<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/09/18/771394.atom"/>
  <meta name="HW.pisa" content="biorxiv;771394v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Evolutionary origins of epidemic potential among human RNA viruses"/>
  <meta name="DC.Identifier" content="10.1101/771394"/>
  <meta name="DC.Date" content="2019-09-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated “Disease X”. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts."/>
  <meta name="DC.Contributor" content="Lu Lu"/>
  <meta name="DC.Contributor" content="Liam Brierley"/>
  <meta name="DC.Contributor" content="Gail Robertson"/>
  <meta name="DC.Contributor" content="Feifei Zhang"/>
  <meta name="DC.Contributor" content="Samantha Lycett"/>
  <meta name="DC.Contributor" content="Donald Smith"/>
  <meta name="DC.Contributor" content="Margo Chase-Topping"/>
  <meta name="DC.Contributor" content="Peter Simmonds"/>
  <meta name="DC.Contributor" content="Mark Woolhouse"/>
  <meta name="article:published_time" content="2019-09-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Evolutionary origins of epidemic potential among human RNA viruses"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated “Disease X”. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;771394v1"/>
  <meta name="citation_id" content="771394v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/771394v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/771394v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/771394v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394.full.pdf"/>
  <meta name="citation_doi" content="10.1101/771394"/>
  <meta name="citation_num_pages" content="44"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="771394"/>
  <meta name="citation_author" content="Lu Lu"/>
  <meta name="citation_author_institution" content="Usher Institute of Population Health Sciences &amp;; Informatics, Ashworth Laboratories, Kings Buildings, University of Edinburgh"/>
  <meta name="citation_author_email" content="lu.lu@ed.ac.uk"/>
  <meta name="citation_author" content="Liam Brierley"/>
  <meta name="citation_author_institution" content="Department of Biostatistics, Waterhouse Building, University of Liverpool"/>
  <meta name="citation_author" content="Gail Robertson"/>
  <meta name="citation_author_institution" content="School of Mathematics, James Clerk Maxwell Building, King’s Buildings, University of Edinburgh"/>
  <meta name="citation_author" content="Feifei Zhang"/>
  <meta name="citation_author_institution" content="Usher Institute of Population Health Sciences &amp;; Informatics, Ashworth Laboratories, Kings Buildings, University of Edinburgh"/>
  <meta name="citation_author" content="Samantha Lycett"/>
  <meta name="citation_author_institution" content="Roslin Institute, University of Edinburgh"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3159-596X"/>
  <meta name="citation_author" content="Donald Smith"/>
  <meta name="citation_author_institution" content="Institute of Evolutionary Biology, Ashworth Laboratories, Kings Buildings, University of Edinburgh"/>
  <meta name="citation_author" content="Margo Chase-Topping"/>
  <meta name="citation_author_institution" content="Usher Institute of Population Health Sciences &amp;; Informatics, Ashworth Laboratories, Kings Buildings, University of Edinburgh"/>
  <meta name="citation_author_institution" content="Roslin Institute, University of Edinburgh"/>
  <meta name="citation_author" content="Peter Simmonds"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author" content="Mark Woolhouse"/>
  <meta name="citation_author_institution" content="Usher Institute of Population Health Sciences &amp;; Informatics, Ashworth Laboratories, Kings Buildings, University of Edinburgh"/>
  <meta name="citation_reference" content="Woolhouse, M. E. J., Rambaut, A. &amp;; Kellam, P. Lessons from Ebola: Improving infectious disease surveillance to inform outbreak management. Sci Transl Med 7, (2015)."/>
  <meta name="citation_reference" content="Wolfe, N. D., Dunavan, C. P. &amp;; Diamond, J. Origins of major human infectious diseases. Nature 447, 279–283, (2007)."/>
  <meta name="citation_reference" content="Olival, K. J. et al. Host and viral traits predict zoonotic spillover from mammals. Nature 546, 646–650, (2017)."/>
  <meta name="citation_reference" content="ME, J. W., Adair, K. &amp;; Brierley, L. RNA Viruses: A Case Study of the Biology of Emerging Infectious Diseases. Microbiol Spectr 1, (2013)."/>
  <meta name="citation_reference" content="Geoghegan, J. L., Senior, A. M., Di Giallonardo, F. &amp;; Holmes, E. C. Virological factors that increase the transmissibility of emerging human viruses. P Natl Acad Sci USA 113, 4170–4175, (2016)."/>
  <meta name="citation_reference" content="Woolhouse, M. E. J., Brierley, L., McCaffery, C. &amp;; Lycett, S. Assessing the Epidemic Potential of RNA and DNA Viruses. Emerg Infect Dis 22, 2037–2044, (2016)."/>
  <meta name="citation_reference" content="Walker, J. W., Han, B. A., Ott, I. M. &amp;; Drake, J. M. Transmissibility of emerging viral zoonoses. PLoS One 13, e0206926, (2018)."/>
  <meta name="citation_reference" content="Woolhouse, M. E. J. &amp;; Brierley, L. Epidemiological characteristics of human-infective RNA viruses. Sci Data 5, 180017, (2018)."/>
  <meta name="citation_reference" content="Lefkowitz, E. J. et al. Virus taxonomy: the database of the International Committee on Taxonomy of Viruses (ICTV). Nucleic Acids Res 46, D708–D717, (2018)."/>
  <meta name="citation_reference" content="Geoghegan, J. L. &amp;; Holmes, E. C. Predicting virus emergence amid evolutionary noise. Open Biol 7, (2017)."/>
  <meta name="citation_reference" content="Carroll, D. et al. The Global Virome Project. Science 359, 872–874, (2018)."/>
  <meta name="citation_reference" content="Carlson, C. J., Zipfel, C. M., Garnier, R. &amp;; Bansal, S. Global estimates of mammalian viral diversity accounting for host sharing. Nat Ecol Evol, (2019)."/>
  <meta name="citation_reference" content="Antia, R., Regoes, R. R., Koella, J. C. &amp;; Bergstrom, C. T. The role of evolution in the emergence of infectious diseases. Nature 426, 658–661, (2003)."/>
  <meta name="citation_reference" content="Pepin, K. M., Lass, S., Pulliam, J. R. C., Read, A. F. &amp;; Lloyd-Smith, J. O. Identifying genetic markers of adaptation for surveillance of viral host jumps. Nat Rev Microbiol 8, 802–813, (2010)."/>
  <meta name="citation_reference" content="Geoghegan, J. L., Duchene, S. &amp;; Holmes, E. C. Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. PLoS Pathog 13, e1006215, (2017)."/>
  <meta name="citation_reference" content="Woolhouse, M., Scott, F., Hudson, Z., Howey, R. &amp;; Chase-Topping, M. Human viruses: discovery and emergence. P Roy Soc B-Biol Sci 367, 2864–2871, (2012)."/>
  <meta name="citation_reference" content="Woolhouse, M. E., Taylor, L. H. &amp;; Haydon, D. T. Population biology of multihost pathogens. Science 292, 1109–1112, (2001)."/>
  <meta name="citation_reference" content="Kuiken, T. et al. Host species barriers to influenza virus infections. Science 312, 394–397, (2006)."/>
  <meta name="citation_reference" content="Holmes, E. C., Rambaut, A. &amp;; Andersen, K. G. Pandemics: spend on surveillance, not prediction. Nature 558, 180–182, (2018)."/>
  <meta name="citation_reference" content="Babayan, S. A., Orton, R. J. &amp;; Streicker, D. G. Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes. Science 362, 577-+, (2018)."/>
  <meta name="citation_reference" content="Friedrich, M. J. WHO’s Blueprint List of Priority Diseases. JAMA 319, 1973, (2018)."/>
  <meta name="citation_reference" content="Benson, D. A., Karsch-Mizrachi, I., Lipman, D. J., Ostell, J. &amp;; Wheeler, D. L. GenBank. Nucleic Acids Res 33, D34–38, (2005)."/>
  <meta name="citation_reference" content="Woolhouse, M. E. J., Brierley, L., McCaffery, C. &amp;; Lycett, S. Assessing the Epidemic Potential of RNA and DNA Viruses. Emerg Infect Dis 22, 2037–2044, (2016)."/>
  <meta name="citation_reference" content="Lefkowitz, E. J. et al. Virus taxonomy: the database of the International Committee on Taxonomy of Viruses (ICTV). Nucleic Acids Res 46, D708–D717, (2018)."/>
  <meta name="citation_reference" content="Drummond, A. J. &amp;; Rambaut, A. BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol 7, 214, (2007)."/>
  <meta name="citation_reference" content="Simmonds, P., Aiewsakun, P. &amp;; Katzourakis, A. Prisoners of war - host adaptation and its constraints on virus evolution. Nat Rev Microbiol, (2018)."/>
  <meta name="citation_reference" content="Edgar, R. C. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32, 1792–1797, (2004)."/>
  <meta name="citation_reference" content="Faria, N. R., Suchard, M. A., Rambaut, A., Streicker, D. G. &amp;; Lemey, P. Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints. Philos T R Soc B 368, 20120196, (2013)."/>
  <meta name="citation_reference" content="Parker, J., Rambaut, A. &amp;; Pybus, O. G. Correlating viral phenotypes with phylogeny: accounting for phylogenetic uncertainty. Infect Genet Evol 8, 239–246, (2008)."/>
  <meta name="citation_reference" content="Baele, G. et al. Improving the Accuracy of Demographic and Molecular Clock Model Comparison While Accommodating Phylogenetic Uncertainty. Molecular Biology and Evolution 29, 2157–2167, (2012)."/>
  <meta name="citation_reference" content="Edwards, C. J. et al. Ancient hybridization and an Irish origin for the modern polar bear matriline. Curr Biol 21, 1251–1258, (2011)."/>
  <meta name="citation_reference" content="O’Brien, J. D., Minin, V. N. &amp;; Suchard, M. A. Learning to count: robust estimates for labeled distances between molecular sequences. Mol Biol Evol 26, 801–814, (2009)."/>
  <meta name="citation_reference" content="Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313, (2014)."/>
  <meta name="citation_reference" content="Brooks, M. E. et al. glmmTMB Balances Speed and Flexibility Among Packages for Zero-inflated Generalized Linear Mixed Modeling. R J 9, 378–400, (2017)."/>
  <meta name="citation_reference" content="Sharp, P. M. &amp;; Hahn, B. H. Origins of HIV and the AIDS pandemic. Cold Spring Harb Perspect Med 1, a006841, (2011)."/>
  <meta name="twitter:title" content="Evolutionary origins of epidemic potential among human RNA viruses"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated “Disease X”. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts."/>
  <meta name="og-title" property="og:title" content="Evolutionary origins of epidemic potential among human RNA viruses"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/771394v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated “Disease X”. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-09-18"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Evolutionary origins of epidemic potential among human RNA viruses | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-912396 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated “Disease X”. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">Ebolavirus in 2014-15 and SARS coronavirus in 2003 are examples of infectious disease epidemics resulting from the emergence of RNA viruses from non-human reservoirs
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. However, the majority of spill-overs from non-human reservoirs involve viruses that do not spread in human populations (e.g., the strictly zoonotic orthohantaviruses and bat lyssaviruses)
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. There have been various studies of ecological predictors of virus infectivity to humans (e.g. 
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, 
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>) but less attention has been paid to transmissibility in human populations even though this trait is integral to epidemic potential
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. Here, we aim to identify phylogenetic and ecological characteristics of RNA virus lineages that are associated with the evolution of human transmissibility.
    </p>
    <p id="p-4">Members of over 200 non-reverse transcribing (non-RT) RNA virus species (i.e. excluding retroviruses) are known to infect humans
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </sup>. Human-infective viruses are found in 20 of the 22 currently recognised families of non-RT RNA viruses of mammals and/or birds (the exceptions being the 
     <em>Arteriviridae</em> and 
     <em>Birnaviridae</em>). Although members of only 82 human-infective non-RT RNA virus species (40%) are known to transmit within human populations (other than by vertical or iatrogenic routes)
     <sup>
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>
     </sup>, these too have broad taxonomic distributions, being found in 19 families (the 
     <em>Bornaviridae</em> having no human-transmissible species).
    </p>
    <p id="p-5">New human RNA viruses are still discovered regularly
     <sup>
      <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>
     </sup> and it is likely that a large number of currently unknown RNA viruses are circulating in non-human mammals and birds, some of which may have the potential to infect and be transmitted by humans
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>. All known human RNA viruses form part of purely mammal or mammal/bird virus clades
     <sup>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>
     </sup> and there is no evidence of human-infective RNA viruses emerging from other kinds of host
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>.
    </p>
    <p id="p-6">Human transmissibility (here, broadly defined as spread directly from person to person or via an indirect route, such as environmental contamination or an arthropod vector) is a necessary but not sufficient condition for a virus to have epidemic potential. Pathogens that are transmissible but with a basic reproduction number (
     <em>R</em>0) less than 1 in humans (e.g. Lassa virus) are restricted to self-limiting outbreaks and their long-term persistence requires a non-human reservoir. Previous studies have addressed the ecology
     <sup>
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>
     </sup> and evolution
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup> of the transition between 
     <em>R</em>0&lt;;1 and 
     <em>R</em>0&gt;;1; here, we focus on the evolution of human transmissibility, i.e. 
     <em>R</em>0&gt;;0. We categorised RNA viruses into three infectivity/transmissibility (IT) levels
     <sup>
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>
     </sup>: Level 1 (L1) viruses are infective to non-human mammals and/or birds but not to humans; L2 viruses are human-infective but not human-transmissible; and L3/4 viruses are human-infective and transmissible (combining viruses with 0&lt;;
     <em>R</em>0&lt;;1 (L3) and those with 
     <em>R</em>0&gt;;1 (L4)).
    </p>
    <p id="p-7">Here, we propose a conceptual model of RNA virus emergence where that latent capacities to infect and transmit between humans are traits inherent to some L1 virus lineages circulating in non-human reservoirs (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); this has been termed “fortuitous adaptation”
     <sup>
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>. This view emphasizes the role of virus genetic diversity within the reservoir, with humans acting as “sentinels”
     <sup>
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>
     </sup>, revealing the presence of L2 or L3/4 viruses through contact with the reservoir. New L3/4 viruses can emerge by any of three routes: i) directly from viruses in a non-human reservoir (L1) at a rate given by parameter 
     <em>a</em>; ii) step-wise via a strictly zoonotic (L2) phase in a non-human reservoir or, less probably, in infected humans at rates given by parameters 
     <em>b</em> and 
     <em>c</em>; or iii) diversification of human-transmissible (L3/4) virus lineages within humans or, possibly, a non-human reservoir. We note that human infectivity and transmissibility are products of convergent evolution, with their molecular and genetic determinants differing across virus taxa
     <sup>
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>
     </sup>, and that the rates of evolution of these traits represented by parameters 
     <em>a</em>, 
     <em>b</em> and 
     <em>c</em> are likely to vary between lineages.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Conceptual model of non-RT RNA virus emergence into human populations. Viruses circulating in the non-human reservoir may not be infective to humans (L1) or have the potential to infect or infect and transmit between humans (L2 and L3/4 respectively). Virus lineages in the reservoir may acquire (thick grey arrows showing rates a, b and c) or lose (thin grey arrows) the capacities to infect and/or transmit in humans through genetic drift [[#x2013]] these traits are not under direct selection in the non-human reservoir. L2 and L3/4 viruses in non-human hosts may cross the species barrier and enter human populations. L3/4 (but not L2) viruses may then spread and evolve independently in the human population (multiple grey arrows). Phylogenetic analyses identify past transitions between L1 and L3/4, L1 and L2 and L2 and L3/4 viruses and can be used to estimate the relative values of parameters a, b and c." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Conceptual model of non-RT RNA virus emergence into human populations. Viruses circulating in the non-human reservoir may not be infective to humans (L1) or have the potential to infect or infect and transmit between humans (L2 and L3/4 respectively). Virus lineages in the reservoir may acquire (thick grey arrows showing rates &lt;;em&gt;;a&lt;;/em&gt;;, &lt;;em&gt;;b&lt;;/em&gt;; and &lt;;em&gt;;c&lt;;/em&gt;;) or lose (thin grey arrows) the capacities to infect and/or transmit in humans through genetic drift [[#x2013]] these traits are not under direct selection in the non-human reservoir. L2 and L3/4 viruses in non-human hosts may cross the species barrier and enter human populations. L3/4 (but not L2) viruses may then spread and evolve independently in the human population (multiple grey arrows). Phylogenetic analyses identify past transitions between L1 and L3/4, L1 and L2 and L2 and L3/4 viruses and can be used to estimate the relative values of parameters &lt;;em&gt;;a&lt;;/em&gt;;, &lt;;em&gt;;b&lt;;/em&gt;; and &lt;;em&gt;;c&lt;;/em&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F1.medium.gif" width="440" height="357"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F1.medium.gif" width="440" height="357"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-8" class="first-child">Conceptual model of non-RT RNA virus emergence into human populations. Viruses circulating in the non-human reservoir may not be infective to humans (L1) or have the potential to infect or infect and transmit between humans (L2 and L3/4 respectively). Virus lineages in the reservoir may acquire (thick grey arrows showing rates 
       <em>a</em>, 
       <em>b</em> and 
       <em>c</em>) or lose (thin grey arrows) the capacities to infect and/or transmit in humans through genetic drift – these traits are not under direct selection in the non-human reservoir. L2 and L3/4 viruses in non-human hosts may cross the species barrier and enter human populations. L3/4 (but not L2) viruses may then spread and evolve independently in the human population (multiple grey arrows). Phylogenetic analyses identify past transitions between L1 and L3/4, L1 and L2 and L2 and L3/4 viruses and can be used to estimate the relative values of parameters 
       <em>a</em>, 
       <em>b</em> and 
       <em>c</em>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <p id="p-9">For phylogenetic analysis we used polymerase protein sequences of human and non-human RNA viruses classified into 39 genera (see Supplementary Materials). We only considered genera containing human-infective virus species, noting that not all eligible genera had sufficient sequences available for analysis. We also excluded all retroviruses 
     <em>a priori</em> and the 
     <em>Influenza A virus</em> genus 
     <em>a posteriori</em> from the main analysis (see Supplementary Materials) but, given their importance, we performed separate analyses of these taxa.
    </p>
    <p id="p-10">Of the 1755 polymerase gene sequences used in our main study, 737 were from L1 viruses, 466 were from L2 viruses (90 from humans, 376 from non-human hosts) and 552 from L3/4 viruses (400 from humans, 152 from non-human hosts). These included virus sequences from natural infections of all available mammal or bird hosts. A plot of the cumulative number of sequences from L1, L2 and L3/4 viruses in our data set over time (
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1</a>), reveals the rapid recent increase in numbers of L1 virus sequences that makes our study feasible at this time. The sequences represent 79% and 81% respectively of L2 and L3/4 non-RT RNA virus species currently listed by the International Committee for the Taxonomy of Viruses (ICTV)
     <sup>
      <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>
     </sup>. Within some virus species, we differentiated between recognised subtypes that differ in infectivity/transmissibility (IT) level (see Data File 4).
    </p>
    <p id="p-11">We estimated the phylogenies of the mammal/bird virus clade for each of the 39 non-RT RNA virus genera (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We then used a Bayesian discrete trait approach (see Supplementary Materials) to estimate IT level for every node within those phylogenies. Our primary interest was tree topology; we did not attempt to date the nodes. The phylogenies indicated that for at least 22/39 genera containing human-infective viruses (L2 and/or L3/4), human infectivity is unlikely (P&lt;;0.20) to be an ancestral trait, and for at least 22/31 genera containing human-transmissible (L3/4) viruses, transmissibility is unlikely (P&lt;;0.20) to be an ancestral trait (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3a</a>, 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">S2</a>, Data File 1). In total, we identified 77 distinct lineages of human-infective (L2 and/or L3/4) and 57 of human-transmissible (L3/4) viruses (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, Data File 1).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Bayesian maximum clade credibility (MCC) trees for members of 39 virus genera showing the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type. Raw data are provided in Data File 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Bayesian maximum clade credibility (MCC) trees for members of 39 virus genera showing the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type. Raw data are provided in Data File 5.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F2.medium.gif" width="310" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F2.medium.gif" width="310" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-12" class="first-child">Bayesian maximum clade credibility (MCC) trees for members of 39 virus genera showing the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type. Raw data are provided in Data File 5.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Outputs of discrete trait analyses. a) Frequency of estimated infection/transmission (IT) level (L1, L2 or L3/4) for the ancestral nodes of the mammal/bird clade in each genus (see Figure 2). b) Frequency of forward transitions by virus genus and whether enveloped. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan). N=96. Vector-borne genera indicated with asterisks. c) As b by virus family. d) As b by virus genome type. e) As b by enveloped/non-enveloped. f) As b by vector-borne/non-vector-borne. g) Boxplots showing distribution of genus-level, loge-transformed relative transition rates for level transitions: L1-L2 (N=21), L1-L3/4 (N=26), L2-L3/4 (N=18). The rates used in the plot are shown in Figure S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Outputs of discrete trait analyses. &lt;;strong&gt;;a&lt;;/strong&gt;;) Frequency of estimated infection/transmission (IT) level (L1, L2 or L3/4) for the ancestral nodes of the mammal/bird clade in each genus (see Figure 2). &lt;;strong&gt;;b&lt;;/strong&gt;;) Frequency of forward transitions by virus genus and whether enveloped. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan). N=96. Vector-borne genera indicated with asterisks. &lt;;strong&gt;;c&lt;;/strong&gt;;) As &lt;;strong&gt;;b&lt;;/strong&gt;; by virus family. &lt;;strong&gt;;d&lt;;/strong&gt;;) As &lt;;strong&gt;;b&lt;;/strong&gt;; by virus genome type. &lt;;strong&gt;;e&lt;;/strong&gt;;) As &lt;;strong&gt;;b&lt;;/strong&gt;; by enveloped/non-enveloped. &lt;;strong&gt;;f&lt;;/strong&gt;;) As &lt;;strong&gt;;b&lt;;/strong&gt;; by vector-borne/non-vector-borne. &lt;;strong&gt;;g&lt;;/strong&gt;;) Boxplots showing distribution of genus-level, loge-transformed relative transition rates for level transitions: L1-L2 (N=21), L1-L3/4 (N=26), L2-L3/4 (N=18). The rates used in the plot are shown in Figure S6.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F3.medium.gif" width="361" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F3.medium.gif" width="361" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-13" class="first-child">Outputs of discrete trait analyses. 
       <strong>a</strong>) Frequency of estimated infection/transmission (IT) level (L1, L2 or L3/4) for the ancestral nodes of the mammal/bird clade in each genus (see 
       <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). 
       <strong>b</strong>) Frequency of forward transitions by virus genus and whether enveloped. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan). N=96. Vector-borne genera indicated with asterisks. 
       <strong>c</strong>) As 
       <strong>b</strong> by virus family. 
       <strong>d</strong>) As 
       <strong>b</strong> by virus genome type. 
       <strong>e</strong>) As 
       <strong>b</strong> by enveloped/non-enveloped. 
       <strong>f</strong>) As 
       <strong>b</strong> by vector-borne/non-vector-borne. 
       <strong>g</strong>) Boxplots showing distribution of genus-level, loge-transformed relative transition rates for level transitions: L1-L2 (N=21), L1-L3/4 (N=26), L2-L3/4 (N=18). The rates used in the plot are shown in 
       <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S6</a>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-14">We found evidence of 86 transitions involving gain of human infectivity and 52 involving gain of human transmissibility within the 39 virus genera (
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1</a>, 
     <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a>, 
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">S3</a>, Data File 2). Human infectivity and transmissibility are likely to have evolved more than once in 20 and 17 genera respectively (
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>). We note that we also found evidence of backward transitions: seven involving loss of human transmissibility and 11 involving loss of human infectivity (Data File 3). We compared our estimated numbers of transitions with outputs from both Markov jumps and a parsimony reconstruction method (see Supplementary Materials). These checks confirm that our estimates of the total numbers of transitions are well supported by the phylogenetic data, even though the precise location of a transition within a phylogeny could not always be estimated with high confidence.
    </p>
    <p id="p-15">One-third of the genera (13/39) have both L1 and L3/4 but no known, extant L2 viruses. Among genera containing L2 viruses, a L1-L2 transition was followed by a L2-L3/4 transition in just four virus lineages (from the genera 
     <em>Phlebovirus</em>, 
     <em>Orthonairovirus</em>, 
     <em>Henipavirus</em> and 
     <em>Orthohepevirus</em>). Overall, we estimate that direct evolution from L1 virus lineages is the most likely origin of 74% of the 57 L3/4 lineages, with just 17% from L2 virus lineages and 9% unknown (
     <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figures 2</a>, 
     <a id="xref-fig-7-2" class="xref-fig" href="#F7">S3</a>). Eight genera show species-level diversification within L3/4 lineages, potentially within human populations, but this accounts for only 21/70 (30%) of L3/4 RNA virus species in our data set (
     <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figures 2</a>, 
     <a id="xref-fig-7-3" class="xref-fig" href="#F7">S3</a>).
    </p>
    <p id="p-16">The relative frequencies of L1-L2, L1-L3/4 and L2-L3/4 transitions varied across genera, families and genome types (
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3b-d</a>, 
     <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S2</a>). The odds of L1-L3/4 relative to L1-L2 transitions were lower for (-)ssRNA versus (+)ssRNA or dsRNA (
     <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2d</a>), for enveloped versus non-enveloped (
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3e</a>) and for vector-borne versus non-vector-borne viruses (
     <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3f</a>) with unadjusted ORs=0.15, 0.23 and 0.23 respectively, all P&lt;;0.05. However, these traits are correlated and multivariable analysis suggested that the dominant effect was the low odds of L1-L3/4 transitions among (-)ssRNA viruses relative to dsRNA and (+)ssRNA viruses (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We note that L2-L3/4 transitions were highly over-represented (unadjusted OR=11.5, P&lt;;0.001) among vector-borne viruses (
     <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3f</a>), occurring in only three non-vector-borne virus genera (
     <em>Henipavirus</em>, 
     <em>Metapneumovirus</em> and 
     <em>Orthohepevirus</em>).
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/914387/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/914387/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/914387" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <p id="p-17" class="first-child">Ancestor node traits as predictors of IT level transitions.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-18">L1-L2, L1-L3/4 and L2-L3/4 transitions had similar distributions of relative node depth (
     <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S3</a>, 
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S4</a>). There was no indication that L2 viruses tended to arise earlier in lineages than L3/4 viruses. Nor was there a consistent trend towards evolutionary recent (as measured by relative node depth) increases in the number of viruses entering human populations. We also estimated the instantaneous transition rates between IT level (see Supplementary Materials). The rates of L2-L3/4 transitions were not higher than the rates of L1-L3/4 transitions (
     <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3g</a>, 
     <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S4</a>). This implies that the distributions of values of parameters 
     <em>a</em> and 
     <em>c</em> in our conceptual model (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) are similar. The estimated rates were highly variable across virus taxa (
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figures S5</a>, 
     <a id="xref-fig-10-2" class="xref-fig" href="#F10">S6</a>) but this was not associated with genome type/enveloped or being vector borne (
     <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table S4</a>).
    </p>
    <p id="p-19">In contrast to IT levels, host categories are not mutually exclusive traits of a virus species/subtype, and for some genera (e.g. 
     <em>Ebolavirus</em>) it is likely that we are missing sequences from the reservoir host(s). Therefore, we did not attempt a discrete traits analysis for host range but compared the known non-human hosts for L2 and L3/4 viruses at the species/subtype and genus levels (see Supplementary Materials). At the species/subtype level human viruses also found in non-human primates were most likely to be L3/4 rather than L2; those also found in birds were least likely (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). At the genus level the pattern was similar (Spearman’s rank correlation of coefficient estimates by host category = +0.89, P&lt;;0.01) but the confidence intervals are wider due to the smaller sample sizes (
     <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure S7</a>). These results are consistent with primates being disproportionately frequent ancestral hosts of L3/4 virus lineages, but there are too few virus sequences from wild primates available (74 in our data set) to test that hypothesis more formally through phylogenetic analysis.
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Human-transmissible virus in nonhuman host. a) The proportions of virus species of different IT levels being known to infect a given non-human host category (six orders of mammal and the Class Aves). b) Results for GLMMs with binary responses for association between viruses within a species/subtype being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves of L3/4 virus species relative to L2 virus species. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Genus and family are included as random effects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Human-transmissible virus in nonhuman host. a) The proportions of virus species of different IT levels being known to infect a given non-human host category (six orders of mammal and the Class Aves). b) Results for GLMMs with binary responses for association between viruses within a species/subtype being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves of L3/4 virus species relative to L2 virus species. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Genus and family are included as random effects.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F4.medium.gif" width="295" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F4.medium.gif" width="295" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-20" class="first-child">Human-transmissible virus in nonhuman host. a) The proportions of virus species of different IT levels being known to infect a given non-human host category (six orders of mammal and the Class Aves). b) Results for GLMMs with binary responses for association between viruses within a species/subtype being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves of L3/4 virus species relative to L2 virus species. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Genus and family are included as random effects.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-21">In keeping with earlier work
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup>, we used RNA-dependent RNA polymerase protein-based phylogenies in our main analysis. However, we tested the possibility that alternative phylogenies would generate different numbers and distributions of transitions between traits by constructing alternatives trees based on surface protein sequences (
     <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure S8</a>). We obtained results that differed minimally from our main analysis, most importantly that an estimated 71% of L3/4 lineages have L1 ancestors (see Data File 9).
    </p>
    <p id="p-22">We recognise that there will be significant gaps in our knowledge of mammal/bird non-RT RNA virus diversity: new species are routinely being identified
     <sup>
      <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>
     </sup> and virus genome sequences are accumulating rapidly (
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure S1</a>). Our selection procedure resulted in a data set with only 28% of virus sequences from human hosts, but it is still likely that the phylogenetic diversity of viruses from non-human hosts is greatly underestimated relative to that from humans
     <sup>
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, 
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
     </sup>, noting that for many mammal/bird taxa no RNA viruses at all have yet been reported. There will also be gaps in our knowledge of IT level: some viruses originally classified as L1 have been subsequently found to human infective; and some viruses originally classified as L2 have been subsequently found to be human transmissible as epidemiological data accumulates
     <sup>
      <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>
     </sup>. As new virus sequences are added and IT levels are assigned or (occasionally) re-assigned it is entirely possible that the estimated ancestors of some L3/4 viruses will change. However, we anticipate that the impact of such changes across the whole data set will be to strengthen our main conclusions. This is because we anticipate that far more L1 viruses will be added than L2, and more L2 than L3/4, continuing the current trend (
     <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure S1</a>). For this reason, we consider our estimate that 74% of L3/4 viruses arose from L1 lineages to be conservative.
    </p>
    <p id="p-23">We did not include two important taxa in our main analysis. The 
     <em>Influenza A virus</em> genus was excluded 
     <em>a posteriori</em> because of a very weak phylogenetic signal for IT level (
     <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure S9</a>). On inspection of the HA gene phylogeny it is apparent that although human infectivity and transmissibility have evolved in only a few lineages (HA types) there is considerable variability in IT level within those lineages, which would generate inflated counts of IT level change (
     <a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure S10</a>). Nonetheless, the phylogeny is broadly consistent with the pattern that most L3/4 lineages evolve from L1 lineages. We excluded Retroviridae from our main analysis 
     <em>a priori</em> because they cause chronic infections that allow far more time for within-host evolution prior to transmission
     <sup>
      <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>
     </sup>. Consistent with this, we find that transmissible human lentiviruses (lineages of HIV-1 and HIV-2) are most likely have evolved through L1-L2-L3/4 transitions rather than L1-L3/4 (
     <a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure S11</a>).
    </p>
    <p id="p-24">Our results show that at least 74% of L3/4 non-RT RNA virus lineages (e.g. human coronaviruses 229E and NL63, measles virus, human respiratory syncytial virus, hepatitis C virus and Aichi virus A) are most likely to have emerged directly from reservoirs of L1 viruses. Stepwise emergence to L2 then L3/4 occurs infrequently and mainly in vector-borne viruses. This result is consistent with epidemiological observation: there are no well-supported examples of any of the &gt;;100 species of L2 non-RT RNA viruses evolving the capacity to transmit in human populations
     <sup>
      <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>
     </sup>. We also estimate that diversification within L3/4 lineages has contributed significantly to human RNA virus diversity, accounting for 30% of species (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). However, a larger fraction, at least 51%, has been generated directly from L1 lineages.
    </p>
    <p id="p-25">The evolution of a new (and extant) lineage of L3/4 non-RT RNA viruses appears a relatively infrequent event; there are just 57 instances over the entire evolutionary history of the genera considered here, with no suggestion of any recent (on an evolutionary time scale) increase in frequency (
     <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S4</a>). Unexpectedly, the same applies to L2 viruses; these are generated from L1 viruses at similar rates as L3/4 viruses and at similar relative node depths (
     <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figures 3g</a> and 
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">S4</a>, 
     <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table S3</a>). We have previously suggested an alternative model where the L2 trait is easily evolved and easily lost
     <sup>
      <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>
     </sup>, but this model is not supported by the analysis reported here. Importantly, these findings do not preclude extant pools of L2 and L3/4 viruses that have yet to be recognised and may not yet have had the opportunity to enter human populations
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </sup>, but they do imply that the drivers for the emergence of novel human viruses, with or without epidemic potential, would be ecological rather than evolutionary.
    </p>
    <p id="p-26">That there was no detectable difference in the relative rates at which L3/4 viruses emerge from L2 and non-human (L1) virus lineages (
     <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3g</a>) implies that the distributions of parameters 
     <em>a</em> and 
     <em>c</em> in our conceptual model (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) are similar. Given that, an obvious explanation for the rarity of L2-L3/4 transitions is that the pool of L1 viruses is much larger than that of L2 viruses, as is widely supposed
     <sup>
      <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>, 
      <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>
     </sup>. Moreover, most L2-L3/4 transitions involve vector-borne viruses. Vector-borne pathogens tend to have wide host ranges
     <sup>
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>, possibly due to the transmission route providing direct access to the blood system. There is a substantial deficit of vector-borne L1 viruses in our database (Data File 4), consistent with the L2 pool being relatively larger for vector-borne viruses, thus explaining why we found most L2-L3/4 transitions in this category. In addition, we found that L1 (-)ssRNA virus lineages are relatively more likely to generate L2 than L3/4 viruses (
     <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and, once this was accounted for, there was no additional effect of a virus having an envelope, as has been suggested previously
     <sup>
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
     </sup>. Nonetheless, this is a large group and (-)ssRNA virus lineages are an important source of L3/4 viruses (
     <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3d</a>).
    </p>
    <p id="p-27">The absence of a clear association between human infectivity and human transmissibility may reflect the key role that cell receptors play in determining a virus’s capacity to infect and be transmitted by humans
     <sup>
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, 
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </sup>. Cell receptor usage varies between virus genera and sometimes within genera
     <sup>
      <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">9</a>
     </sup>. Host switching (to humans, or to any other new host) is facilitated by a virus using a cell receptor with an amino acid sequence that is conserved between hosts
     <sup>
      <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>
     </sup>. However, if the receptor has a different tissue distribution in humans then infectivity may not equate with transmissibility
     <sup>
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>
     </sup>. Evolutionary shifts in receptor usage can therefore lead to changes in human infectivity (L1 to L2) or transmissibility (L2 to L3/4) or both (L1 to L3/4).
    </p>
    <p id="p-28">Our study supports a recent proposal for a large-scale survey of viral diversity in non-human reservoirs, the Global Virome Project (GVP)
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>. Though this idea is controversial and would be costly to implement
     <sup>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup>, its relevance is underlined by our finding that most species of human-transmissible viruses (L3/4) evolve from mammal/bird RNA virus lineages not known to be infective to humans (L1), coupled with the expectation that the great majority of mammal and bird viruses are still unrecognised
     <sup>
      <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>
     </sup>. We note recent progress in using machine learning to predict host range from sequence data
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
     </sup> and the potential of this kind of approach to help identify human-infective and human-transmissible viruses even in the absence of human cases.
    </p>
    <p id="p-29">Finally, our study also provides empirical support for, and underlines the public health importance of, “Disease X”, the scenario that a future serious international epidemic might be caused by a pathogen taxon not currently known to affect humans
     <sup>
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup>. We find that the ability of RNA viruses to transmit between humans most frequently evolves in a manner consistent with the “Disease X” model of emerging infections.
    </p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Funding</h2>
    <p id="p-31">This work was supported by the Wellcome Trust (VIZIONS, ref. 093724), the European Commission H2020 programme (COMPARE, contract number 643476) and BBSRC Institute Strategic Programme Grant: Control of Infectious Diseases (BBS/E/D/20002173).</p>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Competing interests</h2>
    <p id="p-32">None declared.</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Author contributions</h2>
    <p id="p-33">L.L., S.L., P.S. and M.W. designed the study. L.L., L.B. and F.Z. assembled the data sets. L.L., G.R. and M.C-T performed the analyses. D.S. validated some of the phylogenies. L.L. and M.W. wrote the manuscript. All authors reviewed, commented on and approved the manuscript.</p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Data availability</h2>
    <p id="p-34">The raw data used in these analyses are all freely available in the Data Files. These include links to sequence data in GenBank.</p>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">SUPPLEMENTARY MATERIALS</h2>
    <div class="section" id="sec-9">
     <h3>Materials and Methods</h3>
     <div class="section" id="sec-10">
      <h4>Data</h4>
      <p id="p-35">We compiled a data pool of 7488 polymerase (or functional equivalent) gene sequences from non-RT RNA viruses found naturally (i.e. excluding deliberate laboratory exposures) in humans, other mammals or birds and reported in the NCBI GenBank database
       <sup>
        <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">1</a>
       </sup>. Inclusion criteria were that complete or nearly complete polymerase protein sequences were available, together with information of natural infected host (mammal or bird only) species and isolation date (year). No sequence data were available for 30 otherwise eligible human-infective virus species and these could not be included in the analysis – all of these were rare viruses, the majority of which have not been reported in humans for at least 10 years. For some zoonotic virus species, sequences were only available from non-human hosts. To reduce sampling bias, especially of human viruses, we then subsampled the dataset to keep no more than five sequences per host species per virus species/subtype and to avoid clusters of sequences from single outbreaks.
      </p>
      <p id="p-36">We linked individual sequences to infectivity/transmissibility (IT) level: L1, L2 or L3/4 as defined in the main text. IT level was attributed using an updated version of a recently published epidemiological data set but six viruses that are transmitted between humans only by vertical and/or iatrogenic routes
       <sup>
        <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2</a>
       </sup> were categorised here as L2. Four virus species (
       <em>Norovirus, Sapovirus, Orthohepevirus A, Aichivirus A</em>) for which there are recognised subtypes that differ in IT level were included as subtypes rather than single species.
      </p>
      <p id="p-37">Seven virus genera (
       <em>Coltivirus</em>, 
       <em>Erbovirus</em>, 
       <em>Marburgvirus</em>, 
       <em>Seadornavirus</em>, 
       <em>Thogotovirus</em>, 
       <em>Tibrovirus</em> and 
       <em>Torovirus</em>) were excluded because there were too few sequences (&lt;;15) for analysis. Three genera (
       <em>Deltavirus</em>, 
       <em>Rubivirus</em> and 
       <em>Salivirus</em>) provided no useful information because they each contain a single L3/4 species known only from humans. Two genera (
       <em>Influenza B virus</em> and 
       <em>Influenza C virus</em>) had too few non-L3/4 sequences available (one and three respectively). The 
       <em>Influenza A virus</em> genus was considered separately from the main analysis as BaTS tests indicated that there is a uniquely poor association between IT level and polymerase phylogeny for this taxon (see below). Retroviridae (three genera with human-infective species) were excluded 
       <em>a priori</em> from our main analysis given their markedly different biology
       <sup>
        <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">3</a>
       </sup>. However, we carried out a separate analysis for the one genus – 
       <em>Lentivirus</em> – with sufficient 
       <em>pol</em> protein sequences available.
      </p>
      <p id="p-38">Our main data set comprised 1755 sequences from 39 non-RT RNA virus genera (31 with human-transmissible (L3/4) viruses) for analysis. We linked these sequences to the host category from which the virus was isolated, as nine taxonomic orders of the Class Mammalia (distinguishing humans and non-human primates), plus marsupials and collectively the Class Aves (noting that &lt;;5 sequences were available from Eulipotyphla, Lagomorpha, Scandentia and marsupials, with the remaining mammalian orders not represented at all). One multi-species genus, 
       <em>Cosavirus</em>, was represented only by sequences from human hosts. Links to the sequences and associated metadata are provided in Data File 4.
      </p>
     </div>
     <div class="section" id="sec-11">
      <h4>Phylogenetic analysis</h4>
      <p id="p-39">We performed parallel phylogenetic analyses using the BEAST software package
       <sup>
        <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">4</a>
       </sup> (V1.8.2). We were primarily interested in tree topology; branch lengths scale to numbers of amino acid substitutions and we did not attempt to date the nodes – such estimates are likely to be unreliable given the long time scales involved
       <sup>
        <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">5</a>
       </sup>. For every virus genus, polymerase gene sequences were translated to amino acid and aligned with MUSCLE V3.8.4
       <sup>
        <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">6</a>
       </sup>, and we reconstructed Bayesian time-scaled phylogenies for each of the 39 genera and perform discrete traits analysis with respect to IT level. Our methodology is comparable to the approach used in Ref.
       <sup>
        <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">7</a>
       </sup>.
      </p>
      <p id="p-40">We tested the associations between IT levels and polymerase trees per genus using Phylogeny-trait association test (BaTS)
       <sup>
        <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">8</a>
       </sup>, inspecting both the Association index (AI) and Parsimony score (PS) between genera. We observed that the 
       <em>Influenza A virus</em> genus showed evidence of a uniquely weak phylogenetic signal (
       <a id="xref-fig-13-2" class="xref-fig" href="#F13">Fig. S9a</a>) so this genus was subsequently considered separately from the main analysis. Similar results were obtained using phylogenies based on surface proteins as described below (
       <a id="xref-fig-13-3" class="xref-fig" href="#F13">Fig. S9b</a>).
      </p>
      <p id="p-41">We estimated the phylogenies of the sequences using a Bayesian Markov chain Monte Carlo (MCMC) method that was implemented using the Bayesian evolutionary analysis in BEAST. Different combinations of substitution models, clock models and population size models were evaluated by using the path sampling (PS) to estimate marginal likelihoods
       <sup>
        <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">9</a>
       </sup>. The best fitting model was a WAG model with a gamma distribution (G) across sites as the substitution model, with an uncorrelated log-normal relaxed molecular clock model and with a constant size coalescent process or Yule process prior over the phylogenies. Here, we allowed the branch length to be scaled by substitution per site rather than by time (with ucld.mean equal to 1). The MCMC chains were run for 100 million iterations with sub-sampling every 10,000 iterations and 10% burn-in and at least two replicates were performed per genus. MCMC convergence and effective sample size of parameter estimates were evaluated using Tracer 1.5 (
       <a href="http://beast.bio.ed.ac.uk">http://beast.bio.ed.ac.uk</a>). Summary Maximum Clade Credibility phylogenies were created from the posterior samples of trees using TreeAnnotator, and a further sample of 1000 trees was extracted from each genus-level posterior sample for additional processing. We validated our phylogenies of the mammal/bird virus clade by comparing their topologies with the representative phylogenies on the family/genus level published by ICTV
       <sup>
        <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">3</a>
       </sup>, noting that we are missing sequences/species that did not meet our inclusion criteria.
      </p>
      <p id="p-42">We applied asymmetric discrete trait models using BEAST to estimate IT level over each genus-level posterior sample of 1000 trees
       <sup>
        <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">10</a>
       </sup>. The instantaneous transition probabilities between IT levels (equivalent to the relative rates matrix) were estimated simultaneously. All the Bayesian phylogenies mapped by IT level traits are provided in Data File 5.
      </p>
      <p id="p-43">For each genus-level phylogeny we obtained the estimated the probability distributions for IT level at the root ancestral node. Transitions were identified as a change in the most probable IT level estimated for any two adjacent nodes on the phylogeny (
       <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2</a>). We counted all occurrences within each tree of the three possible types of ‘forward’ transition: L1 to L2 (non-human virus acquiring human infectivity); L1 to L3/4 (non-human virus acquiring human infectivity and transmissibility); and L2 to L3/4 (infective but non-transmissible virus acquiring human transmissibility). Instances of the possible ‘backward’ transitions (L2 to L1, L3/4 to L1 and L3/4 to L2) were also recorded.
      </p>
      <p id="p-44">We also applied Markov Jumps 
       <sup>
        <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">11</a>
       </sup> with discrete trait model to estimate and validate the expected number of level transitions in the phylogenetic trees (an example XML file shown in Data File 6). We obtained excellent agreement between median estimates of numbers of forward transitions using Markov jumps and counts estimated from our discrete traits analysis (
       <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S1</a>, 
       <a id="xref-fig-16-1" class="xref-fig" href="#F16">Fig. S12a</a>). We were also able to reproduce the patterns of variation between virus types and taxa with minimal discrepancies (compare 
       <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figs 2b-f</a> and 
       <a id="xref-fig-17-1" class="xref-fig" href="#F17">S13a-e</a>). Outputs are compared in full in Data File 7.
      </p>
      <p id="p-45">In addition, we applied parsimony reconstruction models to trace trait evolution, using Mesquite V 3.5.1 (
       <a href="http://www.mesquiteproject.org">http://www.mesquiteproject.org</a>), with the input phylogenies being generated with the protein sequences of each virus genus via maximum likelihood method, using RaxML V 8 (WAG+G, bootstraps n=1000)
       <sup>
        <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">12</a>
       </sup>. We obtained excellent agreement between mean estimates of numbers of forward transitions using parsimony analysis and counts estimated from our discrete traits analysis (
       <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table S1</a>, 
       <a id="xref-fig-16-2" class="xref-fig" href="#F16">Fig. S12b</a>), noting that the parsimony approach does not identify individual nodes involved in transitions. Outputs are compared in full in Data File 7.
      </p>
      <p id="p-46">To test the robustness of our results to the choice of RNA-dependent RNA polymerase protein sequences to construct our phylogenies we compared results using different proteins. We used 1560 surface protein amino acid sequences (Data File 8) to generate alternative phylogenies for 35 genera (
       <a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig. S8</a>), by mapping the correspondent surface protein with polymerase protein from the same virus strain. We found only a small number of discrepancies between numbers of L1-L2, L1-L3/4 and L2-L3/4 transitions for the two sets of phylogenies (
       <a id="xref-fig-18-1" class="xref-fig" href="#F18">Fig. S14</a>), with total numbers of the three transition types differing by -5, -1 and +1 respectively (Data File 9).
      </p>
     </div>
     <div class="section" id="sec-12">
      <h4>Statistical analysis</h4>
      <p id="p-47">We used Fisher’s exact tests to examine whether relative frequencies of transitions between different IT levels varied among families, genera, genome types, and between vector-borne/non-vector-borne viruses. We then compared frequencies of L1-L2 and L1-L3/4 transitions using a binomial generalised linear mixed model (GLMM) with transition type as the response variable, genome/enveloped type and vector-borne/non-vector-borne as explanatory variables and genus as a random factor (using the R package ‘lme4’). We constructed a composite variable representing the genome type and enveloped structure of viruses involved in transition events. This had four levels: (-)ssRNA (all enveloped), (+)ssRNA enveloped, (+)ssRNA non-enveloped, and dsRNA (all non-enveloped). Models with genus as the only random factor and models with both genus and family as random factors had similar AIC values (ΔAIC&lt;;2), so the former were used in all analyses.</p>
      <p id="p-48">We calculated the depth of the node of each transition on the genus-level phylogenetic tree relative to the ancestral node (
       <a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. S4</a> and Data Files 2 and 3). We used a beta distributed GLMM (using the R package ‘glmmTMB’
       <sup>
        <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">13</a>
       </sup>) to compare node depth (response variable) across level transition types (explanatory variable) with genus as a random factor. We used a linear mixed model (LMM) with a normal distribution to compare log
       <sub>e</sub>-transformed transition rates (response variable) by level transition groups (explanatory variable) with genus as a random factor. We also included genome type/enveloped and vector-borne in the model as potential confounding variables.
      </p>
      <p id="p-49">We tested for an association between host range and IT level in two ways. First, we considered known non-human hosts of the 179 human virus species/subtypes in our data set, using availability of virus sequence as a gold standard indicator of host range. Second, we considered known non-human hosts for all viruses from each of 39 genera using the same criteria. For these analyses, we classified species/subtypes and genera as non-human-transmissible if they occurred in a genus or family respectively that contained only viruses incapable of epidemic spread in human populations (i.e. L2 or L3). To compare host range distributions at species/subtype and genus levels we used GLMMs with binary responses to generate odds ratios for containing human-transmissible viruses, with 95% confidence limits, by host category. We included genus and family plus genus respectively as random effects to allow for taxonomic relatedness.</p>
     </div>
    </div>
   </div>
   <div id="sec-13" class="subsection">
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Numbers of sequences used in this study cumulated by year of submission to Genbank. L1, L2 and L3/4 viruses are distinguished." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Numbers of sequences used in this study cumulated by year of submission to Genbank. L1, L2 and L3/4 viruses are distinguished.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F5.medium.gif" width="440" height="354"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S1." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F5.medium.gif" width="440" height="354"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S1.</span>
      <p id="p-50" class="first-child">Numbers of sequences used in this study cumulated by year of submission to Genbank. L1, L2 and L3/4 viruses are distinguished.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Heat map showing the estimated probabilities that the ancestor for each genus (N=39) was a L1, L2 or L3/4 virus. Blank entries indicate that no sequences from viruses at this IT level were available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Heat map showing the estimated probabilities that the ancestor for each genus (N=39) was a L1, L2 or L3/4 virus. Blank entries indicate that no sequences from viruses at this IT level were available.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F6.medium.gif" width="322" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S2." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F6.medium.gif" width="322" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S2.</span>
      <p id="p-51" class="first-child">Heat map showing the estimated probabilities that the ancestor for each genus (N=39) was a L1, L2 or L3/4 virus. Blank entries indicate that no sequences from viruses at this IT level were available.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Heat map showing the total estimated probabilities that the ancestral node for each forward transition (N=96) was a L1, L2 or L3/4 virus. Three transitions are distinguished: L1 to L2 (greylabels), L1 to L3/4 (purple) and L2 to L3/4 (cyan). Blank entries indicate that no sequences from viruses at this level were present in the sequence database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Heat map showing the total estimated probabilities that the ancestral node for each forward transition (N=96) was a L1, L2 or L3/4 virus. Three transitions are distinguished: L1 to L2 (greylabels), L1 to L3/4 (purple) and L2 to L3/4 (cyan). Blank entries indicate that no sequences from viruses at this level were present in the sequence database.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F7.medium.gif" width="214" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S3." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F7.medium.gif" width="214" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S3.</span>
      <p id="p-52" class="first-child">Heat map showing the total estimated probabilities that the ancestral node for each forward transition (N=96) was a L1, L2 or L3/4 virus. Three transitions are distinguished: L1 to L2 (greylabels), L1 to L3/4 (purple) and L2 to L3/4 (cyan). Blank entries indicate that no sequences from viruses at this level were present in the sequence database.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Relative node depths of transitions by genus. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan), shown in a) dotplot per genus, b) boxplot per transition. Relative node depths are given in Data File 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Relative node depths of transitions by genus. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan), shown in a) dotplot per genus, b) boxplot per transition. Relative node depths are given in Data File 2.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F8.medium.gif" width="440" height="329"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S4." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F8.medium.gif" width="440" height="329"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S4.</span>
      <p id="p-53" class="first-child">Relative node depths of transitions by genus. Three transitions are distinguished: L1 to L2 (grey), L1 to L3/4 (purple) and L2 to L3/4 (cyan), shown in a) dotplot per genus, b) boxplot per transition. Relative node depths are given in Data File 2.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of instantaneous rates of L1-L2, L1-L3/4 and L2-L3/4 transitions for each genus, using mean relative transition rates on a log scale (see Fig. S6). Blank entries indicate that a transition was not possible in this genus given the information available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Comparison of instantaneous rates of L1-L2, L1-L3/4 and L2-L3/4 transitions for each genus, using mean relative transition rates on a log scale (see Fig. S6). Blank entries indicate that a transition was not possible in this genus given the information available.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F9.medium.gif" width="297" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S5." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F9.medium.gif" width="297" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S5.</span>
      <p id="p-54" class="first-child">Comparison of instantaneous rates of L1-L2, L1-L3/4 and L2-L3/4 transitions for each genus, using mean relative transition rates on a log scale (see 
       <a id="xref-fig-10-3" class="xref-fig" href="#F10">Fig. S6</a>). Blank entries indicate that a transition was not possible in this genus given the information available.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Transmission matrices of relative transition rates between IT levels in each genus with [[#x2265]]1 transition (N=37). The mean estimates of instantaneous transition rates are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Transmission matrices of relative transition rates between IT levels in each genus with [[#x2265]]1 transition (N=37). The mean estimates of instantaneous transition rates are shown.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F10.medium.gif" width="440" height="352"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S6." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F10.medium.gif" width="440" height="352"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S6.</span>
      <p id="p-55" class="first-child">Transmission matrices of relative transition rates between IT levels in each genus with ≥1 transition (N=37). The mean estimates of instantaneous transition rates are shown.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F11" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Results for GLMMs with binary responses of the association between viruses within a genus being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Family is included as a random effect. See also Fig. 3b." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Results for GLMMs with binary responses of the association between viruses within a genus being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Family is included as a random effect. See also Fig. 3b.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F11.medium.gif" width="440" height="437"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S7." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F11.medium.gif" width="440" height="437"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S7.</span>
      <p id="p-56" class="first-child">Results for GLMMs with binary responses of the association between viruses within a genus being human-transmissible (L3/4) and being known to infect a given non-human host category. Coefficient estimates (with 95% CIs) are compared for six orders of mammal and the Class Aves. Coefficient estimates greater than zero correspond to a positive association. Models include a random term for each observation to account for over-dispersion. Family is included as a random effect. See also 
       <a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3b</a>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F12" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Bayesian maximum clade credibility (MCC) trees for members of 35 virus genera using surface protein sequences (listed in Data File 8). Phylogenies show the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Bayesian maximum clade credibility (MCC) trees for members of 35 virus genera using surface protein sequences (listed in Data File 8). Phylogenies show the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F12.medium.gif" width="300" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S8." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F12.medium.gif" width="300" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S8." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S8.</span>
      <p id="p-57" class="first-child">Bayesian maximum clade credibility (MCC) trees for members of 35 virus genera using surface protein sequences (listed in Data File 8). Phylogenies show the most probable transitions between non-human viruses (red), viruses infective to humans (green) and viruses transmissible in human populations (blue). Phylogenies are arranged by genome type.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F13" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F13.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Outputs of Phylogeny-trait association (BaTS)analysis as estimated IT level trait and phylogenies associations per genus. Scaled (i.e. normalised as (x - mean)/standard deviation) values of Association Index (AI) in light blue; scaled Parsimony Score (PS) in dark blue. Higher AI and PS indicate lower association between trait and phylogeny. a) RNA-dependent RNA polymerase proteins; and b) surface protein proteins. N= 38 and 36 respectively (see Data File 8). Genera with segmented genomes are indicated with asterisk." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Outputs of Phylogeny-trait association (BaTS)analysis as estimated IT level trait and phylogenies associations per genus. Scaled (i.e. normalised as (x - mean)/standard deviation) values of Association Index (AI) in light blue; scaled Parsimony Score (PS) in dark blue. Higher AI and PS indicate lower association between trait and phylogeny. &lt;;strong&gt;;a&lt;;/strong&gt;;) RNA-dependent RNA polymerase proteins; and &lt;;strong&gt;;b&lt;;/strong&gt;;) surface protein proteins. N= 38 and 36 respectively (see Data File 8). Genera with segmented genomes are indicated with asterisk.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F13.medium.gif" width="440" height="412"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S9." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F13.medium.gif" width="440" height="412"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S9." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S9.</span>
      <p id="p-58" class="first-child">Outputs of Phylogeny-trait association (BaTS)analysis as estimated IT level trait and phylogenies associations per genus. Scaled (i.e. normalised as (x - mean)/standard deviation) values of Association Index (AI) in light blue; scaled Parsimony Score (PS) in dark blue. Higher AI and PS indicate lower association between trait and phylogeny. 
       <strong>a</strong>) RNA-dependent RNA polymerase proteins; and 
       <strong>b</strong>) surface protein proteins. N= 38 and 36 respectively (see Data File 8). Genera with segmented genomes are indicated with asterisk.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F14" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F14.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Bayesian maximum clade credibility (MCC) trees for Influenza A virus genus. Phylogeny generated using full length hemagglutinin (HA) protein sequences (N=181, representing 67 subtypes), mapped with IT levels using discrete trait models with Markov jumps. Nine subtypes (H10N7, H5N1, H5N2, H6N1, H7N2, H7N3, H7N4, H7N9, H9N2) are IT level 2; 5 subtypes (H7N7, H1N1, H1N2, H2N2, H3N2) are L3/4; other subtypes are L1. Colour coding as Fig. 1. The scale bar represents amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Bayesian maximum clade credibility (MCC) trees for &lt;;em&gt;;Influenza A virus&lt;;/em&gt;; genus. Phylogeny generated using full length hemagglutinin (HA) protein sequences (N=181, representing 67 subtypes), mapped with IT levels using discrete trait models with Markov jumps. Nine subtypes (H10N7, H5N1, H5N2, H6N1, H7N2, H7N3, H7N4, H7N9, H9N2) are IT level 2; 5 subtypes (H7N7, H1N1, H1N2, H2N2, H3N2) are L3/4; other subtypes are L1. Colour coding as Fig. 1. The scale bar represents amino acid substitutions per site.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F14.medium.gif" width="308" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S10." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F14.medium.gif" width="308" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S10." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S10.</span>
      <p id="p-59" class="first-child">Bayesian maximum clade credibility (MCC) trees for 
       <em>Influenza A virus</em> genus. Phylogeny generated using full length hemagglutinin (HA) protein sequences (N=181, representing 67 subtypes), mapped with IT levels using discrete trait models with Markov jumps. Nine subtypes (H10N7, H5N1, H5N2, H6N1, H7N2, H7N3, H7N4, H7N9, H9N2) are IT level 2; 5 subtypes (H7N7, H1N1, H1N2, H2N2, H3N2) are L3/4; other subtypes are L1. Colour coding as 
       <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a>. The scale bar represents amino acid substitutions per site.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F15" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F15.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Bayesian maximum clade credibility (MCC) trees for the Lentivirus genus. Phylogeny generated using pol protein sequences (N=35, representing 10 species), mapped with IT levels using discrete trait models with Markov jumps. HIV-1 (group O, P, N, M) and HIV-2 (group A and B) are IT level 3/4; SIV (in chimpanzees, gorillas and sooty mangabeys) are L2, other species are L1 14. Colour coding as Fig. 1. The scale bar represents amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Bayesian maximum clade credibility (MCC) trees for the &lt;;em&gt;;Lentivirus&lt;;/em&gt;; genus. Phylogeny generated using &lt;;em&gt;;pol&lt;;/em&gt;; protein sequences (N=35, representing 10 species), mapped with IT levels using discrete trait models with Markov jumps. HIV-1 (group O, P, N, M) and HIV-2 (group A and B) are IT level 3/4; SIV (in chimpanzees, gorillas and sooty mangabeys) are L2, other species are L1 &lt;;sup&gt;;14&lt;;/sup&gt;;. Colour coding as Fig. 1. The scale bar represents amino acid substitutions per site.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F15.medium.gif" width="430" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S11." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F15.medium.gif" width="430" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S11." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S11.</span>
      <p id="p-60" class="first-child">Bayesian maximum clade credibility (MCC) trees for the 
       <em>Lentivirus</em> genus. Phylogeny generated using 
       <em>pol</em> protein sequences (N=35, representing 10 species), mapped with IT levels using discrete trait models with Markov jumps. HIV-1 (group O, P, N, M) and HIV-2 (group A and B) are IT level 3/4; SIV (in chimpanzees, gorillas and sooty mangabeys) are L2, other species are L1 
       <sup>
        <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">14</a>
       </sup>. Colour coding as 
       <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1</a>. The scale bar represents amino acid substitutions per site.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F16" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F16.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of number of transitions (mean) in different genera (N=37) using different methods (see Table S1). Comparing a) discrete trait count versus number of transitions (mean) estimated by Markov jumps and b) observed node changes on discrete trait phylogenies (discrete trait count) versus number of transitions (mean) estimated by parsimony. Lines of equivalence are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Comparison of number of transitions (mean) in different genera (N=37) using different methods (see Table S1). Comparing &lt;;strong&gt;;a&lt;;/strong&gt;;) discrete trait count versus number of transitions (mean) estimated by Markov jumps and &lt;;strong&gt;;b&lt;;/strong&gt;;) observed node changes on discrete trait phylogenies (discrete trait count) versus number of transitions (mean) estimated by parsimony. Lines of equivalence are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F16.medium.gif" width="440" height="310"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S12." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F16.medium.gif" width="440" height="310"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S12." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S12.</span>
      <p id="p-61" class="first-child">Comparison of number of transitions (mean) in different genera (N=37) using different methods (see 
       <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table S1</a>). Comparing 
       <strong>a</strong>) discrete trait count versus number of transitions (mean) estimated by Markov jumps and 
       <strong>b</strong>) observed node changes on discrete trait phylogenies (discrete trait count) versus number of transitions (mean) estimated by parsimony. Lines of equivalence are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F17" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F17.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="As Fig. 2b-f respectively but based on the results of parsimony analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;As Fig. 2b-f respectively but based on the results of parsimony analysis.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F17.medium.gif" width="440" height="315"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S13." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F17.medium.gif" width="440" height="315"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S13." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S13.</span>
      <p id="p-62" class="first-child">As 
       <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2b-f</a> respectively but based on the results of parsimony analysis.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F18" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F18.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of number of transitions in different genera (N=35) estimated using phylogenies based on polymerase (rdrp) vs surface protein genes. Graphs compare: a) number of transitions (mean) estimated by Markov jumps and b) observed node changes on discrete trait phylogenies (discrete trait count). Lines of equivalence are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383358214" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Comparison of number of transitions in different genera (N=35) estimated using phylogenies based on polymerase (rdrp) vs surface protein genes. Graphs compare: &lt;;strong&gt;;a)&lt;;/strong&gt;; number of transitions (mean) estimated by Markov jumps and &lt;;strong&gt;;b&lt;;/strong&gt;;) observed node changes on discrete trait phylogenies (discrete trait count). Lines of equivalence are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. S14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F18.medium.gif" width="440" height="321"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. S14." src="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F18.medium.gif" width="440" height="321"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S14." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/18/771394/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. S14.</span>
      <p id="p-63" class="first-child">Comparison of number of transitions in different genera (N=35) estimated using phylogenies based on polymerase (rdrp) vs surface protein genes. Graphs compare: 
       <strong>a)</strong> number of transitions (mean) estimated by Markov jumps and 
       <strong>b</strong>) observed node changes on discrete trait phylogenies (discrete trait count). Lines of equivalence are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/914388/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/914388/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/914388" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S1.</span>
      <p id="p-64" class="first-child">Numbers of estimated transitions between human-infective/transmissible (IT) levels 1, 2 and 3/4 across all genera (N=39) compared for three methodologies: 1) counts of the number of internal node changes, transitions, observed from discrete traits analysis* and 2) number of expected transitions (mean) from Markov jumps, both with input trees generated by Bayesian interference; 3) number of expected transitions (mean) from the parsimony reconstruction method with input trees generated using maximum likelihood methods.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T3" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/914384/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/914384/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/914384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S2.</span>
      <p id="p-65" class="first-child">Variation in numbers of types of transitions – univariate analyses.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T4" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/914392/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/914392/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/914392" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S3.</span>
      <p id="p-66" class="first-child">Results of GLMM with beta distribution and genus as a random factor comparing relative node depth among transition types.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T5" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/914391/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/914391/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/914391" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S4.</span>
      <p id="p-67" class="first-child">Predictors of loge-transformed genus-level relative transition rates distinguishing L1 to L2, L1 to L3/4 and L2 to L3/4 transitions.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-68">Data File 1</p>
    <p id="p-69">The ancestral level at the root of all genus trees. The posterior support of the ancestral node as well as the posterior probability of each discrete trait state (level) are listed.</p>
    <p id="p-70">Data File 2</p>
    <p id="p-71">Summary of forward transition events (L1-L2, L1-L3/4 and L2-L3/4) and relative node depth. The posterior support of the ancestral node as well as the posterior probability of each discrete trait state (level) are listed.</p>
    <p id="p-72">Data File 3</p>
    <p id="p-73">Summary of reverse transition events (L2-L1, L3/4-L1 and L3/4-L1) and relative node depth. The posterior support of the ancestral node as well as the posterior probability of each discrete trait state (level) are listed.</p>
    <p id="p-74">Data File 4</p>
    <p id="p-75">Sequence data used in this analysis. For each sequence being used in this study, the information of accession number, virus family, genus, species (whether ICTV recognized), transmission level and host type are given.</p>
    <p id="p-76">Data File 5</p>
    <p id="p-77">Maximum clade credibility (MCC) trees of 39 genera mapped with level traits (as shown in 
     <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1</a>). The branches and nodes are coloured according to the most probable ancestral trait inferred by an asymmetric discrete trait model upon the posterior tree set, with correlated trait posterior probabilities indicated by branch widths. Virus species are labelled on tips of the trees. Trees are scaled by number of substitutions per site, with scale bars labelled underneath each genus tree.
    </p>
    <p id="p-78">Data File 6</p>
    <p id="p-79">Example XML file (
     <em>Mamastrovirus</em>) used in the discrete trait analysis.
    </p>
    <p id="p-80">Data File 7</p>
    <p id="p-81">Comparison of IT level trait reconstructions from discrete trait models and Parsimony methods.</p>
    <p id="p-82">Data File 8</p>
    <p id="p-83">Surface proteins selected for phylogenetic analysis by genus (N=35).</p>
    <p id="p-84">Data File 9</p>
    <p id="p-85">Comparison of number of L3/4 lineages and number of IT changes found in each genus phylogeny using polymerase and surface protein sequences.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-30">We are grateful to Andrew Rambaut and Paul Sharp for helpful discussions, and to Alex Bhattacharya for assistance with data preparation.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M. E. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kellam</span>, 
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">Lessons from Ebola: Improving infectious disease surveillance to inform outbreak management</span>. 
         <abbr class="cit-jnl-abbrev">Sci Transl Med</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.2" data-doi="10.1038/nature05775">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wolfe</span>, 
          <span class="cit-name-given-names">N. D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dunavan</span>, 
          <span class="cit-name-given-names">C. P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Diamond</span>, 
          <span class="cit-name-given-names">J</span>
         </span>. 
         <span class="cit-article-title">Origins of major human infectious diseases</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">447</span>, 
         <span class="cit-fpage">279</span>–
         <span class="cit-lpage">283</span>, (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DN.%2BD.%26rft.volume%253D447%26rft.issue%253D7142%26rft.spage%253D279%26rft.epage%253D283%26rft.atitle%253DOrigins%2Bof%2Bmajor%2Bhuman%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05775%26rft_id%253Dinfo%253Apmid%252F17507975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature05775&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17507975&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000246520300036&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.3" data-doi="10.1038/nature22975">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Olival</span>, 
          <span class="cit-name-given-names">K. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">546</span>, 
         <span class="cit-fpage">646</span>–
         <span class="cit-lpage">650</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28636590&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.4">
       <div class="cit-metadata">
        <cite>ME, J. W., 
         <span class="cit-auth">
          <span class="cit-name-surname">Adair</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-article-title">RNA Viruses: A Case Study of the Biology of Emerging Infectious Diseases</span>. 
         <abbr class="cit-jnl-abbrev">Microbiol Spectr</abbr>
         <span class="cit-vol">1</span>, (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.5" data-doi="10.1073/pnas.1521582113">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Geoghegan</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Senior</span>, 
          <span class="cit-name-given-names">A. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Di Giallonardo</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>, 
          <span class="cit-name-given-names">E. C</span>
         </span>. 
         <span class="cit-article-title">Virological factors that increase the transmissibility of emerging human viruses</span>. 
         <abbr class="cit-jnl-abbrev">P Natl Acad Sci USA</abbr>
         <span class="cit-vol">113</span>, 
         <span class="cit-fpage">4170</span>–
         <span class="cit-lpage">4175</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DP%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1521582113%26rft_id%253Dinfo%253Apmid%252F27001840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE1LzQxNzAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xOC83NzEzOTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.6" data-doi="10.3201/eid2212.160123">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M. E. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCaffery</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lycett</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Assessing the Epidemic Potential of RNA and DNA Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>
         <span class="cit-vol">22</span>, 
         <span class="cit-fpage">2037</span>–
         <span class="cit-lpage">2044</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D22%26rft.spage%253D2037%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2212.160123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid2212.160123&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walker</span>, 
          <span class="cit-name-given-names">J. W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">B. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ott</span>, 
          <span class="cit-name-given-names">I. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Drake</span>, 
          <span class="cit-name-given-names">J. M</span>
         </span>. 
         <span class="cit-article-title">Transmissibility of emerging viral zoonoses</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">e0206926</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0206926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M. E. J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>, 
          <span class="cit-name-given-names">L</span>
         </span>. 
         <span class="cit-article-title">Epidemiological characteristics of human-infective RNA viruses</span>. 
         <abbr class="cit-jnl-abbrev">Sci Data</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">180017</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BData%26rft.volume%253D5%26rft.spage%253D180017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.9" data-doi="10.1093/nar/gkx932">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lefkowitz</span>, 
          <span class="cit-name-given-names">E. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Virus taxonomy: the database of the International Committee on Taxonomy of Viruses (ICTV)</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">D708</span>–
         <span class="cit-lpage">D717</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD708%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx932&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Geoghegan</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>, 
          <span class="cit-name-given-names">E. C</span>
         </span>. 
         <span class="cit-article-title">Predicting virus emergence amid evolutionary noise</span>. 
         <abbr class="cit-jnl-abbrev">Open Biol</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.11" data-doi="10.1126/science.aap7463">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Carroll</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The Global Virome Project</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">359</span>, 
         <span class="cit-fpage">872</span>–
         <span class="cit-lpage">874</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarroll%26rft.auinit1%253DD.%26rft.volume%253D359%26rft.issue%253D6378%26rft.spage%253D872%26rft.epage%253D874%26rft.atitle%253DThe%2BGlobal%2BVirome%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap7463%26rft_id%253Dinfo%253Apmid%252F29472471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3OC84NzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xOC83NzEzOTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Carlson</span>, 
          <span class="cit-name-given-names">C. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zipfel</span>, 
          <span class="cit-name-given-names">C. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garnier</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bansal</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Global estimates of mammalian viral diversity accounting for host sharing</span>. 
         <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.13" data-doi="10.1038/nature02104">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Antia</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Regoes</span>, 
          <span class="cit-name-given-names">R. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koella</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bergstrom</span>, 
          <span class="cit-name-given-names">C. T</span>
         </span>. 
         <span class="cit-article-title">The role of evolution in the emergence of infectious diseases</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">426</span>, 
         <span class="cit-fpage">658</span>–
         <span class="cit-lpage">661</span>, (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAntia%26rft.auinit1%253DR.%26rft.volume%253D426%26rft.issue%253D6967%26rft.spage%253D658%26rft.epage%253D661%26rft.atitle%253DThe%2Brole%2Bof%2Bevolution%2Bin%2Bthe%2Bemergence%2Bof%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02104%26rft_id%253Dinfo%253Apmid%252F14668863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature02104&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14668863&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000187132800040&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.14" data-doi="10.1038/nrmicro2440">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pepin</span>, 
          <span class="cit-name-given-names">K. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lass</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pulliam</span>, 
          <span class="cit-name-given-names">J. R. C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Read</span>, 
          <span class="cit-name-given-names">A. F.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd-Smith</span>, 
          <span class="cit-name-given-names">J. O</span>
         </span>. 
         <span class="cit-article-title">Identifying genetic markers of adaptation for surveillance of viral host jumps</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">802</span>–
         <span class="cit-lpage">813</span>, (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DPepin%26rft.auinit1%253DK.%2BM.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D802%26rft.epage%253D813%26rft.atitle%253DIdentifying%2Bgenetic%2Bmarkers%2Bof%2Badaptation%2Bfor%2Bsurveillance%2Bof%2Bviral%2Bhost%2Bjumps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2440%26rft_id%253Dinfo%253Apmid%252F20938453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrmicro2440&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20938453&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.15" data-doi="10.1371/journal.ppat.1006215">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Geoghegan</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>, 
          <span class="cit-name-given-names">E. C</span>
         </span>. 
         <span class="cit-article-title">Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">e1006215</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006215%26rft_id%253Dinfo%253Apmid%252F28178344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006215&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28178344&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scott</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hudson</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howey</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chase-Topping</span>, 
          <span class="cit-name-given-names">M</span>
         </span>. 
         <span class="cit-article-title">Human viruses: discovery and emergence</span>. 
         <abbr class="cit-jnl-abbrev">P Roy Soc B-Biol Sci</abbr>
         <span class="cit-vol">367</span>, 
         <span class="cit-fpage">2864</span>–
         <span class="cit-lpage">2871</span>, (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DP%2BRoy%2BSoc%2BB-Biol%2BSci%26rft.volume%253D367%26rft.spage%253D2864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.17" data-doi="10.1126/science.1059026">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M. E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>, 
          <span class="cit-name-given-names">L. H.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Haydon</span>, 
          <span class="cit-name-given-names">D. T</span>
         </span>. 
         <span class="cit-article-title">Population biology of multihost pathogens</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">292</span>, 
         <span class="cit-fpage">1109</span>–
         <span class="cit-lpage">1112</span>, (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWoolhouse%26rft.auinit1%253DM.%2BE.%2BJ.%26rft.volume%253D292%26rft.issue%253D5519%26rft.spage%253D1109%26rft.epage%253D1112%26rft.atitle%253DPopulation%2BBiology%2Bof%2BMultihost%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1059026%26rft_id%253Dinfo%253Apmid%252F11352066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUxOS8xMTA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTgvNzcxMzk0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.18" data-doi="10.1126/science.1122818">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kuiken</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Host species barriers to influenza virus infections</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">312</span>, 
         <span class="cit-fpage">394</span>–
         <span class="cit-lpage">397</span>, (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKuiken%26rft.auinit1%253DT.%26rft.volume%253D312%26rft.issue%253D5772%26rft.spage%253D394%26rft.epage%253D397%26rft.atitle%253DHost%2BSpecies%2BBarriers%2Bto%2BInfluenza%2BVirus%2BInfections%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1122818%26rft_id%253Dinfo%253Apmid%252F16627737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3Mi8zOTQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xOC83NzEzOTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.19" data-doi="10.1038/d41586-018-05373-w">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>, 
          <span class="cit-name-given-names">E. C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Andersen</span>, 
          <span class="cit-name-given-names">K. G</span>
         </span>. 
         <span class="cit-article-title">Pandemics: spend on surveillance, not prediction</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">558</span>, 
         <span class="cit-fpage">180</span>–
         <span class="cit-lpage">182</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D558%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fd41586-018-05373-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/d41586-018-05373-w&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.20" data-doi="10.1126/science.aap9072">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Babayan</span>, 
          <span class="cit-name-given-names">S. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Orton</span>, 
          <span class="cit-name-given-names">R. J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Streicker</span>, 
          <span class="cit-name-given-names">D. G</span>
         </span>. 
         <span class="cit-article-title">Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">362</span>, 
         <span class="cit-fpage">577</span>-+, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBabayan%26rft.auinit1%253DS.%2BA.%26rft.volume%253D362%26rft.issue%253D6414%26rft.spage%253D577%26rft.epage%253D580%26rft.atitle%253DPredicting%2Breservoir%2Bhosts%2Band%2Barthropod%2Bvectors%2Bfrom%2Bevolutionary%2Bsignatures%2Bin%2BRNA%2Bvirus%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap9072%26rft_id%253Dinfo%253Apmid%252F30385576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNC81NzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xOC83NzEzOTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Friedrich</span>, 
          <span class="cit-name-given-names">M. J</span>
         </span>. 
         <span class="cit-article-title">WHO’s Blueprint List of Priority Diseases</span>. 
         <abbr class="cit-jnl-abbrev">JAMA</abbr>
         <span class="cit-vol">319</span>, 
         <span class="cit-fpage">1973</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.volume%253D319%26rft.spage%253D1973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <h2 class="">Supplementary references</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 1. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.22" data-doi="10.1093/nar/gni032">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Benson</span>, 
          <span class="cit-name-given-names">D. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Karsch-Mizrachi</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lipman</span>, 
          <span class="cit-name-given-names">D. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ostell</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wheeler</span>, 
          <span class="cit-name-given-names">D. L</span>
         </span>. 
         <span class="cit-auth cit-collab">GenBank</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">33</span>, 
         <span class="cit-fpage">D34</span>–
         <span class="cit-lpage">38</span>, (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBenson%26rft.auinit1%253DD.%2BA.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD34%26rft.epage%253DD38%26rft.atitle%253DGenBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgni032%26rft_id%253Dinfo%253Apmid%252F15608212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gni032&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15608212&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000226524300006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 2. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.23" data-doi="10.3201/eid2212.160123">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M. E. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCaffery</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lycett</span>, 
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Assessing the Epidemic Potential of RNA and DNA Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>
         <span class="cit-vol">22</span>, 
         <span class="cit-fpage">2037</span>–
         <span class="cit-lpage">2044</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D22%26rft.spage%253D2037%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2212.160123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid2212.160123&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 3. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.24" data-doi="10.1093/nar/gkx932">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lefkowitz</span>, 
          <span class="cit-name-given-names">E. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Virus taxonomy: the database of the International Committee on Taxonomy of Viruses (ICTV)</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">D708</span>–
         <span class="cit-lpage">D717</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD708%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx932&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 4. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>, 
          <span class="cit-name-given-names">A. J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. 
         <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-issue">214</span>, (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 5. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Simmonds</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aiewsakun</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Katzourakis</span>, 
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">Prisoners of war - host adaptation and its constraints on virus evolution</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 6. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.27" data-doi="10.1093/nar/gkh340">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edgar</span>, 
          <span class="cit-name-given-names">R. C</span>
         </span>. 
         <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">1792</span>–
         <span class="cit-lpage">1797</span>, (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15034147&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220487200025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 7. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.28" data-doi="10.1098/rstb.2012.0196">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Faria</span>, 
          <span class="cit-name-given-names">N. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>, 
          <span class="cit-name-given-names">M. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Streicker</span>, 
          <span class="cit-name-given-names">D. G.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-article-title">Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints</span>. 
         <abbr class="cit-jnl-abbrev">Philos T R Soc B</abbr>
         <span class="cit-vol">368</span>, 
         <span class="cit-fpage">20120196</span>, (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPhilos%2BT%2BR%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0196%26rft_id%253Dinfo%253Apmid%252F23382420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0196&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23382420&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 8. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.29" data-doi="10.1016/j.meegid.2007.08.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Parker</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>, 
          <span class="cit-name-given-names">O. G</span>
         </span>. 
         <span class="cit-article-title">Correlating viral phenotypes with phylogeny: accounting for phylogenetic uncertainty</span>. 
         <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">239</span>–
         <span class="cit-lpage">246</span>, (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DParker%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D239%26rft.epage%253D246%26rft.atitle%253DCorrelating%2Bviral%2Bphenotypes%2Bwith%2Bphylogeny%253A%2Baccounting%2Bfor%2Bphylogenetic%2Buncertainty.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2007.08.001%26rft_id%253Dinfo%253Apmid%252F17921073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.meegid.2007.08.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17921073&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000256289400002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 9. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.30" data-doi="10.1093/molbev/mss084">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Improving the Accuracy of Demographic and Molecular Clock Model Comparison While Accommodating Phylogenetic Uncertainty</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-vol">29</span>, 
         <span class="cit-fpage">2157</span>–
         <span class="cit-lpage">2167</span>, (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss084%26rft_id%253Dinfo%253Apmid%252F22403239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mss084&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22403239&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000308851600009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 10. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.31" data-doi="10.1016/j.cub.2011.05.058">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edwards</span>, 
          <span class="cit-name-given-names">C. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Ancient hybridization and an Irish origin for the modern polar bear matriline</span>. 
         <abbr class="cit-jnl-abbrev">Curr Biol</abbr>
         <span class="cit-vol">21</span>, 
         <span class="cit-fpage">1251</span>–
         <span class="cit-lpage">1258</span>, (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DEdwards%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D21%26rft.issue%253D15%26rft.spage%253D1251%26rft.epage%253D1258%26rft.atitle%253DAncient%2Bhybridization%2Band%2Ban%2BIrish%2Borigin%2Bfor%2Bthe%2Bmodern%2Bpolar%2Bbear%2Bmatriline.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.05.058%26rft_id%253Dinfo%253Apmid%252F21737280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.05.058&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21737280&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 11. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.32" data-doi="10.1093/molbev/msp003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">O’Brien</span>, 
          <span class="cit-name-given-names">J. D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Minin</span>, 
          <span class="cit-name-given-names">V. N.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>, 
          <span class="cit-name-given-names">M. A</span>
         </span>. 
         <span class="cit-article-title">Learning to count: robust estimates for labeled distances between molecular sequences</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>
         <span class="cit-vol">26</span>, 
         <span class="cit-fpage">801</span>–
         <span class="cit-lpage">814</span>, (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp003%26rft_id%253Dinfo%253Apmid%252F19131426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msp003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19131426&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000264188700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 12. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.33" data-doi="10.1093/bioinformatics/btu033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stamatakis</span>, 
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">1312</span>–
         <span class="cit-lpage">1313</span>, (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24451623&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F18%2F771394.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000336095100024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 13. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brooks</span>, 
          <span class="cit-name-given-names">M. E.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">glmmTMB Balances Speed and Flexibility Among Packages for Zero-inflated Generalized Linear Mixed Modeling</span>. 
         <abbr class="cit-jnl-abbrev">R J</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">378</span>–
         <span class="cit-lpage">400</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DR%2BJ%26rft.volume%253D9%26rft.spage%253D378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 14. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-771394v1.35" data-doi="10.1101/cshperspect.a006841">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sharp</span>, 
          <span class="cit-name-given-names">P. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hahn</span>, 
          <span class="cit-name-given-names">B. H</span>
         </span>. 
         <span class="cit-article-title">Origins of HIV and the AIDS pandemic</span>. 
         <abbr class="cit-jnl-abbrev">Cold Spring Harb Perspect Med</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">a006841</span>, (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BMedicine%26rft.stitle%253DCold%2BSpring%2BHarb%2BPerspect%2BMed%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253Da006841%26rft.epage%253Da006841%26rft.atitle%253DOrigins%2Bof%2BHIV%2Band%2Bthe%2BAIDS%2BPandemic%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a006841%26rft_id%253Dinfo%253Apmid%252F22229120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNzaHBlcnNwZWN0bWVkIjtzOjU6InJlc2lkIjtzOjExOiIxLzEvYTAwNjg0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzE4Lzc3MTM5NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
