<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6687192">6687192</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31356638">31356638</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1007938">10.1371/journal.ppat.1007938</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-18-01905</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Processes</span>: <span class="subject" title="subject">Endocytosis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Processes</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Secretory Pathway</span>: <span class="subject" title="subject">Endocytosis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Types</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epithelial Cells</span>: <span class="subject" title="subject">Endothelial Cells</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Tissue</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epithelium</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epithelial Cells</span>: <span class="subject" title="subject">Endothelial Cells</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Tissue</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epithelium</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epithelial Cells</span>: <span class="subject" title="subject">Endothelial Cells</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">West Nile virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">West Nile virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">West Nile virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">West Nile virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Glycobiology</span>: <span class="subject" title="subject">Glycosylation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Proteins</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Post-Translational Modification</span>: <span class="subject" title="subject">Glycosylation</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemical Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Salts</span>: <span class="subject" title="subject">Sulfates</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene regulation</span>: <span class="subject" title="subject">Small interfering RNAs</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">Small interfering RNAs</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Physiology</span>: <span class="subject" title="subject">Cell Binding</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Proteins</span>: <span class="subject" title="subject">Recombinant Proteins</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Endocytosis of flavivirus NS1 is required for NS1-mediated endothelial hyperpermeability and is abolished by a single N-glycosylation site mutation</div> <div class="alt-title" title="alt-title">
N207-glycosylation and endocytosis of flavivirus NS1 are required for endothelial hyperpermeability</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Chunling</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3050-4480</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">Henry</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1991-629X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biering</span><span class="given-names" tagx="given-names" title="given-names">Scott B.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">Dustin R.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7225-2261</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">Edwina B.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8523-1104</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patana</span><span class="given-names" tagx="given-names" title="given-names">Mark</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomberg</span><span class="given-names" tagx="given-names" title="given-names">Trent A.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malvar</span><span class="given-names" tagx="given-names" title="given-names">Carmel</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4289-1222</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">Nicholas T. N.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4364-5031</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinosa</span><span class="given-names" tagx="given-names" title="given-names">Diego A.</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7238-4037</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">Eva</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001" /><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[], <span class="addr-line" title="addr-line">Division of Infectious Diseases and Vaccinology, School of Public Health, University of California, Berkeley, Berkeley, California, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherry</span><span class="given-names" tagx="given-names" title="given-names">Sara</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Pennsylvania School of Medicine, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">eharris@berkeley.edu</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-7-7</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-7</span></span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007938</span><span class="history" title="history"><span class="received" title="received">received: 2018-10-1</span><span class="accepted" title="accepted">accepted: 2019-6-22</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Wang et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1007938.pdf" title="self-uri">ppat.1007938.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Arthropod-borne flaviviruses cause life-threatening diseases associated with endothelial hyperpermeability and vascular leak. We recently found that vascular leak can be triggered by dengue virus (DENV) non-structural protein 1 (NS1) via the disruption of the endothelial glycocalyx-like layer (EGL). However, the molecular determinants of NS1 required to trigger EGL disruption and the cellular pathway(s) involved remain unknown. Here we report that mutation of a single glycosylated residue of NS1 (N207Q) abolishes the ability of NS1 to trigger EGL disruption and induce endothelial hyperpermeability. Intriguingly, while this mutant bound to the surface of endothelial cells comparably to wild-type NS1, it was no longer internalized, suggesting that NS1 binding and internalization are distinct steps. Using endocytic pathway inhibitors and gene-specific siRNAs, we determined that NS1 was endocytosed into endothelial cells in a dynamin- and clathrin-dependent manner, which was required to trigger endothelial dysfunction <i>in vitro</i> and vascular leak <i>in vivo</i>. Finally, we found that the N207 glycosylation site is highly conserved among flaviviruses and is also essential for West Nile and Zika virus NS1 to trigger endothelial hyperpermeability via clathrin-mediated endocytosis. These data provide critical mechanistic insight into flavivirus NS1-induced pathogenesis, presenting novel therapeutic and vaccine targets for flaviviral diseases.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>Vascular leak is a hallmark of severe dengue disease. Recently, our group revealed a critical role for NS1 in induction of endothelial hyperpermeability and vascular leakage in an endothelial cell-intrinsic manner. However, the upstream pathway triggered by NS1, as well as the molecular determinants of NS1 required for this phenomenon, remain obscure. Gaining insight into this endothelial cell-intrinsic pathway is critical for understanding dengue pathogenesis, developing novel antiviral therapies, and developing NS1-based vaccine approaches that pose a minimal risk of antibody-dependent enhancement. Our current study expands our knowledge of this novel pathway not only by identifying the requirement of internalization of secreted NS1 via clathrin-mediated endocytosis, but also by pinpointing the NS1 molecular determinant (N207) required to trigger vascular leak. Further, our work identifies N207 as a residue conserved among multiple flaviviruses (Zika virus and West Nile virus, in addition to DENV), which is critical for NS1-mediated vascular leak in biologically relevant human endothelial cells modeling interstitial compartments in the lung or the blood-brain barrier. Thus, our study identifies endocytosis and a single amino acid (N207) of the NS1 viral toxin as critical for pan-flavivirus pathogenesis, representing a novel target for anti-flaviviral therapy and vaccine development.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">NIAID/NIH</span></div><span class="award-id" tagx="award-id" title="award-id">R01 AI24493</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7238-4037</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">Eva</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported by NIAID/NIH grant R01 AI24493 (E.H.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="PLOS Publication Stage" value="vor-update-to-uncorrected-proof" /> <meta name="Publication Update" value="2019-08-08" /> <meta name="Data Availability" value="All relevant data are within the manuscript and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the manuscript and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Dengue virus (DENV) is a mosquito-borne flavivirus, and infection with any of its four serotypes (DENV1-4) can result in inapparent infection, classic dengue fever, or dengue hemorrhagic fever/dengue shock syndrome–severe manifestations characterized by vascular leak that can lead to shock and death [<a href="#ppat.1007938.ref001">1</a>]. West Nile virus (WNV) is a related flavivirus that causes encephalitis [<a href="#ppat.1007938.ref002">2</a>], and Zika virus (ZIKV), which recently emerged and generated large epidemics across the Americas, can cause Guillain-Barré syndrome in adults and microcephaly and other congenital birth defects in babies born to women infected during pregnancy [<a href="#ppat.1007938.ref003">3</a>–<a href="#ppat.1007938.ref007">7</a>]. The 10.7-kb RNA genome of flaviviruses encodes three structural and seven non-structural (NS) proteins, including NS1. We recently described a novel cell-intrinsic role for DENV NS1 in increasing permeability of human endothelial cell monolayers <i>in vitro</i> and systemic vascular leak <i>in vivo</i> via disruption of components of the endothelial glycocalyx-like layer (EGL) [<a href="#ppat.1007938.ref008">8</a>–<a href="#ppat.1007938.ref010">10</a>]. This EGL pathway is distinct from the cytokine-mediated pathway involving activation of peripheral blood mononuclear cells (PBMCs) previously described [<a href="#ppat.1007938.ref010">10</a>, <a href="#ppat.1007938.ref011">11</a>]. The NS1-mediated degradation of the EGL involves activation of the protease cathepsin L, heparanase, and sialidases, which in turn disrupt the EGL. However, the upstream pathway initiated by NS1, resulting in cathepsin L activation, EGL degradation, and therefore vascular leak, as well as the molecular determinants of NS1 required for inducing endothelial hyperpermeability, have not yet been identified.</p> <p>All flavivirus NS1 genes are highly homologous and encode a 352-amino acid polypeptide with slight variation in glycosylation status that results in a molecular weight of 46–55 kDa [<a href="#ppat.1007938.ref012">12</a>]. High-resolution structures of the C-terminal half [<a href="#ppat.1007938.ref013">13</a>] and full-length [<a href="#ppat.1007938.ref014">14</a>] DENV NS1, as well as the closely related WNV NS1, revealed dimers containing distinct domains for membrane association and interactions with the immune system and provided structural insights into flavivirus NS1 assembly and antibody recognition [<a href="#ppat.1007938.ref014">14</a>]. In flavivirus-infected cells, NS1 dimers are found intracellularly and also on the cell surface, while secreted forms are comprised of atypical barrel-shaped hexamers containing lipid cargo [<a href="#ppat.1007938.ref015">15</a>].</p> <p>DENV NS1 contains two conserved N-linked glycans (N-glycans) at Asn-130 (N130) and Asn-207 (N207), and NS1 proteins derived from different cells/species exhibit distinct types of N-glycosylation [<a href="#ppat.1007938.ref016">16</a>]. Previous studies investigating the importance of the N-glycans on NS1 found that DENV2 deglycosylated at either site exhibited significant attenuation of neurovirulence in mice compared to the wild-type (WT) virus [<a href="#ppat.1007938.ref017">17</a>]. However, the mechanism of this attenuated virulence has never been elucidated, due in part to the fact that NS1 N-glycosylation mutant viruses are unstable, with significant defects in viral replication and proliferation that prevent further conclusive mechanistic study [<a href="#ppat.1007938.ref017">17</a>]. This has been a major roadblock in efforts to study the dual role of NS1 in viral replication and pathogenesis. In mammalian cell-derived DENV NS1, complex glycans are attached at Asn-130 and high-mannose glycans are attached at Asn-207 [<a href="#ppat.1007938.ref018">18</a>]; these N-glycans are involved in NS1 dimer stability and secretion [<a href="#ppat.1007938.ref018">18</a>, <a href="#ppat.1007938.ref019">19</a>].</p> <p>In this study, to investigate the role of these N-glycosylation sites directly on NS1-induced vascular leak independently from their roles in viral replication, we generated recombinant NS1 in a mammalian expression system, which incorporates human glycosylation patterns. We sought to investigate whether the glycosylation status of NS1 may contribute to its ability to induce endothelial hyperpermeability. Using both transendothelial electrical resistance (TEER) and an immunofluorescence assay (IFA) with confocal microscopy, we show that the NS1-N207Q mutant does not induce endothelial barrier dysfunction or degradation of the EGL of human pulmonary microvascular endothelial cells (HPMEC). Further, we demonstrate that there are no differences in cell binding <i>per se</i> between DENV WT NS1 and the NS1-N207Q mutant, but the latter is not efficiently internalized by endothelial cells, indicating that internalization of NS1 is required for endothelial cell-intrinsic pathogenesis. Furthermore, we show that endothelial cells internalize NS1 via clathrin-mediated endocytosis and that its inhibition by either small molecule inhibitors or siRNA knock-down prevents NS1-induced hyperpermeability and EGL degradation <i>in vitro</i> and vascular leak <i>in vivo</i>. Finally, we identify N207 as a conserved residue of NS1 among multiple flaviviruses (such as DENV, ZIKV, and WNV) that is critical for NS1 internalization and endothelial hyperpermeability in biologically relevant human cells. Taken together, our results indicate that the N207 glycosylation site is required for NS1 internalization by endothelial cells and that clathrin-mediated endocytosis is a key step in the pathway leading to flavivirus NS1-induced endothelial hyperpermeability and barrier dysfunction.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="denvns1-n207qmutantproteindoesnotinduceendothelialhyperpermeabilityinhumanendothelialcells" title="sec"> <div class="title" tagx="title" title="title">
DENV NS1-N207Q mutant protein does not induce endothelial hyperpermeability in human endothelial cells</div> <p>To examine the role of NS1 glycosylation status on endothelial barrier dysfunction, we generated several NS1 constructs targeting glycosylation sites in DENV2 NS1 (henceforth referred to as DENV NS1), including WT, the single N-glycosylation mutants N130Q and N207Q, and the double mutant N130Q+N207Q (<a href="#ppat.1007938.g001">Fig 1A</a> and <a href="#ppat.1007938.s001">S1 Table</a>). DENV NS1-WT and DENV NS1-N207Q were successfully secreted and purified as oligomers (MW: &amp;gt;250 kDa) (<a href="#ppat.1007938.g001">Fig 1B and 1C</a> and <a href="#ppat.1007938.s002">S1A Fig</a>); the DENV NS1-N207Q mutant (monomer ~43 kDa) was slightly smaller than the NS1-WT monomer (~48 kDa) due to loss of the N207 glycosylation site (<a href="#ppat.1007938.g001">Fig 1B</a>), consistent with previous publications [<a href="#ppat.1007938.ref019">19</a>]. In contrast, the N130Q mutant was not efficiently secreted, nor was the double mutant N130Q+N207Q (<a href="#ppat.1007938.s002">S1B Fig</a>); thus, these two constructs were not further characterized. 6xHis-tagged recombinant DENV NS1 proteins were purified using nickel-nitrilotriacetic acid resin agarose beads and subjected to dialysis; the resulting purified DENV NS1 proteins (WT and N207Q) were visualized by silver stain and found to exhibit purity similar to commercially available NS1 (&amp;gt;95%), as previously described [<a href="#ppat.1007938.ref008">8</a>] (<a href="#ppat.1007938.g001">Fig 1D</a>). To confirm the successful removal of an N-glycan at position 207, we digested WT NS1 (both commercial and in-house produced) and the NS1-N207Q mutant with endoglycosidase H (Endo H; removes high mannose glycans) or PNGase F (removes complex glycans and high mannose glycans), and measured their digestion sensitivity through observation of a gel shift by Western blot. The commercial and in-house produced WT NS1 proteins were sensitive to digestion by both Endo H and PNGase F, suggesting they possessed both the high mannose glycan at position 207 as well as the complex glycan at position 130. In contrast, the NS1-N207Q mutant was sensitive to digestion with PNGase F but not Endo H, suggesting that it retained the complex glycan at position 130 but no longer possessed the high mannose N-glycan at position 207, confirming its removal by the N207Q mutation (<a href="#ppat.1007938.s002">S1C Fig</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Mutation of the N-glycosylation site 207 prevents DENV NS1-induced hyperpermeability of endothelial cells.</div> <p><b>(A)</b> Structural model of a DENV2 NS1 dimer showing site-specific mutations. Color-coding highlights two NS1 monomers (one in green, one in yellow) and amino acid changes in corresponding constructs (blue highlights changes in the green monomer; pink highlights changes in the yellow monomer). WT, N130Q, N207Q, and N130Q+N207Q are all on the DENV2 NS1 backbone. PDB ID: 4O6B [<a href="#ppat.1007938.ref014">14</a>]. (<b>B)</b> Western blot of SDS-PAGE of in-house purified and dialyzed DENV NS1-WT and NS1-N207Q mutant, showing monomeric form, using anti-NS1 mAb (7E11) for detection. <b>(C)</b> Silver stain of native PAGE showing oligomeric forms of both purified and dialyzed in-house DENV NS1-WT and NS1-N207Q mutant (MW: &amp;gt;250 kDa). <b>(D)</b> Silver stain of SDS-PAGE gels of purified and dialyzed in-house DENV NS1-WT and NS1-N207Q NS1 mutant, showing purity, using the Pierce Silver Stain Kit. Positive control, NS1<sup>+</sup>, DENV NS1 from Native Antigen Company; Neg, dialyzed elution buffer. Molecular weight marker, 10-250-kDa protein ladder. <b>(E-F)</b> DENV NS1 triggers endothelial permeability of HPMEC (<b>E</b>) and HBMEC (<b>F</b>) monolayers evaluated by TEER (Ohms) at the indicated time-points. Relative TEER was calculated as previously described [<a href="#ppat.1007938.ref008">8</a>]. Data are representative of 2–3 individual experiments. NS1+, NS1 from Native Antigen Company.</p>   </div> <p>We next compared the size and stability of the NS1-WT and NS1-N207Q proteins. When subjected to size-exclusion chromatography, both our in-house produced NS1-WT and NS1-N207Q eluted at similar fractions, suggesting they are of comparable size and conformation (<a href="#ppat.1007938.s003">S2A and S2B Fig</a>). We then utilized an NS1-specific monoclonal antibody (9NS1) previously reported to detect NS1 in a native but not in a denatured conformation [<a href="#ppat.1007938.ref020">20</a>], to further confirm that NS1-WT and NS1-N207Q were in a comparable conformation. Indeed, we found that 9NS1 recognized WT NS1 (both commercial and in-house produced) comparably to NS1-N207Q in native conditions but did not detect any of the three under denaturing conditions. This confirms both that 9NS1 functions as a conformational antibody and that the conformation of NS1-WT and NS1-N207Q is comparable (<a href="#ppat.1007938.s004">S3A Fig</a>). As expected, several additional NS1-specific monoclonal antibodies that are not conformation-dependent, as well as an antibody targeting the 6xHis-tag, recognized all NS1 proteins comparably in native and denaturing conditions (<a href="#ppat.1007938.s004">S3A Fig</a>). Next, to confirm that our purified and dialyzed NS1 proteins were stable at 37°C over the time-frame relevant for our <i>in vitro</i> cell culture-based assays, we diluted the NS1 proteins in our standard cell culture medium and incubated these proteins at 37°C for varying times. We found that both NS1-WT and NS1-N207Q are stable under these conditions both at time-points relevant for our <i>in vitro</i> assays (7 hours) and even at longer time-points (48 hours) (<a href="#ppat.1007938.s004">S3B–S3D Fig</a>). As expected, in these experiments, the addition of SDS or proteinase K degraded NS1 (<a href="#ppat.1007938.s004">S3B–S3D Fig</a>). Taken together, these data indicate that our purified NS1 proteins (both NS1-WT and NS1-N207Q) are comparable in size, conformation, and stability.</p> <p>To investigate the functional role of NS1 glycosylation on NS1-mediated pathogenesis, we first studied the effects of the proteins on endothelial barrier function by measuring the TEER of HPMEC, since a major site of vascular leak in severe dengue occurs in the lungs [<a href="#ppat.1007938.ref008">8</a>]. Our DENV NS1-WT triggered a similar decrease in TEER values as commercial DENV2 NS1 (DENV NS1+) (<a href="#ppat.1007938.g001">Fig 1E</a>), indicating disruption of endothelial barrier integrity. In contrast, the relative TEER of the DENV NS1-N207Q mutant was similar to that of mock-treated cells, demonstrating that mutation of the N207 glycosylation site in DENV NS1 completely prevents induction of endothelial hyperpermeability in HPMEC (<a href="#ppat.1007938.g001">Fig 1E</a> and <a href="#ppat.1007938.s005">S4A Fig</a>). Similar results were observed in human brain microvascular endothelial cells (HBMEC) (<a href="#ppat.1007938.g001">Fig 1F</a> and <a href="#ppat.1007938.s005">S4B Fig</a>); these cells model barrier function of the blood-brain barrier, which is relevant due to cases of dengue encephalitis and reports of DENV detected in the brain in autopsies of dengue shock cases [<a href="#ppat.1007938.ref021">21</a>, <a href="#ppat.1007938.ref022">22</a>]. Together, these data suggest that the N207 glycosylation site is a critical molecular determinant of DENV NS1-mediated pathogenesis.</p> </div> <div class="denvns1-n207qmutantproteinbindstocellsbutdoesnottriggerdegradationoftheendothelialglycocalyx-likelayer" title="sec"> <div class="title" tagx="title" title="title">
DENV NS1-N207Q mutant protein binds to cells but does not trigger degradation of the endothelial glycocalyx-like layer</div> <p>We previously demonstrated that DENV NS1 binds to endothelial cells and activates cathepsin L, which then activates heparanase, leading to cleavage of heparan sulfate from proteoglycans on the cell surface [<a href="#ppat.1007938.ref009">9</a>]. NS1 also activates sialidases, which cleave sialic acid on the cell surface [<a href="#ppat.1007938.ref009">9</a>]. Together, these events contribute to NS1-induced degradation of the EGL, leading to endothelial barrier dysfunction [<a href="#ppat.1007938.ref009">9</a>, <a href="#ppat.1007938.ref010">10</a>]. To mechanistically determine why the DENV NS1-N207Q mutant no longer triggers hyperpermeability of HPMEC, we first asked whether the DENV NS1-N207Q mutant was still able to bind to the surface of endothelial cells. We found that both the DENV NS1-N207Q mutant and WT DENV NS1 (both commercial and in-house-produced) bound to the surface of HPMEC as visualized by IFA (<a href="#ppat.1007938.g002">Fig 2A and 2B</a> and <a href="#ppat.1007938.s006">S5A and S5B Fig</a>). However, the DENV NS1-N207Q mutant did not trigger EGL disruption, as determined by disruption of sialic acid, activation of cathepsin L, and cleavage of heparan sulfate, visualized by IFA at 1 and 6 hours post-treatment (hpt) with NS1 (<a href="#ppat.1007938.g002">Fig 2A–2E</a> and <a href="#ppat.1007938.s006">S5A and S5C Fig</a>). These results suggest that the N-glycosylation of DENV NS1 at residue 207 plays a role in the activation of the endothelial cell-intrinsic mechanisms leading to disruption of the EGL.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Mutation of the N-glycosylation site 207 prevents DENV NS1-induced cathepsin L and sialidase activation and EGL disruption on endothelial cells.</div> <p><b>(A)</b> NS1 protein binding (green, top row) to HPMEC 6 hpt at 37°C was visualized via IFA. The integrity of the EGL was assessed by the presence of sialic acid (Sia) surface expression, stained with WGA-A647 (red, 2<sup>nd</sup> row; merge of NS1 and Sia), as well as cathepsin L (CTSL) activity (red, 3<sup>rd</sup> row) and heparan sulfate [<a href="#ppat.1007938.ref009">9</a>] surface expression (green, bottom row), in HPMEC 6 hpt with NS1 at 37°C, as visualized via IFA. Nuclei were stained with Hoechst (blue). Images (20X; scale bars, 50 μm) are representative of 2 independent experiments. <b>(B)</b> Mean Fluorescence Intensity (MFI) was used to quantify the amount of NS1 binding to the cell surface, <b>(C)</b> sialic acid surface expression, <b>(D)</b> cathepsin L activity, and <b>(E)</b> heparan sulfate expression on HPMEC in <b>Fig 2A</b>. The means ± standard error of the mean (SEM) of two individual experiments run in duplicate are shown. ns = not significant; *, p&amp;lt;0.05; ***, p&amp;lt;0.001; ****, p&amp;lt;0.0001.</p>   </div> </div> <div class="denvns1-n207qmutantisnotinternalizedbyendothelialcells" title="sec"> <div class="title" tagx="title" title="title">
DENV NS1-N207Q mutant is not internalized by endothelial cells</div> <p>Though the DENV NS1-N207Q mutant was still able to bind to the surface of endothelial cells without leading to disruption of the EGL, it surprisingly appeared to bind to the surface of HPMEC at substantially higher levels than WT DENV NS1 (both commercial and in-house-produced) after 6 hours at 37°C (<a href="#ppat.1007938.g002">Fig 2A and 2B</a>). To further investigate this finding, we evaluated NS1 levels on the surface of HPMEC at 4°C, a temperature at which internalization via endocytosis should be greatly reduced [<a href="#ppat.1007938.ref023">23</a>, <a href="#ppat.1007938.ref024">24</a>]. This examines whether the apparent enhanced binding of DENV NS1-N207Q at 37°C was due to an intrinsically enhanced ability of the DENV NS1-N207Q mutant to bind to endothelial cells or through some other mechanism, such as deficient endocytosis. We found that the DENV NS1-N207Q mutant bound to HPMEC at similar levels as both commercial and in-house-produced WT DENV NS1 when incubated at 4°C for 1 hour, suggesting the DENV NS1-N207Q does not intrinsically bind more to HPMEC compared to WT DENV NS1 (<a href="#ppat.1007938.g003">Fig 3A</a>, top row, and <a href="#ppat.1007938.g003">Fig 3B</a>, left panel). Instead, when NS1 proteins were incubated with HPMEC at 37°C (allowing for protein internalization) for 1 hour, the DENV NS1-N207Q mutant again bound at higher levels compared to WT NS1, suggesting that it remained on the cell surface, while both commercial and in-house-produced WT NS1 proteins were internalized (<a href="#ppat.1007938.g003">Fig 3A</a>, bottom row, and <a href="#ppat.1007938.g003">Fig 3B</a>, right panel).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Mutation of the N-glycosylation site 207 of DENV NS1 results in defective NS1 internalization.</div> <p><b>(A)</b> NS1 protein binding (green) to HPMEC 1 hpt at 4°C (top row) or 37°C (bottom row) was visualized via IFA. Images (20X; scale bars, 50 μm) are representative of 2 independent experiments. <b>(B)</b> MFI was used to quantify the amount of NS1 binding to the cell surface 1 hpt at 4°C and 37°C in <b>Fig 3A</b>. The means ± SEM of two individual experiments run in duplicate are shown. ns = not significant; ****, p&amp;lt;0.0001. <b>(C)</b> Confluent HPMEC monolayers were exposed to 10 μg/ml of different NS1 proteins and incubated at 37°C for 1 hours. Trypsin was used to remove surface-bound NS1, and cell lysates were analyzed by Western blot. Western blot shows detection of the internalized DENV NS1 protein, GAPDH (loading control), and Rab5, an early endosome marker. (<b>D</b>) Quantitation of C. Each bar represents the mean ± standard error of the mean (SEM) of densitometry values normalized to the control protein α-tubulin from two independent experiments. <b>(E)</b> Co-localization of NS1 proteins (red), as indicated, with Rab5 (green) in HPMEC. Co-localization is shown in yellow in merge image or white in co-localization panel (JACoP, ImageJ). Nuclei are stained with Hoechst (blue). Images (40X; scale bars, 10 μm) are representative of 2 individual experiments run in duplicate. NS1+, NS1 from Native Antigen Company. <b>(F)</b> Quantification of DENV NS1 and Rab5 colocalization in HPMEC in <b>Fig 3E</b>. Quantification of the amount of spatial overlap between the two signals (NS1 in red and Rab5 in green) in <b>Fig 3E</b> was obtained using four different frames from the maximum projections of two RGB images based on the object-based approach (JACoP) and defined by the Manders’ Coefficient as previously described [<a href="#ppat.1007938.ref059">59</a>]. The colocalization coefficient was normalized taking into account the signal obtained from 300 cells per image. Each bar represents the mean ± SEM of MFI values in the colocalization analyses obtained from two independent experiments. NS1+, NS1 from Native Antigen Company. ns = not significant; ***, p&amp;lt;0.001.</p>   </div> <p>NS1 is known to bind to heparan sulfate moieties on the surface of endothelial cells [<a href="#ppat.1007938.ref025">25</a>]. To determine whether the NS1-N207Q mutant still required heparan sulfate moieties to bind to the surface of endothelial cells, we pretreated cells with recombinant heparanase to selectively remove heparan sulfate from the cell surface, and then compared the ability of NS1-WT and the NS1-N207Q mutant to bind to these cells. As expected, the in-house produced NS1-WT exhibited a significant decrease in binding to heparanase-treated cells compared to untreated cells. Similarly, the NS1-N207Q mutant also exhibited a significant decrease in binding to heparanase-treated cells compared to the untreated cells, suggesting that the NS1-N207Q mutant still requires heparan sulfate to bind to the endothelial cell surface (<a href="#ppat.1007938.s007">S6A and S6B Fig</a>). As expected, levels of heparan sulfate were dramatically decreased in endothelial cells pretreated with heparanase compared to untreated control cells (<a href="#ppat.1007938.s007">S6C Fig</a>).</p> <p>Our previous observations, and the observations of others, support the concept of NS1 internalization by endothelial cells [<a href="#ppat.1007938.ref009">9</a>, <a href="#ppat.1007938.ref026">26</a>]. To further support the notion that the DENV NS1-N207Q mutant has a defect in internalization, we incubated WT DENV NS1 and DENV NS1-N207Q with HPMEC at 37°C and compared levels of NS1 after stripping surface-bound NS1 from HPMEC; the remaining NS1 is presumed to have been internalized. We found that WT DENV NS1 (both commercial and in-house-produced) appeared at higher levels than the DENV NS1-N207Q mutant in the HPMEC cell lysate at 1 hpt following trypsin-mediated removal of surface-bound NS1, as measured by Western blot, supporting our hypothesis that the DENV NS1-N207Q mutant has an internalization defect (<a href="#ppat.1007938.g003">Fig 3C and 3D</a>). Non-trypsin-treated cells served as a control for the initial levels of NS1 added to cells (<a href="#ppat.1007938.s008">S7A Fig</a>). To visualize intracellular WT NS1, we incubated NS1 with HPMEC at 37°C for 1 hour and examined permeabilized cells by IFA for colocalization of WT DENV NS1-WT with Rab5, a regulatory GTPase associated with early endosomes [<a href="#ppat.1007938.ref027">27</a>]. Both WT NS1 and NS1-N207Q proteins were detected; however, only WT DENV NS1, but not DENV NS1-N207Q, colocalized with Rab5 (<a href="#ppat.1007938.g003">Fig 3E and 3F</a> and <a href="#ppat.1007938.s008">S7B Fig</a>). This pattern can also be visualized in an animated single 2D projection of NS1 internalization in HPMEC created using multiple confocal Z-stack images (<a href="#ppat.1007938.s017">S1</a> and <a href="#ppat.1007938.s018">S2</a> Movies). After binding to the cell surface, the WT NS1 protein (in <i>red</i>, indicated by white arrowheads) moves inside the cell, where it encounters Rab5 (in <i>green</i>), which results in a coalescence signal (in <i>yellow</i>), suggesting their spatial colocalization in the cell. Interestingly, as the Z stack images move into a deeper Z axial space, this colocalizing signal disappears, and only the red signal for NS1 persists, suggesting its mobilization into a different intracellular compartment (<a href="#ppat.1007938.s017">S1 Movie</a>). This process does not occur with the NS1-N207Q mutant protein, where most of the NS1 (in <i>red</i>) seems to remain on the cell surface, as little to no signal appears to colocalize with Rab5, suggesting a lack of internalized protein (<a href="#ppat.1007938.s018">S2 Movie</a>). To further demonstrate the localization of NS1 in the early endosome, we show that NS1 also colocalizes with overexpressed Rab5-GFP as well as an additional early endosome marker (EEA1) (<a href="#ppat.1007938.s008">S7C and S7D Fig</a>) Collectively, these data demonstrate that while both WT DENV NS1 and DENV NS1-N207Q bind comparably to endothelial cells, only WT DENV NS1 is internalized, suggesting that endothelial cell binding and internalization may be distinct stages of NS1-induced cell-intrinsic endothelial hyperpermeability. Based on these results, we hypothesized that DENV NS1 internalization by endothelial cells is a critical step in the activation of EGL disruption and endothelial hyperpermeability.</p> </div> <div class="denvns1isinternalizedviaclathrin-mediatedendocytosis,whichisrequiredtotriggerendothelialhyperpermeability,egldegradation,andvascularleak" title="sec"> <div class="title" tagx="title" title="title">
DENV NS1 is internalized via clathrin-mediated endocytosis, which is required to trigger endothelial hyperpermeability, EGL degradation, and vascular leak</div> <p>Endocytosis is a major route for transporting molecules into cells to regulate many cellular signaling processes and is commonly facilitated by two main endocytic molecules: clathrin and caveolin [<a href="#ppat.1007938.ref028">28</a>]. To interrogate the mechanism by which DENV NS1 is internalized by human endothelial cells, we incubated DENV NS1 with HPMEC and examined localization of internalized NS1 with clathrin and caveolin. We detected colocalization of WT DENV NS1 (both commercial and in-house-produced) with clathrin (<a href="#ppat.1007938.g004">Fig 4A and 4C</a>) but not with caveolin in both HPMEC (<a href="#ppat.1007938.g004">Fig 4B and 4D</a>) and HBMEC (<a href="#ppat.1007938.s009">S8A–S8D Fig</a>), suggesting that NS1 is internalized into endothelial cells in a clathrin-dependent manner. Interestingly, although DENV NS1-N207Q colocalized with clathrin comparably to DENV WT NS1 in HPMEC (<a href="#ppat.1007938.g004">Fig 4A and 4C</a>), only WT DENV NS1 colocalized with the early endosome marker Rab5 (<a href="#ppat.1007938.g003">Fig 3E–3F</a>). These data suggest that while binding of the DENV NS1-N207Q mutant to endothelial cells is sufficient to recruit clathrin to the NS1 binding site, it is not sufficient to promote internalization [<a href="#ppat.1007938.ref029">29</a>, <a href="#ppat.1007938.ref030">30</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
DENV NS1 colocalizes with clathrin but not caveolin on the surface of HPMEC, and clathrin-mediated endocytosis inhibitors prevent colocalization of NS1 and Rab5.</div> <p>Confluent HPMEC monolayers were exposed to 10 μg/ml of different NS1 proteins and incubated at 37°C for 30 minutes. Colocalization of NS1 proteins (red, left column), as indicated, with either <b>(A)</b> clathrin or <b>(B)</b> caveolin (green, second column) in HPMEC. Colocalization is shown in yellow in merge images (third column) or white in colocalization panels (fourth column; JACoP, ImageJ). Nuclei were stained with Hoechst (blue). Images (40X; scale bars, 5 μm) are representative of 2 independent experiments performed in duplicate. <b>(C-D)</b> Quantification of the amount of spatial overlap between the two signals, NS1 and clathrin <b>(C)</b>, or NS1 and caveolin <b>(D)</b> in <b>Fig 4A</b> and <b>4B</b>, respectively. The means ± SEM of two individual experiments run in duplicate are shown. <b>(E)</b> Clathrin-mediated endocytosis inhibitor Pitstop 2 prevents colocalization of WT DENV NS1 with Rab5 in HPMEC. Colocalization of NS1 proteins (red, first column) on HPMEC treated with either DMSO (top row) or Pitstop 2 (bottom row) with Rab5 (green, second column) 1.5 hpt with NS1. Colocalization is shown in yellow in merge images (third column) or white in colocalization panels (fourth column; JACoP, ImageJ). Nuclei were stained with Hoechst (blue). Images (40X; scale bars, 5 μm) are representative of 2 independent experiments performed in duplicate. <b>(F)</b> Quantification of the amount of spatial overlap between the two signals, NS1 and Rab5 in <b>Fig 4E</b>. The means ± SEM of two individual experiments run in duplicate are shown. ns = not significant; **, p&amp;lt;0.01.</p>   </div> <p>To determine whether internalization of DENV NS1 is dependent on clathrin, we utilized a common chemical inhibitor of clathrin-mediated endocytosis. Pitstop 2 is a selective inhibitor of clathrin-mediated endocytosis that acts by blocking ligand access to the clathrin N-terminal domain [<a href="#ppat.1007938.ref031">31</a>]. Pretreating HPMEC with Pitstop 2 abrogated NS1 colocalization with Rab5, suggesting inhibition of internalization (<a href="#ppat.1007938.g004">Fig 4E and 4F</a>). These data suggest that DENV NS1 is internalized in a clathrin-dependent manner.</p> <p>Cathepsin L and downstream cellular enzymes are activated in endothelial cells treated with NS1 [<a href="#ppat.1007938.ref009">9</a>]; as such, we hypothesized that internalized NS1 would colocalize with cathepsin L. Since the peak of NS1-induced cathepsin L activation occurs as early as 30 minutes post-treatment [<a href="#ppat.1007938.ref009">9</a>], we conducted an internalization time-course experiment to monitor the lifespan of NS1 within cells to determine whether internalization kinetics correlate with cathepsin L activation. WT NS1 (DENV NS1+) was incubated with HPMEC for 45 minutes at 4°C to allow NS1 to bind to cells. Afterwards, cells were washed with cold PBS to remove any unbound NS1 and were then allowed to incubate at 37°C for varying times. Upon temperature shift to 37°C, NS1 accumulated rapidly in HPMEC, reaching a maximum at 15 minutes post-temperature shift, and then rapidly decreased until dropping to near undetectable levels at 3 hours post-temperature shift (<a href="#ppat.1007938.s010">S9A–S9C Fig</a>). These kinetics correlate well with the previously observed kinetics of NS1-triggered activation of cathepsin L [<a href="#ppat.1007938.ref009">9</a>]. Intriguingly, NS1 colocalized with the early endosome marker Rab5, the lysosomal marker LAMP1, and cathepsin L, suggesting that the presence of NS1 in these intracellular compartments may activate cathepsin L (<a href="#ppat.1007938.g003">Fig 3E</a>, <a href="#ppat.1007938.s010">S9D and S9E Fig</a>). These data indicate that NS1 rapidly and transiently internalizes into cells and localizes to compartments where cathepsin L is present, suggesting that internalization of NS1 may be required for activation of cathepsin L, and thus downstream disruption of the EGL.</p> <p>To test whether DENV NS1 internalization is required for EGL disruption, we evaluated the effect of Pitstop 2 treatment on cathepsin L activation and heparan sulfate degradation (<a href="#ppat.1007938.g005">Fig 5A and 5B</a>). Pretreatment of HPMEC with Pitstop 2 prevented the activation of cathepsin L and subsequent EGL degradation by WT DENV NS1, as demonstrated by decreased cathepsin L activity and reduced heparan sulfate on the surface of HPMEC, respectively (<a href="#ppat.1007938.g005">Fig 5A and 5B</a>). We next asked whether DENV NS1 internalization is required to trigger endothelial hyperpermeability <i>in vitro</i>. Dynasore, a common inhibitor of endocytosis, antagonizes the small GTPase dynamin, which is required for clathrin-mediated endocytosis [<a href="#ppat.1007938.ref032">32</a>, <a href="#ppat.1007938.ref033">33</a>]. Pretreatment of HPMEC with either Dynasore or Pitstop 2 abolished DENV NS1-triggered endothelial hyperpermeability as measured by TEER (<a href="#ppat.1007938.g005">Fig 5C</a> and <a href="#ppat.1007938.s011">S10 Fig</a>). To confirm that these inhibitors blocked clathrin-mediated endocytosis in our system, we measured the capacity of both Pitstop 2 and Dynasore to block internalization of transferrin, a protein well appreciated to be taken up via clathrin-mediated endocytosis [<a href="#ppat.1007938.ref034">34</a>]. As expected, Pitstop2 and Dynasore, but not the DMSO vehicle control, blocked internalization of transferrin into HPMEC (<a href="#ppat.1007938.s012">S11A and S11B Fig</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Addition of clathrin-mediated endocytosis inhibitors prevents EGL disruption and endothelial hyperpermeability triggered by DENV NS1.</div> <p><b>(A)</b> The integrity of the EGL on HPMEC was assessed by the presence of heparan sulfate [<a href="#ppat.1007938.ref009">9</a>] surface expression (green, bottom row), as well as cathepsin L (CTSL) activity (red, top row), at 6 hpt with DENV NS1 (5 μg/ml) in the presence or absence of Pitstop 2 (12.5 μM), or Pitstop 2 alone, at 37°C, as visualized via IFA. Nuclei were stained with Hoechst (blue). Images (20X; scale bars, 50 μm) are representative of 3 independent experiments. <b>(B)</b> Quantification of MFI in <b>(A)</b> from 3 independent experiments expressed as either fold change of cathepsin L activity from Pitstop 2 control values (left) or normalized percentage of heparan sulfate from Pitstop 2 control values [<a href="#ppat.1007938.ref010">10</a>]. Error bars indicate SEM. **, p&amp;lt;0.01; ***, p&amp;lt;0.001. <b>(C)</b> HPMEC were grown on Transwell semi-permeable membranes (0.4 μm pore size), and the apical chamber was treated with either DENV NS1+ (5 μg/ml) and DMSO (blue squares); the dynamin inhibitor Dynasore (25 μM) alone (purple diamonds) or Dynasore and NS1 (orange triangles); or the clathrin-mediated endocytosis inhibitor Pitstop 2 (12.5 μM) alone (light green triangles) or Pitstop 2 and NS1 (pink circles). A TEER assay was used to evaluate the effect of NS1 and inhibitors on endothelial permeability at indicated time-points over 48 hours. (^) represents change of medium. Relative TEER values from 2 independent experiments performed in duplicate are plotted. Error bars indicate standard error of the mean (SEM).</p>   </div> <p>Because chemical inhibitors can have off-target effects, we genetically knocked down clathrin, dynamin, and caveolin to confirm the role of clathrin-mediated endocytosis in DENV NS1 EGL disruption. Utilization of small interfering RNAs (siRNA) targeting either the clathrin heavy chain, dynamin I and II (I/II), or caveolin-1 mRNA enabled us to knock down expression of the proteins encoded by these transcripts. Knocking down clathrin and dynamin, but not caveolin, abolished NS1-mediated EGL disruption, as determined by heparan sulfate surface expression (<a href="#ppat.1007938.g006">Fig 6A and 6B</a>, <a href="#ppat.1007938.s013">S12 Fig</a>). To confirm that our siRNA knock-down experiments successfully blocked clathrin-mediated and caveolin-mediated endocytosis in our system, we conducted internalization assays in these cells using transferrin as well as human serum albumin, which is known to be internalized via caveolin-mediated endocytosis [<a href="#ppat.1007938.ref035">35</a>]. As expected, transferrin internalization was significantly inhibited by knocking down clathrin and dynamin expression, while internalization of human serum albumin was only inhibited by knocking down caveolin expression in cells (<a href="#ppat.1007938.s012">S11C, S11D</a> and <a href="#ppat.1007938.s013">S12</a> Figs), confirming the specificity of our siRNA system. These results indicate that NS1 is internalized via clathrin-mediated endocytosis, which is critical for NS1-induced hyperpermeability and EGL degradation.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Clathrin and dynamin, but not caveolin are required for the NS1-mediated disruption of the EGL.</div> <p><b>(A)</b> HPMEC were transfected with the indicated siRNA for 72 hours then treated with 5 ug/ml DENV2 NS1. The integrity of the EGL on HPMEC was assessed 6 hours post NS1-treatment by the presence of heparan sulfate (green) [<a href="#ppat.1007938.ref009">9</a>] on the cell surface. Nuclei were stained with Hoechst (blue). Images (20X; scale bars, 50 μm). <b>(B)</b> Quantification of MFI in (A) from 3 independent experiments. ns = not significant, **, P &amp;lt; 0.01.</p>   </div> <p>To determine whether clathrin-mediated endocytosis is required for NS1 to trigger vascular leak <i>in vivo</i>, we utilized a mouse model of localized leak in the dermis [<a href="#ppat.1007938.ref010">10</a>]. In this assay, NS1 or control proteins are injected intradermally (ID) into the depilated backs of mice; up to four different conditions can be tested per mouse. Fluorescently labeled soluble dextran (Alexa Fluor 680-dextran) is then administered intravenously, and accumulation of Alexa Fluor 680-dextran in the backs of mice (where ID injections of NS1 or controls occurred) is monitored at 2 hours post-dextran injection. We injected mice with NS1 or PBS mixed with a vehicle control or with a cocktail of Pitstop 2 and Dynasore to antagonize clathrin-mediated endocytosis locally (4 conditions total). Injection with NS1 mixed with a vehicle control showed significantly greater fluorescent signal (i.e., leakage) compared to the PBS conditions or NS1 mixed with the clathrin-mediated endocytosis inhibitor cocktail, suggesting that this cocktail blocked NS1-mediated leak in the dermis. These data suggest that NS1-triggered leakage <i>in vivo</i>, in the mouse dermis, requires clathrin-mediated endocytosis (<a href="#ppat.1007938.g007">Fig 7A and 7B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g007"><span class="label" tagx="label" title="label">Fig 7</span></a></div>   <div class="title" tagx="title" title="title">
DENV2 NS1-triggered vascular leak in the dorsal dermis of mice is prevented by clathrin-mediated endocytosis inhibitors.</div> <p>Hair was removed from the dorsal dermis of mice, and mice were allowed to recover for 3 days. On the day of the assay, retro-orbital injections of Alexa Fluor 680-conjugated dextran were administered, followed by intradermal injections of PBS plus vehicle (DMSO, top left circle), PBS plus inhibitor cocktail (top right circle), 15 μg DENV2 NS1 plus the DMSO vehicle (bottom left circle), and 15 μg DENV2 NS1 plus inhibitors (bottom right circle). The dermis from each mouse was collected and processed two hours post-injection. <b>(A)</b> Representative image of the dorsal dermis from 2 mice following the fluorescent dextran assay. <b>(B)</b> Dermises were scanned using a fluorescent detection system (LI-COR Odyssey CLx Imaging System) at a wavelength of 700 nm, and extravasated fluorescent dextran was quantified in tissue using Image Studio software (LI-COR Biosciences). Data represent the quantification of mean fluorescent intensity from mice in (A): PBS plus DMSO vehicle (closed black circles, n = 8); PBS plus inhibitors (open black circles, n = 4); DENV2 NS1–15 μg plus DMSO vehicle (closed blue diamonds, n = 8); and DENV2 NS1–15 μg plus inhibitors (open blue diamonds, n = 8). Data in (B) represent mean +/- SD and were collected from two independent experiments. All DENV NS1 proteins used in this experiment were purchased from the Native Antigen Company. A nonparametric Mann-Whitney U test was used to determine significance between treatment groups. ns = not significant, *, p &amp;lt; 0.05; **, p &amp;lt; 0.01. The cocktail of inhibitors was composed of Pitstop 2 (0.25 mg/ml) and Dynasore (0.5 mg/ml) to antagonize local clathrin-mediated endocytosis.</p>   </div> </div> <div class="then207glycosylationsiteofns1isconservedandisrequiredtotriggerendothelialhyperpermeabilitybymultipleflaviviruses" title="sec"> <div class="title" tagx="title" title="title">
The N207 glycosylation site of NS1 is conserved and is required to trigger endothelial hyperpermeability by multiple flaviviruses</div> <p>Given the importance of the N207 glycosylation site for DENV NS1-mediated endothelial hyperpermeability, we investigated its prevalence among related viruses and found a high level of conservation of this site in multiple flaviviruses (<a href="#ppat.1007938.g008">Fig 8A</a>). We then expanded our study to examine the role of the N207 glycosylation site of NS1 on the endothelial permeability of HBMEC in two closely related neurotropic flaviviruses–WNV and ZIKV. We first generated the WNV and ZIKV NS1-WT and NS1-N207Q mutant recombinant proteins <b>(</b><a href="#ppat.1007938.s001">S1 Table</a>). All proteins were secreted equivalently and appeared stable throughout the purification/dialysis procedure; further, all proteins were determined to be pure by silver staining and in a hexameric/oligomeric structure by native gels (<a href="#ppat.1007938.s014">S13A–S13F Fig</a>). We found that WT WNV and ZIKV NS1 (commercial or produced in-house) triggered endothelial hyperpermeability as measured by TEER, indicating that WT NS1 proteins are capable of inducing endothelial hyperpermeability in HBMEC [<a href="#ppat.1007938.ref036">36</a>, <a href="#ppat.1007938.ref037">37</a>] (<a href="#ppat.1007938.g008">Fig 8B and 8C</a> and <a href="#ppat.1007938.s015">S14A and 14B Fig</a>). In contrast to the WT NS1 proteins, both the WNV NS1-N207Q and ZIKV NS1-N207Q mutants failed to trigger endothelial hyperpermeability of HBMEC, suggesting that the N207 glycosylation site is a critical determinant for endothelial hyperpermeability induced by NS1 from multiple flaviviruses (<a href="#ppat.1007938.g008">Fig 8B and 8C</a> and <a href="#ppat.1007938.s015">S14A and S14B Fig</a>). To determine whether the N207 residue is also required for internalization of NS1 from related flavivirus such as WNV and ZIKV, as it is for DENV NS1, we conducted a binding/internalization assay of NS1-WT and NS1-N207Q from DENV, WNV, and ZIKV using HPMEC and HBMEC. In this internalization assay, we measured colocalization of NS1 with Rab5 and interpreted Rab5-positive NS1 puncta to represent binding and internalization while Rab5-negative puncta represent binding only. As observed previously [<a href="#ppat.1007938.ref037">37</a>], DENV NS1-WT bound well to both HPMEC and HBMEC while WNV and ZIKV NS1 only efficiently bound to HBMEC, and this was also the case for the NS1-N207Q mutants (<a href="#ppat.1007938.s016">S15A–S15C Fig</a>). Further, within HBMEC, to which ZIKV and WNV NS1 efficiently bound, the NS1-WT proteins but not the NS1-N207Q mutants of DENV, WNV, and ZIKV were efficiently internalized (<a href="#ppat.1007938.s016">S15A–S15C Fig</a>). These data suggest that like DENV NS1, the N207-glycosylation site is critical for internalization of WNV and ZIKV NS1 proteins. We next investigated whether clathrin-mediated endocytosis was a common pathway for flavivirus NS1-mediated pathogenesis. We found that Pitstop 2 blocked heparan sulfate shedding on HBMEC induced by WT WNV NS1 and WT ZIKV NS1 (<a href="#ppat.1007938.g008">Fig 8D–8E</a>). Taken together, these results indicate that internalization of NS1 via a clathrin-mediated endocytosis pathway is a conserved mechanism for NS1-induced endothelial hyperpermeability for multiple flaviviruses.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g008"><span class="label" tagx="label" title="label">Fig 8</span></a></div>   <div class="title" tagx="title" title="title">
Mutation of the N207 glycosylation site abrogates endothelial hyperpermeability induced by NS1 proteins from WNV and ZIKV.</div> <p>(<b>A</b>) Amino acid sequence alignment of flavivirus NS1 sequences indicate that N207 glycosylation site is highly conserved. Accession numbers for each NS1 protein sequences are listed after the corresponding flavivirus names. Sequence analysis was performed using MultAlin and recreated in Excel. The amino acid sequence of NS1 from aa 205–211 is shown. <b>(B-C)</b> Human brain microvascular endothelial cells (HBMEC) were grown on Transwell semi-permeable membranes (0.4 μm pore size), and WNV NS1 <b>(B)</b> or ZIKV NS1 <b>(C)</b> proteins (5 μg/ml) were added to the apical chamber (NS1+, commercially purchased NS1, blue squares; NS1-WT, produced in-house, purple diamonds; NS1-N207Q mutant, orange triangles). A TEER assay was used to evaluate the effect of these NS1 proteins on endothelial permeability at indicated time-points over 48 hours. (^) represents change of medium. Relative TEER values from 2 independent experiments performed in duplicate are plotted. Error bars indicate SEM. <b>(D)</b> The integrity of the EGL on HBMEC was assessed by the presence of heparan sulfate surface expression (green) at 6 hpt with WNV NS1 or ZIKV NS1 (5 μg/ml) in the presence or absence of Pitstop 2 (12.5 μM) at 37°C, as visualized via IFA. Nuclei were stained with Hoechst (blue). Images (20X; scale bars, 50 μm) are representative of 3 independent experiments. (<b>E</b>) Quantification of MFI in (D) from 3 independent experiments. ***, p &amp;lt; 0.001.</p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this study, we demonstrate that the N207 glycosylation site of DENV NS1 is essential for inducing EGL degradation and hyperpermeability of human endothelial cells. We found that the DENV NS1-N207Q mutant binds to cells at similar levels as WT DENV NS1 but is retained on the cell surface, in contrast to WT DENV NS1, which is rapidly internalized. Additionally, internalization of WT DENV NS1 was dynamin- and clathrin-dependent, but independent of caveolin. Further, clathrin-mediated endocytosis of WT DENV, WNV, and ZIKV NS1 was required for endothelial hyperpermeability and EGL disruption. Taken together, these results indicate that the N207 glycosylation site is required for endothelial cell internalization via clathrin-mediated endocytosis and endosomal trafficking of NS1, which is necessary for the activation of enzymes such as cathepsin L that lead to EGL degradation and increased endothelial permeability (<a href="#ppat.1007938.g009">Fig 9A and 9B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007938.g009"><span class="label" tagx="label" title="label">Fig 9</span></a></div>   <div class="title" tagx="title" title="title">
A single N-glycosylation mutant (N207Q) abrogates clathrin mediated-endocytosis of different flavivirus NS1 proteins in human endothelial cells, which is required to trigger EGL disruption and vascular leakage.</div> <p><b>(A)</b> The flavivirus NS1 proteins are secreted as hexamers and contain two glycosylation sites (DENV, ZIKV, JEV, and YFV; aa 130 and 207) or three glycosylation sites (WNV; aa 130, 175, and 207). <b>(B)</b> Overview of NS1 effects on endothelial cells. <b>(B1)</b> Both NS1-WT and the NS1-N207Q mutant bind to glycosaminoglycans (GAGs; e.g., heparan sulfate) on endothelial cells (ECs), decorating the endothelial glycocalyx layer (EGL) and the cell surface. <b>(B2)</b> Both WT and the N207Q mutant of NS1 trigger recruitment of clathrin triskelion proteins to the plasma membrane. However, only the WT protein can induce maturation and scission of the clathrin-coated pit in a dynamin-dependent manner, while the N207Q mutant appears to remain sequestered on the cell surface. The use of specific chemical inhibitors for clathrin (i.e., Pitstop2) and dynamin (i.e. Dynasore) and siRNA transfection with siClathrin and siDynamin prevent NS1 internalization into HPMEC. <b>(B3)</b> This internalization process results in NS1 colocalization with an early endosome marker, Rab5. Clathrin-mediated endocytosis of NS1 (WT) triggers remodeling of the EGL after activation of endosomal-resident proteases such as cathepsin L, which induces activation of the endoglycosidase heparanase, inducing degradation of heparan sulfate on the EGL, leading to endothelial hyperpermeability and vascular leakage.</p>   </div> <p>Using a transgenic expression system, it was previously shown that the N130 glycan of DENV NS1 was required for stabilization of the secreted NS1 hexamer whereas the N207 glycan facilitated secretion and extracellular protein stability [<a href="#ppat.1007938.ref019">19</a>]. In our system, the N-glycan at position 130 was required for protein secretion while the N-glycan at position 207 was dispensable for protein secretion and stability, in contrast to Somnuke et al. These differences could be explained by the methods used to produce and purify NS1 proteins between the two studies (<i>e</i>.<i>g</i>., a different signal sequence mediating secretion and a different strategy and methodology utilized for protein purification). In our current mammalian protein expression system utilizing 293F suspension cells, the DENV NS1-N207Q mutant was efficiently secreted; however, neither the DENV NS1-N130Q mutant nor the double mutant N130Q+N207Q was efficiently secreted, suggesting that the N130 N-glycan site is essential for NS1 secretion. These observations are supported by a previous study that demonstrated that ablation of the homologous glycosylation sites in yellow fever virus NS1 resulted in viral mutants with impaired NS1 secretion [<a href="#ppat.1007938.ref012">12</a>]. Due to this technical limitation, the N130Q and N130Q+N207Q constructs were not further characterized in this study. However, based on the significant homology at NS1-N130 across all flaviviruses, it is possible that NS1-N130 may also carry out important pathological functions, which urge further investigation.</p> <p>We have previously shown that NS1 activates key endothelial cell-intrinsic pathways, including the sialidase and cathepsin L/heparanase pathways, and that these contribute to NS1-induced hyperpermeability of human endothelial cells [<a href="#ppat.1007938.ref009">9</a>]; however, how these enzymes become activated remain unknown [<a href="#ppat.1007938.ref009">9</a>]. Our results here provide evidence that internalization of NS1 via clathrin-mediated endocytosis is required for the activation of these pathogenic processes and demonstrate that NS1 localization in endosomes and lysosomes, where cathepsin L is localized, correlates with cathepsin L activation. Further studies defining the biochemical interactions required for this activation are needed and are an area of active investigation.</p> <p>Our data suggest that NS1 binding and internalization are distinct steps in triggering endothelial hyperpermeability. Because both WT DENV NS1 and DENV NS1-N207Q are able to bind to HPMEC, but only WT DENV NS1 is internalized, we hypothesize that NS1 initially binds to heparan sulfate moieties on endothelial cells [<a href="#ppat.1007938.ref025">25</a>], independently of the N207 N-glycan. DENV NS1 may then interact with an additional binding partner(s) in a manner dependent on the N207 N-glycan, which leads to internalization, endosomal trafficking, and activation of cathepsin L and/or sialidases. Why is the N207 N-glycosylation site critical for internalization of DENV NS1? First, the N207 N-glycan may be required for engaging a specific receptor on endothelial cells, either directly through a physical interaction or indirectly through a conformational change. Based on the available crystal structure of NS1, amino acid 207 is not predicted to be surface-exposed [<a href="#ppat.1007938.ref014">14</a>]. This observation favors a model where a conformational change of NS1 allows this N-glycan to be exposed and mediate an interaction required for internalization or where the N-glycan itself mediates a conformational change of NS1 that allows another domain to mediate an interaction required for internalization. Second, glycosylation patterns have been shown to be important for lateral diffusion of proteins on the plasma membrane [<a href="#ppat.1007938.ref038">38</a>, <a href="#ppat.1007938.ref039">39</a>]. Thus, an alternate possibility is that the N207 N-glycan is required for NS1 to migrate to the proper microdomain to interact with functional components on the lipid membrane required for internalization. Furthermore, NS1 is secreted from DENV-infected cells predominantly as a soluble hexameric barrel-shaped, high-density lipoprotein with a hydrophobic core containing lipid cargos such as triglycerides, cholesteryl esters, and phospholipids [<a href="#ppat.1007938.ref015">15</a>, <a href="#ppat.1007938.ref040">40</a>, <a href="#ppat.1007938.ref041">41</a>]. The functional importance, if any, of these lipid cargos is yet to be determined. A third possibility for the role of the N207 N-glycan in NS1 internalization is that this sugar moiety may be important for acquisition of specific lipid cargos that play a role during NS1 binding to endothelial cells or even internalization; thus, further investigation of the lipid contents of the mutant NS1 is needed to uncover a potential role of the N207 N-glycan in lipid cargo acquisition as well as a possible role of these lipid cargos in NS1-mediated pathogenesis [<a href="#ppat.1007938.ref042">42</a>].</p> <p>Intriguingly, the NS1-N207Q mutant binds to cells and even recruits clathrin to the cell surface, but fails to be internalized and traffic into early endosomes. Because of these observations, an important remaining question is at what stage of the internalization process is the NS1-N207Q mutant blocked? Cell binding of NS1-N207Q alone may be sufficient to trigger the proper signaling pathway(s) to recruit clathrin to the cell surface but may not be sufficient to trigger the next stage of internalization, perhaps mediated through the engagement of the N207 N-glycan to a receptor. Previous reports of stalled clathrin-coated pits suggest that recruitment of additional factors, such as the clathrin-interacting protein epsin, are required for full maturation and budding of the endosome into the cell [<a href="#ppat.1007938.ref030">30</a>, <a href="#ppat.1007938.ref043">43</a>]. Thus, another potential explanation may be that the N207 N-glycan is required to recruit additional factors to the clathrin-coated pits that promote internalization. An understanding of the role that other cellular factors, like epsins, play in the maturation of NS1-containing clathrin-coated pits and subsequent internalization is needed to fully elucidate the internalization defect of NS1-N207Q.</p> <p>Clathrin-mediated endocytosis of NS1 may have implications beyond NS1-triggered endothelial hyperpermeability. It is known that DENV invades cells via endocytosis [<a href="#ppat.1007938.ref044">44</a>], and pharmacologically blocking clathrin-mediated endocytosis retards DENV replication [<a href="#ppat.1007938.ref045">45</a>]. Further, a recent study demonstrated that DENV NS1 interacts with DENV structural proteins, which modulates infectious viral particle production [<a href="#ppat.1007938.ref046">46</a>]. In light of our finding that NS1 is internalized via clathrin-mediated endocytosis, NS1 could potentially play a role in mediating clathrin-mediated endocytosis of flavivirus virions. Further studies are needed to elucidate, and potentially differentiate, clathrin-mediated endocytosis of virions via the E protein and/or NS1.</p> <p>Regarding overall pathogenic mechanisms, our investigation specifically focuses on a cell-intrinsic pathway by which NS1 mediates the disruption of the EGL, but flavivirus-induced vascular dysfunction is a complex and multifactorial process. It can be mediated through the NS1 endothelial cell-intrinsic pathway [<a href="#ppat.1007938.ref009">9</a>, <a href="#ppat.1007938.ref010">10</a>, <a href="#ppat.1007938.ref036">36</a>, <a href="#ppat.1007938.ref037">37</a>], through production of proinflammatory cytokines by immune cells induced by either the virus or NS1 [<a href="#ppat.1007938.ref011">11</a>, <a href="#ppat.1007938.ref047">47</a>], through NS1-activated platelets [<a href="#ppat.1007938.ref048">48</a>], or even potentially through direct viral infection leading to apoptosis of endothelial cells [<a href="#ppat.1007938.ref049">49</a>, <a href="#ppat.1007938.ref050">50</a>]. Though endothelial cells can be infected <i>in vitro</i> by DENV, findings from autopsy studies suggest that this does not occur <i>in vivo</i> [<a href="#ppat.1007938.ref022">22</a>, <a href="#ppat.1007938.ref051">51</a>, <a href="#ppat.1007938.ref052">52</a>]. ZIKV infects human endothelial cells <i>in vitro</i> and fetal endothelial cells in a mouse model [<a href="#ppat.1007938.ref004">4</a>, <a href="#ppat.1007938.ref053">53</a>, <a href="#ppat.1007938.ref054">54</a>], suggesting another pathway by which ZIKV may trigger endothelial dysfunction. Dissecting the relative contributions of each of these pathways in flavivirus-induced pathology <i>in vivo</i> is critical for designing appropriate therapeutics that could be used to treat flavivirus-induced vascular dysfunction.</p> <p>All flaviviruses must disseminate from the blood into specific tissues, where they replicate to high levels. They can do this by infecting migrating immune cells, direct infection of endothelial cells [<a href="#ppat.1007938.ref055">55</a>], or through dysfunctional endothelial cell barriers [<a href="#ppat.1007938.ref036">36</a>, <a href="#ppat.1007938.ref056">56</a>]. Our data indicate that the cell-intrinsic endothelial dysfunction pathway is triggered not only by DENV NS1, but also by neurotropic WNV and ZIKV NS1 in HBMEC. Our previous study demonstrates that NS1 from multiple flaviviruses (including DENV, WNV, and ZIKV) mediates endothelial dysfunction and vascular leak in a tissue-specific manner reflecting viral tropism [<a href="#ppat.1007938.ref037">37</a>]. We speculate that this conserved NS1-mediated cell-intrinsic endothelial dysfunction pathway mediates virus dissemination of different flaviviruses into an appropriate tissue, promoting viral replication.</p> <p>In summary, our current study not only identifies the mechanism of internalization of secreted NS1 by human endothelial cells via a dynamin- and clathrin-mediated, caveolin-independent endocytosis pathway, but also pinpoints a single amino acid, the N207 glycosylation site of NS1, as a key determinant of NS1-mediated endothelial hyperpermeability. Mutation of N207 prevents NS1 internalization by human endothelial cells and inhibits disruption of the EGL and endothelial barrier function <i>in vitro</i> and <i>in vivo</i>. Further, our work identifies N207 of NS1 as a conserved functional residue among multiple flaviviruses (ZIKV and WNV, in addition to DENV) that is essential for flavivirus NS1-mediated endothelial dysfunction via clathrin-mediated endocytosis in human cells. This new molecular insight into the mechanism of flavivirus NS1-triggered endothelial hyperpermeability is crucial for understanding pan-flaviviral pathogenesis, as well as for developing antiviral therapies and NS1-based vaccine approaches.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="cloningofrecombinantns1constructs" title="sec"> <div class="title" tagx="title" title="title">
Cloning of recombinant NS1 constructs</div> <p>A panel of recombinant DENV2 (strain 16681), WNV (NY99), and ZIKV (Nica1-16) NS1 proteins was generated, including wild-type (WT) and site-specific mutant constructs, and the primers utilized are listed in <a href="#ppat.1007938.s001">S1 Table</a>. Amplicons consisting of the WT DENV or WNV NS1 gene preceded by the CD33 signal sequence and followed by a 6xHis-tag were a gift from M.S. Diamond (Washington University in St. Louis), along with the pmab vector. The WT ZIKV NS1 gene fragment was designed according to the ZIKV Nica1-16 strain sequence [<a href="#ppat.1007938.ref004">4</a>] and synthesized by Bio Basic Inc. To introduce mutations, the Phusion High-Fidelity DNA Polymerase (New England BioLabs; NEB) was used to perform overlap-extension PCR according to the manufacturer’s instructions. XbaI and MluI-HF restriction endonucleases (NEB) were used to digest the PCR amplicons and the pmab vector. A Quick Ligation (NEB) was performed to ligate the inserts with the vector, and 5-alpha Competent <i>E</i>. <i>coli</i> cells (NEB) were used for transformation. Colonies were selected based on expected XbaI and MluI-HF double digestion profiles and correct Sanger sequencing results.</p> </div> <div class="cellculture" title="sec"> <div class="title" tagx="title" title="title">
Cell culture</div> <p>FreeStyle 293F suspension cells (Thermo Fisher Scientific) derived from human embryonic kidney cells were cultured in FreeStyle 293 Expression medium with 1% penicillin/streptomycin at 37°C and 8% CO<sub>2</sub> and were maintained at 0.15–1.2 x 10<sup>6</sup> cells/ml on a cell shaker at 135 rpm. The HPMEC-ST1.6r line was kindly donated by Dr. J.C. Kirkpatrick at Johannes Gutenberg University, Germany, and was grown using the EGM-2 bullet kit (Clonetics, Lonza) and maintained as previously described [<a href="#ppat.1007938.ref008">8</a>]. HBMEC were donated by Dr. Ana Rodriguez at New York University and maintained using endothelial cell medium with growth supplement (ScienCell Research Labs).</p> </div> <div class="293ftransfectionandns1proteinpurification" title="sec"> <div class="title" tagx="title" title="title">
293F transfection and NS1 protein purification</div> <p>Transfections were performed using FreeStyle 293F suspension cells and FreeStyle MAX transfection reagent (Thermo Fisher Scientific) according to the manufacturer’s protocol, and supernatant containing NS1 were collected 48 hours post-transfection. To prevent protein degradation, a tablet of EDTA-free protease inhibitor cocktail (Roche Life Sciences) was added to every 150 ml of supernatant collected. Cell supernatants were stored at -80°C prior to purification. NS1-containing supernatants were then thawed and concentrated in an Amicon filter with a 100-KDa size cutoff (UFC9100, Millipore). The 6xHis-tagged recombinant DENV NS1 proteins were purified by batch method using nickel nitrilotriacetic acid (Ni-NTA) resin agarose beads (Thermo Fisher Scientific). In brief, Ni-NTA resin agarose beads were first equilibrated in binding buffer (20 mM sodium phosphate, 500 mM sodium chloride, 20 mM imidazole, pH 7.4), and then supernatants containing NS1 were diluted 1:2 in 2X binding buffer and added to the previously equilibrated Ni-NTA resin agarose beads. NS1-containing supernatants were allowed to bind to the Ni-NTA resin for 30 minutes at 4°C rocking end-over-end. Once binding was completed, the resin was washed 4X in wash buffer (20 mM sodium phosphate, 500 mM sodium chloride, 30 mM imidazole, pH 7.4). Concentrated NS1 was eluted from the Ni-NTA resin using an imidazole-containing elution buffer (20 mM sodium phosphate, 500 mM sodium chloride, and 200 mM imidazole, pH 7.4). This purified NS1 stock was then subjected to dialysis against 1X PBS for 48 hours at 4°C. Typical concentrations of NS1 stocks obtained ranged from 0.1–0.5 mg/ml. The Pierce BCA Protein Assay Kit (Thermo Fisher Scientific) was used to quantify the purified recombinant proteins.</p> </div> <div class="westernblot" title="sec"> <div class="title" tagx="title" title="title">
Western blot</div> <p>For the Endo H/PNGase F digestion Western blots, protein stability assays Western blots, NS1 internalization kinetic Western blots, siRNA knockdown Western blots, and ZIKV/WNV native-PAGE Western blots total cell lysates or recombinant proteins were collected in protein sample buffer (0.1 M Tris [pH 6.8], 4% SDS, 4 mM EDTA, 286 mM 2-mercaptoethanol, 3.2 M glycerol, 0.05% bromophenol blue) and then resolved by SDS-PAGE. The same protocol and buffer were used for native-PAGE but without SDS. The resolved proteins were then transferred onto nitrocellulose membranes and probed with primary antibodies diluted in TBS/0.1% Tween 20 (TBST) containing 5% skim milk, rocking overnight at 4°C. Secondary antibodies were diluted in TBST with 5% skim milk for 1-hour rocking at room temperature. After each antibody incubation, membranes were washed three times for 5 minutes each with TBST. Membranes were visualized using ECL reagents on a ChemiDoc system with Image Lab software from Bio-Rad. All antibodies were probed by the above method except rabbit anti-Caveolin1 and rabbit anti-DynaminI/II which were probed in TBST with 5% bovine serum albumin according to the manufacturer’s instructions. The following antibodies were used: mouse anti-6xHis (MA1-21315, Thermo Scientific), anti-NS1 monoclonal antibody (7E11), rabbit anti-clathrin heavy chain (ab172958, Abcam), rabbit anti-Dynamin I/II (2342S, Cell Signaling), rabbit anti-caveolin 1 (3267, Cell Signaling), mouse anti-βactin (sc-47778, Santa Cruz Biotechnology), mouse anti- αtubulin (ab4074, Abcam), goat anti-mouse HRP (405306, Biolegend), and donkey anti-rabbit HRP (406401, Biolegend). For the N-glycan digestion Western blots, 1 μg of denatured NS1s were digested with 750 units Endo H (NEB) or 500 units PNGase F (NEB) treatment at 37°C for 1 hour according to manufacturer’s protocol. For the NS1 stability assays 100 ng of protein were mixed with EGM-2 media (Lonza) and incubated in a tissue-culture incubator for the times indicated in the figures.</p> <p>For all other Western blots, SDS-PAGE gels were transferred semi-dry onto a polyvinylidene difluoride (PVDF) membrane for 7 minutes [<a href="#ppat.1007938.ref057">57</a>] at 25V. The membrane was blocked and incubated with primary anti-6xHis-tag antibody overnight. After washing, a goat anti-mouse IgG secondary antibody conjugated to Alexa Fluor 680 (Thermo Fisher Scientific) was added at a dilution of 1:5,000 for 1 hour, and blots were imaged using the LI-COR Odyssey imaging system. Anti-NS1 mAb 7E11 (generously donated by R. Putnak at the Walter Reed Army Institute of Research) was used for detection of DENV NS1.</p> </div> <div class="size-exclusionchromatography" title="sec"> <div class="title" tagx="title" title="title">
Size-exclusion chromatography</div> <p>0.25 milligrams (500 ul of a 0.5 mg/ml stock) of purified and dialyzed NS1-WT and NS1-N207Q were injected into a Superose 6 Increase 10/300 GL column (GE Life Sciences) previously equilibrated with 1X PBS. Validation of peaks, detected by UV light, were done by evaluating 2.5 ul of a given fraction by SDS-PAGE and probing with a NS1-speficic monoclonal antibody (7E11).</p> </div> <div class="enzyme-linkedimmunosorbentassay(elisa)" title="sec"> <div class="title" tagx="title" title="title">
Enzyme-linked Immunosorbent Assay (ELISA)</div> <p>NS1 capture ELISA: MaxiSorp ELISA plates (Thermo Scientific Nunc) were coated with 50 μl of an anti-NS1 specific monoclonal antibody (7E11) (5 μg/ml) and incubated overnight at room temperature. The next day the plate was blocked with 1% BSA in PBS for 1 hour then washed twice with 1X PBS. Then 50 ul of NS1 containing supernatant was added to wells and incubated for 1 hour at room temperature. Plates were then washed 3 times in PBS + 0.1% Tween20 and incubated for 1 hour at room temperature with another NS1-specific monoclonal antibody (2B7). After an additional three washes in PBS + 0.1% Tween20 a peroxidase-labeled goat anti-mouse secondary antibody (Jackson ImmunoResearch) at 0.5 μg/ml in 1% BSA-PBS was added to the wells and incubated for an additional hour. After 3 additional washing steps, the assay was developed using an ABTS-HRP substrate kit (KPL), according to the manufacturer’s specifications.</p> <p>NS1 direct ELISA: To examine the conformational status of the recombinant NS1 proteins used in this study, we performed a NS1 direct ELISA using a panel of recombinant NS1 proteins (NS1+, NS1-WT, and NS1-N207Q) and anti-NS1 monoclonal antibodies (mAb) that interacts with the soluble NS1 protein in a conformational-dependent manner. Briefly, ELISA plates (Nunc, Thermo Scientific) were coated overnight at 4°C with a single concentration of NS1 (200 ng/mL) diluted in sterile DPBS (1X, Gibco) under two conditions: native (no SDS) or denaturing (SDS 0.1%, boiling at 99°C for 5 minutes). The next day, plates were washed twice using wash buffer (PBS + 0.05% Tween-20) and blocked using blocking buffer (3% bovine albumin, BSA in washing buffer) for 45 minutes at room temperature (RT). Then, plates were probed individually with anti-NS1 mAbs (0.25 μg/mL) which bind NS1 proteins under denaturing conditions (e.g. 7E11 and 2B7) or native conditions (e.g. 9NS1). All recombinant NS1 proteins in this study contain a 6xHis-tag. The anti-6xHis-tag mAb (0.25 μg/mL, Thermo Scientific) was used as positive control for NS1 detection. A peroxidase conjugated goat-anti mouse IgG (H+L) antibody (Jackson Immunoresearch Inc.,)(1:5000) and TMB (3,3′,5,5′-Tetramethylbenzidine, Sigma) were used to develop the assay. Plates were allowed to develop for 30 minutes at room temperature and were then stopped with a solution of sulfuric acid (H<sub>2</sub>SO<sub>4</sub>, 2N). Plates were read at a 450 nm wavelength using a microplate reader.</p> </div> <div class="transendothelialelectricalresistance(teer)" title="sec"> <div class="title" tagx="title" title="title">
Transendothelial electrical resistance (TEER)</div> <p>To measure the functional effects of the recombinant NS1 proteins, HPMEC and HBMEC were cultured in duplicate wells (80,000 cells in 0.3 ml) in the apical chamber of 24-well Transwell polycarbonate membrane inserts (Transwell permeable support, 0.4 μm, 6.5 mm insert; Corning Inc.) and incubated with 5 μg/ml NS1 protein (WT or mutants, 1.5 μg total protein). TEER was measured as previously described [<a href="#ppat.1007938.ref008">8</a>, <a href="#ppat.1007938.ref009">9</a>]. Resistance values were measured in Ohms (Ω) at sequential 2-hour time-points following the addition of test proteins using an Epithelial Volt Ohm Meter (EVOM) with "chopstick�? electrodes (World Precision Instruments). Untreated endothelial cells grown on Transwell inserts were used as negative untreated controls, and inserts with medium alone were used for blank resistance measurements. Relative TEER represents a ratio of resistance values (Ω) as follows: (Ω experimental condition-Ω medium alone)/(Ω non-treated endothelial cells-Ω medium alone).</p> </div> <div class="ns1proteinbindingassay" title="sec"> <div class="title" tagx="title" title="title">
NS1 protein binding assay</div> <p>Confluent HPMEC monolayers grown on gelatin-coated coverslips (0.2%, Sigma) were treated with 5 μg/ml of DENV2 NS1+ from the Native Antigen Company (Oxfordshire, UK) as a positive control, WT DENV NS1 produced in-house, or the DENV NS1-N207Q mutant and incubated for 1 hour at 37°C/4°C or for 6 hours at 37°C, depending on the experiment. To remove heparan sulfate from the cell surface, HPMEC were treated for 24 hours, prior to NS1 treatment with 0.5 units of recombinant heparanase (H3917-50 UN, Sigma). An anti-6xHis-tag antibody conjugated to DyLight 647 (Thermo Fisher Scientific) was used to detect NS1 protein bound to the cell surface. Untreated cells were used as a negative control. Images were acquired using a Zeiss LSM 710 AxioObserver-34-channel spectral detector confocal microscope and processed using ImageJ software. For MFI quantification, the threshold for each individual channel (RGB) was adjusted and converted to grayscale. Then, mean grayscale values and integrated density from selected areas were obtained, along with adjacent background readings, and plotted as mean fluorescence intensity (MFI).</p> </div> <div class="immunofluorescentmicroscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescent microscopy</div> <p>HPMEC were grown and imaged as described above. To assess the effect of flavivirus NS1 on the integrity of EGL components such as sialic acid and heparan sulfate, HPMEC monolayers were treated with DENV WT (Native Antigen or produced in-house) or NS1-N207Q mutant proteins (5 μg/ml) and fixed with 4% paraformaldehyde (PFA) at different time-points (1, 6 hpt). A primary antibody against heparan sulfate (clone F58-10E4, Amsbio) was incubated overnight at 4°C, and detection was performed using a secondary goat anti-mouse IgM antibody conjugated to Alexa 488 (Thermo Fisher Scientific). The cell surface expression of sialic acid was visualized using the lectin wheat germ agglutinin (WGA) conjugated to Alexa 647 (Thermo Fisher Scientific) [<a href="#ppat.1007938.ref009">9</a>]. The proteolytic activity of cathepsin L was evaluated using the Magic Red assay cathepsin L detection kit (ImmunoChemistry Technologies) [<a href="#ppat.1007938.ref009">9</a>]. Nuclei were stained with Hoechst (blue). Untreated cells were used as a control for basal sialic acid and heparan sulfate expression and cathepsin L activity. Images were acquired using the Zen 2010 software (Zeiss) and analyzed with ImageJ software. For representative pictures, an area of ~1.8 μm<sup>2</sup> (1.25x1.40 μm) containing ~28–30 cells was used. MFI quantification was performed as described above.</p> </div> <div class="ns1proteininternalizationinhpmecandhbmec" title="sec"> <div class="title" tagx="title" title="title">
NS1 protein internalization in HPMEC and HBMEC</div> <p>To evaluate the internalization of DENV NS1 proteins in HPMEC, we used Western blot and IFA. Briefly, confluent monolayers (pre-chilled for 10 minutes at 4°C) grown on culture plates and/or glass coverslips were exposed to 10 μg/ml of different NS1 proteins (as indicated above) and incubated for 45 minutes at 4°C to facilitate NS1 protein adsorption but not internalization. Cultures were then transferred to 37°C to allow protein internalization. For the internalization kinetic experiments, cells were washed twice with PBS, before transfer to 37°C, to remove any unbound NS1. One hour later (or at the indicated times for kinetic analysis), cell supernatants were removed, and cell monolayers were rinsed 3X with PBS and detached with trypsin-EDTA, which also removed any surface-bound, non-internalized NS1 proteins from the outer cell membrane. Cells were pelleted and lysed using SDS gel loading buffer, then the cell lysate was separated using an SDS-PAGE gradient gel (4–20%) and visualized by Western blot with primary anti-NS1 mAbs (1G5.3 and 1H7.4, obtained from Paul Young, University of Queensland, Australia) [<a href="#ppat.1007938.ref058">58</a>] and secondary anti-mouse antibody conjugated to Alexa 750 (ab175740, Abcam). α-Tubulin was used as a protein loading control (anti-α-tubulin ab4074, Abcam), and Rab5 was used as an early endosome marker (ab18211, Abcam). Anti-rabbit IgG conjugated to Alexa 680 was used as secondary antibody (ab175773, Abcam). Images were acquired and processed using a LI-COR Odyssey system and ImageJ software, respectively. For the internalization Western blot, at the indicated time post temperature shift, cell monolayers were rinsed 3X with PBS and then 2X with pre-chilled acid wash buffer (glycine 100 mM, and 150mM NaCl, pH 2.5), which removed any surface-bound, non-internalized NS1 proteins from the outer cell membrane. Cells were then collected in western blot sample buffer.</p> <p>Additionally, internalized NS1 (10 μg/ml) was visualized by IFA by co-staining NS1 and Rab5. Briefly, after 45 minutes at 4°C (to normalize protein adsorption), cultures were transferred to 37°C for 1 hour (or the time indicated in the figure for the kinetic analysis) to facilitate protein internalization. For the internalization kinetic experiments, cells were washed twice with PBS before transfer to 37°C, to remove any unbound NS1. Before fixation of samples within the kinetic analysis, cell supernatants were removed, and cell monolayers were rinsed 3X with PBS and then 2X with pre-chilled acid wash buffer (glycine 100 mM, and 150mM NaCl, pH 2.5), removing surface-bound NS1. After fixation with 4% PFA, samples were processed by indirect IFA and confocal microscopy imaging. For experiments using inhibitors, compounds were added to wells 30 minutes before the addition of DENV NS1 protein. Initially, a co-localization plugin analysis in ImageJ software was used to define the co-localizing spots between NS1 and Rab5 in each experimental condition. The amount of spatial overlap between the two signals (NS1 "red�? and Rab5 "green�?) was obtained using four different frames from the maximum projections of two RGB images based on the object-based approach (JACoP) and defined by the Manders’ Coefficient, as previously described [<a href="#ppat.1007938.ref059">59</a>]. As an additional analysis to quantify the amount of internalized NS1 protein in HBMEC and HPMEC monolayers by IFA, the number of NS1-positive (NS1<sup>+</sup>) puncta per cell (n = 200 cells) and those co-stained with the early endosome marker Rab5 (NS1<sup>+</sup>Rab5<sup>+</sup>) were manually counted using the multi-point tool in ImageJ.</p> </div> <div class="z-stackvideooverlayofns1internalization" title="sec"> <div class="title" tagx="title" title="title">
Z-stack video overlay of NS1 internalization</div> <p>An animated single 2D projection of NS1 internalization in HPMEC was created by collapsing multiple image-sections (maximum projections) obtained from confocal Z-stack acquisition analyses. Briefly, HPMEC monolayers grown on glass coverslips were treated with DENV NS1 (WT), and the glycosylation mutant (DENV NS1-N207Q) for 1 hour at 37°C. Cell monolayers were processed for immunofluorescence assay as described above co-staining for NS1 protein and the early endosome marker Rab5. Z-stacks were acquired on a Zeiss LSM 710 inverted confocal microscope (CRL Molecular Imaging Center, UC Berkeley) using a Plan-Apochromat 40X dry objective. Projections of multiple Z-stack sections (20 μm range: 20 slices; 1 μm interval) were performed and animated using Z stack reconstruction plugin in Image J, then saved as Audio Video Interleave (AVI) format. Scale bar, 1 μm.</p> </div> <div class="ns1colocalizationwithclathrinandcaveolin" title="sec"> <div class="title" tagx="title" title="title">
NS1 colocalization with clathrin and caveolin</div> <p>IFA was used to determine whether clathrin-mediated endocytosis or caveolin-mediated uptake contribute to internalization of NS1 by endothelial cells. HPMEC or HBMEC were grown and imaged as described above. Briefly, after 45 minutes at 4°C (to normalize protein adsorption), plates were transferred to 37°C for 30 minutes to facilitate protein internalization. After fixation with 4% PFA, samples were processed by IFA and confocal microscopy imaging. A colocalization plugin analysis in ImageJ software was used to define the colocalizing spots between NS1 and clathrin or caveolin in each experimental condition. The amount of spatial overlap between the two signals (NS1, red, and Clathrin, Caveolin, Rab5, green) was obtained using four different frames from the maximum projections of two RGB images based on the object-based approach (JACoP) and defined by the Manders’ Coefficient as previously described [<a href="#ppat.1007938.ref059">59</a>].</p> </div> <div class="antibodiesandinhibitorsforendocytosisexperiments" title="sec"> <div class="title" tagx="title" title="title">
Antibodies and inhibitors for endocytosis experiments</div> <p>Primary antibodies against clathrin heavy chain (Clathrin Heavy Chain [D3C6] XP Rabbit mAb [4796S], Cell Signaling), caveolin (Caveolin-1 (D46G3) XP Rabbit mAb #3267, Cell Signaling and Anti-Caveolin-2 antibody EPR5471 [ab133484], Abcam), Rab5 (Anti-Rab5 antibody—Early Endosome Marker [ab18211], Abcam), EEA1-early endosome marker (AB2900, Abcam), cathepsin L (BMS166, eBioscience), LAMP1 (ab25630, Abcam), mouse anti-6xHis (MA1-21315, Thermo Scientific), and rabbit anti-6xHis (ab232492, Abcam), together with the secondary antibodies goat anti-mouse IgG conjugated to Alexa 647 (ab150115, Abcam), goat anti-mouse IgG conjugated to Alexa 488 (ab150117, Abcam), donkey anti-rabbit IgG conjugated to Alexa 568 (ab175470, Abcam) were used in IFA experiments. An anti-6xHis-tag mAb (HIS.H8) conjugated to DyLight 647 (Thermo Fisher Scientific) was used for NS1 protein binding assays. Additionally, cells were infected with a baculovirus Rab5-GFP overexpression system (CellLight Early Endosomes-GFP, BacMam 2.0, C10586, Thermo Fisher Scientific), according to the manufacturer’s instructions, to visualize colocalization of NS1 with Rab5. Selective inhibitors of clathrin-mediated endocytosis (Pitstop 2, Abcam) and dynamin (Dynasore, Sigma) were used in IFA experiments and/or TEER assays at concentrations that do not affect cell viability. Cell viability was determined using the Promega CellTox Green Cytotoxicity Assay following the manufacturer’s instructions. The internalization of human transferrin, an iron-binding protein well-known to be internalized via clathrin-mediated endocytosis, was used as a control to examine the effectiveness of clathrin-mediated endocytosis inhibitors. Briefly, 30 minutes post-treatment with Pitstop 2 (25 μM) and Dynasore (50 μM) at 37°C, transferrin conjugated to Alexa 568 (20 μg/ml) (T23365, Thermo Fisher Scientific) was added to HPMEC monolayers cultured on cover slips and incubated for 10 minutes at 37°C. Then, plates were immediately transferred to an ice-bed; the transferrin inoculum was removed and cell monolayers were washed twice with cold PBS, and then washed twice with pre-chilled acid wash buffer (glycine 100 mM, and 150mM NaCl, pH 2.5) to remove cell-surface associated but non-internalized transferrin [<a href="#ppat.1007938.ref033">33</a>]. After this, coverslips were processed as described above for imaging after fixation with 4% PFA.</p> </div> <div class="sirnaknock-downanalysis" title="sec"> <div class="title" tagx="title" title="title">
siRNA knock-down analysis</div> <p>For transient knock-down analysis of clathrin heavy chain (HC), dynamin I/II, and caveolin-1, the siRNA-mediated knock-down system from Santa Cruz Biotechnology was used. In brief, Control siRNA-A (sc-37007), Clathrin HC siRNA (h) (sc-35067), Dynamin I/II (sc-43736), and caveolin-1 siRNA (h) (sc-29241) were resuspended in siRNA Dilution Buffer (sc-29527) according to the manufacturer’s instructions. HPMEC were then transfected with siRNAs using siRNA Transfection Reagent (sc-29528) in siRNA Transfection Medium (sc-36868) according to the manufacturer’s instructions. Seventy-two hours post-transfection, cells were assayed for knock-down efficiency by Western blot analysis and used for further experiments. Primary antibodies used for Western blot were as follows: clathrin HC (ab172958, Abcam), Dynamin I/II (2342S, Cell Signaling), caveolin-1 (D46G3) XP Rabbit mAb #3267 (Cell Signaling), and α-tubulin (ab4074, Abcam). Additionally, HPMEC monolayers transiently transfected with distinct siRNAs were analyzed for internalization of fluorescently labeled-transferrin-A568 (20 μg/ml) (T23365, Thermo Scientific) and albumin-FITC (100 μg/ml) (GTX28030, GeneTex) after incubation at 37°C for 10 minutes and 1.5 hours, respectively. The amount of internalized proteins was evaluated by IFA analyses using the same acid wash buffer method described above.</p> </div> <div class="localizedvascularleakmurinemodel" title="sec"> <div class="title" tagx="title" title="title">
Localized vascular leak murine model</div> <p><i>In vivo</i> NS1-induced endothelial hyperpermeability was measured using a rodent model of localized vascular leak, as previously described [<a href="#ppat.1007938.ref010">10</a>]. Briefly, the dorsal area of 6-week old female WT C57BL/6 mice (Jackson Labs) was depilated 3–4 days prior to each experiment. Hair was initially trimmed using hair clippers and then removed using Nair (Church &amp;amp; Dwight). Excess Nair was thoroughly wiped off using 70% ethanol and water. On the day of the experiment, DENV2 NS1 (15 μg) was incubated for 15 minutes at room temperature in the presence of 0.25 mg/ml Pitstop 2 (abcam) and 0.5 mg/ml Dynasore (abcam). NS1-inhibitor mixtures and controls were then were injected intradermally (ID) into distinct sites in the depilated dorsal skin of mice. Then, 150 μl of 10-kDa dextran conjugated with Alexa Fluor 680 (1 mg/ml; Sigma) was delivered by retro-orbital (RO) injection. Two hours after injection, mice were euthanized using isoflurane, and the dorsal skin was removed and placed in Petri dishes. Tissues were then visualized and scanned using a fluorescent detection system (LI-COR Odyssey CLx Imaging System) at a wavelength of 700 nm, and leakage surrounding the injection sites was quantified using Image Studio software (LI-COR Biosciences).</p> </div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p>Statistical analyses were performed using GraphPad Prism 6 software, and all graphs were generated using Prism 6. For quantification of MFI in in Figs <a href="#ppat.1007938.g002">2B–2E</a>, <a href="#ppat.1007938.g003">3B, 3D and 3F</a> and <a href="#ppat.1007938.s006">S5B and S5C Fig</a>, an ordinary one-way ANOVA using Dunnett’s multiple comparison test with multiple comparisons was used. For MFI quantification of IFA images in Figs <a href="#ppat.1007938.g004">4F</a>, <a href="#ppat.1007938.g005">5B</a>, <a href="#ppat.1007938.g006">6B</a> and <a href="#ppat.1007938.g008">8E</a> and <a href="#ppat.1007938.s007">S6B Fig</a>, unpaired parametric t-tests were used to determine the significance of treatment with inhibitors or siRNA knockdown of various proteins. For dermal leak experiments in <a href="#ppat.1007938.g007">Fig 7</a>, a nonparametric Mann-Whitney U test was used to determine significance of inhibitor treatment. For MFI quantification of IFA images in <a href="#ppat.1007938.s012">S11B and S11D Fig</a>, ordinary one-way ANOVA analyses using Dunnett’s multiple comparison test with multiple comparisons to either the DMSO group (<a href="#ppat.1007938.s012">S11B Fig</a>) or the untreated group (<a href="#ppat.1007938.s012">S11D Fig</a>) were used to determine the significance of inhibitor treatment or siRNA knockdown on internalization of transferrin or albumin by HPMEC.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Related to Figs <a href="#ppat.1007938.g001">1</a> and <a href="#ppat.1007938.g008">8</a>. Cloning and primer sequences for the DENV2, ZIKV, and WNV NS1 constructs.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="ppat.1007938.s001.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g001">Fig 1</a>. DENV WT NS1 and N-glycosylation site mutant N207Q are efficiently secreted, and the NS1-N207Q mutant lacks a high mannose N-glycan.</div> <p><b>(A)</b> Supernatants from 293F cells transfected with WT DENV NS1 or N-glycosylation mutant NS1-N207Q were concentrated using Amicon Ultra filters (100-kDa), and the concentrated top fraction (Top) and flow-through (FT) were both collected. NS1 monomers were detected via Western Blot of SDS-PAGE, using an anti-6xHis-tag antibody. (<b>B)</b> Western blot of SDS-PAGE showing the expression and secretion of WT DENV NS1, NS1 glycosylation mutant (N130Q), and double mutant (N130Q+N207Q). Anti-6xHis-tag antibody was used for detection of the NS1 monomer. S/N, supernatant; CL, cell lysate; NS1+, DENV2 NS1 (Native Antigen Company); Neg, gel loading buffer only. (<b>C</b>) Western blot of purified NS1 proteins treated with Endo H or PNGase F (NEB) for 1 hour at 37°C, using an anti-6xHis-tag antibody and demonstrating absence of the high mannose N-glycan at position 207 of the NS1-N207Q mutant. -, untreated; +E, Endo H-treated; +P, PNGase F-treated; arrows indicate which N-glycan species are present for each band.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g001">Fig 1</a>. Size-exclusion chromatography reveals a comparable size and elution profile between NS1-WT and NS1-N207Q.</div> <p><b>(A)</b> Size-exclusion chromatography of 0.25 milligrams of purified and dialyzed DENV NS1-WT (black) and DENV NS1-N207Q (gray). <b>(B)</b> Western blot analysis, under denaturing conditions, revealing NS1 monomers from the indicated fractions from Panel A with NS1-WT on the top and NS1-N207Q on the bottom. Proteins are detected using an NS1-specific monoclonal antibody (7E11).</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g001">Fig 1</a>. Purified NS1-WT and NS1-N207Q exist in a comparable conformation and are equally stable over time.</div> <p><b>(A)</b> NS1 direct ELISA comparing binding of three non-conformational mouse monoclonal antibodies (7E11, 2B7, and anti-6xHis) and one conformational mouse monoclonal antibody (9NS1) to NS1+, NS1-WT, or NS1-N207Q at a concentration of 200 ng/ml in native conditions (PBS) or denaturing conditions (PBS + 0.1% SDS with boiling for 5 minutes). <b>(B)</b> NS1 capture-ELISA comparing stability of 100 ng of NS1+, NS1-WT, or NS1-N207Q over time. One hundred ng of the indicated NS1 was diluted in EGM-2 tissue culture medium, mixed with 0.1% SDS or 200 ug/ml Proteinase K when indicated, and placed in a tissue culture incubator (37°C with 5% CO<sub>2</sub>) for the indicated times. The NS1-specific monoclonal antibody (7E11) was used to capture NS1 in the medium and another NS1-specific monoclonal antibody (2B7) was used to detect the captured NS1 proteins. <b>(C)</b> Western blot analysis of the indicated samples from panel B from an SDS-PAGE gel. NS1 was detected with a mouse anti-6xHis-tag monoclonal antibody. <b>(D)</b> Same experimental setup and Western blot analysis as Panel C but measuring the later time points indicated.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g001">Fig 1</a>. WT NS1 but not the NS1-N207Q mutant increase the permeability of HPMEC and HBMEC monolayers.</div> <p>Transendothelial electrical resistance (TEER) assays were used to determine the effect of the NS1-N207Q mutant on NS1-induced hyperpermeability. TEER data here are the non-normalized raw data from <a href="#ppat.1007938.g001">Fig 1</a> displayed in Ohms (Ω). <b>(A)</b> HPMEC values from <a href="#ppat.1007938.g001">Fig 1E</a> and <b>(B)</b> HBMEC values from <a href="#ppat.1007938.g001">Fig 1F</a>.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g002">Fig 2</a>. Mutation of the N-glycosylation site 207 prevents NS1-induced sialic acid degradation.</div> <p>(<b>A</b>) The binding of DENV NS1 (NS1+, Native Antigen Company), the in-house-produced DENV NS1-WT, and NS1-N207Q mutant (green) to HPMEC 1 hour post-treatment (hpt) was visualized via immunofluorescence assay (IFA). The integrity of the EGL component sialic acid (Sia) was assessed after 1 hpt at 37°C. Sia, stained with WGA-A647 (red); nuclei, stained with Hoechst (blue). Images (20X; scale bars, 50μm) are representative of two independent experiments run in duplicate. (<b>B</b>) Quantitation of A (top, NS1 binding). (<b>C</b>) Quantitation of A (bottom, sialic acid). The means ± standard error of the mean (SEM) of two individual experiments run in duplicate are shown. ns, not significant; *, p&amp;lt;0.05; **, p&amp;lt;0.01.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s006.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g003">Fig 3</a>. NS1-WT and NS1-N207Q both require heparan sulfate to bind to the surface of HPMEC.</div> <p>(<b>A</b>) The binding of in-house-produced NS1-WT and the NS1-N207Q mutant (10 μg/ml) (red) to HPMEC was visualized via IFA 24 hpt with 0.5 units of recombinant heparanase; untreated cells were used as a control. The nuclei of cells are stained with Hoechst (blue). Images (20X; scale bars, 50μm) are representative of three independent experiments. (<b>B</b>) Quantitation of cell binding in A. **, p&amp;lt;0.01. (<b>C</b>) Heparan sulfate surface expression (green) in HPMEC 24 hpt with 0.5 units of recombinant heparanase at 37°C, as visualized via IFA. Nuclei were stained with Hoechst (blue). Images (20X; scale bars, 50 μm) are representative of 3 independent experiments.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s007.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g003">Fig 3</a>. NS1-WT but not NS1-N207Q are internalized into HPMEC and localize to the early endosome.</div> <p><b>(A)</b> Western blot analysis of HPMEC from <a href="#ppat.1007938.g003">Fig 3C</a> treated with 10 μg/ml of the indicated NS1 and incubated at 37°C for 1.5 hours. Cells were not trypsinized to demonstrate the addition of equivalent NS1 levels to cells. NS1 was detected using an NS1-specific monoclonal antibody (7E11). <b>(B)</b> IFA analysis of HPMEC from <a href="#ppat.1007938.g003">Fig 3E</a> not treated with NS1. NS1 signal (red) is the first panel from the left and Rab5 signal (green) is the second panel from the left. Colocalization is shown in yellow in the merged image or white in the colocalization panel (ImageJ). <b>(C)</b> IFA analysis of HPMEC cells overexpressing Rab5 (CellLight-Rab5-GFP) (green) treated with NS1 (10 μg/ml) (red). Colocalization is shown in yellow in the merged image. <b>(D)</b> IFA analysis of HPMEC treated with NS1 (10 μg/ml) (red) and stained for the early-endosome marker EEA1 (green). Colocalization is shown in yellow in the merged image. For all images, nuclei are stained with Hoechst (blue). Images (40X; scale bars, 10 μm) are representative of 2 individual experiments run in duplicate. NS1+, NS1 from the Native Antigen Company.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s008.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g004">Fig 4</a>. WT DENV NS1 colocalizes with clathrin but not with caveolin on the surface of HBMEC.</div> <p>Colocalization of DENV NS1 proteins with either clathrin <b>(A)</b> or caveolin <b>(B)</b> in HBMEC was performed as described in the legend for <a href="#ppat.1007938.g004">Fig 4A</a>, except HBMEC were plated instead of HPMEC. Images are 40X; scale bars, 5 μm. Quantification of the amount of spatial overlap between the two signals, NS1 and clathrin <b>(C)</b>, or NS1 and caveolin <b>(D)</b> in S8A Fig and S8B Fig, respectively. The means ± SEM of two individual experiments run in duplicate are shown.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s009.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g004">Fig 4</a>. NS1 rapidly and transiently accumulates inside HPMEC, where it colocalizes with the lysosomal marker Lamp1 and cathepsin L.</div> <p>(<b>A</b>) Internalization of NS1 (10 μg/ml) into HPMEC was measured at the indicated time-points after allowing NS1 to adsorb to the cell surface at 4°C. After NS1 adsorption, cells were washed twice in cold PBS to remove unbound NS1 and were moved to 37°C for the indicated times. At the indicated time-points, cells were fixed, permeabilized, and visualized by IFA. Before fixation, cells were washed in a glycine-acid buffer to remove NS1 that is bound to the surface but not yet internalized. NS1 is visualized by an anti-6xHis-tag antibody (green), and nuclei of cells are stained with Hoechst (blue). Images (20X; scale bars, 50 μm) are representative of two independent experiments. (<b>B</b>) MFI quantification of internalized NS1 in A. (<b>C</b>) Same as A, except cells were collected in protein sample buffer for Western blot analysis. Before cell lysis, cells were washed in a glycine-acid buffer to remove NS1 that is bound to the surface but not yet internalized. <b>(D)</b> NS1 was allowed to internalize into HPMEC for 90 minutes (unbound NS1 was not washed away), cells were then fixed and permeabilized, and colocalization of NS1 and cellular markers was visualized by IFA. NS1 was detected with an anti-6xHis-tag antibody (green, the secondary antibody was conjugated to AF488). Lamp1 (red, the secondary antibody was conjugated to AF568); CTSL, cathepsin L (red, the secondary antibody was conjugated to AF647). Images (20X; scale bars, 25 μm) are representative of two independent experiments.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s010.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S10 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g005">Fig 5</a>. Clathrin-mediated endocytosis is required for NS1 to trigger endothelial hyperpermeability.</div> <p>Transendothelial electrical resistance (TEER) assays were used to determine the effect of inhibiting clathrin-mediated endocytosis on NS1-induced hyperpermeability. TEER data here are the non-normalized raw data from <a href="#ppat.1007938.g005">Fig 5C</a> displayed in Ohms (Ω).</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s011.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S11 Fig</span> <div class="title" tagx="title" title="title">
Related to Figs <a href="#ppat.1007938.g005">5</a>–<a href="#ppat.1007938.g007">7</a>. Chemical inhibition and siRNAs successfully antagonize clathrin- and caveolin-mediated endocytosis.</div> <p>Internalization of <b>(A-D)</b> transferrin-Alexa 568 and <b>(C, D)</b> albumin-FITC in HPMEC monolayers grown on coverslips and <b>(A, B)</b> treated with chemical inhibitors (30 minutes at 37°C; Pitstop 2 = 25 μM; Dynasore = 50 μM) or <b>(C, D)</b> transfected with distinct siRNAs. Internalization of <b>(A-D)</b> transferrin conjugated to Alexa-568 (20 μg/ml), to control for clathrin-mediated endocytosis, and <b>(C, D)</b> albumin-FITC (100 μg/ml), to control for caveolin-mediated endocytosis, was determined after 10 minutes and 1.5 hours at 37°C, respectively. After this time, cell monolayers were washed (4X) with an acid buffer to remove bound-non-internalized proteins. The amount of internalized transferrin and albumin was determined by confocal microscopy imaging (images: 20X; scale bars, 25 μm) and expressed as mean fluorescence intensity (MFI). The means ± SEM of two (B, D right) or three (D left) individual experiments run in duplicate are shown. ns, not significant; *, p&amp;lt;0.05; **, p&amp;lt;0.01.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s012.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S12 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g006">Fig 6</a>. Successful knockdown of siRNA targets.</div> <p>Western blot analysis of cell lysates from <a href="#ppat.1007938.g006">Fig 6A and 6B</a> and <a href="#ppat.1007938.s012">S11 Fig</a>., to test knock-down efficiency of clathrin heavy chain, dynamin I/II, and caveolin-1. Representative image shown from 3 independent Western blots. α-Tubulin is included as a loading control.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s013.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S13 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g008">Fig 8</a>. In-house-produced WNV and ZIKV NS1-WT and NS1-N207Q are equivalently secreted, exhibit high purity and stability, and exist primarily as putative hexamers.</div> <p><b>(A and D)</b> Western blot analysis, under denaturing conditions, of supernatants from 293F cells transfected with the indicated construct. <b>(B and E)</b> Silver staining analysis after SDS-PAGE of the indicated purified NS1 proteins. Gels depict NS1 monomers. <b>(C and F)</b> Western blot analysis, in native condition, of the indicated purified NS1 proteins or commercially purchased (NS1+). For all Western blots NS1 was detected using an anti-6xHis-tag antibody.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s014.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S14 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g008">Fig 8</a>. Flavivirus NS1-WT but not the NS1-N207Q mutant increase the permeability of HBMEC monolayers.</div> <p>Transendothelial electrical resistance (TEER) assays were used to determine the effect of the WNV and ZIKV NS1-N207Q mutant on NS1-induced hyperpermeability. TEER data here are the non-normalized raw data from <a href="#ppat.1007938.g008">Fig 8</a> displayed in Ohms (Ω). <b>(A)</b> Values from HBMEC treated with WNV NS1 from <a href="#ppat.1007938.g008">Fig 8B</a>. <b>(B)</b> Values from HBMEC treated with ZIKV NS1 from <a href="#ppat.1007938.g008">Fig 8C</a>.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s015.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S15 Fig</span> <div class="title" tagx="title" title="title">
Related to <a href="#ppat.1007938.g008">Fig 8</a>. Flavivirus NS1 residue N207 plays a conserved role in NS1 internalization.</div> <p>HPMEC (<b>A top, B</b>) and HBMEC (<b>A bottom, C</b>) monolayers grown on coverslips were treated with three different flavivirus NS1 proteins (DENV, ZIKV and WNV: 10 μg/ml) including NS1-WT and the NS1-N207Q mutant. NS1 binding and internalization were examined after 1.5 hours of incubation at 37°C using confocal microscopy to detect the co-staining of NS1 protein (red) and the early endosomal marker Rab5 (green). Nuclei were stained using Hoechst (blue). Images (40X; scale bars, 5 μm) are representative of 2 independent experiments. <b>(B, C)</b> The amount of total NS1 in each endothelial cell (red staining in <b>A</b>) was expressed as NS1<sup>+</sup>, total puncta per cell (light blue open circles), and NS1 puncta colocalized with Rab5 (yellow staining in <b>A</b>) was expressed as NS1<sup>+</sup> Rab5<sup>+</sup> puncta, indicating NS1 colocalization and therefore internalization (dark blue open squares). NS1<sup>+</sup> puncta were counted from a total of 200 cells collected from two independent experiments and analyzed using ImageJ. Dashed lines (light grey) and solid lines (black) represent the mean values for NS1<sup>+</sup> total puncta and NS1<sup>+</sup>Rab5<sup>+</sup> puncta, respectively.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007938.s016.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Movie</span> <div class="title" tagx="title" title="title">
Relating to <a href="#ppat.1007938.g003">Fig 3</a>: DENV NS1-WT but not the DENV NS1-N207Q mutant are internalized into endothelial cells where they colocalize with Rab5.</div> <p>An animated single 2D projection of NS1 internalization in HPMEC created using multiple confocal z-stack images. (<b>S1 Movie</b>) After binding to the cell surface, the WT NS1 protein (in <i>red</i>, indicated by white arrowheads) moves inside the cell, where it encounters Rab5 (in <i>green</i>), which results in a coalescence signal (in <i>yellow</i>), suggesting spatial colocalization. As the Z stack images move into a deeper Z axial space, this colocalizing signal disappears, and only the red signal for NS1 persists, suggesting its mobilization into a different intracellular compartment. (<a href="#ppat.1007938.s018">S2 Movie</a>) This process does not occur with the NS1-N207Q mutant protein, where most of the NS1 (in <i>red</i>) remain at a higher Z axial space, as little to no signal appears to colocalize with Rab5, suggesting this mutant NS1 protein is retained on the cell surface.</p> <p>(MP4)</p>  <div class="media" title="media"><a href="ppat.1007938.s017.mp4">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Movie</span> <div class="title" tagx="title" title="title">
Relating to <a href="#ppat.1007938.g003">Fig 3</a>: DENV NS1-WT but not the DENV NS1-N207Q mutant are internalized into endothelial cells where they colocalize with Rab5.</div> <p>An animated single 2D projection of NS1 internalization in HPMEC created using multiple confocal z-stack images. (<a href="#ppat.1007938.s017">S1 Movie</a>) After binding to the cell surface, the WT NS1 protein (in <i>red</i>, indicated by white arrowheads) moves inside the cell, where it encounters Rab5 (in <i>green</i>), which results in a coalescence signal (in <i>yellow</i>), suggesting spatial colocalization. As the Z stack images move into a deeper Z axial space, this colocalizing signal disappears, and only the red signal for NS1 persists, suggesting its mobilization into a different intracellular compartment. (<b>S2 Movie</b>) This process does not occur with the NS1-N207Q mutant protein, where most of the NS1 (in <i>red</i>) remain at a higher Z axial space, as little to no signal appears to colocalize with Rab5, suggesting this mutant NS1 protein is retained on the cell surface.</p> <p>(MP4)</p>  <div class="media" title="media"><a href="ppat.1007938.s018.mp4">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Dr. Allison Didychuk for technical assistance with size-exclusion chromatography and helpful discussion. We thank Dr. Michael Diamond (Washington University in St. Louis) for the pmab vector and for amplicons containing the WT DENV and WNV NS1 gene. We thank Kendall E. Lee for technical assistance and Dr. Robert Beatty for valuable discussions. Confocal imaging experiments were conducted on a Zeiss LSM 710 at the CRL Molecular Imaging Center, supported by the Gordon and Betty Moore Foundation. We would like to thank Holly Aaron and Feather Ives for their microscopy training and assistance.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1007938.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue and dengue hemorrhagic fever</span>. <span class="source" tagx="source" title="source">Clin Microbiol Rev</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">480</span>–<span class="lpage" tagx="lpage" title="lpage">96</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9665979">9665979</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marfin</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">West Nile encephalitis: an emerging disease in the United States</span>. <span class="source" tagx="source" title="source">Clinical infectious diseases: an official publication of the Infectious Diseases Society of America</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">1713</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/322700">10.1086/322700</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11595987">11595987</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao-Lormeau</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blake</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mons</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lastere</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roche</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanhomwegen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Guillain-Barre Syndrome outbreak associated with Zika virus infection in French Polynesia: a case-control study</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">387</span>(<span class="issue" tagx="issue" title="issue">10027</span>):<span class="fpage" tagx="fpage" title="fpage">1531</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(16)00562-6">10.1016/S0140-6736(16)00562-6</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26948433">26948433</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabata</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petitt</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michlmayr</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang-Hoover</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Targets Different Primary Human Placental Cells, Suggesting Two Routes for Vertical Transmission</span>. <span class="source" tagx="source" title="source">Cell Host Microbe</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">155</span>–<span class="lpage" tagx="lpage" title="lpage">66</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2016.07.002">10.1016/j.chom.2016.07.002</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27443522">27443522</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarno</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacramento</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khouri</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">do Rosario</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archanjo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Infection and Stillbirths: A Case of Hydrops Fetalis, Hydranencephaly and Fetal Demise</span>. <span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">e0004517</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004517">10.1371/journal.pntd.0004517</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26914330">26914330</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brasil</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreira</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro Nogueira</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Damasceno</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakimoto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Infection in Pregnant Women in Rio de Janeiro</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">375</span>(<span class="issue" tagx="issue" title="issue">24</span>):<span class="fpage" tagx="fpage" title="fpage">2321</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>. Epub 2016/03/04. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa1602412">10.1056/NEJMoa1602412</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26943629">26943629</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasmussen</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jamieson</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honein</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersen</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zika Virus and Birth Defects—Reviewing the Evidence for Causality</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">374</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">1981</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2016/04/13. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMsr1604338">10.1056/NEJMsr1604338</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27074377">27074377</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beatty</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Killingbeck</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopkins</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue virus NS1 triggers endothelial permeability and vascular leak that is prevented by NS1 vaccination</span>. <span class="source" tagx="source" title="source">Sci Transl Med</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">304</span>):<span class="fpage" tagx="fpage" title="fpage">304ra141</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/scitranslmed.aaa3787">10.1126/scitranslmed.aaa3787</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26355030">26355030</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue Virus NS1 Disrupts the Endothelial Glycocalyx, Leading to Hyperpermeability</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e1005738</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1005738">10.1371/journal.ppat.1005738</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27416066">27416066</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratnasiri</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinosa</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beatty</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue virus NS1 cytokine-independent vascular leak is dependent on endothelial glycocalyx components</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e1006673</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006673">10.1371/journal.ppat.1006673</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29121099">29121099</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modhiran</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watterson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panetta</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sester</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue virus NS1 protein activates cells via Toll-like receptor 4 and disrupts endothelial cell monolayer integrity</span>. <span class="source" tagx="source" title="source">Sci Transl Med</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">304</span>):<span class="fpage" tagx="fpage" title="fpage">304ra142</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/scitranslmed.aaa3863">10.1126/scitranslmed.aaa3863</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26355031">26355031</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>. <span class="mixed-article-title" title="mixed-article-title">The flavivirus NS1 protein: molecular and structural biology, immunology, role in pathogenesis and application as a diagnostic biomarker</span>. <span class="source" tagx="source" title="source">Antiviral Res</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">98</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">192</span>–<span class="lpage" tagx="lpage" title="lpage">208</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2013.03.008">10.1016/j.antiviral.2013.03.008</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23523765">23523765</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edeling</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fremont</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Structural basis of Flavivirus NS1 assembly and antibody recognition</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">111</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">4285</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1322036111">10.1073/pnas.1322036111</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24594604">24594604</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akey</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Konwerski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jose</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jurkiw</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Flavivirus NS1 structures reveal surfaces for associations with membranes and the immune system</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">343</span>(<span class="issue" tagx="issue" title="issue">6173</span>):<span class="fpage" tagx="fpage" title="fpage">881</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1247749">10.1126/science.1247749</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24505133">24505133</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutsche</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coulibaly</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voss</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salmon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">d’Alayer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ermonval</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Secreted dengue virus nonstructural protein NS1 is an atypical barrel-shaped high-density lipoprotein</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">108</span>(<span class="issue" tagx="issue" title="issue">19</span>):<span class="fpage" tagx="fpage" title="fpage">8003</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1017338108">10.1073/pnas.1017338108</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21518917">21518917</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiemmeca</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamdet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Punyadee</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prommool</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Songjaeng</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noisakran</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Secreted NS1 Protects Dengue Virus from Mannose-Binding Lectin-Mediated Neutralization</span>. <span class="source" tagx="source" title="source">J Immunol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">197</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">4053</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.1600323">10.4049/jimmunol.1600323</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27798151">27798151</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crabtree</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kinney</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Deglycosylation of the NS1 protein of dengue 2 virus, strain 16681: construction and characterization of mutant viruses</span>. <span class="source" tagx="source" title="source">Arch Virol</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">150</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">771</span>–<span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-004-0430-8">10.1007/s00705-004-0430-8</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15592895">15592895</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pryor</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Glycosylation mutants of dengue virus NS1 protein</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">1994</span>;<span class="volume" tagx="volume" title="volume">75</span>(<span class="issue" tagx="issue" title="issue">Pt 5</span>):<span class="fpage" tagx="fpage" title="fpage">1183</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-75-5-1183">10.1099/0022-1317-75-5-1183</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8176380">8176380</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somnuke</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauhart</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avirutnan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">N-linked glycosylation of dengue virus NS1 protein modulates secretion, cell-surface expression, hexamer stability, and interactions with human complement</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">413</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">253</span>–<span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2011.02.022">10.1016/j.virol.2011.02.022</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21429549">21429549</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nybakken</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engle</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marri</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fremont</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Antibodies against West Nile Virus nonstructural protein NS1 prevent lethal infection through Fc gamma receptor-dependent and -independent mechanisms</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">80</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">1340</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.80.3.1340-1351.2006">10.1128/JVI.80.3.1340-1351.2006</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16415011">16415011</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">DHL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadarangani</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>. <span class="mixed-article-title" title="mixed-article-title">"Locked in a cage"-A case of dengue virus 4 encephalitis</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">e0005369</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0005369">10.1371/journal.pntd.0005369</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28472166">28472166</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balsitis</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coloma</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alava</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flores</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKerrow</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Tropism of dengue virus in mice and humans defined by viral nonstructural protein 3-specific immunostaining</span>. <span class="source" tagx="source" title="source">Am J Trop Med Hyg</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">80</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">416</span>–<span class="lpage" tagx="lpage" title="lpage">24</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19270292">19270292</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harding</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heuser</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stahl</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Receptor-mediated endocytosis of transferrin and recycling of the transferrin receptor in rat reticulocytes</span>. <span class="source" tagx="source" title="source">The Journal of cell biology</span>. <span class="year" tagx="year" title="year">1983</span>;<span class="volume" tagx="volume" title="volume">97</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">329</span>–<span class="lpage" tagx="lpage" title="lpage">39</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.97.2.329">10.1083/jcb.97.2.329</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6309857">6309857</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dombu</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroubi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zibouche</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matran</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betbeder</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of endocytosis and exocytosis of cationic nanoparticles in airway epithelium cells</span>. <span class="source" tagx="source" title="source">Nanotechnology</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">35</span>):<span class="fpage" tagx="fpage" title="fpage">355102</span><span class="pub-id"><a href="https://dx.doi.org/10.1088/0957-4484/21/35/355102">10.1088/0957-4484/21/35/355102</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20689164">20689164</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avirutnan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Punyadee</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manuyakorn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puttikhunt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasinrerk</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Secreted NS1 of dengue virus attaches to the surface of cells via interactions with heparan sulfate and chondroitin sulfate E</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e183</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030183">10.1371/journal.ppat.0030183</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18052531">18052531</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcon-LePoder</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drouet</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frenkiel</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arborio</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durand-Schneider</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The secreted form of dengue virus nonstructural protein NS1 is endocytosed by hepatocytes and accumulates in late endosomes: implications for viral infectivity</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">79</span>(<span class="issue" tagx="issue" title="issue">17</span>):<span class="fpage" tagx="fpage" title="fpage">11403</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.79.17.11403-11411.2005">10.1128/JVI.79.17.11403-11411.2005</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16103191">16103191</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jovic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahajeng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caplan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">The early endosome: a busy sorting station for proteins at the crossroads</span>. <span class="source" tagx="source" title="source">Histology and histopathology</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">99</span>–<span class="lpage" tagx="lpage" title="lpage">112</span>. <span class="pub-id"><a href="https://dx.doi.org/10.14670/HH-25.99">10.14670/HH-25.99</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19924646">19924646</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donaldson</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pathways and mechanisms of endocytic recycling</span>. <span class="source" tagx="source" title="source">Nature reviews Molecular cell biology</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">597</span>–<span class="lpage" tagx="lpage" title="lpage">608</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2755">10.1038/nrm2755</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19696797">19696797</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santini</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marks</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keen</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Endocytic clathrin-coated pit formation is independent of receptor internalization signal levels</span>. <span class="source" tagx="source" title="source">Molecular biology of the cell</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1177</span>–<span class="lpage" tagx="lpage" title="lpage">94</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.9.5.1177">10.1091/mbc.9.5.1177</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9571248">9571248</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez-Busnadiego</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czapla</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrasman</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epsin deficiency impairs endocytosis by stalling the actin-dependent invagination of endocytic clathrin-coated pits</span>. <span class="source" tagx="source" title="source">eLife</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">e03311</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.03311">10.7554/eLife.03311</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25122462">25122462</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Kleist</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stahlschmidt</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulut</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromova</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puchkov</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Role of the clathrin terminal domain in regulating coated pit dynamics revealed by small molecule inhibition</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">146</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">471</span>–<span class="lpage" tagx="lpage" title="lpage">84</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2011.06.025">10.1016/j.cell.2011.06.025</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816279">21816279</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mettlen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pucadyil</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramachandran</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmid</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dissecting dynamin’s role in clathrin-mediated endocytosis</span>. <span class="source" tagx="source" title="source">Biochem Soc Trans</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">Pt 5</span>):<span class="fpage" tagx="fpage" title="fpage">1022</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1042/BST0371022">10.1042/BST0371022</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19754444">19754444</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirchhausen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macia</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelish</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Use of dynasore, the small molecule inhibitor of dynamin, in the regulation of endocytosis</span>. <span class="source" tagx="source" title="source">Method Enzymol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">438</span>:<span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0076-6879(07)38006-3">10.1016/S0076-6879(07)38006-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18413242">18413242</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayle</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamei</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span>. <span class="mixed-article-title" title="mixed-article-title">The intracellular trafficking pathway of transferrin</span>. <span class="source" tagx="source" title="source">Biochimica et biophysica acta</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">1820</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">264</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbagen.2011.09.009">10.1016/j.bbagen.2011.09.009</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21968002">21968002</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minshall</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiruppathi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niles</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilchrist</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamm</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Endothelial cell-surface gp60 activates vesicle formation and trafficking via G(i)-coupled Src kinase signaling pathway</span>. <span class="source" tagx="source" title="source">The Journal of cell biology</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">150</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1057</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.150.5.1057">10.1083/jcb.150.5.1057</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10973995">10973995</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beatty</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Good, the Bad, and the Shocking: The Multiple Roles of Dengue Virus Nonstructural Protein 1 in Protection and Pathogenesis</span>. <span class="source" tagx="source" title="source">Annu Rev Virol</span>. <span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-101416-041848">10.1146/annurev-virology-101416-041848</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30044715">30044715</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puerta-Guardo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glasner</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinosa</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biering</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patana</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratnasiri</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Flavivirus NS1 Triggers Tissue-Specific Vascular Endothelial Dysfunction Reflecting Disease Tropism</span>. <span class="source" tagx="source" title="source">Cell Rep</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1598</span>–<span class="lpage" tagx="lpage" title="lpage">613.e8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2019.01.036">10.1016/j.celrep.2019.01.036</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30726741">30726741</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torreno-Pina</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manzo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buschow</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cambi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Parajo</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enhanced receptor-clathrin interactions induced by N-glycan-mediated membrane micropatterning</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">111</span>(<span class="issue" tagx="issue" title="issue">30</span>):<span class="fpage" tagx="fpage" title="fpage">11037</span>–<span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1402041111">10.1073/pnas.1402041111</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25030450">25030450</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blouin</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamon</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonnord</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boularan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kagan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viaris de Lesegno</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Glycosylation-Dependent IFN-gammaR Partitioning in Lipid and Actin Nanodomains Is Critical for JAK Activation</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">166</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">920</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.07.003">10.1016/j.cell.2016.07.003</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27499022">27499022</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flamand</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megret</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathieu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepault</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rey</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deubel</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue virus type 1 nonstructural glycoprotein NS1 is secreted from mammalian cells as a soluble hexamer in a glycosylation-dependent fashion</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">73</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">6104</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10364366">10364366</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landsberg</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bletchly</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothnagel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddington</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hankamer</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Structure of the dengue virus glycoprotein non-structural protein 1 by electron microscopy and single-particle analysis</span>. <span class="source" tagx="source" title="source">Journal of General Virology</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">93</span>:<span class="fpage" tagx="fpage" title="fpage">771</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.039321-0">10.1099/vir.0.039321-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22238236">22238236</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Igl</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polasek</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gornik</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knezevic</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pucic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novokmet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Glycomics meets lipidomics—associations of N-glycans with classical lipids, glycerophospholipids, and sphingolipids in three European populations</span>. <span class="source" tagx="source" title="source">Molecular bioSystems</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1852</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c0mb00095g">10.1039/c0mb00095g</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21445428">21445428</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henry</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hislop</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grove</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorn</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Zastrow</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Regulation of endocytic clathrin dynamics by cargo ubiquitination</span>. <span class="source" tagx="source" title="source">Developmental cell</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">519</span>–<span class="lpage" tagx="lpage" title="lpage">32</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2012.08.003">10.1016/j.devcel.2012.08.003</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22940114">22940114</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruz-Oliveira</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freire</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conceicao</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higa</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castanho</span><span class="given-names" tagx="given-names" title="given-names">MARB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da Poian</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Receptors and routes of dengue virus entry into the host cells</span>. <span class="source" tagx="source" title="source">Fems Microbiol Rev</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">39</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">155</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/femsre/fuu004">10.1093/femsre/fuu004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25725010">25725010</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jhan</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Blockade of dengue virus infection and viral cytotoxicity in neuronal cells in vitro and in vivo by targeting endocytic pathways</span>. <span class="source" tagx="source" title="source">Scientific reports</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">7</span> Artn 6910 <span class="pub-id"><a href="https://dx.doi.org/10.1038/S41598-017-07023-Z">10.1038/S41598-017-07023-Z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28761128">28761128</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scaturro</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortese</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatel-Chaix</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischl</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartenschlager</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue Virus Non-structural Protein 1 Modulates Infectious Particle Production via Interaction with the Structural Proteins</span>. <span class="source" tagx="source" title="source">PLoS pathogens</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">11</span>). ARTN e1005277 <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1005277">10.1371/journal.ppat.1005277</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26562291">26562291</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothman</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Immunity to dengue virus: a tale of original antigenic sin and tropical cytokine storms</span>. <span class="source" tagx="source" title="source">Nat Rev Immunol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">532</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nri3014">10.1038/nri3014</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21760609">21760609</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chao</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perng</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue virus nonstructural protein 1 activates platelets via Toll-like receptor 4, leading to thrombocytopenia and hemorrhage</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e1007625</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007625">10.1371/journal.ppat.1007625</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31009511">31009511</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avirutnan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malasit</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seliger</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhakdi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Husmann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue virus infection of human endothelial cells leads to chemokine production, complement activation, and apoptosis</span>. <span class="source" tagx="source" title="source">J Immunol</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">161</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">6338</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9834124">9834124</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anfasa</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goeijenbier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widagdo</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siegers</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumtaz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okba</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Infection Induces Elevation of Tissue Factor Production and Apoptosis on Human Umbilical Vein Endothelial Cells</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">817</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.00817">10.3389/fmicb.2019.00817</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31068911">31068911</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jessie</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Localization of dengue virus in naturally infected human tissues, by immunohistochemistry and in situ hybridization</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">189</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">1411</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. Epub 2004/04/05. <span class="pub-id"><a href="https://dx.doi.org/10.1086/383043">10.1086/383043</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15073678">15073678</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aye</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charngkaew</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Win</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wai</span><span class="given-names" tagx="given-names" title="given-names">KZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Punyadee</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Pathologic highlights of dengue hemorrhagic fever in 13 autopsy cases from Myanmar</span>. <span class="source" tagx="source" title="source">Hum Pathol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1221</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.humpath.2014.01.022">10.1016/j.humpath.2014.01.022</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24767772">24767772</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miner</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Govero</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabrera</span><span class="given-names" tagx="given-names" title="given-names">OH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Infection during Pregnancy in Mice Causes Placental Damage and Fetal Demise</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">165</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1081</span>–<span class="lpage" tagx="lpage" title="lpage">91</span>. Epub 2016/05/11. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.05.008">10.1016/j.cell.2016.05.008</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27180225">27180225</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miner</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zika Virus Pathogenesis and Tissue Tropism</span>. <span class="source" tagx="source" title="source">Cell Host Microbe</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">134</span>–<span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2017.01.004">10.1016/j.chom.2017.01.004</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28182948">28182948</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mladinich</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwedes</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackow</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zika Virus Persistently Infects and Is Basolaterally Released from Primary Human Brain Microvascular Endothelial Cells</span>. <span class="source" tagx="source" title="source">MBio</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">4</span>). Epub 2017/07/11. <span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00952-17">10.1128/mBio.00952-17</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28698279">28698279</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustafá</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meuren</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coelho</span><span class="given-names" tagx="given-names" title="given-names">SVA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Arruda</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pathways Exploited by Flaviviruses to Counteract the Blood-Brain Barrier and Invade the Central Nervous System</span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">525</span> Epub 2019/03/28. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.00525">10.3389/fmicb.2019.00525</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30984122">30984122</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishidoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kominami</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Procathepsin L degrades extracellular matrix proteins in the presence of glycosaminoglycans in vitro</span>. <span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">217</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">624</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/bbrc.1995.2820">10.1006/bbrc.1995.2820</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7503744">7503744</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falconar</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Production of dimer-specific and dengue virus group cross-reactive mouse monoclonal antibodies to the dengue 2 virus non-structural glycoprotein NS1</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">1991</span>;<span class="volume" tagx="volume" title="volume">72</span> (<span class="issue" tagx="issue" title="issue">Pt 4</span>):<span class="fpage" tagx="fpage" title="fpage">961</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-72-4-961">10.1099/0022-1317-72-4-961</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1849979">1849979</a></span></span></li> <li tag="ref"><a name="ppat.1007938.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolte</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cordelieres</span><span class="given-names" tagx="given-names" title="given-names">FP</span></span>. <span class="mixed-article-title" title="mixed-article-title">A guided tour into subcellular colocalization analysis in light microscopy</span>. <span class="source" tagx="source" title="source">J Microsc</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">224</span>(<span class="issue" tagx="issue" title="issue">Pt 3</span>):<span class="fpage" tagx="fpage" title="fpage">213</span>–<span class="lpage" tagx="lpage" title="lpage">32</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2818.2006.01706.x">10.1111/j.1365-2818.2006.01706.x</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17210054">17210054</a></span></span></li> </ul> </div> </div>  </body></html>