<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Emerg Microbes Infect</span><span class="iso-abbrev" title="iso-abbrev">Emerg Microbes Infect</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Emerging Microbes &amp;amp; Infections</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2222-1751</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6143596">6143596</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30228270">30228270</a></span><span class="publisher-id" title="publisher-id">157</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41426-018-0157-3">10.1038/s41426-018-0157-3</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
miR-1303 regulates BBB permeability and promotes CNS lesions following CA16 infections by directly targeting MMP9</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Yajie</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Hongzhe</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Xing</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Huiwen</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Yunguang</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jingjing</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Xi</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jiaqi</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Zening</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Haitao</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Lei</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Haijing</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Zhanlong</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Fengmei</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Xi</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">Shaozhong</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">dsz@imbcams.com.cn</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Qihan</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">liqihan@imbcams.com.cn</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2623-5847</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Longding</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">longdingl@gmail.com</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0662 3178</span><span class="institution-id" title="institution-id">grid.12527.33</span><span class="institution" title="institution">Key Laboratory of Systemic Innovative Research on Virus Vaccine, </span><span class="institution" title="institution">Chinese Academy of Medical Sciences, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">155</span><span class="history" title="history"><span class="received" title="received">received: 2018-5-2</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-8-9</span><span class="accepted" title="accepted">accepted: 2018-8-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Coxsackievirus A16 (CA16) is a member of the Picornaviridae family and causes mild and self-limiting hand, foot, and mouth disease (HFMD) in infants and young children. CA16 infection can also progress to central nervous system (CNS) complications; however, the underlying mechanism by which CA16 penetrates the blood-brain barrier (BBB) and then causes CNS damage remains unclear. This study aimed to explore the mechanism of CA16 neurotropic tropism by establishing an in vitro BBB model with CA16 infection and an in vivo CA16 rhesus monkey infant infection model. The results showed that CA16 infection induced increased permeability of the BBB accompanied by upregulation of matrix metalloproteinase 9 (MMP9) expression. Subsequently, high-throughput miRNA sequencing technology and bioinformatics analysis revealed that miR-1303 may regulate BBB permeability by targeting MMP9. Next, we used dual-luciferase, qRT-PCR, and western blot assays to provide evidence of MMP9 targeting by miR-1303. Further experiments revealed that CA16 infection promoted the degradation of junctional complexes (Claudin4, Claudin5, VE-Cadherin, and ZO-1), likely by downregulating miR-1303 and upregulating MMP9. Finally, EGFP-CA16 infection could enter the CNS by facilitating the degradation of junctional complexes, eventually causing neuroinflammation and injury to the CNS, which was confirmed using the in vivo rhesus monkey model. Our results indicate that CA16 might penetrate the BBB and then enter the CNS by downregulating miR-1303, which disrupts junctional complexes by directly regulating MMP9 and ultimately causing pathological CNS changes. These results provide new therapeutic targets in HFMD patients following CA16 infection.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100005150</span><span class="institution" title="institution">Chinese Academy of Medical Sciences (CAMS)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2016-I2M-1-014</span><span class="award-id" tagx="award-id" title="award-id">2017-I2M-2-006</span><span class="award-id" tagx="award-id" title="award-id">2017310039</span><span class="award-id" tagx="award-id" title="award-id">2016-I2M-1-019</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Hongzhe</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100007471</span><span class="institution" title="institution">Applied Basic Research Foundation of Yunnan Province (Yunnan Province Applied and Basic Research Foundation)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2016HC009)</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Hongzhe</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China (National Science Foundation of China)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">31700153</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100004572</span><span class="institution" title="institution">National Health and Family Planning Commission of the People's Republic of China (Ministry of Health of China)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2017 Yunnan Medical Reserve Talents (H-2017034)</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Coxsackievirus A16 (CA16), which belongs to the human enterovirus A species of the Picornaviridae family, is one of the major causative agents of hand, foot, and mouth disease (HFMD), which is a common infectious disease mostly seen in children under 5 years old<sup><a href="#CR1">1</a></sup>. In most instances, HFMD is a benign, self-limiting disease that is often characterized by fever, pharyngitis, and vesicular eruptions on the hands and feet, as well as in the mouth<sup><a href="#CR2">2</a></sup>. However, in some epidemic outbreaks, severe progressive HFMD with fatalities has been observed<sup><a href="#CR3">3</a></sup>. The occurrence of high mortality is usually concomitant with central nervous system (CNS) complications, such as acute flaccid paralysis, aseptic meningitis, brainstem encephalitis, and pulmonary edema<sup><a href="#CR4">4</a></sup>. Currently, three companies in mainland China, namely, Beijing Vigoo Biological Co., Ltd. (Vigoo), Sinovac Biotech Co., Ltd. (Sinovac), and the Chinese Academy of Medical Sciences (CAMS) have successfully developed inactivated EV71 vaccines. All three vaccines have been reported to have high safety and efficacy for preventing EV71 infections, but they have no cross-protective effects on other enterovirus infections, including CA16<sup><a href="#CR5">5</a></sup>. Previous studies have indicated that most CA16 infections present only mild symptoms. However, accumulating evidence in recent years has demonstrated that CA16 might actually be more virulent in children and has induced severe cases of CNS complications and fatal outcomes, which were similar to those of EV71 infections<sup><a href="#CR6">6</a></sup>. Furthermore, some severe and fatal HFMD cases with CA6 and CA10 infections have shown a clear increasing trend in recent years<sup><a href="#CR7">7</a></sup>. Taken together, these findings demonstrate that it is important to explore the underlying molecular basis of HFMD pathogenesis and progression induced by CA16 infections, which might provide novel approaches for the development of vaccines that cover multiple species of enterovirus.</p> <p id="Par3">Similar to poliovirus, CA16 is a highly neurotropic virus<sup><a href="#CR8">8</a></sup>. Two likely routes by which CA16 enter the CNS have been considered: the virus either is transmitted to the CNS from the blood across the blood–brain barrier (BBB) or enters the CNS through peripheral nerves via retrograde axonal transport<sup><a href="#CR9">9</a>,<a href="#CR10">10</a></sup>. Several studies have revealed that the BBB, which is primarily composed of brain endothelial cells, astrocyte end-feet, pericytes, perivascular macrophages, and a basal membrane, acts in concert with the immune system to restrict the entry of pathogens into the CNS. Meanwhile, many pathogens manage to cross the BBB via modifying a host of BBB functions, altering BBB integrity, and disrupting BBB permeability, which ultimately allows the establishment of infection within the CNS<sup><a href="#CR11">11</a></sup>. For example, West Nile virus enters the brain by enhancing the BBB permeability with matrix metalloproteinase 9 (MMP9) release. MMP9 is a member of the zinc-dependent and calcium-dependent endopeptidase family that is capable of degrading most extracellular matrix components<sup><a href="#CR12">12</a></sup>. Moreover, in the CNS, MMP9 can degrade components of the basal lamina and junctional complexes, leading to the disruption of BBB integrity and induction of the neuroinflammatory response in many neurological diseases<sup><a href="#CR13">13</a></sup>. For instance, alterations in BBB permeability were consistent with the increased expression of MMP9 in stroke and traumatic brain injury; further, it was found that disruption of BBB integrity in the setting of elevated MMP9 expression was closely associated with changes in junction assembly and function<sup><a href="#CR14">14</a></sup>. Junctional complexes in the BBB primarily include tight junctions (TJs), adhesion junctions (AJs), and gap junctions, which contribute differently to BBB dysfunction by directly regulating BBB permeability, altering transcellular exchange, affecting the expression pattern of certain transporters, and modifying endothelial metabolic processes<sup><a href="#CR15">15</a></sup>. Hence, destruction of these junctional complexes indirectly mirrors the breakdown of BBB integrity.</p> <p id="Par4">Our previous study analyzed transcriptome changes in a CA16-infected rhesus monkey model and found that the expression of MMP9 was significantly increased<sup><a href="#CR16">16</a></sup>. Thus, we speculated that the CNS injuries induced by CA16 infection are derived from direct penetration of the BBB by CA16 and that the molecular mechanism of BBB penetration by CA16 is related to the deregulation of junctional complexes via MMP9. Furthermore, we aimed to explore the upstream regulatory factors of MMP9 using miRNA high-throughput sequencing technology.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="proliferationkineticsofca16inhuvecs" title="sec"> <div class="title" tagx="title" title="title">
Proliferation kinetics of CA16 in HUVECs</div> <p id="Par5">To assess whether CA16 can infect and proliferate in HUVECs, the cell line was inoculated with each virus and analyzed for proliferation kinetics; EV71 experiments were performed in the same manner as the control. The level of EV71 and CA16 viral load showed a constant rise over time (Fig S<a href="#MOESM2">1A</a>). Moreover, the production of infectious virus particles gradually increased over time with EV71 and CA16 infections (Fig S1B), suggesting that HUVECs are highly susceptible to EV71 and CA16.</p> </div> <div class="ca16infectionspromotehuvecpermeability" title="sec"> <div class="title" tagx="title" title="title">
CA16 infections promote HUVEC permeability</div> <p id="Par6">To determine the effects of CA16 infection on the integrity of HUVECs, we performed a HUVEC endocytosis assay using FITC-dextran. The results revealed that FITC-dextran leakage was significantly upregulated in CA16-treated HUVECs (Fig. <a href="#Fig1">1a</a>), indicating that CA16 infections might enhance cell permeability. Coincidently, CA16 infection induced increased permeability of the BBB that was accompanied by upregulated MMP9 expression (Fig. <a href="#Fig1">1b</a>). Subsequently, an apoptosis assay was used to evaluate whether the alterations in HUVEC permeability were attributed to apoptosis (Fig. <a href="#Fig1">1c</a>). There were no significant differences in HUVEC apoptosis induced by CA16 infections, implying that apoptosis might not be responsible for HUVEC permeability during CA16 infection. Similar results were observed in EV71-infected HUVECs. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
CA16 infections increased permeability of monolayer HUVECs.</div> <p><b>a</b> Permeability of monolayer HUVECs was detected by a paracellular FITC-dextran flux assay. Significant differences among these groups are indicated by *<i>P</i> &amp;lt; 0.05. <b>b</b> Detection of the expression of MMP9 induced by CA16 infection using ELISA and WB. <b>c</b> CA16 infections does not induced HUVECs apoptosis, including early apoptotic and late apoptotic</p>   </div> <p /> </div> <div class="globalanalysisofmirnasequencingdatainhuvecsfollowingca16andev71infections" title="sec"> <div class="title" tagx="title" title="title">
Global analysis of miRNA sequencing data in HUVECs following CA16 and EV71 infections</div> <p id="Par7">Because CA16 and EV71 both caused remarkable changes in HUVEC permeability, we hypothesized that CA16 and EV71 infections might induce similar alterations. Thus, in this study, we screened the differentially expressed miRNAs, which showed the same expression pattern in the trend analysis (Fig S<a href="#MOESM3">2</a>). Subsequently, the 34 miRNAs expressed at the same trend were used to predict potential targets via TargetScan and the miRDB algorithm. As shown in Fig S<a href="#MOESM4">3</a>, the 658 overlapping targets identified by both programs were considered key targets of these miRNAs for the following GO and pathway analysis. Moreover, the GO results revealed that these target genes were categorized as follows: twelve as a biological process (Fig S<a href="#MOESM5">4A</a>), eight as a molecular function (Fig S<a href="#MOESM5">4B</a>), and eight as a cellular component (Fig S<a href="#MOESM5">4C</a>). Then, the pathway results revealed 79 pathways (Fig S<a href="#MOESM6">5</a>). To further narrow down the key targets, we identified intersections of related genes in the GO and pathway analyses, and 71 targets were identified, as shown in Fig S<a href="#MOESM7">6</a>. Consequently, a miRNA–gene network (Fig. <a href="#Fig2">2a</a>) and miRNA–GO network (Fig. <a href="#Fig2">2b</a>) were built based on interactions between miRNAs and genes or miRNAs and GOs. Furthermore, these genes were used for a hierarchical GO category and coexpression network analysis. The results uncovered that signaling process, signal transmission, intracellular signal transduction, small GTPase-mediated signal transduction, regulation of exocyst localization, regulation of exocyst assembly, cell–cell adhesion, regulation of keratinocyte migration, and positive regulation of keratinocyte migration were significantly different (Fig. <a href="#Fig2">2c</a>). Moreover, coexpression results revealed that junctional complexes, such as PCDHA11, PCDH1, and PCDHA7, were coexpressed. Therefore, these findings suggested that the deregulated miRNA-associated target genes might be involved in cell junctions. In addition, junctional complexes of the coexpression network contained MMP9 molecules (Fig. <a href="#Fig2">2d</a>), which is an important factor that allows some neurotropic viruses to cross the BBB. However, our results also revealed that CA16 and EV71 infections might enhance BBB permeability. Therefore, we focused our attention on MMP9. To seek the major miRNA regulator of MMP9, we focused on the miRNA-target network and found that the only miRNA that interacted with MMP9 was miR-1303. Hence, we speculated that the abnormal expression of miR-1303 induced by CA16 and EV71 infections might enhance the increased BBB permeability via regulating MMP9 expression. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Predicted networks between the coincident trend expressed miRNAs and their putative target genes, their putative target gene-associated GOs, Hierarchical tree graphs of GO terms and co-expression network.</div> <p> All miRNAs are depicted as a blue colored node among the networks. The larger the area of the nodes, the bigger the number of connections between a miRNA and other nodes in the network.<b>a</b> Predicted network between the same expressed miRNAs and their putative target genes. Putative targets are presented wathet blue rounded rectangles. The width of the line represents the free energy between the miRNAs and their putative target genes. <b>b</b> Predicted network between the same expressed miRNAs and their putative target gene-associated GOs. Red rectangle nodes denote GOs. Edges show the inhibitory effects of miRNAs on GOs. <b>c</b> In total, 71 genes were used to generate the GO tree. Edges represent "parent–child�? relationships of GO terms. The sizes of the yellow rounded rectangles are proportional to the number of GO terms annotated to each node. <b>d</b> Co-expression network were constructed by the top ten significantly putative targets. Green nodes are target genes and pink nodes are co-expression genes. Genes with bigger size are more centralized in the network and have a stronger capacity of modulating adjacent genes. Different color lines mean the different interactions between these genes</p>   </div> <p /> </div> <div class="mir-1303directlytargetsmmp9anddegradesjunctionalproteins" title="sec"> <div class="title" tagx="title" title="title">
miR-1303 directly targets MMP9 and degrades junctional proteins</div> <p id="Par8">qRT-PCR was applied to detect miR-1303 and MMP9 expression levels in HUVECs following EV71 and CA16 infections. The level of miR-1303 was markedly decreased, which was consistent with our miRNA sequence results (Fig. <a href="#Fig3">3a</a>), whereas the level of MMP9 was remarkably increased (Fig. <a href="#Fig3">3b</a>). Subsequently, based on the inverse correlation between miR-1303 and MMP9 expression, we further explored the target interaction between miR-1303 and MMP9 by a dual-luciferase reporter assay. The results revealed that the luciferase activity was notably decreased in the WT-MMP9 vector transfected with miR-1303 mimics compared with that in the other four groups, while the luciferase activity was not markedly changed in the mutant-MMP9 vector transfected with the five plasmids (Fig. <a href="#Fig3">3c</a>). We further assessed the influence of miR-1303 on the expression of MMP9 by detecting the mRNA and protein levels after transfecting miR-1303 mimics and found that forced miR-1303 expression resulted in a sharp decline of MMP9 mRNA and protein levels in 293T cells (Fig. <a href="#Fig3">3d</a>). Hence, these data demonstrated that miR-1303 might directly target MMP9. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
MMP9 is a target gene of miR-1303.</div> <p><b>a</b> Verification of the expression of miR-1303 by qRT-PCR compared to miRNA sequencing. <b>b</b> Detection of the expression of MMP9 induced by CA16 infection using qRT-PCR. <b>c</b> Dual-luciferase reporter analysis verified the targetting relationship between miR-1303 and MMP9. <b>d</b> qRT-PCR and WB analysis of MMP9 expression in miR-1303 transfected cells. Error bars represent the mean ± SEM, and the data are averages from three biological replicates, *<i>P</i> &amp;lt; 0.05</p>   </div> <p /> <p id="Par9">Since junctional proteins play a major role in the regulation of BBB permeability, and their disruption is a hallmark of CNS abnormalities, we determined whether miR-1303 might regulate the expression of junctional proteins in HUVECs through MMP9. Based on the above results, which showed that MMP9 expression was upregulated in HUVECs following CA16 and EV71 infections, we prepared plasmids for MMP9 depletion (si-MMP9) and found that the protein knockdown efficiency of si-MMP9 was approximately 50% compared to the Mock and si-control in HUVECs (Fig S<a href="#MOESM8">7</a>). As illustrated in Fig. <a href="#Fig4">4a</a>, cells in the mock, si-control, and si-MMP9 groups showed the normal "chicken wire�? appearance of the Claudin4 structure. Compared with that in CA16-infected and EV71-infected cells, which displayed an obscure localization of Claudin4 at the cell–cell contact sites, Claudin4 staining was markedly enhanced in the si-MMP9-CA16 and si-MMP9-EV71 groups (Fig. <a href="#Fig4">4a</a>). Subsequently, IF staining was performed for other junctional proteins, such as Claudin5, VE-Cadherin, and ZO-1. The location of these proteins presented an alteration similar to that of Claudin4 in each group (Fig. <a href="#Fig4">4b–d</a>). Therefore, these observations indicated that CA16 and EV71 infections could interfere with junctional protein assembly in HUVECs, which might be intimately linked to breakdown of the BBB because of the upregulation of MMP9. However, knockdown of MMP9 could maintain the locations of junctional proteins in CA16-infected and EV71-infected HUVECs. Furthermore, western blot results also confirmed that CA16 and EV71 infections in HUVECs could upregulate MMP9 protein levels and downregulate the previously mentioned junctional protein levels, whereas these results were reversed when CA16-infected and EV71-infected HUVECs were transfected with a si-MMP9 plasmid (Fig. <a href="#Fig5">5</a>). Finally, according to the target relationship of miR-1303 and MMP9, we further detected the influences of miR-1303 at the locations of junctional proteins in CA16-infected and EV71-infected HUVECs. After transfection with a miR-1303 overexpression plasmid, Claudin4, Claudin5, VE-Cadherin, and ZO-1 proteins were not destroyed in EV71-infected and CA16-infected HUVECs (Fig. <a href="#Fig6">6</a>). Collectively, these results implied that miR-1303 might target MMP9 to inhibit junctional protein degradation, which might protect the BBB from damage. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Confocal imaging showed that the junctional protein including <b>a</b> Claudin4, <b>b</b> Claudin5, <b>c</b> VE-Cadherin, and <b>d</b> ZO-1 were degraded by CA16 infection and knockout MMP9 rescued these degradation. The si-control and si-MMP9 plasmids contain a GFP tag (green color). EV71/CA16-VP1, junctional proteins and the cell nucleus are indicated by red, purple and mazarine, respectively</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Junctional proteins expression including Claudin4, Claudin5, VE-Cadherin, and ZO-1 were degraded by CA16 infection and knockout MMP9 rescued this degradation.</div> <p><b>a</b> Protein expression of junctional proteins in HUVECs with different treatments normalized against GAPDH. <b>b</b> Densitometry of WBs of three independent experiments performed with lysates at 72 and 96 hpi. The signal intensity was normalized to GAPDH levels from the same blots. The results are expressed as the mean ± SEM for three subjects, *<i>P</i> <i>&amp;lt;</i> 0.05</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Overexpression of miR-1303 inhibites the degradation of junctional protein including <b>a</b> Claudin4, <b>b</b> Claudin5, <b>c</b> VE-Cadherin, and <b>d</b> ZO-1. EV71/CA16-VP1, junctional proteins and the cell nucleus are indicated by red, purple and mazarine, respectively</p>   </div> <p /> </div> <div class="mmp9promotestheentryofca16intothecnsandexacerbatescnslesionsbydegradingjunctionalproteinsinvivo" title="sec"> <div class="title" tagx="title" title="title">
MMP9 promotes the entry of CA16 into the CNS and exacerbates CNS lesions by degrading junctional proteins in vivo</div> <p id="Par10">In a CA16-infected rhesus monkey model, levels of junctional proteins, including Claudin5, VE-Cadherin, and ZO-1, were significantly reduced in EGFP-CA16-infected monkeys compared with control monkeys. Although Claudin4 did not present a notable reduction, it exhibited a decrease trend in EGFP-CA16-infected monkeys compared to control monkeys. In addition, these protein levels were further reduced in EGFP-CA16 + MMP9-infected monkeys compared with EGFP-CA16-infected monkeys. However, the expression of these proteins was notably compensated in EGFP-CA16 + TIMP-1-infected monkeys relative to EGFP-CA16 + MMP9-infected monkeys (Fig. <a href="#Fig7">7a, b</a>). Therefore, these data suggested that MMP9 plays a key role in degrading junctional proteins in HUVECs. The viral infection rates of brain tissues and viral genomic copies in EGFP-CA16 + MMP9 monkeys were the highest, followed by those in EGFP-CA16 monkeys, and they were the lowest in EGFP-CA16 + TIMP-1 monkeys (Fig. <a href="#Fig7">7c, d</a> and S8). This result indicated that MMP9 accelerates CA16 entry into the brain. Ultimately, the morphological examination showed no obvious pathological changes in the thalamus or lungs of control monkeys, whereas different pathological responses to EGFP-CA16 infection and EGFP-CA16 + MMP9 and EGFP-CA16 + TIMP-1 treatments were found in the thalamus and lungs. As presented in Fig. <a href="#Fig7">7e</a>, mild hemorrhagic injuries were observed in the thalamus tissues of the EGFP-CA16 monkeys; serious hemorrhagic injuries were discovered in the thalamus tissues of the EGFP-CA16 + MMP9 monkeys, but these morphological changes were drastically improved in the EGFP-CA16 + TIMP-1 monkeys. Additionally, the alveolar wall in the EGFP-CA16 monkeys and EGFP-CA16 + MMP9 monkeys was visibly thicker than that in the control monkeys (Fig. <a href="#Fig7">7e</a>). Nevertheless, the alveolar wall in the EGFP-CA16 + TIMP-1 monkeys was not apparently thickened (Fig. <a href="#Fig7">7e</a>). Therefore, these findings indicate that MMP9 may exacerbate CNS lesions and lung damage. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <div class="title" tagx="title" title="title">
MMP9 promotes the degradation of junctional protein and CA16 entry into the CNS, which leading to CNS lesion in CA16-infected rhesus monkeys.</div> <p><b>a</b> Junctional protein expression in thalamus of different treated rhesus monkeys. <b>b</b> Statistical analysis of the quantitative value of intensity about the blots. The ratios indicated the intensity of the bands compare to the GAPDH. <b>c</b> The efficacy of CA16 infections in thalamus of different treated rhesus monkeys. CA16-VP1 and the cell nucleus are indicated by green and mazarine, respectively. <b>d</b> Statistical analysis of viral infection rates. The ratios indicated the infected cells compared to the total cells from ten different visual fields. <b>e</b> Histopathological changes in the thalamus and lungs of different treated rhesus monkeys. The green arrow points to a place where pathological changes are obvious</p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par11">Epidemics of CA16 infections have caused many fatalities from serious CNS complications in the Asia-Pacific region since 1997 and have become a serious public health problem in these areas<sup><a href="#CR17">17</a></sup>. Nevertheless, the mechanism of CNS damage induced by CA16 remains obscure. MMPs are broad, nonspecific extracellular matrix-degrading enzymes that comprise collagenases (MMP1, 8, and 13), gelatinases (MMP2 and 9), stromelysins (MMP3, 10, and 11), and matrilysins (MMP7 and MMP26). Previous studies have shown that the increased MMP2 and MMP9 concentration in cerebrospinal fluid degrades junctional complex proteins between brain capillary endothelial cells, which further causes the disruption of BBB functions and ultimately promotes the formation of CNS-related diseases, such as ischemic stroke, multiple sclerosis, and glioblastoma<sup><a href="#CR18">18</a></sup>. The BBB is nonfenestrated to large molecules or xenobiotics, thus providing protection against invasion of the CNS by macromolecules and microorganisms<sup><a href="#CR19">19</a></sup>. The integrity of the BBB is a result of a tightly sealed monolayer of brain microvascular endothelial cells (BMECs), with TJs and AJs forming the seal between cells at junctional complexes<sup><a href="#CR20">20</a></sup>. Furthermore, many viruses (e.g., mouse adenovirus type 1) can trigger changes in BBB permeability by direct disruption of the TJ complex in endothelial cells<sup><a href="#CR21">21</a></sup>. Additionally, evidence is emerging that MMP9 plays an important role in the pathogenesis of BBB breakdown through regulating the complex intercellular junctional structure in BMECs<sup><a href="#CR22">22</a></sup>. For example, the HIV-1-specific protein Tat enhances the production of MMP9, which can decrease TJ protein expression and finally result in an increase of BBB permeability<sup><a href="#CR22">22</a></sup>. Yang et al.<sup><a href="#CR23">23</a></sup> recently speculated that miR-3473a is involved in destruction of the BBB by targeting MMP9 during EV71 infections based on miRNA expression profiles, but this speculation lacks further experimental support. In our study, it was first found that CA16 could increase the permeability of HUVECs in an in vitro BBB model. Previous research has shown that Japanese encephalitis virus is a neurotropic flavivirus that can compromise the structural and functional integrity of the BBB, likely via enhancing the viral load in BMECs, and has the potential to subvert host cell apoptosis to optimize the course of the viral infection through deactivation of pro-apoptotic proteins. This suggests that apoptosis induced by the virus and occurring in BMECs might promote breakdown of the BBB<sup><a href="#CR24">24</a></sup>. Nevertheless, the BBB opening in this study was not related to cell apoptosis induced by CA16 infections (Fig. <a href="#Fig1">1</a>). Furthermore, HUVECs are an important constituent of the BBB in an in vitro model<sup><a href="#CR25">25</a></sup>. Therefore, to further characterize the mechanism by which CA16 modulates the upregulation of BBB permeability, we performed a miRNA profile analysis with high-throughput sequencing technology in CA16-infected HUVECs.</p> <p id="Par12">miRNAs are known to play a role in the replication and propagation of viruses, including cellular antiviral responses and/or the promotion of viral infections through complex regulatory pathways<sup><a href="#CR26">26</a></sup>. In this study, we screened the differentially expressed miRNAs in HUVECs after CA16 infection, focusing our attention on the similarly expressed miRNAs (Fig S<a href="#MOESM3">2</a>). Meanwhile, the potential targets of these miRNAs were predicted by TargetScan and miRDB (Fig S<a href="#MOESM4">3</a>). Overall, 658 intersection targets were used to further analyze the GO and pathways. The results of this analysis revealed that there were many intercellular adhesion-related GOs and pathways, such as biological adhesion, cell junction, and cadherin signaling pathways (Fig S<a href="#MOESM5">4</a>). Subsequently, the overlaying targets involved in both GOs and pathways were applied to construct miRNA-regulatory networks and gene coexpression networks. The results showed that MMP9 was only targeted by miR-1303, which was also implicated in the regulation of biological adhesion (Fig. <a href="#Fig2">2a–c</a>). In addition, there was an interaction between MMP9 and PVLR1 in a coexpression network (Fig. <a href="#Fig2">2d</a>). PVRL1, also known as nectin1, was first identified as an afadin-binding protein and serves as a cell adhesion molecule at AJs<sup><a href="#CR27">27</a></sup>. However, the formation of TJs in BMECs generally requires the existence of AJs. Thus, according to the above clues, we hypothesized whether the increased BBB permeability during EV71 and CA16 infections was attributed to the downregulation of miR-1303, which might negatively regulate the secretion of MMP9 and eventually lead to alterations in the junctional complexes between HUVECs.</p> <p id="Par13">To address this hypothesis, it was necessary to confirm the expression levels of miR-1303 and MMP9 in an in vitro model of the BBB. miR-1303 and MMP9 expression was detected by qRT-PCR, ELISA, and western blot. It was observed that miR-1303 expression was reduced, whereas MMP9 expression was elevated in HUVECs following CA16 infections, which was consistent with the high-throughput sequencing data (Fig. <a href="#Fig3">3a, b</a>). Moreover, the dual-luciferase reporter assay directly indicated that MMP9 was the target gene of miR-1303 (Fig. <a href="#Fig3">3c</a>). Additionally, the downregulated expression of MMP9 at the mRNA and protein levels further demonstrated the target interaction between miR-1303 and MMP9 by overexpression of miR-1303 in 293T cells (Fig. <a href="#Fig3">3d</a>). To date, the investigations between miR-1303 and disease have mainly focused on cancers. For example, Zhang et al.<sup><a href="#CR28">28</a></sup> verified that miR-1303 facilitates the proliferation, migration, and invasion of gastric cancer through targeting claudin-18. However, the role of miR-1303 in CNS-related diseases has not been reported; therefore, based on the target interaction between miR-1303 and MMP9, we aimed to further investigate the effects of miR-1303 on changes in BBB permeability in HUVECs following CA16 infections.</p> <p id="Par14">Junctional complexes between BMECs, such as TJs and AJs, play a crucial role in the regulation of BBB permeability<sup><a href="#CR15">15</a></sup>. TJs are the luminal-most cellular junction and are composed of claudins, occludin, and zona occludens, which seal the paracellular route between BMECs, creating a high transendothelial cell electrical resistance barrier that impedes ions and small charged molecules from crossing the BBB. However, AJs are positioned abluminal to TJs and consist of VE-cadherin dimers that mediate cell–cell membrane adhesion and bind to the actin cytoskeleton via catenins in BMECs<sup><a href="#CR29">29</a></sup>. For instance, rabies virus infection promotes the release of a large number of inflammatory cytokines, which further reduces the expression of TJ proteins in the vascular endothelial cells of the BBB and ultimately results in increased BBB permeability<sup><a href="#CR30">30</a></sup>. Thus, maintaining the intact structure and function of TJ-associated and AJ-associated proteins is essential for BBB integrity. In our study, the expression levels of TJ-associated proteins Claudin4, Claudin5, and ZO-1, as well as the AJ-associated protein VE-cadherin, were significantly reduced in HUVECs after infection with CA16 (Fig. <a href="#Fig4">4</a>). Moreover, the expression levels of these proteins were not remarkably affected in CA16-infected HUVECs that were pretransfected with a si-MMP9 plasmid (Fig. <a href="#Fig5">5</a>). Claudin4, Claudin5, and ZO-1 are often considered sensitive indicators of normal and disturbed functional states of the BBB, while VE-cadherin, a member of a transmembrane Ca<sup>2+</sup>-dependent adhesion molecule family, regulates vascular permeability in the BBB<sup><a href="#CR31">31</a></sup>. Therefore, our results suggested that CA16 infections result in BBB dysfunction likely through modulating the expression of TJ-associated and AJ-associated proteins, which might be intimately related to MMP9. MMP9 is normally only present at low levels in the brain tissue and cerebrospinal fluid but is markedly upregulated in many infectious diseases of the CNS<sup><a href="#CR32">32</a></sup>. Subsequently, based on the target interaction between miR-1303 and MMP9, we continued to investigate the effects of miR-1303 on the above junctional proteins<sup><a href="#CR21">21</a></sup>. The results revealed that the expression levels of Claudin4, Claudin5, ZO-1, and VE-cadherin were not notably reduced in CA16-infected HUVECs pretransfected with a miR-1303 plasmid (Fig. <a href="#Fig6">6</a>). Hence, this result indicated that the downregulation of miR-1303 induced by CA16 infections in HUVECs reduced the expression levels of Claudin4, Claudin5, ZO-1, and VE-cadherin by accelerating the release of MMP9, which might explain why the BBB permeability increased during EV71 and CA16 infections.</p> <p id="Par15">To further confirm whether the degradation of junctional complexes could facilitate entry of CA16 into the CNS by breaking the BBB, which eventually leads to CNS damage, we conducted in vivo animal experiments (Fig. <a href="#Fig7">7a</a>). The viral infection rate in EGFP-CA16 + MMP9 monkeys was higher than that in EGFP-CA16 monkeys and further higher than that in EGFP-CA16 + TIMP-1 monkeys (Fig. <a href="#Fig7">7c</a> and Fig S<a href="#MOESM9">8</a>). Thus, these results confirmed that the brain tissues might have been susceptible to CA16 infection mainly because of the effects of MMP9 on BBB permeability, which eventually caused more severe CNS damage. Moreover, morphological examination revealed that the neuropathological lesions were the most marked in brain tissues of EGFP-CA16 + MMP9 monkeys, with obvious hemorrhagic injuries. Nevertheless, the above pathological changes were mild in the EGFP-CA16 monkeys and were not apparent in the EGFP-CA16 + TIMP-1 monkeys (Fig. <a href="#Fig7">7e</a>).</p> <p id="Par16">Our results are the first to demonstrate that CA16 infections in HUVECs can decrease the expression of miR-1303, which relieves the inhibitory role in MMP9 release and thus promotes MMP9 expression. Therefore, increased MMP9 expression enhanced the BBB permeability by directly degrading junctional complexes, which allowed the virus to enter the CNS and ultimately led to CNS lesions (Fig. <a href="#Fig8">8</a>). We will further consider verifying this conclusion in humanized mice or genetically modified rhesus monkeys. These results might illustrate the potential molecular mechanism of BBB permeability during CA16 infections. This study not only provides a target for the clinical treatment of CA16 infections but also offers an early warning molecule for severe HFMD patients. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <div class="title" tagx="title" title="title">
Summary of the CA16-infected CNS lesion process.</div> <p>CA16 penetrates the BBB and enter the CNS by downregulating miR-1303, which induced the disruption of junctional complexes by directly regulating MMP9 and ultimately caused the pathological CNS changes</p>   </div> <p /> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="ethicsstatementandanimalexperiments" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement and animal experiments</div> <p id="Par17">All animal experimentation was performed with the approval of the Yunnan Provincial Experimental Animal Management Association (Approval number: SCXK [Dian] 2011-0005), the Experimental Animal Ethics Committee of the institute (Approval number: YISHENGLUNZI [2011]15), and the Experimental Animal Administration and Ethics Committee of the Institute of Medical Biology, Chinese Academy of Medical Sciences (IMBCAMS) (Approval number: [2017]16) and in accordance with the principles of "Guide for the Care and Use of Laboratory Animals�? and "The Guidance to Experimental Animal Welfare and Ethical Treatment.�?</p> <p id="Par18">The rhesus monkeys were reared separately in a large cage (BSL-2 conditions) with sufficient fresh air and natural light, which allowed visual, olfactory, and auditory interactions with other monkeys. Each room had a temperature control valve guaranteed to maintain the room temperature at ~25 °C, and food, water, and fruit were readily available. Appropriate treats and vitamins were provided. The animals were given access to environmental enrichment (such as approved toys) to promote their psychological wellbeing. All animals were fully under the care of veterinarians at IMBCAMS.</p> <p id="Par19">A CA16-infected rhesus monkey model was generated by nasal spray using 5 lgCCID<sub>50</sub>/100 µL EGFP-CA16, while two rhesus monkeys were given MMP9 (150 ng/100 μL) or TIMP-1 (150 ng/100 μL) (an MMP9 inhibitor) recombinant proteins through intravenous injection every 2 days. One rhesus monkey injected with phosphate-buffered saline (PBS) served as a control. After 6 days, all rhesus monkeys were euthanized using ketamine as the anesthetic (10 mg/kg of body weight), and samples were collected for viral load examination, morphological analysis, and junctional protein evaluation.</p> </div> <div class="cellcultureandvirusinfection" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and virus infection</div> <p id="Par20">Human umbilical vein endothelial cells (HUVECs) purchased from the American Type Culture Collection (Manassas, USA) were maintained in Dulbecco’s modified Eagle’s medium (Corning, USA) supplemented with 10% fetal bovine serum (Gibco, USA), 2 mmol/L L-glutamine, 100 U/mL penicillin, and 100 µg/mL streptomycin at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>. Strains EV71 (subgenotype C4, GenBank: EU812515.1) or CA16-G20 (subgenotype B, GenBank: JN590244.1) were respectively isolated from an epidemic in Fuyang, China in 2008 and from an HFMD patient in Guangxi, China in 2010 and were used throughout the study. The HUVECs were cultivated in 6-well plates to 80% confluence and then infected with EV71 or CA16 at a multiplicity of infection (MOI) of 0.1. The infected cells were collected at 0, 72, and 96 h post infection (hpi). Cells infected with EV71 and CA16 at 0 hpi were used as controls.</p> <p id="Par21">African green monkey kidney (Vero) cells used for viral plaque assays were obtained from IMBCAMS (Kunming, China) and grown in minimum essential medium with 10% newborn bovine serum (Gibco, USA), L-glutamine, and 1% penicillin/streptomycin at 37 °C in a 95% air and 5% CO<sub>2</sub> atmosphere.</p> <p id="Par22">293T cells for the dual-luciferase reporter assay were provided by IMBCAMS (Kunming, China) and were cultured in Dulbecco’s modified Eagle medium containing 10% fetal bovine serum and 1% penicillin/streptomycin solution in a humidified incubator at 37 °C with 5% CO<sub>2</sub>.</p> </div> <div class="virustitrationofev71andca16inhuvecs" title="sec"> <div class="title" tagx="title" title="title">
Virus titration of EV71 and CA16 in HUVECs</div> <p id="Par23">A plaque assay was used to quantify infectious viruses recovered from HUVECs at the abovementioned timepoints. Briefly, the harvested cells were disrupted by three freeze-thaw cycles, and serial dilutions of all samples were inoculated into confluent Vero cells on 6-well plates. After 3 h of incubation to allow virus attachment, the wells were gently washed with PBS, covered with 2 mL of minimum essential medium containing 1% agarose, and inverted into a 37 °C CO<sub>2</sub> incubator for 48 h. Next, the cells were fixed with 2 mL of 4% paraformaldehyde (Solarbio, China) at 15 °C to 25 °C for 30 min, and the 1% agarose was removed. The monolayer of cells was stained with a crystal violet staining solution for 15 min and washed with ddH<sub>2</sub>O. Finally, visible plaques were counted by the naked eye, and the plaque-forming units (pfu/mL) were calculated with the virus titer formula, in which the virus titer equals the number of plaques × (1 mL) × dilution factor.</p> </div> <div class="measurementofhuvecpermeability" title="sec"> <div class="title" tagx="title" title="title">
Measurement of HUVEC permeability</div> <p id="Par24">HUVECs were grown to confluent monolayers on clear polyester Transwell filters (0.4 mm pore size, 12 mm diameter; Corning, USA). After completely treating with EV71 and CA16, we assessed the integrity and uniformity of the HUVEC monolayers with an In Vitro Vascular Permeability Assay Kit (Millipore, USA). Briefly, the medium was carefully removed from the Transwell insert, and 150 μL of FITC-dextran was added to the top chamber for incubation at room temperature for 20 min in the dark. At the end of the incubation period, the FITC-dextran content of the lower chamber was measured using a fluorescence plate reader (Bio-Rad, USA) at an excitation wavelength of 485 nm and an emission wavelength of 530 nm.</p> </div> <div class="flowcytometryassay" title="sec"> <div class="title" tagx="title" title="title">
Flow cytometry assay</div> <p id="Par25">Cell apoptosis evaluation was performed by flow cytometry. The harvested cells were washed with PBS and resuspended at a density of 1 × 10<sup>6</sup> cells/mL in 1 × Annexin V binding buffer followed by thorough mixing. After double staining with 5 μL of Annexin V-FITC and 5 μL of propidium iodide per sample for 15 min at 25 °C in the dark using the Annexin V-FITC Apoptosis Detection Kit (Beyotime, China), the cells were analyzed using a FACScan<sup>®</sup> flow cytometer equipped with Cell Quest software (BD Biosciences, USA), according to manufacturer’s instructions, to detect early and late apoptosis of cells.</p> </div> <div class="bioinformaticsanalysisofmirnasequencingdata" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics analysis of miRNA sequencing data</div> <p id="Par26">According to previous bioinformatics analysis methods, principal component analysis, differential expression analysis of miRNAs, unsupervised hierarchical clustering, trend analysis, target prediction, gene ontology (GO) and pathway enrichment of differentially expressed miRNAs, and coexpression network analysis of key target genes were systematically conducted study utilizing the corresponding software<sup><a href="#CR33">33</a></sup>. The sequencing data were deposited in the National Center for Biotechnology Information’s Gene Expression Omnibus (GEO) database (<a href="http://www.ncbi.nlm.nih.gov/geo/">www.ncbi.nlm.nih.gov/geo/</a>) and are accessible at series number GSE94551.</p> </div> <div class="quantitativereversetranscription-polymerasechainreaction(qrt-pcr)forviralloadandmir-1303andmmp9expression" title="sec"> <div class="title" tagx="title" title="title">
Quantitative reverse transcription-polymerase chain reaction (qRT-PCR) for viral load and miR-1303 and MMP9 expression</div> <p id="Par27">For viral load examination, total RNA was extracted from all samples using the standard TRIzol reagent (TIANGEN, China) protocol. The regions of standards generated from EV71-VP1 and CA16-VP1 RNA were amplified by the PrimeSTAR<sup>®</sup> HS reagent (TAKARA, Japan) with the primers shown in Table S<a href="#MOESM1">1</a>. The target 624-bp region (EV71-VP1) and 682-bp region (CA16-VP1) were cloned into the pGM-T vector (TIANGEN, China) following the manufacturer’s guidelines. A single recombinant clone containing the insert under the T7 promoter was selected, and the insert was transcribed in vitro using the TranscriptAid T7 High Yield Transcription Kit (Thermo Scientific, USA) according to the manufacturer’s recommendations. The in vitro-transcribed RNA was purified using the GeneJET RNA Purification kit (Thermo Scientific, USA) and quantified spectrophotometrically using the Nanodrop 2000 (Thermo Scientific, USA). The qRT-PCR process was performed using a One Step PrimeScript™ RT-PCR Kit (TAKARA, Japan). To generate a standard curve for cycle thresholds (Cts) versus virus copy number, the purified RNA was serially diluted to known concentrations in the range of 10<sup>1</sup> to 10<sup>8</sup> copies/μL and assayed concurrently with the test samples. The PCR amplification cycle was set at 42 °C for 5 min for reverse transcription, followed by 95 °C for 10 s for initial denaturation and 40 cycles of 5-s denaturation at 95 °C and 34-s annealing at 60 °C. The reactions were run on a 7500 Fast Real-time PCR System (Applied Biosystems, USA).</p> <p id="Par28">For the miR-1303 analysis, miRNA was enriched by a miRcute miRNA Isolation Kit (TIANGEN, China). Subsequently, 2 μg of RNA was first polyadenylated with polyA polymerase and reverse-transcribed into cDNA with a poly(T) adapter primer under the following reaction conditions: 60 min at 37 °C, 5 min at 85 °C, and maintenance at 4 °C. Finally, PCR was performed using SYBR Premix Ex Taq (TAKARA, Japan) in accordance with the manufacturer’s protocols, on an ABI7500 real-time PCR system (Applied Biosystems, USA). The amplification conditions were as follows: 95 °C for 10 s, 40 cycles of 95 °C for 10 s, and 60 °C for 40 s, and dissociation at 95 °C for 60 s, 55 °C for 30 s, and 95 °C for 30 s. The qRT-PCR results were calculated with the comparative threshold cycle (Ct) method and U6 snRNA, which was provided with the kit and served as an internal control. The primers used in this study are listed in Table S<a href="#MOESM1">1</a>. The relative amount of expressed miRNAs and mRNAs was also calculated by the 2<sup>−ΔΔCt</sup> method.</p> </div> <div class="dual-luciferasereporterassay" title="sec"> <div class="title" tagx="title" title="title">
Dual-luciferase reporter assay</div> <p id="Par29">The miR-1303 normal and mutated seed sequences in the 3′-untranslated (UTR) of the MMP9 gene were chemically synthesized in vitro (Sangon Biotech, China), and the Xho I and Not I restriction sites were added to both ends. The two DNA fragments were cloned into the psiCHECK-2 luciferase reporter plasmid (Promega, USA) to generate a wild-type (WT) MMP9-3′UTR plasmid and a mutant MMP9-3′UTR plasmid. For the dual-luciferase reporter assay, 293T cells were seeded into 96-well plates and transfected with different combinations of miR-1303 mimics, miR-1303 inhibitor, negative control (NC) plasmid, NC inhibitor, WT MMP9-3′UTR plasmid, and mutant MMP9-3′UTR plasmid using Lipofectamine 2000 (Promega, USA), strictly according to the manufacturer’s instructions. After 48 h of incubation, the cells from each group were lysed, and the fluorescence intensity was measured with a GloMax 20/20 luminometer (Promega, USA) using Renilla luciferase as the internal reference.</p> </div> <div class="enzyme-linkedimmunosorbentassay(elisa)" title="sec"> <div class="title" tagx="title" title="title">
Enzyme-linked immunosorbent assay (ELISA)</div> <p id="Par30">Cell samples were centrifuged at 3000 × <i>g</i> for 10 min at 4 °C, and supernatants were assessed using a commercial MMP9 ELISA kit (Abcam, USA) according to the manufacturer’s recommendations. In microplates, standards (50 µL), samples (10 µL sample liquid and 40 µL diluent), and blanks were added into predefined wells and incubated for 1 h at room temperature. After the plates were washed twice with wash buffer, 100 µL of horseradish peroxidase (HPR)-labeled conjugates was added prior to sealing the plates for incubation at 37 °C for 1 h. Following five stringent washes, substrates A (50 µL) and B (50 µL) were added into each well for incubation at 37 °C for 15 min. Eventually, a stop solution (50 µL) was added into each well, and the absorbance of each well was measured at 450 nm within 15 min using a microplate reader (Thermo Fisher, USA). MMP9 levels in these samples were located within the linear range of the standard curves.</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western blot analysis</div> <p id="Par31">Cultured cells and tissue specimens were homogenized with radioimmunoprecipitation assay lysis buffer containing 50 mM Tris-HCl (pH 7.5), 150 mM NaCl, 0.1% Nonidet P-40, and a mixture of protease inhibitors for 20 min on ice. Further, the lysates were centrifuged to remove the cellular debris, and protein concentrations were measured with the Bradford protein assay (Thermo Fisher, USA) following the manufacturer’s protocols. Equal amounts of protein (30 μg) were subjected to 8–10% sodium dodecyl sulfate polyacrylamide gel electrophoresis based on their molecular weight and transferred to a PVDF membrane (Millipore, USA). After blocking with 5% skim milk in Tris-buffered saline with 0.1% Tween-20 solution for 2 h at room temperature, the membrane was incubated with primary antibodies against MMP9 (1:1000 dilution), Claudin4 (1:1500 dilution), Claudin5 (1:1500 dilution), VE-Cadherin (1:1200 dilution), ZO-1 (1:2000 dilution), and GAPDH (1:500 dilution; as a loading control) at 4 °C overnight. Antibodies specific to the above proteins were obtained from Abcam, USA. Subsequently, the blot was thoroughly washed with TBST three times for 15 min each and incubated with HRP-conjugated secondary antibodies (1:12,000 dilution; Abmart, USA) for 1 h at room temperature. Immunoreactive bands were developed with an enhanced chemiluminescence kit (Beyotime, China) following the manufacturer’s manual and then detected with a ChemiDoc™ Imaging System (Bio-Rad, USA). Finally, the gray values of the protein bands were quantified using ImageJ software.</p> </div> <div class="immunofluorescence(if)microscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence (IF) microscopy</div> <p id="Par32">HUVECs were seeded onto poly-L-lysine-coated coverslips (Solarbio, China). At the indicated time, the cells were washed twice with PBS, fixed with 4% paraformaldehyde for 30 min, and permeabilized with 1% Triton X-100 in PBS for 15 min at room temperature. After blocking with 5% bovine serum albumin, the cells were incubated with the primary antibodies in blocking solution against EV71/CA16-VP1 (1:1000 dilution; Millipore, USA), Nectin1 (1:100 dilution), Claudin4 (1:200 dilution), VE-cadherin (1:500 dilution), and ZO-1 (1:200 dilution) overnight at 4 °C. Next, the cells were washed with PBS three times and then incubated with Alexa Fluor 594-conjugated donkey anti-rabbit IgG (1:1000 dilution; Abcam, USA) or Alexa Fluor 647-conjugated donkey anti-mouse IgG (1:1000 dilution; Millipore, USA) for 1 h at room temperature. The nuclei were counterstained with 4′, 6-diamidino-2-phenylindole (DAPI, 1:4000 dilution; Beyotime, China). Following three washes in PBS, the coverslips were mounted on slides with an antifade reagent (Solarbio, China). Finally, images of the cells were captured with a laser-scanning confocal microscope (Leica, Germany) and processed using Adobe Photoshop 7.0 software.</p> </div> <div class="hematoxylinandeosinstaining" title="sec"> <div class="title" tagx="title" title="title">
Hematoxylin and eosin staining</div> <p id="Par33">After sacrifice, a rhesus monkey was necropsied to obtain lung and brain tissues. All the tissue samples were fixed in 10% formalin at pH 7.4, embedded in paraffin, and sectioned with a microtome into 5-μm-thick sections. The sections were deparaffinized in xylene, dehydrated with graded ethanol, and washed three times with PBS (5 min per wash). Subsequently, the sections were subjected to hematoxylin and eosin staining to evaluate the damage to the lung and brain tissues. Finally, the slides were passed through stepwise dehydration in increasing concentrations of ethanol (70–100%), 5 min for each step, followed by 5~10 min in xylene. The sections were observed and photographed under a light microscope (Leica, Germany).</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par34">The data are expressed as the means ± standard deviation of experiments repeated at least three independent times and were analyzed with SPSS 20.0 software (IBM, USA). Two-tailed Student’s <i>t</i>-tests were used for comparisons between two groups. Multigroup measurement data were analyzed using one-way analysis of variance. All <i>P</i> values were two-sided, and a <i>P</i> value less than 0.05 was considered statistically significant.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM1_ESM.docx">LINK</a> <p>Table S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM2_ESM.tif">LINK</a> <p>Figure S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM3_ESM.tif">LINK</a> <p>Figure S2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM4_ESM.tif">LINK</a> <p>Figure S3</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM5_ESM.tif">LINK</a> <p>Figure S4</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM6_ESM.tif">LINK</a> <p>Figure S5</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM7_ESM.tif">LINK</a> <p>Figure S6</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM8_ESM.tif">LINK</a> <p>Figure S7</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_157_MOESM9_ESM.tif">LINK</a> <p>Figure S8</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Jie Song, Yajie Hu and Hongzhe Li</p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p><b>Supplementary Information</b> accompanies this paper at (10.1038/s41426-018-0157-3).</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This study is funded by CAMS Innovation Fund for Medical Sciences (2016-I2M-1-014 and 2016-I2M-1-019), National Natural Sciences Foundations of China (31700153), Fundamental Research Funds for the Central Universities and PUMC Youth Fund (2017310039), Yunnan Applied Basic Research Projects (2018FB026), Yunnan provincial innovation team (2016HC009) and 2017 Yunnan Medical Reserve Talents (H-2017034). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p id="Par35">Conceived and designed the experiments: L.D.L., Q.H.L., and SZD. Performed the experiments: J.S., Y.J.H., H.Z.L., and X.H. Analyzed the data: H.W.Z. and Y.G.H. Contributed reagents, materials, analysis tools: J.J.W., X.J., J.Q.L., Z.N.Y., H.T.F., L.G. Animal experiment: H.J.S., Z.L.H., F.M.Y., and X.W. Wrote the paper: J.S. and Y.J.H. Edited the manuscript: L.D.L., Q.H.L., and S.Z.D.</p> </div> </div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par36">The authors declare that they have no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aswathyraj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arunkumar</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alidjinou</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hober</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hand, foot and mouth disease (HFMD): emerging epidemiology and the need for a vaccine strategy</span><span class="source" tagx="source" title="source">Med. Microbiol. Immunol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">205</span><span class="fpage" tagx="fpage" title="fpage">397</span><span class="lpage" tagx="lpage" title="lpage">407</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00430-016-0465-y">10.1007/s00430-016-0465-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27406374">27406374</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bian</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">EV71 vaccine, a new tool to control outbreaks of hand, foot and mouth disease (HFMD)</span><span class="source" tagx="source" title="source">Expert. Rev. Vaccin.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">599</span><span class="lpage" tagx="lpage" title="lpage">606</span><span class="pub-id"><a href="https://dx.doi.org/10.1586/14760584.2016.1138862">10.1586/14760584.2016.1138862</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pathologic and immunologic characteristics of coxsackievirus A16 infection in rhesus macaques</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">500</span><span class="fpage" tagx="fpage" title="fpage">198</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2016.10.031">10.1016/j.virol.2016.10.031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27829175">27829175</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Coxsackievirus A16: epidemiology, diagnosis, and vaccine</span><span class="source" tagx="source" title="source">Hum. Vaccin. Immunother.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">360</span><span class="lpage" tagx="lpage" title="lpage">367</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/hv.27087">10.4161/hv.27087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24231751">24231751</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardosa</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Status of research and development of vaccines for enterovirus 71</span><span class="source" tagx="source" title="source">Vaccine</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">2967</span><span class="lpage" tagx="lpage" title="lpage">2970</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2016.02.077">10.1016/j.vaccine.2016.02.077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26973065">26973065</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Acute pancreatitis in hand, foot and mouth disease caused by Coxsackievirus A16: case report</span><span class="source" tagx="source" title="source">Korean J. Pediatr.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">333</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.3345/kjp.2017.60.10.333">10.3345/kjp.2017.60.10.333</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29158768">29158768</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">SZ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemics and aetiology of hand, foot and mouth disease in Xiamen, China, from 2008 to 2015</span><span class="source" tagx="source" title="source">Epidemiol. Infect.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">145</span><span class="fpage" tagx="fpage" title="fpage">1865</span><span class="lpage" tagx="lpage" title="lpage">1874</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268817000309">10.1017/S0950268817000309</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28367766">28367766</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Pin</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bao</span><span class="given-names" tagx="given-names" title="given-names">Linlin</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Lili</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Fengdi</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Qi</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Wei</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Yanfeng</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">Chuan</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neurotropism In Vitro and Mouse Models of Severe and Mild Infection with Clinical Strains of Enterovirus 71</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">351</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v9110351">10.3390/v9110351</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytokine immunopathogenesis of enterovirus 71 brain stem encephalitis</span><span class="source" tagx="source" title="source">Clin. Dev. Immunol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">2012</span><span class="fpage" tagx="fpage" title="fpage">876241</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22956971">22956971</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">PN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shih</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neural pathogenesis of enterovirus 71 infection</span><span class="source" tagx="source" title="source">Microbes Infect.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">505</span><span class="lpage" tagx="lpage" title="lpage">510</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micinf.2010.03.006">10.1016/j.micinf.2010.03.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20348010">20348010</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">The molecular constituents of the blood-brain barrier</span><span class="source" tagx="source" title="source">Trends Neurosci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">598</span><span class="lpage" tagx="lpage" title="lpage">608</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tins.2015.08.003">10.1016/j.tins.2015.08.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26442694">26442694</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orillo</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">West Nile virus-induced cell adhesion molecules on human brain microvascular endothelial cells regulate leukocyte adhesion and modulate permeability of the in vitro blood-brain barrier model</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e102598</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0102598">10.1371/journal.pone.0102598</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25036379">25036379</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hannocks, M. J. et al. The gelatinases, MMP-2 and MMP-9, as fine tuners of neuroinflammatory processes. <i>Matrix. Biol</i>. 10.1016/j.matbio.2017.11.007 (2017).</span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">FR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Implications of MMP9 for blood brain barrier disruption and hemorrhagic transformation following ischemic stroke</span><span class="source" tagx="source" title="source">Front. Cell. Neurosci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fncel.2016.00056">10.3389/fncel.2016.00056</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26973468">26973468</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatovic</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keep</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andjelkovic</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Junctional proteins of the blood-brain barrier: new insights into function and dysfunction</span><span class="source" tagx="source" title="source">Tissue Barriers</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e1154641</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/21688370.2016.1154641">10.1080/21688370.2016.1154641</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27141427">27141427</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global gene expression analysis of peripheral blood mononuclear cells in rhesus monkey infants with CA16 infection-induced HFMD</span><span class="source" tagx="source" title="source">Virus Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2016.01.002">10.1016/j.virusres.2016.01.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26775814">26775814</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Coxsackievirus A16 utilizes cell surface heparan sulfate glycosaminoglycans as its attachment receptor</span><span class="source" tagx="source" title="source">Emerg. Microbes Infect.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e65</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2017.55">10.1038/emi.2017.55</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28745308">28745308</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikonomidou</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Matrix metalloproteinases and epileptogenesis</span><span class="source" tagx="source" title="source">Mol. Cell. Pediatr.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40348-014-0006-y">10.1186/s40348-014-0006-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26567100">26567100</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span></span><span class="mixed-article-title" title="mixed-article-title">From blood-brain barrier to blood-brain interface: new opportunities for CNS drug delivery</span><span class="source" tagx="source" title="source">Nat. Rev. Drug. Discov.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">275</span><span class="lpage" tagx="lpage" title="lpage">292</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd.2015.21">10.1038/nrd.2015.21</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26794270">26794270</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frey</span><span class="given-names" tagx="given-names" title="given-names">BN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Disruption in the blood-brain barrier: the missing link between brain and body inflammation in bipolar disorder?</span><span class="source" tagx="source" title="source">Neural Plast.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">2015</span><span class="fpage" tagx="fpage" title="fpage">708306</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2015/708306">10.1155/2015/708306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26075104">26075104</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tirumuru, N., Pretto, C. D., Castro Jorge, L. A. &amp;amp; Spindler, K. R. Mouse adenovirus type 1 early region 1A effects on the blood-brain barrier. <i>mSphere</i><b>1</b>, 10.1128/mSphere.00079-16 (2016).</span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Kooij</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Role for MMP-9 in stress-induced downregulation of nectin-3 in hippocampal CA1 and associated behavioural alterations</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">4995</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms5995">10.1038/ncomms5995</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25232752">25232752</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of miRNAs involved in mouse brain damage upon enterovirus 71 infection</span><span class="source" tagx="source" title="source">Front. Cell. Infect. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">133</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28469998">28469998</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ablation of CD11c(hi) dendritic cells exacerbates Japanese encephalitis by regulating blood-brain barrier permeability and altering tight junction/adhesion molecules</span><span class="source" tagx="source" title="source">Comp. Immunol. Microbiol. Infect. Dis.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="lpage" tagx="lpage" title="lpage">32</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cimid.2016.07.007">10.1016/j.cimid.2016.07.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27638116">27638116</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betsholtz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zlokovic</span><span class="given-names" tagx="given-names" title="given-names">BV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Establishment and dysfunction of the blood-brain barrier</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">163</span><span class="fpage" tagx="fpage" title="fpage">1064</span><span class="lpage" tagx="lpage" title="lpage">1078</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2015.10.067">10.1016/j.cell.2015.10.067</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26590417">26590417</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Progress and challenge of microRNA research in immunity</span><span class="source" tagx="source" title="source">Front. Genet.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fgene.2014.00178">10.3389/fgene.2014.00178</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24971086">24971086</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nectin and junctional adhesion molecule are critical cell adhesion molecules for the apico-basal alignment of adherens and tight junctions in epithelial cells</span><span class="source" tagx="source" title="source">Genes Cells</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">985</span><span class="lpage" tagx="lpage" title="lpage">998</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/gtc.12091">10.1111/gtc.12091</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24112238">24112238</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">miR-1303 targets claudin-18 gene to modulate proliferation and invasion of gastric cancer cells</span><span class="source" tagx="source" title="source">Dig. Dis. Sci.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">1754</span><span class="lpage" tagx="lpage" title="lpage">1763</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10620-014-3107-5">10.1007/s10620-014-3107-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24647998">24647998</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geldenhuys</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammad</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adkins</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lockman</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular determinants of blood-brain barrier permeation</span><span class="source" tagx="source" title="source">Ther. Deliv.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">961</span><span class="lpage" tagx="lpage" title="lpage">971</span><span class="pub-id"><a href="https://dx.doi.org/10.4155/tde.15.32">10.4155/tde.15.32</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26305616">26305616</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">WQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">ZF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enhancement of blood-brain barrier permeability and reduction of tight junction protein expression are modulated by chemokines/cytokines induced by rabies virus infection</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">4698</span><span class="lpage" tagx="lpage" title="lpage">4710</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03149-13">10.1128/JVI.03149-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522913">24522913</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tietz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engelhardt</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Brain barriers: crosstalk between complex tight junctions and adherens junctions</span><span class="source" tagx="source" title="source">J. Cell. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">209</span><span class="fpage" tagx="fpage" title="fpage">493</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201412147">10.1083/jcb.201412147</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26008742">26008742</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sellebjerg</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorensen</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemokines and matrix metalloproteinase-9 in leukocyte recruitment to the central nervous system</span><span class="source" tagx="source" title="source">Brain Res. Bull.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">347</span><span class="lpage" tagx="lpage" title="lpage">355</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0361-9230(03)00097-2">10.1016/S0361-9230(03)00097-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12909304">12909304</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Different microRNA profiles reveal the diverse outcomes induced by EV71 and CA16 infection in human umbilical vein endothelial cells using high-throughput sequencing</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">e0177657</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0177657">10.1371/journal.pone.0177657</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28531227">28531227</a></span></span></li> </ul> </div> </div>  </body></html>