<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/11/2020.04.10.029454.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.10.029454v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.10.029454"/>
  <meta name="DC.Date" content="2020-04-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &gt;;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Shay Leary"/>
  <meta name="DC.Contributor" content="Silvana Gaudieri"/>
  <meta name="DC.Contributor" content="Abha Chopra"/>
  <meta name="DC.Contributor" content="Suman Pakala"/>
  <meta name="DC.Contributor" content="Eric Alves"/>
  <meta name="DC.Contributor" content="Mina John"/>
  <meta name="DC.Contributor" content="Suman Das"/>
  <meta name="DC.Contributor" content="Simon Mallal"/>
  <meta name="DC.Contributor" content="Phillips Jane Phillips"/>
  <meta name="article:published_time" content="2020-04-11"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &amp;;gt;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.10.029454v1"/>
  <meta name="citation_id" content="2020.04.10.029454v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.10.029454v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.10.029454v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.10.029454v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/11/2020.04.10.029454.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.10.029454"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.10.029454"/>
  <meta name="citation_author" content="Shay Leary"/>
  <meta name="citation_author_institution" content="Institute for Immunology and Infectious Diseases, Murdoch University;"/>
  <meta name="citation_author_email" content="s.leary@iiid.com.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6966-2294"/>
  <meta name="citation_author" content="Silvana Gaudieri"/>
  <meta name="citation_author_institution" content="University of Western Australia;"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6873-0198"/>
  <meta name="citation_author" content="Abha Chopra"/>
  <meta name="citation_author_institution" content="Institute for Immunology and Infectious Diseases, Murdoch University;"/>
  <meta name="citation_author_email" content="a.chopra@iiid.com.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5008-9080"/>
  <meta name="citation_author" content="Suman Pakala"/>
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center"/>
  <meta name="citation_author_email" content="suman.b.pakala@vumc.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5156-0823"/>
  <meta name="citation_author" content="Eric Alves"/>
  <meta name="citation_author_institution" content="University of Western Australia;"/>
  <meta name="citation_author_email" content="eric.alves@research.uwa.edu.au"/>
  <meta name="citation_author" content="Mina John"/>
  <meta name="citation_author_institution" content="Institute for Immunology and Infectious Diseases, Murdoch University;"/>
  <meta name="citation_author_email" content="mina.john@health.wa.gov.au"/>
  <meta name="citation_author" content="Suman Das"/>
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center"/>
  <meta name="citation_author_email" content="suman.r.das@vumc.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2496-9724"/>
  <meta name="citation_author" content="Simon Mallal"/>
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center"/>
  <meta name="citation_author_email" content="s.mallal@vumc.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7036-1309"/>
  <meta name="citation_author" content="Phillips Jane Phillips"/>
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center"/>
  <meta name="citation_author_email" content="e.phillips@iiid.com.au"/>
  <meta name="twitter:title" content="Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &gt;;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.10.029454v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &gt;;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-11"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1240364 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-category-new-results"> 
  <div class="article abstract-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-2">The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &gt;;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations.</p>
   </div>
   <h3>Competing Interest Statement</h3>
   <p id="p-3">The authors have declared no competing interest.</p>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
