<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/01/03/242495.atom" />
<meta name="HW.pisa" content="biorxiv;242495v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Viral gain-of-function experiments uncover residues under diversifying selection in nature" />
<meta name="DC.Identifier" content="10.1101/242495" />
<meta name="DC.Date" content="2018-01-03" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology." />
<meta name="DC.Contributor" content="Rohan Maddamsetti" />
<meta name="DC.Contributor" content="Daniel T. Johnson" />
<meta name="DC.Contributor" content="Stephanie J. Spielman" />
<meta name="DC.Contributor" content="Katherine L. Petrie" />
<meta name="DC.Contributor" content="Debora S. Marks" />
<meta name="DC.Contributor" content="Justin R. Meyer" />
<meta name="article:published_time" content="2018-01-03" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Viral gain-of-function experiments uncover residues under diversifying selection in nature" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;242495v1" />
<meta name="citation_id" content="242495v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/242495v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/242495v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/242495v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495.full.pdf" />
<meta name="citation_doi" content="10.1101/242495" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="242495" />
<meta name="citation_author" content="Rohan Maddamsetti" />
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School" />
<meta name="citation_author" content="Daniel T. Johnson" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of California San Diego" />
<meta name="citation_author" content="Stephanie J. Spielman" />
<meta name="citation_author_institution" content="Institute for Genomics and Evolutionary Medicine, Temple University" />
<meta name="citation_author" content="Katherine L. Petrie" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of California San Diego" />
<meta name="citation_author_institution" content="Earth-Life Science Institute, Tokyo Institute of Technology" />
<meta name="citation_author" content="Debora S. Marks" />
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School" />
<meta name="citation_author" content="Justin R. Meyer" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of California San Diego" />
<meta name="citation_reference" content="Longdon, B., Brockhurst, M. A., Russell, C. A., Welch, J. J. &amp; Jiggins, F. M. The evolution and genetics of virus host shifts. PLoS Pathog 10, e1004395, doi:10.1371/journal.ppat.1004395 (2014)." />
<meta name="citation_reference" content="Imai, M. et al. Experimental adaptation of an influenza H5 HA confers respiratory droplet transmission to a reassortant H5 HA/H1N1 virus in ferrets. Nature 486, 420–428, doi:10.1038/nature 10831 (2012)." />
<meta name="citation_reference" content="Meyer, J. R. et al. Repeatability and contingency in the evolution of a key innovation in phage lambda. Science 335, 428–432, doi:10.1126/science.1214449 (2012)." />
<meta name="citation_reference" content="Antia, R., Regoes, R. R., Koella, J. C. &amp; Bergstrom, C. T. The role of evolution in the emergence of infectious diseases. Nature 426, 658–661, doi:10.1038/nature02104 (2003)." />
<meta name="citation_reference" content="Shi, Y., Wu, Y., Zhang, W., Qi, J. &amp; Gao, G. F. Enabling the ‘host jump’: structural determinants of receptor-binding specificity in influenza A viruses. Nature reviews. Microbiology 12, 822–831, doi:10.1038/nrmicro3362 (2014)." />
<meta name="citation_reference" content="Song, H. et al. Avian-to-Human Receptor-Binding Adaptation by Influenza A Virus Hemagglutinin H4. Cell reports 20, 1201–1214, doi:10.1016/j.celrep.2017.07.028 (2017)." />
<meta name="citation_reference" content="Koel, B. F. et al. Substitutions near the receptor binding site determine major antigenic change during influenza virus evolution. Science 342, 976–979, doi:10.1126/science.1244730 (2013)." />
<meta name="citation_reference" content="Linster, M. et al. Identification, characterization, and natural selection of mutations driving airborne transmission of A/H5N1 virus. Cell 157, 329–339, doi:10.1016/j.cell.2014.02.040 (2014)." />
<meta name="citation_reference" content="de Wit, E., van Doremalen, N., Falzarano, D. &amp; Munster, V. J. SARS and MERS: recent insights into emerging coronaviruses. Nature reviews. Microbiology 14, 523–534, doi:10.1038/nrmicro.2016.81 (2016)." />
<meta name="citation_reference" content="Lu, G. et al. Molecular basis of binding between novel human coronavirus MERS-CoV and its receptor CD26. Nature 500, 227–231, doi:10.1038/nature12328 (2013)." />
<meta name="citation_reference" content="Rambaut, A., Posada, D., Crandall, K. A. &amp; Holmes, E. C. The causes and consequences of HIV evolution. Nature Reviews Genetics 5, 52, doi:10.1038/nrg1246 (2004)." />
<meta name="citation_reference" content="Holmes, E. C., Dudas, G., Rambaut, A. &amp; Andersen, K. G. The evolution of Ebola virus: Insights from the 2013-2016 epidemic. Nature 538, 193–200, doi:10.1038/nature19790 (2016)." />
<meta name="citation_reference" content="Casadevall, A. &amp; Imperiale, M. J. Risks and benefits of gain-of-function experiments with pathogens of pandemic potential, such as influenza virus: a call for a science-based discussion. MBio 5, e01730–01714, doi:10.1128/mBio.01730-14 (2014)." />
<meta name="citation_reference" content="Chatterjee, S. &amp; Rothenberg, E. Interaction of bacteriophage l with its E. coli receptor, LamB. Viruses 4, 3162–3178, doi:10.3390/v4113162 (2012)." />
<meta name="citation_reference" content="Wang, H. H. et al. Programming cells by multiplex genome engineering and accelerated evolution. Nature 460, 894–898, doi:10.1038/nature08187 (2009)." />
<meta name="citation_reference" content="Suzek, B. E., Wang, Y., Huang, H., McGarvey, P. B. &amp; Wu, C. H. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches. Bioinformatics (Oxford, England) 31, 926–932, doi:10.1093/bioinformatics/btu739 (2015)." />
<meta name="citation_reference" content="Porcek, N. B. &amp; Parent, K. N. Key residues of S. flexneri OmpA mediate infection by bacteriophage Sf6. Journal of molecular biology 427, 1964–1976, doi:10.1016/j.jmb.2015.03.012 (2015)." />
<meta name="citation_reference" content="Arpino, J. A., Reddington, S. C., Halliwell, L. M., Rizkallah, P. J. &amp; Jones, D. D. Random single amino acid deletion sampling unveils structural tolerance and the benefits of helical registry shift on GFP folding and structure. Structure (London, England: 1993) 22, 889–898, doi:10.1016/j.str.2014.03.014 (2014)." />
<meta name="citation_reference" content="Weitz, J. S., Hartman, H. &amp; Levin, S. A. Coevolutionary arms races between bacteria and bacteriophage. Proceedings of the National Academy of Sciences of the United States of America 102, 9535–9540, doi:10.1073/pnas.0504062102 (2005)." />
<meta name="citation_reference" content="Meyer, J. R. et al. Ecological speciation of bacteriophage lambda in allopatry and sympatry. Science 354, 1301–1304, doi:10.1126/science.aai8446 (2016)." />
<meta name="citation_reference" content="Maddamsetti, R. et al. Core Genes Evolve Rapidly in the Long-term Evolution Experiment with Escherichia coli. Genome biology and evolution, doi:10.1093/gbe/evx064 (2017)." />
<meta name="citation_reference" content="Wichman, H. A., Scott, L. A., Yarber, C. D. &amp; Bull, J. J. Experimental evolution recapitulates natural evolution. Philosophical transactions of the Royal Society of London. Series B, Biological sciences 355, 1677–1684, doi:10.1098/rstb.2000.0731 (2000)." />
<meta name="citation_reference" content="Bloom, J. D. Identification of positive selection in genes is greatly improved by using experimentally informed site-specific models. Biology direct 12, 1, doi:10.1186/s13062-016-0172-z (2017)." />
<meta name="citation_reference" content="Gire, S. K. et al. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science 345, 1369–1372, doi:10.1126/science.1259657 (2014)." />
<meta name="citation_reference" content="Grubaugh, N. D. et al. Genomic epidemiology reveals multiple introductions of Zika virus into the United States. Nature 546, 401–405, doi:10.1038/nature22400 (2017)." />
<meta name="citation_reference" content="Wang, H. H. &amp; Church, G. M. Multiplexed genome engineering and genotyping methods applications for synthetic biology and metabolic engineering. Methods in enzymology 498, 409–426, doi:10.1016/b978-0-12-385120-8.00018-8 (2011)." />
<meta name="citation_reference" content="Datsenko, K. A. &amp; Wanner, B. L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences of the United States of America 97, 6640–6645, doi:10.1073/pnas.120163297 (2000)." />
<meta name="citation_reference" content="Baba, T. et al. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular systems biology 2, 2006.0008, doi:10.1038/msb4100050 (2006)." />
<meta name="citation_reference" content="Hopf, T. A. et al. Mutation effects predicted from sequence co-variation. Nature biotechnology 35, 128–135, doi:10.1038/nbt.3769 (2017)." />
<meta name="citation_reference" content="Eddy, S. R. Accelerated Profile HMM Searches. PLoS computational biology 7, e1002195, doi:10.1371/journal.pcbi.1002195 (2011)." />
<meta name="citation_reference" content="Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics (Oxford, England) 30, 1312–1313, doi:10.1093/bioinformatics/btu033 (2014)." />
<meta name="citation_reference" content="Kosakovsky Pond, S. L., Posada, D., Gravenor, M. B., Woelk, C. H. &amp; Frost, S. D. Automated phylogenetic detection of recombination using a genetic algorithm. Molecular biology and evolution 23, 1891–1901, doi:10.1093/molbev/msl051 (2006)." />
<meta name="citation_reference" content="Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2–approximately maximum-likelihood trees for large alignments. PloS one 5, e9490, doi:10.1371/journal.pone.0009490 (2010)." />
<meta name="citation_reference" content="Le, S. Q., &amp; Gascuel, O. An improved general amino acid replacement matrix. Molecular biology and evolution 25, 1307–1320, doi:10.1093/molbev/msn067 (2008)." />
<meta name="citation_reference" content="Spielman, S. J., &amp; Pond, S. L. K. Relative evolutionary rate inference in HyPhy with LEISR. bioRxiv (2017)." />
<meta name="citation_reference" content="Pupko, T., Bell, R. E., Mayrose, I., Glaser, F. &amp; Ben-Tal, N. Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues. Bioinformatics (Oxford, England) 18 Suppl 1, S71–77 (2002)." />
<meta name="citation_reference" content="Huerta-Cepas, J., Serra, F. &amp; Bork, P. ETE 3: Reconstruction, Analysis, and Visualization of Phylogenomic Data. Molecular biology and evolution 33, 1635–1638, doi:10.1093/molbev/msw046 (2016)." />
<meta name="twitter:title" content="Viral gain-of-function experiments uncover residues under diversifying selection in nature" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology." />
<meta name="og-title" property="og:title" content="Viral gain-of-function experiments uncover residues under diversifying selection in nature" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/242495v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-01-03" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/242495v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/242495v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/242495v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/242495v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/74182" />
    <title>Viral gain-of-function experiments uncover residues under diversifying selection in nature | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ODKfs5Pr-SKpJcZW9YYsW1c9a4LaXkneeWAAdUrA_64","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"74182","apath":"\/biorxiv\/early\/2018\/01\/03\/242495.atom","pisa":"biorxiv;242495v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;242495v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/242495v1","title":"Viral gain-of-function experiments uncover residues under diversifying selection in nature","identifier":"node\/74182"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022byTf++uMnanDY5ljHdM3dpuc10WUaRXmYJm7p65Jfq35LZfQ2JQQNCLOMmG\\\\\\\/65tB0+e0CuhJu7bhf48As41ZPj+CdRkRo+Tnd8cya1MF27orZI7TLxTvDthcSWiVtjD3pSB2Xu4Fjya7GcHAXBszNtH1bqdUifDL4lTaxx\\\\\\\/6\\\\\\\/GP8hSdOeeJGjbTtmXD3wnjxRojOg2ywJiobyIuR9JnqadBu0GkStew6tMaZOfABhXsQx\\\\\\\/sTefEkWK06xpNynXWd1saCKLaL37j0eOZHI8Td+\\\\\\\/SLLhDRE5DN\\\\\\\/xnttVYAUCQ4yWsnZPKnqFDePnZfQf4m8k39y3AGnYcuPxj9s0B6kEFCLyII8abSwUKP\\\\\\\/MOtQy+dDFX0ZU+GnLPiJ2QoVuGDDOR8KEV5pLcjvO+pL5Yf1fOos0MPsOIhbYQSYR8YHRPPRyOJYF1ti2ZavKZf0YoHiKXMyp9OuTJyEsRSg5zQna5xp3imdSill2vKTXZDyMHzsOIScKzp1UDSQH6Meof13dZZ5FGjIJIJ79JufSZQG8FhcMfkZ6YOzeujj94H5oCR+rnBtCBS6d0BlPuFx6Nx8RHO3E\\\\\\\/SImHUXGItwc1wUeX0+0UihZzy0ECQ9LHIniV2qOwJpXtcbBPqRvbrf5vWhx1wffSYFo2FAU2IZGHv94wItQrK2cvXafXZb+EaeWyHLzCHSmHEwjy9aGtNKGYKhit5zMNrKhxvL7v5CNxIaXfHbZdFax\\\\\\\/5NOqM8kMmIPTlpq70v2uUzLOL\\\\\\\/bcFtB+Dss0MQeKrtOwZNC1si5XHaZO2ADqVRYLarB6Fr3tVqsnRmGPRrhhYhWLovt5ygEsvFjuaY0W827Sfj06ouUoycyRhWa8nippKGRFHHIpC7P5sh+1uVNf1Qu84Gql7O6wHiPTmJe\\\\\\\/lC9pRRlLBmua7labSh3vlz+dpBJ7D+ihDP10dz78FLTLenc11tteayrBsuJXvHb4P43KVHIFij0bLGUjYC\\\\\\\/xFfXxTghKibDY+J+SNZgKHVVWr9nFlXDceKuEHVBKR+oiDyNlt\\\\\\\/ws2Ygb1q9KMUnBafhm0LnI2pbVdFRLSxKTVWspM8WnvvgNi6Uiuw\\\\\\\/jJKjUfi0+wbV57gmvue88wZWAE9oVQtjy\\\\\\\/zOz36hP91pxFSfcxAtipQcDQNBo2iv3QiTcUmVWILXCxgLkciIBCtbfYKg3K85e+r8VpVK8nKaCkXvixVUk\\\\\\\/iH9XXCglokX1er+fmPBkxZMx8+PbRTisrLxJY8fZDTPcqg+PqXXlOsa0f3i5s6IUyIzdFrPBnI8pyddGpo6I3lq+L0UdM5xye2W7e0HHGJBdJB3Nm8k=\\\u0022,\\\u0022iv\\\u0022:\\\u0022OB8YjWPLYo\\\\\\\/fDMZCQv+wOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228f84087d2aa50e3fd537d757fa259f6cb3e33ee65cf4f3b5d3c586ef87dde965\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/242495v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-74182 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/242495v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rxPiJODuy24r8ftozsURMuwkIoWp-hOGoJ_r1KT4jvw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74182" id="node74182" data-pisa="biorxiv;242495v1" data-pisa-master="biorxiv;242495" data-apath="/biorxiv/early/2018/01/03/242495.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Viral gain-of-function experiments uncover residues under diversifying selection in nature</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rohan</span> <span class="nlm-surname">Maddamsetti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel T.</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Stephanie J.</span> <span class="nlm-surname">Spielman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Katherine L.</span> <span class="nlm-surname">Petrie</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Debora S.</span> <span class="nlm-surname">Marks</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Justin R.</span> <span class="nlm-surname">Meyer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/242495 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node74182" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rohan Maddamsetti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Systems Biology, Harvard Medical School</span>, Boston, MA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BMaddamsetti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maddamsetti%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARohan%2BMaddamsetti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel T. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Biological Sciences, University of California San Diego</span>, La Jolla, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BT.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BT.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephanie J. Spielman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Institute for Genomics and Evolutionary Medicine, Temple University</span>, Philadelphia, PA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BJ.%2BSpielman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spielman%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BJ.%2BSpielman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katherine L. Petrie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Biological Sciences, University of California San Diego</span>, La Jolla, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Earth-Life Science Institute, Tokyo Institute of Technology</span>, <span class='nlm-country'>Japan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BL.%2BPetrie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrie%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BL.%2BPetrie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Debora S. Marks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Systems Biology, Harvard Medical School</span>, Boston, MA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debora%2BS.%2BMarks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marks%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADebora%2BS.%2BMarks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin R. Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Biological Sciences, University of California San Diego</span>, La Jolla, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BR.%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BR.%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/242495v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:74182" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:74182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/242495v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:74182" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:74182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/242495v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:74182" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:74182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/242495v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:74182" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:74182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/242495v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:74182" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:74182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology.</p></div><p id="p-4">Many viruses can expand their host range with a few mutations<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup> that enable the exploitation of new cellular receptors<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. Such mutations may be the first steps toward an epidemic outbreak; this observation has driven an expansion of theoretical<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>, experimental and surveillance studies of host-range shifts in emergent pathogens, including avian influenza<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, coronaviruses<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>, HIV<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup> and ebolavirus<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Ideally, laboratory evolution experiments could be used to accelerate our understanding of how viruses expand their host range. However, it is not clear whether viral evolution observed under the chemical and physical environments of a laboratory can faithfully inform us about viral evolution in nature, for at least two reasons. First, evolutionary trajectories might be sensitive to differences in environmental conditions between the laboratory and nature. Second, the number of evolutionary paths sampled in laboratory experiments is very small compared to natural virus diversity due to the enormous size of viral populations. Indeed, some researchers have called for the suspension of virus gain-of-function laboratory experiments<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> on the grounds that they would tell us little about real-world viral evolution at the risk of constructing viral strains that are pandemic to new hosts. Here, we use a harmless virus, bacteriophage λ, to demonstrate how gain-of-function experiments can identify mutations that mimic those that occur in nature: we find that two amino acid residues that are critical for gain of function in the laboratory recurrently evolve in nature.</p><p id="p-5">Typical laboratory strains of λ infect <em>Escherichia coli</em> by binding to the outer membrane protein LamB<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, but the phage rapidly evolves in the laboratory to exploit a different membrane protein, OmpF<sup>3</sup>. Since OmpF is not the usual <em>E. coli</em> receptor for λ, these experiments are a proxy for the ability of the phage to switch hosts. The evolved gain-of-function phenotype in λ, OmpF<sup>+</sup>, involves multiple non-synonymous mutations in the host-recognition gene <em>J</em>. Each OmpF<sup>+</sup> isolate had between 4 and 10 single nucleotide substitutions in <em>J</em>, and none had insertions or deletions (indels). 97% of the substitutions in 24 independently evolved OmpF<sup>+</sup> λ phage occurred in the <em>J</em> protein<sup>3</sup> between residues 960 —1132, which we call the ‘specificity region’ of <em>J</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Distribution of J mutations evolved en route to OmpF+ and B) synthetic phage genotypes capable of using OmpF. A) Mutations summed across 24 genomes independently evolved to exploit OmpF in Meyer et al. (2012). Protein domains as annotated in the Pfam database are shown in gray. The majority of mutations either occur in the DUF3672 domain, or in the C-terminus past the annotated boundary of DUF3672. B) Synthetic OmpF+ genotypes indicated by colored cells (red marks the canonical three) observed after combinatorial engineering of 10 common mutations or 19 mutations when the canonical three were fixed. Amino acid and nucleotide changes are indicated by positions bookended by the wild-type state and then the evolved state. Amino acids in position 997 and 1107 have multiple derived states." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-917951937" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) Distribution of J mutations evolved &lt;em&gt;en route&lt;/em&gt; to OmpF&lt;sup&gt;+&lt;/sup&gt; and B) synthetic phage genotypes capable of using OmpF. A) Mutations summed across 24 genomes independently evolved to exploit OmpF in Meyer &lt;em&gt;et al&lt;/em&gt;. (2012). Protein domains as annotated in the Pfam database are shown in gray. The majority of mutations either occur in the DUF3672 domain, or in the C-terminus past the annotated boundary of DUF3672. B) Synthetic OmpF&lt;sup&gt;+&lt;/sup&gt; genotypes indicated by colored cells (red marks the canonical three) observed after combinatorial engineering of 10 common mutations or 19 mutations when the canonical three were fixed. Amino acid and nucleotide changes are indicated by positions bookended by the wild-type state and then the evolved state. Amino acids in position 997 and 1107 have multiple derived states.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F1.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F1.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">A) Distribution of J mutations evolved <em>en route</em> to OmpF<sup>+</sup> and B) synthetic phage genotypes capable of using OmpF. A) Mutations summed across 24 genomes independently evolved to exploit OmpF in Meyer <em>et al</em>. (2012). Protein domains as annotated in the Pfam database are shown in gray. The majority of mutations either occur in the DUF3672 domain, or in the C-terminus past the annotated boundary of DUF3672. B) Synthetic OmpF<sup>+</sup> genotypes indicated by colored cells (red marks the canonical three) observed after combinatorial engineering of 10 common mutations or 19 mutations when the canonical three were fixed. Amino acid and nucleotide changes are indicated by positions bookended by the wild-type state and then the evolved state. Amino acids in position 997 and 1107 have multiple derived states.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">By comparing <em>J</em> among OmpF<sup>+</sup> and OmpF<sup>−</sup> λ, Meyer <em>et al</em>. (2012) suggested that the OmpF<sup>+</sup> innovation required four mutations: one at residue 1012, two in the codon for residue 1107, and a fourth mutation somewhere between residues 990 to 1000<sup>3</sup>. In this study, we directly tested this hypothesis by using Multiplexed Automated Genome Engineering (MAGE)<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> to construct two combinatorial genetic libraries of the <em>J</em> mutations that evolved in Meyer <em>et al</em>. (2012). We constructed the first library by focusing on 10 commonly evolved <em>J</em> mutations<sup><a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a></sup>, from which we sequenced 33 OmpF<sup>+</sup> isolates. All strains possessed the target mutation at residue 1012 and both target mutations at residue 1107; by contrast, some strains lacked a fourth mutation between residues 990 and 1000 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). We call the mutations at residues 1012 and 1107 the “canonical mutations”.</p><p id="p-8">To test whether the three canonical mutations were sufficient to confer the OmpF<sup>+</sup> phenotype, we constructed a synthetic phage with just these mutations. Even though the synthetic phage was viable, it proved unable to exploit cells without the ancestral LamB receptor, demonstrating that at least four <em>J</em> mutations were necessary for laboratory-evolved OmpF<sup>+</sup> phage. To find what further mutations might be needed to confer the OmpF<sup>+</sup> phenotype, we constructed a second MAGE library using the phage with the three canonical <em>J</em> mutations as the baseline, and random combinations of 19 other <em>J</em> mutations found in the OmpF<sup>+</sup> λ evolved by Meyer <em>et al</em>. (2012). Screening this second library produced 88 OmpF<sup>+</sup> isolates. One OmpF<sup>+</sup> isolate had just a single extra mutation in amino acid position 1083 in addition to the three canonical mutations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). However, the majority of the OmpF<sup>+</sup> genotypes did not possess this specific mutation, signifying that its function could be substituted by other <em>J</em> mutations. In total, this experiment revealed that four mutations are sufficient to evolve the innovation, but only two specific amino acid changes (at residues 1012 and 1107) are universally required to access OmpF in the context of laboratory experiments.</p><p id="p-9">We then asked whether natural J sequences reflected the host-shift evolutionary dynamics seen in our gain-of-function experiments. We collected and aligned full-length homologous J protein sequences from UniRef100<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup> (1,207 highly similar sequences). Most sequences were prophage uncovered in the genomes of their <em>Enterobacteriaceae</em> hosts, including bacterial genera <em>Escherichia, Salmonella, Citrobacter, Edwardsiella</em>, and even <em>Cronobacter</em>.</p><p id="p-10">Recall that 97% of substitutions in OmpF<sup>+</sup> gain-of-function experiments occur in the specificity region. Likewise, the natural J homologs had disproportionate variation here: 30% of the total amino acid variation occurred in the specificity region, despite only being 15% of the total length of J (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). This nonrandom clustering of variation in the specificity region strongly suggests that J has experienced substantial diversifying selection on host attachment. Furthermore, the 17 residues engineered into the MAGE libraries were significantly more variable than randomly chosen groups of 17 residues from J (non-parametric bootstrap: <em>p</em> &lt; 10<sup>−5</sup>) and from the specificity region 960-1132 (non-parametric bootstrap: <em>p</em> = 0.0054) showing the experiments had identified evolutionary hotspots. However, the specific 17 amino acid substitutions evolved in the laboratory were not common in natural homologs suggesting that the selection experiments had the resolution to predict where changes would evolve, but not the exact change (Supplementary <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Sequence entropy averaged over a 10-residue window in J. Dashed red lines indicate residues 1012 and 1107. B) Gap entropy over the J protein. Dashed red lines indicate residues 1012 and 1107. C) Phylogeny for J protein. Only a subset of the operational taxonomic units (OTUs) are displayed: just OTUs with sequences &#x2265; 5% divergence from each other. The wild-type J protein sequence is labeled in tan. Clades with indels at residue 1012 are labeled in lavender. Clades with indels at residue 1107 are labeled in light green. Sequences with indels at both positions are labeled in light steel gray. D) Phylogeny for residues 960&#x2013;1132 of J protein. OTUs have at least 5% divergence from each other, and are labeled as in (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-917951937" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A) Sequence entropy averaged over a 10-residue window in J. Dashed red lines indicate residues 1012 and 1107. B) Gap entropy over the J protein. Dashed red lines indicate residues 1012 and 1107. C) Phylogeny for J protein. Only a subset of the operational taxonomic units (OTUs) are displayed: just OTUs with sequences &#x2265; 5% divergence from each other. The wild-type J protein sequence is labeled in tan. Clades with indels at residue 1012 are labeled in lavender. Clades with indels at residue 1107 are labeled in light green. Sequences with indels at both positions are labeled in light steel gray. D) Phylogeny for residues 960&#x2013;1132 of J protein. OTUs have at least 5% divergence from each other, and are labeled as in (C).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F2.medium.gif" width="406" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F2.medium.gif" width="406" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/03/242495/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">A) Sequence entropy averaged over a 10-residue window in J. Dashed red lines indicate residues 1012 and 1107. B) Gap entropy over the J protein. Dashed red lines indicate residues 1012 and 1107. C) Phylogeny for J protein. Only a subset of the operational taxonomic units (OTUs) are displayed: just OTUs with sequences ≥ 5% divergence from each other. The wild-type J protein sequence is labeled in tan. Clades with indels at residue 1012 are labeled in lavender. Clades with indels at residue 1107 are labeled in light green. Sequences with indels at both positions are labeled in light steel gray. D) Phylogeny for residues 960–1132 of J protein. OTUs have at least 5% divergence from each other, and are labeled as in (C).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Focusing in on the two residues critical for the OmpF<sup>+</sup> gain of function, 1012 and 1107, we find that they alone are more variable than random pairs of sites in J (non-parametric bootstrap: <em>p</em> = 0.00101) and the specificity region (non-parametric bootstrap: <em>p</em> = 0.036). But what is more remarkable is the observation that in-frame indels are common in the regions around sites 1012 and 1107 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>; Supplementary Data 1). We measured indel variation over J homologs using gap entropy: a two-state entropy counting each position in each aligned sequence as a gap or non-gap. Residues 1010–1013 and 1107–1109 have the highest gap entropy in J, ranging from 0.799 to 0.893 out of a maximum of 1.000 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>; Supplementary Data 2). For comparison, the typical amino acid site in the J alignment had no indels (mode and median gap entropy = 0). That our experiments would correctly identify both regions with the most natural indel variation by chance is extremely unlikely (given the size of each region and the protein length <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/03/242495/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/03/242495/embed/inline-graphic-1.gif"/></noscript></span></span>. Indels can have large beneficial effects on proteins, including altering specificity by changing surface loops<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>, or causing structural rearrangements that improve function<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>.</p><p id="p-13">To analyze J protein evolutionary dynamics in more depth, we built maximum-likelihood phylogenetic trees for J (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>) and its specificity region (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). The trees are very different (normalized Robinson-Foulds distance = 0.94 out of 1.00), indicating that recombination has caused the history of the specificity region to differ from the history of the rest of the protein. This means that the specificity region is an evolutionary module that commonly recombines and circulates as a unit in the phage population. This observation supports the notion that this region has elevated rates of evolution and diversification.</p><p id="p-14">Using the second phylogeny, we calculated site-specific evolutionary rates for the specificity region. The 17 residues studied in the MAGE experiments evolve faster than equally-sized random samples taken from the specificity region (non-parametric bootstrap, <em>p</em> = 0.0067) and residues 1012 and 1107 evolve faster than random pairs of sites sampled from the specificity region (non-parametric bootstrap, <em>p</em> = 0.0073).</p><p id="p-15">Together, the accelerated rate of change and increased entropy at residues 1012 and 1107 suggest that these sites have experienced strong diversifying selection. Our combinatorial genetic studies showed that these particular residues determine receptor tropism, so it is reasonable to conclude that their unique evolution is due to host-range evolution. Mathematical theory has demonstrated that receptor-use evolution should lead to diversification since accessing new host types alters the phages’ ecological niches<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. Indeed, diversification was observed in a related experiment where λ evolved to exploit two host cell types with different receptors<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. To test whether indels at 1012 and 1107 cause diversification, we compared the branches where indels occur versus all others. If the indels cause ecological differentiation, then they should reduce intraspecific competition for hosts and facilitate the long-term maintenance of distinct evolutionary lineages. Indeed, the branches on which indels occur are significantly longer than the other branches of the specificity-region phylogeny (Kruskal-Wallis test, <em>p</em> &lt; 10<sup>−8</sup>; visualized by the discrete indel clades colored in lavender and light green in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>).</p><p id="p-16">The congruence we find between laboratory and natural evolution in λ contrasts with work showing that beneficial mutations in Richard Lenski’s long-term experiment anticorrelate<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> with natural protein variation in <em>E. coli</em>. By design, Lenski’s experiment lacks the complex and variable selection pressures found in <em>E. coli’s</em> natural gut environment, leading the bacteria down evolutionary paths not taken in nature. By contrast, the dominant selection pressure in evolution experiments with bacteriophage— attachment to bacterial host cells— is probably also a dominant selection pressure on phage in the wild. Notably, another study on experimental evolution of bacteriophage has observed residues under positive selection in both the lab and nature<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>.</p><p id="p-17">Together, these studies show that selection experiments on viruses cultured in the laboratory can inform evolution in nature. Based on the patterns of J sequence variation observed, we suggest that host-range evolution is common in this group of viruses, and perhaps others too. While the frequency of host-range evolution may be unsettling, our work also demonstrates potential methods to combat host shifts. In particular, worrisome mutations can be identified with functional genetic experiments as described here or with other laboratory techniques such as deep mutational scanning<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. This information can be combined with genomic surveillance efforts underway<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> to devise more effective disease management strategies to eradicate problematic strains.</p><div class="section" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>MAGE <em>experiments</em></h3><p id="p-18">We modified λ strain cI857 (provided by Ing-Nang Wang, SUNY Albany) integrated into the genome of HWEC106 (provided by Harris Wang, Columbia University). We engineered the J substitutions into λ by using MAGE<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> on strain HWEC106. MAGE uses the λ-red recombineering system provided on the pKD46 plasmid<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. For a description of the oligos used, see Supplementary <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a>.</p><p id="p-19">For the first, 10-mutation library, we ran 18 rounds of MAGE and then screened for OmpF<sup>+</sup> isolates. For the second library, we edited λ with oligos ‘a3034g’ and ‘g3319a t3321a’ to introduce the canonical three mutations. Next, we performed a number of different MAGE trials in order to maximize the number of unique alleles we observed. See Supplementary <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S2</a> for our MAGE strategy.</p><p id="p-20">To screen the edited phage for OmpF<sup>+</sup> genotypes, we induced the MAGE lysogen libraries and plated the phage on lawns of <em>lamB<sup>−</sup> E. coli</em> strain JW3996 from the KEIO collection<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Plaques were picked from the lawns and the C-terminus of the J gene was Sanger sequenced at the Genewiz La Jolla, CA facility. Unpurified PCR products (Forward primer: 5’ CGCATCGTTCACCTCTCACT; Reverse primer: 5’ CCTGCGGGCGGTTTGTCATTT) were submitted.</p></div><div id="sec-3" class="subsection"><h3>Analysis of natural <em>J</em> variation</h3><p id="p-21">We used the <em>evcouplings</em> pipeline<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> to generate a jackhmmer<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> alignment of 1,207 full-length J homologs (parameter settings: bitscore = 0.2; theta = 0.999; seqid_filter = 95 and 99 (thresholds for filtering highly similar sequences); minimum_sequence_coverage = 99 (require sequences to align to 99% of wild-type J protein); and minimum_sequence_coverage = 50. When analyzing the specificity region (say, constructing a phylogeny), we used residues 960-1132 of this full-length alignment.</p><p id="p-22">We used RAxML<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> to generate maximum-likelihood phylogenies using automatically selected amino acid substitution matrices (PROTGAMMAAUTO) based on model fit. To account for recombination when estimating site-specific evolutionary rates in the specificity region, we first ran a modified SBP algorithm<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> on the specificity region. We found a putative recombination breakpoint at residue 1000, supported by Akaike’s information criterion but not the Bayesian information criterion. We therefore partitioned the specificity region at residue 1000 and re-calculated trees for each partition using FastTree<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> with an LG substitution matrix<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. We next calculated site-specific evolutionary rates with LEISR<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>, a scalable implementation of Rate4Site<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> that accounts for recombination breakpoints, using an LG substitution matrix. We calculated the Robinson-Foulds distance between our maximum-likelihood phylogenies for J and its specificity region using <em>ete3-compare<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup></em>. We divided the Robinson-Foulds distance (2276.00) by its maximum possible value (2412.00) to get the normalized distance (0.94).</p><p id="p-23">We calculated gap entropy at each position as: − [<em>p · log</em><sub>2</sub> (<em>p</em>) + (1 − <em>p</em>) · <em>log</em><sub>2</sub>(1<em>−p</em>)], where <em>p</em> is the frequency of gap characters at that position, and 1 -<em>p</em> is the frequency of all amino acids at that position.</p><p id="p-24">For all non-parametric bootstrap calculations, we used 100,000 bootstraps, and chose groups of sites without replacement. For example, when testing for high entropy in the 17 MAGE residues, we generated an empirical null distribution by randomly choosing 17 different residues for one sample, and resampling 100,000 times. We used the same procedure for our evolutionary-rate tests.</p></div><div id="sec-4" class="subsection"><h3>Data Availability</h3><p id="p-25">All data and analysis code for this project are available at the Dryad digital repository [doi: pending publication]</p></div></div><div class="section" id="sec-5"><h2 class="">Author contributions</h2><p id="p-27">R.M. designed and conceived the study. D.T.J. conducted MAGE experiments. K.L.P. analyzed initial MAGE data. R.M. and S.S. performed statistical and phylogenetic analyses. R.M., D.S.M., and J.R.M. wrote the paper and everyone edited it.</p></div><div class="section COI-statement" id="sec-6"><h2 class="">Competing Interests</h2><p id="p-28">The authors declare no competing financial interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-26">We thank John Ingraham, Adam Riesselman, Kelly Brock, David Ding, and Alita Burmeister for helpful discussions. K.L.P. is supported by the ELSI Origins Network (EON), which is funded by the John Templeton Foundation. The ideas expressed in this publication are those of the authors and not necessarily those of the funding sources.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> <span class="em-link"><span class="em-addr">jrmeyer{at}ucsd.edu</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.1" data-doi="10.1371/journal.ppat.1004395"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Longdon</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brockhurst</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Welch</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jiggins</span>,  <span class="cit-name-given-names">F. M.</span></span> <span class="cit-article-title">The evolution and genetics of virus host shifts</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e1004395</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1004395</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004395%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004395%26rft_id%253Dinfo%253Apmid%252F25375777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.2" data-doi="10.1038/nature10831"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Imai</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Experimental adaptation of an influenza H5 HA confers respiratory droplet transmission to a reassortant H5 HA/H1N1 virus in ferrets</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">486</span>, <span class="cit-fpage">420</span>–<span class="cit-lpage">428</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature 10831</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DImai%26rft.auinit1%253DM.%26rft.volume%253D486%26rft.issue%253D7403%26rft.spage%253D420%26rft.epage%253D428%26rft.atitle%253DExperimental%2Badaptation%2Bof%2Ban%2Binfluenza%2BH5%2BHA%2Bconfers%2Brespiratory%2Bdroplet%2Btransmission%2Bto%2Ba%2Breassortant%2BH5%2BHA%252FH1N1%2Bvirus%2Bin%2Bferrets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10831%26rft_id%253Dinfo%253Apmid%252F22722205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22722205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305466800047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.3" data-doi="10.1126/science.1214449"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Repeatability and contingency in the evolution of a key innovation in phage lambda</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>, <span class="cit-fpage">428</span>–<span class="cit-lpage">432</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1214449</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMeyer%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D335%26rft.issue%253D6067%26rft.spage%253D428%26rft.epage%253D432%26rft.atitle%253DRepeatability%2Band%2BContingency%2Bin%2Bthe%2BEvolution%2Bof%2Ba%2BKey%2BInnovation%2Bin%2BPhage%2BLambda%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214449%26rft_id%253Dinfo%253Apmid%252F22282803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2Ny80MjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8wMy8yNDI0OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.4" data-doi="10.1038/nature02104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Antia</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regoes</span>,  <span class="cit-name-given-names">R. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koella</span>,  <span class="cit-name-given-names">J. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>,  <span class="cit-name-given-names">C. T.</span></span> <span class="cit-article-title">The role of evolution in the emergence of infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">658</span>–<span class="cit-lpage">661</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature02104</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAntia%26rft.auinit1%253DR.%26rft.volume%253D426%26rft.issue%253D6967%26rft.spage%253D658%26rft.epage%253D661%26rft.atitle%253DThe%2Brole%2Bof%2Bevolution%2Bin%2Bthe%2Bemergence%2Bof%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02104%26rft_id%253Dinfo%253Apmid%252F14668863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14668863&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187132800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.5" data-doi="10.1038/nrmicro3362"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G. F.</span></span> <span class="cit-article-title">Enabling the ‘host jump’: structural determinants of receptor-binding specificity in influenza A viruses</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Microbiology</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">822</span>–<span class="cit-lpage">831</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro3362</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.volume%253D12%26rft.spage%253D822%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3362%26rft_id%253Dinfo%253Apmid%252F25383601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25383601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-242495v1.6" data-doi="10.1016/j.celrep.2017.07.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Avian-to-Human Receptor-Binding Adaptation by Influenza A Virus Hemagglutinin H4</span>. <abbr class="cit-jnl-abbrev">Cell reports</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.celrep.2017.07.028</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D20%26rft.spage%253D1201%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.07.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.07.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-242495v1.7" data-doi="10.1126/science.1244730"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koel</span>,  <span class="cit-name-given-names">B. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Substitutions near the receptor binding site determine major antigenic change during influenza virus evolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>, <span class="cit-fpage">976</span>–<span class="cit-lpage">979</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1244730</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKoel%26rft.auinit1%253DB.%2BF.%26rft.volume%253D342%26rft.issue%253D6161%26rft.spage%253D976%26rft.epage%253D979%26rft.atitle%253DSubstitutions%2BNear%2Bthe%2BReceptor%2BBinding%2BSite%2BDetermine%2BMajor%2BAntigenic%2BChange%2BDuring%2BInfluenza%2BVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1244730%26rft_id%253Dinfo%253Apmid%252F24264991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE2MS85NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8wMy8yNDI0OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.8" data-doi="10.1016/j.cell.2014.02.040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Linster</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification, characterization, and natural selection of mutations driving airborne transmission of A/H5N1 virus</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>, <span class="cit-fpage">329</span>–<span class="cit-lpage">339</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2014.02.040</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.040%26rft_id%253Dinfo%253Apmid%252F24725402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24725402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.9" data-doi="10.1038/nrmicro.2016.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Wit</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falzarano</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V. J.</span></span> <span class="cit-article-title">SARS and MERS: recent insights into emerging coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Microbiology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">523</span>–<span class="cit-lpage">534</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro.2016.81</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.10" data-doi="10.1038/nature12328"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular basis of binding between novel human coronavirus MERS-CoV and its receptor CD26</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>, <span class="cit-fpage">227</span>–<span class="cit-lpage">231</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature12328</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.11" data-doi="10.1038/nrg1246"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>,  <span class="cit-name-given-names">K. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-article-title">The causes and consequences of HIV evolution</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">52</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg1246</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRambaut%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D61%26rft.atitle%253DThe%2Bcauses%2Band%2Bconsequences%2Bof%2BHIV%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1246%26rft_id%253Dinfo%253Apmid%252F14708016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14708016&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187641100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.12" data-doi="10.1038/nature19790"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudas</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">K. G.</span></span> <span class="cit-article-title">The evolution of Ebola virus: Insights from the 2013-2016 epidemic</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">200</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature19790</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19790%26rft_id%253Dinfo%253Apmid%252F27734858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19790&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27734858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.13" data-doi="10.1128/mBio.01730-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Imperiale</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Risks and benefits of gain-of-function experiments with pathogens of pandemic potential, such as influenza virus: a call for a science-based discussion</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e01730</span>–<span class="cit-lpage">01714</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.01730-14</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De01730%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01730-14%26rft_id%253Dinfo%253Apmid%252F25085113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01730-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25085113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.14" data-doi="10.3390/v4113162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Interaction of bacteriophage l with its E. coli receptor, LamB</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">3162</span>–<span class="cit-lpage">3178</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v4113162</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DChatterjee%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D3162%26rft.epage%253D3178%26rft.atitle%253DInteraction%2Bof%2BBacteriophage%2Bl%2Bwith%2BIts%2BE.%2Bcoli%2BReceptor%252C%2BLamB.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4113162%26rft_id%253Dinfo%253Apmid%252F23202520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4113162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23202520&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311429900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.15" data-doi="10.1038/nature08187"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Programming cells by multiplex genome engineering and accelerated evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">460</span>, <span class="cit-fpage">894</span>–<span class="cit-lpage">898</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature08187</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWang%26rft.auinit1%253DH.%2BH.%26rft.volume%253D460%26rft.issue%253D7257%26rft.spage%253D894%26rft.epage%253D898%26rft.atitle%253DProgramming%2Bcells%2Bby%2Bmultiplex%2Bgenome%2Bengineering%2Band%2Baccelerated%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08187%26rft_id%253Dinfo%253Apmid%252F19633652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19633652&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268938300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.16" data-doi="10.1093/bioinformatics/btu739"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suzek</span>,  <span class="cit-name-given-names">B. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGarvey</span>,  <span class="cit-name-given-names">P. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C. H.</span></span> <span class="cit-article-title">UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches</span>. <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">926</span>–<span class="cit-lpage">932</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btu739</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu739%26rft_id%253Dinfo%253Apmid%252F25398609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.17" data-doi="10.1016/j.jmb.2015.03.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Porcek</span>,  <span class="cit-name-given-names">N. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Parent</span>,  <span class="cit-name-given-names">K. N.</span></span> <span class="cit-article-title">Key residues of S. flexneri OmpA mediate infection by bacteriophage Sf6</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr> <span class="cit-vol">427</span>, <span class="cit-fpage">1964</span>–<span class="cit-lpage">1976</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2015.03.012</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.volume%253D427%26rft.spage%253D1964%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.03.012%26rft_id%253Dinfo%253Apmid%252F25816773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25816773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.18" data-doi="10.1016/j.str.2014.03.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arpino</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reddington</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halliwell</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rizkallah</span>,  <span class="cit-name-given-names">P. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">D. D.</span></span> <span class="cit-article-title">Random single amino acid deletion sampling unveils structural tolerance and the benefits of helical registry shift on GFP folding and structure</span>. <abbr class="cit-jnl-abbrev">Structure (London, England: 1993)</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">889</span>–<span class="cit-lpage">898</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.str.2014.03.014</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%252C%2BEngland%253A%2B1993%2529%26rft.volume%253D22%26rft.spage%253D889%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2014.03.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2014.03.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.19" data-doi="10.1073/pnas.0504062102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weitz</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartman</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Levin</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">Coevolutionary arms races between bacteria and bacteriophage</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">9535</span>–<span class="cit-lpage">9540</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0504062102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504062102%26rft_id%253Dinfo%253Apmid%252F15976021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzI3Lzk1MzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8wMy8yNDI0OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.20" data-doi="10.1126/science.aai8446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ecological speciation of bacteriophage lambda in allopatry and sympatry</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>, <span class="cit-fpage">1301</span>–<span class="cit-lpage">1304</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aai8446</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aai8446%26rft_id%253Dinfo%253Apmid%252F27884940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTQvNjMxNy8xMzAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDEvMDMvMjQyNDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.21" data-doi="10.1093/gbe/evx064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddamsetti</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Core Genes Evolve Rapidly in the Long-term Evolution Experiment with Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Genome biology and evolution</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/gbe/evx064</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.22" data-doi="10.1098/rstb.2000.0731"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarber</span>,  <span class="cit-name-given-names">C. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bull</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Experimental evolution recapitulates natural evolution</span>. <abbr class="cit-jnl-abbrev">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</abbr> <span class="cit-vol">355</span>, <span class="cit-fpage">1677</span>–<span class="cit-lpage">1684</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rstb.2000.0731</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DWichman%26rft.auinit1%253DH.%2BA.%26rft.volume%253D355%26rft.issue%253D1403%26rft.spage%253D1677%26rft.epage%253D1684%26rft.atitle%253DExperimental%2Bevolution%2Brecapitulates%2Bnatural%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2000.0731%26rft_id%253Dinfo%253Apmid%252F11127915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2000.0731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11127915&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166088400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.23" data-doi="10.1186/s13062-016-0172-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bloom</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Identification of positive selection in genes is greatly improved by using experimentally informed site-specific models</span>. <abbr class="cit-jnl-abbrev">Biology direct</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">1</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13062-016-0172-z</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2Bdirect%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13062-016-0172-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13062-016-0172-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.24" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>,  <span class="cit-name-given-names">S. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>, <span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1259657</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDEvMDMvMjQyNDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.25" data-doi="10.1038/nature22400"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grubaugh</span>,  <span class="cit-name-given-names">N. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic epidemiology reveals multiple introductions of Zika virus into the United States</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>, <span class="cit-fpage">401</span>–<span class="cit-lpage">405</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature22400</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22400%26rft_id%253Dinfo%253Apmid%252F28538723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22400&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28538723&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.26" data-doi="10.1016/B978-0-12-385120-8.00018-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-article-title">Multiplexed genome engineering and genotyping methods applications for synthetic biology and metabolic engineering</span>. <abbr class="cit-jnl-abbrev">Methods in enzymology</abbr> <span class="cit-vol">498</span>, <span class="cit-fpage">409</span>–<span class="cit-lpage">426</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/b978-0-12-385120-8.00018-8</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DWang%26rft.auinit1%253DH.%2BH.%26rft.volume%253D498%26rft.spage%253D409%26rft.epage%253D426%26rft.atitle%253DMultiplexed%2Bgenome%2Bengineering%2Band%2Bgenotyping%2Bmethods%2Bapplications%2Bfor%2Bsynthetic%2Bbiology%2Band%2Bmetabolic%2Bengineering.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385120-8.00018-8%26rft_id%253Dinfo%253Apmid%252F21601688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385120-8.00018-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21601688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292007500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.27" data-doi="10.1073/pnas.120163297"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Datsenko</span>,  <span class="cit-name-given-names">K. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wanner</span>,  <span class="cit-name-given-names">B. L.</span></span> <span class="cit-article-title">One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">97</span>, <span class="cit-fpage">6640</span>–<span class="cit-lpage">6645</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.120163297</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.120163297%26rft_id%253Dinfo%253Apmid%252F10829079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTIvNjY0MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAxLzAzLzI0MjQ5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.28" data-doi="10.1038/msb4100050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baba</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection</span>. <abbr class="cit-jnl-abbrev">Molecular systems biology</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">2006</span>.0008<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/msb4100050</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bsystems%2Bbiology%26rft.volume%253D2%26rft.spage%253D2006%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb4100050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.29" data-doi="10.1038/nbt.3769"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hopf</span>,  <span class="cit-name-given-names">T. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mutation effects predicted from sequence co-variation</span>. <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">128</span>–<span class="cit-lpage">135</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.3769</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.volume%253D35%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3769%26rft_id%253Dinfo%253Apmid%252F28092658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28092658&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.30" data-doi="10.1371/journal.pcbi.1002195"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-article-title">Accelerated Profile HMM Searches</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e1002195</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1002195</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002195%26rft.epage%253De1002195%26rft.atitle%253DAccelerated%2BProfile%2BHMM%2BSearches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft_id%253Dinfo%253Apmid%252F22039361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039361&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.31" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btu033</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.32" data-doi="10.1093/molbev/msl051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gravenor</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woelk</span>,  <span class="cit-name-given-names">C. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Frost</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-article-title">Automated phylogenetic detection of recombination using a genetic algorithm</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1891</span>–<span class="cit-lpage">1901</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msl051</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl051%26rft_id%253Dinfo%253Apmid%252F16818476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16818476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240586600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.33" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">M. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dehal</span>,  <span class="cit-name-given-names">P. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Arkin</span>,  <span class="cit-name-given-names">A. P.</span></span> <span class="cit-article-title">FastTree 2–approximately maximum-likelihood trees for large alignments</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e9490</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0009490</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPrice%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9490%26rft.epage%253De9490%26rft.atitle%253DFastTree%2B2--approximately%2Bmaximum-likelihood%2Btrees%2Bfor%2Blarge%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft_id%253Dinfo%253Apmid%252F20224823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224823&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.34" data-doi="10.1093/molbev/msn067"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le</span>,  <span class="cit-name-given-names">S. Q.</span>,</span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">An improved general amino acid replacement matrix</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1307</span>–<span class="cit-lpage">1320</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msn067</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn067%26rft_id%253Dinfo%253Apmid%252F18367465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18367465&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256979100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spielman</span>,  <span class="cit-name-given-names">S. J.</span>,</span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pond</span>,  <span class="cit-name-given-names">S. L. K.</span></span> <span class="cit-article-title">Relative evolutionary rate inference in HyPhy with LEISR</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pupko</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayrose</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaser</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ben-Tal</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues</span>. <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr> <span class="cit-vol">18 Suppl 1</span>, <span class="cit-fpage">S71</span>–<span class="cit-lpage">77</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft.volume%253D181%26rft.spage%253DS71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-242495v1.37" data-doi="10.1093/molbev/msw046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serra</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bork</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">ETE 3: Reconstruction, Analysis, and Visualization of Phylogenomic Data</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">1635</span>–<span class="cit-lpage">1638</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msw046</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw046%26rft_id%253Dinfo%253Apmid%252F26921390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26921390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F03%2F242495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/242495v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F242495v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/177311v3" title="Ligand-regulated entry into the HRD ERAD pathway—the dark side of allostery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/242370v1" title="NvERTx: A gene expression database to compare Embryogenesis and Regeneration in the sea anemone Nematostella vectensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 03, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/242495v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/242495v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/242495v1" class="active" data-icon-position="" data-hide-link-title="0">Viral gain-of-function experiments uncover residues under diversifying selection in nature</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/74182" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jk_r_K7jv0JFNlKBdNPhVs6YoQQ3dC0h3jf_nxY8VNQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74182" id="node74182--2" data-pisa="biorxiv;242495v1" data-pisa-master="biorxiv;242495" data-seqnum="74182" data-apath="/biorxiv/early/2018/01/03/242495.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Viral gain-of-function experiments uncover residues under diversifying selection in nature</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rohan</span> <span class="nlm-surname">Maddamsetti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel T.</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Stephanie J.</span> <span class="nlm-surname">Spielman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Katherine L.</span> <span class="nlm-surname">Petrie</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Debora S.</span> <span class="nlm-surname">Marks</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Justin R.</span> <span class="nlm-surname">Meyer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">242495; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/242495 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/242495v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;title=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;title=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;text=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;title=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;t=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;title=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;title=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74182" id="node74182--3" data-pisa="biorxiv;242495v1" data-pisa-master="biorxiv;242495" data-seqnum="74182" data-apath="/biorxiv/early/2018/01/03/242495.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Viral gain-of-function experiments uncover residues under diversifying selection in nature</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rohan</span> <span class="nlm-surname">Maddamsetti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel T.</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Stephanie J.</span> <span class="nlm-surname">Spielman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Katherine L.</span> <span class="nlm-surname">Petrie</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Debora S.</span> <span class="nlm-surname">Marks</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Justin R.</span> <span class="nlm-surname">Meyer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">242495; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/242495 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Maddamsetti%2C+Rohan&amp;rft.au=Johnson%2C+Daniel+T.&amp;rft.au=Spielman%2C+Stephanie+J.&amp;rft.au=Petrie%2C+Katherine+L.&amp;rft.au=Marks%2C+Debora+S.&amp;rft.au=Meyer%2C+Justin+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/74182/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/74182/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/74182/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/74182/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/74182/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/74182/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/74182/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/74182/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/74182/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/74182/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/74182/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/74182/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;count=horizontal&amp;via=&amp;text=Viral%20gain-of-function%20experiments%20uncover%20residues%20under%20diversifying%20selection%20in%20nature&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/242495v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/242495v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/242495v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
