<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/15/804716.atom" />
<meta name="HW.pisa" content="biorxiv;804716v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins" />
<meta name="DC.Identifier" content="10.1101/804716" />
<meta name="DC.Date" content="2019-10-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="As part of the type I IFN signaling, the 2’-5’- oligoadenylate synthetase (OAS) proteins have been involved in the progression of several non-viral diseases. Notably, OAS has been correlated with immune-modulatory functions that promote chronic inflammatory conditions, autoimmune disorders, cancer, and infectious diseases. In spite of this, OAS enzymes have been ignored as drug targets, and to date, there are no reports of compounds that can inhibit their activity. In this study, we have used homology modeling and virtual high-throughput screening to identify potential inhibitors of the human proteins OAS1, OAS2, and OAS3. Altogether, we have found 37 molecules that could exert a competitive inhibition in the ATP binding sites of OAS proteins, independently of the activation state of the enzyme. This latter characteristic, which might be crucial for a versatile inhibitor, was observed in compounds interacting with the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3. Although there was little correlation between specific chemical fragments and particular interactions, intermolecular contacts with OAS catalytic triad and other critical amino acids were mainly promoted by heterocycles with π electrons and hydrogen bond acceptors. In conclusion, this study provides a potential set of OAS inhibitors as well as valuable information for their design, development, and optimization." />
<meta name="DC.Contributor" content="Karen J. Gonzalez" />
<meta name="DC.Contributor" content="Diego Moncada-Giraldo" />
<meta name="DC.Contributor" content="Juan B. Gutierrez" />
<meta name="article:published_time" content="2019-10-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;As part of the type I IFN signaling, the 2’-5’- oligoadenylate synthetase (OAS) proteins have been involved in the progression of several non-viral diseases. Notably, OAS has been correlated with immune-modulatory functions that promote chronic inflammatory conditions, autoimmune disorders, cancer, and infectious diseases. In spite of this, OAS enzymes have been ignored as drug targets, and to date, there are no reports of compounds that can inhibit their activity. In this study, we have used homology modeling and virtual high-throughput screening to identify potential inhibitors of the human proteins OAS1, OAS2, and OAS3. Altogether, we have found 37 molecules that could exert a competitive inhibition in the ATP binding sites of OAS proteins, independently of the activation state of the enzyme. This latter characteristic, which might be crucial for a versatile inhibitor, was observed in compounds interacting with the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3. Although there was little correlation between specific chemical fragments and particular interactions, intermolecular contacts with OAS catalytic triad and other critical amino acids were mainly promoted by heterocycles with π electrons and hydrogen bond acceptors. In conclusion, this study provides a potential set of OAS inhibitors as well as valuable information for their design, development, and optimization.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;804716v1" />
<meta name="citation_id" content="804716v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/804716v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/804716v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/804716v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716.full.pdf" />
<meta name="citation_doi" content="10.1101/804716" />
<meta name="citation_num_pages" content="44" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="804716" />
<meta name="citation_author" content="Karen J. Gonzalez" />
<meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia" />
<meta name="citation_author" content="Diego Moncada-Giraldo" />
<meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia" />
<meta name="citation_author" content="Juan B. Gutierrez" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Texas at San Antonio" />
<meta name="citation_author_email" content="juan.gutierrez3@utsa.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0627-5035" />
<meta name="citation_reference" content="R. Medzhitov, “Origin and physiological roles of inflammation,” Nature, vol. 454, no. 7203, pp. 428–435, 2008." />
<meta name="citation_reference" content="H. Kumar, T. Kawai, and S. Akira, “Pathogen recognition by the innate immune system,” Int. Rev. Immunol., vol. 30, no. 1, pp. 16–34, 2011." />
<meta name="citation_reference" content="P. Broz and D. M. Monack, “Newly described pattern recognition receptors team up against intracellular pathogens,” Nat. Rev. Immunol., 2013." />
<meta name="citation_reference" content="R. Medzhitov, “Inflammation 2010: New Adventures of an Old Flame,” Cell, vol. 140, no. 6. Cell Press, pp. 771–776, 19-Mar-2010." />
<meta name="citation_reference" content="G. Hartmann, Nucleic Acid Immunity, 1st ed., vol. 133. Elsevier Inc., 2017." />
<meta name="citation_reference" content="H. H. Gad, P. Despres, R. Hartmann, S. Eskildsen-Larsen, and H. Kristiansen, “The Oligoadenylate Synthetase Family: An Ancient Protein Family with Multiple Antiviral Activities,” J. Interf. Cytokine Res., vol. 31, no. 1, pp. 41–47, 2010." />
<meta name="citation_reference" content="J. Chebath, P. Benech, A. Hovanessian, J. Galabru, and M. Revel, “Four different forms of interferon-induced 2’,5’-oligo(A) synthetase identified by immunoblotting in human cells,” J. Biol. Chem., vol. 262, no. 8, pp. 3852–3857, Mar. 1987." />
<meta name="citation_reference" content="J. Zhu etal., “Suplemental information. Antiviral activity of human OASL protein is mediated by enhancing signaling of the RIG-1 RNA sensor.,” Immunity, vol. 40, no. 6, pp. 936–48, Jun. 2014." />
<meta name="citation_reference" content="J. Justesen, R. Hartmann, and N. O. Kjeldgaard, “Gene structure and function of the 2’-5’-oligoadenylate synthetase family,” Cell. Mol. Life Sci., vol. 57, no. 11, pp. 1593–1612, 2000." />
<meta name="citation_reference" content="A. J. Sadler and B. R. G. Williams, “Interferon-inducible antiviral effectors,” Nature Reviews Immunology, vol. 8, no. 7. NIH Public Access, pp. 559–568, Jul-2008." />
<meta name="citation_reference" content="R. Hartmann, J. Justesen, S. N. Sarkar, G. C. Sen, and V. C. Yee, “Crystal Structure of the 2’-Specific and Double-Stranded RNA-Activated Interferon-Induced Antiviral Protein 2’-5’-Oligoadenylate Synthetase,” Structure, vol. 19, pp. 1173–1185, 2003." />
<meta name="citation_reference" content="Jan Lohofener et al., “The activation mechanism of 2’ −5’-oligoadenylate synthetase gives new insights into OAS / cGAS triggers of innate immunity.,” Cell Press, vol. 23, pp. 851–862, 2015." />
<meta name="citation_reference" content="J. Donovan, M. Dufner, and A. Korennykh, “Structural basis for cytosolic double-stranded RNA surveillance by human oligoadenylate synthetase 1,” Proc. Natl. Acad. Sci., vol. 110, no. 5, pp. 1652–1657, 2013." />
<meta name="citation_reference" content="R. Hartmann, G. Walko, and J. Justesen, “Inhibition of 2’-5’ oligoadenylate synthetase by divalent metal ions,” FEBS Lett., vol. 507, no. 1, pp. 54–58, 2001." />
<meta name="citation_reference" content="S. N. Sarkar, A. Ghosh, H. Wang, S. Sung, and G. C. Sen, “The Nature of the Catalytic Domain of 2-5-Oligoadenylate Synthetases,” vol. 274, no. 36, pp. 25535–25542,1999." />
<meta name="citation_reference" content="M. Drappier and T. Mîchîels, “Inhibition of the OAS/RNase L pathway by viruses,” Curr. Opin. Virol., vol. 15, pp. 19–26, 2015." />
<meta name="citation_reference" content="S. Kôks, K. Kingo, M. Keermann, E. Reimann, E. Prans, and K. Abram, “Transcriptional landscape of psoriasis identifies the involvement of IL36 and IL36RN,” BMC Genomics, vol. 16, no. 1, 2015." />
<meta name="citation_reference" content="C. Sanfilippo etal., “OAS Gene Family Expression Is Associated with HIV-Related Neurocognitive Disorders,” Mol. Neurobiol., vol. 55, no. 3, pp. 1905–1914, 2018." />
<meta name="citation_reference" content="M. Dan, D. Zheng, L. L. Field, and V. Bonnevie-Nielsen, “Induction and activation of antiviral enzyme 213,513-oligoadenylate synthetase by in vitro transcribed insulin mRNA and other cellular RNAs,” Mol. Biol. Rep., vol. 39, no. 7, pp. 7813–7822, Jul. 2012." />
<meta name="citation_reference" content="L. L. Field, V. Bonnevie-Nielsen, F. Pociot, S. Lu, T. B. Nielsen, and H. Beck-Nielsen, “OAS1 splice site polymorphism controlling antiviral enzyme activity influences susceptibility to type 1 diabetes,” Diabetes, vol. 54, no. 5, pp. 1588–1591, 2005." />
<meta name="citation_reference" content="X. Feng etal., “Association of increased interferon-inducible gene expression with disease activity and lupus nephritis in patients with systemic lupus erythematosus,” Arthritis Rheum., vol. 54, no. 9, pp. 2951–2962, 2006." />
<meta name="citation_reference" content="A. A. Dar et al., “Extracellular 213513-oligoadenylate synthetase 2 mediates T-cell receptor Οϋ3-ζ chain down-regulation via caspase-3 activation in oral cancer,” Immunology, vol. 147, no. 2, pp. 251–264, 2016." />
<meta name="citation_reference" content="G. Leisching, I. Wiid, and B. Baker, “OAS1, 2 and 3: Significance During Active Tuberculosis?,” J. Infect. Dis., vol. 217, no. 10, pp. 1517–1521, 2018." />
<meta name="citation_reference" content="D. J. Conway et al., “Integrated pathogen load and dual transcriptome analysis of systemic host-pathogen interactions in severe malaria,” Sci. Transl. Med., vol. 10, no. 447, p. eaar3619, 2018." />
<meta name="citation_reference" content="C. Yu et al., “Prediction of key genes and pathways involved in trastuzumab-resistant gastric cancer,” World J. Surg. Oncol., vol. 16, no. 1, 2018." />
<meta name="citation_reference" content="A. M. Murphy, P. Mukherjee, M. Moerdyk-Schauwecker, V. Z. Grdzelishvili, N. R. Shah, and E. Hastie, “Resistance of pancreatic cancer cells to oncolytic vesicular stomatitis virus: Role of type I interferon signaling,” Virology, vol. 436, no. 1, pp. 221–234, 2012." />
<meta name="citation_reference" content="J. Tang et al., “Increased expression of the type I interferon-inducible gene, lymphocyte antigen 6 complex locus E, in peripheral blood cells is predictive of lupus activity in a large cohort of Chinese lupus patients,” Lupus, vol. 17, no. 9, pp. 805–813, 2008." />
<meta name="citation_reference" content="L. G. M. van Baarsen et al., “A subtype of multiple sclerosis defined by an activated immune defense program,” Genes Immun., vol. 7, no. 6, pp. 522–531, 2006." />
<meta name="citation_reference" content="A. C. M. Guedes et al., “ Differential upregulation of human 2BI5BI OAS genes on systemic sclerosis: Detection of increased basal levels of OASL and OAS 2 genes through a qPCR based assay,” Autoimmunity, vol. 47, no. 2, pp. 119–126, 2013." />
<meta name="citation_reference" content="H. Berman, K. Henrick, and H. Nakamura, “Announcing the worldwide Protein Data Bank,” Nat. Struct. Biol., vol. 10, no. 12, p. 980, Dec. 2003." />
<meta name="citation_reference" content="H. M. Berman etal., “The Protein Data Bank,” Struct. Bioinforma., vol. 28, no. 1, pp. 181–198, Jan. 2005." />
<meta name="citation_reference" content="L. Y. Geer etal., “The NCBI BioSystems database,” Nucleic Acids Res., vol. 38, no. SUPPL.1, pp. D492–6, Jan. 2009." />
<meta name="citation_reference" content="A. Bateman, etal., “UniProt: The universal protein knowledgebase,” Nucleic Adds Res., vol. 45, no. D1, pp. D158–D169, Jan. 2017." />
<meta name="citation_reference" content="J. Yang, R. Yan, A. Roy, D. Xu, J. Poisson, and Y. Zhang, “The l-TASSER Suite: protein structure and function prediction,” Nat Meth, vol. 12, no. 1, pp. 7–8, 2015." />
<meta name="citation_reference" content="A. Roy, A. Kucukural, and Y. Zhang, “l-TASSER: a unified platform for automated protein structure and function prediction,” Nat. Protoc., vol. 5, no. 4, pp. 725–738, 2010." />
<meta name="citation_reference" content="Y. Zhang, “l-TASSER server for protein 3D structure prediction,” BMC Bioinformatics, vol. 9, pp. 1–8, 2008." />
<meta name="citation_reference" content="G. M. Morris et al., “Software news and updates AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility,” J. Comput. Chem., vol. 30, no. 16, pp. 2785–2791, Dec. 2009." />
<meta name="citation_reference" content="Ο. Trott and A. J. Olson, Software News and Update AutoDock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading, vol. 31. 2010." />
<meta name="citation_reference" content="V. Hornung, R. Hartmann, A. Ablasser, and K. P. Hopfner, “OAS proteins and cGAS: Unifying concepts in sensing and responding to cytosolic nucleic acids,” Nat. Rev. Immunol., vol. 14, no. 8, pp. 521–528, 2014." />
<meta name="citation_reference" content="F. Madeira etal., “The EMBL-EBI search and sequence analysis tools APIs in 2019.,” Nucleic Acids Res., Apr. 2019." />
<meta name="citation_reference" content="E. F. Pettersen et al., “UCSF Chimera—A Visualization System for Exploratory Research and Analysis,” J Comput Chem, vol. 25, pp. 1605–1612, 2004." />
<meta name="citation_reference" content="J. J. Irwin, T. Sterling, Μ. M. Mysinger, E. S. Bolstad, and R. G. Coleman, “ZINC: A free tool to discover chemistry for biology,” Journal of Chemical Information and Modeling, vol. 52, no. 7. pp. 1757–1768, 2012." />
<meta name="citation_reference" content="U. Viswanathan, S. M. Tomlinson, J. M. Fonner, S. A. Mock, and S. J. Watowich, “Identification of a Novel Inhibitor of Dengue Virus Protease through Use of a Virtual Screening Drug Discovery Web Portal,” J. Chem. Inf. Model., vol. 54, no. 10, pp. 2816–2825, 2014." />
<meta name="citation_reference" content="The-University-of-Texas-at-Austin, “Drug Discovery @ TEXAS ADVANCED COMPUTING CENTER (TACC),” 2012, 2019. [Online]. Available: https://drugdiscovery.tacc.utexas.edu." />
<meta name="citation_reference" content="R. A. Laskowski and Μ. B. Swindells, “LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery,” J. Chem. Inf. Model., vol. 51, no. 10, pp. 2778–2786, Oct. 2011." />
<meta name="citation_reference" content="D. S. Biovia, “Discovery Studio Visualizer vl7.2.0.16349.” San Diego: Dassault Systèmes, 2016." />
<meta name="citation_reference" content="T. Sander, “OSIRIS Property Explorer,” 2013. [Online]. Available: https://www.organic-chemistry.org/prog/peo/. [Accessed: 01-Apr-2019]." />
<meta name="citation_reference" content="T. W. H. Backman, Y. Cao, and T. Girke, “ChemMine tools: An online service for analyzing and clustering small molecules,” Nucleic Acids Res., vol. 39, no. SUPPL. 2, pp. W486–91, Jul. 2011." />
<meta name="citation_reference" content="J. Donovan, G. Whitney, S. Rath, and A. Korennykh, “Structural mechanism of sensing long dsRNA via a noncatalytic domain in human oligoadenylate synthetase 3,” Proc. Natl. Acad. Sci., vol. 112, no. 13, pp. 3949–3954, 2015." />
<meta name="citation_reference" content="M. Drappier and T. M îchîels, “Inhibition of the OAS/RNase L pathway by viruses,” Current Opinion in Virology, vol. 15. Elsevier, pp. 19–26, 01-Dec-2015." />
<meta name="citation_reference" content="L. Zhao etal., “Antagonism of the interferon-induced OAS-RNase L pathway by murine coronavirus ns2 protein is required for virus replication and liver pathology,” Cell Host Microbe, vol. 11, no. 6, pp. 607–616, Jun. 2012." />
<meta name="citation_reference" content="R. Zhang etal., “Homologous 2’,5’-phosphodiesterases from disparate RNA viruses antagonize antiviral innate immunity,” Proc. Natl. Acad. Sci. U. S. A., vol. 110, no. 32, pp. 13114–13119, Aug. 2013." />
<meta name="citation_reference" content="K. M. Ogden etal., “Structural Basis for 2ffl-5l2-Oligoadenylate Binding and Enzyme Activity of a Viral RNase L Antagonist,” J. Virol., vol. 89, no. 13, pp. 6633–6645, Jul. 2015." />
<meta name="citation_reference" content="S. Kim, etal., “PubChem 2019 update: Improved access to chemical data,” Nucleic Acids Res., vol. 47, no. D1, pp. D1102–D1109, Jan. 2019." />
<meta name="twitter:title" content="In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="As part of the type I IFN signaling, the 2’-5’- oligoadenylate synthetase (OAS) proteins have been involved in the progression of several non-viral diseases. Notably, OAS has been correlated with immune-modulatory functions that promote chronic inflammatory conditions, autoimmune disorders, cancer, and infectious diseases. In spite of this, OAS enzymes have been ignored as drug targets, and to date, there are no reports of compounds that can inhibit their activity. In this study, we have used homology modeling and virtual high-throughput screening to identify potential inhibitors of the human proteins OAS1, OAS2, and OAS3. Altogether, we have found 37 molecules that could exert a competitive inhibition in the ATP binding sites of OAS proteins, independently of the activation state of the enzyme. This latter characteristic, which might be crucial for a versatile inhibitor, was observed in compounds interacting with the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3. Although there was little correlation between specific chemical fragments and particular interactions, intermolecular contacts with OAS catalytic triad and other critical amino acids were mainly promoted by heterocycles with π electrons and hydrogen bond acceptors. In conclusion, this study provides a potential set of OAS inhibitors as well as valuable information for their design, development, and optimization." />
<meta name="og-title" property="og:title" content="In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/804716v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="As part of the type I IFN signaling, the 2’-5’- oligoadenylate synthetase (OAS) proteins have been involved in the progression of several non-viral diseases. Notably, OAS has been correlated with immune-modulatory functions that promote chronic inflammatory conditions, autoimmune disorders, cancer, and infectious diseases. In spite of this, OAS enzymes have been ignored as drug targets, and to date, there are no reports of compounds that can inhibit their activity. In this study, we have used homology modeling and virtual high-throughput screening to identify potential inhibitors of the human proteins OAS1, OAS2, and OAS3. Altogether, we have found 37 molecules that could exert a competitive inhibition in the ATP binding sites of OAS proteins, independently of the activation state of the enzyme. This latter characteristic, which might be crucial for a versatile inhibitor, was observed in compounds interacting with the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3. Although there was little correlation between specific chemical fragments and particular interactions, intermolecular contacts with OAS catalytic triad and other critical amino acids were mainly promoted by heterocycles with π electrons and hydrogen bond acceptors. In conclusion, this study provides a potential set of OAS inhibitors as well as valuable information for their design, development, and optimization." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/804716v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/804716v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/804716v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/804716v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/951032" />
    <title>In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UjVE87aYmxg9p7sUxlvJkbMB_ehzhfSzLF_SEHmn0qo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"951032","apath":"\/biorxiv\/early\/2019\/10\/15\/804716.atom","pisa":"biorxiv;804716v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;804716v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/804716v1","title":"In silico identification of potential inhibitors against human 2\u2019-5\u2019-oligoadenylate synthetase (OAS) proteins","identifier":"node\/951032"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sB9vawLTozxZbpF+L2CqdljPgtP8IUfCjuCVwgA+d8kKRIZ40Ob97+O+ruB8\\\\\\\/Df217Bj6MoCqmufAsRfKOCnOcBrgkdIivniPTy8CR5FolTpcbm4RwsMuWC9ZSzx96nQSdkemUWlAVHGJw7xRsSombs9FHnx2wJM\\\\\\\/S6zvaWR5q8uQAdxkKTe5zfMdQELrWeIqqsuXtPFEDPKasU6N13LwGIVR7yjIflaMPaiv7i9U1vFdt4+l3vY4bA7MF\\\\\\\/nMdkFD\\\\\\\/\\\\\\\/zdK9UmwBlGKpfzsr5g1NKRUuFzcCf11K9YzPoVPqyfv87R+nI5qSncXnHolbedGFElG2B3hiPw8DJtxXGrzvvphrcNTCqSXh9cUVGPGjR1gjl0KODbtmL+i9xTResWu7647ghjy9Cwc8CFxjVy5BxjQ9XihpFZaGZz0ymTacsqzYZGGXCwSp3\\\\\\\/e6BNvgen2hOOA\\\\\\\/mWSW1AtsOcWloTT8OoDIwRL3vvpaUUkSYrvhDFGovQAlX\\\\\\\/iDkSRMCh2bBb5BolcWsPuAGWWTLRG+xr27Nckru5jz9hxOSboNxCAKYmMSd7DbgY2O6ndCj6sudw4QUP2RLXFwMfW31lchw18RhAB2LYsa2lE3skP7NHTNZfnrqZKFvjNVohh2vEKW3FZDh6PZRWIB6l\\\\\\\/t6KwzbOVZ1wERZcJLaxyqAzQOVp7Jivt6QmN\\\\\\\/yETfKBNjnmY6JmAII+uhF\\\\\\\/WxQbvp4ElTYIJwEA8+ah2MukBu\\\\\\\/Taye49pqkSEAqZ5o3aFzQz5zAi8FIc+NDMinZIevU+ADs1OhpKP6VnOC0xvE8myrGTKjbHjYOAtk7JeeORHPh7H1BMMv\\\\\\\/hUVF57aoPwjRRCWXsUqMBaKHgL1w3FwElOLkBtQk4ruzI+PQjztzJqifcoNLk6P7HSgZ3\\\\\\\/bZCOM1xvwUB6ZNYH104dcELC5FoacG1gM4Kcmsdf1bk9eUka5FzUKlFAyfVZ4RRbKGqH0HkVEYdcytPH8hMVD2HhV\\\\\\\/Y++J4YSZuR6HyjMh0zwQ9AX04mS4+OmY7GgmPzrTd8H2tZnCyxqNbE7aOuMKJKNCWXLg1WTNCOCQED28kLC6Zqm2Qm47MOcQmRJwQLz9k2R8n0cGAAwhXXPz6bEXVyagvLfiBr2f97d0\\\\\\\/Q+bcrLRUD1adJXKTCF+GMwQOq6CQiueFwqZIWveheYkvEk6ckuCvgG5OEJpMIRb2q9cgwiJhWRzuoLhxhzAOTOxbebQ4Xl5U4wtyObPtAJBQbpQFehOgk3CW951+hWAWQHcwyOdE2STXVfHbPLBdx4fspOw+JQFgqsnhZ9L4ZHpeOvDUCYgO2DPKupUGM=\\\u0022,\\\u0022iv\\\u0022:\\\u00223NRFMQpSG1fr4RgMrBlXMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222ff27bd1af6d06e0e1857b980bee409fb7ae1c5db058cec3e229604bd4f19472\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/804716v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-951032 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/804716v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hZ5N2Kj_DE3wMxGidDMiayfbCQd6bcUpKkb-IsDdWbo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="951032" id="node951032" data-pisa="biorxiv;804716v1" data-pisa-master="biorxiv;804716" data-apath="/biorxiv/early/2019/10/15/804716.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title"><em>In silico</em> identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Karen J.</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Moncada-Giraldo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0627-5035" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Juan B.</span> <span class="nlm-surname">Gutierrez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/804716 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node951032" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karen J. Gonzalez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Bioinformatics, University of Georgia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BJ.%2BGonzalez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BJ.%2BGonzalez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diego Moncada-Giraldo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Bioinformatics, University of Georgia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BMoncada-Giraldo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moncada-Giraldo%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BMoncada-Giraldo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan B. Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics, University of Texas at San Antonio</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BB.%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BB.%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0627-5035" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan B. Gutierrez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:juan.gutierrez3@utsa.edu" class="" data-icon-position="" data-hide-link-title="0">juan.gutierrez3@utsa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/804716v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:951032" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:951032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/804716v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:951032" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:951032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/804716v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:951032" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:951032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/804716v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:951032" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:951032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/804716v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:951032" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:951032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">As part of the type I IFN signaling, the 2’-5’- oligoadenylate synthetase (OAS) proteins have been involved in the progression of several non-viral diseases. Notably, OAS has been correlated with immune-modulatory functions that promote chronic inflammatory conditions, autoimmune disorders, cancer, and infectious diseases. In spite of this, OAS enzymes have been ignored as drug targets, and to date, there are no reports of compounds that can inhibit their activity. In this study, we have used homology modeling and virtual high-throughput screening to identify potential inhibitors of the human proteins OAS1, OAS2, and OAS3. Altogether, we have found 37 molecules that could exert a competitive inhibition in the ATP binding sites of OAS proteins, independently of the activation state of the enzyme. This latter characteristic, which might be crucial for a versatile inhibitor, was observed in compounds interacting with the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3. Although there was little correlation between specific chemical fragments and particular interactions, intermolecular contacts with OAS catalytic triad and other critical amino acids were mainly promoted by heterocycles with π electrons and hydrogen bond acceptors. In conclusion, this study provides a potential set of OAS inhibitors as well as valuable information for their design, development, and optimization.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Inflammation is the defense mechanism employed by the immune system to counter potentially harmful agents [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. During viral infections, inflammatory responses are initiated by the activation of specific molecular sensors—known as pattern-recognition receptors (PRRs)—that detect pathogen-associated molecular patterns (PAMPs) [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Unlike bacteria, where PAMPs constitute a broad set of molecules, viral PAMPs consist mainly of nucleic acids and viral fusion glycoproteins [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. As a result of PRRs activity, type I interferons (IFN) and other mediators of the innate immune response are produced to exert an antiviral function [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>].</p><p id="p-4">In the presence of double-stranded (ds) RNA viruses, interferon-stimulated genes such as the 2’-5’-oligoadenylate synthetase (OAS) are activated to detect and restrict viral replication [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. In humans, the OAS family consists of three catalytically active enzymes, OAS1, OAS2, OAS3, and one OAS-like protein lacking oligoadenylate synthase activity [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Whereas OASL exerts its antiviral role through a ubiquitin-like domain [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], the remaining family members use a nucleotidyltransferase domain to catalyze the conversion of ATP into 2’-5’ -linked oligoadenylates (2-5A) [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. These 2-5A act as unique second messengers that trigger the activation of the ribonuclease L and induce the degradation of cellular and viral RNA [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-5">The activation mechanism of OAS has been shown to involve the recognition of dsRNA, followed by a significant structural rearrangement on the protein [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]-[<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Specifically, the inactive conformation of the enzyme senses dsRNA through a channel of positively charged residues located opposite to the active site [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. As a result of this interaction, OAS experiences a conformational change in its catalytic domain, bringing the active site into proximity and acquiring an optimal configuration for the catalytic reaction [<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>]. Although the mechanistic details of OAS activation have been studied mainly in OAS1, it is believed that the process is conserved for both OAS2 and OAS3 [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>].</p><p id="p-6">After stimulation with dsRNA, OAS requires two Mg<sup>2+</sup> ions and two molecules of ATP to perform its enzymatic activity [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. During the synthesis of 2-5A, one ATP, known as the donor, transfers its AMP moiety to a second ATP, known as the acceptor. For polymerization, the acceptor ATP is replaced by a growing chain of 2-5A [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Since the 2’-specificity of the oligomer is controlled by the binding mode of the acceptor ATP [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>], the Mg<sup>2+</sup> ions play a critical role in positioning the substrates in an ideal geometry for the reaction. Furthermore, Mg<sup>2+</sup> is also responsible for increasing the reactivity of the 2’ oxygen of the acceptor ATP [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>]. The binding of ATP and two Mg<sup>2+</sup> ions leave the OAS proteins fully activated with a structure referred to as the active conformation [<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>].</p><p id="p-7">Although the OAS family of proteins is known as an essential player in the effector response against viral infections [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. OAS has also been correlated with non-viral diseases such as chronic inflammatory conditions [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], autoimmune disorders [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]–[<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], oral cancer [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], tuberculosis [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], and malaria [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. As a common characteristic, OAS has been described as part of the type I IFN signaling, exerting a significant immunoregulatory role. Besides being involved in the progression of several diseases, OAS proteins have also been shown to contribute with resistance to the only approved treatment for gastric cancer [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] and oncolytic virus therapies for pancreatic ductal adenocarcinoma (PDA) [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p><p id="p-8">In oral cancer, it has been demonstrated that high expression of OAS2 promotes tumor progression by negative modulation of the T-cell functions. Particularly, OAS2 can regulate the TCR-ζ chain (CD3 ζ) expression and with it, limit the response required for controlling tumor growth [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>]. Other diseases associated with CD3-ζ chain expression have also shown up-regulation of OAS genes. In systemic lupus erythematosus (SLE), an autoimmune disorder related to the deficiency of CD3-ζ chain expression, increased levels of OAS were found to be correlated with disease activity [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>], [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Likewise, although no association has been confirmed with clinical parameters, the up-regulation of OAS was observed in relapsing-remitting multiple sclerosis [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] and systemic sclerosis [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>].</p><p id="p-9">In pancreatic β cells, high activation of OAS proteins increases basal apoptosis and augment the risk of developing type 1 diabetes (T1D) [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>]. This relationship was also found at the genotype level, where individuals with T1D had higher frequencies of polymorphisms associated with abnormal enzymatic activity than nondiabetic subjects [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Regarding infectious diseases, OAS expression has been reported as a signature that differentiates active tuberculosis from latent tuberculosis. As a consequence, it has been proposed that OAS has an immune-modulatory function that contributes to disease progression [<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>]. Similarly, the analysis of the OAS role during malaria infections has demonstrated that deficient suppression of type I IFN signaling may promote severe infections [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>].</p><p id="p-10">The critical role of OAS in non-viral diseases shows that this family of proteins might be a new target for immune modulation [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>] or improvement of existing treatments. Nonetheless, this approach has gone unnoticed in the drug discovery field, and to date, there are no reports of small compounds that can prevent OAS activity. Consequently, this study aims at the <em>in-silico</em> identification of potential OAS inhibitors. By virtual high-throughput screening, we identified 37 small drug-like molecules that could block all substrate binding sites of OAS and limit its 2-5A synthesis activity.</p></div><div class="section" id="sec-2"><h2 class="">Methodology</h2><div id="sec-3" class="subsection"><h3>Structural modeling</h3><p id="p-11">The nucleotidyltransferase domains of the human proteins OAS1, OAS2, and OAS3 were used as targets for virtual screening. The crystal structure of OAS1 was retrieved from the Protein Data Bank [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], with the PDB identifier <span class="underline">4IG8</span> [<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>]. In the absence of crystal structures for OAS2 and OAS3, these proteins were modeled by homology modeling. The sequences of OAS2 and OAS3 were obtained from NCBI [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] (accession codes <span class="underline">NP 058197.2,</span> and <span class="underline">NP 006178.2</span>), and their domains were defined with the UniProt database [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] (UniProt ID: <span class="underline">P29728,</span> and <span class="underline">Q9Y6K5</span>). Using l-TASSER [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]–[<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], the 3D structures of OAS2 and OAS3 were constructed considering the usual active and inactive conformations of OAS proteins. The template used for modeling was the porcine OAS1 (pOASl) in its active and inactive (PDB:) states (PDB: <span class="underline">4RWN</span> and <span class="underline">1PX5</span>) [<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>]. Since the crystal structure of human OAS1 is only available in an active conformation, the inactive state of this enzyme was also modeled. The template pOASl has 75.1, 53.1, and 56.6% sequence identity with human OAS1, OAS2, and OAS3, respectively.</p><p id="p-12">To consider the different activation states of OAS proteins during the virtual screening, the structures in an active conformation were turned into three models: active protein without ligands, active protein bound to Mg2+ ions, and active protein in complex with Mg2+ ions and a donor substrate. The holoenzymes were created according to pOASl (PDB: <span class="underline">4RWN</span>) as this enzyme has been crystallized with donor and acceptor ligands [<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>]. The donor substrate in the models corresponded to an ATP analog called α, β-Methyleneadenosine-5’-triphosphate (ApCpp). Altogether, twelve targets consisting of nine models and one crystal structure (counting for three activation states), were used for virtual screening (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). An example of the 3D structures representing each activation state studied is presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="3D models representing the OAS activation states considered for virtual screening. On the top left is shown the structural alignment of the active (blue) and inactive (orange) conformations of human 0AS3. On the top right are displayed the conformational changes produced by the binding of RNA such as the movement of the &#x3B2;-floor (black arrows), and the formation of a new a-helix in the active conformation of the protein (red sguare) [11]&#x2014;[13], [49]. To consider the catalytic mechanisms of OAS proteins, the active conformation is divided into three activation states representing the protein without ligands, the protein bound to Mg2+ ions, and the complex formed by Mg2+ ions and a donor substrate (ApCpp). Mg2+ ions are shown in green, and the ApCpp substrate is shown in yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;3D models representing the OAS activation states considered for virtual screening. On the top left is shown the structural alignment of the active (blue) and inactive (orange) conformations of human 0AS3. On the top right are displayed the conformational changes produced by the binding of RNA such as the movement of the &#x3B2;-floor (black arrows), and the formation of a new a-helix in the active conformation of the protein (red sguare) [11]&#x2014;[13], [49]. To consider the catalytic mechanisms of OAS proteins, the active conformation is divided into three activation states representing the protein without ligands, the protein bound to Mg&lt;sup&gt;2+&lt;/sup&gt; ions, and the complex formed by Mg&lt;sup&gt;2+&lt;/sup&gt; ions and a donor substrate (ApCpp). Mg&lt;sup&gt;2+&lt;/sup&gt; ions are shown in green, and the ApCpp substrate is shown in yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F1.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F1.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-13" class="first-child">3D models representing the OAS activation states considered for virtual screening. On the top left is shown the structural alignment of the active (blue) and inactive (orange) conformations of human 0AS3. On the top right are displayed the conformational changes produced by the binding of RNA such as the movement of the β-floor (black arrows), and the formation of a new a-helix in the active conformation of the protein (red sguare) [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]—[<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a>], [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. To consider the catalytic mechanisms of OAS proteins, the active conformation is divided into three activation states representing the protein without ligands, the protein bound to Mg<sup>2+</sup> ions, and the complex formed by Mg<sup>2+</sup> ions and a donor substrate (ApCpp). Mg<sup>2+</sup> ions are shown in green, and the ApCpp substrate is shown in yellow.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953520/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/953520/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/953520" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-14" class="first-child">OAS activation states considered for virtual screening. Human proteins OAS1, OAS2, and OAS3 were studied in four different activation states derived from two structural conformations: Inactive, before dsRNA binding; and active, after dsRNA binding.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3><em>In silico</em> high-throughput screening</h3><div id="sec-5" class="subsection"><h4>Target preparation for molecular docking</h4><p id="p-15">The crystal structure of human OAS1 <span class="underline">(4IG8</span>) was initially treated by removing crystallographic waters, cofactors, and ligands. Subsequently, all targets were prepared with the AutoDockTools 1-5.6 software [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>], adding Gasteiger’s charges and polar hydrogens, specifying AutoDock atom types, and converting PDB files to PDBQT file formats.</p></div><div id="sec-6" class="subsection"><h4>Search space selection for virtual screening</h4><p id="p-16">Prior to the virtual screening, docking calculations with ATP and ATP analogs were performed on OAS1, OAS2, and OAS3 to identify search spaces that favored the natural binding mode of OAS substrates. Using the AutoDock Vina 1.1.2 software [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], different search volumes were evaluated around amino acids involved in OAS activity (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) [<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a>], [<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">13</a>], [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Important residues for OAS2 and OAS3 were established by sequence and structural alignments against human and porcine OAS1, using CLUSTAL O(1.2.4) [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] and UCSF Chimera 1.10.2 [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. Docking calculations were carried out on all activation states of the proteins utilizing ATP, dATP, and ApCpp as substrates. The ATP structure was downloaded from the ZINC database [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], and dATP and ApCpp molecules were retrieved from the PDB files <span class="underline">4IG8</span> and <span class="underline">4RWN,</span> respectively. All the ligands were converted to the PDBQT format using the “prepare ligand” module available in AutoDockTools. The search spaces selected for virtual screening corresponded to search volumes where the substrates mimicked the binding modes reported in <span class="underline">4IG8</span> and <span class="underline">4RWN</span> (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S1</a>). For the activation states that have not been crystallized with a ligand, the search spaces were chosen based on interactions with amino acids involved in OAS activity. The docking results with the native substrates were later used as positive controls to determine whether a molecule could be a competitive inhibitor (see Selection criteria section).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Docking results approximating the natural binding of OAS substrates. The proteins displayed corresponds to human OAS1 4IG8 (dark green) and pOASl 4RWN (light green). On the left are compared the binding modes of donor substrates on the OAS1 &#x201C;Active-Mg2+&#x201D; state. The ligands docked (dATP, ApCpp, and ATP) were contrasted against the corresponding human and porcine crystal structures. In yellow is presented the binding of the ligand in the crystallization and in blue are shown the docking results. Green spheres were used to display the Mg2+ ions. Since ATP has not been crystallized in complex with OAS, the binding mode of this molecule was compared against the binding of dATP and ApCpp. On the right are analyzed the binding modes of the acceptor substrates on the OAS1 &#x201C;Active -Mg2+-donor&#x201D; state. The compounds studied as acceptors were ApCpp and ATP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Docking results approximating the natural binding of OAS substrates. The proteins displayed corresponds to human OAS1 4IG8 (dark green) and pOASl 4RWN (light green). On the left are compared the binding modes of donor substrates on the OAS1 &#x201C;Active-Mg&lt;sup&gt;2+&lt;/sup&gt;&#x201D; state. The ligands docked (dATP, ApCpp, and ATP) were contrasted against the corresponding human and porcine crystal structures. In yellow is presented the binding of the ligand in the crystallization and in blue are shown the docking results. Green spheres were used to display the Mg&lt;sup&gt;2+&lt;/sup&gt; ions. Since ATP has not been crystallized in complex with OAS, the binding mode of this molecule was compared against the binding of dATP and ApCpp. On the right are analyzed the binding modes of the acceptor substrates on the OAS1 &#x201C;Active -Mg&lt;sup&gt;2+&lt;/sup&gt;-donor&#x201D; state. The compounds studied as acceptors were ApCpp and ATP.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F7.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F7.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI.</span> <p id="p-41" class="first-child">Docking results approximating the natural binding of OAS substrates. The proteins displayed corresponds to human OAS1 <span class="underline">4IG8</span> (dark green) and pOASl <span class="underline">4RWN</span> (light green). On the left are compared the binding modes of donor substrates on the OAS1 “Active-Mg<sup>2+</sup>” state. The ligands docked (dATP, ApCpp, and ATP) were contrasted against the corresponding human and porcine crystal structures. In yellow is presented the binding of the ligand in the crystallization and in blue are shown the docking results. Green spheres were used to display the Mg<sup>2+</sup> ions. Since ATP has not been crystallized in complex with OAS, the binding mode of this molecule was compared against the binding of dATP and ApCpp. On the right are analyzed the binding modes of the acceptor substrates on the OAS1 “Active -Mg<sup>2+</sup>-donor” state. The compounds studied as acceptors were ApCpp and ATP.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953537/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/953537/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/953537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-17" class="first-child">Amino acids involved in the enzymatic activity of human OAS proteins. These amino acids were obtained by sequence and structural alignments against human and porcine OAS1. The residues are ordered so that analogous amino acids among the proteins are in the same position in the lists (e.g., Asp75, Asp408, and Asp816 are corresponding residues in OAS sequences).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Virtual screening</h4><p id="p-18">Virtual screenings against the donor and acceptor binding sites of OAS proteins were performed using the DrugDiscovery@TACC virtual portal [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>], [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. This web portal executed the Autodock Vina program on the “Lonestar” supercomputer at TACC (Texas Advanced Computing Center). The required input files were the target structures, the search spaces used for docking (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S1</a>), and the specification of the Virtual chemical library. While the protein structures in the “active-Mg<sup>2+</sup>-donor” state were used for screening on the acceptor binding site, the remaining targets were employed for screening on the donor binding site (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The chemical library used was a subset of the ZINC database termed the “ZINC (Lrg)” library [<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>], [<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>]. This library includes 642,769 small molecules with “clean, drug-like” constraints such as molecular weight between 150 and 500 Da, clogP ≤ 5, five or fewer hydrogen-bond donors, and ten or fewer hydrogen-bond acceptors. The analysis and visualization of the docking simulations were performed with UCSF Chimera 1.10.2, LIGPLOT+ 1.4.5 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], and Discovery Studio Visualizer V17.2.0.16349 [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953545/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/953545/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/953545" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.</span> <p id="p-46" class="first-child">Search spaces used for virtual screening against donor and acceptor binding sites of human OAS proteins.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-8" class="subsection"><h3>Selection criteria</h3><div id="sec-9" class="subsection"><h4>First selection filter</h4><p id="p-19">Two sets of selection criteria were established based on the binding site targeted on each protein state. Thus, ligands aiming at the donor binding site were filtered according to i) docking score lower than or equal to the positive control (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S2</a>); and ii) ability to interact with a minimum of six amino acids involved in the binding of the donor substrate, being at least two from the catalytic triad. Similarly, competitive inhibitors for the acceptor binding site were selected based on i) docking score lower than or equal to the positive control (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table S2</a>); and ii) ability to interact with at least six amino acids implicated in the binding of the acceptor substrate. Any attractive contact within 3.7 Â was considered as an interaction if it was predicted by more than one analysis tool (UCSF Chimera 1.10.2, LIGPLOT+ 1.4.5, and Discovery Studio Visualizer v17.2.0.16349). Moreover, interactions with the catalytic triad included either direct contact with the aspartic acids or indirect interactions through Mg<sup>2+</sup> ions.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/953553/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/953553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2.</span> <p id="p-47" class="first-child">Docking scores of ATP and ATP analogs in complex with human OAS1, OAS2, and OAS3. In red are highlighted the docking scores used as positive controls during the candidate selection. These values correspond to the lowest docking score among the docking results with ATP, dATP, and ApCpp.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Second selection filter</h4><p id="p-20">After selecting candidate molecules for each protein state, the ligands were docked at the remaining activation states within the same enzyme. Subsequently, the potential inhibitors were selected based on i) docking score lower than or equal to the positive control (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table S2</a>); ii) ability to interact—in all protein states—with at least two amino acids from the catalytic triad, and two amino acids in charge of placing the donor substrate; and iii) ability to interact with a minimum of four amino acids implicated in positioning the acceptor substrate. Finally, to overcome the stochasticity behind docking calculations, all screening experiments were repeated five times; only candidate inhibitors found in all repetitions were considered for further analysis.</p></div><div id="sec-11" class="subsection"><h4>Toxicity and structure similarity</h4><p id="p-21">Using the OSIRIS Property Explorer [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>], the candidate inhibitors were filtered based on the absence of risk alerts of mutagenicity or tumorigenicity. The final list of potential inhibitors was subjected to a structure similarity analysis utilizing the ChemMine Tools with a Tanimoto coefficient cutoff of 0.6 [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>].</p></div></div></div><div class="section" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Structural modeling and sequence alignment</h3><p id="p-22">To consider the activation mechanism of OAS proteins during the search for inhibitors of the 2-5A synthesis, homology models were constructed for the active and inactive states of human OAS1, OAS2, and OAS3. Five models preserving the characteristic folding of the active and inactive conformations were obtained with confidence scores (C-score) ranging from 1.5 to 1.79. The C-score, which usually goes from −5 to 2, being 2 the highest confidence [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]–[<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>], evidences the good quality of the predicted structures. Moreover, all models reproduced the conformational changes caused by the binding of RNA, such as the β-floor sliding and the formation of a new α-helix in the active site (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>)[<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>]. Given the above, the models obtained were suitable representatives of the different activation states required for OAS activity and can be used for downstream analysis.</p><p id="p-23">The amino acids that might be critical for the activity of OAS2 and OAS3 were determined based on human and porcine OAS1. By structural and sequence alignment, it was found that the catalytic triad and amino acids associated with the binding of the acceptor substrate were conserved among OAS1, OAS2, and OAS3 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S2</a>). Likewise, although slight differences were present at the donor site, all proteins preserved about 80% of the residues needed to position the donor substrate. The variations observed at OAS donor sites comprise the change of glutamine by lysine at position 566 in OAS2, and tyrosine by histidine at position 970 in OAS3 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S2</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence alignment between pOASl and human OAS1, OAS2, and OAS3. The color code used corresponds to the default Clustal X color scheme, which is assigned depending on the chemical properties of the amino acids. In red squares are highlighted the amino acids belonging to the catalytic triad, in blue squares are underlined the residues involved in positioning the donor substrate, and in black squares are emphasized the amino acids associated with the binding of the acceptor substrate. The numbering does not refer to a position in a specific sequence, but the position in the alignment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence alignment between pOASl and human OAS1, OAS2, and OAS3. The color code used corresponds to the default Clustal X color scheme, which is assigned depending on the chemical properties of the amino acids. In red squares are highlighted the amino acids belonging to the catalytic triad, in blue squares are underlined the residues involved in positioning the donor substrate, and in black squares are emphasized the amino acids associated with the binding of the acceptor substrate. The numbering does not refer to a position in a specific sequence, but the position in the alignment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F8.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F8.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.</span> <p id="p-42" class="first-child">Sequence alignment between pOASl and human OAS1, OAS2, and OAS3. The color code used corresponds to the default Clustal X color scheme, which is assigned depending on the chemical properties of the amino acids. In red squares are highlighted the amino acids belonging to the catalytic triad, in blue squares are underlined the residues involved in positioning the donor substrate, and in black squares are emphasized the amino acids associated with the binding of the acceptor substrate. The numbering does not refer to a position in a specific sequence, but the position in the alignment.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Virtual screening and candidate inhibitor selection</h3><p id="p-24">Virtual screenings against the two ATP binding sites of OAS1, OAS2, and OAS3, considering four different activation states for each protein, led to the identification of molecules that could block OAS substrate sites at any protein state. A total of 12,000 compounds (1,000 molecules per activation state) were observed occupying, partially or entirely, the donor or acceptor binding site of OAS. No molecules were common to all the proteins; instead, most of the ligands were unique to specific states. After a filtering process (see Methodology: Selection criteria), it was found that a total of 454 compounds for OAS1, 672 compounds for OAS2, and 523 compounds for OAS3, could obstruct the binding of ATP in at least one of the activation states of the proteins. Notably, 15 molecules for OAS1,19 molecules for OAS2, and 7 molecules for OAS3 docked at all ATP binding sites, regardless of the activation state of the enzyme. These 41 compounds were subjected to a toxicity analysis as they represented the most likely competitive inhibitors of OAS proteins. Four candidates were discarded because of the presence of chemical fragments that could be associated with tumorigenesis or mutagenesis. The final 37 potential inhibitors of OAS enzymes are presented in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Tables S4</a>, <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">S5</a>, and <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">S6</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953555/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/953555/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/953555" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S3.</span> <p id="p-48" class="first-child">Common chemical groups among OAS potential inhibitors.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953554/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/953554/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4.</span> <p id="p-49" class="first-child">Information associated with potential inhibitors for OAS1. The total interaction score was calculated as the sum of the interaction scores per residue (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). The intermolecular interactions presented only consider amino acids involved in the binding of the donor or the acceptor substrate. Web pages with details about the candidate molecules and biological tests are attached to each ZINC code. This information was retrieved from [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>].</p><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953550/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/953550/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S5.</span> <p id="p-50" class="first-child">Information associated with potential inhibitors for OAS2. The total interaction score was calculated as the sum of the interaction scores per residue (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). The intermolecular interactions presented only consider amino acids involved in the binding of the donor or the acceptor substrate. Web pages with details about the candidate molecules and biological tests are attached to each ZINC code. This information was retrieved from [<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>].</p><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/953556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/953556/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S6.</span> <p id="p-51" class="first-child">Information associated with potential inhibitors for OAS3. The total interaction score was calculated as the sum of the interaction scores per residue (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). The intermolecular interactions presented only consider amino acids involved in the binding of the donor or the acceptor substrate. Web pages with details about the candidate molecules and biological tests are attached to each ZINC code. This information was retrieved from [<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a>].</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Protein-ligand interactions</h3><p id="p-25">Most of the activation states of OAS proteins take place at the donor binding site of the enzyme [<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">12</a>]. Consequently, the ability to interact with this region in the inactive, active, and active-Mg<sup>2+</sup> states of the proteins was considered as an essential feature during the inhibitor selection. To identify characteristics associated with this property, the intermolecular contacts between the 37 candidate molecules and the OAS donor sites were analyzed using the interaction score method (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). As a result, it was observed that most of the compounds could interact with the same residues independently of the activation state of the enzyme. Specifically, OAS1 exhibited a consistent interaction profile where the amino acids Asp75, Asp77, Gln229, Gln194, and Tyr230 displayed the highest interactions score as they were in permanent contact with the inhibitors. Similarly, although OAS2 and OAS3 did not present a uniform pattern of interactions, the amino acids analogous to those described in OAS1 (except Gln194) also interacted with the ligands in the three activation states of these proteins (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overall contacts between potential OAS inhibitors and OAS donor sites. The colormap represents the number of OAS activation states where a candidate inhibitor interacted with an amino acid involved in the binding of the donor substrate. Only attractive contacts within 3.7&#xC0; were considered as an interaction. The color code and scores are assigned as follows: score three (yellow) corresponds to interactions with the three activation states of the OAS donor site (inactive, active, and active-Mg2+); score two (green) is assigned to interactions presented in two activation states; score one (light blue) refers to interactions in one activation state; and score zero (dark blue) indicates no interaction. On the y-axes are presented the ZINC codes of the candidate molecules, sorted from the highest to the lowest total interaction score (see Tables S4, S5, and S6). On the x-axes are shown the residues involved in the binding of the donor substrate. These amino acids are ordered so that analogous residues among the three proteins are in the same position across the x-axes (e.g., Asp75, Asp408, andAsp816 are corresponding residues in OAS seguences)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overall contacts between potential OAS inhibitors and OAS donor sites. The colormap represents the number of OAS activation states where a candidate inhibitor interacted with an amino acid involved in the binding of the donor substrate. Only attractive contacts within 3.7&#xC0; were considered as an interaction. The color code and scores are assigned as follows: score three (yellow) corresponds to interactions with the three activation states of the OAS donor site (inactive, active, and active-Mg&lt;sup&gt;2+&lt;/sup&gt;); score two (green) is assigned to interactions presented in two activation states; score one (light blue) refers to interactions in one activation state; and score zero (dark blue) indicates no interaction. On the y-axes are presented the ZINC codes of the candidate molecules, sorted from the highest to the lowest total interaction score (see Tables S4, S5, and S6). On the x-axes are shown the residues involved in the binding of the donor substrate. These amino acids are ordered so that analogous residues among the three proteins are in the same position across the x-axes (e.g., Asp75, Asp408, andAsp816 are corresponding residues in OAS seguences).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F2.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F2.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-26" class="first-child">Overall contacts between potential OAS inhibitors and OAS donor sites. The colormap represents the number of OAS activation states where a candidate inhibitor interacted with an amino acid involved in the binding of the donor substrate. Only attractive contacts within 3.7À were considered as an interaction. The color code and scores are assigned as follows: score three (yellow) corresponds to interactions with the three activation states of the OAS donor site (inactive, active, and active-Mg<sup>2+</sup>); score two (green) is assigned to interactions presented in two activation states; score one (light blue) refers to interactions in one activation state; and score zero (dark blue) indicates no interaction. On the y-axes are presented the ZINC codes of the candidate molecules, sorted from the highest to the lowest total interaction score (see <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Tables S4</a>, <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">S5</a>, and <a id="xref-table-wrap-8-2" class="xref-table" href="#T8">S6</a>). On the x-axes are shown the residues involved in the binding of the donor substrate. These amino acids are ordered so that analogous residues among the three proteins are in the same position across the x-axes (e.g., Asp75, Asp408, andAsp816 are corresponding residues in OAS seguences).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To determine the predominant intermolecular contacts between the potential inhibitors and important amino acids for OAS activity, the frequency of five types of protein-ligand interactions was calculated. Thus, at OAS donor sites, although weak electrostatic contacts were dominant, strong non-covalent interactions were mainly established with the residues bearing the highest interaction scores (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Tables S4</a>, <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">S5</a>, and <a id="xref-table-wrap-8-3" class="xref-table" href="#T8">S6</a>). Accordingly, Asp75, Asp77, Gln229, and Tyr230 in OAS1 and their equivalents in OAS2 and OAS3 interacted commonly by anion-π interactions, hydrogen bonds, and π-π interactions. Remarkably, the non-conserved residues, such as Lys566 in OAS2 and His970 in OAS3, did not follow the previous tendency and showed a high frequency of alkyl-π contacts and hydrogen bonds, respectively. Regarding the acceptor binding site, similar interactions were observed between analogous residues, being electrostatic contacts and alkyl-π interactions the most frequent protein-ligand interactions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3.B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency histogram of protein-ligand interactions between critical amino acids for OAS activity and potential OAS inhibitors. A. Protein-ligand interactions at OAS donor sites, considering three activation states for each protein (inactive, active, and active-Mg2+). B. Protein-ligand interactions at OAS acceptor sites. On the top of the bars is shown the frequency of each type of interaction, and on the x-axes are presented the residues involved in the binding of the donor and the acceptor substrates. In dark blue are shown Van der Waals contacts, in gray anion/cation-&#x3C0;effects, in yellow &#x3C0;-&#x3C0; interactions, in magenta alkyl-&#x3C0; and &#x3C3;&#x3C0; effects, in orange hydrogen bonds, and in green no interaction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency histogram of protein-ligand interactions between critical amino acids for OAS activity and potential OAS inhibitors. &lt;strong&gt;A.&lt;/strong&gt; Protein-ligand interactions at OAS donor sites, considering three activation states for each protein (inactive, active, and active-Mg&lt;sup&gt;2+&lt;/sup&gt;). &lt;strong&gt;B.&lt;/strong&gt; Protein-ligand interactions at OAS acceptor sites. On the top of the bars is shown the frequency of each type of interaction, and on the x-axes are presented the residues involved in the binding of the donor and the acceptor substrates. In dark blue are shown Van der Waals contacts, in gray anion/cation-&#x3C0;effects, in yellow &#x3C0;-&#x3C0; interactions, in magenta alkyl-&#x3C0; and &#x3C3;&#x3C0; effects, in orange hydrogen bonds, and in green no interaction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F3.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F3.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-28" class="first-child">Frequency histogram of protein-ligand interactions between critical amino acids for OAS activity and potential OAS inhibitors. <strong>A.</strong> Protein-ligand interactions at OAS donor sites, considering three activation states for each protein (inactive, active, and active-Mg<sup>2+</sup>). <strong>B.</strong> Protein-ligand interactions at OAS acceptor sites. On the top of the bars is shown the frequency of each type of interaction, and on the x-axes are presented the residues involved in the binding of the donor and the acceptor substrates. In dark blue are shown Van der Waals contacts, in gray anion/cation-πeffects, in yellow π-π interactions, in magenta alkyl-π and σπ effects, in orange hydrogen bonds, and in green no interaction.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Chemical structure similarity</h3><p id="p-29">Given the shared pattern of interactions across OAS1, OAS2, and OAS3, a structural similarity analysis was conducted among the potential inhibitors to identify scaffolds that could promote this behavior. As a consequence, two sets of highly similar molecules were identified for OAS1 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), and small chemical fragments were found common to several compounds for OAS1, OAS2, and OAS3 (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shared structures among potential inhibitors for OAS1. The scaffolds were identified using ChemMine Tools with a Tanimoto coefficient cutoff of 0.6 [48]. In orange and blue ovals are highlighted the differences among the molecules." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Shared structures among potential inhibitors for OAS1. The scaffolds were identified using ChemMine Tools with a Tanimoto coefficient cutoff of 0.6 [48]. In orange and blue ovals are highlighted the differences among the molecules.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F4.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F4.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-30" class="first-child">Shared structures among potential inhibitors for OAS1. The scaffolds were identified using ChemMine Tools with a Tanimoto coefficient cutoff of 0.6 [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>]. In orange and blue ovals are highlighted the differences among the molecules.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The molecules containing shared structures were subjected to a detailed analysis of protein-ligand interactions to recognize intermolecular contacts associated with each chemical fragment. Although there was no consistency between interactions promoted by small chemical groups (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table S3</a>), the scaffolds presented in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> did evidence a general interaction profile among the different activation states of OAS1. In this regarding, at the donor binding site of OAS1, the tetrazolo[1,5-a]quinoxaline ring (cluster 1), and the [1,2,4]triazolo[4,3-a]quinoxaline ring (cluster 2) established π-anion interactions with Asp75 and Asp77, and electrostatic contacts or hydrogen bonds with Gln229 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>, and <a id="xref-fig-10-1" class="xref-fig" href="#F10">S4</a>). Similarly, at the acceptor binding site, the [1,2,4]triazolo[4,3-a]quinoxaline ring contacted Leul50 through π-alkyl interactions while the tetrazolo[1,5-a]quinoxaline ring tended to form hydrogen bonds with Arg130 (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure S5</a>). On the other hand, even though the phenylpiperazinyl group in cluster 1 exerted different types of contacts with OAS1, this chemical fragment was usually positioned towards amino acids involved in the binding of the acceptor substrate (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">6</a>). This latter characteristic was also observed in other compounds containing a phenylpiperazinyl group, such as ZINC33034925 (OAS2), and ZINC20417720 (OAS3) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3S</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein-ligand interactions among OAS donor sites and candidate inhibitors containing a phenylpiperazinyl fragment. ZINC33034925 is docked on OAS2, and ZINC20417720 is interacting with OAS3. The phenylpiperazinyl group is usually observed in contact with residues involved in the binding of OAS acceptor substrates (red ovals). The carbon atoms of the ligands are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Protein-ligand interactions among OAS donor sites and candidate inhibitors containing a phenylpiperazinyl fragment. ZINC33034925 is docked on OAS2, and ZINC20417720 is interacting with OAS3. The phenylpiperazinyl group is usually observed in contact with residues involved in the binding of OAS acceptor substrates (red ovals). The carbon atoms of the ligands are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F9.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F9.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3.</span> <p id="p-43" class="first-child">Protein-ligand interactions among OAS donor sites and candidate inhibitors containing a phenylpiperazinyl fragment. ZINC33034925 is docked on OAS2, and ZINC20417720 is interacting with OAS3. The phenylpiperazinyl group is usually observed in contact with residues involved in the binding of OAS acceptor substrates (red ovals). The carbon atoms of the ligands are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-π or cation-π effects, in pink or purple hydrophobic interactions, in magenta π-π effects, and in gray metal-acceptor contacts.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intermolecular interactions between the donor binding site of OAS1 and candidate molecules sharing the scaffold 3-[([1,2,4]triazolo[4,3-a]guinoxalin-4-yl)amino]benzamide. The molecules presented belong to cluster 2 in Figure 4. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Intermolecular interactions between the donor binding site of OAS1 and candidate molecules sharing the scaffold 3-[([1,2,4]triazolo[4,3-a]guinoxalin-4-yl)amino]benzamide. The molecules presented belong to cluster 2 in Figure 4. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F10.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F10.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.</span> <p id="p-44" class="first-child">Intermolecular interactions between the donor binding site of OAS1 and candidate molecules sharing the scaffold 3-[([1,2,4]triazolo[4,3-a]guinoxalin-4-yl)amino]benzamide. The molecules presented belong to cluster 2 in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-π or cation-π effects, in pink or purple hydrophobic interactions, in magenta π-π effects, and in gray metal-acceptor contacts.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intermolecular interactions between the acceptor binding site of OAS1 and candidate molecules with high structural similarity. Referring to Figure 4, on the left are presented molecules belonging to cluster 1, and on the right are shown molecules grouped on cluster 2. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intermolecular interactions between the acceptor binding site of OAS1 and candidate molecules with high structural similarity. Referring to Figure 4, on the left are presented molecules belonging to cluster 1, and on the right are shown molecules grouped on cluster 2. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&lt;em&gt;&#x3C0;&lt;/em&gt; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F11.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F11.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5.</span> <p id="p-45" class="first-child">Intermolecular interactions between the acceptor binding site of OAS1 and candidate molecules with high structural similarity. Referring to <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, on the left are presented molecules belonging to cluster 1, and on the right are shown molecules grouped on cluster 2. The carbon atoms of the potential inhibitors are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-<em>π</em> or cation-π effects, in pink or purple hydrophobic interactions, in magenta π-π effects, and in gray metal-acceptor contacts.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Donor binding site of OAS1 in complex with potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[1,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in Figure 4. In yellow is displayed ZINC09561285, in orange is presented ZINC09561278, in green is shown ZINC09561296, and green spheres represent Mg2+ atoms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Donor binding site of OAS1 in complex with potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[1,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in Figure 4. In yellow is displayed ZINC09561285, in orange is presented ZINC09561278, in green is shown ZINC09561296, and green spheres represent Mg&lt;sup&gt;2+&lt;/sup&gt; atoms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F5.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F5.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-32" class="first-child">Donor binding site of OAS1 in complex with potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[1,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. In yellow is displayed ZINC09561285, in orange is presented ZINC09561278, in green is shown ZINC09561296, and green spheres represent Mg<sup>2+</sup> atoms.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted intermolecular interactions between the donor binding site of OAS1 and potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[l,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in Figure 4. The ZINC codes of the compounds are colored according to their binding modes in Figure 5. The carbon atoms in the molecules are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1333495408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted intermolecular interactions between the donor binding site of OAS1 and potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[l,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in Figure 4. The ZINC codes of the compounds are colored according to their binding modes in Figure 5. The carbon atoms in the molecules are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-&#x3C0; or cation-&#x3C0; effects, in pink or purple hydrophobic interactions, in magenta &#x3C0;-&#x3C0; effects, and in gray metal-acceptor contacts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F6.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F6.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/15/804716/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-33" class="first-child">Predicted intermolecular interactions between the donor binding site of OAS1 and potential inhibitors sharing the scaffold (4-phenylpiperazin-l-yl)-[3-(tetrazolo[l,5-a]guinoxalin-4-ylamino)phenyl]methanone. The molecules presented belong to cluster 1 in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>. The ZINC codes of the compounds are colored according to their binding modes in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>. The carbon atoms in the molecules are shown in gray, nitrogen atoms are presented in blue, and oxygen atoms are indicated in red. The color code used to represent the intermolecular interactions is: In light green are presented Van der Waals interactions, in bright green hydrogen bonds, in orange anion-π or cation-π effects, in pink or purple hydrophobic interactions, in magenta π-π effects, and in gray metal-acceptor contacts.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-17"><h2 class="">Discussion</h2><p id="p-34">The OAS family of proteins have been involved in a variety of immunoregulatory functions, promoting the progression of diseases such as cancer [<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>], autoimmune and autoinflammatory disorders [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>]–[<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>], and several infectious diseases [<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>], [<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>]. These detrimental outcomes are generally the result of a continuous positive feedback loop between the activation of OAS and the production of type I interferon [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>]. The possibility of disrupting this cycle of excessive inflammatory responses by the inhibition of OAS proteins places these enzymes as a new target for immune modulation.</p><p id="p-35">In nature, many viruses counteract OAS activity by sequestering their dsRNA or through modification or degradation of the 2-5A messenger [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]–[<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. Moreover, zinc, copper, and iron ions have also demonstrated to impair OAS function by altering the protein oligomerization [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>]. Besides this, there is no evidence of other mechanisms or compounds that can interrupt OAS action. In this study, we propose an ATP-competitive inhibition as a potential strategy to impede the 2-5A synthesis. As a result, we have identified 37 small molecules that can dock at the two ATP binding sites of OAS1, OAS2, or OAS3, with lower energy scores than the natural complexes. Furthermore, the compounds here presented can adjust to the dynamic behavior of these proteins and interact with their substrate sites independently of the activation state of the enzyme.</p><p id="p-36">Despite the inherent variations among OAS proteins, a general pattern of protein-ligand contacts was observed at the donor binding site of OAS1, OAS2, and OAS3. Particularly, the residues Asp75, Asp77, Gln229, and Tyr230 in OAS1, and their equivalents in OAS2 and OAS3 were in constant interaction with the candidate inhibitors (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Notably, although this region undergoes several conformational changes, the stated amino acids seem to be stable as they keep their position throughout OAS activity. Even if our homology models might not capture the precise spatial location of these residues, likely, interactions with the “stable” regions of the binding site promote the docking of ligands at any activation state of the protein. Indeed, numerous strong non-covalent interactions (hydrogen bonds and π-effects) were observed between the steady amino acids and the potential inhibitors (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>).</p><p id="p-37">Although distinct chemical groups contributed to a general pattern of protein-ligand interactions, certain molecular properties were common within the candidate inhibitors. Overall, it was observed that π-systems containing hydrogen bond acceptors (mostly heterocycles) could interact with OAS active site regardless of the activation state of the protein. In this respect, the π-clouds played a fundamental role in the binding of the inhibitors as they promoted π-anion interactions with the catalytic triad, π-cation contacts in the presence of Mg<sup>2+</sup>, and π-alky or π-π effects with residues neighboring the active site (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>, and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figures S3</a>-<a id="xref-fig-11-2" class="xref-fig" href="#F11">S5</a>).</p><p id="p-38">In summary, this work proposes 37 small drug-like molecules as potential ATP-competitive inhibitors of OAS proteins. Altogether, our results suggest that the candidate compounds can interrupt the formation of the 2-5A messengers by binding at the substrate sites of the active and inactive conformations of OAS proteins. Considering the influence of the OAS family in the development of inflammatory disorders, the ligands here identified represent potential alternatives for host-centric interventions aiming at modulating excessive inflammatory responses. As a future direction, experimental validation will be needed to determine the precise inhibitory potential of these compounds.</p></div><div class="section" id="sec-18"><h2 class="">Conclusion</h2><p id="p-39">The inhibition of the OAS family of proteins represents a host-centric strategy that could reduce the severity of a broad spectrum of diseases. In this study, we have identified 37 small drug-like compounds (12 molecules for OAS1, 18 molecules for OAS2, and 7 molecules for OAS3) that could exert a competitive inhibition at the donor and acceptor binding sites of OAS proteins. Several commonalities among the candidate inhibitors, such as general patterns of protein-ligand interactions as well as shared chemical fragments, are valuable findings that could be further explored in the rational design and optimization of general or selective OAS inhibitors. Overall, the molecules here presented offer a new set of candidates for in vitro testing and characterization as potential OAS inhibitors.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by the Defense Advanced Research Projects Agency and the US Army Research Office through the program Technologies for Host Resilience - Host Acute Models of Malaria to study Experimental Resilience (THoR’s HAMMER), DARPA contract #W911NF-16-C-0008, 2016-2019. Additionally, this project was also supported by the Voluntary Tuition Incentive Program for Research Grants funded by the Graduate School at the University of Georgia. The authors acknowledge the Texas Advanced Computing Center (TACC) at The University of Texas at Austin for providing High-performance computing (HPC) and the drug databases that have contributed to the research results reported within this paper. URL: <a href="http://www.tacc.utexas.edu">http://www.tacc.utexas.edu</a></p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Bibliography</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.1" data-doi="10.1038/nature07201"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Medzhitov</span></span>, “<span class="cit-article-title">Origin and physiological roles of inflammation</span>,” <abbr class="cit-jnl-abbrev">Nature</abbr>, vol. <span class="cit-vol">454</span>, no. <span class="cit-issue">7203</span>, pp. <span class="cit-fpage">428</span>–<span class="cit-lpage">435</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMedzhitov%26rft.auinit1%253DR.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D428%26rft.epage%253D435%26rft.atitle%253DOrigin%2Band%2Bphysiological%2Broles%2Bof%2Binflammation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07201%26rft_id%253Dinfo%253Apmid%252F18650913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18650913&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.2" data-doi="10.3109/08830185.2010.529976"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kawai</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Akira</span></span>, “<span class="cit-article-title">Pathogen recognition by the innate immune system</span>,” <abbr class="cit-jnl-abbrev">Int. Rev. Immunol</abbr>., vol. <span class="cit-vol">30</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">16</span>–<span class="cit-lpage">34</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Breviews%2Bof%2Bimmunology%26rft.stitle%253DInt%2BRev%2BImmunol%26rft.aulast%253DKumar%26rft.auinit1%253DH.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D34%26rft.atitle%253DPathogen%2Brecognition%2Bby%2Bthe%2Binnate%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F08830185.2010.529976%26rft_id%253Dinfo%253Apmid%252F21235323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/08830185.2010.529976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21235323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Broz</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Monack</span></span>, “<span class="cit-article-title">Newly described pattern recognition receptors team up against intracellular pathogens</span>,” <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol</abbr>., <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-804716v1.4" data-doi="10.1016/j.cell.2010.03.006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Medzhitov</span></span>, “<span class="cit-chapter-title">Inflammation 2010: New Adventures of an Old Flame</span>,” <span class="cit-source">Cell</span>, vol. <span class="cit-vol">140</span>, no. <span class="cit-issue">6</span>. <span class="cit-publ-name">Cell Press</span>, pp. <span class="cit-fpage">771</span>–<span class="cit-lpage">776</span>, 19-Mar-<span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMedzhitov%26rft.auinit1%253DR.%26rft.volume%253D140%26rft.issue%253D6%26rft.spage%253D771%26rft.epage%253D776%26rft.atitle%253DInflammation%2B2010%253A%2Bnew%2Badventures%2Bof%2Ban%2Bold%2Bflame.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.006%26rft_id%253Dinfo%253Apmid%252F20303867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20303867&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275746600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-804716v1.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hartmann</span></span>, <span class="cit-chapter-title">Nucleic Acid Immunity</span>, <span class="cit-edition">1st ed.</span>, vol. <span class="cit-vol">133</span>. <span class="cit-publ-name">Elsevier Inc.</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Gad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Despres</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hartmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Eskildsen-Larsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kristiansen</span></span>, “<span class="cit-article-title">The Oligoadenylate Synthetase Family: An Ancient Protein Family with Multiple Antiviral Activities</span>,” <abbr class="cit-jnl-abbrev">J. Interf. Cytokine Res</abbr>., vol. <span class="cit-vol">31</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">41</span>–<span class="cit-lpage">47</span>, <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInterf.%2BCytokine%2BRes%26rft.volume%253D31%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chebath</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Benech</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hovanessian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galabru</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Revel</span></span>, “<span class="cit-article-title">Four different forms of interferon-induced 2’,5’-oligo(A) synthetase identified by immunoblotting in human cells</span>,” <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>., vol. <span class="cit-vol">262</span>, no. <span class="cit-issue">8</span>, pp. <span class="cit-fpage">3852</span>–<span class="cit-lpage">3857</span>, Mar. <span class="cit-pub-date">1987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DChebath%26rft.auinit1%253DJ.%26rft.volume%253D262%26rft.issue%253D8%26rft.spage%253D3852%26rft.epage%253D3857%26rft.atitle%253DFour%2Bdifferent%2Bforms%2Bof%2Binterferon-induced%2B2%2527%252C5%2527-oligo%2528A%2529%2Bsynthetase%2Bidentified%2Bby%2Bimmunoblotting%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Apmid%252F2434505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNjIvOC8zODUyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMTUvODA0NzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.8" data-doi="10.1016/j.immuni.2014.05.007"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Suplemental information. Antiviral activity of human OASL protein is mediated by enhancing signaling of the RIG-1 RNA sensor</span>.,” <abbr class="cit-jnl-abbrev">Immunity</abbr>, vol. <span class="cit-vol">40</span>, no. <span class="cit-issue">6</span>, pp. <span class="cit-fpage">936</span>–<span class="cit-lpage">48</span>, Jun. <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D40%26rft.spage%253D936%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2014.05.007%26rft_id%253Dinfo%253Apmid%252F24931123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2014.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24931123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341440200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.9" data-doi="10.1007/PL00000644"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hartmann</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N. O.</span> <span class="cit-name-surname">Kjeldgaard</span></span>, “<span class="cit-article-title">Gene structure and function of the 2’-5’-oligoadenylate synthetase family</span>,” <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci</abbr>., vol. <span class="cit-vol">57</span>, no. <span class="cit-issue">11</span>, pp. <span class="cit-fpage">1593</span>–<span class="cit-lpage">1612</span>, <span class="cit-pub-date">2000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DJustesen%26rft.auinit1%253DJ.%26rft.volume%253D57%26rft.issue%253D11%26rft.spage%253D1593%26rft.epage%253D1612%26rft.atitle%253DGene%2Bstructure%2Band%2Bfunction%2Bof%2Bthe%2B2%2527-5%2527-oligoadenylate%2Bsynthetase%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00000644%26rft_id%253Dinfo%253Apmid%252F11092454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00000644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11092454&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165179800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-804716v1.10" data-doi="10.1038/nri2314"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Sadler</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B. R. G.</span> <span class="cit-name-surname">Williams</span></span>, “<span class="cit-chapter-title">Interferon-inducible antiviral effectors</span>,” <span class="cit-source">Nature Reviews Immunology</span>, vol. <span class="cit-vol">8</span>, no. <span class="cit-issue">7</span>. <span class="cit-publ-name">NIH Public Access</span>, pp. <span class="cit-fpage">559</span>–<span class="cit-lpage">568</span>, Jul-<span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DSadler%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253D559%26rft.epage%253D568%26rft.atitle%253DInterferon-inducible%2Bantiviral%2Beffectors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri2314%26rft_id%253Dinfo%253Apmid%252F18575461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18575461&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257089200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hartmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Sarkar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Sen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">V. C.</span> <span class="cit-name-surname">Yee</span></span>, “<span class="cit-article-title">Crystal Structure of the 2’-Specific and Double-Stranded RNA-Activated Interferon-Induced Antiviral Protein 2’-5’-Oligoadenylate Synthetase</span>,” <abbr class="cit-jnl-abbrev">Structure</abbr>, vol. <span class="cit-vol">19</span>, pp. <span class="cit-fpage">1173</span>–<span class="cit-lpage">1185</span>, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D19%26rft.spage%253D1173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-804716v1.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Jan</span> <span class="cit-name-surname">Lohofener</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-chapter-title">The activation mechanism of 2’ −5’-oligoadenylate synthetase gives new insights into OAS / cGAS triggers of innate immunity</span>.,” <span class="cit-publ-name">Cell Press</span>, vol. <span class="cit-vol">23</span>, pp. <span class="cit-fpage">851</span>–<span class="cit-lpage">862</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Donovan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dufner</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Korennykh</span></span>, “<span class="cit-article-title">Structural basis for cytosolic double-stranded RNA surveillance by human oligoadenylate synthetase 1</span>,” <span class="cit-publ-name">Proc. Natl. Acad. Sci</span>., vol. <span class="cit-vol">110</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">1652</span>–<span class="cit-lpage">1657</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hartmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Walko</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Justesen</span></span>, “<span class="cit-article-title">Inhibition of 2’-5’ oligoadenylate synthetase by divalent metal ions</span>,” <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>., vol. <span class="cit-vol">507</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">54</span>–<span class="cit-lpage">58</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DHartmann%26rft.auinit1%253DR.%26rft.volume%253D507%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D58%26rft.atitle%253DInhibition%2Bof%2B2%2527-5%2527%2Boligoadenylate%2Bsynthetase%2Bby%2Bdivalent%2Bmetal%2Bions.%26rft_id%253Dinfo%253Apmid%252F11682059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11682059&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Sarkar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ghosh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sung</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Sen</span></span>, “<abbr class="cit-jnl-abbrev">The Nature of the Catalytic Domain of 2-5-Oligoadenylate Synthetases</abbr>,” vol. <span class="cit-vol">274</span>, no. <span class="cit-issue">36</span>, pp. <span class="cit-fpage">25535</span>–<span class="cit-lpage">25542</span>,<span class="cit-pub-date">1999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNature%2Bof%2Bthe%2BCatalytic%2BDomain%2Bof%2B2-5-Oligoadenylate%2BSynthetases%26rft.volume%253D274%26rft.spage%253D25535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.16" data-doi="10.1016/j.coviro.2015.07.002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Drappier</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mîchîels</span></span>, “<span class="cit-article-title">Inhibition of the OAS/RNase L pathway by viruses</span>,” <abbr class="cit-jnl-abbrev">Curr. Opin. Virol</abbr>., vol. <span class="cit-vol">15</span>, pp. <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BVirol%26rft.volume%253D15%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2015.07.002%26rft_id%253Dinfo%253Apmid%252F26231767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2015.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26231767&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kôks</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kingo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Keermann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Reimann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Prans</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Abram</span></span>, “<span class="cit-article-title">Transcriptional landscape of psoriasis identifies the involvement of IL36 and IL36RN</span>,” <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, vol. <span class="cit-vol">16</span>, no. <span class="cit-issue">1</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Sanfilippo</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">OAS Gene Family Expression Is Associated with HIV-Related Neurocognitive Disorders</span>,” <abbr class="cit-jnl-abbrev">Mol. Neurobiol</abbr>., vol. <span class="cit-vol">55</span>, no. <span class="cit-issue">3</span>, pp. <span class="cit-fpage">1905</span>–<span class="cit-lpage">1914</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BNeurobiol%26rft.volume%253D55%26rft.spage%253D1905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.19" data-doi="10.1007/s11033-012-1624-x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Field</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bonnevie-Nielsen</span></span>, “<span class="cit-article-title">Induction and activation of antiviral enzyme 213,513-oligoadenylate synthetase by in vitro transcribed insulin mRNA and other cellular RNAs</span>,” <abbr class="cit-jnl-abbrev">Mol. Biol. Rep</abbr>., vol. <span class="cit-vol">39</span>, no. <span class="cit-issue">7</span>, pp. <span class="cit-fpage">7813</span>–<span class="cit-lpage">7822</span>, Jul. <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DDan%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D7813%26rft.epage%253D7822%26rft.atitle%253DInduction%2Band%2Bactivation%2Bof%2Bantiviral%2Benzyme%2B2%2527%252C5%2527-oligoadenylate%2Bsynthetase%2Bby%2Bin%2Bvitro%2Btranscribed%2Binsulin%2BmRNA%2Band%2Bother%2Bcellular%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-012-1624-x%26rft_id%253Dinfo%253Apmid%252F22547268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-012-1624-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22547268&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.20" data-doi="10.2337/diabetes.54.5.1588"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bonnevie-Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pociot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. B.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Beck-Nielsen</span></span>, “<span class="cit-article-title">OAS1 splice site polymorphism controlling antiviral enzyme activity influences susceptibility to type 1 diabetes</span>,” <abbr class="cit-jnl-abbrev">Diabetes</abbr>, vol. <span class="cit-vol">54</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">1588</span>–<span class="cit-lpage">1591</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiabetes%26rft.stitle%253DDiabetes%26rft.aulast%253DField%26rft.auinit1%253DL.%2BL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1588%26rft.epage%253D1591%26rft.atitle%253DOAS1%2BSplice%2BSite%2BPolymorphism%2BControlling%2BAntiviral%2BEnzyme%2BActivity%2BInfluences%2BSusceptibility%2Bto%2BType%2B1%2BDiabetes%26rft_id%253Dinfo%253Adoi%252F10.2337%252Fdiabetes.54.5.1588%26rft_id%253Dinfo%253Apmid%252F15855350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZGlhYmV0ZXMiO3M6NToicmVzaWQiO3M6OToiNTQvNS8xNTg4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMTUvODA0NzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.21" data-doi="10.1002/art.22044"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Feng</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Association of increased interferon-inducible gene expression with disease activity and lupus nephritis in patients with systemic lupus erythematosus</span>,” <abbr class="cit-jnl-abbrev">Arthritis Rheum</abbr>., vol. <span class="cit-vol">54</span>, no. <span class="cit-issue">9</span>, pp. <span class="cit-fpage">2951</span>–<span class="cit-lpage">2962</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2Band%2Brheumatism%26rft.stitle%253DArthritis%2BRheum%26rft.aulast%253DFeng%26rft.auinit1%253DX.%26rft.volume%253D54%26rft.issue%253D9%26rft.spage%253D2951%26rft.epage%253D2962%26rft.atitle%253DAssociation%2Bof%2Bincreased%2Binterferon-inducible%2Bgene%2Bexpression%2Bwith%2Bdisease%2Bactivity%2Band%2Blupus%2Bnephritis%2Bin%2Bpatients%2Bwith%2Bsystemic%2Blupus%2Berythematosus.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fart.22044%26rft_id%253Dinfo%253Apmid%252F16947629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/art.22044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16947629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240872900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Dar</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Extracellular 213513-oligoadenylate synthetase 2 mediates T-cell receptor Οϋ3-ζ chain down-regulation via caspase-3 activation in oral cancer</span>,” <abbr class="cit-jnl-abbrev">Immunology</abbr>, vol. <span class="cit-vol">147</span>, no. <span class="cit-issue">2</span>, pp. <span class="cit-fpage">251</span>–<span class="cit-lpage">264</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D147%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Leisching</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Wiid</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Baker</span></span>, “<span class="cit-article-title">OAS1, 2 and 3: Significance During Active Tuberculosis?</span>,” <abbr class="cit-jnl-abbrev">J. Infect. Dis</abbr>., vol. <span class="cit-vol">217</span>, no. <span class="cit-issue">10</span>, pp. <span class="cit-fpage">1517</span>–<span class="cit-lpage">1521</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis%26rft.volume%253D217%26rft.spage%253D1517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.24" data-doi="10.1126/scitranslmed.aar3619"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Conway</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Integrated pathogen load and dual transcriptome analysis of systemic host-pathogen interactions in severe malaria</span>,” <abbr class="cit-jnl-abbrev">Sci. Transl. Med.</abbr>, vol. <span class="cit-vol">10</span>, no. <span class="cit-issue">447</span>, p. <span class="cit-fpage">eaar3619</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DLee%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D10%26rft.issue%253D447%26rft.spage%253Deaar3619%26rft.epage%253Deaar3619%26rft.atitle%253DIntegrated%2Bpathogen%2Bload%2Band%2Bdual%2Btranscriptome%2Banalysis%2Bof%2Bsystemic%2Bhost-pathogen%2Binteractions%2Bin%2Bsevere%2Bmalaria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.aar3619%26rft_id%253Dinfo%253Apmid%252F29950443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjE1OiIxMC80NDcvZWFhcjM2MTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8xNS84MDQ3MTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.25"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Yu</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Prediction of key genes and pathways involved in trastuzumab-resistant gastric cancer</span>,” <abbr class="cit-jnl-abbrev">World J. Surg. Oncol</abbr>., vol. <span class="cit-vol">16</span>, no. <span class="cit-issue">1</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Murphy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Mukherjee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Moerdyk-Schauwecker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. Z.</span> <span class="cit-name-surname">Grdzelishvili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Shah</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hastie</span></span>, “<span class="cit-article-title">Resistance of pancreatic cancer cells to oncolytic vesicular stomatitis virus: Role of type I interferon signaling</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">436</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">221</span>–<span class="cit-lpage">234</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D436%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.27" data-doi="10.1177/0961203308089694"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tang</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Increased expression of the type I interferon-inducible gene, lymphocyte antigen 6 complex locus E, in peripheral blood cells is predictive of lupus activity in a large cohort of Chinese lupus patients</span>,” <abbr class="cit-jnl-abbrev">Lupus</abbr>, vol. <span class="cit-vol">17</span>, no. <span class="cit-issue">9</span>, pp. <span class="cit-fpage">805</span>–<span class="cit-lpage">813</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLupus%26rft.stitle%253DLupus%26rft.aulast%253DTang%26rft.auinit1%253DJ%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D805%26rft.epage%253D813%26rft.atitle%253DIncreased%2Bexpression%2Bof%2Bthe%2Btype%2BI%2Binterferon-inducible%2Bgene%252C%2Blymphocyte%2Bantigen%2B6%2Bcomplex%2Blocus%2BE%252C%2Bin%2Bperipheral%2Bblood%2Bcells%2Bis%2Bpredictive%2Bof%2Blupus%2Bactivity%2Bin%2Ba%2Blarge%2Bcohort%2Bof%2BChinese%2Blupus%2Bpatients%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0961203308089694%26rft_id%253Dinfo%253Apmid%252F18755862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0961203308089694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18755862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.28" data-doi="10.1038/sj.gene.6364324"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. G. M.</span> <span class="cit-name-surname">van Baarsen</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">A subtype of multiple sclerosis defined by an activated immune defense program</span>,” <abbr class="cit-jnl-abbrev">Genes Immun</abbr>., vol. <span class="cit-vol">7</span>, no. <span class="cit-issue">6</span>, pp. <span class="cit-fpage">522</span>–<span class="cit-lpage">531</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Band%2Bimmunity%26rft.stitle%253DGenes%2BImmun%26rft.aulast%253Dvan%2BBaarsen%26rft.auinit1%253DL.%2BG.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D522%26rft.epage%253D531%26rft.atitle%253DA%2Bsubtype%2Bof%2Bmultiple%2Bsclerosis%2Bdefined%2Bby%2Ban%2Bactivated%2Bimmune%2Bdefense%2Bprogram.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.gene.6364324%26rft_id%253Dinfo%253Apmid%252F16837931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.gene.6364324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16837931&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240370200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C. M.</span> <span class="cit-name-surname">Guedes</span></span> <span class="cit-etal">et al.</span>, “ <span class="cit-article-title">Differential upregulation of human 2BI5BI OAS genes on systemic sclerosis: Detection of increased basal levels of OASL and OAS 2 genes through a qPCR based assay</span>,” <abbr class="cit-jnl-abbrev">Autoimmunity</abbr>, vol. <span class="cit-vol">47</span>, no. <span class="cit-issue">2</span>, pp. <span class="cit-fpage">119</span>–<span class="cit-lpage">126</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAutoimmunity%26rft.volume%253D47%26rft.spage%253D119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Berman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Henrick</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nakamura</span></span>, “<span class="cit-article-title">Announcing the worldwide Protein Data Bank</span>,” <abbr class="cit-jnl-abbrev">Nat. Struct. Biol</abbr>., vol. <span class="cit-vol">10</span>, no. <span class="cit-issue">12</span>, p. 980, Dec. <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Berman</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The Protein Data Bank</span>,” <abbr class="cit-jnl-abbrev">Struct. Bioinforma</abbr>., vol. <span class="cit-vol">28</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">181</span>–<span class="cit-lpage">198</span>, Jan. <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStruct.%2BBioinforma%26rft.volume%253D28%26rft.spage%253D181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.32"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. Y.</span> <span class="cit-name-surname">Geer</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The NCBI BioSystems database</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., vol. <span class="cit-vol">38</span>, no. <span class="cit-issue">SUPPL.1</span>, pp. <span class="cit-fpage">D492</span>–<span class="cit-lpage">6</span>, Jan. <span class="cit-pub-date">2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D38%26rft.spage%253DD492%26rft_id%253Dinfo%253Apmid%252F19854944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19854944&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.33" data-doi="10.1093/nar/gkw1099"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, <span class="cit-etal">et al.</span>, “<span class="cit-article-title">UniProt: The universal protein knowledgebase</span>,” <abbr class="cit-jnl-abbrev">Nucleic Adds Res</abbr>., vol. <span class="cit-vol">45</span>, no. <span class="cit-issue">D1</span>, pp. <span class="cit-fpage">D158</span>–<span class="cit-lpage">D169</span>, Jan. <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAdds%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1099%26rft_id%253Dinfo%253Apmid%252F27899622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Roy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poisson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, “<span class="cit-article-title">The l-TASSER Suite: protein structure and function prediction</span>,” <abbr class="cit-jnl-abbrev">Nat Meth</abbr>, vol. <span class="cit-vol">12</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">7</span>–<span class="cit-lpage">8</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMeth%26rft.volume%253D12%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[35].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-804716v1.35" data-doi="10.1038/nprot.2010.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Roy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kucukural</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, “<span class="cit-article-title">l-TASSER: a unified platform for automated protein structure and function prediction</span>,” <abbr class="cit-jnl-abbrev">Nat. Protoc</abbr>., vol. <span class="cit-vol">5</span>, no. <span class="cit-issue">4</span>, pp. <span class="cit-fpage">725</span>–<span class="cit-lpage">738</span>, <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DRoy%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D738%26rft.atitle%253DI-TASSER%253A%2Ba%2Bunified%2Bplatform%2Bfor%2Bautomated%2Bprotein%2Bstructure%2Band%2Bfunction%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2010.5%26rft_id%253Dinfo%253Apmid%252F20360767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2010.5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20360767&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276196500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.36" data-doi="10.1186/1471-2105-9-1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, “<span class="cit-article-title">l-TASSER server for protein 3D structure prediction</span>,” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, vol. <span class="cit-vol">9</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DFierro%26rft.auinit1%253DA.%2BC.%26rft.volume%253D9%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DEvaluation%2Bof%2Btime%2Bprofile%2Breconstruction%2Bfrom%2Bcomplex%2Btwo-color%2Bmicroarray%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-1%26rft_id%253Dinfo%253Apmid%252F18173834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18173834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.37" data-doi="10.1002/jcc.21256"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Morris</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Software news and updates AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>,” <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>., vol. <span class="cit-vol">30</span>, no. <span class="cit-issue">16</span>, pp. <span class="cit-fpage">2785</span>–<span class="cit-lpage">2791</span>, Dec. <span class="cit-pub-date">2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DMorris%26rft.auinit1%253DG.%2BM.%26rft.volume%253D30%26rft.issue%253D16%26rft.spage%253D2785%26rft.epage%253D2791%26rft.atitle%253DAutoDock4%2Band%2BAutoDockTools4%253A%2BAutomated%2Bdocking%2Bwith%2Bselective%2Breceptor%2Bflexibility.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.21256%26rft_id%253Dinfo%253Apmid%252F19399780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.21256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19399780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271112100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Ο.</span> <span class="cit-name-surname">Trott</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Olson</span></span>, <abbr class="cit-jnl-abbrev">Software News and Update AutoDock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading</abbr>, vol. <span class="cit-vol">31</span>. <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.39" data-doi="10.1038/nri3719"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Hornung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hartmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ablasser</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Hopfner</span></span>, “<span class="cit-article-title">OAS proteins and cGAS: Unifying concepts in sensing and responding to cytosolic nucleic acids</span>,” <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol</abbr>., vol. <span class="cit-vol">14</span>, no. <span class="cit-issue">8</span>, pp. <span class="cit-fpage">521</span>–<span class="cit-lpage">528</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BImmunol%26rft.volume%253D14%26rft.spage%253D521%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3719%26rft_id%253Dinfo%253Apmid%252F25033909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri3719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25033909&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.40"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Madeira</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The EMBL-EBI search and sequence analysis tools APIs in 2019</span>.,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, Apr. <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.41" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. F.</span> <span class="cit-name-surname">Pettersen</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">UCSF Chimera—A Visualization System for Exploratory Research and Analysis</span>,” <abbr class="cit-jnl-abbrev">J Comput Chem</abbr>, vol. <span class="cit-vol">25</span>, pp. <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>, <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.42" data-doi="10.1021/ci3001277"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Irwin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sterling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Μ. M.</span> <span class="cit-name-surname">Mysinger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Bolstad</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Coleman</span></span>, “<span class="cit-article-title">ZINC: A free tool to discover chemistry for biology</span>,” <abbr class="cit-jnl-abbrev">Journal of Chemical Information and Modeling</abbr>, vol. <span class="cit-vol">52</span>, no. <span class="cit-issue">7</span>. pp. <span class="cit-fpage">1757</span>–<span class="cit-lpage">1768</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DIrwin%26rft.volume%253D52%26rft.spage%253D1757%26rft.atitle%253DZINC%2B-%2BA%2Bfree%2Btool%2Bto%2Bdiscover%2Bchemistry%2Bfor%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci3001277%26rft_id%253Dinfo%253Apmid%252F22587354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci3001277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22587354&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.43"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Viswanathan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Tomlinson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Fonner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Mock</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Watowich</span></span>, “<span class="cit-article-title">Identification of a Novel Inhibitor of Dengue Virus Protease through Use of a Virtual Screening Drug Discovery Web Portal</span>,” <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model.</abbr>, vol. <span class="cit-vol">54</span>, no. <span class="cit-issue">10</span>, pp. <span class="cit-fpage">2816</span>–<span class="cit-lpage">2825</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel.%26rft.volume%253D54%26rft.spage%253D2816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.44"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The-University-of-Texas-at-Austin</span>, “<abbr class="cit-jnl-abbrev">Drug Discovery @ TEXAS ADVANCED COMPUTING CENTER (TACC)</abbr>,” 2012, <span class="cit-pub-date">2019</span>. [Online]. Available: <a href="https://drugdiscovery.tacc.utexas.edu">https://drugdiscovery.tacc.utexas.edu</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.45" data-doi="10.1021/ci200227u"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Laskowski</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Μ. B.</span> <span class="cit-name-surname">Swindells</span></span>, “<span class="cit-article-title">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span>,” <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model</abbr>., vol. <span class="cit-vol">51</span>, no. <span class="cit-issue">10</span>, pp. <span class="cit-fpage">2778</span>–<span class="cit-lpage">2786</span>, Oct. <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DLaskowski%26rft.auinit1%253DR.%2BA.%26rft.volume%253D51%26rft.issue%253D10%26rft.spage%253D2778%26rft.epage%253D2786%26rft.atitle%253DLigPlot%252B%253A%2Bmultiple%2Bligand-protein%2Binteraction%2Bdiagrams%2Bfor%2Bdrug%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci200227u%26rft_id%253Dinfo%253Apmid%252F21919503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci200227u&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21919503&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Biovia</span></span>, “<span class="cit-article-title">Discovery Studio Visualizer vl7.2.0.16349</span>.” <abbr class="cit-jnl-abbrev">San Diego: Dassault Systèmes</abbr>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.47"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sander</span></span>, “<span class="cit-article-title">OSIRIS Property Explorer</span>,” <span class="cit-pub-date">2013</span>. [Online]. Available: <a href="https://www.organic-chemistry.org/prog/peo/">https://www.organic-chemistry.org/prog/peo/</a>. [Accessed: <span class="cit-date-in-citation">01-Apr-2019</span>].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.48" data-doi="10.1093/nar/gkr320"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. W. H.</span> <span class="cit-name-surname">Backman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cao</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Girke</span></span>, “<span class="cit-article-title">ChemMine tools: An online service for analyzing and clustering small molecules</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., vol. <span class="cit-vol">39</span>, no. <span class="cit-issue">SUPPL. 2</span>, pp. <span class="cit-fpage">W486</span>–<span class="cit-lpage">91</span>, Jul. <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr320%26rft_id%253Dinfo%253Apmid%252F21576229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21576229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300079&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.49" data-doi="10.1073/pnas.1419409112"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Donovan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Whitney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rath</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Korennykh</span></span>, “<span class="cit-article-title">Structural mechanism of sensing long dsRNA via a noncatalytic domain in human oligoadenylate synthetase 3</span>,” <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>, vol. <span class="cit-vol">112</span>, no. <span class="cit-issue">13</span>, pp. <span class="cit-fpage">3949</span>–<span class="cit-lpage">3954</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1419409112%26rft_id%253Dinfo%253Apmid%252F25775560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEzLzM5NDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8xNS84MDQ3MTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-804716v1.50"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Drappier</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">T. M</span> <span class="cit-name-surname">îchîels</span></span>, “<span class="cit-chapter-title">Inhibition of the OAS/RNase L pathway by viruses</span>,” <span class="cit-source">Current Opinion in Virology</span>, vol. <span class="cit-vol">15</span>. <span class="cit-publ-name">Elsevier</span>, pp. <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>, 01-Dec-<span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BVirology%26rft.volume%253D15%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[51].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-804716v1.51" data-doi="10.1016/j.chom.2012.04.011"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhao</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Antagonism of the interferon-induced OAS-RNase L pathway by murine coronavirus ns2 protein is required for virus replication and liver pathology</span>,” <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>, vol. <span class="cit-vol">11</span>, no. <span class="cit-issue">6</span>, pp. <span class="cit-fpage">607</span>–<span class="cit-lpage">616</span>, Jun. <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DZhao%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D607%26rft.epage%253D616%26rft.atitle%253DAntagonism%2Bof%2Bthe%2Binterferon-induced%2BOAS-RNase%2BL%2Bpathway%2Bby%2Bmurine%2Bcoronavirus%2Bns2%2Bprotein%2Bis%2Brequired%2Bfor%2Bvirus%2Breplication%2Band%2Bliver%2Bpathology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.04.011%26rft_id%253Dinfo%253Apmid%252F22704621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.04.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22704621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F15%2F804716.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305821800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[52].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-804716v1.52" data-doi="10.1073/pnas.1306917110"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Homologous 2’,5’-phosphodiesterases from disparate RNA viruses antagonize antiviral innate immunity</span>,” <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr>, vol. <span class="cit-vol">110</span>, no. <span class="cit-issue">32</span>, pp. <span class="cit-fpage">13114</span>–<span class="cit-lpage">13119</span>, Aug. <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1306917110%26rft_id%253Dinfo%253Apmid%252F23878220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMyLzEzMTE0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMTUvODA0NzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.53" data-doi="10.1128/JVI.00701-15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Ogden</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Structural Basis for 2ffl-5l2-Oligoadenylate Binding and Enzyme Activity of a Viral RNase L Antagonist</span>,” <abbr class="cit-jnl-abbrev">J. Virol</abbr>., vol. <span class="cit-vol">89</span>, no. <span class="cit-issue">13</span>, pp. <span class="cit-fpage">6633</span>–<span class="cit-lpage">6645</span>, Jul. <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00701-15%26rft_id%253Dinfo%253Apmid%252F25878106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xMy82NjMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMTUvODA0NzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-804716v1.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-etal">et al.</span>, “<span class="cit-article-title">PubChem 2019 update: Improved access to chemical data</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, vol. <span class="cit-vol">47</span>, no. <span class="cit-issue">D1</span>, pp. <span class="cit-fpage">D1102</span>–<span class="cit-lpage">D1109</span>, Jan. <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD1102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/804716v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F804716v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/804641v1" title="XTRACT - Standardised protocols for automated tractography and connectivity blueprints in the human and macaque brain" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 15, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/804716v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/804716v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/804716v1" class="active" data-icon-position="" data-hide-link-title="0">In silico identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/951032" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fNMZk3ygD2xBVhQmW8St771X18J4XM_dkLg39qiwStI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="951032" id="node951032--2" data-pisa="biorxiv;804716v1" data-pisa-master="biorxiv;804716" data-seqnum="951032" data-apath="/biorxiv/early/2019/10/15/804716.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title"><em>In silico</em> identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Karen J.</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Moncada-Giraldo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Juan B.</span> <span class="nlm-surname">Gutierrez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">804716; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/804716 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/804716v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;title=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;title=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;text=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;title=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;t=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;title=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;title=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="951032" id="node951032--3" data-pisa="biorxiv;804716v1" data-pisa-master="biorxiv;804716" data-seqnum="951032" data-apath="/biorxiv/early/2019/10/15/804716.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title"><em>In silico</em> identification of potential inhibitors against human 2’-5’-oligoadenylate synthetase (OAS) proteins</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Karen J.</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Moncada-Giraldo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Juan B.</span> <span class="nlm-surname">Gutierrez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">804716; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/804716 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Gonzalez%2C+Karen+J.&amp;rft.au=Moncada-Giraldo%2C+Diego&amp;rft.au=Gutierrez%2C+Juan+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/951032/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/951032/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/951032/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/951032/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/951032/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/951032/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/951032/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/951032/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/951032/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/951032/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/951032/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/951032/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;count=horizontal&amp;via=&amp;text=In%20silico%20identification%20of%20potential%20inhibitors%20against%20human%202%E2%80%99-5%E2%80%99-oligoadenylate%20synthetase%20%28OAS%29%20proteins&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/804716v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/804716v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/804716v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
