<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/embed/graphic-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/03/06/569012.atom" />
<meta name="HW.pisa" content="biorxiv;569012v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure" />
<meta name="DC.Identifier" content="10.1101/569012" />
<meta name="DC.Date" content="2019-03-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="1.  Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common.

2.  Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus.

3.  Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years.

4.  When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns." />
<meta name="DC.Contributor" content="Nicholas M. Fountain-Jones" />
<meta name="DC.Contributor" content="Gustavo Machado" />
<meta name="DC.Contributor" content="Scott Carver" />
<meta name="DC.Contributor" content="Craig Packer" />
<meta name="DC.Contributor" content="Mariana Recamonde-Mendoza" />
<meta name="DC.Contributor" content="Meggan E. Craft" />
<meta name="article:published_time" content="2019-03-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common.&lt;/p&gt;&lt;p&gt;Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus.&lt;/p&gt;&lt;p&gt;Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years.&lt;/p&gt;&lt;p&gt;When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;569012v1" />
<meta name="citation_id" content="569012v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/569012v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/569012v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/569012v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012.full.pdf" />
<meta name="citation_doi" content="10.1101/569012" />
<meta name="citation_num_pages" content="33" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="569012" />
<meta name="citation_author" content="Nicholas M. Fountain-Jones" />
<meta name="citation_author_institution" content="Department of Veterinary Population Medicine, University of Minnesota" />
<meta name="citation_author" content="Gustavo Machado" />
<meta name="citation_author_institution" content="Department of Population Health and Pathobiology, North Carolina State University" />
<meta name="citation_author" content="Scott Carver" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Tasmania" />
<meta name="citation_author" content="Craig Packer" />
<meta name="citation_author_institution" content="Department of Ecology Evolution and Behavior, University of Minnesota" />
<meta name="citation_author" content="Mariana Recamonde-Mendoza" />
<meta name="citation_author_institution" content="Institute of Informatics, Universidade Federal do Rio Grande do Sul" />
<meta name="citation_author" content="Meggan E. Craft" />
<meta name="citation_author_institution" content="Department of Veterinary Population Medicine, University of Minnesota" />
<meta name="citation_reference" content="Altizer, S., Nunn, C. L., Thrall, P. H., Gittleman, J. L., Antonovics, J., Cunningham, A. a., … Pulliam, J. R. C. (2003). Social organization and parasite risk in mammals : Integrating theory and empirical studies. Annual Review of Ecology, Evolution, and Systematics, 34(1), 517–547. doi:10.1146/annurev.ecolsys.34.030102.151725" />
<meta name="citation_reference" content="Anderson, R. M., &amp; May, R. M. (1985). Age-related changes in the rate of disease transmission: implications for the design of vaccination programmes. The Journal of Hygiene, 94(3), 365–436. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/4008922" />
<meta name="citation_reference" content="Babayan, S. A., Orton, R. J., &amp; Streicker, D. G. (2018). Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes. Science, 362(6414), 577–580. doi:10.1126/science.aap9072" />
<meta name="citation_reference" content="Behdenna, A., Lembo, T., Cleaveland, S., Halliday, J. E. B., Packer, C., Lankester, F., … Viana, M. (in press). Transmission ecology of canine parvovirus in a multi-host, multi-pathogen system. Proc R Soc B." />
<meta name="citation_reference" content="Brdar, S., Gavrić, K., Ćulibrk, D., &amp; Crnojević, V. (2016). Unveiling spatial epidemiology of HIV with mobile phone data. Scientific Reports, 6(1), 19342. doi:10.1038/srep19342" />
<meta name="citation_reference" content="Breiman, L. (2001). Random forests. Machine Learning, 45(1), 5–32. doi:10.1023/A:1010933404324" />
<meta name="citation_reference" content="Carver, S., Bevins, S. N., Lappin, M. R., Boydston, E. E., Lyren, L. M., Alldredge, M., … VandeWoude, S. (2016). Pathogen exposure varies widely among sympatric populations of wild and domestic felids across the United States. Ecological Applications, 26(2), 367–381." />
<meta name="citation_reference" content="Chipman, H. A., George, E. I., &amp; McCulloch, R. E. (2010). BART: Bayesian additive regression trees. The Annals of Applied Statistics, 4(1), 266–298. doi:10.1214/09-AOAS285" />
<meta name="citation_reference" content="Denison, D. G. T., &amp; Holmes, C. C. (2001). Bayesian partitioning for estimating disease risk. Biometrics, 57(1), 143–149. doi:10.1111/j.0006-341X.2001.00143.x" />
<meta name="citation_reference" content="Diez-Roux, A. V. (1998). Bringing context back into epidemiology: variables and fallacies in multilevel analysis. American Journal of Public Health, 88(2), 216–22. doi:10.2105/AJPH.88.2.216" />
<meta name="citation_reference" content="Elith, J., Leathwick, J. R., &amp; Hastie, T. (2008). A working guide to boosted regression trees. Journal of Animal Ecology, 77(4), 802–813. doi:10.1111/j.1365-2656.2008.01390.x" />
<meta name="citation_reference" content="Ezenwa, V. O. (2004). Host social behavior and parasitic infection: a multifactorial approach. Behavioral Ecology, 15(3), 446–454. doi:10.1093/beheco/arh028" />
<meta name="citation_reference" content="Fisher, A., Rudin, C., &amp; Dominici, F. (2018). Model class reliance: Variable importance measures for any machine learning model class, from the “Rashomon” Perspective. Retrieved from http://arxiv.org/abs/1801.01489" />
<meta name="citation_reference" content="Fountain-Jones, N., Packer, C., Jacquot, M., Blanchet, G., Terio, K., &amp; Craft, M. (in press). Endemic infection can shape exposure to novel pathogens: Pathogen co-occurrence networks in the Serengeti lions. Ecology Letters." />
<meta name="citation_reference" content="Friedman, J. H. (2002). Stochastic gradient boosting. Computational Statistics &amp; Data Analysis, 38(4), 367–378. doi:10.1016/S0167-9473(01)00065-2" />
<meta name="citation_reference" content="Friedman, J. H., &amp; Popescu, B. E. (2008). Predictive learning via rule ensembles. The Annals of Applied Statistics, 2(3), 916–954. doi:10.1214/07-AOAS148" />
<meta name="citation_reference" content="Gilbert, A. T., Fooks, A. R., Hayman, D. T. S., Horton, D. L., Müller, T., Plowright, R., … Rupprecht, C. E. (2013). Deciphering serology to understand the ecology of infectious diseases in wildlife. EcoHealth, 10(3), 298–313. doi:10.1007/s10393-013-0856-0" />
<meta name="citation_reference" content="Goldstein, A., Kapelner, A., Bleich, J., &amp; Pitkin, E. (2015). Peeking inside the black box: Visualizing statistical learning with plots of individual conditional expectation. Journal of Computational and Graphical Statistics, 24(1), 44–65. doi:10.1080/10618600.2014.907095" />
<meta name="citation_reference" content="Goldstein, E., Pitzer, V. E., O’Hagan, J. J., &amp; Lipsitch, M. (2017). Temporally varying relative risks for infectious diseases. Epidemiology. NIH Public Access. doi:10.1097/EDE.0000000000000571" />
<meta name="citation_reference" content="Han, B. A., Schmidt, J. P., Alexander, L. W., Bowden, S. E., Hayman, D. T. S., &amp; Drake, J. M. (2016). Undiscovered bat hosts of Filoviruses. PLOS Neglected Tropical Diseases, 10(7), e0004815. doi:10.1371/journal.pntd.0004815" />
<meta name="citation_reference" content="Ho, Y. C., &amp; Pepyne, D. L. (2002). Simple explanation of the no-free-lunch theorem and its implications. Journal of Optimization Theory and Applications, 115(3), 549–570. doi:10.1023/A:1021251113462" />
<meta name="citation_reference" content="Hofmann-Lehmann, R., Fehr, D., Grob, M., Elgizoli, M., Packer, C., Martenson, J. S., … Lutz, H. (1996). Prevalence of antibodies to feline parvovirus, calicivirus, herpesvirus, coronavirus, and immunodeficiency virus and of feline leukemia virus antigen and the interrelationship of these viral infections in free-ranging lions in East Africa. Clinical and Vaccine Immunology, 3(5), 554–562. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC170405/pdf/030554.pdf" />
<meta name="citation_reference" content="Hollings, T., Robinson, A., van Andel, M., Jewell, C., &amp; Burgman, M. (2017). Species distribution models: A comparison of statistical approaches for livestock and disease epidemics. PLOS ONE, 12(8), e0183626. doi:10.1371/journal.pone.0183626" />
<meta name="citation_reference" content="Kohavi, R., &amp; John, G. H. (1997). Wrappers for feature subset selection. Artificial Intelligence, 97(1-2), 273–324. doi:10.1016/S0004-3702(97)00043-X" />
<meta name="citation_reference" content="Kuhn, M. (2008). Building Predictive Models in R Using the caret Package. Journal of Statistical Software, 28(5), 1–26. doi:10.18637/jss.v028.i05" />
<meta name="citation_reference" content="Kursa, M. B., &amp; Rudnicki, W. R. (2010). Feature selection with the Boruta Package. Journal of Statistical Software, 36(11), 1–13. doi:10.18637/jss.v036.i11" />
<meta name="citation_reference" content="Lachish, S., Gopalaswamy, A. M., Knowles, S. C. L., &amp; Sheldon, B. C. (2012). Site-occupancy modelling as a novel framework for assessing test sensitivity and estimating wildlife disease prevalence from imperfect diagnostic tests. Methods in Ecology and Evolution, 3(2), 339–348. doi:10.1111/j.2041-210X.2011.00156.x" />
<meta name="citation_reference" content="Lundberg, S., &amp; Lee, S.-I. (2016). An unexpected unity among methods for interpreting model predictions. In NIPS 2016 Workshop on Interpretable Machine Learning in Complex Systems. Retrieved from http://arxiv.org/abs/1611.07478" />
<meta name="citation_reference" content="Machado, G., Mendoza, M. R., &amp; Corbellini, L. G. (2015). What variables are important in predicting bovine viral diarrhea virus? A random forest approach. Veterinary Research, 46(1), 85. doi:10.1186/s13567-015-0219-7" />
<meta name="citation_reference" content="Marmion, M., Parviainen, M., Luoto, M., Heikkinen, R. K., &amp; Thuiller, W. (2009). Evaluation of consensus methods in predictive species distribution modelling. Diversity and Distributions, 15(1), 59–69. doi:10.1111/j.1472-4642.2008.00491.x" />
<meta name="citation_reference" content="Molnar, C. (2018a). iml: An R package for Interpretable Machine Learning. Journal of Open Source Software, 3(26), 786. doi:10.21105/joss.00786" />
<meta name="citation_reference" content="Molnar, C. (2018b). Interpretable machine learning (Retrieved)." />
<meta name="citation_reference" content="Munson, L., Terio, K. A., Kock, R., Mlengeya, T., Roelke, M. E., Dubovi, E., … Packer, C. (2008). Climate extremes promote fatal co-infections during canine distemper epidemics in African lions. PLoS ONE, 3(6), e2545. doi:10.1371/journal.pone.0002545" />
<meta name="citation_reference" content="Nakagawa, S., &amp; Freckleton, R. P. (2008). Missing inaction: the dangers of ignoring missing data. Trends in Ecology &amp; Evolution, 23(11), 592–596. doi:10.1016/J.TREE.2008.06.014" />
<meta name="citation_reference" content="Nunn, C. L., Jordán, F., McCabe, C. M., Verdolin, J. L., &amp; Fewell, J. H. (2015). Infectious disease and group size: more than just a numbers game. Philosophical Transactions of the Royal Society of London B: Biological Sciences, 370(1669). Retrieved from http://rstb.royalsocietypublishing.org/content/370/1669/20140111.long" />
<meta name="citation_reference" content="Ogutu, J. O., Piepho, H.-P., &amp; Schulz-Streeck, T. (2011). A comparison of random forests, boosting and support vector machines for genomic selection. BMC Proceedings, 5(Suppl 3), S11. doi:10.1186/1753-6561-5-S3-S11" />
<meta name="citation_reference" content="Packer, C., Altizer, S., Appel, M., Brown, E., Martenson, J., O’Brien, S. J., … Lutz, H. (1999). Viruses of the Serengeti: patterns of infection and mortality in African lions. Journal of Animal Ecology, 68(6), 1161–1178." />
<meta name="citation_reference" content="Penone, C., Davidson, A. D., Shoemaker, K. T., Di Marco, M., Rondinini, C., Brooks, T. M., … Costa, G. C. (2014). Imputation of missing data in life-history trait datasets: which approach performs the best? Methods in Ecology and Evolution, 5(9), 961–970. doi:10.1111/2041-210X.12232" />
<meta name="citation_reference" content="Pollock, R. V, &amp; Coyne, M. J. (1993). Canine parvovirus. The Veterinary Clinics of North America. Small Animal Practice, 23(3), 555–68. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8389070" />
<meta name="citation_reference" content="Roelke-Parker, M. E., Munson, L., Packer, C., Kock, R., Cleaveland, S., Carpenter, M., … Appel, M. J. (1996). A canine distemper virus epidemic in Serengeti lions (Panthera leo). Nature, 379(6564), 441–445." />
<meta name="citation_reference" content="Schölkopf, B., &amp; Smola, A. J. (2002). Learning with kernels : support vector machines, regularization, optimization, and beyond. MIT Press. Retrieved from https://dl.acm.org/citation.cfm?id=559923" />
<meta name="citation_reference" content="Shapley, L. S. (1953). Stochastic games. Proceedings of the National Academy of Sciences of the United States of America, 39(10), 1095–100. doi:10.1073/PNAS.39.10.1095" />
<meta name="citation_reference" content="Stekhoven, D. J., &amp; Bühlmann, P. (2012). Missforest-Non-parametric missing value imputation for mixed-type data. Bioinformatics, 28(1), 112–118. doi:10.1093/bioinformatics/btr597" />
<meta name="citation_reference" content="Štrumbelj, E., &amp; Kononenko, I. (2014). Explaining prediction models and individual predictions with feature contributions. Knowledge and Information Systems, 41(3), 647–665. doi:10.1007/s10115-013-0679-x" />
<meta name="citation_reference" content="Tu, J. V. (1996). Advantages and disadvantages of using artificial neural networks versus logistic regression for predicting medical outcomes. Journal of Clinical Epidemiology, 49(11), 1225–1231. doi:10.1016/S0895-4356(96)00002-9" />
<meta name="citation_reference" content="Viana, M., Cleaveland, S., Matthiopoulos, J., Halliday, J., Packer, C., Craft, M. E., … Lembo, T. (2015). Dynamics of a morbillivirus at the domestic–wildlife interface: Canine distemper virus in domestic dogs and lions. Proceedings of the National Academy of Sciences of the United States of America, 112(5), 1464–1469. doi:10.1073/pnas.1411623112" />
<meta name="twitter:title" content="How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="1.  Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common.

2.  Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus.

3.  Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years.

4.  When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns." />
<meta name="og-title" property="og:title" content="How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/569012v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="1.  Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common.

2.  Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus.

3.  Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years.

4.  When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-03-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/569012v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/569012v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/569012v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/569012v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/217895" />
    <title>How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0GyQOajXv-wV7_TkjudxrTiw1p3Ztw4kTsBmaA5c4ag","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"217895","apath":"\/biorxiv\/early\/2019\/03\/06\/569012.atom","pisa":"biorxiv;569012v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;569012v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/569012v1","title":"How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure","identifier":"node\/217895"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TqGjMZjMRrxf\\\\\\\/ZHgXdjvbFExDey7oMq58JrTAhP4ssWHryc+VXv5GCP17Nm3ktIg\\\\\\\/lQ71s935+H435RUUswEaoWxG7odDj0HLcCwiCslQ88phL3DNZq+A59gb7CqTnB\\\\\\\/EFI5\\\\\\\/RkPOHofLTlR0pQ9er1CfbsGWTXuuiETFu2wL82N+vKSYJ4XPX7VJSVtm1zAVsQo8eP6giE2OyKZyOA4dEduJy0b7z0dc09FPXVdyzuITxqUuvrv24bF1sE66j0PMChlUjukkBB30xRSICjXPtIwalmB8THywyNbN88zGmXKs2bnJNHlMkTmfBQe2S3+Ri3EKCzI974DAmVGyLdjqe2UFJgvcsWLdl6Z\\\\\\\/Q7+ZpklNABuA3vF3WNZcwe0d9ZTknQAWh8Mox4T0vwFcm5uwB+8zPUK3sI5BD1i+FkSaFvLtGZR3zOkTs\\\\\\\/DDODDBtAoaLfQKVy+XqHGiwDB+9w4vJeZ5\\\\\\\/uThn7GjMjvRIcNBjNha6bYt1\\\\\\\/v3r+9XeW\\\\\\\/\\\\\\\/NPoRjcc5rUsdtQGgUAKQBKlZc6psXDKNYhNSgvdGxj3Z7w16mY462pxQFmPgFbBvZ0sGz6jHKclz\\\\\\\/zptM5AK0V+HlXMB4+1D2uBuG5CVLqhnohkwt6KOr5Qc4CRUKbdVAlPLihVnw19vzccB9PF3jfQzb0f0ffGkcYP7rLMttEGaemPHi7b+OH5ZztPF87BbSPRjKuEYPXN6ASGv\\\\\\\/LSmMrDKHOjs4ukECfXNyWIPj4eCV9yPdfsTth0X6zGuQe4Vaj4OU9iNEcp+pt7EzZHHGx7KHJA+E4fjQoEi2wYnd1aLCsng6TUI2pGg55Gc\\\\\\\/r4z8Y3i34pVQ4WYdF4UDJcyVyazhUc4mvtO6CqJ7UhsQUuRiQDptfmyCUcLO0nsYB6TKyNmCUVS3WskJJ+Zb7XjCYKNUIJPNDjKHtZ4ngkErkbJuWRsNn5x4001I1sJvmYHD7zTpIMo6rFvkCYv\\\\\\\/uJwsIumOmKTC7c\\\\\\\/USD8yFTMVUhQ4REiiZ7XoDnuxmzmHsyctvEdQbBN7DiN3uoZJ9LLJtaFQEm+Sq72gjE73ojc97PbG6rM5eTiPXaktSOJIKUMBYyHQvar6fNGWdMjouOu5+y9269brQOyplJupo1mPvF5ywKgoXBWn6OMdBIfMhNfZJPbL84ViThzPHws79ZQP4dDA4dnlXlwTb6M3eBgdkfYYWjqKbGm0wTquJqMACiyXdNfGqDPuMh8R46tHpqxm2Xf7fcWBJ11\\\\\\\/yz8Pjjs3W8J6lLMtgq8pUwyzyUBrq9i6ewTex+mF9STI+kcJyE6mOiYNVv961nL5Jd4zTpWp37l3s=\\\u0022,\\\u0022iv\\\u0022:\\\u0022n7taPHKBbY8DHsuZ3YFXDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220ec22ba232521e33dbd602fa4724ba1b448e675fb6c101f0fa3d572ab851005b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/569012v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-217895 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/569012v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CSBwvqsFbDZleSjUGN_amQmk6TOaSbp3ZXfKHhzWF-k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="217895" id="node217895" data-pisa="biorxiv;569012v1" data-pisa-master="biorxiv;569012" data-apath="/biorxiv/early/2019/03/06/569012.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Fountain-Jones</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gustavo</span> <span class="nlm-surname">Machado</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott</span> <span class="nlm-surname">Carver</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Packer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mariana</span> <span class="nlm-surname">Recamonde-Mendoza</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Meggan E.</span> <span class="nlm-surname">Craft</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/569012 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node217895" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas M. Fountain-Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Veterinary Population Medicine, University of Minnesota</span>, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BM.%2BFountain-Jones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fountain-Jones%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BM.%2BFountain-Jones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gustavo Machado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Population Health and Pathobiology, North Carolina State University</span>, North Carolina, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2BMachado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Machado%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2BMachado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Scott Carver </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biological Sciences, University of Tasmania</span>, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BCarver%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carver%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BCarver%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Craig Packer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Ecology Evolution and Behavior, University of Minnesota</span>, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BPacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Packer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BPacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mariana Recamonde-Mendoza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Institute of Informatics, Universidade Federal do Rio Grande do Sul</span>, <span class='nlm-country'>Brazil</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mariana%2BRecamonde-Mendoza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Recamonde-Mendoza%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMariana%2BRecamonde-Mendoza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Meggan E. Craft </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Veterinary Population Medicine, University of Minnesota</span>, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meggan%2BE.%2BCraft%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Craft%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeggan%2BE.%2BCraft%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/569012v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/569012v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/569012v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/569012v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/569012v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/569012v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:217895" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:217895/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-4">Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common.</p></li><li id="list-item-2"><p id="p-5">Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus.</p></li><li id="list-item-3"><p id="p-6">Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years.</p></li><li id="list-item-4"><p id="p-7">When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns.</p></li></ol></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-8">An individual’s risk of infection by a pathogen is dependent upon a wide variety of host and pathogen traits that are often complicated by heterogeneities in landscapes and individual behavior (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altizer et al., 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carver et al., 2016</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ezenwa, 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nunn, Jordán, McCabe, Verdolin, &amp; Fewell, 2015</a>). Evidence of infection in animals is often determined by serology, which can determine if a host has likely been infected (or not) sometime in the past (‘exposed’ vs ‘non-exposed’ host, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gilbert et al 2013</a>). Models that attempt to quantify exposure risk often assume exposed and non-exposed hosts can be linearly separable by a predictor even though non-linear relationships may be more biologically plausible, e.g., where exposure risks reach an asymptote by a certain age (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Denison &amp; Holmes, 2001</a>). Further, exposure risk may be dependent on interactions between predictors (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Diez-Roux, 1998</a>) (e.g., exposure risk is only elevated in the youngest and oldest individuals in one subset of the population). Generalized linear models are not effective in capturing such non-linear responses and complex interactions, which is an important advantage of most machine-learning algorithms (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elith, Leathwick, &amp; Hastie, 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tu, 1996</a>).</p><p id="p-9">Machine learning methods offer a powerful and flexible alternative to explore important patterns in exposure data, often with substantially better predictive performance (e.g., <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Elith et al., 2008</a>). Machine learning models can include thousands of predictors without overfitting and can analyze either categorical or continuous data, even in datasets containing large numbers of missing values. However, choosing the most appropriate machine learning model can be challenging, thus the development of methods for identifying the best-performing model has become an important area of research over the last 15 years (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ho &amp; Pepyne, 2002</a>). Supervised machine learning models, such as support vector machines (SVMs), or tree-based algorithms, such as random forests (RF), gradient boosting models (GBMs), or boosted regression trees (BRTs), operate in markedly different ways, potentially leading to different performance and results (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Machado, Mendoza, &amp; Corbellini, 2015</a>). While machine learning models are increasingly popular in disease ecology and epidemiology (e.g., <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Babayan, Orton, &amp; Streicker, 2018</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Han et al., 2016</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hollings et al., 2017</a>), they have not yet been widely adopted.</p><p id="p-10">In part, the limited application of machine learning models to disease ecology is due to the perception of these models as black boxes. Recent advances have helped unveil the black box by clarifying the predictive capabilities of these models (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brdar, Gavrić, Ćulibrk, &amp; Crnojević, 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lundberg &amp; Lee, 2016</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>). For example, coalitional game theory can be used to untangle how variables contribute to a machine learning model prediction (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Štrumbelj &amp; Kononenko, 2014</a>), providing an explanation for why an individual host was predicted to be positive or negative for pathogen exposure.</p><p id="p-11">The conditions of when and where an individual was infected by a pathogen are often unknown (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gilbert et al., 2013</a>). Exposure risk is often quantified by analyzing variables such as age or group size or landscape context based on the date the individual was sampled, even though the start of the infection was likely to have occurred sometime in the past (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Carver et al., 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>). Further, risk factors of pathogen exposure are often assumed to be fixed in time, even though the ecological context could change between exposure and time of sampling (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goldstein et al., 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Munson et al., 2008</a>). The spatio-temporal mismatch between the location and date of infection and the location and date of sampling is well known (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gilbert et al., 2013</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Goldstein et al., 2017</a>), yet few pathogen risk models have explicitly considered these limitations.</p><p id="p-12">Here we introduce a multi-algorithm machine learning ensemble pipeline that incorporates recent advances in data science to better predict pathogen exposure risk. Our pipeline is flexible and many of the post-processing methods are model agnostic in that they can be used to interrogate any regression or classification model. To demonstrate the utility of our approach, we model pathogen exposure risk in two different viruses that infect African lions (<em>Panthera leo</em>): canine distemper virus (CDV) and feline parvovirus (parvovirus). To build our exposure risk models, we leverage age estimates, long term behavioural observations, and environmental data from the Serengeti Lion Project (SLP) as predictors (hereafter called ‘features’ in line with computer science terminology). Using this data, we develop exposure risk models based on features calibrated to reflect potential date of exposure (rather that sampling date), and also compare exposure risk factors between the two pathogens. While we focus on pathogens in wildlife populations, our pipeline could be applied in a similar way to human, domestic animal, or livestock exposure data, as well as to any classification or regression problem. We provide code and a fully worked example in a vignette (Text S1).</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><p id="p-13">If exposure data has been collected across multiple years and there are reliable estimates of host age, researchers can take advantage of our calibration steps below. If not, researchers can proceed to <em>2.2 Pre-processing</em>.</p><div id="sec-3" class="subsection"><h3>2.1 Data calibration</h3><p id="p-14">Plotting prevalence relationships across host age and time is an essential first step to start calibrating exposure models. When evaluating year-prevalence curves, spikes in prevalence in certain years may estimate the timing of likely epidemics, particularly when the year-prevalence curves of juveniles are considered (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>)(<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>). Knowledge of pathogen natural history (e.g., chronic vs acute infection) and tests such as chi-squared testing for temporal fluctuations can be used to support these estimates (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>., <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>). In populations with more precise age estimates beyond juvenile/adult classes, age of likely exposure can be calculated based on how old the individual was when the last epidemic occurred. All other features can be calibrated to the year of exposure (e.g., what was the group size for that individual for the potential year of exposure). Some host characteristics such as sex do not change with time. If the analysis uses serological evidence, detection of exposure is often based on a cut-off threshold (i.e., above a certain titre an individual is classified as exposed). This is imperfect as antibody titre tends to diminish over time (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Gilbert et al., 2013</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>), therefore adding ‘time since epidemic’ as a feature can account for imperfect detection. Accounting for imperfect detection can allow for more accurate measures of infection risk in disease surveys (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lachish, Gopalaswamy, Knowles, &amp; Sheldon, 2012</a>)</p><p id="p-15">Age-prevalence relationships can also help calibrate possible exposure times for pathogens more endemic, or constantly present, in a population (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). If prevalence peaks at an early age and is stable over time, this may be an indication that the pathogen is more endemic in the population; the relevant features can be calibrated to reflect that the individual was likely exposed early in life (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>). Age-specific force of infection models (FOI, or the per capita rate at which susceptible individuals acquire infection) can help support these estimates (i.e., at what age is the FOI highest; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson &amp; May, 1985</a>). If prevalence increases in a linear fashion with age and the FOI is equal across age groups (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a>), gaining a reliable estimate of the year of exposure (and therefore calibration, as described below) is difficult.</p><p id="p-16">Depending on the interpretation of the age- and year-prevalence curves, temporal features can be estimated, or calibrated, including age exposed, time since last epidemic, last epidemic year, year exposed, and age sampled (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flow chart showing the pre-processing steps in our machine learning pipeline. Each colour represents a separate pathogen. FOI: Force of infection. *: indicates optional step." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-387130837" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flow chart showing the pre-processing steps in our machine learning pipeline. Each colour represents a separate pathogen. FOI: Force of infection. *: indicates optional step.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig.1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F1.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig.1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F1.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig.1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig.1:</span> <p id="p-17" class="first-child">Flow chart showing the pre-processing steps in our machine learning pipeline. Each colour represents a separate pathogen. FOI: Force of infection. *: indicates optional step.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>2.2 Pre-processing</h3><p id="p-18">It is important to account for missing data either by imputation or removal prior to model construction (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Some machine learning algorithms, such as gradient boosting, bin missing data as a separate node in the decision tree (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Friedman, 2002</a>, Fig. S1), however other algorithms such as SVM are less flexible. In order to compare predictive performance across models, missing data can either be imputed or removed from the dataset. Although providing specific advice on whether to include missing data or not is outside the scope of this paper (see <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nakagawa &amp; Freckleton, 2008</a>), we provide an option if imputation is suitable for the study problem. We integrated the ‘missForest’(<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Stekhoven &amp; Bühlmann, 2012</a>) machine-learning imputation routine (using the RF algorithm) into our pipeline, as it has been found to have low error rates with ecological data (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Penone et al., 2014</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flow chart showing the steps in our machine learning pipeline. Vertical text in bold indicates how the chart relates to sections of the main text and vignette (Text S1, sections 1-3 respectively). TP: True positive, FP: False positive, FN: False negative, TN: True negative. Yellow boxes indicate which data split is being tested in that particular &#x2018;fold&#x2019;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-387130837" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flow chart showing the steps in our machine learning pipeline. Vertical text in bold indicates how the chart relates to sections of the main text and vignette (Text S1, sections 1-3 respectively). TP: True positive, FP: False positive, FN: False negative, TN: True negative. Yellow boxes indicate which data split is being tested in that particular &#x2018;fold&#x2019;.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F2.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F2.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2:</span> <p id="p-19" class="first-child">Flow chart showing the steps in our machine learning pipeline. Vertical text in bold indicates how the chart relates to sections of the main text and vignette (Text S1, <a id="xref-sec-1-1" class="xref-sec" href="#sec-1">sections 1</a>-<a id="xref-sec-8-1" class="xref-sec" href="#sec-8">3</a> respectively). TP: True positive, FP: False positive, FN: False negative, TN: True negative. Yellow boxes indicate which data split is being tested in that particular ‘fold’.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Another important step before constructing a model, particularly for relatively small data sets with large numbers of features, is to select features that are relevant and informative for prediction. Reducing the feature set used in models not only increases the efficiency of the algorithms, but also improves the accuracy of many machine learning algorithms (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kohavi &amp; John, 1997</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kursa &amp; Rudnicki, 2010</a>). In our pipeline, we use the ‘Boruta’ (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kursa &amp; Rudnicki, 2010</a>) feature selection algorithm for reducing the feature set to just those relevant for prediction prior to model building. The Boruta routine has been found to be the one of the most powerful approaches to select relevant features (Degenhardt, Seifert, &amp; Szymczak, 2017) and does so by measuring the importance of each variable using an RF-based algorithm (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Kursa &amp; Rudnicki, 2010</a>).</p></div><div id="sec-5" class="subsection"><h3>2.3 Model training</h3><p id="p-21">We incorporated an internal repeated 10-fold cross-validation (CV) process to estimate model performance. CV can help prevent overfitting and artificial inflation of accuracy due to use of the same data for training and validation steps (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). To run and evaluate each model, our pipeline uses the ‘caret’ (classification and regression training) package in R (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kuhn, 2008</a>). Not only does this package provide a streamlined approach to tuning parameters for a wide variety of models, including machine learning models, it also offers functions that directly enable robust comparisons of model performance. The ‘train’ function uses resampling to evaluate how tuning parameters such as learning rate (see <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Elith et al., 2008</a>) can impact model performance and ‘chooses’ the optimal model with the highest performance via the confusion matrix (e.g., sensitivity and specificity for classification models). Another advantage of this package is that it can perform classification or regression using 237 different types of models from generalized linear models (GLMs such as logistic regression) to complex machine learning and Bayesian models using a standardized approach (see <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kuhn, 2008</a> for a complete list of models).</p><p id="p-22">In our pipeline, we compare supervised machine learning algorithms (RF, SVM, and GBM) as well as logistic regression. These models are among the most popular and best tested machine learning methods, but all operate in different ways, and this can in turn can impact predictive performance as measured by AUC (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Marmion, et al., 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ogutu, Piepho, &amp; Schulz-Streeck, 2011</a>). In brief, for classification problems, SVMs use vectors and kernel functions that maximizes the margin between classes of data on a hyperplane (see <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schölkopf &amp; Smola, 2002</a> for model details). In contrast, RF and GBM are tree-based algorithms that iteratively split data into increasingly pure ‘sets’, with the main difference being that GBM fits trees sequentially with each new tree helping correct errors from the previous (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Friedman, 2002</a>). For RF, the trees are fit independently with each iteration (see Fig. S1 for a more detailed comparison).</p><p id="p-23">Imbalanced proportions of the outcome of interest are common in disease ecology, where for example, prevalence of the pathogen is &lt; 50%; imbalanced proportions can influence the predictive performance of the algorithm (Haibo He &amp; Garcia, 2009). To address this issue, we included a down-sampling strategy in our pipeline that can be used to compare model performance for datasets with imbalanced data (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>).</p></div><div id="sec-6" class="subsection"><h3>2.4 Interpreting the model</h3><p id="p-24">Once the best model is selected, we provide a variety of tool sets to visualize and interrogate model results (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2</a>) based on the ‘iml’ (interpretable machine learning) R package (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Molnar, 2018a</a>). Measures of variable importance are often used to understand which features contributed to the model in ranked order of importance. However, comparing variable importance across multiple types of models can be challenging as models often quantify variable importance in different ways. Further, in some cases, different models can have the same predictive performance but rely on different features or the ‘Rashomon’ effect (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Breiman, 2001</a>). Here, to help overcome these limitations, our pipeline presents a model agnostic method called ‘model class reliance’ (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fisher, Rudin, &amp; Dominici, 2018</a>) as an extension of Breiman’s (2001) permutation approach. In essence, the algorithm quantifies the expected loss in predictive performance (i.e., how well the model classifies exposed vs non-exposed individuals) for a pair of observations compared to the full model in which the particular feature has been switched (see <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fisher et al., 2018</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Molnar, 2018b</a> for more details). Because this algorithm compares the ratio of error rather than the differences in model error rates it can be used to compare different models. A feature is not considered important if switching it does not alter model performance. Further, this method is probabilistic and has the advantage that feature permutation automatically includes feature interaction effects into account into the importance calculation (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>).</p><p id="p-25">Our pipeline also provides the appropriate code to calculate and plot both partial dependence plots (PD) and individual conditional expectation (ICE) to examine the marginal effect of a particular feature on the response (Goldstein et. al., 2015). PD plots visualize the global (i.e., average model) effect of a feature on the response, whilst ICE plots examine the effect of each feature for each observation, whilst holding all other features constant (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goldstein et al., 2015</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>). For example, to assess the effect of a feature on exposure risk of individual ‘A’, the ICE approach would vary the feature across a grid of values. For each feature variant, the selected machine learning model is used to make new predictions of exposure risk for individual A. Then a line is fit to the predictions across feature values and then the process is repeated for each individual observation. The PD plot represents the average of the lines an ICE plot (e.g., yellow line is the average whereas the black lines are the individual observations in the bottom panel of <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2</a>). Combined they overcome limitations of each approach separately as PD plots alone obscure when interactions shape a feature-response relationship, whilst ICE plots can be challenging to read without plotting the average effect (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>). We recommend ICE plots centred on the minimum feature value (cICE) in order to more easily visualize when ICE estimates vary between observations (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Molnar, 2018a</a>).</p><p id="p-26">To further interrogate interactions in each model, our pipeline provides the option to calculate Friedman’s H-statistic (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Friedman &amp; Popescu, 2008</a>) for all features (and combinations thereof) in the model. The strongest interactions can then be visualized using 3D PD plots or heatmaps (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2</a>). In brief, Friedman’s H-statistic is a flexible way to quantify feature interaction strength using partial dependency decomposition and represents the portion of variance explained by the interaction (see <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Friedman &amp; Popescu, 2008</a> for details). Whilst computationally expensive, this method is also model agnostic and can be upscaled to quantify high order interactions, i.e., three- or four-way interactions between features (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>).</p><p id="p-27">Finally, our pipeline provides functions to calculate Shapely values (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shapley, 1953</a>) to assess how individual features effect the prediction of single observations (i.e., an individual lion). This approach calculates Shapely values from the final model that assign ‘payouts’ to ‘players’ depending on their contribution towards the prediction (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brdar et al., 2016</a>; <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Štrumbelj &amp; Kononenko, 2014</a>). The players cooperate with each other and obtain a certain ‘gain’ from that cooperation. In this case, the ‘payout’ is the model prediction, ‘players’ are the feature values of the observation (e.g., host sex) and the ‘gain’ is the model prediction for this observation (e.g., was the host exposed or not) minus the average prediction of all observations (on average what is probability of a host being exposed) (<a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Molnar, 2018b</a>). More specifically, Shapely values, defined by a value function (<em>Va</em>), compute feature effects (<em>ϕ</em><sub><em>ij</em></sub>) for any predictive model <em>f</em><sup>(<em>i</em>)</sup>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/06/569012/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/06/569012/embed/graphic-3.gif"/></noscript></span></span></p><p id="p-28">Where <em>S</em> is a subset of the features (or players), <em>x</em> is the vector features values for observation <em>i</em> and <em>p</em> is the number of features. <em>Va</em><sub><em>xi</em></sub> is the prediction of feature values in <em>S</em> marginalized by features not in <em>S</em>:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/06/569012/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/06/569012/embed/graphic-4.gif"/></noscript></span></span></p></div><div id="sec-7" class="subsection"><h3>2.5 Modelling lion exposure risk</h3><p id="p-29">Blood samples were collected from 300 individual lions of known age (estimated to the month) in the Serengeti Ecosystem, Tanzania between 1984 and 1994. Of these individuals, 40% were seropositive for CDV and 30% for parvovirus (see <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hofmann-Lehmann et al., 1996</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Roelke-Parker et al., 1996</a> for method details). Both viruses can infect multiple species, yet CDV is directly transmitted via aerosol, whereas parvovirus can be either transmitted directly or through environmental contamination. CDV and parvovirus are considered to have epidemic cycles in the Serengeti lion population (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hofmann-Lehmann et al., 1996</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>). Chi-square tests of year-prevalence relationships supported three epidemic years for both viruses: ~1977, 1981 and 1994 for CDV and 1977, 1985 and 1992 for parvovirus (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>). Subsequent Bayesian state-space analysis of this data further confirmed these years as epidemic years for CDV and parvovirus (Behdenna et al., <em>in press</em>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Viana et al., 2015</a>).</p><p id="p-30">We selected 17 features and calibrated them where appropriate (as detailed in <em>2.1 Data calibration</em>). See Text S2 for feature selection details. Each model, including logistic regression, was performed following the steps outlined in our pipeline. We compared the predictive performance of the models using calibrated and uncalibrated feature sets (hereafter calibrated or uncalibrated models) to assess how differences in calibration could change the exposure risk predictions. Uncalibrated feature sets were calculated based on the date an individual was sampled, rather than going through the process outlined in <em>2.1 Data calibration</em>.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Machine learning models with calibrated feauture sets have higher predictive performance</h3><p id="p-31">For both CDV and parvovirus, machine learning models had higher predictive performance (higher AUC) compared to logistic regression models (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For example, predictions of parvovirus were only just better than random using logistic regression model (AUC = 0.54), whereas the GBM model was correctly predicting exposure 73% of the time (AUC = 72.98). Using our calibration approach further improved the overall predictive performance of each pathogen by increasing the sensitivity of the models (i.e., they more able to correctly identify positives), however, there was a trade-off with reduced specificity. For example, our calibrated CDV model had a 7% increase in AUC with an 23% increase in sensitivity but 18% decrease in specificity compared to the uncalibrated model (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The features considered relevant in each model were relatively consistent (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). However, there were some important exceptions with, for example, sex only relevant for parvovirus exposure in the calibrated model. In contrast, rainfall was relevant for predicting parvovirus exposure in the uncalibrated model but not in the calibrated model.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/243001/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/243001/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/243001" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-32" class="first-child">Model performance and relevant features for each model. If a feature was not relevant in any model it was excluded using the Boruta algorithm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Model calibration alters feature-risk relationships</h3><p id="p-33">Age and rainfall were the most important features predicting CDV exposure, but both features were relatively less important in the calibrated models (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Even though the features associated with exposure risk in each model were broadly similar for both pathogens, the relationships between each feature and exposure risk varied. Partial dependency plots showed that risk of CDV increased relatively linearly across age classes in in the uncalibrated model (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b</a>), whereas in the calibrated model exposure risk was much more constant across age classes with an increase in risk in individuals between 1-2 y.o. (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3f</a>). Rainfall also showed different relationships in each model with reduced exposure risk when the average monthly rainfall &gt; 40 mm in the age calibrated model (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3c</a>). There was a much shallower decline in CDV risk associated with rainfall in the calibrated model (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3e</a>) compared to the uncalibrated model (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3c</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots showing the differences in model predictions and the features that contribute to CDV exposure risk in the Serengeti lions for the uncalibrated models (a-c) compared to the calibrated models (d-f). Age sampled followed by rainfall are the most important features associated with CDV exposure risk in the uncalibrated models (a). In contrast, rainfall followed by age exposed were the most important features in the calibrated model (d). However, cICE plots show that the relationship with exposure risk differs substantially (b &amp; c: uncalibrated model, e &amp; f: calibrated model). Feature name colours reflect feature type (orange = individual, blue = pride-level, red = environmental) and * indicates that the variable was averaged across the pride&#x2019;s territory. See Text S2 for further details including sample size. Tick marks on the x-axes (b, c, e &amp; f) show the distribution of data. Yellow/red line indicates the average value across all individuals. The y-axes reflect probability of being positive for CDV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-387130837" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plots showing the differences in model predictions and the features that contribute to CDV exposure risk in the Serengeti lions for the uncalibrated models (a-c) compared to the calibrated models (d-f). Age sampled followed by rainfall are the most important features associated with CDV exposure risk in the uncalibrated models (a). In contrast, rainfall followed by age exposed were the most important features in the calibrated model (d). However, cICE plots show that the relationship with exposure risk differs substantially (b &amp; c: uncalibrated model, e &amp; f: calibrated model). Feature name colours reflect feature type (orange = individual, blue = pride-level, red = environmental) and * indicates that the variable was averaged across the pride&#x2019;s territory. See Text S2 for further details including sample size. Tick marks on the x-axes (b, c, e &amp; f) show the distribution of data. Yellow/red line indicates the average value across all individuals. The y-axes reflect probability of being positive for CDV.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F3.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F3.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3:</span> <p id="p-34" class="first-child">Plots showing the differences in model predictions and the features that contribute to CDV exposure risk in the Serengeti lions for the uncalibrated models (a-c) compared to the calibrated models (d-f). Age sampled followed by rainfall are the most important features associated with CDV exposure risk in the uncalibrated models (a). In contrast, rainfall followed by age exposed were the most important features in the calibrated model (d). However, cICE plots show that the relationship with exposure risk differs substantially (b &amp; c: uncalibrated model, e &amp; f: calibrated model). Feature name colours reflect feature type (orange = individual, blue = pride-level, red = environmental) and * indicates that the variable was averaged across the pride’s territory. See Text S2 for further details including sample size. Tick marks on the x-axes (b, c, e &amp; f) show the distribution of data. Yellow/red line indicates the average value across all individuals. The y-axes reflect probability of being positive for CDV.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Similar to CDV, age sampled followed by rainfall were the most important features associated with parvovirus exposure risk in the uncalibrated models (Fig. S4a). Parvovirus exposure risk slightly increased across age classes in the uncalibrated models, however in the calibrated models exposure risk increased rapidly at early ages (0-1), but then was relatively constant across ages &gt;3 (Fig. S4b). The signature of rainfall on parvovirus risk in the uncalibrated model was remarkably like that of CDV with a large drop in risk when the monthly rainfall was &gt; 40 mm a month (Fig. S4c). However, rainfall was much less important in the calibrated model (Fig. S4d). Strikingly epidemic year was important in the calibrated model with exposure risk much higher for animals likely exposed in the 1992 epidemic (Fig. S4f).</p></div><div id="sec-11" class="subsection"><h3>Interactions</h3><p id="p-36">We further interrogated the calibrated models to visualize how interactions between features could be important for exposure risk of both pathogens. We focussed on interactions with epidemic year, as we were interested to see if exposure risk could vary with each outbreak (see Fig. S5 for a summary of all interactions detected). For CDV, the strongest interaction with epidemic year was age exposed (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>). Even though exposure risk was predicted to be reasonably similar in each CDV outbreak (Fig. S6), exposure risk was higher for lions 2-5 y.o. in the 1994 epidemic compared to the 1981 epidemic (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b</a>). For parvovirus, territory overlap had the strongest interaction with epidemic year (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4c</a>) with individuals in prides with low overlap having an increased risk of parvovirus during the 1992 epidemic only. There was also an interaction with epidemic year and age exposed with lions aged 10-12 y.o. more likely to be exposed to parvovirus during the 1992 epidemic (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4d</a>). See Text S3 and Fig. S7 for a description of how individual features effects the prediction of individual lion’s exposure risk.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interactions between epidemic year and the other features in shaping exposure risk of CDV (a-b) and parvovirus (c-d) in the Serengeti lions. Differing coloured partial dependence curves (b &amp; d) represent different epidemics. The y axis (y hat) is the relative risk of being exposed (with all other feature combinations marginalized). See Fig. S5 for the interactions detected for other features." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-387130837" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Interactions between epidemic year and the other features in shaping exposure risk of CDV (a-b) and parvovirus (c-d) in the Serengeti lions. Differing coloured partial dependence curves (b &amp; d) represent different epidemics. The y axis (y hat) is the relative risk of being exposed (with all other feature combinations marginalized). See Fig. S5 for the interactions detected for other features.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F4.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F4.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/06/569012/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4:</span> <p id="p-37" class="first-child">Interactions between epidemic year and the other features in shaping exposure risk of CDV (a-b) and parvovirus (c-d) in the Serengeti lions. Differing coloured partial dependence curves (b &amp; d) represent different epidemics. The y axis (y hat) is the relative risk of being exposed (with all other feature combinations marginalized). See Fig. S5 for the interactions detected for other features.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-12"><h2 class="">Discussion</h2><p id="p-38">Our machine learning pipeline offers a powerful approach that can enable researchers to generate robust exposure risk models and interrogate models in new ways. We found that models, where host and environmental features were calibrated based on likely exposure year, had higher predictive power compared to uncalibrated models (i.e., comprised of features based on sample date). The calibrated models not only provided better predictive power but also more robust insights into pathogen dynamics in a wild population that can aid future surveillance. Furthermore, use of this pipeline is not restricted to research on pathogen exposure, but also could be employed with minimal change to model other ecological systems.</p><div id="sec-13" class="subsection"><h3>Robust insights in exposure risk for CDV and parvovirus</h3><p id="p-39">The machine learning algorithms employed in our pipeline had higher predictive performance compared to the logistic regression models because they were able to quantify non-linear relationships in our data. For both pathogens, non-linear relationships between host and environmental features were common and provided new perspectives of pathogen dynamics in this lion population. For example, in our calibrated model of parvovirus, risk was saturated when individuals &gt; 3 y.o but decreased after age 7 (Fig. S4d-e). Puppies are thought to be the highest risk category for canine parvovirus (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pollock &amp; Coyne, 1993</a>) and, based on our calibrated models, this may be true for feline parvovirus in lions as well. Even though this pattern is relatively intuitive, neither the logistic regression (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), the uncalibrated model (Fig. S4a-b), or age-prevalence curves (Fountain-Jones et al., <em>in press</em>) captured this pattern, highlighting the need for these types of models to quantify exposure risk.</p><p id="p-40">Calibrating features provided not only a more nuanced interpretation of pathogen dynamics but also higher predictive performance. The greater predictive performance could be because we removed individuals that were not on the landscape during the inter-epidemic years, and thus were negative for the pathogen, not due a behavioral or landscape trait, but due to the cyclic nature of epidemics. As exposure risk changed over time, including ‘epidemic year’ in the models may have been responsible for the greater predictive power of the calibrated models. Further, even though it was not relevant in the best performing CDV and parvovirus models, including ‘time since exposure’ as a feature provides support that the observed relationships were less likely to be due to differences in detectability due to titre decline over time(<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Packer et al., 1999</a>).</p><p id="p-41">Another advantage of the machine learning methods is the ability to model the non-linear interactions shaping exposure risk in the lions. In our case we focused on how host and environmental characteristics interacted with epidemic year to better understand the temporal dimension of exposure risk: an important question rarely tested in wildlife populations (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Gilbert et al., 2013</a>). For parvovirus, in particular, the risk associated with lion age varied between epidemics with the 1992 epidemic standing out with increased risk in older as well as younger individuals (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a>). For CDV in contrast, there were remarkably similar risk factors for the 1981 and 1994 epidemics, except for an increase in risk in juveniles in 1994 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4b</a>). However, there was no known increase in mortality in 1981 as there was in the 1994, further supporting the idea that co-infection with <em>Babesia</em> was the principal agent driving mortality (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Munson et al., 2008</a>). When and why exposure risk is stable or dynamic across epidemic years is an open question in disease ecology, and analysis pipelines such as ours can help address this gap.</p><p id="p-42">Further, calculating Shapely values from predictive models can unpack what each model means in terms of exposure risk in an intuitive way that can be used to inform practitioners and guide surveillance and forecast future risk. For example, when rainfall is below average individuals from prides with relatively high overlap are more at risk of being exposed to CDV perhaps due to increased chances of congregating at waterholes. The sex and age of individuals are less important when choosing which individuals to test or vaccinate.</p></div><div id="sec-14" class="subsection"><h3>Caveats and future directions</h3><p id="p-43">Even though serological or qPCR evidence is often used (including here) as evidence of past or current infection, there are numerous limitations associated with this type of data that need to be considered (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Gilbert et al., 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lachish et al., 2012</a>). Further, for pathogens that cause high morality, the individuals sampled represent survivors of the epidemic, and this may bias exposure risk models for wildlife populations that are infrequently monitored. For example, if males experienced high mortality in a particular epidemic compared to females, post epidemic sampling of this population may incorrectly infer that females were more likely to be exposed. How to incorporate biases such as these into exposure risk models could be a fruitful future direction for research.</p><p id="p-44">A more general limitation of this approach is that uncertainty in model predictions for the machine learning algorithms we used is not well characterized. Bayesian additive regression trees (BART) may be a promising technique that could fill this gap, but this technique comes at a computational cost that may be prohibitive for larger datasets (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chipman, George, &amp; McCulloch, 2010</a>). Furthermore, none of the methods we employed in our pipeline explicitly account for spatial relationships which could be a limitation for studies that use data across a larger geographic area where spatial autocorrelation could be an important problem.</p><p id="p-45">Lastly this analysis pipeline could be used for a variety of ecological applications including both classification and regression problems with continuous or categorical response variables. For example, our pipeline could easily be incorporated into species distribution models. Moreover, as the R package ‘caret’ is highly flexible and can perform and compare a large number of algorithms with minimal change to the R syntax, the end user could easily test any model of their choosing in a similar way (e.g. generalized additive models or discriminant analysis). Similarly, the code we provide from the R package ‘iml’ is model agnostic and can provide an increase in the interpretability for many regression or classification models (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Molnar, 2018a</a>).</p></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-46">Our analysis pipeline offers a flexible unified approach to not only better understand pathogen exposure risk, but also could be applied to any system where non-linear responses and interactions are important in shaping a response variable. This study has highlighted the value of using advances in data science coupled with feature calibration to increase our understanding of how host and landscape characteristics shape pathogen risk in populations, and how exposure risk could change over time. Machine learning approaches are rapidly evolving and future developments coupled with advances in pathogen detection are likely to provide even more resolution on the drivers of pathogen exposure risk.</p></div></div><div id="sec-17" class="subsection"><p id="p-48"><strong>Author Contributions</strong></p><p id="p-49">The study was designed by NMFJ, CP and MEC and the data were provided by CP. NMFJ, GM and MRM conducted all statistical analyses, MRM led the development of the R optimization functions, and NMFJ wrote the first draft of the manuscript. All authors (NMFJ, GM, SC, CP, MRM and MEC) contributed substantially to revisions. NMFJ and GM prepared the vignette.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-47">N.M. F-J. and M.E.C. were funded by National Science Foundation (DEB-1413925) and M.E.C. was funded by National Science Foundation (DEB-1654609) and CVM Research Office UMN Ag Experiment Station General Ag Research Funds.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> Department of Veterinary Population Medicine, University of Minnesota, 1365 Gortner Avenue, St Paul, Minnesota 55108. <span class="em-link"><span class="em-addr">nfj{at}umn.edu</span></span></p></li><li class="fn-other" id="fn-2"><p id="p-2"><strong>Data Accessibility Statement</strong> Should the manuscript be accepted, the data supporting the results will be archived on GitHub and Dryad with DOI provided.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.1" data-doi="10.1146/annurev.ecolsys.34.030102.151725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thrall</span>,  <span class="cit-name-given-names">P. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antonovics</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>,  <span class="cit-name-given-names">A. a.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Pulliam</span>,  <span class="cit-name-given-names">J. R. C.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Social organization and parasite risk in mammals : Integrating theory and empirical studies</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">517</span>–<span class="cit-lpage">547</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.ecolsys.34.030102.151725</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.volume%253D34%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.34.030102.151725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.34.030102.151725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220102000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.2" data-doi="10.1017/S002217240006160X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">R. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">May</span>,  <span class="cit-name-given-names">R. M.</span></span> (<span class="cit-pub-date">1985</span>). <span class="cit-article-title">Age-related changes in the rate of disease transmission: implications for the design of vaccination programmes</span>. <abbr class="cit-jnl-abbrev">The Journal of Hygiene</abbr>, <span class="cit-vol">94</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">365</span>–<span class="cit-lpage">436</span>. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/4008922">http://www.ncbi.nlm.nih.gov/pubmed/4008922</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bhygiene%26rft.stitle%253DJ%2BHyg%2B%2528Lond%2529%26rft.aulast%253DAnderson%26rft.auinit1%253DR.%2BM.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D365%26rft.epage%253D436%26rft.atitle%253DAge-related%2Bchanges%2Bin%2Bthe%2Brate%2Bof%2Bdisease%2Btransmission%253A%2Bimplications%2Bfor%2Bthe%2Bdesign%2Bof%2Bvaccination%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS002217240006160X%26rft_id%253Dinfo%253Apmid%252F4008922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S002217240006160X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4008922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.3" data-doi="10.1126/science.aap9072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babayan</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orton</span>,  <span class="cit-name-given-names">R. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">362</span>(<span class="cit-issue">6414</span>), <span class="cit-fpage">577</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aap9072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBabayan%26rft.auinit1%253DS.%2BA.%26rft.volume%253D362%26rft.issue%253D6414%26rft.spage%253D577%26rft.epage%253D580%26rft.atitle%253DPredicting%2Breservoir%2Bhosts%2Band%2Barthropod%2Bvectors%2Bfrom%2Bevolutionary%2Bsignatures%2Bin%2BRNA%2Bvirus%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap9072%26rft_id%253Dinfo%253Apmid%252F30385576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNC81NzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wNi81NjkwMTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-569012v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Behdenna</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lembo</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cleaveland</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halliday</span>,  <span class="cit-name-given-names">J. E. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lankester</span>,  <span class="cit-name-given-names">F.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Viana</span>,  <span class="cit-name-given-names">M.</span></span> (in press). <span class="cit-article-title">Transmission ecology of canine parvovirus in a multi-host, multi-pathogen system</span>. <abbr class="cit-jnl-abbrev">Proc R Soc B</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.5" data-doi="10.1038/srep19342"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brdar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gavrić</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ćulibrk</span>,  <span class="cit-name-given-names">D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Crnojević</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Unveiling spatial epidemiology of HIV with mobile phone data</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">19342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/srep19342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D6%26rft.spage%253D19342%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.6" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breiman</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Random forests</span>. <abbr class="cit-jnl-abbrev">Machine Learning</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1023/A:1010933404324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bevins</span>,  <span class="cit-name-given-names">S. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lappin</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boydston</span>,  <span class="cit-name-given-names">E. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lyren</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alldredge</span>,  <span class="cit-name-given-names">M.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">VandeWoude</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Pathogen exposure varies widely among sympatric populations of wild and domestic felids across the United States</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">367</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D26%26rft.spage%253D367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.8" data-doi="10.1214/09-AOAS285"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chipman</span>,  <span class="cit-name-given-names">H. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">George</span>,  <span class="cit-name-given-names">E. I.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">McCulloch</span>,  <span class="cit-name-given-names">R. E.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">BART: Bayesian additive regression trees</span>. <abbr class="cit-jnl-abbrev">The Annals of Applied Statistics</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">266</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1214/09-AOAS285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D4%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-AOAS285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-AOAS285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000283528300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.9" data-doi="10.1111/j.0006-341X.2001.00143.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denison</span>,  <span class="cit-name-given-names">D. G. T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">C. C.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Bayesian partitioning for estimating disease risk</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr>, <span class="cit-vol">57</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">143</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.0006-341X.2001.00143.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDenison%26rft.auinit1%253DD.%2BG.%26rft.volume%253D57%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D149%26rft.atitle%253DBayesian%2Bpartitioning%2Bfor%2Bestimating%2Bdisease%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.2001.00143.x%26rft_id%253Dinfo%253Apmid%252F11252589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.2001.00143.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11252589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167376900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.10" data-doi="10.2105/AJPH.88.2.216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Diez-Roux</span>,  <span class="cit-name-given-names">A. V.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Bringing context back into epidemiology: variables and fallacies in multilevel analysis</span>. <abbr class="cit-jnl-abbrev">American Journal of Public Health</abbr>, <span class="cit-vol">88</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">216</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2105/AJPH.88.2.216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPublic%2BHealth%26rft.stitle%253DAJPH%26rft.aulast%253DDiez-Roux%26rft.auinit1%253DA%2BV%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D216%26rft.epage%253D222%26rft.atitle%253DBringing%2Bcontext%2Bback%2Binto%2Bepidemiology%253A%2Bvariables%2Band%2Bfallacies%2Bin%2Bmultilevel%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.2105%252FAJPH.88.2.216%26rft_id%253Dinfo%253Apmid%252F9491010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2105/AJPH.88.2.216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9491010&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074113400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.11" data-doi="10.1111/j.1365-2656.2008.01390.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>,  <span class="cit-name-given-names">J. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hastie</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">A working guide to boosted regression trees</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>, <span class="cit-vol">77</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">802</span>–<span class="cit-lpage">813</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2656.2008.01390.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BEcol%26rft.aulast%253DElith%26rft.auinit1%253DJ.%26rft.volume%253D77%26rft.issue%253D4%26rft.spage%253D802%26rft.epage%253D813%26rft.atitle%253DA%2Bworking%2Bguide%2Bto%2Bboosted%2Bregression%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2008.01390.x%26rft_id%253Dinfo%253Apmid%252F18397250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2008.01390.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256539800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.12" data-doi="10.1093/beheco/arh028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ezenwa</span>,  <span class="cit-name-given-names">V. O.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Host social behavior and parasitic infection: a multifactorial approach</span>. <abbr class="cit-jnl-abbrev">Behavioral Ecology</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">446</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/beheco/arh028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavioral%2BEcology%26rft.stitle%253DBehav%2BEcol%26rft.aulast%253DEzenwa%26rft.auinit1%253DV.%2BO.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D446%26rft.epage%253D454%26rft.atitle%253DHost%2Bsocial%2Bbehavior%2Band%2Bparasitic%2Binfection%253A%2Ba%2Bmultifactorial%2Bapproach%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbeheco%252Farh028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/beheco/arh028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000221556500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-569012v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rudin</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Dominici</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Model class reliance: Variable importance measures for any machine learning model class, from the “Rashomon” Perspective</span>. Retrieved from <a href="http://arxiv.org/abs/1801.01489">http://arxiv.org/abs/1801.01489</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-569012v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fountain-Jones</span>,  <span class="cit-name-given-names">N.</span></span>., <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacquot</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchet</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terio</span>,  <span class="cit-name-given-names">K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Craft</span>,  <span class="cit-name-given-names">M.</span></span>. (in press). <span class="cit-article-title">Endemic infection can shape exposure to novel pathogens: Pathogen co-occurrence networks in the Serengeti lions</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.15" data-doi="10.1016/S0167-9473(01)00065-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">J. H.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Stochastic gradient boosting</span>. <abbr class="cit-jnl-abbrev">Computational Statistics &amp; Data Analysis</abbr>, <span class="cit-vol">38</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">367</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0167-9473(01)00065-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2BStatistics%2B%2526%2BData%2BAnalysis%26rft.volume%253D38%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-9473%252801%252900065-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-9473(01)00065-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173918200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.16" data-doi="10.1214/07-AOAS148"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">J. H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Popescu</span>,  <span class="cit-name-given-names">B. E.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Predictive learning via rule ensembles</span>. <abbr class="cit-jnl-abbrev">The Annals of Applied Statistics</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">916</span>–<span class="cit-lpage">954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1214/07-AOAS148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D2%26rft.spage%253D916%26rft_id%253Dinfo%253Adoi%252F10.1214%252F07-AOAS148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/07-AOAS148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.17" data-doi="10.1007/s10393-013-0856-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">A. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fooks</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayman</span>,  <span class="cit-name-given-names">D. T. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Rupprecht</span>,  <span class="cit-name-given-names">C. E.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Deciphering serology to understand the ecology of infectious diseases in wildlife</span>. <abbr class="cit-jnl-abbrev">EcoHealth</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">298</span>–<span class="cit-lpage">313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10393-013-0856-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D10%26rft.spage%253D298%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10393-013-0856-0%26rft_id%253Dinfo%253Apmid%252F23918033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10393-013-0856-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918033&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328339700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.18" data-doi="10.1080/10618600.2014.907095"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapelner</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bleich</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pitkin</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Peeking inside the black box: Visualizing statistical learning with plots of individual conditional expectation</span>. <abbr class="cit-jnl-abbrev">Journal of Computational and Graphical Statistics</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">44</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1080/10618600.2014.907095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComputational%2Band%2BGraphical%2BStatistics%26rft.volume%253D24%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10618600.2014.907095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10618600.2014.907095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.19" data-doi="10.1097/EDE.0000000000000571"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pitzer</span>,  <span class="cit-name-given-names">V. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hagan</span>,  <span class="cit-name-given-names">J. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lipsitch</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Temporally varying relative risks for infectious diseases</span>. <abbr class="cit-jnl-abbrev">Epidemiology</abbr>. <span class="cit-auth cit-collab">NIH Public Access</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1097/EDE.0000000000000571</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpidemiology%26rft_id%253Dinfo%253Adoi%252F10.1097%252FEDE.0000000000000571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/EDE.0000000000000571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.20" data-doi="10.1371/journal.pntd.0004815"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowden</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayman</span>,  <span class="cit-name-given-names">D. T. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Undiscovered bat hosts of Filoviruses</span>. <abbr class="cit-jnl-abbrev">PLOS Neglected Tropical Diseases</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">e0004815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0004815</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D10%26rft.spage%253De0004815%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0004815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0004815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.21" data-doi="10.1023/A:1021251113462"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">Y. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pepyne</span>,  <span class="cit-name-given-names">D. L.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Simple explanation of the no-free-lunch theorem and its implications</span>. <abbr class="cit-jnl-abbrev">Journal of Optimization Theory and Applications</abbr>, <span class="cit-vol">115</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">549</span>–<span class="cit-lpage">570</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1023/A:1021251113462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BOptimization%2BTheory%2Band%2BApplications%26rft.volume%253D115%26rft.spage%253D549%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021251113462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021251113462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofmann-Lehmann</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grob</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elgizoli</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martenson</span>,  <span class="cit-name-given-names">J. S.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Lutz</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Prevalence of antibodies to feline parvovirus, calicivirus, herpesvirus, coronavirus, and immunodeficiency virus and of feline leukemia virus antigen and the interrelationship of these viral infections in free-ranging lions in East Africa</span>. <abbr class="cit-jnl-abbrev">Clinical and Vaccine Immunology</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">554</span>–<span class="cit-lpage">562</span>. Retrieved from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC170405/pdf/030554.pdf">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC170405/pdf/030554.pdf</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Band%2BVaccine%2BImmunology%26rft.volume%253D3%26rft.spage%253D554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.23" data-doi="10.1371/journal.pone.0183626"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hollings</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">A.</span></span>, van <span class="cit-auth"><span class="cit-name-surname">Andel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jewell</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Burgman</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Species distribution models: A comparison of statistical approaches for livestock and disease epidemics</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">e0183626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0183626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D12%26rft.spage%253De0183626%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0183626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0183626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.24" data-doi="10.1016/S0004-3702(97)00043-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kohavi</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">John</span>,  <span class="cit-name-given-names">G. H.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Wrappers for feature subset selection</span>. <abbr class="cit-jnl-abbrev">Artificial Intelligence</abbr>, <span class="cit-vol">97</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">273</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0004-3702(97)00043-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArtificial%2BIntelligence%26rft.volume%253D97%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0004-3702%252897%252900043-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0004-3702(97)00043-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000071321500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.25" data-doi="10.18637/jss.v028.i05"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Building Predictive Models in <em>R</em> Using the caret Package</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.18637/jss.v028.i05</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D28%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v028.i05%26rft_id%253Dinfo%253Apmid%252F27774042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v028.i05&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774042&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.26" data-doi="10.18637/jss.v036.i11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kursa</span>,  <span class="cit-name-given-names">M. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rudnicki</span>,  <span class="cit-name-given-names">W. R.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Feature selection with the Boruta Package</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.18637/jss.v036.i11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D36%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v036.i11%26rft_id%253Dinfo%253Apmid%252F25285054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v036.i11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25285054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.27" data-doi="10.1111/j.2041-210X.2011.00156.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lachish</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gopalaswamy</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>,  <span class="cit-name-given-names">S. C. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sheldon</span>,  <span class="cit-name-given-names">B. C.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Site-occupancy modelling as a novel framework for assessing test sensitivity and estimating wildlife disease prevalence from imperfect diagnostic tests</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">339</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.2041-210X.2011.00156.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00156.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00156.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-confproc" id="cit-569012v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lundberg</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.-I.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">An unexpected unity among methods for interpreting model predictions</span>. In <span class="cit-conf-name">NIPS 2016 Workshop on Interpretable Machine Learning in Complex Systems</span>. Retrieved from <a href="http://arxiv.org/abs/1611.07478">http://arxiv.org/abs/1611.07478</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.29" data-doi="10.1186/s13567-015-0219-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Machado</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendoza</span>,  <span class="cit-name-given-names">M. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Corbellini</span>,  <span class="cit-name-given-names">L. G.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">What variables are important in predicting bovine viral diarrhea virus? A random forest approach</span>. <abbr class="cit-jnl-abbrev">Veterinary Research</abbr>, <span class="cit-vol">46</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13567-015-0219-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2BResearch%26rft.volume%253D46%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13567-015-0219-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13567-015-0219-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.30" data-doi="10.1111/j.1472-4642.2008.00491.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marmion</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parviainen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luoto</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heikkinen</span>,  <span class="cit-name-given-names">R. K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Evaluation of consensus methods in predictive species distribution modelling</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">59</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1472-4642.2008.00491.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D15%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1472-4642.2008.00491.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1472-4642.2008.00491.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.31" data-doi="10.21105/joss.00786"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Molnar</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2018a</span>). <span class="cit-article-title">iml: An R package for Interpretable Machine Learning</span>. <abbr class="cit-jnl-abbrev">Journal of Open Source Software</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">26</span>), <span class="cit-fpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.21105/joss.00786</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BOpen%2BSource%2BSoftware%26rft.volume%253D3%26rft.spage%253D786%26rft_id%253Dinfo%253Adoi%252F10.21105%252Fjoss.00786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21105/joss.00786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Molnar</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2018b</span>). <abbr class="cit-jnl-abbrev">Interpretable machine learning</abbr> (Retrieved).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.33" data-doi="10.1371/journal.pone.0002545"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Munson</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terio</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kock</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mlengeya</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roelke</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dubovi</span>,  <span class="cit-name-given-names">E.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Climate extremes promote fatal co-infections during canine distemper epidemics in African lions</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">e2545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0002545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMunson%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De2545%26rft.epage%253De2545%26rft.atitle%253DClimate%2Bextremes%2Bpromote%2Bfatal%2Bco-infections%2Bduring%2Bcanine%2Bdistemper%2Bepidemics%2Bin%2BAfrican%2Blions.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002545%26rft_id%253Dinfo%253Apmid%252F18575601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18575601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.34" data-doi="10.1016/J.TREE.2008.06.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Freckleton</span>,  <span class="cit-name-given-names">R. P.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Missing inaction: the dangers of ignoring missing data</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">592</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/J.TREE.2008.06.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D23%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.TREE.2008.06.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/J.TREE.2008.06.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordán</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCabe</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdolin</span>,  <span class="cit-name-given-names">J. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Fewell</span>,  <span class="cit-name-given-names">J. H.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Infectious disease and group size: more than just a numbers game</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society of London B: Biological Sciences</abbr>, <span class="cit-vol">370</span>(<span class="cit-issue">1669</span>). Retrieved from <a href="http://rstb.royalsocietypublishing.org/content/370/1669/20140111.long">http://rstb.royalsocietypublishing.org/content/370/1669/20140111.long</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.36" data-doi="10.1186/1753-6561-5-S3-S11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>,  <span class="cit-name-given-names">J. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piepho</span>,  <span class="cit-name-given-names">H.-P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A comparison of random forests, boosting and support vector machines for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Proceedings</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">Suppl 3</span>), <span class="cit-fpage">S11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1753-6561-5-S3-S11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProceedings%26rft.volume%253D5%26rft.spage%253D11S%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-5-S3-S11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-5-S3-S11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Appel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martenson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Brien</span>,  <span class="cit-name-given-names">S. J.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Lutz</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Viruses of the Serengeti: patterns of infection and mortality in African lions</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>, <span class="cit-vol">68</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1161</span>–<span class="cit-lpage">1178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D68%26rft.spage%253D1161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.38" data-doi="10.1111/2041-210X.12232"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Penone</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>,  <span class="cit-name-given-names">K. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Marco</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rondinini</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">T. M.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Costa</span>,  <span class="cit-name-given-names">G. C.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Imputation of missing data in life-history trait datasets: which approach performs the best?</span> <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">961</span>–<span class="cit-lpage">970</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.12232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pollock</span>,  <span class="cit-name-given-names">R. V</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Coyne</span>,  <span class="cit-name-given-names">M. J.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Canine parvovirus</span>. <abbr class="cit-jnl-abbrev">The Veterinary Clinics of North America. Small Animal Practice</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">555</span>–<span class="cit-lpage">68</span>. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/8389070">http://www.ncbi.nlm.nih.gov/pubmed/8389070</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BVeterinary%2Bclinics%2Bof%2BNorth%2BAmerica.%2BSmall%2Banimal%2Bpractice%26rft.stitle%253DVet%2BClin%2BNorth%2BAm%2BSmall%2BAnim%2BPract%26rft.aulast%253DPollock%26rft.auinit1%253DR.%2BV.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D555%26rft.epage%253D568%26rft.atitle%253DCanine%2Bparvovirus.%26rft_id%253Dinfo%253Apmid%252F8389070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8389070&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LA12500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.40" data-doi="10.1038/379441a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roelke-Parker</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munson</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kock</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cleaveland</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">M.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Appel</span>,  <span class="cit-name-given-names">M. J.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">A canine distemper virus epidemic in Serengeti lions (<em>Panthera leo</em>)</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">379</span>(<span class="cit-issue">6564</span>), <span class="cit-fpage">441</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRoelke-Parker%26rft.auinit1%253DM.%2BE.%26rft.volume%253D379%26rft.issue%253D6564%26rft.spage%253D441%26rft.epage%253D445%26rft.atitle%253DA%2Bcanine%2Bdistemper%2Bvirus%2Bepidemic%2Bin%2BSerengeti%2Blions%2B%2528Panthera%2Bleo%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F379441a0%26rft_id%253Dinfo%253Apmid%252F8559247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/379441a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8559247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TT30400057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-569012v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schölkopf</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Smola</span>,  <span class="cit-name-given-names">A. J.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-chapter-title">Learning with kernels : support vector machines, regularization, optimization, and beyond</span>. <span class="cit-publ-name">MIT Press</span>. Retrieved from <a href="https://dl.acm.org/citation.cfm?id=559923">https://dl.acm.org/citation.cfm?id=559923</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.42" data-doi="10.1073/pnas.39.10.1095"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shapley</span>,  <span class="cit-name-given-names">L. S.</span></span> (<span class="cit-pub-date">1953</span>). <span class="cit-article-title">Stochastic games</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-vol">39</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">1095</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/PNAS.39.10.1095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShapley%26rft.auinit1%253DL.%2BS.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1095%26rft.epage%253D1100%26rft.atitle%253DStochastic%2BGames%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.39.10.1095%26rft_id%253Dinfo%253Apmid%252F16589380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMzkvMTAvMTA5NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA2LzU2OTAxMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.43" data-doi="10.1093/bioinformatics/btr597"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>,  <span class="cit-name-given-names">D. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Missforest-Non-parametric missing value imputation for mixed-type data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr597%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298383800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.44" data-doi="10.1007/s10115-013-0679-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Štrumbelj</span>,  <span class="cit-name-given-names">E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kononenko</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Explaining prediction models and individual predictions with feature contributions</span>. <abbr class="cit-jnl-abbrev">Knowledge and Information Systems</abbr>, <span class="cit-vol">41</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">647</span>–<span class="cit-lpage">665</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10115-013-0679-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DKnowledge%2Band%2BInformation%2BSystems%26rft.volume%253D41%26rft.spage%253D647%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10115-013-0679-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10115-013-0679-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.45" data-doi="10.1016/S0895-4356(96)00002-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">J. V.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Advantages and disadvantages of using artificial neural networks versus logistic regression for predicting medical outcomes</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Epidemiology</abbr>, <span class="cit-vol">49</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1225</span>–<span class="cit-lpage">1231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0895-4356(96)00002-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bepidemiology%26rft.stitle%253DJ%2BClin%2BEpidemiol%26rft.aulast%253DTu%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D49%26rft.issue%253D11%26rft.spage%253D1225%26rft.epage%253D1231%26rft.atitle%253DAdvantages%2Band%2Bdisadvantages%2Bof%2Busing%2Bartificial%2Bneural%2Bnetworks%2Bversus%2Blogistic%2Bregression%2Bfor%2Bpredicting%2Bmedical%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0895-4356%252896%252900002-9%26rft_id%253Dinfo%253Apmid%252F8892489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0895-4356(96)00002-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8892489&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F06%2F569012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VP56600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-569012v1.46" data-doi="10.1073/pnas.1411623112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viana</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cleaveland</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthiopoulos</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halliday</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Craft</span>,  <span class="cit-name-given-names">M. E.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Lembo</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Dynamics of a morbillivirus at the domestic–wildlife interface: Canine distemper virus in domestic dogs and lions</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-vol">112</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1464</span>–<span class="cit-lpage">1469</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1411623112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1411623112%26rft_id%253Dinfo%253Apmid%252F25605919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzUvMTQ2NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA2LzU2OTAxMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/569012v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F569012v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/557082v1" title="Quantitative multiplexed ChIP reveals global alterations that shape promoter bivalency in ground state embryonic stem cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 06, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/569012v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/569012v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/569012v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/569012v1" class="active" data-icon-position="" data-hide-link-title="0">How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/217895" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ScjJWBebkd3NyhhMnI1-IbAbLuE6HyLRNJb2inAiybw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="217895" id="node217895--2" data-pisa="biorxiv;569012v1" data-pisa-master="biorxiv;569012" data-seqnum="217895" data-apath="/biorxiv/early/2019/03/06/569012.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Fountain-Jones</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gustavo</span> <span class="nlm-surname">Machado</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott</span> <span class="nlm-surname">Carver</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Packer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mariana</span> <span class="nlm-surname">Recamonde-Mendoza</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Meggan E.</span> <span class="nlm-surname">Craft</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569012; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/569012 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/569012v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;title=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;title=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;text=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;title=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;t=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;title=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;title=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="217895" id="node217895--3" data-pisa="biorxiv;569012v1" data-pisa-master="biorxiv;569012" data-seqnum="217895" data-apath="/biorxiv/early/2019/03/06/569012.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Fountain-Jones</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gustavo</span> <span class="nlm-surname">Machado</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott</span> <span class="nlm-surname">Carver</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Packer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mariana</span> <span class="nlm-surname">Recamonde-Mendoza</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Meggan E.</span> <span class="nlm-surname">Craft</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569012; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/569012 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Fountain-Jones%2C+Nicholas+M.&amp;rft.au=Machado%2C+Gustavo&amp;rft.au=Carver%2C+Scott&amp;rft.au=Packer%2C+Craig&amp;rft.au=Recamonde-Mendoza%2C+Mariana&amp;rft.au=Craft%2C+Meggan+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/217895/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/217895/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/217895/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/217895/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/217895/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/217895/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/217895/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/217895/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/217895/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/217895/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/217895/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/217895/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;count=horizontal&amp;via=&amp;text=How%20to%20make%20more%20from%20exposure%20data%3F%20An%20integrated%20machine%20learning%20pipeline%20to%20predict%20pathogen%20exposure&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/569012v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/569012v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/569012v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
