<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/02/2020.03.31.019216.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.31.019216v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis" />
<meta name="DC.Identifier" content="10.1101/2020.03.31.019216" />
<meta name="DC.Date" content="2020-04-02" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-КB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention." />
<meta name="DC.Contributor" content="Jingjiao Li" />
<meta name="DC.Contributor" content="Mingquan Guo" />
<meta name="DC.Contributor" content="Xiaoxu Tian" />
<meta name="DC.Contributor" content="Chengrong Liu" />
<meta name="DC.Contributor" content="Xin Wang" />
<meta name="DC.Contributor" content="Xing Yang" />
<meta name="DC.Contributor" content="Ping Wu" />
<meta name="DC.Contributor" content="Zixuan Xiao" />
<meta name="DC.Contributor" content="Yafei Qu" />
<meta name="DC.Contributor" content="Yue Yin" />
<meta name="DC.Contributor" content="Joyce Fu" />
<meta name="DC.Contributor" content="Zhaoqin Zhu" />
<meta name="DC.Contributor" content="Zhenshan Liu" />
<meta name="DC.Contributor" content="Chao Peng" />
<meta name="DC.Contributor" content="Tongyu Zhu" />
<meta name="DC.Contributor" content="Qiming Liang" />
<meta name="article:published_time" content="2020-04-02" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-kB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.31.019216v1" />
<meta name="citation_id" content="2020.03.31.019216v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.31.019216" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.31.019216" />
<meta name="citation_author" content="Jingjiao Li" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author" content="Mingquan Guo" />
<meta name="citation_author_institution" content="Shanghai Institute of Phage, Shanghai Public Health Clinical Center, Fudan University" />
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Public Health Clinical Center, Fudan University" />
<meta name="citation_author" content="Xiaoxu Tian" />
<meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences" />
<meta name="citation_author" content="Chengrong Liu" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author" content="Xin Wang" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author" content="Xing Yang" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author" content="Ping Wu" />
<meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences" />
<meta name="citation_author" content="Zixuan Xiao" />
<meta name="citation_author_institution" content="Faculty of Medicine &amp; Dentistry, University of Alberta" />
<meta name="citation_author" content="Yafei Qu" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9367-1302" />
<meta name="citation_author" content="Yue Yin" />
<meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences" />
<meta name="citation_author" content="Joyce Fu" />
<meta name="citation_author_institution" content="Department of Statistics, University of California" />
<meta name="citation_author" content="Zhaoqin Zhu" />
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Public Health Clinical Center, Fudan University" />
<meta name="citation_author" content="Zhenshan Liu" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author_email" content="liangqiming@shsmu.edu.cn" />
<meta name="citation_author_email" content="tyzhu@shphc.org.cn" />
<meta name="citation_author_email" content="pengchao@sari.ac.cn" />
<meta name="citation_author_email" content="zhenshan6@shsmu.edu.cn" />
<meta name="citation_author" content="Chao Peng" />
<meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences" />
<meta name="citation_author_email" content="liangqiming@shsmu.edu.cn" />
<meta name="citation_author_email" content="tyzhu@shphc.org.cn" />
<meta name="citation_author_email" content="pengchao@sari.ac.cn" />
<meta name="citation_author_email" content="zhenshan6@shsmu.edu.cn" />
<meta name="citation_author" content="Tongyu Zhu" />
<meta name="citation_author_institution" content="Shanghai Institute of Phage, Shanghai Public Health Clinical Center, Fudan University" />
<meta name="citation_author_institution" content="Shanghai Key Laboratory of Organ Transplantation, Zhongshan Hospital, Fudan University" />
<meta name="citation_author_email" content="liangqiming@shsmu.edu.cn" />
<meta name="citation_author_email" content="tyzhu@shphc.org.cn" />
<meta name="citation_author_email" content="pengchao@sari.ac.cn" />
<meta name="citation_author_email" content="zhenshan6@shsmu.edu.cn" />
<meta name="citation_author" content="Qiming Liang" />
<meta name="citation_author_institution" content="Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University" />
<meta name="citation_author_email" content="liangqiming@shsmu.edu.cn" />
<meta name="citation_author_email" content="tyzhu@shphc.org.cn" />
<meta name="citation_author_email" content="pengchao@sari.ac.cn" />
<meta name="citation_author_email" content="zhenshan6@shsmu.edu.cn" />
<meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature (2020) doi:10.1038/s41586-020-2008-3." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature (2020) doi:10.1038/s41586-020-2012-7." />
<meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N. Engl. J. Med. (2020) doi:10.1056/NEJMoa2001017." />
<meta name="citation_reference" content="Wilson, M. E. &amp; Chen, L. H. Travelers Give Wings to Novel Coronavirus (2019-nCoV). J. Travel Med. (2020) doi:10.1093/jtm/taaa015." />
<meta name="citation_reference" content="The Lancet, null. Emerging understandings of 2019-nCoV. Lancet Lond. Engl. 395, 311 (2020)." />
<meta name="citation_reference" content="Guan, W. et al. Clinical characteristics of 2019 novel coronavirus infection in China. medRxiv 2020.02.06.20020974 (2020) doi:10.1101/2020.02.06.20020974." />
<meta name="citation_reference" content="Hoffmann, M. et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell (2020) doi:10.1016/j.cell.2020.02.052." />
<meta name="citation_reference" content="Walls, A. C. et al. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell (2020) doi:10.1016/j.cell.2020.02.058." />
<meta name="citation_reference" content="Yan, R. et al. Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science (2020) doi:10.1126/science.abb2762." />
<meta name="citation_reference" content="Zhang, L. et al. Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors. Science (2020) doi:10.1126/science.abb3405." />
<meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet Lond. Engl. 395, 497–506 (2020)." />
<meta name="citation_reference" content="Ren, L.-L. et al. Identification of a novel coronavirus causing severe pneumonia in human: a descriptive study. Chin. Med. J. (Engl.) (2020) doi:10.1097/CM9.0000000000000722." />
<meta name="citation_reference" content="Kolaczkowska, E. &amp; Kubes, P. Neutrophil recruitment and function in health and inflammation. Nat. Rev. Immunol. 13, 159–175 (2013)." />
<meta name="citation_reference" content="Wang, C. et al. Aveolar Macrophage Activation and Cytokine Storm in the Pathogenesis of Severe COVID-19. (2020) doi:10.21203/rs.3.rs-19346/v1." />
<meta name="citation_reference" content="Wu, A. et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe (2020) doi:10.1016/j.chom.2020.02.001." />
<meta name="citation_reference" content="Decroly, E. et al. Crystal structure and functional analysis of the SARS-coronavirus RNA cap 2’-O-methyltransferase nsp10/nsp16 complex. PLoS Pathog. 7, e1002059 (2011)." />
<meta name="citation_reference" content="Ma, Y. et al. Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex. Proc. Natl. Acad. Sci. U. S. A. 112, 9436–9441 (2015)." />
<meta name="citation_reference" content="Wang, Y. et al. Coronavirus nsp10/nsp16 Methyltransferase Can Be Targeted by nsp10-Derived Peptide In Vitro and In Vivo To Reduce Replication and Pathogenesis. J. Virol. 89, 8416–8427 (2015)." />
<meta name="citation_reference" content="von Brunn, A. et al. Analysis of intraviral protein-protein interactions of the SARS coronavirus ORFeome. PloS One 2, e459 (2007)." />
<meta name="citation_reference" content="Nourbakhsh, M. et al. The NF-κB Repressing Factor Is Involved in Basal Repression and Interleukin (IL)-1-induced Activation of IL-8 Transcription by Binding to a Conserved NF-κB-flanking Sequence Element. J. Biol. Chem. 276, 4501–4508 (2001)." />
<meta name="citation_reference" content="Huang, K.-H., Wang, C.-H., Lin, C.-H. &amp; Kuo, H.-P. NF-κB repressing factor downregulates basal expression and mycobacterium tuberculosis induced IP-10 and IL-8 synthesis via interference with NF-κB in monocytes. J. Biomed. Sci. 21, 71 (2014)." />
<meta name="citation_reference" content="Ghebrehiwet, B., Lim, B. L., Peerschke, E. I., Willis, A. C. &amp; Reid, K. B. Isolation, cDNA cloning, and overexpression of a 33-kD cell surface glycoprotein that binds to the globular ‘heads’ of C1q. J. Exp. Med. 179, 1809–1821 (1994)." />
<meta name="citation_reference" content="Scholte, F. E. M. et al. Stress granule components G3BP1 and G3BP2 play a proviral role early in Chikungunya virus replication. J. Virol. 89, 4457–4469 (2015)." />
<meta name="citation_reference" content="Faria, P. A. et al. VSV disrupts the Rae1/mrnp41 mRNA nuclear export pathway. Mol. Cell 17, 93–102 (2005)." />
<meta name="citation_reference" content="Plager, D. A. et al. A novel and highly divergent homolog of human eosinophil granule major basic protein. J. Biol. Chem. 274, 14464–14473 (1999)." />
<meta name="citation_reference" content="Eash, K. J., Greenbaum, A. M., Gopalan, P. K. &amp; Link, D. C. CXCR2 and CXCR4 antagonistically regulate neutrophil trafficking from murine bone marrow. J. Clin. Invest. 120, 2423–2431 (2010)." />
<meta name="citation_reference" content="Nakatomi, K., Aida, Y., Kusumoto, K., Pabst, M. J. &amp; Maeda, K. Neutrophils responded to immobilized lipopolysaccharide in the absence of lipopolysaccharide-binding protein. J. Leukoc. Biol. 64, 177–184 (1998)." />
<meta name="citation_reference" content="Soria-Valles, C. et al. MMP-25 Metalloprotease Regulates Innate Immune Response through NF-κB Signaling. J. Immunol. Baltim. Md 1950 197, 296–302 (2016)." />
<meta name="citation_reference" content="Huang, L. et al. Down-regulation of miR-301a suppresses pro-inflammatory cytokines in Toll-like receptor-triggered macrophages. Immunology 140, 314–322 (2013)." />
<meta name="citation_reference" content="Xiong, C., Jiang, L., Chen, Y. &amp; Jiang, Q. Evolution and variation of 2019-novel coronavirus. bioRxiv 2020.01.30.926477 (2020) doi:10.1101/2020.01.30.926477." />
<meta name="citation_reference" content="Kumar, S., Stecher, G., Li, M., Knyaz, C. &amp; Tamura, K. MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Mol. Biol. Evol. 35, 1547–1549 (2018)." />
<meta name="citation_reference" content="Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015)." />
<meta name="citation_reference" content="Yu, G., Wang, L.-G., Han, Y. &amp; He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Omics J. Integr. Biol. 16, 284–287 (2012)." />
<meta name="twitter:title" content="Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-КB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention." />
<meta name="og-title" property="og:title" content="Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-КB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-02" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.31.019216v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.31.019216v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.31.019216v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1223734" />
    <title>Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fE57MVpbHQPT2QWWeVFrTzJpIDClJXiTv5c9yqr6Tm8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1223734","apath":"\/biorxiv\/early\/2020\/04\/02\/2020.03.31.019216.atom","pisa":"biorxiv;2020.03.31.019216v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.31.019216v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.31.019216v1","title":"Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis","identifier":"node\/1223734"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rQA0QcG2PEj3s3zYyCVf4xEr6Xev8IF5M47j3DMsNi4SU9oN0UVpAyt9fwPI6DlbwiBm7aymLHdX0ZC\\\\\\\/QoRdSaHpcDPrbWPq9ipilM5neQJGDe\\\\\\\/6NctHMInfgy3iCrbm5fjcgAoHc+Aq5H+FuoWEcWiKpCzz6ZuPRnV1AqIeMQytkhinFWyz5OlC8bpWD\\\\\\\/bQmltQ5AYHBz2zOZpgUmrQ+yZ+Jw8cJLT9qtAL+wnF4986uPDK1Xm4aAnCBp+9VwiEIWYl+g8p62OzxjvzFPHGZlSD6bHiJgfRgQEYxgNUGFF6TerqSHQjUGYPk6PC0hfdpHGyQXTkQtUjwx2EbSidj\\\\\\\/8w2xtY\\\\\\\/Dh4QuHlSs9eYiWzPHLdU6ZGzV90eS56vkUp9w+dDEldY6d4XGvS+akE7sSXuTgIfOlCKesUD2TwTfUwH1sqBkCiimoWbsijxSxJRzGVp2FJ7r9SbD\\\\\\\/4uclF7xv4hELkGMJ1xj9d+ESb68i\\\\\\\/PJN5ksSudlSbnaNe59x2agPXo2oBmDucoShVb9X5qnuhA2JrBQgySYSyuzi75ZIppEr4nLpM2WTycUGa3F4\\\\\\\/DZLYKw7NgV7eTRleJ1K4gaTb3HfZ2wJv4khqODRNzYXCvtQWrnk328epteWreXctql9xeYDZkf9ebyj4a7pJTzyT22qtcVMWymHuknkb0q4anWjls\\\\\\\/bsKBFAaFjYdFZFizsSv95uiZeGhhI80kCUMH15S\\\\\\\/EhpaaU9U9hQoQlU6nobF5kDc5txRkBcdI8sIOwITqPJjEUwwU\\\\\\\/ReZy9kugLjAhEYUumdxjJAscZDB+hIP6iuxxn48ti\\\\\\\/+9xQbhuwNoIiYpunZwH65OWJObBueAtemzXDVuMg+0c22kparKnHWPBWOPdHKZOoEKLOMwDWRpR+38fUs4Z6n9lxV+oP2YaZgJsXGAngb7ZiD8qZOteoL7rKSeGwkn6KJaacuhUPuRbCH7FefPIdgesa6DbPh40ubypxb8YcCnw+H5Dr\\\\\\\/oDeoiXksO1JUPrVrF0JuqTYudqZbDZasd\\\\\\\/P1bz5m3RTmLAyaegwIsX7KPkl+UmBBrv\\\\\\\/lsjrfvDYTrXJrYKjiU6He7Wk2vV3HeoKoxNpCGSUDvicy6NO2EW5BrDMSfuss3u1KREzG6m\\\\\\\/ailhNv1j2d3X0\\\\\\\/QQWnoEZsp+hXWhkQ0Kx8YNWL0IhBqLB9UP41NOSUnsp7sofa5aqHDYr8dwhikeZEqr8G6w\\\\\\\/hqQrXxCAsGxuON15LYKXaVbogLo7sWB4CrLK5aBshMvy\\\\\\\/bkXk0IATrlgRQUgJXzGDBzQNzFKUUsPwZhpqxylTZ6AMFwlGc+CqwIV9D6dycMaCqga+IWHdXuRF\\\u0022,\\\u0022iv\\\u0022:\\\u0022xK45Xx5YYW+lOwuc3o6YXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221f49e6e2f9fb0fd4e95bf0d887b92756c016c7e56a9deb582884bf2cdf8e4b91\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.31.019216v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1223734 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.31.019216v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zaZqagfehUX9jlD5nL4s_P8rHw18hLAkES-5D0bN2Zg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1223734" id="node1223734" data-pisa="biorxiv;2020.03.31.019216v1" data-pisa-master="biorxiv;2020.03.31.019216" data-apath="/biorxiv/early/2020/04/02/2020.03.31.019216.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingjiao</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mingquan</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Chengrong</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xin</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xing</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zixuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9367-1302" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Yin</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Joyce</span> <span class="nlm-surname">Fu</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Zhaoqin</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Zhenshan</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Tongyu</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Qiming</span> <span class="nlm-surname">Liang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.31.019216 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1223734" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jingjiao Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingjiao%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingjiao%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mingquan Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Shanghai Institute of Phage, Shanghai Public Health Clinical Center, Fudan University</span>, Shanghai, 201508, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Laboratory Medicine, Shanghai Public Health Clinical Center, Fudan University</span>, Shanghai, 201508, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingquan%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingquan%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaoxu Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences</span>, Shanghai, 201210, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxu%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxu%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chengrong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengrong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengrong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xing Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xing%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXing%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ping Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences</span>, Shanghai, 201210, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ping%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Zixuan Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Faculty of Medicine &#38; Dentistry, University of Alberta</span>, Alberta, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zixuan%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZixuan%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yafei Qu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yafei%2BQu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYafei%2BQu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9367-1302" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yafei Qu</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yue Yin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences</span>, Shanghai, 201210, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BYin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BYin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joyce Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Statistics, University of California</span>, Riverside, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joyce%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoyce%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Zhaoqin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Laboratory Medicine, Shanghai Public Health Clinical Center, Fudan University</span>, Shanghai, 201508, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaoqin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaoqin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Zhenshan Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenshan%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhenshan%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangqiming@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liangqiming@shsmu.edu.cn</a>
<a href="mailto:tyzhu@shphc.org.cn" class="" data-icon-position="" data-hide-link-title="0">tyzhu@shphc.org.cn</a>
<a href="mailto:pengchao@sari.ac.cn" class="" data-icon-position="" data-hide-link-title="0">pengchao@sari.ac.cn</a>
<a href="mailto:zhenshan6@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhenshan6@shsmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Chao Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Sciences</span>, Shanghai, 201210, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChao%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangqiming@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liangqiming@shsmu.edu.cn</a>
<a href="mailto:tyzhu@shphc.org.cn" class="" data-icon-position="" data-hide-link-title="0">tyzhu@shphc.org.cn</a>
<a href="mailto:pengchao@sari.ac.cn" class="" data-icon-position="" data-hide-link-title="0">pengchao@sari.ac.cn</a>
<a href="mailto:zhenshan6@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhenshan6@shsmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Tongyu Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Shanghai Institute of Phage, Shanghai Public Health Clinical Center, Fudan University</span>, Shanghai, 201508, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Shanghai Key Laboratory of Organ Transplantation, Zhongshan Hospital, Fudan University</span>, Shanghai, 200032, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongyu%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATongyu%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangqiming@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liangqiming@shsmu.edu.cn</a>
<a href="mailto:tyzhu@shphc.org.cn" class="" data-icon-position="" data-hide-link-title="0">tyzhu@shphc.org.cn</a>
<a href="mailto:pengchao@sari.ac.cn" class="" data-icon-position="" data-hide-link-title="0">pengchao@sari.ac.cn</a>
<a href="mailto:zhenshan6@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhenshan6@shsmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Qiming Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Research Center of Translational Medicine, Shanghai Children’s Hospital; Shanghai Institute of Immunology, Department of Immunology and Microbiology, School of Medicine, Shanghai Jiao Tong University</span>, Shanghai, 200025, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qiming%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQiming%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangqiming@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liangqiming@shsmu.edu.cn</a>
<a href="mailto:tyzhu@shphc.org.cn" class="" data-icon-position="" data-hide-link-title="0">tyzhu@shphc.org.cn</a>
<a href="mailto:pengchao@sari.ac.cn" class="" data-icon-position="" data-hide-link-title="0">pengchao@sari.ac.cn</a>
<a href="mailto:zhenshan6@shsmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhenshan6@shsmu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.31.019216v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1223734" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1223734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.31.019216v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1223734" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1223734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.31.019216v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1223734" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1223734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.31.019216v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1223734" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1223734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.31.019216v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1223734" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1223734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">The ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-КB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention.</p></div><div class="section" id="sec-1"><h2 class="">MAIN TEXT</h2><p id="p-4">In December 2019, Severe Acute Respiratory Syndrome Coronavirus-2 [SARS-CoV-2, previously known by the provisional name ‘2019 novel coronavirus’ (2019-nCoV) given by the World Health Organization (WHO)], was first isolated from a cluster of patients with pneumonia. This novel strain of coronavirus has been identified as the causative agent for the epidemic of acute respiratory distress syndrome [also referred to as coronavirus disease 2019 (COVID-19)] in Wuhan, China<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. Since then, the COVID-19 outbreak has rapidly swept all the provinces of China and has disseminated globally to 202 countries and territories through traveling<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. On 31 January 2020, the WHO declared SARS-CoV-2 a Public Health Emergency of International Concern<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. As of 27 March 2020, the current pandemic of SARS-CoV-2 has caused 512,701 confirmed infection cases including 23,495 deaths in 202 countries and territories, calling for extended efforts to understand the molecular drivers of its immune evasion and pathogenesis.</p><p id="p-5">COVID-19 is a potential zoonotic disease with low to moderate mortality rate (2-5%) and is primarily transmitted through droplet and direct contact with infected persons or incubation carriers<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. SARS-CoV-2 has a preferential tropism for airway epithelial cells mediated through the accessibility of ACE2 receptor<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. The common clinical manifestations of fever and cough in conjunction with laboratory results of progressively increasing neutrophil counts and leukocytopenia in serum samples taken from COVID-19 patients at various stages of the illness indicates uncontrolled neutrophil extravasation and activation<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Neutrophils are the most abundant type of white blood cells in human blood and serve as the first responders in the line of defense against acute infection. Chemical signals such as Interleukin-8 (IL-8) and IL-6 stimulate neutrophil chemotaxis and recruitment to the site of infection where they congregate and cause the release of more proinflammatory mediators that in turn activate additional inflammatory signals, leading to cytokine storms that may progress to vital organ failure<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. Indeed, a large number of macrophages and neutrophils were found in alveolar cavities in COVID-19 fatal patients<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, suggesting their roles for cytokine storms. While uncontrolled neutrophil activity and aberrant cytokine response contributes to SARS-CoV-2 pathogenesis in COVID-19, how SARS-CoV-2 causes these immunological changes, especially the enhancement of neutrophil chemotaxis and activation remains largely undetermined.</p><p id="p-6">SARS-CoV-2 is a single-stranded positive-sense RNA virus and belongs to the β-coronavirus family consisting of many animal species and human pathogens including SARS-CoV and Middle East Respiratory Syndrome (MERS)-CoV. The ~30 kb non-segmented genome of SARS-CoV-2 encodes 14 major open reading frames (ORFs), which are further processed into 4 structural proteins [glycoprotein spike (S), membrane (M), envelope (E), and nucleocapsid (N)], 16 non-structural proteins (nsp1 to nsp16), and at least 8 accessory proteins (ORF3a, ORF6, ORF7a, ORF7b, ORF8, ORF9a, ORF9b, and ORF10)<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Similar to SARS-CoV, spike protein of SARS-CoV-2 directly interacts with human ACE2 expressed on human airway epithelia to facilitate virus entry<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>. However, host targets for other viral proteins remain largely unknown, which limits our understanding of SARS-CoV-2 immune evasion and pathogenesis, further impeding anti-viral drug development and design of therapeutic approaches.</p><div id="sec-2" class="subsection"><h3>Genome-wide screens for SARS-CoV-2 intra-viral protein-protein interactions</h3><p id="p-7">Limited by their relatively small genomes, viral proteins form different combinations of protein complexes to fulfill a variety of critical functions during viral life cycle, making the investigation of intra-viral protein-protein interactions (PPIs) a requirement. To systematically characterize the intra-viral PPI networks, both genome-wide yeast-two hybrid (Y2H) screens and co-immunoprecipitations (co-IP) experiments with SARS-CoV-2 genes were performed. All the predicted viral genes were synthesized based on the SARS-CoV-2 Wuhan-Hu-1 isolate (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom">MN908947</a>). The entire library was cloned into Y2H DNA-binding domain (pACT2) and activation domain (pGBTK7) vectors and then transformed into the Y2HGold yeast strain (Clontech). Nsp3 was truncated into two smaller fragments in this experiment since large genes tend to yield fewer interactions in the Y2H assay. All the possible pair-wise combinations were examined independently for the activation of the reporter genes HIS3 and ADE2 in triplicates. We captured 19 PPIs from the Y2H screens, including nsp15-nsp2, ORF7b-nsp1, N-nsp4, ORF3a-nsp4, ORF7b-nsp7, nsp7-nsp8, nsp12-nsp8, nsp7-nsp9, nsp14-nsp10, nsp16-nsp10, ORF6-nsp14, E-E, N-E, ORF3a-E, ORF9b-E, N-N, nsp7-ORF7a, nsp8-ORF8, and ORF9b-ORF9b (<strong>Extended Data Fig.1</strong>). The Co-IP assay resulted in the identification of 52 PPIs (<strong>Extended Data Fig.1-2</strong>). Some of the interactions between viral proteins, such as the association between nsp12 and nsp8, were detected by both Y2H and co-IP. Some viral proteins, such as M, N, E, nsp2, nsp5, nsp8, ORF6, ORF7a, ORF7b, ORF9b, and ORF10 displayed self-association, suggesting their dimer or oligomer forms may carry out important functions during infection. In total, we characterized 58 distinct intra-viral PPIs amongst 28 SARS-CoV-2 genes that may promote virus replication and can be expected to be important to immune evasion and viral pathogenesis. (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig.1a</a></strong>).</p><p id="p-8">Among the 4 classical structural proteins, 3 PPIs were detected (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig.1b</a></strong>); these interactions of structural proteins may be critical in the organization of viral particles during viral replication and assembly. We also found 8 PPIs between structural proteins and accessory proteins (ORF3a, ORF6, ORF7a, ORF7b, ORF9b, and ORF10) and 6 PPIs between structural proteins and non-structural proteins (nsp2, nsp4, nsp5, nsp8, and nsp16) (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a></strong>), These PPI may play a pivotal role in viral structural protein processing, modification, and trafficking, supporting their importance for viral particle assembly and egress.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV-2 intra-viral protein-protein interaction network (a) Viral interactions based on Y2H screens and co-IP experiments as described in Extended Data Fig. 1. Different types of viral proteins are labelled with the indicated colors. Self-associated viral proteins are circled in blue. (b) Interactions between viral structural proteins and other viral proteins. (c) Interaction network amongst viral non-structural proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1483586763" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SARS-CoV-2 intra-viral protein-protein interaction network&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) Viral interactions based on Y2H screens and co-IP experiments as described in Extended Data Fig. 1. Different types of viral proteins are labelled with the indicated colors. Self-associated viral proteins are circled in blue. (&lt;strong&gt;b&lt;/strong&gt;) Interactions between viral structural proteins and other viral proteins. (&lt;strong&gt;c&lt;/strong&gt;) Interaction network amongst viral non-structural proteins.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F1.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F1.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">SARS-CoV-2 intra-viral protein-protein interaction network</span><p id="p-9" class="first-child">(<strong>a</strong>) Viral interactions based on Y2H screens and co-IP experiments as described in Extended Data Fig. 1. Different types of viral proteins are labelled with the indicated colors. Self-associated viral proteins are circled in blue. (<strong>b</strong>) Interactions between viral structural proteins and other viral proteins. (<strong>c</strong>) Interaction network amongst viral non-structural proteins.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">While structural proteins are components for the capsid and the envelope of the virus, several non-structural proteins (nsps), such as nsp3, nsp5, nsp12, nsp14, and nsp16 have associated enzymatic activities essential for viral RNA replication and viral protein translation. In addition, nsps can function as virulence factors that inhibit the host’s immune system defenses. We speculate that the relatively large number of nsp interactions detected (12) might imply a crucial role of nsps in the formation of multimeric complexes during the viral life cycle (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig.1c</a></strong>). The SARS-CoV-2 ORF1ab (the precursor of all 16 nsps) shares 94.4% amino acid identity to SARS-CoV ORF1ab. A number of structural studies posits the methyltransferase activity of SARS-CoV nsp14 or nsp16 are facilitated by nsp10-nsp14 and nsp10-nsp16 associations, respectively<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>, which are also observed in SARS-CoV-2. Thus, our intra-viral PPI network provides the foundation to dissect the role of these viral proteins in mediating the synthesis, assembly and processing of viral proteins for both SARS-CoV-2 and the closely related SARS-CoV. Disruption of these intra-viral PPIs by repurposing small molecule or peptide drugs that target specific epitopes can be a treatment strategy against COVID-19<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>.</p></div><div id="sec-3" class="subsection"><h3>Genome-wide proteomic screen for SARS-CoV-2 viral-host protein-protein interactions</h3><p id="p-11">To better understand the roles of these viral proteins during the virus life cycle, it is pivotal to establish the SARS-CoV-2-host interactome in human cells. We overexpressed the plasmids encoding each SARS-CoV-2 encoded gene with N-terminal 3xFlag-epitope in HEK293 cells and purified each SARS-CoV-2 protein complex by affinity purification. Co-purified cellular proteins were subsequently analyzed by liquid chromatography and tandem mass spectrometry (AP-LC-MS). From AP-LC-MS analysis, 251 cellular proteins were identified to interact with SARS-CoV-2 proteins, resulting in 631 high-confidence interactions (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig.2</a></strong> and <strong>Table 1</strong>). We identified previously reported proteins that had bona fide associations with SARS-CoV proteins, such as ROA0 and subunits of the ATPase (AT1A1 and AT1A2)<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>, further substantiating these cellular processes as vital targets of the coronavirus family. SARS-CoV-2 viral proteins also interacts with the subunits of 40S ribosomal proteins (RS5, RS10, RS12, RS14, RS17, RS18, and RS27), T complex proteins (TCPA, TCPD, TCPE, TCPG, TCPH, TCPQ, and TCPZ), and proteasome-related proteins (PRS4, PRS7, PRS8, PSA, PSA4, PSA6, PSA7, PSD11, PSD13, PSMD1, PSMD2, PSMD6, and PSMD7). These viral-cellular protein interactions might have extensive cytopathic effects on the host cell, often altering host gene translation, protein folding, and degradation pathways during viral infection to assist viral growth and proliferation (<strong>Extended Data Fig.3a-b)</strong>. The virus may also co-opt the complex membranous network surrounding and residing within the host cell, such as the endoplasmic reticulum, Golgi apparatus, and plasma membranes, so long as the membrane systems possess host proteins that lead to a productive infection (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig.2</a></strong>). Most importantly, the AP-LC-MS analysis uncovered host targets pertinent in inflammatory and innate immune responses, which may help explain the common respiratory symptoms. Key viral-cellular PPIs that may influence the pathogenesis of SARS-CoV-2 include interaction between NKRF (an endogenous repressor of IL-8/IL-6 synthesis) and nsp9/nsp10/nsp12/nsp13/nsp15<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>, C1QBP (involved in complement C1 activation) and nsp9/nsp10<sup>22</sup>, G3BP2 (anti-viral stress granule protein) and N<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, and RAE1 (an interferon-inducible mRNA nuclear export protein) and ORF6<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. These observations provide an overview of host proteins or pathways that are modulated by SARS-CoV-2 viral proteins and these PPIs might serve as potential targets for drug development or repurposing against COVID-19.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV-2-human protein-protein interaction network Network representation of the high-confidence SARS-CoV-2-host interactome in HEK293 cells. There are 27 SARS-CoV-2 bait proteins (red squares) and 251 interacting host protein (circles). Subcellular localizations of host proteins are labelled with indicated colors. Proteins with known functions in inflammation or immune responses are circled in purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1483586763" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;SARS-CoV-2-human protein-protein interaction network&lt;/span&gt; Network representation of the high-confidence SARS-CoV-2-host interactome in HEK293 cells. There are 27 SARS-CoV-2 bait proteins (red squares) and 251 interacting host protein (circles). Subcellular localizations of host proteins are labelled with indicated colors. Proteins with known functions in inflammation or immune responses are circled in purple.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F2.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F2.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">SARS-CoV-2-human protein-protein interaction network</span><p id="p-12" class="first-child">Network representation of the high-confidence SARS-CoV-2-host interactome in HEK293 cells. There are 27 SARS-CoV-2 bait proteins (red squares) and 251 interacting host protein (circles). Subcellular localizations of host proteins are labelled with indicated colors. Proteins with known functions in inflammation or immune responses are circled in purple.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Global proteome profiling of PBMCs from COVID-19 patients</h3><p id="p-13">To elucidate the mechanism and effect of SARS-CoV-2 infection in patients, we determined the global proteome profile changes in PBMCs samples taken from healthy donors compared with PBMCs collected from COVID-19 patients with mild or severe symptoms. The PBMC samples were obtained from the Shanghai Public Health Clinical Center and includes 6 healthy donors, 22 COVID-19 patients with mild symptoms (COVID-19-MS), and 13 COVID-19 patients with severe symptoms (COVID-19-SS) (<strong>Extended Data Fig.4</strong>). PBMCs were harvested from the leftover blood samples and all the patients were confirmed negative for influenza A virus. Following the Guideline of Novel Coronavirus Laboratory Safety (Edition 2), PBMCs were collected, lysed in denaturing buffer (see Methods) at the Shanghai Public Health Clinical Center and sent for proteome analysis at the National Facility for Protein Science in Shanghai.</p><p id="p-14">A total of 6,739 proteins were identified in our study, of which 4,274 proteins could be quantified in all 26 PMBCs samples. Proteins with less than three quantitation values did not pass filtering threshold and were excluded from the quantitation and statistical analysis. 5,522 proteins were quantified according to criteria and included in global proteomic profiling to reveal a list of 220 proteins found to be differentially expressed [|log2 (fold change)| ≥ 1, n≥3] in the PBMCs of COVID-19-MS compared with that of healthy donors: 115 proteins showed a statistically significant increase while 105 proteins showed reduction in expression when comparing COVID-19-MS with control. (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig.3a</a></strong>, <strong>Extended Data Fig.5a-b and Table 2</strong>). Functional analysis by DAVID and Cluster Profiler revealed that these genes were enriched in specific cellular biological processes, including positive regulation of neutrophil chemotaxis, neutrophil activation, type I interferon signaling pathway, inflammatory response, and antigen processing and presentation (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig.3b</a></strong> and <strong>Extended Data Fig.5c-d</strong>), suggesting an active innate immune response against SARS-CoV-2 infection. Analysis of the global proteomic changes reveal significant upregulation of CXCR2, PRG3, LBP, MMP25, CRP, NLRP1 in the COVID-19-MS PBMC cohort (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig.3c</a></strong>). CXCR2 is a subunit of IL-8 receptor and facilitates IL-8-induced neutrophil chemotaxis while PRG3 has been reported to stimulate neutrophil superoxide production and IL-8 release<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. LBP positively regulates neutrophil response upon LPS stimulation and MMP25 inactivates alpha-1 proteinase inhibitor to facilitate the trans-endothelial migration of neutrophil to inflammatory sites during infection<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. These differentially expressed proteins support a model by which SARS-CoV-2 triggers cytokine release, such as IL-8, in the early stage of infection, resulting in uncontrolled infiltration and activation of neutrophils to the sites of infection followed by eventual progression to cytokine storm and respiratory distress. Indeed, laboratory results reveal increased neutrophils to ~90% of the white blood cell count as COVID-19 progresses to critical stages<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. The elevation of IL-8 and IL-6 but not IL-17 in PMBCs collected from COVID-19 patients indicates possible roles for IL-8/IL-6 in the severe inflammatory response induced by SARS-CoV-2 (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig.3d-e</a></strong> and <strong>Extended Data Fig.5e</strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proteome profile change in PBMCs of COVID-19 patients suggests a potential role of the nsp9/nsp10-NKRF-IL-6/8 axis in neutrophil activation (a-c) Comparative proteomic analysis of COVID-19 patients with mild symptoms (N = 13) compared to healthy control (N = 3). In the volcano plot, each circle represents a protein that was quantifiable in at least three biological replicates. Significantly up- or down-regulated proteins (Adjusted two-sided P&#x2264;0.05; |log2(fold change)| &#x2265; 1) are shown in red and blue, respectively. Differentially expressed proteins related to inflammatory pathways are upregulated and labeled (a). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (b). Heat map of the differential expressed proteins (|log2(fold change)| &#x2265; 1) (c). (d-e) The levels of IL-8 (d) and IL-6 (e) in PBMCs from COVID-19 patients with mild or severe symptoms or recovered patients (viral RNA negative) were determined by FACS analysis. (f-h) Comparative proteomic analysis of COVID-19 patients with severe symptoms (N = 10) compared to mild symptoms (N = 13). Each circle represents a protein that was quantifiable in at least three biological replicates in volcano plot. Significantly up- or down-regulated proteins (Adjusted two-sided P&#x2264;0.05; |log2(fold change)| &#x2265; 1) are shown in red and blue, respectively. Adaptive immunity pathway related differentially expressed proteins are downregulated and labeled (f). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (g). Heat map of the differential expressed proteins (|log2(fold change)| &#x2265; 1) (h). (i-j) Individual stable expression of nsp9 or nsp10 from SARS-CoV-2 significantly promotes the inductions of IL-8 (i) and IL-6 (j) in the lung epithelial cell line. Mean &#xB1; SD; ns, and ***p &lt; 0.001 by Student&#x2019;s t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1483586763" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Proteome profile change in PBMCs of COVID-19 patients suggests a potential role of the nsp9/nsp10-NKRF-IL-6/8 axis in neutrophil activation&lt;/span&gt; (&lt;strong&gt;a-c&lt;/strong&gt;) Comparative proteomic analysis of COVID-19 patients with mild symptoms (N = 13) compared to healthy control (N = 3). In the volcano plot, each circle represents a protein that was quantifiable in at least three biological replicates. Significantly up- or down-regulated proteins (Adjusted two-sided P&#x2264;0.05; |log2(fold change)| &#x2265; 1) are shown in red and blue, respectively. Differentially expressed proteins related to inflammatory pathways are upregulated and labeled (&lt;strong&gt;a&lt;/strong&gt;). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (&lt;strong&gt;b&lt;/strong&gt;). Heat map of the differential expressed proteins (|log2(fold change)| &#x2265; 1) (&lt;strong&gt;c&lt;/strong&gt;). (&lt;strong&gt;d-e&lt;/strong&gt;) The levels of IL-8 (&lt;strong&gt;d&lt;/strong&gt;) and IL-6 (&lt;strong&gt;e&lt;/strong&gt;) in PBMCs from COVID-19 patients with mild or severe symptoms or recovered patients (viral RNA negative) were determined by FACS analysis. (&lt;strong&gt;f-h&lt;/strong&gt;) Comparative proteomic analysis of COVID-19 patients with severe symptoms (N = 10) compared to mild symptoms (N = 13). Each circle represents a protein that was quantifiable in at least three biological replicates in volcano plot. Significantly up- or down-regulated proteins (Adjusted two-sided P&#x2264;0.05; |log2(fold change)| &#x2265; 1) are shown in red and blue, respectively. Adaptive immunity pathway related differentially expressed proteins are downregulated and labeled (&lt;strong&gt;f&lt;/strong&gt;). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (&lt;strong&gt;g&lt;/strong&gt;). Heat map of the differential expressed proteins (|log2(fold change)| &#x2265; 1) (&lt;strong&gt;h&lt;/strong&gt;). (&lt;strong&gt;i-j&lt;/strong&gt;) Individual stable expression of nsp9 or nsp10 from SARS-CoV-2 significantly promotes the inductions of IL-8 (&lt;strong&gt;i&lt;/strong&gt;) and IL-6 (&lt;strong&gt;j&lt;/strong&gt;) in the lung epithelial cell line. Mean &#xB1; SD; ns, and ***&lt;em&gt;p&lt;/em&gt; &lt; 0.001 by Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F3.medium.gif" width="275" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F3.medium.gif" width="275" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Proteome profile change in PBMCs of COVID-19 patients suggests a potential role of the nsp9/nsp10-NKRF-IL-6/8 axis in neutrophil activation</span><p id="p-15" class="first-child">(<strong>a-c</strong>) Comparative proteomic analysis of COVID-19 patients with mild symptoms (N = 13) compared to healthy control (N = 3). In the volcano plot, each circle represents a protein that was quantifiable in at least three biological replicates. Significantly up- or down-regulated proteins (Adjusted two-sided P≤0.05; |log2(fold change)| ≥ 1) are shown in red and blue, respectively. Differentially expressed proteins related to inflammatory pathways are upregulated and labeled (<strong>a</strong>). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (<strong>b</strong>). Heat map of the differential expressed proteins (|log2(fold change)| ≥ 1) (<strong>c</strong>). (<strong>d-e</strong>) The levels of IL-8 (<strong>d</strong>) and IL-6 (<strong>e</strong>) in PBMCs from COVID-19 patients with mild or severe symptoms or recovered patients (viral RNA negative) were determined by FACS analysis. (<strong>f-h</strong>) Comparative proteomic analysis of COVID-19 patients with severe symptoms (N = 10) compared to mild symptoms (N = 13). Each circle represents a protein that was quantifiable in at least three biological replicates in volcano plot. Significantly up- or down-regulated proteins (Adjusted two-sided P≤0.05; |log2(fold change)| ≥ 1) are shown in red and blue, respectively. Adaptive immunity pathway related differentially expressed proteins are downregulated and labeled (<strong>f</strong>). Gene Ontology analysis shows the enrichment of biological processes with significantly up- or down-regulated proteins (<strong>g</strong>). Heat map of the differential expressed proteins (|log2(fold change)| ≥ 1) (<strong>h</strong>). (<strong>i-j</strong>) Individual stable expression of nsp9 or nsp10 from SARS-CoV-2 significantly promotes the inductions of IL-8 (<strong>i</strong>) and IL-6 (<strong>j</strong>) in the lung epithelial cell line. Mean ± SD; ns, and ***<em>p</em> &lt; 0.001 by Student’s <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Next, we compared the proteome change between PBMCs isolated from COVID-19-SS and COVID-19-MS patients. Amongst 6,235 quantified proteins (n≥3), 553 proteins were found to be significantly differentially expressed [|log2 (fold change)| ⩾ 1, adjusted two-sided <em>p</em> Value ⩽0.05] between COVID-19SS and COVID-19 MS’s PMBCs. 27 of these proteins were upregulated and 526 of these proteins were downregulated in COVID-19-SS as compared to COVID-19-MS (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig.3f</a></strong>, <strong>Extended Data Fig.6a-b</strong> and <strong>Table 3</strong>). Functional analysis by DAVID and Cluster Profiler revealed that these genes are highly enriched in pathways associated with regulation of immune response, T cell co-stimulation and activation, complement activation, B cell receptor signaling pathway, and T cell receptor signaling (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig.3g</a></strong> and <strong>Extended Data Fig.6c-d</strong>). The expressions of T cell receptor (TCR) subunits (CD3e, CD3g, CD247, TRAC, and TRBC1), T cell surface molecules (CD4, CD8a, CD8b, and CD2), MHC class II molecules (HLA-DRA, HLA-DRB1, HLA-DRB4, and HLA-DRB5), T cell migration stimulators (DDP4), and TCR signaling kinases (ZAP70, LCK, and FYN) were dramatically diminished in COVID-19-SS as compared to COVID-19-MS (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig.3a,h</a></strong>), suggesting a failure of T cells activation and function. The expressions of immunoglobulin subunits (IGHG1, IGHG2, IGHG4, IGLC2, IGLL1, and IGHE) were also significant lower in COVID-19-SS (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig.3a,h</a></strong>), indicating reduced antibody secretion by B cells. These results demonstrated that more severe COVID-19 patients undergo a functional decline in adaptive immunity, which is correlated with their lower T cell and B cell populations<sup><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a></sup>.</p></div><div id="sec-5" class="subsection"><h3>SARS-CoV-2 nsp9 and nsp10 target NKRF to facilitate IL-8/IL-6 production</h3><p id="p-17">From the quantitative proteome analysis of PBMCs, it is clear that SARS-CoV-2 induces IL-8/IL-6 expression and increases neutrophil count which may mediate the manifestation of acute respiratory distress syndrome in COVID-19 patients. Therefore, we hypothesized that SARS-CoV-2 virulence factors may potentiate or regulate the induction of IL-8/IL-6 during infection, leading to the uncontrolled infiltration and activation of neutrophils. NKRF is a transcriptional repressor that inhibit IL-8 and IL-6 induction by competing with NF-КB for promoter binding<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. From our virus-host interactome, we discovered that NKRF interacts with SARS-CoV-2 nsp9, nsp10, nsp12, nsp13, and nsp15 (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig.2</a></strong> and <strong>Extended Data Fig.7a</strong>). Among these SARS-CoV-2 viral proteins, individually expressed nsp9 or nsp10 facilitated the induction of both IL-8 and IL-6 in lung epithelial A549 cells, while individually expressed nsp12, nsp13, or nsp15 had little or no effects (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3i-j</a></strong>). The levels of TNFα was not affected by either nsp9 or nsp10 (<strong>Extended Data Fig.7b</strong>), suggesting nsp9 and nsp10 does not affect NK-КB signaling transduction overall. Importantly, NKRF deficiency by specific siRNA facilitated the induction of IL-8 and diminished the effect of nsp10, indicating that nsp10 potentially targeted NKRF to regulate IL-8 induction (<strong>Extended Data Fig.7c-d</strong>).</p><p id="p-18">Similar to other coronaviruses, SARS-CoV-2 has underwent rapid adaptive changes against evolutionary pressures since its isolation<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. Phylogenetic and genetic analysis<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> of 160 public SARS-CoV-2 sequences reveals 137 amino acid substitutions in 23 viral proteins with no mutations occurring in nsp9 (<strong>Extended Data Fig.7e</strong> and <strong>Table 4</strong>), which indicates nsp9 has remained relatively stable. Unlike nsp9, nsp10 picked up only two amino acid substitutions (A61G and F68L), but with very low variation rate (1/160 for each site) during evolution (<strong>Extended Data Fig.7f</strong>), indicating that nsp9 and nsp10 are potential stable targets to curb SARS-CoV-2-mediated IL-8/IL-6 activation and subsequently recruitment of neutrophils.</p><p id="p-19">Our study integrated virus-host interaction network and quantitative proteome approaches to identify host targets of SARS-CoV-2 viral proteins, demonstrating that SARS-CoV-2 evolved multiple mechanisms to modulate host fundamental cellular processes, which ultimately contributes to its immune evasion and pathogenesis. To our knowledge, this is the first study to characterize the PBMC proteome profile in both mild and severe cases of COVID-19, corroborating the overall clinical features of COVID-19 patients at molecular levels. These systematic analyses not only generate groundwork for future SARS-CoV-2 research on the nature of this virus but also provide valuable targets for specific drug development or repurposing. With these platforms, we identified nsp9 and nsp10 as the potential virulence factors for SARS-CoV-2, which interacts with NKRF to facilitate IL-8/IL-6 induction, ultimately leading to uncontrolled infiltration and activation of neutrophils (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig.4</a></strong>). Specific agents targeting IL-8/IL-6 or the downstream JAK kinases could be repurposed as suitable drug candidates for clinical evaluation. These small molecule inhibitors (i.e. IL-6 or IL-8 receptor antagonists) or agents blocking virulence factor PPIs would have enormous value as anti-virulence therapies in the treatment of SARS-CoV-2 infections (<strong>Fig.4</strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models of nsp9/nsp10-mediated SARS-CoV-2 pathogenesis and potential design or repurposing for specific inhibitors " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1483586763" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Models of nsp9/nsp10-mediated SARS-CoV-2 pathogenesis and potential design or repurposing for specific inhibitors&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F4.medium.gif" width="440" height="121"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F4.medium.gif" width="440" height="121"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.03.31.019216/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Models of nsp9/nsp10-mediated SARS-CoV-2 pathogenesis and potential design or repurposing for specific inhibitors</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">AUTHOR CONTRIBUTION</h2><p id="p-20">Q.L., T.Z., and C.P. conceived of the research, designed the study, and wrote the manuscript. M.G. isolated and lysed PBMCs in denaturing buffer. Z.L., J.L., C.L., X.W., and X.Y. performed the experiments and analyzed data. X.T., P.W., and Y.Y performed the LC-MS analysis. Z.X. helped with visual representation of data and edited the manuscript. All authors commented on the manuscript.</p></div><div class="section" id="sec-7"><h2 class="">METHODS</h2><div class="section" id="sec-8"><h3>Ethics statement</h3><p id="p-22">This study was approved by the Ethics Committee of Shanghai Public Health Clinical Center (#YJ-2020-S052-02). The proteomic analysis of PBMC samples were performed on existing samples collected during standard diagnostic tests, posing no extra burden to patients.</p></div><div class="section" id="sec-9"><h3>Cell lines, plasmids, and antibodies</h3><p id="p-23">HEK293 and A549 cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM; Gibco-BRL) containing 4 mM glutamine and 10% FBS. Vero cells were cultured in DMEM with 5% FBS and 4 mM glutamine. Transient transfections were performed with Lipofectamine 3000 (Thermo Fisher Scientific, #3000015). SARS-CoV-2 genes were synthesized as DNA fragments (Sangon Biotech) and these expression constructs were amplified by PCR and cloned into pLVX-3xFlag-MCS-P2A-tagRFP (puro), pEF-MCS-3xHA, pGBTK7, or pACT2 vectors. Mutations of nsp10 were generated using QuikChange site-directed mutagenesis kit (Stratagene). All constructs were sequenced using an ABI PRISM 377 automatic DNA sequencer to verify 100% correspondence with the original sequence. Primary antibodies were purchased from the following venters: mouse anti-Flag</p></div><div class="section" id="sec-10"><h3>Purification of ZIKV viral protein complexes</h3><p id="p-24">HEK293 were transfected with 3xFlag-tagged plasmids encoding each SARS-CoV-2 protein by Lipofectamine 3000 (Thermo Fisher Scientific, #L3000015). 3×10<sup>7</sup> HEK293 cells were harvested in 10 ml lysis buffer (50 mM Tris-HCl [pH 7.5], 150 mM NaCl, 0.5% Nonidet P40, 10% glycerol, phosphatase inhibitors and protease inhibitors) at 72 h post transfection, followed by centrifugation and filtration (0.45 μm) to remove debris. Cell lysates were precleared with 100 μl protein A/G resin, and then incubated with 25 μl anti-Flag M2 resin (Sigma, #F2426) for 12 h at 4°C on a rotator. The resin was washed 5 times and transferred to a spin column with 40 μl of 3 X Flag peptide (Sigma, #F4799) for 1 h at 4°C on a rotator. The purified complexes were sent for mass spectrometry analysis for protein-protein interaction network at the National Facility for Protein Science in Shanghai.</p></div><div class="section" id="sec-11"><h3>Virus-host Interaction network analysis</h3><p id="p-25">The samples (purified complexes) were precipitated by TCA and resolved in 8 M urea, and then treated with 5 mM TCEP and 10 mM IAA to reduce the disulfide bonds and alkylate the resulting thiol groups, sequentially. The mixture was digested for 16 h at 37 °C by trypsin at an enzyme-to-substrate ratio of 1:50 (wt/wt). The trypsin-digested peptides were desalted by monospin C18 and loaded on a home-made 40 cm-long pulled-tip analytical column (75 μm ID x 360μm OD, ReproSil-Pur C18-AQ 1.9 μm resin, Dr. Maisch GmbH), connected to a Thermo Easy-nLC1000 HPLC system. The samples were analyzed with a 120 min-HPLC gradient from 0 to 100% of buffer B (buffer A: 0.1% formic acid in water; buffer B: 0.1% formic acid in 80% acetonitrile) at 300 nL/min : 4-8 min, 0%-4% B;1-96 min, 8-35% B; 97-104min, 35-60% B; 104-105min, 60-100% B; 105-120min, 100% B. The eluted peptides were ionized and directly introduced into a Q-Exactive mass spectrometer using a nano-spray source with a distal 2.0-kV spray voltage. Survey full-scan MS spectra (from m/z 300-1,800) was acquired in the Orbitrap analyzer with resolution r = 70,000 at m/z 200. One acquisition cycle includes one full-scan MS spectrum followed by top 20 MS/MS events, sequentially generated on the first to the twentieth most intense ions selected from the full MS spectrum at a 28% normalized collision energy. The acquired MS/MS data were analyzed against a UniProtKB Human database (database released on Sept. 30, 2018) containing SARS-CoV-2 viral proteins using Maxquant V1.6.10.43. In order to accurately estimate peptide probabilities and false discovery rates (FDR), we used a decoy database containing the reversed sequences of all the proteins appended to the target database. FDR was set at 0.01. Mass tolerance for precursor ions was set at 20 ppm. Trypsin was defined as cleavage enzyme and the maximal number of missed cleavage sites was set at 3. Carbamidomethylation (+57.02146) of cysteine was considered as a static modification. Methionine oxidation was set as variable modifications.</p></div><div class="section" id="sec-12"><h3>Proteomic study of PBMCs from COVID-19 patients</h3><div class="section" id="sec-13"><h4>Preparation of PMBC cells from patients</h4><p id="p-26">The information of 35 patients (13 COVID-19 patients with severe symptoms and 22 COVID-19 patients with mild symptoms) with confirmed SARS-CoV-2 infection and 6 health donors as control was listed in Extended Data Fig.4, including age (47-73), sex, and symptoms. This study followed the Guideline of Novel Coronavirus Laboratory Safety (Edition 2) and was approved by Shanghai Public Health Clinical Center (#YJ-2020-S052-02). The peripheral venous whole blood samples were collected from all the patients and PBMCs were prepared using density-gradient centrifugation followed by washing with cold PBS twice. The cell pellets were lysed by 8 Urea (in 20mM Tris-HCl with protein inhibitor cocktail and phosphor-STOP cocktail, pH 8.0) to denature the proteins and sonicated twice on ice to break the protein-protein or DNA-protein interactions. All the procedures were carried out within the certified laboratory for studies of infectious materials. Then protein samples were combined to generate 3 control samples from 6 healthy donors, 13 mild-case samples from 22 COVID-19-MS and 10 severe-case samples from 13 COVID-19-SS according to the protein concentrations of all the samples. The resulted protein soups were then applied for protein digestion for the following proteomic experiments.</p></div><div class="section" id="sec-14"><h4>Protein Digestion</h4><p id="p-27">TCEP (tris(2-carboxyethyl) phosphine, final concentration is 5 mM) (Thermo Scientific) and iodoacetamide (final concentration is 10mM) (Sigma) for reduction and alkylation were added to the solution and incubated at room temperature for 30 minutes, respectively. The protein mixture was diluted four times and digested with LysC at 1:50(w/w) for four hours and then with Trypsin at 1:40 (w/w) (Promega, <a href="http://www.promega.com/">http://www.promega.com/</a>) overnight.</p></div><div class="section" id="sec-15"><h4>TMT labeling</h4><p id="p-28">The concentrations of the peptide soups were measured with Quantitative Colorimetric Peptide Assay (Thermo Scientific). TMT10plex amino reactive reagents (0.8 mg per vial, Thermo Scientific) were suspended in 41 μl of anhydrous acetonitrile and added to peptide soups (100 μg peptides in total). Reactions were allowed to proceed at room temperature for 1 hr, and then quenched by the addition of 8 μl of 5% hydroxylamine for 15 min. The TMT-labeled samples were pooled at a 1:1:1:1:1:1:1:1:1:1 ratio. The mixture was vacuum centrifuged to near dryness and desalted on a MonoSpin™ C18 (GL Science).</p></div><div class="section" id="sec-16"><h4>Off-line high pH-reversed phase fractionation</h4><p id="p-29">The desalted TMT-labeled peptides were resuspended in 300 ul 10% buffer C (50 mM ammonium hydroxide, pH 10). Off-line fractionations were performed using a Agilent 1260 HPLC with a 4.6 mm × 250 mm BEH C18, 3.5 μm column (Waters) by a 58 min gradient from 5% to 80% buffer B (100% acetonitrile) with buffer A (100% water) at a flow rate of 0.5 mL/min while buffer C was running constantly at 10% of total flow rate during the process. Fractions were collected over 75 min at 1 min intervals after the start of the gradient in 1.5 ml Eppendorf tubes. The peptide mixture was fractionated into a total of 50 fractions which were consolidated into 18 vials. Samples were subsequently desalted and vacuum centrifuged to near dryness. The desalted peptides were reconstituted in 0.1% fomic acid for downstream LC-MS/MS analysis.</p></div><div class="section" id="sec-17"><h4>LC-MS/MS analysis of peptides</h4><p id="p-30">For proteome study, all the 18 fractions for each TMT experiment were analyzed by a home-made 40 cm-long pulled-tip analytical column (75 μm ID x 360μm OD, ReproSil-Pur C18-AQ 1.9 μm resin, Dr. Maisch GmbH), the column was then placed in-line with an Easy-nLC 1200 nano HPLC (Thermo Scientific, San Jose, CA) for mass spectrometry analysis. The analytical column temperature was set at 55 ℃ during the experiments. The mobile phase and elution gradient used for peptide separation were as follows: 0.1% formic acid in water as buffer A and 0.1% formic acid in 80% acetonitrile as buffer B, 0-1 min, 5%-10% B; 1-96 min, 10-30% B; 96-104 min, 30%-50% B, 104-105 min, 50%-100% B, 105-120 min, 100% B. The flow rate was set at 300 nL/min. Data-dependent tandem mass spectrometry (MS/MS) analysis was performed with a Q Exactive Orbitrap mass spectrometer (Thermo Scientific, San Jose, CA). Peptides eluted from the LC column were directly electrosprayed into the mass spectrometer with the application of a distal 2.2-kV spray voltage. A cycle of one full-scan MS spectrum (m/z 300-1800) was acquired followed by top 20 MS/MS events (first mass fixed at 100 m/z scan range), sequentially generated on the first to the twentieth most intense ions selected from the full MS spectrum at a 35% normalized collision energy. Full scan resolution was set to 70,000 with automated gain control (AGC) target of 3e6. MS/MS scan resolution was set to 35,000 with isolation window of 1.8 m/z and AGC target of 1e5. The number of microscans was one for both MS and MS/MS scans and the maximum ion injection time was 50 and 100 ms, respectively. The dynamic exclusion settings used were as follows: charge exclusion, 1 and &gt;8; exclude isotopes, on; and exclusion duration, 30 seconds.</p></div><div class="section" id="sec-18"><h4>Data Analysis</h4><p id="p-31">The acquired MS/MS data were analyzed against a UniProtKB human (database released on Sept. 30, 2018) by Maxquant V1.6.10.43. Tolerance of precursor mass and fragment mass were set to ±20ppm. Carbamidomethylation of cysteine (+57.021 Da), TMT-labeled N-terminus and lysine (+229.163 Da) were set as static modifications. Oxidation of methionine (+15.995 Da) was set as variable modification. All identified proteins had a FDR ≤1%, which was calculated at the peptide level.</p></div><div class="section" id="sec-19"><h4>Merging of datasets and statistical analysis</h4><p id="p-32">The interpretation of three sets of proteomic results (protein Groups.txt) from MaxQuant were done in the Perseus software environment (version 1.6.2.3) and Python3. Briefly, the datasets were merged by matching the protein accession numbers of each individual dataset for combination by Python in-house developed scripts (comparepg.py). And the protein abundances were normalized by the TMT-labeled internal standard samples which were consisted of an aliquot of each sample and spiked into the 3 sets of TMT labeling experiments. All the normalized reporter ion abundances of biological repeats were averaged for fold-change calculation. For AP-MS analysis, a two-sided two sample t-test was used to obtain the statistical significance to compare the results of the two groups. For proteome analysis, p value was calculated by empirical Bayes moderated t-tests and then adjusted by Benjamini and Hochberg method, using <em>limma</em> package<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> built in R environment. Proteins were required to have a |log2(fold change)|⩾1 and adj. p value cut-off of 0.05 to be considered as being significantly differentially expressed. Gene ontology (GO) analysis was performed using DAVID Bioinformatics Resources 6.8 (<a href="https://david.ncifcrf.gov/">https://david.ncifcrf.gov/</a>) and ClusterProfiler v3.14.3<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>.</p></div></div><div class="section" id="sec-20"><h3>Detection of SARS-CoV-2 intra-viral PPIs by yeast-two-hybrid</h3><p id="p-33">28 SAR-CoV-2 genes were cloned into both pGBTK7 (bait) and pACT2 (prey) vectors. The reporter Y2HGold yeast strain (Clontech) was co-transformed with different bait-prey combinations as indicated in Extended Data Fig.1 in accordance with instructions for Matchmaker GAL4 Two-Hybrid System 3 (Clontech). The transformed yeast was then screened in 4-dropout plates for the protein-protein interaction.</p></div><div class="section" id="sec-21"><h3>RNA extraction and quantitative RT-PCR</h3><p id="p-34">Total RNA was isolated from cells with the RNeasy Mini Kit (Qiagen, #74106) and treated with RNase-free DNase according to the manufacturer’s protocol. All gene transcripts were quantified by quantitative PCR using qScript<sup>TM</sup> One-Step qRT-PCR Kit (Quanta Biosciences, #95057-050) on CFX96 real-time PCR system (Bio-Rad). Primer sequences for qPCR were as follow: IL-6 forward: AACCTGAACCTTCCAAAGATGG, IL-6 reverse: TCTGGCTTGTTCCTCACTACT; IL-8 forward: TCTTGCACAAATATTTGATGC, IL-8 reverse: CCACTGTGCCTTGGTTTC; GAPGH forward: GAGTCAACGGATTTGGTCGT, GAPGH reverse: TTGATTTTGGAGGGATCTCG; TNFα forward: CTCCAGGCGGTGCTTGTTC, TNFα reverse: GGCTACAGGCTTGTCACTCG; NKRF forward: GTAAACATGCAGCTGCCGAC, NKRF reverse: CGTGCACACGGGATTTGAAG.</p></div><div class="section" id="sec-22"><h3>siRNA gene silencing</h3><p id="p-35">Small interfering RNA (siRNA) targeting human NKRF (#A10001) and negative control siRNAs were purchased from GenePharma. siRNAs were delivered to the A549 cells using Lipofectamine RNAiMAX Transfection Reagent (Thermo Fisher Scientific, #13778030) according to the manufacturer’s instructions. After 48 h of transfection, cells were verified for target gene knock-down by quantitative RT-PCR.</p></div><div class="section" id="sec-23"><h3>Phylogenetic and genetic analysis of SARS-CoV-2 sequences</h3><p id="p-36">160 SARS-CoV-2 isolates with complete genome sequences from 18 different countries during December 23, 2019 to February 29, 2020 were used for molecular evolutionary analysis in this study. Multiple alignment was performed by Clustal W and phylogenetic tree was constructed by Neighbor-Joining statistical method based on the Maximum Composite Likelihood model with 500 replicates of bootstrap testing using MEGA<sup><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup>.</p></div><div class="section" id="sec-24"><h3>Quantification and Statistical Analysis</h3><p id="p-37">All data were expressed as Mean ± SD as indicated. For parametric analysis, the F test was used to determine the equality of variances between the groups compared; statistical significance across two groups was tested by Student’s t-test; one-way analysis of variance (ANOVA) followed by Bonferroni’s <em>post hoc</em> test were used to determine statistically significant differences between multiple groups. <em>P</em>-values of less than 0.05 were considered significant.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-21">This work was supported by grants from National Key Research and Development Project of China (2018YFA0900802), National Natural Science Foundation of China (31770176 and 31500667), National Science and Technology Major Project (2020ZX09201001), the Program for Professor of Special Appointment (Eastern Scholar) at Shanghai Institutions of Higher learning, Shanghai Science and Technology Commission (2017QA1403200), Shanghai Municipal Health Commission (2018YQ40 and 201940179), Shanghai Municipal Key Clinical Specialty (shslczdzk01102), Innovative Research Team of High-level Local Universities in Shanghai, the Interdisciplinary Program of Shanghai Jiao Tong University (YG2020YQ14), Chinese Academy of Sciences Large Research Infrastructure of Maintenance and Remolding Project (DSS-WXGZ-2020-0001), and Chinese Academy of Science Key Technology Talent Program.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCE</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.1" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.31.019216v1.2" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.3" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China</span>, 2019. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31978945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.4" data-doi="10.1093/jtm/taaa015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">M. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L. H.</span></span> <span class="cit-article-title">Travelers Give Wings to Novel Coronavirus (2019-nCoV)</span>. <abbr class="cit-jnl-abbrev">J. Travel Med</abbr>. (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/jtm/taaa015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BTravel%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjtm%252Ftaaa015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jtm/taaa015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.5"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The Lancet, null. Emerging understandings of 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">Lancet Lond. Engl</abbr>. <span class="cit-vol">395</span>, <span class="cit-fpage">311</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BLond.%2BEngl%26rft.volume%253D395%26rft.spage%253D311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.6" data-doi="10.1101/2020.02.06.20020974"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">W</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical characteristics of 2019 novel coronavirus infection in China</span>. <abbr class="cit-jnl-abbrev">medRxiv 2020.02.06.20020974</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.06.20020974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmedRxiv%2B2020.02.06.20020974%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.06.20020974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMC4wMi4wNi4yMDAyMDk3NHYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wMy4zMS4wMTkyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.7" data-doi="10.1016/j.cell.2020.02.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.052%26rft_id%253Dinfo%253Apmid%252F32142651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32142651&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.31.019216v1.8" data-doi="10.1016/j.cell.2020.02.058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.058%26rft_id%253Dinfo%253Apmid%252F32155444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32155444&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.31.019216v1.9" data-doi="10.1126/science.abb2762"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2762%26rft_id%253Dinfo%253Apmid%252F32132184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4NS8xNDQ0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wMy4zMS4wMTkyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.10" data-doi="10.1126/science.abb3405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb3405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb3405%26rft_id%253Dinfo%253Apmid%252F32198291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiJzY2llbmNlLmFiYjM0MDV2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAyLzIwMjAuMDMuMzEuMDE5MjE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet Lond. Engl</abbr>. <span class="cit-vol">395</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BLond.%2BEngl%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.12" data-doi="10.1097/CM9.0000000000000722"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.-L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of a novel coronavirus causing severe pneumonia in human: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Chin. Med. J. (Engl.)</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1097/CM9.0000000000000722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin.%2BMed.%2BJ.%2B%2528Engl.%2529%26rft_id%253Dinfo%253Adoi%252F10.1097%252FCM9.0000000000000722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/CM9.0000000000000722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.13" data-doi="10.1038/nri3399"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolaczkowska</span>,  <span class="cit-name-given-names">E</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Kubes</span>,  <span class="cit-name-given-names">P</span></span>. <span class="cit-article-title">Neutrophil recruitment and function in health and inflammation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">159</span>–<span class="cit-lpage">175</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BImmunol%26rft.volume%253D13%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3399%26rft_id%253Dinfo%253Apmid%252F23435331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri3399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23435331&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.14" data-doi="10.21203/rs.3.rs-19346/v1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Aveolar Macrophage Activation and Cytokine Storm in the Pathogenesis of Severe COVID-19</abbr>. (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.21203/rs.3.rs-19346/v1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAveolar%2BMacrophage%2BActivation%2Band%2BCytokine%2BStorm%2Bin%2Bthe%2BPathogenesis%2Bof%2BSevere%2BCOVID-19%26rft_id%253Dinfo%253Adoi%252F10.21203%252Frs.3.rs-19346%252Fv1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21203/rs.3.rs-19346/v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.15" data-doi="10.1016/j.chom.2020.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2020.02.001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2020.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.16" data-doi="10.1371/journal.ppat.1002059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>,  <span class="cit-name-given-names">E</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Crystal structure and functional analysis of the SARS-coronavirus RNA cap 2’-O-methyltransferase nsp10/nsp16 complex</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">e1002059</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DDecroly%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1002059%26rft.epage%253De1002059%26rft.atitle%253DCrystal%2Bstructure%2Band%2Bfunctional%2Banalysis%2Bof%2Bthe%2BSARS-coronavirus%2BRNA%2Bcap%2B2%2527-O-methyltransferase%2Bnsp10%252Fnsp16%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002059%26rft_id%253Dinfo%253Apmid%252F21637813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.17" data-doi="10.1073/pnas.1508686112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">Y</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">112</span>, <span class="cit-fpage">9436</span>–<span class="cit-lpage">9441</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508686112%26rft_id%253Dinfo%253Apmid%252F26159422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzMwLzk0MzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMi8yMDIwLjAzLjMxLjAxOTIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.18" data-doi="10.1128/JVI.00948-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Coronavirus nsp10/nsp16 Methyltransferase Can Be Targeted by nsp10-Derived Peptide In Vitro and In Vivo To Reduce Replication and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">8416</span>–<span class="cit-lpage">8427</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00948-15%26rft_id%253Dinfo%253Apmid%252F26041293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNi84NDE2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wMy4zMS4wMTkyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.19" data-doi="10.1371/journal.pone.0000459"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">von Brunn</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of intraviral protein-protein interactions of the SARS coronavirus ORFeome</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e459</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dvon%2BBrunn%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253De459%26rft.epage%253De459%26rft.atitle%253DAnalysis%2Bof%2Bintraviral%2Bprotein-protein%2Binteractions%2Bof%2Bthe%2BSARS%2Bcoronavirus%2BORFeome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000459%26rft_id%253Dinfo%253Apmid%252F17520018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17520018&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.20" data-doi="10.1074/jbc.M007532200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nourbakhsh</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">The NF-κB Repressing Factor Is Involved in Basal Repression and Interleukin (IL)-1-induced Activation of IL-8 Transcription by Binding to a Conserved NF-κB-flanking Sequence Element</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>, <span class="cit-fpage">4501</span>–<span class="cit-lpage">4508</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M007532200%26rft_id%253Dinfo%253Apmid%252F11071890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzYvNi80NTAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wMy4zMS4wMTkyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.21" data-doi="10.1186/s12929-014-0071-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">K.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">C.-H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kuo</span>,  <span class="cit-name-given-names">H.-P.</span></span> <span class="cit-article-title">NF-κB repressing factor downregulates basal expression and mycobacterium tuberculosis induced IP-10 and IL-8 synthesis via interference with NF-κB in monocytes</span>. <abbr class="cit-jnl-abbrev">J. Biomed. Sci.</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">71</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiomed.%2BSci.%26rft.volume%253D21%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12929-014-0071-5%26rft_id%253Dinfo%253Apmid%252F25135111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12929-014-0071-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25135111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.31.019216v1.22" data-doi="10.1084/jem.179.6.1809"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghebrehiwet</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">B. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peerschke</span>,  <span class="cit-name-given-names">E. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>,  <span class="cit-name-given-names">A. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">K. B.</span></span> <span class="cit-article-title">Isolation, cDNA cloning, and overexpression of a 33-kD cell surface glycoprotein that binds to the globular ‘heads’ of C1q. J</span>. <abbr class="cit-jnl-abbrev">Exp. Med.</abbr> <span class="cit-vol">179</span>, <span class="cit-fpage">1809</span>–<span class="cit-lpage">1821</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BMed.%26rft.volume%253D179%26rft.spage%253D1809%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.179.6.1809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1084/jem.179.6.1809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.23" data-doi="10.1128/JVI.03612-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scholte</span>,  <span class="cit-name-given-names">F. E. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Stress granule components G3BP1 and G3BP2 play a proviral role early in Chikungunya virus replication</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">4457</span>–<span class="cit-lpage">4469</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03612-14%26rft_id%253Dinfo%253Apmid%252F25653451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzgvNDQ1NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAyLzIwMjAuMDMuMzEuMDE5MjE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.24" data-doi="10.1016/j.molcel.2004.11.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faria</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">VSV disrupts the Rae1/mrnp41 mRNA nuclear export pathway</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">93</span>–<span class="cit-lpage">102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DFaria%26rft.auinit1%253DP.%2BA.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D102%26rft.atitle%253DVSV%2Bdisrupts%2Bthe%2BRae1%252Fmrnp41%2BmRNA%2Bnuclear%2Bexport%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.11.023%26rft_id%253Dinfo%253Apmid%252F15629720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.11.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15629720&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226371100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.25" data-doi="10.1074/jbc.274.20.14464"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plager</span>,  <span class="cit-name-given-names">D. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A novel and highly divergent homolog of human eosinophil granule major basic protein</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>, <span class="cit-fpage">14464</span>–<span class="cit-lpage">14473</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DPlager%26rft.auinit1%253DD.%2BA.%26rft.volume%253D274%26rft.issue%253D20%26rft.spage%253D14464%26rft.epage%253D14473%26rft.atitle%253DA%2BNovel%2Band%2BHighly%2BDivergent%2BHomolog%2Bof%2BHuman%2BEosinophil%2BGranule%2BMajor%2BBasic%2BProtein%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.20.14464%26rft_id%253Dinfo%253Apmid%252F10318872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMjAvMTQ0NjQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMi8yMDIwLjAzLjMxLjAxOTIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.26" data-doi="10.1172/JCI41649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eash</span>,  <span class="cit-name-given-names">K. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gopalan</span>,  <span class="cit-name-given-names">P. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Link</span>,  <span class="cit-name-given-names">D. C.</span></span> <span class="cit-article-title">CXCR2 and CXCR4 antagonistically regulate neutrophil trafficking from murine bone marrow</span>. <abbr class="cit-jnl-abbrev">J. Clin. Invest</abbr>. <span class="cit-vol">120</span>, <span class="cit-fpage">2423</span>–<span class="cit-lpage">2431</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DEash%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D120%26rft.issue%253D7%26rft.spage%253D2423%26rft.epage%253D2431%26rft.atitle%253DCXCR2%2Band%2BCXCR4%2Bantagonistically%2B%2Bregulate%2Bneutrophil%2Btrafficking%2B%2Bfrom%2Bmurine%2Bbone%2Bmarrow.%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI41649%26rft_id%253Dinfo%253Apmid%252F20516641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI41649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20516641&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279544000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakatomi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aida</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kusumoto</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pabst</span>,  <span class="cit-name-given-names">M. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Maeda</span>,  <span class="cit-name-given-names">K</span></span>. <span class="cit-article-title">Neutrophils responded to immobilized lipopolysaccharide in the absence of lipopolysaccharide-binding protein</span>. <abbr class="cit-jnl-abbrev">J. Leukoc. Biol.</abbr> <span class="cit-vol">64</span>, <span class="cit-fpage">177</span>–<span class="cit-lpage">184</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BLeukocyte%2BBiology%26rft.stitle%253DJ.%2BLeukoc.%2BBiol.%26rft.aulast%253DNakatomi%26rft.auinit1%253DK%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D177%26rft.epage%253D184%26rft.atitle%253DNeutrophils%2Bresponded%2Bto%2Bimmobilized%2Blipopolysaccharide%2Bin%2Bthe%2Babsence%2Bof%2Blipopolysaccharide-binding%2Bprotein%26rft_id%253Dinfo%253Apmid%252F9715256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9715256&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soria-Valles</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">MMP-25 Metalloprotease Regulates Innate Immune Response through NF-κB Signaling. J. Immunol</span>. <abbr class="cit-jnl-abbrev">Baltim. Md 1950</abbr> <span class="cit-vol">197</span>, <span class="cit-fpage">296</span>–<span class="cit-lpage">302</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBaltim.%2BMd%2B1950%26rft.volume%253D197%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Down-regulation of miR-301a suppresses pro-inflammatory cytokines in Toll-like receptor-triggered macrophages</span>. <abbr class="cit-jnl-abbrev">Immunology</abbr> <span class="cit-vol">140</span>, <span class="cit-fpage">314</span>–<span class="cit-lpage">322</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D140%26rft.spage%253D314%26rft_id%253Dinfo%253Apmid%252F23808420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23808420&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.30" data-doi="10.1101/2020.01.30.926477"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Q</span></span>. <span class="cit-article-title">Evolution and variation of 2019-novel coronavirus</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.01.30.926477</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.30.926477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.30.926477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4zMC45MjY0Nzd2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAyLzIwMjAuMDMuMzEuMDE5MjE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.31" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knyaz</span>,  <span class="cit-name-given-names">C</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K</span></span>. <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Mol</span>. <abbr class="cit-jnl-abbrev">Biol. Evol.</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">ADDITIONAL REFERENCE</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.32" data-doi="10.1093/nar/gkv007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">limma powers differential expression analyses for RNA-sequencing and microarray studies</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">e47</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv007%26rft_id%253Dinfo%253Apmid%252F25605792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25605792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.03.31.019216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.31.019216v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Q.-Y.</span></span> <span class="cit-article-title">clusterProfiler: an R package for comparing biological themes among gene clusters. Omics J</span>. <abbr class="cit-jnl-abbrev">Integr. Biol.</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">284</span>–<span class="cit-lpage">287</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr.%2BBiol.%26rft.volume%253D16%26rft.spage%253D284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.31.019216v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/758326v2" title="Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.31.015941v1" title="Identification of a common deletion in the spike protein of SARS-CoV-2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 02, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.31.019216v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.31.019216v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.31.019216v1" class="active" data-icon-position="" data-hide-link-title="0">Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1223734" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VEjnap9gDIC4v02W6H2t5Gx2ciYg2YAlkK8ueMeGUqs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1223734" id="node1223734--2" data-pisa="biorxiv;2020.03.31.019216v1" data-pisa-master="biorxiv;2020.03.31.019216" data-seqnum="1223734" data-apath="/biorxiv/early/2020/04/02/2020.03.31.019216.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingjiao</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mingquan</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Chengrong</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xin</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xing</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zixuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Yin</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Joyce</span> <span class="nlm-surname">Fu</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Zhaoqin</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Zhenshan</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Tongyu</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Qiming</span> <span class="nlm-surname">Liang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.31.019216; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.31.019216 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;title=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;title=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;text=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;title=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;t=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;title=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;title=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1223734" id="node1223734--3" data-pisa="biorxiv;2020.03.31.019216v1" data-pisa-master="biorxiv;2020.03.31.019216" data-seqnum="1223734" data-apath="/biorxiv/early/2020/04/02/2020.03.31.019216.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingjiao</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mingquan</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Chengrong</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xin</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xing</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zixuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Yin</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Joyce</span> <span class="nlm-surname">Fu</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Zhaoqin</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Zhenshan</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Tongyu</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Qiming</span> <span class="nlm-surname">Liang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.31.019216; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.31.019216 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Li%2C+Jingjiao&amp;rft.au=Guo%2C+Mingquan&amp;rft.au=Tian%2C+Xiaoxu&amp;rft.au=Liu%2C+Chengrong&amp;rft.au=Wang%2C+Xin&amp;rft.au=Yang%2C+Xing&amp;rft.au=Wu%2C+Ping&amp;rft.au=Xiao%2C+Zixuan&amp;rft.au=Qu%2C+Yafei&amp;rft.au=Yin%2C+Yue&amp;rft.au=Fu%2C+Joyce&amp;rft.au=Zhu%2C+Zhaoqin&amp;rft.au=Liu%2C+Zhenshan&amp;rft.au=Peng%2C+Chao&amp;rft.au=Zhu%2C+Tongyu&amp;rft.au=Liang%2C+Qiming"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1223734/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1223734/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1223734/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1223734/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1223734/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1223734/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1223734/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1223734/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1223734/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1223734/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1223734/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1223734/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;count=horizontal&amp;via=&amp;text=Virus-host%20interactome%20and%20proteomic%20survey%20of%20PMBCs%20from%20COVID-19%20patients%20reveal%20potential%20virulence%20factors%20influencing%20SARS-CoV-2%20pathogenesis&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.31.019216v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.31.019216v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
