<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/20/2020.02.19.950253.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.19.950253v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Pangolin homology associated with 2019-nCoV"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.19.950253"/>
  <meta name="DC.Date" content="2020-02-20"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV."/>
  <meta name="DC.Contributor" content="Tao Zhang"/>
  <meta name="DC.Contributor" content="Qunfu Wu"/>
  <meta name="DC.Contributor" content="Zhigang Zhang"/>
  <meta name="article:published_time" content="2020-02-20"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Pangolin homology associated with 2019-nCoV"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.19.950253v1"/>
  <meta name="citation_id" content="2020.02.19.950253v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.950253v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.950253v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.950253v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.19.950253"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.19.950253"/>
  <meta name="citation_author" content="Tao Zhang"/>
  <meta name="citation_author_institution" content="State key Laboratory for Conservation and Utilization of Bio-Resources in Yunnan, School of Life Sciences, Yunnan University"/>
  <meta name="citation_author" content="Qunfu Wu"/>
  <meta name="citation_author_institution" content="State key Laboratory for Conservation and Utilization of Bio-Resources in Yunnan, School of Life Sciences, Yunnan University"/>
  <meta name="citation_author" content="Zhigang Zhang"/>
  <meta name="citation_author_institution" content="State key Laboratory for Conservation and Utilization of Bio-Resources in Yunnan, School of Life Sciences, Yunnan University"/>
  <meta name="citation_author_email" content="zhangzhigang@ynu.edu.cn"/>
  <meta name="citation_reference" content="A. E. Gorbalenya, Severe acute respiratory syndrome-related coronavirus – The species and its viruses, a statement of the Coronavirus Study Group. bioRxiv, 2020.2002.2007.937862 (2020)."/>
  <meta name="citation_reference" content="P. Zhou et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, https://doi.org/10.1038/s41586-020-2008-3 (2020)."/>
  <meta name="citation_reference" content="F. Wu et al., A new coronavirus associated with human respiratory disease in China. Nature, https://doi.org/10.1038/s41586-020-2012-7 (2020)."/>
  <meta name="citation_reference" content="J. Cui, F. Li, Z.-L. Shi, Origin and evolution of pathogenic coronaviruses. Nat. Rev. Microbiol. 17, 181–192 (2019)."/>
  <meta name="citation_reference" content="W. Li et al., Bats are natural reservoirs of SARS-like coronaviruses. Science 310, 676–679 (2005)."/>
  <meta name="citation_reference" content="C. Huang et al., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506 (2020)."/>
  <meta name="citation_reference" content="P. Liu, W. Chen, J.-P. Chen, Viral metagenomics revealed sendai virus and coronavirus infection of Malayan Pangolins (Manis javanica). Viruses 11, 979 (2019)."/>
  <meta name="citation_reference" content="X.-Y. Ge et al., Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538 (2013)."/>
  <meta name="citation_reference" content="S. K. Wong, W. Li, M. J. Moore, H. Choe, M. Farzan, A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2. J. Biol. Chem. 279, 3197–3201 (2004)."/>
  <meta name="citation_reference" content="A. M. Bolger, M. Lohse, B. Usadel, Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)."/>
  <meta name="citation_reference" content="B. Langmead, S. L. Salzberg, Fast gapped-read alignment with Bowtie 2. Nat. Meth. 9, 357–359 (2012)."/>
  <meta name="citation_reference" content="D. Li, C.-M. Liu, R. Luo, K. Sadakane, T.-W. Lam, MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics 31, 1674–1676 (2015)."/>
  <meta name="citation_reference" content="H. Li et al., The sequence alignment/map format and SAMtools. Bioinformatics 25, 2078–2079 (2009)."/>
  <meta name="citation_reference" content="R. C. Edgar, MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32, 1792–1797 (2004)."/>
  <meta name="citation_reference" content="G. Talavera, J. Castresana, Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst. Biol. 56, 564–577 (2007)."/>
  <meta name="citation_reference" content="S. Kumar, G. Stecher, M. Li, C. Knyaz, K. Tamura, MEGA X: molecular evolutionary genetics analysis across computing platforms. Mol. Biol. Evol. 35, 1547–1549 (2018)."/>
  <meta name="twitter:title" content="Pangolin homology associated with 2019-nCoV"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV."/>
  <meta name="og-title" property="og:title" content="Pangolin homology associated with 2019-nCoV"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.950253v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-20"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Pangolin homology associated with 2019-nCoV | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1160267 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.</p>
   </div>
   <p id="p-4">In the late December of 2019, an epidemic pneumonia (the W.H.O. announced it recently as Corona Virus Disease (COVID-19)(
    <em>
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
    </em>)) outbreak in city of Wuhan in China and soon widely spreads all over the world. According to authoritative statistics, the COVID-19 has caused more than 40,000 laboratory-confirmed infections with more than 1000 deaths by 12 February 2020 and is still increasing. It was caused by a novel identified coronavirus 2019-nCoV (the International Committee on Taxonomy of Viruses (ICTV) renamed this virus as severe acute respiratory syndrome coronavirus 2, SARS-CoV-2 (
    <em>
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
    </em>)). Released complete genomes of 2019-nCoVs (
    <em>
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, 
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
    </em>) have helped rapid identification and diagnosis of the COVID-19. Another key task is to find potential origin of 2019-nCoV(
    <em>
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
    </em>). Unsurprisingly, like SARS-CoV and MERS-CoV(
    <em>
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
    </em>), the bat is still a probable origin of the 2019-nCoV because the 2019-nCoV shared 96% whole genome identity with a bat coronavirus Bat-CoV-RaTG13 from 
    <em>Rhinolophus affinis</em> from Yunnan Province(
    <em>
     <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
    </em>). However, SARS-CoV and MERS-CoV usually pass into medium host like civets or camels before leaping to humans(
    <em>
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
    </em>). It indicates that the 2019n-Cov was probably transmitted to humans by some other animals. Considering the earliest COVID-19 patient reported no exposure at the seafood market(
    <em>
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
    </em>), finding intermediate host of 2019-nCoV is vital to block its transmission.
   </p>
   <p id="p-5">On 24 October 2019, Liu and his colleagues from the Guangdong Wildlife Rescue Center of China (
    <em>
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
    </em>) firstly detected the existence of SARS-liked coronavirus from lung samples of two dead Malayan Pangolins with a frothy liquid in lung and pulmonary fibrosis, which is close on the outbreak of COVID-19. From their published results, all virus contigs assembled from 2 lung samples (lung07, lung08) showed not high identities ranging from 80.24% to 88.93% with known SARS coronavirus. Hence, we conjectured that dead Malayan pangolin may carry a new coronavirus close to 2019-nCoV.
   </p>
   <p id="p-6">To confirm our assumption, we downloaded raw RNA-seq data (SRA accession number PRJNA573298) of those two lung samples from SRA and conducted consistent quality control and contamination removing as described by Liu’s study(
    <em>
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
    </em>). We found 1882 clean reads from lung08 sample mapped upon 2019-nCoV reference genome (GenBank Accession 
    <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom">MN908947</a>)(
    <em>
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
    </em>) with high genome coverage of 76.02%. We performed 
    <em>de novo</em> assembly of those reads and totally obtained 36 contigs with length ranging from 287bp to 2187bp with mean length of 700bp. Blasting against proteins from 2845 coronavirus reference genomes including RaTG13, 2019-nCoVs and other known coronaviruses, we found 22 contigs can be best matched to 2019-nCoVs (70.6%-100% aa identity; average: 95.41%) and 12 contigs matched to Bat SARS-like coronavirus (92.7%-100% aa identity; average: 97.48%) (
    <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a>). These results indicate Malayan pangolin indeed carries a novel coronavirus (here named as Pangolin-CoV) close to 2019-nCoV.
   </p>
   <div id="T1" class="table pos-float">
    <div class="table-inline table-callout-links">
     <div class="callout">
      <span>View this table:</span>
      <ul class="callout-links">
       <li class="view-inline first">
        <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1163306/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
       </li>
       <li class="view-popup">
        <a href="/highwire/markup/1163306/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
       </li>
       <li class="download-ppt last">
        <a href="/highwire/powerpoint/1163306" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
       </li>
      </ul>
     </div>
    </div>
    <div class="table-caption">
     <span class="table-label">Table S1.</span>
     <span class="caption-title">Contigs taxonomically annotated by using BLASTx against 2845 Coronavirus reference genomes.</span>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-7">Using reference-guided scaffolding approach, we created Pangolin-CoV draft genome (19,587 bp) based on the above 34 contigs. Remapping 1882 reads against the draft genome resulted in 99.99% genome coverage at a mean depth of 7.71 X (range: 1X-47X) (
    <strong>
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>
    </strong>). Based on Simplot analysis, Pangolin-Cov showed highly overall genome sequence identity throughout the genomes to RaTG13 (90.55%) and 2019-nCoV (91.02%) (
    <strong>
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>
    </strong>), although there is greatly high identity 96.2% between 2019-nCoV and RaTG13(
    <em>
     <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>
    </em>). Another SARS-like coronavirus more similar to Pangolin-Cov were Bat SARSr-CoV ZXC21(85.65%) and Bat SARSr-CoV ZC45 (85.01%). These results indicate Pangolin-Cov may be the common origin of 2019-nCoV and RaTG13.
   </p>
   <div id="F1" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genome-related analysis. (A) Sequence depth of mapped reads remapping to Pangolin-Cov. (B) Similarity plot based on the full-length genome sequence of Pangolin-Cov. Full-length genome sequences of 2019-nCoV (Beta-CoV/Wuhan-Hu-1), BatCov-RaTG13, Bat SARSr-CoV 21, Bat SARSr-CoV45, Bat SARSr-CoV WIV1, and SARS-CoV BJ01 were used as reference sequences. (C) Comparison of common genome organization similarity among 2019-nCoV, Pangolin-Cov and BatCov-RaTG13 linked to Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genome-related analysis.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Sequence depth of mapped reads remapping to Pangolin-Cov. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Similarity plot based on the full-length genome sequence of Pangolin-Cov. Full-length genome sequences of 2019-nCoV (Beta-CoV/Wuhan-Hu-1), BatCov-RaTG13, Bat SARSr-CoV 21, Bat SARSr-CoV45, Bat SARSr-CoV WIV1, and SARS-CoV BJ01 were used as reference sequences. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Comparison of common genome organization similarity among 2019-nCoV, Pangolin-Cov and BatCov-RaTG13 linked to &lt;;strong&gt;;Table S2&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F1.medium.gif" width="419" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F1.medium.gif" width="419" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 1.</span>
     <span class="caption-title">Genome-related analysis.</span>
     <p id="p-8" class="first-child">(
      <strong>A</strong>) Sequence depth of mapped reads remapping to Pangolin-Cov. (
      <strong>B</strong>) Similarity plot based on the full-length genome sequence of Pangolin-Cov. Full-length genome sequences of 2019-nCoV (Beta-CoV/Wuhan-Hu-1), BatCov-RaTG13, Bat SARSr-CoV 21, Bat SARSr-CoV45, Bat SARSr-CoV WIV1, and SARS-CoV BJ01 were used as reference sequences. (
      <strong>C</strong>) Comparison of common genome organization similarity among 2019-nCoV, Pangolin-Cov and BatCov-RaTG13 linked to 
      <strong>
       <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S2</a>
      </strong>.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="T2" class="table pos-float">
    <div class="table-inline table-callout-links">
     <div class="callout">
      <span>View this table:</span>
      <ul class="callout-links">
       <li class="view-inline first">
        <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1163307/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
       </li>
       <li class="view-popup">
        <a href="/highwire/markup/1163307/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
       </li>
       <li class="download-ppt last">
        <a href="/highwire/powerpoint/1163307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
       </li>
      </ul>
     </div>
    </div>
    <div class="table-caption">
     <span class="table-label">Table S2.</span>
     <span class="caption-title">Comparing nt and aa sequence identity difference of ten genes among Pangolin-Cov, 2019-nCoV, and BatCov-RaTG13</span>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-9">The viral genome organization of Pangolin-Cov was characterized by sequence alignment against 2019-nCoV (GenBank Accession 
    <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom">MN908947</a>) and RaTG13. The Pangolin-Cov genome consists of six major open reading frames (ORFs) common to coronaviruses and other four accessory genes (
    <strong>
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>
    </strong> and 
    <strong>
     <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S2</a>
    </strong>). Further analysis indicates that Pangolin-Cov genes covered 2019-nCoV genes with coverage ranging from 45.8% to 100% (average coverage 76.9%). Pangolin-Cov genes shared high average nucleotide and amino identity with both 2019-nCoV (MN908947) (93.2% nt/94.1% aa) and RaTG13 (92.8% nt/93.5% aa) (
    <strong>
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a> and 
     <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table S2</a>
    </strong>). Surprisingly, some of Pangolin-Cov genes showed higher aa sequence identity to 2019-nCoV than RaTG13, including orf1b (73.4/72.8), S-protein (97.5/95.4), orf7a (96.9/93.6), and orf10 (97.3/94.6). High S-protein amino acid identity implies function similarity between Pangolin-Cov and 2019-nCoV.
   </p>
   <p id="p-10">To determine the evolutionary relationships among Pangolin-Cov, 2019-nCoV and previously identified coronaviruses, we estimated phylogenetic trees based on the nucleotide sequences of the whole genome sequence, RNA-dependent RNA polymerase gene (RdRp), non-structural protein genes ORF1a and 1b, and the main structural proteins encoded by the S and M genes. In all phylogenies, Pangolin-CoV, RaTG13 and 2019-nCoV were clustered into a well-supported group, here named as “SARS-CoV-2 group” (
    <strong>
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>
    </strong> and 
    <strong>
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figures S1</a> to 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">S2</a>
    </strong>). This group represents a novel Beta-coronaviruses group. Within this group, RaTG13 and 2019-nCoV was grouped together, and the Pangolin-CoV was their lowest common ancestor. However, whether the basal position of the SARS-CoV-2 group is SARSr-CoV ZXC21 and/or SARSr-CoV ZC45 or not is still in debate. Such debate also occurred in both Wu et al.(
    <em>
     <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>
    </em>) and Zhou et al. (
    <em>
     <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>
    </em>)studies. Possible explanation is due to a past history of recombination in Beta-CoV group(
    <em>
     <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a>
    </em>). It is noteworthy that our discovered evolutionary relationships of coronaviruses shown by the whole genome, RdRp gene, and S-gene were highly consistent with that discovered by complete genome information in Zhou et al. study(
    <em>
     <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>
    </em>). It indicates our Pangolin-CoV draft genome has enough genomic information to trace true evolutionary position of Pangolin-CoV in coronaviruses.
   </p>
   <div id="F5" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic relationship of coronavirus based on ORF1a gene (A) and ORF1b gene (B) nucleotide sequences. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Phylogenetic relationship of coronavirus based on ORF1a gene (A) and ORF1b gene (B) nucleotide sequences.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F5.medium.gif" width="440" height="346"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F5.medium.gif" width="440" height="346"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure S1.</span>
     <span class="caption-title">Phylogenetic relationship of coronavirus based on ORF1a gene (A) and ORF1b gene (B) nucleotide sequences.</span>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="F6" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic relationship of coronavirus based on S gene (A) and M gene (B) nucleotide sequences. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Phylogenetic relationship of coronavirus based on S gene (A) and M gene (B) nucleotide sequences.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F6.medium.gif" width="440" height="344"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F6.medium.gif" width="440" height="344"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure S2.</span>
     <span class="caption-title">Phylogenetic relationship of coronavirus based on S gene (A) and M gene (B) nucleotide sequences.</span>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="F2" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic relationship of coronavirus based on whole genome and RdRp gene nucleotide sequences. Red text denotes the Malayan Pangolin-CoV. Pink text denotes 2019-nCoV (SARS-CoV-2). Green text denotes a bat coronavirus having 96% similarity at genome level to SARS-CoV-2. Blue text denotes reference coronaviruses used in Figure1B. Detailed information can be found in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Phylogenetic relationship of coronavirus based on whole genome and RdRp gene nucleotide sequences.&lt;;/span&gt;; Red text denotes the Malayan Pangolin-CoV. Pink text denotes 2019-nCoV (SARS-CoV-2). Green text denotes a bat coronavirus having 96% similarity at genome level to SARS-CoV-2. Blue text denotes reference coronaviruses used in Figure1B. Detailed information can be found in &lt;;strong&gt;;Materials and Methods&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F2.medium.gif" width="440" height="348"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F2.medium.gif" width="440" height="348"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 2.</span>
     <span class="caption-title">Phylogenetic relationship of coronavirus based on whole genome and RdRp gene nucleotide sequences.</span>
     <p id="p-11" class="first-child">Red text denotes the Malayan Pangolin-CoV. Pink text denotes 2019-nCoV (SARS-CoV-2). Green text denotes a bat coronavirus having 96% similarity at genome level to SARS-CoV-2. Blue text denotes reference coronaviruses used in 
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure1B</a>. Detailed information can be found in 
      <strong>Materials and Methods</strong>.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-12">The coronavirus spike (S) protein consisting of 2 subunits (S1 and S2) mediates infection of receptor-expressing host cells and is a critical target for antiviral neutralizing antibodies. S1 contains a receptor binding domain (RBD) about 193 amino acid fragment, which is responsible for recognizing and binding with the cell surface receptor(
    <em>
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, 
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
    </em>). Zhou et al. had experimentally confirmed that 2019-nCoV is able to use human, Chinese horseshoe bats, civet, and pig ACE2 as an entry receptor in the ACE2-expressing cells(
    <em>
     <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a>
    </em>), suggesting the RBD of 2019-nCoV mediates infection to human and other animals. To gain a sequence-level insight into understanding pathogen potential of Pangolin-Cov, we investigated the amino acid variation pattern of S1 protein from Pangolin-CoV, 2019-nCoV, RaTG13, and other representative SARS-CoVs. The amino acid phylogenetic tree showed the S1 protein of Pangolin-CoV is more closely related to that of 2019-CoV than RaTG13. Within the RBD, we further found Pangolin-CoV and 2019-nCOV was highly conserved with only one amino acid change (500H/500Q) (
    <strong>
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>
    </strong>) but not belonging to five key residues involved in the interaction with human ACE2(
    <em>
     <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a>, 
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>
    </em>). In contrast, RaTG13 has 17 amino acid residue changes and 4 of them belonged to key amino acid residue (
    <strong>
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>
    </strong>). These results indicate Pangolin-Cov has similar pathogen potential to 2019-nCoV.
   </p>
   <div id="F3" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Amino acid sequence alignment of the S1 protein and its phylogeny. The receptor-binding motif of SARS-CoV and the homologous region of other coronaviruses are indicated by the grey box. The key amino acid residues involved in the interaction with human ACE2 are marked with the orange box. Bat SARS-like CoVs had been reported not to use ACE2, had amino acid deletions at two motifs marked by the yellow box. Detailed information can be found in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Amino acid sequence alignment of the S1 protein and its phylogeny.&lt;;/span&gt;; The receptor-binding motif of SARS-CoV and the homologous region of other coronaviruses are indicated by the grey box. The key amino acid residues involved in the interaction with human ACE2 are marked with the orange box. Bat SARS-like CoVs had been reported not to use ACE2, had amino acid deletions at two motifs marked by the yellow box. Detailed information can be found in &lt;;strong&gt;;Materials and Methods&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F3.medium.gif" width="440" height="312"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F3.medium.gif" width="440" height="312"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 3.</span>
     <span class="caption-title">Amino acid sequence alignment of the S1 protein and its phylogeny.</span>
     <p id="p-13" class="first-child">The receptor-binding motif of SARS-CoV and the homologous region of other coronaviruses are indicated by the grey box. The key amino acid residues involved in the interaction with human ACE2 are marked with the orange box. Bat SARS-like CoVs had been reported not to use ACE2, had amino acid deletions at two motifs marked by the yellow box. Detailed information can be found in 
      <strong>Materials and Methods</strong>.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-14">The nucleocapsid protein (N-protein) is the most abundant protein in coronavirus. The N-protein is a highly immunogenic phosphoprotein, and it is normally very conserved. The N protein of coronavirus is often used as a marker in diagnostic assays. To gain a further insight into the diagnostic potential for Pangolin-Cov, we investigated the amino acid variation pattern of N-protein from Pangolin-CoV, 2019-nCoV, RaTG13, and other representative SARS-CoV. Phylogenetic analysis based on N protein supports Pangolin-Cov as a sister taxon of 2019-nCoV and RaTG13 (
    <strong>
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>
    </strong>). We further found seven amino acid mutations can differentiate our defined “SAR-CoV-2 group” (12N, 26 G, 27S, 104D, 218A, 335T, 346N, 350Q) from other known SARS-CoVs (12S, 26D, 27N, 104E, 218T, 335H, 346Q, 350N). Two amino acid sites (38P and 268Q) are shared by Pangolin-Cov, RaTG13 and SARS-CoVs, which are mutated as 38S and 268A in 2019-nCoV. Only one amino acid residue shared by Pangolin-CoV and other SARS-CoVs (129E) is consistently changed in both 2019-nCoV and RaTG13 (129D). Our observed amino acid changes in N-protein would be useful for developing antigen for much more sensitive serological detection of 2019-nCoV.
   </p>
   <div id="F4" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Amino acid sequence alignment of N protein and its phylogeny. Highly-conserved amino acid residues in N-protein marked by colors have diagnostic potential. Detailed information can be found in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014843270" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Amino acid sequence alignment of N protein and its phylogeny.&lt;;/span&gt;; Highly-conserved amino acid residues in N-protein marked by colors have diagnostic potential. Detailed information can be found in &lt;;strong&gt;;Materials and Methods.&lt;;/strong&gt;;&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F4.medium.gif" width="440" height="315"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F4.medium.gif" width="440" height="315"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.950253/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 4.</span>
     <span class="caption-title">Amino acid sequence alignment of N protein and its phylogeny.</span>
     <p id="p-15" class="first-child">Highly-conserved amino acid residues in N-protein marked by colors have diagnostic potential. Detailed information can be found in 
      <strong>Materials and Methods.</strong>
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-16">Based on published metagenomic data, this study provides the first report on a potential closely related kin (Pangolin-CoV) of 2019-nCoV, which was discovered from dead Malayan Pangolins after extensive rescue efforts. Aside from RaTG13, the Pangolin-CoV is the most closely related to 2019-nCoV. Due to original sample unavailable, we did not perform further experiments to confirm our findings, including PCR validation, serological detection, and even the isolation of virus particle etc. However, on 7 February, researchers from the South China Agricultural University in Guangzhou reported pangolin would be the potential candidate host of 2019-nCoV for isolating a virus 99% similar to 2019-nCoV in genome (Data unpublished). Our discovered Pangolin-CoV genome showed 91.02% nt identity with 2019-nCoV, implying Pangolin-CoV could be different from that unpublished. Whether pangolin species is a good candidate for 2019-nCoV still need to be further investigated. Considering the wide spread of SARSr-CoV in their natural reservoirs, our findings would be meaningful to find novel intermediate hosts of 2019-nCoV for blocking interspecies transmission.</p>
   <div class="section" id="sec-1">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-2" class="subsection">
     <h3>Data preparation</h3>
     <p id="p-17">We downloaded raw data of lung08 and lung07 published by Liu’s study(
      <em>
       <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>
      </em>) from NCBI sequence read archive (SRA) under Bio Project PRJNA573298. Raw reads were first adaptor- and quality-trimmed using the Trimmomatic program (version 0.39)(
      <em>
       <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
      </em>). For removing host contamination, Bowtie2 (version 2.3.4.3) (
      <em>
       <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
      </em>)was used to map clean reads to the host reference genome of 
      <em>Manis javanica</em> (NCBI Project ID: PRJNA256023). Only unmapped reads were mapped to 2019-nCoV reference genome (GenBank Accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom">MN908947</a>) for identifying virus reads.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Read Assembly and construction of consensus sequence</h3>
     <p id="p-18">Virus-targeted reads were assembled 
      <em>de novo</em> using MEGAHIT (v1.1.3)(
      <em>
       <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
      </em>). Read remapping to assembled contigs was performed by using Bowtie2(
      <em>
       <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>
      </em>). Mapping coverage and depth were produced using Samtools (version 1.9)(
      <em>
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
      </em>). Contigs were taxonomically annotated using BLAST 2.9.0+ against 2845 Coronavirus reference genomes (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table S1</a>). Bat-Cov-RaTG13 genome was downloaded from NGDC database (
      <a href="https://bigd.big.ac.cn/">https://bigd.big.ac.cn/</a>) (Accession no. GWHABKP00000000)(
      <em>
       <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a>
      </em>). 2019-nCoV reference genome was downloaded from NCBI (Accession no. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom">MN908947</a>)(
      <em>
       <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">3</a>
      </em>). Other coronavirus genomes were downloaded ViPR database (
      <a href="https://www.viprbrc.org/brc/home.spg?decorator=corona">https://www.viprbrc.org/brc/home.spg?decorator=corona</a>) on 6 February 2020. We further used reference-guided strategy to construct draft genome based on those contigs taxonomically annotated to 2019-nCoVs, SARS-CoV, and Bat SARS-like CoV. Each contig was aligned against 2019-nCoV reference genome with MUSCLE software (version 3.8.31)(
      <em>
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </em>). Aligned contigs were merged into consensus scaffold with BioEdit version 7.2.5 (
      <a href="http://www.mybiosoftware.com/bioedit-7-0-9-biological-sequence-alignment-editor.html">http://www.mybiosoftware.com/bioedit-7-0-9-biological-sequence-alignment-editor.html</a>) following manually quality checking. Small fragments with less than length 25bp were discarded, if these fragments were not covered by any large fragments. The potential open reading frames (ORFs) of finally obtained draft genome were annotated by the alignment to 2019-nCoV reference genome (Accession no. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom">MN908947</a>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Phylogenetic relationship analysis</h3>
     <p id="p-19">Sequence alignment was carried out using MUSCLE software(
      <em>
       <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
      </em>). Alignment accuracy was checked manually base-by-base. Gblocks(
      <em>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
      </em>) was used to process the gap in aligned sequence. Using MegaX (version 10.1.7)(
      <em>
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
      </em>), we inferred all maximum likelihood (ML) phylogenetic trees under the best-fit DNA/anima acid substitute model with 1000 bootstrap replications. Phylogenetic analyses were performed using the nucleotide sequences of various CoV gene data sets: the whole genome and ORF1a, ORF1b, Membrane (M) gene, spike (S) and RNA-dependent RNA polymerase (RdRp) gene. The best model of M is GTR+G and all others are GTR+G+I. Two additional protein-based trees were constructed under WAG+G (S1 subunit of S protein) and JTT+G (N-protein), respectively. Branches with values&lt;; 70% bootstrap were hidden in all phylogenetic trees.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-20">This study was supported by the Second Tibetan Plateau Scientific Expedition and Research (STEP) program (no. 2019QZKK0503), the National Key Research and Development Program of China (no. 2018YFC2000500), the Key Research Program of the Chinese Academy of Sciences (no. KFZD-SW-219), and the Chinese National Natural Science Foundation (no. 31970571).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. E.</span>
          <span class="cit-name-surname">Gorbalenya</span>
         </span>, 
         <span class="cit-article-title">Severe acute respiratory syndrome-related coronavirus – The species and its viruses, a statement of the Coronavirus Study Group</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, 2020.2002.2007.937862 (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.19.950253v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Zhou</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <span class="cit-source">Nature</span>, 
         <a href="https://doi.org/10.1038/s41586-020-2008-3">https://doi.org/10.1038/s41586-020-2008-3</a> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.19.950253v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Wu</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <span class="cit-source">Nature</span>, 
         <a href="https://doi.org/10.1038/s41586-020-2012-7">https://doi.org/10.1038/s41586-020-2012-7</a> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.4" data-doi="10.1038/s41579-018-0118-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Cui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.-L.</span>
          <span class="cit-name-surname">Shi</span>
         </span>, 
         <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol</abbr>. 
         <span class="cit-vol">17</span>, 
         <span class="cit-fpage">181</span>–
         <span class="cit-lpage">192</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.5" data-doi="10.1126/science.1118391">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Li</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">310</span>, 
         <span class="cit-fpage">676</span>–
         <span class="cit-lpage">679</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yMC8yMDIwLjAyLjE5Ljk1MDI1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Huang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-P.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-article-title">Viral metagenomics revealed sendai virus and coronavirus infection of Malayan Pangolins (
          <em>Manis javanica</em>)
         </span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-vol">11</span>, 
         <span class="cit-fpage">979</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.8" data-doi="10.1038/nature12711">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.-Y.</span>
          <span class="cit-name-surname">Ge</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">503</span>, 
         <span class="cit-fpage">535</span>–
         <span class="cit-lpage">538</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24172901&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000327464200045&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.9" data-doi="10.1074/jbc.C300520200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. K.</span>
          <span class="cit-name-surname">Wong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. J.</span>
          <span class="cit-name-surname">Moore</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Choe</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Farzan</span>
         </span>, 
         <span class="cit-article-title">A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2</span>. 
         <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. 
         <span class="cit-vol">279</span>, 
         <span class="cit-fpage">3197</span>–
         <span class="cit-lpage">3201</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C300520200%26rft_id%253Dinfo%253Apmid%252F14670965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvNS8zMTk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4xOS45NTAyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.10" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. M.</span>
          <span class="cit-name-surname">Bolger</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Lohse</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Usadel</span>
         </span>, 
         <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">2120</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.11" data-doi="10.0.4.14/nmeth.1923">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Langmead</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. L.</span>
          <span class="cit-name-surname">Salzberg</span>
         </span>, 
         <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Meth</abbr>. 
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">357</span>–
         <span class="cit-lpage">359</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BMeth%26rft.volume%253D9%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.0.4.14%252Fnmeth.1923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.0.4.14/nmeth.1923&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.12" data-doi="10.1093/bioinformatics/btv033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.-M.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Luo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Sadakane</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.-W.</span>
          <span class="cit-name-surname">Lam</span>
         </span>, 
         <span class="cit-article-title">MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">1674</span>–
         <span class="cit-lpage">1676</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv033%26rft_id%253Dinfo%253Apmid%252F25609793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25609793&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.13" data-doi="10.1093/bioinformatics/btp352">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Li</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The sequence alignment/map format and SAMtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">2078</span>–
         <span class="cit-lpage">2079</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268808600014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.14" data-doi="10.1093/nar/gkh340">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R. C.</span>
          <span class="cit-name-surname">Edgar</span>
         </span>, 
         <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. 
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">1792</span>–
         <span class="cit-lpage">1797</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15034147&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220487200025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.15" data-doi="10.1080/10635150701472164">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Talavera</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Castresana</span>
         </span>, 
         <span class="cit-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">56</span>, 
         <span class="cit-fpage">564</span>–
         <span class="cit-lpage">577</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DTalavera%26rft.auinit1%253DG.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D564%26rft.epage%253D577%26rft.atitle%253DImprovement%2Bof%2Bphylogenies%2Bafter%2Bremoving%2Bdivergent%2Band%2Bambiguously%2Baligned%2Bblocks%2Bfrom%2Bprotein%2Bsequence%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701472164%26rft_id%253Dinfo%253Apmid%252F17654362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10635150701472164&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17654362&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000248359900002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.950253v1.16" data-doi="10.1093/molbev/msy096">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Kumar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Stecher</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Knyaz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Tamura</span>
         </span>, 
         <span class="cit-article-title">MEGA X: molecular evolutionary genetics analysis across computing platforms</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">35</span>, 
         <span class="cit-fpage">1547</span>–
         <span class="cit-lpage">1549</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29722887&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.19.950253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
