<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/27/2020.02.25.964775.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.25.964775v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.25.964775"/>
  <meta name="DC.Date" content="2020-02-27"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV\_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV\_N2, N3 (USA), and NIID\_2019-nCOV\_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2."/>
  <meta name="DC.Contributor" content="Yu Jin Jung"/>
  <meta name="DC.Contributor" content="Gun-Soo Park"/>
  <meta name="DC.Contributor" content="Jun Hye Moon"/>
  <meta name="DC.Contributor" content="Keunbon Ku"/>
  <meta name="DC.Contributor" content="Seung-Hwa Beak"/>
  <meta name="DC.Contributor" content="Seil Kim"/>
  <meta name="DC.Contributor" content="Edmond Changkyun Park"/>
  <meta name="DC.Contributor" content="Daeui Park"/>
  <meta name="DC.Contributor" content="Jong-Hwan Lee"/>
  <meta name="DC.Contributor" content="Cheol Woo Byeon"/>
  <meta name="DC.Contributor" content="Joong Jin Lee"/>
  <meta name="DC.Contributor" content="Jin-Soo Maeng"/>
  <meta name="DC.Contributor" content="Seong Jun Kim"/>
  <meta name="DC.Contributor" content="Seung Il Kim"/>
  <meta name="DC.Contributor" content="Bum-Tae Kim"/>
  <meta name="DC.Contributor" content="Min Jun Lee"/>
  <meta name="DC.Contributor" content="Hong Gi Kim"/>
  <meta name="article:published_time" content="2020-02-27"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the 2019-nCoV_N2, N3 of USA and the ORF1ab of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.25.964775v1"/>
  <meta name="citation_id" content="2020.02.25.964775v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.25.964775v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.25.964775v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.25.964775v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.25.964775"/>
  <meta name="citation_num_pages" content="13"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.25.964775"/>
  <meta name="citation_author" content="Yu Jin Jung"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author" content="Gun-Soo Park"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Research Group of Food Processing, Korea Food Research Institute"/>
  <meta name="citation_author" content="Jun Hye Moon"/>
  <meta name="citation_author_institution" content="Department of Molecular Diagnostics, WELLS BIO, INC"/>
  <meta name="citation_author" content="Keunbon Ku"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author" content="Seung-Hwa Beak"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Department of Predictive Toxicology, Korea Institute of Toxicology"/>
  <meta name="citation_author" content="Seil Kim"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Division of Chemical and Medical Metrology, Center for Bioanalysis, Korea Research Institute of Standards and Science"/>
  <meta name="citation_author" content="Edmond Changkyun Park"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute"/>
  <meta name="citation_author" content="Daeui Park"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Department of Predictive Toxicology, Korea Institute of Toxicology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9452-5849"/>
  <meta name="citation_author" content="Jong-Hwan Lee"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author" content="Cheol Woo Byeon"/>
  <meta name="citation_author_institution" content="Department of Molecular Diagnostics, WELLS BIO, INC"/>
  <meta name="citation_author" content="Joong Jin Lee"/>
  <meta name="citation_author_institution" content="Department of Molecular Diagnostics, WELLS BIO, INC"/>
  <meta name="citation_author" content="Jin-Soo Maeng"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Research Group of Food Processing, Korea Food Research Institute"/>
  <meta name="citation_author" content="Seong Jun Kim"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author" content="Seung Il Kim"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute"/>
  <meta name="citation_author" content="Bum-Tae Kim"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author" content="Min Jun Lee"/>
  <meta name="citation_author_institution" content="Department of Molecular Diagnostics, WELLS BIO, INC"/>
  <meta name="citation_author_email" content="tenork@krict.re.kr"/>
  <meta name="citation_author_email" content="mjlee@wellsbio.net"/>
  <meta name="citation_author" content="Hong Gi Kim"/>
  <meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology"/>
  <meta name="citation_author_email" content="tenork@krict.re.kr"/>
  <meta name="citation_author_email" content="mjlee@wellsbio.net"/>
  <meta name="citation_reference" content="WHO Coronavirus disease 2019 (COVID-19): Situation Report – 34. 2020. DOI: https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200223-sitrep-34-covid-19.pdf?sfvrsn=44ff8fd3_2."/>
  <meta name="citation_reference" content="Guan, W.J., et al. Clinical characteristics of 2019 novel coronavirus infection in China. 2020. DOI: https://doi.org/10.1101/2020.02.06.20020974."/>
  <meta name="citation_reference" content="Huang, C., et al., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet, 2020. 395(10223): p. 497–506."/>
  <meta name="citation_reference" content="Wu, F., et al., A new coronavirus associated with human respiratory disease in China. Nature, 2020."/>
  <meta name="citation_reference" content="Zhou, P., et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 2020."/>
  <meta name="citation_reference" content="Zhu, N., et al., A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med, 2020. 382(8): p. 727–733."/>
  <meta name="citation_reference" content="Gorbalenya, A.E., et al. Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group. 2020. DOI: https://doi.org/10.1101/2020.02.07.937862."/>
  <meta name="citation_reference" content="WHO Coronavirus disease (COVID-19) technical guidance: Laboratory testing for 2019-nCoV in humans. 2020. DOI: https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance."/>
  <meta name="citation_reference" content="Chu, D.K.W., et al., Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clin Chem, 2020."/>
  <meta name="citation_reference" content="Corman, V.M., et al., Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill, 2020. 25(3)."/>
  <meta name="citation_reference" content="WHO Coronavirus disease 2019 (COVID-19): Situation Report – 28. 2020. DOI: https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200217-sitrep-28-covid-19.pdf?sfvrsn=a19cf2ad_2."/>
  <meta name="citation_reference" content="Drosten, C., et al., Identification of a novel coronavirus in patients with severe acute respiratory syndrome. N Engl J Med, 2003. 348(20): p. 1967–76."/>
  <meta name="citation_reference" content="Poon, L.L., et al., A one step quantitative RT-PCR for detection of SARS coronavirus with an internal control for PCR inhibitors. J Clin Virol, 2004. 30(3): p. 214–7."/>
  <meta name="citation_reference" content="Corman, V.M., et al., Assays for laboratory confirmation of novel human coronavirus (hCoV-EMC) infections. Euro Surveill, 2012. 17(49)."/>
  <meta name="citation_reference" content="Corman, V.M., et al., Detection of a novel human coronavirus by real-time reverse-transcription polymerase chain reaction. Euro Surveill, 2012. 17(39)."/>
  <meta name="citation_reference" content="Xie, X., et al., Chest CT for Typical 2019-nCoV Pneumonia: Relationship to Negative RT-PCR Testing. Radiology, 2020:p. 200343."/>
  <meta name="citation_reference" content="China_CDC Specific primers and probes for detection 2019 novel coronavirus. 2020. DOI: http://ivdc.chinacdc.cn/kyjz/202001/t20200121_211337.html."/>
  <meta name="citation_reference" content="Nao, N., et al. Detection of second case of 2019-nCoV infection in Japan. 2020. DOI: https://www.who.int/docs/default-source/coronaviruse/method-niid-20200123-2.pdf?sfvrsn=fbf75320_7."/>
  <meta name="citation_reference" content="Health, T.M.o.P. Diagnostic detection of Novel coronavirus 2019 by Real time RT-PCR. 2020. DOI: https://www.who.int/docs/default-source/coronaviruse/conventional-rt-pcr-followed-by-sequencing-for-detection-of-ncov-rirl-nat-inst-health-t.pdf?sfvrsn=42271c6d_4."/>
  <meta name="citation_reference" content="US_CDC 2019-Novel Coronavirus (2019-nCoV) Real-time rRT-PCR Panel: Primers and Probes. 2020. DOI: https://www.who.int/docs/default-source/coronaviruse/uscdcrt-pcr-panel-primer-probes.pdf?sfvrsn=fa29cb4b_2."/>
  <meta name="twitter:title" content="Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV\_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV\_N2, N3 (USA), and NIID\_2019-nCOV\_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2."/>
  <meta name="og-title" property="og:title" content="Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.25.964775v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV\_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV\_N2, N3 (USA), and NIID\_2019-nCOV\_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-27"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1171523 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Firstly informed to World Health Organization (WHO) on 31 December 2019, the current outbreak of Coronavirus Disease (COVID-19) involves 78,811 confirmed cases over 28 countries as of 23 February 2020 [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. The majority of COVID-19 patients had pneumonia and showed symptoms include fever and cough [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, 
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The genome sequence of causative novel coronavirus was shared through Global Initiative on Sharing All Influenza Data (GISAID) platform from 12 January 2020. The sequences of novel coronavirus (CoV) showed close similarity to that of severe acute respiratory syndrome-related coronaviruses (SARSr-CoV) and the virus uses ACE2 as the entry receptor like SARS-CoV [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. The Coronavirus Study Group of the International Committee on Taxonomy of Viruses designated the virus as SARS-CoV-2 [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>].
    </p>
    <p id="p-5">Molecular diagnosis of COVID-19 is currently carried out by one-step quantitative RT-PCR (qRT-PCR) targetting SARS-CoV-2 by which primers and probes being suggested by institutes of China, Germany, Hong Kong, Japan, Thailand, and USA were posted through WHO [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>-
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Clinical diagnosis methods including CT scan are also utilized to identify COVID-19 cases in Hubei province, China, from 13 February 2020 [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Although qRT-PCR assay served as a gold-standard method to detect respiratory infectious viruses such as SARS-CoV and MERS-CoV [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>-
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], current qRT-PCR assays targetting SARS-CoV-2 have some caveats. First, due to the high similarity of SARS-CoV-2 to SARS-CoV, primer-probe sets would cross-react. Second, the sensitivity of the assays may not enough to confirm suspicious patients in early time points after admission. Indeed, cases of positive CT scan results and negative RT-PCR results at initial presentation were reported [
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. The performance of molecular diagnosis might be dependent on primers, probes, and reagents. There have been no comparative results of the current qRT-PCR analysis for the molecular diagnosis of SARS-CoV-2.
    </p>
    <p id="p-6">In this present study, the qRT-PCR analysis was performed with previously reported primer-probe sets targeting RdRp/Orf1 and N region of SARS-CoV-2. This is the first comparative analysis of various primer-probe sets for the laboratory confirmation of SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Materials and methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Primer Information of qPCR</h3>
     <p id="p-7">For the comparative analysis of laboratory confirmation for SARS-CoV-2, ten primer-probe sets were selected based on sequence information from the six different national institutions; the Centers for Disease Control and Prevention (CDC) (USA), Charité – Universitätsmedizin Berlin Institute of Virology (Germany), The University of Hong Kong (Hong Kong), National Institute of Infectious Disease, Department of virology III (Japan), China CDC (China), and National Institute of Health (Thailand). All of the DNA oligonucleotides were synthesized from Neoprobe (Daejeon, South Korea). The sequences of primer-probe sets and their locations at viral RNA (GenBank 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.25.964775.atom">MN908947.3</a>) were listed in 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Seven of the ten sets were derived from the N gene, and the other three sets were derived from Orf1 gene (RdRp, ORF 1b-Nsp14, and ORF 1-Nsp10). All DNA oligonucleotides were resuspended in nuclease-free water before use.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1174103/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1174103/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1174103" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Information of primers and probes analyzed in the study</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Relative positions of qRT-PCR primer-probe set on the SARS-CoV-2. The target genes and sequences of primers were searched from WHO website (http://www.who.int). The number below amplicons are genome positions according to SARS-CoV-2, GenBank MN908947.3. The sets were published by China CDC (Orf1ab and N), Charit[[#xE9]] [[#x2013]] universit[[#xE4]]tsmedizin berlin institute of virology in Germany (RdRp_SARSr and E), the University of Hong Kong (HKU-ORF1b_nsp14 and HKU-N), USA CDC (2019-nCoV_N1, N2, and N3), National Institute of Health in Thailand (WH-NIC N), and National Institute of Infectious Disease in Japan (NIID_2019-nCoV_N). Orf1: open reading frame 1; RdRp: RNA-dependent RNA polymerase gene; Nsp14: non-structural protein 14 gene; S: spike protein gene; E: envelop protein gene, N: nucleocapsid protein gene" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1526180750" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Relative positions of qRT-PCR primer-probe set on the SARS-CoV-2. The target genes and sequences of primers were searched from WHO website (http://www.who.int). The number below amplicons are genome positions according to SARS-CoV-2, GenBank MN908947.3. The sets were published by China CDC (Orf1ab and N), Charit[[#xE9]] [[#x2013]] universit[[#xE4]]tsmedizin berlin institute of virology in Germany (RdRp_SARSr and E), the University of Hong Kong (HKU-ORF1b_nsp14 and HKU-N), USA CDC (2019-nCoV_N1, N2, and N3), National Institute of Health in Thailand (WH-NIC N), and National Institute of Infectious Disease in Japan (NIID_2019-nCoV_N). Orf1: open reading frame 1; RdRp: RNA-dependent RNA polymerase gene; Nsp14: non-structural protein 14 gene; S: spike protein gene; E: envelop protein gene, N: nucleocapsid protein gene&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775/F1.medium.gif" width="440" height="142"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775/F1.medium.gif" width="440" height="142"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.25.964775/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-8" class="first-child">Relative positions of qRT-PCR primer-probe set on the SARS-CoV-2. The target genes and sequences of primers were searched from WHO website (
        <a href="http://www.who.int">http://www.who.int</a>). The number below amplicons are genome positions according to SARS-CoV-2, GenBank 
        <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.25.964775.atom">MN908947.3</a>. The sets were published by China CDC (Orf1ab and N), Charité – universitätsmedizin berlin institute of virology in Germany (RdRp_SARSr and E), the University of Hong Kong (HKU-ORF1b_nsp14 and HKU-N), USA CDC (2019-nCoV_N1, N2, and N3), National Institute of Health in Thailand (WH-NIC N), and National Institute of Infectious Disease in Japan (NIID_2019-nCoV_N). Orf1: open reading frame 1; RdRp: RNA-dependent RNA polymerase gene; Nsp14: non-structural protein 14 gene; S: spike protein gene; E: envelop protein gene, N: nucleocapsid protein gene
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Viral RNA preparation</h3>
     <p id="p-9">The infection experiments were performed in a biosafety level-3 (BSL-3) laboratory. African green monkey kidney Vero cells (ATCC CCL-81) were infected with a clinical isolate SARS-CoV-2 (BetaCoV/Korea/KCDC03/2020 provided from Korea CDC). After 72 h, the culture medium containing mature infectious virions (virus medium) was collected and viral RNA was isolated from the culture medium using the QIAamp viral RNA extraction Kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Preparation of in vitro transcribed RNA standard</h3>
     <p id="p-10">The coding sequence of SARS-CoV-2 Envelope (E) protein, which cloned in pET21a plasmid was PCR amplified with T7 promoter primer (5’ – AATACGACTCACTATAG – 3’, Macrogen Inc., South Korea) and T7 terminator primer (5’ – GCTAGTTATTGCTCAGCGG – 3’, Macrogen) with AccuPower® PCR PreMix (-dye) kit (Bioneer Inc., South Korea). PCR product was then used as 
      <em>in vitro</em> transcription template using MEGAscript™ T7 Transcription Kit (Invitrogen Inc., CA, USA). The copy number of 
      <em>in vitro</em> transcribed RNA was calculated from RNA concentration measured with Quantus™ Fluorometer (Promega Inc., WI, USA). Standardized amounts of 
      <em>in vitro</em> produced RNA were used E primer and qRT-PCR to produce a standard curve.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Confirmatory qRT-PCR in RdRp and N</h3>
     <p id="p-11">Extracted nucleic acid samples were tested for comparative analysis of SARS-CoV-2 by qRT-PCR. The Orf1 and N region of SARS-CoV-2 were used as the target sequences for SARS-CoV-2 specific gene. Briefly, 10 μL of purified viral RNA was amplified in a 20 μL reaction solution containing 1X 1 step RT-PCR mix (WELLS BIO INC., South Korea), and 300 nM of primers and probes for the target detection. The qRT-PCR was performed with a CFX 96 touch real-time PCR detection system (Bio-rad, Hercules, CA, USA). The qRT-PCR conditions applied in this study were programmed as follows: UNG incubation, RT incubation, and enzyme activation were serially performed at 25 °C for 2 minutes, at 55 °C for 10 minutes, at 94 °C for 3 minutes respectively. Thermal cycling was then performed at 94 °C for 15 seconds (denaturation), and at 60 °C for 30 seconds (annealing and amplification) for forty-five cycles.</p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Results and Discussion</h2>
    <div id="sec-8" class="subsection">
     <h3>Validation of qRT-PCR assay</h3>
     <p id="p-12">The Ct value was not produced from negative control, indicating the reaction was done aseptically. The standard curve from E gene primer-probe set also showed the reaction was done accordingly. The R
      <sup>2</sup> value of the standard curve was 0.999 and the calculated amplification efficiency was 101.6%. These indicated that the qRT-PCR reaction was done in optimal condition. The viral concentration of supernatant and cell lysate was determined by E gene-based assay (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1174101/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1174101/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1174101" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <span class="caption-title">Comparative analysis of Ct values obtained by employing each primer-probe set</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>RdRp/Orf1 Assays</h3>
     <p id="p-13">The Ct value of RdRp_SARSr (Germany), HKU-ORF1b-nsp14 (Hong Kong), and ORF1ab (China) from low concentration (15 copies/reaction) were 43.00, 38.97, and 36.85, respectively (
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The assay with RdRp_SARSr (Germany) set showed a positive signal from the single reaction of triplicate in the concentration of 15 copies/reaction. The assay with HKU-ORF1b-nsp14 (Hong Kong), and ORF1ab (China) sets showed positive signals in the concentration of 1.5 copies/reaction (data not shown). The R
      <sup>2</sup> value from RdRp_SARSr (Germany), HKU-ORF1b-nsp14 (Hong Kong), and ORF1ab (China) were 0.983, 0.997 and 0.997, respectively. The calculated amplification efficiency of RdRp_SARSr (Germany), HKU-ORF1b-nsp14 (Hong Kong), and ORF1ab (China) was 101.6%, 96.1%, and 109.8%, respectively. As a result, ORF1ab (China) set may be recommended for the laboratory confirmation of the RdRp/Orf1 gene.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>N Assays</h3>
     <p id="p-14">The Ct value of N (China), HKU-N (Hong Kong), NIID_2019-nCOV_N (Japan), WH-NIC N (Thailand), 2019-nCoV_N1, N2, and N3 (USA) from low concentration (15 copies/reaction) were 34.86, 35.43, 33.13, 38.13, 34.71, 33.14, and 33.09, respectively (
      <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The Ct value of 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets were similar to each other, and the sets could be regarded as the most sensitive sets. The moderately sensitive assay was based on 2019-nCoV_N1 (USA) and N (China). These sets had higher Ct value than the most sensitive sets, however, the Ct values from low concentration (15 copies/μl) were still within the cut-off value (Ct &lt;;37). WH-NIC N (Thailand) set was less sensitive than other sets. The Ct value from low concentration (15 copies/μl) was close to the cut-off value (Ct &lt;;38). The R
      <sup>2</sup> value from of N (China), HKU-N (Hong Kong), NIID_2019-nCOV_N (Japan), WH-NIC N (Thailand), 2019-nCoV_N1, N2, and N3 (USA) were 0.989, 0.980, 0.987, 0.987, 0.986, 0.952, and 0.991, respectively. The calculated amplification efficiency of N (China), HKU-N (Hong Kong), NIID_2019-nCOV_N (Japan), WH-NIC N (Thailand), 2019-nCoV_N1, N2, and N3 (USA) were 89.4, 105.3, 100.7, 106.2, 95.2, 97.3, and 93.9, respectively. Therefore, 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be beneficial for the laboratory confirmation of SARS-CoV-2 by qRT-PCR assay of N gene.
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Conclusions</h2>
    <p id="p-15">Various primer-probe sets were previously reported to detect SARS-CoV-2 by the qRT-PCR assay. The sensitivity of the assay may not enough to confirm suspicious patients in the early stage of SARS-CoV-2 infection. Nevertheless, there have been no comparative results of the current qRT-PCR analysis for the molecular diagnosis of SARS-CoV-2. In the present study, the first comparative analysis of various primer-probe sets targeting RdRp/Orf1 and N region of SARS-CoV-2 was performed by qRT-PCR for the laboratory confirmation. In the case of targeting RdRp/Orf1 region, ORF1ab (China) set might be the most sensitive than other sets. 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets may be recommended for the sensitive qRT-PCR assay of N region. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-16">We appreciated to National Culture Collection for Pathogen of Korea CDC for providing clinical SARS-CoV-2 isolate. This work was supported by National Research Council of Science and Technology grant by the Ministry of Science and ICT (Grant No. CRC-16-01-KRICT).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.25.964775v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>
         <span class="cit-source">Coronavirus disease 2019 (COVID-19): Situation Report – 34</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200223-sitrep-34-covid-19.pdf?sfvrsn=44ff8fd3_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200223-sitrep-34-covid-19.pdf?sfvrsn=44ff8fd3_2</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.25.964775v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>, 
          <span class="cit-name-given-names">W.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">Clinical characteristics of 2019 novel coronavirus infection in China</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://doi.org/10.1101/2020.02.06.20020974">https://doi.org/10.1101/2020.02.06.20020974</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">395</span>(
         <span class="cit-issue">10223</span>): p. 
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.25.964775v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China</span>, 
         <span class="cit-pub-date">2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, 2020. 
         <span class="cit-vol">382</span>(
         <span class="cit-issue">8</span>): p. 
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.25.964775v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gorbalenya</span>, 
          <span class="cit-name-given-names">A.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://doi.org/10.1101/2020.02.07.937862">https://doi.org/10.1101/2020.02.07.937862</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.25.964775v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>
         <span class="cit-source">Coronavirus disease (COVID-19) technical guidance: Laboratory testing for 2019-nCoV in humans</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.25.964775v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">D.K.W.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Clin Chem</abbr>, 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">3</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.25.964775v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>
         <span class="cit-source">Coronavirus disease 2019 (COVID-19): Situation Report – 28</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200217-sitrep-28-covid-19.pdf?sfvrsn=a19cf2ad_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200217-sitrep-28-covid-19.pdf?sfvrsn=a19cf2ad_2</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.12" data-doi="10.1056/NEJMoa030747">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-vol">348</span>(
         <span class="cit-issue">20</span>): p. 
         <span class="cit-fpage">1967</span>–
         <span class="cit-lpage">76</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12690091&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.25.964775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000182823400005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.25.964775v1.13" data-doi="10.1016/j.jcv.2003.12.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A one step quantitative RT-PCR for detection of SARS coronavirus with an internal control for PCR inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Virol</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">30</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">214</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.stitle%253DJ%2BClin%2BVirol%26rft.aulast%253DPoon%26rft.auinit1%253DL.%2BL.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D214%26rft.epage%253D217%26rft.atitle%253DA%2Bone%2Bstep%2Bquantitative%2BRT-PCR%2Bfor%2Bdetection%2Bof%2BSARS%2Bcoronavirus%2Bwith%2Ban%2Binternal%2Bcontrol%2Bfor%2BPCR%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2003.12.007%26rft_id%253Dinfo%253Apmid%252F15135737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jcv.2003.12.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15135737&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.25.964775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221845900003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.25.964775v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Assays for laboratory confirmation of novel human coronavirus (hCoV-EMC) infections</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">49</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Detection of a novel human coronavirus by real-time reverse-transcription polymerase chain reaction</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">39</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.25.964775v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Chest CT for Typical 2019-nCoV Pneumonia: Relationship to Negative RT-PCR Testing</span>. 
         <abbr class="cit-jnl-abbrev">Radiology</abbr>, 
         <span class="cit-pub-date">2020</span>:p. 
         <span class="cit-fpage">200343</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.02.25.964775v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">China_CDC</span>
         <span class="cit-source">Specific primers and probes for detection 2019 novel coronavirus</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="http://ivdc.chinacdc.cn/kyjz/202001/t20200121_211337.html">http://ivdc.chinacdc.cn/kyjz/202001/t20200121_211337.html</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.02.25.964775v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nao</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">Detection of second case of 2019-nCoV infection in Japan</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/docs/default-source/coronaviruse/method-niid-20200123-2.pdf?sfvrsn=fbf75320_7">https://www.who.int/docs/default-source/coronaviruse/method-niid-20200123-2.pdf?sfvrsn=fbf75320_7</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.02.25.964775v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Health</span>, 
          <span class="cit-name-given-names">T.M.o.P.</span>
         </span>
         <span class="cit-source">Diagnostic detection of Novel coronavirus 2019 by Real time RT-PCR</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/docs/default-source/coronaviruse/conventional-rt-pcr-followed-by-sequencing-for-detection-of-ncov-rirl-nat-inst-health-t.pdf?sfvrsn=42271c6d_4">https://www.who.int/docs/default-source/coronaviruse/conventional-rt-pcr-followed-by-sequencing-for-detection-of-ncov-rirl-nat-inst-health-t.pdf?sfvrsn=42271c6d_4</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.02.25.964775v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">US_CDC</span>
         <span class="cit-source">2019-Novel Coronavirus (2019-nCoV) Real-time rRT-PCR Panel: Primers and Probes</span>. 
         <span class="cit-pub-date">2020</span>. DOI: 
         <a href="https://www.who.int/docs/default-source/coronaviruse/uscdcrt-pcr-panel-primer-probes.pdf?sfvrsn=fa29cb4b_2">https://www.who.int/docs/default-source/coronaviruse/uscdcrt-pcr-panel-primer-probes.pdf?sfvrsn=fa29cb4b_2</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
