<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.04.07.030445.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.07.030445v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development" />
<meta name="DC.Identifier" content="10.1101/2020.04.07.030445" />
<meta name="DC.Date" content="2020-04-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Here we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells. We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses.

These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity. Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.

### Competing Interest Statement" />
<meta name="DC.Contributor" content="Oliver C. Grant" />
<meta name="DC.Contributor" content="David Montgomery" />
<meta name="DC.Contributor" content="Keigo Ito" />
<meta name="DC.Contributor" content="Robert J. Woods" />
<meta name="article:published_time" content="2020-04-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Here we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells.  We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses. These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity.  Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.07.030445v1" />
<meta name="citation_id" content="2020.04.07.030445v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.07.030445" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.07.030445" />
<meta name="citation_author" content="Oliver C. Grant" />
<meta name="citation_author_institution" content="Complex Carbohydrate Research Center, University of Georgia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4835-5161" />
<meta name="citation_author" content="David Montgomery" />
<meta name="citation_author_institution" content="Complex Carbohydrate Research Center, University of Georgia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8112-8695" />
<meta name="citation_author" content="Keigo Ito" />
<meta name="citation_author_institution" content="Complex Carbohydrate Research Center, University of Georgia" />
<meta name="citation_author" content="Robert J. Woods" />
<meta name="citation_author_institution" content="Complex Carbohydrate Research Center, University of Georgia" />
<meta name="citation_author_email" content="rwoods@ccrc.uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2400-6293" />
<meta name="citation_reference" content="W.H.O. Coronavirus disease 2019 (Covid-19) Situation Report. Report No. 77, (2020)." />
<meta name="citation_reference" content="Depetris, R. S. et al. Partial enzymatic deglycosylation preserves the structure of cleaved recombinant HIV-1 envelope glycoprotein trimers. J Biol Chem 287, 24239–24254 (2012)." />
<meta name="citation_reference" content="Pereira, M. S. et al. Glycans as Key Checkpoints of T Cell Activity and Function. Front Immunol 9, 2754 (2018)." />
<meta name="citation_reference" content="Baum, L. G. &amp; Cobb, B. A. The direct and indirect effects of glycans on immune function. Glycobiology 27, 619–624 (2017)." />
<meta name="citation_reference" content="Vigerust, D. J. &amp; Shepherd, V. L. Virus glycosylation: role in virulence and immune interactions. Trends in microbiology 15, 211–218 (2007)." />
<meta name="citation_reference" content="Casals, C., Campanero-Rhodes, M. A., Garcia-Fojeda, B. &amp; Solis, D. The Role of Collectins and Galectins in Lung Innate Immune Defense. Front Immunol 9, 1998 (2018)." />
<meta name="citation_reference" content="Hutter, J. et al. Toward animal cell culture-based influenza vaccine design: viral hemagglutinin N-glycosylation markedly impacts immunogenicity. J Immunol 190, 220–230 (2013)." />
<meta name="citation_reference" content="Stevens, J. et al. Structure and receptor specificity of the hemagglutinin from an H5N1 influenza virus. Science 312, 404–410 (2006)." />
<meta name="citation_reference" content="Cotter, C. R., Jin, H. &amp; Chen, Z. A single amino acid in the stalk region of the H1N1pdm influenza virus HA protein affects viral fusion, stability and infectivity. PLoS Pathog 10, e1003831 (2014)." />
<meta name="citation_reference" content="Li, Y. et al. Single hemagglutinin mutations that alter both antigenicity and receptor binding avidity influence influenza virus antigenic clustering. J Virol 87, 9904–9910 (2013)." />
<meta name="citation_reference" content="Altman, M. O. et al. Human Influenza A Virus Hemagglutinin Glycan Evolution Follows a Temporal Pattern to a Glycan Limit. mBio 10 (2019)." />
<meta name="citation_reference" content="Zost, S. J. et al. Contemporary H3N2 influenza viruses have a glycosylation site that alters binding of antibodies elicited by egg-adapted vaccine strains. Proc Natl Acad Sci U S A 114, 12578–12583 (2017)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (2020)." />
<meta name="citation_reference" content="Yuan, Y. et al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat Commun 8, 15092 (2017)." />
<meta name="citation_reference" content="Watanabe, Y., Allen, J. D., Wrapp, D., McLellan, J. S. &amp; Crispin, M. Site-specific analysis of the SARS-CoV-2 glycan shield. Preprint at https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1 (2020)." />
<meta name="citation_reference" content="Khatri, K. et al. Integrated Omics and Computational Glycobiology Reveal Structural Basis for Influenza A Virus Glycan Microheterogeneity and Host Interactions. Mol. Cell. Proteomics 15, 1895–1912 (2016)." />
<meta name="citation_reference" content="An, Y. et al. N-Glycosylation of Seasonal Influenza Vaccine Hemagglutinins: Implication for Potency Testing and Immune Processing. J Virol 93 (2019)." />
<meta name="citation_reference" content="An, Y. et al. Comparative Glycomics Analysis of Influenza Hemagglutinin (H5N1) Produced in Vaccine Relevant Cell Platforms. J. Proteome Res. 12, 3707–3720 (2013)." />
<meta name="citation_reference" content="Urbanowicz, R. A. et al. Antigenicity and Immunogenicity of Differentially Glycosylated Hepatitis C Virus E2 Envelope Proteins Expressed in Mammalian and Insect Cells. J Virol 93 (2019)." />
<meta name="citation_reference" content="Hang, I. et al. Analysis of site-specific N-glycan remodeling in the endoplasmic reticulum and the Golgi. Glycobiology 25 (2015)." />
<meta name="citation_reference" content="Losfeld, M. E. et al. Influence of protein/glycan interaction on site-specific glycan heterogeneity. FASEB J 31, 4623–4635 (2017)." />
<meta name="citation_reference" content="Arigoni-Affolter, I. et al. Mechanistic reconstruction of glycoprotein secretion through monitoring of intracellular N-glycan processing. Sci Adv 5, eaax8930 (2019)." />
<meta name="citation_reference" content="York, I. A., Stevens, J. &amp; Alymova, I. V. Influenza virus N-linked glycosylation and innate immunity. Biosci Rep 39 (2019)." />
<meta name="citation_reference" content="Skehel, J. J. et al. A carbohydrate side chain on hemagglutinins of Hong Kong influenza viruses inhibits recognition by a monoclonal antibody. Proc Natl Acad Sci U S A 81, 1779–1783 (1984)." />
<meta name="citation_reference" content="An, Y., McCullers, J. A., Alymova, I., Parsons, L. M. &amp; Cipollo, J. F. Glycosylation Analysis of Engineered H3N2 Influenza A Virus Hemagglutinins with Sequentially Added Historically Relevant Glycosylation Sites. J Proteome Res 14, 3957–3969 (2015)." />
<meta name="citation_reference" content="Homans, S. W., Dwek, R. A. &amp; Rademacher, T. W. Solution Conformations of N-Linked Oligosaccharides. Biochemistry 26, 6571–6578 (1987)." />
<meta name="citation_reference" content="Homans, S. W. et al. Conformational Transitions in N-Linked Oligosaccharides. Biochemistry 25, 6342–6350 (1986)." />
<meta name="citation_reference" content="Jo, S., Qi, Y. &amp; Im, W. Preferred conformations of N-glycan core pentasaccharide in solution and in glycoproteins. Glycobiology 26, 19–29 (2016)." />
<meta name="citation_reference" content="Harbison, A. &amp; Fadda, E. An atomistic perspective on ADCC quenching by core-fucosylation of IgG1 Fc N-glycans from enhanced sampling molecular dynamics. Glycobiology (2019)." />
<meta name="citation_reference" content="Amaro, R. E. &amp; Li, W. W. Molecular-level simulation of pandemic influenza glycoproteins. Methods Mol Biol 819, 575–594 (2012)." />
<meta name="citation_reference" content="Bernardi, A., Kirschner, K. N. &amp; Faller, R. Structural analysis of human glycoprotein butyrylcholinesterase using atomistic molecular dynamics: The importance of glycosylation site ASN241. PLoS One 12, e0187994 (2017)." />
<meta name="citation_reference" content="Yanaka, S. et al. Dynamic Views of the Fc Region of Immunoglobulin G Provided by Experimental and Computational Observations. Antibodies (Basel) 8 (2019)." />
<meta name="citation_reference" content="Peng, W. et al. Recent H3N2 Viruses Have Evolved Specificity for Extended, Branched Human-type Receptors, Conferring Potential for Increased Avidity. Cell Host Microbe 21, 23–34 (2017)." />
<meta name="citation_reference" content="Vigerust, D. J. et al. N-linked glycosylation attenuates H3N2 influenza viruses. J Virol 81, 8593–8600 (2007)." />
<meta name="citation_reference" content="Lin, Y. P. et al. Evolution of the receptor binding properties of the influenza A(H3N2) hemagglutinin. Proc Natl Acad Sci U S A 109, 21474–21479 (2012)." />
<meta name="citation_reference" content="Gulati, S. et al. Human H3N2 Influenza Viruses Isolated from 1968 To 2012 Show Varying Preference for Receptor Substructures with No Apparent Consequences for Disease or Spread. PLoS One 8, e66325 (2013)." />
<meta name="citation_reference" content="Hulswit, R. J. G. et al. Human coronaviruses OC43 and HKU1 bind to 9-O-acetylated sialic acids via a conserved receptor-binding site in spike protein domain A. Proc Natl Acad Sci U S A 116, 2681–2690 (2019)." />
<meta name="citation_reference" content="Tortorici, M. A. et al. Structural basis for human coronavirus attachment to sialic acid receptors. Nature Structural &amp; Molecular Biology 26, 481–489 (2019)." />
<meta name="citation_reference" content="Vlasak, R., Luytjes, W., Spaan, W. &amp; Palese, P. Human and bovine coronaviruses recognize sialic acid-containing receptors similar to those of influenza C viruses. Proc Natl Acad Sci U S A 85, 4526–4529 (1988)." />
<meta name="citation_reference" content="Li, W. et al. Identification of sialic acid-binding function for the Middle East respiratory syndrome coronavirus spike glycoprotein. Proc Natl Acad Sci U S A 114, E8508–E8517 (2017)." />
<meta name="citation_reference" content="Park, Y. J. et al. Structures of MERS-CoV spike glycoprotein in complex with sialoside attachment receptors. Nature Structural &amp; Molecular Biology 26, 1151–1157 (2019)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)." />
<meta name="citation_reference" content="Yan, R. et al. Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science (2020)." />
<meta name="citation_reference" content="Hoffmann, M. et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell (2020)." />
<meta name="citation_reference" content="Connell, B. J. &amp; Lortat-Jacob, H. Human immunodeficiency virus and heparan sulfate: from attachment to entry inhibition. Front Immunol 4 (2013)." />
<meta name="citation_reference" content="Zheng, M. &amp; Song, L. Novel antibody epitopes dominate the antigenicity of spike glycoprotein in SARS-CoV-2 compared to SARS-CoV. Cell Mol Immunol (2020)." />
<meta name="citation_reference" content="Gui, M. et al. Electron microscopy studies of the coronavirus ribonucleoprotein complex. Protein Cell 8, 219–224 (2017)." />
<meta name="citation_reference" content="Pallesen, J. et al. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proc Natl Acad Sci U S A 114, E7348–E7357 (2017)." />
<meta name="citation_reference" content="Lan, J. et al. Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. Preprint at https://www.biorxiv.org/content/10.1101/2020.02.19.956235v1 (2020)." />
<meta name="citation_reference" content="Prabakaran, P. et al. Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody. J Biol Chem 281, 15829–15836 (2006)." />
<meta name="citation_reference" content="Hwang, W. C. et al. Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. J Biol Chem 281, 34610–34616 (2006)." />
<meta name="citation_reference" content="Pak, J. E. et al. Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain. J Mol Biol 388, 815–823 (2009)." />
<meta name="citation_reference" content="Ying, T. et al. Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody. Nat Commun 6, 8223 (2015)." />
<meta name="citation_reference" content="Wang, L. et al. Evaluation of candidate vaccine approaches for MERS-CoV. Nat Commun 6, 7712 (2015)." />
<meta name="citation_reference" content="Li, Y. et al. A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein. Cell Res 25, 1237–1249 (2015)." />
<meta name="citation_reference" content="Chen, Z. et al. Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset. J Infect Dis 215, 1807–1815 (2017)." />
<meta name="citation_reference" content="Zhang, S. et al. Structural Definition of a Unique Neutralization Epitope on the Receptor-Binding Domain of MERS-CoV Spike Glycoprotein. Cell Rep 24, 441–452 (2018)." />
<meta name="citation_reference" content="Wang, L. et al. Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape. J Virol 92 (2018)." />
<meta name="citation_reference" content="Zhou, H. et al. Structural definition of a neutralization epitope on the N-terminal domain of MERS-CoV spike glycoprotein. Nat Commun 10, 3068 (2019)." />
<meta name="citation_reference" content="Walls, A. C. et al. Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 176, 1026–1039 e1015 (2019)." />
<meta name="citation_reference" content="Wang, N. et al. Structural Definition of a Neutralization-Sensitive Epitope on the MERS-CoV S1-NTD. Cell Rep 28, 3395–3405 e3396 (2019)." />
<meta name="citation_reference" content="Humphrey, W., Dalke, A. &amp; Schulten, K. VMD - Visual Molecular Dynamics. J. Mol. Graphics 14, 33–38 (1996)." />
<meta name="citation_reference" content="Huang, C. C. et al. Structure of a V3-containing HIV-1 gp120 core. Science 310, 1025–1028 (2005)." />
<meta name="citation_reference" content="Zhou, T. et al. Structural definition of a conserved neutralization epitope on HIV-1 gp120. Nature 445, 732–737 (2007)." />
<meta name="citation_reference" content="Kwong, P. D. et al. Structures of HIV-1 gp120 envelope glycoproteins from laboratory-adapted and primary isolates. Structure 8, 1329–1339 (2000)." />
<meta name="citation_reference" content="Pejchal, R. et al. A Potent and Broad Neutralizing Antibody Recognizes and Penetrates the HIV Glycan Shield. Science 334, 1097–1103 (2011)." />
<meta name="citation_reference" content="Pettersen, E. F. et al. UCSF Chimera - A Visualization System for Exploratory Research and Analysis. J. Comp. Chem. 25, 1605–1612 (2004)." />
<meta name="citation_reference" content="NACCESS v. 2.1.1 (University College London, London, 1993)." />
<meta name="citation_reference" content="Horiya, S., MacPherson, I. S. &amp; Krauss, I. J. Recent strategies targeting HIV glycans in vaccine design. Nat. Chem. Biol. 10, 990–999 (2014)." />
<meta name="citation_reference" content="Doores, K. J. The HIV glycan shield as a target for broadly neutralizing antibodies. FEBS J 282, 4679–4691 (2015)." />
<meta name="citation_reference" content="Reche, P. A., Glutting, J. P., Zhang, H. &amp; Reinherz, E. L. Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles. Immunogenetics 56, 405–419 (2004)." />
<meta name="citation_reference" content="Reche, P. A., Glutting, J. P. &amp; Reinherz, E. L. Prediction of MHC class I binding peptides using profile motifs. Hum Immunol 63, 701–709 (2002)." />
<meta name="citation_reference" content="Avci, F. Y., Li, X., Tsuji, M. &amp; Kasper, D. L. A mechanism for glycoconjugate vaccine activation of the adaptive immune system and its implications for vaccine design. Nature: Medicine 17, 1602–1609 (2011)." />
<meta name="citation_reference" content="Malaker, S. A. et al. Identification and Characterization of Complex Glycosylated Peptides Presented by the MHC Class II Processing Pathway in Melanoma. J Proteome Res 16, 228–237 (2017)." />
<meta name="citation_reference" content="Haji-Ghassemi, O., Blackler, R. J., Martin Young, N. &amp; Evans, S. V. Antibody recognition of carbohydrate epitopes. Glycobiology 25, 920–952 (2015)." />
<meta name="citation_reference" content="Chang, D. &amp; Zaia, J. Why Glycosylation Matters in Building a Better Flu Vaccine. Mol Cell Proteomics 18, 2348–2358 (2019)." />
<meta name="citation_reference" content="Sun, X. et al. N-linked glycosylation of the hemagglutinin protein influences virulence and antigenicity of the 1918 pandemic and seasonal H1N1 influenza A viruses. J Virol 87, 8756–8766 (2013)." />
<meta name="citation_reference" content="Nivedha, A. K., Makeneni, S., Foley, B. L., Tessier, M. B. &amp; Woods, R. J. Importance of ligand conformational energies in carbohydrate docking: Sorting the wheat from the chaff. J Comput Chem 35, 526–539 (2014)." />
<meta name="citation_reference" content="Grant, O. C. et al. Gly-Spec: a webtool for predicting glycan specificity by integrating glycan array screening data and 3D structure. Glycobiology 26, 1027–1028 (2016)." />
<meta name="citation_reference" content="Jorgensen, W. L. Quantum and Statistical Mechanical Studies of Liquids. 10. Transferable Intermolecular Potential Functions for Water, Alcohols, and Ethers. Application to Liquid Water. J. Am. Chem. Soc. 103, 335–340 (1981)." />
<meta name="citation_reference" content="Salomon-Ferrer, R., Götz, A. W., Poole, D., Le Grand, S. &amp; Walker, R. C. Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald. J. Chem. Theory Comput. 9, 3878–3888 (2013)." />
<meta name="citation_reference" content="Gotz, A. W. et al. Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born. J. Chem. Theory Comput. 8, 1542–1555 (2012)." />
<meta name="citation_reference" content="AMBER 14 (University of California, San Francisco, 2014)." />
<meta name="citation_reference" content="Kirschner, K. N. et al. GLYCAM06: a generalizable biomolecular force field. Carbohydrates. J. Comput. Chem. 29, 622–655 (2008)." />
<meta name="citation_reference" content="Maier, J. A. et al. ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB. J Chem Theory Comput 11 (2015)." />
<meta name="citation_reference" content="Darden, T., York, D. &amp; Pedersen, L. Particle mesh Ewald: An N⋅ log (N) method for Ewald sums in large systems. The Journal of chemical physics 98, 10089 (1993)." />
<meta name="citation_reference" content="Ryckaert, J.-P., Ciccotti, G. &amp; Berendsen, H. J. C. Numerical integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes. J. Comput. Phys. 23, 327–341 (1977)." />
<meta name="twitter:title" content="3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Here we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells. We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses.

These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity. Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.

### Competing Interest Statement" />
<meta name="og-title" property="og:title" content="3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Here we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells. We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses.

These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity. Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.

### Competing Interest Statement" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.07.030445v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.07.030445v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.07.030445v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1237294" />
    <title>3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"THvxnyfyczv1PHkcTa6fA6RR9NsuoLlJlIGigq010Kw","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1237294","apath":"\/biorxiv\/early\/2020\/04\/09\/2020.04.07.030445.atom","pisa":"biorxiv;2020.04.07.030445v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.07.030445v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.07.030445v1","title":"3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development","identifier":"node\/1237294"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TlcJWVDfGR1mjWAi3fcPPgbRgMv\\\\\\\/gFmRDO6+1pI6+sSYzdBJ3RyQ2idkV8h0mJu2OcGjjkIkq7punveAB322VsQaWoawTjqg38QvUW1dk0tulqMXKU+7kGKJedDAZRlQ7CH4jvD5mAKIOnGqFZBD\\\\\\\/DIpUZ9si2i+qctty+sOFP4E\\\\\\\/c0Wf11jJSyjPfS7Ad0L793nGb5kzR\\\\\\\/gysdMkJNJHH+URMuu4Wqkv00nVSOtIWeLC9aHut+XnEVCj\\\\\\\/SFVQvJiM3Y3vsCj4xr5LkVGLN89+zh65Es8tfxwVp+HPl9dwFGVwzwGRLlreKJatTa8X6GWoqk0Zvfdfm1TgUl0RBiwEqCWnElMoJQSznSJ9x5Aw9B+uVk1EPwPJWy0Bla6Yh6p3eCX4Aj5c0M\\\\\\\/zjOGoJvFsexL1evqlWyoKX5rH2K6jvg+LPNyvpEsyE1RHq5baRFnBcij4yLI5Ki1XNG+FMBNM6L6XlKnVbIxIJw8V1h5bZHdJ\\\\\\\/k\\\\\\\/m5K095LSFJn5M2l7rj1D94h3J9XCXg1QPq2s8EGGmRxhaR1Dq1q60MQlV8H9nP70o8hvn\\\\\\\/4WK77P18SBykIydAfyPsZUPoCJB3BuzaBlaaT4pGrv+K3z+seavBYYaBPNrOdJ86CV8ZsEoktmXAvUo7nFHx12Ks3g2A8h1XPHe4L1InjtolcPkb1PXKLicNd\\\\\\\/cKAyfZk9En+EOV0dkHUPoS3RCtRzM9sTZ4xKaDwwakrZdpCpQgenq3N6jREFVNTmNw1CuZAVozpNDfV82g76xtri\\\\\\\/y8WFVtieILk5oGoWHNHu6eDChCXEw\\\\\\\/Ia4Y+bZ43GrTEuQ0Nu\\\\\\\/xZO+jQAMS8SYkxXDqP0K7cYseNs69m7jd8KdBTzy38NStbhlt+HF3bBChcYsWbvs1vcUR7v\\\\\\\/2iNEMNBuGw7UjfdieOuEOHGU2615+K7r40URMFxeAkiPmp\\\\\\\/f0z+snaGnH0MjDMnY9figozpODvhXD0CDS9sEqP\\\\\\\/0RiEcehw4PDghWv0RiJu4JR7+\\\\\\\/qL7b7ZqQDpVZBAtm\\\\\\\/zk5fBNKO6hHMtQm62gFa1SqXIy0dCEzIbD7yN7+kFKPrt7+aDhuUpCpgzom2w+ZFSYH9t5ispjdZiFSCxTlueyM3znWnfy+n1MyriJjVnNHeJbLA0hv8KtPl\\\\\\\/EAFiW6cjgFrKi5Qnfe6bqLp6HW1Gtwk0MtcKWO2sOvCakWGtXcO2VH9CbNuxXq1yOZAKzQiWR0sHFh8lA6rkjHf\\\\\\\/Erwb3Ilv\\\\\\\/wOWieMHZFhjDHYUiiWEWNLtZO+XT99qkTyNQVa+Kvf5wZwDz3wiOrBmc3yDhJrG\\\\\\\/IvPd7iAYNqg8XudGoSKPGJslvJY19A06E\\\u0022,\\\u0022iv\\\u0022:\\\u00224ycmoPqMQqmVtdXRPYH3JQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022cc50d741c2304402c14f69c5ffab80056a88a4295ab3c07b56949694ab45275b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.07.030445v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1237294 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.07.030445v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7uogRrPJBIJKM0xuQDoq1gy6L9F8etxFFg--bKah8ao" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237294" id="node1237294" data-pisa="biorxiv;2020.04.07.030445v1" data-pisa-master="biorxiv;2020.04.07.030445" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030445.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4835-5161" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Oliver C.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8112-8695" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">David</span> <span class="nlm-surname">Montgomery</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keigo</span> <span class="nlm-surname">Ito</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2400-6293" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Woods</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.07.030445 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1237294" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Oliver C. Grant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Complex Carbohydrate Research Center, University of Georgia</span>, 315 Riverbend Rd, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BC.%2BGrant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grant%20OC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BC.%2BGrant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4835-5161" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Oliver C. Grant</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Montgomery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Complex Carbohydrate Research Center, University of Georgia</span>, 315 Riverbend Rd, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMontgomery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montgomery%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BMontgomery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8112-8695" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Montgomery</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Keigo Ito </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Complex Carbohydrate Research Center, University of Georgia</span>, 315 Riverbend Rd, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keigo%2BIto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ito%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeigo%2BIto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert J. Woods </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Complex Carbohydrate Research Center, University of Georgia</span>, 315 Riverbend Rd, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BWoods%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woods%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BJ.%2BWoods%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2400-6293" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert J. Woods</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwoods@ccrc.uga.edu" class="" data-icon-position="" data-hide-link-title="0">rwoods@ccrc.uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.07.030445v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030445v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030445v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030445v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030445v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030445v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.07.030445v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1237294" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1237294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Here we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells. We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses.</p><p id="p-3">These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity. Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">The present COVID-19 pandemic has led to over a million confirmed infections with a fatality rate of approximately 5 percent <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> since the first reports of a severe acute respiratory syndrome (SARS) infection by a novel coronavirus (SARS-CoV-2) at the end of 2019. As of April 2020, there is still no vaccine or approved therapeutic to treat this disease. Here we examine the structure of the SARS-CoV-2 envelope spike (S) protein that mediates host cell infection, with a specific focus on the extent to which S protein glycosylation masks this virus antigen from the host immune response.</p><p id="p-7">Viral envelope proteins are often modified by the attachment of complex glycans that can account for up to half of the molecular weight of these glycoproteins, as in HIV gp120 <sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. The glycosylation of these surface antigens helps the pathogen evade recognition by the host immune system by cloaking the protein surface from detection by antibodies, and can influence the ability of the host to raise an effective adaptive immune response <sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> or even be exploited by the virus to enhance infectivity <sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. Fortunately, the innate immune system has evolved a range of strategies for responding to glycosylated pathogens <sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, but antigen glycosylation nevertheless complicates the development of vaccines <sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Over time, the protein sequences in viral antigens undergo mutations, which can alter the species specificity of the virus <sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, modulate its infectivity <sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>, and alter the antigenicity of the surface proteins <sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. These mutations can also impact the degree to which the protein is glycosylated by creating new or removing existing locations of the glycans (glycosites) on the antigens <sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Varying surface antigen glycosylation is thus a mechanism by which new virus strains can evade the host immune response <sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup>, and attenuate the efficacy of existing vaccines <sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup>.</p><p id="p-8">Very recently, a cryo-EM structure of the SARS-CoV-2 S glycoprotein has been reported <sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>, which led to conclusion that, like the related protein from the 2002 - 2003 SARS pandemic (SARS-CoV-1) <sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, the CoV-2 S protein is also extensively glycosylated <sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup>. Furthermore, an analysis of the glycan structures present at each glycosite in the S protein produced recombinantly in human embryonic kidney 293 cells has also been recently reported <sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>.</p><p id="p-9">Here we have generated 3D structures of several glycoforms of the SARS-CoV-2 S glycoprotein, in which the glycans represent those present in the S protein produced in HEK293 cells <sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>, as well as those corresponding to the nascent glycoprotein (prior to enzymatic modifications in the Golgi), as well as those that are commonly observed on antigens present in other viruses <sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. We have subjected these models to long MD simulations and compare the extent to which glycan microheterogeneity impacts epitope exposure. Additionally, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-10">The impact of glycosylation on the ability of antibodies to bind to a pathogenic glycoprotein may be estimated by quantifying the fraction of the surface area of the protein antigen that is physically shielded by glycans from antibody recognition. Such an analytical approach to defining glycoprotein antigenicity led to the conclusion that glycosylation of the E2 envelope protein of hepatitis C virus reduces its surface area by up to 65% <sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. The structure of proteins may be readily determined experimentally using techniques such as x-ray crystallography or cryo-electron microscopy (cryo-EM), however, it is far more challenging to determine the structure of the glycans. This is because glycans are often significantly more flexible than the proteins to which they are attached, and also because multiple different glycan structures may be present at any given glycosite, giving rise to an ensemble of glycosylated protein variants (glycoforms). This glycan heterogeneity is a natural result of the mechanism of protein glycosylation. While all glycans that are added to proteins through attachment to asparagine side chains (N-linked glycans) are transferred <em>en bloc</em> as a single glycan precursor structure, subsequent passage of the nascent glycoprotein through the cellular machinery (Endoplasmic reticulum, Golgi apparatus) exposes the glycans to enzymes, glycosidases and glycosyl transferases, that modify their structure. The ability of enzymes to alter or process each glycan depends in part on the 3D shape of the protein to which the glycan is attached, particularly in the region of the glycosite <sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>, and on the time that any given glycan is exposed to these enzymes <sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. The presence of multiple glycosites and the variation in the composition of the glycans at a glycosite, can result in a staggering number of permutations of protein glycoforms. Although the particular structure of the glycans at the glycosites may not impact protein immunogenicity, as reported in the case of hepatitis C virus <sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>, the location and number or density of glycosites can have a profound impact on the innate immune response to virus <sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> and on the antigenicity of viral envelope proteins <sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>, as observed in the case of influenza A. Lastly, to further complicate our understanding of the effect of glycosylation on protein antigenicity, glycans display large internal motions that prevents their accurate description by any single 3D shape, in contrast to proteins <sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>.</p><p id="p-11">Fortunately, computational simulations can play a key role in characterizing glycoproteins by predicting the likely 3D shapes of the glycans <sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. In a typical simulation, the glycans are added to the experimentally-determined 3D structure of the protein, and the resulting static model of the glycoprotein is then subjected to molecular dynamics (MD) simulations that approximate the effects of water and temperature on the temporal and spatial properties of the glycoprotein over the simulation time frame. The 3D structures obtained from the simulation can then be used to determine the extent to which glycosylation has altered the protein’s antigenicity <sup><a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a></sup> or other properties. When the structure of specific glycans at particular glycosites is known, these glycans can be employed in the simulation <sup><a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, however this data is often unavailable and plausible glycosylation states may have to be assumed, particularly in the case of proteins with multiple permutations of glycoforms <sup><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a></sup>.</p><div id="sec-3" class="subsection"><h3>Model glycoforms</h3><p id="p-12">It is well established that there is a strong dependence of both the composition and relative glycan abundance (glycan microheterogeneity) on the cell type used in glycoprotein production. And there is a large body of data relating to the influence of host cell line on viral envelop protein glycosylation. For example, a glycomics analysis of influenza A virus produced in five different cell lines, all of relevance to vaccine production, let to the observation of profound differences in the compositions of the glycans at a given site; with structures varying from paucimannose (Sf9 cells) to core-fucosylated hybrid with bisecting N-acetylglucosamine (Egg) to sialylated biantennary glycans (HEK293) <sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>. For these reasons, we have modeled the S glycoprotein with reported site-specific glycosylation <sup><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a></sup>, as well as hypothetical homogeneously glycosylated glycoforms of the high mannose (M9), paucimannose (M3), biantennary complex (Complex) and core-fucosylated biantennary complex (Complex Core F) types. Comparisons among the glycoforms permits an assessment of the impact of cell-based differential glycan processing on S protein antigenicity.</p><p id="p-13">Notably, differences in glycan microheterogeneity have been shown not to significantly affect the potency of influenza antigens, as determined in single radial immunodiffusion (SRID) assays; an assay that is typically used to measure the potency of commercial influenza vaccines <sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. That the precise composition of the glycans may not alter antigenicity, suggests that glycan microheterogeneity may not heavily impact the exposed surface area of the protein. Nevertheless, such glycan variations have been shown to impact the ability of innate immune lectins, such as surfactant protein-D (SPD), to respond to the antigen <sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>. Because SPD requires a cluster of high mannose glycans on the pathogenic protein in order to bind <sup><a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">16</a></sup>, variations in the glycan composition can impair such a response. Again, in the case of influenza, increases in the number of glycosites have been shown to impact receptor binding, protein folding, virulence, antigenicity, and shielding of immunogenic sites <sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>–<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. To assess the impact glycosylation on S protein antigenicity, an initial comparison with other more-fully characterized coronaviruses was undertaken.</p></div><div id="sec-4" class="subsection"><h3>Comparison with epitopes in related coronavirus S glycoproteins</h3><p id="p-14">Viral adhesion to host cells is enabled by proteins on the virus surface (or envelope) that evolve to recognize specific receptors on the host cell. The receptors may be glycans, as is well known in the case of influenza, endemic human CoVs, and MERS-CoV <sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>–<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>, or proteins, as in the case of the SARS-CoV-1 <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> and current SARS-CoV-2 viruses <sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>, or both, as in human immunodeficiency virus (HIV) <sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. The receptor binding domain (RBD) within domain 1 of the S protein (S1) in SARS-CoV-1 and -2 has been shown to interact with the host protein angiotensin-converting enzyme 2 (ACE2). The affinity of S1 for ACE2 enables the virus to adhere to and infect the host cell <sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>. Being exposed on the viral surface, S proteins are a major target for host antibodies and are referred to as viral antigens; these antigens are therefore targets for vaccine development.</p><p id="p-15">The S glycoproteins of SARS-CoV-1 and CoV-2 share a high degree of structural similarity, with an average root-mean-squared difference (RMSD) in the Cα positions of only 3.8 Å <sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a></sup>. They also share a relatively high sequence homology of 75.5% <sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. The MERS S glycoprotein also shares a similar trimeric structure with CoV-1 and Cov-2. Cryo-EM studies show that the receptor binding domains of the S1 subunits of the CoV-2 S glycoprotein can exist in at least two conformations by undergoing a hinge-like movement, which has also been reported in SARS-CoV-1 and MERS, as well as other more distantly related coronaviruses <sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>. Despite the structural and sequence similarities between the SARS-CoV-1 and CoV-2 S glycoproteins, sequence based epitope prediction indicates that approximately 85% of the theoretical sequential epitopes are unique between the two proteins <sup><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a></sup>.</p><p id="p-16">To better understand the antigenic properties of the SARS-CoV-2 S protein from a structural perspective, we aligned the 3D structure of the S glycoprotein from SARS-CoV-2 <sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a></sup> with those from co-crystal structures of SARS-CoV-1 and MERS that contained bound antibody fragments. From this alignment, the extent to which epitopes in the CoV-2 S glycoprotein might be inaccessible to known antibodies on the basis of structural differences in the glycoproteins or due to shielding by glycans on the CoV-2 S glycoprotein surface (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) was inferred. The previously reported antibodies bind almost exclusively to the S1 RBD. When aligned on the glycosylated CoV-2 S protein, it is evident that approximately 50% of the corresponding CoV-1 or MERS epitopes are shielded by glycans from antibody binding in the CoV-2 S glycoprotein, and only areas of the protein surface at the apex of the S1 domain appear to be accessible to known antibodies. The epitope surface area in the known antibody co-complexes ranges between approximately 800 to 1000 Å<sup>2</sup>, which is comparable to the contact surface between the S glycoprotein and its ACE2 receptor <sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="a. Side view (upper panels) and Top view (lower panels) of the SARS-CoV-2 S protein (grey surface) with homogeneous Complex glycosylation (magenta) showing aligned antibody fragments (ribbons) from co-complexes with the S glycoproteins from SARS-CoV-1 (orange) and MERS (cyan) 48,50&#x2013;61. b. Glycans present on SARS-CoV-2 S glycoprotein that are incompatible with known antibody positions due to steric overlap are shown in yellow. c. Potential antibody poses after elimination of epitopes blocked by S protein glycosylation. Images generated using Visual Molecular Dynamics (VMD) 62 version 1.9.3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2080772347" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;a.&lt;/strong&gt; Side view (upper panels) and Top view (lower panels) of the SARS-CoV-2 S protein (grey surface) with homogeneous Complex glycosylation (magenta) showing aligned antibody fragments (ribbons) from co-complexes with the S glycoproteins from SARS-CoV-1 (orange) and MERS (cyan) &lt;sup&gt;48,50&#x2013;61&lt;/sup&gt;. &lt;strong&gt;b.&lt;/strong&gt; Glycans present on SARS-CoV-2 S glycoprotein that are incompatible with known antibody positions due to steric overlap are shown in yellow. &lt;strong&gt;c.&lt;/strong&gt; Potential antibody poses after elimination of epitopes blocked by S protein glycosylation. Images generated using Visual Molecular Dynamics (VMD) &lt;sup&gt;62&lt;/sup&gt; version 1.9.3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F1.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F1.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-17" class="first-child"><strong>a.</strong> Side view (upper panels) and Top view (lower panels) of the SARS-CoV-2 S protein (grey surface) with homogeneous Complex glycosylation (magenta) showing aligned antibody fragments (ribbons) from co-complexes with the S glycoproteins from SARS-CoV-1 (orange) and MERS (cyan) <sup><a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>–<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>. <strong>b.</strong> Glycans present on SARS-CoV-2 S glycoprotein that are incompatible with known antibody positions due to steric overlap are shown in yellow. <strong>c.</strong> Potential antibody poses after elimination of epitopes blocked by S protein glycosylation. Images generated using Visual Molecular Dynamics (VMD) <sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup> version 1.9.3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">This preliminary epitope analysis does not take into account variations in site-specific glycan composition or the plasticity of the glycans or the protein. Furthermore, the number of antibodies that have been co-crystallized with the related S proteins represents only a minute fraction of the possible repertoire. To address these limitations, we subjected multiple glycoforms of the CoV-2 S glycoprotein to MD simulation and interpreted the results in terms of the impact of glycan structure on the theoretical S glycoprotein antigenic surface area (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="To estimate the antibody accessible surface area (AbASA), a spherical probe was derived (radius 7.2 &#xC5;, smaller sphere) that approximates the average size of the hypervariable loops from four anti-gp120 antibodies, in which the epitopes were either protein surface residues (PDB IDs: 2B4C 63, 2NY7 64, 1G9M 65) or both carbohydrate and protein residues: (3TYG 66). This probe size may be compared to values of 5 and 10 &#xC5; employed previously to estimate antigenic surface area 19. Changes in the solvent accessible surface area (SASA) showed no shielding by glycans and thus a simple SASA model was not useful for this analysis19. Additionally, to account for the presence of the beta-sheet framework in the antibody variable fragment (Fv), we introduced a second larger probe (18.6 &#xC5;) sufficient to approximately enclose that domain. The antigenic surface area is then defined as sum of the surface areas of any protein residues that make contact with the CDR probe, provided that the CDR probe is proximal to residues accessible to the Fv probe. This latter requirement is governed by Lmax, which requires that the distance between the CDR-antigen contact site and the Fv probe surface be less than the length (10.4 &#xC5;) of the longest CDR loop in mAb PGT128. PGT128 was chosen for this reference as it contains a particularly long CDR loop that penetrates the glycan shield of gp120. Images generated with UCSF Chimera 67." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2080772347" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;To estimate the antibody accessible surface area (AbASA), a spherical probe was derived (radius 7.2 &#xC5;, smaller sphere) that approximates the average size of the hypervariable loops from four anti-gp120 antibodies, in which the epitopes were either protein surface residues (PDB IDs: 2B4C &lt;sup&gt;63&lt;/sup&gt;, 2NY7 &lt;sup&gt;64&lt;/sup&gt;, 1G9M &lt;sup&gt;65&lt;/sup&gt;) or both carbohydrate and protein residues: (3TYG &lt;sup&gt;66&lt;/sup&gt;). This probe size may be compared to values of 5 and 10 &#xC5; employed previously to estimate antigenic surface area &lt;sup&gt;19&lt;/sup&gt;. Changes in the solvent accessible surface area (SASA) showed no shielding by glycans and thus a simple SASA model was not useful for this analysis&lt;sup&gt;19&lt;/sup&gt;. Additionally, to account for the presence of the beta-sheet framework in the antibody variable fragment (Fv), we introduced a second larger probe (18.6 &#xC5;) sufficient to approximately enclose that domain. The antigenic surface area is then defined as sum of the surface areas of any protein residues that make contact with the CDR probe, provided that the CDR probe is proximal to residues accessible to the Fv probe. This latter requirement is governed by L&lt;sub&gt;max&lt;/sub&gt;, which requires that the distance between the CDR-antigen contact site and the Fv probe surface be less than the length (10.4 &#xC5;) of the longest CDR loop in mAb PGT128. PGT128 was chosen for this reference as it contains a particularly long CDR loop that penetrates the glycan shield of gp120. Images generated with UCSF Chimera &lt;sup&gt;67&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F2.medium.gif" width="440" height="211"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F2.medium.gif" width="440" height="211"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-19" class="first-child">To estimate the antibody accessible surface area (AbASA), a spherical probe was derived (radius 7.2 Å, smaller sphere) that approximates the average size of the hypervariable loops from four anti-gp120 antibodies, in which the epitopes were either protein surface residues (PDB IDs: 2B4C <sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>, 2NY7 <sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>, 1G9M <sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup>) or both carbohydrate and protein residues: (3TYG <sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>). This probe size may be compared to values of 5 and 10 Å employed previously to estimate antigenic surface area <sup><a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">19</a></sup>. Changes in the solvent accessible surface area (SASA) showed no shielding by glycans and thus a simple SASA model was not useful for this analysis<sup><a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">19</a></sup>. Additionally, to account for the presence of the beta-sheet framework in the antibody variable fragment (Fv), we introduced a second larger probe (18.6 Å) sufficient to approximately enclose that domain. The antigenic surface area is then defined as sum of the surface areas of any protein residues that make contact with the CDR probe, provided that the CDR probe is proximal to residues accessible to the Fv probe. This latter requirement is governed by L<sub>max</sub>, which requires that the distance between the CDR-antigen contact site and the Fv probe surface be less than the length (10.4 Å) of the longest CDR loop in mAb PGT128. PGT128 was chosen for this reference as it contains a particularly long CDR loop that penetrates the glycan shield of gp120. Images generated with UCSF Chimera <sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Assessment of the impact of glycosylation on antigenicity</h3><p id="p-20">Three independent MD simulations of five glycoforms of the S protein were performed for a combined total of 0.25 μs per glycoform and used to compute the extent to the antigenic surface area was sensitive to glycan microheterogeneity (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The data indicate that uniform glycosylation with the smallest of the glycans (paucimannose, M3), which is a sub-structure within all N-linked glycans, provided the least shielding of the S protein surface, leaving 71% of the surface exposed to an antibody probe. In contrast, the largest high mannose N-linked glycans (M9), which corresponds to the nascent glycoform that would exist prior to processing through the Endoplasmic reticulum and Golgi apparatus, led to the highest level of surface shielding.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242749/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242749/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242749" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-21" class="first-child">SARS-CoV-2 S glycoprotein antigenic surface areas (Å<sup>2</sup>) as a function of glycoform.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The extent of shielding offered by the two complex types of glycans are not statistically different from that of M9 at 53-55% antigenic surface exposure. A site-specific model generated from the most commonly observed glycans at each glycosite for the S glycoprotein produced recombinantly in HEK293 cells was also included for comparison. This glycoform resulted in shielding of just over 40% of the S protein antigenic surface; a value similar to the models based on uniform M9 and Complex glycans. These results are broadly consistent with the conclusion that antigenicity of the S protein is insensitive to glycan microheterogeneity, with the exception of the glycoform composed solely of M3 glycans. Nevertheless, differences in glycosylation may impact other features, such as local interactions between the glycan and the protein surface, or local structural fluctuations in either the protein or glycan conformations that are only partially captured by the CRD probe analysis.</p><p id="p-23">A visual examination of the structures from MD simulation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and S1) broadly confirms the observation (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) that the most exposed epitopes comprise the ACE2 receptor site, specifically the apex region of the S1 domain when that domain is in the open conformation. Moreover, the extensive motion displayed by each glycan illustrates that no single static model can fully capture the extent of glycan shielding. It can also be observed that a ring of antigenic sites appears to encircle the S1 domain, independent of glycoform. Unlike the extremely high level of glycan shielding in gp120 that challenges HIV vaccine development <sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>,<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup>, the level of shielding by glycans in the S protein is more moderate, with approximately 60% of the surface potentially accessible to antibodies.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overlay of snapshots from MD simulation of the S glycoprotein with site-specific glycosylation. The glycans are shown in ball-and-stick representation: M9 (green), M5 (dark yellow), hybrid (orange), complex (pink) (See Table S1 for details). The protein surface is colored according to antibody accessibility from black to red (least to most accessible). Images generated using Visual Molecular Dynamics (VMD) 62 version 1.9.3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2080772347" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overlay of snapshots from MD simulation of the S glycoprotein with site-specific glycosylation. The glycans are shown in ball-and-stick representation: M9 (green), M5 (dark yellow), hybrid (orange), complex (pink) (See Table S1 for details). The protein surface is colored according to antibody accessibility from black to red (least to most accessible). Images generated using Visual Molecular Dynamics (VMD) &lt;sup&gt;62&lt;/sup&gt; version 1.9.3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F3.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F3.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-24" class="first-child">Overlay of snapshots from MD simulation of the S glycoprotein with site-specific glycosylation. The glycans are shown in ball-and-stick representation: M9 (green), M5 (dark yellow), hybrid (orange), complex (pink) (See Table S1 for details). The protein surface is colored according to antibody accessibility from black to red (least to most accessible). Images generated using Visual Molecular Dynamics (VMD) <sup><a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a></sup> version 1.9.3.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Adaptive immune response to SARS-CoV-2</h3><p id="p-25">Beyond a role in shielding the underlying protein from recognition by antibodies, the glycans on pathogenic proteins may also attenuate the ability of the host immune system to raise antibodies against any epitopes that include the glycan. In a T-cell dependent adaptive immune response, peptides from the endocytosed pathogen are presented on antigen presenting cells by major histocompatibility complex II molecules, known as human leukocyte antigen (HLA) complexes. HLA complexes have preferred peptide motifs, and based on a knowledge of these preferences it is possible to predict which peptides in a protein are likely to be HLA antigens <sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>,<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></sup>. However, when that peptide contains a glycosylation site, the ability of the peptide to be presented in an HLA complex may be compromised, if for example the peptide cannot bind to the HLA molecule due to the steric presence of the glycan. However, glycopeptides may be presented in HLA complexes <sup><a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></sup> if the glycan is small enough or if it is found on the end of the peptide antigen where it doesn’t interfere with HLA binding <sup><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></sup>. The glycan-mediated shielding of predicted HLA antigens (Table S2) derived from the S protein are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a>, S2 and S3 for all HLA peptide sequences that also contain a glycosite.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence of the S protein (NCBI: YP_009724390.1) used to generate the 3D model of the glycoprotein. Residues 1-26 and 1147-1273 were not included in the 3D structure due to a lack of relevant template structures. Sequences within a rectangle were predicted to consist of one or more HLA antigens using the RankPep server (imed.med.ucm.es/Tools/rankpep 71,72). Glycosites are indicated with asterisks, residues reported to interact with the ACE2 receptor 49 are underlined, and the protease cleavage site is indicated with a triangle above the RS junction. a. The sequence is colored according to antibody accessibility computed for the site-specific glycoform from white to red (least to most accessible). b. Antibody accessibility computed for the non-glycosylated protein. c. The difference in accessibilities between the site-specific and non-glycosylated glycoforms is plotted as the fold change in antigen accessibility during the simulation from &#x2212;4 to 0 (blue to white), where blue indicates glycosylation-dependent surface shielding." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2080772347" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequence of the S protein (NCBI: YP_009724390.1) used to generate the 3D model of the glycoprotein. Residues 1-26 and 1147-1273 were not included in the 3D structure due to a lack of relevant template structures. Sequences within a rectangle were predicted to consist of one or more HLA antigens using the RankPep server (imed.med.ucm.es/Tools/rankpep &lt;sup&gt;71,72&lt;/sup&gt;). Glycosites are indicated with asterisks, residues reported to interact with the ACE2 receptor &lt;sup&gt;49&lt;/sup&gt; are underlined, and the protease cleavage site is indicated with a triangle above the RS junction. &lt;strong&gt;a.&lt;/strong&gt; The sequence is colored according to antibody accessibility computed for the site-specific glycoform from white to red (least to most accessible). &lt;strong&gt;b&lt;/strong&gt;. Antibody accessibility computed for the non-glycosylated protein. &lt;strong&gt;c&lt;/strong&gt;. The difference in accessibilities between the site-specific and non-glycosylated glycoforms is plotted as the fold change in antigen accessibility during the simulation from &#x2212;4 to 0 (blue to white), where blue indicates glycosylation-dependent surface shielding.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F4.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F4.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030445/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-26" class="first-child">Sequence of the S protein (NCBI: YP_009724390.1) used to generate the 3D model of the glycoprotein. Residues 1-26 and 1147-1273 were not included in the 3D structure due to a lack of relevant template structures. Sequences within a rectangle were predicted to consist of one or more HLA antigens using the RankPep server (imed.med.ucm.es/Tools/rankpep <sup><a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">71</a>,<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">72</a></sup>). Glycosites are indicated with asterisks, residues reported to interact with the ACE2 receptor <sup><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a></sup> are underlined, and the protease cleavage site is indicated with a triangle above the RS junction. <strong>a.</strong> The sequence is colored according to antibody accessibility computed for the site-specific glycoform from white to red (least to most accessible). <strong>b</strong>. Antibody accessibility computed for the non-glycosylated protein. <strong>c</strong>. The difference in accessibilities between the site-specific and non-glycosylated glycoforms is plotted as the fold change in antigen accessibility during the simulation from −4 to 0 (blue to white), where blue indicates glycosylation-dependent surface shielding.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">As expected, glycosylation consistently decreased the surface exposure of the residues proximal to the glycosites (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4.c</a>), but also led to non-sequential changes in exposure, as a result of the 3D topology in the vicinity of each glycosite. Of the 18 glycosites in the 3D structure, 16 are predicted to be present in HLA peptides. Although the glycans may occur throughout the HLA sequences (Table S2), in 12 of these sequences the glycans are predicted to be present at the terminus of at least one putative HLA antigen. This observation suggests that these 12 glycosites may not interfere with antigen presentation in an HLA complex. This property is essential for the potential generation of antibodies against the underlying epitopes, but moreover, may lead to antibodies that target these carbohydrates on the S glycoprotein <sup><a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">73</a></sup>. Anti-carbohydrate antibodies have been shown to be neutralizing in other viruses, such as HIV <sup><a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a></sup>, and therefore glycosylated peptides can offer an alternative to more traditional peptide epitopes. From the perspective of vaccine development <sup><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup>, targeting glycans as epitopes would be expected to benefit from matching the glycan microheterogeneity in the vaccine to that in the circulating virus, which requires additional consideration of the choice of cell type for vaccine production.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-28">The present study indicates that glycans shield approximately 40% of the underlying protein surface of the S glycoprotein from antibody recognition, and that this value is relatively insensitive to glycan type. This suggests that although glycan microheterogeneity varies according to host cell type, the efficacy of antisera should not be impacted by such differences. In contrast, by analogy with influenza hemagglutinin <sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></sup>, variations in glycosite location arising from antigenic drift can be expected to have a profound effect on S protein antigenicity and potentially vaccine efficacy. Fortunately, the most accessible and largest antigenic surface in the S protein consists of the ACE2 binding domain, where the virus can’t exploit glycan shielding or mutational changes to evade host immune response without potentially attenuating viral fitness. The requirement that the virus maintain the integrity of the ACE2 RBD suggests that a vaccine that includes this epitope may be effective, as long as the virus continues to target the same host receptor.</p><p id="p-29">Glycan microheterogeneity may impact the innate immune response by altering the ability of collectins and other lectins of the immune system to neutralize the virus, and may impact the adaptive immune response by altering the number of viable HLA antigens. But such heterogeneity has little impact on the antigenic surface area of the S protein. From a vaccine perspective, efficacy may benefit from ensuring that the production method results in glycosylation profiles that match those of the circulating virus, or by engineering a vaccine so as to avoid glycosylated sequences.</p><p id="p-30">Lastly, the observation that homogeneously glycosylated glycoforms are predicted to display approximately the same antigenic properties as those computed for the more relevant site-specific glycoform suggests that such models can be usefully applied in advance of the report of experimental glycomics data. This final conclusion is significant as it enables the effects of glycosite alterations to be estimated in anticipation of antigenic shift or drift.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>SARS-CoV2 spike (S) protein structure</h3><p id="p-31">A 3D structure of the prefusion form of the S protein (RefSeq: YP_009724390.1, UniProt: P0DTC2 SPIKE_SARS2), based on a Cryo-EM structure (PDB code 6VSB) <sup><a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a></sup>, was obtained from the SWISS-MODEL server (swissmodel.expasy.org). The model has 95% coverage (residues 27 to 1146) of the S protein.</p></div><div id="sec-10" class="subsection"><h3>S protein glycoform generation</h3><p id="p-32">Five unique 3D models for the glycosylated glycoprotein were generated using the glycoprotein builder available at GLYCAM-Web (<a href="http://www.glycam.org">www.glycam.org</a>) together with an in-house program that adjusts the asparagine side chain torsion angles and glycosidic linkages within known low-energy ranges <sup><a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a></sup> to relieve any atomic overlaps with the core protein, as described previously <sup><a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a></sup>. The site specific glycans used to model a glycoform representative of the data obtained from the S glycoprotein expressed in HEK293 cells <sup><a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a></sup>, are presented in Table S1.</p></div><div id="sec-11" class="subsection"><h3>Energy minimization and Molecular dynamics (MD) simulations</h3><p id="p-33">Each glycosylated structure was placed in a periodic box of approximately 130,000 TIP3P water molecules <sup><a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a></sup> with a 10 Å buffer between the glycoprotein and the box edge. Energy minimization of all atoms was performed for 20,000 steps (10,000 steepest decent, followed by 10,000 conjugant gradient) under constant pressure (1 atm) and temperature (300 K) (nPT) conditions. All MD simulations were performed under nPT conditions with the CUDA implementation of the PMEMD <sup><a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>,<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup> simulation code, as present in the Amber14 software suite <sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>. The GLYCAM06j force field <sup><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a></sup> and Amber14SB force field <sup><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup> were employed for the carbohydrate and protein moieties, respectively. A Berendsen barostat with a time constant of 1 ps was employed for pressure regulation, while a Langevin thermostat with a collision frequency of 2 ps<sup>−1</sup> was employed for temperature regulation. A nonbonded interaction cut-off of 8 Å was employed. Long-range electrostatics were treated with the particle-mesh Ewald (PME) method <sup><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup>. Covalent bonds involving hydrogen were constrained with the SHAKE algorithm, allowing an integration time step of 2 fs <sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></sup> to be employed. The energy minimized coordinates were equilibrated at 300K over 400 ps with restraints on the solute heavy atoms. Each system was then equilibrated with restraints on the Cα atoms of the protein for 1ns, prior to initiating 3 independent production MD simulations with random starting seeds for a total time of 0.25 μs, with no restraints applied.</p></div></div><div class="section" id="sec-13"><h2 class="">Supporting Information</h2><p id="p-34">Coordinates in pdb format for each glycoform will be made available for download from GLYCAM-Web (<a href="http://www.glycam.org">www.glycam.org</a>).</p></div><div class="section" id="sec-14"><h2 class="">Author Contributions</h2><p id="p-36">O.C.G. and R.J.W. designed the research; O.C.G., D. M., K. I., and R.J.W. performed the research; R.J.W. wrote the paper.</p></div><div class="section COI-statement" id="sec-15"><h2 class="">Competing Interests</h2><p id="p-37">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">R.J.W. thanks the National Institutes of Health (U01 CA207824 and P41 GM103390) for financial support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-5"><a href="http://glycam.org/">http://glycam.org/</a></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">W.H.O</span>. <abbr class="cit-jnl-abbrev">Coronavirus disease 2019 (Covid-19) Situation Report</abbr>. Report No. 77, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.2" data-doi="10.1074/jbc.M112.371898"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Depetris</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Partial enzymatic deglycosylation preserves the structure of cleaved recombinant HIV-1 envelope glycoprotein trimers</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>, <span class="cit-fpage">24239</span>–<span class="cit-lpage">24254</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.371898%26rft_id%253Dinfo%253Apmid%252F22645128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMjkvMjQyMzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDQ0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pereira</span>,  <span class="cit-name-given-names">M. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Glycans as Key Checkpoints of T Cell Activity and Function</span>. <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">2754</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol%26rft.volume%253D9%26rft.spage%253D2754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baum</span>,  <span class="cit-name-given-names">L. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cobb</span>,  <span class="cit-name-given-names">B. A.</span></span> <span class="cit-article-title">The direct and indirect effects of glycans on immune function</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">619</span>–<span class="cit-lpage">624</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft.volume%253D27%26rft.spage%253D619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.5" data-doi="10.1016/j.tim.2007.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vigerust</span>,  <span class="cit-name-given-names">D. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shepherd</span>,  <span class="cit-name-given-names">V. L.</span></span> <span class="cit-article-title">Virus glycosylation: role in virulence and immune interactions</span>. <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">211</span>–<span class="cit-lpage">218</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DVigerust%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D211%26rft.epage%253D218%26rft.atitle%253DVirus%2Bglycosylation%253A%2Brole%2Bin%2Bvirulence%2Band%2Bimmune%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2007.03.003%26rft_id%253Dinfo%253Apmid%252F17398101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2007.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17398101&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246944100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.6" data-doi="10.3389/fimmu.2018.01998"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casals</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanero-Rhodes</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Fojeda</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Solis</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">The Role of Collectins and Galectins in Lung Innate Immune Defense</span>. <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1998</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol%26rft.volume%253D9%26rft.spage%253D1998%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2018.01998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2018.01998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.7" data-doi="10.4049/jimmunol.1201060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutter</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Toward animal cell culture-based influenza vaccine design: viral hemagglutinin N-glycosylation markedly impacts immunogenicity</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-vol">190</span>, <span class="cit-fpage">220</span>–<span class="cit-lpage">230</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1201060%26rft_id%253Dinfo%253Apmid%252F23225881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMjIwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.8" data-doi="10.1126/science.1124513"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure and receptor specificity of the hemagglutinin from an H5N1 influenza virus</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>, <span class="cit-fpage">404</span>–<span class="cit-lpage">410</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1124513%26rft_id%253Dinfo%253Apmid%252F16543414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3Mi80MDQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDQ0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.9" data-doi="10.1371/journal.ppat.1003831"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotter</span>,  <span class="cit-name-given-names">C. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">A single amino acid in the stalk region of the H1N1pdm influenza virus HA protein affects viral fusion, stability and infectivity</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e1003831</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1003831%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003831%26rft_id%253Dinfo%253Apmid%252F24391498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24391498&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.10" data-doi="10.1128/JVI.01023-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single hemagglutinin mutations that alter both antigenicity and receptor binding avidity influence influenza virus antigenic clustering</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">9904</span>–<span class="cit-lpage">9910</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01023-13%26rft_id%253Dinfo%253Apmid%252F23824816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xNy85OTA0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altman</span>,  <span class="cit-name-given-names">M. O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human Influenza A Virus Hemagglutinin Glycan Evolution Follows a Temporal Pattern to a Glycan Limit</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">10</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.12" data-doi="10.1073/pnas.1712377114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zost</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Contemporary H3N2 influenza viruses have a glycosylation site that alters binding of antibodies elicited by egg-adapted vaccine strains</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">12578</span>–<span class="cit-lpage">12583</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1712377114%26rft_id%253Dinfo%253Apmid%252F29109276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQ3LzEyNTc4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.14" data-doi="10.1038/ncomms15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15092</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft_id%253Dinfo%253Apmid%252F28393837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28393837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030445v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crispin</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-source">Site-specific analysis of the SARS-CoV-2 glycan shield</span>. Preprint at <a href="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1">https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.16" data-doi="10.1074/mcp.M116.058016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khatri</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrated Omics and Computational Glycobiology Reveal Structural Basis for Influenza A Virus Glycan Microheterogeneity and Host Interactions</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">1895</span>–<span class="cit-lpage">1912</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M116.058016%26rft_id%253Dinfo%253Apmid%252F26984886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjk6IjE1LzYvMTg5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNDQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">N-Glycosylation of Seasonal Influenza Vaccine Hemagglutinins: Implication for Potency Testing and Immune Processing</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">93</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.18" data-doi="10.1021/pr400329k"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative Glycomics Analysis of Influenza Hemagglutinin (H5N1) Produced in Vaccine Relevant Cell Platforms</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">3707</span>–<span class="cit-lpage">3720</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes.%26rft.volume%253D12%26rft.spage%253D3707%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr400329k%26rft_id%253Dinfo%253Apmid%252F23848607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr400329k&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23848607&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urbanowicz</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antigenicity and Immunogenicity of Differentially Glycosylated Hepatitis C Virus E2 Envelope Proteins Expressed in Mammalian and Insect Cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">93</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hang</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of site-specific N-glycan remodeling in the endoplasmic reticulum and the Golgi</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">25</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.21" data-doi="10.1096/fj.201700403R"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Losfeld</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Influence of protein/glycan interaction on site-specific glycan heterogeneity</span>. <abbr class="cit-jnl-abbrev">FASEB J</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">4623</span>–<span class="cit-lpage">4635</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.201700403R%26rft_id%253Dinfo%253Apmid%252F28679530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1096/fj.201700403R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28679530&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.22" data-doi="10.1126/sciadv.aax8930"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arigoni-Affolter</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mechanistic reconstruction of glycoprotein secretion through monitoring of intracellular N-glycan processing</span>. <abbr class="cit-jnl-abbrev">Sci Adv</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">eaax8930</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DArigoni-Affolter%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253Deaax8930%26rft.epage%253Deaax8930%26rft.atitle%253DMechanistic%2Breconstruction%2Bof%2Bglycoprotein%2Bsecretion%2Bthrough%2Bmonitoring%2Bof%2Bintracellular%2BN-glycan%2Bprocessing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aax8930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiNS8xMS9lYWF4ODkzMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNDQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">York</span>,  <span class="cit-name-given-names">I. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Alymova</span>,  <span class="cit-name-given-names">I. V.</span></span> <span class="cit-article-title">Influenza virus N-linked glycosylation and innate immunity</span>. <abbr class="cit-jnl-abbrev">Biosci Rep</abbr> <span class="cit-vol">39</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.24" data-doi="10.1073/pnas.81.6.1779"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skehel</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A carbohydrate side chain on hemagglutinins of Hong Kong influenza viruses inhibits recognition by a monoclonal antibody</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">1779</span>–<span class="cit-lpage">1783</span> (<span class="cit-pub-date">1984</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSkehel%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D1779%26rft.epage%253D1783%26rft.atitle%253DA%2Bcarbohydrate%2Bside%2Bchain%2Bon%2Bhemagglutinins%2Bof%2BHong%2BKong%2Binfluenza%2Bviruses%2Binhibits%2Brecognition%2Bby%2Ba%2Bmonoclonal%2Bantibody.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.6.1779%26rft_id%253Dinfo%253Apmid%252F6584912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4MS82LzE3NzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDQ0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.25" data-doi="10.1021/acs.jproteome.5b00416"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCullers</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alymova</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parsons</span>,  <span class="cit-name-given-names">L. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cipollo</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-article-title">Glycosylation Analysis of Engineered H3N2 Influenza A Virus Hemagglutinins with Sequentially Added Historically Relevant Glycosylation Sites</span>. <abbr class="cit-jnl-abbrev">J Proteome Res</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">3957</span>–<span class="cit-lpage">3969</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BProteome%2BRes%26rft.volume%253D14%26rft.spage%253D3957%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jproteome.5b00416%26rft_id%253Dinfo%253Apmid%252F26202417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.jproteome.5b00416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26202417&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Homans</span>,  <span class="cit-name-given-names">S. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dwek</span>,  <span class="cit-name-given-names">R. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rademacher</span>,  <span class="cit-name-given-names">T. W.</span></span> <span class="cit-article-title">Solution Conformations of N-Linked Oligosaccharides</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">6571</span>–<span class="cit-lpage">6578</span> (<span class="cit-pub-date">1987</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D26%26rft.spage%253D6571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.27" data-doi="10.1021/bi00368a076"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Homans</span>,  <span class="cit-name-given-names">S. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Conformational Transitions in N-Linked Oligosaccharides</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">6342</span>–<span class="cit-lpage">6350</span> (<span class="cit-pub-date">1986</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D25%26rft.spage%253D6342%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00368a076%26rft_id%253Dinfo%253Apmid%252F3790526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00368a076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3790526&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.28" data-doi="10.1093/glycob/cwv083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Im</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Preferred conformations of N-glycan core pentasaccharide in solution and in glycoproteins</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">19</span>–<span class="cit-lpage">29</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fglycob%252Fcwv083%26rft_id%253Dinfo%253Apmid%252F26405106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/glycob/cwv083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26405106&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harbison</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fadda</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">An atomistic perspective on ADCC quenching by core-fucosylation of IgG1 Fc N-glycans from enhanced sampling molecular dynamics</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amaro</span>,  <span class="cit-name-given-names">R. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W. W.</span></span> <span class="cit-article-title">Molecular-level simulation of pandemic influenza glycoproteins</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">819</span>, <span class="cit-fpage">575</span>–<span class="cit-lpage">594</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DAmaro%26rft.auinit1%253DR.%2BE.%26rft.volume%253D819%26rft.spage%253D575%26rft.epage%253D594%26rft.atitle%253DMolecular-level%2Bsimulation%2Bof%2Bpandemic%2Binfluenza%2Bglycoproteins.%26rft_id%253Dinfo%253Apmid%252F22183559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22183559&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirschner</span>,  <span class="cit-name-given-names">K. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Faller</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Structural analysis of human glycoprotein butyrylcholinesterase using atomistic molecular dynamics: The importance of glycosylation site ASN241</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0187994</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0187994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yanaka</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dynamic Views of the Fc Region of Immunoglobulin G Provided by Experimental and Computational Observations</span>. <abbr class="cit-jnl-abbrev">Antibodies (Basel)</abbr> <span class="cit-vol">8</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.33" data-doi="10.1016/j.chom.2016.11.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Recent H3N2 Viruses Have Evolved Specificity for Extended, Branched Human-type Receptors, Conferring Potential for Increased Avidity</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">34</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D21%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.11.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.34" data-doi="10.1128/JVI.00769-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vigerust</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">N-linked glycosylation attenuates H3N2 influenza viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">8593</span>–<span class="cit-lpage">8600</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00769-07%26rft_id%253Dinfo%253Apmid%252F17553891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xNi84NTkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.35" data-doi="10.1073/pnas.1218841110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolution of the receptor binding properties of the influenza A(H3N2) hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">21474</span>–<span class="cit-lpage">21479</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1218841110%26rft_id%253Dinfo%253Apmid%252F23236176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzUyLzIxNDc0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.36" data-doi="10.1371/journal.pone.0066325"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gulati</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human H3N2 Influenza Viruses Isolated from 1968 To 2012 Show Varying Preference for Receptor Substructures with No Apparent Consequences for Disease or Spread</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e66325</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66325%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066325%26rft_id%253Dinfo%253Apmid%252F23805213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23805213&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.37" data-doi="10.1073/pnas.1809667116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hulswit</span>,  <span class="cit-name-given-names">R. J. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human coronaviruses OC43 and HKU1 bind to 9-O-acetylated sialic acids via a conserved receptor-binding site in spike protein domain A</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">2681</span>–<span class="cit-lpage">2690</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1809667116%26rft_id%253Dinfo%253Apmid%252F30679277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE2LzcvMjY4MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNDQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis for human coronavirus attachment to sialic acid receptors</span>. <abbr class="cit-jnl-abbrev">Nature Structural &amp; Molecular Biology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D26%26rft.spage%253D481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.39" data-doi="10.1073/pnas.85.12.4526"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vlasak</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luytjes</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Palese</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Human and bovine coronaviruses recognize sialic acid-containing receptors similar to those of influenza C viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">4526</span>–<span class="cit-lpage">4529</span> (<span class="cit-pub-date">1988</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DVlasak%26rft.auinit1%253DR.%26rft.volume%253D85%26rft.issue%253D12%26rft.spage%253D4526%26rft.epage%253D4529%26rft.atitle%253DHuman%2Band%2Bbovine%2Bcoronaviruses%2Brecognize%2Bsialic%2Bacid-containing%2Breceptors%2Bsimilar%2Bto%2Bthose%2Bof%2Binfluenza%2BC%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.12.4526%26rft_id%253Dinfo%253Apmid%252F3380803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTIvNDUyNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNDQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.40" data-doi="10.1073/pnas.1712592114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of sialic acid-binding function for the Middle East respiratory syndrome coronavirus spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E8508</span>–<span class="cit-lpage">E8517</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1712592114%26rft_id%253Dinfo%253Apmid%252F28923942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQwL0U4NTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of MERS-CoV spike glycoprotein in complex with sialoside attachment receptors</span>. <abbr class="cit-jnl-abbrev">Nature Structural &amp; Molecular Biology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">1151</span>–<span class="cit-lpage">1157</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D26%26rft.spage%253D1151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.42" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Connell</span>,  <span class="cit-name-given-names">B. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lortat-Jacob</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Human immunodeficiency virus and heparan sulfate: from attachment to entry inhibition</span>. <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-vol">4</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Novel antibody epitopes dominate the antigenicity of spike glycoprotein in SARS-CoV-2 compared to SARS-CoV</span>. <abbr class="cit-jnl-abbrev">Cell Mol Immunol</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.47" data-doi="10.1007/s13238-016-0352-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Electron microscopy studies of the coronavirus ribonucleoprotein complex</span>. <abbr class="cit-jnl-abbrev">Protein Cell</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">219</span>–<span class="cit-lpage">224</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BCell%26rft.volume%253D8%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-016-0352-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13238-016-0352-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48 in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.48" data-doi="10.1073/pnas.1707304114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E7348</span>–<span class="cit-lpage">E7357</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1707304114%26rft_id%253Dinfo%253Apmid%252F28807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM1L0U3MzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49 in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030445v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lan</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-source">Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</span>. Preprint at <a href="https://www.biorxiv.org/content/10.1101/2020.02.19.956235v1">https://www.biorxiv.org/content/10.1101/2020.02.19.956235v1</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50 in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.50" data-doi="10.1074/jbc.M600697200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">281</span>, <span class="cit-fpage">15829</span>–<span class="cit-lpage">15836</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M600697200%26rft_id%253Dinfo%253Apmid%252F16597622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjMvMTU4MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDQ0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.51" data-doi="10.1074/jbc.M603275200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">W. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">281</span>, <span class="cit-fpage">34610</span>–<span class="cit-lpage">34616</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDQ0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.52" data-doi="10.1016/j.jmb.2009.03.042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pak</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">388</span>, <span class="cit-fpage">815</span>–<span class="cit-lpage">823</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPak%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D388%26rft.issue%253D4%26rft.spage%253D815%26rft.epage%253D823%26rft.atitle%253DStructural%2Binsights%2Binto%2Bimmune%2Brecognition%2Bof%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2BS%2Bprotein%2Breceptor%2Bbinding%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.03.042%26rft_id%253Dinfo%253Apmid%252F19324051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.03.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19324051&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266302500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.53" data-doi="10.1038/ncomms9223"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ying</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">8223</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D8223%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9223%26rft_id%253Dinfo%253Apmid%252F26370782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.54" data-doi="10.1038/ncomms8712"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evaluation of candidate vaccine approaches for MERS-CoV</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">7712</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D7712%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms8712%26rft_id%253Dinfo%253Apmid%252F26218507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms8712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26218507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.55" data-doi="10.1038/cr.2015.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1237</span>–<span class="cit-lpage">1249</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D25%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2015.113%26rft_id%253Dinfo%253Apmid%252F26391698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2015.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26391698&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.56" data-doi="10.1093/infdis/jix209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">1807</span>–<span class="cit-lpage">1815</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D215%26rft.spage%253D1807%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjix209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jix209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.57" data-doi="10.1016/j.celrep.2018.06.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural Definition of a Unique Neutralization Epitope on the Receptor-Binding Domain of MERS-CoV Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D24%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2018.06.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2018.06.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">92</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.59" data-doi="10.1038/s41467-019-10897-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural definition of a neutralization epitope on the N-terminal domain of MERS-CoV spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">3068</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D3068%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-10897-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-10897-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030445v1.60" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">1026</span>–<span class="cit-lpage">1039 e1015</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30712865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61 in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural Definition of a Neutralization-Sensitive Epitope on the MERS-CoV S1-NTD</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">3395</span>–<span class="cit-lpage">3405 e3396</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D28%26rft.spage%253D3395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62 in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.62" data-doi="10.1016/0263-7855(96)00018-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Humphrey</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalke</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schulten</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">VMD - Visual Molecular Dynamics</span>. <abbr class="cit-jnl-abbrev">J. Mol. Graphics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bgraphics%26rft.stitle%253DJ%2BMol%2BGraph%26rft.aulast%253DHumphrey%26rft.auinit1%253DW.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D33-8%252C%2B28%26rft.atitle%253DVMD%253A%2Bvisual%2Bmolecular%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0263-7855%252896%252900018-5%26rft_id%253Dinfo%253Apmid%252F8744570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0263-7855(96)00018-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8744570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH51500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63 in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.63" data-doi="10.1126/science.1118398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of a V3-containing HIV-1 gp120 core</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>, <span class="cit-fpage">1025</span>–<span class="cit-lpage">1028</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHuang%26rft.auinit1%253DC.-c.%26rft.volume%253D310%26rft.issue%253D5750%26rft.spage%253D1025%26rft.epage%253D1028%26rft.atitle%253DStructure%2Bof%2Ba%2BV3-Containing%2BHIV-1%2Bgp120%2BCore%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118398%26rft_id%253Dinfo%253Apmid%252F16284180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1MC8xMDI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64 in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.64" data-doi="10.1038/nature05580"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural definition of a conserved neutralization epitope on HIV-1 gp120</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>, <span class="cit-fpage">732</span>–<span class="cit-lpage">737</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhou%26rft.auinit1%253DT.%26rft.volume%253D445%26rft.issue%253D7129%26rft.spage%253D732%26rft.epage%253D737%26rft.atitle%253DStructural%2Bdefinition%2Bof%2Ba%2Bconserved%2Bneutralization%2Bepitope%2Bon%2BHIV-1%2Bgp120.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05580%26rft_id%253Dinfo%253Apmid%252F17301785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17301785&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244205200037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65 in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.65" data-doi="10.1016/S0969-2126(00)00547-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kwong</span>,  <span class="cit-name-given-names">P. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of HIV-1 gp120 envelope glycoproteins from laboratory-adapted and primary isolates</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1329</span>–<span class="cit-lpage">1339</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DKwong%26rft.auinit1%253DP.%2BD.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D1329%26rft.epage%253D1339%26rft.atitle%253DStructures%2Bof%2BHIV-1%2Bgp120%2Benvelope%2Bglycoproteins%2Bfrom%2Blaboratory-adapted%2Band%2Bprimary%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0969-2126%252800%252900547-5%26rft_id%253Dinfo%253Apmid%252F11188697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0969-2126(00)00547-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11188697&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66 in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.66" data-doi="10.1126/science.1213256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pejchal</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Potent and Broad Neutralizing Antibody Recognizes and Penetrates the HIV Glycan Shield</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>, <span class="cit-fpage">1097</span>–<span class="cit-lpage">1103</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1213256%26rft_id%253Dinfo%253Apmid%252F21998254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzQvNjA1OS8xMDk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67 in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.67" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>,  <span class="cit-name-given-names">E. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">UCSF Chimera - A Visualization System for Exploratory Research and Analysis</span>. <abbr class="cit-jnl-abbrev">J. Comp. Chem.</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComp.%2BChem.%26rft.volume%253D25%26rft.spage%253D1605%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.04.07.030445v1.68"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">NACCESS v. 2.1.1</span> (<span class="cit-publ-name">University College London</span>, <span class="cit-publ-loc">London</span>, <span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69 in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.69" data-doi="10.1038/nchembio.1685"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Horiya</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacPherson</span>,  <span class="cit-name-given-names">I. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Krauss</span>,  <span class="cit-name-given-names">I. J.</span></span> <span class="cit-article-title">Recent strategies targeting HIV glycans in vaccine design</span>. <abbr class="cit-jnl-abbrev">Nat. Chem. Biol.</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">990</span>–<span class="cit-lpage">999</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BChem.%2BBiol.%26rft.volume%253D10%26rft.spage%253D990%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.1685%26rft_id%253Dinfo%253Apmid%252F25393493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.1685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25393493&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70 in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.70" data-doi="10.1111/febs.13530"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doores</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">The HIV glycan shield as a target for broadly neutralizing antibodies</span>. <abbr class="cit-jnl-abbrev">FEBS J</abbr> <span class="cit-vol">282</span>, <span class="cit-fpage">4679</span>–<span class="cit-lpage">4691</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJ%26rft.volume%253D282%26rft.spage%253D4679%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ffebs.13530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/febs.13530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71 in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.71" data-doi="10.1007/s00251-004-0709-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reche</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glutting</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reinherz</span>,  <span class="cit-name-given-names">E. L.</span></span> <span class="cit-article-title">Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles</span>. <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">56</span>, <span class="cit-fpage">405</span>–<span class="cit-lpage">419</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DReche%26rft.auinit1%253DP.%2BA.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D405%26rft.epage%253D419%26rft.atitle%253DEnhancement%2Bto%2Bthe%2BRANKPEP%2Bresource%2Bfor%2Bthe%2Bprediction%2Bof%2Bpeptide%2Bbinding%2Bto%2BMHC%2Bmolecules%2Busing%2Bprofiles.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-004-0709-7%26rft_id%253Dinfo%253Apmid%252F15349703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-004-0709-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15349703&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224555500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72 in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.72" data-doi="10.1016/S0198-8859(02)00432-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reche</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glutting</span>,  <span class="cit-name-given-names">J. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reinherz</span>,  <span class="cit-name-given-names">E. L.</span></span> <span class="cit-article-title">Prediction of MHC class I binding peptides using profile motifs</span>. <abbr class="cit-jnl-abbrev">Hum Immunol</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">701</span>–<span class="cit-lpage">709</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bimmunology%26rft.stitle%253DHum%2BImmunol%26rft.aulast%253DReche%26rft.auinit1%253DP.%2BA.%26rft.volume%253D63%26rft.issue%253D9%26rft.spage%253D701%26rft.epage%253D709%26rft.atitle%253DPrediction%2Bof%2BMHC%2Bclass%2BI%2Bbinding%2Bpeptides%2Busing%2Bprofile%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0198-8859%252802%252900432-9%26rft_id%253Dinfo%253Apmid%252F12175724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0198-8859(02)00432-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12175724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177804200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73 in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.73" data-doi="10.1038/nm.2535"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Avci</span>,  <span class="cit-name-given-names">F. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsuji</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kasper</span>,  <span class="cit-name-given-names">D. L.</span></span> <span class="cit-article-title">A mechanism for glycoconjugate vaccine activation of the adaptive immune system and its implications for vaccine design</span>. <abbr class="cit-jnl-abbrev">Nature: Medicine</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">1602</span>–<span class="cit-lpage">1609</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DAvci%26rft.auinit1%253DF.%2BY.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D1602%26rft.epage%253D1609%26rft.atitle%253DA%2Bmechanism%2Bfor%2Bglycoconjugate%2Bvaccine%2Bactivation%2Bof%2Bthe%2Badaptive%2Bimmune%2Bsystem%2Band%2Bits%2Bimplications%2Bfor%2Bvaccine%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.2535%26rft_id%253Dinfo%253Apmid%252F22101769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.2535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74 in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malaker</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification and Characterization of Complex Glycosylated Peptides Presented by the MHC Class II Processing Pathway in Melanoma</span>. <abbr class="cit-jnl-abbrev">J Proteome Res</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">228</span>–<span class="cit-lpage">237</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BProteome%2BRes%26rft.volume%253D16%26rft.spage%253D228%26rft_id%253Dinfo%253Apmid%252F27550523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27550523&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75 in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.75" data-doi="10.1093/glycob/cwv037"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haji-Ghassemi</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blackler</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin Young</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">S. V.</span></span> <span class="cit-article-title">Antibody recognition of carbohydrate epitopes</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">920</span>–<span class="cit-lpage">952</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fglycob%252Fcwv037%26rft_id%253Dinfo%253Apmid%252F26033938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/glycob/cwv037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26033938&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76 in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.76" data-doi="10.1074/mcp.R119.001491"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zaia</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Why Glycosylation Matters in Building a Better Flu Vaccine</span>. <abbr class="cit-jnl-abbrev">Mol Cell Proteomics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">2348</span>–<span class="cit-lpage">2358</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.R119.001491%26rft_id%253Dinfo%253Apmid%252F31604803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77 in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.77" data-doi="10.1128/JVI.00593-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">N-linked glycosylation of the hemagglutinin protein influences virulence and antigenicity of the 1918 pandemic and seasonal H1N1 influenza A viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">8756</span>–<span class="cit-lpage">8766</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00593-13%26rft_id%253Dinfo%253Apmid%252F23740978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xNS84NzU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA0NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78 in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.78" data-doi="10.1002/jcc.23517"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nivedha</span>,  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makeneni</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foley</span>,  <span class="cit-name-given-names">B. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tessier</span>,  <span class="cit-name-given-names">M. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Importance of ligand conformational energies in carbohydrate docking: Sorting the wheat from the chaff</span>. <abbr class="cit-jnl-abbrev">J Comput Chem</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">526</span>–<span class="cit-lpage">539</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BComput%2BChem%26rft.volume%253D35%26rft.spage%253D526%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.23517%26rft_id%253Dinfo%253Apmid%252F24375430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.23517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24375430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79 in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.79" data-doi="10.1093/glycob/cww094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">O. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Gly-Spec: a webtool for predicting glycan specificity by integrating glycan array screening data and 3D structure</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">1027</span>–<span class="cit-lpage">1028</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fglycob%252Fcww094%26rft_id%253Dinfo%253Apmid%252F28120784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/glycob/cww094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28120784&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80 in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.80" data-doi="10.1021/ja00392a016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>,  <span class="cit-name-given-names">W. L.</span></span> <span class="cit-article-title">Quantum and Statistical Mechanical Studies of Liquids. 10. Transferable Intermolecular Potential Functions for Water, Alcohols, and Ethers. Application to Liquid Water</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc.</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">335</span>–<span class="cit-lpage">340</span> (<span class="cit-pub-date">1981</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc.%26rft.volume%253D103%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja00392a016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja00392a016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981KZ44800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81 in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.81" data-doi="10.1021/ct400314y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salomon-Ferrer</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Götz</span>,  <span class="cit-name-given-names">A. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poole</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le Grand</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">R. C.</span></span> <span class="cit-article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">3878</span>–<span class="cit-lpage">3888</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BTheory%2BComput.%26rft.volume%253D9%26rft.spage%253D3878%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct400314y%26rft_id%253Dinfo%253Apmid%252F26592383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct400314y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26592383&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82 in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.82" data-doi="10.1021/ct200909j"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gotz</span>,  <span class="cit-name-given-names">A. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput.</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1542</span>–<span class="cit-lpage">1555</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BChem%2BTheory%2BComput%26rft.aulast%253DGotz%26rft.auinit1%253DA.%2BW.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D1542%26rft.epage%253D1555%26rft.atitle%253DRoutine%2BMicrosecond%2BMolecular%2BDynamics%2BSimulations%2Bwith%2BAMBER%2Bon%2BGPUs.%2B1.%2BGeneralized%2BBorn.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct200909j%26rft_id%253Dinfo%253Apmid%252F22582031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct200909j&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22582031&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83 in text" id="ref-83">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.07.030445v1.83"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">AMBER 14</span> (<span class="cit-publ-name">University of California</span>, <span class="cit-publ-loc">San Francisco</span>, <span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84 in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.84" data-doi="10.1002/jcc.20820"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirschner</span>,  <span class="cit-name-given-names">K. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">GLYCAM06: a generalizable biomolecular force field. Carbohydrates</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem.</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">622</span>–<span class="cit-lpage">655</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DKirschner%26rft.auinit1%253DK.%2BN.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D622%26rft.epage%253D655%26rft.atitle%253DGLYCAM06%253A%2Ba%2Bgeneralizable%2Bbiomolecular%2Bforce%2Bfield.%2BCarbohydrates.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20820%26rft_id%253Dinfo%253Apmid%252F17849372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17849372&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253648500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85 in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">J. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span>. <abbr class="cit-jnl-abbrev">J Chem Theory Comput</abbr> <span class="cit-vol">11</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86 in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.86" data-doi="10.1063/1.464397"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darden</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">York</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Particle mesh Ewald: An N⋅ log (N) method for Ewald sums in large systems</span>. <abbr class="cit-jnl-abbrev">The Journal of chemical physics</abbr> <span class="cit-vol">98</span>, <span class="cit-fpage">10089</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bchemical%2Bphysics%26rft.volume%253D98%26rft.spage%253D10089%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.464397%26rft_id%253Dinfo%253Apmid%252F10.1063%252F1.46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1063/1.464397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10.1063/1.46&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030445.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LG10100091&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87 in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030445v1.87" data-doi="10.1016/0021-9991(77)90098-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ryckaert</span>,  <span class="cit-name-given-names">J.-P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciccotti</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berendsen</span>,  <span class="cit-name-given-names">H. J.</span></span> <span class="cit-article-title">C. Numerical integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes</span>. <abbr class="cit-jnl-abbrev">J. Comput. Phys.</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">341</span> (<span class="cit-pub-date">1977</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BPhys.%26rft.volume%253D23%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0021-9991%252877%252990098-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0021-9991(77)90098-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.07.030445v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.08.029769v1" title="Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.07.029934v1" title="Room-temperature-storable PCR Mixes for SARS-CoV-2 Detection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.07.030445v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.04.07.030445v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/04/09/2020.04.07.030445.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.07.030445v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.07.030445v1" class="active" data-icon-position="" data-hide-link-title="0">3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1237294" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-onvshzOTlU8hlA1Z9pMvly73ZAPM8ystASsmeOogCgA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237294" id="node1237294" data-pisa="biorxiv;2020.04.07.030445v1" data-pisa-master="biorxiv;2020.04.07.030445" data-seqnum="1237294" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030445.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Oliver C.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">David</span> <span class="nlm-surname">Montgomery</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keigo</span> <span class="nlm-surname">Ito</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Woods</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.07.030445; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.07.030445 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;title=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;title=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;text=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;title=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;t=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;title=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;title=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237294" id="node1237294--2" data-pisa="biorxiv;2020.04.07.030445v1" data-pisa-master="biorxiv;2020.04.07.030445" data-seqnum="1237294" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030445.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Oliver C.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">David</span> <span class="nlm-surname">Montgomery</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keigo</span> <span class="nlm-surname">Ito</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Woods</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.07.030445; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.07.030445 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Grant%2C+Oliver+C.&amp;rft.au=Montgomery%2C+David&amp;rft.au=Ito%2C+Keigo&amp;rft.au=Woods%2C+Robert+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1237294/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1237294/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1237294/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1237294/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1237294/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1237294/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1237294/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1237294/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1237294/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1237294/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1237294/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1237294/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;count=horizontal&amp;via=&amp;text=3D%20Models%20of%20glycosylated%20SARS-CoV-2%20spike%20protein%20suggest%20challenges%20and%20opportunities%20for%20vaccine%20development&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030445v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.07.030445v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
