<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Ecol Evol</span><span class="iso-abbrev" title="iso-abbrev">Ecol Evol</span><span class="doi" title="doi">10.1002/(ISSN)2045-7758</span><span class="publisher-id" title="publisher-id">ECE3</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Ecology and Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-7758</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7319105">7319105</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/ece3.6287">10.1002/ece3.6287</a></span><span class="publisher-id" title="publisher-id">ECE36287</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Research</span>: <span class="subject" title="subject">Original Research</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Prior evolution in stochastic versus constant temperatures affects RNA virus evolvability at a thermal extreme</div> <div class="alt-title" title="alt-title">
GLORIA�?SORIA et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gloria�?Soria</span><span class="given-names" tagx="given-names" title="given-names">Andrea</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5401-3988</span><a href="#ece36287-aff-0001"><sup>1</sup></a><a href="#ece36287-curr-0001"><sup>4</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">andrea.gloria-soria@ct.gov</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendiola</span><span class="given-names" tagx="given-names" title="given-names">Sandra Y.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9809-0603</span><a href="#ece36287-aff-0001"><sup>1</sup></a><a href="#ece36287-curr-0002"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morley</span><span class="given-names" tagx="given-names" title="given-names">Valerie J.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6805-7562</span><a href="#ece36287-aff-0001"><sup>1</sup></a><a href="#ece36287-curr-0003"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alto</span><span class="given-names" tagx="given-names" title="given-names">Barry W.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7054-2081</span><a href="#ece36287-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">Paul E.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3490-7498</span><a href="#ece36287-aff-0001"><sup>1</sup></a><a href="#ece36287-aff-0003"><sup>3</sup></a></span></div><span class="citation_author_institution" id="ece36287-aff-0001">[1], <span class="named-content" title="named-content">Department of Ecology and Evolutionary Biology</span><span class="institution" title="institution">Yale University</span><div class="unknown" title="city" tag="city">
New Haven</div><span class="named-content" title="named-content">CT</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece36287-aff-0002">[2], <span class="named-content" title="named-content">Florida Medical Entomology Laboratory</span><span class="institution" title="institution">University of Florida</span><div class="unknown" title="city" tag="city">
Vero Beach</div><span class="named-content" title="named-content">FL</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece36287-aff-0003">[3], <span class="named-content" title="named-content">Program in Microbiology</span><span class="institution" title="institution">Yale School of Medicine</span><div class="unknown" title="city" tag="city">
New Haven</div><span class="named-content" title="named-content">CT</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece36287-curr-0001">[4], <span class="named-content" title="named-content">Department of Environmental Sciences, Center for Vector Biology and Zoonotic Diseases</span><span class="institution" title="institution">The Connecticut Agricultural Experiment Station</span><div class="unknown" title="city" tag="city">
New Haven</div><span class="named-content" title="named-content">CT</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece36287-curr-0002">[5], <span class="named-content" title="named-content">Department of Biology</span><span class="institution" title="institution">Emory University</span><div class="unknown" title="city" tag="city">
Atlanta</div><span class="named-content" title="named-content">GA</span><div class="unknown" title="postal-code" tag="postal-code">
30322</div><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece36287-curr-0003">[6], <span class="named-content" title="named-content">Department of Biology</span><span class="institution" title="institution">Pennsylvania State University</span><div class="unknown" title="city" tag="city">
University Park</div><span class="named-content" title="named-content">PA</span><div class="unknown" title="postal-code" tag="postal-code">
16802</div><span class="country" tagx="country" title="country">USA</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b><br /><br /> Andrea Gloria�?Soria, Department of Environmental Sciences, Center for Vector Biology &amp;amp; Zoonotic Diseases, The Connecticut Agricultural Experiment Station. 123 Huntington St. New Haven, CT 06504, USA.<br /><br /> Email: <span class="email" tagx="email" title="email">andrea.gloria-soria@ct.gov</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-6</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">12</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/ece3.v10.12</span><span class="fpage" tagx="fpage" title="fpage">5440</span><span class="lpage" tagx="lpage" title="lpage">5450</span><span class="history" title="history"><span class="received" title="received">received: 2019-12-11</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-3-26</span><span class="accepted" title="accepted">accepted: 2020-4-01</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="file:ECE3-10-5440.pdf" title="self-uri">file:ECE3-10-5440.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>It is unclear how historical adaptation versus maladaptation in a prior environment affects population evolvability in a novel habitat. Prior work showed that vesicular stomatitis virus (VSV) populations evolved at constant 37°C improved in cellular infection at both 29°C and 37°C; in contrast, those evolved under random changing temperatures between 29°C and 37°C failed to improve. Here, we tested whether prior evolution affected the rate of adaptation at the thermal�?niche edge: 40°C. After 40 virus generations in the new environment, we observed that populations historically evolved at random temperatures showed greater adaptability. Deep sequencing revealed that most of the newly evolved mutations were de novo. Also, two novel evolved mutations in the VSV glycoprotein and replicase genes tended to co�?occur in the populations previously evolved at constant 37°C, whereas this parallelism was not seen in populations with prior random temperature evolution. These results suggest that prior adaptation under constant versus random temperatures constrained the mutation landscape that could improve fitness in the novel 40°C environment, perhaps owing to differing epistatic effects of new mutations entering genetic architectures that earlier diverged. We concluded that RNA viruses maladapted to their previous environment could "leapfrog�? over counterparts of higher fitness, to achieve faster adaptability in a novel environment.</p> </div> <div class="abstract" title="abstract"> <p>It is unclear how historical adaptation versus maladaptation in a prior environment affects population evolvability in a novel habitat. We tested whether prior evolution of vesicular stomatitis virus (VSV) populations at either at constant temperature or under random changing temperatures affected the rate of adaptation at the thermal�?niche edge. Our results suggest that prior adaptation under constant versus random temperatures constrained the mutation landscape that could improve fitness in the novel environment and conclude that RNA viruses maladapted to their previous environment could "leapfrog�? over counterparts of higher fitness, to achieve faster adaptability in a novel environment. </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="graphic_UNKNOWN" position="anchor" id="nlm-graphic-1" orientation="portrait" /> </div> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
adaptation</div> <div class="kwd" title="kwd">
experimental evolution</div> <div class="kwd" title="kwd">
historical contingency</div> <div class="kwd" title="kwd">
molecular evolution</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases </span><span class="institution-id" title="institution-id">10.13039/100000060</span></div> </div><span class="award-id" tagx="award-id" title="award-id">3R01AI091646�?04S1 </span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Yale University </span><span class="institution-id" title="institution-id">10.13039/100005326</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Mellon Mays Undergraduate Fellowship</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Centers for Disease Control and Prevention </span><span class="institution-id" title="institution-id">10.13039/100000030</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01CK000509</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Science Foundation </span><span class="institution-id" title="institution-id">10.13039/100000001</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DGE�?1122492 </span><span class="award-id" tagx="award-id" title="award-id">NSF BEACON Center for Evolution in Action</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Institute for Biospheric Studies, Yale University </span><span class="institution-id" title="institution-id">10.13039/100011492</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Gaylord Donnelly Environmental Postdoctoral Fellow</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="June 2020" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.8.4 mode:remove_FC converted:26.06.2020" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gloria�?Soria</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mendiola</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morley</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alto</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Prior evolution in stochastic versus constant temperatures affects RNA virus evolvability at a thermal extreme</span>. <span class="source" tagx="source" title="source">Ecol Evol</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">5440</span>–<span class="lpage" tagx="lpage" title="lpage">5450</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.6287">10.1002/ece3.6287</a></span></span></p> <div class="fn-group" title="fn-group"> <div class="fn-type-funding" title="funding"> <p><b>Funding information</b></p> <p>This work was supported by: U.S. National Institute of Allergy and Infectious Diseases (NIAID) Research Supplement to Promote Diversity in Health�?Related Research 3R01AI091646�?04S1 to A.G�?S (PI: J. Yin and co�?PI: P.E.T.); U.S. National Science Foundation (NSF) Graduate Research Fellowship Grant DGE�?1122492 to V.J.M.; a Mellon Mays Undergraduate Fellowship to S.Y.M.; the Centers for Disease Control and Prevention Cooperative Agreement Number U01CK000509 (A.G.�?S, PI. T. Andreadis); and P.E.T. affiliate membership in the NSF BEACON Center for Evolution in Action. The contents of this work are solely the responsibility of the authors and do not necessarily represent the official views of the Centers for Disease Control and Prevention or the Department of Health and Human Services. The virus isolates originated from a previous experiment supported by a Gaylord Donnelly Environmental Postdoctoral Fellowship from the Yale Institute for Biospheric Studies to B.W.A.</p> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>It is generally assumed that all biological populations experience changes in their environments, at least occasionally. The frequency of environmental turnover can affect which phenotypic variants within a population are favored by natural selection, thus dictating whether the population is relatively homogeneous versus heterogeneous (polymorphic) for genetic variation underlying these phenotypes. Frequent environmental turnover may present differing targets for selection, whereby the relative fitness of certain phenotypes are favored in some environments, but the relative performances of these variants are disfavored in other environments (Buckling, Kassen, Bell, &amp;amp; Rainey, <a href="#ece36287-bib-0008">2000</a>; Kassen, <a href="#ece36287-bib-0025">2002</a>). In this way, an interaction between the tempo of environmental change and the mutations underlying fitness traits may cause a population to harbor greater versus lesser standing genetic variation. Stochastic environmental changes may be particularly challenging for evolving populations, because mismatches between variants and the selective environments can cause genetic variation to accumulate and yet prove insufficient to fuel adaptation (i.e., inability for one or more variants to experience sustained positive selection across varying environments causes adaptation to "stall out,�? and increases vulnerability to extinction) (Bürger and Lynch, <a href="#ece36287-bib-0066">1995</a>; Donaldson�?Matasci, Lachmann, &amp;amp; Bergstrom, <a href="#ece36287-bib-0013">2008</a>; Lande &amp;amp; Shannon, <a href="#ece36287-bib-0030">1996</a>; Botero, Weissing, Wright, &amp;amp; Rubenstein, <a href="#ece36287-bib-0006">2015</a>). Thus, stochastic (unpredictable) environments can cause genetic variance to increase in a population over time, without mean fitness improvement (maladaptation). In contrast, selection in a constant environment should cause relatively lower standing genetic variation to be present in an adapting population. Beyond the expected difference in standing variation, the fixation of alleles in a population successfully adapting to its environment should cause its genetic architecture to diverge from a population that is maladapted. By definition, the underlying genetic basis of phenotypic traits that improve fitness (i.e., adaptations) constitutes genetic architecture; therefore, populations that share recent common ancestry but are adapted versus maladapted to their environments should necessarily diverge in genetic architecture, as well as differ in standing variation.</p> <p>Despite these expected differences, it is unclear whether previous adaptation versus maladaptation in a selective environment should be consequential for evolvability in a novel habitat where all populations are equally unfit. One possibility is that prior successful adaptation molds genetic architecture, such that the population experiences a relatively greater likelihood of conditionally beneficial mutations in the new environment. That is, previously adapted versus maladapted populations can undergo the same spontaneous mutation rate in the novel environment, but the well�?adapted population could more easily evolve via beneficial mutations that modify existing phenotypic traits to achieve more rapid fitness improvement (Bjorklund, <a href="#ece36287-bib-0005">1996</a>; Crespi, <a href="#ece36287-bib-0011">2000</a>; Kirkpatrick &amp;amp; Barton, <a href="#ece36287-bib-0027">1997</a>). This idea seems especially plausible if the novel environment is constant and an extension of the prior habitat (e.g., selection at a novel elevated temperature that is close to the thermal condition of the prior environment). On the other hand, if greater standing genetic variation increases the likelihood that a population will harbor conditionally beneficial alleles in the novel environment, the prior�?maladapted population may experience a relative advantage in evolvability. Importantly, this idea assumes that accumulation of mutations in the prior environment is proportional to the probability that this variation will be useful for fitness in the novel condition. Although some experimental evolution studies have tested the role of historical contingency in further evolution (e.g., Kryazhimskiy, Rice, Jenison, &amp;amp; Desai, <a href="#ece36287-bib-0029">2014</a>; Meyer et al., <a href="#ece36287-bib-0043">2012</a>; Travisano, Mongold, Bennett, &amp;amp; Lenski, <a href="#ece36287-bib-0054">1995</a>; Zachary, Borland, &amp;amp; Lenski, <a href="#ece36287-bib-0063">2008</a>), the evolutionary fate of adapted versus maladapted populations in a novel environment has rarely been tested.</p> <p>Vesicular stomatitis virus (VSV) is a zoonotic <i>Vesiculovirus</i> of veterinary importance in the family <i>Rhabdoviridae</i>, which infects domesticated cattle, horses, and swine, and whose definitive natural host reservoir remains unclear (Rozo�?Lopez, Drolet, &amp;amp; Londoño�?Renteria, <a href="#ece36287-bib-0053">2018</a>). The virus is transmitted between hosts via insect vectors, such as mosquitoes, sand flies, black flies, and biting midges, as well as through direct contact (Lyles &amp;amp; Rupprecht, <a href="#ece36287-bib-0038">2006</a>; Rozo�?Lopez et al., <a href="#ece36287-bib-0053">2018</a>). In addition, VSV provides a biological model for studying generalities of RNA virus evolution (Elena et al., 2001; Holland et al., <a href="#ece36287-bib-0067">1991</a>; Morley, Sistrom, Usme�?Ciro, <a href="#ece36287-bib-0047">2016</a>; Remold, Rambaut, &amp;amp; Turner, <a href="#ece36287-bib-0051">2008</a>; Turner &amp;amp; Elena, <a href="#ece36287-bib-0055">2000</a>; Turner, Morales, Alto, &amp;amp; Remold, <a href="#ece36287-bib-0068">2010</a>; Williams et al., <a href="#ece36287-bib-0061">2016</a>). VSV has an ~11kb negative�?sense ssRNA genome, encoding 5 proteins: the nucleocapsid (N) protein that tightly encapsidates the genomic RNA, phosphoprotein (P), and large (L) protein which make up the polymerase, glycoprotein (G) involved in cell�?surface binding and infection initiation via membrane fusion, and matrix (M) protein important for virion formation and inhibition of host antiviral gene expression (Rose &amp;amp; Whitt, <a href="#ece36287-bib-0069">2001</a>). In laboratory tissue culture, replication of wild�?type VSV on baby hamster kidney (BHK) cells is roughly equal at temperatures ranging from 28°C to 37°C, with virus populations achieving equivalent sizes in these environments (Alto &amp;amp; Turner, <a href="#ece36287-bib-0001">2010</a>).</p> <p>We used VSV and tissue culture cells as a model to compare and contrast phenotypic and molecular evolution of RNA viruses, when evolved in constant versus fluctuating temperature environments for host�?cell infection (Alto, Wasik, Morales, &amp;amp; Turner, <a href="#ece36287-bib-0002">2013</a>). A single ancestral genotype (clone) of VSV was used to found five replicate lineages in a high temperature treatment that imposed constant selection at 37°C and in a random temperature treatment where the selective environment stochastically changed each passage day, in the interval ranging between 29°C and 37°C. The experiment comprised 25 passages total (i.e., 100 generations of VSV evolution; Alto et al., <a href="#ece36287-bib-0002">2013</a>). Results showed that virus populations passaged at 37°C improved in fitness relative to their ancestor, in both the selective environment and via correlated improvement at the lower edge of the thermal niche, 29°C. In contrast, viruses evolved under stochastic temperatures failed to increase in fitness relative to their ancestor at either temperature extremes, 29°C and 37°C (Alto et al., <a href="#ece36287-bib-0002">2013</a>). For these reasons, here we define the constant 37°C �?evolved virus lineages as adapted to their selective environment, whereas the random temperature�?selected lineages of viruses are maladapted to their prior environment.</p> <p>Here, we used the two sets of viruses generated by Alto et al. (<a href="#ece36287-bib-0002">2013</a>) to test whether prior adaptation versus maladaptation was consequential for the rate of fitness improvement in a novel temperature for host�?cell infection: 40°C. We confirmed that the virus lineages previously passaged under stochastic temperatures harbored relatively greater genetic variation, but that both sets of populations were equally of low fitness in the novel environment. We then created a gene pool (mixture) of variants drawn from populations evolved at constant 37°C and similarly for viruses that previously experienced a random temperature environment. These two gene pools were each used to found five replicate lineages that were passaged for an additional 10 days (40 generations) of virus evolution at 40°C. We determined whether historical contingency (prior adaptation vs. maladaptation) led to differing rates of evolvability in the evolutionary time allowed by the current experiment. In addition, we conducted whole�?genome sequencing of evolved viruses to infer which mutations may be responsible for fitness improvement at 40°C and to discern whether populations showed identical (parallel) versus unique mutational solutions to the common selective problem: improved host�?cell infection at 40°C.</p> </div> <div class="methods" title="methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
METHODS</div> <div class="strainsandcultureconditions" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Strains and culture conditions</div> <p>Baby hamster kidney (BHK) cells were obtained from the laboratory of Esteban Domingo (University of Madrid) and were used as hosts for virus infection. Cells were grown in 6�?well tissue culture plates under Dulbecco's modified Eagle's minimum essential medium (DMEM) with 10% fetal bovine serum and 1% penicillin and streptomycin. Cells were incubated at 37°C, 95% relative humidity, and 5% CO<sub>2</sub> to achieve confluent monolayers of ~1 × 10<sup>6</sup> cells/cm<sup>2</sup>. Viruses were originally derived from the Mudd Summer strain of VSV Indiana serotype.</p> </div> <div class="priorexperimentalevolution" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Prior experimental evolution</div> <p>All viruses in the current study came from a prior evolution experiment (Alto et al., <a href="#ece36287-bib-0002">2013</a>). Briefly, an ancestral clone of VSV was used to found five replicate populations (lineages) in each of two treatments (Figure <a href="#ece36287-fig-0001">1a</a>). The high temperature treatment imposed constant selection at 37°C. The random treatment imposed a randomly chosen temperature on each passage day, within the interval ranging between 29°C and 37°C (i.e., exposure to 1 of 9 possible temperatures each day with median = 33.2°C; see Figure <a href="#ece36287-fig-0001">1</a> of Alto et al., <a href="#ece36287-bib-0002">2013</a>). To initiate each treatment population, a cell monolayer was infected with ancestral VSV at a multiplicity of infection (MOI) of 0.001 viruses per cell. After one hour incubation at 37°C, the infected monolayer was incubated for an additional 23 hr according to the treatment regime; the common 37°C adsorption step avoided potential confounding effects of temperature�?dependent differences in virus attachment. After 24 hr, each population was diluted 10<sup>5</sup>�?fold to begin the subsequent passage. This process was repeated for 25 passages total (i.e., ~100 generations of VSV evolution; Miralles, Moya, &amp;amp; Elena, <a href="#ece36287-bib-0044">2000</a>). At each passage, aliquots of supernatant containing the virus progeny were sampled from each lineage and stored at −80°C for future analysis.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Evolutionary history of the constant (37°C) and random lineages used in this study. (a) Alto et al. (<a href="#ece36287-bib-0002">2013</a>) experimental evolution that yielded the constant and random individual lineages that were combined in the current study to generate two ancestral gene pools. (b) Experimental evolution at 40°C of five VSV lineages founded by each ancestral gene pool</p>   </div> </div> <div class="currentexperimentalevolution—selectioninnovel40°cenvironment" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Current experimental evolution—selection in novel 40°C environment</div> <p>In the current study, two ancestral "gene pools�? were created, using viruses from the Alto et al., (<a href="#ece36287-bib-0002">2013</a>) study (Figure <a href="#ece36287-fig-0001">1b</a>). One ancestral pool contained a mixture of equimolar samples from the five VSV lineages that experienced prior evolution at 37°C treatment; hereafter, this gene pool is referred to as previously selected under Constant temperature. The other ancestral pool was created identically but using the five VSV lineages that experienced random temperature selection. The current experiment used each ancestral pool to initiate five replicate populations (10 populations total; Figure <a href="#ece36287-fig-0001">1b</a>) that were allowed to infect a cell monolayer at MOI = 0.01 plaque�?forming units (PFU) per cell (this MOI was chosen to avoid selection favoring defective�?interfering particles, characteristic of high MOI infections with VSV; Horodyski, Nichol, Spindler, &amp;amp; Holland, <a href="#ece36287-bib-0022">1983</a>). After initial 45�?min incubation at 37°C, the infected monolayers were incubated at 40°C for 23 hr. The next day, each population was diluted 10<sup>4</sup>�?fold to begin the following passage by transferring an aliquot to a fresh BHK monolayer at MOI = 0.01. This process was repeated for a total of 10 passages. At the end of the experiment, aliquots of the supernatant of each virus population were sampled and stored at −80°C for later analysis. We note that mixing samples from the five evolutionary lineages in the 37°C treatment, and similarly combining those from lineages evolved under Random temperatures, ensures that a broad spectrum of mutations arising in the prior treatment conditions are present together in the same ancestral pool in the current study. An alternative design is to preserve treatment�?lineage independence by creating five separate ancestors per treatment group to study adaptation at 40°C. One important difference is that preserving lineage independence should increase the probability that low�?frequency variants in the population are represented in the founding population, potentially affecting adaptability in the new 40°C environment. This possible evolvability effect of pooling lineage samples versus preserving lineage independence is the subject of our future work.</p> </div> <div class="absolutefitness" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Absolute fitness</div> <p>Absolute fitness is defined as the observed mean capacity (titer reported as log10 PFU per mL) for a test virus to produce infectious virus particles when grown on cells at MOI = 0.01, where direct virus–virus interactions within coinfected cells are minimized (Wasik, Muñoz�?Rojas, Okamoto, Miller�?Jensen, &amp;amp; Turner, <a href="#ece36287-bib-0057">2016</a>). Absolute fitness was measured by allowing a test virus to infect BHK cells under the above�?described culture conditions, followed by plaque assays of serially diluted aliquots to estimate titer. Aliquots grown on BHK cells were plated under DMEM with 10% heat�?inactivated fetal bovine serum and solidified with 1% low�?melt agarose and incubated for 24 hr at 37°C. After incubation, agarose was removed and cells were stained with crystal violet to visualize plaques. Each plaque was assumed to have originated from a single infecting virus.</p> </div> <div class="rnaisolation,cdnasynthesis,andgenomeamplification" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
RNA isolation, cDNA synthesis, and genome amplification</div> <p>Genomic viral RNA was extracted from 22 test viruses: 10 lineages from the Alto et al. (<a href="#ece36287-bib-0002">2013</a>) study that were used to establish the two ancestral pools in the current experiment (i.e., each endpoint 37°C and random lineage; Figure <a href="#ece36287-fig-0001">1a</a>); two ancestral pools (constant, random); and 10 endpoint (passage 10) VSV lineages in the current study. RNA isolation was performed via QIAamp Viral RNA Mini Kit (Qiagen) and then subjected to cDNA synthesis using <i>SuperScript II</i> Reverse Transcriptase (Life Technologies) and random hexamer primers. The majority of the VSV genome was amplified via polymerase chain reaction (PCR) as 8 overlapping fragments ranging between 1 and 2.5kb in length, using primers listed in Appendix <a href="#ece36287-sup-0005">S1</a>.</p> </div> <div class="genomesequencingandbioinformatics" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Genome sequencing and bioinformatics</div> <p>For Sanger sequencing of evolved populations, the resulting PCR fragments were purified using a mixture of 1ul of Exonuclease I (<i>Exo1</i>; NEB), 1ul Alkaline phosphatase (NEB), and 1ul of Alkaline phosphatase buffer (NEB) and incubated at 37°C for 20 min followed by 80°C for an additional 20 min. Sequencing was performed via dye termination (Sanger) at the Yale University DNA Analysis Facility at Science Hill using primers listed in Appendix <a href="#ece36287-sup-0005">S1</a>. Sequence processing and genome assembly were performed in Geneious 11.1.4 (<a href="http://www.geneious.com">http://www.geneious.com</a>), and polymorphic sites were confirmed by visual inspection of chromatograms. The resulting sequence represented the population consensus genome. Variable sites within a sample were identified as secondary peaks. However, minority variants were likely missed by this approach, since only high frequency variants were detectable (Remold et al., <a href="#ece36287-bib-0051">2008</a>). Consensus sequences of evolved populations were aligned with each other and with the consensus sequences from the ancestral 37°C and random populations from Alto et al., (<a href="#ece36287-bib-0002">2013</a>).</p> <p>For library preparation and deep sequencing of ancestral populations, Illumina�?sequencing libraries were constructed from the 10 lineages from Alto et al., (<a href="#ece36287-bib-0002">2013</a>) that were used to generate the two ancestral pools. For these samples, each step from reverse transcription through sequencing was completed in duplicate, resulting in two technical replicates of each sample. Amplified genome fragments were purified using the QIAquick PCR Purification Kit (Qiagen), and the eight fragments from each virus population were combined to create an equimolar mixture. Libraries were prepared from the pooled viral amplicons using the Nextera XT Kit (Illumina Inc.) and the Nextera Index Kit (Illumina Inc.). Samples were sequenced via paired�?end, 75bp�?read Illumina HiSeq 2000 (Illumina Inc.) at the Yale Center for Genome Analysis. Quality control of reads was conducted using FastQC (Andrews, <a href="#ece36287-bib-0003">2010</a>) with a minimum quality score of 20 and a minimum length of 30bp.</p> <p>Ion Torrent�?sequencing libraries were constructed from the two ancestral pools (constant, random) and samples from passage 10, for each of the evolved lineages (<i>N</i> = 12). Amplified genomes were purified using SPRI beads (Rohland &amp;amp; Reich, <a href="#ece36287-bib-0052">2012</a>) and sheared to an average size of 400bp using a Covaris E210 fragmentation (Covaris). Viral fragments belonging to the same sample were then combined in equimolar concentrations. Barcoded libraries were prepared from each sample pool with the Ion Plus Fragment Library Kit (Life Technologies). Samples were sequenced at the Yale University DNA Analysis Facility at Science Hill, using an Ion PGM<sup>TM</sup> System (Life Technologies).</p> </div> <div class="genomeassemblyandalignment" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Genome assembly and alignment</div> <div class="illuminasequencing" title="sec"><span class="label" tagx="label" title="label">2.7.1</span><div class="title" tagx="title" title="title">
Illumina sequencing</div> <p>Illumina sequences were mapped to the publicly available VSV genome (GI:9627229) using BWA v0.7.12 (Li &amp;amp; Durbin, <a href="#ece36287-bib-0036">2009</a>) with default parameters. Consensus sequences and minority�?variant tables were generated using the QUASR pipeline (Watson et al., <a href="#ece36287-bib-0059">2013</a>) with a minimum variant frequency cutoff of 0.01. Only polymorphisms detected in both technical sequencing replicates of each sample were retained to reduce false�?positive results arising during the sequencing process.</p> </div> <div class="iontorrentsequencing" title="sec"><span class="label" tagx="label" title="label">2.7.2</span><div class="title" tagx="title" title="title">
Ion Torrent sequencing</div> <p>Quality of the Ion Torrent runs was inspected using FastQC v.0.11.5 (Andrews, <a href="#ece36287-bib-0003">2010</a>). Reads were quality (�?t 27 –l 30) and size�?filtered (30�? 250bp) using the FASTX�?Toolkit v.0.0.14 (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a><i>)</i> and then mapped to the VSV reference genome (GI:9627229) using Bowtie2 v.2.2.9 (Langmead, Trapnell, Pop, &amp;amp; Salzberg, <a href="#ece36287-bib-0031">2009</a>). Average genome coverage was 2123.7X ± 1131.3 (49.2X�?4061.9X), with population B from the random treatment failing to yield reads. Consensus sequences and pileup files were generated with SAMtools v.1.5 (Li and Durbin, <a href="#ece36287-bib-0036">2009</a>; Li et al., <a href="#ece36287-bib-0037">2009</a>) and VCFtools v.0.1.14 (Danecek et al., <a href="#ece36287-bib-0012">2011</a>,). Variable sites were identified using VarScan2 v.2.4.3 (Koboldt et al., <a href="#ece36287-bib-0028">2012</a>); only variants present at frequencies above 1% (0.01) were considered and indels were excluded. Variants at frequencies above 90% were also excluded, since they referred to 107 sites confirmed to have been fixed between the VSV reference genome (GI:9627229) and the ancestral populations from Alto et al., (<a href="#ece36287-bib-0002">2013</a>).</p> <p>Note that while we use two different NGS technologies differing in their manufacturers’ reported error rates (Illumina: 0.001 vs. Ion Torrent: 0.01), Ion Torrent substitution error rates (excluding indels) are an order of magnitude lower (Bragg, Stone, Butler, Hugenholtz, &amp;amp; Tyson, <a href="#ece36287-bib-0007">2013</a>; Huang, Hung, &amp;amp; Wang, <a href="#ece36287-bib-0023">2019</a>) and thus similar to that of Illumina technology. Thus, a minority�?variant filter of &amp;lt;0.01 is appropriate and in agreement with cutoffs suggested by previous reports studying viral quasi�?species (Van den Hoecke, Verhelst, Vuylsteke, &amp;amp; Saelens, <a href="#ece36287-bib-0056">2015</a>).</p> </div> </div> <div class="statisticalanalysis" title="sec"><span class="label" tagx="label" title="label">2.8</span><div class="title" tagx="title" title="title">
Statistical analysis</div> <p>A linear mixed model was used to assess the fixed effect of evolutionary history (historical contingency) and random effect of lineage on viral fitness at the final passage of the experimental evolution at 40°C (PROC MIXED, SAS Institute 2004). One�?way ANOVA and Welch two�?sample <i>t</i> test were used for other comparisons (R v. 3.4.0, R Core Team, <a href="#ece36287-bib-0049">2017</a>; Prism 8 v. 8.0.2, GraphPad Software Inc.).</p> </div> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <div class="differinggeneticvariationinancestralgenepools" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
Differing genetic variation in ancestral gene pools</div> <p>Sanger sequencing data in the previous study (Alto et al., <a href="#ece36287-bib-0002">2013</a>) suggested that greater average genetic variation was harbored in the five populations evolved in the random temperature treatment, relative to the five lineages evolved at 37°C. These sets of populations were used to create the two ancestral gene pools in the current study (Figure <a href="#ece36287-fig-0001">1</a>).</p> <p>Next�?generation sequencing (NGS) offers a more powerful approach than Sanger sequencing, to identify rare alleles present at low frequencies in a population. In the current study, we obtained NGS data for the five populations evolved previously at 37°C and for those evolved under random temperatures, to determine whether they differed in average genetic diversity. Consensus sequences derived from NGS confirmed that there are more variable sites across the VSV genomes of lineages in the random treatment (<i>N</i> = 13), compared with those that experienced the constant 37°C treatment (<i>N</i> = 6) (Files <a href="#ece36287-sup-0001">S1</a>,<a href="#ece36287-sup-0002"> S2</a>). Although the number of minor allele frequency variant sites (MAFV; &amp;gt;1%) was not significantly different between the treatments (t = −0.87959, <i>df</i> = 15.328, <i>p</i> = .3927; File <a href="#ece36287-sup-0003">S3</a>), the frequency spectrum of the MAFVs in the random lineages was shifted toward higher frequencies compared with those from the 37°C treatment, with an average MAFV<sub>Rnd</sub> frequency of 0.04 and a median of 0.05, compared with an average MAFV<sub>37C</sub> frequency of 0.02 and a median of 0.03 (t = −3.0237, <i>df</i> = 664.88, <i>p</i> = .0025; Appendix <a href="#ece36287-sup-0005">S2</a>). We concluded that the lineages evolved previously under random temperatures contained greater average allelic diversity than those evolved at strictly 37°C.</p> <p>We then sought to confirm that the sampling of the sets of evolved populations to create the two ancestral gene pools (Figure <a href="#ece36287-fig-0001">1b</a>) was successful in maintaining the observed difference in mean genetic variation described above. To do so, we generated consensus sequences and conducted variant analysis on NGS data for each of the two ancestral pools created for the current study. Results confirmed that the diversity difference remained after the sets of lineages were combined to create the ancestral pools; the constant ancestral pool showed 71 variable sites, whereas the random ancestral pool presented 100 variable sites (Appendix <a href="#ece36287-sup-0005">S3 </a>and<a href="#ece36287-sup-0005"> S4</a>). We concluded that the random ancestral pool was more genetically variable than the constant ancestral pool.</p> </div> <div class="equivalentfitnessofancestralpoolsinthenovel40°cenvironment" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Equivalent fitness of ancestral pools in the novel 40°C environment</div> <p>Prior to initiation of the current experimental evolution, we used replicated (<i>n</i> = 5) fitness assays to test whether the ancestral pools (constant, random) differed in growth performance in the 40°C environment intended to impose virus selection. Results showed that the constant ancestral pool had a mean absolute fitness of 9.64 ± 0.21 s.d.m. log10 PFU/ml, and the ancestral random pool had a mean absolute fitness of 9.57 ± 0.29 s.d.m. log10 PFU/ml; outlined symbols in Figure <a href="#ece36287-fig-0002">2</a>. Comparison of these data revealed that the two ancestral pools did not statistically differ in mean fitness (t = 0.3345, <i>df</i> = 7.9, <i>p</i> = .74). We concluded that neither ancestral mixture was advantaged in fitness performance in the 40°C selective environment, prior to initiation of the current experimental evolution.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Evolved fitness of VSV lineages after 10 passages at 40°C on BHK cells. Fitness measured via plaque assay as viral growth after 24 hr at 40°C starting with a multiplicity of infection (MOI) of 0.01 particles/cell. Circles are the 37°C lineages and squares the random lineages. Each symbol indicates the mean fitness of a single virus lineage with the 95% CI shown. Fitness of the ancestral strains under the same conditions is indicated by outlined symbols, evolved populations by filled symbols. The random treatment group evolved higher grand mean fitness than the 37°C group (t = −4.8036, <i>df</i> = 4.4, <i>p</i> = .0068)</p>   </div> </div> <div class="fasteradaptationat40°cforvirusespreviouslyevolvedunderrandomtemperatures" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Faster adaptation at 40°C for viruses previously evolved under random temperatures</div> <p>We hypothesized that prior selection at constant 37°C temperature versus at random temperatures ranging between 29°C and 37°C would impact further ability for RNA viruses to adapt to constant elevated temperature of 40°C (i.e., effect of historical contingency). To do so, we used each ancestral gene pool to create five new lineages that underwent 10 passages (40 virus generations) of selection at 40°C (Figure <a href="#ece36287-fig-0001">1b</a>).</p> <p>After 10 experimental passages, we used replicated (<i>n</i> = 3) fitness assays to measure the average growth performance at 40°C of each endpoint population in the two treatment groups (constant, random). Results (Figure <a href="#ece36287-fig-0002">2</a>) yielded mean log10 absolute fitness values for each constant population (37A: 9.73 ± 0.06 s.d.m.; 37B: 9.54 ± 0.104 s.d.m.; 37C: 9.85 ± 0.07 s.d.m.; 37D: 9.51 ± 0.09 s.d.m.; 37E: 9.72 ± 0.03 s.d.m. log10 PFU/ml). We then compared the dataset for each population to the mean fitness of its founding ancestral gene pool (9.64 ± 0.21 s.d.m. log10 PFU/ml). This statistical analysis showed that none of the 5 populations evolved increased fitness at 40°C relative to the common ancestor (File <a href="#ece36287-sup-0004">S4</a>).</p> <p>Similarly, we obtained mean log10 absolute fitness values (Figure <a href="#ece36287-fig-0002">2</a>) for each random population (RndA: 10.51 ± 0.05 s.d.m.; RndB: 10.27 ± 0.135 s.d.m.; RndC: 10.180 ± 0.14 s.d.m.; RndD: 10.19 ± 0.14 s.d.m.; RndE: 10.23 ± 0.14 s.d.m. log10 PFU/ml). We then compared the dataset for each population to the mean fitness of its founding ancestral gene pool (9.57 ± 0.29 s.d.m. log10 PFU/ml). This statistical analysis showed that all five of these populations evolved increased fitness at 40°C relative to their common ancestor (File <a href="#ece36287-sup-0004">S4</a>).</p> <p>To test the main hypothesis that historical contingency should cause viruses evolved under constant 37°C temperature versus random temperatures to generally differ in adaptability at 40°C, we compared the grand mean fitness improvement for the two sets of evolved populations. Results (Figure <a href="#ece36287-fig-0002">2</a>) showed that the grand mean log10 fitness of the random populations was numerically greater than that of the constant lineages: LS means ± SE: 10.28 ± 0.06 versus 9.67 ± 0.06 PFU/ml. Furthermore, a statistical analysis showed that these two values significantly differed (linear mixed model: F<sub>1,8</sub> = 47.21, <i>p</i> = .0001; t = −4.8036, <i>df</i> = 4.4, <i>p</i> = .0068). We concluded from the data and analysis that the prior selection under random temperatures led to an adaptive advantage, on average, in fitness gained at 40°C, relative to viruses evolved under prior constant 37°C selection.</p> </div> <div class="historicalselectionaffectsmolecularevolutionofpopulationsat40°c" title="sec"><span class="label" tagx="label" title="label">3.4</span><div class="title" tagx="title" title="title">
Historical selection affects molecular evolution of populations at 40°C</div> <p>We obtained NGS data (Figure <a href="#ece36287-fig-0003">3</a>) for each of the endpoint populations in the current study, to identify variable sites associated with each treatment group, and to evaluate whether identical (parallel) mutations fixed within and between the two sets of treatment populations.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Novel alleles observed in the 10 evolved lineages, relative to the VSV reference genome (GI:9627229). (a) Random pool and (b) 37°C pool. The vertical lines on the top panel are the relative position of the observed mutations in the VSV genome. Numbers on the bottom panel indicate the position of the mutations. Symbols indicate the nucleotide of the derived allele: filled square (T), filled circle (A), filled triangle (G), filled diamond (C), circle outline (M), triangle outline (R), square outline (S), and diamond outline (Y). Highlighted in black are derived alleles detected in the ancestral pool at &amp;gt;1% frequency; in grey are derived alleles not detected in the pool but present at low frequency (&amp;gt;1%) in any of the original populations from Alto et al. (<a href="#ece36287-bib-0002">2013</a>) that gave rise to the corresponding pool. VSV genes are shown: nucleoprotein (N), phosphoprotein (P), matrix protein (M), glycoprotein (G), and large protein (L). Alleles known to have fixed between the reference genome and the ancestral lineages from Alto et al. (<a href="#ece36287-bib-0002">2013</a>) are not shown</p>   </div> <p>We observed that consensus sequences of the terminal (passage 10; Figure <a href="#ece36287-fig-0003">3</a>) populations showed different amounts of variation across the two treatment groups. Variant analysis on the NGS data confirmed more variable sites (frequency &amp;gt; 1%) (broader mutational spectrum) in the random lineages (<i>N</i> = 123), relative to the 37°C lineages (<i>N</i> = 96), although the difference was not significant (t = 1.88, <i>df</i> = 6.05, <i>p</i> = .1082; Appendix <a href="#ece36287-sup-0005">S5 </a>and<a href="#ece36287-sup-0005"> S6</a>). Interestingly, of the 33 mutations depicted in Figure <a href="#ece36287-fig-0003">3</a>, only 11 were detectable (frequency &amp;gt; 1%) in the respective ancestral gene pools, or detected when inspecting the individual NGS data for the five original lineages that were used to create each of the ancestral gene pools (Figure <a href="#ece36287-fig-0001">1</a> and Figure <a href="#ece36287-fig-0003">3</a>, File <a href="#ece36287-sup-0003">S3</a>, Appendix <a href="#ece36287-sup-0005">S3 </a>and<a href="#ece36287-sup-0005"> S4</a>). These results suggest that the majority of mutations relevant for adaptation to the 40°C environment appeared de novo via spontaneous mutation in the current study, after the populations had entered the new selective environment.</p> <p>In addition, we observed differing patterns of parallel molecular evolution in our study (Figure <a href="#ece36287-fig-0003">3a and 3b</a>). In particular, greater parallelism was observed in the populations that experienced prior selection at 37°C (Figure <a href="#ece36287-fig-0003">3b</a>). All five of these lineages showed an identical fixed amino acid substitution in the VSV large (L) protein gene (A9594G: Tyr to Cys). Four out of five of these populations also showed an identical fixed or polymorphic change in the VSV glycoprotein (G) gene (A4148C) that does not change the amino acid sequence. In contrast, none of the populations previously evolved under random temperatures showed this allele change in G, and three of these lineages showed evidence of the same change or a different one in the allele position in L (Figure <a href="#ece36287-fig-0003">3a</a>). Otherwise, the random A�?E populations showed no detectable evidence for parallel molecular substitutions.</p> </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>A fundamental goal of evolutionary biology is to improve its predictive power as a science, such as accurately predicting genotype–phenotype associations, and the future adaptive trajectories of evolving lineages (Graves &amp;amp; Weinreich, <a href="#ece36287-bib-0016">2017</a>; Lassig, Mustonen, &amp;amp; Walczak, <a href="#ece36287-bib-0032">2017</a>). These aims are ambitious, because there are many ways that the effect of a beneficial allele on individual fitness can change over time, making it difficult to predict its long�?term evolutionary fate. For example, the biotic and abiotic environments in which the allele resides may change over time, presenting new challenges that may alter whether allele effects on gene expression, pleiotropy, and epistasis remain beneficial for individual fitness. Moreover, the fitness effects of a beneficial allele can change when additional mutations at other loci cause additions/deletions in the genome, and its effects can be altered if fitness depends on the relative abundance of other genotypes in the population (frequency dependence). Essentially, these myriad variables indicate that the fitness benefit experienced by an allele or genotype can change across generations and environments (e.g., evolutionary landscapes, Remold, <a href="#ece36287-bib-0050">2012</a>), making the long�?term success of an allele and the persistence of its lineage difficult to predict.</p> <p>Experimental evolution studies offer useful "proving grounds�? to test how biological populations adapt to new environmental challenges, and to gauge whether theory accurately predicts how current performance relates to future evolution (Garland &amp;amp; Rose, <a href="#ece36287-bib-0015">2009</a>). Here, we showed that virus population adaptation and maladaptation in a selective environment were consequential for fitness improvements in a novel environment. Specifically, RNA virus populations evolved in a constant high temperature (37°C) were disadvantaged in evolvability at the thermal niche edge (40°C), compared to their counterparts with a history of evolution in stochastic (random) temperatures. Furthermore, we found that virus populations previously evolved in stochastic temperatures contained greater genetic variation coupled with their greater adaptability than virus populations evolved in constant high temperature (37°C). Our empirical studies support the notion that RNA viruses maladapted to a previous environment (i.e., random populations) may have a fitness advantage in novel environments, in part, through access to conditionally beneficial alleles which may influence their adaptive potential (McBride &amp;amp; Turner, <a href="#ece36287-bib-0041">2008</a>; evolutionary revolutions, Lenormand, Roze, &amp;amp; Rousset, <a href="#ece36287-bib-0034">2009</a>). Theory predicts and empirical observations support the notion that fluctuating thermal environments should enhance tolerance at thermal extremes (Duncan, Fellous, Quillery, &amp;amp; Kaltz, <a href="#ece36287-bib-0014">2011</a>; Levins, <a href="#ece36287-bib-0035">1968</a>; Lynch &amp;amp; Gabriel, <a href="#ece36287-bib-0039">1987</a>). Thus, historical contingency may facilitate preadaptation and the emergence of organisms (pathogens) in novel environments (Arnold, Jackson, Waterfield, &amp;amp; Mansfield, <a href="#ece36287-bib-0004">2007</a>; Ketola et al., <a href="#ece36287-bib-0026">2013</a>; Lee &amp;amp; Gelembiuk, <a href="#ece36287-bib-0033">2008</a>). On the one hand, the repeatability of this outcome suggests it might be helpful for the overall goal of refining our power to make evolutionary predictions: Lineages that perform relatively worse in their current environment can still be potentiated to "leapfrog�? in evolvability over counterpart lineages historically evolved near the new selective target. On the other hand, it is unknown whether the result would be repeatable if the new environment was slightly different, such as adaptation to cellular infection at 38°C or 39°C or adaptation to another type of stressful environment (pH, salinity). For example, populations of <i>Drosophila melanogaster</i> selected for increased desiccation resistance showed correlated responses to resistance to other stressors including heat shock and starvation (Hoffman &amp;amp; Parsons, <a href="#ece36287-bib-0020">1989a</a>,<a href="#ece36287-bib-0021"> b</a>). Other experiments on viral adaptation to novel hosts have tested whether host�?selection regimes result in positive or negative correlated performance in either the original or unselected (novel) hosts, and multiple outcomes have been observed (Cooper &amp;amp; Scott, <a href="#ece36287-bib-0010">2001</a>; Greene et al., <a href="#ece36287-bib-0017">2005</a>; Novella, Hershey, Escarmis, Domingo, &amp;amp; Holland, <a href="#ece36287-bib-0048">1999</a>; Turner &amp;amp; Elena, <a href="#ece36287-bib-0055">2000</a>; Weaver, Brault, Kang, &amp;amp; Holland, <a href="#ece36287-bib-0060">1999</a>; Zárate &amp;amp; Novella, <a href="#ece36287-bib-0064">2004</a>). This dichotomy is precisely the reason why historical contingency and evolvability studies are intriguing and vital areas for future work. If underlying patterns truly exist, it may be plausible (albeit dismaying) to conceive that advancements may only come from increased sheer effort: use more attempts to examine effects of historical contingency on evolvability in empirical studies across various biological systems, and observe whether patterns emerge. Notably, differences in the current study design might have unknown effects on the observed outcome. We created gene pools for the 37°C and random temperature evolved populations and sub�?sampled from each pool to establish the replicate populations challenged to evolve at 40°C; this design emphasized the effect of relatively low (37°C evolved) versus high (random evolved) genetic variation on 40°C adaptation, when initial diversity of replicate populations is sampled from the same source pool. Alternatively, each population from the prior study could have served as an independent replicate in the current experiment, emphasizing the role of low versus high diversity on adaptation, when founding pools differ by a greater extent. Likewise, different outcomes may have resulted from imposed alternative randomly changing temperature regimes each experimental passage, or if the period between temperature switches had been modified. A possible future study could examine these alternative designs, to test whether it changes the observed outcome.</p> <p>Our study capitalized on using the set of RNA viruses in Alto et al., (<a href="#ece36287-bib-0002">2013</a>) as an empirical model to test the general scenario of populations well�?adapted versus maladapted in their current environments, and whether this differing prior history affected rate of fitness improvement under a new challenge. Environmental fluctuations (e.g., random temperatures) can either cause maladaptation or select for adaptive generalism, which may fortuitously increase ability to invade novel environments (Ketola et al., <a href="#ece36287-bib-0026">2013</a>); whereas a static environment (e.g., constant 37°C) tends to select for specialists (Kassen, <a href="#ece36287-bib-0025">2002</a>). Furthermore, the genetic architectures of populations evolved under these scenarios also should differ. For example, a population selected in environments A and B should have a different genetic signature than a population selected in environment A or B alone, attributable to mutations with fitness trade�?offs that are unlikely to fix in a population selected in both environments. Our data are consistent with the idea that populations selected in stochastically fluctuating (random) environments are advantaged at niche limits (Kassen, <a href="#ece36287-bib-0025">2002</a>; Levins, <a href="#ece36287-bib-0035">1968</a>). Also, altered thermal tolerance limits have been observed in <i>E. coli</i> ("stepping stone�? and "sliding niche�? hypotheses, Mongold, Bennett, &amp;amp; Lenski, <a href="#ece36287-bib-0045">1996</a>; Mongold, Bennett, &amp;amp; Lenski, <a href="#ece36287-bib-0046">1999</a>) and <i>Drosophila</i> (Huey, Partridge, &amp;amp; Fowler, <a href="#ece36287-bib-0024">1991</a>; Watson &amp;amp; Hoffmann, <a href="#ece36287-bib-0058">1996</a>) following selection in higher or fluctuating temperatures. Although the current study was not intended to test a specific circumstance in the evolution of natural virus populations, our results could help inform how virus adaptation might proceed in roughly analogous situations.</p> <p>Foremost, it is clear that predictions of RNA virus emergence pose important and difficult challenges in biomedicine and public health, and we believe that the results of our study are both relevant and useful for refining emergence predictions. In particular, certain bat species are known to serve as animal reservoirs of RNA viruses that emerge as zoonotic disease agents in humans and other mammals (Calisher, Childs, Field, Holmes, &amp;amp; Schountz, <a href="#ece36287-bib-0009">2006</a>). Bats in the family <i>Pteropodidae</i> (commonly known as "flying foxes�?) can harbor henipaviruses, such as Hendra virus and Nipah virus, with high case�?fatality rates when these viruses spillover into hosts such as horses and humans. It is unclear whether particular physiological and behavioral traits can best explain why some bat species tend to serve as RNA�?virus reservoirs. The fluctuating body temperatures of bats offer one key possibility: the body temperatures of bats can be very low when daily roosting and during extended hibernation, whereas these temperatures can be greatly elevated when bats are actively flying and foraging for food (Calisher et al., <a href="#ece36287-bib-0009">2006</a>). Because viremias in apparently healthy bats are observed to be unaffected by these temperature extremes, the immunobiology of bats has received attention to explain why these mammals can harbor diverse communities of viruses without succumbing to illness (e.g., Xie et al., <a href="#ece36287-bib-0062">2018</a>; Zhang et al., <a href="#ece36287-bib-0065">2013</a>). Regardless of the explanatory mechanism, our study suggests a related phenomenon that may merit attention. Our observed results support the concept that RNA viruses with greater access to genetic variation enhanced evolvability compared to RNA viruses with less genetic variation when encountering novel environments distinct from the prior habitat. The apparent persistence of RNA virus populations and communities within individual bats might create diverse variant pools that fail to reach high titers (i.e., viremias within bats are constant but do not reach the high loads expected when a virus overwhelms the host immune system). The relatively diverse viruses evolved in stochastic (random) temperatures in the Alto et al., (<a href="#ece36287-bib-0002">2013</a>) study could be viewed as analogous to those within a bat reservoir. By further analogy, the observed evolvability advantage of these viruses in a constant new environment (40°C) might be similar to the success of viruses that spillover from bats into novel hosts, such as humans and horses, with constant and higher body temperatures than those experienced in the bat reservoirs. For this reason, it might be useful to test how the genetic variance of important emerging RNA viruses is affected when cultured in the laboratory under constant versus fluctuating temperatures, and whether this history impacts performance and evolution at novel temperatures seen in endpoint (emergence) hosts.</p> <p>Our observed findings that some previously selected RNA virus populations (random populations) have improved performance at the thermal niche limits may be applicable to another real�?life example relevant to public health. Cold�?adapted genotypes of Influenza A virus may have a selective advantage in terms of increased survival outside of the preferred niche, potentially contributing to additional cases and flu epidemics (Hatta et al., <a href="#ece36287-bib-0018">2007</a>; Hayashi, Wills, Bussey, &amp;amp; Takimoto, <a href="#ece36287-bib-0019">2015</a>; Massin, Werf, &amp;amp; Naffakh, <a href="#ece36287-bib-0040">2001</a>). Virulent H5N1 influenza variants have a replicative advantage and are more tolerant of cooler temperatures in the upper respiratory tract of mammals attributable to an amino acid change in the PB2 protein.</p> <p>The genetic differences that separated the evolved lineages from their ancestral gene pools suggested that genetic architectures and epistasis may have played key roles. The virus populations historically evolved in random temperatures achieved relatively higher fitness in the novel 40°C environment in the time allowed, but these lineages showed few similarities to one another in terms of genetic changes that could account for their highly similar phenotypic improvements. This result clearly indicates that relatively more genetic "solutions�? existed for these populations to meet the adaptation challenge, relative to the obvious parallelism observed among the 37°C �?evolved populations. Moreover, the latter 37°C �?evolved populations did not manage to improve in fitness in the new environment, on average, despite the strong selection to evolve similar—and fewer—solutions to the selection problem. So, prior adaptive success among the 37°C �?evolved populations constrained mutational solutions in the novel temperature environment. Altogether, these data suggest that the prior experiment by Alto et al., (<a href="#ece36287-bib-0002">2013</a>) somehow molded the populations drawn from the two treatments to experience two very different outcomes in the current study.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
CONFLICT OF INTERESTS</div> <p>The authors declare no competing interests.</p> </div> <div class="author'scontributions" title="sec"> <div class="title" tagx="title" title="title">
AUTHOR'S CONTRIBUTIONS</div> <p><b>Andrea Gloria�?Soria:</b> Data curation (lead); formal analysis (lead); investigation (lead); visualization (lead); writing – original draft (lead); writing – review &amp;amp; editing (equal). <b>Sandra Y. Mendiola:</b> Data curation (equal); investigation (equal); writing – review &amp;amp; editing (equal). <b>Valerie J. Morley:</b> Data curation (equal); formal analysis (equal); methodology (equal); writing – review &amp;amp; editing (equal). <b>Barry W. Alto:</b> Formal analysis (equal); methodology (equal); validation (equal); writing – review &amp;amp; editing (equal). <b>Paul E. Turner:</b> Conceptualization (lead); funding acquisition (lead); investigation (equal); methodology (equal); project administration (lead); resources (equal); supervision (equal); writing – original draft (equal); writing – review &amp;amp; editing (equal).</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p>File S1</p>  <div class="media" title="media"><a href="ECE3-10-5440-s001.fasta">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>File S2</p>  <div class="media" title="media"><a href="ECE3-10-5440-s002.fasta">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>File S3</p>  <div class="media" title="media"><a href="ECE3-10-5440-s003.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>File S4</p>  <div class="media" title="media"><a href="ECE3-10-5440-s004.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Appendix S1�?S6</p>  <div class="media" title="media"><a href="ECE3-10-5440-s005.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGEMENTS</div> <p>We thank two anonymous reviewers, I. Roxie and members of the Turner research group for valuable feedback on this study, and E.S.C.P. Williams and P. Mamillapalli for technical assistance.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA AVAILABILITY STATEMENT</div> <p>Tables and consensus sequence are available in the Appendix or as supplementary files S1–S4. Raw Illumina and Ion Torrent sequence reads are available through Dryad <a href="https://doi.org/10.5061/dryad.qf0bs33">https://doi.org/10.5061/dryad.qf0bs33</a>.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="ece36287-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alto</span>, <span class="given-names" tagx="given-names" title="given-names">B. W.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Consequences of host adaptation for performance of vesicular stomatitis virus in novel thermal environments</span>. <span class="source" tagx="source" title="source">Evolutionary Ecology</span>, <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">299</span>–<span class="lpage" tagx="lpage" title="lpage">315</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10682-009-9307-3">10.1007/s10682-009-9307-3</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alto</span>, <span class="given-names" tagx="given-names" title="given-names">B. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wasik</span>, <span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morales</span>, <span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Stochastic temperatures impede RNA virus adaptation</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">969</span>–<span class="lpage" tagx="lpage" title="lpage">979</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/evo.12034">10.1111/evo.12034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23550749">23550749</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andrews</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">FastQC: A quality control tool for high throughput sequence data</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arnold</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jackson</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Waterfield</span>, <span class="given-names" tagx="given-names" title="given-names">N. R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mansfield</span>, <span class="given-names" tagx="given-names" title="given-names">J. W.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of microbial virulence: The benefits of stress</span>. <span class="source" tagx="source" title="source">Trends in Genetics</span>, <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">293</span>–<span class="lpage" tagx="lpage" title="lpage">300</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2007.03.017">10.1016/j.tig.2007.03.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17434232">17434232</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bjorklund</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">The importance of evolutionary constraints in ecological time scales</span>. <span class="source" tagx="source" title="source">Evolutionary Ecology</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">423</span>–<span class="lpage" tagx="lpage" title="lpage">431</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/BF01237727">10.1007/BF01237727</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Botero</span>, <span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weissing</span>, <span class="given-names" tagx="given-names" title="given-names">F. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wright</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rubenstein</span>, <span class="given-names" tagx="given-names" title="given-names">D. R.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary tipping points in the capacity to adapt to environmental change</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">112</span>, <span class="fpage" tagx="fpage" title="fpage">184</span>–<span class="lpage" tagx="lpage" title="lpage">189</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1408589111">10.1073/pnas.1408589111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25422451">25422451</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bragg</span>, <span class="given-names" tagx="given-names" title="given-names">L. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stone</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Butler</span>, <span class="given-names" tagx="given-names" title="given-names">M. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hugenholtz</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tyson</span>, <span class="given-names" tagx="given-names" title="given-names">G. W.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Shining a light on dark sequencing: Characterising errors in Ion Torrent PGM data</span>. <span class="source" tagx="source" title="source">PLoS Computational Biology</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003031</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003031">10.1371/journal.pcbi.1003031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23592973">23592973</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Buckling</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kassen</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bell</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rainey</span>, <span class="given-names" tagx="given-names" title="given-names">P. B.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Disturbance and diversity in experimental microcosms</span>. <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">408</span>, <span class="fpage" tagx="fpage" title="fpage">961</span>–<span class="lpage" tagx="lpage" title="lpage">964</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/35050080">10.1038/35050080</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11140680">11140680</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0066" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bürger</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">Evolution and extinction in a changing environment: A quantitative�?genetic analysis</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">151</span>–<span class="lpage" tagx="lpage" title="lpage">163</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28593664">28593664</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Calisher</span>, <span class="given-names" tagx="given-names" title="given-names">C. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Childs</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Field</span>, <span class="given-names" tagx="given-names" title="given-names">H. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span>, <span class="given-names" tagx="given-names" title="given-names">K. V.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schountz</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Bats: Important reservoir hosts of emerging viruses</span>. <span class="source" tagx="source" title="source">Clinical Microbiology Reviews</span>, <span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">531</span>–<span class="lpage" tagx="lpage" title="lpage">545</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/cmr.00017-06">10.1128/cmr.00017-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16847084">16847084</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cooper</span>, <span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scott</span>, <span class="given-names" tagx="given-names" title="given-names">T. W.</span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Differential evolution of eastern equine encephalitis virus populations in response to host cell type</span>. <span class="source" tagx="source" title="source">Genetics</span>, <span class="volume" tagx="volume" title="volume">157</span>, <span class="fpage" tagx="fpage" title="fpage">1403</span>–<span class="lpage" tagx="lpage" title="lpage">1412</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11290699">11290699</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Crespi</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The evolution of maladaptation</span>. <span class="source" tagx="source" title="source">Heredity</span>, <span class="volume" tagx="volume" title="volume">84</span>, <span class="fpage" tagx="fpage" title="fpage">623</span>–<span class="lpage" tagx="lpage" title="lpage">629</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2540.2000.00746.x">10.1046/j.1365-2540.2000.00746.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10886377">10886377</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Danecek</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Auton</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abecasis</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Albers</span>, <span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Banks</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">DePristo</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Durbin</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The variant call format and VCFtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">15</span>), <span class="fpage" tagx="fpage" title="fpage">2156</span>–<span class="lpage" tagx="lpage" title="lpage">2158</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr330">10.1093/bioinformatics/btr330</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522">21653522</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Donaldson�?Matasci</span>, <span class="given-names" tagx="given-names" title="given-names">M. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lachmann</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bergstrom</span>, <span class="given-names" tagx="given-names" title="given-names">C. T.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Phenotypic diversity as an adaptation to environmental uncertainty</span>. <span class="source" tagx="source" title="source">Evolutionary Ecology Research</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">493</span>–<span class="lpage" tagx="lpage" title="lpage">515</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Duncan</span>, <span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fellous</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quillery</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kaltz</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Adaptation of <i>Paramecium caudatum</i> to variable conditions of temperature stress</span>. <span class="source" tagx="source" title="source">Research in Microbiology</span>, <span class="volume" tagx="volume" title="volume">162</span>, <span class="fpage" tagx="fpage" title="fpage">939</span>–<span class="lpage" tagx="lpage" title="lpage">944</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.resmic.2011.04.012">10.1016/j.resmic.2011.04.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21575715">21575715</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garland</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rose</span>, <span class="given-names" tagx="given-names" title="given-names">M. R.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="source" tagx="source" title="source">Experimental evolution: Concepts, methods and applications of selection experiments</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berkeley, CA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">University of California Press</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Graves</span>, <span class="given-names" tagx="given-names" title="given-names">C. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weinreich</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Variability in fitness effects can preclude selection of the fittest</span>. <span class="source" tagx="source" title="source">Annual Review of Ecology, Evolution, and Systematics</span>, <span class="volume" tagx="volume" title="volume">48</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">399</span>–<span class="lpage" tagx="lpage" title="lpage">417</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ecolsys-110316-022722">10.1146/annurev-ecolsys-110316-022722</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Greene</span>, <span class="given-names" tagx="given-names" title="given-names">I. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deardorff</span>, <span class="given-names" tagx="given-names" title="given-names">E. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Milleron</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Domingo</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weaver</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Effect of alternating passage on adaptation of Sindbis virus to vertebrate and invertebrate cells</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">79</span>, <span class="fpage" tagx="fpage" title="fpage">14253</span>–<span class="lpage" tagx="lpage" title="lpage">14260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.79.22.14253-14260.2005">10.1128/JVI.79.22.14253-14260.2005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16254360">16254360</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hatta</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hatta</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kim</span>, <span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Watanabe</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shinya</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nguyen</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kawaoka</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Growth of H5N1 influenza A viruses in the upper respiratory tracts of mice</span>. <span class="source" tagx="source" title="source">PLoS Path</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">1374</span>–<span class="lpage" tagx="lpage" title="lpage">1379</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030133">10.1371/journal.ppat.0030133</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hayashi</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wills</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bussey</span>, <span class="given-names" tagx="given-names" title="given-names">K. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Takimoto</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of Influenza A virus PB2 residue involved in enhanced polymerase activity and virus growth in mammalian cells at low temperatures</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">15</span>), <span class="fpage" tagx="fpage" title="fpage">8042</span>–<span class="lpage" tagx="lpage" title="lpage">8049</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26018156">26018156</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoffmann</span>, <span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parsons</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span> (<span class="year" tagx="year" title="year">1989a</span>). <span class="mixed-article-title" title="mixed-article-title">An integrated approach to environmental stress tolerance and life�?history variation: Desiccation tolerance in <i>Drosophila</i></span>. <span class="source" tagx="source" title="source">Biological Journal of the Linnaean Society</span>, <span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">117</span>–<span class="lpage" tagx="lpage" title="lpage">136</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1095-8312.1989.tb02098.x">10.1111/j.1095-8312.1989.tb02098.x</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoffmann</span>, <span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parsons</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span> (<span class="year" tagx="year" title="year">1989b</span>). <span class="mixed-article-title" title="mixed-article-title">Selection for increased desiccation resistance in <i>Drosophila melanogaster</i>: Additive genetic control and correlated responses for other stresses</span>. <span class="source" tagx="source" title="source">Genetics</span>, <span class="volume" tagx="volume" title="volume">122</span>, <span class="fpage" tagx="fpage" title="fpage">837</span>–<span class="lpage" tagx="lpage" title="lpage">845</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2503423">2503423</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0067" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holland</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de la Torre</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clarke</span><span class="given-names" tagx="given-names" title="given-names">D. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Duarte</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Quantitation of relative fitness and great adaptability of clonal populations of RNA viruses</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">65</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">2960</span>–<span class="lpage" tagx="lpage" title="lpage">2967</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2033662">2033662</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horodyski</span>, <span class="given-names" tagx="given-names" title="given-names">F. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nichol</span>, <span class="given-names" tagx="given-names" title="given-names">S. T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Spindler</span>, <span class="given-names" tagx="given-names" title="given-names">K. R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holland</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span></span> (<span class="year" tagx="year" title="year">1983</span>). <span class="mixed-article-title" title="mixed-article-title">Properties of DI particle resistant mutants of vesicular stomatitis virus isolated from persistent infections and from undiluted passages</span>. <span class="source" tagx="source" title="source">Cell</span>, <span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">801</span>–<span class="lpage" tagx="lpage" title="lpage">810</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(83)90022-3">10.1016/0092-8674(83)90022-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6307527">6307527</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span>, <span class="given-names" tagx="given-names" title="given-names">S. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hung</span>, <span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Application of deep sequencing methods for inferring viral population diversity</span>. <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">266</span>, <span class="fpage" tagx="fpage" title="fpage">95</span>–<span class="lpage" tagx="lpage" title="lpage">102</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jviromet.2019.01.013">10.1016/j.jviromet.2019.01.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30690049">30690049</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huey</span>, <span class="given-names" tagx="given-names" title="given-names">R. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Partridge</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fowler</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Thermal sensitivity of <i>Drosophila melanogaster</i> responds rapidly to laboratory natural selection</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">751</span>–<span class="lpage" tagx="lpage" title="lpage">756</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1991.tb04343.x">10.1111/j.1558-5646.1991.tb04343.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28568829">28568829</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kassen</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">The experimental evolution of specialists, generalists, and the maintenance of diversity</span>. <span class="source" tagx="source" title="source">Journal of Evolutionary Biology</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">173</span>–<span class="lpage" tagx="lpage" title="lpage">190</span>. <a href="1046/j.1420-9101.2002.00377.x">1046/j.1420�?9101.2002.00377.x</a></span></li> <li tag="ref"><a name="ece36287-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ketola</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mikonranta</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Saarinen</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ӧrmälä</span>, <span class="given-names" tagx="given-names" title="given-names">A.�?M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Friman</span>, <span class="given-names" tagx="given-names" title="given-names">V.�?P.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Laakso</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Fluctuating temperature leads to evolution of thermal generalism and preadaptation to novel environments</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">2936</span>–<span class="lpage" tagx="lpage" title="lpage">2944</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/evo.12148">10.1111/evo.12148</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24094344">24094344</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kirkpatrick</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barton</span>, <span class="given-names" tagx="given-names" title="given-names">N. H.</span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of a species' range</span>. <span class="source" tagx="source" title="source">American Naturalist</span>, <span class="volume" tagx="volume" title="volume">150</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/286054">10.1086/286054</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Koboldt</span>, <span class="given-names" tagx="given-names" title="given-names">D. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larson</span>, <span class="given-names" tagx="given-names" title="given-names">D. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shen</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McLellan</span>, <span class="given-names" tagx="given-names" title="given-names">M. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lin</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilson</span>, <span class="given-names" tagx="given-names" title="given-names">R. K.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing</span>. <span class="source" tagx="source" title="source">Genome Research</span>, <span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">568</span>–<span class="lpage" tagx="lpage" title="lpage">576</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.129684.111">10.1101/gr.129684.111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22300766">22300766</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kryazhimskiy</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rice</span>, <span class="given-names" tagx="given-names" title="given-names">D. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jenison</span>, <span class="given-names" tagx="given-names" title="given-names">E. R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Desai</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Global epistasis makes adaptation predictable despite sequence�?level stochasticity</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">344</span>, <span class="fpage" tagx="fpage" title="fpage">1519</span>–<span class="lpage" tagx="lpage" title="lpage">1522</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1250939">10.1126/science.1250939</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24970088">24970088</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lande</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shannon</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">The role of genetic variation in adaptation and population persistence in a changing environment</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">50</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">434</span>–<span class="lpage" tagx="lpage" title="lpage">437</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1996.tb04504.x">10.1111/j.1558-5646.1996.tb04504.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28568879">28568879</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Langmead</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Trapnell</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pop</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salzberg</span>, <span class="given-names" tagx="given-names" title="given-names">S. L.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Ultrafast and memory�?efficient alignment of short DNA sequences to the human genome</span>. <span class="source" tagx="source" title="source">Genome Biology</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">R25</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174">19261174</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lassig</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mustonen</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walczak</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Predicting Evolution</span>. <span class="source" tagx="source" title="source">Nature Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">77</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28812721">28812721</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span>, <span class="given-names" tagx="given-names" title="given-names">C. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gelembiuk</span>, <span class="given-names" tagx="given-names" title="given-names">G. W.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary origins of invasive populations</span>. <span class="source" tagx="source" title="source">Evolutionary Applications</span>, <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">427</span>–<span class="lpage" tagx="lpage" title="lpage">448</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1752-4571.2008.00039.x">10.1111/j.1752-4571.2008.00039.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25567726">25567726</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenormand</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roze</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rousset</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Stochasticity in evolution</span>. <span class="source" tagx="source" title="source">Trends in Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">157</span>–<span class="lpage" tagx="lpage" title="lpage">165</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19178980">19178980</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Levins</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">1968</span>). <span class="source" tagx="source" title="source">Evolution in changing environments: Some theoretical explorations</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Princeton, NJ</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton University Press</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Durbin</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows�?Wheeler transform</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1754</span>–<span class="lpage" tagx="lpage" title="lpage">1760</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Handsaker</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wysoker</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fennell</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruan</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Homer</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Durbin</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The sequence alignment/map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">2078</span>–<span class="lpage" tagx="lpage" title="lpage">2079</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lyles</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rupprecht</span>, <span class="given-names" tagx="given-names" title="given-names">C. E.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Rhabdoviridae</span> In <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knipe</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span></span> et al. <span class="source" tagx="source" title="source">Fields Virology</span> (pp. <span class="fpage" tagx="fpage" title="fpage">1363</span>–<span class="lpage" tagx="lpage" title="lpage">1408</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Philadelphia, PA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Lippincott, Williams and Wilkins</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lynch</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gabriel</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> (<span class="year" tagx="year" title="year">1987</span>). <span class="mixed-article-title" title="mixed-article-title">Environmental tolerance</span>. <span class="source" tagx="source" title="source">American Naturalist</span>, <span class="volume" tagx="volume" title="volume">129</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>–<span class="lpage" tagx="lpage" title="lpage">303</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Massin</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van der Werf</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Naffakh</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Residue 627 of PB2 is a determinant of cold sensitivity in RNA replication of avian influenza viruses</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">75</span>(<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">5398</span>–<span class="lpage" tagx="lpage" title="lpage">5404</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.75.11.5398-5404.2001">10.1128/jvi.75.11.5398-5404.2001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11333924">11333924</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McBride</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic robustness and adaptability of viruses</span>. <span class="source" tagx="source" title="source">Microbe</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">409</span>–<span class="lpage" tagx="lpage" title="lpage">415</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/microbe.3.409.1">10.1128/microbe.3.409.1</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meyer</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dobias</span>, <span class="given-names" tagx="given-names" title="given-names">D. T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weitz</span>, <span class="given-names" tagx="given-names" title="given-names">J. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barrick</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quick</span>, <span class="given-names" tagx="given-names" title="given-names">R. T.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenski</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Repeatability and contingency in the evolution of a key innovation in phage Lambda</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">335</span>, <span class="fpage" tagx="fpage" title="fpage">428</span>–<span class="lpage" tagx="lpage" title="lpage">432</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1214449">10.1126/science.1214449</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22282803">22282803</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miralles</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moya</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elena</span>, <span class="given-names" tagx="given-names" title="given-names">S. F.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Diminishing returns of population size in the rate of RNA virus adaptation</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">74</span>, <span class="fpage" tagx="fpage" title="fpage">3566</span>–<span class="lpage" tagx="lpage" title="lpage">3571</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.74.8.3566-3571.2000">10.1128/JVI.74.8.3566-3571.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10729131">10729131</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mongold</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bennett</span>, <span class="given-names" tagx="given-names" title="given-names">A. F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenski</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary adaptation to temperature. IV. Adaptation of <i>Escherichia coli</i> at a niche boundary</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">35</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28568880">28568880</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mongold</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bennett</span>, <span class="given-names" tagx="given-names" title="given-names">A. F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenski</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary adaptation to temperature. VII. Extension of the upper thermal limit of <i>Escherichia coli</i></span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">386</span>–<span class="lpage" tagx="lpage" title="lpage">394</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1999.tb03774.x">10.1111/j.1558-5646.1999.tb03774.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28565406">28565406</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morley</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sistrom</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Usme�?Ciro</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Remold</span>, <span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution in spatially mixed host environments increases divergence for evolved fitness and intrapopulation genetic diversity in RNA viruses</span>. <span class="source" tagx="source" title="source">Virus Evolution</span>, <span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">vev022</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev022">10.1093/ve/vev022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774292">27774292</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Novella</span>, <span class="given-names" tagx="given-names" title="given-names">I. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hershey</span>, <span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Escarmis</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Domingo</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holland</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Lack of evolutionary stasis during alternating replication of an arbovirus in insect and mammalian cells</span>. <span class="source" tagx="source" title="source">Journal of Molecular Biology</span>, <span class="volume" tagx="volume" title="volume">287</span>, <span class="fpage" tagx="fpage" title="fpage">459</span>–<span class="lpage" tagx="lpage" title="lpage">465</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/jmbi.1999.2635">10.1006/jmbi.1999.2635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10092452">10092452</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R Core Team</span> . (<span class="year" tagx="year" title="year">2017</span>). <span class="source" tagx="source" title="source">R: A language and environment for statistical computing</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span> Retrieved from <a href="https://www.R-project.org">https://www.R�?project.org</a>.</span></li> <li tag="ref"><a name="ece36287-bib-0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Remold</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Understanding specialism when the jack of all trades can be the master of all</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society B</span>, <span class="volume" tagx="volume" title="volume">279</span>, <span class="fpage" tagx="fpage" title="fpage">4861</span>–<span class="lpage" tagx="lpage" title="lpage">4869</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2012.1990">10.1098/rspb.2012.1990</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23097515">23097515</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Remold</span>, <span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rambaut</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary genomics of host adaptation in vesicular stomatitis virus</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1138</span>–<span class="lpage" tagx="lpage" title="lpage">1147</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msn059">10.1093/molbev/msn059</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18353798">18353798</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rohland</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Reich</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Cost�?effective, high�?throughput DNA sequencing libraries for multiplexed target capture</span>. <span class="source" tagx="source" title="source">Genome Research</span>, <span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">939</span>–<span class="lpage" tagx="lpage" title="lpage">946</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.128124.111">10.1101/gr.128124.111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22267522">22267522</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0069" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rose</span>, <span class="given-names" tagx="given-names" title="given-names">J. K.</span></span> &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Whitt</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Rhabdoviridae: The viruses and their replication</span> In <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fields</span><span class="given-names" tagx="given-names" title="given-names">B. N.</span></span></span> et al. <span class="source" tagx="source" title="source">Fields Virology</span> (pp. <span class="fpage" tagx="fpage" title="fpage">1221</span>–<span class="lpage" tagx="lpage" title="lpage">1244</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Philadelphia, PA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Lippincott, Williams and Wilkins</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rozo�?Lopez</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drolet</span>, <span class="given-names" tagx="given-names" title="given-names">B. S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Londoño�?Renteria</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Vesicular stomatitis transmission: A comparison of incriminated vectors</span>. <span class="source" tagx="source" title="source">Insects</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">190</span>.</span></li> <li tag="ref"><a name="ece36287-bib-0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Travisano</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mongold</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bennett</span>, <span class="given-names" tagx="given-names" title="given-names">A. F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenski</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Experimental tests of the roles of adaptation, chance, and history in evolution</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">267</span>, <span class="fpage" tagx="fpage" title="fpage">87</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.7809610">10.1126/science.7809610</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7809610">7809610</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elena</span>, <span class="given-names" tagx="given-names" title="given-names">S. F.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Cost of host radiation in an RNA virus</span>. <span class="source" tagx="source" title="source">Genetics</span>, <span class="volume" tagx="volume" title="volume">156</span>, <span class="fpage" tagx="fpage" title="fpage">1465</span>–<span class="lpage" tagx="lpage" title="lpage">1470</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11102349">11102349</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0068" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">P. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morales</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alto</span><span class="given-names" tagx="given-names" title="given-names">B. W.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Remold</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>. (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Role of evolved host breadth in the initial emergence of an RNA virus</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">64</span>(<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">3273</span>–<span class="lpage" tagx="lpage" title="lpage">3286</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20633045">20633045</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van den Hoecke</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Verhelst</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vuylsteke</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Saelens</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of the genetic diversity of influenza A viruses using next�?generation DNA sequencing</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">79</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-015-1284-z">10.1186/s12864-015-1284-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25758772">25758772</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wasik</span>, <span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Muñoz�?Rojas</span>, <span class="given-names" tagx="given-names" title="given-names">A. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Okamoto</span>, <span class="given-names" tagx="given-names" title="given-names">K. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miller�?Jensen</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Generalized selection to overcome innate immunity selects for host breadth in an RNA virus</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">70</span>, <span class="fpage" tagx="fpage" title="fpage">270</span>–<span class="lpage" tagx="lpage" title="lpage">281</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/evo.12845">10.1111/evo.12845</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26882316">26882316</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Watson</span>, <span class="given-names" tagx="given-names" title="given-names">M. J. O.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoffmann</span>, <span class="given-names" tagx="given-names" title="given-names">A. A.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Acclimation, cross�?generation effects, and the response to selection for increased cold resistance in <i>Drosophila</i></span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">1182</span>–<span class="lpage" tagx="lpage" title="lpage">1192</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1996.tb02359.x">10.1111/j.1558-5646.1996.tb02359.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28565271">28565271</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Watson</span>, <span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Welkers</span>, <span class="given-names" tagx="given-names" title="given-names">M. R. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Depledge</span>, <span class="given-names" tagx="given-names" title="given-names">D. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coulter</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Breuer</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de Jong</span>, <span class="given-names" tagx="given-names" title="given-names">M. D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kellam</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Viral population analysis and minority�?variant detection using short read next�?generation sequencing</span>. <span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B: Biological Sciences</span>, <span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">1614</span>), <span class="fpage" tagx="fpage" title="fpage">20120205</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2012.0205">10.1098/rstb.2012.0205</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weaver</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brault</span>, <span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kang</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holland</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic and fitness changes accompanying adaptation of an arbovirus to vertebrate and invertebrate cells</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">4316</span>–<span class="lpage" tagx="lpage" title="lpage">4326</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.73.5.4316-4326.1999">10.1128/JVI.73.5.4316-4326.1999</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10196330">10196330</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0061" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Williams</span>, <span class="given-names" tagx="given-names" title="given-names">E. S. C. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morales</span>, <span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wasik</span>, <span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brusic</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Whelan</span>, <span class="given-names" tagx="given-names" title="given-names">S. P. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">P. E.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Repeatable population dynamics among vesicular stomatitis virus lineages evolved under high co�?infection</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">370</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.00370">10.3389/fmicb.2016.00370</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27065953">27065953</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0062" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xie</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shen</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goh</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cui</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Dampened STING�?dependent interferon activation in bats</span>. <span class="source" tagx="source" title="source">Cell Host and Microbe</span>, <span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">297</span>–<span class="lpage" tagx="lpage" title="lpage">301</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2018.01.006">10.1016/j.chom.2018.01.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29478775">29478775</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0063" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zachary</span>, <span class="given-names" tagx="given-names" title="given-names">D. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Borland</span>, <span class="given-names" tagx="given-names" title="given-names">C. Z.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lenski</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Historical contingency and the evolution of a key innovation in an experimental population of <i>Escherichia coli</i></span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">105</span>(<span class="issue" tagx="issue" title="issue">23</span>), <span class="fpage" tagx="fpage" title="fpage">7899</span>–<span class="lpage" tagx="lpage" title="lpage">7906</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18524956">18524956</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0064" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zárate</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Novella</span>, <span class="given-names" tagx="given-names" title="given-names">I. S.</span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Vesicular stomatitis virus evolution during alternation between persistent infection in insect cells and acute infection in mammalian cells is dominated by the persistence phase</span>. <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">78</span>, <span class="fpage" tagx="fpage" title="fpage">12236</span>–<span class="lpage" tagx="lpage" title="lpage">12242</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.78.22.12236-12242.2004">10.1128/JVI.78.22.12236-12242.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15507610">15507610</a></span></span></li> <li tag="ref"><a name="ece36287-bib-0065" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cowled</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shi</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bishop�?Lilly</span>, <span class="given-names" tagx="given-names" title="given-names">K. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fang</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, … <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">339</span>(<span class="issue" tagx="issue" title="issue">6118</span>), <span class="fpage" tagx="fpage" title="fpage">456</span>–<span class="lpage" tagx="lpage" title="lpage">460</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1230835">10.1126/science.1230835</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258410">23258410</a></span></span></li> </ul> </div> </div>  </body></html>