<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/23/2020.03.21.001933.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.21.001933v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability" />
<meta name="DC.Identifier" content="10.1101/2020.03.21.001933" />
<meta name="DC.Date" content="2020-03-23" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells." />
<meta name="DC.Contributor" content="Vinicio Armijos-Jaramillo" />
<meta name="DC.Contributor" content="Justin Yeager" />
<meta name="DC.Contributor" content="Claire Muslin" />
<meta name="DC.Contributor" content="Yunierkis Perez-Castillo" />
<meta name="article:published_time" content="2020-03-23" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.21.001933v1" />
<meta name="citation_id" content="2020.03.21.001933v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.21.001933" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.21.001933" />
<meta name="citation_author" content="Vinicio Armijos-Jaramillo" />
<meta name="citation_author_institution" content="Grupo de Bio-Quimioinformática, Carrera de Ingeniería en Biotecnología, Facultad de Ingeniería y Ciencias Agropecuarias, Universidad de Las Américas" />
<meta name="citation_author_email" content="vinicio.armijos@udla.edu.ec" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2965-2515" />
<meta name="citation_author" content="Justin Yeager" />
<meta name="citation_author_institution" content="Biodiversidad Medio Ambiente y Salud (BIOMAS), Dirección General de Investigación, Universidad de Las Américas" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8692-6311" />
<meta name="citation_author" content="Claire Muslin" />
<meta name="citation_author_institution" content="One Health Research Group, Faculty of Health Sciences, Universidad de Las Américas" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2746-6914" />
<meta name="citation_author" content="Yunierkis Perez-Castillo" />
<meta name="citation_author_institution" content="Grupo de Bio-Quimioinformática y Escuela de Ciencias Físicas y Matemáticas, Universidad de Las Américas" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3710-0035" />
<meta name="citation_reference" content="Altschul, S. F., Madden, T. L., Schäffer, A. A., Zhang, J., Zhang, Z., Miller, W., &amp; Lipman, D. J. (1997). Gapped BLAST and PSI-BLAST: A new generation of protein database search programs. Nucleic Acids Research, 25(17), 3389–3402." />
<meta name="citation_reference" content="Benvenuto, D., Giovanetti, M., Ciccozzi, A., Spoto, S., Angeletti, S., &amp; Ciccozzi, M. (2020). The 2019-new coronavirus epidemic: Evidence for virus evolution. Journal of Medical Virology, 92(4), 455–459. https://doi.org/10.1002/jmv.25688" />
<meta name="citation_reference" content="Berman, H. M., Westbrook, J., Feng, Z., Gilliland, G., Bhat, T. N., Weissig, H., Shindyalov, I. N., &amp; Bourne, P. E. (2000). The Protein Data Bank. Nucleic Acids Research, 28(1), 235–242. https://doi.org/10.1093/nar/28.1.235" />
<meta name="citation_reference" content="Case, D. A., Ben-Shalom, I. Y., Brozell, S. R., Cerutti, D. S., Cheatham, T. E., Cruzeiro, V. W. D., Darden, T. A., Duke, R. E., Ghoreishi, D., Gilson, M. K., Gohlke, H., Goetz, A. W., Greene, D., Harris, R., Homeyer, N., Izadi, S., Kovalenko, A., Kurtzman, T., Lee, T. S., … Kollman, P. A. (2018). AMBER 2018." />
<meta name="citation_reference" content="Cui, J., Li, F., &amp; Shi, Z.-L. (2019). Origin and evolution of pathogenic coronaviruses. Nature Reviews. Microbiology, 17(3), 181–192. https://doi.org/10.1038/s41579-018-0118-9" />
<meta name="citation_reference" content="Darling, A. C. E., Mau, B., Blattner, F. R., &amp; Perna, N. T. (2004). Mauve: Multiple alignment of conserved genomic sequence with rearrangements. Genome Research, 14(7), 1394–1403. https://doi.org/10.1101/gr.2289704" />
<meta name="citation_reference" content="Drosten, C., Günther, S., Preiser, W., van der Werf, S., Brodt, H.-R., Becker, S., Rabenau, H., Panning, M., Kolesnikova, L., Fouchier, R. A. M., Berger, A., Burguière, A.-M., Cinatl, J., Eickmann, M., Escriou, N., Grywna, K., Kramme, S., Manuguerra, J.-C., Müller, S., … Doerr, H. W. (2003). Identification of a novel coronavirus in patients with severe acute respiratory syndrome. The New England Journal of Medicine, 348(20), 1967–1976. https://doi.org/10.1056/NEJMoa030747" />
<meta name="citation_reference" content="Fehr, A. R., Channappanavar, R., &amp; Perlman, S. (2017). Middle East Respiratory Syndrome: Emergence of a Pathogenic Human Coronavirus. Annual Review of Medicine, 68(1), 387–399. https://doi.org/10.1146/annurev-med-051215-031152" />
<meta name="citation_reference" content="Gallagher, T. M., &amp; Buchmeier, M. J. (2001). Coronavirus spike proteins in viral entry and pathogenesis. Virology, 279(2), 371–374. https://doi.org/10.1006/viro.2000.0757" />
<meta name="citation_reference" content="Guindon, S., Dufayard, J.-F., Lefort, V., Anisimova, M., Hordijk, W., &amp; Gascuel, O. (2010). New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0. Systematic biology, 59(3), 307–321. https://doi.org/10.1093/sysbio/syq010" />
<meta name="citation_reference" content="Hoffmann, M., Kleine-Weber, H., Schroeder, S., Krüger, N., Herrler, T., Erichsen, S., Schiergens, T. S., Herrler, G., Wu, N.-H., Nitsche, A., Müller, M. A., Drosten, C., &amp; Pöhlmann, S. (2020). SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell, 181, 1–10. https://doi.org/10.1016/j.cell.2020.02.052" />
<meta name="citation_reference" content="Hu, B., Zeng, L.-P., Yang, X.-L., Ge, X.-Y., Zhang, W., Li, B., Xie, J.-Z., Shen, X.-R., Zhang, Y.-Z., Wang, N., Luo, D.-S., Zheng, X.-S., Wang, M.-N., Daszak, P., Wang, L.-F., Cui, J., &amp; Shi, Z.-L. (2017). Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLOS Pathogens, 13(11), e1006698. https://doi.org/10.1371/journal.ppat.1006698" />
<meta name="citation_reference" content="Huang, C., Wang, Y., Li, X., Ren, L., Zhao, J., Hu, Y., Zhang, L., Fan, G., Xu, J., Gu, X., Cheng, Z., Yu, T., Xia, J., Wei, Y., Wu, W., Xie, X., Yin, W., Li, H., Liu, M., … Cao, B. (2020). Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet (London, England), 395(10223), 497–506. https://doi.org/10.1016/S0140-6736(20)30183-5" />
<meta name="citation_reference" content="Huerta-Cepas, J., Dopazo, J., &amp; Gabaldón, T. (2010). ETE: A python Environment for Tree Exploration. BMC Bioinformatics, 11, 24. https://doi.org/10.1186/1471-2105-11-24" />
<meta name="citation_reference" content="Hughes, A. L., &amp; Hughes, M. A. K. (2007). More effective purifying selection on RNA viruses than in DNA viruses. Gene, 404(1-2), 117–125. https://doi.org/10.1016/j.gene.2007.09.013" />
<meta name="citation_reference" content="Katoh, K., &amp; Standley, D. M. (2013). MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Molecular Biology and Evolution, 30(4), 772–780. https://doi.org/10.1093/molbev/mst010" />
<meta name="citation_reference" content="Kearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., Buxton, S., Cooper, A., Markowitz, S., Duran, C., Thierer, T., Ashton, B., Meintjes, P., &amp; Drummond, A. (2012). Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics (Oxford, England), 28(12), 1647–1649. https://doi.org/10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="Kirchdoerfer, R. N., Cottrell, C. A., Wang, N., Pallesen, J., Yassine, H. M., Turner, H. L., Corbett, K. S., Graham, B. S., McLellan, J. S., &amp; Ward, A. B. (2016). Pre-fusion structure of a human coronavirus spike protein. Nature, 531(7592), 118–121. https://doi.org/10.1038/nature17200" />
<meta name="citation_reference" content="Kosakovsky Pond, S. L., Poon, A. F. Y., Velazquez, R., Weaver, S., Hepler, N. L., Murrell, B., Shank, S. D., Magalis, B. R., Bouvier, D., Nekrutenko, A., Wisotsky, S., Spielman, S. J., Frost, S. D. W., &amp; Muse, S. V. (2020). HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies. Molecular Biology and Evolution, 37(1), 295–299. https://doi.org/10.1093/molbev/msz197" />
<meta name="citation_reference" content="Lam, T. T.-Y., Shum, M. H.-H., Zhu, H.-C., Tong, Y.-G., Ni, X.-B., Liao, Y.-S., Wei, W., Cheung, W. Y.-M., Li, W.-J., Li, L.-F., Leung, G. M., Holmes, E. C., Hu, Y.-L., &amp; Guan, Y. (2020). Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China. BioRxiv, 2020.02.13.945485. https://doi.org/10.1101/2020.02.13.945485" />
<meta name="citation_reference" content="Li, W., Wong, S.-K., Li, F., Kuhn, J. H., Huang, I.-C., Choe, H., &amp; Farzan, M. (2006). Animal Origins of the Severe Acute Respiratory Syndrome Coronavirus: Insight from ACE2-S-Protein Interactions. Journal of Virology, 80(9), 4211–4219. https://doi.org/10.1128/jvi.80.9.4211-4219.2006" />
<meta name="citation_reference" content="Li, Wenhui, Moore, M. J., Vasllieva, N., Sui, J., Wong, S. K., Berne, M. A., Somasundaran, M., Sullivan, J. L., Luzuriaga, K., Greeneugh, T. C., Choe, H., &amp; Farzan, M. (2003). Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature, 426(6965), 450–454. https://doi.org/10.1038/nature02145" />
<meta name="citation_reference" content="Li, Wenhui, Zhang, C., Sui, J., Kuhn, J. H., Moore, M. J., Luo, S., Wong, S.-K., Huang, I.-C., Xu, K., Vasilieva, N., Murakami, A., He, Y., Marasco, W. A., Guan, Y., Choe, H., &amp; Farzan, M. (2005). Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2. The EMBO journal, 24(8), 1634–1643. https://doi.org/10.1038/sj.emboj.7600640" />
<meta name="citation_reference" content="McBee, R. M., Rozmiarek, S. A., Meyerson, N. R., Rowley, P. A., &amp; Sawyer, S. L. (2015). The Effect of Species Representation on the Detection of Positive Selection in Primate Gene Data Sets. Molecular Biology and Evolution, 32(4), 1091–1096. https://doi.org/10.1093/molbev/msu399" />
<meta name="citation_reference" content="Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., &amp; Ferrin, T. E. (2004). UCSF Chimera—A visualization system for exploratory research and analysis. Journal of Computational Chemistry, 25(13), 1605–1612. https://doi.org/10.1002/jcc.20084" />
<meta name="citation_reference" content="Shang, J., Ye, G., Shi, K., Wan, Y., Luo, C., Aihara, H., Geng, Q., Auerbach, A., &amp; Li, F. (2020). Structural basis for receptor recognition by the novel coronavirus from Wuhan. Research Square Pre-Print. https://doi.org/10.21203/rs.2.24749/v1" />
<meta name="citation_reference" content="Tang, X., Wu, C., Li, X., Song, Y., Yao, X., Wu, X., Duan, Y., Zhang, H., Wang, Y., Qian, Z., Cui, J., &amp; Lu, J. (2020). On the origin and continuing evolution of SARS-CoV-2. National Science Review. https://doi.org/10.1093/nsr/nwaa036" />
<meta name="citation_reference" content="Tortorici, M. A., &amp; Veesler, D. (2019). Structural insights into coronavirus entry. En Advances in Virus Research (Vol. 105, pp. 93–116). Academic Press Inc. https://doi.org/10.1016/bs.aivir.2019.08.002" />
<meta name="citation_reference" content="Walls, A. C., Park, Y.-J., Tortorici, M. A., Wall, A., McGuire, A. T., &amp; Veesler, D. (2020). Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell. https://doi.org/10.1016/j.cell.2020.02.058" />
<meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R. S., &amp; Li, F. (2020). Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. Journal of Virology. https://doi.org/10.1128/JVI.00127-20" />
<meta name="citation_reference" content="Weaver, S., Shank, S. D., Spielman, S. J., Li, M., Muse, S. V., &amp; Kosakovsky Pond, S. L. (2018). Datamonkey 2.0: A Modern Web Application for Characterizing Selective and Other Evolutionary Processes. Molecular Biology and Evolution, 35(3), 773–777. https://doi.org/10.1093/molbev/msx335" />
<meta name="citation_reference" content="Webb, B., &amp; Sali, A. (2016). Comparative Protein Structure Modeling Using MODELLER. Current Protocols in Bioinformatics, 54, 5.6.1-5.6.37. https://doi.org/10.1002/cpbi.3" />
<meta name="citation_reference" content="WHO. (2004). Summary of probable SARS cases with onset of illness from 1 November 2002 to 31 July 2003. https://www.who.int/csr/sars/country/table2004_04_21/en/" />
<meta name="citation_reference" content="WHO. (2020). Coronavirus disease 2019 (COVID-19) Situation Report—59. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200319-sitrep-59-covid-19.pdf?sfvrsn=c3dcdef9_2" />
<meta name="citation_reference" content="Wong, M. C., Cregeen, S. J. J., Ajami, N. J., &amp; Petrosino, J. F. (2020). Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019. BioRxiv, 2020.02.07.939207. https://doi.org/10.1101/2020.02.07.939207" />
<meta name="citation_reference" content="Woolley, S., Johnson, J., Smith, M. J., Crandall, K. A., &amp; McClellan, D. A. (2003). TreeSAAP: Selection on amino acid properties using phylogenetic trees. Bioinformatics (Oxford, England), 19(5), 671–672. https://doi.org/10.1093/bioinformatics/btg043" />
<meta name="citation_reference" content="Xiao, K., Zhai, J., Feng, Y., Zhou, N., Zhang, X., Zou, J.-J., Li, N., Guo, Y., Li, X., Shen, X., Zhang, Z., Shu, F., Huang, W., Li, Y., Zhang, Z., Chen, R.-A., Wu, Y.-J., Peng, S.-M., Huang, M., … Shen, Y. (2020). Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins. BioRxiv, 2020.02.17.951335. https://doi.org/10.1101/2020.02.17.951335" />
<meta name="citation_reference" content="Yan, R., Zhang, Y., Li, Y., Xia, L., Guo, Y., &amp; Zhou, Q. (2020). Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science (New York, N.Y.). https://doi.org/10.1126/science.abb2762" />
<meta name="citation_reference" content="Yang, Z. (2007). PAML 4: Phylogenetic analysis by maximum likelihood. Molecular Biology and Evolution, 24(8), 1586–1591. https://doi.org/10.1093/molbev/msm088" />
<meta name="citation_reference" content="Yuan, Y., Cao, D., Zhang, Y., Ma, J., Qi, J., Wang, Q., Lu, G., Wu, Y., Yan, J., Shi, Y., Zhang, X., &amp; Gao, G. F. (2017). Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nature communications, 8, 15092–15092. https://doi.org/10.1038/ncomms15092" />
<meta name="citation_reference" content="Zaki, A. M., Van Boheemen, S., Bestebroer, T. M., Osterhaus, A. D. M. E., &amp; Fouchier, R. A. M. (2012). Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. New England Journal of Medicine, 367(19), 1814–1820. https://doi.org/10.1056/NEJMoa1211721" />
<meta name="citation_reference" content="Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W., Si, H.-R., Zhu, Y., Li, B., Huang, C.-L., Chen, H.-D., Chen, J., Luo, Y., Guo, H., Jiang, R.-D., Liu, M.-Q., Chen, Y., Shen, X.-R., Wang, X., … Shi, Z.-L. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature. https://doi.org/10.1038/s41586-020-2012-7" />
<meta name="citation_reference" content="Zhu, N., Zhang, D., Wang, W., Li, X., Yang, B., Song, J., Zhao, X., Huang, B., Shi, W., Lu, R., Niu, P., Zhan, F., Ma, X., Wang, D., Xu, W., Wu, G., Gao, G. F., &amp; Tan, W. (2020). A novel coronavirus from patients with pneumonia in China, 2019. New England Journal of Medicine, 382(8), 727–733. https://doi.org/10.1056/NEJMoa2001017" />
<meta name="twitter:title" content="SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells." />
<meta name="og-title" property="og:title" content="SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-23" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.21.001933v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.21.001933v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.21.001933v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1208247" />
    <title>SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7dcHhq47QmdD6yAnWFBINNN2i_cFbeS4hmi1bbAxblQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1208247","apath":"\/biorxiv\/early\/2020\/03\/23\/2020.03.21.001933.atom","pisa":"biorxiv;2020.03.21.001933v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.21.001933v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.21.001933v1","title":"SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability","identifier":"node\/1208247"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022D91VOjNtpB+p+VPbc539qgwAOSC+LKxc4q7KvmwaPwI6xnOPK8WXLrHOr1Oalsm2IHIxret4+UMBONBrOVF9dw\\\\\\\/Zs\\\\\\\/51\\\\\\\/OlQJ0TyNes48DUcLnN7YmdRmIDh7+3cJTaNzCleWqfEkJspiXNdTsuhwBo5A4bJzFSaqZCgYuVlY2EXnz2lZb8DvREVj00Emu6avSarl9Su8b9J5gWrQZnq314jFTFfhkDD789v0jujOwS7L\\\\\\\/+\\\\\\\/X4mzbm5qlUgQ1cyTDJPW7JygcjxzQyUQDLXuDzNk8wCSiHnNLLv+5tBb+PzdRaksa4k\\\\\\\/40WN4SiKm\\\\\\\/aSf3VsNbxSWz4wAvR4pPoUHUw32lWyc+36rfK00MRE8IFYwYUMrIz45j7AcuryIzTh6YKS2QCnLfTQ6\\\\\\\/ymSEzOFrVcIOQuhVKIHPOFB8fjeq\\\\\\\/+KeXMzi\\\\\\\/WNIA0xgO0EBzuMFn4j1HHxENL5LaStdukQa70Mu4MVAmgtJ6HmfvqKTnSNwCxtSN8dSEmkXqvt+JIv9H+udXhd0Wmov6iPA183orXqOb+bnCU8GnJRQCvU8Ki5EbZWbsgnB1BRO76xbfS\\\\\\\/QU2zG9nT4DEZFLNZxm1hMl7B0DGTzbC04VOI+3huI5WUL51L6FFJIDHixCdtbqXeC82hsm9EBbqIOrTuWGHXu30zDETWvaHdvav1uY10q5qMYiXy52+hmKlQ1ODc0+2PsrLV\\\\\\\/eWVZlEwktTG1S\\\\\\\/Z6LU9sBICirnG\\\\\\\/bHEbyQHUuxEhBXV8rhstnjvNJMfLWG174p\\\\\\\/pG+cB0qR7MVAW3KqjjdaLBMDuFTHm8vgTEBrQJVW2sfMQMSlJ93WuheqwwxCSufI4SQYntKup9EQIlR91HibXa4\\\\\\\/CYyKKb9HI1q\\\\\\\/EtFuvj0XG86lNpQ4ip9ChxKvAHB2up8VxikrkLLaKlXWJichWqRhDH1CQNW5LgGoWwUQ0LGfTbms8ZRm5Dr97QdNuMWET6nJWeQ1L66pmNiW9tm74iBnn+h2biESAVU4AMSwDsJXa+SkuUdrS1PBRYB7lciNwl\\\\\\\/IGGEq\\\\\\\/7JkBNQWDgdrtnRxG0BNZyHW0ZVKSNu6I+0M\\\\\\\/EHApBJiavkLJp9BoikY0EDMC+Jf9w+LEK0bPsP0zIT1XfFd5NMLQRetCNvHF7JiueW9UkNcoD+SU0n48zqfvg2qNB6M3jH\\\\\\\/R3FVYgLg3hrOh4ZpzOMpzWvC\\\\\\\/Q7nwxvDCg5zsZkYRyae0DH22bnIatAnAXkDqLUf8rzytgYT9ZVIMWQBS7b6VEWQ7\\\\\\\/KR96OcyfIo08WtWifWTRGFOsu5D22mgfGI\\\\\\\/fmWfkfTgaiWiBPKbeXJh9FOcoBPJfutpuZOPcjrI2UBT2rtTMQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022G7eXn+e9S3XWBYSt8CxDcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022100fe88a68911db61902d9d4c30093bc97f1b72e1ce8f26318194dee259d1729\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.21.001933v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1208247 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.21.001933v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SMCaEFZdFzc-MCpIzABTfSf181vZPR-Uf6Y0dFUG99o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208247" id="node1208247" data-pisa="biorxiv;2020.03.21.001933v1" data-pisa-master="biorxiv;2020.03.21.001933" data-apath="/biorxiv/early/2020/03/23/2020.03.21.001933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2965-2515" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Vinicio</span> <span class="nlm-surname">Armijos-Jaramillo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8692-6311" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Yeager</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2746-6914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Muslin</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3710-0035" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yunierkis</span> <span class="nlm-surname">Perez-Castillo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.21.001933 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1208247" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vinicio Armijos-Jaramillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Grupo de Bio-Quimioinformática, Carrera de Ingeniería en Biotecnología, Facultad de Ingeniería y Ciencias Agropecuarias, Universidad de Las Américas</span>, Quito, <span class='nlm-country'>Ecuador</span>, EC170125</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinicio%2BArmijos-Jaramillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armijos-Jaramillo%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVinicio%2BArmijos-Jaramillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2965-2515" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vinicio Armijos-Jaramillo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vinicio.armijos@udla.edu.ec" class="" data-icon-position="" data-hide-link-title="0">vinicio.armijos@udla.edu.ec</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Justin Yeager </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Biodiversidad Medio Ambiente y Salud (BIOMAS), Dirección General de Investigación, Universidad de Las Américas</span>, Quito, <span class='nlm-country'>Ecuador</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BYeager%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeager%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BYeager%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8692-6311" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin Yeager</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Claire Muslin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>One Health Research Group, Faculty of Health Sciences, Universidad de Las Américas</span>, Quito, <span class='nlm-country'>Ecuador</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BMuslin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muslin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaire%2BMuslin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2746-6914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claire Muslin</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yunierkis Perez-Castillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Grupo de Bio-Quimioinformática y Escuela de Ciencias Físicas y Matemáticas, Universidad de Las Américas</span>, Quito, <span class='nlm-country'>Ecuador</span>, EC170125</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunierkis%2BPerez-Castillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perez-Castillo%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYunierkis%2BPerez-Castillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3710-0035" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yunierkis Perez-Castillo</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.21.001933v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.001933v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.001933v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.001933v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.001933v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.21.001933v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1208247" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1208247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">The recent emergence of the novel SARS coronavirus 2 (SARS-CoV-2) marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century, following the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV). The first, SARS-CoV emerged in November 2002 in the Guangdong province of China and spread globally during 2002-03, infecting more than 8000 people and causing 774 deaths (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Drosten et al., 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">WHO, 2004</a>). MERS-CoV was the second emergence and was first detected in Saudi Arabia in 2012 and resulted in nearly 2500 human infections and 858 deaths in 27 countries (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fehr et al., 2017</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zaki et al., 2012</a>). In December 2019, SARS-CoV-2, a previously unknown coronavirus capable of infecting humans was discovered in the Chinese city of Wuhan, in the Hubei province (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huang et al., 2020</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhu et al., 2020</a>). SARS-CoV-2 is associated with an ongoing pandemic of atypical pneumonia, now termed coronavirus disease 19 (Covid-2019) that has affected over 209,000 people with 8778 fatalities as of March 19, 2020 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">WHO, 2020</a>). Both SARS-CoV and MERS-CoV are thought to have originated in colonies of bats, eventually transmitted to humans, putatively facilitated by intermediate hosts such as palm civets and dromedary camels, respectively (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cui et al., 2019</a>). The genome of SARS-CoV-2 shares about 80% nucleotide identity with that of SARS-CoV and is 96% identical to the bat coronavirus BatCoV RaTG13 genome, reinforcing the probable bat origin of the virus (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zhou et al., 2020</a>). However, better assessing the evolutionary dynamics of SARS-CoV-2 is an active research priority worldwide.</p><p id="p-4">SARS-CoV, MERS-CoV and SARS-CoV-2 belong to the genus <em>Betacoronavirus</em> within the subfamily <em>Coronavirinae</em> of the family <em>Coronaviridae</em>. Members of this family are enveloped viruses containing a single positive-strand RNA genome of 27-32 kb in length, the largest known RNA virus genome. The coronavirus spherical virion consists of four structural proteins: the spike glycoprotein (S-protein), the envelope protein, membrane protein and nucleocapsid. The transmembrane trimeric S-protein plays a critical role in virus entry into host cells (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gallagher &amp; Buchmeier, 2001</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tortorici &amp; Veesler, 2019</a>). It comprises two functional subunits: S1 subunit, where the receptor-binding domain (RBD) is found, is responsible for binding host cell surface receptors and S2 subunit mediates subsequent fusion between the viral and cellular membranes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kirchdoerfer et al., 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yuan et al., 2017</a>). Both SARS-CoV and SARS-CoV-2 interact directly with angiotensin-converting enzyme 2 (ACE2) to enter host target cells (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hoffmann et al., 2020</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li et al., 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Walls et al., 2020</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yan et al., 2020</a>). In the case of SARS-CoV, ACE2 binding was found to be a critical determinant for the virus host range and key amino acid residues in the RBD were identified to be essential for ACE2-mediated SARS-CoV infection and adaptation to humans (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li et al., 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li et al., 2005</a>).</p><p id="p-5">Understanding the dynamics that permits a virus to shift hosts is of considerable interest, and further be an essential preliminary step towards facilitating the development of vaccines and the discovery of specific drug therapies. We employ a multidisciplinary approach to look for evidence of diversifying selection on the S-protein gene, and model the interactions between human ACE2 (hACE2) and the RBD of selected coronavirus strains, which ultimately afforded us novel insights detailing virus and host cell interactions. Given the rapid pace of discovery we aim to add clarity to evolutionary dynamics of diseases strains by more precisely understand the dynamics at the S-protein and its interaction with hACE2.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Phylogenetic reconstruction, and analysis testing for evidence of positive/purifying selection in the coronavirus S-protein region</h3><p id="p-6">The most similar genomes to SARS-CoV-2 MN908947 were retrieved using BLASTp (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul et al., 1997</a>) vs the NR database of GenBank (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Genomes were then aligned using MAUVE (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Darling et al., 2004</a>) and the S-protein gene was trimmed. The extracted genomic sections were aligned using a translation align option of Geneious (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kearse et al., 2012</a>) with a MAFFT plugin (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Katoh &amp; Standley, 2013</a>). The phylogenetic reconstruction of S-protein genes was performed with PhyML (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Guindon et al., 2010</a>), using a GTR+I+G model, using 100 non-parametric bootstrap replicates. Both, the alignment and the tree were used as input for PAML Codeml (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yang, 2007</a>). The presence of sites under positive selection was tested by the comparison of M2 (it allows a proportion of positive, neutral and negative selection sites in the alignment) vs M1 (it allows a proportion of neutral and negative selection sites in the alignment) and M8 (ω follows a beta distribution plus a proportion of sites with ω&gt;1) vs M7 (ω follows a beta distribution) models using the ETE toolkit 3.0 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huerta-Cepas et al., 2010</a>). The presence of tree nodes under positive selection was obtained with the free branch model and then tested by the comparison of branch free (different ω for each selected branches) vs M0 (negative selection for all sites and branches-null model) and branch free vs branch neutral (ω=1 for selected branches) models. The presence of sites with positive selection under specific branches of the tree was tested with bsA (proportion of sites with positive selection in a specific branch of the tree) vs bsA1 (proportion of sites with neutral and purifying selection in a specific branch of the tree) models. Likelihood ratio test (LRT) was performed (p≤ 0.05) to compare the hypothesis contrasted by each model.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1214701/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1214701/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1214701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-7" class="first-child">List of coronavirus isolates used for positive selection analysis (closer dataset)</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">We used the set of programs available in HyPhy (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kosakovsky Pond et al., 2020</a>), Fast Unconstrained Bayesian AppRoximation (FUBAR) to detect overall sites under positive selection, and Fixed Effects Likelihood (FEL) to detect specific sites under positive selection in specific branches. We used Mixed Effects Model of Evolution (MEME) to detect episodic positive/diversifying selection and adaptive Branch Site REL (aBSREL) to detect branches in the tree under positive selection. The web server Datamonkey (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Weaver et al., 2018</a>) was used to perform the HyPhy analyses.</p><p id="p-9">Finally, TreeSAAP 3.2 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Woolley et al., 2003</a>) was used to detect sites under adaptation (in terms of physicochemical properties). The same alignment and tree described above were used for this analysis. All these experiments were performed again using the S-protein genes of a shorter list of accessions and more distantly related (broad dataset) to SARS-COV-2 (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY304488&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">AY304488</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY395003&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">AY395003</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=DQ412043&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">DQ412043</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=FJ882957&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">FJ882957</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY417144&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">KY417144</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG772933&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">MG772933</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG772934&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">MG772934</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">MN908947</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_004718&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom">NC_004718</a>) to test the reproducibility of the predicted branches and sites under positive selection.</p></div><div id="sec-4" class="subsection"><h3>Assessing molecular structure and host-specific interactions</h3><p id="p-10">The crystal structure of the SARS-CoV S-protein RBD (GeneBank ID NC_004718) in complex with hACE2 was retrieved from the Protein Data Bank (code 2AJF) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berman et al., 2000</a>). Homology models were constructed using this structure as template for the RBDs of SARS-CoV-2 (SARS2, GeneBank ID MN908947), the Bat SARS-like coronavirus isolate Rm1 (Rm1, GeneBank ID DQ412043) and the Bat SARS-like coronavirus isolate Rs4231 (Rs4231, GeneBank ID KY417146). One additional homology model for the G496D mutant of the SARS-CoV-2 RBD (SARS2-MUT) was constructed. Homology models were built with Modeller v. 9 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Webb &amp; Sali, 2016</a>) using its UCSF Chimera interface (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pettersen et al., 2004</a>). Five models were constructed for each target sequence and the one with the lowest DOPE score was selected for the final model.</p><p id="p-11">All non-amino acidic residues were removed from the SARS-CoV RBD-hACE2 complex to obtain a clean complex. The homology models of the SARS2, Rm1, Rs4231 RBDs and SARS2-MUT were superimposed into the SARS-CoV RBD to obtain their initial complexes with hACE2. These complexes were then subject to molecular dynamics (MD) simulations and estimation of their free energies of binding using Amber 18 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Case et al., 2018</a>). For the later, ACE2 was considered as the receptor and the RBDs as ligands. The protocol described below was employed for all complexes and otherwise noted default software parameters were employed.</p><p id="p-12">Systems preparation was performed with the tleap program of the Amber 18 suite. Each complex was enclosed in a truncated octahedron box extending 10 Å from any atom. Next, the boxes were solvated with TIP3P water molecules and Na+ ions were added to neutralize the excess charge. Systems were minimized in two steps, the first of which consisted in 500 steps of the steepest descent algorithm followed by 500 cycles of conjugate gradient with protein atoms restrained using a force constant of 500 kcal/mol.Å<sup>2</sup>. The PME method with a cutoff of 12 Å was used to treat long range electrostatic interactions. During the second minimization step the PME cutoff was set to 10 Å and it proceeded for 1500 steps of the steepest descent algorithm followed by 1000 cycles of conjugate gradient with no restrains. The same PME cutoff of 10 Å was used in all simulation steps from here on. Both minimization stages were performed at constant volume.</p><p id="p-13">The minimized systems were heated from 0 to 300 K at constant volume constraining all protein atoms with a force constant of 10 kcal/mol.Å<sup>2</sup>. The SHAKE algorithm was used to constrain all bonds involving hydrogens and their interactions were omitted from this step on. Heating took place for 10000 steps, with a time step of 2 fs and a Langevin thermostat with a collision frequency of 1.0 ps<sup>−1</sup> was employed. All subsequent MD steps utilized the same thermostat settings. Afterward, the systems were equilibrated for 100 ps at a constant temperature of 300 K and a constant pressure of 1 bar. Pressure was controlled with isotropic position scaling with a relaxation time of 2 ps. The equilibrated systems were used as input for 10 ns length production MD simulations.</p><p id="p-14">The free energies of binding were computed under the MM-PBSA approach implemented in AmberTools 18 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Case et al., 2018</a>). A total of 100 MD snapshots were evenly selected, one every 50 ps, from the last 5 ns of the production run for MM-PBSA calculations. The ionic strength was set to 100 mM and the solute dielectric factor was set to 4 for all systems.</p></div></div><div class="section" id="sec-5"><h2 class="">Results and discussion</h2><p id="p-15">In order to detect branches and sites under positive/negative selection, two datasets were explored. The first (‘closer’ dataset) harbors the most similar genomes to Wuhan-Hu-1 coronavirus (SARS-CoV-2) (MN908947). For this dataset, several genomes were excluded from the analysis because they showed minimal variation to other sequences. We used a preliminary phylogeny to select a representative isolate of each clade (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) in order to exclude highly similar sequences. The second dataset (‘broad’ dataset) includes some accessions of the first dataset plus isolates less related to SARS-CoV-2, like SARS-like coronavirus isolates from different countries (see methods). We compare the results of two dataset because the phylogenetic distance between orthologues in a given dataset has been demonstrated to alter the ability to detect selection in PAML and MEME (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McBee et al., 2015</a>).</p><p id="p-16">In both datasets, we observed evidence of purifying selection in the majority of nodes of the tree. Specifically, in the ‘closer’ dataset we identified 38 nodes with evidence of negative selection, and 4 under positive selection when free ratios model of CODEML model was applied. To confirm the four nodes under positive selection we use LTR test for contrasting hypothesis using branch free, branch neutral and M0 models of CODEML. Using these approximations, any node predicted by free ratios model with ω&gt;1 was significantly different to the purifying (ω&lt;1) or neutral (ω=1) models. An equivalent analysis was performed using aBSREL of HyPhy, observing episodic diversifying selection in at least 8 of 41 nodes of the phylogenetic tree reconstructed with the ‘closer’ dataset (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Interestingly, one of the divisions detected with diversifying selection was the branch that contains SARS-CoV-2, Pangolin coronavirus isolate MP789 and Bat coronavirus RaTG13 (called SARS-CoV-2 group) but not the specific branch that contains SARS-CoV-2.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of S-protein genes of selected betacoronaviruses (&#x2018;closer&#x2019; dataset). Predicted nodes under episodic diversifying selection (according to predictions generated by aBSREL of HyPhy) were colored in blue. Numbers in the inner nodes represent the number of non-parametric bootstrap replicates" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-348422543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree of S-protein genes of selected betacoronaviruses (&#x2018;closer&#x2019; dataset). Predicted nodes under episodic diversifying selection (according to predictions generated by aBSREL of HyPhy) were colored in blue. Numbers in the inner nodes represent the number of non-parametric bootstrap replicates&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F1.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F1.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-17" class="first-child">Phylogenetic tree of S-protein genes of selected betacoronaviruses (‘closer’ dataset). Predicted nodes under episodic diversifying selection (according to predictions generated by aBSREL of HyPhy) were colored in blue. Numbers in the inner nodes represent the number of non-parametric bootstrap replicates</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">MEME of HyPhy detected 40 sites in different branches with signal of episodic positive selection, but FEL did not identify sites under positive selection (with p &lt; 0.1) in the SARS-CoV-2 when ‘closer’ dataset was used. Depending on the focal dataset analyzed (‘close’ vs. ‘broad’), FEL detected several sites under positive selection in SARS-CoV-2, demonstrating the influence of relatedness within the dataset in branch-sites model predictions. FEL predicted the site F486 under positive selection in SARS-CoV-2 using the closer dataset without Pangolin coronavirus isolate MP789. It is interesting despite the influence of the dataset in the results, because site F486 is directly involved in hACE2-RBD interaction (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Shang et al., 2020</a>), explaining at least in part strong selection at this site. Moreover, the branch-site model bsA (positive selection) vs bsA1 (relaxation) of CODEML were compared to find evidence of sites under positive selection in branch of SARS-CoV-2 using the ‘closer’ dataset, but bsA does not show significant differences with bsA1 (p&gt;0.05) indicating selection cannot be confidently implicated, but it was when other datasets were used (including F486). In summary, we do find evidence of sites under positive/episodic selection in branches of close related strains of Wuhan-Hu-1 isolate coronavirus. However, there is not strong evidence of specific sites under positive selection in SARS-CoV-2 using the tools mentioned in this work. This result does not disregard the presence of positive selection sites in SARS-CoV-2, nonetheless, it shows the limitation of the methods to identify with precision specific sites under positive selection in a precise taxon of a phylogenetic tree. We further warn researchers need to be conservative with interpretations of studies utilizing these methodologies, given the equivocal results can be generated by datasets varying in genetic similarity.</p><p id="p-19">To complement our analyses looking for evidence of selection among lineages, we specifically analyzed for patterns of selection across sites in the S-protein genes, we used the sites models available in CODEML and HyPhY. Model M2 of CODEML detected 0.133 % of sites under positive selection (ω&gt;1) and models M1 and M2 detected 85% of sites under purifying selection (ω&lt;1). Model M2 explains the significant data better (p=7e-4) than M1 model, that takes in account only sites with neutral and purifying selection. FUBAR of HyPhy also detected 1070 sites under purifying selection and only 2 sites under positive selection (alignment size 1284). A similar analysis performed by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benvenuto et al. (2020)</a> with FUBAR detects 1065 sites under purifying selection and sites 536 and 644 as under positive selection.</p><p id="p-20">These results suggests by in large strong purifying selection is acting over the vast majority of the S-protein gene, with a comparatively low proportion of sites under positive selection. The evidence of a high amount of purifying selection strengthens similar findings previously reported in RNA viruses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hughes &amp; Hughes, 2007</a>). These results are congruent with the analysis of <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tang et al. (2020)</a> performed with entire coronavirus genomes. However, the sites under positive selection reported differ from our results, and from those of <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Benvenuto et al. (2020)</a>. They found the sites 439, 483 and 439 (counting from the first amino acid of S-protein), our FUBAR analysis found the sites 443 and 445, and FEL found the residue 486 as specific site under positive selection in SARS-COV-2. To resolve these ambiguities in positive selection sites we calculate putative selection sites with CODEML (using Bayes Empirical Bayes from M2 and M8 models) and FUBAR with different datasets reflecting the addition of novel sequences to online repositories (broad, closer, closer without MN996532 and MT084071 and closer without MT084071) and we obtain different results.</p><p id="p-21">It is becoming increasingly clear that predictions of positive selected sites are highly influenced simply by the diversity of the individual sequences included in the datasets. In any case, the majority of predicted sites converge in the region between 439 to 508, a section of the RBD. Additionally, we used TreeSaap to detect important biochemical amino acid properties changes over regions and/or sites along betacoronavirus S-protein. Using a sliding window size of 20 (increasing by 1) we detect that the region between 466 to 500 (using SARS-COV-2 S-protein as a reference) have drastic amino acid changes for alpha-helical tendencies. In addition, the section between 448 to 485 residues registers radical changes in amino acids implicated in the equilibrium constant (ionization of COOH). In the structural analysis we performed, the section between 472 to 486 forms a loop that is not present in certain S-proteins of coronavirus isolated in bats. This loop extends the interaction area between RBD of S-protein and human ACE2, in fact, the lack of these loop decreases the negative energy of interaction (increasing the binding) among these two molecules (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These results obtained from independent analysis strongly highlight the importance of 439 to 508 section. Additionally, important hACE2-binding residues in the RBD from SARS-COV-2 obtained from the crystallography and structure determination performed by <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Shang et al. (2020)</a> are also present in the section we highlight here. We propose that this region is the most probable to contain the sites under positive selection due to predictions by our CODEML and FUBAR models. In that sense, we refer to this section as Region under Positive Selection (RPS). It is important to additionally clarify that even inside the RPS we found at least 20 aa highly conserved between coronaviruses, several of them are predicted as sites under purifying selection. This shows that it is necessary to maintain sites without change around polymorphic sites, probably to conserve the protein structure and at the same time to have the ability to colonize more than one host.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1214700/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1214700/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1214700" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-22" class="first-child">Estimated free energies of binding of the coronaviruses RBDs to ACE2. Energy values are expressed in kcal/mol.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Interestingly, the RPS of the pangolin coronavirus isolate MP789 differs only in one amino acid with the homologous region of SARS-CoV-2, whereas in contrast the bat coronavirus RaTG13 (the overall most similar isolate to SARS-CoV-2 sequenced at the moment) shows 17 differences in the same region. Several explanations could derive from this observation. The hypothesis of recombination inside the pangolin between a native coronavirus strain and a bat coronavirus (like RaTG13) is congruent with our observation. This scenario was proposed and discussed as the origin of SARS-CoV-2 by (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lam et al., 2020</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wong et al., 2020</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Xiao et al., 2020</a>), however, other explanations are possible. If the SARS-CoV-2, RaTG13 and pangolin coronavirus MP789 isolate are closely related as shown in the tree of the <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, we are observing the ancestral sequence of RPS in human and pangolin coronaviruses, and a mutated version in bat virus. Elucidating the origin of SARS-CoV-2 is beyond the scope of this work, nevertheless sequencing of new coronavirus isolates in the near future could resolve this question.</p><p id="p-24">With a list of broader observations related to the role of selection across viral genomes we aimed to specifically understand how these regions could affect virus/host interactions. To understand more in deep the importance of RPS in the evolution of SARS-COV-2, we quantified the relative importance of this region in the interaction between RBD and hACE2. In that sense, MD simulations were run for five complexes (listed in methods). In all cases the systems were stable with Root Mean Square Deviations (RMSD) of their backbones between 1.11 Å and 3.30 Å relative to the initial complexes structures during the last 5 ns of the production run. We first investigated the network of contacts between the ligands (coronaviruses RDB) with the receptor (hACE2). Overall, all complexes present a large number of contacts between the ligands and the receptor in at least 50% of the MD snapshots selected for MM-PBSA calculations. Common interactions with T27, F28, K31, H34, Y41, K353, G354, D355 and R357 of the receptor are observed in all systems. The full networks of interactions between the coronaviruses and the hACE2 receptor are provided as Supporting Information.</p><p id="p-25">Next we estimated the free energies of binding of the coronaviruses’ RBDs to hACE2 and the results of these evaluations are summarized in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. These calculations show that the SARS2, SARS and Rs4231 viruses are predicted to favorably bind to the human hACE2 receptor, while the Rm1 and SARS2-MUT variants present unfavorable free energies of binding. The fact that the bat’s coronavirus Rs4231, in addition to SARS and SARS2, presents favorable interaction with hACE2 is in accordance with the previous observation that it is able to infect human cells expressing this protein (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hu et al., 2017</a>). To get more insights into the contribution of the receptor and the RBDs to the binding process, we performed energy decomposition experiments.</p><p id="p-26">The contribution of each residue in the studied coronaviruses that interact with the hACE2 receptor are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Rows are presented in such a way that each of them contains the residues occupying the same position in the viruses RBDs structures as in the SAR2 RBD structure. From here on, residues numeration will take that of SARS2 as reference. In general, most RBDs residues show negative values of contribution to the free energies of binding to the human receptor. All studied RBDs, except that of the Rm1 coronavirus, have amino acids with large favorable contributions to the free energies of binding that directly interact with hACE2: K417 of SARS2 and SARS2-MUT, R426 in SARS and R480 in Rs4231. On the other hand, the G476D mutation (D463 present in bat coronavirus strains) have a negative contribution to the binding of the RDB to hACE2. This site was predicted to be under purifying selection by FUBAR analyses, and is located within the RPS.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1214699/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1214699/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1214699" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-27" class="first-child">Contribution of the residues in the RDB of the coronaviruses interacting with the hACE2 receptor. Energy values are presented as kcal/mol.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Strikingly, the G496D mutation (SARS2 numeration) has a large negative influence in the free energy of binding in the two complexes that contain it. It is also worth noting that the three aspartic acid substitutions present in all systems negatively contribute to the systems stability. Taking into account that the only difference between SARS2 and SARS2-MUT is the G496D mutation, we postulate that this RBD position is critical for the human receptor recognition by coronaviruses. To the best of our knowledge, no coronavirus having aspartic acid at this position is able to infect human cells. This result supports the prediction from FUBAR analyses indicating that the site G496D is under purifying selection. Combined, our results strongly suggest that the mutation of the D496 residue present in the coronaviruses from bats is critical for their RBDs to recognize the human hACE2 receptor. Additionally, it shows the importance of sites under purifying selection in RPS for the RBD evolution.</p><p id="p-29">To better interpret the influence of the key interactions between the coronaviruses RBDs and their hACE2 receptor, their interactions were analyzed. To select the representative structure of each system the MD snapshots employed for MM-PBSA calculations were clustered. Then, the representative structure of a system was selected as the centroid of the most populated cluster. The predicted RBD-hACE2 complexes for SARS2, SARS and SARS2-MUT are depicted in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted interaction of SARS2 (top), SARS2-MUT (middle) and SARS (bottom) with the human ACE2 receptor. hACE2 in depicted in gray and the RBD of coronaviruses in cyan. Oxygen atoms are colored red and nitrogen blue. The numbering of the residues corresponds to that of the sequence of each spike glycoprotein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-348422543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted interaction of SARS2 (top), SARS2-MUT (middle) and SARS (bottom) with the human ACE2 receptor. hACE2 in depicted in gray and the RBD of coronaviruses in cyan. Oxygen atoms are colored red and nitrogen blue. The numbering of the residues corresponds to that of the sequence of each spike glycoprotein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F2.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F2.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.03.21.001933/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-30" class="first-child">Predicted interaction of SARS2 (top), SARS2-MUT (middle) and SARS (bottom) with the human ACE2 receptor. hACE2 in depicted in gray and the RBD of coronaviruses in cyan. Oxygen atoms are colored red and nitrogen blue. The numbering of the residues corresponds to that of the sequence of each spike glycoprotein.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Many studies have focused on coronaviruses mutations that favor adaptations for human hosts infections. For example, it has been shown that specific substitutions at positions 455, 486, 483, 494 and 501 (442, 472, 479, 480 and 487 in SARS) of the RBD of SARS favors the interaction between the RBD of SARS and hACE2 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cui et al., 2019</a>). Likewise, homology modeling studies found favorable interactions between the residues occupying these positions in the SARS2 RBD and the human receptor (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wan et al., 2020</a>). The cornerstone of these favorable interactions is the complementarity of the RBDs with hot spots 31 and 353. These are salt bridges between K31 and E35 and between D38 and K353 of ACE2 which are buried in a hydrophobic environment (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure. 2</a>). In the cases of SARS2 and SARS, Q493 (N479 in SARS) and N501 (T487 in SARS) add support to the hot spots according to these previous studies. These observations should also hold for the Rs4231 strain, however the N501A change in the later compared to SARS2 (A488 in Rs4231) add little support to hot spot 353. In this case, to continue permitting human infection, the large favorable contribution of R480 in Rs4231 to the free energy of binding could compensate the weak support provided by A488 to hot spot 353.</p><p id="p-32">Interestingly, K353 is the residue forming the largest network of contacts with the analyzed RBDs among those belonging to both hot spots. Our simulations also show that in SARS2 and SARS the RBD amino acids with the largest contribution to the free energy of binding, K417 and R426 (see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) respectively, do not interact with any hot spot residue. Instead, they interact with D30 of hACE2 in the SARS2 complex and with E329 of the human receptor in the SARS complex. This could indicate that interactions additional to those previously identified with the hACE2 hotspots could be critical for the stabilization of the RDB-human receptor complexes. Finally, we analyzed the possible reasons for the predicted negative impact that the G496D mutation has on the predicted free energies of binding of the RBD to hACE2. As depicted in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, G496 directly interacts with K353 in hot spot 353 and its mutation interferes with the D38-K353 salt bridge. Specifically, D496 of the RDB point to D38 of hACE yields a high electric repulsion between these amino acids. Consequently, this portion of the RBD is pushed to a position further from hACE2 than that observed in the wild type receptor, resulting in the reduction of its network of contacts with K353. As a result, the binding of the RBD to hACE2 is considerably inhibited and unlikely to occur.</p></div><div class="section" id="sec-6"><h2 class="">Conclusions</h2><p id="p-33">A priority in ongoing research is to better understand coronavirus evolution, with specific interests in understanding the role of selection pressures in viral evolution, and clarifying how viral strains can infect novel hosts. Our experiments suggest that there are sites under positive selection in the S-protein gene of SARS-CoV-2 and other betacoronaviruses, particularly in a region that we called RPS (Region under Positive Selection) inside of the RBD. However, we have identified that by in large, sites in this region (and overall, in the S-protein gene) are under purifying selection. Particularly, for the site D496G, the presence of aspartic acid seems indispensable for the interaction with the hACE2. Additionally, we performed MD simulations and free energies of binding predictions for five different complexes of coronaviruses that do and do not infect human cells. Our results suggest that as long as no disrupting interference occur with both salt bridges at hot spots 31 and 353 coronaviruses are able to bind with hACE2. Modeling results suggest that interference with the hot spot 353 could be and effective strategy for inhibiting the recognition of the RBD of the SARS-COV-2 spike protein by its human host receptor ACE2 and hence prevent infections. Although additional simulations and experiments are required, all evidence suggests that the mutation of D496 in the bat variants of the coronaviruses permit infection of human cells. Giving the large contribution of SARS2 K417 to the free energy of binding of the RBD to hACE2 we propose that blocking its interaction with the receptor D30 could be a promising strategy for future drug discovery efforts.</p></div><div class="section" id="sec-8"><h2 class="">Conflict of interest</h2><p id="p-34">The authors declare that they have no conflicts of interest.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madden</span>,  <span class="cit-name-given-names">T. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schäffer</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Gapped BLAST and PSI-BLAST: A new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">17</span>), <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benvenuto</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giovanetti</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciccozzi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spoto</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Angeletti</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ciccozzi</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">The 2019-new coronavirus epidemic: Evidence for virus evolution</span>. <abbr class="cit-jnl-abbrev">Journal of Medical Virology</abbr>, <span class="cit-vol">92</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">455</span>–<span class="cit-lpage">459</span>. <a href="https://doi.org/10.1002/jmv.25688">https://doi.org/10.1002/jmv.25688</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BVirology%26rft.volume%253D92%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.3" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berman</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westbrook</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilliland</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhat</span>,  <span class="cit-name-given-names">T. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissig</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shindyalov</span>,  <span class="cit-name-given-names">I. N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bourne</span>,  <span class="cit-name-given-names">P. E.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">The Protein Data Bank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>. <a href="https://doi.org/10.1093/nar/28.1.235">https://doi.org/10.1093/nar/28.1.235</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Case</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ben-Shalom</span>,  <span class="cit-name-given-names">I. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brozell</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cerutti</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheatham</span>,  <span class="cit-name-given-names">T. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruzeiro</span>,  <span class="cit-name-given-names">V. W. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darden</span>,  <span class="cit-name-given-names">T. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duke</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghoreishi</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilson</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gohlke</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goetz</span>,  <span class="cit-name-given-names">A. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greene</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homeyer</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Izadi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kovalenko</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">T. S.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Kollman</span>,  <span class="cit-name-given-names">P. A.</span></span> (<span class="cit-pub-date">2018</span>). <abbr class="cit-jnl-abbrev">AMBER 2018</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.5" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature Reviews. Microbiology</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span>. <a href="https://doi.org/10.1038/s41579-018-0118-9">https://doi.org/10.1038/s41579-018-0118-9</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews.%2BMicrobiology%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.6" data-doi="10.1101/gr.2289704"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darling</span>,  <span class="cit-name-given-names">A. C. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mau</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blattner</span>,  <span class="cit-name-given-names">F. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Perna</span>,  <span class="cit-name-given-names">N. T.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Mauve: Multiple alignment of conserved genomic sequence with rearrangements</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1394</span>–<span class="cit-lpage">1403</span>. <a href="https://doi.org/10.1101/gr.2289704">https://doi.org/10.1101/gr.2289704</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BC.E.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1394%26rft.epage%253D1403%26rft.atitle%253DMauve%253A%2BMultiple%2BAlignment%2Bof%2BConserved%2BGenomic%2BSequence%2BWith%2BRearrangements%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2289704%26rft_id%253Dinfo%253Apmid%252F15231754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTM5NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDMuMjEuMDAxOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.7" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Günther</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiser</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Werf</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brodt</span>,  <span class="cit-name-given-names">H.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabenau</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panning</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolesnikova</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burguière</span>,  <span class="cit-name-given-names">A.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cinatl</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eickmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Escriou</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grywna</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramme</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manuguerra</span>,  <span class="cit-name-given-names">J.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">S.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Doerr</span>,  <span class="cit-name-given-names">H. W.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">The New England Journal of Medicine</abbr>, <span class="cit-vol">348</span>(<span class="cit-issue">20</span>), <span class="cit-fpage">1967</span>–<span class="cit-lpage">1976</span>. <a href="https://doi.org/10.1056/NEJMoa030747">https://doi.org/10.1056/NEJMoa030747</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Middle East Respiratory Syndrome: Emergence of a Pathogenic Human Coronavirus</span>. <abbr class="cit-jnl-abbrev">Annual Review of Medicine</abbr>, <span class="cit-vol">68</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">387</span>–<span class="cit-lpage">399</span>. <a href="https://doi.org/10.1146/annurev-med-051215-031152">https://doi.org/10.1146/annurev-med-051215-031152</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BMedicine%26rft.volume%253D68%26rft.spage%253D387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.9" data-doi="10.1006/viro.2000.0757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>,  <span class="cit-name-given-names">T. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M. J.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Coronavirus spike proteins in viral entry and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-vol">279</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">371</span>–<span class="cit-lpage">374</span>. <a href="https://doi.org/10.1006/viro.2000.0757">https://doi.org/10.1006/viro.2000.0757</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DGallagher%26rft.auinit1%253DT.%2BM.%26rft.volume%253D279%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D374%26rft.atitle%253DCoronavirus%2Bspike%2Bproteins%2Bin%2Bviral%2Bentry%2Band%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2000.0757%26rft_id%253Dinfo%253Apmid%252F11162792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.2000.0757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166695000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.10" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>,  <span class="cit-name-given-names">J.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>,  <span class="cit-name-given-names">W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">59</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>. <a href="https://doi.org/10.1093/sysbio/syq010">https://doi.org/10.1093/sysbio/syq010</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleine-Weber</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schroeder</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krüger</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrler</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erichsen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schiergens</span>,  <span class="cit-name-given-names">T. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrler</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">N.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitsche</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pöhlmann</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">181</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>. <a href="https://doi.org/10.1016/j.cell.2020.02.052">https://doi.org/10.1016/j.cell.2020.02.052</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D181%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.12" data-doi="10.1371/journal.ppat.1006698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">L.-P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">J.-Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">D.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">X.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.-N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">e1006698</span>. <a href="https://doi.org/10.1371/journal.ppat.1006698">https://doi.org/10.1371/journal.ppat.1006698</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>, <span class="cit-vol">395</span>(<span class="cit-issue">10223</span>), <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>. <a href="https://doi.org/10.1016/S0140-6736(20)30183-5">https://doi.org/10.1016/S0140-6736(20)30183-5</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.14" data-doi="10.1186/1471-2105-11-24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dopazo</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">ETE: A python Environment for Tree Exploration</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">24</span>. <a href="https://doi.org/10.1186/1471-2105-11-24">https://doi.org/10.1186/1471-2105-11-24</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuerta-Cepas%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DETE%253A%2Ba%2Bpython%2BEnvironment%2Bfor%2BTree%2BExploration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-24%26rft_id%253Dinfo%253Apmid%252F20070885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070885&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.15" data-doi="10.1016/j.gene.2007.09.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">A. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">M. A. K.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">More effective purifying selection on RNA viruses than in DNA viruses</span>. <abbr class="cit-jnl-abbrev">Gene</abbr>, <span class="cit-vol">404</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">117</span>–<span class="cit-lpage">125</span>. <a href="https://doi.org/10.1016/j.gene.2007.09.013">https://doi.org/10.1016/j.gene.2007.09.013</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D404%26rft.issue%253D1-2%26rft.spage%253D117%26rft.epage%253D125%26rft.atitle%253DMore%2Beffective%2Bpurifying%2Bselection%2Bon%2BRNA%2Bviruses%2Bthan%2Bin%2BDNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2007.09.013%26rft_id%253Dinfo%253Apmid%252F17928171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2007.09.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17928171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251074300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.16" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Standley</span>,  <span class="cit-name-given-names">D. M.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>. <a href="https://doi.org/10.1093/molbev/mst010">https://doi.org/10.1093/molbev/mst010</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.17" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moir</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sturrock</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duran</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thierer</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashton</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meintjes</span>,  <span class="cit-name-given-names">P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>. <a href="https://doi.org/10.1093/bioinformatics/bts199">https://doi.org/10.1093/bioinformatics/bts199</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.18" data-doi="10.1038/nature17200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassine</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">H. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A. B.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Pre-fusion structure of a human coronavirus spike protein</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">531</span>(<span class="cit-issue">7592</span>), <span class="cit-fpage">118</span>–<span class="cit-lpage">121</span>. <a href="https://doi.org/10.1038/nature17200">https://doi.org/10.1038/nature17200</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17200%26rft_id%253Dinfo%253Apmid%252F26935699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26935699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">A. F. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velazquez</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weaver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hepler</span>,  <span class="cit-name-given-names">N. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murrell</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shank</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magalis</span>,  <span class="cit-name-given-names">B. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouvier</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wisotsky</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spielman</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>,  <span class="cit-name-given-names">S. D. W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Muse</span>,  <span class="cit-name-given-names">S. V.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">295</span>–<span class="cit-lpage">299</span>. <a href="https://doi.org/10.1093/molbev/msz197">https://doi.org/10.1093/molbev/msz197</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D37%26rft.spage%253D295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T. T.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shum</span>,  <span class="cit-name-given-names">M. H.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">H.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">Y.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>,  <span class="cit-name-given-names">X.-B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>,  <span class="cit-name-given-names">W. Y.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">G. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.-L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China</span>. <span class="cit-source">BioRxiv, 2020.02.13.945485</span>. <a href="https://doi.org/10.1101/2020.02.13.945485">https://doi.org/10.1101/2020.02.13.945485</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.21" data-doi="10.1128/JVI.80.9.4211-4219.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">I.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Animal Origins of the Severe Acute Respiratory Syndrome Coronavirus: Insight from ACE2-S-Protein Interactions</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-vol">80</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">4211</span>–<span class="cit-lpage">4219</span>. <a href="https://doi.org/10.1128/jvi.80.9.4211-4219.2006">https://doi.org/10.1128/jvi.80.9.4211-4219.2006</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D80%26rft.issue%253D9%26rft.spage%253D4211%26rft.epage%253D4219%26rft.atitle%253DAnimal%2BOrigins%2Bof%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%253A%2BInsight%2Bfrom%2BACE2-S-Protein%2BInteractions%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.9.4211-4219.2006%26rft_id%253Dinfo%253Apmid%252F16611880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzkvNDIxMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDMuMjEuMDAxOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.22" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Wenhui</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasllieva</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berne</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Somasundaran</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luzuriaga</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greeneugh</span>,  <span class="cit-name-given-names">T. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">426</span>(<span class="cit-issue">6965</span>), <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span>. <a href="https://doi.org/10.1038/nature02145">https://doi.org/10.1038/nature02145</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.23" data-doi="10.1038/sj.emboj.7600640"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Wenhui</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">I.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasilieva</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marasco</span>,  <span class="cit-name-given-names">W. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2</span>. <abbr class="cit-jnl-abbrev">The EMBO journal</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1634</span>–<span class="cit-lpage">1643</span>. <a href="https://doi.org/10.1038/sj.emboj.7600640">https://doi.org/10.1038/sj.emboj.7600640</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D1634%26rft.epage%253D1643%26rft.atitle%253DReceptor%2Band%2Bviral%2Bdeterminants%2Bof%2BSARS-coronavirus%2Badaptation%2Bto%2Bhuman%2BACE2%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600640%26rft_id%253Dinfo%253Apmid%252F15791205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyNC84LzE2MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMy8yMDIwLjAzLjIxLjAwMTkzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.24" data-doi="10.1093/molbev/msu399"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBee</span>,  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozmiarek</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyerson</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rowley</span>,  <span class="cit-name-given-names">P. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sawyer</span>,  <span class="cit-name-given-names">S. L.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">The Effect of Species Representation on the Detection of Positive Selection in Primate Gene Data Sets</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">1091</span>–<span class="cit-lpage">1096</span>. <a href="https://doi.org/10.1093/molbev/msu399">https://doi.org/10.1093/molbev/msu399</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu399%26rft_id%253Dinfo%253Apmid%252F25556235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25556235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.25" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>,  <span class="cit-name-given-names">E. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">T. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Couch</span>,  <span class="cit-name-given-names">G. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>,  <span class="cit-name-given-names">D. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">E. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ferrin</span>,  <span class="cit-name-given-names">T. E.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">UCSF Chimera—A visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">Journal of Computational Chemistry</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">13</span>), <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>. <a href="https://doi.org/10.1002/jcc.20084">https://doi.org/10.1002/jcc.20084</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aihara</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geng</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auerbach</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structural basis for receptor recognition by the novel coronavirus from Wuhan</span>. <span class="cit-source">Research Square Pre-Print</span>. <a href="https://doi.org/10.21203/rs.2.24749/v1">https://doi.org/10.21203/rs.2.24749/v1</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">On the origin and continuing evolution of SARS-CoV-2</span>. <span class="cit-source">National Science Review</span>. <a href="https://doi.org/10.1093/nsr/nwaa036">https://doi.org/10.1093/nsr/nwaa036</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.21.001933v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-chapter-title">Structural insights into coronavirus entry</span>. <span class="cit-source">En Advances in Virus Research</span> (Vol. <span class="cit-vol">105</span>, pp. <span class="cit-fpage">93</span>–<span class="cit-lpage">116</span>). <span class="cit-publ-name">Academic Press Inc</span>. <a href="https://doi.org/10.1016/bs.aivir.2019.08.002">https://doi.org/10.1016/bs.aivir.2019.08.002</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEn%2BAdvances%2Bin%2BVirus%2BResearch%26rft.volume%253D105%26rft.spage%253D93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">A. T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <span class="cit-source">Cell</span>. <a href="https://doi.org/10.1016/j.cell.2020.02.058">https://doi.org/10.1016/j.cell.2020.02.058</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <span class="cit-source">Journal of Virology</span>. <a href="https://doi.org/10.1128/JVI.00127-20">https://doi.org/10.1128/JVI.00127-20</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weaver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shank</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spielman</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>,  <span class="cit-name-given-names">S. V.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S. L.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Datamonkey 2.0: A Modern Web Application for Characterizing Selective and Other Evolutionary Processes</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">773</span>–<span class="cit-lpage">777</span>. <a href="https://doi.org/10.1093/molbev/msx335">https://doi.org/10.1093/molbev/msx335</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D35%26rft.spage%253D773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sali</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Comparative Protein Structure Modeling Using MODELLER</span>. <abbr class="cit-jnl-abbrev">Current Protocols in Bioinformatics</abbr>, <span class="cit-vol">54</span>, <span class="cit-fpage">5.6.1</span>-<span class="cit-lpage">5.6.37</span>. <a href="https://doi.org/10.1002/cpbi.3">https://doi.org/10.1002/cpbi.3</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BProtocols%2Bin%2BBioinformatics%26rft.volume%253D54%26rft.spage%253D561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.33"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. (<span class="cit-pub-date">2004</span>). <span class="cit-source">Summary of probable SARS cases with onset of illness from 1 November 2002 to 31 July 2003</span>. <a href="https://www.who.int/csr/sars/country/table2004_04_21/en/">https://www.who.int/csr/sars/country/table2004_04_21/en/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.34"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. (<span class="cit-pub-date">2020</span>). <span class="cit-source">Coronavirus disease 2019 (COVID-19) Situation Report—59</span>. <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200319-sitrep-59-covid-19.pdf?sfvrsn=c3dcdef9_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200319-sitrep-59-covid-19.pdf?sfvrsn=c3dcdef9_2</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cregeen</span>,  <span class="cit-name-given-names">S. J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ajami</span>,  <span class="cit-name-given-names">N. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Petrosino</span>,  <span class="cit-name-given-names">J. F.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019</span>. <span class="cit-source">BioRxiv, 2020.02.07.939207</span>. <a href="https://doi.org/10.1101/2020.02.07.939207">https://doi.org/10.1101/2020.02.07.939207</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.36" data-doi="10.1093/bioinformatics/btg043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolley</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>,  <span class="cit-name-given-names">K. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">McClellan</span>,  <span class="cit-name-given-names">D. A.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">TreeSAAP: Selection on amino acid properties using phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">671</span>–<span class="cit-lpage">672</span>. <a href="https://doi.org/10.1093/bioinformatics/btg043">https://doi.org/10.1093/bioinformatics/btg043</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DWoolley%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D671%26rft.epage%253D672%26rft.atitle%253DTreeSAAP%253A%2BSelection%2Bon%2BAmino%2BAcid%2BProperties%2Busing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg043%26rft_id%253Dinfo%253Apmid%252F12651734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12651734&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181964600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">J.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">R.-A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">S.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">M.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</span>. <span class="cit-source">BioRxiv, 2020.02.17.951335</span>. <a href="https://doi.org/10.1101/2020.02.17.951335">https://doi.org/10.1101/2020.02.17.951335</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. <span class="cit-source">Science (New York, N.Y.)</span>. <a href="https://doi.org/10.1126/science.abb2762">https://doi.org/10.1126/science.abb2762</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.39" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">PAML 4: Phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>. <a href="https://doi.org/10.1093/molbev/msm088">https://doi.org/10.1093/molbev/msm088</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G. F.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">15092</span>–<span class="cit-lpage">15092</span>. <a href="https://doi.org/10.1038/ncomms15092">https://doi.org/10.1038/ncomms15092</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D8%26rft.spage%253D15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.41" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Boheemen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>,  <span class="cit-name-given-names">A. D. M. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>, <span class="cit-vol">367</span>(<span class="cit-issue">19</span>), <span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span>. <a href="https://doi.org/10.1056/NEJMoa1211721">https://doi.org/10.1056/NEJMoa1211721</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.03.21.001933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.21.001933v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.-D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">R.-D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.-Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <span class="cit-source">Nature</span>. <a href="https://doi.org/10.1038/s41586-020-2012-7">https://doi.org/10.1038/s41586-020-2012-7</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.001933v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhan</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G. F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>, <span class="cit-vol">382</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>. <a href="https://doi.org/10.1056/NEJMoa2001017">https://doi.org/10.1056/NEJMoa2001017</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.21.001933v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/628933v3" title="Toffee – a highly efficient, lossless file format for DIA-MS" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.21.000398v1" title="Automatic identification of relevant genes from low-dimensional embeddings of single cell RNAseq data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 23, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.21.001933v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.21.001933v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.21.001933v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.21.001933v1" class="active" data-icon-position="" data-hide-link-title="0">SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1208247" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Rcv_I2GQ_CN0c03bLG_TBCC7Agdh5Wb4OqEYylJRnq0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208247" id="node1208247--2" data-pisa="biorxiv;2020.03.21.001933v1" data-pisa-master="biorxiv;2020.03.21.001933" data-seqnum="1208247" data-apath="/biorxiv/early/2020/03/23/2020.03.21.001933.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vinicio</span> <span class="nlm-surname">Armijos-Jaramillo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Yeager</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Muslin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yunierkis</span> <span class="nlm-surname">Perez-Castillo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.21.001933; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.21.001933 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;title=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;title=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;text=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;title=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;t=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;title=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;title=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208247" id="node1208247--3" data-pisa="biorxiv;2020.03.21.001933v1" data-pisa-master="biorxiv;2020.03.21.001933" data-seqnum="1208247" data-apath="/biorxiv/early/2020/03/23/2020.03.21.001933.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vinicio</span> <span class="nlm-surname">Armijos-Jaramillo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Yeager</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Muslin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yunierkis</span> <span class="nlm-surname">Perez-Castillo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.21.001933; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.21.001933 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Armijos-Jaramillo%2C+Vinicio&amp;rft.au=Yeager%2C+Justin&amp;rft.au=Muslin%2C+Claire&amp;rft.au=Perez-Castillo%2C+Yunierkis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1208247/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1208247/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1208247/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1208247/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1208247/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1208247/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1208247/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1208247/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1208247/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1208247/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1208247/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1208247/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;count=horizontal&amp;via=&amp;text=SARS-CoV-2%2C%20an%20evolutionary%20perspective%20of%20interaction%20with%20human%20ACE2%20reveals%20undiscovered%20amino%20acids%20necessary%20for%20complex%20stability&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.001933v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.001933v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
