<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/05/13/320929.atom"/>
  <meta name="HW.pisa" content="biorxiv;320929v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX"/>
  <meta name="DC.Identifier" content="10.1101/320929"/>
  <meta name="DC.Date" content="2018-05-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="During lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells. Significance Statement Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting."/>
  <meta name="DC.Contributor" content="Aaron S. Mendez"/>
  <meta name="DC.Contributor" content="Carolin Vogt"/>
  <meta name="DC.Contributor" content="Jens Bohne"/>
  <meta name="DC.Contributor" content="Britt A. Glaunsinger"/>
  <meta name="article:published_time" content="2018-05-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;During lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction &lt;;i&gt;;in vitro&lt;;/i&gt;; and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells.&lt;;/p&gt;;&lt;;h3&gt;;Significance Statement&lt;;/h3&gt;; &lt;;p&gt;;Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;320929v1"/>
  <meta name="citation_id" content="320929v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/320929v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/320929v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/320929v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929.full.pdf"/>
  <meta name="citation_doi" content="10.1101/320929"/>
  <meta name="citation_num_pages" content="44"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="320929"/>
  <meta name="citation_author" content="Aaron S. Mendez"/>
  <meta name="citation_author_institution" content="Department of Plant &amp;; Microbial Biology, University of California Berkeley"/>
  <meta name="citation_author" content="Carolin Vogt"/>
  <meta name="citation_author_institution" content="Department of Plant &amp;; Microbial Biology, University of California Berkeley"/>
  <meta name="citation_author_institution" content="Hannover Medical School Institute of Virology"/>
  <meta name="citation_author" content="Jens Bohne"/>
  <meta name="citation_author_institution" content="Hannover Medical School Institute of Virology"/>
  <meta name="citation_author" content="Britt A. Glaunsinger"/>
  <meta name="citation_author_institution" content="Department of Plant &amp;; Microbial Biology, University of California Berkeley"/>
  <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California Berkeley"/>
  <meta name="citation_author_institution" content="Howard Hughes Medical Institute"/>
  <meta name="citation_author_email" content="Glaunsinger@berkeley.edu"/>
  <meta name="citation_reference" content="Abernathy E &amp;; Glaunsinger B (2015) Emerging roles for RNA degradation in viral replication and antiviral defense. Virology 479–480:600–608."/>
  <meta name="citation_reference" content="Glaunsinger BA (2015) Modulation of the Translational Landscape During Herpesvirus Infection. Annu Rev Virol 2(1):311–333."/>
  <meta name="citation_reference" content="Gaglia MM, Covarrubias S, Wong W, &amp;; Glaunsinger BA (2012) A common strategy for host RNA degradation by divergent viruses. J Virol 86(17):9527–9530."/>
  <meta name="citation_reference" content="Kamitani W, Huang C, Narayanan K, Lokugamage KG, &amp;; Makino S (2009) A two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein. Nat Struct Mol Biol 16(11):1134–1140."/>
  <meta name="citation_reference" content="Hayashi T, MacDonald LA, &amp;; Takimoto T (2015) Influenza A Virus Protein PA-X Contributes to Viral Growth and Suppression of the Host Antiviral and Immune Responses. J Virol 89(12):6442–6452."/>
  <meta name="citation_reference" content="Rabkin CS, Biggar RJ, &amp;; Horm JW (1991) Increasing incidence of cancers associated with the human immunodeficiency virus epidemic. Int J Cancer 47(5):692–696."/>
  <meta name="citation_reference" content="Kaplan LD (2013) Human herpesvirus-8: Kaposi sarcoma, multicentric Castleman disease, and primary effusion lymphoma. Hematology Am Soc Hematol Educ Program 2013:103–108."/>
  <meta name="citation_reference" content="Glaunsinger B, Chavez L, &amp;; Ganem D (2005) The exonuclease and host shutoff functions of the SOX protein of Kaposi’s sarcoma-associated herpesvirus are genetically separable. J Virol 79(12):7396–7401."/>
  <meta name="citation_reference" content="Dahlroth SL, et al. (2009) Crystal structure of the shutoff and exonuclease protein from the oncogenic Kaposi’s sarcoma-associated herpesvirus. FEBS J 276(22):6636–6645."/>
  <meta name="citation_reference" content="Bagneris C, Briggs LC, Savva R, Ebrahimi B, &amp;; Barrett TE (2011) Crystal structure of a KSHV-SOX-DNA complex: insights into the molecular mechanisms underlying DNase activity and host shutoff. Nucleic Acids Res 39(13):5744–5756."/>
  <meta name="citation_reference" content="Richner JM, et al. (2011) Global mRNA degradation during lytic gammaherpesvirus infection contributes to establishment of viral latency. PLoS Pathog 7(7):e1002150."/>
  <meta name="citation_reference" content="Abernathy E, et al. (2014) Gammaherpesviral gene expression and virion composition are broadly controlled by accelerated mRNA degradation. PLoS Pathog 10(1):e1003882."/>
  <meta name="citation_reference" content="Rowe M, et al. (2007) Host shutoff during productive Epstein-Barr virus infection is mediated by BGLF5 and may contribute to immune evasion. Proc Natl Acad Sci U S A 104(9):3366–3371."/>
  <meta name="citation_reference" content="Lee YJ &amp;; Glaunsinger BA (2009) Aberrant herpesvirus-induced polyadenylation correlates with cellular messenger RNA destruction. PLoS Biol 7(5):e1000107."/>
  <meta name="citation_reference" content="Glaunsinger B &amp;; Ganem D (2004) Lytic KSHV infection inhibits host gene expression by accelerating global mRNA turnover. Mol Cell 13(5):713–723."/>
  <meta name="citation_reference" content="Covarrubias S, et al. (2011) Coordinated destruction of cellular messages in translation complexes by the gammaherpesvirus host shutoff factor and the mammalian exonuclease Xrn1. PLoS Pathog 7(10):e1002339."/>
  <meta name="citation_reference" content="Clyde K &amp;; Glaunsinger BA (2011) Deep sequencing reveals direct targets of gammaherpesvirus-induced mRNA decay and suggests that multiple mechanisms govern cellular transcript escape. PLoS One 6(5):e19655."/>
  <meta name="citation_reference" content="Hutin S, Lee Y, &amp;; Glaunsinger BA (2013) An RNA element in human interleukin 6 confers escape from degradation by the gammaherpesvirus SOX protein. J Virol 87(8):4672–4682."/>
  <meta name="citation_reference" content="Muller M &amp;; Glaunsinger BA (2017) Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases. PLoS Pathog 13(8):e1006593."/>
  <meta name="citation_reference" content="Gaglia MM, Rycroft CH, &amp;; Glaunsinger BA (2015) Transcriptome-Wide Cleavage Site Mapping on Cellular mRNAs Reveals Features Underlying Sequence-Specific Cleavage by the Viral Ribonuclease SOX. PLoS Pathog 11(12):e1005305."/>
  <meta name="citation_reference" content="Lee H, et al. (2017) KSHV SOX mediated host shutoff: the molecular mechanism underlying mRNA transcript processing. Nucleic Acids Res 45(8):4756–4767."/>
  <meta name="citation_reference" content="Laganeckas M, Margelevicius M, &amp;; Venclovas C (2011) Identification of new homologs of PD-(D/E)XK nucleases by support vector machines trained on data derived from profile-profile alignments. Nucleic Acids Res 39(4):1187–1196."/>
  <meta name="citation_reference" content="Zhang J, McCabe KA, &amp;; Bell CE (2011) Crystal structures of lambda exonuclease in complex with DNA suggest an electrostatic ratchet mechanism for processivity. Proc Natl Acad Sci U S A 108(29):11872–11877."/>
  <meta name="citation_reference" content="Mordasini T, Curioni A, &amp;; Andreoni W (2003) Why do divalent metal ions either promote or inhibit enzymatic reactions? The case of BamHI restriction endonuclease from combined quantum-classical simulations. J Biol Chem 278(7):4381–4384."/>
  <meta name="citation_reference" content="Korennykh AV, et al. (2011) Cofactor-mediated conformational control in the bifunctional kinase/RNase Ire1. BMC Biol 9:48."/>
  <meta name="citation_reference" content="Morton DJ, et al. (2018) The RNA exosome and RNA exosome-linked disease. RNA 24(2):127–142."/>
  <meta name="citation_reference" content="Gillespie A, Gabunilas J, Jen JC, &amp;; Chanfreau GF (2017) Mutations of EXOSC3/Rrp40p associated with neurological diseases impact ribosomal RNA processing functions of the exosome in S. cerevisiae. RNA 23(4):466–472."/>
  <meta name="citation_reference" content="Reis FP, Pobre V, Silva IJ, Malecki M, &amp;; Arraiano CM (2013) The RNase II/RNB family of exoribonucleases: putting the ‘Dis’ in disease. Wiley Interdiscip Rev RNA 4(5):607–615."/>
  <meta name="citation_reference" content="Popp MW &amp;; Maquat LE (2018) Nonsense-mediated mRNA Decay and Cancer. Curr Opin Genet Dev 48:44–50."/>
  <meta name="citation_reference" content="Kuzmiak HA &amp;; Maquat LE (2006) Applying nonsense-mediated mRNA decay research to the clinic: progress and challenges. Trends Mol Med 12(7):306–316."/>
  <meta name="citation_reference" content="Hui E, et al. (2017) T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition. Science 355(6332):1428–1433."/>
  <meta name="citation_reference" content="Korennykh AV, et al. (2009) The unfolded protein response signals through high-order assembly of Ire1. Nature 457(7230):687–693."/>
  <meta name="citation_reference" content="Regulski EE &amp;; Breaker RR (2008) In-line probing analysis of riboswitches. Methods Mol Biol 419:53–67."/>
  <meta name="twitter:title" content="Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="During lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells. Significance Statement Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting."/>
  <meta name="og-title" property="og:title" content="Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/320929v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="During lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells. Significance Statement Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-05-13"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-98270 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">During lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction 
     <em>in vitro</em> and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells.
    </p>
    <div id="sec-1" class="subsection">
     <p id="p-3">
      <strong>Significance Statement</strong> Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting.
     </p>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Introduction</h2>
    <p id="p-4">Viral infection dramatically reshapes the gene expression landscape of the host cell. By changing overall messenger RNA (mRNA) abundance or translation, viruses can redirect host machinery towards viral gene expression while simultaneously dampening immune stimulatory signals (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Suppression of host gene expression, termed host shutoff, can occur via a variety of mechanisms, but one common strategy is to accelerate degradation of mRNA (
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). This occurs during infection with DNA viruses such as alphaherpesviruses, gammaherpesvirues, and vaccinia virus, as well as with RNA viruses such as influenza A virus and SARS and MERS coronaviruses (
     <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). In the majority of these cases, a viral factor promotes endonucleolytic cleavage of target mRNAs. This strategy bypasses the normally rate limiting steps of deadenylation and decapping to effect rapid mRNA degradation by host exonucleases (
     <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>).
    </p>
    <p id="p-5">Virally encoded host shutoff endonucleases are usually specific for mRNA, yet broad-acting in that they target the majority of the mRNA population. This is exemplified by herpesviral nucleases, including the SOX endonuclease encoded by Kaposi’s sarcoma-associated herpesvirus (KSHV), an oncogenic human gammaherpesvirus that causes Kaposi’s sarcoma and B cell lymphoproliferative diseases (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, 
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). KSHV SOX is a member of the PD-(D/E)xK type II restriction endonuclease superfamily that possesses mechanistically distinct DNase and RNase activities (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). The RNase activity of the gammaherpesvirus SOX protein has been shown to play key roles in various aspects of the viral lifecycle, including immune evasion, cell type specific replication, and controlling the gene expression landscape of infected cells (
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). However, the mechanism by which SOX targets mRNAs remains largely unknown.
    </p>
    <p id="p-6">Sequencing data indicate that within the mRNA pool there appears to be a range of SOX targeting efficiencies; some transcripts are efficiently cleaved in cells, while others are partially or fully refractory to cleavage (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>–
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Additionally, SOX has been shown to cut within specific locations of mRNAs in cells, further emphasizing that there must be transcript features that confer selectivity (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, 
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Indeed, a transcriptome-wide cleavage analysis indicated that SOX targeting is directed by a relatively degenerate motif, often containing an unpaired polyadenosine stretch shortly upstream of the cleavage site, which is located in a loop structure (
     <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>). Cleavage within an unpaired loop was confirmed in a recent crystal structure of SOX with RNA, although additional contacts that could confer sequence specificity were not observed (
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>).
    </p>
    <p id="p-7">Thus, a major outstanding question is how RNA sequence and/or structure contribute to SOX target recognition. In this context, it is unclear how sequence features surrounding the RNA cleavage site might impact SOX targeting, for example by changing its affinity for a given RNA or the efficiency with which cleavage occurs. To address these questions, we sought to reconstitute the SOX cleavage reaction 
     <em>in vitro</em> using purified components. Using an RNA substrate that is efficiently cleaved by SOX in cells, we revealed that specific RNA sequences within and outside of the cleavage site significantly contribute to SOX binding efficiency and target processing. In particular, we found that the polyadenosine stretch previously predicted to be important is critical for SOX binding, and we experimentally verified the importance of an open loop structure surrounding the cleavage site. Finally, we demonstrated that this 
     <em>in vitro</em> system faithfully recapitulates the initial endonucleolytic cleavage event that is an essential component of mRNA target specificity 
     <em>in vivo.</em> Collectively, our data reveal that specific sequence features potently impact SOX binding, and thus provide key insight into the breadth of SOX targeting efficiency observed across the transcriptome. More broadly, this information provides a framework for better understanding the target specificity of endonucleases, which play central roles in mammalian quality control processes and viral infection outcomes.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results</h2>
    <div id="sec-4" class="subsection">
     <h3>KSHV SOX cleaves RNA substrates endonucleolytically as a monomer</h3>
     <p id="p-8">In cells, the mRNA fragments resulting from the primary SOX endonucleolytic cleavage are predominantly cleared by the host 5’-3’ exonuclease XRN1, while 
      <em>in vitro,</em> RNA fragments are rapidly degraded by 5’-3’ exonucleolytic activity intrinsic to purified SOX (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Thus, it has been challenging to analyze the initial endonucleolytic cleavage event that is an essential component of mRNA target specificity 
      <em>in vivo.</em> Here, we sought to develop a biochemical system to address these questions.
     </p>
     <p id="p-9">Our prior analysis of SOX targets in cells identified the human 
      <em>LIMD1</em> mRNA, which codes for a protein essential for P body formation and integrity, as being highly susceptible to cleavage by SOX (
      <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>). The minimum sequence required to directly cut the putative cleavage site in 
      <em>LIMD1</em> in cells was mapped to a 54 nucleotide segment (
      <em>LIMD1</em> 54), and we therefore chose this as our model substrate to study SOX targeting 
      <em>in vitro</em> (
      <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>). We first expressed and purified KSHV SOX to greater then 95 percent purity from SF9 insect cells (
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">fig. S1
       <em>A</em>
      </a>). Using the 
      <em>LIMD1</em> 54 substrate, we plotted the observed rate constant (
      <em>k</em>
      <sub>obs</sub>) as a function of SOX concentration, yielding a Hill coefficient of 
      <em>n = 1.11</em> (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1
       <em>A</em>
      </a>). Thus, in agreement with previous observations (
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>), SOX appears to function predominantly as a monomer. Under conditions of half maximal activity (2 μM; 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1
       <em>A</em>
      </a>), SOX displayed a strong preference for the “hard” divalent metal Mg
      <sup>2+</sup> and a weaker preference for the “softer” and larger metals Mn
      <sup>2+</sup>, Co
      <sup>2+</sup>, and Zn
      <sup>2+</sup> (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1
       <em>B</em>
      </a>). This is again consistent with other characterized members of the P/DExK family of enzymes (
      <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Notably, SOX activity in the presence of Mg
      <sup>2+</sup> was inhibited in a dose-dependent manner upon competitive addition of Ca
      <sup>2+</sup> (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1
       <em>C</em>
      </a> and 
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">S1
       <em>B</em>
      </a>). This is likely the result of increased coordination partners engaged by Ca
      <sup>2+</sup>, which decreases the ability of catalytic residues to promote proper base hydrolysis (
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). Finally, increasing the NaCl concentration above 100 mM led to substantially decreased SOX activity (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>), in accordance with the observation that high salt concentrations frequently inhibit nuclease activity by disrupting protein-protein or protein-substrate interactions (
      <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>). Given that recombinant SOX displays robust 5’-3’ exonuclease activity (
      <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>), we sought to confirm that 
      <em>LIMD1</em> 54 was subject to endonucleolytic SOX cleavage, as this is the predominant event that directs mRNA turnover in SOX expressing cells (
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, 
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). Both the 5’ and 3’ ends 
      <em>LIMD1</em> 54 were blocked by capping the 5’ end with a Cy5 fluorophore and the 3’ end with an Iowa Black quencher (
      <em>LIMD1</em> 54 Flo). We confirmed this RNA was resistant to degradation by the 5’-phosphate dependent exonuclease terminator (
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1
       <em>E</em>
      </a>, lane 3). However, in the presence of SOX, a cleavage product was observed that correlated with an endonucleolytic cut (
      <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1
       <em>E</em>
      </a>, lane 2).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Kinetic characterization of recombinant SOX. (A) The observed rate constant (kobs) was plotted as a function of SOX concentration using the 5[[#x2019]] 32P-labeled LIMD1 54 RNA substrate, showing a hill coefficient (n) of 1.11. (B) The catalytic efficiency of SOX (2[[#x3BC]]M) in the presence of MgCl2, MnCl2, CoCl2, and ZnCl2 was plotted as a function of cofactor concentration. (C) The impact of adding increasing concentrations of CaCl2 on SOX-induced degradation of a 5[[#x2019]] 32P-labeled LIMD1 54 RNA probe. Reactions were carried out in the presence of 0.7 mM MgCl2. (D) SOX catalytic efficiency was determined under increasing concentrations of NaCl. (E) The 5[[#x2019]] and 3[[#x2019]] ends of LIMD 54 were blocked with Cy5 and Iowa Black, respectively, to prevent exonucleolytic degradation. Reactions were incubated for 30 min in the presence of SOX or, as a control, the Terminator 5[[#x2019]] exonuclease (EXO). Input refers to RNA in reaction buffer without enzyme." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Kinetic characterization of recombinant SOX.&lt;;/span&gt;; (A) The observed rate constant (&lt;;em&gt;;k&lt;;/em&gt;;&lt;;sub&gt;;obs&lt;;/sub&gt;;) was plotted as a function of SOX concentration using the 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; RNA substrate, showing a hill coefficient (&lt;;em&gt;;n&lt;;/em&gt;;) of 1.11. (B) The catalytic efficiency of SOX (2[[#x3BC]]M) in the presence of MgCl&lt;;sub&gt;;2&lt;;/sub&gt;;, MnCl&lt;;sub&gt;;2&lt;;/sub&gt;;, CoCl&lt;;sub&gt;;2&lt;;/sub&gt;;, and ZnCl&lt;;sub&gt;;2&lt;;/sub&gt;; was plotted as a function of cofactor concentration. (C) The impact of adding increasing concentrations of CaCl&lt;;sub&gt;;2&lt;;/sub&gt;; on SOX-induced degradation of a 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; RNA probe. Reactions were carried out in the presence of 0.7 mM MgCl&lt;;sub&gt;;2&lt;;/sub&gt;;. (D) SOX catalytic efficiency was determined under increasing concentrations of NaCl. (E) The 5[[#x2019]] and 3[[#x2019]] ends of &lt;;em&gt;;LIMD 54&lt;;/em&gt;; were blocked with Cy5 and Iowa Black, respectively, to prevent exonucleolytic degradation. Reactions were incubated for 30 min in the presence of SOX or, as a control, the Terminator 5[[#x2019]] exonuclease (EXO). Input refers to RNA in reaction buffer without enzyme.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F1.medium.gif" width="364" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F1.medium.gif" width="364" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1.</span>
       <span class="caption-title">Kinetic characterization of recombinant SOX.</span>
       <p id="p-10" class="first-child">(A) The observed rate constant (
        <em>k</em>
        <sub>obs</sub>) was plotted as a function of SOX concentration using the 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> RNA substrate, showing a hill coefficient (
        <em>n</em>) of 1.11. (B) The catalytic efficiency of SOX (2μM) in the presence of MgCl
        <sub>2</sub>, MnCl
        <sub>2</sub>, CoCl
        <sub>2</sub>, and ZnCl
        <sub>2</sub> was plotted as a function of cofactor concentration. (C) The impact of adding increasing concentrations of CaCl
        <sub>2</sub> on SOX-induced degradation of a 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> RNA probe. Reactions were carried out in the presence of 0.7 mM MgCl
        <sub>2</sub>. (D) SOX catalytic efficiency was determined under increasing concentrations of NaCl. (E) The 5’ and 3’ ends of 
        <em>LIMD 54</em> were blocked with Cy5 and Iowa Black, respectively, to prevent exonucleolytic degradation. Reactions were incubated for 30 min in the presence of SOX or, as a control, the Terminator 5’ exonuclease (EXO). Input refers to RNA in reaction buffer without enzyme.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">To confirm this processing event was not a result of contamination, we purified a SOX mutant containing mutations within two key residues of the SOX active site (D221N/E244Q). Incubation of this mutant with 
      <em>LIMD1</em> 54 over the course of 1.5 h yielded no RNA cleavage (
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. S1C</a>). Thus, recombinant SOX appears to target 
      <em>LIMD1</em> 54 for endonucleolytic cleavage 
      <em>in vitro</em>, as has been observed for this substrate in cells.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>KSHV SOX shows RNA substrate selectivity 
      <em>in vitro</em>
     </h3>
     <p id="p-12">To analyze RNA substrate selectivity using our 
      <em>in vitro</em> assay, we first compared SOX degradation of 
      <em>LIMD1</em> 54 to a 51-nucleotide sequence of the mRNA encoding GFP (
      <em>GFP</em> 51). We have previously shown that 
      <em>GFP</em> mRNA is cleaved by SOX in cells, and that 
      <em>GFP</em> 51 is the minimal sequence required to elicit cleavage (
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, 
      <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>). The cleavage sites for 
      <em>LIMD1</em> 54 and 
      <em>GFP 51</em> are predicted to occur in an open loop region (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2
       <em>A</em>
      </a>, red arrow). Upon direct comparison of these two RNAs, we observed a 5-fold increase in the catalytic efficiency of SOX for the 
      <em>LIMD1</em> 54 substrate compared to 
      <em>GFP 51</em> (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2
       <em>B</em>
      </a>). This difference was not exclusively due to the fact that the GFP substrate was slightly shorter than 
      <em>LIMD1 54,</em> as SOX also displayed a 2.5-fold reduction of catalytic efficiency on a longer, 100 nt GFP substrate (
      <em>GFP</em> 100; 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2
       <em>B</em>
      </a>). Electrophoretic mobility shift assays (EMSA) further revealed a 10-fold increase in SOX binding to 
      <em>LIMD1</em> 54 compared to 
      <em>GFP</em> 51 (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2C</a>). Given that both substrates contain the requisite unpaired bulge at the predicted cleavage site (see 
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2
       <em>A</em>
      </a>), these observations suggest that additional sequence or structural features impact SOX targeting efficiency on individual RNAs.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Substrate specificity, salt concentration and the bridge motif play important roles in SOX activity. (A) The predicted folding of the LIMD1 54 and GFP 51 RNAs was determined using mFold [Cite Nucleic Acids Res. 31 (13), 3406[[#x2013]]15, (2003)]. Red arrows mark the predicted SOX cleavage site. (B) Catalytic efficiencies were determined for SOX (2 [[#x3BC]]M) in the presence of GFP 51, GFP 100 or LIMD1 54 substrate. Reactions were performed in triplicate. (C) Binding curves of SOX with GFP 51 and LIMD1 54 RNA. Percent binding of substrates was determined by EMSA, whereupon curves were fit to a single binding model from three independent measurements. (D) Catalytic efficiency of WT SOX or the host shutoff mutants P176S and F179A was determined at a constant enzyme concentration (2 [[#x3BC]]M) using a 5[[#x2019]] 32P-labeled LIMD1 54 RNA probe. Experiments were performed in triplicate. (E) EMSAs were used to determine percent binding of 5[[#x2019]] 32P-labeled LIMD1 54 RNA probe to WT SOX, P176S, and F179A. Curves were fit to a single binding model from three independent measurements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Substrate specificity, salt concentration and the bridge motif play important roles in SOX activity.&lt;;/span&gt;; (A) The predicted folding of the &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 and &lt;;em&gt;;GFP&lt;;/em&gt;; 51 RNAs was determined using mFold [Cite &lt;;em&gt;;Nucleic Acids Res.&lt;;/em&gt;; &lt;;strong&gt;;31 (13)&lt;;/strong&gt;;, 3406[[#x2013]]15, (2003)]. Red arrows mark the predicted SOX cleavage site. (B) Catalytic efficiencies were determined for SOX (2 [[#x3BC]]M) in the presence of &lt;;em&gt;;GFP&lt;;/em&gt;; 51, &lt;;em&gt;;GFP&lt;;/em&gt;; 100 or &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 substrate. Reactions were performed in triplicate. (C) Binding curves of SOX with &lt;;em&gt;;GFP&lt;;/em&gt;; 51 and &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 RNA. Percent binding of substrates was determined by EMSA, whereupon curves were fit to a single binding model from three independent measurements. (D) Catalytic efficiency of WT SOX or the host shutoff mutants P176S and F179A was determined at a constant enzyme concentration (2 [[#x3BC]]M) using a 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; RNA probe. Experiments were performed in triplicate. (E) EMSAs were used to determine percent binding of 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; RNA probe to WT SOX, P176S, and F179A. Curves were fit to a single binding model from three independent measurements.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F2.medium.gif" width="368" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F2.medium.gif" width="368" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2.</span>
       <span class="caption-title">Substrate specificity, salt concentration and the bridge motif play important roles in SOX activity.</span>
       <p id="p-13" class="first-child">(A) The predicted folding of the 
        <em>LIMD1</em> 54 and 
        <em>GFP</em> 51 RNAs was determined using mFold [Cite 
        <em>Nucleic Acids Res.</em>
        <strong>31 (13)</strong>, 3406–15, (2003)]. Red arrows mark the predicted SOX cleavage site. (B) Catalytic efficiencies were determined for SOX (2 μM) in the presence of 
        <em>GFP</em> 51, 
        <em>GFP</em> 100 or 
        <em>LIMD1</em> 54 substrate. Reactions were performed in triplicate. (C) Binding curves of SOX with 
        <em>GFP</em> 51 and 
        <em>LIMD1</em> 54 RNA. Percent binding of substrates was determined by EMSA, whereupon curves were fit to a single binding model from three independent measurements. (D) Catalytic efficiency of WT SOX or the host shutoff mutants P176S and F179A was determined at a constant enzyme concentration (2 μM) using a 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> RNA probe. Experiments were performed in triplicate. (E) EMSAs were used to determine percent binding of 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> RNA probe to WT SOX, P176S, and F179A. Curves were fit to a single binding model from three independent measurements.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-14">Two SOX point mutants, P176S and F179A, located in an unstructured region of the protein that bridges domains I and II have been shown to be selectively required for its endonucleolytic processing of RNA substrates (
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig S2
       <em>A</em>
      </a> and 
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">S2
       <em>B</em>
      </a>) (
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, 
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>). Structural data indicate that residue F179 forms a stacking interaction with an adenine base in the RNA, likely stabilizing the protein-RNA interaction, while P176 is hypothesized to contribute to structural rearrangements required for F179 engagement (
      <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>). We purified both mutants to evaluate their relative RNA processing and RNA binding activity against the optimal 
      <em>LIMD1 54</em> substrate. Both mutants displayed purity and elution profiles similar to wild type (WT) SOX (see 
      <a id="xref-fig-8-4" class="xref-fig" href="#F8">Fig. S1A</a>). However, the catalytic efficiency of each mutant was &gt;;5-fold less than WT SOX (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2D</a>). Furthermore, RNA binding was severely perturbed; the binding kinetics of WT SOX for 
      <em>LIMD1</em> 54 are in the single digit nanomolar range (K
      <sub>d</sub> = 7 nM), while P176S and F179A display &gt;;2 log defects (K
      <sub>d</sub> = 702 nM and 831 nM, respectively) (
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 
       <em>2E</em>
      </a> and 
      <a id="xref-fig-9-3" class="xref-fig" href="#F9">S3
       <em>A-C</em>
      </a>). Thus, the large defect in RNA binding likely explains the decreased efficiency of RNA processing. Notably, while there was a dramatic decrease in the relative affinities of the two mutants for 
      <em>LIMD1</em> 54, there was not a complete loss of binding or RNA processing. This could be a result of secondary nonspecific interactions and/or nonspecific exonucleolytic degradation by SOX from the 5’ monophosphorylated end of the probe.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Secondary structure determination of the 
      <em>LIMD1</em> 54 substrate
     </h3>
     <p id="p-15">
      <em>In silico</em> RNA folding predictions of SOX targeting motifs, coupled with RNA mutagenesis experiments, have indicated that an RNA stem loop structure is an important determinant in SOX targeting both 
      <em>in vitro</em> and 
      <em>in vivo</em> (
      <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">20</a>, 
      <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>). Given the importance of this predicted motif, and in particular the proposed requirement for unpaired sequence at the cut site, we sought to experimentally determine the structure of 
      <em>LIMD1</em> 54 using chemical based in-line probing (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). This showed that the 
      <em>LIMD1</em> 54 structure contains a largely base paired stem region, followed by a loop at positions 15–27 that encompasses the predicted SOX cleavage site between nt 26 and 27, and a short hairpin structure at positions 29–40 (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). Notably, some differences exist between the predicted and observed structures of 
      <em>LIMD1</em> 54, including a larger loop region and the subsequent short stem-loop (compare 
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a> to 
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2A</a>) However, in both cases the predicted cleavage site of SOX resides in a loop region.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="In-line structure probing of LIMD1 54. (A) An in-line reaction (Rxn) was performed at room temperature for 24 h (lane 2) or 48 h (lane 3) at pH 8.3 to identify structured regions of the LIMD1 54 RNA. Ladders were generated by subjecting the RNA to cleavage by RNase T1 (lane 8) or alkaline hydrolysis (-OH, lane 4[[#x2013]]7). Products were separated by 8% urea PAGE, whereupon structured regions protected from cleavage were identified (green bars). No reaction (NR, lane1) refers to the input RNA. (B) Diagram showing the LIMD1 54 structure as deduced from the in-line probing gel. Green color corresponds to the structured regions denoted by the bars in (A), while blue refers to unstructured regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;In-line structure probing of &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54.&lt;;/span&gt;; (A) An in-line reaction (Rxn) was performed at room temperature for 24 h (lane 2) or 48 h (lane 3) at pH 8.3 to identify structured regions of the &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 RNA. Ladders were generated by subjecting the RNA to cleavage by RNase T1 (lane 8) or alkaline hydrolysis (-OH, lane 4[[#x2013]]7). Products were separated by 8% urea PAGE, whereupon structured regions protected from cleavage were identified (green bars). No reaction (NR, lane1) refers to the input RNA. (B) Diagram showing the &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; structure as deduced from the in-line probing gel. Green color corresponds to the structured regions denoted by the bars in (A), while blue refers to unstructured regions.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F3.medium.gif" width="440" height="344"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F3.medium.gif" width="440" height="344"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3.</span>
       <span class="caption-title">In-line structure probing of 
        <em>LIMD1</em> 54.
       </span>
       <p id="p-16" class="first-child">(A) An in-line reaction (Rxn) was performed at room temperature for 24 h (lane 2) or 48 h (lane 3) at pH 8.3 to identify structured regions of the 
        <em>LIMD1</em> 54 RNA. Ladders were generated by subjecting the RNA to cleavage by RNase T1 (lane 8) or alkaline hydrolysis (-OH, lane 4–7). Products were separated by 8% urea PAGE, whereupon structured regions protected from cleavage were identified (green bars). No reaction (NR, lane1) refers to the input RNA. (B) Diagram showing the 
        <em>LIMD1 54</em> structure as deduced from the in-line probing gel. Green color corresponds to the structured regions denoted by the bars in (A), while blue refers to unstructured regions.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>SOX binds to an unpaired stretch of adenosine repeats</h3>
     <p id="p-17">Recently, a high-resolution crystal structure was solved of SOX bound to a 31nt fragment of the KSHV pre-microRNA K12–2 (K2–31). In this structure, the only observed contacts between SOX and K2–31 occurred between the four active site residues of SOX (Y373, R248, C247, F179) and the UGAAG motif surrounding the cleavage site of the RNA (
      <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">21</a>). It was therefore hypothesized that no other residues beyond this unpaired UGAAG motif were involved in transcript recognition (
      <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">21</a>). However, our data showing a 10-fold difference in binding affinity between two 
      <em>in vivo</em> validated SOX substrates suggested that a more extended interaction surface might distinguish optimal from sub-optimal RNA substrates. We therefore used RNA footprinting to map the SOX binding sites on 
      <em>LIMD1 54.</em> Indeed, SOX robustly protected 3 
      <em>LIMD1 54</em> adenosines (positions 20–24) from RNase I digestion in a dose dependent manner (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). Notably, this mapped binding site is the same one predicted from 
      <em>in vivo</em> PARE-seq data (
      <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">20</a>). We also observed a modest protection of base 27 (G) located directly adjacent to the predicted cleavage site of SOX, which represents the region detected in the crystal structure of K2–31 bound to SOX. Collectively, these findings suggest that while SOX may interact with residues directly adjacent to the cut site, a more extensive interaction interface exists for its preferred 
      <em>in vivo</em> targets.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SOX binds to a stretch of adenosines upstream of the cleavage site (A) An RNA footprinting assay was carried out by incubating 5[[#x2019]] 32P-labeled LIMD1 54 with RNase I in the presence (lanes 3[[#x2013]]7) or absence (lane 2) of a dilution series of SOX (8[[#x2013]]0.5 [[#x3BC]]M). Hydrolysis (-OH, lane 1) and RNase T1 (T1, lane 8) ladders of the RNA were also generated in order to map the location of protected sites. (B) Diagram of LIMD1 54 indicating sites protected from RNase I cleavage by SOX. The upstream SOX binding site is colored orange while the protected residues surrounding the cut site are shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SOX binds to a stretch of adenosines upstream of the cleavage site&lt;;/span&gt;; (A) An RNA footprinting assay was carried out by incubating 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; with RNase I in the presence (lanes 3[[#x2013]]7) or absence (lane 2) of a dilution series of SOX (8[[#x2013]]0.5 [[#x3BC]]M). Hydrolysis (-OH, lane 1) and RNase T1 (T1, lane 8) ladders of the RNA were also generated in order to map the location of protected sites. (B) Diagram of &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 indicating sites protected from RNase I cleavage by SOX. The upstream SOX binding site is colored orange while the protected residues surrounding the cut site are shown in red.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F4.medium.gif" width="440" height="433"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F4.medium.gif" width="440" height="433"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 4.</span>
       <span class="caption-title">SOX binds to a stretch of adenosines upstream of the cleavage site</span>
       <p id="p-18" class="first-child">(A) An RNA footprinting assay was carried out by incubating 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> with RNase I in the presence (lanes 3–7) or absence (lane 2) of a dilution series of SOX (8–0.5 μM). Hydrolysis (-OH, lane 1) and RNase T1 (T1, lane 8) ladders of the RNA were also generated in order to map the location of protected sites. (B) Diagram of 
        <em>LIMD1</em> 54 indicating sites protected from RNase I cleavage by SOX. The upstream SOX binding site is colored orange while the protected residues surrounding the cut site are shown in red.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Residues surrounding the SOX binding and cleavage sites contribute to efficient substrate degradation</h3>
     <p id="p-19">To explore the importance of the residues involved in SOX binding and cleavage, we engineered 3 mutants of the 
      <em>LIMD1 54</em> substrate (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>). First, we preserved the loop structure but replaced the 3 adenosines bound by SOX (residues 39–43) with guanosines (
      <em>LIMD1</em> 3xA-G). Second, we mutated the residue located at the predicted SOX cut site that was also protected in the footprinting assay (
      <em>LIMD1</em> A-G). This mutant has been previously identified to block SOX cleavage 
      <em>in vivo</em> (
      <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">20</a>). Third, we largely abolished the loop structure by providing complementary base pairing (
      <em>LIMD1</em> Zipper). The predicted structures of the 
      <em>LIMD1</em> 3XA-G and 
      <em>LIMD1</em> zipper mutants were verified by in-line probing (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a>, 
      <a id="xref-fig-13-1" class="xref-fig" href="#F13">S6</a>). Real-time binding kinetics for SOX with WT 
      <em>LIMD1</em> 54 and each of the 3 mutant substrates were then measured using bio-layer interferometry (BLI). All RNA probes were 3’ biotinylated and immobilized to a streptavidin-coated BLI probe, whereupon the binding and dissociation of SOX was measured. To prevent degradation of the probe, excess calcium ion was used in place of magnesium (
      <a id="xref-fig-8-5" class="xref-fig" href="#F8">Fig. S1B</a>). SOX retained similar binding affinity to the cut site mutant 
      <em>LIMD1</em> A-G (K
      <sub>d</sub> = 25 nM) as to WT 
      <em>LIMD1</em> 54 (K
      <sub>d</sub> = 16.3 nM) (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>, 
      <a id="xref-fig-11-1" class="xref-fig" href="#F11">S4A-B</a> and 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">table S1</a>). In contrast, SOX exhibited dramatically reduced binding to both the predicted binding site mutant 
      <em>LIMD1</em> 3xA-G (K
      <sub>d</sub> = 710 nM) and to the 
      <em>LIMD1</em> zipper mutant lacking the loop region (K
      <sub>d</sub> = 904 nM) (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5B</a>, 
      <a id="xref-fig-11-2" class="xref-fig" href="#F11">S4C-D</a> and 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">table S1</a>). We also measured SOX binding to the KSHV pre-miRNA sequence used to obtain the SOX-RNA co-crystal structure (K2–31) (
      <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">21</a>). Notably, the affinity of SOX for K2–31 was even less than that of the 
      <em>LIMD1</em> 3xA-G and Zipper mutants (K
      <sub>d</sub> = 1.8 μM), suggesting that despite having an UGAAG motif upstream of a predicted bulge, this may not be a bona fide SOX target (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5
       <em>B</em>
      </a>, 
      <a id="xref-fig-11-3" class="xref-fig" href="#F11">S4
       <em>E</em>
      </a> and 
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">table S1</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Mutations that disrupt LIMD1 54 structure reduce SOX binding affinity. (A) Diagram showing the location of mutations made within LIMD1 54. Binding site mutants and cut site mutants are labeled accordingly. (B) The binding affinity of SOX for each of the LIMD1 54 mutants was tested in parallel using Bio-Layer Interferometry (BLI). (C) SOX catalytic efficiency was tested at a constant concentration (2 [[#x3BC]]M) for all of the LIMD1 54 mutant substrates. All assays were performed in triplicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Mutations that disrupt &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 structure reduce SOX binding affinity.&lt;;/span&gt;; (A) Diagram showing the location of mutations made within &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54. Binding site mutants and cut site mutants are labeled accordingly. (B) The binding affinity of SOX for each of the &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 mutants was tested in parallel using Bio-Layer Interferometry (BLI). (C) SOX catalytic efficiency was tested at a constant concentration (2 [[#x3BC]]M) for all of the &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 mutant substrates. All assays were performed in triplicate.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F5.medium.gif" width="440" height="432"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F5.medium.gif" width="440" height="432"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 5.</span>
       <span class="caption-title">Mutations that disrupt 
        <em>LIMD1</em> 54 structure reduce SOX binding affinity.
       </span>
       <p id="p-20" class="first-child">(A) Diagram showing the location of mutations made within 
        <em>LIMD1</em> 54. Binding site mutants and cut site mutants are labeled accordingly. (B) The binding affinity of SOX for each of the 
        <em>LIMD1</em> 54 mutants was tested in parallel using Bio-Layer Interferometry (BLI). (C) SOX catalytic efficiency was tested at a constant concentration (2 μM) for all of the 
        <em>LIMD1</em> 54 mutant substrates. All assays were performed in triplicate.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">We next quantitatively measured the catalytic efficiency of SOX towards each of the above RNA substrates. Despite SOX having WT binding affinity for the predicted cleavage site mutant 
      <em>LIMD1</em> A-G, there was a 4-fold defect in its ability to degrade this mutant (
      <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5C</a>). Even more marked defects in SOX catalytic efficiency were observed for the binding site mutant 
      <em>LIMD1</em> 3XA-G, the loop mutant 
      <em>LIMD1</em> zipper, and the pre-miRNA K2–31 (
      <a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5C</a>). Collectively, these data indicate that efficient RNA cleavage requires both an appropriate SOX binding site and a suitable cut site.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Site-specific endonucleolytic cleavage of target RNA occurs in vitro</h3>
     <p id="p-22">In cells, SOX cleaves its mRNA substrates site-specifically. Mutagenesis of residues in mapped cleavage sites generally abolishes SOX cleavage at that location (
      <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">20</a>). To determine if our 
      <em>in vitro</em> assay faithfully recapitulated the site specificity of SOX endonucleolytic targeting observed in cells, we established reaction conditions that enabled trapping of the early cleavage events. By combining Ca
      <sup>2+</sup> and Mg
      <sup>2+</sup> in our reaction buffer, we were able to sufficiently slow SOX processing to visualize cleavage products derived from 5’ 
      <sup>32</sup>P labeled substrates. Indeed, we observed a predominant 27 nt band, which is the size of the product released upon 
      <em>LIMD1 54</em> cleavage at the predicted cut site (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6
       <em>A</em>
      </a>, lane 3). Additional bands also appeared, likely representing subsequent processing events. Importantly, when we incubate SOX with the cut site mutant 
      <em>LIMD1 A-G,</em> there is a complete loss of this 27 nt product, as well as the additionally processed intermediates (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6A</a>, lane 4). Production of these cleavage intermediates required SOX, as no decay was observed in the RNA-only controls (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6A</a>, lanes 1–2).
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SOX endonucleolytically cleaves LIMD1 54 in a site-specific manner. (A) 5[[#x2019]] 32P-labeled LIMD1 54 and LIMD1 54 A_G were incubated for 10 min in 0.1 mM CaCl2 and 0.7 mM MgCl2 in the presence or absence of 2 [[#x3BC]]M SOX. The 5[[#x2019]] cleavage product with a size corresponding to cleavage at the mapped cut site (27 nt) is shown. (B) Cleavage assay was performed as in (A), except using 3[[#x2019]] 32P pCp-labeled LIMD1 54. The 3[[#x2019]] cleavage product with a size corresponding to cleavage at the mapped cut site (28 nt) is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SOX endonucleolytically cleaves &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54 in a site-specific manner.&lt;;/span&gt;; (A) 5[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P-labeled &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; and &lt;;em&gt;;LIMD1 54&lt;;/em&gt;; A_G were incubated for 10 min in 0.1 mM CaCl&lt;;sub&gt;;2&lt;;/sub&gt;; and 0.7 mM MgCl&lt;;sub&gt;;2&lt;;/sub&gt;; in the presence or absence of 2 [[#x3BC]]M SOX. The 5[[#x2019]] cleavage product with a size corresponding to cleavage at the mapped cut site (27 nt) is shown. (B) Cleavage assay was performed as in (A), except using 3[[#x2019]] &lt;;sup&gt;;32&lt;;/sup&gt;;P pCp-labeled &lt;;em&gt;;LIMD1&lt;;/em&gt;; 54. The 3[[#x2019]] cleavage product with a size corresponding to cleavage at the mapped cut site (28 nt) is shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F6.medium.gif" width="440" height="342"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F6.medium.gif" width="440" height="342"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 6.</span>
       <span class="caption-title">SOX endonucleolytically cleaves 
        <em>LIMD1</em> 54 in a site-specific manner.
       </span>
       <p id="p-23" class="first-child">(A) 5’ 
        <sup>32</sup>P-labeled 
        <em>LIMD1 54</em> and 
        <em>LIMD1 54</em> A_G were incubated for 10 min in 0.1 mM CaCl
        <sub>2</sub> and 0.7 mM MgCl
        <sub>2</sub> in the presence or absence of 2 μM SOX. The 5’ cleavage product with a size corresponding to cleavage at the mapped cut site (27 nt) is shown. (B) Cleavage assay was performed as in (A), except using 3’ 
        <sup>32</sup>P pCp-labeled 
        <em>LIMD1</em> 54. The 3’ cleavage product with a size corresponding to cleavage at the mapped cut site (28 nt) is shown.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-24">Finally, we sought to verify that the predominant 27 nt cleavage product we observed was a result of an endonucleolytic cleavage and not 5’ end processing. To this end, we generated a 
      <em>LIMD1</em> 54 substrate containing a 3’ 
      <sup>32</sup>P pCp label and a free 5’ OH to block 5’ end processing. Again, in the presence of SOX, WT 
      <em>LIMD1 54</em> but not the A-G mutant produced a cleavage product whose size corresponded to cleavage at the predicted site (
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig 6
       <em>B</em>
      </a>). Taken together, these data confirm that our 
      <em>in vitro</em> assay faithfully recapitulates SOX cleavage site specificity on a true substrate.
     </p>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Discussion</h2>
    <p id="p-25">Endonuclease-directed mRNA degradation plays key roles in the lifecycle of gammaherpesviruses, yet the fundamental principles governing target specificity by SOX are not well understood. Here, through the development of the first biochemical system to faithfully recapitulate the internal cleavage specificity observed for SOX in cells, we revealed how both RNA sequence and structure contribute to targeting. These findings resolve a central feature of the current model of SOX activity (
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Previous observations established that sequences flanking the cut site were required to direct cleavage by SOX (
     <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>, 
     <a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">20</a>). However, it was unresolved whether they played a strictly structural role in presenting an exposed loop for cleavage, served as a platform for SOX binding, or created a binding site for one or more cellular factors that then indirectly recruited SOX to its targets. Through a combination of mutational analyses, RNA structure probing, and RNA footprinting assays, we showed that efficient SOX targeting requires both an exposed loop structure and upstream sequences that serve as a SOX binding platform. This combination of sequence and structural features within the targeting motif helps explain why some mRNAs are efficiently cleaved by SOX, whereas others are weaker substrates.
    </p>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Model of mRNA targeting by SOX. SOX is able to distinguish mRNA from other types of RNA in cells by an as yet unknown mechanism. Subsequently, it endonucleolytically cleaves its targets at specific sites, whereupon the fragments are degraded by host exonucleases such as Xrn1 and Dis3L2. Here, we revealed that in addition to the requirement for an unpaired loop at the cleavage site, additional upstream RNA sequences increase the affinity of SOX for individual targets, thereby controlling cleavage efficiency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15057050" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Model of mRNA targeting by SOX.&lt;;/span&gt;; SOX is able to distinguish mRNA from other types of RNA in cells by an as yet unknown mechanism. Subsequently, it endonucleolytically cleaves its targets at specific sites, whereupon the fragments are degraded by host exonucleases such as Xrn1 and Dis3L2. Here, we revealed that in addition to the requirement for an unpaired loop at the cleavage site, additional upstream RNA sequences increase the affinity of SOX for individual targets, thereby controlling cleavage efficiency.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F7.medium.gif" width="440" height="176"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F7.medium.gif" width="440" height="176"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/13/320929/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 7.</span>
      <span class="caption-title">Model of mRNA targeting by SOX.</span>
      <p id="p-26" class="first-child">SOX is able to distinguish mRNA from other types of RNA in cells by an as yet unknown mechanism. Subsequently, it endonucleolytically cleaves its targets at specific sites, whereupon the fragments are degraded by host exonucleases such as Xrn1 and Dis3L2. Here, we revealed that in addition to the requirement for an unpaired loop at the cleavage site, additional upstream RNA sequences increase the affinity of SOX for individual targets, thereby controlling cleavage efficiency.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-27">A key open question related to SOX function is how it can target the majority of mRNAs in cells, yet with significant site specificity. These observations suggest that there must be specific mRNA features that influence targeting. Indeed, PARE-seq analyses of cleavage intermediates in SOX expressing cells revealed that cleavage sites were associated with a degenerate sequence motif (
     <a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">20</a>). Sequences proximal to the cleavage site were predicted to be un-base paired and frequently contained a polyadenosine stretch followed by a purine (
     <a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">20</a>). The importance of these sequence features for SOX targeting was validated for the 
     <em>LIMD1</em> transcript in cells (
     <a id="xref-ref-20-12" class="xref-bibr" href="#ref-20">20</a>). Because 
     <em>LIMD1</em> has been established as a particularly robust SOX target in cells (
     <a id="xref-ref-20-13" class="xref-bibr" href="#ref-20">20</a>), we reasoned that it must contain features optimal for SOX processing and therefore would be an ideal substrate to dissect these features biochemically. Indeed, SOX binding to 
     <em>LIMD1</em> 54 was 10-fold better than to the commonly used reporter substrate GFP, and an order of magnitude better than to the K2–31 pre-miRNA, which has not been demonstrated to be processed by SOX in cells. Importantly, these binding differences correlated with the efficiency of SOX cleavage 
     <em>in vitro,</em> arguing that the ability to bind the targeting motif is a key step in target recognition. Through RNA footprinting assays, we were able to show that SOX binds to a bulge structure proximal to the cleavage site containing the polyadenosine stretch previously predicted to be important (
     <a id="xref-ref-20-14" class="xref-bibr" href="#ref-20">20</a>). Mutating either the bulge structure (
     <em>LIMD1</em> zipper) or maintaining the bulge but mutating the polyadenosine stretch (
     <em>LIMD1</em> 3xA-G) resulted in an order of magnitude reduction in binding affinity, correlating with a dramatic decrease in cleavage efficiency. Collectively, these data demonstrate that variability in the efficiency of SOX targeting observed in cells is likely due to differences in RNA sequences that mediate SOX binding.
    </p>
    <p id="p-28">A recent crystal structure of SOX bound to the K2–31 pre-miRNA captured the importance of the exposed loop region for SOX cleavage (
     <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">21</a>). However, the structure did not reveal additional interactions between SOX and the RNA beyond the 3 residues surrounding the cut site. Our data suggest that this is likely because the K2–31 RNA lacks the additional residues necessary for SOX binding site found in both 
     <em>LIMD1</em> and 
     <em>GFP.</em> While the K2–31 RNA does contain adenosines upstream of the cleavage site, structural predictions indicate these residues are within a stem region (
     <a id="xref-ref-21-9" class="xref-bibr" href="#ref-21">21</a>), rather than in an exposed loop as is the case for 
     <em>LIMD1</em> and 
     <em>GFP.</em> Together, these observations indicate that while upstream adenosines are important for binding, they must be present in an unpaired state to promote SOX binding. It is notable that prior studies reported much weaker interactions between SOX and RNA (K
     <sub>d</sub> = 75 μM) compared to its DNA substrates (K
     <sub>d</sub> = 1 μm) (
     <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">9</a>, 
     <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>, 
     <a id="xref-ref-21-10" class="xref-bibr" href="#ref-21">21</a>). However, in these cases binding assays were conducted with scrambled RNA sequences. We found that SOX binding affinities to RNA substrates vary over several orders of magnitude, in a manner that correlates with cleavage efficiency. Interestingly, the crystal structure of SOX bound to DNA showed more dynamic interactions along the length of the protein (~480 Å
     <sup>2</sup> interaction surface), when compared to the K2–31 RNA bound structure (~240 Å
     <sup>2</sup> interaction surface). It is therefore possible that more interaction along the length of SOX protein might occur with optimal substrates such as 
     <em>LIMD1</em> that are more tightly bound.
    </p>
    <p id="p-29">The fact that purified SOX endonucleolytically cleaved 
     <em>LIMD1</em> 54 at the precise site observed in SOX-expressing cells demonstrates that cleavage site selection on an mRNA is not mediated by a cellular cofactor. Instead, targeting at particular RNA motifs is strongly influenced by the strength of SOX binding. Our observation that the P176S and F179A SOX mutants display significant RNA binding defects indicates that their failure to cleave mRNAs in cells is due to an inability to efficiently bind the targeting motif.
    </p>
    <p id="p-30">The mechanism by which SOX initially distinguishes RNA polymerase II transcribed mRNAs from other types of RNA in cells remains an important open question, as this feature of SOX selectivity is not preserved 
     <em>in vitro.</em> We hypothesize that cellular co-factors, perhaps though interactions with SOX, enable this distinction. More broadly, endonucleases are instrumental in RNA processing and degradation. Nuclease processing defects lead to several human pathologies ranging from cancer to neurodegeneration (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–
     <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>), and our study provides a framework for better understanding the mechanistic features governing endonuclease targeting.
    </p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Materials and Methods</h2>
    <p id="p-31">Full details are available in 
     <em>SI Materials and Methods</em>
    </p>
    <div id="sec-12" class="subsection">
     <h3>Recombinant protein expression and purification</h3>
     <p id="p-32">KSHV SOX was codon optimized for Sf9 expression and synthesized from GENEWIZ. SOX was then subcloned using restriction sites BamHI and SalI (New England BioLabs) into pFastBac HTD. This vector was modified to carry a GST affinity tag and PreScission protease cut site as described (
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). All SOX mutants were generated using single primer site-directed mutagenesis (
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Sequences were validated using standard pGEX forward and reverse primers. Generation of viral bacmids and transfections were prepared as described in the Bac-to-Bac
      <sup>®</sup> Baculovirus Expression System (Thermo Fisher Scientific) manual. After transfection, Sf9 cells (Thermo Fisher Scientific) were grown for 96 hr at 22 °C using SF-900 SMF media (Gibco) substituted with 5% fetal bovine serum (FBS) and 1% antibiotic antimycotic (AA). Supernatant was transferred to a 6-well tissue culture plate containing 1 ml of 2×10
      <sup>^6</sup> cells/well. Cells were incubated for 96hr to generate passage 1 (P1). The P1 supernatant was transferred to a flask containing 50 mL of 1×10
      <sup>^6</sup> cells/ml and incubated for 96 h, a time point sufficient to yield 5 mg of SOX per 50 mL of cells. Protein expression was confirmed by Western blot with an anti-GST antibody (GE Health Care Life Sciences).
     </p>
     <p id="p-33">Sf9 cell pellets were suspended in lysis buffer containing 600 mM NaCl, 5% glycerol, 0.5% Triton X-100, 3 mM DTT, 20 mM HEPES pH 7.0 with a cOmplete, EDTA-Free protease inhibitor Cocktail tablet (Roche). Cells were sonicated on ice using a macro trip for 30 sec bursts with 1 min rests for 12 min at 50 Amps. Cell lysate was cleared using a pre-chilled (4°C) Sorvall LYNX 6000 Superspeed Centrifuge spun at 18,000 rpm for 45 min. The cleared lysate was incubated for 4 hr at 4 °C with rotation with 4 mL of a GST bead slurry (GE Healthcare Life Sciences) that had been pre-washed 3x with wash buffer (WB) containing 600 mM NaCl, 5% glycerol, 3 mM DTT, 20 mM HEPES pH 7.0. The bead-protein mixture was washed 3X times with 15 mL of WB, then transferred to a 10 ml disposable column (Qiagen) and washed with an additional 50 mL of WB followed by 100 mL of low salt buffer (LSB) containing 250 mM NaCl, 5% glycerol, 3 mM DTT, 20 mM HEPES pH 7.0 with periodic resuspension to prevent compaction. SOX was then cleaved on column with PreScission protease (GE Healthcare Life Sciences) overnight at 4°C, and remaining protein was collected with a final 8 mL LSB wash.</p>
     <p id="p-34">Cleaved protein was concentrated to ~ 1 mL using Amicon filter concentrator membrane cut off 30 kDa (EMD Millipore), then loaded onto a HiLoad Superdex S200 pg gel filtration column (GE Healthcare Life Sciences). Protein elutions were concentrated using an Amicon concentrator described above to 5 mg/mL and aliquots were snap frozen in liquid N
      <sub>2</sub> and stored at − 80 °C.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>RNA substrate preparation and end labeling</h3>
     <p id="p-35">All RNA substrates (sequences in 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S2</a>) unless stated otherwise were synthesized by Dharmacon (GE Healthcare) with HPLC and page purification. RNAs were 5’ end labeled with γ-[
      <sup>32</sup>P]-ATP-6000 Ci/mmol 150 mCi/ml (Perkin Elmer) using T4 PNK (New England BioLabs). RNAs were 3’ end labeled with 5’-[
      <sup>32</sup>P]-pCp 3000 Ci/mMol 10 mCi/ml using T4 RNA ligase 1 (New England BioLabs). Labeled RNA substrates were purified using 20% urea-PAGE and were isolated from gel slices by incubating overnight at 8°C in a buffer containing 10 mM Tris-HCl, 1 mM EDTA pH 8.0. Eluted RNAs were ethanol precipitated and resuspended in RNase-free ddH
      <sub>2</sub>O.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Ribonuclease assays</h3>
     <p id="p-36">
      <em>k</em>
      <sub>obs</sub> and Hill coefficients of SOX were determined from the cleavage kinetics of [
      <sup>32</sup>P]-labeled RNA substrates as previously described (
      <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>). Briefly, 1 μl (≤1 pM) of [
      <sup>32</sup>P]-labeled RNA was added to 9 μl of premixture containing 20 mM Hepes pH 7.1, 70 mM NaCl, 2 mM MgCl
      <sub>2</sub>, 1 mM TCEP, 1% glycerol, and increasing concentrations of purified SOX. Reactions were performed at room temperature under single turnover conditions, and quenched at the indicated time intervals with 8 μl stop solution (10 M urea, 0.1% SDS, 0.1 mM EDTA, 0.05% xylene cyanol, 0.05% bromophenol blue). Samples were resolved by 15% urea-PAGE, imaged using a Typhoon variable mode imager (GE Healthcare), and quantified using ImageQuant and GelQuant software packages (Molecular Devices). The data were plotted and fit to exponential curves using Prism 7 software package (GraphPad) to determine observed rate constants.
     </p>
     <p id="p-37">A FRET probe with excitation at 646 nM and emission at 662 nM (
      <em>LIMD1</em> 54 Flo) was purchased from Dharmacon. The RNA FRET probe was added at a final concentration of 100 nM to 9 μl of premixture containing 20 mM Hepes pH 7.1, 70 mM NaCl, 2 mM MgCl
      <sub>2</sub>, 1 mM TCEP, 1% glycerol with 2 μM of SOX (
      <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>). Terminator 5’ exonuclease (Lucigen) was added to reactions using a 1:1000 dilution of the enzyme. reactions were quenched at indicated time intervals with equal volumes of stop solution containing 95 
      <em>%</em> formamide and 10 mM EDTA, then resolved using urea-PAGE and visualized using a Typhoon variable mode imager (GE Halthcare). The data were plotted using Prism 7 software package (GraphPad). All experiments were repeated &gt;;3 times and mean values were computed.
     </p>
     <p id="p-38">For assays designed to detect endonucleolytic cleavage intermediates, 1 μl of labeled RNA substrate was combined with 9 μl of reaction solution (20 mM HEPES pH 7.1, 70mM NaCl, 0.200 mM CaCl, 0.700 mM MgCl
      <sub>2</sub>, 1 % glycerol, 0.5 mM TCEP) in the presence or absence of 2 μM SOX for 10 min at room temperature. RNA was then ethanol precipitated, resuspended in 95 % formamide solution containing 10 mM EDTA, and resolved on a 12% urea-PAGE analytical grade sequencing gel together with a ssRNA Decade ladder (Ambion Life Technologies) for 1.5 hours at 22 watts before imaging as described above.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-39">We thank members of the Glaunsinger lab for their suggestions and critical reading of the manuscript. This work was supported by NIH R01CA136367 (B.A.G.), C.V. was supported by the DAAD P.R.I.M.E. program (57222872) for postdoctoral fellows. B.A.G. is an investigator of the Howard Hughes Medical Institute.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Abernathy</span>
          <span class="cit-name-given-names">E</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">B</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Emerging roles for RNA degradation in viral replication and antiviral defense</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">479–480</span>:
         <span class="cit-fpage">600</span>–
         <span class="cit-lpage">608</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Modulation of the Translational Landscape During Herpesvirus Infection</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr>
         <span class="cit-vol">2</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">311</span>–
         <span class="cit-lpage">333</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D2%26rft.spage%253D311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.3" data-doi="10.1128/JVI.01230-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gaglia</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Covarrubias</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">W</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">A common strategy for host RNA degradation by divergent viruses</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">86</span>(
         <span class="cit-issue">17</span>):
         <span class="cit-fpage">9527</span>–
         <span class="cit-lpage">9530</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01230-12%26rft_id%253Dinfo%253Apmid%252F22740404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNy85NTI3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.4" data-doi="10.1038/nsmb.1680">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kamitani</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Narayanan</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lokugamage</span>
          <span class="cit-name-given-names">KG</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Makino</span>
          <span class="cit-name-given-names">S</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">A two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein</span>. 
         <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr>
         <span class="cit-vol">16</span>(
         <span class="cit-issue">11</span>):
         <span class="cit-fpage">1134</span>–
         <span class="cit-lpage">1140</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DKamitani%26rft.auinit1%253DW.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D1134%26rft.epage%253D1140%26rft.atitle%253DA%2Btwo-pronged%2Bstrategy%2Bto%2Bsuppress%2Bhost%2Bprotein%2Bsynthesis%2Bby%2BSARS%2Bcoronavirus%2BNsp1%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1680%26rft_id%253Dinfo%253Apmid%252F19838190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nsmb.1680&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19838190&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.5" data-doi="10.1128/JVI.00319-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashi</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">MacDonald</span>
          <span class="cit-name-given-names">LA</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Takimoto</span>
          <span class="cit-name-given-names">T</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Influenza A Virus Protein PA-X Contributes to Viral Growth and Suppression of the Host Antiviral and Immune Responses</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">89</span>(
         <span class="cit-issue">12</span>):
         <span class="cit-fpage">6442</span>–
         <span class="cit-lpage">6452</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00319-15%26rft_id%253Dinfo%253Apmid%252F25855745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xMi82NDQyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rabkin</span>
          <span class="cit-name-given-names">CS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggar</span>
          <span class="cit-name-given-names">RJ</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Horm</span>
          <span class="cit-name-given-names">JW</span>
         </span> (
         <span class="cit-pub-date">1991</span>) 
         <span class="cit-article-title">Increasing incidence of cancers associated with the human immunodeficiency virus epidemic</span>. 
         <abbr class="cit-jnl-abbrev">Int J Cancer</abbr>
         <span class="cit-vol">47</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">692</span>–
         <span class="cit-lpage">696</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bcancer.%2B%2BJournal%2Binternational%2Bdu%2Bcancer%26rft.stitle%253DInt%2BJ%2BCancer%26rft.aulast%253DRabkin%26rft.auinit1%253DC.%2BS.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D692%26rft.epage%253D696%26rft.atitle%253DIncreasing%2Bincidence%2Bof%2Bcancers%2Bassociated%2Bwith%2Bthe%2Bhuman%2Bimmunodeficiency%2Bvirus%2Bepidemic.%26rft_id%253Dinfo%253Apmid%252F2004849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=2004849&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1991FC75700010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.7" data-doi="10.1182/asheducation-2013.1.103">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kaplan</span>
          <span class="cit-name-given-names">LD</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Human herpesvirus-8: Kaposi sarcoma, multicentric Castleman disease, and primary effusion lymphoma</span>. 
         <abbr class="cit-jnl-abbrev">Hematology Am Soc Hematol Educ Program</abbr>
         <span class="cit-vol">2013</span>:
         <span class="cit-fpage">103</span>–
         <span class="cit-lpage">108</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DASH%2BEducation%2BProgram%2BBook%26rft.stitle%253DASH%2BEducation%2BBook%26rft.aulast%253DKaplan%26rft.auinit1%253DL.%2BD.%26rft.volume%253D2013%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D108%26rft.atitle%253DHuman%2Bherpesvirus-8%253A%2BKaposi%2Bsarcoma%252C%2Bmulticentric%2BCastleman%2Bdisease%252C%2Band%2Bprimary%2Beffusion%2Blymphoma%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fasheducation-2013.1.103%26rft_id%253Dinfo%253Apmid%252F24319170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiYmxvb2Rib29rIjtzOjU6InJlc2lkIjtzOjEwOiIyMDEzLzEvMTAzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.8" data-doi="10.1128/JVI.79.12.7396-7401.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chavez</span>
          <span class="cit-name-given-names">L</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ganem</span>
          <span class="cit-name-given-names">D</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">The exonuclease and host shutoff functions of the SOX protein of Kaposi’s sarcoma-associated herpesvirus are genetically separable</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">79</span>(
         <span class="cit-issue">12</span>):
         <span class="cit-fpage">7396</span>–
         <span class="cit-lpage">7401</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGlaunsinger%26rft.auinit1%253DB.%26rft.volume%253D79%26rft.issue%253D12%26rft.spage%253D7396%26rft.epage%253D7401%26rft.atitle%253DThe%2BExonuclease%2Band%2BHost%2BShutoff%2BFunctions%2Bof%2Bthe%2BSOX%2BProtein%2Bof%2BKaposi%2527s%2BSarcoma-Associated%2BHerpesvirus%2BAre%2BGenetically%2BSeparable%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.12.7396-7401.2005%26rft_id%253Dinfo%253Apmid%252F15919895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xMi83Mzk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.9" data-doi="10.1111/j.1742-4658.2009.07374.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dahlroth</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Crystal structure of the shutoff and exonuclease protein from the oncogenic Kaposi’s sarcoma-associated herpesvirus</span>. 
         <abbr class="cit-jnl-abbrev">FEBS J</abbr>
         <span class="cit-vol">276</span>(
         <span class="cit-issue">22</span>):
         <span class="cit-fpage">6636</span>–
         <span class="cit-lpage">6645</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DDahlroth%26rft.auinit1%253DS.%2BL.%26rft.volume%253D276%26rft.issue%253D22%26rft.spage%253D6636%26rft.epage%253D6645%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bshutoff%2Band%2Bexonuclease%2Bprotein%2Bfrom%2Bthe%2Boncogenic%2BKaposi%2527s%2Bsarcoma-associated%2Bherpesvirus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2009.07374.x%26rft_id%253Dinfo%253Apmid%252F19843164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2009.07374.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19843164&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.10" data-doi="10.1093/nar/gkr111">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bagneris</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Briggs</span>
          <span class="cit-name-given-names">LC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Savva</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ebrahimi</span>
          <span class="cit-name-given-names">B</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Barrett</span>
          <span class="cit-name-given-names">TE</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Crystal structure of a KSHV-SOX-DNA complex: insights into the molecular mechanisms underlying DNase activity and host shutoff</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">39</span>(
         <span class="cit-issue">13</span>):
         <span class="cit-fpage">5744</span>–
         <span class="cit-lpage">5756</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr111%26rft_id%253Dinfo%253Apmid%252F21421561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkr111&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21421561&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000293020000042&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.11" data-doi="10.1371/journal.ppat.1002150">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Richner</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Global mRNA degradation during lytic gammaherpesvirus infection contributes to establishment of viral latency</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">7</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">e1002150</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRichner%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002150%26rft.epage%253De1002150%26rft.atitle%253DGlobal%2BmRNA%2Bdegradation%2Bduring%2Blytic%2Bgammaherpesvirus%2Binfection%2Bcontributes%2Bto%2Bestablishment%2Bof%2Bviral%2Blatency.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002150%26rft_id%253Dinfo%253Apmid%252F21811408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002150&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21811408&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.12" data-doi="10.1371/journal.ppat.1003882">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Abernathy</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Gammaherpesviral gene expression and virion composition are broadly controlled by accelerated mRNA degradation</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">10</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">e1003882</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1003882%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003882%26rft_id%253Dinfo%253Apmid%252F24453974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003882&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24453974&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.13" data-doi="10.1073/pnas.0611128104">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rowe</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Host shutoff during productive Epstein-Barr virus infection is mediated by BGLF5 and may contribute to immune evasion</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">104</span>(
         <span class="cit-issue">9</span>):
         <span class="cit-fpage">3366</span>–
         <span class="cit-lpage">3371</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0611128104%26rft_id%253Dinfo%253Apmid%252F17360652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzkvMzM2NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzEzLzMyMDkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.14" data-doi="10.1371/journal.pbio.1000107">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">YJ</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Aberrant herpesvirus-induced polyadenylation correlates with cellular messenger RNA destruction</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>
         <span class="cit-vol">7</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">e1000107</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1000107%26rft.epage%253De1000107%26rft.atitle%253DAberrant%2Bherpesvirus-induced%2Bpolyadenylation%2Bcorrelates%2Bwith%2Bcellular%2Bmessenger%2BRNA%2Bdestruction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000107%26rft_id%253Dinfo%253Apmid%252F19468299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000107&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19468299&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.15" data-doi="10.1016/S1097-2765(04)00091-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">B</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ganem</span>
          <span class="cit-name-given-names">D</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">Lytic KSHV infection inhibits host gene expression by accelerating global mRNA turnover</span>. 
         <abbr class="cit-jnl-abbrev">Mol Cell</abbr>
         <span class="cit-vol">13</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">713</span>–
         <span class="cit-lpage">723</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGlaunsinger%26rft.auinit1%253DB.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D713%26rft.epage%253D723%26rft.atitle%253DLytic%2BKSHV%2Binfection%2Binhibits%2Bhost%2Bgene%2Bexpression%2Bby%2Baccelerating%2Bglobal%2BmRNA%2Bturnover.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252804%252900091-7%26rft_id%253Dinfo%253Apmid%252F15023341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1097-2765(04)00091-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15023341&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220209000009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.16" data-doi="10.1371/journal.ppat.1002339">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Covarrubias</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Coordinated destruction of cellular messages in translation complexes by the gammaherpesvirus host shutoff factor and the mammalian exonuclease Xrn1</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">7</span>(
         <span class="cit-issue">10</span>):
         <span class="cit-fpage">e1002339</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DCovarrubias%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002339%26rft.epage%253De1002339%26rft.atitle%253DCoordinated%2Bdestruction%2Bof%2Bcellular%2Bmessages%2Bin%2Btranslation%2Bcomplexes%2Bby%2Bthe%2Bgammaherpesvirus%2Bhost%2Bshutoff%2Bfactor%2Band%2Bthe%2Bmammalian%2Bexonuclease%2BXrn1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002339%26rft_id%253Dinfo%253Apmid%252F22046136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002339&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22046136&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.17" data-doi="10.1371/journal.pone.0019655">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Clyde</span>
          <span class="cit-name-given-names">K</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Deep sequencing reveals direct targets of gammaherpesvirus-induced mRNA decay and suggests that multiple mechanisms govern cellular transcript escape</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">6</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">e19655</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DClyde%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19655%26rft.epage%253De19655%26rft.atitle%253DDeep%2Bsequencing%2Breveals%2Bdirect%2Btargets%2Bof%2Bgammaherpesvirus-induced%2BmRNA%2Bdecay%2Band%2Bsuggests%2Bthat%2Bmultiple%2Bmechanisms%2Bgovern%2Bcellular%2Btranscript%2Bescape.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019655%26rft_id%253Dinfo%253Apmid%252F21573023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019655&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21573023&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.18" data-doi="10.1128/JVI.00159-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hutin</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">Y</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">An RNA element in human interleukin 6 confers escape from degradation by the gammaherpesvirus SOX protein</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">87</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">4672</span>–
         <span class="cit-lpage">4682</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00159-13%26rft_id%253Dinfo%253Apmid%252F23408619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg3LzgvNDY3MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzEzLzMyMDkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Muller</span>
          <span class="cit-name-given-names">M</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">13</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">e1006593</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gaglia</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rycroft</span>
          <span class="cit-name-given-names">CH</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Glaunsinger</span>
          <span class="cit-name-given-names">BA</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Transcriptome-Wide Cleavage Site Mapping on Cellular mRNAs Reveals Features Underlying Sequence-Specific Cleavage by the Viral Ribonuclease SOX</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">11</span>(
         <span class="cit-issue">12</span>):
         <span class="cit-fpage">e1005305</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D11%26rft.spage%253De1005305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">KSHV SOX mediated host shutoff: the molecular mechanism underlying mRNA transcript processing</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">45</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">4756</span>–
         <span class="cit-lpage">4767</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D45%26rft.spage%253D4756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.22" data-doi="10.1093/nar/gkq958">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Laganeckas</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Margelevicius</span>
          <span class="cit-name-given-names">M</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Venclovas</span>
          <span class="cit-name-given-names">C</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Identification of new homologs of PD-(D/E)XK nucleases by support vector machines trained on data derived from profile-profile alignments</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">39</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">1187</span>–
         <span class="cit-lpage">1196</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq958%26rft_id%253Dinfo%253Apmid%252F20961958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkq958&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20961958&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000288019400009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.23" data-doi="10.1073/pnas.1103467108">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCabe</span>
          <span class="cit-name-given-names">KA</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bell</span>
          <span class="cit-name-given-names">CE</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Crystal structures of lambda exonuclease in complex with DNA suggest an electrostatic ratchet mechanism for processivity</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">108</span>(
         <span class="cit-issue">29</span>):
         <span class="cit-fpage">11872</span>–
         <span class="cit-lpage">11877</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1103467108%26rft_id%253Dinfo%253Apmid%252F21730170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI5LzExODcyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.24" data-doi="10.1074/jbc.C200664200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mordasini</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Curioni</span>
          <span class="cit-name-given-names">A</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Andreoni</span>
          <span class="cit-name-given-names">W</span>
         </span> (
         <span class="cit-pub-date">2003</span>) 
         <span class="cit-article-title">Why do divalent metal ions either promote or inhibit enzymatic reactions? The case of BamHI restriction endonuclease from combined quantum-classical simulations</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>
         <span class="cit-vol">278</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">4381</span>–
         <span class="cit-lpage">4384</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C200664200%26rft_id%253Dinfo%253Apmid%252F12496295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzgvNy80MzgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.25" data-doi="10.1186/1741-7007-9-48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Korennykh</span>
          <span class="cit-name-given-names">AV</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Cofactor-mediated conformational control in the bifunctional kinase/RNase Ire1</span>. 
         <abbr class="cit-jnl-abbrev">BMC Biol</abbr>
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DKorennykh%26rft.auinit1%253DA.%2BV.%26rft.volume%253D9%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DCofactor-mediated%2Bconformational%2Bcontrol%2Bin%2Bthe%2Bbifunctional%2Bkinase%252FRNase%2BIre1.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-9-48%26rft_id%253Dinfo%253Apmid%252F21729334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1741-7007-9-48&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21729334&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.26" data-doi="10.1261/rna.064626.117">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Morton</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">The RNA exosome and RNA exosome-linked disease</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>
         <span class="cit-vol">24</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">127</span>–
         <span class="cit-lpage">142</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.064626.117%26rft_id%253Dinfo%253Apmid%252F29093021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMTI3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.27" data-doi="10.1261/rna.060004.116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gillespie</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gabunilas</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jen</span>
          <span class="cit-name-given-names">JC</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chanfreau</span>
          <span class="cit-name-given-names">GF</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Mutations of EXOSC3/Rrp40p associated with neurological diseases impact ribosomal RNA processing functions of the exosome in S. cerevisiae</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>
         <span class="cit-vol">23</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">466</span>–
         <span class="cit-lpage">472</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.060004.116%26rft_id%253Dinfo%253Apmid%252F28053271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNDY2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reis</span>
          <span class="cit-name-given-names">FP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pobre</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Silva</span>
          <span class="cit-name-given-names">IJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Malecki</span>
          <span class="cit-name-given-names">M</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Arraiano</span>
          <span class="cit-name-given-names">CM</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">The RNase II/RNB family of exoribonucleases: putting the ‘Dis’ in disease</span>. 
         <abbr class="cit-jnl-abbrev">Wiley Interdiscip Rev RNA</abbr>
         <span class="cit-vol">4</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">607</span>–
         <span class="cit-lpage">615</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip%2BRev%2BRNA%26rft.volume%253D4%26rft.spage%253D607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-320929v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Popp</span>
          <span class="cit-name-given-names">MW</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Maquat</span>
          <span class="cit-name-given-names">LE</span>
         </span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Nonsense-mediated mRNA Decay and Cancer</span>. 
         <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr>
         <span class="cit-vol">48</span>:
         <span class="cit-fpage">44</span>–
         <span class="cit-lpage">50</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.volume%253D48%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.30" data-doi="10.1016/j.molmed.2006.05.005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kuzmiak</span>
          <span class="cit-name-given-names">HA</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Maquat</span>
          <span class="cit-name-given-names">LE</span>
         </span> (
         <span class="cit-pub-date">2006</span>) 
         <span class="cit-article-title">Applying nonsense-mediated mRNA decay research to the clinic: progress and challenges</span>. 
         <abbr class="cit-jnl-abbrev">Trends Mol Med</abbr>
         <span class="cit-vol">12</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">306</span>–
         <span class="cit-lpage">316</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DTrends%2BMol%2BMed%26rft.aulast%253DKuzmiak%26rft.auinit1%253DH.%2BA.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D306%26rft.epage%253D316%26rft.atitle%253DApplying%2Bnonsense-mediated%2BmRNA%2Bdecay%2Bresearch%2Bto%2Bthe%2Bclinic%253A%2Bprogress%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molmed.2006.05.005%26rft_id%253Dinfo%253Apmid%252F16782405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.molmed.2006.05.005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16782405&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000239703800005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.31" data-doi="10.1126/science.aaf1292">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">355</span>(
         <span class="cit-issue">6332</span>):
         <span class="cit-fpage">1428</span>–
         <span class="cit-lpage">1433</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf1292%26rft_id%253Dinfo%253Apmid%252F28280247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTUvNjMzMi8xNDI4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTMvMzIwOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.32" data-doi="10.1038/nature07661">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Korennykh</span>
          <span class="cit-name-given-names">AV</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">The unfolded protein response signals through high-order assembly of Ire1</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">457</span>(
         <span class="cit-issue">7230</span>):
         <span class="cit-fpage">687</span>–
         <span class="cit-lpage">693</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKorennykh%26rft.auinit1%253DA.%2BV.%26rft.volume%253D457%26rft.issue%253D7230%26rft.spage%253D687%26rft.epage%253D693%26rft.atitle%253DThe%2Bunfolded%2Bprotein%2Bresponse%2Bsignals%2Bthrough%2Bhigh-order%2Bassembly%2Bof%2BIre1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07661%26rft_id%253Dinfo%253Apmid%252F19079236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature07661&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19079236&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000263064700033&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 1. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-320929v1.33" data-doi="10.1007/978-1-59745-033-1_4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Regulski</span>
          <span class="cit-name-given-names">EE</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Breaker</span>
          <span class="cit-name-given-names">RR</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">In-line probing analysis of riboswitches</span>. 
         <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>
         <span class="cit-vol">419</span>:
         <span class="cit-fpage">53</span>–
         <span class="cit-lpage">67</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DRegulski%26rft.auinit1%253DE.%2BE.%26rft.volume%253D419%26rft.spage%253D53%26rft.epage%253D67%26rft.atitle%253DIn-line%2Bprobing%2Banalysis%2Bof%2Briboswitches.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-033-1_4%26rft_id%253Dinfo%253Apmid%252F18369975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/978-1-59745-033-1_4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18369975&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F13%2F320929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
