<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/11/2020.04.07.030924.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.07.030924v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.07.030924"/>
  <meta name="DC.Date" content="2020-04-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation. ### Competing Interest Statement"/>
  <meta name="DC.Contributor" content="Syed Mohammad Lokman"/>
  <meta name="DC.Contributor" content="Md. Rasheduzzaman"/>
  <meta name="DC.Contributor" content="Asma Salauddin"/>
  <meta name="DC.Contributor" content="Rocktim Barua"/>
  <meta name="DC.Contributor" content="Afsana Yeasmin Tanzina"/>
  <meta name="DC.Contributor" content="Meheadi Hasan Rumi"/>
  <meta name="DC.Contributor" content="Md. Imran Hossain"/>
  <meta name="DC.Contributor" content="Amam Zonaed Siddiki"/>
  <meta name="DC.Contributor" content="Adnan Mannan"/>
  <meta name="DC.Contributor" content="Md. Mahbub Hasan"/>
  <meta name="article:published_time" content="2020-04-11"/>
  <meta name="article:section" content="Confirmatory Results"/>
  <meta name="citation_title" content="Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.07.030924v1"/>
  <meta name="citation_id" content="2020.04.07.030924v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030924v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030924v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030924v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/11/2020.04.07.030924.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.07.030924"/>
  <meta name="citation_section" content="Confirmatory Results"/>
  <meta name="citation_firstpage" content="2020.04.07.030924"/>
  <meta name="citation_author" content="Syed Mohammad Lokman"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="syedlokman3@gmail.com"/>
  <meta name="citation_author" content="Md. Rasheduzzaman"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="rashed.geb.cu@gmail.com"/>
  <meta name="citation_author" content="Asma Salauddin"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="asmageb11@gmail.com"/>
  <meta name="citation_author" content="Rocktim Barua"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="rocktim23@gmail.com"/>
  <meta name="citation_author" content="Afsana Yeasmin Tanzina"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="tanziageb@gmail.com"/>
  <meta name="citation_author" content="Meheadi Hasan Rumi"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="rumicu0@gmail.com"/>
  <meta name="citation_author" content="Md. Imran Hossain"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="imranemu951@gmail.com"/>
  <meta name="citation_author" content="Amam Zonaed Siddiki"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="zsiddiki@gmail.com"/>
  <meta name="citation_author" content="Adnan Mannan"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="adnan.mannan@cu.ac.bd"/>
  <meta name="citation_author" content="Md. Mahbub Hasan"/>
  <meta name="citation_author_institution" content="University of Chittagong"/>
  <meta name="citation_author_email" content="mahbub.hasan@cu.ac.bd"/>
  <meta name="twitter:title" content="Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation. ### Competing Interest Statement"/>
  <meta name="og-title" property="og:title" content="Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030924v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation. ### Competing Interest Statement"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-11"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1239290 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-category-confirmatory-results"> 
  <div class="article abstract-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-2">The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.</p>
   </div>
   <h3>Competing Interest Statement</h3>
   <p id="p-3"/>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
