<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/25/2020.02.21.959973.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.21.959973v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.21.959973"/>
  <meta name="DC.Date" content="2020-02-25"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at &lt;;https://github.com/galaxyproject/SARS-CoV-2&gt;;."/>
  <meta name="DC.Contributor" content="Galaxy and HyPhy developments teams"/>
  <meta name="DC.Contributor" content="Anton Nekrutenko"/>
  <meta name="DC.Contributor" content="Sergei L Kosakovsky Pond"/>
  <meta name="article:published_time" content="2020-02-25"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.21.959973v2"/>
  <meta name="citation_id" content="2020.02.21.959973v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959973v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959973v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959973v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.21.959973"/>
  <meta name="citation_num_pages" content="12"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.21.959973"/>
  <meta name="citation_author" content="Galaxy and HyPhy developments teams"/>
  <meta name="citation_author_institution" content="The Pennsylvania State University"/>
  <meta name="citation_author" content="Anton Nekrutenko"/>
  <meta name="citation_author_email" content="anton@nekrut.org"/>
  <meta name="citation_author_email" content="spond@temple.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5987-8032"/>
  <meta name="citation_author" content="Sergei L Kosakovsky Pond"/>
  <meta name="citation_author_email" content="anton@nekrut.org"/>
  <meta name="citation_author_email" content="spond@temple.edu"/>
  <meta name="citation_reference" content="Zhou P, Yang X-L, Wang X-G, Hu B, Zhang L, Zhang W, Si H-R, Zhu Y, Li B, Huang C-L, Chen H-D, Chen J, Luo Y, Guo H, Jiang R-D, Liu M-Q, Chen Y, Shen X-R, Wang X, Zheng X-S, Zhao K, Chen Q-J, Deng F, Liu L-L, Yan B, Zhan F-X, Wang Y-Y, Xiao G-F, Shi Z-L. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature [Internet]. 2020 Feb 3; Available from: http://dx.doi.org/10.1038/s41586-020-2012-7 PMID: 32015507"/>
  <meta name="citation_reference" content="Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, Wang W, Song H, Huang B, Zhu N, Bi Y, Ma X, Zhan F, Wang L, Hu T, Zhou H, Hu Z, Zhou W, Zhao L, Chen J, Meng Y, Wang J, Lin Y, Yuan J, Xie Z, Ma J, Liu WJ, Wang D, Xu W, Holmes EC, Gao GF, Wu G, Chen W, Shi W, Tan W. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet [Internet]. 2020 Jan 30; Available from: http://dx.doi.org/10.1016/S0140-6736(20)30251-8 PMID: 32007145"/>
  <meta name="citation_reference" content="Wu F, Zhao S, Yu B, Chen Y-M, Wang W, Song Z-G, Hu Y, Tao Z-W, Tian J-H, Pei Y-Y, Yuan M-L, Zhang Y-L, Dai F-H, Liu Y, Wang Q-M, Zheng J-J, Xu L, Holmes EC, Zhang Y-Z. A new coronavirus associated with human respiratory disease in China. Nature [Internet]. 2020 Feb 3; Available from: http://dx.doi.org/10.1038/s41586-020-2008-3 PMID: 32015508"/>
  <meta name="citation_reference" content="Holshue ML, DeBolt C, Lindquist S, Lofy KH, Wiesman J, Bruce H, Spitters C, Ericson K, Wilkerson S, Tural A, Diaz G, Cohn A, Fox L, Patel A, Gerber SI, Kim L, Tong S, Lu X, Lindstrom S, Pallansch MA, Weldon WC, Biggs HM, Uyeki TM, Pillai SK, Washington State 2019-nCoV Case Investigation Team. First Case of 2019 Novel Coronavirus in the United States. N Engl J Med [Internet]. 2020 Jan 31; Available from: http://dx.doi.org/10.1056/NEJMoa2001191 PMID: 32004427"/>
  <meta name="citation_reference" content="Grüning B, Dale R, Sjödin A, Chapman BA, Rowe J, Tomkins-Tinch CH, Valieris R, Köster J, Bioconda Team. Bioconda: sustainable and comprehensive software distribution for the life sciences. Nat Methods. 2018 Jul;15(7):475–476. PMID: 29967506"/>
  <meta name="citation_reference" content="Goecks J, Nekrutenko A, Taylor J, Team G. Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences. Genome Biol. Department of Biology and Department of Mathematics and Computer Science, Emory University, 1510 Clifton Road NE, Atlanta, GA 30322, USA. jeremy.goecks@emory.edu: BioMed Central Ltd; 2010 Jan 1;11(8):R86."/>
  <meta name="citation_reference" content="Grüning BA, Rasche E, Rebolledo-Jaramillo B, Eberhard C, Houwaart T, Chilton J, Coraor N, Backofen R, Taylor J, Nekrutenko A. Jupyter and Galaxy: Easing entry barriers into complex data analyses for biomedical researchers. PLoS Comput Biol. 2017 May;13(5):e1005425. PMCID: PMC5444614"/>
  <meta name="citation_reference" content="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. J Comput Biol. 2012 May 1;19(5):455–477."/>
  <meta name="citation_reference" content="Wick RR, Judd LM, Gorrie CL, Holt KE. Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads. PLoS Comput Biol. 2017 Jun;13(6):e1005595. PMCID: PMC5481147"/>
  <meta name="citation_reference" content="Korber B, Muldoon M, Theiler J, Gao F, Gupta R, Lapedes A, Hahn BH, Wolinsky S, Bhattacharya T. Timing the ancestor of the HIV-1 pandemic strains. Science. 2000 Jun 9;288(5472):1789–1796. PMID: 10846155"/>
  <meta name="citation_reference" content="arambaut, Pinned A, Duchene S, Duplessis L, Volz E, Unpinned A, Globally AP. Phylodynamic Analysis | 93 genomes | 15 Feb 2020 [Internet]. Virological. 2020 [cited 2020 Feb 17]. Available from: http://virological.org/t/phylodynamic-analysis-93-genomes-15-feb-2020/356"/>
  <meta name="citation_reference" content="Schmitt MW, Kennedy SR, Salk JJ, Fox EJ, Hiatt JB, Loeb LA. Detection of ultra-rare mutations by next-generation sequencing. Proc Natl Acad Sci U S A. Departments of Pathology, Genome Sciences, and Biochemistry, University of Washington School of Medicine, Seattle, WA 98195, USA.; 2012 Sep 4;109(36):14508–14513."/>
  <meta name="citation_reference" content="Walls AC, Tortorici MA, Bosch B-J, Frenz B, Rottier PJM, DiMaio F, Rey FA, Veesler D. Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer. Nature. 2016 Mar 3;531(7592):114–117. PMCID: PMC5018210"/>
  <meta name="citation_reference" content="Duquerroy S, Vigouroux A, Rottier PJM, Rey FA, Bosch BJ. Central ions and lateral asparagine/glutamine zippers stabilize the post-fusion hairpin conformation of the SARS coronavirus spike glycoprotein. Virology. Elsevier; 2005 May 10;335(2):276–285. PMID: 15840526"/>
  <meta name="citation_reference" content="Kosakovsky Pond SL, Posada D, Gravenor MB, Woelk CH, Frost SDW. Automated phylogenetic detection of recombination using a genetic algorithm. Mol Biol Evol. 2006 Oct;23(10):1891–1901. PMID: 16818476"/>
  <meta name="citation_reference" content="Batut B, Hiltemann S, Bagnacani A, Baker D, Bhardwaj V, Blank C, Bretaudeau A, Brillet-Guéguen L, Čech M, Chilton J, Clements D, Doppelt-Azeroual O, Erxleben A, Freeberg MA, Gladman S, Hoogstrate Y, Hotz H-R, Houwaart T, Jagtap P, Larivière D, Le Corguillé G, Manke T, Mareuil F, Ramírez F, Ryan D, Sigloch FC, Soranzo N, Wolff J, Videm P, Wolfien M, Wubuli A, Yusuf D, Galaxy Training Network, Taylor J, Backofen R, Nekrutenko A, Grüning B. Community-Driven Data Analysis Training for Biology. Cell Syst. 2018 Jun 27;6(6):752–758.e1. PMCID: PMC6296361"/>
  <meta name="citation_reference" content="CDC. 2019 Novel Coronavirus (2019-nCoV) [Internet]. Centers for Disease Control and Prevention. 2020 [cited 2020 Feb 19]. Available from: https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html"/>
  <meta name="citation_reference" content="Garrison E, Marth G. Haplotype-based variant detection from short-read sequencing. arXiv.org [Internet]. Cornell University Library; 2012 Jul 17;q-bio.GN. Available from: http://arxiv.org/abs/1207.3907v2"/>
  <meta name="citation_reference" content="Common Workflow Language [Internet]. [cited 2020 Feb 21]. Available from: https://www.commonwl.org/"/>
  <meta name="citation_reference" content="Nextflow - A DSL for parallel and scalable computational pipelines [Internet]. [cited 2020 Feb 21]. Available from: https://www.nextflow.io/"/>
  <meta name="citation_reference" content="Köster J, Rahmann S. Snakemake--a scalable bioinformatics workflow engine. Bioinformatics. 2012 Oct 1;28(19):2520–2522. PMID: 22908215"/>
  <meta name="twitter:title" content="No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at &lt;;https://github.com/galaxyproject/SARS-CoV-2&gt;;."/>
  <meta name="og-title" property="og:title" content="No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959973v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at &lt;;https://github.com/galaxyproject/SARS-CoV-2&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-25"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1167268 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-3">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at 
     <a href="https://github.com/galaxyproject/SARS-CoV-2">https://github.com/galaxyproject/SARS-CoV-2</a>.
    </p>
   </div>
   <div id="sec-1" class="subsection">
    <p id="p-7">The initial publications describing genomic features of COVID-19 [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>] used Illumina and Oxford nanopore data to elucidate the sequence composition of patient specimens (although only Wu et al. [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>] explicitly provided the accession numbers for their raw short read sequencing data). However, their approaches to processing, assembly, and analysis of raw data differed widely (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and ranged from transparent [
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>] to entirely opaque [
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]. Such lack of analytical transparency sets a dangerous precedent. Infectious disease outbreaks often occur in locations where infrastructure necessary for data analysis may be inaccessible or unbiased interpretation of results may be politically untenable. As a consequence, there is a global need to ensure access to free, open, and robust analytical approaches that can be used by anyone in the world to analyze, interpret, and share data. Can existing tools and computational resources support such a global need? Here we show that they can: we analyzed all available raw COVID-19 data to demonstrate that analyses described in [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>] can be reproduced on public infrastructure using open source tools by any researcher with an Internet connection.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1169490/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1169490/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1169490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <p id="p-8" class="first-child">Methods used for the analysis of primary COVID-19 data. ? = uncertain (e.g., Holshue et al. [
       <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>] identify FreeBayes as an assembly tool).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-9">We exclusively used free software tools publicly available from the BioConda package distribution system [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], deployed through the worldwide network of open Galaxy platforms [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] and executed using public high throughput computational infrastructure (XSEDE in the US, VSC in Belgium, de.NBI and ELIXIR in EU, NeCTAR Research Cloud in Australia). We also used an open source Jupyter environment [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] for exploratory analysis of data. All analyses performed here are fully documented and accessible at 
     <a href="https://github.com/galaxyproject/SARS-CoV-2/">https://github.com/galaxyproject/SARS-CoV-2/</a> and 
     <a href="https://doi.org/10.5281/zenodo.3685264">https://doi.org/10.5281/zenodo.3685264</a> (note that these are being continuously updated).
    </p>
    <p id="p-10">We divided our analysis into the following stages: (1) read pre-processing, (2) genome assembly, (3) timing the most recent common ancestor (MRCA), (4) analysis of genomic variation within individual samples, and (5) recombination and selection analyses.</p>
    <p id="p-11">We pre-processed six currently available (as of Feb 19, 2020) sequencing read datasets for COVID-19 (
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1</a>) by removing adapter contamination and reads derived from human transcripts and combined the resulting datasets. This was done to COVID-19-specific reads that constitute only a fraction of the original data. These were used as inputs for SPAdes assembler [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>] and Unicycler [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]—an assembly pipeline based on SPAdes that includes a number of pre-processing and polishing steps. Both approaches were able to reconstruct a full length COVID-19 genome with Unicycler producing a cleaner assembly graph. Its largest contig (29,781bp) had 100% identity to the published assembly NC_045512.
    </p>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1169493/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1169493/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1169493" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S1.</span>
      <p id="p-27" class="first-child">Raw COVID-19 sequencing data available at the time of writing (Feb 20, 2020). BALF = bronchoalveolar lavage fluid, * indicates that data may not be reliable (for example, the link between SRR10903402 and [
       <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>] is inferred: neither the SRA record nor the manuscript establish this relationship).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-12">Next we estimated the date of the most recent common ancestor (MRCA) of COVID-19. For this we used simple root-to-tip regression [
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>] (more complex and powerful phylodynamics methods could certainly be used, but for this data with very low levels of sequence divergence, simpler and faster methods suffice). Using a set of sequences from all COVID-19 sequences available as of Feb 16, 2020 we obtained an MRCA date of Oct 24, 2019, which is close to other existing estimates [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>].
    </p>
    <p id="p-13">The vast majority of COVID-19 genomic data available at the time of writing are partially or fully assembled genomes. There is no public access to sequence reads that were used to produce these assemblies: as of Feb 19, 2020—more than two months since the beginning of the outbreak—there are only six raw datasets (
     <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S1</a>).
    </p>
    <p id="p-14">This should be unacceptable, since raw read data can be used to uncover viral diversity within individual samples and to evaluate robustness and reliability of the assembly. To demonstrate that such diversity exists, we mapped Illumina reads against COVID-19 reference (NC_045512) and identified sequence variants with frequencies above 5% while taking into account quality of alternative bases and strand bias. Five percent was selected as a conservative threshold that can be reliably resolved from Illumina data [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Using this threshold, thirty nine single nucleotide variants (SNVs) were identified in total across all samples (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). The most prominent sequence variant was observed in sample SRR10903401. It is an A-to-C substitution with alternate allele frequency of 38% that causes a Lys
     <sup>921</sup>Gln amino acid replacement within the spike glycoprotein S (product of gene 
     <em>S</em>). S is a homotrimeric protein containing S1 and S2 subdomains mediating receptor recognition and membrane fusion, respectively [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. S2 subdomains contain two heptad repeat (repeats of units containing seven amino acids) regions: HR1 and HR2. The Lys
     <sup>921</sup>Gln substitution we observed is located in HR1 and forms a salt bridge with Gln
     <sup>1188</sup> within HR2. This is one in a series of salt bridges involved in the formation of the HR1/HR2 hairpin structures [
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. This site invariably contains Lys in all human SARS-related coronaviruses (S protein residue 903) as well as in many other coronaviruses (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). However, more distantly related coronaviruses including transmissible gastroenteritis coronavirus (TGEV), the porcine respiratory coronavirus (PRCV), the canine coronavirus (CCV), the feline peritonitis virus (FIPV), and the porcine epidemic diarrhea virus (PDEV) all contain Gln at the corresponding position ([
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>] and 
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). The Lys
     <sup>921</sup>Gln change would prevent the formation of the salt bridge with Gln
     <sup>1188</sup> and may have structural and functional implications for the spike protein structure and, consequently, COVID-19 virulence. This potentially adaptive change was not observed in the other two samples and lack of raw read data prevented us from identifying it in other geographically and temporally distributed samples.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Distribution of nucleotide changes across COVID-19 genome. AF = minor allele frequency, POS = position" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-744263562" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Distribution of nucleotide changes across COVID-19 genome. AF = minor allele frequency, POS = position&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F1.medium.gif" width="440" height="127"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F1.medium.gif" width="440" height="127"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-15" class="first-child">Distribution of nucleotide changes across COVID-19 genome. AF = minor allele frequency, POS = position</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Amino acid alignment of spike glycoprotein regions HR1 (A) and HR2 (B). The site of the Lys921Gln substitution observed by us in a COVID-19 isolate is highlighted with a black rectangle in panel A. Its corresponding salt bridge partner is highlighted with black rectangle in panel B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-744263562" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Amino acid alignment of spike glycoprotein regions HR1 (A) and HR2 (B). The site of the Lys&lt;;sup&gt;;921&lt;;/sup&gt;;Gln substitution observed by us in a COVID-19 isolate is highlighted with a black rectangle in panel A. Its corresponding salt bridge partner is highlighted with black rectangle in panel B.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F2.medium.gif" width="440" height="257"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F2.medium.gif" width="440" height="257"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-16" class="first-child">Amino acid alignment of spike glycoprotein regions HR1 (A) and HR2 (B). The site of the Lys
       <sup>921</sup>Gln substitution observed by us in a COVID-19 isolate is highlighted with a black rectangle in panel A. Its corresponding salt bridge partner is highlighted with black rectangle in panel B.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-17">To detect potential genome rearrangement events that might have led to the emergence of COVID-19 we performed analysis of recombination using a genetic algorithm approach [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Wu et al. [
     <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>] identified two potential recombination breakpoints within the COVID-19 S-gene with some segments having higher similarity to Bat ZC45 and ZXC21 coronaviruses (accessions 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MG772933&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom">MG772933</a> and 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MG772934&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom">MG772934</a>, respectively), while others were more similar to SARS Tor2 and SZ3 isolates (accessions 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY274119&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom">AY274119</a> and 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY304486&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom">AY304486</a>). Our attempt at reproducing this analysis did identify a set of potential breakpoints similar to the ones reported by Wu et al. [
     <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>], but lacking robust statistical support (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>).
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Location of potential recombination breakpoints along the S-gene (GARD analysis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-744263562" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Location of potential recombination breakpoints along the &lt;;em&gt;;S-&lt;;/em&gt;;gene (GARD analysis).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F3.medium.gif" width="440" height="200"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F3.medium.gif" width="440" height="200"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-18" class="first-child">Location of potential recombination breakpoints along the 
       <em>S-</em>gene (GARD analysis).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-19">Finally, we performed a branch-level test for positive selection on a codon-alignment of the 
     <em>S</em> gene from COVID-19, SARS-Tor2 as well as Bat ZC45, ZXC21, and Rp3 coronaviruses, specifically to identify if there was any evidence of diversifying selection along the ancestral branch leading to COVID-19 isolates. We found statistically significant evidence of positive diversifying selection (~7% of 
     <em>S</em>-gene sites) along the branch leading to COVID-19 (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>).
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Analysis of branch-specific positive diversifying selection (aBSREL) along the branch leading to COVID-19 (MN988688)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-744263562" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Analysis of branch-specific positive diversifying selection (aBSREL) along the branch leading to COVID-19 (MN988688).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F4.medium.gif" width="434" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F4.medium.gif" width="434" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-20" class="first-child">Analysis of branch-specific positive diversifying selection (aBSREL) along the branch leading to COVID-19 (MN988688).</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-21">The goal of our study was to (1) raise awareness of the lack of primary data necessary to effectively respond to global emergencies such as the COVID-19 outbreak and (2) demonstrate that all analyses can be performed transparently with already existing open source publicly available tools and computational infrastructure. The first problem—reluctance to share primary data—has its roots in the fact that the ultimate reward for research efforts is a highly cited publication. As a result, individual researchers are naturally averse to sharing primary data prior to manuscript acceptance. The second issue—underutilization of existing, community supported tools and analysis frameworks—may be due to the lack of sustained efforts to educate the biomedical community about best practices in (genomic) data analysis. Such efforts exist (e.g., [
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]) but have difficulties reaching a wide audience because prominent scientific publication outlets are reluctant to accept data analysis tutorials or reviews. Yet the only way to improve accessibility and reproducibility of biomedical research is through dissemination of best analysis practices.
    </p>
    <p id="p-22">We want to particularly emphasize the issue of irreproducibility. All researchers involved in any given outbreak research should have access to a set of community-curated tools in the same way as they have access to COVID-19 RT-PCR primers [
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. Moreover they should have access to computational infrastructure that can execute these tools and apply them to potentially large NGS datasets. This is essential as precious time is spent on “reinventing the wheel”. Instead, in an ideal world, after reading any of the original COVID-19 manuscripts any researcher should be able to apply the same analytical procedures to their own data. To illustrate these points we assessed the reproducibility of the four initial manuscripts describing the COVID-19 genome (Table 2). All manuscripts reported versions of the software used, but none listed parameters used. This effectively prevents quality control and replication because outcomes of complex procedures such as genome assembly, phylogenetic reconstruction, and recombination analysis are notoriously parameter-dependent. One of the manuscripts [
     <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a>] explicitly lists FreeBayes [
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], a variant discovery tool, as software used for short read assembly—something that FreeBayes is not capable of doing. Finally, only [
     <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a>] provided access to the raw data, rendering the other three manuscripts unverifiable and irreproducible.
    </p>
    <p id="p-23">Our short study demonstrates that viral genome analyses can be performed using open worldwide scientific infrastructure, relying entirely on community-curated and -supported open-source software. While we used Galaxy as the platform to execute all analyses described here, the individual software components can be obtained directly from BioConda and run independently. They can be combined into workflows using systems like the CWL[
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], Nextflow[
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], or Snakemake[
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Whatever the execution environment or workflow engine, using community supported, versioned, open source tools makes data analyses robust and transparent. This increases the quality, efficiency and, ultimately, impact of biomedical research.
    </p>
    <p id="p-24">As we were submitting this manuscript, several new sets of raw Illumina reads have been deposited to the Short Read Archive at the NCBI. We did not analyze them in this version of our (Feb 23) manuscript. However, this reinforces the key point we are making in this study—anyone can use the open workflows described here to analyze the new data. In an age of digital connectedness, open, highly accessible, globally shared data and analysis platforms have the potential to transform the way biomedical research is done, opening the way to ‘global research markets’, where competition arises from deriving understanding rather than access to samples and data. Other disciplines have embraced the benefits of global data generation and sharing, astronomy and high energy physics being two highly successful examples. We have the opportunity to mirror their successes in infrastructure funding by demonstrating that biological research can embrace the same global perspective on common infrastructure investment and data sharing.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Authors and contributions</h2>
    <p id="p-25">Dannon Baker
     <sup>1</sup>, Marius Van Den Beek
     <sup>2</sup>, Daniel Blankenberg
     <sup>3</sup>, Dave Bouvier
     <sup>4</sup>, John Chilton
     <sup>5</sup>, Nate Coraor
     <sup>6</sup>, Frederik Coppens
     <sup>7</sup>, Ignacio Eguinoa
     <sup>8</sup>, Simon Gladman
     <sup>9</sup>, Björn Grüning
     <sup>10</sup>, Delphine Larivière
     <sup>11</sup>, Andrew Lonie
     <sup>12</sup>, Sergei Kosakovsky Pond
     <sup>13</sup>, Wolfgang Maier
     <sup>14</sup>, Anton Nekrutenko
     <sup>15</sup>, James Taylor
     <sup>16</sup>, Steven Weaver
     <sup>17</sup>
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-744263562" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F5.medium.gif" width="440" height="184"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure5" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F5.medium.gif" width="440" height="184"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure5" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/25/2020.02.21.959973/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-26">Usegalaxy.org efforts are funded by NIH Grants U41 HG006620 and NSF ABI Grant 1661497. Usegalaxy.eu is supported by the German Federal Ministry of Education and Research grants 031L0101C and de.NBI-epi. Galaxy and HyPhy integration is supported by NIH grant R01 AI134384. Usegalaxy.org.au is supported by Bioplatforms Australia and the Australian Research Data Commons through funding from the Australian Government National Collaborative Research Infrastructure Strategy. Hyphy.org development team is supported by NIH grant R01GM093939. Usegalaxy.be is supported by the Research Foundation-Flanders (FWO) grant I002919N and the Flemish Supercomputer Center (VSC).</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-1">
      <p id="p-1">usegalaxy.org
       <sup>1,2,3</sup>, usegalaxy.eu
       <sup>4</sup>, usegalaxy.org.au
       <sup>5,6</sup>, usegalaxy.be
       <sup>7,8</sup> and hyphy.org
       <sup>9</sup> development teams
      </p>
     </li>
     <li class="fn-update fn-group-summary-of-updates" id="fn-2">
      <p id="p-4">General edits and updates to author affiliations and funding</p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-3">
      <p id="p-5">
       <a href="https://github.com/galaxyproject/SARS-CoV-2">https://github.com/galaxyproject/SARS-CoV-2</a>
      </p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-4">
      <p id="p-6">
       <a href="https://doi.org/10.5281/zenodo.3685264">https://doi.org/10.5281/zenodo.3685264</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.1" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X-G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>
          <span class="cit-name-given-names">H-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">C-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">H-D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">R-D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">M-Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>
          <span class="cit-name-given-names">X-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">X-S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Q-J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">L-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>
          <span class="cit-name-given-names">F-X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y-Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>
          <span class="cit-name-given-names">G-F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z-L.</span>
         </span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature [Internet]</abbr>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Feb</span>
         <span class="cit-day">3</span>; Available from: 
         <a href="http://dx.doi.org/10.1038/s41586-020-2012-7">http://dx.doi.org/10.1038/s41586-020-2012-7</a>
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">32015507</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2B%255BInternet%255D%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015507&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.21.959973v2.2" data-doi="10.1016/S0140-6736(20)30251-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bi</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">WJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">GF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet [Internet]</abbr>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Jan</span>
         <span class="cit-day">30</span>; Available from: 
         <a href="http://dx.doi.org/10.1016/S0140-6736(20)30251-8">http://dx.doi.org/10.1016/S0140-6736(20)30251-8</a>
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">32007145</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%2B%255BInternet%255D%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32007145&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.3" data-doi="10.1038/s41586-020-2008-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Y-M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">Z-G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>
          <span class="cit-name-given-names">Z-W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>
          <span class="cit-name-given-names">J-H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pei</span>
          <span class="cit-name-given-names">Y-Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">M-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dai</span>
          <span class="cit-name-given-names">F-H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Q-M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">J-J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y-Z.</span>
         </span>
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature [Internet]</abbr>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Feb</span>
         <span class="cit-day">3</span>; Available from: 
         <a href="http://dx.doi.org/10.1038/s41586-020-2008-3">http://dx.doi.org/10.1038/s41586-020-2008-3</a>
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">32015508</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2B%255BInternet%255D%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015508&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Holshue</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DeBolt</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindquist</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lofy</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wiesman</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Spitters</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ericson</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilkerson</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tural</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diaz</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cohn</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fox</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patel</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gerber</span>
          <span class="cit-name-given-names">SI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindstrom</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pallansch</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weldon</span>
          <span class="cit-name-given-names">WC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggs</span>
          <span class="cit-name-given-names">HM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Uyeki</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pillai</span>
          <span class="cit-name-given-names">SK</span>
         </span>, 
         <span class="cit-auth cit-collab">Washington State 2019-nCoV Case Investigation Team</span>. 
         <span class="cit-article-title">First Case of 2019 Novel Coronavirus in the United States</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med [Internet]</abbr>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Jan</span>
         <span class="cit-day">31</span>; Available from: 
         <a href="http://dx.doi.org/10.1056/NEJMoa2001191">http://dx.doi.org/10.1056/NEJMoa2001191</a>
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">32004427</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%2B%255BInternet%255D%26rft.volume%253D31%26rft.atitle%253DFIRST%2BCASE%2BOF%2B2019%2BNOVEL%2BCORONAVIRUS%2BIN%2BTHE%2BUNITED%2BSTATES%26rft_id%253Dinfo%253Apmid%252F32004427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=32004427&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.5" data-doi="10.1038/s41592-018-0046-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grüning</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dale</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sjödin</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chapman</span>
          <span class="cit-name-given-names">BA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rowe</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tomkins-Tinch</span>
          <span class="cit-name-given-names">CH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Valieris</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Köster</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth cit-collab">Bioconda Team</span>. 
         <span class="cit-article-title">Bioconda: sustainable and comprehensive software distribution for the life sciences</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. 
         <span class="cit-pub-date">2018</span>
         <span class="cit-month">Jul</span>;
         <span class="cit-vol">15</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">475</span>–
         <span class="cit-lpage">476</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">29967506</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D15%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41592-018-0046-7%26rft_id%253Dinfo%253Apmid%252F29967506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41592-018-0046-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29967506&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.6" data-doi="10.1186/gb-2010-11-8-r86">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Goecks</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nekrutenko</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Team</span>
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. 
         <span class="cit-auth cit-collab">Department of Biology and Department of Mathematics and Computer Science, Emory University, 1510 Clifton Road NE, Atlanta, GA 30322, USA</span>. 
         <span class="em-link">
          <span class="em-addr">jeremy.goecks{at}emory.edu</span>
         </span>: 
         <span class="cit-auth cit-collab">BioMed Central Ltd;</span>
         <span class="cit-pub-date">2010</span>
         <span class="cit-month">Jan</span>
         <span class="cit-day">1</span>;
         <span class="cit-vol">11</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">R86</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20738864&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grüning</span>
          <span class="cit-name-given-names">BA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rasche</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rebolledo-Jaramillo</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eberhard</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Houwaart</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chilton</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coraor</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Backofen</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nekrutenko</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Jupyter and Galaxy: Easing entry barriers into complex data analyses for biomedical researchers</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Comput Biol.</abbr>
         <span class="cit-pub-date">2017</span>
         <span class="cit-month">May</span>;
         <span class="cit-vol">13</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">e1005425</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span>
         <span class="cit-pub-id-scheme">PMCID: </span>
         <span class="cit-pub-id cit-pub-id-pmcid">PMC5444614</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol.%26rft.volume%253D13%26rft.spage%253De1005425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.8" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nurk</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antipov</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gurevich</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dvorkin</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kulikov</span>
          <span class="cit-name-given-names">AS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lesin</span>
          <span class="cit-name-given-names">VM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nikolenko</span>
          <span class="cit-name-given-names">SI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pham</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Prjibelski</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pyshkin</span>
          <span class="cit-name-given-names">AV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sirotkin</span>
          <span class="cit-name-given-names">AV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vyahhi</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tesler</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alekseyev</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pevzner</span>
          <span class="cit-name-given-names">PA.</span>
         </span>
         <span class="cit-article-title">SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Biol.</abbr>
         <span class="cit-pub-date">2012</span>
         <span class="cit-month">May</span>
         <span class="cit-day">1</span>;
         <span class="cit-vol">19</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22506599&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.9" data-doi="10.1371/journal.pcbi.1005595">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wick</span>
          <span class="cit-name-given-names">RR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Judd</span>
          <span class="cit-name-given-names">LM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gorrie</span>
          <span class="cit-name-given-names">CL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holt</span>
          <span class="cit-name-given-names">KE.</span>
         </span>
         <span class="cit-article-title">Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Comput Biol.</abbr>
         <span class="cit-pub-date">2017</span>
         <span class="cit-month">Jun</span>;
         <span class="cit-vol">13</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">e1005595</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span>
         <span class="cit-pub-id-scheme">PMCID: </span>
         <span class="cit-pub-id cit-pub-id-pmcid">PMC5481147</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol.%26rft.volume%253D13%26rft.spage%253De1005595%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005595%26rft_id%253Dinfo%253Apmid%252F28594827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005595&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28594827&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.10" data-doi="10.1126/science.288.5472.1789">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Korber</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muldoon</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Theiler</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lapedes</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hahn</span>
          <span class="cit-name-given-names">BH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolinsky</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhattacharya</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Timing the ancestor of the HIV-1 pandemic strains</span>. 
         <abbr class="cit-jnl-abbrev">Science.</abbr>
         <span class="cit-pub-date">2000</span>
         <span class="cit-month">Jun</span>
         <span class="cit-day">9</span>;
         <span class="cit-vol">288</span>(
         <span class="cit-issue">5472</span>):
         <span class="cit-fpage">1789</span>–
         <span class="cit-lpage">1796</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">10846155</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKorber%26rft.auinit1%253DB.%26rft.volume%253D288%26rft.issue%253D5472%26rft.spage%253D1789%26rft.epage%253D1796%26rft.atitle%253DTiming%2Bthe%2BAncestor%2Bof%2Bthe%2BHIV-1%2BPandemic%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5472.1789%26rft_id%253Dinfo%253Apmid%252F10846155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODgvNTQ3Mi8xNzg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjUvMjAyMC4wMi4yMS45NTk5NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">arambaut</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pinned</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duplessis</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Volz</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Unpinned</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Globally</span>
          <span class="cit-name-given-names">AP.</span>
         </span>
         <span class="cit-article-title">Phylodynamic Analysis | 93 genomes | 15 Feb 2020 [Internet]</span>. 
         <abbr class="cit-jnl-abbrev">Virological</abbr>. 
         <span class="cit-pub-date">2020</span> [cited 2020 Feb 17]. Available from: 
         <a href="http://virological.org/t/phylodynamic-analysis-93-genomes-15-feb-2020/356">http://virological.org/t/phylodynamic-analysis-93-genomes-15-feb-2020/356</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.12" data-doi="10.1073/pnas.1208715109">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmitt</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kennedy</span>
          <span class="cit-name-given-names">SR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salk</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fox</span>
          <span class="cit-name-given-names">EJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hiatt</span>
          <span class="cit-name-given-names">JB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loeb</span>
          <span class="cit-name-given-names">LA.</span>
         </span>
         <span class="cit-article-title">Detection of ultra-rare mutations by next-generation sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. 
         <span class="cit-auth cit-collab">Departments of Pathology, Genome Sciences, and Biochemistry, University of Washington School of Medicine, Seattle, WA 98195, USA.</span>; 
         <span class="cit-pub-date">2012</span>
         <span class="cit-month">Sep</span>
         <span class="cit-day">4</span>;
         <span class="cit-vol">109</span>(
         <span class="cit-issue">36</span>):
         <span class="cit-fpage">14508</span>–
         <span class="cit-lpage">14513</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208715109%26rft_id%253Dinfo%253Apmid%252F22853953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM2LzE0NTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjUvMjAyMC4wMi4yMS45NTk5NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.13" data-doi="10.1038/nature16988">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tortorici</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bosch</span>
          <span class="cit-name-given-names">B-J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frenz</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rottier</span>
          <span class="cit-name-given-names">PJM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DiMaio</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rey</span>
          <span class="cit-name-given-names">FA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Veesler</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</span>. 
         <abbr class="cit-jnl-abbrev">Nature.</abbr>
         <span class="cit-pub-date">2016</span>
         <span class="cit-month">Mar</span>
         <span class="cit-day">3</span>;
         <span class="cit-vol">531</span>(
         <span class="cit-issue">7592</span>):
         <span class="cit-fpage">114</span>–
         <span class="cit-lpage">117</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span>
         <span class="cit-pub-id-scheme">PMCID: </span>
         <span class="cit-pub-id cit-pub-id-pmcid">PMC5018210</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature.%26rft.volume%253D531%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16988%26rft_id%253Dinfo%253Apmid%252F26855426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature16988&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26855426&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.21.959973v2.14" data-doi="10.1016/j.virol.2005.02.022">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duquerroy</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vigouroux</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rottier</span>
          <span class="cit-name-given-names">PJM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rey</span>
          <span class="cit-name-given-names">FA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bosch</span>
          <span class="cit-name-given-names">BJ.</span>
         </span>
         <span class="cit-chapter-title">Central ions and lateral asparagine/glutamine zippers stabilize the post-fusion hairpin conformation of the SARS coronavirus spike glycoprotein</span>. 
         <span class="cit-source">Virology</span>. 
         <span class="cit-publ-name">Elsevier</span>; 
         <span class="cit-pub-date">2005</span>
         <span class="cit-month">May</span>
         <span class="cit-day">10</span>;
         <span class="cit-vol">335</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">276</span>–
         <span class="cit-lpage">285</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">15840526</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DDuquerroy%26rft.auinit1%253DS.%26rft.volume%253D335%26rft.issue%253D2%26rft.spage%253D276%26rft.epage%253D285%26rft.atitle%253DCentral%2Bions%2Band%2Blateral%2Basparagine%252Fglutamine%2Bzippers%2Bstabilize%2Bthe%2Bpost-fusion%2Bhairpin%2Bconformation%2Bof%2Bthe%2BSARS%2Bcoronavirus%2Bspike%2Bglycoprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2005.02.022%26rft_id%253Dinfo%253Apmid%252F15840526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2005.02.022&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15840526&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.15" data-doi="10.1093/molbev/msl051">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kosakovsky Pond</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Posada</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gravenor</span>
          <span class="cit-name-given-names">MB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Woelk</span>
          <span class="cit-name-given-names">CH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frost</span>
          <span class="cit-name-given-names">SDW.</span>
         </span>
         <span class="cit-article-title">Automated phylogenetic detection of recombination using a genetic algorithm</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol.</abbr>
         <span class="cit-pub-date">2006</span>
         <span class="cit-month">Oct</span>;
         <span class="cit-vol">23</span>(
         <span class="cit-issue">10</span>):
         <span class="cit-fpage">1891</span>–
         <span class="cit-lpage">1901</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">16818476</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl051%26rft_id%253Dinfo%253Apmid%252F16818476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msl051&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16818476&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240586600009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Batut</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hiltemann</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bagnacani</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baker</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhardwaj</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blank</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bretaudeau</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brillet-Guéguen</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Čech</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chilton</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clements</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Doppelt-Azeroual</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Erxleben</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Freeberg</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gladman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoogstrate</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hotz</span>
          <span class="cit-name-given-names">H-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Houwaart</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jagtap</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Larivière</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Le Corguillé</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manke</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mareuil</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ramírez</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryan</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sigloch</span>
          <span class="cit-name-given-names">FC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Soranzo</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolff</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Videm</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolfien</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wubuli</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yusuf</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth cit-collab">Galaxy Training Network</span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Backofen</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nekrutenko</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grüning</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">Community-Driven Data Analysis Training for Biology</span>. 
         <abbr class="cit-jnl-abbrev">Cell Syst.</abbr>
         <span class="cit-pub-date">2018</span>
         <span class="cit-month">Jun</span>
         <span class="cit-day">27</span>;
         <span class="cit-vol">6</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">752</span>–
         <span class="cit-lpage">758.e1</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span>
         <span class="cit-pub-id-scheme">PMCID: </span>
         <span class="cit-pub-id cit-pub-id-pmcid">PMC6296361</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D6%26rft.spage%253D752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.21.959973v2.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">CDC</span>. 
         <span class="cit-article-title">2019 Novel Coronavirus (2019-nCoV) [Internet]</span>. 
         <span class="cit-source">Centers for Disease Control and Prevention</span>. 
         <span class="cit-pub-date">2020</span> [cited 2020 Feb 19]. Available from: 
         <a href="https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html">https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Garrison</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Haplotype-based variant detection from short-read sequencing</span>. 
         <abbr class="cit-jnl-abbrev">arXiv.org [Internet]. Cornell University Library</abbr>; 
         <span class="cit-pub-date">2012</span>
         <span class="cit-month">Jul</span>
         <span class="cit-day">17</span>;
         <span class="cit-fpage">q-bio.GN</span>. Available from: 
         <a href="http://arxiv.org/abs/1207.3907v2">http://arxiv.org/abs/1207.3907v2</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.21.959973v2.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">Common Workflow Language [Internet]</span>. [cited 2020 Feb 21]. Available from: 
         <a href="https://www.commonwl.org/">https://www.commonwl.org/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.21.959973v2.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">Nextflow - A DSL for parallel and scalable computational pipelines [Internet]</span>. [cited 2020 Feb 21]. Available from: 
         <a href="https://www.nextflow.io/">https://www.nextflow.io/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959973v2.21" data-doi="10.1093/bioinformatics/bts480">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Köster</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rahmann</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Snakemake--a scalable bioinformatics workflow engine</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics.</abbr>
         <span class="cit-pub-date">2012</span>
         <span class="cit-month">Oct</span>
         <span class="cit-day">1</span>;
         <span class="cit-vol">28</span>(
         <span class="cit-issue">19</span>):
         <span class="cit-fpage">2520</span>–
         <span class="cit-lpage">2522</span>.
         <span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span>
         <span class="cit-pub-id-scheme">PMID: </span>
         <span class="cit-pub-id cit-pub-id-pmid">22908215</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts480%26rft_id%253Dinfo%253Apmid%252F22908215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts480&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22908215&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F25%2F2020.02.21.959973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000309687500018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
