<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5864297">5864297</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29485041">29485041</a></span><span class="publisher-id" title="publisher-id">30756</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.30756">10.7554/eLife.30756</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Epidemiology and Global Health</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Conjunction of factors triggering waves of seasonal influenza</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">Ishanu</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8339-8162</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiciman</span><span class="given-names" tagx="given-names" title="given-names">Emre</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5429-468X</span><a href="#aff3">3</a><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elliott</span><span class="given-names" tagx="given-names" title="given-names">Joshua W</span></span><a href="#aff4">4</a><a href="#con3" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey L</span></span><a href="#aff5">5</a><a href="#fund1" /><a href="#fund2" /><a href="#fund3" /><a href="#con4" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rzhetsky</span><span class="given-names" tagx="given-names" title="given-names">Andrey</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6959-7405</span><span class="email" tagx="email" title="email">arzhetsky@uchicago.edu</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#aff4">4</a><a href="#aff6">6</a><a href="#fund4" /><a href="#fund5" /><a href="#fund6" /><a href="#con5" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Institute of Genomics and Systems Biology</span><span class="institution" title="institution">University of Chicago</span><span class="addr-line" title="addr-line">Chicago</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Department of Medicine</span><span class="institution" title="institution">University of Chicago</span><span class="addr-line" title="addr-line">Chicago</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Information and Data Science Group</span><span class="institution" title="institution">Microsoft Research</span><span class="addr-line" title="addr-line">Redmond</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Computation Institute</span><span class="institution" title="institution">University of Chicago</span><span class="addr-line" title="addr-line">Chicago</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Department of Environmental Health Sciences, Mailman School of Public Health</span><span class="institution" title="institution">Columbia University</span><span class="addr-line" title="addr-line">New York</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Departments of Human Genetics</span><span class="institution" title="institution">University of Chicago</span><span class="addr-line" title="addr-line">Chicago</span><span class="country" tagx="country" title="country">United States</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jit</span><span class="given-names" tagx="given-names" title="given-names">Mark</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="aff7">[], <span class="institution" title="institution">London School of Hygiene &amp;amp; Tropical Medicine, and Public Health England</span><span class="country" tagx="country" title="country">United Kingdom</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2018-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e30756</span><span class="history" title="history"><span class="received" title="received">received: 2017-7-26</span><span class="accepted" title="accepted">accepted: 2018-2-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Chattopadhyay et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license">[license_ref] http://creativecommons.org/licenses/by/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-30756.pdf" title="self-uri">elife-30756.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Using several longitudinal datasets describing putative factors affecting influenza incidence and clinical data on the disease and health status of over 150 million human subjects observed over a decade, we investigated the source and the mechanistic triggers of influenza epidemics. We conclude that the initiation of a pan-continental influenza wave emerges from the simultaneous realization of a complex set of conditions. The strongest predictor groups are as follows, ranked by importance: (1) the host population’s socio- and ethno-demographic properties; (2) weather variables pertaining to specific humidity, temperature, and solar radiation; (3) the virus’ antigenic drift over time; (4) the host population’€™s land-based travel habits, and; (5) recent spatio-temporal dynamics, as reflected in the influenza wave auto-correlation. The models we infer are demonstrably predictive (area under the Receiver Operating Characteristic curve 80%) when tested with out-of-sample data, opening the door to the potential formulation of new population-level intervention and mitigation policies.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
eLife digest</div> <p>Influenza – or ‘the flu’ – is a contagious disease which sweeps across the globe like clockwork, claiming tens of thousands of lives. This is known as ‘seasonal flu’.</p> <p>Many scientists have tried to identify the factors that spark these yearly outbreaks. Some past studies have found that seasonal flu occurs when air that is normally humid turns dry, suggesting weather patterns play an important part. Other research has shown that air travel contributes to the flu spreading across the world. However, these studies typically focus on just one or two factors on their own. It is still not clear how exactly these factors combine to drive outbreaks, and then sustain the wave of infection.</p> <p>To address this, Chattopadhyay et al. analyze the medical histories of 150 million American people over a decade, combining this information with large datasets about the different factors that trigger flu outbreaks. This includes detailed data about air travel and weather patterns, as well as census data that describe features of the population. Patterns of movement are also examined, for example by processing billions of Twitter messages "tagged�? with a location. Chattopadhyay et al. used all of these datasets to model outbreaks of the flu in the United States, and see which factors play the biggest role.</p> <p>It turns out that yearly outbreaks of seasonal flu are a result of a combination of elements. Some factors interact to help trigger the start of the wave, like humid weather in a highly populated area with nearby airports. Other factors, such how people move, encourage the spread of the infection. Finally, certain features of the population, for example how closely knitted a community is, make specific areas of the country more susceptible to the arrival of the disease. Overall, some of the most important elements of the model relate to the characteristics of the populations, the weather, the type of virus, and the number of short-distance journeys (rather than air travel).</p> <p>Understanding how and why outbreaks occur can help policy-makers design strategies that reduce the spread and impact of seasonal flu, which could potentially save thousands of lives. Ultimately, the model developed by Chattopadhyay et al. could be used to test whether these policies would work before they are implemented in the real world.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
poisson regression</div> <div class="kwd" title="kwd">
Granger causality</div> <div class="kwd" title="kwd">
matching analysis</div> <div class="kwd" title="kwd">
US influenza epidemic</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Virus</div> <div class="kwd" title="kwd">
Human</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01HL108634-01</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey L</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01GM100467</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey L</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01GM110748</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey L</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">1P50MH094267</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rzhetsky</span><span class="given-names" tagx="given-names" title="given-names">Andrey</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100006502</span><span class="institution" title="institution">Defense Sciences Office, DARPA</span></div> </div><span class="award-id" tagx="award-id" title="award-id">W911NF1410333</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rzhetsky</span><span class="given-names" tagx="given-names" title="given-names">Andrey</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01HL122712</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rzhetsky</span><span class="given-names" tagx="given-names" title="given-names">Andrey</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="Three complementary computational approaches reveal a set of putative causes of initiation of pan-USA waves of influenza." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Seasonal influenza is a serious threat to public health, claiming tens of thousands of lives every year. A large number of past studies have focused on identifying the likely factors responsible for initiating each seasonal disease wave. Typically, each such study focused on one or a few hypothetical factors. Our study aimed at an integrative, joint analysis of numerous suggested disease triggers, comparing their relative importance and possible cooperation in triggering pan-US waves of seasonal influenza infection. The goal of this study was to identify the most informative combinations of statistical predictors associated with the initiation of pan-US influenza infection waves.</p> <div class="recentcomputationalstudiesofinfluenza: " title="sec"> <div class="title" tagx="title" title="title">
Recent computational studies of influenza: </div> <p>Computational study of the dynamics and factors influencing infectious disease spread began with compartmental models, such as the Susceptible-Infected-Resistant (SIR) model, which traces its origins to the beginning of the last century (<a href="#bib57">Kermack and McKendrick, 1927</a>). Initially a purely theoretical tool, these SIR-style models were subsequently enhanced with population and geographic data, allowing their application to specific cities and the distances between them (<a href="#bib55">Keeling and Rohani, 2002</a>). For instance, one approach, termed ‘gravity wave’ modeling, used geographic, short- and long-range, work-related human movement and demographic data to formulate gravity potentials between US counties in order to infer the dynamics of infection spread (<a href="#bib85">Viboud et al., 2006</a>).</p> <p>Some studies have focused on one specific factor affecting infection, such as air travel, to simulate the spread of influenza (<a href="#bib27">Colizza et al., 2006</a>); other studies used SIR models, generalized for a collection of interconnected geographic areas, spatial-network or patch models, to model a number of common infections, including influenza, measles, and foot-and-mouth disease (<a href="#bib72">Riley, 2007</a>).</p> <p>More ambitious network model approaches have simulated the global transmission of infectious disease using high-resolution, worldwide population data and the locations of International Air Transport Association (IATA)-indexed airports (<a href="#bib8">Balcan et al., 2009</a>). Similar to (<a href="#bib85">Viboud et al., 2006</a>), the authors of the study computed the global infection-pre-disposing ‘gravity field’ over the network of international airports. This network-based approach was subsequently developed further (<a href="#bib9">Balcan and Vespignani, 2011</a>) through the modeling of ‘phase transition’–that is the chain-reaction switch of geographic infection status–in complex networks, utilizing approaches introduced in theoretical physics.</p> <p>Another layer of sophistication was achieved by incorporating rich historical records. For example, Eggo <i>et al.</i> (<a href="#bib36">Eggo et al., 2011</a>) modeled the Spanish influenza epidemic of 1918–1919, using mortality documents from both the UK and the US, explicitly accounting for the size and distances between cities. In the same spirit, Brockmann and Helbing (<a href="#bib16">Brockmann and Helbing, 2013</a>) represented infection as diffusion on a complex network, estimating arrival times for infection across the globe.</p> <p>Following the formulation of the hypothesis that absolute humidity modulates influenza survival and transmission (<a href="#bib76">Shaman and Kohn, 2009</a>), researchers began incorporating climate variables into SIR-like models (<a href="#bib24">Chowell et al., 2012</a>). More recent dynamic models have incorporated a probabilistic description of influenza infection’s spatial transitions in space and time, accounting for selected demographic confounders (<a href="#bib45">Gog et al., 2014</a>) and (<a href="#bib19">Charu et al., 2017</a>).</p> <p>In this study, rather than following the SIR-style modeling tradition, we used statistical epidemiology- and econometric-like approaches, in addition to a causality-network method presented here for the first time. There are some prior studies that are close to ours in spirit (but not in details). For example, (<a href="#bib10">Barreca and Shimshack, 2012</a>) used historical US influenza mortality data (1973–2002) in conjunction with collinear humidity and temperature records to establish county-level statistical associations between variables in the datasets. They concluded that absolute humidity was ‘an especially critical determinant of observed human influenza mortality, even after controlling for temperature.’ Another study, focusing on historical influenza records in the Netherlands, (<a href="#bib80">te Beest et al., 2013</a>) used the number of weekly influenza-like patient visits (transformed into an estimated rate of infection) as a response variable in a regression analysis of climate data. They concluded that the bulk of explained variation (57%) was attributed to the depletion of susceptible hosts during the disease season and non-weather-related ‘between-season effects,’ with only 3% explained by absolute humidity, represented as a continuous predictor variable. Additionally, this study observed that school holidays did not have a statistically significant effect on influenza transmission.</p> <p>As all causality detection methods come with dissimilar limitations and are imperfect in unique ways, we designed our study intentionally to attack the same target problem using three different statistical approaches: Approach 1: A non-parametric Granger analysis (<a href="#bib46">Granger, 1980</a>) focusing on infection flows’ directionalities across the US and whether influenza propagates via long- vs. short-distance travel (we run analysis across all pairs of air- and land-travel county neighbors, respectively). Approach 2: A mixed-effect Poisson regression (<a href="#bib49">Hedeker and Gibbons, 2006</a>) explicitly accounting for the auto-correlation of infection waves in time and space, along with the full set of socioeconomic, climate, and geographic predictors. Approach 3: A county-matching, non-parametric analysis to identify the minimum predictive set of factors that distinguish those counties associated with the onset of the influenza season (<a href="#bib64">Morgan and Winship, 2015</a>).</p> <p>Our study became possible through access to several, very large longitudinal datasets: (1) a nine-year collection of insurance records capturing the dynamics of influenza-like illnesses (ILIs) in the United States (Truven MarketScan database, see Materials and methods); (2) temporally collinear, high-resolution weather measurements over every US county; (3) detailed air travel (<a href="#bib81">The United States Bureau of Transportation Statistics, 2010</a>) and geographic proximity data (<a href="#bib82">The United States Census, 2016</a>) showing connectivity between US counties; (4) billions of geo-located Twitter messages reflecting long- and short-distance human movement patterns, and; (5) US census data accounting for US county and county-equivalent population distribution, demographic, and socioeconomic properties (<a href="#bib52">HRSA, 2016</a>). An explicit comparison of the ILI data in the insurance claims to the influenza records provided by the Center for Disease Control and Prevention (<a href="#bib17">CDC, 2016</a>) showed that the two sources agree well (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.91</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3.5</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
201</div> </div> </div> </div> </div> </div></div></span>), with insurance claims providing higher data resolution, see <a href="#fig1s1">Figure 1—figure supplement 1</a>. Curiously, the relationship between the two sources of ILI observations is not linear: We attribute this to the lower resolution of the CDC data. These three types of analysis produce congruent–albeit not identical–results.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>The logical flow of our analysis is as follows: (1) We first show that our definition of ILIs corresponds well with CDC data, and that our causality coefficients, defined in Approach 1, have similar meanings to coefficients in the regression analysis; (2) We then explain the outcomes of the analysis according to Approach 1; (3) We then analyze the importance of putative casual factors, <a href="#fig1">Figure 1</a>, as applied to an initiation of influenza season, <a href="#fig2">Figure 2</a>; (4) In Approach 2, we pay special attention to the relative importance of short- and long-distance travel in influenza propagation, Figure 4; (5) We further test the best regression model in terms of predictive accuracy, Figure 5, using disjointed data parts for training and testing, and; (6) We culminate our analysis with a county-matching analysis, Figure 6.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.003"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Putative determinants of seasonal influenza onset in the continental US and Poisson mixed-effect regression analysis (Approach 2).</div> <p>Plate A shows the significant variables along with their computed influence coefficients from the mixed-effect Poisson regression analysis (the best model chosen from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
126</div></div></span> different regression equations with different variable combinations). The statistically significant estimates of fixed effects are grouped into several classes: climate variables, economic and demographic variables, auto-regression variables, variables related to travel, and those related to antigenic diversity (see the last entry in <a href="#table5">Table 5</a> for the detailed regression equation used. The complete list of all models considered is given in Table S-D7). The fixed-effect regression coefficients plotted in Plate A are shown on a logarithmic scale, meaning that the absolute magnitude of predictor-specific effect is obtained by exponentiating the parameter value. A negative coefficient for a predictor variable suggests that the influenza rate falls as this factor increases, while a positive coefficient predicts a growing rate of infection as the parameter value grows. The integrated influence of individual predictors, under this model, is additive with respect to the county-specific rate of infection. For example, a coefficient of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
0.6</div> </div></div></span> for parameter AVG_PRESS_mean tells us that the average atmospheric pressure has a negative association with the influenza rate. As the mean atmospheric pressure for the county grows, the probability that the county would participate in an infection initiation wave falls. As <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
0.6</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.54</div> </div></div></span>, the rate of infection drops by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
46</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> when atmospheric pressure increases by one unit of zero-centered and standard-deviation-normalized atmospheric pressure. Similarly, an increase in the share of a white Hispanic population predicts an increase in influenza rate: A coefficient of 1.3 translates into a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1.3</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
267</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span> rate increase, possibly, because of the higher social network connectivity associated with this segment of population. Plates B - I enumerate the average spatial distribution of a few key significant factors considered in Poisson regression: (<b>B</b>) average temperature; (<b>C</b>) average maximum specific humidity; (<b>D</b>) average wind velocity in miles per hour; (<b>E</b>) average solar flux; (<b>F</b>) logarithm of population density (people per square mile); (<b>G</b>) total precipitation; (<b>H</b>) income, and; (<b>I</b>) percent of poor as deviations about the country average. Plates J-M show the strong dependence between our estimated antigenic diversity (normalized, see Definition in text) corresponding to the HA, NA, M1, and M2 viral proteins, and the cumulative fraction of the inoculated population (normalized between 0 and 1), where both sets of variables are geo-spatially and temporally stratified. Pearson’s correlation tests shown in Plates J-M were performed under null hypothesis that there the two quantities (plotted along axes X and Y) are statistically independent (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.010"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Characteristics of seasonal influenza in the continental US An analysis of county-specific, weekly reports on the number of influenza cases for a period of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
471</div></div></span> weeks spanning January 2003 to December 2013 (Plates A-H) for recurrent patterns of disease propagation.</div> <p>In particular, the weeks leading up to that in which an epidemic season peaks (determined by significant infection reports from the maximum number of counties for that season) demonstrate an apparent flow of disease from south to north, which cannot be explained by population density alone (also see movie in Supplement). Plate I illustrates the near-perfect time table for a seasonal epidemic. Plates J and K compare the county-specific initiation probabilities of an influenza season, and the causality streamlines.</p>   </div> <div class="table-wrap_UNKNOWN" id="table1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.30756.011">10.7554/eLife.30756.011</a><span class="label" tagx="label" title="label">Table 1.</span> <div class="title" tagx="title" title="title">
Social connectivity: The US Southern region appears to have an unusually high level of social connectivity.</div> <p>(In GSS survey results, the number of close friends, close friends who are neighbors, and number of friends who all or mostly know each other is higher in the South, especially in the East/South/Central census region, than in the country at large.)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1">WSC (TX, OK, AR, LA)</th> <th rowspan="1" colspan="1">ESC (MS, AL, TN, KY)</th> <th rowspan="1" colspan="1">SA (FL, GA, SC, NC, VA, WV, MD, DC)</th> <th rowspan="1" colspan="1">Country-at-large</th> <th rowspan="1" colspan="1">WNC (ND, SD, NE, KS, MO, IA, MN) (not in South/Southeast) this is the second most social region following ESC (MS, AL, TN, KY)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Close friends</td> <td rowspan="1" colspan="1">7.22</td> <td rowspan="1" colspan="1">12.76</td> <td rowspan="1" colspan="1">8.20</td> <td rowspan="1" colspan="1">7.57</td> <td rowspan="1" colspan="1">10.56</td> </tr> <tr> <td rowspan="1" colspan="1">Close friends who are neighbors</td> <td rowspan="1" colspan="1">1.02</td> <td rowspan="1" colspan="1">3.40</td> <td rowspan="1" colspan="1">1.32</td> <td rowspan="1" colspan="1">1.45</td> <td rowspan="1" colspan="1">3.15</td> </tr> <tr> <td rowspan="1" colspan="1">% of friends who all or mostly know each other</td> <td rowspan="1" colspan="1">All:20% Mostly: 43%</td> <td rowspan="1" colspan="1">All:18% Mostly: 58%</td> <td rowspan="1" colspan="1">All: 11% Mostly: 52%</td> <td rowspan="1" colspan="1">All: 12 Mostly: 50%</td> <td rowspan="1" colspan="1">All: 16% Mostly: 58%</td> </tr> <tr> <td rowspan="1" colspan="1">How often visit closest friends*</td> <td rowspan="1" colspan="1">107</td> <td rowspan="1" colspan="1">151</td> <td rowspan="1" colspan="1">126</td> <td rowspan="1" colspan="1">122</td> <td rowspan="1" colspan="1">129</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>*Survey options are: lives in household, daily, several times a week, once a week, once a month, several times a year, and less often. These are converted to approximate number of visits per year (see Supplement for more information about the GSS analysis).</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="table2" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.30756.012">10.7554/eLife.30756.012</a><span class="label" tagx="label" title="label">Table 2.</span> <div class="title" tagx="title" title="title">
Fisher’s exact test results on matched treatment combinations</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">YR</th> <th rowspan="1" colspan="1">dh<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="unknown" title="mmultiscripts" tag="mmultiscripts"> <div class="mi_UNKNOWN" /> <div class="unknown" title="none" tag="none" /> <div class="mo_UNKNOWN">
⋆</div> <div class="unknown" title="mprescripts" tag="mprescripts" /> <div class="mn_UNKNOWN">
0</div> <div class="unknown" title="none" tag="none" /> </div></div></span></th> <th rowspan="1" colspan="1">dh<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="unknown" title="mmultiscripts" tag="mmultiscripts"> <div class="mi_UNKNOWN" /> <div class="unknown" title="none" tag="none" /> <div class="mo_UNKNOWN">
⋆</div> <div class="unknown" title="mprescripts" tag="mprescripts" /> <div class="mn_UNKNOWN">
1</div> <div class="unknown" title="none" tag="none" /> </div></div></span></th> <th rowspan="1" colspan="1">dt<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="unknown" title="mmultiscripts" tag="mmultiscripts"> <div class="mi_UNKNOWN" /> <div class="unknown" title="none" tag="none" /> <div class="mo_UNKNOWN">
⋆</div> <div class="unknown" title="mprescripts" tag="mprescripts" /> <div class="mn_UNKNOWN">
0</div> <div class="unknown" title="none" tag="none" /> </div></div></span></th> <th rowspan="1" colspan="1">h<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">t<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">u<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">M1<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">M2<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">V<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="unknown" title="mmultiscripts" tag="mmultiscripts"> <div class="mi_UNKNOWN" /> <div class="unknown" title="none" tag="none" /> <div class="mo_UNKNOWN">
⋆</div> <div class="unknown" title="mprescripts" tag="mprescripts" /> <div class="mn_UNKNOWN">
0</div> <div class="unknown" title="none" tag="none" /> </div></div></span></th> <th rowspan="1" colspan="1">V<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="unknown" title="mmultiscripts" tag="mmultiscripts"> <div class="mi_UNKNOWN" /> <div class="unknown" title="none" tag="none" /> <div class="mo_UNKNOWN">
⋆</div> <div class="unknown" title="mprescripts" tag="mprescripts" /> <div class="mn_UNKNOWN">
1</div> <div class="unknown" title="none" tag="none" /> </div></div></span></th> <th rowspan="1" colspan="1">a<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
⋆</div> </div></div></span></th> <th rowspan="1" colspan="1">p-value</th> <th rowspan="1" colspan="1">Odds ratio</th> <th rowspan="1" colspan="1">Lower 99% cnf. bnds.</th> <th rowspan="1" colspan="1">Upper 99% cnf. bnds.</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">2003</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.9</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
8</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">2.83</td> <td rowspan="1" colspan="1">1.73</td> <td rowspan="1" colspan="1">4.66</td> </tr> <tr> <td rowspan="1" colspan="1">2004</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
6.5</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
3</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">6.22</td> <td rowspan="1" colspan="1">1.08</td> <td rowspan="1" colspan="1">132.03</td> </tr> <tr> <td rowspan="1" colspan="1">2005</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3.4</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
6</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">8.31</td> <td rowspan="1" colspan="1">2.16</td> <td rowspan="1" colspan="1">54.93</td> </tr> <tr> <td rowspan="1" colspan="1">2006</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
5.3</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
7</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">4.56</td> <td rowspan="1" colspan="1">1.96</td> <td rowspan="1" colspan="1">12.0</td> </tr> <tr> <td rowspan="1" colspan="1">2007</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2.1</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">3.85</td> <td rowspan="1" colspan="1">0.82</td> <td rowspan="1" colspan="1">28.16</td> </tr> <tr> <td rowspan="1" colspan="1">2008</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.9</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
3</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">5.26</td> <td rowspan="1" colspan="1">1.23</td> <td rowspan="1" colspan="1">50.2</td> </tr> <tr> <td rowspan="1" colspan="1">2009</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3.1</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
10</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">4.78</td> <td rowspan="1" colspan="1">2.38</td> <td rowspan="1" colspan="1">10.34</td> </tr> <tr> <td rowspan="1" colspan="1">2010</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.4</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">3.64</td> <td rowspan="1" colspan="1">0.93</td> <td rowspan="1" colspan="1">24.27</td> </tr> <tr> <td rowspan="1" colspan="1">2011</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4.9</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
11</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">4.91</td> <td rowspan="1" colspan="1">2.51</td> <td rowspan="1" colspan="1">10.05</td> </tr> <tr> <td rowspan="1" colspan="1">All Years</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">X</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
7.2</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
9</div> </div> </div> </div></div></span></td> <td rowspan="1" colspan="1">3.88</td> <td rowspan="1" colspan="1">2.10</td> <td rowspan="1" colspan="1">7.89</td> </tr> <tr> <td rowspan="1" colspan="1">All Years</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1">Y</td> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1.0</div></div></span></td> <td rowspan="1" colspan="1">1.0</td> <td rowspan="1" colspan="1">0.48</td> <td rowspan="1" colspan="1">2.15</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="table3" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.30756.013">10.7554/eLife.30756.013</a><span class="label" tagx="label" title="label">Table 3.</span> <div class="title" tagx="title" title="title">
Fisher’s exact test results on matched treatment combinations</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1">(a)</th> <th valign="top" rowspan="1" colspan="1" /> <th valign="top" rowspan="1" colspan="1" /> </tr> <tr> <th valign="top" rowspan="1" colspan="1">YR p</th> <th valign="top" rowspan="1" colspan="1">-value 99%</th> <th valign="top" rowspan="1" colspan="1">Conf. Bnd.</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">max hus avg</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1">0.003603</td> <td valign="top" rowspan="1" colspan="1">1.0, 4.21</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1">0.6919</td> <td valign="top" rowspan="1" colspan="1">0.16, 17.63</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005 0.</td> <td valign="top" rowspan="1" colspan="1">1948</td> <td valign="top" rowspan="1" colspan="1">0.61, 7.89</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006 0.</td> <td valign="top" rowspan="1" colspan="1">6525 0.28, 3.06</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007 0.</td> <td valign="top" rowspan="1" colspan="1">3574 0.49, 18.85</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008 0.</td> <td valign="top" rowspan="1" colspan="1">103</td> <td valign="top" rowspan="1" colspan="1">0.55, 1.23</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009 0</td> <td valign="top" rowspan="1" colspan="1">.1067</td> <td valign="top" rowspan="1" colspan="1">0.68, 8.77</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 0.5318</td> <td valign="top" rowspan="1" colspan="1"> 0.27, 41.03</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1">0.09054</td> <td valign="top" rowspan="1" colspan="1">0.74, 5.17</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1">1[1]10-4</td> <td valign="top" rowspan="1" colspan="1">1.12, 1.88</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">t avg mean</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1">0.06439</td> <td valign="top" rowspan="1" colspan="1">0.81, 3.65</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">0.27, 10.62</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1">0.003339</td> <td valign="top" rowspan="1" colspan="1">1.17, 123.0</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1"> 0.8172</td> <td valign="top" rowspan="1" colspan="1">0.29, 4.0</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1">0.537</td> <td valign="top" rowspan="1" colspan="1">0.47, 7.42</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1">0.05985</td> <td valign="top" rowspan="1" colspan="1">0.59, Inf</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1"> 0.0006337</td> <td valign="top" rowspan="1" colspan="1">1.37, 51.68</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 0.2853</td> <td valign="top" rowspan="1" colspan="1"> 0.50, 9.28</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1"> 0.05729</td> <td valign="top" rowspan="1" colspan="1"> 0.85, 3.49</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS 5.87</td> <td valign="top" rowspan="1" colspan="1">[1]10-9</td> <td valign="top" rowspan="1" colspan="1">1.36, 2.23</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">d hus 0</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1"> 0.5374</td> <td valign="top" rowspan="1" colspan="1"> 0.55, 3.41</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">0.27, 11.01</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1">0.04401</td> <td valign="top" rowspan="1" colspan="1"> 0.81, 7.13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1"> 0.001708</td> <td valign="top" rowspan="1" colspan="1">1.31, Inf</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1">0.009199</td> <td valign="top" rowspan="1" colspan="1"> 1.0, 37.34</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1">0.3051</td> <td valign="top" rowspan="1" colspan="1">0.60, 5.92</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1">0.02726</td> <td valign="top" rowspan="1" colspan="1"> 0.82, 90.16</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 1</td> <td valign="top" rowspan="1" colspan="1">0.41, 2.78</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1"> 0.577</td> <td valign="top" rowspan="1" colspan="1">0.57, 2.77</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1">1.48[1]10-5</td> <td valign="top" rowspan="1" colspan="1">1.12, 1.64</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">d t avg mean 0</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1"> 0.004956</td> <td valign="top" rowspan="1" colspan="1">1.0, 24.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1">0.445</td> <td valign="top" rowspan="1" colspan="1">0.14, 6.0</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1">0.001164</td> <td valign="top" rowspan="1" colspan="1"> 1.23, 12.03</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1"> 0.01198</td> <td valign="top" rowspan="1" colspan="1">0.97, 11.08</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1">0.01147</td> <td valign="top" rowspan="1" colspan="1">0.96, 11.05</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1"> 0.08552</td> <td valign="top" rowspan="1" colspan="1"> 0.74, 11.18</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1">0.06847</td> <td valign="top" rowspan="1" colspan="1"> 0.73, 17.69</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1">0.08251</td> <td valign="top" rowspan="1" colspan="1">0.15, 1.63</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1">0.6031</td> <td valign="top" rowspan="1" colspan="1">0.47, 3.55</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL</td> <td valign="top" rowspan="1" colspan="1"> YRS 4.98[1]10-11</td> <td valign="top" rowspan="1" colspan="1">1.35, 2.06</td> </tr> </tbody> </table> <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1">(b)</th> <th valign="top" rowspan="1" colspan="1" /> <th valign="top" rowspan="1" colspan="1" /> </tr> <tr> <th valign="top" rowspan="1" colspan="1">YR p-</th> <th valign="top" rowspan="1" colspan="1">value</th> <th valign="top" rowspan="1" colspan="1"> 99% Conf. Bnd.</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">hus 1</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1">0.1652</td> <td valign="top" rowspan="1" colspan="1"> 0.10, 2.43</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1"> 1</td> <td valign="top" rowspan="1" colspan="1"> 0.22, 24.42</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1"> 0.002004</td> <td valign="top" rowspan="1" colspan="1"> 0.09, 0.87</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1"> 0.32, 7.65</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1"> 0.389</td> <td valign="top" rowspan="1" colspan="1">0.33, 1.90</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1">0.02142</td> <td valign="top" rowspan="1" colspan="1"> 0.9, 8.48</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1"> 0.1822</td> <td valign="top" rowspan="1" colspan="1"> 0.67, 6.06</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1">0.6005</td> <td valign="top" rowspan="1" colspan="1"> 0.23, 4.22</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1"> 0.9166</td> <td valign="top" rowspan="1" colspan="1"> 0.6, 1.88</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1">0.07</td> <td valign="top" rowspan="1" colspan="1"> 0.72, 1.06</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">d hus 2</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1"> 0.0083</td> <td valign="top" rowspan="1" colspan="1"> 1.01, 5.77</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1"> 0.79</td> <td valign="top" rowspan="1" colspan="1">0.36, 13.33</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1"> 0.275</td> <td valign="top" rowspan="1" colspan="1"> 0.71, 2.54</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1"> 0.24</td> <td valign="top" rowspan="1" colspan="1"> 0.66, 4.36</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1"> 0.19</td> <td valign="top" rowspan="1" colspan="1"> 0.62, 9.33</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1"> 0.18</td> <td valign="top" rowspan="1" colspan="1"> 0.65, 7.52</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1"> 0.53</td> <td valign="top" rowspan="1" colspan="1"> 0.44, 6.25</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 0.08</td> <td valign="top" rowspan="1" colspan="1"> 0.21, 1.51</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1">0.59</td> <td valign="top" rowspan="1" colspan="1"> 0.69, 1.87</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1"> 0.13</td> <td valign="top" rowspan="1" colspan="1"> 0.78, 1.07</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">urbanity</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1">0.0083</td> <td valign="top" rowspan="1" colspan="1"> 1.01, 5.77</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1"> 0.79</td> <td valign="top" rowspan="1" colspan="1"> 0.36, 13.33</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1"> 0.275</td> <td valign="top" rowspan="1" colspan="1">0.71, 2.54</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1">0.24</td> <td valign="top" rowspan="1" colspan="1"> 0.66, 4.36</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1"> 0.19</td> <td valign="top" rowspan="1" colspan="1"> 0.62, 9.33</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1"> 0.18</td> <td valign="top" rowspan="1" colspan="1">0.65, 7.52</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1">0.53</td> <td valign="top" rowspan="1" colspan="1"> 0.44, 6.25</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 0.08</td> <td valign="top" rowspan="1" colspan="1"> 0.21, 1.51</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1"> 0.59</td> <td valign="top" rowspan="1" colspan="1"> 0.69, 1.87</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1">2.2_10-16</td> <td valign="top" rowspan="1" colspan="1"> 3.67, 5.06</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">airport proximity</td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2003</td> <td valign="top" rowspan="1" colspan="1"> 0.004956</td> <td valign="top" rowspan="1" colspan="1"> 1.0, 24.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2004</td> <td valign="top" rowspan="1" colspan="1">0.445</td> <td valign="top" rowspan="1" colspan="1"> 0.14, 6.0</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2005</td> <td valign="top" rowspan="1" colspan="1"> 0.001164</td> <td valign="top" rowspan="1" colspan="1">1.23, 12.03</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2006</td> <td valign="top" rowspan="1" colspan="1"> 0.01198</td> <td valign="top" rowspan="1" colspan="1"> 0.97, 11.08</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2007</td> <td valign="top" rowspan="1" colspan="1"> 0.01147</td> <td valign="top" rowspan="1" colspan="1"> 0.96, 11.05</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2008</td> <td valign="top" rowspan="1" colspan="1"> 0.08552</td> <td valign="top" rowspan="1" colspan="1"> 0.74, 11.18</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2009</td> <td valign="top" rowspan="1" colspan="1">0.06847</td> <td valign="top" rowspan="1" colspan="1">0.73, 17.69</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2010</td> <td valign="top" rowspan="1" colspan="1"> 0.08251</td> <td valign="top" rowspan="1" colspan="1"> 0.15, 1.63</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">2011</td> <td valign="top" rowspan="1" colspan="1"> 0.6031</td> <td valign="top" rowspan="1" colspan="1">0.47, 3.55</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">ALL YRS</td> <td valign="top" rowspan="1" colspan="1">1_10-16</td> <td valign="top" rowspan="1" colspan="1">1.73, 2.93</td> </tr> </tbody> </table> </div> <div class="approach1:causalitystreamlinesfromnon-parametricgrangeranalysis" title="sec"> <div class="title" tagx="title" title="title">
Approach 1: Causality streamlines from non-parametric granger analysis</div> <p>Our analysis of health insurance claims covers nine years of influenza cycles (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
2003</div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
2011</div></div></span>, inclusively), see <a href="#fig2">Figure 2</a>. We visualized weekly, county-level prevalence as a movie (see Supplement); <a href="#fig2">Figure 2A–H</a> show a few relevant weekly snapshots from different years. The plates in <a href="#fig2">Figure 2A–H</a>, and especially the movie, clearly show that seasonal influenza cycles initiate in the South/Southeastern US and sweep the country from south to north. This pattern is repeated, with some variation, each season.</p> <p><a href="#fig3">Figure 3G</a> shows the country-wide propagation dynamics as represented by our computed causality streamlines. The alignment of causality flow vectors into long, continuous streamlines suggests a stable propagation mechanism across the country; the probability of a long sequence of summary movement vectors accidentally matching in the direction by mere chance is vanishingly small (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
16</div> </div> </div> </div></div></span> for longer streamlines).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.015"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Computation of causality field, Approach 1 Plates A and B: Incidence data from neighboring counties in Alabama, US.</div> <p>Plates C and D: Transformation to difference-series, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, change in the number of reported cases between weeks. We imposed a binary quantization, with positive changes mapping to ‘1,’ and negative changes mapping to ‘0.’ From a pair of such symbol streams, we computed the direction-specific coefficients of Granger causality (see Supplement). For each county, we obtained a coefficient for each of its neighbors, which captured the degree of influence flowing outward to its respective neighbors (Plate L). We computed the expected outgoing influence by considering these coefficients as representative of the vector lengths from the centroid of the originating county to centroids of its neighbors. Viewed across the continental US, we then observed the emergence of clearly discernible paths outlining the ‘causality field’ (Plate G). The long streamlines shown are highly significant, with the probability of chance occurrence due to accidental alignment of component stitched vectors less than <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
185</div> </div> </div></div></span>; while each individual relationship has a chance occurrence probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
∼</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
6</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span> (Plates E and F). Plate H: Spatially-averaged travel patterns (see text in Materials and methods) and the sink distribution between expected travel patterns. These patterns (Plate H), along with the inferred causality field (Plate I), match up closely, with sinks showing up largely in the Southern US, explaining the central role played there. In Plate H, the size of the blue circles indicate the percentage of movement streamlines (computed by interpreting the locally averaged movement directions as a vector field) that sink to those locations. In Plate I, the size of the red circles indicate the percentage of causality streamlines that sink to the indicated locations. We note that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
∼</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
75</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span> of the movement streamlines sink in counties belonging to the Southern states, which matches up well with the sinks of the causality streamlines. In Plates J and K show spatial analysis results for two different infections (HIV and <i>E. coli</i>, respectively) and which exhibit very different causality fields, negating the possibility of boundary effects.</p>   </div> <p>Do epidemics originate from the same counties season after season? To answer this question, we follow ‘causality streamlines’ back to their source county. Informally, influenza onset in these source counties has little or no causal dependency on their neighbors. That is, their epidemic states are seemingly caused by factors outside of disease prevalence in other counties. <a href="#fig2">Figure 2K</a> presents the county-specific likelihood of streamline initiation across our nine years of data. To verify the near-shores position of these source counties is not a mere manifestation of a boundary effect of shore counties (no neighbors at the side of large water body), we carried out identical causality analyses with two different infections, specifically choosing diseases less likely to share etiologies with influenza: HIV and <i>Escherichia coli</i>. The results for both HIV and <i>Escherichia coli</i> infections are shown in <a href="#fig3">Figure 3J and K</a>, which exhibit flow patterns very different from those obtained for influenza. These streamlines almost never originate from the coasts, thus reducing the likelihood that the pattern observed for influenza is a geo-spatial boundary effect. Combined with the exceedingly low probability (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
∼</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
185</div> </div> </div> </div></div></span>) of chance inference for the streamlines, this strongly supports our conclusion that the epidemics are of coastal origin.</p> <p>We directly validated our conclusion that influenza waves tend to start in the South by identifying counties which seem to trigger the epidemic. We computed a ‘trigger period’ of five to six weeks for each season, defined as the period immediately preceding an exponential increase in influenza dispersion. To calculate this weekly dispersion, we treated each county as a node in an undirected graph, each with an edge connecting two geographically adjacent counties–only if they had both reported at least one influenza case in the specified week. We defined dispersion as the size of the largest, connected component in this undirected graph. Thus, a trigger period describes the period in which the size of the giant component of the infection graph rises above <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
250</div></div></span> counties from being under <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
100</div></div></span> as shown in <a href="#fig2">Figure 2I</a>, and then proceeds to the seasonal peak. <a href="#fig2">Figure 2J</a> presents the likelihood of a county being part of this largest, connected component during the trigger period. In the second approach, we followed causality streamlines back to their source county. <a href="#fig2">Figure 2K</a> presents the county-specific likelihood of streamline initiation across nine years.</p> <p>These approaches produced qualitatively similar results (<a href="#fig2">Figure 2J and K</a>). While epidemics seem to start in many places around the country (see the origins of streamlines in <a href="#fig3">Figure 3J and I</a>), they successfully gain traction near large bodies of water (as evidenced by the most likely places of epidemic onset, see <a href="#fig2">Figure 2J and K</a>)). Otherwise, they fizzle out before triggering an actual epidemic cycle (see <a href="#fig2">Figure 2J</a>). Seasonal initiation is neither spatially uniform nor simply a reflection of county-specific population density.</p> <p>Our analysis of the Twitter movement matrix indicates that people most frequently travel between neighboring counties, preferentially towards higher-population-density areas, which shows that the maximum-probability movement patterns follow the local gradient of increasing population density (see <a href="#fig4s1">Figure 4—figure supplement 1</a>). In contrast, the geo-spatially-averaged movement vectors for each county reveal global flows in the movement patterns (see <a href="#fig3">Figure 3H</a>, along with Methods for the calculation of spatial averages).</p> <p><a href="#fig3">Figure 3H–I</a> suggest that average movement patterns largely agree with the influenza streamlines: Both patterns, especially in the South/Southeast of the country, are associated with flow pointing away from large bodies of water.</p> <p>In addition to looking at the direction of short-range travel, we used our non-parametric Granger analysis to investigate the comparative strength of short- vs. long-range influenza propagation. In the first case, we considered the neighborhood map shown in <a href="#fig4">Figure 4A</a> (for a detailed definition of �?neighbors,’ see Materials and methods), and the in the second case, we considered associations between major, airport-bearing counties (see <a href="#fig4">Figure 4B</a>). We then plotted the distribution of the maximum pairwise coefficient of causality, where the maximization is carried out by fixing the source and the target and varying the delay in weeks, after which we attempt to predict the target stream.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.016"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
Comparing influence of short- and long-distance travel on infection propagation Plate A shows land connectivity visualized as a graph with edges between neighboring counties.</div> <p>Plate B shows air connectivity as links between airports, with edge thickness proportional to traffic volume. Plate C shows the delay in weeks for the propagation of Granger-causal influence between counties in which major airports are located, and Plate E shows the distribution of the inferred causality coefficient between those same counties. Plates D and F show the delay and the causality coefficient distribution respectively, which we computed by considering spatially neighboring counties. The results show that local connectivity is more important. We reached a similar conclusion using mixed-effect Poisson regression, as shown in Plate G: The inferred coefficients for land connectivity are significantly larger than those for air connectivity, tweet-based connectivity, or exponential diffusion from the top <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
30</div></div></span> largest airports. The coefficients shown in Plate G are exponentiated, allowing us to visualize probability magnitudes (see ‘Model Definition’).</p>   </div> <p>Conclusions associated with Approach 1: The inferred causality streamlines computed from the infection time series in all counties (<a href="#fig3">Figure 3</a>) show that epidemics are mostly triggered near large water bodies and flow inland and away. They also illustrate that the US continental Southern states act as ‘sinks’ to a large proportion of these streamlines. (‘Sinks,’ in our definition here, are geographic areas that multiple streamlines converge <i>towards</i>; sinks are especially obvious when we look at the vector representation of causality direction. The opposite of a ‘sink’ is a ‘source,’ defined as an area at which at least one streamline starts.) This might explain the increased prevalence in the designated region. Additionally, the analysis shows that human travel is a very important driver of emergent epidemiological patterns, and that short-range, land-based travel is more important than air-travel. This result is cross-corroborated by our Poisson regression analysis (described next in Approach 2).</p> <p>Approaches 2 and 3 are motivated by the ‘why’ questions: (1) Why do epidemics initiate where and when they do? and; (2) Why do some disease initiations become epidemics while others do not?</p> </div> <div class="approach2:importanceoffactorsfrompoissonregression" title="sec"> <div class="title" tagx="title" title="title">
Approach 2: Importance of factors from poisson regression</div> <p>We focused on a subset of weeks associated with the initial rise of influenza waves (indicated by the gray bars in <a href="#fig2">Figure 2I</a>, and calculated as discussed earlier). The results from our best-fit model are illustrated in <a href="#fig1">Figure 1A</a>. We selected this particular model out of a total of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
126</div></div></span> compared in the Bayesian analysis, a few of which we list in <a href="#table5">Table 5</a>, ranked by their decreasing goodness-of-fit, measured with the Deviance Information Criterion, DIC (see Supplement). From the values of the inferred coefficients corresponding to the different factors, and taking into account their significance levels and credible intervals, we concluded that the roles played by weather variables, particularly humidity, appear to be substantially more complicated compared to what has been suggested in the literature.</p> <div class="thesurprisinglyunimportantfactors" title="sec"> <div class="title" tagx="title" title="title">
The surprisingly unimportant factors</div> <p>School schedule was not predictive of influenza onset in our analysis: We ended up with a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
p</div></div></span>-value of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.84</div></div></span> and an odds ratio of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.8403</div></div></span>, strongly suggesting that school opening dates are not a significant factor in triggering the seasonal epidemic.</p> <p>We are not claiming here that closing down schools during the seasonal peak, or during an initial phase of a seasonal epidemic, would not have a beneficial effect on maximum incidence. Rather, the observed epidemiological patterns over the time period we analyzed (2003–2011) do not seem to name ‘school opening times’ as a significant predictive factor–at least in the continental US.</p> <p>We factored in the effect of vaccination coverage by estimating the cumulative fraction of the population that received the current influenza vaccine stratified by geo-spatial location and time of inoculation within each influenza season. Our analysis indicated that vaccination coverage is not a significant predictor of influenza onset/triggering period. It could be a reflection of overall vaccine ineffectiveness, or the choice of outcome predicted (i.e. vaccination might effect overall infection numbers over the entire outbreak, but not the timing of the trigger). It could also reflect the fact that different influenza type/subtypes have different virulence–so a vaccine against H3N2 during an H3N2 year, may be more effective, but due to that fact that H3N2 is more virulent, more people still wind up seeking medical care.</p> <p>Vaccination coverage failed to reach predictive significance. The variables corresponding to spatio-temporal indicators of the cumulative fraction of the inoculated population are included in our best model (see the last entry in <a href="#table5">Table 5</a>), but their effect fails to be significant. However, if we drop those variables from the model, the DIC increases. We suggest that the strong dependence between antigenic diversity and vaccination coverage (see <a href="#fig1">Figure 1J–M</a>) is responsible for this effect: Vaccination coverage is important, but its influence is captured by the antigenic variation.</p> </div> <div class="themostimportantfactors" title="sec"> <div class="title" tagx="title" title="title">
The most important factors</div> <p>The strongest predictor groups (ranked by importance) are the population’s socio-demographic properties, weather, antigenic drift of the virus, land-based travel, and auto-correlation of influenza waves.</p> <p><i>Weather</i> As far as weather effects are concerned, epidemics tend to originate in places with high mean, maximum specific humidity, high average temperature, and low average air pressure, namely, in counties at the Southern, and, to a lesser extent, Eastern and Western US coastlines. Additionally, the spread of an epidemic is significantly influenced by a drop in specific humidity up to four weeks before its onset. However, this effect is weaker than the mean maximum specific humidity effect. Drop in average temperature that dips one to three weeks prior to the epidemic onset is also significantly important (this is consistent with earlier experiments [<a href="#bib61">Lowen et al., 2008</a>]), especially when the temperature drop is accompanied by a decrease in specific humidity, average wind speed, and solar flux. However, high levels of solar flux in the week of onset are also important. This complicated set of weather conditions, a signature of the cold air front (<a href="#bib77">Shaman et al., 2010</a>), is validated by our out-of sample predictions to increase the risk of triggering the seasonal epidemic. Total precipitation also plays a positive role.</p> </div> <div class="theweather/humidityparadox" title="sec"> <div class="title" tagx="title" title="title">
The weather/humidity paradox</div> <p>How can colder weather and lower humidity be a predictor of influenza, if influenza epidemic waves tend to start in the South with warmer climates and higher humidity? Our resolution of this seeming controversy is as follows: The stress is on the drop in both humidity and temperature in those areas with high average annual values of these measurements. A blast of colder, lower-humidity weather in these warm-climate areas has two effects: (1) The influenza virus can stay viable in water droplets longer than in hot, sunny weather, and; (2) Humans tend to interact indoors, in more crowded conditions. Both of these factors are favorable for transmission of the virus to the population at large.</p> <p><i>Antigenic variation</i> Antigenic diversity for HA, NA, M1, and M2 are important predictors. While HA, NA, and M1 inhibit the trigger, M2 diversity enhances it. This peculiar difference in the direction of influence might be a manifestation of the roles played by the individual viral proteins in its life-cycle.</p> <p>The first three proteins are directly involved in the viral binding to host cell surface receptors, while M2 activity is needed only during HA biosynthesis. Additionally, proteolysis experiments indicated that M2 proton channel activity helped to protect (H1N1)pdm09 HA from premature conformational changes as it traversed low-pH compartments during transport to the cell surface (<a href="#bib1">Alvarado-Facundo et al., 2015</a>).</p> <p>We found that antigenic diversity is a significant predictor in all four of the viral proteins we considered. Interestingly, while the increasing diversity found in HA, NA, and M1 inhibits the epidemic trigger, the higher diversity in M2 enhances it (see Discussion).</p> <p><i>Travel</i> Land travel intensity one to three weeks before epidemic onset is a strong predictor. Air travel is also predictive, but its strength is an order of magnitude weaker than that of land travel.</p> <p><i>Autocorrelation</i> The increase in an influenza outbreak’s infection weekly rate one and two weeks before an epidemic onset (in the epidemic source county itself) is predictive of epidemic wave origin.</p> <p>We have used substantially richer datasets than those used by earlier studies (<a href="#bib77">Shaman et al., 2010</a>; <a href="#bib79">Tamerius et al., 2013</a>), which lends strong statistical support to our conclusions. It also allowed us to disentangle and make precise the contributions from different factors, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, mean county humidity vs. drops in humidity before an infection. While we found the former effect to be clearly stronger (in accordance to previously reported results [<a href="#bib77">Shaman et al., 2010</a>]), the other diverse set of factors were also found to be significant.</p> </div> <div class="validationofpredictivecapability" title="sec"> <div class="title" tagx="title" title="title">
Validation of predictive capability</div> <p>The robustness of our results is established in a number of ways:</p> <ol> <li> <p>In mixed-effect regression (Approach 2), we compared over <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
120</div></div></span> chosen model variations (see <a href="#table5">Table 5</a> for an abridged list, and <a href="#supp3">Supplementary file 3</a>for the complete enumeration of considered models); the results appear to be qualitatively stable, though the quantitative performance of the models vary somewhat as measured by DIC for different configurations of regression equations.</p> </li> <li> <p>We carried out a direct validation of predictive performance by estimating model parameters using the first six seasons and predicting the epidemic trigger locations using the last three (see <a href="#fig5">Figure 5</a> and Materials and methods). The out-of-sample predictions of influenza incidence are always positively correlated with observed incidence (Plate A). Perhaps more importantly, we obtained good predictability as measured by the area under the curve (AUC <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
≈</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
80</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span>) for the receiver operating characteristics (ROC, See Plates B-D). Plates E-G show that our out-of-sample predictions correctly identified epidemic initiation in the Southern and Southeastern counties of the continental US.</p> </li> <li> <p>As we discuss in the next section, in Approach 3, we conducted a corroborating matched effect analysis on the counties, using combinations of county-specific factors as a ‘treatment,’ not unlike clinical trials in which patients on a drug regimen are matched to patients receiving a placebo (<a href="#bib64">Morgan and Winship, 2015</a>).</p> </li> </ol> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.018"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
Prediction performance with training data from the first six seasons and validation on the last three.</div> <p>Plate A shows the correlation between the observed incidence and the model-predicted response. We show significant positive correlation, particularly within the trigger periods, between the model predictions and the actual held-out data. This gives us confidence to construct ROC curves for each week. Plates B-D show the ROC curves for the last three weeks of each of the three seasons in the out-of-sample period (potentially, these computations can be repeated for all possible partitions of study weeks into training and test samples). Plates E-G illustrate that the normalized decision variable, which is the normalized response from the model, identifies the South and Southeastern counties as the trigger zones.</p>   </div> </div> </div> <div class="approach3:matchingcountiesandfactorcombinations" title="sec"> <div class="title" tagx="title" title="title">
Approach 3: Matching counties and factor combinations</div> <p>In Approach 3, we investigated combinations of factors presented as ‘treatment’ via a non-parametric, exact-matching analysis of US counties during the weeks of epidemic onset on a season-by-season basis.</p> <p>First, we collected the list of all counties with a drop in maximum specific humidity during the weeks leading up to an influenza season in a particular year. This is the ‘treated set’: the set of counties that may be thought of as subjected to the positive ‘treatment’ of a drop in specific humidity. We split this set into two, considering counties that also experience increased influenza prevalence during the epidemic onset, and ones that do not (counties with two different values of the outcome variable). The number of counties in these two sets define the first row of a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> contingency table. In the second row (the ‘control set’), we focused on counties that do not experience a drop in the maximum specific humidity. However, we only considered counties that have a matching counterpart in the treated set in the following sense: For each county in the control set, we found at least one in the treated set such that the rest of the significant variables (other than specific humidity) had similar variation patterns in both counties. Once we defined the control set, we split it in the manner described for the treated set: We counted the number of control counties that experienced an increased influenza prevalence during epidemic onset, and those which did not. This defined the second row of the contingency table. Finally, we used Fisher’s exact test to compute an odds ratio (the odds of realizing these numbers by chance), along with the test-derived significance of the association between the ‘treatment’ and epidemic wave initiation (<i>p</i>-value). Furthermore, we defined our treatment to consist of multiple factors simultaneously, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span> specific humidity and its change in the preceding week, along with average temperature and degree of urbanity, see <a href="#fig6">Figure 6</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.019"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <div class="title" tagx="title" title="title">
Results for our analysis involving county-matching (Approach 3).</div> <p>Plate A illustrates the factor combinations that turn out to be significant over the nine seasons. Notably, for each season, we have multiple, distinct factor sets that turn out to be significant (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.05</div> </div></div></span>) and yield a greater-than-unity odd ratio. Plotting the probability with which different factors are selected when we look at season-specific county matchings (the top panel in Plate A), we see a corroboration of the conclusions drawn in Approach 2. We find that specific humidity and average temperature, along with their variations, are almost always included. We do see some new factors that fail to be significant in the regression analysis, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, degree of urbanity and vaccination coverage. While vaccination coverage is indeed included as a factor in our best performing model, in Approach two it failed to achieve significance, perhaps due to its strong dependence on antigenic variation (see <a href="#fig1">Figure 1J–M</a>). Degree of urbanity is indeed significant for some of the regression models we considered (see Supplementary Information), but was not significant for the model with the smallest DIC. Note that ‘Treatment’ here is defined as a logical combination of weather factors. A treatment is typically a conjunction of several weather variables. For example, the treatment shown in top left panel of Plate B involves a conjunction of: (1) a drop in average temperature during the week of infection; (2) a drop in temperature during the week of infection; (3) a higher-than-average specific humidity; (4) a higher-than-average temperature, and; (5) a high degree of urbanity. With respect to the ‘treatment,’ we can divide counties into three groups: (1) ‘treated counties,’ shown in green; (2) at least one matching county for each of the treated counties (matching counties are very close to the treated counties in all aspects but in treatment, which we called ‘control’ counties), shown in black, and; (3) other counties, shown in grey. The counties in the ‘treatment’ and ‘control’ groups are further subdivided into those counties that initiated an influenza wave and those that have not, resulting in four counts arranged into a two-by-two contingency table. We then used the Fisher exact test to test for association between treatment and influenza onset. Panels in Plate B show both the treated and control sets for the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
9</div></div></span> seasons for a subset of chosen factors. The results are significant, as shown in <a href="#table2">Tables 2</a> and <a href="#table3">3</a>. The variable definitions are given in <a href="#table4">Table 4</a>. Notably, some of the variables found significant in the regression analysis are not included above, and some which are not found to be significant in the best regression model show up here. This is not to imply that they are not predictive or lack causal influence. The matched treatment approach, as described above, is not very effective if we use more than <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
∼</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
15</div> </div> </div></div></span> factors simultaneously to define the treated set (for the amount of data we have); this results in a contingency table populated with zero entries.</p>   </div> <p>Note that geographic clustering of ‘treated’ and ‘untreated’ counties arose automatically as a result of similar weather patterns being imposed via the constraint of multiple climate variables.</p> <div class="table-wrap_UNKNOWN" id="table4" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.30756.020">10.7554/eLife.30756.020</a><span class="label" tagx="label" title="label">Table 4.</span> <div class="title" tagx="title" title="title">
Variables in mixed-effect Poisson regression analysis (Approach 2)</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Variable name</th> <th rowspan="1" colspan="1">physical effect</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">N</td> <td rowspan="1" colspan="1">Total number of patient visits given week and county (the offset)</td> </tr> <tr> <td rowspan="1" colspan="1">max_HUS_mean</td> <td rowspan="1" colspan="1">Mean county-specific maximum specific humidity over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_max_HUS_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of maximum humidity, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">t_avg_mean</td> <td rowspan="1" colspan="1">Mean county-specific temperature over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_t_avg_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of mean temperature, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">RSDS_mean</td> <td rowspan="1" colspan="1">Mean county-specific solar insolation over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_RSDS_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of mean solar insolation, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">AVG_PRESS mean</td> <td rowspan="1" colspan="1">Mean county-specific solar insolation over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_AVG_PRESS_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of mean surface pressure, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">tot_prec_mean</td> <td rowspan="1" colspan="1">Mean county-specific total precipitation over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_tot_prec_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of mean total precipitation, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">Wind_avg_mean</td> <td rowspan="1" colspan="1">Mean county-specific average wind speed over nine years</td> </tr> <tr> <td rowspan="1" colspan="1">d_Wind_avg_i</td> <td rowspan="1" colspan="1">Normalized and zero-centered deviations of average wind speed, i = 0–4 weeks before</td> </tr> <tr> <td rowspan="1" colspan="1">Income</td> <td rowspan="1" colspan="1">County-specific mean income</td> </tr> <tr> <td rowspan="1" colspan="1">airport_diffusion</td> <td rowspan="1" colspan="1">Influence from proximity to airports, modeled as human traffic-weighted exponential diffusion from the 30 largest US airports</td> </tr> <tr> <td rowspan="1" colspan="1">Am_Ind</td> <td rowspan="1" colspan="1">% of American Indians in the county</td> </tr> <tr> <td rowspan="1" colspan="1">Asian</td> <td rowspan="1" colspan="1">% of Asians</td> </tr> <tr> <td rowspan="1" colspan="1">White_Hisp</td> <td rowspan="1" colspan="1">% of Caucasian/Hispanics</td> </tr> <tr> <td rowspan="1" colspan="1">W_non_Hisp</td> <td rowspan="1" colspan="1">% of Caucasian/Non-Hispanics</td> </tr> <tr> <td rowspan="1" colspan="1">Black_Hisp</td> <td rowspan="1" colspan="1">% of Black/Hispanics</td> </tr> <tr> <td rowspan="1" colspan="1">B_non_Hisp</td> <td rowspan="1" colspan="1">% of Black/Non-Hispanics</td> </tr> <tr> <td rowspan="1" colspan="1">Pacific</td> <td rowspan="1" colspan="1">% of Pacific Islanders</td> </tr> <tr> <td rowspan="1" colspan="1">Insured</td> <td rowspan="1" colspan="1">% of county population insured</td> </tr> <tr> <td rowspan="1" colspan="1">Poor</td> <td rowspan="1" colspan="1">% of county population under poverty line</td> </tr> <tr> <td rowspan="1" colspan="1">Urban</td> <td rowspan="1" colspan="1">% of county population classified as urban</td> </tr> <tr> <td rowspan="1" colspan="1">land_i</td> <td rowspan="1" colspan="1">influenza velocity change in the land neighbors of the county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> weeks before the current week</td> </tr> <tr> <td rowspan="1" colspan="1">tweet_i</td> <td rowspan="1" colspan="1">influenza velocity change in the Twitter neighbors of the county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> weeks before the current week</td> </tr> <tr> <td rowspan="1" colspan="1">air_i</td> <td rowspan="1" colspan="1">influenza velocity change in airport neighbors of the county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> weeks before the current week</td> </tr> <tr> <td rowspan="1" colspan="1">v_i</td> <td rowspan="1" colspan="1">change in rate of infection in the county itself <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> weeks from the current</td> </tr> <tr> <td rowspan="1" colspan="1">M1</td> <td rowspan="1" colspan="1">Diversity in M1 protein primary structure</td> </tr> <tr> <td rowspan="1" colspan="1">M2</td> <td rowspan="1" colspan="1">Diversity in M2 protein primary structure</td> </tr> <tr> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">Diversity in NA protein primary structure</td> </tr> <tr> <td rowspan="1" colspan="1">HA</td> <td rowspan="1" colspan="1">Diversity in HA protein primary structure</td> </tr> <tr> <td rowspan="1" colspan="1">Cum_vac_per_N_i</td> <td rowspan="1" colspan="1">vaccination coverage in the county cumulated over past 20 weeks <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> weeks from the current</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>(a) Definition of variables.</p> </div> </div> </div> <p>Unlike the mixed-effect regression approach (Approach 2), this matching analysis is non-parametric, and intended to reveal whether multiple factors are, indeed, simultaneously necessary.</p> <p>The results of Approach 3’s analysis are presented in <a href="#fig6">Figure 6</a> and <a href="#table2">Tables 2</a> and <a href="#table3">3</a>. We found that no single variable was able to consistently yield a statistically significant odds ratio greater than one; multiple factors interacted to shape an epidemic trigger (see <a href="#table3">Table 3</a> for a few examples). With a total of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
47</div></div></span> significant variables in our best mixed-effect model, an exhaustive search for all combinations was not feasible. Instead, we performed a standard evolutionary search, looking for combinations that yielded a significant odds ratio for individual seasons. Additionally, we considered all seasons together (by simply adding the contingency tables, element-wise) in order to increase the test’s statistical power.</p> <p>We isolated ten variables (as shown in <a href="#fig6">Figure 6</a>, Plate B) in this manner which included maximum specific humidity and average temperature along with their variations, the degree of urbanity, antigenic variation, and vaccination coverage.</p> <p>The factors that appeared most often in our analysis are illustrated in Plate A: It appears that maximum specific humidity and average temperature, along with their variations, and the degree of urbanity have the most frequent contribution, followed by antigenic variation and vaccination coverage.</p> <p>We did see some new factors here that failed to be of significance in the regression analysis (Approach 2), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, degree of urbanity and vaccination coverage. While vaccination coverage was included as a factor in our best performing model in Approach 2, it failed to achieve significance, perhaps due to its strong dependence on antigenic variation (see <a href="#fig1">Figure 1J–M</a>). Degree of urbanity was indeed significant for some of the regression models we considered (see Supplementary Information), but failed to be so for the model with the least DIC.</p> <p>Thus, Approach three corroborates and strengthens key claims of Approach 2.</p> <p>The exact set of factors varied somewhat over the seasons; nevertheless, together, they yield significant results when all seasons are considered together. The matching analysis corroborates our results from both the mixed-effect regression and the geographic streamline analyses: The sets of counties initiating the wave are near coasts on the Southern region of the continental US (see Plates A - I in <a href="#fig6">Figure 6</a>).</p> </div> <div class="localtravelvs.long-distancetravelandinfluenza" title="sec"> <div class="title" tagx="title" title="title">
Local travel vs. long-distance travel and influenza</div> <p>Our conclusion that local travel is predominantly responsible for disease wave propagation is supported by several lines of analysis.</p> <p>First, continuous land-movement infection waves are visible in the weekly influenza rate movie; we computed this movie from insurance claim data and made it available with results of this study.</p> <p>Second, because our all-weeks-included dataset was too large for the R MCMCglmm package to handle, we performed mixed-effect Poisson regression calculations using a 50 percent random sample of all the weeks for which data were available. In this computation, the airport-proximity, fixed-effect coefficient turned out to be statistically significantly negative (see Supplement A, as well as the editable output file ‘flu-50-percent-weeks.txt’).</p> <p>Third, the results from our Granger-causality inference showed that:</p> <ol> <li> <p>Local county-to-county movements were much more predictive of influenza wave change than airport movements. In comparing Plates E and F in <a href="#fig4">Figure 4</a>, we see that the local movement causality coefficient (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div></div></span>) is, on average, twice as large as that for long-range movement (<a href="#fig4">Figure 4E</a>). <a href="#fig4">Figure 4E</a> shows that the mean long-range causality coefficient is approximately 0.05, whereas it is just over 0.1 in the local propagation. As the causality coefficient quantifies the amount of predictability (measured as information in bits) communicated about the target data stream per observed bit in the source data stream, it follows that, on average, every ten bits of sequential incidence data from an influencing location tells us one bit about the unfolding incidence dynamics in the target location. Therefore, in case of the long-range movement, informativeness is twice as low, so we need on average 20 bits to infer one bit about the state of infection. These calculations strongly suggest that local movement is predictively stronger with regards to influenza infection propagation.</p> </li> <li> <p>While the most frequent value of the computed time delay in influence propagation between counties with large airports is zero weeks, this distribution is significantly flatter compared to that for local, county-to-county influence propagation.</p> </li> </ol> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>A summary of the complex relationship between the driving factors that contribute to the trigger and subsequent development of a seasonal epidemic can be clarified with a forest fire metaphor.</p> <p>The maturation of a forest fire requires the collusion of multiple factors—namely the presence of flammable media, an initiating spark, and a wind current to help spread the fire. Our conceived mapping of this analogy to influenza infection is as follows:</p> <ol> <li> <p>Flammable Media: The Southern US appears to have an unusually high level of social connectivity; it is at least one order of magnitude higher than that in the north of the country (see GSS survey results (<a href="#bib78">Smith et al., 1972</a>) and <a href="#table1">Table 1</a>). The number of close friends, close friends who are neighbors, and communities of people who all, or mostly, know each other is much higher in the South than in the country at large. Our conjecture is that a manifestation of this high-connectivity is the highest <i>apparent</i> percentage of people infected with influenza (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
20</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> as opposed to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> in other parts of the country).</p> </li> <li> <p>Initiation Spark: An initial spark for the infection wave is generated by a combination of weather and demographic factors. Specifically, warm, humid places are conducive to influenza wave initiation - particularly in weeks where specific humidity drops. Airport proximity is important, as well as demographic and economic makeup and also the degree of urbanization. Note that the first static condition (warm humid places) is highly correlated with areas in the South with greater social connectivity. It is possible that static meteorological variables (warm mean temperature and high mean humidity) serve as proxies for high social connectivity or other correlated socioeconomic factors.</p> </li> <li> <p>Wind: The ‘wind’ in this analogy is the collective movement of a large number of people, integrated over time, revealing persistent ‘currents.’ These currents reproducibly point from coastlines and move inwards towards the center of the continent, making them perfect vehicles to transmit the infection inland from the shores.</p> </li> </ol> <p>Each of our three types of computational approaches has their strengths and weaknesses: (1) The Poisson mixed-effect regression allows for the direct comparison of the predictive strength of numerous predictor variables and accounts for spatial and temporal autocorrelation, but relies on strong modeling assumptions; (2) The non-parametric Granger analysis is not limited by restrictive modeling assumptions in our implementation, though it focuses only on trends of infection propagation between counties, and; (3) The county-matching analysis is also model-free, but this freedom comes at the expense of lesser statistical power.</p> <div class="whatisnewinthisstudy?" title="sec"> <div class="title" tagx="title" title="title">
What is new in this study?</div> <p>The following aspects make our study of influenza triggers new in the influenza literature: (1) Instead of simulating the plausibility of one particular epidemic trigger model with a dynamic disease transmission model, we used formal model selection tools to compare the goodness of fit of hundreds of plausible models; (2) We explicitly attempted to systematically cross-compare the importance of numerous individual factors typically hypothesized to contribute to epidemic onset; (3) To accomplish this, we collected an unprecedented volume of temporal and spatial data on disease dynamics and the dynamics of putative predisposing factors; (4) We used several orthogonal, computational causality-inference techniques (one of which was developed specifically for this study) to probe associations between disease onset and putative epidemic triggers; (5) We tested our best models for their predictive potential and demonstrated that they are, indeed, suitable for forecasting disease waves, and; (6) We combined, for the first time, numerous candidate factors in a single, integrative study.</p> <p>Convergent conclusions, culled from these radically different techniques, strengthen our claims and make it statistically unlikely that we are observing analysis artifacts. First, the Granger causality analysis results (Approach 1) provide insights into the details of influenza’s epidemiological dynamics. <a href="#fig3">Figure 3G</a> traces out the paths most likely followed by the infection, on average, across the continental US. We note that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
∼</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
75</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span> of the streamlines sink in counties belonging to the Southern states, which matches up well with the streamline-encoded dynamics of weekly disease incidence over nine years (see <a href="#fig3">Figure 3I</a>). What drives this particular causality field’s geometry? While we cannot definitively answer this question, a comparison of the global patterns emerges from the local mobility data culled from the aforementioned Twitter database and offers a tentative explanation (see <a href="#fig3">Figure 3H</a>). Second, contrary to reported human travel pattern influence on seasonal epidemics (<a href="#bib85">Viboud et al., 2006</a>) (but consistent with [<a href="#bib45">Gog et al., 2014</a>]), we find that short-distance travel contributes more significantly to disease spread (see <a href="#fig4">Figure 4</a>). In particular, we find that long-range air travel is important as an epidemic <i>trigger</i>, but once infection waves are triggered, air travel patterns (or proximity to major airports) become less important. Short-range mobility, on the other hand, is apparently important for sustaining infection transmission over each season. Thus, we find short-range travel to be more important for defining the emergent spatio-temporal geometry of infection waves, while proximity to airports is more important for actually triggering an influenza season; the latter loses positive influence once an infection is under way. This conclusion is justified as follows: (1) When we performed regression calculations using all weeks for which data were available (as opposed to wave initiation weeks only) the airport proximity predictor coefficient turned out to be statistically significantly negative (see Supplement). (2) Results from our Granger-causal inference indicate that, on average, the local, putatively causal connections are far stronger compared to the putatively causal connections between counties within which the major airports are located (see <a href="#fig4">Figure 4C and F</a>). Additionally, from our best mixed-effect regression model (<a href="#fig1">Figure 1A</a>), we find that land connectivity effects are significantly stronger than air connectivity effects. The predictive value of Twitter connectivity, which intuitively captures both local and long-distance travel, lies in-between land and air connectivity coefficients. Note that Twitter connectivity is represented as a directed graph, where for each pair of counties, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> edge weight represents the conditional probability of ending at county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, given that a traveler/Twitter user started her journey in county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>. Transition probabilities from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> sum to one over all <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>. Therefore, intuitively, the Twitter connectivity graph should have the features of both a land-connectivity and an air travel graph; which indeed appears to be close to reality. 3) While airport diffusion is a significant factor in our best Poisson model (using data from the initiation period), the causal streamlines (constructed with the complete, all-year incidence data) do not seem to originate from airport-bearing counties.</p> <p>The role of short-distance travel is particularly crucial in explaining influenza’s time-averaged, geo-spatial prevalence. While the mixed-effect regression analysis explains seasonal initiation in the vicinity of the continental US Southern shores, it might not, by itself, adequately explain its average prevalence patterns across the country.</p> <p>Also not explained solely by our regression models is the occurrence of relatively high infection prevalence in the central parts of the country. These differences cannot be attributed to long-distance air travel, as discussed before. However, the routes taken by the causality streamlines (as computed by the non-parametric Granger analysis), interpreted as paths followed by an infection on average, suggest an explanation: The close match between the Granger-causal flow and the short-range mobility patterns (derived from Twitter analysis) strongly suggest that average disease prevalence is modulated by short-range mobility.</p> </div> <div class="rationaleforobservationalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Rationale for observational analysis</div> <p>The traditional empirical approach of testing a causal link between a factor and an outcome of an experiment was to vary one factor at a time, while keeping the other factors (experimental conditions) constant. This ‘all the rest of the conditions are equal’ assumption is often referred to by its Latin form as <i>ceteris paribus</i>. R.A. Fisher ([<a href="#bib40">Fisher, 1935</a>], p. 18) noted that, in real-life experiments, perfect <i>ceteris paribus</i> is not achievable ‘because uncontrollable causes which may influence the results are always … innumerable.’ Fisher’s proposed solution to this problem is to design experiments to involve random assignment of treatment (the putative causal factorâ€™s states) to individual trials and then use regression analysis to estimate the value and significance of the putative causal effect.</p> <p>Likewise, hypothesis-driven science, wherein investigators formulate a single, testable hypothesis and design specific experiments to test it, is a core element of the scientific method, and works well in most scientific fields. However, a new challenge emerges in data-rich scientific fields, such as genomics, epidemiology, economics, climate modeling, and astronomy: How do we choose the most promising hypotheses among millions of eligible candidates that potentially fit data? One solution to this challenge is the many-hypotheses approach, a method of automated hypothesis generation in which many hypotheses are systematically produced and simultaneously tested against all available data. This approach is currently used, for example, in whole-genome association or genetic linkage studies, and often enables truly unexpected discoveries. In contrast to the single-hypothesis approach, the many-hypotheses approach explicitly accounts for the large universe of possible hypotheses through calibrated statistical tests, effectively reducing the likelihood of accidentally accepting a mediocre hypothesis as a proxy for the truth (<a href="#bib66">Nuzzo, 2014</a>).</p> <p>The many-hypotheses approach provides a complement to carefully controlled and highly focused wet laboratory experiments. Running controlled experiments to test a single hypothesis necessarily ignores many of the complexities of a real-world phenomenon; these complexities are necessarily present in large, longitudinal datasets. Of course, the data-driven ‘many-hypotheses’ approach is only one aspect of the broader scientific process progressing toward the development of verifiable general theories.</p> </div> <div class="agreementanddisagreementbetweenmethods" title="sec"> <div class="title" tagx="title" title="title">
Agreement and disagreement between methods</div> <p>Intuitively, we expected that all three approaches would produce similar, if not identical results. In practice, while the three approaches agreed in most cases, this agreement was not perfect. For example, the highlighted areas (greater incidence) in the first influenza season snapshot for <a href="#fig2">Figure 2A–H</a> each should match relatively well to the maps in <a href="#fig6">Figure 6B</a> (or at least some unspecified subset of ‘high incidence treatment counties’). While the county-matching results point to initiation at coasts, in <a href="#fig2">Figure 2</a>, 2006-2007 initiation seems to spread from the West Coast and, in , 2010 has a scattered pattern across the middle of the US.</p> <p>The intuitive explanation of perceived discrepancy is that the matching method agrees with other analysis types predominantly, but not in all cases. Each analysis has limitations. In the case of the matching analysis, we have less statistical power than in, for example, Poisson regression; matching by numerous parameters reduces the initial set of thousands of counties to a handful of matching ‘treated’ counties (which meet a particular combination of weather and sociodemographic conditions) and ‘untreated’ counties (very similar to ‘treated’ ones in all respects but treatment). The difficult-to-match, ‘weeded out’ counties may happen to be in the coastal areas indicated as the most likely places of influenza wave origin by other analyses.</p> <p>In the case of the 2007 and 2010 results, the matching analyses pick patterns that are different from those produced by the causality streamline analysis and mixed-effect Poisson regression models.</p> <p><a href="#fig6">Figure 6</a>’s Plate B shows the distribution of the treatment counties and matched-non-treatment counties. Note that here, we are not directly predicting initiation, so while the patterns in <a href="#fig2">Figure 2</a> and <a href="#fig6">Figure 6</a> should indeed show some similarity, they are not required to match up perfectly. The most similar treated counties do indeed show up in the Southern shores.</p> </div> <div class="generalizability" title="sec"> <div class="title" tagx="title" title="title">
Generalizability</div> <p>Our analysis uses no prior knowledge specific to influenza epidemiology. As such, these methods are not limited by either the pathogen under consideration (influenza), or the geospatial context (United States). The tools developed here are expected to be equally applicable to analyzing general epidemiological dynamics for pathogens other than influenza, unfolding in arbitrary geographical regions. The specific conclusions we draw about the initiation and propagation of the seasonal influenza in US might not hold true for influenza epidemiology in a different geographical context. However, the analysis tools are still applicable. More broadly, our tools delineate a general approach to modeling complex spatio-temporal dynamics, with applications beyond solely disease epidemiology.</p> <p>We conclude by highlighting the structure of overlapping conclusions delivered by our three approaches. Approach 1: Granger-causality analysis suggests that an epidemic tends to begin in the South, near water bodies and that short-range, land-based travel is more influential compared to air-travel for infection propagation, providing a map of mean infection flow across the continental US. Approach 2: Poisson regression identifies significant predictive factors, ranks these factors by importance, suggests that Southern shores are where the epidemic begins, and corroborates Approach 1’s result on short-range vs. long-range travel. Approach 3 (county-matching): This approach drills down further to the epidemic onset source to the Southeastern shores of continental US, and identifies a smaller validated subset of predictive factors.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <p>The methods and putative predictors identified in our study should be directly applicable to data outside the US. While the balance (relative importance) of putative triggers of influenza waves may vary, the factors themselves should be universal globally, because the virus and host biology are universal.</p> <div class="candidatefactorsininfluenzainitiation" title="sec"> <div class="title" tagx="title" title="title">
Candidate factors in influenza initiation</div> <p>To investigate county-specific variability, we grouped candidate factors into several categories: demographic, relation to human movement, infection state of county neighbors, and county’s own recent state, and climatic.</p> <p><i>Major hypotheses</i> regarding putative causal factors affecting infection dynamics can be traced to a handful of earlier publications:</p> <ul> <li> <p>Short- and long-range, work-related human movement (<a href="#bib85">Viboud et al., 2006</a>), including air travel (<a href="#bib27">Colizza et al., 2006</a>; <a href="#bib8">Balcan et al., 2009</a>; <a href="#bib85">Viboud et al., 2006</a>)</p> </li> <li> <p>Demographic confounders (<a href="#bib45">Gog et al., 2014</a>; <a href="#bib19">Charu et al., 2017</a>)</p> </li> <li> <p>Social contact among children in schools, or the ‘Return-to-school effect’ (<a href="#bib45">Gog et al., 2014</a>)</p> </li> <li> <p>Absolute humidity (<a href="#bib76">Shaman and Kohn, 2009</a>)</p> </li> <li> <p>Other climate variables (<a href="#bib24">Chowell et al., 2012</a>; <a href="#bib80">te Beest et al., 2013</a>)</p> </li> <li> <p>Host immunity, as affected by vaccination coverage, previous infections, and antigenic variation (<a href="#bib18">Centers for Disease Control and Prevention et al., 2009</a>).</p> </li> </ul> <div class="humanmovement" title="sec"> <div class="title" tagx="title" title="title">
Human movement</div> <p>We considered two measurements of human movement: (1) The first measurement reflects the proximity of counties to major airports. We computed an exponentially-diffusing influence from counties with major US airports, weighted by passenger enplanements at their respective locations. This accounted for people moving to and from both major airports and neighboring counties, and; (2) The second measurement is a large-scale movement matrix representing people’s week-to-week travels between counties. These data were culled from a complete collection of geo-located Twitter messages, captured over <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
3.5</div></div></span> years, and constituted a large-scale, longitudinal sample of individual movements. We used only automatically geo-tagged tweets.</p> </div> <div class="demographic" title="sec"> <div class="title" tagx="title" title="title">
Demographic</div> <p>Influenza is transmitted through direct contact with infected individuals, via contaminated objects, and virus-laden aerosols. Thus, human population density (how many people happen to be around) and social connectivity (how many people interact with each other and how frequently [<a href="#bib11">Bedford et al., 2015</a>]) are factors expected to affect local virus incubation and spread. In addition to population density, we considered socioeconomic factors such as county-mean household income, levels of poverty and urbanity, as well as the prevalence of ethnic and age groups. All these socio-demographic and socio-economic data were derived from reports provided by the US Census ([Web Page]; 2017. Available from: <a href="https://www.census.gov/geo/reference/ua/uafacts.html">https://www.census.gov/geo/reference/ua/uafacts.html</a>).</p> </div> <div class="return-to-schooleffect" title="sec"> <div class="title" tagx="title" title="title">
Return-to-school effect</div> <p>Social contact among children in schools has been extensively investigated as a determinant of the peak incidence rate. This is one of the few factors that might lend itself to intervention relatively easily, and hence the interest is well-justified. While any reduction in social contact should, in theory, directly impact transmission, quantifying the effect of this specific mode of contact on the incidence rate has been difficult to calculate. Predictions of the reduction in the peak incidence associated with reduced social contact were typically 20–60% (<a href="#bib39">Ferguson et al., 2006</a>; <a href="#bib47">Haber et al., 2007</a>), with some studies predicting much larger reductions of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
≧</div></div></span>90% (<a href="#bib63">Mniszewski et al., 2008</a>; <a href="#bib42">Ghosh and Heffernan, 2010</a>). Reductions in the cumulative attack rate (AR, ratio of the number of new cases to the size of the population at risk) were usually smaller than those in the peak incidence. Several studies predicted small (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>10%) or no reduction in the cumulative AR (<a href="#bib39">Ferguson et al., 2006</a>; <a href="#bib47">Haber et al., 2007</a>; <a href="#bib91">Yasuda et al., 2005</a>; <a href="#bib26">Ciofi degli Atti et al., 2008</a>; <a href="#bib90">Yasuda et al., 2008</a>; <a href="#bib56">Kelso et al., 2009</a>; <a href="#bib31">Davey et al., 2008</a>; <a href="#bib73">Rizzo et al., 2008</a>; <a href="#bib87">Vynnycky and Edmunds, 2008</a>; <a href="#bib43">Glass and Barnes, 2007</a>; <a href="#bib59">Lee et al., 2010</a>; <a href="#bib89">Yang et al., 2011</a>; <a href="#bib92">Zhang et al., 2012</a>), whilst a few predicted substantial reductions (e.g. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
≧</div></div></span>90%) (<a href="#bib44">Glass et al., 2006</a>; <a href="#bib31">Davey et al., 2008</a>; <a href="#bib37">Elveback et al., 1976</a>; <a href="#bib32">Davey and Glass, 2008</a>). Only two studies (<a href="#bib44">Glass et al., 2006</a>; <a href="#bib59">Lee et al., 2010</a>) predicted that peak incidence might increase markedly under certain circumstances following school closures, for example by 27% if school closures caused a doubling in the number of contacts in the household and community, or by 13% if school systems were closed for two weeks at a prevalence of 1% in the general population. Studies have also investigated the effect of such interventions on children vs. adults; one study predicted an overall reduction in the cumulative AR, but an increase of up to 48% in the cumulative AR for adults in some situations (<a href="#bib4">Araz et al., 2012</a>).</p> <p>While this diverse set of predictions in the literature often pertains to the effect of school closures as an intervention tool, we are more interested in the influence that the current school schedule has, if any, on triggering an epidemic. To answer this specific question, we formulated a simple statistical test to determine whether the timing of return-to-school after summer and winter holidays significantly predicts influenza season initiation. We found insufficient evidence in support of this effect (see Methods and materials).</p> </div> <div class="climatevariables" title="sec"> <div class="title" tagx="title" title="title">
Climate variables</div> <p>Specific humidity and a drop in temperature have been suggested as the key drivers in triggering seasonal influenza epidemics (<a href="#bib61">Lowen et al., 2008</a>; <a href="#bib77">Shaman et al., 2010</a>). These initial conclusions were drawn from experiments conducted using an animal model (guinea pig), under controlled laboratory conditions (<a href="#bib61">Lowen et al., 2008</a>), followed by indirect support from epidemiological modeling (<a href="#bib77">Shaman et al., 2010</a>).</p> </div> <div class="vaccinationcoverage" title="sec"> <div class="title" tagx="title" title="title">
Vaccination coverage</div> <p>Vaccination is widely regarded as our most promising tool to combat influenza, though antigenic variation between seasons makes it difficult to craft an effective vaccination strategy (<a href="#bib14">Boni, 2008</a>). Understanding how the virus will evolve in the short-term is key to finding the correct antigenic match for an upcoming influenza season. Additionally, short-term molecular evolution might rapidly give rise to immune-escape variants that, if detected, might dictate intra-season updates in vaccine composition. More importantly, vaccination itself might exert significant selection pressure to influence antigenic drift. The effect of vaccination on viral evolution has been documented in an avian H5N2 lineage in vaccinated chickens (<a href="#bib60">Lee et al., 2004</a>), suggesting that similar processes might be occurring in human counterparts. The diversity of the surface proteins at any point in time between seasons suggests that our current vaccination strategies are limited to confer partial immunity, which can result in a highly immune or vaccinated population selectively pressurizing the viral population to evolve more quickly than usual. Given that influenza moves quickly across geographies and that there are multiple, co-circulating strains that may confer partial cross-protection, changing viral service proteins represent a ‘moving target’ for the human immune system. There are additional complexities due to early-life immune imprinting in humans.</p> </div> <div class="accountingfornon-vaccinationhostimmunity" title="sec"> <div class="title" tagx="title" title="title">
Accounting for non-vaccination host immunity</div> <p>Resistance to influenza infection in human hosts arises via two related mechanisms: immunological memory from a previous infection by an identical or sufficiently similar strain, or vaccination against the current strain (or a sufficiently similar strain). In our analysis, we explicitly account for the degree of vaccination coverage. Accounting for host immunity is more difficult, because resistance to infection is not directly observable; however, our analysis also uses the antigenic variation of influenza virus as a proxy to host resistance: If the relative antigenic variation is large, then the susceptibility of non-vaccinated hosts increases, while low antigenic variation increases the probability of encountering a resistant host. In addition, we use the absolute antigenic deviation of the later-season virus from the first-season virus in our data set (winter of 2003–2004) as a predictor, in order to capture longer-term effects of immunological memory. Thus, if the magnitude of the relative drift between two succeeding years is small, we have a likely decrease in susceptibility. Likewise, if the absolute deviation from a few years back starts decreasing, we would also register a decrease in susceptibility. Incorporating these factors in the diverse set of models that we investigate, guarantees that we are indeed considering host susceptibility contributions from a wide range of possible mechanisms.</p> </div> <div class="antigenicvariation" title="sec"> <div class="title" tagx="title" title="title">
Antigenic Variation</div> <p>The influenza virus counteracts host immunity via subtle genomic changes over time. The more gradual process, known as antigenic drift, is a manifestation of a gradual accumulation of mutations within viral surface proteins recognized by human antibodies, such as hemagglutinin (HA) and neuraminidase (NA). These mutations are typically introduced during cycles of viral replication (<a href="#bib12">Boni et al., 2004</a>). Most of these mutations are neutral, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span> they do not affect the functional conformation of the viral proteins. However, some of these alterations do, in fact, sufficiently change secondary and tertiary protein structures to have a negative impact on the binding of host antibodies raised in response to previously circulating strains (<a href="#bib88">Webby and Webster, 2001</a>). (Many such mutations also reduce the virus’s viability.) Thus, while a single infection episode is potentially enough to provide long-term host immunity to the invading strain, antigenic variation due to intense selection pressure gives rise to novel viral strains, making re-infections possible within the span of a few years (<a href="#bib3">Andreasen, 2003</a>). This kind of perpetual Red-Queen arms race injects influenza dynamics with auto-correlative dependencies over multiple seasons. It has been suggested that substantial antigenic drift might be associated with more severe, early-onset influenza epidemics, resulting in increased mortality (<a href="#bib83">Treanor, 2004</a>). In contrast to antigenic drift, <i>antigenic shift</i> is an abrupt, major change in virus structure due to gene segment re-assortment that occurs during simultaneous infection of a single host by multiple influenza subtypes (<a href="#bib33">De Clercq, 2006</a>). Antigenic shift results in new versions of viral surface proteins. Antigenic shift due to re-assortment gives rise to novel influenza subtypes that, if capable of sustained human-to-human transmission, can have devastating consequences for human populations, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span> the 2009 H1N1 pandemic (<a href="#bib65">Neumann et al., 2009</a>).</p> <p>In our analysis, we factor in the potential effect of antigenic variation by estimating the surface protein’s population diversity – hemagglutinin (HA), neuraminidase (NA), matrix protein 1 (M1), and matrix protien M2 – as a function of time and geographical sample collection location. Our rationale for our focus on these proteins is that HA, NA, and to some degree M1, are all present on the viral surface (<a href="#bib58">Lamb et al., 1985</a>), contribute to viral assembly, and mediate the release of membrane-enveloped particles (<a href="#bib23">Chlanda et al., 2015</a>). M2 has been shown to have enhanced the pandemic 2009 influenza A virus [(H1N1)pdm09] (<a href="#bib41">Friedman et al., 2017</a>) HA-pseudovirus infectivity (<a href="#bib1">Alvarado-Facundo et al., 2015</a>).</p> </div> </div> <div class="datasources" title="sec"> <div class="title" tagx="title" title="title">
Data sources</div> <div class="socialconnectivity" title="sec"> <div class="title" tagx="title" title="title">
Social connectivity</div> <p>Data was obtained from the General Social Surveys, NORC, See <a href="#table1">Table 1</a>, [].</p> </div> <div class="clinicaldatasource" title="sec"> <div class="title" tagx="title" title="title">
Clinical data source</div> <p>The source of the clinical incidence data used in this study is the Truven Health Analytics MarketScan<sup>®</sup> Commercial Claims and Encounters Database for the years 2003 to 2012 (<a href="#bib48">Hansen, 2017</a>). The database consists of approved commercial health insurance claims for between 17.5 and 45.2 million people annually, with linkage across years, for a total of approximately 150 million individuals. This US national database contains data contributed by over 150 insurance carriers and large, self-insuring companies. We scanned 4.6 billion inpatient and outpatient service claims and identified almost six billion diagnosis codes. After un-duplication, we identified approximately <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
12.8</div></div></span> unique diagnostic codes per individual. We processed the Truven database to obtain the reported weekly number of influenza cases over a period of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
471</div></div></span> weeks spanning from January <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
2003</div></div></span> to December <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
2013</div></div></span>, at the spatial resolution of US counties. To define influenza in insurance claims, we used the following set of ICD9 codes: 487.8, 488.12, 488.1, 488.0, 488.01, 488.02, 487.0, 487.1, 488.19, 488.09, 488, 487, and 488.11. We also considered including non-specific codes representing <i>unspecified viral infection</i> (079.99), as suggested in (<a href="#bib86">Viboud et al., 2014</a>), and decided against it. (We provide explicit annotation for each code, as well as rationale for choosing narrower ICD-code definition of ILIs, in the Supplement Section S-D).</p> <p>A peak percentage of ILI-affected people of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
20</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> may seem high–especially given that it is a much greater percentage than the change in seropositivity over most influenza seasons. Note that our <i>apparent</i> estimates of influenza infection rate are unavoidably inflated, because what we are measuring is the week- and county-specific prevalence of influenza-like illnesses <i>among insured patients who contacted their physician for any reason during the week in question</i>. In other words, the denominator that we used for computing prevalence was almost certain to be much smaller than the overall population of the corresponding county.</p> </div> <div class="dataonantigenicdriftforinfluenzaa" title="sec"> <div class="title" tagx="title" title="title">
Data on antigenic drift for influenza A</div> <p>Sequence data for this computation was obtained from the National Institute of Allergy and Infectious Diseases (NIAID) Influenza Research Database (IRD) (<a href="#bib93">Zhang et al., 2017</a>) through their web site at <a href="http://www.fludb.org.">http://www.fludb.org.</a></p> </div> <div class="dataonvaccinationcoverage" title="sec"> <div class="title" tagx="title" title="title">
Data on vaccination coverage</div> <p>Data on vaccinations was extracted from our EHR database corresponding to the procedural codes 90661 and Q2037, which corresponds to the dominant influenza vaccines. (<a href="http://flu.seqirus.com/files/billing_and_coding_guide.pdf">http://flu.seqirus.com/files/billing_and_coding_guide.pdf</a>)</p> </div> <div class="dataonhumanairtravel" title="sec"> <div class="title" tagx="title" title="title">
Data on human air travel</div> <p>We used a complete, directed graph of passenger air travel for 2010, accounting for the number of passengers transported in each direction (<a href="#bib81">The United States Bureau of Transportation Statistics, 2010</a>). For each county, we computed an air neighborhood network: For counties <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, the incoming edge to county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> represents the proportion of passengers, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span> represents the ratio of all passengers who traveled from county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> to county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> by plane,mto the total number of travelers who left county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> by plane during the year, so that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
≠</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>.</p> </div> <div class="dataongeneralhumantravelpatternsintheus" title="sec"> <div class="title" tagx="title" title="title">
Data on general human travel patterns in the US</div> <p>Using the complete Twitter dataset, we aggregated a movement matrix to capture people’s week-to-week travels between counties from geo-located Twitter messages (captured during the period of January 1, 2011 through June 30, 2014). This dataset includes approximately <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.7</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mn_UNKNOWN">
9</div> </div> </div></div></span> messages and represents <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mn_UNKNOWN">
8</div> </div> </div></div></span> user-days of location information. A small, but significant, percentage of Twitter messages are automatically tagged with the author’s current latitude/longitude information, as determined by their mobile device. Each latitude/longitude-annotated tweet was mapped to a FIPS county code based on Tiger/Line shape files from the 2013 Census dataset (<a href="http://www.census.gov/geo/maps-data/data/tiger-line.html">http://www.census.gov/geo/maps-data/data/tiger-line.html</a>). In addition, we calculated a variant of our movement matrix to capture seasonality and other temporal dynamics: A set of 52 movement matrices captured weekly, county-to-county movements, observed in each week of the year, aggregating each week based on the corresponding observations, from each year from 2011 through 2014.</p> <p>This dataset constitutes a large-scale, longitudinal sample of individual movements. We found that the movement patterns are consistent with intuitive patterns and prior studies of large-scale movements; the majority of people in a county remain in the same county week-over-week. Most travel between counties occurs between neighboring counties, and between counties and large metropolitan areas, conditioned on distance and size of the metropolitan area. We represent our movement data as an <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
n</div> </div></div></span> matrix <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> that captures the likelihood that a person observed in county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> during the course of a week will be observed in county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> in the following week. We calculate the entries <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> of matrix <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> </div> </div> </div></div></div> <p /> <p>For each time interval, we computed the mean movement vector from a given county by summing all county-to-county movement vectors, weighted by the proportion of people moving in each direction.</p> <p>To investigate the role of proximity to major airports, we modeled influence diffusion as follows: Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> be the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> county, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> be the total contribution obtained by diffusing influences from the major airport bearing counties. Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
⋆</div> </div></div></span> be the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> airport-bearing county, and let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
N</div></div></span> be the total number of major airport locations considered, in our case, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
27</div> </div></div></span>. Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mi_UNKNOWN">
k</div> </div></div></span> be the volume of traffic for the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> major airport location. We then compute <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
Θ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
⋆</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Θ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
⋅</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
⋅</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the distance in miles between two locations, we computed using the Haversine approximation. The value of the constant <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> was chosen to be <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.1</div></div></span>. Small variations in the constant do not significantly alter our conclusions. As noted above, proximity to airports had a significant positive influence in sparking seasonal epidemics; the influence is significantly weaker once an outbreak is well under way.<p /> </div> <div class="estimatingantigenicdiversity" title="sec"> <div class="title" tagx="title" title="title">
Estimating antigenic diversity</div> <p>In this study, we measured antigenic diversity as follows: Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span> be the set of amino acid sequences for the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> protein (one of HA, NA, M1, or M2), collected in year <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> ranging between 2003 and 2011), in state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
x</div></div></span> of the continental US. The temporal resolution of the sequence data is thus set to years instead of weeks, and the spatial resolution to states instead of counties. These resolutions are coarse compared to our EHR data on infection incidences, and is set in this manner to maintain sufficient statistical power. For each such set of amino acid sequences <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span>, we compute the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> of pairwise edit distances: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ℒ</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
≠</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ℒ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the standard edit distance (also known as the Levenshtein distance) between the sequences <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span>. Mathematically, the Levenshtein distance between two strings <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> (of length <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
|</div> </div> </div></div></span> respectively) is given by: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ℒ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
where </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
if</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mtd_UNKNOWN" /> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mtd_UNKNOWN" /> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
≠</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mtd_UNKNOWN" /> </div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mtext_UNKNOWN">
otherwise.</div> </div> </div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
≠</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the indicator function equal to 0 when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> and equal to one otherwise, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
lev</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the distance between the first <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> characters of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
a</div></div></span> and the first <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> characters of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
b</div></div></span>.<p /> <p>The antigenic diversity of the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> protein at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> in state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
x</div></div></span> was then defined as the median of the distribution of the values in the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>. Clearly, as the sequences became more diverse at a point in time and space due to molecular variations brought about by either drifts or shifts, the measure deviated more from zero. Use of the median provided robustness to outliers.</p> <p>Importantly, here we did not compare temporal changes in viral antigenic makeup directly, but estimated the time-specific diversity in the viral protein primary structure. We expected our diversity measure to be representative of the cumulative changes that occurred within each of the nine influenza seasons.</p> </div> <div class="estimatingvaccinationcoverage" title="sec"> <div class="title" tagx="title" title="title">
Estimating vaccination coverage</div> <p>We incorporated the effect of vaccination coverage by estimating the cumulative fraction of the population that received the current influenza vaccine within the previous 20 week period. This approach to estimating vaccination coverage does not correct for season-specific antigenic match, or the lack thereof. Nevertheless, because we explicitly included measures of antigenic diversity in addition to vaccination coverage, we expected that effects arising from the degree of antigenic match would indeed be factored in; if there is a significant mismatch, we expected the antigenic diversity in that year to be less and vice verse. This assumes implicitly that vaccination does indeed play a major role in exerting significant selection pressure, the assumption that was reinforced by our observation of a strong dependency between vaccination coverage and normalized antigenic diversity.</p> <p>We found that antigenic diversity is quite strongly affected by vaccination coverage. This reflects the theoretical predictions in Boni et al. (<a href="#bib13">Boni et al., 2006</a>), where it is shown that the amount of observed antigenic drift increases as immunity in the host population increases and pressures the virus population to evolve.</p> </div> <div class="estimatingtheeffectofreturn-to-schooldays" title="sec"> <div class="title" tagx="title" title="title">
Estimating the effect of return-to-school days</div> <p>The absence of consensus, and the diversity of modeling assumptions pertaining to this effect (described above) makes it difficult to validate conclusions in large scale epidemiological data. We carried out a simple test to determine whether there is statistical evidence that after-holiday, return-to-school periods in August-September and in January predict or trigger the seasonal epidemic.</p> <p>For this test, we assumed a broad window to cover all such school openings across the continental US including the last week in August, the entire month of September, two weeks in October, and two weeks in January. We next carried out a Fisher’s exact test to determine whether the overlap between these weeks and the identified ‘trigger period’ (See Table LABEL:SI-tabschool) for the seasonal epidemic are sufficiently non-random.</p> </div> <div class="weatherdata" title="sec"> <div class="title" tagx="title" title="title">
Weather data</div> <p>The dataset starts with the week beginning December 31 st, 2002 and includes 522 weeks (which ends exactly on the week ending December 31 st, 2012). Temperature and precipitation data come from the 2.5 arcminute (approximately 4 km) PRISM (<a href="#bib67">Oregon State University, 2014</a>) dataset and other variables (wind speed, specific humidity, surface pressure, downward incident, and shortwave solar radiation) come from the 7.5 arcminute (approximately 12 km) Phase 2 North American Land Data Assimilation System (NLDAS-2) dataset (<a href="#bib62">Mitchell, 2004</a>; <a href="#bib28">Cosgrove et al., 2003</a>). These datasets were selected in large part due to the fact that both are updated in near real-time, making it possible to use these datasets for future monitoring applications. PRISM is released daily, with an approximately 13 hr delay (data for the previous day is released at 1pm EST each day) while NLDAS is released daily, with an approximately 2.5 day delay).</p> <p>Variables were aggregated to county boundaries based on shapefiles from the GADM database of Global Administrative Areas (<a href="#bib5">Areas, 2014</a>). Where appropriate, we considered both the average daily climate variable (for example, the daily maximum temperature averaged over the week) as well as the the maximum and/or minimum of the variable experienced over the week. For precipitation, we considered only the cumulative total precipitation experienced during the week.</p> </div> </div> <div class="threeapproaches" title="sec"> <div class="title" tagx="title" title="title">
Three approaches</div> <div class="approach1:non-parametricgangeranalysisofcounty-specificincidence" title="sec"> <div class="title" tagx="title" title="title">
Approach 1: Non-parametric Ganger analysis of county-specific incidence</div> <div class="analysisofquantizedclinicaldata" title="sec"> <div class="title" tagx="title" title="title">
Analysis of quantized clinical data</div> <p>For the causality analysis, we started with an integer-valued time series for each US county, and to carry out the causality analysis, we first quantized the series in two steps:</p> <ol> <li> <p>1. Computing the difference series, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, the weekly change in the number of reported cases.</p> </li> <li> <p>2. Mapping positive changes to symbol ‘1’ and negative changes to symbol ‘0’.</p> </li> </ol> <p>This mapped each data series to a symbol stream over a binary alphabet. The binary quantization is not a restriction imposed by the inference algorithm; while we do require quantized magnitudes, longer data streams can be encoded with finer alphabets to accommodate an arbitrary precision. For this specific dataset, the relatively short length of the county-specific time-series necessitated a coarse quantization in order for the results to have a meaningful statistical significance.</p> <p>Given a pair of such quantized streams <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>, the algorithm described in the Supplement computes two coefficients of causality, one for each direction. Intuitively, the coefficient <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> is a non-dimensional number between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span> that quantifies the amount of information that one may acquire about the second stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> from observing the first stream (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span>). More specifically, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> is the average reduction in the uncertainty of the next predicted symbol in stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> in bits, per bit, acquired from observed symbols in stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span>. It can be shown that the coefficient of causality <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span> if and only if there is no causal influence from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> in the sense of Granger, and assumes the maximum value <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span> if and only if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> is deterministically predictable from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span>. Moreover, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> if and only if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
b</div> </div></div></span> are statistically independent processes (<a href="#bib22">Chattopadhyay, 2014</a>). It is trivial to produce examples where we would have <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> illustrating the ability of the algorithm to capture the asymmetric flow of causal influence in one preferred direction and not the other.</p> <p>Additionally, whenever we computed the causality coefficient, it was always associated with a time delay: We calculated the coefficient for predicting the target stream some specified number of steps in the future, and the computed coefficient was thus parameterized by this delay. In our analysis, we computed coefficients up to a maximum delay of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
10</div></div></span> weeks, and, for each pair of counties, selected the optimum delay which gave rise to the largest coefficient. For more details on the algorithm, see the Supplement.</p> </div> <div class="computationofcausalityfieldsandcausalitystreamlines" title="sec"> <div class="title" tagx="title" title="title">
Computation of causality fields and causality streamlines</div> <p>To analyze the propagation dynamics of disease, we defined the notion of a Granger <i>causal flow</i> between two counties. Treating county-specific changes in disease prevalence as a time-stamped data stream, we quantified the directional strength of the causal flow between two counties as the degree of predictability of one stream’s future, given the history of the other (see <a href="#fig3">Figure 3A–D</a>). Our new algorithm was able to compute a <i>coefficient of causality</i>: an information-theoretic measure of the information in bits that one stream communicates about another in a direction-specific manner. A strong coefficient of this type suggests that either the disease itself, or an underlying cause, may be propagating from the former county to the latter.</p> <p>To explore the country-wide propagation dynamics, we stitched together the properly-aligned, adjacent between-county causal flow vectors across the whole US map into <i>causality streamlines</i>, representing the average of multiple spatial infection propagation waves over time. To compute causality streamlines, we needed a precise notion of county neighbors. We considered two counties to be neighbors if either they shared a common border, or if one county was reachable within a line-of-sight distance of 50 miles from any point within another. The latter condition removed ambiguities as to whether counties touching at a point should be still considered as neighbors. The exact distance value (50 miles) does not significantly change our results, as long as it does not vary significantly. With the definition of the neighborhood map in place, we proceeded to compute the direction-specific coefficients of causality between neighboring counties. It follows that we would obtain a set of coefficients for each county and one for each of its neighbors, capturing the degree of causal influence from a given particular county to its respective neighbors. Our algorithm also computed the probability of each coefficient arising by chance alone, and we ignored coefficients that have more than a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
6</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> probability that two independent processes lacking any causal connection gave rise to the particular computed value of the coefficient. Once the coefficients had been calculated for each neighbor, we computed the resultant direction of causal influence outflow from that particular county. This was carried out by visualizing the causality coefficients as weights on the length of the vectors, from the centroid of the considered county to the centroids of its neighbors. We then calculated the resultant vector (see <a href="#fig3">Figure 3</a>). Viewed systematically across the continental US, these local vectors formed a discernible pattern; we observed the emergence of a non-random structure with clearly discernible paths outlining the ‘causality field’ (see <a href="#fig3">Figure 3</a>, Plates G, J, K). To interpret the plots, note that streamlines start at their thinnest part, their direction is indicted with thickening line; typically multiple streamlines coalesce into a river-like pattern.</p> </div> <div class="inferringstatistical‘granger-causality’fromdata" title="sec"> <div class="title" tagx="title" title="title">
Inferring statistical ‘Granger-causality’ from data</div> <p>Granger attempted to obtain a precise definition of causal influence (<a href="#bib46">Granger, 1980</a>) from the following intuitive notion: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div></div></span> is a cause of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div></div></span>, if it has unique information that alters the probabilistic estimate of the immediate future of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div></div></span>.</p> <p>Here, we used a new, non-parametric approach to Granger causal inference (<a href="#bib22">Chattopadhyay, 2014</a>) (Approach 1). In contrast to state-of-the-art binary tests (<a href="#bib7">Baek and Brock, 1992</a>; <a href="#bib50">Hiemstra and Jones, 1994</a>), we computed the degree of causal dependence between quantized data streams from stationary ergodic sources in a non-parametric, non-linear setting.</p> <p>Our approach was significantly more general to common, regression-based implementations of Granger causal inference, and did not involve any autoregressive moving average (ARMA) modeling. All such commonly used techniques impose an <i>a priori</i> linear structure on the data, which then constrains the class of dependency structures we can hope to distill.</p> <p>True causality, in a Humean sense, cannot be inferred (<a href="#bib53">Hume, 1993</a>; <a href="#bib54">Kant, 1998</a>). Among other reported approaches to ascertaining causal dependency relationships, the work of J. Pearl (<a href="#bib69">Pearl, 2009a</a>) is perhaps most visible, and builds on the paradigm of structural causal models (SCM) (<a href="#bib70">Pearl, 2009b</a>). While Pearl’s work is often claimed to be able to answer causal queries regarding the effects of potential interventions, as well as regarding counterfactuals, our objective in this paper is somewhat different. We are interested in delineating whether infection transmission pathways can be distilled from the patterns of infection’s spatio-temporal incidence.</p> </div> </div> <div class="approach2:poissonregressiononputativefactors" title="sec"> <div class="title" tagx="title" title="title">
Approach 2: Poisson regression on putative factors</div> <p>In Approach 2, we investigated the relative importance of putative factors as follows: Specifically, let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div></div></span> denote the total number of patients in county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>, who are of age <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, and gender <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span>. Denoting the number of individuals diagnosed with influenza in a given county during given week as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div></div></span>, we modeled the within-county disease incidence counts for every county (for which data was available) in the US using the following mixed-effect Poisson regression model: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
with </div> </div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mi_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div></div></div>where, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
α</div></div></span> is the intercept, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> are a fixed-effect design matrix and a vector of fixed effects, respectively, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> vector of changes in rate of infection in the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
.</div> <div class="mo_UNKNOWN">
.</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> weeks prior to the current, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span> is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> vector of auto-correlation fixed effects, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
×</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> vector of changes in the rate of infection in the neighbors of the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
.</div> <div class="mo_UNKNOWN">
.</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> weeks prior to the current (the neighbors are subdivided into <i>land</i> neighbors, <i>Twitter</i> neighbors, and <i>air</i> neighbors), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span> is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> vector of county-neighborhoods fixed effects, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> are random effects and their design matrix, respectively. Variable <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
m</div></div></span> represents the depth of ‘memory’ of auto-regression in weeks, in our case <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4</div> </div></div></span>.<p /> <p>In this way, the total number of rows in matrix <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div></div></span> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
510</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
143</div> </div></div></span> (weeks <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
×</div></div></span> counties), with county-specific socioeconomic covariates and week-specific weather covariates. For disease initiation analysis, we included only a subset of time series covering approximately <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
50</div></div></span> weeks.</p> <p>We did not use an explicit spatial smoothing. The county-level random effects were used to account for possible heterogeneity in disease reporting across counties. However, as predictors, we used average infection rate values of immediate neighbors for the county (one, two, three, and four weeks in the past). This prediction structure afforded an implicit spatial smoothing, dampening spurious fluctuations in infection rates.</p> <div class="out-of-samplepredictionandrocanalysiswithmixed-effectpoissonregression" title="sec"> <div class="title" tagx="title" title="title">
Out-of-sample prediction and ROC analysis with mixed-effect poisson regression</div> <p>We carried our out-of-sample prediction with the models inferred with mixed-effect regression. The steps were as follows:</p> <ol> <li> <p>We trained the model parameters with data from the trigger periods corresponding to the first six seasons.</p> </li> <li> <p>Once we identified the coefficient’s variables, we used it to predict the response variable (influenza incidence) for the last three seasons.</p> </li> </ol> <p>As expected, the predicted incidence does not exactly match the observed out-of-sample data. Nevertheless, we see positive correlation (<a href="#fig5">Figure 5</a>, Plate A). Because we were modeling a necessarily spatio-temporal stochastic process, the predictive ability of the model is difficult to judge simply from the observed positive correlation. To resolve this, we investigated the performance of our model by computing how well it predicted the counties that experience flare-ups during the trigger-periods in the out-of-sample data. This exercise is reduced to a classification problem, by first choosing a threshold on the number of reported cases per week to define what is meant by a ‘flare-up’ (see description below). To quantify the prediction performance, we constructed ROC curves for each of the three target seasons, for each fixed week, as follows:</p> <ol> <li> <p>We first quantized the incidence data to reduce it to a binary stream. In particular, we chose a threshold (ten), such that for each county, and each week, we reported a ‘1’ if the number of reported cases was greater than the threshold, and ‘0’ otherwise. Note this quantization is different to what we used in carrying out our non-parametric Granger analysis in Approach 1.</p> </li> <li> <p>For each county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>, and week <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, we then ended up with the binary class variable <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
}</div> </div> </div></div></span> and a decision variable <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?</div> </div></div></span>, where the former is the quantized incidence described above, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span> is the response predicted by the model, normalized between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span>.</p> </li> <li> <p>For a chosen decision threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
D</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?</div> </div></div></span>, we could determine the predicted class <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
}</div> </div> </div></div></span> as:<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
 if </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
≧</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
 otherwise</div> </div> </div> </div> </div> <div class="mo_UNKNOWN" /> </div> </div> </div> </div> </div> </div> </div> </div></div></span></p> </li> <li> <p>Comparing the observed and predicted classes, we computed the false positive rate (FPR: defined as the ratio of false positives to the sum of false positives and true negatives), and the true positive rate (TPR: defined as the ratio of true positives to the sum of true positives and false negatives). Finally, we constructed the ROC curve, which shows the relationship of the TPR and the FPR as the decision-threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
D</div> </div></div></span> is varied.</p> </li> <li> <p>We constructed Receiver Operating Characteristic (ROC) curves for each week in the out-of-sample period, and estimated the area under curve (AUC). The AUC measures the performance of the predictor (our model) to correctly classify the counties that would go on to have a disease incidence greater than the initial set threshold (ten cases in our analysis). In the perfect case, we would have an AUC of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> </div></div></span>, which implies that we can achieve zero false positives, while getting a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> true positive rate. Our best model achieves approximately <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
80</div> <div class="mo_UNKNOWN">
%</div> </div></div></span> AUC for the trigger weeks, as shown in <a href="#fig5">Figure 5</a>.</p> </li> </ol> </div> </div> <div class="variableandmodelselection" title="sec"> <div class="title" tagx="title" title="title">
Variable and model selection</div> <p>Approaches 2 and 3 have different limitations; these affect which approach is most effective at which statistical detection task.</p> <p>Approach 2 (mixed-effect regression analysis) can use a whole dataset for model selection, that can span individual predictors, their combinations, and even interactions between factors (see <a href="#table5">Table 5</a>; <a href="#fig1s2">Figure 1—figure supplement 2</a>; <a href="#fig1s3">Figure 1—figure supplement 3</a>; <a href="#fig1s4">Figure 1—figure supplement 4</a>; <a href="#sdata2">Source data 2</a>; <a href="#sdata3">Source data 3</a>; <a href="#supp3">Supplementray file 3</a>; <a href="#scode1">Source code 1</a>). Model selection allows us to choose a model with the right balance of complexity and explanatory power, thus enabling this analysis to detect collinear explanatory variables and drop the weaker predictors that increase model complexity but do not add explanatory power.</p> <div class="table-wrap_UNKNOWN" id="table5" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.30756.021">10.7554/eLife.30756.021</a><span class="label" tagx="label" title="label">Table 5.</span> <div class="title" tagx="title" title="title">
Different Models Considered and DIC Ranking</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Equation used in Poisson regression</th> <th rowspan="1" colspan="1">DIC</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + max_HUS_mean * t_avg_mean + d_max_HUS_0 * d_t_avg_0 + d_max_HUS_min_1 * d_t_avg_min_1 + d_max_HUS_min_2 * d_t_avg_min_2 + d_max_HUS_min_3 * d_t_avg_min_3 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.</td> <td rowspan="1" colspan="1">185942</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + d_max_HUS_min_4 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + d_t_avg_min_4 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + d_RSDS_min_4 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + d_AVG_PRESS_min_4 + tot_prec_mean + d_tot_prec_0 + d_tot_prec_min_1 + d_tot_prec_min_2 + d_tot_prec_min_3 + d_tot_prec_min_4 + Wind_avg_mean + d_Wind_avg_0 + d_Wind_avg_min_1 + d_Wind_avg_min_2 + d_Wind_avg_min_3 + d_Wind_avg_min_4 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.+HA * NA.</td> <td rowspan="1" colspan="1">185940.6</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + max_HUS_mean * t_avg_mean + d_max_HUS_0 * d_t_avg_0 + d_max_HUS_min_1 * d_t_avg_min_1 + d_max_HUS_min_2 * d_t_avg_min_2 + d_max_HUS_min_3 * d_t_avg_min_3 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4air1+air2+air3+air4+HA + M1+M2+NA.+Cum_vac_per_N_0</td> <td rowspan="1" colspan="1">185938.1</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + d_max_HUS_min_4 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + d_t_avg_min_4 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + d_RSDS_min_4 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + d_AVG_PRESS_min_4 + tot_prec_mean + d_tot_prec_0 + d_tot_prec_min_1 + d_tot_prec_min_2 + d_tot_prec_min_3 + d_tot_prec_min_4 + Wind_avg_mean + d_Wind_avg_0 + d_Wind_avg_min_1 + d_Wind_avg_min_2 + d_Wind_avg_min_3 + d_Wind_avg_min_4 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.+Cum_vac_per_N_diff_1 + Cum_vac_per_N_diff_2 + Cum_vac_per_N_diff_3 + Cum_vac_per_N_diff_4</td> <td rowspan="1" colspan="1">185935.9</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + d_max_HUS_min_4 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + d_t_avg_min_4 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + d_RSDS_min_4 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + d_AVG_PRESS_min_4 + tot_prec_mean + d_tot_prec_0 + d_tot_prec_min_1 + d_tot_prec_min_2 + d_tot_prec_min_3 + d_tot_prec_min_4 + Wind_avg_mean + d_Wind_avg_0 + d_Wind_avg_min_1 + d_Wind_avg_min_2 + d_Wind_avg_min_3 + d_Wind_avg_min_4 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.</td> <td rowspan="1" colspan="1">185933.7</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + d_max_HUS_min_4 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + d_t_avg_min_4 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + d_RSDS_min_4 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + d_AVG_PRESS_min_4 + tot_prec_mean + d_tot_prec_0 + d_tot_prec_min_1 + d_tot_prec_min_2 + d_tot_prec_min_3 + d_tot_prec_min_4 + Wind_avg_mean + d_Wind_avg_0 + d_Wind_avg_min_1 + d_Wind_avg_min_2 + d_Wind_avg_min_3 + d_Wind_avg_min_4 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.+Cum_vac_per_N_0</td> <td rowspan="1" colspan="1">185932.3</td> </tr> <tr> <td rowspan="1" colspan="1">flu <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>LOGN + 1 + max_HUS_mean + d_max_HUS_0 + d_max_HUS_min_1 + d_max_HUS_min_2 + d_max_HUS_min_3 + d_max_HUS_min_4 + t_avg_mean + d_t_avg_0 + d_t_avg_min_1 + d_t_avg_min_2 + d_t_avg_min_3 + d_t_avg_min_4 + RSDS_mean + d_RSDS_0 + d_RSDS_min_1 + d_RSDS_min_2 + d_RSDS_min_3 + d_RSDS_min_4 + AVG_PRESS_mean + d_AVG_PRESS_0 + d_AVG_PRESS_min_1 + d_AVG_PRESS_min_2 + d_AVG_PRESS_min_3 + d_AVG_PRESS_min_4 + tot_prec_mean + d_tot_prec_0 + d_tot_prec_min_1 + d_tot_prec_min_2 + d_tot_prec_min_3 + d_tot_prec_min_4 + Wind_avg_mean + d_Wind_avg_0 + d_Wind_avg_min_1 + d_Wind_avg_min_2 + d_Wind_avg_min_3 + d_Wind_avg_min_4 + Income + airport_diffusion + Am_Ind + Asian + White_Hisp + W_non_Hisp + Black_Hisp + B_non_Hisp + Pacific + Insured+Poor + Urban+v1+v2+land1+land2+land3+land4+tweet1+tweet2+tweet3+tweet4+air1+air2+air3+air4+HA + M1+M2+NA.+Cum_vac_per_N_0 + Cum_vac_per_N_1 + Cum_vac_per_N_2 + Cum_vac_per_N_3</td> <td rowspan="1" colspan="1">185926.6</td> </tr> </tbody> </table> </div> <p>For example, the best model in our mixed-effect regression model series, with deviance information criterion (DIC) <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
185</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
926.6</div> </div></div></span> (see <a href="#table5">Table 5</a>, the last model), includes a term d_max_HUS_min_3 * d_t_avg_min_3 that denotes an interaction between the weekly change in the maximum specific humidity and the weekly change in the average temperature, both of which are weather parameter changes recorded three weeks before the current week. Because it does not split datasets, Approach two is more powerful at detecting interactions between explanatory variables and rather complicated models. However, this approach is more susceptible to bias (picking non-causal correlations that are inherent in the raw data), if the data are unbalanced, unlike with Approach 3.</p> <p>We added and removed random effects to the model structure, taking out and adding fixed effects, executing the regression algorithm, and plotting the DIC achieved against model complexity. Here, model complexity is simply proxied by the descriptional complexity in terms of how large the model was (the number of factors in the regression equation). We stopped when the drop in DIC stabilized (See <a href="#fig1s3">Figure 1—figure supplement 3</a> ). Notably, the DIC has been shown to select overfits (<a href="#bib2">Ando, 2011</a>; <a href="#bib71">Plummer, 2008</a>; <a href="#bib84">van der Linde, 2012</a>), and does not properly account for model complexity in practice. Our driving idea was the identification of the Pareto front that trades off accuracy (in terms of DIC) with model complexity in a transparent manner. This is, of course, a ‘greedy approach,’ in the sense that we do not guarantee that we have indeed found the ‘best’ possible model. However, because our cross-validation (<a href="#fig5">Figure 5</a>) yielded good results, we deemed the stopping rule to be satisfactory.</p> </div> <div class="approach3:county-matchingeffectanalysis" title="sec"> <div class="title" tagx="title" title="title">
Approach 3: County-matching effect analysis</div> <p>We designed Approach three specifically to balance biased observational data. While Approach three does not allow for explicit model selection, it is good at detecting simpler combinations of predictors putatively ‘causally’ affecting the outcome variable. However, because this approach involves splitting data into smaller and smaller subsets, as the complexity of predictor function (‘treatment’) grows, it quickly runs out of statistical power.</p> <p>These two approaches’ properties explain the perceived misalignment of results from the two streams of analysis. For an intuitive understanding of matching approaches, consider specific humidity’s effect on influenza prevalence. The county-matching method’s goal is to deduce associations putatively interpreted as causality relations. For example, consider testing the question of whether counties with higher-than-average mean maximum specific humidity do, indeed, have higher influenza prevalence, in a statistically significant sense, when compared to counties that do not, provided all other factors are held constant.</p> <p>Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�?�?</div></div></span> denote the set of all US counties, and let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div></div></span> be the set of all factors we find to be significant in our mixed-effect regression analysis. Now, for any subset of factors <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
⫅</div> <div class="mi_UNKNOWN">
�?��</div> </div></div></span>, we denote the complement set as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
∖</div> <div class="mi_UNKNOWN">
K</div> </div> </div></div></span>. Additionally, we define the Boolean function <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
�?�?</div> </div> <div class="mo_UNKNOWN">
→</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div> </div></div></span>, (the treatment function) where for some factors <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?��</div> </div></div></span>, and some counties <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?�?</div> </div></div></span>, the Boolean value <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is true if the signal or treatment corresponding to the factor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
s</div></div></span> is present in county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
y</div></div></span>. We used the sign for the coefficients we obtained in our best mixed-effect regression model to determine what counts as a positive signal. For example, because maximum specific humidity (denoted by the variable <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
_</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
_</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
g</div> </div></div></span>) enters with a positive coefficient, if county <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
y</div></div></span> experiences a higher-than-average maximum specific humidity, then <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
_</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
_</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is true.</p> <p>To simplify the notation, we used <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
s</div> </div></div></span> to denote a true signal, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
¬</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
s</div> </div> </div></div></span> to denote a false one.</p> <p>Finally, for any <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
⫅</div> <div class="mi_UNKNOWN">
�?��</div> </div></div></span>, we define the following three sets, which we refer to as the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
W</div></div></span>-sets: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
K</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?�?</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
⋀</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
K</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mi_UNKNOWN">
K</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN" /> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
�?�?</div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
¬</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
⋀</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
K</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
⋀</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
∃</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
∀</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∈</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
∧</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
∨</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
¬</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
∧</div> <div class="mi_UNKNOWN">
¬</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
}</div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
∈</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
¬</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
⋀</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
K</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
∖</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
l</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> </div> </div> </div> </div></div></div> <p /> <p>Clearly, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
K</div> </div></div></span> is the treated set, that is, the set of counties which exhibit the signal encoded by the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> </div></div></span> is then the matched control set of counties, which lack the signal, but each county in this set has a matching counterpart in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
K</div> </div></div></span>.</p> <p>The <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
W</div></div></span>-sets allow us to set up a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> contingency table for any chosen subset of factors <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
⫅</div> <div class="mi_UNKNOWN">
�?��</div> </div></div></span> as described before. Specifically, we split the sets <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
K</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mi_UNKNOWN">
K</div> </div></div></span> into two subsets each, representing those counties which experience a spike in influenza prevalence and which do not. The <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> contingency table is then subjected to Fisher’s exact test.</p> </div> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01HL108634-01</span> to Jeffrey L Shaman.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01GM100467</span> to Jeffrey L Shaman.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01GM110748</span> to Jeffrey L Shaman.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">1P50MH094267</span> to Andrey Rzhetsky.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100006502</span><span class="institution" title="institution">Defense Sciences Office, DARPA</span></div> </div><span class="award-id" tagx="award-id" title="award-id">W911NF1410333</span> to Andrey Rzhetsky.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01HL122712</span> to Andrey Rzhetsky.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Erin Gannon, Mark Jit, Prabhat Jha, Elizabeth C Lee, and Margarita Rzhetsky for numerous comments on earlier versions of the manuscript. This work was supported by NIH grants 1P50MH094267, U01HL108634-01, R01HL122712-02, R01GM100467, and U01GM110748, in addition to DARPA contract W911NF1410333, and a gift from Liz and Kent Dauten.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Software, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Formal analysis, Methodology, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Data curation, Formal analysis, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Software, Formal analysis, Supervision, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft, Project administration, Writing—review and editing.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.022">10.7554/eLife.30756.022</a><span class="label" tagx="label" title="label">Source code 1.</span> <div class="title" tagx="title" title="title">
R code for analyses in Approach 2: mcmc_flu.r</div> <p>Raw outputs of the regression experiments in Approach 2: R Output Logs: models computed using only weeks associated with initiation of influenza wave.</p>  <div class="media" title="media"><a href="elife-30756-code1.r">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.023">10.7554/eLife.30756.023</a><span class="label" tagx="label" title="label">Source code 2.</span> <div class="title" tagx="title" title="title">
County-matching code for Approach 3: counterfactual.cc</div>  <div class="media" title="media"><a href="elife-30756-code2.cc">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.024">10.7554/eLife.30756.024</a><span class="label" tagx="label" title="label">Source data 1.</span> <div class="title" tagx="title" title="title">
Streamline raw data for <a href="#fig4">Figure 4G</a>: streamline_data.zip</div> <p>Each streamline is a a series of physical coordinates (latitude and longitude in degrees), with a linebreak after each streamline.</p>  <div class="media" title="media"><a href="elife-30756-data1.zip">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.025">10.7554/eLife.30756.025</a><span class="label" tagx="label" title="label">Source data 2.</span> <div class="title" tagx="title" title="title">
Analysis of 50% of all weeks in study with mixed-effect Poisson regression: flu-50-percent-weeks.txt</div>  <div class="media" title="media"><a href="elife-30756-data2.txt">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.026">10.7554/eLife.30756.026</a><span class="label" tagx="label" title="label">Source data 3.</span> <div class="title" tagx="title" title="title">
Raw outputs of all mixed-effect regression analyses in this study.</div>  <div class="media" title="media"><a href="elife-30756-data3.txt">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.027">10.7554/eLife.30756.027</a><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
Summarized Algorithms for Non-parametric Granger Causal Inference:</div> <p>SI2.docx</p>  <div class="media" title="media"><a href="elife-30756-supp1.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.028">10.7554/eLife.30756.028</a><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Back to School Effect Analysis.</div>  <div class="media" title="media"><a href="elife-30756-supp2.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.029">10.7554/eLife.30756.029</a><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
Regression Equations and Corresponding DIC values.</div>  <div class="media" title="media"><a href="elife-30756-supp3.txt">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.030">10.7554/eLife.30756.030</a><span class="label" tagx="label" title="label">Supplementary file 4.</span> <div class="title" tagx="title" title="title">
Regression Equations for all models tested in our regression analysis along with corresponding DIC values.</div> <p>Balancing Precision and Sensitivity when Identifying Influenza-like Illnesses from Electronic Medical Records.</p>  <div class="media" title="media"><a href="elife-30756-supp4.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.30756.031">10.7554/eLife.30756.031</a><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-30756-transrepform.docx">LINK</a></div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarado-Facundo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribas-Aparicio</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiménez-Alberto</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Influenza virus M2 protein ion channel activity helps to maintain pandemic 2009 H1N1 virus hemagglutinin fusion competence during transport to the cell surface</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">1975</span><span class="lpage" tagx="lpage" title="lpage">1985</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03253-14">10.1128/JVI.03253-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25473053">25473053</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ando</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Predictive bayesian model selection</span><span class="source" tagx="source" title="source">American Journal of Mathematical and Management Sciences</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/01966324.2011.10737798">10.1080/01966324.2011.10737798</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreasen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Dynamics of annual influenza A epidemics with immuno-selection</span><span class="source" tagx="source" title="source">Journal of Mathematical Biology</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">504</span><span class="lpage" tagx="lpage" title="lpage">536</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00285-002-0186-2">10.1007/s00285-002-0186-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12783180">12783180</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Araz</span><span class="given-names" tagx="given-names" title="given-names">OM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Damien</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paltiel</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van de Geijn</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galvani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyers</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Simulating school closure policies for cost effective pandemic decision making</span><span class="source" tagx="source" title="source">BMC Public Health</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">449</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2458-12-449">10.1186/1471-2458-12-449</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22713694">22713694</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Areas</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span></span><span class="year" tagx="year" title="year">2014</span>[data-title] GADM database<a href="http://gadm.org/">http://gadm.org/</a></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asimakopoulos</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansor Mahmood</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Non-linear Granger causality in the currency futures returns</span><span class="source" tagx="source" title="source">Economics Letters</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0165-1765(00)00219-6">10.1016/S0165-1765(00)00219-6</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baek</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brock</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="source" tagx="source" title="source">A General Test for Nonlinear Granger Causality: Bivariate Model</span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balcan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colizza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonçalves</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramasco</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Multiscale mobility networks and the spatial spreading of infectious diseases</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">21484</span><span class="lpage" tagx="lpage" title="lpage">21489</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0906910106">10.1073/pnas.0906910106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20018697">20018697</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balcan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Phase transitions in contagion processes mediated by recurrent mobility patterns</span><span class="source" tagx="source" title="source">Nature Physics</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">581</span><span class="lpage" tagx="lpage" title="lpage">586</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nphys1944">10.1038/nphys1944</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21799702">21799702</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barreca</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimshack</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Absolute humidity, temperature, and influenza mortality: 30 years of county-level evidence from the United States</span><span class="source" tagx="source" title="source">American Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">176</span><span class="fpage" tagx="fpage" title="fpage">S114</span><span class="lpage" tagx="lpage" title="lpage">S122</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aje/kws259">10.1093/aje/kws259</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23035135">23035135</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broor</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chadha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunasekaran</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCauley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potdar</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tashiro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Global circulation patterns of seasonal influenza viruses vary with antigenic drift</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">523</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature14460">10.1038/nature14460</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26053121">26053121</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreasen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christiansen</span><span class="given-names" tagx="given-names" title="given-names">FB</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Influenza drift and epidemic size: the race between generating and escaping immunity</span><span class="source" tagx="source" title="source">Theoretical Population Biology</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="lpage" tagx="lpage" title="lpage">191</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tpb.2003.10.002">10.1016/j.tpb.2003.10.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14766191">14766191</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreasen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldman</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Epidemic dynamics and antigenic evolution in a single season of influenza A</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">273</span><span class="fpage" tagx="fpage" title="fpage">1307</span><span class="lpage" tagx="lpage" title="lpage">1316</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2006.3466">10.1098/rspb.2006.3466</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Vaccination and antigenic drift in influenza</span><span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">C8</span><span class="lpage" tagx="lpage" title="lpage">C14</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2008.04.011">10.1016/j.vaccine.2008.04.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18773534">18773534</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brock</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Causality, chaos, explanation and prediction in economics and finance</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casti</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlqvist</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="source" tagx="source" title="source">Beyond Belief: Randomness, Prediction, and Explanation in Science</span><span class="fpage" tagx="fpage" title="fpage">230</span><span class="lpage" tagx="lpage" title="lpage">279</span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helbing</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The hidden geometry of complex, network-driven contagion phenomena</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">342</span><span class="fpage" tagx="fpage" title="fpage">1337</span><span class="lpage" tagx="lpage" title="lpage">1342</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1245200">10.1126/science.1245200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24337289">24337289</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: CDC</span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The United states center for disease control and prevention</span><a href="http://www.cdc.gov/flu/weekly/overview.htm">http://www.cdc.gov/flu/weekly/overview.htm</a><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">August, 2016</span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Centers for Disease Control and Prevention</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiore</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shay</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broder</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iskander</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uyeki</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mootrey</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bresee</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Prevention and control of seasonal influenza with vaccines: recommendations of the Advisory Committee on Immunization Practices (ACIP), 2009</span><span class="source" tagx="source" title="source">MMWR. Recommendations and Reports : Morbidity and Mortality Weekly Report. Recommendations and Reports</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">52</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19644442">19644442</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charu</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeger</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjørnstad</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kissler</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Human mobility and the spatial transmission of influenza in the United States</span><span class="source" tagx="source" title="source">PLoS Computational Biology</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1005382</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1005382">10.1371/journal.pcbi.1005382</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28187123">28187123</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Abductive learning of quantized stochastic processes with probabilistic finite automata</span><span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences</span><span class="volume" tagx="volume" title="volume">371</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20110543</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsta.2011.0543">10.1098/rsta.2011.0543</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23277601">23277601</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Structural transformations of probabilistic finite state machines</span><span class="source" tagx="source" title="source">International Journal of Control</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">820</span><span class="lpage" tagx="lpage" title="lpage">835</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00207170701704746">10.1080/00207170701704746</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Causality networks</span><span class="source" tagx="source" title="source">arXiv</span><a href="http://arxiv.org/abs/1406.6651">http://arxiv.org/abs/1406.6651</a></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chlanda</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schraidt</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kummer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riches</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberwinkler</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prinz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kräusslich</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briggs</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Structural analysis of the roles of influenza A virus membrane-associated proteins in assembly and morphology</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">8957</span><span class="lpage" tagx="lpage" title="lpage">8966</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00592-15">10.1128/JVI.00592-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26085153">26085153</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowell</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Towers</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuentes</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sotomayor</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lima</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villarroel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villarroel</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olea</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The influence of climatic conditions on the transmission dynamics of the 2009 A/H1N1 influenza pandemic in Chile</span><span class="source" tagx="source" title="source">BMC Infectious Diseases</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">298</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-12-298">10.1186/1471-2334-12-298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23148597">23148597</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danks</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glymour</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="source" tagx="source" title="source">Data Driven Methods for Nonlinear Granger Causality: Climate Teleconnection Mechanisms</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Pittsburgh, Pennsylvania</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Carnegie Mellon University</span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciofi degli Atti</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merler</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizzo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ajelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massari</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manfredi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furlanello</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scalia Tomba</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iannelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Mitigation measures for pandemic influenza in Italy: an individual based model considering different scenarios</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1790</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0001790">10.1371/journal.pone.0001790</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18335060">18335060</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colizza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barthélemy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">The role of the airline transportation network in the prediction and predictability of global epidemics</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">2015</span><span class="lpage" tagx="lpage" title="lpage">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0510525103">10.1073/pnas.0510525103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16461461">16461461</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cosgrove</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houser</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaake</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robock</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheffield</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinker</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarpley</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Real-time and retrospective forcing in the North American land data assimilation system (NLDAS) project</span><span class="source" tagx="source" title="source">Journal of Geophysical Research</span><span class="volume" tagx="volume" title="volume">108</span><span class="pub-id"><a href="https://dx.doi.org/10.1029/2002JD003118">10.1029/2002JD003118</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crutchfield</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">The calculi of emergence: computation, dynamics and induction</span><span class="source" tagx="source" title="source">Physica D: Nonlinear Phenomena</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0167-2789(94)90273-9">10.1016/0167-2789(94)90273-9</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darnell</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="source" tagx="source" title="source">The Limits of Econometrics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Aldershot, Hampshire</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Gower</span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davey</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beyeler</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Effective, robust design of community mitigation for pandemic influenza: a systematic examination of proposed US guidance</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e2606</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0002606">10.1371/journal.pone.0002606</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18596963">18596963</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davey</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Rescinding community mitigation strategies in an influenza pandemic</span><span class="source" tagx="source" title="source">Emerging Infectious Diseases</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">372</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1403.070673">10.3201/eid1403.070673</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18325247">18325247</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Clercq</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Antiviral agents active against influenza A viruses</span><span class="source" tagx="source" title="source">Nature Reviews Drug Discovery</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1015</span><span class="lpage" tagx="lpage" title="lpage">1025</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd2175">10.1038/nrd2175</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17139286">17139286</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denker</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1983</span><span class="mixed-article-title" title="mixed-article-title">On U-statistics and v. mise? statistics for weakly dependent processes</span><span class="source" tagx="source" title="source">Zeitschrift fur Wahrscheinlichkeitstheorie Und Verwandte Gebiete</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">505</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00534953">10.1007/BF00534953</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diks</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panchenko</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">A new statistic and practical guidelines for nonparametric Granger causality testing</span><span class="source" tagx="source" title="source">Journal of Economic Dynamics and Control</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1669</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jedc.2005.08.008">10.1016/j.jedc.2005.08.008</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eggo</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cauchemez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Spatial dynamics of the 1918 influenza pandemic in England, Wales and the United States</span><span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">233</span><span class="lpage" tagx="lpage" title="lpage">243</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2010.0216">10.1098/rsif.2010.0216</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elveback</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fox</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ackerman</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langworthy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyd</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatewood</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">1976</span><span class="mixed-article-title" title="mixed-article-title">An influenza simulation model for immunization studies</span><span class="source" tagx="source" title="source">American Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">152</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.aje.a112213">10.1093/oxfordjournals.aje.a112213</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/814808">814808</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Epstein</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="source" tagx="source" title="source">A History of Econometrics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">North-Holland</span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cajka</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooley</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Strategies for mitigating an influenza pandemic</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">442</span><span class="fpage" tagx="fpage" title="fpage">448</span><span class="lpage" tagx="lpage" title="lpage">452</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature04795">10.1038/nature04795</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16642006">16642006</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">1935</span><span class="source" tagx="source" title="source">The Design of Experiments</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Edinburgh, London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oliver and Boyd</span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drori</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pando</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glatman-Freedman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sefty</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassal</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shohat</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendelson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hindiyeh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandelboim</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">A(H1N1)pdm09 influenza infection: vaccine inefficiency</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">32856</span><span class="lpage" tagx="lpage" title="lpage">32863</span><span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.16459">10.18632/oncotarget.16459</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28415629">28415629</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heffernan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Influenza pandemic waves under various mitigation strategies with 2009 H1N1 as a case study</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e14307</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0014307">10.1371/journal.pone.0014307</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21187938">21187938</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnes</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">How much would closing schools reduce transmission during an influenza pandemic?</span><span class="source" tagx="source" title="source">Epidemiology</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">623</span><span class="lpage" tagx="lpage" title="lpage">628</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/EDE.0b013e31812713b4">10.1097/EDE.0b013e31812713b4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17700251">17700251</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beyeler</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Targeted social distancing design for pandemic influenza</span><span class="source" tagx="source" title="source">Emerging Infectious Diseases</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1671</span><span class="lpage" tagx="lpage" title="lpage">1681</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1211.060255">10.3201/eid1211.060255</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17283616">17283616</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballesteros</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjornstad</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chao</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Spatial Transmission of 2009 Pandemic Influenza in the US</span><span class="source" tagx="source" title="source">PLoS Computational Biology</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003635</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003635">10.1371/journal.pcbi.1003635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24921923">24921923</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granger</span><span class="given-names" tagx="given-names" title="given-names">CWJ</span></span></span><span class="year" tagx="year" title="year">1980</span><span class="mixed-article-title" title="mixed-article-title">Testing for causality</span><span class="source" tagx="source" title="source">Journal of Economic Dynamics and Control</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">329</span><span class="lpage" tagx="lpage" title="lpage">352</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0165-1889(80)90069-X">10.1016/0165-1889(80)90069-X</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haber</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shay</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">XM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weintraub</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orenstein</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Effectiveness of interventions to reduce contact rates during a simulated influenza pandemic</span><span class="source" tagx="source" title="source">Emerging Infectious Diseases</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">581</span><span class="lpage" tagx="lpage" title="lpage">589</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1304.060828">10.3201/eid1304.060828</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17553273">17553273</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The truven health marketscan databases for life sciences researchers</span><span class="source" tagx="source" title="source">Truven Health Ananlytics IBM Watson Health</span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedeker</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Longitudinal data analysis</span><span class="source" tagx="source" title="source">Wiley Series in Probability and Statistics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken, N.J</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley-Interscience</span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiemstra</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Testing for linear and nonlinear granger causality in the stock price-volume relation</span><span class="source" tagx="source" title="source">The Journal of Finance</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">1639</span><span class="lpage" tagx="lpage" title="lpage">1664</span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiemstra</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kramer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Nonlinearity and endogeneity in macro-asset pricing</span><span class="source" tagx="source" title="source">IMF Working Papers</span><span class="volume" tagx="volume" title="volume">95</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1</span><span class="pub-id"><a href="https://dx.doi.org/10.5089/9781451845082.001">10.5089/9781451845082.001</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: HRSA</span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Health reasources and services administration</span><a href="http://datawarehouse.hrsa.gov/tools/dataportal.aspx">http://datawarehouse.hrsa.gov/tools/dataportal.aspx</a></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hume</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="source" tagx="source" title="source">An Enquiry Concerning Human Understanding</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">United States</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Hackett Publishing Country</span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kant</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Critique of pure reason</span><span class="source" tagx="source" title="source">The Cambridge Edition of the Works of Immanuel Kant</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/CBO9780511804649">10.1017/CBO9780511804649</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Estimating spatial coupling in epidemiological systems: a mechanistic approach</span><span class="source" tagx="source" title="source">Ecology Letters</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">29</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1461-0248.2002.00268.x">10.1046/j.1461-0248.2002.00268.x</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Simulation suggests that rapid activation of social distancing can arrest epidemic development due to a novel strain of influenza</span><span class="source" tagx="source" title="source">BMC Public Health</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">117</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2458-9-117">10.1186/1471-2458-9-117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19400970">19400970</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kermack</span><span class="given-names" tagx="given-names" title="given-names">WO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKendrick</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="year" tagx="year" title="year">1927</span><span class="mixed-article-title" title="mixed-article-title">A contribution to the mathematical theory of epidemics</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society A: Mathematical, Physical and Engineering Sciences</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">700</span><span class="lpage" tagx="lpage" title="lpage">721</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspa.1927.0118">10.1098/rspa.1927.0118</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamb</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zebedee</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Influenza virus M2 protein is an integral membrane protein expressed on the infected-cell surface</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">627</span><span class="lpage" tagx="lpage" title="lpage">633</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(85)90211-9">10.1016/0092-8674(85)90211-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3882238">3882238</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">BY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooley</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potter</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wheaton</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voorhees</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stebbins</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grefenstette</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmer</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmerman</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Assi</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagener</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Simulating school closure strategies to mitigate an influenza epidemic</span><span class="source" tagx="source" title="source">Journal of Public Health Management and Practice</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">252</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/PHH.0b013e3181ce594e">10.1097/PHH.0b013e3181ce594e</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20035236">20035236</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senne</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Effect of vaccine use in the evolution of Mexican lineage H5N2 avian influenza virus</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">8372</span><span class="lpage" tagx="lpage" title="lpage">8381</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.78.15.8372-8381.2004">10.1128/JVI.78.15.8372-8381.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15254209">15254209</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowen</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mubareka</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palese</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">High temperature (30 degrees C) blocks aerosol but not contact transmission of influenza virus</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">5650</span><span class="lpage" tagx="lpage" title="lpage">5652</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00325-08">10.1128/JVI.00325-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18367530">18367530</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The multi-institution North American land data assimilation system (NLDAS): utilizing multiple gcip products and partners in a continental distributed hydrological modeling system</span><span class="source" tagx="source" title="source">Journal of Geophysical Research</span><span class="volume" tagx="volume" title="volume">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1029/2003JD003823">10.1029/2003JD003823</a></span></span></li> <li tag="ref"><a name="bib63" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mniszewski</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Valle</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stroud</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riese</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sydoriak</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Pandemic simulation of antivirals + school closures: buying time until strain-specific vaccine is available</span><span class="source" tagx="source" title="source">Computational and Mathematical Organization Theory</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">221</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10588-008-9027-1">10.1007/s10588-008-9027-1</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winship</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Counterfactuals and causal inference : methods and principles for social research</span><span class="source" tagx="source" title="source">Analytical Methods for Social Research</span><span class="edition" tagx="edition" title="edition">Second edition</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span></span></li> <li tag="ref"><a name="bib65" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Emergence and pandemic potential of swine-origin H1N1 influenza virus</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">459</span><span class="fpage" tagx="fpage" title="fpage">931</span><span class="lpage" tagx="lpage" title="lpage">939</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature08157">10.1038/nature08157</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19525932">19525932</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuzzo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Scientific method: statistical errors</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">506</span><span class="fpage" tagx="fpage" title="fpage">150</span><span class="lpage" tagx="lpage" title="lpage">152</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/506150a">10.1038/506150a</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522584">24522584</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Oregon State University</span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">PRISM climate group</span><a href="http://prism.oregonstate.edu">http://prism.oregonstate.edu</a><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">August, 2016</span></span></li> <li tag="ref"><a name="bib68" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papadimitriou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockwell</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faloutsos</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Adaptive, hands-off stream mining</span><span class="source" tagx="source" title="source">Proceedings 2003 VLDB Conference</span><span class="fpage" tagx="fpage" title="fpage">560</span><span class="lpage" tagx="lpage" title="lpage">571</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-012722442-8/50056-2">10.1016/B978-012722442-8/50056-2</a></span></span></li> <li tag="ref"><a name="bib69" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2009a</span><span class="mixed-article-title" title="mixed-article-title">Causal inference in statistics: An overview</span><span class="source" tagx="source" title="source">Statistics Surveys</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">96</span><span class="lpage" tagx="lpage" title="lpage">146</span><span class="pub-id"><a href="https://dx.doi.org/10.1214/09-SS057">10.1214/09-SS057</a></span></span></li> <li tag="ref"><a name="bib70" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2009b</span><span class="edition" tagx="edition" title="edition">2nd edition</span><span class="source" tagx="source" title="source">Causality: models, reasoning and inference</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/CBO9780511803161">10.1017/CBO9780511803161</a></span></span></li> <li tag="ref"><a name="bib71" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plummer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Penalized loss functions for Bayesian model comparison</span><span class="source" tagx="source" title="source">Biostatistics</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">523</span><span class="lpage" tagx="lpage" title="lpage">539</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/biostatistics/kxm049">10.1093/biostatistics/kxm049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18209015">18209015</a></span></span></li> <li tag="ref"><a name="bib72" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Large-scale spatial-transmission models of infectious disease</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">316</span><span class="fpage" tagx="fpage" title="fpage">1298</span><span class="lpage" tagx="lpage" title="lpage">1301</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1134695">10.1126/science.1134695</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540894">17540894</a></span></span></li> <li tag="ref"><a name="bib73" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizzo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunelli</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pugliese</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bella</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manfredi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomba</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iannelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciofi degli Atti</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degli Atti</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Scenarios of diffusion and control of an influenza pandemic in Italy</span><span class="source" tagx="source" title="source">Epidemiology and Infection</span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">1650</span><span class="lpage" tagx="lpage" title="lpage">1657</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S095026880800037X">10.1017/S095026880800037X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18272019">18272019</a></span></span></li> <li tag="ref"><a name="bib74" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nord</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Causality tests and functional form sensitivity</span><span class="source" tagx="source" title="source">Applied Economics</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">135</span><span class="lpage" tagx="lpage" title="lpage">141</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00036848500000011">10.1080/00036848500000011</a></span></span></li> <li tag="ref"><a name="bib75" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seth</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Principe</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">A test of granger non-causality based on nonparametric conditional independence</span><span class="conf-name" tagx="conf-name" title="conf-name">2010 20th International Conference on Pattern Recognition</span><span class="fpage" tagx="fpage" title="fpage">2620</span><span class="lpage" tagx="lpage" title="lpage">2623</span></span></li> <li tag="ref"><a name="bib76" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kohn</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Absolute humidity modulates influenza survival, transmission, and seasonality</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">3243</span><span class="lpage" tagx="lpage" title="lpage">3248</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0806852106">10.1073/pnas.0806852106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19204283">19204283</a></span></span></li> <li tag="ref"><a name="bib77" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pitzer</span><span class="given-names" tagx="given-names" title="given-names">VE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Absolute humidity and the seasonal onset of influenza in the continental United States</span><span class="source" tagx="source" title="source">PLoS Biology</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000316</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.1000316">10.1371/journal.pbio.1000316</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20186267">20186267</a></span></span></li> <li tag="ref"><a name="bib78" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsden</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hout</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1972</span><span class="mixed-article-title" title="mixed-article-title">General Social Survey Series</span><a href="http://www3.norc.org/GSS+Website/About+GSS/">http://www3.norc.org/GSS+Website/About+GSS/</a><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">August, 2016</span></span></li> <li tag="ref"><a name="bib79" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamerius</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom-Feshbach</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uejio</span><span class="given-names" tagx="given-names" title="given-names">CK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comrie</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Environmental predictors of seasonal influenza epidemics across temperate and tropical climates</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003194</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003194">10.1371/journal.ppat.1003194</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23505366">23505366</a></span></span></li> <li tag="ref"><a name="bib80" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">te Beest</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Boven</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooiveld</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Dool</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallinga</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Driving factors of influenza transmission in the Netherlands</span><span class="source" tagx="source" title="source">American Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">1469</span><span class="lpage" tagx="lpage" title="lpage">1477</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aje/kwt132">10.1093/aje/kwt132</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24029683">24029683</a></span></span></li> <li tag="ref"><a name="bib81" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: The United States Bureau of Transportation Statistics</span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Complete US airport network in 2010</span><a href="http://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=292">http://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=292</a><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">August, 2016</span></span></li> <li tag="ref"><a name="bib82" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: The United States Census</span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">US county adjacency</span><a href="https://www.census.gov/geo/reference/county-adjacency.html">https://www.census.gov/geo/reference/county-adjacency.html</a><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">August, 2016</span></span></li> <li tag="ref"><a name="bib83" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Treanor</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Influenza vaccine--outmaneuvering antigenic shift and drift</span><span class="source" tagx="source" title="source">New England Journal of Medicine</span><span class="volume" tagx="volume" title="volume">350</span><span class="fpage" tagx="fpage" title="fpage">218</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMp038238">10.1056/NEJMp038238</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14724300">14724300</a></span></span></li> <li tag="ref"><a name="bib84" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Linde</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A Bayesian view of model complexity</span><span class="source" tagx="source" title="source">Statistica Neerlandica</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">253</span><span class="lpage" tagx="lpage" title="lpage">271</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1467-9574.2011.00518.x">10.1111/j.1467-9574.2011.00518.x</a></span></span></li> <li tag="ref"><a name="bib85" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjørnstad</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Synchrony, waves, and spatial hierarchies in the spread of influenza</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">312</span><span class="fpage" tagx="fpage" title="fpage">447</span><span class="lpage" tagx="lpage" title="lpage">451</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1125237">10.1126/science.1125237</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16574822">16574822</a></span></span></li> <li tag="ref"><a name="bib86" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charu</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballesteros</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Demonstrating the use of high-volume electronic medical claims data to monitor local and regional influenza activity in the US</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e102429</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0102429">10.1371/journal.pone.0102429</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25072598">25072598</a></span></span></li> <li tag="ref"><a name="bib87" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vynnycky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edmunds</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Analyses of the 1957 (Asian) influenza pandemic in the United Kingdom and the impact of school closures</span><span class="source" tagx="source" title="source">Epidemiology and Infection</span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">166</span><span class="lpage" tagx="lpage" title="lpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268807008369">10.1017/S0950268807008369</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17445311">17445311</a></span></span></li> <li tag="ref"><a name="bib88" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webby</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Emergence of influenza A viruses</span><span class="source" tagx="source" title="source">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</span><span class="volume" tagx="volume" title="volume">356</span><span class="fpage" tagx="fpage" title="fpage">1817</span><span class="lpage" tagx="lpage" title="lpage">1828</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2001.0997">10.1098/rstb.2001.0997</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11779380">11779380</a></span></span></li> <li tag="ref"><a name="bib89" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ettema</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Analysis of CDC social control measures using an agent-based simulation of an influenza epidemic in a city</span><span class="source" tagx="source" title="source">BMC Infectious Diseases</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">199</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-11-199">10.1186/1471-2334-11-199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21767379">21767379</a></span></span></li> <li tag="ref"><a name="bib90" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasuda</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshizawa</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kimura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shigematsu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumoto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawachi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oshima</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Preparedness for the spread of influenza: prohibition of traffic, school closure, and vaccination of children in the commuter towns of Tokyo</span><span class="source" tagx="source" title="source">Journal of Urban Health</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">619</span><span class="lpage" tagx="lpage" title="lpage">635</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11524-008-9264-0">10.1007/s11524-008-9264-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18449643">18449643</a></span></span></li> <li tag="ref"><a name="bib91" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasuda</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshizawa</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Modeling on social spread from immunity</span><span class="source" tagx="source" title="source">Japanese Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">58</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">S14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16377856">16377856</a></span></span></li> <li tag="ref"><a name="bib92" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwoh</span><span class="given-names" tagx="given-names" title="given-names">CK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lees</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hung</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Evaluating temporal factors in combined interventions of workforce shift and school closure for mitigating the spread of influenza</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e32203</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0032203">10.1371/journal.pone.0032203</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22403634">22403634</a></span></span></li> <li tag="ref"><a name="bib93" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aevermann</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauphin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macken</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahaffey</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reardon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suloway</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walters</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaremba</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zmasek</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klem</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheuermann</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Influenza research database: an integrated bioinformatics resource for influenza virus research</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">D466</span><span class="lpage" tagx="lpage" title="lpage">D474</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw857">10.1093/nar/gkw857</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27679478">27679478</a></span></span></li> </ul> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Appendix 1</div> <div class="appendix" title="appendix"> <div class="title" tagx="title" title="title">
Non-parametric Granger Causal Inference</div> <div class="boxed-text" tagx="boxed-text" title="boxed-text"><a href="https://dx.doi.org/10.7554/eLife.30756.032">10.7554/eLife.30756.032</a><div class="standardimplementationsofgrangercausalinference" title="sec"> <div class="title" tagx="title" title="title">
Standard Implementations of Granger Causal Inference</div> <p>Suppose that one is interested in the possibility that a vector series <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> causes another vector <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div></div></span> be an information set available at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
n</div></div></span>, consisting of terms of the vector series <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>, that is, </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(11)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
≦</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div></div></div> <p /> <p><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div></div></span> is said to be a proper information set with respect to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>, if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> is included within <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. Further, suppose that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> does not include any component of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>, and define </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(12)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Z</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
≦</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div></div></div> <p /> <p>Inferring causality in the mean is easier, and if one is satisfied with using minimum mean square prediction error as the criterion to evaluate incremental predictive power, then one may use linear, one-step-ahead least squares predictors to obtain an operational procedure: If <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the variance of one-step forecast error of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> given <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div></div></span>, then <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div></div></span> is a prima facie cause of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div></div></span> with respect to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
′</div> </div></div></span> if: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(13)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
|</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
J</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> <p /> <p>Testing for bivariate Granger causality <i>in the mean</i> involves estimating a linear, reduced-form vector autoregression: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(14)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
U</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(15)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> are one-sided lag polynomials in the lag operator <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div></div></span> with roots all-distinct, and outside the unit circle. The regression errors <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
U</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div></div></span> are assumed to be mutually independent and individually i.i.d. with zero mean and constant variance. We used a standard joint test (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div></div></span> or <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
χ</div> <div class="mn_UNKNOWN">
2</div> </div></div></span>-test) to determine whether lagged <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div></div></span> has significant linear predictive power for current <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div></div></span>.<p /> <p>The null hypothesis that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div></div></span> does not strictly Granger cause <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div></div></span> is rejected if the coefficients of the elements in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> are jointly significantly different from zero.</p> <p>Linear tests presuppose restrictive and often unrealistic (<a href="#bib30">Darnell and Evans, 1990</a>; <a href="#bib38">Epstein, 1987</a>) structure on data. Brock (<a href="#bib15">Brock, 1991</a>) presents a simple bivariate model to analytically demonstrate the limitations of linear tests in uncovering nonlinear influence. To address this issue, a number of nonlinear tests have been suggested, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, with generalized, autoregressive, conditional heteroskedasticity (GARCH) models (<a href="#bib6">Asimakopoulos et al., 2000</a>), using wavelet transforms (<a href="#bib68">Papadimitriou et al., 2003</a>), or heuristic, additive relationships (<a href="#bib25">Chu et al., 2005</a>). However, these approaches often assume a class of allowable non-linearities; thus not quite alleviating the problem of presupposed structure. This is not just an academic issue; Granger causality has been shown to be significantly sensitive to non-linear transformations (<a href="#bib74">Roberts and Nord, 1985</a>).</p> <p>Non-parametric approaches, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span> the Hiemstra-Jones (HJ) test (<a href="#bib50">Hiemstra and Jones, 1994</a>) on the other hand, attempt to completely dispense with presuppositions regarding causality structure. Given two series, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>, the HJ test (which is a modification of the Baek-Brock test [<a href="#bib7">Baek and Brock, 1992</a>]) uses correlation integrals to test if the probability of similar futures for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> given similar pasts, change significantly if we condition instead on similar pasts for both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> simultaneously.</p> <p>Nevertheless, in order to achieve consistent estimation of the correlation integrals, the data series are required to be ergodic, stationary, and absolutely regular <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div> <div class="mi_UNKNOWN">
β</div></div></span>-mixing, with an upper bound on the rate at which the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
β</div></div></span>-coefficients approach zero (<a href="#bib34">Denker and Keller, 1983</a>). The additional assumptions beyond ergodicity and stationarity serve to guarantee that sufficiently separated fragments of the data series are nearly independent. The HJ test and its variants (<a href="#bib35">Diks and Panchenko, 2006</a>; <a href="#bib75">Seth and Principe, 2010</a>) have been quite successful in econometrics; uncovering nonlinear causal relations between money and income (<a href="#bib7">Baek and Brock, 1992</a>), aggregate stock returns and macroeconomic factors (<a href="#bib51">Hiemstra and Kramer, 1995</a>), currency future returns (<a href="#bib6">Asimakopoulos et al., 2000</a>) and stock price and trading volume (<a href="#bib50">Hiemstra and Jones, 1994</a>). Surprisingly, despite clear evidence that linear tests typically have low power in uncovering nonlinear causation (<a href="#bib50">Hiemstra and Jones, 1994</a>; <a href="#bib6">Asimakopoulos et al., 2000</a>), the application of non-parametric tests has been limited in areas beyond financial and macroeconomic interests.</p> <div class="1.1ourapproach:non-parametricapproachbasedonprobabilisticautomata" title="sec"> <div class="title" tagx="title" title="title">
1.1 Our Approach: Non-parametric Approach Based on Probabilistic Automata</div> <p>We use a new, non-parametric Granger causality-based test for quantized processes (<a href="#bib22">Chattopadhyay, 2014</a>).</p> <p>Going beyond binary hypothesis testing, we quantify the notion of the degree of causal influence between observed data streams, without presupposing any particular model structure. Generative models of causal influence are inferred with no a priori, imposed dynamical structure beyond ergodicity, stationarity, and a form of weak dependence. The explicit generative models may be used for prediction. The proposed inference algorithms are PAC-efficient (<a href="#bib22">Chattopadhyay, 2014</a>), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div></div></span>, we are guaranteed to find good models with high probability, and with small sample complexity.</p> </div> </div> <div class="modelingframework:probabilisticautomataandcrossedprobabilisticautomata" title="sec"> <div class="title" tagx="title" title="title">
Modeling Framework: Probabilistic Automata and Crossed Probabilistic Automata</div> <p>In this section, we briefly describe the notion of probabilistic automata, the inference problem, the notion of causal states, and the decision fusion strategy for multiple models.</p> <div class="selfmodels" title="sec"> <div class="title" tagx="title" title="title">
Self Models</div> <p>The event catalogue’s quantized data streams may be viewed as sample paths from hidden, quantized stochastic processes that drive the observed dynamics. A self-model for such a stream is a generative model that captures statistically significant symbol patterns that causally determine (in a probabilistic sense) future symbols. A good modeling framework for such self-models are probabilistic finite state automata (PFSA) (<a href="#bib20">Chattopadhyay and Lipson, 2013</a>; <a href="#bib22">Chattopadhyay, 2014</a>).</p> <p>One such self model, shown in <a href="#fig1">Figure 1</a> (Plate A(i)), is an example of a simple PFSA with two states generated from a binary symbol sequence. Note that, for this particular example, a sequence <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
11</div></div></span> localizes or synchronizes the machine to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
2</div> </div></div></span>, which then implies that the next symbol is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span> with a probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.1</div></div></span> and a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span> with a probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.9</div></div></span>. Thus, given this stream’s self-model, along with the short history <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
11</div></div></span>, we can predict the symbol distribution in the next time step. Notably, the states in a PFSA are not the alphabet symbols themselves, but are equivalence classes of symbol sequences (histories) that lead to statistically equivalent futures. This particular example has two states–not because of the fact that the alphabet is binary, but because there are only two distinct distributions that dictate the next symbol from any point in the dynamical evolution. The symbol value is distributed as either <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
0.1 0.9</div> <div class="mo_UNKNOWN">
]</div> </div></div></span> or <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
0.3 0.7</div> <div class="mo_UNKNOWN">
]</div> </div></div></span> over the binary event alphabet, and the current context or ‘state’ dictates which distribution is in effect chosen. These contexts are simply historical sequence equivalence classes that could have potentially lead to the present state; the specific history belonging to whichever current equivalence class actually transpired makes no difference in the future. Hence, these classes are dynamical states for the discrete stochastic evolution of the system at hand. The transition structure of the PFSA (represented in the graph as the labeled edges with probabilities) specifies how we move between these classes as new symbols are generated, or as new data is acquired.</p> <p>The inference problem here is to determine both the number of states and the transition structure given a sufficiently long, quantized data stream. In our unsupervised approach, there are no restrictions on admissible graphs for the inferred PFSA, and no a priori constraints on the number of states that may appear (<a href="#bib20">Chattopadhyay and Lipson, 2013</a>).</p> <p>Thus, the number of states inferred by our algorithm is a direct measure of the statistical complexity of the underlying process (<a href="#bib29">Crutchfield, 1994</a>). An inferred single-state, self model implies that the stream is a sequence of independently generated symbols (white noise or a Bernoulli process), and is therefore uninformative.</p> <p>The inference algorithm identifies the number of causal states by searching for distinct ‘contexts’–sequences which, once transpired, lead to a distinct probability distribution of the next symbol. The computation proceeds as follows (<a href="#bib20">Chattopadhyay and Lipson, 2013</a>):</p> <ol> <li> <p>Let the set of all possible sequences up to a length <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div></div></span> be denoted as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
L</div> </div></div></span>.</p> </li> <li> <p>Compute the probability of a future symbol (at a specified time shift) being <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> or <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> after a specific string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span> from the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
L</div> </div></div></span> is encountered; call this distribution <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
ω</div> </div></div></span> for the string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span>.</p> </li> <li> <p>Call the set of probability distributions obtained by this method as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Φ</div></div></span>.</p> </li> <li> <p>Find clusters in the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Φ</div></div></span>, such that individual clusters are separated by some pre-specified distance <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>. These clusters represent the causal states, as they are classes of histories (sequences) that lead to identical, immediate future.</p> </li> <li> <p>Suppose string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span> is in cluster <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
i</div> </div></div></span>, and sequence <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> is in cluster <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
j</div> </div></div></span>; it then follows that, in the inferred PFSA, there is a transition labeled <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> from the corresponding state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
j</div> </div></div></span>. Carrying out this procedure for each symbol of the alphabet for each inferred cluster or state identifies the complete transition structure of the model.</p> </li> <li> <p>Once the transition structure is identified, we choose an arbitrary initial state and step through the model as dictated by the input data stream. We count the number of times each edge or transition is traversed, and, by normalizing the count, we arrive at an estimate of the edge probabilities.</p> </li> </ol> <p>This completes the PFSA inference.</p> </div> <div class="formaldefinitionofpfsasandfutureprediction" title="sec"> <div class="title" tagx="title" title="title">
Formal Definition of PFSAs and Future Prediction</div> <p>Mathematically, a PFSA is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
4</div></div></span>-tuple <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Σ</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div></div></span> is the set of states, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div></div></span> is the symbol alphabet, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
Σ</div> </div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
Q</div> </div> </div></div></span> is the transition function specifying the graph structure such that for any state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div></div></span> is the end state of the transition from state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
q</div></div></span> via symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mo_UNKNOWN">
→</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div></div></span> is the symbol probability function such that for any state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span> is the probability of generating the symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Σ</div> </div></div></span> from the state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
q</div></div></span>, with the constraint: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(16)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∀</div> <div class="mi_UNKNOWN">
q</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Σ</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></div> <p /> <p>We specify a time shift <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> when we infer a PFSA <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></span> from a given input stream; this time shift is the delay with which the inferred model makes predictions. Specifically, each transition step in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></span> is translated to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> steps in the quantized stream. Thus, if we know that the current state in the model is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>, then the predicted symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> steps for the future are <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div></div></span> with probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.30756.033"><span class="label" tagx="label" title="label">Appendix 1—figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Intuitive Description of Self and Cross Models.</div> <p><i>Plate A</i> illustrates the notion of self-models. Historical data is first represented as a symbol sequence (denoted as ‘Data stream’ in Plate A) using space-time discretization and magnitude quantization. For example, we may use a spatial discretization of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
±</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
∘</div> </div> </div></div></span> in both latitudes and longitudes, a temporal discretization of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span> week, and a binary magnitude quantization that maps all magnitudes below <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
4.0</div></div></span> to symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span>, and all higher magnitudes to symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span>. This symbol stream then represents a sample path from a hidden, quantized stochastic process. A self-model is a generative model of this data stream, which captures symbol patterns that causally determine (in a probabilistic sense) future symbols. Specifically, our inferred self-model (see Plate A(i) for an example) is a probabilistic, finite state automata (PFSA). <i>Plate B</i> illustrates the notion of cross-models. Instead of inferring a model from a given stream to predict future symbols in the same stream, we now have two symbol streams (Data Stream I and Data Stream II), and the cross-model is essentially a generative model that attempts to predict symbols in one stream by reading historical data in another. Notably, as shown in Plate B(i), the cross-model is syntactically not exactly a PFSA (arcs have no probabilities in the cross-model, but each state has an output distribution). We call such models �?crossed probabilistic finite state automata,’ or XPFSA. Once these models are inferred, they may be used to predict the future evolution of the data streams. Thus, the self-model in Plate A may be initialized with its unique stationary distribution, after which a relatively short observed history would dictate the current distribution on the model states. This, in turn, would yield a distribution over the symbol alphabet in the next time step. For a cross-model, we would be able to obtain future symbol distribution in the second stream, given a short history in the first stream. Note that the cross-model from I<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
→</div></div></span> II is not necessarily the same as the cross-model in the other direction.</p>   </div> <p>Note that, if we have two models <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Σ</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Σ</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> with time shifts <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> respectively, it follows that with each step the models make predictions at different points in time in the future.</p> <p>Note that to make a prediction using a PFSA, we must know its current state. This is non-trivial in general due to the synchronization problem (<a href="#bib20">Chattopadhyay and Lipson, 2013</a>; <a href="#bib22">Chattopadhyay, 2014</a>). We find an approximation of the current state as follows:</p> <ol> <li> <p>Compute the he PFSA states’ stationary distribution using standard Markov chain tools.</p> </li> <li> <p>Treat the stationary distribution row vector <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div></div></span> as the initial state distribution. The rationale here is that if we assume that the model has been operating for a sufficient length of time, the current distribution must be very close to the stationary distribution in the expected sense.</p> </li> <li> <p>Compute the square matrices, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Γ</div> <div class="mi_UNKNOWN">
σ</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
|</div> </div> </div> </div> </div></div></span>, for each symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Σ</div> </div></div></span>, such that the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div> </div></div></span> element of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Γ</div> <div class="mi_UNKNOWN">
σ</div> </div></div></span>, denoted as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Γ</div> <div class="mi_UNKNOWN">
σ</div> </div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span>, which signifies the probability of going from state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
j</div> </div></div></span> via the symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div></div></span>.</p> </li> <li> <p>Use a relatively short history <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
⋯</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mi_UNKNOWN">
ℓ</div> </div> </div> </div></div></span> of past symbols (before the current time) to update the state distribution as follows:</p> </li> </ol> <p> </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(17)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?���?���?���?���?���?���?���?���?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Γ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
⋯</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
ℓ</div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
k</div> </div></div></span> is the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
h</div> </div> </div></div></span> entry of the short history <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
h</div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span> is the state distribution at step <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span>. It follows from ergodicity that the updated distribution <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mi_UNKNOWN">
ℓ</div> </div></div></span> is an estimate of the current state distrbution.<p /> <p>The predicted distribution over the symbol alphabet <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> steps in the future (if the time shift for the model is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span>), is then simply: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
predicted</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ℓ</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
ℓ</div> </div></div></span> is the current probability for state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
q</div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, the entry corresponding to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
q</div></div></span> in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mi_UNKNOWN">
ℓ</div> </div></div></span>.<p /> </div> <div class="crossmodels" title="sec"> <div class="title" tagx="title" title="title">
Cross Models</div> <p>Unlike a self model, which attempts to predict future symbols in the same stream, a cross model attempts to predict symbols in a target stream after reading a short history in the source stream. Plate B(i) in <a href="#fig1">Figure 1</a> illustrates a simple cross model with two states. Note that the model syntax is different from a PFSA; in particular, there are no probabilities on the arcs, though each state has a specified output distribution. Thus, if we see sequence <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
101101</div></div></span> in data stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> (See <a href="#fig1">Figure 1</a>, Plate B), then we are in state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> in the cross-model, and hence the next symbol in the data stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> can be predicted to be <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0</div></div></span> with a probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.7</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div></div></span> with a probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
0.3</div></div></span>. We call such models crossed Probabilistic Finite State Automata (XPFSA) (<a href="#bib22">Chattopadhyay, 2014</a>). A cross model state or an XPFSA has a slightly different interpretation from PFSA; while PFSA states are equivalence classes of histories that lead to identical futures in the same stream, the XPFSA are equivalence classes of histories in the source stream that lead to identical futures in the target stream (the future evolution in the target stream does not matter).</p> <p>XPFSA inference ˜ (<a href="#bib22">Chattopadhyay, 2014</a>) is similar in principle to PFSA inference. Here, we have two input data streams, the source stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> over the source alphabet <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, and the target stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> over the source alphabet <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>. The broad steps are as follows:</p> <ol> <li> <p>Let the set of all possible sequences up to a length <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div></div></span> in source stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> be denoted as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>.</p> </li> <li> <p>Compute the probability of a future symbol (at a specified time shift) being <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span> in the target stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> after a specific string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span> from the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> is encountered in the source stream; call this distribution <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
ω</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div></div></span> for the string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span>.</p> </li> <li> <p>Call the set of probability distributions obtained in this manner as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Φ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div></div></span>.</p> </li> <li> <p>Find clusters in the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Φ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div></div></span>, such that the clusters are separated by some pre-specified distance <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>. These clusters now represent causal states of the cross-dependence between the stream (from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>), because they are classes of histories (sequences) in the source stream that leads to identical, immediate futures in the target stream.</p> </li> <li> <p>Suppose string <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div></div></span> is in a cluster corresponding to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
i</div> </div></div></span>, and sequence <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> is in a cluster corresponding to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
j</div> </div></div></span>; it then follows that, in the inferred XPFSA, there is a transition labeled <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> from the corresponding state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
i</div> </div></div></span> to state <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
j</div> </div></div></span>. Carrying out this procedure for each symbol of the source alphabet for each inferred cluster or state identifies the complete transition structure of the cross model. Note that the cross-model transitions are only labelled with symbols from the source alphabet.</p> </li> <li> <p>Once we have identified the transition structure, we choose an arbitrary initial state in the XPFSA and step through the model as dictated by the source data stream. Each time we reach a particular state in the cross-model, we note which symbol from target alphabet <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> transpires in the target stream (at the specified time shift). Once we reach the end of the input source stream, we normalize the symbol count vectors corresponding to the cross-model states, and this determines the output distribution at each state. Note that each output distribution is a probability distribution over the target alphabet. This completes the XPFSA inference.</p> </li> </ol> <p>An XPFSA may be thought of as a direct generalization of a PFSA, and any PFSA can be represented as an XPFSA. To illustrate this, note that predicting future symbols in the same stream can be thought of as predicting symbols is a second, but identical, copy of the first stream.</p> <p>If all histories in a source stream are equivalent in this sense, then we have a single state XPFSA, which implies that the source cannot provide any new information on what is going to happen in the target stream based on its own history. Hence, that source lacks any causal influence on the target. Thus, as before, the complexity of the inferred models is directly related to the statistical complexity (<a href="#bib29">Crutchfield, 1994</a>) of the learnable, dynamical relationships underlying data streams themselves.</p> <p>XPFSAs are asymmetric in general; the model for how a source influences a target does not need to be identical when the roles played by the streams are reversed. However, two streams are statistically independent if and only if the XPFSAs in both directions are single state machines (<a href="#bib22">Chattopadhyay, 2014</a>).</p> <p>This leads us to the notion of the causality coefficient, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, from data stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> to data stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>. True to Granger’s notion of statistical causality, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> rigorously quantifies the amount of <i>additional</i> information we can obtain about the immediate future in stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> through observing stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>.</p> <p>Explicitly, the coefficient is defined as the ratio of the expected change in the entropy of the next-symbol distribution in stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> conditioned over observations in the stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> to the entropy of the next-symbol distribution in stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>, conditioned on the fact that no observations are made on stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>. We show that causality coefficient <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> takes values on the closed unit interval and that higher values indicate a stronger predictability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, therefore, a higher degree of causal influence. We have <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> if and only if the inferred machine has a single state, and streams <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span> are statistically independent if and only if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
B</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>.</p> <p>Thus, the interpretation of the causality coefficient which is central to our development is as follows:</p> <p><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.3</div> </div></div></span><i>(say) from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> means that we can acquire, on average, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> </div></div></span></i></p><div class="unknown" title="mpadded" tag="mpadded"><i> <div class="mn_UNKNOWN">
0.3</div> </i></div><i> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div>  of additional information about <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> from each bit read from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, over what is already available from the past history of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>.</i><p /> <p>This is very different from computing correlations, and it assumes no model structure a priori for the hidden dynamics driving the data streams.</p> <p>Importantly, it turns out that inferring self models is simply a special case of cross model inference, in which the target stream is simply a time-shifted version of the first stream.</p> </div> <div class="formaldefinitionofxpfsasandfutureprediction" title="sec"> <div class="title" tagx="title" title="title">
Formal Definition of XPFSAs and Future Prediction</div> <p>Formally, an XPFSA is also a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
4</div></div></span>-tuple <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Δ</div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div></div></span> is a set of states, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> is the input alphabet. The transition function <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
×</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
Q</div> </div> </div></div></span> is defined as before (in the case of the PFSA), but the symbol probability function is defined as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mo_UNKNOWN">
×</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div> <div class="mo_UNKNOWN">
→</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div></div></span>, with </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(18)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∀</div> <div class="mi_UNKNOWN">
q</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> is the target alphabet, which is distinct from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> in general. Note that the symbol distribution specified by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div></div></span> is over the target alphabet <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>.<p /> <p>As before, there is a time shift <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> associated with inferred XPFSA, such that each symbol transition in the model maps to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> steps in the target stream. If we know the current state of the XPFSA to be <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>, and we observe symbol <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div> </div></div></span> in the source stream, then the predicted symbol in the target stream (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span> steps in the future) is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span>, with a probability of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>.</p> <p>As before, determining the current state in the XPFSA is non-trivial. We estimate the current state distribution in a manner similar to that used for self models:</p> <ol> <li> <p>Note that the XPFSA graph does not have probabilities on it edges. However, the corresponding source stream’s self model does have symbol probabilities on its own edges, and it models the source stream. The problem is that the graph for the self model, and that of the cross model, might not be identical. We solve this problem using projective composition of probabilistic automata (<a href="#bib21">Chattopadhyay and Ray, 2008</a>). This operation takes the self model for the source stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, and projects it on the inferred XPFSA graph <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span>, and the result is a PFSA <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> </div></div></span></p><div class="unknown" title="mpadded" tag="mpadded"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mover_UNKNOWN"> <div class="mo_UNKNOWN">
⊗</div> <div class="mo_UNKNOWN">
→</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div>  with the same structure as that of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span>.<p /> </li> <li> <p>Once we obtain <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> </div></div></span></p><div class="unknown" title="mpadded" tag="mpadded"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mover_UNKNOWN"> <div class="mo_UNKNOWN">
⊗</div> <div class="mo_UNKNOWN">
→</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div> , we can estimate its current state as described in the case for self models (using observed symbols in the source stream).<p /> </li> <li> <p>Let the current state distribution be <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mi_UNKNOWN">
ℓ</div> </div></div></span>. Then, the predicted future symbol (at time shift <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div></div></span>) in the target stream is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span> with a probability of:</p> </li> </ol> <p> </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mtext_UNKNOWN">
�??��??��??��???�??��??��??��??��???</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
Q</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
℘</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mi_UNKNOWN">
q</div> <div class="mi_UNKNOWN">
ℓ</div> </div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
,</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mover_UNKNOWN"> <div class="mo_UNKNOWN">
⊗</div> <div class="mo_UNKNOWN">
→</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div></div></span> (note we are using the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mo_UNKNOWN">
~</div> </div></div></span> from the XPFSA, and not from the projected PFSA).<p /> </div> <div class="computationofthecoefficientofcausality" title="sec"> <div class="title" tagx="title" title="title">
Computation of the Coefficient of Causality</div> <p>Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span> be stationary ergodic processes over finite alphabets <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="mi_UNKNOWN">
B</div> </div> </div></div></span> respectively. Then, the causal dependence coefficient of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span> on <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, denoted as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, is formally defined as the ratio of the expected change in the entropy of the next symbol distribution in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>. This is due to observations in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, which show the entropy of the next symbol distribution in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div></div></span>. In the absence of data in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></span>, we have (<a href="#bib22">Chattopadhyay, 2014</a>): </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(19)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
�??�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
x</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
λ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ℋ</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div></div></div>where the entropy <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> of discrete probability distribution <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
u</div></div></span> is given by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span> is the empty string, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�??�</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
⋅</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the expectation operator over all possible sequences in the source stream. Once we infer the cross model, computing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
A</div> </div></div></span> is straightforward.<p /> </div> <div class="choiceofquantizationschemes" title="sec"> <div class="title" tagx="title" title="title">
Choice of Quantization Schemes</div> <p>The weekly time series of county-specific records for the number of reported influenza cases is quantized to a binary stream. If <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
k</div> </div></div></span> is the number of reported cases for a given fixed county at the week index <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span>, then the quantized binary stream <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mi_UNKNOWN">
k</div> </div></div></span> is obtained as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(20)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
if </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> <div class="mtd_UNKNOWN"> <div class="mtext_UNKNOWN">
otherwise</div> </div> </div> </div> <div class="mi_UNKNOWN" /> </div> </div></div></div> <p /> </div> </div> </div> </div> </div> </div> </div>  </body></html>