<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2017/12/13/185876.atom"/>
  <meta name="HW.pisa" content="biorxiv;185876v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Complemented palindrome small RNAs first discovered from SARS coronavirus"/>
  <meta name="DC.Identifier" content="10.1101/185876"/>
  <meta name="DC.Date" content="2017-12-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="In this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes."/>
  <meta name="DC.Contributor" content="Chang Liu"/>
  <meta name="DC.Contributor" content="Ze Chen"/>
  <meta name="DC.Contributor" content="Wenyuan Shen"/>
  <meta name="DC.Contributor" content="Deshui Yu"/>
  <meta name="DC.Contributor" content="Siyu Li"/>
  <meta name="DC.Contributor" content="Yue Hu"/>
  <meta name="DC.Contributor" content="Haishuo Ji"/>
  <meta name="DC.Contributor" content="Wenjun Bu"/>
  <meta name="DC.Contributor" content="Qingsong Wang"/>
  <meta name="DC.Contributor" content="Shan Gao"/>
  <meta name="article:published_time" content="2017-12-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Complemented palindrome small RNAs first discovered from SARS coronavirus"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;In this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2017/01/01"/>
  <meta name="citation_mjid" content="biorxiv;185876v2"/>
  <meta name="citation_id" content="185876v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/185876v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/185876v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/185876v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876.full.pdf"/>
  <meta name="citation_doi" content="10.1101/185876"/>
  <meta name="citation_num_pages" content="13"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="185876"/>
  <meta name="citation_author" content="Chang Liu"/>
  <meta name="citation_author_institution" content="Laboratory of Medical Molecular Virology, School of Medicine, Nankai University"/>
  <meta name="citation_author" content="Ze Chen"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology and Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Science"/>
  <meta name="citation_author_institution" content="Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonoses"/>
  <meta name="citation_author" content="Wenyuan Shen"/>
  <meta name="citation_author_institution" content="Laboratory of Medical Molecular Virology, School of Medicine, Nankai University"/>
  <meta name="citation_author" content="Deshui Yu"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Siyu Li"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Yue Hu"/>
  <meta name="citation_author_institution" content="Laboratory of Medical Molecular Virology, School of Medicine, Nankai University"/>
  <meta name="citation_author" content="Haishuo Ji"/>
  <meta name="citation_author_institution" content="Institute of Statistics, Nankai University"/>
  <meta name="citation_author" content="Wenjun Bu"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Qingsong Wang"/>
  <meta name="citation_author_institution" content="Tianjin Research Center of Basic Medical Sciences and Department of Cell Biology, Tianjin Medical University"/>
  <meta name="citation_author_email" content="gao_shan@mail.nankai.edu.cn"/>
  <meta name="citation_author" content="Shan Gao"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author_institution" content="Institute of Statistics, Nankai University"/>
  <meta name="citation_author_email" content="gao_shan@mail.nankai.edu.cn"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8919-1338"/>
  <meta name="citation_reference" content="Chen, Z., Sun, Y., Yang, X., Wu, Z., Guo, K., Niu, X., Wang, Q., Ruan, J., Bu, W., and Gao, S., Two featured series of rRNA-derived RNA fragments (rRFs) constitute a novel class of small RNAs. Plos One, 2017. 12(4): e0176458."/>
  <meta name="citation_reference" content="Kreuze, J.F.; Perez, A.; Untiveros, M.; Quispe, D.; Fuentes, S.; Barker, I.; Simon, R. Complete viral genome sequence and discovery of novel viruses by deep sequencing of small RNAs: A generic method for diagnosis, discovery and sequencing of viruses. Virology 2009, 388, 1–7."/>
  <meta name="citation_reference" content="Li, R.; Gao, S.; Hernandez, A.G.; Wechter, W.P.; Fei, Z.; Ling, K. Deep sequencing of small RNAs in tomato for virus and viroid identification and strain differentiation. PLoS ONE 2012, 7, e37127."/>
  <meta name="citation_reference" content="Zheng, Y.; Gao, S.; Chellappan, P.; Li, R.; Marco, G.; Dina, G.; Segundo, F.; Ling, K.; Jan, K.; Fei, Z. VirusDetect: An automated pipeline for efficient virus discovery using deep sequencing of small RNAs. Virology 2017, 500, 130–138."/>
  <meta name="citation_reference" content="Nayak, A.; Tassetto, M.; Kunitomi, M.; Andino, R. RNA interference-mediated intrinsic antiviral immunity in invertebrates. Springer Berlin Heidelberg 2013, 371, 183–200."/>
  <meta name="citation_reference" content="Wang, F.; Sun, Y.; Ruan, J.; Chen, R.; Chen, X.; Chen, C.; Kreuze, J.F.; Fei, Z.; Zhu, X.; Gao, S. Using small RNA deep sequencing to detect human viruses. BioMed Res. Int. 2016, 2016, 2596782."/>
  <meta name="citation_reference" content="Niu, X., Sun, Y., Chen, Z., Li, R., Padmanabhan, C., Ruan, J., Kreuze, J.F., Ling, K., Fei, Z., and Gao, S., Using Small RNA-seq Data to Detect siRNA Duplexes Induced by Plant Viruses. Genes, 2017. 8(6): 163."/>
  <meta name="citation_reference" content="Roberts, A., Deming, D., Paddock, C.D., Cheng, A., Yount, B., Vogel, L., Herman, B.D., Sheahan, T., Heise, M., and Genrich, G.L., A mouse-adapted SARS-coronavirus causes disease and mortality in BALB/c mice. PLoS pathogens, 2007. 3(1): e5."/>
  <meta name="citation_reference" content="Chew D.S., Choi K.P., Heidner H., et al. Palindromes in SARS and Other Coronaviruses. Informs Journal on Computing, 2004, 16(4): 331."/>
  <meta name="citation_reference" content="Gao, S., Tian, X., Chang, H., Sun, Y., Wu, Z., Cheng, Z., Dong, P., Zhao, Q., Ruan, J., and Bu, W., Two novel lncRNAs discovered in human mitochondrial DNA using PacBio full-length transcriptome data. Mitochondrion 2017, 36:10.1016/j.mito.2017.08.002."/>
  <meta name="citation_reference" content="Elbashir, S.M.; Harborth, J.; Weber, K.; Tuschl, T. Analysis of gene function in somatic mammalian cells using small interfering RNAs. Methods 2002, 26, 199–213."/>
  <meta name="citation_reference" content="Srivastava, S.K. and Robins, H.S., Palindromic nucleotide analysis in human T cell receptor rearrangements. PLoS One, 2012. 7(12): e52250."/>
  <meta name="citation_reference" content="Enserink, M., Clues to the animal origins of SARS. Science, 2003. 300(5624): 1351–1351."/>
  <meta name="citation_reference" content="Li, W., Shi, Z., Yu, M., Ren, W., Smith, C., Epstein, J.H., Wang, H., Crameri, G., Hu, Z., and Zhang, H., Bats are natural reservoirs of SARS-like coronaviruses. Science, 2005. 310(5748): 676–679."/>
  <meta name="citation_reference" content="Zhang, M., Zhan, F., Sun, H., Gong, X., Fei, Z., and Gao, S. Fastq_clean: An optimized pipeline to clean the Illumina sequencing data with quality control. in Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on. 2014. IEEE."/>
  <meta name="citation_reference" content="Gao, S., Ou, J., and Xiao, K., R language and Bioconductor in bioinformatics applications(Chinese Edition). 2014, Tianjin: Tianjin Science and Technology Translation Publishing Co., Ltd."/>
  <meta name="twitter:title" content="Complemented palindrome small RNAs first discovered from SARS coronavirus"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="In this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes."/>
  <meta name="og-title" property="og:title" content="Complemented palindrome small RNAs first discovered from SARS coronavirus"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/185876v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="In this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2017-12-13"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Complemented palindrome small RNAs first discovered from SARS coronavirus | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-71429 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">In this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAG
     <span class="underline">CUUGUUAAAGA</span> from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Small RNA sequencing (small RNA-seq or sRNA-seq) is used to acquire thousands of short RNA sequences with lengths of usually less than 50 bp. With sRNA-seq, many novel non-coding RNAs (ncRNAs) have been discovered. For example, two featured series of rRNA-derived RNA fragments (rRFs) constitute a novel class of small RNAs [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Small RNA-seq has also been used for virus detection in plants [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>] and invertebrates [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. In 2016, Wang 
     <em>et al.</em> first used sRNA-seq data from the NCBI SRA database to prove that sRNA-seq can be used to detect and identify human viruses [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], but the detection results were not as good as those of plant or invertebrate viruses. To improve virus detection in mammals, our strategy was to detect and compare featured RNA fragments in plants, invertebrates and mammals using sRNA-seq data. In one previous study [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], we detected siRNA duplexes induced by plant viruses and analyzed these siRNA duplexes as an important class of featured RNA fragments. In this study, we detected siRNA duplexes induced by invertebrate and mammal viruses and unexpectedly discovered another important class of featured RNA fragments, which were complemented palindrome small RNAs (cpsRNAs). Among all the detected cpsRNAs, we found a typical 22-nt cpsRNA UCUUUAACAA
     <span class="underline">GCUUGUUAAAGA</span> from SARS coronavirus (SARS-CoV) strain MA15, which deserved further studies because mice infected with SARS-CoV MA15 died from an overwhelming viral infection with virally mediated destruction of pneumocytes and ciliated epithelial cells [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Although the palindromic motif TCTTTAACAAG
     <span class="underline">CTTGTTAAAGA</span> was already observed in a previous study [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], it never be considered to be transcribed as cpsRNAs before our studies.
    </p>
    <p id="p-5">The first discovered cpsRNA named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. In our previous study of mitochondrial genomes, we had reported for the first time a 20-nt palindrome small RNA (psRNA) named hsa-tiR-MDL1-20 [
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. The biological functions of hsa-tiR-MDL1-20 had been preliminarily studied in our previous study, while the biological functions of SARS-CoV-cpsR-22 were still unknown. In this study, we compared the features of siRNA duplexes induced by mammal viruses with those induced by plant and invertebrate viruses and found that siRNA duplexes induced by mammal viruses had significantly lower percentages of total sequenced reads and it seemed that they were only produced from a few sites on the virus genomes. One possible reason could be a large proportion of sRNA-seq data is from other small RNA fragments caused by the presence of a number of dsRNA-triggered nonspecific responses such as the type I interferon (IFN) synthesis [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Another possible reason could be the missing siRNA duplexes or siRNA fragments functions in cells by interaction with host RNAs or proteins. Based on this idea, we suspected that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. Then, we performed RNAi experiments to test the cellular effects induced by SARS-CoV-cpsR-22 and its segments.
    </p>
    <p id="p-6">In this study, we reported for the first time the existence of cpsRNAs. Further sequence analysis supported that SARS-CoV-cpsR-22 could originate from bat betacoronaviruses. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. This study aims to provide useful information for a better understanding of psRNAs and cpsRNAs, which constitute a novel class of small RNAs. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results and Discussion</h2>
    <div id="sec-3" class="subsection">
     <h3>Comparison of siRNA-duplexes induced by plant, invertebrate and mammal viruses</h3>
     <p id="p-7">In this study, 11 invertebrate viruses were detected using 51 runs of sRNA-seq data (
      <strong>Supplementary file 1</strong>) and two mammal viruses (H1N1 and SARS-CoV) were detected using 12 runs of sRNA-seq data. In our previous study, six mammal viruses were detected using 36 runs of sRNA-seq data [
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>]. The detection of siRNA-duplexes by 11 invertebrate and eight mammal viruses was performed using a published program in our previous study [
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]. Then, we compared the features of siRNA duplexes induced by invertebrate viruses (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">
       <strong>Figure 1A</strong>
      </a>) with those induced by plant viruses (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">
       <strong>Figure 1B</strong>
      </a>). The results showed that the duplex length was the principal factor to determine the read count in both plants and invertebrates. 21-nt siRNA duplexes were the most abundant duplexes in both plants and invertebrates, followed by 22-nt siRNA duplexes in plants but 20-nt siRNA duplexes in invertebrates. 21-nt siRNA duplexes with 2-nt overhangs were the most abundant 21-nt duplexes in plants, while 21-nt siRNA duplexes with 1-nt overhangs were the most abundant 21-nt duplexes in invertebrates but they had a very close read count to that of 21-nt siRNA duplexes with 2-nt overhangs. 18-nt, 19-nt, 20-nt and 22-nt siRNA duplexes in invertebrates had much higher percentages of total sequenced reads that those in plants. In addition, 18-nt and 19-nt siRNA duplexes had very close read counts and 20-nt and 22-nt siRNA duplexes had very close read counts in invertebrates. Since siRNA duplexes induced by mammal viruses had significantly lower percentages of total sequenced reads, the comparison of siRNA-duplex features between mammals and invertebrates/or plants could not provide meaningful results using the existing public data with standard sequencing depth. However, as an unexpected result from siRNA-duplex analysis, we discovered cpsRNAs from invertebrate and mammal viruses.
     </p>
     <div id="F1" class="fig pos-float odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of siRNA-duplexes induced by plant and invertebrate viruses All the cleaned sRNA-seq reads were aligned to viral genome sequences using the software bowtie v0.12.7 with one mismatch. The detection of siRNA duplexes was performed using the program duplexfinder [7]. A. The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from 11 invertebrate viral genomes. B. The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from seven plant viral genomes [7]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-329592131" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Comparison of siRNA-duplexes induced by plant and invertebrate viruses&lt;;/span&gt;; All the cleaned sRNA-seq reads were aligned to viral genome sequences using the software bowtie v0.12.7 with one mismatch. The detection of siRNA duplexes was performed using the program duplexfinder [7]. &lt;;strong&gt;;A.&lt;;/strong&gt;; The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from 11 invertebrate viral genomes. &lt;;strong&gt;;B.&lt;;/strong&gt;; The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from seven plant viral genomes [7].&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F1.medium.gif" width="440" height="188"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F1.medium.gif" width="440" height="188"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Comparison of siRNA-duplexes induced by plant and invertebrate viruses</span>
       <p id="p-8" class="first-child">All the cleaned sRNA-seq reads were aligned to viral genome sequences using the software bowtie v0.12.7 with one mismatch. The detection of siRNA duplexes was performed using the program duplexfinder [
        <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>]. 
        <strong>A.</strong> The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from 11 invertebrate viral genomes. 
        <strong>B.</strong> The read count of siRNA duplexes varies with the duplex length and the overhang length, using data from seven plant viral genomes [
        <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>].
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">One typical cpsRNA UCUUUAACAAG
      <span class="underline">CUUGUUAAAGA</span> (DQ497008: 25962-25983) located in the orf3b gene on the SARS-CoV strain MA15 genome was detected in four runs of sRNA-seq data (SRA: SRR452404, SRR452406, SRR452408 and SRR452410). This cpsRNA was named SARS-CoV-cpsR-22, which contained 22 nucleotides perfectly matching its reverse complementary sequence (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">
       <strong>Figure 2A</strong>
      </a>). We also detected one 18-nt and one 19-nt segment of SARS-CoV-cpsR-22, which could also be derived from siRNA duplexes (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">
       <strong>Figure 2B</strong>
      </a>) but their strands (positive or negative) could not be determined. Among SARS-CoV-cpsR-22 and its two segments, the 19-nt segment was the most abundant and the 22-nt SARS-CoV-cpsR-22 was the least abundant.
     </p>
     <div id="F2" class="fig pos-float odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Clues to origins of SARS-CoV-cpsR-22 All the genome sequences are represented by their GenBank accession numbers (e.g. DQ497008). A. 16-nt, 18-nt, 19-nt, 20-nt and 22-nt siRNA duplexes were used for RNAi experiments. *16-nt and 20-nt had not been detected in this study. B. All the homologous sequences at the 22-nt locus on civet betacoronavirus genomes were identical to SARS-CoV-cpsR-22, while one of four genotypes at the 22-nt locus on bat betacoronavirus genomes was identical to SARS-CoV-cpsR-22. C. The psRNA (heteropalindromic) hsa-tiR-MDL1 and cpsRNA SARS-CoV-cpsR-22 can form hairpins. D. The phylogenetic tree was built by the Neighbor Joining (NJ) method using the orf3b gene from human betacoronavirus, eight homologous sequences from bat betacoronavirus and two homologous sequences from civet betacoronavirus. The branch[[#x2019]]s length corresponds to an average number of nucleotide changes per 100 nucleotides." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-329592131" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Clues to origins of SARS-CoV-cpsR-22&lt;;/span&gt;; All the genome sequences are represented by their GenBank accession numbers (e.g. DQ497008). &lt;;strong&gt;;A.&lt;;/strong&gt;; 16-nt, 18-nt, 19-nt, 20-nt and 22-nt siRNA duplexes were used for RNAi experiments. *16-nt and 20-nt had not been detected in this study. &lt;;strong&gt;;B.&lt;;/strong&gt;; All the homologous sequences at the 22-nt locus on civet betacoronavirus genomes were identical to SARS-CoV-cpsR-22, while one of four genotypes at the 22-nt locus on bat betacoronavirus genomes was identical to SARS-CoV-cpsR-22. &lt;;strong&gt;;C.&lt;;/strong&gt;; The psRNA (heteropalindromic) hsa-tiR-MDL1 and cpsRNA SARS-CoV-cpsR-22 can form hairpins. &lt;;strong&gt;;D.&lt;;/strong&gt;; The phylogenetic tree was built by the Neighbor Joining (NJ) method using the orf3b gene from human betacoronavirus, eight homologous sequences from bat betacoronavirus and two homologous sequences from civet betacoronavirus. The branch[[#x2019]]s length corresponds to an average number of nucleotide changes per 100 nucleotides.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F2.medium.gif" width="440" height="275"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F2.medium.gif" width="440" height="275"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Clues to origins of SARS-CoV-cpsR-22</span>
       <p id="p-10" class="first-child">All the genome sequences are represented by their GenBank accession numbers (e.g. DQ497008). 
        <strong>A.</strong> 16-nt, 18-nt, 19-nt, 20-nt and 22-nt siRNA duplexes were used for RNAi experiments. *16-nt and 20-nt had not been detected in this study. 
        <strong>B.</strong> All the homologous sequences at the 22-nt locus on civet betacoronavirus genomes were identical to SARS-CoV-cpsR-22, while one of four genotypes at the 22-nt locus on bat betacoronavirus genomes was identical to SARS-CoV-cpsR-22. 
        <strong>C.</strong> The psRNA (heteropalindromic) hsa-tiR-MDL1 and cpsRNA SARS-CoV-cpsR-22 can form hairpins. 
        <strong>D.</strong> The phylogenetic tree was built by the Neighbor Joining (NJ) method using the orf3b gene from human betacoronavirus, eight homologous sequences from bat betacoronavirus and two homologous sequences from civet betacoronavirus. The branch’s length corresponds to an average number of nucleotide changes per 100 nucleotides.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Discovery of psRNAs and cpsRNAs</h3>
     <p id="p-11">Palindromic motifs are found in published genomes of most species and play important roles in biological processes. The well-known samples of palindromic DNA motifs include restriction enzyme sites, methylation sites and palindromic motifs in T cell receptors [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. In this study, we found that palindromic or complemented palindrome small RNAs motifs existed ubiquitously in animal virus genomes, but not all of them were detected to be transcribed and processed into cpsRNAs probably due to inadequate sequencing depth of the sRNA-seq data. For example, we only found two psRNAs (CUACUGAC
      <span class="underline">CAGUCAUC</span> and AAGGUCUCC
      <span class="underline">CUCUGGAA</span>) from 14 palindrome motifs and one cpsRNA SARS-CoV-cpsR-22 from 29 complemented palindrome motifs with size from 14 to 26 nt (
      <strong>Supplementary file 1</strong>) on the SARS-CoV genome (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ497008.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom">DQ497008.1</a>) using four runs of sRNA-seq data (
      <strong>Materials and Methods</strong>). A DNA palindromic motif is defined as a nucleic acid sequence which is reverse complimentary to itself, while small RNAs which are reverse complimentary to themselves are defined as cpsRNAs. Accordingly, the typical psRNA should have a sequence which is 100% identical to its reverse sequence, but most psRNAs are heteropalindromic or semipalindromic as hsa-tiR-MDL1-20 reported in our previous study [
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>]. The psRNA hsa-tiR-MDL1-20 AAAGACACCCCCCACAGUUU (NC_012920: 561-580) contains a 14-nt region (underlined) which reads the same backward as forward and a 3-nt flanking sequence at the 5′ end which is reverse complimentary to a 3-nt flanking sequence at the 3′ end. With these 3-nt flanking sequences, hsa-tiR-MDL1-20 can form a hairpin as cpsRNAs usually do (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">
       <strong>Figure 2A</strong>
      </a>). Although SARS-CoV-cpsR-22 is also typical, most of cpsRNAs have mismatches or InDels (Insertions/Deletions) in the reverse complimentary matches (
      <strong>Supplementary file 1</strong>). one example is a new Epstein-Barr Virus (EBV) microRNA precursor (pre-miRNA) with the length of 89 nt reported in our previous study [
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>]. This pre-miRNA sequence contains six mismatches and five insertions and only 87.64% (78/89) of the total nucleotides contributes to reverse complimentary matches.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Clues to origins of SARS-CoV-cpsR-22</h3>
     <p id="p-12">The previously unknown SARS virus generated widespread panic in 2002 and 2003 when it caused 774 deaths and more than 8000 cases of illness. Scientists immediately suspected that civet cats which were only distantly relatives to house cats may had been SARS-CoV’s springboard to human [
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Later, scientists concluded that civets were not the original source of SARS. Further investigation showed that genetic diversity of coronaviruses in bats increased the possibility of variants crossing the species barrier and causing outbreaks of diseases in human populations [
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. To investigate the origins of SARS-CoV-cpsR-22, we obtained coronavirus genome sequences associated to bats, palm civets, rats and mice, monkeys, dogs, bovines, hedgehogs, giraffes, waterbucks and equines from the NCBI GenBank database. The results of sequence analysis showed that SARS-CoV-cpsR-22 was only located in the orf3b gene encoded by the betacoronavirus genome. Then, we blasted the orf3b gene from human betacoronavirus ( GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ497008.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom">DQ497008.1</a>) to all the obtained betacoronavirus genomes, except those for experiments on mice and monkeys. The results showed that the orf3b gene from human betacoronavirus had homologous genes from betacoronavirus in bats and palm civets (
      <strong>Supplementary file 2</strong>) rather than in other species. SARS-CoV-cpsR-22 also had homologous sequences at a 22-nt locus on bat and civet betacoronavirus genomes. All the homologous sequences at the 22-nt locus on civet betacoronavirus genomes were identical to SARS-CoV-cpsR-22, while one of four genotypes at the 22-nt locus on bat betacoronavirus genomes was identical to SARS-CoV-cpsR-22 (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">
       <strong>Figure 2C</strong>
      </a>). Four genotypes at the 22-nt locus on bat betacoronavirus genomes had no, one, two and three mismatchs to SARS-CoV-cpsR-22 and their corresponding orf3b homologous sequences had identities of 96.77%, 96.13%, 87.96% and 85.16%. This suggested that one variant containing SARS-CoV-cpsR-22 could originate from betacoronavirus in bats, then be passed to palm civets and finally to human. This was consistent with the results of phylogenetic analysis using the orf3b homologous sequences from bat and civet betacoronavirus genomes (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">
       <strong>Figure 2D</strong>
      </a>). In the phylogenetic tree, all human and civet betacoronavirus containing SARS-CoV-cpsR-22 was grouped into one clade. The nearest relative of the human and civet clade was the bat betacoronavirus (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=JX993988.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom">JX993988.1</a>) containing SARS-CoV-cpsR-22 and the next nearest relative was the bat betacoronavirus (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ412042.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom">DQ412042.1</a>) containing 22-nt homologous sequence with one mismatch to SARS-CoV-cpsR-22.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Preliminary studies on biological functions of SARS-CoV-cpsR-22</h3>
     <p id="p-13">Our previous study showed that the psRNA hsa-tiR-MDL1-20 contained the Transcription Initiation Site (TIS) and the Transcription Termination Site (TTS) of the human mitochondrial H-strand and could involve in mtDNA transcription regulation [
      <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>]. This inspired us to speculate that cpsRNAs could have similar biological functions and we investigated SARS-CoV-cpsR-22 using RNA interference (RNAi). Then, SARS-CoV-cpsR-22 and its 16-nt, 18-nt, 19-nt and 20-nt segments were designed as siRNA duplexes and introduced into PC-9 cells by pSIREN-RetroQ plasmid transfection (
      <strong>Materials and Methods</strong>). As a result, the 19-nt and 20-nt segment significantly induced cell apoptosis to 2.76 and 1.48 times at 48 hours after their transfection, respectively. Particularly, the 19-nt segment significantly induced cell apoptosis to 7.94 (36.04/4.54) times at 72 hours after their transfection (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">
       <strong>Figure 3</strong>
      </a>). This corresponded to the 19-nt segment being the most abundant among all the segments of SARS-CoV-cpsR-22. Using the 19-nt segment, we also tested cell apoptosis in five other human cell lines and one mouse cell line. The results showed the 19-nt segment significantly induced cell apoptosis in the cell lines of A549, MCF-7 and H1299, but not in the cell lines of MB231, H520 and L929 (mouse). Since different cell types express specific genes, the 19-nt segment could silence cell-type specific transcripts to induce cell apoptosis through RNAi. These results suggested that the 19-nt segment had significant biological functions and could play a role in SARS-CoV infection or pathogenicity.
     </p>
     <div id="F3" class="fig pos-float odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV-cpsR-22 induced cell apoptosis PC-9 cells were divided into six groups named 22, 16, 18, 19, 20 and control for transfection using plasmids containing SARS-CoV-cpsR-22, its 16-nt, 18-nt, 19-nt, 20-nt segments. Each group had three replicate samples for plasmid transfection and cell apoptosis measurement. Each sample was processed following the same procedure. Cell apoptosis was measured at 72 h after transfection. The samples in this figure were selected randomly from their corresponding groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-329592131" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;SARS-CoV-cpsR-22 induced cell apoptosis&lt;;/span&gt;; PC-9 cells were divided into six groups named 22, 16, 18, 19, 20 and control for transfection using plasmids containing SARS-CoV-cpsR-22, its 16-nt, 18-nt, 19-nt, 20-nt segments. Each group had three replicate samples for plasmid transfection and cell apoptosis measurement. Each sample was processed following the same procedure. Cell apoptosis was measured at 72 h after transfection. The samples in this figure were selected randomly from their corresponding groups.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F3.medium.gif" width="421" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F3.medium.gif" width="421" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/13/185876/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">SARS-CoV-cpsR-22 induced cell apoptosis</span>
       <p id="p-14" class="first-child">PC-9 cells were divided into six groups named 22, 16, 18, 19, 20 and control for transfection using plasmids containing SARS-CoV-cpsR-22, its 16-nt, 18-nt, 19-nt, 20-nt segments. Each group had three replicate samples for plasmid transfection and cell apoptosis measurement. Each sample was processed following the same procedure. Cell apoptosis was measured at 72 h after transfection. The samples in this figure were selected randomly from their corresponding groups.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-8" class="subsection">
     <h3>Datasets and data analysis</h3>
     <p id="p-15">All sRNA-seq data were downloaded from the NCBI SRA database. The invertebrate and mammal viruses were detected from sRNA-seq data using VirusDetect [
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>] and their genome sequences were downloaed from the NCBI GenBank database. The description of sRNA-seq data and virus genomes is presented in 
      <strong>Supplementary file 1</strong>. The cleaning and quality control of sRNA-seq data were performed using the pipeline Fastq_clean [
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] that was optimized to clean the raw reads from Illumina platforms. Using the software bowtie v0.12.7 with one mismatch, we aligned all the cleaned sRNA-seq reads to viral genome sequences and obtained alignment results in SAM format for detection of siRNA duplexes using the program duplexfinder [
      <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>]. Statistical computation and plotting were performed using the software R v2.15.3 with the Bioconductor packages [
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. The orf3b gene from human betacoronavirus (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ497008.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom">DQ497008.1</a>), its 20 homologous sequences from bat betacoronavirus and nine homologous sequences from civet betacoronavirus were aligned using ClustalW2 (
      <strong>Supplementary file 2</strong>). After removal of identical sequences, the orf3b gene from human betacoronavirus, eight homologous sequences from bat betacoronavirus and two homologous sequences from civet betacoronavirus were used for phylogenetic analysis. Since these homologous sequences had high identities (from 85.16% to 99.78%) to the orf3b gene from DQ497008, the Neighbor Joining (NJ) method was used for phylogenetic analysis.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>RNAi and cellular experiments</h3>
     <p id="p-16">Based on the shRNA design protocol [
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>], the sequences of SARS-CoV-cpsR-22, its 16-nt, 18-nt, 19-nt 20-nt segments (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">
       <strong>Figure 2B</strong>
      </a>) and their control "CGTACGCGGAATACTTCGA" were selected to use as target sequences for pSIREN-RetroQ vector construction (Clontech, USA), respectively. PC-9 cells were divided into six groups named 22, 16, 18, 19, 20 and control for transfection using plasmids containing SARS-CoV-cpsR-22, its 16-nt, 18-nt, 19-nt, 20-nt segments and control sequences. Each group had three replicate samples for plasmid transfection and cell apoptosis measurement. Each sample was processed following the same procedure described below. At 12 h prior to transfection, the PC-9 cells were washed with PBS and trypsinized. Gbico RPMI-1640 medium was added into the cells, which were then centrifuged at 1000 rpm for 10 min at 4°C to remove the supernatant. Gbico RPMI-1640 medium (Thermo Fisher Scientific, USA) containing 10% fetal bovine serum was added to adjust the solvent to reach a volume of 2 μL and contain 2 × 10
      <sup>5</sup> cells. These cells were seeded in one well of a 6-well plate for plasmid transfection. The transfection of 2 μg plasmids was performed using 5 μL Lipofectamine 2000 (Life technology, USA) following the manufacturer’s instructions. Cell apoptosis was measured with FITC Annexin V Apoptosis Detection Kit I (BD Biosciences, USA) following the procedure described below. The cells were washed by PBS, trypsinized and collectd using a 5-mL culture tube at 48 h or 72 h after transfection. The culture tube was then centrifuged at 1000 rpm for 10 min at 4°C to remove the supernatant. The cells were washed twice with cold PBS and resuspended in 1X Binding Buffer at a concentration of 1 × 10
      <sup>6</sup> cells/mL. 100 μL of the solution (1 10
      <sup>5</sup> cells) was transfered to a new culture tube with 5 L of FITC-Annexin V and 5 L PI. The cells were gently vortexed and incubated for 15 min at room temperature in the dark. 400 μL of 1X Binding Buffer was added to the tube. Finally, this sample was analyzed using a FACSCalibur flow cytometer (BD Biosciences, USA) within 1 hour. Apoptotic cells were quantified by summing the count of early apoptotic cells (FITC-Annexin V+/PI-) and that of late apoptotic cells (FITC-Annexin V+/PI+).
     </p>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Funding</h2>
    <p id="p-18">This work was supported by grants from by Fundamental Research Funds for the Central Universities (for Nankai University) to Shan Gao, Central Public-Interest Scientific Institution Basal Research Fund of Lanzhou Veterinary Research Institute of CAAS to Ze Chen, and Natural Science Foundation of China (81472052) to Qingsong Wang.</p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Competing interests</h2>
    <p id="p-19">Non-financial competing interests are claimed in this study.</p>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Authors’ contributions</h2>
    <p id="p-20">SG conceived this project. SG and QW supervised this project. SG, ZC and ZW analyzed the data. YS and HC curated the sequences and prepared all the figures, tables and additional files. CL, WS and YH performed qPCR experiments. SG drafted the main manuscript. DY and WB revised the manuscript. 
     <strong>All authors have read and approved the manuscript</strong>.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-17">We thank Yangguang Han and Jinke Huang from Tianjin KeYiJiaXin Technology Co., Ltd for his professional guidance on the RNAi and cellular experiments.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-article-title">Two featured series of rRNA-derived RNA fragments (rRFs) constitute a novel class of small RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Plos One</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">4</span>): 
         <span class="cit-elocation-id">e0176458</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.2" data-doi="10.1016/j.virol.2009.03.024">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kreuze</span>, 
          <span class="cit-name-given-names">J.F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perez</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Untiveros</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Quispe</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fuentes</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Barker</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Simon</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Complete viral genome sequence and discovery of novel viruses by deep sequencing of small RNAs: A generic method for diagnosis, discovery and sequencing of viruses</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-pub-date">2009</span>, 
         <span class="cit-vol">388</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D388%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2009.03.024%26rft_id%253Dinfo%253Apmid%252F19394993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2009.03.024&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19394993&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-185876v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hernandez</span>, 
          <span class="cit-name-given-names">A.G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wechter</span>, 
          <span class="cit-name-given-names">W.P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fei</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ling</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-article-title">Deep sequencing of small RNAs in tomato for virus and viroid identification and strain differentiation</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">7</span>, 
         <span class="cit-elocation-id">e37127</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chellappan</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Marco</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dina</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Segundo</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ling</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jan</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fei</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-article-title">VirusDetect: An automated pipeline for efficient virus discovery using deep sequencing of small RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">500</span>, 
         <span class="cit-fpage">130</span>–
         <span class="cit-lpage">138</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D500%26rft.spage%253D130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nayak</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tassetto</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kunitomi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Andino</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">RNA interference-mediated intrinsic antiviral immunity in invertebrates</span>. 
         <abbr class="cit-jnl-abbrev">Springer Berlin Heidelberg</abbr>
         <span class="cit-pub-date">2013</span>, 
         <span class="cit-vol">371</span>, 
         <span class="cit-fpage">183</span>–
         <span class="cit-lpage">200</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSpringer%2BBerlin%2BHeidelberg%26rft.volume%253D371%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kreuze</span>, 
          <span class="cit-name-given-names">J.F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fei</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Using small RNA deep sequencing to detect human viruses</span>. 
         <abbr class="cit-jnl-abbrev">BioMed Res. Int.</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">2016</span>, 
         <span class="cit-fpage">2596782</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioMed%2BRes.%2BInt.%26rft.volume%253D2016%26rft.spage%253D2596782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Padmanabhan</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kreuze</span>, 
          <span class="cit-name-given-names">J.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ling</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fei</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-article-title">Using Small RNA-seq Data to Detect siRNA Duplexes Induced by Plant Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Genes</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">6</span>): 
         <span class="cit-fpage">163</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D8%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.8" data-doi="10.1371/journal.ppat.0030005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deming</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Paddock</span>, 
          <span class="cit-name-given-names">C.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yount</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vogel</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Herman</span>, 
          <span class="cit-name-given-names">B.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sheahan</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heise</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Genrich</span>, 
          <span class="cit-name-given-names">G.L.</span>
         </span>, 
         <span class="cit-article-title">A mouse-adapted SARS-coronavirus causes disease and mortality in BALB/c mice</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">e5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De5%26rft.epage%253De5%26rft.atitle%253DA%2Bmouse-adapted%2BSARS-coronavirus%2Bcauses%2Bdisease%2Band%2Bmortality%2Bin%2BBALB%252Fc%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030005%26rft_id%253Dinfo%253Apmid%252F17222058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17222058&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.9" data-doi="10.1287/ijoc.1040.0087">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chew</span>
          <span class="cit-name-given-names">D.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>
          <span class="cit-name-given-names">K.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heidner</span>
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Palindromes in SARS and Other Coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Informs Journal on Computing</abbr>, 
         <span class="cit-pub-date">2004</span>, 
         <span class="cit-vol">16</span>(
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">331</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DINFORMS%2BJournal%2Bon%2BComputing%26rft.stitle%253DINFORMS%2BJournal%2Bon%2BComputing%26rft.aulast%253DChew%26rft.auinit1%253DD.%2BS.%2BH.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D331%26rft.epage%253D340%26rft.atitle%253DPalindromes%2Bin%2BSARS%2Band%2BOther%2BCoronaviruses%26rft_id%253Dinfo%253Adoi%252F10.1287%252Fijoc.1040.0087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1287/ijoc.1040.0087&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.10" data-doi="10.1016/j.mito.2017.08.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dong</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Bu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-article-title">Two novel lncRNAs discovered in human mitochondrial DNA using PacBio full-length transcriptome data</span>. 
         <abbr class="cit-jnl-abbrev">Mitochondrion</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">36</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">:</span>
         <span class="cit-pub-id cit-pub-id-doi">
          <span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mito.2017.08.002
         </span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMitochondrion%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mito.2017.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.mito.2017.08.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.11" data-doi="10.1016/S1046-2023(02)00023-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Elbashir</span>, 
          <span class="cit-name-given-names">S.M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Harborth</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Weber</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tuschl</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Analysis of gene function in somatic mammalian cells using small interfering RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Methods</abbr>
         <span class="cit-pub-date">2002</span>, 
         <span class="cit-vol">26</span>, 
         <span class="cit-fpage">199</span>–
         <span class="cit-lpage">213</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DElbashir%26rft.auinit1%253DS.%2BM.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D213%26rft.atitle%253DAnalysis%2Bof%2Bgene%2Bfunction%2Bin%2Bsomatic%2Bmammalian%2Bcells%2Busing%2Bsmall%2Binterfering%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252802%252900023-3%26rft_id%253Dinfo%253Apmid%252F12054897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1046-2023(02)00023-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12054897&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F13%2F185876.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000175971400013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Srivastava</span>, 
          <span class="cit-name-given-names">S.K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Robins</span>, 
          <span class="cit-name-given-names">H.S.</span>
         </span>, 
         <span class="cit-article-title">Palindromic nucleotide analysis in human T cell receptor rearrangements</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">12</span>): 
         <span class="cit-elocation-id">e52250</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Enserink</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-article-title">Clues to the animal origins of SARS</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-vol">300</span>(
         <span class="cit-issue">5624</span>): 
         <span class="cit-fpage">1351</span>–
         <span class="cit-lpage">1351</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D300%26rft.spage%253D1351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-185876v2.14" data-doi="10.1126/science.1118391">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Epstein</span>, 
          <span class="cit-name-given-names">J.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crameri</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">310</span>(
         <span class="cit-issue">5748</span>): 
         <span class="cit-fpage">676</span>–
         <span class="cit-lpage">679</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8xMi8xMy8xODU4NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-185876v2.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gong</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fei</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Fastq_clean: An optimized pipeline to clean the Illumina sequencing data with quality control</span>. in 
         <span class="cit-source">Bioinformatics and Biomedicine (BIBM)</span>, 
         <span class="cit-conf-name">2014 IEEE International Conference on</span>. 
         <span class="cit-conf-date">2014</span>. IEEE.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-book" id="cit-185876v2.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ou</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-source">R language and Bioconductor in bioinformatics applications(Chinese Edition)</span>. 
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-publ-loc">Tianjin</span>: 
         <span class="cit-publ-name">Tianjin Science and Technology Translation Publishing Co., Ltd</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
