<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/embed/graphic-10.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/08/2020.02.04.933135.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.04.933135v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.04.933135"/>
  <meta name="DC.Date" content="2020-02-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al . (&lt;;https://doi.org/10.1002/jmv.25682&gt;;) and by Pradhan et al . (&lt;;https://doi.org/10.1101/2020.01.30.927871&gt;;), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses."/>
  <meta name="DC.Contributor" content="Chengxin Zhang"/>
  <meta name="DC.Contributor" content="Wei Zheng"/>
  <meta name="DC.Contributor" content="Xiaoqiang Huang"/>
  <meta name="DC.Contributor" content="Eric W. Bell"/>
  <meta name="DC.Contributor" content="Xiaogen Zhou"/>
  <meta name="DC.Contributor" content="Yang Zhang"/>
  <meta name="article:published_time" content="2020-02-08"/>
  <meta name="article:section" content="Contradictory Results"/>
  <meta name="citation_title" content="Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji &lt;;i&gt;;et al&lt;;/i&gt;;. (https://doi.org/10.1002/jmv.25682) and by Pradhan &lt;;i&gt;;et al&lt;;/i&gt;;. (https://doi.org/10.1101/2020.01.30.927871), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.04.933135v1"/>
  <meta name="citation_id" content="2020.02.04.933135v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.933135v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.933135v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.933135v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.04.933135"/>
  <meta name="citation_num_pages" content="11"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="Contradictory Results"/>
  <meta name="citation_firstpage" content="2020.02.04.933135"/>
  <meta name="citation_author" content="Chengxin Zhang"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7290-1324"/>
  <meta name="citation_author" content="Wei Zheng"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author" content="Xiaoqiang Huang"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author" content="Eric W. Bell"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author" content="Xiaogen Zhou"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author" content="Yang Zhang"/>
  <meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan"/>
  <meta name="citation_author_email" content="zhng@umich.edu"/>
  <meta name="citation_reference" content="Zhou, P.; Yang, X.-L.; Wang, X.-G.; Hu, B.; Zhang, L.; Zhang, W.; Si, H.-R.; Zhu, Y.; Li, B.; Huang, C.-L.; Chen, H.-D.; Chen, J.; Luo, Y.; Guo, H.; Jiang, R.-D.; Liu, M.-Q.; Chen, Y.; Shen, X.-R.; Wang, X.; Zheng, X.-S.; Zhao, K.; Chen, Q.-J.; Deng, F.; Liu, L.-L.; Yan, B.; Zhan, F.-X.; Wang, Y.-Y.; Xiao, G.-F.; Shi, Z.-L., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020."/>
  <meta name="citation_reference" content="Letko, M.; Munster, V., Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV. bioRxiv 2020, 2020.01.22.915660."/>
  <meta name="citation_reference" content="Pradhan, P.; Pandey, A. K.; Mishra, A.; Gupta, P.; Tripathi, P. K.; Menon, M. B.; Gomes, J.; Vivekanandan, P.; Kundu, B., Uncanny similarity of unique inserts in the 2019-nCoV spike protein to HIV-1 gp120 and Gag. bioRxiv 2020, 2020.01.30.927871."/>
  <meta name="citation_reference" content="Ji, W.; Wang, W.; Zhao, X.; Zai, J.; Li, X., Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost cross-species transmission from snake to human. Journal of Medical Virology 2020."/>
  <meta name="citation_reference" content="Zheng, W.; Li, Y.; Zhang, C. X.; Pearce, R.; Mortuza, S. M.; Zhang, Y., Deep-learning contact-map guided protein structure prediction in CASP13. Proteins 2019."/>
  <meta name="citation_reference" content="Zhang, Y.; Skolnick, J., Scoring function for automated assessment of protein structure template quality. Proteins 2004, 57 (4), 702–710."/>
  <meta name="citation_reference" content="Song, W.; Gui, M.; Wang, X.; Xiang, Y., Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog 2018, 14 (8), e1007236."/>
  <meta name="citation_reference" content="Zhou, X. G.; Hu, J.; Zhang, C. X.; Zhang, G. J.; Zhang, Y., Assembling multidomain protein structures through analogous global structural alignments. Proceedings of the National Academy of Sciences of the United States of America 2019, 116 (32), 15930–15938."/>
  <meta name="citation_reference" content="Altschul, S. F.; Madden, T. L.; Schäffer, A. A.; Zhang, J.; Zhang, Z.; Miller, W.; Lipman, D. J., Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Research 1997, 25 (17), 3389–3402."/>
  <meta name="citation_reference" content="Li, W.; Shi, Z.; Yu, M.; Ren, W.; Smith, C.; Epstein, J. H.; Wang, H.; Crameri, G.; Hu, Z.; Zhang, H.; Zhang, J.; McEachern, J.; Field, H.; Daszak, P.; Eaton, B. T.; Zhang, S.; Wang, L.-F., Bats Are Natural Reservoirs of SARS-Like Coronaviruses. Science 2005, 310 (5748), 676."/>
  <meta name="citation_reference" content="Wang, Q.; Qi, J.; Yuan, Y.; Xuan, Y.; Han, P.; Wan, Y.; Ji, W.; Li, Y.; Wu, Y.; Wang, J.; Iwamoto, A.; Woo, Patrick C. Y.; Yuen, K.-Y.; Yan, J.; Lu, G.; Gao, George F., Bat Origins of MERS-CoV Supported by Bat Coronavirus HKU4 Usage of Human Receptor CD26. Cell Host &amp;; Microbe 2014, 16 (3), 328–337."/>
  <meta name="citation_reference" content="Corman, V. M.; Baldwin, H. J.; Tateno, A. F.; Zerbinati, R. M.; Annan, A.; Owusu, M.; Nkrumah, E. E.; Maganga, G. D.; Oppong, S.; Adu-Sarkodie, Y.; Vallo, P.; da Silva Filho, L. V. R. F.; Leroy, E. M.; Thiel, V.; van der Hoek, L.; Poon, L. L. M.; Tschapka, M.; Drosten, C.; Drexler, J. F., Evidence for an Ancestral Association of Human Coronavirus 229E with Bats. Journal of Virology 2015, 89 (23), 11858."/>
  <meta name="citation_reference" content="Hu, B.; Zeng, L.-P.; Yang, X.-L.; Ge, X.-Y.; Zhang, W.; Li, B.; Xie, J.-Z.; Shen, X.-R.; Zhang, Y.-Z.; Wang, N.; Luo, D.-S.; Zheng, X.-S.; Wang, M.-N.; Daszak, P.; Wang, L.-F.; Cui, J.; Shi, Z.-L., Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLOS Pathogens 2017, 13 (11), e1006698."/>
  <meta name="citation_reference" content="GISAID, Phylogenetic tree of CoV families (larger context, including nearest bat precursor) (2020-02-03). https://platform.gisaid.org/epi3/frontend#lightbox983930542 2020."/>
  <meta name="citation_reference" content="Callaway, E.; Cyranoski, D., Why snakes probably aren’t spreading the new China virus. Nature 2020."/>
  <meta name="citation_reference" content="Meintjes, P. L.; Rodrigo, A. G., Evolution of relative synonymous codon usage in Human Immunodeficiency Virus type-1. Journal of bioinformatics and computational biology 2005, 3 (01), 157–168."/>
  <meta name="citation_reference" content="Edgar, R. C., MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Research 2004, 32 (5), 1792–1797."/>
  <meta name="citation_reference" content="Gouy, M.; Guindon, S.; Gascuel, O., SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building. Molecular Biology and Evolution 2009, 27 (2), 221–224."/>
  <meta name="citation_reference" content="Yang, J.; Yan, R.; Roy, A.; Xu, D.; Poisson, J.; Zhang, Y., The I-TASSER Suite: protein structure and function prediction. Nature Methods 2015, 12 (1), 7–8."/>
  <meta name="citation_reference" content="Li, Y.; Zhang, C.; Bell, E. W.; Yu, D. J.; Zhang, Y., Ensembling multiple raw coevolutionary features with deep residual neural networks for contact-map prediction in CASP13. Proteins 2019."/>
  <meta name="citation_reference" content="Alexaki, A.; Kames, J.; Holcomb, D. D.; Athey, J.; Santana-Quintero, L. V.; Lam, P. V. N.; Hamasaki-Katagiri, N.; Osipova, E.; Simonyan, V.; Bar, H., Codon and Codon-Pair Usage Tables (CoCoPUTs): facilitating genetic variation analyses and recombinant gene design. Journal of molecular biology 2019, 431 (13), 2434–2441."/>
  <meta name="citation_reference" content="Nakamura, Y.; Gojobori, T.; Ikemura, T., Codon usage tabulated from international DNA sequence databases: status for the year 2000. Nucleic Acids Research 2000, 28 (1), 292–292."/>
  <meta name="citation_reference" content="Peden, J. F., Analysis of codon usage. 2000."/>
  <meta name="citation_reference" content="Towns, J.; Cockerill, T.; Dahan, M.; Foster, I.; Gaither, K.; Grimshaw, A.; Hazlewood, V.; Lathrop, S.; Lifka, D.; Peterson, G. D.; Roskies, R.; Scott, J. R.; Wilkins-Diehr, N., XSEDE: Accelerating Scientific Discovery. Comput Sci Eng 2014, 16 (5), 62–74."/>
  <meta name="twitter:title" content="Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al . (&lt;;https://doi.org/10.1002/jmv.25682&gt;;) and by Pradhan et al . (&lt;;https://doi.org/10.1101/2020.01.30.927871&gt;;), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses."/>
  <meta name="og-title" property="og:title" content="Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.933135v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al . (&lt;;https://doi.org/10.1002/jmv.25682&gt;;) and by Pradhan et al . (&lt;;https://doi.org/10.1101/2020.01.30.927871&gt;;), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1141963 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-contradictory-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji 
     <em>et al</em>. (
     <a href="https://doi.org/10.1002/jmv.25682">https://doi.org/10.1002/jmv.25682</a>) and by Pradhan 
     <em>et al</em>. (
     <a href="https://doi.org/10.1101/2020.01.30.927871">https://doi.org/10.1101/2020.01.30.927871</a>), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The 2019 novel conronavirus, or 2019-nCoV, recently caused a major pneumonia outbreak in Wuhan, China and has spread to at least 20 countries in the world. As of February 3, 2020, there are more than 20,000 patients diagnosed with 2019-nCoV infection in China alone with more than 400 deaths. Understanding the viral infection mechanisms and animal hosts are of high urgency for the control and treatment of the 2019-nCoV virus. While it is now commonly recognized that bats such as 
     <em>Rhinolophus affinis</em> may serve as the natural reservoir of 2019-nCoV
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>, it is still unclear which animal serves as an intermediate host that brought the bat coronavirus to human hosts. During the 2019-nCoV’s infection of host cells, a critical virion protein is the Spike surface glycoprotein, also known as the S protein. Spike proteins constitute the outermost component in a coronavirus virion particle and are responsible for viral recognition of Angiotensin Converting Enzyme 2 (ACE2), a transmembrane receptor on mammalian hosts that facilitate viral entrance into host cells
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>–
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. Therefore, the spike protein largely determines host specificity and infectivity of a coronavirus. In this report, we re-analyzed two recent studies
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>, which have spurred numerous interests and discussions in the community and the society, regarding the sequence and structure of spike protein in the 2019-nCoV and the identification of its intermediate hosts.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>2019-nCoV Spike Protein does not Include Insertions Unique to HIV-1</h3>
     <p id="p-5">In a recent manuscript entitled “Uncanny similarity of unique inserts in the 2019-nCoV spike protein to HIV-1 gp120 and Gag” 
      <sup>
       <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
      </sup>, Pradhan 
      <em>et al</em>. presented a discovery of four novel insertions unique to 2019-nCoV spike protein (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). They further concluded that these four insertions are part of the receptor binding site of 2019-nCoV, and that these insertions shared “uncanny similarity” to Human Immunodeficiency Virus 1 (HIV-1) proteins but not to other coronaviruses. These claims have resulted in considerable controversy in the community. To investigate whether the conclusions by Pradhan 
      <em>et al</em>. are scientifically precise, we re-analyzed the structural location and sequence homology of the four spike protein insertions discussed therein.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequence alignment of 2019-nCoV spike protein (NCBI accession: QHD43416) and SARS-CoV spike protein (UniProt ID: P59594). The four [[#x201C]]novel[[#x201D]] insertions [[#x201C]]GTNGTKR[[#x201D]] (IS1), [[#x201C]]YYHKNNKS[[#x201D]] (IS2), [[#x201C]]GDSSSG[[#x201D]] (IS3) and [[#x201C]]QTNSPRRA[[#x201D]] (IS4) by Pradhan et al. are highlighted in dashed rectangles. We noted that these fragments are not bona fide [[#x201C]]insertions[[#x201D]]; in fact, at least three out of all four fragments are also shared with Bat Coronavirus RaTG13 spike glycoprotein (NCBI accession: QHR63300.1), as shown in Table 1[[#x2013]]3. Nevertheless, we still refer these fragments as [[#x201C]]insertions[[#x201D]] in this manuscript for consistency with the original report. The receptor binding domain of spike is marked within the solid box, which corresponds to residue positions 323 to 545 in the above alignment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377235563" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Sequence alignment of 2019-nCoV spike protein (NCBI accession: QHD43416) and SARS-CoV spike protein (UniProt ID: P59594). The four [[#x201C]]novel[[#x201D]] insertions [[#x201C]]GTNGTKR[[#x201D]] (IS1), [[#x201C]]YYHKNNKS[[#x201D]] (IS2), [[#x201C]]GDSSSG[[#x201D]] (IS3) and [[#x201C]]QTNSPRRA[[#x201D]] (IS4) by Pradhan &lt;;em&gt;;et al.&lt;;/em&gt;; are highlighted in dashed rectangles. We noted that these fragments are not &lt;;em&gt;;bona fide&lt;;/em&gt;; [[#x201C]]insertions[[#x201D]]; in fact, at least three out of all four fragments are also shared with Bat Coronavirus RaTG13 spike glycoprotein (NCBI accession: QHR63300.1), as shown in Table 1[[#x2013]]3. Nevertheless, we still refer these fragments as [[#x201C]]insertions[[#x201D]] in this manuscript for consistency with the original report. The receptor binding domain of spike is marked within the solid box, which corresponds to residue positions 323 to 545 in the above alignment.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F1.medium.gif" width="440" height="264"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F1.medium.gif" width="440" height="264"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-6" class="first-child">Sequence alignment of 2019-nCoV spike protein (NCBI accession: QHD43416) and SARS-CoV spike protein (UniProt ID: 
        <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=P59594&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom">P59594</a>). The four “novel” insertions “GTNGTKR” (IS1), “YYHKNNKS” (IS2), “GDSSSG” (IS3) and “QTNSPRRA” (IS4) by Pradhan 
        <em>et al.</em> are highlighted in dashed rectangles. We noted that these fragments are not 
        <em>bona fide</em> “insertions”; in fact, at least three out of all four fragments are also shared with Bat Coronavirus RaTG13 spike glycoprotein (NCBI accession: QHR63300.1), as shown in 
        <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>–
        <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>. Nevertheless, we still refer these fragments as “insertions” in this manuscript for consistency with the original report. The receptor binding domain of spike is marked within the solid box, which corresponds to residue positions 323 to 545 in the above alignment.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1143901/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1143901/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1143901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-7" class="first-child">BLAST search result for IS1. For 
        <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> to 
        <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">4</a>, if there are multiple redundant hits for the same gene from different strains of the same species removed, only one hit is shown. The residue non-identical to query is highlighted in bold. Sequence identity is calculated as the number of identical residues divided by query length. Only the sequence portion aligned to the query is shown. In this table, we also list the closest BLAST hit from bat coronavirus, which is known to be closely related to 2019-nCoV 
        <sup>
         <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>
        </sup>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1143906/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1143906/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1143906" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-8" class="first-child">BLAST search result for IS2.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1143904/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1143904/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1143904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3.</span>
       <p id="p-9" class="first-child">BLAST search result for IS3.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-10">Since the structure of the spike protein in 2019-nCoV is not yet available, we used C-I-TASSER
      <sup>
       <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
      </sup> to model its tertiary structure, as part of our efforts for full genome structure and function analyses of 2019-nCoV that are available at 
      <a href="https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/">https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/</a>. The C-I-TASSER model has a reasonable confidence score which corresponds to an estimated TM-score 
      <sup>
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
      </sup> of 0.61. The 2019-nCoV spike model was then assembled with the human ACE2 structure (PDB ID: 6ACJ)
      <sup>
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
      </sup> by DEMO
      <sup>
       <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
      </sup> to form a spike-ACE2 complex. As shown in 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, all four insertions are located outside the Receptor Binding Domain (RBD) of spike, in contrast to the original conclusion made by Pradhan 
      <em>et al</em>. which stated that the insertions are located on the interface with ACE2.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Complex structure model between human ACE2 (left yellow) and 2019-nCoV spike protein trimer (right, with three chains colored in magenta, cyan, and blue respectively) constructed by C-I-TASSER. The four insertions are shown as spheres. During different stages of coronavirus infection, the spike proteins may be post-processed (i.e. cleaved) to produce different isoforms. Therefore, the eventual spike complex might not include all residues of a full-length spike protein. Nevertheless, we construct the complex model using full-length spike sequence to illustrate the locations of the four insertions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377235563" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Complex structure model between human ACE2 (left yellow) and 2019-nCoV spike protein trimer (right, with three chains colored in magenta, cyan, and blue respectively) constructed by C-I-TASSER. The four insertions are shown as spheres. During different stages of coronavirus infection, the spike proteins may be post-processed (i.e. cleaved) to produce different isoforms. Therefore, the eventual spike complex might not include all residues of a full-length spike protein. Nevertheless, we construct the complex model using full-length spike sequence to illustrate the locations of the four insertions.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F2.medium.gif" width="440" height="265"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F2.medium.gif" width="440" height="265"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-11" class="first-child">Complex structure model between human ACE2 (left yellow) and 2019-nCoV spike protein trimer (right, with three chains colored in magenta, cyan, and blue respectively) constructed by C-I-TASSER. The four insertions are shown as spheres. During different stages of coronavirus infection, the spike proteins may be post-processed (i.e. cleaved) to produce different isoforms. Therefore, the eventual spike complex might not include all residues of a full-length spike protein. Nevertheless, we construct the complex model using full-length spike sequence to illustrate the locations of the four insertions.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-12">To investigate viral homologs of the four insertions, we further performed a BLAST sequence search of these four insertions against the non-redundant (NR) sequence database, restricting the search results to viruses (taxid:10239), but leaving other search parameters at default values. The choice of BLAST instead of the more sensitive PSI-BLAST algorithm
      <sup>
       <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
      </sup> is to emulate the original report by Pradhan 
      <em>et al</em>. which mainly aimed to identify near identical sequences. The top 5 sequence homologs (including the query itself) identified for each insertion are listed in 
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a>–
      <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">4</a>. In contrast to the previous claim that the four insertions are unique to 2019-nCoV and HIV-1, all four insertion fragments can be found in other viruses. In fact, an HIV-1 protein is among the top BLAST hits for only one of the four insertion fragments, while three of the four insertion fragments are found in bat coronavirus RaTG13. Moreover, partly due to the very short length of these insertions, which range from 6 to 8 amino acids, the E-value of the BLAST hits, which is a parameter used by BLAST for assessing the statistical significance of the alignments and usually needs to be below 0.01 to be considered as significant
      <sup>
       <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>
      </sup>, are all greater than 4, except for a bat coronavirus hit for IS2. These high E-values suggest that the majority of these similarities are likely to be coincidental.
     </p>
     <p id="p-13">Given that 3 out of the 4 insertion fragments are found in the bat coronavirus RaTG13, it is attempting to assume that these “insertions” may be directly inherited from bat coronaviruses. Currently, there are at least 7 known human coronaviruses (2019-nCoV, SARS-CoV, MERS-CoV, HCoV-229E, HCoV-OC43, HCoV-NL63 and HCoV-HKU1), where many of them, including Severe Acute Respiratory Syndrome-related Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome-related Coronavirus (MERS-CoV), were shown to be transmitted from bat
      <sup>
       <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </sup> (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). To further examine the evolutionary relationship between the 2019-nCoV genome and the bat coronavirus, in comparison with other human coronaviruses, in 
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> we created by MUSCLE a multiple sequence alignment (MSA) for all the 7 human coronaviruses and the bat coronavirus RaTG13 that is currently known to be the closest relative to 2019-nCoV
      <sup>
       <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
      </sup> among known coronaviruses. Among the 4 “insertions” of the 2019-nCoV, IS1 has only 1 residue different from the bat coronavirus, and 3 out of 7 residues are identical with MERS-CoV. IS2 and IS3 are all identical to the bat coronavirus. For IS4, although the local sequence alignment by BLAST did not hit the bat coronavirus in 
      <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, it has a close evolutionary relation with the bat coronavirus in the MSA. In particular, the first 6 residues in the IS4 fragment “QTQTNSPRRA” from 2019-nCoV are identical to the bat CoV, while the last 4 residues, which were absent in the bat coronavirus or SARS-CoV, have at least 50% identity to MERS-CoV and HCoV-HKU1.
     </p>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1143907/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1143907/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1143907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 4.</span>
       <p id="p-14" class="first-child">BLAST search result for IS4.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Proposed transmission history of representative coronaviruses (A) and multiple sequence alignment for the bat coronavirus and 7 known human coronaviruses (B). Bat CoV (QHR63300.1), 2019-nCoV (QHD43416.1), MERS-CoV (YP_009047204.1), HCoV-NL63 (YP_003767.1), HCoV-229E (NP_073551.1), SARS-CoV (P59594), HCoV-OC43 YP_009555241.1) and HCoV-HKU1 (YP_173238.1) are downloaded from the NCBI and UniProt databases. For brevity, only the regions near the four [[#x201C]]insertions[[#x201D]] are displayed in the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377235563" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Proposed transmission history of representative coronaviruses &lt;;strong&gt;;(A)&lt;;/strong&gt;; and multiple sequence alignment for the bat coronavirus and 7 known human coronaviruses &lt;;strong&gt;;(B)&lt;;/strong&gt;;. Bat CoV (QHR63300.1), 2019-nCoV (QHD43416.1), MERS-CoV (YP_009047204.1), HCoV-NL63 (YP_003767.1), HCoV-229E (NP_073551.1), SARS-CoV (P59594), HCoV-OC43 YP_009555241.1) and HCoV-HKU1 (YP_173238.1) are downloaded from the NCBI and UniProt databases. For brevity, only the regions near the four [[#x201C]]insertions[[#x201D]] are displayed in the figure.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F3.medium.gif" width="440" height="305"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F3.medium.gif" width="440" height="305"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-15" class="first-child">Proposed transmission history of representative coronaviruses 
        <strong>(A)</strong> and multiple sequence alignment for the bat coronavirus and 7 known human coronaviruses 
        <strong>(B)</strong>. Bat CoV (QHR63300.1), 2019-nCoV (QHD43416.1), MERS-CoV (YP_009047204.1), HCoV-NL63 (YP_003767.1), HCoV-229E (NP_073551.1), SARS-CoV (P59594), HCoV-OC43 YP_009555241.1) and HCoV-HKU1 (YP_173238.1) are downloaded from the NCBI and UniProt databases. For brevity, only the regions near the four “insertions” are displayed in the figure.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">Putting these together, we believe that there is a close evolutionary relation between 2019-nCoV and bat coronavirus RaTG13. The four insertions highlighted by Pradhan 
      <em>et al</em>. in the spike protein are not unique to 2019-nCoV and HIV-1. In fact, they are shared with many other viruses including the bat coronavirus. Yes, the similarities in the sequence-based alignments built on these very short fragments are statistically insignificant, as assessed by the BLAST E-values. Structurally, these “insertions” are far away from the binding interface of the spike protein with the ACE2 receptor, as shown in 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, which are also contradictory with the conclusion made by Pradhan 
      <em>et al</em>.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Codon Usage Analysis does not Suggest Snakes as Potential Hosts of 2019-nCoV</h3>
     <p id="p-17">Another early study attempting to understanding the infection of 2019-nCoV was performed by Ji 
      <em>et al</em>. 
      <sup>
       <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
      </sup>. In this study, the authors analyzed the Relative Synonymous Codon Usage (RSCU) of 2019-nCoV and 8 vertebrates, including two species of snakes (
      <em>Bungarus multicinctus</em> and 
      <em>Naja atra</em>), hedgehog (
      <em>Erinaceus europaeus</em>), bat (
      <em>Rhinolophus sinicus</em>), marmot (
      <em>Marmota</em>), pangolin (
      <em>manis javanica</em>), chicken (
      <em>Gallus gallus</em>), and human (
      <em>Homo sapiens</em>). Among these vertebrates, snakes have the smallest codon usage difference (squared Euclidiean distance of RSCU) from 2019-nCoV and were therefore proposed by Ji 
      <em>et al</em>. as the intermediate hosts of 2019-nCoV. This conclusion is controversial among virologists, mainly due to the lack of prior biological evidence that coronavirus can infect animals other than mammals and birds 
      <sup>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
      </sup>. In this report, we further scrutinize the bioinformatics approach and the underlying biological assumptions by a large-scale replication of the RSCU analysis.
     </p>
     <p id="p-18">The bioinformatics analysis performed in Ji 
      <em>et al</em>. study has several limitations. First, there are only approximately 300 protein coding sequences (CDSs) in the NCBI GenBank for the snake species they chose from 
      <em>Bungarus multicinctus</em> and 
      <em>Naja atra</em>. These CDSs represent &lt;;2% of all protein coding genes in a typical snake genome; the genome of King cobra (
      <em>Naja hannah</em>), for example, encodes 18,387 proteins according to UniProt (
      <a href="https://www.uniprot.org/proteomes/UP000018936">https://www.uniprot.org/proteomes/UP000018936</a>). The limited numbers of known CDSs in 
      <em>Bungarus multicinctus</em> and 
      <em>Naja atra</em> mean that the RSCU statistics may not reflect the actual RSCU distribution in the whole genome. Second, the Codon Usage Database used in the analysis of Ji 
      <em>et al</em>. has not been updated since 2007; a re-analysis using more recent codon usage database such as CoCoPUTs is needed. Third, only 8 vertebrates were analyzed in their study, while there are &gt;;100,000 vertebrates with at least one CDS in the NCBI GenBank database. Finally, there is no established evidence that viruses evolve their codon usage to resemble that of their animal hosts 
      <sup>
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
      </sup>; this calls for a careful benchmark of RSCU analysis in terms of its ability to re-discover known hosts of characterized viruses.
     </p>
     <p id="p-19">To address these issues, we re-implemented the RSCU comparison algorithm proposed by Ji 
      <em>et al</em>. to analyze the codon usage in the 2019-nCoV genome (NCBI accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom">MN908947.3</a>) and those of all 102,367 vertebrate species in the CoCoPUTS database. To test whether this kind of analysis can recover known hosts of well-studied coronavirus, severe acute respiratory syndrome-related coronavirus (SARS-CoV, NCBI accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_004718&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom">NC_004718</a>) and Middle East respiratory syndrome-related coronavirus (MERS-CoV, NCBI accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_019843&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom">NC_019843</a>) are also included. Codon usage frequency are converted to squared Euclidean distance of RSCU in two separate analyses: one based on all vertebrates (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A-C</a>) and the other on the subset of vertebrates with enough statistics, i.e. &gt;;2000 known CDSs (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4D-F</a>), roughly corresponding to 10% of all protein coding genes in a typical vertebrate genome.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Top 20 vertebrate species ranked in ascending order of squared Euclidean distance of RSCU to 2019-nCoV (A, D), SARS-CoV (B, E), and to MERS-CoV (C, F). The upper panels (A-C) are for all vertebrates with known codon usage in CoCoPUTS database, while the lower panel (D-F) is for the subset of species where more than 2000 CDSs are available for codon usage calculation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377235563" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Top 20 vertebrate species ranked in ascending order of squared Euclidean distance of RSCU to 2019-nCoV &lt;;strong&gt;;(A, D)&lt;;/strong&gt;;, SARS-CoV &lt;;strong&gt;;(B, E)&lt;;/strong&gt;;, and to MERS-CoV &lt;;strong&gt;;(C, F)&lt;;/strong&gt;;. The upper panels &lt;;strong&gt;;(A-C)&lt;;/strong&gt;; are for all vertebrates with known codon usage in CoCoPUTS database, while the lower panel (&lt;;strong&gt;;D-F&lt;;/strong&gt;;) is for the subset of species where more than 2000 CDSs are available for codon usage calculation.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F4.medium.gif" width="437" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F4.medium.gif" width="437" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/08/2020.02.04.933135/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-20" class="first-child">Top 20 vertebrate species ranked in ascending order of squared Euclidean distance of RSCU to 2019-nCoV 
        <strong>(A, D)</strong>, SARS-CoV 
        <strong>(B, E)</strong>, and to MERS-CoV 
        <strong>(C, F)</strong>. The upper panels 
        <strong>(A-C)</strong> are for all vertebrates with known codon usage in CoCoPUTS database, while the lower panel (
        <strong>D-F</strong>) is for the subset of species where more than 2000 CDSs are available for codon usage calculation.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">As shown in 
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, the top-ranking vertebrates with lowest RSCU distances to the three different coronavirus are very similar. For example, for all three viruses, two kinds of frogs (
      <em>Megophrys feae</em> and 
      <em>Liophryne schlaginhaufeni</em>) have the smallest RSCU distances among all vertebrates, while another two kinds of frogs (
      <em>Xenopus laevis</em> and 
      <em>Xenopus tropicalis</em>) have the smallest RSCU distances among all vertebrates with sufficient sequences. In fact, the RSCU distances between coronaviruses and frogs are consistently smaller than that between coronaviruses and the two snakes (
      <em>Bungarus multicinctus and Naja atra</em>) analyzed in Ji 
      <em>et al</em>., as shown in 
      <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.
     </p>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1143899/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1143899/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1143899" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 5.</span>
       <p id="p-22" class="first-child">Squared Euclidean among coronaviruses and representative vertebrates.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-23">
      <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> also showed that the RSCU analysis cannot identify the known natural, intermediate, and eventual hosts (
      <em>Rhinolophus sinicus</em>, 
      <em>Paguma larvata</em>, and 
      <em>Homo sapiens</em>) of SARS-CoV or the hosts (
      <em>Camelus dromedarius</em> and 
      <em>Homo sapiens</em>) of MERS-CoV, as the RSCU distances of these vertebrates are all larger than the cold blooded vertebrates (frogs and snakes). In fact, SARS-CoV and MERS-CoV, the two coronaviruses known to utilize different intermediate hosts (
      <em>Paguma larvata</em> and 
      <em>Camelus dromedarius</em>), have almost no difference in RSCU. These data suggest that the RSCU analysis proposed by Ji 
      <em>et al</em>. cannot provide conclusive evidence for the natural, intermediate or eventual hosts of coronavirus. Based on our current knowledge of existing zoonotic coronaviruses 
      <sup>
       <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
      </sup>, we conceive that, if 2019-nCoV requires an intermediate host, this host is more likely to be a warm-blooded vertebrate (a mammal or a bird), than snakes.
     </p>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Discussion and Conclusion</h2>
    <p id="p-24">Due to the scarcity of experimental and clinical data, as well as the urgency to understand the infectivity of the deadly coronaviruses, we have been increasingly relying on computational analyses to study the 2019-nCoV virus in terms of protein structures, functions, phylogeny, and interactions at both molecular and organismal levels. Indeed, within less than a month of the publication of the 2019-nCoV genome in January 2020, multiple bioinformatics analyses regarding 2019-nCoV have been either published or posted as preprint. While such expeditious analyses provide much needed insights into the biology of the 2019-nCoV virus, there is a caution to avoid over-interpretation of the data at the absence of comprehensive benchmarks or follow-up experimental validations. In this report, we have investigated two recently published computational analyses regarding intermediate host identification and the analysis of spike protein insertions. In both cases, we found that the conclusions proposed by the original studies do not hold in the face of more comprehensive replications of these analyses. We hope our analysis presented herein can clarify some of the misinterpretations resulted from previous bioinformatics analyses for the 2019-nCoV virus.</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Methods</h2>
    <div id="sec-7" class="subsection">
     <h3>Protein Sequence Alignment</h3>
     <p id="p-25">Global protein sequence alignment of the full-length coronavirus spike proteins was performed by MUSCLE 
      <sup>
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
      </sup> and visualized by SeaView 
      <sup>
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
      </sup>.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Structure Prediction of Spike-ACE2 complex</h3>
     <p id="p-26">We use C-I-TASSER 
      <sup>
       <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
      </sup> to create structural models of the full-length spike protein. Here, C-I-TASSER is an extended pipeline of I-TASSER
      <sup>
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
      </sup> and utilizes the deep convolutional neural-network based contact-maps 
      <sup>
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
      </sup> to guide the Monte Carlo fragment assembly simulations. Since the RBD domain of spike exhibits different conformations relative to the remaining portion of the protein, the DEMO pipeline 
      <sup>
       <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>
      </sup> was then used to re-assembly the domains and to construct a complex structure consisting of spike trimer and the extracellular domain of human ACE2, using the ACE2-bound conformation 2 of SARS-CoV spike glycoprotein (PDB ID: 6ACJ) as a template. Our complex modeling did not use the template originally used in the Pradhan 
      <em>et al</em>. study (PDB ID: 6ACD) because it did not include the ACE2 receptor.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Relative Synonymous Codon Usage (RSCU) Analysis</h3>
     <p id="p-27">As per the previous study 
      <sup>
       <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>
      </sup>, the RSCU for codon 
      <em>j</em> in a species is calculated as 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/08/2020.02.04.933135/embed/graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/08/2020.02.04.933135/embed/graphic-10.gif"/>
        </noscript>
       </span>
      </span> where 
      <em>k</em>
      <sub>
       <em>j</em>
      </sub> is the number of codons synonymous to codon 
      <em>j</em> (including 
      <em>j</em> itself), 
      <em>p</em>
      <sub>
       <em>j</em>
      </sub> is the probability of the respective amino acid being encoded by codon 
      <em>j</em> among all 
      <em>k</em>
      <sub>
       <em>j</em>
      </sub> synonymous codons in the CDSs of the whole genome. The difference of codon usage in two different species (a virus versus a vertebrate in our case) is defined by squared Euclidean distance of RSCU, i.e., 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/08/2020.02.04.933135/embed/graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/08/2020.02.04.933135/embed/graphic-11.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-28">Here, 
      <em>N</em>=61 is the number of codons that encodes amino acids, thereby excluding the 3 stop codons. 
      <em>X</em>
      <sub>
       <em>j</em>
      </sub> and 
      <em>X</em>
      <sub>
       <em>j</em>
      </sub>’ are the RSCU for codon 
      <em>j</em> in the virus and in the vertebrate, respectively. In our report, the codon usages of all vertebrates are taken from the CoCoPUTS 
      <sup>
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
      </sup> database, which was last updated in January 2020. This database was therefore much more recent than the Codon Usage Database 
      <sup>
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
      </sup> last updated in 2007 that was used in the previous research 
      <sup>
       <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>
      </sup>. To obtain the codon usage of coronaviruses, we imported the GenBank annotations of the three coronavirus genomes to SnapGene (GSL Biotech LLC] to export the codon usage table based on the GenBank annotations. CodonW 
      <sup>
       <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
      </sup> was not used for codon usage calculation as in the previous study because it cannot account for the −1 frameshift translation of the first open reading frame (ORF) in the coronavirus genome.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-29">This work is supported in part by the National Institute of General Medical Sciences (GM083107, GM116960), the National Institute of Allergy and Infectious Diseases (AI134678), and the National Science Foundation (DBI1564756, IIS1901191). This work used the Extreme Science and Engineering Discovery Environment (XSEDE) 
     <sup>
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
     </sup>, which is supported by National Science Foundation (ACI-1548562).
    </p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/">https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.-G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>, 
          <span class="cit-name-given-names">H.-R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">H.-D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">R.-D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">M.-Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>, 
          <span class="cit-name-given-names">X.-R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">X.-S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Q.-J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">L.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>, 
          <span class="cit-name-given-names">F.-X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.-Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">G.-F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.-L.</span>
         </span>, 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Letko</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-article-title">Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>, 2020.01.22.915660.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pradhan</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pandey</span>, 
          <span class="cit-name-given-names">A. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mishra</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tripathi</span>, 
          <span class="cit-name-given-names">P. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Menon</span>, 
          <span class="cit-name-given-names">M. B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gomes</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Vivekanandan</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kundu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-article-title">Uncanny similarity of unique inserts in the 2019-nCoV spike protein to HIV-1 gp120 and Gag</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>, 2020.01.30.927871.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ji</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zai</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-article-title">Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost cross-species transmission from snake to human</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medical Virology</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">C. X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pearce</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mortuza</span>, 
          <span class="cit-name-given-names">S. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">Deep-learning contact-map guided protein structure prediction in CASP13</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-pub-date">2019</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.6" data-doi="10.1002/prot.20264">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Skolnick</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-article-title">Scoring function for automated assessment of protein structure template quality</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-pub-date">2004</span>, 
         <span class="cit-vol">57</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">702</span>–
         <span class="cit-lpage">710</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D702%26rft.epage%253D710%26rft.atitle%253DScoring%2Bfunction%2Bfor%2Bautomated%2Bassessment%2Bof%2Bprotein%2Bstructure%2Btemplate%2Bquality.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.20264%26rft_id%253Dinfo%253Apmid%252F15476259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.20264&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15476259&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000225351100007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.7" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gui</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-pub-date">2018</span>, 
         <span class="cit-vol">14</span> (
         <span class="cit-issue">8</span>), 
         <span class="cit-fpage">e1007236</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.8" data-doi="10.1073/pnas.1905068116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">X. G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">C. X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">G. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">Assembling multidomain protein structures through analogous global structural alignments</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-pub-date">2019</span>, 
         <span class="cit-vol">116</span> (
         <span class="cit-issue">32</span>), 
         <span class="cit-fpage">15930</span>–
         <span class="cit-lpage">15938</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1905068116%26rft_id%253Dinfo%253Apmid%252F31341084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzMyLzE1OTMwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDgvMjAyMC4wMi4wNC45MzMxMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.9" data-doi="10.1093/nar/25.17.3389">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Altschul</span>, 
          <span class="cit-name-given-names">S. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Madden</span>, 
          <span class="cit-name-given-names">T. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schäffer</span>, 
          <span class="cit-name-given-names">A. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lipman</span>, 
          <span class="cit-name-given-names">D. J.</span>
         </span>, 
         <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-pub-date">1997</span>, 
         <span class="cit-vol">25</span> (
         <span class="cit-issue">17</span>), 
         <span class="cit-fpage">3389</span>–
         <span class="cit-lpage">3402</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9254694&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XU79300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.10" data-doi="10.1126/science.1118391">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Epstein</span>, 
          <span class="cit-name-given-names">J. H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Crameri</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">McEachern</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Field</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Daszak</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Eaton</span>, 
          <span class="cit-name-given-names">B. T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.-F.</span>
         </span>, 
         <span class="cit-article-title">Bats Are Natural Reservoirs of SARS-Like Coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-pub-date">2005</span>, 
         <span class="cit-vol">310</span> (
         <span class="cit-issue">5748</span>), 
         <span class="cit-fpage">676</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wOC8yMDIwLjAyLjA0LjkzMzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.933135v1.11" data-doi="10.1016/j.chom.2014.08.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Qi</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xuan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ji</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Iwamoto</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Woo</span>, 
          <span class="cit-name-given-names">Patrick C. Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuen</span>, 
          <span class="cit-name-given-names">K.-Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">George F.</span>
         </span>, 
         <span class="cit-article-title">Bat Origins of MERS-CoV Supported by Bat Coronavirus HKU4 Usage of Human Receptor CD26</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host &amp;; Microbe</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">16</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">328</span>–
         <span class="cit-lpage">337</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D16%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25211075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.chom.2014.08.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25211075&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.933135v1.12" data-doi="10.1128/JVI.01755-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Baldwin</span>, 
          <span class="cit-name-given-names">H. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tateno</span>, 
          <span class="cit-name-given-names">A. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zerbinati</span>, 
          <span class="cit-name-given-names">R. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Annan</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Owusu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Nkrumah</span>, 
          <span class="cit-name-given-names">E. E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Maganga</span>, 
          <span class="cit-name-given-names">G. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Oppong</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Adu-Sarkodie</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Vallo</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">da Silva Filho</span>, 
          <span class="cit-name-given-names">L. V. R. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Leroy</span>, 
          <span class="cit-name-given-names">E. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Thiel</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Hoek</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L. L. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tschapka</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Drexler</span>, 
          <span class="cit-name-given-names">J. F.</span>
         </span>, 
         <span class="cit-article-title">Evidence for an Ancestral Association of Human Coronavirus 229E with Bats</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">89</span> (
         <span class="cit-issue">23</span>), 
         <span class="cit-fpage">11858</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01755-15%26rft_id%253Dinfo%253Apmid%252F26378164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMy8xMTg1OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzA4LzIwMjAuMDIuMDQuOTMzMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.933135v1.13" data-doi="10.1371/journal.ppat.1006698">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>, 
          <span class="cit-name-given-names">L.-P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.-Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">J.-Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>, 
          <span class="cit-name-given-names">X.-R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.-Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">D.-S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">X.-S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.-N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Daszak</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.-F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cui</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.-L.</span>
         </span>, 
         <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">13</span> (
         <span class="cit-issue">11</span>), 
         <span class="cit-fpage">e1006698</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.04.933135v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-article-title">GISAID, Phylogenetic tree of CoV families (larger context, including nearest bat precursor)</span> (2020-02-03). 
         <a href="https://platform.gisaid.org/epi3/frontend#lightbox983930542">https://platform.gisaid.org/epi3/frontend#lightbox983930542</a>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Callaway</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cyranoski</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-article-title">Why snakes probably aren’t spreading the new China virus</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.16" data-doi="10.1142/S0219720005000953">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Meintjes</span>, 
          <span class="cit-name-given-names">P. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rodrigo</span>, 
          <span class="cit-name-given-names">A. G.</span>
         </span>, 
         <span class="cit-article-title">Evolution of relative synonymous codon usage in Human Immunodeficiency Virus type-1</span>. 
         <abbr class="cit-jnl-abbrev">Journal of bioinformatics and computational biology</abbr>
         <span class="cit-pub-date">2005</span>, 
         <span class="cit-vol">3</span> (
         <span class="cit-issue">01</span>), 
         <span class="cit-fpage">157</span>–
         <span class="cit-lpage">168</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbioinformatics%2Band%2Bcomputational%2Bbiology%26rft.stitle%253DJ%2BBioinform%2BComput%2BBiol%26rft.aulast%253DMeintjes%26rft.auinit1%253DP.%2BL.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D168%26rft.atitle%253DEvolution%2Bof%2Brelative%2Bsynonymous%2Bcodon%2Busage%2Bin%2BHuman%2BImmunodeficiency%2BVirus%2Btype-1.%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS0219720005000953%26rft_id%253Dinfo%253Apmid%252F15751118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1142/S0219720005000953&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15751118&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.17" data-doi="10.1093/nar/gkh340">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edgar</span>, 
          <span class="cit-name-given-names">R. C.</span>
         </span>, 
         <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-pub-date">2004</span>, 
         <span class="cit-vol">32</span> (
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">1792</span>–
         <span class="cit-lpage">1797</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15034147&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220487200025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gouy</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guindon</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gascuel</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-article-title">SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-pub-date">2009</span>, 
         <span class="cit-vol">27</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">221</span>–
         <span class="cit-lpage">224</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D27%26rft.spage%253D221%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=19854763&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273704400003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Roy</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Poisson</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">The I-TASSER Suite: protein structure and function prediction</span>. 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">12</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">7</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D12%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bell</span>, 
          <span class="cit-name-given-names">E. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">D. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">Ensembling multiple raw coevolutionary features with deep residual neural networks for contact-map prediction in CASP13</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-pub-date">2019</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Alexaki</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kames</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Holcomb</span>, 
          <span class="cit-name-given-names">D. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Athey</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Santana-Quintero</span>, 
          <span class="cit-name-given-names">L. V.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lam</span>, 
          <span class="cit-name-given-names">P. V. N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hamasaki-Katagiri</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Osipova</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Simonyan</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bar</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">Codon and Codon-Pair Usage Tables (CoCoPUTs): facilitating genetic variation analyses and recombinant gene design</span>. 
         <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>
         <span class="cit-pub-date">2019</span>, 
         <span class="cit-vol">431</span> (
         <span class="cit-issue">13</span>), 
         <span class="cit-fpage">2434</span>–
         <span class="cit-lpage">2441</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.volume%253D431%26rft.spage%253D2434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.22" data-doi="10.1093/nar/28.1.292">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nakamura</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gojobori</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ikemura</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-article-title">Codon usage tabulated from international DNA sequence databases: status for the year 2000</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-pub-date">2000</span>, 
         <span class="cit-vol">28</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">292</span>–
         <span class="cit-lpage">292</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNakamura%26rft.auinit1%253DY.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D292%26rft.epage%253D292%26rft.atitle%253DCodon%2Busage%2Btabulated%2Bfrom%2Binternational%2BDNA%2Bsequence%2Bdatabases%253A%2Bstatus%2Bfor%2Bthe%2Byear%2B2000%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.292%26rft_id%253Dinfo%253Apmid%252F10592250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/28.1.292&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10592250&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F08%2F2020.02.04.933135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000084896300084&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Peden</span>, 
          <span class="cit-name-given-names">J. F.</span>
         </span>, 
         <span class="cit-article-title">Analysis of codon usage</span>. 
         <span class="cit-pub-date">2000</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.04.933135v1.24" data-doi="10.1109/MCSE.2014.80">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Towns</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cockerill</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dahan</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Foster</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gaither</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Grimshaw</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hazlewood</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lathrop</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lifka</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Peterson</span>, 
          <span class="cit-name-given-names">G. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Roskies</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Scott</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilkins-Diehr</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-article-title">XSEDE: Accelerating Scientific Discovery</span>. 
         <abbr class="cit-jnl-abbrev">Comput Sci Eng</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">16</span> (
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">62</span>–
         <span class="cit-lpage">74</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComput%2BSci%2BEng%26rft.volume%253D16%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2014.80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1109/MCSE.2014.80&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
