<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head charset="UTF-8"/>
 <style> * { font-family : helvetica;} .font16 { font-size : 16; } .font18 { font-size : 18; } .font25 { font-size : 25; } .font10 { font-size : 10; } .font11 { font-size : 11; } .font12 { font-size : 12; } .font13 { font-size : 13; } .font30 { font-size : 30; } .font14 { font-size : 14; } .font15 { font-size : 15; }</style>
 <body>
  <p>
   <span class="font30">subtype </span>
   <span class="font25">sequences </span>
   <span class="font18">viruses </span>
   <span class="font16">A-derived </span>
   <span class="font16">selection </span>
   <span class="font16">CRFs </span>
   <span class="font16">HIV-M </span>
   <span class="font15">fragments </span>
   <span class="font15">different </span>
   <span class="font15">Africa </span>
   <span class="font15">HIV </span>
   <span class="font14">genome </span>
   <span class="font14">lineages </span>
   <span class="font14">tree </span>
   <span class="font14">HIV- </span>
   <span class="font14">likely </span>
   <span class="font14">recombination </span>
   <span class="font14">A-attributed </span>
   <span class="font14">sites </span>
   <span class="font14">using </span>
   <span class="font13">sub-subtype </span>
   <span class="font13">{border-style </span>
   <span class="font13">CM/GA/CF </span>
   <span class="font13">circulating </span>
   <span class="font13">px;} </span>
   <span class="font13">#ffffff;} </span>
   <span class="font13">negative </span>
   <span class="font13">sequence </span>
   <span class="font13">codon </span>
   <span class="font12">(Fig. </span>
   <span class="font12">AAfr </span>
   <span class="font12">analysed </span>
   <span class="font12">parental </span>
   <span class="font12">subtypes </span>
   <span class="font12">evidence </span>
   <span class="font12">lineage </span>
   <span class="font12">position </span>
   <span class="font12">recombinant </span>
   <span class="font12">segments </span>
   <span class="font12">DRC </span>
   <span class="font12">Evolution, </span>
   <span class="font12">amino </span>
   <span class="font12">full-length </span>
   <span class="font12">Table </span>
   <span class="font12">dataset </span>
   <span class="font12">inferred </span>
   <span class="font12">near </span>
   <span class="font12">region </span>
   <span class="font12">Molecular </span>
   <span class="font12">West </span>
   <span class="font12">belonging </span>
   <span class="font12">genomic </span>
   <span class="font12">new </span>
   <span class="font12">sampled </span>
   <span class="font11">East </span>
   <span class="font11">Europe </span>
   <span class="font11">Fig. </span>
   <span class="font11">MRCA </span>
   <span class="font11">Research </span>
   <span class="font11">Subtype </span>
   <span class="font11">Virology, </span>
   <span class="font11">_AE </span>
   <span class="font11">_AG </span>
   <span class="font11">acid </span>
   <span class="font11">clade </span>
   <span class="font11">diversity </span>
   <span class="font11">events </span>
   <span class="font11">evolutionary </span>
   <span class="font11">global </span>
   <span class="font11">high </span>
   <span class="font11">identified </span>
   <span class="font11">indicated </span>
   <span class="font11">prevalence </span>
   <span class="font11">that, </span>
   <span class="font11">(HXB </span>
   <span class="font11">African </span>
   <span class="font11">CRF_cpx </span>
   <span class="font11">Human </span>
   <span class="font11">MCC </span>
   <span class="font11">S). </span>
   <span class="font11">Supplementary </span>
   <span class="font11">Virus </span>
   <span class="font11">_cpx, </span>
   <span class="font11">analyses </span>
   <span class="font11">cluster </span>
   <span class="font11">consistent </span>
   <span class="font11">genes </span>
   <span class="font11">genomes </span>
   <span class="font11">included </span>
   <span class="font11">multiple </span>
   <span class="font11">positive </span>
   <span class="font11">previously </span>
   <span class="font11">signals </span>
   <span class="font11">CRF </span>
   <span class="font11">Cameroon </span>
   <span class="font11">HPD </span>
   <span class="font11">Recombinant </span>
   <span class="font11">Republic </span>
   <span class="font11">Type </span>
   <span class="font11">alignments </span>
   <span class="font11">and/or </span>
   <span class="font11">available </span>
   <span class="font11">compared </span>
   <span class="font11">constructed </span>
   <span class="font11">existed </span>
   <span class="font11">gag </span>
   <span class="font11">groups </span>
   <span class="font11">major </span>
   <span class="font11">model </span>
   <span class="font11">prior </span>
   <span class="font11">px; </span>
   <span class="font11">substitution </span>
   <span class="font11">time </span>
   <span class="font11">{font-family </span>
   <span class="font11">(Supplementary </span>
   <span class="font11">Analysis </span>
   <span class="font11">Bayesian </span>
   <span class="font11">Biology </span>
   <span class="font11">Diversity </span>
   <span class="font11">Group </span>
   <span class="font11">[], </span>
   <span class="font11">_AE, </span>
   <span class="font11">_AG, </span>
   <span class="font11">acids </span>
   <span class="font11">analysis </span>
   <span class="font11">event </span>
   <span class="font11">fitness </span>
   <span class="font11">font-size </span>
   <span class="font11">highly </span>
   <span class="font11">independent </span>
   <span class="font11">individual </span>
   <span class="font11">isolates </span>
   <span class="font11">lineages. </span>
   <span class="font11">monophyletic </span>
   <span class="font11">occurred </span>
   <span class="font11">present </span>
   <span class="font11">  _cpx </span>
   <span class="font11">–); </span>
   <span class="font11">(P = .) </span>
   <span class="font11">(Table </span>
   <span class="font11">(i.e. </span>
   <span class="font11">AIDS </span>
   <span class="font11">BEAST </span>
   <span class="font11">CRF_cpx, </span>
   <span class="font11">Cameroon/Gabon/CAR </span>
   <span class="font11">Evolutionary </span>
   <span class="font11">New </span>
   <span class="font11">Similarly, </span>
   <span class="font11">South </span>
   <span class="font11">_cpx </span>
   <span class="font11">ancestral </span>
   <span class="font11">containing </span>
   <span class="font11">contains </span>
   <span class="font11">data </span>
   <span class="font11">database </span>
   <span class="font11">derived </span>
   <span class="font11">dissemination </span>
   <span class="font11">divergent </span>
   <span class="font11">double; </span>
   <span class="font11">evolving </span>
   <span class="font11">fact </span>
   <span class="font11">favouring </span>
   <span class="font11">genetically </span>
   <span class="font11">however, </span>
   <span class="font11">implemented </span>
   <span class="font11">infections </span>
   <span class="font11">known </span>
   <span class="font11">originated </span>
   <span class="font11">parts </span>
   <span class="font11">patterns </span>
   <span class="font11">phylogenetic </span>
   <span class="font11">predisposed </span>
   <span class="font11">rate </span>
   <span class="font11">regions </span>
   <span class="font11">relative </span>
   <span class="font11">remains </span>
   <span class="font11">sampling </span>
   <span class="font11">sequences. </span>
   <span class="font11">supported </span>
   <span class="font11">world. </span>
   <span class="font10">(LANL </span>
   <span class="font10">A.et </span>
   <span class="font10">AEur, </span>
   <span class="font10">Central </span>
   <span class="font10">Circulating </span>
   <span class="font10">Congo </span>
   <span class="font10">Further, </span>
   <span class="font10">Gabon, </span>
   <span class="font10">LANL </span>
   <span class="font10">Leu) </span>
   <span class="font10">Retroviruses, </span>
   <span class="font10">Strains </span>
   <span class="font10">aligned </span>
   <span class="font10">border-color </span>
   <span class="font10">border-width </span>
   <span class="font10">bound </span>
   <span class="font10">contained </span>
   <span class="font10">date </span>
   <span class="font10">dates </span>
   <span class="font10">detected </span>
   <span class="font10">differences </span>
   <span class="font10">distribution </span>
   <span class="font10">diverse </span>
   <span class="font10">env </span>
   <span class="font10">epidemic </span>
   <span class="font10">estimates </span>
   <span class="font10">gag, </span>
   <span class="font10">gene </span>
   <span class="font10">geographical </span>
   <span class="font10">includes </span>
   <span class="font10">indicate </span>
   <span class="font10">locations </span>
   <span class="font10">nucleotide </span>
   <span class="font10">possibility </span>
   <span class="font10">results </span>
   <span class="font10">selectively </span>
   <span class="font10">solid; </span>
   <span class="font10">sub-lineage </span>
   <span class="font10">times </span>
   <span class="font10">variants </span>
   <span class="font10">world </span>
   <span class="font10">{font-style </span>
   <span class="font10">{font-weight </span>
   <span class="font10">–), </span>
   <span class="font10">‘pure’ </span>
   <span class="font10">(Stamatakis </span>
   <span class="font10">(Worobey </span>
   <span class="font10">(–) </span>
   <span class="font10">Arg(Lys) </span>
   <span class="font10">BMC </span>
   <span class="font10">Biology, </span>
   <span class="font10">CRF_AG </span>
   <span class="font10">Cameroon, </span>
   <span class="font10">Cameroonian </span>
   <span class="font10">Equatorial </span>
   <span class="font10">Evidence </span>
   <span class="font10">Evolution </span>
   <span class="font10">Evolution’, </span>
   <span class="font10">Gag- </span>
   <span class="font10">Genetics </span>
   <span class="font10">Ile(Val) </span>
   <span class="font10">Immunodeficiency </span>
   <span class="font10">June </span>
   <span class="font10">K.et </span>
   <span class="font10">Large </span>
   <span class="font10">Location </span>
   <span class="font10">National </span>
   <span class="font10">Patterns </span>
   <span class="font10">Phylogenetic </span>
   <span class="font10">RAxML </span>
   <span class="font10">Subtypes </span>
   <span class="font10">_AD </span>
   <span class="font10">_cpx_ = fragment </span>
   <span class="font10">account </span>
   <span class="font10">acting </span>
   <span class="font10">adapted </span>
   <span class="font10">approximately </span>
   <span class="font10">bold </span>
   <span class="font10">clades </span>
   <span class="font10">classified </span>
   <span class="font10">courier; </span>
   <span class="font10">dataset. </span>
   <span class="font10">directional </span>
   <span class="font10">display </span>
   <span class="font10">distributed </span>
   <span class="font10">em; </span>
   <span class="font10">ensure </span>
   <span class="font10">estimated </span>
   <span class="font10">evolution </span>
   <span class="font10">example, </span>
   <span class="font10">explain </span>
   <span class="font10">favoured </span>
   <span class="font10">forms </span>
   <span class="font10">fragment </span>
   <span class="font10">frequencies </span>
   <span class="font10">genetic </span>
   <span class="font10">history </span>
   <span class="font10">increased </span>
   <span class="font10">indicating </span>
   <span class="font10">involving </span>
   <span class="font10">large </span>
   <span class="font10">length </span>
   <span class="font10">lineages, </span>
   <span class="font10">location </span>
   <span class="font10">low </span>
   <span class="font10">lower </span>
   <span class="font10">maintenance </span>
   <span class="font10">node </span>
   <span class="font10">nucleotides </span>
   <span class="font10">numbers </span>
   <span class="font10">obtained </span>
   <span class="font10">phylogenetically </span>
   <span class="font10">pol, </span>
   <span class="font10">positions </span>
   <span class="font10">primarily </span>
   <span class="font10">probable </span>
   <span class="font10">probably </span>
   <span class="font10">provided </span>
   <span class="font10">pt;} </span>
   <span class="font10">recombinants </span>
   <span class="font10">respectively </span>
   <span class="font10">rest </span>
   <span class="font10">selected </span>
   <span class="font10">sequences, </span>
   <span class="font10">similar </span>
   <span class="font10">site </span>
   <span class="font10">spread </span>
   <span class="font10">suggests </span>
   <span class="font10">trees </span>
   <span class="font10">yielded </span>
   <span class="font10">  _AD </span>
   <span class="font10">‘HIV- </span>
  </p>
 </body>
</html>
