<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/10/31/458877.atom" />
<meta name="HW.pisa" content="biorxiv;458877v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size" />
<meta name="DC.Identifier" content="10.1101/458877" />
<meta name="DC.Date" content="2018-10-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam’s strain explained more than the sire’s strain (13.2% vs. 1.8%), and the dam’s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains." />
<meta name="DC.Contributor" content="John R. Shorter" />
<meta name="DC.Contributor" content="Paul L. Maurizio" />
<meta name="DC.Contributor" content="Timothy A. Bell" />
<meta name="DC.Contributor" content="Ginger D. Shaw" />
<meta name="DC.Contributor" content="Darla R. Miller" />
<meta name="DC.Contributor" content="Terry J. Gooch" />
<meta name="DC.Contributor" content="Jason S. Spence" />
<meta name="DC.Contributor" content="Leonard McMillan" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="article:published_time" content="2018-10-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam’s strain explained more than the sire’s strain (13.2% vs. 1.8%), and the dam’s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;458877v1" />
<meta name="citation_id" content="458877v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/458877v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/458877v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/458877v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877.full.pdf" />
<meta name="citation_doi" content="10.1101/458877" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="458877" />
<meta name="citation_author" content="John R. Shorter" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author" content="Paul L. Maurizio" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5859-6260" />
<meta name="citation_author" content="Timothy A. Bell" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina" />
<meta name="citation_author" content="Ginger D. Shaw" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author" content="Darla R. Miller" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author" content="Terry J. Gooch" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author" content="Jason S. Spence" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author" content="Leonard McMillan" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_reference" content="Avalos, E. and C. Smith, 1987 Genetic improvement of litter size in pigs. Animal Science 44: 153–163." />
<meta name="citation_reference" content="Aylor, D. L., W. Valdar, W. Foulds-Mathes, R. J. Buus, R. A. Verdugo, et al., 2011 Genetic analysis of complex traits in the emerging collaborative cross. Genome Research 21: 1213–1222." />
<meta name="citation_reference" content="Bandy, T. and E. Eisen, 1984 Prenatal and postnatal effects in mouse lines selected for body weight and litter size: performance of postnatal dams and growth of progeny. Journal of Animal Science 59: 896–907." />
<meta name="citation_reference" content="Beauchamp, G. and A. Kacelnik, 1990 On the fitness functions relating parental care to reproductive value. Journal of Theoretical Biology 146: 513–522." />
<meta name="citation_reference" content="Bogue, M. A., S. C. Grubb, D. O. Walton, V. M. Philip, G. Kolishovski, et al., 2018 Mouse phenome database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice. Nucleic Acids Research 46: D843–D850." />
<meta name="citation_reference" content="Calaway, J. D., A. B. Lenarcic, J. P. Didion, J. R. Wang, J. B. Searle, et al., 2013 Genetic architecture of skewed x inactivation in the laboratory mouse. PLoS Genetics 9: e1003853." />
<meta name="citation_reference" content="Chesler, E. J., 2014 Out of the bottleneck: the diversity outcross and collaborative cross mouse populations in behavioral genetics research. Mammalian Genome 25: 3–11." />
<meta name="citation_reference" content="Chesler, E. J., D. R. Miller, L. R. Branstetter, L. D. Galloway, B. L. Jackson, et al., 2008 The collaborative cross at oak ridge national laboratory: developing a powerful resource for systems genetics. Mammalian Genome 19: 382–389." />
<meta name="citation_reference" content="Cocquet, J., P. J. Ellis, S. K. Mahadevaiah, N. A. Affara, D. Vaiman, et al., 2012 A genetic basis for a postmeiotic x versus y chromosome intragenomic conflict in the mouse. PLoS genetics 8: e1002900." />
<meta name="citation_reference" content="Cocquet, J., P. J. Ellis, Y. Yamauchi, S. K. Mahadevaiah, N. A. Affara, et al., 2009 The multicopy gene sly represses the sex chromosomes in the male mouse germline after meiosis. PLoS Biology 7: e1000244." />
<meta name="citation_reference" content="Collaborative Cross Consortium, 2012 The genome architecture of the collaborative cross mouse genetic reference population. Genetics 190: 389–401." />
<meta name="citation_reference" content="Conway, S., S. Mahadevaiah, S. Darling, B. Capel, A. Rattigan, et al., 1994 Y353/b: a candidate multiple-copy spermiogenesis gene on the mouse y chromosome. Mammalian Genome 5: 203–210." />
<meta name="citation_reference" content="Crowley, J. J., Y. Kim, A. B. Lenarcic, C. R. Quackenbush, C. J. Barrick, et al., 2014 Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis. Genetics 196: 321–47." />
<meta name="citation_reference" content="Crowley, J. J., V. Zhabotynsky, W. Sun, S. Huang, I. K. Pakatci, et al., 2015 Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance. Nature Genetics 47: 353." />
<meta name="citation_reference" content="De la Fuente, L. and F. San Primitivo, 1985 Selection for large and small litter size of the first three litters in mice. Genetique, Selection, Evolution 17: 251." />
<meta name="citation_reference" content="Didion, J. P., H. Yang, K. Sheppard, C.-P. Fu, L. McMillan, et al., 2012 Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias. BMC Genomics 13: 34." />
<meta name="citation_reference" content="Drickamer, L., 1990 Seasonal variation in fertility, fecundity and litter sex ratio in laboratory and wild stocks of house mice (mus domesticus). Laboratory Animal Science 40: 284–288." />
<meta name="citation_reference" content="Ellis, P. J., J. Bacon, and N. A. Affara, 2011 Association of sly with sex-linked gene amplification during mouse evolution: a side effect of genomic conflict in spermatids? Human Molecular Genetics 20: 3010–3021." />
<meta name="citation_reference" content="Falconer, D., 1960 The genetics of litter size in mice. Journal of Cellular Physiology 56: 153–167." />
<meta name="citation_reference" content="Falconer, D. and T. Mackay, 1996 Introduction to Quantitative Genetics. Longmans Green, Harlow, Essex, UK." />
<meta name="citation_reference" content="Ferris, M. T., D. L. Aylor, D. Bottomly, A. C. Whitmore, L. D. Aicher, et al., 2013 Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross. PLoS Pathogens 9: e1003196." />
<meta name="citation_reference" content="Garcia, M. and M. Baselga, 2002 Estimation of genetic response to selection in litter size of rabbits using a cryopreserved control population. Livestock Production Science 74: 45–53." />
<meta name="citation_reference" content="Gralinski, L. E., M. T. Ferris, D. L. Aylor, A. C. Whitmore, R. Green, et al., 2015 Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross. PLOS Genetics 11: e1005504." />
<meta name="citation_reference" content="Gralinski, L. E., V. D. Menachery, A. P. Morgan, A. L. Totura, A. Beall, et al., 2017 Allelic variation in the toll-like receptor adaptor protein Ticam2 contributes to SARS-Coronavirus pathogenesis in mice. G3: Genes, Genomes, Genetics 7: 1653–1663." />
<meta name="citation_reference" content="Gutiérrez, J. P., B. Nieto, P. Piqueras, N. Ibánez, and C. Salgado, 2006 Genetic parameters for canalisation analysis of litter size and litter weight traits at birth in mice. Genetics Selection Evolution 38: 445." />
<meta name="citation_reference" content="Hadfield, J. D., 2010 MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Journal of Statistical Software 33: 1–22." />
<meta name="citation_reference" content="Hager, R. and R. A. Johnstone, 2003 The genetic basis of family conflict resolution in mice. Nature 421: 533." />
<meta name="citation_reference" content="Hoornbeek, F., 1968 Mating success and litter size variation within and between inbred and hybrid generations of rats. Journal of Animal Science 27: 1378–1382." />
<meta name="citation_reference" content="Jinks, J. and P. Broadhurst, 1963 Diallel analysis of litter size and body weight in rats. Heredity 18: 319." />
<meta name="citation_reference" content="Johnson, R. K., M. K. Nielsen, and D. S. Casey, 1999 Responses in ovulation rate, embryonal survival, and litter traits in swine to 14 generations of selection to increase litter size. Journal of Animal Science 77: 541–557." />
<meta name="citation_reference" content="Kass, R. E. and A. E. Raftery, 1995 Bayes factors. Journal of the American Statistical Association 90: 773–795." />
<meta name="citation_reference" content="Kelada, S. N., D. L. Aylor, B. C. Peck, J. F. Ryan, U. Tavarez, et al., 2012 Genetic analysis of hematological parameters in incipient lines of the collaborative cross. G3: Genes, Genomes, Genetics 2: 157–165." />
<meta name="citation_reference" content="Koturbash, I., A. Scherhag, J. Sorrentino, K. Sexton, W. Bodnar, et al., 2011 Epigenetic mechanisms of mouse interstrain variability in genotoxicity of the environmental toxicant 1, 3-butadiene. Toxicological Sciences 122: 448–456." />
<meta name="citation_reference" content="Larson, E. L., S. Keeble, D. Vanderpool, M. D. Dean, and J. M. Good, 2016 The composite regulatory basis of the large x-effect in mouse speciation. Molecular biology and evolution 34: 282–295." />
<meta name="citation_reference" content="Lenarcic, A. B., K. L. Svenson, G. A. Churchill, and W. Valdar, 2012 A general Bayesian approach to analyzing diallel crosses of inbred strains. Genetics 190: 413–35." />
<meta name="citation_reference" content="Macholán, M., S. J. Baird, P. Munclinger, P. Dufková, B. Bímová, et al., 2008 Genetic conflict outweighs heterogametic incompatibility in the mouse hybrid zone? BMC Evolutionary Biology 8: 271." />
<meta name="citation_reference" content="Mathes, W. F., D. L. Aylor, D. R. Miller, G. A. Churchill, E. J. Chesler, et al., 2011 Architecture of energy balance traits in emerging lines of the collaborative cross. American Journal of Physiology-Endocrinology and Metabolism 300: E1124-E1134." />
<meta name="citation_reference" content="Maurizio, P. L., M. T. Ferris, G. R. Keele, D. R. Miller, G. D. Shaw, et al., 2018 Bayesian diallel analysis reveals Mx1-dependent and Mx1-independent effects on response to influenza A virus in mice. G3: Genes, Genomes, Genetics 8: 427–445." />
<meta name="citation_reference" content="Merilä, J. and B. C. Sheldon, 1999 Genetic architecture of fitness and nonfitness traits: empirical patterns and development of ideas. Heredity 83: 103." />
<meta name="citation_reference" content="Morgan, A. P., C.-P. Fu, C.-Y. Kao, C. E. Welsh, J. P. Didion, et al., 2016 The mouse universal genotyping array: from substrains to subspecies. G3: Genes, Genomes, Genetics 6: 263–279." />
<meta name="citation_reference" content="Mousseau, T. A. and D. A. Roff, 1987 Natural selection and the heritability of fitness components. Heredity 59: 181." />
<meta name="citation_reference" content="Odet, F., W. Pan, T. A. Bell, S. G. Goodson, A. M. Stevans, et al., 2015 The founder strains of the Collaborative Cross express a complex combination of advantageous and deleterious traits for male reproduction. G3: Genes, Genomes, Genetics 5: 2671–2683." />
<meta name="citation_reference" content="Oreper, D., Y. Cai, L. M. Tarantino, F. P.-M. de Villena, and W. Valdar, 2017 Inbred strain variant database (isvdb): a repository for probabilistically informed sequence differences among the collaborative cross strains and their founders. G3: Genes, Genomes, Genetics 7: 1623–1630." />
<meta name="citation_reference" content="Percival, C. J., D. K. Liberton, F. Pardo-Manuel de Villena, R. Spritz, R. Marcucio, et al., 2016 Genetics of murine craniofacial morphology: diallel analysis of the eight founders of the collaborative cross. Journal of Anatomy 228: 96–112." />
<meta name="citation_reference" content="Peripato, A., R. De Brito, S. Matioli, L. Pletscher, T. Vaughn, et al., 2004 Epistasis affecting litter size in mice. Journal of Evolutionary Biology 17: 593–602." />
<meta name="citation_reference" content="Philip, V. M., G. Sokoloff, C. L. Ackert-Bicknell, M. Striz, L. Branstetter, et al., 2011 Genetic analysis in the collaborative cross breeding population. Genome Research 21: 1223–1238." />
<meta name="citation_reference" content="Phillippi, J., Y. Xie, D. R. Miller, T. A. Bell, Z. Zhang, et al., 2014 Using the emerging collaborative cross to probe the immune system. Genes and Immunity 15: 38." />
<meta name="citation_reference" content="Price, T. and D. Schluter, 1991 On the low heritability of life-history traits. Evolution 45: 853–861." />
<meta name="citation_reference" content="Rasmussen, A. L., A. Okumura, M. T. Ferris, R. Green, F. Feldmann, et al., 2014 Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance. Science 346: 987–991." />
<meta name="citation_reference" content="Roberts, R., 1960 The effects on litter size of crossing lines of mice inbred without selection. Genetics Research 1: 239–252." />
<meta name="citation_reference" content="Rogala, A. R., A. P. Morgan, A. M. Christensen, T. J. Gooch, T. a. Bell, et al., 2014 The Collaborative Cross as a Resource for Modeling Human Disease: CC011/Unc, a New Mouse Model for Spontaneous Colitis. Mammalian Genome 25: 95–108." />
<meta name="citation_reference" content="Schoenrock, S., D. Oreper, J. Farrington, R. McMullan, R. Ervin, et al., 2017 Perinatal nutrition interacts with genetic background to alter behavior in a parent-of-origin-dependent manner in adult Collaborative Cross mice. Genes, Brain and Behavior." />
<meta name="citation_reference" content="Shorter, J. R., F. Odet, D. L. Aylor, W. Pan, C.-Y. Kao, et al., 2017 Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross. Genetics 206: 557–572." />
<meta name="citation_reference" content="Smith, C. C. and S. D. Fretwell, 1974 The optimal balance between size and number of offspring. The American Naturalist 108: 499–506." />
<meta name="citation_reference" content="Sorge, R. E., L. J. Martin, K. A. Isbester, S. G. Sotocinal, S. Rosen, et al., 2014 Olfactory exposure to males, including men, causes stress and related analgesia in rodents. Nature Methods 11: 629." />
<meta name="citation_reference" content="Srivastava, A., A. P. Morgan, M. L. Najarian, V. K. Sarsani, J. S. Sigmon, et al., 2017 Genomes of the mouse Collaborative Cross. Genetics 206: 537–556." />
<meta name="citation_reference" content="Stirling, D., D. Réale, and D. Roff, 2002 Selection, structure and the heritability of behaviour. Journal of Evolutionary Biology 15: 277–289." />
<meta name="citation_reference" content="Trivers, R. L., 1974 Parent-offspring conflict. Integrative and Comparative Biology 14: 249–264." />
<meta name="citation_reference" content="Turner, L. M., D. J. Schwahn, and B. Harr, 2012 Reduced male fertility is common but highly variable in form and severity in a natural house mouse hybrid zone. Evolution 66: 443–458." />
<meta name="citation_reference" content="Varona, L. and D. Sorensen, 2014 Joint analysis of binomial and continuous traits with a recursive model: a case study using mortality and litter size of pigs. Genetics 196: 643–651." />
<meta name="citation_reference" content="Yu, G., 2009 Variance stabilizing transformations of Poisson, binomial and negative binomial distributions. Statistics &amp; Probability Letters 79: 1621–1629." />
<meta name="twitter:title" content="A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam’s strain explained more than the sire’s strain (13.2% vs. 1.8%), and the dam’s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains." />
<meta name="og-title" property="og:title" content="A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/458877v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam’s strain explained more than the sire’s strain (13.2% vs. 1.8%), and the dam’s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-10-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/458877v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/458877v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/458877v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/458877v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/137542" />
    <title>A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mz5XgjnVAejwjU8gSiBBNihhBGfxYAiPahv3v7itX4w","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"137542","apath":"\/biorxiv\/early\/2018\/10\/31\/458877.atom","pisa":"biorxiv;458877v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;458877v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/458877v1","title":"A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size","identifier":"node\/137542"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227\\\\\\\/Z8AseSd1cagpt78Px0aoH07jfE4olqrbtzIbpVYYeN9IPa7SiznWLJjU3FhQh\\\\\\\/ejtH3pvOBmdNwX3cjm2FBTEF6dSlVLL6qWorqb21kTlbGxLc9isuNWDl0g7ZQiyoQLDgr9rL811d+9\\\\\\\/8XDMdLfu\\\\\\\/suSZNSsrD79vAG17qRPSoKt7zm\\\\\\\/uVCMeR+WtHP\\\\\\\/dwJoxv1VZjBiAFGyIr9B0c0kAYFdOYAT84iIXgC0mp1Iph6ywzei2IcdBFZeyXu3nbzw0UQWAX9AopwY8vs0rbnJui1Q+N519fuoTVs6049hq5M8Tl\\\\\\\/lNK2+ni6dcw2d21Fe6EXNtN08YdlmuoTcbzJGkxrL3Yn7sLJ007c4zj4YwGQNX2BG1\\\\\\\/mgdU25xa72fGVfnChQ9eN1OZQOHxT38Snt6V9as6B\\\\\\\/eu5tkW70\\\\\\\/6yub+Jj4EiB+xEKYPSwglGqhnqPYscbW0xaDa0j5s8EkPX2E8\\\\\\\/K8y2FJ30UMty7KsLWJI\\\\\\\/kVp1O2BqiIy7OlKy5bFENpyFrKEe0ZA\\\\\\\/ibcXCkhJagUEFv1kdfmHUmxV6mPV536H+q137aFiJZuqmppWGZayP9D1Fx3ZpuJxFLlp1yyOux5aD7eCgCLHlVNt47q05PokZNTi\\\\\\\/ffDQKV04AU2rakkhIU2w+OuwVPx+m0T9e4RUVA7ZNcKdrrnGrR+AVuTB9n\\\\\\\/tt8\\\\\\\/rE99MSLeR9JJkKDSCj4U+rTASeEJwt0kUmZLhMQCrpbf1JJlFx9Nx0kAPL\\\\\\\/4JggvXGa2JRgCOk329ztpO1B9v7WrrvlBR6dvm\\\\\\\/nDmPruBawme8R19jX0zjYgeszMP3gCoMzcFfqbf\\\\\\\/SClwwhNAT0oJn0TwMywroKUj\\\\\\\/YDSojehqu88vh\\\\\\\/MQb77AtKHzUu\\\\\\\/nDux59nD1iUkCPtrGVJWJYAPSUFSOBYzZeg3uBCoxoAxu8RlJOTyhmaCNW6mzl57wURbboiyjpoAop5JCcvY966UNx\\\\\\\/iloWdmpDcogfXq1WaE3r0SsO1WLURFdis0rtCF6b5iabrcwpJnwhhU7S01IPyCqid2kvTDmWkCQbTljMX\\\\\\\/Uwds4nujtpJ1ZejKITDySDYcHCoCxt+SRmWos8trXemeQfo0x2BmLzU+wTdVd9pGq0+aRZm9coFvwa3zdnPyStkiVc9cesGsOgoDbRJqrAIiu9HMt9FTfFiAwVLPxnngNhl+rxpQrGyKkXzqGgi5zv9oM\\\\\\\/DxVfV8N77VjwXknDWiAkVOrUhgbdu61wO0UlWZL7VSqGlbLDYeqDLfOpmga0yaHwMqZKW\\\\\\\/XirrI9gJYU4xi1shB\\\\\\\/7zwzQ+gQinhmAkOlLfP9P0QfSVrI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jHqLhNB+j\\\\\\\/kHDv86VYvvpg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002249a2bae2b83ec1936194364d4e256d2897f7bf3834a79bec4f5e149893d9cf84\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/458877v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-137542 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/458877v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bJl6zkV2dMSxGrqX2F_gI4Wutc5I2DlWNqxQlfct7hs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="137542" id="node137542" data-pisa="biorxiv;458877v1" data-pisa-master="biorxiv;458877" data-apath="/biorxiv/early/2018/10/31/458877.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">John R.</span> <span class="nlm-surname">Shorter</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Paul L.</span> <span class="nlm-surname">Maurizio</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Bell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ginger D.</span> <span class="nlm-surname">Shaw</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Darla R.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Terry J.</span> <span class="nlm-surname">Gooch</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">Spence</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leonard</span> <span class="nlm-surname">McMillan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">William</span> <span class="nlm-surname">Valdar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Pardo-Manuel de Villena</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/458877 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node137542" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John R. Shorter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BShorter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shorter%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BR.%2BShorter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Paul L. Maurizio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span><span class='nlm-institution'>Curriculum in Bioinformatics and Computational Biology, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BL.%2BMaurizio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurizio%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BL.%2BMaurizio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul L. Maurizio</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span><span class='nlm-institution'>Lineberger Comprehensive Cancer Center, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ginger D. Shaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ginger%2BD.%2BShaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shaw%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGinger%2BD.%2BShaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Terry J. Gooch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Terry%2BJ.%2BGooch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gooch%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATerry%2BJ.%2BGooch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jason S. Spence </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BS.%2BSpence%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spence%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BS.%2BSpence%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span><span class='nlm-institution'>Department of Computer Science, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span><span class='nlm-institution'>Lineberger Comprehensive Cancer Center, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a>
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Department of Genetics, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span><span class='nlm-institution'>Lineberger Comprehensive Cancer Center, University of North Carolina</span>, Chapel Hill, North Carolina, 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a>
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/458877v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:137542" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:137542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/458877v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:137542" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:137542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/458877v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:137542" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:137542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/458877v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:137542" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:137542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/458877v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:137542" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:137542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam’s strain explained more than the sire’s strain (13.2% vs. 1.8%), and the dam’s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">A critical part of fitness is optimizing the balance between numbers of offspring, offspring body size, and reproductive timing, leading to strong selection for these traits (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Smith and Fretwell 1974</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beauchamp and Kacelnik 1990</a>). Fundamental fitness traits, like litter size (<em>i.e.,</em> the number of pups born per litter), should have low contributions of genetic variance due to rapid fixation of alleles that increase fitness (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mousseau and Roff 1987</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Price and Schluter 1991</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Merilä and Sheldon 1999</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stirling <em>et al.</em> 2002</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Peripato <em>et al.</em> 2004</a>). Estimates of litter size heritability vary from 5% to 20% in mouse, rabbit, and porcine populations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Falconer 1960</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Avalos and Smith 1987</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Falconer and Mackay 1996</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Johnson <em>et al.</em> 1999</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Garcia and Baselga 2002</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Peripato <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gutiérrez <em>et al.</em> 2006</a>) leading to the majority of variation in litter size being non-heritable. Non-heritable factors include seasonality, food availability, paternal and maternal effects, dominance and epistasis, as well as unknown factors. Additionally, sexual conflict over parental investment may exist between females and males, and this in turn could affect litter size (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Trivers 1974</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hager and Johnstone 2003</a>). These factors make it difficult to obtain clear measurements of litter size in wild populations. A controlled laboratory environment is therefore necessary to measure genetic effects of litter size (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jinks and Broadhurst 1963</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Roberts 1960</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bandy and Eisen 1984</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hoornbeek 1968</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Peripato <em>et al.</em> 2004</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gutiérrez <em>et al.</em> 2006</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Varona and Sorensen 2014</a>).</p><p id="p-5">The Collaborative Cross (CC) and its eight founder strains are an important resource for studying complex traits, establishing mouse models for human disease, and understanding the mouse Diversity Outbred (DO), which originated from the CC (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chesler 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rogala <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rasmussen <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gralinski <em>et al.</em> 2015</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2017</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schoenrock <em>et al.</em> 2017</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Maurizio <em>et al.</em> 2018</a>). The CC resource and its founder strains can also be used to study reproductive ability, which is especially interesting because this population has an established record both of breeding successes and of failures. For example, litter size in the early CC lines shows a steady decline over the first six generations of inbreeding (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Philip <em>et al.</em> 2011</a>). In addition, nearly 95% of all CC lines have become extinct, primarily due to subspecific genomic incompatibilities (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shorter <em>et al.</em> 2017</a>). Although standard reproductive phenotypes have been measured, mostly in male CC founders (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Odet <em>et al.</em> 2015</a>), the genetic and non-genetic control over CC breeding success has never been thoroughly characterized, and is likely to contain vital information that may be used to improve CC breeding in the future.</p><p id="p-6">Given the growing popularity of the CC as a community resource, we investigated reproductive ability across the eight founders of the CC to better understand the genetic and nongenetic factors that affect their fertility and breeding. Using an 8 × 8 diallel design, we measured weaned litter sizes from over 4400 litters arising from 62 crosses across four years of breeding. Adapting a recently developed statistical model of diallel effects (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lenarcic <em>et al.</em> 2012</a>), we quantified both the genetic contributions that shape litter size and the contributions of several environmental factors. Our results provide a detailed account of breeding patterns across and between the eight founders. This experiment also informs us about genetic combinations that are highly or marginally productive in the CC, helping us to better understand CC fertility problems and line extinction.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><p id="p-7">The mouse inbred strains used in these experiments are the eight founder strains of the Collaborative Cross (CC) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium 2012</a>). The founders of the CC include five classical strains, A/J (AJ), C57BL/6J (B6), 129S1/SvImJ (129S1), NOD/ShiLtJ (NOD), NZO/HlLtJ (NZO), and three wild-derived strains, CAST/EiJ (CAST), PWK/PhJ (PWK), and WSB/EiJ (WSB). Mice originated from a colony maintained by Gary Churchill at Jackson Laboratory, and were transferred to the FMPV lab at the University of North Carolina (UNC) in 2008. The original colony also produced most of the G1 breeders that populated the inbred funnels at ORNL, TAU and Geniad (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Srivastava <em>et al.</em> 2017</a>). All mice described here were reared by the FPMV lab at UNC. Mice were bred at the UNC Hillsborough vivarium from 2008-2010 and bred at the UNC Genetic Medicine Building (GMB) vivarium from 2010-2012. A total of 4,448 litters resulting from crosses between 1,478 individual dams and 1,238 individual sires were born from all eight inbred crosses and 54 of 56 reciprocal F1 hybrid crosses, excluding hybrids between NZO × CAST and NZO × PWK, which are known unproductive crosses (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chesler et al. 2008</a>). The directions of all crosses are described as female by male (i.e., dam.strain × sire.strain), unless otherwise noted. Litter size and sex were determined at weaning by visual inspection. Animals were kept on a 14-hour, 10-hour light/dark schedule with lights turned on at 7:00 AM; temperature was maintained at 20°-24° with relative humidity between 40-50%. Mice were housed in standard 20×30-cm ventilated polysulfone cages with standard laboratory grade Bed-O-Cob bedding. Water and Purina Prolab RMH3000 were available <em>ad libitum.</em> Mouse chow was supplemented with Fenbendazole (Feb 2010) two weeks before and two weeks after transportation to the GMB facility to eliminate possible pinworms. Selemectin treatment was dropped onto the coats of mice before transfer to remove mites from the cages. These treated cages were not opened until after their arrival at UNC GMB. All animal rearing and breeding was conducted in strict compliance with the <em>Guide for the Care and Use of Laboratory Animals</em> (Institute of Laboratory Animals Resources, National Research Council 1996, <a href="https://www.ncbi.nlm.nih.gov/books/NBK232589/">https://www.ncbi.nlm.nih.gov/books/NBK232589/</a>). The Institutional Animal Care and Use Committee of the University of North Carolina approved all animal use and research described here.</p><div id="sec-3" class="subsection"><h3>Statistical Analysis</h3><div id="sec-4" class="subsection"><h4>Testing environmental interactions</h4><p id="p-8">Significant environmental interactions were determined using ANOVA, using JMP 12 software (JMP®, Version 12. SAS Institute Inc., Cary, NC, 1989-2007). Tested effects included a season effect (average weaned litter size in each month over every year), non-seasonal factors using a year-by-month effect (average weaned litter size for all months across all years), and a litter order effect (average weaned litter size in each subsequent dam litter). We performed the ANOVA on litter size counts from the eight inbred matings only, because of their robust sample sizes throughout the four years of breeding.</p></div><div id="sec-5" class="subsection"><h4>Diallel analysis of litter size</h4><p id="p-9">To estimate the overall contributions of heritable factors affecting litter size in our population, we adapted a previously published linear mixed model, Bayes-Diallel (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lenarcic et al. 2012</a>), which performs this estimation for continuous phenotypes, to the setting of a discrete count-based phenotype. To do this we reimplemented the BayesDiallel Gibbs sampler as a generalized linear mixed model (GLMM) using the R software package MCMCglmm (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hadfield 2010</a>). Let <em>y<sub>i</sub></em> be the number of pups born to litter <em>i</em>, where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-1.gif"/></noscript></span></span>, and denote the corresponding pair of parental strains, namely maternal strain <em>j</em> and paternal strain <em>k</em>, using the notation {<em>jk</em>} [<em>i</em>]. The effect of parental strains on y<sub>i</sub> was modeled using an overdispersed zero-truncated Poisson (ZTP) regression (data scales of the model in brackets):
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-1.gif"/></noscript></span>

</span>
where ZTPois(<em>λ<sub>i</sub></em>) denotes a Poisson distribution with expected value <em>λ<sub>i</sub></em> but conditional on having observed that <em>y<sub>i</sub></em> ≠ 0 (<strong>Appendix A</strong>), g is the link function <em>g(x)</em> = log(x), with inverse <em>g <sup>−1</sup>(x) = e<sup>x</sup></em>, that relates <em>λ<sub>i</sub></em> to a latent scale <em>ℓ<sub>i</sub></em>, and <em>η<sub>i</sub></em> is a linear predictor on that latent scale with an error term <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-2.gif"/></noscript></span></span> providing overdispersion. The linear predictor <em>η<sub>i</sub></em> is then composed of an intercept μ, random effects for litter order (<em>i.e.,</em> parity) <em>u</em><sup>order[i]</sup> and batch (month) <em>u</em><sup>batch[i]</sup> respectively (detailed below), where for example batch <em>[i]</em> refers to the batch containing litter <em>i</em>, and a further linear predictor d<sup>T</sup><em><sub>{jk}[i]</sub> β</em> (detailed below) that specifies the contributions of parental strains <em>j</em> and <em>k</em> (detailed below). The random effects for litter order and batch were both modeled as categorical random intercepts, <em>u</em><sup>batch</sup> ∼ N(0, <em>τ</em><sup>2</sup><sub>batch</sub>) and <em>u</em><sup>order</sup> ∼ N(<em>c</em><sup>order</sup> · <em>α, τ</em><sup>2</sup><sub>order</sub>), where litter order is additionally predicted by <em>c</em><sup>order</sup>, the order as a number, with fixed effect <em>α.</em></p><p id="p-10">The parental strain contribution d<sup>T</sup><sub>{jk}</sub><em>β</em>, is equivalent to the ‘fullu’ (full, unsexed) model described in (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Lenarcic <em>et al.</em> 2012</a>), and is given below:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-2.gif"/></noscript></span>

</span>
where I<sub>{A}</sub> is an indicator and equal to 1 if A is true and 0 otherwise, S<sub>{A}</sub> is a sign variable equal to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-3.gif"/></noscript></span></span> if A is true and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-4.gif"/></noscript></span></span> otherwise, β<sub>inbred</sub> is a fixed effect of being inbred, and all other variables (in classes <em>a, m, b, v, w</em>) are modeled as random effects with class-specific variances, for example, a1,…,a8 are modeled as a<sub>j</sub> ∼ N(0, τ<sup>2</sup><sub>a</sub>). The <em>a</em> (“additive”) class represents strain-specific dosage effects, the <em>m</em> (“parental sex”) class represents asymmetric parent-of-origin effects, where positive values indicate that maternal effects are greater than paternal effects for a given strain, inbred penalty effects are comprised of overall (fixed) and strain-specific (random) b parameters, and epistatic effects are strain-pair specific deviations, both symmetric (<em>v</em>) and asymmetric (<em>w</em>).</p><p id="p-11">To provide directional parent-of-origin effects for maternal and paternal strain we defined, by reparameterization of the additive and parental sex effect, the following two additional types of effect:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-3.gif"/></noscript></span>

</span></p><p id="p-12">For example, dam<sub>B6</sub> = <em>a</em><sub>B6</sub> + <em>m</em><sub>B6</sub> is the B6-specific dam (maternal strain) effect and sire<sub>B6</sub> = <em>a</em><sub>B6</sub> - <em>m</em><sub>B6</sub> is the B6-specific sire (paternal strain) effect. Posterior samples of these quantities were obtained as a post-processing step by reparameterizing posterior samples of <em>a<sub>j</sub></em> and <em>m<sub>j</sub></em>. Obtaining these as a post-processing step, rather than explicit modification of <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>, preserves the original Bayes-Diallel model and therefore has no effect on the effect estimates (or variance projection estimates) for the other diallel categories. (Parameter definitions summarized in Table S1.)</p><p id="p-13">Priors were chosen to be minimally informative. For fixed effects (<em>μ, α</em>, and <em>β</em><sub>inbred</sub>) we used N (0,1 x 10<sup>3</sup>). For variances of random effects <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/inline-graphic-5.gif"/></noscript></span></span> we used an in verse gamma distribution with scale and shape both equal to 0.001. Posterior effect estimates are presented as posterior mean (and median), and the 95% highest posterior density (HPD) interval.</p><p id="p-14">In order to stably estimate the contribution of each variance class to the overall phenotype, we used the diallel variance projection [VarP; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crowley et al. (2014)</a>]. This is a heritability-like measure that partitions the overall phenotypic variance of an idealized future diallel experiment into additive, parent-of-origin (parental sex), inbred (dominance), epistatic, and other random/fixed effects categories in the diallel. VarPs were calculated both from the ZTP model described above, and, for comparison, from the standard Gaussian-outcome BayesDiallel model (using the Bayes-Diallel software) (<strong>Appendix A</strong>). The standard BayesDiallel model was applied to our data after litter size was subject to a variance-stabilizing transformation, the square root, this corresponding to the linear mixed model approximation to the ZTP,
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-4.gif"/></noscript></span>

</span></p><p id="p-15">The standard BayesDiallel model under this transformation was also used to evaluate evidence in our data set for the inclusion (or exclusion) of each variable or class of variables in improving the model fit. Specifically, we calculated for each variance class a posterior model inclusion probability (MIP) <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Lenarcic <em>et al.</em> (2012)</a>. MIPs with a value of 1 indicate strong evidence to retain a variable or inheritance class in the model, whereas MIPs near 0 indicate strong evidence for exclusion of a variable/class from the model. MIPs within the range of (0.05,0.25] or [0.75,0.95) indicate positive evidence, (0.01,0.05] or [0.95,0.99) indicate strong evidence, and [0,0.01] or [0.99,1] indicate very strong evidence for model inclusion, approximately following the scheme of <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kass and Raftery (1995)</a>.</p></div><div id="sec-6" class="subsection"><h4>Diallel analysis of sex ratio</h4><p id="p-16">To model diallel effects on sex ratio we recast the BayesDiallel model as a binomial GLMM. Letting <em>y<sub>i</sub></em> be the number of males out of a total of <em>n<sub>i</sub></em> pups for litter <em>i</em>, we model where <em>π<sub>i</sub></em> is the expected proportion of males predicted for litter <em>i</em>, the link function is g(x) = logit(x) = log(x) / (1 — x), with inverse link <em>g<sup>−1</sup></em> (x) = logit<sup>−1</sup> (x) = expit(x) = <em>e</em><sup>x</sup>/ (1 + <em>e</em><sup>x</sup>), and <em>ℓi</em>, which represents <em>π<sub>i</sub></em> on the latent scale, is modeled using the BayesDiallel hierarchy as in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-17">Additional details about our statistical modeling approaches are provided in <strong>Appendix A</strong> (litter size) and <strong>Appendix B</strong> (sex ratio).</p></div></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-18">File S1 contains all breeding data used for the analysis in this study. File S2 contains the scripts and software used for the analysis in this study. File S3 contains the Supplemental Figures and Tables. The data and code used to analyze the data are available at <a href="https://github.com/mauriziopaul/litterDiallel">https://github.com/mauriziopaul/litterDiallel</a>, with a static version available at <a href="https://doi.org/10.5281/zenodo.1435829">https://doi.org/10.5281/zenodo.1435829</a>. Phenotype data from this study is available on the Mouse Phenome Database (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bogue <em>et al.</em> 2018</a>) at <a href="https://phenome.jax.org/projects/Shorter">https://phenome.jax.org/projects/Shorter</a> with persistent identifier RRID:SCR003212, Accession number MPD:623, project Shorter3.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Litter size is affected by housing facility but not season</h3><p id="p-19">We bred litters in an 8x 8 inbred diallel of the CC founder strains, and generated all eight inbreds and 54 of 56 possible reciprocal F1 hybrids (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Figure S1-S3). The eight inbred strains, displayed across the diagonal, were mated at higher frequencies both for maintenance of inbred strains and propagation of the diallel. For all 62 genetic inbred and hybrid crosses, we recorded the following information: mated pairs, wean dates, litter size at weaning, including total and sex-specific counts (File S1). This diallel cross was originally designed and maintained for the generation of F1 mice for several experimental projects (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koturbash <em>et al.</em> 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mathes <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kelada <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Didion <em>et al.</em> 2012</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calaway <em>et al.</em> 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Crowley <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Phillippi <em>et al.</em> 2014</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Odet <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> 2015</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Morgan <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Percival <em>et al.</em> 2016</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Shorter <em>et al.</em> 2017</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Oreper <em>et al.</em> 2017</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Maurizio <em>et al.</em> 2018</a>). As a result, certain reproductive measurements such as time between litters and maximum number of offspring per cross were necessarily biased by experimental breeding requirements. Average weaned litter size, however, is a reproductive trait that should be well-estimated independently of these factors.</p><div id="F1" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diallel crossing scheme and weaned pup distribution. The number of litters observed per cross is given by the integers, with the largest sample sizes, along the diagonal, corresponding to the production of inbred parental strains. Column and row sums are given along the bottom row and rightmost column, respectively. A total of 4,448 litters were evaluated for this analysis, resulting in a total of 24,782 weaned pups. The shading within each box corresponds to the average number of weaned pups per litter in each cross, with averages ranging from 3.6 to 9.1 pups per litter. Litters for which no pups survived until weaning were not included in our analysis. The symbol &#x201C;&#xD7;&#x201D; is used to indicate incompatible crosses that do not produce any litters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1130460735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diallel crossing scheme and weaned pup distribution. The number of litters observed per cross is given by the integers, with the largest sample sizes, along the diagonal, corresponding to the production of inbred parental strains. Column and row sums are given along the bottom row and rightmost column, respectively. A total of 4,448 litters were evaluated for this analysis, resulting in a total of 24,782 weaned pups. The shading within each box corresponds to the average number of weaned pups per litter in each cross, with averages ranging from 3.6 to 9.1 pups per litter. Litters for which no pups survived until weaning were not included in our analysis. The symbol &#x201C;&#xD7;&#x201D; is used to indicate incompatible crosses that do not produce any litters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F1.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F1.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Diallel crossing scheme and weaned pup distribution. The number of litters observed per cross is given by the integers, with the largest sample sizes, along the diagonal, corresponding to the production of inbred parental strains. Column and row sums are given along the bottom row and rightmost column, respectively. A total of 4,448 litters were evaluated for this analysis, resulting in a total of 24,782 weaned pups. The shading within each box corresponds to the average number of weaned pups per litter in each cross, with averages ranging from 3.6 to 9.1 pups per litter. Litters for which no pups survived until weaning were not included in our analysis. The symbol “×” is used to indicate incompatible crosses that do not produce any litters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We measured litter size and report the mean number of weaned pups per litter for the 62 viable crosses in the diallel (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A wide distribution of litter sizes was observed, ranging from an average of 3.6 weaned pups for WSB × WSB crosses, to an average of 9.1 weaned pups for NOD × PWK crosses, with an overall mean of 5.5 weaned pups per litter. Examining average litter sizes of the inbred strains across all 4 years, we found neglible evidence of consistent seasonal effects (f<sub>11,2283</sub> = 1.272 <em>p</em> = 0.23) (Figure S4) but positive evidence of non-seasonal patterns: a modeled ‘year.month’ covariate significantly affected litter size (f<sub>47,2247</sub> = 2.44 <em>p</em> &lt; .0001; see also Figure S5). The non-seasonal effect could be driven by the relocation of mice between vivariums, which occurred in February 2010. The housing facility may have an impact on litter size as well. To test this, we measured average litter size differences between the two housing facilities and found that two founder strains, 129S1 and WSB, had significantly larger average litter sizes at the Hillsborough facility than at UNC GMB with a difference of 5.87 to 4.42 weaned pups per litter (p &lt; 0.0001) for 129S1, and 3.84 to 3.42 weaned pups per litter (p = 0.029) for WSB. The six other founder strains did not significantly differ in their average litter sizes between the two facilities, but tended to have smaller weaned litters at UNC GMB (see Discussion). For average litter size across the diallel, we tested for an effect of litter number (birth parity number, for a given mating pair) on the number of pups weaned in each litter. Previous research suggests that the first litter can be significantly smaller than subsequent litters due to various biological factors (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">De la Fuente and San Primitivo 1985</a>). This effect was consistent in the diallel: we observed that there was significant reduction in litter size in the first litter (<em>p</em> &lt; 0.0001) compared to the overall linear effect that parity has on reducing litter size (Figure S5).</p></div><div id="sec-10" class="subsection"><h3>Litter size is moderately heritable, and maternal effects account for the majority of explained variation</h3><p id="p-22">To estimate founder strain effects on litter size, we used a Bayesian regression model that decomposes the phenotypic variation in a diallel design into genetic and parent-of-origin contributions (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Lenarcic <em>et al.</em> 2012</a>). Using this model, litter size was estimated to have narrow sense heritability (<em>h</em><sup>2</sup> approximated by VarP[additive]) accounting for 9.18% of the variance, with parent-of-origin effects (VarP[parental.sex]) accounting for 5.77%, the fact of being inbred (VarP[inbred.overall]) at 1.43%, and strain-by-strain interactions (VarP[epistatic.symmetric] + VarP[asymmetric.epistatic]) at 3.40% (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>); in aggregate, these factors explained 17.73% of the total variance for litter size in the founder diallel.</p><div id="F2" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diallel effects and variance contributions for weaned litter size. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Diallel effects, including (left) strain-specific additive, parental sex, and inbred effects, and (right) epistatic effects between each pairwise cross. For each parameter, thin and thick horizontal lines represent 95% and 50% highest posterior density (HPD) intervals of effects, respectively, and vertical break and dash give posterior median and mean, respectively. The effects are in relation to an overall mean litter size of 5.46 (95% HPD: 5.00-6.10). The gray vertical lines indicate zero. Effects are shown as the log, or latent, scale effects on the mean litter size attributable to each strain or strainpair and inheritance group, where values are centered at 0 for each random effect class. Intervals that exclude zero have non-negligible effects on the mean litter size. Labels with &#x201C;v&#x201D; or &#x201C;w&#x201D; refer to symmetric or asymmetric epistatic effects, respectively. Colored bars indicate corresponding variance classes in (A) and (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1130460735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diallel effects and variance contributions for weaned litter size. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Diallel effects, including (left) strain-specific additive, parental sex, and inbred effects, and (right) epistatic effects between each pairwise cross. For each parameter, thin and thick horizontal lines represent 95% and 50% highest posterior density (HPD) intervals of effects, respectively, and vertical break and dash give posterior median and mean, respectively. The effects are in relation to an overall mean litter size of 5.46 (95% HPD: 5.00-6.10). The gray vertical lines indicate zero. Effects are shown as the log, or latent, scale effects on the mean litter size attributable to each strain or strainpair and inheritance group, where values are centered at 0 for each random effect class. Intervals that exclude zero have non-negligible effects on the mean litter size. Labels with &#x201C;v&#x201D; or &#x201C;w&#x201D; refer to symmetric or asymmetric epistatic effects, respectively. Colored bars indicate corresponding variance classes in (A) and (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F2.medium.gif" width="243" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F2.medium.gif" width="243" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Diallel effects and variance contributions for weaned litter size. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Diallel effects, including (left) strain-specific additive, parental sex, and inbred effects, and (right) epistatic effects between each pairwise cross. For each parameter, thin and thick horizontal lines represent 95% and 50% highest posterior density (HPD) intervals of effects, respectively, and vertical break and dash give posterior median and mean, respectively. The effects are in relation to an overall mean litter size of 5.46 (95% HPD: 5.00-6.10). The gray vertical lines indicate zero. Effects are shown as the log, or latent, scale effects on the mean litter size attributable to each strain or strainpair and inheritance group, where values are centered at 0 for each random effect class. Intervals that exclude zero have non-negligible effects on the mean litter size. Labels with “v” or “w” refer to symmetric or asymmetric epistatic effects, respectively. Colored bars indicate corresponding variance classes in (A) and (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">In more detail, we present estimates for all modeled diallel effects as posterior means and highest posterior density (HPD) intervals (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Parameters are divided into two groups: general effects and strain pair-specific effects. General effects comprise strain-specific additive effects (additive), strain-specific and overall inbred (inbred), and strain-specific parent-of-origin (parental sex) effects. Strain pair-specific (epistatic) effects are the effects that arise specifically in crosses of two heterologous strains, with ‘v’ referring to symmetric epistatic and ‘w’ referring to asymmetric epistatic effects (pairwise parent-of-origin effects). All classes of effect were determined to add positive value to the model fit (either strong or very strong evidence for inclusion; MIP &gt; 0.986) except parental sex (the difference between male and female), whose value was equivocal (MIP=0.34) Table S2).</p><p id="p-25">Under the general effects, we see significant positive additive effects on average litter size from B6, NOD, and NZO and significant negative additive effects on litter size from CAST, PWK, and WSB strain dosages. A similar pattern is seen in the parental sex effects, where B6 and NOD dosages have a significant positive effect on average litter size, whereas CAST, PWK, and WSB have significant negative effects. The “inbred” effect is negative indicating that inbred status has a negative overall effect on average litter size, regardless of parental strain. Each strain also has an individual inbred effect in addition to an overall inbred effect; the only strain with a positive inbred effect is NOD. When the individual inbred effects are taken into account with the overall “inbred” effect, inbred litters are on average slightly smaller than their heterozygous counterparts. For strain pair-specific epistatic effects, there are a few marginally noteworthy effects, with the most prominent being a negative asymmetric epistatic effect for PWK × NOD.</p><p id="p-26">To more clearly differentiate the contributions of the mother vs. the father strain, we reparameterized the BayesDiallel model to capture effects specific to dam.strain and sire.strain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). B6 and NOD dams increase litter sizes by more than 1.26 fold, by an average of 1.66 and 1.79 pups, respectively, regardless of sire. CAST, PWK, and WSB dams tend to decrease average litter size by 0.90, 0.81, and 1.51 pups. The sire effect is similar, with NOD and NZO sires having larger litters and CAST sires producing smaller litters. As expected, we see that the dam.strain has a much larger influence on the variation of litter size compared with the sire.strain (13.15% vs. 1.81%).</p><div id="F3" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dam.strain and sire.strain variance contributions and estimates of effects on weaned litter size. These effects are a reparameterization of additive and parental.sex effects from the previous analysis. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Estimates for the maternal (&#x201C;dam.strain&#x201D;) and paternal (&#x201C;sire.strain&#x201D;) effects on litter size, as calculated from the additive and parental sex parameters in Figure 2, with HPD intervals defined correspondingly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1130460735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dam.strain and sire.strain variance contributions and estimates of effects on weaned litter size. These effects are a reparameterization of additive and parental.sex effects from the previous analysis. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Estimates for the maternal (&#x201C;dam.strain&#x201D;) and paternal (&#x201C;sire.strain&#x201D;) effects on litter size, as calculated from the additive and parental sex parameters in Figure 2, with HPD intervals defined correspondingly.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F3.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F3.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Dam.strain and sire.strain variance contributions and estimates of effects on weaned litter size. These effects are a reparameterization of additive and parental.sex effects from the previous analysis. (A) Variance contributions of distinct effect classes, reported as posterior means and 95% HPDs of variance projections (VarPs) on weaned litter size. (B) Estimates for the maternal (“dam.strain”) and paternal (“sire.strain”) effects on litter size, as calculated from the additive and parental sex parameters in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, with HPD intervals defined correspondingly.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>No significant strain effects on sex ratio</h3><p id="p-28">We examined genetic and non-genetic effects on the average sex ratio per litter and found no evidence that sex ratio was skewed (Figure S6;Figure S7). The overall mean for sex bias, quantified as number of male pups weaned divided by the total number of weaned pups, was 0.4979, and did not significantly differ from our expectation of a 50:50 sex ratio (Binominal test, two-tailed <em>p</em> = 0.219). For the eight inbred founders, 129S1 is the only strain that departed significantly from expectation, with slight reduction of males 0.475:0.525 (Binominal test, two-tailed <em>p</em> = 0.041). However, correction for multiple testing shows no significant sex ratio bias of any inbred strain. The outbred crosses have substantially fewer litters and offspring than the inbred matings, leading to less balanced sex ratios; however, when multiple testing is accounted for, they also show no significant deviations from expected sex ratios.</p></div></div><div class="section" id="sec-12"><h2 class="">Discussion</h2><p id="p-29">We have investigated factors influencing litter size in the eight CC founders and their F1 hybrids. We note that litter size is a component of reproductive performance, but distinct from total strain productivity; a study on total strain productivity would need to take into account litter size, numbers of litters, maximum reproductive age, and pup survival until breeding. Our results illustrate how mammalian litter size in a full diallel design is influenced by genotypic and environmental variation. They also address some of the factors that contributed to line extinction and breeding problems in generating the CC (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chesler <em>et al.</em> 2008</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Philip <em>et al.</em> 2011</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Shorter <em>et al.</em> 2017</a>).</p><p id="p-30">We estimated that genetic (additive, inbred, and epistatic) effects on average litter size explained 17.73% of variation, suggesting that most of the phenotypic variation arises from unexplained environmental effects. Within the explained variance, our estimate of the narrow sense heritability (<em>h</em><sup>2</sup> = 0.092) for litter size may seem small, but it is within the range uncovered in previous research (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Falconer 1960</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Peripato <em>et al.</em> 2004</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gutiérrez <em>et al.</em> 2006</a>). Compared with the overall average litter, we observed substantial positive effects of B6 and NOD strains, from both additive genetic and parent-of-origin parameters, and substantial negative effects of PWK and WSB (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). We also observed, as expected, that lower litter size was associated with being inbred (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-31">During the G1 and G2 out-crossing generations of the CC, mean litter size was lower for crosses involving wild-derived strains, CAST, PWK and WSB (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Philip <em>et al.</em> 2011</a>). A similar pattern is observed here, but these effects are determined to be specifically through the maternal strain, with CAST, PWK and WSB having negative “dam” effects on litter size (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3)</a>. It is likely that selection pressure in classical lab strains is associated with larger litters compared with the wild-derived strains. Additionally, two of the wild-derived strains, CAST and PWK, are from a different subspecific origin than the other six CC founders. This likely contributes to decreased productivity through subspecific incompatibilities (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Shorter <em>et al.</em> 2017</a>).</p><p id="p-32">We identify and report environmental factors that may influence litter size. The breeding of this diallel was performed across two different vivariums over the course of 4 years, and we see a significant effect from housing facility. The Hillsborough facility was associated with larger litters for all strains, especially 129S1 and WSB. The two facilities have many different factors that could explain these differences. The Hillsborough facility housed multiple species, including dogs and mice, had smaller rooms that held approximately 300 mouse cages, was remotely located in a rural area, had different laboratory personnel, had cage changes once a week, and was supplied with filtered well water. The diallel breeding at the GMB facility took place in a large central room, contained only laboratory mice, is located in a basement of a large seven story research building on campus, has cage changes every other week, and is supplied with filtered city water. It is possible that one or more of these factors, independently or in combination, affected productivity. Another finding was that seasonality, which has previously been shown to influence litter size and frequency of litters in mammals (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Drickamer 1990</a>), did not seem to significantly impact litter size in this study. This is likely due to consistent light-dark cycles and temperatures as well as a steady diet. We did observe a significant effect on litter size after the transfer of the mice to the UNC GMB facility (February 2010), which reduced overall litter sizes from March to June 2010. This may have been due to the use of ivermectin during the time of the transfer. Other factors, such as the sex of the laboratory personnel interacting with the animals, are generally known to influence rodent behavior and could contribute to some of the environmentally-induced variation we observed(<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sorge <em>et al.</em> 2014</a>).</p><p id="p-33">Last, we measured sex ratio across all inbred and outbred crosses. Despite some departures from equality at the nominal significance threshold (alpha = 0.05), no associations with founder strain dosage remain significant after correction for multiple testing. Recent work has suggested a potential for bias in sex ratio driven by the male germline in <em>Mus musculus</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Conway <em>et al.</em> 1994</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Macholán <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cocquet <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ellis <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cocquet <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Turner <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Larson <em>et al.</em> 2016</a>), particularly in intersubspecific hybrids that are mismatched for copy number of X- and Y-linked genes expressed in postmeiotic spermatids. Although there are no previous observations that suggest a bias in sex ratio in the CC or its founders, it remains an important characteristic to measure in a study on reproductive productivity.</p><p id="p-34">To estimate heritable effects on liter size and sex ratio we extended the original BayesDiallel model of <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Lenarcic <em>et al.</em> (2012)</a> in two new ways. First, to better understand and distinguish the effects arising from female and male parents, we reparameterized our strain-specific additive and parental-sex effects such that we could provide estimates of maternal strain and paternal strain effects separately. In the original BayesDiallel model, maternal and paternal strain effects are split into “additive” effects, which consolidates the effects they have in common, and “parental sex”, which models any remaining deviation between the two. Recognizing that additive effects from the sire are essentially wiped out by the additive effects, we instead collapsed the additive and parental-sex effects into “dam.strain” and “sire.strain” effects in a post-processing step on the posterior output. This allowed us to run the original BayesDiallel model while also viewing our data from the perspective of dam strain and sire strain contributions.</p><p id="p-35">Second, we reimplemented the original MCMC sampler, designed for modeling a continuous outcome variable, in a general package MCMCglmm (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hadfield 2010</a>) in order to model count and binary responses. Litter size, as measured, is most naturally distributed as Poisson, with zero-truncation owing to the fact that only successful litters were recorded. Sex ratio is most naturally modeled as a binomial, with an underlying (male) proportion between 0 and 1. Although it would be possible to obtain an approximate analysis by transformations to normality using the original BayesDiallel (and we do this for litter size for some otherwise hard-to-obtain quantities), we found such approximations to be inadequate for reliable estimation of higher order effects in the case of litter size and deeply flawed in the case of sex ratio.</p><p id="p-36">Although there is a computational cost, and added complexity to determining variance contributions, this new implementation achieves several objectives: 1) we no longer break the assumptions in the original model regarding normally distributed errors; 2) we easily accommodate overdispersion in our data; and 3) we can select from a large number of GLMMs ones that more closely resemble the forms of our data observations. In addition, we believe this flexibility will be appealing to many other researchers who would like to model non-Gaussian distributed phenotypes using diallel designs, and we have provided the code in an R package litterDiallel (<a href="https://doi.org/10.5281/zenodo.1435829">https://doi.org/10.5281/zenodo.1435829</a>).</p><p id="p-37">Overall, these results have implications for other avenues of future research. Future multiparental research populations should test for strain incompatibilities, reproductive phenotyping, and other health traits in a full diallel before the recombinant inbreeding begins (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Odet <em>et al.</em> 2015</a>). These future research populations should also use non-related wild-derived individuals from the same subspecific origin in order to increase genetic diversity without introducing hybrid incompatibilities.</p></div><div class="section" id="sec-13"><h2 class="">Appendix A: Diallel Model For Litter Size</h2><p id="p-39">We collected data on litter size at weaning for 62 genetic crosses of inbred lines, across four years of breeding.</p><p id="p-40">We use zero-truncated poisson (ZTP) regression for modeling our data. This type of regression is explicit in its framework accounting for discrete observations, flexible in its ability to use linear mixed models on the latent scale, and allows for parameterization of excess variance observed, in a way that standard Poisson regression does not. We account for the zero depletion in our data by using ZTP regression instead of standard Poisson regression, since we exclude observations of birth cohorts where no pups survived to weaning.</p><p id="p-41">In <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, the distribution of the observed data (litter size) is displayed for the WSBxWSB inbred mating, along with simulated data from a zero-truncated Poisson distribution based on the data mean.</p><div id="F4" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of observed data and (pseudo-)randomly generated data for the WSBxWSB inbreds. (A) Distribution of the litter diallel data for WSBxWSB (nlitters=341, npups=1244). (B) Frequencies expected from a zero-truncated Poisson distribution with mean=3.648 (same as in A). The vertical red lines indicate the mean value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1130460735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of observed data and (pseudo-)randomly generated data for the WSBxWSB inbreds. (A) Distribution of the litter diallel data for WSBxWSB (nlitters=341, npups=1244). (B) Frequencies expected from a zero-truncated Poisson distribution with mean=3.648 (same as in A). The vertical red lines indicate the mean value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F4.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F4.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Comparison of observed data and (pseudo-)randomly generated data for the WSBxWSB inbreds. (A) Distribution of the litter diallel data for WSBxWSB (nlitters=341, npups=1244). (B) Frequencies expected from a zero-truncated Poisson distribution with mean=3.648 (same as in A). The vertical red lines indicate the mean value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">For the ZTP, the first two moments (mean and variance), for values <em>y<sub>i</sub></em> &gt; 0, are given by:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-10.gif"/></noscript></span>

</span></p><p id="p-44">The relationship between the standard Poisson regression model (with overdispersion) is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. The ZTP modifies this by truncating values on the expected (log) data scale and observed data scale to be 1 or greater.
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/10/31/458877/embed/graphic-11.gif"/></noscript></span>

</span></p><div id="F5" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Latent and data scale comparison. A toy example illustrating standard Poisson regression of count data, with overdispersion, giving values drawn from pseudorandom normal variables drawn from ?(&#x3BC; = 0, sd = 0.5), showing how the continuous latent scale values [top], which correspond to the scale of the linear predictor, map onto their expected (data) scale value (through the inverse link, f (x) = ex) [middle], and how the mean expected (data) scale value corresponds to the observed (data) scale values (integers). In each frame, the vertical red line corresponds to the mean of the values shown in that frame." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1130460735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Latent and data scale comparison. A toy example illustrating standard Poisson regression of count data, with overdispersion, giving values drawn from pseudorandom normal variables drawn from &lt;em&gt;?(&#x3BC;&lt;/em&gt; = 0, sd = 0.5), showing how the continuous &lt;em&gt;latent&lt;/em&gt; scale values [top], which correspond to the scale of the linear predictor, map onto their &lt;em&gt;expected&lt;/em&gt; (data) scale value (through the inverse link, &lt;em&gt;f&lt;/em&gt; (x) = &lt;em&gt;e&lt;/em&gt;&lt;sup&gt;x&lt;/sup&gt;) [middle], and how the mean &lt;em&gt;expected&lt;/em&gt; (data) scale value corresponds to the &lt;em&gt;observed&lt;/em&gt; (data) scale values (integers). In each frame, the vertical red line corresponds to the mean of the values shown in that frame.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F5.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F5.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/31/458877/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Latent and data scale comparison. A toy example illustrating standard Poisson regression of count data, with overdispersion, giving values drawn from pseudorandom normal variables drawn from <em>?(μ</em> = 0, sd = 0.5), showing how the continuous <em>latent</em> scale values [top], which correspond to the scale of the linear predictor, map onto their <em>expected</em> (data) scale value (through the inverse link, <em>f</em> (x) = <em>e</em><sup>x</sup>) [middle], and how the mean <em>expected</em> (data) scale value corresponds to the <em>observed</em> (data) scale values (integers). In each frame, the vertical red line corresponds to the mean of the values shown in that frame.</p><div class="sb-div caption-clear"></div></div></div><div class="section" id="sec-14"><h3>Diallel effect estimates</h3><p id="p-46">Diallel effect estimates are obtained using an MCMCglmm (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Hadfield 2010</a>) implementation of BayesDiallel (BayesDiallel-glmm, henceforth), with 1.515 x 10<sup>6</sup> iterations, 1.5 x 10<sup>4</sup> iterations of burn-in, and thinning by 500 (saving only every 1/500<sup>th</sup> iteration), to obtain 3000 independent samples with minimal autocorrelation, and plotted as highest posterior densities (HPDs) in the results.</p></div><div class="section" id="sec-15"><h3>Conversion to the expected data scale</h3><p id="p-47">The effects estimated from the BayesDiallel-glmm model are transformed from the latent scale to the (expected) data scale via the inverse link function, <em>i.e.</em> exp(a<sub>AJ</sub>), for interpretability of effects on the original data scale.</p></div><div class="section" id="sec-16"><h3>Variance Explained Using Variance Projection</h3><p id="p-48">To avoid the problem of interpretability in transforming variance parameter (and variance projection) estimates from the latent to the observed data scales, we instead calculate and report variance projections, as calculated on the Gaussian version of BayesDiallel. In order to account for heteroscedasticity (unequal variance) of the model residuals that arises from the approximately ZTPoisson nature of the data, we use a variance-stabilizing transformation (VST) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yu 2009</a>) and run BayesDiallel again (Gaussian) to obtain Variance Projections on the modeled data. The VarPs that are calculated from these parameter estimates are an approximation of the variance contributions that we would observe in the GLMM BayesDiallel model.</p></div></div><div class="section" id="sec-17"><h2 class="">Appendix B: Diallel Model For Sex Ratio</h2><p id="p-49">We model the male pup counts and female pup counts jointly, using the BayesDiallel linear model, formulated for binomial GLMM regression. This model directly considers genetic effects on the imbalance in male vs. female pups by parameterizing the number of males and number of total pups (or the total, and the fraction of males in the total). The model is elaborated in the methods section of the main manuscript.</p><p id="p-50">The proportion of weaned pups that are male, or equivalently, the proportion of weaned pups that are female, is approximated by a binomial distribution. In our data, the mean and the variance of male ratio are 0.494 and 0.063, respectively.</p><p id="p-51">To generate the upper and lower 95% boundaries, as shown in Figure S6, for the expected phenotype under the null hypothesis of male pup proportion = 0.5, we used the qbinom function in the stats package in R.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">This work was supported in part by the following grants from the National Institutes of Health: P50GM076468, P50HG006582/P50MH090338, R01HD065024 (FP-MV), T32HD040127 (JRS), T32AI007419 (PLM), R01GM104125 (PLM), and R35GM127000 (WV). The Collaborative Cross project is also supported by the University Cancer Research Funds granted to Lineberger Comprehensive Cancer Center (MCR012CCRI).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Avalos</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Genetic improvement of litter size in pigs</span>. <abbr class="cit-jnl-abbrev">Animal Science</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BScience%26rft.volume%253D44%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.2" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aylor</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Foulds-Mathes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Verdugo</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging collaborative cross</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzMxLzQ1ODg3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.3" data-doi="10.2527/jas1984.594896x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bandy</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eisen</span></span>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Prenatal and postnatal effects in mouse lines selected for body weight and litter size: performance of postnatal dams and growth of progeny</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Science</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">896</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DBandy%26rft.auinit1%253DT.%2BR.%26rft.volume%253D59%26rft.issue%253D4%26rft.spage%253D896%26rft.epage%253D907%26rft.atitle%253DPrenatal%2Band%2BPostnatal%2BEffects%2Bin%2BMouse%2BLines%2BSelected%2Bfor%2BBody%2BWeight%2Band%2BLitter%2BSize%253A%2BPerformance%2Bof%2BPostnatal%2BDams%2Band%2BGrowth%2Bof%2BProgeny%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1984.594896x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1984.594896x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.4" data-doi="10.1016/S0022-5193(05)80376-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beauchamp</span>,  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kacelnik</span></span>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">On the fitness functions relating parental care to reproductive value</span>. <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D146%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252805%252980376-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(05)80376-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.5" data-doi="10.1093/nar/gkx1082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bogue</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Grubb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. O.</span> <span class="cit-name-surname">Walton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Philip</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kolishovski</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mouse phenome database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D843</span>–<span class="cit-lpage">D850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253DD843%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1082%26rft_id%253Dinfo%253Apmid%252F29136208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29136208&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calaway</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Lenarcic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Didion</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Searle</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic architecture of skewed x inactivation in the laboratory mouse</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr> <span class="cit-vol">9</span>: e1003853.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.7" data-doi="10.1007/s00335-013-9492-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chesler</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Out of the bottleneck: the diversity outcross and collaborative cross mouse populations in behavioral genetics research</span>. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2BGenome%26rft.volume%253D25%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9492-9%26rft_id%253Dinfo%253Apmid%252F24272351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9492-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24272351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.8" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chesler</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Branstetter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Galloway</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Jackson</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The collaborative cross at oak ridge national laboratory: developing a powerful resource for systems genetics</span>. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cocquet</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Ellis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Mahadevaiah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Affara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vaiman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genetic basis for a postmeiotic x versus y chromosome intragenomic conflict in the mouse</span>. <abbr class="cit-jnl-abbrev">PLoS genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002900</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cocquet</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Ellis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yamauchi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Mahadevaiah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Affara</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The multicopy gene sly represses the sex chromosomes in the male mouse germline after meiosis</span>. <abbr class="cit-jnl-abbrev">PLoS Biology</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1000244</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.11" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Collaborative Cross Consortium</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.12" data-doi="10.1007/BF00360546"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Conway</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mahadevaiah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Darling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Capel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rattigan</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Y353/b: a candidate multiple-copy spermiogenesis gene on the mouse y chromosome</span>. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DConway%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D203%26rft.epage%253D210%26rft.atitle%253DY353%252FB%253A%2Ba%2Bcandidate%2Bmultiple-copy%2Bspermiogenesis%2Bgene%2Bon%2Bthe%2Bmouse%2BY%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00360546%26rft_id%253Dinfo%253Apmid%252F7516747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00360546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7516747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994ND31300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.13" data-doi="10.1534/genetics.113.156901"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crowley</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Lenarcic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Quackenbush</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Barrick</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156901%26rft_id%253Dinfo%253Apmid%252F24240528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMzIxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.14" data-doi="10.1038/ng.3222"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crowley</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Zhabotynsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. K.</span> <span class="cit-name-surname">Pakatci</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D47%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3222%26rft_id%253Dinfo%253Apmid%252F25730764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25730764&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De la Fuente</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">San Primitivo</span></span>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Selection for large and small litter size of the first three litters in mice</span>. <abbr class="cit-jnl-abbrev">Genetique, Selection, Evolution</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetique%252C%2BSelection%252C%2BEvolution%26rft.volume%253D17%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.16" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Didion</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sheppard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-P.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">McMillan</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drickamer</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Seasonal variation in fertility, fecundity and litter sex ratio in laboratory and wild stocks of house mice (mus domesticus)</span>. <abbr class="cit-jnl-abbrev">Laboratory Animal Science</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaboratory%2Banimal%2Bscience%26rft.stitle%253DLab%2BAnim%2BSci%26rft.aulast%253DDrickamer%26rft.auinit1%253DL.%2BC.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D284%26rft.epage%253D288%26rft.atitle%253DSeasonal%2Bvariation%2Bin%2Bfertility%252C%2Bfecundity%2Band%2Blitter%2Bsex%2Bratio%2Bin%2Blaboratory%2Band%2Bwild%2Bstocks%2Bof%2Bhouse%2Bmice%2B%2528Mus%2Bdomesticus%2529%26rft_id%253Dinfo%253Apmid%252F2162984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2162984&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DH66200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.18" data-doi="10.1093/hmg/ddr204"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">P. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bacon</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Affara</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association of sly with sex-linked gene amplification during mouse evolution: a side effect of genomic conflict in spermatids</span>? <abbr class="cit-jnl-abbrev">Human Molecular Genetics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3010</span>–<span class="cit-lpage">3021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddr204%26rft_id%253Dinfo%253Apmid%252F21551453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddr204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21551453&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292560000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.19" data-doi="10.1002/jcp.1030560414"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">The genetics of litter size in mice</span>. <abbr class="cit-jnl-abbrev">Journal of Cellular Physiology</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCellular%2BPhysiology%26rft.volume%253D56%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.1030560414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.1030560414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-458877v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mackay</span></span>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-publ-name">Longmans Green, Harlow, Essex</span>, <span class="cit-publ-loc">UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferris</span>,  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bottomly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Whitmore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Aicher</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">PLoS Pathogens</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003196</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Baselga</span></span>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation of genetic response to selection in litter size of rabbits using a cryopreserved control population</span>. <abbr class="cit-jnl-abbrev">Livestock Production Science</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivestock%2BProduction%2BScience%26rft.volume%253D74%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Ferris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Whitmore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Green</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">PLOS Genetics</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005504</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. D.</span> <span class="cit-name-surname">Menachery</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Morgan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Totura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beall</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Allelic variation in the toll-like receptor adaptor protein <em>Ticam2</em> contributes to SARS-Coronavirus pathogenesis in mice</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1653</span>–<span class="cit-lpage">1663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D7%26rft.spage%253D1653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.25" data-doi="10.1186/1297-9686-38-5-445"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nieto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Piqueras</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ibánez</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Salgado</span></span>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic parameters for canalisation analysis of litter size and litter weight traits at birth in mice</span>. <abbr class="cit-jnl-abbrev">Genetics Selection Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGutierrez%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D445%26rft.epage%253D462%26rft.atitle%253DGenetic%2Bparameters%2Bfor%2Bcanalisation%2Banalysis%2Bof%2Blitter%2Bsize%2Band%2Blitter%2Bweight%2Btraits%2Bat%2Bbirth%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-38-5-445%26rft_id%253Dinfo%253Apmid%252F16954039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-38-5-445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16954039&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.26" data-doi="10.18637/jss.v033.i01"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v033.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v033.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.27" data-doi="10.1038/nature01239"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hager</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Johnstone</span></span>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genetic basis of family conflict resolution in mice</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHager%26rft.auinit1%253DR.%26rft.volume%253D421%26rft.issue%253D6922%26rft.spage%253D533%26rft.epage%253D535%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2Bfamily%2Bconflict%2Bresolution%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01239%26rft_id%253Dinfo%253Apmid%252F12556892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12556892&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180670600045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.28" data-doi="10.2527/jas1968.2751378x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoornbeek</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Mating success and litter size variation within and between inbred and hybrid generations of rats</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Science</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1378</span>–<span class="cit-lpage">1382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHoornbeek%26rft.auinit1%253DF.%2BK.%26rft.volume%253D27%26rft.issue%253D5%26rft.spage%253D1378%26rft.epage%253D1382%26rft.atitle%253DMating%2BSuccess%2Band%2BLitter%2BSize%2BVariation%2Bwithin%2Band%2Bbetween%2BInbred%2Band%2BHybrid%2BGenerations%2Bof%2BRats%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1968.2751378x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1968.2751378x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.29" data-doi="10.1038/hdy.1963.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jinks</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Broadhurst</span></span>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Diallel analysis of litter size and body weight in rats</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D18%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1963.33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1963.33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.30" data-doi="10.2527/1999.773541x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">R. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Casey</span></span>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Responses in ovulation rate, embryonal survival, and litter traits in swine to 14 generations of selection to increase litter size</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Science</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%2BK.%26rft.volume%253D77%26rft.issue%253D3%26rft.spage%253D541%26rft.epage%253D557%26rft.atitle%253DResponses%2Bin%2Bovulation%2Brate%252C%2Bembryonal%2Bsurvival%252C%2Band%2Blitter%2Btraits%2Bin%2Bswine%2Bto%2B14%2Bgenerations%2Bof%2Bselection%2Bto%2Bincrease%2Blitter%2Bsize%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1999.773541x%26rft_id%253Dinfo%253Apmid%252F10229349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1999.773541x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10229349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084444700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.31" data-doi="10.2307/2291091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kass</span>,  <span class="cit-name-given-names">R. E.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Raftery</span></span>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayes factors</span>. <abbr class="cit-jnl-abbrev">Journal of the American Statistical Association</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2BAssociation%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10783804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RA10400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelada</span>,  <span class="cit-name-given-names">S. N.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Peck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Ryan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Tavarez</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic analysis of hematological parameters in incipient lines of the collaborative cross</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D2%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.33" data-doi="10.1093/toxsci/kfr133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koturbash</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Scherhag</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sorrentino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sexton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bodnar</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Epigenetic mechanisms of mouse interstrain variability in genotoxicity of the environmental toxicant 1, 3-butadiene</span>. <abbr class="cit-jnl-abbrev">Toxicological Sciences</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfr133%26rft_id%253Dinfo%253Apmid%252F21602187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfr133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21602187&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293914500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larson</span>,  <span class="cit-name-given-names">E. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Keeble</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vanderpool</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Dean</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Good</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The composite regulatory basis of the large x-effect in mouse speciation</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft.volume%253D34%26rft.spage%253D282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.35" data-doi="10.1534/genetics.111.132563"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Svenson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A general Bayesian approach to analyzing diallel crosses of inbred strains</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132563%26rft_id%253Dinfo%253Apmid%252F22345610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macholán</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Baird</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Munclinger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Dufková</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bímová</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic conflict outweighs heterogametic incompatibility in the mouse hybrid zone</span>? <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D8%26rft.spage%253D271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathes</span>,  <span class="cit-name-given-names">W. F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Chesler</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Architecture of energy balance traits in emerging lines of the collaborative cross</span>. <abbr class="cit-jnl-abbrev">American Journal of Physiology-Endocrinology and Metabolism</abbr> <span class="cit-vol">300</span>: E<span class="cit-fpage">1124</span>-E<span class="cit-lpage">1134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPhysiology-Endocrinology%2Band%2BMetabolism%26rft.volume%253D300%26rft.spage%253D1124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maurizio</span>,  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Ferris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Keele</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. D.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Bayesian diallel analysis reveals Mx1-dependent and Mx1-independent effects on response to influenza A virus in mice</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D8%26rft.spage%253D427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.39" data-doi="10.1038/sj.hdy.6885850"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Merilä</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Sheldon</span></span>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic architecture of fitness and nonfitness traits: empirical patterns and development of ideas</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D83%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6885850%26rft_id%253Dinfo%253Apmid%252F10469197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6885850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10469197&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082889600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-P.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-Y.</span> <span class="cit-name-surname">Kao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Welsh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Didion</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D6%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.41" data-doi="10.1038/hdy.1987.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mousseau</span>,  <span class="cit-name-given-names">T. A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Roff</span></span>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Natural selection and the heritability of fitness components</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D59%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1987.113%26rft_id%253Dinfo%253Apmid%252F3316130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1987.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3316130&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K315700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Odet</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Bell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Goodson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Stevans</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The founder strains of the Collaborative Cross express a complex combination of advantageous and deleterious traits for male reproduction</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2671</span>–<span class="cit-lpage">2683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D5%26rft.spage%253D2671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oreper</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Tarantino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. P.-M.</span> <span class="cit-name-surname">de Villena</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inbred strain variant database (isvdb): a repository for probabilistically informed sequence differences among the collaborative cross strains and their founders</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D7%26rft.spage%253D1623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.44" data-doi="10.1111/joa.12382"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Percival</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Liberton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pardo-Manuel de Villena</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Spritz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Marcucio</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetics of murine craniofacial morphology: diallel analysis of the eight founders of the collaborative cross</span>. <abbr class="cit-jnl-abbrev">Journal of Anatomy</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnatomy%26rft.volume%253D228%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjoa.12382%26rft_id%253Dinfo%253Apmid%252F26426826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/joa.12382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426826&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.45" data-doi="10.1111/j.1420-9101.2004.00702.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peripato</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">De Brito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Matioli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pletscher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vaughn</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis affecting litter size in mice</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DPeripato%26rft.auinit1%253DA.%2BC.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D593%26rft.epage%253D602%26rft.atitle%253DEpistasis%2Baffecting%2Blitter%2Bsize%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2004.00702.x%26rft_id%253Dinfo%253Apmid%252F15149402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2004.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15149402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221197900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.46" data-doi="10.1101/gr.113886.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Philip</span>,  <span class="cit-name-given-names">V. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Ackert-Bicknell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Striz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Branstetter</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis in the collaborative cross breeding population</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113886.110%26rft_id%253Dinfo%253Apmid%252F21734011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIyMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzMxLzQ1ODg3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.47" data-doi="10.1038/gene.2013.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillippi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Bell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using the emerging collaborative cross to probe the immune system</span>. <abbr class="cit-jnl-abbrev">Genes and Immunity</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Band%2BImmunity%26rft.volume%253D15%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgene.2013.59%26rft_id%253Dinfo%253Apmid%252F24195963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gene.2013.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24195963&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.48" data-doi="10.2307/2409693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schluter</span></span>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">On the low heritability of life-history traits</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D45%26rft.spage%253D853%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FW83600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.49" data-doi="10.1126/science.1259595"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Okumura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Ferris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Green</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Feldmann</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259595%26rft_id%253Dinfo%253Apmid%252F25359852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMi85ODciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8zMS80NTg4NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">The effects on litter size of crossing lines of mice inbred without selection</span>. <abbr class="cit-jnl-abbrev">Genetics Research</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D1%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.51" data-doi="10.1007/s00335-013-9499-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rogala</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Morgan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Christensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Gooch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. a.</span> <span class="cit-name-surname">Bell</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Collaborative Cross as a Resource for Modeling Human Disease: CC011/Unc, a New Mouse Model for Spontaneous Colitis</span>. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2BGenome%26rft.volume%253D25%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9499-2%26rft_id%253Dinfo%253Apmid%252F24487921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9499-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487921&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoenrock</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Oreper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Farrington</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">McMullan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ervin</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Perinatal nutrition interacts with genetic background to alter behavior in a parent-of-origin-dependent manner in adult Collaborative Cross mice</span>. <abbr class="cit-jnl-abbrev">Genes, Brain and Behavior</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.53" data-doi="10.1534/genetics.116.199596"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shorter</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Odet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-Y</span>. <span class="cit-name-surname">Kao</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199596%26rft_id%253Dinfo%253Apmid%252F28592496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.54" data-doi="10.1086/282929"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">C. C.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Fretwell</span></span>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The optimal balance between size and number of offspring</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D108%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974T647300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sorge</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Martin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Isbester</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Sotocinal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rosen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Olfactory exposure to males, including men, causes stress and related analgesia in rodents</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D11%26rft.spage%253D629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.56" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Morgan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Najarian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. K.</span> <span class="cit-name-surname">Sarsani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Sigmon</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.57" data-doi="10.1046/j.1420-9101.2002.00389.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stirling</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Réale</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Roff</span></span>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Selection, structure and the heritability of behaviour</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D15%26rft.spage%253D277%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.2002.00389.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.2002.00389.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174709000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.58" data-doi="10.1093/icb/14.1.249"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trivers</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Parent-offspring conflict</span>. <abbr class="cit-jnl-abbrev">Integrative and Comparative Biology</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegrative%2Band%2BComparative%2BBiology%26rft.stitle%253DIntegr.%2BComp.%2BBiol.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D249%26rft.epage%253D264%26rft.atitle%253DParent-Offspring%2BConflict%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252F14.1.249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/14.1.249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.59" data-doi="10.1111/j.1558-5646.2011.01445.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Schwahn</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Harr</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reduced male fertility is common but highly variable in form and severity in a natural house mouse hybrid zone</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DTurner%26rft.auinit1%253DL.%2BM.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D443%26rft.epage%253D458%26rft.atitle%253DReduced%2Bmale%2Bfertility%2Bis%2Bcommon%2Bbut%2Bhighly%2Bvariable%2Bin%2Bform%2Band%2Bseverity%2Bin%2Ba%2Bnatural%2Bhouse%2Bmouse%2Bhybrid%2Bzone.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01445.x%26rft_id%253Dinfo%253Apmid%252F22276540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01445.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22276540&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F31%2F458877.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299547200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.60" data-doi="10.1534/genetics.113.159475"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varona</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sorensen</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Joint analysis of binomial and continuous traits with a recursive model: a case study using mortality and litter size of pigs</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159475%26rft_id%253Dinfo%253Apmid%252F24414548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvNjQzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMzEvNDU4ODc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-458877v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Variance stabilizing transformations of Poisson, binomial and negative binomial distributions</span>. <abbr class="cit-jnl-abbrev">Statistics &amp; Probability Letters</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1621</span>–<span class="cit-lpage">1629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%2B%2526%2BProbability%2BLetters%26rft.volume%253D79%26rft.spage%253D1621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/458877v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F458877v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/458851v1" title="Integrated Cell Expression Toolkit for low input transcript profiling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 31, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/458877v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/458877v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/458877v1" class="active" data-icon-position="" data-hide-link-title="0">A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/137542" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iEtUwUiq8KqjIr1vLF_oq6ksjd2keJJ97hJ1rdW6ey0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="137542" id="node137542--2" data-pisa="biorxiv;458877v1" data-pisa-master="biorxiv;458877" data-seqnum="137542" data-apath="/biorxiv/early/2018/10/31/458877.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">John R.</span> <span class="nlm-surname">Shorter</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Paul L.</span> <span class="nlm-surname">Maurizio</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Bell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ginger D.</span> <span class="nlm-surname">Shaw</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Darla R.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Terry J.</span> <span class="nlm-surname">Gooch</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">Spence</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leonard</span> <span class="nlm-surname">McMillan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">William</span> <span class="nlm-surname">Valdar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Pardo-Manuel de Villena</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">458877; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/458877 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/458877v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;title=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;title=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;text=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;title=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;t=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;title=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;title=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="137542" id="node137542--3" data-pisa="biorxiv;458877v1" data-pisa-master="biorxiv;458877" data-seqnum="137542" data-apath="/biorxiv/early/2018/10/31/458877.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">John R.</span> <span class="nlm-surname">Shorter</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Paul L.</span> <span class="nlm-surname">Maurizio</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Bell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ginger D.</span> <span class="nlm-surname">Shaw</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Darla R.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Terry J.</span> <span class="nlm-surname">Gooch</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">Spence</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leonard</span> <span class="nlm-surname">McMillan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">William</span> <span class="nlm-surname">Valdar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Pardo-Manuel de Villena</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">458877; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/458877 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Shorter%2C+John+R.&amp;rft.au=Maurizio%2C+Paul+L.&amp;rft.au=Bell%2C+Timothy+A.&amp;rft.au=Shaw%2C+Ginger+D.&amp;rft.au=Miller%2C+Darla+R.&amp;rft.au=Gooch%2C+Terry+J.&amp;rft.au=Spence%2C+Jason+S.&amp;rft.au=McMillan%2C+Leonard&amp;rft.au=Valdar%2C+William&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/137542/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/137542/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/137542/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/137542/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/137542/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/137542/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/137542/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/137542/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/137542/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/137542/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/137542/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/137542/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;count=horizontal&amp;via=&amp;text=A%20diallel%20of%20the%20mouse%20Collaborative%20Cross%20founders%20reveals%20strong%20strain-specific%20maternal%20effects%20on%20litter%20size&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/458877v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/458877v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/458877v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
