<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Biomed Res Int</span> <span class="iso-abbrev" title="iso-abbrev">Biomed Res Int</span> <span class="publisher-id" title="publisher-id">BMRI</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BioMed Research International</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">2314-6133</span> <span class="issn-epub" tagx="issn" title="issn-epub">2314-6141</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Hindawi</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5435908">5435908</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1155/2017/5279065">10.1155/2017/5279065</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Ribonuclease from <i> Bacillus</i> Acts as an Antiviral Agent against Negative- and Positive-Sense Single Stranded Human Respiratory RNA Viruses </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6543-688X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">Raihan</span> </span> <a href="#I1"> <sup>1</sup> </a> <a href="#cor1"> <sup>*</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MÃ¼ller</span> <span class="given-names" tagx="given-names" title="given-names">Christin</span> </span> <a href="#I2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romanova</span> <span class="given-names" tagx="given-names" title="given-names">Yulia</span> </span> <a href="#I1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mostafa</span> <span class="given-names" tagx="given-names" title="given-names">Ahmed</span> </span> <a href="#I2"> <sup>2</sup> </a> <a href="#I3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulyanova</span> <span class="given-names" tagx="given-names" title="given-names">Vera</span> </span> <a href="#I1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pleschka</span> <span class="given-names" tagx="given-names" title="given-names">Stephan</span> </span> <a href="#I2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">Olga</span> </span> <a href="#I1"> <sup>1</sup> </a> </span> </div> <span class="citation_author_institution" id="I1">[], <sup>1</sup> </span> <span class="citation_author_institution" id="I2">[], <sup>2</sup> </span> <span class="citation_author_institution" id="I3">[], <sup>3</sup> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
*Raihan Shah Mahmud: <span class="email" tagx="email" title="email">raihan.shah@gmail.com</span> </div> <div class="fn-type-other" title="other"> <p>Academic Editor: Syed Abdul Qadir</p> </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2017</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-5-5</span> </span> <span class="volume" tagx="volume" title="volume">2017</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">5279065</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-1-6</span> <span class="accepted" title="accepted">accepted: 2017-4-4</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Bacillus pumilus</i> ribonuclease (binase) was shown to be a promising antiviral agent in animal models and cell cultures. However, the mode of its antiviral action remains unknown. To assess the binase effect on intracellular viral RNA we have selected single stranded negative- and positive-sense RNA viruses, influenza virus, and rhinovirus, respectively, which annually cause respiratory illnesses and are characterized by high contagious nature, mutation rate, and antigen variability. We have shown that binase exerts an antiviral effect on both viruses at the same concentration, which does not alter the spectrum of A549 cellular proteins and expression of housekeeping genes. The titers of influenza A (H1N1pdm) virus and human rhinovirus serotype 1A were reduced by 40% and 65%, respectively. A preincubation of influenza virus with binase before infection significantly reduced viral titer after single-cycle replication of the virus. Using influenza A virus mini genome system we showed that binase reduced GFP reporter signaling indicating a binase action on the expression of viral mRNA. Binase reduced the level of H1N1pdm viral NP mRNA accumulation in A549 cells by 20%. Since the viral mRNA is a possible target for binase this agent could be potentially applied in the antiviral therapy against both negative- and positive-sense RNA viruses. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
KFU, Russia</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
JLU Giessen</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Russian Science Foundation</div> <span class="award-id" tagx="award-id" title="award-id">14-14-00522</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
German Academic Exchange Service</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
DZIF</div> <span class="award-id" tagx="award-id" title="award-id">RFMEFI59414X0003</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Ministry of Education and Science of the Russian Federation</div> </div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The single stranded RNA viruses, such as negative-sense Ebola, Marburg, Lassa, and influenza and positive-sense human immunodeficiency viruses, are very important human pathogens in the world. Recent virus outbreaks with a large number of human deaths were caused by RNA viruses like Ebola, corona, Zika, and different strains of influenza A viruses. Among RNA viruses, the respiratory viruses are highly contagious and cause annually worldwide epidemics and occasional pandemic outbreaks. The influenza A virus even without a pandemic outbreak kills up to half million humans each year. A great number of diseases are attributable to human rhinoviruses (HRV) which are the major cause of the common cold. HRV infections are suffered by everyone. Recent reports suggest that HRV infection is associated with severe respiratory illness in children [ <a href="#B1">1</a>]. </p> <p>The main problem of antiviral therapy is drug resistance. Although the majority of common viral diseases are self-limited illnesses that do not require specific antiviral therapy, antiviral drugs help to prevent complications or shorten the severity and duration of viral diseases symptoms. Considerable economic losses from annual epidemics promote constant search for new antiviral agents, which become useless with time due to the high variability of viruses. </p> <p>Usually, two groups of drugs are used for the treatment of influenza. The first group including rimantadine and its analogs inhibits the ion channel function of M2 protein, thereby inhibiting viral uncoating. The second one represented by oseltamivir inhibits the enzyme neuraminidase [ <a href="#B2">3</a>]. Inducing adaptive immunity through the vaccination with inactivated or attenuated antigenic material of viruses helps to prevent influenza illness or to reduce its impact in individuals. There is no permanent vaccination against influenza virus and each year countries lose a large amount of money from their budget to prepare new vaccines and rescue the national public health and life. Widespread antiviral resistance due to the high mutation rate in viral genome limits the clinical impact of all types of medicinal products for the prevention and treatment of influenza infections. </p> <p>Currently, there is no vaccine against HRV [ <a href="#B1">1</a>]. The HRV has almost the highest number of serotypes (more than 100) among all respiratory viruses due to mutations of its genome. Antigenic heterogeneity of HRVs is regarded as a major barrier to effective vaccine development which has to be highly polyvalent and has resulted in little progress over 50 years [ <a href="#B1">1</a>, <a href="#B3">4</a>]. Other problems, which have impeded rhinoviral vaccine development, include the lack of convenient animal models and as expected the absence of great impact on respiratory morbidity and mortality while reducing the frequency of common colds [ <a href="#B4">5</a>] </p> <p>Today, there are no drugs that destroy viruses; all antiviral agents can only inhibit viral development. Therefore, degradation of the viral RNA seems to be a very promising approach of antiviral therapy. The earliest studies on the antiviral activity of RNases were performed using pancreatic RNase as an agent that quickly normalized the state and decreased the symptoms of meningitis and cerebrospinal pleocytosis in patients with tick borne encephalitis [ <a href="#B5">6</a>]. Shortcoming of mammalian RNases is their affinity to cytosolic RNase inhibitor protein (RI) in human cells by which adventitious mammalian RNases are inhibited. In contrast, bacterial RNases are not inhibited by RI and can retain their catalytic activity in mammalian tissues. One of the well-studied bacterial RNases is binase, the guanyl-preferring low molecular weight RNase secreted by <i> Bacillus pumilus</i> [ <a href="#B6">7</a>]. </p> <p>We proposed earlier that mechanisms of antiviral activity of RNases include both the direct action on nucleic acid and indirect effects, that is, intervention into the RNA interference, immunomodulation, and induction of infected cell apoptosis (for review see [ <a href="#B7">8</a>]). We have demonstrated that binase reduced the titer of pandemic influenza A/Hamburg/4/2009 (H1N1pdm), reovirus serotype 1 (Reo 1-Lang), herpes virus type I (pseudorabies), Middle East respiratory syndrome coronavirus (MERS-CoV), and human corona virus (HCoV-229E) in infected Madin-Darby canine kidney (MDCK) epithelial cells, African green monkey kidney (Vero) cells, Madin-Darby bovine kidney (MDBK) epithelial cells, human fetal lung fibroblast (MRC5) cells, and hepatocellular carcinoma (Huh7) cells, respectively [ <a href="#B8">9</a>â <a href="#B10">11</a>]. Influenza viruses contain single stranded negative-sense RNA; reoviruses have a double stranded RNA genome, while herpes virus possesses genomic DNA. Because of the specific catalytic activity of binase towards RNA (mainly single stranded) and its inhibitory effect towards RNA- and DNA-containing viruses, it could be proposed that this enzyme should affect mRNA, which is synthesized by all viruses. The aim of our study was to prove that viral mRNA is a direct target of binase. </p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.cells,viruses,andstocks" title="sec"> <div class="title" tagx="title" title="title">
2.1. Cells, Viruses, and Stocks</div> <p>A546 (human lung adenocarcinoma epithelial), HeLa (human cervical cancer), and MDCK-II (Madin-Darby canine kidney epithelial) cells were cultured in growth Dulbecco's modified Eagle medium (DMEM) (Gibco, USA) supplemented with 10% fetal bovine serum (PAA, Austria), 100âU/ml penicillin, and 0.1âmg/ml streptomycin (P/S) (Gibco, USA) and incubated at 37Â°C in a 5% CO <sub>2</sub> atmosphere. The pandemic influenza A/Hamburg/04/09 (H1N1pdm) and human rhinovirus serotype 1A (HRV1A) were provided by the strain collection of the Institute of Medical Virology, Justus Liebig University, Giessen, Germany, and propagated in MDCK-II and HeLa cells, respectively. Briefly, MDCK-II and HeLa monolayers were incubated with H1N1pdm and HRV1A at a multiplicity of infection (MOI) of 1 diluted in PBS <sup>++</sup>/BA (PBS containing 0.2% bovine albumin (PAA, Germany), 1âmM MgCl <sub>2</sub>, 0.9âmM CaCl <sub>2</sub>, 100âU/ml penicillin, and 0.1âmg/ml streptomycin). After 1âh of virus adsorption in the dark, the cell monolayers were supplemented with medium and incubated at either 37Â°C (H1N1pdm) or 33Â°C (HRV1A) in humidified 5% CO <sub>2</sub> conditions. </p> </div> <div class="2.2.bacteriagrowthconditionandbinaseisolation" title="sec"> <div class="title" tagx="title" title="title">
2.2. Bacteria Growth Condition and Binase Isolation</div> <p>Binase (EC 3.1.27.3; bacterial extracellular guanyl-preferring ribonuclease) was collected from the culture medium of <i> Bacillus pumilus</i> B3073 (the former name is <i> B. intermedius</i> B3073 [ <a href="#B6">7</a>]). The cultures were grown in a 1âL conical flask containing 1/5 flask volume of the phosphate deficient liquid medium (2.0% low phosphate peptone, 1.0% glucose, 1.0% CaCl <sub>2</sub>, 0.3% NaCl, 0.03% MgSO <sub>4</sub>ââÃ 7H <sub>2</sub>O, and 0.01% MnSO <sub>4</sub>, pH 8.5) with 200ârpm in a Multitron shaking incubator (INFORS HT, Switzerland) at 37Â°C until the beginning of the stationary phase of bacterial growth. After centrifugation at 6000 <i>g</i> for 30âmin, the pelleted bacterial cells were removed; the supernatant was acidified by glacial acetic acid to pH 5.0, diluted 1â:â5 in double-distilled water (ddH <sub>2</sub>O), and applied on the column with DE-32 cellulose (Whatman, UK) equilibrated with 10âmM sodium acetate buffer (pH 5.0). The flow through from DE-32 was transferred to column packed with phosphocellulose P11 (Whatman, UK) equilibrated with 10âmM sodium acetate buffer (pH 5.0). The system was then equilibrated with 20âmM sodium phosphate buffer (pH 7.0). Binase was eluted with 200âmM sodium phosphate buffer (pH 7.0). Further polishing purification of binase was performed using the UnoS column and BioLogic DuoFlow chromatography system (Bio-Rad, USA) equilibrated with 20âmM sodium acetate buffer (pH 5.0) and eluted using a linear gradient of 0â0.25âM NaCl. Elution fractions of 5âml were collected. The protein samples were frozen overnight at â20Â°C, followed by freezing for 15âmin at â80Â°C and then lyophilized for 48âh at â50Â°C under vacuum using a Labconco FreeZone 2.5 Liter Freeze Dry System (Labconco, USA). Homogeneity and authenticity of the purified enzyme were confirmed by PAAG electrophoresis, Western blotting, and MALDI TOF/TOF mass spectrometry on UltrafleXtreme (Bruker Corporation, Germany) as described in [ <a href="#B11">12</a>]. </p> </div> <div class="2.3.enzymaticactivityofbinase" title="sec"> <div class="title" tagx="title" title="title">
2.3. Enzymatic Activity of Binase</div> <p>The ribonuclease activity of binase was registered by 260ânm absorbance measuring of the products of binase hydrolysis of high molecular weight yeast RNA. Binase was incubated in 0.25âM Tris-HCl buffer (pH 8.5) in the presence of 0.5â <i>Î¼</i>g/ <i>Î¼</i>l RNA (final concentration) in a water bath for 15âmin at 37Â°C. The reaction was stopped by addition of ice cold 6.8% perchloric acid followed by incubation on ice bath for 10âmin followed by centrifugation for 10âmin at 12000 <i>g</i>. The supernatant was diluted 1â:â20 in ddH <sub>2</sub>O and used for the 260ânm absorbance measurement using spectrophotometer SmartSpec Plus (Bio-Rad, USA). One unit of ribonuclease activity corresponded to the quantity of enzyme that increased the extinction of acid-soluble products of RNA hydrolysis at 260ânm by 1 optical unit after incubation for 1âh at 37Â°C, calculated per ml of the enzyme solution. </p> </div> <div class="2.4.ribonucleaseactivityincellculturemedium" title="sec"> <div class="title" tagx="title" title="title">
2.4. Ribonuclease Activity in Cell Culture Medium</div> <p>To identify the enzymatic activity of binase, the growth medium from subconfluence (90â95%) A549 cells in a 96-well plate (in triplicate) was replaced with 100â <i>Î¼</i>g/ml binase (final concentration) containing complete DMEM and incubated for 48âh at 37Â°C in 5% CO <sub>2</sub>. The supernatant (cell culture fluid) containing binase was collected to measure the binase activity using RNase activity assay as described before (see <a href="#sec2.3">Section 2.3</a>). </p> </div> <div class="2.5.cellviabilitymeasurement" title="sec"> <div class="title" tagx="title" title="title">
2.5. Cell Viability Measurement</div> <p>The cytotoxic concentration (CC <sub>50</sub>, the concentration of compound that causes death to 50% of viable cells) of HeLa cells under binase treatment was measured using MTT assay as described by Shah Mahmud and Ilinskaya [ <a href="#B12">13</a>]. Briefly, growth medium of subconfluence HeLa cells in a 96-well plate (in triplicate) was replaced with DMEM containing different concentration of binase (at final concentrations of 0.005â500â <i>Î¼</i>g/ml) and kept for 24âh at 33Â°C in a humidified 5% CO <sub>2</sub> incubator. After aspirating the media and washing with PBS <sup>++</sup> (phosphate-buffered saline containing 1âmM MgCl <sub>2</sub>, 0.9âmM CaCl <sub>2</sub>) the cells were left to recover for 60âmin in growth medium and then replaced by 200â <i>Î¼</i>l of MTT-mix (growth DMEM medium containing 0.123âmg/ml of MTT-reagent = 1-(4,5-dimethylthiazol-2- yl)-3,5-diphenylformazan, Sigma Aldrich, USA). The cells were further incubated for 60âmin at 37Â°C in 5% CO <sub>2</sub>. After discarding the MTT-mix, the cells were fixed with 4% PFA paraformaldehyde (Roth, Germany) for 30âmin at RT (room temperature) and then the fixing solution was discarded. The cells were air dried for 10â15âmin and the tetrazolium crystals were dissolved by adding 200â <i>Î¼</i>l of isopropanol to each well. The plates were shaken for 10âmin and analyzed photometrically at 492ânm excitation in an ELISA reader EL808 (BioTek, USA). The MTT value of the binase nontreated control was used as 100%. The percentage of cell viability after binase exposure was calculated as follows: percentage viability = 100/(MTT value of control Ã MTT value of binase-treated sample). The CC <sub>50</sub> was determined using GraphPad Prism 5.0 Software (GraphPad Software Inc., USA) by plotting the percentage of viable cells as a function of the compound concentration. </p> </div> <div class="2.6.binasetreatment" title="sec"> <div class="title" tagx="title" title="title">
2.6. Binase Treatment</div> <p>To determine the effect of 100â <i>Î¼</i>g/ml binase against H1N1pdm virus, the A549 monolayers were subcultured to prepare 96-well plates (10 <sup>6</sup> cells/well, <i>n</i> = 8). Following the overnight 90â95% confluency of the monolayers, the growth medium was removed and cell monolayers were washed with PBS <sup>++</sup>. Virus suspension containing H1N1pdm with MOI = 1 diluted in PBS <sup>++</sup>/BA (50â <i>Î¼</i>l) was added to each well and incubated for 1âh at RT. The viral supernatant was then removed and binase at a final concentration of 10 or 100â <i>Î¼</i>g/ml in an infection medium (DMEM containing 100âUâml <sup>â1</sup> penicillin, 100â <i>Î¼</i>g/ml streptomycin, 0.2% BSA (PAA, Germany), and 1â <i>Î¼</i>g/ml TPCK-treated trypsin) was added. The cells were incubated for 12âh and then the viral loads in the collected supernatants were estimated using focus assay as described in [ <a href="#B12">13</a>, <a href="#B13">14</a>]. The cell monolayers were subjected to cellular total protein or RNA extractions for further investigations. The binase and virus preincubation for 45âmin and infection and incubation steps were performed as previously described by Shah Mahmud and Ilinskaya [ <a href="#B12">13</a>]. </p> </div> <div class="2.7.binaseeffectiveconcentrationdetermination" title="sec"> <div class="title" tagx="title" title="title">
2.7. Binase Effective Concentration Determination</div> <p>The effective concentration (EC <sub>50</sub>, the concentration of compound which reduces the virus titer by 50%) of binase against HRV1A was determined in HeLa cells. The cells were grown in a 48-well plate (in triplicate) overnight at 37Â°C and 5% CO <sub>2</sub>. Cell monolayers were then washed and infected with 75â <i>Î¼</i>l of virus suspension in PBS <sup>++</sup>/BA (MOI = 1) for 1âh at 33Â°C. After removing the virus inoculum, cells were incubated with 250â <i>Î¼</i>l growth medium in each well containing different concentrations of binase (at final concentrations of 0.01â500â <i>Î¼</i>g/ml) for 24âh, at 33Â°C and 5% CO <sub>2</sub>. The supernatants containing HRV1A virus were assayed for their viral titers using plaque assay. </p> </div> <div class="2.8.virustitration" title="sec"> <div class="title" tagx="title" title="title">
2.8. Virus Titration</div> <p>The HRV1A titers were determined in a 24-well cell culture plate containing subconfluence (90â95%) HeLa cell monolayers using plaque assay. Briefly, the HRV1A virus inoculum was diluted 1â:â10 in PBS <sup>++</sup>/BA. The growth media from HeLa cells were removed and the cell monolayers were washed using PBS <sup>++</sup>. The diluted virus was added to each well and incubated for 1âh at 33Â°C and 5% CO <sub>2</sub> and then removed. 500â <i>Î¼</i>l of Avicel-media containing 1x MEM and 1.25% Avicel (FMC BioPolymer, Belgium) was added and cells were further incubated for 48âh at 33Â°C with 5% CO <sub>2</sub>. After removal of the overlay media, the cell monolayers were washed with PBS <sup>++</sup>, stained with 0.1% crystal violet solution, further washed, and dried. The plaques corresponding to propagating viral particles were counted. The H1N1pdm titers were determined in MDCK-II using focus assay. To analyze the binase effect against H1N1pdm or HRV1A and identify the EC <sub>50</sub> of binase, the plaques were counted and PFU/ml was measured using MS Excel Software package. The PFU/ml values of the binase nontreated infected HeLa cells were used as 100%. The EC <sub>50</sub> was determined using GraphPad Prism 5.0 Software (GraphPad Software Inc., USA) by plotting the percentage of viable cells as a function of the compound concentration. </p> </div> <div class="2.9.proteinextractionfromcells" title="sec"> <div class="title" tagx="title" title="title">
2.9. Protein Extraction from Cells</div> <p>To perform a two-dimensional (2D) difference gel electrophoresis (DIGE) protein expression analysis, the total cellular proteins of binase-treated A549 cells (100â <i>Î¼</i>g/ml) were extracted after 12âh of incubation in 6-well plates (in triplicate). Binase nontreated cells were taken as control. After removal of the cell culture media, the cell monolayers were washed twice with PBS <sup>++</sup>, scratched off, and collected in 1âml PBS. The cells were then pelleted by centrifugation for 30âsec at 13000 <i>g</i> and 4Â°C. The supernatant was discarded and cell pellets were lysed on ice using lysis buffer (TLB: 25âmM Tris, pH 8.0; 137âmM NaCl; 10% glycerol; 0.1% sodium dodecyl sulphate; 0.5% sodium deoxycholate; 1% NP-40; 2âmM EDTA, pH 8.0; 0.2âmM Pefablock; 5â <i>Î¼</i>g/ml aprotinin; 5â <i>Î¼</i>g/ml leupeptin; 1âmM Na-vanadate; and 5âmM benzamidine). After adding the lysis buffer, the cells were incubated for 10âmin on ice and successively centrifuged for 30âmin at 13000 <i>g</i> with 4Â°C and supernatants were collected. The protein concentration of each sample was determined according to the manufacturer's instruction using Bradford Assay Kit (Bio-Rad, USA). </p> </div> <div class="2.10.cyaninedyelabelingofproteinsamples" title="sec"> <div class="title" tagx="title" title="title">
2.10. Cyanine Dye Labeling of Protein Samples</div> <p>Total cellular proteins (100â <i>Î¼</i>g) from binase nontreated (control) and binase-treated samples were labeled with green (Cy3) and red (Cy5) cyanine dyes at a concentration of 800 pmol dye/100â <i>Î¼</i>l of proteins, respectively. The dye and protein mix were incubated on ice for 1âh in the dark at RT. The labeling reaction was terminated by incubating with 10âmM lysine for 10âmin. The labeled protein was mixed and used for isoelectric focusing (IEF). </p> </div> <div class="2.11.2ddifferencegelelectrophoresis" title="sec"> <div class="title" tagx="title" title="title">
2.11. 2D Difference Gel Electrophoresis</div> <p>The isoelectric pH gradient (IPG) tube gel was prepared using 8âM urea; 4% acrylamide; 2.4% ampholytes; 2.4% CHAPS; and NP40 according to Protean IEF Cell (Bio-Rad, USA) manufacturer's instructions. 100â <i>Î¼</i>g of each labeled protein mix from control and test samples was combined in a 50â <i>Î¼</i>l standard buffer (8âM urea; 4% CHAPS and NP-40; 10âmM Tris-HCl, pH 8.5) containing 2% DTT (dithiothreitol) and added onto the prepared IEF gel in a tube. The first-dimensional isoelectric focusing was carried out on a PROTEAN II xi Cell (Bio-Rad, USA) chamber at RT using the following program: 100âV for 1âh, 200âV for 1âh, a gradient of 200âVâ700âV for 6âh, 700âV for 5âh, 900âV for 1.5âh, and 1000âV for 1âh. </p> <p>The 9â16% polyacrylamide gradient gel for two-dimensional gel electrophoresis was prepared according to Protean IEF Cell (Bio-Rad, USA) manufacturer protocol. The IEF gel was equilibrated for 15âmin in SDS-PAAG equilibration buffer (6âM urea; 0.375âM Tris-HCl, pH 8.8; 2% SDS; 20% glycerol; 2% (w/v) DTT) for 15âmin. The IEF gel was carefully placed on top of 9â16% SDS-PAAG with 400âml upper electrolyte (20âmM NaOH) on glass plate. 2 liters of lower electrolyte contained 20âmM of H <sub>3</sub>PO <sub>4</sub>. The electrophoresis was carried out using the following program: 20âmA for 20âmin, 40âmA for 2âh, and 35âmA for 3âh. The proteins separated on gels covering the 15â150âkDa range. A Typhoon FLA 9500 Imager (GE Healthcare, UK) was used to visualize protein spot on the gels. The excitation and emission wavelength combinations of 532ânm and 635ânm were used to capture the images of protein labeled with Cy3 and Cy5, respectively. The spot detection matching was carried out by using ImageQuant TL software (GE Healthcare, UK). </p> </div> <div class="2.12.plasmidsandprimers" title="sec"> <div class="title" tagx="title" title="title">
2.12. Plasmids and Primers</div> <p>For the in vitro reconstitution of viral ribonucleoprotein (vRNP) complex, four plasmids pHW-PB1-Hamburg, pHW-PB2-Hamburg, pHW-PA-Hamburg, and pHW-NP-Hamburg generating the mRNAs of polymerases and NP segments of H1N1pdm virus together with pPol1-GFP-RT generating a vRNA-like Pol1-transcript encoding the green fluorescent protein (GFP) were used. GFP, a reporter protein in negative-sense, are flanked by the 3â²- and 5â²-noncoding region of the NS RNA segment of influenza A/WSN/33 virus [ <a href="#B14">2</a>] placed between a truncated human RNA polymerase I promotor (POLI) and hepatitis delta virus ribozyme (R). The expressed subunits of the viral polymerases and nucleoprotein replicated and transcribed the influenza virus-like RNA expressed by pPOLI-GFP-RT into mRNA. </p> <p>To perform the real-time RT-PCR the primer pairs (Actin_SYBR_F: 5â²-AGG CAC CAG GGC GTG AT-3â²; Actin_SYBR_R: 5â²-GCC CAC ATA GGA ATC CTT CTG AC-3â² and GAPDH_SYBR_F: 5â²-CCA GGT GGT CTC CTC TGA CTT C-3â²; GAPDH_SYBR_R: 5â²-CAA AGT GGT CGT TGA GGG CAA TG-3â²) were used to amplify the cellular ACTB (beta-actin) and GAPDH (glyceraldehyde-3-phosphate dehydrogenase) genes, respectively. The H1N1pdm viral NP mRNA specific primer pairs (HH_SYBR_F: 5â²-GGC CAT AAG GAC CAG GAG TG-3â²; HH_SYBR_R: 5â²-CCG CTG AAT GCT GCC ATA AC-3â²) were used to identify the binase effect on influenza A viral RNA using real-time RT-PCR. </p> </div> <div class="2.13.rnaextractionandcdnapreparation" title="sec"> <div class="title" tagx="title" title="title">
2.13. RNA Extraction and cDNA Preparation</div> <p>To identify the binase action on the cellular and viral mRNA expression levels in A549-infected cells with H1N1pdm, binase-treated/infected, binase-treated/noninfected, noninfected, and nontreated A549 cells were kept for 12âh in 6-well plate (in triplicate). After washing with PBS <sup>++</sup>, the scratched cells were collected in PBS and immediately counted. 10 <sup>6</sup> cells from each sample were used for RNA extraction and cDNA preparation. The cells from each sample were lysed with RLT buffer (Qiagen, Germany). Total RNA was purified using a RNeasy mini Kit (Qiagen, Germany) according to recommended protocol. The total RNAs of each sample were eluted from each filter using the same volume of elution buffer. Equal volumes of total RNA were used to amplify A549 cellular mRNA of ACTB, mRNA of GAPDH, and H1N1pdm viral NP mRNA genes cDNA using MMLV RT Kit (Evrogen, Russia). In brief, 0.5â2â <i>Î¼</i>g of each total RNA in equal volume of each eluted sample was mixed with 20â <i>Î¼</i>M specific reverse primer, preincubated for 2âmin at 70Â°C, and then immediately snap-cooled on ice. The reverse transcription was performed in 20â <i>Î¼</i>l solution containing preprepared RNA and primer mix, 100âU of MMLV reverse transcriptase, 1âmM of dNTP mix, 2âmM of DTT, 1x buffer of MMLV RT Kit (Evrogen, Russia), and 1âU RiboLock RNase Inhibitor (Thermo Fisher Scientific, USA). The RT-PCR was performed for 60âmin at 42Â°C. The cDNA of each sample were diluted 1â:â10 using RNase free water (Thermo Fisher Scientific, USA) and used as a template for real-time PCR. </p> </div> <div class="2.14.real-timepcr" title="sec"> <div class="title" tagx="title" title="title">
2.14. Real-Time PCR</div> <p>To quantify the cellular and viral mRNA in A549 cells using qPCRmix-HS SYBR Kit (Evrogen, Russia), the 4â <i>Î¼</i>l of each cDNA (1â100âng to each reaction) (see <a href="#sec2.13">Section 2.13</a>), 0.4â <i>Î¼</i>M of each (forward and reverse) primers, and 1x qPCRmix-HS SYBR (Evrogen, Russia) were mixed in a LightCycler 480 Multiwell Plate 96 (Roche Diagnostics, Switzerland) on ice in the dark and covered by LightCycler 480 Sealing Foil. Thermal cycling was done on a LightCycler 480 (Roche Diagnostics, Switzerland) system under the following conditions: 10âmin at 95Â°C; 40 cycles of 15âsec at 95Â°C; 30âsec at 60Â°C; and 30âsec at 30Â°C. The quantification and data analysis was performed using the LightCycler 480 Service Software (Roche Diagnostics, Switzerland). For identification of binase action on cellular mRNA or H1N1pdm viral NP mRNA accumulation according to the relative fold-difference of expression levels (relative changes in gene expression from real-time quantitative PCR experiments) using internal control, mRNA expression data of a reference GAPDH gene was used to correct the control/calibrator (binase nontreated) and test/experimental (binase-treated) samples data. The ÎCt value of the calibrator from influenza virus-infected and binase nontreated cells (Ct <sub>target</sub> (Ct value obtained using H1N1pdm viral NP mRNA specific primers in A549 cells) âCt <sub>reference</sub> (Ct value obtained using GAPDH mRNA primers in A549 cells)) and ÎCt value of the test from influenza virus-infected and binase-treated cells (Ct <sub>target</sub> â Ct <sub>reference</sub>) were calculated. The fold change was calculated using 2 <sup>âÎÎCt</sup> value, where ÎÎCt = ÎCt, test â ÎCt, calibrator. </p> </div> <div class="2.15.transfection" title="sec"> <div class="title" tagx="title" title="title">
2.15. Transfection</div> <p>To investigate the effect of binase on the gene expression activity of H1N1pdm virus, 1â <i>Î¼</i>g of every five plasmids (see <a href="#sec2.12">Section 2.12</a>) encoding viral ribonucleoprotein complex together with pPOLI-GFP-RT was cotransfected into A549 cells as previously described with minor modifications (in triplicate) [ <a href="#B15">15</a>, <a href="#B16">16</a>]. Briefly, the transfection mixture consisting of 180â <i>Î¼</i>l "Opti-MEM" (Thermo Fisher Scientific, USA) and total of 5â <i>Î¼</i>g of plasmid DNA along with 12â <i>Î¼</i>l "TransIT-2020" (Mirus Bio, USA) was incubated for 45âmin at RT. Then, the mixture was diluted to 1âml using "Opti-MEM" and transferred to 80â90% confluent A549 cell monolayer (grown in 6-well plate) to allow transfection. The cells were then kept for 12âh at 37Â°C in 5% CO <sub>2</sub> incubator. The transfection medium was replaced with 2âml of Opti-MEM containing 0.2% BA and P/S with binase (10â <i>Î¼</i>g/ml) or without binase. After 36âh, the GFP expression was visualized and counted by Axiovert 35 inverted fluorescent microscope (Zeiss, Germany). </p> </div> <div class="2.16.statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.16. Statistical Analysis</div> <p>Statistical tests and graphical data presentation were performed using GraphPad Prism 5 Software (GraphPad Software, Inc., USA) and MS Excel 2010 (Microsoft, USA). All data are presented as the mean Â± standard deviation of the mean (SD). The significance between two groups was determined via "Student <i> t</i>-test." A significant difference was considered to be a <i>P</i> value of &amp;lt;0.05. </p> </div> <div class="2.17.biosafety" title="sec"> <div class="title" tagx="title" title="title">
2.17. Biosafety</div> <p>All experiments involving influenza A (H1N1pdm) and rhinovirus serotype 1A (HRV1A) were performed using biosafety level 2 containment laboratory approved for such use by the local authorities (RP, Giessen, Germany). </p> </div> </div> <div class="3.resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
3. Results and Discussion</div> <p>In the present work, we have studied the effect of bacterial ribonuclease binase on infectious titer of negative-sense single stranded RNA influenza A (H1N1pdm) virus after its single-cycle replication in A549 cell culture (12âh p.i.). In contradistinction to the previous report on the maximum anti-influenza effect of binase after multicycle virus replication [ <a href="#B12">13</a>], here, we have focused on the time period of 12âh p.i. to study virus gene expression at the moment of first round progeny virions formation. It was found that 45âmin preincubation of binase at concentration of 10â <i>Î¼</i>g/ml with virus led to decrease in infectious virus titer by 5-fold at an MOI of 0.1 ( <a href="#fig1">Figure 1(a)</a>). At an MOI of 0.01 the reduction of virus titer after binase treatment was more significant (8-fold). The higher antiviral effectiveness of binase on lower virus titer suggests the possible administration of binase at the early phase of virus infection or for prophylaxis. </p> <p>The catalytic activity of binase (100â <i>Î¼</i>g/ml) in A549 cell culture media decreased to 50% after 48âh incubation ( <a href="#fig1">Figure 1(b)</a>) that corresponded to the data obtained earlier for 1â <i>Î¼</i>g/ml binase [ <a href="#B12">13</a>]. Binase is a thermo- and pH-stable agent, which can be used in vivo retaining its activity. The 49% reduction of binase activity in the cell culture fluid can be explained by internalization of the enzyme to the A549 cells. Binase internalization to A549 cell cytoplasm and nucleus was previously illustrated by immunofluorescence methods [ <a href="#B17">17</a>]. Possible inhibition of catalytic activity and partial degradation of binase by extracellular proteases are not excluded though. Thus, binase acts as the antiviral agent under wide range of concentrations after short and long period of incubation. Moreover, the additional binase administration is not needed during 48âh. </p> <p>It is known that binase at concentrations up to 100â <i>Î¼</i>g/ml is not toxic towards A549 cells [ <a href="#B12">13</a>, <a href="#B17">17</a>] while inhibiting H1N1pdm virus propagation [ <a href="#B7">8</a>, <a href="#B12">13</a>]. We have shown here that a concentration of binase of 100â <i>Î¼</i>g/ml without preincubation of the virus (at an MOI of 1) with binase reduced the infectious titer in A549 cell culture by 40% after a single replication round ( <a href="#fig1">Figure 1(c)</a>). Therefore, binase seems to be able to act intracellularly on viruses before they exit the cells. This raises a question regarding antiviral targets of binase in the virus-infected cells. We performed a 2D-DIGE of binase-treated and nontreated cells. It was determined that binase at 100â <i>Î¼</i>g/ml does not alter the spectrum of A549 cellular proteins ( <a href="#fig2">Figure 2(a)</a>). Moreover, the expression of cellular housekeeping genes, measured by mRNA levels of genes encoding structural protein ACTB and metabolic protein GAPDH, was not changed upon treatment of A549 cells with 100â <i>Î¼</i>g/ml of binase followed by 12âh incubation ( <a href="#fig2">Figure 2(b)</a>). At the same time, binase reduced H1N1pdm viral NP mRNA accumulation in the A549 cells infected with virus at an MOI of 1 ( <a href="#fig2">Figure 2(c)</a>), probably, reflecting the specific reduction of viral RNA. Expression of the gene was 20% lower in binase-treated virus-infected cells than in the virus-infected binase nontreated cells. These results support the idea of an inhibitory effect of binase on virus replication ( <a href="#fig1">Figure 1(c)</a>). </p> <p>To gain a more detailed evidence of binase effect on H1N1pdm viral gene expression in A549 cells we used a plasmid based reverse genetic system containing a set of plasmids encoding PB1, PB2, PA, and NP genes of H1N1pdm viral RNP complex and an influenza virus-like RNA encoding a GFP reporter protein (see Sections <a href="#sec2.12">2.12</a> and <a href="#sec2.15">2.15</a>). GFP fluorescence is detectable only if plasmid-encoded genes for the RNP complex and the virus-like RNA are expressed. It was determined that binase reduced the viral RNA expression by 4.4 times indicating the binase inhibition of the expression of viral genes (Figures <a href="#fig3">3(a)</a> and <a href="#fig3">3(b)</a>). Therefore, a target of binase antiviral action could be viral mRNA. Its degradation prevents synthesis of viral proteins. </p> <p>In <a href="#fig4">Figure 4</a>, we have shown a scheme reflecting different possible mechanisms of a binase effect on influenza A viral functional ribonucleoprotein (RNP) complex gene expression using a vRNA-like Pol1-transcript encoding a reporter gene (GFP) step by step starting from internalization of binase (steps A and B) to the step of blocking of viral translation (steps C, D). Steps depicting production of reporter mRNA (7-8) and transcription of viral RNP mRNA (4-5) can be inhibited by binase. </p> <p>Based on the conducted experiments, we can conclude that binase acts on synthesized positive-sense mRNA; however, we do not rule out that binase would digest negative-sense vRNA of H1N1pdm virus or its cRNA. With respect to vRNA it seems less possible, because vRNA is protected by NP proteins and, moreover, its degradation would result in complete elimination of the virus from the cell culture. To confirm the ability of binase to degrade positive-sense RNA of virus, which ignores the internalization and localization in nuclei in order to serve as a template during translation of viral proteins, we used a different system: HeLa cells were infected by positive-sense RNA-containing HRV1A virus. In HeLa cells HRV1A virus is easily propagated; it replicates in HeLa cells to high titers. We show binase is nontoxic for HeLa cells in concentrations under 100â <i>Î¼</i>g/ml ( <a href="#fig5">Figure 5(a)</a>). It reduced the titer of HRV1A virus by 3 times at 100â <i>Î¼</i>g/ml ( <a href="#fig5">Figure 5(b)</a>). The results confirmed the possibility of binase to degrade viral positive-sense RNA without internalization into the nucleus. Thus, regardless of the negative- or positive-sense genome of single stranded RNA viruses, binase might be able to cleave viral RNA catalytically and thereby inhibit the synthesis of the viral proteins. This property of the bacterial enzyme uncovers the promising aspects of its application as a new antiviral agent. </p> <p>To replicate successfully, viruses protect their genomic material from degradation by the host cell. RNA viruses must contend with numerous destabilizing host cell processes including mRNA decay pathways and viral RNA degradation resulting from the antiviral response [ <a href="#B18">18</a>]. However, viruses do not possess specific defense system against bacterial RNases applied exogenously, and viral mRNA remains an important target for antiviral drug discovery. </p> <p>To impair viral RNA transcription, different strategies have been used. For example, siRNA duplexes directed against the Rift Valley fever virus (RVFV) nucleoprotein can effectively inhibit RVFV replication in human (MRC5 cells) and African green monkey cells (Vero E6 cells). It was shown that pretreatment of cells with the nucleoprotein-specific siRNAs markedly reduces the virus titer [ <a href="#B19">19</a>]. Protein-protein interactions of PB1-PB2 subunits play pivotal roles in assembling the functional polymerase complex, which is essential for the replication and transcription of influenza virus RNA. Novel small molecule compounds which inhibited the interaction of RNA-dependent RNA polymerase subunits were identified. Recent efforts of discovery of antiviral drugs targeting RNA have provided drug-like small molecules that inhibit viral replication and include inhibitors of human immunodeficiency virus (HIV), hepatitis C virus (HCV), severe acute respiratory syndrome coronavirus (SARS CoV), and influenza A virus [ <a href="#B20">20</a>]. Highly conserved noncoding RNA (ncRNA) elements in viral genomes and transcripts offer new opportunities to expand the repertoire of drug targets for the development of anti-infective therapy. Viruses for which targeting ncRNA components in the genome or transcripts may be promising include insect-borne flaviviruses (Dengue, Zika, and West Nile) and filoviruses (Ebola and Marburg) [ <a href="#B21">21</a>]. </p> <p>According to our data, both negative-sense and positive-sense RNA-containing viruses can be targeted by binase. The +ssRNA transcript of influenza virus is used as mRNA for the synthesis of structural proteins; +ssRNA of rhinovirus is used both as mRNA and as the packaged genome. The rhinovirus +ssRNA does not need to be transported into the nucleus; the influenza +ssRNA is also located in the cytoplasm. Cytoplasmic allocation of +ssRNAs facilitates their targeting by binase. We have shown previously that massive internalization of binase into nucleus had already started after 3âh of binase incubation with A549 cells [ <a href="#B17">17</a>]. Therefore, we can propose that binase targets âssRNA too. </p> </div> <div class="4.conclusion" title="sec"> <div class="title" tagx="title" title="title">
4. Conclusion</div> <p>We found that binase at nontoxic concentrations to eukaryotic cells does not alter the cellular protein production or mRNA expression but affects viral RNA production. Binase exerts its antiviral effect both against positive- and negative-sense RNA viruses suggesting binase as a promising antiviral agent against different viruses. </p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The work was supported by the KFU, Russia, and JLU Giessen, Germany, partnership program and the program for competitive growth of KFU. This work was also supported by the Russian Science Foundation Grant no. 14-14-00522, by the program "Research Stays for University Academics and Scientists" by the German Academic Exchange Service (DAAD), and in part by the German Centre for Infection Research (DZIF), partner site Giessen, Germany (TTU Emerging Infections to SP). Equipment of the Institute of Medical Virology JLU, Giessen, Germany, and the Interdisciplinary Center for Collective Use (ID RFMEFI59414X0003) sponsored by Ministry of Education and Science of the Russian Federation was used. </p> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
RNase:</div> <div class="def" title="def"> <p>Ribonuclease</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RI:</div> <div class="def" title="def"> <p>RNase inhibitor protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ACTB:</div> <div class="def" title="def"> <p> <i>Î²</i>-Actin </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GAPDH:</div> <div class="def" title="def"> <p>Glyceraldehyde-3-phosphate dehydrogenase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNP:</div> <div class="def" title="def"> <p>Ribonucleoprotein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GFP:</div> <div class="def" title="def"> <p>Green fluorescent protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MOI:</div> <div class="def" title="def"> <p>Multiplicity of infection</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FFU:</div> <div class="def" title="def"> <p>Focus forming units</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
p.i.:</div> <div class="def" title="def"> <p>Postinfection</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IEF:</div> <div class="def" title="def"> <p>Isoelectric focusing</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
+ssRNA:</div> <div class="def" title="def"> <p>Positive-sense single stranded RNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
âssRNA:</div> <div class="def" title="def"> <p>Negative-sense single stranded RNA.</p> </div> </div> </div> </div> <div class="disclosure" title="sec"> <div class="title" tagx="title" title="title">
Disclosure</div> <p>The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </p> </div> <div class="conflictsofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that there are no conflicts of interest regarding the publication of this article. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glanville</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnston</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Challenges in developing a cross-serotype rhinovirus vaccine</span> <span class="source" tagx="source" title="source"> <i>Current Opinion in Virology</i> </span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">83</span> <span class="lpage" tagx="lpage" title="lpage">88</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.coviro.2015.03.004">10.1016/j.coviro.2015.03.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25829255">25829255</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pleschka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaskunas</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engelhardt</span> <span class="given-names" tagx="given-names" title="given-names">O. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">ZÃ¼rcher</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palese</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Sastre</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A plasmid-based reverse genetics system for influenza A virus</span> <span class="source" tagx="source" title="source"> <i>Journal of Virology</i> </span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">4188</span> <span class="lpage" tagx="lpage" title="lpage">4192</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8648766">8648766</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Razonable</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antiviral drugs for viruses other than human immunodeficiency virus</span> <span class="source" tagx="source" title="source"> <i>Mayo Clinic Proceedings</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">1009</span> <span class="lpage" tagx="lpage" title="lpage">1026</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.4065/mcp.2011.0309">10.4065/mcp.2011.0309</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21964179">21964179</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milanoi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ongus</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gachara</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coldren</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bulimo</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Serotype and genetic diversity of human rhinovirus strains that circulated in Kenya in 2008 </span> <span class="source" tagx="source" title="source"> <i>Influenza and Other Respiratory Viruses</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">185</span> <span class="lpage" tagx="lpage" title="lpage">191</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/irv.12373">10.1111/irv.12373</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26822469">26822469</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olszewska</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zambon</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Openshaw</span> <span class="given-names" tagx="given-names" title="given-names">P. J. M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of vaccines against common colds</span> <span class="source" tagx="source" title="source"> <i>British Medical Bulletin</i> </span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">111</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bmb/62.1.99">10.1093/bmb/62.1.99</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12176853">12176853</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glukhov</span> <span class="given-names" tagx="given-names" title="given-names">B. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jerusalimsky</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canter</span> <span class="given-names" tagx="given-names" title="given-names">V. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salganik</span> <span class="given-names" tagx="given-names" title="given-names">R. I.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ribonuclease treatment of tick-borne encephalitis</span> <span class="source" tagx="source" title="source"> <i>Archives of Neurology</i> </span> <span class="year" tagx="year" title="year">1976</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">598</span> <span class="lpage" tagx="lpage" title="lpage">603</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1001/archneur.1976.00500090004002">10.1001/archneur.1976.00500090004002</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/60988">60988</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulyanova</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudkina</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vershinina</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Domann</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic distribution of extracellular guanyl-preferring ribonucleases renews taxonomic status of two <i>Bacillus</i> strains </span> <span class="source" tagx="source" title="source"> <i>Journal of General and Applied Microbiology</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">181</span> <span class="lpage" tagx="lpage" title="lpage">188</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2323/jgam.2016.02.005">10.2323/jgam.2016.02.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27373509">27373509</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ribonucleases as antiviral agents</span> <span class="source" tagx="source" title="source"> <i>Molecular Biology</i> </span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">615</span> <span class="lpage" tagx="lpage" title="lpage">623</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1134/S0026893314040050">10.1134/S0026893314040050</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B8" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Efimova</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mostafa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulyanova</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antiviral activity of bacterial extracellular ribonuclease against single-, double-stranded RNA and DNA containing viruses in cell cultures </span> <span class="source" tagx="source" title="source"> <i>BioNanoScience</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">561</span> <span class="lpage" tagx="lpage" title="lpage">563</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12668-016-0279-9">10.1007/s12668-016-0279-9</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Efimova</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zelenikhin</span> <span class="given-names" tagx="given-names" title="given-names">P. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabirova</span> <span class="given-names" tagx="given-names" title="given-names">M. I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolpakov</span> <span class="given-names" tagx="given-names" title="given-names">A. I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O. N.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Exogenous <i>Bacillus pumilus</i> RNase (binase) suppresses the reproduction of reovirus serotype 1 </span> <span class="source" tagx="source" title="source"> <i>Molecular Biology</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">96</span> <span class="lpage" tagx="lpage" title="lpage">101</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1134/S0026893316060042">10.1134/S0026893316060042</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MÃ¼ller</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulyanova</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pleschka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel antiviral strategy against MERS-CoV and HCoV-229E using binase to target viral genome replication </span> <span class="source" tagx="source" title="source"> <i>BioNanoScience</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">3</span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudkina</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulyanova</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Three-step procedure for preparation of pure <i>Bacillus altitudinis</i> ribonuclease </span> <span class="source" tagx="source" title="source"> <i>FEBS Open Bio</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">24</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/2211-5463.12023">10.1002/2211-5463.12023</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B12" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah Mahmud</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilinskaya</span> <span class="given-names" tagx="given-names" title="given-names">O. N.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antiviral activity of binase against the pandemic influenza A (H1N1) virus</span> <span class="source" tagx="source" title="source"> <i>Acta Naturae</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">19</span> <span class="fpage" tagx="fpage" title="fpage">44</span> <span class="lpage" tagx="lpage" title="lpage">51</span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24455182">24455182</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mostafa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanrai</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ziebuhr</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pleschka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improved dual promotor-driven reverse genetics system for influenza viruses</span> <span class="source" tagx="source" title="source"> <i>Journal of Virological Methods</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">193</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">603</span> <span class="lpage" tagx="lpage" title="lpage">610</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jviromet.2013.07.021">10.1016/j.jviromet.2013.07.021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23886561">23886561</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neumann</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watanabe</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generation of influenza A viruses entirely from cloned cDNAs</span> <span class="source" tagx="source" title="source"> <i>Proceedings of the National Academy of Sciences of the United States of America</i> </span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">96</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">9345</span> <span class="lpage" tagx="lpage" title="lpage">9350</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.96.16.9345">10.1073/pnas.96.16.9345</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10430945">10430945</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Webster</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Unidirectional RNA polymerase I-polymerase II transcription system for the generation of influenza A virus from eight plasmids </span> <span class="source" tagx="source" title="source"> <i>Journal of General Virology</i> </span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2843</span> <span class="lpage" tagx="lpage" title="lpage">2847</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/0022-1317-81-12-2843">10.1099/0022-1317-81-12-2843</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11086114">11086114</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cabrera-Fuentes</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aslam</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saffarzadeh</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Internalization of <i>Bacillus intermedius</i> ribonuclease (BINASE) induces human alveolar adenocarcinoma cell death </span> <span class="source" tagx="source" title="source"> <i>Toxicon</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">219</span> <span class="lpage" tagx="lpage" title="lpage">226</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.toxicon.2013.03.015">10.1016/j.toxicon.2013.03.015</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23567038">23567038</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ullmer</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Semler</span> <span class="given-names" tagx="given-names" title="given-names">B. L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Diverse strategies used by picornaviruses to escape host RNA decay pathways</span> <span class="source" tagx="source" title="source"> <i>Viruses</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">p. E335</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/v8120335">10.3390/v8120335</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faburay</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richt</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Short interfering RNA inhibits rift valley fever virus replication and degradation of protein kinase R in human cells </span> <span class="source" tagx="source" title="source"> <i>Frontiers in Microbiology</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">1889</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2016.01889">10.3389/fmicb.2016.01889</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Screening for novel small molecule inhibitors targeting the assembling of influenza virus polymerase complex by a BiLC-based reporter system </span> <span class="source" tagx="source" title="source"> <i>Journal of Virology</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="issue" tagx="issue" title="issue">5</span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermann</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Small molecules targeting viral RNA</span> <span class="source" tagx="source" title="source"> <i>Wiley Interdisciplinary Reviews: RNA</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">726</span> <span class="lpage" tagx="lpage" title="lpage">743</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/wrna.1373">10.1002/wrna.1373</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27307213">27307213</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>(a) Decrease of H1N1pdm virus titer in A549 cells infected by virus at an MOI of 0.01 and 0.1 which was preincubated with 10â <i>Î¼</i>g/ml binase for 45âmin prior to cell infection; (b) binase catalytic activity in cell-free medium and A549 cell culture medium after 48âh incubation; (c) virus titer after 12âh incubation of A549 cells treated by 100â <i>Î¼</i>g/ml binase and infected by H1N1pdm virus at an MOI of 1. FFU/ml were calculated using supernatant after single-cycle replication of virus. Statistical significance was calculated using GraphPad Prism 5.0 Software and assessed by Student's <i> t</i>-test: <sup> <i>â</i> </sup> <i>P</i> &amp;lt; 0.001. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>(a) Influence of 100â <i>Î¼</i>g/ml binase at 12âh p.i. on proteome of A549 cells. Two-dimensional difference gel electrophoresis (2D-DIGE) of proteins isolated from nontreated ((i), green labeling by Cy3) and binase-treated cells ((ii), red labeling by Cy5); (iii) merged image, unchanged protein spots appeared yellow. High resolution 2D-DIGE images were generated using Typhoon FLA 9500 Imager; samples were separated using narrow range of IEF tube gel, pH 3â8; gels cover 15â150âkDa range. (b) Real-time RT-PCR analysis of mRNA transcripts of housekeeping genes (beta-actin, ACTB; glyceraldehyde-3-phosphate dehydrogenase, GAPDH) in noninfected, binase-treated (100â <i>Î¼</i>g/ml, 12âh), and nontreated A549 cells. (c) H1N1pdm viral NP mRNA in infected nontreated A549 cells (1) and treated by 100â <i>Î¼</i>g/ml binase after 12âh (2). GAPDH mRNA was used as a loading control to normalize viral mRNA accumulation. 10 <sup>6</sup> cells from each sample were used to measure mRNA levels using Roche Light Cycler 480 system and target-specific primers. Statistical significance was calculated using GraphPad Prism 5.0 Software and assessed by Student's <i> t</i>-test: <sup> <i>â</i> </sup> <i>P</i> &amp;lt; 0.01. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Effect of binase on functional H1N1pdm viral RNP polymerase complex formation in A549 cells transfected by plasmids (pHW) encoding vRNP subunits (PB1, PB2, and PA) and NP of H1N1pdm virus together with vRNA-like transcript pPOLI-GFP-RT encoding GFP. Decrease of GFP expression in the cells treated by binase (a) and visualization of this effect using Axiovert 35 inverted fluorescent microscope (b). (i) A549 cells carrying PAM 503 plasmid as GFP expression control; (ii) A549 cells without binase treatment; (iii) A549 cells treated by 10â <i>Î¼</i>g/ml binase after transfections. Statistical significance was calculated using GraphPad Prism 5.0 Software and assessed by Student's <i> t</i>-test: <sup> <i>â</i> </sup> <i>P</i> &amp;lt; 0.001. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Hypothetic scheme of binase influence on functional viral RNP polymerase complex formation monitored by GFP fluorescence in the cells using the influenza A virus mini genome system. For the in vitro reconstitution of viral ribonucleoprotein complex (vRNP), plasmids pHW-PB1-Hamburg, pHW-PB2-Hamburg, pHW-PA-Hamburg, and pHW-NP-Hamburg were used. They carry genes for generating the mRNAs of polymerases and NP segments of H1N1pdm virus. These plasmids were transfected into A549 cells along with pPOLI-GFP-RT plasmid, which generates a vRNA-like POLI-transcript encoding the green fluorescent protein. GFP gene in a latter construct is encoded in negative-sense and flanked by the 3â²- and 5â²-noncoding region of the NS RNA segment of influenza A/WSN/33 virus [ <a href="#B14">2</a>] placed between a truncated human RNA polymerase I promotor (POLI) and hepatitis delta virus ribozyme (R). The expressed subunits of the viral polymerases and nucleoprotein replicate and transcribe the influenza virus-like RNA expressed by pPOLI-GFP-RT into GFP mRNA, resulting in the detection of GFP activity in transfected cells. The described steps are labeled as follows: (1) plasmid constructs encoding vRNP subunits and GFP as a reporter protein, (2-3) transfection and internalization of plasmids into nucleus of A549 cells, (4â6) expression of viral polymerase complex, (7) maturation of GFP transcript by viral polymerase complex, and (8) GFP signaling. Possible binase actions are represented in the steps: (A) binase internalization into cytoplasm, (B) binase internalization into nucleus, (C) and (D) inhibitory effect of binase on the expression of influenza A virus mini genome system in cytoplasm and nucleus, respectively. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Viability of HeLa cells treated by binase (24âh) (a) and the influence of binase on HRV1A virus titer after 24âh incubation of HeLa cells with different concentrations of the RNase. CC <sub>50</sub> and EC <sub>50</sub> are the binase concentration, which induces 50% cell death and 50% reduction of virus titer, respectively. 1â <i>Î¼</i>g/ <i>Î¼</i>L of binase corresponds to 82â <i>Î¼</i>M. The <i> R</i>-squared value and cytotoxic and effective concentrations were calculated using GraphPad Prism 5.0 Software. </p>   </div> </div>  </body></html>