<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/02/2020.02.26.967026.1.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.26.967026v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.26.967026"/>
  <meta name="DC.Date" content="2020-03-02"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at &lt;;https://adapt.sabetilab.org/&gt;;."/>
  <meta name="DC.Contributor" content="Hayden C. Metsky"/>
  <meta name="DC.Contributor" content="Catherine A. Freije"/>
  <meta name="DC.Contributor" content="Tinna-Solveig F. Kosoko-Thoroddsen"/>
  <meta name="DC.Contributor" content="Pardis C. Sabeti"/>
  <meta name="DC.Contributor" content="Cameron Myhrvold"/>
  <meta name="article:published_time" content="2020-03-02"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.26.967026v2"/>
  <meta name="citation_id" content="2020.02.26.967026v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.967026v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.967026v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.967026v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.26.967026"/>
  <meta name="citation_num_pages" content="11"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.26.967026"/>
  <meta name="citation_author" content="Hayden C. Metsky"/>
  <meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology (MIT) and Harvard"/>
  <meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science"/>
  <meta name="citation_author_email" content="hmetsky@broadinstitute.org"/>
  <meta name="citation_author_email" content="cmyhrvol@broadinstitute.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8871-2349"/>
  <meta name="citation_author" content="Catherine A. Freije"/>
  <meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology (MIT) and Harvard"/>
  <meta name="citation_author_institution" content="Program in Virology, Harvard Medical School"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1939-3102"/>
  <meta name="citation_author" content="Tinna-Solveig F. Kosoko-Thoroddsen"/>
  <meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology (MIT) and Harvard"/>
  <meta name="citation_author" content="Pardis C. Sabeti"/>
  <meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology (MIT) and Harvard"/>
  <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University"/>
  <meta name="citation_author_institution" content="Howard Hughes Medical Institute"/>
  <meta name="citation_author_institution" content="Harvard T.H. Chan School of Public Health"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9843-1890"/>
  <meta name="citation_author" content="Cameron Myhrvold"/>
  <meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology (MIT) and Harvard"/>
  <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University"/>
  <meta name="citation_author_email" content="hmetsky@broadinstitute.org"/>
  <meta name="citation_author_email" content="cmyhrvol@broadinstitute.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8971-184X"/>
  <meta name="citation_reference" content="Zhou P, Yang X-L, Wang X-G, Hu B, Zhang L, Zhang W, et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature. 2020. doi:10.1038/s41586-020-2012-7"/>
  <meta name="citation_reference" content="V-Respiratory probe set (2020-01). Available: https://github.com/broadinstitute/catch/tree/master/probe-designs"/>
  <meta name="citation_reference" content="Metsky HC, Siddle KJ, Gladden-Young A, Qu J, Yang DK, Brehio P, et al. Capturing sequence diversity in metagenomes with comprehensive and scalable probe design. Nat Biotechnol. 2019;37: 160–168. doi:10.1038/s41587-018-0006-x"/>
  <meta name="citation_reference" content="Li B, Si H-R, Zhu Y, Yang X-L, Anderson DE, Shi Z-L, et al. Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing. mSphere. 2020;5. doi:10.1128/mSphere.00807-19"/>
  <meta name="citation_reference" content="Wee S-L. As Deaths Mount, China Tries to Speed Up Coronavirus Testing. The New York Times. 9 Feb 2020. Available: https://www.nytimes.com/2020/02/09/world/asia/china-coronavirus-tests.html. Accessed 24 Feb 2020."/>
  <meta name="citation_reference" content="Myhrvold C, Freije CA, Gootenberg JS, Abudayyeh OO, Metsky HC, Durbin AF, et al. Field-deployable viral diagnostics using CRISPR-Cas13. Science. 2018;360: 444–448. doi:10.1126/science.aas8836"/>
  <meta name="citation_reference" content="Wang M, Wu Q, Xu W, Qiao B, Wang J, Zheng H, et al. Clinical diagnosis of 8274 samples with 2019-novel coronavirus in Wuhan. Infectious Diseases (except HIV/AIDS). medRxiv; 2020. doi:10.1101/2020.02.12.20022327"/>
  <meta name="citation_reference" content="Gootenberg JS, Abudayyeh OO, Lee JW, Essletzbichler P, Dy AJ, Joung J, et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science. 2017;356: 438–442. doi:10.1126/science.aam9321"/>
  <meta name="citation_reference" content="Gootenberg JS, Abudayyeh OO, Kellner MJ, Joung J, Collins JJ, Zhang F. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. Science. 2018;360: 439–444. doi:10.1126/science.aaq0179"/>
  <meta name="citation_reference" content="Chen JS, Ma E, Harrington LB, Da Costa M, Tian X, Palefsky JM, et al. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science. 2018;360: 436–439. doi:10.1126/science.aar6245"/>
  <meta name="citation_reference" content="James P. Broughton, Wayne Deng, Clare L. Fasching, Jasmeet Singh, Charles Y. Chiu, Janice S. Chen. A protocol for rapid detection of the 2019 novel coronavirus SARS-CoV-2 using CRISPR diagnostics: SARS-CoV-2 DETECTR. 2020. Available: https://mammoth.bio/wp-content/uploads/2020/02/A-protocol-for-rapid-detection-of-the-2019-novel-coronavirus-SARS-CoV-2-using-CRISPR-diagnostics-SARS-CoV-2-DETECTR.pdf"/>
  <meta name="citation_reference" content="Feng Zhang, Omar O. Abudayyeh, Jonathan S. Gootenberg. A protocol for detection of COVID-19 using CRISPR diagnostics. Available: https://www.broadinstitute.org/files/publications/special/COVID-19%20detection%20(updated).pdf"/>
  <meta name="citation_reference" content="Shu Y, McCauley J. GISAID: Global initiative on sharing all influenza data - from vision to reality. Euro Surveill. 2017;22. doi:10.2807/1560-7917.ES.2017.22.13.30494"/>
  <meta name="citation_reference" content="Hou T, Zeng W, Yang M, Chen W, Ren L, Ai J, et al. Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus. Infectious Diseases (except HIV/AIDS). medRxiv; 2020. doi:10.1101/2020.02.22.20025460"/>
  <meta name="citation_reference" content="Kellner MJ, Koob JG, Gootenberg JS, Abudayyeh OO, Zhang F. SHERLOCK: nucleic acid detection with CRISPR nucleases. Nat Protoc. 2019;14: 2986–3012. doi:10.1038/s41596-019-0210-2"/>
  <meta name="twitter:title" content="CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at &lt;;https://adapt.sabetilab.org/&gt;;."/>
  <meta name="og-title" property="og:title" content="CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.967026v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at &lt;;https://adapt.sabetilab.org/&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-02"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1177437 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at 
     <a href="https://adapt.sabetilab.org/">https://adapt.sabetilab.org/</a>.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-5">A novel 
     <em>Severe acute respiratory syndrome-related coronavirus</em>, SARS-CoV-2 (family: 
     <em>Coronaviridae</em>), is the virus behind a severe outbreak originating in China [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. SARS-CoV-2 surveillance is essential to slowing widespread transmission. In January 2020, we quickly made available a capture enrichment panel [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>] using CATCH [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>] that is aimed at enhancing sequencing of SARS-CoV-2 and other respiratory viruses. Capture has also been important for ongoing SARS-like coronavirus surveillance [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], and the panel’s inclusion of SARS-like bat and pangolin coronaviruses can aid surveillance efforts.
    </p>
    <p id="p-6">There are several challenges associated with surveillance during the current SARS-CoV-2 outbreak. First, high case counts overwhelm diagnostic testing capacity, underscoring the need for a rapid pipeline for sample processing [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Second, SARS-CoV-2 is closely related to other important coronavirus subspecies and species, so detection assays can yield false positives if they are not exquisitely specific to SARS-CoV-2. Third, suspected SARS-CoV-2 patients sometimes have a different respiratory viral infection or have co-infections with SARS-CoV-2 and other respiratory viruses [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Therefore, it is important to characterize these other pathogens, for both patient diagnostics and outbreak response.
    </p>
    <p id="p-7">Here, we help address the challenge of identifying SARS-CoV-2 and the numerous other respiratory viral pathogens by reporting a set of comprehensive design options for 67 species and subspecies for CRISPR-based detection assays. We have not yet experimentally tested most of these designs, instead focusing our efforts so far on extensively testing a point-of-care assay for SARS-CoV-2 using the Cas13-based SHERLOCK technology [
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Using this assay, we demonstrate sensitive detection of synthetic SARS-CoV-2 RNA at 10 copies per microliter.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Designs for single assay and multiplex panels</h3>
     <p id="p-8">We have been developing algorithms and machine learning models for rapidly designing nucleic acid detection assays, linked in a system called ADAPT (manuscript in preparation). The designs satisfy several constraints, including on:</p>
     <ul class="list-unord " id="list-1">
      <li id="list-item-1">
       <p id="p-9">
        <strong>Comprehensiveness</strong>: Assays account for a high fraction of known sequence diversity in their species or subspecies (&gt;;97% for most assays), and are meant to be effective against variable targets.
       </p>
      </li>
      <li id="list-item-2">
       <p id="p-10">
        <strong>Predicted sensitivity</strong>: Assays are predicted by our machine learning model to have high detection activity against the full scope of targeted genomic diversity (here, based on 
        <em>Lwa</em>Cas13a activity only).
       </p>
      </li>
      <li id="list-item-3">
       <p id="p-11">
        <strong>Predicted specificity</strong>: Assays have high predicted specificity to their species or subspecies, factoring in the full extent of known strain diversity, allowing them to be grouped into panels that are accurate in differentiating between related taxa.
       </p>
      </li>
     </ul>
     <p id="p-12">Comprehensiveness and—to some extent—specificity of the designs can be verified 
      <em>in silico</em>.
     </p>
     <p id="p-13">Using ADAPT we designed 67 assays, satisfying the above constraints, to identify: the SARS-related coronavirus species; SARS-CoV-2; two other subspecies in that species with high similarity to SARS-CoV-2; all other known 
      <em>Coronaviridae</em> species, including 4 other species that commonly cause human illness; and other common respiratory viral species and subspecies (
      <strong>
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
      </strong>). Each assay targets a single species or subspecies and can be used individually (e.g., point-of-care detection); additionally, owing to how they are designed, multiple assays can be grouped together to test for multiple targets and distinguish them with high specificity.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1181018/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1181018/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1181018" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-14" class="first-child">A summary of the species and subspecies constituting the 67 designs at 
        <a href="https://adapt.sabetilab.org/">https://adapt.sabetilab.org/</a>. SARS-CoV-2 is designed to exclude detection of the highly similar RaTG13 sequence, and other similar bat and pangolin SARS-like coronaviruses; the SARS-like subspecies includes most bat and pangolin SARS-like coronaviruses.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-15">Sequences for single assays and multiplex panels are available at 
      <strong>
       <a href="https://adapt.sabetilab.org/">https://adapt.sabetilab.org/</a>
      </strong>.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>SARS-CoV-2 SHERLOCK assay testing</h3>
     <p id="p-16">We initially screened a set of 4 designs for SHERLOCK [
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>] assays, output by ADAPT to detect SARS-CoV-2. We identified an assay, which was the best-performing and also our highest ranked design 
      <em>a priori</em>. We extensively tested this assay using a synthetic RNA target and determined the limit of detection to be 10 copies/µl using both fluorescent and lateral flow detection (
      <strong>
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>
      </strong>). This assay performs well in comparison to the recently disclosed DETECTR [
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>] assay (sensitivity: 70–300 cp/µl) [
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] and SHERLOCK assay (10–100 cp/µl) [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] for SARS-CoV-2. A protocol for performing this assay is provided in the Methods section and can be used for testing any of the other designs we have provided.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Testing an assay for SARS-CoV-2 using synthetic RNA targets. We show data from both fluorescent (a) and lateral flow detection (b). Target concentrations (in cp/[[#x3BC]]l) are indicated. RPA NTC: water input into RPA; Det NTC: water input into Cas13 detection reaction. In (a), error bars indicate one standard deviation based on n=3 technical replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060900928" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Testing an assay for SARS-CoV-2 using synthetic RNA targets. We show data from both fluorescent (a) and lateral flow detection (b). Target concentrations (in cp/[[#x3BC]]l) are indicated. RPA NTC: water input into RPA; Det NTC: water input into Cas13 detection reaction. In (a), error bars indicate one standard deviation based on &lt;;em&gt;;n&lt;;/em&gt;;=3 technical replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1/F1.medium.gif" width="440" height="204"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1/F1.medium.gif" width="440" height="204"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.26.967026.1/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-17" class="first-child">Testing an assay for SARS-CoV-2 using synthetic RNA targets. We show data from both fluorescent (a) and lateral flow detection (b). Target concentrations (in cp/μl) are indicated. RPA NTC: water input into RPA; Det NTC: water input into Cas13 detection reaction. In (a), error bars indicate one standard deviation based on 
        <em>n</em>=3 technical replicates.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Discussion</h2>
    <p id="p-18">Ongoing SARS-CoV-2 sequencing is key to developing and monitoring diagnostics and similar surveillance tools. In the case of the SARS-CoV-2 outbreak, genomes have been generated and shared at a remarkable pace, and we thank those who have contributed their data through GISAID [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. We and others, relying on this data [
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], have shown that it is possible to rapidly design CRISPR-based tools for detection and surveillance during an outbreak.
    </p>
    <p id="p-19">Among other goals for this work, we plan to evaluate: (1) sensitivity of the SARS-CoV-2 assay against clinical isolates and patient samples—including sputum, throat, and nasal swabs—some of which may be challenging sample types to test; (2) specificity at both the species and subspecies levels against highly related viruses. For the latter, we intend to use a mixture of synthetic targets reflecting different viral sequences, and patient samples or viral seedstocks when available. We hope that the comprehensiveness and high predicted sensitivity and specificity of our designs will enable many groups to proceed rapidly and successfully from assay testing through deployment.</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Conflicts of interest</h2>
    <p id="p-112">H.C.M., C.A.F., P.C.S., and C.M. are inventors on patent filings related to this work. P.C.S. is a co-founder of, shareholder in, and advisor to Sherlock Biosciences, Inc, as well as a Board member of and shareholder in Danaher Corporation.</p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Methods</h2>
    <div class="section" id="sec-8">
     <h3>SHERLOCK protocol for SARS-CoV-2</h3>
     <div class="section" id="sec-9">
      <h4>Isothermal amplification (RT-RPA)</h4>
      <div class="section" id="sec-10">
       <h5>List of equipment and materials</h5>
       <ul class="list-unord " id="list-3">
        <li id="list-item-94">
         <p id="p-113">Heat block, water bath, or thermocycler, prewarmed to 41 °C</p>
        </li>
        <li id="list-item-95">
         <p id="p-114">RevertAid Reverse Transcriptase (Thermo)</p>
        </li>
        <li id="list-item-96">
         <p id="p-115">RNase inhibitor (NEB Murine)</p>
        </li>
        <li id="list-item-97">
         <p id="p-116">Primer mix @ 5 µM of each primer (see 
          <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Supplementary Table 1</a> for primer sequences)
         </p>
        </li>
        <li id="list-item-98">
         <p id="p-117">Synthetic DNA or RNA target (see 
          <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Supplementary Table 1</a> for sequences), or extracted RNA from a viral seedstock or patient sample
         </p>
        </li>
        <li id="list-item-99">
         <p id="p-118">Rehydration buffer, lyophilized RPA pellets, MgAc @ 280 mM from RPA kit (TwistAmp Basic kit, TwistDx)</p>
        </li>
        <li id="list-item-100">
         <p id="p-119">Nuclease-free water</p>
        </li>
       </ul>
       <div id="T2" class="table pos-float">
        <div class="table-inline table-callout-links">
         <div class="callout">
          <span>View this table:</span>
          <ul class="callout-links">
           <li class="view-inline first">
            <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1181021/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
           </li>
           <li class="view-popup">
            <a href="/highwire/markup/1181021/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
           </li>
           <li class="download-ppt last">
            <a href="/highwire/powerpoint/1181021" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
           </li>
          </ul>
         </div>
        </div>
       </div>
      </div>
      <div class="section" id="sec-11">
       <h5>Step by step protocol</h5>
       <ol class="list-ord " id="list-4">
        <li id="list-item-101">
         <p id="p-121">Determine the number of pellets needed based on the size of the experiment (2 samples per pellet, if doing 20 µl RPA reactions).</p>
        </li>
        <li id="list-item-102">
         <p id="p-122">Make a master mix for N pellets, consisting of Rehydration buffer, RPA primer mix, water, RNase inhibitors, and RevertAid RT. Do not include MgAc at this step.</p>
        </li>
        <li id="list-item-103">
         <p id="p-123">Resuspend each RPA pellet using the total volume for 1 RPA pellet (∼30 µl) of master mix.</p>
        </li>
        <li id="list-item-104">
         <p id="p-124">Add the MgAc to the master mix. Keep the master mix on ice.</p>
        </li>
        <li id="list-item-105">
         <p id="p-125">Aliquot the master mix (containing MgAc) into wells of a 96-well plate or strip tubes pre-chilled on ice. For 20 µl reactions, aliquot 18 µl of master mix.</p>
        </li>
        <li id="list-item-106">
         <p id="p-126">Add sample or a control target to each aliquot of master mix (2 µl, if using 20 µl reactions), mix thoroughly, and incubate at 41 °C for 20 minutes.</p>
        </li>
       </ol>
      </div>
     </div>
     <div class="section" id="sec-12">
      <h4>Cas13 detection</h4>
      <div class="section" id="sec-13">
       <h5>List of equipment and materials</h5>
       <ul class="list-unord " id="list-5">
        <li id="list-item-107">
         <p id="p-128">Heat block, water bath, or thermocycler, prewarmed to 37 °C</p>
        </li>
        <li id="list-item-108">
         <p id="p-129">
          <strong>For visual readout:</strong> camera or cell phone with camera; 
          <strong>for fluorescent readout:</strong> qPCR machine / plate reader capable of detecting FAM
         </p>
        </li>
        <li id="list-item-109">
         <p id="p-130">
          <strong>For fluorescent readout:</strong> optical 96-well plate, optical strip-tubes, or black 96-well plate with clear bottom
         </p>
        </li>
        <li id="list-item-110">
         <p id="p-131">10× Cleavage buffer (1× CB is 40 mM Tris pH 7.5, 1 mM DTT)</p>
        </li>
        <li id="list-item-111">
         <p id="p-132">RNase inhibitors (NEB Murine)</p>
        </li>
        <li id="list-item-112">
         <p id="p-133">Cleavage reporter, 
          <strong>for visual readout:</strong> IDT for lateral flow (sequences in 
          <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Supplementary Table 1</a>); 
          <strong>for fluorescent readout:</strong> RNase Alert v2 (Thermo)
         </p>
        </li>
        <li id="list-item-113">
         <p id="p-134">
          <em>Lwa</em>Cas13 protein @ 0.5 mg/ml, in 16 ul aliquots, diluted in 1× Storage buffer (50 mM Tris pH 7.5, 600 mM NaCl, 5% glycerol, 2 mM DTT) [
          <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]
         </p>
        </li>
        <li id="list-item-114">
         <p id="p-135">Cas13 crRNA @ 2 µM (see 
          <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Supplementary Table 1</a> for sequences)
         </p>
        </li>
        <li id="list-item-115">
         <p id="p-136">T7 RNA polymerase (Lucigen NxGen)</p>
        </li>
        <li id="list-item-116">
         <p id="p-137">rNTP mix @ 25 mM each (NEB)</p>
        </li>
        <li id="list-item-117">
         <p id="p-138">MgCl
          <sub>2</sub> @ 100 mM
         </p>
        </li>
        <li id="list-item-118">
         <p id="p-139">Nuclease-free water</p>
        </li>
       </ul>
       <div id="T3" class="table pos-float">
        <div class="table-inline table-callout-links">
         <div class="callout">
          <span>View this table:</span>
          <ul class="callout-links">
           <li class="view-inline first">
            <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1181024/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
           </li>
           <li class="view-popup">
            <a href="/highwire/markup/1181024/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
           </li>
           <li class="download-ppt last">
            <a href="/highwire/powerpoint/1181024" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
           </li>
          </ul>
         </div>
        </div>
       </div>
      </div>
      <div class="section" id="sec-14">
       <h5>Step by step protocol</h5>
       <ol class="list-ord " id="list-6">
        <li id="list-item-119">
         <p id="p-141">Dilute 
          <em>Lwa</em>Cas13a by adding 110.5 µl of 1× Storage buffer to a 16 µl aliquot of Cas13 protein (prior to dilution @ 0.5 mg/ml).
         </p>
        </li>
        <li id="list-item-120">
         <p id="p-142">Prepare a master mix based on the table above, adding the components in the order they are listed in the table. Do not add RPA product at this step.</p>
        </li>
        <li id="list-item-121">
         <p id="p-143">Aliquot 19 µl of master mix into wells of a 96-well plate or strip tubes pre-chilled on ice. If using a fluorescent readout and depending on the instrument used, an optical PCR or black plate with a clear base can be used.</p>
        </li>
        <li id="list-item-122">
         <p id="p-144">Add 1 µL of RPA product to each master mix aliquot. Seal the plate and incubate at 37 °C for 30 minutes to 3 hours. 
          <strong>For fluorescent readout</strong>, measure fluorescence every 5 minutes. 
          <strong>For visual readout</strong>, see additional details below.
         </p>
        </li>
       </ol>
      </div>
     </div>
     <div class="section" id="sec-15">
      <h4>Visual readout</h4>
      <div class="section" id="sec-16">
       <h5>List of equipment and materials</h5>
       <ul class="list-unord " id="list-7">
        <li id="list-item-123">
         <p id="p-146">HybriDetect 1 lateral flow strips (Milenia)</p>
        </li>
        <li id="list-item-124">
         <p id="p-147">Hybridetect Assay Buffer (Milenia)</p>
        </li>
       </ul>
      </div>
      <div class="section" id="sec-17">
       <h5>Step by step protocol</h5>
       <ol class="list-ord " id="list-8">
        <li id="list-item-125">
         <p id="p-149">After incubation at 37 °C for 30 minutes to 3 hours, add 80 µl of Hybridetect Assay Buffer to the total volume of each detection reaction.</p>
        </li>
        <li id="list-item-126">
         <p id="p-150">Add a lateral flow strip to each well and incubate for 2-5 minutes at room temperature.</p>
         <ol class="list-alphaupper " id="list-9">
          <li id="list-item-127">
           <p id="p-151">Take care to avoid contamination of the strips, by using tweezers to remove individual strips and place in buffer.</p>
          </li>
         </ol>
        </li>
        <li id="list-item-128">
         <p id="p-152">Remove strips, place on flat, well-lit surface, and analyze or acquire images of the strips.</p>
        </li>
       </ol>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Supplementary Information</h2>
    <div id="T4" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1181025/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1181025/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1181025" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Supplementary Table 1.</span>
      <p id="p-153" class="first-child">Primer, target, and crRNA sequences for the SARS-CoV-2 assay.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-20">We thank Brittany Petros, Christopher Tomkins-Tinch, Molly Kemball, and Megan Vodzak for their contributions and help, and other members of the Sabeti lab for helpful conversations related to this work. Funding was provided by HHMI and DARPA D18AC00006.</p>
    <p id="p-21">We also wish to thank the following labs that have generously made available SARS-CoV-2 samples and genome sequences via GISAID:</p>
    <ul class="list-unord " id="list-2">
     <li id="list-item-4">
      <p id="p-22">Amalea Dulcene Nicolasora Research Institute for Tropical Medicine, Molecular Biology Laboratory</p>
     </li>
     <li id="list-item-5">
      <p id="p-23">Arizona Department of Health Services</p>
     </li>
     <li id="list-item-6">
      <p id="p-24">Bamrasnaradura Hospital</p>
     </li>
     <li id="list-item-7">
      <p id="p-25">Beijing Genomics Institute (BGI)</p>
     </li>
     <li id="list-item-8">
      <p id="p-26">Beijing Institute of Microbiology and Epidemiology</p>
     </li>
     <li id="list-item-9">
      <p id="p-27">BGI &amp;; Institute of Microbiology, Chinese Academy of Sciences &amp;; Shandong First Medical University &amp;; Shandong Academy of Medical Sciences &amp;; General Hospital of Central Theater Command of People’s Liberation Army of China</p>
     </li>
     <li id="list-item-10">
      <p id="p-28">California Department of Public Health</p>
     </li>
     <li id="list-item-11">
      <p id="p-29">Centers for Disease Control, R.O.C. (Taiwan)</p>
     </li>
     <li id="list-item-12">
      <p id="p-30">Centre for Infectious Diseases and Microbiology Laboratory Services</p>
     </li>
     <li id="list-item-13">
      <p id="p-31">Charité Universitätsmedizin Berlin, Institute of Virology</p>
     </li>
     <li id="list-item-14">
      <p id="p-32">Chongqing Municipal Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-15">
      <p id="p-33">CNR Virus des Infections Respiratoires - France SUD</p>
     </li>
     <li id="list-item-16">
      <p id="p-34">Collaboration between the University of Melbourne at The Peter Doherty Institute for Infection and Immunity, and the Victorian Infectious Disease Reference Laboratory</p>
     </li>
     <li id="list-item-17">
      <p id="p-35">Department of Disease Control, Ministry of Public Health, Thailand</p>
     </li>
     <li id="list-item-18">
      <p id="p-36">Department of Infectious and Tropical Diseases, Bichat Claude Bernard Hospital, Paris</p>
     </li>
     <li id="list-item-19">
      <p id="p-37">Department of Laboratory Medicine, Lin-Kou Chang Gung Memorial Hospital, Taoyuan, Taiwan.</p>
     </li>
     <li id="list-item-20">
      <p id="p-38">Department of Laboratory Medicine, National Taiwan University Hospital</p>
     </li>
     <li id="list-item-21">
      <p id="p-39">Department of Medical Sciences, Ministry of Public Health, Thailand</p>
     </li>
     <li id="list-item-22">
      <p id="p-40">Department of Microbiology, Guangdong Provincial Center for Diseases Control and Prevention</p>
     </li>
     <li id="list-item-23">
      <p id="p-41">Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-24">
      <p id="p-42">Department of Virology III, National Institute of Infectious Diseases</p>
     </li>
     <li id="list-item-25">
      <p id="p-43">Department of Virology, University of Helsinki and Helsinki University Hospital, Helsinki, Finland</p>
     </li>
     <li id="list-item-26">
      <p id="p-44">Dept. of Pathology, National Institute of Infectious Diseases</p>
     </li>
     <li id="list-item-27">
      <p id="p-45">Division of Viral Diseases, Centers for Disease Control and Prevention</p>
     </li>
     <li id="list-item-28">
      <p id="p-46">Fujian Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-29">
      <p id="p-47">General Hospital of Central Theater Command of People’s Liberation Army of China</p>
     </li>
     <li id="list-item-30">
      <p id="p-48">Guangdong Provincial Center for Diseases Control and Prevention; Guangdong Provincial Public Health</p>
     </li>
     <li id="list-item-31">
      <p id="p-49">Hangzhou Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-32">
      <p id="p-50">Hong Kong Department of Health</p>
     </li>
     <li id="list-item-33">
      <p id="p-51">Hubei Provincial Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-34">
      <p id="p-52">Illinois Department of Public Health Chicago Laboratory</p>
     </li>
     <li id="list-item-35">
      <p id="p-53">INMI Lazzaro Spallanzani IRCCS</p>
     </li>
     <li id="list-item-36">
      <p id="p-54">Institut für Mikrobiologie der Bundeswehr, Munich</p>
     </li>
     <li id="list-item-37">
      <p id="p-55">Institute of Pathogen Biology, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College</p>
     </li>
     <li id="list-item-38">
      <p id="p-56">Institute of Viral Disease Control and Prevention, China CDC</p>
     </li>
     <li id="list-item-39">
      <p id="p-57">Jiangsu Provincial Center for Disease Control &amp;; Prevention</p>
     </li>
     <li id="list-item-40">
      <p id="p-58">Joanna Ina Manalo Research Institute for Tropical Medicine, Molecular Biology Laboratory</p>
     </li>
     <li id="list-item-41">
      <p id="p-59">Korea Centers for Disease Control &amp;; Prevention (KCDC) Center for Laboratory Control of Infectious Diseases Division of Viral Diseases</p>
     </li>
     <li id="list-item-42">
      <p id="p-60">KU Leuven, Clinical and Epidemiological Virology</p>
     </li>
     <li id="list-item-43">
      <p id="p-61">Laboratoire Virpath, CIRI U111, UCBL1, INSERM, CNRS, ENS Lyon</p>
     </li>
     <li id="list-item-44">
      <p id="p-62">Laboratory of Virology, INMI Lazzaro Spallanzani IRCCS</p>
     </li>
     <li id="list-item-45">
      <p id="p-63">Lapland Central Hospital</p>
     </li>
     <li id="list-item-46">
      <p id="p-64">Li Ka Shing Faculty of Medicine, The University of Hong Kong</p>
     </li>
     <li id="list-item-47">
      <p id="p-65">Massachusetts Department of Public Health</p>
     </li>
     <li id="list-item-48">
      <p id="p-66">Microbial Genomics Core Lab, National Taiwan University Centers of Genomic and Precision Medicine</p>
     </li>
     <li id="list-item-49">
      <p id="p-67">Monash Medical Centre</p>
     </li>
     <li id="list-item-50">
      <p id="p-68">National Centre for Infectious Diseases</p>
     </li>
     <li id="list-item-51">
      <p id="p-69">National Influenza Centre, National Public Health Laboratory, Kathmandu, Nepal</p>
     </li>
     <li id="list-item-52">
      <p id="p-70">National Institute for Communicable Disease Control and Prevention (ICDC) Chinese Center for Disease Control and Prevention (China CDC)</p>
     </li>
     <li id="list-item-53">
      <p id="p-71">National Institute for Viral Disease Control &amp;; Prevention, China CDC</p>
     </li>
     <li id="list-item-54">
      <p id="p-72">National Public Health Laboratory, National Centre for Infectious Diseases</p>
     </li>
     <li id="list-item-55">
      <p id="p-73">National Reference Center for Viruses of Respiratory Infections, Institut Pasteur, Paris</p>
     </li>
     <li id="list-item-56">
      <p id="p-74">NHC Key laboratory of Enteric Pathogenic Microbiology, Institute of Pathogenic Microbiology</p>
     </li>
     <li id="list-item-57">
      <p id="p-75">NSW Health Pathology - Institute of Clinical Pathology and Medical Research; Westmead Hospital; University of Sydney</p>
     </li>
     <li id="list-item-58">
      <p id="p-76">Pamela Youde Nethersole Eastern Hospital</p>
     </li>
     <li id="list-item-59">
      <p id="p-77">Pathogen Discovery, Respiratory Viruses Branch, Division of Viral Diseases, Centers for Disease Control and Prevention</p>
     </li>
     <li id="list-item-60">
      <p id="p-78">Pathogen Genomics Center, National Institute of Infectious Diseases</p>
     </li>
     <li id="list-item-61">
      <p id="p-79">Pathology Queensland</p>
     </li>
     <li id="list-item-62">
      <p id="p-80">Prince of Wales Hospital (PWH)</p>
     </li>
     <li id="list-item-63">
      <p id="p-81">Princess Margaret Hospital (PMH)</p>
     </li>
     <li id="list-item-64">
      <p id="p-82">Programme in Emerging Infectious Diseases, Duke-NUS Medical School</p>
     </li>
     <li id="list-item-65">
      <p id="p-83">Providence Regional Medical Center</p>
     </li>
     <li id="list-item-66">
      <p id="p-84">Public Health Virology Laboratory</p>
     </li>
     <li id="list-item-67">
      <p id="p-85">Queen Elizabeth Hospital (QEH)</p>
     </li>
     <li id="list-item-68">
      <p id="p-86">Queen Mary Hospital (QMH)</p>
     </li>
     <li id="list-item-69">
      <p id="p-87">Respiratory Virus Unit, Microbiology Services Colindale, Public Health England</p>
     </li>
     <li id="list-item-70">
      <p id="p-88">Serology, Virology and OTDS Laboratories (SAViD), NSW Health Pathology Randwick</p>
     </li>
     <li id="list-item-71">
      <p id="p-89">Shenzhen Key Laboratory of Pathogen and Immunity, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</p>
     </li>
     <li id="list-item-72">
      <p id="p-90">Singapore General Hospital, Molecular Laboratory, Division of Pathology</p>
     </li>
     <li id="list-item-73">
      <p id="p-91">Sorbonne Université, Inserm et Assistance Publique-Hôpitaux de Paris (Pitié Salpêtrière)</p>
     </li>
     <li id="list-item-74">
      <p id="p-92">South China Agricultural University</p>
     </li>
     <li id="list-item-75">
      <p id="p-93">State Key Laboratory of Virology, Wuhan University</p>
     </li>
     <li id="list-item-76">
      <p id="p-94">Taiwan Centers for Disease Control</p>
     </li>
     <li id="list-item-77">
      <p id="p-95">Texas Department of State Health Services</p>
     </li>
     <li id="list-item-78">
      <p id="p-96">Thai Red Cross Emerging Infectious Diseases - Health Science Centre</p>
     </li>
     <li id="list-item-79">
      <p id="p-97">The University of Hong Kong - Shenzhen Hospital</p>
     </li>
     <li id="list-item-80">
      <p id="p-98">Tuen Mun Hospital (TMH)</p>
     </li>
     <li id="list-item-81">
      <p id="p-99">Unit for Laboratory Development and Technology Transfer, Public Health Agency of Sweden</p>
     </li>
     <li id="list-item-82">
      <p id="p-100">United Christian Hospital</p>
     </li>
     <li id="list-item-83">
      <p id="p-101">Virology Laboratory National Institute for Infectious Diseases ‘Lazzaro Spallanzani’ IRCCS</p>
     </li>
     <li id="list-item-84">
      <p id="p-102">Virology Laboratory, INMI L. Spallanzani</p>
     </li>
     <li id="list-item-85">
      <p id="p-103">Virology Unit, Institut Pasteur du Cambodge</p>
     </li>
     <li id="list-item-86">
      <p id="p-104">Washington State Department of Health</p>
     </li>
     <li id="list-item-87">
      <p id="p-105">Wisconsin Department of Health Services</p>
     </li>
     <li id="list-item-88">
      <p id="p-106">Wuhan Fourth Hospital</p>
     </li>
     <li id="list-item-89">
      <p id="p-107">Wuhan Institute of Virology, Chinese Academy of Sciences</p>
     </li>
     <li id="list-item-90">
      <p id="p-108">Wuhan Jinyintan Hospital</p>
     </li>
     <li id="list-item-91">
      <p id="p-109">Yongchuan District Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-92">
      <p id="p-110">Zhejiang Provincial Center for Disease Control and Prevention</p>
     </li>
     <li id="list-item-93">
      <p id="p-111">Zhongxian Center for Disease Control and Prevention</p>
     </li>
    </ul>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-3">Amended title; revised language around diagnostics to more accurately reflect current state; fixed issue with links.</p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-2">
      <p id="p-4">
       <a href="https://adapt.sabetilab.org">https://adapt.sabetilab.org</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.1" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X-G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2020</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015507&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.26.967026.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.26.967026v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">V-Respiratory probe set</span> (2020-01). Available: 
         <a href="https://github.com/broadinstitute/catch/tree/master/probe-designs">https://github.com/broadinstitute/catch/tree/master/probe-designs</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.3" data-doi="10.1038/s41587-018-0006-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Metsky</span>
          <span class="cit-name-given-names">HC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Siddle</span>
          <span class="cit-name-given-names">KJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gladden-Young</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qu</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">DK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brehio</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Capturing sequence diversity in metagenomes with comprehensive and scalable probe design</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">37</span>: 
         <span class="cit-fpage">160</span>–
         <span class="cit-lpage">168</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41587-018-0006-x</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D37%26rft.spage%253D160%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-018-0006-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41587-018-0006-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.4" data-doi="10.1128/mSphere.00807-19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>
          <span class="cit-name-given-names">H-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z-L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">mSphere</abbr>. 
         <span class="cit-pub-date">2020;5</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/mSphere.00807-19</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.stitle%253DmSphere%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De00807-19%26rft.epage%253De00807-19%26rft.atitle%253DDiscovery%2Bof%2BBat%2BCoronaviruses%2Bthrough%2BSurveillance%2Band%2BProbe%2BCapture-Based%2BNext-Generation%2BSequencing%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSphere.00807-19%26rft_id%253Dinfo%253Apmid%252F31996413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibXNwaCI7czo1OiJyZXNpZCI7czoxMzoiNS8xL2UwMDgwNy0xOSI7czo0OiJhdG9tIjtzOjUwOiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjYuOTY3MDI2LjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.26.967026v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Wee S-L. As Deaths</span>
          <span class="cit-name-surname">Mount</span>
         </span>, 
         <span class="cit-source">China Tries to Speed Up Coronavirus Testing. The New York Times</span>. 
         <span class="cit-vol">9</span>
         <span class="cit-month">Feb</span>
         <span class="cit-pub-date">2020</span>. Available: 
         <a href="https://www.nytimes.com/2020/02/09/world/asia/china-coronavirus-tests.html">https://www.nytimes.com/2020/02/09/world/asia/china-coronavirus-tests.html</a>. 
         <span class="cit-date-in-citation">Accessed 24 Feb 2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.6" data-doi="10.1126/science.aas8836">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Myhrvold</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Freije</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gootenberg</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abudayyeh</span>
          <span class="cit-name-given-names">OO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Metsky</span>
          <span class="cit-name-given-names">HC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">AF</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Field-deployable viral diagnostics using CRISPR-Cas13</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">360</span>: 
         <span class="cit-fpage">444</span>–
         <span class="cit-lpage">448</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aas8836</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyhrvold%26rft.auinit1%253DC.%26rft.volume%253D360%26rft.issue%253D6387%26rft.spage%253D444%26rft.epage%253D448%26rft.atitle%253DField-deployable%2Bviral%2Bdiagnostics%2Busing%2BCRISPR-Cas13%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aas8836%26rft_id%253Dinfo%253Apmid%252F29700266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM4Ny80NDQiO3M6NDoiYXRvbSI7czo1MDoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjI2Ljk2NzAyNi4xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.7" data-doi="10.1101/2020.02.12.20022327">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qiao</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical diagnosis of 8274 samples with 2019-novel coronavirus in Wuhan</span>. 
         <abbr class="cit-jnl-abbrev">Infectious Diseases (except HIV/AIDS). medRxiv</abbr>; 
         <span class="cit-pub-date">2020</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.12.20022327</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfectious%2BDiseases%2B%2528except%2BHIV%252FAIDS%2529.%2BmedRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.12.20022327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMC4wMi4xMi4yMDAyMjMyN3YyIjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4yNi45NjcwMjYuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.8" data-doi="10.1126/science.aam9321">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gootenberg</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abudayyeh</span>
          <span class="cit-name-given-names">OO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">JW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Essletzbichler</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dy</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joung</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Nucleic acid detection with CRISPR-Cas13a/C2c2</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">356</span>: 
         <span class="cit-fpage">438</span>–
         <span class="cit-lpage">442</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aam9321</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam9321%26rft_id%253Dinfo%253Apmid%252F28408723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTYvNjMzNi80MzgiO3M6NDoiYXRvbSI7czo1MDoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjI2Ljk2NzAyNi4xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.9" data-doi="10.1126/science.aaq0179">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gootenberg</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abudayyeh</span>
          <span class="cit-name-given-names">OO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kellner</span>
          <span class="cit-name-given-names">MJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joung</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Collins</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">360</span>: 
         <span class="cit-fpage">439</span>–
         <span class="cit-lpage">444</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aaq0179</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaq0179%26rft_id%253Dinfo%253Apmid%252F29449508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM4Ny80MzkiO3M6NDoiYXRvbSI7czo1MDoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjI2Ljk2NzAyNi4xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.10" data-doi="10.1126/science.aar6245">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harrington</span>
          <span class="cit-name-given-names">LB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Da Costa</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palefsky</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">360</span>: 
         <span class="cit-fpage">436</span>–
         <span class="cit-lpage">439</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aar6245</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aar6245%26rft_id%253Dinfo%253Apmid%252F29449511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM4Ny80MzYiO3M6NDoiYXRvbSI7czo1MDoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjI2Ljk2NzAyNi4xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.26.967026v2.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">James P.</span>
          <span class="cit-name-surname">Broughton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wayne</span>
          <span class="cit-name-surname">Deng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Clare L.</span>
          <span class="cit-name-surname">Fasching</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jasmeet</span>
          <span class="cit-name-surname">Singh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Charles Y.</span>
          <span class="cit-name-surname">Chiu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Janice</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-source">Chen. A protocol for rapid detection of the 2019 novel coronavirus SARS-CoV-2 using CRISPR diagnostics: SARS-CoV-2 DETECTR</span>. 
         <span class="cit-pub-date">2020</span>. Available: 
         <a href="https://mammoth.bio/wp-content/uploads/2020/02/A-protocol-for-rapid-detection-of-the-2019-novel-coronavirus-SARS-CoV-2-using-CRISPR-diagnostics-SARS-CoV-2-DETECTR.pdf">https://mammoth.bio/wp-content/uploads/2020/02/A-protocol-for-rapid-detection-of-the-2019-novel-coronavirus-SARS-CoV-2-using-CRISPR-diagnostics-SARS-CoV-2-DETECTR.pdf</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.26.967026v2.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Feng</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Omar O.</span>
          <span class="cit-name-surname">Abudayyeh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jonathan S.</span>
          <span class="cit-name-surname">Gootenberg</span>
         </span>. 
         <span class="cit-source">A protocol for detection of COVID-19 using CRISPR diagnostics</span>. Available: 
         <a href="https://www.broadinstitute.org/files/publications/special/COVID-19%20detection%20(updated).pdf">https://www.broadinstitute.org/files/publications/special/COVID-19%20detection%20(updated).pdf</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.13" data-doi="10.2807/1560-7917.ES.2017.22.13.30494">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCauley</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data - from vision to reality</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">22</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.2807/1560-7917.ES.2017.22.13.30494</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft.volume%253D22%26rft.spage%253Dpii30494%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2017.22.13.30494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2017.22.13.30494&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.14" data-doi="10.1101/2020.02.22.20025460">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hou</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ai</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Infectious Diseases (except HIV/AIDS). medRxiv</abbr>; 
         <span class="cit-pub-date">2020</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.22.20025460</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfectious%2BDiseases%2B%2528except%2BHIV%252FAIDS%2529.%2BmedRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.22.20025460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMC4wMi4yMi4yMDAyNTQ2MHYyIjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4yNi45NjcwMjYuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.967026v2.15" data-doi="10.1038/s41596-019-0210-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kellner</span>
          <span class="cit-name-given-names">MJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koob</span>
          <span class="cit-name-given-names">JG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gootenberg</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abudayyeh</span>
          <span class="cit-name-given-names">OO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">SHERLOCK: nucleic acid detection with CRISPR nucleases</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">14</span>: 
         <span class="cit-fpage">2986</span>–
         <span class="cit-lpage">3012</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41596-019-0210-2</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D14%26rft.spage%253D2986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41596-019-0210-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41596-019-0210-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
