<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/26/2020.03.26.009803.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.26.009803v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="The potential SARS-CoV-2 entry inhibitor"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.26.009803"/>
  <meta name="DC.Date" content="2020-03-26"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of −991.21 kcal/mol and estimated ΔG of −8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study."/>
  <meta name="DC.Contributor" content="Jrhau Lung"/>
  <meta name="DC.Contributor" content="Yu-Shih Lin"/>
  <meta name="DC.Contributor" content="Yao-Hsu Yang"/>
  <meta name="DC.Contributor" content="Yu-Lun Chou"/>
  <meta name="DC.Contributor" content="Geng-He Chang"/>
  <meta name="DC.Contributor" content="Ming-Shao Tsai"/>
  <meta name="DC.Contributor" content="Cheng-Ming Hsu"/>
  <meta name="DC.Contributor" content="Reming-Albert Yeh"/>
  <meta name="DC.Contributor" content="Li-Hsin Shu"/>
  <meta name="DC.Contributor" content="Yu-Ching Cheng"/>
  <meta name="DC.Contributor" content="Hung Te Liu"/>
  <meta name="DC.Contributor" content="Ching-Yuan Wu"/>
  <meta name="article:published_time" content="2020-03-26"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="The potential SARS-CoV-2 entry inhibitor"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of -991.21 kcal/mol and estimated ΔG of -8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.26.009803v1"/>
  <meta name="citation_id" content="2020.03.26.009803v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009803v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009803v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009803v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.26.009803"/>
  <meta name="citation_num_pages" content="22"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.26.009803"/>
  <meta name="citation_author" content="Jrhau Lung"/>
  <meta name="citation_author_institution" content="Department of Research and Development, Chiayi Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Yu-Shih Lin"/>
  <meta name="citation_author_institution" content="Department of Pharmacy, Chiayi Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Yao-Hsu Yang"/>
  <meta name="citation_author_institution" content="Department of Traditional Chinese Medicine, Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Health Information and Epidemiology Laboratory of Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="School of Chinese medicine, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8080-0504"/>
  <meta name="citation_author" content="Yu-Lun Chou"/>
  <meta name="citation_author_institution" content="Department of surgery, Kaohsiung Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Geng-He Chang"/>
  <meta name="citation_author_institution" content="Department of Otolaryngology, Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Center of Excellence for Chang Gung Research Datalink, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Ming-Shao Tsai"/>
  <meta name="citation_author_institution" content="Department of Otolaryngology, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Cheng-Ming Hsu"/>
  <meta name="citation_author_institution" content="Department of Otolaryngology, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Reming-Albert Yeh"/>
  <meta name="citation_author_institution" content="Department of Otolaryngology, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Li-Hsin Shu"/>
  <meta name="citation_author_institution" content="Department of Traditional Chinese Medicine, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Yu-Ching Cheng"/>
  <meta name="citation_author_institution" content="Department of Traditional Chinese Medicine, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Hung Te Liu"/>
  <meta name="citation_author_institution" content="Department of Traditional Chinese Medicine, Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Ching-Yuan Wu"/>
  <meta name="citation_author_institution" content="Department of Traditional Chinese Medicine, Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="School of Chinese medicine, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_email" content="smbepigwu77@gmail.com"/>
  <meta name="citation_author_email" content="smbepig@adm.cgmh.org.tw"/>
  <meta name="citation_reference" content="Huang C, Wang Y, Li X, Ren L, Zhao J, Hu Y, et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet. 2020;395(10223):497–506."/>
  <meta name="citation_reference" content="Dey SK, Rahman MM, Siddiqi UR, Howlader A. Analyzing the Epidemiological Outbreak of COVID-19: A Visual Exploratory Data Analysis (EDA) Approach. J Med Virol. 2020."/>
  <meta name="citation_reference" content="Lai CC, Shih TP, Ko WC, Tang HJ, Hsueh PR. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronavirus disease-2019 (COVID-19): The epidemic and the challenges. Int J Antimicrob Agents. 2020;55(3):105924."/>
  <meta name="citation_reference" content="Walls AC, Park YJ, Tortorici MA, Wall A, McGuire AT, Veesler D. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell. 2020."/>
  <meta name="citation_reference" content="Yan R, Zhang Y, Li Y, Xia L, Guo Y, Zhou Q. Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science. 2020."/>
  <meta name="citation_reference" content="Hoffmann M, Kleine-Weber H, Schroeder S, Kruger N, Herrler T, Erichsen S, et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell. 2020."/>
  <meta name="citation_reference" content="Liu W, Morse JS, Lalonde T, Xu S. Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV. Chembiochem. 2020."/>
  <meta name="citation_reference" content="Hwang WC, Lin Y, Santelli E, Sui J, Jaroszewski L, Stec B, et al. Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. The Journal of biological chemistry. 2006;281(45):34610–6."/>
  <meta name="citation_reference" content="Lung J, Lin YS, Yang YH, Chou YL, Shu LH, Cheng YC, et al. The potential chemical structure of anti-SARS-CoV-2 RNA-dependent RNA polymerase. J Med Virol. 2020."/>
  <meta name="citation_reference" content="Eswar N, Eramian D, Webb B, Shen MY, Sali A. Protein structure modeling with MODELLER. Methods Mol Biol. 2008;426:145–59."/>
  <meta name="citation_reference" content="Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, et al. UCSF Chimera--a visualization system for exploratory research and analysis. J Comput Chem. 2004;25(13):1605–12."/>
  <meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic acids research. 2018;46(W1):W296–W303."/>
  <meta name="citation_reference" content="Grosdidier A, Zoete V, Michielin O. EADock: docking of small molecules into protein active sites with a multiobjective evolutionary optimization. Proteins. 2007;67(4):1010–25."/>
  <meta name="citation_reference" content="Kim JY, Kim YI, Park SJ, Kim IK, Choi YK, Kim SH. Safe, high-throughput screening of natural compounds of MERS-CoV entry inhibitors using a pseudovirus expressing MERS-CoV spike protein. Int J Antimicrob Agents. 2018;52(5):730–2."/>
  <meta name="citation_reference" content="Zhang DH, Wu KL, Zhang X, Deng SQ, Peng B. In silico screening of Chinese herbal medicines with the potential to directly inhibit 2019 novel coronavirus. J Integr Med. 2020."/>
  <meta name="twitter:title" content="The potential SARS-CoV-2 entry inhibitor"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of −991.21 kcal/mol and estimated ΔG of −8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study."/>
  <meta name="og-title" property="og:title" content="The potential SARS-CoV-2 entry inhibitor"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009803v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of −991.21 kcal/mol and estimated ΔG of −8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-26"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>The potential SARS-CoV-2 entry inhibitor | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1213246 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-2">Outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of −991.21 kcal/mol and estimated ΔG of −8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan, China, induced by SARS-CoV-2. Human-to-human spread rapidly to the whole world triggering a global public health emergency (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). In coronaviruses, spike protein on the surface envelop of SARS-CoV-2 is responsible for promoting viral entry into the host cells (
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). The spike protein is composed of two functional subunits: the S1 subunit that binds to the host cell receptors and the S2 subunit that mediates the fusion of the viral and host cellular membranes. The distal part of the S1 subunit contains the receptor binding domain (RBD) that directly binds to the peptidase domain of angiotensin converting enzyme 2 (ACE2)(
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, 
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). The directly contact area between RBD and ACE2 included several regions. Gln498, Thr500, Asn501 residues of the amino terminus (N) of RBD form several hydrogen bonds with the Tyr41, Gln42, Lys353, and Arg357 residues of ACE2. Lys417 and Tyr453 residues of RBD can interact with the Asp30 and His34 residues of ACE2, respectively. Gln474 in the carboxyl terminus (C) of RBD forms hydrogen bond with Gln24 of ACE2. In addition, Phe486 of RBD forms van der Waals forces with Met82 of ACE24 (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>)(
     <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>). However, the sequence identity of the spike protein between SARS-CoV-2 and SARS-CoV is 76%, and major variation exists at the N-terminus encoding the RBD(
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). Since RBD of spike protein is surface-exposed and promotes entry into host cells, it could be a potential target for therapy and vaccination using small molecules and neutralizing antibodies to treat COVID-19 (
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>).
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) A modeled structure of SARS-CoV-2 spike protein. Orange molecule represent crystallographic binding contact surface of RBD in SARS-CoV-2. Read molecule represent crystallographic directly binding amino acid residues of RBD in SARS-CoV-2. Grid box size for binding site is used by Modeller. The contact area and grid box size (light yellow) is showed for binding site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1617444164" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;(&lt;;strong&gt;;A&lt;;/strong&gt;;) A modeled structure of SARS-CoV-2 spike protein. Orange molecule represent crystallographic binding contact surface of RBD in SARS-CoV-2. Read molecule represent crystallographic directly binding amino acid residues of RBD in SARS-CoV-2. Grid box size for binding site is used by Modeller. The contact area and grid box size (light yellow) is showed for binding site.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F1.medium.gif" width="440" height="212"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F1.medium.gif" width="440" height="212"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 1.</span>
      <p id="p-4" class="first-child">(
       <strong>A</strong>) A modeled structure of SARS-CoV-2 spike protein. Orange molecule represent crystallographic binding contact surface of RBD in SARS-CoV-2. Read molecule represent crystallographic directly binding amino acid residues of RBD in SARS-CoV-2. Grid box size for binding site is used by Modeller. The contact area and grid box size (light yellow) is showed for binding site.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-5">In a recent study, we discovered that theaflavin possesses a potential chemical structure of anti-SARS-CoV-2 RNA-dependent RNA polymerase(
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Some Chinese medicinal compounds are also used for prophylaxis of SARS-CoV-2 infection. However, the actual mechanisms and efficiency of theaflavin for inhibiting SARS-CoV-2 entry into host cells are still unclear. Therefore, we used molecular docking to target the RBD of SARS-CoV-2, with the aim of screening these chemical structures of traditional Chinese medicinal compounds demonstrating commonly used against SARS-CoV and identifying alternative potential chemical structures for antiviral therapy.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods and Materials</h2>
    <div id="sec-3" class="subsection">
     <h3>Structure preparation</h3>
     <p id="p-6">The 3D structure of spike protein of SARS-CoV-2 (NCBI Reference Sequence: YP_009724390.1) was generated based on homologous modeling using Modeller(
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>) incorporated within the UCSF Chimera(
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>) and SWISS-MODEL(
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Compound dataset collection</h3>
     <p id="p-7">Eighty-three chemical structures from traditional Chinese medicinal compounds and their similar structures were retrieved from ZINC15 database.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Molecular docking and virtual screening</h3>
     <p id="p-8">We used two molecular docking methods for analysis. First, molecular docking and virtual screening was performed using idock download from Github (
      <a href="https://github.com/HongjianLi/idock">https://github.com/HongjianLi/idock</a>) in a local linux machine. For each structure, nine docking posed were generated and the scores for the best docking poses of each structure were used for ranking by idock. Second, SwissDock server (Swiss Institute of Bioinformatics, University of Lausanne, Switzerland) was used for in silico prediction of the lowest free binding energy. The calculation was running online (accessible from 
      <a href="http://www.swissdock.ch/">http://www.swissdock.ch/</a>) in the Internet browser(
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). The grid box encompassed by Lys417 and Tyr453 of the middle of the bridge, Gln498, Thr500, Asn501 of the N-terminus, Gln474 of C-terminus and surrounding amino acids around contact surface of SARS-CoV-2 RBD was using for ligand docking and virtual screening (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>).
     </p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Result</h2>
    <p id="p-9">Screening of these chemical structures, we discovered that theaflavin (ZINC3978446, 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>) also has a lower idock score in the contact area of RBD in SARS-CoV-2 (−7.95 kcal/mol). The contact modes between theaflavin and RBD of SARS-CoV-2 with the lowest idocking scores are illustrated in 
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>. Regarding contact modes by idock, hydrophobic interactions contribute significantly in binding and additional hydrogen bond was found between theaflavin and SARS-CoV-2 RBD (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a>).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) The structure of theaflavin (ZINC3978446). (B) The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by idock. Their relative distances between amino acid residues and catechin pentabenzoate or theaflavin digallte are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in [[#xC5]]) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1617444164" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; The structure of theaflavin (ZINC3978446). (&lt;;strong&gt;;B&lt;;/strong&gt;;) The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by idock. Their relative distances between amino acid residues and catechin pentabenzoate or theaflavin digallte are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in [[#xC5]]) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F2.medium.gif" width="440" height="273"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F2.medium.gif" width="440" height="273"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 2.</span>
      <p id="p-10" class="first-child">
       <strong>(A)</strong> The structure of theaflavin (ZINC3978446). (
       <strong>B</strong>) The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by idock. Their relative distances between amino acid residues and catechin pentabenzoate or theaflavin digallte are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in Å) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">Because theaflavin has the lower idock score in the contact area of SARS-CoV-2 RBD (−7.95 kcal/mol), we used the SwissDock server to confirm the result. Favorable binding modes were scored based on FullFitness scores and binding energy (Estimated ΔG (kcal/mol)) by the SwissDock server. The FullFitness scores and binding energy results and were obtained from the docking of theaflavin (ZINC3978446) into contact area of SARS-CoV-2 RBD. Theaflavin showed FullFitness scores of −991.21 kcal/mol and estimated ΔG of −8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>). The 3D and 2D contact modes between contact area of SARS-CoV-2 RBD and theaflavin with the lowest binding energy are illustrated in 
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B, C</a> and 
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">D</a>. Regarding the contact modes by SwissDock server, hydrophobic interactions contribute significantly for binding (
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3D</a> and 
     <a id="xref-fig-3-5" class="xref-fig" href="#F3">E</a>). The additional hydrogen bonds hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Clustering results obtained from the docking of theaflavin into RBD by SwissDock service. (B) The molecules represent crystallographic and predicted pose for theaflavin in the pocket of RBD. (C) The hydrogen bonds interaction established by theaflavin with the closest residues of RBD are showed through Protein-Ligand Interaction Profiler (PLIP). (D) The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by SwissDock. Their relative distances between amino acid residues and theaflavin are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in [[#xC5]]) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin. (E) The table of hydrogen bonds between theaflavin and RBD by SwissDock." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1617444164" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Clustering results obtained from the docking of theaflavin into RBD by SwissDock service. &lt;;strong&gt;;(B)&lt;;/strong&gt;; The molecules represent crystallographic and predicted pose for theaflavin in the pocket of RBD. &lt;;strong&gt;;(C)&lt;;/strong&gt;; The hydrogen bonds interaction established by theaflavin with the closest residues of RBD are showed through Protein-Ligand Interaction Profiler (PLIP). &lt;;strong&gt;;(D)&lt;;/strong&gt;; The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by SwissDock. Their relative distances between amino acid residues and theaflavin are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in [[#xC5]]) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin. &lt;;strong&gt;;(E)&lt;;/strong&gt;; The table of hydrogen bonds between theaflavin and RBD by SwissDock.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F3.medium.gif" width="348" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F3.medium.gif" width="348" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.26.009803/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 3.</span>
      <p id="p-12" class="first-child">
       <strong>(A)</strong> Clustering results obtained from the docking of theaflavin into RBD by SwissDock service. 
       <strong>(B)</strong> The molecules represent crystallographic and predicted pose for theaflavin in the pocket of RBD. 
       <strong>(C)</strong> The hydrogen bonds interaction established by theaflavin with the closest residues of RBD are showed through Protein-Ligand Interaction Profiler (PLIP). 
       <strong>(D)</strong> The contact model between theaflavin and SARS-CoV-2 RBD are showed in 2D interaction diagram by SwissDock. Their relative distances between amino acid residues and theaflavin are analyzed and illustrated by LigPolt+. Carbon, oxygen, nitrogen, and fluoride molecules are marked as white, red, blue, and green circles, respectively. Covalent bonds in theaflavin and amino acid residues of RBD are labeled in purple and orange solid lines, respectively. The light blue dot lines label the distance (in Å) of hydrogen bonds formed between the functional moieties of theaflavin and amino acid residues. Hydrophobic interactions between theaflavin and RBD are depicted by the name of involving amino acid residues, which are labeled with dark green with dark red eyelashes pointing to the involved functional moiety of theaflavin. 
       <strong>(E)</strong> The table of hydrogen bonds between theaflavin and RBD by SwissDock.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Discussion</h2>
    <p id="p-13">The previous study reported that dihydrotanshinone I also had inhibitory effects against viral entry and replication in the MERS-CoV through the cell model (
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). In addition, the recent study showed dihydrotanshinone I could inhibit viral entry through binding to the fusion cone of spike protein which is important for viral membrane fusion(
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). The docking binding energy between dihydrotanshinone I and spike protein is −5.16 kcal/mol (
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>). In our previous study, we discovered that theaflavin (ZINC3978446) is a potential chemical structure of anti-SARS-CoV-2 RNA-dependent RNA polymerase(
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>). Moreover, we also found that theaflvin was able to dock in contact area in RBD of SARS-CoV-2. Theaflvin formed hydrophobic interactions and additional hydrogen bonds with the direct contact area of RBD. It is possible that theaflavin could ocuppy the contact area of RBD and block the interaction between ACE2 and RBD of SARS-CoV-2. Nevertheless, the mechanisms of viral entry blockage by dihydrotanshinone I and theaflavin could vary.
    </p>
    <p id="p-14">Our results suggest that theaflavin could be the candidate for prophylaxis or treatment of SARS-CoV-2 infection through RBD targeting. However, the exact in vivo effect remains unclear and further studies are necessary to confirm the effects of theaflavin against SARS-CoV-2 entry.</p>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Declarations</h2>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Availability of data and materials</h2>
    <p id="p-16">All data and materials are contained and described within the manuscript.</p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Authors’ contributions</h2>
    <p id="p-17">J. L. performed the experiments; C.Y.W. conceived the idea and designed experiments and wrote manuscript. Y.H.Y., L.H.S., G.H.C., M.S.T., Y.C.C., H.T.L., Y.S.L. and J.L. analyzed the data. Y.L.C., C.M.H. and R.A.Y. revised English writing of the manuscript. All authors reviewed and approved the final version.</p>
   </div>
   <div class="section COI-statement" id="sec-11">
    <h2 class="">Competing interests</h2>
    <p id="p-18">The authors declare that they have no competing interests.</p>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Consent for publication</h2>
    <p id="p-19">Not applicable.</p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">Ethics approval and consent to participate</h2>
    <p id="p-20">Not applicable.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgement</h2>
    <p id="p-15">This work was supported by grant CMRPG6H0162 from Chang Gung Memorial Hospital, and MOST 108-2320-B-182-021 from Ministry of Science and Technology to Dr. Ching Yuan Wu, and NMRPG6H0041 from Ministry of Science and Technology to Dr. Jrhau Lung. The authors thank the Health Information and Epidemiology Laboratory at the Chiayi Chang Gung Memorial Hospital for the comments and assistance in data analysis.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.1" data-doi="10.1016/S0140-6736(20)30183-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">395</span>(
         <span class="cit-issue">10223</span>):
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.26.009803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.009803v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dey</span>
          <span class="cit-name-given-names">SK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rahman</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Siddiqi</span>
          <span class="cit-name-given-names">UR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howlader</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Analyzing the Epidemiological Outbreak of COVID-19: A Visual Exploratory Data Analysis (EDA) Approach</span>. 
         <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lai</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shih</span>
          <span class="cit-name-given-names">TP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ko</span>
          <span class="cit-name-given-names">WC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">HJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsueh</span>
          <span class="cit-name-given-names">PR.</span>
         </span>
         <span class="cit-article-title">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronavirus disease-2019 (COVID-19): The epidemic and the challenges</span>. 
         <abbr class="cit-jnl-abbrev">Int J Antimicrob Agents</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">55</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">105924</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BAntimicrob%2BAgents%26rft.volume%253D55%26rft.spage%253D105924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>
          <span class="cit-name-given-names">YJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tortorici</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wall</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McGuire</span>
          <span class="cit-name-given-names">AT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Veesler</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">Q.</span>
         </span>
         <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffmann</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kleine-Weber</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schroeder</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kruger</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Herrler</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Erichsen</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morse</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lalonde</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV</span>. 
         <abbr class="cit-jnl-abbrev">Chembiochem</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.8" data-doi="10.1074/jbc.M603275200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hwang</span>
          <span class="cit-name-given-names">WC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Santelli</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sui</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jaroszewski</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stec</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of biological chemistry</abbr>. 
         <span class="cit-pub-date">2006</span>;
         <span class="cit-vol">281</span>(
         <span class="cit-issue">45</span>):
         <span class="cit-fpage">34610</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bbiological%2Bchemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAzLjI2LjAwOTgwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lung</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">YS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">YH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chou</span>
          <span class="cit-name-given-names">YL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shu</span>
          <span class="cit-name-given-names">LH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">YC</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The potential chemical structure of anti-SARS-CoV-2 RNA-dependent RNA polymerase</span>. 
         <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.10" data-doi="10.1007/978-1-60327-058-8_8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eswar</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eramian</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Webb</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>
          <span class="cit-name-given-names">MY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sali</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Protein structure modeling with MODELLER</span>. 
         <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>. 
         <span class="cit-pub-date">2008</span>;
         <span class="cit-vol">426</span>:
         <span class="cit-fpage">145</span>–
         <span class="cit-lpage">59</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEswar%26rft.auinit1%253DN.%26rft.volume%253D426%26rft.spage%253D145%26rft.epage%253D159%26rft.atitle%253DProtein%2Bstructure%2Bmodeling%2Bwith%2BMODELLER.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-058-8_8%26rft_id%253Dinfo%253Apmid%252F18542861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/978-1-60327-058-8_8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18542861&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.26.009803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.11" data-doi="10.1002/jcc.20084">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pettersen</span>
          <span class="cit-name-given-names">EF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goddard</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Couch</span>
          <span class="cit-name-given-names">GS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greenblatt</span>
          <span class="cit-name-given-names">DM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meng</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Chem</abbr>. 
         <span class="cit-pub-date">2004</span>;
         <span class="cit-vol">25</span>(
         <span class="cit-issue">13</span>):
         <span class="cit-fpage">1605</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15264254&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.26.009803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000223379100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.12" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertoni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bienert</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Studer</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tauriello</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gumienny</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>(
         <span class="cit-issue">W1</span>):
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">W303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.26.009803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.13" data-doi="10.1002/prot.21367">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grosdidier</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zoete</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michielin</span>
          <span class="cit-name-given-names">O.</span>
         </span>
         <span class="cit-article-title">EADock: docking of small molecules into protein active sites with a multiobjective evolutionary optimization</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>. 
         <span class="cit-pub-date">2007</span>;
         <span class="cit-vol">67</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">1010</span>–
         <span class="cit-lpage">25</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DGrosdidier%26rft.auinit1%253DA.%26rft.volume%253D67%26rft.issue%253D4%26rft.spage%253D1010%26rft.epage%253D1025%26rft.atitle%253DEADock%253A%2Bdocking%2Bof%2Bsmall%2Bmolecules%2Binto%2Bprotein%2Bactive%2Bsites%2Bwith%2Ba%2Bmultiobjective%2Bevolutionary%2Boptimization.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.21367%26rft_id%253Dinfo%253Apmid%252F17380512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.21367&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17380512&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.26.009803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000246415700021&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">JY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">YI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>
          <span class="cit-name-given-names">SJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">IK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>
          <span class="cit-name-given-names">YK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">SH.</span>
         </span>
         <span class="cit-article-title">Safe, high-throughput screening of natural compounds of MERS-CoV entry inhibitors using a pseudovirus expressing MERS-CoV spike protein</span>. 
         <abbr class="cit-jnl-abbrev">Int J Antimicrob Agents</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">52</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">730</span>–
         <span class="cit-lpage">2</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BAntimicrob%2BAgents%26rft.volume%253D52%26rft.spage%253D730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.009803v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">DH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">KL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">SQ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">In silico screening of Chinese herbal medicines with the potential to directly inhibit 2019 novel coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J Integr Med</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
