<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Indian J Med Res</span><span class="iso-abbrev" title="iso-abbrev">Indian J. Med. Res</span><span class="publisher-id" title="publisher-id">IJMR</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">The Indian Journal of Medical Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0971-5916</span><span class="issn-epub" tagx="issn" title="issn-epub">0975-9174</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Wolters Kluwer - Medknow</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">India</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7258754">7258754</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32242875">32242875</a></span><span class="publisher-id" title="publisher-id">IJMR-151-216</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.4103/ijmr.IJMR_594_20">10.4103/ijmr.IJMR_594_20</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Laboratory preparedness for SARS-CoV-2 testing in India: Harnessing a network of Virus Research &amp;amp; Diagnostic Laboratories</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">Nivedita</span></span><a href="#aff1">1</a><a href="#fn1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potdar</span><span class="given-names" tagx="given-names" title="given-names">Varsha</span></span><a href="#aff4">4</a><a href="#fn1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Praharaj</span><span class="given-names" tagx="given-names" title="given-names">Ira</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giri</span><span class="given-names" tagx="given-names" title="given-names">Sidhartha</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sapkal</span><span class="given-names" tagx="given-names" title="given-names">Gajanan</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">Pragya</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudhary</span><span class="given-names" tagx="given-names" title="given-names">Manohar Lal</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dar</span><span class="given-names" tagx="given-names" title="given-names">Lalit</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugunan</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span><a href="#aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">Harmanmeet</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munivenkatappa</span><span class="given-names" tagx="given-names" title="given-names">Ashok</span></span><a href="#aff9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shastri</span><span class="given-names" tagx="given-names" title="given-names">Jayanthi</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaveri</span><span class="given-names" tagx="given-names" title="given-names">Krishnasamy</span></span><a href="#aff11">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutta</span><span class="given-names" tagx="given-names" title="given-names">Shanta</span></span><a href="#aff12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malhotra</span><span class="given-names" tagx="given-names" title="given-names">Bharti</span></span><a href="#aff13">13</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">Amita</span></span><a href="#aff14">14</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagamani</span><span class="given-names" tagx="given-names" title="given-names">Kammilli</span></span><a href="#aff15">15</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shantala</span><span class="given-names" tagx="given-names" title="given-names">G.B.</span></span><a href="#aff10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raut</span><span class="given-names" tagx="given-names" title="given-names">Sharmila</span></span><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vegad</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><a href="#aff16">16</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">Ajanta</span></span><a href="#aff17">17</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudhary</span><span class="given-names" tagx="given-names" title="given-names">Aashish</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brijwal</span><span class="given-names" tagx="given-names" title="given-names">Megha</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balakrishnan</span><span class="given-names" tagx="given-names" title="given-names">Anukumar</span></span><a href="#aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manjunatha</span><span class="given-names" tagx="given-names" title="given-names">Jayaswamy</span></span><a href="#aff9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">Manish</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srinivasan</span><span class="given-names" tagx="given-names" title="given-names">Sivasubramanian</span></span><a href="#aff11">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banu</span><span class="given-names" tagx="given-names" title="given-names">Hasina</span></span><a href="#aff12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">Himanshu</span></span><a href="#aff13">13</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">Parul</span></span><a href="#aff14">14</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunita</span><span class="given-names" tagx="given-names" title="given-names">Pakalpati</span></span><a href="#aff15">15</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ambica</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><a href="#aff10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fageria</span><span class="given-names" tagx="given-names" title="given-names">Babita</span></span><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">Disha</span></span><a href="#aff16">16</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajbongshi</span><span class="given-names" tagx="given-names" title="given-names">Gitika</span></span><a href="#aff17">17</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijay</span><span class="given-names" tagx="given-names" title="given-names">Neetu</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narayan</span><span class="given-names" tagx="given-names" title="given-names">Jitendra</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aggarwal</span><span class="given-names" tagx="given-names" title="given-names">Neeraj</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagar</span><span class="given-names" tagx="given-names" title="given-names">Anu</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gangakhedkar</span><span class="given-names" tagx="given-names" title="given-names">Raman R.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">Priya</span></span><a href="#aff18">#</a><a href="#cor1" /></span></div><span class="citation_author_institution" id="aff1">[1], <i>Division of Epidemiology &amp;amp; Communicable Diseases, Indian Council of Medical Research, New Delhi, India</i></span><span class="citation_author_institution" id="aff2">[2], <i>Department of Microbiology, All India Institute of Medical Sciences, New Delhi, India</i></span><span class="citation_author_institution" id="aff3">[3], <i>Department of Health Research, Ministry of Health &amp;amp; Family Welfare, Government of India, New Delhi, India</i></span><span class="citation_author_institution" id="aff4">[4], <i>Influenza Group, ICMR-National Institute of Virology, Pune, India</i></span><span class="citation_author_institution" id="aff5">[5], <i>Maximum Containment Laboratory, ICMR-National Institute of Virology, Pune, India</i></span><span class="citation_author_institution" id="aff6">[6], <i>Department of Microbiology, Kasturba Hospital for Infectious Diseases, Mumbai, India</i></span><span class="citation_author_institution" id="aff7">[7], <i>Department of Microbiology, Indira Gandhi Government Medical College &amp;amp; Hospital, Nagpur, Maharashtra, India</i></span><span class="citation_author_institution" id="aff8">[8], <i>ICMR-National Institute of Virology Kerala Unit, Alappuzha, Kerala, India</i></span><span class="citation_author_institution" id="aff9">[9], <i>ICMR-National Institute of Virology Bangalore Field Unit, Bengaluru, Karnataka, India</i></span><span class="citation_author_institution" id="aff10">[10], <i>Bangalore Medical College &amp;amp; Research Institute, Bengaluru, Karnataka, India</i></span><span class="citation_author_institution" id="aff11">[11], <i>Department of Virology, King Institute of Preventive Medicine &amp;amp; Research, Chennai, Tamil Nadu, India</i></span><span class="citation_author_institution" id="aff12">[12], <i>ICMR-National Institute of Cholera &amp;amp; Enteric Diseases, Kolkata,West Bengal, India</i></span><span class="citation_author_institution" id="aff13">[13], <i>Department of Microbiology, Sawai Man Singh Medical College, Jaipur, Rajasthan, India</i></span><span class="citation_author_institution" id="aff14">[14], <i>Department of Microbiology, King George's Medical University, Lucknow, Uttar Pradesh, India</i></span><span class="citation_author_institution" id="aff15">[15], <i>Department of Microbiology, Gandhi Medical College &amp;amp; Hospital, Secunderabad, Telangana, India</i></span><span class="citation_author_institution" id="aff16">[16], <i>Department of Microbiology, Byramjee Jeejeebhoy Medical College, Ahmedabad, Gujarat, India</i></span><span class="citation_author_institution" id="aff17">[17], <i>Department of Microbiology, Gauhati Medical College &amp;amp; Hospital, Guwahati, Assam, India</i></span><span class="citation_author_institution" id="aff18">[#], <i>ICMR-National Institute of Virology, Pune, India</i></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><i>For correspondence:</i> Dr Priya Abraham, Director, ICMR-National Institute of Virology, Pune 411 001, Maharashtra, India e-mail: <span class="email" tagx="email" title="email">priya.abraham@icmr.gov.in</span>, <span class="email" tagx="email" title="email">director.niv@icmr.gov.in</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="volume" tagx="volume" title="volume">151</span><span class="issue" tagx="issue" title="issue">2-3</span><span class="fpage" tagx="fpage" title="fpage">216</span><span class="lpage" tagx="lpage" title="lpage">225</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access journal, and articles are distributed under the terms of the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 License, which allows others to remix, tweak, and build upon the work non-commercially, as long as appropriate credit is given and the new creations are licensed under the identical terms.</span><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="ijmr-151-216.pdf" title="self-uri">ijmr-151-216.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background&amp;objectives:" title="sec"> <div class="title" tagx="title" title="title">
Background &amp;amp; objectives:</div> <p>An outbreak of respiratory illness of unknown aetiology was reported from Hubei province of Wuhan, People's Republic of China, in December 2019. The outbreak was attributed to a novel coronavirus (CoV), named as severe acute respiratory syndrome (SARS)-CoV-2 and the disease as COVID-19. Within one month, cases were reported from 25 countries. In view of the novel viral strain with reported high morbidity, establishing early countrywide diagnosis to detect imported cases became critical. Here we describe the role of a countrywide network of VRDLs in early diagnosis of COVID-19.</p> </div> <div class="methods:" title="sec"> <div class="title" tagx="title" title="title">
Methods:</div> <p>The Indian Council of Medical Research (ICMR)-National Institute of Virology (NIV), Pune, established screening as well as confirmatory assays for SARS-CoV-2. A total of 13 VRDLs were provided with the <i>E</i> gene screening real-time reverse transcription-polymerase chain reaction (rRT-PCR) assay. VRDLs were selected on the basis of their presence near an international airport/seaport and their past performance. The case definition for testing included all individuals with travel history to Wuhan and symptomatic individuals with travel history to other parts of China. This was later expanded to include symptomatic individuals returning from Singapore, Japan, Hong Kong, Thailand and South Korea.</p> </div> <div class="results:" title="sec"> <div class="title" tagx="title" title="title">
Results:</div> <p>Within a week of standardization of the test at NIV, all VRDLs could initiate testing for SARS-CoV-2. Till February 29, 2020, a total of 2,913 samples were tested. This included both 654 individuals quarantined in the two camps and others fitting within the case definition. The quarantined individuals were tested twice - at days 0 and 14. All tested negative on both occasions. Only three individuals belonging to different districts in Kerala were found to be positive.</p> </div> <div class="interpretation&amp;conclusions:" title="sec"> <div class="title" tagx="title" title="title">
Interpretation &amp;amp; conclusions:</div> <p>Sudden emergence of SARS-CoV-2 and its potential to cause a pandemic posed an unsurmountable challenge to the public health system of India. However, concerted efforts of various arms of the Government of India resulted in a well-coordinated action at each level. India has successfully demonstrated its ability to establish quick diagnosis of SARS-CoV-2 at NIV, Pune, and the testing VRDLs.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
diagnosis</div> <div class="kwd" title="kwd">
preparedness</div> <div class="kwd" title="kwd">
quality control</div> <div class="kwd" title="kwd">
quarantine</div> <div class="kwd" title="kwd">
severe acute respiratory syndrome-CoV-2</div> <div class="kwd" title="kwd">
Virus Research and Diagnostic Laboratory</div> </div> </div> </div> <div class="body" title="body"> <p>Coronaviruses (CoVs) are a group of enveloped viruses with non-segmented positive sense RNA belonging to the family Coronaviridae and the order Nidovirales. On the basis of phylogenetic clustering, they are classified into three different genera: alpha, beta and gamma. While alpha and beta types have mammalian hosts, gamma type CoVs have avian hosts. Alpha- and beta-CoVs are widely distributed in humans and other mammals including bats and cause mild respiratory infections<a href="#ref1">1</a>. However, two beta coronaviruses causing severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) were responsible for widespread epidemics with a case fatality rate of 10 per cent for SARS<a href="#ref2">2</a> and 35 per cent for MERS CoVs<a href="#ref3">3</a>.</p> <p>The World Health Organization (WHO) reported cases of pneumonia of unknown aetiology in Wuhan city, Hubei province of People's Republic of China, on December 31, 2019<a href="#ref4">4</a>. On January 7, 2020, Chinese authorities officially announced that the illness was caused by a novel CoV. The WHO has named the disease as COVID-19<a href="#ref5">5</a>, and based on its similarity to SARS-CoV (2002-2003), the CoV Study Group of the International Committee on Taxonomy of Viruses (ICTV) has named the virus as SARS-CoV-2<a href="#ref6">6</a>. A viral genome sequence was released in public domain on January 10, 2020 (Wuhan-Hu-1, GenBank accession number MN908947<a href="#ref7">7</a>), followed by four other genomes deposited on January 12, in the viral sequence database curated by the Global Initiative on Sharing All Influenza Data (GISAID). The novel beta coronavirus shows 89 and 82 per cent nucleotide identity to bat CoV, CoVZXC21, and SARS-CoV (2002-2003), respectively<a href="#ref8">8</a>. Since its emergence, the disease has rapidly spread to neighbouring provinces of China as well 53 other countries through international travel<a href="#ref9">9</a>. Infection is spread through droplets or prolonged contact with infected patients<a href="#ref10">10</a>.</p> <p>Virus isolate is the gold standard for establishment and standardization of assay performance. Since SARS- CoV-2 virus isolate was not available earlier, based on the genetic sequence of SARS-CoV-2 and closely related SARS-CoV (2002-2003), the WHO shared protocols (<i>E, N, RdRp</i> and <i>S</i> genes) for screening and confirmation of probable cases<a href="#ref11">11</a>.</p> <p>Here, we briefly describe the efforts made by the Government of India (GoI) towards reducing the risk of emergence of COVID-19 in India. We also provide a detailed description of the role of a well-established countrywide network of Virus Research and Diagnostic Laboratories (VRDLs) which could be rapidly enabled to scale up testing capacity for SARS-CoV-2 in different parts of India.</p> <div class="material_methods" title="material|methods"> <div class="title" tagx="title" title="title">
Material &amp;amp; Methods</div> <p><i>Identifying suspected cases and contacts</i>: Screening of passengers returning from China was initiated on January 18, 2020, at seven different airports throughout India and subsequently extended to 21 different airports. Universal thermal screening of all passengers has been made mandatory for all flights from Singapore, Japan and South Korea besides China and Hong Kong. Screening of passengers was also initiated at 12 major seaports and all minor ports in the country to identify crew and passengers travelling back from China and to undertake required measures for isolation if found symptomatic. In addition, the GoI also initiated screening at all integrated checkposts from Nepal in the States of Uttar Pradesh (UP), Uttarakhand, West Bengal, Sikkim and Bihar.</p> <p><i>Individuals evacuated from China and Japan</i>: The GoI evacuated Indian citizens residing in Wuhan and neighbouring cities in the Hubei province of China. Two subsequent evacuations were undertaken on January 31 and February 1, 2020. A total of 654 individuals (including 645 Indians residing in Wuhan, 7 Maldivian nationals and 2 members from the crew) were brought back in dedicated aircrafts. Similar operations were conducted to evacuate 112 individuals from Wuhan (including 76 Indians and 36 people from Madagascar, Maldives, Myanmar, South Africa and the USA) and 124 from Diamond Princess ship, Japan (including 119 Indians and 5 people from Nepal, Peru, South Africa and Sri Lanka) on February 27, 2020. All the individuals were quarantined in two separate facilities close to New Delhi, India. The facilities were managed by the Army and Indo-Tibetan Border Police.</p> <p>Inclusion criteria for testing for COVID-19: The following inclusion criteria were followed for considering samples for COVID-19 testing: (<i>i</i>) Symptomatic (fever, sore throat, running nose, dyspnoea, <i>etc</i>.)/asymptomatic individuals returned from Wuhan, China, after 15 January 2020; (<i>ii</i>) Only symptomatic individuals returned from rest of China, which was subsequently expanded to include Hong Kong, Japan, South Korea, Singapore, Iran and Italy, were also included; (<i>iii</i>) Close contacts of confirmed positive cases of COVID-19 infection; and (<i>iv</i>) All individuals evacuated from Wuhan, China, and Diamond Princess ship, Japan, in four different operations.</p> <p>Inclusion criteria for laboratory testing are evolving in India, given the increased number of cases of COVID-19 being reported from different countries<a href="#ref10">10</a>.</p> <p><i>Sample collection and transport</i>: For all suspected cases with symptoms and/or travel history, nasopharyngeal and/or oropharyngeal swabs were collected in viral transport medium. Samples were sent to testing laboratories at 4Â°C with ice packs within 24 h of collection. Special provision was made with the national and international courier agencies to facilitate uninterrupted sample transport from field to laboratories and for quality control purpose.</p> <p>For a subset of the individuals, blood samples were collected in serum separator and EDTA (ethylenediaminetetraacetic acid) tubes for serum and plasma, respectively. Stool and urine samples were also collected from the laboratory-confirmed positive cases. Since SARS-CoV-2 is a new virus and diagnostic tests are evolving rapidly, it was considered critical to have various types of clinical samples for standardization and validation of new tests. However, owing to very low positivity rates of SARS-CoV-2 infection, it was subsequently decided to collect blood samples from only laboratory-confirmed positive cases and review the strategy later.</p> <p><i>Diagnostic assays for SARS-CoV-2</i>: Following release of the first sequence results of the SARS-CoV-2 virus by China<a href="#ref7">7</a>, candidate diagnostic real-time reverse transcription-polymerase chain reaction (rRT-PCR) assays were designed and made available in the public domain for researchers<a href="#ref12">12</a>. The first-line screening assay targeted the SARS-CoV-2-specific <i>E</i> gene. Confirmatory assays targeted the '<i>RdRp</i> gene', '<i>N</i> gene' and '<i>ORF-1b</i>'. Positive control materials for these assays were obtained from CharitÃ©, Berlin, via EVAg<a href="#ref13">13</a>. Known copy numbers of <i>in vitro</i> transcribed RNA standard were used as the positive controls for the rRT-PCR assays. Detection of RNAse <i>P</i> gene was used as an internal positive control to monitor sample quality, RNA extraction and detection of PCR inhibitors. The Indian Council of Medical Research (ICMR)-National Institute of Virology (NIV), Pune, which is the apex laboratory for viral diagnosis and research in India, optimized the conventional and real-time PCR assays targeting different genomic regions of SARS-CoV-2 and initiated testing of suspected cases.</p> <p><i>Laboratory organization for SARS-CoV-2 testing in India</i>: The Department of Health Research (DHR)/ICMR initiated establishment of a network of public health laboratories to enhance capacity for diagnosis and detection of viruses of public health importance in the Indian setting in 2013. Over the years, this network has been expanded, and currently, a total of 106 VRDLs have been established throughout the country (Fig. <a href="#F1">1</a>). Detection of viral pathogens using serological methods and molecular diagnostic tools is the major focus of the VRDLs. In addition, a subset of these also have capabilities to perform cell culture and virus cultivation. All established VRDLs are equipped to perform testing for viral aetiologies at least under Biosafety Level 2 (BSL-2) conditions. Further, 10 VRDLs are in various stages of operationalization of a BSL-3 facility for detection of high-risk pathogens<a href="#ref14">14</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Virus Research and Diagnostic Laboratory Network of 106 laboratories in India. <i>Source</i>: Map outline reproduced with permission from Survey of India, Department of Science &amp;amp; Technology.</p>   </div> <p>The NIV, Pune, functions as the resource centre for the VRDL network and is responsible for providing technical training for performing molecular and serological assays for virological diagnosis. It also performs the important task of standardizing assay procedures for the network as well as quality control and quality assurance activities.</p> <p><i>VRDLs as State nodal centres for coordination of sample collection and shipment</i>: In the last week of January 2020, the DHR/ICMR provided directives on sample collection and transport to all the 106 VRDLs currently under the network. Video conferences were held with VRDLs where the directives were explained and issues regarding sample collection and shipment were discussed. From each State or Union Territory, one VRDL was designated as the nodal centre for coordination of collection and shipment of samples to testing laboratories. On collection of sample(s) from any suspected case, the hospital or nearest VRDL was expected to contact the designated nodal centre for that State. The role of the nodal centre was to enable timely and proper transport of the collected samples from the VRDLs to the designated testing laboratory.</p> <p>The National Centre for Disease Control (NCDC), Ministry of Health and Family Welfare, Integrated Disease Surveillance Programme (IDSP) and State public health departments were informed regarding the nodal centres and assigned testing laboratories for the respective States. The VRDLs complemented the IDSP in an effective way. Flow diagram depicting the complete process from sample collection in the field, transportation, testing and reporting of results is shown in Figure <a href="#F2">2</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Process from sample collection in the field, transportation, testing and reporting of results. NCDC, National Centre for Disease Control; MoH&amp;amp;FW, Ministry of Health &amp;amp; Family Welfare; IDSP, Integrated Disease Surveillance Programme; DHR, Department of Health Research; ICMR, Indian Council of Medical Research; VRDLs, Virus Research and Diagnostic Laboratories; ICMR-NIV, ICMR-National Institute of Virology.</p>   </div> <p>All the laboratories were supplied with the primers, probes, PCR reagents, positive and negative controls and standard operating procedure (SOP) for the real-time RT-PCR (rRT-PCR) assay by NIV, Pune. A stringent inventory control was maintained at NIV and VRDLs based on the upsurge in number of cases and evolution of the disease.</p> <p><i>Expansion of testing capabilities and selection of testing laboratories for SARS-CoV-2</i>: Following the increase in the load of screening samples from suspected cases with symptoms and travel history to China or asymptomatic persons with travel history to Wuhan after January 15, 2020, it was decided that strategically located VRDLs needed to start testing for SARS-CoV-2 in addition to the apex laboratory at NIV, Pune. VRDLs were chosen based on their location in cities with international airports receiving travellers from China, the capability of the VRDLs to perform real-time PCR assays and their involvement in the ongoing testing for influenza viruses within the network. A total of 13 VRDLs across 11 States were selected for SARS-CoV-2 testing. After February 29, 2020, based on the upsurge in the number of suspect cases, primarily due to outbreaks reported from countries other than China (Iran, South Korea, Italy and Japan), the number of testing laboratories was scaled up to a total of 31 laboratories. It is eventually proposed to involve all 106 VRDLs in COVID-19 testing or sample collection and transportation. The distribution of COVID-19 testing (operational and new) laboratories in India is depicted in Figure <a href="#F3">3</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Testing laboratories for severe acute respiratory syndrome coronavirus 2. <i>Source</i>: Map outline reproduced with permission from Survey of India, Department of Science &amp;amp; Technology.</p>   </div> <p>Before initiating testing of clinical specimens from suspected cases of SARS-CoV-2, each VRDL shared results from the rRT-PCR runs performed with positive and negative controls with the apex laboratory (NIV, Pune). After satisfactory performance of the laboratories, independent testing was initiated in the designated catchment areas. Frontline screening rRT-PCR test targeting SARS-CoV-2-specific <i>E</i> gene was performed. Initial support was also provided to NCDC, New Delhi, for initiation of SARS-CoV-2 testing. Thereafter, NCDC, Delhi, initiated independent testing; however, results were shared with ICMR on a daily basis.</p> <p>Figure <a href="#F4">4</a> depicts the organization of laboratories for SARS-CoV-2 diagnosis and their respective roles.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Organization of laboratories for severe acute respiratory syndrome coronavirus 2 diagnosis in India and their respective roles.</p>   </div> <p><i>Quality control for SARS-CoV-2 testing at VRDLs</i>: NIV, Pune, coordinated the quality control activities for SARS-CoV-2 testing VRDLs. The laboratories shared first 10 negatives, all positive and equivocal samples for SARS-CoV-2 to NIV, Pune, for confirmation. Samples tested positive at VRDLs were subjected to confirmatory tests at NIV, Pune. The final result for any SARS-CoV-2-positive samples was released only after confirmation at the apex laboratory. Ten negative samples from VRDLs were also randomly picked up and subjected to next-generation sequencing (NGS).</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p><i>Testing of suspected cases and contacts</i>: Based on the inclusion criteria, as on February 29, 2020, a total of 1,369 individuals including suspected cases and contacts were tested at the designated laboratories. The testing VRDLs as well as NIV, Pune, had evaluated a varying number of samples from suspected cases/contacts as follows: NIV, Pune (n=343); All India Institute of Medical Sciences (AIIMS), New Delhi (n=16); NIV Field Unit, Bangaluru (n=152); NIV Field Unit, Kerala (n=435); National Institute of Cholera and Enteric Diseases, Kolkata (n=42); Kasturba Hospital for Infectious Diseases, Mumbai (n=49); Sawai Man Singh Hospital, Jaipur (n=68); King George's Medical University, Lucknow (n=54); Gandhi Medical College, Secunderabad (n=93); King Institute of Preventive Medicine and Research, Chennai (n=52); Indira Gandhi Government Medical College &amp;amp; Hospital, Nagpur (n=21); Bangalore Medical College and Research Institute, Bengaluru (n=26); Gauhati Medical College, Guwahati (n=6); and B.J. Medical College, Ahmedabad (n=12). Repeat testing of suspect cases was also done at VRDLs (numbers excluded). Eleven VRDLs initiated independent testing between February 1 and 3, 2020. Two VRDLs at Ahmedabad and Guwahati were added from February 8, 2020 as per need. Time taken by all VRDLs from receipt of reagents to initiation of independent testing ranged between 24-36 h. The turnaround time for testing was 12-24 h depending on the number of samples tested. Results were communicated to the State IDSP and/or the referring contact point. Initial numbers referred to the testing network were high due to increased number of travellers returning from affected countries. However, this number declined after travel restrictions were imposed. As the VRDLs established testing for SARS-CoV-2, the number of samples referred to NIV dropped.</p> <p>Figure <a href="#F5">5</a> depicts the incremental decline in number of samples referred to NIV after testing at VRDLs was established.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>The incremental decline in number of samples referred to the National Institute of Virology after testing at the Virus Research and Diagnostic Laboratories was established.</p>   </div> <p><i>Testing of evacuees from quarantine camps</i>: On January 31 and February 1, 2020, the GoI evacuated 654 individuals from Wuhan. All these evacuees were quarantined for 14 days at two centres (Armed Forces Medical Services camp at Manesar and Indo-Tibetan Border Police Chhawla camp)<a href="#ref15">15</a>. On the day of arrival (day 0), nasopharyngeal and/or oropharyngeal samples were collected from all these individuals. The samples were tested for SARS-CoV-2 at ICMR-NIV, Pune, and DHR/ICMR Regional VRDL at AIIMS, New Delhi. All samples were tested negative. Between days 0 and 14, symptomatic quarantined individuals were subjected to repeat testing of SARS-CoV-2. At the end of two weeks, repeat samples were collected from all the 654 individuals and retested at the two sites. All samples were again tested negative. Subsequently, samples were tested from the second set of evacuees from Wuhan (112) and Diamond Princess ship, Japan (124), on day 0 at the same sites. A total of 1,544 samples were tested, and all were negative.</p> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="demographicprofileofindividualsscreenedforcovid-19" title="sec"> <div class="title" tagx="title" title="title">
Demographic profile of individuals screened for COVID-19</div> <p><u>Evacuees from China and Japan quarantined at army and border police camps</u>: Of the 654 evacuees from Wuhan, 29.2 per cent (191/654) were female and 70.5 per cent (461/654) were male. Gender information was not available for two evacuees. Manesar camp had 248 males, whereas the remaining 406 including 191 females were quarantined at ITBP Chhawla camp. The mean age of the evacuees was 23.5 yr (median: 22 yr; range: 1-72 yr); 77.2 per cent (492/637) of the evacuees for whom information about age was available were â‰¤25 yr of age.</p> <p>The second batch of quarantined individuals at these two camps arrived on February 27, 2020 which included a total of 236 evacuees from China and from the cruise ship Diamond Princess off the coast of Japan. The second batch of quarantined individuals included a total of 41 females and 195 males. The mean age was 29.4 yr (median: 29 yr; range: one year nine months-56 yr); 76.3 per cent (180/236) were between 25 and 50 yr of age. All these were also tested negative.</p> <p><u>Individuals tested for SARS-CoV-2 other than evacuees who were quarantined</u>: Of the 1,369 individuals other than evacuees who were tested, all demographic details were available for a total of 1,263 (843 male, 420 female) individuals. The mean age of the individuals who underwent testing was 25.9 yr (median: 27 yr; range: eight months-76 yr). A little more than half of these individuals were in the 25-50 yr age group (54.2%).</p> <p>Samples from three suspected cases with symptoms of acute respiratory illness were tested positive for SARS-CoV-2 at NIV, Pune. The first positive case was confirmed on January 30, 2020, followed by two other cases on February 2 and 3, 2020. All the three positive cases had travel history to Wuhan, China, and belonged to the State of Kerala. A total of 67 samples from 64 individuals with contact history with the three confirmed cases who developed respiratory symptoms were also tested for SARS-CoV-2. All the samples from the 64 contacts were negative for SARS-CoV-2.</p> <p><i>Symptomatology and travel history of individuals screened for COVID-19 infection outside the quarantine camps</i>: Of the total 1263 individuals, 19 per cent (n=240) did not report any specific symptoms for inclusion for COVID-19 testing. Among those reported as symptomatic, complete details of clinical history were accessed for 292 individuals. The presence of cough was the most common symptom reported in 49.7 per cent (n=145). Sore throat was reported by 21.2 per cent (n=62), whereas 34.6 per cent of the individuals (n=101) were febrile at the time of presentation; 17.1 per cent (n=50) of all symptomatic individuals presented with nasal discharge and rhinitis. Diarrhoea was a presenting symptom for only 1.4 per cent (4/292) of all symptomatic patients with available clinical history. Of the 1,263 individuals for whom travel-related data were available, any history of foreign travel was documented for 1,081 individuals (85.6%); 19.32 per cent (n=244) had a history of travel to Wuhan. A further 38.1 per cent (481/1263) had a history of travel to areas in China other than Wuhan, whereas 15.7 per cent (198/1263) had a history of travel to South-East Asian countries such as Thailand, Singapore, Malaysia, Indonesia and Vietnam. The remaining 158 (12.5%) individuals did not have any pertinent overseas travel history.</p> <p><i>Quality control for ongoing testing</i>: For quality control purposes, all testing VRDLs performing the <i>E</i> gene screening assay shared the first 10 negative samples and all positive or equivocal/borderline testing samples with ICMR-NIV, Pune, for reconfirmation. Eleven negative samples further subjected to NGS revealed negative results for SARS-CoV-2 but tested positive for other viruses: influenza A (2) and rhinovirus (3).</p> <p>A total of 126 samples which tested negative by the <i>E</i> gene rRT-PCR screening assay at the 13 testing VRDLs were shared with NIV, Pune, till February 29, 2020, and these were all confirmed negative. The concordance for the negative samples was, therefore, 100 per cent.</p> <p>The VRDLs also shared 13 samples which were borderline positives and showed amplification at late Ct values (range: 33-37 cycles) for confirmation. These included seven follow up samples collected from the three laboratory-confirmed positive cases which were also tested positive for the <i>E</i> gene at the testing VRDL. Testing results were concordant for five of seven follow up samples. The remaining six samples which showed borderline positivity with the screening qPCR at the testing VRDLs were found to be negative with the confirmatory qPCR assays performed at NIV, Pune.</p> <p><i>Inventory control</i>: Inventory control posed several challenges. Primers and probes were ordered by the NIV, Pune, once the laboratory protocols were shared by the WHO on January 15, 2020. Since nature of spread of the disease was unknown, testing reagents were stockpiled for only 5000 tests initially. NIV initiated testing from January 21, 2020. However, in view of upsurge in global cases and deaths, the testing capacity was upscaled to another 13 laboratories and inventory at NIV was scaled up to 25,000 tests by February 1, 2020. Following further upsurge in global cases and deaths by February 25, 2020, the inventory was upscaled to 70,000 tests. Daily situational analysis of inventory at the apex and testing laboratories was undertaken to ensure optimum supplies and avoid exhaustion of reagents at the apex and testing laboratories.</p> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Global epidemics due to emerging/re-emerging infectious diseases have become common over the past two decades, with the major reason being increased trade and travel, leading to movement of people across borders. A recent example is the epidemic of SARS-CoV-2 in Hubei province of China which has spread to more than 70 countries within a two-month time. During the initial period, virus sequences were unknown, there were no known sources for acquiring positive controls and the virus isolates were not available. In view of this, the challenge to develop diagnostics for a new disease with a pandemic potential initially seemed unsurmountable. However, once the virus sequences were made available in public domain and source of positive controls and probes could be identified, India immediately established a network of testing laboratories for the new SARS-CoV-2 virus very swiftly. Starting with availability of validated diagnosis at the ICMR-NIV, Pune, testing capacity was further upscaled to another 13 DHR/ICMR VRDLs. The apex laboratory standardized the testing protocols within one week, and the VRDLs initiated testing within two weeks of release of laboratory protocols by the WHO. In addition, 35 laboratories fully equipped in terms of availability of adequate infrastructure and trained staff for SARS-CoV-2 testing were kept on a standby. An impressive model wherein the VRDLs have closely worked with the field surveillance programme of GoI (IDSP) has been operationalized during this public health crisis. However, this model worked well only in some States, but there were gaps in other states. Kerala, Karnataka, Telangana, Rajasthan and Uttar Pradesh clearly leveraged the capacity of existing VRDLs. The sample transport mechanisms from IDSP to VRDLs at these places were also good. The number of tests conducted by VRDLs in these States depicted the effective screening of suspect cases implemented at nearby airports, IDSP and State health programmes. However, in cities such as Kolkata, Chennai and Mumbai, despite the presence of a testing facility, the number of suspect cases referred was low. Liasoning between VRDLs and IDSP needs to be further strengthened in such areas.</p> <p>Quality control programme implemented by NIV, Pune, for the VRDLs revealed 100 per cent concordance between the testing at VRDLs and NIV for negative samples. For borderline positive samples (Ct values between 33 and 35 cycles), concordance was low. This emphasized the need to continue reconfirmation of all positive samples at NIV, Pune. This has also indicated that <i>E</i> gene screening assay may not be sufficient to declare positivity. Further confirmatory tests as currently done by NIV, Pune, are required for confirming true positivity. To enhance the robustness of testing, it is essential to equip the SARS-CoV-2 testing VRDLs with additional confirmatory assays.</p> <p>Inventory control posed challenges at every step. It was imperative to maintain optimum stocks of reagents, but simultaneously, it was important not to overestimate and waste the limited resources. Thoughtful optimization of inventory was undertaken. Planning of availability of tests at a given point was done in such a way that the excess tests could be used for routine surveillance in case the situation of COVID-19 does not worsen in India.</p> <p>The threat of a potential pandemic due to SARS-CoV-2 has brought out the strength of the judiciously established network of VRDLs in India and also the capability of this network to rapidly adapt in times of any public health emergency, with appropriate quality checks. This already operational platform was able to switch gears to provide countrywide diagnosis for SARS-CoV-2 within a month after discovery of this novel virus. VRDLs offer a robust platform for early detection of emerging/re-emerging viral infections in all parts of India, facilitating early containment and prevention of larger epidemics.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-supported-by" title="supported-by"> <p><b><i>Financial support &amp;amp; sponsorship</i>:</b> Authors acknowledge the Department of Health Research, Ministry of Health &amp;amp; Family Welfare, Government of India, New Delhi, for financial support.</p> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p><b><i>Conflicts of Interest</i>:</b> None.</p> </div> <div class="fn-type-equal" title="equal"><span class="label" tagx="label" title="label">â€ </span><p>Equal contribution</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="ref1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knipe</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howley</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="source" tagx="source" title="source">Fields virology</span><span class="year" tagx="year" title="year">2013</span><span class="edition" tagx="edition" title="edition">6th ed</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Philadelphia</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Lippincott Williams &amp;amp; Wilkins</span></span></li> <li tag="ref"><a name="ref2" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">VCC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">SKP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">PCY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome coronavirus as an agent of emerging and reemerging infection</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">660</span><span class="lpage" tagx="lpage" title="lpage">94</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934078">17934078</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JFW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">SKP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">KKW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">VCC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">PCY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus: Another zoonotic betacoronavirus causing SARS-like disease</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">465</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25810418">25810418</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source">Coronavirus</span><span class="year" tagx="year" title="year">2020</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on January 21, 2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">WHO</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://wwwwhoint/health-topics/coronavirus">https://wwwwhoint/health-topics/coronavirus</a></span></span></span></li> <li tag="ref"><a name="ref5" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source">Naming the coronavirus disease (COVID-19) and the virus that causes it</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on February 29, 2020</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://wwwwhoint/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it">https://wwwwhoint/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it</a></span></span></span></li> <li tag="ref"><a name="ref6" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Groot</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gulyaeva</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome-related coronavirus: The species and its viruses - A statement of the Coronavirus Study Group</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="comment">doi: 20200207937862</span></span></li> <li tag="ref"><a name="ref7" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YZ</span></span></span><span class="source" tagx="source" title="source">Novel 2019 Coronavirus genome Virological</span><span class="year" tagx="year" title="year">2020</span><span class="month" tagx="month" title="month">1</span><span class="day" tagx="day" title="day">10</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on January 21, 2020</span><span class="comment">Available from: <span class="uri" title="uri"><a href="http://virologicalorg/t/novel-2019-coronavirus-genome/319">http://virologicalorg/t/novel-2019-coronavirus-genome/319</a></span></span></span></li> <li tag="ref"><a name="ref8" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span><span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">221</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31987001">31987001</a></span></span></li> <li tag="ref"><a name="ref9" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source">Coronavirus disease 2019 (COVID-19) Situation Report - 40</span><span class="year" tagx="year" title="year">2020</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on February 29, 2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">WHO</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://wwwwhoint/docs/default-source/coronaviruse/situation-reports/20200304-sitrep-40-covid-19pdfsfvrsn=783b4c9d_2">https://wwwwhoint/docs/default-source/coronaviruse/situation-reports/20200304-sitrep-40-covid-19pdfsfvrsn=783b4c9d_2</a></span></span></span></li> <li tag="ref"><a name="ref10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: A study of a family cluster</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">514</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a></span></span></li> <li tag="ref"><a name="ref11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source">Laboratory testing for 2019 novel coronavirus (2019-nCoV) in suspected human cases: Interim guidance</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">WHO</span></span></li> <li tag="ref"><a name="ref12" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landt</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molenkamp</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meijer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">DKW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span><span class="source" tagx="source" title="source">Euro Surveill</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2000045</span></span></li> <li tag="ref"><a name="ref13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="source" tagx="source" title="source">European Virus Archive - GLOBAL</span><span class="year" tagx="year" title="year">2020</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on February 29, 2020</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://www.european-virus-archive.com/">https://www.european-virus-archive.com/</a></span></span></span></li> <li tag="ref"><a name="ref14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="collab'">collab: Department of Health Research, Ministry of Health &amp;amp; Family Welfare, Government of India</span><span class="source" tagx="source" title="source">Establishment of a network of laboratories for managing epidemics and natural calamities (VRDL)</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on February 29, 2020</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://dhr.gov.in/schemes/esablishment-network-laboratories-managingepidemics-and-natural-calamities">https://dhr.gov.in/schemes/esablishment-network-laboratories-managingepidemics-and-natural-calamities</a></span></span></span></li> <li tag="ref"><a name="ref15" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="source" tagx="source" title="source">Press Information Bureau, Ministry of Health &amp;amp; Family Welfare, Government of India</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">accessed on February 29, 2020</span><span class="comment">Available from: <span class="uri" title="uri"><a href="https://pib.gov.in/newsite/PrintRelease.aspx?relid=199146">https://pib.gov.in/newsite/PrintRelease.aspx?relid=199146</a></span></span></span></li> </ul> </div> </div>  </body></html>