<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Mol Biol Evol</span><span class="iso-abbrev" title="iso-abbrev">Mol. Biol. Evol</span><span class="publisher-id" title="publisher-id">molbev</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Biology and Evolution</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0737-4038</span><span class="issn-epub" tagx="issn" title="issn-epub">1537-1719</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7086167">7086167</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/molbev/msz276">10.1093/molbev/msz276</a></span><span class="publisher-id" title="publisher-id">msz276</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Methods</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Machine Learning Methods for Predicting Human-Adaptive Influenza A Viruses Based on Viral Nucleotide Compositions</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6860-6135</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jing</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Sen</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Bo</span></span><a href="#msz276-aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Yi</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">Xiao-Ping</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Xiao-Yan</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Meng-Ting</span></span><a href="#msz276-aff1">1</a><a href="#msz276-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Yu-Chang</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Zhong-Peng</span></span><a href="#msz276-aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">Cheng-Feng</span></span><a href="#msz276-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Tao</span></span><a href="#msz276-cor1" /><a href="#msz276-aff1">1</a><a href="#msz276-aff3">3</a></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">Koichiro</span></span><span class="role" tagx="role" title="role">Associate Editor</span></span></div><span class="citation_author_institution" id="msz276-aff1">[1], <span class="institution" title="institution">Department of Virology, State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="msz276-aff2">[2], <span class="institution" title="institution">Department of Clinical Laboratory, the Fifth Medical Centre of Chinese PLA General Hospital</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="msz276-aff3">[3], <span class="institution" title="institution">Graduate School, Anhui Medical University</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="msz276-aff4">[4], <span class="institution" title="institution">Department of Infection and Immunology, State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology</span><span class="country" tagx="country" title="country">China</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><b>Corresponding author:</b> E-mail: <span class="email" tagx="email" title="email">jiangtao@bmi.ac.cn</span>. </div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-4</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-11-11</span></span><span class="volume" tagx="volume" title="volume">37</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">1224</span><span class="lpage" tagx="lpage" title="lpage">1236</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="msz276.pdf" title="self-uri">msz276.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Each influenza pandemic was caused at least partly by avian- and/or swine-origin influenza A viruses (IAVs). The timing of and the potential IAVs involved in the next pandemic are currently unpredictable. We aim to build machine learning (ML) models to predict human-adaptive IAV nucleotide composition. A total of 217,549 IAV full-length coding sequences of the PB2 (polymerase basic protein-2), PB1, PA (polymerase acidic protein), HA (hemagglutinin), NP (nucleoprotein), and NA (neuraminidase) segments were decomposed for their codon position-based mononucleotides (12 nts) and dinucleotides (48 dnts). A total of 68,742 human sequences and 68,739 avian sequences (1:1) were resampled to characterize the human adaptation-associated (d)nts with principal component analysis (PCA) and other ML models. Then, the human adaptation of IAV sequences was predicted based on the characterized (d)nts. Respectively, 9, 12, 11, 13, 10 and 9 human-adaptive (d)nts were optimized for the six segments. PCA and hierarchical clustering analysis revealed the linear separability of the optimized (d)nts between the human-adaptive and avian-adaptive sets. The results of the confusion matrix and the area under the receiver operating characteristic curve indicated a high performance of the ML models to predict human adaptation of IAVs. Our model performed well in predicting the human adaptation of the swine/avian IAVs before and after the 2009 H1N1 pandemic. In conclusion, we identified the human adaptation-associated genomic composition of IAV segments. ML models for IAV human adaptation prediction using large IAV genomic data sets can facilitate the identification of key viral factors that affect virus transmission/pathogenicity. Most importantly, it allows the prediction of pandemic influenza.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
human adaptation</div> <div class="kwd" title="kwd">
influenza A viruses (IAVs)</div> <div class="kwd" title="kwd">
genomic nucleotide composition</div> <div class="kwd" title="kwd">
machine learning (ML)</div> <div class="kwd" title="kwd">
dinucleotide</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Science and Technology Major Project of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2017ZX10305501</span><span class="award-id" tagx="award-id" title="award-id">2018ZX10732401</span><span class="award-id" tagx="award-id" title="award-id">2018ZX10711001-003</span><span class="award-id" tagx="award-id" title="award-id">AWS16J020</span><span class="award-id" tagx="award-id" title="award-id">BWS16J008</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Key Project of China State Key Laboratory of Pathogen and Biosecurity of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">SKLPBS1806</span><span class="award-id" tagx="award-id" title="award-id">SKLPBS1407</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Natural Science Foundation of China</span><span class="institution-id" title="institution-id">10.13039/501100001809</span></div> </div><span class="award-id" tagx="award-id" title="award-id">31200119</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Key Research and Development Program of China</span><span class="institution-id" title="institution-id">10.13039/501100012166</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2018YFC1200100</span></div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Type A influenza viruses (IAVs) infect a wide range of avian and mammalian hosts, generally with species specificity. Avian influenza viruses (AIVs) typically exist in natural reservoirs, waterfowl, and shorebirds (<a href="#msz276-B50">Yoon <i>et al.</i> 2014</a>), which mostly cause subclinical bird infection (<a href="#msz276-B46">Webster et al. 1978</a>; <a href="#msz276-B23">Long et al. 2019</a>). AIVs sporadically infect mammalian hosts, such as swine (<a href="#msz276-B28">Pensaert et al. 1981</a>), human beings (<a href="#msz276-B38">Subbarao and Katz 2000</a>; <a href="#msz276-B6">de Jong et al. 2006</a>; <a href="#msz276-B21">Lam et al. 2013</a>), and other mammals (<a href="#msz276-B47">White 2013</a>; <a href="#msz276-B22">Lee et al. 2017</a>) and are incapable of intraspecies transmission (<a href="#msz276-B43">Tran et al. 2004</a>; <a href="#msz276-B24">Maines et al. 2006</a>; <a href="#msz276-B23">Long et al. 2019</a>). However, the high frequency of mutation and segment recombination endows AIVs with the chance to obtain human-adaptive genomes, which pose a high pandemic risk. Notably, swine adaptation and swine-adapted IAVs are closely related to human pandemics. All of the last five recorded influenza pandemics were caused by avian-origin, swine-origin, or reassortant IAVs (<a href="#msz276-B32">Reid et al. 2004</a>; <a href="#msz276-B19">Kislinger et al. 2006</a>; <a href="#msz276-B4">Bragstad et al. 2011</a>; <a href="#msz276-B23">Long et al. 2019</a>). Thus, it is of great importance to predict the adaptation of avian or swine IAVs to humans.</p> <p>Human-adaptive IAVs are capable of infecting and causing disease in humans easily and of spreading among human populations efficiently. To date, H3N2 and H1N1 (including seasonal H1N1 and A(H1N1)pdm09) are dominant human-adaptive IAV subtypes that cause epidemics in humans (<a href="#msz276-B33">Ren et al. 2016</a>). H5N1, H7N9, and other IAV subtypes occasionally infect humans but are not yet capable of spreading in human populations (<a href="#msz276-B49">Yang et al. 2007</a>; <a href="#msz276-B34">Rudge and Coker 2013</a>; <a href="#msz276-B14">Hu et al. 2014</a>; <a href="#msz276-B7">Deng et al. 2017</a>). Laboratory studies have identified numerous viral determinants that are associated with the human adaptation of IAVs via mediating receptor binding, regulating the virus’s replication cycle, and antagonizing host immunity (<a href="#msz276-B42">Taubenberger and Kash 2010</a>; <a href="#msz276-B3">Bouvier 2015</a>; <a href="#msz276-B23">Long et al. 2019</a>). However, there are no universal human adaptation determinants for IAVs.</p> <p>Gene sequencing technology and machine/deep learning methods have facilitated virus sequence identification of a considerably large data set, including IAVs. Machine learning (ML) methods have recently demonstrated their effectiveness in multiple disciplinary fields, including virology. The distinct host tropism protein signatures of IAVs (<a href="#msz276-B9">Eng et al. 2016</a>), the zoonotic risk of various viruses (<a href="#msz276-B10">Eng et al. 2017</a>), and even the avian-to-human transmission risk of IAVs (<a href="#msz276-B31">Qiang et al. 2018</a>) have been recognized. The host dependence of mononucleotides (nts) and tetranucleotide compositions of influenza viruses has also been studied with ML methods (<a href="#msz276-B15">Iwasaki et al. 2013</a>). Notably, the prominent role of dinucleotides in virus genomes has been implicated in both experimental and computational reports. Viral dinucleotides are targets for the host innate immune system (<a href="#msz276-B40">Takata et al. 2017</a>), and they independently regulate the virulence (<a href="#msz276-B1">Atkinson et al. 2014</a>; <a href="#msz276-B44">Tulloch et al. 2014</a>) and replication (<a href="#msz276-B48">Witteveldt et al. 2016</a>) of IAV viruses. Species-specific (<a href="#msz276-B13">Glass et al. 2007</a>) and virus-family-specific (<a href="#msz276-B8">Di Giallonardo et al. 2017</a>) dinucleotide compositions have also been computationally recognized. More recently, the dinucleotide composition in RNA virus genomes accurately predicts viral reservoir hosts and arthropod vectors using ML methods (<a href="#msz276-B2">Babayan et al. 2018</a>). Therefore, we hypothesize that genomic dinucleotide composition is another crucial genomic feature for influenza viruses, which is most likely amino acid independent, and may be useful for characterizing the human adaptation feature of IAVs.</p> <p>In the present study, 60 types of mono- and dinucleotide compositions were analyzed based on the nucleotide position within a codon in the full-length coding sequences of the first six genomic segments of IAVs: PB2 (polymerase basic protein 2), PB1, PA (polymerase acidic protein), HA (hemagglutinin), NP (nucleoprotein), and NA (neuraminidase). These (d)nts were optimized based on their relative importance, with principal component analysis (PCA) and support vector classifier (SVC) methods. Then, ML models of gradient-boosted regression trees (GBRT), multilayer perceptron (MLP) classifier, random forest (RF) classifier, and SVC were built to analyze and predict the human adaptation of human-, swine-, or avian-origin IAVs. Our models perform well in predicting human-adaptive swine or avian IAVs.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="predictionpipelineanddataprocessingofthegenomicnucleotidecompositioniniavs" title="sec"> <div class="title" tagx="title" title="title">
Prediction Pipeline and Data Processing of the Genomic Nucleotide Composition in IAVs</div> <p>As the workflow diagram in <a href="#msz276-F1">figure 1<i>A</i></a> shows, data wrangling was performed for IAV open reading frame (ORF) sequences. Twelve types of mononucleotides (nts) and 48 types of dinucleotides (dnts) in the ORF were counted for all the sequence samples. The phylogeny of the sample ORF sequences, the hierarchical clustering of sequence samples based on the 60 (d)nts,, and the distribution of the sequence samples, in each type of sequence label, were analyzed (<a href="#msz276-F1">fig. 1<i>A</i></a>). The 60 (d)nts were sorted based on their importance (cross-validation score, cv_Score) for the classifier with PCA and SVC methods (<a href="#msz276-F1">fig. 1<i>B</i></a>), and the best (d)nts (<a href="#msz276-F1">fig. 1<i>B</i></a>), which were optimized with ML approaches from the sorted (d)nts, were utilized for the final data optimization and the final prediction (<a href="#msz276-F1">fig. 1<i>B</i></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 1.</span></a></div>   <p>The workflow of data processing, feature optimization, and model construction. (<i>A</i>) The workflow of data processing and model construction. Original ORF sequences within the length range were utilized for codon dependently counting 12 mononucleotides (nts) and 48 dinucleotides (dnts). The counting file for the 60 (d)nts for the PB2, PB1, PA, HA, NP, or NA segments was randomly resampled to maintain a balanced distribution of human and avian sequences. The (d)nt composition distribution and the sequence phylogeny were analyzed, and then the randomly split (d)nt counting data were utilized for model building and testing. (<i>B</i>) Workflow of the feature extraction with PCA/ SVC methods. The 60 (d)nts were sorted according to the score (AUC) from the SVC analysis, with the PCA-extracted principal component from the randomly selected (d)nt counting information. Accumulated (d)nts in the sorted list were analyzed with ML models, with the cross of the AUC score curve down its MA score (<i>n</i> = 3) as the threshold of the best number of (d)nts. "DF2_60 (d)nts.csv�? for the (d)nt sorting and "Best x (d)nts�? for data optimization were, respectively, labeled with blue and green text boxes filled in both (<i>A</i>) and (<i>B</i>). </p>   </div> <p>In total, 226,183 full-length coding sequences for the first six segments (PB2, PB1, PA, HA, NP, and NA), available up to December 31, 2018, had skewed distributions for the labels of country/area, host, subtype, segment, or year. In total, 8,634 sequences were dropped, beyond the length range had repeated sequence IDs (<a href="#sup1">supplementary table 1</a>, <a href="#sup1">Supplementary Material</a> online). The remaining 217,549 sequences were still predominantly from the United States for the country/area and from 2009 to 2018 for the year (<a href="#sup1">supplementary fig. 1</a>, <a href="#sup1">Supplementary Material</a> online). Random resampling was performed to maintain a ratio of approximately 1:1 for the sequences from the United States and mainland China, the second largest influenza sample country. The resampled 83,980 avian and human sequences available for feature extraction and model building were dominantly from North America and East and Southeast Asia, particularly from the United States and China (<a href="#sup1">supplementary fig. 2</a><i>A</i>, <a href="#sup1">Supplementary Material</a> online); the sample distributions of the different types of hosts, subtypes, segments and years are indicated (<a href="#sup1">supplementary fig. 2</a><i>B</i>–<i>F</i>, <a href="#sup1">Supplementary Material</a> online). The 34,990 swine IAV sequences were not included in the training data because of their double biological adaptation to both human and avian hosts.</p> </div> <div class="thecharacterizationofthehumanadaptation-associatednucleotidecompositionofiavsequences" title="sec"> <div class="title" tagx="title" title="title">
The Characterization of the Human Adaptation-Associated Nucleotide Composition of IAV Sequences</div> <p>The (d)nt composition was counted and compared within and between species based on the profile of relative dinucleotide abundance values according to previous reports (<a href="#msz276-B17">Karlin and Mrazek 1997</a>). Hierarchical clustering was performed for 3.59–5.01‰ (59–61) randomly sampled sequences from each segment sequence set according to the (d)nt composition. The majority of human and avian IAVs were not clustered into human and avian branches, respectively, for the PB1, PA, HA, and NA segments (<a href="#sup1">supplementary fig. 3</a>, <a href="#sup1">Supplementary Material</a> online), and these selected sequence samples were not clustered into human and avian groups in a phylogeny tree (<a href="#sup1">supplementary fig. 4</a>, <a href="#sup1">Supplementary Material</a> online). Additionally, a PCA transversion of the 60 (d)nts was performed to evaluate the linear separability between major human sequences and avian sequences. There was no such separability in the principal component 1 or principal component 2 of the 60 (d)nts in the PB2, PA, HA, or NA segments; only the PB1 and NP segments were separable for both groups of subtypes for principal component 1 (<a href="#sup1">supplementary fig. 5</a>, <a href="#sup1">Supplementary Material</a> online).</p> <p>An ML analysis combining PCA and SVC was performed to characterize the human adaptation-associated nucleotide composition of IAVs from the 60 (d)nts. As the workflow (<a href="#msz276-F1">fig. 1<i>B</i></a>) shows, 3,540 iterations of PCA/SVC analysis (guaranteeing more than 200 repeat analyses for each (d)nt) (<a href="#sup1">supplementary fig. 6</a>, <a href="#sup1">Supplementary Material</a> online) were performed to reduce every four (d)nts into one principal component, which was then utilized for the SVC for avian and human IAV sequences. The statistical analysis results of the cross-validation score (Cross_val score) from the PCA/SVC for the 60 (d)nts of each segment are listed in <a href="#sup1">supplementary tables 2–7</a>, <a href="#sup1">Supplementary Material</a> online. The 60 (d)nts were sorted according to the mean Cross_val score, in other words, based on the feature importance (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 7<i>A</i>–<i>F</i></a>, <a href="#sup1">Supplementary Material</a> online, for the six segments). The plotting of the relative (d)nt composition for each segment demonstrated that there was a significant difference between human and avian sets in terms of the (d)nts.</p> <p>The ML-based GBRT, MLP classifier, RF classifier, and SVC models were utilized to identify the optimal number of (d)nts for the human/avian IAV classification. As indicated in <a href="#msz276-F2">figure 2<i>A</i></a>–<i>F</i>, a leveling off of the crossing of the cross-validation score (Cross_val score) with its moving average 3 (MA3) level, along with the (d)nt accumulation, was defined as the indicator of the optimal number of (d)nts. Accordingly, an average of 9–13 top (d)nts in the sorted list was identified as the best/optimized (d)nts by the four types of ML classifiers (<a href="#msz276-F2">fig. 2<i>G</i></a>). For the PB2 segment, there was a significant (<i>P</i> &amp;lt; 0.001, Mann–Whitney <i>U</i> test, <a href="#sup1">supplementary table 8</a>, <a href="#sup1">Supplementary Material</a> online) difference between avian and human sets for each of the nine optimized (d)nts (<a href="#msz276-F2">fig. 2<i>H</i></a>, <a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>A</i></a>, <a href="#sup1">Supplementary Material</a> online). A significant (<i>P</i> = 0.049461 for p_ag_N12 in the HA segment, <i>P</i> &amp;lt; 0.001 for the others, Mann–Whitney <i>U</i> test, <a href="#sup1">supplementary table 8</a>, <a href="#sup1">Supplementary Material</a> online) difference was also observed for each of the other (d)nts for the PB1 (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>B</i></a>, <a href="#sup1">Supplementary Material</a> online), PA (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>C</i></a>, <a href="#sup1">Supplementary Material</a> online), HA (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>D</i></a>, <a href="#sup1">Supplementary Material</a> online), NP (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>E</i></a>, <a href="#sup1">Supplementary Material</a> online), and NA (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 8<i>F</i></a>, <a href="#sup1">Supplementary Material</a> online) segments.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 2.</span></a></div>   <p>Optimization by SVC with the most different (d)nts between human and avian virus segments. The sorted 60 (d)nts were successively put into the accumulating feature list for the SVC (random state = 1 and cv = 5); the MA of the three cross-validation scores (cross_val importance) (MA3) and the cross_val importance itself for each segment were curved (<i>A</i>–<i>F</i>). The first 9, 12, 11, 13, 10, 9 (d)nts in each list were defined as the best features for PB2, PB1, PA, HA, NP, and NA (<i>G</i>), respectively, when the cross_val importance curve crossed the MA3 curve. (<i>H</i>) Boxplot of the best 9 (d)nts for the PB2 segment; the Mann–Whitney <i>U</i> test was performed between the two groups for each (d)nt, and the <i>P</i>-value is indicated.</p>   </div> </div> <div class="predictinghumanadaptationofiavsbasedonthecharacterizednucleotidecomposition" title="sec"> <div class="title" tagx="title" title="title">
Predicting Human Adaptation of IAVs Based on the Characterized Nucleotide Composition</div> <p>The unsupervised clustering and supervised two-category classification with the four ML classifiers mentioned above were performed to evaluate the effectiveness of the characterized (d)nts for human/avian IAV classification. It was demonstrated that the two principal components of the nine optimized (d)nts were separable in distribution between avian and human sequence sets for the PB2 (<a href="#msz276-F3">fig. 3<i>A</i></a>) and PB1 (<a href="#msz276-F3">fig. 3<i>B</i></a>) segments. Such separability was also observed for the PA (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 9<i>A</i></a>, <a href="#sup1">Supplementary Material</a> online), HA (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 9<i>A</i></a>, <a href="#sup1">Supplementary Material</a> online), NP (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 10<i>A</i></a>, <a href="#sup1">Supplementary Material</a> online), and NA segments to varying degrees. Such separability was also indicated by the hierarchical clustering of both sequence sets. The majority of the human and avian PB2 sequences were clustered into two groups for the PB2 segment (<a href="#msz276-F4">fig. 4</a>) and the other five segments (<a href="#sup1">supplementary figs. 11–15</a>, <a href="#sup1">Supplementary Material</a> online), particularly for the ribonucleoprotein complex of the PB2, PB1, and NP segments. Interestingly, the PA sequences of A(H1N1)pdm09 were clustered into the avian sequence group, thought distinctive from other human sequences and avian sequences, as indicated previously (<a href="#msz276-B37">Smith et al. 2009</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 3.</span></a></div>   <p>PCA analysis of the optimized (d)nts for the PB2 and PB1 segments between human and avian IAV sequences. The optimized 9 and 12 (d)nts for the PB2 (<i>A</i>) and PB1 (<i>B</i>) segments, respectively, were converted into two principal components and were then plotted with pairplot (seaborn package, Python) (lower-left panel and upper-right panel in each figure subpart). The distribution of principal components 1 (PCA_1) and 2 (PCA_2) of avian (blue) and human (orange) sequences were indicated by kernel density estimation (KDE) (upper-left panel and lower-right panel in each figure subpart), and the separability between avian and human sequences was shown for the PB2 (<i>A</i>) and PB1 (<i>B</i>) segments with the pair plots and KDE.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 4.</span></a></div>   <p>Heatmap and hierarchical clustering of human and avian IAV PB2 sequences based on the Euclidean distance of the optimized (d)nts. The 61 PB2 sequence samples were randomly (random state = 1) selected from PB2 (3.59‰) and then clustered with a heatmap and hierarchical clustering for PB2 based on the Euclidean distance of the optimized 9 (d)nts; the sequence identity and (d)nts were clustered. Standardized scaling was performed for data with the function (x-x.mean)/x.std. The color in the heatmap presented the value for each (d)nt on the <i>x</i>-axis, as shown by the color bar in the upper-left corner. The hierarchical relationships for the sampled sequences and (d)nts are indicated on the left and upper sides, respectively, in each image. The red-blue column to the left of the heatmap was utilized to show the human (red) and avian (blue) groups.</p>   </div> <p>An SVC was used to predict the human adaptation of all IAV sequence samples, with the optimized (d)nts, with the same number of tail (d)nts in the sorted list as control. The true negative rate and the true positive rate for the control (d)nts were 64.76% and 95.58%, respectively, for the PB2 segment (upper-left panel, <a href="#msz276-F5">fig. 5<i>A</i>);</a> their average AUC for the 5-fold tests was 0.861 ± 0.004 (upper-right panel, <a href="#msz276-F5">fig. 5<i>A</i></a>). However, the true negative/positive rates for the optimized (d)nts for the PB2 segment were 98.45% and 94.10%, respectively (lower-left panel, <a href="#msz276-F5">fig. 5<i>A</i></a>), and the average AUC increased to 0.995 ± 0.001 (lower-right panel, <a href="#msz276-F5">fig. 5<i>A</i></a>). As indicated in <a href="#msz276-F5">figure 5<i>B</i>–<i>F</i></a>, the prediction and the probability of the optimized (d)nt-based SVC was markedly higher than that of the control (d)nt-based SVC. High performance with the optimized (d)nts was also obtained with the other three supervised learning models (GBRT, RF classifier, and MLP classifier) (<a href="#sup1">supplementary Figs. 16–18</a>, <a href="#sup1">Supplementary Material</a> online, respectively).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 5.</span></a></div>   <p>The prediction of human adaptation classes (true/false) and the human adaptation probability by the SVC model, with optimized (d)nts for each segment. The human adaptation classes (true/false) and the human adaptation probability of avian and human sequences were predicted by the SVC with the optimized (best) 9–13 (d)nts for the six segments, with the same optimized-(d)nt number tail (worst) (d)nts as the control. The confusion matrix for human adaptation class prediction, the ROC curve and the area under the ROC curve (AUC) for the SVC model with the worst or best (d)nts are indicated, respectively, for PB2 (<i>A</i>), PB1 (<i>B</i>), PA (<i>C</i>), HA (<i>D</i>), NP (<i>E</i>), and NA (<i>F</i>).</p>   </div> <p>The human adaptation of the sequence-resampled sequence set from the United States and all-sequence set was predicted (with an SVC probability threshold of 0.5) by the above-mentioned SVC model. Then, the association of such adaptation was analyzed with sequence labels, such as subtype and host. Regardless of the other labels, the H1N2 subtype was highly adaptive to humans (approximately 75% by both sequence sets, the left and right parts in <a href="#msz276-F6">fig. 6<i>A</i></a>), as well as the designated human-adaptive H3N2 and H1N1 subtypes. There were 10% or more human-adaptive sequences for H2N2 and H11N9, mixed or H4N8 sequences in the sequence-resampled sequence set from the United States (left part of <a href="#msz276-F6">fig. 6<i>A</i></a>), and H16N3 and H4N6 in the all-sequence set (right part of <a href="#msz276-F6">fig. 6<i>A</i></a>). In terms of the host, 4.4% or 2.5% human sequences, mainly from human-infected AIVs, were not adaptive to humans (<a href="#msz276-F6">fig. 6<i>B</i></a>), and almost 70% of swine sequences were human-adaptive sets (<a href="#msz276-F6">fig. 6<i>B</i></a>). Surprisingly, 16.4% and 19.4% of the turkey sequences from the two sets were human-adaptive sets, ranking first for avian hosts; more than 10% sequences were human-adaptive sets from other birds, such as American black ducks, shorebirds, gulls, blue-winged teal, and quails in either sequence set (<a href="#msz276-F6">fig. 6<i>B</i></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 6.</span></a></div>   <p>The distribution of the subtypes and hosts of human-adaptive IAV sequences before the 2009 influenza pandemic. The human adaptation was predicted by the SVC model with a probability threshold of 0.5. The total number of sequences, the number of human-adaptive sequences, and the human adaptation ratio are quantitatively presented by the size of the red and green circles and the height of the blue histogram in polar coordinates. The labels of the top 20 subtypes (<i>A</i>) or the top 20 hosts (<i>B</i>) for the US-resampled sequences (left part of A/B) or for all IAV sequences (right part of A/B) are indicated. The subtypes and hosts are listed in descending order.</p>   </div> <p>Additionally, when both the segment and subtype labels were taken into account, more different details appeared in such adaptations. As indicated in <a href="#msz276-F7">figure 7<i>A</i></a>, most of the human-adaptive sequences (more than 50%) from swine were H3 or H1 in the hemagglutinin subtype and were N1 or N2 in neuraminidase subtype, with H3N8 as an exception. For avian sequences, H11N9, H4N8, H16N3, H3N2, H4N6, H5N2, H1N1, and H3N8 were at the top of the list (<a href="#msz276-F7">fig. 7<i>B</i></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">F <div class="sc_UNKNOWN">
Ig</div>. 7.</span></a></div>   <p>The prediction of the human adaptation of swine and avian IAV sequences before the 2009 influenza pandemic. Human-adaptive swine (<i>A</i>) and avian (<i>B</i>) IAV sequences before the 2009 influenza pandemic were predicted by the SVC model with a probability threshold of 0.5. The total number of sequences, the number of adaptive sequences, and the human adaptation ratio are represented by the orange, blue, and yellow histograms, respectively. The human adaptation ratio is also presented as a floating number above the stacked histogram. IAV subtypes are indicated by the labels along the <i>x</i>-axis.</p>   </div> </div> <div class="humanadaptationofavianandswineiavsbeforeandafterthe2009h1n1influenzapandemic" title="sec"> <div class="title" tagx="title" title="title">
Human Adaptation of Avian and Swine IAVs before and after the 2009 H1N1 Influenza Pandemic</div> <p>A(H1N1)pdm09 viruses, also known as swine-origin IAVs (H1N1) (S-OIVs), first emerged in North America (<a href="#msz276-B37">Smith et al. 2009</a>) and spread all over the world within the following 6 months (<a href="#msz276-B39">Swerdlow et al. 2011</a>; <a href="#msz276-B11">Fineberg 2014</a>). Origins and evolutionary genomics of S-OIVs have been well identified (<a href="#msz276-B12">Garten et al. 2009</a>; <a href="#msz276-B37">Smith et al. 2009</a>). Given the high importance in influenza pandemics of swine as a mixer for human and avian IAVs (<a href="#msz276-B45">Vijaykrishna et al. 2011</a>; <a href="#msz276-B26">Nelson et al. 2015</a>), we analyzed the worldwide distribution of human-adaptive swine and avian sequences before the 2009 H1N1 influenza pandemic. It was indicated that North America and East Asia were the high-risk areas of human-adaptive swine sequences (<a href="#msz276-T1">table 1</a>) for the six IAV segments, based on the human adaptation ratio. Notably, the United States is ranked first for the absolute number of human-adaptive sequences for each segment. East Asia, particularly Hong Kong and mainland China, had larger human adaptation ratios. Human-adaptive avian sequences are dominantly distributed in the United States and China (<a href="#msz276-T1">table 1</a>). Interestingly, the United States still led the world in the number or proportion of human-adaptive sequences for the five segments (<a href="#msz276-T1">table 1</a>), except NA (maximum in China) (<a href="#msz276-T1">table 1</a>). Accordingly, such distribution bias of human-adaptive swine and avian sequences was consistent with the origin of each segment for the 2009 H1N1 pandemic (<a href="#msz276-B37">Smith et al. 2009</a>).</p> <div class="table-wrap_UNKNOWN" id="msz276-T1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Human Adaptation for Swine and Avian IAV Sequences before the 2009 H1N1 Influenza Pandemic in Different Countries/Areas.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="(" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Host</th> <th colspan="12" rowspan="1">Country/Area, Adaptation ratio (%) (Number of adaptive/total sequences) <hr /> </th> </tr> <tr> <th colspan="2" rowspan="1">PB2</th> <th colspan="2" rowspan="1">PB1</th> <th colspan="2" rowspan="1">PA</th> <th colspan="2" rowspan="1">HA</th> <th colspan="2" rowspan="1">NP</th> <th colspan="2" rowspan="1">NA</th> </tr> </thead> <tbody> <tr> <td rowspan="10" colspan="1">Swine</td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>100.0 (20/20)</b></td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>99.2 (397/400)</b></td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>100.0 (19/19)</b></td> <td rowspan="1" colspan="1"><b>DK</b></td> <td rowspan="1" colspan="1"><b>100.0 (21/21)</b></td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>100.0 (20/20)</b></td> <td rowspan="1" colspan="1"><b>DK</b></td> <td rowspan="1" colspan="1"><b>84.2 (32/38)</b></td> </tr> <tr> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>69.7 (85/122)</b></td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>94.7 (18/19)</b></td> <td rowspan="1" colspan="1"><b>UK</b></td> <td rowspan="1" colspan="1"><b>96.9 (62/64)</b></td> <td rowspan="1" colspan="1"><b>UK</b></td> <td rowspan="1" colspan="1"><b>97.2 (69/71)</b></td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>97.3 (427/439)</b></td> <td rowspan="1" colspan="1"><b>KR</b></td> <td rowspan="1" colspan="1"><b>68.8 (22/32)</b></td> </tr> <tr> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>57.7 (45/78)</b></td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>91.7 (44/48)</b></td> <td rowspan="1" colspan="1"><b>IT</b></td> <td rowspan="1" colspan="1"><b>94.7 (18/19)</b></td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>93.1 (27/29)</b></td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>91.7 (44/48)</b></td> <td rowspan="1" colspan="1"><b>IT</b></td> <td rowspan="1" colspan="1"><b>57.5 (23/40)</b></td> </tr> <tr> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">32.6 (125/384)</td> <td rowspan="1" colspan="1"><b>KR</b></td> <td rowspan="1" colspan="1"><b>82.1 (23/28)</b></td> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>91.9 (136/148)</b></td> <td rowspan="1" colspan="1"><b>IT</b></td> <td rowspan="1" colspan="1"><b>90.5 (38/42)</b></td> <td rowspan="1" colspan="1"><b>KR</b></td> <td rowspan="1" colspan="1"><b>83.3 (20/24)</b></td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>55.3 (26/47)</b></td> </tr> <tr> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">31.1 (14/45)</td> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>78.1 (121/155)</b></td> <td rowspan="1" colspan="1"><b>TH</b></td> <td rowspan="1" colspan="1"><b>77.8 (7/9)</b></td> <td rowspan="1" colspan="1"><b>DE</b></td> <td rowspan="1" colspan="1"><b>87.8 (36/41)</b></td> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>78.6 (125/159)</b></td> <td rowspan="1" colspan="1"><b>UK</b></td> <td rowspan="1" colspan="1"><b>53.0 (35/66)</b></td> </tr> <tr> <td rowspan="1" colspan="1">KR</td> <td rowspan="1" colspan="1">19.4 (6/31)</td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>60.0 (48/80)</b></td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>66.7 (42/63)</b></td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>78.0 (39/50)</b></td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>60.8 (59/97)</b></td> <td rowspan="1" colspan="1">JP</td> <td rowspan="1" colspan="1">42.9 (12/28)</td> </tr> <tr> <td rowspan="1" colspan="1">UK</td> <td rowspan="1" colspan="1">11.1 (7/63)</td> <td rowspan="1" colspan="1">UK</td> <td rowspan="1" colspan="1">9.2 (6/65)</td> <td rowspan="1" colspan="1"><b>KR</b></td> <td rowspan="1" colspan="1"><b>59.3 (16/27)</b></td> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>60.2 (100/166)</b></td> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">19.4 (6/31)</td> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">36.6 (158/432)</td> </tr> <tr> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">7.4 (2/27)</td> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">7.4 (2/27)</td> <td rowspan="1" colspan="1"><b>DE</b></td> <td rowspan="1" colspan="1"><b>52.6 (10/19)</b></td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>59.6 (68/114)</b></td> <td rowspan="1" colspan="1">UK</td> <td rowspan="1" colspan="1">12.1 (8/66)</td> <td rowspan="1" colspan="1">DE</td> <td rowspan="1" colspan="1">32.6 (14/43)</td> </tr> <tr> <td rowspan="1" colspan="1">FR</td> <td rowspan="1" colspan="1">6.2 (1/16)</td> <td rowspan="1" colspan="1">ES</td> <td rowspan="1" colspan="1">0.0 (0/16)</td> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">48.6 (192/395)</td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>59.2 (274/463)</b></td> <td rowspan="1" colspan="1">FR</td> <td rowspan="1" colspan="1">5.9 (1/17)</td> <td rowspan="1" colspan="1">CN</td> <td rowspan="1" colspan="1">27.2 (25/92)</td> </tr> <tr> <td rowspan="1" colspan="1">DE</td> <td rowspan="1" colspan="1">0.0 (0/25<sup>b</sup>)</td> <td rowspan="1" colspan="1">DE</td> <td rowspan="1" colspan="1">0.0 (0/33)</td> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">42.2 (19/45)</td> <td rowspan="1" colspan="1">TH</td> <td rowspan="1" colspan="1">48.4 (15/31)</td> <td rowspan="1" colspan="1">DE</td> <td rowspan="1" colspan="1">0.0 (0/30)</td> <td rowspan="1" colspan="1">HK</td> <td rowspan="1" colspan="1">16.8 (46/274)</td> </tr> <tr> <td rowspan="10" colspan="1">Avian</td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>8.4 (190/2,266)</b></td> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">3.5 (17/479)</td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>33.2 (758/2,286)</b></td> <td rowspan="1" colspan="1"><b>AUS</b></td> <td rowspan="1" colspan="1"><b>51.2 (22/43)</b></td> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">1.7 (33/1,935)</td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>7.0 (3/43)</b></td> </tr> <tr> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>5.0 (27/542)</b></td> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">2.2 (51/2,282)</td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>28.9 (129/446)</b></td> <td rowspan="1" colspan="1"><b>DE</b></td> <td rowspan="1" colspan="1"><b>36.6 (15/41)</b></td> <td rowspan="1" colspan="1">HK</td> <td rowspan="1" colspan="1">1.6 (1/63)</td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>5.7 (23/407)</b></td> </tr> <tr> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">3.4 (16/474)</td> <td rowspan="1" colspan="1">CN</td> <td rowspan="1" colspan="1">0.4 (2/534)</td> <td rowspan="1" colspan="1"><b>NL</b></td> <td rowspan="1" colspan="1"><b>26.7 (58/217)</b></td> <td rowspan="1" colspan="1"><b>RU</b></td> <td rowspan="1" colspan="1"><b>19.5 (8/41)</b></td> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">0.2 (1/417)</td> <td rowspan="1" colspan="1">TH</td> <td rowspan="1" colspan="1">3.0 (1/33)</td> </tr> <tr> <td rowspan="1" colspan="1">HK</td> <td rowspan="1" colspan="1">3.2 (2/62)</td> <td rowspan="1" colspan="1">VN</td> <td rowspan="1" colspan="1">0.0 (0/89)</td> <td rowspan="1" colspan="1"><b>AUS</b></td> <td rowspan="1" colspan="1"><b>17.0 (8/47)</b></td> <td rowspan="1" colspan="1"><b>CA</b></td> <td rowspan="1" colspan="1"><b>18.2 (87/478)</b></td> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">0.0 (0/79)</td> <td rowspan="1" colspan="1">TW</td> <td rowspan="1" colspan="1">2.6 (1/39)</td> </tr> <tr> <td rowspan="1" colspan="1">JP</td> <td rowspan="1" colspan="1">3.1 (2/65)</td> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">0.0 (0/91)</td> <td rowspan="1" colspan="1"><b>TW</b></td> <td rowspan="1" colspan="1"><b>8.1 (3/37)</b></td> <td rowspan="1" colspan="1"><b>US</b></td> <td rowspan="1" colspan="1"><b>17.5 (352/2,008)</b></td> <td rowspan="1" colspan="1">VN</td> <td rowspan="1" colspan="1">0.0 (0/92)</td> <td rowspan="1" colspan="1">HK</td> <td rowspan="1" colspan="1">1.9 (1/52)</td> </tr> <tr> <td rowspan="1" colspan="1">NL</td> <td rowspan="1" colspan="1">0.4 (1/225)</td> <td rowspan="1" colspan="1">NL</td> <td rowspan="1" colspan="1">0.0 (0/219)</td> <td rowspan="1" colspan="1"><b>CN</b></td> <td rowspan="1" colspan="1"><b>6.8 (30/440)</b></td> <td rowspan="1" colspan="1"><b>JP</b></td> <td rowspan="1" colspan="1"><b>14.3 (10/70)</b></td> <td rowspan="1" colspan="1">NL</td> <td rowspan="1" colspan="1">0.0 (0/209)</td> <td rowspan="1" colspan="1">US</td> <td rowspan="1" colspan="1">1.7 (17/976)</td> </tr> <tr> <td rowspan="1" colspan="1">AUS</td> <td rowspan="1" colspan="1">0.0 (0/47)</td> <td rowspan="1" colspan="1">SE</td> <td rowspan="1" colspan="1">0.0 (0/237)</td> <td rowspan="1" colspan="1"><b>DE</b></td> <td rowspan="1" colspan="1"><b>6.2 (2/32)</b></td> <td rowspan="1" colspan="1"><b>NL</b></td> <td rowspan="1" colspan="1"><b>13.4 (28/209)</b></td> <td rowspan="1" colspan="1">SE</td> <td rowspan="1" colspan="1">0.0 (0/232)</td> <td rowspan="1" colspan="1">SE</td> <td rowspan="1" colspan="1">1.6 (1/62)</td> </tr> <tr> <td rowspan="1" colspan="1">RU</td> <td rowspan="1" colspan="1">0.0 (0/47)</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1"><b>IL</b></td> <td rowspan="1" colspan="1"><b>5.4 (2/37)</b></td> <td rowspan="1" colspan="1"><b>HK</b></td> <td rowspan="1" colspan="1"><b>9.4 (6/64)</b></td> <td rowspan="1" colspan="1">CN</td> <td rowspan="1" colspan="1">0.0 (0/522)</td> <td rowspan="1" colspan="1">NL</td> <td rowspan="1" colspan="1">0.9 (1/106)</td> </tr> <tr> <td rowspan="1" colspan="1">TH</td> <td rowspan="1" colspan="1">0.0 (0/54)</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">JP</td> <td rowspan="1" colspan="1">4.8 (3/63)</td> <td rowspan="1" colspan="1"><b>SE</b></td> <td rowspan="1" colspan="1"><b>8.5 (20/236)</b></td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">CA</td> <td rowspan="1" colspan="1">0.5 (1/184)</td> </tr> <tr> <td rowspan="1" colspan="1">IT</td> <td rowspan="1" colspan="1">0.0 (0/82)</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">HK</td> <td rowspan="1" colspan="1">3.1 (2/64)</td> <td rowspan="1" colspan="1"><b>TW</b></td> <td rowspan="1" colspan="1"><b>7.7 (3/39)</b></td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">\</td> <td rowspan="1" colspan="1">IL</td> <td rowspan="1" colspan="1">0.0 (0/36)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label" /><p>Countries/areas were abbreviated as Australia: AUS, Canada: CA, China: CN, Spain: ES, France: FR, Germany: DE, Denmark: DK, Hong Kong, China: HK, Israel: IL, Italy: IT, Japan: JP, South Korea: KR, Netherlands: NL, Russia: RU, Sweden: SE, Taiwan, China: TW, Thailand: TH, United Kingdom: UK, USA: US, Vietnam: VN.</p> </div> </div> </div> <p>Furthermore, the hierarchical clustering analysis of the S-OIVs with the IAVs before 2009 was performed based on the SVC model-characterized (d)nts by random sampling 1,000 samples from the total samples. As indicated (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 19<i>A</i></a>, <a href="#sup1">Supplementary Material</a> online), the S-OIV (A(H1N1)pdm09) PB2 sequences were clustered most closely with the PB2 sequences from several avian viruses (H6N8, H11N2, H11N9, and mixed subtypes) in Delaware in 1994 and 1995 and then with the avian and swine viruses (H1N1, H3N2, and H5N2) in Delaware and other US states in the 2000s. S-OIV PB1 was most closely clustered with the human H3N2 viruses before or after 2000 in the United States (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 19<i>B</i></a>, <a href="#sup1">Supplementary Material</a> online). The close clustering with various subtypes of avian/swine viruses mainly in the United States and sporadically in East Asia for the PA segment (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 19<i>C</i></a>, <a href="#sup1">Supplementary Material</a> online) and the neighboring with swine H1N1, H1N2, and H3N2 viruses in the United States /Asia/Europe for the HA, NP, and NA segments were indicated (<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msz276#supplementary-data">supplementary fig. 19<i>D</i>–<i>F</i></a>, <a href="#sup1">Supplementary Material</a> online). Such kinds of clustering for the six segments were also indicated by the other nine rounds of hierarchical clustering analysis with random-resampled sequences (random state = 2–10) (supporting data for <a href="#sup1">supplementary fig. 19</a>, <a href="#sup1">Supplementary Material</a> online). Interestingly, the results were consistent with a previous evolutionary analysis with full cDNA sequences (<a href="#msz276-B37">Smith et al. 2009</a>). Moreover, as indicated by the human adaptation probability in each clustering hierarchy (the last floating number in each sequence name, <a href="#sup1">supplementary fig. 19</a>, <a href="#sup1">Supplementary Material</a> online), almost all of the A(H1N1)pdm09-neighboring avian or swine sequences were predicted by our model to be human-adaptive sequences.</p> <p>The influence of the 2009 H1N1 influenza pandemic on the human adaptation of avian and swine IAVs post-2009 was also evaluated. As indicated in <a href="#msz276-T2">table 2</a>, all six segments of avian sequences decreased in terms of the human adaptation ratio, although the adaptive sequence number increased. In particular, the adaptive sequences of PB1, PA, and HA increased much more (over the median level). For swine viruses, all six segments of the sequences, particularly for NA, PA, and HA (over the median level), increased in terms of both the human adaptation ratio and adaptive sequence number.</p> <div class="table-wrap_UNKNOWN" id="msz276-T2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Human Adaptation Changes in Avian, Swine, and Human IAVs after the 2009 H1N1 Influenza Pandemic.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Segment</th> <th rowspan="2" colspan="1">Period/Change</th> <th colspan="3" rowspan="1">Adaptation Ratio (%) (adaptive/total sequences) and the Change (%) <hr /> </th> </tr> <tr> <th rowspan="1" colspan="1">Avian</th> <th rowspan="1" colspan="1">Swine</th> <th rowspan="1" colspan="1">Human</th> </tr> </thead> <tbody> <tr> <td rowspan="3" colspan="1">PB2</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1">5.4 (249/4,608)</td> <td rowspan="1" colspan="1">35.6 (311/873)</td> <td rowspan="1" colspan="1">87.3 (3,397/3,893)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1">2.6 (334/12,846)</td> <td rowspan="1" colspan="1">35.9 (1,574/4,379)</td> <td rowspan="1" colspan="1">95.6 (16,415/17,179)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−51.85</td> <td rowspan="1" colspan="1">0.84</td> <td rowspan="1" colspan="1">9.51</td> </tr> <tr> <td rowspan="3" colspan="1">PB1</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1"><b>1.91 (88/4,610)</b></td> <td rowspan="1" colspan="1">71.2 (664/933)</td> <td rowspan="1" colspan="1">93.2 (3,627/3,890)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1"><b>1.87 (236/12,598)</b></td> <td rowspan="1" colspan="1">84.5 (3,734/4,421)</td> <td rowspan="1" colspan="1">97.6 (16,456/16,856)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−<b>2.11</b></td> <td rowspan="1" colspan="1">18.68</td> <td rowspan="1" colspan="1">4.72</td> </tr> <tr> <td rowspan="3" colspan="1">PA</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1"><b>22.7 (1,019/4,487)</b></td> <td rowspan="1" colspan="1"><b>64.7 (549/849)</b></td> <td rowspan="1" colspan="1">95.9 (3,633/3,790)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1"><b>19.0 (2,351/12,398)</b></td> <td rowspan="1" colspan="1"><b>84.2 (3,508/4,168)</b></td> <td rowspan="1" colspan="1">98.2 (16,550/16,856)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−<b>16.30</b></td> <td rowspan="1" colspan="1"><b>30.14</b></td> <td rowspan="1" colspan="1">2.40</td> </tr> <tr> <td rowspan="3" colspan="1">HA</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1"><b>14.0 (604/4,314)</b></td> <td rowspan="1" colspan="1"><b>66.7 (753/1,129)</b></td> <td rowspan="1" colspan="1">94.5 (4,809/5,091)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1"><b>11.5 (1,398/12,158)</b></td> <td rowspan="1" colspan="1"><b>79.4 (7,234/9,113)</b></td> <td rowspan="1" colspan="1">97.7 (27,508/28,158)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−<b>17.86</b></td> <td rowspan="1" colspan="1"><b>19.04</b></td> <td rowspan="1" colspan="1">3.39</td> </tr> <tr> <td rowspan="3" colspan="1">NP</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1">1.1 (47/4,133)</td> <td rowspan="1" colspan="1">72 (717/996)</td> <td rowspan="1" colspan="1">95.7 (3,517/3,676)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1">0.6 (65/11,005)</td> <td rowspan="1" colspan="1">83.8 (3,709/4,427)</td> <td rowspan="1" colspan="1">98 (14,101/14,385)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−45.45</td> <td rowspan="1" colspan="1">16.39</td> <td rowspan="1" colspan="1">2.40</td> </tr> <tr> <td rowspan="3" colspan="1">NA</td> <td rowspan="1" colspan="1">Before 2009</td> <td rowspan="1" colspan="1">2.4 (58/2,379)</td> <td rowspan="1" colspan="1"><b>35.8 (429/1,200)</b></td> <td rowspan="1" colspan="1">93.6 (3,544/3,787)</td> </tr> <tr> <td rowspan="1" colspan="1">2009–2018</td> <td rowspan="1" colspan="1">1.4 (109/7,734)</td> <td rowspan="1" colspan="1"><b>54.8 (4,648/8,482)</b></td> <td rowspan="1" colspan="1">97.8 (19,944/20,386)</td> </tr> <tr> <td rowspan="1" colspan="1">Change</td> <td rowspan="1" colspan="1">−41.67</td> <td rowspan="1" colspan="1"><b>53.07</b></td> <td rowspan="1" colspan="1">4.49</td> </tr> </tbody> </table> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>IAVs must acquire sufficient human adaptation before they can promote human pandemics. To date, there has been no universal definition of IAVs’ human adaptation, although numerous human-adaptive viral determinants have been reported (<a href="#msz276-B42">Taubenberger and Kash 2010</a>; <a href="#msz276-B3">Bouvier 2015</a>; <a href="#msz276-B23">Long et al. 2019</a>). In the present study, we defined it as the capability to infect humans easily, to transmit among populations efficiently, and to be virulent to some degree to humans. Accordingly, the human-adaptive IAVs were limited to the H3N2 and H1N1 viruses, either of which can continuously cause endemics or even pandemics in humans (<a href="#msz276-B33">Ren et al. 2016</a>), whereas other subtypes of avian IAVs (<a href="#msz276-B49">Yang et al. 2007</a>; <a href="#msz276-B34">Rudge and Coker 2013</a>; <a href="#msz276-B14">Hu et al. 2014</a>) were classified into the avian-adaptive group. There might be a concern about a selection bias of the human adaptation criteria. If so, the "human adaptation�? label for the four segments (PB2, PB1, PA, and NP) would be wrongly associated with "H3N2�? or "H1N1,�? which would be inconsistent with the "true�? human adaptation of these segments. Under such circumstances, the "true�? human-adapted PB2, PB1, PA, and NP sequences of the human-adaptive H2N2 virus (<a href="#msz276-B5">Cox and Subbarao 2000</a>) might be underestimated, and the host adaptation of the four segments of swine H1N2 virus would not be correctly predicted. However, high HA and NA of the human adaptation frequencies were unanimously predicted for the PB2, PB1, PA, and NP sequences for both the H2N2 and H1N2 viruses. Interestingly, since 2005, dozens of human H1N2 infection cases have been reported in the United States (Pulit-Penaloza et al. 2018) and the Netherlands (<a href="#msz276-B25">Meijer et al. 2018</a>); the high human adaptation of the H1N2 virus was also experimentally supported (Pulit-Penaloza et al. 2018). Therefore, the IAV human adaptation criteria are acceptable to some degree, according to the existing human-adapted IAVs.</p> <p>In the last few decades, the overwhelming majority of studies on viral determinants have focused on protein levels for virus infection, transmission, virulence, and host adaptation. In particular, distinct protein signatures for host tropism (<a href="#msz276-B9">Eng et al. 2016</a>) and avian-to-human transmission (<a href="#msz276-B31">Qiang et al. 2018</a>) have been recognized with ML methods. Recently, accumulating reports found a significant influence of synonymous viral nucleotide or dinucleotide mutation on the virus response to the host’s innate immune system (<a href="#msz276-B40">Takata et al. 2017</a>) on virus virulence (<a href="#msz276-B1">Atkinson et al. 2014</a>; <a href="#msz276-B44">Tulloch et al. 2014</a>) and virus replication (<a href="#msz276-B48">Witteveldt, Martin-Gans and Simmonds 2016</a>). Here, we compressed the full-length coding information of the six segments into the counting information of 12 nts and 48 dnts, all of which were sorted according to their classification importance with the PCA/SVC method. Optimization is one of the crucial parts of machine/deep learning. A moving average (MA; <a href="#msz276-B18">Kashyap 1982</a>), also known as the rolling mean, was utilized here to optimize the number of features for the ML models, with the crossing of the MA with its MA3 value as a cutoff point, at which the number of (d)nts was the best.</p> <p>Interestingly, the counting information of each mono- or dinucleotide varied in the importance of each segment. Besides, 9–13 optimized (d)nts were enough to predict the human adaptation for each of the six segments. Given the high performance in the avian/human adaptation classification, no other optimization methods were explored here.</p> <p>The species-specific (<a href="#msz276-B13">Glass et al. 2007</a>) and virus-family-specific (<a href="#msz276-B8">Di Giallonardo et al. 2017</a>) dinucleotide composition has been computationally explored for viruses. The genomic dinucleotide composition of RNA viruses is useful for predicting viral reservoir hosts and arthropod vectors (<a href="#msz276-B2">Babayan et al. 2018</a>). Therefore, we speculated here that the mono-/dinucleotide composition should be another critical genomic feature, and we assume here that there should be a species selection bias of IAV nucleotides/dinucleotides. Taking PB2 as an example, the frequency of T, C, A, or G at the first position and G at the third position within a codon, the odds ratios of ct_N3M1, ag_N12 and at_N12 determined the human adaptation of IAVs. According to the eukaryotic codon list (<a href="#msz276-B35">Shu 2017</a>), every amino acid is coded by one (for methionine and tryptophan) to six trinucleotide codons (for leucine, serine, and arginine); six (phenylalanine, leucine, serine, tyrosine, cysteine, and tryptophan), five, seven, and five types of amino acids were respectively dependent on the nucleotides of T, C, A, and G, at the first nucleotide position within a codon, 4–7 types of amino acids were dependent on the four types of nucleotides at the first nucleotide position, and 13–15 types of amino acids were dependent on the four types of nucleotides at the third nucleotide position. Therefore, each mononucleotide feature is theoretically associated only with 5–15 possible amino acids (the stop codon is not considered). Accordingly, every dinucleotide is associated with 1–4 types of amino acids (<a href="#msz276-B35">Shu 2017</a>). Therefore, the nucleotide composition was associated only with the amino acid compositional information, with less than 50% probability. Thus, we speculated that the genomic composition of mono-/dinucleotides is another essential genomic characteristic of IAVs and is probably biologically associated with the host adaptation of IAVs.</p> <p>An A(H1N1)pdm09 virus caused the latest worldwide influenza pandemic (<a href="#msz276-B39">Swerdlow et al. 2011</a>; <a href="#msz276-B11">Fineberg 2014</a>). Here, our results regarding the high human adaptability of swine IAVs before 2009 in the United States precisely predicted the high risk of these IAVs. Of course, a possible underestimation of human-adaptive swine viruses/sequences was not excluded in many high-risk developing countries, such as China and Vietnam, due to a likely undeveloped monitoring/detection program for swine influenza. However, a marked lower human adaptation of the swine IAV sequence was also indicated by our model in the area of the European Union, which is the second largest pig plantation area, with twice as much pig production in this area in 2018 than the United States (<a href="https://www.statista.com/statistics/273232/net-pork-production-worldwide-by-country/">https://www.statista.com/statistics/273232/net-pork-production-worldwide-by-country/</a>). Moreover, the 2009 H1N1 pandemic was not initiated in China, although pork production in China is 4-fold that of the United States. This phenomenon implies that our results might reveal the "true�? severity of swine infection of human-adaptive swine viruses in the United States rather than in other areas before 2009.</p> <p>In summary, the human-adaptive and avian-adaptive nucleotide compositions of influenza A viruses (IAVs) were determined with supervised/unsupervised ML methods. ML, based on human-adaptive nucleotide composition, performed well in predicting the human adaptation of IAVs before the 2009 H1N1 pandemic. This approach might be promising for the prediction of the risk of an influenza pandemic and global vulnerability to influenza.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="sequencedataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Sequence Data Processing</div> <p>Full-length coding sequences of the first six IAV segments of PB2, PB1, PA, HA, NP, and NA (the M and NS segments are not included due to their short length) were utilized for the nucleotide composition analysis. In total, 115,917 human sequence samples, 76,538 avian sequences, and 35,569 swine sequences, up to December 31, 2018, were downloaded from the Influenza Research Database (IRD) (<a href="#msz276-B51">Zhang et al. 2017</a>) or from the Global Initiative on Sharing All Influenza Data (GISAID) database (Shu and McCauley 2017). The ID, strain name, sequence length, and other labels were extracted from the definition content of sequence file in FASTA format via a Python script (Script-1, <a href="#sup1">supplementary data</a>, <a href="#sup1">Supplementary Material</a> online). The ID number and other labels are listed in the <a href="#sup1">supplementary data</a>, <a href="#sup1">Supplementary Material</a> online. The composition of mononucleotide (nt, T, C, A, and G) and dinucleotides (dnts, 16 types of combination of every two nts) were counted and calculated according to formulas 1 and 2/3 (Script-2, supplementary file, <a href="#sup1">Supplementary Material</a> online) for each of the three types of nucleotide positions within a trinucleotide codon (<a href="#msz276-B17">Karlin and Mrazek 1997</a>). In total, there were 60 (d)nts, including the frequency of 12 types of nts (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div></div></span>) and the relative frequency of 48 types of dnts (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> </div></div></span>). </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
codon</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
nt</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
position</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
)</div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Σ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
16</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
for</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
for</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
codon</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
nt</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
position</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
)</div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mi_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
fr</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN"> </div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN"> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
codon</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
nt</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
position</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
for</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
for</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
)</div></div></div> <p /> <p>Sequences with duplicate ID, incorrect labels, or out-of-length ranges were excluded, and the remaining 217,549 sequences are listed in <a href="#sup1">supplementary table 1</a>, <a href="#sup1">Supplementary Material</a> online. To avoid a country/area bias for ML modeling, due to the overwhelming majority of US samples, we randomly resampled the US sequences, with the United States to China ratio of approximately 1:1 for each segment (Script-3, supplementary file, <a href="#sup1">Supplementary Material</a> online), via the pandas.DataFrame.sample (Python) model. In total, 46,042 randomly resampled human-adaptive sequences and 46,488 human-inadaptive avian sequences were utilized for feature extraction and model building. Human-originated H5N1, H7N9, and other subtypes were excluded from the human adaptation set and were not included in the avian set for model building.</p> </div> <div class="thephylogeneticanalysisofrandomlysampledhuman/avianiavsequencesusingthemaximumlikelihoodmethod" title="sec"> <div class="title" tagx="title" title="title">
The Phylogenetic Analysis of Randomly Sampled Human/Avian IAV Sequences Using the Maximum Likelihood Method</div> <p>In total, 59–61 sequence samples were randomly selected (random state = 1) from each segment sequence set (3.59–5.01% of the total sequences) via pandas.DataFrame.sample (Python). Then, MEGA (MEGA 7.0.26, <a href="#msz276-B20">Kumar et al. 2016</a>) was utilized to build a maximum likelihood tree with the Tamura–Nei model (<a href="#msz276-B41">Tamura and Nei 1993</a>) for PB2 (A) and the other five segments (B–F). The parameters were set as follows: uniform rates among sites, gaps complete deletion, the ML heuristic method set to the nearest-neighbor interchange, and with making the initial tree automatically (default—NJ/BioNJ) as an initial tree. Multiple and pairwise alignments were performed with ClustalW with a gap-opening penalty of 15, a gap extension penalty of 6.66, an IUB DNA weight matrix, and a transition weight of 0.5 before a phylogenetic tree was built. Another two rounds of random resampling were performed from each segment sequence set with the same sampling ratio as mentioned above, and the maximum likelihood tree was built with the same parameters.</p> </div> <div class="machinelearninganalysis" title="sec"> <div class="title" tagx="title" title="title">
Machine Learning Analysis</div> <p>ML analysis was performed with Python. The Scikit-learn package (version = 0.18.1, <a href="https://scikit-learn.org/stable/#">https://scikit-learn.org/stable/#</a>) was utilized for PCA (sklearn.decomposition.PCA) analysis and the supervised ML methods of SVC, GBRT, MLP classifier, and RF classifier from the submodel of sklearn.svm.SVC, sklearn.ensemble.GradientBoostingClassifier, sklearn.neural_network.MLPClassifier, and sklearn.ensemble.RandomForestClassifier, respectively. The data were split with StratifiedKFold from sklearn.model_selection (n_splits = 5, random_state = 1, shuffle = True) into five training/test sets before supervised learning was implemented (Script-4, supplementary file, <a href="#sup1">Supplementary Material</a> online). The SciPy package (cluster.hierarchy, version = 0.19.0, <a href="https://www.scipy.org">https://www.scipy.org</a>) was utilized to build a hierarchical clustering of the IAV sequences based on the Euclidean distance between/among sequences.</p> <p>PCA is a widely utilized unsupervised ML model for constructing a low-rank model of a data matrix. For the following (d)nt sorting, an orthogonal transformation by PCA (<a href="#msz276-B16">Jolliffe and Cadima 2016</a>) was performed to convert every four (d)nts with possible correlations into one principal component, with the most significant possible variance (formula 4). For the evaluation of the separability between avian and human sequences, PCA was also utilized to transform the information of all (60) or the optimized (d)nts into two principal components.</p> <p>Hierarchical clustering is another important unsupervised ML method for hierarchical cluster analysis. Strategies for hierarchical clustering generally fall into two types: "bottom-up�? approaches, by which each observation starts from a lower cluster and then are clustered with its paired cluster(s) in a higher hierarchy; and "top-down�? approaches, by which all the observations start from the top cluster and then are split into lower hierarchies recursively down the hierarchy axis. For the hierarchical clustering of the IAV sequences based on all of or the optimized (d)nts, the Euclidean distance was calculated and utilized as a hierarchical clustering scalar (formula 5). </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
minimize</div> <div class="mi_UNKNOWN"> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
‖</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
XY</div> </div> <div class="mo_UNKNOWN">
‖</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ij</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
}</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
ϵ</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
Y</div> <div class="mo_UNKNOWN">
ϵ</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
or</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
n</div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN"> </div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
avian</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN"> </div> <div class="mi_UNKNOWN">
human</div> <div class="mo_UNKNOWN"> </div> <div class="mi_UNKNOWN">
d</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
nt</div> </div> </div> <div class="mo_UNKNOWN">
;</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
60</div></div></div> <p /> <p>An SVC, also known as a support vector machine or a support vector network (<a href="#msz276-B27">Noble 2006</a>), is one of the most popular supervised learning models for classification and regression analysis. An SVM training algorithm builds a model that assigns new samples to one category or another, making it a nonprobabilistic binary linear classifier (<a href="#msz276-B27">Noble 2006</a>) (formula 6). The other three ML models were the GBRT MLP classifier and RFC classifier algorithms, by which a prediction is made to evaluate the probability of human adaptation for each sequence. The GBRT algorithm, also known as gradient tree boosting, is a greedy generalized boosting model for differentiable loss functions. </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
ω</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN"> </div> <div class="mi_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
ω</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mo_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN"> </div> <div class="mn_UNKNOWN">
60</div></div></div> <p /> </div> <div class="featureextraction" title="sec"> <div class="title" tagx="title" title="title">
Feature Extraction</div> <p>To evaluate the importance of each (d)nt, we first sorted the (d)nts with a PCA/SVC combined model. Three thousand and five hundred and forty iterations of PCA/SVC analysis were performed to transform every four (d)nts into one PCA component, which was then utilized for the SVC classification of the avian and human IAV sequences. Thus, the 60 (d)nts were sorted according to their average area under the curve (AUC) (a) of the repeated above-mentioned PCA/SVC analysis. Supervised ML models (GBRT, MLP, RFC, and SVC) were utilized to evaluate the efficiency of the sorted (d)nts as human/avian classification features. Accumulated (d)nts, from 1 to 60 from the sorted list, were input into each of the four models, and the Cross_val score was utilized as an efficiency indicator. The optimized ML (d)nt number was defined as the number of the accumulated (d)nts, with which the Cross_val score did not increase as much as the (d)nt number accumulation, was evaluated by the MA strategy (<a href="#msz276-B36">Slawnych et al. 2009</a>) and was determined at the crossing point of the Cross_val score curve with its MA3 curve.</p> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Original sequence data are available from the Influenza Research Database (IRD, up to December 31, 2018) (<a href="#msz276-B51">Zhang et al. 2017</a>) via the website of <a href="http://www.fludb.org">www.fludb.org</a> and from the Global Initiative on Sharing All Influenza Data (GISAID) (Shu and McCauley 2017) via the website of <a href="http://www.gisad.org">www.gisad.org</a>. All the original data for the results are available online: <a href="https://github.com/Jamalijama/Predict_IAV_Host">https://github.com/Jamalijama/Predict_IAV_Host</a>.</p> </div> <div class="codeavailability" title="sec"> <div class="title" tagx="title" title="title">
Code Availability</div> <p>The source code for the present study is available online: <a href="https://github.com/Jamalijama/Predict_IAV_Host">https://github.com/Jamalijama/Predict_IAV_Host</a>.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">msz276_Supplementary_Data</span><div class="media" title="media"><a href="msz276_supplementary_data.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This work was supported by grants from the National Science and Technology Major Project of China (2017ZX10305501, 2018ZX10732401, 2018ZX10711001-003, AWS16J020, and BWS16J008), the Key Project of China State Key Laboratory of Pathogen and Biosecurity of China (SKLPBS1806 and SKLPBS1407), the National Natural Science Foundation of China (31200119), and the National Key Research and Development Program of China (2018YFC1200100). We gratefully acknowledge the authors, originating and submitting laboratories of the sequences from GISAID’s EpiFlu Database on which this research is based.</p> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>J.L. designed the study with help from T.J., S.Z., B.L., Y.H., and X.P.K.; T.J. and C.F.Q. supervised the project; J.L. coded the scripts for the (d)nt counting, for the feature extraction, and the machine learning models; J.L. performed the data analysis under assistance from T.J., S.Z., X.Y.W., M.T.H., Y.C.L., and Z.P.Z.; C.F.Q. and B.L. gave constructive comments on the article. J.L. wrote and edited the manuscript, with suggestions from T.J., S.Z., Y.H., X.P.K., and Z.P.Z.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="msz276-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witteveldt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="volume" tagx="volume" title="volume">42</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">4527</span>–<span class="lpage" tagx="lpage" title="lpage">4545</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24470146">24470146</a></span></span></li> <li tag="ref"><a name="msz276-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babayan</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orton</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG.</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">362</span>(<span class="issue" tagx="issue" title="issue">6414</span>):<span class="fpage" tagx="fpage" title="fpage">577</span>–<span class="lpage" tagx="lpage" title="lpage">580</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30385576">30385576</a></span></span></li> <li tag="ref"><a name="msz276-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouvier</span><span class="given-names" tagx="given-names" title="given-names">NM.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Animal models for influenza virus transmission studies: a historical perspective</span>. <span class="source" tagx="source" title="source">Curr Opin Virol</span>. <span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">101</span>–<span class="lpage" tagx="lpage" title="lpage">108</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26126082">26126082</a></span></span></li> <li tag="ref"><a name="msz276-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bragstad</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martel</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomsen</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aasted</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fomsgaard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Pandemic influenza 1918 H1N1 and 1968 H3N2 DNA vaccines induce cross-reactive immunity in ferrets against infection with viruses drifted for decades</span>. <span class="source" tagx="source" title="source">Influenza Other Resp</span>. <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">13</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>.</span></li> <li tag="ref"><a name="msz276-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subbarao</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Global epidemiology of influenza: past and present</span>. <span class="source" tagx="source" title="source">Annu Rev Med</span>. <span class="volume" tagx="volume" title="volume">51</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">407</span>–<span class="lpage" tagx="lpage" title="lpage">421</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10774473">10774473</a></span></span></li> <li tag="ref"><a name="msz276-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thanh</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chau</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoang</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chau</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khanh</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">VC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qui</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Fatal outcome of human influenza A (H5N1) is associated with high viral load and hypercytokinemia</span>. <span class="source" tagx="source" title="source">Nat Med</span>. <span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">1203</span>–<span class="lpage" tagx="lpage" title="lpage">1207</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16964257">16964257</a></span></span></li> <li tag="ref"><a name="msz276-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic and genetic characterization of a 2017 clinical isolate of the H7N9 virus in Guangzhou, China, during the fifth epidemic wave</span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1331</span>–<span class="lpage" tagx="lpage" title="lpage">1339</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29019145">29019145</a></span></span></li> <li tag="ref"><a name="msz276-B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Giallonardo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlub</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Dinucleotide composition in animal RNA viruses is shaped more by virus family than by host species</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="volume" tagx="volume" title="volume">91</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">e02381-16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28148785">28148785</a></span></span></li> <li tag="ref"><a name="msz276-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eng</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">TW.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Distinct host tropism protein signatures to identify possible zoonotic influenza A viruses</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">e0150173.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26915079">26915079</a></span></span></li> <li tag="ref"><a name="msz276-B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eng</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">TW.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Predicting zoonotic risk of influenza A viruses from host tropism protein signature using random forest</span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span>. <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1135.</span></span></li> <li tag="ref"><a name="msz276-B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fineberg</span><span class="given-names" tagx="given-names" title="given-names">HV.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Pandemic preparedness and response–lessons from the H1N1 influenza of 2009</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="volume" tagx="volume" title="volume">370</span>(<span class="issue" tagx="issue" title="issue">14</span>):<span class="fpage" tagx="fpage" title="fpage">1335</span>–<span class="lpage" tagx="lpage" title="lpage">1342</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24693893">24693893</a></span></span></li> <li tag="ref"><a name="msz276-B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balish</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessions</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deyde</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Antigenic and genetic characteristics of swine-origin 2009 A(H1N1) influenza viruses are circulating in humans</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">325</span>(<span class="issue" tagx="issue" title="issue">5937</span>):<span class="fpage" tagx="fpage" title="fpage">197</span>–<span class="lpage" tagx="lpage" title="lpage">201</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465683">19465683</a></span></span></li> <li tag="ref"><a name="msz276-B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khulan</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figueroa</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olivier</span><span class="given-names" tagx="given-names" title="given-names">EN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oakley</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Zant</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouhassira</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melnick</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golden</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">CG dinucleotide clustering is a species-specific property of the genome</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="volume" tagx="volume" title="volume">35</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">6798</span>–<span class="lpage" tagx="lpage" title="lpage">6807</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932072">17932072</a></span></span></li> <li tag="ref"><a name="msz276-B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Limited human-to-human transmission of avian influenza A(H7N9) virus, Shanghai, China, March to April 2013</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">25</span>):<span class="fpage" tagx="fpage" title="fpage">pii: 20838</span>.</span></li> <li tag="ref"><a name="msz276-B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwasaki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wada</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikemura</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Novel bioinformatics strategies for prediction of directional sequence changes in influenza virus genomes and for surveillance of potentially hazardous strains</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span>. <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">386.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23964903">23964903</a></span></span></li> <li tag="ref"><a name="msz276-B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolliffe</span><span class="given-names" tagx="given-names" title="given-names">IT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cadima</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Principal component analysis: a review and recent developments</span>. <span class="source" tagx="source" title="source">Phil Trans R Soc A</span>. <span class="volume" tagx="volume" title="volume">374</span>(<span class="issue" tagx="issue" title="issue">2065</span>):<span class="fpage" tagx="fpage" title="fpage">20150202.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26953178">26953178</a></span></span></li> <li tag="ref"><a name="msz276-B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mrazek</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Compositional differences within and between eukaryotic genomes</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="volume" tagx="volume" title="volume">94</span>(<span class="issue" tagx="issue" title="issue">19</span>):<span class="fpage" tagx="fpage" title="fpage">10227</span>–<span class="lpage" tagx="lpage" title="lpage">10232</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9294192">9294192</a></span></span></li> <li tag="ref"><a name="msz276-B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashyap</span><span class="given-names" tagx="given-names" title="given-names">RL.</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Optimal choice of AR and MA parts in autoregressive moving average models</span>. <span class="source" tagx="source" title="source">IEEE Trans Pattern Anal Mach Intell</span>. <span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">99</span>–<span class="lpage" tagx="lpage" title="lpage">104</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21869012">21869012</a></span></span></li> <li tag="ref"><a name="msz276-B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kislinger</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kannan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ignatchenko</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gramolini</span><span class="given-names" tagx="given-names" title="given-names">AO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hallett</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Global survey of organ and organelle protein expression in mouse: combined proteomic and transcriptomic profiling</span>. <span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">125</span>:<span class="fpage" tagx="fpage" title="fpage">173</span>–<span class="lpage" tagx="lpage" title="lpage">186</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16615898">16615898</a></span></span></li> <li tag="ref"><a name="msz276-B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">1870</span>–<span class="lpage" tagx="lpage" title="lpage">1874</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a></span></span></li> <li tag="ref"><a name="msz276-B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The genesis and source of the H7N9 influenza viruses causing human infections in China</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">502</span>(<span class="issue" tagx="issue" title="issue">7470</span>):<span class="fpage" tagx="fpage" title="fpage">241</span>–<span class="lpage" tagx="lpage" title="lpage">244</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23965623">23965623</a></span></span></li> <li tag="ref"><a name="msz276-B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slavinski</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiff</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlino</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daskalakis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rakeman</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misener</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Outbreak of influenza A(H7N2) among cats in an animal shelter with cat-to-human transmission-New York City, 2016</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span>. <span class="volume" tagx="volume" title="volume">65</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1927</span>–<span class="lpage" tagx="lpage" title="lpage">1929</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29020187">29020187</a></span></span></li> <li tag="ref"><a name="msz276-B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mistry</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haslam</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barclay</span><span class="given-names" tagx="given-names" title="given-names">WS.</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Host and viral determinants of influenza A virus species specificity</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="volume" tagx="volume" title="volume">17</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">67</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30487536">30487536</a></span></span></li> <li tag="ref"><a name="msz276-B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maines</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L-M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rowe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falcon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mai</span><span class="given-names" tagx="given-names" title="given-names">LQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sedyaningsih</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Lack of transmission of H5N1 avian-human reassortant influenza viruses in a ferret model</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="volume" tagx="volume" title="volume">103</span>(<span class="issue" tagx="issue" title="issue">32</span>):<span class="fpage" tagx="fpage" title="fpage">12121</span>–<span class="lpage" tagx="lpage" title="lpage">12126</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16880383">16880383</a></span></span></li> <li tag="ref"><a name="msz276-B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meijer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swaan</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voerknecht</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jusic</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Brink</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wijsman</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voordouw</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donker</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sleven</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorigo-Zetsma</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Case of seasonal reassortant A(H1N2) influenza virus infection, the Netherlands, March 2018</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">15</span>):pii=18-00160.</span></li> <li tag="ref"><a name="msz276-B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culhane</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Detmer</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Global migration of influenza A viruses in swine</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">6696.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25813399">25813399</a></span></span></li> <li tag="ref"><a name="msz276-B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noble</span><span class="given-names" tagx="given-names" title="given-names">WS.</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">What is a support vector machine?</span><span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1565</span>–<span class="lpage" tagx="lpage" title="lpage">1567</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17160063">17160063</a></span></span></li> <li tag="ref"><a name="msz276-B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ottis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandeputte</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaplan</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachmann</span><span class="given-names" tagx="given-names" title="given-names">PA.</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">Evidence for the natural transmission of influenza A virus from wild ducks to swine and its potential importance for man</span>. <span class="source" tagx="source" title="source">Bull World Health Organ</span>. <span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">75</span>–<span class="lpage" tagx="lpage" title="lpage">78</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6973418">6973418</a></span></span></li> <li tag="ref"><a name="msz276-B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pulit-Penaloza</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thor</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zanders</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creager</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridenour</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2018a</span><span class="mixed-article-title" title="mixed-article-title">Antigenically diverse swine origin H1N1 variant influenza viruses exhibit differential ferret pathogenesis and transmission phenotypes</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">pii: e00095-18</span>.</span></li> <li tag="ref"><a name="msz276-B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pulit-Penaloza</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pappas</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brock</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumpey</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maines</span><span class="given-names" tagx="given-names" title="given-names">TR.</span></span></span><span class="year" tagx="year" title="year">2018b</span><span class="mixed-article-title" title="mixed-article-title">Comparative in vitro and in vivo analysis of H1N1 and H1N2 variant influenza viruses isolated from humans between 2011 and 2016</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">pii: e01444-18</span>.</span></li> <li tag="ref"><a name="msz276-B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Scoring amino acid mutations to predict avian-to-human transmission of avian influenza viruses</span>. <span class="source" tagx="source" title="source">Molecules</span><span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1584</span>.</span></li> <li tag="ref"><a name="msz276-B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubenberger</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanning</span><span class="given-names" tagx="given-names" title="given-names">TG.</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Evidence of an absence: the genetic origins of the 1918 pandemic influenza virus</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">909</span>–<span class="lpage" tagx="lpage" title="lpage">914</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15494747">15494747</a></span></span></li> <li tag="ref"><a name="msz276-B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Ecological dynamics of influenza A viruses: cross-species transmission and global migration</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">36839.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27827462">27827462</a></span></span></li> <li tag="ref"><a name="msz276-B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rudge</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coker</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Human to human transmission of H7N9</span>. <span class="source" tagx="source" title="source">BMJ</span><span class="volume" tagx="volume" title="volume">347</span>:<span class="fpage" tagx="fpage" title="fpage">f4730</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23920349">23920349</a></span></span></li> <li tag="ref"><a name="msz276-B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">JJ.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">A new integrated symmetrical table for genetic codes</span>. <span class="source" tagx="source" title="source">Biosystems</span><span class="volume" tagx="volume" title="volume">151</span>:<span class="fpage" tagx="fpage" title="fpage">21</span>–<span class="lpage" tagx="lpage" title="lpage">26</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27887904">27887904</a></span></span></li> <li tag="ref"><a name="msz276-B301" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shu Y and McCauley J. 2017. GISAID: Global initiative on sharing all influenza data - from vision to reality. <i>Euro Surveill</i>. 22(13): pii: 30494.</span></li> <li tag="ref"><a name="msz276-B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slawnych</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nieminen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kahonen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kavanagh</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehtimaki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramadan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viik</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aggarwal</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehtinen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellis</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikus</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Post-exercise assessment of cardiac repolarization alternans in patients with coronary artery disease using the modified moving average method</span>. <span class="source" tagx="source" title="source">J Am Coll Cardiol</span>. <span class="volume" tagx="volume" title="volume">53</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">1130</span>–<span class="lpage" tagx="lpage" title="lpage">1137</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19324258">19324258</a></span></span></li> <li tag="ref"><a name="msz276-B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worobey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghwani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">459</span>(<span class="issue" tagx="issue" title="issue">7250</span>):<span class="fpage" tagx="fpage" title="fpage">1122</span>–<span class="lpage" tagx="lpage" title="lpage">1125</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19516283">19516283</a></span></span></li> <li tag="ref"><a name="msz276-B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subbarao</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Avian influenza viruses infecting humans</span>. <span class="source" tagx="source" title="source">Cell Mol Life Sci</span>. <span class="volume" tagx="volume" title="volume">57</span>:<span class="fpage" tagx="fpage" title="fpage">1770</span>–<span class="lpage" tagx="lpage" title="lpage">1784</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130181">11130181</a></span></span></li> <li tag="ref"><a name="msz276-B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swerdlow</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finelli</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridges</span><span class="given-names" tagx="given-names" title="given-names">CB.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">2009 H1N1 influenza pandemic: field and epidemiologic investigations in the United States at the start of the first pandemic of the 21st century</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span>. <span class="volume" tagx="volume" title="volume">52(1 Suppl)</span>:<span class="fpage" tagx="fpage" title="fpage">S1</span>–<span class="lpage" tagx="lpage" title="lpage">S3</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21342879">21342879</a></span></span></li> <li tag="ref"><a name="msz276-B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takata</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goncalves-Carneiro</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zang</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soll</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">York</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanco-Melo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bieniasz</span><span class="given-names" tagx="given-names" title="given-names">PD.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">CG dinucleotide suppression enables antiviral defence targeting non-self RNA</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">550</span>(<span class="issue" tagx="issue" title="issue">7674</span>):<span class="fpage" tagx="fpage" title="fpage">124</span>–<span class="lpage" tagx="lpage" title="lpage">127</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28953888">28953888</a></span></span></li> <li tag="ref"><a name="msz276-B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">512</span>–<span class="lpage" tagx="lpage" title="lpage">526</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8336541">8336541</a></span></span></li> <li tag="ref"><a name="msz276-B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubenberger</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kash</span><span class="given-names" tagx="given-names" title="given-names">JC.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Influenza virus evolution, host adaptation, and pandemic formation</span>. <span class="source" tagx="source" title="source">Cell Host Microbe</span>. <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">440</span>–<span class="lpage" tagx="lpage" title="lpage">451</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20542248">20542248</a></span></span></li> <li tag="ref"><a name="msz276-B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luong</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vo</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">TQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngo</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dao</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Avian influenza A (H5N1) in 10 patients in Vietnam</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="volume" tagx="volume" title="volume">350</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1179</span>–<span class="lpage" tagx="lpage" title="lpage">1188</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14985470">14985470</a></span></span></li> <li tag="ref"><a name="msz276-B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tulloch</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">RNA virus attenuation by codon pair deoptimisation is an artefact of increases in CpG/UpA dinucleotide frequencies</span>. <span class="source" tagx="source" title="source">eLife</span>. <span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">e4531.</span></span></li> <li tag="ref"><a name="msz276-B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Long-term evolution and transmission dynamics of swine influenza A virus</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="volume" tagx="volume" title="volume">473</span>(<span class="issue" tagx="issue" title="issue">7348</span>):<span class="fpage" tagx="fpage" title="fpage">519</span>–<span class="lpage" tagx="lpage" title="lpage">522</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21614079">21614079</a></span></span></li> <li tag="ref"><a name="msz276-B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yakhno</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinshaw</span><span class="given-names" tagx="given-names" title="given-names">VS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bean</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murti</span><span class="given-names" tagx="given-names" title="given-names">KG.</span></span></span><span class="year" tagx="year" title="year">1978</span><span class="mixed-article-title" title="mixed-article-title">Intestinal influenza: replication and characterization of influenza viruses in ducks</span>. <span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">84</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">268</span>–<span class="lpage" tagx="lpage" title="lpage">278</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23604">23604</a></span></span></li> <li tag="ref"><a name="msz276-B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">VC.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">A review of influenza viruses in seals and the implications for public health</span>. <span class="source" tagx="source" title="source">US Army Med Dep J</span>. <span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23277445">23277445</a></span></span></li> <li tag="ref"><a name="msz276-B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witteveldt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Gans</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Enhancement of the replication of hepatitis C virus replicons of genotypes 1 to 4 by manipulation of CpG and UpA dinucleotide frequencies and use of cell lines expressing SECL14L2 for antiviral resistance testing</span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span>. <span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">2981</span>–<span class="lpage" tagx="lpage" title="lpage">2992</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26953209">26953209</a></span></span></li> <li tag="ref"><a name="msz276-B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halloran</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugimoto</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longini</span><span class="given-names" tagx="given-names" title="given-names">IJ.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Detecting human-to-human transmission of avian influenza A (H5N1)</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1348</span>–<span class="lpage" tagx="lpage" title="lpage">1353</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18252106">18252106</a></span></span></li> <li tag="ref"><a name="msz276-B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webby</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Evolution and ecology of influenza A viruses</span>. <span class="source" tagx="source" title="source">Curr Top Microbiol Immunol</span>. <span class="volume" tagx="volume" title="volume">385</span>:<span class="fpage" tagx="fpage" title="fpage">359</span>–<span class="lpage" tagx="lpage" title="lpage">375</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24990620">24990620</a></span></span></li> <li tag="ref"><a name="msz276-B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aevermann</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauphin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Influenza Research Database: an integrated bioinformatics resource for influenza virus research</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">D1</span>):<span class="fpage" tagx="fpage" title="fpage">D466</span>–<span class="lpage" tagx="lpage" title="lpage">D474</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27679478">27679478</a></span></span></li> </ul> </div> </div>  </body></html>