<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/24/2020.03.20.000885.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.20.000885v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection" />
<meta name="DC.Identifier" content="10.1101/2020.03.20.000885" />
<meta name="DC.Date" content="2020-03-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2’s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2’s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.

One Sentence Summary Extensive Recombination and Strong Purifying Selection among coronaviruses from different hosts facilitate the emergence of SARS-CoV-2" />
<meta name="DC.Contributor" content="Xiaojun Li" />
<meta name="DC.Contributor" content="Elena E. Giorgi" />
<meta name="DC.Contributor" content="Manukumar Honnayakanahalli Marichann" />
<meta name="DC.Contributor" content="Brian Foley" />
<meta name="DC.Contributor" content="Chuan Xiao" />
<meta name="DC.Contributor" content="Xiang-Peng Kong" />
<meta name="DC.Contributor" content="Yue Chen" />
<meta name="DC.Contributor" content="Bette Korber" />
<meta name="DC.Contributor" content="Feng Gao" />
<meta name="article:published_time" content="2020-03-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-29s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-29s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.20.000885v2" />
<meta name="citation_id" content="2020.03.20.000885v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.20.000885" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.20.000885" />
<meta name="citation_author" content="Xiaojun Li" />
<meta name="citation_author_institution" content="Department of Medicine, Duke University Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5780-0880" />
<meta name="citation_author" content="Elena E. Giorgi" />
<meta name="citation_author_institution" content="Theoretical Biology and Biophysics, Los Alamos National Laboratory" />
<meta name="citation_author" content="Manukumar Honnayakanahalli Marichann" />
<meta name="citation_author_institution" content="Department of Medicine, Duke University Medical Center" />
<meta name="citation_author" content="Brian Foley" />
<meta name="citation_author_institution" content="Theoretical Biology and Biophysics, Los Alamos National Laboratory" />
<meta name="citation_author" content="Chuan Xiao" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The University of Texas at El Paso" />
<meta name="citation_author" content="Xiang-Peng Kong" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, Grossman School of Medicine, New York University" />
<meta name="citation_author" content="Yue Chen" />
<meta name="citation_author_institution" content="Department of Medicine, Duke University Medical Center" />
<meta name="citation_author" content="Bette Korber" />
<meta name="citation_author_institution" content="Theoretical Biology and Biophysics, Los Alamos National Laboratory" />
<meta name="citation_author" content="Feng Gao" />
<meta name="citation_author_institution" content="Department of Medicine, Duke University Medical Center" />
<meta name="citation_author_email" content="fgao@duke.edu" />
<meta name="citation_reference" content="N. Zhu et al., A Novel Coronavirus from Patients with Pneumonia in China, 2019. The New England journal of medicine 382, 727–733 (2020)." />
<meta name="citation_reference" content="W. H. Organization, Novel Coronavirus (COVID-19) Situation. https://experience.arcgis.com/experience/685d0ace521648f8a5beeeee1b9125cd, (2020)." />
<meta name="citation_reference" content="F. Wu et al., A new coronavirus associated with human respiratory disease in China. Nature 579, 265–269 (2020)." />
<meta name="citation_reference" content="A. E. Gorbalenya et al., The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nature Microbiology, (2020)." />
<meta name="citation_reference" content="J. Cui, F. Li, Z. L. Shi, Origin and evolution of pathogenic coronaviruses. Nat Rev Microbiol 17, 181–192 (2019)." />
<meta name="citation_reference" content="X.-D. Lin et al., Extensive diversity of coronaviruses in bats from China. Virology 507, 1–10 (2017)." />
<meta name="citation_reference" content="A. Banerjee, K. Kulcsar, V. Misra, M. Frieman, K. Mossman, Bats and Coronaviruses. Viruses 11, 41 (2019)." />
<meta name="citation_reference" content="P. Zhou et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273 (2020)." />
<meta name="citation_reference" content="Roujian Lu et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet, (2020)." />
<meta name="citation_reference" content="T. T.-Y. Lam et al., Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China. bioRxiv, 2020.2002.2013.945485 (2020)." />
<meta name="citation_reference" content="P. Liu, W. Chen, J.-P. Chen, Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica). Viruses 11, 979 (2019)." />
<meta name="citation_reference" content="Y. Guan et al., Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China. Science (New York, N.Y.) 302, 276–278 (2003)." />
<meta name="citation_reference" content="E. I. Azhar et al., Evidence for camel-to-human transmission of MERS coronavirus. The New England journal of medicine 370, 2499–2505 (2014)." />
<meta name="citation_reference" content="K. S. Lole et al., Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination. Journal of virology 73, 152–160 (1999)." />
<meta name="citation_reference" content="M. C. Wong, S. J. Javornik Cregeen, N. J. Ajami, J. F. Petrosino, Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019. bioRxiv, 2020.2002.2007.939207 (2020)." />
<meta name="citation_reference" content="A. C. Siepel, A. L. Halpern, C. Macken, B. T. Korber, A computer program designed to screen rapidly for HIV type 1 intersubtype recombinant sequences. AIDS research and human retroviruses 11, 1413–1416 (1995)." />
<meta name="citation_reference" content="R. L. Graham, R. S. Baric, Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission. Journal of virology 84, 3134–3146 (2010)." />
<meta name="citation_reference" content="F. Li, Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol 3, 237–261 (2016)." />
<meta name="citation_reference" content="B. Hu et al., Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS pathogens 13, e1006698 (2017)." />
<meta name="citation_reference" content="W. Li et al., Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)." />
<meta name="citation_reference" content="J. Lan et al., Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. bioRxiv, 2020.2002.2019.956235 (2020)." />
<meta name="citation_reference" content="D. Wrapp et al., Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (New York, N.Y.), eabb2507 (2020)." />
<meta name="citation_reference" content="A. C. Walls et al., Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell, S0092-8674(0020)30262-30262 (2020)." />
<meta name="citation_reference" content="C. Xiao et al., HIV-1 did not contribute to the 2019-nCoV genome. Emerging Microbes &amp; Infections 9, 378–381 (2020)." />
<meta name="citation_reference" content="B. Coutard et al., The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade. Antiviral Res 176, 104742 (2020)." />
<meta name="citation_reference" content="H. Zhou et al., A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19. bioRxiv, 2020.2003.2002.974139 (2020)." />
<meta name="citation_reference" content="A. C. Walls et al., Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein. bioRxiv, 2020.2002.2019.956581 (2020)." />
<meta name="citation_reference" content="C. C. Hon et al., Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus. Journal of virology 82, 1819–1826 (2008)." />
<meta name="citation_reference" content="S. K. Lau et al., Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination. Journal of virology 89, 10532–10547 (2015)." />
<meta name="citation_reference" content="J. F. Chan et al., Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerg Microbes Infect 9, 221–236 (2020)." />
<meta name="citation_reference" content="V. D. Menachery et al., A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence. Nature medicine 21, 1508–1513 (2015)." />
<meta name="citation_reference" content="V. D. Menachery et al., SARS-like WIV1-CoV poised for human emergence. Proceedings of the National Academy of Sciences 113, 3048–3053 (2016)." />
<meta name="citation_reference" content="X.-Y. Ge et al., Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538 (2013)." />
<meta name="citation_reference" content="N. Wang et al., Serological Evidence of Bat SARS-Related Coronavirus Infection in Humans, China. Virol Sin 33, 104–107 (2018)." />
<meta name="citation_reference" content="R. Abagyan, M. Totrov, D. Kuznetsov, ICM—A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation. Journal of computational chemistry 15, 488–506 (1994)." />
<meta name="citation_reference" content="Y. Shu, J. McCauley, GISAID: Global initiative on sharing all influenza data - from vision to reality. Euro Surveill 22, (2017)." />
<meta name="citation_reference" content="P. Liu et al., Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ? bioRxiv, 2020.2002.2018.954628 (2020)." />
<meta name="citation_reference" content="K. Xiao et al., Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins. bioRxiv, 2020.2002.2017.951335 (2020)." />
<meta name="citation_reference" content="M. A. Larkin et al., Clustal W and Clustal X version 2.0. Bioinformatics 23, 2947–2948 (2007)." />
<meta name="citation_reference" content="S. Guindon, O. Gascuel, A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic biology 52, 696–704 (2003)." />
<meta name="citation_reference" content="B. Foley et al., HIV sequence compendium 2018. (Theoretical Biology and Biophysics Group, Los Alamos National Laboratory, NM, LA-UR 18-25673, Los Alamos, New Mexico, 2018)." />
<meta name="citation_reference" content="citation_title=Computational Analysis of HIV Molecular Sequences;citation_pages=55-72;citation_year=2000" />
<meta name="citation_reference" content="B. Murrell et al., Detecting individual sites subject to episodic diversifying selection. PLoS Genet 8, e1002764–e1002764 (2012)." />
<meta name="citation_reference" content="E. F. Pettersen et al., UCSF Chimera--a visualization system for exploratory research and analysis. Journal of computational chemistry 25, 1605–1612 (2004)." />
<meta name="citation_reference" content="J. Yang, Y. Zhang, Protein Structure and Function Prediction Using I-TASSER. Current protocols in bioinformatics 52, 5.8.1-5.8.15 (2015)." />
<meta name="citation_reference" content="J. Shang et al., Structural basis for receptor recognition by the novel coronavirus from Wuhan. DOI: 10.21203/rs.2.24749/v1, (2020)." />
<meta name="twitter:title" content="Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2’s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2’s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.

One Sentence Summary Extensive Recombination and Strong Purifying Selection among coronaviruses from different hosts facilitate the emergence of SARS-CoV-2" />
<meta name="og-title" property="og:title" content="Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2’s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2’s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.

One Sentence Summary Extensive Recombination and Strong Purifying Selection among coronaviruses from different hosts facilitate the emergence of SARS-CoV-2" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.20.000885v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.20.000885v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.20.000885v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1210263" />
    <title>Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7Qa5IE0G_Vw91OwdU-2rttsETlU1mYeoHnCB6RbpdpY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1210263","apath":"\/biorxiv\/early\/2020\/03\/24\/2020.03.20.000885.atom","pisa":"biorxiv;2020.03.20.000885v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.20.000885v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.20.000885v2","title":"Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection","identifier":"node\/1210263"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UIGFf1YWq\\\\\\\/uzBNH4A4cnW0bBmdEdSS9qaDZQbSAN6GodAkFathuLl6n5BIdQ5YIi2g9\\\\\\\/avxZ\\\\\\\/7sh8RD3QfndDSJS\\\\\\\/IYUvxiLNUq12cXaMlamlUQ+aB2PVKjMlNlKS4EF0JYUGQ6bI1srA2f0KFPNJb8LCH0jMP81E5Ji00XsvJKsr6h7ZTmWgoYKKN+We4rJ3ju0aeSay7\\\\\\\/ci1l4InhxmTy91ay7jlbwcakMLoRJYGWLLnZ+tOK7B0ucvm0w0+O31nyFhVweCeH8bmu8PPQP1QozLSUjLQk4Y4DW1c7A8N5SfmMSCsA9TRQ0kF9yvMN5Ngm8pXzN4oIqxt9oEjyipISYmpOxoUf2+LBtkBMSLEWseOZeXLZ97mE466X1XE9eA46vdyDjHToJQ2G8l2VywSDutWBQWdf24kJzjAmQPbN3MHWNPMymQvpd2gjK\\\\\\\/jbW9xxGtN45PuahxwLJ5Wuj7lKiYfWs\\\\\\\/VayknDNUBZani3HUV3ZAm0NR0AOYV4cupFnjbeUkDR+nrIeaVm8lszoHrYi6uvV5KnQ507XhVrSm6FwQ7ZJL5RIotyr0pnWW+NXelxA+WVAdghzj+WZDnDwUca1WvoUyzG7GmlL0T1ZkTNfBgvNWxSYXiMMStO0\\\\\\\/2sbUmG1ndtt6sikEzCdTLZXt5Z1HMSqeCUcMTfvDzEA34ac46KnAqFpcEz5rm51AOJsLRWUdpLMsngobIThlj7uIActyLYqEqRBbue9ZOZkwbKE48kuWtnHnDRSgmeHjXUXdVymbFOcSDtv5+Gv+T4VXAYGU3iylvIdpM0xOHih5BtMQI1AsXjkvbP03upuztx80uz88NufS8IV84r6ZGwe5YE0Q8fJMf4zaJE\\\\\\\/WwnycNnxvPR+y6UaPu\\\\\\\/eMsqybfNtvLuE19P2AFdbHngROqhmOP4yW3NmLwXqj6nD44OERpj2VcVHL4o2W8jafvcx5eDir8Kkt86a1hw7xzv2Nq1+rmqSlOoSVlbRZzzidqcNnA9U3bwxOexjpM3lSVLDZ+aIrh+6FIE6iQtrfpuVMP6ZyCk3sa0VwqV0hmtONbHT4I+16JlifNCo\\\\\\\/R4z8bI3TZekG6bsOiQvrW3csB2cAbBmubxbXHugYJnF+6LW1cGhry3tlcOlaUdjOdKaqdEUlx+0SKAc7hQbIv+\\\\\\\/XaV5yM2GdadKflZr7ifydCN+dv3Db1I6Ti+6UAPFPLyZeLxOtObTnsv6TDRDxsPzyzMCfVVUnNKC94rzoKapZV6L2DcahYfA8lJVyWbY5ZSTcnm3Gy4\\\\\\\/Cv6MRQ4dlYiDHfIlqfwyEDnhDVEubalyhUruohK+8ligykcdiKn6OnqW0Qld1evj\\\u0022,\\\u0022iv\\\u0022:\\\u0022S\\\\\\\/bRy4I30RpUQgSMWm\\\\\\\/1Hw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228a3bd2506f28145133ef1718d1e866dc8103298044655982f31a29d07b4578b9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.20.000885v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1210263 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.20.000885v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZyWb-ZxJZf247veLGICc1VaL4QhcQVRsa9Lk01Rx9jY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210263" id="node1210263" data-pisa="biorxiv;2020.03.20.000885v2" data-pisa-master="biorxiv;2020.03.20.000885" data-apath="/biorxiv/early/2020/03/24/2020.03.20.000885.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5780-0880" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elena E.</span> <span class="nlm-surname">Giorgi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Manukumar Honnayakanahalli</span> <span class="nlm-surname">Marichann</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Foley</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xiang-Peng</span> <span class="nlm-surname">Kong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bette</span> <span class="nlm-surname">Korber</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Gao</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.20.000885 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1210263" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaojun Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Medicine, Duke University Medical Center</span>, Durham, NC 27710, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaojun%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaojun%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5780-0880" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaojun Li</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elena E. Giorgi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Theoretical Biology and Biophysics, Los Alamos National Laboratory</span>, Los Alamos, NM 87544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BE.%2BGiorgi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giorgi%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BE.%2BGiorgi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Manukumar Honnayakanahalli Marichann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Medicine, Duke University Medical Center</span>, Durham, NC 27710, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manukumar%2BHonnayakanahalli%2BMarichann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marichann%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManukumar%2BHonnayakanahalli%2BMarichann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Brian Foley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Theoretical Biology and Biophysics, Los Alamos National Laboratory</span>, Los Alamos, NM 87544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BFoley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foley%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BFoley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chuan Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Chemistry and Biochemistry, The University of Texas at El Paso</span>, El Paso, TX 79968, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuan%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuan%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiang-Peng Kong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Biochemistry and Molecular Pharmacology, Grossman School of Medicine, New York University</span>, New York, NY 10016</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiang-Peng%2BKong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kong%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiang-Peng%2BKong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yue Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Medicine, Duke University Medical Center</span>, Durham, NC 27710, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bette Korber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Theoretical Biology and Biophysics, Los Alamos National Laboratory</span>, Los Alamos, NM 87544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bette%2BKorber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Korber%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABette%2BKorber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Feng Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Medicine, Duke University Medical Center</span>, Durham, NC 27710, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Feng%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFeng%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fgao@duke.edu" class="" data-icon-position="" data-hide-link-title="0">fgao@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.20.000885v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000885v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000885v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000885v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000885v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.20.000885v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1210263" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1210263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2’s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2’s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>One Sentence Summary</strong> Extensive Recombination and Strong Purifying Selection among coronaviruses from different hosts facilitate the emergence of SARS-CoV-2</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-6">The severe respiratory disease COVID-19 was first noticed in late December 2019 (<em><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></em>). It rapidly became epidemic in China, devastating public health and finance. By mid-March, COVID-19 had spread to ∼150 countries and infected over 150,000 people (<em><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></em>). On March 11, 2020, the World Health Organization (WHO) officially declared it a pandemic.</p><p id="p-7">A complete genome sequence of the etiological agent of COVID-19 (<em><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></em>), severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (<em><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></em>), identified it as a new member of the genus <em>Betacoronavirus</em>, which include a diverse reservoir of coronaviruses (CoVs) isolated from bats (<em><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></em>). While genetically distinct from the betacoronaviruses that cause SARS and MERS in humans (<em><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></em>), SARS-CoV-2 shares the highest level of genetic similarity (96.3%) with CoV RaTG13, sampled from a bat in Yunnan in 2013 (<em><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></em>). Recently, CoV sequences closely related to SARS-CoV-2 were obtained from confiscated Malaya pangolins in two separate studies (<em><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></em>). Pangolin SARS-like CoVs (Pan_SL-CoV) form two distinct clades corresponding to their collection location. Pan_SL-CoV_GD from Guangdong (GD) province in China and are genetically more similar to SARS-CoV-2 (91.2%) than Pan_SL-CoV_GX from Guangxi (GX) province (85.4%).</p><p id="p-8">Understanding the origin of SARS-CoV-2 may help resolve strategies to deter future cross-species transmissions and to establish appropriate animal models. Viral sequences nearly identical to SARS and MERS viruses were found in civets and domestic camels, respectively (<em><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></em>), demonstrating that they originated from zoonotic transmissions with intermediate host species between the bat reservoirs and humans—a common pattern leading to CoV zoonosis (<em><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></em>). Viruses nearly identical to SARS-CoV-2 have not yet been found. In this paper we demonstrate, through localized genomic analysis, a complex pattern of evolutionary recombination between CoVs from distinct host species and cross-species infections that likely originated SARS-CoV-2.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Acquisition of receptor binding motif through recombination</h3><p id="p-9">Phylogenetic analysis of 43 complete genome sequences from three clades (SARS-CoVs and bat_SL-CoVs; SARS-CoV-2, bat_SL-CoVs and pan_SL-CoVs; and two divergent bat_SL-CoVs) within the Sarbecovirus group (<em><a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a></em>) confirms that RaTG13 is overall the closest sequence to SARS-CoV-2 (fig. S1). It is followed by Pan_SL-CoV_GD viruses next, and then Pan_SL-CoV_GX. Among the bat-CoV sequences in clade 2 (fig. S1), ZXC21 and ZC45, sampled from bats in 2005 in Zhoushan, Zhejiang, China, are the most divergent, with the exception of the beginning of the ORF1a gene (region 1, <a id="xref-fig-1-1" class="xref-fig" href="#F1">fig. 1A</a>). All other Bat_SL-CoV and SARS-CoV sequences form a separate clade 3, while clade 1 comprises BtKY72 and BM48-31, the two most divergent Bat_SL-CoV sequences, in the Sarbecovirus group (fig. S1). Recombination in the first SARS-CoV-2 sequence (Wuhan-Hu-1) with other divergent CoVs has been previously observed (<em><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></em>). Here, to better understand the role of recombination in the origin of SARS-CoV-2 among these genetically similar CoVs, we compare Wuhan-Hu-1 to six representative Bat_SL-CoVs, one SARS-CoV, and the two Pan_SL-CoV_GD sequences using SimPlot analysis (<em><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></em>). RaTG13 has the highest similarity across the genome (<em><a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a></em>), with two notable exceptions where a switch occurs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">fig. 1A</a>). In phylogenetic reconstructions, SARS-CoV-2 clusters closer to ZXC21 and ZC45 than RaTG13 at the beginning of the ORF1a gene (region 1, <a id="xref-fig-1-3" class="xref-fig" href="#F1">fig. 1B</a>), and, as reported (<em><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></em>), to a Pan_SL-CoV_GD in region 2 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">fig.s 1C</a> and S2), which spans the receptor angiotensin-converting enzyme 2 (ACE2) binding site in the spike (S) glycoprotein gene. Comparing Wuhan-Hu-1 to Pan_SL-CoV_GD and RaTG13, as representative of distinct host-species branches in the evolutionary history of SARS-CoV-2, using the recombination detection tool RIP (<em><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></em>), we find significant recombination breakpoints before and after the ACE2 binding site (fig. S2A), suggesting that SARS-CoV-2 carries a history of cross-species recombination between the bat and the pangolin CoVs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV-2 recombination with Pan_SL-CoV and Bat_SL-CoV. (A) SimPlot genetic similarity plot between SARS-CoV-2 Wuhan-Hu-1 and representative CoV sequences, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. (B-E) Phylogenetic trees of regions of disproportional similarities, showing high similarities with ZXC21 (B) or GD/P1La (C), or high divergences with both GD/P1La and GX/P4L (D) or GD/P1La (E). All positions are relative to Wuhan-Hu-1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508063396" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;SARS-CoV-2 recombination with Pan_SL-CoV and Bat_SL-CoV.&lt;/span&gt; (A) SimPlot genetic similarity plot between SARS-CoV-2 Wuhan-Hu-1 and representative CoV sequences, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. (B-E) Phylogenetic trees of regions of disproportional similarities, showing high similarities with ZXC21 (B) or GD/P1La (C), or high divergences with both GD/P1La and GX/P4L (D) or GD/P1La (E). All positions are relative to Wuhan-Hu-1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F1.medium.gif" width="253" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F1.medium.gif" width="253" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">SARS-CoV-2 recombination with Pan_SL-CoV and Bat_SL-CoV.</span><p id="p-10" class="first-child">(A) SimPlot genetic similarity plot between SARS-CoV-2 Wuhan-Hu-1 and representative CoV sequences, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. (B-E) Phylogenetic trees of regions of disproportional similarities, showing high similarities with ZXC21 (B) or GD/P1La (C), or high divergences with both GD/P1La and GX/P4L (D) or GD/P1La (E). All positions are relative to Wuhan-Hu-1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Pan_SL-CoV sequences are generally more similar to SARS-CoV-2 than CoV sequences, other than RaTG13 and ZXC21, but are more divergent from SARS-CoV-2 at two regions in particular: the beginning of the ORF1b gene and the highly divergent N terminus of the S gene (regions 3 and 4, respectively, <a id="xref-fig-1-5" class="xref-fig" href="#F1">fig. 1A</a>). Within-region phylogenetic reconstructions show that Pan_SL-CoV sequences become as divergent as BtKY72 and BM48-31 in region 3 (<a id="xref-fig-1-6" class="xref-fig" href="#F1">fig. 1D</a>), while less divergent in region 4, where Pan_SL-CoV_GD clusters with ZXC21 and ZC45 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">fig. 1E</a>). Together, these observations suggest ancestral cross-species recombination between pangolin and bat CoVs in the evolution of SARS-CoV-2 at the ORF1a and S genes. Furthermore the discordant phylogenetic clustering at various regions of the genome among clade 2 CoVs also supports extensive recombination among these viruses isolated from bats and pangolins.</p><p id="p-12">The SARS-CoV-2 S glycoprotein mediates viral entry into host cells and therefore represents a prime target for drug and vaccine development (<em><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></em>). While SARS-CoV-2 sequences share the greatest overall genetic similarity with RaTG13, this is no longer the case in parts of the S gene. Specifically, amino acid sequences of the receptor binding motif (RBM) in the C terminal of the S1 subunit are nearly identical to those in two Pan_SL-CoV_GD viruses, with only one amino acid difference (Q498H)—although the RBM region has not been fully sequenced in one of Guangdong pangolin virus (Pan_SL-CoV_GD/P2S) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">fig. 2A</a>). Pangolin CoVs from Guangxi are much more divergent. Phylogenetic analysis based on the amino acid sequences of this region shows three distinct clusters of SARS-CoV, SARS-CoV-2 and bat-CoV only viruses, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">fig. 2B</a>). Interestingly, while SARS-CoV and SARS-CoV-2 viruses use ACE2 for viral entry, all CoVs in the third cluster have a 5-aa deletion and a 13-14-aa deletion in RBM (<a id="xref-fig-2-3" class="xref-fig" href="#F2">fig. 2A</a>) and cannot infect human target cells (<em><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Impact of SARS-CoV-2 recombination on coreceptor binding. (A) AA sequences of the receptor binding motif (RBM) in the spike (S) gene among Sarbecovirus CoVs compared to Wuhan-Hu-1 (top). Dashes indicate identical aa&#x2019;s, dots indicate deletions. ACE2 critical contact sites highlighted in blue, two large deletions in green. (B) RBM aa phylogenetic tree, showing three distinct clusters, with large deletions Bat-SL-CoVs in divergent cluster. (C) SARS-CoV and SARS-CoV-2 receptor binding domains (RBD). Human ACE2 in green at the top and the S1 unit of the S-protein at the bottom; SARS-CoV S-protein (PDB 2AJF) in red, and SARS-CoV-2 S-protein in magenta with RBM in blue. All structure backbones shown as ribbons with key residues at the interface shown as stick models, labeled using the same color scheme. (D) Impact of different RBM aa between SARS-CoV-2 RaTG13 on ACE2 binding. (E) Impact of different aa at position 498 (Q in SARS-CoV-2, top, and H in RaTG13, bottom) on ACE2 binding. Same color-coding as in (C) with additional hydrogen bonding as light blue lines. (F) Impact of two deletions on ACE2 binding interface in some bat-SL-CoVs, positions indicated in yellow, and modeled structure with long deletion between residue 473 in light blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508063396" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Impact of SARS-CoV-2 recombination on coreceptor binding.&lt;/span&gt; (A) AA sequences of the receptor binding motif (RBM) in the spike (S) gene among Sarbecovirus CoVs compared to Wuhan-Hu-1 (top). Dashes indicate identical aa&#x2019;s, dots indicate deletions. ACE2 critical contact sites highlighted in blue, two large deletions in green. (B) RBM aa phylogenetic tree, showing three distinct clusters, with large deletions Bat-SL-CoVs in divergent cluster. (C) SARS-CoV and SARS-CoV-2 receptor binding domains (RBD). Human ACE2 in green at the top and the S1 unit of the S-protein at the bottom; SARS-CoV S-protein (PDB 2AJF) in red, and SARS-CoV-2 S-protein in magenta with RBM in blue. All structure backbones shown as ribbons with key residues at the interface shown as stick models, labeled using the same color scheme. (D) Impact of different RBM aa between SARS-CoV-2 RaTG13 on ACE2 binding. (E) Impact of different aa at position 498 (Q in SARS-CoV-2, top, and H in RaTG13, bottom) on ACE2 binding. Same color-coding as in (C) with additional hydrogen bonding as light blue lines. (F) Impact of two deletions on ACE2 binding interface in some bat-SL-CoVs, positions indicated in yellow, and modeled structure with long deletion between residue 473 in light blue.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F2.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F2.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">Impact of SARS-CoV-2 recombination on coreceptor binding.</span><p id="p-13" class="first-child">(A) AA sequences of the receptor binding motif (RBM) in the spike (S) gene among Sarbecovirus CoVs compared to Wuhan-Hu-1 (top). Dashes indicate identical aa’s, dots indicate deletions. ACE2 critical contact sites highlighted in blue, two large deletions in green. (B) RBM aa phylogenetic tree, showing three distinct clusters, with large deletions Bat-SL-CoVs in divergent cluster. (C) SARS-CoV and SARS-CoV-2 receptor binding domains (RBD). Human ACE2 in green at the top and the S1 unit of the S-protein at the bottom; SARS-CoV S-protein (PDB 2AJF) in red, and SARS-CoV-2 S-protein in magenta with RBM in blue. All structure backbones shown as ribbons with key residues at the interface shown as stick models, labeled using the same color scheme. (D) Impact of different RBM aa between SARS-CoV-2 RaTG13 on ACE2 binding. (E) Impact of different aa at position 498 (Q in SARS-CoV-2, top, and H in RaTG13, bottom) on ACE2 binding. Same color-coding as in (C) with additional hydrogen bonding as light blue lines. (F) Impact of two deletions on ACE2 binding interface in some bat-SL-CoVs, positions indicated in yellow, and modeled structure with long deletion between residue 473 in light blue.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Although both SARS-CoV and SARS-CoV-2 use the human ACE2 as their receptors (<em><a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></em>) they show a high level of genetic divergence (<a id="xref-fig-1-8" class="xref-fig" href="#F1">figs. 1</a> and S1). However, structures of the S1 unit of the S protein from both viruses are highly similar (<em><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>-<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></em>), with the exception of a loop, not proximal to the binding site, that bends differently (<a id="xref-fig-2-4" class="xref-fig" href="#F2">fig. 2C</a>). This suggests that viral entry through binding of ACE2 is structurally constrained to maintain the correct conformation. Among 17 distinct amino acids between SARS-CoV-2 and RaTG13 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">fig. 2A</a>), five contact sites are different, likely impacting RaTG13’s binding to ACE2 (<a id="xref-fig-2-6" class="xref-fig" href="#F2">fig. 2D</a> and Table S1). The single amino acid difference (Q or H at position 498) between SARS-CoV-2 and Pan_SL-CoV_GD is at the edge of the ACE2 contact interface; neither Q or H at this position form hydrogen bonds with ACE2 residues (<a id="xref-fig-2-7" class="xref-fig" href="#F2">fig. 2E</a>). Thus, a functional RBM nearly identical to the one in SARS-CoV-2 is naturally present in Pan_SL-CoV_GD viruses. The very distinctive RaTG13 RBM suggests that this virus is unlikely to infect human cells, and that the acquisition of a complete functional RBM by a RaTG13-like CoV through a recombination event with a Pan_SL-CoV_GD-like virus enabled it to use ACE2 for human infection.</p><p id="p-15">Three small insertions are identical in SARS-CoV-2 and RaTG13 but not found in other CoVs in the Sarbecovirus group (<em><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></em>). The RaTG13 sequence was sampled in 2013, years before SARS-CoV-2 was first identified. It is unlikely that both SARS-CoV-2 and RaTG13 independently acquired identical insertions at three different locations in the S gene. Thus, it is plausible that an RaTG13-like virus served as a progenitor to generate SARS-CoV-2 by gaining a complete human ACE2 binding RBM from Pan_SL-CoV_GD-like viruses through recombination. Genetic divergence at the nucleic acid level between Wuhan-Hu-1 and Pan_SL-CoV_GD viruses is significantly reduced from 13.9% (<a id="xref-fig-1-9" class="xref-fig" href="#F1">fig. 1E</a>) to 1.4% at the amino acid level (<a id="xref-fig-2-8" class="xref-fig" href="#F2">fig. 2B</a>) in the RBM region, indicating recombination between RaTG13-like CoVs and Pan_SL-CoV_GD-like CoVs. Furthermore, SARS-CoV-2 has a unique furin cleavage site insertion (PRRA) not found in any other CoVs in the Sarbecovirus group (<em><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></em>), although similar motifs are also found in MERS and more divergent bat CoVs (<em><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></em>) (Fig. S3). This PRRA motif makes the S1/S2 cleavage in SARS-CoV-2 much more efficiently than in SARS-CoV and may expand its tropism and/or enhance its transmissibility (<em><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></em>). A recent study of bat CoVs in Yunnan, China, identified a three-amino acid insertion (PAA) at the same site (<em><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></em>). Although it is not known if this PAA motif can function like the PRRA motif, the presence of a similar insertion at the same site indicates that such insertion may already be present in the wild bat CoVs. The more efficient cleavage of S1 and S2 units of the spike glycoprotein (<em><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></em>) and efficient binding to ACE2 by SARS-CoV-2 (<em><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></em>) may have allowed SARS-CoV-2 to jump to humans, leading to the rapid spread of SARS-CoV-2 in China and the rest of the world.</p></div><div id="sec-5" class="subsection"><h3>Strong purifying selection among SRAS-CoV-2 and closely related viruses</h3><p id="p-16">Recombination from Pan_SL-CoV_GD at the RBM and at the unique furin cleavage site insertion prompted us to examine the SARS-CoV-2 sequences within these regions. Amino acid sequences from SARS-CoV-2, RaTG13, and all Pan_SL-CoV viruses are identical or nearly identical before, between, and after the RBM and the furin cleavage site, while all other CoVs are very distinctive (<a id="xref-fig-3-1" class="xref-fig" href="#F3">fig. 3A</a> and S3). The average of all pairwise dN/dS ratios (ω) among SARS-CoV-2, RaTG13, and Pan_SL-CoV viruses at the 3’-end of the S gene (after the furin cleavage site) is 0.013, compared to ω =0.05 in the S region preceding the furin cleavage site, and to ω =0.04 after the site for all other CoVs. The much lower ω value at the 3’-end of the S gene among the SARS-CoV-2, RaTG13, and Pan_SL-CoV viruses indicates that this region is under strong purifying selection within these sequences (<a id="xref-fig-3-2" class="xref-fig" href="#F3">fig. 3A</a>). A plot of synonymous and nonsynonymous substitutions relative to Wuhan-Hu-1 highlights the regional differences across the region before and after the RBM and the furin cleavage site (<a id="xref-fig-3-3" class="xref-fig" href="#F3">fig. 3A</a>): the 3’ end of the region is highly conserved among the SARS-CoV-2, RaTG13, and Pan_SL-CoV viruses (Group A), while far more nonsynonymous mutations are observed in the rest of the CoV sequences (Group B). The shift in selective pressure in the 3’ -end of the gene among these related viruses versus other CoVs begins near codon 368 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">fig. 3B</a>), and such a shift was not evident among other compared CoVs (<a id="xref-fig-3-5" class="xref-fig" href="#F3">fig. 3B-D</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strong purifying selection after furin cleavage in S gene among SARS-CoV-2 and closely related viruses. (A) Phylogenetic tree (left) and Highlighter plot (right) of sequences around the RBM and furin cleavage site compared to SARS-CoV-2 Wuhan-Hu-1 (na positions 22541-24391). ACE2 receptor binding motif (RBM) and furin cleavage site highlighted in light-gray boxes. Mutations compared to Wuahn-Hu-1 are light blue for synonymous, red for non-synonymous. Dominance of synonymous mutations within group A compared to group B highlighted on the right. (B) Cumulative plots of each codon average behavior for all pairwise comparisons for indels and synonymous (light blue) and non-synonymous (red) mutations, by group. A vertical steps in group A at around codon 370 (na 1105) shows a shift in localized accumulations; non-synonymous mutations end after the furin cleavage site. Group B instead lacks this abrupt change in slope. &#x3C9;&#x2019;s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group and region half. (C) Sequence dS/dN ratios compared to Wuhan-Hu-1 within codons 1-370 (na 1-1104, green) and codons 371-620 (na 1105-1893, dark blue). (D) Proportion of tree branches under positive and negative selection (right and left respectively) per site for the two groups using the mixed effects model of evolution (MEME) from datamonkey (www.datamonkey.org)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508063396" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Strong purifying selection after furin cleavage in S gene among SARS-CoV-2 and closely related viruses.&lt;/span&gt; (A) Phylogenetic tree (left) and Highlighter plot (right) of sequences around the RBM and furin cleavage site compared to SARS-CoV-2 Wuhan-Hu-1 (na positions 22541-24391). ACE2 receptor binding motif (RBM) and furin cleavage site highlighted in light-gray boxes. Mutations compared to Wuahn-Hu-1 are light blue for synonymous, red for non-synonymous. Dominance of synonymous mutations within group A compared to group B highlighted on the right. (B) Cumulative plots of each codon average behavior for all pairwise comparisons for indels and synonymous (light blue) and non-synonymous (red) mutations, by group. A vertical steps in group A at around codon 370 (na 1105) shows a shift in localized accumulations; non-synonymous mutations end after the furin cleavage site. Group B instead lacks this abrupt change in slope. &#x3C9;&#x2019;s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group and region half. (C) Sequence dS/dN ratios compared to Wuhan-Hu-1 within codons 1-370 (na 1-1104, green) and codons 371-620 (na 1105-1893, dark blue). (D) Proportion of tree branches under positive and negative selection (right and left respectively) per site for the two groups using the mixed effects model of evolution (MEME) from datamonkey (www.datamonkey.org).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Strong purifying selection after furin cleavage in S gene among SARS-CoV-2 and closely related viruses.</span><p id="p-17" class="first-child">(A) Phylogenetic tree (left) and Highlighter plot (right) of sequences around the RBM and furin cleavage site compared to SARS-CoV-2 Wuhan-Hu-1 (na positions 22541-24391). ACE2 receptor binding motif (RBM) and furin cleavage site highlighted in light-gray boxes. Mutations compared to Wuahn-Hu-1 are light blue for synonymous, red for non-synonymous. Dominance of synonymous mutations within group A compared to group B highlighted on the right. (B) Cumulative plots of each codon average behavior for all pairwise comparisons for indels and synonymous (light blue) and non-synonymous (red) mutations, by group. A vertical steps in group A at around codon 370 (na 1105) shows a shift in localized accumulations; non-synonymous mutations end after the furin cleavage site. Group B instead lacks this abrupt change in slope. ω’s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group and region half. (C) Sequence dS/dN ratios compared to Wuhan-Hu-1 within codons 1-370 (na 1-1104, green) and codons 371-620 (na 1105-1893, dark blue). (D) Proportion of tree branches under positive and negative selection (right and left respectively) per site for the two groups using the mixed effects model of evolution (MEME) from datamonkey (<a href="http://www.datamonkey.org">www.datamonkey.org</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We observe similar patterns of purifying selection pressure in other parts of the genome, including the E and M genes, as well as the partial ORF1a and ORF1b genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">fig. 4</a>). Interestingly, the purifying selection pressure varies among different viruses depending on which genes are analyzed. The broadest group includes SARS-CoV-2, RaTG13, all Pan_SL-CoV and the two bat CoVs (ZXC21 and ZC45) for both E and M genes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">figs. 4</a> and S5). The second group includes SARS-CoV-2, RaTG13, and all Pan_SL-CoV only for the 3’ end of the S gene. The narrowest selection group only contains SARS-CoV-2, RaTG13, and pangolin CoVs from Guangdong for the partial regions of ORF1a and ORF1b (<a id="xref-fig-4-3" class="xref-fig" href="#F4">figs. 4</a> and S6). Consistently low ω values and strong purifying selection pressure on SARS-CoV-2, RaTG13 and Pan_SL-CoV_GD viruses suggest that these complete and partial genes are under similar functional/structural constraints among the different host species. In two extreme cases, amino acid sequences of the E gene and the 3’ end of ORF1a are identical among the compared CoV sequences, although genetic distances are quite large among these viruses at the nucleic acid level. Such evolutionary constraints across viral genomes, especially at functional domains in the S gene, which plays an important role in cross-species transmission (<em><a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></em>), coupled with frequent recombination, may facilitate cross-species transmissions between RaTG13-like bat and/or Pan_SL-CoV_GD-like viruses.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strong purifying selection on complete and partial gene regions among SARS-CoV-2, RaTG13 and Pan_SL-CoV viruses. Purifying selection pressure on complete and partial genes within different viruses (red boxes) as evident by shorter branches in aa phylogenetic trees compared to na trees. Distinct purifying selection patterns are observed among different viruses: (A) SARS-CoV-2, RaTG13, Pan_SL-CoV and bat CoV ZXC21 and ZC45; (B) SARS-CoV-2, RaTG13, all Pan_SL-CoV sequences; (C) SARS-CoV-2, RaTG13 and Pan_SL-CoV_GD. Cumulative plots of the average behavior of each codon for all pairwise comparisons for synonymous mutations, non-synonymous mutations and indels within each gene region. &#x3C9;&#x2019;s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508063396" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Strong purifying selection on complete and partial gene regions among SARS-CoV-2, RaTG13 and Pan_SL-CoV viruses.&lt;/span&gt; Purifying selection pressure on complete and partial genes within different viruses (red boxes) as evident by shorter branches in aa phylogenetic trees compared to na trees. Distinct purifying selection patterns are observed among different viruses: (A) SARS-CoV-2, RaTG13, Pan_SL-CoV and bat CoV ZXC21 and ZC45; (B) SARS-CoV-2, RaTG13, all Pan_SL-CoV sequences; (C) SARS-CoV-2, RaTG13 and Pan_SL-CoV_GD. Cumulative plots of the average behavior of each codon for all pairwise comparisons for synonymous mutations, non-synonymous mutations and indels within each gene region. &#x3C9;&#x2019;s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F4.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F4.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Strong purifying selection on complete and partial gene regions among SARS-CoV-2, RaTG13 and Pan_SL-CoV viruses.</span><p id="p-19" class="first-child">Purifying selection pressure on complete and partial genes within different viruses (red boxes) as evident by shorter branches in aa phylogenetic trees compared to na trees. Distinct purifying selection patterns are observed among different viruses: (A) SARS-CoV-2, RaTG13, Pan_SL-CoV and bat CoV ZXC21 and ZC45; (B) SARS-CoV-2, RaTG13, all Pan_SL-CoV sequences; (C) SARS-CoV-2, RaTG13 and Pan_SL-CoV_GD. Cumulative plots of the average behavior of each codon for all pairwise comparisons for synonymous mutations, non-synonymous mutations and indels within each gene region. ω’s denote average ratios of the rate of nonsynonymous substitutions per nonsynonymous site (dN/dS) for each group.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Frequent recombination between SARS-CoVs and bat_SL-CoVs</h3><p id="p-20">Previous studies using more limited sequence sets found that SARS-CoVs originated through multiple recombination events between different bat-CoVs (<em><a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></em>). Our phylogenetic analyses of individual genes show that SARS-CoV sequences tend to cluster with YN2018B, Rs9401, Rs7327, WIV16 and Rs4231 (group A) for some genes and Rf4092, YN2013, Anlong-112 and GX2013 (group B) for others (fig. S7). SimPlot analysis using both groups of bat_SL-CoVs and the closely related bat CoV YNLF-34C (<em><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></em>) shows that SARS-CoV GZ02 shifts in similarity across different bat SL-CoVs at various regions of the genome (<a id="xref-fig-5-1" class="xref-fig" href="#F5">fig. 5A</a>). In particular, phylogenetic reconstruction of the beginning of ORF1a (region 1) confirms that SARS-CoVs cluster with YNLF-34C (<em><a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></em>), and this region is distinctive comparing to all other CoVs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">fig. 5B</a>). YNLF-34C is more divergent from SARS-CoV than other bat-CoV viruses before and after this region, confirming the previously reported complex recombinant nature of YNLF-34C (<em><a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a></em>) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">fig. 5A</a>). At the end of the S gene (region 2), SARS-CoVs cluster with group A CoVs, forming a highly divergent clade (<a id="xref-fig-5-4" class="xref-fig" href="#F5">fig. 5C</a>). In region 3 (ORF8), SARS-CoVs and group B CoVs, together with YNLF-34C, form a very divergent and distinctive cluster (<a id="xref-fig-5-5" class="xref-fig" href="#F5">fig. 5D</a>). To further explore the recombinant nature of SARS-CoVs, we compared GZ02 to representative bat CoV sequences using the RIP recombination detection tool (<em><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></em>). We identified four significant breakpoints (at 99% confidence) between the two parental lineages (fig. S8A), further supported by phylogenetic analysis (fig. S8B-S8D). In addition, the two aforementioned groups of bat CoVs (shown in light brown and light blue in the trees) show similar cluster changes across the five recombinant regions, suggesting multiple events of historic recombination among bat SL-CoVs. These results demonstrate that SARS-CoV shares a recombinant history with at least three different groups of bat-CoVs and confirms the major role of recombination in the evolution of these viruses.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple recombination of SARS-CoVs with different bat_SL-CoVs. (A) SimPlot genetic similarity plot between SARS-CoV GZ02 and SARS_SL-CoVs, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. Group A reference CoVs (YN2018B, Rs9401, Rs7327, WIV16 and Rs4231) in blue, group B CoVs (Rf4092, YN2013, Anlong-112 and GX2013) in orange, YNLF-34C in green, and outlier control HKU3-12 in red. (B-D) Phylogenetic trees for high similarity regions between GZ02 and YNLF-34C (B), group (C), and group B (D). All positions are relative to Wuhan-Hu-1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508063396" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Multiple recombination of SARS-CoVs with different bat_SL-CoVs.&lt;/span&gt; (A) SimPlot genetic similarity plot between SARS-CoV GZ02 and SARS_SL-CoVs, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. Group A reference CoVs (YN2018B, Rs9401, Rs7327, WIV16 and Rs4231) in blue, group B CoVs (Rf4092, YN2013, Anlong-112 and GX2013) in orange, YNLF-34C in green, and outlier control HKU3-12 in red. (B-D) Phylogenetic trees for high similarity regions between GZ02 and YNLF-34C (B), group (C), and group B (D). All positions are relative to Wuhan-Hu-1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F5.medium.gif" width="376" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F5.medium.gif" width="376" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.20.000885/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <span class="caption-title">Multiple recombination of SARS-CoVs with different bat_SL-CoVs.</span><p id="p-21" class="first-child">(A) SimPlot genetic similarity plot between SARS-CoV GZ02 and SARS_SL-CoVs, using a 400-bp window at a 50-bp step and the Kimura 2-parameter model. Group A reference CoVs (YN2018B, Rs9401, Rs7327, WIV16 and Rs4231) in blue, group B CoVs (Rf4092, YN2013, Anlong-112 and GX2013) in orange, YNLF-34C in green, and outlier control HKU3-12 in red. (B-D) Phylogenetic trees for high similarity regions between GZ02 and YNLF-34C (B), group (C), and group B (D). All positions are relative to Wuhan-Hu-1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Of the bat SL-CoVs that contributed to the recombinant origin of SARS-CoV, only group A viruses bind to ACE2. Group B bat SL-CoVs do not infect human cells (<em><a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a></em>) and have two deletions in the RBM (<a id="xref-fig-1-10" class="xref-fig" href="#F1">figs. 1E</a> and <a id="xref-fig-2-9" class="xref-fig" href="#F2">2A</a>). The short deletion between residues 445 and 449, and in particular the loss of Y449, which forms three hydrogen bonds with ACE2, will significantly affect the overall structure of the RBM (<a id="xref-fig-2-10" class="xref-fig" href="#F2">figs. 2F</a> and <a id="xref-fig-2-11" class="xref-fig" href="#F2">2E</a>). The region encompassing the large deletion between residues 473 and 486 contains the loop structure that accounts for the major differences between the S protein of SARS-CoV and SARS-CoV-2 (<a id="xref-fig-2-12" class="xref-fig" href="#F2">fig. 2C</a>). This deletion causes the loss of contact site F486 and affects the conserved residue F498’s hydrophobic interaction with residue M82 on ACE2 (<a id="xref-fig-2-13" class="xref-fig" href="#F2">fig. 2F</a>). These two deletions will render RBM in those CoVs incapable to bind human ACE2. Therefore, recombination may play a role in enabling cross-species transmission in SARS-CoVs through the acquisition of an S gene type that can efficiently bind to the human ACE2 receptor.</p><p id="p-23">ORF8 is one of the highly variable genes in coronaviruses (<em><a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a></em>) and its function has not yet been elucidated (<em><a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></em>). Breakpoints within this region show that recombination occurred at the beginning and the end of ORF8 (fig. S9), where nucleic acid sequences are nearly identical among both SARS-CoVs and group B bat CoVs. Moreover, all compared viruses form three highly distinct clusters (<a id="xref-fig-5-6" class="xref-fig" href="#F5">fig. 5D</a>), suggesting that the ORF8 gene may be biologically constrained and evolves through modular recombination. The third recombination region at the beginning of ORF1a is where SARS-CoV-2 also recombined with other bat CoVs (region 1, <a id="xref-fig-1-11" class="xref-fig" href="#F1">fig. 1A</a>). This region is highly variable (<em><a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a></em>) and recombination within this part of the genome was also found in many other CoVs, suggesting that it may be a recombination hotspot and may factor into cross-species transmission.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-24">There are three important aspects to betacoronavirus evolution that should be carefully considered in phylogenetic reconstructions among more distant coronaviruses. First, there is extensive recombination among all of these viruses (<em><a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">29</a></em>) (<a id="xref-fig-1-12" class="xref-fig" href="#F1">figs. 1</a> and <a id="xref-fig-5-7" class="xref-fig" href="#F5">5</a>), making standard phylogenetic reconstructions based on full genomes problematic, as different regions of the genome have distinct ancestral relationships. Second, between more distant sequences, synonymous substitutions are often fully saturated, which can confound analyses of selective pressure and add noise to phylogenetic analysis. Finally, there are different selective pressures at work in different lineages, which is worth consideration interpreting trees.</p><p id="p-25">The currently sampled pangolin CoVs are too divergent from SARS-CoV-2 for them to be SARS-CoV-2 progenitors, but it is noteworthy that these sequences contain an RBM that can most likely bind to human ACE2. While RaTG13 is the most closely related CoV sequence to SARS-CoV-2, it has a distinctive RBM, which is not expected to bind to human ACE2. SARS-CoV-2 has a nearly identical RBM to the one found in the pangolin CoVs from Guangdong. Thus, it is plausible that RaTG13-like bat-CoV viruses may have obtained the RBM sequence binding to human ACE2 through recombination with Pan_SL-CoV_GD-like viruses. We hypothesize that this, and/or other ancestral recombination events between viruses infecting bats and pangolins, may have had a key role in the evolution of the strain that lead to the introduction of SARS-CoV-2 into humans.</p><p id="p-26">All three human CoVs (SARS, MERS and SARS-2) are the result of recombination among CoVs. Recombination in all three viruses involved the S gene, likely a precondition to zoonosis that enabled efficient binding to human receptors (<em><a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a></em>). Extensive recombination among bat coronaviruses and strong purifying selection pressure among viruses from humans, bats and pangolin may allow such closely related viruses ready jump between species and adapt to the new hosts. Many bat CoVs have been found able to bind to human ACE2 and replication in human cells (<em><a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>-<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></em>). Serological evidence has revealed that additional otherwise undetected spillovers have occurred in people in China living in proximity to wild bat populations (<em><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></em>). Continuous surveillance of coronaviruses in their natural hosts and in humans will be key to rapid control of new coronavirus outbreaks.</p><p id="p-27">So far efforts have failed to find the original pathway of SARS-CoV-2 into humans by identifying a coronavirus that is nearly identical to SARS-CoV-2, as those found for SARS and MERS in civets and domestic camels respectively (<em><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></em>). However, if the new SARS-CoV-2 strain did not cause widespread infections in its natural or intermediate hosts, such a strain may never be identified. The close proximity of animals of different species in a wet market setting may increase the potential for cross-species spillover infections, by enabling recombination between more distant coronaviruses and the emergence of recombinants with novel phenotypes. While the direct reservoir of SARS-CoV-2 is still being sought, one thing is clear: reducing or eliminating direct human contact with wild animals is critical to preventing new coronavirus zoonosis in the future.</p></div><div class="section" id="sec-8"><h2 class="">Materials and Methods</h2><div id="sec-9" class="subsection"><h3>Sequences analysis</h3><p id="p-28">All 43 CoV complete genome sequences were obtained from GenBank and GISAID (Global Initiative on Sharing All Influenza Data) (<em><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></em>), and were selected to be representative of the diversity. Pan_SL-CoV_GD/P1La sequence was generated by combining Pan_SL-CoV_GD/P1L (<em><a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a></em>) with some additional sequences from the NCBI BioProject database PRJNA5732983 (<em><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></em>) to have a maximal coverage of the complete genome sequence for analysis. A new CoV sequence from pangolin (EPI_ISL_410721) (<em><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></em>)was not inclued because that it became available after we had already completed the analyses in this study, and it was not as close to SARS-CoV-2 sequences and did not change the interpretation of our results. The whole genome sequences were first aligned using Clustal X2 (<em><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></em>). The alignments for all coding regions were manually optimized based on the amino acid sequence alignment using SeaView 5.0.1.</p></div><div id="sec-10" class="subsection"><h3>Recombination Analyses</h3><p id="p-29">SimPlot 3.5.15 (<em><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></em>) was used determine the percent identity of the query sequence to reference sequences. Potential recombinant regions among analyzed sequences were identified by sliding a 400bp-window at a 50bp-step across the alignment using the Kimura 2-parameter model. Phylogenetic trees were constructed by the maximum likelihood method using the GTR model (<em><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></em>), and their reliability was estimated from 1,000 bootstrap replicates. The positions of analyzed sequence regions were based on those in the reference SARS-CoV-2 Wuhan-Hu-1 (MN908947). Recombination regions and breakpoints were also analyzed using the LANL database (<em><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></em>) tool RIP (<em><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a></em>).</p></div><div id="sec-11" class="subsection"><h3>Selection Analyses</h3><p id="p-30">Cumulative plots of the average behavior of each codon for all pairwise comparisons in the input data, for insertions and deletions (indels), synonymous (syn), and nonsynonymous (nonsyn) mutations and values of the ratios of the rate of synonymous nucleotide substitutions per synonymous site and nonsynonymous substitutions per nonsynonymous site (dN/dS, or ω) were obtained using the LANL database tool SNAP (<em><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></em>). In order to avoid counting instances where synonymous mutations were saturated, averages of all pairwise dN/dS ratios were calculated excluding pairs that yielded dS values greater than 1. Sequences were analyzed for episodic selection pressure using the mixed effects model of evolution (MEME) (<em><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></em>) from the datamonkey server (<a href="http://www.datamonkey.org">www.datamonkey.org</a>).</p></div><div id="sec-12" class="subsection"><h3>Structure modeling of receptor binding</h3><p id="p-31">To investigate the single mutation Q498H in RBM between SARS-CoV-2 and Pan_SL-CoV_GD, Q498 in the crystal structure of S/ACE2 complex was mutated to H498 using Chimera (<em><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></em>). Local energy minimization (only H498 was allowed to move) was computed using Chimera’s built-in functions. To investigate the impact of the deletion between residue 473 to 486 to the binding interface between SARS-CoV-2 and human ACE2, a homology model with the deletion was generated using I-TASSER (<em><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></em>). The top five best models provided by the server have Confidence Score (C-score) of 0.86, -2.33, -4.01, -4.17, and -4.49. The C-score was used to estimate the quality of the models, which should be between -5.0 to 2; the higher the value, the higher the confidence in the model (<em><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></em>). Based on the C-score, model 1 was used in <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2F</a>. The interaction of the RBD of RaTG13 and ACE2 was modeled on PDB 6VW1, a hybrid structure of human SARS-CoV2 (<em><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></em>) using ICM software package (<em><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></em>), and the mutational differences of the Gibbs free energy (Table S1) were calculated with the built-in algorithm.</p></div></div><div class="section" id="sec-14"><h2 class="">Supplementary Material</h2><p id="p-33">Materials and Methods</p><p id="p-34">Figures S1-S9</p><p id="p-35">Table S1</p><p id="p-36">References (35-46)</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">We thank all those who have contributed SARS-CoV-2 genome sequences to the GISAID database (<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) and Virological.org (<a href="http://virological.org/">http://virological.org/</a>). We also thank Dr. Xinquan Wang from Tsinghua University for sharing the PDB 6M0J structure with us before its official release date. Funding: This work was supported by NIH Grants (AI122909, AI118571, GM129525 and AI145655). EEG, BK, and BF acknowledge support by the LDRD program at Los Alamos National Laboratory. Author contributions: Project conceptualization: F.G., B.K., E.E.G; Structure analysis: C.X., X-P.K; Sequence analysis: F.G., B.K., X.L., E.E.G., M.H.M., Y.C., B.F; Phylogenetic analysis: F.G., B.K., X.L., E.E.G., M.H.M., Y.C.; Recombination analysis: F.G., E.E.G., B.K., X.L., E.E.G., M.H.M., B.F.; Manuscript writing: F.G., B.K., E.E.G. Manuscript editing: F.G., B.K., E.E.G., X.L., C.X., X-P.K., F.G. and B.F. supervised the project. Competing interests: All authors declare no competing interests. All data is available in the main text or the supplementary materials.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-5">author contributions and methods updated</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References and Notes</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000885v2.2"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">W. H. Organization</span>, <span class="cit-source">Novel Coronavirus (COVID-19) Situation</span>. <a href="https://experience.arcgis.com/experience/685d0ace521648f8a5beeeee1b9125cd">https://experience.arcgis.com/experience/685d0ace521648f8a5beeeee1b9125cd</a>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Gorbalenya</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.5" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. L.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000885v2.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.-D.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Extensive diversity of coronaviruses in bats from China</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">507</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D507%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Banerjee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kulcsar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Misra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Frieman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mossman</span></span>, <span class="cit-article-title">Bats and Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">41</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.8" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhou</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Roujian</span> <span class="cit-name-surname">Lu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. T.-Y.</span> <span class="cit-name-surname">Lam</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2013.945485</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-P.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-article-title">Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica)</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">979</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guan</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">302</span>, <span class="cit-fpage">276</span>–<span class="cit-lpage">278</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.volume%253D302%26rft.spage%253D276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.13" data-doi="10.1056/NEJMoa1401505"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. I.</span> <span class="cit-name-surname">Azhar</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evidence for camel-to-human transmission of MERS coronavirus</span>. <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr> <span class="cit-vol">370</span>, <span class="cit-fpage">2499</span>–<span class="cit-lpage">2505</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft.volume%253D370%26rft.spage%253D2499%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1401505%26rft_id%253Dinfo%253Apmid%252F24896817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1401505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24896817&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337804400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.14" data-doi="10.1128/JVI.73.1.152-160.1999"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Lole</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">73</span>, <span class="cit-fpage">152</span>–<span class="cit-lpage">160</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLole%26rft.auinit1%253DK.%2BS.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D152%26rft.epage%253D160%26rft.atitle%253DFull-Length%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%25A0Genomes%2Bfrom%2BSubtype%2BC-Infected%2BSeroconverters%2Bin%2BIndia%252C%2Bwith%2BEvidence%2Bof%2BIntersubtype%2BRecombination%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.73.1.152-160.1999%26rft_id%253Dinfo%253Apmid%252F9847317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjczLzEvMTUyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjQvMjAyMC4wMy4yMC4wMDA4ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Wong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J. Javornik</span> <span class="cit-name-surname">Cregeen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Ajami</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Petrosino</span></span>, <span class="cit-article-title">Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2007.939207</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.16" data-doi="10.1089/aid.1995.11.1413"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Siepel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Halpern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Macken</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Korber</span></span>, <span class="cit-article-title">A computer program designed to screen rapidly for HIV type 1 intersubtype recombinant sequences</span>. <abbr class="cit-jnl-abbrev">AIDS research and human retroviruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">1413</span>–<span class="cit-lpage">1416</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Bresearch%2Band%2Bhuman%2Bretroviruses%26rft.stitle%253DAIDS%2BRes%2BHum%2BRetroviruses%26rft.aulast%253DSiepel%26rft.auinit1%253DA.%2BC.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1413%26rft.epage%253D1416%26rft.atitle%253DA%2Bcomputer%2Bprogram%2Bdesigned%2Bto%2Bscreen%2Brapidly%2Bfor%2BHIV%2Btype%2B1%2Bintersubtype%2Brecombinant%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.1995.11.1413%26rft_id%253Dinfo%253Apmid%252F8573400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.1995.11.1413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8573400&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TG26400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.17" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Graham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Baric</span></span>, <span class="cit-article-title">Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI0LzIwMjAuMDMuMjAuMDAwODg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.19" data-doi="10.1371/journal.ppat.1006698"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1006698</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.20" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lan</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2019.956235</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wrapp</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr>, <span class="cit-fpage">eabb2507</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Walls</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, S0092-8674(0020)30262-30262 (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xiao</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">HIV-1 did not contribute to the 2019-nCoV genome</span>. <abbr class="cit-jnl-abbrev">Emerging Microbes &amp; Infections</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">378</span>–<span class="cit-lpage">381</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BMicrobes%2B%2526%2BInfections%26rft.volume%253D9%26rft.spage%253D378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.25" data-doi="10.1016/j.antiviral.2020.104742"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Coutard</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">104742</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D176%26rft.spage%253D104742%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2020.104742%26rft_id%253Dinfo%253Apmid%252F32057769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2020.104742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32057769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhou</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2003.2002.974139</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Walls</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2019.956581</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.28" data-doi="10.1128/JVI.01926-07"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Hon</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">1819</span>–<span class="cit-lpage">1826</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI0LzIwMjAuMDMuMjAuMDAwODg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.29" data-doi="10.1128/JVI.01048-15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Lau</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">10532</span>–<span class="cit-lpage">10547</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01048-15%26rft_id%253Dinfo%253Apmid%252F26269185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMC8xMDUzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI0LzIwMjAuMDMuMjAuMDAwODg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.30" data-doi="10.1080/22221751.2020.1719902"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Chan</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">221</span>–<span class="cit-lpage">236</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.31" data-doi="10.1038/nm.3985"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. D.</span> <span class="cit-name-surname">Menachery</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</span>. <abbr class="cit-jnl-abbrev">Nature medicine</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">1508</span>–<span class="cit-lpage">1513</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.volume%253D21%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3985%26rft_id%253Dinfo%253Apmid%252F26552008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.3985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26552008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000885v2.32" data-doi="10.1073/pnas.1517719113"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. D.</span> <span class="cit-name-surname">Menachery</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">SARS-like WIV1-CoV poised for human emergence</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">3048</span>–<span class="cit-lpage">3053</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517719113%26rft_id%253Dinfo%253Apmid%252F26976607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzExLzMwNDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNC8yMDIwLjAzLjIwLjAwMDg4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.33" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.-Y.</span> <span class="cit-name-surname">Ge</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Serological Evidence of Bat SARS-Related Coronavirus Infection in Humans, China</span>. <abbr class="cit-jnl-abbrev">Virol Sin</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">104</span>–<span class="cit-lpage">107</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BSin%26rft.volume%253D33%26rft.spage%253D104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.35" data-doi="10.1002/jcc.540150503"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Abagyan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Totrov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kuznetsov</span></span>, <span class="cit-article-title">ICM—A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation</span>. <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">488</span>–<span class="cit-lpage">506</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.volume%253D15%26rft.spage%253D488%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.540150503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.540150503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NG57200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.36"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McCauley</span></span>, <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data - from vision to reality</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr> <span class="cit-vol">22</span>, (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?</span> <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2018.954628</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Xiao</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 2020.2002.2017.951335</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.39" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Larkin</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.40" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Guindon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Gascuel</span></span>, <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr> <span class="cit-vol">52</span>, <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.20.000885v2.41"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Foley</span></span> <span class="cit-etal">et al.</span>, <span class="cit-source">HIV sequence compendium 2018</span>. (<span class="cit-publ-name">Theoretical Biology and Biophysics Group, Los Alamos National Laboratory, NM, LA-UR 18-25673</span>, <span class="cit-publ-loc">Los Alamos, New Mexico</span>, <span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.20.000885v2.42"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Rodrigo</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Learn</span></span></li></ol><cite><span class="cit-auth"> <span class="cit-name-given-names">B. B. T.</span> <span class="cit-name-surname">Korber</span></span>, in <span class="cit-source">Computational Analysis of HIV Molecular Sequences</span>, <span class="cit-ed"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Rodrigo</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Learn</span></span>, Eds. (<span class="cit-publ-name">Kluwer Academic Publishers, Dordrecht, Netherlands</span>, <span class="cit-pub-date">2000</span>), chap. 4, pp. <span class="cit-fpage">55</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.43" data-doi="10.1371/journal.pgen.1002764"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Murrell</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detecting individual sites subject to episodic diversifying selection</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e1002764</span>–<span class="cit-lpage">e1002764</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMurrell%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002764%26rft.epage%253De1002764%26rft.atitle%253DDetecting%2Bindividual%2Bsites%2Bsubject%2Bto%2Bepisodic%2Bdiversifying%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002764%26rft_id%253Dinfo%253Apmid%252F22807683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22807683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.44" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. F.</span> <span class="cit-name-surname">Pettersen</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.20.000885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.45"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-article-title">Protein Structure and Function Prediction Using I-TASSER</span>. <abbr class="cit-jnl-abbrev">Current protocols in bioinformatics</abbr> <span class="cit-vol">52</span>, <span class="cit-fpage">5.8.1</span>-<span class="cit-lpage">5.8.15</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bprotocols%2Bin%2Bbioinformatics%26rft.volume%253D52%26rft.spage%253D581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000885v2.46" data-doi="10.21203/rs.2.24749/v1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shang</span></span> <span class="cit-etal">et al.</span>, <abbr class="cit-jnl-abbrev">Structural basis for receptor recognition by the novel coronavirus from Wuhan</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.21203/rs.2.24749/v1</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructural%2Bbasis%2Bfor%2Breceptor%2Brecognition%2Bby%2Bthe%2Bnovel%2Bcoronavirus%2Bfrom%2BWuhan%26rft_id%253Dinfo%253Adoi%252F10.21203%252Frs.2.24749%252Fv1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21203/rs.2.24749/v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.20.000885v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.24.004655v1" title="SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.02.15.951095v3" title="Long non-coding RNA Gm12664 up-regulates CAV1 expression to promote hepatocellular lipid accumulation by sponging miR-295-5p" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 24, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.20.000885v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.20.000885v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.20.000885v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.20.000885v2" class="active" data-icon-position="" data-hide-link-title="0">Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1210263" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ukEi-RCkp2_5vbJFMPxmGTOaAZ62MXt2s8H44flZiXU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210263" id="node1210263--2" data-pisa="biorxiv;2020.03.20.000885v2" data-pisa-master="biorxiv;2020.03.20.000885" data-seqnum="1210263" data-apath="/biorxiv/early/2020/03/24/2020.03.20.000885.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elena E.</span> <span class="nlm-surname">Giorgi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Manukumar Honnayakanahalli</span> <span class="nlm-surname">Marichann</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Foley</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xiang-Peng</span> <span class="nlm-surname">Kong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bette</span> <span class="nlm-surname">Korber</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Gao</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.20.000885; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.20.000885 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;title=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;title=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;text=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;title=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;t=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;title=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;title=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210263" id="node1210263--3" data-pisa="biorxiv;2020.03.20.000885v2" data-pisa-master="biorxiv;2020.03.20.000885" data-seqnum="1210263" data-apath="/biorxiv/early/2020/03/24/2020.03.20.000885.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elena E.</span> <span class="nlm-surname">Giorgi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Manukumar Honnayakanahalli</span> <span class="nlm-surname">Marichann</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Foley</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Xiang-Peng</span> <span class="nlm-surname">Kong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yue</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bette</span> <span class="nlm-surname">Korber</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Gao</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.20.000885; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.20.000885 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Li%2C+Xiaojun&amp;rft.au=Giorgi%2C+Elena+E.&amp;rft.au=Marichann%2C+Manukumar+Honnayakanahalli&amp;rft.au=Foley%2C+Brian&amp;rft.au=Xiao%2C+Chuan&amp;rft.au=Kong%2C+Xiang-Peng&amp;rft.au=Chen%2C+Yue&amp;rft.au=Korber%2C+Bette&amp;rft.au=Gao%2C+Feng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1210263/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1210263/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1210263/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1210263/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1210263/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1210263/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1210263/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1210263/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1210263/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1210263/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1210263/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1210263/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;count=horizontal&amp;via=&amp;text=Emergence%20of%20SARS-CoV-2%20through%20Recombination%20and%20Strong%20Purifying%20Selection&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000885v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.20.000885v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
