<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/01/29/528174.atom" />
<meta name="HW.pisa" content="biorxiv;528174v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Virome heterogeneity and connectivity in waterfowl and shorebird communities" />
<meta name="DC.Identifier" content="10.1101/528174" />
<meta name="DC.Date" content="2019-01-29" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses ( Picobirnaviridae and Reoviridae ), single-stranded RNA viruses ( Astroviridae, Caliciviridae, Picornaviridae ), a retro-transcribing DNA virus ( Hepadnaviridae ), and a single-stranded DNA virus ( Parvoviridae ). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel Rotavirus species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations." />
<meta name="DC.Contributor" content="Michelle Wille" />
<meta name="DC.Contributor" content="Mang Shi" />
<meta name="DC.Contributor" content="Marcel Klaassen" />
<meta name="DC.Contributor" content="Aeron C. Hurt" />
<meta name="DC.Contributor" content="Edward C. Holmes" />
<meta name="article:published_time" content="2019-01-29" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Virome heterogeneity and connectivity in waterfowl and shorebird communities" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses (&lt;i&gt;Picobirnaviridae&lt;/i&gt; and &lt;i&gt;Reoviridae&lt;/i&gt;), single-stranded RNA viruses (&lt;i&gt;Astroviridae, Caliciviridae, Picornaviridae&lt;/i&gt;), a retro-transcribing DNA virus (&lt;i&gt;Hepadnaviridae&lt;/i&gt;), and a single-stranded DNA virus (&lt;i&gt;Parvoviridae&lt;/i&gt;). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel &lt;i&gt;Rotavirus&lt;/i&gt; species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;528174v1" />
<meta name="citation_id" content="528174v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/528174v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/528174v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/528174v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174.full.pdf" />
<meta name="citation_doi" content="10.1101/528174" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="528174" />
<meta name="citation_author" content="Michelle Wille" />
<meta name="citation_author_institution" content="WHO Collaborating Centre for Reference and Research on Influenza, at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_email" content="michelle.wille@influenzacentre.org" />
<meta name="citation_author_email" content="edward.holmes@sydney.edu.au" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5629-0196" />
<meta name="citation_author" content="Mang Shi" />
<meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, Charles Perkins Centre, School of Life and Environmental Sciences and Sydney Medical School, The University of Sydney" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6154-4437" />
<meta name="citation_author" content="Marcel Klaassen" />
<meta name="citation_author_institution" content="Centre for Integrative Ecology, Deakin University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3907-9599" />
<meta name="citation_author" content="Aeron C. Hurt" />
<meta name="citation_author_institution" content="WHO Collaborating Centre for Reference and Research on Influenza, at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1826-4314" />
<meta name="citation_author" content="Edward C. Holmes" />
<meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, Charles Perkins Centre, School of Life and Environmental Sciences and Sydney Medical School, The University of Sydney" />
<meta name="citation_author_email" content="michelle.wille@influenzacentre.org" />
<meta name="citation_author_email" content="edward.holmes@sydney.edu.au" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9596-3552" />
<meta name="citation_reference" content="Ostfeld RS,Keesing F. Effects of Host Diversity on Infectious Disease. Annu Rev Ecol Evol S. 2012;43:157–82." />
<meta name="citation_reference" content="Keesing F, Holt RD, Ostfeld RS. Effects of species diversity on disease risk. Ecol Lett. 2006;9(4):485–98." />
<meta name="citation_reference" content="Johnson PT, Preston DL, Hoverman JT, LaFonte BE. Host and parasite diversity jointly control disease risk in complex communities. Proc Natl Acad Sci U S A. 2013;110(42):16916–21." />
<meta name="citation_reference" content="LoGiudice K, Ostfeld RS, Schmidt KA, Keesing F. The ecology of infectious disease: Effects of host diversity and community composition on Lyme disease risk. Proc Natl Acad Sci U S A. 2003;100(2):567–71." />
<meta name="citation_reference" content="Milholland MT, Castro-Arellano I, Suzan G, Garcia-Pena GE, Lee TE, Rohde RE, et al. Global Diversity and Distribution of Hantaviruses and Their Hosts. EcoHealth. 2018;15(1):163–208." />
<meta name="citation_reference" content="Haydon DT, Cleaveland S, Taylor LH, Laurenson MK. Identifying reservoirs of infection: A conceptual and practical challenge. Emerg Infect Dis. 2002;8(12):1468–73." />
<meta name="citation_reference" content="Altizer S, Nunn CL, Thrall PH, Gittleman JL, Antonovics J, Cunningham AA, et al. Social organization and parasite risk in mammals: Integrating theory and empirical studies. Annual Review of Ecology Evolution and Systematics. 2003;34:517–47." />
<meta name="citation_reference" content="Streicker DG, Fenton A, Pedersen AB. Differential sources of host species heterogeneity influence the transmission and control of multihost parasites. Ecology Letters. 2013;16(8):975–84." />
<meta name="citation_reference" content="Geoghegan JL, Holmes EC. Predicting virus emergence amid evolutionary noise. Open Biol. 2017;7(10)." />
<meta name="citation_reference" content="Wille M, Avril A, Tolf C, Schager A, Larsson S, Borg O, et al. Temporal dynamics, diversity, and interplay in three components of the viriodiversity of a Mallard population: Influenza A virus, avian paramyxovirus and avian coronavirus. Infect Genet Evol. 2015;29:129–37." />
<meta name="citation_reference" content="van Dijk JGB, Verhagen JH, Wille M, Waldenström J. Host and virus ecology as determinants of influenza A virus transmission in wild birds. Curr Opin Virol. 2018;28:26–36." />
<meta name="citation_reference" content="Dann P. Foraging behaviour and diets of red-necked stints and curlew sandpipers in south-eastern Australia. Wildlife Research. 2000;27(1):61–8." />
<meta name="citation_reference" content="Wikramaratna PS, Pybus OG, Gupta S. Contact between bird species of different lifespans can promote the emergence of highly pathogenic avian influenza strains. Proc Natl Acad Sci U S A. 2014;111(29):10767–72." />
<meta name="citation_reference" content="Ren HG, Jin Y, Hu MD, Zhou J, Song T, Huang ZS, et al. Ecological dynamics of influenza A viruses: cross-species transmission and global migration. Sci Rep-Uk. 2016;6." />
<meta name="citation_reference" content="Olsen B, Munster VJ, Wallensten A, Waldenström J, Osterhaus ADME, Fouchier RAM. Global patterns of influenza A virus in wild birds. Science. 2006;312:384–8." />
<meta name="citation_reference" content="Munster VJ, Baas C, Lexmond P, Waldenström J, Wallensten A, Fransson T, et al. Spatial, temporal, and species variation in prevalence of influenza A viruses in wild migratory birds. PloS Pathog. 2007;3:e61. doi:10.1371/journal.ppat.0030061." />
<meta name="citation_reference" content="Maxted AM, Luttrell MP, Goekjian VH, Brown JD, Niles LJ, Dey AD, et al. Avian influenza virus infection dynamics in shorebird hosts. J Wildl Dis. 2012;48(2):322–34." />
<meta name="citation_reference" content="Bröjer C, Ågren EO, Uhlhorn H, Bernodt K, Mörner T, Jansson DS, et al. Pathology of natural highly pathogenic avian influenza H5N1 infection in wild tufted ducks (Aythya fuligula). J Vet Diagn Invest. 2009;21:579–87." />
<meta name="citation_reference" content="Pantin-Jackwood MJ, Costa-Hurtado M, Shepherd E, DeJesus E, Smith D, Spackman E, et al. Pathogenicity and Transmission of H5 and H7 Highly Pathogenic Avian Influenza Viruses in Mallards. J Virol. 2016;90(21):9967–82." />
<meta name="citation_reference" content="Kim J-K, Negovetich NJ, Forrest HL, Webster RG. Ducks: the “trojan horses” of H5N1 influenza. Influenza and Other Respiratory Viruses. 2009;3:121–8." />
<meta name="citation_reference" content="Magor KE. Immunoglobulin genetics and antibody responses to influenza in ducks. Dev Comp Immunol. 2011;35:1008–16." />
<meta name="citation_reference" content="Hill SC, Manvell RJ, Schulenburg B, Shell W, Wikramaratna PS, Perrins C, et al. Antibody responses to avian influenza viruses in wild birds broaden with age. Proceedings Biological sciences. 2016;283(1845):doi:10.1098/rspb.2016.159." />
<meta name="citation_reference" content="Ferenczi M. Avian influenza virus dynamics in Australian wild birds. PhD Thesis: Deakin University; 2016." />
<meta name="citation_reference" content="Ferenczi M, Beckmann C, Warner S, Loyn R, O’Riley K, Wang X, et al. Avian influenza infection dynamics under variable climatic conditions, viral prevalence is rainfall driven in waterfowl from temperate, south-east Australia. Vet Res. 2016;47:23." />
<meta name="citation_reference" content="Wille M, Eden JS, Shi M, Klaassen M, Hurt AC, Holmes EC. Virus-virus interactions and host ecology are associated with RNA virome structure in wild birds. Mol Ecol. 2018: doi:10.1111/mec.14918." />
<meta name="citation_reference" content="Shi M, Lin XD, Tian JH, Chen LJ, Chen X, Li CX, et al. Redefining the invertebrate RNA virosphere. Nature. 2016;540(7634):539–43." />
<meta name="citation_reference" content="Shi M, Lin XD, Chen X, Tian JH, Chen LJ, Li K, et al. The evolutionary history of vertebrate RNA viruses. Nature. 2018;556(7700):197–202." />
<meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–U54." />
<meta name="citation_reference" content="Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80." />
<meta name="citation_reference" content="Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009;25(15):1972–3." />
<meta name="citation_reference" content="Guindon S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Systematic biology. 2010;59(3):307–21." />
<meta name="citation_reference" content="Lagkouvardos I, Fischer S, Kumar N, Clavel T. Rhea: a transparent and modular R pipeline for microbial profiling based on 16S rRNA gene amplicons. Peerj. 2017;5:e2836. doi:10.7717/peerj.2836." />
<meta name="citation_reference" content="Oksanen J, Kindt R, Legendre P, O’Hara B, Stevens MHH, Oksanen MJ, et al. The vegan package. Community Ecology Package. 2007;10:631–7." />
<meta name="citation_reference" content="McMurdie PJ, Holmes S. phyloseq: An R package for reproducible interactive analysis and graphics of microbiome census data. PloS ONE. 2013;8(4):e61217. doi:10.1371/journal.pone.0061217." />
<meta name="citation_reference" content="Brooks AW, Kohl KD, Brucker RM, van Opstal EJ, Bordenstein SR. Phylosymbiosis: Relationships and Functional Effects of Microbial Communities across Host Evolutionary History. PloS Biology. 2016;14(11)." />
<meta name="citation_reference" content="Robinson DF, Foulds LR. Comparison of Phylogenetic Trees. Math Biosci. 1981;53(1-2):131–47." />
<meta name="citation_reference" content="Schliep KP. phangorn: phylogenetic analysis in R. Bioinformatics. 2011;27(4):592–3." />
<meta name="citation_reference" content="Penny D, Hendy MD. The Use of Tree Comparison Metrics. Syst Zool. 1985;34(1):75–82." />
<meta name="citation_reference" content="Geoghegan JL, Duchene S, Holmes EC. Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. PloS Pathog. 2017;13(2)." />
<meta name="citation_reference" content="Paradis E, Claude J, Strimmer K. APE: Analyses of Phylogenetics and Evolution in R language. Bioinformatics. 2004;20(2):289–90." />
<meta name="citation_reference" content="Barth JMI, Matschiner M, Robertson BC. Phylogenetic Position and Subspecies Divergence of the Endangered New Zealand Dotterel (Charadrius obscurus). PloS One. 2013;8(10)." />
<meta name="citation_reference" content="Baker AJ, Pereira SL, Paton TA. Phylogenetic relationships and divergence times of Charadriiformes genera: multigene evidence for the Cretaceous origin of at least 14 clades of shorebirds. Biol Lett-Uk. 2007;3(2):205–9." />
<meta name="citation_reference" content="Gibson R, Baker A. Multiple gene sequences resolve phylogenetic relationships in the shorebird suborder Scolopaci (Aves: Charadriiformes). Mol Phylogenet Evol. 2012;64(1):66–72." />
<meta name="citation_reference" content="Sun ZL, Pan T, Hu CC, Sun L, Ding HW, Wang H, et al. Rapid and recent diversification patterns in Anseriformes birds: Inferred from molecular phylogeny and diversification analyses. PloS One. 2017;12(9)." />
<meta name="citation_reference" content="Revell LJ. phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol Evol. 2012;3(2):217–23." />
<meta name="citation_reference" content="Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biology. 2014;15(12)." />
<meta name="citation_reference" content="Latorre-Margalef N, Tolf C, Grosbois V, Avril A, Bengtsson D, Wille M, et al. Long-term variation in influenza A virus prevalence and subtype diversity in a migratory Mallards in Northern Europe. Proceedings Biological sciences. 2014;281:doi:10.1098/rspb.2014.0098" />
<meta name="citation_reference" content="Wilcox BR, Knutsen GA, Berdeen J, Goekjian VH, Poulson R, Goyal S, et al. Influenza A viruses in ducks in Northwestern Minnesota: fine scale spatial and temporal variation in prevalence and subtype diversity. PloS ONE. 2011;6:e24010. doi:10.1371/journal.pone.0024010." />
<meta name="citation_reference" content="Wille M, Lindqvist K, Muradrasoli S, Olsen B, Jarhult JD. Urbanization and the dynamics of RNA viruses in Mallards (Anas platyrhynchos). Infect Genet Evol. 2017;51:89–97." />
<meta name="citation_reference" content="Wille M, Muradrasoli S, Nilsson A, Jarhult JD. High prevalence and putative lineage maintenance of avian coronaviruses in Scandinavian waterfowl. PloS ONE. 2016;11(3):e0150198. doi:10.1371/journal.pone." />
<meta name="citation_reference" content="Chamings A, Nelson TM, Vibin J, Wille M, Klaassen M, Alexandersen S. Detection and characterisation of coronaviruses in migratory and non-migratory Australian wild birds. Sci Rep-Uk. 2018;8." />
<meta name="citation_reference" content="Hoque MA, Burgess GW, Cheam AL, Skerratt LF. Epidemiology of avian influenza in wild aquatic birds in a biosecurity hotspot, North Queensland, Australia. Preventive veterinary medicine. 2015;118(1):169–81." />
<meta name="citation_reference" content="Hoque MA, Burgess GW, Karo-Karo D, Cheam AL, Skerratt LF. Monitoring of wild birds for Newcastle disease virus in north Queensland, Australia. Preventive veterinary medicine. 2012;103(1):49–62." />
<meta name="citation_reference" content="de Souza WM, Romeiro MF, Fumagalli MJ, Modha S, de Araujo J, Queiroz LH, et al. Chapparvoviruses occur in at least three vertebrate classes and have a broad biogeographic distribution. J Gen Virol. 2017;98(2):225–9." />
<meta name="citation_reference" content="Roediger B, Lee Q, Tikoo S, Cobbin JCA, Henderson JM, Jormakka M, et al. An Atypical Parvovirus Drives Chronic Tubulointerstitial Nephropathy and Kidney Fibrosis. Cell. 2018;175(2):530–43 e24." />
<meta name="citation_reference" content="Jourdain E, van Riel D, Munster V, Kuiken T, Waldenström J, Olsen B, et al. The pattern of influenza virus attachment varies among wild bird species. PloS ONE. 2011;6(9):doi:10.1371/journal.pone.0024155." />
<meta name="citation_reference" content="Hall SR, Sivars-Becker L, Becker C, Duffy MA, Tessier AJ, Caceres CE. Eating yourself sick: transmission of disease as a function of foraging ecology. Ecology Letters. 2007;10(3):207–18." />
<meta name="citation_reference" content="Lozano GA. Optimal Foraging Theory - a Possible Role for Parasites. Oikos. 1991;60(3):391–5." />
<meta name="citation_reference" content="Roche B, Lebarbenchon C, Gauthier-Clerc M, Chang CM, Thomas F, Renaud F, et al. Water-borne transmission drives avian influenza dynamics in wild birds: the case of the 2005-2006 epidemics in the Camargue area. Infect Genet Evol. 2009;9(5):800–5." />
<meta name="citation_reference" content="Hoye BJ, Fouchier RAM, Klaassen M. Host behaviour and physiology underpin individual variation in avian influenza virus infection in migratory Bewick’s Swans. Proceedings Biological sciences. 2012;279(1728):529–34." />
<meta name="citation_reference" content="Guo H, Mason WS, Aldrich CE, Saputelli JR, Miller DS, Jilbert AR, et al. Identification and characterization of avihepadnaviruses isolated from exotic anseriformes maintained in captivity.J Virol. 2005;79(5):2729–42." />
<meta name="twitter:title" content="Virome heterogeneity and connectivity in waterfowl and shorebird communities" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses ( Picobirnaviridae and Reoviridae ), single-stranded RNA viruses ( Astroviridae, Caliciviridae, Picornaviridae ), a retro-transcribing DNA virus ( Hepadnaviridae ), and a single-stranded DNA virus ( Parvoviridae ). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel Rotavirus species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations." />
<meta name="og-title" property="og:title" content="Virome heterogeneity and connectivity in waterfowl and shorebird communities" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/528174v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses ( Picobirnaviridae and Reoviridae ), single-stranded RNA viruses ( Astroviridae, Caliciviridae, Picornaviridae ), a retro-transcribing DNA virus ( Hepadnaviridae ), and a single-stranded DNA virus ( Parvoviridae ). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel Rotavirus species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-01-29" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/528174v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/528174v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/528174v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/528174v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/156923" />
    <title>Virome heterogeneity and connectivity in waterfowl and shorebird communities | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"phpyjAbyIctDwm0H5LWsN5w-TKyi19CCxIDQ-OblQ6Y","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"156923","apath":"\/biorxiv\/early\/2019\/01\/29\/528174.atom","pisa":"biorxiv;528174v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;528174v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/528174v1","title":"Virome heterogeneity and connectivity in waterfowl and shorebird communities","identifier":"node\/156923"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h0sifK1UsMMgNOU1hYEiVOmT\\\\\\\/k9E0VrY7X4BUVUECuOF2BeCzkQd+FlYmvu2MlMWfG7zh2D6c1PNyYwUtT4tSUEqvBQaCjrZCy8bO\\\\\\\/w8I9EBZYptwoa8hMaGn56qR7vv6R8\\\\\\\/JVV5CJsdzAcEeYSSfWeIvgAfIftP0a9wje1pb0dO0vx74+t4slwBFOO665yWJT42Rsle4s4smuGP2QepL0iRB3fOgoi5WiER1DJ4X45lUhae18i26WXpXpRYeqd2aHISf+tObvhVZ35T9hXb4wASB\\\\\\\/gDSNpv0zdBqjnmSlB0rGo2wC4vArdUK2R5u2hYxIlvGQfKtmuoTIpvcinzUI6XKt5pAMrothWn7eh5dr8KrBhK\\\\\\\/8jw8sL1LgULcWI6PUSvyLGR8m+C6xVYNgPtgg52NWPdYvPLS0CgBQjQn64+BnwKqTkitMJnzTa00UL7WiRyMm5k9ALuspAmuwlEzhGYxmk9oWUYHtaRYLcciBUmIji7ZLaqxQVqEDoNNpelCexSB4U7aFqTwIIg+4qWtfpsxr2qkcAzF6zJISAIlHfm\\\\\\\/hXWZ+C7oXYUaTAa3EmoHq0i1vFqW9i+4gyTWrlk7tEwrRZ9ci3AL0ZAPnnCi+7pUb7bgJ9Hu1K+kmlMTBwB03ygOAvBd9momScFc0Ks9eB4T\\\\\\\/NNm\\\\\\\/2UWsomTNVo+h3cWSY\\\\\\\/i5FhXjg9GZlyzA+YDUUMkXsyU\\\\\\\/PuvgO0DzTPjU3EuH98jrIaStx0v+zTF40bUGzqu0Aktea9oIHd4fjwGS8\\\\\\\/2aROMoYnlFzzDelSVDUA4x6rgoXzpWKLAYKDl1SryJuxhUJ9s8AyGB\\\\\\\/SifBsslwEdyx5\\\\\\\/4lDBW3AKMOGzTZd3yODg4yYKKrxZoM4RhHkvYYR603\\\\\\\/dFU8RCgJJMBCa+32R4S9i+i5O2R7FJ7KCQ+5KVvOBxhXZYOojIxxzU2zrcAy0BLMt4bojN2BO1Sdemh2K86sccmgeUTG2LHWTR26cx3ZMLUbh+HnvHzzayF6YyLkQXSAflg9gO10pCJyAu2kkMmIbHpk8mPq5yTOmjgbfimQh3S8vj89f0KDwiGseO7XvlmmLs2AkllbadKKhcL81Dyd3SPjUg8nH6nUkF8Lo5a87RobsPe9KQHmKwIR8Dm6MgHYGUY4Ux1P6BIQl0zxQzFhzwwLpAcVyiqEVRi3nzSUHk4Psl\\\\\\\/AVgZWlKan5E38UzlmUvMT1JDSO4usyRb6jDo+HB1Oe+4gMUklY9ASDoqOP9uJfpoArIjOfw4JAr8+kNC1n6c9nAYJJbVPfndcLTcwpK1MlIwmjBkSULL5Kx8Ggisofv4p+smaXek=\\\u0022,\\\u0022iv\\\u0022:\\\u0022y9Ar+WZKq8d7lgxHkAnPsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002239df0060c62087cc4c7c6cf6a458094746ee6cdf8c6f8b2644324730ca2d0de9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/528174v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-156923 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/528174v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fvr0A8tLpP4U7OoRP2S239Hun9FGV87Z0nB9of2_wk4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="156923" id="node156923" data-pisa="biorxiv;528174v1" data-pisa-master="biorxiv;528174" data-apath="/biorxiv/early/2019/01/29/528174.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Virome heterogeneity and connectivity in waterfowl and shorebird communities</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5629-0196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Wille</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6154-4437" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Mang</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3907-9599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Klaassen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1826-4314" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Aeron C.</span> <span class="nlm-surname">Hurt</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9596-3552" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Edward C.</span> <span class="nlm-surname">Holmes</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/528174 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node156923" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michelle Wille </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>WHO Collaborating Centre for Reference and Research on Influenza, at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BWille%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wille%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichelle%2BWille%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5629-0196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michelle Wille</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michelle.wille@influenzacentre.org" class="" data-icon-position="" data-hide-link-title="0">michelle.wille@influenzacentre.org</a>
<a href="mailto:edward.holmes@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">edward.holmes@sydney.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mang Shi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Marie Bashir Institute for Infectious Diseases and Biosecurity, Charles Perkins Centre, School of Life and Environmental Sciences and Sydney Medical School, The University of Sydney</span>, Sydney, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mang%2BShi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMang%2BShi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6154-4437" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mang Shi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcel Klaassen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Centre for Integrative Ecology, Deakin University</span>, Geelong, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcel%2BKlaassen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klaassen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcel%2BKlaassen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3907-9599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcel Klaassen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aeron C. Hurt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>WHO Collaborating Centre for Reference and Research on Influenza, at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aeron%2BC.%2BHurt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hurt%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAeron%2BC.%2BHurt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1826-4314" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aeron C. Hurt</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Edward C. Holmes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Marie Bashir Institute for Infectious Diseases and Biosecurity, Charles Perkins Centre, School of Life and Environmental Sciences and Sydney Medical School, The University of Sydney</span>, Sydney, <span class='nlm-country'>Australia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BC.%2BHolmes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holmes%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BC.%2BHolmes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9596-3552" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edward C. Holmes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michelle.wille@influenzacentre.org" class="" data-icon-position="" data-hide-link-title="0">michelle.wille@influenzacentre.org</a>
<a href="mailto:edward.holmes@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">edward.holmes@sydney.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/528174v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:156923" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:156923/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/528174v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:156923" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:156923/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/528174v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:156923" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:156923/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/528174v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:156923" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:156923/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/528174v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:156923" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:156923/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses (<em>Picobirnaviridae</em> and <em>Reoviridae</em>), single-stranded RNA viruses (<em>Astroviridae, Caliciviridae, Picornaviridae</em>), a retro-transcribing DNA virus (<em>Hepadnaviridae</em>), and a single-stranded DNA virus (<em>Parvoviridae</em>). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel <em>Rotavirus</em> species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Many hosts are members of multi-species aggregations and may be infected by an assemblage of specialist and/or multi-host generalist infectious agents. Host community diversity is central to pathogen dynamics (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), and pathogen species richness, relative abundance, specificity and intra- and inter-species interactions within assemblages likely have complex roles in modulating pathogen levels within populations (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). A significant limitation in studying viral communities in hosts is that most viral species remained undescribed (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>), such that viral ecology across multi-host systems has been limited to “single-virus” dynamics, particularly in vertebrate systems (for example, Influenza A virus [IAV] in avian populations). With the advent of unbiased, bulk ‘meta-transcriptomic’ RNA sequencing we can now explore, in more detail, how viral community structure may be shaped by host-species interactions.</p><p id="p-4">Birds of the orders Anseriformes and Charadriiformes, the central reservoirs for avian viruses such as IAV, avian avulavirus and avian coronavirus (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), form multi-host flocks, in which many species may migrate, forage, or roost together (e.g. (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). These flocks may comprise species along a taxonomically related gradient and may utilize similar or different ecological niches in the same environment. For example, in Australia, taxonomically related dabbling Grey Teals (<em>Anas gracilis</em>) and Pacific Black Ducks (<em>Anas superciliosa</em>) may share the environment with the distantly related filter feeding Pink-eared Duck (<em>Malacorhynchus membranaceus</em>). These multi-host flocks form multi-host maintenance communities (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>), with consequences for virus ecology, transmission, and virulence (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>).</p><p id="p-5">Studies of the ecology of IAV, the best studied multi-host virus in wild birds, have shown that not all hosts are equal (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). In particular, there are marked differences in susceptibility, pathology and the subsequent immune response in taxonomically related species, or distantly related species within similar ecological niches. For example, dramatic differences in viral prevalence exist within the Charadriiformes, such that Ruddy Turnstones (<em>Arenaria interpres</em>) may have an IAV prevalence of ∼15%, compared to the negligible IAV prevalence in co-sampled Sanderlings (<em>Calidris alba</em>) sampled at Delaware Bay, USA (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). There are also major differences in the pathology of highly pathogenic IAV in Anseriformes both in the field and in experimental infections. Mallards (<em>Anas platyrhynchos</em>) infected with the highly pathogenic IAV are thought to move the virus large distances and remain free of clinical signs, while Tufted Ducks (<em>Aythya fuligula</em>), in contrast, experience severe mortality (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Following IAV infection, dabbling ducks of the genus <em>Anas</em> are believed to develop poor immune memory (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>), allowing IAV re-infections throughout their lives, in contrast to swans that have long term immune memory (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) and where re-infection probability is likely very low in adults. These differences are driven by factors encompassing both virus (e.g. virulence, transmission) and host (e.g. receptor availability, immune responses) biology (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-6">The goal of this study was to use the analysis of comparative virome structures, particularly virome composition, diversity and abundance, as a means to describe the nature of host-virus interactions beyond the “one-host, one-virus” model. Given their role as hosts for multi-host viruses such as IAV, coronaviruses and avian avulaviruses, we used apparently healthy members of the Anseriformes and Charadriiformes as model avian taxa in these comparisons. In particular, we aimed to (i) reveal the viromes and describe novel viruses in the bird taxa sampled, (ii) determine whether viromes of different host orders have different abundance and viral diversity such that they cluster separately, (iii) determine whether closely taxonomically related and co-occuring avian hosts have viromes that are more homogenous, and (iv) identify the impact of host taxonomy, which we use as a proxy for differences in relevant host traits (such as host physiology, cell receptors), in determining virome structure. Overall, we reveal a combination of virome heterogeneity and connectivity across avian species that are important reservoirs of avian viruses.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Ethics statement</h3><p id="p-7">This research was conducted under approval of the Deakin University Animal Ethics Committee (permit numbers A113-2010 and B37-2013). Banding was performed under Australian Bird Banding Scheme permit (banding authority numbers 2915 and 2703). Research permits were approved by the Department of Environment, Land, Water and Planning Victoria (permit numbers 10006663 and 10005726)</p></div><div id="sec-4" class="subsection"><h3>Sample selection</h3><p id="p-8">Samples were collected as part of a long-term IAV surveillance study (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). Shorebirds were captured using cannon nets at the Western Treatment Plant near Melbourne (37°59′11.62′′S, 113 144°39′38.66′′E) during the same sampling event (n=434). Waterfowl were sampled post-mortem (within 12 hours) following harvest from lakes in south-west Victoria in March 2017 (n=125) (36°58’S 141°05′E) (Table S1). No birds showed any signs of disease.</p><p id="p-9">A combination or oropharyngeal and cloacal samples were collected using a sterile-tipped swab and were placed in viral transport media (VTM, Brain-heart infusion broth containing 2×106 122 U/l penicillin, 0.2 mg/ml streptomycin, 0.5 mg/ml gentamicin, 500 U/ml amphotericin B, Sigma).</p></div><div id="sec-5" class="subsection"><h3>RNA virus discovery</h3><p id="p-10">RNA was extracted and libraries constructed as per (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) (Table S1). Paired end sequencing (100bp) of the RNA library was performed on an Illumina platform at the Australian Genome Research Facility (AGRF, Melbourne). Similarly, sequence reads were demultiplexed and contigs assembled and contig abundance calculated as per (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>).</p><p id="p-11">All contigs that returned blast hits with paired abundance estimates were filtered to remove plants and invertebrate reads that likely correspond to the host diet, as well as fungal, bacterial and host sequences. The virus list was further filtered to remove viruses with likely invertebrate (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>), lower vertebrate (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>), plant or bacterial host associations using the Virus-Host database (<a href="http://www.genome.jp/virushostdb/">http://www.genome.jp/virushostdb/</a>) such that only those viruses that grouped within the previously defined vertebrate virus groups are identified as bird associated.</p></div><div id="sec-6" class="subsection"><h3>Virus genome annotation and phylogenetic analysis</h3><p id="p-12">Contigs were annotated, and phylogenetic treesinferred as per (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>). Briefly, contigs greater than 1000bp in length were inspected using Geneious R10 (Biomatters, New Zealand), and open reading frame corresponding to predicted genome architectures based on the closest reference genomes were interrogated. Reads were subsequently mapped back to viral contigs to identify mis-assembly using bowtie2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Viruses with full length genomes, or incomplete genomes but that possess the full-length RNA-dependant RNA polymerase (RdRp) gene, were used for phylogenetic analysis. Briefly, sequences of the polyprotein or gene encoding for the RdRp were aligned using MAFFT (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>), and gaps and ambiguously aligned regions were stripped using trimAL (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Final alignment lengths for each data set are presented in Table S2. The most appropriate amino acid substitution model was then determined for each data set, and maximum likelihood trees were estimated using PhyML 3.0 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>) with 1000 bootstrap replicates. Novel viral species were identified as those that had &lt;90% RdRp protein identity, or &lt;80% genome identity to previously described viruses. All reads have been deposited in the Short Read Archive (PRJNA505206) and viral sequences described have been deposited in GenBank (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK204384&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom">MK204384</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK20441&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom">MK20441</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK213322&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom">MK213322</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK213337&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom">MK213337</a>).</p></div><div id="sec-7" class="subsection"><h3>Diversity and abundance across libraries</h3><p id="p-13">Relative virus abundance was estimated as the proportion of the total viral reads in each library (excluding rRNA). All ecological measures were calculated using the data set comprising viruses associated with “higher” vertebrates (i.e. birds and mammals), albeit with all retroviruses and retrotransposons removed (hereafter, “avian virus data set”).</p><p id="p-14">Analyses were performed using R v 3.4.0 integrated into RStudio v 1.0.143, and plotted using <em>ggplot2</em>. Both the observed virome richness and Shannon effective (i.e. alpha diversity) were calculated for each library at the virus family and genus levels using the Rhea script sets (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>), and compared between avian orders using the Kruskal-Wallis rank sum test. Beta diversity was calculated using the Bray Curtis dissimilarity matrix and virome structure was plotted as a function of nonmetric multidimensional scaling (NMDS) ordination and tested using Adonis tests using the <em>vegan</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>) and <em>phyloseq</em> packages (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). To determine whether differences in virome composition can be explained by host phylogeny (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>), dendograms of beta diversity were constructed using the bray curtis dissimilarity matrix using the <em>vegan</em> package incorporated into the hclust() function. Dendograms representing library abundance and composition at the viral family, genera and species level of taxonomy were also compared to the host phylogeny. For the species level comparison, we used only those presented in Table S3. Two phylogenetic tree congruence metrics were then calculated to compare the match between the virome metric and host phylogeny: the matching cluster Robinson-Foulds tree metric (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>) was calculated using the <em>phangorn</em> package (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), and the normalized PH85 metric (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>) was determined using the <em>ape</em> package (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). For both metrics, a distance of 0 indicates complete congruence and 1, incomplete congruence. The phylogenetic relationships among the avian host species were inferred using a maximum likelihood tree of the cytochrome B mitochondrial sequences and accords with those determined previously (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>-<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). The overall co-phylogenetic analysis was visualized using the <em>phytools</em> package (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>).</p><p id="p-15">Finally, the relative abundance of virus families between each avian host order (Charadriiformes versus Anseriformes) was assessed using the wilcox test. Subsequently, log2 relative abundance was calculated using <em>DESeq2</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>) implemented in <em>phyloseq</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). Given the large number of genera detected within the <em>Picornaviridae</em>, this analysis was repeated at the viral genus level in this case.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Substantial undescribed diversity of RNA viruses in wild birds</h3><p id="p-16">We characterized the total transcriptome of nine avian species from the Anseriformes and Charadriiformes (Table S1). Within each avian order, bird species were sampled across the same spatial and temporal scales, although members of the Anseriformes and Charadriiformes were sampled at different time points (i.e. years) and locations. This resulted in two bird communities comprising five and four species, respectively (Table S1). RNA sequencing of rRNA depleted libraries resulted in an average of 50,906,440 reads (range 44,178,936-64,149,180) assembled into a mean of 27,108 contigs (range 22,175-496,319) (Table S1). There was no significant correlation between RNA concentration and the number of reads, contigs or abundance demonstrating such that this was not a source of bias (Fig S1). There was a large range in total viral abundance (0.14-10.67% viral reads) and putative avian viral abundance (0.00083-0.327% viral reads) in each library (Table S1, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>). In additional to avian viral reads, libraries had numerous reads matching arthropod viruses and retroviruses (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>). Although these retroviruses are likely bird associated, the challenge of differentiating between endogenous and exogenous retroviruses meant that they were excluded from the analysis, as were those viruses most likely associated with arthropods, plants and bacteria.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of viral contigs identified in this study. (A) Host association of viral contigs identified in this study: avian, retroviruses and retrotransposons, all other hosts including lower vertebrate, invertebrate, plant, bacterial hosts, and host reference gene ribosomal protein L4 (RPL4). (B) Abundance and alpha diversity of avian viral families identified in each library. Relative abundance and alpha diversity calculations are presented in Fig S15. Abundance and alpha diversity of viral genera and species is presented in Fig S16 and S17, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Overview of viral contigs identified in this study.&lt;/span&gt; (A) Host association of viral contigs identified in this study: avian, retroviruses and retrotransposons, all other hosts including lower vertebrate, invertebrate, plant, bacterial hosts, and host reference gene ribosomal protein L4 (RPL4). (B) Abundance and alpha diversity of avian viral families identified in each library. Relative abundance and alpha diversity calculations are presented in Fig S15. Abundance and alpha diversity of viral genera and species is presented in Fig S16 and S17, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F1.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F1.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Overview of viral contigs identified in this study.</span><p id="p-17" class="first-child">(A) Host association of viral contigs identified in this study: avian, retroviruses and retrotransposons, all other hosts including lower vertebrate, invertebrate, plant, bacterial hosts, and host reference gene ribosomal protein L4 (RPL4). (B) Abundance and alpha diversity of avian viral families identified in each library. Relative abundance and alpha diversity calculations are presented in Fig S15. Abundance and alpha diversity of viral genera and species is presented in Fig S16 and S17, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">A total of 24 of the 27 viruses identified in this study likely represent novel avian viral species (Table S3, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2</a>, Fig S2). Novel species were identified in the double-stranded RNA viruses (<em>Picobirnaviridae</em> and <em>Reoviridae</em>, genus Rotavirus), positive-sense single-stranded (ss) RNA viruses (<em>Astroviridae, Caliciviridae, Picornaviridae</em> genus Megrivirus, Gallivirus, and unassigned genera), and both retro-transcribing (<em>Avihepadnaviridae</em>) and single-stranded DNA viruses (<em>Parvoviridae</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bipartite network illustrating the species for which complete viral genomes were found in each library. Each library is represented as a central node, with a pictogram of the avian species, surrounded by each viral species. Where no complete viral genomes were revealed, the pictogram is shown with no viruses. Where two libraries share a virus species the networks between the two libraries are linked. Virus colour corresponds to virus taxonomy. A list of viruses from each library is presented in Table S3, and phylogenetic trees for each virus family and species can be found in (Fig 3-6, Figs S2-S14)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bipartite network illustrating the species for which complete viral genomes were found in each library. Each library is represented as a central node, with a pictogram of the avian species, surrounded by each viral species. Where no complete viral genomes were revealed, the pictogram is shown with no viruses. Where two libraries share a virus species the networks between the two libraries are linked. Virus colour corresponds to virus taxonomy. A list of viruses from each library is presented in Table S3, and phylogenetic trees for each virus family and species can be found in (Fig 3-6, Figs S2-S14).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F2.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F2.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-19" class="first-child">Bipartite network illustrating the species for which complete viral genomes were found in each library. Each library is represented as a central node, with a pictogram of the avian species, surrounded by each viral species. Where no complete viral genomes were revealed, the pictogram is shown with no viruses. Where two libraries share a virus species the networks between the two libraries are linked. Virus colour corresponds to virus taxonomy. A list of viruses from each library is presented in Table S3, and phylogenetic trees for each virus family and species can be found in (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a>-<a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>, Figs S2-S14).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h4>Novel ssRNA viruses</h4><p id="p-20">Two novel avastroviruses were identified. Red-necked Stint avastrovirus likely comprises a new virus “Group” as it falls basal to the Group 1 and 2 viruses in our phylogenetic analysis (Fig S3). Although no full genome Group 3 viruses exist, phylogenetic analysis of a short region of the RdRp demonstrated that this virus does not belong to Group 3 avastroviruses (Fig S4). Analysis of this short RdRp region also suggested that Red-necked Stint avastrovirus is sister to a virus detected in Swedish Mallards, indicating that this new group may be globally distributed (Fig S4). A Group 1 Avastrovirus, Pink-eared Duck astrovirus was also identified, and was a sister to Turkey astrovirus 2 (Fig S3, S4).</p><p id="p-21">Our study further expanded an unassigned avian specific genus of the <em>Caliciviridae</em> with the identification of three new species, two from Pink-eared Ducks and one from Grey Teal. These viruses form a clade comprising Goose calicivirus, Turkey calicivirus and a calicivirus previously identified in Red-necked Avocets from Australia (<em>Recurvirostra novaehollandiae</em>) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of the virus polyprotein, including the RdRp, of representatives of the vertebrate RNA virus family the Caliciviridae. Viruses identified in this study are denoted with a filled circle and in bold. The most divergent calicivirus, Atlantic Salmon calicivirus, was used as outgroup to root the tree. Bootstrap values &gt;70% are shown for key nodes. The scale bar represents the number of amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of the virus polyprotein, including the RdRp, of representatives of the vertebrate RNA virus family the &lt;em&gt;Caliciviridae&lt;/em&gt;. Viruses identified in this study are denoted with a filled circle and in bold. The most divergent calicivirus, Atlantic Salmon calicivirus, was used as outgroup to root the tree. Bootstrap values &gt;70% are shown for key nodes. The scale bar represents the number of amino acid substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F3.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F3.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-22" class="first-child">Phylogenetic tree of the virus polyprotein, including the RdRp, of representatives of the vertebrate RNA virus family the <em>Caliciviridae</em>. Viruses identified in this study are denoted with a filled circle and in bold. The most divergent calicivirus, Atlantic Salmon calicivirus, was used as outgroup to root the tree. Bootstrap values &gt;70% are shown for key nodes. The scale bar represents the number of amino acid substitutions per site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Members of the <em>Picornaviridae</em> were commonplace and genomes or partial genomes were identified in almost every library sequenced (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4</a>, Fig S5). Further, two different species of Picornaviruses were detected in the Red-necked Stint (Red-necked Stint Gallivirus and Red-necked Stint Picornavirus B-like) and Pink-eared Duck (Pink-eared Duck Megrivirus and Pink-eared Duck Picornavirus) libraries. To date, Galliviruses have only been isolated from Galliformes, so it was unexpected to identify a virus that was sister to this genus in the Red-necked Stint, although long branch lengths involved may indicate a novel viral genus. A virus similar to Avian sapeloviruses was identified in an Australian Shelduck (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4</a>, Fig S5). Three different Megriviruses were also identified, two from Anseriformes and one from Charadriiformes. Finally, a number of picornaviruses from unassigned genera similar to Pigeon picornavirus B were identified in Charadriiformes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4</a>, Fig S5). These form a clade with a number of picornaviruses previously detected in Red-necked Avocets from Australia. Two host libraries had more than one picornavirus species (Red-necked Stint and Pink-eared Duck) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogeny of the virus polyprotein, containing the RdRp, of selected members of the Picornaviridae. An expanded tree containing reference viruses for all main avian and mammalian genera is presented in Figure S5. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogeny of the virus polyprotein, containing the RdRp, of selected members of the &lt;em&gt;Picornaviridae&lt;/em&gt;. An expanded tree containing reference viruses for all main avian and mammalian genera is presented in Figure S5. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F4.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F4.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-24" class="first-child">Phylogeny of the virus polyprotein, containing the RdRp, of selected members of the <em>Picornaviridae</em>. An expanded tree containing reference viruses for all main avian and mammalian genera is presented in Figure S5. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">In addition to a previously described coronavirus, we identified a potentially novel species of deltacoronavirus. Specifically, sharp-tailed Sandpiper deltacoronavirus was most closely related to deltaviruses in wild birds from the United Arab Emirates and gulls from Europe (Fig S11), although the limited number of deltacoronaviruses sequences available inhibits a detailed analysis of its geographic range.</p></div><div id="sec-11" class="subsection"><h4>Novel dsRNA viruses</h4><p id="p-26">Three picobirnavirus species from two waterfowl species were newly identified here. Grey Teal picobirnavirus X and Pink-eared Duck picobirnavirus are members of a broad clade closely related to picobirnaviruses sampled in a number of species including Turkeys (from which only very short sequences are available and hence not analysed here (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5</a>). Grey Teal picobirnavirus Y falls into a divergent clade largely comprised of human and porcine picobirnaviruses (and no turkey picobirnaviruses), potentially representing an interesting host-switching event (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5</a>). However, due to limited sampling in wild birds, this virus could be related to other, currently unsampled, avian picorbirnaviruses. Two novel rotavirus species were also revealed from these host species. Indeed, Grey Teals and Pink-eared ducks shared a rotavirus species, distantly related to rotavirus G. This virus is one of three shared viruses in our entire dataset. Grey Teals also carried a second rotavirus species, distantly related to rotavirus F (Fig S7).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogeny of virus segment 2, containing the RdRp, of the Picobirnaviridae. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogeny of virus segment 2, containing the RdRp, of the &lt;em&gt;Picobirnaviridae&lt;/em&gt;. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F5.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F5.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-27" class="first-child">Phylogeny of virus segment 2, containing the RdRp, of the <em>Picobirnaviridae</em>. The tree was midpoint rooted for clarity only. Viruses described in this study are marked in bold, adjacent to a filled circle. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Novel DNA viruses</h4><p id="p-28">These data also provided evidence for the presence of a novel single-stranded DNA virus and retro-transcribing DNA virus. An Australian Shelduck parvovirus (ssDNA) was revealed in the Grey Teal (Grey Teal Parvovirus) that belongs to the highly divergent genus Chapparvovirus of the <em>Parvoviridae</em> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6</a>). Exogenous hepadnaviruses (retro-transcribing DNA viruses) from waterfowl are host specific, and the novel Wood Duck Hepatitis B virus identified here is most closely related to Shelgoose Hepatitis B virus, Duck Hepatitis B virus and Snow Goose Hepatitis Virus (Fig S8).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of the NS protein of representative Parvovirinae (ssDNA). The sequence generated in this study are indicated by a filled circle and are shown in bold. The tree was midpoint rooted for clarity only. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of the NS protein of representative &lt;em&gt;Parvovirinae&lt;/em&gt; (ssDNA). The sequence generated in this study are indicated by a filled circle and are shown in bold. The tree was midpoint rooted for clarity only. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F6.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F6.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-29" class="first-child">Phylogenetic tree of the NS protein of representative <em>Parvovirinae</em> (ssDNA). The sequence generated in this study are indicated by a filled circle and are shown in bold. The tree was midpoint rooted for clarity only. Bootstrap values &gt;70% are shown for key nodes. The scale bar indicates the number of amino acid substitutions per site.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-13" class="subsection"><h3>Previously described avian RNA viruses</h3><p id="p-30">Given their frequency in avian populations as described in numerous surveillance schemes, we anticipated finding IAV, Avian avulavirus type 1 (formerly avian paramyxovirus type 1), and members of the <em>Coronaviridae</em> in some of the species sampled. Not only did we detect these viruses, but IAV and avian gammacoronavirus were shared across three different waterfowl libraries (Fig S9-S10). Phylogenetic analysis of a partial RdRp revealed that the avian gammacoronavirus identified here were most closely related to those already found in Australian wild birds (Fig S10).</p><p id="p-31">We identified two subtypes of influenza A virus – H9N1 and H3N1 (Fig S12, S13) – in Grey Teal and Pink-eared Duck, respectively. Both H9N1 and H3N1 are rarely detected subtype combinations in large waterfowl surveillance schemes (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). Segments of these two viruses generally fell into the geographically isolated “Eurasian” clade, with the exception of the NP segment that fell within the “North American” clade, thereby demonstrating intercontinental reassortment in Australian waterfowl viruses (Fig S12, S13). Finally, although avian avulavirus Type 1 Class II Genotype 1b are frequently isolated from wild birds globally, we detected a Class II Genotype 1a virus infrequently isolated in wild birds (Fig S14). This genotype has been previously isolated from Australian chickens, although the F gene cleavage site (GRQGR*L) indicates this virus is of the low pathogenic phenotype.</p></div><div id="sec-14" class="subsection"><h3>Host heterogeneity and connectivity of avian viromes</h3><p id="p-32">There was a large variation in the abundance of avian viral reads across the libraries. The highest abundance of avian viruses was in Red-necked Stint and Sharp-tailed Sandpiper, with 0.08% of reads in both cases; the lowest viral abundance was also in this avian order (Curlew Sandpiper, 0.00018% of reads). Of the Anseriformes, Grey Teal, Australian Shelduck and Pink-eared Duck had high abundance (0.051%, 0.035%, and 0.047% of reads, respectively) while Wood Duck and Pacific Black Duck had very low abundance, albeit only one order of magnitude lower (0.0024% and 0.0019%, respectively) (Table S1).</p><p id="p-33">As with abundance, there was marked heterogeneity in alpha diversity (i.e. the diversity of viruses in each library) within the Anseriformes and Charadriiformes. In contrast to abundance, alpha diversity was higher in Anseriform than Charadriiform libraries at the viral family, genus, and species levels, although this difference was not statistically significant (Fig S15-S17). The lack of significance is potentially due to the surprisingly high alpha diversity in Red-necked Stint compared to other Charadriiformes, and the low alpha diversity in Pacific Black Ducks and Australian Wood Ducks in the Anseriformes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig 7A</a>, Fig S15-S18). Hence, despite sampling multispecies flocks, there can be a large variation in virome structure across species and potentially individuals.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterogeneity and lack of taxonomic structure in avian viromes. (A) Abundance of avian viral genera identities in each library. Libraries are arranged taxonomically, with cladograms illustrating host species phylogenetic relationships within the Charadriiformes and Anseriformes. The taxonomic identification presented is that of the top blastx hit of all avian viral contigs. Asterisks indicate cases in which at least one complete or partial (&gt;1000bp) virus was obtained. Alpha diversity metrics are presented in Fig S17. (B) Co-phylogeny demonstrating the discordance between host taxonomic relationship and virome composition. Host (phylogenetic) taxonomic relationship was inferred using the mitochondrial cytochrome B gene. Virome composition dendogram generated by clustering of bray-curtis dissimilarity matrix. The relationship between host taxonomy and virome composition was tested using two discordance metrics: Robinson-Foulds and nPH85, where 1 is discordance and 0 is agreement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-61894405" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Heterogeneity and lack of taxonomic structure in avian viromes.&lt;/span&gt; (A) Abundance of avian viral genera identities in each library. Libraries are arranged taxonomically, with cladograms illustrating host species phylogenetic relationships within the Charadriiformes and Anseriformes. The taxonomic identification presented is that of the top blastx hit of all avian viral contigs. Asterisks indicate cases in which at least one complete or partial (&gt;1000bp) virus was obtained. Alpha diversity metrics are presented in Fig S17. (B) Co-phylogeny demonstrating the discordance between host taxonomic relationship and virome composition. Host (phylogenetic) taxonomic relationship was inferred using the mitochondrial cytochrome B gene. Virome composition dendogram generated by clustering of bray-curtis dissimilarity matrix. The relationship between host taxonomy and virome composition was tested using two discordance metrics: Robinson-Foulds and nPH85, where 1 is discordance and 0 is agreement.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F7.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F7.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/528174/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">Heterogeneity and lack of taxonomic structure in avian viromes.</span><p id="p-34" class="first-child">(A) Abundance of avian viral genera identities in each library. Libraries are arranged taxonomically, with cladograms illustrating host species phylogenetic relationships within the Charadriiformes and Anseriformes. The taxonomic identification presented is that of the top blastx hit of all avian viral contigs. Asterisks indicate cases in which at least one complete or partial (&gt;1000bp) virus was obtained. Alpha diversity metrics are presented in Fig S17. (B) Co-phylogeny demonstrating the discordance between host taxonomic relationship and virome composition. Host (phylogenetic) taxonomic relationship was inferred using the mitochondrial cytochrome B gene. Virome composition dendogram generated by clustering of bray-curtis dissimilarity matrix. The relationship between host taxonomy and virome composition was tested using two discordance metrics: Robinson-Foulds and nPH85, where 1 is discordance and 0 is agreement.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">There was also substantial variation in the viral genera and species composition within each library (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig 7A</a>, Fig S15-S18). Members of the <em>Picornaviridae, Caliciviridae</em> and <em>Reoviridae</em> (genus <em>Rotavirus</em>) were ubiquitous and full genomes or short contigs were found in almost every library, often at high abundance (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig 7A</a>, Fig S14, S16). In addition to picornaviruses and rotaviruses, the Red-necked Stint had a highly abundant astrovirus (0.012%) and avulavirus (0.035%), while Sharp-tailed Sandpipers had a highly abundant deltacoronavirus (0.073%) that were not detected in other Charadriiform libraries (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig 7A</a>, S17). More viral families, genera and species were shared amongst the Anseriformes, particularly between Grey Teal, Pink-eared Duck and Australian Shelduck. Specifically, Grey Teal and Australian Shelduck shed avian gammacoronavirus at high abundance (0.021% and 0.029%, respectively) and Grey Teal and Pink-eared Duck shed IAV, although at lower abundances (0.0041% and 0.000728%, respectively) (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Fig 7A</a>, Fig S17). Overall, there were no significant trends towards differential abundance of viral families between the Charadriiformes and Anseriformes (Fig S19-S21). Viruses from the <em>Astroviridae, Calciviridae, Coronaviridae, Picobirnaviridae, Picornaviridae, Reoviridae</em> and <em>Rhabdoviridae</em> were found in both Anseriformes and Charadriiformes, albeit with different abundance patterns (Fig S19-S21). Multiple genera of picornaviruses were detected, but similarly without significant differences: most families and genera were detected in both avian orders (Fig S19-S21).</p><p id="p-36">Using a non-metric ordination method (NMDS) we found no clustering of viral family or viral genus by host order with respect to virus abundance and diversity between libraries (i.e. beta-diversity) (Fig S22). Similarly, there was no statistically significant clustering of viral families (R<sup>2</sup> = 0.13, df = 8, p = 0.267) or genera (R<sup>2</sup> = 0.157, df = 8, p = 0.173) in the Charadriiformes compared to Anseriformes, suggesting that despite taxonomic differences waterbirds share numerous viral genera and families. We utilized a co-phylogenetic approach to better determine whether this lack of clustering was associated with host phylogeny. Accordingly, a phylogram of beta-diversity was not congruent with host phylogeny at the viral family, genus or species levels (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Fig 7B</a>, Fig S23-S24). Hence, evolutionary relationships among hosts may not play a major role in structuring viromes. For example, closely related sister species (the <em>Anas</em> ducks Grey Teal and Pacific Black Ducks, or <em>Calidris</em> sandpipers Sharp-tailed Sandpipers, Red-necked Stint and Curlew Sandpipers) do not possess viromes that are more similar to each other than to those of more distantly related species within the same avian order. Rather, the phylogram of beta diversity has clusters with a mix of Anseriform and Charadriiform libraries, indicating connectivity of viral families and genera across species of both avian orders (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Fig 7B</a>, Fig S23-S24). Similarly, there was also no statistically significant clustering by feeding mechanism in the Anseriformes (R<sup>2</sup>=0.56, df=4, p=0.2) or migratory propensity in the Charadriiformes (R<sup>2</sup>=0.43, df=3, p=0.25) (Fig S22) suggesting that host ecology may also play a limited role in shaping virome composition at the levels of viral family and genus.</p><p id="p-37">Assessing comparative virome structure at the viral family and genera level is critical in demonstrating core viral families and genera in waterbirds. Species level analysis, albeit limited to viruses in which we were able to assemble &gt;1000bp, is a more accurate measure of connectivity and heterogeneity of avian viromes. First, while there was no marked division in virome composition at the level of viral family or genera between the Anseriformes and the Charadriiformes, there was such a distinction at the level of viral species. Specifically, no viral species were shared between the Anseriformes and Charadriiformes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3</a>), although the Anseriformes and Charadriiformes were sampled at different time points and at different locations. Within the Anseriformes, three viruses (influenza A virus, avian gammacoronavirus, and duck rotavirus G-like) were shared between three libraries: Grey Teal, Pink-eared Duck and Australian Shelduck (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3</a>, Fig S23). These shared viruses were especially common in the viromes of the Grey Teal (80% of avian viral reads), Australian Shelduck (82% of avian viral reads), and a small proportion of Pink-eared Duck avian viral reads (17%). The two <em>Anas</em> ducks (Grey Teal and Pacific Black Duck), the most closely related Anseriformes, did not share any viral species; surprisingly, the virome of the Pacific-Black Duck was different from the three connected host species. Further, Grey Teal and Pink-eared Ducks, the most taxonomically distinct waterfowl, shared two viral species, demonstrating the limited impact of host phylogeny (Fig S23). These viruses were also shared across ecological niches (dabbling ducks and filter feeding ducks), suggesting that co-occurrence was potentially responsible for their spread.</p><p id="p-38">Within the Anseriformes we tested for the effect of virus-virus interactions on alpha diversity, specifically whether the presence of IAV had an effect on virome abundance or composition. Despite a low abundance of IAV in the libraries, there was a trend towards a higher alpha diversity of viromes in Anseriform species positive for IAV (Grey Teal, Pink-eared Duck; Fig S25) at the viral family, genus and species levels, although this was not statistically significant due to small sample size. Furthermore, IAV positive libraries were adjacent to each on the NMDS plots, although this clustering was not statistically significant (Viral Family R<sup>2</sup>=0.14, df=8, p=0.307, Viral Genera, R<sup>2</sup>=0.14, df=8, p=0.224). This trend did not hold for avian gammacoronavirus in the Grey Teal and Australian Shelduck libraries (Fig S26).</p><p id="p-39">Finally, the phylogenetic analysis did not reveal a clear host-taxonomic gradient in viral species relationships. However, within the megriviruses (<em>Picornaviridae</em>), there appear to be large clades that may reflect avian order, with the viruses identified in the Anseriformes and Charadriiformes falling into two different clades. Furthermore, viruses from wild Anseriformes fall as sister taxa to previously described duck and goose megriviruses (Fig S6). In addition, this and our previous study (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a>) identified a number of picornaviruses from an unassigned genus only found in Charadriiformes, such that it might similarly represent a virus genus that is specific to a particular host order.</p></div></div><div class="section" id="sec-15"><h2 class="">Discussion</h2><p id="p-40">We identified 27 novel and previously described viral species from nine waterbirds falling into two avian orders. Anseriformes and Charadriiformes are important reservoirs for the best described avian virus, IAV, but are also central to the epidemiology of other multi-host viruses such as avian coronavirus and avian avulavirus type 1 (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>-<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). As such, these avian hosts are excellent model species for understanding the determinants of virome composition. Indeed, we detected all these previously described low pathogenic avian viruses in our sample set, and coronaviruses and influenza A viruses were shared across different Anseriform species. We also genomically described 24 novel viral species belonging to 10 viral families, including both RNA and DNA viruses. The largest number and diversity of viruses belonged to the <em>Picornaviridae</em>, although a number of rotaviruses and caliciviruses were also described.</p><p id="p-41">Overall, the avian viruses identified in this study were most closely related to other avian viruses, or in genera containing avian viruses. The exception is Grey Teal picorbirnavirus Y that occupies a clade dominated by viruses from human and porcine hosts. Whether this represents a host switch, or is due to lack of sampling in other hosts, will likely be revealed in additional metatranscriptomic studies. The Shelduck parvovirus described here is of particular interest as it is a member of the genus <em>Chapparvovirus</em>. Metagenomic analyses have recently identified members of this genus in a large number of vertebrates (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>), and are known agents of severe disease (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>).</p><p id="p-42">Beyond viral discovery, our study revealed no predictable clustering of viromes according to host taxonomy in either the Anseriformes and/or Charadriiformes. Given the data on IAV, we might expect see differences in virome structure due to a number of host factors (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>), including differences in biology/physiology. For example, different host species have different cell receptors which in turn results in different cell and tissue tropisms and patterns of viral attachment (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>). Further, following infection, different species have differences in long-term immune memory (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). However, we saw no clear distinction between the viromes of Anseriformes or Charadriiformes based on host taxonomy, suggesting these host factors are not central to virome structuring. For example, within the Charadriiformes, the closely related <em>Calidris</em> sandpipers (Scolopacidae) did not have similar viromes and did not cluster as a group independently from Red-capped Plover, a member of a different avian family (Charadridae). Alternatively, it is possible that aspects of host ecology, such as foraging ecology, may be more important in shaping virome composition than host taxonomy (a proxy for physiology). Specifically, differences in ecology may generate differences in virus exposure across closely related hosts (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>). The five Anseriform species studied here utilize three different feeding ecologies - dabbling, grazing and filter feeding - while the four Charadriiform species have different bill lengths and forage in different layers of sediment (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). Notably, however, there was no clear difference in virome composition as a result of ecology.</p><p id="p-43">Central to our study was considering virome structure in the context of a multi-host and multi-virus model of virus-host interactions. Accordingly, the data generated here revealed large-scale heterogeneity (abundance and alpha diversity) in virome composition at the levels of virus family, genus and even species. Despite this heterogeneity, there was also some connectivity between viromes at the levels of virus family, genus and even species: some viral families and genera were ubiquitous in almost all avian libraries, including members of the <em>Picornaviridae</em> and <em>Reoviridae</em>. More striking was the connectivity between three avian species (Grey Teal, Australian Shelduck, and Pink-eared Duck) at the level of viral species: these hosts shared Influenza A virus, gammacoronavirus, and Duck G-like rotavirus. As these Anseriformes were sampled in the same temporal and spatial frames, it is not unexpected that there was connectivity despite differences in ecology and taxonomy. However, at the level of viral species there was no connectivity between the Anseriform and Charadriiform libraries, although this may be due to the physiological differences noted above or, more simply, that the ducks and wader viruses were sampled at different times and places. Despite the lack of connectivity between the Anseriformes and Charadriiformes at a viral species level, avian avulavirus 1 and deltacoronavirus detected in Red-necked Stint and Sharp-tailed Sandpiper, respectively, have been previously described in Anseriformes (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). This connectivity between viromes is likely facilitated by the association of these birds to water. Specifically, viruses such as IAV are thought to be primarily transmitted by the faecal-oral route, in which viruses contaminate water through the faeces and birds ingest the viruses while feeding or preening (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>). Such water-borne transmission is critical to the dynamics of infection in bird communities (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). Furthermore, aquatic habitats seemingly support a higher risk of infection as compared to terrestrial habitats (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>). In support of this was the observation of lower viral diversity and abundance in the grazing Australian Wood duck which has a more terrestrial dietary strategy compared to the other Anseriform species.</p><p id="p-44">In sum, viral families and genera appeared to be readily shared among hosts, suggesting that waterbirds are key hosts for these families and genera. More importantly, our results indicate that avian viromes are largely comprised of what appear to be multi-host generalist viruses (here, influenza A virus, avian coronavirus, avian avulavirus type 1, duck rotavirus D-like) along with potential host-specific specialist viruses, which likely play a role in driving both heterogeneity and connectivity. While we found no evidence for viral species shared across avian orders, known multi-host virus species were revealed in both avian orders. Cases of clear host specificity were rare, but we speculate that Wood Duck Hepatitis B virus is likely host specific given high host specificity in this viral family (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>). In addition, the clade level structuring of Megriviruses (<em>Picornaviridae</em>), and previous report of an identical Megrivirus species found in the same avian species in very different locations (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">25</a>), similarly suggests some level of host specificity in this viral genus. Viral discovery efforts are imperative to better understanding factors that shape virome structure and the scope of host specificity in the avian reservoir. Importantly, we believe it is also imperative to consider multi-host, multi-virus systems in virus ecology.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-45">The sampling was supported by NIAID (HHSN266200700010C), ARC discovery grants (DP130101935 and DP160102146). The Melbourne WHO Collaborating Centre for Reference and Research on Influenza is supported by the Australian Department of Health. ECH is funded by an ARC Australian Laureate Fellowship (FL170100022). We thank the Victorian Wader Study Group the logistic support from Melbourne Water, and duck hunters who kindly sampled duck carcasses. We also thank Simeon Lisovski and Marta Ferenczi for their support</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ostfeld</span>  <span class="cit-name-given-names">RS</span></span>,<span class="cit-auth"><span class="cit-name-surname">Keesing</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Effects of Host Diversity on Infectious Disease</span>. <abbr class="cit-jnl-abbrev">Annu Rev Ecol Evol S</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">43</span>:<span class="cit-fpage">157</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BEcol%2BEvol%2BS%26rft.volume%253D43%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.2" data-doi="10.1111/j.1461-0248.2006.00885.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keesing</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostfeld</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-article-title">Effects of species diversity on disease risk</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">485</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DKeesing%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D485%26rft.epage%253D498%26rft.atitle%253DEffects%2Bof%2Bspecies%2Bdiversity%2Bon%2Bdisease%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00885.x%26rft_id%253Dinfo%253Apmid%252F16623733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00885.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16623733&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236384100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.3" data-doi="10.1073/pnas.1310557110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoverman</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">LaFonte</span>  <span class="cit-name-given-names">BE</span></span>. <span class="cit-article-title">Host and parasite diversity jointly control disease risk in complex communities</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">110</span>(<span class="cit-issue">42</span>):<span class="cit-fpage">16916</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310557110%26rft_id%253Dinfo%253Apmid%252F24082092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQyLzE2OTE2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMjkvNTI4MTc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.4" data-doi="10.1073/pnas.0233733100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">LoGiudice</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostfeld</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keesing</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">The ecology of infectious disease: Effects of host diversity and community composition on Lyme disease risk</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">100</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">567</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0233733100%26rft_id%253Dinfo%253Apmid%252F12525705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDAvMi81NjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8yOS81MjgxNzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milholland</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castro-Arellano</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzan</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Pena</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Global Diversity and Distribution of Hantaviruses and Their Hosts</span>. <abbr class="cit-jnl-abbrev">EcoHealth</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">15</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">163</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D15%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.6" data-doi="10.3201/eid0812.010317"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haydon</span>  <span class="cit-name-given-names">DT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cleaveland</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">LH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurenson</span>  <span class="cit-name-given-names">MK</span></span>. <span class="cit-article-title">Identifying reservoirs of infection: A conceptual and practical challenge</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-pub-date">2002</span>;<span class="cit-vol">8</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1468</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DHaydon%26rft.auinit1%253DD.%2BT.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D1468%26rft.epage%253D1473%26rft.atitle%253DIdentifying%2Breservoirs%2Bof%2Binfection%253A%2Ba%2Bconceptual%2Band%2Bpractical%2Bchallenge.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid0812.010317%26rft_id%253Dinfo%253Apmid%252F12498665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid0812.010317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12498665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179834300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.7" data-doi="10.1146/annurev.ecolsys.34.030102.151725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altizer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thrall</span>  <span class="cit-name-given-names">PH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antonovics</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Social organization and parasite risk in mammals: Integrating theory and empirical studies</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology Evolution and Systematics</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">34</span>:<span class="cit-fpage">517</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2BEvolution%2Band%2BSystematics%26rft.volume%253D34%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.34.030102.151725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.34.030102.151725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220102000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.8" data-doi="10.1111/ele.12122"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Streicker</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">AB</span></span>. <span class="cit-article-title">Differential sources of host species heterogeneity influence the transmission and control of multihost parasites</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">16</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">975</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D16%26rft.spage%253D975%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12122%26rft_id%253Dinfo%253Apmid%252F23714379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23714379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-article-title">Predicting virus emergence amid evolutionary noise. Open Biol</span>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">7</span>(<span class="cit-issue">10</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avril</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tolf</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schager</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borg</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Temporal dynamics, diversity, and interplay in three components of the viriodiversity of a Mallard population: Influenza A virus, avian paramyxovirus and avian coronavirus</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">29</span>:<span class="cit-fpage">129</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2BEvol%26rft.volume%253D29%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>  <span class="cit-name-given-names">JGB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verhagen</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waldenström</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Host and virus ecology as determinants of influenza A virus transmission in wild birds</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">28</span>:<span class="cit-fpage">26</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D28%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dann</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Foraging behaviour and diets of red-necked stints and curlew sandpipers in south-eastern Australia</span>. <abbr class="cit-jnl-abbrev">Wildlife Research</abbr>. <span class="cit-pub-date">2000</span>;<span class="cit-vol">27</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">61</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWildlife%2BResearch%26rft.volume%253D27%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.13" data-doi="10.1073/pnas.1401849111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wikramaratna</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">OG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Contact between bird species of different lifespans can promote the emergence of highly pathogenic avian influenza strains</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">111</span>(<span class="cit-issue">29</span>):<span class="cit-fpage">10767</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1401849111%26rft_id%253Dinfo%253Apmid%252F24958867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI5LzEwNzY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMjkvNTI4MTc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">HG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">ZS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ecological dynamics of influenza A viruses: cross-species transmission and global migration</span>. <abbr class="cit-jnl-abbrev">Sci Rep-Uk</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.15" data-doi="10.1126/science.1122438"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">VJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallensten</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waldenström</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">ADME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RAM.</span></span> <span class="cit-article-title">Global patterns of influenza A virus in wild birds</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">312</span>:<span class="cit-fpage">384</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOlsen%26rft.auinit1%253DB.%26rft.volume%253D312%26rft.issue%253D5772%26rft.spage%253D384%26rft.epage%253D388%26rft.atitle%253DGlobal%2BPatterns%2Bof%2BInfluenza%2BA%2BVirus%2Bin%2BWild%2BBirds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1122438%26rft_id%253Dinfo%253Apmid%252F16627734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3Mi8zODQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8yOS81MjgxNzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.16" data-doi="10.1371/journal.ppat.0030061"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">VJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baas</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lexmond</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waldenström</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallensten</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fransson</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Spatial, temporal, and species variation in prevalence of influenza A viruses in wild migratory birds</span>. <abbr class="cit-jnl-abbrev">PloS Pathog</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">e61</span>. doi:10.1371/journal.ppat.0030061.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DMunster%26rft.auinit1%253DV.%2BJ.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De61%26rft.epage%253De61%26rft.atitle%253DSpatial%252C%2Btemporal%252C%2Band%2Bspecies%2Bvariation%2Bin%2Bprevalence%2Bof%2Binfluenza%2BA%2Bviruses%2Bin%2Bwild%2Bmigratory%2Bbirds.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030061%26rft_id%253Dinfo%253Apmid%252F17500589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17500589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.17" data-doi="10.7589/0090-3558-48.2.322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maxted</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luttrell</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goekjian</span>  <span class="cit-name-given-names">VH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niles</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dey</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Avian influenza virus infection dynamics in shorebird hosts</span>. <abbr class="cit-jnl-abbrev">J Wildl Dis</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">48</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">322</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BWildlife%2BDiseases%26rft.stitle%253DJ%2BWildl%2BDis%26rft.aulast%253DMaxted%26rft.auinit1%253DA.%2BM.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D322%26rft.epage%253D334%26rft.atitle%253DAVIAN%2BINFLUENZA%2BVIRUS%2BINFECTION%2BDYNAMICS%2BIN%2BSHOREBIRD%2BHOSTS%26rft_id%253Dinfo%253Adoi%252F10.7589%252F0090-3558-48.2.322%26rft_id%253Dinfo%253Apmid%252F22493108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7589/0090-3558-48.2.322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22493108&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302589000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.18" data-doi="10.1177/104063870902100501"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bröjer</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ågren</span>  <span class="cit-name-given-names">EO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uhlhorn</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernodt</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mörner</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Pathology of natural highly pathogenic avian influenza H5N1 infection in wild tufted ducks (<em>Aythya fuligula</em>)</span>. <abbr class="cit-jnl-abbrev">J Vet Diagn Invest</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">21</span>:<span class="cit-fpage">579</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.stitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.aulast%253DBrojer%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D579%26rft.epage%253D587%26rft.atitle%253DPathology%2Bof%2BNatural%2BHighly%2BPathogenic%2BAvian%2BInfluenza%2BH5N1%2BInfection%2Bin%2BWild%2BTufted%2BDucks%2B%2528Aythya%2BFuligula%2529%26rft_id%253Dinfo%253Adoi%252F10.1177%252F104063870902100501%26rft_id%253Dinfo%253Apmid%252F19737752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/104063870902100501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19737752&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.19" data-doi="10.1128/JVI.01165-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pantin-Jackwood</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costa-Hurtado</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeJesus</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spackman</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Pathogenicity and Transmission of H5 and H7 Highly Pathogenic Avian Influenza Viruses in Mallards</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">90</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">9967</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01165-16%26rft_id%253Dinfo%253Apmid%252F27558429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8yMS85OTY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMjkvNTI4MTc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J-K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Negovetich</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">RG</span></span>. <span class="cit-article-title">Ducks: the “trojan horses” of H5N1 influenza</span>. <abbr class="cit-jnl-abbrev">Influenza and Other Respiratory Viruses</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">121</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfluenza%2Band%2BOther%2BRespiratory%2BViruses%26rft.volume%253D3%26rft.spage%253D121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.21" data-doi="10.1016/j.dci.2011.02.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magor</span>  <span class="cit-name-given-names">KE</span></span>. <span class="cit-article-title">Immunoglobulin genetics and antibody responses to influenza in ducks</span>. <abbr class="cit-jnl-abbrev">Dev Comp Immunol</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">35</span>:<span class="cit-fpage">1008</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DMagor%26rft.auinit1%253DK.%2BE.%26rft.volume%253D35%26rft.issue%253D9%26rft.spage%253D1008%26rft.epage%253D1016%26rft.atitle%253DImmunoglobulin%2Bgenetics%2Band%2Bantibody%2Bresponses%2Bto%2Binfluenza%2Bin%2Bducks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2011.02.011%26rft_id%253Dinfo%253Apmid%252F21377488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2011.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21377488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manvell</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulenburg</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shell</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wikramaratna</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perrins</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Antibody responses to avian influenza viruses in wild birds broaden with age</span>. <abbr class="cit-jnl-abbrev">Proceedings Biological sciences</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">283</span>(<span class="cit-issue">1845</span>):doi:10.1098/rspb.2016.159.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-thesis" id="cit-528174v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferenczi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-chapter-title">Avian influenza virus dynamics in Australian wild birds</span>. PhD Thesis: <span class="cit-publ-name">Deakin University</span>; <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferenczi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loyn</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Riley</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Avian influenza infection dynamics under variable climatic conditions, viral prevalence is rainfall driven in waterfowl from temperate, south-east Australia</span>. <abbr class="cit-jnl-abbrev">Vet Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">47</span>:<span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet%2BRes%26rft.volume%253D47%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaassen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurt</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-article-title">Virus-virus interactions and host ecology are associated with RNA virome structure in wild birds</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>. <span class="cit-pub-date">2018</span>: doi:10.1111/mec.14918.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.26" data-doi="10.1038/nature20167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">XD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">CX</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Redefining the invertebrate RNA virosphere</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">540</span>(<span class="cit-issue">7634</span>):<span class="cit-fpage">539</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D540%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20167%26rft_id%253Dinfo%253Apmid%252F27880757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27880757&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.27" data-doi="10.1038/s41586-018-0012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">XD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The evolutionary history of vertebrate RNA viruses</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">556</span>(<span class="cit-issue">7700</span>):<span class="cit-fpage">197</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0012-7%26rft_id%253Dinfo%253Apmid%252F29618816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29618816&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.28" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">357</span>–<span class="cit-lpage">U54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.29" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">30</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">772</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.30" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Capella-Gutierrez</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silla-Martinez</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">25</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">1972</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.31" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">59</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">307</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.32" data-doi="10.7717/peerj.2836"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lagkouvardos</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clavel</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Rhea: a transparent and modular R pipeline for microbial profiling based on 16S rRNA gene amplicons</span>. <abbr class="cit-jnl-abbrev">Peerj</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">5</span>:<span class="cit-fpage">e2836</span>. doi:10.7717/peerj.2836.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerj%26rft.volume%253D5%26rft.spage%253De2836%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.2836%26rft_id%253Dinfo%253Apmid%252F28097056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.2836&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28097056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oksanen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindt</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">MHH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oksanen</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The vegan package</span>. <abbr class="cit-jnl-abbrev">Community Ecology Package</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">10</span>:<span class="cit-fpage">631</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommunity%2BEcology%2BPackage%26rft.volume%253D10%26rft.spage%253D631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.34" data-doi="10.1371/journal.pone.0061217"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McMurdie</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">phyloseq: An R package for reproducible interactive analysis and graphics of microbiome census data</span>. <abbr class="cit-jnl-abbrev">PloS ONE</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e61217</span>. doi:10.1371/journal.pone.0061217.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BONE%26rft.volume%253D8%26rft.spage%253De61217%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061217%26rft_id%253Dinfo%253Apmid%252F23630581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23630581&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohl</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brucker</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Opstal</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordenstein</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-article-title">Phylosymbiosis: Relationships and Functional Effects of Microbial Communities across Host Evolutionary History</span>. <abbr class="cit-jnl-abbrev">PloS Biology</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">14</span>(<span class="cit-issue">11</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.36" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">LR</span></span>. <span class="cit-article-title">Comparison of Phylogenetic Trees</span>. <abbr class="cit-jnl-abbrev">Math Biosci</abbr>. <span class="cit-pub-date">1981</span>;<span class="cit-vol">53</span>(<span class="cit-issue">1-2</span>):<span class="cit-fpage">131</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath%2BBiosci%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.37" data-doi="10.1093/bioinformatics/btq706"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schliep</span>  <span class="cit-name-given-names">KP</span></span>. <span class="cit-article-title">phangorn: phylogenetic analysis in R</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">27</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">592</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq706%26rft_id%253Dinfo%253Apmid%252F21169378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21169378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287246000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.38" data-doi="10.1093/sysbio/34.1.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Penny</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hendy</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-article-title">The Use of Tree Comparison Metrics</span>. <abbr class="cit-jnl-abbrev">Syst Zool</abbr>. <span class="cit-pub-date">1985</span>;<span class="cit-vol">34</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">75</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BZool%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F34.1.75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/34.1.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duchene</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-article-title">Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families</span>. <abbr class="cit-jnl-abbrev">PloS Pathog</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">13</span>(<span class="cit-issue">2</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.40" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">APE: Analyses of Phylogenetics and Evolution in R language</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">20</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">289</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barth</span>  <span class="cit-name-given-names">JMI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matschiner</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">BC</span></span>. <span class="cit-article-title">Phylogenetic Position and Subspecies Divergence of the Endangered New Zealand Dotterel (Charadrius obscurus</span>). <abbr class="cit-jnl-abbrev">PloS One</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>(<span class="cit-issue">10</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paton</span>  <span class="cit-name-given-names">TA</span></span>. <span class="cit-article-title">Phylogenetic relationships and divergence times of Charadriiformes genera: multigene evidence for the Cretaceous origin of at least 14 clades of shorebirds</span>. <abbr class="cit-jnl-abbrev">Biol Lett-Uk</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">3</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">205</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BLett-Uk%26rft.volume%253D3%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Multiple gene sequences resolve phylogenetic relationships in the shorebird suborder Scolopaci (Aves: Charadriiformes</span>). <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">64</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">66</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPhylogenet%2BEvol%26rft.volume%253D64%26rft.spage%253D66%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">HW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid and recent diversification patterns in Anseriformes birds: Inferred from molecular phylogeny and diversification analyses</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">12</span>(<span class="cit-issue">9</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.45" data-doi="10.1111/j.2041-210X.2011.00169.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>  <span class="cit-name-given-names">LJ</span></span>. <span class="cit-article-title">phytools: an R package for phylogenetic comparative biology (and other things</span>). <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">3</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">217</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D3%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00169.x%26rft_id%253Dinfo%253Apmid%252F23653398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00169.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23653398&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">15</span>(<span class="cit-issue">12</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Latorre-Margalef</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tolf</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grosbois</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avril</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bengtsson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Long-term variation in influenza A virus prevalence and subtype diversity in a migratory Mallards in Northern Europe</span>. <abbr class="cit-jnl-abbrev">Proceedings Biological sciences</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">281</span>:doi:10.1098/rspb.2014.0098</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.48" data-doi="10.1371/journal.pone.0024010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knutsen</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berdeen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goekjian</span>  <span class="cit-name-given-names">VH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poulson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goyal</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Influenza A viruses in ducks in Northwestern Minnesota: fine scale spatial and temporal variation in prevalence and subtype diversity</span>. <abbr class="cit-jnl-abbrev">PloS ONE</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">e24010</span>. doi:10.1371/journal.pone.0024010.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWilcox%26rft.auinit1%253DB.%2BR.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24010%26rft.epage%253De24010%26rft.atitle%253DInfluenza-A%2Bviruses%2Bin%2Bducks%2Bin%2Bnorthwestern%2BMinnesota%253A%2Bfine%2Bscale%2Bspatial%2Band%2Btemporal%2Bvariation%2Bin%2Bprevalence%2Band%2Bsubtype%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024010%26rft_id%253Dinfo%253Apmid%252F21931636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21931636&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindqvist</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muradrasoli</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarhult</span>  <span class="cit-name-given-names">JD</span></span>. <span class="cit-article-title">Urbanization and the dynamics of RNA viruses in Mallards (<em>Anas platyrhynchos</em>)</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">51</span>:<span class="cit-fpage">89</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2BEvol%26rft.volume%253D51%26rft.spage%253D89%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muradrasoli</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarhult</span>  <span class="cit-name-given-names">JD</span></span>. <span class="cit-article-title">High prevalence and putative lineage maintenance of avian coronaviruses in Scandinavian waterfowl</span>. <abbr class="cit-jnl-abbrev">PloS ONE</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">11</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e0150198</span>. doi:10.1371/journal.pone.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BONE%26rft.volume%253D11%26rft.spage%253De0150198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chamings</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vibin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaassen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexandersen</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Detection and characterisation of coronaviruses in migratory and non-migratory Australian wild birds</span>. <abbr class="cit-jnl-abbrev">Sci Rep-Uk</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-528174v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoque</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheam</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skerratt</span>  <span class="cit-name-given-names">LF</span></span>. <span class="cit-article-title">Epidemiology of avian influenza in wild aquatic birds in a biosecurity hotspot, North Queensland, Australia</span>. <abbr class="cit-jnl-abbrev">Preventive veterinary medicine</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">118</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">169</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPreventive%2Bveterinary%2Bmedicine%26rft.volume%253D118%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoque</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karo-Karo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheam</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skerratt</span>  <span class="cit-name-given-names">LF</span></span>. <span class="cit-article-title">Monitoring of wild birds for Newcastle disease virus in north Queensland, Australia</span>. <abbr class="cit-jnl-abbrev">Preventive veterinary medicine</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">103</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">49</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPreventive%2Bveterinary%2Bmedicine%26rft.stitle%253DPrev%2BVet%2BMed%26rft.aulast%253DHoque%26rft.auinit1%253DM.%2BA.%26rft.volume%253D103%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D62%26rft.atitle%253DMonitoring%2Bof%2Bwild%2Bbirds%2Bfor%2BNewcastle%2Bdisease%2Bvirus%2Bin%2Bnorth%2BQueensland%252C%2BAustralia.%26rft_id%253Dinfo%253Apmid%252F21945812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21945812&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Souza</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romeiro</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Modha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Araujo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Queiroz</span>  <span class="cit-name-given-names">LH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Chapparvoviruses occur in at least three vertebrate classes and have a broad biogeographic distribution</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">98</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">225</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.55" data-doi="10.1016/j.cell.2018.08.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roediger</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tikoo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobbin</span>  <span class="cit-name-given-names">JCA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jormakka</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">An Atypical Parvovirus Drives Chronic Tubulointerstitial Nephropathy and Kidney Fibrosis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">175</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">530</span>–<span class="cit-lpage">43 e24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D175%26rft.spage%253D530%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.08.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.08.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jourdain</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Riel</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuiken</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waldenström</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The pattern of influenza virus attachment varies among wild bird species</span>. <abbr class="cit-jnl-abbrev">PloS ONE</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">6</span>(<span class="cit-issue">9</span>):doi:10.1371/journal.pone.0024155.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.57" data-doi="10.1111/j.1461-0248.2007.01011.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sivars-Becker</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tessier</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caceres</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-article-title">Eating yourself sick: transmission of disease as a function of foraging ecology</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">10</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">207</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DHall%26rft.auinit1%253DS.%2BR.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D218%26rft.atitle%253DEating%2Byourself%2Bsick%253A%2Btransmission%2Bof%2Bdisease%2Bas%2Ba%2Bfunction%2Bof%2Bforaging%2Becology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01011.x%26rft_id%253Dinfo%253Apmid%252F17305804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01011.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244227800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.58" data-doi="10.2307/3545084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">GA</span></span>. <span class="cit-article-title">Optimal Foraging Theory - a Possible Role for Parasites</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>. <span class="cit-pub-date">1991</span>;<span class="cit-vol">60</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">391</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D60%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3545084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3545084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FJ41200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.59" data-doi="10.1016/j.meegid.2009.04.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roche</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lebarbenchon</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gauthier-Clerc</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renaud</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Water-borne transmission drives avian influenza dynamics in wild birds: the case of the 2005-2006 epidemics in the Camargue area</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">9</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">800</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DRoche%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DWater-borne%2Btransmission%2Bdrives%2Bavian%2Binfluenza%2Bdynamics%2Bin%2Bwild%2Bbirds%253A%2Bthe%2Bcase%2Bof%2Bthe%2B2005-2006%2Bepidemics%2Bin%2Bthe%2BCamargue%2Barea.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2009.04.009%26rft_id%253Dinfo%253Apmid%252F19379841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2009.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19379841&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269271400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.60" data-doi="10.1098/rspb.2011.0958"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoye</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaassen</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Host behaviour and physiology underpin individual variation in avian influenza virus infection in migratory Bewick’s Swans</span>. <abbr class="cit-jnl-abbrev">Proceedings Biological sciences</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">279</span>(<span class="cit-issue">1728</span>):<span class="cit-fpage">529</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2BBiological%2Bsciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.0958%26rft_id%253Dinfo%253Apmid%252F21733894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.0958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21733894&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F29%2F528174.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-528174v1.61" data-doi="10.1128/JVI.79.5.2729-2742.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">WS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aldrich</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saputelli</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jilbert</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification and characterization of avihepadnaviruses isolated from exotic anseriformes maintained in captivity</span>.<abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">79</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">2729</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGuo%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D5%26rft.spage%253D2729%26rft.epage%253D2742%26rft.atitle%253DIdentification%2Band%2BCharacterization%2Bof%2BAvihepadnaviruses%2BIsolated%2Bfrom%2BExotic%2BAnseriformes%2BMaintained%2Bin%2BCaptivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.5.2729-2742.2005%26rft_id%253Dinfo%253Apmid%252F15708992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzUvMjcyOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzI5LzUyODE3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/528174v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F528174v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/533828v1" title="Shared mechanisms between coronary heart disease and depression: findings from a large UK general population-based cohort" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 29, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/528174v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/528174v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/528174v1" class="active" data-icon-position="" data-hide-link-title="0">Virome heterogeneity and connectivity in waterfowl and shorebird communities</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/156923" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XY4-29TMkngieIznKcqwWQRHDLsbkWgOHYbH0t9czPo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="156923" id="node156923--2" data-pisa="biorxiv;528174v1" data-pisa-master="biorxiv;528174" data-seqnum="156923" data-apath="/biorxiv/early/2019/01/29/528174.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Virome heterogeneity and connectivity in waterfowl and shorebird communities</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Wille</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mang</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Klaassen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Aeron C.</span> <span class="nlm-surname">Hurt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Edward C.</span> <span class="nlm-surname">Holmes</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">528174; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/528174 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/528174v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;title=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;title=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;text=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;title=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;t=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;title=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;title=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="156923" id="node156923--3" data-pisa="biorxiv;528174v1" data-pisa-master="biorxiv;528174" data-seqnum="156923" data-apath="/biorxiv/early/2019/01/29/528174.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Virome heterogeneity and connectivity in waterfowl and shorebird communities</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Wille</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mang</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Klaassen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Aeron C.</span> <span class="nlm-surname">Hurt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Edward C.</span> <span class="nlm-surname">Holmes</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">528174; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/528174 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Wille%2C+Michelle&amp;rft.au=Shi%2C+Mang&amp;rft.au=Klaassen%2C+Marcel&amp;rft.au=Hurt%2C+Aeron+C.&amp;rft.au=Holmes%2C+Edward+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/156923/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/156923/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/156923/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/156923/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/156923/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/156923/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/156923/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/156923/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/156923/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/156923/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/156923/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/156923/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;count=horizontal&amp;via=&amp;text=Virome%20heterogeneity%20and%20connectivity%20in%20waterfowl%20and%20shorebird%20communities&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/528174v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/528174v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/528174v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
