<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="publisher-id" title="publisher-id">978-0-85729-883-6</span><span class="doi" title="doi">10.1007/978-0-85729-883-6</span><span class="nlm-ta" title="nlm-ta">Human-in-the-Loop Simulations</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Human-in-the-Loop Simulations</span><div class="unknown" title="journal-subtitle" tag="journal-subtitle">
Methods and Practice</div> </div><span class="isbn" tagx="isbn" title="isbn">978-0-85729-882-9</span><span class="isbn" tagx="isbn" title="isbn">978-0-85729-883-6</span></div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7121606">7121606</a></span><span class="publisher-id" title="publisher-id">8</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/978-0-85729-883-6_8">10.1007/978-0-85729-883-6_8</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Analyzing Global Epidemiology of Diseases Using Human-in-the-Loop Bio-Simulations</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothrock</span><span class="given-names" tagx="given-names" title="given-names">Ling</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+1 814 865 7241</span><span class="fax" tagx="fax" title="fax">+1 814 863 4745</span><span class="email" tagx="email" title="email">lxr28@engr.psu.edu</span></span><a href="#AffID1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narayanan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+1 937 775 5007</span><span class="fax" tagx="fax" title="fax">+1 937 775 5009</span><span class="email" tagx="email" title="email">s.narayanan@wright.edu</span></span><a href="#AffID2">2</a></span><span class="citation_author_institution" id="AffID1">[ID1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.29857.31</span><span class="institution-id" title="institution-id">0000000120974281</span><span class="institution" title="institution">, The Harold and Inge Marcus Department of, </span><span class="institution" title="institution">The Pennsylvania State University, </span></div></span><span class="citation_author_institution" id="AffID2">[ID2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.268333.f</span><span class="institution-id" title="institution-id">0000000419367937</span><span class="institution" title="institution">, 405 Russ Engineering Center, </span><span class="institution" title="institution">Wright State University, </span></div></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Dhananjai M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">raodm@muohio.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chernyakhovsky</span><span class="given-names" tagx="given-names" title="given-names">Alexander</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">achernya@mit.edu</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Victoria</span></span><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.259956.4</span><span class="institution-id" title="institution-id">0000000121956763</span><span class="institution" title="institution">CSE Department, </span><span class="institution" title="institution">Miami University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.116068.8</span><span class="institution-id" title="institution-id">0000000123412786</span><span class="institution" title="institution">Massachusetts Institute of Technology, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2011-9-9</span></span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">174</span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Humanity is facing an increasing number of highly virulent and communicable diseases such as influenza. Combating such global diseases requires in-depth knowledge of their epidemiology. The only practical method for discovering global epidemiological knowledge and identifying prophylactic strategies is simulation. However, several interrelated factors, including increasing model complexity, stochastic nature of diseases, and short analysis timeframes render exhaustive analysis an infeasible task. An effective approach to alleviate the aforementioned issues and enable efficient epidemiological analysis is to manually steer bio-simulations to scenarios of interest. Selective steering preserves causality, inter-dependencies, and stochastic characteristics in the model better than â€œseedingâ€�, i.e., manually setting simulation state. Accordingly, we have developed a novel Eco-modeling and bio-simulation environment called SEARUMS. The bio-simulation infrastructure of SEARUMS permits a human-in-the-loop to steer the simulation to scenarios of interest so that epidemics can be effectively modeled and analyzed. This article discusses mathematical principles underlying SEARUMS along with its software architecture and design. In addition, the article also presents the bio-simulations and multi-faceted case studies conducted using SEARUMS to elucidate its ability to forecast timelines, epicenters, and socio-economic impacts of epidemics. Currently, the primary emphasis of SEARUMS is to ease global epidemiological analysis of avian influenza. However, the methodology is sufficiently generic and it can be adapted for other epidemiological analysis required to effectively combat various diseases.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Markov Process</div> <div class="kwd" title="kwd">
Avian Influenza</div> <div class="kwd" title="kwd">
Severe Acute Respiratory Syndrome</div> <div class="kwd" title="kwd">
Epidemiological Analysis</div> <div class="kwd" title="kwd">
Antiviral Prophylaxis</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© Springer-Verlag London Limited 2011" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Communicable and vector-borne diseases, primarily induced by viral and bacterial infections, are the most common forms of ailments amongst humans and domesticated farm animals (Flint et al. <a href="#CR8">2004</a>). Despite numerous advances in epidemiology and medicine many of these diseases, particularly zoonotic diseases, continue to defy human efforts to treat and control them (Flint et al. <a href="#CR8">2004</a>). The source of such resistance to treatment and control is twofold. First, the incessant evolutionary processes such as genetic reassortment, recombination, and mutation induce changes to organisms, thereby constantly morphing their antigenic characteristics (Russell et al. <a href="#CR29">2008</a>). Second, the vectors of the diseases, such as birds and animals, often disperse the pathogens over wide areas (Hagemeijer and Mundkur <a href="#CR14">2006</a>; Normile <a href="#CR23">2006</a>). The scenario is further convoluted due to complicated symbiotic processes between the vectors and other organisms including humans. Rapid changes in phylogeny and bio-diversity pose significant challenges in vaccine manufacturing (WHO <a href="#CR39">2007a</a>). In addition, socio-political restrictions and technological limitations further magnify the challenges involved in controlling intercontinental spread of infectious diseases (WHO <a href="#CR39">2007a</a>).</p> <p>The aforementioned epidemiological, antigenic, and socio-political traits of communicable diseases are salient characteristics of several recent global epidemics, such as: severe acute respiratory syndrome (SARS) (Hufnagel et al. <a href="#CR17">2004</a>), avian influenza (H5N1) outbreaks (WHO <a href="#CR37">2006</a>), and swine flu (H1N1) outbreak (Garten et al. <a href="#CR9">2009</a>; Smith et al. <a href="#CR31">2009</a>). Such emergent viral epidemics cause significant morbidity and mortality, both in humans and livestock (CDC <a href="#CR4">2006</a>; WHO <a href="#CR35">2005</a>). For example, human influenza-A viruses causes more than 500,000 moralities in humans annually (Russell et al. <a href="#CR29">2008</a>). Its highly pathogenic avian homologe, namely H5N1, has caused economic devastation, particularly to poultry farming, in excess of 10 billion dollars (Kilpatrick et al. <a href="#CR18">2006</a>).</p> <p>The primary approach for combating emergent diseases, particularly viral diseases, is proactive, targeted antiviral prophylaxis (CDC <a href="#CR4">2006</a>; WHO <a href="#CR40">2007b</a>). Unfortunately, the constantly changing antigenic characteristics of viruses, particularly all influenza viruses, reduce efficacy of vaccinations (WHO <a href="#CR38">2006c</a>). Moreover a myriad of technological issues pose serious hurdles to manufacturing and distribution of even small volumes of H5N1 vaccines (WHO <a href="#CR38">2006c</a>). Furthermore, intercontinental migration of birds and international human travels further exacerbate effective containment and mitigation of epidemics.</p> <div class="epidemiologicalsimulations:abriefoverview" title="sec"> <div class="title" tagx="title" title="title">
Epidemiological Simulations: A Brief Overview</div> <p>The technological, macro-economic, and socio-political issues surrounding emergent, viral and communicable diseases (discussed in <a href="#Sec1">Sect. 8.1</a>) have led to a rapidly growing emphasis on analyzing the global epidemiology of diseases. Epidemiology broadly covers the study of various facets of communicable diseases in a selected population, including: transmission characteristics of diseases, epidemic and endemic states, environmental factors impacting disease transmission, and impacts of containment and prophylactic interventions (Anderson and May <a href="#CR1">1992</a>). Epidemiology is a multidisciplinary method involving biology, biostatistics, geographic information science, social science, and computer science (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). It has steadily evolved from its inception in early nineteenth century and is now a mainstream methodology that forms the corner stone for public health measures and preventive medicine (Anderson and May <a href="#CR1">1992</a>). Epidemiological analysis enables more accurate forecasting of epidemics and aids improving efficacy of geographically targeted antiviral prophylaxis (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>). Nevertheless, epidemiology is an active area of research as it continues to evolve in conjunction with breakthroughs in other disciples, such as: microbiology, genetics, proteomics, and meta-genomics (Rao et al. <a href="#CR26">2007a</a>).</p> <p>A fundamental and widely used framework underlying epidemiological analysis is compartmentalized model of communicable diseases (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). In a compartmentalized model, the sub-population that are in the same epidemic state are partitioned into non-intersecting sets called compartments (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). The temporal progress of an epidemic is modeled by transitioning a suitable fraction of the population from one compartment to another (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). The transition functions are typically modeled as ordinary differential equations (ODEs) with time as the independent variable. Constants in the ODEs are determined based on characteristics of the disease being analyzed. A more detailed description of the compartmentalized models is presented in <a href="#Sec7">Sect. 8.2.1</a>.</p> <p>Analysis of compartmentalized epidemiological models is performed by simultaneously solving the ODEs at different time steps to obtain characteristics of the epidemic being studied (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). One of the most powerful and widely used approaches for such an analysis is computer-based simulation. Simulations have gained significant importance epidemiology because they are the only practical approach to analyze large and complex epidemiological models (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>; Rao and Chernyakhovsky <a href="#CR25">2008</a>). Furthermore, the need to conduct a variety of multi-faceted analysis within short time frames necessitates the use of computer-based simulations. Moreover, simulation is a cost effective and non-destructive methodology from which results can be easily displayed in an intuitive form. More importantly, it enables effective explorations of policies and procedures associated with complicated control measures, such as: targeted layered containment [the main concept behind US governmentâ€™s containment strategies (Halloran et al. <a href="#CR15">2008</a>)], quarantine, social distancing, school closing, and targeted antiviral prophylaxis (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Longini et al. <a href="#CR22">2005</a>; Rao et al. <a href="#CR26">2007a</a>).</p> </div> <div class="motivationforhuman-in-the-loopsimulation" title="sec"> <div class="title" tagx="title" title="title">
Motivation for Human-in-the-Loop Simulation</div> <p>Currently, epidemiological models are used by many international agencies, including the World Health Organization (WHO) and the Centers for Disease Control (CDC), for large-scale, multi-faceted analysis required to propose and validate multi-national targeted layered containment policies as well as prophylactic measures (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Longini et al. <a href="#CR22">2005</a>; Rao et al. <a href="#CR26">2007a</a>). Such epidemiological analysis is performed using detailed, stochastic models that involve complex, symbiotic interactions between the various entities involved in the simulation.</p> <p>Typically, only a selected subset of scenarios is analyzed to evaluate effectiveness of a candidate set of containment strategies (Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>). Narrowing the subset of scenarios to be analyzed is critical because hundreds of thousands of simulation runs are needed even to analyze a specific subset of scenarios. The numerous runs are necessary to provide sufficiently accurate and statistically significant results using realistic, stochastic models (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Longini et al. <a href="#CR22">2005</a>).</p> <p>One of the important prerequisites of simulating selected scenarios is to have the model in a state that accurately reflects the conditions associated with the scenario. One strategy is to seed the simulation, i.e., manually initialize the state to appropriately model the scenario being analyzed. However, such an approach is typically tedious, cumbersome, and error prone. Moreover, in stochastic models, accurately capturing temporal, causal relationships (Lamport <a href="#CR19">1978</a>) between entities in the model can prove to be a greater challenge. These issues often lead to conspicuous inefficiencies when multiple scenarios have to be analyzed. An alternative approach is to commence simulations in a given, verified initial state and then steer the simulations to desired scenarios of interest. Strategies for steering simulations can be broadly classified into three main categories, namely: automatic, semi-automatic, and manual steering. Note that this taxonomy is based on the degree of human involvement in the steering process.</p> <p>The complex, stochastic nature of global epidemiological models necessitates the use of manual steering. Manual steering of a simulation requires a human-in-the-loop (HITL) to intermittently modify the state of the simulation; thereby altering its trajectory to the desired scenario. Such HITL epidemiological bio-simulations enable efficient generation of various scenarios to be analyzed. In addition, this approach preserves the causal inter-dependencies in a stochastic model. Moreover, simulating from a validated initial state minimizes modeling errors and reduces continued verification efforts. Last but not the least, HITL steering enables some degree of adaptation and modeling of unplanned or unanticipated scenarios that could occur in emergent epidemics. Consequently, HITL-simulations hold significant potential to play a vital role in epidemiology and in establishing pertinent public policies.</p> </div> <div class="searums:ahuman-in-the-loopepidemiologicalsimulator" title="sec"> <div class="title" tagx="title" title="title">
SEARUMS: A Human-in-the-Loop Epidemiological Simulator</div> <p>Realizing the advantages of simulation-based epidemiological analysis requires the use of an effective software environment for Eco-modeling,HITL bio-simulation, and analysis (Rao et al. <a href="#CR26">2007a</a>). In addition, the software environment must facilitate rapid simulation to minimize analysis time frames. We also envision the software must be portable and accessible to enable its widespread use (Rao et al. <a href="#CR26">2007a</a>). Accordingly, we have endeavored to design and develop an Eco-modeling and bio-simulation software environment called SEARUMS.</p> <p>Currently, SEARUMS is geared for analyzing the global epidemiology of avian influenza, with migrating waterfowl as the primary vectors for intercontinental spread of the disease. However, the methodology is sufficiently generic and can be easily adapted for other diseases. Accordingly, this chapter elucidates the modeling, HITL-simulation, and software design principles underlying SEARUMS so that the concepts can be readily adapted and applied to other diseases as well. Furthermore, SEARUMS is envisioned to serve as a global, multi-disciplinary environment that seamlessly integrates knowledge from various fields so that epidemiologists, economists, and disease control centers can collaboratively use it to combat global epidemics.</p> </div> <div class="sectionorganizationandaudience" title="sec"> <div class="title" tagx="title" title="title">
Section Organization and Audience</div> <p>The principles underlying the Eco-modeling and HITL bio-simulation infrastructure of SEARUMS span a broad range of disciplines, including: mathematical modeling, statistical analysis, computer science, and software engineering. Furthermore, the multi-disciplinary nature of SEARUMS garners interest from a broad range of audience, including: epidemiologists, economists, software architects, and disease control centers. Consequently, the various sections in this chapter are organized to emphasize a specific aspect of SEARUMS. The objective of this organization is to enable readers from a particular discipline to focus on pertinent topics. Accordingly, common concepts that are pervasive across the various sections are discussed as the background information in <a href="#Sec6">Sect. 8.2</a>. This section also reviews some of the pertinent related research investigations. <a href="#Sec11">Section 8.3</a> discusses the software architecture and design principles underlying SEARUMS. The procedure used for modeling, incorporation of real-world statistical data, and analysis using SEARUMS is presented in <a href="#Sec13">Sect. 8.4</a>. Results from some of the analysis conducted using SEARUMS are summarized in <a href="#Sec15">Sect. 8.5</a>. <a href="#Sec16">SectionÂ 8.6</a> concludes the chapter while emphasizing the utility and broader applicability of the HITL-simulations infrastructure of SEARUMS to other diseases.</p> </div> </div> <div class="backgroundandrelatedresearch" title="sec"> <div class="title" tagx="title" title="title">
Background and Related Research</div> <p>Simulation-based analysis of global epidemiology of diseases is a multi-disciplinary task that involves ecology, mathematics, statistics, computer science, and software engineering (Anderson and May <a href="#CR1">1992</a>; Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>; Rao et al. <a href="#CR28">2009</a>). Accordingly, this section covers relevant prerequisite information and terminology on the aforementioned topics along with brief surveys of closely related research investigations. Specifically, <a href="#Sec7">Sect. 8.2.1</a> covers the compartmental models used in epidemiology. Although the compartmental models can be applied to many diseases, in this article we focus on its use for analyzing global epidemiology of avian influenza. Consequently, the ecology of avian influenza is summarized in <a href="#Sec8">Sect. 8.2.2</a> to further motivate the Eco-modeling and bio-simulation environment constituting SEARUMS. Next <a href="#Sec9">Sect. 8.2.3</a> presents an overview of the Markov processes that are used to model the natural interactions occurring in the ecology. This section also illustrates the interactions between the compartmental models and Markov processes. <a href="#Sec10">Section 8.2.4</a> presents a brief survey of the concepts and software frameworks pertaining to HITL steering of simulators and software systems.</p> <div class="compartmentalmodelsinepidemiology" title="sec"> <div class="title" tagx="title" title="title">
Compartmental Models in Epidemiology</div> <p>The most widely used mathematical framework for epidemiological analysis are compartmentalized models (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). In a compartmentalized model the population being analyzed is partitioned into a few non-intersecting subsets called compartments. Compartments are defined such that the sub-population within a compartment exhibits a vital disease characteristic (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>), such as: </p><ul> <li> <p><i>Susceptible</i>: population lacks immunity</p> </li> <li> <p><i>Exposed</i>: infected, but not yet infectious sub-population</p> </li> <li> <p><i>Infected</i>: sub-population is actively spreading the disease, and</p> </li> <li> <p><i>Recovered</i>: sub-population acquired immunity or died due to disease.</p> </li> </ul> <p /> <p>Additional compartments are used to appropriately model other epidemiological states of diseases depending on the analysis needs (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>; Rao et al. <a href="#CR26">2007a</a>).</p> <p>The classical susceptible-exposed-infected-removed (SEIR) compartmentalized mathematical model shown in Fig.Â <a href="#Fig1">8.1</a> is used to model the epidemiology of various diseases (Anderson and May <a href="#CR1">1992</a>). The characteristics of the SEIR model are represented using the following system of differential equations: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{dS}{dt} = \mu N - \left[\lambda + \mu \right]S(t) $$\end{document}</div> </div> <div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M2">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{dE}{dt} = \lambda S(t) - (\beta + \mu)E(t) $$\end{document}</div> </div> <div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{dI}{dt} = \beta E(t) - (v + \mu)I(t) $$\end{document}</div> </div> <div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M4">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{dR}{dt} = vI(t) - \mu R(t) $$\end{document}</div> </div> <div class="graphic" position="anchor" /> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.1</span></a></div>   <p>A classical SEIR epidemiology model illustrating the four typical compartments used to describe the progression of a disease through a population (of <i>N</i> individuals)</p>   </div> <p /> <p>where, <i>S(t)</i>, <i>E(t)</i>, <i>I(t)</i>, and <i>R(t)</i> represent the number of susceptible, exposed, infected, and removed hosts at any given instant of time <i>t</i>. The temporal progress of an epidemic is modeled by transitioning a fraction of the population from one compartment to another (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>). The transition functions are typically modeled as ODEs with time as the independent variable.</p> <p>Constants in the ODEs are determined based on characteristics of the disease being analyzed. Specifically, the parameters Î¼, Î», Î², and <i>Î½ </i>are: </p><ol> <li> <p>Î¼: the per capita host birth/death rate,</p> </li> <li> <p>Î»: the force of infection</p> </li> <li> <p>Î²: latency period</p> </li> <li> <p><i>v</i>: per capita recovery rate</p> </li> </ol> <p /> <p>Typically, the ODEs involve stochastic components to account for uncertainties introduced by various environmental or external factors (Anderson and May <a href="#CR1">1992</a>). An important and distinguishing property of the compartmentalized models and the transition functions is that the total population (<i>N</i> in Fig.Â <a href="#Fig1">8.1</a>) being modeled is held a constant (Anderson and May <a href="#CR1">1992</a>; Daley and Gani <a href="#CR5">2001</a>).</p> <p>Adaptations of the aforementioned classical SEIR model have been widely used to model the epidemiology of influenza (Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Halloran et al. <a href="#CR15">2008</a>; Longini et al. <a href="#CR22">2005</a>; Rao et al. <a href="#CR26">2007a</a>). Investigations conducted by Longini et al. (<a href="#CR22">2005</a>) and Ferguson et al. (<a href="#CR7">2006</a>) focus on analyzing pandemic mode of H5N1 in Thailand. In pandemic mode rapid and sustained human-to-human transmission is assumed. Since human-to-human transmission is assumed, these investigations use a highly detailed spatially explicit model based on SEIR concepts. The modeling approach used by Halloran et al. (<a href="#CR15">2008</a>) is similar to those proposed by Ferguson et al. (<a href="#CR7">2006</a>) and Longini et al. (<a href="#CR22">2005</a>). Moreover, these three investigations are based on the premise that H5N1 has already mutated to a pandemic form and epidemics are being caused primarily due to human-to-human transmission.</p> <p>In contrast, our investigations assume and reflect the current, real-world situation; i.e., H5N1 is yet to mutate into its pandemic state and human-to-human transmission is unsustained. Furthermore, our research aims to utilize waterfowl migration data to forecast zoonotic epicenters and timelines of potential epidemics. In addition, we also emphasize the role of poultry (extensible to swine) as intermediate hosts. These aspects notably distinguish our efforts from the aforementioned recent related investigations.</p> </div> <div class="avianinfluenza" title="sec"> <div class="title" tagx="title" title="title">
Avian Influenza</div> <p>Avian influenza is a viral disease caused by H5N1, a highly virulent strain of the influenza-A virus, that has the potential to cause a global pandemic (CDC <a href="#CR4">2006</a>; WHO <a href="#CR38">2006c</a>). The ecological interactions contributing to the transcontinental spread of the disease is illustrated in Fig.Â <a href="#Fig2">8.2</a>. As shown in the figure, infected migrating waterfowl, in which the virus is endemic, are the primary vectors for causing intercontinental spread of the disease (Normile <a href="#CR23">2006</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.2</span></a></div>   <p>The figure illustrates the salient pathways for transmission of H5N1 to various bird species and humans. The <i>arrows</i> in the diagram show the various directions of infection that occur in the ecology of avian influenza</p>   </div> <p /> <p>The virus rapidly spreads from waterfowl to poultry and humans through contaminated water, feed, feces, and surfaces. Once infected, the disease has a devastating impact on poultry farms causing 100% mortality within 48Â h (WHO <a href="#CR40">2007b</a>). In humans, the virus causes disease with a high mortality rate of nearly 60% (CDC <a href="#CR4">2006</a>; Normile <a href="#CR23">2006</a>). Furthermore, it is known to induce primary viral pneumonia in the host (Normile <a href="#CR23">2006</a>). Researchers believe that avian influenza has significant potential to become one of the deadliest pandemics in human history (CDC <a href="#CR4">2006</a>; WHO <a href="#CR38">2006c</a>). This inference has been drawn based on statistics from recent epizootic outbreaks and the highly pathogenic characteristics of H5N1. Moreover, manufacturing and distribution of vaccinations is facing multi-faceted challenges (WHO <a href="#CR39">2007a</a>). The aforementioned issues make it imperative the epidemiology of avian influenza is thoroughly analyzed in order to empower various national and international organizations with the knowledge to strategically combat the disease (Anderson and May <a href="#CR1">1992</a>; Epstein <a href="#CR6">2009</a>; Ferguson et al. <a href="#CR7">2006</a>; Longini et al. <a href="#CR22">2005</a>; GLiPHA <a href="#CR11">2007</a>; Rao et al. <a href="#CR27">2009</a>).</p> </div> <div class="markovprocesses" title="sec"> <div class="title" tagx="title" title="title">
Markov Processes</div> <p>The compartmental epidemiological models focus purely on the temporal progression of the disease in a given entity, may it be waterfowl, human, or poultry. However, the compartmental SEIR models (see <a href="#Sec7">Sect. 8.2.1</a>) do not embody the complete ecology, such as seasonal migration, occurring in nature. Consequently, such ecological processes need to be suitably modeled in conjunction with the epidemiological process. Accordingly, we envision using the concept of a Markov processes to model the overall ecological processes. Furthermore, our Eco-modeling approach suitably incorporates the SEIR models to provide a complete, holistic representation of the real-world ecology.</p> <p>A Markov process is a mathematical formalism used to describe changes occurring to the state of a stochastic system in discrete time steps (Solow and Smith <a href="#CR32">2006</a>; Winston <a href="#CR41">1994</a>). A Markov process consists of a number of states (or values) through which the system may transition at any given time. Mathematically, a Markov process is defined as a sequence of time dependent random variables <i>X</i><sub>0</sub>,<i> X</i><sub>1</sub>,<i> X</i><sub>2</sub>, ..., where <i>X</i><sub><i>t</i></sub> is a random variable that describes the state of the process at discrete time t. The initial or starting state of the system is typically represented by <i>X</i><sub>0</sub>. Transitions from one state to another are governed by the following three laws: (1) a Markov process may be in only one given state at any instant of time; (2) transition from one state to another occurs instantaneously in discrete time steps; and (3) the next state to which the process transitions is purely determined by the current state of the system and not its past. In other words, the past, present, and future states of a Markov process are independent of each other (Winston <a href="#CR41">1994</a>).</p> <p>In our approach, the SEIR operations are repeatedly preformed in an appropriate state in the Markov process. It must be noted that the SEIR models involving ODEs are inherently based on the notion of continuous simulation-time (Anderson and May <a href="#CR1">1992</a>). However, the Markov processes operate in discrete simulation-time steps (Winston <a href="#CR41">1994</a>). Consequently, our Markov process-based approach approximates the SEIR model. However, the discrete simulation-time steps are chosen to be sufficiently small to provide adequate accuracy without degrading performance (Rao et al. <a href="#CR28">2009</a>).</p> <p>Another important aspect of our Eco-modeling methodology is the aggregate representation of sub-populations. At a global scale, the varying populations of entities are typically modeled at a coarser granularity or aggregate entities (Booth <a href="#CR3">1997</a>; GROMS <a href="#CR12">2006</a>; Law et al. <a href="#CR21">2005</a>; Rao et al. <a href="#CR28">2009</a>). For example, certain species of waterfowl that live and migrate as a large flock in nature are typically modeled as a single entity. A large collection of birds, such as a poultry farms are represented as a single entity. In an analogous manner, humans living in geographic proximity to each other are modeled as a single entity.</p> <p>The motivation for such a coarser grained, aggregate representation is two fold (Booth <a href="#CR3">1997</a>; GROMS <a href="#CR12">2006</a>; Law et al. <a href="#CR21">2005</a>; Rao et al. <a href="#CR28">2009</a>). First, such an approach is necessary to reduce the size and complexity of the model to more tractable scales. Aggregated models continue to provide the necessary fidelity as long as the aggregation is performed at a sufficiently fine granularity (Law et al. <a href="#CR21">2005</a>). Second, various real-world statistical data on population dynamics, seasonal migratory behaviors, and disease progresses are determined through random sampling of a large subset of the population. Such data is more meaningful and relatively straightforward to apply at an aggregate level.</p> <p>In our software system the three primary aggregate entities, namely: (1) flock of waterfowl; (2) poultry; and (3) group of humans have been modeled using the aforementioned Eco-modeling approach. The Markov processes have been suitably implemented using the modeling framework provided by SEARUMS. The Markov process for the waterfowl entity is shown in Fig.Â <a href="#Fig3">8.3</a>. Readers are referred to the literature for more details on the Markov processes for various entities and the associated mathematical equations (Rao et al. <a href="#CR28">2009</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.3</span></a></div>   <p>Overview of Markov process for a waterfowl flock illustrating the states through which the process transitions to model the behavioral life cycle of waterfowl. The state of the process <i>S</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> at discrete time t is represented by the four-tuple <i>S</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup>Â =Â <div class="â_UNKNOWN"><i>X</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup>, <i>Y</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup>, <i>R</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup>, <i>I</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup>Â &amp;gt;, where <i>X</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> is the longitude, <i>Y</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> is the latitude, <i>R</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> is the instantaneous radius of the flock, <i>I</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> is the current measure of infection in the flock. The set <i>N</i><sub arrange="stack"><i>t</i></sub><sup arrange="stack">mp</sup> represents the neighborhood at time <i>t</i>. Readers are referred to the literature (Rao et al. <a href="#CR28">2009</a>) for details on the mathematical notations, spherical geometric equations, and state transitions</div></p>   </div> <p /> </div> <div class="human-in-the-loopsimulationandsteering" title="sec"> <div class="title" tagx="title" title="title">
Human-in-the-Loop Simulation and Steering</div> <p>In conjunction with advancements in microprocessor technologies, computer-based simulations have become an important, indispensable, and multi-disciplinary methodology for study and analysis of complex systems (Railsback et al. <a href="#CR24">2006</a>). Today, simulations are widely used in fundamental sciences, applied sciences, engineering, medicine, economics, and in the military (Tobias and Hofmann <a href="#CR33">2004</a>). During its inception, simulations were run purely as an offline or batch processing task without requiring any interaction with a human operator. However, the growing demand and diversification in needs in various disciplines have led to the development of simulations which involve interactions with a human operator (Rao et al. <a href="#CR27">2007b</a>). Such simulations, which permit or require a human to interact, are classified as HITL-simulations. The process of impacting the state of the simulation to divert the simulation to another state is called simulation steering. Simulation steering is similar to program steering, which is a more generic concept (Gu et al. <a href="#CR13">1994</a>).</p> <p>HITL-simulations can be further classified into constructive HITL-simulations and virtual simulations. In a virtual simulation, simulation-time progressed as the same rate as the real, wall-clock time (Rao et al. <a href="#CR27">2007b</a>). Furthermore, interaction between the human and the simulation occurs in a synchronous manner. Virtual simulations are primarily used for training, gaming, and education (Rao et al. <a href="#CR27">2007b</a>). On the other hand, in a constructive simulation, simulation-time progresses at a different rate than real time. Human interactions with a constructive simulation are typically intermittent or on a â€œas neededâ€� basis. The interactions are triggered based on an user gesture or when predefined events occur in the simulation.</p> <p>Constructive simulations are used for simulating large and complex systems such as the ecological and epidemiological models. Several such general purpose simulators have been described in the literature (Gilbert and Bankes <a href="#CR10">2002</a>; Railsback et al. <a href="#CR24">2006</a>; WHO <a href="#CR35">2005</a>). Some of the commonly used simulators include NetLogo, SWARM, SWARM-Java, Repast, and MASON. Railsback et al. (<a href="#CR24">2006</a>) highly recommended NetLogo for its ease-of-use. However, it uses a custom language for modeling and its source code is proprietary. On the other hand, the latter four platforms uses traditional programming languages and source codes are freely available (Railsback et al. <a href="#CR24">2006</a>). These four platforms essentially provide a core framework for model development and a collection of library modules. The library modules are built using the core framework and can be readily reused for modeling. Moreover, the simulators provide some support for HITL interactions.</p> <p>The aforementioned simulators are mostly general purpose simulators and are not specifically designed for epidemiology. However, our objective is to minimize learning curves for both developers and users, maximize portability, include intuitive interfaces for modeling, and seamlessly incorporate epidemiological analysis tools. Furthermore, the simulators had some disadvantages (Rao et al. <a href="#CR28">2009</a>). Consequently, we endeavored to develop SEARUMS, a custom Eco-modeling and HITL bio-simulation environment.</p> </div> </div> <div class="searums" title="sec"> <div class="title" tagx="title" title="title">
SEARUMS</div> <p>SEARUMS is an Eco-modeling and HITL bio-simulation environment. Currently, it is optimized to enable study and analysis of global epidemiology of avian influenza. However, the design of its modeling and simulation framework is sufficiently generic. Therefore, it can be adapted for epidemiological analysis of other diseases. SEARUMS has been developed in Java by utilizing many of the languageâ€™s object oriented programming features (Bloch <a href="#CR2">2001</a>). SEARUMS is designed to be a user friendly, integrated, graphical modeling, simulation, visualization, and analysis environment for conducting epidemiological analysis. These design goals have been achieved by composing the system using a collection of interdependent but loosely coupled modules as shown in Fig.Â <a href="#Fig4">8.4</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.4</span></a></div>   <p>Architectural overview of SEARUMS. GUI modules are highlighted with a striped background. SEARUMS can be downloaded from <a href="http://www.searums.org">http://www.searums.org</a></p>   </div> <p /> <p>Each module shown in Fig.Â <a href="#Fig4">8.4</a> has a well-defined functionality that can be accessed and utilized via a set of application program interface (API) method calls. APIs of the modules are Java interface classes that are implemented by each module. Interactions between modules are achieved through interface classes to ensure loose coupling. This approach permits seamless â€œplug and playâ€� of modules and the environment is composed by loading suitable modules dynamically on-demand via Javaâ€™s reflection API (Bloch <a href="#CR2">2001</a>). Such an implementation approach has been adopted to ease customization and extension of SEARUMS without requiring changes to its design or impacting existing modules.</p> <p>The modules constituting SEARUMS are broadly classified as core modules and graphical user interface (GUI) modules. The core modules of SEARUMS are the agent repository, agent customizer, persistence module, HITL steering module, simulation module, and logging module. These modules provide the core M&amp;amp;S functionality of SEARUMS. The GUI facilitates interactions with the core modules via convenient and intuitive user interfaces. The GUI modules can be further categorized into the editor subsystem, the simulation controller, and the visualization and analysis subsystem. The GUI presented by these modules is shown in Fig.Â <a href="#Fig5">8.5</a>. SEARUMS uses the model-view-controller pattern to couple the core modules, the GUI models, and the Eco-description. The design permits the GUI modules to be easily replaced with a minimal command-line text interface for running SEARUMS in offline batch mode. The batch mode is useful for performing repeated runs or analyzing different scenarios on computational clusters. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.5</span></a></div>   <p>The figure presents a screenshot of SEARUMS illustrating the graphical layout as seen by a user. The various modules constituting SEARUMS have been marked using <i>black dashed lines</i>. <i>Purple circles</i> indicate human groups and <i>orange circles</i> indicate poultry flocks. Variation in colors arises due to overlap of human and poultry flocks. <i>Colored squares</i> and corresponding <i>colored dashed</i> lines illustrate migration paths of waterfowl flocks</p>   </div> <p /> <p>The modules and subsystems constituting SEARUMS cooperatively operate on a shared, in-memory representation of the model called the Eco-description. The Eco-description is a centralized data structure that includes all the information necessary for modeling, simulation, and analysis. It is composed using a collection of Java classes and that provide efficient access to data and information required by the various modules. The primary information encapsulated by the Eco-description parameters for analysis relates to the smart agents (Hare and Deadman <a href="#CR16">2004</a>) that constitute the model. As shown in Fig.Â <a href="#Fig4">8.4</a>, the agents are organized into an AGENT REPOSITORY to facilitate instantiation and use via Java reflection API.</p> <p>Currently, SEARUMS includes the following three smart agents: WATERFOWL AGENT that represents a migrating waterfowl flock, POULTRY AGENT that models behavior of poultry flocks, and HUMAN AGENT that models humans. Each agent has its own behavior that reflects the characteristics of its real-world counterpart. The behaviors are customized to represent specific instances of an agent by specifying suitable values for the exposed attributes via the ATTRIBUTE EDITOR GUI module. The attributes of an agent include: </p><ol> <li> <p><i>Geographic attributes</i> that indicate the location (latitude and longitude) and logical association with countries and continents. In addition, each agent has a circle or influence that circumscribes its neighborhood.</p> </li> <li> <p><i>Migratory attributes</i> are specified only for agents whose location changes over the lifetime of the simulation. The migratory attributes are described as a sequence of migration points. Each migration point has geographical and chronological (arrival and departure dates) attributes associated with it. In SEARUMS, only one complete migration cycle needs to be specified. The software automatically reuses the information to simulate annual migratory cycles.</p> </li> <li> <p><i>Statistical attributes</i> for agent instances include their initial population, density and distribution, initial infection percentage, infection spread parameters, incubation periods, mortality rates, and population re-growth parameters.</p> </li> </ol> <p /> <p>The agents implement the conceptual, mathematical model of the system developed using Markov processes as described in <a href="#Sec9">Sect. 8.2.3</a>. They are added to a model via suitable toolbar buttons or menu options provided by SEARUMS. Agent instances are created with default attributes from the agent repository by the AGENT CUSTOMIZER module using Javaâ€™s reflection API. Once instantiated, the attributes for agents can be modified via the ATTRIBUTE EDITOR module. The agents are implemented as a family of Java classes by extending a common base class called AGENT. The AGENT class provides methods for interacting with the simulation kernel, inspecting the neighborhood, scheduling events, and interfacing with the GUI modules.</p> <p>The agents in a model are logically organized into hierarchical sets called groups. SEARUMS permits multiple top-level groups with an arbitrary number of hierarchies, with one or more sub-groups at each hierarchical level. An agent can be a member of multiple groups. The groups serve several different purposes in SEARUMS. A group can be used as a parameter for statistical analysis and for plotting charts. For example, a group called US can be created with 50 different sub-groups, one for each state, encompassing various agents. The main US group can be selected for plotting charts and SEARUMS automatically collates and plots data for each state. Note that even though graph plotting is restricted to one hierarchical level, statistics for plotting are collated in a recursive, depth-rst manner and includes data from all agents in underlying hierarchies. A modeler can use a combination of groups to perform multi-faceted analysis at different scales. In addition, groups can be included or excluded from simulations for analyzing different scenarios. The GUI modules utilize groups to provide control on visibility of agents to manage details displayed on the screen. The group editor module provides the user interface for managing group entries and hierarchies.</p> <p>Once all the agent instances and groups have been established in a model, the parameters for observation are added to the Eco-description. These parameters are selected by the user via the Statistics and Charts editor from a list of options. The list includes the attributes of the agents and the groups in the Eco-description. Each parameter is configured to be sampled hourly, daily, or weekly in terms of simulation-time. Moreover, each parameter can be subjected to statistical operations, such as sum, mean, and median. SEARUMS can dynamically (i.e., during simulation) plot and save a variety of charts including: line graphs and pie charts. Multiple charts can be simultaneously used for analyzing a variety of data.</p> <p>All of the aforementioned information is stored as an integral part of the Eco-description. The Eco-description can be saved for future reuse via the PERSISTENCE MODULE. The Eco-description is unmarshalled into a XML document that is compliant with a predefined XML schema. Serializing to a XML document provides a few advantages. First, it enables simple scripts to be developed that can modify specific values and perform multiple simulation runs in batch mode. Second, XML documents can be readily version controlled and archived using commonly available revision control systems such as CVS and subversion. Third, it eases documentation, validation, sharing, and reuse of valuable domain-specific statistical data collated by different researchers from diverse sources. Lastly, it is used to create checkpoints that reflect different scenarios. Such features play an important role in facilitating large-scale, collaborative epidemiological studies.</p> <p>The SIMULATION MODULE performs the task of conducting a discrete event simulation (DES) using the Eco-description. This module utilizes a multi-threaded DES kernel that manages and schedules the discrete events generated by the agents. Multi-threading enables the DES kernel to exploit the compute power of multiprocessor or multi-core machines thereby reducing the wall-clock time for simulation. The number of threads spawned by the DES kernel is configurable. Each thread processes concurrent events (events with the same timestamp) in parallel without violating the causal constraints between events.</p> <div class="human-in-the-loopsteeringmodule" title="sec"> <div class="title" tagx="title" title="title">
Human-in-the-Loop Steering Module</div> <p>In our current research, we have drawn inferences and conclusions from a variety of case studies conducted using a calibrated Eco-description. First an initial Eco-description was developed via SEARUMS using the modeling methodology described in <a href="#Sec11">Sect. 8.3</a>. TableÂ <a href="#Tab1">8.1</a> lists the waterfowl species, including high risk species (Hagemeijer and Mundkur <a href="#CR14">2006</a>), used to develop the Eco-description. The migratory flyways of the waterfowl and their population have been collated from data published by various organizations (CDC <a href="#CR4">2006</a>; WHO <a href="#CR36">2006c</a>; GROMS <a href="#CR12">2006</a>; GLiPHA <a href="#CR11">2007</a>, Hagemeijer and Mundkur <a href="#CR14">2006</a>). For modeling and simulation purposes the dates for migration were approximated to the middle of the months reported in the statistics. Due to the significant variation in migration patterns the approximated migration dates are expected to have deviations ofÂ Â±2Â weeks which are accounted for through stochastic changes in migration dates each time a simulation is performed. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">TableÂ 8.1</span> </div></p><p>The different agent instances used to develop the Eco-description used for case studies</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Description of agent type</th> <th align="left">No. of instances</th> <th align="left">Total population</th> <th align="left">No. of countries</th> </tr> </thead> <tbody> <tr> <td align="left">Bar-tailed godwit</td> <td char="." align="char">4</td> <td char="." align="char">40,000</td> <td align="left">18</td> </tr> <tr> <td align="left">Canada goose</td> <td char="." align="char">16</td> <td char="." align="char">231,700</td> <td align="left">5</td> </tr> <tr> <td align="left">Common crane</td> <td char="." align="char">9</td> <td char="." align="char">22,500</td> <td align="left">21</td> </tr> <tr> <td align="left">Eurasian widgeon<sup>a</sup></td> <td char="." align="char">3</td> <td char="." align="char">1,296,000</td> <td align="left">17</td> </tr> <tr> <td align="left">Great knot</td> <td char="." align="char">3</td> <td char="." align="char">231,000</td> <td align="left">8</td> </tr> <tr> <td align="left">Mallard<sup>a</sup></td> <td char="." align="char">1</td> <td char="." align="char">5,000</td> <td align="left">1</td> </tr> <tr> <td align="left">Razorbill</td> <td char="." align="char">1</td> <td char="." align="char">148,000</td> <td align="left">4</td> </tr> <tr> <td align="left">Red-breasted goose<sup>a</sup></td> <td char="." align="char">1</td> <td char="." align="char">44,000</td> <td align="left">4</td> </tr> <tr> <td align="left">Red-crowned crane</td> <td char="." align="char">1</td> <td char="." align="char">15,000</td> <td align="left">4</td> </tr> <tr> <td align="left">Siberian crane</td> <td char="." align="char">3</td> <td char="." align="char">30,000</td> <td align="left">12</td> </tr> <tr> <td align="left">Yellow-billed duck</td> <td char="." align="char">2</td> <td char="." align="char">20,000</td> <td align="left">8</td> </tr> <tr> <td align="left">Total waterfowl flocks</td> <td char="." align="char">44</td> <td char="." align="char">4,371,000</td> <td align="left">40</td> </tr> <tr> <td align="left">Total poultry flocks</td> <td char="." align="char">1,315</td> <td char="." align="char">18,136,146,826</td> <td align="left">All</td> </tr> <tr> <td align="left">Total human groups</td> <td char="." align="char">1,314</td> <td char="." align="char">6,646,739,849</td> <td align="left">All</td> </tr> <tr> <td align="left">Total</td> <td char="." align="char">2,673</td> <td char="." align="char">24,787,572,675</td> <td align="left">All</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Note that each agent is used to represent a group</p> <p><sup>a</sup>High risk waterfowl species (Hagemeijer and Mundkur <a href="#CR14">2006</a>). The total of 44 waterfowl flocks with different migratory pathways were used. The total population column shows the sum of the populations of all agent instances in each category</p> </div>  <p /> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <p>In our current research, we have drawn inferences and conclusions from a variety of case studies conducted using a calibrated Eco-description. First an initial Eco-description was developed via SEARUMS using the modeling methodology described in <a href="#Sec11">Sect. 8.3</a>. TableÂ <a href="#Tab1">8.1</a> list the waterfowl species, including high risk species (Hagemeijer and Mundkur <a href="#CR14">2006</a>), used to develop the Eco-description. The migratory flyways of the waterfowl and their population have been collated from data published by various organizations (CDC <a href="#CR4">2006</a>; GLiPHA <a href="#CR11">2007</a>; GROMS <a href="#CR12">2006</a>; Hagemeijer and Mundkur <a href="#CR14">2006</a>; USCB <a href="#CR34">2006</a>; WHO <a href="#CR36">2006c</a>). For modeling and simulation purposes the dates for migration were approximated to the middle of the months reported in the statistics. Due to the significant variation in migration patterns the approximated migration dates are expected to have deviations of Â±2Â weeks which are accounted for through stochastic changes in migration dates each time a simulation is performed. The radius of the waterfowl agents were computed using the population, density, and dispersion data obtained from published statistics (GROMS <a href="#CR12">2006</a>; Hagemeijer and Mundkur <a href="#CR14">2006</a>).</p> <p>The dispersion of poultry population in different continents has been approximated to circular regions with even density (GLiPHA <a href="#CR11">2007</a>; Law et al. <a href="#CR21">2005</a>; Booth <a href="#CR3">1997</a>). Such a modeling approach is commonly used in spatially explicit ecological models (Booth <a href="#CR3">1997</a>; Hare and Deadman <a href="#CR16">2004</a>; Law et al. <a href="#CR21">2005</a>; Winston <a href="#CR41">1994</a>; Wolfram MathWorld <a href="#CR42">2006</a>). Global poultry and human population density data have been collated from statistics published by national organizations and government databases (GLiPHA <a href="#CR11">2007</a>; SEDAC <a href="#CR30">2007</a>; USCB <a href="#CR34">2006</a>). As shown in TableÂ <a href="#Tab1">8.1</a>, our model includes the complete human population (approximately 6.646 billion) humans represented by 1,314 agents. On an average, each human agent models 5.058 million humans living in a contiguous circular region. However, the precise population represented by an agent varies depending on the density of the region it models. Agents modeling dense metropolitan areas have higher human populations while agents modeling rural areas of the world have lower population. In contrast, the radius of all the human agents in the model is equal. The radius was computed using the grid size of gridded human population data from SEDAC (<a href="#CR30">2007</a>).</p> <p>A similar strategy has also been applied to distribute the 18.136 billion poultry birds to 1,315 poultry agents as shown in TableÂ <a href="#Tab1">8.1</a>. All the waterfowl agents have equal radius as determined from the grid size of the gridded poultry data obtained from GLiPHA (<a href="#CR11">2007</a>). However, the poultry population represented by each agent varies depending on the world region being modeled by the agent. Note that our Eco-description includes only a selected subset of the waterfowl as complete migration data is unavailable. However, to the best of our knowledge, it is the most comprehensive model of its kind reported to date. Furthermore, it can be readily extended to include additional waterfowl entities from other parts of the world.</p> <div class="human-in-the-loopsteeringandcalibration" title="sec"> <div class="title" tagx="title" title="title">
Human-in-the-Loop Steering and Calibration</div> <p>The first step in our study was to calibrate the model to ensure that it provides a sufficiently accurate representation of real-world epidemiology. The calibration was performed injunction with verification and validation of the model. We verified the accuracy and fidelity of the aforementioned Eco-description by performing extensive HITL-simulations with initial source of infection set to outbreak in Indonesia, a notable epicenter of H5N1 epidemics (WHO <a href="#CR36">2006a</a>). The Eco-description was interactively calibrated by suitably tuning the following attributes: start date for simulation, initial infection percentage, intra-flock disease spread rate, and inter-flock transmission mechanism. Note that we calibrated only the attributes that were indirectly derived from published statistics. We established validity of the Eco-description and SEARUMS by confirming that the timing and chronology of several outbreaks observed in the simulations correlate with the significant real-world incidents reported (WHO <a href="#CR36">2006a</a>) as shown in TableÂ <a href="#Tab2">8.2</a>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">TableÂ 8.2</span> </div></p><p>Comparison of chronology of significant Real-world outbreaks as reported by WHO (<a href="#CR36">2006</a>) against the simulated outbreaks</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Incident</th> <th align="left">Real-world date</th> <th align="left">Simulated date</th> <th align="left">Error (days)</th> </tr> </thead> <tbody> <tr> <td align="left">Outbreak in Indonesia</td> <td align="left">23â€"Janâ€"2006</td> <td align="left">01â€"Janâ€"2006</td> <td align="left">22</td> </tr> <tr> <td align="left">Infection in Iraq/Iran</td> <td align="left">01â€"Marâ€"2006</td> <td align="left">25â€"Marâ€"2006</td> <td align="left">24</td> </tr> <tr> <td align="left">Infection in China</td> <td align="left">27â€"Aprâ€"2006</td> <td align="left">02â€"Aprâ€"2006</td> <td align="left">âˆ’25</td> </tr> <tr> <td align="left">Infection in Egypt</td> <td align="left">11â€"Octâ€"2006</td> <td align="left">14â€"Sepâ€"2006</td> <td align="left">âˆ’27</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The data was recorded after the model was calibrated</p> </div>  <p /> <p>It must be noted that the HITL-simulations steering played an important role in calibrating the model, particularly given its size and complexity. Using the validated and calibrated Eco-description we performed several case studies to analyze the spread and pandemic threat posed by avian influenza to US. Some of these case studies are discussed in <a href="#Sec15">Sect. 8.5</a>.</p> </div> </div> <div class="experiments" title="sec"> <div class="title" tagx="title" title="title">
Experiments</div> <p>The calibrated Eco-description (see <a href="#Sec13">Sect. 8.4</a> for details) has been used to conduct several experiments to analyze the potential impact of avian influenza to poultry farming in the US. The study was conducted using a number of bio-simulations with three different experimental groups of migrating waterfowl. The three flocks were chosen based on their close proximity to known primary sites of disease outbreaks (Rao et al. <a href="#CR28">2009</a>). The initial infection in each experimental group was varied for analysis.</p> <p>FigureÂ <a href="#Fig6">8.6</a> illustrates one of the trans-Atlantic transmission pathways to the continental US. We observed that the spread was determined by migratory pathways and timelines of different species of waterfowl rather than initial infection percentages (Rao et al. <a href="#CR28">2009</a>). One of the interesting observations is that our experiments correctly predicted an outbreak in the UK (WHO <a href="#CR38">2006c</a>). The graph in Fig.Â <a href="#Fig7">8.7</a> presents the impact of avian influenza outbreaks on poultry population in the continental US. Decrease in poultry population corresponds to H5N1 induced death and culling of birds to control the disease. Increase in poultry population reflects regeneration of poultry flocks after an outbreak. As illustrated by the graph, infections in poultry also follow a cyclic pattern that correlate with annual migration of waterfowl. The mortality figures can be translated to corresponding dollar figures for financial analysis. Additional case studies and experiments conducted using SEARUMS are discussed in the literature (Rao et al. <a href="#CR28">2009</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.6</span></a></div>   <p>Timelines for H5N1 infection spread from Indonesia (a known epicenter of H5N1 outbreaks) to US. The time chart also illustrates the epicenters of the intermediate infections</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 8.7</span></a></div>   <p>Fluctuation in poultry population due to H5N1 outbreak which is controlled by mass culling of infected poultry</p>   </div> <p /> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>This article motivated the need for HITL-simulation steering (see <a href="#Sec3">Sect.Â 8.1.2</a>) to effectively analyze the global epidemiology of emergent diseases. As discussed in <a href="#Sec1">Sect. 8.1</a>, epidemiology is a vital discipline with far reaching impacts on human health and economics. Specifically, the article discussed the conceptual mathematical models (refer to <a href="#Sec7">Sects.Â 8.2.1</a> and <a href="#Sec9">8.2.3</a>) and their implementation in a custom Eco-modeling and HITL bio-simulation environment called SEARUMS. The article also discussed the software architecture of SEARUMS and its design in <a href="#Sec11">Sect.Â 8.3</a>.</p> <p>The procedure involved in utilizing SEARUMS for Eco-modeling and model calibration using HITL-simulation steering was discussed in <a href="#Sec13">Sect.Â 8.4</a>. HITL-simulation steering played a vital role in enabling calibration of the model. The HITL infrastructure was developed because conventional approaches for suitably seeding the model were unsuccessful despite hundreds of attempts and simulation runs. Furthermore, the ability to create checkpoints using the HITL infrastructure enabled rapid analysis of various scenarios.</p> <p>The Eco-modeling and HITL-simulation infrastructure of SEARUMS has been used to model and analyze the global epidemiology of avian influenza. It must be noted that, even though the current emphasis of SEARUMS is on avian influenza, the underlying conceptual models and the software infrastructure are generic. Consequently, the concepts and software tools can be adapted for epidemiological analysis.</p> <p>Some of the experiments conducted using SEARUMS were presented in <a href="#Sec15">Sect.Â 8.5</a> to illustrate the multi-disciplinary applicability of SEARUMS. Researchers, epidemiologists, and ornithologists can utilize HITL-simulations for rapid â€œwhat-ifâ€� types of analysis to study impacts of other factors influencing epidemics. It can be used to analyze other scenarios such as those simulated by Los Alamos National Laboratory (<a href="#CR20">2006</a>). SEARUMS and our Eco-description provide an excellent foundation for further enhancements. Note that use of SEARUMS does not require any special computing infrastructure or programming knowledge. Consequently, experts from multiple domains can collaboratively use SEARUMS to perform various types of analysis on a global scale, assess threats, and measure effectiveness of countermeasures. Our methodology and HITL-simulation environment will enable mankind to strategically invest precious time and resources to combat avian influenza, minimize its impacts on human life and global economy thereby averting a pandemic.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="source" tagx="source" title="source">Infectious diseases of humans: dynamics and control</span><span class="year" tagx="year" title="year">1992</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford University Press</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University</span></span></li> <li tag="ref"><a name="CR2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bloch J (2001) Effective Java programming language guide, 1st edn. Pearson, Prentice Hall </span></li> <li tag="ref"><a name="CR3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Booth</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gecko: a continuous 2d world for ecological modeling</span><span class="source" tagx="source" title="source">Artif Life</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">147163</span><span class="pub-id"><a href="https://dx.doi.org/10.1162/artl.1997.3.3.147">10.1162/artl.1997.3.3.147</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">CDC (2006) Centers for disease control &amp;amp; prevention: avian Influenza: current situation. <a href="http://www.cdc.gov/flu/avian/outbreaks/current.htm">http://www.cdc.gov/flu/avian/outbreaks/current.htm</a></span></li> <li tag="ref"><a name="CR5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daley</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="source" tagx="source" title="source">Epidemic modelling: an introduction</span><span class="year" tagx="year" title="year">2001</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span></span></li> <li tag="ref"><a name="CR6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Epstein</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modelling to contain pandemics</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">460</span><span class="fpage" tagx="fpage" title="fpage">687</span><span class="lpage" tagx="lpage" title="lpage">687</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/460687a">10.1038/460687a</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19661897">19661897</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DAT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraserl</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cajka</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooley Burke DS</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Strategies for mitigating an influenza pandemic</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">442</span><span class="fpage" tagx="fpage" title="fpage">448</span><span class="lpage" tagx="lpage" title="lpage">452</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature04795">10.1038/nature04795</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16642006">16642006</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Flint SJ, Enquist LW, Racaniello VR, Skalka AM (2004) Principles of virology, 2nd edn. American Society for Microbiology (ASM), Washington, DC</span></li> <li tag="ref"><a name="CR9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Garten RJ, Davis CT, Russell CA, Shu B, Lindstrom S, Balish A, Sessions WM, Xu X, Skepner E, Deyde V, Okomo-Adhiambo M, Gubareva L, Barnes J, Smith CB, Emery SL, Hillman MJ, Rivailler P, Smagala J, de Graaf M, Burke DF, Fouchier RAM, Pappas C, Alpuche-Aranda CM, Lopez-Gatell H, Olivera H, Lopez I, Myers CA, Faix D, Blair PJ, Yu C, Keene KM, Dotson JP David, Boxrud D, Sambol AR, Abid SH, St George K, Bannerman T, Moore AL, Stringer DJ, Blevins P, Demmler-Harrison GJ, Ginsberg M, Kriner P, Waterman S, Smole S, Guevara HF, Belongia EA, Clark PA, Beatrice ST, Donis R, Katz J, Finelli L, Bridges CB, Shaw M, Jernigan DB, Uyeki TM, Smith DJ, Klimov AI, Cox NJ (2009) Antigenic and genetic characteristics of swine-origin 2009 A(H1N1) influenza viruses circulating in humans. Science 325(5937):197â€"201. doi:<a href="http://DOI:10.1126/science.1176225">10.1126/science.1176225</a>, <a href="http://www.sciencemag.org/cgi/reprint/325/5937/197.pdf">http://www.sciencemag.org/cgi/reprint/325/5937/197.pdf</a></span></li> <li tag="ref"><a name="CR10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gilbert N, Bankes S (2002) Platforms and methods for agent-based modeling. In: Proceedings of the National Academy of Sciences of the USA, vol 99, pp 7197â€"7198. doi:<a href="http://DOI:10.1073/pnas.072079499">10.1073/pnas.072079499</a></span></li> <li tag="ref"><a name="CR11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">GLiPHA (2007) Global livestock production and health atlas (GLiPHA): animal production and health division of food and agriculture organization of the UN. <a href="http://www.fao.org/ag/aga/glipha/index.jsp">http://www.fao.org/ag/aga/glipha/index.jsp</a></span></li> <li tag="ref"><a name="CR12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">GROMS (2006) Global register of migratory species: summarising knowledge about migratory species for conservation. <a href="http://www.groms.de/">http://www.groms.de/</a></span></li> <li tag="ref"><a name="CR13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gu W, Vetter J, Schwan K (1994) An annotated bibliography of interactive program steering. ACm SIGPLAN notices 29(9):140â€"148. doi:<a href="http://doi.acm.org/10.1145/185009.185038">10.1145/185009.185038</a></span></li> <li tag="ref"><a name="CR14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hagemeijer W, Mundkur T (2006) Migratory flyways in Europe, Africa, and Asia and the spread of HPAI H5N1. In: International scientific conference on avian influenza and wild birds, FAO and OIE, Rome, Italy. <a href="http://www.fao.org/ag/againfo/subjects/en/health/diseases-cards/conference/documents/hagemeijer-mundkur.pdf">http://www.fao.org/ag/againfo/subjects/en/health/diseases-cards/conference/documents/hagemeijer-mundkur.pdf</a></span></li> <li tag="ref"><a name="CR15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Halloran ME, Ferguson NM, Eubank S, Ira M Longini J, Cummings DAT, Lewis B, Xu S, Fraser C, Vullikanti A, Germann TC, Wagener D, Beckman R, Kadau K, Barrett C, Macken CA, Burke DS, Cooley P (2008) Modeling targeted layered containment of an influenza pandemic in the US. Proc Natl Acad Sci USA 105(12):4639â€"4644. doi:<a href="http://DOI:10.1073/pnas.0706849105">10.1073/pnas.0706849105</a>, <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.0706849105">http://www.pnas.org/cgi/doi/10.1073/pnas.0706849105</a></span></li> <li tag="ref"><a name="CR16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hare M, Deadman PJ (2004) Further towards a taxonomy of agent-based simulation models in environmental management. Mathematics and computers in simulation 64(1):25â€"40. doi:<a href="http://dx.doi.org/10.1016/S0378-4754(03)00118-6">10.1016/S0378-4754(03)00118-6</a></span></li> <li tag="ref"><a name="CR17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hufnagel L, Brockmann D, Geisel T (2004) Forecast and control of epidemics in a globalized world. Proc Natl Acad Sci USA 101(42):15124â€"15129. doi:<a href="http://DOI:10.1073/pnas.0308344101">10.1073/pnas.0308344101</a>,<a href="http://www.pnas.org/content/101/42/15124.abstract">http://www.pnas.org/content/101/42/15124.abstract</a></span></li> <li tag="ref"><a name="CR18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kilpatrick AM, Chmura AA, Gibbons DW, Fleischer RC, Marra PP, Daszak P (2006) Predicting the global spread of H5N1 avian influenza. Proc Natl Acad Sci USA103(51):19368â€"19373. doi:<a href="http://DOI:10.1073/pnas.0609227103">10.1073/pnas.0609227103</a></span></li> <li tag="ref"><a name="CR19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamport</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time, clocks, and the ordering of events in a distributed system</span><span class="source" tagx="source" title="source">Commut ACM</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">558565</span></span></li> <li tag="ref"><a name="CR20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">LANL (2006) Los Alamos National Laboratory: avian flu modeled on supercomputer, explores vaccine and isolation options for thwarting a pandemic. <a href="http://www.lanl.gov/news/index.php?fuseaction=home.story&amp;story_id=8171">http://www.lanl.gov/news/index.php?fuseaction=home.story&amp;amp;story_id=8171</a></span></li> <li tag="ref"><a name="CR21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Law R, Dieckmann U, Metz JA (2005) The geometry of ecological interactions: simplifying spatial complexity. Cambridge University Press, New York</span></li> <li tag="ref"><a name="CR22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longini</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nizam</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ungchusak</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanshaoworakul</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummunings</span><span class="given-names" tagx="given-names" title="given-names">DAT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halloran</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span></span><span class="mixed-article-title" title="mixed-article-title">Containing pandemic influenza at the source</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">309</span><span class="issue" tagx="issue" title="issue">5737</span><span class="fpage" tagx="fpage" title="fpage">1083</span><span class="lpage" tagx="lpage" title="lpage">1087</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1115717">10.1126/science.1115717</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16079251">16079251</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Normile</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Avian influenza: evidence points to migratory birds in H5N1 spread</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">311</span><span class="issue" tagx="issue" title="issue">5765</span><span class="fpage" tagx="fpage" title="fpage">1225</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.311.5765.1225">10.1126/science.311.5765.1225</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16513949">16513949</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Railsback</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lytinen</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Agent-based simulation platforms: review and development recommendations</span><span class="source" tagx="source" title="source">Simulation</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">609</span><span class="lpage" tagx="lpage" title="lpage">623</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/0037549706073695">10.1177/0037549706073695</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rao DM, Chernyakhovsky A (2008) Parallel simulation of the global epidemiology of avian influenza. In: Proceedings of the 2008 winter simulation conference, pp 1583â€"1591</span></li> <li tag="ref"><a name="CR26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rao DM, Chernyakhovsk A, Rao V (2007a) SEARUMS: studying epidemiology of avian influenza rapidly using simulation. In: Proceedings of ICMHAâ€™07. Berkeley, pp 667â€"673</span></li> <li tag="ref"><a name="CR27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin Stieger Jr</span><span class="given-names" tagx="given-names" title="given-names">DDH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kidambi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narayanan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">Design and implementation of virtual and constructive simulations using open EAAGLES</span><span class="year" tagx="year" title="year">2007b</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Linus Publications</span></span></li> <li tag="ref"><a name="CR28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chernyakhovsky</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling and analysis of global epidemiology of avian influenza</span><span class="source" tagx="source" title="source">Environ Model Softw</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">124</span><span class="lpage" tagx="lpage" title="lpage">134</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.envsoft.2008.06.011">10.1016/j.envsoft.2008.06.011</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregory</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gust</span><span class="given-names" tagx="given-names" title="given-names">ID</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hay</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimov</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kageyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Komadina</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapedes</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosterin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obuchi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">ADME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rimmelzwaan</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stohr</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tashiro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RAM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">The global circulation of seasonal influenza A (H3N2) viruses</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">320</span><span class="issue" tagx="issue" title="issue">5874</span><span class="fpage" tagx="fpage" title="fpage">340</span><span class="lpage" tagx="lpage" title="lpage">346</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1154137">10.1126/science.1154137</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18420927">18420927</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">SEDAC (2007) SocioEconomic Data and Applications Center (SEDAC): gridded population of the world. <a href="http://sedac.ciesin.columbia.edu/">http://sedac.ciesin.columbia.edu/</a></span></li> <li tag="ref"><a name="CR31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worobey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MaSK</span><span class="given-names" tagx="given-names" title="given-names">Cheung CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghwani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">JSM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origins and evolutionary genomics of the 2009 swine-origin h1n1 influenza a epidemic</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">459</span><span class="fpage" tagx="fpage" title="fpage">1122</span><span class="lpage" tagx="lpage" title="lpage">1125</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature08182">10.1038/nature08182</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19516283">19516283</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solow</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using markov chain successional models backwards</span><span class="source" tagx="source" title="source">J Appl Ecol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">43</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">185188</span></span></li> <li tag="ref"><a name="CR33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tobias R, Hofmann C (2004) Evaluation of free java-libraries for social-scientific agent based simulation. J Artif Soc Soc Simul(JASS) 7(1). <a href="http://jasss.soc.surrey.ac.uk/7/1/6.html">http://jasss.soc.surrey.ac.uk/7/1/6.html</a></span></li> <li tag="ref"><a name="CR34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">USCB (2006) U.S. Census Bureau: 100 largest counties. <a href="http://www.census.gov/popest/counties/CO-EST2006-07.html">http://www.census.gov/popest/counties/CO-EST2006-07.html</a></span></li> <li tag="ref"><a name="CR35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2005) World Health Organization: ten things you need to know about pandemic influenza.<a href="http://www.who.int/csr/disease/influenza/pandemic10things/">http://www.who.int/csr/disease/influenza/pandemic10things/</a></span></li> <li tag="ref"><a name="CR36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2006a) World Health Organization: avian influenza: updates. <a href="http://www.who.int/csr/disease/avian_influenza/updates/en/">http://www.who.int/csr/disease/avian_influenza/updates/en/</a></span></li> <li tag="ref"><a name="CR37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2006b) World Health Organization: H5N1 avian influenza: timeline. <a href="http://www.who.int/entity/csr/disease/avian_influenza/timeline.pdf">http://www.who.int/entity/csr/disease/avian_influenza/timeline.pdf</a></span></li> <li tag="ref"><a name="CR38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2006c) World Health Organization (WHO): avian influenza. <a href="http://www.who.int/csr/disease/avian_influenza/en/">http://www.who.int/csr/disease/avian_influenza/en/</a></span></li> <li tag="ref"><a name="CR39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2007a) A description of the process of seasonal and H5N1 influenza vaccine virus selection and development. <a href="http://www.who.int/entity/csr/disease/avian_influenza/Fluvaccvirusselection.pdf">http://www.who.int/entity/csr/disease/avian_influenza/Fluvaccvirusselection.pdf</a></span></li> <li tag="ref"><a name="CR40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO (2007b) World Health Organization: avian influenza: situation updates. <a href="http://www.who.int/entity/csr/disease/influenza/H5N1-9reduit.pdf">http://www.who.int/entity/csr/disease/influenza/H5N1-9reduit.pdf</a></span></li> <li tag="ref"><a name="CR41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winston</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span></span><span class="source" tagx="source" title="source">Operations research (applications and algorithms), 3rd edn</span><span class="year" tagx="year" title="year">1994</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Belmont</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Duxbury Press</span></span></li> <li tag="ref"><a name="CR42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wolfram MathWorld (2006) Mercator projection. <a href="http://mathworld.wolfram.com/MercatorProjection.html">http://mathworld.wolfram.com/MercatorProjection.html</a></span></li> </ul> </div> </div>  </body></html>