<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="abstract" title="abstract">
      <div class="abstract-title" title="abstract-title">Abstract</div>
      <div class="title" tagx="title" title="title">ABSTRACT</div>
      <p>Influenza is a major global public health threat as a result of its highly pathogenic
         variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing
         offers the potential for a diagnostic test for influenza virus which also provides
         insights on transmission, evolution, and drug resistance and simultaneously detects
         other viruses. We therefore set out to apply the Oxford Nanopore Technologies sequencing
         method to metagenomic sequencing of respiratory samples. We generated influenza virus
         reads down to a limit of detection of 10
         
         <sup>2</sup> to 10
         
         <sup>3</sup> genome copies/ml in pooled samples, observing a strong relationship between the viral
         titer and the proportion of influenza virus reads (
         
         <i>P</i> = 4.7 × 10
         
         <sup>−5</sup>). Applying our methods to clinical throat swabs, we generated influenza virus reads
         for 27/27 samples with mid-to-high viral titers (cycle threshold [
         
         <i>C
            
            <sub>T</sub>
         </i>] values, &lt;30) and 6/13 samples with low viral titers (
         
         <i>C
            
            <sub>T</sub>
         </i> values, 30 to 40). No false-positive reads were generated from 10 influenza virus-negative
         samples. Thus, Nanopore sequencing operated with 83% sensitivity (95% confidence interval
         [CI], 67 to 93%) and 100% specificity (95% CI, 69 to 100%) compared to the current
         diagnostic standard. Coverage of full-length virus was dependent on sample composition,
         being negatively influenced by increased host and bacterial reads. However, at high
         influenza virus titers, we were able to reconstruct &gt;99% complete sequences for all
         eight gene segments. We also detected a human coronavirus coinfection in one clinical
         sample. While further optimization is required to improve sensitivity, this approach
         shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis
         of influenza virus and other respiratory viruses.
         
      </p>
   </div>
</html>