<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.02.27.968008.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.27.968008v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design" />
<meta name="DC.Identifier" content="10.1101/2020.02.27.968008" />
<meta name="DC.Date" content="2020-04-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site’s conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus’ mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.

*   Mpro
    :   Main protease
    CoVs
    :   Coronaviruses
    ORFs
    :   Open reading frames
    3CLpro
    :   Chymotrypsin-like cysteine protease
    S
    :   Spike surface glycoprotein
    E
    :   Small envelope protein
    M
    :   Matrix protein
    N
    :   Nucleocapsid protein
    N3
    :   N-[(5 methylisoxazol-3-yl)carbonyl] alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1--{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide
    cMD
    :   Classical molecular dynamics simulations
    MixMD
    :   Mixed-solvent molecular dynamics simulations
    PDB
    :   Protein Data Bank
    MAV
    :   Maximal accessible volume
    ACN
    :   Acetonitrile
    BNZ
    :   Benzene
    DMSO
    :   Dimethylsulfoxide
    MEO
    :   Methanol
    PHN
    :   Phenol
    URE
    :   Urea
    CMA
    :   Correlated mutation analysis" />
<meta name="DC.Contributor" content="Maria Bzówka" />
<meta name="DC.Contributor" content="Karolina Mitusińska" />
<meta name="DC.Contributor" content="Agata Raczyńska" />
<meta name="DC.Contributor" content="Aleksandra Samol" />
<meta name="DC.Contributor" content="Jack A. Tuszyński" />
<meta name="DC.Contributor" content="Artur Góra" />
<meta name="article:published_time" content="2020-04-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site9s conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus9 mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.27.968008v3" />
<meta name="citation_id" content="2020.02.27.968008v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.27.968008" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.27.968008" />
<meta name="citation_author" content="Maria Bzówka" />
<meta name="citation_author_institution" content="Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6802-8753" />
<meta name="citation_author" content="Karolina Mitusińska" />
<meta name="citation_author_institution" content="Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0183-0845" />
<meta name="citation_author" content="Agata Raczyńska" />
<meta name="citation_author_institution" content="Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1884-3669" />
<meta name="citation_author" content="Aleksandra Samol" />
<meta name="citation_author_institution" content="Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6372-8125" />
<meta name="citation_author" content="Jack A. Tuszyński" />
<meta name="citation_author_institution" content="Department of Physics, University of Alberta" />
<meta name="citation_author_institution" content="DIMEAS, Politecnino di Torino, Corso Duca degli Abruzzi" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9976-0429" />
<meta name="citation_author" content="Artur Góra" />
<meta name="citation_author_institution" content="Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology" />
<meta name="citation_author_email" content="a.gora@tunnelinggroup.pl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2530-6957" />
<meta name="citation_reference" content="Huang, C.; Wang, Y.; Li, X.; Ren, L.; Zhao, J.; Hu, Y.; Zhang, L.; Fan, G.; Xu, J.; Gu, X.; et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 2020." />
<meta name="citation_reference" content="Zhu, N.; Zhang, D.; Wang, W.; Li, X.; Yang, B.; Song, J.; Zhao, X.; Huang, B.; Shi, W.; Lu, R.; et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N. Engl. J. Med. 2020, 382, 727–733." />
<meta name="citation_reference" content="Woo, P.C.Y.; Huang, Y.; Lau, S.K.P.; Yuen, K.-Y. Coronavirus Genomics and Bioinformatics Analysis. Viruses 2010, 2, 1804–1820." />
<meta name="citation_reference" content="Tang, Q.; Song, Y.; Shi, M.; Cheng, Y.; Zhang, W.; Xia, X.-Q. Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition. Sci. Rep. 2015, 5, 17155." />
<meta name="citation_reference" content="Cui, J.; Li, F.; Shi, Z.-L. Origin and evolution of pathogenic coronaviruses. Nat. Rev. Microbiol. 2019, 17, 181–192." />
<meta name="citation_reference" content="Fehr, A.R.; Perlman, S. Coronaviruses: An Overview of Their Replication and Pathogenesis. Methods Mol Biol. 2015, 1282, 1–23." />
<meta name="citation_reference" content="Zhang, L.; Shen, F.; Chen, F.; Lin, Z. Origin and evolution of the 2019 novel coronavirus. Clin. Infect. Dis. 2020." />
<meta name="citation_reference" content="Song, Z.; Xu, Y.; Bao, L.; Zhang, L.; Yu, P.; Qu, Y.; Zhu, H.; Zhao, W.; Han, Y.; Qin, C. From SARS to MERS, Thrusting Coronaviruses into the Spotlight. Viruses 2019, 11, 59." />
<meta name="citation_reference" content="Xue, X.; Yu, H.; Yang, H.; Xue, F.; Wu, Z.; Shen, W.; Li, J.; Zhou, Z.; Ding, Y.; Zhao, Q.; et al. Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design. J. Virol. 2008, 82, 2515–2527." />
<meta name="citation_reference" content="Wu, A.; Peng, Y.; Huang, B.; Ding, X.; Wang, X.; Niu, P.; Meng, J.; Zhu, Z.; Zhang, Z.; Wang, J.; et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe 2020." />
<meta name="citation_reference" content="Zumla, A.; Chan, J.F.W.; Azhar, E.I.; Hui, D.S.C.; Yuen, K.-Y. Coronaviruses — drug discovery and therapeutic options. Nat. Rev. Drug Discov. 2016, 15, 327–347." />
<meta name="citation_reference" content="Liu, W.; Morse, J.S.; Lalonde, T.; Xu, S. Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV. ChemBioChem 2020, cbic.202000047." />
<meta name="citation_reference" content="Jin, Z.; Du, X.; Xu, Y.; Deng, Y.; Liu, M.; Zhao, Y.; Zhang, B.; Li, X.; Zhang, L.; Peng, C.; et al. Structure of Mpro from COVID-19 virus and discovery of its inhibitors. bioRxiv 2020." />
<meta name="citation_reference" content="Zhong, N.; Zhang, S.; Zou, P.; Chen, J.; Kang, X.; Li, Z.; Liang, C.; Jin, C.; Xia, B. Without Its N-Finger, the Main Protease of Severe Acute Respiratory Syndrome Coronavirus Can Form a Novel Dimer through Its C-Terminal Domain. J. Virol. 2008, 82, 4227–4234." />
<meta name="citation_reference" content="Ton, A.-T.; Gentile, F.; Hsing, M.; Ban, F.; Cherkasov, A. Rapid Identification of Potential Inhibitors of SARS-CoV-2 Main Protease by Deep Docking of 1.3 Billion Compounds. ChemRxiv 2020." />
<meta name="citation_reference" content="Xu, Z.; Peng, C.; Shi, Y.; Zhu, Z.; Mu, K.; Wang, X.; Zhu, W. Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation. bioRxiv 2020." />
<meta name="citation_reference" content="Liu, X.; Wang, X.-J. Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines. bioRxiv 2020." />
<meta name="citation_reference" content="Li, Y.; Zhang, J.; Wang, N.; Li, H.; Shi, Y.; Guo, G.; Liu, K.; Hao, Z.; Zou, Q. Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening. bioRxiv 2020." />
<meta name="citation_reference" content="Nguyen, D.D.; Gao, K.; Chen, J.; Wang, R.; Wei, G.-W. Potentially highly potent drugs for 2019-nCoV. bioRxiv 2020." />
<meta name="citation_reference" content="Talluri, S. Virtual High Throughput Screening Based Prediction of Potential Drugs for COVID-19 8 March 2020. Preprints 2020." />
<meta name="citation_reference" content="Chen, Y.W.; Yiu, C.-P.; Wong, K.-Y. Prediction of the 2019-nCoV 3C-like Protease (3CLpro) Structure: Virtual Screening Reveals Velpatasvir, Ledipasvir, and Other Drug Repurposing Candidates. ChemRxiv 2020." />
<meta name="citation_reference" content="Fischer, A.; Sellner, M.; Neranjan, S.; Lill, M.A.; Smieško, M. Inhibitors for Novel Coronavirus Protease Identified by Virtual Screeningof 687 Million Compounds. ChemRxiv 2020." />
<meta name="citation_reference" content="Gentile, D.; Patamia, V.; Scala, A.; Sciortino, M.T.; Piperno, A. Inhibitors of SARS-CoV-2 Main Protease from a Library of Marine Natural Products: A Virtual Screening and Molecular Modeling Study. Preprints." />
<meta name="citation_reference" content="Adem, S.; Eyupoglu, V.; Sarfraz, I.; Rasul, A.; Ali, M. Identification of potent COVID-19 main protease (Mpro) inhibitors from natural polyphenols: An in silico strategy unveils a hope against CORONA. Preprints 2020." />
<meta name="citation_reference" content="Berman, H.M. The Protein Data Bank. Nucleic Acids Res. 2000, 28, 235–242." />
<meta name="citation_reference" content="Zhang, L.; Lin, D.; Sun, X.; Curth, U.; Drosten, C.; Sauerhering, L.; Becker, S.; Rox, K.; Hilgenfeld, R. Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved a-ketoamide inhibitors. Science (80-.). 2020, eabb3405." />
<meta name="citation_reference" content="Bacha, U.; Barrila, J.; Velazquez-Campoy, A.; Leavitt, S.A.; Freire, E. Identification of Novel Inhibitors of the SARS Coronavirus Main Protease 3CL pro +. Biochemistry 2004, 43, 4906–4912." />
<meta name="citation_reference" content="Anand, K. Coronavirus Main Proteinase (3CLpro) Structure: Basis for Design of Anti-SARS Drugs. Science (80-.). 2003, 300, 1763–1767." />
<meta name="citation_reference" content="Mitusińska, K.; Raczyńska, A.; Bzówka, M.; Bagrowska, W.; Góra, A. Applications of water molecules for analysis of macromolecule properties. Comput. Struct. Biotechnol. J. 2020, 18, 355–365." />
<meta name="citation_reference" content="Magdziarz, T.; Mitusińska, K.; Bzówka, M.; Raczyńska, A.; Stańczak, A.; Banas, M.; Bagrowska, W.; Góra, A. AQUA-DUCT 1.0: structural and functional analysis of macromolecules from an intramolecular voids perspective. Bioinformatics 2019." />
<meta name="citation_reference" content="Li, C.; Qi, Y.; Teng, X.; Yang, Z.; Wei, P.; Zhang, C.; Tan, L.; Zhou, L.; Liu, Y.; Lai, L. Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase. J. Biol. Chem. 2010, 285, 28134–28140." />
<meta name="citation_reference" content="Kuipers, R.K.; Joosten, H.-J.; van Berkel, W.J.H.; Leferink, N.G.H.; Rooijen, E.; Ittmann, E.; van Zimmeren, F.; Jochens, H.; Bornscheuer, U.; Vriend, G.; et al. 3DM: Systematic analysis of heterogeneous superfamily data to discover protein functionalities. Proteins Struct. Funct. Bioinforma. 2010, 78, 2101–2113." />
<meta name="citation_reference" content="Subramanian, K.; Mitusińska, K.; Raedts, J.; Almourfi, F.; Joosten, H.-J.; Hendriks, S.; Sedelnikova, S.E.; Kengen, S.W.M.; Hagen, W.R.; Góra, A.; et al. Distant Non-Obvious Mutations Influence the Activity of a Hyperthermophilic Pyrococcus furiosus Phosphoglucose Isomerase. Biomolecules 2019, 9, 212." />
<meta name="citation_reference" content="Schymkowitz, J.; Borg, J.; Stricher, F.; Nys, R.; Rousseau, F.; Serrano, L. The FoldX web server: an online force field. Nucleic Acids Res. 2005, 33, W382–W388." />
<meta name="citation_reference" content="Klausen, M.S.; Jespersen, M.C.; Nielsen, H.; Jensen, K.K.; Jurtz, V.I.; Sønderby, C.K.; Sommer, M.O.A.; Winther, O.; Nielsen, M.; Petersen, B.; et al. NetSurfP-2.0: Improved prediction of protein structural features by integrated deep learning. Proteins Struct. Funct. Bioinforma. 2019, 87, 520–527." />
<meta name="citation_reference" content="Mitusińska, K.; Magdziarz, T.; Bzówka, M.; Stańczak, A.; Gora, A. Exploring Solanum tuberosum Epoxide Hydrolase Internal Architecture by Water Molecules Tracking. Biomolecules 2018, 8, 143." />
<meta name="citation_reference" content="Needle, D.; Lountos, G.T.; Waugh, D.S. Structures of the Middle East respiratory syndrome coronavirus 3C-like protease reveal insights into substrate specificity. Acta Crystallogr. Sect. D Biol. Crystallogr. 2015, 71, 1102–1111." />
<meta name="citation_reference" content="Zhang, L.; Lin, D.; Sun, X.; Rox, K.; Hilgenfeld, R. X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of a-Ketoamide Inhibitors. bioRxiv 2020." />
<meta name="citation_reference" content="Tsai, M.-Y.; Chang, W.-H.; Liang, J.-Y.; Lin, L.-L.; Chang, G.-G.; Chang, H.-P. Essential covalent linkage between the chymotrypsin-like domain and the extra domain of the SARS-CoV main protease. J. Biochem. 2010, 148, 349–358." />
<meta name="citation_reference" content="Anand, K. Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain. EMBO J. 2002, 21, 3213–3224." />
<meta name="citation_reference" content="Lim, L.; Shi, J.; Mu, Y.; Song, J. Dynamically-Driven Enhancement of the Catalytic Machinery of the SARS 3C-Like Protease by the S284-T285-I286/A Mutations on the Extra Domain. PLoS One 2014, 9, e101941." />
<meta name="citation_reference" content="Chang, C.; Jeyachandran, S.; Hu, N.-J.; Liu, C.-L.; Lin, S.-Y.; Wang, Y.-S.; Chang, Y.-M.; Hou, M.-H. Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein. Mol. Biosyst. 2016, 12, 59–66." />
<meta name="citation_reference" content="Dayer, M.R.; Taleb-Gassabi, S.; Dayer, M.S. Lopinavir; A Potent Drug against Coronavirus Infection: Insight from Molecular Docking Study. Arch. Clin. Infect. Dis. 2017, 12." />
<meta name="citation_reference" content="Resnick, E.; Bradley, A.; Gan, J.; Douangamath, A.; Krojer, T.; Sethi, R.; Geurink, P.P.; Aimon, A.; Amitai, G.; Bellini, D.; et al. Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening. J. Am. Chem. Soc. 2019, 141, 8951–8968." />
<meta name="citation_reference" content="Anandakrishnan, R.; Aguilar, B.; Onufriev, A. V. H++ 3.0: automating pK prediction and the preparation of biomolecular structures for atomistic molecular modeling and simulations. Nucleic Acids Res. 2012, 40, W537–W541." />
<meta name="citation_reference" content="Luchko, T.; Gusarov, S.; Roe, D.R.; Simmerling, C.; Case, D.A.; Tuszynski, J.; Kovalenko, A. Three-Dimensional Molecular Theory of Solvation Coupled with Molecular Dynamics in Amber. J. Chem. Theory Comput. 2010, 6, 607–624." />
<meta name="citation_reference" content="Sindhikara, D.J.; Yoshida, N.; Hirata, F. Placevent: An algorithm for prediction of explicit solvent atom distribution-Application to HIV-1 protease and F-ATP synthase. J. Comput. Chem. 2012, 33, 1536–1543." />
<meta name="citation_reference" content="Case, D.A.; Ben-Shalom, I.Y.; Brozell, S.R.; Cerutti, D.S., C.T.E.; Cruzeiro, V.W.D.; Darden, T.A.; Duke, R.E.; Ghoreishi, D.; Gilson, M.K.; Gohlke, H.; et al. AMBER 2018 2018." />
<meta name="citation_reference" content="Pence, H.E.; Williams, A. ChemSpider: An Online Chemical Information Resource. J. Chem. Educ. 2010, 87, 1123–1124." />
<meta name="citation_reference" content="Nikitin, A.M.; Lyubartsev, A.P. New six-site acetonitrile model for simulations of liquid acetonitrile and its aqueous mixtures. J. Comput. Chem. 2007, 28, 2020–2026." />
<meta name="citation_reference" content="Wang, J.; Wang, W.; Kollman, P.A.; Case, D.A. Automatic atom type and bond type perception in molecular mechanical calculations. J. Mol. Graph. Model. 2006, 25, 247–260." />
<meta name="citation_reference" content="Gasteiger, J.; Marsili, M. Iterative partial equalization of orbital electronegativity—a rapid access to atomic charges. Tetrahedron 1980, 36, 3219–3228." />
<meta name="citation_reference" content="Martinez, L.; Andrade, R.; Birgin, E.G.; Martínez, J.M. PACKMOL: A package for building initial configurations for molecular dynamics simulations. J. Comput. Chem. 2009, 30, 2157–2164." />
<meta name="citation_reference" content="Delano, W.L. PyMOL: An Open-Source Molecular Graphics Tool. Ccp4 Newslett Protein Crystallogr 2002, 40." />
<meta name="citation_reference" content="Gertz, E.M.; Yu, Y.-K.; Agarwala, R.; Schäffer, A.A.; Altschul, S.F. Composition-based statistics and translated nucleotide searches: Improving the TBLASTN module of BLAST. BMC Biol. 2006, 4, 41." />
<meta name="citation_reference" content="Camacho, C.; Coulouris, G.; Avagyan, V.; Ma, N.; Papadopoulos, J.; Bealer, K.; Madden, T.L. BLAST+: architecture and applications. BMC Bioinformatics 2009, 10, 421." />
<meta name="citation_reference" content="Altschul, S.F.; Gish, W.; Miller, W.; Myers, E.W.; Lipman, D.J. Basic local alignment search tool. J. Mol. Biol. 1990, 215, 403–410." />
<meta name="citation_reference" content="Sievers, F.; Higgins, D.G. Clustal Omega, Accurate Alignment of Very Large Numbers of Sequences. In Multiple Sequence Alignment Methods; 2014; pp. 105–116." />
<meta name="twitter:title" content="Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site’s conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus’ mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.

*   Mpro
    :   Main protease
    CoVs
    :   Coronaviruses
    ORFs
    :   Open reading frames
    3CLpro
    :   Chymotrypsin-like cysteine protease
    S
    :   Spike surface glycoprotein
    E
    :   Small envelope protein
    M
    :   Matrix protein
    N
    :   Nucleocapsid protein
    N3
    :   N-[(5 methylisoxazol-3-yl)carbonyl] alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1--{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide
    cMD
    :   Classical molecular dynamics simulations
    MixMD
    :   Mixed-solvent molecular dynamics simulations
    PDB
    :   Protein Data Bank
    MAV
    :   Maximal accessible volume
    ACN
    :   Acetonitrile
    BNZ
    :   Benzene
    DMSO
    :   Dimethylsulfoxide
    MEO
    :   Methanol
    PHN
    :   Phenol
    URE
    :   Urea
    CMA
    :   Correlated mutation analysis" />
<meta name="og-title" property="og:title" content="Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site’s conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus’ mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.

*   Mpro
    :   Main protease
    CoVs
    :   Coronaviruses
    ORFs
    :   Open reading frames
    3CLpro
    :   Chymotrypsin-like cysteine protease
    S
    :   Spike surface glycoprotein
    E
    :   Small envelope protein
    M
    :   Matrix protein
    N
    :   Nucleocapsid protein
    N3
    :   N-[(5 methylisoxazol-3-yl)carbonyl] alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1--{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide
    cMD
    :   Classical molecular dynamics simulations
    MixMD
    :   Mixed-solvent molecular dynamics simulations
    PDB
    :   Protein Data Bank
    MAV
    :   Maximal accessible volume
    ACN
    :   Acetonitrile
    BNZ
    :   Benzene
    DMSO
    :   Dimethylsulfoxide
    MEO
    :   Methanol
    PHN
    :   Phenol
    URE
    :   Urea
    CMA
    :   Correlated mutation analysis" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.27.968008v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.27.968008v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.27.968008v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1231086" />
    <title>Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rapsky67-T4JzxHphJtKKljmBuKnTj5ASYWQKlHOvUQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1231086","apath":"\/biorxiv\/early\/2020\/04\/06\/2020.02.27.968008.atom","pisa":"biorxiv;2020.02.27.968008v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.27.968008v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.27.968008v3","title":"Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design","identifier":"node\/1231086"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F5VULS5X78iLtvxOzM8VV807oTZXUNaGW29aJcctY+RRxoXCyAd49K8fFJIo7\\\\\\\/4BmxF2WSUMUyS8wVt0F7Y\\\\\\\/sOXkH3B7B6QwmDgGp0CEXWjUIpBimse1FjxiVE5K3LYGREROzIaNiOvP\\\\\\\/1Yus3fZJVfrS2QgocP6ygiseqfB6raCEQdsYYXieEIubLP0maIeCBmzZnrBqon5GKeI+mB1EWzDv06RAk5PMSg9s9ZbQ2m4cvTeKoGHKVzDoXNVRY\\\\\\\/ATdyoRShGho7QJm3TaDu4EkG908wXCaNDY0\\\\\\\/TCwpBw0h2usyT21qgXigDkzkPLbhd8yNZi2UAU0ihd++X\\\\\\\/fN3\\\\\\\/d5nsEyNX3CuRRNnj8LcOEKnFJAj1uKtGt6nB9LfLgPdh7Y64c+FTJN\\\\\\\/6SNCGHfvRl6N8yf3qubd5i5V765r53l3pET7jBDpM+JRjsvHOpoy\\\\\\\/LYeRNUhF\\\\\\\/5r4PojVJKNwaOJcOyjyW0cmUre5g02PsKJj+P\\\\\\\/Afz12m82rV3LmDIG0HSQFakr9P8owTtFlqDD+JbPUdBo\\\\\\\/SFO95omZiqdVWBChOiirV57XRDNu4rYI4sztrXAQhrqG8K7nH7lJStcDNdFxutKU99iDsFwLvAHBLkgEdeIJtRfq\\\\\\\/5RzwT4DfcDfdKgJdkb7uXMoplqPpbb2YszU0kkN0k0FSqz2cDhB5Y3UQ0iiGvPgrrXEEOSOgKBI0PRoWsXYFfAYbXSY2uekDNhO3T2zvb5wZ\\\\\\\/AlauvBtK4OXvURfhiDMdD0\\\\\\\/H+RviEA3Io3oC03rOhmJQKluZ4MtQig8qDW80sgaGhasyz2JFHHlfal64usgv\\\\\\\/m0bOWG2VW8ZmhyqhftIYQWhgRH1V1v6vuyLn15QhoZYgdui\\\\\\\/8Ii74jU+GWW7O5UVBdTLisG9DglHNebJOWbv0Iw59Vt6MWEbdvDmf6U13diSsz79K6ArI6O83BjVQh2dnU0roQEZbjPPzsIb4VByZHiKe08ViXNId1sZyaJl+aiKa9Yfh05zvcHKNkdAmQe6TDYauL1b\\\\\\\/4Mkd\\\\\\\/jSe5HHXbGeYwn4XF+yd\\\\\\\/1eTOo8n5IMkOO\\\\\\\/5mnLQIMvz0ZJwuI+5J7dOaYOvdR\\\\\\\/222Kr\\\\\\\/24JyvPhKlYkeV6ufAcbh5jOLs53hNB6EE71c+nHF4CWfmXB6ppEfor4vK+hlImSdXTUYqnST650C22mv7Y5htwKQy9SVz1Rx7yHrUnWpcapK7NvhOlLXJ1adFWkMLw2EBTGyJTatScLRAPTm0j71OnopKkX1g0Cfo8t+jee2ERt36LiEqTAMIaSdrvMH35IscEukjL9eOz86lB5f3prmDtQJXFsayrxymwpa0MDnz4dyPIE7S03u0v\\\u0022,\\\u0022iv\\\u0022:\\\u002296LGOxKWCC70G36Gmt5Dfw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e43e9fc8c8d4ce70f99bc729c23fe711195391af6d4d0f12276b5d91e90b14ed\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.27.968008v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1231086 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.27.968008v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hth9B0Cw5U9_DZW_r5uK2gKjQqPcePRXx-B6yWLelWw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231086" id="node1231086" data-pisa="biorxiv;2020.02.27.968008v3" data-pisa-master="biorxiv;2020.02.27.968008" data-apath="/biorxiv/early/2020/04/06/2020.02.27.968008.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6802-8753" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bzówka</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0183-0845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Karolina</span> <span class="nlm-surname">Mitusińska</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1884-3669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Agata</span> <span class="nlm-surname">Raczyńska</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6372-8125" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Aleksandra</span> <span class="nlm-surname">Samol</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9976-0429" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jack A.</span> <span class="nlm-surname">Tuszyński</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2530-6957" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Artur</span> <span class="nlm-surname">Góra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.27.968008 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1231086" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maria Bzówka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology</span>, Gliwice, 44-100, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BBz%C3%B3wka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bz%C3%B3wka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BBz%25C3%25B3wka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6802-8753" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maria Bzówka</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karolina Mitusińska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology</span>, Gliwice, 44-100, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karolina%2BMitusi%C5%84ska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitusi%C5%84ska%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarolina%2BMitusi%25C5%2584ska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0183-0845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karolina Mitusińska</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Agata Raczyńska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology</span>, Gliwice, 44-100, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agata%2BRaczy%C5%84ska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raczy%C5%84ska%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAgata%2BRaczy%25C5%2584ska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1884-3669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Agata Raczyńska</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aleksandra Samol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology</span>, Gliwice, 44-100, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aleksandra%2BSamol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samol%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAleksandra%2BSamol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6372-8125" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aleksandra Samol</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jack A. Tuszyński </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Physics, University of Alberta</span>, Edmont, AB, T6G 2E1, <span class='nlm-country'>Canada</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>DIMEAS, Politecnino di Torino, Corso Duca degli Abruzzi</span>, 24, Turin, 10129, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jack%2BA.%2BTuszy%C5%84ski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuszy%C5%84ski%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJack%2BA.%2BTuszy%25C5%2584ski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9976-0429" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jack A. Tuszyński</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Artur Góra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Tunneling Group, Biotechnology Centre, ul. Krzywoustego 8, Silesian University of Technology</span>, Gliwice, 44-100, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Artur%2BG%C3%B3ra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B3ra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArtur%2BG%25C3%25B3ra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2530-6957" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Artur Góra</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:a.gora@tunnelinggroup.pl" class="" data-icon-position="" data-hide-link-title="0">a.gora@tunnelinggroup.pl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.27.968008v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.968008v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.968008v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.968008v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.968008v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.27.968008v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1231086" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1231086/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site’s conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus’ mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-4">In early December 2019, the first atypical pneumonia outbreak associated with the novel coronavirus of zoonotic origin (SARS-CoV-2) appeared in Wuhan City, Hubei Province, China [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. In general, coronaviruses (CoVs) are classified into four major genera: Alphacoronavirus, Betacoronavirus (which primarily infect mammals), Gammacoronavirus, and Deltacoronavirus (which primarily infect birds) [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. In humans, coronaviruses usually cause mild to moderate upper-respiratory tract illnesses, e.g., the common cold, however, the rarer forms of CoVs can be lethal. By the end of 2019, six kinds of human CoV have been identified: HCoV-NL63, HCoV-229E, belonging to Alphacoronavirus genera, HCoV-OC43, HCoV-HKU1, severe acute respiratory syndrome SARS-CoV, and Middle East respiratory syndrome MERS-CoV, belonging to Betacoronavirus genera [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Of the aforementioned CoVs, the last two are the most dangerous and they were associated with the outbreak of two epidemics at the beginning of the 21st century [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. On January 2020, SARS-CoV-2 was isolated and announced as a new, seventh, type of human coronavirus. It was classified as Betacoronavirus [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. Based on the phylogenetic analysis of the genomic data of SARS-CoV-2, Zhang et al. indicated that the SARS-CoV-2 is most closely related to two SARS-CoV sequences isolated from bats in 2015 and 2017. This is suggestive that the bat’s CoV and SARS-CoV-2 share a common ancestor, and the new virus can be considered as a SARS-like virus [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>].</p><p id="p-5">The genome of coronaviruses typically contains a positive-sense, single-stranded RNA but it differs in size ranging between ~26 and ~32 kb. It also includes a variable number of open reading frames (ORFs) – from 6 to 11. The first ORF is the largest, encoding nearly 70% of the entire genome and 16 non-structural proteins (nsps) [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Of the nsps, the main protease (Mpro, also known as a chymotrypsin-like cysteine protease, 3CLpro), encoded by nsp5, has been found to play a fundamental role in viral gene expression and replication, thus it is an attractive target for anti-CoV drug design [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. The remaining ORFs encode accessory and structural proteins, including spike surface glycoprotein (S), small envelope protein (E), matrix protein (M), and nucleocapsid protein (N).</p><p id="p-6">Based on the three sequenced genomes of SARS-CoV-2 (Wuhan/IVDC-HB-01/2019, Wuhan/IVDC-HB-04/2019, and Wuhan/IVDC-HB-05/2019, provided by the National Institute for Viral Disease Control and Prevention, CDC, China), Wu et al., performed a detailed genome annotation. The results were further compared to related coronaviruses – 1,008 human SARS-CoV, 338 bat SARS-like CoV, and 3,131 human MERS-CoV indicating that the three strains of SARS-CoV-2 have almost identical genomes with 14 ORFs, encoding 27 proteins including 15 non-structural proteins (nsp1-10 and nsp12-16), 4 structural proteins (S, E, M, N), and 8 accessory proteins (3a, 3b, p6, 7a, 7b, 8b, 9b, and orf14). The only identified difference in the genome consisting of ~29.8 kb nucleotides consisted of five nucleotides. The genome annotation revealed that SARS-CoV-2 is fairly similar to SARS-CoV at the amino acid level, however, there are some differences in the occurrence of accessory proteins, e.g., the 8a accessory protein, present in SARS-CoV, is absent in SARS-CoV-2 and the lengths of 8b and 3b proteins do not match. The phylogenetic analysis of SARS-CoV-2 showed it to be most closely related to SARS-like bat viruses, but no strain of SARS-like bat virus was found to cover all equivalent proteins of SARS-CoV-2 [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-7">As previously mentioned, the main protease is one of the key enzymes in the viral life cycle. Together with other non-structural proteins (papain-like protease, helicase, RNA-dependent RNA polymerase) and the spike glycoprotein structural protein, it is essential for interactions between the virus and host cell receptor during viral entry [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Initial analyses of genomic sequences of the four nsps mentioned above indicate that those enzymes are highly conserved sharing more than 90% sequence similarity with the corresponding SARS-CoV enzymes [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>].</p><p id="p-8">The first released crystal structure of the Mpro of SARS-CoV-2 (PDB ID: 6lu7) was obtained by Prof. Yang’s group from ShanghaiTech by co-crystallisation with a peptide-like inhibitor N-[(5 methylisoxazol-3-yl)carbonyl]alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1-{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide (N3 or PRD_002214) [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. The same inhibitor was co-crystallised with other human coronaviruses, e.g., HCoV-NL63 (PDB ID: 5gwy), HCoV-KU1 (PDB ID: 3d23), or SARS-CoV (PDB ID: 2amq). This enzyme naturally forms a dimer whose each monomer consists of the N-terminal catalytic region and a C-terminal region [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. While 12 residues differ between both CoVs, only one, namely S46 in SARS-CoV-2 (A46 in SARS-CoV), is located in the proximity of the entrance to the active site. However, such a small structural change would typically be not expected to substantially affect the binding of small molecules [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. Such an assumption would routinely involve the generation of a library of derivatives and analogous based on the scaffold of a drug that inhibits the corresponding protein in the SARS-CoV case. As shown in the present paper, regrettably, this strategy is not likely to succeed with SARS-CoV-2 for Mpro as a molecular target.</p><p id="p-9">In this study, we investigate how only 12 different residues, located mostly on the protein’s surface, may affect the behaviour of the active site pocket of the SARS-CoV-2 Mpro structure. To this end, we performed classical molecular dynamics simulations (cMD) of both SARS and SARS-CoV-2 Mpros as well as mixed-solvents molecular dynamics simulations (MixMD) combined with small molecules’ tracking approach to analyse the conformational changes in the binding site. In spite of the structural differences in the active sites of both Mpro proteins, major issues involving plasticity and flexibility of the binding site could result in significant difficulties in inhibitor design for this molecular target. Indeed, an in silico attempt has already been made involving a massive virtual screening for Mpro inhibitors of SARS-CoV-2 using Deep Docking [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Other recent attempts focused on virtual screening for putative inhibitors of the same main protease of SARS-CoV-2 based on the clinically approved drugs [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], and also based on the compounds from different databases or libraries [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>–<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. However, none of such attempts is likely to lead to clinical advances in the fight against SARS-CoV-2 for reasons we elaborate below.</p></div><div class="section" id="sec-2"><h2 class="">2. Results</h2><div id="sec-3" class="subsection"><h3>2.1. Crystal structure comparison, and location of the replaced amino acids distal to the active site</h3><p id="p-10">The first SARS-CoV-2 main protease’s crystallographic structure was made publicly available through the Protein Data Bank (PDB) [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] as a complex with an N3 inhibitor (PDB ID: 6lu7) [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. Next, the structure without the inhibitor was also made available (PDB ID: 6y2e) [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. We refer to those structures as SARS-CoV-2 Mpro<sup>N3</sup> and SARS-CoV-2 Mpro, respectively. We also used two structures of the SARS-CoV main protease: one, referred to as SARS-CoV Mpro<sup>N3</sup> (PDB ID: 2amq), co-crystallised with the same N3 inhibitor, and the other without an inhibitor (PDB ID: 1q2w), which we refer to as SARS-CoV Mpro. The SARS-CoV-2 Mpro and SARS-CoV Mpro structures differ by only 12 amino acids located mostly on the proteins’ surface (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, Supplementary Table S1). Both enzymes share the same structural composition; they comprise three domains: domains I (residues 1-101) and II (residues 102-184) consist of an antiparallel β-barrel, and the α-helical domain III (residues 201-301) is required for the enzymatic activity [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Both enzymes resemble the structure of cysteine proteases, although their active site is lacking the third catalytic residue [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]; their active site comprises a catalytic dyad, namely H41 and C145, and a particularly stable water molecule forms at least three hydrogen bond interactions with surrounding residues, including the catalytic histidine, which corresponds to the position of a third catalytic member (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). It should be also noted that one of the differing amino acids in SARS-CoV-2 Mpro, namely S46, is located on a C44-P52 loop, which is flanking the active site cavity.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The differences between the SARS-CoV Mpro and SARS-CoV-2 Mpro structures. (A) The overall structure of both SARS-CoV and SARS-CoV-2 Mpros with differing amino acids marked as black (SARS-CoV Mpro) and blue (SARS-CoV-2 Mpro). (B) Close-up of the active site cavity and bound N3 inhibitor into SARS-CoV (black sticks) and SARS-CoV-2 (blue sticks) Mpros. The catalytic water molecule that resembles the position of the third member of the catalytic triad adopted from the cysteine proteases is shown for both SARS-CoV (black sphere) and SARS-CoV-2 (blue sphere) Mpros. The active site residues are shown as red sticks and the proteins&#x2019; structures are shown in surface representation. The differing residues in position 46 located near the entrance to the active site are marked with an asterisk (*) on the (A) and as blue and black lines on the (B) panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The differences between the SARS-CoV Mpro and SARS-CoV-2 Mpro structures. (A) The overall structure of both SARS-CoV and SARS-CoV-2 Mpros with differing amino acids marked as black (SARS-CoV Mpro) and blue (SARS-CoV-2 Mpro). (B) Close-up of the active site cavity and bound N3 inhibitor into SARS-CoV (black sticks) and SARS-CoV-2 (blue sticks) Mpros. The catalytic water molecule that resembles the position of the third member of the catalytic triad adopted from the cysteine proteases is shown for both SARS-CoV (black sphere) and SARS-CoV-2 (blue sphere) Mpros. The active site residues are shown as red sticks and the proteins&#x2019; structures are shown in surface representation. The differing residues in position 46 located near the entrance to the active site are marked with an asterisk (*) on the (A) and as blue and black lines on the (B) panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F1.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F1.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">The differences between the SARS-CoV Mpro and SARS-CoV-2 Mpro structures. (A) The overall structure of both SARS-CoV and SARS-CoV-2 Mpros with differing amino acids marked as black (SARS-CoV Mpro) and blue (SARS-CoV-2 Mpro). (B) Close-up of the active site cavity and bound N3 inhibitor into SARS-CoV (black sticks) and SARS-CoV-2 (blue sticks) Mpros. The catalytic water molecule that resembles the position of the third member of the catalytic triad adopted from the cysteine proteases is shown for both SARS-CoV (black sphere) and SARS-CoV-2 (blue sphere) Mpros. The active site residues are shown as red sticks and the proteins’ structures are shown in surface representation. The differing residues in position 46 located near the entrance to the active site are marked with an asterisk (*) on the (A) and as blue and black lines on the (B) panel.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>2.2. Plasticity of the binding cavities</h3><p id="p-12">A total of 2 μs classical molecular dynamics (cMD) simulations of both SARS-CoV-2 and SARS-CoV Mpros with different starting points were run to examine the plasticity of their binding cavities. A combination of the cMD approach with water molecules used as molecular probes is assumed to provide a highly detailed picture of the protein’s interior dynamics [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. The small molecules tracking approach was used to determine the accessibility of the active site pocket in both SARS-CoV and SARS-CoV-2 Mpros, and a local distribution approach was used to provide information about an overall distribution of solvent in the proteins’ interior. To properly examine the flexibility of both active site cavities, we used the time-window mode implemented in AQUA-DUCT software [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] to analyse the water molecules’ flow through the cavity in a 10 ns time step and combined that with the outer pocket calculations to examine the plasticity and maximal accessible volume (MAV) of the binding cavity.</p><p id="p-13">Surprisingly, despite their high similarity, the binding cavities of SARS-CoV and SARS-CoV-2 Mpros show significantly different MAV. Both proteins reduce their MAV upon inhibitor binding by approximately 20%, but the maximal volume of SARS-CoV is over 50% larger than those of SARS-CoV-2 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Supplementary Figure S1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The differences between the maximal accessible volume of the binding cavities calculated during MD simulations of both apo structures of Mpros (SARS-CoV and SARS-CoV-2) and structures with co-crystallised N3 inhibitor (SARS-CoVN3 and SARS-CoV-2N3) used as different starting points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The differences between the maximal accessible volume of the binding cavities calculated during MD simulations of both apo structures of Mpros (SARS-CoV and SARS-CoV-2) and structures with co-crystallised N3 inhibitor (SARS-CoV&lt;sup&gt;N3&lt;/sup&gt; and SARS-CoV-2&lt;sup&gt;N3&lt;/sup&gt;) used as different starting points.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F2.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F2.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-14" class="first-child">The differences between the maximal accessible volume of the binding cavities calculated during MD simulations of both apo structures of Mpros (SARS-CoV and SARS-CoV-2) and structures with co-crystallised N3 inhibitor (SARS-CoV<sup>N3</sup> and SARS-CoV-2<sup>N3</sup>) used as different starting points.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>2.3. Flexibility of the active site entrance</h3><p id="p-15">To further examine the plasticity and flexibility of the main proteases binding cavities, we focused on the movements of loops surrounding their entrances and regulating the active sites’ accessibility. We found that one of the analysed loops of the SARS-CoV Mpro, namely C44-P52 loop, is more flexible than the corresponding loops of SARS-CoV-2 Mpro structure, while the adjacent loops are mildly flexible (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This could be indirectly assumed from the absence of the C44-P52 loop in the crystallographic structure of SARS-CoV Mpro structure. On the other hand, such flexibility could suggest that the presence of an inhibitor might stabilise the loops surrounding the active site. It is worth adding, that this loop is carrying the unique SARS-CoV-2 Mpro residue S46.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flexibility of loops surrounding the entrance to the binding cavity of (A) SARS-CoV-2 Mpro, (B) SARS-CoV Mpro, (C) SARS-CoV MproN3, and (D) SARS-CoV MproN3. For the picture clarity, only residues creating loops were shown. The active site residues are shown as red sticks and the A46S replacement between SARS-CoV and SARS-CoV-2 main proteases is shown as light blue sticks. The width and colour of the shown residues reflect the level of loop flexibility. The wider and darker residues are more flexible." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Flexibility of loops surrounding the entrance to the binding cavity of (A) SARS-CoV-2 Mpro, (B) SARS-CoV Mpro, (C) SARS-CoV Mpro&lt;sup&gt;N3&lt;/sup&gt;, and (D) SARS-CoV Mpro&lt;sup&gt;N3&lt;/sup&gt;. For the picture clarity, only residues creating loops were shown. The active site residues are shown as red sticks and the A46S replacement between SARS-CoV and SARS-CoV-2 main proteases is shown as light blue sticks. The width and colour of the shown residues reflect the level of loop flexibility. The wider and darker residues are more flexible.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F3.medium.gif" width="440" height="103"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F3.medium.gif" width="440" height="103"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-16" class="first-child">Flexibility of loops surrounding the entrance to the binding cavity of (A) SARS-CoV-2 Mpro, (B) SARS-CoV Mpro, (C) SARS-CoV Mpro<sup>N3</sup>, and (D) SARS-CoV Mpro<sup>N3</sup>. For the picture clarity, only residues creating loops were shown. The active site residues are shown as red sticks and the A46S replacement between SARS-CoV and SARS-CoV-2 main proteases is shown as light blue sticks. The width and colour of the shown residues reflect the level of loop flexibility. The wider and darker residues are more flexible.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>2.4. Cosolvent hot-spots analysis</h3><p id="p-17">The mixed-solvent MD simulations were run with 6 cosolvents: acetonitrile (ACN), benzene (BNZ), dimethylsulfoxide (DMSO), methanol (MEO), phenol (PHN), and urea (URE). Cosolvents were used as specific molecular probes, representing different chemical properties and functional groups that would complement the different regions of the binding site and the protein itself. Using small molecules tracking approach we analysed the flow through the Mpros structures and identified the regions in which those molecules are being trapped and/or caged, located within the protein itself (global hot-spots; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, Supplementary Figure S2) and inside the binding cavity (local hot-spots; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, Supplementary Figure S3). The size and location of both types of hot-spots differ and provide complementary information. The global hot-spots identify potential binding/interacting sites in the whole protein structure and additionally provide information about regions attracting particular types of molecules, whereas local hot-spots describe the actual available binding space of a specific cavity.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Localisation of the global hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Localisation of the global hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F4.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F4.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-18" class="first-child">Localisation of the global hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The general distribution of the global hot-spots from particular cosolvents is quite similar and verifies specific interactions with the particular regions of the analysed proteins. A notable number of hot-spots are located around the amino acids that vary between the SARS-CoV-2 and SARS-CoV Mpros (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, Supplementary Figure S2). The largest number and the densest hot-spots are located within the binding cavity and the region essential for Mpros dimerisation [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], between the II and III domains. The binding cavity is particularly occupied by urea, benzene and phenol hot-spots, which is especially interesting, since these solvents exhibit different chemical properties.</p><p id="p-20">A close inspection of the binding site cavity provides further details of cosolvent distribution. The benzene hot-spots for the SARS-CoV-2 Mpro structure are localised deep inside the active site cavity, while SARS-CoV Mpro features mostly benzene hot-spots at the cavity entrance (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). This is interesting since, in the absence of cosolvent molecules, the water accessible volume for SARS-CoV-2 Mpro was 50% smaller than in the case of SARS-CoV Mpro, underlining huge plasticity of the binding cavity and suggesting large conformational changes induced by interaction with a potential ligand. It is also interesting, that both global and local hot-spots of the SARS-CoV Mpro structure are located in the proximity of the C44-P52 loop, which potentially regulates the access to the active site.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Localisation of the local hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene. The active site residues are shown as red sticks, and the proteins&#x2019; structures are shown in cartoon representation, loop 44-52 is grey." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Localisation of the local hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene. The active site residues are shown as red sticks, and the proteins&#x2019; structures are shown in cartoon representation, loop 44-52 is grey.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F5.medium.gif" width="440" height="158"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F5.medium.gif" width="440" height="158"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-21" class="first-child">Localisation of the local hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spots density. The colour coding is as follows: purple - urea, green - dimethylsulfoxide, yellow - methanol, orange - acetonitrile, pink - phenol, red - benzene. The active site residues are shown as red sticks, and the proteins’ structures are shown in cartoon representation, loop 44-52 is grey.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>2.5. Potential mutability of SARS-CoV-2</h3><p id="p-22">In general, all the above-mentioned findings indicate potential difficulties in the identification of specific inhibitors toward Mpro proteins. First, the binding site itself is characterised by huge plasticity and probably even distant to active site mutations modify Mpro binding properties. Secondly, the C44-P52 loop regulates access to the active site and can contribute to the discrimination of potential inhibitors. Therefore, additional mutations in the above-mentioned regions, which could appear during further SARS-CoV-2 evolution, can significantly change the affinity between Mpro and its ligands. To verify potential threat of further mutability of the Mpro protein we performed: i) correlated mutation analysis (CMA) on multiple sequence alignment, ii) the analysis of the contribution of already identified differences between the SARS-CoV and SARS-CoV-2 Mpros to protein stability, and iii) prediction of further possible mutations caused by the most probable mutations, the substitution of single nucleotides in the mRNA sequence of Mpro.</p><p id="p-23">Indeed, the analysis performed with Comulator software [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] shows that within viral Mpros evolutionary-correlated residues are dispersed throughout the structure. This indirectly supports our previous findings that distant amino acids mutation can contribute significantly to the binding site plasticity. It is worth adding that among evolutionary-correlated residues we identified also those that differ between SARS-CoV-2 and SARS-CoV Mpros, located on the C44-P52 loop (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) and the F185-T201 linker loop. The CMA analysis indicates that particular residues in both loops are evolutionary-correlated. The Q189 from the linker loop correlates with residues from the C44-P52 loop, whereas R188, A191, and A194 correlate with selected residues from all domains, but not with the C44-P52 loop. As was shown, the mutation of amino acids distant from active site residues, that are evolutionary correlated, is most likely to modify the active site accessibility [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>].</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Localisation of the evolutionary-correlated residues of Mpros (black sticks). The CMA analysis provided four groups of evolutionary-correlated residues. The SARS-CoV-2 Mpro structure is presented as cartoon, the active site residues are shown as red sticks, the unique residues of the SARS-CoV-2 Mpro as blue sticks, the asterisk (*) indicates the residue belonging to the evolutionary-correlated residues, unique for SARS-CoV Mpro. The loop C44-P52 is coloured black and the F185-T201 loop is orange. Please note, that within one of the correlated groups (upper left) the residues from C44-P52 loop are correlated with Q189 from the linker loop and with residue from III domain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Localisation of the evolutionary-correlated residues of Mpros (black sticks). The CMA analysis provided four groups of evolutionary-correlated residues. The SARS-CoV-2 Mpro structure is presented as cartoon, the active site residues are shown as red sticks, the unique residues of the SARS-CoV-2 Mpro as blue sticks, the asterisk (*) indicates the residue belonging to the evolutionary-correlated residues, unique for SARS-CoV Mpro. The loop C44-P52 is coloured black and the F185-T201 loop is orange. Please note, that within one of the correlated groups (upper left) the residues from C44-P52 loop are correlated with Q189 from the linker loop and with residue from III domain.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F6.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F6.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-24" class="first-child">Localisation of the evolutionary-correlated residues of Mpros (black sticks). The CMA analysis provided four groups of evolutionary-correlated residues. The SARS-CoV-2 Mpro structure is presented as cartoon, the active site residues are shown as red sticks, the unique residues of the SARS-CoV-2 Mpro as blue sticks, the asterisk (*) indicates the residue belonging to the evolutionary-correlated residues, unique for SARS-CoV Mpro. The loop C44-P52 is coloured black and the F185-T201 loop is orange. Please note, that within one of the correlated groups (upper left) the residues from C44-P52 loop are correlated with Q189 from the linker loop and with residue from III domain.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">In the interest of examining the energetical effect of the 12 amino acid replacement in the SARS-CoV-2 Mpro structure, we calculated their energetic contributions to the protein’s stability using FoldX [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. As expected, the differences in total energies of the SARS-CoV Mpro and variants with introduced mutation from SARS-CoV-2 Mpro residue did not represent a significant energy change (Supplementary Table S1). The biggest energy reduction was found for mutation H134F (−0.85 kcal/mol) and mutations R88K, S94A, T285A, I286L only slightly reduced the total energy (Supplementary Table S1).</p><p id="p-26">To investigate further possible mutations of SARS-CoV-2 Mpro, single nucleotide substitutions were introduced to the SARS-CoV-2 main protease gene. If a substitution of a single nucleotide caused translation to a different amino acid compared to the corresponding residue in the wild-type structure, an appropriate mutation was proposed with FoldX calculations. The most energetically favourable potential mutations were chosen based on a −1.5 kcal/mol threshold (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, Supplementary Table S2). Most of the energetically favourable potential mutations include amino acids that are solvent-exposed on the protein’s surface, according to NetSurfP [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>] results. These results show that in general, exposed amino acids are more likely to mutate.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Potential mutability of SARS-CoV-2 Mpro. (A) Structure of SARS-CoV-2 Mpro with the most energetically favourable potential mutations of amino acids marked as green surface. Positions of amino acids that differ from the ones in SARS-CoV Mpro structure marked as blue sticks. Catalytic dyad marked as red. (B) The catalytic site of SARS-CoV-2 Mpro is shown as surface with the most energetically favourable potential mutations shown as green, neutral as white and unfavourable as red. The C44-P52 loop is shown as black mesh." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-69261035" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Potential mutability of SARS-CoV-2 Mpro. (A) Structure of SARS-CoV-2 Mpro with the most energetically favourable potential mutations of amino acids marked as green surface. Positions of amino acids that differ from the ones in SARS-CoV Mpro structure marked as blue sticks. Catalytic dyad marked as red. (B) The catalytic site of SARS-CoV-2 Mpro is shown as surface with the most energetically favourable potential mutations shown as green, neutral as white and unfavourable as red. The C44-P52 loop is shown as black mesh.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F7.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F7.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.02.27.968008/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <p id="p-27" class="first-child">Potential mutability of SARS-CoV-2 Mpro. (A) Structure of SARS-CoV-2 Mpro with the most energetically favourable potential mutations of amino acids marked as green surface. Positions of amino acids that differ from the ones in SARS-CoV Mpro structure marked as blue sticks. Catalytic dyad marked as red. (B) The catalytic site of SARS-CoV-2 Mpro is shown as surface with the most energetically favourable potential mutations shown as green, neutral as white and unfavourable as red. The C44-P52 loop is shown as black mesh.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Additionally, the potential mutability of the binding cavity was investigated. Residues belonging to the binding cavity were found within 7 Å from the N3 inhibitor. Then, we calculated the differences in the Gibbs free energy of protein folding with respect to the wild-type protein (Supplementary Table S3) and presented the results as a heat map. The most energetically favourable potential mutations are shown as green, neutral as white and unfavourable as red (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Interestingly, residues forming the catalytic dyad, namely H41 and C145, are also prone to mutate. However, probably the most important message comes from the analysis of the potential mutability of the C44-P52 loop. Mutation of four of them has a stabilising effect for the protein and for the rest the effect is near-neutral. These results indicate that the future evolution of the Mpro protein can significantly reduce the potential use of this protein as a molecular target for coronavirus treatment due to a highly probable development of drug resistance of this virus through mutations.</p></div></div><div class="section" id="sec-8"><h2 class="">3. Discussion</h2><p id="p-29">As we have shown in previous research, tracking of water molecules in the binding cavity combined with the local distribution approach can identify catalytic water positions [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Indeed, despite differences in the size and dynamics of the binding cavities of SARS-CoV and SARS-CoV-2 Mpros, the main identified water was always found in a position next to the H41 residue, and this location is assumed to indicate catalytic water of Mpro replacing the missing third catalytic site amino acid [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>]. That was the first quality check of our methodology that approved our approach, and has initiated further investigations.</p><p id="p-30">As reported in the previous research, the overall plasticity of Mpro is required for proper enzyme functioning [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. In the case of SARS-CoV the truncation of the linker loop (F185-T201) gave rise to a significant reduction in protein activity and confirmed that the proper orientation of the linker allows the shift between dimeric and monomeric forms [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Dimerisation of the enzyme is necessary for its catalytic activity and the proper conformation of the seven N-terminal residues (N-finger) is required [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. In SARS-CoV-2 Mpro, the T285 is replaced by alanine, and the I286 by leucine. It has been shown that replacing S284, T285, and I286 by alanine residues in SARS-CoV Mpro leads to a 3.6-fold enhancement of the catalytic activity of the enzyme. This is accompanied by changes in the structural dynamics of the enzyme that transmit the effect of the mutation to the catalytic centre. Indeed, the T285A replacement observed in the SARS-CoV-2 Mpro allows the two domains III to approach each other a little closer [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>].</p><p id="p-31">The comparison of MD simulations of both main proteases initiated from different starting conformations (with and without N3 inhibitor) suggests that beside plasticity of the whole protein, there can be large differences between the accessibility to the binding cavity and/or the accommodation of the shape of the cavity in response to the inhibitor that can be bound. There are also differences in the outer pockets’ maximal accessible volumes between the two structures of SARS-CoV main proteases; the apo SARS-CoV Mpro structure used as a starting point of MD simulations has shown the largest MAV of all the analysed systems. These results suggest that the SARS-CoV main proteases’ binding cavity is highly flexible and changes both in volume and shape significantly alter the ligand binding. This finding indicates a serious obstacle for a classical virtual screening approach and drug design in general. Numerous novel compounds that are considered as potential inhibitors of SARS-CoV, have not reached the stage of clinical trials. The lack of success might be related to the above-mentioned plasticity of the binding cavity. Some of these compounds have been used for docking and virtual screening research aimed not only at SARS-CoV [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>] but also at the novel SARS-CoV-2 [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>–<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. Such an approach focuses mostly on the structural similarity between the binding pockets, but ignores the fact that the actual available binding space differs significantly. In general, a rational drug design can be a very successful tool in the identification of possible inhibitors in cases where the atomic resolution structure of the target protein is known. For a new target, when a highly homologous structure is available, a very logical strategy would be seeking chemically similar compounds or creating derivatives of this inhibitor, and finding those compounds that are predicted to have a higher affinity for the new target structure than the original one. This would be expected to work for SARS-CoV-2 proteins (such as Mpro) using SARS-CoV proteins as templates. However, our in-depth analysis indicates a very different situation taking place, with major shape and size differences emerging due to the binding site flexibility. The continuous effort of Diamond Light Source group [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>] performing massive XChem crystallographic fragment screen against Mpro supports our finding on large binding site flexibility (Supplementary Figure S4). Therefore, repurposing SARS drugs against COVID-19 may not be successful due to major shape and size differences, and despite docking methods, the enhanced sampling should be considered.</p><p id="p-32">The analysis of the water hot-spots shows that the catalytic water hot-spot dominates water distribution inside the binding cavity. The remaining water hot-spots correspond to a much lower water density level and are on the borders of the binding cavity, which suggests a rather hydrophobic or neutral interior of the binding cavity. The MixMD simulations performed with various cosolvents have further confirmed these observations. The largest number and the densest hot-spots were located within the binding cavity and the region essential for Mpros dimerisation [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>], between the II and III domains. The deep insight into the local hot-spots distribution of the various cosolvents underlines the large differences in binding sites plasticity. The smaller binding cavity of the SARS-CoV-2 enlarged significantly in the presence of a highly hydrophobic cosolvent. The benzene hot-spots were detected deep inside the cavity, and also near the C44-P52 loop. In contrast, in the case of the SARS-CoV, benzene hot-spots were located only in the vicinity of the C44-P52 loop. Such a conclusion may also imply that a sufficiently potent inhibitor of SARS-CoV and/or SARS-CoV-2 Mpros needs to be able to open its way to the active site before it can successfully bind to its cavity. These results support the regulatory role of the C44-P52 loop and again alert against unwarranted use of simplified approaches for drug repositioning or docking.</p><p id="p-33">The difficulties in targeting the active site of the Mpros are also explained by evolutionary study and potential mutability analysis. As already pointed out, the C44-P52 loop is likely to regulate the access to the active site by enabling entrance of favourable small molecules and blocking the entry of unfavourable ones. The second important loop, F185-T201, which starts in the vicinity of the binding site and links I and II domains with the III domain contributes significantly to Mpro dimerisation [<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>].</p><p id="p-34">The initial analysis of the effect of the 12 amino acid replacements in SARS-CoV-Mpro on the SARS-CoV-2 Mpro structure stability was expected to provide neutral or stabilising contribution to proteins folding. Indeed, all replacements were found to stabilise the protein’s folding (e.g., H134F: −0.85 kcal/mol) or have almost neutral character (e.g., R88K, S94A, T285A, I286L). The analysis of the potential risk of further Mpro structure evolution within the binding cavity suggests that mutations of residues that contribute to ligand binding or access to the active site are energetically favourable, and likely to occur. Some of the residues that are prone to mutate would provide the inactive enzyme (e.g., the residues forming the catalytic dyad) and therefore could be considered as a blind alley in enzyme evolution, but others (e.g., amino acids from the C44-P52 loop, T45, S46, E47, L50) could significantly modify the inhibitors binding mode of Mpro. The location of prone to mutate residues on the regulatory loop questions the effort for Mpro active site inhibitors design as a long term strategy. However, our results indicate also residues that are energetically unfavourable to mutate (e.g., P39, R40, P52, G143, G146, or L167), which could provide an anchor for successful drug design that can outlast coronavirus’ Mpros variability in future. Alternatively, we would suggest targeting the region between II and III domains, which contributes to the dimer formation.</p></div><div class="section" id="sec-9"><h2 class="">4. Materials and Methods</h2><div id="sec-10" class="subsection"><h3>4.4 Classical MD simulations</h3><p id="p-35">The H++ server [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>] was used to protonate the SARS-CoV-2 and SARS-CoV main proteases’ structures (PDB IDs: 6lu7, 6y2e, 2amq and 1q2w, respectively) using standard parameters and pH 7.4. The missing 4-amino-acids-long loop of the 1q2w model was added using the corresponding loop of the 6lu7 model. Water molecules were placed using the combination of 3D-RISM [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>] and the Placevent algorithm [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. The AMBER 18 LEaP [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>] was used to immerse models in a truncated octahedral box of TIP3P water molecules and prepare the systems for simulation using the ff14SB force field41. Additionally, 4 and 3 Na + ions were added to the SARS-CoV-2 and the SARS-CoV, respectively. AMBER 18 software [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>] was used to run 2 μs simulations of both systems. The minimisation procedure consisted of 2000 steps, involving 1000 steepest descent steps followed by 1000 steps of conjugate gradient energy minimisation, with decreasing constraints on the protein backbone (500, 125 and 25 kcal x mol-1 x Å2) and a final minimisation with no constraints of conjugate gradient energy minimization. Next, gradual heating was performed from 0 K to 300 K over 20 ps using a Langevin thermostat with a temperature coupling constant of 1.0 ps in a constant volume periodic box. Equilibration and production stages were run using the constant pressure periodic boundary conditions for 1 ns with 1 fs step and 50 ns with a 2 fs time step, respectively. Constant temperature was maintained using the weak-coupling algorithm for 50 ns of the production simulation time, with a temperature coupling constant of 1.0 ps. Long-range electrostatic interactions were modelled using the Particle Mesh Ewald method with a non-bonded cut-off of 10 Å and the SHAKE algorithm. The coordinates were saved at an interval of 1 ps.</p></div><div id="sec-11" class="subsection"><h3>4.2. Mixed-solvent MD simulations – cosolvent preparation</h3><p id="p-36">Six different cosolvents: acetonitrile (ACN), benzene (BNZ), dimethylsulfoxide (DMSO), methanol (MEO), phenol (PHN), and urea (URE) were selected to perform the mixed-solvent MD simulations. The chemical structures of cosolvents molecules were downloaded from the ChemSpider database [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>] and a dedicated set of parameters was prepared. Parameters for CAN were adopted from the work by Nikitin and Lyubartsev [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>], and parameters for URE were modified using the 8Mureabox force field to obtain parameters for a single molecule. For the rest of the co-solvent molecules, parameters were prepared using Antechamber [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>] with Gasteiger charges [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>].</p></div><div id="sec-12" class="subsection"><h3>4.3. Mixed-solvent MD simulations – initial configuration</h3><p id="p-37">The Packmol software [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>] was used to build the initial systems consisting of protein (protonated according to the previously described procedure), water, and particular cosolvent molecules. 4 and 3 Na<sup>+</sup> ions were added to the SARS-CoV-2 Mpro and the SARS-CoV Mpros, respectively. It was assumed that the percentage concentration of the cosolvent should not exceed 5% (in the case of ACN, DMSO, MEO, and URE), or should be about 1% in the case of BNZ and PHN phenol (see Supplementary Table S4). The mixed-solvent MD simulation procedures (minimization, equilibration, and production) carried out using the AMBER 18 package were identical as for the classical MD simulations. Only the heating stage differed – it was extended up to 40 ps.</p></div><div id="sec-13" class="subsection"><h3>4.4. Water and cosolvent molecules tracking</h3><p id="p-38">The AQUA-DUCT 1.0 software was used to track water and cosolvent molecules. Molecules of interests, which have entered the so-called <em>Object,</em> defined as 5Å sphere around the centre of geometry of active site residues, namely H41, C145, H164, and D187, were traced within the <em>Scope</em> region, defined as the interior of a convex hull of both COVID-19 Mpro and SARS Mpro Cα atoms. All visualisations were made in PyMol [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>].</p><p id="p-39">AQUA-DUCT was used to analyse maximal accessible volume (MAV), defined as the <em>outer</em> pockets; the <em>outer</em> pocket represents the maximal possible space that could be explored by tracked molecules.</p></div><div id="sec-14" class="subsection"><h3>4.5. Hot-spot identification and selection</h3><p id="p-40">AQUA-DUCT was used to detect regions occupied by molecules of interests, and identify the densest sites using a local solvent distribution approach. Those so-called hot-spots could be calculated as local and/or global, based on the distribution of tracked molecules which visited the <em>Object</em> (local) or just the <em>Scope</em> without visiting the <em>Object</em> (global); here, they are considered as potential binding sites. For clarity, the size of each sphere representing a particular hot-spot has been changed to reflect its occupation level. The selection of the most significant hot-spots consisted of indicating points showing the highest density in particular regions. From the set of points in the space, small groups of hot-spots were determined. Groups were further defined by distance (radius) from each other. Any point found within a distance shorter than the determined radius (3Å) from any other point being part of a given group was counted toward the group. For each so designated group of points, one showing the highest density was chosen as representing the place.</p></div><div id="sec-15" class="subsection"><h3>4.6. Obtaining SARS-CoV-2 Mpro gene sequence</h3><p id="p-41">SARS-CoV-2 Mpro was downloaded from the PDB as a complex with an N3 inhibitor (PDB ID: 6lu7). Tblastn [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] was run based on the protein amino acid sequence. 100% identity with 10055-10972 region of SARS-CoV-2 Mpro complete genome (Sequence ID: MN985262.1) was obtained. Blastx [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>] calculations were run with the selected region, and orf1a polyprotein (NCBI Reference Sequence: YP_009725295.1) amino acid sequence, identical with the previously downloaded SARS-CoV-2 Mpro, was received.</p></div><div id="sec-16" class="subsection"><h3>4.7. Energetic effect of amino acids substitutions</h3><p id="p-42">FoldX software was used to insert substitutions into the structures of SARS-CoV and SARS-CoV-2 Mpros. To analyse the changes in the two structures, 12 single-point mutations were introduced to the SARS structure. Each of the residues in SARS-CoV Mpro was mutated to the respective SARS-CoV-2 Mpro residue, and the difference in total energies of the wild-type SARS-CoV-2 Mpro and the mutant structures were calculated. Then, to investigate further possible mutations of SARS-CoV-2 Mpro, single nucleotide substitutions were introduced to the SARS-CoV-2 main protease gene. If a substitution of a single nucleotide caused translation to a different amino acid than the corresponding residue in the wild-type structure, an appropriate mutation was proposed with FoldX software.</p></div><div id="sec-17" class="subsection"><h3>4.8 Comulator calculations of correlation between amino acids</h3><p id="p-43">SARS-CoV Mpro was downloaded from the PDB (PDB ID: 1q2w). Blast [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>] was run based on the amino acid sequence. As a result, 2643 sequences of viral main proteases similar to chain A SARS-CoV Mpro were obtained. Clustal Omega [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>] was used to prepare an alignment of those sequences. Comulator was then employed to calculate the correlation between amino acids and based on the results, groups of positions in SARS-CoV Mpro sequence were selected whose amino acid occurrences strongly depended on each other.</p></div></div><div class="section" id="sec-18"><h2 class="">5. Conclusions</h2><p id="p-44">In this paper, we reported on molecular dynamics simulations of the main protease (Mpro), whose crystal structures have been released. We compared the Mpro for SARS-CoV-2 with a highly similar SARS-CoV protein. In spite of a high level of sequence similarity between these two homologous proteins, their active sites show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, a detailed analysis of the binding pocket’s conformational changes during simulation time indicates its flexibility and plasticity, which dashes hope for rapid and reliable drug design. Moreover, our findings show the presence of a flexible loop regulating the access to the binding site pocket. A successful inhibitor may need to have an ability to relocate the loop from the entrance to bind to the catalytic pocket. However, mutations leading to changes in the amino acid sequence of the loop, while not affecting the folding of the protein, may result in the putative inhibitors’ inability to access the binding pocket and provide a probable development of drug resistance. To avoid that situation that the future evolution of the Mpros can wipe out all our efforts, we should focus on key functional residues or those whose further mutation will destabilise the protein.</p></div><div class="section" id="sec-20"><h2 class="">Supplementary Materials</h2><p id="p-45">Supplementary materials are available</p></div><div class="section" id="sec-21"><h2 class="">Author Contributions</h2><p id="p-46">Conceptualization, A.G.; methodology, M.B., K.M. A.R and A.S.; software, M.B., K.M. A.R and A.S.; validation, M.B., K.M. and A.G.; formal analysis, M.B., K.M. A.R and A.S.; investigation, M.B., K.M. A.R and A.S.; resources, M.B and K.M.; data curation, M.B and K.M.; writing—original draft preparation, M.B and K.M.; writing—review and editing, J.A.T. and A.G.; visualization, M.B., K.M. A.R and A.G; supervision, A.G.; project administration, A.G.; funding acquisition, J.A.T. and A.G. All authors have read and agreed to the published version of the manuscript.</p></div><div class="section" id="sec-22"><h2 class="">Funding</h2><p id="p-47">KM, MB, AR, AS and AG work was supported by the National Science Centre, Poland, grant no DEC-2013/10/E/NZ1/00649 and DEC-2015/18/M/NZ1/00427. JAT expresses gratitude for research support for this project received from IBM CAS and NSERC (Canada).</p></div><div class="section" id="sec-23"><h2 class="">Conflicts of Interest</h2><p id="p-48">The authors declare no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">The extended version of the manuscript with detailed description of methods and improved supplementary informations.</p></li></ul></div><div class="section glossary"><h2 class="">Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">Mpro<span class="def-item-dt-sep"></span></dt><dd>Main protease<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">CoVs<span class="def-item-dt-sep"></span></dt><dd>Coronaviruses<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">ORFs<span class="def-item-dt-sep"></span></dt><dd>Open reading frames<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">3CLpro<span class="def-item-dt-sep"></span></dt><dd>Chymotrypsin-like cysteine protease<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">S<span class="def-item-dt-sep"></span></dt><dd>Spike surface glycoprotein<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">E<span class="def-item-dt-sep"></span></dt><dd>Small envelope protein<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">M<span class="def-item-dt-sep"></span></dt><dd>Matrix protein<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">N<span class="def-item-dt-sep"></span></dt><dd>Nucleocapsid protein<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">N3<span class="def-item-dt-sep"></span></dt><dd>N-[(5 methylisoxazol-3-yl)carbonyl] alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1--{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">cMD<span class="def-item-dt-sep"></span></dt><dd>Classical molecular dynamics simulations<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">MixMD<span class="def-item-dt-sep"></span></dt><dd>Mixed-solvent molecular dynamics simulations<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">PDB<span class="def-item-dt-sep"></span></dt><dd>Protein Data Bank<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">MAV<span class="def-item-dt-sep"></span></dt><dd>Maximal accessible volume<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">ACN<span class="def-item-dt-sep"></span></dt><dd>Acetonitrile<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">BNZ<span class="def-item-dt-sep"></span></dt><dd>Benzene<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">DMSO<span class="def-item-dt-sep"></span></dt><dd>Dimethylsulfoxide<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">MEO<span class="def-item-dt-sep"></span></dt><dd>Methanol<span class="def-item-dd-sep"></span></dd><dt id="def-item-18">PHN<span class="def-item-dt-sep"></span></dt><dd>Phenol<span class="def-item-dd-sep"></span></dd><dt id="def-item-19">URE<span class="def-item-dt-sep"></span></dt><dd>Urea<span class="def-item-dd-sep"></span></dd><dt id="def-item-20">CMA<span class="def-item-dt-sep"></span></dt><dd>Correlated mutation analysis</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.2" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2020</span>, <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.3" data-doi="10.3390/v2081803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P.C.Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.K.P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.-Y.</span></span> <span class="cit-article-title">Coronavirus Genomics and Bioinformatics Analysis</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">2</span>, <span class="cit-fpage">1804</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D1804%26rft.epage%253D1820%26rft.atitle%253DCoronavirus%2Bgenomics%2Band%2Bbioinformatics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv2081803%26rft_id%253Dinfo%253Apmid%252F21994708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v2081803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">X.-Q.</span></span> <span class="cit-article-title">Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">5</span>, <span class="cit-fpage">17155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D5%26rft.spage%253D17155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.5" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span> <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">17</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.6" data-doi="10.1007/978-1-4939-2438-7_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Coronaviruses: An Overview of Their Replication and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">1282</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1282%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2438-7_1%26rft_id%253Dinfo%253Apmid%252F25720466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2438-7_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25720466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Origin and evolution of the 2019 novel coronavirus</span>. <abbr class="cit-jnl-abbrev">Clin. Infect. Dis</abbr>. <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">From SARS to MERS, Thrusting Coronaviruses into the Spotlight</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">11</span>, <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.9" data-doi="10.1128/JVI.02114-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">82</span>, <span class="cit-fpage">2515</span>–<span class="cit-lpage">2527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02114-07%26rft_id%253Dinfo%253Apmid%252F18094151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzUvMjUxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDIuMjcuOTY4MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.11" data-doi="10.1038/nrd.2015.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zumla</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J.F.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Azhar</span>,  <span class="cit-name-given-names">E.I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">D.S.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.-Y.</span></span> <span class="cit-article-title">Coronaviruses — drug discovery and therapeutic options</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Drug Discov</abbr>. <span class="cit-pub-date">2016</span>, <span class="cit-vol">15</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BDrug%2BDiscov%26rft.volume%253D15%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd.2015.37%26rft_id%253Dinfo%253Apmid%252F26868298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd.2015.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26868298&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morse</span>,  <span class="cit-name-given-names">J.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lalonde</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">ChemBioChem</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-fpage">cbic.202000047</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of Mpro from COVID-19 virus and discovery of its inhibitors</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.14" data-doi="10.1128/JVI.02612-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Without Its N-Finger, the Main Protease of Severe Acute Respiratory Syndrome Coronavirus Can Form a Novel Dimer through Its C-Terminal Domain</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">82</span>, <span class="cit-fpage">4227</span>–<span class="cit-lpage">4234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02612-07%26rft_id%253Dinfo%253Apmid%252F18305043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzkvNDIyNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDIuMjcuOTY4MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ton</span>,  <span class="cit-name-given-names">A.-T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gentile</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hsing</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ban</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cherkasov</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Rapid Identification of Potential Inhibitors of SARS-CoV-2 Main Protease by Deep Docking of 1.3 Billion Compounds</span>. <abbr class="cit-jnl-abbrev">ChemRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mu</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968008v3.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.-J.</span></span> <span class="cit-article-title">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968008v3.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">Q.</span></span> <span class="cit-article-title">Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968008v3.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">D.D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">G.-W.</span></span> <span class="cit-article-title">Potentially highly potent drugs for 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968008v3.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Talluri</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Virtual High Throughput Screening Based Prediction of Potential Drugs for COVID-19 8 March 2020</span>. <abbr class="cit-jnl-abbrev">Preprints</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yiu</span>,  <span class="cit-name-given-names">C.-P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">K.-Y.</span></span> <span class="cit-article-title">Prediction of the 2019-nCoV 3C-like Protease (3CLpro) Structure: Virtual Screening Reveals Velpatasvir, Ledipasvir, and Other Drug Repurposing Candidates</span>. <abbr class="cit-jnl-abbrev">ChemRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fischer</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sellner</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Neranjan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lill</span>,  <span class="cit-name-given-names">M.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Smieško</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Inhibitors for Novel Coronavirus Protease Identified by Virtual Screeningof 687 Million Compounds</span>. <abbr class="cit-jnl-abbrev">ChemRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968008v3.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gentile</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Patamia</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Scala</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sciortino</span>,  <span class="cit-name-given-names">M.T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Piperno</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Inhibitors of SARS-CoV-2 Main Protease from a Library of Marine Natural Products: A Virtual Screening and Molecular Modeling Study</span>. <abbr class="cit-jnl-abbrev">Preprints</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adem</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eyupoglu</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sarfraz</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rasul</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ali</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Identification of potent COVID-19 main protease (Mpro) inhibitors from natural polyphenols: An in silico strategy unveils a hope against CORONA</span>. <abbr class="cit-jnl-abbrev">Preprints</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.25" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berman</span>,  <span class="cit-name-given-names">H.M.</span></span> <span class="cit-article-title">The Protein Data Bank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2000</span>, <span class="cit-vol">28</span>, <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Curth</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sauerhering</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rox</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved a-ketoamide inhibitors</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-pub-date">2020</span>, <span class="cit-fpage">eabb3405</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.27" data-doi="10.1021/bi0361766"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bacha</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barrila</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Velazquez-Campoy</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leavitt</span>,  <span class="cit-name-given-names">S.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Freire</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Identification of Novel Inhibitors of the SARS Coronavirus Main Protease 3CL pro +</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">43</span>, <span class="cit-fpage">4906</span>–<span class="cit-lpage">4912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D43%26rft.spage%253D4906%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0361766%26rft_id%253Dinfo%253Apmid%252F15109248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0361766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15109248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.28" data-doi="10.1126/science.1085658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Coronavirus Main Proteinase (3CLpro) Structure: Basis for Design of Anti-SARS Drugs</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-pub-date">2003</span>, <span class="cit-vol">300</span>, <span class="cit-fpage">1763</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085658%26rft_id%253Dinfo%253Apmid%252F12746549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wMi4yNy45NjgwMDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitusińska</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Raczyńska</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bzówka</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bagrowska</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Góra</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Applications of water molecules for analysis of macromolecule properties</span>. <abbr class="cit-jnl-abbrev">Comput. Struct. Biotechnol. J</abbr>. <span class="cit-pub-date">2020</span>, <span class="cit-vol">18</span>, <span class="cit-fpage">355</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStruct.%2BBiotechnol.%2BJ%26rft.volume%253D18%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magdziarz</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mitusińska</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bzówka</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Raczyńska</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stańczak</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Banas</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bagrowska</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Góra</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">AQUA-DUCT 1.0: structural and functional analysis of macromolecules from an intramolecular voids perspective</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.31" data-doi="10.1074/jbc.M109.095851"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Teng</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">285</span>, <span class="cit-fpage">28134</span>–<span class="cit-lpage">28140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.095851%26rft_id%253Dinfo%253Apmid%252F20489209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMzYvMjgxMzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNi8yMDIwLjAyLjI3Ljk2ODAwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuipers</span>,  <span class="cit-name-given-names">R.K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Joosten</span>,  <span class="cit-name-given-names">H.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van Berkel</span>,  <span class="cit-name-given-names">W.J.H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leferink</span>,  <span class="cit-name-given-names">N.G.H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rooijen</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ittmann</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van Zimmeren</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jochens</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bornscheuer</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vriend</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">3DM: Systematic analysis of heterogeneous superfamily data to discover protein functionalities</span>. <abbr class="cit-jnl-abbrev">Proteins Struct. Funct. Bioinforma</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">78</span>, <span class="cit-fpage">2101</span>–<span class="cit-lpage">2113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%2BStruct.%2BFunct.%2BBioinforma%26rft.volume%253D78%26rft.spage%253D2101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mitusińska</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Raedts</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Almourfi</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Joosten</span>,  <span class="cit-name-given-names">H.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hendriks</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sedelnikova</span>,  <span class="cit-name-given-names">S.E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kengen</span>,  <span class="cit-name-given-names">S.W.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hagen</span>,  <span class="cit-name-given-names">W.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Góra</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Distant Non-Obvious Mutations Influence the Activity of a Hyperthermophilic Pyrococcus furiosus Phosphoglucose Isomerase</span>. <abbr class="cit-jnl-abbrev">Biomolecules</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">9</span>, <span class="cit-fpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomolecules%26rft.volume%253D9%26rft.spage%253D212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.34" data-doi="10.1093/nar/gki387"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schymkowitz</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Borg</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stricher</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nys</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rousseau</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Serrano</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The FoldX web server: an online force field</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">33</span>, <span class="cit-fpage">W382</span>–<span class="cit-lpage">W388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSchymkowitz%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW382%26rft.epage%253DW388%26rft.atitle%253DThe%2BFoldX%2Bweb%2Bserver%253A%2Ban%2Bonline%2Bforce%2Bfield%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki387%26rft_id%253Dinfo%253Apmid%252F15980494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400077&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klausen</span>,  <span class="cit-name-given-names">M.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jespersen</span>,  <span class="cit-name-given-names">M.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">K.K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jurtz</span>,  <span class="cit-name-given-names">V.I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sønderby</span>,  <span class="cit-name-given-names">C.K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sommer</span>,  <span class="cit-name-given-names">M.O.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Winther</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">NetSurfP-2.0: Improved prediction of protein structural features by integrated deep learning</span>. <abbr class="cit-jnl-abbrev">Proteins Struct. Funct. Bioinforma</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">87</span>, <span class="cit-fpage">520</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%2BStruct.%2BFunct.%2BBioinforma%26rft.volume%253D87%26rft.spage%253D520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitusińska</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Magdziarz</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bzówka</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stańczak</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gora</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Exploring Solanum tuberosum Epoxide Hydrolase Internal Architecture by Water Molecules Tracking</span>. <abbr class="cit-jnl-abbrev">Biomolecules</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">8</span>, <span class="cit-fpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomolecules%26rft.volume%253D8%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.37" data-doi="10.1107/S1399004715003521"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Needle</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lountos</span>,  <span class="cit-name-given-names">G.T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Waugh</span>,  <span class="cit-name-given-names">D.S.</span></span> <span class="cit-article-title">Structures of the Middle East respiratory syndrome coronavirus 3C-like protease reveal insights into substrate specificity</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr. Sect. D Biol. Crystallogr</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">71</span>, <span class="cit-fpage">1102</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr.%2BSect.%2BD%2BBiol.%2BCrystallogr%26rft.volume%253D71%26rft.spage%253D1102%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS1399004715003521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S1399004715003521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rox</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of a-Ketoamide Inhibitors</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.39" data-doi="10.1093/jb/mvq071"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">M.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">W.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">J.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">L.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">G.-G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">H.-P.</span></span> <span class="cit-article-title">Essential covalent linkage between the chymotrypsin-like domain and the extra domain of the SARS-CoV main protease</span>. <abbr class="cit-jnl-abbrev">J. Biochem</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">148</span>, <span class="cit-fpage">349</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiochem%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjb%252Fmvq071%26rft_id%253Dinfo%253Apmid%252F20587646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jb/mvq071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20587646&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.40" data-doi="10.1093/emboj/cdf327"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>. <span class="cit-pub-date">2002</span>, <span class="cit-vol">21</span>, <span class="cit-fpage">3213</span>–<span class="cit-lpage">3224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DAnand%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D3213%26rft.epage%253D3224%26rft.atitle%253DStructure%2Bof%2Bcoronavirus%2Bmain%2Bproteinase%2Breveals%2Bcombination%2Bof%2Ba%2Bchymotrypsin%2Bfold%2Bwith%2Ban%2Bextra%2B%257Balpha%257D-helical%2Bdomain%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf327%26rft_id%253Dinfo%253Apmid%252F12093723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTMvMzIxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDIuMjcuOTY4MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.41" data-doi="10.1371/journal.pone.0101941"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Dynamically-Driven Enhancement of the Catalytic Machinery of the SARS 3C-Like Protease by the S284-T285-I286/A Mutations on the Extra Domain</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">9</span>, <span class="cit-fpage">e101941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101941%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101941%26rft_id%253Dinfo%253Apmid%252F25036652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25036652&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jeyachandran</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">N.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">C.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">S.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.-S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">Y.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">M.-H.</span></span> <span class="cit-article-title">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span>. <abbr class="cit-jnl-abbrev">Mol. Biosyst</abbr>. <span class="cit-pub-date">2016</span>, <span class="cit-vol">12</span>, <span class="cit-fpage">59</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiosyst%26rft.volume%253D12%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dayer</span>,  <span class="cit-name-given-names">M.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Taleb-Gassabi</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dayer</span>,  <span class="cit-name-given-names">M.S.</span></span> <span class="cit-article-title">Lopinavir; A Potent Drug against Coronavirus Infection: Insight from Molecular Docking Study</span>. <abbr class="cit-jnl-abbrev">Arch. Clin. Infect. Dis</abbr>. <span class="cit-pub-date">2017</span>, <span class="cit-vol">12</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Resnick</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bradley</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Douangamath</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Krojer</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sethi</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Geurink</span>,  <span class="cit-name-given-names">P.P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aimon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Amitai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bellini</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">141</span>, <span class="cit-fpage">8951</span>–<span class="cit-lpage">8968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc%26rft.volume%253D141%26rft.spage%253D8951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.45" data-doi="10.1093/nar/gks375"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anandakrishnan</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aguilar</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Onufriev</span>,  <span class="cit-name-given-names">A. V.</span></span> <span class="cit-article-title">H++ 3.0: automating pK prediction and the preparation of biomolecular structures for atomistic molecular modeling and simulations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">40</span>, <span class="cit-fpage">W537</span>–<span class="cit-lpage">W541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks375%26rft_id%253Dinfo%253Apmid%252F22570416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570416&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306670900088&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.46" data-doi="10.1021/ct900460m"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luchko</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gusarov</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Roe</span>,  <span class="cit-name-given-names">D.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Simmerling</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Case</span>,  <span class="cit-name-given-names">D.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tuszynski</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kovalenko</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Three-Dimensional Molecular Theory of Solvation Coupled with Molecular Dynamics in Amber</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">6</span>, <span class="cit-fpage">607</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BChem%2BTheory%2BComput%26rft.aulast%253DLuchko%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D607%26rft.epage%253D624%26rft.atitle%253DThree-dimensional%2Bmolecular%2Btheory%2Bof%2Bsolvation%2Bcoupled%2Bwith%2Bmolecular%2Bdynamics%2Bin%2BAmber.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct900460m%26rft_id%253Dinfo%253Apmid%252F20440377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct900460m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20440377&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sindhikara</span>,  <span class="cit-name-given-names">D.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yoshida</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hirata</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Placevent: An algorithm for prediction of explicit solvent atom distribution-Application to HIV-1 protease and F-ATP synthase</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">33</span>, <span class="cit-fpage">1536</span>–<span class="cit-lpage">1543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DSindhikara%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D33%26rft.issue%253D18%26rft.spage%253D1536%26rft.epage%253D1543%26rft.atitle%253DPlacevent%253A%2Ban%2Balgorithm%2Bfor%2Bprediction%2Bof%2Bexplicit%2Bsolvent%2Batom%2Bdistribution-application%2Bto%2BHIV-1%2Bprotease%2Band%2BF-ATP%2Bsynthase.%26rft_id%253Dinfo%253Apmid%252F22522665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22522665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Case</span>,  <span class="cit-name-given-names">D.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ben-Shalom</span>,  <span class="cit-name-given-names">I.Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brozell</span>,  <span class="cit-name-given-names">S.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cerutti</span>,  <span class="cit-name-given-names">D.S.</span></span>, C.T.E.; <span class="cit-auth"><span class="cit-name-surname">Cruzeiro</span>,  <span class="cit-name-given-names">V.W.D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Darden</span>,  <span class="cit-name-given-names">T.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Duke</span>,  <span class="cit-name-given-names">R.E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ghoreishi</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gilson</span>,  <span class="cit-name-given-names">M.K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gohlke</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">AMBER 2018</abbr> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.49" data-doi="10.1021/ed100697w"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pence</span>,  <span class="cit-name-given-names">H.E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">ChemSpider: An Online Chemical Information Resource</span>. <abbr class="cit-jnl-abbrev">J. Chem. Educ</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">87</span>, <span class="cit-fpage">1123</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BEduc%26rft.volume%253D87%26rft.spage%253D1123%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fed100697w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ed100697w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000282868100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.50" data-doi="10.1002/jcc.20721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nikitin</span>,  <span class="cit-name-given-names">A.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lyubartsev</span>,  <span class="cit-name-given-names">A.P.</span></span> <span class="cit-article-title">New six-site acetonitrile model for simulations of liquid acetonitrile and its aqueous mixtures</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. <span class="cit-pub-date">2007</span>, <span class="cit-vol">28</span>, <span class="cit-fpage">2020</span>–<span class="cit-lpage">2026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DNikitin%26rft.auinit1%253DA.%2BM.%26rft.volume%253D28%26rft.issue%253D12%26rft.spage%253D2020%26rft.epage%253D2026%26rft.atitle%253DNew%2Bsix-site%2Bacetonitrile%2Bmodel%2Bfor%2Bsimulations%2Bof%2Bliquid%2Bacetonitrile%2Band%2Bits%2Baqueous%2Bmixtures.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20721%26rft_id%253Dinfo%253Apmid%252F17450554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17450554&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.51" data-doi="10.1016/j.jmgm.2005.12.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kollman</span>,  <span class="cit-name-given-names">P.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Case</span>,  <span class="cit-name-given-names">D.A.</span></span> <span class="cit-article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <abbr class="cit-jnl-abbrev">J. Mol. Graph. Model</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">25</span>, <span class="cit-fpage">247</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bgraphics%2B%2526%2Bmodelling%26rft.stitle%253DJ%2BMol%2BGraph%2BModel%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D247%26rft.epage%253D260%26rft.atitle%253DAutomatic%2Batom%2Btype%2Band%2Bbond%2Btype%2Bperception%2Bin%2Bmolecular%2Bmechanical%2Bcalculations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmgm.2005.12.005%26rft_id%253Dinfo%253Apmid%252F16458552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmgm.2005.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16458552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241713000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.52" data-doi="10.1016/0040-4020(80)80168-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gasteiger</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marsili</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Iterative partial equalization of orbital electronegativity—a rapid access to atomic charges</span>. <abbr class="cit-jnl-abbrev">Tetrahedron</abbr> <span class="cit-pub-date">1980</span>, <span class="cit-vol">36</span>, <span class="cit-fpage">3219</span>–<span class="cit-lpage">3228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTetrahedron%26rft.volume%253D36%26rft.spage%253D3219%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-4020%252880%252980168-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-4020(80)80168-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1980KW36200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.53" data-doi="10.1002/jcc.21224"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Andrade</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Birgin</span>,  <span class="cit-name-given-names">E.G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martínez</span>,  <span class="cit-name-given-names">J.M.</span></span> <span class="cit-article-title">PACKMOL: A package for building initial configurations for molecular dynamics simulations</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. <span class="cit-pub-date">2009</span>, <span class="cit-vol">30</span>, <span class="cit-fpage">2157</span>–<span class="cit-lpage">2164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DMartinez%26rft.auinit1%253DL.%26rft.volume%253D30%26rft.issue%253D13%26rft.spage%253D2157%26rft.epage%253D2164%26rft.atitle%253DPACKMOL%253A%2Ba%2Bpackage%2Bfor%2Bbuilding%2Binitial%2Bconfigurations%2Bfor%2Bmolecular%2Bdynamics%2Bsimulations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.21224%26rft_id%253Dinfo%253Apmid%252F19229944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.21224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19229944&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269095700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delano</span>,  <span class="cit-name-given-names">W.L.</span></span> <span class="cit-article-title">PyMOL: An Open-Source Molecular Graphics Tool</span>. <abbr class="cit-jnl-abbrev">Ccp4 Newslett Protein Crystallogr</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">40</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.55" data-doi="10.1186/1741-7007-4-41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gertz</span>,  <span class="cit-name-given-names">E.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.-K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Agarwala</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schäffer</span>,  <span class="cit-name-given-names">A.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S.F.</span></span> <span class="cit-article-title">Composition-based statistics and translated nucleotide searches: Improving the TBLASTN module of BLAST</span>. <abbr class="cit-jnl-abbrev">BMC Biol</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">4</span>, <span class="cit-fpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DGertz%26rft.auinit1%253DE.%2BM.%26rft.volume%253D4%26rft.spage%253D41%26rft.epage%253D41%26rft.atitle%253DComposition-based%2Bstatistics%2Band%2Btranslated%2Bnucleotide%2Bsearches%253A%2Bimproving%2Bthe%2BTBLASTN%2Bmodule%2Bof%2BBLAST.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-4-41%26rft_id%253Dinfo%253Apmid%252F17156431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-4-41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17156431&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.56" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Camacho</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Coulouris</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Avagyan</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bealer</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Madden</span>,  <span class="cit-name-given-names">T.L.</span></span> <span class="cit-article-title">BLAST+: architecture and applications</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">10</span>, <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.57" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S.F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gish</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">E.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D.J.</span></span> <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">1990</span>, <span class="cit-vol">215</span>, <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.02.27.968008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.968008v3.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Higgins</span>,  <span class="cit-name-given-names">D.G.</span></span> <span class="cit-article-title">Clustal Omega, Accurate Alignment of Very Large Numbers of Sequences</span>. In <abbr class="cit-jnl-abbrev">Multiple Sequence Alignment Methods</abbr>; <span class="cit-pub-date">2014</span>; pp. <span class="cit-fpage">105</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.27.968008v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.24.006197v2" title="In-Silico evidence for two receptors based strategy of SARS-CoV-2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.16.994236v2" title="The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 06, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.27.968008v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.27.968008v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.27.968008v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.27.968008v3" class="active" data-icon-position="" data-hide-link-title="0">Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1231086" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-88CFLxeBKhZ1EfD0CQcPWEmfdFU31PD_xm9Sw6bmYIs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231086" id="node1231086--2" data-pisa="biorxiv;2020.02.27.968008v3" data-pisa-master="biorxiv;2020.02.27.968008" data-seqnum="1231086" data-apath="/biorxiv/early/2020/04/06/2020.02.27.968008.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bzówka</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karolina</span> <span class="nlm-surname">Mitusińska</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Agata</span> <span class="nlm-surname">Raczyńska</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Aleksandra</span> <span class="nlm-surname">Samol</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jack A.</span> <span class="nlm-surname">Tuszyński</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Artur</span> <span class="nlm-surname">Góra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.27.968008; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.27.968008 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;title=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;title=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;text=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;title=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;t=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;title=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;title=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231086" id="node1231086--3" data-pisa="biorxiv;2020.02.27.968008v3" data-pisa-master="biorxiv;2020.02.27.968008" data-seqnum="1231086" data-apath="/biorxiv/early/2020/04/06/2020.02.27.968008.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural and Evolutionary Analysis Indicate that the SARS-CoV-2 Mpro is an Inconvenient Target for Small-Molecule Inhibitors Design</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bzówka</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karolina</span> <span class="nlm-surname">Mitusińska</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Agata</span> <span class="nlm-surname">Raczyńska</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Aleksandra</span> <span class="nlm-surname">Samol</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jack A.</span> <span class="nlm-surname">Tuszyński</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Artur</span> <span class="nlm-surname">Góra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.27.968008; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.27.968008 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Bz%C3%B3wka%2C+Maria&amp;rft.au=Mitusi%C5%84ska%2C+Karolina&amp;rft.au=Raczy%C5%84ska%2C+Agata&amp;rft.au=Samol%2C+Aleksandra&amp;rft.au=Tuszy%C5%84ski%2C+Jack+A.&amp;rft.au=G%C3%B3ra%2C+Artur"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1231086/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1231086/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1231086/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1231086/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1231086/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1231086/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1231086/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1231086/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1231086/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1231086/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1231086/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1231086/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;count=horizontal&amp;via=&amp;text=Structural%20and%20Evolutionary%20Analysis%20Indicate%20that%20the%20SARS-CoV-2%20Mpro%20is%20an%20Inconvenient%20Target%20for%20Small-Molecule%20Inhibitors%20Design&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968008v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.27.968008v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/molecular-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Molecular Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
