<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Antimicrob Agents</span><span class="iso-abbrev" title="iso-abbrev">Int. J. Antimicrob. Agents</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Antimicrobial Agents</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0924-8579</span><span class="issn-epub" tagx="issn" title="issn-epub">1872-7913</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier B.V. and International Society of Chemotherapy.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7128678">7128678</a></span><span class="publisher-id" title="publisher-id">S0924-8579(20)30110-2</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.ijantimicag.2020.105960">10.1016/j.ijantimicag.2020.105960</a></span><span class="publisher-id" title="publisher-id">105960</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Structural and molecular modelling studies reveal a new mechanism of action of chloroquine and hydroxychloroquine against SARS-CoV-2 infection</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">Jacques</span></span><span class="email" tagx="email" title="email">jacques.fantini@univ-amu.fr</span><a href="#aff0001">a</a><a href="#aff0002">b</a><a href="#cor0001">�?�</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Scala</span><span class="given-names" tagx="given-names" title="given-names">Coralie</span></span><a href="#aff0003">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chahinian</span><span class="given-names" tagx="given-names" title="given-names">Henri</span></span><a href="#aff0001">a</a><a href="#aff0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">Nouara</span></span><a href="#aff0001">a</a><a href="#aff0002">b</a></span><span class="citation_author_institution" id="aff0001">[a], </span><span class="citation_author_institution" id="aff0002">[b], </span><span class="citation_author_institution" id="aff0003">[c], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">�?�</span>Corresponding author. Address: INSERM UMR_S 1072, Boulevard Pierre Dramard, 13015 Marseille, France. <span class="email" tagx="email" title="email">jacques.fantini@univ-amu.fr</span></div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-5</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="volume" tagx="volume" title="volume">55</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">105960</span><span class="lpage" tagx="lpage" title="lpage">105960</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier B.V. and International Society of Chemotherapy</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">•</span><p id="para0001">The COVID�?19 pandemic caused by SARS�?CoV�?2 poses a global health emergency.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="para0002">Promising results suggest that chloroquine could stop the spread of SARS�?CoV�?2.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="para0003">In-silico studies confirm the antiviral properties of chloroquine.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="para0004">New mechanism of action of chloroquine elucidated.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="para0005">Hydroxychloroquine is more potent than chloroquine.</p> </li> </ul> <p /> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The recent emergence of the novel pathogenic SARS-coronavirus 2 (SARS-CoV-2) is responsible for a worldwide pandemic. Given the global health emergency, drug repositioning is the most reliable option to design an efficient therapy for infected patients without delay. The first step of the viral replication cycle [i.e. attachment to the surface of respiratory cells, mediated by the spike (S) viral protein] offers several potential therapeutic targets. The S protein uses the angiotension-converting enzyme-2 (ACE-2) receptor for entry, but also sialic acids linked to host cell surface gangliosides. Using a combination of structural and molecular modelling approaches, this study showed that chloroquine (CLQ), one of the drugs currently under investigation for SARS-CoV-2 treatment, binds sialic acids and gangliosides with high affinity. A new type of ganglioside-binding domain at the tip of the N-terminal domain of the SARS-CoV-2 S protein was identified. This domain (111–158), which is fully conserved among clinical isolates worldwide, may improve attachment of the virus to lipid rafts and facilitate contact with the ACE-2 receptor. This study showed that, in the presence of CLQ [or its more active derivative, hydroxychloroquine (CLQ-OH)], the viral S protein is no longer able to bind gangliosides. The identification of this new mechanism of action of CLQ and CLQ-OH supports the use of these repositioned drugs to cure patients infected with SARS-CoV-2. The in-silico approaches used in this study might also be used to assess the efficiency of a broad range of repositioned and/or innovative drug candidates before clinical evaluation.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Graphical Abstract</div> <p> </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> <p /> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Coronavirus</div> <div class="kwd" title="kwd">
Pandemic</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
Ganglioside</div> <div class="kwd" title="kwd">
Spike</div> <div class="kwd" title="kwd">
Chloroquine</div> </div> </div> <p id="misc0001">Editor: Jean-Marc Rolain</p> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="para0006">The recent emergence of the novel pathogenic SARS-coronavirus 2 (SARS-CoV-2) is responsible for a global pandemic <a href="#bib0001">[1]</a>, and there is an urgent need to identify active antiviral agents. Given the global health emergency, drug repositioning is the most reliable option to design an efficient therapy for infected patients without delay <a href="#bib0002">[2</a>,<a href="#bib0003">3]</a>. Several drugs have already been tested, among which chloroquine (CLQ), a well-known antimalarial drug, is one of the most promising as it has shown apparent efficacy in the treatment of COVID-19-associated pneumonia in recent clinical studies <a href="#bib0004">[4]</a>. However, the mechanism of action of CLQ against SARS-CoV-2 is unclear as the drug seems to exert a broad range of potential antiviral effects <a href="#bib0005">[5]</a>. Therefore, although CLQ is classically considered as an inhibitor of endocytic pathways through elevation of endosomal pH <a href="#bib0006">[6]</a>, its detailed molecular mechanism of action as an antiviral compound remains unclear <a href="#bib0003">[3</a>,<a href="#bib0005">5]</a>. Interestingly, CLQ has been shown to interfere with the terminal glycosylation of angiotensin-converting enzyme-2 (ACE-2) <a href="#bib0007">[7]</a>, which acts as a plasma membrane receptor for both SARS-CoV <a href="#bib0008">[8]</a> and SARS-CoV-2 <a href="#bib0009">[9]</a>, and CLQ could act at several steps of the coronavirus replication cycle <a href="#bib0007">[7]</a>. These data suggest the interesting and mostly unexplored possibility that CLQ could prevent viral attachment through a direct effect on host cell surface molecules.</p> <p id="para0007">One important characteristic of human coronaviruses is that besides their protein membrane receptor, they also depend upon sialic-acid-containing glycoproteins and gangliosides that act as primary attachment factors along the respiratory tract <a href="#bib0010">[10]</a>. The present study used a combination of structural and molecular modelling approaches <a href="#bib0011">[11]</a> to investigate the potential interaction between CLQ and sialic acids. A ganglioside-binding site in the N-terminal domain (NTD) of the spike (S) glycoprotein of SARS-CoV-2 was identified, and CLQ was shown to be a potential blocker of the S–ganglioside interaction which occurs in the first step of the viral replication cycle (i.e. attachment to the surface of respiratory cells, mediated by the S protein). In addition, the antiviral potential of CLQ and its derivative hydroxychloroquine (CLQ-OH) against SARS-CoV-2 were compared. Overall, this study found that CLQ and CLQ-OH may be used to fight pathogenic human coronaviruses <a href="#bib0003">[3]</a>, <a href="#bib0004">[4]</a>, <a href="#bib0005">[5]</a>, <a href="#bib0006">[6]</a>, including SARS-CoV-2 <a href="#bib0012">[12]</a>.</p> </div> <div class="materialsandmethods" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Materials and methods</div> <p id="para0008">In-silico analyses were performed using Hyperchem and Molegro Molecular viewer as described <a href="#bib0011">[11</a>,<a href="#bib0013">13</a>,<a href="#bib0014">14]</a>. The initial coordinates of GM1 were obtained from CHARMM-GUI Glycolipid Modeler (<a href="http://www.charmmgui.org/?doc=input/glycolipid">http://www.charmmgui.org/?doc=input/glycolipid</a>; <a href="#bib0015">[15]</a>), which uses the internal coordinate information of common glycosidic torsion angle values, orients the ganglioside perpendicular to the membrane, and performs Langevin dynamics with a cylindrical restraint potential to keep the whole GM1 molecule cylindrical, especially the membrane-embedded ceramide part. In the next step, the ganglioside was included in a periodic box solvated with 1128 water molecules. The system was energy-minimized six times, switching between runs using the steepest descent gradients and runs using Polak–Ribière conjugate gradients until convergence to machine precision. The ganglioside was subsequently merged with the NTD domain of SARS-CoV-2 S protein as obtained from pdb file # 6VSB <a href="#bib0016">[16]</a>. Initial conditions corresponded to minimized structures obtained with the Polak–Ribière algorithm. Docked complexes were subsequently submitted to iterative cycles of molecular dynamics using the CHARMM36 force field optimized for carbohydrates <a href="#bib0017">[17]</a>. Interaction energies were calculated from stable complexes using the Ligand Energy Inspector function of Molegro <a href="#bib0013">[13]</a>.</p> <p id="para0009">N-acetylneuraminic acid (Neu5Ac) was generated with the Hyperchem database. 9-O-acetyl-N-acetylneuraminic acid (9-O-SIA) was retrieved from pdb file 6Q06 <a href="#bib0018">[18]</a>. CLQ is N-(7-chloroquinolin-4-yl)-N,N-diéthyl-pentane-1,4-diamine. Its three-dimensioanl structure was retrieved from pdb file # 4V2O <a href="#bib0019">[19]</a>. CLQ-OH is (RS)-2-[{4-[(7-chloroquinolin-4-yl)amino]pentyl}(ethyl)amino]ethanol. CLQ-OH was generated by hydroxylation of CLQ with Hyperchem. Both CLQ and CLQ-OH were energy-minimized and merged with water molecules as described below.</p> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Results</div> <div class="structuralandconformationalanalysisofclqandclq-ohinwater" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
Structural and conformational analysis of CLQ and CLQ-OH in water</div> <p id="para0010">The chemical structures of CLQ and CLQ-OH are shown in <a href="#fig0001">Fig. 1</a> (a,b). The only difference between the two molecules is the presence of a terminal hydroxyl group in CLQ-OH. This OH group has a marked influence on the conformation and water-solubilization properties of the drug. CLQ-OH may adopt a wide range of conformations, the most stable being the extended one shown in <a href="#fig0001">Fig. 1</a>(c). When immersed in a periodic box of 31.5 Å<sup>2</sup> with 1042 water molecules, the system reached, at equilibrium, an estimated energy of interaction of -92 kJ.mol<sup>−1</sup>, accounting for 56 water molecules solvating CLQ-OH <a href="#fig0001">Fig. 1</a>(d). In contrast, due to an intramolecular hydrophobic effect, CLQ appeared to be more condensed than CLQ-OH <a href="#fig0001">Fig. 1</a>(e). At equilibrium, CLQ was surrounded by 58 water molecules with an energy of interaction of -79 kJ.mol<sup>−1</sup><a href="#fig0001">Fig. 1</a>(f). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Chemical structure of chloroquine (CLQ) and hydroxychloroquine (CLQ-OH). (a) CLQ. (b) CLQ-OH. (c) CLQ-OH extended conformer. (d) CLQ-OH in water. (e) Typical condensed conformer of CLQ. (f) CLQ in water. The molecules in (c–f) are shown in either tube or sphere rendering (carbon, green; nitrogen, blue; oxygen, red; hydrogen, white). In (c) and (e), the chlorine atom of CLQ and CLQ-OH is indicated by an arrow.</p>   </div> <p /> <p id="para0011">These water-compatible conformations of CLQ and CLQ-OH were used as initial conditions for studying the interaction of these drugs with sialic acids and gangliosides.</p> </div> <div class="sialicacidsasmoleculartargetsofclqandclq-oh" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Sialic acids as molecular targets of CLQ and CLQ-OH</div> <p id="para0012">Neu5Ac is the predominant sialic acid found in human glycoproteins and gangliosides. When CLQ was merged with Neu5Ac, a quasi-instantaneous fit occurred between the two molecules, whose global shapes in water are geometrically complementary <a href="#fig0002">Fig. 2</a> (a). This is particularly obvious in the views of the CLQ–Neu5Ac complex in mixed surface/balls and sticks rendition <a href="#fig0002">Fig. 2</a>(a,b). The interaction was driven by the positioning of the negative charge of the carboxylate group of Neu5Ac and one of the two cationic charges of CLQ (pKa 10.2) <a href="#fig0002">Fig. 2</a>(c). The energy of interaction of this complex was estimated to be -47 kJ.mol<sup>−1</sup>. As coronaviruses preferentially interact with 9-O-acetyl-N-acetylneuraminic acid (9-O-SIA) <a href="#bib0010">[10]</a>, this study used a similar molecular modelling approach to assess whether CLQ could also interact with this specific sialic acid. A good fit between CLQ and 9-O-SIA was obtained <a href="#fig0002">Fig. 2</a>(d–f), with an energy of interaction of -45 kJ.mol<sup>−1</sup>. In this case, the carboxylate group of the sialic acid interacted with the cationic group of the nitrogen-containing ring of CLQ (pKa 8.1) <a href="#fig0002">Fig. 2</a>(d). The complex was further stabilized by OH-π and van der Waals interactions. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Molecular modelling of chloroquine (CLQ) interaction with sialic acids. (a,b) Surface representation of the CLQ–sialic acid (Neu5Ac) complex. Two opposite views of the complex are shown. Note the geometric complementarity between the L-shape conformer of CLQ dissolved in water (in blue) and Neu5Ac (in red). (c) Neu5Ac bound to CLQ via a combination of CH-π and electrostatic interactions with one of the cationic groups of CLQ (+). (d) Molecular modelling of CLQ bound to N-acetyl-9-O-acetylneuraminic acid (9-O-SIA). From right to left, the dashed lines indicate a series of van der Waals, OH-π and electrostatic contacts with both cationic groups of CLQ (+). (e,f) Surface representations of the CLQ–9-O-SIA complex.</p>   </div> <p /> <p id="para0013">Next, CLQ-OH was tested to assess whether it could, as CLQ, bind to 9-O-SIA (<a href="#fig0003">Fig. 3</a> ). The complex obtained with CLQ-OH was very similar to that obtained with CLQ [compare <a href="#fig0003">Fig. 3</a>(a,b) with <a href="#fig0002">Fig. 2</a>(e,f), although several conformational adjustments occurred during the simulations. Interestingly, the OH group of CLQ-OH reinforced the binding of CLQ to sialic acid through establishment of a hydrogen bond <a href="#fig0003">Fig. 3</a>(c,d). Overall, this hydrogen bond compensated for the slight loss of energy caused by the conformational rearrangement, and the energy of interaction of the complex was estimated to be -46 kJ.mol<sup>−1</sup>, which is very close to the value obtained for CLQ (-45 kJ.mol<sup>−1</sup>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Molecular modelling of hydroxychloroquine (CLQ-OH) interaction with sialic acids. (a,b) Surface representation of CLQ-OH bound to N-acetyl-9-O-acetylneuraminic acid (9-O-SIA). Two opposite views of the complex are shown. Note the geometric complementarity between CLQ-OH (in blue) and 9-O-SIA (in red). (c,d) Molecular mechanism of CLQ-OH binding to 9-O-SIA: combination of electrostatic interactions and hydrogen bonding.</p>   </div> <p /> </div> <div class="molecularrecognitionofgangliosidesbyclqandclq-oh" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Molecular recognition of gangliosides by CLQ and CLQ-OH</div> <p id="para0014">In the respiratory tract, sialic acids are usually part of glycoproteins and gangliosides. Molecular modelling approaches were used to assess whether CLQ and CLQ-OH can recognize sialic acid units in their natural molecular environment. In these simulations, ganglioside GM1 was chosen as a representative example of human plasma membrane gangliosides. A first series of simulations was performed with CLQ. When merged with the ganglioside, CLQ had two distinct binding sites, both located in the polar saccharide part of GM1. The first site was located at the tip of the saccharide moiety of the ganglioside <a href="#fig0004">Fig. 4</a> (a,b). The energy of interaction was estimated to be -47 kJ.mol<sup>−1</sup>. CLQ retained the typical L-shape structure of the water-soluble conformer bound to isolated sialic acids [compare <a href="#fig0002">Figs 2</a>(c) and <a href="#fig0004">4</a>(a)]. From a mechanistic point of view, the carboxylate group of the sialic acid of GM1 was oriented towards the cationic groups of CLQ. The rings of CLQ faced the N-acetylgalactosamine (GalNAc) residue of GM1, establishing both OH-π interaction and hydrogen bonding <a href="#fig0004">Fig. 4</a>(b). The second site was in a large area including both the ceramide–sugar junction and the saccharide moiety <a href="#fig0004">Fig. 4</a>(c). The chlorine atom of CLQ was oriented towards the ceramide axis, allowing the nitrogen-containing ring of CLQ to stack on to the pyrane ring of the first sugar residue [i.e. glucose (Glc)]. The perfect geometric complementarity of the two partners <a href="#fig0004">Fig. 4</a>(c,d) accounted for a particularly high energy of interaction in this case (-61 kJ.mol<sup>−1</sup>). Interestingly, there was no overlap between the two CLQ-binding sites on GM1, so the ganglioside could accommodate two CLQ molecules <a href="#fig0004">Fig. 4</a>(e), reaching a global energy of interaction of -108 kJ.mol<sup>−1</sup>. A similar situation was observed with CLQ-OH, which occupies the same binding site as CLQ <a href="#fig0004">Fig. 4</a>(f). In this case, the energy of interaction was further increased by stabilizing contacts established between the two CLQ-OH molecules, reaching -120 kJ.mol<sup>−1</sup>. Overall, these data showed that CLQ and CLQ-OH have a good fit for sialic acids, either isolated or bound to gangliosides. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Molecular modelling simulations of chloroquine (CLQ) and hydroxychloroquine (CLQ-OH) binding to ganglioside GM1. The surface electrostatic potential of GM1 indicates a non-polar, membrane-embedded part corresponding to ceramide (white areas), and an acidic part protruding in the extracellular space corresponding to the sialic-acid-containing saccharide part (red areas). (a) CLQ bound to the tip of the carbohydrate moiety of GM1. (b) Molecular mechanism of CLQ–ganglioside interactions. (c) Molecular dynamics simulations revealed a second site of interaction. In this case, the aromatic cycles of CLQ are positioned at the ceramide–sugar junction, whereas the nitrogen atoms interact with the acidic part of the ganglioside (not illustrated). (d,e) Surface views of GM1 complexed with one (d) or two (e) CLQ molecules (both in blue), illustrating the geometric complementarity of GM1 and CLQ molecules. (f) One GM1 molecule can also accommodate two distinct CLQ-OH molecules simultaneously, after slight rearrangement allowing increased fit due to CLQ-OH/CLQ-OH interactions. To improve clarity, CLQ-OH molecules bound to GM1 are represented in two distinct colours (blue and green).</p>   </div> <p /> </div> <div class="structuralanalysisofthentdofsars-cov-2sprotein" title="sec"><span class="label" tagx="label" title="label">3.4</span><div class="title" tagx="title" title="title">
Structural analysis of the NTD of SARS-CoV-2 S protein</div> <p id="para0015">The next step of this study was to determine how SARS-CoV-2 could interact with plasma membrane gangliosides, and whether such interaction could be affected by CLQ and CLQ-OH. The global structure of the SARS-CoV-2 S protein <a href="#bib0016">[16]</a> is shown in <a href="#fig0005">Fig. 5</a> (a–d). It consists of a trimer of S proteins, each harbouring two distinct domains distant from the viral envelope: the receptor-binding region (RBD) and the NTD. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Structural features of the SARS-CoV-2 spike (S) protein. (a) Trimeric structure (each S protein has a distinct surface colour, ‘blue’, ‘yellow’ and ‘purple’). (b) Ribbon representation of ‘blue’ S protein in the trimer (α-helix, red; β-strand, blue; coil, grey). (c) Surface structure of the ‘blue’ S protein isolated from the trimer. (d) Ribbon structure of the ‘blue’ S protein. (e) Zoom on the N-terminal domain (NTD) of the ‘blue’ S protein. (f,g) Molecular model of a minimal NTD obtained with Hyperchem [ribbon in representation in (f), surface rendering in (g)]. (h) Highlighting of the amino acid residues of the NTD that could belong to a potential ganglioside-binding domain.</p>   </div> <p /> <p id="para0016">It was reasoned that if the RBD is engaged in functional interactions with the ACE-2 receptor, it would be interesting to search for potential ganglioside-binding sites on the other cell-accessible domain of the S glycoprotein (i.e. the NTD). The NTD contains approximately 290 amino acid residues. The tip of the NTD was of particular interest, as it displays a flat interface <a href="#fig0005">Fig. 5</a>(f) ideally positioned for targeting a ganglioside-rich plasma membrane microdomain, such as a lipid raft.</p> <p id="para0017">The amino acid sequence of the planar interfacial surface located at the tip of the NTD was analysed for the presence of consensus ganglioside-binding domains <a href="#bib0020">[20]</a>. These motifs are constituted by a triad of mandatory amino acid residues such as (K,R)-X<sub>n</sub>-(F,Y,W)-X<sub>n</sub>-(K,R). The X<sub>n</sub> intercalating segments, usually four to five residues, may contain any amino acid, but often Gly, Pro and/or Ser residues. The strict application of this algorithm did not allow the detection of any potential ganglioside-binding domain in this region of the NTD. However, an intriguing over-representation of aromatic and basic residues was found in the 129–158 segment: 129-<u>K</u>VCE<u>F</u>Q<u>F</u>CNDP<u>F</u>LGV<u>YY</u>H<u>K</u>NN<u>K</u>S<u>W</u>MESE<u>FR</u>-158. This 30-amino acid stretch also contains Gly, Pro and/or Ser residues that are often found in ganglioside-binding motifs. These observations supported the notion that the tip of the NTD could display a large ganglioside-attachment interface.</p> </div> <div class="molecularinteractionsbetweengangliosidesandthentdofsars-cov-2sprotein" title="sec"><span class="label" tagx="label" title="label">3.5</span><div class="title" tagx="title" title="title">
Molecular interactions between gangliosides and the NTD of SARS-CoV-2 S protein</div> <p id="para0018">Molecular dynamic simulations of a structural motif encompassing amino acid residues 100–175 of the NTD <a href="#fig0005">Fig. 5</a>(f–h) merged with ganglioside GM1 further supported this concept. As shown in <a href="#fig0006">Fig. 6</a> , the large flat area of this structural domain fitted very well with the protruding oligosaccharide part of the ganglioside. Several amino acid residues appear to be critical for this interaction, especially Phe-135, Asn-137 and Arg-158 (<a href="#tbl0001">Table 1</a> ). Overall, the complex involved 10 amino acid residues for a total energy of interaction of -100 kJ.mol<sup>−1</sup>. At this stage, it was observed that approximately 50% of the interface was involved in the complex, leaving the remaining 50% available for interaction with a second GM1 molecule. As expected, merging a second GM1 molecule with the preformed GM1–NTD complex led to a trimolecular complex consisting of two gangliosides in a typical symmetrical chalice-like structure into which the NTD could insert its interfacial ganglioside-binding domain (<a href="#fig0007">Fig. 7</a> ). The formation of this trimolecular complex was progressive, starting with a conformational rearrangement of the first ganglioside–NTD complex triggered by the second GM1 molecule. The energy of interaction of the new complex was consistently increased by 37%, reaching an estimated value of -137 kJ.mol<sup>−1</sup>. At this stage, attachment of the NTD to the ganglioside-rich microdomain involved the whole interface (i.e. 15 surface-accessible residues from Asp-111 to Ser-162). The critical residues were Asp-111, Gln-134, Phe-135, Arg-158 and Ser-161 (<a href="#tbl0001">Table 1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Molecular complex between the N-terminal domain (NTD) of SARS-CoV-2 spike protein and a single GM1 ganglioside. The NTD is represented in ribbons superposed with a transparent surface rendering (light green). Two symmetric views of the complex are shown (a,b). The amino acid residues Q-134 to D-138 located in the centre of the ganglioside-binding domain are represented as green spheres. The saccharide part of the ganglioside forms a landing surface for the tip of the NTD.</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="tbl0001"><span class="label" tagx="label" title="label">Table 1</span> <p>Energy of interaction of each amino acid residue of SARS�?CoV�?2 spike protein in contact with GM1 molecules.</p> [alt-text] Table 1 <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top">Amino acid residues</th> <th valign="top">Energy of interaction (kJ.mol�?1)</th> </tr> </thead> <tbody> <tr> <td valign="top">First step: one GM1 molecule</td> <td valign="top" /> </tr> <tr> <td valign="top"> Asp�?111</td> <td valign="top">−5.6</td> </tr> <tr> <td valign="top"> Lys�?113</td> <td valign="top">−8.2</td> </tr> <tr> <td valign="top"> Gln�?134</td> <td valign="top">−8.6</td> </tr> <tr> <td valign="top"> Phe�?135</td> <td valign="top">−20.1</td> </tr> <tr> <td valign="top"> Cys�?136</td> <td valign="top">−7.0</td> </tr> <tr> <td valign="top"> Asn�?137</td> <td valign="top">−15.2</td> </tr> <tr> <td valign="top"> Asp�?138</td> <td valign="top">−6.4</td> </tr> <tr> <td valign="top"> Arg�?158</td> <td valign="top">−17.4</td> </tr> <tr> <td valign="top"> Ser�?161</td> <td valign="top">−9.7</td> </tr> <tr> <td valign="top"> Ser�?162</td> <td valign="top">−2.0</td> </tr> <tr> <td valign="top"> Total</td> <td valign="top">−100.2</td> </tr> <tr> <td valign="top">Second step: two GM1 molecules</td> <td valign="top" /> </tr> <tr> <td valign="top"> Asp�?111</td> <td valign="top">−15.8</td> </tr> <tr> <td valign="top"> Ser�?112</td> <td valign="top">−10.7</td> </tr> <tr> <td valign="top"> Lys�?113</td> <td valign="top">−9.2</td> </tr> <tr> <td valign="top"> Gln�?134</td> <td valign="top">−11.2</td> </tr> <tr> <td valign="top"> Phe�?135</td> <td valign="top">−10.5</td> </tr> <tr> <td valign="top"> Cys�?136</td> <td valign="top">−6.2</td> </tr> <tr> <td valign="top"> Asn�?137</td> <td valign="top">−4.7</td> </tr> <tr> <td valign="top"> Phe�?140</td> <td valign="top">−5.2</td> </tr> <tr> <td valign="top"> Gly�?142</td> <td valign="top">−5.6</td> </tr> <tr> <td valign="top"> Glu�?156</td> <td valign="top">−9.0</td> </tr> <tr> <td valign="top"> Phe�?157</td> <td valign="top">−13.8</td> </tr> <tr> <td valign="top"> Arg�?158</td> <td valign="top">−19.8</td> </tr> <tr> <td valign="top"> Tyr�?160</td> <td valign="top">−3.2</td> </tr> <tr> <td valign="top"> Ser�?161</td> <td valign="top">−9.7</td> </tr> <tr> <td valign="top"> Ser�?162</td> <td valign="top">−2.0</td> </tr> <tr> <td valign="top"> Total</td> <td valign="top">−136.6</td> </tr> </tbody> </table> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>Molecular complex between the N-terminal domain (NTD) of SARS-CoV-2 spike protein and a dimer of GM1. In (a), (c) and (d), the NTD is represented as in <a href="#fig0004">Fig. 4</a>. In (b), the surface of the NTD is shown without any transparency. The amino acid residues Q-134 to S-162 belonging to the ganglioside-binding domain (GBD) are represented as green spheres. Compared with a single GM1 molecule, the dimer of gangliosides forms a larger attractive surface for the NTD. In the above view of (d), the anchorage of the NTD to the gangliosides is particularly obvious. As chloroquine also interacts with the saccharide part of GM1, its presence would clearly mask most of the landing surface available for the NTD, preventing attachment of the virus to the plasma membrane of host cells.</p>   </div> <p /> </div> <div class="potentialcoordinatedinteractionsbetweensars-cov-2andtheplasmamembraneofahostcell:keyroleofgangliosidesinlipidrafts" title="sec"><span class="label" tagx="label" title="label">3.6</span><div class="title" tagx="title" title="title">
Potential coordinated interactions between SARS-CoV-2 and the plasma membrane of a host cell: key role of gangliosides in lipid rafts</div> <p id="para0019">Taken together, these data strongly support the concept of a dual receptor/attachment model for SARS-CoV-2, with the RBD domain being involved in ACE-2 receptor recognition, and the NTD interface responsible for finding a ganglioside-rich landing area (lipid raft) at the cell surface.</p> <p id="para0020">Such a dual receptor model, consistent with the topology of the SARS-CoV-2 S protein, is proposed in <a href="#fig0008">Fig. 8</a> . With this model in mind, the potential effects of CLQ and CLQ-OH were studied, both of which, according to the molecular modelling data, have a high affinity for sialic acids and gangliosides. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <p>Dual recognition of gangliosides and angiotensin-converting enzyme-2 (ACE-2) by SARS-CoV-2 spike (S) protein. The viral protein displays two distinct domains, the tips of which are available for distinct types of interactions. The receptor-binding domain binds to the ACE-2 receptor, and the N-terminal domain (NTD) binds to the ganglioside-rich domain of the plasma membrane. Lipid rafts, which are membrane domains enriched in gangliosides (in yellow) and cholesterol (in blue), provide a perfect attractive interface for adequately positioning the viral S protein at the first step of the infection process. These structural and molecular modelling studies suggest that amino acid residues 111–162 of the NTD form a functional ganglioside-binding domain, the interaction of which with lipid rafts can be efficiently prevented by chloroquine and hydroxychloroquine.</p>   </div> <p /> </div> <div class="molecularmechanismofclqandclq-ohantiviraleffect:preventingsars-cov-2sproteinaccesstocellsurfacegangliosides" title="sec"><span class="label" tagx="label" title="label">3.7</span><div class="title" tagx="title" title="title">
Molecular mechanism of CLQ and CLQ-OH antiviral effect: preventing SARS-CoV-2 S protein access to cell surface gangliosides</div> <p id="para0021">With the aim of establishing whether CLQ and CLQ-OH could prevent the attachment of SARS-CoV-2 to plasma membrane gangliosides, the initial NTD–GM1 complex was superposed with a drug–GM1 complex (<a href="#fig0009">Fig. 9</a> ). To improve clarity, the ganglioside is not presented in <a href="#fig0009">Fig. 9</a>. This superposition shows that the NTD and the drug (CLQ-OH in this case) share the same spatial position when bound to GM1, so GM1 cannot bind the viral protein and the drug simultaneously. This is due to the fact that the NTD and the drugs (CLQ and CLQ-OH) bind to GM1 with a similar mechanism controlled by a dyad of functional interactions: a hydrogen bond and a geometrically perfect CH-π stacking interaction. In the case of the NTD, the hydrogen bond involves Asn-167, whereas CH-π stacking is mediated by the aromatic ring of Phe-135 <a href="#fig0006">Fig. 6</a>(b). On one hand, Asn-167 establishes a network of hydrogen bonds with the GalNAc residue of GM1. On the other hand, the flat aromatic ring of Phe-135 stacks on to the cycle of the Glc residue of GM1. In the case of CLQ and CLQ-OH, it is the nitrogen-containing ring of the drug that stacks on to the Glc ring <a href="#fig0004">Fig. 4</a>(c). Note that both the Phe-135 (in red) and CLQ-OH (in green) rings are located in the same position (<a href="#fig0009">Fig. 9</a>). The other CLQ-OH molecule, which covers the tip of the sugar part of the ganglioside, interacts with the GalNAc ganglioside <a href="#fig0004">Fig. 4</a>(b). When the NTD is bound to the ganglioside, the side chain of Asn-137 is found in this exact position (<a href="#fig0009">Fig. 9</a>). Thus, once two CLQ-OH (or two CLQ) molecules are bound to a ganglioside <a href="#fig0004">Fig. 4</a>(e,f), any binding of a SARS-Cov-2 S protein to the same ganglioside is totally prevented. The energy required to overcome this steric incompatibility is estimated to be several hundred kJ.mol<sup>−1</sup>, which is far too high to occur. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 9</span></a></div>   <p>Mechanism of action of hydroxychloroquine (CLQ-OH). The N-terminal domain (NTD) bound to GM1 was superposed onto GM1 in interaction with two CLQ-OH molecules. The models only show the NTD and both CLQ-OH molecules (not GM1, to improve clarity). (a,b) The aromatic ring of F-135 (in red), which stacks onto the glucose cycle of GM1, overlaps the aromatic CLQ-OH rings (in green) which also bind to GM1 via a CH-π stacking mechanism. The N-137 residue of the NTD interacts with the N-acetylgalactosamine residue of GM1, but this interaction cannot occur in the presence of CLQ-OH as this part of GM1 is totally masked by the drug. (c,d) Superposition of the NTD surface (in purple) with the two CLQ-OH molecules bound to GM1, illustrating the steric impossibility that prevents NTD binding to GM1 when both CLQ-OH molecules are already interacting with the ganglioside.</p>   </div> <p /> </div> <div class="sequencealignmentanalysisofsars-cov-2andrelatedcoronavirus:evolutionoftheganglioside-bindingdomainatcriticalaminoacidresidues" title="sec"><span class="label" tagx="label" title="label">3.8</span><div class="title" tagx="title" title="title">
Sequence alignment analysis of SARS-CoV-2 and related coronavirus: evolution of the ganglioside-binding domain at critical amino acid residues</div> <p id="para0022">As CLQ and CLQ-OH are potential therapies for SARS-CoV-2 infection, it is important to check whether the amino acid residues identified as critical for ganglioside binding are conserved among clinical isolates. The alignment of the 111–162 domain of 11 clinical isolates of SARS-CoV-2 from various geographic origins (including Asia and USA) is shown in <a href="#fig0010">Fig. 10</a> . In this region, which contains the ganglioside-binding domain identified in the present report, all amino acids are fully conserved. Interestingly, the motif is built like a giant consensus ganglioside-binding domain: a central region displaying the critical aromatic residue (Phe-135) and a basic residue at each end (Lys-113 and Arg-158). In the middle of each stretch separating this typical triad, there is a N-glycosylation site (Asn-122 and Asn-149). These last regions are not directly involved in ganglioside binding, so the oligosaccharide linked to these asparagine residues could be perfectly intercalated between the sugar head group of gangliosides. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 10</span></a></div>   <p>Amino acid sequence alignments of the ganglioside-binding domain (GBD) of the SARS-CoV-2 spike protein. (a) Clinical SARS-CoV-2 isolates aligned with the reference sequence (6VSB_A, fragment 111–162). The amino acid residues involved in GM1 binding are indicated in red. Two asparagine residues acting as glycosylation sites are highlighted in yellow. (b) Alignments of human and animal viruses compared with SARS-CoV-2 (6VSB_A, fragment 111–162). Deletions are highlighted in green, amino acid changes in residues involved in ganglioside binding are highlighted in blue, conserved residues of the GBD are highlighted in red, and asparagine residues acting as glycosylation sites are highlighted in yellow.</p>   </div> <p /> <p id="para0023">It was also noted that the ganglioside-binding domain of the NTD is fully conserved in bat RaTG13, which indicates a close relationship between the bat coronavirus and the human isolates that are currently circulating around the world. However, the motif is slightly different in other bat- and human-related coronaviruses (<a href="#fig0010">Fig. 10</a>), suggesting a recent evolution which could explain, at least in part, why SARS-CoV-2 is more contagious than previously characterized human coronaviruses.</p> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Discussion</div> <p id="para0024">Sialic acids linked to glycoproteins and gangliosides are used by a broad range of viruses as receptors and/or attachment factors for cell entry <a href="#bib0010">[10]</a>. These viruses include major human pathogens affecting the respiratory tract, such as influenza <a href="#bib0021">[21]</a> and coronaviruses <a href="#bib0022">[22</a>,<a href="#bib0023">23]</a>. The attachment to sialic-acid-containing cell surface structures is mediated by receptor-binding proteins that belong to the viral spike. In the case of coronaviruses, this function is fulfilled by the S glycoprotein <a href="#bib0009">[9</a>,<a href="#bib0024">24]</a>. SARS-CoV and SARS-CoV-2 interact with the ACE-2 protein, which has been identified as a key determinant of the contagiousness of viruses <a href="#bib0008">[8]</a>. However, considering the increased transmissibility of SARS-CoV-2 compared with SARS-CoV, binding to ACE-2 alone might not be enough to ensure robust infection of the upper respiratory tract. Thus, it is likely that SARS-CoV-2 might also bind to other cell surface attachment factors, such as sialic-acid-containing glycoproteins and gangliosides. Consistent with this notion, it has been shown that depletion of cell surface sialic acids by neuraminidase treatment inhibited MERS-CoV entry of human airway cells <a href="#bib0025">[25]</a>. These data, which provide direct evidence that sialic acids play a critical role in human coronavirus attachment, broaden the therapeutic options to block the replication of SARS-CoV-2 that is responsible for the COVID-19 pandemic.</p> <p id="para0025">Few drugs have shown consistent antiviral efficiency <i>in vitro</i> together with reported efficiency in patients infected with SARS-CoV-2 <a href="#bib0003">[3</a>,<a href="#bib0012">12]</a>. Of these, CLQ is of interest as its chemical structure is based on a combination of cationic nitrogen atoms and aromatic rings. Both features have been shown to be key determinants of the recognition of sialic acids and gangliosides by proteins <a href="#bib0020">[20</a>,<a href="#bib0026">26]</a>. Modelling approaches have been used successfully to decipher various molecular mechanisms of protein–sugar interactions accounting for the interaction of virus <a href="#bib0027">[27]</a>, bacteria <a href="#bib0028">[28]</a>, membrane <a href="#bib0013">[13]</a> and amyloid proteins <a href="#bib0020">[20]</a> with cell surface glycolipids. This in-silico strategy was applied to unravel the molecular mechanisms underlying the antiviral mechanisms of CLQ and CLQ-OH against SARS-CoV-2 infection. First, it was shown that CLQ and CLQ-OH bind readily to sialic acids with high affinity, including the typical 9-O-SIA subtype recognized by coronaviruses <a href="#bib0023">[23]</a>. Next, it was shown that CLQ and CLQ-OH also bind to sialic-acid-containing gangliosides. Based on these data, the drugs may also recognize the sialic acid residues of glycoproteins. Further studies will help clarify this point.</p> <p id="para0026">This molecular modelling study has identified a new type of ganglioside-binding domain in the NTD of the SARS-CoV-2 S protein. This ganglioside-binding domain consists of a large flat interface enriched in aromatic and basic amino acid residues. It covers a stretch of 52 amino acid residues (111–162), which is longer than all linear ganglioside-binding domains characterized to date <a href="#bib0029">[29]</a>. However, the new SARS-CoV-2 motif is organized into three distinct regions, including a central aromatic domain and two terminal basic domains (<a href="#fig0010">Fig. 10</a>). Thus, this motif displays the typical features that determine optimal binding to gangliosides (i.e. CH-π stacking and electrostatic interactions).</p> <p id="para0027">A major outcome of this study is the demonstration that CLQ and CLQ-OH display molecular groups that fully mimic the way in which the S protein binds to gangliosides. Two CLQ (or two CLQ-OH) molecules can bind simultaneously to the polar head group of ganglioside GM1. Interestingly, these simulations indicated that CLQ-OH is more potent than CLQ, in line with the reported increased antiviral activity of CLQ-OH against SARS-CoV-2 <a href="#bib0030">[30]</a>. Once bound to GM1, the drugs prevent any access to the Glc and GalNAc units of the ganglioside, which are the critical binding residues for Phe-135 and Asn-137, respectively. This amino acid dyad, as well as all the other residues that mediate ganglioside binding by the SARS-CoV-2 spike, is fully conserved among clinical isolates worldwide. It is also conserved in the bat RaTG13 isolate, which reinforces the hypothesis of bat-to-human transmission. From an epidemiological point of view, it can be hypothesized that the evolution of this motif has conferred an enhanced attachment capacity of human coronaviruses to the respiratory tract through improved S–ganglioside interactions.</p> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
Conclusion</div> <p id="para0028">Given the global health emergency, drug repurposing is obviously the option of choice <a href="#bib0002">[2</a>,<a href="#bib0003">3]</a>. However, a considerable amount of time could be saved by in-silico testing to determine the capability of any potential anti-SARS-CoV-2 to disrupt the interaction of the S protein with the host cell membrane. Applied to both RBD–ACE-2 and NTD–ganglioside interactions, this molecular modelling method will help select those drugs that are likely to interfere with the initial attachment of virus particles to the respiratory tract surface epithelium. The study data support the use of CLQ, and preferentially CLQ-OH, as initial therapy for patients infected with SARS-CoV-2.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib0001" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">579</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="bib0002" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mullard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug repurposing programmes get lift off</span><span class="source" tagx="source" title="source">Nat Rev Drug Discov</span><span class="volume" tagx="volume" title="volume">11</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">505</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22743966">22743966</a></span></span></li> <li tag="ref"><a name="bib0003" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colson</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rolain</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raoult</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chloroquine for the 2019 novel coronavirus SARS-CoV-2</span><span class="source" tagx="source" title="source">Int J Antimicrob Agents</span><span class="volume" tagx="volume" title="volume">55</span><span class="year" tagx="year" title="year">2020</span>105923</span></li> <li tag="ref"><a name="bib0004" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Breakthrough: chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies</span><span class="source" tagx="source" title="source">Biosci Trends</span><span class="year" tagx="year" title="year">2020 Feb 19</span><span class="pub-id"><a href="https://dx.doi.org/10.5582/bst.2020.01047">10.5582/bst.2020.01047</a></span><span class="comment">[Epub ahead of print]</span></span></li> <li tag="ref"><a name="bib0005" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savarino</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Trani</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donatelli</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cauda</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassone</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">New insights into the antiviral effects of chloroquine</span><span class="source" tagx="source" title="source">Lancet Infect Dis</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">69</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16439323">16439323</a></span></span></li> <li tag="ref"><a name="bib0006" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mauthe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orhon</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocchi</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luhr</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hijlkema</span><span class="given-names" tagx="given-names" title="given-names">K.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chloroquine inhibits autophagic flux by decreasing autophagosome–lysosome fusion</span><span class="source" tagx="source" title="source">Autophagy</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1435</span><span class="lpage" tagx="lpage" title="lpage">1455</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29940786">29940786</a></span></span></li> <li tag="ref"><a name="bib0007" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergeron</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benjannet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erickson</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rollin</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ksiazek</span><span class="given-names" tagx="given-names" title="given-names">T.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chloroquine is a potent inhibitor of SARS coronavirus infection and spread</span><span class="source" tagx="source" title="source">Virol J</span><span class="volume" tagx="volume" title="volume">2</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16115318">16115318</a></span></span></li> <li tag="ref"><a name="bib0008" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilieva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sui</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berne</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">426</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">450</span><span class="lpage" tagx="lpage" title="lpage">454</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a></span></span></li> <li tag="ref"><a name="bib0009" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020 Mar 4</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb2762">10.1126/science.abb2762</a></span><span class="comment">pii: eabb2762[Epub ahead of print]</span></span></li> <li tag="ref"><a name="bib0010" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matrosovich</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrler</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klenk</span><span class="given-names" tagx="given-names" title="given-names">H.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sialic acid receptors of viruses</span><span class="source" tagx="source" title="source">Top Curr Chem</span><span class="volume" tagx="volume" title="volume">367</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23873408">23873408</a></span></span></li> <li tag="ref"><a name="bib0011" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Scala</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hybrid in silico/in vitro approaches for the identification of functional cholesterol-binding domains in membrane proteins</span><span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="volume" tagx="volume" title="volume">1583</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28205163">28205163</a></span></span></li> <li tag="ref"><a name="bib0012" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colson</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rolain</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lagier</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brouqui</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raoult</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chloroquine and hydroxychloroquine as available weapons to fight COVID-19</span><span class="source" tagx="source" title="source">Int J Antimicrob Agents</span><span class="year" tagx="year" title="year">2020 Mar 4</span>105932<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijantimicag.2020.105932">10.1016/j.ijantimicag.2020.105932</a></span><span class="comment">[Epub ahead of print]</span></span></li> <li tag="ref"><a name="bib0013" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flores</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramirez-Franco</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desplantes</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debreux</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferracci</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wernert</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gangliosides interact with synaptotagmin to form the high-affinity receptor complex for botulinum neurotoxin B</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">116</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">18098</span><span class="lpage" tagx="lpage" title="lpage">18108</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31431523">31431523</a></span></span></li> <li tag="ref"><a name="bib0014" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Scala</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wandall</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The glycosphingolipid MacCer promotes synaptic bouton formation in drosophila by interacting with Wnt</span><span class="source" tagx="source" title="source">Elife</span><span class="volume" tagx="volume" title="volume">7</span><span class="year" tagx="year" title="year">2018</span><span class="comment">pii:e38183</span></span></li> <li tag="ref"><a name="bib0015" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ståhle</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kern</span><span class="given-names" tagx="given-names" title="given-names">N.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">CHARMM-GUI membrane builder for complex biological membrane simulations with glycolipids and lipoglycans</span><span class="source" tagx="source" title="source">J Chem Theory Comput</span><span class="volume" tagx="volume" title="volume">15</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">775</span><span class="lpage" tagx="lpage" title="lpage">786</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30525595">30525595</a></span></span></li> <li tag="ref"><a name="bib0016" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrapp</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbett</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsieh</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abiona</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">367</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1260</span><span class="lpage" tagx="lpage" title="lpage">1263</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a></span></span></li> <li tag="ref"><a name="bib0017" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guvench</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greene</span><span class="given-names" tagx="given-names" title="given-names">S.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamath</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venable</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastor</span><span class="given-names" tagx="given-names" title="given-names">R.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Additive empirical force field for hexopyranose monosaccharides</span><span class="source" tagx="source" title="source">J Comput Chem</span><span class="volume" tagx="volume" title="volume">29</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">2543</span><span class="lpage" tagx="lpage" title="lpage">2564</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18470966">18470966</a></span></span></li> <li tag="ref"><a name="bib0018" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">Y.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walls</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauer</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tortorici</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structures of MERS-CoV spike glycoprotein in complex with sialoside attachment receptors</span><span class="source" tagx="source" title="source">Nat Struct Mol Biol</span><span class="volume" tagx="volume" title="volume">26</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">1151</span><span class="lpage" tagx="lpage" title="lpage">1157</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31792450">31792450</a></span></span></li> <li tag="ref"><a name="bib0019" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huta</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehlenbacher</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zubieta</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bou-Abdallah</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The lysosomal protein saposin B binds chloroquine</span><span class="source" tagx="source" title="source">Chem Med Chem</span><span class="volume" tagx="volume" title="volume">11</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">277</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26616259">26616259</a></span></span></li> <li tag="ref"><a name="bib0020" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Deciphering the glycolipid code of Alzheimer's and Parkinson's amyloid proteins allowed the creation of a universal ganglioside-binding peptide</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2014</span>e104751</span></li> <li tag="ref"><a name="bib0021" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">D.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lal</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host lipid rafts play a major role in binding and endocytosis of influenza A virus</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">10</span><span class="year" tagx="year" title="year">2018</span><span class="comment">pii:E650</span></span></li> <li tag="ref"><a name="bib0022" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tam</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid rafts are involved in SARS-CoV entry into Vero E6 cells</span><span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span><span class="volume" tagx="volume" title="volume">369</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">344</span><span class="lpage" tagx="lpage" title="lpage">349</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18279660">18279660</a></span></span></li> <li tag="ref"><a name="bib0023" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tortorici</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walls</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koerhuis</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for human coronavirus attachment to sialic acid receptors</span><span class="source" tagx="source" title="source">Nat Struct Mol Biol</span><span class="volume" tagx="volume" title="volume">26</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">481</span><span class="lpage" tagx="lpage" title="lpage">489</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31160783">31160783</a></span></span></li> <li tag="ref"><a name="bib0024" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleine-Weber</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krüger</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erichsen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2020.02.052">10.1016/j.cell.2020.02.052</a></span><span class="comment">pii:S0092-8674(20)30229-4[Epub ahead of print]</span></span></li> <li tag="ref"><a name="bib0025" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulswit</span><span class="given-names" tagx="given-names" title="given-names">R.J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widjaja</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">V.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McBride</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of sialic acid-binding function for the Middle East respiratory syndrome coronavirus spike glycoprotein</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">114</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">E8508</span><span class="lpage" tagx="lpage" title="lpage">E8517</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28923942">28923942</a></span></span></li> <li tag="ref"><a name="bib0026" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular insights into amyloid regulation by membrane cholesterol and sphingolipids: common mechanisms in neurodegenerative diseases</span><span class="source" tagx="source" title="source">Expert Rev Mol Med</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">e27</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20807455">20807455</a></span></span></li> <li tag="ref"><a name="bib0027" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahfoud</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garmy</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maresca</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puigserver</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a common sphingolipid-binding domain in Alzheimer, prion, and HIV-1 proteins</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="volume" tagx="volume" title="volume">277</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">11292</span><span class="lpage" tagx="lpage" title="lpage">11296</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11792705">11792705</a></span></span></li> <li tag="ref"><a name="bib0028" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garmy</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prediction of glycolipid-binding domains from the amino acid sequence of lipid raft-associated proteins: application to HpaA, a protein involved in the adhesion of <i>Helicobacter pylori</i> to gastrointestinal cells</span><span class="source" tagx="source" title="source">Biochemistry</span><span class="volume" tagx="volume" title="volume">45</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">10957</span><span class="lpage" tagx="lpage" title="lpage">10962</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16953581">16953581</a></span></span></li> <li tag="ref"><a name="bib0029" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fantini</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yahi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Brain lipids in synaptic function and neurological disease. Clues to innovative therapeutic strategies for brain disorders</span><span class="year" tagx="year" title="year">2015</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco</span></span></li> <li tag="ref"><a name="bib0030" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In vitro antiviral activity and projection of optimized dosing design of hydroxychloroquine for the treatment of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)</span><span class="source" tagx="source" title="source">Clin Infect Dis</span><span class="year" tagx="year" title="year">2020 Mar 9</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/ciaa237">10.1093/cid/ciaa237</a></span><span class="comment">pii:ciaa237</span></span></li> </ul> </div> <div class="ack" title="ack"> <p><b>Funding:</b> None.</p> <p><b>Competing interests:</b> None declared.</p> <p><b>Ethical approval:</b> Not required.</p> </div> </div>  </body></html>