<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/22/2020.02.28.970343.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.28.970343v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches" />
<meta name="DC.Identifier" content="10.1101/2020.02.28.970343" />
<meta name="DC.Date" content="2020-03-22" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world’s population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated." />
<meta name="DC.Contributor" content="Muhammad Tahir ul Qamar" />
<meta name="DC.Contributor" content="Abdur Rehman" />
<meta name="DC.Contributor" content="Usman Ali Ashfaq" />
<meta name="DC.Contributor" content="Muhammad Qasim Awan" />
<meta name="DC.Contributor" content="Israr Fatima" />
<meta name="DC.Contributor" content="Farah Shahid" />
<meta name="DC.Contributor" content="Ling-Ling Chen" />
<meta name="article:published_time" content="2020-03-22" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2.  Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.28.970343v2" />
<meta name="citation_id" content="2020.02.28.970343v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.28.970343" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.28.970343" />
<meta name="citation_author" content="Muhammad Tahir ul Qamar" />
<meta name="citation_author_institution" content="College of Life Science and Technology, Guangxi University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4832-4250" />
<meta name="citation_author" content="Abdur Rehman" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad" />
<meta name="citation_author" content="Usman Ali Ashfaq" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad" />
<meta name="citation_author_email" content="llchen@mail.hzau.edu.cn" />
<meta name="citation_author_email" content="usmancemb@gmail.com" />
<meta name="citation_author" content="Muhammad Qasim Awan" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad" />
<meta name="citation_author" content="Israr Fatima" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad" />
<meta name="citation_author" content="Farah Shahid" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Biotechnology, Government College University Faisalabad" />
<meta name="citation_author" content="Ling-Ling Chen" />
<meta name="citation_author_institution" content="College of Life Science and Technology, Guangxi University" />
<meta name="citation_author_email" content="llchen@mail.hzau.edu.cn" />
<meta name="citation_author_email" content="usmancemb@gmail.com" />
<meta name="citation_reference" content="R.A. Tripp, S.M. Tompkins, Roles of Host Gene and Non-coding RNA Expression in Virus Infection, Springer 2018." />
<meta name="citation_reference" content="X. Xu, P. Chen, J. Wang, J. Feng, H. Zhou, X. Li, W. Zhong, P. Hao, Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission, Sci China Life Sci 63(3) (2020) 457–460." />
<meta name="citation_reference" content="N. Zhu, D. Zhang, W. Wang, X. Li, B. Yang, J. Song, X. Zhao, B. Huang, W. Shi, R. Lu, P. Niu, F. Zhan, X. Ma, D. Wang, W. Xu, G. Wu, G.F. Gao, W. Tan, I. China Novel Coronavirus, T. Research, A Novel Coronavirus from Patients with Pneumonia in China, 2019, N Engl J Med 382(8) (2020) 727–733." />
<meta name="citation_reference" content="C. Huang, Y. Wang, X. Li, L. Ren, J. Zhao, Y. Hu, L. Zhang, G. Fan, J. Xu, X. Gu, Z. Cheng, T. Yu, J. Xia, Y. Wei, W. Wu, X. Xie, W. Yin, H. Li, M. Liu, Y. Xiao, H. Gao, L. Guo, J. Xie, G. Wang, R. Jiang, Z. Gao, Q. Jin, J. Wang, B. Cao, Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China, Lancet 395(10223) (2020) 497–506." />
<meta name="citation_reference" content="M. Tahir ul Qamar, S.M. Alqahtani, M.A. Alamri, L.-L. Chen, Structural Basis of SARS-CoV-2 3CLpro and Anti-COVID-19 Drug Discovery from Medicinal Plants, Preprints 2020020193 (2020)." />
<meta name="citation_reference" content="D.S. Hui, E. I Azhar, T.A. Madani, F. Ntoumi, R. Kock, O. Dar, G. Ippolito, T.D. Mchugh, Z.A. Memish, C. Drosten, The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health—The latest 2019 novel coronavirus outbreak in Wuhan, China, International Journal of Infectious Diseases 91 (2020) 264–266." />
<meta name="citation_reference" content="M.A. Alamri, M. Tahir ul Qamar, S.M. Alqahtani, Pharmacoinformatics and Molecular Dynamic Simulation Studies Reveal Potential Inhibitors of SARS-CoV-2 Main Protease 3CLpro, Preprints 2020020308 (2020)." />
<meta name="citation_reference" content="T. Ahmed, M. Noman, A. Almatroudi, M. Shahid, M. Khurshid, F. Tariq, M.T. ul Qamar, R. Yu, B. Li, Coronavirus Disease 2019 Assosiated Pneumonia in China: Current Status and Future Prospects, Preprints 2020020358 (2020)." />
<meta name="citation_reference" content="H. Lu, Drug treatment options for the 2019-new coronavirus (2019-nCoV), Biosci Trends 14(1) (2020) 69–71." />
<meta name="citation_reference" content="M.I. Abdelmageed, A.H. Abdelmoneim, M.I. Mustafa, N.M. Elfadol, N.S. Murshed, S.W. Shantier, A.M. Makhawi, Design of multi epitope-based peptide vaccine against E protein of human 2019-nCoV: An immunoinformatics approach, BioRxiv (2020)." />
<meta name="citation_reference" content="M. Wang, R. Cao, L. Zhang, X. Yang, J. Liu, M. Xu, Z. Shi, Z. Hu, W. Zhong, G. Xiao, Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro, Cell Res 30(3) (2020) 269–271." />
<meta name="citation_reference" content="E. De Gregorio, R. Rappuoli, Vaccines for the future: learning from human immunology, Microb Biotechnol 5(2) (2012) 149–55." />
<meta name="citation_reference" content="A. Patronov, I. Doytchinova, T-cell epitope vaccine design by immunoinformatics, Open biology 3(1) (2013) 120139." />
<meta name="citation_reference" content="N. Gupta, H. Regar, V.K. Verma, D. Prusty, A. Mishra, V.K. Prajapati, Receptor-ligand based molecular interaction to discover adjuvant for immune cell TLRs to develop next-generation vaccine, Int J Biol Macromol 152 (2020) 535–545." />
<meta name="citation_reference" content="P. Kalita, D.L. Lyngdoh, A.K. Padhi, H. Shukla, T. Tripathi, Development of multi-epitope driven subunit vaccine against Fasciola gigantica using immunoinformatics approach, International journal of biological macromolecules 138 (2019) 224–233." />
<meta name="citation_reference" content="M. Saadi, A. Karkhah, H.R. Nouri, Development of a multi-epitope peptide vaccine inducing robust T cell responses against brucellosis using immunoinformatics based approaches, Infection, Genetics and Evolution 51 (2017) 227–234." />
<meta name="citation_reference" content="Z. Nain, M.M. Karim, M.K. Sen, U.K. Adhikari, Structural Basis and Designing of Peptide Vaccine using PE-PGRS Family Protein of Mycobacterium ulcerans–An Integrated Vaccinomics Approach, bioRxiv (2019) 795146." />
<meta name="citation_reference" content="M. Tahir Ul Qamar, S. Saleem, U.A. Ashfaq, A. Bari, F. Anwar, S. Alqahtani, Epitope-based peptide vaccine design and target site depiction against Middle East Respiratory Syndrome Coronavirus: an immune-informatics study, J Transl Med 17(1) (2019) 362." />
<meta name="citation_reference" content="M. Tahir Ul Qamar, A. Bari, M.M. Adeel, A. Maryam, U.A. Ashfaq, X. Du, I. Muneer, H.I. Ahmad, J. Wang, Peptide vaccine against chikungunya virus: immuno-informatics combined with molecular docking approach, J Transl Med 16(1) (2018) 298." />
<meta name="citation_reference" content="B. Ahmad, U.A. Ashfaq, M.U. Rahman, M.S. Masoud, M.Z. Yousaf, Conserved B and T cell epitopes prediction of ebola virus glycoprotein for vaccine development: An immuno-informatics approach, Microb Pathog 132 (2019) 243–253." />
<meta name="citation_reference" content="F. Shahid, U.A. Ashfaq, A. Javaid, H. Khalid, Immunoinformatics guided rational design of a next generation multi epitope based peptide (MEBP) vaccine by exploring Zika virus proteome, Infect Genet Evol 80 (2020) 104199." />
<meta name="citation_reference" content="A. Ikram, T. Zaheer, F.M. Awan, A. Obaid, A. Naz, R. Hanif, R.Z. Paracha, A. Ali, A.K. Naveed, H.A. Janjua, Exploring NS3/4A, NS5A and NS5B proteins to design conserved subunit multi-epitope vaccine against HCV utilizing immunoinformatics approaches, Sci Rep 8(1) (2018) 16107." />
<meta name="citation_reference" content="R.K. Pandey, S. Dahiya, J. Mahita, R. Sowdhamini, V.K. Prajapati, Vaccination and immunization strategies to design Aedes aegypti salivary protein based subunit vaccine tackling Flavivirus infection, International journal of biological macromolecules 122 (2019) 1203–1211." />
<meta name="citation_reference" content="V. Chauhan, M.P. Singh, Immuno-informatics approach to design a multi-epitope vaccine to combat cytomegalovirus infection, Eur J Pharm Sci 147 (2020) 105279." />
<meta name="citation_reference" content="M.U. Mirza, S. Rafique, A. Ali, M. Munir, N. Ikram, A. Manan, O.M. Salo-Ahen, M. Idrees, Towards peptide vaccines against Zika virus: Immunoinformatics combined with molecular dynamics simulations to predict antigenic epitopes of Zika viral proteins, Sci Rep 6 (2016) 37313." />
<meta name="citation_reference" content="A. Khan, M. Junaid, A.C. Kaushik, A. Ali, S.S. Ali, A. Mehmood, D.-Q. Wei, Computational identification, characterization and validation of potential antigenic peptide vaccines from hrHPVs E6 proteins using immunoinformatics and computational systems biology approaches, PloS one 13(5) (2018)." />
<meta name="citation_reference" content="C. Katalani, G. Nematzadeh, G. Ahmadian, J. Amani, G. Kiani, P. Ehsani, In silico design and in vitro analysis of a recombinant trivalent fusion protein candidate vaccine targeting virulence factor of Clostridium perfringens, Int J Biol Macromol 146 (2020) 1015–1023." />
<meta name="citation_reference" content="R.L. Hunter, Overview of vaccine adjuvants: present and future, Vaccine 20 Suppl 3 (2002) S7–12." />
<meta name="citation_reference" content="B. Guy, The perfect mix: recent progress in adjuvant research, Nat Rev Microbiol 5(7) (2007) 505–17." />
<meta name="citation_reference" content="D.A. Benson, I. Karsch-Mizrachi, D.J. Lipman, J. Ostell, E.W. Sayers, GenBank, Nucleic acids research 37(suppl_1) (2008) D26–D31." />
<meta name="citation_reference" content="J.M. Walker, The proteomics protocols handbook, Springer 2005." />
<meta name="citation_reference" content="I.A. Doytchinova, D.R. Flower, VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines, BMC Bioinformatics 8(1) (2007) 4." />
<meta name="citation_reference" content="G. Deléage, ALIGNSEC: viewing protein secondary structure predictions within large multiple sequence alignments, Bioinformatics (2017)." />
<meta name="citation_reference" content="M. Kallberg, H. Wang, S. Wang, J. Peng, Z. Wang, H. Lu, J. Xu, Template-based protein structure modeling using the RaptorX web server, Nat Protoc 7(8) (2012) 1511–22." />
<meta name="citation_reference" content="R.K. Pandey, T.K. Bhatt, V.K. Prajapati, Novel Immunoinformatics Approaches to Design Multi-epitope Subunit Vaccine for Malaria by Investigating Anopheles Salivary Protein, Sci Rep 8(1) (2018) 1125." />
<meta name="citation_reference" content="A. Waterhouse, M. Bertoni, S. Bienert, G. Studer, G. Tauriello, R. Gumienny, F.T. Heer, T.A.P. de Beer, C. Rempfer, L. Bordoli, R. Lepore, T. Schwede, SWISS-MODEL: homology modelling of protein structures and complexes, Nucleic Acids Res 46(W1) (2018) W296–W303." />
<meta name="citation_reference" content="M.D. Cooper, The early history of B cells, Nat Rev Immunol 15(3) (2015) 191–7." />
<meta name="citation_reference" content="J. Zheng, In Silico Analysis of Epitope-Based Vaccine Candidates against Hepatitis B Virus Polymerase Protein, Viruses 9(5) (2017)." />
<meta name="citation_reference" content="A.S. Kolaskar, P.C. Tongaonkar, A semi-empirical method for prediction of antigenic determinants on protein antigens, FEBS Lett 276(1-2) (1990) 172–4." />
<meta name="citation_reference" content="J. Ponomarenko, H.-H. Bui, W. Li, N. Fusseder, P.E. Bourne, A. Sette, B. Peters, ElliPro: a new structure-based tool for the prediction of antibody epitopes, BMC bioinformatics 9(1) (2008) 514." />
<meta name="citation_reference" content="Z. Nain, F. Abdulla, M.M. Rahman, M.M. Karim, M.S.A. Khan, S.B. Sayed, S. Mahmud, S.R. Rahman, M.M. Sheam, Z. Haque, Proteome-wide screening for designing a multi-epitope vaccine against emerging pathogen Elizabethkingia anophelis using immunoinformatic approaches, Journal of Biomolecular Structure and Dynamics (2019) 1–18." />
<meta name="citation_reference" content="R. Vita, J.A. Overton, J.A. Greenbaum, J. Ponomarenko, J.D. Clark, J.R. Cantrell, D.K. Wheeler, J.L. Gabbard, D. Hix, A. Sette, B. Peters, The immune epitope database (IEDB) 3.0, Nucleic Acids Res 43(Database issue) (2015) D405–12." />
<meta name="citation_reference" content="W. Fleri, S. Paul, S.K. Dhanda, S. Mahajan, X. Xu, B. Peters, A. Sette, The Immune Epitope Database and Analysis Resource in Epitope Discovery and Synthetic Vaccine Design, Front Immunol 8 (2017) 278." />
<meta name="citation_reference" content="H.H. Bui, J. Sidney, W. Li, N. Fusseder, A. Sette, Development of an epitope conservancy analysis tool to facilitate the design of epitope-based diagnostics and vaccines, BMC Bioinformatics 8(1) (2007) 361." />
<meta name="citation_reference" content="S.N. Sarkar, G.C. Sen, Novel functions of proteins encoded by viral stress-inducible genes, Pharmacol Ther 103(3) (2004) 245–59." />
<meta name="citation_reference" content="S. Fuse, M.J. Molloy, E.J. Usherwood, Immune responses against persistent viral infections: possible avenues for immunotherapeutic interventions, Crit Rev Immunol 28(2) (2008) 159–83." />
<meta name="citation_reference" content="M. Tahir ul Qamar, F. Shahid, U. Ali, A.Z. Fareed, L.-L. Chen, Structural modeling and conserved epitopes prediction against SARS-COV-2 structural proteins for vaccine development, Research Square (2020)." />
<meta name="citation_reference" content="A. Lamiable, P. Thevenet, J. Rey, M. Vavrusa, P. Derreumaux, P. Tuffery, PEP-FOLD3: faster de novo structure prediction for linear peptides in solution and in complex, Nucleic Acids Res 44(W1) (2016) W449–54." />
<meta name="citation_reference" content="M. Tahir Ul Qamar, A. Maryam, I. Muneer, F. Xing, U.A. Ashfaq, F.A. Khan, F. Anwar, M.H. Geesi, R.R. Khalid, S.A. Rauf, A.R. Siddiqi, Computational screening of medicinal plant phytochemicals to discover potent pan-serotype inhibitors against dengue virus, Sci Rep 9(1) (2019) 1433." />
<meta name="citation_reference" content="S. Durdagi, M. Tahir Ul Qamar, R.E. Salmas, Q. Tariq, F. Anwar, U.A. Ashfaq, Investigating the molecular mechanism of staphylococcal DNA gyrase inhibitors: A combined ligand-based and structure-based resources pipeline, J Mol Graph Model 85 (2018) 122–129." />
<meta name="citation_reference" content="W.L. DeLano, Pymol: An open-source molecular graphics tool, CCP4 Newsletter on protein crystallography 40(1) (2002) 82–92." />
<meta name="citation_reference" content="S.J. de Vries, M. van Dijk, A.M. Bonvin, The HADDOCK web server for data-driven biomolecular docking, Nat Protoc 5(5) (2010) 883–97." />
<meta name="citation_reference" content="E. Gasteiger, C. Hoogland, A. Gattiker, M.R. Wilkins, R.D. Appel, A. Bairoch, Protein identification and analysis tools on the ExPASy server, The proteomics protocols handbook, Springer 2005, pp. 571–607." />
<meta name="citation_reference" content="I. Dimitrov, D.R. Flower, I. Doytchinova, AllerTOP-a server for in silico prediction of allergens, BMC bioinformatics, BioMed Central, 2013, p. S4." />
<meta name="citation_reference" content="L.J. McGuffin, K. Bryson, D.T. Jones, The PSIPRED protein structure prediction server, Bioinformatics 16(4) (2000) 404–5." />
<meta name="citation_reference" content="L. Heo, H. Park, C. Seok, GalaxyRefine: Protein structure refinement driven by side-chain repacking, Nucleic Acids Res 41(Web Server issue) (2013) W384–8." />
<meta name="citation_reference" content="S.C. Lovell, I.W. Davis, W.B. Arendall III, P.I. De Bakker, J.M. Word, M.G. Prisant, J.S. Richardson, D.C. Richardson, Structure validation by Cα geometry: ϕ, ? and Cβ deviation, Proteins: Structure, Function, and Bioinformatics 50(3) (2003) 437–450." />
<meta name="citation_reference" content="M. Wiederstein, M.J. Sippl, ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins, Nucleic Acids Res 35(Web Server issue) (2007) W407–10." />
<meta name="citation_reference" content="M. Lengths, M. Angles, Limitations of structure evaluation tools errat, Quick Guideline Comput Drug Des 16 (2018) 75." />
<meta name="citation_reference" content="A. Kuriata, A.M. Gierut, T. Oleniecki, M.P. Ciemny, A. Kolinski, M. Kurcinski, S. Kmiecik, CABS-flex 2.0: a web server for fast simulations of flexibility of protein structures, Nucleic Acids Res 46(W1) (2018) W338–W343." />
<meta name="citation_reference" content="M. Kurcinski, T. Oleniecki, M.P. Ciemny, A. Kuriata, A. Kolinski, S. Kmiecik, CABS-flex standalone: a simulation environment for fast modeling of protein flexibility, Bioinformatics 35(4) (2019) 694–695." />
<meta name="citation_reference" content="N. Rapin, O. Lund, M. Bernaschi, F. Castiglione, Computational immunology meets bioinformatics: the use of prediction tools for molecular binding in the simulation of the immune system, PLoS One 5(4) (2010)." />
<meta name="citation_reference" content="A.T. Kroger, General recommendations on immunization, US Department of Health and Human Services, Public Health Service, Centers for Disease Control (2013)." />
<meta name="citation_reference" content="F. Castiglione, F. Mantile, P. De Berardinis, A. Prisco, How the interval between prime and boost injection affects the immune response in a computational model of the immune system, Computational and mathematical methods in medicine 2012 (2012)." />
<meta name="citation_reference" content="R.A. Laskowski, PDBsum new things, Nucleic Acids Res 37(Database issue) (2009) D355–9." />
<meta name="citation_reference" content="T. Hansson, C. Oostenbrink, W. van Gunsteren, Molecular dynamics simulations, Curr Opin Struct Biol 12(2) (2002) 190–6." />
<meta name="citation_reference" content="M.J. Abraham, T. Murtola, R. Schulz, S. Páll, J.C. Smith, B. Hess, E. Lindahl, GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers, SoftwareX 1 (2015) 19–25." />
<meta name="citation_reference" content="I. Muneer, M.T. Ul Qamar, K. Tusleem, S. Abdul Rauf, H.M.J. Hussain, A.R. Siddiqi, Discovery of selective inhibitors for cyclic AMP response element-binding protein: a combined ligand and structure-based resources pipeline, Anticancer Drugs 30(4) (2019) 363–373." />
<meta name="citation_reference" content="A. Grote, K. Hiller, M. Scheer, R. Munch, B. Nortemann, D.C. Hempel, D. Jahn, JCat: a novel tool to adapt codon usage of a target gene to its potential expression host, Nucleic Acids Res 33(Web Server issue) (2005) W526–31." />
<meta name="citation_reference" content="C.L. Smith, J.G. Econome, A. Schutt, S. Klco, C.R. Cantor, A physical map of the Escherichia coli K12 genome, Science 236(4807) (1987) 1448–53." />
<meta name="citation_reference" content="P.M. Sharp, W.H. Li, The codon Adaptation Index--a measure of directional synonymous codon usage bias, and its potential applications, Nucleic Acids Res 15(3) (1987) 1281–95." />
<meta name="citation_reference" content="R.V. Luckheeram, R. Zhou, A.D. Verma, B. Xia, CD4+ T cells: differentiation and functions, Clinical and developmental immunology 2012 (2012)." />
<meta name="citation_reference" content="R. Arai, H. Ueda, A. Kitayama, N. Kamiya, T. Nagamune, Design of the linkers which effectively separate domains of a bifunctional fusion protein, Protein engineering 14(8) (2001) 529–532." />
<meta name="citation_reference" content="R.K. Pandey, A. Narula, M. Naskar, S. Srivastava, P. Verma, R. Malik, P. Shah, V.K. Prajapati, Exploring dual inhibitory role of febrifugine analogues against Plasmodium utilizing structure-based virtual screening and molecular dynamic simulation, Journal of Biomolecular Structure and Dynamics 35(4) (2017) 791–804." />
<meta name="citation_reference" content="N. Nezafat, Z. Karimi, M. Eslami, M. Mohkam, S. Zandian, Y. Ghasemi, Designing an efficient multi-epitope peptide vaccine against Vibrio cholerae via combined immunoinformatics and protein interaction based approaches, Comput Biol Chem 62 (2016) 82–95." />
<meta name="citation_reference" content="B. Livingston, C. Crimi, M. Newman, Y. Higashimoto, E. Appella, J. Sidney, A. Sette, A rational strategy to design multiepitope immunogens based on multiple Th lymphocyte epitopes, J Immunol 168(11) (2002) 5499–506." />
<meta name="citation_reference" content="F.E. Lund, Cytokine-producing B lymphocytes—key regulators of immunity, Current opinion in immunology 20(3) (2008) 332–338." />
<meta name="citation_reference" content="R. Rehan Khalid, M. Tahir Ul Qamar, A. Maryam, A. Ashique, F. Anwar, H.G. M, A.R. Siddiqi, Comparative Studies of the Dynamics Effects of BAY60-2770 and BAY58-2667 Binding with Human and Bacterial H-NOX Domains, Molecules 23(9) (2018) 2141." />
<meta name="citation_reference" content="N. Khatoon, R.K. Pandey, V.K. Prajapati, Exploring Leishmania secretory proteins to design B and T cell multi-epitope subunit vaccine using immunoinformatics approach, Sci Rep 7(1) (2017) 8285." />
<meta name="citation_reference" content="K.K. Dubey, G.A. Luke, C. Knox, P. Kumar, B.I. Pletschke, P.K. Singh, P. Shukla, Vaccine and antibody production in plants: developments and computational tools, Brief Funct Genomics 17(5) (2018) 295–307." />
<meta name="citation_reference" content="X. Yang, X. Yu, An introduction to epitope prediction methods and software, Rev Med Virol 19(2) (2009) 77–96." />
<meta name="citation_reference" content="I.J. Amanna, M.K. Slifka, Contributions of humoral and cellular immunity to vaccine-induced protection in humans, Virology 411(2) (2011) 206–15." />
<meta name="citation_reference" content="A. Pashine, N.M. Valiante, J.B. Ulmer, Targeting the innate immune response with improved vaccine adjuvants, Nature medicine 11(4) (2005) S63–S68." />
<meta name="citation_reference" content="S. Shamriz, H. Ofoghi, N. Moazami, Effect of linker length and residues on the structure and stability of a fusion protein with malaria vaccine application, Comput Biol Med 76 (2016) 24–9." />
<meta name="twitter:title" content="Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world’s population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated." />
<meta name="og-title" property="og:title" content="Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world’s population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-22" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.28.970343v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.28.970343v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.28.970343v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1206600" />
    <title>Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ddJc0p9iA4GdhTWHANMUWqqJ0WvBfHCdxxSVGTvmn1s","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1206600","apath":"\/biorxiv\/early\/2020\/03\/22\/2020.02.28.970343.atom","pisa":"biorxiv;2020.02.28.970343v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.28.970343v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.28.970343v2","title":"Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches","identifier":"node\/1206600"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zPFI+xFuHnafQVMsUPYpZ5emdUjhhnKMHNI00vHTyQssNIZnjDKyOcP4EJS4x6XiKR2+maNw1Wju2G5SBp787vN6vwBMTUi9EHD9hAUZRv\\\\\\\/fYYVZqCcAyINTuZr83GLPCpQhLpDekblrSE09OCc5nq+YOzlADG1kPk0fp\\\\\\\/QbEB9ZjVPrL2zsr089VmHdlek5fP02FqseKf+48l0RmrYSunwJy1rX4rvAFXwriw2gj3v5KpNGfmtyUs9CHMbNMXRJOj\\\\\\\/pEKe8M322dcAIjkwn0BTgiNj35+pJ5FIXNcfwm4Nb6BTBxaUHcdhvEXmt94rsqC0x9xGcnIbJtEyVKNj9gSanVMY+DjR47bywRC2HUMpkB7SF2jWHgQS1AC5nLUTC\\\\\\\/k8Rl7s5B7DjSTXy0ah0szkmQdkfWAp8ye3NWwhMDt\\\\\\\/H1OjKvuekB1LSwdkpv8MFz39gRPKuA\\\\\\\/P9XHaniibh8pJfO0PxgOPS+pSgcNgW51Yjpruot4iLYlOO6YGE0+WynDMNA9U3b0Kges4DEn3yhizktJBtqhAy9MVPTfmD5IQ8W6ApEFM7Ay3Yb3v5HIs3Vw7wfxy36DQNwyz70LI2x6S5hFYIVeLM+5l3\\\\\\\/x3qoi+pKr\\\\\\\/F068rYs0jl1tkH3+WBOClkRcD34OfNefGxIGrXZJh2HGkYcrokn0s\\\\\\\/0cJExW14ucfIAwdxvRDOXbKr8xF5veGsMks6W3NOwC8Qzx1hAQ63psF9oDpUUdz0cEe5J+zB32VQhT6Kad0pY7TLGZVgur1V8PhuKpNvCWyr2Lfmh5WLL2qYWpCUNtVFHLqjykNqkz8al4hbaYoHYCUZEVIp+yl7MtrmzeVOsi+h1kHUaKavFBtrz8Oz6FkdHfv9sXsPROLr+elCYvdEEQH0vKhHkqdmQSEf9O\\\\\\\/bniKy\\\\\\\/lTMIPO0Q33uvAHtWa5HcmvK1lJRgDGNLQ3wVqctO7z96w9MUUDM0cMo7kUDPrlkSbALsXfVFd242Lt5LG3598lt7vFgunXYGtLzJdrnpYvh1KjaeW6nx+qrvyh3yc9ridIWIP59pN6dZcEzqT\\\\\\\/CBy68czc8F6omBU1J4Gmt29sYMMy8G0MpFahJc6yXrU6xyibwahAEhgM7+zdDnKt1GTkUVe5\\\\\\\/JX1YU50FO6js+ht0q63VD+fwDNlK8Wkf3vXU5bJW3ecBYQVmLAvaJ0kuf\\\\\\\/0TPSntlJkTZVPwbPK83P6xUy9JQZwb3JDdawwliYj046BlB7VKPj\\\\\\\/vQF1igyKhjiOwViUz0CTs5aBUTYdlJ60fRNlW8DxEcHyr5r1L2BRx6slM7ZHsz0NEfj+pR3YJ3VFE\\\\\\\/gtq9R\\\\\\\/u9+WhGZbG7+Cf0bw\\\u0022,\\\u0022iv\\\u0022:\\\u0022vAKEymnz3AggkZGignDZOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022abe60f8c759556962acbb4f339fa0bb502128a81a90648a80c2e616b187366f9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.28.970343v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1206600 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.28.970343v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ofcxW4vdgqbFvySKO7HNUam3sK4C94MdAoFzhn5G52I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1206600" id="node1206600" data-pisa="biorxiv;2020.02.28.970343v2" data-pisa-master="biorxiv;2020.02.28.970343" data-apath="/biorxiv/early/2020/03/22/2020.02.28.970343.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and <em>in silico</em> approaches</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4832-4250" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Muhammad Tahir</span> <span class="nlm-surname">ul Qamar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Abdur</span> <span class="nlm-surname">Rehman</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Usman Ali</span> <span class="nlm-surname">Ashfaq</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhammad Qasim</span> <span class="nlm-surname">Awan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Israr</span> <span class="nlm-surname">Fatima</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Farah</span> <span class="nlm-surname">Shahid</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ling-Ling</span> <span class="nlm-surname">Chen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.28.970343 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1206600" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Muhammad Tahir ul Qamar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>College of Life Science and Technology, Guangxi University</span>, Nanning, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhammad%2BTahir%2Bul%2BQamar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=ul%20Qamar%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMuhammad%2BTahir%2Bul%2BQamar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4832-4250" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Muhammad Tahir ul Qamar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abdur Rehman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Bioinformatics and Biotechnology, Government College University Faisalabad</span>, <span class='nlm-country'>Pakistan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdur%2BRehman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rehman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdur%2BRehman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Usman Ali Ashfaq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Bioinformatics and Biotechnology, Government College University Faisalabad</span>, <span class='nlm-country'>Pakistan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Usman%2BAli%2BAshfaq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashfaq%20UA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUsman%2BAli%2BAshfaq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:llchen@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">llchen@mail.hzau.edu.cn</a>
<a href="mailto:usmancemb@gmail.com" class="" data-icon-position="" data-hide-link-title="0">usmancemb@gmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Muhammad Qasim Awan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Bioinformatics and Biotechnology, Government College University Faisalabad</span>, <span class='nlm-country'>Pakistan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhammad%2BQasim%2BAwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Awan%20MQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMuhammad%2BQasim%2BAwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Israr Fatima </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Bioinformatics and Biotechnology, Government College University Faisalabad</span>, <span class='nlm-country'>Pakistan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Israr%2BFatima%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fatima%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsrar%2BFatima%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Farah Shahid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Bioinformatics and Biotechnology, Government College University Faisalabad</span>, <span class='nlm-country'>Pakistan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Farah%2BShahid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shahid%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFarah%2BShahid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ling-Ling Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>College of Life Science and Technology, Guangxi University</span>, Nanning, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ling-Ling%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALing-Ling%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:llchen@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">llchen@mail.hzau.edu.cn</a>
<a href="mailto:usmancemb@gmail.com" class="" data-icon-position="" data-hide-link-title="0">usmancemb@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.28.970343v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.28.970343v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.28.970343v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.28.970343v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.28.970343v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.28.970343v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1206600" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1206600/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world’s population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and <em>in silico</em> cloning of MEV ensured increased expression in the <em>Escherichia coli</em> K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-4">Viruses are dangerous pathogens and can cause irreversible losses to human lives and economy. The world hardly learns to deal with a virus when new emerges and threatens the future of humanity. A similar situation arises when a new strain of coronavirus (CoV) not previously identified in humans was reported last year (2019) [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Positive-sense RNA viruses called corona viruses belong to the Coronaviridae family that are distributed broadly among human and mammals. In the last two decades there have been more than 10,000 reported infections of two types of coronaviruses: severe acute respiratory coronavirus (SARS-COV) or Middle East Coronavirus (MERS-COV) [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Last year December (2019), few cases of unknown viral based pneumonia were reported in Wuhan city (Hubei province, P.R. China). Later it was identified that, this pneumonia is linked with a novel type of CoV outbreak. World Health Organization (WHO) named the virus SARS-CoV-2 and linked disease as coronavirus disease 2019 (COVID-19). The COVID-19 quickly widespread around the globe and claimed thousands of human lives. WHO announced global health emergency and recognized COVID-19 as pandemic [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. SARS-CoV-2 was indicated root cause of COVID-19 through deep sequencing analysis from lower respiratory tract samples of patients [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. SARS-COV-2 genome sequence is almost 70% similar to SARS-COV, and 40% similar to the MERS-COV [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Symptoms of SARS-COV-2 may arise within 2 days or up-to 14 days of exposure. Symptoms such as fever, diarrhea and respiratory disorder are found in infected patients [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. According to latest research SARS-CoV-2 has an identical genomic organization as of beta-coronaviruses; 5’-untranslated region (UTR), orf1ab (replicas complex), nsps (encoding non-structural proteins), S (spike) protein, E (envelope) protein, M (membrane) protein, Oraf6, orf7a, orf8, N (nucleocapsid) protein, orf10, 3’-UTR and several unknown non-structural open reading frames [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>].</p><p id="p-5">There is currently no vaccine or approved treatment for COVID-19. Few traditional Chinese medicine such as Shufengjiedu capsules and Lianhuaqingwen capsules were reported effective against COVID-19 [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Nonetheless, no clinical trials support the safety and efficacy of these medicinal products. Similarly, clinical trials are in process for few experimental drugs, including remdesivir and chloroquine which were found effective <em>in vitro</em> against COVID-19 [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. However, there is no clinical trial-based drug or vaccine reported yet. To prevent viral diseases vaccine is the most effective approach. Now a days, availability of genomic information, advance software and immunological data sets could greatly facilitate researchers to identify the effective epitopes from pathogens’ proteins that can be used to develop active sub-unit vaccines [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>-<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. The subunit vaccine contains the fragments of antigenic proteins of pathogen that can trigger an immune response against the target pathogen [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. In recent studies, few candidate vaccine constructs were reported against MERS-CoV [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], Chikungunya virus [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], Ebola virus [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], Zika virus [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], HCV [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], Flavivirus [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] and Cytomegalovirus [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] with promising results. The <em>in silico</em> methods reduce the number of <em>in vitro</em> experiments and save time, overcome cost obstacles and increase the potential for successful vaccine design [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>-<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>].</p><p id="p-6">In present study, SARS-CoV-2 proteome was explored to determine the potent antigenic proteins and their further screening for B-cell, T-cell and IFN-γ inducing epitopes prediction with their MHC (major histocompatibility complex) alleles. Antigenicity, allergenicity and toxicity of predicted epitopes were analyzed. Multiepitope based vaccine (MEV) construct was designed by using the most potential and interacting epitopes, with the addition of suitable linkers and an adjuvant. Adjuvants are generally defined as molecules that may increase or modulate the intrinsic immunogenicity of an antigen [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Adjuvants are essential to reduce the amount of antigen and the number of injections, as they help to induce effective and persistent immune responses [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Several <em>in silico</em> approaches were utilized to validate the antigenicity, immunogenicity, allergenicity, structural stability/flexibility and physiochemical properties of designed MEV. Furthermore, molecular docking and MD simulations analyses were carried out to investigate binding interaction and stability of the MEV with human pathogenic receptors. At the end, the MEV codons were optimized for <em>E. coli</em> system and <em>in silico</em> cloning was performed to ensure its expression profiling (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The overall experimental workflow used to develop MEV construct against SARS-COV-2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The overall experimental workflow used to develop MEV construct against SARS-COV-2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F1.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F1.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-7" class="first-child">The overall experimental workflow used to develop MEV construct against SARS-COV-2.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">2. Material and methods</h2><div id="sec-3" class="subsection"><h3>2.1. Sequence retrieval and analysis of antigenic proteins</h3><p id="p-8">In the first step, whole proteome of SARS-CoV-2 was retrieved from GENBANK [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. After that, individual protein sequences were extracted and stored as FASTA format for further analysis.</p><div id="sec-4" class="subsection"><h4>2.1.1. Antigenicity and physiochemical properties evaluation</h4><p id="p-9">The Expassy Protparam tool was used to determine the physical and chemical properties of selected proteins [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. To check protein antigenicity, the Vaxijen 2.0 software was used [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. The threshold value was held at 0.5, and the secondary structure of proteins was predicted by using SOPMA tool [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>].</p></div><div id="sec-5" class="subsection"><h4>2.1.2. Tertiary structure prediction and refinement of target proteins</h4><p id="p-10">Tertiary stores of most of SARS-CoV-2 proteins are not reported yet. Therefore, combinations of different approaches were employed to predict good quality structures for further analysis. Online tools such as Swiss model which work on homology based modeling algorithms and Raptor X which work on deep learning modules, were primarily used for the tertiary structure prediction of SARS-COV-2 proteins [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>-<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Predicted Models were then refined by galaxy refine server and validated by Ramachandran plot analysis.</p></div></div><div id="sec-6" class="subsection"><h3>2.2. Epitopes prediction and validation</h3><div id="sec-7" class="subsection"><h4>2.2.1. B-cell epitope prediction</h4><p id="p-11">In immune system the B-Cell epitope helps to detect viral infection and activities. B-cells provide humoral immunity by secreting immunoglobulins which can neutralize antigen upon binding. A surface receptor of B-cell recognizes B-cell epitopes, resulting the generation of antigen-specific immunoglobulins [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. B-cell epitopes are two types, linear (continuous) and conformational (discontinuous). An online database of ABCPred was used to predict linear B-cell epitopes [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Conformational epitopes were predicted by Ellipro server [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>].</p></div><div id="sec-8" class="subsection"><h4>2.2.2. T-cell Epitope prediction</h4><p id="p-12">T-cells express TCRs (T-cell receptors), that could recognize specific antigens and can generate cellular and humoral immune response against them [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. The Immune Epitope Database (IEDB) consensus method [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>] was used to predict MHC classes I and MHC II epitopes with consensus score less than 2 filter cut.</p></div><div id="sec-9" class="subsection"><h4>2.2.3. Immunogenicity prediction of epitopes</h4><p id="p-13">To evaluate the antigenicity, allergenicity and toxicity of B-cell and T-cell epitope, Vaxijen v2.0 and AllerTOP v2.0 were used. The filtering criteria 0.5 was used. Only filtered, non-antigenic, non-allergenic and non-toxic epitopes were selected for further studies.</p></div><div id="sec-10" class="subsection"><h4>2.2.4. Conservation analysis of epitopes</h4><p id="p-14">IEDB conservancy analysis tool [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>] was used, to monitor the degree of conservation in the sequence of B-cell and T-cell epitopes. Epitopes showing 100 percent conservation were shortlisted for further analysis.</p></div><div id="sec-11" class="subsection"><h4>2.2.5. Interferon-γ <em>inducing epitope prediction</em></h4><p id="p-15">IFN-γ is acknowledged to elicit intrinsic safe responses and can directly detain viral duplication [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. Besides, they can trigger the versatile immune reactions by preparing cytotoxic T lymphocyte (CTL) and Helper T lymphocyte (HTL). IFN epitope server was used to calculate IFN-γ epitopes of selected proteins of SARS-CoV-2 using SVM hybrid algorithms along with Motif [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>].</p></div><div id="sec-12" class="subsection"><h4>2.2.6. Epitopes modeling and molecular docking</h4><p id="p-16">Epitopes which show favorable strong binding affinities with a common experimentally validated allele, are good choice to design MEV construct. Therefore, molecular docking between screened epitopes and human allele was performed. Molecular docking is an <em>in silico</em> approach which determined the binning affinity between ligand and its target proteins, and also highlight the important residues involve in the interaction. 3D structures of overlapping, highly antigenic and conserved epitopes with corresponding common alleles were predicted using PEPFOLD [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. The X-ray crystallographic structure of a common human allele (HLA-B7) was retrieved from protein data bank (PDB ID: 3VCL). Molecular docking was performed using same protocol of our previously published studies [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. To visualize the docked complexes and draw figures, the PyMOL molecular graphics system was used [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>].</p></div><div id="sec-13" class="subsection"><h4>2.2.7. Population coverage analysis of selected epitopes</h4><p id="p-17">The selected epitopes for MEV construct should effectively cover major populations across the globe. For population coverage, overlapping, antigenic, conserved and strongly interacting with HLA-B7 allele epitopes were selected and further analysed using the IEDB population coverage analysis tool by maintaining the default analysis parameters <sup>[<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>]</sup>. This tool is designed to estimate the population coverage of epitopes from diverse counties based on the distribution of their MHC-binding alleles. As SARS-CoV-2 is global pandemic, therefore, worldwide analysis has been performed.</p></div></div><div id="sec-14" class="subsection"><h3>2.3. Construction of multi epitope vaccine</h3><p id="p-18">To construct a sub-unit vaccine, the epitopes with following properties are usually preferred: (a) 100% conserved, (b) overlapping, (c) highly immunogenic, (d) non-allergic, (e) non-toxic, (f) with significant population coverage, (g) having strong binding affinity with common human allele and (h) have no similarity with the human proteins. Therefore, only those CTLs, HTLs and B-cell epitope were further selected which passed all the eight above parameters to construct SARS-CoV-2 MEV. To boost the immune response an adjuvant was attached with the first CTL epitope with the EAAAK linker, while other epitopes were connected using AAY and GPGPG linkers to preserve their independent immunogenic activities after their inter-interaction compatibility validation using HADDOCK Guru interface [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>].</p><div id="sec-15" class="subsection"><h4>2.3.1. Immunogenic and physicochemical properties evaluation of the vaccine construct</h4><p id="p-19">The vaccine construct should be stable, antigenic and non-allergic. The ProtParam tool [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>] was used to evaluate the physicochemical properties of MEV. It analyzes different physical and chemical features of proteins including grand average hydropathy, half-life, stability/instability index, theoretical pI, aliphatic index hydropathy and half-life. The MEV properties were further verified using VaxiJen 2.0 and AllerTOP V2.0 servers [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Moreover, PSIPRED was used to analysed the secondary structure of MEV [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>].</p></div><div id="sec-16" class="subsection"><h4>2.3.2. Tertiary structure prediction of vaccine construct</h4><p id="p-20">As vaccine construct is combination of different epitopes, therefore, the RaptorX server was used to develop MEV 3D tertiary structure. The RaptorX server use a multi-template threading approach to determine the tertiary structure of query protein [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>].</p></div><div id="sec-17" class="subsection"><h4>2.3.3. Refinement and validation of vaccine construct</h4><p id="p-21">Galaxy Refine server MD simulation approach was used to refine the MEV predicted 3D structure [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. To verify refined MEV structure quality, Ramachandran plot analysis was performed using RAMPAGE server [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], followed by structural validation analysis using ProSA-web server [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. The quality scores outside the normal range of natural proteins during structure validation process reveal potential defects in the protein structure model. Therefore, ERRAT server was further used to evaluate the statistics of non-bonded interactions in MEV construct [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]. Furthermore, MEV structural flexibility was also analysed using CABS-Flex 2.0 server [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. The flexibility of vaccine is an important aspect for its functionating, and CABS-Flex server provides a detail overview of flexibility and stability of query protein by simulating its residues [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>].</p></div><div id="sec-18" class="subsection"><h4>2.3.4. Immunogenicity evaluation of the vaccine construct</h4><p id="p-22">An <em>in silico</em> immune simulation was performed using C-ImmSim 10.1 server [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>], in order to validate immunological response of constructed MEV. This server simulates the three major functional mamal system components (bone marrow, thymus and lymph node) [<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a>]. The MEV has been tested for the ability to simulate various types of immune cells such as HTL, CTL, B-cells, NK cells, dendritic cells, Immunoglobulins and cytokines. Clinically the minimum recommended interval between two doses of vaccines is four weeks [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Consequently, three injections (each injection contain a thousand units of MEV) were administered using C-ImmSim immunostimulatory, with the recommended interval of four weeks (1, 84 and 168 time-steps parameters were set as 1 time-step is equal to eight hours of real life) for a total of 1000 steps of simulation. Other parameters were kept as default.</p></div></div><div id="sec-19" class="subsection"><h3>2.4. Molecular docking of vaccine construct with human immune receptor</h3><p id="p-23">The host produces an efficient immune response if an antigen / vaccine interacts properly with the target immune cells. Therefore, molecular docking analysis was performed to analyse integrations between the MEV and the human immune receptors. TLR3 and TLR8 have been extensively studied and studies found their vital roles in the generation of antiviral immune response. HADDOCK was used for the MEV docking with TLR3 (PDB ID: 1ZIW) and TLR8 (PDB ID: 3W3 G). To visualize the docked complex and draw figures, the PyMOL educational version was used [<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a>]. In addition, the online database PDBsum was used to demonstrate the interacting residues of docked complexes [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>].</p></div><div id="sec-20" class="subsection"><h3>2.5. Molecular dynamics simulation analysis of vaccine construct and receptors complexes</h3><p id="p-24">MD simulation is an important approach to analyse the stability of the receptor-ligand complex [<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. Complexes of MEV with TLR3 and TLR8 were simulated at 20 ns using GROMACS 5.1.4 [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>] by following the same protocol of our previously published studies [<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. The trajectories were saved for each complex after every 2 fs and root mean square deviation (RMSD) and root mean square fluctuations (RMSF) analysis were performed using GROMCAS tools.</p></div><div id="sec-21" class="subsection"><h3>2.6. Codon optimization and in silico cloning</h3><p id="p-25">Codon adaptation is a way of increasing the translation efficacy of external genes in the host if the use of codon in both species varies. After carefully evaluating MEV properties and immune response, finally its codon optimization was performed followed by <em>in silico</em> cloning. The Java Codon Adaptation Tool (JCAT) server [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>] was used for codon optimization of MEV to make it compatible with widely used prokaryotic expression system; <em>E. coli</em> K12 [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. The available additional options were selected to evade (a) rho-independent transcription termination, (b) prokaryote ribosome binding site and (c) restriction enzymes cleavage sites. The GC (guanine and cytosine) contents together with codon adaptation index (CAI) [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>] were evaluated. Furthermore, to facilitate restriction and cloning, sticky ends restriction sites of XhoI and HindIII restriction enzymes were introduced at the start/N-terminal and end/C-terminal of the optimized MEV sequence, respectively. Finally, the adapted nucleotide sequence of MEV was cloned into the <em>E. Coli</em> pET28a(+) vector with SnapGene 4.2 tool (<a href="https://snapgene.com/">https:/snapgene.com/</a>) to ensure the <em>in vitro</em> expression.</p></div></div><div class="section" id="sec-22"><h2 class="">3. Results</h2><div id="sec-23" class="subsection"><h3>3.1. Target proteins sequence and structural analysis</h3><p id="p-26">The amino acid sequences of SARS-CoV-2 important vaccine target proteins (ORF1 [QHD43415.1], S [QHD43416.1], ORF3a [QHD43417.1], E [QHD43418.1], M [QHD43419.1], ORF6 [QHD43420.1], ORF7a [QHD443421.1], ORF8 [QHD43422.1], N [QHD43423.2] and ORF10 [QHI42199.1]) were retrieved and saved in FASTA format. Vaxijen server was used to check the antigenicity of proteins. Total 7 highly antigenic proteins were detected. The most antigenic protein found was ORF10, followed by E, M, ORF6, ORF7a, ORF8, and N, with antigenic values 0.7185, 0.6502, 0.6441, 0.6131, 0.6025, 0.5102 and 0.5059 respectively. ORF1, S, and ORF3 proteins had antigenic values less than 0.5, therefore they were excluded from further analysis. Among selected 7 highly antigenic SARS-CoV-2 proteins for further analysis, 5 were non-structural proteins (M, N, ORF6, ORF7a, and ORF10) and 2 were structural proteins (E, ORF8). Other physicochemical characteristics of selected proteins including theoretical pi, molecular weight, half-life, stability profile, aliphatic index, etc were analysed using ProtParam server (Supplementary Table 1) and their secondary structures was predicted using SOPMA tool (Supplementary Table 2).</p><p id="p-27">The 3D models of selected proteins tertiary structures were predicted using Swiss model tool and Raptor X tool, and in order to select best quality models, predicted structures were further refined by galaxy refine server followed by Ramachandran plot validation. The most of structures predicted using Swiss model were of better quality than Raptor X predicted structures, except for N and Orf7a proteins. Therefore, good quality models were selected for further analysis (Supplementary Figure 1). There was no suitable structure predicted for ORF10 because of small number of residues. So, its structure was predicted by PEPFOLD server [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>] (Supplementary Table 3).</p></div><div id="sec-24" class="subsection"><h3>3.2. Prediction of B-cell, T-cell and IFN-γ inducing epitopes</h3><p id="p-28">Screened out B-cell epitopes were 100% conserved in all protein sequences and ere highly antigenic. All the target proteins were predicted to have a total 55 linear epitopes (E-4, M-12, ORF6-1, ORF7a-6, ORF8-9, N-22, and ORF10-1) (Supplementary Table 4). Moreover, a total of 24 (E-4, M-2, ORF6-3, ORF7a-4, ORF8-4, NC-4, and ORF10-3) conformational epitopes were predicted in all target proteins (Supplementary Table 5).</p><p id="p-29">As mentioned before, epitopes that can bind to multiple alleles because of their strong defense capabilities are considered the most appropriate epitopes. Therefore, total 31 MHC class I (E-9, M-4, ORF6-2, ORF7a-3, ORF8-7, N-4, ORF10-2) (Supplementary Table 6) and 40 MHC class II (E-4, M-5, ORF6-4, ORF7a-7, ORF8-12, N-4, ORF10-4) epitopes, 100% conserved among target protein sequences and highly antigenic were chosen for further study (Supplementary Table 7).</p><p id="p-30">The HTLs helps to activate CTLs together with other immune cells upon various types of cytokines release i.e., IFN-γ, interleukin-4 and interleukin-10 [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>]. Thus, HTL epitopes that induce cytokines are vital for the progress of vaccines or immunotherapy. A total of 988 IFN-γ inducing epitopes from target proteins (E-67, M-214, ORF6-53, ORF7a-100, ORF8-113, N-411, ORF10-30) were obtained (Supplementary Table 8).</p></div><div id="sec-25" class="subsection"><h3>3.3. Interaction analysis of epitopes with HLA-B7 allele</h3><p id="p-31">As stated before, to construct a sub-unit vaccine, the chosen epitopes should be 100% conserved, overlapping and antigenic. Therefore, total 50 conserved/antigenic epitopes from selected proteins overlapping in all 3 categories (B-cell, T-cell and IFN-γ) were selected for further validation of their interactions with a common human allele. The 3D structures of selected epitopes were predicted using PEPFOLD. The binding patterns of selected epitopes with a common conserved allele HLA-B7 were analyzed through molecular docking and it was found that only 27 epitopes bind deep inside in the HLA-B7 binding pocket (Supplementary Figure 2-3). Each bound epitope to HLA-B7 depict stronger than −10.00 Kcal/mol docking affinity. Docking binding energy scores together with their detail information is mentioned in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. All the 27 selected epitopes ensured their binding efficiency as well as their suitability to be used in multiple epitope-based vaccine construct.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1211159/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1211159/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1211159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-32" class="first-child">Final selected epitopes from SARS-CoV-2 antigenic proteins used to design MEV construct.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>3.4. Population coverage</h3><p id="p-33">The distribution and expression of HLA alleles vary by ethnic groups and regions of the world. Therefore, it affects the successful development of an epitope-based vaccine. The IEDB population tool was used to check the population coverage of selected epitopes and their suitability for vaccine construct. Selected epitopes showed the 99.29% world population coverage. The highest coverage of population found within a country Sweden 99.79%. The population coverage for worst COVID-19 hit countries; China, Italy, Spain and Singapore were 84.51%, 96.59%, 99.26% and 85.23%, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Likewise, significant population coverage was identified for other seriously affected countries with SARS-CoV-2 including Australia, USA and France. The result suggested that our MEV could help to combat against COVID-19 in most of the regions round the globe.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Worldwide population coverage of MEV epitopes based on their respective HLA binding alleles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Worldwide population coverage of MEV epitopes based on their respective HLA binding alleles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F2.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F2.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-34" class="first-child">Worldwide population coverage of MEV epitopes based on their respective HLA binding alleles.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h3>3.5. Construction of multiepitope based vaccine</h3><p id="p-35">All 27 selected epitopes (E-2, M-15, ORF6-2, ORF7a-4, ORF8-7, N-5, ORF10-2) were validated for their inter-interactions and further used to develop an MEV construct. An adjuvant (45 amino acid long β-defensin) was linked with the help of EAAAK linker at the start (to the N-terminal of the MEV). EAAAK linker reduce interaction with other protein regions with efficient separation and increase stability [<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]. The vaccine’s immunogenicity may increase with an adjuvant. Epitopes were merge together based on their interaction’s compatibility in sequential manner with AAY and GPGPG linkers respectively. AAY and GPGPG prevents the generation of junctional epitopes, which is a major concern in the design of multiepitope vaccines; On the other hand, they facilitates the immunization and presentation of epitopes [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>, <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. The final vaccine construct comprises 505 amino acid residues (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic diagram of MEV construct: an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic diagram of MEV construct: an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F3.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F3.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-36" class="first-child">Schematic diagram of MEV construct: an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>3.6. Evaluation of multiepitope based vaccine</h3><p id="p-37">First, Blast-p analysis was performed against <em>Homo sapiens</em> proteome with default parameters to validate that MEV is non-homologous. Protein with less than 37% identity generally considered as non-homologous protein. However, MEV showed no similarity (higher or equal to 37%) with the human proteins.</p><p id="p-38">Next, allergenicity, antigenicity and toxicity of the vaccine construct were evaluated. Results described that MEV is highly antigenic (0.6741 at 0.5% threshold), non-allergenic and non-toxic.</p><p id="p-39">Further, the physiochemical properties of the SARS-CoV-2 MEV construct were determined using Protparam. It contains 505 amino acids with 55426.35 KDa molecular weight, indicating good antigenic nature. The isoelectric point (PI) of MEV was 9.12 which show its negative. Negatively charged MEV showed the value of isoelectric point (pI) less than 7. MEV was categorized as stable as instability index computed by the Protparam was 33.41. The aliphatic index was 82.75, which represents the idea of the proportional volume of the aliphatic side chains. The protein sequence has a GRAVY value of 0.105, indicating the hydrophobic nature of the MEV. Total time taken for a protein to disappear after it has been synthesized in cell depicted as the half-life of the protein which was computed as &gt; 20 h for yeast, 30 h for mammalian-reticulocytes and &gt; 10 h for <em>Escherichia coli</em>. MEV computed formula wa C2549H3850N666O669S28, depicting the cumulative numbers of Carbon (C), Oxygen (O), Nitrogen (N), Hydrogen (H) and Sulfur (S). Above results indicated MEV as a suitable potential vaccine.</p></div><div id="sec-29" class="subsection"><h3>3.7. Structural analysis of multiepitope based vaccine</h3><p id="p-40">Secondary structure of MEV was predicted by PSIPRED. Among the 505 amino acids, the formation of α-helix is comprised of 176 amino acids representing 35.20%, 109 amino acids in β-strands representing 21.59% and 215 amino acids forms the coils which are 42.58% of the whole MEV construct (Supplementary Figure 4).</p><p id="p-41">To determine the tertiary structure of vaccine RaptorX server was used. Structure was refined by Galaxy refine server (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The improved model depicted that 95% amino acids are in favorable region, 96.3% of residues in most favorable region, 3.7% of residues in permitted region and 0.0% in outer region according to Ramachandran plot analysis. Further analyses revealed qRMSD is 0.428, poor rotamers are 0%, MolProbity is 1.889, clash score is 13.6, and Z score is −2.25 Z. In addition, the refined model showed 0 errors with PROCHECK validation. The refined model score was 85.7143 in quality check analysis through ERRAT. These results show that the refined model is of good quality.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) A 505 amino acid long MEV sequence consisting an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green). (B) The predicted three-dimensional structure of the MEV vaccine construct (Alpha helixes: Brown; Beta sheets: Blue; Loops: Cyan). (C) Ramachandran plot analysis result of predicted MEV structure where 96.3% of residues present in the most favorable region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(&lt;strong&gt;A&lt;/strong&gt;) A 505 amino acid long MEV sequence consisting an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green). (&lt;strong&gt;B&lt;/strong&gt;) The predicted three-dimensional structure of the MEV vaccine construct (Alpha helixes: Brown; Beta sheets: Blue; Loops: Cyan). (&lt;strong&gt;C&lt;/strong&gt;) Ramachandran plot analysis result of predicted MEV structure where 96.3% of residues present in the most favorable region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F4.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F4.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-42" class="first-child">(<strong>A</strong>) A 505 amino acid long MEV sequence consisting an adjuvant (Maroon) linked at N-terminal with the help of EAAAK linker (Pink). CTL epitopes are joined by AAY linkers (Blue) while HTL epitopes are joined by GPGPG linkers (Green). (<strong>B</strong>) The predicted three-dimensional structure of the MEV vaccine construct (Alpha helixes: Brown; Beta sheets: Blue; Loops: Cyan). (<strong>C</strong>) Ramachandran plot analysis result of predicted MEV structure where 96.3% of residues present in the most favorable region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In addition, flexibility of MEV structure was evaluated using CABS-flex 2.0 server with 50 cycles simulation at 1.4 °C temperature. Among the 10 final retrieved 3D structures, regions near to N-terminal depicted lesser fluctuation compared with the regions near the C-terminal (Supplementary Figure 5A). Resultant contact-map presented the residue-residue interaction pattern for all the 10 final retrieved models (Supplementary Figure 5B). Finally, root mean square fluctuation (RMSF) plot revealed the fluctuation of each of the amino acid of MEV model ranging from 0.0 Å to 3.5 Å. The presence of fluctuations in the MEV structure, indicated its high flexibility and endorse it as a potential vaccine construct (Supplementary Figure 5B).</p></div><div id="sec-30" class="subsection"><h3>3.8. Prediction of B-cell epitopes in multiepitope based vaccine</h3><p id="p-44">B-lymphocytes besides secreting cytokines, also produce antigens, which in return provide humoral immunity [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. Therefore, MEV ideally should have B-cell epitopes with its domains. Three conformational/discontinuous and 92 linear/continuous B-cell epitopes from the MEV were predicted without altering the prediction parameters of Ellipro and ABCPred 2.0 (Supplementary Tables 9-10).</p></div><div id="sec-31" class="subsection"><h3>3.9. Immunogenicity evaluation of multiepitope based vaccine</h3><p id="p-45">The <em>in silico</em> immune system simulation against MEV showed significant activity of B-cells and T-cells, which was consistent with the actual immune responses. Primary immune response was observed with the increased levels of IgM in the start. Later, secondary and tertiary immune responses were observed comparitively higher than the primary response. The immunoglobulin activity (i.e., IgG1 + IgG2, IgM, and IgG + IgM antibodies) was normally at high levels with a consistent decrease in MEV concentration. The high levels of triggered B-cells and memory B-cell formation was observed, that is indicator of an effective long-lasting immune response produced by the MEV (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A-D</a>). Similarly, higher levels of HTL and CTL as well as development of memory Th and Tc were observed which is vital for triggering the immune response (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5E-H</a>). The significant levels of T regulatory cells and the continuous and increased proliferation of macrophages and dendritic cells were observed during introduction of MEV. The higher levels of cytokines like IL-2, and IFN-γ were also observed (Figure I-L). These observations indicated the generation of promising antiviral response by the proposed MEV construct.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In silico simulation of immune response against MEV as an antigen: (A) Antigen and immunoglobulins, (B) B-cell population, (C) B-cell population per state, (D) Plasma B-cell population, (E) Helper T-cell population, (F) Helper T-cell population per state, (G) Cytotoxic T-cell population, (H) Cytotoxic T-cell population per state, (I) Reduced levels of T regulatory cells, (J) Dendritic cell population per state, (K) Macrophage population per state, and (L) Different levels of cytokine and interleukins with Simpson index (D) of immune response." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In silico&lt;/em&gt; simulation of immune response against MEV as an antigen: (&lt;strong&gt;A&lt;/strong&gt;) Antigen and immunoglobulins, (&lt;strong&gt;B&lt;/strong&gt;) B-cell population, (&lt;strong&gt;C&lt;/strong&gt;) B-cell population per state, (&lt;strong&gt;D&lt;/strong&gt;) Plasma B-cell population, (&lt;strong&gt;E&lt;/strong&gt;) Helper T-cell population, (&lt;strong&gt;F&lt;/strong&gt;) Helper T-cell population per state, (&lt;strong&gt;G&lt;/strong&gt;) Cytotoxic T-cell population, (&lt;strong&gt;H&lt;/strong&gt;) Cytotoxic T-cell population per state, (&lt;strong&gt;I&lt;/strong&gt;) Reduced levels of T regulatory cells, (&lt;strong&gt;J&lt;/strong&gt;) Dendritic cell population per state, (&lt;strong&gt;K&lt;/strong&gt;) Macrophage population per state, and (&lt;strong&gt;L&lt;/strong&gt;) Different levels of cytokine and interleukins with Simpson index (D) of immune response.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F5.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F5.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-46" class="first-child"><em>In silico</em> simulation of immune response against MEV as an antigen: (<strong>A</strong>) Antigen and immunoglobulins, (<strong>B</strong>) B-cell population, (<strong>C</strong>) B-cell population per state, (<strong>D</strong>) Plasma B-cell population, (<strong>E</strong>) Helper T-cell population, (<strong>F</strong>) Helper T-cell population per state, (<strong>G</strong>) Cytotoxic T-cell population, (<strong>H</strong>) Cytotoxic T-cell population per state, (<strong>I</strong>) Reduced levels of T regulatory cells, (<strong>J</strong>) Dendritic cell population per state, (<strong>K</strong>) Macrophage population per state, and (<strong>L</strong>) Different levels of cytokine and interleukins with Simpson index (D) of immune response.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-32" class="subsection"><h3>3.10. Molecular docking of multiepitope based vaccine with TLR3 and TLR8</h3><p id="p-47">An appropriate association between immune receptor molecules and the antigen molecule is essential to activate an immune responsiveness. HADDOCK server has thus been used to perform the docking of the MEV with human immune receptors TLR3 and TLR8. TLR3 and TLR8 can efficiently induce the immune response after virus recognition. The docking analysis showed good interactions between the MEV and TLR3/TLR8. The binding scores of MEV-TLR3 and MEV-TLR8 were −293.90 Kcal/mol and −283.20 Kcal/mol, respectively. TLR3 is shown in the orange color, while the MEV is shown in the blue color, respectively, in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>. It was observed that MEV made 11 hydrogen bond interactions within range of 3.00 Å with TLR3 <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B-C</a>. MEV interacting amino acids with hydrogen bonding to TLR3 shown in green color stick representation, while similarly TLR3 amino acids interacting through hydrogen bonding with MEV shown in red color stick representation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MEV construct docking with human TLR3: (A) MEV-TLR3 docked complex in cartoon representation. TLR3 displayed with brown color and MEV vaccine construct displayed with blue color. (B) Interacting residues illustration between MEV and TLR3 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR3 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (C) All interacting residues of MEV and TLR3. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MEV construct docking with human TLR3: (&lt;strong&gt;A&lt;/strong&gt;) MEV-TLR3 docked complex in cartoon representation. TLR3 displayed with brown color and MEV vaccine construct displayed with blue color. (&lt;strong&gt;B&lt;/strong&gt;) Interacting residues illustration between MEV and TLR3 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR3 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (&lt;strong&gt;C&lt;/strong&gt;) All interacting residues of MEV and TLR3. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F6.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F6.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-48" class="first-child">MEV construct docking with human TLR3: (<strong>A</strong>) MEV-TLR3 docked complex in cartoon representation. TLR3 displayed with brown color and MEV vaccine construct displayed with blue color. (<strong>B</strong>) Interacting residues illustration between MEV and TLR3 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR3 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (<strong>C</strong>) All interacting residues of MEV and TLR3. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">In case of TLR8, it is shown in the grey color, while the MEV is shown in the blue color, respectively, in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>. It was observed that MEV made 9 hydrogen bond interactions within range of 3.00 Å with TLR8 <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B-C</a>. Similar to TLR3, MEV interacting amino acids with hydrogen bonding to TLR8 shown in green color stick representation, while TLR8 amino acids interacting through hydrogen bonding with MEV shown in red color stick representation. These results indicated that this MEV is best suitable candidate for vaccine production.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MEV construct docking with human TLR8: (A) MEV-TLR8 docked complex in cartoon representation. TLR8 displayed with grey color and MEV vaccine construct displayed with blue color. (B) Interacting residues illustration between MEV and TLR8 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR8 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (C) All interacting residues of MEV and TLR8. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MEV construct docking with human TLR8: (&lt;strong&gt;A&lt;/strong&gt;) MEV-TLR8 docked complex in cartoon representation. TLR8 displayed with grey color and MEV vaccine construct displayed with blue color. (&lt;strong&gt;B&lt;/strong&gt;) Interacting residues illustration between MEV and TLR8 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR8 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (&lt;strong&gt;C&lt;/strong&gt;) All interacting residues of MEV and TLR8. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F7.medium.gif" width="440" height="378"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F7.medium.gif" width="440" height="378"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <p id="p-50" class="first-child">MEV construct docking with human TLR8: (<strong>A</strong>) MEV-TLR8 docked complex in cartoon representation. TLR8 displayed with grey color and MEV vaccine construct displayed with blue color. (<strong>B</strong>) Interacting residues illustration between MEV and TLR8 complex. Interacting residues of MEV are highlighted with green color stick representation, while interacting residues of TLR8 are highlighted with red color stick representation. Hydrogen bonds are represented with yellow color dotted lines. (<strong>C</strong>) All interacting residues of MEV and TLR8. Hydrogen bonds are shown with blue color lines. The colors of interacting residues are representing properties of amino acids (Positive: Blue, Negative: Red, Neutral: Green, Aliphatic: Grey, Aromatic: Pink, Pro&amp;Gly: Orange and Cys: Yellow).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-33" class="subsection"><h3>3.11. MD simulation of TLR3-MEV and TLR8-MEV complexes</h3><p id="p-51">MD simulation is a common approach used to analyse the micro-interactions between ligand/vaccine and protein/receptor structures [<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>]. In order to further assess MEV dynamics and stability, its docking complexes with TLR3 and TLR8 were simulated by 20 ns MD followed by Root Mean Square Deviations (RMSD) and Root Mean Square Fluctuations (RMSF) analysis. The values of RMSD of backbone atoms were computed to monitor the structural stability of MEV and immune receptor complexes (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). The RMSD average values for both complexes TLR3-MEV and TLR8-MEV are 4.5 ± 0.02 Å and 5.6 ± 0.02 Å, respectively. There were no significant variations noticed in both docked complexes. Both the systems remain stable throughout 20 ns simulation. To further compute, the residual and side-chain flexibility, RMSF over 20 ns time were computed. Little fluctuations at the terminal constituent residues of both complexes were observed, while middle residues depicted stable behavior with an average 1.5 ± 0.02 Å RMSF (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). In general, both complexes were stable without obvious fluctuations. These results validate the docking interaction analysis and endorsed that MEV can strongly bind with immune receptors to generate significant immune response against SARS-CoV-2.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MD simulation at 20 ns results: (A) The RMSD plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSD represented with purple color line while MEV-TLR8 complex RMSD is represented with orange color line. (B) The RMSF plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSF represented with purple color line while MEV-TLR8 complex RMSF is represented with orange color line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MD simulation at 20 ns results: (A) The RMSD plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSD represented with purple color line while MEV-TLR8 complex RMSD is represented with orange color line. (B) The RMSF plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSF represented with purple color line while MEV-TLR8 complex RMSF is represented with orange color line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F8.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F8.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <p id="p-52" class="first-child">MD simulation at 20 ns results: (A) The RMSD plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSD represented with purple color line while MEV-TLR8 complex RMSD is represented with orange color line. (B) The RMSF plot of the MEV-TLR3 and MEV-TLR8 complexes. MEV-TLR3 complex RMSF represented with purple color line while MEV-TLR8 complex RMSF is represented with orange color line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-34" class="subsection"><h3>3.12. In silico cloning</h3><p id="p-53"><em>In silico</em> cloning was performed to ensure expression of SARS-CoV-2 derived MEV in commonly used <em>E. coli</em> host. First, codons of MEV construct were adapted as per codon usage of <em>E. coli</em> expression system. JCAT server was used to optimize the MEV codons according to <em>E. coli</em> (strain K12). The optimized MEV construct contained 1515 nucleotides (Supplementary Table 11), an ideal range of GC content 56.30% (30-70%) and CAI value 0.93 (0.8-1.0), and showing the high possibility of positive protein expression and reliability. In next step, buffer compatible restriction enzymes XhoI and HindIII restriction sites were added to the both ends of MEV optimized nucleotide sequence to aid the cloning/purification process. Finally, refined MEV sequence was cloned between XhoI and HindIII restriction sites at the multiple cloning site of pET28a (+) vector (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). The total length of the clone was 6.875 kbp.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MEV codon optimized nucleotide sequence for E. coli expression system, in silico cloned at multiple cloning site (MCS) of pET28a(+) vector, shown with blue color in vector backbone (black color) and with grey color functional annotation. MEV cloned sequence is surrounded by XhoI (158) and HindIII (1679) restriction enzyme site, and 6-histidine residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2040113929" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MEV codon optimized nucleotide sequence for &lt;em&gt;E. coli&lt;/em&gt; expression system, &lt;em&gt;in silico&lt;/em&gt; cloned at multiple cloning site (MCS) of pET28a(+) vector, shown with blue color in vector backbone (black color) and with grey color functional annotation. MEV cloned sequence is surrounded by XhoI (158) and HindIII (1679) restriction enzyme site, and 6-histidine residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F9.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F9.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/22/2020.02.28.970343/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9.</span> <p id="p-54" class="first-child">MEV codon optimized nucleotide sequence for <em>E. coli</em> expression system, <em>in silico</em> cloned at multiple cloning site (MCS) of pET28a(+) vector, shown with blue color in vector backbone (black color) and with grey color functional annotation. MEV cloned sequence is surrounded by XhoI (158) and HindIII (1679) restriction enzyme site, and 6-histidine residues.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-35"><h2 class="">4. Discussion</h2><p id="p-55">Vaccination has many useful effects for improving people health at a low-cost and best aid to inhibit transmission of diseases around the world. However, the development and production of the vaccine is labour-intensive and costly. Immunoinformatics approaches can reduce this burden. Today researchers are searching methods for the development of subunit vaccines from complete genome/proteome of pathogens [<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>]. Epitope prediction for antibodies becomes more significant with the advancement of the computational tools for designing a vaccine [<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. In the field of bioinformatics, Immuno-informatics is a sub-branch that includes a lot of tools &amp; databases. Immunological datasets prediction and <em>in silico</em> analysis are done with the help of those tools. With the advancement of tools and a variety of data availability like genomic, proteomic, and different algorithms made it more effective for scientists to accurately predict epitopes that are much effective in the development of the sub-unit vaccines [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>].</p><p id="p-56">An outbreak of SARS-COV-2 in late December 2019 that later turned into a global pandemic resulted in thousands of deaths around the globe [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>]. After the outbreak, remarkable progress has been made towards the structural genomics, proteomics and drug repurposing against the SARS-CoV-2, yet there is no exact cure has been identified for COVID-19. Till date, no effective vaccine is available against SARS-CoV-2. Therefore, COVID-19 vaccine need to be designed so that global pandemic situation could be controlled effectively.</p><p id="p-57">The recent study was conducted to design a multiepitope based vaccine (MEV) against SARS-COV-2 by using immunoinformatics and <em>in silico</em> approaches. MEV is advantageous compared to monovalent vaccine because it can elicit humoral, innate and cellular immunity responses together [<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>]. Three different studies have been reported yet, which reported lists of potential epitopes using Surface glycoprotein, Envelope protein, and Membrane glycoprotein [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>]. However, no other sub-unit vaccine construct has been reported against SARS-CoV-2 to date. The current MEV construct is therefore is very important to curb COVID-19. Amino acid sequences of ten proteins of SARS-COV-2 were taken from Genbank and their antigenicity was checked. Highly antigenic proteins were selected for further analysis. After the complete physiochemical analysis of antigenic proteins, B-cell, T-cell and IFN-γ inducing epitopes were predicted. Antigenic and overlapped T-cell epitopes with B-cells and IFN-γ epitopes were further validated using molecular docking with HLA-B7 allele and their population coverage was estimated. Finally, MEV was designed with the help of linkers and adjuvant. An adjuvant was added to the N-terminal of the MEV and epitopes were linked with the help of AAY and GPGPG linkers. Adjuvant was added to increase the immunogenicity of the vaccine [<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>]. Linkers were added to help maintain the function of each epitope so that after being imported into the human body they can function independently/properly [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>].</p><p id="p-58">MEV was found to be highly antigenic, immunogenic, non-toxic and non-allergenic indicating an epitope-based vaccine’s potential to cause robust immune responses. During <em>in silico</em> immunological validation analysis, higher B and T-cell activity was observed that is rationally identical with typical immune responses. B and T-cell memory formation was evident, and results depicted it last for several months. Molecular docking analysis followed by MD simulation revealed that the MEV is potentially able to properly/firmly occupy the TLR3/TLR8 receptors with minimal energy. Furthermore, the expression of MEV construct inside the host <em>E. coli</em> K12 was improved through codon optimization. The results of recent study suggest that the MEV being designed is a potential candidate to undergo <em>in vivo</em> and <em>in vitro</em> experimental analysis to develop a potential vaccine against COVID-19.</p></div><div class="section" id="sec-36"><h2 class="">5. Conclusion</h2><p id="p-59">Recent global pandemic of SARS-CoV-2 claimed hundreds of precious lives in various regions of the world and crumble the economies of several countries. There is no drug or vaccine reported against SARS-CoV-2 yet. Few antiviral drugs and vaccines are in clinical trials, but none have yet been declared as clinically approved anti-COVID-19 therapeutic. In this study, a successful attempt was made to design a sub-unit MEV against SARS-COV-2. Immunoinformatics and <em>in silico</em> approaches were used to develop a potential and safe MEV that could trigger three types of the immune responses: humoral, innate and cellular. A highly immunogenic, safe, stable and strongly interacting with human receptors, MEV has been reported in present study that could be a potential candidate for vaccine production. However, current research is the result of an integrated vaccinomics approach. Therefore, further experimental research by the vaccinologists is required to demonstrate the efficacy of the designed vaccine.</p></div><div class="section" id="sec-38"><h2 class="">Authors’ contributions</h2><p id="p-61">MTQ, LLC and UAA conceived and designed this study; MTQ and AR performed the experiments; MQA, IF and FS analyse the results; MTQ and AR wrote the manuscript; UAA and LLC improved and revised the manuscript, and all the authors approved the final version.</p></div><div class="section" id="sec-39"><h2 class="">Conflicts of interest</h2><p id="p-62">The authors have no conflicts of interest to declare.</p></div><div class="section" id="sec-40"><h2 class="">Supplementary data</h2><p id="p-63"><strong>Supplementary Figure 1</strong>. Cartoon 3D structural representation of SARS-CoV-2 antigenic proteins. (<strong>A</strong>) E protein tertiary structure, (<strong>B</strong>) M protein tertiary structure, (<strong>C</strong>) N protein tertiary structure, (<strong>D</strong>) ORF6 protein tertiary structure, (<strong>E</strong>) ORF7a protein tertiary structure, (<strong>F</strong>) ORF8 protein tertiary structure and (<strong>G</strong>) ORF10 protein tertiary structure.</p><p id="p-64"><strong>Supplementary Figure 2</strong>. The 3D binding pattern of the selected 16 MHC Class I epitopes (meshed-pink) docked with HLA-B7 allele (cyan). Hydrogen bond interactions are highlighted with yellow color dotted lines. Numbering 1-16 is consistent with the <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> of main text.</p><p id="p-65"><strong>Supplementary Figure 3</strong>. The 3D binding pattern of the selected 11 MHC Class II epitopes (purple blue) docked with HLA-B7 allele (cyan). Hydrogen bond interactions are highlighted with yellow color dotted lines. Numbering 1-11 is consistent with the <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> of main text.</p><p id="p-66"><strong>Supplementary Figure 4</strong>. Secondary structure depiction of MEV construct.</p><p id="p-67"><strong>Supplementary Figure 5</strong>. MEV structural flexibility results. (<strong>A</strong>) Cartoon representation of top 10 final models showing obvious fluctuation throughout. (<strong>B</strong>) MEV residue-residue interaction/contact map. The interactive area is represented in the central panel. (<strong>C</strong>) RMSF plot representing the obvious fluctuations of MEV residues during simulation.</p><p id="p-68"><strong>Supplementary Table 1.</strong> Physiochemical properties of the SARS-COV-2 proteins.</p><p id="p-69"><strong>Supplementary Table 2.</strong> Secondary structure of the SARS-COV-2 proteins.</p><p id="p-70"><strong>Supplementary Table 3.</strong> 3D structural details of selected SARS-COV-2 proteins.</p><p id="p-71"><strong>Supplementary Table 4.</strong> Complete list of predicted potential linear B-cell epitopes of SARS-COV-2 proteins.</p><p id="p-72"><strong>Supplementary Table 5.</strong> List of predicted potential conformational B-cell epitopes of SARS-COV-2 proteins.</p><p id="p-73"><strong>Supplementary Table 6.</strong> List of predicted potential MHC Class I epitopes of SARS-COV-2 proteins.</p><p id="p-74"><strong>Supplementary Table 7.</strong> List of predicted potential MHC Class II epitopes of SARS-COV-2 proteins.</p><p id="p-75"><strong>Supplementary Table 8.</strong> List of predicted potential IFN-γ inducing epitopes of SARS-COV-2 proteins.</p><p id="p-76"><strong>Supplementary Table 9.</strong> Linear B-cell epitopes in the final MEV construct.</p><p id="p-77"><strong>Supplementary Table 10.</strong> Conformational B-cell epitopes in the final MEV construct.</p><p id="p-78"><strong>Supplementary Table 11.</strong> Codon optimized nucleotide sequence of MEV construct for cloning in <em>E. coli</em> strain K12. Green color bold sequence at 5’ site (N-terminal) is representing XhoI restriction enzyme site, while maroon color bold sequence at 3’ site (C-terminal) is representing HindIII restriction enzyme site.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-60">This work was supported by the Starting Research Grant for High-level Talents from Guangxi University and Postdoctoral research platform grant of Guangxi University.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.28.970343v2.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.A.</span> <span class="cit-name-surname">Tripp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.M.</span> <span class="cit-name-surname">Tompkins</span></span>, <span class="cit-source">Roles of Host Gene and Non-coding RNA Expression in Virus Infection</span>, <span class="cit-publ-name">Springer</span> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hao</span></span>, <span class="cit-article-title">Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission</span>, <abbr class="cit-jnl-abbrev">Sci China Life Sci</abbr> <span class="cit-vol">63</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">457</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BChina%2BLife%2BSci%26rft.volume%253D63%26rft.spage%253D457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Niu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.F.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-article-title">I. China Novel Coronavirus, T. Research, A Novel Coronavirus from Patients with Pneumonia in China</span>, 2019, <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">382</span>(<span class="cit-issue">8</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.4" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Cheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Yin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Jin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cao</span></span>, <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>, <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>(<span class="cit-issue">10223</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. Tahir ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.M.</span> <span class="cit-name-surname">Alqahtani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.A.</span> <span class="cit-name-surname">Alamri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-L.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-article-title">Structural Basis of SARS-CoV-2 3CLpro and Anti-COVID-19 Drug Discovery from Medicinal Plants</span>, <abbr class="cit-jnl-abbrev">Preprints</abbr> <span class="cit-vol">2020020193</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.6" data-doi="10.1016/j.ijid.2020.01.009"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.S.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. I</span> <span class="cit-name-surname">Azhar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.A.</span> <span class="cit-name-surname">Madani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ntoumi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Dar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ippolito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.D.</span> <span class="cit-name-surname">Mchugh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.A.</span> <span class="cit-name-surname">Memish</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health—The latest 2019 novel coronavirus outbreak in Wuhan, China</span>, <abbr class="cit-jnl-abbrev">International Journal of Infectious Diseases</abbr> <span class="cit-vol">91</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">264</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BInfectious%2BDiseases%26rft.volume%253D91%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijid.2020.01.009%26rft_id%253Dinfo%253Apmid%252F31953166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijid.2020.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31953166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.A.</span> <span class="cit-name-surname">Alamri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. Tahir ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.M.</span> <span class="cit-name-surname">Alqahtani</span></span>, <span class="cit-article-title">Pharmacoinformatics and Molecular Dynamic Simulation Studies Reveal Potential Inhibitors of SARS-CoV-2 Main Protease 3CLpro</span>, <abbr class="cit-jnl-abbrev">Preprints 2020020308</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ahmed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Noman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Almatroudi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shahid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Khurshid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tariq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.T. ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">Coronavirus Disease 2019 Assosiated Pneumonia in China: Current Status and Future Prospects</span>, <abbr class="cit-jnl-abbrev">Preprints 2020020358</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-article-title">Drug treatment options for the 2019-new coronavirus (2019-nCoV)</span>, <abbr class="cit-jnl-abbrev">Biosci Trends</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">69</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiosci%2BTrends%26rft.volume%253D14%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.I.</span> <span class="cit-name-surname">Abdelmageed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.H.</span> <span class="cit-name-surname">Abdelmoneim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.I.</span> <span class="cit-name-surname">Mustafa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.M.</span> <span class="cit-name-surname">Elfadol</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.S.</span> <span class="cit-name-surname">Murshed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.W.</span> <span class="cit-name-surname">Shantier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.M.</span> <span class="cit-name-surname">Makhawi</span></span>, <span class="cit-article-title">Design of multi epitope-based peptide vaccine against E protein of human 2019-nCoV: An immunoinformatics approach</span>, <abbr class="cit-jnl-abbrev">BioRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.11" data-doi="10.1038/s41422-020-0282-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>, <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">269</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D30%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41422-020-0282-0%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41422-020-0282-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">De Gregorio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rappuoli</span></span>, <span class="cit-article-title">Vaccines for the future: learning from human immunology</span>, <abbr class="cit-jnl-abbrev">Microb Biotechnol</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>) (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">149</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMicrob%2BBiotechnol%26rft.aulast%253DDe%2BGregorio%26rft.auinit1%253DE.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D155%26rft.atitle%253DVaccines%2Bfor%2Bthe%2Bfuture%253A%2Blearning%2Bfrom%2Bhuman%2Bimmunology.%26rft_id%253Dinfo%253Apmid%252F21880117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21880117&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.13" data-doi="10.1098/rsob.120139"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Patronov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Doytchinova</span></span>, <span class="cit-article-title">T-cell epitope vaccine design by immunoinformatics</span>, <abbr class="cit-jnl-abbrev">Open biology</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">120139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsob.120139%26rft_id%253Dinfo%253Apmid%252F23303307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsob.120139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23303307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[14].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.28.970343v2.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gupta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Regar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Verma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Prusty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mishra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Prajapati</span></span>, <span class="cit-article-title">Receptor-ligand based molecular interaction to discover adjuvant for immune cell TLRs to develop next-generation vaccine</span>, <abbr class="cit-jnl-abbrev">Int J Biol Macromol</abbr> <span class="cit-vol">152</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">535</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BBiol%2BMacromol%26rft.volume%253D152%26rft.spage%253D535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kalita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.L.</span> <span class="cit-name-surname">Lyngdoh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.K.</span> <span class="cit-name-surname">Padhi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shukla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tripathi</span></span>, <span class="cit-article-title">Development of multi-epitope driven subunit vaccine against Fasciola gigantica using immunoinformatics approach</span>, <abbr class="cit-jnl-abbrev">International journal of biological macromolecules</abbr> <span class="cit-vol">138</span> (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">224</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bbiological%2Bmacromolecules%26rft.volume%253D138%26rft.spage%253D224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.16"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Saadi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Karkhah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.R.</span> <span class="cit-name-surname">Nouri</span></span>, <span class="cit-article-title">Development of a multi-epitope peptide vaccine inducing robust T cell responses against brucellosis using immunoinformatics based approaches, Infection</span>, <abbr class="cit-jnl-abbrev">Genetics and Evolution</abbr> <span class="cit-vol">51</span> (<span class="cit-pub-date">2017</span>) <span class="cit-fpage">227</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2Band%2BEvolution%26rft.volume%253D51%26rft.spage%253D227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Nain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Karim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.K.</span> <span class="cit-name-surname">Sen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.K.</span> <span class="cit-name-surname">Adhikari</span></span>, <span class="cit-article-title">Structural Basis and Designing of Peptide Vaccine using PE-PGRS Family Protein of Mycobacterium ulcerans–An Integrated Vaccinomics Approach</span>, <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2019</span>) <span class="cit-vol">795146</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. Tahir Ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Saleem</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Anwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Alqahtani</span></span>, <span class="cit-article-title">Epitope-based peptide vaccine design and target site depiction against Middle East Respiratory Syndrome Coronavirus: an immune-informatics study</span>, <abbr class="cit-jnl-abbrev">J Transl Med</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTransl%2BMed%26rft.volume%253D17%26rft.spage%253D362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. Tahir Ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Adeel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Maryam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Muneer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.I.</span> <span class="cit-name-surname">Ahmad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-article-title">Peptide vaccine against chikungunya virus: immuno-informatics combined with molecular docking approach</span>, <abbr class="cit-jnl-abbrev">J Transl Med</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTransl%2BMed%26rft.volume%253D16%26rft.spage%253D298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ahmad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.U.</span> <span class="cit-name-surname">Rahman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.S.</span> <span class="cit-name-surname">Masoud</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.Z.</span> <span class="cit-name-surname">Yousaf</span></span>, <span class="cit-article-title">Conserved B and T cell epitopes prediction of ebola virus glycoprotein for vaccine development: An immuno-informatics approach</span>, <abbr class="cit-jnl-abbrev">Microb Pathog</abbr> <span class="cit-vol">132</span> (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob%2BPathog%26rft.volume%253D132%26rft.spage%253D243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Shahid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Javaid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Khalid</span></span>, <span class="cit-article-title">Immunoinformatics guided rational design of a next generation multi epitope based peptide (MEBP) vaccine by exploring Zika virus proteome</span>, <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr> <span class="cit-vol">80</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">104199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2BEvol%26rft.volume%253D80%26rft.spage%253D104199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.22" data-doi="10.1038/s41598-018-34254-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ikram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zaheer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.M.</span> <span class="cit-name-surname">Awan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Obaid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Naz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hanif</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.Z.</span> <span class="cit-name-surname">Paracha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.K.</span> <span class="cit-name-surname">Naveed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.A.</span> <span class="cit-name-surname">Janjua</span></span>, <span class="cit-article-title">Exploring NS3/4A, NS5A and NS5B proteins to design conserved subunit multi-epitope vaccine against HCV utilizing immunoinformatics approaches</span>, <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">16107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D16107%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-34254-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-34254-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.K.</span> <span class="cit-name-surname">Pandey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dahiya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mahita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sowdhamini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Prajapati</span></span>, <span class="cit-article-title">Vaccination and immunization strategies to design Aedes aegypti salivary protein based subunit vaccine tackling Flavivirus infection</span>, <abbr class="cit-jnl-abbrev">International journal of biological macromolecules</abbr> <span class="cit-vol">122</span> (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">1203</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bbiological%2Bmacromolecules%26rft.volume%253D122%26rft.spage%253D1203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Chauhan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.P.</span> <span class="cit-name-surname">Singh</span></span>, <span class="cit-article-title">Immuno-informatics approach to design a multi-epitope vaccine to combat cytomegalovirus infection</span>, <abbr class="cit-jnl-abbrev">Eur J Pharm Sci</abbr> <span class="cit-vol">147</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">105279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur%2BJ%2BPharm%2BSci%26rft.volume%253D147%26rft.spage%253D105279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.25" data-doi="10.1038/srep37313"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.U.</span> <span class="cit-name-surname">Mirza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rafique</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Munir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ikram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Manan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.M.</span> <span class="cit-name-surname">Salo-Ahen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Idrees</span></span>, <span class="cit-article-title">Towards peptide vaccines against Zika virus: Immunoinformatics combined with molecular dynamics simulations to predict antigenic epitopes of Zika viral proteins</span>, <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span> (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">37313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D37313%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep37313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep37313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[26].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.28.970343v2.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Junaid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.C.</span> <span class="cit-name-surname">Kaushik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.S.</span> <span class="cit-name-surname">Ali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mehmood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.-Q.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-article-title">Computational identification, characterization and validation of potential antigenic peptide vaccines from hrHPVs E6 proteins using immunoinformatics and computational systems biology approaches</span>, <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">5</span>) (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Katalani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Nematzadeh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ahmadian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Amani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kiani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ehsani</span></span>, <span class="cit-article-title">In silico design and in vitro analysis of a recombinant trivalent fusion protein candidate vaccine targeting virulence factor of Clostridium perfringens</span>, <abbr class="cit-jnl-abbrev">Int J Biol Macromol</abbr> <span class="cit-vol">146</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">1015</span>–<span class="cit-lpage">1023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BBiol%2BMacromol%26rft.volume%253D146%26rft.spage%253D1015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.28" data-doi="10.1016/S0264-410X(02)00164-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.L.</span> <span class="cit-name-surname">Hunter</span></span>, <span class="cit-article-title">Overview of vaccine adjuvants: present and future</span>, <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">20</span> <span class="cit-issue">Suppl 3</span> (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">S7</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D20%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0264-410X%252802%252900164-0%26rft_id%253Dinfo%253Apmid%252F12184369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0264-410X(02)00164-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12184369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.29" data-doi="10.1038/nrmicro1681"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Guy</span></span>, <span class="cit-article-title">The perfect mix: recent progress in adjuvant research</span>, <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">7</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">505</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DGuy%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D505%26rft.epage%253D517%26rft.atitle%253DThe%2Bperfect%2Bmix%253A%2Brecent%2Bprogress%2Bin%2Badjuvant%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1681%26rft_id%253Dinfo%253Apmid%252F17558426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17558426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247344600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.A.</span> <span class="cit-name-surname">Benson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Karsch-Mizrachi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.J.</span> <span class="cit-name-surname">Lipman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ostell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.W.</span> <span class="cit-name-surname">Sayers</span></span>, <span class="cit-article-title">GenBank</span>, <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">suppl_1</span>) (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">D26</span>–<span class="cit-lpage">D31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D37%26rft.spage%253DD26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.28.970343v2.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.M.</span> <span class="cit-name-surname">Walker</span></span>, <span class="cit-source">The proteomics protocols handbook</span>, <span class="cit-publ-name">Springer</span> <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.32" data-doi="10.1186/1471-2105-8-4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.A.</span> <span class="cit-name-surname">Doytchinova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.R.</span> <span class="cit-name-surname">Flower</span></span>, <span class="cit-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines</span>, <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDoytchinova%26rft.auinit1%253DI.%2BA.%26rft.volume%253D8%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DVaxiJen%253A%2Ba%2Bserver%2Bfor%2Bprediction%2Bof%2Bprotective%2Bantigens%252C%2Btumour%2Bantigens%2Band%2Bsubunit%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-4%26rft_id%253Dinfo%253Apmid%252F17207271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17207271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.33"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Deléage</span></span>, <span class="cit-article-title">ALIGNSEC: viewing protein secondary structure predictions within large multiple sequence alignments</span>, <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.34" data-doi="10.1038/nprot.2012.085"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kallberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-article-title">Template-based protein structure modeling using the RaptorX web server</span>, <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">8</span>) (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">1511</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKallberg%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D1511%26rft.epage%253D1522%26rft.atitle%253DTemplate-based%2Bprotein%2Bstructure%2Bmodeling%2Busing%2Bthe%2BRaptorX%2Bweb%2Bserver.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.085%26rft_id%253Dinfo%253Apmid%252F22814390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22814390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.35" data-doi="10.1038/s41598-018-19456-1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.K.</span> <span class="cit-name-surname">Pandey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.K.</span> <span class="cit-name-surname">Bhatt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Prajapati</span></span>, <span class="cit-article-title">Novel Immunoinformatics Approaches to Design Multi-epitope Subunit Vaccine for Malaria by Investigating Anopheles Salivary Protein</span>, <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D1125%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-19456-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-19456-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.36" data-doi="10.1093/nar/gky427"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Waterhouse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bertoni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bienert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Studer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tauriello</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gumienny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.T.</span> <span class="cit-name-surname">Heer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.A.P.</span> <span class="cit-name-surname">de Beer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rempfer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bordoli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lepore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schwede</span></span>, <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">W1</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">W296</span>–<span class="cit-lpage">W303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.37" data-doi="10.1038/nri3801"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.D.</span> <span class="cit-name-surname">Cooper</span></span>, <span class="cit-article-title">The early history of B cells</span>, <abbr class="cit-jnl-abbrev">Nat Rev Immunol</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2015</span>) <span class="cit-fpage">191</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BImmunol%26rft.volume%253D15%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3801%26rft_id%253Dinfo%253Apmid%252F25656707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri3801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25656707&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-article-title">In Silico Analysis of Epitope-Based Vaccine Candidates against Hepatitis B Virus Polymerase Protein</span>, <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">5</span>) (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.39" data-doi="10.1016/0014-5793(90)80535-Q"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.S.</span> <span class="cit-name-surname">Kolaskar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.C.</span> <span class="cit-name-surname">Tongaonkar</span></span>, <span class="cit-article-title">A semi-empirical method for prediction of antigenic determinants on protein antigens</span>, <abbr class="cit-jnl-abbrev">FEBS Lett</abbr> <span class="cit-vol">276</span>(<span class="cit-issue">1-2</span>) (<span class="cit-pub-date">1990</span>) <span class="cit-fpage">172</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKolaskar%26rft.auinit1%253DA.%2BS.%26rft.volume%253D276%26rft.issue%253D1-2%26rft.spage%253D172%26rft.epage%253D174%26rft.atitle%253DA%2Bsemi-empirical%2Bmethod%2Bfor%2Bprediction%2Bof%2Bantigenic%2Bdeterminants%2Bon%2Bprotein%2Bantigens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252890%252980535-Q%26rft_id%253Dinfo%253Apmid%252F1702393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(90)80535-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1702393&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EP34200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.40" data-doi="10.1186/1471-2105-9-514"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ponomarenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-H.</span> <span class="cit-name-surname">Bui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Fusseder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.E.</span> <span class="cit-name-surname">Bourne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-article-title">ElliPro: a new structure-based tool for the prediction of antibody epitopes</span>, <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DPonomarenko%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D9%26rft.spage%253D514%26rft.epage%253D514%26rft.atitle%253DElliPro%253A%2Ba%2Bnew%2Bstructure-based%2Btool%2Bfor%2Bthe%2Bprediction%2Bof%2Bantibody%2Bepitopes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-514%26rft_id%253Dinfo%253Apmid%252F19055730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19055730&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.41"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Nain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Abdulla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Rahman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Karim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.S.A.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.B.</span> <span class="cit-name-surname">Sayed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mahmud</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.R.</span> <span class="cit-name-surname">Rahman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Sheam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Haque</span></span>, <span class="cit-article-title">Proteome-wide screening for designing a multi-epitope vaccine against emerging pathogen Elizabethkingia anophelis using immunoinformatic approaches</span>, <abbr class="cit-jnl-abbrev">Journal of Biomolecular Structure and Dynamics</abbr> (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.42" data-doi="10.1093/nar/gku938"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.A.</span> <span class="cit-name-surname">Overton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.A.</span> <span class="cit-name-surname">Greenbaum</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ponomarenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.D.</span> <span class="cit-name-surname">Clark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.R.</span> <span class="cit-name-surname">Cantrell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.K.</span> <span class="cit-name-surname">Wheeler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.L.</span> <span class="cit-name-surname">Gabbard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hix</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-article-title">The immune epitope database (IEDB) 3.0</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>) (<span class="cit-pub-date">2015</span>) <span class="cit-fpage">D405</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku938%26rft_id%253Dinfo%253Apmid%252F25300482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25300482&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.43"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Fleri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Paul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.K.</span> <span class="cit-name-surname">Dhanda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mahajan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-article-title">The Immune Epitope Database and Analysis Resource in Epitope Discovery and Synthetic Vaccine Design</span>, <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-vol">8</span> (<span class="cit-pub-date">2017</span>) <span class="cit-fpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol%26rft.volume%253D8%26rft.spage%253D278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.44" data-doi="10.1186/1471-2105-8-361"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.H.</span> <span class="cit-name-surname">Bui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Fusseder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-article-title">Development of an epitope conservancy analysis tool to facilitate the design of epitope-based diagnostics and vaccines</span>, <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBui%26rft.auinit1%253DH.%2BH.%26rft.volume%253D8%26rft.spage%253D361%26rft.epage%253D361%26rft.atitle%253DDevelopment%2Bof%2Ban%2Bepitope%2Bconservancy%2Banalysis%2Btool%2Bto%2Bfacilitate%2Bthe%2Bdesign%2Bof%2Bepitope-based%2Bdiagnostics%2Band%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-361%26rft_id%253Dinfo%253Apmid%252F17897458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897458&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.45" data-doi="10.1016/j.pharmthera.2004.07.007"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.N.</span> <span class="cit-name-surname">Sarkar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.C.</span> <span class="cit-name-surname">Sen</span></span>, <span class="cit-article-title">Novel functions of proteins encoded by viral stress-inducible genes</span>, <abbr class="cit-jnl-abbrev">Pharmacol Ther</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2004</span>) <span class="cit-fpage">245</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacology%2B%2526%2Btherapeutics%26rft.stitle%253DPharmacol%2BTher%26rft.aulast%253DSarkar%26rft.auinit1%253DS.%2BN.%26rft.volume%253D103%26rft.issue%253D3%26rft.spage%253D245%26rft.epage%253D259%26rft.atitle%253DNovel%2Bfunctions%2Bof%2Bproteins%2Bencoded%2Bby%2Bviral%2Bstress-inducible%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pharmthera.2004.07.007%26rft_id%253Dinfo%253Apmid%252F15464592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pharmthera.2004.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15464592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224485000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.46" data-doi="10.1615/CritRevImmunol.v28.i2.40"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fuse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.J.</span> <span class="cit-name-surname">Molloy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.J.</span> <span class="cit-name-surname">Usherwood</span></span>, <span class="cit-article-title">Immune responses against persistent viral infections: possible avenues for immunotherapeutic interventions</span>, <abbr class="cit-jnl-abbrev">Crit Rev Immunol</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">2</span>) (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">159</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bimmunology%26rft.stitle%253DCrit%2BRev%2BImmunol%26rft.aulast%253DFuse%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D183%26rft.atitle%253DImmune%2Bresponses%2Bagainst%2Bpersistent%2Bviral%2Binfections%253A%2Bpossible%2Bavenues%2Bfor%2Bimmunotherapeutic%2Binterventions.%26rft_id%253Dinfo%253Adoi%252F10.1615%252FCritRevImmunol.v28.i2.40%26rft_id%253Dinfo%253Apmid%252F18540829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1615/CritRevImmunol.v28.i2.40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18540829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254868400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.47"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. Tahir ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Shahid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Ali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.Z.</span> <span class="cit-name-surname">Fareed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-L.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-article-title">Structural modeling and conserved epitopes prediction against SARS-COV-2 structural proteins for vaccine development</span>, <abbr class="cit-jnl-abbrev">Research Square</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.48" data-doi="10.1093/nar/gkw329"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lamiable</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Thevenet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vavrusa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Derreumaux</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tuffery</span></span>, <span class="cit-article-title">PEP-FOLD3: faster de novo structure prediction for linear peptides in solution and in complex</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">W1</span>) (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">W449</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw329%26rft_id%253Dinfo%253Apmid%252F27131374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27131374&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.49"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. Tahir Ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Maryam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Muneer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Xing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.A.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Anwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.H.</span> <span class="cit-name-surname">Geesi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.R.</span> <span class="cit-name-surname">Khalid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.A.</span> <span class="cit-name-surname">Rauf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.R.</span> <span class="cit-name-surname">Siddiqi</span></span>, <span class="cit-article-title">Computational screening of medicinal plant phytochemicals to discover potent pan-serotype inhibitors against dengue virus</span>, <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">1433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D1433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.50"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Durdagi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. Tahir Ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.E.</span> <span class="cit-name-surname">Salmas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Tariq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Anwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-article-title">Investigating the molecular mechanism of staphylococcal DNA gyrase inhibitors: A combined ligand-based and structure-based resources pipeline</span>, <abbr class="cit-jnl-abbrev">J Mol Graph Model</abbr> <span class="cit-vol">85</span> (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">122</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BGraph%2BModel%26rft.volume%253D85%26rft.spage%253D122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.51"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.L.</span> <span class="cit-name-surname">DeLano</span></span>, <span class="cit-article-title">Pymol: An open-source molecular graphics tool</span>, <abbr class="cit-jnl-abbrev">CCP4 Newsletter on protein crystallography</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">82</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCCP4%2BNewsletter%2Bon%2Bprotein%2Bcrystallography%26rft.volume%253D40%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.52" data-doi="10.1038/nprot.2010.32"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.J.</span> <span class="cit-name-surname">de Vries</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">van Dijk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.M.</span> <span class="cit-name-surname">Bonvin</span></span>, <span class="cit-article-title">The HADDOCK web server for data-driven biomolecular docking</span>, <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5</span>) (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">883</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253Dde%2BVries%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D883%26rft.epage%253D897%26rft.atitle%253DThe%2BHADDOCK%2Bweb%2Bserver%2Bfor%2Bdata-driven%2Bbiomolecular%2Bdocking.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2010.32%26rft_id%253Dinfo%253Apmid%252F20431534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2010.32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20431534&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277174100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.28.970343v2.53"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Gasteiger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hoogland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gattiker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.R.</span> <span class="cit-name-surname">Wilkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.D.</span> <span class="cit-name-surname">Appel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bairoch</span></span>, <span class="cit-chapter-title">Protein identification and analysis tools on the ExPASy server</span>, <span class="cit-source">The proteomics protocols handbook</span>, <span class="cit-publ-name">Springer</span> <span class="cit-pub-date">2005</span>, pp. <span class="cit-fpage">571</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.28.970343v2.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Dimitrov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.R.</span> <span class="cit-name-surname">Flower</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Doytchinova</span></span>, <span class="cit-chapter-title">AllerTOP-a server for in silico prediction of allergens</span>, <span class="cit-source">BMC bioinformatics</span>, <span class="cit-publ-name">BioMed Central</span>, <span class="cit-pub-date">2013</span>, p. <span class="cit-fpage">S4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.55" data-doi="10.1093/bioinformatics/16.4.404"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.J.</span> <span class="cit-name-surname">McGuffin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bryson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.T.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-article-title">The PSIPRED protein structure prediction server</span>, <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2000</span>) <span class="cit-fpage">404</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMcGuffin%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D404%26rft.epage%253D405%26rft.atitle%253DThe%2BPSIPRED%2Bprotein%2Bstructure%2Bprediction%2Bserver%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F16.4.404%26rft_id%253Dinfo%253Apmid%252F10869041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/16.4.404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10869041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087933900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.56" data-doi="10.1093/nar/gkt458"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Heo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Seok</span></span>, <span class="cit-article-title">GalaxyRefine: Protein structure refinement driven by side-chain repacking</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">Web Server issue</span>) (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">W384</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt458%26rft_id%253Dinfo%253Apmid%252F23737448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23737448&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323603200061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.57"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.C.</span> <span class="cit-name-surname">Lovell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.W.</span> <span class="cit-name-surname">Davis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.B.</span> <span class="cit-name-surname">Arendall</span>   <span class="cit-name-suffix">III</span>.</span>, <span class="cit-auth"> <span class="cit-name-given-names">P.I.</span> <span class="cit-name-surname">De Bakker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.M.</span> <span class="cit-name-surname">Word</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.G.</span> <span class="cit-name-surname">Prisant</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.S.</span> <span class="cit-name-surname">Richardson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.C.</span> <span class="cit-name-surname">Richardson</span></span>, <span class="cit-article-title">Structure validation by Cα geometry: ϕ, ? and Cβ deviation</span>, <abbr class="cit-jnl-abbrev">Proteins: Structure, Function, and Bioinformatics</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">437</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStructure%252C%2BFunction%252C%2Band%2BBioinformatics%26rft.volume%253D50%26rft.spage%253D437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.58" data-doi="10.1093/nar/gkm290"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wiederstein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.J.</span> <span class="cit-name-surname">Sippl</span></span>, <span class="cit-article-title">ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">Web Server issue</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">W407</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm290%26rft_id%253Dinfo%253Apmid%252F17517781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17517781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500076&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.59"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lengths</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Angles</span></span>, <span class="cit-article-title">Limitations of structure evaluation tools errat</span>, <abbr class="cit-jnl-abbrev">Quick Guideline Comput Drug Des</abbr> <span class="cit-vol">16</span> (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuick%2BGuideline%2BComput%2BDrug%2BDes%26rft.volume%253D16%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[60].</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.60"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kuriata</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.M.</span> <span class="cit-name-surname">Gierut</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Oleniecki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.P.</span> <span class="cit-name-surname">Ciemny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kolinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kurcinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kmiecik</span></span>, <span class="cit-article-title">CABS-flex 2.0: a web server for fast simulations of flexibility of protein structures</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">W1</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">W338</span>–<span class="cit-lpage">W343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[61].</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.61"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kurcinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Oleniecki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.P.</span> <span class="cit-name-surname">Ciemny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kuriata</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kolinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kmiecik</span></span>, <span class="cit-article-title">CABS-flex standalone: a simulation environment for fast modeling of protein flexibility</span>, <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">694</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[62].</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.62"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rapin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lund</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bernaschi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Castiglione</span></span>, <span class="cit-article-title">Computational immunology meets bioinformatics: the use of prediction tools for molecular binding in the simulation of the immune system</span>, <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[63].</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.28.970343v2.63"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.T.</span> <span class="cit-name-surname">Kroger</span></span>, <span class="cit-source">General recommendations on immunization, US Department of Health and Human Services</span>, <span class="cit-publ-name">Public Health Service, Centers for Disease Control</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[64].</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.64"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Castiglione</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Mantile</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">De Berardinis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Prisco</span></span>, <span class="cit-article-title">How the interval between prime and boost injection affects the immune response in a computational model of the immune system</span>, <abbr class="cit-jnl-abbrev">Computational and mathematical methods in medicine 2012</abbr> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[65].</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.65" data-doi="10.1093/nar/gkn860"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.A.</span> <span class="cit-name-surname">Laskowski</span></span>, <span class="cit-article-title">PDBsum new things</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">Database issue</span>) (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">D355</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn860%26rft_id%253Dinfo%253Apmid%252F18996896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn860&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18996896&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261906200064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[66].</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.66" data-doi="10.1016/S0959-440X(02)00308-1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hansson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Oostenbrink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">van Gunsteren</span></span>, <span class="cit-article-title">Molecular dynamics simulations</span>, <abbr class="cit-jnl-abbrev">Curr Opin Struct Biol</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>) (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">190</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DHansson%26rft.auinit1%253DT.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D190%26rft.epage%253D196%26rft.atitle%253DMolecular%2Bdynamics%2Bsimulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252802%252900308-1%26rft_id%253Dinfo%253Apmid%252F11959496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(02)00308-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11959496&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175043000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[67].</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.67" data-doi="10.1016/j.softx.2015.06.001"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.J.</span> <span class="cit-name-surname">Abraham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Murtola</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Schulz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Páll</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.C.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hess</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lindahl</span></span>, <span class="cit-article-title">GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers</span>, <abbr class="cit-jnl-abbrev">SoftwareX</abbr> <span class="cit-vol">1</span> (<span class="cit-pub-date">2015</span>) <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftwareX%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.softx.2015.06.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.softx.2015.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[68].</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.68"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Muneer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.T.</span> <span class="cit-name-surname">Ul Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tusleem</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Abdul Rauf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.M.J.</span> <span class="cit-name-surname">Hussain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.R.</span> <span class="cit-name-surname">Siddiqi</span></span>, <span class="cit-article-title">Discovery of selective inhibitors for cyclic AMP response element-binding protein: a combined ligand and structure-based resources pipeline</span>, <abbr class="cit-jnl-abbrev">Anticancer Drugs</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">363</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnticancer%2BDrugs%26rft.volume%253D30%26rft.spage%253D363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[69].</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [69] in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.69" data-doi="10.1093/nar/gki376"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Grote</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hiller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Scheer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Munch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nortemann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.C.</span> <span class="cit-name-surname">Hempel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jahn</span></span>, <span class="cit-article-title">JCat: a novel tool to adapt codon usage of a target gene to its potential expression host</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">Web Server issue</span>) (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">W526</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGrote%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW526%26rft.epage%253DW531%26rft.atitle%253DJCat%253A%2Ba%2Bnovel%2Btool%2Bto%2Badapt%2Bcodon%2Busage%2Bof%2Ba%2Btarget%2Bgene%2Bto%2Bits%2Bpotential%2Bexpression%2Bhost%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki376%26rft_id%253Dinfo%253Apmid%252F15980527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400107&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[70].</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.70" data-doi="10.1126/science.3296194"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.L.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.G.</span> <span class="cit-name-surname">Econome</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schutt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Klco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.R.</span> <span class="cit-name-surname">Cantor</span></span>, <span class="cit-article-title">A physical map of the Escherichia coli K12 genome</span>, <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">236</span>(<span class="cit-issue">4807</span>) (<span class="cit-pub-date">1987</span>) <span class="cit-fpage">1448</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSmith%26rft.auinit1%253DC.%26rft.volume%253D236%26rft.issue%253D4807%26rft.spage%253D1448%26rft.epage%253D1453%26rft.atitle%253DA%2Bphysical%2Bmap%2Bof%2Bthe%2BEscherichia%2Bcoli%2BK12%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3296194%26rft_id%253Dinfo%253Apmid%252F3296194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzYvNDgwNy8xNDQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjIvMjAyMC4wMi4yOC45NzAzNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[71].</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [71] in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.71" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.M.</span> <span class="cit-name-surname">Sharp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.H.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">The codon Adaptation Index--a measure of directional synonymous codon usage bias, and its potential applications</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">1987</span>) <span class="cit-fpage">1281</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.3.1281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3547335&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987F976100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[72].</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference [72] in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.72"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.V.</span> <span class="cit-name-surname">Luckheeram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.D.</span> <span class="cit-name-surname">Verma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Xia</span></span>, <span class="cit-article-title">CD4+ T cells: differentiation and functions</span>, <abbr class="cit-jnl-abbrev">Clinical and developmental immunology 2012</abbr> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[73].</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [73] in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.73" data-doi="10.1093/protein/14.8.529"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Arai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ueda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kitayama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kamiya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nagamune</span></span>, <span class="cit-article-title">Design of the linkers which effectively separate domains of a bifunctional fusion protein</span>, <abbr class="cit-jnl-abbrev">Protein engineering</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">8</span>) (<span class="cit-pub-date">2001</span>) <span class="cit-fpage">529</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DArai%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D529%26rft.epage%253D532%26rft.atitle%253DDesign%2Bof%2Bthe%2Blinkers%2Bwhich%2Beffectively%2Bseparate%2Bdomains%2Bof%2Ba%2Bbifunctional%2Bfusion%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252F14.8.529%26rft_id%253Dinfo%253Apmid%252F11579220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/14.8.529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11579220&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171561700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[74].</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [74] in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.74"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.K.</span> <span class="cit-name-surname">Pandey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Narula</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Naskar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Srivastava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Verma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Malik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Shah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Prajapati</span></span>, <span class="cit-article-title">Exploring dual inhibitory role of febrifugine analogues against Plasmodium utilizing structure-based virtual screening and molecular dynamic simulation</span>, <abbr class="cit-jnl-abbrev">Journal of Biomolecular Structure and Dynamics</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2017</span>) <span class="cit-fpage">791</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiomolecular%2BStructure%2Band%2BDynamics%26rft.volume%253D35%26rft.spage%253D791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[75].</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [75] in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.75" data-doi="10.1016/j.compbiolchem.2016.04.006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nezafat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Karimi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eslami</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mohkam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zandian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ghasemi</span></span>, <span class="cit-article-title">Designing an efficient multi-epitope peptide vaccine against Vibrio cholerae via combined immunoinformatics and protein interaction based approaches</span>, <abbr class="cit-jnl-abbrev">Comput Biol Chem</abbr> <span class="cit-vol">62</span> (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">82</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput%2BBiol%2BChem%26rft.volume%253D62%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2016.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2016.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[76].</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [76] in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.76" data-doi="10.4049/jimmunol.168.11.5499"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Livingston</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Crimi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Newman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Higashimoto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Appella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-article-title">A rational strategy to design multiepitope immunogens based on multiple Th lymphocyte epitopes</span>, <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-vol">168</span>(<span class="cit-issue">11</span>) (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">5499</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DLivingston%26rft.auinit1%253DB.%26rft.volume%253D168%26rft.issue%253D11%26rft.spage%253D5499%26rft.epage%253D5506%26rft.atitle%253DA%2BRational%2BStrategy%2Bto%2BDesign%2BMultiepitope%2BImmunogens%2BBased%2Bon%2BMultiple%2BTh%2BLymphocyte%2BEpitopes%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.168.11.5499%26rft_id%253Dinfo%253Apmid%252F12023344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE2OC8xMS81NDk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjIvMjAyMC4wMi4yOC45NzAzNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[77].</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [77] in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.77" data-doi="10.1016/j.coi.2008.03.003"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.E.</span> <span class="cit-name-surname">Lund</span></span>, <span class="cit-article-title">Cytokine-producing B lymphocytes—key regulators of immunity</span>, <abbr class="cit-jnl-abbrev">Current opinion in immunology</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">3</span>) (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">332</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bimmunology%26rft.stitle%253DCurr%2BOpin%2BImmunol%26rft.aulast%253DLund%26rft.auinit1%253DF.%2BE.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D332%26rft.epage%253D338%26rft.atitle%253DCytokine-producing%2BB%2Blymphocytes-key%2Bregulators%2Bof%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coi.2008.03.003%26rft_id%253Dinfo%253Apmid%252F18417336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coi.2008.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18417336&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257655200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[78].</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [78] in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.78"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. Rehan</span> <span class="cit-name-surname">Khalid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. Tahir Ul</span> <span class="cit-name-surname">Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Maryam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ashique</span></span>, <span class="cit-auth"><span class="cit-name-surname">F. Anwar</span>,  <span class="cit-name-given-names">H.G. M</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.R.</span> <span class="cit-name-surname">Siddiqi</span></span>, <span class="cit-article-title">Comparative Studies of the Dynamics Effects of BAY60-2770 and BAY58-2667 Binding with Human and Bacterial H-NOX Domains</span>, <abbr class="cit-jnl-abbrev">Molecules</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">9</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">2141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecules%26rft.volume%253D23%26rft.spage%253D2141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[79].</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference [79] in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.79" data-doi="10.1038/s41598-017-08842-w"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Khatoon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.K.</span> <span class="cit-name-surname">Pandey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.K.</span> <span class="cit-name-surname">Prajapati</span></span>, <span class="cit-article-title">Exploring Leishmania secretory proteins to design B and T cell multi-epitope subunit vaccine using immunoinformatics approach</span>, <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>) (<span class="cit-pub-date">2017</span>) <span class="cit-fpage">8285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D8285%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-08842-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-08842-w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[80].</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference [80] in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.80"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.K.</span> <span class="cit-name-surname">Dubey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.A.</span> <span class="cit-name-surname">Luke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Knox</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.I.</span> <span class="cit-name-surname">Pletschke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.K.</span> <span class="cit-name-surname">Singh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Shukla</span></span>, <span class="cit-article-title">Vaccine and antibody production in plants: developments and computational tools</span>, <abbr class="cit-jnl-abbrev">Brief Funct Genomics</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">5</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">295</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct%2BGenomics%26rft.volume%253D17%26rft.spage%253D295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[81].</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference [81] in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.81" data-doi="10.1002/rmv.602"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-article-title">An introduction to epitope prediction methods and software</span>, <abbr class="cit-jnl-abbrev">Rev Med Virol</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">2</span>) (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">77</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReviews%2Bin%2Bmedical%2Bvirology%26rft.stitle%253DRev%2BMed%2BVirol%26rft.aulast%253DYang%26rft.auinit1%253DX.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D77%26rft.epage%253D96%26rft.atitle%253DAn%2Bintroduction%2Bto%2Bepitope%2Bprediction%2Bmethods%2Band%2Bsoftware.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frmv.602%26rft_id%253Dinfo%253Apmid%252F19101924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/rmv.602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19101924&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264423400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[82].</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference [82] in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.82" data-doi="10.1016/j.virol.2010.12.016"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.J.</span> <span class="cit-name-surname">Amanna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.K.</span> <span class="cit-name-surname">Slifka</span></span>, <span class="cit-article-title">Contributions of humoral and cellular immunity to vaccine-induced protection in humans</span>, <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">411</span>(<span class="cit-issue">2</span>) (<span class="cit-pub-date">2011</span>) <span class="cit-fpage">206</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D411%26rft.spage%253D206%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.12.016%26rft_id%253Dinfo%253Apmid%252F21216425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21216425&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288343000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[83].</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference [83] in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.83" data-doi="10.1038/nm1210"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pashine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.M.</span> <span class="cit-name-surname">Valiante</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.B.</span> <span class="cit-name-surname">Ulmer</span></span>, <span class="cit-article-title">Targeting the innate immune response with improved vaccine adjuvants</span>, <abbr class="cit-jnl-abbrev">Nature medicine</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">S63</span>–<span class="cit-lpage">S68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DPashine%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D4%2BSuppl%26rft.spage%253DS63%26rft.epage%253DS68%26rft.atitle%253DTargeting%2Bthe%2Binnate%2Bimmune%2Bresponse%2Bwith%2Bimproved%2Bvaccine%2Badjuvants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1210%26rft_id%253Dinfo%253Apmid%252F15812492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15812492&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F22%2F2020.02.28.970343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228339600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[84].</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference [84] in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.28.970343v2.84"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shamriz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ofoghi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Moazami</span></span>, <span class="cit-article-title">Effect of linker length and residues on the structure and stability of a fusion protein with malaria vaccine application</span>, <abbr class="cit-jnl-abbrev">Comput Biol Med</abbr> <span class="cit-vol">76</span> (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">24</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput%2BBiol%2BMed%26rft.volume%253D76%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.28.970343v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.20.999979v1" title="True S-cones are concentrated in the ventral mouse retina for color detection in the upper visual field" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/673509v2" title="Body size but not age influences phototaxis in bumble bee (Bombus terrestris, L.) workers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 22, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.28.970343v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.28.970343v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.28.970343v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.28.970343v2" class="active" data-icon-position="" data-hide-link-title="0">Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1206600" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GNWZeC_ySwtUgDwsJAdiBnbhnf-6DgHCoatbpvUtH2Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1206600" id="node1206600" data-pisa="biorxiv;2020.02.28.970343v2" data-pisa-master="biorxiv;2020.02.28.970343" data-seqnum="1206600" data-apath="/biorxiv/early/2020/03/22/2020.02.28.970343.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and <em>in silico</em> approaches</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Muhammad Tahir</span> <span class="nlm-surname">ul Qamar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Abdur</span> <span class="nlm-surname">Rehman</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Usman Ali</span> <span class="nlm-surname">Ashfaq</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhammad Qasim</span> <span class="nlm-surname">Awan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Israr</span> <span class="nlm-surname">Fatima</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Farah</span> <span class="nlm-surname">Shahid</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ling-Ling</span> <span class="nlm-surname">Chen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.28.970343; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.28.970343 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;title=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;title=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;text=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;title=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;t=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;title=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;title=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1206600" id="node1206600--2" data-pisa="biorxiv;2020.02.28.970343v2" data-pisa-master="biorxiv;2020.02.28.970343" data-seqnum="1206600" data-apath="/biorxiv/early/2020/03/22/2020.02.28.970343.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and <em>in silico</em> approaches</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Muhammad Tahir</span> <span class="nlm-surname">ul Qamar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Abdur</span> <span class="nlm-surname">Rehman</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Usman Ali</span> <span class="nlm-surname">Ashfaq</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhammad Qasim</span> <span class="nlm-surname">Awan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Israr</span> <span class="nlm-surname">Fatima</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Farah</span> <span class="nlm-surname">Shahid</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ling-Ling</span> <span class="nlm-surname">Chen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.28.970343; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.28.970343 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=ul+Qamar%2C+Muhammad+Tahir&amp;rft.au=Rehman%2C+Abdur&amp;rft.au=Ashfaq%2C+Usman+Ali&amp;rft.au=Awan%2C+Muhammad+Qasim&amp;rft.au=Fatima%2C+Israr&amp;rft.au=Shahid%2C+Farah&amp;rft.au=Chen%2C+Ling-Ling"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1206600/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1206600/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1206600/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1206600/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1206600/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1206600/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1206600/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1206600/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1206600/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1206600/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1206600/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1206600/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;count=horizontal&amp;via=&amp;text=Designing%20of%20a%20next%20generation%20multiepitope%20based%20vaccine%20%28MEV%29%20against%20SARS-COV-2%3A%20Immunoinformatics%20and%20in%20silico%20approaches&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.28.970343v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.28.970343v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
