<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/25/2020.03.23.004580.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.23.004580v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="FEP-based screening prompts drug repositioning against COVID-19"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.23.004580"/>
  <meta name="DC.Date" content="2020-03-25"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated in vitro exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-κB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19."/>
  <meta name="DC.Contributor" content="Zhe Li"/>
  <meta name="DC.Contributor" content="Xin Li"/>
  <meta name="DC.Contributor" content="Yi-You Huang"/>
  <meta name="DC.Contributor" content="Yaoxing Wu"/>
  <meta name="DC.Contributor" content="Lingli Zhou"/>
  <meta name="DC.Contributor" content="Runduo Liu"/>
  <meta name="DC.Contributor" content="Deyan Wu"/>
  <meta name="DC.Contributor" content="Lei Zhang"/>
  <meta name="DC.Contributor" content="Hao Liu"/>
  <meta name="DC.Contributor" content="Ximing Xu"/>
  <meta name="DC.Contributor" content="Yuxia Zhang"/>
  <meta name="DC.Contributor" content="Jun Cui"/>
  <meta name="DC.Contributor" content="Xin Wang"/>
  <meta name="DC.Contributor" content="Hai-Bin Luo"/>
  <meta name="article:published_time" content="2020-03-25"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="FEP-based screening prompts drug repositioning against COVID-19"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated in vitro exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-kB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.23.004580v1"/>
  <meta name="citation_id" content="2020.03.23.004580v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004580v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004580v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004580v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.23.004580"/>
  <meta name="citation_num_pages" content="34"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.23.004580"/>
  <meta name="citation_author" content="Zhe Li"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, School of Pharmaceutical Sciences, Sun Yat-Sen University"/>
  <meta name="citation_author" content="Xin Li"/>
  <meta name="citation_author_institution" content="Center for Innovative Marine Drug Screening &amp;; Evaluation (QNLM), School of Medicine and Pharmacy, Ocean University of China"/>
  <meta name="citation_author_institution" content="School of Life Sciences, Lanzhou University"/>
  <meta name="citation_author" content="Yi-You Huang"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, School of Pharmaceutical Sciences, Sun Yat-Sen University"/>
  <meta name="citation_author" content="Yaoxing Wu"/>
  <meta name="citation_author_institution" content="MOE Key Laboratory of Gene Function and Regulation, State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University"/>
  <meta name="citation_author" content="Lingli Zhou"/>
  <meta name="citation_author_institution" content="MOE Key Laboratory of Gene Function and Regulation, State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University"/>
  <meta name="citation_author" content="Runduo Liu"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, School of Pharmaceutical Sciences, Sun Yat-Sen University"/>
  <meta name="citation_author" content="Deyan Wu"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, School of Pharmaceutical Sciences, Sun Yat-Sen University"/>
  <meta name="citation_author" content="Lei Zhang"/>
  <meta name="citation_author_institution" content="MOE Key Laboratory of Gene Function and Regulation, State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University"/>
  <meta name="citation_author" content="Hao Liu"/>
  <meta name="citation_author_institution" content="High Performance Computing Center, Pilot National Laboratory for Marine Science and Technology (QNLM)"/>
  <meta name="citation_author" content="Ximing Xu"/>
  <meta name="citation_author_institution" content="Center for Innovative Marine Drug Screening &amp;; Evaluation (QNLM), School of Medicine and Pharmacy, Ocean University of China"/>
  <meta name="citation_author_institution" content="Marine Biomedical Research Institute of Qingdao"/>
  <meta name="citation_author" content="Yuxia Zhang"/>
  <meta name="citation_author_institution" content="Guangzhou Institute of Pediatrics, Guangzhou Women and Children’s Medical Center, State Key Laboratory of Respiratory Diseases, Guangzhou Medical University"/>
  <meta name="citation_author" content="Jun Cui"/>
  <meta name="citation_author_institution" content="MOE Key Laboratory of Gene Function and Regulation, State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University"/>
  <meta name="citation_author" content="Xin Wang"/>
  <meta name="citation_author_institution" content="Center for Innovative Marine Drug Screening &amp;; Evaluation (QNLM), School of Medicine and Pharmacy, Ocean University of China"/>
  <meta name="citation_author_institution" content="Marine Biomedical Research Institute of Qingdao"/>
  <meta name="citation_author" content="Hai-Bin Luo"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of New Drug Design and Evaluation, School of Pharmaceutical Sciences, Sun Yat-Sen University"/>
  <meta name="citation_author_email" content="luohb77@mail.sysu.edu.cn"/>
  <meta name="citation_reference" content="Chen, N.; Zhou, M.; Dong, X.; Qu, J.; Gong, F.; Han, Y.; Qiu, Y.; Wang, J.; Liu, Y.; Wei, Y.; Xia, J.; Yu, T.; Zhang, X.; Zhang, L., Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet 2020."/>
  <meta name="citation_reference" content="Huang, C.; Wang, Y.; Li, X.; Ren, L.; Zhao, J.; Hu, Y.; Zhang, L.; Fan, G.; Xu, J.; Gu, X.; Cheng, Z.; Yu, T.; Xia, J.; Wei, Y.; Wu, W.; Xie, X.; Yin, W.; Li, H.; Liu, M.; Xiao, Y.; Gao, H.; Guo, L.; Xie, J.; Wang, G.; Jiang, R.; Gao, Z.; Jin, Q.; Wang, J.; Cao, B., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 2020."/>
  <meta name="citation_reference" content="Holshue, M. L.; DeBolt, C.; Lindquist, S.; Lofy, K. H.; Wiesman, J.; Bruce, H.; Spitters, C.; Ericson, K.; Wilkerson, S.; Tural, A.; Diaz, G.; Cohn, A.; Fox, L.; Patel, A.; Gerber, S. I.; Kim, L.; Tong, S.; Lu, X.; Lindstrom, S.; Pallansch, M. A.; Weldon, W. C.; Biggs, H. M.; Uyeki, T. M.; Pillai, S. K.; Washington State -nCo, V. C. I. T., First Case of 2019 Novel Coronavirus in the United States. N Engl J Med 2020."/>
  <meta name="citation_reference" content="Adedeji, A. O.; Sarafianos, S. G., Antiviral drugs specific for coronaviruses in preclinical development. Curr Opin Virol 2014, 8, 45–53."/>
  <meta name="citation_reference" content="Jin, Z.; Du, X.; Xu, Y.; Deng, Y.; Liu, M.; Zhao, Y.; Zhang, B.; Li, X.; Zhang, L.; Duan, Y., et al, Structure of Mpro from COVID-19 virus and discovery of its inhibitors. bioRxiv 2020."/>
  <meta name="citation_reference" content="Li, Y., Zhang, J.; Wang, N.; Li, H.; Shi Y.; Guo, G.; Liu, K.; Zeng, H.; Zou, Q. Therapeutic drugs targeting 2019-nCoV main protease by high-throughput screening. bioRxiv 2020."/>
  <meta name="citation_reference" content="Xiong, R.; Zhang, L.; Li, S.; Sun, Y.; Ding, M.; Wang, Y.; Zhao, Y.; Wu, Y.; Shang, W.; Jiang, X., Novel and potent inhibitors targeting DHODH, a rate-limiting enzyme in de novo pyrimidine biosynthesis, are broad-spectrum antiviral against RNA viruses including newly emerged coronavirus SARS-CoV-2. bioRxiv 2020."/>
  <meta name="citation_reference" content="Wang, L.; Wu, Y.; Deng, Y.; Kim, B.; Pierce, L.; Krilov, G.; Lupyan, D.; Robinson, S.; Dahlgren, M. K.; Greenwood, J.; Romero, D. L.; Masse, C.; Knight, J. L.; Steinbrecher, T.; Beuming, T.; Damm, W.; Harder, E.; Sherman, W.; Brewer, M.; Wester, R.; Murcko, M.; Frye, L.; Farid, R.; Lin, T.; Mobley, D. L.; Jorgensen, W. L.; Berne, B. J.; Friesner, R. A.; Abel, R., Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field. J. Am. Chem. Soc. 2015, 137 (7), 2695–703."/>
  <meta name="citation_reference" content="Steinbrecher, T. B.; Dahlgren, M.; Cappel, D.; Lin, T.; Wang, L.; Krilov, G.; Abel, R.; Friesner, R.; Sherman, W., Accurate Binding Free Energy Predictions in Fragment Optimization. Journal of chemical information and modeling 2015, 55 (11), 2411–20."/>
  <meta name="citation_reference" content="Sun, Y.; Kollman, P. A., Determination of solvation free energy using molecular dynamics with solute Cartesian mapping: An application to the solvation of 18-crown-6. J. Chem. Phys. 1992, 97 (7), 5108–5112."/>
  <meta name="citation_reference" content="Radmer, R. J.; Kollman, P. A., Free energy calculation methods: a theoretical and empirical comparison of numerical errors and a new method qualitative estimates of free energy changes. J. Comput. Chem. 1997, 18 (7), 902–919."/>
  <meta name="citation_reference" content="Briggs, J. M.; Marrone, T. J.; McCammon, J. A., Computational science new horizons and relevance to pharmaceutical design. Trends Cardiovas. Med. 1996, 6 (6), 198–203."/>
  <meta name="citation_reference" content="Kim, J. T.; Hamilton, A. D.; Bailey, C. M.; Domoal, R. A.; Wang, L.; Anderson, K. S.; Jorgensen, W. L., FEP-guided selection of bicyclic heterocycles in lead optimization for non-nucleoside inhibitors of HIV-1 reverse transcriptase. J. Am. Chem. Soc. 2006, 128 (48), 15372–15373."/>
  <meta name="citation_reference" content="Jorgensen, W. L., Computer-aided discovery of anti-HIV agents. Bioorganic &amp;; medicinal chemistry 2016, 24 (20), 4768–4778."/>
  <meta name="citation_reference" content="Abel, R.; Wang, L.; Harder, E. D.; Berne, B.; Friesner, R. A., Advancing Drug Discovery through Enhanced Free Energy Calculations. Accounts of chemical research 2017, 50 (7), 1625–1632."/>
  <meta name="citation_reference" content="Aldeghi, M.; Heifetz, A.; Bodkin, M. J.; Knapp, S.; Biggin, P. C., Accurate calculation of the absolute free energy of binding for drug molecules. Chemical science 2016, 7 (1), 207–218."/>
  <meta name="citation_reference" content="Aldeghi, M.; Heifetz, A.; Bodkin, M. J.; Knapp, S.; Biggin, P. C., Predictions of ligand selectivity from absolute binding free energy calculations. Journal of the American Chemical Society 2017, 139 (2), 946–957."/>
  <meta name="citation_reference" content="Liu, X.; Li, Z.; Liu, S.; Chen, Z.; Zhao, Z.; Huang, Y.-y.; Zhang, Q.; Wang, J.; Shi, Y.; Xu, Y., et al, Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction. medRxiv 2020."/>
  <meta name="citation_reference" content="Jin, Z.; Du, X.; Xu, Y.; Deng, Y.; Liu, M.; Zhao, Y.; Zhang, B.; Li, X.; Zhang, L.; Peng, C., et al, Structure of Mpro from COVID-19 virus and discovery of its inhibitors. bioRxiv 2020."/>
  <meta name="citation_reference" content="Halgren, T. A.; Murphy, R. B.; Friesner, R. A.; Beard, H. S.; Frye, L. L.; Pollard, W. T.; Banks, J. L., Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening. Journal of medicinal chemistry 2004, 47 (7), 1750–1759."/>
  <meta name="citation_reference" content="Hess, B.; Kutzner, C.; Van Der Spoel, D.; Lindahl, E., GROMACS 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation. Journal of chemical theory computation 2008, 4 (3), 435–447."/>
  <meta name="citation_reference" content="Pronk, S.; Páll, S.; Schulz, R.; Larsson, P.; Bjelkmar, P.; Apostolov, R.; Shirts, M. R.; Smith, J. C.; Kasson, P. M.; van der Spoel, D., GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit. J Bioinformatics 2013, 29 (7), 845–854."/>
  <meta name="citation_reference" content="Wang, J.; Wolf, R. M.; Caldwell, J. W.; Kollman, P. A.; Case, D. A., Development and testing of a general amber force field. Journal of computational chemistry 2004, 25 (9), 1157–1174."/>
  <meta name="citation_reference" content="Frisch, M.; Trucks, G.; Schlegel, H.; Scuseria, G.; Robb, M.; Cheeseman, J.; Montgomery Jr, J.; Vreven, T.; Kudin, K.; Burant, J., Gaussian 03, revision C. 02; Gaussian, Inc.: Wallingford, CT, 2004. 2013."/>
  <meta name="citation_reference" content="Case, D.; Babin, V.; Berryman, J.; Betz, R.; Cai, Q.; Cerutti, D.; Cheatham III, T.; Darden, T.; Duke, R.; Gohlke, H., The FF14SB force field. Amber 2014, 14, 29–31."/>
  <meta name="citation_reference" content="Jorgensen, W. L.; Chandrasekhar, J.; Madura, J. D.; Impey, R. W.; Klein, M. L., Comparison of simple potential functions for simulating liquid water. The Journal of chemical physics 1983, 79 (2), 926–935."/>
  <meta name="citation_reference" content="Boresch, S.; Tettinger, F.; Leitgeb, M.; Karplus, M., Absolute binding free energies: a quantitative approach for their calculation. The Journal of Physical Chemistry B 2003, 107 (35), 9535–9551."/>
  <meta name="citation_reference" content="Hummer, G.; Pratt, L. R.; García, A. E., Multistate Gaussian model for electrostatic solvation fr ee energies. Journal of the American Chemical Society 1997, 119 (36), 8523–8527."/>
  <meta name="citation_reference" content="Li, Z.; Huang, Y.; Wu, Y.; Chen, J.; Wu, D.; Zhan, C.-G.; Luo, H.-B., Absolute binding free energy calculation and design of a subnanomolar inhibitor of phosphodiesterase-10. Journal of medicinal chemistry 2019, 62 (4), 2099–2111."/>
  <meta name="citation_reference" content="Van Gunsteren, W. F.; Berendsen, H. J., A leap-frog algorithm for stochastic dynamics. J Molecular Simulation 1988, 1 (3), 173–185."/>
  <meta name="citation_reference" content="Goga, N.; Rzepiela, A.; De Vries, A.; Marrink, S.; Berendsen, H., Efficient algorithms for Langevin and DPD dynamics. Journal of chemical theory computation 2012, 8 (10), 3637–3649."/>
  <meta name="citation_reference" content="Parrinello, M.; Rahman, A., Polymorphic transitions in single crystals: A new molecular dynamics method. Journal of Applied physics 1981, 52 (12), 7182–7190."/>
  <meta name="citation_reference" content="Hess, B.; Bekker, H.; Berendsen, H. J.; Fraaije, J. G., LINCS: a linear constraint solver for molecular simulations. Journal of computational chemistry 1997, 18 (12), 1463–1472."/>
  <meta name="citation_reference" content="Essmann, U.; Perera, L.; Berkowitz, M. L.; Darden, T.; Lee, H.; Pedersen, L. G., A smooth particle mesh Ewald method. The Journal of chemical physics 1995, 103 (19), 8577–8593."/>
  <meta name="citation_reference" content="Bennett, C. H., Efficient estimation of free energy differences from Monte Carlo data. Journal of Computational Physics 1976, 22 (2), 245–268."/>
  <meta name="citation_reference" content="Shirts, M. R.; Chodera, J. D., Statistically optimal analysis of samples from multiple equilibrium states. The Journal of chemical physics 2008, 129 (12), 124105."/>
  <meta name="citation_reference" content="Wang, J.; Deng, Y.; Roux, B., Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials. Biophysical journal 2006, 91 (8), 2798–2814."/>
  <meta name="citation_reference" content="Fujitani, H.; Tanida, Y.; Ito, M.; Jayachandran, G.; Snow, C. D.; Shirts, M. R.; Sorin, E. J.; Pande, V. S., Direct calculation of the binding free energies of FKBP ligands. The Journal of chemical physics 2005, 123 (8), 084108."/>
  <meta name="citation_reference" content="Lapelosa, M.; Gallicchio, E.; Levy, R. M., Conformational transitions and convergence of absolute binding free energy calculations. Journal of chemical theory computation 2012, 8 (1), 47–60."/>
  <meta name="citation_reference" content="Wang, M.; Cao, R.; Zhang, L.; Yang, X.; Liu, J.; Xu, M.; Shi, Z.; Hu, Z.; Zhong, W.; Xiao, G., Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro. Cell research 2020, 30 (3), 269–271."/>
  <meta name="citation_reference" content="Dong, L.; Hu, S.; Gao, J. J. D. D.; Therapeutics, Discovering drugs to treat coronavirus disease 2019 (COVID-19). Drug Discoveries &amp;; Therapeutics 2020, 14 (1), 58–60."/>
  <meta name="citation_reference" content="Aldeghi, M.; Heifetz, A.; Bodkin, M. J.; Knapp, S.; Biggin, P. C., Accurate calculation of the absolute free energy of binding for drug molecules. Chemical Science 2016, 7 (1), 207–218."/>
  <meta name="citation_reference" content="Aldeghi, M.; Heifetz, A.; Bodkin, M. J.; Knapp, S.; Biggin, P. C., Predictions of ligand selectivity from absolute binding free energy calculations. Journal of the American Chemical Society 2017, 139 (2), 946–957."/>
  <meta name="twitter:title" content="FEP-based screening prompts drug repositioning against COVID-19"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated in vitro exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-κB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19."/>
  <meta name="og-title" property="og:title" content="FEP-based screening prompts drug repositioning against COVID-19"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004580v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated in vitro exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-κB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-25"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>FEP-based screening prompts drug repositioning against COVID-19 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1211713 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated 
     <em>in vitro</em> exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-κB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">1. Introduction</h2>
    <p id="p-3">The novel coronavirus 2019-nCoV (also known as HCoV-19 or SARS-CoV-2) outbreak had emerged from Wuhan, Hubei Province, China in December 2019
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, 
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. On March 22, there were 813,00 confirmed COVID-19 cases including 3,253 deaths in China. This virus has also infected more than 220,000 patients in all of the continents and over 180 other countries, such as Italy, Spain, U.S.A, Germany, France, and Iran gradually became a global pandemic due to the very high risk of spread and impact of COVID-19. To date, there is no specific treatment or vaccine against COVID-19, thus it is urgently need to repositioning potential agents against COVID-19.
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>
    </p>
    <p id="p-4">The COVID-19’s replicase gene encodes two over-lapping translation products, polyproteins 1a and 1ab (pp1a and pp1ab), which mediate all of the functions required for viral replication. Mpro, as the key enzyme in proteolytic processing of viral replication, is initially released by the auto-cleavage of pp1a and pp1ab. Then Mpro in turn cleaves pp1a and pp1ab to release functional proteins necessary for viral replication.
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> In the view of essential functions of Mpro in viral life cycle and its high conservatism, it is an attractive target for the discovery of anti-COVID-19 agents.
    </p>
    <p id="p-5">Great efforts from various research groups have been done to discover new agents from several databases by targeting the target Mpro via several virtual screening strategy,
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">6</a>
     </sup> which consists of pharmacophore, molecule docking, and molecular simulations approaches. As a result, six drugs inhibited Mpro with IC
     <sub>50</sub> values ranging from 0.67 to 21.4 μM.
     <sup>
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
     </sup> These drug design methods contributed considerably to the lead discovery, but the computational accuracy and efficiency need to be improved especially when dealing with emergency situations such as the COVID-19 outbreak. Free energy perturbation (FEP) method is a promising method with satisfactory accuracy
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">7</a>-
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">14</a>
     </sup>, but their actual applications to drug design are still limited to simulate minor structural changes of the ligands, thus predicting the relative binding free energy (RBFE).
     <sup>
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">7</a>, 
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">14</a>
     </sup> In order to perform virtual screening of a large molecule database, the absolute binding free energy (ABFE) calculation must be performed for each ligand without using of a reference ligand structure. The FEP approach has an advantage in predicting the affinities more precisely between drugs and their targets than conventional methods, such as pharmacophore, molecule docking, and molecular simulations. However, the FEP-ABFE approaches are extremely expensive/time-consumpting and therefore not used for virtual screening purpose.
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">15</a>, 
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">16</a>
     </sup>
    </p>
    <p id="p-6">To accelerate the discovery of Mpro inhibitors from the small molecule database to combat COVID-19, we represent a newly derived FEP-ABFE-accelerated screening strategy together with bioassay validation to rapidly reposition potential agents against COVID-19 by targeting viral proteinase Mpro. As a result, fifteen of twenty-five drugs were validated 
     <em>in vitro</em> to exhibit considerable inhibitory potencies towards Mpro. Among them, the most potent and representative Mpro inhibitor dipyridamole just finished its first-round clinical trials, and showed significant clinical outcomes.
     <sup>
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">17</a>
     </sup> In short, this is the first report to screen the FDA-approved database by using the FEP-ABFE approach, and this FEP-based method showed significant advantages by means of improving the hit rates and repositioning more potent leads.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">2. Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>2.1 Molecular docking</h3>
     <p id="p-7">The crystal structure of viral proteinase Mpro (PDB ID: 6LU7)
      <sup>
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">18</a>
      </sup> for COVID-19 was used for the molecule docking purpose. Based on the crystal structure, more than 2500 small molecules in the FDA-approved drug database were first screened by using molecular docking program Glide
      <sup>
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">19</a>
      </sup>. Considering Mpro being a protease, Cys145-His41/Ser144-His163 can act as the nucleophilic agent and acid that assist the hydrolysis reaction of the substrate proteins, and Gly143 and Gln166 can form hydrogen bonds with the “CO-NH-Cα-CO-NH-Cα” structure of the backbone of the substrate protein. Thus, these 6 residues were considered as the key residues of Mpro in the screening. After docking, the binding modes of all the ligands were carefully checked, and 100 molecules with specific interaction with the key residues and relatively high docking scores were selected for further FEP studies to evaluate their ABFE.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>2.2 Free energy perturbation (FEP)</h3>
     <div id="sec-5" class="subsection">
      <h4>2.2.1 Preliminary MD simulations</h4>
      <p id="p-8">All the 100 ligands selected by molecular docking were further evaluated by FEP calculations carried out in Gromacs-2019
       <sup>
        <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">20</a>,
        <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">21</a>
       </sup>. Before FEP calculations, 4 ns preliminary MD simulations were performed for each receptor-ligand complex to improve the fit of the ligand into the binding pocket. All the ligands are parameterized by the general AMBER force field (GAFF)
       <sup>
        <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">22</a>
       </sup>. Restrained electrostatic potential (RESP) charges calculation of relative ligand was performed with Gaussian 03 program
       <sup>
        <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">23</a>
       </sup> at the HF/6-31G* level. The parameters of protein were described by the AMBER FF14SB force field
       <sup>
        <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">24</a>
       </sup>. The TIP3P model
       <sup>
        <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">25</a>
       </sup> was used for water molecules. The systems were neutralized by adding counter ions (either Na
       <sup>+</sup> or Cl
       <sup>-</sup> ions). The systems were first minimized by using steepest descent method for 5000 cycles and then heated from 0 to 298 K in an NVT ensemble within 100 ps. The systems were then equilibrated in an NPT ensemble with weak restraints of 1000 kJ/mol/nm
       <sup>2</sup> for 500 ps followed by a 4 ns unconstrained production simulation. The last snapshot of the MD simulations was used for the following FEP calculations, and the trajectory of the last 2 ns was analyzed to get the parameters for adding restraints between receptors and ligands.
      </p>
     </div>
     <div id="sec-6" class="subsection">
      <h4>2.2.3 Protocol for automatically adding restraints</h4>
      <p id="p-9">Based on the preliminary MD simulations results, the FEP-ABFE calculations were carried based on the thermodynamic cycle given in 
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. As shown in the thermodynamic cycle, a restraint should be added to the receptor and ligand for each FEP calculation. The strategy of adding restraints first reported by Boresch 
       <em>et al.</em>
       <sup>
        <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">26</a>
       </sup> was used in this study, which consists of one distance, two angles, and three dihedrals harmonic potentials with a force constant of 10 kcal/mol/Å
       <sup>
        <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
       </sup> [rad
       <sup>
        <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>
       </sup>]. The contribution of the restraints to the Lig 
       <span class="inline-formula" id="inline-formula-1">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-1.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-1.gif"/>
         </noscript>
        </span>
       </span> system was calculated analytically, and the contribution of the restraint to the Rec-Lig system 
       <span class="inline-formula" id="inline-formula-2">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-2.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-2.gif"/>
         </noscript>
        </span>
       </span> was calculated by FEP. According to the strategy, three atoms of the ligand and three atoms of the receptor will be selected to add the restraint. In order to add the restraints at the equilibrium position, a program was designed to automatically detect the required parameters and select the three ligand atoms and the three receptors atoms. For ligand, the heavy atom which is closest to the geometry center was selected as the first atom; the heavy atom which is most distant from the first atom was selected as the second atom; the heavy atom forms an angle that is larger than 90 degrees with the first two atoms and most distant from the first atom is selected as the third atom. For the receptor, based on the last 2 ns trajectory of the 4 ns preliminary MD simulations, the distances, angles, dihedrals between the three ligand atoms and C
       <sub>α</sub>, C
       <sub>c</sub> (carbon of the carboxyl group) and N (N atom of the amino group) atoms of all the residues within 5 Å of the ligand were calculated along the MD trajectory. C
       <sub>α</sub>, C
       <sub>c</sub> and N atoms from the same residue with the most stable (lowest standard deviation) distances, angles, and dihedrals values were selected as the three receptor atoms. The determined mean values of distance, angles, and dihedrals were used for adding the restraints between the three ligand atoms and the three receptor atoms.
      </p>
      <div id="F1" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The thermodynamic cycle used for the FEP-ABFE calculations derived from our previous version28." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The thermodynamic cycle used for the FEP-ABFE calculations derived from our previous version&lt;;sup&gt;;28&lt;;/sup&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F1.medium.gif" width="381" height="440"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F1.medium.gif" width="381" height="440"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 1.</span>
        <p id="p-10" class="first-child">The thermodynamic cycle used for the FEP-ABFE calculations derived from our previous version
         <sup>
          <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">28</a>
         </sup>.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-7" class="subsection">
      <h4>2.2.3. ABFE calculation</h4>
      <p id="p-11">In order to calculate the ABFE value between a ligand and its receptor, the ligand electrostatic and van der Waals interactions will be decoupled. In the recently published works 
       <sup>
        <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">15</a>, 
        <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">16</a>
       </sup>, 12 λ were used for adding restraints, 10 λ were used for decoupling electrostatic interactions, and 20 λ were used for decoupling vdW interactions, respectively, which need a lot of computation resources and thus is not applicable for the virtual screening purpose.
      </p>
      <p id="p-12">In order to make it possible to use FEP-ABFE in rapid discovery of inhibitors against COVID-19, the alchemical pathway was optimized to find one that can significantly not only improve the calculation speed but also keep the accuracy. According to the FEP theory, in order to calculate the free energy difference Δ
       <em>A</em>, probability distribution of potential energy difference between adjacent λ, denoted as 
       <em>P</em>(Δ
       <em>U</em>), will be sampled. In the reported studies
       <sup>
        <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">27</a>, 
        <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">28</a>
       </sup>, 
       <em>P</em>(Δ
       <em>U</em>) ‘s are considered as Gaussian-like distributions, which is true for the steps of electrostatic interactions and vdW interactions decoupling. However, for the restraint addition steps, the 
       <em>P</em>(Δ
       <em>U</em>)’s are far from Gaussian distributions. In this study, we first derived and introduce the restrain energy distribution (RED) function which can be used to correctly describe the 
       <em>P</em>(Δ
       <em>U</em>) of the restraint addition steps. By using the automatic restrain adding program described above and fitting 
       <em>P</em>(Δ
       <em>U</em>) by the RED function, restraint energy can be accurately estimated by just using single-step perturbation (λ from 0.0 directly to 1.0), which greatly decreased the calculation resources needed for the restraint addition. For the decoupling of electrostatic and vdW interactions, the alchemical pathways which can significantly decrease the number of λ needed and keep the accuracy were also studied. After all these procedures have been done, the FEP-ABFE calculations were greatly accelerated, which made the application of FEP-ABFE for virtual screening possible.
      </p>
      <p id="p-13">After the alchemical pathway is determined, for each window, 5000 cycles of steepest descent energy minimization were first carried out, and then 100 ps simulations in the NVT ensemble along with Langevin dynamics
       <sup>
        <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">29</a>, 
        <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">30</a>
       </sup> for temperature coupling was performed to heat the system to 298 K with weak position restraints of 1000 kJ/mol/nm
       <sup>
        <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>
       </sup> applied to the receptor and the ligand heavy atoms. The simulation system was subsequently equilibrated in an NPT ensemble for 500 ps with the position restraints still applied followed by a 4 ns unconstrained production simulation. Pressure was coupled using Parrinello–Rahman pressure coupling scheme
       <sup>
        <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">31</a>
       </sup>. The LINCS constraint algorithm
       <sup>
        <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">32</a>
       </sup> was used only on H-bonds. In all simulations, the particle mesh Ewald (PME) algorithm
       <sup>
        <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">33</a>
       </sup> was used for calculation of long-range electrostatic interactions. Δ
       <em>U</em> values were sampled along the unconstrained simulation, and the free energy differences between each window were calculated by using the Bennet acceptance ratio (BAR) method
       <sup>
        <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">34</a>, 
        <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">35</a>
       </sup>.
      </p>
      <p id="p-14">Several researches
       <sup>
        <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">36</a>-
        <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">38</a>
       </sup> have reported that the FEP-ABFE method is relatively accurate for electrically neutral ligands, but when the net charge of the ligand is not 0, there will be a systematic error. Considering many of the FDA approved drugs are not electrically neutral, all the molecules evaluated via the FEP-ABFE calculations were grouped by their net charges, and in each group, the molecules with highest binding free energies were selected for further bioassay validation.
      </p>
     </div>
    </div>
    <div id="sec-8" class="subsection">
     <h3>2.3 Enzymatic assay for Mpro inhibitors</h3>
     <p id="p-15">The pGEX4T1-Mpro plasmid was constructed (Atagenix, Wuhan) and transfected the E. coli strain BL21 (Codonplus, Stratagene). GST-tagged protein was purified by GST-glutathione affinity chromatography and cleaved with FXa. The purity of recombinant protein was greater than 95% as judged by SDS–PAGE. The binding of DIP to Mpro was measured by the Biacore 8K system (GE healthcare) at 25 °C. Mpro was immobilized on a CM5 chip surface via covalent linkage to Mpro N-terminus. First, the CM5 chip was activated using 1:4 
      <em>N</em>-hydroxysuccinimide (NHS)/1-ethyl-3-(3-dimethylaminopropyl) carbodiimide (EDC) at the flow rate of 10 
      <em>µ</em>L/min for 7 min. Then, Mpro (90 
      <em>µ</em>g/mL) in 10 mM acetate buffer (pH 4.5) was passed over separate flow cells at 10 
      <em>µ</em>L/min for 3 min (1800 response units), which followed by a blocking step using ethanolamine (1 M, pH 8.5) at 10 
      <em>µ</em>L/min for 7 min. Binding studies were performed by passing 5-80 
      <em>µ</em>M of DIP over the immobilized Mpro at the flow rate of 30 
      <em>µ</em>L/min and the contact time was set to be 200 s. A sample volume of 120 
      <em>µ</em>L DIP in running buffer was injected into the flow cell and the bound ligand was washed by running buffer which contained 10 mM phosphate, 137 mM NaCl, 2.7 mM KCl, pH 7.4, and 0.5% DMSO.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>2.4 Surface plasmon resonance (SPR) between ligands and Mpro</h3>
     <p id="p-16">Surface plasmon resonance experiments were performed with a BIAcore T200 (GE Healthcare). The running buffer contained 1.05×PBS, 0.005% (vol/vol) surfactant P20, pH 7.4, and 1% DMSO. The purified DHODH, which was diluted in sodium acetate solution (pH 5.5) with a final concentration of 30 μg/mL, was immobilized on a CM5 sensor chip by amine coupling. All analyte measurements were performed at a flow rate of 30 μL/min. The analyte was diluted in the running buffer from the top concentration. Data processing and analysis were performed using BIAevaluation 1.1 software.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>2.5. Inhibition of NF-κB signaling pathway and inflammatory responses</h3>
     <p id="p-17">Luciferase reporter assay was applied to determine the function of indicated drugs in NF-κB signaling pathway. 293T cells were transfected with a mixture of luciferase reporter (firefly luciferase), TK-luc (renilla luciferase plasmid), then the cells were transfected with the active form of viral receptor RIG-I (RIG-I-CARD), or infected with Sendai virus (SeV). Luciferase activity was measured at 24 hrs after transfection or infection using a luminometer (Thermo scientific) with a dual-luciferase reporter assay system according to the manufacturer’s instructions (Promega). Data represent relative firefly luciferase activity, normalized to renilla luciferase activity. qRT-PCR and ELISA assay was applied to determine the expression of pro-inflammatory cytokines. Total RNA was extracted from the cells using TriZol reagent (Invitrogen) according to the manufacturer’s instructions. For RT-PCR analysis, cDNA was generated with HiScript II Q RT SuperMix for qPCR (+gDNA wiper) (Vazyme, R223-01) and analyzed by real-time qRT-PCR using 2 × RealStar SYBR Mixture (Genestar). All data were normalized to GAPDH expression. Primer sequences were as follows: GAPDH, sense: 50-CGGAGTCAACGGATTTGGTC-30, anti-sense: 50-GACAAGCTTCCCGTTCTCAG-30; TNF-a, sense: 50-CCAGACCAAGGTCAACCTCC-30, and anti-sense: 50-CAGACTCGGCAAAGTCGAGA-30. For measurement of cytokines, human IL-1β in cell culture supernatants was detected with an ELISA kit (BD Biosciences, No. 557953) according to the manufacturer’s protocols.</p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">3. Result and Discussion</h2>
    <div id="sec-12" class="subsection">
     <h3>3.1 The RED function accelerated the FEP-ABFE approach</h3>
     <p id="p-18">The probability distribution of energy difference between different windows 
      <em>P</em>(Δ
      <em>U</em>) for calculating 
      <span class="inline-formula" id="inline-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-3.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-3.gif"/>
        </noscript>
       </span>
      </span> can be described by the following restrain energy distribution (RED) function (
      <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Eq. 1</a>), and the derivation and detailed discussion of the RED function is given Supporting Information 
      <a id="xref-sec-19-1" class="xref-sec" href="#sec-19">(SI) section S1</a>. 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-2.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-19">The RED function can correctly describe the sampled restrain energy distribution 
      <em>P</em>(Δ
      <em>U</em>) and greatly increase the convergence for calculating 
      <span class="inline-formula" id="inline-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-4.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-4.gif"/>
        </noscript>
       </span>
      </span> As shown in 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2(a)</a>, the sampled 
      <em>P</em>(Δ
      <em>U</em>) (yellow dots) for adding restraint can be fitted quite well by the RED function (red line). By using 3 targets and 28 ligands as a test set, with the use of the automatic restrain adding program and fitting the 
      <em>P</em>(Δ
      <em>U</em>) by the RED function, the 
      <span class="inline-formula" id="inline-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-5.gif"/>
        </noscript>
       </span>
      </span> calculated by using one-step perturbation (2 λ with values of 0.0 and 1.0) has good correlation with that calculated by using the previously reported 12-λ perturbation with R
      <sup>2</sup> larger than 0.97 (results are shown in 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2(b)</a>). The energy difference between the two alchemical pathways for all the tested systems are less than 0.5 kcal/mol. Besides using one-step perturbation for the calculation of 
      <span class="inline-formula" id="inline-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-6.gif"/>
        </noscript>
       </span>
      </span>, the alchemical pathway for calculating ΔA
      <sub>annihilation</sub> is also optimized which further decreased the number of λ values needed. Detailed discussion about the correctness and rationality of the RED function are given in 
      <a id="xref-sec-20-1" class="xref-sec" href="#sec-20">SI sections S2</a> and 
      <a id="xref-sec-21-1" class="xref-sec" href="#sec-21">S3</a>; detailed results about the comparison between one-step perturbation and 12-λ perturbation are given in 
      <a id="xref-sec-20-2" class="xref-sec" href="#sec-20">SI section S2</a>; and detailed strategy for further optimizing the alchemical pathway for calculating ΔA
      <sub>annihilation</sub> are given in 
      <a id="xref-sec-22-1" class="xref-sec" href="#sec-22">SI section S4</a>.After all the above efforts, the FEP-ABFE can be accurately calculated by using just 16 λ values which need only 38 % computation resources compared with the reported FEP protocols, and the calculation is accelerated by more than 2.5 times. With such acceleration, FEP-ABFE calculation was first made possible to be applied in virtual screening to rapidly discover inhibitors against the coronavirus COVID-19.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(a) P([[#x394]]U) can be fitted by RED function quite well. The yellow dots are the corresponding sampled P([[#x394]]U), the red line is the fitted RED function, the green line is the first term in the RED function, and the blue line is the second term in the RED function. (b) The linear regression between the results of calculated from 12[[#x3BB]] perturbation and one-step perturbation (denoted as 2[[#x3BB]]), which showed quite an excellent correlation with slope [[#x223C]]1.0, interception [[#x223C]]0.0, and R2 larger than 0.97." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;(a) P([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;) can be fitted by RED function quite well. The yellow dots are the corresponding sampled P([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;), the red line is the fitted RED function, the green line is the first term in the RED function, and the blue line is the second term in the RED function. (b) The linear regression between the results of calculated from 12[[#x3BB]] perturbation and one-step perturbation (denoted as 2[[#x3BB]]), which showed quite an excellent correlation with slope [[#x223C]]1.0, interception [[#x223C]]0.0, and R&lt;;sup&gt;;2&lt;;/sup&gt;; larger than 0.97.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F2.medium.gif" width="440" height="168"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F2.medium.gif" width="440" height="168"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-20" class="first-child">(a) P(Δ
        <em>U</em>) can be fitted by RED function quite well. The yellow dots are the corresponding sampled P(Δ
        <em>U</em>), the red line is the fitted RED function, the green line is the first term in the RED function, and the blue line is the second term in the RED function. (b) The linear regression between the results of 
        <span class="inline-formula" id="inline-formula-7">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/F2/embed/inline-graphic-7.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/F2/embed/inline-graphic-7.gif"/>
          </noscript>
         </span>
        </span> calculated from 12λ perturbation and one-step perturbation (denoted as 2λ), which showed quite an excellent correlation with slope ∼1.0, interception ∼0.0, and R
        <sup>2</sup> larger than 0.97.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-13" class="subsection">
     <h3>3.2 Remarkable accuracy of FEP-ABFE calculations based on test results from 28 receptor-ligand systems</h3>
     <p id="p-21">The accuracy of the accelerated 16-λ-FEP-ABFE calculation was first tested against 3 targets with 28 receptor-ligand systems. One of the targets is BRD4 which is also used in the study of the FEP-ABFE method using 42 λ values previously reported by Aldeghi 
      <em>et al.</em>,
      <sup>
       <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">15</a>, 
       <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">16</a>
      </sup> and since FEP-ABFE calculations have systematic bias when the net charge of the ligand is not 0, all the 8 neutral ligands from Aldeghi’s work are used in this study.
      <sup>
       <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">15</a>
      </sup> Besides the target of BRD4, the other two targets include HIV-1 protease (10 electrically neutral ligands) and Human factor Xa (10 electrically neutral ligands). All the binding modes of the 28 calculated test systems are known from the protein data bank (
      <a href="http://www.rcsb.org">www.rcsb.org</a>), which ensured the correctness of the starting structures. As given in details in 
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S3</a>, the ligands are quite diverse to each other with the following properties: molecular weight from 241 to 662 Da; number of atoms from 22 to 89; number of rotatable bonds from 0 to 21; number of hydrogen bond acceptors from 1 to 7; number of hydrogen bond donors from 0 to 6; calculated log
      <em>P</em> from 1.17 to 4.85. The FEP-ABFE calculation results for all the 28 systems are summarized in 
      <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S4</a>, and the linear regression statistics between the calculation and experimental results are given in 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Figure For the HIV-1 protease and Human factor Xa, most of the predicted binding free energies ΔG
      <sub>pred</sub> are consistent with the experimental binding free energies ΔG
      <sub>exp</sub> with the average prediction errors less than 2.0 kcal/mol. For BRD4, although the calculations showed considerable systematic error with all the calculation results shifting to the negative direction, the calculation results still have a linear correlation with the experimental ones. The test results based on the 28 ligands with diverse chemical scaffolds indicate the FEP-ABFE method achieves a remarkable accuracy, which encourage us to perform further virtual screening target the FDA-approved drug database via the FEP-ABFE method.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1216434/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1216434/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1216434" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">The statistical results of the FEP-ABFE calculations for the three tested targets.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-14" class="subsection">
     <h3>3.3 FEP-ABFE screening led to drug repositioning of potent Mpro inhibitors</h3>
     <p id="p-22">After all the FDA-approved drugs were docked into the binding site of Mpro, 100 molecules which have specific interactions with the six key amino acid residues including Cys145, His41, Ser144, His163, Gly143, and Gln166, were subjected to perform further FEP-ABFE calculations. Among these 100 drugs, 49, 46, and 5 of them are electrically neutral, with negative charges, and with positive charges, respectively. Since the FEP method is known to have systematic error when the ligands are not electrically neutral, to select molecules based on FEP-ABFE results, the molecules were grouped by their charges in order to make sure that there is an error cancellation inside each group. From each group, top 20% to 40% molecules were selected based on their ABFE values and stability along MD simulations. As a result, 25 molecules were bought for further experimental activity tests. According to the bioassay results, 15 out of the 25 molecules exhibited considerable inhibitory potencies against Mpro (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Since candesartan cilextil is a prodrug, its metabolite candesartan was also proved to be an Mpro inhibitor and showed an IC
      <sub>50</sub> of 9.45 μM. Thus, totally 16 active molecules were found in this study, and their structures and activities are shown in 
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. The inhibitory curves of the most potent leads are given in 
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The linear regression models between the experimental [[#x394]]Gexp and the FEP-ABFE predicted [[#x394]]Gcal for the three targets and 28 ligands." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The linear regression models between the experimental [[#x394]]G&lt;;sub&gt;;exp&lt;;/sub&gt;; and the FEP-ABFE predicted [[#x394]]G&lt;;sub&gt;;cal&lt;;/sub&gt;; for the three targets and 28 ligands.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F3.medium.gif" width="440" height="131"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F3.medium.gif" width="440" height="131"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-23" class="first-child">The linear regression models between the experimental ΔG
        <sub>exp</sub> and the FEP-ABFE predicted ΔG
        <sub>cal</sub> for the three targets and 28 ligands.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Chemical structures and IC50 values of 16 Mpro inhibitors (six compounds in blue color previously reported by other groups5,39-40) from the FDA-approved drug database. Disulfiram worked as the positive control for the bioassay (the IC50 value of 5.72 [[#x3BC]]M in the literature)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Chemical structures and IC&lt;;sub&gt;;50&lt;;/sub&gt;; values of 16 Mpro inhibitors (six compounds in blue color previously reported by other groups&lt;;sup&gt;;5,39-40&lt;;/sup&gt;;) from the FDA-approved drug database. Disulfiram worked as the positive control for the bioassay (the IC&lt;;sub&gt;;50&lt;;/sub&gt;; value of 5.72 [[#x3BC]]M in the literature).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F4.medium.gif" width="381" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F4.medium.gif" width="381" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-24" class="first-child">Chemical structures and IC
        <sub>50</sub> values of 16 Mpro inhibitors (six compounds in blue color previously reported by other groups
        <sup>
         <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>,
         <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">39</a>-
         <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">40</a>
        </sup>) from the FDA-approved drug database. Disulfiram worked as the positive control for the bioassay (the IC
        <sub>50</sub> value of 5.72 μM in the literature).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The inhibitory curves for the most potent Mpro inhibitors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The inhibitory curves for the most potent Mpro inhibitors.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F5.medium.gif" width="440" height="171"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F5.medium.gif" width="440" height="171"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <p id="p-25" class="first-child">The inhibitory curves for the most potent Mpro inhibitors.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-26">Among these 16 active molecules, ten leads (names shown in black in 
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) are originally discovered in this study, while other six compounds including disulfiram, chloroquine, montelukast sodium, atazanavir, indinavir, and maribavir are also reported in other previous studies
      <sup>
       <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>, 
       <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">39</a>, 
       <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">40</a>
      </sup> (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Dipyridamole was identified to be the most potent Mpro inhibitor with an IC
      <sub>50</sub> of 0.55 μM, which was first discovered by this FEP-ABFE method. In addition, the SPR assay was performed to determine the binding affinity between dipyridamole and Mpro 
      <em>in vitro</em>. This has revealed that DIP bound to Mpro with an experimentally confirmed affinity of 34 µM (K
      <sub>
       <em>D,eq</em>
      </sub>) (
      <strong>
       <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>
      </strong>). In short, the relative hit rate of 60% and the ten originally discovered leads including dipyridamole indicated the significant advantage of the FEP-ABFE method in discovering biologically active compounds over traditional methods such as molecular docking and molecular dynamics simulations.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Dipyridamole suppresses Mpro activity by the SPR assay. Experimental SPR sensorgram from Biacore 8K between dipyridamole and Mpro, color points overlaid with a 1:1 Langmuir binding model kinetic fit (black). (A) Relative response from injection of DIP at 5, 10, 20, 40, 80, and 100 [[#x3BC]]M. Global analysis of the shape of the response curves yielded the parameters of on-rate (Kon = 3.20E+3), off-rate (Koff = 2.19E-1) and the ratio of Koff divided by Kon (KD = 68 [[#xB5]]M). (B)Equilibrium binding responses plotted versus dipyridamole concentration and fitted to a simple binding isotherm to yield an affinity of 34 [[#xB5]]M (K D,eq)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Dipyridamole suppresses Mpro activity by the SPR assay. Experimental SPR sensorgram from Biacore 8K between dipyridamole and Mpro, color points overlaid with a 1:1 Langmuir binding model kinetic fit (black). (A) Relative response from injection of DIP at 5, 10, 20, 40, 80, and 100 [[#x3BC]]M. Global analysis of the shape of the response curves yielded the parameters of on-rate (&lt;;em&gt;;K&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;on&lt;;/em&gt;;&lt;;/sub&gt;; = 3.20E+3), off-rate (&lt;;em&gt;;K&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;off&lt;;/em&gt;;&lt;;/sub&gt;; = 2.19E-1) and the ratio of &lt;;em&gt;;K&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;off&lt;;/em&gt;;&lt;;/sub&gt;; divided by &lt;;em&gt;;K&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;on&lt;;/em&gt;;&lt;;/sub&gt;; (&lt;;em&gt;;K&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;D&lt;;/em&gt;;&lt;;/sub&gt;; = 68 [[#xB5]]M). (B)Equilibrium binding responses plotted versus dipyridamole concentration and fitted to a simple binding isotherm to yield an affinity of 34 [[#xB5]]M (K &lt;;sub&gt;;&lt;;em&gt;;D,eq&lt;;/em&gt;;&lt;;/sub&gt;;).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F6.medium.gif" width="440" height="272"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F6.medium.gif" width="440" height="272"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <p id="p-27" class="first-child">Dipyridamole suppresses Mpro activity by the SPR assay. Experimental SPR sensorgram from Biacore 8K between dipyridamole and Mpro, color points overlaid with a 1:1 Langmuir binding model kinetic fit (black). (A) Relative response from injection of DIP at 5, 10, 20, 40, 80, and 100 μM. Global analysis of the shape of the response curves yielded the parameters of on-rate (
        <em>K</em>
        <sub>
         <em>on</em>
        </sub> = 3.20E+3), off-rate (
        <em>K</em>
        <sub>
         <em>off</em>
        </sub> = 2.19E-1) and the ratio of 
        <em>K</em>
        <sub>
         <em>off</em>
        </sub> divided by 
        <em>K</em>
        <sub>
         <em>on</em>
        </sub> (
        <em>K</em>
        <sub>
         <em>D</em>
        </sub> = 68 µM). (B)Equilibrium binding responses plotted versus dipyridamole concentration and fitted to a simple binding isotherm to yield an affinity of 34 µM (K 
        <sub>
         <em>D,eq</em>
        </sub>).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-28">The putative binding pattern between dipyridamole and Mpro is given in 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7a</a>, and as shown in this figure, hydrogen bond interactions and hydrophobic interactions are the main driving forces for the binding between dipyridamole and Mpro. The protein-ligand interaction fingerprint (PLIF) is given in 
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7b</a>, which shows the key residues (Thr25, Asn142, Gly143, Ser144, His163 and Glu166) that played key roles in the binding between the 16 active molecules and the target Mpro. Recently, dipyridamole has finished the first-round clinical trials including 31 patients with COVID-19 and showed significant clinical improvement.
      <sup>
       <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">17</a>
      </sup> The second-round clinical trials including more than 130 patients with COVID-19 are subjected to further investigation into its therapeutic use in COVID-19, particularly in the shadow of its rapid spread into the world.
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(a) Binding mode between Mpro and dipyridamole after molecular dynamics simulations. Dipyridamole is shown in yellow stick model, key amino acid residues of Mpro are shown in cyan stick model, and hydrogen-bonds are shown as red dashed lines. (b) The protein-ligand interaction fingerprint (PLIF) between the 16 active molecules and Mpro." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;(a) Binding mode between Mpro and dipyridamole after molecular dynamics simulations. Dipyridamole is shown in yellow stick model, key amino acid residues of Mpro are shown in cyan stick model, and hydrogen-bonds are shown as red dashed lines. (b) The protein-ligand interaction fingerprint (PLIF) between the 16 active molecules and Mpro.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F7.medium.gif" width="440" height="195"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F7.medium.gif" width="440" height="195"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7.</span>
       <p id="p-29" class="first-child">(a) Binding mode between Mpro and dipyridamole after molecular dynamics simulations. Dipyridamole is shown in yellow stick model, key amino acid residues of Mpro are shown in cyan stick model, and hydrogen-bonds are shown as red dashed lines. (b) The protein-ligand interaction fingerprint (PLIF) between the 16 active molecules and Mpro.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-15" class="subsection">
     <h3>3.4 The effect of dipyridamole and montelukast sodium on NF-kB signaling and inflammation activation</h3>
     <p id="p-30">It has been reported COVID-19 led to the increased amounts of pro-inflammatory cytokines in serum, which may be associated with pulmonary inflammation and lung damage in patients.
      <sup>
       <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
      </sup> During viral infection, host pattern recognition receptors (PRRs) such as RIG-I, sense viral RNA and hierarchically trigger signaling cascades, which include adaptor proteins (MAVS and TRAF), a set of well-studied serine/threonine kinases (IKK complex and TBK1/IKKi) and several transcription factors (NF-κB and IRF3/IRF7), to turn on the transcription of antiviral type I interferon (IFN) and pro-inflammatory cytokine genes, including TNFα and pro-IL-1β. Subsequently, pro-IL-1β can be further processed by active caspase-1 from inflammasome, to become mature IL-1β and initiates inflammatory responses (
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). Here we investigated the function of active molecules in virus-induced pro-inflammatory cytokines. Among the active molecules in 
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, two molecules dipyridamole and montelukast sodium, inhibited the NF-κB activation induced by SeV infection or RIG-I (CARD) (an active form of RNA viral receptor RIG-I) overexpression (
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). In addition, both dipyridamole and montelukast sodium decreased SeV-induced mRNA expression of pro-inflammatory cytokine TNFα (
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8C</a>). Interestingly, dipyridamole, but not montelukast sodium could also inhibit the activity of NLRP3 inflammasome and reduced IL-1β secretion (
      <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8D</a>). Collectively, we found both dipyridamole and montelukast sodium showed a global inhibitory function on NF-κB signaling and inflammatory responses during viral infection.
     </p>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Dipyridamole and montelukast sodium inhibit NF-[[#x3BA]]B signaling and inflammation. (A) Illustration of virus-induced activation of NF-[[#x3BA]]B and inflammasome. (B) Dipyridamole (DIP) and montelukast sodium (MON) inhibited SeV- and RIG-I-mediated activation of NF-[[#x3BA]]B signaling. 293T cells were transfected with the NF-[[#x3BA]]B luciferase reporter, together with TK-luc, followed by SeV infection and dipyridamole or montelukast sodium treatment. (C) Real-time PCR analysis of TNF[[#x3B1]] mRNA in A549 cells that were treated with dipyridamole or montelukast sodium, followed by SeV infection or TNF[[#x3B1]] treatment. (D) HEK293T cells were transiently transfected with NLRP3, ASC, pro-caspase-1, and pro-IL-1[[#x3B2]] plasmid, followed by dipyridamole or montelukast sodium treatment. Cell culture supernatants were collected 24 hr post-transfection to measure IL-1[[#x3B2]] release by ELISA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Dipyridamole and montelukast sodium inhibit NF-[[#x3BA]]B signaling and inflammation.&lt;;/span&gt;; (A) Illustration of virus-induced activation of NF-[[#x3BA]]B and inflammasome. (B) Dipyridamole (DIP) and montelukast sodium (MON) inhibited SeV- and RIG-I-mediated activation of NF-[[#x3BA]]B signaling. 293T cells were transfected with the NF-[[#x3BA]]B luciferase reporter, together with TK-luc, followed by SeV infection and dipyridamole or montelukast sodium treatment. (C) Real-time PCR analysis of &lt;;em&gt;;TNF[[#x3B1]]&lt;;/em&gt;; mRNA in A549 cells that were treated with dipyridamole or montelukast sodium, followed by SeV infection or TNF[[#x3B1]] treatment. (D) HEK293T cells were transiently transfected with NLRP3, ASC, pro-caspase-1, and pro-IL-1[[#x3B2]] plasmid, followed by dipyridamole or montelukast sodium treatment. Cell culture supernatants were collected 24 hr post-transfection to measure IL-1[[#x3B2]] release by ELISA.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F8.medium.gif" width="440" height="302"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F8.medium.gif" width="440" height="302"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 8.</span>
       <span class="caption-title">Dipyridamole and montelukast sodium inhibit NF-κB signaling and inflammation.</span>
       <p id="p-31" class="first-child">(A) Illustration of virus-induced activation of NF-κB and inflammasome. (B) Dipyridamole (DIP) and montelukast sodium (MON) inhibited SeV- and RIG-I-mediated activation of NF-κB signaling. 293T cells were transfected with the NF-κB luciferase reporter, together with TK-luc, followed by SeV infection and dipyridamole or montelukast sodium treatment. (C) Real-time PCR analysis of 
        <em>TNFα</em> mRNA in A549 cells that were treated with dipyridamole or montelukast sodium, followed by SeV infection or TNFα treatment. (D) HEK293T cells were transiently transfected with NLRP3, ASC, pro-caspase-1, and pro-IL-1β plasmid, followed by dipyridamole or montelukast sodium treatment. Cell culture supernatants were collected 24 hr post-transfection to measure IL-1β release by ELISA.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-32">Encouraged by the above results, dipyridamole has been approved for clinical studies by Dawu County People’s Hospital and the First Affiliated Hospital of Guangzhou Medical University. Clinical trial (ChiCTR2000030055) was registered. On March 08, it has finished the first-round clinical trials including 31 patients with COVID-19 and showed significant clinical outcomes. Its second round of clinical trials are on-going with over 130 enrolled patients with COVID-19.</p>
    </div>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">4. Conclusion</h2>
    <p id="p-33">In this study, an accelerated FEP-ABFE based virtual screening strategy with bioassay validation is introduced as a rapid protocol to accurately discover anti-COVID-19 proteinase Mpro inhibitors. The RED function was derived and introduced to optimize the alchemical pathway of FEP-ABFE, which greatly accelerated the calculation, and FEP-ABFE was first made possible in the application of screening a molecule database. As a result, 15 out of 25 drugs tested by bioassay exhibited considerable inhibitory affinities against Mpro. Besides candesartan cilextil’s metabolite candesartan, totally 16 active molecules were found in this study. Among them, we found that the most potent Mpro inhibitors (dipyridamole and montelukast) sodium showed a global inhibitory function on NF-κB signaling and inflammatory responses during viral infection. Dipyridamole has been approved for clinical studies by Dawu County People’s Hospital and the First Affiliated Hospital of Guangzhou Medical University. Clinical trial (ChiCTR2000030055) was registered. As a result, we has finished its first-round clinical trials including 31 patients with COVID-19 and it demonstrated markedly improved clinical outcomes. In summary, the FEP-ABFE-based method showed significant advantages by means of improving the hit rates and repositioning potent agents, which is especially suitable in prompting drug repositioning when not only against emergency situations such as the COVID-19 outbreak and virus mutations, but also against other important drug targets.</p>
   </div>
   <div class="section" id="sec-17">
    <h2 class="">Funding</h2>
    <p id="p-35">National Natural Science Foundation of China (8152204), Science Foundation of Guangzhou City (201904020023), Guangdong Provincial Key Laboratory of Construction Foundation (2017B030314030), Local Innovative and Research Teams Project of Guangdong Pearl River Talents Program (2017BT01Y093), Sun Yat-Sen University and Zhejiang University special scientific research fund for COVID-19 prevention and control, and philanthropy donation from individuals. The funders had no roles in the design and execution of the study.</p>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Supporting Information</h2>
    <div class="section" id="sec-19">
     <h3>Section S1. Derivation of the restrain energy distribution (RED) function</h3>
     <p id="p-36">Considering the situation where the restraint between the receptor and ligand is added at the equilibrium position, the interaction between receptor and the ligand consists of two parts including the force field and the restraint. Interaction comes from force field can be simplified as a harmonic potential since the ligand is near the equilibrium, and the restraint force is also a harmonic biasing force. Thus, the total interaction potential between receptor and ligand can be simplified as 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-11.gif"/>
        </noscript>
       </span>
      </span> where 
      <em>k</em>
      <sub>
       <em>total</em>
      </sub> is the apparent force constant, and (
      <em>r</em> − 
      <em>r</em>
      <sub>0</sub>) is the generalized distance between the current position and the equilibrium. When adding restraints, the Δ
      <em>U</em> between adjacent 
      <em>λ</em> will be sampled in order to calculate the free energy of the added restraints, and the Δ
      <em>U</em> can be represented by the following equation, 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-12.gif"/>
        </noscript>
       </span>
      </span> where i and i+1 means the adjacent 
      <em>i</em>th and (
      <em>i</em> + 1)th window, and 
      <em>k</em>
      <sub>
       <em>res</em>
      </sub> is the force constant of the added restraint.
     </p>
     <p id="p-37">Under the above circumstances, the probability distribution 
      <em>P</em>(Δ
      <em>U</em>
      <sub>
       <em>i</em>+1,
       <em>i</em>
      </sub>) can be represented as 
      <span class="disp-formula" id="disp-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-13.gif"/>
        </noscript>
       </span>
      </span> where Z is the partition function, 
      <em>β</em> = (
      <em>k</em>
      <sub>
       <em>B</em>
      </sub>
      <em>T</em>)
      <sup>−1</sup>, 
      <em>Ω</em>(
      <em>E</em>
      <sub>
       <em>rec</em>−
       <em>lig</em>
      </sub>) is the density of state. Since the generalized distance between the current position and the equilibrium (
      <em>r</em> − 
      <em>r</em>
      <sub>0</sub>) can be considered as a three dimensional vector, the density of state 
      <em>Ω</em>(
      <em>E</em>
      <sub>
       <em>rec</em>−
       <em>lig</em>
      </sub>) can be represented as 
      <span class="disp-formula" id="disp-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-14.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-38">Here, in 
      <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Eq. (S4)</a> we omitted 
      <span class="inline-formula" id="inline-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-8.gif"/>
        </noscript>
       </span>
      </span> which is a constant factor and won’t affect the form of the derived RED function.
     </p>
     <p id="p-39">By substituting 
      <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Eq. (S1)</a>
      <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">(S2)</a> and 
      <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">(S4)</a> to 
      <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Eq. (S3)</a>, we can get 
      <span class="disp-formula" id="disp-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-15.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-15.gif"/>
        </noscript>
       </span>
      </span> and by substituting (
      <em>r</em> − 
      <em>r</em>
      <sub>0</sub>)
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2</a>
      </sup> term in 
      <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Eq. (S5)</a> with Δ
      <em>U</em>
      <sub>
       <em>i</em>+1,
       <em>i</em>
      </sub>, representingΔ
      <em>U</em>
      <sub>
       <em>i</em>+1,
       <em>i</em>
      </sub>by Δ
      <em>U</em>, combining the terms which are constant factors, and normalizing the distribution function, the distribution for 
      <em>P</em>(Δ
      <em>U</em>) near the equilibrium has the form of 
      <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Eq. (S6)</a>
      <span class="disp-formula" id="disp-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-16.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-16.gif"/>
        </noscript>
       </span>
      </span> where a and b are two constants, and 
      <span class="inline-formula" id="inline-formula-9">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-9.gif"/>
        </noscript>
       </span>
      </span> is a factor for normalization purpose.
     </p>
     <p id="p-40">During the simulation, when some of the sampled points are far from the equilibrium, we need to add a Gaussian term to Eq. 6 to represent such situations, and the 
      <em>P</em>(Δ
      <em>U</em>) can be represented as 
      <span class="disp-formula" id="disp-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-17.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/graphic-17.gif"/>
        </noscript>
       </span>
      </span> which is the restrain energy distribution (RED) function used in this study, where a, b, c, d, h, 
      <em>μ</em>, and σ are 7 parameters to be fitted. In the RED function,
      <span class="inline-formula" id="inline-formula-10">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-10.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-11">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-11.gif"/>
        </noscript>
       </span>
      </span> are two factors used to combine the two terms and keep the function to be described only by the first term (denoted as harmonic energy term hereinafter) when Δ
      <em>U</em> is small and only by the second term (denoted as Gaussian term hereinafter) when Δ
      <em>U</em> is large. The two constants n
      <sub>1</sub> and n
      <sub>2</sub> can be set to two relatively big integers, and we chose 10 and 4 for n
      <sub>1</sub> and n
      <sub>2</sub>, respectively. We can also choose other values for n
      <sub>1</sub> and n
      <sub>2</sub> from 2 to 10, which do not affect the fitting results too much. The results of fitting 
      <em>P</em>(Δ
      <em>U</em>) by RED function is discussed in detail in the following 
      <a id="xref-sec-19-2" class="xref-sec" href="#sec-19">section (SI</a>
      <a id="xref-sec-20-3" class="xref-sec" href="#sec-20">Section S2</a>).
     </p>
    </div>
    <div class="section" id="sec-20">
     <h3>Section S2. Fitting by the RED function greatly increased the convergence of restraints steps and reduced the calculation needed in the FEP-ABFE prediction</h3>
     <p id="p-41">Although the accuracy of FEP calculation has been reported by several studies
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">1</a>, 
       <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">2</a>
      </sup>, the calculation resources needed for a single FEP-ABFE prediction is massive. Thus, in order to rapidly discover inhibitors for clinical use against the coronavirus COVID-19, it is important to increase the convergence and decrease the calculation while keep the accuracy of FEP. Normally, more than 10 λ values will calculated during the addition of restraints, such as the works recently reported by Aldeghi 
      <em>et al.</em> that 12 non-uniformly distributed λ values are used (0.0, 0.01, 0.025, 0.05, 0.075, 0.1, 0.15, 0.2, 0.3, 0.5, 0.75, 1.0) for the addition of restraints.
      <sup>
       <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">1</a>, 
       <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">2</a>
      </sup> However, with the use of the automatic restrain adding program and fitting the probability distribution of the sampled energy difference 
      <em>P</em>(Δ
      <em>U</em>) function, the 
      <em>P</em>(Δ
      <em>U</em>) can be fitted quite well, and the convergence for calculating restraint energy was greatly improved. As a result, the restraint energy can be accurately calculated by just one-step perturbation with λ values change directly from 0.0 to 1.0, and the calculation can be greatly accelerated. Using a HIV-1 protease-ligand complex (crystal structure PDB ID: 2QHY) as an example, the restraint was added by using one-step perturbation and 12-λ perturbation, respectively. The sampled 
      <em>P</em>(Δ
      <em>U</em>)’s for adding restraints are fitted by RED function, and the results are shown in 
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure S1</a>. In each picture, the yellow dots are the corresponding sampled 
      <em>P</em>(Δ
      <em>U</em>), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function. As expected, the RED function is described only by the harmonic energy term when Δ
      <em>U</em> is small and only by the Gaussian term when Δ
      <em>U</em> is large, and the sampled 
      <em>P</em>(Δ
      <em>U</em>) can be fitted well. The free energies of adding restraints are 0.912 and 0.948 kcal/mol for the 12-λ perturbation and the one-step perturbation, respectively, which means the convergence for the one-step perturbation is pretty good.
     </p>
     <div id="F9" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The sampled P([[#x394]]U)[[#x2019]]s fitted by the RED function. The yellow dots are the corresponding sampled P([[#x394]]U), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The sampled &lt;;em&gt;;P&lt;;/em&gt;;([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;)[[#x2019]]s fitted by the RED function. The yellow dots are the corresponding sampled &lt;;em&gt;;P&lt;;/em&gt;;([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F9.medium.gif" width="440" height="366"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F9.medium.gif" width="440" height="366"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure S1.</span>
       <p id="p-42" class="first-child">The sampled 
        <em>P</em>(Δ
        <em>U</em>)’s fitted by the RED function. The yellow dots are the corresponding sampled 
        <em>P</em>(Δ
        <em>U</em>), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-43">In order to further justify the applicability of one-step perturbation (2 λ with values of 0.0 and 1.0, denoted as 2 λ below) in adding restraints, three different targets and 28 ligands were tested to compare the 
      <span class="inline-formula" id="inline-formula-12">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-12.gif"/>
        </noscript>
       </span>
      </span> results calculated from 12 λ and 2 λ. The bond, angle, and dihedral parameters for the restraints are the same for 12 λ and 2 λ calculations. 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1</a> summarized the results for the 
      <span class="inline-formula" id="inline-formula-13">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/embed/inline-graphic-13.gif"/>
        </noscript>
       </span>
      </span>, and the calculation results revealed that the single-step perturbation have good convergence with the energy difference for all the tested systems less than 0.5 kcal/mol. Thus, by using automatic restrain adding program and fitting the sampled 
      <em>P</em>(Δ
      <em>U</em>) by RED function, the calculation resources needed for adding restraints can be greatly decreased while keeping the accuracy.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1216426/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1216426/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1216426" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S1.</span>
       <p id="p-44" class="first-child">The details of free energy changes for the complex in the annihilation for the traditional 42-window schemes and the 16-window schemes 
        <sup>a</sup>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div class="section" id="sec-21">
     <h3>Section S3. RED function can also correctly describe the restrain energy distribution even if the restrain is not added to the equilibrium position</h3>
     <p id="p-45">Although the automatic restrain adding program can make sure that the parameters chose for the restrain are close to the equilibrium, it is still important to have an understanding about the behavior of the probability distribution of Δ
      <em>U</em> when the parameters are not at the equilibrium. As an example, we determined the restraint parameters for 2QHY by using the original crystal structure instead by using the pre-equilibrated trajectory, and the resulted 
      <em>P</em>(Δ
      <em>U</em>)′
      <em>s</em> are shown in 
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S2</a>. As can be seen, the probability distributions for the initial several windows showed distributions with “double-peak” shapes. In theory, the “double-peak” shape distribution 
      <em>P</em>(Δ
      <em>U</em>) will happen when relative position between the receptor and ligand, which we use to determine the ligand restraints addition parameters, is far from the equilibrium state, and at the same time, the harmonic potentials used to apply the restraint are not strong enough. According to FEP theory, the probability distribution near the lower tail of 
      <em>P</em>(Δ
      <em>U</em>) will greatly influence the accuracy of Δ
      <em>A</em> calculation. The “double-peak” shape of the probability distribution is far from the equilibrium, and the distribution pattern near the lower tail varies from time to time during the simulation, which will result in a very unstable free energy estimation. Thus, to deal with the instability in energy calculation, in the reported studies
      <sup>
       <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">1</a>, 
       <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">2</a>
      </sup>, the restraints are added with more than 10 λ values, and the first several λ are selected rather close to each other (0.0, 0.01, 0.025, 0.05, 0.075, 0.1). Although the purpose of the RED function is to describe the 
      <em>P</em>(Δ
      <em>U</em>) when the restraints are added near the equilibrium state, the RED function can also describe the 
      <em>P</em>(Δ
      <em>U</em>) well even when restraints are not added near the equilibrium. Using the forward perturbation from 0.00 to 0.01 as an example, which is shown in 
      <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure S2</a>, when the added restrain is relatively weak and not at the equilibrium position, the peak on the right which is described by the Gaussian term of the RED function is caused by the ligand occupying the equilibrium position, and the peak on the left is caused by the added restraints which is described by the harmonic energy term of the RED function. The RED function can describe the 
      <em>P</em>(Δ
      <em>U</em>) of adding restraints quite well no matter whether the restraints are added near the equilibrium, which further proved the correctness and rationality of the function. The RED function can be used to further optimize and give a deeper understanding of the FEP-ABFE calculation in the future studies.
     </p>
     <div id="F10" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The sampled P([[#x394]]U)[[#x2019]]s fitted by the RED function when the restraints are not added near the equilibrium position. The yellow dots are the corresponding sampled P([[#x394]]U), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1050915851" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The sampled &lt;;em&gt;;P&lt;;/em&gt;;([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;)[[#x2019]]s fitted by the RED function when the restraints are not added near the equilibrium position. The yellow dots are the corresponding sampled &lt;;em&gt;;P&lt;;/em&gt;;([[#x394]]&lt;;em&gt;;U&lt;;/em&gt;;), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F10.medium.gif" width="440" height="219"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F10.medium.gif" width="440" height="219"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.004580/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure S2.</span>
       <p id="p-46" class="first-child">The sampled 
        <em>P</em>(Δ
        <em>U</em>)’s fitted by the RED function when the restraints are not added near the equilibrium position. The yellow dots are the corresponding sampled 
        <em>P</em>(Δ
        <em>U</em>), the green line is the harmonic energy term in the RED function, and the blue line is the Gaussian term in the RED function.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div class="section" id="sec-22">
     <h3>Section S4. FEP calculations can be accelerated by more than 2.5 times while keeping the accuracy with appropriate selection of λ values</h3>
     <p id="p-47">In order to further accelerate the calculation, we studied the effect of decreasing the number of λ values for both charge and vdW interactions decoupling by using 4 different receptor-ligand systems. Corresponding crystal structures (PDB codes: 3SVG, 3U5L, 3U5J, 4HBV) were used in order to make sure that the binding modes between the receptor and ligands are correct. For all the 4 crystal structures, the ABFE was first calculated by using 42 λ (12 λ for restrain, 10 λ for charge, 20 λ for vdW) as reported by the recently published works
      <sup>
       <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">1</a>, 
       <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">2</a>
      </sup>, and then the number of λ values were decreased for the decoupling of both charge and vdW interactions. As can be seen in 
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S2</a>, by using 16 λ values in total (2 λ for restrain with fitting by RED function, 5 λ for charge, and 9 λ for vdW) the energy calculation results are quite close to that calculated by using 42 λ values for all the 4 systems tested (all the energy differences are within 1 kcal/mol). Compared with 42 λ, FEP-ABFE calculation using just 16 λ values need only 38 % computation resources, and the calculation is accelerated by more than 2.5 times. With such acceleration, FEP-ABFE calculation is now possible to be applied in virtual screening to rapidly discover inhibitors against the coronavirus COVID-19.
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1216424/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1216424/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1216424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S2.</span>
       <p id="p-48" class="first-child">Comparison of 
        <span class="inline-formula" id="inline-formula-14">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/T3/embed/inline-graphic-14.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/25/2020.03.23.004580/T3/embed/inline-graphic-14.gif"/>
          </noscript>
         </span>
        </span> between 12 λ and 2 λ calculations.
        <sup>a</sup>
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1216421/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1216421/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1216421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S3.</span>
       <p id="p-49" class="first-child">Physical chemical properties of the ligands. PDB is the PDB code the ligand belongs to. MW is the molecular weight in Daltons; Netq is the net charge; NROT is the number of rotatable bonds; HBA is the number of hydrogen bond acceptors; HBD is the number of hydrogen bond donors; cLogP is the calculated octanol/water partition coefficient (computed with XLOGP3). All the properties are gained from the PDBbind-CN Database.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1216431/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1216431/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1216431" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S4.</span>
       <p id="p-50" class="first-child">Summary of free energy calculation results (kcal/mol) among HIV-1 protease, Human factor Xa, and BRD4.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-34">We cordially acknowledge Tencent Cloud and National Supercomputing centers in Shenzhen and Guangzhou for providing HPC resources for virtual screening and free energy perturbation calculations. We also acknowledge the Beijing Super Cloud Computing Center (BSCC) for providing HPC resources that have contributed to the research results reported within this paper. URL: 
     <a href="http://www.blsc.cn/">http://www.blsc.cn/</a>.
    </p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dong</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Qu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gong</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Qiu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-article-title">Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yin</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Holshue</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">DeBolt</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindquist</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lofy</span>, 
          <span class="cit-name-given-names">K. H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wiesman</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Spitters</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ericson</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilkerson</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tural</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Diaz</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cohn</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fox</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Patel</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gerber</span>, 
          <span class="cit-name-given-names">S. I.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindstrom</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pallansch</span>, 
          <span class="cit-name-given-names">M. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Weldon</span>, 
          <span class="cit-name-given-names">W. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggs</span>, 
          <span class="cit-name-given-names">H. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Uyeki</span>, 
          <span class="cit-name-given-names">T. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pillai</span>, 
          <span class="cit-name-given-names">S. K.</span>
         </span>; 
         <span class="cit-auth cit-collab">Washington State -nCo, V. C. I. T</span>., 
         <span class="cit-article-title">First Case of 2019 Novel Coronavirus in the United States</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.4" data-doi="10.1016/j.coviro.2014.06.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Adedeji</span>, 
          <span class="cit-name-given-names">A. O.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sarafianos</span>, 
          <span class="cit-name-given-names">S. G.</span>
         </span>, 
         <span class="cit-article-title">Antiviral drugs specific for coronaviruses in preclinical development</span>. 
         <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">45</span>–
         <span class="cit-lpage">53</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D8%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2014.06.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.coviro.2014.06.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Duan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure of Mpro from COVID-19 virus and discovery of its inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>
         <span class="cit-article-title">Therapeutic drugs targeting 2019-nCoV main protease by high-throughput screening</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 6. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xiong</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-article-title">Novel and potent inhibitors targeting DHODH, a rate-limiting enzyme in de novo pyrimidine biosynthesis, are broad-spectrum antiviral against RNA viruses including newly emerged coronavirus SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 7. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.8" data-doi="10.1021/ja512751q">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pierce</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Krilov</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lupyan</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Robinson</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dahlgren</span>, 
          <span class="cit-name-given-names">M. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Greenwood</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Romero</span>, 
          <span class="cit-name-given-names">D. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Masse</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Knight</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Steinbrecher</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Beuming</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Damm</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Harder</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sherman</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Brewer</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wester</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Murcko</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Frye</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Farid</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mobley</span>, 
          <span class="cit-name-given-names">D. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jorgensen</span>, 
          <span class="cit-name-given-names">W. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berne</span>, 
          <span class="cit-name-given-names">B. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Abel</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span>. 
         <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. 
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">137</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">2695</span>–
         <span class="cit-lpage">703</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc%26rft.volume%253D137%26rft.spage%253D2695%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja512751q%26rft_id%253Dinfo%253Apmid%252F25625324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ja512751q&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25625324&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.9" data-doi="10.1021/acs.jcim.5b00538.">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Steinbrecher</span>, 
          <span class="cit-name-given-names">T. B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dahlgren</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cappel</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Krilov</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Abel</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sherman</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-article-title">Accurate Binding Free Energy Predictions in Fragment Optimization</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical information and modeling</abbr>
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">55</span> (
         <span class="cit-issue">11</span>), 
         <span class="cit-fpage">2411</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.volume%253D55%26rft.spage%253D2411%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jcim.5b00538.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/acs.jcim.5b00538.&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kollman</span>, 
          <span class="cit-name-given-names">P. A.</span>
         </span>, 
         <span class="cit-article-title">Determination of solvation free energy using molecular dynamics with solute Cartesian mapping: An application to the solvation of 18-crown-6</span>. 
         <abbr class="cit-jnl-abbrev">J. Chem. Phys</abbr>. 
         <span class="cit-pub-date">1992</span>, 
         <span class="cit-vol">97</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">5108</span>–
         <span class="cit-lpage">5112</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D97%26rft.spage%253D5108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.11" data-doi="10.1002/(SICI)1096-987X(199705)18:7&lt;;902::AID-JCC4&gt;;3.0.CO;2-V">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Radmer</span>, 
          <span class="cit-name-given-names">R. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kollman</span>, 
          <span class="cit-name-given-names">P. A.</span>
         </span>, 
         <span class="cit-article-title">Free energy calculation methods: a theoretical and empirical comparison of numerical errors and a new method qualitative estimates of free energy changes</span>. 
         <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. 
         <span class="cit-pub-date">1997</span>, 
         <span class="cit-vol">18</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">902</span>–
         <span class="cit-lpage">919</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BChem%26rft.volume%253D18%26rft.spage%253D902%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-987X%2528199705%252918%253A7%253C902%253A%253AAID-JCC4%253E3.0.CO%253B2-V%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-987X(199705)18:7&lt;;902::AID-JCC4&gt;;3.0.CO;2-V&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Briggs</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Marrone</span>, 
          <span class="cit-name-given-names">T. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">McCammon</span>, 
          <span class="cit-name-given-names">J. A.</span>
         </span>, 
         <span class="cit-article-title">Computational science new horizons and relevance to pharmaceutical design</span>. 
         <abbr class="cit-jnl-abbrev">Trends Cardiovas. Med</abbr>. 
         <span class="cit-pub-date">1996</span>, 
         <span class="cit-vol">6</span> (
         <span class="cit-issue">6</span>), 
         <span class="cit-fpage">198</span>–
         <span class="cit-lpage">203</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcardiovascular%2Bmedicine%26rft.stitle%253DTrends%2BCardiovasc%2BMed%26rft.aulast%253DBriggs%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D198%26rft.epage%253D203%26rft.atitle%253DComputational%2Bscience%2Bnew%2Bhorizons%2Band%2Brelevance%2Bto%2Bpharmaceutical%2Bdesign.%26rft_id%253Dinfo%253Apmid%252F21232297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=21232297&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">J. T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hamilton</span>, 
          <span class="cit-name-given-names">A. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bailey</span>, 
          <span class="cit-name-given-names">C. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Domoal</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>, 
          <span class="cit-name-given-names">K. S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jorgensen</span>, 
          <span class="cit-name-given-names">W. L.</span>
         </span>, 
         <span class="cit-article-title">FEP-guided selection of bicyclic heterocycles in lead optimization for non-nucleoside inhibitors of HIV-1 reverse transcriptase</span>. 
         <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. 
         <span class="cit-pub-date">2006</span>, 
         <span class="cit-vol">128</span> (
         <span class="cit-issue">48</span>), 
         <span class="cit-fpage">15372</span>–
         <span class="cit-lpage">15373</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DKim%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D128%26rft.issue%253D48%26rft.spage%253D15372%26rft.epage%253D15373%26rft.atitle%253DFEP-guided%2Bselection%2Bof%2Bbicyclic%2Bheterocycles%2Bin%2Blead%2Boptimization%2Bfor%2Bnon-nucleoside%2Binhibitors%2Bof%2BHIV-1%2Breverse%2Btranscriptase.%26rft_id%253Dinfo%253Apmid%252F17131993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=17131993&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jorgensen</span>, 
          <span class="cit-name-given-names">W. L.</span>
         </span>, 
         <span class="cit-article-title">Computer-aided discovery of anti-HIV agents</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">24</span> (
         <span class="cit-issue">20</span>), 
         <span class="cit-fpage">4768</span>–
         <span class="cit-lpage">4778</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D24%26rft.spage%253D4768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 14. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.15" data-doi="10.1021/acs.accounts.7b00083">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Abel</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Harder</span>, 
          <span class="cit-name-given-names">E. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berne</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>, 
         <span class="cit-article-title">Advancing Drug Discovery through Enhanced Free Energy Calculations</span>. 
         <abbr class="cit-jnl-abbrev">Accounts of chemical research</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">50</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">1625</span>–
         <span class="cit-lpage">1632</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAccounts%2Bof%2Bchemical%2Bresearch%26rft.volume%253D50%26rft.spage%253D1625%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.accounts.7b00083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/acs.accounts.7b00083&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 15. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aldeghi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heifetz</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bodkin</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Knapp</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggin</span>, 
          <span class="cit-name-given-names">P. C.</span>
         </span>, 
         <span class="cit-article-title">Accurate calculation of the absolute free energy of binding for drug molecules</span>. 
         <abbr class="cit-jnl-abbrev">Chemical science</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">7</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">207</span>–
         <span class="cit-lpage">218</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemical%2Bscience%26rft.volume%253D7%26rft.spage%253D207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 16. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.17" data-doi="10.1021/jacs.6b11467">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aldeghi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heifetz</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bodkin</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Knapp</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggin</span>, 
          <span class="cit-name-given-names">P. C.</span>
         </span>, 
         <span class="cit-article-title">Predictions of ligand selectivity from absolute binding free energy calculations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of the American Chemical Society</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">139</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">946</span>–
         <span class="cit-lpage">957</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.volume%253D139%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjacs.6b11467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jacs.6b11467&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 17. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">Y.-y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction</span>. 
         <abbr class="cit-jnl-abbrev">medRxiv</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 18. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure of Mpro from COVID-19 virus and discovery of its inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 19. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.20" data-doi="10.1021/jm030644s">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Halgren</span>, 
          <span class="cit-name-given-names">T. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Murphy</span>, 
          <span class="cit-name-given-names">R. B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Beard</span>, 
          <span class="cit-name-given-names">H. S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Frye</span>, 
          <span class="cit-name-given-names">L. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pollard</span>, 
          <span class="cit-name-given-names">W. T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Banks</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span>, 
         <span class="cit-article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-pub-date">2004</span>, 
         <span class="cit-vol">47</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">1750</span>–
         <span class="cit-lpage">1759</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DHalgren%26rft.auinit1%253DT.%2BA.%26rft.volume%253D47%26rft.issue%253D7%26rft.spage%253D1750%26rft.epage%253D1759%26rft.atitle%253DGlide%253A%2Ba%2Bnew%2Bapproach%2Bfor%2Brapid%252C%2Baccurate%2Bdocking%2Band%2Bscoring.%2B2.%2BEnrichment%2Bfactors%2Bin%2Bdatabase%2Bscreening.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm030644s%26rft_id%253Dinfo%253Apmid%252F15027866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm030644s&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15027866&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220317600020&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 20. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hess</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kutzner</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Van Der Spoel</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindahl</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-article-title">GROMACS 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical theory computation</abbr>
         <span class="cit-pub-date">2008</span>, 
         <span class="cit-vol">4</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">435</span>–
         <span class="cit-lpage">447</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Btheory%2Bcomputation%26rft.volume%253D4%26rft.spage%253D435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 21. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.22" data-doi="10.1093/bioinformatics/btt055">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pronk</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Páll</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schulz</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Larsson</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bjelkmar</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Apostolov</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shirts</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kasson</span>, 
          <span class="cit-name-given-names">P. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Spoel</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-article-title">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span>. 
         <abbr class="cit-jnl-abbrev">J Bioinformatics</abbr>
         <span class="cit-pub-date">2013</span>, 
         <span class="cit-vol">29</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">845</span>–
         <span class="cit-lpage">854</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt055%26rft_id%253Dinfo%253Apmid%252F23407358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt055&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23407358&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000316695700004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 22. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.23" data-doi="10.1002/jcc.20035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolf</span>, 
          <span class="cit-name-given-names">R. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Caldwell</span>, 
          <span class="cit-name-given-names">J. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kollman</span>, 
          <span class="cit-name-given-names">P. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Case</span>, 
          <span class="cit-name-given-names">D. A.</span>
         </span>, 
         <span class="cit-article-title">Development and testing of a general amber force field</span>. 
         <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr>
         <span class="cit-pub-date">2004</span>, 
         <span class="cit-vol">25</span> (
         <span class="cit-issue">9</span>), 
         <span class="cit-fpage">1157</span>–
         <span class="cit-lpage">1174</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D9%26rft.spage%253D1157%26rft.epage%253D1174%26rft.atitle%253DDevelopment%2Band%2Btesting%2Bof%2Ba%2Bgeneral%2Bamber%2Bforce%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20035%26rft_id%253Dinfo%253Apmid%252F15116359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jcc.20035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15116359&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221624200007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 23. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.03.23.004580v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Frisch</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Trucks</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlegel</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Scuseria</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Robb</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheeseman</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Montgomery</span>
          <span class="cit-name-suffix">Jr</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Vreven</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kudin</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Burant</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-source">Gaussian 03, revision C. 02</span>; 
         <span class="cit-publ-name">Gaussian, Inc</span>.: 
         <span class="cit-publ-loc">Wallingford, CT, 2004</span>. 
         <span class="cit-pub-date">2013</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 24. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Case</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Babin</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berryman</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Betz</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cai</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cerutti</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheatham</span>
          <span class="cit-name-suffix">III</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Darden</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Duke</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gohlke</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">The FF14SB force field</span>. 
         <abbr class="cit-jnl-abbrev">Amber</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">29</span>–
         <span class="cit-lpage">31</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAmber%26rft.volume%253D14%26rft.spage%253D29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 25. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.26" data-doi="10.1063/1.445869">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jorgensen</span>, 
          <span class="cit-name-given-names">W. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chandrasekhar</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Madura</span>, 
          <span class="cit-name-given-names">J. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Impey</span>, 
          <span class="cit-name-given-names">R. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Klein</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>, 
         <span class="cit-article-title">Comparison of simple potential functions for simulating liquid water</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of chemical physics</abbr>
         <span class="cit-pub-date">1983</span>, 
         <span class="cit-vol">79</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">926</span>–
         <span class="cit-lpage">935</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bchemical%2Bphysics%26rft.volume%253D79%26rft.spage%253D926%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.445869%26rft_id%253Dinfo%253Apmid%252F24341749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1063/1.445869&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24341749&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1983QZ31500046&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 26. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Boresch</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tettinger</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Leitgeb</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Karplus</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-article-title">Absolute binding free energies: a quantitative approach for their calculation</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of Physical Chemistry B</abbr>
         <span class="cit-pub-date">2003</span>, 
         <span class="cit-vol">107</span> (
         <span class="cit-issue">35</span>), 
         <span class="cit-fpage">9535</span>–
         <span class="cit-lpage">9551</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysical%2BChemistry%2BB%26rft.volume%253D107%26rft.spage%253D9535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 27. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.28" data-doi="10.1021/ja971148u">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hummer</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pratt</span>, 
          <span class="cit-name-given-names">L. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">García</span>, 
          <span class="cit-name-given-names">A. E.</span>
         </span>, 
         <span class="cit-article-title">Multistate Gaussian model for electrostatic solvation fr ee energies</span>. 
         <abbr class="cit-jnl-abbrev">Journal of the American Chemical Society</abbr>
         <span class="cit-pub-date">1997</span>, 
         <span class="cit-vol">119</span> (
         <span class="cit-issue">36</span>), 
         <span class="cit-fpage">8523</span>–
         <span class="cit-lpage">8527</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.volume%253D119%26rft.spage%253D8523%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja971148u%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ja971148u&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XV70000015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 28. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>, 
          <span class="cit-name-given-names">C.-G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">H.-B.</span>
         </span>, 
         <span class="cit-article-title">Absolute binding free energy calculation and design of a subnanomolar inhibitor of phosphodiesterase-10</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-pub-date">2019</span>, 
         <span class="cit-vol">62</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">2099</span>–
         <span class="cit-lpage">2111</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.volume%253D62%26rft.spage%253D2099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 29. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Van Gunsteren</span>, 
          <span class="cit-name-given-names">W. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berendsen</span>, 
          <span class="cit-name-given-names">H. J.</span>
         </span>, 
         <span class="cit-article-title">A leap-frog algorithm for stochastic dynamics</span>. 
         <abbr class="cit-jnl-abbrev">J Molecular Simulation</abbr>
         <span class="cit-pub-date">1988</span>, 
         <span class="cit-vol">1</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">173</span>–
         <span class="cit-lpage">185</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMolecular%2BSimulation%26rft.volume%253D1%26rft.spage%253D173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 30. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Goga</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rzepiela</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">De Vries</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Marrink</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berendsen</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">Efficient algorithms for Langevin and DPD dynamics</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical theory computation</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">8</span> (
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">3637</span>–
         <span class="cit-lpage">3649</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Btheory%2Bcomputation%26rft.volume%253D8%26rft.spage%253D3637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 31. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.32" data-doi="10.1063/1.328693">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Parrinello</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rahman</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-article-title">Polymorphic transitions in single crystals: A new molecular dynamics method</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Applied physics</abbr>
         <span class="cit-pub-date">1981</span>, 
         <span class="cit-vol">52</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">7182</span>–
         <span class="cit-lpage">7190</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2Bphysics%26rft.volume%253D52%26rft.spage%253D7182%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.328693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1063/1.328693&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 32. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.33" data-doi="10.1002/(SICI)1096-987X(199709)18:12&lt;;1463::AID-JCC4&gt;;3.3.CO;2-L">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hess</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bekker</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berendsen</span>, 
          <span class="cit-name-given-names">H. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fraaije</span>, 
          <span class="cit-name-given-names">J. G.</span>
         </span>, 
         <span class="cit-article-title">LINCS: a linear constraint solver for molecular simulations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr>
         <span class="cit-pub-date">1997</span>, 
         <span class="cit-vol">18</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">1463</span>–
         <span class="cit-lpage">1472</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.volume%253D18%26rft.spage%253D1463%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.3.CO%253B2-L%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-987X(199709)18:12&lt;;1463::AID-JCC4&gt;;3.3.CO;2-L&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XT81100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 33. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.34" data-doi="10.1063/1.470117">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Essmann</span>, 
          <span class="cit-name-given-names">U.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perera</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Berkowitz</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Darden</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pedersen</span>, 
          <span class="cit-name-given-names">L. G.</span>
         </span>, 
         <span class="cit-article-title">A smooth particle mesh Ewald method</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of chemical physics</abbr>
         <span class="cit-pub-date">1995</span>, 
         <span class="cit-vol">103</span> (
         <span class="cit-issue">19</span>), 
         <span class="cit-fpage">8577</span>–
         <span class="cit-lpage">8593</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bchemical%2Bphysics%26rft.volume%253D103%26rft.spage%253D8577%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.470117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1063/1.470117&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1995TE36400026&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 34. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.35" data-doi="10.1016/0021-9991(76)90078-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bennett</span>, 
          <span class="cit-name-given-names">C. H.</span>
         </span>, 
         <span class="cit-article-title">Efficient estimation of free energy differences from Monte Carlo data</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Computational Physics</abbr>
         <span class="cit-pub-date">1976</span>, 
         <span class="cit-vol">22</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">245</span>–
         <span class="cit-lpage">268</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComputational%2BPhysics%26rft.volume%253D22%26rft.spage%253D245%26rft.atitle%253DEFFICIENT%2BESTIMATION%2BOF%2BFREE-ENERGY%2BDIFFERENCES%2BFROM%2BMONTE-CARLO%2BDATA%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0021-9991%252876%252990078-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0021-9991(76)90078-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 35. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.36" data-doi="10.1063/1.2978177">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shirts</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chodera</span>, 
          <span class="cit-name-given-names">J. D.</span>
         </span>, 
         <span class="cit-article-title">Statistically optimal analysis of samples from multiple equilibrium states</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of chemical physics</abbr>
         <span class="cit-pub-date">2008</span>, 
         <span class="cit-vol">129</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">124105</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BChemical%2BPhysics%26rft.stitle%253DThe%2BJournal%2Bof%2BChemical%2BPhysics%26rft.aulast%253DShirts%26rft.auinit1%253DM.%2BR.%26rft.volume%253D129%26rft.issue%253D12%26rft.spage%253D124105%26rft.epage%253D124105%26rft.atitle%253DStatistically%2Boptimal%2Banalysis%2Bof%2Bsamples%2Bfrom%2Bmultiple%2Bequilibrium%2Bstates.%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.2978177%26rft_id%253Dinfo%253Apmid%252F19045004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1063/1.2978177&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19045004&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 36. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.37" data-doi="10.1529/biophysj.106.084301">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Roux</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span>. 
         <abbr class="cit-jnl-abbrev">Biophysical journal</abbr>
         <span class="cit-pub-date">2006</span>, 
         <span class="cit-vol">91</span> (
         <span class="cit-issue">8</span>), 
         <span class="cit-fpage">2798</span>–
         <span class="cit-lpage">2814</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D2798%26rft.epage%253D2814%26rft.atitle%253DAbsolute%2Bbinding%2Bfree%2Benergy%2Bcalculations%2Busing%2Bmolecular%2Bdynamics%2Bsimulations%2Bwith%2Brestraining%2Bpotentials.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.106.084301%26rft_id%253Dinfo%253Apmid%252F16844742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1529/biophysj.106.084301&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16844742&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240700700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004580v1.38" data-doi="10.1063/1.1999637">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fujitani</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tanida</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ito</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jayachandran</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Snow</span>, 
          <span class="cit-name-given-names">C. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shirts</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sorin</span>, 
          <span class="cit-name-given-names">E. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pande</span>, 
          <span class="cit-name-given-names">V. S.</span>
         </span>, 
         <span class="cit-article-title">Direct calculation of the binding free energies of FKBP ligands</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of chemical physics</abbr>
         <span class="cit-pub-date">2005</span>, 
         <span class="cit-vol">123</span> (
         <span class="cit-issue">8</span>), 
         <span class="cit-fpage">084108</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BChemical%2BPhysics%26rft.stitle%253DThe%2BJournal%2Bof%2BChemical%2BPhysics%26rft.aulast%253DFujitani%26rft.auinit1%253DH.%26rft.volume%253D123%26rft.issue%253D8%26rft.spage%253D084108%26rft.epage%253D084108%26rft.atitle%253DDirect%2Bcalculation%2Bof%2Bthe%2Bbinding%2Bfree%2Benergies%2Bof%2BFKBP%2Bligands.%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.1999637%26rft_id%253Dinfo%253Apmid%252F16164283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1063/1.1999637&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16164283&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 38. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lapelosa</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gallicchio</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Levy</span>, 
          <span class="cit-name-given-names">R. M.</span>
         </span>, 
         <span class="cit-article-title">Conformational transitions and convergence of absolute binding free energy calculations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical theory computation</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">8</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">47</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Btheory%2Bcomputation%26rft.volume%253D8%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 39. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.40" data-doi="10.1038/s41422-020-0282-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhong</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>. 
         <abbr class="cit-jnl-abbrev">Cell research</abbr>
         <span class="cit-pub-date">2020</span>, 
         <span class="cit-vol">30</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">269</span>–
         <span class="cit-lpage">271</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.volume%253D30%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41422-020-0282-0%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41422-020-0282-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.004580.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 40. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dong</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">J. J. D. D.</span>
         </span>; 
         <span class="cit-article-title">Therapeutics, Discovering drugs to treat coronavirus disease 2019 (COVID-19)</span>. 
         <abbr class="cit-jnl-abbrev">Drug Discoveries &amp;; Therapeutics</abbr>
         <span class="cit-pub-date">2020</span>, 
         <span class="cit-vol">14</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">58</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDrug%2BDiscoveries%2B%2526%2BTherapeutics%26rft.volume%253D14%26rft.spage%253D58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <h2 class="">References for SI</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 1. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.42">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aldeghi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heifetz</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bodkin</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Knapp</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggin</span>, 
          <span class="cit-name-given-names">P. C.</span>
         </span>, 
         <span class="cit-article-title">Accurate calculation of the absolute free energy of binding for drug molecules</span>. 
         <abbr class="cit-jnl-abbrev">Chemical Science</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">7</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">207</span>–
         <span class="cit-lpage">218</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemical%2BScience%26rft.volume%253D7%26rft.spage%253D207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 2. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.004580v1.43" data-doi="10.1021/jacs.6b11467">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aldeghi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heifetz</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bodkin</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Knapp</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Biggin</span>, 
          <span class="cit-name-given-names">P. C.</span>
         </span>, 
         <span class="cit-article-title">Predictions of ligand selectivity from absolute binding free energy calculations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of the American Chemical Society</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">139</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">946</span>–
         <span class="cit-lpage">957</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.volume%253D139%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjacs.6b11467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jacs.6b11467&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
