<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/11/2020.02.10.942847.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.10.942847v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="ΦX174 Attenuation by Whole Genome Codon Deoptimization" />
<meta name="DC.Identifier" content="10.1101/2020.02.10.942847" />
<meta name="DC.Date" content="2020-02-11" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence." />
<meta name="DC.Contributor" content="James T. Van Leuven" />
<meta name="DC.Contributor" content="Martina M. Ederer" />
<meta name="DC.Contributor" content="Katelyn Burleigh" />
<meta name="DC.Contributor" content="LuAnn Scott" />
<meta name="DC.Contributor" content="Randall A. Hughes" />
<meta name="DC.Contributor" content="Vlad Codrea" />
<meta name="DC.Contributor" content="Andrew D. Ellington" />
<meta name="DC.Contributor" content="Holly Wichman" />
<meta name="DC.Contributor" content="Craig Miller" />
<meta name="article:published_time" content="2020-02-11" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="ΦX174 Attenuation by Whole Genome Codon Deoptimization" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its &lt;i&gt;E. coli&lt;/i&gt; host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.10.942847v1" />
<meta name="citation_id" content="2020.02.10.942847v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.10.942847" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.10.942847" />
<meta name="citation_author" content="James T. Van Leuven" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2730-0287" />
<meta name="citation_author" content="Martina M. Ederer" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author" content="Katelyn Burleigh" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author_institution" content="Seattle Children’s Research Institute" />
<meta name="citation_author" content="LuAnn Scott" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author" content="Randall A. Hughes" />
<meta name="citation_author_institution" content="Applied Research Laboratories, University of Texas" />
<meta name="citation_author_institution" content="Biotechnology Branch, CCDC US Army Research Laboratory" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4665-0335" />
<meta name="citation_author" content="Vlad Codrea" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, University of Texas" />
<meta name="citation_author" content="Andrew D. Ellington" />
<meta name="citation_author_institution" content="Applied Research Laboratories, University of Texas" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, University of Texas" />
<meta name="citation_author" content="Holly Wichman" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author" content="Craig Miller" />
<meta name="citation_author_institution" content="Department of Biological Science, University of Idaho" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author_email" content="crmiller@uidaho.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7920-4994" />
<meta name="citation_reference" content="Plotkin JB, Kudla G. 2010. Synonymous but not the same: the causes and consequences of codon bias. Nat Rev Genet 12:32–42." />
<meta name="citation_reference" content="Bulmer M. 1991. The selection-mutation-drift theory of synonymous codon usage. Genetics 129:897–907." />
<meta name="citation_reference" content="Long H, Sung W, Kucukyildirim S, Williams E, Miller SF, Guo W, Patterson C, Gregory C, Strauss C, Stone C, Berne C, Kysela D, Shoemaker WR, Muscarella ME, Luo H, Lennon JT, Brun YV, Lynch M. 2018. Evolutionary determinants of genome-wide nucleotide composition. Nat Ecol Evol 1." />
<meta name="citation_reference" content="Kelkar YD, Phillips DS, Ochman H. 2015. Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes. G3 Genes Genomes Genet 5:1247–1252." />
<meta name="citation_reference" content="Raghavan R, Kelkar YD, Ochman H. 2012. A selective force favoring increased G+C content in bacterial genes. Proc Natl Acad Sci 109:14504–14507." />
<meta name="citation_reference" content="Newman ZR, Young JM, Ingolia NT, Barton GM. 2016. Differences in codon bias and GC content contribute to the balanced expression of TLR7 and TLR9. Proc Natl Acad Sci 113:E1362–E1371." />
<meta name="citation_reference" content="Giallonardo FD, Schlub TE, Shi M, Holmes EC. 2017. Dinucleotide composition in animal RNA viruses is shaped more by virus family than host species. J Virol JVI.02381-16." />
<meta name="citation_reference" content="Atkinson NJ, Witteveldt J, Evans DJ, Simmonds P. 2014. The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication. Nucleic Acids Res 42:4527–4545." />
<meta name="citation_reference" content="Fros JJ, Dietrich I, Alshaikhahmed K, Passchier TC, Evans DJ, Simmonds P. 2017. CpG and UpA dinucleotides in both coding and non-coding regions of echovirus 7 inhibit replication initiation post-entry. eLife 6:e29112." />
<meta name="citation_reference" content="Gaunt E, Wise HM, Zhang H, Lee LN, Atkinson NJ, Nicol MQ, Highton AJ, Klenerman P, Beard PM, Dutia BM, Digard P, Simmonds P. 2016. Elevation of CpG frequencies in influenza A genome attenuates pathogenicity but enhances host response to infection. eLife 5:e12735." />
<meta name="citation_reference" content="Burns CC, Campagnoli R, Shaw J, Vincent A, Jorba J, Kew O. 2009. Genetic Inactivation of Poliovirus Infectivity by Increasing the Frequencies of CpG and UpA Dinucleotides within and across Synonymous Capsid Region Codons. J Virol 83:9957–9969." />
<meta name="citation_reference" content="Irwin B, Heck JD, Hatfield GW. 1995. Codon Pair Utilization Biases Influence Translational Elongation Step Times. J Biol Chem 270:22801–22806." />
<meta name="citation_reference" content="Tats A, Tenson T, Remm M. 2008. Preferred and avoided codon pairs in three domains of life. BMC Genomics 9:463." />
<meta name="citation_reference" content="Gutman GA, Hatfield GW. 1989. Nonrandom utilization of codon pairs in Escherichia coli. Proc Natl Acad Sci U S A 86:3699–3703." />
<meta name="citation_reference" content="Gamble CE, Brule CE, Dean KM, Fields S, Grayhack EJ. 2016. Adjacent Codons Act in Concert to Modulate Translation Efficiency in Yeast. Cell 166:679–690." />
<meta name="citation_reference" content="Rocha EPC, Danchin A, Viari A. 2001. Evolutionary Role of Restriction/Modification Systems as Revealed by Comparative Genome Analysis. Genome Res 11:946–958." />
<meta name="citation_reference" content="Levin BR. 1993. The accessory genetic elements of bacteria: existence conditions and (co)evolution. Curr Opin Genet Dev 3:849–854." />
<meta name="citation_reference" content="Karlin S, Burge C, Campbell AM. 1992. Statistical analyses of counts and distributions of restriction sites in DNA sequences. Nucleic Acids Res 20:1363–1370." />
<meta name="citation_reference" content="Pleška M, Guet CC. 2017. Effects of mutations in phage restriction sites during escape from restriction–modification. Biol Lett 13:20170646." />
<meta name="citation_reference" content="Cáceres EF, Hurst LD. 2013. The evolution, impact and properties of exonic splice enhancers. Genome Biol 14:R143." />
<meta name="citation_reference" content="Kudla G, Murray AW, Tollervey D, Plotkin JB. 2009. Coding-Sequence Determinants of Gene Expression in Escherichia coli. Science 324:255–258." />
<meta name="citation_reference" content="Burkhardt DH, Rouskin S, Zhang Y, Li G-W, Weissman JS, Gross CA. 2017. Operon mRNAs are organized into ORF-centric structures that predict translation efficiency. eLife 6:e22037." />
<meta name="citation_reference" content="Boël G, Letso R, Neely H, Price WN, Wong K-H, Su M, Luff JD, Valecha M, Everett JK, Acton TB, Xiao R, Montelione GT, Aalberts DP, Hunt JF. 2016. Codon influence on protein expression in E. coli correlates with mRNA levels. Nature 529:358–363." />
<meta name="citation_reference" content="Kelsic ED, Chung H, Cohen N, Park J, Wang HH, Kishony R. 2016. RNA Structural Determinants of Optimal Codons Revealed by MAGE-Seq. Cell Syst 3:563–571.e6." />
<meta name="citation_reference" content="Presnyak V, Alhusaini N, Chen Y-H, Martin S, Morris N, Kline N, Olson S, Weinberg D, Baker KE, Graveley BR, Coller J. 2015. Codon Optimality Is a Major Determinant of mRNA Stability. Cell 160:1111–1124." />
<meta name="citation_reference" content="Ponnala L. 2010. A Plausible Role for the Presence of Internal Shine-Dalgarno Sites. Bioinforma Biol Insights 4:55–60." />
<meta name="citation_reference" content="Li G-W, Oh E, Weissman JS. 2012. The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria. Nature 484:538–541." />
<meta name="citation_reference" content="Goodman DB, Church GM, Kosuri S. 2013. Causes and Effects of N-Terminal Codon Bias in Bacterial Genes. Science 342:475–479." />
<meta name="citation_reference" content="Tuller T, Carmi A, Vestsigian K, Navon S, Dorfan Y, Zaborske J, Pan T, Dahan O, Furman I, Pilpel Y. 2010. An Evolutionarily Conserved Mechanism for Controlling the Efficiency of Protein Translation. Cell 141:344–354." />
<meta name="citation_reference" content="Chen GF, Inouye M. 1990. Suppression of the negative effect of minor arginine codons on gene expression; preferential usage of minor codons within the first 25 codons of the Escherichia coli genes. Nucleic Acids Res 18:1465–1473." />
<meta name="citation_reference" content="Cannarozzi G, Schraudolph NN, Faty M, von Rohr P, Friberg MT, Roth AC, Gonnet P, Gonnet G, Barral Y. 2010. A Role for Codon Order in Translation Dynamics. Cell 141:355–367." />
<meta name="citation_reference" content="Zhang G, Hubalewska M, Ignatova Z. 2009. Transient ribosomal attenuation coordinates protein synthesis and co-translational folding. Nat Struct Mol Biol 16:274–280." />
<meta name="citation_reference" content="Yu C-H, Dang Y, Zhou Z, Wu C, Zhao F, Sachs MS, Liu Y. 2015. Codon Usage Influences the Local Rate of Translation Elongation to Regulate Co-translational Protein Folding. Mol Cell 59:744–754." />
<meta name="citation_reference" content="Ikemura T. 1985. Codon usage and tRNA content in unicellular and multicellular organisms. Mol Biol Evol 2:13–34." />
<meta name="citation_reference" content="Grantham R, Gautier C, Gouy M, Mercier R, Pavé A. 1980. Codon catalog usage and the genome hypothesis. Nucleic Acids Res 8:r49–r62." />
<meta name="citation_reference" content="Hershberg R, Petrov DA. 2008. Selection on codon bias. Annu Rev Genet 42:287–299." />
<meta name="citation_reference" content="Lawrie DS, Messer PW, Hershberg R, Petrov DA. 2013. Strong Purifying Selection at Synonymous Sites in D. melanogaster. PLOS Genet 9:e1003527." />
<meta name="citation_reference" content="Machado HE, Lawrie DS, Petrov DA. 2017. Strong purifying selection on codon usage bias. bioRxiv 106476." />
<meta name="citation_reference" content="Lynch M, Conery JS. 2003. The Origins of Genome Complexity. Science 302:1401–1404." />
<meta name="citation_reference" content="Kristofich J, Morgenthaler AB, Kinney WR, Ebmeier CC, Snyder DJ, Old WM, Cooper VS, Copley SD. 2018. Synonymous mutations make dramatic contributions to fitness when growth is limited by a weak-link enzyme. PLOS Genet 14:e1007615." />
<meta name="citation_reference" content="Lucks JB, Nelson DR, Kudla GR, Plotkin JB. 2008. Genome Landscapes and Bacteriophage Codon Usage. PLoS Comput Biol 4." />
<meta name="citation_reference" content="Shin YC, Bischof GF, Lauer WA, Desrosiers RC. 2015. Importance of codon usage for the temporal regulation of viral gene expression. Proc Natl Acad Sci U S A 112:14030–14035." />
<meta name="citation_reference" content="Sau K, Gupta SK, Sau S, Ghosh TC. 2005. Synonymous codon usage bias in 16 Staphylococcus aureus phages: Implication in phage therapy. Virus Res 113:123–131." />
<meta name="citation_reference" content="Kunisawa T, Kanaya S, Kutter E. 1998. Comparison of synonymous codon distribution patterns of bacteriophage and host genomes. DNA Res Int J Rapid Publ Rep Genes Genomes 5:319–326." />
<meta name="citation_reference" content="Kunisawa T. 1992. Synonymous codon preferences in bacteriophage T4: A distinctive use of transfer RNAs from T4 and from its host Escherichia coli. J Theor Biol 159:287–298." />
<meta name="citation_reference" content="Nabel GJ. 2013. Designing Tomorrow’s Vaccines. N Engl J Med 368:551–560." />
<meta name="citation_reference" content="Rueckert C, Guzmán CA. 2012. Vaccines: From Empirical Development to Rational Design. PLOS Pathog 8:e1003001." />
<meta name="citation_reference" content="Minor PD. 2015. Live attenuated vaccines: Historical successes and current challenges. Virology 479–480:379–392." />
<meta name="citation_reference" content="Ramezanpour B, Haan I, Osterhaus A, Claassen E. 2016. Vector-based genetically modified vaccines: Exploiting Jenner’s legacy. Vaccine 34:6436–6448." />
<meta name="citation_reference" content="Lim K, Lang T, Lam V, Yin J. 2006. Model-Based Design of Growth-Attenuated Viruses. PLOS Comput Biol 2:e116." />
<meta name="citation_reference" content="Mueller S, Coleman JR, Papamichail D, Ward CB, Nimnual A, Futcher B, Skiena S, Wimmer E. 2010. Live attenuated influenza virus vaccines by computer-aided rational design. Nat Biotechnol 28:723–726." />
<meta name="citation_reference" content="Cann AJ, Stanway G, Hughes PJ, Minor PD, Evans DM, Schild GC, Almond JW. 1984. Reversion to neurovirulence of the live-attenuated Sabin type 3 oral poliovirus vaccine. Nucleic Acids Res 12:7787–7792." />
<meta name="citation_reference" content="Famulare M, Chang S, Iber J, Zhao K, Adeniji JA, Bukbuk D, Baba M, Behrend M, Burns CC, Oberste MS. 2015. Sabin Vaccine Reversion in the Field: a Comprehensive Analysis of Sabin-Like Poliovirus Isolates in Nigeria. J Virol 90:317–331." />
<meta name="citation_reference" content="Jorba J, Diop OM, Iber J, Sutter RW, Wassilak SG, Burns CC. 2016. Update on Vaccine-Derived Polioviruses — Worldwide, January 2015–May 2016. MMWR Morb Mortal Wkly Rep 65:763–769." />
<meta name="citation_reference" content="Kew O. 2012. Reaching the last one per cent: progress and challenges in global polio eradication. Curr Opin Virol 2:188–198." />
<meta name="citation_reference" content="Cello J, Paul AV, Wimmer E. 2002. Chemical Synthesis of Poliovirus cDNA: Generation of Infectious Virus in the Absence of Natural Template. Science." />
<meta name="citation_reference" content="Burns CC, Shaw J, Campagnoli R, Jorba J, Vincent A, Quay J, Kew O. 2006. Modulation of Poliovirus Replicative Fitness in HeLa Cells by Deoptimization of Synonymous Codon Usage in the Capsid Region. J Virol 80:3259–3272." />
<meta name="citation_reference" content="Mueller S, Papamichail D, Coleman JR, Skiena S, Wimmer E. 2006. Reduction of the Rate of Poliovirus Protein Synthesis through Large-Scale Codon Deoptimization Causes Attenuation of Viral Virulence by Lowering Specific Infectivity. J Virol 80:9687–9696." />
<meta name="citation_reference" content="Coleman JR, Papamichail D, Skiena S, Futcher B, Wimmer E, Mueller S. 2008. Virus Attenuation by Genome-Scale Changes in Codon Pair Bias. Science 320:1784–1787." />
<meta name="citation_reference" content="Martínez MA, Jordan-Paiz A, Franco S, Nevot M. 2016. Synonymous Virus Genome Recoding as a Tool to Impact Viral Fitness. Trends Microbiol 24:134–147." />
<meta name="citation_reference" content="Bull JJ, Molineux IJ, Wilke CO. 2012. Slow Fitness Recovery in a Codon-Modified Viral Genome. Mol Biol Evol 29:2997–3004." />
<meta name="citation_reference" content="Cheng BYH, Ortiz-Riaño E, Nogales A, Torre JC de la, Martínez-Sobrido L. 2015. Development of Live-Attenuated Arenavirus Vaccines Based on Codon Deoptimization. J Virol 89:3523–3533." />
<meta name="citation_reference" content="Cheng BYH, Nogales A, de la Torre JC, Martínez-Sobrido L. 2017. Development of live-attenuated arenavirus vaccines based on codon deoptimization of the viral glycoprotein. Virology 501:35–46." />
<meta name="citation_reference" content="Cladel NM, Budgeon LR, Hu J, Balogh KK, Christensen ND. 2013. Synonymous codon changes in the oncogenes of the cottontail rabbit papillomavirus lead to increased oncogenicity and immunogenicity of the virus. Virology 438:70–83." />
<meta name="citation_reference" content="Luan S, Pan W, Li T, Yang H, Zhang B, Li F, Chen L. 2009. Rescued influenza A virus with codon deoptimized NS1 gene is attenuated both in vitro and in vivo. Chin J Biotechnol 25:720–726." />
<meta name="citation_reference" content="Meng J, Lee S, Hotard AL, Moore ML. 2014. Refining the Balance of Attenuation and Immunogenicity of Respiratory Syncytial Virus by Targeted Codon Deoptimization of Virulence Genes. mBio 5:e01704–14." />
<meta name="citation_reference" content="Nogales A, Baker SF, Ortiz-Riaño E, Dewhurst S, Topham DJ, Martínez-Sobrido L. 2014. Influenza A Virus Attenuation by Codon Deoptimization of the NS Gene for Vaccine Development. J Virol 88:10525–10540." />
<meta name="citation_reference" content="Rostad CA, Stobart CC, Gilbert BE, Pickles RJ, Hotard AL, Meng J, Blanco JCG, Moin SM, Graham BS, Piedra PA, Moore ML. 2016. A Recombinant Respiratory Syncytial Virus Vaccine Candidate Attenuated by a Low-Fusion F Protein Is Immunogenic and Protective against Challenge in Cotton Rats. J Virol 90:7508–7518." />
<meta name="citation_reference" content="Velazquez-Salinas L, Risatti GR, Holinka LG, O’Donnell V, Carlson J, Alfano M, Rodriguez LL, Carrillo C, Gladue DP, Borca MV. 2016. Recoding structural glycoprotein E2 in classical swine fever virus (CSFV) produces complete virus attenuation in swine and protects infected animals against disease. Virology 494:178–189." />
<meta name="citation_reference" content="Nougairede A, Fabritus LD, Aubry F, Gould EA, Holmes EC, Lamballerie X de. 2013. Random Codon Re-encoding Induces Stable Reduction of Replicative Fitness of Chikungunya Virus in Primate and Mosquito Cells. PLOS Pathog 9:e1003172." />
<meta name="citation_reference" content="Fabritus L de, Nougairède A, Aubry F, Gould EA, Lamballerie X de. 2015. Attenuation of Tick-Borne Encephalitis Virus Using Large-Scale Random Codon Re-encoding. PLOS Pathog 11:e1004738." />
<meta name="citation_reference" content="Fabritus L de, Nougairède A, Aubry F, Gould EA, Lamballerie X de. 2016. Utilisation of ISA Reverse Genetics and Large-Scale Random Codon Re-Encoding to Produce Attenuated Strains of Tick-Borne Encephalitis Virus within Days. PLOS ONE 11:e0159564." />
<meta name="citation_reference" content="Villanueva E, Martí-Solano M, Fillat C. 2016. Codon optimization of the adenoviral fiber negatively impacts structural protein expression and viral fitness. Sci Rep 6:srep27546." />
<meta name="citation_reference" content="Liang B, Ngwuta JO, Surman S, Kabatova B, Liu X, Lingemann M, Liu X, Yang L, Herbert R, Swerczek J, Chen M, Moin SM, Kumar A, McLellan JS, Kwong PD, Graham BS, Collins PL, Munir S. 2017. Improved Prefusion Stability, Optimized Codon Usage, and Augmented Virion Packaging Enhance the Immunogenicity of Respiratory Syncytial Virus Fusion Protein in a Vectored-Vaccine Candidate. J Virol 91:e00189–17." />
<meta name="citation_reference" content="Vabret N, Bailly-Bechet M, Lepelley A, Najburg V, Schwartz O, Verrier B, Tangy F. 2014. Large-Scale Nucleotide Optimization of Simian Immunodeficiency Virus Reduces Its Capacity To Stimulate Type I Interferon In Vitro. J Virol 88:4161–4172." />
<meta name="citation_reference" content="Moratorio G, Henningsson R, Barbezange C, Carrau L, Bordería AV, Blanc H, Beaucourt S, Poirier EZ, Vallet T, Boussier J, Mounce BC, Fontes M, Vignuzzi M. 2017. Attenuation of RNA viruses by redirecting their evolution in sequence space. Nat Microbiol 2:17088." />
<meta name="citation_reference" content="Le Nouën CL, Brock LG, Luongo C, McCarty T, Yang L, Mehedi M, Wimmer E, Mueller S, Collins PL, Buchholz UJ, DiNapoli JM. 2014. Attenuation of human respiratory syncytial virus by genome-scale codon-pair deoptimization. Proc Natl Acad Sci 111:13169–13174." />
<meta name="citation_reference" content="Martrus G, Nevot M, Andres C, Clotet B, Martinez MA. 2013. Changes in codon-pair bias of human immunodeficiency virus type 1 have profound effects on virus replication in cell culture. Retrovirology 10:78." />
<meta name="citation_reference" content="Shen SH, Stauft CB, Gorbatsevych O, Song Y, Ward CB, Yurovsky A, Mueller S, Futcher B, Wimmer E. 2015. Large-scale recoding of an arbovirus genome to rebalance its insect versus mammalian preference. Proc Natl Acad Sci 112:4749–4754." />
<meta name="citation_reference" content="Sharp PM, Li WH. 1987. The codon Adaptation Index--a measure of directional synonymous codon usage bias, and its potential applications. Nucleic Acids Res 15:1281–1295." />
<meta name="citation_reference" content="Xia X. 2007. An Improved Implementation of Codon Adaptation Index. Evol Bioinforma Online 3:53–58." />
<meta name="citation_reference" content="Reis M dos, Savva R, Wernisch L. 2004. Solving the riddle of codon usage preferences: a test for translational selection. Nucleic Acids Res 32:5036–5044." />
<meta name="citation_reference" content="Xia X. 2015. A Major Controversy in Codon-Anticodon Adaptation Resolved by a New Codon Usage Index. Genetics 199:573–579." />
<meta name="citation_reference" content="Fox JM, Erill I. 2010. Relative Codon Adaptation: A Generic Codon Bias Index for Prediction of Gene Expression. DNA Res 17:185–196." />
<meta name="citation_reference" content="Wright F. 1990. The ‘effective number of codons’ used in a gene. Gene 87:23–29." />
<meta name="citation_reference" content="Elf J, Nilsson D, Tenson T, Ehrenberg M. 2003. Selective Charging of tRNA Isoacceptors Explains Patterns of Codon Usage. Science 300:1718–1722." />
<meta name="citation_reference" content="Bentele K, Saffert P, Rauscher R, Ignatova Z, Blüthgen N. 2013. Efficient translation initiation dictates codon usage at gene start. Mol Syst Biol 9:675." />
<meta name="citation_reference" content="Miller CR, Van Leuven JT, Wichman HA, Joyce P. 2017. Selecting among three basic fitness landscape models: Additive, multiplicative and stickbreaking. Theor Popul Biol." />
<meta name="citation_reference" content="Nagel AC, Joyce P, Wichman HA, Miller CR. 2012. Stickbreaking: A Novel Fitness Landscape Model That Harbors Epistasis and Is Consistent with Commonly Observed Patterns of Adaptive Evolution. Genetics 190:655–667." />
<meta name="citation_reference" content="Robinson M, Lilley R, Little S, Emtage JS, Yarranton G, Stephens P, Millican A, Eaton M, Humphreys G. 1984. Codon usage can affect efficiency of translation of genes in Escherichia coli. Nucleic Acids Res 12:6663–6671." />
<meta name="citation_reference" content="Quax TEF, Wolf YI, Koehorst JJ, Wurtzel O, van der Oost R, Ran W, Blombach F, Makarova KS, Brouns SJJ, Forster AC, Wagner EGH, Sorek R, Koonin EV, van der Oost J. 2013. Differential Translation Tunes Uneven Production of Operon-Encoded Proteins. Cell Rep 4:938–944." />
<meta name="citation_reference" content="Aragonès L, Guix S, Ribes E, Bosch A, Pintó RM. 2010. Fine-Tuning Translation Kinetics Selection as the Driving Force of Codon Usage Bias in the Hepatitis A Virus Capsid. PLOS Pathog 6:e1000797." />
<meta name="citation_reference" content="Zhao K-N, Chen J. 2011. Codon usage roles in human papillomavirus. Rev Med Virol 21:397–411." />
<meta name="citation_reference" content="Kula A, Saelens J, Cox J, Schubert AM, Travisano M, Putonti C. 2018. The Evolution of Molecular Compatibility between Bacteriophage FX174 and its Host. Sci Rep 8:8350." />
<meta name="citation_reference" content="Kimchi-Sarfaty C, Oh JM, Kim I-W, Sauna ZE, Calcagno AM, Ambudkar SV, Gottesman MM. 2007. A “Silent” Polymorphism in the MDR1 Gene Changes Substrate Specificity. Science 315:525–528." />
<meta name="citation_reference" content="Bailey SF, Hinz A, Kassen R. 2014. Adaptive synonymous mutations in an experimentally evolved Pseudomonas fluorescens population. Nat Commun 5:4076." />
<meta name="citation_reference" content="Agashe D, Sane M, Phalnikar K, Diwan GD, Habibullah A, Martinez-Gomez NC, Sahasrabuddhe V, Polachek W, Wang J, Chubiz LM, Marx CJ. 2016. Large-Effect Beneficial Synonymous Mutations Mediate Rapid and Parallel Adaptation in a Bacterium. Mol Biol Evol 33:1542–1553." />
<meta name="citation_reference" content="Schenk MF, Szendro IG, Krug J, Visser JAGM de. 2012. Quantifying the Adaptive Potential of an Antibiotic Resistance Enzyme. PLOS Genet 8:e1002783." />
<meta name="citation_reference" content="Zwart MP, Schenk MF, Hwang S, Koopmanschap B, Lange N de, Pol L van de, Nga TTT, Szendro IG, Krug J, Visser JAGM de. 2018. Unraveling the causes of adaptive benefits of synonymous mutations in TEM-1 β-lactamase. Heredity 121:406." />
<meta name="citation_reference" content="Couce A, Tenaillon OA. 2015. The rule of declining adaptability in microbial evolution experiments. Front Genet 6." />
<meta name="citation_reference" content="Chou H-H, Chiu H-C, Delaney NF, Segrè D, Marx CJ. 2011. Diminishing Returns Epistasis Among Beneficial Mutations Decelerates Adaptation. Science 332:1190–1192." />
<meta name="citation_reference" content="West SA, Peters AD, Barton NH. 1998. Testing for Epistasis Between Deleterious Mutations. Genetics 149:435–444." />
<meta name="citation_reference" content="de Visser JAGM, Cooper TF, Elena SF. 2011. The causes of epistasis. Proc R Soc B Biol Sci 278:3617–3624." />
<meta name="citation_reference" content="Kouyos RD, Silander OK, Bonhoeffer S. 2007. Epistasis between deleterious mutations and the evolution of recombination. Trends Ecol Evol 22:308–315." />
<meta name="citation_reference" content="Segrè D, DeLuna A, Church GM, Kishony R. 2005. Modular epistasis in yeast metabolism. Nat Genet 37:77–83." />
<meta name="citation_reference" content="Elena SF, Lenski RE. 1997. Test of synergistic interactions among deleterious mutations in bacteria. Nature 390:395–398." />
<meta name="citation_reference" content="Sanjuán R, Moya A, Elena SF. 2004. The contribution of epistasis to the architecture of fitness in an RNA virus. Proc Natl Acad Sci U S A 101:15376–15379." />
<meta name="citation_reference" content="Jasnos L, Korona R. 2007. Epistatic buffering of fitness loss in yeast double deletion strains. Nat Genet 39:550–554." />
<meta name="citation_reference" content="Guerrero RF, Muir CD, Josway S, Moyle LC. 2017. Pervasive antagonistic interactions among hybrid incompatibility loci. PLOS Genet 13:e1006817." />
<meta name="citation_reference" content="Parera M, Perez-Alvarez N, Clotet B, Martínez MA. 2009. Epistasis among Deleterious Mutations in the HIV-1 Protease. J Mol Biol 392:243–250." />
<meta name="citation_reference" content="Wang X, Minasov G, Shoichet BK. 2002. Evolution of an Antibiotic Resistance Enzyme Constrained by Stability and Activity Trade-offs. J Mol Biol 320:85–95." />
<meta name="citation_reference" content="Lalic J, Elena SF. 2012. Magnitude and sign epistasis among deleterious mutations in a positive-sense plant RNA virus. Heredity 109:71–77." />
<meta name="citation_reference" content="Johnson MS, Martsul A, Kryazhimskiy S, Desai MM. 2019. Higher-fitness yeast genotypes are less robust to deleterious mutations. Science 366:490–493." />
<meta name="citation_reference" content="Kunec D, Osterrieder N. 2016. Codon Pair Bias Is a Direct Consequence of Dinucleotide Bias. Cell Rep 14:55–67." />
<meta name="citation_reference" content="Futcher B, Gorbatsevych O, Shen SH, Stauft CB, Song Y, Wang B, Leatherwood J, Gardin J, Yurovsky A, Mueller S, Wimmer E. 2015. Reply to Simmonds et al.: Codon pair and dinucleotide bias have not been functionally distinguished. Proc Natl Acad Sci 112:E3635–E3636." />
<meta name="citation_reference" content="Jorge DM de M, Mills RE, Lauring AS. 2015. CodonShuffle: a tool for generating and analyzing synonymously mutated sequences. Virus Evol 1." />
<meta name="citation_reference" content="Grote A, Hiller K, Scheer M, Münch R, Nörtemann B, Hempel DC, Jahn D. 2005. JCat: a novel tool to adapt codon usage of a target gene to its potential expression host. Nucleic Acids Res 33:W526–W531." />
<meta name="citation_reference" content="Chin JX, Chung BK-S, Lee D-Y. 2014. Codon Optimization OnLine (COOL): a web-based multi-objective optimization platform for synthetic gene design. Bioinformatics 30:2210–2212." />
<meta name="citation_reference" content="Paff ML, Jack BR, Smith BL, Bull JJ, Wilke CO. 2018. Combinatorial Approaches to Viral Attenuation. mSystems 3:e00046–18." />
<meta name="citation_reference" content="Donaldson EF, Yount B, Sims AC, Burkett S, Pickles RJ, Baric RS. 2008. Systematic Assembly of a Full-Length Infectious Clone of Human Coronavirus NL63. J Virol 82:11948–11957." />
<meta name="citation_reference" content="Wichman HA, Millstein J, Bull JJ. 2005. Adaptive Molecular Evolution for 13,000 Phage Generations. Genetics 170:19–31." />
<meta name="citation_reference" content="Karlin S, Mrázek J, Campbell A, Kaiser D. 2001. Characterizations of Highly Expressed Genes of Four Fast-Growing Bacteria. J Bacteriol 183:5025–5040." />
<meta name="twitter:title" content="ΦX174 Attenuation by Whole Genome Codon Deoptimization" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence." />
<meta name="og-title" property="og:title" content="ΦX174 Attenuation by Whole Genome Codon Deoptimization" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-11" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.10.942847v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.10.942847v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.10.942847v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1145901" />
    <title>ΦX174 Attenuation by Whole Genome Codon Deoptimization | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"np7Prdp741wb-IS-pZ3w6diWZ7U06DOO2fk9T4avH_4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1145901","apath":"\/biorxiv\/early\/2020\/02\/11\/2020.02.10.942847.atom","pisa":"biorxiv;2020.02.10.942847v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.10.942847v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.10.942847v1","title":"\u03a6X174 Attenuation by Whole Genome Codon Deoptimization","identifier":"node\/1145901"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dM\\\\\\\/fJUrzec3gBKHIFZ6L0hHQ6mkL9kTIKiKNn\\\\\\\/LvTN9dHQzNr5RJpvvAOLK9RtGOVV+TnlHrhCenuaRZXCo8aLx76P0jt4XF2H1uU9lh+nWeB9uadjd8zZQO5OiujizQRWsuv4Cb2AVkCHIxS2Y0MHUenliZvQez9NllKYqrnuyxIRwWda\\\\\\\/zu\\\\\\\/nOMmcSmA3M5Kmq9N1Fw4DKgMpVp9SkVZezSZjcKpi9dJUVO2WkKVNPrSzjtx\\\\\\\/vbmTXE+Rxe9454Xm54\\\\\\\/wP800j1JOsmeIH0+q\\\\\\\/aZtjSqR5kNT3BDy0KQq1xnpWpB3HrEoC7Pa0K7ioPQfXDwHzNPqEDJXilnoWhRRgPQsx+GS8jHDL\\\\\\\/TrnQrk1l3DbOpaO2HsTchqgHlYt0ywL+rSZYTUrauNXLmST7reaGyeTki2vREJfnIHhuFUPfPu+K0x4W5yfxajz9WeUy2+z37Zjciuh6y11jH7Iuq9lFOtezEkePGV5jOzmab6HJsFrSBfczBnuTuzKQkZn\\\\\\\/lj8xtIDZOJipfp9QRMmqhfssjROzsu4Oc9X6Y35jy9QGLBT8RLPwW9wREzuDQbl14xSpCCCijMuFfPIL+hS48LHkznTX+IZiQs1CUI5Sw0iVDNVY3Km7GCCqsOLhYMb1VMcaB5germHELCH6gq163YPVjd7SpdORhuTkgxRUufxrOsKHGHgIKOY+y0KOzVQghA\\\\\\\/ckeGgXFrGctK7srBqi7Obg\\\\\\\/NKcZ\\\\\\\/E5Z\\\\\\\/DpgPYX2JkqERAqYI33Oi7WYl++QaynkNke+gN4BojSZle6BmNLD1\\\\\\\/Ga\\\\\\\/ePtIi2EWT2MLY211JSbIfwBEt1u\\\\\\\/hsXsaQWNUHw6YeHn3DqW5TyYJ43CIz\\\\\\\/VcHlWe3HPcp5kw+0c5QmUP9LrBEg1rHKmXV7CiEIw12AkBr8JwqaDqpvGdxBzzaA2EL32nkpPomoW\\\\\\\/sbGabc5Z7z++olX5lPpZe5zuRBHWW9aF3UF8bymrmhCb6aMb8ZECKR8\\\\\\\/isJc43URj\\\\\\\/l3+bXhpn6wZOIm81mBCcF8jn2j6QKHwFTKZSsSn7QMCP3MFiMelucBZLR7\\\\\\\/LyZRdfE1AXjh86we6DuA6uyvX5P8WLSdOpwkKRnRAEKwiHw4rbma16CFWTuyVuTTpUr6MG+rROBuPRD2kROJ699\\\\\\\/goC\\\\\\\/jk+fIO5JY2Ngz4wLOwPddFGH8akzvmXMbXvDdsZu4mK\\\\\\\/KsrXVxOU47gd7IrMAl9KTtf\\\\\\\/vY9Gxdn7k2H9Q6GoaXHHa4iDzdtTYC2fpJd6uZZSdb91pQfawXdvjSeO200Qw0SVTN2wwdSiRjapo2XzrOsTIoL4LpY6CXFOqJaGp\\\\\\\/\\\\\\\/olZJ3AVprQRKHBrOe2Ix179\\\u0022,\\\u0022iv\\\u0022:\\\u0022xnH5Lm2aQQIbqycGOIWdqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002222dc35fff8b3a2f9befed95c4d558dd706c69b3f87414e54bdbfd3715f2c0a0f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.10.942847v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1145901 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.10.942847v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YJvxWHR1Wt3855WWAfRXI9aBQf-DQcePiBjhAnNXTqY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1145901" id="node1145901" data-pisa="biorxiv;2020.02.10.942847v1" data-pisa-master="biorxiv;2020.02.10.942847" data-apath="/biorxiv/early/2020/02/11/2020.02.10.942847.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">ΦX174 Attenuation by Whole Genome Codon Deoptimization</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2730-0287" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">Burleigh</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">LuAnn</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4665-0335" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Randall A.</span> <span class="nlm-surname">Hughes</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Vlad</span> <span class="nlm-surname">Codrea</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew D.</span> <span class="nlm-surname">Ellington</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Holly</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7920-4994" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.10.942847 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1145901" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">James T. Van Leuven </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow, ID 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BT.%2BVan%2BLeuven%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Leuven%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BT.%2BVan%2BLeuven%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2730-0287" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James T. Van Leuven</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martina M. Ederer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BM.%2BEderer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ederer%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartina%2BM.%2BEderer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Katelyn Burleigh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Seattle Children’s Research Institute</span>, Seattle, WA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katelyn%2BBurleigh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burleigh%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatelyn%2BBurleigh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">LuAnn Scott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=LuAnn%2BScott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuAnn%2BScott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Randall A. Hughes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Applied Research Laboratories, University of Texas</span>, Austin, TX 78758</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Biotechnology Branch, CCDC US Army Research Laboratory</span>, 2800 Powder Mill Rd, Adelphi, MD 20783</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randall%2BA.%2BHughes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hughes%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARandall%2BA.%2BHughes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4665-0335" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Randall A. Hughes</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vlad Codrea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Institute for Cellular and Molecular Biology, University of Texas</span>, Austin, TX 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vlad%2BCodrea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Codrea%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVlad%2BCodrea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Andrew D. Ellington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Applied Research Laboratories, University of Texas</span>, Austin, TX 78758</div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Institute for Cellular and Molecular Biology, University of Texas</span>, Austin, TX 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BEllington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ellington%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.%2BEllington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Holly Wichman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow, ID 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Holly%2BWichman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wichman%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHolly%2BWichman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Craig Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Science, University of Idaho</span>, Moscow, ID 83844</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow, ID 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7920-4994" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Craig Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:crmiller@uidaho.edu" class="" data-icon-position="" data-hide-link-title="0">crmiller@uidaho.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.10.942847v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.10.942847v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.10.942847v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.10.942847v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.10.942847v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.10.942847v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1145901" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1145901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its <em>E. coli</em> host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><div id="sec-2" class="subsection"><h3>Synonymous compositional features of viral genomes</h3><p id="p-4">The unequal use of synonymous codons is known as codon usage bias (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Codon biases are the result of an interaction between mutational and selective pressures (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Understanding how these two forces combine to determine codon usage in a genome is challenging. A commonly accepted explanation is that the rate of translation (i.e. the amount of proteins being made) is affected by the abundance of tRNAs that pair with codons on a strand of mRNA. Codons that result in the optimal amount of protein then have a selective advantage. However, there are many other compositional features within the DNA sequences of protein coding genes upon which selection acts. These features include: the genic GC content (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>), CpG or TpA dinucleotides (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), codon pairs (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>), endonuclease recognition sites (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>), intron splicing motifs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), mRNA folding stability (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>), ribosomal pausing sites (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>), concentration of non-preferred codons at transcripts 5’ end (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>), autocorrelation of codons on transcript (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>), and capacity of codon order to influence co-translational folding of proteins (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). At any given site, natural selection may act on one or more of these features to favor the use of certain codons over other synonymous ones (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). While the strength of selection on synonymous mutations is generally considered weak to neutral and genetic drift should therefore also exert a strong influence, the overall signature of selection in the form of non-random codon usage is broadly observed across life (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>–<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>) where selection has been shown to strongly favor one synonymous mutation over others (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>–<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>).</p><p id="p-5">As viruses must utilize their hosts’ cellular machinery, there is an expectation that virus genomes are enriched for host-preferred codons. This appears to be only partially true. Many viral genomes contain more host-preferred codons than expected by chance, especially in highly expressed genes encoding viral structural proteins (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). However, many viral genes are not enriched in host-preferred codons. Sometimes unpreferred codons are used to regulate viral gene expression (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). Other virus genomes appear to have little preference for codons abundant in the host genome. For instance, Lucks et al. found that the majority of 74 bacteriophage genomes show no significant preference for host-preferred codons (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>). Similar discordance between host and viral codon usage patterns are observed in other studies (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>–<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). This discordance could be caused by insufficient selection on codon usage, host-phage relationships that are too short-lived for selection to fine tune codon usage in the phage, or inadequate understanding of the features under selection.</p></div><div id="sec-3" class="subsection"><h3>Vaccine development by synonymous recoding</h3><p id="p-6">Empirically developed (e.g. serial passage viral adaptation) vaccines have saved millions of lives over the last century, yet methodological improvements make rationally designed, recombinant vaccines attractive because they can be rapidly produced and specifically engineered for safety and effectiveness (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>–<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). One proposed method of generating recombinant vaccines involves making many synonymous, attenuating changes to viral genomes, i.e. “deoptimizing” the viral genes (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). The recombinant vaccine can be made by either editing the genome of the wild-type virus or by generating a viral genome entirely from synthesized nucleic acids. Synonymous deoptimization offers a potentially efficient and effective way of making vaccines: the protein sequences of recoded vaccines are identical to their target viruses, they replicate in their host to provide prolonged exposure to the antigen, and the introduction of many synonymous changes presumably assures evolutionary robustness, preventing the evolution of virulence by reversion.</p><p id="p-7">Poliovirus serves as a very good example for the synonymous recoding strategy. Development of a robust, live-attenuated poliovirus vaccine is desired because in some areas on Earth wild poliovirus and the emergent vaccine-derived polioviruses (cVDPVs) continues to cause concern over the resurgence of poliomyelitis (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>–<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). A synthetic poliovirus was assembled in 2002 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), codon deoptimized in 2006 (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>), codon pair deoptimized in 2008 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>), and dinucleotide deoptimized in 2009 (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). In all cases, attenuated viruses were produced by recoding the P1/capsid region of the genome. <em>In vitro</em>, these viruses replicate slower and produce lower viral titers than wild-type virus. <em>In vivo</em> the codon pair deoptimized strain protected mice against challenge by wild poliovirus (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>). While the mechanism of attenuation is not yet fully elucidated, reduced protein expression of the deoptimized genes is observed (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>–<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">59</a>). These deoptimized poliovirus constructs are genetically stable and remain non-virulent for up to 25 passages in cell culture (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">59</a>).</p><p id="p-8">The apparent success of building poliovirus vaccine candidates using synonymous recoding led to similar attempts to develop vaccines for influenza, adeno-associated, human immunodeficiency, papilloma, chikungunya, respiratory syncytial, simian immunodeficiency, porcine reproductive and respiratory syndrome, echovirus 7, tick-borne encephalitis, vesicular stomatitis, dengue, T7, Lassa, adeno, and swine fever viruses (reviewed in (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>)). The most common method for synonymously deoptimizing viruses is recoding wildtype genes with increased proportions of non preferred codons (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>–<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>, <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">65</a>–<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>) although other methods of recoding have been successful as well. For example, viral fitness was decreased when synonymous substitutions were randomly introduced (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>–<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>), when codons were replaced by those infrequently used in viral (not host) genes (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>), when the proportion of optimal codons was <em>increased</em> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>, <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>), or when the number of codons one substitution away from a translational termination codon was increased (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>).</p></div><div id="sec-4" class="subsection"><h3>No predictive understanding of synonymous recoding</h3><p id="p-9">While it is clear that synonymous recoding causes attenuation and the strategy holds promise for vaccine development, we lack a predictive understanding of the process. Part of this results from the biological complexity and variation in the systems involved. In many cases, the fitness impact of recoding is cell-line dependent (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>, <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>–<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">70</a>, <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>–<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>), is inconsistent between <em>in vivo</em> and <em>in vitro</em> experiments (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>, <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a>, <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">79</a>, <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">79</a>), or is temporally variable (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">73</a>). Another obstacle is the nature of the genetic code itself: it is generally challenging to manipulate one compositional feature of the genome and hold all the others fixed. For example, when codons are shuffled to change codon pair frequency, mRNA stability may be affected. Or when codons are deoptimized, codon pair frequencies also change. This makes it difficult to attribute the cause of fitness decreases to one factor (<em>e.g.</em> codon usage adaptation), especially when the features are correlated. As we do here, most studies have focused on manipulating a single compositional feature of the genome and measuring its impact on fitness. Standardizing recoding methodologies and features measured across studies would greatly improve our understanding of the factors that drive fitness decreases and other phenotypic effects caused by synonymous deoptimization.</p><p id="p-10">Despite the optimistic results achieved in studies on synonymous recoding to date, basic questions underlying the method itself remain unanswered (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">60</a>). What is the best strategy to perform synonymous recoding to achieve attenuation? Can generalities be made about the extent of recoding and the degree of attenuation—or will the biological details and idiosyncrasies of each system preclude this? What is the mechanistic cause of attenuation from synonymous recoding? Are viruses recoded this way robust against fitness recovery? Is it more effective to maximally recode less of the genome (say one gene), or make the recoding less severe, but distribute it across the genome? As an increasing proportion of the genome is recoded, or equivalently, as multiple recoded parts are combined, does attenuation respond in an additive or non-additive manner? A deeper understanding of genome evolution and synonymous sequence choice is required to answer these questions.</p><p id="p-11">In this paper, we focus on two issues related to the codon deoptimization of viruses. First, we seek to compare the fitness effects of mutating different genes in the same virus. Second, we seek to understand how, when attenuating mutations are combined, they interact to affect fitness (<em>i.e.</em> epistasis of deleterious mutations). The nature of epistasis among fragments is crucial for modeling fitness effects: if mutational effects combine synergistically (<em>i.e.</em> the combined fitness being even lower than predicted from the observed individual effects), the range in the number of mutations needed to achieve the targeted attenuation level would tend to be reduced (<a id="xref-fig-1-1" class="xref-fig" href="#F1">fig. 1</a>). Conversely, if they combine antagonistically (<em>i.e.</em> the combination of mutations are less attenuated than predicted from individual effects), it may be easier to achieve a target attenuation level, but there may be a limit to how much attenuation is possible. If mutations, in combination, display sign epistasis, irregular magnitude epistasis, or even vary between synergistic and antagonistic epistasis, it will suggest the underlying process is complex and difficult to predict and generalize. To evaluate these issues, herein we have recoded all the non-overlapping genes of the bacteriophage ΦX174 in fragments and combined recoded fragments in all possible within-gene permutations, and measure the fitness of the resulting recoded bacteriophage.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hypothetical data illustrates that epistasis determines the number of mutations needed to attenuate a virus. The &#x201C;targeted attenuation&#x201D; window is more difficult to achieve if mutational effects combine synergistically than if mutations combine antagonistically because the slope of the curve is steeper (fewer mutations in the window). Notice that this pattern is reversed if a slight level of attenuation is desired (near y=0). If sign epistasis, or even irregular magnitude epistasis is observed, then the underlying nature of interactions are more difficult to predict and generalize." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Hypothetical data illustrates that epistasis determines the number of mutations needed to attenuate a virus.&lt;/span&gt; The &#x201C;targeted attenuation&#x201D; window is more difficult to achieve if mutational effects combine synergistically than if mutations combine antagonistically because the slope of the curve is steeper (fewer mutations in the window). Notice that this pattern is reversed if a slight level of attenuation is desired (near y=0). If sign epistasis, or even irregular magnitude epistasis is observed, then the underlying nature of interactions are more difficult to predict and generalize.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F1.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F1.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <span class="caption-title">Hypothetical data illustrates that epistasis determines the number of mutations needed to attenuate a virus.</span><p id="p-12" class="first-child">The “targeted attenuation” window is more difficult to achieve if mutational effects combine synergistically than if mutations combine antagonistically because the slope of the curve is steeper (fewer mutations in the window). Notice that this pattern is reversed if a slight level of attenuation is desired (near y=0). If sign epistasis, or even irregular magnitude epistasis is observed, then the underlying nature of interactions are more difficult to predict and generalize.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Synonymously deoptimizing ΦX174 genes</h3><p id="p-13">ΦX174 is a bacteriophage with a 5.4kb single-stranded DNA genome containing 11 genes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">fig. 2a</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">table 1</a>). We measured codon usage bias of ΦX174 genes using the codon adaptation index (CAI). CAI is a gene-level statistic running from zero to one that summarizes the extent that codons in a gene are used, rarely (CAI nearer 0) or commonly (CAI nearer 1) among highly expressed host genes (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>). We found that most ΦX174 genes are not particularly enriched for preferred <em>E. coli</em> codons (<a id="xref-fig-2-2" class="xref-fig" href="#F2">fig. 2b</a>). Only gene J has a codon adaptation index (CAI) value equivalent to highly expressed <em>E. coli</em> genes. Gene K uses the most unpreferred codons and has a CAI value near the lowest value observed for genes in the <em>E. coli</em> genome. All other ΦX174 genes have CAI values within the range of most <em>E. coli</em> protein-coding genes. ΦX174 structural proteins (B, D, F, G, H, J) have higher CAI values than non-structural genes (A, C, E, K), suggesting that high expression of these proteins is important for viral fitness. When we computationally deoptimized entire ΦX174 genes (<em>i.e.</em> recoded them to use the least-preferred codons throughout), the resulting CAI values were in the lower tail or even below the tail for all <em>E. coli</em> genes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">fig. 2</a>, supplementary table S1). These reductions in CAI were the result of changing between 42% (20/48 codons for gene C) and 75% (24/32 codons for gene J) of the codons of a gene, corresponding 15-32% of its base pairs (supplementary table S2). All the other ΦX174 genes fall within this range of recoding. We calculated additional metrics of codon adaptation including an alternative version of CAI (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>), tRNA adaptation index: tAI (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>), index of translation elongation: I<sub>TE</sub> (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>), relative codon adaptation: RCA (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>), the number of effective codons: Nc (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>), and the starvation codon adaptation index: sCAI (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>). Nc differs from the other indices in that it measures the deviation from uniform codon usage and does not score codon preference. The other indices compare the codon usage of any given gene to a set of genes that are known to be highly expressed in a cell. These latter indices differ from one another in how codon optimality is scored (<em>e.g.</em> taking into account tRNA gene abundances). Calculation details are in the Materials and Methods. All analyses produced qualitatively similar rank-orders for ΦX174 genes (supplementary table S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1148867/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1148867/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1148867" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-14" class="first-child">ΦX174 gene function and protein copy number required for the assembly of one virion.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="&#x3A6;X174 genome organization and capacity for deoptimization relative to host genes. (a) Genes on the &#x3A6;X174 genome are shown as empty black boxes. Recoded regions are shown as filled blue boxes. These fragments are named consecutively (e.g., A1, A2, A3, A4, F1, F2, F3,&#x2026;). Approximate transcript expression levels are shown as filled grey bands whose heights are proportional to the relative number of transcripts by RT-qPCR. (b) Codon adaptation index (CAI) of E. coli genes, genes in wild-type &#x3A6;X174, and deoptimized &#x3A6;X174 genes. Genes highly expressed in E. coli are rescaled to the secondary y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&#x3A6;X174 genome organization and capacity for deoptimization relative to host genes.&lt;/span&gt; (a) Genes on the &#x3A6;X174 genome are shown as empty black boxes. Recoded regions are shown as filled blue boxes. These fragments are named consecutively (e.g., A1, A2, A3, A4, F1, F2, F3,&#x2026;). Approximate transcript expression levels are shown as filled grey bands whose heights are proportional to the relative number of transcripts by RT-qPCR. (b) Codon adaptation index (CAI) of &lt;em&gt;E. coli&lt;/em&gt; genes, genes in wild-type &#x3A6;X174, and deoptimized &#x3A6;X174 genes. Genes highly expressed in &lt;em&gt;E. coli&lt;/em&gt; are rescaled to the secondary y-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F2.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F2.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <span class="caption-title">ΦX174 genome organization and capacity for deoptimization relative to host genes.</span><p id="p-15" class="first-child">(a) Genes on the ΦX174 genome are shown as empty black boxes. Recoded regions are shown as filled blue boxes. These fragments are named consecutively (e.g., A1, A2, A3, A4, F1, F2, F3,…). Approximate transcript expression levels are shown as filled grey bands whose heights are proportional to the relative number of transcripts by RT-qPCR. (b) Codon adaptation index (CAI) of <em>E. coli</em> genes, genes in wild-type ΦX174, and deoptimized ΦX174 genes. Genes highly expressed in <em>E. coli</em> are rescaled to the secondary y-axis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Codon deoptimization of ΦX174 genes reduces viral fitness</h3><p id="p-16">We codon deoptimized the genes of ΦX174 by recoding nearly all synonymous residues to the least preferred <em>E. coli</em> codons. We did not recode regions of genes that overlapped with other genes nor the first six residues of each gene since these residues are known to have strong effects on gene expression (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>). Four of the six recoded variants were less fit than wild type ΦX174 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">fig. 3a</a>). The construct containing the fully deoptimized G gene could not be recovered, even after growing the strain overnight on the susceptible host cells in an attempt to obtain a recovery mutation. Recoding highly expressed genes (J, F, and G) resulted in larger fitness decreases than recoding lowly expressed genes (A and H). Although the number of variants built was small, the fitness effects of deoptimization were correlated to the proportion of codons edited and the CAI of the recoded genes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">fig. 3b</a> and supplementary table S2).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness effects of deoptimizing &#x3A6;X174 genes. (a) The fitness of wild type and deoptimized &#x3A6;X174 strains containing recoded genes are shown in replicate. Means and standard error bars are shown in black. Fitness values that are significantly different from wild type are indicated with asterisks. (b) Fitness is plotted against measures that potentially explain fitness decreases. Protein copy number is the number of proteins needed to virion assembly (see Table 1). Fitness is the number of doublings per hour (log2 of the ratio of the phage concentration at 60 min divided by the phage concentration at time zero). The fitness of wild-type is indicated with a grey horizontal line. Structural proteins are shown as empty circles. Deoptimizing gene G yielded no viable phage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Fitness effects of deoptimizing &#x3A6;X174 genes.&lt;/span&gt; (a) The fitness of wild type and deoptimized &#x3A6;X174 strains containing recoded genes are shown in replicate. Means and standard error bars are shown in black. Fitness values that are significantly different from wild type are indicated with asterisks. (b) Fitness is plotted against measures that potentially explain fitness decreases. Protein copy number is the number of proteins needed to virion assembly (see Table 1). Fitness is the number of doublings per hour (log2 of the ratio of the phage concentration at 60 min divided by the phage concentration at time zero). The fitness of wild-type is indicated with a grey horizontal line. Structural proteins are shown as empty circles. Deoptimizing gene G yielded no viable phage.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F3.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F3.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <span class="caption-title">Fitness effects of deoptimizing ΦX174 genes.</span><p id="p-17" class="first-child">(a) The fitness of wild type and deoptimized ΦX174 strains containing recoded genes are shown in replicate. Means and standard error bars are shown in black. Fitness values that are significantly different from wild type are indicated with asterisks. (b) Fitness is plotted against measures that potentially explain fitness decreases. Protein copy number is the number of proteins needed to virion assembly (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Fitness is the number of doublings per hour (log2 of the ratio of the phage concentration at 60 min divided by the phage concentration at time zero). The fitness of wild-type is indicated with a grey horizontal line. Structural proteins are shown as empty circles. Deoptimizing gene G yielded no viable phage.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Reconstructing a combinatorial fitness landscape for deoptimized genes</h3><p id="p-18">We segmented ΦX174 genes into gene fragments (<a id="xref-fig-2-4" class="xref-fig" href="#F2">fig. 2</a>) and measured the fitness of all of the possible within-gene combinations of deoptimized fragments (<a id="xref-fig-4-1" class="xref-fig" href="#F4">fig. 4</a> and supplementary table S3). Of the 12 deoptimized strains with only one deoptimized fragment, only 6 have fitness values below wild type. The moderate fitness effects of these partially recoded genes allowed us to observe how deleterious effects combine as compared to the more deleterious effects observed when deoptimizing complete genes. As additional deoptimized fragments are joined, the fitness of the resulting viruses decrease (<a id="xref-fig-4-2" class="xref-fig" href="#F4">fig. 4b</a>). In most cases combining deoptimized fragments results in less fit viruses. The exception is gene A where instances of sign epistasis are observed. Specifically, the average fitness of A1+A3, A2+A3, A1+A3+A4, A1+A2+A4, A2+A3+A4, and A1+A2+A3+A4 are all higher than at least one of their constituent fitness values (<a id="xref-fig-4-3" class="xref-fig" href="#F4">fig. 4b</a>). To further investigate how deleterious effects combine, we employed a statistical framework for calculating the best-fitting model of epistasis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness of &#x3A6;X174 when deoptimized gene fragments are combinatorially joined. The fitness of variants containing one deoptimized fragment (a) and all possible within-gene variants (b) was measured and compared to wild type (grey horizontal line). In (b), fragment lengths are drawn to scale. Filled colors indicate the deoptimized fragments while unfilled blocks indicate wild-type fragments. In both (a) and (b), fitness is shown as log2 fold increase in the number of phage per hour." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Fitness of &#x3A6;X174 when deoptimized gene fragments are combinatorially joined.&lt;/span&gt; The fitness of variants containing one deoptimized fragment (a) and all possible within-gene variants (b) was measured and compared to wild type (grey horizontal line). In (b), fragment lengths are drawn to scale. Filled colors indicate the deoptimized fragments while unfilled blocks indicate wild-type fragments. In both (a) and (b), fitness is shown as log2 fold increase in the number of phage per hour.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F4.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F4.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <span class="caption-title">Fitness of ΦX174 when deoptimized gene fragments are combinatorially joined.</span><p id="p-19" class="first-child">The fitness of variants containing one deoptimized fragment (a) and all possible within-gene variants (b) was measured and compared to wild type (grey horizontal line). In (b), fragment lengths are drawn to scale. Filled colors indicate the deoptimized fragments while unfilled blocks indicate wild-type fragments. In both (a) and (b), fitness is shown as log2 fold increase in the number of phage per hour.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Fitting models of epistasis to combinatorial fitness data</h3><p id="p-20">The combinatorial network of genotypes that we generated in this work can be analyzed by applying simple models of epistasis (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>) to determine how the effects of mutations combine. We fit the data for genes A, F and H to three basic models—additive, multiplicative, and stickbreaking—which gave rise to no, antagonistic, and synergistic epistasis respectively (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">figure 1</a> and (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>)). In fitting the three models, we conducted two analyses for each gene: one of absolute fit where we assess if the data is consistent with each model individually, and one of relative fit wherein one of the three models is assumed to be correct. The results from this analysis were not highly conclusive, but suggest the nature of epistasis is heterogeneous across different genes. For genes F and H, none of the three models could be rejected based on absolute goodness of fit (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">table 2</a>). For gene F, the additive model provides the best fit to the data. For gene H, stickbreaking gives the best fit (R<sup>2</sup>=0.885), consistent with synergistic epistasis. This is visually clear in <a id="xref-fig-4-4" class="xref-fig" href="#F4">figure 4b</a>, where the fully recoded gene H (3 recoded fragments) has far lower fitness than one would expect based on the individually recoded fragments—all of which were basically neutral.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1148868/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1148868/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1148868" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Models of epistasis fit to combinatorial fitness data.</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">Visually, a pattern of antagonistic epistasis was observed for gene A, as several of the variants with two recoded fragments had fitnesses as low or even slightly lower than the three- and four-recoded fragment variants (<a id="xref-fig-4-5" class="xref-fig" href="#F4">fig. 4b</a>). Indeed, the additive and stickbreaking models were rejected for gene A based on absolute goodness of fit (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">table 2</a>). The multiplicative model, with its antagonistic pattern of epistasis, was not rejected, but the p-value was marginal (p=0.066). Strong antagonistic epistasis is occuring for gene A—even stronger than that predicted under the multiplicative model. This was revealed by regressing background fitness against fitness effect (<a id="xref-fig-5-1" class="xref-fig" href="#F5">fig. 5a</a> and supplementary figs. S1-S3). When effects were measured as differences (the additive model), negative/positive slopes corresponded to antagonistic/synergistic epistasis. Under the correct model, no correlation exists and slopes are expected to be random deviations around zero. Under the additive model (<a id="xref-fig-5-2" class="xref-fig" href="#F5">fig 5a</a>), a clear pattern across all four recoded fragments where the effect of the fragment becomes more strongly deleterious on higher fitness backgrounds (negative regression slopes) was observed. When the p-values of the individual fragments were combined, their result is significant (supplementary fig. S1). The analogous regression under the multiplicative model was less extreme, but even here the slopes were consistently negative, indicating a level of antagonistic epistasis beyond multiplicative (supplementary fig. S2).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness effect of each recoded fragment on all possible backgrounds. (a) Regressions of each recoded fragment&#x2019;s fitness effect under the additive model against background fitness. The y=0 line is a perfect fit to the additive model with no residual effects of background. The p-value for each linear regression is shown. The overall fit of every model for each gene is shown in Table 3. In (b), the effect a fragment has on CAI is shown against its effect on fitness from the background to the background plus the recoded fragment (A2 -&gt; A1+A2, A2+A3 -&gt; A1 + A2+A3) is shown for all fragments. Slight point jitter was used for visualization." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Fitness effect of each recoded fragment on all possible backgrounds.&lt;/span&gt; (a) Regressions of each recoded fragment&#x2019;s fitness effect under the additive model against background fitness. The y=0 line is a perfect fit to the additive model with no residual effects of background. The p-value for each linear regression is shown. The overall fit of every model for each gene is shown in Table 3. In (b), the effect a fragment has on CAI is shown against its effect on fitness from the background to the background plus the recoded fragment (A2 -&gt; A1+A2, A2+A3 -&gt; A1 + A2+A3) is shown for all fragments. Slight point jitter was used for visualization.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F5.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F5.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5</span> <span class="caption-title">Fitness effect of each recoded fragment on all possible backgrounds.</span><p id="p-22" class="first-child">(a) Regressions of each recoded fragment’s fitness effect under the additive model against background fitness. The y=0 line is a perfect fit to the additive model with no residual effects of background. The p-value for each linear regression is shown. The overall fit of every model for each gene is shown in Table 3. In (b), the effect a fragment has on CAI is shown against its effect on fitness from the background to the background plus the recoded fragment (A2 -&gt; A1+A2, A2+A3 -&gt; A1 + A2+A3) is shown for all fragments. Slight point jitter was used for visualization.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Correlating codon deoptimization to combinatorial fitness data</h3><p id="p-23">Ultimately, our goal was to correlate changes in genomic properties (<em>e.g.</em> codon preference) to changes in viral fitness. The most straightforward method of analysis would be to regress the two measurements, and indeed the fitness of deoptimized variants was linearly correlated to CAI (R<sup>2</sup>=0.36, p=2E-16, <a id="xref-fig-6-1" class="xref-fig" href="#F6">fig. 6a</a>). However, it is worth noting that the data points used in this regression were not independent because the deoptimized fragments were combined to achieve higher levels of deoptimization. Our combinometic method of making variants also allowed us to correct for the cumulative fitness effect of combined fragments by calculating the effect of adding any particular fragment to different backgrounds (<a id="xref-fig-5-3" class="xref-fig" href="#F5">fig. 5b</a>). For example, the effect of deoptimizing the F1 fragment was measured by comparing the fitness values of WT to deoptimized F1 (20-21 = −1), or F2 to F1+F2 (18-20 = −2), or F3 to F1+F3 (14-16 = −2), or F2+F3 to F1+F2+F3 (8-10 = −2). Thus, deoptimizing F1 resulted in an average fitness effect of about -2. This background subtraction approach corrected for the non-independence of data points in regressions between change in fitness from WT and change in CAI from WT.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness of recoded viruses correlates with codon usage bias and mRNA folding stability. (a) Codon usage bias (CAI) compared to fitness for viruses optimized and deoptimized in genes A, F, G, and H. Fitness and CAI of wild type are indicated with grey horizontal and vertical lines. Points to the right of these lines are optimized. Points to the left are deoptimized, (b) Viral fitness compared to mRNA folding stability (mfold). Wild type values are indicated with grey horizontal and vertical lines. Less stable transcripts (deoptimized genes) have less negative values and are right of wild type. More stable transcripts (optimized genes) have more negative values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116886456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Fitness of recoded viruses correlates with codon usage bias and mRNA folding stability.&lt;/span&gt; (a) Codon usage bias (CAI) compared to fitness for viruses optimized and deoptimized in genes A, F, G, and H. Fitness and CAI of wild type are indicated with grey horizontal and vertical lines. Points to the right of these lines are optimized. Points to the left are deoptimized, (b) Viral fitness compared to mRNA folding stability (mfold). Wild type values are indicated with grey horizontal and vertical lines. Less stable transcripts (deoptimized genes) have less negative values and are right of wild type. More stable transcripts (optimized genes) have more negative values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F6.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F6.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/11/2020.02.10.942847/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6</span> <span class="caption-title">Fitness of recoded viruses correlates with codon usage bias and mRNA folding stability.</span><p id="p-24" class="first-child">(a) Codon usage bias (CAI) compared to fitness for viruses optimized and deoptimized in genes A, F, G, and H. Fitness and CAI of wild type are indicated with grey horizontal and vertical lines. Points to the right of these lines are optimized. Points to the left are deoptimized, (b) Viral fitness compared to mRNA folding stability (mfold). Wild type values are indicated with grey horizontal and vertical lines. Less stable transcripts (deoptimized genes) have less negative values and are right of wild type. More stable transcripts (optimized genes) have more negative values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">When we applied this correction, we observed a wide variance in fitness effects (<a id="xref-fig-5-4" class="xref-fig" href="#F5">fig. 5b</a>). For example, in some backgrounds, adding deoptimized H1 reduced fitness by only ~1 doublings/hr. In other backgrounds, H1 reduced fitness by ~8 doublings per hour. Despite this variation, there is a good correlation between change in CAI and change in fitness (R<sup>2</sup>=0.58, p=4.8E-10, <a id="xref-fig-5-5" class="xref-fig" href="#F5">fig. 5</a> and supplementary table S4). Applying this background correction indicates that only a portion of the fitness changes can be explained by changes in codon usage bias. This is particularly true for genes A and H. Fragments in gene F seem to have more consistent effects (<a id="xref-fig-5-6" class="xref-fig" href="#F5">fig. 5b</a>).</p></div><div id="sec-11" class="subsection"><h3>How different synonymous features correlate with fitness</h3><p id="p-26">We replaced ΦX174 codons with less preferred codons without consideration for how alterations might affect other features in the genome. As mentioned in the introduction, many such features may be under selection. To investigate unintended consequences of codon deoptimization, we calculated numerous genome characteristics to see if any correlate with the fitness decreases observed in deoptimized fragments (supplementary table S1). We included many different measures of codon usage bias (CAI, tAI, ITE, etc), codon pair bias (CPB), frequency of Shine-Dalgarno motifs, mRNA folding stability, as well as simply the number of changes made. The best predictor of fitness is the folding stability of the codon deoptimized mRNA (R<sup>2</sup>=0.67, p.adj=0.02), which performed significantly better than the best measure of codon usage bias which was CAI using the Xia2007 method (ΔAIC=15.5). This correlation is easily observed when mRNA stability values are plotted against change in fitness (<a id="xref-fig-6-2" class="xref-fig" href="#F6">fig. 6b</a>). We were interested to see if the correlation between genomic features like CAI and fitness held up even when features were optimized, so we replaced ΦX174 codons with codons frequently used in <em>E. coli</em> (supplementary table S2), expecting to observe beneficial fitness effects. However, in all cases, fitness was either unaffected or reduced (<a id="xref-fig-6-3" class="xref-fig" href="#F6">fig. 6a</a> and supplementary table S3). Because of this, if these optimized constructs are included in the regression models, the number of sites changed and fraction of gene edited become the metrics that best predict fitness from genomic measures. We observed a peak-shaped fitness landscape when combining the optimized and deoptimized data set; this is discussed below.</p></div></div><div class="section" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Patterns of synonymous codon usage biases</h3><p id="p-27">Synonymous codon usage biases are present in genomes across the tree of life (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). We often think of these biases as having little consequence during the natural evolution of organisms because the strength of selection acting on any one synonymous mutation is generally weak. Nevertheless, the presence of biases shows that selection acts with sufficient strength to maintain them in the face of genetic drift. The prevailing theories on the preservation of codon biases suggest that codon choice is primarily driven by selection on translational speed and mRNA stability (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>). The enrichment of codons that use abundant tRNAs in highly expressed genes points towards a model where translation speed is correlated to tRNA abundance. We find that the most highly expressed ΦX174 gene ranks second best in its use of host-preferred codons, but only marginally better than the average <em>E. coli</em> gene (<a id="xref-fig-2-5" class="xref-fig" href="#F2">fig. 2</a>). That the majority of ΦX174 genes fall below average in preferred codon usage bias according to <em>E. coli</em> usage patterns is not surprising—many viruses do not favor the most preferred host codons. A myriad of reasons may explain why viral genes do not evolve to their full codon usage potential. Assuming a selective explanation, it could be that codon usage controls the stoichiometric ratio between viral genes (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>), temporally regulates gene expression (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">73</a>, <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>), facilitates co-translational folding (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>), dampens protein expression to avoid host immune responses (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">64</a>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>), or is limited by other compositional features. Comparative genomics between ΦX174, G4, α3, and ΦMH2K suggest that the codon usage biases are a conserved feature of these microviruses (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>). Our data cannot point towards any one of these explanations, although we note that the high-copy number proteins of ΦX174 tend to have higher CAI values and are more easily deoptimized.</p></div><div id="sec-14" class="subsection"><h3>Recoding of ΦX174</h3><p id="p-28">Synonymous mutations that disrupt important genome compositional features can have substantial phenotypic effects (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a>–<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>). For example, of 48 cefotaxime resistance mutations in TEM-1 β-lactamase that were identified from a randomly mutagenized pool of variants, 10 were silent (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>). One mutation increased nitrocefin hydrolysis by 7.5-fold, probably by increasing the amount of functional TEM-1 (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>). Our work on ΦX174 also shows that synonymous mutations can have massive fitness effects, as the virus can be completely attenuated by recoding with non-preferred, synonymous codons. Although we introduced many synonymous mutations in each recoded ΦX174 strain, the largest observed fitness impact of any single deoptimized fragment contained only 29 synonymous codon changes. These 29 synonymous changes resulted in a 50% decrease in fitness, which is a decrease of about 10 doublings per hour or about a 1000-fold change in the number of offspring. Aside from using codons that ensure the use of preferred tRNAs, organisms must also balance codon usage with codon pair biases, di-nucleotide preferences, mRNA structural motifs, restriction enzyme site avoidances, etc. Selection can act on many types of genome compositional features. In recoding ΦX174 with better or worse codons, we observed correlative changes in other compositional features (<a id="xref-fig-6-4" class="xref-fig" href="#F6">fig 6</a>, supplementary table S4).</p><p id="p-29">A good predictor of translational throughput is mRNA secondary structure (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>). Tightly folded mRNAs are traversed by the ribosome more slowly than unfolded mRNAs, although faster translation is not necessarily beneficial. mRNA structure is used to slow translation where pausing is needed, most notably before protein structures that require co-translational folding (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a>) and at the 5’ end of a transcript, where proper loading of a mRNA onto a ribosome foreshadows correct and efficient translation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). Our data support the importance of mRNA folding stability for organismal fitness, as the stability of recoded ΦX174 genes is correlated with ΦX174 fitness (<a id="xref-fig-6-5" class="xref-fig" href="#F6">fig. 6b</a>). Interestingly, the optimized ΦX174 strains almost always have increased mRNA folding stabilities and decreased fitness values, suggesting the folding stability of each mRNA may be near optimal in the wild-type virus. While it is unclear to us why the optimized strains have increased mRNA folding stabilities, departure from wild-type—either more or less stable—results in fitness declines. This suggests that wild-type ΦX174 sits at a peak in its fitness landscape.</p></div><div id="sec-15" class="subsection"><h3>Combining mutations and epistasis</h3><p id="p-30">Genomes accumulate deleterious mutations over time. The detrimental effect of accumulating deleterious mutations is prevented by sex, recombination, and purifying selection which purge them from populations. In contrast to beneficial mutations which generally combine with diminishing returns (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>, <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>), the way that the individual effects of deleterious mutations combine is less well understood. Among many issues preventing these predictions is a paucity of empirical phenotype data for networks of deleterious mutations (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a>–<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a>). This is especially true for combinations beyond two. A number of studies have investigated epistasis among pairs or triple sets of deleterious mutations, but the findings are mixed (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>–<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a>). Sometimes the combined effect is the sum of the individual effects (additive/no epistasis), sometimes it is less than predicted from the individual effects (antagonistic/positive epistasis) (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a>, <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a>), and sometimes it is more than predicted (synergistic/negative epistasis) (<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a>). Among these three scenarios, antagonistic epistasis seems to be most common (<a id="xref-ref-104-2" class="xref-bibr" href="#ref-104">104</a>, <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a>). If one considers sign epistasis to be an extreme form of antagonistic epistasis, then more support is garnished for this model as a number of studies on deleterious mutations uncover some degree of deleterious mutations becoming beneficial in combination (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">112</a>). Our data is novel in that it builds several complete combinatorial networks of deleterious mutations, but it does lack large samples sizes. Of the networks we built, only the one for gene A had a sufficient number of data points to reject poorly fitting models. For gene A, strong antagonistic epistasis was observed. Johnson <em>et al.</em> recently found that this type of epistasis is common among loss-of-function mutations in yeast. They called it “increasing cost epistasis” because a given deleterious mutation tends to have a greater cost on more fit backgrounds (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">113</a>). For genes F and H, no models can be rejected, but the data suggest that mutations in gene F are additive while mutations in gene H combine synergistically. For the purposes of building synonymously recoded viruses for vaccines, it is promising to see gene A displaying antagonistic epistasis. With this type of epistasis where fitness flattens out, less trial and error should be required to build attenuated, but still viable, viruses.</p></div><div id="sec-16" class="subsection"><h3>Synonymous virus genome recoding for vaccines</h3><p id="p-31">Synonymously recoding viral genomes has a potentially useful application in making live-attenuated vaccines. The antigenicity of synonymously attenuated viruses is maintained because the viral protein sequences remain unchanged. However, the process of choosing how many codons to change and what type of synonymous changes to implement is currently done without guiding principles. If fact, which synonymous features most strongly affect recoded viruses is debated (<a id="xref-ref-79-4" class="xref-bibr" href="#ref-79">79</a>, <a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">114</a>, <a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">115</a>, <a id="xref-ref-115-2" class="xref-bibr" href="#ref-115">115</a>). Of the dozens of viruses that have been deoptimized, a minority of them measure compositional features different from the one being directly targeted for deoptimization. At the very least, we suggest that researchers must measure a variety of compositional features when designing deoptimized constructs. A better approach would be to develop construct design software that supports researchers to engineer deoptimized viral genes (see (<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">116</a>) for an example using codon shuffling). This software exists for optimizing genes for expression in host cells (<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">117</a>, <a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">118</a>) and may be co-opted for deoptimization purposes. In our experiments we made no effort to isolate changes to one type of compositional feature. In exploring this possibility, we found it difficult to generate sufficient deoptimization of one feature (CAI) while keeping other features (mfold, CpB, Shine-Dalgarno frequency) unchanged. Recently, Paff et al. demonstrated that promoter ablation attenuated T7 bacteriophage in a predictable manner (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">119</a>). Combining these edits with previous codon deoptimized strains showed increased attenuation. Targeting intragenic attenuating mutations is a promising way to test how deleterious effects combine without the added complication of trying to isolate correlative compositional features.</p><p id="p-32">Like many other studies, our data showed virus codon deoptimization is an effective way to generate attenuated viruses. In cell culture and animal studies, deoptimized viruses were shown to protect from viral challenge and were stable over small numbers of passages (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">60</a>). However, much concern remains about the potential for attenuated viruses to recover virulence. It is therefore important to understand how many and what types of synonymous mutations can be made to viral genomes without completely ablating their ability to replicate in host cells. What viral genes should be attenuated? How many attenuating mutations should be made to the genome? What synonymous features should be targeted for deoptimization? In most studies to date, a limited number of deoptimized constructs (usually structural proteins) were tested. We showed that fitness decreases can be obtained by deoptimizing many of the ΦX174 genes, indicating that nonstructural genes may also be good targets for attenuation. One approach to avoid evolutionary reversion might be recoding multiple genes or entire viral genomes, balancing optimization and deoptimization to maintain sufficient virulence while increasing the genetic distance to wild type. This strategy could prevent recovery by mutation or by recombination with wild-type viruses. However, our work suggests that the effects of recoding will not be uniform across a genome. We found that the attenuating effects of recoding and the nature of epistatic interactions from combining fragments differ dramatically between genes.</p></div></div><div class="section" id="sec-18"><h2 class="">Methods and Methods</h2><div class="section" id="sec-19"><h3>Bacterial cultures and phage stocks</h3><p id="p-34">A laboratory strain of bacteriophage ΦX174 (GenBank accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AF176034&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom">AF176034</a>) was used in this study. All experiments were carried out using <em>E. coli</em> C as a host in modified Luria-Bertani media (10 g/l tryptone, 5 g/l Bacto yeast extract, 10 g/l NaCl, 2 mM CaCl2).</p></div><div class="section" id="sec-20"><h3>Synthetic ΦX174 genomes</h3><p id="p-35">A phage assembly platform for ΦX174 was devised following (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">120</a>). The ΦX174 chromosome was divided into 15 genomic fragments designed to avoid host cell toxicity by separating genes from their promoters and breaking large genes into multiple segments. Each segment is flanked by unique five nucleotide overlaps of WT ΦX174 sequence so that they can be amplified from the ancestral ΦX174 using PCR primers designed to incorporate terminal BsmB1 restriction sites. Amplicons were cloned into pCR2.1 using the Invitrogen TOPO TA cloning system (Life Technologies, Grand Island, NY). We pooled plasmid DNA containing all 15 of the phage DNA fragments in equimolar amounts and digested them with BsmB1 (Fermentas Fast Digest, Life Technologies, Grand Island, NY) for 30 minutes to 1 hour at 37°C. The digested plasmids were subjected to agarose gel electrophoresis for 10 to 15 minutes using a 1.2% agarose gel to separate the vector from the inserts. The inserts were excised from the gel, purified using the GeneJET gel extraction kit (Fermentas), ligated overnight at 14°C with T4 DNA ligase (Promega Corporation, Madison, WI), and transformed by electroporation into 100 μl competent <em>E. coli</em> C cells. The transformation mix was resuspended with 1 ml of ΦLB and either plated immediately or incubated for about 20 minutes at 37°C to allow for one viral burst. The ΦLB was added to 3 ml of ΦLB top agar and plated onto a ΦLB agar plate. After four to five hours of incubation at 37°C, recombinant phage plaques were visible and plates were removed from the incubator. To verify that the recombinant phage encoded the intended sequence, the resulting phage genome was sequenced in its entirety as previously described (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">121</a>).</p></div><div class="section" id="sec-21"><h3>Codon deoptimization of ΦX174</h3><p id="p-36">Codon deoptimized and optimized fragments were synthesized by in-house at the University of Texas at Austin, Applied Research Laboratories’ Gene Synthesis Facility or purchased from Biomatik USA, LLC (Wilmington, DE) according to the codon usage of five representative <em>E. coli</em> genomes (<em>E. coli</em> 536, <em>E. coli</em> UT 189, <em>E. coli</em> O157:H7 str. Sakai, <em>E. coli</em> O157:H7EDL933, and <em>E. coli</em> CFT073). Codon usage was calculated by averaging each codon’s usage frequency in CDS of these <em>E. coli</em> genomes. Each wild-type ΦX174 codon that could be changed to a more or less frequently used codon was changed. Some coding regions were not modified; these were either identified a priori or excluded based on failure of a genome construct to yield viable phage. Unmodified regions in our final dataset include overlapping reading frames, promoter regions that occurred within other reading frames, and the region from A4299-A4328 which encodes the ΦX174 origin of replication. In addition, approximately 50 bases surrounding the initiation codon (AUG) of each gene were left unmodified to assure efficient translation initiation. Unsuccessful attempts to create live virus using synthesized fragments were repeated at least three times, then passaged in liquid culture for 24 hours to allow for recovery mutants to arise.</p></div><div class="section" id="sec-22"><h3>Measuring viral fitness</h3><p id="p-37">Fitness assays and fitness calculations were performed as previously described (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>). The assay is a determination of growth rate at low MOI in 10 mL LB and is carried out at 37°C. Host cells were prepared by growing to ~10<sup>8</sup> cells/mL and aliquoted into 8.5 mL of warm LB just prior to adding phage. Phage fitness is expressed as the log2 fold increase in the total number of phage per hour. All measurements were done in triplicate. At 45min, virus titers were done on LB-agar plates with 0.3% top agar.</p></div><div class="section" id="sec-23"><h3>Calculating genome statistics</h3><p id="p-38">Calculations of genome statistics were done as follows: Codon Adaptation Index (CAI) was calculated using the seqinr cai() function in R which uses the Sharp and Li 1987 method and the <em>E. coli</em> codon usage table. An alternative form of CAI was also calculated using an updated method outlined in Xia 2007. The number of effective codons (Nc) was calculated following (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">85</a>). The Index of translational elongation (I<sub>TE</sub>) was calculated according to (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">83</a>). The Starvation codon adaptation index (sCAI) was calculated according to (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">86</a>). This measure scores genes by how susceptible their codons are to a scarcity of amino-acylated tRNAs. The number of Shine-Dalgarno motifs with binding values &gt; 0 were counted in any given sequence and included in the model. The strength of the Shine-Dalgarno sequences was also considered by including a per-codon average binding strength of all Shine-Dalgarno motifs in a gene (sum of binding strengths/gene gene length).</p><p id="p-39">For <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, all protein coding sequences were parsed from the <em>E. coli</em> O103:H2 genome (GCA_000010745.1) and CAI was calculated as described above. The list of the most highly expressed genes is from (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">122</a>).</p></div><div class="section" id="sec-24"><h3>Analysis of Epistasis</h3><p id="p-40">We analyzed the network fitness data from genes A, F, and H using the Stickbreaker R package (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">88</a>) and functions therein. This package fits such data to the additive, multiplicative, and stickbreaking models. While the additive and multiplicative models assume a mutation (a recoded block in this context) changes background fitness by a difference or a factor (respectively), the stickbreaking model assumes a mutation’s effect is scaled by the distance between the background and a fitness boundary. For fitting the stickbreaking model, we could not obtain reasonable estimates for the fitness boundary from the data (beneficial mutations are much more useful for estimating the boundary than deleterious ones). Instead, we assumed a fitness boundary of 24.5 dbl/hr (wildtype has fitness of 20.5); using a larger fitness boundary simply makes the sticking model more and more like the additive model. Relative fit (posterior probabilities) was calculated following the methods in (<a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">88</a>). To estimate the absolute goodness of fit, we used parametric bootstrap. Specifically, for each gene and each model, we extracted the observed effect of each block on each background it appeared on. For each recoded fragment, we then regressed the background’s fitness against the fitness effect by fitting a simple linear model and obtained a p-value associated with a slope of zero (illustrated for gene A in <a id="xref-fig-5-7" class="xref-fig" href="#F5">fig 5</a>). When a model is correct, the slope of this line is expected to be zero. For a given gene and model, we take the sum of the logs of the p-values, P<sub>obs</sub>, as a summary statistic. Noticed that the data points involved in these regressions were not independent and, as such, the p-values were not valid. We accounted for this by simulating 10,000 datasets (using the estimated coefficient of each block and the estimated Gaussian noise parameter, that captures both experimental noise and variation from model expectations). For each simulated dataset, we repeated the regression for each block and combined across blocks to obtain a summary P<sub>sim</sub>. Across 10,000 simulations, this generated the approximate distribution of P when the model is correct. We then located P<sub>obs</sub> in this distribution and calculated the p-value as the proportion of simulations where P<sub>sim</sub> &lt; P<sub>obs</sub>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">This study was supported by the National Institutes of Health grants P20 GM104420 and GM076040. We thank the Institute for Bioinformatic and Evolutionary Studies (IBEST) Genomics Resources Core for assistance with DNA sequencing. Sequences for all recoded ΦX174 genes are available on Genbank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN045299&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom">MN045299</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN045346&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom">MN045346</a>.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.1" data-doi="10.1038/nrg2899"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D12%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">The selection-mutation-drift theory of synonymous codon usage</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>:<span class="cit-fpage">897</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D897%26rft.epage%253D907%26rft.atitle%253DThe%2Bselection-mutation-drift%2Btheory%2Bof%2Bsynonymous%2Bcodon%2Busage.%26rft_id%253Dinfo%253Apmid%252F1752426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvODk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kucukyildirim</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berne</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kysela</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">WR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muscarella</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brun</span>  <span class="cit-name-given-names">YV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Evolutionary determinants of genome-wide nucleotide composition</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr> <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">YD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes</span>. <abbr class="cit-jnl-abbrev">G3 Genes Genomes Genet</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">1247</span>–<span class="cit-lpage">1252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%2BGenomes%2BGenet%26rft.volume%253D5%26rft.spage%253D1247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.5" data-doi="10.1073/pnas.1205683109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">YD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A selective force favoring increased G+C content in bacterial genes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">14504</span>–<span class="cit-lpage">14507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1205683109%26rft_id%253Dinfo%253Apmid%252F22908296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM2LzE0NTA0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.6" data-doi="10.1073/pnas.1518976113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">ZR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">NT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">GM</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Differences in codon bias and GC content contribute to the balanced expression of TLR7 and TLR9</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">113</span>:<span class="cit-fpage">E1362</span>–<span class="cit-lpage">E1371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518976113%26rft_id%253Dinfo%253Apmid%252F26903634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzEwL0UxMzYyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giallonardo</span>  <span class="cit-name-given-names">FD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlub</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Dinucleotide composition in animal RNA viruses is shaped more by virus family than host species</span>. <abbr class="cit-jnl-abbrev">J Virol JVI</abbr>.02381-16.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.8" data-doi="10.1093/nar/gku075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witteveldt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">4527</span>–<span class="cit-lpage">4545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku075%26rft_id%253Dinfo%253Apmid%252F24470146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24470146&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.9" data-doi="10.7554/eLife.29112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fros</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alshaikhahmed</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Passchier</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">CpG and UpA dinucleotides in both coding and non-coding regions of echovirus 7 inhibit replication initiation post-entry</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e29112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De29112%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.29112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.29112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaunt</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">LN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">MQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Highton</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beard</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutia</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Digard</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Elevation of CpG frequencies in influenza A genome attenuates pathogenicity but enhances host response to infection</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e12735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De12735%26rft_id%253Dinfo%253Apmid%252F26878752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26878752&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.11" data-doi="10.1128/JVI.00508-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Genetic Inactivation of Poliovirus Infectivity by Increasing the Frequencies of CpG and UpA Dinucleotides within and across Synonymous Capsid Region Codons</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">83</span>:<span class="cit-fpage">9957</span>–<span class="cit-lpage">9969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00508-09%26rft_id%253Dinfo%253Apmid%252F19605476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xOS85OTU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.12" data-doi="10.1074/jbc.270.39.22801"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">GW</span></span>. <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Codon Pair Utilization Biases Influence Translational Elongation Step Times</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">270</span>:<span class="cit-fpage">22801</span>–<span class="cit-lpage">22806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DIrwin%26rft.auinit1%253DB.%26rft.volume%253D270%26rft.issue%253D39%26rft.spage%253D22801%26rft.epage%253D22806%26rft.atitle%253DCodon%2BPair%2BUtilization%2BBiases%2BInfluence%2BTranslational%2BElongation%2BStep%2BTimes%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.270.39.22801%26rft_id%253Dinfo%253Apmid%252F7559409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzAvMzkvMjI4MDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.13" data-doi="10.1186/1471-2164-9-463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tats</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenson</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Preferred and avoided codon pairs in three domains of life</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTats%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D463%26rft.epage%253D463%26rft.atitle%253DPreferred%2Band%2Bavoided%2Bcodon%2Bpairs%2Bin%2Bthree%2Bdomains%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-463%26rft_id%253Dinfo%253Apmid%252F18842120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18842120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.14" data-doi="10.1073/pnas.86.10.3699"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gutman</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">GW</span></span>. <span class="cit-pub-date">1989</span>. <span class="cit-article-title">Nonrandom utilization of codon pairs in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">86</span>:<span class="cit-fpage">3699</span>–<span class="cit-lpage">3703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGutman%26rft.auinit1%253DG.%2BA.%26rft.volume%253D86%26rft.issue%253D10%26rft.spage%253D3699%26rft.epage%253D3703%26rft.atitle%253DNonrandom%2Butilization%2Bof%2Bcodon%2Bpairs%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.10.3699%26rft_id%253Dinfo%253Apmid%252F2657727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODYvMTAvMzY5OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.15" data-doi="10.1016/j.cell.2016.05.070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gamble</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brule</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grayhack</span>  <span class="cit-name-given-names">EJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Adjacent Codons Act in Concert to Modulate Translation Efficiency in Yeast</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>:<span class="cit-fpage">679</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D679%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.070%26rft_id%253Dinfo%253Apmid%252F27374328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27374328&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.16" data-doi="10.1101/gr.GR-1531RR"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">EPC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viari</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Evolutionary Role of Restriction/Modification Systems as Revealed by Comparative Genome Analysis</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">946</span>–<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.GR-1531RR%26rft_id%253Dinfo%253Apmid%252F11381024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjExLzYvOTQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.17" data-doi="10.1016/0959-437X(93)90004-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">BR</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">The accessory genetic elements of bacteria: existence conditions and (co)evolution</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">849</span>–<span class="cit-lpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DLevin%26rft.auinit1%253DB.%2BR.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D849%26rft.epage%253D854%26rft.atitle%253DThe%2Baccessory%2Bgenetic%2Belements%2Bof%2Bbacteria%253A%2Bexistence%2Bconditions%2Band%2B%2528co%2529evolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0959-437X%252893%252990004-9%26rft_id%253Dinfo%253Apmid%252F8118209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0959-437X(93)90004-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8118209&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.18" data-doi="10.1093/nar/20.6.1363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">AM</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Statistical analyses of counts and distributions of restriction sites in DNA sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">1363</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKarlin%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D1363%26rft.epage%253D1370%26rft.atitle%253DStatistical%2Banalyses%2Bof%2Bcounts%2Band%2Bdistributions%2Bof%2Brestriction%2Bsites%2Bin%2BDNA%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.6.1363%26rft_id%253Dinfo%253Apmid%252F1313968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/20.6.1363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1313968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HM42000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.19" data-doi="10.1098/rsbl.2017.0646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pleška</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guet</span>  <span class="cit-name-given-names">CC</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Effects of mutations in phage restriction sites during escape from restriction–modification</span>. <abbr class="cit-jnl-abbrev">Biol Lett</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">20170646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2017.0646%26rft_id%253Dinfo%253Apmid%252F29237814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2017.0646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29237814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.20" data-doi="10.1186/gb-2013-14-12-r143"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cáceres</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">LD</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The evolution, impact and properties of exonic splice enhancers</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">R143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR143%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-12-r143%26rft_id%253Dinfo%253Apmid%252F24359918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-12-r143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24359918&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.21" data-doi="10.1126/science.1170160"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">JB</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Coding-Sequence Determinants of Gene Expression in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>:<span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.22" data-doi="10.7554/eLife.22037"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burkhardt</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouskin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">CA</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Operon mRNAs are organized into ORF-centric structures that predict translation efficiency</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e22037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De22037%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.22037%26rft_id%253Dinfo%253Apmid%252F28139975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.22037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28139975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.23" data-doi="10.1038/nature16509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boël</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Letso</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neely</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">WN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luff</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valecha</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acton</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montelione</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aalberts</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">JF</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Codon influence on protein expression in E. coli correlates with mRNA levels</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">529</span>:<span class="cit-fpage">358</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D529%26rft.spage%253D358%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16509%26rft_id%253Dinfo%253Apmid%252F26760206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26760206&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelsic</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishony</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">RNA Structural Determinants of Optimal Codons Revealed by MAGE-Seq</span>. <abbr class="cit-jnl-abbrev">Cell Syst</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">563</span>–<span class="cit-lpage">571</span>.e6.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst%26rft.volume%253D3%26rft.spage%253D563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.25" data-doi="10.1016/j.cell.2015.02.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Presnyak</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alhusaini</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kline</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Codon Optimality Is a Major Determinant of mRNA Stability</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>:<span class="cit-fpage">1111</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D1111%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.02.029%26rft_id%253Dinfo%253Apmid%252F25768907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.02.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25768907&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ponnala</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">A Plausible Role for the Presence of Internal Shine-Dalgarno Sites</span>. <abbr class="cit-jnl-abbrev">Bioinforma Biol Insights</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">55</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinforma%2BBiol%2BInsights%26rft.volume%253D4%26rft.spage%253D55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.27" data-doi="10.1038/nature10965"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">JS</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>:<span class="cit-fpage">538</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DG.%2BW.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D538%26rft.epage%253D541%26rft.atitle%253DThe%2Banti-Shine-Dalgarno%2Bsequence%2Bdrives%2Btranslational%2Bpausing%2Band%2Bcodon%2Bchoice%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10965%26rft_id%253Dinfo%253Apmid%252F22456704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22456704&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.28" data-doi="10.1126/science.1241934"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosuri</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Causes and Effects of N-Terminal Codon Bias in Bacterial Genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>:<span class="cit-fpage">475</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241934%26rft_id%253Dinfo%253Apmid%252F24072823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1Ny80NzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.29" data-doi="10.1016/j.cell.2010.03.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carmi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vestsigian</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dorfan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaborske</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dahan</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furman</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">An Evolutionarily Conserved Mechanism for Controlling the Efficiency of Protein Translation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>:<span class="cit-fpage">344</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTuller%26rft.auinit1%253DT.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D344%26rft.epage%253D354%26rft.atitle%253DAn%2Bevolutionarily%2Bconserved%2Bmechanism%2Bfor%2Bcontrolling%2Bthe%2Befficiency%2Bof%2Bprotein%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.031%26rft_id%253Dinfo%253Apmid%252F20403328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403328&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.30" data-doi="10.1093/nar/18.6.1465"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Suppression of the negative effect of minor arginine codons on gene expression; preferential usage of minor codons within the first 25 codons of the Escherichia coli genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">1465</span>–<span class="cit-lpage">1473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DChen%26rft.auinit1%253DG.-F.%2BT.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D1465%26rft.epage%253D1473%26rft.atitle%253DSuppression%2Bof%2Bthe%2Bnegative%2Beffect%2Bof%2Bminor%2Barginine%2Bcodons%2Bon%2Bgene%2Bexpression%253B%2Bpreferential%2Busage%2Bof%2Bminor%2Bcodons%2Bwithin%2Bthe%2Bfirst%2B25%2Bcodons%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.6.1465%26rft_id%253Dinfo%253Apmid%252F2109307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.6.1465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2109307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CW96700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.31" data-doi="10.1016/j.cell.2010.02.036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cannarozzi</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schraudolph</span>  <span class="cit-name-given-names">NN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faty</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Rohr</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friberg</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonnet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonnet</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barral</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">A Role for Codon Order in Translation Dynamics</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>:<span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCannarrozzi%26rft.auinit1%253DG.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D367%26rft.atitle%253DA%2Brole%2Bfor%2Bcodon%2Border%2Bin%2Btranslation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.02.036%26rft_id%253Dinfo%253Apmid%252F20403329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.02.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403329&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.32" data-doi="10.1038/nsmb.1554"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubalewska</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ignatova</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Transient ribosomal attenuation coordinates protein synthesis and co-translational folding</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">274</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D274%26rft.epage%253D280%26rft.atitle%253DTransient%2Bribosomal%2Battenuation%2Bcoordinates%2Bprotein%2Bsynthesis%2Band%2Bco-translational%2Bfolding.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1554%26rft_id%253Dinfo%253Apmid%252F19198590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263906000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.33" data-doi="10.1016/j.molcel.2015.07.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Codon Usage Influences the Local Rate of Translation Elongation to Regulate Co-translational Protein Folding</span>. <abbr class="cit-jnl-abbrev">Mol Cell</abbr> <span class="cit-vol">59</span>:<span class="cit-fpage">744</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D59%26rft.spage%253D744%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.07.018%26rft_id%253Dinfo%253Apmid%252F26321254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.07.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26321254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.34" data-doi="10.1093/oxfordjournals.molbev.a040335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040335%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3916708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AAA8800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.35" data-doi="10.1093/nar/8.1.197-c"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavé</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">1980</span>. <span class="cit-article-title">Codon catalog usage and the genome hypothesis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">r49</span>–<span class="cit-lpage">r62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253Dr49%26rft.epage%253D62%26rft.atitle%253DCodon%2Bcatalog%2Busage%2Band%2Bthe%2Bgenome%2Bhypothesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.1.197-c%26rft_id%253Dinfo%253Apmid%252F6986610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.1.197-c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6986610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JC46600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.36" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Selection on codon bias</span>. <abbr class="cit-jnl-abbrev">Annu Rev Genet</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.37" data-doi="10.1371/journal.pgen.1003527"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lawrie</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">PW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Strong Purifying Selection at Synonymous Sites in D. melanogaster</span>. <abbr class="cit-jnl-abbrev">PLOS Genet</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet%26rft.volume%253D9%26rft.spage%253De1003527%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003527%26rft_id%253Dinfo%253Apmid%252F23737754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23737754&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawrie</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Strong purifying selection on codon usage bias</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> 106476.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.39" data-doi="10.1126/science.1089370"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">JS</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">The Origins of Genome Complexity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>:<span class="cit-fpage">1401</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D302%26rft.issue%253D5649%26rft.spage%253D1401%26rft.epage%253D1404%26rft.atitle%253DThe%2BOrigins%2Bof%2BGenome%2BComplexity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089370%26rft_id%253Dinfo%253Apmid%252F14631042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY0OS8xNDAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kristofich</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgenthaler</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kinney</span>  <span class="cit-name-given-names">WR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebmeier</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Old</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">VS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Copley</span>  <span class="cit-name-given-names">SD</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Synonymous mutations make dramatic contributions to fitness when growth is limited by a weak-link enzyme</span>. <abbr class="cit-jnl-abbrev">PLOS Genet</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">e1007615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet%26rft.volume%253D14%26rft.spage%253De1007615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lucks</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">JB</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Genome Landscapes and Bacteriophage Codon Usage</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.42" data-doi="10.1073/pnas.1515387112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">YC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bischof</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desrosiers</span>  <span class="cit-name-given-names">RC</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Importance of codon usage for the temporal regulation of viral gene expression</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">14030</span>–<span class="cit-lpage">14035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1515387112%26rft_id%253Dinfo%253Apmid%252F26504241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ1LzE0MDMwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.43" data-doi="10.1016/j.virusres.2005.05.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">TC</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Synonymous codon usage bias in 16 Staphylococcus aureus phages: Implication in phage therapy</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">113</span>:<span class="cit-fpage">123</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DSau%26rft.auinit1%253DK.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D131%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bbias%2Bin%2B16%2BStaphylococcus%2Baureus%2Bphages%253A%2Bimplication%2Bin%2Bphage%2Btherapy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.05.001%26rft_id%253Dinfo%253Apmid%252F15970346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15970346&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232500300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kunisawa</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanaya</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kutter</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Comparison of synonymous codon distribution patterns of bacteriophage and host genomes</span>. <abbr class="cit-jnl-abbrev">DNA Res Int J Rapid Publ Rep Genes Genomes</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">319</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes%2BInt%2BJ%2BRapid%2BPubl%2BRep%2BGenes%2BGenomes%26rft.volume%253D5%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.45" data-doi="10.1016/S0022-5193(05)80725-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kunisawa</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Synonymous codon preferences in bacteriophage T4: A distinctive use of transfer RNAs from T4 and from its host Escherichia coli</span>. <abbr class="cit-jnl-abbrev">J Theor Biol</abbr> <span class="cit-vol">159</span>:<span class="cit-fpage">287</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DKunisawa%26rft.auinit1%253DT.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D287%26rft.epage%253D298%26rft.atitle%253DSynonymous%2Bcodon%2Bpreferences%2Bin%2Bbacteriophage%2BT4%253A%2Ba%2Bdistinctive%2Buse%2Bof%2Btransfer%2BRNAs%2Bfrom%2BT4%2Band%2Bfrom%2Bits%2Bhost%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252805%252980725-8%26rft_id%253Dinfo%253Apmid%252F1296091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(05)80725-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1296091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KH22900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.46" data-doi="10.1056/NEJMra1204186"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nabel</span>  <span class="cit-name-given-names">GJ</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Designing Tomorrow’s Vaccines</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">368</span>:<span class="cit-fpage">551</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D368%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra1204186%26rft_id%253Dinfo%253Apmid%252F23388006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra1204186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23388006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314494100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.47" data-doi="10.1371/journal.ppat.1003001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rueckert</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guzmán</span>  <span class="cit-name-given-names">CA.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Vaccines: From Empirical Development to Rational Design</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e1003001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRueckert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003001%26rft.epage%253De1003001%26rft.atitle%253DVaccines%253A%2Bfrom%2Bempirical%2Bdevelopment%2Bto%2Brational%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003001%26rft_id%253Dinfo%253Apmid%252F23144616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23144616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.48" data-doi="10.1016/j.virol.2015.03.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minor</span>  <span class="cit-name-given-names">PD</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Live attenuated vaccines: Historical successes and current challenges</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">479–480</span>:<span class="cit-fpage">379</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2015.03.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2015.03.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramezanpour</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haan</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claassen</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Vector-based genetically modified vaccines: Exploiting Jenner’s legacy</span>. <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">34</span>:<span class="cit-fpage">6436</span>–<span class="cit-lpage">6448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D34%26rft.spage%253D6436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.50" data-doi="10.1371/journal.pcbi.0020116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Model-Based Design of Growth-Attenuated Viruses</span>. <abbr class="cit-jnl-abbrev">PLOS Comput Biol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">e116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DLim%26rft.auinit1%253DK.%2BI.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De116%26rft.epage%253De116%26rft.atitle%253DModel-based%2Bdesign%2Bof%2Bgrowth-attenuated%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020116%26rft_id%253Dinfo%253Apmid%252F16948530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16948530&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.51" data-doi="10.1038/nbt.1636"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nimnual</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Live attenuated influenza virus vaccines by computer-aided rational design</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">28</span>:<span class="cit-fpage">723</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D723%26rft.epage%253D726%26rft.atitle%253DLive%2Battenuated%2Binfluenza%2Bvirus%2Bvaccines%2Bby%2Bcomputer-aided%2Brational%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1636%26rft_id%253Dinfo%253Apmid%252F20543832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20543832&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279723900029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.52" data-doi="10.1093/nar/12.20.7787"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanway</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minor</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schild</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almond</span>  <span class="cit-name-given-names">JW</span></span>. <span class="cit-pub-date">1984</span>. <span class="cit-article-title">Reversion to neurovirulence of the live-attenuated Sabin type 3 oral poliovirus vaccine</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">7787</span>–<span class="cit-lpage">7792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DCann%26rft.auinit1%253DA.J.%26rft.volume%253D12%26rft.issue%253D20%26rft.spage%253D7787%26rft.epage%253D7792%26rft.atitle%253DReversion%2Bto%2Bneurovirulence%2Bof%2Bthe%2Blive-attenuated%2BSabin%2Btype%2B3%2Boral%2Bpollovirus%2Bvaccine%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F12.20.7787%26rft_id%253Dinfo%253Apmid%252F6093054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/12.20.7787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6093054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TP73900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Famulare</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iber</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adeniji</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bukbuk</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Behrend</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberste</span>  <span class="cit-name-given-names">MS</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Sabin Vaccine Reversion in the Field: a Comprehensive Analysis of Sabin-Like Poliovirus Isolates in Nigeria</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">317</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D90%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.54" data-doi="10.15585/mmwr.mm6530a3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diop</span>  <span class="cit-name-given-names">OM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iber</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sutter</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wassilak</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Update on Vaccine-Derived Polioviruses — Worldwide, January 2015–May 2016</span>. <abbr class="cit-jnl-abbrev">MMWR Morb Mortal Wkly Rep</abbr> <span class="cit-vol">65</span>:<span class="cit-fpage">763</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMMWR%2BMorb%2BMortal%2BWkly%2BRep%26rft.volume%253D65%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.15585%252Fmmwr.mm6530a3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15585/mmwr.mm6530a3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.55" data-doi="10.1016/j.coviro.2012.02.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Reaching the last one per cent: progress and challenges in global polio eradication</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">188</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BOpin%2BVirol%26rft.aulast%253DKew%26rft.auinit1%253DO.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D188%26rft.epage%253D198%26rft.atitle%253DReaching%2Bthe%2Blast%2Bone%2Bper%2Bcent%253A%2Bprogress%2Band%2Bchallenges%2Bin%2Bglobal%2Bpolio%2Beradication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2012.02.006%26rft_id%253Dinfo%253Apmid%252F22482715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2012.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22482715&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cello</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Chemical Synthesis of Poliovirus cDNA: Generation of Infectious Virus in the Absence of Natural Template</span>. <abbr class="cit-jnl-abbrev">Science</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.57" data-doi="10.1128/JVI.80.7.3259-3272.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quay</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Modulation of Poliovirus Replicative Fitness in HeLa Cells by Deoptimization of Synonymous Codon Usage in the Capsid Region</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">3259</span>–<span class="cit-lpage">3272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBurns%26rft.auinit1%253DC.%2BC.%26rft.volume%253D80%26rft.issue%253D7%26rft.spage%253D3259%26rft.epage%253D3272%26rft.atitle%253DModulation%2Bof%2BPoliovirus%2BReplicative%2BFitness%2Bin%2BHeLa%2BCells%2Bby%2BDeoptimization%2Bof%2BSynonymous%2BCodon%2BUsage%2Bin%2Bthe%2BCapsid%2BRegion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.7.3259-3272.2006%26rft_id%253Dinfo%253Apmid%252F16537593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzcvMzI1OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.58" data-doi="10.1128/JVI.00738-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Reduction of the Rate of Poliovirus Protein Synthesis through Large-Scale Codon Deoptimization Causes Attenuation of Viral Virulence by Lowering Specific Infectivity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">9687</span>–<span class="cit-lpage">9696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D19%26rft.spage%253D9687%26rft.epage%253D9696%26rft.atitle%253DReduction%2Bof%2Bthe%2BRate%2Bof%2BPoliovirus%2BProtein%2BSynthesis%2Bthrough%2BLarge-Scale%2BCodon%2BDeoptimization%2BCauses%2BAttenuation%2Bof%2BViral%2BVirulence%2Bby%2BLowering%2BSpecific%2BInfectivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00738-06%26rft_id%253Dinfo%253Apmid%252F16973573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xOS85Njg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.59" data-doi="10.1126/science.1155761"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Virus Attenuation by Genome-Scale Changes in Codon Pair Bias</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>:<span class="cit-fpage">1784</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColeman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D320%26rft.issue%253D5884%26rft.spage%253D1784%26rft.epage%253D1787%26rft.atitle%253DVirus%2BAttenuation%2Bby%2BGenome-Scale%2BChanges%2Bin%2BCodon%2BPair%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155761%26rft_id%253Dinfo%253Apmid%252F18583614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4NC8xNzg0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.60" data-doi="10.1016/j.tim.2015.11.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordan-Paiz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nevot</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Synonymous Virus Genome Recoding as a Tool to Impact Viral Fitness</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">134</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D24%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2015.11.002%26rft_id%253Dinfo%253Apmid%252F26646373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2015.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26646373&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.61" data-doi="10.1093/molbev/mss119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molineux</span>  <span class="cit-name-given-names">IJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">CO</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Slow Fitness Recovery in a Codon-Modified Viral Genome</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">2997</span>–<span class="cit-lpage">3004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss119%26rft_id%253Dinfo%253Apmid%252F22532576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22532576&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.62" data-doi="10.1128/JVI.03401-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">BYH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortiz-Riaño</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nogales</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torre</span>  <span class="cit-name-given-names">JC de la</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Sobrido</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Development of Live-Attenuated Arenavirus Vaccines Based on Codon Deoptimization</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>:<span class="cit-fpage">3523</span>–<span class="cit-lpage">3533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03401-14%26rft_id%253Dinfo%253Apmid%252F25589652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzcvMzUyMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">BYH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nogales</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Torre</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Sobrido</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Development of live-attenuated arenavirus vaccines based on codon deoptimization of the viral glycoprotein</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">501</span>:<span class="cit-fpage">35</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D501%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cladel</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Budgeon</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balogh</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">ND</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Synonymous codon changes in the oncogenes of the cottontail rabbit papillomavirus lead to increased oncogenicity and immunogenicity of the virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">438</span>:<span class="cit-fpage">70</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D438%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Rescued influenza A virus with codon deoptimized NS1 gene is attenuated both in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">Chin J Biotechnol</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">720</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin%2BJ%2BBiotechnol%26rft.volume%253D25%26rft.spage%253D720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.66" data-doi="10.1128/mBio.01704-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hotard</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">ML</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Refining the Balance of Attenuation and Immunogenicity of Respiratory Syncytial Virus by Targeted Codon Deoptimization of Virulence Genes</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e01704</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D5%26rft.spage%253De01704%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01704-14%26rft_id%253Dinfo%253Apmid%252F25249281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01704-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25249281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.67" data-doi="10.1128/JVI.01565-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nogales</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortiz-Riaño</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dewhurst</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Topham</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Sobrido</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Influenza A Virus Attenuation by Codon Deoptimization of the NS Gene for Vaccine Development</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">10525</span>–<span class="cit-lpage">10540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01565-14%26rft_id%253Dinfo%253Apmid%252F24965472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOC8xMDUyNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.68" data-doi="10.1128/JVI.00012-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rostad</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stobart</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickles</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hotard</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">JCG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moin</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piedra</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">ML</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">A Recombinant Respiratory Syncytial Virus Vaccine Candidate Attenuated by a Low-Fusion F Protein Is Immunogenic and Protective against Challenge in Cotton Rats</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">7508</span>–<span class="cit-lpage">7518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00012-16%26rft_id%253Dinfo%253Apmid%252F27279612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNi83NTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Velazquez-Salinas</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Risatti</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holinka</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alfano</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrillo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gladue</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borca</span>  <span class="cit-name-given-names">MV</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Recoding structural glycoprotein E2 in classical swine fever virus (CSFV) produces complete virus attenuation in swine and protects infected animals against disease</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">494</span>:<span class="cit-fpage">178</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D494%26rft.spage%253D178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.70" data-doi="10.1371/journal.ppat.1003172"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nougairede</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fabritus</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamballerie</span>  <span class="cit-name-given-names">X de</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Random Codon Re-encoding Induces Stable Reduction of Replicative Fitness of Chikungunya Virus in Primate and Mosquito Cells</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003172%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003172%26rft_id%253Dinfo%253Apmid%252F23436995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23436995&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.71" data-doi="10.1371/journal.ppat.1004738"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fabritus</span>  <span class="cit-name-given-names">L de</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nougairède</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamballerie</span>  <span class="cit-name-given-names">X de</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Attenuation of Tick-Borne Encephalitis Virus Using Large-Scale Random Codon Re-encoding</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">e1004738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D11%26rft.spage%253De1004738%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004738%26rft_id%253Dinfo%253Apmid%252F25734338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25734338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fabritus</span>  <span class="cit-name-given-names">L de</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nougairède</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamballerie</span>  <span class="cit-name-given-names">X de</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Utilisation of ISA Reverse Genetics and Large-Scale Random Codon Re-Encoding to Produce Attenuated Strains of Tick-Borne Encephalitis Virus within Days</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">e0159564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D11%26rft.spage%253De0159564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martí-Solano</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fillat</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Codon optimization of the adenoviral fiber negatively impacts structural protein expression and viral fitness</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">srep27546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253Dsrep27546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ngwuta</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Surman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kabatova</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lingemann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herbert</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swerczek</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moin</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwong</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munir</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Improved Prefusion Stability, Optimized Codon Usage, and Augmented Virion Packaging Enhance the Immunogenicity of Respiratory Syncytial Virus Fusion Protein in a Vectored-Vaccine Candidate</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">91</span>:<span class="cit-fpage">e00189</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D91%26rft.spage%253De00189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.75" data-doi="10.1128/JVI.03223-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vabret</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailly-Bechet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lepelley</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Najburg</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verrier</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tangy</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Large-Scale Nucleotide Optimization of Simian Immunodeficiency Virus Reduces Its Capacity To Stimulate Type I Interferon In Vitro</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">4161</span>–<span class="cit-lpage">4172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03223-13%26rft_id%253Dinfo%253Apmid%252F24478441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDE2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moratorio</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henningsson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barbezange</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrau</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordería</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaucourt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poirier</span>  <span class="cit-name-given-names">EZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vallet</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boussier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mounce</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fontes</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Attenuation of RNA viruses by redirecting their evolution in sequence space</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">17088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D2%26rft.spage%253D17088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.77" data-doi="10.1073/pnas.1411290111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Nouën</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luongo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mehedi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchholz</span>  <span class="cit-name-given-names">UJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">DiNapoli</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Attenuation of human respiratory syncytial virus by genome-scale codon-pair deoptimization</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">111</span>:<span class="cit-fpage">13169</span>–<span class="cit-lpage">13174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1411290111%26rft_id%253Dinfo%253Apmid%252F25157129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM2LzEzMTY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">78.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.78" data-doi="10.1186/1742-4690-10-78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martrus</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nevot</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Changes in codon-pair bias of human immunodeficiency virus type 1 have profound effects on virus replication in cell culture</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRetrovirology%26rft.volume%253D10%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-10-78%26rft_id%253Dinfo%253Apmid%252F23885919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-10-78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23885919&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.79" data-doi="10.1073/pnas.1502864112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stauft</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbatsevych</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurovsky</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Large-scale recoding of an arbovirus genome to rebalance its insect versus mammalian preference</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">4749</span>–<span class="cit-lpage">4754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1502864112%26rft_id%253Dinfo%253Apmid%252F25825721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE1LzQ3NDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.80" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">WH</span></span>. <span class="cit-pub-date">1987</span>. <span class="cit-article-title">The codon Adaptation Index--a measure of directional synonymous codon usage bias, and its potential applications</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">15</span>:<span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.3.1281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3547335&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987F976100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">An Improved Implementation of Codon Adaptation Index</span>. <abbr class="cit-jnl-abbrev">Evol Bioinforma Online</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">53</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BBioinforma%2BOnline%26rft.volume%253D3%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.82" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Reis M</span> <span class="cit-name-surname">dos</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15448185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224355400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.83" data-doi="10.1534/genetics.114.172106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A Major Controversy in Codon-Anticodon Adaptation Resolved by a New Codon Usage Index</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>:<span class="cit-fpage">573</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172106%26rft_id%253Dinfo%253Apmid%252F25480780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvNTczIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.84" data-doi="10.1093/dnares/dsq012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erill</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Relative Codon Adaptation: A Generic Codon Bias Index for Prediction of Gene Expression</span>. <abbr class="cit-jnl-abbrev">DNA Res</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">185</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsq012%26rft_id%253Dinfo%253Apmid%252F20453079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsq012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20453079&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279411300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.85" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">The ‘effective number of codons’ used in a gene</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>:<span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.86" data-doi="10.1126/science.1083811"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elf</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenson</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrenberg</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Selective Charging of tRNA Isoacceptors Explains Patterns of Codon Usage</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>:<span class="cit-fpage">1718</span>–<span class="cit-lpage">1722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DElf%26rft.auinit1%253DJ.%26rft.volume%253D300%26rft.issue%253D5626%26rft.spage%253D1718%26rft.epage%253D1722%26rft.atitle%253DSelective%2BCharging%2Bof%2BtRNA%2BIsoacceptors%2BExplains%2BPatterns%2Bof%2BCodon%2BUsage%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1083811%26rft_id%253Dinfo%253Apmid%252F12805541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.87" data-doi="10.1038/msb.2013.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bentele</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saffert</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rauscher</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ignatova</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blüthgen</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Efficient translation initiation dictates codon usage at gene start</span>. <abbr class="cit-jnl-abbrev">Mol Syst Biol</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2013.32%26rft_id%253Dinfo%253Apmid%252F23774758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjkvMS82NzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Leuven</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Selecting among three basic fitness landscape models: Additive, multiplicative and stickbreaking</span>. <abbr class="cit-jnl-abbrev">Theor Popul Biol</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.89" data-doi="10.1534/genetics.111.132134"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagel</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">CR</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Stickbreaking: A Novel Fitness Landscape Model That Harbors Epistasis and Is Consistent with Commonly Observed Patterns of Adaptive Evolution</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>:<span class="cit-fpage">655</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132134%26rft_id%253Dinfo%253Apmid%252F22095084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNjU1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.90" data-doi="10.1093/nar/12.17.6663"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emtage</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarranton</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millican</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Humphreys</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">1984</span>. <span class="cit-article-title">Codon usage can affect efficiency of translation of genes in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">6663</span>–<span class="cit-lpage">6671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DRobinson%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D17%26rft.spage%253D6663%26rft.epage%253D6671%26rft.atitle%253DCodon%2Busage%2Bcan%2Baffect%2Befficiency%2Bof%2Btranslation%2Bof%2Bgenes%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F12.17.6663%26rft_id%253Dinfo%253Apmid%252F6091031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/12.17.6663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6091031&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TH92900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.91" data-doi="10.1016/j.celrep.2013.07.049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quax</span>  <span class="cit-name-given-names">TEF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">YI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koehorst</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wurtzel</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Oost</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blombach</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brouns</span>  <span class="cit-name-given-names">SJJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">EGH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorek</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Oost</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Differential Translation Tunes Uneven Production of Operon-Encoded Proteins</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">938</span>–<span class="cit-lpage">944</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D4%26rft.spage%253D938%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.07.049%26rft_id%253Dinfo%253Apmid%252F24012761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.07.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24012761&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.92" data-doi="10.1371/journal.ppat.1000797"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aragonès</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guix</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ribes</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosch</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pintó</span>  <span class="cit-name-given-names">RM.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Fine-Tuning Translation Kinetics Selection as the Driving Force of Codon Usage Bias in the Hepatitis A Virus Capsid</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e1000797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DAragones%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De1000797%26rft.epage%253De1000797%26rft.atitle%253DFine-tuning%2Btranslation%2Bkinetics%2Bselection%2Bas%2Bthe%2Bdriving%2Bforce%2Bof%2Bcodon%2Busage%2Bbias%2Bin%2Bthe%2Bhepatitis%2BA%2Bvirus%2Bcapsid.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000797%26rft_id%253Dinfo%253Apmid%252F20221432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20221432&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.93" data-doi="10.1002/rmv.707"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K-N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Codon usage roles in human papillomavirus</span>. <abbr class="cit-jnl-abbrev">Rev Med Virol</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">397</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReviews%2Bin%2Bmedical%2Bvirology%26rft.stitle%253DRev%2BMed%2BVirol%26rft.aulast%253DZhao%26rft.auinit1%253DK.%2BN.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D397%26rft.epage%253D411%26rft.atitle%253DCodon%2Busage%2Broles%2Bin%2Bhuman%2Bpapillomavirus.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frmv.707%26rft_id%253Dinfo%253Apmid%252F22025363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/rmv.707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22025363&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kula</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saelens</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Travisano</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putonti</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The Evolution of Molecular Compatibility between Bacteriophage FX174 and its Host</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">8350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D8350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.95" data-doi="10.1126/science.1135308"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">I-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">ZE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calcagno</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ambudkar</span>  <span class="cit-name-given-names">SV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gottesman</span>  <span class="cit-name-given-names">MM</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">A “Silent” Polymorphism in the MDR1 Gene Changes Substrate Specificity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">315</span>:<span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1135308%26rft_id%253Dinfo%253Apmid%252F17185560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTUvNTgxMS81MjUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">96.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.96" data-doi="10.1038/ncomms5076"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kassen</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Adaptive synonymous mutations in an experimentally evolved <em>Pseudomonas fluorescens</em> population</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">4076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D5%26rft.spage%253D4076%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5076%26rft_id%253Dinfo%253Apmid%252F24912567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24912567&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.97" data-doi="10.1093/molbev/msw035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agashe</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sane</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phalnikar</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diwan</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Habibullah</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Gomez</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sahasrabuddhe</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Polachek</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chubiz</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">CJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Large-Effect Beneficial Synonymous Mutations Mediate Rapid and Parallel Adaptation in a Bacterium</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">1542</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw035%26rft_id%253Dinfo%253Apmid%252F26908584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26908584&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.98" data-doi="10.1371/journal.pgen.1002783"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schenk</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szendro</span>  <span class="cit-name-given-names">IG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Visser JAGM</span> <span class="cit-name-surname">de</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Quantifying the Adaptive Potential of an Antibiotic Resistance Enzyme</span>. <abbr class="cit-jnl-abbrev">PLOS Genet</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e1002783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSchenk%26rft.auinit1%253DM.%2BF.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002783%26rft.epage%253De1002783%26rft.atitle%253DQuantifying%2Bthe%2Badaptive%2Bpotential%2Bof%2Ban%2Bantibiotic%2Bresistance%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002783%26rft_id%253Dinfo%253Apmid%252F22761587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.99" data-doi="10.1038/s41437-018-0104-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zwart</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schenk</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koopmanschap</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lange N</span> <span class="cit-name-surname">de</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Pol L</span> <span class="cit-name-surname">van de</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">TTT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szendro</span>  <span class="cit-name-given-names">IG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Visser JAGM</span> <span class="cit-name-surname">de</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Unraveling the causes of adaptive benefits of synonymous mutations in TEM-1 β-lactamase</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">121</span>:<span class="cit-fpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D121%26rft.spage%253D406%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0104-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0104-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Couce</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">OA</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The rule of declining adaptability in microbial evolution experiments</span>. <abbr class="cit-jnl-abbrev">Front Genet</abbr> <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.101" data-doi="10.1126/science.1203799"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">H-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">H-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segrè</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">CJ</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Diminishing Returns Epistasis Among Beneficial Mutations Decelerates Adaptation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>:<span class="cit-fpage">1190</span>–<span class="cit-lpage">1192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChou%26rft.auinit1%253DH.-H.%26rft.volume%253D332%26rft.issue%253D6034%26rft.spage%253D1190%26rft.epage%253D1192%26rft.atitle%253DDiminishing%2BReturns%2BEpistasis%2BAmong%2BBeneficial%2BMutations%2BDecelerates%2BAdaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203799%26rft_id%253Dinfo%253Apmid%252F21636771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzIvNjAzNC8xMTkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">NH</span></span>. <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Testing for Epistasis Between Deleterious Mutations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>:<span class="cit-fpage">435</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWest%26rft.auinit1%253DS.%2BA.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D435%26rft.epage%253D444%26rft.atitle%253DTesting%2Bfor%2BEpistasis%2BBetween%2BDeleterious%2BMutations%26rft_id%253Dinfo%253Apmid%252F9584115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ5LzEvNDM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">103.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.103" data-doi="10.1098/rspb.2011.1537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Visser</span>  <span class="cit-name-given-names">JAGM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">TF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">The causes of epistasis</span>. <abbr class="cit-jnl-abbrev">Proc R Soc B Biol Sci</abbr> <span class="cit-vol">278</span>:<span class="cit-fpage">3617</span>–<span class="cit-lpage">3624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BR%2BSoc%2BB%2BBiol%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.1537%26rft_id%253Dinfo%253Apmid%252F21976687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.1537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21976687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.104" data-doi="10.1016/j.tree.2007.02.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kouyos</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silander</span>  <span class="cit-name-given-names">OK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonhoeffer</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Epistasis between deleterious mutations and the evolution of recombination</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">308</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DKouyos%26rft.auinit1%253DR.%2BD.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D308%26rft.epage%253D315%26rft.atitle%253DEpistasis%2Bbetween%2Bdeleterious%2Bmutations%2Band%2Bthe%2Bevolution%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.02.014%26rft_id%253Dinfo%253Apmid%252F17337087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.02.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17337087&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247194400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.105" data-doi="10.1038/nm1161"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Segrè</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeLuna</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishony</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Modular epistasis in yeast metabolism</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">37</span>:<span class="cit-fpage">77</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegr%25C3%25A8%26rft.auinit1%253DD.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D83%26rft.atitle%253DModular%2Bepistasis%2Bin%2Byeast%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1161%26rft_id%253Dinfo%253Apmid%252F15592468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592468&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225997500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">106.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.10.942847v1.106" data-doi="10.1038/37108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">RE</span></span>. <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Test of synergistic interactions among deleterious mutations in bacteria</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">390</span>:<span class="cit-fpage">395</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D390%26rft.issue%253D6658%26rft.spage%253D395%26rft.epage%253D398%26rft.atitle%253DTest%2Bof%2Bsynergistic%2Binteractions%2Bamong%2Bdeleterious%2Bmutations%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F37108%26rft_id%253Dinfo%253Apmid%252F9389477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/37108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9389477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YH54900061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.107" data-doi="10.1073/pnas.0404125101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The contribution of epistasis to the architecture of fitness in an RNA virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">101</span>:<span class="cit-fpage">15376</span>–<span class="cit-lpage">15379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404125101%26rft_id%253Dinfo%253Apmid%252F15492220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQzLzE1Mzc2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.108" data-doi="10.1038/ng1986"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jasnos</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korona</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Epistatic buffering of fitness loss in yeast double deletion strains</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">39</span>:<span class="cit-fpage">550</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJasnos%26rft.auinit1%253DL.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D550%26rft.epage%253D554%26rft.atitle%253DEpistatic%2Bbuffering%2Bof%2Bfitness%2Bloss%2Bin%2Byeast%2Bdouble%2Bdeletion%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1986%26rft_id%253Dinfo%253Apmid%252F17322879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17322879&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.109" data-doi="10.1371/journal.pgen.1006817"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guerrero</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Josway</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">LC</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Pervasive antagonistic interactions among hybrid incompatibility loci</span>. <abbr class="cit-jnl-abbrev">PLOS Genet</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">e1006817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet%26rft.volume%253D13%26rft.spage%253De1006817%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">110.</span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110. in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.110" data-doi="10.1016/j.jmb.2009.07.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parera</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perez-Alvarez</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Epistasis among Deleterious Mutations in the HIV-1 Protease</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">392</span>:<span class="cit-fpage">243</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DParera%26rft.auinit1%253DM.%26rft.volume%253D392%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D250%26rft.atitle%253DEpistasis%2Bamong%2Bdeleterious%2Bmutations%2Bin%2Bthe%2BHIV-1%2Bprotease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.07.015%26rft_id%253Dinfo%253Apmid%252F19607838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.07.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19607838&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">111.</span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111. in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.111" data-doi="10.1016/S0022-2836(02)00400-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minasov</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoichet</span>  <span class="cit-name-given-names">BK</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Evolution of an Antibiotic Resistance Enzyme Constrained by Stability and Activity Trade-offs</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">320</span>:<span class="cit-fpage">85</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D320%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D95%26rft.atitle%253DEvolution%2Bof%2Ban%2Bantibiotic%2Bresistance%2Benzyme%2Bconstrained%2Bby%2Bstability%2Band%2Bactivity%2Btrade-offs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252802%252900400-X%26rft_id%253Dinfo%253Apmid%252F12079336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(02)00400-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12079336&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176609700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">112.</span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference 112. in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.112" data-doi="10.1038/hdy.2012.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lalic</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Magnitude and sign epistasis among deleterious mutations in a positive-sense plant RNA virus</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">71</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DLalic%26rft.auinit1%253DJ.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D71%26rft.epage%253D77%26rft.atitle%253DMagnitude%2Band%2Bsign%2Bepistasis%2Bamong%2Bdeleterious%2Bmutations%2Bin%2Ba%2Bpositive-sense%2Bplant%2BRNA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.15%26rft_id%253Dinfo%253Apmid%252F22491062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22491062&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306567400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">113.</span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference 113. in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.113" data-doi="10.1126/science.aay4199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martsul</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">MM</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Higher-fitness yeast genotypes are less robust to deleterious mutations</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">366</span>:<span class="cit-fpage">490</span>–<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnson%26rft.auinit1%253DM.%2BS.%26rft.volume%253D366%26rft.issue%253D6464%26rft.spage%253D490%26rft.epage%253D493%26rft.atitle%253DHigher-fitness%2Byeast%2Bgenotypes%2Bare%2Bless%2Brobust%2Bto%2Bdeleterious%2Bmutations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aay4199%26rft_id%253Dinfo%253Apmid%252F31649199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjYvNjQ2NC80OTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">114.</span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference 114. in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.114" data-doi="10.1016/j.celrep.2015.12.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kunec</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterrieder</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Codon Pair Bias Is a Direct Consequence of Dinucleotide Bias</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">55</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D14%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.12.011%26rft_id%253Dinfo%253Apmid%252F26725119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.12.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26725119&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">115.</span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference 115. in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.115" data-doi="10.1073/pnas.1507710112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbatsevych</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stauft</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leatherwood</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurovsky</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Reply to Simmonds et al.: Codon pair and dinucleotide bias have not been functionally distinguished</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">E3635</span>–<span class="cit-lpage">E3636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1507710112%26rft_id%253Dinfo%253Apmid%252F26071446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzI4L0UzNjM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">116.</span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference 116. in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jorge</span>  <span class="cit-name-given-names">DM de M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">CodonShuffle: a tool for generating and analyzing synonymously mutated sequences</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">117.</span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference 117. in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.117" data-doi="10.1093/nar/gki376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grote</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Münch</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nörtemann</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hempel</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">JCat: a novel tool to adapt codon usage of a target gene to its potential expression host</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">W526</span>–<span class="cit-lpage">W531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGrote%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW526%26rft.epage%253DW531%26rft.atitle%253DJCat%253A%2Ba%2Bnovel%2Btool%2Bto%2Badapt%2Bcodon%2Busage%2Bof%2Ba%2Btarget%2Bgene%2Bto%2Bits%2Bpotential%2Bexpression%2Bhost%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki376%26rft_id%253Dinfo%253Apmid%252F15980527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400107&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">118.</span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference 118. in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.118" data-doi="10.1093/bioinformatics/btu192"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">BK-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D-Y.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Codon Optimization OnLine (COOL): a web-based multi-objective optimization platform for synthetic gene design</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">2210</span>–<span class="cit-lpage">2212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu192%26rft_id%253Dinfo%253Apmid%252F24728853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24728853&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F11%2F2020.02.10.942847.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">119.</span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference 119. in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paff</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jack</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">CO</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Combinatorial Approaches to Viral Attenuation</span>. <abbr class="cit-jnl-abbrev">mSystems</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e00046</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSystems%26rft.volume%253D3%26rft.spage%253De00046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">120.</span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference 120. in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.120" data-doi="10.1128/JVI.01804-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burkett</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickles</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Systematic Assembly of a Full-Length Infectious Clone of Human Coronavirus NL63</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">11948</span>–<span class="cit-lpage">11957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01804-08%26rft_id%253Dinfo%253Apmid%252F18818320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yMy8xMTk0OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzExLzIwMjAuMDIuMTAuOTQyODQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">121.</span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference 121. in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.121" data-doi="10.1534/genetics.104.034488"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millstein</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Adaptive Molecular Evolution for 13,000 Phage Generations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>:<span class="cit-fpage">19</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.034488%26rft_id%253Dinfo%253Apmid%252F15687276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTcwLzEvMTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMS8yMDIwLjAyLjEwLjk0Mjg0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">122.</span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference 122. in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.10.942847v1.122" data-doi="10.1128/JB.183.17.5025-5040.2001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mrázek</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Characterizations of Highly Expressed Genes of Four Fast-Growing Bacteria</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-vol">183</span>:<span class="cit-fpage">5025</span>–<span class="cit-lpage">5040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKarlin%26rft.auinit1%253DS.%26rft.volume%253D183%26rft.issue%253D17%26rft.spage%253D5025%26rft.epage%253D5040%26rft.atitle%253DCharacterizations%2Bof%2BHighly%2BExpressed%2BGenes%2Bof%2BFour%2BFast-Growing%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.17.5025-5040.2001%26rft_id%253Dinfo%253Apmid%252F11489855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4My8xNy81MDI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTEvMjAyMC4wMi4xMC45NDI4NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.10.942847v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.10.942979v1" title="Recent bloom of filamentous algae in Lake Baikal is caused by Spirogyra Link., 1820 of local origin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.02.10.939231v1" title="Plasticity in gene expression facilitates invasion of the desert environment in house mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 11, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.10.942847v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.10.942847v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.10.942847v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.10.942847v1" class="active" data-icon-position="" data-hide-link-title="0">ΦX174 Attenuation by Whole Genome Codon Deoptimization</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1145901" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XUoJFy63rJAiqiYd7EcVxkvWKxj-q70DQOroauApWUU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1145901" id="node1145901--2" data-pisa="biorxiv;2020.02.10.942847v1" data-pisa-master="biorxiv;2020.02.10.942847" data-seqnum="1145901" data-apath="/biorxiv/early/2020/02/11/2020.02.10.942847.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">ΦX174 Attenuation by Whole Genome Codon Deoptimization</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">Burleigh</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">LuAnn</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Randall A.</span> <span class="nlm-surname">Hughes</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Vlad</span> <span class="nlm-surname">Codrea</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew D.</span> <span class="nlm-surname">Ellington</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Holly</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.10.942847; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.10.942847 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;title=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;title=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;text=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;title=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;t=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;title=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;title=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1145901" id="node1145901--3" data-pisa="biorxiv;2020.02.10.942847v1" data-pisa-master="biorxiv;2020.02.10.942847" data-seqnum="1145901" data-apath="/biorxiv/early/2020/02/11/2020.02.10.942847.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">ΦX174 Attenuation by Whole Genome Codon Deoptimization</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">Burleigh</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">LuAnn</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Randall A.</span> <span class="nlm-surname">Hughes</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Vlad</span> <span class="nlm-surname">Codrea</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew D.</span> <span class="nlm-surname">Ellington</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Holly</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Craig</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.10.942847; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.10.942847 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Van+Leuven%2C+James+T.&amp;rft.au=Ederer%2C+Martina+M.&amp;rft.au=Burleigh%2C+Katelyn&amp;rft.au=Scott%2C+LuAnn&amp;rft.au=Hughes%2C+Randall+A.&amp;rft.au=Codrea%2C+Vlad&amp;rft.au=Ellington%2C+Andrew+D.&amp;rft.au=Wichman%2C+Holly&amp;rft.au=Miller%2C+Craig"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1145901/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1145901/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1145901/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1145901/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1145901/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1145901/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1145901/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1145901/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1145901/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1145901/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1145901/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1145901/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;count=horizontal&amp;via=&amp;text=%CE%A6X174%20Attenuation%20by%20Whole%20Genome%20Codon%20Deoptimization&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.10.942847v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.10.942847v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
