<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/25/2020.03.23.002931.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.23.002931v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.23.002931"/>
  <meta name="DC.Date" content="2020-03-25"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor ( i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3°C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low power – capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS 900 ) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003)."/>
  <meta name="DC.Contributor" content="Estefania Nunez-Bajo"/>
  <meta name="DC.Contributor" content="Michael Kasimatis"/>
  <meta name="DC.Contributor" content="Yasin Cotur"/>
  <meta name="DC.Contributor" content="Tarek Asfour"/>
  <meta name="DC.Contributor" content="Alex Collins"/>
  <meta name="DC.Contributor" content="Ugur Tanriverdi"/>
  <meta name="DC.Contributor" content="Max Grell"/>
  <meta name="DC.Contributor" content="Matti Kaisti"/>
  <meta name="DC.Contributor" content="Guglielmo Senesi"/>
  <meta name="DC.Contributor" content="Karen Stevenson"/>
  <meta name="DC.Contributor" content="Firat Güder"/>
  <meta name="article:published_time" content="2020-03-25"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3 °C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~0.35 USD per device. The system is operated digitally, portable and low power — capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS900) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003).&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.23.002931v1"/>
  <meta name="citation_id" content="2020.03.23.002931v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.002931v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.002931v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.002931v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.23.002931"/>
  <meta name="citation_num_pages" content="28"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.23.002931"/>
  <meta name="citation_author" content="Estefania Nunez-Bajo"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_email" content="e.nunez-bajo@imperial.ac.uk"/>
  <meta name="citation_author_email" content="guder@imperial.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3478-7759"/>
  <meta name="citation_author" content="Michael Kasimatis"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4509-4907"/>
  <meta name="citation_author" content="Yasin Cotur"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8898-0917"/>
  <meta name="citation_author" content="Tarek Asfour"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author" content="Alex Collins"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6889-9038"/>
  <meta name="citation_author" content="Ugur Tanriverdi"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author" content="Max Grell"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3718-3360"/>
  <meta name="citation_author" content="Matti Kaisti"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_institution" content="Department of Future Technologies, University of Turku"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8118-2851"/>
  <meta name="citation_author" content="Guglielmo Senesi"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author" content="Karen Stevenson"/>
  <meta name="citation_author_institution" content="Moredun Research Institute, Pentlands Science Park"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0270-9550"/>
  <meta name="citation_author" content="Firat Güder"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Imperial College London"/>
  <meta name="citation_author_email" content="e.nunez-bajo@imperial.ac.uk"/>
  <meta name="citation_author_email" content="guder@imperial.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5454-0609"/>
  <meta name="citation_reference" content="Chen, A. &amp;; Yang, S. Replacing antibodies with aptamers in lateral flow immunoassay. Biosens. Bioelectron. 71, 230–242 (2015)."/>
  <meta name="citation_reference" content="Zhang, Y. &amp;; Ozdemir, P. Microfluidic DNA amplification:a review. Anal. Chim. Acta 638, 115–125 (2009)."/>
  <meta name="citation_reference" content="Lee, S. H. et al. Emerging ultrafast nucleic acid amplification technologies for next generation molecular diagnostics. Biosens. Bioelectron 141, 111448 (2019)."/>
  <meta name="citation_reference" content="Menzel, A., Gübeli, R. J., Güder, F., Weber, W. &amp;; Zacharias, M. Detection of real-time dynamics of drug-target interactions by ultralong nanowalls. Lab Chip 13, 4173–4179 (2013)."/>
  <meta name="citation_reference" content="Jokerst, J. C. et al. Development of a paper-based analytical device for colorimetric detection of select foodborne pathogens. Anal. Chem. 84, 2900−2907 (2012)."/>
  <meta name="citation_reference" content="Li, Y., Cu, Y. T. H. &amp;; Luo, D. Multiplexed detection of pathogen DNA with DNA-based fluorescence nanobarcodes. Nat. Biotechnol. 23, 885−889 (2005)."/>
  <meta name="citation_reference" content="Donhauser, S. C., Niessner, R. &amp;; Seidel, M. Sensitive quantification of Escherichia coli O157:H7, Salmonella enterica, and Campylobacter jejuni by combining stopped polymerase chain reaction with chemiluminescence flow-through DNA microarray analysis. Anal. Chem. 83, 3153−3160 (2011)."/>
  <meta name="citation_reference" content="Chen, Y. et al. In situ hybridization chain reaction amplification for universal and highly sensitive electrochemiluminescent detection of DNA. Anal. Chem. 84, 7750–7755 (2012)."/>
  <meta name="citation_reference" content="Reboud, J. et al. Paper-based microfluidics for DNA diagnostics of malaria in low resource underserved rural communities. PNAS 116, 4834–4842 (2019)."/>
  <meta name="citation_reference" content="Tsaloglou, M.-N. et al. Handheld isothermal amplification and electrochemical detection of DNA in resource-limited settings. Anal. Biochem. 543, 116–121 (2018)."/>
  <meta name="citation_reference" content="Fang, X. et al. Real-time monitoring of strand-displacement DNA amplification by a contactless electrochemical microsystem using interdigitated electrodes. Lab Chip 12, 3190–3196 (2012)."/>
  <meta name="citation_reference" content="Yeung, S.-W., Lee, T. M.-H., Cai, H. &amp;; Hsing, I.-M. A DNA biochip for on-the-spot multiplexed pathogen identification. Nucleic Acids Res. 34, e118 (2006)."/>
  <meta name="citation_reference" content="Petralia, S. et al. A miniaturized silicon based device for nucleic acids electrochemical detection. Sens. Bio-Sensing Res. 6, 90–94 (2015)."/>
  <meta name="citation_reference" content="Yeung, S. S. W., Lee, T. M. H. &amp;; Hsing, I.-M. Electrochemistry-based real-time PCR on a microchip. Anal. Chem. 80, 363–368 (2008)."/>
  <meta name="citation_reference" content="Lee, T. M.-H., Carles, M. C. &amp;; Hsing, I.-M. Microfabricated PCR-electrochemical device for simultaneous DNA amplification and detection. Lab Chip 100–10 (2003)."/>
  <meta name="citation_reference" content="Güder, F. et al. Toward discrete multilayered composite structures: do hollow networks form in a polycrystalline infinite nanoplane by the Kirkendall effect? Chem. Mater. 23, 4445–4451 (2011)."/>
  <meta name="citation_reference" content="Grell, M. et al. Autocatalytic metallization of fabrics using Si ink, for biosensors, batteries and energy harvesting. Adv. Funct. Mater. 29, 1804798 (2019)."/>
  <meta name="citation_reference" content="Güder, F., Yang, Y., Kucukbayrak, U. M. &amp;; Zahcarias, M. Tracing the migration history of metal catalysts in metal-assisted chemically etched Silicon. ACS Nano 7, 1583–1590 (2013)."/>
  <meta name="citation_reference" content="Kasimatis, M. et al. Monolithic solder-on nanoporous Si-Cu contacts for stretchable silicone composite. ACS Appl. Mater. Interfaces 11, 47577−47586 (2019)."/>
  <meta name="citation_reference" content="Gesele, G., Linsmeierdag, J., Drachdag, V., Frickedag, J. &amp;; Arens-Fischer, R. Temperature-dependent thermal conductivity of porous silicon. J. Phys. D Appl. Phys. 30, 2911–2915 (1997)."/>
  <meta name="citation_reference" content="Weisse, J. M. et al. Thermal conductivity in porous silicon nanowire arrays. Nanoscale Res Lett 7, 554 (2012)."/>
  <meta name="citation_reference" content="Boukai, A. I. et al. Silicon nanowires as efficient thermoelectric materials. Nature 168–171 (2008)."/>
  <meta name="citation_reference" content="Deféver, T., Druet, M., Evrard, D., Marchal, D. &amp;; Limoges, B. Real-Time electrochemical PCR with a DNA intercalating redox probe. Anal. Chem. 83, 1815–1821 (2011)."/>
  <meta name="citation_reference" content="Fang, H. et al. Huey Fang, Naveen Ramalingam, Dong Xian-Dui, Tan Swee Ngin, Zeng Xianting, Annie Tan Lai Kuan, Eric Yap Peng Huat, GongHai-Qing. Biosens. Bioelectron. 24, 2131–2136 (2009)."/>
  <meta name="citation_reference" content="Won, B. Y. et al. Investigation of the signaling mechanism and verification of the performance of an electrochemical real-time PCR system based on the interaction of methylene blue with DNA. Analyst. 136, 1573–1579 (2011)."/>
  <meta name="citation_reference" content="Wildgoose, G. G., Giovanelli, D., Lawrence, N. S. &amp;; Compton, R. G. High-temperature electrochemistry: A review. Electroanalysis 16, 421–433 (2004)."/>
  <meta name="citation_reference" content="Watanabe, E., Mizuguchi, S., Hayashi, T. &amp;; Toyama, K. The effect of temperature on the electrochemical properties of copper-DNA membranes. Biosensors. 3, 117–125."/>
  <meta name="citation_reference" content="Schulzke, C. Temperature dependent electrochemistry—a versatile tool for investigations of biology related topics. Dalt. Trans. 34, 6683–6691 (2009)."/>
  <meta name="citation_reference" content="Mobius, P., Hotzel, H., Rassbach, A. &amp;; Kohler, H. Comparison of 13 single-round and nested PCR assays targeting IS900, ISMav2, f57 and locus 255 for detection of Mycobacterium avium subsp. paratuberculosis. Vet Microbiol. 126, 324–333 (2008)."/>
  <meta name="citation_reference" content="Vansnick, E. et al. Newly developed primers for the detection of Mycobacterium avium subspecies paratuberculosis. Vet Microbiol. 100, 197–204 (2004)."/>
  <meta name="citation_reference" content="Plain, K., Marsh, I., Waldron, A., Galea, F. &amp;; Whittington, AM Saunders, V. High-throughput direct fecal PCR assay for detection of Mycobacterium avium subsp. paratuberculosis in sheep and cattle. J Clin Microbiol 52, 745–57 (2014)."/>
  <meta name="citation_reference" content="Chan, J. F.-W. et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet 395, 514–523 (2020)."/>
  <meta name="citation_reference" content="Poon, L. L. M. et al. Rapid detection of the Severe Acute Respiratory Syndrome (SARS) Coronavirus by a Loop-mediated isothermal amplification assay. Clin. Chem. 50, 1050–1052 (2004)."/>
  <meta name="citation_reference" content="Deféver, T. et al. Real-time electrochemical monitoring of the polymerase chain reaction by mediated redox catalysis, J. Am. Chem. Soc. 131 (2009) 11433–11441."/>
  <meta name="citation_reference" content="Ferapontova, E. E. Electrochemical indicators for DNA electroanalysis,. Curr. Anal. Chem 7, 51–62 (2011)."/>
  <meta name="citation_reference" content="Patterson, A. S., Hsieh, K., Soh, H. T. &amp;; Plaxco, K. W. Electrochemical real-time nucleic acid amplification: towards point-of-care quantification of pathogens. Trends Biotechnol. 31, 705–712 (2013)."/>
  <meta name="citation_reference" content="Baradun, G. et al. Cellulose fibers enable near zero-cost electrical sensing of water-soluble gases. ACS Sensors 4, 1662–1669 (2019)."/>
  <meta name="citation_reference" content="Güder, F. et al. Paper-based electrical respiration S sensor. Angew. Chemie Int. Ed. 55, 5727–5732 (2016)."/>
  <meta name="citation_reference" content="Hansen, S. et al. Development of rapid extraction method of Mycobacterium avium subspecies paratuberculosis DNA from bovine stool samples. Diagnostics 6, 36–44 (2019)."/>
  <meta name="citation_reference" content="Heiniger, E. K. et al. Comparison of point-of-care-compatible lysis methods for bacteria and viruses. J Microbiol Methods 128, 80–87 (2016)."/>
  <meta name="twitter:title" content="Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor ( i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3°C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low power – capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS 900 ) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003)."/>
  <meta name="og-title" property="og:title" content="Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.002931v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor ( i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3°C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low power – capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS 900 ) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003)."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-25"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1210548 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention 
     <em>e.g.,</em> drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (
     <em>i.e.</em> thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3°C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low power – capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS
     <em>900</em>) of the genomic DNA of 
     <em>M. avium</em> subsp. 
     <em>paratuberculosis</em> (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30
     <sup>th</sup> cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003).
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">1. Introduction</h2>
    <p id="p-4">Despite the advancement of diagnostic technologies targeting nucleic acids (NA), there are still no rapid, handheld, low-cost, easy-to-use and integrated solutions for the testing of infectious diseases at the Point-of-Need (PoN). This is unfortunately the case for pathogens infecting humans, animals or even plants. This large gap in the diagnostic workflow hampers attempts to contain infectious pathogens from spreading by rapid and early detection. This technological gap has become, once again, evident with the spread of COVID-19, the diagnosis of which continues to depend heavily on centralized laboratories with specialized personnel and facilities, which in turn slows down testing and delays treatment.</p>
    <p id="p-5">NA make excellent targets for the direct detection of pathogens due to their high specificity. Unlike other biomarkers, such as antibodies or non-genetic molecules originating from pathogens, NA can be chemically amplified, enabling direct detection of low numbers of pathogens (down to single organisms). Hence tests targeting NA tend to be exceptionally sensitive. NA-based reagents (
     <em>e.g.,</em> primers) for NA testing can also be produced synthetically rapidly and on a large scale; DNA-based molecules, in particular, are highly stable, therefore, do not require a cold-chain for storage, making it especially suitable for PoN testing.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup> Ability to produce NA quickly also increases the speed of development and deployment of new test kits to health systems, providing much needed diagnostic capabilities for the detection of novel pathogens (such as SARS-CoV-2). Despite the massive advantages, on-site testing of NA has been limited.
    </p>
    <p id="p-6">There are several molecular methods to amplify NA for detection with high specificity 
     <em>e.g.,</em> polymerase chain reaction (PCR), loop-mediated isothermal amplification (LAMP), strand-displacement amplification (SDA), recombinase-polymerase amplification (RPA). While PCR requires thermocycling; emerging isothermal methods such as LAMP and RPA, do not, removing the need for sophisticated instruments.
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup> On the other hand, PCR is a simple process, requiring few reagents. It is also the gold standard for NA-based laboratory diagnostics with a solid support infrastructure. Amplification strategies are traditionally combined with fluorescence-based optical methods of detection to produce a quantitative analytical signal. Optical methods, even though sensitive, require expensive instruments.
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> The fluorescent labels used in optical detection are also difficult to handle and susceptible to photobleaching. Optical detection systems have been difficult to miniaturize and largely limited to centralized laboratories, however, some early concepts have been proposed by various groups in the literature.
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>
    </p>
    <p id="p-7">Despite low-cost onsite testing for infectious diseases is being the holy grail of NA diagnostics, there are still no inexpensive and handheld solutions in the market that can provide truly portable, rapid NA detection. Commercially available, benchtop optical lab instruments such as GeneXpert (Cephied) have already made a substantial difference in the speed of NA-based diagnosis of infectious diseases but the high cost of the instrument/tests, large size and power consumption have prevented the adoption of these systems for use in the field.</p>
    <p id="p-8">The ideal, miniaturized, low-cost portable detection system for NA must be able to heat up the sample from the patient to the desired temperature setpoint with high precision (using a heater and temperature sensor) and measure the results of the amplification reactions quantitatively, all in an integrated fashion. Using cleanroom-based semiconductor fabrication methods, such systems have been reported;
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup> however, these devices require advanced methods of microfabrication that can only be performed in a cleanroom, such as photolithography, vacuum etching/deposition etc. They also do not exploit the intrinsic properties of the semiconductor materials that are used as substrates (
     <em>i.e.</em> Si itself can be used both as a temperature sensor and electrical heater). Hence the devices reported are complex and expensive. Because semiconductor foundries are mainly located in East Asia, manufacturing and logistics are susceptible to disruptions in the supply chain due to unexpected events (
     <em>e.g.,</em> COVID-19 pandemic). There is, therefore, still no viable low-cost, rapid, quantitative, integrated handheld NA testing solution available for use in the field with the potential for market wide availability.
    </p>
    <p id="p-9">In this work, we report an integrated silicon-based Point-of-Need tri-modal NA transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of NA quantitatively in real-time (
     <strong>
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>
     </strong> and Supporting Information 
     <strong>Video V1</strong>). Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory and does not require fabrication in an advanced semiconductor foundry. Hence it is resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To achieve cleanroom-free, low-cost fabrication, we have developed a series of fabrication methodologies based on wet etching to form porous Silicon, electroplating, thermal bonding and laser-cutting (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). To reduce costs and complexity, TriSilix exploits the intrinsic properties of the semiconductor Si which can be used as a resistive heating device and thermistor simultaneously. TriSilix has three modes of operation: i) electrical (Joule) heater, ii) thermistor with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) label-free electrochemical sensor for detecting target NA with methylene blue as a redox-active reporter. TriSilix works with both cyclic and isothermal methods of amplification. We demonstrate that TriSilix can detect NA from both bacteria (
     <em>Mycobacterium avium</em> subspecies 
     <em>paratuberculosis</em>) and viruses (SARS-CoV-2) with high specificity.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Schematic illustration of construction of a TriSilix chip and photographs of the actual device. (B) Wafer-scale fabrication of TriSilix using a 4-inch Si wafer. Each wafer yield 37 chips. (C) Schematic illustration of the functional building blocks of TriSilix that provide trimodal operation for integrated nucleic acid amplification and detection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1496110387" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Schematic illustration of construction of a TriSilix chip and photographs of the actual device. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Wafer-scale fabrication of TriSilix using a 4-inch Si wafer. Each wafer yield 37 chips. (&lt;;strong&gt;;C)&lt;;/strong&gt;; Schematic illustration of the functional building blocks of TriSilix that provide trimodal operation for integrated nucleic acid amplification and detection.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F1.medium.gif" width="440" height="381"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F1.medium.gif" width="440" height="381"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-10" class="first-child">
       <strong>(A)</strong> Schematic illustration of construction of a TriSilix chip and photographs of the actual device. 
       <strong>(B)</strong> Wafer-scale fabrication of TriSilix using a 4-inch Si wafer. Each wafer yield 37 chips. (
       <strong>C)</strong> Schematic illustration of the functional building blocks of TriSilix that provide trimodal operation for integrated nucleic acid amplification and detection.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">2. Fabrication of TriSilix Chips</h2>
    <p id="p-11">The fabrication of multilayered, composite TriSilix chips (dimension of each chip: 10×10×0.65 mm) starts with a Si wafer.
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </sup> In this study we used lightly-doped p-type 4” Si wafers, however, wafers with a larger diameter would also work.
    </p>
    <p id="p-12">First, each wafer is plated electrolessly for 20s (in 80 μM KAuCl
     <sub>4</sub> and 0.5% HF) to form a thin layer of gold particulate film. The wafer is then placed inside an etching bath containing an aqueous solution of H
     <sub>2</sub>O
     <sub>2</sub> and HF with a ratio of 1:20 v/v (30% H
     <sub>2</sub>O
     <sub>2</sub>: 10% HF) to perform metal-assisted chemical etching (MACE) of Si for 10 min.
     <sup>
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup> This process forms a 500 nm thick nanoporous Silicon (pSi) layer on each side of the wafer (
     <strong>
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>
     </strong>). pSi plays a critical role in the fabrication of TriSilix. The porous surface allows electroplating of high-quality metal films on the surface of the Si substrate by creating an interlocking, high porosity surface to improve adhesion 
     <strong>(
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2B</a> &amp;; 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">C</a>)
     </strong>. Without this step, the metal films electroplated do not adhere to the surface of the substrate.
     <sup>
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>
     </sup> The pSi layer also allows thermal bonding of sheets of polymer films in an ordinary heat press after patterning through laser-cutting (
     <strong>
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>
     </strong>). After the formation of pSi surface, two layers of polyethylene terephthalate-polyethylene (PET-PE) are thermally bonded on the bottom and top surfaces of the Si wafer by heat pressing at 180 °C for 5 min. The heat pressed layers define the shape of the Working Electrode (WE) on the top surface and electrical contacts for Joule heating/resistance measurements on the bottom surface of the wafer. The patterned, and heat pressed polymer sheets essentially act as masking layers for electroplating of the metal electrodes on the pSi surface equivalent to photolithography-based patterning in conventional microfabrication. Next, the unmasked areas of the pSi surface were cleaned in 5% HF and electroplated by Au in a bath containing an aqueous solution of 10 mM KAu(CN)
     <sub>2</sub>/KCN for 10 min under a constant current of 10 mA versus Pt electrode (5×5 cm) yielding ~100 nm thick porous Au films. To achieve high uniformity when electroplating across the wafer, we have designed, and 3D printed a custom, electroplating platform that allows circular electrodes to be attached around the perimeter of the Si wafer (
     <strong>Figure S1</strong>). To form a three-electrode electrochemical cell on the top surface, the Counter (CE) and Reference Electrodes (RE) were created by heat pressing an Ag-plated Cu-PET-PE film, patterned once again by laser cutting. Ag plating was performed in an aqueous solution of 10 mM AgCN/KCN under an applied current of 20 mA versus Pt electrode (using the electroplating holder). Once the basic device structure was created, the wafer was laser-diced into individual chips (37 chips / 4” Si wafer, 
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Using the Randles–Sevcik equation with experimentally measured gradient values of 0.71 ± 0.08, we determined that the Au film has a 2x larger electroactive area than the geometrically defined area due to the high porosity of the Au film, which is beneficial for high-performance electrochemical analysis (
     <strong>Figure S2</strong>). A sample reservoir (polyethylene; diameter: 5 mm; volume: 40 μl, 
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>) with two holes was thermally bonded across the three-electrode electrochemical cell at 110±10 °C. The sample reservoir is specifically designed to prevent the evaporation of the solvent during amplification of DNA at elevated temperatures. The final TriSilix chip has a thickness of 490 μm (N=7) in the WE region and 650 μm in the RE/CE region (N=7) without the sample reservoir. 
     <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a> shows a schematic illustration of the functional building blocks of TriSilix that provide trimodal operation: The three-electrode electrochemical cell fabricated on the top surface is connected to an electrochemical reader (potentiostat) to provide an electrochemical signal concerning the detection of NA (Mode 1). The electrodes fabricated at the bottom of the device are connected to custom electronics to enable heating (Mode 2) and temperature sensing (Mode 3) for thermoregulation.
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Scanning electron micrographs of the surface of (A) Si wafer after metal-assisted chemical etching with Au catalyst producing a nanoporous Si surface (pSi). (B) pSi surface after electroplating with gold. (C) Cross-section of the sample shown in (B). The inset is a close-up view of the cross-section where the Au and porous Si layers are clearly visible. (D) Optical micrograph of the cross section of a TriSilix Chip with thermally bonded sheets of polymer films. Note that while the cross-sectional image shown in (C) was cleaved, (D) was laser-cut hence the roughness." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1496110387" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Scanning electron micrographs of the surface of &lt;;strong&gt;;(A)&lt;;/strong&gt;; Si wafer after metal-assisted chemical etching with Au catalyst producing a nanoporous Si surface (pSi). (&lt;;strong&gt;;B)&lt;;/strong&gt;; pSi surface after electroplating with gold. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Cross-section of the sample shown in (B). The inset is a close-up view of the cross-section where the Au and porous Si layers are clearly visible. &lt;;strong&gt;;(D)&lt;;/strong&gt;; Optical micrograph of the cross section of a TriSilix Chip with thermally bonded sheets of polymer films. Note that while the cross-sectional image shown in (C) was cleaved, (D) was laser-cut hence the roughness.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F2.medium.gif" width="436" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F2.medium.gif" width="436" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-13" class="first-child">Scanning electron micrographs of the surface of 
       <strong>(A)</strong> Si wafer after metal-assisted chemical etching with Au catalyst producing a nanoporous Si surface (pSi). (
       <strong>B)</strong> pSi surface after electroplating with gold. 
       <strong>(C)</strong> Cross-section of the sample shown in (B). The inset is a close-up view of the cross-section where the Au and porous Si layers are clearly visible. 
       <strong>(D)</strong> Optical micrograph of the cross section of a TriSilix Chip with thermally bonded sheets of polymer films. Note that while the cross-sectional image shown in (C) was cleaved, (D) was laser-cut hence the roughness.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">3. Characterization of Temperature Transduction</h2>
    <p id="p-14">NA amplification reactions require maintaining the sample at a temperature setpoint with high °precision. For PCR, the duration of each heating step must also be carefully controlled. Because the TriSilix uses Si, the substrate itself can be used both as an electrical heater and temperature sensor (
     <em>i.e.</em> thermistor) without adding any additional components for temperature transduction. Si, a semiconductor, heats up when an electrical current passes through it (known as Joule heating). Because Si has a high thermal conductivity (~150 W m
     <sup>−1</sup> °K
     <sup>−1</sup> at 300 °K), the substrate can be heated uniformly regardless of the path the current flows. The electrical resistance of Si also varies with temperature with a negative correlation (within our experimental range of temperatures from RT to 110 °C); the electrical resistance of Si drops with increasing temperature due to generation of mobile charge carriers allowing the use of Si substrate itself as a sensitive sensor of temperature. 
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>–
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
     </sup>
    </p>
    <p id="p-15">We have applied electrical currents in the range 0-400 mA between two Au electrodes deposited on the bottom of TriSilix chip to heat up the device electrically (the experiments were performed at room temperature; ~25 °C). During this experiment, we used a thermal camera (FLIR E4) to measure the temperature across the chip as a reference measurement. As illustrated in 
     <strong>
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>,
     </strong> TriSilix chip can be heated up to 110 °C, electrically. The relationship between the current applied and substrate temperature was linear (with a positive slope) at steady-state with high repeatability (slope: 232.3±8.7 °C A
     <sup>−1</sup>; R
     <sup>2</sup> = 0.9971, N=5).
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Relationship between temperature and (constant) current applied when TriSilix is used as an electrical heater (N=5). The temperature measurements were cross correlated with a thermal camera as a reference measurement. (B) Temperature vs normalized resistance for five TriSilix chips. The relationship between the normalized resistance (resistance/initial resistance) were linear for all devices with a negative slope. (C) I-V measurements across the Au-Si-Au junction which yielded a linear, Ohmic relationship between [[#x2212]]5V and 5V (R2=0.9996, N=5). (D) Photograph (top) and simplified circuit schematic diagram (bottom) of the custom electronics used for heating and temperature sensing. E) Temperature program to maintain the TriSilix chip at a temperature setpoint over time using software control and custom electronics. This program is an exemplar fifth cycle of a TriSilix PCR program in which the sample is kept at 30 sec for each step at 94 [[#xB0]]C, 63 [[#xB0]]C and 72 [[#xB0]]C before being lowered to 40 [[#xB0]]C for the electrochemical measurement. F) Photograph (left) and schematic illustration (right) of the silicone-based holder with embedded contacts to run TriSilix in trimodal operation. Error bars=SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1496110387" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Relationship between temperature and (constant) current applied when TriSilix is used as an electrical heater (N=5). The temperature measurements were cross correlated with a thermal camera as a reference measurement. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Temperature vs normalized resistance for five TriSilix chips. The relationship between the normalized resistance (resistance/initial resistance) were linear for all devices with a negative slope. &lt;;strong&gt;;(C)&lt;;/strong&gt;; I-V measurements across the Au-Si-Au junction which yielded a linear, Ohmic relationship between [[#x2212]]5V and 5V (R&lt;;sup&gt;;2&lt;;/sup&gt;;=0.9996, N=5). &lt;;strong&gt;;(D)&lt;;/strong&gt;; Photograph (top) and simplified circuit schematic diagram (bottom) of the custom electronics used for heating and temperature sensing. &lt;;strong&gt;;E)&lt;;/strong&gt;; Temperature program to maintain the TriSilix chip at a temperature setpoint over time using software control and custom electronics. This program is an exemplar fifth cycle of a TriSilix PCR program in which the sample is kept at 30 sec for each step at 94 [[#xB0]]C, 63 [[#xB0]]C and 72 [[#xB0]]C before being lowered to 40 [[#xB0]]C for the electrochemical measurement. &lt;;strong&gt;;F)&lt;;/strong&gt;; Photograph (left) and schematic illustration (right) of the silicone-based holder with embedded contacts to run TriSilix in trimodal operation. Error bars=SD.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F3.medium.gif" width="417" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F3.medium.gif" width="417" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-16" class="first-child">
       <strong>(A)</strong> Relationship between temperature and (constant) current applied when TriSilix is used as an electrical heater (N=5). The temperature measurements were cross correlated with a thermal camera as a reference measurement. 
       <strong>(B)</strong> Temperature vs normalized resistance for five TriSilix chips. The relationship between the normalized resistance (resistance/initial resistance) were linear for all devices with a negative slope. 
       <strong>(C)</strong> I-V measurements across the Au-Si-Au junction which yielded a linear, Ohmic relationship between −5V and 5V (R
       <sup>2</sup>=0.9996, N=5). 
       <strong>(D)</strong> Photograph (top) and simplified circuit schematic diagram (bottom) of the custom electronics used for heating and temperature sensing. 
       <strong>E)</strong> Temperature program to maintain the TriSilix chip at a temperature setpoint over time using software control and custom electronics. This program is an exemplar fifth cycle of a TriSilix PCR program in which the sample is kept at 30 sec for each step at 94 °C, 63 °C and 72 °C before being lowered to 40 °C for the electrochemical measurement. 
       <strong>F)</strong> Photograph (left) and schematic illustration (right) of the silicone-based holder with embedded contacts to run TriSilix in trimodal operation. Error bars=SD.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-17">At room temperature, the electrical resistance of a batch of TriSilix chips was 5.4±0.6 Ω (N=37) which varied linearly (with a negative slope of −6.1±0.4×10
     <sup>−3</sup> °C
     <sup>−1</sup>; R
     <sup>2</sup> = 0.9991, N=5) with the temperature measured within the range from room temperature to 110 °C (
     <strong>
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>
     </strong>). By measuring the electrical resistance of the Si substrate, using the two electrical contacts at the bottom of the chip, the temperature of the TriSilix chip can be precisely identified after calibration against a reference measurement (see Supporting Information 
     <strong>Section S3</strong> for more information on the calibration of the TriSilix thermistor). Temperature sensing is important for correct and fast execution of a NA amplification program for at least two reasons: i) Temperature can be maintained at a precise setpoint using a control algorithm such as proportional–integral–derivative control loop. ii) When cooling (passively) or heating (actively), the speed of reaching a temperature setpoint can be maximized as both heating and cooling depend on the outside temperature and device packaging (
     <em>e.g.,</em> the construction of the holder or cartridge). By only relying on the power applied, temperature control cannot be achieved with high precision.
    </p>
    <p id="p-18">Although we expected formation of Schottky barriers and non-linear electrical characteristics across the Au-Si interface, the I-V measurements (
     <strong>
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>
     </strong>) demonstrated that the junction exhibits relatively ohmic behavior with a linear I-V relationship (R
     <sup>2</sup>=0.9996) between −5V and +5V. The Si substrate plus the Au contacts can, therefore, be modeled as a simple variable resistor, the magnitude of which changes with temperature. We have designed a custom electrical circuit (
     <strong>
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>
     </strong>) and MATLAB-based graphical user interface (
     <strong>Figure S4</strong>) to control the temperature of the TriSilix chip at a precise setpoint through software control. The circuit (
     <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>) implements a voltage controlled constant current source which can be adjusted digitally using a low-cost microcontroller (Arduino Due). This circuit can also measure the voltage drop across the TriSilix chip; when a constant current is applied, the resistance (hence the temperature) of the TriSilix thermistor can be calculated using the voltage reading, and Ohm’s law. Using the MATLAB program and the custom electrical circuit board, we were able to run a temperature program (
     <strong>
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a>
     </strong>) and maintain the TriSilix chip at a given temperature setpoint over time. This is crucial for both cyclic and isothermal NA amplification reactions.
    </p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">4. Characterization of Electrochemical Sensing</h2>
    <p id="p-19">We produced a thermally-stable silicone-based holder encased in a 3D printed polylactic acid (PLA) housing (
     <strong>
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figures 3F</a> and S5
     </strong>) to characterize the performance of electrochemical sensing using the TriSilix chip. The silicone holder also contained five embedded gold-plated stainless-steel electrodes to make electrical contacts with the three electrodes positioned at the top for electrochemical NA sensing and four at the bottom for heating and temperature sensing.
    </p>
    <p id="p-20">First, we characterized the electrochemical redox processes for methylene blue (MB) using cyclic voltammetry (
     <strong>
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>
     </strong>). We chose MB because the electrochemical approach we used for the detection of DNA involves the use of MB
     <sup>
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
     </sup> as an intercalating redox reporter. During NA amplification, MB is intercalated between guanine-cytosine base pairs of the double-stranded DNA (ds-DNA) which provides an electroanalytical signal correlated with the concentration of ds-DNA in the sample. We prepared a 125 μg mL
     <sup>−1</sup> solution of MB in 10 mM phosphate-buffered saline pH 7 (PBS) and swept across a range of potentials between −400 to 200 mV at a scan rate of 100 mV s
     <sup>−1</sup> to characterize the electrochemical processes involving MB and electrodes. We determined that the anodic peak current, originating from the oxidation of MB on the electrode surface, appears at −67±2 and cathodic peak (due to reduction) at −97±4 mV versus Ag (
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). The absence of peaks with nearly overlapping anodic and cathodic curves in buffer alone indicates that the electrodes are electrochemically and mechanically stable and high performance (
     <em>e.g.,</em> low capacitive current). We have performed (
     <strong>
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>
     </strong>) square wave voltammetry (SWV), a substantially more sensitive electroanalytical method, to measure the concentration of MB in PBS in a range between 0 - 125 μg mL
     <sup>−1</sup> using a potential window from −500 mV to −250 mV versus Ag (corresponding to the anodic process). The SWV measurements using TriSilix produces an electroanalytical signal (peak current intensity) that is linearly related to the concentration of MB in the range 0.5 μg mL
     <sup>−1</sup> – 80 μg mL
     <sup>−1</sup> with an R
     <sup>2</sup> = 0.9963 (
     <strong>
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>
     </strong> – curve denoted by ‘RT’). We have also characterized the analytical performance of TriSilix for performing SWV measurements when the chip was operated at elevated temperatures. To prevent evaporation of the solvent (
     <em>i.e.</em> water) from the sample solution, a small amount (10 μL) of mineral oil was added to the reservoir. Since both isothermal and cyclic NA amplification reactions require heating; the effect of temperature on the electrochemical measurements is important. As shown in 
     <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>, the peak current intensity (measured from the recorded SWVs) increased two to four times in comparison to room temperature when operated at higher temperatures; this is due to enhanced transport of the analyte to the surface of the electrode.
     <sup>
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
     </sup> With increasing temperature, however, the error bars also widen, indicating that the precision of the measurement decreases. We speculate that this could be related to thermal and electrical crosstalk between the heater and the electrochemical sensing structures which introduce additional noise to the electroanalytical measurements at higher temperatures. We have also studied the thermal stability of MB over time (
     <strong>
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>
     </strong>). In this experiment, we added MB to the PCR and RPA mastermix solutions to simulate the conditions during isothermal and cyclic amplification reactions. As illustrated in 
     <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4D</a> (and 
     <strong>Figure S6</strong>), MB remained relatively stable both at 40 °C over 45 mins (
     <em>i.e.</em> RPA conditions) and 35 cycles of PCR conditions (30 sec for each step at 94 °C, 63 °C and 72 °C; every 5
     <sup>th</sup> cycle, the temperature is reduced to 40 °C for the measurement to increase precision as illustrated in 
     <strong>Figure S7</strong>) indicated by the small spread of the measurements. The SWV measurements at elevated temperatures produced similar results to the measurements taken at room temperature (25°C) in terms of repeatability, in agreement with the findings in the literature by other groups although we did observe a slight decrease in stability for the PCR conditions (
     <em>i.e.</em> higher temperatures).
     <sup>
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>
     </sup> The results of the thermal stability experiments indicate that MB is a sufficiently stable redox-active reporter for use in NA amplification reactions at elevated temperatures.
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) CVs recorded in 0 and 125 [[#x3BC]]g mL[[#x2212]]1 aqueous solutions of MB in PBS. Scan rate=100 mV s[[#x2212]]1. (B) SWVs recorded in MB solutions in PBS at room temperature in a range of concentrations from 0 to 125 [[#x3BC]]g mL[[#x2212]]1. (C) Calibration curves of peak current intensities vs concentration from SWVs recorded in MB solutions in PBS in a range of concentrations from 0 to 80 [[#x3BC]]g mL[[#x2212]]1 at different temperatures. Error bars=SD, N=5. (D) Histogram showing average values from SWVs recorded in RPA mastermix and PCR mastermix at room temperature and under NA amplification conditions. Error bars correspond to standard deviations from peak current intensities measured from 17 (RPA) and 8 (PCR) SWVs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1496110387" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; CVs recorded in 0 and 125 [[#x3BC]]g mL&lt;;sup&gt;;[[#x2212]]1&lt;;/sup&gt;; aqueous solutions of MB in PBS. Scan rate=100 mV s&lt;;sup&gt;;[[#x2212]]1&lt;;/sup&gt;;. (&lt;;strong&gt;;B)&lt;;/strong&gt;; SWVs recorded in MB solutions in PBS at room temperature in a range of concentrations from 0 to 125 [[#x3BC]]g mL&lt;;sup&gt;;[[#x2212]]1&lt;;/sup&gt;;. (&lt;;strong&gt;;C)&lt;;/strong&gt;; Calibration curves of peak current intensities vs concentration from SWVs recorded in MB solutions in PBS in a range of concentrations from 0 to 80 [[#x3BC]]g mL&lt;;sup&gt;;[[#x2212]]1&lt;;/sup&gt;; at different temperatures. Error bars=SD, N=5. &lt;;strong&gt;;(D)&lt;;/strong&gt;; Histogram showing average values from SWVs recorded in RPA mastermix and PCR mastermix at room temperature and under NA amplification conditions. Error bars correspond to standard deviations from peak current intensities measured from 17 (RPA) and 8 (PCR) SWVs.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F4.medium.gif" width="440" height="396"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F4.medium.gif" width="440" height="396"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-21" class="first-child">
       <strong>(A)</strong> CVs recorded in 0 and 125 μg mL
       <sup>−1</sup> aqueous solutions of MB in PBS. Scan rate=100 mV s
       <sup>−1</sup>. (
       <strong>B)</strong> SWVs recorded in MB solutions in PBS at room temperature in a range of concentrations from 0 to 125 μg mL
       <sup>−1</sup>. (
       <strong>C)</strong> Calibration curves of peak current intensities vs concentration from SWVs recorded in MB solutions in PBS in a range of concentrations from 0 to 80 μg mL
       <sup>−1</sup> at different temperatures. Error bars=SD, N=5. 
       <strong>(D)</strong> Histogram showing average values from SWVs recorded in RPA mastermix and PCR mastermix at room temperature and under NA amplification conditions. Error bars correspond to standard deviations from peak current intensities measured from 17 (RPA) and 8 (PCR) SWVs.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">5. Real-time Quantitative Detection of DNA During Amplification Reactions</h2>
    <p id="p-22">Using the TriSilix chip, we have performed real-time and quantitative RPA (isothermal, qRPA) and PCR (cyclic, qPCR) amplification of DNA using its temperature transduction capabilities and measured the concentration of reaction products – smaller fragments of ds-DNA, 
     <em>i.e.</em> amplicons – using the electrochemical DNA sensor, quantitatively and in real-time. During amplification, the number of amplicons increases exponentially with time if the DNA target is present in the sample. When new amplicons are generated, MB interacts with the G-C pairs and can no longer participate in the electron-transfer reactions with the working electrode hence the electroanalytical signal, originating from the redox-active reporter, decreases with increasing concentration of amplicons (and the target DNA in the original sample).
    </p>
    <p id="p-23">We performed qRPA using TriSilix with MB as the redox-active reporter. We used the TwistAmp Basic kit (TwistDx, UK) with the positive (3.2 kbp template, 144-bp amplicons) and negative (100-bp CTX-M ESBL; extended-spectrum beta-lactamases) DNA controls in the experiments. We have slightly modified the mastermix solution provided in the kit to contain 10 μg mL
     <sup>−1</sup> of MB to enable electrochemical detection in real time. 30 μL of the modified-mastermix was introduced into sample reservoir of TriSilix followed by the addition of 10 μL of mineral oil to prevent evaporation. The sample was heated to 40 °C for 40 min and a square-wave voltammogram was recorded every 2.5 min during the reaction. As shown in 
     <strong>
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>,
     </strong> while the electroanalytical signal (normalized peak current intensity) remained relatively unchanged for the negative control, the positive control (50 – 150 nM according to the manufacturer) showed a large drop in signal in the form of a characteristic sigmoidal profile. This result demonstrates that TriSilix can perform real-time and quantitative DNA analysis through (isothermal) RPA reactions.
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) qRPA measurements performed using TriSilix. TriSilix produces a clear signal between the positive and negative control sample within 10 min (N=5). (B) Titration curve for direct electrochemical detection of genomic DNA from MAP K10 DNA without amplification over 0 to 200 pg (N=3). (C) qPCR measurements performed using TriSilix of genomic DNA from MAP K10 against CTMX-ESBL as negative control (0 fg). (D) TriSilix qPCR measurement for a synthetic fragment of cDNA from SARS-CoV-2 against a synthetic fragment of cDNA from SARS-CoV (2003) as negative sample. Clear separation of signals after 20 cycles (~40 mins), indicating rapid, on-site detection is possible using TriSilix (N=3). Error bars in all plots=SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1496110387" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; qRPA measurements performed using TriSilix. TriSilix produces a clear signal between the positive and negative control sample within 10 min (N=5). &lt;;strong&gt;;(B)&lt;;/strong&gt;; Titration curve for direct electrochemical detection of genomic DNA from MAP K10 DNA without amplification over 0 to 200 pg (N=3). (&lt;;strong&gt;;C)&lt;;/strong&gt;; qPCR measurements performed using TriSilix of genomic DNA from MAP K10 against CTMX-ESBL as negative control (0 fg). &lt;;strong&gt;;(D)&lt;;/strong&gt;; TriSilix qPCR measurement for a synthetic fragment of cDNA from SARS-CoV-2 against a synthetic fragment of cDNA from SARS-CoV (2003) as negative sample. Clear separation of signals after 20 cycles (~40 mins), indicating rapid, on-site detection is possible using TriSilix (N=3). Error bars in all plots=SD.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F5.medium.gif" width="440" height="392"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F5.medium.gif" width="440" height="392"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/25/2020.03.23.002931/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 5.</span>
      <p id="p-24" class="first-child">
       <strong>(A)</strong> qRPA measurements performed using TriSilix. TriSilix produces a clear signal between the positive and negative control sample within 10 min (N=5). 
       <strong>(B)</strong> Titration curve for direct electrochemical detection of genomic DNA from MAP K10 DNA without amplification over 0 to 200 pg (N=3). (
       <strong>C)</strong> qPCR measurements performed using TriSilix of genomic DNA from MAP K10 against CTMX-ESBL as negative control (0 fg). 
       <strong>(D)</strong> TriSilix qPCR measurement for a synthetic fragment of cDNA from SARS-CoV-2 against a synthetic fragment of cDNA from SARS-CoV (2003) as negative sample. Clear separation of signals after 20 cycles (~40 mins), indicating rapid, on-site detection is possible using TriSilix (N=3). Error bars in all plots=SD.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-25">In the next experiment, we have performed qPCR analysis of genomic DNA extracted from clinical samples of 
     <em>Mycobacterium avium</em> subspecies 
     <em>paratuberculosis</em> (MAP) K10 strain. MAP is the causative microorganism of Johne’s Disease affecting ruminant animals and is controversially associated with Crohn’s Disease in humans. MAP is, therefore, an important analytical target which requires routine testing in centralized laboratories through microbial cultures or conventional fluorescence qPCR (lasting up to 3 hours). 
     <sup>
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>–
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>
     </sup> Using TriSilix, we have first performed a titration experiment to determine the limit-of-detection (LoD) of genomic DNA from MAP K10 without amplification over a range of 0-200 pg (
     <strong>
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>
     </strong>). Once again, we performed SWV as the analytical method and measured anodic and cathodic peak current intensities originating from MB as the electroanalytical signal. Without DNA amplification, we were able to detect genomic DNA of MAP K10 (4,829 kbp) down to an LoD of 0.8 pg from cathodic peak current intensities in comparison to a LoD of 1.2 pg provided by the anodic peak current intensities; the cathodic peak current intensity was, therefore, used as the electroanalytical signal in the following PCR experiments. We performed qPCR using the TriSilix chip to amplify and quantify small amounts (0 – 500 fg) of DNA of MAP K10 in real-time (
     <strong>
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>
     </strong>). The forward primer (5’-GCC GCG CTG CTG GAG TTG A-3’) and reverse primer (5’- CGC GGC ACG GCT CTT GTT -3’), were used to amplify a 563 nucleotide segment of IS
     <em>900</em> (204–766 of GenBank Accesion Number 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AE016958.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom">AE016958.1</a>; National Center for Biotechnology Information, USA), which has 17 repeats in the genome of MAP K10. Using the qPCR approach shown in Figure S7, we were able to measure as low as 20 fg of genomic DNA of MAP K10 which is equivalent to detection of a single bacterium in the sample. We also characterized the same samples using a commercial laboratory qPCR (IDvet, UK) using a test manufactured by ID Gene™ as a gold-standard (
     <strong>Figure S8</strong>). The results produced by TriSilix were similar or better in comparison to those produced by the commercially available, sophisticated laboratory instrument; for 20 fg of MAP K10 DNA, the resulting C
     <sub>t</sub> (cycle threshold) value was 30 using TriSilix, 5 cycles shorter compared to the value obtained using the commercial qPCR (C
     <sub>t</sub>=35).
    </p>
    <p id="p-26">In a final experiment, using TriSilix, we performed qPCR analysis of the complimentary DNA (cDNA) of SARS-CoV-2 (22712–22869 nucleotides of GenBank accession number 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom">MN908947</a>)
     <sup>
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
     </sup> the causative agent of COVID-19, (
     <strong>
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>
     </strong>). We have selected the forward primer (5’-CCTA CTA AAT TAA ATG ATC TCT GCT TTA CT-3’) and reverse primer (5’-CAA GCT ATA ACG CAG CCT GTA -3’), to amplify the target sequence. As a negative control, we used a cDNA fragment of another coronavirus, SARS-CoV (17741–17984 nucleotides of GenBank accession number 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY274119&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom">AY274119</a>)
     <sup>
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
     </sup> responsible for the SARS outbreak in 2003. Because we did not have access to patient samples, we used cDNA fragments produced synthetically by Integrated DNA Technologies Inc. We used the same PCR thermal cycling and measurement program described in the previous experiment involving MAP K10. Using TriSilix, we were able detect 1 pg of cDNA of SARS-CoV-2 quantitatively, in real-time with specificity against a cDNA sequence from a similar virus (SARS-CoV), in as low as 20 cycles of PCR (lasting ~40 min). After 35 cycles, the difference in the electroanalytical signal was five times, showing clear separation.
    </p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">6. Conclusions</h2>
    <p id="p-27">TriSilix is a low-cost, high-precision, integrated nucleic acid amplification and detection technology that is ideally suited for point-of-need diagnosis of infectious pathogens (bacteria, viruses etc.) affecting, humans, animals and plants. TriSilix can perform both isothermal and cyclic nucleic acid amplification; integrates temperature control and DNA detection on the same device, is label-free, requires minimal sample handling and allow operation by minimally trained personnel.</p>
    <p id="p-28">Although TriSilix is a silicon-based technology, it does not need a cleanroom for fabrication; chips can be produced in a standard wet-lab with easily accessible laboratory equipment worldwide. Because fabrication in an advanced silicon foundry is not needed, reliance on the global supply chains is substantially reduced; silicon foundries are located only in a few countries in the world. Each TriSilix chip costs ~0.35 USD (cost of materials is shown in 
     <strong>Table S9</strong>) when produced at wafer-scale with 4-inch wafers but the cost could be reduced further by moving to 8-inch wafers, decreasing the size of each chip and optimizing the fabrication process (for example, the gold contacts at the bottom of the TriSilix chip could be replaced by another appropriate metal to form Ohmic contacts). Due to its small size, integrated form-factor, and low-power requirements, TriSilix can be controlled with a portable, battery-operated handheld analyzer. Using a Li-ion battery with a 4000 mAh capacity (a typical rating for batteries in modern smartphones), TriSilix is estimated to perform at least 40 tests using isothermal amplification reactions at 40 °C lasting 30 min or 13 tests using 35 cycles of PCR (these estimates assume that the electronics for the handheld analyzer also draw 50 mA while operating).
    </p>
    <p id="p-29">The TriSilix technology has the following three disadvantages: (i) currently TriSilix uses MB as the redox-active reporter to quantify the products of the NA amplification reactions in real-time. MB is known to polymerize and its activity also slightly decreases over time at elevated temperatures. This issue, however, can be addressed by the use of redox-active metal-complexes that also intercalate with DNA, several of which have already been reported in the literature for real-time NA amplification
     <sup>
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,
      <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>,
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>–
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
     </sup> (ii) TriSilix currently requires purified DNA samples for operation. (iii) Because sample reservoir and polymer films are attached thermally at 110 °C, TriSilix cannot be operated beyond this temperature without causing debonding of the layers and leaks from the sample reservoir. For most NA amplification processes, however, 110 °C is sufficiently high. In case higher temperatures are needed (for example for on-chip sample prep), the fabrication process would need to be modified and thermoplastics with a higher melting point could be used.
    </p>
    <p id="p-30">In the future, we will design a handheld analyzer and include all electronics needed for the operation of TriSilix in the same device to simplify and enable its use in the field (currently the potentiostat for electrochemical NA sensing and driver electronics for heating + temperature sensing are separate). We will also add capabilities to communicate with smartphones to have access to the cloud so that the result of each diagnostic test can be passed to health agencies remotely.
     <sup>
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>,
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
     </sup> To enable on-site testing, we will implement on-chip lysis capabilities (such as mechanical
     <sup>
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
     </sup> or heat-induced cell lysis
     <sup>
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>
     </sup>) to analyze the samples taken directly from the subject without further sample prep. We will also implement on-chip reverse transcription to enable analysis of RNA. TriSilix would be particularly useful in emergency situations, such as the COVID-19 pandemic, where rapid, early, on-site detection of the virus would accelerate intervention and reduce the spread of the pathogen. Since TriSilix is a low-cost, high-performance NA detection technology, it is expected to find applications beyond infectious diseases which may include genetic analysis of various plants, animals or detection of genetic diseases. TriSilix is a versatile platform and could also be adapted for electrochemical detection of non-genetic targets (
     <em>e.g.,</em> through the use of antibodies) with on-chip incubation capabilities to deliver accelerated results at the Point-of-Need.
    </p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">7. Methods and Materials</h2>
    <div id="sec-8" class="subsection">
     <h3>Fabrication</h3>
     <div id="sec-9" class="subsection">
      <h4>Preparation of pSi</h4>
      <p id="p-31">Silicon wafers (p-type Siegert Wafer, 525 ± 25 μm thickness, 0−100 Ω·cm, lightly doped) were cleaned with acetone, rinsed with distilled water, and then immersed in a 3:1 piranha solution (95% H
       <sub>2</sub>SO
       <sub>4</sub>/30% H
       <sub>2</sub>O
       <sub>2</sub> v/ v) at 80 °C to remove organic residues. The wafers were dried in air and immersed in a batch containing an aqueous solution of 80 μM KAuCl
       <sub>4</sub> and 0.5% HF for 20 s to deposit Au particles on the surface of the wafer electrolessly. The Si wafers were rinsed with distilled water, dried and immersed in an etching bath (30% H
       <sub>2</sub>O
       <sub>2</sub>: 10% HF, 1:20 v/v ratio) for MACE with for 10 min. All chemicals, unless otherwise stated, were purchased from Sigma.
      </p>
     </div>
     <div id="sec-10" class="subsection">
      <h4>Patterning and heat-transfer of polymer layers</h4>
      <p id="p-32">PE sheets were laser-cut (Model: Trotec Speedy 100; Wavelength: 1μm; Power: 20 W) and the patterns were transferred to the porous surfaces of the Si wafer (both front and back) by heat-pressing at 180 °C for 5 min. The regions cut out were later electroplated to form the WE and electrical contacts. The Cu-PET (20 μm Cu; 23 μm PET) sheets were purchased from UK Insulations Ltd. and laminated with an additional sheet of PE film at 180 °C for 5 min in a heat press to improve thermal bonding on pSi surfaces. The Cu-PET-PE laminates were also laser-cut using a fiber laser, electroplated with Ag and heat-pressed on the top side of the Si wafer to form the electrodes for the CE and RE.</p>
     </div>
     <div id="sec-11" class="subsection">
      <h4>Electrodeposition of Au ang Ag</h4>
      <p id="p-33">Immediately before electroplating with Au to form the electrical contacts at the bottom of the wafer and WE on the top, each Si wafer was dipped into 5% HF to remove the native oxide and produce an electrically conductive surface. The wafers were electroplated with Au in a bath containing 10 mM KAu(CN)
       <sub>2</sub>/KCN using a 5 cm x 5 cm Pt electrode (SPA plating) as anode at a constant current of 10 mA. The Cu-PET-PE film was first treated with 0.1 M H
       <sub>2</sub>SO
       <sub>4</sub>, rinsed and electroplated with Ag in an aqueous solution of 10 mM AgCN/KCN under an applied current of 20 mA using a 5 cm x 5 cm Pt electrode as anode. All chemicals, unless otherwise stated, were purchased from Sigma.
      </p>
     </div>
    </div>
    <div id="sec-12" class="subsection">
     <h3>DNA Amplification and Extraction</h3>
     <div id="sec-13" class="subsection">
      <h4>RPA</h4>
      <p id="p-34">The RPA Premix solution was prepared by adding and vortexing 25 μL of 2X buffer, 8 μL of 10 mM dNTPs (Fisher Scientific), 5 μL of 10X E Mix and 5 μL of control primer Mix (30 bp) in a 0.2 mL Eppendorf tube. 2.5 μL of 20X core solution and 1 μL of 0.05 % MB were placed in the lid of the tube and mixed with 10 inversions. The RPA Mastermix was made by mixing 3.5 μL of Nuclease-free ultrapure water (Fisher Scientific) with the Premix. This solution was subsequently mixed with 2.5 μL of 280 mM OAc and 1 μL of positive control DNA or 50 nM negative control (CTX-M ESBL: 5’-ATTGACGTGC TTTTCCGCAA TCGGATTATA GTTAACAAGG TCAGATTTTT TGATCTCAAC TCGCTGATTT AACAGATTCG GTTCGCTTTC ACTTTTCTTC-3’; Sigma) to create the final RPA mix. 30 μL of the final solution were added into the device chamber followed by 10 μL of mineral oil (Sigma) to prevent evaporation of water. All of the chemicals (unless otherwise stated) were included in the TwistAmp® Basic Kit.</p>
     </div>
     <div id="sec-14" class="subsection">
      <h4>MAP K10 PCR</h4>
      <p id="p-35">50 μL of PCR mastermix solution in Nuclease-free ultrapure water contained 60 mM Tris-HCl Buffer pH 8.8 (Bio-Rad), 200 μM dNTPs, 200 nmol μL
       <sup>−1</sup> of each primer (Biomers), 0.5 U Taq DNA polymerase and 20 μg mL
       <sup>−1</sup> MB. 50 μL of PCR mix solution in nuclease-free water contained 60 mM Tris-HCl Buffer pH 8.8 (Bio-Rad), 200 μM dNTPs, 200 nmol μL
       <sup>−1</sup> of each primer (Biomers), 0.5 U Taq DNA polymerase, 2 mM MgCl
       <sub>2</sub>, 20 μg mL
       <sup>−1</sup> MB and 1 μL of MAP or Nuclease-free ultrapure water. 30 μL of PCR mastermix or PCR mix solutions were added into the sample reservoir of TriSilix for reaction, followed by 10 μL of mineral oil (Sigma) to prevent evaporation. All chemicals, unless otherwise stated, were purchased from Fisher Scientific.
      </p>
     </div>
     <div id="sec-15" class="subsection">
      <h4>SARS-CoV PCR</h4>
      <p id="p-36">The PCR Mix consisted of 0.5 U Taq polymerase, 1X Taq Polymerase buffer, 1 μM of each primer (Biomers), 2 mM MgCl
       <sub>2</sub>, 0.2 mM dNTPs, 30 μg mL
       <sup>−1</sup> MB and 1 pg cDNA (IDtdna). 30 μL of this solution were added into the sample reservoir of TriSilix for reaction, followed by 10 μL of mineral oil (Sigma) to prevent evaporation. All chemicals, unless otherwise stated, were purchased from Fisher Scientific.
      </p>
     </div>
     <div id="sec-16" class="subsection">
      <h4>Extraction of Genomic DNA</h4>
      <p id="p-37">Genomic DNA was prepared from a 10ml culture of MAP K10 in Middlebrook 7H9 supplemented with 10% Albumin Dextrose Catalase enrichment, 0.2% (vol/vol) glycerol, 0.05% (vol/vol) Tween 80 and 2μg/ml mycobactin j. Mycobacteria were harvested in early to mid log phase of growth and the cells were pelleted at room temperature for 5 min at 14,000g. Pellets were re-suspended in ATL buffer (Qiagen DNeasy® Blood &amp;; Tissue Kit) and the samples were transferred to Lysing Matrix B tubes (0.1mm silica spheres). Samples were homogenized in a FastPrep™ FP120 cell disruptor at 3 x 20s, Speed 6, followed by centrifugation at 14,000g for 5 minutes. Supernatants were transferred to fresh sterile micro centrifuge tubes and Proteinase K (Qiagen DNeasy® Kit) was added followed by incubation overnight at 56°C. DNA was extracted using DNeasy® Kit (Qiagen) according to the manufacturer’s protocol. Concentration (40 μg mL
       <sup>−1</sup>) and purity of the DNA were determined using a Nanodrop and the absorbance ratios 260/280 and 260/230.
      </p>
     </div>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Characterization</h3>
     <div id="sec-18" class="subsection">
      <h4>Microstructural Characterization</h4>
      <p id="p-38">Scanning electron micrographs were acquired using a Zeiss Gemini Sigma 300 FEG SEM at 5 keV electron beam energy. Optical micrographs were obtained with a Brunel SP202XM metallurgical microscope. A Nikon D3200 camera was attached to the microscope for the acquisition of the images using DigiCamControl open-source software.</p>
     </div>
     <div id="sec-19" class="subsection">
      <h4>Electrical and Thermal characterization</h4>
      <p id="p-39">The I-V curves for the characterization of the metal−semiconductor junctions were acquired using a Keithley 2450 SourceMetter. Thermal images were recorded using a FLIR E4 thermal camera.</p>
     </div>
     <div id="sec-20" class="subsection">
      <h4>Electrochemical Measurements</h4>
      <p id="p-40">Cyclic (CV) and square wave (SWV) voltammograms were acquired using a handheld potentiostat (PalmSens3, PalmSens BV, The Netherlands) with the supplied PSTrace 5.3 software in a three-electrode setup. The following parameters were used for all the SWV measurements: pulse amplitude of 100 mV, potential step of 5 mV and 50 Hz of frequency. Prior to the DNA measurements by SWV, the sample reservoir was cleaned with DNA decontamination reagent (Fisher Scientific) followed by Nuclease-free ultrapure water (Fisher Scientific).</p>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">8. Acknowledgements</h2>
    <p id="p-41">F.G., K.S. and E.N-B. would like to thank the Wellcome Trust (Grant No. 207687/Z/17/Z) and the UK Engineering and Physical Sciences Research Council (EPSRC) (EP/ R010242/1) and Imperial College, Department of Bioengineering for their generous support. M. Kasimatis acknowledges EPSRC DTP (Reference: 1846144) and A.C., BBSRC DTP (Reference: 2177734). Y.C. thanks the Turkish Ministry of Education. M. Kaisti acknowledges Finnish Foundation for Technology Promotion. F.G. and M.G. thank the Imperial College Centre for Plastic Electronics. F.G. also acknowledges Agri Futures Lab. We would like to thank George Caldow from SRUC (Midlothian, UK) for performing the commercial qPCR MAP tests and Dr Tolga Bozkurt (Imperial College) for the fruitful discussions.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">9. References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.1" data-doi="10.1016/j.bios.2015.04.041">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Replacing antibodies with aptamers in lateral flow immunoassay</span>. 
         <abbr class="cit-jnl-abbrev">Biosens. Bioelectron</abbr>. 
         <span class="cit-vol">71</span>, 
         <span class="cit-fpage">230</span>–
         <span class="cit-lpage">242</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiosens.%2BBioelectron%26rft.volume%253D71%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bios.2015.04.041%26rft_id%253Dinfo%253Apmid%252F25912679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bios.2015.04.041&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25912679&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.2" data-doi="10.1016/j.aca.2009.02.038">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ozdemir</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-article-title">Microfluidic DNA amplification:a review</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chim. Acta</abbr>
         <span class="cit-vol">638</span>, 
         <span class="cit-fpage">115</span>–
         <span class="cit-lpage">125</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytica%2BChimica%2BActa%26rft.stitle%253DAnalytica%2BChimica%2BActa%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D638%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D125%26rft.atitle%253DMicrofluidic%2BDNA%2Bamplification--a%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aca.2009.02.038%26rft_id%253Dinfo%253Apmid%252F19327449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.aca.2009.02.038&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19327449&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">S. H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Emerging ultrafast nucleic acid amplification technologies for next generation molecular diagnostics</span>. 
         <abbr class="cit-jnl-abbrev">Biosens. Bioelectron</abbr>
         <span class="cit-vol">141</span>, 
         <span class="cit-fpage">111448</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiosens.%2BBioelectron%26rft.volume%253D141%26rft.spage%253D111448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.4" data-doi="10.1039/c3lc50694k">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Menzel</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gübeli</span>, 
          <span class="cit-name-given-names">R. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Güder</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weber</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zacharias</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Detection of real-time dynamics of drug-target interactions by ultralong nanowalls</span>. 
         <abbr class="cit-jnl-abbrev">Lab Chip</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">4173</span>–
         <span class="cit-lpage">4179</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLab%2BChip%26rft.volume%253D13%26rft.spage%253D4173%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc3lc50694k%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1039/c3lc50694k&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.5" data-doi="10.1021/ac203466y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jokerst</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Development of a paper-based analytical device for colorimetric detection of select foodborne pathogens</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr>
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">2900</span>−
         <span class="cit-lpage">2907</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DJokerst%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D84%26rft.issue%253D6%26rft.spage%253D2900%26rft.epage%253D2907%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bpaper-based%2Banalytical%2Bdevice%2Bfor%2Bcolorimetric%2Bdetection%2Bof%2Bselect%2Bfoodborne%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac203466y%26rft_id%253Dinfo%253Apmid%252F22320200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ac203466y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22320200&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.6" data-doi="10.1038/nbt1106">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cu</span>, 
          <span class="cit-name-given-names">Y. T. H.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Multiplexed detection of pathogen DNA with DNA-based fluorescence nanobarcodes</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">885</span>−
         <span class="cit-lpage">889</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D23%26rft.issue%253D7%26rft.spage%253D885%26rft.epage%253D889%26rft.atitle%253DMultiplexed%2Bdetection%2Bof%2Bpathogen%2BDNA%2Bwith%2BDNA-based%2Bfluorescence%2Bnanobarcodes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1106%26rft_id%253Dinfo%253Apmid%252F15951805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt1106&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15951805&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000230427700037&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.7" data-doi="10.1021/ac2002214">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Donhauser</span>, 
          <span class="cit-name-given-names">S. C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niessner</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Seidel</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Sensitive quantification of Escherichia coli O157:H7, Salmonella enterica, and Campylobacter jejuni by combining stopped polymerase chain reaction with chemiluminescence flow-through DNA microarray analysis</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr>
         <span class="cit-vol">83</span>, 
         <span class="cit-fpage">3153</span>−
         <span class="cit-lpage">3160</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DDonhauser%26rft.auinit1%253DS.%2BC.%26rft.volume%253D83%26rft.issue%253D8%26rft.spage%253D3153%26rft.epage%253D3160%26rft.atitle%253DSensitive%2Bquantification%2Bof%2BEscherichia%2Bcoli%2BO157%253AH7%252C%2BSalmonella%2Benterica%2B%252C%2Band%2BCampylobacter%2Bjejuni%2Bby%2Bcombining%2Bstopped%2Bpolymerase%2Bchain%2Breaction%2Bwith%2Bchemiluminescence%2Bflow-through%2BDNA%2Bmicroarray%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac2002214%26rft_id%253Dinfo%253Apmid%252F21417213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ac2002214&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21417213&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">In situ hybridization chain reaction amplification for universal and highly sensitive electrochemiluminescent detection of DNA</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr>
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">7750</span>–
         <span class="cit-lpage">7755</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D84%26rft.issue%253D18%26rft.spage%253D7750%26rft.epage%253D7755%26rft.atitle%253DIn%2Bsitu%2Bhybridization%2Bchain%2Breaction%2Bamplification%2Bfor%2Buniversal%2Band%2Bhighly%2Bsensitive%2Belectrochemiluminescent%2Bdetection%2Bof%2BDNA.%26rft_id%253Dinfo%253Apmid%252F22924989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=22924989&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.9" data-doi="10.1073/pnas.1812296116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reboud</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Paper-based microfluidics for DNA diagnostics of malaria in low resource underserved rural communities</span>. 
         <abbr class="cit-jnl-abbrev">PNAS</abbr>
         <span class="cit-vol">116</span>, 
         <span class="cit-fpage">4834</span>–
         <span class="cit-lpage">4842</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1812296116%26rft_id%253Dinfo%253Apmid%252F30782834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzExLzQ4MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNS8yMDIwLjAzLjIzLjAwMjkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tsaloglou</span>, 
          <span class="cit-name-given-names">M.-N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Handheld isothermal amplification and electrochemical detection of DNA in resource-limited settings</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr>
         <span class="cit-vol">543</span>, 
         <span class="cit-fpage">116</span>–
         <span class="cit-lpage">121</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnal.%2BBiochem.%26rft.volume%253D543%26rft.spage%253D116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Real-time monitoring of strand-displacement DNA amplification by a contactless electrochemical microsystem using interdigitated electrodes</span>. 
         <abbr class="cit-jnl-abbrev">Lab Chip</abbr>
         <span class="cit-vol">12</span>, 
         <span class="cit-fpage">3190</span>–
         <span class="cit-lpage">3196</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLab%2Bon%2Ba%2Bchip%26rft.stitle%253DLab%2BChip%26rft.aulast%253DFang%26rft.auinit1%253DX.%26rft.volume%253D12%26rft.issue%253D17%26rft.spage%253D3190%26rft.epage%253D3196%26rft.atitle%253DReal-time%2Bmonitoring%2Bof%2Bstrand-displacement%2BDNA%2Bamplification%2Bby%2Ba%2Bcontactless%2Belectrochemical%2Bmicrosystem%2Busing%2Binterdigitated%2Belectrodes.%26rft_id%253Dinfo%253Apmid%252F22773155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=22773155&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.12" data-doi="10.1093/nar/gkl702">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yeung</span>, 
          <span class="cit-name-given-names">S.-W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">T. M.-H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cai</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsing</span>, 
          <span class="cit-name-given-names">I.-M.</span>
         </span>
         <span class="cit-article-title">A DNA biochip for on-the-spot multiplexed pathogen identification</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">e118</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl702%26rft_id%253Dinfo%253Apmid%252F17000638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkl702&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17000638&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Petralia</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A miniaturized silicon based device for nucleic acids electrochemical detection</span>. 
         <abbr class="cit-jnl-abbrev">Sens. Bio-Sensing Res.</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">90</span>–
         <span class="cit-lpage">94</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSens.%2BBio-Sensing%2BRes.%26rft.volume%253D6%26rft.spage%253D90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yeung</span>, 
          <span class="cit-name-given-names">S. S. W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">T. M. H.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsing</span>, 
          <span class="cit-name-given-names">I.-M.</span>
         </span>
         <span class="cit-article-title">Electrochemistry-based real-time PCR on a microchip</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr>
         <span class="cit-vol">80</span>, 
         <span class="cit-fpage">363</span>–
         <span class="cit-lpage">368</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DYeung%26rft.auinit1%253DS.%2BS.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D363%26rft.epage%253D368%26rft.atitle%253DElectrochemistry-based%2Breal-time%2BPCR%2Bon%2Ba%2Bmicrochip.%26rft_id%253Dinfo%253Apmid%252F18088101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=18088101&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">T. M.-H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carles</span>, 
          <span class="cit-name-given-names">M. C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsing</span>, 
          <span class="cit-name-given-names">I.-M.</span>
         </span>
         <span class="cit-article-title">Microfabricated PCR-electrochemical device for simultaneous DNA amplification and detection</span>. 
         <abbr class="cit-jnl-abbrev">Lab Chip</abbr>
         <span class="cit-fpage">100</span>–
         <span class="cit-lpage">10</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Güder</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Toward discrete multilayered composite structures: do hollow networks form in a polycrystalline infinite nanoplane by the Kirkendall effect?</span>
         <abbr class="cit-jnl-abbrev">Chem. Mater.</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">4445</span>–
         <span class="cit-lpage">4451</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChem.%2BMater.%26rft.volume%253D23%26rft.spage%253D4445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grell</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Autocatalytic metallization of fabrics using Si ink, for biosensors, batteries and energy harvesting</span>. 
         <abbr class="cit-jnl-abbrev">Adv. Funct. Mater.</abbr>
         <span class="cit-vol">29</span>, 
         <span class="cit-fpage">1804798</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdv.%2BFunct.%2BMater.%26rft.volume%253D29%26rft.spage%253D1804798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Güder</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kucukbayrak</span>, 
          <span class="cit-name-given-names">U. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zahcarias</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Tracing the migration history of metal catalysts in metal-assisted chemically etched Silicon</span>. 
         <abbr class="cit-jnl-abbrev">ACS Nano</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">1583</span>–
         <span class="cit-lpage">1590</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DACS%2BNano%26rft.volume%253D7%26rft.spage%253D1583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kasimatis</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Monolithic solder-on nanoporous Si-Cu contacts for stretchable silicone composite</span>. 
         <abbr class="cit-jnl-abbrev">ACS Appl. Mater. Interfaces</abbr>
         <span class="cit-vol">11</span>, 
         <span class="cit-fpage">47577</span>−
         <span class="cit-lpage">47586</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DACS%2BAppl.%2BMater.%2BInterfaces%26rft.volume%253D11%26rft.spage%253D47577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gesele</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Linsmeierdag</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drachdag</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frickedag</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Arens-Fischer</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Temperature-dependent thermal conductivity of porous silicon</span>. 
         <abbr class="cit-jnl-abbrev">J. Phys. D Appl. Phys.</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">2911</span>–
         <span class="cit-lpage">2915</span> (
         <span class="cit-pub-date">1997</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BD%2BAppl.%2BPhys.%26rft.volume%253D30%26rft.spage%253D2911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Weisse</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Thermal conductivity in porous silicon nanowire arrays</span>. 
         <abbr class="cit-jnl-abbrev">Nanoscale Res Lett</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">554</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNanoscale%2BRes%2BLett%26rft.aulast%253DWeisse%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D554%26rft.epage%253D554%26rft.atitle%253DThermal%2Bconductivity%2Bin%2Bporous%2Bsilicon%2Bnanowire%2Barrays.%26rft_id%253Dinfo%253Apmid%252F23039084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=23039084&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Boukai</span>, 
          <span class="cit-name-given-names">A. I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Silicon nanowires as efficient thermoelectric materials</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-fpage">168</span>–
         <span class="cit-lpage">171</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Deféver</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Druet</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Evrard</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marchal</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Limoges</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">Real-Time electrochemical PCR with a DNA intercalating redox probe</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr>
         <span class="cit-vol">83</span>, 
         <span class="cit-fpage">1815</span>–
         <span class="cit-lpage">1821</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem.%26rft.volume%253D83%26rft.spage%253D1815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.24" data-doi="10.1016/j.bios.2008.11.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Huey Fang, Naveen Ramalingam, Dong Xian-Dui, Tan Swee Ngin, Zeng Xianting, Annie Tan Lai Kuan, Eric Yap Peng Huat, GongHai-Qing</span>. 
         <abbr class="cit-jnl-abbrev">Biosens. Bioelectron</abbr>. 
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">2131</span>–
         <span class="cit-lpage">2136</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiosensors%2B%2526%2Bbioelectronics%26rft.stitle%253DBiosens%2BBioelectron%26rft.aulast%253DFang%26rft.auinit1%253DT.%2BH.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D2131%26rft.epage%253D2136%26rft.atitle%253DReal-time%2BPCR%2Bmicrofluidic%2Bdevices%2Bwith%2Bconcurrent%2Belectrochemical%2Bdetection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bios.2008.11.009%26rft_id%253Dinfo%253Apmid%252F19162460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bios.2008.11.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19162460&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Won</span>, 
          <span class="cit-name-given-names">B. Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Investigation of the signaling mechanism and verification of the performance of an electrochemical real-time PCR system based on the interaction of methylene blue with DNA</span>. 
         <abbr class="cit-jnl-abbrev">Analyst</abbr>. 
         <span class="cit-vol">136</span>, 
         <span class="cit-fpage">1573</span>–
         <span class="cit-lpage">1579</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalyst%2B%2528London%2529%26rft.stitle%253DAnalyst%2B%2528London%2529%26rft.aulast%253DWon%26rft.auinit1%253DB.%2BY.%26rft.volume%253D136%26rft.issue%253D8%26rft.spage%253D1573%26rft.epage%253D1579%26rft.atitle%253DInvestigation%2Bof%2Bthe%2Bsignaling%2Bmechanism%2Band%2Bverification%2Bof%2Bthe%2Bperformance%2Bof%2Ban%2Belectrochemical%2Breal-time%2BPCR%2Bsystem%2Bbased%2Bon%2Bthe%2Binteraction%2Bof%2Bmethylene%2Bblue%2Bwith%2BDNA.%26rft_id%253Dinfo%253Apmid%252F21321686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=21321686&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.26" data-doi="10.1002/elan.200302875">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wildgoose</span>, 
          <span class="cit-name-given-names">G. G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Giovanelli</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lawrence</span>, 
          <span class="cit-name-given-names">N. S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Compton</span>, 
          <span class="cit-name-given-names">R. G.</span>
         </span>
         <span class="cit-article-title">High-temperature electrochemistry: A review</span>. 
         <abbr class="cit-jnl-abbrev">Electroanalysis</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">421</span>–
         <span class="cit-lpage">433</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DElectroanalysis%26rft.volume%253D16%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felan.200302875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/elan.200302875&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Watanabe</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mizuguchi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashi</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Toyama</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-article-title">The effect of temperature on the electrochemical properties of copper-DNA membranes</span>. 
         <abbr class="cit-jnl-abbrev">Biosensors</abbr>. 
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">117</span>–
         <span class="cit-lpage">125</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schulzke</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-article-title">Temperature dependent electrochemistry—a versatile tool for investigations of biology related topics</span>. 
         <abbr class="cit-jnl-abbrev">Dalt. Trans.</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">6683</span>–
         <span class="cit-lpage">6691</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDalt.%2BTrans.%26rft.volume%253D34%26rft.spage%253D6683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.29" data-doi="10.1016/j.vetmic.2007.07.016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mobius</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hotzel</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rassbach</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kohler</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">Comparison of 13 single-round and nested PCR assays targeting IS900, ISMav2, f57 and locus 255 for detection of Mycobacterium avium subsp. paratuberculosis</span>. 
         <abbr class="cit-jnl-abbrev">Vet Microbiol.</abbr>
         <span class="cit-vol">126</span>, 
         <span class="cit-fpage">324</span>–
         <span class="cit-lpage">333</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DMobius%26rft.auinit1%253DP.%26rft.volume%253D126%26rft.issue%253D4%26rft.spage%253D324%26rft.epage%253D333%26rft.atitle%253DComparison%2Bof%2B13%2Bsingle-round%2Band%2Bnested%2BPCR%2Bassays%2Btargeting%2BIS900%252C%2BISMav2%252C%2Bf57%2Band%2Blocus%2B255%2Bfor%2Bdetection%2Bof%2BMycobacterium%2Bavium%2Bsubsp.%2Bparatuberculosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2007.07.016%26rft_id%253Dinfo%253Apmid%252F17889455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2007.07.016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17889455&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000252497700003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.30" data-doi="10.1016/j.vetmic.2004.02.006">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vansnick</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Newly developed primers for the detection of Mycobacterium avium subspecies paratuberculosis</span>. 
         <abbr class="cit-jnl-abbrev">Vet Microbiol.</abbr>
         <span class="cit-vol">100</span>, 
         <span class="cit-fpage">197</span>–
         <span class="cit-lpage">204</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DVansnick%26rft.auinit1%253DE.%26rft.volume%253D100%26rft.issue%253D3-4%26rft.spage%253D197%26rft.epage%253D204%26rft.atitle%253DNewly%2Bdeveloped%2Bprimers%2Bfor%2Bthe%2Bdetection%2Bof%2BMycobacterium%2Bavium%2Bsubspecies%2Bparatuberculosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2004.02.006%26rft_id%253Dinfo%253Apmid%252F15145498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2004.02.006&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15145498&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.31" data-doi="10.1128/JCM.03233-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Plain</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marsh</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Waldron</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Galea</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Whittington</span>, 
          <span class="cit-name-given-names">AM</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-surname">Saunders</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">High-throughput direct fecal PCR assay for detection of Mycobacterium avium subsp. paratuberculosis in sheep and cattle</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-vol">52</span>, 
         <span class="cit-fpage">745</span>–
         <span class="cit-lpage">57</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.03233-13%26rft_id%253Dinfo%253Apmid%252F24352996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjUyLzMvNzQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjUvMjAyMC4wMy4yMy4wMDI5MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">J. F.-W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">514</span>–
         <span class="cit-lpage">523</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.33" data-doi="10.1373/clinchem.2004.032011">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L. L. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Rapid detection of the Severe Acute Respiratory Syndrome (SARS) Coronavirus by a Loop-mediated isothermal amplification assay</span>. 
         <abbr class="cit-jnl-abbrev">Clin. Chem.</abbr>
         <span class="cit-vol">50</span>, 
         <span class="cit-fpage">1050</span>–
         <span class="cit-lpage">1052</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2004.032011%26rft_id%253Dinfo%253Apmid%252F15054079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6OToiNTAvNi8xMDUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjUvMjAyMC4wMy4yMy4wMDI5MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Deféver</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Real-time electrochemical monitoring of the polymerase chain reaction by mediated redox catalysis</span>, 
         <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc.</abbr>
         <span class="cit-vol">131</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-fpage">11433</span>–
         <span class="cit-lpage">11441</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DDefever%26rft.auinit1%253DT.%26rft.volume%253D131%26rft.issue%253D32%26rft.spage%253D11433%26rft.epage%253D11441%26rft.atitle%253DReal-time%2Belectrochemical%2Bmonitoring%2Bof%2Bthe%2Bpolymerase%2Bchain%2Breaction%2Bby%2Bmediated%2Bredox%2Bcatalysis.%26rft_id%253Dinfo%253Apmid%252F19722651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=19722651&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F25%2F2020.03.23.002931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.002931v1.35" data-doi="10.2174/157341111793797617">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ferapontova</span>, 
          <span class="cit-name-given-names">E. E.</span>
         </span>
         <span class="cit-article-title">Electrochemical indicators for DNA electroanalysis</span>,. 
         <abbr class="cit-jnl-abbrev">Curr. Anal. Chem</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">51</span>–
         <span class="cit-lpage">62</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr.%2BAnal.%2BChem%26rft.volume%253D7%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.2174%252F157341111793797617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2174/157341111793797617&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Patterson</span>, 
          <span class="cit-name-given-names">A. S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsieh</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Soh</span>, 
          <span class="cit-name-given-names">H. T.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Plaxco</span>, 
          <span class="cit-name-given-names">K. W.</span>
         </span>
         <span class="cit-article-title">Electrochemical real-time nucleic acid amplification: towards point-of-care quantification of pathogens</span>. 
         <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">705</span>–
         <span class="cit-lpage">712</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baradun</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cellulose fibers enable near zero-cost electrical sensing of water-soluble gases</span>. 
         <abbr class="cit-jnl-abbrev">ACS Sensors</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">1662</span>–
         <span class="cit-lpage">1669</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DACS%2BSensors%26rft.volume%253D4%26rft.spage%253D1662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Güder</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Paper-based electrical respiration S sensor</span>. 
         <abbr class="cit-jnl-abbrev">Angew. Chemie Int. Ed.</abbr>
         <span class="cit-vol">55</span>, 
         <span class="cit-fpage">5727</span>–
         <span class="cit-lpage">5732</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAngew.%2BChemie%2BInt.%2BEd.%26rft.volume%253D55%26rft.spage%253D5727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hansen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Development of rapid extraction method of Mycobacterium avium subspecies paratuberculosis DNA from bovine stool samples</span>. 
         <abbr class="cit-jnl-abbrev">Diagnostics</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">36</span>–
         <span class="cit-lpage">44</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDiagnostics%26rft.volume%253D6%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.23.002931v1.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heiniger</span>, 
          <span class="cit-name-given-names">E. K.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Comparison of point-of-care-compatible lysis methods for bacteria and viruses</span>. 
         <abbr class="cit-jnl-abbrev">J Microbiol Methods</abbr>
         <span class="cit-vol">128</span>, 
         <span class="cit-fpage">80</span>–
         <span class="cit-lpage">87</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMicrobiol%2BMethods%26rft.volume%253D128%26rft.spage%253D80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
