<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.03.15.991844.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.15.991844v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Emergence of RBD mutations from circulating SARS-CoV-2 strains with enhanced structural stability and higher human ACE2 receptor affinity of the spike protein"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.15.991844"/>
  <meta name="DC.Date" content="2020-04-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Spike protein receptor binding domain (RBD) of SARS-CoV-2 is the critical determinant of viral tropism and infectivity. Genomes of global SARS-CoV-2 strains were analyzed, yielding 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types circulating in Wuhan, Shenzhen, Hong Kong, and France, displayed enhanced structural stability along with higher human ACE2 receptor affinity of the spike protein, indicating these mutants may have acquired increased infectivity to humans. ### Competing Interest Statement"/>
  <meta name="DC.Contributor" content="Junxian Ou"/>
  <meta name="DC.Contributor" content="Zhonghua Zhou"/>
  <meta name="DC.Contributor" content="Ruixue Dai"/>
  <meta name="DC.Contributor" content="Jing Zhang"/>
  <meta name="DC.Contributor" content="Wendong Lan"/>
  <meta name="DC.Contributor" content="Shan Zhao"/>
  <meta name="DC.Contributor" content="Jianguo Wu"/>
  <meta name="DC.Contributor" content="Donald Seto"/>
  <meta name="DC.Contributor" content="Lilian Cui"/>
  <meta name="DC.Contributor" content="Gong Zhang"/>
  <meta name="DC.Contributor" content="Qiwei Zhang"/>
  <meta name="article:published_time" content="2020-04-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Emergence of RBD mutations from circulating SARS-CoV-2 strains with enhanced structural stability and higher human ACE2 receptor affinity of the spike protein"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Spike protein receptor binding domain (RBD) of SARS-CoV-2 is the critical determinant of viral tropism and infectivity. Genomes of global SARS-CoV-2 strains were analyzed, yielding 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types circulating in Wuhan, Shenzhen, Hong Kong, and France, displayed enhanced structural stability along with higher human ACE2 receptor affinity of the spike protein, indicating these mutants may have acquired increased infectivity to humans.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.15.991844v3"/>
  <meta name="citation_id" content="2020.03.15.991844v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.991844v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.991844v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.991844v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.15.991844"/>
  <meta name="citation_num_pages" content="25"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.15.991844"/>
  <meta name="citation_author" content="Junxian Ou"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Tropical Disease Research, School of Public Health, Southern Medical University"/>
  <meta name="citation_author" content="Zhonghua Zhou"/>
  <meta name="citation_author_institution" content="Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes, Institute of Life and Health Engineering, College of Life Science and Technology, Jinan University"/>
  <meta name="citation_author" content="Ruixue Dai"/>
  <meta name="citation_author_institution" content="Department of Environmental Science and Engineering, Fudan University"/>
  <meta name="citation_author_institution" content="Novoprotein Scientific Inc"/>
  <meta name="citation_author" content="Jing Zhang"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Virology, Institute of Medical Microbiology, Jinan University"/>
  <meta name="citation_author" content="Wendong Lan"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Tropical Disease Research, School of Public Health, Southern Medical University"/>
  <meta name="citation_author" content="Shan Zhao"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Tropical Disease Research, School of Public Health, Southern Medical University"/>
  <meta name="citation_author" content="Jianguo Wu"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Virology, Institute of Medical Microbiology, Jinan University"/>
  <meta name="citation_author" content="Donald Seto"/>
  <meta name="citation_author_institution" content="Bioinformatics and Computational Biology Program, School of Systems Biology, George Mason University"/>
  <meta name="citation_author" content="Lilian Cui"/>
  <meta name="citation_author_institution" content="Novoprotein Scientific Inc"/>
  <meta name="citation_author" content="Gong Zhang"/>
  <meta name="citation_author_institution" content="Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes, Institute of Life and Health Engineering, College of Life Science and Technology, Jinan University"/>
  <meta name="citation_author_email" content="zhangqw@smu.edu.cn"/>
  <meta name="citation_author_email" content="zhanggong@jnu.edu.cn"/>
  <meta name="citation_author" content="Qiwei Zhang"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Tropical Disease Research, School of Public Health, Southern Medical University"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Virology, Institute of Medical Microbiology, Jinan University"/>
  <meta name="citation_author_email" content="zhangqw@smu.edu.cn"/>
  <meta name="citation_author_email" content="zhanggong@jnu.edu.cn"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2770-111X"/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med. 2020:727–733. doi:10.1056/nejmoa2001017"/>
  <meta name="citation_reference" content="Li Q, Guan X, Wu P, et al. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia. N Engl J Med. 2020:1–9. doi:10.1056/nejmoa2001316"/>
  <meta name="citation_reference" content="Wang D, Hu B, Hu C, et al. Clinical Characteristics of 138 Hospitalized Patients with 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. JAMA - J Am Med Assoc. 2020:1–9. doi:10.1001/jama.2020.1585"/>
  <meta name="citation_reference" content="Chan JFW, Yuan S, Kok KH, et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet. 2020;395(10223):514–523. doi:10.1016/S0140-6736(20)30154-9"/>
  <meta name="citation_reference" content="Mariana N. Coronavirus disease 2019 (COVID-19). 2020;2019(April)."/>
  <meta name="citation_reference" content="Hoffmann M, Kleine-Weber H, Schroeder S, et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell. March 2020:S0092-8674(20)30229–4. doi:10.1016/j.cell.2020.02.052"/>
  <meta name="citation_reference" content="Chen Y, Guo Y, Pan Y, Zhao ZJ. Structure analysis of the receptor binding of 2019-nCoV. Biochem Biophys Res Commun. 2020;2(xxxx):0–5. doi:10.1016/j.bbrc.2020.02.071"/>
  <meta name="citation_reference" content="Wrapp D, Wang N, Corbett KS, et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (80-). 2020;367(6483):1260 LP – 1263. doi:10.1126/science.abb2507"/>
  <meta name="citation_reference" content="Wan Y, Shang J, Graham R, Baric RS, Li F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol. 2020;(January). doi:10.1128/jvi.00127-20"/>
  <meta name="citation_reference" content="Rozas J, Ferrer-Mata A, Sanchez-DelBarrio JC, et al. DnaSP 6: DNA sequence polymorphism analysis of large data sets. Mol Biol Evol. 2017;34(12):3299–3302. doi:10.1093/molbev/msx248"/>
  <meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, et al. SWISS-MODEL: Homology modelling of protein structures and complexes. Nucleic Acids Res. 2018;46(W1):W296–W303. doi:10.1093/nar/gky427"/>
  <meta name="citation_reference" content="Lam TT-Y, Shum MH-H, Zhu H-C, et al. Identifying SARS-CoV-2 related coronaviruses in Malayan pangolins. Nature. 2020. doi:10.1038/s41586-020-2169-0"/>
  <meta name="citation_reference" content="Xiao K, Zhai J, Feng Y, et al. Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins. bioRxiv. January 2020:2020.02.17.951335. doi:10.1101/2020.02.17.951335"/>
  <meta name="twitter:title" content="Emergence of RBD mutations from circulating SARS-CoV-2 strains with enhanced structural stability and higher human ACE2 receptor affinity of the spike protein"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Spike protein receptor binding domain (RBD) of SARS-CoV-2 is the critical determinant of viral tropism and infectivity. Genomes of global SARS-CoV-2 strains were analyzed, yielding 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types circulating in Wuhan, Shenzhen, Hong Kong, and France, displayed enhanced structural stability along with higher human ACE2 receptor affinity of the spike protein, indicating these mutants may have acquired increased infectivity to humans. ### Competing Interest Statement"/>
  <meta name="og-title" property="og:title" content="Emergence of RBD mutations from circulating SARS-CoV-2 strains with enhanced structural stability and higher human ACE2 receptor affinity of the spike protein"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.991844v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Spike protein receptor binding domain (RBD) of SARS-CoV-2 is the critical determinant of viral tropism and infectivity. Genomes of global SARS-CoV-2 strains were analyzed, yielding 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types circulating in Wuhan, Shenzhen, Hong Kong, and France, displayed enhanced structural stability along with higher human ACE2 receptor affinity of the spike protein, indicating these mutants may have acquired increased infectivity to humans. ### Competing Interest Statement"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-09"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Emergence of RBD mutations from circulating SARS-CoV-2 strains with enhanced structural stability and higher human ACE2 receptor affinity of the spike protein | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1237305 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Spike protein receptor binding domain (RBD) of SARS-CoV-2 is the critical determinant of viral tropism and infectivity. Genomes of global SARS-CoV-2 strains were analyzed, yielding 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types circulating in Wuhan, Shenzhen, Hong Kong, and France, displayed enhanced structural stability along with higher human ACE2 receptor affinity of the spike protein, indicating these mutants may have acquired increased infectivity to humans.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-6">A novel coronavirus SARS-CoV-2 has caused the outbreaks of Coronavirus Disease 2019 (COVID-19) globally since the first report in mid-December 2019 in Wuhan, China
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. As of April 5, 2020, SARS-CoV-2 has infected 1,051,635 people world-wide and caused 56,985 deaths
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup> with the estimated fatality rate of 5.41%. This on-going pandemic of COVID-19 has become the most serious threat to public health in recent times.
    </p>
    <p id="p-7">Angiotensin-converting enzyme 2 (ACE2) is the cellular receptor of SARS-CoV-2
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>, which is the same as for SARS-CoV. The receptor binding domain (RBD) of the subunit S1 directly interacts with ACE2, which provides for tight binding to the peptidase domain of ACE2. Therefore, RBD is the critical determinant of virus-receptor interaction and reflects viral host range, tropism and infectivity
     <sup>
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. Although the RBD sequences of different SARS-CoV-2 strains circulating globally are conserved, mutations have appeared, which might account for differences in viral infectivity and contribute to its spread.
    </p>
    <p id="p-8">To investigate whether these mutations in RBD have altered the receptor binding affinities and whether these strains may have been selected for higher infectivity, the binding dynamics between the SARS-CoV-2 RBDs of the mutated strains to date and human ACE2 receptor were modelled and assessed.</p>
    <div id="sec-2" class="subsection">
     <h3>SARS-CoV-2 RBD mutation mapping</h3>
     <p id="p-9">Among the 1609 SARS-CoV-2 strains with whole genome sequences available in the public databases, 32 strains contained amino acid mutations in the RBD (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Table 1</a>). These strains were reported from multiple locations, including China, U.K., Finland, France, Belgium, U.S.A., and India (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Most mutants deviate from the original reported genome (SARS-CoV-2 Wuhan-Hu-1) by only one amino acid (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Figure 1</a>). These 32 mutations parse into 10 types. Mutation V367F was found in six individual strains isolated from four patients: Three in France and one in Hong Kong. This suggested that these strains may have originated as a sub-lineage. Similarly, novel sub-lineages may account for the 13 V483A mutants and 7 G476S mutants isolated in the U.S.A. (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243202/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1243202/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplementary Table 1:</span>
       <span class="caption-title">Meta data of the strains with mutations in the RBD of spike glycoprotein</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-8.medium.gif" width="440" height="328"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-8.medium.gif" width="440" height="328"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-9.medium.gif" width="440" height="387"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-9.medium.gif" width="440" height="387"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-10.medium.gif" width="440" height="388"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-10.medium.gif" width="440" height="388"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-11.medium.gif" width="440" height="372"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-11.medium.gif" width="440" height="372"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-12.medium.gif" width="440" height="175"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-12.medium.gif" width="440" height="175"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-13.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-13.medium.gif" width="440" height="387"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-13.medium.gif" width="440" height="387"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-14.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-14.medium.gif" width="440" height="160"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-14.medium.gif" width="440" height="160"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F5/graphic-14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Supplementary Figure 1:</span>
       <span class="caption-title">Multiple alignments of the RBD amino acid sequences. SARS-CoV-2 Wuhan-Hu-1, the first reported genome, is used as reference. A bat and a pangolin SARS-like coronavirus are also included. Amino acid substitutions are marked. Dots indicate identical amino acids.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Distribution of the SARS-CoV-2 strains mutated in the RBD of the S protein. The geographic distribution of the RBD mutant strains is displayed. The strains with names highlighted in red are mutants with the enhanced binding affinity. The strains with names noted in yellow are mutants with similar binding affinities." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Distribution of the SARS-CoV-2 strains mutated in the RBD of the S protein.&lt;;/span&gt;; The geographic distribution of the RBD mutant strains is displayed. The strains with names highlighted in red are mutants with the enhanced binding affinity. The strains with names noted in yellow are mutants with similar binding affinities.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F1.medium.gif" width="440" height="301"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F1.medium.gif" width="440" height="301"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1:</span>
       <span class="caption-title">Distribution of the SARS-CoV-2 strains mutated in the RBD of the S protein.</span>
       <p id="p-10" class="first-child">The geographic distribution of the RBD mutant strains is displayed. The strains with names highlighted in red are mutants with the enhanced binding affinity. The strains with names noted in yellow are mutants with similar binding affinities.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Nucleotide diversity indicates strong positive selective pressure on RBD</h3>
     <p id="p-11">Since RBD is the only domain to bind human ACE2 and initiate cell entry, it is believed that the RBD should be highly conserved. However, polymorphism and divergence analysis by DnaSP6 (version 6.12.03) 
      <sup>
       <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
      </sup> showed that the RBD sequences were as diverse as the other regions of the S protein (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). The peak signals for diversity distribute across the entire S protein, and the multiple peaks in the RBD also reached the Pi value of ~0.0008, similar to Pi values in the other regions. Therefore, we hypothesize that the RBD would be selected to maintain or even improve its binding affinity to human ACE2.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Polymorphism and divergence graph of SARS-CoV-2 S gene. Polymorphism and divergence were analyzed by DnaSP6 (version 6.12.03). Analyses were conducted using the Nei-Gojobori model. All positions containing gaps and missing data were eliminated. Structural domains are annotated. The Pi values are calculated with window size: 20 nt, step size: 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Polymorphism and divergence graph of SARS-CoV-2 S gene.&lt;;/span&gt;; Polymorphism and divergence were analyzed by DnaSP6 (version 6.12.03). Analyses were conducted using the Nei-Gojobori model. All positions containing gaps and missing data were eliminated. Structural domains are annotated. The Pi values are calculated with window size: 20 nt, step size: 5.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F2.medium.gif" width="440" height="269"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F2.medium.gif" width="440" height="269"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2:</span>
       <span class="caption-title">Polymorphism and divergence graph of SARS-CoV-2 S gene.</span>
       <p id="p-12" class="first-child">Polymorphism and divergence were analyzed by DnaSP6 (version 6.12.03). Analyses were conducted using the Nei-Gojobori model. All positions containing gaps and missing data were eliminated. Structural domains are annotated. The Pi values are calculated with window size: 20 nt, step size: 5.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">To test this hypothesis, we investigated the selective pressures of the S gene by calculating nonsynonymous/synonymous substitution rate ratios (dN/dS ratios) for various segments of the S gene in the 1609 SARS-CoV-2 strains. With respect to our hypothesis, the entire S gene exhibited a dN/dS of 4.6526, remarkably greater than 1, showing that the S gene is indeed under positive selective pressure (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Supplementary Table 2</a>). The RBD showed a similar dN/dS (4.3966) as the entire S protein, indicating that high selective pressure was also applied to this essential domain. Therefore, the functional relevance of these RBD mutations may be inferred.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243208/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1243208/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1243208" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplementary Table 2:</span>
       <span class="caption-title">Estimates of Average Codon-based Evolutionary Divergence over S gene Pairs.</span>
       <p id="p-25" class="first-child">The numbers of nonsynonymous and synonymous differences per sequence from averaging over all sequence pairs are shown. Analyses were conducted using the Nei-Gojobori model. The analysis involved 660 SARS-CoV-2 nucleotide sequences. All positions containing gaps and missing data were discarded.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Three mutation types bind human ACE2 receptor with higher affinity</h3>
     <p id="p-14">To estimate the functional changes suggested by the RBD mutations, we performed molecular dynamics simulation for the prototype SARS-CoV-2 (Wuhan-Hu-1 strain) and the RBD mutants in order to assess their binding energy to human ACE2, which was performed using GROMACS 2019. The complex structure of the SARS-CoV-2 S-protein RBD domain and human ACE2 was obtained from National Microbiology Data Center (ID: NMDCS0000001) (PDB ID: 6LZG) (
      <a href="https://www.rcsb.org/structure/6LZG">https://www.rcsb.org/structure/6LZG</a>). Mutated amino acids of the SARS-CoV-2 RBD mutants were directly replaced in the model, and the bat/pangolin CoV RBD domain was modeled using SWISS-MODEL
      <sup>
       <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
      </sup>. Each simulation was performed at 10ns and each model was simulated in triplicates. All trajectories reached a plateau of RMSD after 2~5ns (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>), indicating that their structures reached an equilibrium. All of the subsequent computations on thermodynamics were based on the 5~10ns trajectories. Three RBD mutation types (N354D and D364Y, V367F, W436R) exhibited significantly lowered ΔG; suggesting a significantly increased affinity to human ACE2; the other mutants showed a similar ΔG compared to the prototype (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). The ΔG of these three mutation types were around −58 kJ/mol, approximately 25% lower than the prototype strain (−46.5 kJ/mol, calculated from the experimentally measured K
      <sub>D</sub>) (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>). Compared to the K
      <sub>D</sub> = 14.7 nM of the prototype RBD
      <sup>
       <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
      </sup>, the equilibrium dissociation constant (KD) of the three mutants are calculated as 0.12 nM, 0.11 nM, and 0.13 nM, respectively (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3C</a>), which were two orders of magnitude lower than for the prototype strain, indicating a remarkably increased affinity to the human ACE2 receptor. In the only double amino acid mutant (N354D, D364Y), the N354D substitution decreased the affinity, while the D364Y provided an even higher affinity than the overall double mutations (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3B</a>). This indicated that the D364Y is the major contributor of the enhanced affinity.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Binding free energy calculated for the SARS-CoV-2 S-RBD to human ACE2. (A) RMSD of typical MD trajectories of the SARS-CoV-2 prototype and the mutant strains. (B) Comparison of the binding free energy ([[#x394]]G) of the RBDs and the human ACE2. Note, the [[#x394]]G is inversely proportional to the binding affinity. Data are presented as mean[[#xB1]]SD. P-values were calculated using single-tailed student t-test. The P-values are shown for those with P &lt;; 0.05. The [[#x394]]G calculated from experimental KD values of SARS and SARS-CoV-2 prototype are marked in dotted and dashed lines, respectively. (C) Comparison of the equilibrium dissociation constants (KD) as calculated with the [[#x394]]G. (D) Comparison of the binding affinity of prototype S protein and V367F mutant to human ACE2 by ligand-receptor binding ELISA assay." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Binding free energy calculated for the SARS-CoV-2 S-RBD to human ACE2.&lt;;/span&gt;; (A) RMSD of typical MD trajectories of the SARS-CoV-2 prototype and the mutant strains. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Comparison of the binding free energy ([[#x394]]G) of the RBDs and the human ACE2. Note, the [[#x394]]G is inversely proportional to the binding affinity. Data are presented as mean[[#xB1]]SD. &lt;;em&gt;;P&lt;;/em&gt;;-values were calculated using single-tailed student t-test. The &lt;;em&gt;;P&lt;;/em&gt;;-values are shown for those with &lt;;em&gt;;P&lt;;/em&gt;; &lt;; 0.05. The [[#x394]]G calculated from experimental K&lt;;sub&gt;;D&lt;;/sub&gt;; values of SARS and SARS-CoV-2 prototype are marked in dotted and dashed lines, respectively. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Comparison of the equilibrium dissociation constants (KD) as calculated with the [[#x394]]G. &lt;;strong&gt;;(D)&lt;;/strong&gt;; Comparison of the binding affinity of prototype S protein and V367F mutant to human ACE2 by ligand-receptor binding ELISA assay.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F3.medium.gif" width="283" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F3.medium.gif" width="283" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3:</span>
       <span class="caption-title">Binding free energy calculated for the SARS-CoV-2 S-RBD to human ACE2.</span>
       <p id="p-15" class="first-child">(A) RMSD of typical MD trajectories of the SARS-CoV-2 prototype and the mutant strains. 
        <strong>(B)</strong> Comparison of the binding free energy (ΔG) of the RBDs and the human ACE2. Note, the ΔG is inversely proportional to the binding affinity. Data are presented as mean±SD. 
        <em>P</em>-values were calculated using single-tailed student t-test. The 
        <em>P</em>-values are shown for those with 
        <em>P</em> &lt;; 0.05. The ΔG calculated from experimental K
        <sub>D</sub> values of SARS and SARS-CoV-2 prototype are marked in dotted and dashed lines, respectively. 
        <strong>(C)</strong> Comparison of the equilibrium dissociation constants (KD) as calculated with the ΔG. 
        <strong>(D)</strong> Comparison of the binding affinity of prototype S protein and V367F mutant to human ACE2 by ligand-receptor binding ELISA assay.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">To validate the affinity change of the mutant S protein (V367F) experimentally, a receptor-ligand binding ELISA assay of the S proteins and the ACE2 was performed. 
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3D</a> showed that the V367F mutant significantly lowered the ED50 concentration (ED50 = 0.8±0.04 μg/ml), as compared to the prototype (ED50 = 1.7±0.14 μg/ml), demonstrating that the V367F mutant has higher affinity than the prototype. This result qualitatively validated our computational simulation.
     </p>
     <p id="p-17">In comparison, the bat CoV RBD (strain RaTG13, with the highest genome similarity) showed a much lower binding affinity (K
      <sub>D</sub>=1.17mM; ΔG=-17.4kJ/mol) to human ACE2 than the pangolin CoV (K
      <sub>D</sub>=1.89μM; ΔG=-33.9kJ/mol). For comparison, the affinity of the pangolin CoV was slightly lower than the SARS-CoV-2 prototype strain (K
      <sub>D</sub>=14.7nM; ΔG=-46.5kJ/mol) (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3B</a>, 
      <a id="xref-fig-3-8" class="xref-fig" href="#F3">3C</a>).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Structural basis for the increased affinity</h3>
     <p id="p-18">The binding surface of the RBD to ACE2 is largely in random coil conformation, which lacks structural rigidity. Therefore, a firm scaffold should be necessary to maintain this conformation of the interaction surface, and thus may facilitate the binding affinity. The beta-sheet structure scaffold, centered by residues 510-524 (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>, marked as red), provides this rigidity. “Higher affinity” mutants (N354D D364Y, V367F, and W436R) showed a considerable decrease of the RMSF (Root Mean Square of Fluctuation) at this region, demonstrating a more rigid structure; this was not observed for the “similar affinity” mutants (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). Coincidentally, the substitutions that account for the affinity increase (D364Y, V367F, and W436R) are all located near this fragment. Indeed, residues 475-485, which is a random coil near the binding site, showed a remarkably higher RMSF for the “similar affinity” group mutants, in contrast to the “higher affinity” group mutants (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>). Moreover, the “higher affinity” group exhibited a general decreased ΔG in the binding site region, but not the “similar affinity” group mutants (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4C</a>). One mutation, W436R, provides a positively charged Arg in the proximity of the complementing highly negative charged ACE2 surface. This potential electrostatic attraction may contribute to the higher affinity (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4D</a>).
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural analysis of RBD mutants and the effects on their binding affinity. (A) Spatial location of the mutated amino acids and the fragment 510-524. (B) RMSF of the nine mutants were compared to that of the prototype. Red arrows denote the fragment of residues 510-524. Black arrows denote the fragment of residues 475-485. (C) Contribution of each amino acid to the binding free energy. Red bars denote the binding site. (D) View of the interaction surface of ACE2, with charge density noted. The arginine of the W436R mutant is in the proximity of the negatively charged amino acids. The electrostatic surface charges of the ACE2 are calculated using Pymol, with the charge unit KbT/ec, as noted in the Pymol manual." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-840926143" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural analysis of RBD mutants and the effects on their binding affinity.&lt;;/span&gt;; &lt;;strong&gt;;(A)&lt;;/strong&gt;; Spatial location of the mutated amino acids and the fragment 510-524. &lt;;strong&gt;;(B)&lt;;/strong&gt;; RMSF of the nine mutants were compared to that of the prototype. Red arrows denote the fragment of residues 510-524. Black arrows denote the fragment of residues 475-485. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Contribution of each amino acid to the binding free energy. Red bars denote the binding site. &lt;;strong&gt;;(D)&lt;;/strong&gt;; View of the interaction surface of ACE2, with charge density noted. The arginine of the W436R mutant is in the proximity of the negatively charged amino acids. The electrostatic surface charges of the ACE2 are calculated using Pymol, with the charge unit &lt;;em&gt;;K&lt;;sub&gt;;b&lt;;/sub&gt;;T/e&lt;;sub&gt;;c&lt;;/sub&gt;;&lt;;/em&gt;;, as noted in the Pymol manual.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F4.medium.gif" width="300" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F4.medium.gif" width="300" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.15.991844/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 4:</span>
       <span class="caption-title">Structural analysis of RBD mutants and the effects on their binding affinity.</span>
       <p id="p-19" class="first-child">
        <strong>(A)</strong> Spatial location of the mutated amino acids and the fragment 510-524. 
        <strong>(B)</strong> RMSF of the nine mutants were compared to that of the prototype. Red arrows denote the fragment of residues 510-524. Black arrows denote the fragment of residues 475-485. 
        <strong>(C)</strong> Contribution of each amino acid to the binding free energy. Red bars denote the binding site. 
        <strong>(D)</strong> View of the interaction surface of ACE2, with charge density noted. The arginine of the W436R mutant is in the proximity of the negatively charged amino acids. The electrostatic surface charges of the ACE2 are calculated using Pymol, with the charge unit 
        <em>K
         <sub>b</sub>T/e
         <sub>c</sub>
        </em>, as noted in the Pymol manual.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Discussion</h2>
    <p id="p-20">Due to the lengthening pandemic and evolving nature of the SARS-CoV-2 virus globally, identifying changes in viral infectivity is crucial to containing the COVID-19 spread. Quarantine policies need to be adapted with respect to the changes in virus infectivity. This report provides computational insight into the functional outcome of mutations in RBD: RBD mutates under positive selection pressure, and several mutants acquired increased binding affinity to human ACE2 receptor, implying higher infectivity to humans (noted for one mutant with experimental validation).</p>
    <p id="p-21">It should be noted that the mutation V367F enhancing the affinity was found in six strains: One in Hong Kong and five in France. As RBD is conserved in SARS-CoV-2, the coincidence of six strains with the same mutation across the geographic distance indicates that this mutant may have evolved to be more robust and that these strains originated as a novel sub-lineage, given the close isolation dates (January 22 and 23, respectively). Combined with the epidemiological data, mutation surveillance is of critical importance as it can reveal more exact transmission routes of the epidemic and provide early warning for additional outbreaks. Emergence of SARS-CoV-2 strains in Hong Kong, France, and other countries with RBD mutations providing higher binding affinity to human ACE2 suggests a higher risk of more severe morbidity and mortality during a sustained pandemic of COVID-19 if no effective precautions are implemented.</p>
    <p id="p-22">In our study, the binding energy of the bat SARS-like CoV RBD suggests it is too high to bind human ACE2 effectively (KD in millimolar range). In contrast, the pangolin CoV showed a KD of binding to human ACE2 at the micromolar range, just ~6x higher than that of human SARS virus (K
     <sub>D</sub> = 0.326μM) (
     <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3</a>), indicating that the pangolin CoV has the potential to infect human in unprotected close contact. The pangolin CoV has been detected among the smuggled Malayan pangolins in multiple provinces in China
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup>, suggesting a risk of zoonotic infection from wild animals to human constantly and widely.
    </p>
    <p id="p-23">The S protein is also important for antigen recognition. In this survey of 1609 strains, 32 had amino acid mutations in the RBD. Since the RBD contains important antigenic epitopes, mutations in RBD, especially those which change the amino acid properties, may weaken the binding of the antibody raised against the prototype strain. This may lead to decreased vaccine efficacy.</p>
    <p id="p-24">In summary, we have identified 32 RBD mutant strains clustering into 10 mutation types under high positive selection pressure. Three mutation types emerged in Asia and Europe; these display enhanced structural stability of the spike protein along with higher binding affinities to human ACE2 receptor, and indicate these mutants may have acquired increased infectivity to humans.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-2">
      <p id="p-5">New version including more data, modified title, and more anlaysis. </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">Reference</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.1" data-doi="10.1056/nejmoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. 
         <span class="cit-pub-date">2020</span>:
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/nejmoa2001017</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252Fnejmoa2001017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/nejmoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.991844v3.2" data-doi="10.1056/nejmoa2001316">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. 
         <span class="cit-pub-date">2020</span>:
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/nejmoa2001316</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252Fnejmoa2001316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/nejmoa2001316&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.991844v3.3" data-doi="10.1001/jama.2020.1585">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical Characteristics of 138 Hospitalized Patients with 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">JAMA - J Am Med Assoc</abbr>. 
         <span class="cit-pub-date">2020</span>:
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.1585</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJAMA%2B-%2BJ%2BAm%2BMed%2BAssoc%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.1585%26rft_id%253Dinfo%253Apmid%252F32031570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1001/jama.2020.1585&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32031570&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.15.991844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.4" data-doi="10.1016/S0140-6736(20)30154-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JFW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kok</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">395</span>(
         <span class="cit-issue">10223</span>):
         <span class="cit-fpage">514</span>–
         <span class="cit-lpage">523</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30154-9</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930154-9%26rft_id%253Dinfo%253Apmid%252F31986261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30154-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31986261&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.15.991844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mariana</span>
          <span class="cit-name-given-names">N.</span>
         </span>
         <abbr class="cit-jnl-abbrev">Coronavirus disease 2019 (COVID-19)</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">2019</span>(
         <span class="cit-month">April</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.6" data-doi="10.1016/j.cell.2020.02.052">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffmann</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kleine-Weber</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schroeder</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>. 
         <span class="cit-month">March</span>
         <span class="cit-pub-date">2020</span>:
         <span class="cit-vol">S0092-8674</span>(
         <span class="cit-issue">20</span>)
         <span class="cit-fpage">30229</span>–
         <span class="cit-lpage">4</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.052</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.052%26rft_id%253Dinfo%253Apmid%252F32142651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.052&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32142651&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.15.991844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.7" data-doi="10.1016/j.bbrc.2020.02.071">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">ZJ.</span>
         </span>
         <span class="cit-article-title">Structure analysis of the receptor binding of 2019-nCoV</span>. 
         <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">2</span>(
         <span class="cit-issue">xxxx</span>):
         <span class="cit-fpage">0</span>–
         <span class="cit-lpage">5</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbrc.2020.02.071</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2020.02.071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2020.02.071&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.8" data-doi="10.1126/science.abb2507">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corbett</span>
          <span class="cit-name-given-names">KS</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science (80-)</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">367</span>(
         <span class="cit-issue">6483</span>):
         <span class="cit-fpage">1260 LP</span> – 
         <span class="cit-lpage">1263</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4xNS45OTE4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.991844v3.9" data-doi="10.1128/jvi.00127-20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>
          <span class="cit-name-given-names">RS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>. 
         <span class="cit-pub-date">2020</span>;(
         <span class="cit-month">January</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00127-20</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.00127-20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/jvi.00127-20&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.10" data-doi="10.1093/molbev/msx248">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rozas</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ferrer-Mata</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sanchez-DelBarrio</span>
          <span class="cit-name-given-names">JC</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">DnaSP 6: DNA sequence polymorphism analysis of large data sets</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">34</span>(
         <span class="cit-issue">12</span>):
         <span class="cit-fpage">3299</span>–
         <span class="cit-lpage">3302</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msx248</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D3299%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx248%26rft_id%253Dinfo%253Apmid%252F29029172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msx248&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29029172&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.15.991844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.11" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertoni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bienert</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: Homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>(
         <span class="cit-issue">W1</span>):
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">W303</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gky427</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.15.991844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.12" data-doi="10.1038/s41586-020-2169-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lam</span>
          <span class="cit-name-given-names">TT-Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shum</span>
          <span class="cit-name-given-names">MH-H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">H-C</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Identifying SARS-CoV-2 related coronaviruses in Malayan pangolins</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2020</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2169-0</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2169-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2169-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.991844v3.13" data-doi="10.1101/2020.02.17.951335">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhai</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-month">January</span>
         <span class="cit-pub-date">2020</span>:2020.02.17.951335.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.17.951335</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.17.951335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4xNy45NTEzMzV2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMTUuOTkxODQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
