<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/embed/graphic-9.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/09/22/157156.atom" />
<meta name="HW.pisa" content="biorxiv;157156v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies" />
<meta name="DC.Identifier" content="10.1101/157156" />
<meta name="DC.Date" content="2017-09-22" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. [www.syndromictrends.com][1] presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.

 [1]: http://www.syndromictrends.com" />
<meta name="DC.Contributor" content="Lindsay Meyers" />
<meta name="DC.Contributor" content="Christine C. Ginocchio" />
<meta name="DC.Contributor" content="Aimie N. Faucett" />
<meta name="DC.Contributor" content="Frederick S. Nolte" />
<meta name="DC.Contributor" content="Per H. Gesteland" />
<meta name="DC.Contributor" content="Amy Leber" />
<meta name="DC.Contributor" content="Diane Janowiak" />
<meta name="DC.Contributor" content="Virginia Donovan" />
<meta name="DC.Contributor" content="Jennifer Dien Bard" />
<meta name="DC.Contributor" content="Silvia Spitzer" />
<meta name="DC.Contributor" content="Kathleen A. Stellrecht" />
<meta name="DC.Contributor" content="Hossein Salimnia" />
<meta name="DC.Contributor" content="Rangaraj Selvarangan" />
<meta name="DC.Contributor" content="Stefan Juretschko" />
<meta name="DC.Contributor" content="Judy A. Daly" />
<meta name="DC.Contributor" content="Jeremy C. Wallentine" />
<meta name="DC.Contributor" content="Kristy Lindsey" />
<meta name="DC.Contributor" content="Franklin Moore" />
<meta name="DC.Contributor" content="Sharon L. Reed" />
<meta name="DC.Contributor" content="Maria Aguero-Rosenfeld" />
<meta name="DC.Contributor" content="Paul D. Fey" />
<meta name="DC.Contributor" content="Gregory A. Storch" />
<meta name="DC.Contributor" content="Steve J. Melnick" />
<meta name="DC.Contributor" content="Christine C. Robinson" />
<meta name="DC.Contributor" content="Jennifer F. Meredith" />
<meta name="DC.Contributor" content="Camille V. Cook" />
<meta name="DC.Contributor" content="Robert K. Nelson" />
<meta name="DC.Contributor" content="Jay D. Jones" />
<meta name="DC.Contributor" content="Samuel V. Scarpino" />
<meta name="DC.Contributor" content="Benjamin M. Althouse" />
<meta name="DC.Contributor" content="Kirk M. Ririe" />
<meta name="DC.Contributor" content="Bradley A. Malin" />
<meta name="DC.Contributor" content="Mark A. Poritz" />
<meta name="article:published_time" content="2017-09-22" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray&lt;sup&gt;®&lt;/sup&gt; Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;157156v2" />
<meta name="citation_id" content="157156v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/157156v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/157156v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/157156v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156.full.pdf" />
<meta name="citation_doi" content="10.1101/157156" />
<meta name="citation_num_pages" content="43" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="157156" />
<meta name="citation_author" content="Lindsay Meyers" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author_email" content="lindsay.meyers@biofiredx.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2994-274X" />
<meta name="citation_author" content="Christine C. Ginocchio" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author_institution" content="bioMérieux USA" />
<meta name="citation_author_institution" content="Hofstra Northwell School of Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8200-0324" />
<meta name="citation_author" content="Aimie N. Faucett" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4590-4276" />
<meta name="citation_author" content="Frederick S. Nolte" />
<meta name="citation_author_institution" content="Medical University of South Carolina" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4943-9220" />
<meta name="citation_author" content="Per H. Gesteland" />
<meta name="citation_author_institution" content="University of Utah" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9869-8122" />
<meta name="citation_author" content="Amy Leber" />
<meta name="citation_author_institution" content="Nationwide Children’s Hospital" />
<meta name="citation_author" content="Diane Janowiak" />
<meta name="citation_author_institution" content="South Bend Medical Foundation" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9208-0647" />
<meta name="citation_author" content="Virginia Donovan" />
<meta name="citation_author_institution" content="Winthrop University Hospital" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9296-6931" />
<meta name="citation_author" content="Jennifer Dien Bard" />
<meta name="citation_author_institution" content="Children’s Hospital of Los Angeles" />
<meta name="citation_author_institution" content="Keck School of Medicine, University of Southern California" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0524-9473" />
<meta name="citation_author" content="Silvia Spitzer" />
<meta name="citation_author_institution" content="Stony Brook University Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2642-4771" />
<meta name="citation_author" content="Kathleen A. Stellrecht" />
<meta name="citation_author_institution" content="Albany Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0160-6034" />
<meta name="citation_author" content="Hossein Salimnia" />
<meta name="citation_author_institution" content="Detroit Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8069-5185" />
<meta name="citation_author" content="Rangaraj Selvarangan" />
<meta name="citation_author_institution" content="Children’s Mercy Hospital" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3275-6657" />
<meta name="citation_author" content="Stefan Juretschko" />
<meta name="citation_author_institution" content="Northwell Health, Lake Success" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4658-6276" />
<meta name="citation_author" content="Judy A. Daly" />
<meta name="citation_author_institution" content="Primary Children’s Hospital" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4801-7023" />
<meta name="citation_author" content="Jeremy C. Wallentine" />
<meta name="citation_author_institution" content="Intermountain Medical Center" />
<meta name="citation_author" content="Kristy Lindsey" />
<meta name="citation_author_institution" content="University of Massachusetts Medical School-Baystate" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8172-5964" />
<meta name="citation_author" content="Franklin Moore" />
<meta name="citation_author_institution" content="University of Massachusetts Medical School-Baystate" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3019-0437" />
<meta name="citation_author" content="Sharon L. Reed" />
<meta name="citation_author_institution" content="UC San Diego Health" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7615-9366" />
<meta name="citation_author" content="Maria Aguero-Rosenfeld" />
<meta name="citation_author_institution" content="New York University Langone Health" />
<meta name="citation_author" content="Paul D. Fey" />
<meta name="citation_author_institution" content="University of Nebraska Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0939-6884" />
<meta name="citation_author" content="Gregory A. Storch" />
<meta name="citation_author_institution" content="Washington University Medical Center" />
<meta name="citation_author" content="Steve J. Melnick" />
<meta name="citation_author_institution" content="Nicklaus Children's Hospital" />
<meta name="citation_author" content="Christine C. Robinson" />
<meta name="citation_author_institution" content="Children's Hospital Colorado" />
<meta name="citation_author" content="Jennifer F. Meredith" />
<meta name="citation_author_institution" content="Greenville Health System" />
<meta name="citation_author" content="Camille V. Cook" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6071-4898" />
<meta name="citation_author" content="Robert K. Nelson" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author" content="Jay D. Jones" />
<meta name="citation_author_institution" content="BioFire Diagnostics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9091-5316" />
<meta name="citation_author" content="Samuel V. Scarpino" />
<meta name="citation_author_institution" content="University of Vermont" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5716-2770" />
<meta name="citation_author" content="Benjamin M. Althouse" />
<meta name="citation_author_institution" content="University of Washington" />
<meta name="citation_author_institution" content="New Mexico State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5464-654X" />
<meta name="citation_author" content="Kirk M. Ririe" />
<meta name="citation_author_institution" content="bioMérieux USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0620-7419" />
<meta name="citation_author" content="Bradley A. Malin" />
<meta name="citation_author_institution" content="Vanderbilt University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3040-5175" />
<meta name="citation_author" content="Mark A. Poritz" />
<meta name="citation_author_institution" content="BioFire Defense" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5060-0560" />
<meta name="citation_reference" content="Marinovic, A.B., et al., Quantifying Reporting Timeliness to Improve Outbreak Con}trol. Emerg Infect Dis, 2015. 21(2)" />
<meta name="citation_reference" content="Yang, W., A. Karspeck, and J. Shaman, Comparison of filtering methods for the modeling and retrospective forecasting of influenza epidemics. PLoS Comput Biol, 2014. 10(4)" />
<meta name="citation_reference" content="van den Wijngaard, C.C., et al., Evaluation of syndromic surveillance in the Netherlands: its added value and recommendations for implementation. Euro Surveill, 2011. 16(9)" />
<meta name="citation_reference" content="CDC. Flu Activity &amp; Surveillance. [cited April 30, 2017]; Available from: https://www.cdc.gov/flu/weekly/fluactivitysurv.htm." />
<meta name="citation_reference" content="CDC. Overview of Influenza Surveillance in the United States. [cited April 30, 2017]; Available from: http://www.cdc.gov/flu/pdf/weekly/overview.pdf." />
<meta name="citation_reference" content="CDC. National Respiratory and Enteric Virus Surveillance System (NREVSS). [cited April 30, 2017]; Available from: http://www.cdc.gov/surveillance/nrevss/." />
<meta name="citation_reference" content="National Electronic Disease Surveillance System Working, G., National Electronic Disease Surveillance System (NEDSS): a standards-based approach to connect public health and clinical medicine. J Public Health Manag Pract, 2001. 7(6)" />
<meta name="citation_reference" content="Lamb, E., et al., Update on progress in electronic reporting of laboratory results to public health agencies United States, 2014. MMWR Morb Mortal Wkly Rep, 2015. 64(12)" />
<meta name="citation_reference" content="United States Department of Defense. Global Emerging Infections Surveillance and Response System. [cited April 30, 2017]; Available from: https://www.health.mil/MilitaryHealthTopics/HealthReadiness/ArmedForcesHealthSurveillanceBranch/GlobalEmergingInfectionsSurveillanceandResponse." />
<meta name="citation_reference" content="Nelson, N.P., et al., Event-based internet biosurveillance: relation to epidemiological observation. Emerg Themes Epidemiol, 2012. 9(1)" />
<meta name="citation_reference" content="Cheng, C.K., et al., A profile of the online dissemination of national influenza surveillance data. BMC Public Health, 2009. 9" />
<meta name="citation_reference" content="Hulth, A. and G. Rydevik, GET WELL: an automated surveillance system for gaining new epidemiological knowledge. BMC Public Health, 2011. 11" />
<meta name="citation_reference" content="CDC. BioSense. [cited September 20, 2017]; Available from: https://www.cdc.gov/nssp/biosense/index.html." />
<meta name="citation_reference" content="Ginsberg, J., et al., Detecting influenza epidemics using search engine query data. Nature, 2009. 457(7232)" />
<meta name="citation_reference" content="GoogleFlu. 2015, The Next Chapter for Flu Trends. [cited April 30, 2017]; Available from: http://googleresearch.blogspot.com/2015/08/thenextchapterforflutrends.html." />
<meta name="citation_reference" content="FluNearYou. Flu Near You | HealthMap. [cited April 30, 2017]; Available from: https://flunearyou.org/." />
<meta name="citation_reference" content="Das, D., et al., Monitoring over-the-counter medication sales for early detection of disease outbreaks New York City. MMWR Suppl, 2005. 54" />
<meta name="citation_reference" content="Patwardhan, A. and R. Bilkovski, Comparison: Flu prescription sales data from a retail pharmacy in the US with Google Flu trends and US ILINet (CDC) data as flu activity indicator. PLoS One, 2012. 7(8)" />
<meta name="citation_reference" content="Broniatowski, D.A., M.J. Paul, and M. Dredze, National and local influenza surveillance through Twitter: an analysis of the 2012-2013 influenza epidemic. PLoS One, 2013. 8(12)" />
<meta name="citation_reference" content="Signorini, A., A.M. Segre, and P.M. Polgreen, The use of Twitter to track levels of disease activity and public concern in the U.S. during the influenza A H1N1 pandemic. PLoS One, 2011. 6(5)" />
<meta name="citation_reference" content="Generous, N., et al., Global disease monitoring and forecasting with Wikipedia. PLoS Comput Biol, 2014. 10(11)" />
<meta name="citation_reference" content="McIver, D.J. and J.S. Brownstein, Wikipedia usage estimates prevalence of influenza-like illness in the United States in near real-time. PLoS Comput Biol, 2014. 10(4)" />
<meta name="citation_reference" content="IHC. GermWatch Intermountain Healthcare Salt Lake City, Utah. [cited April 30, 2017]; Available from: http://intermountainhealthcare.org/healthresources/germwatch/Pages/home.aspx." />
<meta name="citation_reference" content="Gesteland, P.H., et al., Informing the front line about common respiratory viral epidemics. AMIA Annu Symp Proc, 2007" />
<meta name="citation_reference" content="NYSDOH. Electronic Clinical Laboratory Reporting System. [cited April 30, 2017]; Available from: http://www.health.ny.gov/professionals/reportable_diseases/eclrs/." />
<meta name="citation_reference" content="Shah, S. and C. Messina, The time-associated impact of the Newborn Influenza Protection Act on infant influenza rates in New York State. J Perinat Med, 2014. 42(6)" />
<meta name="citation_reference" content="Crawford, J.M., et al., Laboratory surge response to pandemic (H1N1) 2009 outbreak, New York City metropolitan area, USA. Emerg Infect Dis, 2010. 16(1)" />
<meta name="citation_reference" content="Ginocchio, C.C., et al., Evaluation of multiple test methods for the detection of the novel 2009 influenza A (H1N1) during the New York City outbreak. J Clin Virol, 2009. 45(3)" />
<meta name="citation_reference" content="Al Tawfiq, J.A., et al., Surveillance for emerging respiratory viruses. Lancet Infect Dis, 2014. 14(10)" />
<meta name="citation_reference" content="Gautret, P., et al., Emerging viral respiratory tract infections-environmental risk factors and transmission. Lancet Infect Dis, 2014. 14(11)" />
<meta name="citation_reference" content="Hazelton, B., et al., Detection of influenza A and B with the Alere i Influenza A &amp; B: a novel isothermal nucleic acid amplification assay. Influenza Other Respir Viruses, 2015. 9(3)" />
<meta name="citation_reference" content="Salez, N., et al., Prospective and retrospective evaluation of the Cepheid Xpert(R) Flu/RSV XC assay for rapid detection of influenza A, influenza B, and respiratory syncytial virus. Diagn Microbiol Infect Dis, 2015. 81(4)" />
<meta name="citation_reference" content="Nolte, F.S., L. Gauld, and S.B. Barrett, Direct Comparison of Alere i and cobas Liat Influenza A and B Tests for Rapid Detection of Influenza Virus Infection. J Clin Microbiol, 2016. 54(11)" />
<meta name="citation_reference" content="Buchan, B.W. and N.A. Ledeboer, Emerging technologies for the clinical microbiology laboratory. Clin Microbiol Rev, 2014. 27(4)" />
<meta name="citation_reference" content="Zhang, H., S. Morrison, and Y.W. Tang, Multiplex polymerase chain reaction tests for detection of pathogens associated with gastroenteritis. Clin Lab Med, 2015. 35(2)" />
<meta name="citation_reference" content="Salez, N., et al., Evaluation of Four Commercial Multiplex Molecular Tests for the Diagnosis of Acute Respiratory Infections. PLoS One, 2015. 10(6)" />
<meta name="citation_reference" content="Poritz, M.A., et al., FilmArray, an automated nested multiplex PCR system for multipathogen detection: development and application to respiratory tract infection. PLoS One, 2011. 6(10)" />
<meta name="citation_reference" content="Pierce, V.M. and R.L. Hodinka, Comparison of the GenMark Diagnostics eSensor respiratory viral panel to real-time PCR for detection of respiratory viruses in children. J Clin Microbiol, 2012. 50(11)" />
<meta name="citation_reference" content="GenMarkDx. 2016, The ePlex(®) Respiratory Pathogen Panel (RP)*. [cited April 30, 2017]; Available from: https://genmarkdx.com/wpcontent/uploads/2016/09/GNMKIMC1046EePlexRPDataSheetUS.pdf." />
<meta name="citation_reference" content="Gadsby, N.J., et al., Comparison of the Luminex Respiratory Virus Panel fast assay with in house real-time PCR for respiratory viral infection diagnosis. J Clin Microbiol, 2010. 48(6)" />
<meta name="citation_reference" content="Lee, C.K., et al., Comparison of Luminex NxTAG Respiratory Pathogen Panel and xTAG Respiratory Viral Panel FAST Version 2 for the Detection of Respiratory Viruses. Ann Lab Med, 2017. 37(3)" />
<meta name="citation_reference" content="Butt, S.A., et al., Comparison of three commercial RT PCR systems for the detection of respiratory viruses. J Clin Virol, 2014. 61(3)" />
<meta name="citation_reference" content="van Panhuis, W.G., et al., A systematic review of barriers to data sharing in public health. BMC Public Health, 2014. 14" />
<meta name="citation_reference" content="Williams, P.A. and A.J. Woodward, Cybersecurity vulnerabilities in medical devices: a complex environment and multifaceted problem. Med Devices (Auckl), 2015. 8" />
<meta name="citation_reference" content="Benitez, K. and B. Malin, Evaluating re-identification risks with respect to the HIPAA privacy rule. J Am Med Inform Assoc, 2010. 17(2)" />
<meta name="citation_reference" content="El Emam, K., et al., A systematic review of re-identification attacks on health data. PLoS One, 2011. 6(12)" />
<meta name="citation_reference" content="El Emam, K., S. Rodgers, and B. Malin, Anonymising and sharing individual patient data. BMJ, 2015. 350" />
<meta name="citation_reference" content="Sarpatwari, A., et al., Ensuring patient privacy in data sharing for postapproval research. N Engl J Med, 2014. 371(17)" />
<meta name="citation_reference" content="BioFire Diagnostics. FilmArray(®) Panels. [cited April 30, 2017]; Available from: http://www.biofiredx.com/products/thefilmarraypanels/." />
<meta name="citation_reference" content="Hayden, R.T., et al., Comparison of two broadly multiplexed PCR systems for viral detection in clinical respiratory tract specimens from immunocompromised children. J Clin Virol, 2012. 53(4)" />
<meta name="citation_reference" content="Loeffelholz, M.J., et al., Comparison of the FilmArray Respiratory Panel and Prodesse real-time PCR assays for detection of respiratory pathogens. J Clin Microbiol, 2011. 49(12)" />
<meta name="citation_reference" content="Pierce, V.M., et al., Comparison of the Idaho Technology FilmArray system to real-time PCR for detection of respiratory pathogens in children. J Clin Microbiol, 2012. 50(2)" />
<meta name="citation_reference" content="Rand, K.H., H. Rampersaud, and H.J. Houck, Comparison of two multiplex methods for detection of respiratory viruses: FilmArray RP and xTAG RVP. J Clin Microbiol, 2011. 49(7)" />
<meta name="citation_reference" content="Xu, M., et al., Implementation of filmarray respiratory viral panel in a core laboratory improves testing turnaround time and patient care. Am J Clin Pathol, 2013. 139(1)" />
<meta name="citation_reference" content="Couturier, M.R., et al., Evaluation of the FilmArray(R) Respiratory Panel for clinical use in a large children's hospital. J Clin Lab Anal, 2013. 27(2)" />
<meta name="citation_reference" content="Popowitch, E.B., S.S. O'Neill, and M.B. Miller, Comparison of the Biofire FilmArray RP, Genmark eSensor RVP, Luminex xTAG RVPv1, and Luminex xTAG RVP fast multiplex assays for detection of respiratory viruses. J Clin Microbiol, 2013. 51(5)" />
<meta name="citation_reference" content="Bansal, S., et al., Big Data for Infectious Disease Surveillance and Modeling. Journal of Infectious Diseases, 2016. 214(suppl 4)" />
<meta name="citation_reference" content="Magurran, A.E., Measuring biological diversity. 2004, Malden, Ma: Blackwell Pub. viii." />
<meta name="citation_reference" content="U.S. Dept. of Health and Human Services, Standards for privacy of individually identifiable health information, Final Rule. 2002." />
<meta name="citation_reference" content="Malin, B., D. Karp, and R.H. Scheuermann, Technical and policy approaches to balancing patient privacy and data sharing in clinical and translational research. J Investig Med, 2010. 58(1)" />
<meta name="citation_reference" content="Office for Civil Rights, Guidance regarding methods for de-identification of protected health information in accordance with the Health Insurance Portability and Accountability Act (HIPAA) Privacy Rule, U.S. Department of Health and Human Services., Editor. 2012, U.S. Department of Health and Human Services." />
<meta name="citation_reference" content="IBM. Infographic: The Four V's of Big Data. [cited June 8, 2017]; Available from: http://www.ibmbigdatahub.com/infographic/fourvsbigdata." />
<meta name="citation_reference" content="Stockmann, C., et al., Seasonality of acute otitis media and the role of respiratory viral activity in children. Pediatr Infect Dis J, 2013. 32(4)" />
<meta name="citation_reference" content="Radin, J.M., et al., Epidemiology of pathogen-specific respiratory infections among three US populations. PLoS One, 2014. 9(12)" />
<meta name="citation_reference" content="Brittain Long, R., et al., Seasonal variations of 15 respiratory agents illustrated by the application of a multiplex polymerase chain reaction assay. Scand J Infect Dis, 2012. 44(1)" />
<meta name="citation_reference" content="Olofsson, S., et al., PCR for detection of respiratory viruses: seasonal variations of virus infections. Expert Rev Anti Infect Ther, 2011. 9(8)" />
<meta name="citation_reference" content="Reed, C., et al., Estimating influenza disease burden from population-based surveillance data in the United States. PLoS One, 2015. 10(3)" />
<meta name="citation_reference" content="Ruuskanen, O., et al., Viral pneumonia. Lancet, 2011. 377(9773)" />
<meta name="citation_reference" content="Pavia, A.T., Viral infections of the lower respiratory tract: old viruses, new viruses, and the role of diagnosis. Clin Infect Dis, 2011. 52 Suppl 4" />
<meta name="citation_reference" content="Brendish, N.J., et al., Routine molecular point-of-care testing for respiratory viruses in adults presenting to hospital with acute respiratory illness (ResPOC): a pragmatic, open-label, randomised controlled trial. Lancet Respir Med, 2017. 5(5)" />
<meta name="citation_reference" content="Goka, E.A., et al., Single, dual and multiple respiratory virus infections and risk of hospitalization and mortality. Epidemiol Infect, 2015. 143(1)" />
<meta name="citation_reference" content="Martin, E.T., et al., Epidemiology of multiple respiratory viruses in childcare attendees. J Infect Dis, 2013. 207(6)" />
<meta name="citation_reference" content="Rhedin, S., et al., Clinical utility of PCR for common viruses in acute respiratory illness. Pediatrics, 2014. 133(3)" />
<meta name="citation_reference" content="Pinky, L. and H.M. Dobrovolny, Coinfections of the Respiratory Tract: Viral Competition for Resources. PLoS One, 2016. 11(5)" />
<meta name="citation_reference" content="Thorburn, K., et al., RSV outbreak in a paediatric intensive care unit. J Hosp Infect, 2004. 57(3)" />
<meta name="citation_reference" content="Gooskens, J., et al., Morbidity and mortality associated with nosocomial transmission of oseltamivir-resistant influenza A(H1N1) virus. JAMA, 2009. 301(10)" />
<meta name="citation_reference" content="Te Wierik, M.J., et al., An outbreak of severe respiratory tract infection caused by human metapneumovirus in a residential care facility for elderly in Utrecht, the Netherlands, January to March 2010. Euro Surveill, 2012. 17(13)" />
<meta name="citation_reference" content="Casalegno, J.S., et al., Rhinoviruses delayed the circulation of the pandemic influenza A (H1N1) 2009 virus in France. Clin Microbiol Infect, 2010. 16(4)" />
<meta name="citation_reference" content="Karppinen, S., et al., Interference between respiratory syncytial virus and rhinovirus in respiratory tract infections in children. Clin Microbiol Infect, 2016. 22(2)" />
<meta name="citation_reference" content="Hurlbert, S.H., The Nonconcept of Species Diversity: A Critique and Alternative Parameters. Ecology, 1971. 52(4)" />
<meta name="citation_reference" content="Jost, L., Entropy and diversity. Oikos, 2006. 113(2)" />
<meta name="citation_reference" content="Krebs, C.J., Ecological methodology. 2nd ed. 1999, Menlo Park, Calif.: Benjamin/Cummings. xii." />
<meta name="citation_reference" content="Essa, S., et al., Mixed viral infections circulating in hospitalized patients with respiratory tract infections in kuwait. Adv Virol, 2015. 2015" />
<meta name="citation_reference" content="Bhattacharyya, S., et al., Cross-immunity between strains explains the dynamical pattern of paramyxoviruses. Proc Natl Acad Sci U S A, 2015. 112(43)" />
<meta name="citation_reference" content="Wen, X., et al., Structural basis for antibody cross-neutralization of respiratory syncytial virus and human metapneumovirus. Nat Microbiol, 2017. 2" />
<meta name="citation_reference" content="Achten, N.B., et al., Interference Between Respiratory Syncytial Virus and Human Rhinovirus Infection in Infancy. J Infect Dis, 2017. 215(7)" />
<meta name="citation_reference" content="Nsoesie, E.O., et al., A systematic review of studies on forecasting the dynamics of influenza outbreaks. Influenza Other Respir Viruses, 2014. 8(3)" />
<meta name="citation_reference" content="Chretien, J.P., et al., Influenza forecasting in human populations: a scoping review. PLoS One, 2014. 9(4)" />
<meta name="citation_reference" content="Biggerstaff, M., et al., Results from the centers for disease control and prevention's predict the 2013 2014 Influenza Season Challenge. BMC Infect Dis, 2016. 16" />
<meta name="citation_reference" content="Moran, K.R., et al., Epidemic Forecasting is Messier Than Weather Forecasting: The Role of Human Behavior and Internet Data Streams in Epidemic Forecast. Journal of Infectious Diseases, 2016. 214(suppl 4)" />
<meta name="citation_reference" content="HHS. 2003, Disclosures For Public Health Activities. [cited April 30, 2017]; Available from: https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/special/publichealth/publichealth.pdf." />
<meta name="citation_reference" content="Shekalaghe, S., et al., Clinical performance of an automated reader in interpreting malaria rapid diagnostic tests in Tanzania. Malar J, 2013. 12" />
<meta name="citation_reference" content="InSTEDD. Connectivity Solutions for TB Diagnostics. [cited April 30, 2017]; Available from: http://instedd.org/project/connectivitysolutionsfortbdiagnostics/." />
<meta name="citation_reference" content="HHS. Health Information Privacy. [cited; Available from: http://www.hhs.gov/ocr/privacy/index.html." />
<meta name="citation_reference" content="HHS. Summary of the HIPAA Privacy Rule. [cited; Available from: https://www.hhs.gov/sites/default/files/privacysummary.pdf." />
<meta name="citation_reference" content="HHS. Special Topics in Health Information Privacy. [cited; Available from: http://www.hhs.gov/ocr/privacy/hipaa/understanding/special/index.html." />
<meta name="citation_reference" content="Office for Civil Rights. 2012, Guidance Regarding Methods for De-identification of Protected Health Information in Accordance with the Health Insurance Portability and Accountability Act (HIPAA) Privacy Rule. [cited April 30, 2017]; Available from: https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/coveredentities/Deidentification/hhs_deid_guidance.pdf." />
<meta name="citation_reference" content="Federal Committee on Statistical Methodology, Statistical and Science Policy, Statistical Policy Working Paper 22, Report on Statistical Disclosure Limitation Methodology. 2005, Office of Information and Regulatory Affairs, Office of Management and Budget,." />
<meta name="citation_reference" content="Amazon. 2017, Architecting for HIPAA Security and Compliance on Amazon Web Services. [cited April 30, 2017]; Available from: https://d0.awsstatic.com/whitepapers/compliance/AWS_HIPAA_Compliance_Whitepaper.pdf." />
<meta name="citation_reference" content="BioFire Diagnostics. Guidelines for Laboratory Verification of Performance of the FilmArray(r) Respiratory Panel (RP). [cited April 30, 2017]; Available from: http://www.biofiredx.com/wpcontent/uploads/2016/03/TNFLM1PRT006007GuidelinesforLaboratoryVerificationofPerformanceoftheFilmArrayRespiratoryPanel.pdf." />
<meta name="citation_reference" content="CDC. Nation and Regional Level Outpatient Illness and Viral Surveillance. [cited June 8, 2017]; Available from: https://gis.cdc.gov/grasp/fluview/fluportaldashboard.html." />
<meta name="twitter:title" content="Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. [www.syndromictrends.com][1] presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.

 [1]: http://www.syndromictrends.com" />
<meta name="og-title" property="og:title" content="Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/157156v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. [www.syndromictrends.com][1] presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.

 [1]: http://www.syndromictrends.com" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-09-22" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/157156v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/157156v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/157156v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/157156v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/58365" />
    <title>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qc3wshjaPRgX4JyAGv_xtbijoGgl6ap8IyQo7yhZ4M0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"58365","apath":"\/biorxiv\/early\/2017\/09\/22\/157156.atom","pisa":"biorxiv;157156v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;157156v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/157156v2","title":"Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies","identifier":"node\/58365"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002214kGlYynaTCvE1B4KtUOiaHaYUILx+BhXXneiFXMNtELVoXDgq0SHUTlpoCszdB6QA6PS8Mm+u5+tkpwBf4bH8gY9o\\\\\\\/2zxeoXmWJVpqjliYA4Qr6oECb3giqPi2upz33RDl7NXXOCATlxL1joewaYMUUaQ+Vi4VzYP+UCt6lHN7bIR29CzuwMuuWeBrnYZ1Z\\\\\\\/7vIorhnrkMvZkXm5EVcNO\\\\\\\/bF59yWuWfFf49+BS0wGGz6FsE2RRxvDFY0l81RwtG4b6\\\\\\\/PQQINZgIve+loA9jFsq8zwdcZUIOvLHlZbg9ZwTTjsJ4uror6px77r5JY8SC6tHHU8H7I2thhiHYviauWKzF83GYhzN1oLnVM7usFGHt7dHxP6KXo9TGyi9+TDHSgI1e4DqVtlEGqYqb6iLJsh2Awr6GA83zf0jIZPSSTh7LTN\\\\\\\/IOjkMknDnQH0tfm2T4t+lKcrbZr2DUXmc1wGsRMajy1VQyyUtJ5ZPnH67u76aOdTKiSl4QlrN9X42\\\\\\\/cpY3Fw844dztIp8QIZK3rbZuDIDCKpGOoD+YgFq\\\\\\\/9LpobMZCVMaZ+yF6kyP9p7h2a9LcsJ8Fv6EifkI7J\\\\\\\/YrTxLAlAJXM647V2Ivt3j4bLzGPyrYAVf7Vm9lxUpHwArc+ieTTRdfvqUvIiBHTwDW\\\\\\\/X8IyiJW7+1qQXZCUB0tk0MGMXFJbXXvN7McNhCh0vqc+aJgA3XR+F9W965lnYDsXEOM+dsIqZ2EvBISWe6YLOA2hC+Nl1YHIYSx\\\\\\\/YkMoTFP8NsLhcQ9Uqg\\\\\\\/4\\\\\\\/bfDk37CKcbPJHDU3LBANRLCdg2kAOTsLMB7LNiQ\\\\\\\/1zpyeAGHk4lNF4yRp5E33bJgZcApCeXWxSTtOUpYD8CccxddFMe1MtiIT0BskjWok9T\\\\\\\/jNWSH8zeXVZLpjFfbZUxsiA1e\\\\\\\/P+VpjTaJEWYQdORfVQQQo6dWENzMU+O6o9Tu\\\\\\\/pjEVuwPQjswshie\\\\\\\/kJAYifjaTcT9W4IKSRzcbwfvi+REQX1fjg8WQf2kt8X6vE0vP7+c9LGbTNzfRXzBkMeMwwUsTRKVv7sPU4k22jPwrzRbI3rsSBViT7ohEKzj+4xtn0M2JOEp7rsCeX49IGoqbO\\\\\\\/pRGR8JZ9HNIhuggDY8L6MYeOlfjeZDrlzwni51ak6lGbXeRwu6y6Vm63SogcNMgwcNF\\\\\\\/uMul3h4Zyruu0hLPyHqeSvgNpOhQVOZRp6Y+X24GN4XWDqC9GK+HMZUra\\\\\\\/wl\\\\\\\/o75gxmcTrGrkOHTq+G2nv+QyWuRrZAa5pjpiXj\\\\\\\/qMAFNIUx7dHwKEefcKHQ4OpVyAj2x7vX+XvLmN9R7Uf90IZbS1Em+s=\\\u0022,\\\u0022iv\\\u0022:\\\u0022vOwX+UxcshsWegD1kJFhUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228eea1eab4a57852638830aa9d9d5fd32d566d23e20290e37a07b50189277105c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/157156v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-58365 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/157156v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y2mtn84jJZGh-RTan5DaNiXQ4uL41FCsbhPWs3cJNTs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="58365" id="node58365" data-pisa="biorxiv;157156v2" data-pisa-master="biorxiv;157156" data-apath="/biorxiv/early/2017/09/22/157156.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2994-274X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Lindsay</span> <span class="nlm-surname">Meyers</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8200-0324" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Ginocchio</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4590-4276" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Aimie N.</span> <span class="nlm-surname">Faucett</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4943-9220" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Frederick S.</span> <span class="nlm-surname">Nolte</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9869-8122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Per H.</span> <span class="nlm-surname">Gesteland</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Amy</span> <span class="nlm-surname">Leber</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9208-0647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Diane</span> <span class="nlm-surname">Janowiak</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9296-6931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Virginia</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-0524-9473" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Dien Bard</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-2642-4771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Spitzer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-0160-6034" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Stellrecht</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-8069-5185" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Hossein</span> <span class="nlm-surname">Salimnia</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0003-3275-6657" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Rangaraj</span> <span class="nlm-surname">Selvarangan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-4658-6276" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Juretschko</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0003-4801-7023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Judy A.</span> <span class="nlm-surname">Daly</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jeremy C.</span> <span class="nlm-surname">Wallentine</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0002-8172-5964" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kristy</span> <span class="nlm-surname">Lindsey</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0002-3019-0437" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Franklin</span> <span class="nlm-surname">Moore</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18"><a href="http://orcid.org/0000-0001-7615-9366" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sharon L.</span> <span class="nlm-surname">Reed</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Aguero-Rosenfeld</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="20"><a href="http://orcid.org/0000-0003-0939-6884" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Paul D.</span> <span class="nlm-surname">Fey</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Gregory A.</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Steve J.</span> <span class="nlm-surname">Melnick</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Robinson</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Jennifer F.</span> <span class="nlm-surname">Meredith</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="25"><a href="http://orcid.org/0000-0001-6071-4898" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Camille V.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author" data-delta="26"><span class="nlm-given-names">Robert K.</span> <span class="nlm-surname">Nelson</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="27"><a href="http://orcid.org/0000-0001-9091-5316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jay D.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="28"><a href="http://orcid.org/0000-0001-5716-2770" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Samuel V.</span> <span class="nlm-surname">Scarpino</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="29"><a href="http://orcid.org/0000-0002-5464-654X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Benjamin M.</span> <span class="nlm-surname">Althouse</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="30"><a href="http://orcid.org/0000-0003-0620-7419" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kirk M.</span> <span class="nlm-surname">Ririe</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="31"><a href="http://orcid.org/0000-0003-3040-5175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bradley A.</span> <span class="nlm-surname">Malin</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="32"><a href="http://orcid.org/0000-0002-5060-0560" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Poritz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/157156 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node58365" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay Meyers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BMeyers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyers%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALindsay%2BMeyers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2994-274X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lindsay Meyers</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lindsay.meyers@biofiredx.com" class="" data-icon-position="" data-hide-link-title="0">lindsay.meyers@biofiredx.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine C. Ginocchio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>bioMérieux USA</span>, Durham, NC 27712</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Hofstra Northwell School of Medicine</span>, Hempstead, NY 11549</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BC.%2BGinocchio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ginocchio%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristine%2BC.%2BGinocchio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8200-0324" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christine C. Ginocchio</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimie N. Faucett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimie%2BN.%2BFaucett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faucett%20AN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAimie%2BN.%2BFaucett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4590-4276" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimie N. Faucett</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Frederick S. Nolte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Medical University of South Carolina</span>, Charleston, SC 29425</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BS.%2BNolte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nolte%20FS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BS.%2BNolte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4943-9220" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frederick S. Nolte</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Per H. Gesteland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>University of Utah</span>, Salt Lake City, UT 84112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BH.%2BGesteland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gesteland%20PH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APer%2BH.%2BGesteland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9869-8122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Per H. Gesteland</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Amy Leber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Nationwide Children’s Hospital</span>, Columbus, OH 43205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BLeber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leber%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BLeber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Diane Janowiak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>South Bend Medical Foundation</span>, South Bend, IN 46601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BJanowiak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janowiak%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BJanowiak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9208-0647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diane Janowiak</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Virginia Donovan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Winthrop University Hospital</span>, Mineola, NY 11501</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BDonovan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Donovan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVirginia%2BDonovan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9296-6931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Virginia Donovan</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jennifer Dien Bard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Children’s Hospital of Los Angeles</span>, Los Angeles, CA 90027</div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Keck School of Medicine, University of Southern California</span>, Los Angeles, CA 90039</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BDien%2BBard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dien%20Bard%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BDien%2BBard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0524-9473" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer Dien Bard</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Silvia Spitzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>Stony Brook University Medical Center</span>, Stony Brook, NY 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvia%2BSpitzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spitzer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASilvia%2BSpitzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2642-4771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Silvia Spitzer</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kathleen A. Stellrecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>12</span><span class='nlm-institution'>Albany Medical Center</span>, Albany, NY 12208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BA.%2BStellrecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stellrecht%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BA.%2BStellrecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0160-6034" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathleen A. Stellrecht</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Hossein Salimnia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>13</span><span class='nlm-institution'>Detroit Medical Center</span>, Detroit, MI 48201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hossein%2BSalimnia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salimnia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHossein%2BSalimnia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8069-5185" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hossein Salimnia</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rangaraj Selvarangan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>14</span><span class='nlm-institution'>Children’s Mercy Hospital</span>, Kansas City, MO 64108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rangaraj%2BSelvarangan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Selvarangan%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARangaraj%2BSelvarangan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3275-6657" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rangaraj Selvarangan</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Stefan Juretschko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>15</span><span class='nlm-institution'>Northwell Health, Lake Success</span>, NY 11042</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BJuretschko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juretschko%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BJuretschko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4658-6276" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Juretschko</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Judy A. Daly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>16</span><span class='nlm-institution'>Primary Children’s Hospital</span>, Salt Lake City, UT 84132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judy%2BA.%2BDaly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daly%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudy%2BA.%2BDaly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4801-7023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Judy A. Daly</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Jeremy C. Wallentine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>17</span><span class='nlm-institution'>Intermountain Medical Center</span>, Murray, UT 84107</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BC.%2BWallentine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallentine%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeremy%2BC.%2BWallentine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Kristy Lindsey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>18</span><span class='nlm-institution'>University of Massachusetts Medical School-Baystate</span>, Springfield, MA 01199</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristy%2BLindsey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lindsey%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristy%2BLindsey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8172-5964" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kristy Lindsey</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Franklin Moore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>18</span><span class='nlm-institution'>University of Massachusetts Medical School-Baystate</span>, Springfield, MA 01199</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Franklin%2BMoore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moore%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFranklin%2BMoore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3019-0437" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Franklin Moore</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Sharon L. Reed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>19</span><span class='nlm-institution'>UC San Diego Health</span>, San Diego, CA 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BL.%2BReed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reed%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BL.%2BReed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7615-9366" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharon L. Reed</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Maria Aguero-Rosenfeld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>20</span><span class='nlm-institution'>New York University Langone Health</span>, New York, NY 10016</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BAguero-Rosenfeld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguero-Rosenfeld%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BAguero-Rosenfeld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Paul D. Fey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>21</span><span class='nlm-institution'>University of Nebraska Medical Center</span>, Omaha, NE 68198</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BD.%2BFey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fey%20PD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BD.%2BFey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0939-6884" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul D. Fey</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Gregory A. Storch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>22</span><span class='nlm-institution'>Washington University Medical Center</span>, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BA.%2BStorch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storch%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BA.%2BStorch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Steve J. Melnick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>23</span><span class='nlm-institution'>Nicklaus Children's Hospital</span>, Miami, FL 33155</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BJ.%2BMelnick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melnick%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BJ.%2BMelnick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">Christine C. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>24</span><span class='nlm-institution'>Children's Hospital Colorado</span>, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BC.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BC.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Jennifer F. Meredith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>25</span><span class='nlm-institution'>Greenville Health System</span>, Greenville, SC 29605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BF.%2BMeredith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meredith%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BF.%2BMeredith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Camille V. Cook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camille%2BV.%2BCook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cook%20CV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACamille%2BV.%2BCook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6071-4898" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Camille V. Cook</a></li></ul></div><div class="author-tooltip-26"><div class="author-tooltip-name">Robert K. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BK.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BK.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-27"><div class="author-tooltip-name">Jay D. Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>BioFire Diagnostics</span>, Salt Lake City, UT 84103</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BD.%2BJones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jones%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJay%2BD.%2BJones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9091-5316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jay D. Jones</a></li></ul></div><div class="author-tooltip-28"><div class="author-tooltip-name">Samuel V. Scarpino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>26</span><span class='nlm-institution'>University of Vermont</span>, Burlington, VT 05405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BV.%2BScarpino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scarpino%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BV.%2BScarpino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5716-2770" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel V. Scarpino</a></li></ul></div><div class="author-tooltip-29"><div class="author-tooltip-name">Benjamin M. Althouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>27</span><span class='nlm-institution'>University of Washington</span>, Seattle, WA 98105</div><div class='author-affiliation'><span class='nlm-sup'>28</span><span class='nlm-institution'>New Mexico State University</span>, Las Cruces, NM 87131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BM.%2BAlthouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Althouse%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BM.%2BAlthouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5464-654X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin M. Althouse</a></li></ul></div><div class="author-tooltip-30"><div class="author-tooltip-name">Kirk M. Ririe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>bioMérieux USA</span>, Durham, NC 27712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kirk%2BM.%2BRirie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ririe%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKirk%2BM.%2BRirie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0620-7419" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kirk M. Ririe</a></li></ul></div><div class="author-tooltip-31"><div class="author-tooltip-name">Bradley A. Malin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>29</span><span class='nlm-institution'>Vanderbilt University</span>, Nashville, TN 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BA.%2BMalin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Malin%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABradley%2BA.%2BMalin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3040-5175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bradley A. Malin</a></li></ul></div><div class="author-tooltip-32"><div class="author-tooltip-name">Mark A. Poritz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>30</span><span class='nlm-institution'>BioFire Defense</span>, Salt Lake City, UT 84107</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BPoritz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poritz%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BA.%2BPoritz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5060-0560" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark A. Poritz</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/157156v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:58365" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:58365/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/157156v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:58365" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:58365/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/157156v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:58365" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:58365/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/157156v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:58365" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:58365/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/157156v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:58365" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:58365/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray<sup>®</sup> Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. <a href="http://www.syndromictrends.com">www.syndromictrends.com</a> presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">The availability of real-time surveillance data that can monitor the spread of infectious diseases benefits public health [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. At present, tracking of respiratory or foodborne outbreaks relies on a variety of methods ranging from automated real-time electronic reporting to manual web entry of test results. Systems such as the Centers for Disease Control and Prevention’s (CDC) FluView [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], National Respiratory and Enteric Virus Surveillance Systems (NREVSS, [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]), National Electronic Disease Surveillance System (NEDSS [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]), Global Emerging Infections Surveillance (GEIS) [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], and others, although web-based still require manual entry of data from laboratories, resulting in data that are often incomplete or not current.</p><p id="p-4">Newer efforts at syndrome-based surveillance [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] include BioSense (extraction of symptomatic data from electronic health records [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]), Google Flu (tracking of Internet search queries [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>] but recently discontinued [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]), and Flu Near You (voluntary reporting [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]). Additionally, numerous next generation, syndromic surveillance systems; e.g., pharmacy sales records [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], Twitter conversations [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], and Wikipedia hits [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], have come online in the past five years. However, these systems cannot report the specific pathogen causing an increase in a particular set of symptoms. Finally, there are more localized efforts such as GermWatch in Utah [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] and the Electronic Clinical Laboratory Reporting System (ECLRS) in New York [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] that draw from hospital (HIS) and laboratory (LIS) information systems. This disparity in technologies and data collection methods results in incomplete surveillance.</p><p id="p-5">Comprehensive and uniform diagnostic test data will aid in the identification of potential outbreaks. A combination of broad respiratory pathogen testing and an internal electronic surveillance system enabled the rapid dissemination of data across the largest health care system in New York, the North Shore-LIJ Health System (now Northwell Health) during the influenza A H1N1-2009 pandemic in the New York City area. Pathogen-specific molecular testing permitted rapid a) notification to state epidemiologists; b) tracking of the virus so that health care resources could be managed effectively; and c) evaluation of influenza diagnostics [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Today, with the threat of emerging pathogens such as Middle East Respiratory Syndrome coronavirus (MERS CoV), avian influenza, enterovirus D68, and Ebola virus, real-time surveillance programs are critical [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].</p><p id="p-6">It is not always possible to accurately diagnose the causative agents of most infectious diseases from symptoms alone due to overlapping clinical presentation. Thus, to achieve maximal utility, infectious disease surveillance systems should move beyond syndrome-based reporting and be pathogen-specific and comprehensive, reporting on as many of the common pathogens for a particular syndrome as possible. Sensitive and specific automated molecular diagnostic systems that detect up to four different pathogens in a single sample have been available from in vitro diagnostic (IVD) manufacturers for some time [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>-<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. However adoption of IVD platforms with broad multiplexing capability has become widespread only in the last few years. Commercially available systems that can detect most of the known etiological agents for respiratory, gastrointestinal and other multi-pathogen syndromes [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>-<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] include the BioFire (Salt Lake City, UT) FilmArray<sup>®</sup> System [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]; the GenMark (Carlsbad, CA) eSensor XT-8<sup>®</sup> [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>] and ePlex<sup>®</sup> [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]; and the Luminex (Austin, TX) xTAG<sup>®</sup> [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>], nxTag<sup>®</sup> [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>] and Verigene<sup>®</sup> systems [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>].</p><p id="p-7">Multi-analyte diagnostic tests provide the raw data needed for real-time pathogen-specific surveillance but there remain a number of obstacles to sharing these results (reviewed in [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]). The obstacles largely center on information privacy and network security. A real-time surveillance system using diagnostic test results requires safeguards for protected health information (PHI). Medical records and devices have become attractive targets for cyber attackers in recent years [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>], which has made hospitals and clinics reluctant to connect their Local Area Networks (LANs) to the Internet. Releasing patient test results requires the removal of PHI or authorization from the patient. Studies have shown that de-identification of patient data is not as simple as removing all specific identifiers because, in the age of big data, under the right circumstances it is possible to re-associate patients and their data using publicly available information [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>-<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>].</p><p id="p-8">We describe here the implementation of a real-time pathogen-specific surveillance system that overcomes the PHI concerns noted above. FilmArray Trend de-identifies, aggregates and exports test results from FilmArray Instruments in use in US clinical laboratories. Although data from all commercially available FilmArray Panels (Methods and [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]) is exported to the Trend database, we focus here on the Respiratory Panel (RP), which can detect 17 viral (adenovirus [ADV], coronaviruses [CoV OC43, 229E, NL63, HKU-1], human metapneumovirus [hMPV], human rhinovirus/enterovirus [HRV/EV]; influenza A [Flu A, subtyping H1N1, 2009 H1N1, H3N2], influenza B [Flu B]; parainfluenza viruses [PIV1-4], and respiratory syncytial virus [RSV]), and three bacterial (<em>Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae</em>) pathogens [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>-<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]</p><p id="p-9">With more than 344,000 patient results for the FilmArray RP test alone, the Trend database has many of the properties associated with “big data” as it applies to infectious disease [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. After describing how the dataset can be cleaned of non-patient tests, we make some observations on the seasonality of the different respiratory pathogens and we apply the ecological concept of “species diversity” [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>] to observe a correlation between the abundance of each pathogen and the rate at which co-detections (more than one positive result per test) occur.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Sending FilmArray data directly to the cloud</h3><p id="p-10">The FilmArray Trend public website is an outgrowth of the Trend clinical website, which was developed to provide clinical laboratories using the FilmArray System with up-to-date information on the respiratory, gastrointestinal and meningitis/encephalitis pathogens circulating at their institutions. The most general and efficient way to export both the clinical and public Trend information is to follow a “Bottom-Out” approach to data export (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In this scheme, the FilmArray Instrument sends data via the Internet directly to a single cloud database where it can be viewed by health care providers at the originating institution. This data export pathway contrasts with a “Top-Out” approach (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) in which diagnostic test results are pushed from the instrument up to the LIS, then to the HIS and, finally, a subset of this information is forwarded to cloud-based databases.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schema for export of IVD test results to an external database Bottom-Out and Top-Out approaches for data export are indicated by solid and dashed lines respectively. Some institutions have developed their own systems for aggregating and displaying infectious disease data (indicated by &#x201C;Internal website&#x201D;). Other abbreviations are defined in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Schema for export of IVD test results to an external database&lt;/span&gt; Bottom-Out and Top-Out approaches for data export are indicated by solid and dashed lines respectively. Some institutions have developed their own systems for aggregating and displaying infectious disease data (indicated by &#x201C;Internal website&#x201D;). Other abbreviations are defined in the text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F1.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F1.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Schema for export of IVD test results to an external database</span><p id="p-11" class="first-child">Bottom-Out and Top-Out approaches for data export are indicated by solid and dashed lines respectively. Some institutions have developed their own systems for aggregating and displaying infectious disease data (indicated by “Internal website”). Other abbreviations are defined in the text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Initial testing of the Trend export mechanism was performed in collaboration with the Clinical Laboratories of the Medical University of South Carolina. This trial allowed us to develop and test auto-export functions and de-identification protocols for the Trend software. The de-identification requirement of the Health Insurance Portability and Accountability Act (HIPAA) of 1996 [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], specifically the Safe Harbor provision, requires the removal of 18 enumerated variables that could directly or indirectly identify an individual. In accord with this requirement, the first stage study did not export test identifiers, or free form text fields and only returned the year of the test. The initial dataset provided low-resolution information but was a useful platform to evaluate the proposed system. Further development to enable export of higher resolution data required the design of routines that would adhere to an alternative HIPAA de-identification strategy, namely, the Expert Determination approach, which requires a risk assessment demonstrating that the chance of re-identifying an individual is sufficiently small [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. The Expert Determination process identified and made recommendations for fields that could facilitate disclosure of PHI. A summary of the Expert Determination results detailing the risk of Trend data in regard to replicability, availability and distinguishability is shown in Methods, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/371314/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/371314/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/371314" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <span class="caption-title">Distribution of number of positive results per test for Trend RP tests</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/371450/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/371450/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/371450" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <span class="caption-title">Qualitative analysis for the Expert Determination</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">Because the Expert Determination study established that no PHI will be disclosed by the Trend data export algorithm, Data Use Agreements (DUA), rather than Business Associates Agreements (BAA, see Methods for the difference between a DUA and a BAA) were executed with each of the collaborating institutions (listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The DUAs define for the clinical laboratory how BioFire will manage and make use of the Trend data. The Trend client software residing on the FilmArray computer queries the FilmArray test result local database and exports the results to an Amazon Web Services (AWS) database (see Methods). The Trend client software performs de-identification on the FilmArray computer prior to export as detailed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. Health care providers are granted access to their institutions Trend data by the Laboratory Director. Since web access to view the data is restricted to the local site, de-identification of geographic indicators is not required. However, in implementation of the public Trend website, which presents the national syndromic test surveillance (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), we have further aggregated the data with respect to geographic origin and obfuscated the date of the test (see Methods). Since only de-identified data are exported from the clinical institutions, no PHI is sent to or stored on the cloud server.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/371176/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/371176/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/371176" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <span class="caption-title">Clinical laboratories participating in the initial Trend study</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">All sites submitted the Trend project for review by their local Institutional Review Board (IRB); all but one of the 20 IRBs deemed the project “exempt” because of the absence of PHI export. Thus the security requirements for the database and the controls necessary for storage and transport of de-identified data are significantly reduced.</p><p id="p-15">Following the protocol established by Expert Determination review, the Trend software delays the export of data until the number of tests queued for export exceeds a minimum threshold for each type of FilmArray panel. In practice, this results in an average time to export of less than two hours from each site that has multiple instruments. 98.5% of the tests are exported within 24 hours.</p></div><div id="sec-4" class="subsection"><h3>Characteristics of the FilmArray sites used in the Trend pilot study</h3><p id="p-16">The 20 sites contributing to the Trend pilot project (Methods, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) have the same average number of instruments, six (range one to 22) as for all US FilmArray customers. The Trend pilot sites have been using the FilmArray RP test for an average of 3.8 years (range one to six) prior to July 2017. The size of the institutions participating ranges from 300 to 6400 beds with the majority being large hospitals and health care networks with an average of 1,100 beds. Six (30%) sites are pediatric hospitals and one is a reference laboratory. Fifteen (75%) of the sites have uploaded archived FilmArray RP test results to the Trend database, with eight (40%) reporting results dating back to 2012. Unless stated otherwise, the data presented here covers the period from July, 2013 to July, 2017.</p><p id="p-17">The algorithm used to diagnose the cause of respiratory disease varies by site. More than half of the Trend sites do not enforce an institutional respiratory testing protocol and, even within sites that have a required protocol, some discretionary use of Film Array RP is allowed. Without detailed records from each institution’s HIS, it is not possible to determine whether the FilmArray RP was used as a front line test or as a reflex test (typically following a negative result for influenza and/or RSV).</p></div><div id="sec-5" class="subsection"><h3>Cleaning non patient test results from the FilmArray Trend database</h3><p id="p-18">To determine the prevalence of respiratory pathogens, we needed to expunge the Trend database of test results that are not derived from clinical patient samples. Non-patient results come from a variety of sources including verification testing, routine Quality Control (QC) and proficiency testing (PT) (see Methods, Cleaning the Trend data). Despite this complexity, the majority of non-patient test results can be identified and distinguished from the patient-derived data because of the high number of positive organism calls in a single test and because of the temporal aspects of verification and control testing (see Methods). QC tests are estimated to account for half of all FilmArray RP results in which more than 3 organisms are detected. A combination of filters (Methods) removes 3.52% of the total tests from the Trend data set. We estimate that 3% of the remaining dual and 6% of the remaining triple detection tests respectively may come from QC testing. In addition to the exclusion of tests temporally associated with validation events, all results with four or more positives were removed from further analysis (0.9% of the filtered total total). This includes the 0.16% of test results with exactly four organisms (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, “Tests after event removal” column) because the minority are derived from patient testing.</p></div><div id="sec-6" class="subsection"><h3>Detection of respiratory pathogens in Trend samples from 2013 to 2017</h3><p id="p-19">The detection counts and pathogen detection rates derived from the Trend data set for each organism in the FilmArray RP are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Other views of these data, including percent detection of individual organisms or combinations of organisms, are available on the FilmArray Trend website: <a href="http://www.syndromictrends.com">www.syndromictrends.com</a>. The FilmArray RP Test Utilization Rate (RP TUR, see Methods) and the individual organism detection counts increased over this period because the Trend clinical sites increased their utilization of the FilmArray RP tests (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Seasonal fluctuations can also be seen within this growth pattern, with use increasing up to fourfold each winter when compared to the previous summer. HRV/EV, the most common pathogen detected group, is identified in approximately 30% of all samples tested each year (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 2-figure supplement 1</a>). Other pathogens detected in five to 10% of the samples include: RSV, the PIVs, ADV, Flu A and hMPV. <em>M. pneumoniae, C. pneumoniae</em>, <em>B. pertussis</em> and Flu B are detected in less than 2% of all samples. The average percentage of each organism is relatively constant over the four years of data in the Trend database (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>-figure supplement 2).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QC runs are performed when a new lot of FilmArray RP pouches is introduced Histogram of count of FilmArray RP tests with more than three positive results after a new lot of FilmArray RP pouches is introduced at a clinical site. The X-axis is the sequential number of RP tests associated with the new lot that have been performed at that site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;QC runs are performed when a new lot of FilmArray RP pouches is introduced&lt;/span&gt; Histogram of count of FilmArray RP tests with more than three positive results after a new lot of FilmArray RP pouches is introduced at a clinical site. The X-axis is the sequential number of RP tests associated with the new lot that have been performed at that site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Table 1-Figure Supplement 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F6.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Table 1-Figure Supplement 1:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F6.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Table 1-Figure Supplement 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Table 1-Figure Supplement 1:</span> <span class="caption-title">QC runs are performed when a new lot of FilmArray RP pouches is introduced</span><p id="p-102" class="first-child">Histogram of count of FilmArray RP tests with more than three positive results after a new lot of FilmArray RP pouches is introduced at a clinical site. The X-axis is the sequential number of RP tests associated with the new lot that have been performed at that site.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of adenovirus and three bacteria Detection of adenovirus and the three bacterial pathogens (B. pertussis, C. pneumoniae, and M. pneumoniae) in the Trend dataset displayed as a stacked area graph. CDC measured ILI is indicated (orange line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection of adenovirus and three bacteria&lt;/span&gt; Detection of adenovirus and the three bacterial pathogens (&lt;em&gt;B. pertussis, C. pneumoniae&lt;/em&gt;, and &lt;em&gt;M. pneumoniae&lt;/em&gt;) in the Trend dataset displayed as a stacked area graph. CDC measured ILI is indicated (orange line).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2-Figure Supplement 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F7.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2-Figure Supplement 1:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F7.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2-Figure Supplement 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2-Figure Supplement 1:</span> <span class="caption-title">Detection of adenovirus and three bacteria</span><p id="p-103" class="first-child">Detection of adenovirus and the three bacterial pathogens (<em>B. pertussis, C. pneumoniae</em>, and <em>M. pneumoniae</em>) in the Trend dataset displayed as a stacked area graph. CDC measured ILI is indicated (orange line).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of FilmArray RP organisms by type Percent detection rates aggregated from all participating Trend sites. The percent positive detection per test for each organism group is ordered by abundance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Detection of FilmArray RP organisms by type&lt;/span&gt; Percent detection rates aggregated from all participating Trend sites. The percent positive detection per test for each organism group is ordered by abundance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2-Figure Supplement 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F8.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2-Figure Supplement 2:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F8.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2-Figure Supplement 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2-Figure Supplement 2:</span> <span class="caption-title">Detection of FilmArray RP organisms by type</span><p id="p-104" class="first-child">Percent detection rates aggregated from all participating Trend sites. The percent positive detection per test for each organism group is ordered by abundance.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Organism detection by type and year Percent detection rates from all participating Trend sites for each FilmArray RP organism are broken out by respiratory year. Organisms are ordered by total abundance across the four years." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Organism detection by type and year&lt;/span&gt; Percent detection rates from all participating Trend sites for each FilmArray RP organism are broken out by respiratory year. Organisms are ordered by total abundance across the four years.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2-Figure Supplement 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F9.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2-Figure Supplement 3:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F9.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2-Figure Supplement 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2-Figure Supplement 3:</span> <span class="caption-title">Organism detection by type and year</span><p id="p-105" class="first-child">Percent detection rates from all participating Trend sites for each FilmArray RP organism are broken out by respiratory year. Organisms are ordered by total abundance across the four years.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear regression of MIE and observed co detections The time series data in figure 5B shown as a scatter plot. The equation of the linear regression is: MIE = 4.05153* Co-Detection rate-0.0206 with an R2 value of 0.9003" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Linear regression of MIE and observed co detections&lt;/span&gt; The time series data in figure 5B shown as a scatter plot. The equation of the linear regression is: MIE = 4.05153* Co-Detection rate-0.0206 with an R&lt;sup&gt;2&lt;/sup&gt; value of 0.9003&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5-Figure Supplement 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F10.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5-Figure Supplement 1:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F10.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5-Figure Supplement 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5-Figure Supplement 1:</span> <span class="caption-title">Linear regression of MIE and observed co detections</span><p id="p-106" class="first-child">The time series data in figure 5B shown as a scatter plot. The equation of the linear regression is: MIE = 4.05153* Co-Detection rate-0.0206 with an R<sup>2</sup> value of 0.9003</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of RP organisms over time across all sites Figure 2 legend: Detection of FilmArray RP pathogens in the Trend dataset displayed as a stacked area graphs. All panels have the same time period (July 2013 to June 2017). A) Count of each organism. The TUR metric (red line, units are FilmArray RP tests performed), and count of FilmArray RP tests that are negative (white area between pathogen count and TUR) are indicated. The y-axis values are not indicated as this is considered proprietary information. B) Pathogen detection rates for all organisms. C) Pathogen detection rates for the subset of organisms that show seasonality (see Results and the legend for the list of organisms). D) HRV/EV detection rates. CDC measured ILI is indicated (black line) in panels C and D. Organisms follow the same color scheme in all panels; the order of organisms in the legend (down then across) matches the stacked area graph top to bottom. Figure 2-Figure Supplement 1:Detection of adenovirus and three bacteria Figure 2-Figure Supplement 2:Detection of FilmArray RP organisms by type Figure 2-Figure Supplement 3:Organism detection by type and year" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection of RP organisms over time across all sites&lt;/span&gt; &lt;strong&gt;Figure 2 legend:&lt;/strong&gt; Detection of FilmArray RP pathogens in the Trend dataset displayed as a stacked area graphs. All panels have the same time period (July 2013 to June 2017). A) Count of each organism. The TUR metric (red line, units are FilmArray RP tests performed), and count of FilmArray RP tests that are negative (white area between pathogen count and TUR) are indicated. The y-axis values are not indicated as this is considered proprietary information. B) Pathogen detection rates for all organisms. C) Pathogen detection rates for the subset of organisms that show seasonality (see Results and the legend for the list of organisms). D) HRV/EV detection rates. CDC measured ILI is indicated (black line) in panels C and D. Organisms follow the same color scheme in all panels; the order of organisms in the legend (down then across) matches the stacked area graph top to bottom. &lt;strong&gt;Figure 2-Figure Supplement 1:&lt;/strong&gt;Detection of adenovirus and three bacteria &lt;strong&gt;Figure 2-Figure Supplement 2&lt;/strong&gt;:Detection of FilmArray RP organisms by type &lt;strong&gt;Figure 2-Figure Supplement 3&lt;/strong&gt;:Organism detection by type and year&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F2.medium.gif" width="246" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F2.medium.gif" width="246" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <span class="caption-title">Detection of RP organisms over time across all sites</span><p id="p-20" class="first-child"><strong>Figure 2 legend:</strong></p><p id="p-21">Detection of FilmArray RP pathogens in the Trend dataset displayed as a stacked area graphs. All panels have the same time period (July 2013 to June 2017). A) Count of each organism. The TUR metric (red line, units are FilmArray RP tests performed), and count of FilmArray RP tests that are negative (white area between pathogen count and TUR) are indicated. The y-axis values are not indicated as this is considered proprietary information. B) Pathogen detection rates for all organisms. C) Pathogen detection rates for the subset of organisms that show seasonality (see Results and the legend for the list of organisms). D) HRV/EV detection rates. CDC measured ILI is indicated (black line) in panels C and D. Organisms follow the same color scheme in all panels; the order of organisms in the legend (down then across) matches the stacked area graph top to bottom.</p><p id="p-22"><strong>Figure 2-Figure Supplement 1:</strong>Detection of adenovirus and three bacteria</p><p id="p-23"><strong>Figure 2-Figure Supplement 2</strong>:Detection of FilmArray RP organisms by type</p><p id="p-24"><strong>Figure 2-Figure Supplement 3</strong>:Organism detection by type and year</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The pathogens’ seasonal variability measured by percent detection can be classified into at least three groups. <strong>Group 1</strong>: The majority of organisms follow the classical “respiratory” season (October through March) and increase by more than 10-fold above their baseline detection rate (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). These include the CoVs, Flu A, Flu B, hMPV, the PIVs, and RSV (PIV3 is a slight exception to this rule in that it peaks in the summer months, and has a winter peak that is only detected regionally (data not shown)). Within this group, all but five viruses demonstrate significant biennial fluctuations; Flu B, hMPV, OC43, and PIV3 and RSV experience relatively consistent annual peaks. <strong>Group 2</strong>: HRV/EV is in a class by itself in that it is detected in a high percentage of tests over time, (minimum of 10% in winter) and experiences moderate peaks of two to three fold outside the respiratory season baseline, in the early fall and spring (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). <strong>Group 3</strong>: The bacteria and adenovirus are present at a relatively constant rate (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>-Figure supplement 1). The CDC FluView reported rate of Influenza-Like-Illness (ILI) tracks moderately well with the Group 1 organisms (cross correlation of 0.85) and not with HRV/EV or with ADV and the bacteria.</p></div><div id="sec-7" class="subsection"><h3>Comparison of Trend to CDC measures of influenza</h3><p id="p-26">The CDC FluView network [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>] gathers information about influenza prevalence from a large number of public health and clinical laboratories in the US. FluView is considered the gold standard for these measures. We compared the Trend detection rates for Flu A (all subtypes) plus Flu B to the FluView Influenza (A and B) from September 2015 to July 2017 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The analysis was restricted to this time period because of a change in the CDC’s reporting of Flu prevalence in the fall of 2015. A cross-correlation of 0.974 was observed between the Trend Flu A/B percent detection and FluView reported influenza prevalence. Notably, the onset, peak and duration of the influenza season coincide between the two measures.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trend Influenza Detection Rate compared to CDC Influenza activity Percent of combined FilmArray Flu A (all subtypes) and Flu B detections (blue line) and CDC reported Influenza prevalence (black lines). CDC data are aggregated only from regions with participating Trend sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Trend Influenza Detection Rate compared to CDC Influenza activity&lt;/span&gt; Percent of combined FilmArray Flu A (all subtypes) and Flu B detections (blue line) and CDC reported Influenza prevalence (black lines). CDC data are aggregated only from regions with participating Trend sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F3.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F3.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Trend Influenza Detection Rate compared to CDC Influenza activity</span><p id="p-27" class="first-child">Percent of combined FilmArray Flu A (all subtypes) and Flu B detections (blue line) and CDC reported Influenza prevalence (black lines). CDC data are aggregated only from regions with participating Trend sites.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Respiratory Panel Co Detections</h3><p id="p-28">We found that 26,347 FilmArray RP tests in the Trend dataset had 2 or 3 co-detections (7.7% of total FilmArray RP tests) and that the co-detection rate of each organism varies widely (10-55%). Although an additional pathogen was detected in half of the ADV and CoV positive samples, co-detections were observed in only 10-15% of the samples positive for either Flu A or Flu B (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection rates for all organisms compared to co-detections (A) Percent total positive detections for each organism in the RP Trend data set is presented in stacked bars, showing the rate of detection of a single organism (A, blue) and those involved in a co-detection (A, black). Data are calculated for each site during the period of July 2013 to March 2017, when available, and then aggregated. (B) Percentage of each organism involved in a co-detection is shown. Bars are colored by pathogen family (CoV, purple; Bacteria, blue; PIV green; Flu A yellow)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection rates for all organisms compared to co-detections&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Percent total positive detections for each organism in the RP Trend data set is presented in stacked bars, showing the rate of detection of a single organism (A, blue) and those involved in a co-detection (A, black). Data are calculated for each site during the period of July 2013 to March 2017, when available, and then aggregated. &lt;strong&gt;(B)&lt;/strong&gt; Percentage of each organism involved in a co-detection is shown. Bars are colored by pathogen family (CoV, purple; Bacteria, blue; PIV green; Flu A yellow).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F4.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F4.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">Detection rates for all organisms compared to co-detections</span><p id="p-29" class="first-child"><strong>(A)</strong> Percent total positive detections for each organism in the RP Trend data set is presented in stacked bars, showing the rate of detection of a single organism (A, blue) and those involved in a co-detection (A, black). Data are calculated for each site during the period of July 2013 to March 2017, when available, and then aggregated. <strong>(B)</strong> Percentage of each organism involved in a co-detection is shown. Bars are colored by pathogen family (CoV, purple; Bacteria, blue; PIV green; Flu A yellow).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Trend data have high temporal, spatial, and organism-specific resolution. These three properties allow for a novel evaluation of co-detections. The observed rates of co-detections should be influenced by the number of circulating pathogens detected by the FilmArray RP test at a particular site. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> shows the average number of unique organisms detected at each site in a given week (see Methods: Calculation of co-detection rates). This number fluctuates from a summer low of four to a winter high of 11 pathogens. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> (grey bars) shows that the total rate of organism co-detections in the Trend dataset fluctuates annually with peak rates occurring in the winter months. The average rates have been as high as 12% in the winter of 2016 and as low as 2% in the summer of 2014.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Seasonal variation in pathogen diversity and co-detections A) Average circulating pathogen number (black line) and one standard deviation computed across all Trend sites (grey area). B) Rate of co-detections in the RP Trend dataset (grey bars, left axis) the MIE index (purple line, right axis) and MIE confidence intervals (shaded purple area). Figure 5-Figure supplement 1: Linear regression of MIE and observed co-detections" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426873080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Seasonal variation in pathogen diversity and co-detections&lt;/span&gt; &lt;strong&gt;A)&lt;/strong&gt; Average circulating pathogen number (black line) and one standard deviation computed across all Trend sites (grey area). &lt;strong&gt;B)&lt;/strong&gt; Rate of co-detections in the RP Trend dataset (grey bars, left axis) the MIE index (purple line, right axis) and MIE confidence intervals (shaded purple area). &lt;strong&gt;Figure 5-Figure supplement 1&lt;/strong&gt;: Linear regression of MIE and observed co-detections&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F5.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F5.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/09/22/157156/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Seasonal variation in pathogen diversity and co-detections</span><p id="p-31" class="first-child"><strong>A)</strong> Average circulating pathogen number (black line) and one standard deviation computed across all Trend sites (grey area). <strong>B)</strong> Rate of co-detections in the RP Trend dataset (grey bars, left axis) the MIE index (purple line, right axis) and MIE confidence intervals (shaded purple area). <strong>Figure 5-Figure supplement 1</strong>: Linear regression of MIE and observed co-detections</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">From the Trend data, a Measure of Interspecific Encounter (MIE) can be calculated as the probability of a co-detection, weighted by the prevalence of each circulating pathogen at a site. Although the value of the MIE metric is higher than the actual co-detection rate it correlates well (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>, purple line compared to the grey bars has a cross correlation of 0.9488 at a lag of 0). The magnitude adjustment between MIE and the observed co-detections is calculated by the slope of the linear regression of the two metrics (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> n figure Supplement 1) and has a value of 4.05 (R<sup>2</sup> = 0.9003).</p></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-33">This article describes “FilmArray Trend”, a new system for real-time reporting of widespread pathogen-specific syndromic data. This system already has many of the important properties of big data. We consider Trend in terms of the “V”s that are often used to describe big data: volume (amount), velocity (speed of acquisition), veracity (accuracy), variety (diversity of information) and value (utility) [<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>].</p><div id="sec-10" class="subsection"><h3>Trend Volume</h3><p id="p-34">The Trend RP dataset is growing at an average rate of &gt;200,000 pathogen test results per month. Connecting the first 20 clinical sites has provided insight into the principal concerns that will be raised by the legal, IT and administrative departments of the healthcare providers that house FilmArray instruments. It should be possible, therefore, to expand the Trend installed base by 10 to 20-fold over the next few years. Similarly, the existence of Trend should enable other IVD manufacturers to build their own Trend-like systems with greater acceptance on the part of their customers, thereby allowing a more global and comprehensive surveillance perspective.</p></div><div id="sec-11" class="subsection"><h3>Trend Velocity</h3><p id="p-35">The data in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> are similar to previous demonstrations of the seasonality associated with different respiratory viruses [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>-<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. What is novel is that this data is generated automatically, on site, and in close to real-time compared to other surveillance systems. Greater than 98% of the test results are exported to the Trend database within 24 hours of being generated. As part of the de-identification protocol, sequential FilmArray RP tests of the same type are put into the same time bin. This has the effect that test results are exported faster during periods of peak use such as during the peak of the respiratory season or during an outbreak. Trend should be instrumental at a local level to determine the start of respiratory season; many hospitals make significant changes to their operations based on this event however, at present, data collection to track the respiratory season is often slow and manual, or semi-automated at best.</p><p id="p-36">The key to implementing Trend clinical sites was to demonstrate that FilmArray test results can be exported without the risk of breaching PHI confidentiality either directly or through some combination of the data that was exported. Trend successfully used the Expert Determination process as prescribed by the HIPAA guidelines (see Methods), which greatly simplified the data sharing agreement between BioFire Diagnostics and the clinical site and allowed health care providers to use Trend without risk of inadvertently disclosing PHI.</p><p id="p-37">The software architecture underlying the Trend system is both simple and secure: 1) no changes to the institutional firewall or LAN are needed; 2) the Trend database cannot reach back and query the FilmArray computer due to the institutional firewall, which is set to outbound data only; 3) Trend software can only submit data to the cloud database and cannot query the database. Yet, despite this security, authorized users of the Trend database can mine the deidentified data to look for novel patterns in respiratory pathogen epidemiology.</p></div><div id="sec-12" class="subsection"><h3>Trend Veracity</h3><p id="p-38">The goal of an epidemiological surveillance network is to infer which infectious diseases are circulating in the general population based on testing a sample of patients [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>]. Different surveillance systems have different biases in their data, biases that perturb the ability to predict true population prevalence.</p><p id="p-39">While the removal of all PHI has great benefits in terms of implementation, it also has several shortcomings that complicate interpretation of the data. First, Trend cannot account for the variability in the diagnostic testing algorithms applied to the selection of samples to be tested by the FilmArray Instruments. During respiratory season, health care providers may prescreen patients with other diagnostic tests including rapid antigen or molecular assays for influenza and RSV, and/or commercial and laboratory developed molecular tests for a mix of other respiratory pathogens. Depending upon the sensitivity of these upstream tests, 50-90% of influenza and RSV for the subset of the patients screened would be excluded from the Trend dataset if the front line test is positive. This testing protocol may skew the actual prevalence of not only influenza and RSV but all other individual respiratory pathogens and co-infections detected by the FilmArray. In some institutions, testing is reserved for hospitalized patients and others at risk for developing complications of respiratory tract infections including the very young, very old and immunocompromised patients, so Trend data may represent a less healthy patient population and not necessarily general community prevalence. Conversely there are sites that perform a significant number of tests for the outpatient setting. This may create variability among the clinical sites’ percent positivity and introduces a challenge to comparing pathogen intensity between sites.</p><p id="p-40">The uncertainties surrounding the testing algorithm and the precise patient population tested should not interfere with determining the onset, peak and duration of the pathogen season at each institution. These limitations on the data are likely to be common among almost all current surveillance systems for similar reasons. Given these concerns, the agreement between the percent positivity of Flu A/B as determined by Trend and the percent positivity reported by CDC FluView Influenza is striking (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), supporting the validity and utility of the Trend data.</p><p id="p-41">A second source of concern in the Trend data set is a consequence of removal of sample identification such that we cannot directly determine whether the sample was from a patient or was a non-clinical sample (verification test, QC or PT) and should be removed from further epidemiological analysis. We estimate that non-patient testing makes up approximately 1.8% of the total FilmArray RP tests. Automated detection algorithms remove 3.5% of the total RP tests, including approximately half of the non-clinical samples. With the exception of the four positive tests, the clinical samples removed by filtering should be a random sampling of all patient tests. The remaining 1% fraction of non-patient tests have essentially no impact on the Trend evaluation of pathogen prevalence but they do make it more difficult to perform high resolution analysis of pathogen co-detections. This is especially true for co-detections of low prevalence organisms where QC positives are likely to be more common than real positives. Future updates to the FilmArray software will simplify the process by which the instrument operator can tag tests of non-patient samples, thereby largely eliminating the need to filter such test results from the Trend database before analysis.</p></div><div id="sec-13" class="subsection"><h3>Trend Variety</h3><p id="p-42">The total positivity rate of the FilmArray RP test varies from a low of 38% in the summer months to a high of 75% in December and January, with a yearly average of approximately 60%. <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a> shows that the average number of different circulating pathogens at a single institution can vary from eight up to 11 during the winter months. Even during the peak periods of ILI, many respiratory infections are due to other viruses (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>) that can present clinically in a similar fashion [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>]. Therefore, the presumption of an influenza infection based on reported influenza percent positivity, without diagnostic testing for the virus, can lead to the inappropriate use of anti-viral agents [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. Conversely, without comprehensive testing, a negative influenza or RSV test can lead to prescription of an unnecessary antibiotic. FilmArray Trend data can be a valuable aid for antimicrobial stewardship programs because it provides real-time information regarding the causes of respiratory infections and highlights the prevalence of viral infections.</p><p id="p-43">As previously observed [<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>], the viruses that share the winter seasonality of influenza demonstrate annual or biennial behavior. It is possible that the viruses that share an influenza-like seasonality but do not show a two year cycle (RSV and hMPV) are actually alternating strains but the FilmArray RP Test does not detect this difference (e.g. the FilmArray RP does not differentiate between RSV A and RSV B). Adenovirus and the bacteria show constant occurrence through the year; HRV is in a unique class with peaks in the fall and spring.</p><p id="p-44">Detection of multiple respiratory viruses in the same patient has been reported before. In the Trend dataset the rate of dual and triple co-detections was approximately 7.7 %, with HRV/EV as the organism most commonly observed in a co-detection. Some viruses, such as ADV and CoVs are detected in a mixed infection more than 50% of the time (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). In principle a FilmArray RP positive result may represent detection of residual pathogen nucleic acid from a previous infection that has resolved. However, several studies suggest that coinfections are associated with more severe disease ([<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>-<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>], see also discussion in [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]). In such cases, information about multiple detections can provide infection control practitioners with data that can assist in bed management and in the assessment of risk for nosocomial infections in a patient population that has been segregated by the occurrence of a common pathogen. Such information can prevent the introduction of a new pathogen associated with cohorting patients during busy respiratory seasons [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>-<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>].</p><p id="p-45">The question of whether different respiratory pathogens interfere with, or facilitate, growth in a human host is of some interest and not well understood. With the right data it can be studied at the population [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>], individual [<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>], and cellular level [<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a>]. Because the Trend data still includes some non-patient tests, we have chosen not to analyze every possible dual or triple infection individually. Rather we have taken a global approach and compared the overall rate of observed co-detections with MIE, which is a measure of the diversity of viruses circulating in a specific region and time period. MIE is similar, but not identical, to PIE (Probability of Interspecific Encounter [<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]), also referred to as the Gini-Simpson index (1-D, where D is the Simpson s index [<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>]), which is used in ecology as a measure of the species diversity of a region. Similarly, the circulating pathogen number of <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a> is identical to the Species Richness measure of ecology. We calculate MIE using frequencies (P<sub>i</sub>) of pathogen positivity per FilmArray test and note that the sum of all pathogen frequencies can add up to more than 100% because of co-detections or be less than 100% because of the presence of negative tests. In this regard, MIE differs from PIE because it is not a probability measure.</p><p id="p-46"><a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a> shows that the observed rate of co-detections is a constant fraction of MIE (approximately one quarter as indicated by the linear regression of <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>-Figure supplement 1).This observation suggests that, in the aggregate, respiratory pathogens are appearing in co-infections at a rate that can be predicted by their observed abundance. Data however may be biased by the patient population tested and the type of respiratory disease. The data also does not rule out that there are particular respiratory pathogens that occur more or less often in mixed infections than predicted by their individual percent positivity rates. [<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">74</a>, <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>]. As we improve our ability to remove non-patient test results from the Trend dataset we will be able to characterize specific virus co-detection rates and their significance [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>, <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">78</a>, <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">79</a>, <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>-<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>].</p></div><div id="sec-14" class="subsection"><h3>Trend Value</h3><p id="p-47">As with weather forecasting, there is both a theoretical and a practical interest in predicting the next few weeks or months of the respiratory season [<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>-<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>]. FilmArray Trend contributes to infectious disease forecasting efforts because the data is timely and comprehensive. As the number of sites participating in Trend increases it will be possible to localize the reported infections to smaller geographical regions. At a high enough density of Trend sites, patterns of movement of respiratory pathogens across the US will become visible in a way that has not been easily observed before now.</p><p id="p-48">The Trend RP data is summarized as a stacked graph that shows the percentage contribution of each pathogen to what is currently being detected by FilmArray RP testing (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>, <a href="http://www.syndromictrends.com">www.syndromictrends.com</a>). This analysis does not take into account changes in the rate of testing over a given season, information that should provide additional data regarding disease intensity and severity. In contrast the simple metric, Test Utilization Rate (TUR), describes the non-normalized rate of FilmArray test usage and serves as a surrogate for the level of syndromic disease that health care providers observe (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-49">TUR suffers from two defects. First, it is closely linked to the sales of the FilmArray test and thus is proprietary data that BioFire does not share (Google took a similar position in regard to releasing the search queries used by Google Flu Trends [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>]). Second, TUR is driven by both the demand for testing and the growth in FilmArray product adoption and increasing acceptance and usage by health care providers. A useful step beyond TUR would be a normalized Test Utilization Rate that can adjust for the underlying growth of testing unrelated to the intensity and duration of the respiratory disease season. An increase in a normalized TUR metric may indicate the prevalence of circulating respiratory viruses and the intensity of respiratory disease overall. Likewise, an increase in the normalized metric, concomitant with an increase in negative tests, may indicate the occurrence of an outbreak caused by an emerging pathogen.</p><p id="p-50">Public health agencies (PHAs), which include local and state health departments and the CDC, are specifically exempt under an HIPAA provision that allows clinical laboratories to disclose PHI to the PHA for specified public health purposes [<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>]. The exemption includes follow up studies on reportable infectious diseases. Real-time pathogen specific syndromic surveillance systems such as Trend will allow state PHAs to more rapidly identify, acquire, and test residual samples from potential outbreaks. Conversely, perceived “outbreaks” may actually be coincidental multi-organism seasonal surges, and rapid analysis by Trend-like systems could prevent timely and costly outbreak investigation.</p><p id="p-51">Given the movement in healthcare technology towards greater vertical integration of a hospital’s data, the Bottom-Out approach exemplified by Trend will face more competition from Top-Out approaches (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, see for example GermWatch in Utah, [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>]), because these systems can capture patient information (e.g. age, gender, and patient address) that is critical for more detailed epidemiological analysis. However, combining PHI with the diagnostic test result in the Top-Out approach makes these systems more complex and difficult to implement and may limit participation by health care institutions. Ironically, Bottom-Out data export systems have a role to play in the development of Top-Out systems because Bottom-Out export provides a rapid and efficient means to quality check the data flowing from Top-Out systems. FilmArray Trend data could also be combined with data derived from other automated diagnostic platforms [<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>]. This work might best be accomplished by a third party that is viewed as independent and impartial. For example, in the case of data originating in the US, a federal institution or a private foundation could host a database to which IVD manufacturers would contribute their different syndromic test results. The benefits of a more complete view of circulating pathogens should outweigh the complexities of combining data from different platforms.</p></div><div id="sec-15" class="subsection"><h3>Outlook</h3><p id="p-52">FilmArray Trend is a novel surveillance tool for simultaneously monitoring multiple syndromic diseases that has demonstrated promise in expanding our knowledge of the epidemiology of infectious diseases. Indeed, the close correlation of seasonal respiratory viruses tracked by Trend with reported CDC ILI highlights the major contributory role of multiple respiratory pathogens beyond influenza to ILI. The national and global expansion of Trend will provide a comprehensive tool to study the impact of co-infections, understand the role of previously underappreciated pathogens, and clarify true disease epidemiology. Finally, systems like Trend will be essential for the rapid identification of disease anomalies indicating potential emergent outbreaks, thereby providing an independent tool for public health surveillance.</p></div></div><div class="section" id="sec-16"><h2 class="">Methods</h2><div id="sec-17" class="subsection"><h3>FilmArray System and data output</h3><p id="p-53">The FilmArray System performs nucleic acid purification, reverse transcription, nested multiplex PCR amplification and DNA melt curve analysis for up to 30 targets in 63 minutes ([<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>], <a href="http://www.biofiredx.com">www.biofiredx.com</a>). The FilmArray disposable pouch is prepared by injecting hydration solution and, separately, the patient sample, i.e., for the FilmArray RP test, a nasopharyngeal swab in viral transport media. The pouch is loaded into the FilmArray Instrument and the operator scans the barcode (containing panel type, lot and serial numbers) and then scans or manually enters information into the “Sample” field. Additional free form text can be entered into a “Tag” field. Software on the computer connected to the FilmArray Instrument analyses the amplification products and indicates each pathogen target detected. The results are presented as a PDF report and can be exported to the laboratory’s LIS. Information about the patient (gender, age, date of birth etc.) is not recorded by the instrument unless specifically entered.</p><p id="p-54">As of 2016, more than 4,000 FilmArray instruments have been placed in clinical use worldwide. Approximately 80% of these systems are in the US. The FilmArray RP test is United States Food and Drug Administration (US FDA) designated as Clinical Laboratory Improvement Act (CLIA) moderate status (can be used in moderate to high complexity laboratories).</p></div><div id="sec-18" class="subsection"><h3>HIPAA and the use of de identified patient data for epidemiological tracking</h3><p id="p-55">The Health Insurance Portability and Accountability Act of 1996 (HIPAA) and the Health Information Technology for Economic and Clinical Health Act (HITECH, a part of the American Recovery and Reinvestment Act of 2009) apply to the use of IVD test results. The US Department of Health and Human Services (DHHS) has issued regulations that set standards for the use of patient data (see links at [<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>]). Under HIPAA, laboratories that perform the FilmArray test on patient samples are “Covered Entities.” If laboratories send PHI to a third party (such as the IVD manufacturer) then that party is acting as a business associate (BA) of the Covered Entity and a Business Associate Agreement (BAA) would be required.</p><p id="p-56">The HIPAA Privacy Rule [<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a>] dictates that Covered Entities and, by extension, their BAs, may only disclose PHI with the patient’s written authorization for the purposes of treatment, payment or normal health care operations, and a small number of additional exemptions [<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a>]. Trend software resolves HIPAA concerns by removing all PHI before the test results leave the Covered Entity. This makes it highly unlikely that BioFire, or a malicious intruder into the Trend database, could associate the FilmArray test results with a patient. Because PHI is not exported, BioFire is not a BA and a Data Use Agreement (DUA) between the laboratory and BioFire addresses the export of FilmArray data to the Trend database. Demonstrating that Trend database does not contain PHI has been critical to recruiting institutions to this project.</p></div><div id="sec-19" class="subsection"><h3>Expert determination of de identification</h3><p id="p-57">The HIPAA Privacy Rule allows for release of patient data without prior authorization as long as it has been properly de-identified [<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>]. There are two acceptable routes to de-identification,1) the Safe Harbor approach wherein the data are stripped of an enumerated list of 18 identifiers classified as PHI and there is no indication (i.e., actual knowledge, that the remaining information would lead back to the individual) and 2) the Expert Determination approach wherein a person with experience in relevant statistical and scientific principles evaluates the PHI in conjunction with other reasonably available records, establishes a protocol for de-identification and certifies that the protocol allows only a small risk that PHI may be disclosed to an anticipated recipient.</p><p id="p-58">The goal of the Trend project is to provide a near real-time view of the changes in pathogen prevalence; therefore, it is important to be able to retrieve the date when a FilmArray test is performed. However, retrieving the date conflicts with the Safe Harbor approach because the date of a test is PHI (the year of the test is not PHI but working with just the year defeats the purpose of tracking prevalence through the season). For this reason we followed the Expert Determination approach to manage data export.</p><p id="p-59">The study took into consideration data that are available on participating clinical laboratory FilmArray Instruments, BioFire’s own customer database, the proposed Trend database and publicly available data sources. We analyzed how combinations of this information could be used by an adversary to identify an individual in the dataset thereby disclosing PHI [<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>]. The results of this study (summarized in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) provided recommendations for development and site enrollment criteria for Trend, and for BioFire operating procedures.</p><p id="p-60">In accord with the recommended actions, information in the fields that may be used to distinguish a patient is obfuscated (through truncation or binning) to ensure that a combination of these fields cannot be used to identify a specific patient [<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">97</a>]. For example, the time and date of the test are dynamically binned so that a minimum number of tests of one panel type are included in each bin prior to export. This ensures that a sufficient quantity of test results are uploaded to the database from one site at one time so that there is very low risk that patient identity can be inferred from knowledge of the start time of the test.</p><p id="p-61">If an adversary were to infiltrate the safeguards of the database, and wished to know specific patient test results from a specific location on a given day, no unique records would exist. The combination of deleting the sample identification (ID) field, binning the test date range, and truncating the FilmArray pouch serial number ensures that the remaining information is never unique, which indicates that there is a low risk of misuse of data.</p></div><div id="sec-20" class="subsection"><h3>FilmArray Trend Client Software and Database</h3><p id="p-62">The Trend client software resides on the computer associated with the FilmArray Instrument(s). The computer makes a secure, HTTPS, connection to services hosted by Amazon Web Services. Authenticated data submissions are stored in a database hosted by Amazon Relational Database Service. Both services have been configured to be HIPAA Security Rule compliant [<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>]. Trend client software requires that the computer has Internet access to make secure outbound web requests. The HTTPS protocol is industry standard technology used for secure banking and web applications. The Website software authenticates the Trend data before saving the exported de-identified data to the Trend Database.</p></div><div id="sec-21" class="subsection"><h3>Sites used in the Pilot Trend project</h3><p id="p-63">The clinical laboratory sites participating in the pilot Trend project are shown in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>. The HHS regions are as defined by the CDC [<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>].</p></div><div id="sec-22" class="subsection"><h3>Cleaning the Trend data</h3><p id="p-64">FilmArray Trend should display only valid test results acquired as part of the normal clinical testing of patients. Results from tests where an instrument, software or process control failure occurred have been removed from the data set. In addition, there are situations in which FilmArray customers test non-patient samples: FilmArray clinical customers can purchase Quality Control material from several different commercial sources and can make their own QC material by using cultured organism (from, for example, Zeptometrix, Buffalo, NY) either individually or in combination, or by using patient samples that previously tested positive for one or more organisms on the panel. Although BioFire provides recommendations for performing initial test validations [<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>] that involve combining pathogens in pools of 1-4 per test as well as negative controls, clinical laboratories may use other combinations of analytes. The signature of several of the commercial mixes can be identified simply by observing the set of organisms present. For example MMQC (Maine Molecular Quality Controls, Saco, ME) makes synthetic RNA mixes (M211 v1.1 and M212 v1.1) that explain 98% and 97% of the positive signals for seven and 12 analyte test results, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, All, lines 7 and 12).</p><p id="p-65">Additional QC testing is associated with specific events: a site adopts a new panel, qualifies an instrument returned from repair or, more frequently, qualifies a new lot of FilmArray RP pouches. Since we cannot be sure which tests are performed on control material, we identify events that are associated with test results containing more than three positive calls and flag all tests associated with these events for removal. <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>-Figure Supplement 1 presents an example in the case of the QC tests performed to qualify a new lot of FilmArray RP pouches. In the first six days after a new lot of FilmArray pouches is received, tests with more than three positive organism calls make up 50, 48, 18, 10, 7, and 3% of the results. Thus, for a new lot of pouches, we remove the first three days of testing to minimize the contribution from control runs, thereby deleting 1.4% of all test results from Trend.</p><p id="p-66">Clinical laboratories also perform periodic proficiency testing using test samples provided by either CAP (College of American Pathologists, Northfield IL) or API (American Proficiency Institute, Traverse City, MI). This testing occurs at defined time periods during the year and BioFire is notified. Thus, we have knowledge of the organism mixes being tested as well as the number of such tests. Because only 0.05% of the results in the Trend data set are predicted to be part of CAP or API proficiency testing we have not removed them from the present study.</p><p id="p-67">Because all patient identifiers are stripped before data are exported, there is no automated means to detect repeat tests on the same patient. However, the level of this testing is less than 1% of the total due to test cost and the fact that repeat testing should not be used for test-of-cure. For these reasons, the occurrence of patient duplicate testing would be rare and would not impact the overall Trend dataset. At present we cannot easily remove test results that are part of research protocols. Users at the Trend Pilot sites are instructed to add the Tag: “No Trend” to such tests (at present 0.03% of the Trend dataset) but due to the manual nature of the process we do not know how well this request is observed in practice.</p></div><div id="sec-23" class="subsection"><h3>Test Utilization Rate (TUR) and Pathogen Detection Rate</h3><p id="p-68">The FilmArray RP TUR metric is defined as the non-normalized number of RP patient test results generated each week on average across the Trend sites (computed as a centered three week moving average). To calculate the pathogen detection rate (as displayed in <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a> and on the Trend website) we compute the rate for each organism at each institution as a centered three week moving average. In order to adjust for the capacity differences between sites, a national aggregate is calculated as the unweighted average of individual site rates Only data from sites contributing more than 30 tests per week is included to avoid noise from small numbers of tests. Because the calculation of pathogen detection rate includes results from patients with multiple detections the detection rate for all organisms can, in theory, add up to greater than one. In practice this does not occur.</p></div><div id="sec-24" class="subsection"><h3>Comparison to the CDC Influenza observed rate of detection</h3><p id="p-69">The CDC FluView rate of Flu A and Flu B detections as well as the reported incidence of ILI are taken from the CDC website [<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>]. Only the CDC data from the HHS regions that contained Trend Pilot sites were used for calculating the rate of influenza detections (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-25" class="subsection"><h3>Calculation of co detection rates and related measures</h3><p id="p-70">Pathogen co-detections are defined as FilmArray tests in which two or three organisms are detected. We also calculated two other measures that relate to co-detections: the Circulating Pathogen Number and the Measure of Interspecific Encounter (MIE). Both of these time series measures are calculated for each site and week, a centered five-week moving average is computed and then an unweighted average of all sites is used to create a national aggregate. The five week moving average is used to reduce noise due to small numbers of samples within a week at some sites.</p><p id="p-71">More specifically the Circulating Pathogen Number is simply the count of the unique organisms detected at a site during a one-week period. MIE is calculated from the frequencies of each organism at a site (number of positive test results for an organism divided by the number of FilmArray tests performed at that site). To reduce noise, we only include site data if more than 10 FilmArray tests were performed in that week. If P<sub>1</sub> P<sub>N</sub> are the percentage detection of the N different organisms circulating at a single site over a single week then MIE is defined as:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/09/22/157156/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/09/22/157156/embed/graphic-9.gif"/></noscript></span></span>
</p><p id="p-72">Conceptually, MIE is an attempt to estimate the likelihood that a patient infected with one organism may be infected with another unique organism circulating in the population at a given period in time, resulting in a co-infection.</p></div></div><div class="section" id="sec-26"><h2 class="">Additional information</h2><div class="section ack" id="ack-1"><h3>Acknowledgements</h3><p id="p-73">We acknowledge funding from NIH grant 5U01AI074419 (L.M., K.M.R. and M.A.P) and NIH/NIAID contract HHSN272201600002C (MAP). We thank Chris Thurston and Spencer Rose (BioFire Defense) for building the final Trend public website; Andrew Wallin (BioFire Defense) for reviewing the MIE data analysis; Anna Hoffee (BioFire Diagnostics) for assistance with the figures; Mark Pallansch (CDC), Kirsten St. George (New York State Department of Health) and Allyn Nakashima (Utah Department of Health) for useful discussions; Anne Blaschke and colleagues at BioFire Diagnostics and BioFire Defense for reviewing the manuscript.</p></div><div class="section" id="sec-27"><h3>Author contributions</h3><p id="p-74">L.M., K.M.R. and M.A.P. developed the Trend concept. P.H.G. and C.C.G provided early input on the scope of the project. L.M. designed the DUAs, recruited the sites and provided overall coordination of the project. C.V.C. coordinated the clinical site software installations, L.M. and R.N. designed and supervised the coding of the data export software, database and web interface for the data. L.M. and A.N.F. performed the data analysis and prepared the figures. B.A.M. reviewed the expert determination studies. S.V.S., B.M.A. and J.D.J. reviewed the epidemiological analysis. F.S.N. tested the first release of the data export software. F.S.N., P. H. G., A.L., D.J., V.D, J.D.B., S.S., K.A.S., H.S., R.S., S.J., J.A.D., J.C.W., K.L., F.M., S.L.R, M.A-R., P.D.F., G.A.S., S.J.M., C.C.R., J.F.M. implemented the Trend export software at their respective institutions. C.C.G. and M.A.P. provided periodic review of the project. M.A.P., L. M. and C.C.G. wrote the manuscript. All authors reviewed the final manuscript.</p></div><div class="section" id="sec-28"><h3>Author ORCIDs</h3><p id="p-75">Lindsay Meyers 0000-0003-2994-274X</p><p id="p-76">Christine C. Ginocchio 0000-0002-8200-0324</p><p id="p-77">Aimie N. Faucett 0000-0003-4590-4276</p><p id="p-78">Frederick S. Nolte 0000-0002-4943-9220</p><p id="p-79">Per H. Gesteland 0000-0002-9869-8122</p><p id="p-80">Diane Janowiak 0000-0002-9208-0647</p><p id="p-81">Virginia Donovan 0000-0002-9296-6931</p><p id="p-82">Jennifer Dien Bard 0000-0003-0524-9473</p><p id="p-83">Silvia Spitzer 0000-0002-2642-4771</p><p id="p-84">Kathleen A. Stellrecht 0000-0002-0160-6034</p><p id="p-85">Hossein Salimnia 0000-0002-8069-5185</p><p id="p-86">Rangaraj Selvarangan 0000-0003-3275-6657</p><p id="p-87">Stefan Juretschko 0000-0002-4658-6276</p><p id="p-88">Judy A. Daly 0000-0003-4801-7023</p><p id="p-89">Kristy Lindsey 0000-0002-8172-5964</p><p id="p-90">Franklin Moore 0000-0002-3019-0437</p><p id="p-91">Sharon L. Reed 0000-0001-7615-9366</p><p id="p-92">Paul D. Fey 0000-0003-0939-6884</p><p id="p-93">Camille V. Cook 0000-0001-6071-4898</p><p id="p-94">Jay D. Jones 0000-0001-9091-5316</p><p id="p-95">Samuel V. Scarpino 0000-0001-5716-2770</p><p id="p-96">Benjamin M. Althouse 0000-0002-5464-654X</p><p id="p-97">Kirk M. Ririe 0000-0003-0620-7419</p><p id="p-98">Bradley A. Malin 0000-0003-3040-5175</p><p id="p-99">Mark A. Poritz 0000-0002-5060-0560</p></div><div class="section" id="sec-29"><h3>Competing interests</h3><p id="p-100">L.M., A.N.F., R.K.N., C.V.C, J.D.J., K.M.R., C.C.G. and M.A.P. are present or former employees of bioMérieux, Inc. or its subsidiaries. bioMérieux markets the FilmArray System and Trend. F.S.N., P.H.G., D.J., V.D., A.L., J.D.B., S.S., K.A.S., H.S., R.S., S.J., J.A.D., J.C.W.,</p><p id="p-101">K.L., F.M., S.L.R., M.A-R., P.D.F., G.A.S., S.J.M., S.V.S., B.M.A., are research contractors of BioFire Diagnostics for the development of the FilmArray Trend system. CCR and JFM are members of the Trend Working Group. B.A.M. is a paid consultant of BioFire Diagnostics.</p></div></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marinovic</span>,  <span class="cit-name-given-names">A.B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Quantifying Reporting Timeliness to Improve Outbreak Con}trol</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">21</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Karspeck</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shaman</span></span>, <span class="cit-article-title">Comparison of filtering methods for the modeling and retrospective forecasting of influenza epidemics</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">10</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Wijngaard</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of syndromic surveillance in the Netherlands: its added value and recommendations for implementation</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">16</span>(<span class="cit-issue">9</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CDC</span>. <span class="cit-source">Flu Activity &amp; Surveillance</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="https://www.cdc.gov/flu/weekly/fluactivitysurv.htm">https://www.cdc.gov/flu/weekly/fluactivitysurv.htm</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.5"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CDC</span>. <span class="cit-source">Overview of Influenza Surveillance in the United States</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="http://www.cdc.gov/flu/pdf/weekly/overview.pdf">http://www.cdc.gov/flu/pdf/weekly/overview.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CDC</span>. <span class="cit-source">National Respiratory and Enteric Virus Surveillance System (NREVSS)</span>. [cited April 30, 2017]; Available from: <a href="http://www.cdc.gov/surveillance/nrevss/">http://www.cdc.gov/surveillance/nrevss/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">National Electronic Disease Surveillance System Working</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">National Electronic Disease Surveillance System (NEDSS): a standards-based approach to connect public health and clinical medicine</span>. <abbr class="cit-jnl-abbrev">J Public Health Manag Pract</abbr>, <span class="cit-pub-date">2001</span>. <span class="cit-vol">7</span>(<span class="cit-issue">6</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lamb</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Update on progress in electronic reporting of laboratory results to public health agencies United States, 2014</span>. <abbr class="cit-jnl-abbrev">MMWR Morb Mortal Wkly Rep</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">64</span>(<span class="cit-issue">12</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.9"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">United States Department of Defense</span>. <span class="cit-source">Global Emerging Infections Surveillance and Response System</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="https://www.health.mil/MilitaryHealthTopics/HealthReadiness/ArmedForcesHealthSurveillanceBranch/GlobalEmergingInfectionsSurveillanceandResponse">https://www.health.mil/MilitaryHealthTopics/HealthReadiness/ArmedForcesHealthSurveillanceBranch/GlobalEmergingInfectionsSurveillanceandResponse</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">N.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Event-based internet biosurveillance: relation to epidemiological observation</span>. <abbr class="cit-jnl-abbrev">Emerg Themes Epidemiol</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">9</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A profile of the online dissemination of national influenza surveillance data</span>. <abbr class="cit-jnl-abbrev">BMC Public Health</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">9</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hulth</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rydevik</span></span>, <span class="cit-article-title">GET WELL: an automated surveillance system for gaining new epidemiological knowledge</span>. <abbr class="cit-jnl-abbrev">BMC Public Health</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">11</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.13"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CDC</span>. <span class="cit-source">BioSense</span>. [cited September 20, <span class="cit-pub-date">2017</span>]; Available from: <a href="https://www.cdc.gov/nssp/biosense/index.html">https://www.cdc.gov/nssp/biosense/index.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ginsberg</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detecting influenza epidemics using search engine query data</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">457</span>(<span class="cit-issue">7232</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.15"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GoogleFlu</span>. <span class="cit-pub-date">2015</span>, <span class="cit-source">The Next Chapter for Flu Trends</span>. [cited April 30, 2017]; Available from: <a href="http://googleresearch.blogspot.com/2015/08/thenextchapterforflutrends.html">http://googleresearch.blogspot.com/2015/08/thenextchapterforflutrends.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.16"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">FluNearYou</span>. <span class="cit-source">Flu Near You | HealthMap</span>. [cited April 30, 2017]; Available from: <a href="https://flunearyou.org/">https://flunearyou.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Das</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Monitoring over-the-counter medication sales for early detection of disease outbreaks New York City</span>. <abbr class="cit-jnl-abbrev">MMWR</abbr> Suppl, <span class="cit-pub-date">2005</span>. <span class="cit-vol">54</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patwardhan</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bilkovski</span></span>, <span class="cit-article-title">Comparison: Flu prescription sales data from a retail pharmacy in the US with Google Flu trends and US ILINet (CDC) data as flu activity indicator</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">7</span>(<span class="cit-issue">8</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broniatowski</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.J.</span> <span class="cit-name-surname">Paul</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dredze</span></span>, <span class="cit-article-title">National and local influenza surveillance through Twitter: an analysis of the 2012-2013 influenza epidemic</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">8</span>(<span class="cit-issue">12</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Signorini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.M.</span> <span class="cit-name-surname">Segre</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.M.</span> <span class="cit-name-surname">Polgreen</span></span>, <span class="cit-article-title">The use of Twitter to track levels of disease activity and public concern in the U.S. during the influenza A H1N1 pandemic</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">5</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Generous</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Global disease monitoring and forecasting with Wikipedia</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">10</span>(<span class="cit-issue">11</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McIver</span>,  <span class="cit-name-given-names">D.J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.S.</span> <span class="cit-name-surname">Brownstein</span></span>, <span class="cit-article-title">Wikipedia usage estimates prevalence of influenza-like illness in the United States in near real-time</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">10</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.23"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IHC</span>. <span class="cit-source">GermWatch Intermountain Healthcare Salt Lake City, Utah</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="http://intermountainhealthcare.org/healthresources/germwatch/Pages/home.aspx">http://intermountainhealthcare.org/healthresources/germwatch/Pages/home.aspx</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">P.H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Informing the front line about common respiratory viral epidemics</span>. <abbr class="cit-jnl-abbrev">AMIA Annu Symp Proc</abbr>, <span class="cit-pub-date">2007</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.25"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">NYSDOH</span>. <span class="cit-source">Electronic Clinical Laboratory Reporting System</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="http://www.health.ny.gov/professionals/reportable_diseases/eclrs/">http://www.health.ny.gov/professionals/reportable_diseases/eclrs/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shah</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Messina</span></span>, <span class="cit-article-title">The time-associated impact of the Newborn Influenza Protection Act on infant influenza rates in New York State</span>. <abbr class="cit-jnl-abbrev">J Perinat Med</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">42</span>(<span class="cit-issue">6</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crawford</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Laboratory surge response to pandemic (H1N1) 2009 outbreak, New York City metropolitan area, USA</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">16</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ginocchio</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of multiple test methods for the detection of the novel 2009 influenza A (H1N1) during the New York City outbreak</span>. <abbr class="cit-jnl-abbrev">J Clin Virol</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">45</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Al Tawfiq</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Surveillance for emerging respiratory viruses</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">14</span>(<span class="cit-issue">10</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gautret</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Emerging viral respiratory tract infections-environmental risk factors and transmission</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">14</span>(<span class="cit-issue">11</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hazelton</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detection of influenza A and B with the Alere i Influenza A &amp; B: a novel isothermal nucleic acid amplification assay</span>. <abbr class="cit-jnl-abbrev">Influenza Other Respir Viruses</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">9</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salez</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Prospective and retrospective evaluation of the Cepheid Xpert(R) Flu/RSV XC assay for rapid detection of influenza A, influenza B, and respiratory syncytial virus</span>. <abbr class="cit-jnl-abbrev">Diagn Microbiol Infect Dis</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">81</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nolte</span>,  <span class="cit-name-given-names">F.S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gauld</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.B.</span> <span class="cit-name-surname">Barrett</span></span>, <span class="cit-article-title">Direct Comparison of Alere i and cobas Liat Influenza A and B Tests for Rapid Detection of Influenza Virus Infection</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">54</span>(<span class="cit-issue">11</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buchan</span>,  <span class="cit-name-given-names">B.W.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">N.A.</span> <span class="cit-name-surname">Ledeboer</span></span>, <span class="cit-article-title">Emerging technologies for the clinical microbiology laboratory</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">27</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Morrison</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.W.</span> <span class="cit-name-surname">Tang</span></span>, <span class="cit-article-title">Multiplex polymerase chain reaction tests for detection of pathogens associated with gastroenteritis</span>. <abbr class="cit-jnl-abbrev">Clin Lab Med</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">35</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salez</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of Four Commercial Multiplex Molecular Tests for the Diagnosis of Acute Respiratory Infections</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">10</span>(<span class="cit-issue">6</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poritz</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">FilmArray, an automated nested multiplex PCR system for multipathogen detection: development and application to respiratory tract infection</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">10</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pierce</span>,  <span class="cit-name-given-names">V.M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.L.</span> <span class="cit-name-surname">Hodinka</span></span>, <span class="cit-article-title">Comparison of the GenMark Diagnostics eSensor respiratory viral panel to real-time PCR for detection of respiratory viruses in children</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">50</span>(<span class="cit-issue">11</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.39"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GenMarkDx</span>. <span class="cit-pub-date">2016</span>, <span class="cit-source">The ePlex(®) Respiratory Pathogen Panel (RP)*</span>. [cited April 30, 2017]; Available from: <a href="https://genmarkdx.com/wpcontent/uploads/2016/09/GNMKIMC1046EePlexRPDataSheetUS.pdf">https://genmarkdx.com/wpcontent/uploads/2016/09/GNMKIMC1046EePlexRPDataSheetUS.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gadsby</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of the Luminex Respiratory Virus Panel fast assay with in house real-time PCR for respiratory viral infection diagnosis</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">48</span>(<span class="cit-issue">6</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of Luminex NxTAG Respiratory Pathogen Panel and xTAG Respiratory Viral Panel FAST Version 2 for the Detection of Respiratory Viruses</span>. <abbr class="cit-jnl-abbrev">Ann Lab Med</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">37</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butt</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of three commercial RT PCR systems for the detection of respiratory viruses</span>. <abbr class="cit-jnl-abbrev">J Clin Virol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">61</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Panhuis</span>,  <span class="cit-name-given-names">W.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A systematic review of barriers to data sharing in public health</span>. <abbr class="cit-jnl-abbrev">BMC Public Health</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">14</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">P.A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A.J.</span> <span class="cit-name-surname">Woodward</span></span>, <span class="cit-article-title">Cybersecurity vulnerabilities in medical devices: a complex environment and multifaceted problem</span>. <abbr class="cit-jnl-abbrev">Med Devices (Auckl)</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">8</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benitez</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Malin</span></span>, <span class="cit-article-title">Evaluating re-identification risks with respect to the HIPAA privacy rule</span>. <abbr class="cit-jnl-abbrev">J Am Med Inform Assoc</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">17</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Emam</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A systematic review of re-identification attacks on health data</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">12</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Emam</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rodgers</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Malin</span></span>, <span class="cit-article-title">Anonymising and sharing individual patient data</span>. <abbr class="cit-jnl-abbrev">BMJ</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">350</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sarpatwari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ensuring patient privacy in data sharing for postapproval research</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">371</span>(<span class="cit-issue">17</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.49"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BioFire Diagnostics</span>. <span class="cit-source">FilmArray(®) Panels</span>. [cited April 30, 2017]; Available from: <a href="http://www.biofiredx.com/products/thefilmarraypanels/">http://www.biofiredx.com/products/thefilmarraypanels/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayden</span>,  <span class="cit-name-given-names">R.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of two broadly multiplexed PCR systems for viral detection in clinical respiratory tract specimens from immunocompromised children</span>. <abbr class="cit-jnl-abbrev">J Clin Virol</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">53</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">51.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loeffelholz</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of the FilmArray Respiratory Panel and Prodesse real-time PCR assays for detection of respiratory pathogens</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">49</span>(<span class="cit-issue">12</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pierce</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of the Idaho Technology FilmArray system to real-time PCR for detection of respiratory pathogens in children</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">50</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rand</span>,  <span class="cit-name-given-names">K.H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rampersaud</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.J.</span> <span class="cit-name-surname">Houck</span></span>, <span class="cit-article-title">Comparison of two multiplex methods for detection of respiratory viruses: FilmArray RP and xTAG RVP</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">49</span>(<span class="cit-issue">7</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Implementation of filmarray respiratory viral panel in a core laboratory improves testing turnaround time and patient care</span>. <abbr class="cit-jnl-abbrev">Am J Clin Pathol</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">139</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">55.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Couturier</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of the FilmArray(R) Respiratory Panel for clinical use in a large children's hospital</span>. <abbr class="cit-jnl-abbrev">J Clin Lab Anal</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">27</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Popowitch</span>,  <span class="cit-name-given-names">E.B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.S.</span> <span class="cit-name-surname">O'Neill</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.B.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-article-title">Comparison of the Biofire FilmArray RP, Genmark eSensor RVP, Luminex xTAG RVPv1, and Luminex xTAG RVP fast multiplex assays for detection of respiratory viruses</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">51</span>(<span class="cit-issue">5</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bansal</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Big Data for Infectious Disease Surveillance and Modeling</span>. <abbr class="cit-jnl-abbrev">Journal of Infectious Diseases</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">214</span>(<span class="cit-issue">suppl 4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-157156v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magurran</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-source">Measuring biological diversity</span>. <span class="cit-pub-date">2004</span>, <span class="cit-publ-loc">Malden, Ma</span>: <span class="cit-publ-name">Blackwell Pub. viii</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.59"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">U.S. Dept. of Health and Human Services</span>, <abbr class="cit-jnl-abbrev">Standards for privacy of individually identifiable health information, Final Rule</abbr>. <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malin</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Karp</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.H.</span> <span class="cit-name-surname">Scheuermann</span></span>, <span class="cit-article-title">Technical and policy approaches to balancing patient privacy and data sharing in clinical and translational research</span>. <abbr class="cit-jnl-abbrev">J Investig Med</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">58</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-157156v2.61"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Office for Civil Rights</span>, <span class="cit-chapter-title">Guidance regarding methods for de-identification of protected health information in accordance with the Health Insurance Portability and Accountability Act (HIPAA) Privacy Rule</span>, <span class="cit-source">U.S. Department of Health and Human Services., Editor</span>. <span class="cit-pub-date">2012</span>, <span class="cit-publ-name">U.S. Department of Health and Human Services</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.62"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IBM</span>. <span class="cit-source">Infographic: The Four V's of Big Data</span>. [cited June 8, <span class="cit-pub-date">2017</span>]; Available from: <a href="http://www.ibmbigdatahub.com/infographic/fourvsbigdata">http://www.ibmbigdatahub.com/infographic/fourvsbigdata</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stockmann</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Seasonality of acute otitis media and the role of respiratory viral activity in children</span>. <abbr class="cit-jnl-abbrev">Pediatr Infect Dis J</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">32</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">64.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Radin</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Epidemiology of pathogen-specific respiratory infections among three US populations</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">9</span>(<span class="cit-issue">12</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brittain Long</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Seasonal variations of 15 respiratory agents illustrated by the application of a multiplex polymerase chain reaction assay</span>. <abbr class="cit-jnl-abbrev">Scand J Infect Dis</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">44</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olofsson</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">PCR for detection of respiratory viruses: seasonal variations of virus infections</span>. <abbr class="cit-jnl-abbrev">Expert Rev Anti Infect Ther</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">9</span>(<span class="cit-issue">8</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reed</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Estimating influenza disease burden from population-based surveillance data in the United States</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">10</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruuskanen</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Viral pneumonia</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">377</span>(<span class="cit-issue">9773</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavia</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-article-title">Viral infections of the lower respiratory tract: old viruses, new viruses, and the role of diagnosis</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">52</span> <span class="cit-issue">Suppl 4</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brendish</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Routine molecular point-of-care testing for respiratory viruses in adults presenting to hospital with acute respiratory illness (ResPOC): a pragmatic, open-label, randomised controlled trial</span>. <abbr class="cit-jnl-abbrev">Lancet Respir Med</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">5</span>(<span class="cit-issue">5</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goka</span>,  <span class="cit-name-given-names">E.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Single, dual and multiple respiratory virus infections and risk of hospitalization and mortality</span>. <abbr class="cit-jnl-abbrev">Epidemiol Infect</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">143</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">72.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">E.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Epidemiology of multiple respiratory viruses in childcare attendees</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">207</span>(<span class="cit-issue">6</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rhedin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clinical utility of PCR for common viruses in acute respiratory illness</span>. <abbr class="cit-jnl-abbrev">Pediatrics</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">133</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pinky</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">H.M.</span> <span class="cit-name-surname">Dobrovolny</span></span>, <span class="cit-article-title">Coinfections of the Respiratory Tract: Viral Competition for Resources</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">11</span>(<span class="cit-issue">5</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thorburn</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">RSV outbreak in a paediatric intensive care unit</span>. <abbr class="cit-jnl-abbrev">J Hosp Infect</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">57</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">76.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gooskens</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Morbidity and mortality associated with nosocomial transmission of oseltamivir-resistant influenza A(H1N1) virus</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">301</span>(<span class="cit-issue">10</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Te Wierik</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">An outbreak of severe respiratory tract infection caused by human metapneumovirus in a residential care facility for elderly in Utrecht, the Netherlands</span>, January to March <span class="cit-pub-date">2010</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, 2012. <span class="cit-vol">17</span>(<span class="cit-issue">13</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casalegno</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Rhinoviruses delayed the circulation of the pandemic influenza A (H1N1) 2009 virus in France</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">16</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karppinen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Interference between respiratory syncytial virus and rhinovirus in respiratory tract infections in children</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">22</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hurlbert</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-article-title">The Nonconcept of Species Diversity: A Critique and Alternative Parameters</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-pub-date">1971</span>. <span class="cit-vol">52</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jost</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-article-title">Entropy and diversity</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">113</span>(<span class="cit-issue">2</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-book" id="cit-157156v2.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krebs</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-source">Ecological methodology</span>. <span class="cit-edition">2nd ed</span>. <span class="cit-pub-date">1999</span>, <span class="cit-publ-loc">Menlo Park, Calif</span>.: <span class="cit-publ-name">Benjamin/Cummings</span>. <span class="cit-fpage">xii</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Essa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Mixed viral infections circulating in hospitalized patients with respiratory tract infections in kuwait</span>. <abbr class="cit-jnl-abbrev">Adv Virol</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">2015</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Cross-immunity between strains explains the dynamical pattern of paramyxoviruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">112</span>(<span class="cit-issue">43</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">85.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structural basis for antibody cross-neutralization of respiratory syncytial virus and human metapneumovirus</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">2</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Achten</span>,  <span class="cit-name-given-names">N.B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Interference Between Respiratory Syncytial Virus and Human Rhinovirus Infection in Infancy</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">215</span>(<span class="cit-issue">7</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nsoesie</span>,  <span class="cit-name-given-names">E.O.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A systematic review of studies on forecasting the dynamics of influenza outbreaks</span>. <abbr class="cit-jnl-abbrev">Influenza Other Respir Viruses</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">8</span>(<span class="cit-issue">3</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">88.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chretien</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Influenza forecasting in human populations: a scoping review</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">9</span>(<span class="cit-issue">4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">89.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-157156v2.89"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biggerstaff</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Results from the centers for disease control and prevention's predict the 2013 2014 Influenza Season Challenge</span>. <abbr class="cit-jnl-abbrev">BMC Infect Dis</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">16</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.90"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moran</span>,  <span class="cit-name-given-names">K.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Epidemic Forecasting is Messier Than Weather Forecasting: The Role of Human Behavior and Internet Data Streams in Epidemic Forecast</span>. <abbr class="cit-jnl-abbrev">Journal of Infectious Diseases</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">214</span>(<span class="cit-issue">suppl 4</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.91"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">HHS</span>. <span class="cit-pub-date">2003</span>, <span class="cit-source">Disclosures For Public Health Activities</span>. [cited April 30, 2017]; Available from: <a href="https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/special/publichealth/publichealth.pdf">https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/special/publichealth/publichealth.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-157156v2.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shekalaghe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clinical performance of an automated reader in interpreting malaria rapid diagnostic tests in Tanzania</span>. <abbr class="cit-jnl-abbrev">Malar J</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">12</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.93"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">InSTEDD</span>. <span class="cit-source">Connectivity Solutions for TB Diagnostics</span>. [cited April 30, 2017]; Available from: <a href="http://instedd.org/project/connectivitysolutionsfortbdiagnostics/">http://instedd.org/project/connectivitysolutionsfortbdiagnostics/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.94"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">HHS</span>. <span class="cit-source">Health Information Privacy</span>. [cited; Available from: <a href="http://www.hhs.gov/ocr/privacy/index.html">http://www.hhs.gov/ocr/privacy/index.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.95"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">HHS</span>. <span class="cit-source">Summary of the HIPAA Privacy Rule</span>. [cited; Available from: <a href="https://www.hhs.gov/sites/default/files/privacysummary.pdf">https://www.hhs.gov/sites/default/files/privacysummary.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.96"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">HHS</span>. <span class="cit-source">Special Topics in Health Information Privacy</span>. [cited; Available from: <a href="http://www.hhs.gov/ocr/privacy/hipaa/understanding/special/index.html">http://www.hhs.gov/ocr/privacy/hipaa/understanding/special/index.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.97"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Office for Civil Rights</span>. <span class="cit-pub-date">2012</span>, <span class="cit-source">Guidance Regarding Methods for De-identification of Protected Health Information in Accordance with the Health Insurance Portability and Accountability Act (HIPAA) Privacy Rule</span>. [cited April 30, 2017]; Available from: <a href="https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/coveredentities/Deidentification/hhs_deid_guidance.pdf">https://www.hhs.gov/sites/default/files/ocr/privacy/hipaa/understanding/coveredentities/Deidentification/hhs_deid_guidance.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-book" id="cit-157156v2.98"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Federal Committee on Statistical Methodology</span>, <span class="cit-source">Statistical and Science Policy, Statistical Policy Working Paper 22, Report on Statistical Disclosure Limitation Methodology</span>. <span class="cit-pub-date">2005</span>, <span class="cit-publ-name">Office of Information and Regulatory Affairs, Office of Management and Budget</span>,.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.99"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Amazon</span>. <span class="cit-pub-date">2017</span>, <span class="cit-source">Architecting for HIPAA Security and Compliance on Amazon Web Services</span>. [cited April 30, 2017]; Available from: <a href="https://d0.awsstatic.com/whitepapers/compliance/AWS_HIPAA_Compliance_Whitepaper.pdf">https://d0.awsstatic.com/whitepapers/compliance/AWS_HIPAA_Compliance_Whitepaper.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.100"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BioFire Diagnostics</span>. <span class="cit-source">Guidelines for Laboratory Verification of Performance of the FilmArray(r) Respiratory Panel (RP)</span>. [cited April 30, <span class="cit-pub-date">2017</span>]; Available from: <a href="http://www.biofiredx.com/wpcontent/uploads/2016/03/TNFLM1PRT006007GuidelinesforLaboratoryVerificationofPerformanceoftheFilmArrayRespiratoryPanel.pdf">http://www.biofiredx.com/wpcontent/uploads/2016/03/TNFLM1PRT006007GuidelinesforLaboratoryVerificationofPerformanceoftheFilmArrayRespiratoryPanel.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-web" id="cit-157156v2.101"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CDC</span>. <span class="cit-source">Nation and Regional Level Outpatient Illness and Viral Surveillance</span>. [cited June 8, <span class="cit-pub-date">2017</span>]; Available from: <a href="https://gis.cdc.gov/grasp/fluview/fluportaldashboard.html">https://gis.cdc.gov/grasp/fluview/fluportaldashboard.html</a>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/157156v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F157156v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/142992v2" title="Deep transcriptome annotation suggests that small and large proteins encoded in the same genes often cooperate" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/165050v2" title="Sensitive and robust assessment of ChIP-seq read distribution using a strand-shift profile" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 22, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/157156v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/157156v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/157156v2" class="active" data-icon-position="" data-hide-link-title="0">Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/58365" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BSp8FuiRDyaEYcmLEXTsNv2OUaI0XfPviHndVCCmKWE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="58365" id="node58365--2" data-pisa="biorxiv;157156v2" data-pisa-master="biorxiv;157156" data-seqnum="58365" data-apath="/biorxiv/early/2017/09/22/157156.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Lindsay</span> <span class="nlm-surname">Meyers</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Ginocchio</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Aimie N.</span> <span class="nlm-surname">Faucett</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Frederick S.</span> <span class="nlm-surname">Nolte</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Per H.</span> <span class="nlm-surname">Gesteland</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Amy</span> <span class="nlm-surname">Leber</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Diane</span> <span class="nlm-surname">Janowiak</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Virginia</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Dien Bard</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Spitzer</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Stellrecht</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hossein</span> <span class="nlm-surname">Salimnia</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rangaraj</span> <span class="nlm-surname">Selvarangan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Juretschko</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Judy A.</span> <span class="nlm-surname">Daly</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jeremy C.</span> <span class="nlm-surname">Wallentine</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Kristy</span> <span class="nlm-surname">Lindsey</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Franklin</span> <span class="nlm-surname">Moore</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Sharon L.</span> <span class="nlm-surname">Reed</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Aguero-Rosenfeld</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Paul D.</span> <span class="nlm-surname">Fey</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Gregory A.</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Steve J.</span> <span class="nlm-surname">Melnick</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Robinson</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Jennifer F.</span> <span class="nlm-surname">Meredith</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Camille V.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author" data-delta="26"><span class="nlm-given-names">Robert K.</span> <span class="nlm-surname">Nelson</span></span>, <span class="highwire-citation-author" data-delta="27"><span class="nlm-given-names">Jay D.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author" data-delta="28"><span class="nlm-given-names">Samuel V.</span> <span class="nlm-surname">Scarpino</span></span>, <span class="highwire-citation-author" data-delta="29"><span class="nlm-given-names">Benjamin M.</span> <span class="nlm-surname">Althouse</span></span>, <span class="highwire-citation-author" data-delta="30"><span class="nlm-given-names">Kirk M.</span> <span class="nlm-surname">Ririe</span></span>, <span class="highwire-citation-author" data-delta="31"><span class="nlm-given-names">Bradley A.</span> <span class="nlm-surname">Malin</span></span>, <span class="highwire-citation-author" data-delta="32"><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Poritz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">157156; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/157156 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/157156v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;title=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;title=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;text=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;title=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;t=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;title=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;title=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="58365" id="node58365--3" data-pisa="biorxiv;157156v2" data-pisa-master="biorxiv;157156" data-seqnum="58365" data-apath="/biorxiv/early/2017/09/22/157156.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Lindsay</span> <span class="nlm-surname">Meyers</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Ginocchio</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Aimie N.</span> <span class="nlm-surname">Faucett</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Frederick S.</span> <span class="nlm-surname">Nolte</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Per H.</span> <span class="nlm-surname">Gesteland</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Amy</span> <span class="nlm-surname">Leber</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Diane</span> <span class="nlm-surname">Janowiak</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Virginia</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Dien Bard</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Spitzer</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Stellrecht</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hossein</span> <span class="nlm-surname">Salimnia</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rangaraj</span> <span class="nlm-surname">Selvarangan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Juretschko</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Judy A.</span> <span class="nlm-surname">Daly</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jeremy C.</span> <span class="nlm-surname">Wallentine</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Kristy</span> <span class="nlm-surname">Lindsey</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Franklin</span> <span class="nlm-surname">Moore</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Sharon L.</span> <span class="nlm-surname">Reed</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Aguero-Rosenfeld</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Paul D.</span> <span class="nlm-surname">Fey</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Gregory A.</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Steve J.</span> <span class="nlm-surname">Melnick</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Christine C.</span> <span class="nlm-surname">Robinson</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Jennifer F.</span> <span class="nlm-surname">Meredith</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Camille V.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author" data-delta="26"><span class="nlm-given-names">Robert K.</span> <span class="nlm-surname">Nelson</span></span>, <span class="highwire-citation-author" data-delta="27"><span class="nlm-given-names">Jay D.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author" data-delta="28"><span class="nlm-given-names">Samuel V.</span> <span class="nlm-surname">Scarpino</span></span>, <span class="highwire-citation-author" data-delta="29"><span class="nlm-given-names">Benjamin M.</span> <span class="nlm-surname">Althouse</span></span>, <span class="highwire-citation-author" data-delta="30"><span class="nlm-given-names">Kirk M.</span> <span class="nlm-surname">Ririe</span></span>, <span class="highwire-citation-author" data-delta="31"><span class="nlm-given-names">Bradley A.</span> <span class="nlm-surname">Malin</span></span>, <span class="highwire-citation-author" data-delta="32"><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Poritz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">157156; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/157156 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Meyers%2C+Lindsay&amp;rft.au=Ginocchio%2C+Christine+C.&amp;rft.au=Faucett%2C+Aimie+N.&amp;rft.au=Nolte%2C+Frederick+S.&amp;rft.au=Gesteland%2C+Per+H.&amp;rft.au=Leber%2C+Amy&amp;rft.au=Janowiak%2C+Diane&amp;rft.au=Donovan%2C+Virginia&amp;rft.au=Dien+Bard%2C+Jennifer&amp;rft.au=Spitzer%2C+Silvia&amp;rft.au=Stellrecht%2C+Kathleen+A.&amp;rft.au=Salimnia%2C+Hossein&amp;rft.au=Selvarangan%2C+Rangaraj&amp;rft.au=Juretschko%2C+Stefan&amp;rft.au=Daly%2C+Judy+A.&amp;rft.au=Wallentine%2C+Jeremy+C.&amp;rft.au=Lindsey%2C+Kristy&amp;rft.au=Moore%2C+Franklin&amp;rft.au=Reed%2C+Sharon+L.&amp;rft.au=Aguero-Rosenfeld%2C+Maria&amp;rft.au=Fey%2C+Paul+D.&amp;rft.au=Storch%2C+Gregory+A.&amp;rft.au=Melnick%2C+Steve+J.&amp;rft.au=Robinson%2C+Christine+C.&amp;rft.au=Meredith%2C+Jennifer+F.&amp;rft.au=Cook%2C+Camille+V.&amp;rft.au=Nelson%2C+Robert+K.&amp;rft.au=Jones%2C+Jay+D.&amp;rft.au=Scarpino%2C+Samuel+V.&amp;rft.au=Althouse%2C+Benjamin+M.&amp;rft.au=Ririe%2C+Kirk+M.&amp;rft.au=Malin%2C+Bradley+A.&amp;rft.au=Poritz%2C+Mark+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/58365/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/58365/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/58365/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/58365/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/58365/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/58365/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/58365/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/58365/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/58365/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/58365/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/58365/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/58365/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;count=horizontal&amp;via=&amp;text=Automated%20collection%20of%20pathogen-specific%20diagnostic%20data%20for%20real-time%20syndromic%20epidemiological%20studies&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/157156v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/157156v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/157156v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/epidemiology" class="highlight" data-icon-position="" data-hide-link-title="0">Epidemiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
