<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/08/2020.04.02.022384.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.02.022384v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="One-step RNA extraction for RT-qPCR detection of 2019-nCoV"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.02.022384"/>
  <meta name="DC.Date" content="2020-04-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months[1][1]. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey. ### Competing Interest Statement The authors have declared no competing interest. [1]: #ref-1"/>
  <meta name="DC.Contributor" content="Monica Sentmanat"/>
  <meta name="DC.Contributor" content="Evguenia Kouranova"/>
  <meta name="DC.Contributor" content="Xiaoxia Cui"/>
  <meta name="article:published_time" content="2020-04-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="One-step RNA extraction for RT-qPCR detection of 2019-nCoV"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months1. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.02.022384v2"/>
  <meta name="citation_id" content="2020.04.02.022384v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.022384v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.022384v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.022384v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.02.022384"/>
  <meta name="citation_num_pages" content="8"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.02.022384"/>
  <meta name="citation_author" content="Monica Sentmanat"/>
  <meta name="citation_author_institution" content="Genome Engineering &amp;; iPSC Center (GEiC), Department of Genetics, Washington University in St. Louis School of Medicine"/>
  <meta name="citation_author" content="Evguenia Kouranova"/>
  <meta name="citation_author_institution" content="Genome Engineering &amp;; iPSC Center (GEiC), Department of Genetics, Washington University in St. Louis School of Medicine"/>
  <meta name="citation_author" content="Xiaoxia Cui"/>
  <meta name="citation_author_institution" content="Genome Engineering &amp;; iPSC Center (GEiC), Department of Genetics, Washington University in St. Louis School of Medicine"/>
  <meta name="citation_author_email" content="x.cui@wustl.edu"/>
  <meta name="citation_reference" content="Chan, J. F., Yuan, S., et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. The Lancet 395, 514–523 (2020)."/>
  <meta name="citation_reference" content="Roth, Emily. Preparation of viral transport medium.Center for Disease Control and Prevention. SOP#: DSR-052-01 (2020)."/>
  <meta name="citation_reference" content="Kouranova, E. et al. CRISPRs for Optimal Targeting: Delivery of CRISPR Components as DNA, RNA, and Protein into Cultured Cells and Single-Cell Embryos. Human Gene Therapy 27, 464–475 (2016)."/>
  <meta name="citation_reference" content="Pfaender, S., Brinkmann, J., et al. Mechanisms of Methods for Hepatitis C Virus Inactivation. Applied and Environmental Microbiology 81, 1616–1621 (2015)."/>
  <meta name="citation_reference" content="Van Kampen, J.J.A., Tinte, A., et al. Ebola Virus Inactivation by Detergents Is Annulled in Serum. Journal of Infection Diseases. 216, 859–66 (2017)."/>
  <meta name="citation_reference" content="CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel. Centers for Disease Control and Prevention. CDC-006-00019, Revision: 03 (2020)."/>
  <meta name="citation_reference" content="Bruce, E.A., Tighe, S., et al., RT-qPCR Detection Of SARS-CoV-2 RNA From Patient Nasopharyngeal Swab Using QIAGEN RNeasy Kits or Directly Via Omission of An RNA Extraction Step. BioRxiv doi: https://doi.org/10.1101/2020.03.20.001008 (2020)."/>
  <meta name="twitter:title" content="One-step RNA extraction for RT-qPCR detection of 2019-nCoV"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months[1][1]. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey. ### Competing Interest Statement The authors have declared no competing interest. [1]: #ref-1"/>
  <meta name="og-title" property="og:title" content="One-step RNA extraction for RT-qPCR detection of 2019-nCoV"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.022384v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months[1][1]. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey. ### Competing Interest Statement The authors have declared no competing interest. [1]: #ref-1"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>One-step RNA extraction for RT-qPCR detection of 2019-nCoV | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1235748 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-2">The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">RESULTS</h2>
    <p id="p-5">In the CDC protocol, specimens are typically placed in 3 mL of viral transport media consisting of a buffered salt solution with fetal bovine serum and an antimicrobial solution
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. Viral particles in the samples remain infectious until lysed during RNA extraction. The RNA is then prepared by using a column based RNA extraction kit. We showed previously that RNA can be extracted by one-step lysis in QuickExtract DNA Extraction Solution (QE buffer) and directly used in RT-qPCR
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. The QE buffer contains detergents and proteinase K, both of which could inactivate viral particles. Previous work on hepatitis C and ebola virus has shown that detergent alone is sufficient to reduce infectious titer in the absence of serum (e.g. fetal bovine serum) and is even more effective in combination with proteinase K which degrades core viral proteins accessible through lipid viral envelop dissolution by detergent
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>.
    </p>
    <p id="p-6">We placed swabs directly in QE buffer. Prior to heat extraction, samples were vortexed and divided into two. One was used to create a positive control counterpart with 2019-nCoV plasmid DNA template added at 100 copies/ul. Although 2019-nCoV is an RNA virus, this positive DNA control ensures the assay is working while maximizing saftey and the need for Biosaftey Level 2 (BSL2) precautions, as is required for handling 2019-nCoV RNA material. An RNA extraction control using HCT-116 human colorectal cancer cells at 10 cells/ul was included. Samples were then heated at 65°C for 15 min followed by 98°C for 2 min to inactivate proteinase K and then directly used for the test using a single probe and primer set (N1) as well as the set for RNaseP.</p>
    <p id="p-7">All positive control counterpart samples had a Ct value for N1 between 29-32, while all experimental samples had no detectable amplification from the N1 primer/probe set (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We detected RNaseP reference signals in both the OP and human cell line extraction controls, which confirms that the extractions were successful. Although an equal volume of lysis extract was used to generate the positive control counterpart of OP experimental samples, RNaseP reference signals were 2 cycles greater for positive controls, suggesting that the swab containing portion retained more cells. But it’s important to point out that the raw Ct values do not change the interpretation of results for what is intended to be a qualitative yes/no assay - that the positive and extraction controls performed within the range (&lt;;40) to determine the presence or absence of 2019-nCoV N gene in experimental samples
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="An OP specimen was lysed in QE buffer alongside HCT116 cells as controls and amplified by using one-step RT-qPCR. Ct values for OP specimen, HCT-116 extraction control, and no template control containing no cells as input (NTC) shown. A Ct value of 0 indicates no signal was detected. Each sample has a positive control with 100 copies/ul plasmid DNA template spiked-in to confirm N1 probe and primer set performance. Error bars are the standard deviation for three technical replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985041597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;An OP specimen was lysed in QE buffer alongside HCT116 cells as controls and amplified by using one-step RT-qPCR. Ct values for OP specimen, HCT-116 extraction control, and no template control containing no cells as input (NTC) shown. A Ct value of 0 indicates no signal was detected. Each sample has a positive control with 100 copies/ul plasmid DNA template spiked-in to confirm N1 probe and primer set performance. Error bars are the standard deviation for three technical replicates.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F1.medium.gif" width="440" height="237"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F1.medium.gif" width="440" height="237"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-8" class="first-child">An OP specimen was lysed in QE buffer alongside HCT116 cells as controls and amplified by using one-step RT-qPCR. Ct values for OP specimen, HCT-116 extraction control, and no template control containing no cells as input (NTC) shown. A Ct value of 0 indicates no signal was detected. Each sample has a positive control with 100 copies/ul plasmid DNA template spiked-in to confirm N1 probe and primer set performance. Error bars are the standard deviation for three technical replicates.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-9">To determine if NP and OP specimens remain stable in QE buffer until they can be transported from the site of collection to the lab, we stored the samples with positive plasmid DNA template control at 100 copies/ul at room temperature, 4°C, or −20°C for 24 hours. Samples were heat extracted after 24 hours and used for the RT-qPCR test with probe sets N1 and N2. The sample without positive control added was processed for extraction immediately after the collection. All positive control samples had a Ct value between 33-34 for N1 and N2 across conditions. The Ct values for reference RNaseP were within the range of 26-29 when the collection swab was present during heat extraction and a Ct range of 29-31 when the swab was not present. This is consistent with epithelial cells being caught in the polyester fibers during collection. The spiked plasmid DNA does not seem to be retained on the swab. The Ct values of samples stored at different temperatures are not significantly different from those samples processed without storage (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242824/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1242824/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1242824" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1</span>
      <p id="p-10" class="first-child">Comparison of QE lysis performance across specimen storage temperatures after 24 hours. Ct values for oropharyngeal (OP) and nasopharyngeal (NP) specimens that were immediately heat extracted (no storage) or kept at room temperature (RT), 4°C or −20°C for 24 hours. The presence of the collection swab during heat extraction is also shown (+/− swab). The positive control is plasmid DNA template spiked at 100 copies/ul into specimen sample. A value of NS indicates no signal was detected.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">We sought to compare yields from direct input of QE buffer-lysed sample and column purified RNA. An OP specimen was placed in 200 ul of QE buffer and the collection swab removed. We removed 20 ul for our experimental input before adding 100 copies/ul of positive control DNA template to the remaining buffer. We took 120 ul for column purification using the Qiagen RNeasy Mini Kit with a final elution volume of 30 ul. Although this is not one of the CDC recommended RNA extraction kits, it has been previously shown to perform as well as the recommended kits for positive control samples
     <sup>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. The remaining 60 ul of QE sample positive control counterpart and 20 ul of experimental QE sample were heat extracted. Each RT-qPCR reaction had 2 ul of input assayed. We found QE processed and column purified material had comparable Ct values despite 4-fold more material processed for column purification (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Side-by-side comparison of OP specimen from direct lysis in QE buffer vs. Qiagen RNeasy column purification. Ct values for probe sets N1, N2 and RNaseP shown using undilted and 1:10 dilutions of positive control counterparts at 100 copies/ul of plasmid DNA template and experimental OP specimen. A Ct value of 0 indicates no signal was detected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985041597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Side-by-side comparison of OP specimen from direct lysis in QE buffer vs. Qiagen RNeasy column purification. Ct values for probe sets N1, N2 and RNaseP shown using undilted and 1:10 dilutions of positive control counterparts at 100 copies/ul of plasmid DNA template and experimental OP specimen. A Ct value of 0 indicates no signal was detected.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F2.medium.gif" width="440" height="243"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F2.medium.gif" width="440" height="243"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.02.022384/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-12" class="first-child">Side-by-side comparison of OP specimen from direct lysis in QE buffer vs. Qiagen RNeasy column purification. Ct values for probe sets N1, N2 and RNaseP shown using undilted and 1:10 dilutions of positive control counterparts at 100 copies/ul of plasmid DNA template and experimental OP specimen. A Ct value of 0 indicates no signal was detected.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">DISCUSSION</h2>
    <p id="p-13">We presented here an improvement to the standard 2019-nCoV RT-qPCR test. We modified the RNA extraction step by using a one-step lysis buffer, bypassing the use of column purification. Our indicators for adequate assay performance are consistent with CDC assay interpretation guidelines, the detection of a Ct level of &lt;;40 for 2019-nCoV positive control counterparts and for RNaseP reference signal for positive controls and experimental samples
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>. The direct lysis takes less than 20 min to process samples ready for RT-qPCR and can be easily scaled up to a 96-well format and obtain higher throughput. The cost of the lysis buffer is much lower than that of a column purification kit. The lysis only requires a regular PCR machine and does not need a centrifuge or a manifold, as column purification requires.
    </p>
    <p id="p-14">In addition to the QE buffer used here, there are products, such as Lucigen’s QE for RNA Extraction buffer and SingleShot Cell Lysis Kit from Bio-Rad, specifically formulated for RNA extraction, may enhance sensitivity by suppressing RNase activity and improve performance. When necessary, these buffers can be easily produced in large quantities. This way, it eliminates the potential shortage of RNA extraction kits.</p>
    <p id="p-15">We also demonstrated that the samples are stable in the QE buffer for at least 24 hours even when they are stored at ambient temperatures, allowing time flexibility between sample collection and processing. In the meantime, detergents and proteinase K in the buffer likely inactivate the viral particles in positive samples before extraction, making the samples safer to handle during transport and in the testing lab.</p>
    <p id="p-16">In the parallel comparison of QE buffer lysed samples versus column purified samples, the Ct values for positive controls and RNaseP were within 1 cycle or less, even though the samples were concentrated 4x (from 120 ul to 30 ul elute). It indicates that there is significant loss of material by column purification and the benefit of concentrating the samples is evened out. Additionally, our method starts with only 200 ul of lysis buffer, and 2 ul of direct lysis is 1/100 of the total sample. If we use a smaller swab, which is available, the volume can be lowered further. With the standard method, the sample transport buffer is 3 mL and only 120 ul to 140 ul, depending on the extraction kit, is taken for purification, essentially a 1:15 dilution that can translate into up to a difference of 4 Ct’s.</p>
    <p id="p-17">Given the increasing need for more tests to be done quicker, our data presents a feasible option for large scale testing sites. Bypassing the need of column purification and lowering the volume of sample collection buffer together simplify high-throughput process development, shortens turnaround, reduces cost and may improve sensitivity.</p>
    <p id="p-18">A limitation of this work is that DNA template was used as a positive control for the viral RNA. Even though RNaseP demonstrates comparable stability to that of the DNA control, viral RNA might be more susceptible to nucleases in the NP and OP samples. Our next step is to use an in vitro transcribed RNA as spike-in, fully realizing that naked transcripts are likely more easily degraded by the nucleases than viral RNA with the protection of protein coating. The ultimate test is using positive patient samples containing viral particles.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">METHODS</h2>
    <div id="sec-4" class="subsection">
     <h3>Specimen Collection</h3>
     <p id="p-19">Collection of a nasopharyngeal specimen (NP) is the recommended method for testing patients presenting COVID-19 symptoms using the CDC diagnostic RT-qPCR test, with oropharyngeal (OP) being an acceptable alternative according to CDC guidelines. To determine if direct lysis of comparable specimen samples could be used for the RT-qPCR test, we self-collected NP and OP samples using sterile polyester flock swabs (PurFlock Ultra flocked collection swabs by Puritan Diagnostics LLC). The swabs were each placed in 1.5 mL Eppendorf tube containing 200 ul of QuickExtract DNA Extraction Solution (QE buffer, Lucigen LLC, Madison, WI). Each tube was vortexed and stored until extraction.</p>
     <p id="p-20">Extraction control HCT-116 cells were counted and pipetted directly to the QE buffer, and no swab was used. A positive control was made for each sample by transferring 100 ul of sample to a new 1.5 mL Eppendorf tube and adding 10000 copies of COVID19 plasmid DNA template control (IDT Cat. No 10006625), prior to extraction incubation. The final concentration of the positive control is 100 copies/ul. No cell input negative control samples were also used for 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and Table 2. In 
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, the no cell input negative control also had a plasmid DNA template positive control counterpart.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>RNA Extraction</h3>
     <p id="p-21">Samples were incubated in the QE buffer at 65°C for 15 minutes, followed by 98°C for 2 min. Column purification was performed using Qiagen RNeasy Mini Kit (Cat. No. 74104) from 120 ul of each sample, using an elution volume of 30 ul.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>RT-qPCR</h3>
     <p id="p-22">Probe and primer sets were obtained from the research COVID19 RT-qPCR kits 2019-nCoV RUO (IDT Cat. No. 10006713). A one-step reaction mix was prepared using Reliance One-Step Multiplex Supermix (BioRad Cat. No. 12010220) with 2 ul of sample, 1.5 ul of the probe/prime set mix for each reaction, 5 ul of 4x reaction mix, and 11.5 ul of molecular biology grade water to a final volume of 20 ul. The RT-qPCR was run on a QuantStudio™ 3 real-time PCR machine (ThermoFisher).</p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">RECOMMENDED PROTOCOL</h2>
    <div class="section" id="sec-8">
     <h3>Assemble collection tubes</h3>
     <p id="p-23">Aliquot 200 ul of QE buffer to each collection tube.</p>
    </div>
    <div class="section" id="sec-9">
     <h3>Sample collection</h3>
     <ol class="list-ord " id="list-1">
      <li id="list-item-1">
       <p id="p-24">Take an NP or OP sample and break the handle, leaving the swab in the buffer.</p>
      </li>
      <li id="list-item-2">
       <p id="p-25">Vortex the tube and store at RT or 4C until extraction.</p>
      </li>
     </ol>
    </div>
    <div class="section" id="sec-10">
     <h3>Sample processing</h3>
     <ol class="list-ord " id="list-2">
      <li id="list-item-3">
       <p id="p-26">Vortex the sample again, take 50 ul of sample each to two PCR tubes marked with “+” and “-”.</p>
      </li>
      <li id="list-item-4">
       <p id="p-27">Add 5000 copies of positive RNA control of N transcript to the tube marked with “+”.</p>
      </li>
      <li id="list-item-5">
       <p id="p-28">(Extraction) Place both PCR tubes on a PCR machine and run the following program: 65°C, 15 min, 98°C, 2 min.</p>
      </li>
      <li id="list-item-6">
       <p id="p-29">Take 2 ul each from “+” and “-” samples and assemble the one-step RT-qPCR reactions, containing 1.5 ul of each primer/probe set, 5 ul of 4x reaction mix and 11.5 ul of nuclease-free water.</p>
      </li>
      <li id="list-item-7">
       <p id="p-30">Run the reactions on a QuantStudio™ 3 real-time PCR machine (ThermoFisher) using the following program:</p>
       <p id="p-31">50°C for 10 min</p>
       <p id="p-32">95°C, 10 min</p>
       <p id="p-33">40 cycles of 95°C, 3 sec and 55°C, 30 sec</p>
      </li>
     </ol>
    </div>
    <div class="section" id="sec-11">
     <h3>Data analysis</h3>
     <p id="p-34">Data analysis was performed using QuantStudio
      <sup>®</sup> Design and Analysis Desktop Software.
     </p>
    </div>
    <div class="section" id="sec-12">
     <h3>Total time</h3>
     <p id="p-35">Just under two hours (20 min for lysis, about 1hr 20 min for RT-qPCR, plus data analysis).</p>
    </div>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-4">Positive control description has been corrected. A DNA, not RNA template was used for positive control counterparts described in the Results section.</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">BIBLIOGRAPHY</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">J. F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">514</span>–
         <span class="cit-lpage">523</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Roth</span>, 
          <span class="cit-name-given-names">Emily</span>
         </span>. 
         <span class="cit-article-title">Preparation of viral transport medium.Center for Disease Control and Prevention</span>. 
         <abbr class="cit-jnl-abbrev">SOP#: DSR-052-01</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.3" data-doi="10.1089/hum.2016.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kouranova</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">CRISPRs for Optimal Targeting: Delivery of CRISPR Components as DNA, RNA, and Protein into Cultured Cells and Single-Cell Embryos</span>. 
         <abbr class="cit-jnl-abbrev">Human Gene Therapy</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">464</span>–
         <span class="cit-lpage">475</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DHuman%2BGene%2BTherapy%26rft.volume%253D27%26rft.spage%253D464%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fhum.2016.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/hum.2016.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.4" data-doi="10.1128/AEM.03580-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pfaender</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brinkmann</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mechanisms of Methods for Hepatitis C Virus Inactivation</span>. 
         <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>
         <span class="cit-vol">81</span>, 
         <span class="cit-fpage">1616</span>–
         <span class="cit-lpage">1621</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.03580-14%26rft_id%253Dinfo%253Apmid%252F25527548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjgxLzUvMTYxNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA4LzIwMjAuMDQuMDIuMDIyMzg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Van Kampen</span>, 
          <span class="cit-name-given-names">J.J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tinte</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Ebola Virus Inactivation by Detergents Is Annulled in Serum</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Infection Diseases</abbr>. 
         <span class="cit-vol">216</span>, 
         <span class="cit-fpage">859</span>–
         <span class="cit-lpage">66</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInfection%2BDiseases%26rft.volume%253D216%26rft.spage%253D859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.02.022384v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">CDC 2019-Novel Coronavirus (2019-nCoV)</span>
         <span class="cit-article-title">Real-Time RT-PCR Diagnostic Panel</span>. 
         <abbr class="cit-jnl-abbrev">Centers for Disease Control and Prevention. CDC-006-00019, Revision: 03</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.02.022384v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>, 
          <span class="cit-name-given-names">E.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tighe</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">RT-qPCR Detection Of SARS-CoV-2 RNA From Patient Nasopharyngeal Swab Using QIAGEN RNeasy Kits or Directly Via Omission of An RNA Extraction Step</span>. 
         <span class="cit-source">BioRxiv</span> doi: 
         <a href="https://doi.org/10.1101/2020.03.20.001008">https://doi.org/10.1101/2020.03.20.001008</a> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
