<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/17/2020.03.15.992438.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.15.992438v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection" />
<meta name="DC.Identifier" content="10.1101/2020.03.15.992438" />
<meta name="DC.Date" content="2020-03-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that have been found in more than 200 diverse organisms. Although it is still not fully established if RNA viruses could generate miRNAs that would target their own genes or alter the host gene expression, there are examples of miRNAs functioning as an antiviral defense mechanism. In the case of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there are several mechanisms that would make miRNAs impact the virus, like interfering with replication, translation and even modulating the host expression. In this study, we performed a machine learning based miRNA prediction analysis for the SARS-CoV-2 genome to identify miRNA-like hairpins and searched for potential miRNA – based interactions between the viral miRNAs and human genes and human miRNAs and viral genes. Our PANTHER gene function analysis results indicate that viral derived miRNA candidates could target various human genes involved in crucial cellular processes including transcription. For instance, a transcriptional regulator, STAT1 and transcription machinery might be targeted by virus-derived miRNAs. In addition, many known human miRNAs appear to be able to target viral genes. Considering the fact that miRNA-based therapies have been successful before, comprehending mode of actions of miRNAs and their possible roles during SARS-CoV-2 infections could create new opportunities for the development and improvement of new therapeutics." />
<meta name="DC.Contributor" content="Müşerref Duygu Saçar Demirci" />
<meta name="DC.Contributor" content="Aysun Adan" />
<meta name="article:published_time" content="2020-03-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that have been found in more than 200 diverse organisms. Although it is still not fully established if RNA viruses could generate miRNAs that would target their own genes or alter the host gene expression, there are examples of miRNAs functioning as an antiviral defense mechanism. In the case of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there are several mechanisms that would make miRNAs impact the virus, like interfering with replication, translation and even modulating the host expression. In this study, we performed a machine learning based miRNA prediction analysis for the SARS-CoV-2 genome to identify miRNA-like hairpins and searched for potential miRNA - based interactions between the viral miRNAs and human genes and human miRNAs and viral genes. Our PANTHER gene function analysis results indicate that viral derived miRNA candidates could target various human genes involved in crucial cellular processes including transcription. For instance, a transcriptional regulator, STAT1 and transcription machinery might be targeted by virus-derived miRNAs. In addition, many known human miRNAs appear to be able to target viral genes. Considering the fact that miRNA-based therapies have been successful before, comprehending mode of actions of miRNAs and their possible roles during SARS-CoV-2 infections could create new opportunities for the development and improvement of new therapeutics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.15.992438v1" />
<meta name="citation_id" content="2020.03.15.992438v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.15.992438" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.15.992438" />
<meta name="citation_author" content="Müşerref Duygu Saçar Demirci" />
<meta name="citation_author_institution" content="Bioinformatics, Abdullah Gül University" />
<meta name="citation_author_email" content="duygu.sacar@agu.edu.tr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2012-0598" />
<meta name="citation_author" content="Aysun Adan" />
<meta name="citation_author_institution" content="Molecular Biology and Genetics, Abdullah Gül University" />
<meta name="citation_reference" content="de Wilde, A. H., Snijder, E. J., Kikkert, M. &amp; van Hemert, M. J. Host Factors in Coronavirus Replication. Curr. Top. Microbiol. Immunol. 419, 1–42 (2018)." />
<meta name="citation_reference" content="Chang, L., Yan, Y. &amp; Wang, L. Coronavirus Disease 2019: Coronaviruses and Blood Safety. Transfus. Med. Rev. (2020). doi:10.1016/J.TMRV.2020.02.003" />
<meta name="citation_reference" content="Deng, S.-Q. &amp; Peng, H.-J. Characteristics of and Public Health Responses to the Coronavirus Disease 2019 Outbreak in China. J. Clin. Med. 9, (2020)." />
<meta name="citation_reference" content="Cheng, Z. J. &amp; Shan, J. 2019 Novel coronavirus: where we are and what we know. Infection 1–9 (2020). doi:10.1007/s15010-020-01401-y" />
<meta name="citation_reference" content="Chen, Y., Liu, Q. &amp; Guo, D. Emerging coronaviruses: Genome structure, replication, and pathogenesis. J. Med. Virol. 92, 418–423 (2020)." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 1–4 (2020). doi:10.1038/s41586-020-2012-7" />
<meta name="citation_reference" content="Xu, X. et al. Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission. Sci. China Life Sci. 63, 457–460 (2020)." />
<meta name="citation_reference" content="Li, X. &amp; Zou, X. An overview of RNA virus-encoded microRNAs. ExRNA 1, 37 (2019)." />
<meta name="citation_reference" content="Griffiths-Jones, S., Saini, H. K., van Dongen, S. &amp; Enright, A. J. miRBase: tools for microRNA genomics. Nucleic Acids Res. 36, D154–8 (2008)." />
<meta name="citation_reference" content="Pfeffer, S. et al. Identification of virus-encoded microRNAs. Science 304, 734–6 (2004)." />
<meta name="citation_reference" content="Saçar Demirci, M. D., Toprak, M. &amp; Allmer, J. A Machine Learning Approach for MicroRNA Precursor Prediction in Retro-transcribing Virus Genomes. J. Integr. Bioinform. 13, (2016)." />
<meta name="citation_reference" content="Berthold, M. R. et al. KNIME: The Konstanz Information Miner. in SIGKDD Explorations 11, 319–326 (2008)." />
<meta name="citation_reference" content="Saçar Demirci, M. D., Baumbach, J. &amp; Allmer, J. On the performance of pre-microRNA detection algorithms. Nat. Commun. 8, 330 (2017)." />
<meta name="citation_reference" content="Kozomara, A. &amp; Griffiths-Jones, S. miRBase: integrating microRNA annotation and deep-sequencing data. Nucleic Acids Res. 39, D152–7 (2011)." />
<meta name="citation_reference" content="Hofacker, I. L. Vienna RNA secondary structure server. Nucleic Acids Res. 31, 3429–3431 (2003)." />
<meta name="citation_reference" content="Dai, X. &amp; Zhao, P. X. PsRNATarget: A plant small RNA target analysis server. Nucleic Acids Res. 39, W155–9 (2011)." />
<meta name="citation_reference" content="Mi, H., Muruganujan, A. &amp; Thomas, P. D. PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees. Nucleic Acids Res. 41, D377–86 (2013)." />
<meta name="citation_reference" content="Gallagher, T. M. &amp; Buchmeier, M. J. Coronavirus spike proteins in viral entry and pathogenesis. Virology 279, 371–4 (2001)." />
<meta name="citation_reference" content="Wilson, L., McKinlay, C., Gage, P. &amp; Ewart, G. SARS coronavirus E protein forms cation-selective ion channels. Virology 330, 322–31 (2004)." />
<meta name="citation_reference" content="Castaño-Rodriguez, C. et al. Role of Severe Acute Respiratory Syndrome Coronavirus Viroporins E, 3a, and 8a in Replication and Pathogenesis. MBio 9, (2018)." />
<meta name="citation_reference" content="Neuman, B. W. et al. A structural analysis of M protein in coronavirus assembly and morphology. J. Struct. Biol. 174, 11–22 (2011)." />
<meta name="citation_reference" content="Masters, P. S. The molecular biology of coronaviruses. Adv. Virus Res. 66, 193–292 (2006)." />
<meta name="citation_reference" content="Graham, R. L., Sparks, J. S., Eckerle, L. D., Sims, A. C. &amp; Denison, M. R. SARS coronavirus replicase proteins in pathogenesis. Virus Res. 133, 88–100 (2008)." />
<meta name="citation_reference" content="Lau, S. K. P. et al. Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination. J. Virol. 89, 10532–47 (2015)." />
<meta name="citation_reference" content="Schaecher, S. R., Touchette, E., Schriewer, J., Buller, R. M. &amp; Pekosz, A. Severe acute respiratory syndrome coronavirus gene 7 products contribute to virus-induced apoptosis. J. Virol. 81, 11054–68 (2007)." />
<meta name="citation_reference" content="Tan, Y.-X. et al. Induction of apoptosis by the severe acute respiratory syndrome coronavirus 7a protein is dependent on its interaction with the Bcl-XL protein. J. Virol. 81, 6346–55 (2007)." />
<meta name="citation_reference" content="Paraskevis, D. et al. Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event. Infect. Genet. Evol. 79, 104212 (2020)." />
<meta name="citation_reference" content="Frieman, M. et al. Severe acute respiratory syndrome coronavirus ORF6 antagonizes STAT1 function by sequestering nuclear import factors on the rough endoplasmic reticulum/Golgi membrane. J. Virol. 81, 9812–24 (2007)." />
<meta name="citation_reference" content="Ghosh, Z., Mallick, B. &amp; Chakrabarti, J. Cellular versus viral microRNAs in host-virus interaction. Nucleic Acids Res. 37, 1035–48 (2009)." />
<meta name="citation_reference" content="Piedade, D. &amp; Azevedo-Pereira, J. M. The Role of microRNAs in the Pathogenesis of Herpesvirus Infection. Viruses 8, (2016)." />
<meta name="citation_reference" content="Engelmann, I., Alidjinou, E. K., Bertin, A., Sane, F. &amp; Hober, D. miRNAs in enterovirus infection. Crit. Rev. Microbiol. 44, 701–714 (2018)." />
<meta name="citation_reference" content="Khatun, M. &amp; Ray, R. B. Mechanisms Underlying Hepatitis C Virus-Associated Hepatic Fibrosis. Cells 8, (2019)." />
<meta name="citation_reference" content="Mishra, R., Kumar, A., Ingle, H. &amp; Kumar, H. The Interplay Between Viral-Derived miRNAs and Host Immunity During Infection. Front. Immunol. 10, 3079 (2019)." />
<meta name="citation_reference" content="Li, X. et al. H5N1 influenza virus-specific miRNA-like small RNA increases cytokine production and mouse mortality via targeting poly(rC)-binding protein 2. Cell Res. 28, 157–171 (2018)." />
<meta name="citation_reference" content="Liang, H. et al. Identification of Ebola virus microRNAs and their putative pathological function. Sci. China. Life Sci. 57, 973–81 (2014)." />
<meta name="citation_reference" content="Klase, Z. et al. HIV-1 TAR miRNA protects against apoptosis by altering cellular gene expression. Retrovirology 6, 18 (2009)." />
<meta name="citation_reference" content="Zhang, S. et al. Up-regulation of microRNA-203 in influenza A virus infection inhibits viral replication by targeting. Sci. Rep. 8, (2018)." />
<meta name="citation_reference" content="Mallick, B., Ghosh, Z. &amp; Chakrabarti, J. MicroRNome analysis unravels the molecular basis of SARS infection in bronchoalveolar stem cells. PLoS One 4, e7837 (2009)." />
<meta name="citation_reference" content="Ma, Y.-J. et al. Cellular microRNA let-7c inhibits M1 protein expression of the H1N1 influenza A virus in infected human lung epithelial cells. J. Cell. Mol. Med. 16, 2539–46 (2012)." />
<meta name="citation_reference" content="Huang, S.-H. et al. Phage display technique identifies the interaction of severe acute respiratory syndrome coronavirus open reading frame 6 protein with nuclear pore complex interacting protein NPIPB3 in modulating Type I interferon antagonism. J. Microbiol. Immunol. Infect. 50, 277–285 (2017)." />
<meta name="citation_reference" content="Bandiera, S. et al. Hepatitis C Virus-Induced Upregulation of MicroRNA miR-146a-5p in Hepatocytes Promotes Viral Infection and Deregulates Metabolic Pathways Associated with Liver Disease Pathogenesis. J. Virol. 90, 6387–6400 (2016)." />
<meta name="citation_reference" content="Ye, R., Montalto-Morrison, C. &amp; Masters, P. S. Genetic analysis of determinants for spike glycoprotein assembly into murine coronavirus virions: distinct roles for charge-rich and cysteine-rich regions of the endodomain. J. Virol. 78, 9904–17 (2004)." />
<meta name="citation_reference" content="Trobaugh, D. W. &amp; Klimstra, W. B. MicroRNA Regulation of RNA Virus Replication and Pathogenesis. Trends Mol. Med. 23, 80–93 (2017)." />
<meta name="citation_reference" content="Raza Naqvi, A., Ahmad, I., Valverde, A., Siddiqui, H. &amp; Schaller, S. Viral MicroRNAs: Interfering the Interferon Signaling. Curr. Pharm. Des. (2020). doi:10.2174/1381612826666200109181238" />
<meta name="citation_reference" content="Herbert, K. M. &amp; Nag, A. A Tale of Two RNAs during Viral Infection: How Viruses Antagonize mRNAs and Small Non-Coding RNAs in The Host Cell. Viruses 8, (2016)." />
<meta name="citation_reference" content="Yang, Y.-C. &amp; Chang, L.-K. Role of TAF4 in transcriptional activation by Rta of Epstein-Barr Virus. PLoS One 8, e54075 (2013)." />
<meta name="citation_reference" content="Abernathy, E. &amp; Glaunsinger, B. Emerging roles for RNA degradation in viral replication and antiviral defense. Virology 479–480, 600–8 (2015)." />
<meta name="citation_reference" content="Ramalingam, D. &amp; Ziegelbauer, J. M. Viral microRNAs Target a Gene Network, Inhibit STAT Activation, and Suppress Interferon Responses. Sci. Rep. 7, 40813 (2017)." />
<meta name="citation_reference" content="van der Ree, M. H. et al. Safety, tolerability, and antiviral effect of RG-101 in patients with chronic hepatitis C: a phase 1B, double-blind, randomised controlled trial. Lancet (London, England) 389, 709–717 (2017)." />
<meta name="twitter:title" content="Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that have been found in more than 200 diverse organisms. Although it is still not fully established if RNA viruses could generate miRNAs that would target their own genes or alter the host gene expression, there are examples of miRNAs functioning as an antiviral defense mechanism. In the case of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there are several mechanisms that would make miRNAs impact the virus, like interfering with replication, translation and even modulating the host expression. In this study, we performed a machine learning based miRNA prediction analysis for the SARS-CoV-2 genome to identify miRNA-like hairpins and searched for potential miRNA – based interactions between the viral miRNAs and human genes and human miRNAs and viral genes. Our PANTHER gene function analysis results indicate that viral derived miRNA candidates could target various human genes involved in crucial cellular processes including transcription. For instance, a transcriptional regulator, STAT1 and transcription machinery might be targeted by virus-derived miRNAs. In addition, many known human miRNAs appear to be able to target viral genes. Considering the fact that miRNA-based therapies have been successful before, comprehending mode of actions of miRNAs and their possible roles during SARS-CoV-2 infections could create new opportunities for the development and improvement of new therapeutics." />
<meta name="og-title" property="og:title" content="Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that have been found in more than 200 diverse organisms. Although it is still not fully established if RNA viruses could generate miRNAs that would target their own genes or alter the host gene expression, there are examples of miRNAs functioning as an antiviral defense mechanism. In the case of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there are several mechanisms that would make miRNAs impact the virus, like interfering with replication, translation and even modulating the host expression. In this study, we performed a machine learning based miRNA prediction analysis for the SARS-CoV-2 genome to identify miRNA-like hairpins and searched for potential miRNA – based interactions between the viral miRNAs and human genes and human miRNAs and viral genes. Our PANTHER gene function analysis results indicate that viral derived miRNA candidates could target various human genes involved in crucial cellular processes including transcription. For instance, a transcriptional regulator, STAT1 and transcription machinery might be targeted by virus-derived miRNAs. In addition, many known human miRNAs appear to be able to target viral genes. Considering the fact that miRNA-based therapies have been successful before, comprehending mode of actions of miRNAs and their possible roles during SARS-CoV-2 infections could create new opportunities for the development and improvement of new therapeutics." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.15.992438v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.15.992438v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.15.992438v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1199874" />
    <title>Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oOOD_I4kthFLlw3hLlRIqiduH7B8-C_xIf4WEwwP66g","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1199874","apath":"\/biorxiv\/early\/2020\/03\/17\/2020.03.15.992438.atom","pisa":"biorxiv;2020.03.15.992438v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.15.992438v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.15.992438v1","title":"Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection","identifier":"node\/1199874"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hRphTb2Le\\\\\\\/AjOO+AdLThS5R7Ya0KRlFky8HgZEgxk5K3RW7IhA9eF06SIlYisE3QCffH3OuR9hvKbMEczFpeLXbT9WzXJ7D+4lNlzP7GzzNttBgp6L\\\\\\\/ZdyEKcPsCwRU92yB28jrjAZ0rOTv7ZyPi8t99cfYsLzovs621+DkckYtm\\\\\\\/aYDlhT93sTt0bYFDdrJvMF0zTeQKMf90kMXI0rAtfciP+kFiOAZgoFK9h0OUIbhZOPjPpw2Y5cNSE1Ec7y2fdvG0mCjkaE5MRczadaqkUGxpaZqr6j76oZ4G0b9IncLf+gWsblghcW8RDbKaAihzd53Dc5bS7oedS9\\\\\\\/7MwMPLuh6Mho0zxwSFuHusGaneW1IIcBCcTFjTMJ89x06QNUv2f0xyKVb536\\\\\\\/Q7gJV5UR43YC2mEzTNz3rr\\\\\\\/tBZN\\\\\\\/Rl9Zf2rDMQfmEeSun9tasaLZ9bgEJsOAnRY4D82F9\\\\\\\/HMbSU96OHWKJA2R5YiCOIBMqbOOrMB1Drr1UVnDwG5\\\\\\\/W1Vst6GnXiE5efuoyqY7Zdo7mq7aAd5ldHsJwrDN5OuWLx4M4sYFSvfaInIo8CFZLwYu2o24C456Bkj+2W+oCgvs\\\\\\\/fL+xyDnsol1aYaZGU\\\\\\\/N8u9FxeXKUk1rysQ6sg3OSWgPp5I29Jx+dM9DJ7oWFy+DpnINvXRiuvnM4wCh8sxHehZUhge8JKBiFi1BVKNiyeytI08bhtr\\\\\\\/5N7zBElfnVE1+hbA76ELZ9n\\\\\\\/qVtQroY0GNLoS5V\\\\\\\/wfKcLiJ9PkXDinZABcL1oAyizqaTw1FXv+MdPHk+o80D065JjzJuogwIf0mjmX+\\\\\\\/\\\\\\\/ntOUtTyCaiYOQEbYnH1wU\\\\\\\/cxufWWcO88fDPrfRshs4\\\\\\\/Bm5GvH4YLJq8kOxUdpECZpg0WQeliqMt2CgOG5luryLWPTuKgKy\\\\\\\/y\\\\\\\/UXGNqGPfcoJ\\\\\\\/vJK2gstj86XSFEvV+fFHGWQlOeGUZCGt7yPnfIGLAaNkGMbI5s7JBcwGBhbfKCmhSCH1QWYCYf5X+ZGh87tSqvI+xqtKUO8hshHi\\\\\\\/hgGp3HKLPJLtN2mu8w\\\\\\\/k8Ogg8Sl2mQcfS18jXmfPoJiJVOTSBs+L6VnzP3WW5AZ24cx+EYa5hCh+nXht7sv6mleuE16YJsvWOgclR9yRd\\\\\\\/Grqe5uhlivIS4ucNPM6x5yP3vKDzRmukha812wgJlx4u5vdW1dwyI6dJP1zSVnZ+XQ+uaudqxE8\\\\\\\/goMa+mAy4ZlVHxViosAxjEkfN2FvIhzAZqKZcbJH1pliKgF2vU2besPEvyY1g9AHIRotUG1YANfC6OVtW2kuP9x1EO1t1KD1GLtOxc8QTjNzWoEk7PCq7W7noyZ9p72qnU5TP\\\u0022,\\\u0022iv\\\u0022:\\\u0022154Cy2g6iH6Z9b7aasJlkQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022291b21d990d47190538eea4d93f8dab35b54c0acda592e389d579a557965f7a7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.15.992438v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1199874 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.15.992438v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qLsCjwnOxHpISGTW4wnFcnuwGVfmKX5i59iA1gcsDQw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1199874" id="node1199874" data-pisa="biorxiv;2020.03.15.992438v1" data-pisa-master="biorxiv;2020.03.15.992438" data-apath="/biorxiv/early/2020/03/17/2020.03.15.992438.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2012-0598" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Müşerref Duygu</span> <span class="nlm-surname">Saçar Demirci</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Aysun</span> <span class="nlm-surname">Adan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.15.992438 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1199874" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Müşerref Duygu Saçar Demirci </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics, Abdullah Gül University</span>, Kayseri, <span class='nlm-country'>Turkey</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%BC%C5%9Ferref%2BDuygu%2BSa%C3%A7ar%2BDemirci%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sa%C3%A7ar%20Demirci%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM%25C3%25BC%25C5%259Ferref%2BDuygu%2BSa%25C3%25A7ar%2BDemirci%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2012-0598" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Müşerref Duygu Saçar Demirci</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duygu.sacar@agu.edu.tr" class="" data-icon-position="" data-hide-link-title="0">duygu.sacar@agu.edu.tr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aysun Adan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Molecular Biology and Genetics, Abdullah Gül University</span>, Kayseri, <span class='nlm-country'>Turkey</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aysun%2BAdan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Adan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAysun%2BAdan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.15.992438v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.15.992438v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.15.992438v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.15.992438v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.15.992438v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.15.992438v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1199874" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1199874/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that have been found in more than 200 diverse organisms. Although it is still not fully established if RNA viruses could generate miRNAs that would target their own genes or alter the host gene expression, there are examples of miRNAs functioning as an antiviral defense mechanism. In the case of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there are several mechanisms that would make miRNAs impact the virus, like interfering with replication, translation and even modulating the host expression. In this study, we performed a machine learning based miRNA prediction analysis for the SARS-CoV-2 genome to identify miRNA-like hairpins and searched for potential miRNA – based interactions between the viral miRNAs and human genes and human miRNAs and viral genes. Our PANTHER gene function analysis results indicate that viral derived miRNA candidates could target various human genes involved in crucial cellular processes including transcription. For instance, a transcriptional regulator, STAT1 and transcription machinery might be targeted by virus-derived miRNAs. In addition, many known human miRNAs appear to be able to target viral genes. Considering the fact that miRNA-based therapies have been successful before, comprehending mode of actions of miRNAs and their possible roles during SARS-CoV-2 infections could create new opportunities for the development and improvement of new therapeutics.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Coronoviruses (CoVs) are pathogens with serious health effects including enteric, respiratory, hepatic and central nervous diseases on human and animals. Zoonotic CoVs, Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome coronavirus (MERS-CoV), have been identified the sources of outbreaks in 2002/2003 and 2012, respectively with high mortality rates due to severe respiratory syndrome <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Besides zoonotic CoVs, there are four types of human CoVs have been identified known as HCoV-OC43, HCoV-2293, HCoV-NL63 and HCoV-HKU1 <sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a></sup>. Unknown pneumonia have been detected in Wuhan, China and spread globally since December 2019. The World Health Organization (WHO) named this new coronavirus as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) responsible for the new disease termed Coronavirus Disease 2019 (COVID-19) and it is the seventh identified CoV with animal origin infecting human <sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup>.</p><p id="p-4">Coronaviruses are positive-single stranded RNA (+ssRNA) viruses with exceptionally large genomes of ∼ 30 kb with 5′cap structure and 3′polyA tail. The coronavirus subfamily is divided into four genera: α, β, γ, and δ based on serotype and genome features <sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. The genome of a typical CoV codes for at least 6 different open reading frames (ORFs), which has variations based on the CoV type <sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Some ORFs encode non-structural proteins while others code for structural proteins required for viral replication and pathogenesis. Structural proteins include spike (S) glycoprotein, matrix (M) protein, small envelope (E) protein, and nucleocapsid (N) protein with various roles for virus enterance and spread. SARS-CoV-2 belongs to β CoV with 45-90% genetic similarity to SARS-CoV based on sequence analysis and might share similar viral genomic and transcriptomic complexity <sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. Currently, it has been also revealed that SARS-COV-2 has a very high homology with bat CoVs, which indicated how it is transmitted to human without knowing intermediate carriers <sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. S protein of SARS-CoV-2 has a strong interaction with human angiotensin-converting enzyme 2 (ACE2) expressed on alveolar epithelial cells which shows the way of virus infection in human <sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>.</p><p id="p-5">MicroRNAs (miRNAs) are small, noncoding RNAs that play role in regulation of the gene expression in various organisms ranging from viruses to higher eukaryotes. It has been estimated that miRNAs might influence around 60% of mammalian genes and their main effect is on regulatory pathways including cancer, apoptosis, metabolism and development <sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>.</p><p id="p-6">Although the current release of miRNAs, the standard miRNA depository, lists miRNAs of 271 organisms, only 34 of them are viruses <sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. While, the first virus-encoded miRNAs was discovered for the human Epstein-Barr virus (EBV) <sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>, more than 320 viral miRNA precursors were reported so far. Although it has been shown that various DNA viruses express miRNAs, it is still debatable if RNA viruses could also encode. The major concerns regarding miRNAs of RNA viruses are based on <sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-7">- the fact that RNA viruses that replicate in cytoplasm, do not have access to nuclear miRNA machinery</p></li><li id="list-item-2"><p id="p-8">- since RNA is the genetic material, miRNA production would interfere with viral replication</p></li></ul><p id="p-9">On the other hand, many features of miRNA - based gene regulation seems to be especially beneficial for viruses. For instance, through targeting specific human genes by viral miRNAs, it is possible to form an environment suitable for survival and replication of the virus. Furthermore, for viral miRNAs it is likely to escape host defense system since host itself generates miRNAs in the same manner.</p><p id="p-10">Currently, options for the prevention and treatment of CoVs are very limited due to the complexity. Therefore, detailed analysis of CoV-host interactions is quite important to understand viral pathogenesis and to determine the outcomes of infection. Although there are studies regarding to the viral replication and their interaction with host innate immune system, the role of miRNA-mediated RNA-silencing in SARS-CoV-2 infection has not been enlightened yet. In this study, SARS-CoV-2 genome was searched for miRNA-like sequences and potential host-virus interactions based on miRNA actions were analyzed.</p><div id="sec-2" class="subsection"><h3>Methods</h3><p id="p-11">Data analysis, pre-miRNA prediction, mature miRNA detection workflows were generated by using the Konstanz Information Miner (KNIME) platform <sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup></p></div><div id="sec-3" class="subsection"><h3>Data</h3><p id="p-12">Genome data of the virus was obtained from NCBI:</p><p id="p-13">Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1, complete genome</p><p id="p-14">GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom">MN908947</a>.3</p><p id="p-15">MiRNA prediction workflow izMiR <sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> and its related data were taken from Mendeley Data: <a href="https://data.mendeley.com/datasets/mgh5r9wny7/1">https://data.mendeley.com/datasets/mgh5r9wny7/1</a></p><p id="p-16">Mature miRNA sequences of human were downloaded from miRBase Release 22.1 <sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>.</p></div><div id="sec-4" class="subsection"><h3>Pre-miRNA prediction</h3><p id="p-17">Genome sequences of SARS-CoV-2 were transcribed (T-&gt;U) and divided into 500 nt long fragments with 250 overlaps. Then these fragments were folded into their secondary structures by using RNAfold <sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> with default settings and hairpin structures were extracted, producing 950 hairpins in total.</p><p id="p-18">A modified version of izMiR (SVM classifier is changed to Random Forest and latest miRBase version was used for learning) was applied to these hairpins with ranging lengths (from 7 to 176) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Based on the mean value of averages of 3 classifiers’ prediction scores (Decision Tree, Naive Bayes and Random Forest), 29 hairpins passed 0.900 threshold and used for further analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-plots for comparison of general features of human pre-miRNAs and SARS-CoV-2 hairpins. Hairpin lengths shown in top graph. Minimum free energy scores obtained from RNAfold 15 plotted in bottom panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650035410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Box-plots for comparison of general features of human pre-miRNAs and SARS-CoV-2 hairpins. Hairpin lengths shown in top graph. Minimum free energy scores obtained from RNAfold &lt;sup&gt;15&lt;/sup&gt; plotted in bottom panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-1.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-1.medium.gif" width="440" height="263"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-plots for comparison of general features of human pre-miRNAs and SARS-CoV-2 hairpins. Hairpin lengths shown in top graph. Minimum free energy scores obtained from RNAfold 15 plotted in bottom panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650035410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Box-plots for comparison of general features of human pre-miRNAs and SARS-CoV-2 hairpins. Hairpin lengths shown in top graph. Minimum free energy scores obtained from RNAfold &lt;sup&gt;15&lt;/sup&gt; plotted in bottom panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-2.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-2.medium.gif" width="440" height="263"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-19" class="first-child">Box-plots for comparison of general features of human pre-miRNAs and SARS-CoV-2 hairpins. Hairpin lengths shown in top graph. Minimum free energy scores obtained from RNAfold <sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup> plotted in bottom panel.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Mature miRNA prediction and targeting</h3><p id="p-20">Selected hairpins were further processed into smaller sequences; maximum 23 nt length with 6 nt overlaps. Then, these fragments were filtered based on minimum length of 15 and their location on the hairpins (sequences not involving any loop nucleotides were included). Target search of these remaining 30 candidate mature miRNAs were performed against human and SARS-CoV-2 genes by using psRNATarget tool with default settings <sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Moreover, human mature miRNAs’ from miRBase were applied for searching their targets in SARS-CoV-2 genes.</p></div><div id="sec-6" class="subsection"><h3>Gene Ontology</h3><p id="p-21">The targets of viral miRNAs in human genes were further analyzed for their Gene Ontology (GO). To achieve this, PANTHER Classification System (<a href="http://www.pantherdb.org">http://www.pantherdb.org</a>) was used <sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>.</p></div></div><div class="section" id="sec-7"><h2 class="">Results</h2><p id="p-22">Searching SARS-CoV-2 genome for sequences forming hairpin structures resulted in 950 hairpins with varying lengths (Supplementary Files). In order to use machine learning based miRNA prediction approach of izMiR workflows, hundreds of features were calculated for all of the pre-miRNA candidate sequences. Among those, minimum free energy (mfe) values required for the folding of secondary structures of hairpin sequences and hairpin sequence lengths of known human miRNAs from miRBase and predicted hairpins of SARS-CoV-2 were compared (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Based on the box-plots shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, most of the extracted viral hairpins seem to be smaller than human miRNA precursors.</p><p id="p-23">Since viruses would need to use at least some members of host miRNA biogenesis pathway elements, viral miRNAs should be similar to host miRNAs to a certain degree. Therefore, a classification scheme trained with known human miRNAs was applied on SARS-CoV-2 hairpins. Only 29 hairpins out of 950 passed the 0.900 prediction score threshold and used for further analysis. From these hairpins, 30 mature miRNA candidates were extracted and their possible targets for human and SARS-CoV-2 genes were investigated.</p><p id="p-24">SARS-CoV-2 miRNA candidates were further analyzed to test if they were similar to any of the known mature miRNAs from 271 organism listed in miRBase. To achieve this, a basic similarity search was performed based on the Levenshtein distance calculations in KNIME. However, there was no significant similarity between hairpin or mature sequences.</p><p id="p-25">While predicted mature miRNAs of SARS-CoV-2 were used to find their targets in SARS-CoV-2 and human genes, mature miRNAs of human were also applied on SARS-CoV-2 genes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, Supplementary Files). Although miRNA based self-regulation of viral gene expression is a hypothetical case, SARS-CoV-2 ORF1ab polyprotein gene might be the only one that could be a target of viral miRNAs. In total 1367 human genes seem to be targeted by viral miRNAs. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> lists some of the predicted targets of SARS-CoV-2 miRNAs in human genes that have roles in transcription. The full list of miRNA – target predictions are available in Supplementary Files.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1202540/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1202540/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-26" class="first-child">Transcription related human gene targets of viral miRNAs. MiRNA column indicates the accession numbers of mature viral miRNAs; Target gene # column shows the total number of different genes involved in transcription and targeted by the corresponding viral MiRNAs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Considering the potential of miRNAs as biomarkers and therapeutic agents, it is essential to check if any of the known human miRNAs could target viral genes. Among 2654 mature entries of <em>Homo sapiens</em> in miRBase Release 22.1, 479 of them could target SARS-CoV-2 genes (Supplementary Files). While Envelope and ORF6 genes were targeted by single miRNAs, ORF1ab appeared to be the target of 369 different human mature miRNAs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). As expected, number of targeting events appear to be correlated with the gene length.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1202539/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1202539/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-28" class="first-child">Predicted virulent mRNA targets by Human miRNAs: bold miRNAs are the common ones targeting more than one indicated viral proteins. The functions of SARS-CoV-2 proteins are not fully characterized, however, its coding genes might share functional similarity with SARS-CoV as shown in column “Functions of Target Genes”. Gene size indicates the size of genes in terms of number of nucleotides, hsa miRNAs shows the number of different human miRNAs that could target indicated viral genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Lastly, in order to understand the main mechanisms that would be affected by the influence of viral miRNAs on human genes, PANTHER Classification System was applied to targeted human genes. Based on the results presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> – <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a> and Supplementary Files, a wide range of human genes with various molecular functions and pathways could be targeted.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar-chart for the protein classes of human genes that could be targeted by viral miRNAs. Protein classes of genes were obtained from Panther. X-axis shows the number of genes with respected classes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650035410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar-chart for the protein classes of human genes that could be targeted by viral miRNAs. Protein classes of genes were obtained from Panther. X-axis shows the number of genes with respected classes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F2.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F2.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-30" class="first-child">Bar-chart for the protein classes of human genes that could be targeted by viral miRNAs. Protein classes of genes were obtained from Panther. X-axis shows the number of genes with respected classes.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar-chart for the pathways of human genes that could be targeted by viral miRNAs. Graph is limited to the pathways that have at least 10 genes. Pathways of genes were obtained from Panther. Y-axis shows the number of genes with respected pathways. Legend is sorted from maximum to minimum (top to bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650035410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar-chart for the pathways of human genes that could be targeted by viral miRNAs. Graph is limited to the pathways that have at least 10 genes. Pathways of genes were obtained from Panther. Y-axis shows the number of genes with respected pathways. Legend is sorted from maximum to minimum (top to bottom).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F3.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F3.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992438/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-31" class="first-child">Bar-chart for the pathways of human genes that could be targeted by viral miRNAs. Graph is limited to the pathways that have at least 10 genes. Pathways of genes were obtained from Panther. Y-axis shows the number of genes with respected pathways. Legend is sorted from maximum to minimum (top to bottom).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h3>Discussion</h3><p id="p-32">The potential roles of miRNA-mediated RNA interference in infection biology has been defined as an essential regulatory molecular pathway. The involvement of miRNAs in host-pathogen interactions during infection might include the targeting of viral genes by host-cellular miRNAs as well as evolution of DNA and RNA virus-based gene silencing mechanisms to overcome host antiviral response and to maintain viral infection and disease <sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. In addition, viruses might use host cellular miRNAs for their own advantage. Therefore, molecular elucidation of the roles of miRNAs in host-virus interaction might provide a deeper understanding for viral pathogenesis and the development of an effective antiviral therapy as shown in several viruses including Herpesvirus, Enterovirus and Hepatitis C <sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>–<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. Although the detailed knowledge about viral miRNAs have been obtained from DNA viruses, it is still controversial for RNA viruses that whether they produce their own miRNAs or not <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. However, there are several reports showing the presence of non-canonical (due to the lack of classical stem-loop structure in miRNAs), small miRNA-like small RNAs produced during viral infections as shown in H5N1 Influenza <sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>, Ebola virus <sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> and HIV-1 <sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. Although they play crucial roles in several viral infections, miRNAs have not been studied in the pathogenesis of SARS-CoV-2 infection. In this study, we elucidated miRNA-mediated regulation responsible for SAR-CoV-2 infection by predicting possible host genes targeted by viral miRNAs and viral genes targeted by cellular miRNAs, which might provide potential ways to understand the underlying mechanisms of SARS-CoV-2 infection.</p><p id="p-33">We investigated possible virulent genes of SARS-CoV-2 targeted by host-cellular miRNAs as shown in <strong><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a></strong>, which are mainly responsible for viral biogenesis, entrance, replication and infection. Except envelope (E) protein and ORF6, all viral genes (S, M, N, ORF1ab, ORF3a, ORF8, ORF7a and ORF10) are targeted by multiple human miRNAs. For instance, hsa-miR-203b-3p targeted ORF1ab and ORF3a with roles in viral replication was already shown to suppress influenza A virus replication <sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. Even though hsa-miR-148a-3p targeted ORF8 to prevent interspecies transmission and also replication, it was found to target S, E, M and ORF1a protein in closely related SARS-CoV <sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. hsa-let-7c-5p targeted ORF1ab in SARS-CoV-2 while it was found to be involved in H1N1 influenza A suppression by targeting its M1 protein <sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>. In another study, ORF6 of SARS-CoV suppressed type I interferon signaling by blocking the nuclear transport of STAT1 in the presence of interferon β <sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>, therefore, hsa-miR-190a-5p might target ORF6 to overcome immune system escape in SARS-CoV-2. The presence of such miRNAs could be considered as a host’s innate antiviral defense mechanism. On the other hand, virus could use these miRNAs to suppress their own replication to escape from immune system at the beginning of infection and transmission for a stronger infection. For instance, miR-146a-5p was upregulated in hepatitis C virus-infected liver cells of patients and in infected human hepatocytes, which promoted virus particle assembly <sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Moreover, ss RNA viruses could evolve very rapidly to change their gene sequences matching with these host miRNAs, therefore, they increase their host specificity. Once the virus establishes a successful transmission inside the host, they would mutate their genes very rapidly to escape from host miRNAs, which results from their RNA polymerases without proofreading activity <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>.</p><p id="p-34">Virus-derived miRNAs might function by targeting host and virus-encoded transcripts to regulate host-pathogen interaction. The roles of viral miRNAs in pathogenesis include alteration of host defense mechanisms and regulation of crucial biological processes including cell survival, proliferation, modulation of viral life-cycle phase <sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. Although encoding miRNAs seems quite problematic for RNA viruses due to the nature of miRNA biogenesis pathway, it is possible to circumvent these problems through different ways as seen in HIV-1 <sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup>. Therefore, we analyzed possible human genes targeted by predicted miRNA like small RNAs (<strong><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a></strong>). Based on the panther analysis, regulators of eukaryotic transcription would be the most important targets of 18 SARS-CoV-2 derived mature miRNA like candidates out of 29 hairpins in total. These transcriptional regulators are involved in both basal transcription machinery including several components of human mediator complex (MED1, MED9, MED12L, MED19) and basal transcription factors such as TAF4, TAF5 and TAF7L. Viruses might downregulate host gene expression in order to increase their gene expression either co-transcriptionally in the nucleus or post-transcriptionally in the nucleus or cytoplasm <sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. Therefore, targeting basal transcription machinery such as components (TAFs) of TFIID complex by SARS-CoV-2 could prevent RNA polymerase II to assemble on promoters of host genes at the initiation step. Viral factors have been shown to block transcription initiation by inhibiting TFIID or more specifically TAF4 in herpesvirus <sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. Another interesting target gene by SARS-CoV-2 miRNA was different subunits of CCR4-NOT transcription complex including CNOT4, CNOT10 and CNOT6L, which are deadenylases involved in mRNA decay <sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. Therefore, suppression of these genes by viral miRNAs could impede mRNA turnover in the host and provide opportunities for viral mRNA to escape from degradation. Additionally, site specific trans-acting factors such as MAFG, STAT1, STAT5B and SOX11 would be targeted by viral miRNAs. STAT family transcription factors are activated by cytokine-induced stimuli and generally involved in an interferon response. In Kaposi’s sarcoma-associated herpesvirus (KSHV), viral miRNAs were found to inhibit STAT3 and STAT5, resulting in deregulated interferon response and transition into lytic viral replication <sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>.</p><p id="p-35">As a conclusion, viral diseases have been paid attention as a global health problem due to the lack of proper treatment strategies and rapid evolution of viruses. In recent years, studies have focused on identifying miRNAs as targets for the treatment of viral diseases and there are promising clinical phase studies evaluating the therapeutic potential of antagomirs targeting host miRNAs such as miR-122 in hepatitis C <sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. Based on the results shown in <strong><a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a></strong>, it can be concluded that increases in the level of host miRNAs targeting virulent genes such as S, M, N, E and ORF1ab would block viral entry and replication. Moreover, decreasing the levels of host miRNAs would make SARS-CoV-2 more replicative and visible for the host immune system. However, alterations in host miRNA levels would interfere with specific cellular processes which are crucial for the host biology. In our study, we have also identified possible miRNA like small RNAs from SARS-CoV-2 genome which target important human genes. Therefore, antagomirs targeting viral miRNAs could be also designed even though there are only a few studies for DNA viruses <sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup>. However, all these therapeutic possibilities need further mechanistical evaluations to understand how they regulate virus-host interaction. Therefore, further in vitro, ex vivo and in vivo studies will be required to validate candidate miRNAs for SARS-CoV-2 infection.</p></div></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Wilde</span>,  <span class="cit-name-given-names">A. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kikkert</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">van Hemert</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Host Factors in Coronavirus Replication</span>. <abbr class="cit-jnl-abbrev">Curr. Top. Microbiol. Immunol</abbr>. <span class="cit-vol">419</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">42</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BTop.%2BMicrobiol.%2BImmunol%26rft.volume%253D419%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.2" data-doi="10.1016/J.TMRV.2020.02.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Coronavirus Disease 2019: Coronaviruses and Blood Safety</span>. <abbr class="cit-jnl-abbrev">Transfus. Med. Rev</abbr>. (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/J.TMRV.2020.02.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransfus.%2BMed.%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.TMRV.2020.02.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/J.TMRV.2020.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">S.-Q.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">H.-J.</span></span> <span class="cit-article-title">Characteristics of and Public Health Responses to the Coronavirus Disease 2019 Outbreak in China</span>. <abbr class="cit-jnl-abbrev">J. Clin. Med</abbr>. <span class="cit-vol">9</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.4" data-doi="10.1007/s15010-020-01401-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Z. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shan</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">2019 Novel coronavirus: where we are and what we know</span>. <abbr class="cit-jnl-abbrev">Infection</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s15010-020-01401-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs15010-020-01401-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s15010-020-01401-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Q.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Emerging coronaviruses: Genome structure, replication, and pathogenesis</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. <span class="cit-vol">92</span>, <span class="cit-fpage">418</span>–<span class="cit-lpage">423</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirol%26rft.volume%253D92%26rft.spage%253D418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.6" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission</span>. <abbr class="cit-jnl-abbrev">Sci. China Life Sci</abbr>. <span class="cit-vol">63</span>, <span class="cit-fpage">457</span>–<span class="cit-lpage">460</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina%2BLife%2BSci%26rft.volume%253D63%26rft.spage%253D457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-article-title">An overview of RNA virus-encoded microRNAs</span>. <abbr class="cit-jnl-abbrev">ExRNA</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">37</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExRNA%26rft.volume%253D1%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.9" data-doi="10.1093/nar/gkm952"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saini</span>,  <span class="cit-name-given-names">H. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Dongen</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Enright</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-article-title">miRBase: tools for microRNA genomics</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">36</span>, <span class="cit-fpage">D154</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm952%26rft_id%253Dinfo%253Apmid%252F17991681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17991681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.10" data-doi="10.1126/science.1096781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeffer</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of virus-encoded microRNAs</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>, <span class="cit-fpage">734</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPfeffer%26rft.auinit1%253DS.%26rft.volume%253D304%26rft.issue%253D5671%26rft.spage%253D734%26rft.epage%253D736%26rft.atitle%253DIdentification%2Bof%2BVirus-Encoded%2BMicroRNAs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1096781%26rft_id%253Dinfo%253Apmid%252F15118162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MS83MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNy8yMDIwLjAzLjE1Ljk5MjQzOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saçar Demirci</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toprak</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Allmer</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">A Machine Learning Approach for MicroRNA Precursor Prediction in Retro-transcribing Virus Genomes</span>. <abbr class="cit-jnl-abbrev">J. Integr. Bioinform</abbr>. <span class="cit-vol">13</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berthold</span>,  <span class="cit-name-given-names">M. R.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">KNIME: The Konstanz Information Miner. in SIGKDD Explorations</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">319</span>–<span class="cit-lpage">326</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DKNIME%253A%2BThe%2BKonstanz%2BInformation%2BMiner.%2Bin%2BSIGKDD%2BExplorations%26rft.volume%253D11%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saçar Demirci</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baumbach</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Allmer</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">On the performance of pre-microRNA detection algorithms</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">330</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D8%26rft.spage%253D330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.14" data-doi="10.1093/nar/gkq1027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozomara</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">miRBase: integrating microRNA annotation and deep-sequencing data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">39</span>, <span class="cit-fpage">D152</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1027%26rft_id%253Dinfo%253Apmid%252F21037258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21037258&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.15" data-doi="10.1093/nar/gkg599"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I. L.</span></span> <span class="cit-article-title">Vienna RNA secondary structure server</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">31</span>, <span class="cit-fpage">3429</span>–<span class="cit-lpage">3431</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHofacker%26rft.auinit1%253DI.%2BL.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3429%26rft.epage%253D3431%26rft.atitle%253DVienna%2BRNA%2Bsecondary%2Bstructure%2Bserver%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg599%26rft_id%253Dinfo%253Apmid%252F12824340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg599&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824340&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.16" data-doi="10.1093/nar/gkr319"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">P. X.</span></span> <span class="cit-article-title">PsRNATarget: A plant small RNA target analysis server</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">39</span>, <span class="cit-fpage">W155</span>–<span class="cit-lpage">9</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr319%26rft_id%253Dinfo%253Apmid%252F21622958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21622958&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.17" data-doi="10.1093/nar/gks1118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mi</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muruganujan</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">P. D.</span></span> <span class="cit-article-title">PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">41</span>, <span class="cit-fpage">D377</span>–<span class="cit-lpage">86</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1118%26rft_id%253Dinfo%253Apmid%252F23193289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23193289&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.18" data-doi="10.1006/viro.2000.0757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>,  <span class="cit-name-given-names">T. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Coronavirus spike proteins in viral entry and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">279</span>, <span class="cit-fpage">371</span>–<span class="cit-lpage">4</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DGallagher%26rft.auinit1%253DT.%2BM.%26rft.volume%253D279%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D374%26rft.atitle%253DCoronavirus%2Bspike%2Bproteins%2Bin%2Bviral%2Bentry%2Band%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2000.0757%26rft_id%253Dinfo%253Apmid%252F11162792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.2000.0757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166695000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.19" data-doi="10.1016/j.virol.2004.09.033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKinlay</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gage</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ewart</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">SARS coronavirus E protein forms cation-selective ion channels</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">330</span>, <span class="cit-fpage">322</span>–<span class="cit-lpage">31</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DWilson%26rft.auinit1%253DL.%26rft.volume%253D330%26rft.issue%253D1%26rft.spage%253D322%26rft.epage%253D331%26rft.atitle%253DSARS%2Bcoronavirus%2BE%2Bprotein%2Bforms%2Bcation-selective%2Bion%2Bchannels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.09.033%26rft_id%253Dinfo%253Apmid%252F15527857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.09.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15527857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225195600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castaño-Rodriguez</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Role of Severe Acute Respiratory Syndrome Coronavirus Viroporins E, 3a, and 8a in Replication and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">9</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.21" data-doi="10.1016/j.jsb.2010.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A structural analysis of M protein in coronavirus assembly and morphology</span>. <abbr class="cit-jnl-abbrev">J. Struct. Biol</abbr>. <span class="cit-vol">174</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DNeuman%26rft.auinit1%253DB.%2BW.%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D22%26rft.atitle%253DA%2Bstructural%2Banalysis%2Bof%2BM%2Bprotein%2Bin%2Bcoronavirus%2Bassembly%2Band%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2010.11.021%26rft_id%253Dinfo%253Apmid%252F21130884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2010.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21130884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.22" data-doi="10.1016/S0065-3527(06)66005-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Masters</span>,  <span class="cit-name-given-names">P. S.</span></span> <span class="cit-article-title">The molecular biology of coronaviruses</span>. <abbr class="cit-jnl-abbrev">Adv. Virus Res</abbr>. <span class="cit-vol">66</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">292</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DMasters%26rft.auinit1%253DP.%2BS.%26rft.volume%253D66%26rft.spage%253D193%26rft.epage%253D292%26rft.atitle%253DThe%2Bmolecular%2Bbiology%2Bof%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252806%252966005-3%26rft_id%253Dinfo%253Apmid%252F16877062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(06)66005-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16877062&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240137400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.23" data-doi="10.1016/j.virusres.2007.02.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sparks</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>,  <span class="cit-name-given-names">L. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">A. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Denison</span>,  <span class="cit-name-given-names">M. R.</span></span> <span class="cit-article-title">SARS coronavirus replicase proteins in pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-vol">133</span>, <span class="cit-fpage">88</span>–<span class="cit-lpage">100</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGraham%26rft.auinit1%253DR.%2BL.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D100%26rft.atitle%253DSARS%2Bcoronavirus%2Breplicase%2Bproteins%2Bin%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.02.017%26rft_id%253Dinfo%253Apmid%252F17397959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.02.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17397959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.24" data-doi="10.1128/JVI.01048-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S. K. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">10532</span>–<span class="cit-lpage">47</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01048-15%26rft_id%253Dinfo%253Apmid%252F26269185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMC8xMDUzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE3LzIwMjAuMDMuMTUuOTkyNDM4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.25" data-doi="10.1128/JVI.01266-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaecher</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Touchette</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schriewer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buller</span>,  <span class="cit-name-given-names">R. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pekosz</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus gene 7 products contribute to virus-induced apoptosis</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">81</span>, <span class="cit-fpage">11054</span>–<span class="cit-lpage">68</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01266-07%26rft_id%253Dinfo%253Apmid%252F17686858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMC8xMTA1NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE3LzIwMjAuMDMuMTUuOTkyNDM4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.26" data-doi="10.1128/JVI.00090-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Y.-X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Induction of apoptosis by the severe acute respiratory syndrome coronavirus 7a protein is dependent on its interaction with the Bcl-XL protein</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">81</span>, <span class="cit-fpage">6346</span>–<span class="cit-lpage">55</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00090-07%26rft_id%253Dinfo%253Apmid%252F17428862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xMi82MzQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTcvMjAyMC4wMy4xNS45OTI0MzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paraskevis</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol</abbr>. <span class="cit-vol">79</span>, <span class="cit-fpage">104212</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol%26rft.volume%253D79%26rft.spage%253D104212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.28" data-doi="10.1128/JVI.01012-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frieman</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus ORF6 antagonizes STAT1 function by sequestering nuclear import factors on the rough endoplasmic reticulum/Golgi membrane</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">81</span>, <span class="cit-fpage">9812</span>–<span class="cit-lpage">24</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01012-07%26rft_id%253Dinfo%253Apmid%252F17596301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xOC85ODEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTcvMjAyMC4wMy4xNS45OTI0MzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.29" data-doi="10.1093/nar/gkn1004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mallick</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chakrabarti</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Cellular versus viral microRNAs in host-virus interaction</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">37</span>, <span class="cit-fpage">1035</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn1004%26rft_id%253Dinfo%253Apmid%252F19095692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn1004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19095692&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263962600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Piedade</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Azevedo-Pereira</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">The Role of microRNAs in the Pathogenesis of Herpesvirus Infection</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">8</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992438v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engelmann</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alidjinou</span>,  <span class="cit-name-given-names">E. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sane</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hober</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">miRNAs in enterovirus infection</span>. <abbr class="cit-jnl-abbrev">Crit. Rev. Microbiol</abbr>. <span class="cit-vol">44</span>, <span class="cit-fpage">701</span>–<span class="cit-lpage">714</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BMicrobiol%26rft.volume%253D44%26rft.spage%253D701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khatun</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ray</span>,  <span class="cit-name-given-names">R. B.</span></span> <span class="cit-article-title">Mechanisms Underlying Hepatitis C Virus-Associated Hepatic Fibrosis</span>. <abbr class="cit-jnl-abbrev">Cells</abbr> <span class="cit-vol">8</span>, (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishra</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingle</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">The Interplay Between Viral-Derived miRNAs and Host Immunity During Infection</span>. <abbr class="cit-jnl-abbrev">Front. Immunol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">3079</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BImmunol%26rft.volume%253D10%26rft.spage%253D3079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.34" data-doi="10.1038/cr.2018.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">H5N1 influenza virus-specific miRNA-like small RNA increases cytokine production and mouse mortality via targeting poly(rC)-binding protein 2</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">157</span>–<span class="cit-lpage">171</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D28%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2018.3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2018.3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.35" data-doi="10.1007/s11427-014-4759-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of Ebola virus microRNAs and their putative pathological function</span>. <abbr class="cit-jnl-abbrev">Sci. China. Life Sci</abbr>. <span class="cit-vol">57</span>, <span class="cit-fpage">973</span>–<span class="cit-lpage">81</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina.%2BLife%2BSci%26rft.volume%253D57%26rft.spage%253D973%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-014-4759-2%26rft_id%253Dinfo%253Apmid%252F25266153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-014-4759-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266153&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.36" data-doi="10.1186/1742-4690-6-18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klase</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">HIV-1 TAR miRNA protects against apoptosis by altering cellular gene expression</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">18</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRetrovirology%26rft.aulast%253DKlase%26rft.auinit1%253DZ.%26rft.volume%253D6%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DHIV-1%2BTAR%2BmiRNA%2Bprotects%2Bagainst%2Bapoptosis%2Bby%2Baltering%2Bcellular%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-6-18%26rft_id%253Dinfo%253Apmid%252F19220914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-6-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Up-regulation of microRNA-203 in influenza A virus infection inhibits viral replication by targeting</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">8</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.38" data-doi="10.1371/journal.pone.0007837"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mallick</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghosh</span>,  <span class="cit-name-given-names">Z.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chakrabarti</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">MicroRNome analysis unravels the molecular basis of SARS infection in bronchoalveolar stem cells</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e7837</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMallick%26rft.auinit1%253DB.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De7837%26rft.epage%253De7837%26rft.atitle%253DMicroRNome%2Banalysis%2Bunravels%2Bthe%2Bmolecular%2Bbasis%2Bof%2BSARS%2Binfection%2Bin%2Bbronchoalveolar%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007837%26rft_id%253Dinfo%253Apmid%252F19915717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19915717&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.39" data-doi="10.1111/j.1582-4934.2012.01572.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">Y.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cellular microRNA let-7c inhibits M1 protein expression of the H1N1 influenza A virus in infected human lung epithelial cells</span>. <abbr class="cit-jnl-abbrev">J. Cell. Mol. Med</abbr>. <span class="cit-vol">16</span>, <span class="cit-fpage">2539</span>–<span class="cit-lpage">46</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Band%2Bmolecular%2Bmedicine%26rft.stitle%253DJ%2BCell%2BMol%2BMed%26rft.aulast%253DMa%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D2539%26rft.epage%253D2546%26rft.atitle%253DCellular%2BmicroRNA%2Blet-7c%2Binhibits%2BM1%2Bprotein%2Bexpression%2Bof%2Bthe%2BH1N1%2Binfluenza%2BA%2Bvirus%2Bin%2Binfected%2Bhuman%2Blung%2Bepithelial%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1582-4934.2012.01572.x%26rft_id%253Dinfo%253Apmid%252F22452878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1582-4934.2012.01572.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22452878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992438.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.-H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phage display technique identifies the interaction of severe acute respiratory syndrome coronavirus open reading frame 6 protein with nuclear pore complex interacting protein NPIPB3 in modulating Type I interferon antagonism</span>. <abbr class="cit-jnl-abbrev">J. Microbiol. Immunol. Infect</abbr>. <span class="cit-vol">50</span>, <span class="cit-fpage">277</span>–<span class="cit-lpage">285</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMicrobiol.%2BImmunol.%2BInfect%26rft.volume%253D50%26rft.spage%253D277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.41" data-doi="10.1128/JVI.00619-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bandiera</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Hepatitis C Virus-Induced Upregulation of MicroRNA miR-146a-5p in Hepatocytes Promotes Viral Infection and Deregulates Metabolic Pathways Associated with Liver Disease Pathogenesis</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">90</span>, <span class="cit-fpage">6387</span>–<span class="cit-lpage">6400</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00619-16%26rft_id%253Dinfo%253Apmid%252F27147737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNC82Mzg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTcvMjAyMC4wMy4xNS45OTI0MzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.42" data-doi="10.1128/JVI.78.18.9904-9917.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montalto-Morrison</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Masters</span>,  <span class="cit-name-given-names">P. S.</span></span> <span class="cit-article-title">Genetic analysis of determinants for spike glycoprotein assembly into murine coronavirus virions: distinct roles for charge-rich and cysteine-rich regions of the endodomain</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">9904</span>–<span class="cit-lpage">17</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DYe%26rft.auinit1%253DR.%26rft.volume%253D78%26rft.issue%253D18%26rft.spage%253D9904%26rft.epage%253D9917%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BDeterminants%2Bfor%2BSpike%2BGlycoprotein%2BAssembly%2Binto%2BMurine%2BCoronavirus%2BVirions%253A%2BDistinct%2BRoles%2Bfor%2BCharge-Rich%2Band%2BCysteine-Rich%2BRegions%2Bof%2Bthe%2BEndodomain%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.18.9904-9917.2004%26rft_id%253Dinfo%253Apmid%252F15331724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xOC85OTA0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTcvMjAyMC4wMy4xNS45OTI0MzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.43" data-doi="10.1016/j.molmed.2016.11.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trobaugh</span>,  <span class="cit-name-given-names">D. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Klimstra</span>,  <span class="cit-name-given-names">W. B.</span></span> <span class="cit-article-title">MicroRNA Regulation of RNA Virus Replication and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">Trends Mol. Med</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">80</span>–<span class="cit-lpage">93</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMol.%2BMed%26rft.volume%253D23%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molmed.2016.11.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molmed.2016.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.44" data-doi="10.2174/1381612826666200109181238"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raza Naqvi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahmad</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valverde</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddiqui</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schaller</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Viral MicroRNAs: Interfering the Interferon Signaling</span>. <abbr class="cit-jnl-abbrev">Curr. Pharm. Des</abbr>. (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2174/1381612826666200109181238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BPharm.%2BDes%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1381612826666200109181238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1381612826666200109181238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herbert</span>,  <span class="cit-name-given-names">K. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nag</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">A Tale of Two RNAs during Viral Infection: How Viruses Antagonize mRNAs and Small Non-Coding RNAs in The Host Cell</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">8</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.-C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">L.-K.</span></span> <span class="cit-article-title">Role of TAF4 in transcriptional activation by Rta of Epstein-Barr Virus</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e54075</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abernathy</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Glaunsinger</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Emerging roles for RNA degradation in viral replication and antiviral defense</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">479–480</span>, <span class="cit-fpage">600</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.48" data-doi="10.1038/srep40813"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramalingam</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ziegelbauer</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Viral microRNAs Target a Gene Network, Inhibit STAT Activation, and Suppress Interferon Responses</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">40813</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D7%26rft.spage%253D40813%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep40813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep40813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.15.992438v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van der Ree</span>,  <span class="cit-name-given-names">M. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Safety, tolerability, and antiviral effect of RG-101 in patients with chronic hepatitis C: a phase 1B, double-blind, randomised controlled trial</span>. <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr> <span class="cit-vol">389</span>, <span class="cit-fpage">709</span>–<span class="cit-lpage">717</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D389%26rft.spage%253D709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.15.992438v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.15.992883v1" title="A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/742866v2" title="Bat coronavirus phylogeography in the Western Indian Ocean" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 17, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.15.992438v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.15.992438v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.15.992438v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.15.992438v1" class="active" data-icon-position="" data-hide-link-title="0">Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1199874" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_xNDN7KiNV4qVgKZEXjhNM9a24H9ndXkRkADYDuOCh8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1199874" id="node1199874" data-pisa="biorxiv;2020.03.15.992438v1" data-pisa-master="biorxiv;2020.03.15.992438" data-seqnum="1199874" data-apath="/biorxiv/early/2020/03/17/2020.03.15.992438.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Müşerref Duygu</span> <span class="nlm-surname">Saçar Demirci</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Aysun</span> <span class="nlm-surname">Adan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.15.992438; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.15.992438 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;title=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;title=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;text=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;title=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;t=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;title=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;title=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1199874" id="node1199874--2" data-pisa="biorxiv;2020.03.15.992438v1" data-pisa-master="biorxiv;2020.03.15.992438" data-seqnum="1199874" data-apath="/biorxiv/early/2020/03/17/2020.03.15.992438.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Computational analysis of microRNA-mediated interactions in SARS-CoV-2 infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Müşerref Duygu</span> <span class="nlm-surname">Saçar Demirci</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Aysun</span> <span class="nlm-surname">Adan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.15.992438; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.15.992438 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Sa%C3%A7ar+Demirci%2C+M%C3%BC%C5%9Ferref+Duygu&amp;rft.au=Adan%2C+Aysun"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1199874/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1199874/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1199874/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1199874/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1199874/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1199874/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1199874/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1199874/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1199874/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1199874/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1199874/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1199874/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;count=horizontal&amp;via=&amp;text=Computational%20analysis%20of%20microRNA-mediated%20interactions%20in%20SARS-CoV-2%20infection&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.15.992438v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.15.992438v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
