<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4405370">4405370</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25898324">25898324</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1004810">10.1371/journal.ppat.1004810</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-14-02332</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Spatial Structure, Transmission Modes and the Evolution of Viral Exploitation Strategies</div> <div class="alt-title" title="alt-title">
Spatial Structure Molds the Evolution of Viral Latency</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berngruber</span><span class="given-names" tagx="given-names" title="given-names">Thomas W.</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">Sébastien</span></span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">Sylvain</span></span><a href="#aff002"><sup>2</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Phage Technology Center GmbH, Bönen, Germany</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Centre d'Ecologie Fonctionnelle et Evolutive (CEFE), UMR 5175, CNRS-Université de Montpellier—Université Paul-Valéry Montpellier—EPHE. Montpellier, France</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">Sam Paul</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Edinburgh, UNITED KINGDOM</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: TWB SL SG. Performed the experiments: TWB. Analyzed the data: TWB SL SG. Wrote the paper: TWB SL SG.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">berngruber@gmail.com</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2015-4</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004810</span><span class="history" title="history"><span class="received" title="received">received: 2014-9-25</span><span class="accepted" title="accepted">accepted: 2015-3-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Berngruber et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div><a class="self-uri" href="ppat.1004810.pdf" title="self-uri">ppat.1004810.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Spatial structure and local migration are predicted to promote the evolution of less aggressive host exploitation strategies in horizontally transmitted pathogens. Here we explore the effect of spatial structure on the evolution of pathogens that can use both horizontal and vertical routes of transmission. First, we analyse theoretically how vertical transmission can alter evolutionary trajectories and confirm that space can impede the spread of virulent pathogens. Second, we test this prediction using the latent phage λ which transmits horizontally and vertically in <i>Escherichia coli</i> populations. We show that the latent phage λ wins competition against the virulent mutant λcI857 in spatially structured epidemics, but loses when spatial structure is eroded. The vertical transmission of phage λ immunizes its local host pool against superinfection and prevents the spread of the virulent λcI857. This effect breaks down when mixing facilitates horizontal transmission to uninfected hosts. We thus confirm the importance of spatial structure for the evolutionary maintenance of prudent infection strategies in latent viruses.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author Summary</div> <p>Why do some viruses use latent infection strategies and in which environments can such prudent host exploitation strategies evolve? Theory of mathematical epidemiology and social evolution predicts that prudent pathogens can evolve when they cluster in space and share the mutual benefits of prudent exploitation of the host population. Here we extend the theory to study the effect of spatial structure on the evolution of pathogens that can transmit both horizontally and vertically. We explore these effects experimentally by competing the latent bacteriophage λ and the virulent mutant λcI857 in spatially structured epidemics and gradually erode spatial structure. We show that the latent bacteriophage λ only wins in a spatially structured epidemic. Yet, a breakdown of the epidemic structure by long-range transmission reduces the benefit of latency by 500 fold. This demonstrates that long-rage transmission, as for example by air travel, might select for much more virulent types of previously latent viruses.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This research was funded by European Research Council starting grant 243054 ‘EVOLEPID’ (<a href="http://erc.europa.eu">http://erc.europa.eu</a>) to SG and Agence Nationale pour la Recherche grant 10-PDOC-017-01 ‘SPATEVOLEPID’ (<a href="http://www.agence-nationale-recherche.fr/">http://www.agence-nationale-recherche.fr/</a>) to SL. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant experimental data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant experimental data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>When individuals compete for a common resource natural selection often favours more aggressive exploitation strategies. This may lead to resource exhaustion and, consequently, to population extinction, a process known as the 'tragedy of the commons'[<a href="#ppat.1004810.ref001">1</a>]. Because pathogens compete for a common resource (the host population), the same process may select for extreme exploitation strategies. Yet, two main ecological factors can alter this evolutionary outcome and promote the evolution of less aggressive exploitation strategies.</p> <p>First, epidemic spread reduces the density of susceptible hosts and can feed back on the selective pressure that favours intermediate exploitation strategies [<a href="#ppat.1004810.ref002">2</a>–<a href="#ppat.1004810.ref005">5</a>]. Indeed, decreased availability of susceptible hosts weakens selection for higher transmission rates [<a href="#ppat.1004810.ref006">6</a>–<a href="#ppat.1004810.ref008">8</a>]. Ultimately, this may select for intermediate evolutionarily stable strategies, balancing the benefit (transmission) and the cost (virulence: induced host mortality) of host exploitation. We recently tested this idea using the bacteriophage λ in experimental epidemics spreading through well-mixed environments [<a href="#ppat.1004810.ref009">9</a>]. We confirmed that more virulent strains are indeed selected for during early epidemics, when uninfected hosts are abundant, but also that natural selection favours latent strains of the virus as disease prevalence increases.</p> <p>Second, theoretical studies on spatially structured populations indicate that localized transmission often favours more prudent host exploitation strategies [<a href="#ppat.1004810.ref010">10</a>–<a href="#ppat.1004810.ref021">21</a>]. Indeed, when epidemics are spatially structured, extreme strategies of host exploitation may lead to the over-exploitation of the local host supply and therefore fail to invade. The intensity of local competition for susceptible hosts depends, however, on the precise genetic and epidemiological structure of the pathogen population [<a href="#ppat.1004810.ref021">21</a>]. The effect of spatial structure on the evolution of pathogens has had some experimental support [<a href="#ppat.1004810.ref022">22</a>–<a href="#ppat.1004810.ref023">23</a>] which is, however, based on experiments with obligate killing and strictly horizontally transmitting pathogens. Previous experimental work has shown that vertical transmission selects for lower virulence [<a href="#ppat.1004810.ref024">24</a>–<a href="#ppat.1004810.ref025">25</a>] but the effect of spatial structure on vertically transmitted pathogens remains an open question. In the present study we explore the effect of spatial structure on the evolution of pathogens that can be transmitted both vertically and horizontally. We first develop and analyse a theoretical model to understand the evolutionary dynamics of pathogens in spatially structured epidemics (<a href="#ppat.1004810.g001">Fig 1</a>). This model allows infected hosts to reproduce and to transmit their pathogens vertically. We use this model to study the competition between different pathogen genotypes under various levels of mixing (see supporting information in <a href="#ppat.1004810.s001">S1 Text</a>). Then, we confront these predictions with experiments performed with the horizontally and vertically transmitting temperate bacteriophage λ. Bacteriophage λ is an avirulent pathogen that controls its own replication by the virulence repressor <i>cI</i> and integrates into the host genome as a prophage without killing the host cell—a process known as lysogenization. The virulence repressor <i>cI</i> furthermore controls prophage λ reactivation and excludes superinfection by a second viral particle [<a href="#ppat.1004810.ref026">26</a>]. Yet, spontaneous mutations in <i>cI</i> can markedly increase virulence [<a href="#ppat.1004810.ref027">27</a>]. We studied competition between temperate bacteriophage λ and its virulent mutant λcI857. Mutant λ<i>cI857</i> carries an unstable virulence repressor protein (cI857) that biases the viral life cycle towards horizontal transmission: whereas λ lysogenizes about 60% of newly infected cells killing only about 40% of cell, λcI857 kills 96% of host cells upon first infection (see Fig S2B in <a href="#ppat.1004810.s001">S1 Text</a>). In addition λcI857 shows a higher rate of prophage reactivation, sacrificing infected host cells for increased production of free viral particles (Fig S2A in <a href="#ppat.1004810.s001">S1 Text</a>). Furthermore, λcI857 is less efficient in the exclusion of superinfection (see Fig S2B in <a href="#ppat.1004810.s001">S1 Text</a>). Whereas the temperate λ wildtype shows an avirulent host exploitation strategy the virulent mutant λcI857 readily exploits and kills its host cell. To experimentally investigate the effect of spatial structure on the success of these opposing strategies we competed λ and λcI857 in experimental epidemics with different degrees of spatial structure (Manipulation of spatial structure is described in Material &amp;amp; Methods and [<a href="#ppat.1004810.ref028">28</a>]).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Schematic representation of the pathogen life cycle.</div> <p>In our model we assume that hosts can either be uninfected or infected by the wild type (in green) or a mutant genotype (in red). Transmission can take two different routes. Horizontal transmission occurs between a susceptible host and an infected host with genotype <i>i</i> with rate <i>β</i><sub><i>i</i></sub>[<i>I</i>|<i>S</i>] (see supporting information Theory in <a href="#ppat.1004810.s001">S1 Text</a>). Vertical transmission occurs when and infected host reproduces in an empty site with rate <i>b</i><sub><i>I</i></sub><i>δq</i><sub><i>0/i</i></sub>, where <i>b</i><sub><i>I</i></sub> is the reproduction rate of infected hosts, <i>δ</i> is the fidelity of vertical transmission and <i>q</i><sub><i>0/i</i></sub> measures the average local density of empty sites around hosts infected with genotype <i>i</i>. Among hosts infected by the same genotype we distinguish hosts that got infected horizontally (noted <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1004810.e001g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
H</div> </div> </div></div></div>) or vertically (noted <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1004810.e002g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
V</div> </div> </div></div></div></span>). Both types of hosts can reproduce and have an increased mortality rate <i>α</i><sub><i>i</i></sub> (i.e. virulence) due to the infection with genotype <i>i</i>.<p />   </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="theory" title="sec"> <div class="title" tagx="title" title="title">
Theory</div> <p>We study a spatial version of the compartmental model depicted in <a href="#ppat.1004810.g001">Fig 1</a>. This allows us to make predictions on the epidemiological and evolutionary consequences of spatial structure. First, <a href="#ppat.1004810.g002">Fig 2A</a> shows that lower levels of mixing slow down the spread of the infection because in a spatially structured environment, pathogens experience a lower density of susceptible hosts. Second, the change in frequency of a virulent mutant strain is (see Supporting Information Theory in <a href="#ppat.1004810.s001">S1 Text</a>): </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1004810.e003g" position="anchor" mimetype="image" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
w</div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mo_UNKNOWN">
︸</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="munder_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
w</div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> <div class="mspace_UNKNOWN" /> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
︸</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="munder_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mspace_UNKNOWN" /> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
︸</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
e</div> </div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></div><span class="label" tagx="label" title="label">(1)</span></div> The first term measures the contribution of the horizontal transmission route and depends on the local density of susceptible hosts in the neighbourhood of a host infected by a mutant or wild-type parasite The second term measures the contribution of vertical transmission and depends on the local density of empty sites experienced by hosts infected by each type of parasite. Indeed, a vertical transmission event is conditional upon a reproduction event and thus on the availability of empty sites in the neighbourhood. Finally, the last term measures the cost of virulence.<p /> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Effect of spatial structure on epidemiology and evolution in the model.</div> <p>The top figures (A) show simulation results for the density of susceptible (dashed line) and infected hosts (full line) for different level of mixing (left: no mixing (<i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 0), middle: intermediate (<i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 0.5), right: full mixing (<i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 1)). The figures in the middle (B) show simulation results for the change in mutant frequency in the total pathogen population (black), in the horizontally infected hosts (red) or in vertically infected hosts (blue) for different levels of mixing. The figures at the bottom (C) indicate the values of the different components of the selection coefficient identified in <a href="#ppat.1004810.e003">Eq (1)</a>: the horizontal transmission term (red), the vertical transmission component (blue) and the sum of the previous two terms (black) and the magnitude of the cost of virulence (dashed black). Note that the global mutant frequency increases only when the sum of the transmission terms (full black line) is higher than the cost of virulence (dashed line). Our simulations are the result of the numerical integration of the deterministic approximation of the full spatial model, using Improved Pair Approximation (IPA; [<a href="#ppat.1004810.ref031">31</a>]) with parameters <i>ϕ</i> = 1/6 and <i>θ</i> = 2/5, which correspond to a triangular lattice (each site has 6 neighbours, see [<a href="#ppat.1004810.ref031">31</a>] for details). Parameters: <i>α</i><sub><i>w</i></sub> = 0.1, <i>α</i><sub><i>m</i></sub> = 1, <i>β</i><sub><i>w</i></sub> = 2, <i>β</i><sub><i>m</i></sub> = 3.5, <i>δ</i> = 0.99, <i>d</i> = 0.01. Initial conditions: <i>S</i>(0) = 0.8, <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1004810.e004g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
H</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
H</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.01</div> </div></div></div>,<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1004810.e005g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
5</div> </div> </div> </div></div></div></span>.<p />   </div> <p><a href="#ppat.1004810.g002">Fig 2B</a> shows numerical simulations based on the pair approximation [<a href="#ppat.1004810.ref029">29</a>–<a href="#ppat.1004810.ref031">31</a>] under different levels of mixing (see supporting information Theory in <a href="#ppat.1004810.s001">S1 Text</a>). Note that, under all level of mixing, the frequency of the virulent mutant initially increases. Indeed, early on during the epidemics, parasites have access to a large density of susceptible hosts, which favours the spread of more transmissible mutants. This benefit, however, is only transient. As the pool of susceptible hosts is exhausted, more prudent pathogens end up being selected under all mixing treatments. Spatial structure, however, has a dramatic impact on the virulent mutant: less mixing drastically reduces the transient benefit of virulence. To understand this effect, we represent on <a href="#ppat.1004810.g002">Fig 2C</a> the effect of spatial structure on the different components of the selection coefficient in <a href="#ppat.1004810.e003">Eq (1)</a>. Spatial structure acts through two different effects: (i) space reduces the horizontal transmission benefit (red curve in <a href="#ppat.1004810.g002">Fig 2C</a>) and (ii) space generates benefit for the virulence mutant through vertical transmission (blue curve in <a href="#ppat.1004810.g002">Fig 2C</a>). The first effect is the classical effect of space often discussed as a self-shading or kin selection argument [<a href="#ppat.1004810.ref014">14</a>,<a href="#ppat.1004810.ref021">21</a>,<a href="#ppat.1004810.ref032">32</a>]. The second effect is due to the accumulation of empty sites near virulence mutants, which is caused by the death of hosts infected by related (and thus virulent) genotypes. In other words, virulence mutant have more opportunities to transmit vertically than avirulent genotypes.</p> <p>In addition, to better grasp the effect of space on pathogen evolution, we track the change in mutant frequency over two types of infected hosts: hosts that acquired the infection horizontally and hosts that acquired the infection vertically. <a href="#ppat.1004810.g002">Fig 2B</a> reveals the effect of mixing on the change in frequency of the mutant in these two compartments. In a well-mixed environment the frequency of the mutant is always higher in the horizontally infected hosts. In contrast, in a structured environment, the frequency of the virulent mutant can be higher in vertically infected hosts. This is due to the effect of spatial structure on the opportunities for vertical transmission. As explained above the virulent mutant has more opportunities for vertical transmission because the local density of empty sites is higher (see <a href="#ppat.1004810.g002">Fig 2B</a>). Another way to see this effect is to track the ratio W<sub>H</sub> / W<sub>V</sub> which provides a measure of the relative contribution of the two routes of transmission to mutant fitness. When this ratio is above 1 the change in mutant frequency is mainly governed by the horizontal transmission route. In contrast, when it is below one the vertical transmission route contributes more to the mutant fitness. <a href="#ppat.1004810.g003">Fig 3</a> shows that during the early stage of the epidemics horizontal transmission drives the evolution of the virulent mutant under all mixing treatments but that the contribution of vertical transmission is always increased by spatial structure.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Effect of mixing on the relative contribution of horizontal (W<sub>H</sub>) and vertical transmission (W<sub>V</sub>) to the fitness of the virulent mutant in the model.</div> <p>Each curve corresponds to different levels of mixing: no mixing <i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 0 (dotted line), intermediate mixing <i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 0.5 (dashed line) and full mixing with <i>g</i><sub><i>H</i></sub> = <i>g</i><sub><i>P</i></sub> = 1 (full line). The contribution of horizontal transmission is maximal in the early stage of the epidemic because many susceptible hosts are still available. More mixing increases the contribution of horizontal transmission to the fitness of the mutant throughout the epidemics.</p>   </div> </div> <div class="experiment" title="sec"> <div class="title" tagx="title" title="title">
Experiment</div> <p>The above model is an attempt to capture the interplay between the effects of spatial structure and vertical transmission on the evolution of pathogen virulence. To test these predictions experimentally we followed competition of the avirulent λ and virulent λcI857 in spatial epidemics after an initial inoculation of a 1:1 ratio of cells infected by λ or λcI857 onto a biofilm of susceptible cells. Thereupon spatial structure was either left <b><i>undisturbed</i></b> or disturbed for <b><i>30s</i></b> or <b><i>24h</i></b> in the absence of liquid, or 24h in the presence of liquid (<b><i>24h-wet</i></b>). By engineering fluorescent protein expression cassettes (CFP and YFP) into the viral genome we tagged infected cells by a fluorescent colour. This enabled us to directly visualize pathogen-host structure and spatial prevalence in these four environments (see <a href="#ppat.1004810.g004">Fig 4</a>). In the <b><i>undisturbed</i></b> treatment the epidemic spreads in a circular front segregating by viral types (λCFP or λYFP) but large areas remain uninfected (see <a href="#ppat.1004810.g004">Fig 4A</a>). In the <b><i>30s</i></b> disturbance treatment the epidemic spreads through the entire biofilm and creates epidemic clusters dominated by a single type of virus (<a href="#ppat.1004810.g004">Fig 4B</a>). After <b><i>24h</i></b> disturbance epidemic clusters disappear, but the biofilm remains structured (<a href="#ppat.1004810.g004">Fig 4C</a>). In the <b><i>24h-wet</i></b> treatment epidemic structure is completely homogenized (<a href="#ppat.1004810.g004">Fig 4D</a>). Thus, as expected from our model simulation (<a href="#ppat.1004810.g002">Fig 2A</a>), higher mixing is associated with a significant increase in overall prevalence of infection (χ<sub>1</sub><sup>2</sup> = 36.52, p&amp;lt;0.0001, Fig S3 in <a href="#ppat.1004810.s001">S1 Text</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Competition of fluorescently marked virus (λCFP and λYFP—blue and green areas) during invasion into uninfected cells (black areas) at different degrees of mixing (<i>undisturbed</i>, <i>30s</i>, <i>24h</i>, <i>24h-wet</i>).</div> <p><b>(A)</b><i>Undisturbed environment</i>—Circular epidemic front segregates into single-virus sectors (blue and green sectors, white arrow depicts the centre of epidemic inoculation). <b>(B)</b><i>30s disturbance</i>—Epidemic clusters that are dominated by a single type of virus appear. <b>(C,D)</b> Longer periods of disturbance (<i>24h</i>) and a liquid surface layer (<i>24h-wet</i>) progressively homogenize the spatial genetic structure of virus types.</p>   </div> <p>The mixing treatment strongly affects competitive fitness <i>W</i> of λcI857 (χ<sub>1</sub><sup>2</sup> = 79.15, p&amp;lt;0.0001, <a href="#ppat.1004810.g005">Fig 5</a>). In the <b><i>undisturbed</i></b> environment λcI857 fitness decreases by ~100 fold after the first transfer day, in contrast in the <b><i>24h-wet</i></b> environment λcI857 fitness increases by ~5 fold. As expected, intermediate spatial disturbance treatments (<b><i>30s</i></b> and <b><i>24h</i></b>) yield intermediate levels of λcI857 fitness (<a href="#ppat.1004810.g005">Fig 5</a>). This demonstrates that erosion of the spatial structure increases the fitness of the virulent λcI857 by ~500 fold. Note that additional transfers did not affect this pattern except for the treatment 30s where λcI857 fitness increases with time (<i>t</i> = 5.08, p&amp;lt;10–4).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Mixing selects for virulence.</div> <p>A 1:1 mixture of temperate λ and its virulent mutant λcI857 was used to seed spatial epidemics. Spatial structure was disturbed to different degree (<b><i>undisturbed</i>, <i>30s</i>, <i>24h</i>, <i>24h-wet</i></b>) and transferred onto a fresh biofilm of susceptible hosts for 3 consecutive days. Disturbance of the epidemic structure has a strong effect on the competitive fitness of the virulent λcI857. The <b><i>undisturbed</i></b> environment strongly selects against virulence whereas in a well-mixed environment (<b><i>24h-wet</i></b>) virulence is beneficial. As expected, intermediate levels of mixing (<b><i>30s</i>, <i>24h</i></b>) lead to intermediate fitness of λcI857. Symbols represent 6 replications for each mixing treatment (with 3 independent replicates for each marker-mutant combination).</p>   </div> <p>To disentangle the contribution of horizontal and vertical transmission to the fitness of λcI857 we repeated the experiment for the first round of spatial transfer with uninfected cells marked by the red fluorescent protein mCherry. This way we identified new infections from horizontal transmission as doubly coloured cells (mCherry and colour of the new infecting virus) and vertical transmission from the initial lysogen as singly coloured cells (colour of the originally integrated virus). From these data we calculated the competitive fitness of λcI857 among all new infections (horizontal fitness W<sub>H</sub>) and fitness among all vertical transmission events (vertical fitness W<sub>V</sub>) in each spatial environment. <a href="#ppat.1004810.g006">Fig 6</a> presents the effect of mixing on the ratio W<sub>H</sub> / W<sub>V</sub> between horizontal and vertical fitness components of λcI857. In the absence of mixing both routes of transmission contribute to λcI857 fitness. Interestingly, in agreement with our theoretical prediction (<a href="#ppat.1004810.g002">Fig 2C</a>), we found that higher levels of mixing promote horizontal transmission (χ<sub>1</sub><sup>2</sup> = 14,82, p = 0.002). For instance, in the highly mixed environment (<b><i>24h-wet</i></b> treatment), 3 out of 4 new infections result from the horizontal transmission route (<a href="#ppat.1004810.g006">Fig 6</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1004810.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Effect of mixing on the relative contribution of horizontal (W<sub>H</sub>) and vertical transmission (W<sub>V</sub>) to the fitness of the virulent λcI857.</div> <p>More mixing (from <b><i>undisturbed</i>, <i>30s</i>, <i>24h</i></b> to <b><i>24h-wet</i></b>) increases the fitness contribution of horizontal transmission to the competitive ability of λcI857 (W<sub>H</sub>) relative to the fitness contribution of vertical transmission (W<sub>V</sub>) (see <a href="#ppat.1004810.g003">Fig 3</a>). Each box represents the median, the first and the third quartiles. Dashed lines delimit 1.5 times the inter-quartile range, above which individual counts are considered outliers and marked as empty circles.</p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>During an epidemic, disease prevalence increases, and the opportunities to find uninfected hosts dwindle. This may select for prudent exploitation of the remaining host population. In a previous study, we demonstrated the impact of such epidemiological feedback on the evolution of bacteriophage λ [<a href="#ppat.1004810.ref009">9</a>]. Here, we use the same experimental system to study the effect of spatial structure on the evolution of bacteriophage λ. To account for latency and vertical transmission in the life cycle of bacteriophage λ, we extend previous theory on the effect of spatial structure on pathogen evolution by allowing for vertical transmission of the pathogen. Our model tries to capture all the details of the bacteriophage λ life-cycle, but for tractability we do not explicitly model the free living stage of the virus. Our central result is that the fitness of a virulent strain relative to a more prudent strain decreases as the environment becomes increasingly structured. A general insight of previous theory is that the outcome of selection on parasite life-history traits depends on the balance between the genetic and epidemiological structures of the parasite population [<a href="#ppat.1004810.ref021">21</a>]. At the genetic level, parasites infecting a cluster of hosts tend to be related when transmission occurs locally. Hence, competition for the local host supply tends to take place among related pathogens (kin competition) [<a href="#ppat.1004810.ref018">18</a>,<a href="#ppat.1004810.ref021">21</a>,<a href="#ppat.1004810.ref032">32</a>]. At the epidemiological level, local pathogen spread reduces the local density of uninfected hosts, thereby degrading the parasite's local environment and diminishing future opportunities for horizontal transmission [<a href="#ppat.1004810.ref005">5</a>,<a href="#ppat.1004810.ref014">14</a>]. From models without vertical transmission, it has been shown that more prudent strategies of host exploitation are selected for if the genetic structure of the parasite population is above a threshold determined by the epidemiological structure of the host population [<a href="#ppat.1004810.ref033">33</a>]. As interactions become more localized, genetic structure increases and the balance is tilted at the advantage of more prudent strains [<a href="#ppat.1004810.ref021">21</a>]. We show that vertical transmission does not alter the general theoretical expectation that spatial structure should select for more prudent host exploitation strategies. Yet our analysis reveals that in a spatially structured environment vertical transmission can affect evolutionary dynamics via two main effects. First, vertical transmission affects opportunities of horizontal transmission through a modification of the local density of susceptible hosts (the first term in <a href="#ppat.1004810.e003">Eq (1)</a>). Second, vertical transmission generates a force (the second term in <a href="#ppat.1004810.e003">Eq (1)</a>) that may counteract the effect of spatial structure. During the early stage of the epidemic this effect remains very low but further work is required to investigate the impact of the effects of vertical transmission on evolutionary stable virulence strategies. Our model is useful to decouple the change in mutant frequency in horizontally and vertically infected hosts. This distinction between different host types is artificial but it helps to grasp the interplay between spatial structure and transmission routes. As in other situations where the pathogen can appear in different states [<a href="#ppat.1004810.ref009">9</a>] or in different hosts [<a href="#ppat.1004810.ref008">8</a>, <a href="#ppat.1004810.ref034">34</a>–<a href="#ppat.1004810.ref036">36</a>], tracking the change in frequency between different compartments enables us to generate a prediction regarding the effect of spatial structure on the relative contribution of the two transmission routes on pathogen fitness (<a href="#ppat.1004810.g002">Fig 2C</a>).</p> <p>This new theoretical model generates predictions that we tested experimentally with bacteriophage λ. Unlike our previous experimental study in chemostats [<a href="#ppat.1004810.ref009">9</a>], the present experiment has been realized on agar plates to manipulate the amount of spatial structuring. Yet, on a normal agar plate bacterial growth eventually enters stationary phase and the epidemiological and evolutionary dynamics halt. Consequently, our experiments allow us to test predictions during the initial phase of the epidemics only. During this phase we made three qualitative predictions that we verified experimentally. First, we confirmed that lower levels of mixing reduce the speed of the epidemic. Second, we confirmed that a virulent strain is outcompeted by a more prudent strain when the environment becomes increasingly structured. This is our main result and it agrees well with previous experimental studies realized with horizontally transmitted pathogens [<a href="#ppat.1004810.ref022">22</a>–<a href="#ppat.1004810.ref023">23</a>]. Third, our experimental setup allows monitoring the relative contribution of horizontal and vertical transmission among new infections and confirms that mixing decreases the contribution of vertical transmission to pathogen fitness.</p> <p>The evolution of host exploitation strategies can often be explained at the level of within-host selection. For parasites competing for within-host resources, more prudent host exploitation should be selected for when the probability of multiple infections decreases [<a href="#ppat.1004810.ref004">4</a>]. In our model, superinfection exclusion by a latent λ prophage guarantees that only related pathogen individuals compete within the same host cell and, hence, limits selection for increased replication and virulence at the within-host level. Similarly, many covertly infecting and genome-integrating viral pathogens take within- host competition out of the equation by active exclusion of superinfection. In herpes and retroviruses, for example, superinfection exclusion is well described and might be a key requirement for the maintenance of viral latency [<a href="#ppat.1004810.ref037">37</a>–<a href="#ppat.1004810.ref039">39</a>]. Yet some mutations are known to enable λ to break through superinfection exclusion [<a href="#ppat.1004810.ref026">26</a>]. We did not observe such mutants in our present experiments, but we were able to extend our theory to explore the effect of spatial structure on the epidemiology and evolution of such virulence mutants that can superinfect already infected bacteria (see supporting information Theory in <a href="#ppat.1004810.s001">S1 Text</a>). This analysis shows that mixing does affect the epidemiology and evolution in this alternative scenario: more mixing enhances the spread of the virulence mutant. Hence, this analysis suggests that the effect of spatial structure on the transient dynamics we report in the present study is robust to this modification of the life cycle. We believe that similar predictions may be feasible to understand and predict the epidemiology and evolution of a broad range of pathogens with complex transmission modes.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="theory" title="sec"> <div class="title" tagx="title" title="title">
Theory</div> <p>The theoretical analysis is presented in the supporting information (<a href="#ppat.1004810.s001">S1 Text</a>). We examine the evolutionary dynamics of a vertically and horizontally transmitted pathogen in a spatially structured environment. The analytical model reveals that the change in frequency can be decomposed in different terms that characterize the local environment of competing variants of the pathogens. These terms can be tracked in numerical simulations (<a href="#ppat.1004810.g002">Fig 2</a>). Further understanding of the evolutionary dynamics can be obtained by focusing on the change in frequency of a mutant pathogen in hosts that have acquired the pathogen horizontally (<i>I</i><sub><i>H</i></sub>) and in hosts that have acquired the pathogen vertically (<i>I</i><sub><i>V</i></sub>).</p> </div> <div class="experiments" title="sec"> <div class="title" tagx="title" title="title">
Experiments</div> <p>Host exploitation strategies of λ and λcI857 were quantified by three independent assays. <b>(1) Virus production (PFU/mL)</b>: Lysogens were grown to OD<sub>600nm</sub> = 0.6 at 30°C and shifted to 35°C for 2h until lysis occurred. Viral titers were determined by qPCR (Roche LightCycler480 primers F:5’<tt>AATGAAGGCAGGAAGTA</tt>3’ R:5’<tt>GCTTTCCATTCCATCGG</tt>3’) on filtered lysates (Millipore 96-well filtration plate Multiscreen HTS). CP values were calibrated to PFU/ml by top-agar plating a dilution series of a λvir (3x10<sup>9</sup> pfu). <b>(2) Growth of infected hosts (CFU/mL)</b>: Lysogens were diluted to OD<sub>600nm</sub> = 0.07 and grown for 6h at 35°C in eight replicates in 96-well plates (900 rpm, Titramax shaker (Heidolph, Germany)). Final OD<sub>600nm</sub> was measured in an Infinity200 microplate reader (Tecan, Austria). OD<sub>600nm</sub> values were calibrated to CFU/ml by plating a dilution curve. <b>(3) Lysogenization rate</b> was determined by challenging non-infected E.coli MG1655 of OD<sub>600nm</sub> = 0.1 with 10<sup>8</sup> PFU/mL free virus particles of λCFP, λYFP, λcI857CFP and λcI857YFP for 24h. After 24h, the proportion of lysogenized (fluorescent) cells was determined by flow cytometry (Fig S2 in <a href="#ppat.1004810.s001">S1 Text</a>).</p> <p>Fluorescently labelled phage were constructed as described in [<a href="#ppat.1004810.ref009">9</a>]. Spatial epidemics were started by spreading a 10<sup>8</sup> cfu/ml suspension of uninfected E.coli MG1655 (starved 24h in 0.2%Maltose, 10mM MgSO<sub>4</sub>) onto agar plates until dried. Thereafter a 1:1 mixture of cells infected by λ and λcI857 was inoculated in localized spots by an array of 0.4mm stainless steel tattooing needles at about 4mm distance. Epidemic structure was disturbed by rolling 4mm sterile stainless steel beads over the dry agar surface for <i>30s</i> or <i>24h</i> or over agar surface humidified by sterile saline solution (<i>24h-wet</i>) or kept <i>undisturbed</i>. Agar plates were incubated at 35°C in agitation (250rpm), but only in the <i>24h</i> and <i>24h-wet</i> treatment steel beads remained on the agar surface. To control for marker effects all competitions were carried out with reciprocal marker-mutant combinations (λYFP vs. λcI857CFP and λCFP vs. λcI857YFP). Spatial structure was photographed on an Olympus BH-2 RCF microscope with a 2x fluorite objective and filter cube #69380 (Chroma Thechnology Corp., VT, USA). For the daily transfer of the epidemic structure the needle array described above was poked into plates that had been incubated for 24 hours and subsequently the needle array was poked into agar plates containing a layer of uninfected cells as described above. For flowcytometry the bacterial layer was washed off with 2 ml saline and analysed on a B&amp;amp;D Fortessa flowcytometer (Excitation CFP, YFP, mCherry at 405, 488 and 561 nm).</p> <p><b>Horizontal and vertical transmission</b> was estimated by repeating the spatial epidemics for a single cycle with lysogens of λCFP, λYFP, λcI857CFP and λcI857YFP carrying pBAD18 and uninfected E.coli MG1655 carrying plasmid pRSET-mCherry. 20μg/ml Ampicillin was added to the medium to prevent spontaneous plasmid loss. Horizontal transmission events were scored by double coloration (mCherry+CFP and mCherry+YFP). Horizontal and vertical fitness were calculated as mentioned above by only considering mCherry+ or mCherry- cells.</p> <p><b>The relative fitness of λcI857</b> was calculated as <i>W</i> = (<i>f</i><sub>t</sub> /(1- <i>f</i><sub>t</sub>))/ (<i>f</i><sub>0</sub> /(1- <i>f</i><sub>0</sub>)) where <i>f</i><sub>t</sub> measures the frequency at time <i>t</i>. In general the frequency was calculated over the whole host population. In <a href="#ppat.1004810.g006">Fig 6</a> we distinguish horizontal fitness, W<sub>H</sub>, and vertical fitness, W<sub>V</sub>, by focusing either on the frequency in newly infected bacteria (bacteria carrying the plasmid pRSET-mCherry) or on the frequency in previously infected bacteria (bacteria that do not carry the plasmid pRSET-mCherry), respectively. In <a href="#ppat.1004810.g004">Fig 4</a> we present the ratio W<sub>H</sub> / W<sub>V</sub> for different mixing treatments.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Analyses were carried out using the R statistical package (version 3.0.2). The fitness W of λcI857 (<a href="#ppat.1004810.g005">Fig 5</a>, <a href="#ppat.1004810.s002">S1 Data</a>) was log transformed prior to fitting in a mixed effects model (lme, nlme package). Mixing (undisturbed, 30 s, 24h and 24h-wet) and transfer day (1, 2 and 3) were fitted as fixed effects while population was nested within the marker effect (the two reciprocal marker-mutant combinations) to account for the repeated measurements at different points in time. The ratio W<sub>H</sub> / W<sub>V</sub> (<a href="#ppat.1004810.g006">Fig 6</a>, <a href="#ppat.1004810.s002">S1 Data</a>) was fitted in a mixed effect model with mixing as a fixed factor and marker as a random effect. The prevalence of the infection (Fig S3 in <a href="#ppat.1004810.s002">S1 Text</a> and <a href="#ppat.1004810.s002">S1 Data</a>) was fitted in a mixed effect model with mixing as a fixed factor and marker colour as a random effect. Maximal models, including all higher-order interactions, were simplified by sequentially eliminating non-significant terms and interactions to establish a minimal model. The significance of explanatory variables in mixed effect models was established using a likelihood ratio test which is approximately distributed as a chi-square distribution [<a href="#ppat.1004810.ref040">40</a>]. A posteriori contrasts were carried out at the final time point of the experiments in Figs <a href="#ppat.1004810.g005">5</a>, <a href="#ppat.1004810.g006">6</a> and S3 (in <a href="#ppat.1004810.s001">S1 Text</a>) by aggregating factor levels together and by testing the fit of the simplified model using a likelihood ratio test [<a href="#ppat.1004810.ref041">41</a>].</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Text</span> <div class="title" tagx="title" title="title">
Theoretical appendix, extended simulation results and supporting experimental results.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1004810.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Data</span> <div class="title" tagx="title" title="title">
Experimental data for Figs <a href="#ppat.1004810.g005">5</a>, <a href="#ppat.1004810.g006">6</a> and S3 (in <a href="#ppat.1004810.s001">S1 Text</a>).</div> <p>(TXT)</p>  <div class="media" title="media"><a href="ppat.1004810.s002.txt">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Francois Gatchitch for technical assistance and the flow cytometry platform of <i>RIO Imaging Montpellier</i> for technical support.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1004810.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardin</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">1968</span>). <span class="mixed-article-title" title="mixed-article-title">The Tragedy of the Commons. The population problem has no technical solution; It Requires a Fundamental Extension in Morality</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">162</span>(<span class="issue" tagx="issue" title="issue">3859</span>): <span class="fpage" tagx="fpage" title="fpage">1243</span>–<span class="lpage" tagx="lpage" title="lpage">1248</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17756331">17756331</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary Epidemiology and the Dynamics of Adaptation</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">63</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">826</span>–<span class="lpage" tagx="lpage" title="lpage">838</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2009.00609.x">10.1111/j.1558-5646.2009.00609.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19154387">19154387</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span> (<span class="year" tagx="year" title="year">1982</span>). <span class="mixed-article-title" title="mixed-article-title">Coevolution of hosts and parasites</span>. <span class="source" tagx="source" title="source">Parasitology</span><span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">411</span>–<span class="lpage" tagx="lpage" title="lpage">426</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6755367">6755367</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frank</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Models of parasite virulence</span>. <span class="source" tagx="source" title="source">The Q. Rev. Biol</span>. <span class="volume" tagx="volume" title="volume">71</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">37</span>–<span class="lpage" tagx="lpage" title="lpage">78</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8919665">8919665</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dieckmann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metz</span><span class="given-names" tagx="given-names" title="given-names">J.A.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabelis</span><span class="given-names" tagx="given-names" title="given-names">M.W.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sigmund</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="source" tagx="source" title="source">Adaptive Dynamics of Infectious Diseases: In Pursuit of Virulence Management</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge MA</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenski</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">The evolution of virulence in parasites and pathogens: Reconciliation between two competing hypotheses</span>. <span class="source" tagx="source" title="source">J. Theor. Biol</span>. <span class="volume" tagx="volume" title="volume">169</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">253</span>–<span class="lpage" tagx="lpage" title="lpage">265</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7967617">7967617</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proulx</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">A general theory for the evolutionary dynamics of virulence</span>. <span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">163</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">E40</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15122509">15122509</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Applying population-genetic models in theoretical evolutionary epidemiology</span>. <span class="source" tagx="source" title="source">Ecol. Letters</span><span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">876</span>–<span class="lpage" tagx="lpage" title="lpage">888</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17845288">17845288</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berngruber</span><span class="given-names" tagx="given-names" title="given-names">T.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froissart</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choisy</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of virulence in emerging epidemics</span>. <span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">e1003209</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003209">10.1371/journal.ppat.1003209</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23516359">23516359</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boerlijst</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamers</span><span class="given-names" tagx="given-names" title="given-names">M.E.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hogeweg</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary consequences of spiral waves in a host parasitoid system</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B</span>. <span class="volume" tagx="volume" title="volume">253</span>: <span class="fpage" tagx="fpage" title="fpage">15</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuda</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>(<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Pathogen invasion and host extinction in lattice structured populations</span>. <span class="source" tagx="source" title="source">J. Math. Biol</span>. <span class="volume" tagx="volume" title="volume">32</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">251</span>–<span class="lpage" tagx="lpage" title="lpage">268</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8182357">8182357</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabelis</span><span class="given-names" tagx="given-names" title="given-names">M.W.</span></span>(<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">The milker-killer dilemma in spatially structured predator-prey interactions</span>. <span class="source" tagx="source" title="source">Oikos</span><span class="volume" tagx="volume" title="volume">74</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">391</span>–<span class="lpage" tagx="lpage" title="lpage">400</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rand</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">H.B.</span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Invasion, stability and evolution to criticality in spatially extended, artificial host-pathogen ecologies</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B.</span>: <span class="volume" tagx="volume" title="volume">259</span>(<span class="issue" tagx="issue" title="issue">1354</span>): <span class="fpage" tagx="fpage" title="fpage">55</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Small worlds’ and the evolution of virulence: Infection occurs locally and at a distance</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B.</span>: <span class="volume" tagx="volume" title="volume">266</span>(<span class="issue" tagx="issue" title="issue">1432</span>): <span class="fpage" tagx="fpage" title="fpage">1933</span>–<span class="lpage" tagx="lpage" title="lpage">1938</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10584335">10584335</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">The effects of local spatial structure on epidemiological invasions</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B</span>. <span class="volume" tagx="volume" title="volume">266</span>(<span class="issue" tagx="issue" title="issue">1421</span>): <span class="fpage" tagx="fpage" title="fpage">859</span>–<span class="lpage" tagx="lpage" title="lpage">867</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10343409">10343409</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haraguchi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The evolution of parasite virulence and transmission rate in a spatially structured population</span>. <span class="source" tagx="source" title="source">J. Theor. Biol</span>. <span class="volume" tagx="volume" title="volume">203</span>: <span class="fpage" tagx="fpage" title="fpage">85</span>–<span class="lpage" tagx="lpage" title="lpage">96</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10704294">10704294</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">C.R.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boerlijst</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Selection at the level of the community: The importance of spatial structure</span>. <span class="source" tagx="source" title="source">TREE</span><span class="volume" tagx="volume" title="volume">17</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">83</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Contact networks and the evolution of virulence</span>, in <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dieckmann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metz</span><span class="given-names" tagx="given-names" title="given-names">J. A. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabelis</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sigmund</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span> (eds) <span class="source" tagx="source" title="source">Adaptive Dynamics of Infectious Diseases</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Read</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Disease evolution on networks: The role of contact structure</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B.</span>: <span class="volume" tagx="volume" title="volume">270</span>(<span class="issue" tagx="issue" title="issue">1516</span>): <span class="fpage" tagx="fpage" title="fpage">699</span>–<span class="lpage" tagx="lpage" title="lpage">708</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12713743">12713743</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Self-structuring in spatial evolutionary ecology</span>. <span class="source" tagx="source" title="source">Ecol. Letters</span><span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">277</span>–<span class="lpage" tagx="lpage" title="lpage">295</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18070102">18070102</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Are parasites prudent in space?�?</span><span class="source" tagx="source" title="source">Ecol. Letters</span><span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">1245</span>–<span class="lpage" tagx="lpage" title="lpage">1255</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2010.01516.x">10.1111/j.1461-0248.2010.01516.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20727004">20727004</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerr</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuhauser</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohannan</span><span class="given-names" tagx="given-names" title="given-names">B. J. M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dean</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Local migration promotes competitive restraint in a host-pathogen ‘Tragedy of the Commons’</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">442</span>(<span class="issue" tagx="issue" title="issue">7098</span>): <span class="fpage" tagx="fpage" title="fpage">75</span>–<span class="lpage" tagx="lpage" title="lpage">78</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823452">16823452</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mealor</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Local interactions select for lower pathogen infectivity</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">315</span>(<span class="issue" tagx="issue" title="issue">5816</span>): <span class="fpage" tagx="fpage" title="fpage">1284</span>–<span class="lpage" tagx="lpage" title="lpage">1286</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332415">17332415</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messenger</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molineux</span><span class="given-names" tagx="given-names" title="given-names">I.J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bull</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Virulence evolution in a virus obeys a trade-off</span>. <span class="source" tagx="source" title="source">Proc Biol Sci</span><span class="volume" tagx="volume" title="volume">266</span>, <span class="fpage" tagx="fpage" title="fpage">397</span>–<span class="lpage" tagx="lpage" title="lpage">404</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10097397">10097397</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Vertical transmission selects for reduced virulence in a plant virus and for increased resistance in the host</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">e1004293</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004293">10.1371/journal.ppat.1004293</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25077948">25077948</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ptashne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. (<span class="year" tagx="year" title="year">1992</span>) <span class="source" tagx="source" title="source">A Genetic Switch: Phage Lambda and Higher Organisms</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Blackwell Publishers</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Refardt</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rainey</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span><span class="mixed-article-title" title="mixed-article-title">Tuning a genetic switch: Experimental evolution and natural variation of prophage induction</span>. <span class="source" tagx="source" title="source">Evolution</span> (<span class="year" tagx="year" title="year">2010</span>) <span class="volume" tagx="volume" title="volume">64</span>:<span class="fpage" tagx="fpage" title="fpage">1086</span>–<span class="lpage" tagx="lpage" title="lpage">1097</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2009.00882.x">10.1111/j.1558-5646.2009.00882.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19891623">19891623</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berngruber</span><span class="given-names" tagx="given-names" title="given-names">T.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of suicide as a defence strategy against pathogens in a spatially structured environment</span>. <span class="source" tagx="source" title="source">Ecol. Lett</span>. <span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">446</span>–<span class="lpage" tagx="lpage" title="lpage">53</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/ele.12064">10.1111/ele.12064</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23331662">23331662</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuda</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogita</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">K</span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Statistical mechanics of population: The lattice Lotka-Volterra model</span>. <span class="source" tagx="source" title="source">Progress in Theoretical Physics</span><span class="volume" tagx="volume" title="volume">88</span>:<span class="fpage" tagx="fpage" title="fpage">1035</span>–<span class="lpage" tagx="lpage" title="lpage">1049</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rand</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Correlation equations and pair approximations for spatial ecologies</span> In: <span class="source" tagx="source" title="source">Advanced Ecological Theory: Advances in Principles and Applications</span>, ed. <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGlade</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>., p. <span class="fpage" tagx="fpage" title="fpage">100</span>–<span class="lpage" tagx="lpage" title="lpage">142</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Blackwell</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford UK</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Pair Approximations for Different Spatial Geometries</span> In: <span class="source" tagx="source" title="source">The Geometry of Ecological Interactions: Simplifying Spatial Complexity</span>, eds. <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dieckmann</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Law</span><span class="given-names" tagx="given-names" title="given-names">R</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metz</span><span class="given-names" tagx="given-names" title="given-names">JAJ</span></span>, p. <span class="fpage" tagx="fpage" title="fpage">359</span>–<span class="lpage" tagx="lpage" title="lpage">387</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wild</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardner</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">West</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Adaptation and the evolution of parasite virulence in a connected world</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">459</span>: <span class="fpage" tagx="fpage" title="fpage">983</span>–<span class="lpage" tagx="lpage" title="lpage">986</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature08071">10.1038/nature08071</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19474791">19474791</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of spatially structured host—parasite interactions</span>. <span class="source" tagx="source" title="source">J. Evol. Biol</span>. <span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">10</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/jeb.12551">10.1111/jeb.12551</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25439133">25439133</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="chapter-title" tagx="chapter-title" title="chapter-title">Insights from Price’s equation into evolutionary epidemiology</span> In: <span class="source" tagx="source" title="source">Disease evolution: models, concepts and data analyses</span>. eds. <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dieckmann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Mathematical Society</span>, p. <span class="fpage" tagx="fpage" title="fpage">23</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.</span></li> <li tag="ref"><a name="ppat.1004810.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">The evolutionary epidemiology of vaccination</span>. <span class="source" tagx="source" title="source">J R Soc Interface</span><span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">803</span>–<span class="lpage" tagx="lpage" title="lpage">817</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17264055">17264055</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lélu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langlais</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulle</span><span class="given-names" tagx="given-names" title="given-names">M.-L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilot-Fromont</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">When should a trophically and vertically transmitted parasite manipulate its intermediate host? The case of Toxoplasma gondii</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society of London B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">280</span>, <span class="fpage" tagx="fpage" title="fpage">20131143</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2013.1143">10.1098/rspb.2013.1143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23825211">23825211</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mador</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panet</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steiner</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>. <span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">The latency-associated gene of herpes simplex virus type 1 (HSV-1) interferes with superinfection by HSV-1</span>. <span class="source" tagx="source" title="source">J. Neurovirol</span>. <span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">97</span>–<span class="lpage" tagx="lpage" title="lpage">102</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12491159">12491159</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nethe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berkhout</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Kuyl</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>. <span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Retroviral superinfection resistance</span>. <span class="source" tagx="source" title="source">Retrovirology</span><span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">52</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16107223">16107223</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berngruber</span><span class="given-names" tagx="given-names" title="given-names">T.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissing</span><span class="given-names" tagx="given-names" title="given-names">F.J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Inhibition of superinfection and the evolution of viral latency</span>. <span class="source" tagx="source" title="source">J. Virol</span>. <span class="volume" tagx="volume" title="volume">84</span>(<span class="issue" tagx="issue" title="issue">19</span>): <span class="fpage" tagx="fpage" title="fpage">10200</span>–<span class="lpage" tagx="lpage" title="lpage">10208</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00865-10">10.1128/JVI.00865-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20660193">20660193</a></span></span></li> <li tag="ref"><a name="ppat.1004810.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolker</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="source" tagx="source" title="source">Ecological Models and Data in R</span> (<span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton University Press</span>).</span></li> <li tag="ref"><a name="ppat.1004810.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawley</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="source" tagx="source" title="source">The R Book</span>, <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken, NJ</span>.</span></li> </ul> </div> </div>  </body></html>