<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6221319">6221319</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0206926">10.1371/journal.pone.0206926</a></span><span class="publisher-id" title="publisher-id">PONE-D-18-14755</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Transmission and Infection</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Henipavirus</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Henipavirus</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Henipavirus</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Henipavirus</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Paramyxoviruses</span>: <span class="subject" title="subject">Hendra Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bunyaviruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bunyaviruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bunyaviruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bunyaviruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Hemorrhagic fever viruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Hemorrhagic fever viruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Hemorrhagic fever viruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Hemorrhagic fever viruses</span>: <span class="subject" title="subject">Crimean-Congo hemorrhagic fever virus</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span>: <span class="subject" title="subject">Zoonoses</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Engineering and Technology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Management Engineering</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Decision Analysis</span>: <span class="subject" title="subject">Decision Trees</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Decision Analysis</span>: <span class="subject" title="subject">Decision Trees</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Transmissibility of emerging viral zoonoses</div> <div class="alt-title" title="alt-title">
Transmissibility of emerging viral zoonoses</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6130-8189</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">Joseph W.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Barbara A.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ott</span><span class="given-names" tagx="given-names" title="given-names">Isabel M.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">John M.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Odum School of Ecology, University of Georgia, Athens, Georgia, United States of America</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Center for the Ecology of Infectious Diseases, University of Georgia, Athens, Georgia, United States of America</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Cary Institute for Ecosystem Studies, Millbrook, New York, United States of America</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Southeastern Cooperative Wildlife Disease Study, College of Veterinary Medicine, University of Georgia, Athens, Georgia, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsiao</span><span class="given-names" tagx="given-names" title="given-names">Chuhsing Kate</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">National Taiwan University, TAIWAN</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">jw14128@uga.edu</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0206926</span><span class="history" title="history"><span class="received" title="received">received: 2018-5-18</span><span class="accepted" title="accepted">accepted: 2018-10-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Walker et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0206926.pdf" title="self-uri">pone.0206926.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Effective public health research and preparedness requires an accurate understanding of which virus species possess or are at risk of developing human transmissibility. Unfortunately, our ability to identify these viruses is limited by gaps in disease surveillance and an incomplete understanding of the process of viral adaptation. By fitting boosted regression trees to data on 224 human viruses and their associated traits, we developed a model that predicts the human transmission ability of zoonotic viruses with over 84% accuracy. This model identifies several viruses that may have an undocumented capacity for transmission between humans. Viral traits that predicted human transmissibility included infection of nonhuman primates, the absence of a lipid envelope, and detection in the human nervous system and respiratory tract. This predictive model can be used to prioritize high-risk viruses for future research and surveillance, and could inform an integrated early warning system for emerging infectious diseases.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
Authors Walker and Ott were supported by assistantships from the University of Georgia Center for Undergraduate Research Opportunities (<a href="http://curo.uga.edu">curo.uga.edu</a>) and NSF DEB EEID Program Grant #1717282. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="The dataset of viral traits on which we performed our analysis has been deposited on figshare at https://figshare.com/s/df98394c27c2064f0f3e (DOI: 10.6084/m9.figshare.7177085)." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The dataset of viral traits on which we performed our analysis has been deposited on figshare at <a href="https://figshare.com/s/df98394c27c2064f0f3e">https://figshare.com/s/df98394c27c2064f0f3e</a> (DOI: <a href="https://doi.org/10.6084/m9.figshare.7177085">10.6084/m9.figshare.7177085</a>).</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Zoonotic viruses pose major threats to human health [<a href="#pone.0206926.ref001">1</a>–<a href="#pone.0206926.ref005">5</a>]. These viruses, which emerge from animal reservoirs, can cause epidemics that require substantial resources for containment [<a href="#pone.0206926.ref006">6</a>, <a href="#pone.0206926.ref007">7</a>] if sustained human-to-human transmission occurs. Even limited outbreaks from stuttering chains of secondary (human to human) transmission can overwhelm local health systems and undermine social and political stability [<a href="#pone.0206926.ref004">4</a>, <a href="#pone.0206926.ref008">8</a>, <a href="#pone.0206926.ref009">9</a>]. Recent outbreak responses have been largely reactive, rather than preemptive, often resulting in inefficient action and costly delays [<a href="#pone.0206926.ref009">9</a>]. Developing a more proactive strategy for spillover prevention will require novel predictive tools [<a href="#pone.0206926.ref010">10</a>].</p> <p>Animal viruses must pass through a series of highly selective evolutionary bottlenecks to become established in the human population [<a href="#pone.0206926.ref011">11</a>]. First, the host species barrier selects for viruses that establish successful infections in exposed humans [<a href="#pone.0206926.ref011">11</a>, <a href="#pone.0206926.ref012">12</a>]. The next bottleneck selects for viruses capable of relatively efficient human-to-human transmission. This efficiency can be measured by the pathogen’s basic reproduction number, <i>R</i><sub>0</sub>, defined as the mean number of secondary infections propagated by an initial case in an immunologically naïve population. Selection for variants with higher <i>R</i><sub>0</sub> values produces viruses capable of more sustained transmission within the new human population [<a href="#pone.0206926.ref012">12</a>]. During this process, two discrete thresholds can be defined: first, whether or not an animal virus can infect humans, and second, whether or not a zoonotic infection can be transmitted between humans. To supplement knowledge on the first threshold, animal-to-human spillover, several data-driven models have been used to predict undiscovered zoonotic viruses, potential vectors [<a href="#pone.0206926.ref013">13</a>, <a href="#pone.0206926.ref014">14</a>], and animal reservoirs [<a href="#pone.0206926.ref015">15</a>–<a href="#pone.0206926.ref017">17</a>]. With regards to the second threshold, a descriptive statistical analysis by Geoghegan <i>et al</i> found that human transmissibility was generally associated with low host mortality, chronic infection, non-segmented genomes, and the lack of an insect vector [<a href="#pone.0206926.ref018">18</a>].</p> <p>In this study, we use a predictive machine learning model to identify specific virus species that may have undocumented potential for human-to-human transmission. The tree-based machine learning method we employ in this analysis, known as gradient boosted regression trees, offers a number of distinct advantages over more parametric statistical models. Boosted regression trees accommodate diverse data types and are robust to hidden interactions, missing data, and co-linearities among variables (e.g., correlated viral traits that arose through shared evolutionary history) [<a href="#pone.0206926.ref019">19</a>, <a href="#pone.0206926.ref020">20</a>]. Additionally, a machine learning approach also allows us to model and interpret complex nonlinear relationships between predictor variables and the response, which can be obscured in parametric regression models. Finally, these methods identify viruses that best fit the trait profile of a zoonotic virus with secondary transmission in humans, giving more precise targets (particular viruses, or viral clades) for surveillance and future research.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>Consolidating records from the primary literature and existing pathogen databases yielded a list of 224 virus species known to infect humans. For each virus, we collected data on taxonomic grouping and 19 biological features (<a href="#pone.0206926.s004">S1 Table</a>), 16 of which do not have a counterpart in previous analyses [<a href="#pone.0206926.ref018">18</a>, <a href="#pone.0206926.ref018">18</a>]. We then assigned each virus a binary score denoting whether or not it was known to be transmissible between humans; non-transmissible viruses received a score of 0, while viruses transmitted directly, via an arthropod vector, or through environmental contamination received a score of 1. Fitting boosted regression trees to this data produced an ensemble of models that identify strong predictors of human transmissibility (<a href="#pone.0206926.g001">Fig 1</a>) and accurately discriminate transmissible from non-transmissible viruses: among zoonotic viruses, models distinguished those with observed secondary transmission with ~84% accuracy (median AUC = 0.8430 +/- 0.0778); when applied across all human viruses (zoonotic and non-zoonotic), the model achieved even higher accuracy (median AUC = 0.9196 +/- 0.0353), demonstrating that zoonotic viruses with secondary transmission are easily distinguishable from all other human viruses on the basis of observable viral traits (<a href="#pone.0206926.g002">Fig 2</a>). AUC scores, tuning parameters, and the relative influence of covariates are reported in <a href="#pone.0206926.s005">S2 Table</a>. We used partial dependence plots to characterize the relationships between predictor variables and transmissibility in our model (<a href="#pone.0206926.g003">Fig 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0206926.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Relative influence of predictors, averaged across 40 models.</div> <p>For each of the 40 boosted regression tree models in our primary ensemble, the normalized relative influence of each predictor variable was computed using Friedman’s algorithm <sup>45</sup>. This figure shows the average of these scores (mean relative influence) for each predictor variable in our dataset that was included in at least one model of the ensemble (mean relative influence &amp;gt; 0). Horizontal lines represent the interval formed by ± 1 standard deviations. Exact relative influence values are listed in <a href="#pone.0206926.s005">S2 Table</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0206926.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Predicted viral risk index.</div> <p>This figure contrasts the observed transmission ability of all 224 viruses in our dataset (red = human-to-human transmission observed, blue = human-to-human transmission not observed) with their average model-predicted response probabilities, as assigned by the primary boosted regression tree models. This model ensemble accurately discriminates transmissible and non-transmissible viruses, as illustrated by the lack of "overlap�? of the two groups in the rank-ordering. The highest ranked viruses that are not currently known to be transmissible between humans were Carnivore amdoparvovirus 1, Hendra virus, Cardiovirus A, Rosavirus A, Human T-lymphotropic viruses 3 &amp;amp; 4 (HTLV-3/4), and Simian Foamy virus. Crimean-Congo haemorrhagic fever virus was the lowest ranked species for human-to-human transmission has been documented.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0206926.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Variable partial dependence plots.</div> <p>Partial dependence plots show how the model-predicted probability that a virus is able to spread between humans is affected by individual viral traits when the effects all other predictors are controlled for. Dark lines represent the median predicted transmission probability across the 40 boosted regression tree models of the primary ensemble, while shaded regions represent the corresponding 95% confidence interval. Viral features are ordered by their mean relative influence within the primary boosted regression tree models from left to right, then top to bottom. Predictor variables with a mean relative influence score of 0 are not included in this figure. Trait definitions and exact relative influence scores are given in <a href="#pone.0206926.s004">S1</a> and <a href="#pone.0206926.s005">S2</a> Tables, respectively.</p>   </div> <p>Our models identified certain non-human hosts of zoonotic viruses as strong predictors of secondary human transmissibility. Zoonoses carried by nonhuman primates were more likely to be transmissible between humans than other zoonotic viruses, while viruses found in rodents and birds were less likely to be transmissible. Arthropod-borne viruses also showed a significantly reduced probability of being transmissible between humans compared to directly transmitted viruses, corroborating previous findings [<a href="#pone.0206926.ref018">18</a>]. Only 13.21% of the arboviruses in our dataset were also transmissible between humans, compared to 76.61% of non-vector-borne viruses.</p> <p>Virus presence in certain human tissues was also predictive of human transmissibility in our model. Virus detection in the liver, the central nervous system, and the upper and lower respiratory tracts was associated with an increased likelihood of transmission between humans. The respiratory tracts support successful viral entry and establishment, as they place a large surface area of exposed mucosal membrane in direct contact with the environment [<a href="#pone.0206926.ref021">21</a>, <a href="#pone.0206926.ref022">22</a>]. As the livers and central nervous systems of living patients are relatively inaccessible to diagnosticians, false negatives among under-sampled viral zoonoses could bias the observed relationships between viral presence in these locations and transmission ability.</p> <p>A number of structural features were also associated with human transmissibility, including the absence of a lipid envelope, small virus particle size (&amp;lt; 75 nanometers (nm) in diameter), and limited genome segmentation (≤ 2 segments). These relationships have been identified in a previous analysis [<a href="#pone.0206926.ref018">18</a>]. Non-enveloped virions are often more stable in the extra-host environment than enveloped particles, and can therefore remain infectious in the air, in water, and on surfaces for longer periods of time [<a href="#pone.0206926.ref023">23</a>]. This corroborates previous findings [<a href="#pone.0206926.ref018">18</a>]. The elevated stability of non-enveloped viruses could partially explain the association between smaller virion size and human transmission in <a href="#pone.0206926.g003">Fig 3</a>, since the lipid envelope contributes to the diameter of the virion; in our dataset, the average diameter of enveloped and non-enveloped viruses was 109 and 52 nm, respectively. The negative relationship that we observed in our models between number of genome segments and transmission likelihood does not contradict Geoghegan <i>et al</i>, which classified genome segmentation as a binary variable and found that segmented viruses are collectively less likely to be transmissible between humans [<a href="#pone.0206926.ref018">18</a>].</p> <p>Although many viral features, including genome length and strandedness, are phylogenetically conserved, only 11 of 104 (10.6%) binary family and genus variables were estimated to have non-zero relative influence in any model, with a combined mean relative influence of only 2.66% (<a href="#pone.0206926.s005">S2 Table</a>). We did not observe significant changes in AUC scores or the trait profile after removing taxonomy variables from the model (<a href="#pone.0206926.s005">S2 Table</a>), suggesting that the covariates in our model capture the majority of the important family- and genus-wide traits that underpin secondary human transmission.</p> <p>Using our primary model ensemble, we ranked viruses by their mean predicted probability of being transmissible between humans (<a href="#pone.0206926.g002">Fig 2</a> and <a href="#pone.0206926.s006">S3 Table</a>). Of the 85 human viruses not currently known to spread between humans, 47 were assigned higher probabilities than Crimean-Congo hemorrhagic fever virus, the lowest ranked species with known transmissibility between humans. Within this subset, the viruses with the highest probability of secondary human transmission were Carnivore amdoparvovirus 1 [<a href="#pone.0206926.ref024">24</a>], Hendra virus [<a href="#pone.0206926.ref025">25</a>], Cardiovirus A [<a href="#pone.0206926.ref026">26</a>, <a href="#pone.0206926.ref027">27</a>], Rosavirus A [<a href="#pone.0206926.ref028">28</a>], HTLV-3 and 4 [<a href="#pone.0206926.ref029">29</a>], and Simian Foamy virus [<a href="#pone.0206926.ref030">30</a>]. These pathogens may be predisposed to developing human-to-human transmissibility, and some may already be capable of transmission between humans, as underreporting and misdiagnosis of infections often allow viruses to spread unobserved [<a href="#pone.0206926.ref031">31</a>]. Future epidemiologic studies of these pathogens should make efforts to identify potential human-to-human transmission.</p> <p>To determine the potential influence of study bias, we created a separate ensemble of models, trained on the same data-splits as the models in our primary ensemble, that included the log-transformed number of PubMed citations for each virus as a predictor, a measure of research effort. Study effort was relatively influential in these models (mean relative influence = 20.685, 2nd highest of all predictors), but its inclusion did not improve overall predictive performance, and the rankings of the most influential variables were highly similar between the two ensembles (Kendall rank-order correlation = 0.929). Comparing partial dependence plots between the two ensembles shows that including study effort as a predictor does not meaningfully alter the predicted trait profile of transmissible viruses (<a href="#pone.0206926.s001">S1 Fig</a>). Furthermore, rankings of individual virus species by their predicted transmission probability were similar between models that do and do not include study effort, with a Kendall rank-order correlation of 0.86. These comparisons suggest that the primary model ensemble is not severely confounded by study bias.</p> <p>In this analysis, viruses capable of spreading between humans directly, environmentally, or through an intermediate arthropod vector are labeled as "transmissible.�? To assess the robustness of our findings to different definitions of the "transmissible�? label, we fit a separate ensemble of GBM models with a modified response variable definition. For these models, we labeled viruses as "transmissible�? (binary response = 1) if they are capable of spreading between humans directly or through the environmental, while viruses that are not known to spread between humans at all or require an intermediate vector to spread are classified as "non-transmissible�? (binary response = 0). This change in definition had the effect of reclassifying the response of 5 arbovirus species (chikungunya, yellow fever, dengue, o’nyong-nyong, and Oropouche) from "transmissible�? to "non-transmissible�?. The rankings of predictors by mean relative influence were almost identical between ensembles (Kendall rank-order correlation = 0.985), and a comparison of partial dependence plots shows that the trait profiles of directly transmissible viruses and transmissible viruses as a whole do not significantly differ (<a href="#pone.0206926.g003">Fig 3</a> and <a href="#pone.0206926.s002">S2 Fig</a>).</p> <p>Not all viruses were accurately classified by our model. In particular, some viruses with known human-to-human transmissibility were assigned a low predicted transmission probability (<a href="#pone.0206926.g002">Fig 2</a>). The 10 human-transmissible viruses with the lowest predicted response values belong to 5 families: <i>Bunyaviridae</i>, <i>Togaviridae</i>, <i>Arenaviridae</i>, <i>Flaviviridae</i>, and <i>Poxviridae</i> (<a href="#pone.0206926.s006">S3 Table</a>). These families contain relatively high numbers of individual virus species, few of which are known to be transmissible between humans (<a href="#pone.0206926.s003">S3 Fig</a>). Further molecular characterization of these viruses may identify factors that distinguish human-to-human transmissible "outlier�? viruses from their non-transmissible relatives.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>These results are subject to two key qualifications. First, by considering viruses at the species level, our approach does not account for variation in human transmissibility within individual species. For example, the H1N1 subtype of Influenza A has caused several human pandemics [<a href="#pone.0206926.ref032">32</a>], whereas the H3N8 subtype within the same species primarily infects dogs and horses, and has never been isolated from humans [<a href="#pone.0206926.ref033">33</a>]. Second, our approach does not consider the efficiency of observed human transmission per se, and handles viruses with supercritical (R<sub>0</sub> &amp;gt; 1) and subcritical (0 &amp;lt; R<sub>0</sub> &amp;lt; 1) transmission identically. Future research aiming to preempt novel viral spillover events will depend on building mechanistic understanding at these smaller scales.</p> <p>In this study, we show that the human-to-human transmissibility of zoonotic viruses can be predicted with a high degree of accuracy by ensembles of machine learning models trained on observed viral features. This data-driven modeling framework could allow public health workers to broadly characterize the epidemic risk posed by novel viral pathogens. Linking these models with those trained to predict the zoonotic capacity of animal viruses [<a href="#pone.0206926.ref015">15</a>] could provide a data-driven method for focusing broad-scale virome sampling projects.</p> <p>Our model ensemble predicts that zoonoses carried by nonhuman primates are more likely to be transmissible between humans relative to other zoonotic viruses, while viruses found in rodents and birds are less likely to be transmissible. This corroborates current theories on viral emergence, which posit that both phylogenetic and physical proximity between human and animal hosts drives spillover events and the success of subsequent adaptations [<a href="#pone.0206926.ref034">34</a>–<a href="#pone.0206926.ref036">36</a>]. Closer phylogenetic relationships between hosts generally correspond to physiological and molecular similarities that allow secondary transmission in a new host species to occur with less extensive viral adaptation [<a href="#pone.0206926.ref002">2</a>, <a href="#pone.0206926.ref003">3</a>, <a href="#pone.0206926.ref012">12</a>, <a href="#pone.0206926.ref034">34</a>, <a href="#pone.0206926.ref036">36</a>]. In contrast, for viruses that infect phylogenetically distant hosts, beneficial adaptations to one host system can significantly reduce pathogen fitness in the others, impeding the evolution of transmissibility within the new host population due to adaptive trade-offs [<a href="#pone.0206926.ref034">34</a>, <a href="#pone.0206926.ref036">36</a>, <a href="#pone.0206926.ref037">37</a>]. However, close physical proximity can increase contact rates between humans and more distantly related host species, creating additional opportunities for spillover. While genetic dissimilarities may initially inhibit the development of secondary transmissibility in humans, increased contact creates more opportunities for successful viral adaptation [<a href="#pone.0206926.ref021">21</a>, <a href="#pone.0206926.ref038">38</a>].</p> <p>The negative association between arthropod vector infection and human-to-human transmissibility in our model ensemble suggests that arboviruses, none of which exclusively infect humans, are significantly constrained by the evolutionary trade-offs needed to infect both phylogenetically divergent arthropod vectors and vertebrate host species [<a href="#pone.0206926.ref034">34</a>, <a href="#pone.0206926.ref036">36</a>, <a href="#pone.0206926.ref039">39</a>]. Viruses that overcome this barrier most commonly circulate between nonhuman primate hosts and anthropophilic vectors, again illustrating the importance of phylogenetic and physical proximity in successful viral emergence [<a href="#pone.0206926.ref002">2</a>, <a href="#pone.0206926.ref034">34</a>, <a href="#pone.0206926.ref036">36</a>]. In contrast, the vast majority of arboviruses infect genetically dissimilar organisms via zoophilic vectors, typically only infecting humans living in close proximity with reservoir hosts. Humans are predominantly dead-end hosts for such viruses.</p> <p>Few quantitative studies have investigated the associations between viral traits and human transmissibility. Our findings corroborate the results of one such study (Geoghegan <i>et al</i> [<a href="#pone.0206926.ref018">18</a>]) in several important ways: both analyses found that DNA-based genomes were associated with human transmission, while genome segmentation, presence of lipid envelopes, and associations with arthropod vectors predicted dead-end spillover. Beyond these physical attributes, our model ensemble also shows how the human-to-human transmissibility of viral zoonoses is shaped by ecological and evolutionary context, with virus isolation from non-human primates and the human respiratory tract and CNS being positively associated with human transmission (<a href="#pone.0206926.g003">Fig 3</a>). In our model ensemble, the variables describing the animal hosts and human organ systems from which viruses have been isolated together account for over 38% of the overall variable influence (<a href="#pone.0206926.g001">Fig 1</a>, <a href="#pone.0206926.s005">S2 Table</a>).</p> <p>We demonstrate that a highly predictive model ensemble based on evolutionarily conserved and readily observable viral characteristics informs the relative risks posed by different zoonotic virus species, and identifies optimal targets for research and surveillance efforts.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="data" title="sec"> <div class="title" tagx="title" title="title">
Data</div> <p>We compiled data on all viruses suspected to infect humans and candidate predictive features associated with each viral species. Our list was initially derived from the viral pathogens listed in GIDEON [<a href="#pone.0206926.ref040">40</a>], ViralZone [<a href="#pone.0206926.ref041">41</a>] and the Virus Pathogen Resource (ViPR) [<a href="#pone.0206926.ref042">42</a>]. We further supplemented this list by searching the literature for publications describing additional viruses which have had genetic material isolated from humans, obtaining a total of 224 viral species recognized by the International Committee on Taxonomy of Viruses [<a href="#pone.0206926.ref043">43</a>]. We also extracted information from the literature on a variety of biological features, including the genetic and structural attributes, animal hosts, arthropod vector status, and human tissue presence of each virus, and encoded this data as 19 predictor variables (<a href="#pone.0206926.s004">S1 Table</a>). Because their histograms showed highly skewed distributions, we log-transformed the variable for number or genome segments. To measure the influence of taxonomic grouping on transmission ability, we included each unique family and genus (using the taxonomic groupings recognized by the International Committee on Taxonomy of Viruses [<a href="#pone.0206926.ref043">43</a>]) represented among our list of human viruses as 104 separate binary variables, for a total of 123 predictor variables. This dataset contains no missing or incomplete values (100% data coverage), and to our knowledge is more extensive in the number of viruses and viral features than past studies of the association between pathogen traits and transmission ability [<a href="#pone.0206926.ref018">18</a>].</p> <p>Based on published epidemiological information, we assigned a binary response variable to indicate whether or not there is evidence of human-to-human transmission for each virus. Our operational definition of "transmissible virus�? is a pathogen which has spread via direct contact with an infectious human, as well as viruses which may be indirectly transmitted between humans through an intermediate arthropod vector or environmental source (i.e., transmission through the fecal-oral route or following contact with surface fomites). This definition does not encompass infection resulting from organ transplantation.</p> <p>Cross-reactivity between antigenically related viral species can produce false positives in serological tests, so we only used the 224 virus species that have had genetic material isolated from humans to perform our analysis. All exploratory analysis and data transformations were conducted in R [<a href="#pone.0206926.ref044">44</a>].</p> </div> <div class="analysis" title="sec"> <div class="title" tagx="title" title="title">
Analysis</div> <p>Of the 224 virus species confirmed to infect humans, we randomly selected 80% and 20% to create the training and testing sets, respectively. We repeated this process to create 40 unique data-splits, and built our primary model ensemble by fitting a single model to each unique training set using the gradient boosting machine (GBM). The GBM fits a boosted regression tree comprising a sequence of decision trees [<a href="#pone.0206926.ref019">19</a>, <a href="#pone.0206926.ref020">20</a>]. Within each tree, predictors are associated with the response (here, a binary indicator of human-to-human transmission) by recursively breaking down the total pool of training observations by randomly selected splitting variables. At each step (or "split�?), groups become smaller and more homogeneous. After a specified number of splits, the mean response of each group is recorded and assigned to the associated terminal node. This process is iterated to create a set of thousands of trees. Our models were built with 5500 trees and specify a Bernoulli error distribution for the binary response variable. We applied ten-fold cross validation during the fitting process to prevent overfitting. To further investigate the effect of phylogeny on transmissibility, we built and analyzed a second ensemble of 40 models that did not include any taxonomy variables. All partitioning, model building and subsequent analysis was done in R [<a href="#pone.0206926.ref044">44</a>] using the GBM package [<a href="#pone.0206926.ref045">45</a>].</p> <p>The structure of these tree ensembles was analyzed to gain insight into the relationships between predictor and response variables. To determine the relative contribution of each predictor to classification performance, we computed relative influence scores using Friedman’s algorithm [<a href="#pone.0206926.ref046">46</a>]. For each predictor variable in a given model, this algorithm sums the reduction in error across all nodes in the collection of trees that use the variable for splitting within a single model. These raw influence scores are then normalized as percentages and the average and standard deviation of scores across all 40 models in the ensemble were derived. We also constructed partial dependence plots, which show the marginal impact of individual predictors on the model response by integrating over the influence of all other variables [<a href="#pone.0206926.ref019">19</a>]. The plots that we display in <a href="#pone.0206926.g003">Fig 3</a> show the median effect across all 40 models in the primary ensemble, with corresponding 95% confidence intervals.</p> <p>To evaluate classification performance, we computed AUC scores for each model on its associated testing and training partitions, and used these scores to derive the mean training and testing AUC of the ensemble.</p> <p>A central premise of our analysis is that our dataset may contain viruses that have an unobserved ability to transmit between humans. These viruses are conservatively designated as "non-transmissible�? in our analysis to denote that human-to-human transmission has not been observed, and also to minimize type II error (the error associated with classifying a human transmissible virus as unable to transmit between humans). This designation is analogous to the treatment of presence-absence data in ecology [<a href="#pone.0206926.ref047">47</a>], and case-control data with contaminated controls in econometrics [<a href="#pone.0206926.ref048">48</a>]. The statistical literature on this problem shows that while poorly calibrated probability estimates may be a consequence of contaminated controls, the ratio of probabilities for pairs of viruses are not affected [<a href="#pone.0206926.ref047">47</a>, <a href="#pone.0206926.ref048">48</a>]. Thus, AUC scores and the rank ordering of virus species by their predicted response are not affected by the discrepancy either.</p> <p>To investigate the differences in predictive performance between our tree-based machine learning approach and more rigid logistic regression models, we fit a comparable generalized linear model (GLM) to each of the 40 train-test partitions. On average, testing AUC was higher in the primary GBM models than the primary GLM models by 0.149. When evaluated by a paired Wilcoxon signed-rank test, this provides statistically significant evidence at p &amp;lt; 0.00001 that the median of the distribution of testing AUC scores is greater for primary GBM models than primary GLM models. When taxonomy variables were excluded from models, testing AUC was higher in GLM models by an average of 0.001 relative to GBM models, a statistically insignificant difference (p = 0.9808 under a Wilcoxon signed-rank test). These results indicate that the predictive performance of our machine learning methodology is equivalent to or greater than that of parametric logistic regression models in this situation.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Variable partial dependence plots, supplementary models with study effort.</div> <p>Partial dependence plots show how the model-predicted probability that a virus is able to spread between humans is affected by individual viral traits when the effects all other predictors are controlled for. These models include the log10-transformed number of PubMed citations for each virus species as a predictor variable. The relationships between predictors and the transmission response are not meaningfully changed from those in our primary model, which does not include a study effort predictor (<a href="#pone.0206926.g003">Fig 3</a>). This suggests that study effort is not a confounder of variable relationships in our models.</p> <p>(TIFF)</p>  <div class="media" title="media"><a href="pone.0206926.s001.tiff">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Variable partial dependence plots, supplementary models with a modified response definition.</div> <p>Partial dependence plots show how the model-predicted probability that a virus is able to spread between humans is affected by individual viral traits when the effects all other predictors are controlled for. In these models, we modified our definition of the response variable such that viruses that require an arthropod vector to spread between humans are relabeled as "non-transmissible�? The relationships between predictors and the transmission response are not meaningfully changed from those in our primary model, in which the "transmissible�? response group includes viruses that exclusively pass between humans indirectly through arthropod vectors (<a href="#pone.0206926.g003">Fig 3</a>). This indicates that our decision to not differentiate between direct transmission and indirect vector-borne transmission in the response variable did not significantly affect the trait-profile of transmissible virus we present in this study.</p> <p>(TIFF)</p>  <div class="media" title="media"><a href="pone.0206926.s002.tiff">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Family characteristics of false-negative virus species.</div> <p>Each point represents a virus family that contains one or more species known to infect humans. Points represent the families of viruses included in our dataset (those known to infect humans). Red points are the 5 virus families containing the ten known-transmissible species with the lowest model-predicted transmission probability (<a href="#pone.0206926.g002">Fig 2</a>). These families contain relatively high numbers of individual virus species known to infect humans, few of which are known to be capable of human-to-human transmission.</p> <p>(TIFF)</p>  <div class="media" title="media"><a href="pone.0206926.s003.tiff">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Descriptions of the predictor variables included in our models.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="pone.0206926.s004.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Information on the parameters, AUC scores, and variable relative influence scores of the primary and secondary models.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="pone.0206926.s005.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
A ranking of virus species by their mean predicted response probability in the primary GBM model ensemble.</div> <p>Viruses for which there is evidence of human-to-human transmission are given a value of 1 in the column <i>Actual</i>.<i>Response</i>, while viruses which are no known to be transmissible have a value of 0.</p> <p>(CSV)</p>  <div class="media" title="media"><a href="pone.0206926.s006.csv">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This research was supported by the University of Georgia Center for Undergraduate Research Opportunities and NSF DEB EEID Program Grant #1717282. We also would like to thank Sarah Coker and Eamon O’Dea for their advice and guidance on the drafting of this paper.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0206926.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">K. E.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global trends in emerging infectious diseases</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">451</span>, <span class="fpage" tagx="fpage" title="fpage">990</span>–<span class="lpage" tagx="lpage" title="lpage">993</span> (<span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature06536">10.1038/nature06536</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18288193">18288193</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M. E. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brierley</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCaffery</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Assessing the Epidemic Potential of RNA and DNA Viruses</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">2037</span>–<span class="lpage" tagx="lpage" title="lpage">2044</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2212.160123">10.3201/eid2212.160123</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27869592">27869592</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">L. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latham</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M. E. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Risk factors for human disease emergence</span>. <span class="source" tagx="source" title="source"><i>Phil</i>. <i>Trans</i>. <i>R</i>. <i>Soc</i></span>. <span class="source" tagx="source" title="source"><i>B</i></span>. <span class="volume" tagx="volume" title="volume">356</span>, <span class="fpage" tagx="fpage" title="fpage">983</span>–<span class="lpage" tagx="lpage" title="lpage">989</span> (<span class="year" tagx="year" title="year">2001</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd-Smith</span><span class="given-names" tagx="given-names" title="given-names">J. O.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epidemic Dynamics at the Human-Animal Interface</span>. <span class="source" tagx="source" title="source"><i>Science</i></span><span class="volume" tagx="volume" title="volume">362</span>, <span class="fpage" tagx="fpage" title="fpage">1362</span>–<span class="lpage" tagx="lpage" title="lpage">1367</span> (<span class="year" tagx="year" title="year">2009</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morse</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schluederberg</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Emerging Viruses: The Evolution of Viruses and Viral Diseases</span>. <span class="source" tagx="source" title="source"><i>J</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">162</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">7</span> (<span class="year" tagx="year" title="year">1990</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2113071">2113071</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piot</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartos</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghys</span><span class="given-names" tagx="given-names" title="given-names">P. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartländer</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">The global impact of HIV/AIDS</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">410</span>, <span class="fpage" tagx="fpage" title="fpage">968</span>–<span class="lpage" tagx="lpage" title="lpage">973</span> (<span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1038/35073639">10.1038/35073639</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11309626">11309626</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The global distribution and burden of dengue</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">496</span>, <span class="fpage" tagx="fpage" title="fpage">504</span>–<span class="lpage" tagx="lpage" title="lpage">507</span> (<span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12060">10.1038/nature12060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563266">23563266</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">United Nations Development Group (UNDG)—Western and Central Africa. Socio-Economic Impact of Ebola Virus Disease in West African Countries. <a href="http://www.africa.undp.org/content/dam/rba/docs/Reports/ebola-west-africa.pdf">http://www.africa.undp.org/content/dam/rba/docs/Reports/ebola-west-africa.pdf</a> (UNDG-CWA, 2015).</span></li> <li tag="ref"><a name="pone.0206926.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siedner</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gostin</span><span class="given-names" tagx="given-names" title="given-names">L. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cranmer</span><span class="given-names" tagx="given-names" title="given-names">H. H.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Strengthening the Detection of and Early Response to Public Health Emergencies: Lessons from the West African Ebola Epidemic</span>. <span class="source" tagx="source" title="source"><i>PLoS Med</i></span>. <span class="volume" tagx="volume" title="volume">12</span>, <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.1001804">10.1371/journal.pmed.1001804</a></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25803303">25803303</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Future directions in analytics for infectious disease intelligence</span>. <span class="source" tagx="source" title="source"><i>EMBO reports</i></span><span class="comment">doi: <a href="https://doi.org/10.15252/embr.201642534">10.15252/embr.201642534</a></span> (<span class="year" tagx="year" title="year">2016</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plowright</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Pathways to zoonotic spillover</span>. <span class="source" tagx="source" title="source"><i>Nature Rev</i>. <i>Microbiol</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro.2017.45">10.1038/nrmicro.2017.45</a></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28555073">28555073</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">C. R.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cross-species Virus Transmission and the Emergence of New Epidemic Diseases</span>. <span class="source" tagx="source" title="source"><i>Microbiology and Molecular Biology Reviews</i></span><span class="volume" tagx="volume" title="volume">72</span>, <span class="fpage" tagx="fpage" title="fpage">457</span>–<span class="lpage" tagx="lpage" title="lpage">470</span> (<span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.00004-08">10.1128/MMBR.00004-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18772285">18772285</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">M. V.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Data-driven identification of potential Zika virus vectors</span>. <span class="source" tagx="source" title="source"><i>eLife</i></span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">e22053</span> (<span class="year" tagx="year" title="year">2017</span>). <span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.22053">10.7554/eLife.22053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28244371">28244371</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L. H.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Data-driven predictions and novel hypotheses about zoonotic tick vectors from the genus <i>Ixodes</i></span>. <span class="source" tagx="source" title="source">BMC Ecology</span><span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">1</span> (<span class="year" tagx="year" title="year">2018</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29347979">29347979</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olival</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span>. <span class="volume" tagx="volume" title="volume">451</span>, <span class="fpage" tagx="fpage" title="fpage">990</span>–<span class="lpage" tagx="lpage" title="lpage">993</span> (<span class="year" tagx="year" title="year">2017</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowden</span><span class="given-names" tagx="given-names" title="given-names">S. E.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Rodent reservoirs of future zoonotic diseases</span>. <span class="source" tagx="source" title="source"><i>Proc</i>. <i>Natl</i>. <i>Acad</i>. <i>Sci</i></span>. <span class="volume" tagx="volume" title="volume">112</span>, <span class="fpage" tagx="fpage" title="fpage">7039</span>–<span class="lpage" tagx="lpage" title="lpage">7044</span> (<span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1501598112">10.1073/pnas.1501598112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26038558">26038558</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Undiscovered Bat Hosts of Filoviruses</span>. <span class="source" tagx="source" title="source"><i>PLoS</i>. <i>Negl</i>. <i>Trop</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">e0004815</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004815">10.1371/journal.pntd.0004815</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27414412">27414412</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geoghegan</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senior</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giallonardo</span><span class="given-names" tagx="given-names" title="given-names">F. D.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Virological factors that increase the transmissibility of emerging human viruses</span>. <span class="source" tagx="source" title="source"><i>Proc</i>. <i>Natl</i>. <i>Acad</i>. <i>Sci</i></span>. <span class="volume" tagx="volume" title="volume">113</span>, <span class="fpage" tagx="fpage" title="fpage">4170</span>–<span class="lpage" tagx="lpage" title="lpage">4175</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1521582113">10.1073/pnas.1521582113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27001840">27001840</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elith</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leathwick</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hastie</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="mixed-article-title" title="mixed-article-title">A working guide to boosted regression trees</span>. <span class="source" tagx="source" title="source"><i>J</i>. <i>Anim</i>. <i>Ecol</i></span>. <span class="volume" tagx="volume" title="volume">77</span>, <span class="fpage" tagx="fpage" title="fpage">802</span>–<span class="lpage" tagx="lpage" title="lpage">813</span> (<span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2656.2008.01390.x">10.1111/j.1365-2656.2008.01390.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397250">18397250</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hastie</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tibshirani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The Elements of Statistical Learning: Data Mining, Inference, and Prediction</span>, <span class="edition" tagx="edition" title="edition">Second Edition</span> (<span class="collab'">collab: Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>, <span class="year" tagx="year" title="year">2009</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herfst</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Drivers of airborne human-to-human pathogen transmission</span>. <span class="source" tagx="source" title="source"><i>Current Opinion in Virology</i></span><span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">22</span>–<span class="lpage" tagx="lpage" title="lpage">29</span> (<span class="year" tagx="year" title="year">2017</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2016.11.006">10.1016/j.coviro.2016.11.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27918958">27918958</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boone</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerba</span><span class="given-names" tagx="given-names" title="given-names">C. P.</span></span><span class="mixed-article-title" title="mixed-article-title">Significance of Fomites in the Spread of Respiratory and Enteric Viral Disease</span>. <span class="source" tagx="source" title="source"><i>Appl</i>. <i>Environ</i>. <i>Microbiol</i></span>. <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">1687</span>–<span class="lpage" tagx="lpage" title="lpage">1696</span> (<span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.02051-06">10.1128/AEM.02051-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220247">17220247</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucas</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knipe</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Viral Capsids and Envelopes: Structure and Function</span>. <span class="source" tagx="source" title="source"><i>Encyclopedia of Life Sciences</i></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/9780470015902.a0001091.pub2">10.1002/9780470015902.a0001091.pub2</a></span> (<span class="year" tagx="year" title="year">2001</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jepsen</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Aleutian Mink Disease Virus and Humans</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">2040</span>–<span class="lpage" tagx="lpage" title="lpage">2042</span> (<span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1512.090514">10.3201/eid1512.090514</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961696">19961696</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Playford</span><span class="given-names" tagx="given-names" title="given-names">E. G.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Human Hendra Virus Encephalitis Associated with Equine Outbreak, Australia, 2008</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">219</span>–<span class="lpage" tagx="lpage" title="lpage">223</span> (<span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1602.090552">10.3201/eid1602.090552</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20113550">20113550</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberste</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Human Febrile Illness Caused by Encephalomyocarditis Virus Infection, Peru</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">640</span>–<span class="lpage" tagx="lpage" title="lpage">646</span> (<span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1504.081428">10.3201/eid1504.081428</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19331761">19331761</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blinkova</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cardioviruses Are Genetically Diverse and Cause Common Enteric Infections in South Asian Children</span>. <span class="source" tagx="source" title="source"><i>J</i>. <i>Virol</i></span>. <span class="volume" tagx="volume" title="volume">83</span>, <span class="fpage" tagx="fpage" title="fpage">4631</span>–<span class="lpage" tagx="lpage" title="lpage">4641</span>, <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02085-08">10.1128/JVI.02085-08</a></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19193786">19193786</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holtz</span><span class="given-names" tagx="given-names" title="given-names">L. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonio</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stine</span><span class="given-names" tagx="given-names" title="given-names">O. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of rosavirus 2, a novel variant of a rodent-associated picornavirus, in children from The Gambia</span>. <span class="source" tagx="source" title="source"><i>Virology</i></span>. <b>0</b>, <span class="fpage" tagx="fpage" title="fpage">25</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>, <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2014.01.018">10.1016/j.virol.2014.01.018</a></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24725928">24725928</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahieux</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gessain</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">HTLV-3/STLV-3 and HTLV-4 Viruses: Discovery, Epidemiology, Serology and Molecular Aspects</span>. <span class="source" tagx="source" title="source"><i>Viruses</i></span><span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">1074</span>–<span class="lpage" tagx="lpage" title="lpage">1090</span> (<span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3390/v3071074">10.3390/v3071074</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994771">21994771</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calattini</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Simian Foamy Virus Transmission from Apes to Humans, Rural Cameroon</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">1314</span>–<span class="lpage" tagx="lpage" title="lpage">1320</span> (<span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1309.061162">10.3201/eid1309.061162</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18252101">18252101</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halliday</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Bringing together emerging and endemic zoonoses surveillance: shared challenges and a common solution</span>. <span class="source" tagx="source" title="source"><i>Phil</i>. <i>Trans</i>. <i>R</i>. <i>Soc</i></span>. <span class="source" tagx="source" title="source"><i>B</i></span>. <span class="volume" tagx="volume" title="volume">367</span>, <span class="fpage" tagx="fpage" title="fpage">2872</span>–<span class="lpage" tagx="lpage" title="lpage">2880</span> (<span class="year" tagx="year" title="year">2012</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jhung</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology of 2009 Pandemic Influenza A (H1N1) in the United States</span>. <span class="source" tagx="source" title="source"><i>Clin</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">S13</span>–<span class="lpage" tagx="lpage" title="lpage">S26</span>, <span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/ciq008">10.1093/cid/ciq008</a></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21342884">21342884</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payungporn</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influenza A Virus (H3N8) in Dogs with Respiratory Disease, Florida</span>. <span class="source" tagx="source" title="source"><i>Emerg</i>. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">902</span>–<span class="lpage" tagx="lpage" title="lpage">908</span>, <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1406.071270">10.3201/eid1406.071270</a></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18507900">18507900</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geoghegan</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duchêne</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families</span>. <span class="source" tagx="source" title="source"><i>PLoS Pathog</i></span>. <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">e1006215</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006215">10.1371/journal.ppat.1006215</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28178344">28178344</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The Evolution and Emergence of RNA Viruses</span> (<span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford Univ. Press</span>, <span class="year" tagx="year" title="year">2009</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanjuán</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo-Calap</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms of viral mutation</span>. <span class="source" tagx="source" title="source"><i>Cell</i>. <i>Mol</i>. <i>Life</i>. <i>Sci</i></span>. <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">4433</span>–<span class="lpage" tagx="lpage" title="lpage">4448</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00018-016-2299-6">10.1007/s00018-016-2299-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27392606">27392606</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="mixed-article-title" title="mixed-article-title">The comparative genomics of viral emergence</span>. <span class="source" tagx="source" title="source"><i>Proc</i>. <i>Natl</i>. <i>Acad</i>. <i>Sci</i></span>. <span class="volume" tagx="volume" title="volume">107</span>, <span class="fpage" tagx="fpage" title="fpage">1742</span>–<span class="lpage" tagx="lpage" title="lpage">1746</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19858482">19858482</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomley</span><span class="given-names" tagx="given-names" title="given-names">F. M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirley</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span><span class="mixed-article-title" title="mixed-article-title">Livestock infectious diseases and zoonoses</span>. <span class="source" tagx="source" title="source"><i>Phil</i>. <i>Trans</i>. <i>R</i>. <i>Soc</i></span>. <span class="source" tagx="source" title="source"><i>B</i></span>. <span class="volume" tagx="volume" title="volume">364</span>, <span class="fpage" tagx="fpage" title="fpage">2637</span>–<span class="lpage" tagx="lpage" title="lpage">2642</span> (<span class="year" tagx="year" title="year">2009</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woelk</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Reduced Positive Selection in Vector-Borne RNA Viruses</span>. <span class="source" tagx="source" title="source"><i>Mol</i>. <i>Biol</i>. <i>Evol</i></span>. <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">2333</span>–<span class="lpage" tagx="lpage" title="lpage">2336</span> (<span class="year" tagx="year" title="year">2000</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berger</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span><span class="mixed-article-title" title="mixed-article-title">GIDEON: a comprehensive Web-based resource for geographic medicine</span>. <span class="source" tagx="source" title="source"><i>International Journal of Health Geographics</i></span><span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">10</span>, <span class="pub-id"><a href="https://dx.doi.org/10.1186/1476-072X-4-10">10.1186/1476-072X-4-10</a></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15847698">15847698</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">ViralZone: a knowledge resource to understand virus diversity</span>. <span class="source" tagx="source" title="source"><i>Nucleic</i>. <i>Acids</i>. <i>Res</i></span>. <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">576</span>–<span class="lpage" tagx="lpage" title="lpage">582</span> (<span class="year" tagx="year" title="year">2011</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">ViPR: An open bioinformatics database and analysis resource for virology research</span>. <span class="source" tagx="source" title="source"><i>Nucleic</i>. <i>Acids</i>. <i>Res</i></span>. <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">593</span>–<span class="lpage" tagx="lpage" title="lpage">598</span> (<span class="year" tagx="year" title="year">2011</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">International Committee on Taxonomy of Viruses (ICTV). Virus Taxonomy: The Classification and Nomenclature of Viruses—The Online (10th) Report of the ICTV. <a href="http://talk.ictvonline.org/ictv-reports/ictv_online_report/">http://talk.ictvonline.org/ictv-reports/ictv_online_report/</a> (ICTV, 2017).</span></li> <li tag="ref"><a name="pone.0206926.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Core Team. R: A Language and Environment for Statistical Computing. <a href="http://www.R-project.org/">www.R-project.org/</a> (2016).</span></li> <li tag="ref"><a name="pone.0206926.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ridgeway, G. gbm: Generalized Boosted Regression Models. cran.r-project.org/web/packages/gbm/index.html (2017).</span></li> <li tag="ref"><a name="pone.0206926.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="mixed-article-title" title="mixed-article-title">Greedy function approximation: A gradient boosting machine</span>. <span class="source" tagx="source" title="source"><i>The Annals of Statistics</i></span><span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">1189</span>–<span class="lpage" tagx="lpage" title="lpage">1232</span> (<span class="year" tagx="year" title="year">2001</span>).</span></li> <li tag="ref"><a name="pone.0206926.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hastie</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fithian</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="mixed-article-title" title="mixed-article-title">Inference from presence-only data; the ongoing controversy</span>. <span class="source" tagx="source" title="source"><i>Ecography</i></span><span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">864</span>–<span class="lpage" tagx="lpage" title="lpage">867</span> (<span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1600-0587.2013.00321.x">10.1111/j.1600-0587.2013.00321.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25492992">25492992</a></span></span></li> <li tag="ref"><a name="pone.0206926.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancaster</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imbens</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">Case-control studies with contaminated controls</span>. <span class="source" tagx="source" title="source"><i>Journal of Econometrics</i></span><span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">145</span>–<span class="lpage" tagx="lpage" title="lpage">160</span> (<span class="year" tagx="year" title="year">1996</span>).</span></li> </ul> </div> </div>  </body></html>