<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2015/11/06/030742.atom"/>
  <meta name="HW.pisa" content="biorxiv;030742v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="What’s in my pot? Real-time species identification on the MinION™"/>
  <meta name="DC.Identifier" content="10.1101/030742"/>
  <meta name="DC.Date" content="2015-11-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2015, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What’s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies’ MinION™ sensing device with a real-time species identification bioinformatics application."/>
  <meta name="DC.Contributor" content="Sissel Juul"/>
  <meta name="DC.Contributor" content="Fernando Izquierdo"/>
  <meta name="DC.Contributor" content="Adam Hurst"/>
  <meta name="DC.Contributor" content="Xiaoguang Dai"/>
  <meta name="DC.Contributor" content="Amber Wright"/>
  <meta name="DC.Contributor" content="Eugene Kulesha"/>
  <meta name="DC.Contributor" content="Roger Pettett"/>
  <meta name="DC.Contributor" content="Daniel J. Turner"/>
  <meta name="article:published_time" content="2015-11-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="What’s in my pot? Real-time species identification on the MinION™"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What’s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies’ MinION™ sensing device with a real-time species identification bioinformatics application.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2015/01/01"/>
  <meta name="citation_mjid" content="biorxiv;030742v1"/>
  <meta name="citation_id" content="030742v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/030742v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/030742v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/030742v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742.full.pdf"/>
  <meta name="citation_doi" content="10.1101/030742"/>
  <meta name="citation_num_pages" content="9"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="030742"/>
  <meta name="citation_author" content="Sissel Juul"/>
  <meta name="citation_author_institution" content="Oxford Nanopore Technologies Inc."/>
  <meta name="citation_author" content="Fernando Izquierdo"/>
  <meta name="citation_author_institution" content="Metrichor Ltd."/>
  <meta name="citation_author" content="Adam Hurst"/>
  <meta name="citation_author_institution" content="Metrichor Ltd."/>
  <meta name="citation_author" content="Xiaoguang Dai"/>
  <meta name="citation_author_institution" content="Oxford Nanopore Technologies Inc."/>
  <meta name="citation_author" content="Amber Wright"/>
  <meta name="citation_author_institution" content="Metrichor Ltd."/>
  <meta name="citation_author" content="Eugene Kulesha"/>
  <meta name="citation_author_institution" content="Metrichor Ltd."/>
  <meta name="citation_author" content="Roger Pettett"/>
  <meta name="citation_author_institution" content="Metrichor Ltd."/>
  <meta name="citation_author" content="Daniel J. Turner"/>
  <meta name="citation_author_institution" content="Oxford Nanopore Technologies Ltd."/>
  <meta name="citation_author_email" content="daniel.turner@nanoporetech.com"/>
  <meta name="citation_reference" content="Naccache SN, et al. A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples. Genome Research 24 (7), 1180–1192(2014)"/>
  <meta name="citation_reference" content="Schuster SC. Next-generation sequencing transforms today’s biology. Nature Methods 5 (1), 16–18 (2008)"/>
  <meta name="citation_reference" content="Dunne WM, Jr., et al. Next-generation and whole-genome sequencing in the diagnostic clinical microbiology laboratory. European Journal of Clinical Microbiology &amp;; Infectious Diseases 31 (8), 1719–1726 (2012)"/>
  <meta name="citation_reference" content="Stone M. Molecular mug shots make catching foodborne pathogens easier. BioScience 62 (12), 1093 (2012)"/>
  <meta name="citation_reference" content="Adams IP, et al. Next-generation sequencing and metagenomic analysis: a universal diagnostic tool in plant virology. Molecular Plant Pathology 10 (4), 537–545 (2009)"/>
  <meta name="citation_reference" content="Yergeau E, et al. Next-generation sequencing of microbial communities in the Athabasca River and its tributaries in relation to oil sands mining activities. Applied and Environmental Microbiology 78 (21), 7626–7637 (2012)"/>
  <meta name="citation_reference" content="Turner D, et al. Complete assembly of novel environmental bacterial genomes by MinION™ sequencing. BioRxiv (http://dx.doi.org/10.1101/0269301(2015)"/>
  <meta name="citation_reference" content="Greninger AL, et al. Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis. Genome Medicine 7 (1), 99 (2015)"/>
  <meta name="citation_reference" content="Jain M, et al. Improved data analysis for the MinION nanopore sequencer. Nature Methods 12 (4), 351–356 (2015)"/>
  <meta name="citation_reference" content="Brown C. Single molecule ‘strand’ sequencing using protein nanopores and scalable electronic devices. F1000 Research http://fl000research.com/slides/4–1206 (2012)"/>
  <meta name="citation_reference" content="Wood DE &amp;; Salzberg SL. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biology 15 (3), R46 (2014)"/>
  <meta name="citation_reference" content="Tatusova T, et al. RefSeq microbial genomes database: new representation and annotation strategy. Nucleic Acids Research 43 (7), 3872 (2015)"/>
  <meta name="citation_reference" content="Paulsen IT, et al. Complete genome sequence of the plant commensal Pseudomonas fluorescens Pf-5. Nature Biotechnology 23 (7), 873–878 (2005)"/>
  <meta name="citation_reference" content="Bolotin A, et al. The complete genome sequence of the lactic acid bacterium Lactococcus lactis ssp. lactis IL1403. Genome Research 11 (5), 731–753 (2001)"/>
  <meta name="citation_reference" content="Munoz M, et al. Fecal shedding of Klebsiella pneumoniae by dairy cows. Journal of Dairy Science 89 (9), 3425–3430 (2006)"/>
  <meta name="citation_reference" content="Kromker V, et al. Bovine Streptococcus uberis intramammary infections and mastitis. Clinical Microbiology 3 (4), 157 (2014)"/>
  <meta name="citation_reference" content="Hashemi M, et al. The prevalence of clinical and subclinical mastitis in dairy cows in the central region of Fars province, south of Iran Iranian Journal of Veterinary Research 12 (3), 236–241 (2011)"/>
  <meta name="citation_reference" content="Tenhagen BA, et al. Prevalence of mastitis pathogens and their resistance against antimicrobial agents in dairy cows in Brandenburg, Germany. Journal of Dairy Science 89 (7), 2542–2551 (2006)"/>
  <meta name="citation_reference" content="Quigley L, et al. A comparison of methods used to extract bacterial DNA from raw milk and raw milk cheese. Journal of Applied Microbiology 113 (1), 96–105 (2012)"/>
  <meta name="citation_reference" content="Adey A, et al. Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition. Genome Biology 11 (12), R119 (2010)"/>
  <meta name="twitter:title" content="What’s in my pot? Real-time species identification on the MinION™"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What’s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies’ MinION™ sensing device with a real-time species identification bioinformatics application."/>
  <meta name="og-title" property="og:title" content="What’s in my pot? Real-time species identification on the MinION™"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/030742v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What’s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies’ MinION™ sensing device with a real-time species identification bioinformatics application."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2015-11-06"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>What’s in my pot? Real-time species identification on the MinION™ | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-8279 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What’s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies’ MinION™ sensing device with a real-time species identification bioinformatics application.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Next Generation Sequencing generates large amounts of genomic information non-specifically from low quantities of input DNA, and hence provides an unbiased way to survey the content of complex metagenomic samples with no prior knowledge 
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup> and without the requirement for culturing. The approach therefore has potential utility in a wide variety of contexts, such as the clinical diagnosis of infectious diseases
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>, in tracking the source of foodborne illness 
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>, monitoring crop pathogens 
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>, and environmental metagenomic studies 
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. In certain contexts, such as in the diagnosis of infectious disease, the speed with which pathogenic species can be identified is of critical importance, because rapid identification enables earlier treatment
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>, and can permit the use of narrow-spectrum antibiotics which reduces the risk of bacteria developing antibiotic resistance. In an ideal world, species would be identified no more than a few hours after sample collection, which would require rapid extraction of nucleic acids, and library preparation, from the sample of interest, and minimal time spent generating and interpreting data.
    </p>
    <p id="p-5">Oxford Nanopore’s MinION™ is a portable nucleic acid sequencing device, which plugs directly into the USB drive of a standard laptop and requires no additional infrastructure, and is capable of generating very long reads in real time 
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </sup>. The device contains an array of protein nanopores, in a high-salt buffer. During the sequencing reaction, an electrical potential is applied, which causes ions - and hence a current – to flow through the pores. As individual strands of DNA pass through the nanopores, the current changes in a sequence-specific manner
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>. One consequence of this is that basecalled data is generated in almost real-time: as soon as a strand passes through one of the nanopores in the array, the electrical current measurements are basecalled, and the resulting sequences are then available for downstream analysis. The time required for a strand to pass through a pore is proportional to its length at a constant translocation speed. To control the speed of strand translocation, and to ratchet the DNA strand through the pore one base at a time, the system uses an accessory protein, termed a motor. The speed of translocation can be adjusted by increasing or decreasing the concentration of ATP in the sequencing buffer. Currently, we control the speed of strand translocation to approximately ~70 bases / second as standard, meaning that a 500 base strand takes just over 7 seconds to translocate, and a 30 kb strand takes just over 7 minutes.
    </p>
    <p id="p-6">As each strand translocates, current measurements are written to a separate file on the laptop to which the MinION is connected. The Metrichor™ agent monitors the contents of this folder, and when a strand has finished translocating, uploads the file to the Amazon WebServer where basecalling and taxonomic classification are performed. The time taken for basecalling is proportional to the length of the strand: approximately 10 seconds for a 500 bp fragment and 10 minutes for a 30kb fragment. Classification typically takes 1-2 seconds per strand.</p>
    <p id="p-7">To complement the real-time nature of data generation, we have developed the What’s in my Pot? (WIMP) analysis pipeline on the Metrichor platform. The WIMP application classifies and identifies microbial species in real time, using a data structure that is pre-built and is shared by all application runs and which is based on taxonomy and a reference database. This data structure maps all kmers of length 24 present in the chosen database to nodes in the NCBI taxonomy tree. Because of this pre-processing, new reads can be quickly classified by looking up kmers, rather than aligning them against the original reference. During the sequencing run, the Metrichor agent uploads reads and a WIMP report is created, the read is basecalled and is classified against the prebuilt data structure. WIMP uses kraken 
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup> to map all kmers, to calculate the least common ancestor (LCA), to determine the most likely placement in the taxonomy tree, and to give each placement a classification score. This score is the fraction of all kmers found in the sequence that have been mapped to the LCA in the clade that contains the taxonomy ID. The higher this classification score, the more confident we can be that the classification was correct.
    </p>
    <p id="p-8">The WIMP bacterial, viral and fungal identification application works with a reference database which covers all the bacterial, viral and fungal genomes available in RefSeq 
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>. For an organism to be identified to subspecies or strain level in the WIMP report it must be present at this taxonomic level in the database. Organisms not included in the database are reported at a taxonomic group higher than strain / subspecies, such as species or genus.
    </p>
    <p id="p-9">To allow straightforward interpretation of WIMP results we have created an interactive report (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>), which allows the user to change several display features:
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Metrichor WIMP application report, shown for bacteria, virus, and fungi." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452569461" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Metrichor WIMP application report, shown for bacteria, virus, and fungi.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F1.medium.gif" width="440" height="310"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F1.medium.gif" width="440" height="310"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-10" class="first-child">Metrichor WIMP application report, shown for bacteria, virus, and fungi.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">
     <strong>Sample composition:</strong> is a donut chart showing the relative proportions of reads from the species found in the sample, and the calculated confidence level. The area of the donut segment for each species is proportional to the read count. The colour of the arc corresponds to the confidence in the 
     <strong>Classification score</strong> section. Clicking on an individual segment updates the 
     <strong>Selection</strong> table with the sequencing run details for that organism, shown here for Aspergillus fumigatus Af293. The selected organism also becomes highlighted in the 
     <strong>NCBI Taxonomy</strong> panel (bolded text and wider lines) and is shown in a lighter colour in the donut chart.
    </p>
    <p id="p-12">
     <strong>Selection:</strong> gives more detail on an individual segment selected in the 
     <strong>Sample composition</strong> section.
    </p>
    <p id="p-13">
     <strong>Taxonomic lineage:</strong> details the taxonomic classification for the individual organism selected in the 
     <strong>Sample composition</strong> section.
    </p>
    <p id="p-14">
     <strong>Classification score:</strong> this shows the current classification threshold for displaying placements in the 
     <strong>NCBI Taxonomy</strong> and 
     <strong>Sample composition</strong> sections. The higher the score, the greater the confidence in the placement. The threshold can be adjusted by clicking on the coloured bar, and the charts will update automatically.
    </p>
    <p id="p-15">
     <strong>NCBI Taxonomy:</strong> nodes can be expanded or collapsed. Node colours correspond to those used to indicate confidence in the 
     <strong>Filter by classification score</strong> and 
     <strong>Sample composition</strong> sections, allowing easy interpretation of the confidence with which taxonomic calls are made.
    </p>
    <p id="p-16">The WIMP report is automatically updated with classified reads at regular intervals or by refreshing the browser throughout the run. An interactive version of this report is available at 
     <a href="http://metrichor.com/workflow_instance/89822?token=U2FsdGVkXl_cDhPEc6-AkWb69BdZblg">http://metrichor.com/workflow_instance/89822?token=U2FsdGVkXl_cDhPEc6-AkWb69BdZblg</a>.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>What’s in my Pot?</h3>
     <p id="p-17">We took several commercially-available bacterial, viral and fungal genomes, which were represented in the RefSeq database, and prepared either PCR-free genomic DNA or cDNA libraries, as appropriate. We pooled the resulting libraries together in arbitrary ratios, before sequencing, and analysis with the Metrichor application WIMP Bacteria Virus Fungi k24 for SQK-MAP006 version 1.48. All organisms were identified, down to strain or subspecies level in many cases (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Comparison of WIMP counts to concentration measured by qPCR</h3>
     <p id="p-18">Next, we took six bacterial genomes and prepared low-input libraries for each, before pooling them in arbitrary ratios, and sequencing. We chose species for which published qPCR primer sequences were available, and for which the primers were robust and specific in our tests (data not shown). The sequence data was analysed in real time using the WIMP workflow. All species were successfully identified. We then compared the WIMP counts for six of these species to the relative abundance of each species in the pool as measured by qPCR. The qPCR counts agree well with the WIMP data, indicating that WIMP (including the sequencing and library preparation) is quantitative (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). As a consequence, the limit of sensitivity of the workflow is essentially a question of how many reads are obtained from the sequencing run, which is governed by how long the device is left to generate data.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of WIMP read counts with quantitative PCR measurements" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452569461" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Comparison of WIMP read counts with quantitative PCR measurements&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F2.medium.gif" width="440" height="235"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F2.medium.gif" width="440" height="235"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-19" class="first-child">Comparison of WIMP read counts with quantitative PCR measurements</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Unpasteurised cow’s milk</h3>
     <p id="p-20">We applied WIMP to a sample where we reasoned that many of the microbial species would be present in the RefSeq database: unpasteurised cow’s milk. To allow further microbial growth we left the milk samples at room temperature for two weeks before extracting the DNA, and we spiked in genomic DNA from 
      <em>Listeria monocytogenes</em> at the earliest opportunity in the DNA extraction. The WIMP analysis reveals the presence of several bacterial genera such as 
      <em>Lactococcus</em> and 
      <em>Pseudomonas,</em> as well as bacteriophages, with varying degrees of confidence at different taxonomic levels (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Metrichor WIMP application report for microorganisms present in unpasteurised cow[[#x2019]]s milk (Listeria used as a spike-in)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452569461" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Metrichor WIMP application report for microorganisms present in unpasteurised cow[[#x2019]]s milk (Listeria used as a spike-in).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F3.medium.gif" width="440" height="290"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F3.medium.gif" width="440" height="290"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2015/11/06/030742/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-21" class="first-child">Metrichor WIMP application report for microorganisms present in unpasteurised cow’s milk (Listeria used as a spike-in).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-22">These organisms might reasonably be expected to occur in a dairy farm environment 
      <sup>
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </sup>. The relatively low confidence of the three strains of 
      <em>Lactococcus lactis subsp lactis</em> indicates that none of these three strains is in the database, whereas the high confidence at the higher subspecies node indicates the presence of 
      <em>Lactococcus lactis subsp lactis</em> albeit a different strain from those in the database. We were able to detect the spiked-in 
      <em>Listeria</em> DNA with very high confidence, and the number of counts of listeria reads (253) matched closely the added amount, by mass (3.5% of total DNA).
     </p>
     <p id="p-23">We also detected bacterial species known to cause bovine mastitis: 
      <em>Klebsiella pneumoniae</em>
      <sup>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
      </sup>, and 
      <em>Streptococcus uberis</em>
      <sup>
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
      </sup> at very high confidence. Mastitis in dairy cattle is a potentially fatal disease, causing necrosis of udder tissue, and affects large proportions of cows 
      <sup>
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
      </sup>. In addition to the obvious harmful consequences of udder infections in dairy cattle, these organisms can cause urinary tract infections and pneumonia in humans.
     </p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Discussion</h2>
    <p id="p-24">Using the laboratory procedure described in this manuscript, we were able to begin generation and classification of sequence data within approximately 3.5 hours starting from unpasteurised milk samples. There are several steps in this procedure that may possibly be shortened. For instance, several alternative methods are available for microbial DNA extraction from milk 
     <sup>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup>, some of which reportedly take less time to process samples. Additionally, to prepare double stranded DNA for sequencing on the MinION, several manipulations are currently necessary: fragment ends are blunted and phosphorylated, and a single deoxyadenosine is added to both 3’ ends. Adapters are then attached by ligation. One adapter is a hairpin, which allows both strands of the duplex to be sequenced in one read, which improves the accuracy of basecalling. The other adapter has several features which enhance the efficiency of sequencing, including a low secondary structure 5’ end for threading into the pore, and a pre-bound motor enzyme. Because there are no obligatory PCR or size-selection steps, when starting with extracted genomic DNA the entire library preparation can be performed in 90 minutes. More rapid library preparation may be performed with the use of transposases 
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
     </sup>, which simultaneously shear and attach adapters to DNA fragments. In addition, it may be possible to perform the classification analysis on pre-basecalled electrical current data, by aligning this data to reference genomes which have been converted to current space using our basecalling model. In theory, such ‘squiggle-space’ analyses have the benefit of being faster to perform than base-space analyses simply because no time is spent basecalling. We intend to explore these options for possible inclusion in future versions of the WIMP application. Finally, although as of November 2015 our standard strand translocation speed is approximately 70 bases per second, we intend to release a ‘fast mode’ shortly in which strands translocate at 500 bases per second. This will allow the acquisition of data &gt;;7 times more quickly, meaning that high confidence strain identification will be possible after a shorter period of sequencing time than at present. At this translocation rate, the data presented in 
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a> could be obtained in under an hour.
    </p>
    <p id="p-25">The WIMP Metrichor application, in combination with the portable MinION sequencer is well-suited to rapidly monitoring the health and well-being of dairy animals and in performing quality control of dairy produce. With further simplification of the DNA extraction and library preparation procedures, such analyses will become possible in a non-laboratory setting, which raises the possibility of using the device in the future for applications such as the real-time monitoring of infections and their susceptibility to antibiotics.</p>
    <p id="p-26">For information on how to access the MinION and WIMP application, please visit 
     <a href="http://www.nanoporetech.com">www.nanoporetech.com</a>
    </p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-8" class="subsection">
     <h3>1. What’s in my Pot?</h3>
     <p id="p-27">Genomic DNA or RNA from 
      <em>Treponema denticola</em> (ATCC
      <sup>®</sup> 35405D-5™), 
      <em>Akkermansia muciniphila</em> (ATCC
      <sup>®</sup> BAA-835D-5™), 
      <em>Campylobacter jejuni</em> subsp. 
      <em>jejuni</em> (ATCC
      <sup>®</sup> 700819D-5™), 
      <em>Neisseria gonorrhoeae</em> (ATCC
      <sup>®</sup> 700825D-5™), 
      <em>Clostridium difficile</em> (ATCC
      <sup>®</sup> BAA-1382D-5™), 
      <em>Bordetella pertussis</em> (ATCC
      <sup>®</sup> BAA-589D-5™), 
      <em>Salmonella enterica subsp. enterica</em> serovar Typhimurium (ATCC
      <sup>®</sup> 700720D-5™), 
      <em>Helicobacter pylori</em> (ATCC
      <sup>®</sup> 43504D-5™), Human respiratory syncytial virus (ATCC
      <sup>®</sup> VR-955D™), Human coronavirus 229E (ATCC
      <sup>®</sup> VR-740D™), Measles virus, strain Edmonston (ATCC
      <sup>®</sup> VR-24D™), Mumps virus, strain Enders (ATCC
      <sup>®</sup> VR-106D™), Human parainfluenza virus 1 (ATCC
      <sup>®</sup> VR-94D™), 
      <em>Aspergillus fumigatus</em> Fresenius (Af293, ATCC
      <sup>®</sup> MYA-4609D-2™), 
      <em>Candida albicans</em> (strain SC5314, ATCC
      <sup>®</sup> MYA-2876D-5™), 
      <em>Cryptococcus neoformans</em> (JEC21, ATCC
      <sup>®</sup> MYA-565D-5™), 
      <em>Malassezia globosa</em> (CBS 7966, ATCC
      <sup>®</sup> MYA-4612D-5™), and 
      <em>Penicillium marneffei</em> (QM 7333, ATCC
      <sup>®</sup> 18224D-2™) were all purchased from ATCC.
     </p>
     <p id="p-28">The bacterial and fungal genomic DNAs were sheared to ~10kb in a g-TUBE (Covaris) according to the manufacturer’s protocol.</p>
     <p id="p-29">The viral genomic RNAs were reverse-transcribed and PCR-amplified using SuperScript
      <sup>®</sup> III One-Step RT-PCR System (Thermo Fisher) and the appropriate primers (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>):
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/188752/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/188752/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/188752" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-30" class="first-child">Primers used in viral reverse transcription</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-31">Bacterial and fungal genomic DNAs and viral cDNA amplicons were end-repaired and dA-tailed using NEBNext
      <sup>®</sup> Ultra™ II End Repair/dA-Tailing Module (New England Biolabs). We prepared sequencing libraries using Oxford Nanopore Technologies’ version 6 library preparation kit, following Oxford Nanopore’s protocols, and sequenced for 6 hours following Oxford Nanopore’s standard running protocols. The samples were analysed with Metrichor application WIMP Bacteria Virus Fungi k24 for SQK-MAP006 version 1.48 as described in the text.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>2. Comparison of WIMP counts to concentration measured by qPCR</h3>
     <p id="p-32">Genomic DNA from each of 
      <em>Treponema denticola, Akkermansia muciniphila, Campylobacter jejuni</em> subsp. 
      <em>jejuni, Neisseria, Clostridium difficile,</em> and 
      <em>Bordetella pertussis</em> were sheared and end-prepared as described above, and Oxford Nanopore’s low-input adapters were ligated on. The libraries were PCR-amplified according to Oxford Nanopore’s low-input PCR protocol. The PCR product was purified using 0.5x volume of Agencourt AMPure XP beads (Beckman Coulter) and the eluted DNA was used as template for qPCR and for preparing sequencing libraries as described above. The samples were analysed with Metrichor application WIMP Bacteria k24 for SQK-MAP006 version 1.34.
     </p>
     <p id="p-33">qPCR standard curves were made in triplicate for all bacteria individually using the primers listed in 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>:
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/188828/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/188828/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/188828" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-34" class="first-child">qPCR primers for bacteria</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-35">10 ng of pooled bacterial DNA was used in each of three replicates for each primer pair to determine the concentration of each bacterial species in the template mix. When a standard curve is used, qPCR measures the number of copies of a bacterial genome, whereas at a constant fragment size, WIMP counts are proportional to the genome size, the qPCR results were adjusted for genome size before being normalised to 
      <em>Clostridium difficile,</em> which had the fewest WIMP reads.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>3. Unpasteurised milk</h3>
     <p id="p-36">Raw cow’s milk was purchased from Udder Milk Creamery Co-Op (
      <a href="http://www.uddermilk.com/">http://www.uddermilk.com/</a>) and immediately aliquoted into sterilized plastic tubes or glass flasks. Aliquots were left at room temperature for two weeks to allow microbial growth. We extracted genomic DNA from 4 mL of raw milk using the Milk Bacterial DNA Isolation Kit (Norgen Biotek Corp). We followed the manufacturer’s protocol with the addition of an initial centrifugation of the milk at 1,000 rpm for 2 minutes to pellet the host cow cells. The supernatant from this centrifugation included the bacterial cells and was transferred to a new tube. The bacterial cells were pelleted by centrifuging at 14,000 rpm for 3 min. 25 ng corresponding to roughly 3.5% of the total DNA, 
      <em>Listeria monocytogenes</em> (Strain EGDe ATCC
      <sup>®</sup> BAA-679D-5™) was added to the sample before binding of DNA to the column.
     </p>
     <p id="p-37">We prepared PCR-free sequencing libraries using Oxford Nanopore Technologies’ version 6 library preparation kits, following Oxford Nanopore’s protocols, and sequenced the samples with and without spiked-in 
      <em>Listeria</em> on different flow cells, for 3 hours each, following Oxford Nanopore’s standard running protocols. The samples were analysed with Metrichor application WIMP Bacteria Virus Fungi k24 for SQK-MAP006 version 1.48.
     </p>
    </div>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.1" data-doi="10.1101/gr.171934.113">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples</span>. 
         <abbr class="cit-jnl-abbrev">Genome Research</abbr>
         <span class="cit-vol">24</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">1180</span>–
         <span class="cit-lpage">1192</span>(
         <span class="cit-pub-date">2014</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171934.113%26rft_id%253Dinfo%253Apmid%252F24899342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE4MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzExLzA2LzAzMDc0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schuster</span>
          <span class="cit-name-given-names">SC.</span>
         </span>
         <span class="cit-article-title">Next-generation sequencing transforms today’s biology</span>. 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-vol">5</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">16</span>–
         <span class="cit-lpage">18</span> (
         <span class="cit-pub-date">2008</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D5%26rft.spage%253D16%26rft_id%253Dinfo%253Apmid%252F18165802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=18165802&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.3" data-doi="10.1007/s10096-012-1641-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dunne</span>
          <span class="cit-name-given-names">WM</span>, 
          <span class="cit-name-suffix">Jr.</span>.
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Next-generation and whole-genome sequencing in the diagnostic clinical microbiology laboratory</span>. 
         <abbr class="cit-jnl-abbrev">European Journal of Clinical Microbiology &amp;; Infectious Diseases</abbr>
         <span class="cit-vol">31</span> (
         <span class="cit-issue">8</span>), 
         <span class="cit-fpage">1719</span>–
         <span class="cit-lpage">1726</span> (
         <span class="cit-pub-date">2012</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bclinical%2Bmicrobiology%2B%2526%2Binfectious%2Bdiseases%2B%253A%2B%2Bofficial%2Bpublication%2Bof%2Bthe%2BEuropean%2BSociety%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DEur%2BJ%2BClin%2BMicrobiol%2BInfect%2BDis%26rft.aulast%253DDunne%26rft.auinit1%253DW.%2BM.%26rft.volume%253D31%26rft.issue%253D8%26rft.spage%253D1719%26rft.epage%253D1726%26rft.atitle%253DNext-generation%2Band%2Bwhole-genome%2Bsequencing%2Bin%2Bthe%2Bdiagnostic%2Bclinical%2Bmicrobiology%2Blaboratory.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10096-012-1641-7%26rft_id%253Dinfo%253Apmid%252F22678348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s10096-012-1641-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22678348&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.4" data-doi="10.1525/bio.2012.62.12.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stone</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Molecular mug shots make catching foodborne pathogens easier</span>. 
         <abbr class="cit-jnl-abbrev">BioScience</abbr>
         <span class="cit-vol">62</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">1093</span> (
         <span class="cit-pub-date">2012</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioScience%26rft_id%253Dinfo%253Adoi%252F10.1525%252Fbio.2012.62.12.15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1525/bio.2012.62.12.15&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Adams</span>
          <span class="cit-name-given-names">IP</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Next-generation sequencing and metagenomic analysis: a universal diagnostic tool in plant virology</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Plant Pathology</abbr>
         <span class="cit-vol">10</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">537</span>–
         <span class="cit-lpage">545</span> (
         <span class="cit-pub-date">2009</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BPlant%2BPathology%26rft.volume%253D10%26rft.spage%253D537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.6" data-doi="10.1128/AEM.02036-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yergeau</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Next-generation sequencing of microbial communities in the Athabasca River and its tributaries in relation to oil sands mining activities</span>. 
         <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>
         <span class="cit-vol">78</span> (
         <span class="cit-issue">21</span>), 
         <span class="cit-fpage">7626</span>–
         <span class="cit-lpage">7637</span> (
         <span class="cit-pub-date">2012</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02036-12%26rft_id%253Dinfo%253Apmid%252F22923391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8yMS83NjI2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTEvMDYvMDMwNzQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-030742v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Turner</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Complete assembly of novel environmental bacterial genomes by MinION™ sequencing</span>. 
         <span class="cit-source">BioRxiv</span> (
         <a href="http://dx.doi.org/10.1101/0269301">http://dx.doi.org/10.1101/0269301</a>(
         <span class="cit-pub-date">2015</span>)
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.8" data-doi="10.1186/s13073&amp;;ndash;015&amp;;ndash;0220&amp;;ndash;9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis</span>. 
         <abbr class="cit-jnl-abbrev">Genome Medicine</abbr>
         <span class="cit-vol">7</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">99</span> (
         <span class="cit-pub-date">2015</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BMedicine%26rft.volume%253D7%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073%2526ndash%253B015%2526ndash%253B0220%2526ndash%253B9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13073&amp;;ndash;015&amp;;ndash;0220&amp;;ndash;9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jain</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Improved data analysis for the MinION nanopore sequencer</span>. 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-vol">12</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">351</span>–
         <span class="cit-lpage">356</span> (
         <span class="cit-pub-date">2015</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D12%26rft.spage%253D351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-web" id="cit-030742v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brown</span>
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-article-title">Single molecule ‘strand’ sequencing using protein nanopores and scalable electronic devices</span>. 
         <span class="cit-source">F1000 Research</span>
         <a href="http://fl000research.com/slides/">http://fl000research.com/slides/</a>
         <span class="cit-fpage">4</span>–
         <span class="cit-lpage">1206</span> (
         <span class="cit-pub-date">2012</span>)
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.11" data-doi="10.1186/gb-2014-15-3-r46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wood</span>
          <span class="cit-name-given-names">DE</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biology</abbr>
         <span class="cit-vol">15</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">R46</span> (
         <span class="cit-pub-date">2014</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24580807&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.12" data-doi="10.1093/nar/gkv278">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tatusova</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">RefSeq microbial genomes database: new representation and annotation strategy</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">43</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">3872</span> (
         <span class="cit-pub-date">2015</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv278%26rft_id%253Dinfo%253Apmid%252F25824943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv278&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25824943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.13" data-doi="10.1038/nbt1110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Paulsen</span>
          <span class="cit-name-given-names">IT</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Complete genome sequence of the plant commensal Pseudomonas fluorescens Pf-5</span>. 
         <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>
         <span class="cit-vol">23</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">873</span>–
         <span class="cit-lpage">878</span> (
         <span class="cit-pub-date">2005</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DPaulsen%26rft.auinit1%253DI.%2BT.%26rft.volume%253D23%26rft.issue%253D7%26rft.spage%253D873%26rft.epage%253D878%26rft.atitle%253DComplete%2Bgenome%2Bsequence%2Bof%2Bthe%2Bplant%2Bcommensal%2BPseudomonas%2Bfluorescens%2BPf-5.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1110%26rft_id%253Dinfo%253Apmid%252F15980861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt1110&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15980861&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000230427700035&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.14" data-doi="10.1101/gr.GR-1697R">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolotin</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The complete genome sequence of the lactic acid bacterium Lactococcus lactis ssp. lactis IL1403</span>. 
         <abbr class="cit-jnl-abbrev">Genome Research</abbr>
         <span class="cit-vol">11</span> (
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">731</span>–
         <span class="cit-lpage">753</span> (
         <span class="cit-pub-date">2001</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.GR-1697R%26rft_id%253Dinfo%253Apmid%252F11337471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjExLzUvNzMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTEvMDYvMDMwNzQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Munoz</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Fecal shedding of Klebsiella pneumoniae by dairy cows</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Dairy Science</abbr>
         <span class="cit-vol">89</span> (
         <span class="cit-issue">9</span>), 
         <span class="cit-fpage">3425</span>–
         <span class="cit-lpage">3430</span> (
         <span class="cit-pub-date">2006</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMunoz%26rft.auinit1%253DM.%2BA.%26rft.volume%253D89%26rft.issue%253D9%26rft.spage%253D3425%26rft.epage%253D3430%26rft.atitle%253DFecal%2BShedding%2Bof%2BKlebsiella%2Bpneumoniae%2Bby%2BDairy%2BCows.%26rft_id%253Dinfo%253Apmid%252F16899675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=16899675&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kromker</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Bovine Streptococcus uberis intramammary infections and mastitis</span>. 
         <abbr class="cit-jnl-abbrev">Clinical Microbiology</abbr>
         <span class="cit-vol">3</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">157</span> (
         <span class="cit-pub-date">2014</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%26rft.volume%253D3%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hashemi</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The prevalence of clinical and subclinical mastitis in dairy cows in the central region of Fars province, south of Iran</span>
         <abbr class="cit-jnl-abbrev">Iranian Journal of Veterinary Research</abbr>
         <span class="cit-vol">12</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">236</span>–
         <span class="cit-lpage">241</span> (
         <span class="cit-pub-date">2011</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIranian%2BJournal%2Bof%2BVeterinary%2BResearch%26rft.volume%253D12%26rft.spage%253D236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.18" data-doi="10.3168/jds.S0022-0302(06)72330-X">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tenhagen</span>
          <span class="cit-name-given-names">BA</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Prevalence of mastitis pathogens and their resistance against antimicrobial agents in dairy cows in Brandenburg, Germany</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Dairy Science</abbr>
         <span class="cit-vol">89</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">2542</span>–
         <span class="cit-lpage">2551</span> (
         <span class="cit-pub-date">2006</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DTenhagen%26rft.auinit1%253DB.-A.%26rft.volume%253D89%26rft.issue%253D7%26rft.spage%253D2542%26rft.epage%253D2551%26rft.atitle%253DPrevalence%2Bof%2Bmastitis%2Bpathogens%2Band%2Btheir%2Bresistance%2Bagainst%2Bantimicrobial%2Bagents%2Bin%2Bdairy%2Bcows%2Bin%2BBrandenburg%252C%2BGermany.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252806%252972330-X%26rft_id%253Dinfo%253Apmid%252F16772573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(06)72330-X&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16772573&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000238293300024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.19" data-doi="10.1111/j.1365-2672.2012.05294.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Quigley</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A comparison of methods used to extract bacterial DNA from raw milk and raw milk cheese</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Applied Microbiology</abbr>
         <span class="cit-vol">113</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">96</span>–
         <span class="cit-lpage">105</span> (
         <span class="cit-pub-date">2012</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DQuigley%26rft.auinit1%253DL.%26rft.volume%253D113%26rft.issue%253D1%26rft.spage%253D96%26rft.epage%253D105%26rft.atitle%253DA%2Bcomparison%2Bof%2Bmethods%2Bused%2Bto%2Bextract%2Bbacterial%2BDNA%2Bfrom%2Braw%2Bmilk%2Band%2Braw%2Bmilk%2Bcheese.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2012.05294.x%26rft_id%253Dinfo%253Apmid%252F22452460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2012.05294.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22452460&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305184900010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-030742v1.20" data-doi="10.1186/gb-2010-11-12-r119">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Adey</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biology</abbr>
         <span class="cit-vol">11</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">R119</span> (
         <span class="cit-pub-date">2010</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAdey%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253DR119%26rft.epage%253DR119%26rft.atitle%253DRapid%252C%2Blow-input%252C%2Blow-bias%2Bconstruction%2Bof%2Bshotgun%2Bfragment%2Blibraries%2Bby%2Bhigh-density%2Bin%2Bvitro%2Btransposition.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-12-r119%26rft_id%253Dinfo%253Apmid%252F21143862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-12-r119&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21143862&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2015%2F11%2F06%2F030742.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
