<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/16/2019.12.15.877092.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.15.877092v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction" />
<meta name="DC.Identifier" content="10.1101/2019.12.15.877092" />
<meta name="DC.Date" content="2019-12-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement , an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies." />
<meta name="DC.Contributor" content="Ali Punjani" />
<meta name="DC.Contributor" content="Haowei Zhang" />
<meta name="DC.Contributor" content="David J. Fleet" />
<meta name="article:published_time" content="2019-12-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive &lt;i&gt;non-uniform refinement&lt;/i&gt;, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, &lt;i&gt;non-uniform refinement&lt;/i&gt; removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. &lt;i&gt;Non-uniform refinement&lt;/i&gt; is implemented in the &lt;i&gt;cryoSPARC&lt;/i&gt; software package and has already been used successfully in several notable structural studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.15.877092v1" />
<meta name="citation_id" content="2019.12.15.877092v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.15.877092" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.15.877092" />
<meta name="citation_author" content="Ali Punjani" />
<meta name="citation_author_institution" content="University of Toronto" />
<meta name="citation_author_institution" content="Vector Institute" />
<meta name="citation_author_institution" content="Structura Biotechnology Inc" />
<meta name="citation_author_email" content="apunjani@structura.bio" />
<meta name="citation_author" content="Haowei Zhang" />
<meta name="citation_author_institution" content="University of Toronto" />
<meta name="citation_author_institution" content="Structura Biotechnology Inc" />
<meta name="citation_author" content="David J. Fleet" />
<meta name="citation_author_institution" content="University of Toronto" />
<meta name="citation_author_institution" content="Vector Institute" />
<meta name="citation_reference" content="J. M. Bell, M. Chen, P. R. Baldwin, and S. J. Ludtke. High resolution single particle refinement in EMAN2.1. Methods, 100:25–34, 2016." />
<meta name="citation_reference" content="G. Cardone, J. B. Heymann, and A. C. Steven. One number does not fit all: Mapping local variations in resolution in cryo-em reconstructions. Journal of Structural Biology, 184(2):226–236, 2013." />
<meta name="citation_reference" content="S. Chen, G. McMullan, A. R. Faruqi, G. N. Murshudov, J. M. Short, S. H. W. Scheres, and R. Henderson. High-resolution noise substitution to measure overfitting and validate resolution in 3d structure determination by single particle electron cryomicroscopy. Ultramicroscopy, 135:24–35, 2013." />
<meta name="citation_reference" content="Y. Chen1, O. B. Clarke, J. Kim, S. Stowe, Y.-K. Kim, Z. Assur, M. Cavalier, R. Godoy-Ruiz, D. C. von Alpen, C. Manzini, W. S. Blaner, J. Frank, L. Quadro, D. J. Weber, L. Shapiro, W. A. Hendrickson, and F. Mancia. Structure of the stra6 receptor for retinol uptake. Science, 353:aad8266, 2016." />
<meta name="citation_reference" content="Y. Cheng. Single-Particle Cryo-EM at Crystallographic Resolution. Cell, 161(3):450–457, 2015." />
<meta name="citation_reference" content="O. Clarke. Personal Communication, 2018." />
<meta name="citation_reference" content="A. P. Dempster, N. M. Laird, and D. B. Rubin. Maximum likelihood from incomplete data via the EM algorithm. Journal of the Royal Statistical Society, Series B, 39(1):1–38, 1977." />
<meta name="citation_reference" content="M. Felsberg and G. Sommer. The monogenic signal. IEEE transactions on Signal Processing, 49(12):3136–3144, 2001." />
<meta name="citation_reference" content="M. S. A. Gilman, C. Liu, A. Fung, I. Behera, P. Jordan, P. Rigaux, N. Ysebaert, S. Tcherniuk, J. Sourimant, J.-F. Eléouët, P. Sutto-Ortiz, E. Decroly, D. Roymans, Z. Jin, and J. S. McLellan. Structure of the Respiratory Syncytial Virus Polymerase Complex. Cell, 179(1):193–204.e14, sep 2019." />
<meta name="citation_reference" content="G. M. Golub, M. Heath, and G. Wahba. Generalized cross-validationas as a method for choosing a good ridge parameter. Technometrics, 21:215–223, 1979." />
<meta name="citation_reference" content="T. Grant, A. Rohou, and N. Grigorieff. cistem, user-friendly software for single-particle image processing. eLife, 7:e35383, 2018." />
<meta name="citation_reference" content="N. Grigorieff. FREEALIGN: High resolution refinement of single particle structures. Journal of Structural Biology, 157:117–125, 2007." />
<meta name="citation_reference" content="G. Harauz and M. van Heel. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. Optik, 73:146–156, 1986." />
<meta name="citation_reference" content="S. Hofmann, D. Januliene, A. R. Mehdipour, C. Thomas, E. Stefan, S. Brüchert, B. T. Kuhn, E. R. Geertsma, G. Hummer, R. Tampé, and A. Moeller. Conformation space of a heterodimeric ABC exporter under turnover conditions. Nature, 571(7766):580–583, 2019." />
<meta name="citation_reference" content="J. Kim, Y. Z. Tan, K. Wicht, et al. Structure and drug resistance of the plasmodium falciparum transporter pfcrt. Nature, Nov. 2019." />
<meta name="citation_reference" content="A. Kucukelbir, F. J. Sigworth, and H. D. Tagare. A Bayesian adaptive basis algorithm for single particle reconstruction. Journal of Structural Biology, 179:56–67, 2012." />
<meta name="citation_reference" content="A. Kucukelbir, F. J. Sigworth, and H. D. Tagare. Quantifying the local resolution of cryo-em density maps. Nature Methods, 11:63–65, 2014." />
<meta name="citation_reference" content="J. Lehtinen, J. Munkberg, J. Hasselgren, S. Laine, T. Karras, M. Aittala, and T. Aila. Noise2Noise: Learning image restoration without clean data. International Conference on Machine Learning, 2018." />
<meta name="citation_reference" content="Y. Liu, K. Zhou, N. Zhang, H. Wei, Y. Z. Tan, Z. Zhang, B. Carragher, C. S. Potter, S. D’Arcy, and K. Luger. FACT caught in the act of manipulating the nucleosome. Nature, 2019." />
<meta name="citation_reference" content="A. H. Nguyen, A. R. B. Thomsen, T. J. Cahill, R. Huang, L.-Y. Huang, T. Marcink, O. B. Clarke, S. Heissel, A. Masoudi, D. Ben-Hail, F. Samaan, V. P. Dandey, Y. Z. Tan, C. Hong, J. P. Mahoney, S. Triest, J. Little, X. Chen, R. Sunahara, J. Steyaert, H. Molina, Z. Yu, A. des Georges, and R. J. Lefkowitz. Structure of an endosomal signaling GPCR–G protein–β-arrestin megacomplex. Nature Structural &amp; Molecular Biology, 26(12):1123–1131, 2019." />
<meta name="citation_reference" content="C. E. Paulsen, J. P. Armache, Y. Gao, Y. Cheng, and D. Julius. Structure of the TRPA1 ion channel suggests regulatory mechanisms. Nature, 520:511–517, 2015." />
<meta name="citation_reference" content="A. Punjani, J. L. Rubinstein, D. J. Fleet, and M. A. Brubaker. CryoSPARC: Algorithms for rapid unsupervised cryo-em structure determination. Nature Methods, 14:290–296, 2017." />
<meta name="citation_reference" content="K. Ramlaul, C. M. Palmer, and C. H. S. Aylett. A Local Agreement Filtering Algorithm for Transmission EM Reconstructions. Journal of Structural Biology, 205(1):30–40, 2019." />
<meta name="citation_reference" content="K. B. Rogala, X. Gu, J. F. Kedir, M. Abu-Remaileh, L. F. Bianchi1, A. M. Bottino1, R. Dueholm1, A. Niehaus1, D. Overwijn1, A. C. Priso Fils1, S. X. Zhou1, D. Leary, N. N. Laqtom1, E. J. Brignole, and D. M. Sabatini. Structural basis for the docking of mTORC1 on the lysosomal surface. Science, 366(6464):468–475, oct 2019." />
<meta name="citation_reference" content="P. B. Rosenthal and R. Henderson. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. J. Molecular Biology, 333(4):721–745, 2003." />
<meta name="citation_reference" content="S. H. W. Scheres. A Bayesian view on cryo-em structure determination. Journal of Molecular Biology, 415:406–418, 2012." />
<meta name="citation_reference" content="S. H. W. Scheres. RELION: Implementation of a Bayesian approach to cryo-em structure determination. Journal of Structural Biology, 180(3):519 – 530, 2012." />
<meta name="citation_reference" content="S. H. W. Scheres and S. Chen. Prevention of overfitting in cryo-EM structure determination. Nature methods, 9(9):853–854, sep 2012." />
<meta name="citation_reference" content="J. L. Vilas, J. Gomez-Blanco, P. Conesa, R. Melero, J. M. de la Rosa-Trevín, J. Oton, J. Cuenca, R. Marabini, J. M. Carazo, J. Vargas, and C. O. S. Sorzano. Monores: Automatic and accurate estimation of local resolution for electron microscopy maps. Structure, 26(2):337–344, 2018." />
<meta name="citation_reference" content="C. Vonesch, L. Wang, Y. Shkolnisky, and A. Singer. Fast wavelet-based single particle reconstruction in cryo-em. In International Symposium on Biomedical Imaging, pages 1950–1953, 2011." />
<meta name="citation_reference" content="G. Wahba. A comparison of GCV and GML for choosing the smoothing parameter in the generalized spline smoothing problem. The Annals of Statistics, 13(4):1378–1402, 1985." />
<meta name="citation_reference" content="A. C. Walls, X. Xiong, Y.-J. Park, M. A. Tortorici, J. Snijder, J. Quispe, E. Cameroni, R. Gopal, M. Dai, A. Lanzavecchia, M. Zambon, F. A. Rey, D. Corti, and D. Veesler. Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell, 176(5):1026–1039.e15, feb 2019." />
<meta name="citation_reference" content="H. Xu, T. Li, A. Rohou, C. P. Arthur, F. Tzakoniati, E. Wong, A. Estevez, C. Kugel, Y. Franke, J. Chen, C. Ciferri, D. H. Hackos, C. M. Koth, and J. Payandeh. Structural basis of nav1.7 inhibition by a gating-modifier spider toxin. Cell, 2019." />
<meta name="citation_reference" content="Z. Yin, Y. Zheng, and P. Doerschuk. An ab initio algorithm for low-resolution 3d reconstructions from cryoelectron microscopy images. Journal of Structural Biology, 133:132–142, 2001." />
<meta name="citation_reference" content="J. Zivanov, T. Nakane, B. Forsberg, D. Kimanius, W. Hagen, and E. Lindahl. New tools for automated high-resolution cryo-em structure determination in RELION-3. eLife, 7, e42166, 2018." />
<meta name="twitter:title" content="Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement , an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies." />
<meta name="og-title" property="og:title" content="Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement , an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.15.877092v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.15.877092v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.15.877092v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1040133" />
    <title>Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aHuC5hZZOePPEAUqD27sOs0rEPXgV1yvmlRf_YVfgDQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1040133","apath":"\/biorxiv\/early\/2019\/12\/16\/2019.12.15.877092.atom","pisa":"biorxiv;2019.12.15.877092v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.15.877092v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.15.877092v1","title":"Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction","identifier":"node\/1040133"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v6QqMdugIdugRLSl9g21UZvMIfAe41ys+7vOuJIAuzLl3J2UKz\\\\\\\/FOgizq\\\\\\\/+xOiGxZWZQxJQNgO0hxOJ8HPyRF5D+bc35SUqtsM6o0ojyDhnqGoys88PRx8VECgS5WmkdL1wJzAktDqW4FJfYiG6I\\\\\\\/b7RmmfEBMXcGW5DjBpNwynD49weiArNZa\\\\\\\/E59v73ueLHgNUa7vm4bi5Vbt1ARglV6wMRadspjTXAnPbHC1eZnZ9NWH2kF6NL1gYfz\\\\\\\/sJAz\\\\\\\/abo7XT+CqLleAlPBSP\\\\\\\/dXufIZDLsXnTbinF1PP6Zk33Z6siVRDpVlA3E9f3w1CfW1omFZW4GIVAipsVuVVPY6WB8iFckhveCQCtqTPlMOki6I2TBlHJIi4hHLYASDFFagKeZmXp95Eqq2euRkpUcC8sizQ9or+eKb1DVbJxKTtrSjyZTzJssqKpDH46f7V2nUc0Pb7CEIqEmJuj14Atut4508rFg2Cmq5pVvbcLfa1vWvTLe3P8zyJRniUeH+a6ATGIAZwFp6van6icay2V4A8bdfiXaMt9qvtTbfeBxTyZbJLWVG7ZXd28gKw5iYCGJGja2rBbrV62VUBxnV2b2xu0VdRsfF\\\\\\\/dqnHY5T4h4I8oYhZgNrfxf666WRMpKdogxmqrAbPJYERz3AJQX\\\\\\\/Y4DzzjkGKemGxpzViYKjCFwPFigr0SSAnLpbrS+sE+XJfoTNDKmu4fM8Rq9rtDVyamsNKk5t3TTkbxbTXbjYv3tscoDtIqbSb8uC4RXQnk9SsThSJg8Il1WY9ASyRJho\\\\\\\/h32eoNBRlZwp7jFb9zLPKp6SZoBySzjJ8aHWDJ+4+kyKnvXjCMhaCdbqw6nfucbBCSvzIMfZN8MBDotHpk9mPFL66m6dMbjcOVXwiE80NfH\\\\\\\/ATpfRJ8BBL0FK3h01mU4OA7pKtCebl6zT3cxf7IkzYn\\\\\\\/M+hxUdNl4CANfUzECdDX9GQoZL9fichc3lF7GNQPFDzkLeoJZXM5rN6PD7RKzuz48P3b0JivqC6RCOpeL4eOhAybOoOFg+kcQ5rXTxgV9P5F0QaAS0nXOuPAqdxSyOy7WWiFha5N5M5qInxpW\\\\\\\/Fp5m7AphV+Lju6KvsAteYGkDYViHjSiLKzYyO\\\\\\\/SjNE82JhbKr\\\\\\\/9hC1C8zdrh1EUbNo\\\\\\\/ZTihuErIBCt\\\\\\\/SI60HTvmbIyYafv\\\\\\\/Wh6RvTqVEcyoqy4HcdXM6oN5pmjyf\\\\\\\/QChViQmBoo\\\\\\\/EMhtXI79mM6uGOUQzb0pDPNuodgeGaDyEr7ETS8j+0aAj2Rh5GaRFQCw9IlHC1upbWjdFYtvxMgpwIw4IwOutc6AbBLbhM6Zn1m4sSM0U55AMDhucAXWMDYN3BUm\\\u0022,\\\u0022iv\\\u0022:\\\u0022DkDPTHTt+MXO8iqt3foeWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002256b2a64ac484140813bf9cfe09839fcbe4dfdc2d8eea4ab4d3629a3360d07b56\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.15.877092v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1040133 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.15.877092v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oiJ2Fh-P1ZO16N4zeXtxjpyioTioeSo5N3zu8moSj6U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1040133" id="node1040133" data-pisa="biorxiv;2019.12.15.877092v1" data-pisa-master="biorxiv;2019.12.15.877092" data-apath="/biorxiv/early/2019/12/16/2019.12.15.877092.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ali</span> <span class="nlm-surname">Punjani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Haowei</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Fleet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.15.877092 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1040133" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ali Punjani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Toronto</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Vector Institute</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Structura Biotechnology Inc</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ali%2BPunjani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Punjani%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAli%2BPunjani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:apunjani@structura.bio" class="" data-icon-position="" data-hide-link-title="0">apunjani@structura.bio</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Haowei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Toronto</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Structura Biotechnology Inc</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haowei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaowei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David J. Fleet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Toronto</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Vector Institute</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BFleet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fleet%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BFleet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.15.877092v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1040133" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1040133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.15.877092v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1040133" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1040133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.15.877092v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1040133" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1040133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.15.877092v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1040133" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1040133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.15.877092v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1040133" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1040133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive <em>non-uniform refinement</em>, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, <em>non-uniform refinement</em> removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. <em>Non-uniform refinement</em> is implemented in the <em>cryoSPARC</em> software package and has already been used successfully in several notable structural studies.</p></div><div class="section" id="sec-1"><h2 class="">1 Introduction</h2><p id="p-3">Single particle cryogenic electron microscopy (cryo-EM) is a rapidly advancing technique for determining high resolution 3D structures of proteins and protein complexes. There are several reasons that cryo-EM has become a tool of choice for recent structural biology projects [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], but key among them is the fact that cryo-EM can be used to study classes of proteins for which other techniques (e.g., X-ray crystallography and nuclear magnetic resonance) are not effective. These include membrane proteins, proteins larger than ∼30kDa with multiple conformational states, or with flexible or disordered domains. In all of these cases, the proteins are composed of regions with varying structural properties. Membrane proteins in particular are often encapsulated in detergent micelles or lipid nanodiscs, creating a large disordered region of density around the hydrophobic region of the protein. Despite the common occurrence of such spatial variability, most state-of-the-art 3D refinement algorithms are based on the mathematical assumption of uniformity (rigidity) of the particle.</p><p id="p-4">This paper formulates a cross-validation regularization framework for single particle cryo-EM refinement, one that explicitly provides for the spatial inhomogeneity generated by various physical phenomena (disorder, motion, occupancy, etc) found in a typical molecular complex. Our proposed framework incorporates general domain knowledge about proteins, without specific knowledge of any particular molecule. Within this framework we derive a novel refinement algorithm, called <em>non-uniform refinement</em>, that accounts for structural variability, while taking care to ensure that the algorithm retains favourable statistical properties for validation and that it does not introduce avenues for over-fitting during 3D reconstruction.</p><p id="p-5">With a GPU accelerated implementation of non-uniform refinement implemented in the <em>cryoSPARC</em> software package [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], we demonstrate improvements in resolution and 3D density map quality for a range of membrane proteins, yielding state-of-the-art reconstructions in all cases. We show results on a small membrane protein-Fab complex in a lipid nanodisc, a small membrane protein with negligible density outside of the lipid bilayer, a medium-sized membrane complex with a large detergent micelle, and a sodium channel with flexible domains. A beta version of the algorithm was previously released in <em>cryoSPARC</em> and has been used in structural studies of several other interesting proteins (see <a id="xref-sec-13-1" class="xref-sec" href="#sec-13">Section 8</a>). Non-uniform refinement is reliable and automatic, requiring no change in parameters between datasets, and with no reliance on hand-made spatial masks or manual labels.</p></div><div class="section" id="sec-2"><h2 class="">2 Regularization in Iterative Refinement</h2><p id="p-6">In the standard cryo-EM 3D reconstruction problem set up [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>], a generative model describes the formation of images of a target protein in an electron microscope. According to the model, the unobserved protein structure, represented by a 3D density, is rotated by a 3D pose and projected along the direction of the electron beam. The resulting 2D projection is translated in the plane, modulated by the microscope CTF, and corrupted by additive noise, under a certain noise model. The 3D density map <em>m</em> is typically parameterized as a real-space 3D array with density at each voxel, in a Cartesian grid of box size <em>N</em>, and a corresponding discrete Fourier representation, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/inline-graphic-1.gif"/></noscript></span></span>. The goal of reconstruction is to infer the unobserved 3D densities of the voxel array, called <strong>model parameters</strong>. Representing the 3D density, 2D projections, observed images, and the noise model in the Fourier domain is common practice for computational efficiency, exploiting the well-known convolution and Fourier-slice theorems [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]. The unobserved pose variables for each image are <strong>latent variables</strong>.</p><p id="p-7">Iterative refinement methods (<a id="xref-statement-1-1" class="xref-statement" href="#statement-1">Algorithm 1</a>), which provide state of the art results (e.g., [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]), can be interpreted as variants of block-coordinate descent or the expectation-maximization algorithm [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], performing maximum-likelihood or maximum-<em>a-posteriori</em> estimation of model parameters from observations, according to the generative model described above. In cryo-EM, and more generally in inverse problems with noisy, partial observations, a critical component that modulates the quality of the results is regularization. Regularization, in intuitive terms, refers to the inclusion of prior knowledge to help penalize model complexity and avoid over-fitting. In cryo-EM refinement it plays a key role in mitigating the effects of noise, so that signal alone is used to infer model parameters. One can regularize problems explicitly, using a prior distribution over model parameters, or implicitly, by applying a regularization operator to the model parameters during optimization. Iterative refinement methods tend to use implicit regularizers, attenuating noise in the reconstructed map at each iteration. In either case, the separation of signal from noise is the crux of many inference problems.</p><div id="statement-1" class="statement"><span class="statement-label">Algorithm 1</span><h3>Iterative Refinement (Expectation-Maximization)</h3><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F1.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure1" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F1.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-8"></p></div><p id="p-9">In the cryo-EM refinement problem, like many latent variable inverse problems, there is an additional interplay between regularization, noise buildup, and the estimation of latent variables. Retained noise due to under-regularization will contaminate the estimation of latent variables. This contamination is propagated to subsequent iterations and causes over-fitting. In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 1</a>, with the TRPA1 membrane protein [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], over-fitting occurs in the disordered micelle and in the dynamic tail at the bottom of the protein. Conversely, over-regularization attenuates useful signal structure, further degrading latent variable estimates. With TRPA1, signal loss occurs in the central region where the structure is not as well resolved as it otherwise could be. In general, the amount of regularization (or smoothing) depends on a set of regularization parameters, the tuning of which is often critical to successful outcomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A 3D map from uniform iterative refinement (in cryoSPARC) reveals spatial variations in structure properties in a prototypical membrane protein (TRPA1 ion channel, EMPIAR-10024 [21]). The density map is shown with a low threshold, where color depicts local resolution [2] as a proxy for local structure properties. The core inner region (red) is more rigid and resolved to higher resolution. The solvent facing region (yellow) is less well ordered. The detergent micelle (light blue) is largely disordered. A flexible tail at the bottom (blue) is blurred due to motion. In uniform refinement, a shift-invariant regularizer smooths all regions equally. This uniformity leaves high frequency noise to accumulate in disordered regions (under-regularization), while discarding resolvable signal in rigid regions (over-regularization). Non-uniform refinement is designed to alleviate these problems." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A 3D map from uniform iterative refinement (in &lt;em&gt;cryoSPARC&lt;/em&gt;) reveals spatial variations in structure properties in a prototypical membrane protein (TRPA1 ion channel, EMPIAR-10024 [21]). The density map is shown with a low threshold, where color depicts local resolution [2] as a proxy for local structure properties. The core inner region (red) is more rigid and resolved to higher resolution. The solvent facing region (yellow) is less well ordered. The detergent micelle (light blue) is largely disordered. A flexible tail at the bottom (blue) is blurred due to motion. In uniform refinement, a shift-invariant regularizer smooths all regions equally. This uniformity leaves high frequency noise to accumulate in disordered regions (under-regularization), while discarding resolvable signal in rigid regions (over-regularization). Non-uniform refinement is designed to alleviate these problems.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F2.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F2.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-10" class="first-child">A 3D map from uniform iterative refinement (in <em>cryoSPARC</em>) reveals spatial variations in structure properties in a prototypical membrane protein (TRPA1 ion channel, EMPIAR-10024 [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]). The density map is shown with a low threshold, where color depicts local resolution [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>] as a proxy for local structure properties. The core inner region (red) is more rigid and resolved to higher resolution. The solvent facing region (yellow) is less well ordered. The detergent micelle (light blue) is largely disordered. A flexible tail at the bottom (blue) is blurred due to motion. In uniform refinement, a shift-invariant regularizer smooths all regions equally. This uniformity leaves high frequency noise to accumulate in disordered regions (under-regularization), while discarding resolvable signal in rigid regions (over-regularization). Non-uniform refinement is designed to alleviate these problems.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">This paper reconsiders the task of regularization based on the observation that common iterative refinement algorithms often systematically under-fit <em>and</em> over-fit different regions of a 3D structure, simultaneously. This causes a loss of resolvable detail in some parts of a structure, and accumulation of noise in others. The reason stems from the use of frequency-space filtering as a form of regularization. Some programs, like <em>cisTEM</em> [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>], use a strict resolution cutoff, beyond which Fourier amplitudes are set to zero before alignment of particle images to the current 3D structure. In <em>RELION</em> [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], regularization was initially formulated with an explicit Gaussian prior on Fourier amplitudes of the 3D structure, with a hand-tuned parameter that controls Fourier amplitude shrinkage. Later versions of <em>RELION</em> [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] and <em>cryoSPARC</em>’s homogeneous (uniform) refinement [<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>] use a transfer function (or Wiener filter) determined by Fourier Shell Correlation (FSC) computed between two independent half-maps (e.g., [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]).</p><p id="p-12">Such methods presume a Fourier basis and shift-invariance. Although well-suited to stationary processes, they are less well suited to protein structures, which are spatially compact and exhibit non-uniform disorder, motion, or variations in signal resolution. FSC, for instance, provides an aggregate measure of resolution. To the extent that FSC under- or over-estimates resolution in different regions, FSC-based shift-invariant filtering will over-smooth some regions, attenuating useful signal, and under smooth others, leaving unwanted noise. To address these issues we introduce a family of adaptive regularizers that can, in many cases, find better 3D structures with improved estimates of the latent poses during refinement.</p></div><div class="section" id="sec-3"><h2 class="">3 Cross-Validation Regularization</h2><p id="p-13">We formulate a new regularizer for cryo-EM reconstruction in terms of the minimization of a cross-validation objective [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. Cross-validation (CV) is a general principle that is widely used in machine learning and statistics for model selection and parameter estimation with complex models. In CV, observed data are randomly partitioned into a training set and a held-out validation set. Model parameters are inferred using the training data, the quality of which is then assessed by measuring an error function applied to the validation data. In <strong>k-fold CV</strong>, the observations are partitioned into <em>k</em> parts. In each of <em>k</em> trials, one part is selected as the held-out validation set, and the remaining <em>k</em> − 1 parts comprise the training set. The per-trial validation errors are summed, providing the total CV error. This procedure measures agreement between the optimized model and the observations, without bias due to over-fitting. Rather, over-fitting during training is detected directly as an increase in the validation error. Importantly, formulating regularization in a cross-validation setting provides a principled way to design regularization operators that are more complex than the conventional, isotropic frequency-space filters. The CV framework is not restricted to a Fourier basis. One may consider more complex parameterizations, the use of meta-parameters, and incorporate cryo-EM domain knowledge.</p><p id="p-14">Given a family of regularizers <em>r</em><sub><em>θ</em></sub> with parameter <em>θ</em>, the minimization of CV error to find <em>θ</em> is often applied as an <em>outer loop</em>. This requires the optimization of model parameters <em>m</em> to be repeated many times with different values of <em>θ</em>, a prohibitively expensive cost for problems like cryo-EM. Instead, one can also perform CV optimization as an <em>inner loop</em>, while optimization of model parameters occurs in the outer loop. Regularizer parameters <em>θ</em> are then effectively optimized on-the-fly, preventing under- or over-fitting without requiring multiple 3D refinements to be completed.</p><p id="p-15">To that end, consider the use of 2-fold CV optimization to select the regularization operator, denoted <em>r</em><sub><em>θ</em></sub>(<em>m</em>) in the regularization step in <a id="xref-statement-1-2" class="xref-statement" href="#statement-1">Algorithm 1</a> (note that <em>k</em> &gt; 2 is also possible). The dataset 𝒟 is partitioned into two halves, 𝒟<sub>1</sub> and 𝒟<sub>2</sub>, and two (unregularized) refinements are computed, namely <em>m</em><sub>1</sub> and <em>m</em><sub>2</sub>. For each, one half of the data is the ‘training set’, and the other is held out for validation. To find the regularizer parameters <em>θ</em> we wish to minimize the total CV error <em>E</em>, i.e.,
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-3.gif"/></noscript></span>

</span>
where <em>e</em> is the negative log likelihood of the validation half-set given the regularized half-map. The second line simplifies this expression by using the raw reconstruction from the opposite half-set as a proxy for the actual observed images. Note that assumptions for “gold standard” refinement [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>] are not broken in this procedure (see <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">Section 5</a>). With the L2 norm, <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Eqn. 1</a> reduces to a sum of per-voxel squared errors, corresponding to white Gaussian noise between the half-set reconstructions. When the choice of <em>θ</em> causes <em>r</em><sub><em>θ</em></sub> to remove too little noise from the raw reconstruction, the residual error <em>E</em> will be unnecessarily large. If <em>θ</em> causes <em>r</em><sub><em>θ</em></sub> to over-regularize, removing too much structure from the raw reconstruction, then <em>E</em> increases as the structure retained by <em>r</em><sub><em>θ</em></sub> no longer cancels corresponding structure in the opposite half-map. As such, minimizing <em>E</em>(<em>θ</em>) provides the regularizer that optimally separates signal from noise. Similar objectives have been used for image de-noising [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Finally, we note that this formulation can be extended to compare each half-set reconstruction against images directly (dealing appropriately with the latent pose variables) or to use error functions corresponding to different noise models.</p></div><div class="section" id="sec-4"><h2 class="">4 Regularization Parameter Optimization</h2><p id="p-16">The CV formulation in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Eqn. 1</a> provides great flexibility in choosing the family of regularizers <em>r</em><sub><em>θ</em></sub>, taking domain knowledge into account. For non-uniform refinement, we wish to accommodate protein structures with spatial variations in disorder, motion, and resolution. Accordingly, we define the regularizer to be a space-varying linear filter. The filter’s spatial extent is determined by the regularization parameter <em>θ</em>(<em>x</em>), which varies with spatial position:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-4.gif"/></noscript></span>

</span>
where <em>h</em>(<em>x</em>; <em>ψ</em>) is symmetric smoothing kernel, the spatial scale of which is specified by <em>ψ</em>. In practice we let <em>h</em>(<em>x</em>; <em>ψ</em>) be a 2<sup>nd</sup>-order Butterworth kernel, the parameter of which is the upper band-limit (frequency) of its transfer function. In this context <em>θ</em>(<em>x</em>) represents the corresponding wavelength.</p><p id="p-17">When <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Eqn. 2</a> is combined with the CV objective for the estimation of <em>θ</em>(<em>x</em>), one obtains
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-18">With one regularization parameter at each voxel, i.e., <em>θ</em>(<em>x</em>), this reduces to a large set of decoupled optimization problems, one for each voxel. That is, for voxel <em>x</em> one obtains
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-6.gif"/></noscript></span>

</span>
</p><p id="p-19">With this decoupling, <em>θ</em>(<em>x</em>) can transition quickly from voxel to voxel, yielding high spatial resolution. On the other hand, the individual sub-problems (4) are not well constrained since each parameter is estimated from data at a single location, so the parameter estimates are not useful. In essence, our regularizer design has two competing goals, namely, reliable signal detection, and high spatial resolution (i.e. respecting boundaries between regions with different properties). Signal detection improves through aggregation of observations (e.g., neighboring voxels), while high spatial resolution prefers minimal aggregation (as in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Eqn. 4</a>).</p><p id="p-20">To improve signal detection, we further constrain <em>θ</em>* to be smooth. That is, although in some regions <em>θ</em> should change quickly (solvent-protein boundaries), in most regions we expect it to change slowly (solvent and regions of rigid protein mass). Smoothness effectively limits the number of degrees of freedom in <em>θ</em>, which is important to ensure <em>θ</em> itself does not overfit during iterative refinement (see <a id="xref-sec-12-1" class="xref-sec" href="#sec-12">Sec. 7</a>). One can encourage smoothness in <em>θ</em> by explicitly penalizing spatial derivatives of <em>θ</em> in the objective (<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Eqn. 3</a>), but this yields a Markov random field problem that is hard to optimize. Alternatively, one can express <em>θ</em> in a low-dimensional basis (e.g., radial basis functions), but this requires prior knowledge of the expected degree of smoothness. Instead, we adopt a simple but effective approach. Assuming that <em>θ</em> is smoothly varying, we treat measurements in the local neighborhood of <em>x</em> as additional constraints on <em>θ</em>(<em>x</em>). A window function can be used to gives more weight to points close to <em>x</em>. We thereby obtain the following least-squares objective:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-7.gif"/></noscript></span>

</span>
where <em>w</em><sub><em>ρ</em></sub>(<em>x</em>) is positive and integrates to one, with spatial extent <em>ρ</em>. This allows one to estimate <em>θ</em> at each voxel independently, while the overlapping neighborhoods ensure that <em>θ</em>(<em>x</em>) varies smoothly.</p><div id="statement-2" class="statement"><span class="statement-label">Algorithm 2</span><h3>Regularization Step for Non-Uniform Refinement</h3><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F3.medium.gif" width="440" height="74"/><noscript><img class="highwire-fragment fragment-image" alt="Figure3" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F3.medium.gif" width="440" height="74"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-21"></p></div><p id="p-22">This approach also provides a natural way to allow for variable neighbourhood sizes, where <em>ρ</em>(<em>x</em>) depends on location <em>x</em>, so both rigid regions and transition regions are well modeled. Importantly, we want <em>ρ</em>(<em>x</em>) to be large enough to reliably estimate <em>θ</em>(<em>x</em>), but small enough to enable local transitions. A reasonable balance can be specified in terms of the highest frequency with significant power as captured by the regularization parameter <em>θ</em>(<em>x</em>). In particular, as a heuristic it suffices to ensure that <em>ρ</em>(<em>x</em>) &gt; <em>γθ</em>(<em>x</em>) where <em>γ</em>, the adaptive window factor (AWF), is a constant.<sup><a id="xref-fn-1-1" class="xref-fn" href="#fn-1">1</a></sup> This constraint yields the final computational problem solved in non-uniform refinement to regularize 3D electron density at each iteration; i.e.,
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/16/2019.12.15.877092/embed/graphic-9.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-5"><h2 class="">5 Non-Uniform Refinement Algorithm</h2><p id="p-23">Given a set of particle images and a low resolution <em>ab initio</em> 3D map, non-uniform refinement comprises three main steps, similar to conventional uniform (homogeneous) refinement (<a id="xref-statement-1-3" class="xref-statement" href="#statement-1">Alg. 1</a>). The data are randomly partitioned into two halves, each of which is used to independently estimate a 3D half-map. This “gold standard” refinement [<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>] allows use of FSC for evaluating map quality, and for comparison with existing algorithms. The alignment of particle images against their respective half-maps, and the reconstruction of the raw 3D density map (the E and M steps in <a id="xref-statement-1-4" class="xref-statement" href="#statement-1">Alg. 1</a>) are also identical to uniform refinement.</p><p id="p-24">The difference between uniform and non-uniform refinement is in the regularization step. First, in non-uniform refinement, regularization is performed independently in the two half-maps. As such, the estimation of the spatial regularization parameters in <a id="xref-statement-2-1" class="xref-statement" href="#statement-2">Algorithm 2</a> effectively partitions each half-dataset into quarter-datasets, and we often refer to the raw reconstructions in <a id="xref-statement-2-2" class="xref-statement" href="#statement-2">Algorithm 2</a> as quarter-maps. The non-uniform refinements on half-maps are therefore entirely independent, satisfying the assumptions of a “gold-standard” refinement [<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">28</a>]. By contrast, conventional uniform refinement uses FSC between half-maps to determine regularization parameters at each iteration, thereby sharing masks and regularization parameters, both of which contaminate final FSC-based assessment because the two half-maps are no longer reconstructed independently.</p><p id="p-25">Most importantly, non-uniform refinement uses <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Eqn. 6</a> to define the optimal parameters with which to regularize each half-set reconstruction at each refinement iteration. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 2</a> shows an example of the difference between uniform filtering (FSC-based) and the new CV-optimal regularizer used in non-uniform refinement. Uniform regularization removes signal and noise from all parts of the 3D map equally. Non-uniform regularization, on the other hand, removes more noise from disordered regions, while retaining the high-resolution signal in well-structured regions that is critical for aligning 2D particle images in the next iteration.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Images illustrate the difference between uniform (FSC-based) regularization and non-uniform (cross-validation-optimal) regularization on a membrane protein (PfCRT, EMPIAR-10330 [15]). Left: A central slice through a raw reconstruction (M-step of Alg. 1) of a half-map after 9 iterations. Middle: The same reconstruction is shown after uniform isotropic filtering (based on FSC between half-maps). Right: The same half-map reconstruction after non-uniform regularization with the optimal CV regularizer. Non-uniform regularization removes noise from the solvent background and nanodisc region, while preserving the high-resolution structure needed for particle alignments in the well-ordered protein region. It is particularly effective due to its implicitly small number of degrees of freedom (Eqn. 6) and capacity to model sharp transitions between ordered and disordered regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Images illustrate the difference between uniform (FSC-based) regularization and non-uniform (cross-validation-optimal) regularization on a membrane protein (PfCRT, EMPIAR-10330 [15]). &lt;strong&gt;Left:&lt;/strong&gt; A central slice through a raw reconstruction (M-step of Alg. 1) of a half-map after 9 iterations. &lt;strong&gt;Middle:&lt;/strong&gt; The same reconstruction is shown after uniform isotropic filtering (based on FSC between half-maps). &lt;strong&gt;Right:&lt;/strong&gt; The same half-map reconstruction after non-uniform regularization with the optimal CV regularizer. Non-uniform regularization removes noise from the solvent background and nanodisc region, while preserving the high-resolution structure needed for particle alignments in the well-ordered protein region. It is particularly effective due to its implicitly small number of degrees of freedom (Eqn. 6) and capacity to model sharp transitions between ordered and disordered regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F4.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F4.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-26" class="first-child">Images illustrate the difference between uniform (FSC-based) regularization and non-uniform (cross-validation-optimal) regularization on a membrane protein (PfCRT, EMPIAR-10330 [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]). <strong>Left:</strong> A central slice through a raw reconstruction (M-step of <a id="xref-statement-1-5" class="xref-statement" href="#statement-1">Alg. 1</a>) of a half-map after 9 iterations. <strong>Middle:</strong> The same reconstruction is shown after uniform isotropic filtering (based on FSC between half-maps). <strong>Right:</strong> The same half-map reconstruction after non-uniform regularization with the optimal CV regularizer. Non-uniform regularization removes noise from the solvent background and nanodisc region, while preserving the high-resolution structure needed for particle alignments in the well-ordered protein region. It is particularly effective due to its implicitly small number of degrees of freedom (<a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Eqn. 6</a>) and capacity to model sharp transitions between ordered and disordered regions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">In practice, for regularization parameter estimation, <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">Equation 6</a> is solved on a discretized parameter space where a relatively simple discrete search method can be used (e.g., as opposed to continuous gradient-based optimization). The algorithm is implemented in Python within the <em>cryoSPARC</em> software platform [<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">22</a>], with most of the computation implemented on GPU accelerators. An efficient solution to <a id="xref-disp-formula-6-4" class="xref-disp-formula" href="#disp-formula-6">Eqn. 6</a> is important in practice because this subproblem is solved twice for each iteration of a non-uniform refinement.</p><p id="p-28">Finally, the tuning parameters for non-uniform regularization are interpretable and relatively few in number. They include the order of the Butterworth kernel, the discretization of the parameter space, and the scalar relating <em>ρ</em>(<em>x</em>) and <em>θ</em>(<em>x</em>), called the adaptive window factor (AWF). In all experiments below we use a 2<sup>nd</sup>-order Butterworth filter, and a fixed AWF parameter <em>γ</em> = 2. We discretize the regularization parameters into 50 possible values, equispaced in the Fourier domain to provide greater sensitivity to small scale changes at finer resolutions. We find that non-uniform refinement is approximately two times slower than uniform refinement in our current implementation.</p></div><div class="section" id="sec-6"><h2 class="">6 Results</h2><p id="p-29">Using four different membrane proteins as examples, we provide extensive comparisons between the proposed non-uniform refinement algorithm and conventional uniform (homogeneous) refinement. For the purposes of this comparison, all processing is performed in <em>cryoSPARC</em>. Except for the type of regularization used in refinement, all parameters and stages of data analysis are held constant between uniform and non-uniform refinement. Both begin with the same <em>ab-initio</em> structure. The same algorithm is used for estimating alignments between 2D particle images and 3D maps, and the same CTF-corrected back-projection is used for direct (unregularized) 3D reconstruction.</p><p id="p-30">With each dataset the same mask is used for both uniform and non-uniform refinement when computing FSC curves. Masks are tested using phase randomization [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>] to ensure they do not cause FSC bias. In addition, the same B-factor is used to sharpen both uniform and non-uniform refinement maps for each dataset. This consistency helps to ensure that the visible differences in 3D structure are due to algorithmic differences and not to differences in the amplification of high spatial frequencies. Where maps are colored by local resolution estimates, a straightforward implementation of <em>Blocres</em> [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>] is used for resolution estimation.</p><p id="p-31">Finally, all non-uniform refinement parameters (other than symmetry) are held constant across all datasets. No manual masks are used during alignment, no masking is done to identify or separate the micelle or nanodisc regions, and no modifications are made to the 2D particle images.</p><div id="sec-7" class="subsection"><h3>6.1 STRA6-CaM</h3><p id="p-32">Zebrafish STRA6-CaM [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>] is a 180kDa C2 symmetric membrane protein complex comprising the 74kDa STRA6 protein bound to calmodulin (CaM). STRA6 mediates the uptake of retinol in various tissues. We present results of uniform and non-uniform refinement on a dataset of 28,848 particle images (pixel size</p><p id="p-33">1.07<em>Å</em>) of STRA6-CaM in a lipid nanodisc, courtesy of Oliver Clarke and Filippo Mancia [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Note that this dataset differs from the older set used in [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>] which consisted of 56,615 particle images of STRA6-CaM in amphipol.</p><p id="p-34"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 3A</a> shows FSC curves from uniform and non-uniform refinement, computed with the same mask. There is a substantial improvement in nominal resolution, from 4.0<em>Å</em> to 3.6<em>Å</em>. This indicates an improvement due to non-uniform refinement in the average global signal-to-noise over the entire structure. It is well known that different regions of a protein will have different resolution characteristics [<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>] and this is apparent in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 3C</a>. The filtered and sharpened 3D maps from uniform and non-uniform refinements are shown from two viewing directions, colored by local resolution estimates. Both maps are filtered using their respective FSC curves and sharpened using a B-factor of − 140<em>Å</em><sup>2</sup>. No local filtering or sharpening is used. The maps are thresholded to contain the same total volume in both cases. Non-uniform refinement resolves significantly improved structural detail in several regions of the molecule, while peripheral and flexible regions remain at low resolutions.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of uniform and non-uniform refinement from 28,848 particle images of STRA6-CaM [4] in lipid nanodisc. A: FSC curves computed with the same mask for both refinements. Numerical improvement from 4.0&#xC5; to 3.6&#xC5; reflects a spatially global improvement of signal. Improvement in some regions is even larger. B: Histograms of change in particle alignment pose and shift between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations. C: 3D density maps from uniform and non-uniform refinement are filtered using the corresponding FSC curves and sharpened with the same B-factor, &#x2212; 140&#xC5;2. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Map differences are due to algorithmic rather than visualization differences. Map color depicts local resolution (Blocres [2]), on a single color scale. D: Individual &#x3B1;-helical segments from the non-uniform map (purple) resolve backbone and side-chains while the uniform map (grey) does not in most cases. The left-most &#x3B1;-helix is peripheral while the right-most is central." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of uniform and non-uniform refinement from 28,848 particle images of &lt;strong&gt;STRA6-CaM&lt;/strong&gt; [4] in lipid nanodisc. &lt;strong&gt;A&lt;/strong&gt;: FSC curves computed with the same mask for both refinements. Numerical improvement from 4.0&lt;em&gt;&#xC5;&lt;/em&gt; to 3.6&lt;em&gt;&#xC5;&lt;/em&gt; reflects a spatially global improvement of signal. Improvement in some regions is even larger. &lt;strong&gt;B&lt;/strong&gt;: Histograms of change in particle alignment pose and shift between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations. &lt;strong&gt;C&lt;/strong&gt;: 3D density maps from uniform and non-uniform refinement are filtered using the corresponding FSC curves and sharpened with the same B-factor, &#x2212; 140&lt;em&gt;&#xC5;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Map differences are due to algorithmic rather than visualization differences. Map color depicts local resolution (&lt;em&gt;Blocres&lt;/em&gt; [2]), on a single color scale. &lt;strong&gt;D&lt;/strong&gt;: Individual &lt;em&gt;&#x3B1;&lt;/em&gt;-helical segments from the non-uniform map (purple) resolve backbone and side-chains while the uniform map (grey) does not in most cases. The left-most &lt;em&gt;&#x3B1;&lt;/em&gt;-helix is peripheral while the right-most is central.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F5.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F5.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-35" class="first-child">Results of uniform and non-uniform refinement from 28,848 particle images of <strong>STRA6-CaM</strong> [<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>] in lipid nanodisc. <strong>A</strong>: FSC curves computed with the same mask for both refinements. Numerical improvement from 4.0<em>Å</em> to 3.6<em>Å</em> reflects a spatially global improvement of signal. Improvement in some regions is even larger. <strong>B</strong>: Histograms of change in particle alignment pose and shift between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations. <strong>C</strong>: 3D density maps from uniform and non-uniform refinement are filtered using the corresponding FSC curves and sharpened with the same B-factor, − 140<em>Å</em><sup>2</sup>. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Map differences are due to algorithmic rather than visualization differences. Map color depicts local resolution (<em>Blocres</em> [<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>]), on a single color scale. <strong>D</strong>: Individual <em>α</em>-helical segments from the non-uniform map (purple) resolve backbone and side-chains while the uniform map (grey) does not in most cases. The left-most <em>α</em>-helix is peripheral while the right-most is central.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36"><a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 3D</a> shows detailed views of individual helices from within the structure with a docked atomic model, also courtesy of Oliver Clarke [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>]. The visible difference in structure quality between uniform and non-uniform refinement is especially relevant during atomic model building, where in many cases, protein backbone and side-chains can only be traced with confidence in the non-uniform refinement map.</p><p id="p-37">Non-uniform refinement appears to improve 3D reconstruction by improving particle image alignments. The STRA6-CaM particle is encased in a lipid nanodisc that has significant mass relative to the protein itself. Also, the CaM subunits are not completely rigid. The presence of this substantial disordered density breaks the assumptions behind uniform refinement. Noise is permitted to iteratively build up within the lipid nanodisc and flexible regions, while the core protein density is over-regularized and information useful for alignments is lost. In non-uniform refinement, the structure is optimally regularized according to cross-validation. The lipid nanodisc is smoothed while protein structure is maximally retained, leading to improved particle alignments. The change in particle image alignments between uniform and non-uniform refinement is shown in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 3B</a>. The pose changes are particularly large, with most particles changing by more than 3 degrees. This change leads directly to improved structure quality.</p><p id="p-38">In the original STRA6-CaM publication using amphipol, the dataset of 56,615 particles was refined to 3.9<em>Å</em> using a uniform refinement. In the result presented here, about half the number of particles (28,848) in lipid nanodisc yield a uniform refinement result of 4.0<em>Å</em> indicating that the quality of the data is higher. Non-uniform refinement, however, produces a 3.6<em>Å</em> result that is substantially better than both uniform refinements.</p></div><div id="sec-8" class="subsection"><h3>6.2 PfCRT</h3><p id="p-39">The <em>Plasmodium falciparum</em> chloroquine resistance transporter (PfCRT) is a small asymmetric 48kDa membrane protein with no soluble region [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. Mutations in PfCRT are associated with the emergence of resistance to chloroquine and piperaquine as antimalarial treatments. The PfCRT dataset used here comprises 16,905 particle images of PfCRT in lipid nanodisc with a Fab bound, imaged with a 0.5175<em>Å</em> pixel size. It is available publicly as EMPIAR-10330 [<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>]. An early version of the non-uniform refinement algorithm was essential for reconstructing the published high resolution density map, into which a molecular model was built and from which biological insights were drawn [<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>].</p><p id="p-40">For PfCRT, the difference in resolution and map quality between uniform and non-uniform refinement is dramatic. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 4A,C</a> shows the corresponding FSC curves and 3D density maps. Uniform refinement is limited to 6.9<em>Å</em>, a resolution at which transmembrane helices are barely resolvable. At this low resolution, the authors [<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>] were unable to interpret the density map nor improve it with conventional techniques. By contrast, their results and ours here show that non-uniform refinement recovers signal up to 3.6<em>Å</em>, from which an atomic model can be built with confidence. Such levels of improvement are the difference between success and failure in a cryo-EM project.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of uniform and non-uniform refinement from 16,905 particle images of PfCRT [15] in lipid nanodisc with a single Fab bound. A: FSC curves computed with the same mask show numerical improvement from 6.9&#xC5; to 3.6&#xC5;, a dramatic global improvement of signal. B: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal non-uniform regularization yields improved alignments through multiple iterations. Note that the x-axis limits are larger than in other figures. C: 3D density maps from uniform and non-uniform refinement, both filtered using the corresponding FSC curve, and sharpened with the same B-factor of &#x2212; 100&#xC5;2. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Maps are colored by local resolution from Blocres [2], all on the same color scale. D: Individual &#x3B1;-helical segments from the non-uniform map (purple) resolve backbone and side-chains while &#x3B1;-helices are barely resolved in uniform (grey) map density and &#x3B2;-strands are not separated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of uniform and non-uniform refinement from 16,905 particle images of &lt;strong&gt;PfCRT&lt;/strong&gt; [15] in lipid nanodisc with a single Fab bound. &lt;strong&gt;A&lt;/strong&gt;: FSC curves computed with the same mask show numerical improvement from 6.9&lt;em&gt;&#xC5;&lt;/em&gt; to 3.6&lt;em&gt;&#xC5;&lt;/em&gt;, a dramatic global improvement of signal. &lt;strong&gt;B&lt;/strong&gt;: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal non-uniform regularization yields improved alignments through multiple iterations. Note that the &lt;em&gt;x&lt;/em&gt;-axis limits are larger than in other figures. &lt;strong&gt;C&lt;/strong&gt;: 3D density maps from uniform and non-uniform refinement, both filtered using the corresponding FSC curve, and sharpened with the same B-factor of &#x2212; 100&lt;em&gt;&#xC5;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Maps are colored by local resolution from &lt;em&gt;Blocres&lt;/em&gt; [2], all on the same color scale. &lt;strong&gt;D&lt;/strong&gt;: Individual &lt;em&gt;&#x3B1;&lt;/em&gt;-helical segments from the non-uniform map (purple) resolve backbone and side-chains while &lt;em&gt;&#x3B1;&lt;/em&gt;-helices are barely resolved in uniform (grey) map density and &lt;em&gt;&#x3B2;&lt;/em&gt;-strands are not separated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F6.medium.gif" width="387" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F6.medium.gif" width="387" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-41" class="first-child">Results of uniform and non-uniform refinement from 16,905 particle images of <strong>PfCRT</strong> [<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">15</a>] in lipid nanodisc with a single Fab bound. <strong>A</strong>: FSC curves computed with the same mask show numerical improvement from 6.9<em>Å</em> to 3.6<em>Å</em>, a dramatic global improvement of signal. <strong>B</strong>: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal non-uniform regularization yields improved alignments through multiple iterations. Note that the <em>x</em>-axis limits are larger than in other figures. <strong>C</strong>: 3D density maps from uniform and non-uniform refinement, both filtered using the corresponding FSC curve, and sharpened with the same B-factor of − 100<em>Å</em><sup>2</sup>. No local filtering or sharpening is used, and thresholds are set to keep the enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Maps are colored by local resolution from <em>Blocres</em> [<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a>], all on the same color scale. <strong>D</strong>: Individual <em>α</em>-helical segments from the non-uniform map (purple) resolve backbone and side-chains while <em>α</em>-helices are barely resolved in uniform (grey) map density and <em>β</em>-strands are not separated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The PfCRT-Fab complex (100kDa) is especially interesting due to its small size on the spectrum of proteins solvable by cryo-EM. In addition, the lipid nanodisc (∼50kDa) around PfCRT accounts for a large fraction of the total particle molecular weight. It is precisely in cases like this that the non-uniform refinement strategy is critical, as larger disordered regions leave more room for over- and under-regularization to influence particle alignments. For PfCRT in particluar, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 4B</a> shows substantial changes in particle alignments between uniform and non-uniform refinement. Most particle images exhibit pose changes of more than 6 degrees, indicating that regularization has a large impact on alignment. It also indicates that a large fraction of particle images may have been grossly misaligned by uniform refinement.</p><p id="p-43"><a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 4D</a> shows in detail the quality of the non-uniform refinement map for PfCRT. Transmembrane <em>α</em>-helices can be directly traced, including side-chains. In contrast, the uniform refinement map does not show helical pitch, and cannot separate <em>β</em>-strands in the Fab domain.</p></div><div id="sec-9" class="subsection"><h3>6.3 90kDa C3 Symmetric Membrane Protein</h3><p id="p-44">The next example is another small membrane protein with no soluble domain. This particle is C3 symmetric with a molecular weight of 90kDa, but unlike the PfCRT, does not have any Fabs bound. The protein mass is surrounded by a large detergent micelle, making alignments difficult. It is therefore a good test case for non-uniform alignment. The dataset, courtesy of Oliver Clarke [<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>], contains 42,740 particles at a pixel size of 1.05<em>Å</em>. It is part of an on-going study that is not yet published, so statistics and figures are shown here with permission, but we do not name the protein or show the entire 3D map.</p><p id="p-45"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 5A</a> shows FSC curves from uniform and non-uniform refinement. The overall global resolution of the map improves from 3.9<em>Å</em> to 3.6<em>Å</em> when using non-uniform refinement, but as with the STRA6-CaM dataset, local regions of the map improve in quality more than the numerical resolution improvement might suggest. <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 5C</a> shows densities for three transmembrane <em>α</em>-helices from the protein, with two (left, center) being closer to the periphery of the protein near the micelle, and one (right) being closer to the center. The peripheral <em>α</em>-helices show significant improvement in the level of detail resolved in the density map, allowing tracing of backbone and side-chains in the non-uniform refinement result. By comparison, model building would be difficult with the corresponding uniform refinement result. The improvement of the central <em>α</em>-helix is less dramatic, and was chosen to indicate that some parts of the uniform refinement map are resolved at its nominal 3.9<em>Å</em> resolution. <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 5B</a> highlights the large changes in particle alignment pose and shift between uniform and non-uniform refinement that also occur with this dataset.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Small membrane protein: Results of uniform and non-uniform refinement on a dataset of 42,740 particle images of a C3 symmetric 90kDa membrane protein with no soluble domains [6]. A: FSC curves computed using the same mask show numerical improvement from 3.9&#xC5; to 3.6&#xC5;, indicating improved global average resolution. B: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations, causing these changes. D: Density map detail for transmembrane &#x3B1;-helices. Two (left, center) are peripheral, near the micelle. One (right) is central within the protein. Both maps are filtered using the corresponding FSC curve, sharpened with the same B-factor of &#x2212; 180&#xC5;2, no local filtering or sharpening is used, and thresholds are set to keep enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Non-uniform (purple) map has clear density for backbone and side-chains while for peripheral &#x3B1;-helices, helical pitch is barely resolved in uniform (grey) map density. The central &#x3B1;-helix (right) improves less dramatically." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;Small membrane protein:&lt;/strong&gt; Results of uniform and non-uniform refinement on a dataset of 42,740 particle images of a C3 symmetric 90kDa membrane protein with no soluble domains [6]. &lt;strong&gt;A&lt;/strong&gt;: FSC curves computed using the same mask show numerical improvement from 3.9&lt;em&gt;&#xC5;&lt;/em&gt; to 3.6&lt;em&gt;&#xC5;&lt;/em&gt;, indicating improved global average resolution. &lt;strong&gt;B&lt;/strong&gt;: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations, causing these changes. &lt;strong&gt;D&lt;/strong&gt;: Density map detail for transmembrane &lt;em&gt;&#x3B1;&lt;/em&gt;-helices. Two (left, center) are peripheral, near the micelle. One (right) is central within the protein. Both maps are filtered using the corresponding FSC curve, sharpened with the same B-factor of &#x2212; 180&lt;em&gt;&#xC5;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;, no local filtering or sharpening is used, and thresholds are set to keep enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Non-uniform (purple) map has clear density for backbone and side-chains while for peripheral &lt;em&gt;&#x3B1;&lt;/em&gt;-helices, helical pitch is barely resolved in uniform (grey) map density. The central &lt;em&gt;&#x3B1;&lt;/em&gt;-helix (right) improves less dramatically.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F7.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F7.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-46" class="first-child"><strong>Small membrane protein:</strong> Results of uniform and non-uniform refinement on a dataset of 42,740 particle images of a C3 symmetric 90kDa membrane protein with no soluble domains [<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>]. <strong>A</strong>: FSC curves computed using the same mask show numerical improvement from 3.9<em>Å</em> to 3.6<em>Å</em>, indicating improved global average resolution. <strong>B</strong>: Histograms of change in particle alignments between uniform and non-uniform refinement. Optimal regularization in non-uniform refinement improves the ability to align particles over iterations, causing these changes. <strong>D</strong>: Density map detail for transmembrane <em>α</em>-helices. Two (left, center) are peripheral, near the micelle. One (right) is central within the protein. Both maps are filtered using the corresponding FSC curve, sharpened with the same B-factor of − 180<em>Å</em><sup>2</sup>, no local filtering or sharpening is used, and thresholds are set to keep enclosed volume constant. Density differences are thus due to algorithmic rather than visualization differences. Non-uniform (purple) map has clear density for backbone and side-chains while for peripheral <em>α</em>-helices, helical pitch is barely resolved in uniform (grey) map density. The central <em>α</em>-helix (right) improves less dramatically.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>6.4 Na<sub>v</sub>1.7 Ion Channel</h3><p id="p-47">The Na<sub>v</sub>1.7 channel [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] is a voltage-gated sodium channel found in the human nervous system. Nav channels are fundamental to the generation and conduction of action potentials in neurons, are mutated in various diseases, and are targeted by toxins and therapeutic drugs (e.g., for pain relief). We present results of uniform and non-uniform refinement on Na<sub>v</sub>1.7 data originally collected and processed by the authors of [<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>], available publicly as raw microscope movies (EMPIAR-10261). In this dataset, the Na<sub>v</sub>1.7 channel is bound to two Fabs, resulting in a 245kDa C2 symmetric complex solublized in detergent. The data contains 25,084 movies from a Gatan K2 Summit direct electron detector camera in counting mode, with a pixel size of 0.849<em>Å</em>. We processed the entire dataset through motion correction, CTF estimation, particle picking, 2D classification, <em>ab-initio</em> reconstruction and heterogeneous refinement in <em>cryoSPARC</em> v2.11 [<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">22</a>]. A total of 738,436 particle images were extracted from the movies. These were curated using 2D classification, yielding 431,741 particle images. As in [<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a>], we detected two discrete conformational states corresponding to the active and inactive states of the channel. 130,982 particles were found in the inactive state, and the remaining 300,759 were found to be in the active state. We obtained state-of-the-art resolution reconstructions for both the active and inactive states, but here we focus solely on the active state.</p><p id="p-48">Compared to the preceding datasets, the Na<sub>v</sub>1.7 complex has a larger molecular weight, and in relative terms, a smaller detergent micelle. It does, however, have other regions that are disordered or flexible, namely, a central 4-helix bundle, peripheral transmembrane domains, and the Fabs. Notably, non-uniform refinement handles these regions of the structure automatically along with the micelle, and therefore still delivers improved resolution and map quality. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 6A</a> shows FSC curves from uniform and non-uniform refinement. Uniform refinement already reaches a 3.4<em>Å</em> resolution, and non-uniform refinement further improves this to 3.1<em>Å</em>. <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 6C</a> shows 3D density maps colored by local resolution. With non-uniform refinement, map quality is clearly improved in central transmembrane regions, while flexible parts of the structure (membrane <em>α</em>-helices closest to the micelle, Fab domains, and 4-helix bundle) remain at intermediate resolutions.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of uniform and non-uniform refinement on 300,759 particle images of Nav1.7 [33] in a detergent micelle with two Fabs bound. A: FSC curves indicate significant improvement in signal with non-uniform refinement, with numerical improvement from 3.4&#xC5; to 3.1&#xC5;. B: Histograms show the changes in particle alignments enabled by optimal regularization in non-uniform refinement. Changes are somewhat smaller here compared to earlier examples. C: The 3D density maps are filtered using corresponding FSC curves, sharpened with a B-factor of &#x2212; 85&#xC5;2. Thresholds are set to keep the enclosed volume constant. No local filtering or sharpening is used. Color depicts local resolution (Blocres [2]), on the same color scale. D: Individual &#x3B1;-helical segments from the non-uniform map (purple) are better resolved, allowing modelling of side-chain rotamers with confidence while uniform map (grey) often does not. Two &#x3B1;-helices (left, center), from the peripheral region of the transmembrane domain, show larger improvement. The other (right), from the central core, is already well resolved by uniform refinement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of uniform and non-uniform refinement on 300,759 particle images of &lt;strong&gt;Na&lt;/strong&gt;&lt;sub&gt;&lt;strong&gt;v&lt;/strong&gt;&lt;/sub&gt;&lt;strong&gt;1.7&lt;/strong&gt; [33] in a detergent micelle with two Fabs bound. &lt;strong&gt;A&lt;/strong&gt;: FSC curves indicate significant improvement in signal with non-uniform refinement, with numerical improvement from 3.4&lt;em&gt;&#xC5;&lt;/em&gt; to 3.1&lt;em&gt;&#xC5;&lt;/em&gt;. &lt;strong&gt;B&lt;/strong&gt;: Histograms show the changes in particle alignments enabled by optimal regularization in non-uniform refinement. Changes are somewhat smaller here compared to earlier examples. &lt;strong&gt;C&lt;/strong&gt;: The 3D density maps are filtered using corresponding FSC curves, sharpened with a B-factor of &#x2212; 85&lt;em&gt;&#xC5;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. Thresholds are set to keep the enclosed volume constant. No local filtering or sharpening is used. Color depicts local resolution (&lt;em&gt;Blocres&lt;/em&gt; [2]), on the same color scale. &lt;strong&gt;D&lt;/strong&gt;: Individual &lt;em&gt;&#x3B1;&lt;/em&gt;-helical segments from the non-uniform map (purple) are better resolved, allowing modelling of side-chain rotamers with confidence while uniform map (grey) often does not. Two &lt;em&gt;&#x3B1;&lt;/em&gt;-helices (left, center), from the peripheral region of the transmembrane domain, show larger improvement. The other (right), from the central core, is already well resolved by uniform refinement.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F8.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F8.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-49" class="first-child">Results of uniform and non-uniform refinement on 300,759 particle images of <strong>Na</strong><sub><strong>v</strong></sub><strong>1.7</strong> [<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">33</a>] in a detergent micelle with two Fabs bound. <strong>A</strong>: FSC curves indicate significant improvement in signal with non-uniform refinement, with numerical improvement from 3.4<em>Å</em> to 3.1<em>Å</em>. <strong>B</strong>: Histograms show the changes in particle alignments enabled by optimal regularization in non-uniform refinement. Changes are somewhat smaller here compared to earlier examples. <strong>C</strong>: The 3D density maps are filtered using corresponding FSC curves, sharpened with a B-factor of − 85<em>Å</em><sup>2</sup>. Thresholds are set to keep the enclosed volume constant. No local filtering or sharpening is used. Color depicts local resolution (<em>Blocres</em> [<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a>]), on the same color scale. <strong>D</strong>: Individual <em>α</em>-helical segments from the non-uniform map (purple) are better resolved, allowing modelling of side-chain rotamers with confidence while uniform map (grey) often does not. Two <em>α</em>-helices (left, center), from the peripheral region of the transmembrane domain, show larger improvement. The other (right), from the central core, is already well resolved by uniform refinement.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50"><a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 6B</a> shows changes in particle alignment poses and shifts between the two refinement types. These changes are smaller on average than the other examples, congruent with the smaller changes in map features. <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 6D</a> shows three selected regions from the density maps along with a docked atomic model (PDB-6N4Q [<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">33</a>]). In <em>α</em>-helices farther from the center of the protein (left, center), improvement in map quality is readily apparent, with many side chains being somewhat buildable in the uniform refinement map, but having clearly interpretable density in the non-uniform refinement map. Central <em>α</em>-helices (right) show less improvement, but map quality remains equal or slightly improved. This result is significant because it shows that reconstructions of proteins without disorder are not harmed by using non-uniform refinement.</p><p id="p-51">Note that our result on the active state of the Na<sub>v</sub>1.7 channel is also an improvement over the published result of 3.6<em>Å</em> (EMDB-0341) [<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">33</a>]. In [<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">33</a>], the authors performed all processing in <em>cisTEM</em> [<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>]. In our result, a substantial part of the improvement over the published result is due to non-uniform refinement, while some other part may be due to improvements in other components of the processing pipeline in <em>cryoSPARC</em>.</p></div><div id="sec-11" class="subsection"><h3>6.5 Reconstruction B-factor Improvements</h3><p id="p-52">The preceding results indicate that non-uniform refinement improves particle image alignments during iterative refinement, yielding improved map resolution and quality. Here, we show that it also improves B-factors [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>] of 3D reconstructions, indicating a more efficient use of signal from the same particle image data.</p><p id="p-53">In [<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>], a B-factor is fit to the rotationally-averaged power spectrum of the reconstructed density map. The B-factor is composed of two parts, <em>B</em><sub>image</sub> and <em>B</em><sub>computation</sub>. <em>B</em><sub>image</sub> captures the decay of high-frequency signal intrinsic to the imaging process. <em>B</em><sub>computation</sub> captures the additional high-frequency signal lost due to computational errors in alignment or reconstruction from the image data.</p><p id="p-54"><a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 7A</a> (left) shows overall B-factors fit to reconstructions for the four datasets discussed above, with uniform and non-uniform refinement, given the same image data. The Guinier plot in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig. 7</a> (right) illustrates the difference in high-frequency signal decay between the two types of refinement, here for the Na<sub>v</sub>1.7 channel dataset. For all datasets, non-uniform refinement B-factor magnitudes are smaller than for uniform refinement. Since <em>B</em><sub>image</sub> is the same in both methods, it follows that non-uniform refinement has lower <em>B</em><sub>computation</sub>, losing less high frequency signal to misalignment. This difference also suggests that using non-uniform refinement should in general require less image data to reach the same resolutions as uniform refinement, consistent with what we observed with the STRA6-CaM dataset above.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Left: B-factor values for four proteins for uniform and non-uniform refinement computed by cryoSPARC. Larger magnitudes (steep slopes) indicate greater signal lost. Given identical image data, differences in B-factor between uniform and non-uniform refinement indicate relative amounts of signal lost due solely to refinement. Non-uniform refinement generally has a significantly lower magnitude, indicating less less signal lost to noise in reconstruction at higher frequencies. Right: Guinier plot (log amplitude per frequency squared [25]) for the Nav1.7 channel dataset described in Sec. 6.4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;Left&lt;/strong&gt;: B-factor values for four proteins for uniform and non-uniform refinement computed by &lt;em&gt;cryoSPARC&lt;/em&gt;. Larger magnitudes (steep slopes) indicate greater signal lost. Given identical image data, differences in B-factor between uniform and non-uniform refinement indicate relative amounts of signal lost due solely to refinement. Non-uniform refinement generally has a significantly lower magnitude, indicating less less signal lost to noise in reconstruction at higher frequencies. &lt;strong&gt;Right&lt;/strong&gt;: Guinier plot (log amplitude per frequency squared [25]) for the Na&lt;sub&gt;v&lt;/sub&gt;1.7 channel dataset described in Sec. 6.4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F9.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F9.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <p id="p-55" class="first-child"><strong>Left</strong>: B-factor values for four proteins for uniform and non-uniform refinement computed by <em>cryoSPARC</em>. Larger magnitudes (steep slopes) indicate greater signal lost. Given identical image data, differences in B-factor between uniform and non-uniform refinement indicate relative amounts of signal lost due solely to refinement. Non-uniform refinement generally has a significantly lower magnitude, indicating less less signal lost to noise in reconstruction at higher frequencies. <strong>Right</strong>: Guinier plot (log amplitude per frequency squared [<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a>]) for the Na<sub>v</sub>1.7 channel dataset described in <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">Sec. 6.4.</a></p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-12"><h2 class="">7 Discussion of Related Methods</h2><p id="p-56">There is a long line of cryo-EM research aimed at estimating the local resolution at each voxel of a 3D structure once a refinement is complete [<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. These methods are generally based on statistical tests applied to the coefficients of a windowed Fourier transform [<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">2</a>] or some form of wavelet transform [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>]. Although the aim of non-uniform refinement is not to estimate resolution <em>per se</em>, the regularizer parameter <em>θ</em>(<em>x</em>) does correspond to a local frequency band-limit at each voxel. As such, it might be viewed as a proxy for local resolution, but with some important differences. Notably, our formulation defines <em>θ</em>(<em>x</em>) as the optimum of a cross-validation objective that removes noise. It does not depend on a particular definition of “local resolution” nor on an explicit resolution estimator.</p><p id="p-57">Local resolution estimates [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">2</a>] or local statistical tests [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] can also be used to adaptively filter a 3D map. This technique is used extensively for visualization, for assessing map quality in different regions of a particle, and during molecular model building. The family of filters and local filter band-limits are typically selected to maximize subjective visual quality and are therefore not optimized against the estimator used to determine local resolution. Considerations are also not made in this technique to control the number of degrees of freedom or model complexity of the local filter parameters. Furthermore, information from opposite half-set reconstructions is typically shared, breaking independence. While local resolution estimation followed by local filtering is generally satisfactory as a one-time post-processing step for visualization, in our experience it can lead to severe over-fitting when used iteratively in a refinement as a substitute for the regularization step in <a id="xref-statement-1-6" class="xref-statement" href="#statement-1">Alg. 1</a>. <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 8</a> gives an example of the over-fitting that often occurs. During iterative refinement, small mis-estimations of local resolution at a few locations (due to high estimator variance [<a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">2</a>]) cause subtle over- or under-fitting, leaving slight density variations. Over multiple iterations of refinement, these errors can produce strong erroneous density that contaminate particle alignments and the local estimation of resolution itself, creating a vicious cycle. A related technique using iterative local resolution and filtering was described briefly in <em>EMAN2.2</em> documentation [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>] and may suffer the same problem. The resulting artefacts (e.g., streaking and spikey density radiating from the structure) are particularly prevalent in datasets with junk particles, structured outliers, or small particles that are already difficult to align. To mitigate these problems, the approach we advocate couples an implicit resolution measure to a particular choice of local regularizer, with optimization explicitly designed to control model capacity and avoid over-fitting of regularizer parameters.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Side view of a peripheral transmembrane region of the Nav1.7 channel showing that naive forms of local filtering during iterative refinement can lead to significant over-fitting. A: Unsharpened density map at the 8th iteration of non-uniform refinement as presented in this work. B: Unsharpened density map at the 8th iteration of a simpler iterative refinement scheme where local resolution estimation and local filtering are performed at each iteration. Both maps are thresholded at the same level. The map in B has acquired clear artefacts of over-fitting, with incorrect high-resolution density dominating the peripheral transmembrane region of the protein. This artefactual density continues to strengthen over subsequent iterations, contaminating local resolution estimates and leading to errors in particle alignments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839354620" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Side view of a peripheral transmembrane region of the Na&lt;sub&gt;v&lt;/sub&gt;1.7 channel showing that naive forms of local filtering during iterative refinement can lead to significant over-fitting. &lt;strong&gt;A&lt;/strong&gt;: Unsharpened density map at the 8&lt;sup&gt;th&lt;/sup&gt; iteration of non-uniform refinement as presented in this work. &lt;strong&gt;B&lt;/strong&gt;: Unsharpened density map at the 8&lt;sup&gt;th&lt;/sup&gt; iteration of a simpler iterative refinement scheme where local resolution estimation and local filtering are performed at each iteration. Both maps are thresholded at the same level. The map in &lt;strong&gt;B&lt;/strong&gt; has acquired clear artefacts of over-fitting, with incorrect high-resolution density dominating the peripheral transmembrane region of the protein. This artefactual density continues to strengthen over subsequent iterations, contaminating local resolution estimates and leading to errors in particle alignments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F10.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F10.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/16/2019.12.15.877092/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8:</span> <p id="p-58" class="first-child">Side view of a peripheral transmembrane region of the Na<sub>v</sub>1.7 channel showing that naive forms of local filtering during iterative refinement can lead to significant over-fitting. <strong>A</strong>: Unsharpened density map at the 8<sup>th</sup> iteration of non-uniform refinement as presented in this work. <strong>B</strong>: Unsharpened density map at the 8<sup>th</sup> iteration of a simpler iterative refinement scheme where local resolution estimation and local filtering are performed at each iteration. Both maps are thresholded at the same level. The map in <strong>B</strong> has acquired clear artefacts of over-fitting, with incorrect high-resolution density dominating the peripheral transmembrane region of the protein. This artefactual density continues to strengthen over subsequent iterations, contaminating local resolution estimates and leading to errors in particle alignments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">Another related technique, used in <em>cisTEM</em> [<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>] and its predecessor <em>Frealign</em> [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>], entails the manual creation of a mask to directly label a region of a structure that one expects to be disordered (eg., a detergent micelle), followed by low-pass filtering in that region to a single pre-set resolution at each iteration of refinement. This technique, one of the first to acknowledge and address the issue of simultaneous under- and over-fitting, shares the same intuitions behind non-uniform refinement. Nevertheless, it relies on manual decisions about how to regularize a map during refinement. Refinement is sensitive to regularization, and manual choices necessitate a trial and error process that can be tedious and difficult to replicate across datasets or by others.</p><p id="p-60">Another possible approach to local control of regularization is to formulate the problem in another basis. In most single particle EM reconstruction methods, the Fourier basis is chosen primarily because the Fourier Slice Theorem facilitates fast projection evaluation and reconstruction. Another natural choice are the closely related spherical harmonics that permit fast rotation [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Wavelet bases have also been suggested [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], the basis functions of which are simultaneously localized in space and the Fourier domain. Indeed, wavelets are used in methods for local resolution estimation [<a id="xref-ref-2-11" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>]. Kucukelbir et al [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] describe an interesting approach to iterative refinement with an adaptive wavelet basis and a sparsity prior. This approch is also similar in spirit to the goal in this work, but their model has a single regularization parameter for the entire 3D map. It is based on the noise variance estimated from the corners of particle images, which may not capture local variations in noise due to disorder, motion, or partial occupancy. Nevertheless, like the cross-validation approach here, the authors also find that with a data-driven approach they can largely avoid the need for manual parameter setting and the spatial masking that is needed with a Fourier basis.</p></div><div class="section" id="sec-13"><h2 class="">8 Conclusions</h2><p id="p-61">We introduce non-uniform refinement, a new algorithm for achieving higher resolution and higher quality reconstructions of protein structures from single particle cryo-EM data. Non-uniform refinement uses cross-validation regularization as a principled tool for parameter estimation in noisy inverse problems like cryo-EM reconstruction. In this way, we are able to incorporate into the algorithm, and exploit, the domain knowledge that many types of proteins contain regions with varying levels of disorder, flexibility, or occupancy. These variations lead uniform refinement algorithms to under- or over-regularize regions of a 3D density map during refinement. Instead, non-uniform refinement automatically identifies and treats local regions optimally, without introducing bias into “gold standard” estimation of resolution with the FSC and without any specific knowledge about any particular protein.</p><p id="p-62">A range of different proteins are used to show the capability of the algorithm to resolve higher-resolution detail, and in many cases, significantly improve map quality. A further effect of using a principled cross-validation procedure is that the entire algorithm becomes automatic and does not require user interaction to change any parameters between datasets.</p><p id="p-63">In this work, non-uniform refinement is contrasted with uniform refinement in which global FSC-based resolution estimates are used for filtering during iterative refinement. The fact that non-uniform refinement locally regularizes 3D maps during refinement indicates that there may in fact be significant structure resolved beyond the global FSC-based resolution to which final maps are filtered before visualization. We do not address this possibility here, but it does raise the interesting question of how one can then effectively measure resolution in general (similar to the premise of [<a id="xref-ref-2-12" class="xref-bibr" href="#ref-2">2</a>]).</p><p id="p-64">To construct the non-uniform refinement algorithm in this work, we have considered only one specific family of regularizers. It would be possible, within the same framework, to consider other families that may regularize even more effectively. For example, one could look to the extensive denoising literature for different regularizers.</p><p id="p-65">In this paper, we have focused on results demonstrating the applicability of non-uniform refinement to membrane proteins, which are a large class with especially high importance in drug discovery. The algorithm, however, is general, and provides a useful approach for other proteins with local variation in physical properties.</p><p id="p-66">A beta version of the non-uniform refinement algorithm was released in previous versions of <em>cryoSPARC</em> [<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">22</a>]. Indeed, it has been already been successful in helping to resolve a number of interesting new structures. Examples include a structure of FACT manipulating the nucleasome [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], multiple conformations of an ABC exporter [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], mTORC1 docked on the lysosome [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], the respiratory syncytial virus polymerase complex [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], a GPCR-G protein-<em>β</em>-arrestin megacomplex [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], and MERS-CoV/SARS-CoV with neutralizing antibodies [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. An updated non-uniform refinement algorithm implementation, as described in this work, will be released in an upcoming version of <em>cryoSPARC</em> (<a href="http://www.cryosparc.com">www.cryosparc.com</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-67">We are extraordinarily grateful to Oliver Clarke and Yong Zi Tan for providing valuable cryo-EM data early in this project, and for sharing their experience as early adopters of non-uniform refinement and cryoSPARC. We thank the entire team at Structura Biotechnology Inc. that designs, develops, and maintains the cryoSPARC software system in which this project was implemented and tested. We thank John Rubinstein for comments on the manuscript. This work was financially supported in part by NSERC Canada and by the Canadian Institute for Advanced Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-68"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label"><sup>1</sup></span> Two key factors directly influence the choice of <em>ρ</em>(<em>x</em>) for estimating <em>θ</em>(<em>x</em>) reliably. The first is noise in the raw reconstruction. Aggregating squared error measures the residual variance (the residual power); larger windows provide more accurate variance estimates. The second is residual signal structure. When over-regularized, some high-frequency signal is lost, in which case signal residual is expected to have significant amplitudes near the wavelength <em>θ</em>. (Higher frequencies with insignificant amplitudes are dominated by noise, while lower frequencies are left unchanged by the regularizer and thus cancel the signal in the opposite half-map.) Integrating a squared band-pass signal provides a phase-independent measure of the local signal power; in our case the minimum window <em>ρ</em>(<em>x</em>) is determined by pass-band of the residual signal, which is expected to be close to <em>θ</em>. Thus the minimum <em>ρ</em>(<em>x</em>) is naturally constrained to be small multiple of <em>θ</em>.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.1" data-doi="10.1016/j.ymeth.2016.02.018"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Bell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Baldwin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Ludtke</span></span>. <span class="cit-article-title">High resolution single particle refinement in EMAN2.1</span>. <abbr class="cit-jnl-abbrev">Methods</abbr>, <span class="cit-vol">100</span>:<span class="cit-fpage">25</span>–<span class="cit-lpage">34</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D100%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2016.02.018%26rft_id%253Dinfo%253Apmid%252F26931650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2016.02.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26931650&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.2" data-doi="10.1016/j.jsb.2013.08.002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Cardone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Heymann</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Steven</span></span>. <span class="cit-article-title">One number does not fit all: Mapping local variations in resolution in cryo-em reconstructions</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">184</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">226</span>–<span class="cit-lpage">236</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStructural%2BBiology%26rft.volume%253D184%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2013.08.002%26rft_id%253Dinfo%253Apmid%252F23954653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2013.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23954653&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.3" data-doi="10.1016/j.ultramic.2013.06.004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McMullan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Faruqi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. N.</span> <span class="cit-name-surname">Murshudov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Short</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. H. W.</span> <span class="cit-name-surname">Scheres</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Henderson</span></span>. <span class="cit-article-title">High-resolution noise substitution to measure overfitting and validate resolution in 3d structure determination by single particle electron cryomicroscopy</span>. <abbr class="cit-jnl-abbrev">Ultramicroscopy</abbr>, <span class="cit-vol">135</span>:<span class="cit-fpage">24</span>–<span class="cit-lpage">35</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUltramicroscopy%26rft.volume%253D135%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ultramic.2013.06.004%26rft_id%253Dinfo%253Apmid%252F23872039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ultramic.2013.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23872039&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326941500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.4" data-doi="10.1126/science.aad8266"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">O. B.</span> <span class="cit-name-surname">Clarke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Stowe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-K.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Assur</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cavalier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Godoy-Ruiz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">von Alpen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Manzini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Blaner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Frank</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Quadro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Weber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Shapiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Hendrickson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Mancia</span></span>. <span class="cit-article-title">Structure of the stra6 receptor for retinol uptake</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">353</span>:<span class="cit-fpage">aad8266</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D353%26rft.issue%253D6302%26rft.spage%253Daad8266%26rft.epage%253Daad8266%26rft.atitle%253DStructure%2Bof%2Bthe%2BSTRA6%2Breceptor%2Bfor%2Bretinol%2Buptake%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad8266%26rft_id%253Dinfo%253Apmid%252F27563101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTMvNjMwMi9hYWQ4MjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTYvMjAxOS4xMi4xNS44NzcwOTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.5" data-doi="10.1016/j.cell.2015.03.049"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cheng</span></span>. <span class="cit-article-title">Single-Particle Cryo-EM at Crystallographic Resolution</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">161</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">450</span>–<span class="cit-lpage">457</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.03.049%26rft_id%253Dinfo%253Apmid%252F25910205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.03.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25910205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Clarke</span></span>. <abbr class="cit-jnl-abbrev">Personal Communication</abbr>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.7" data-doi="10.2307/2984875"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Dempster</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Laird</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Rubin</span></span>. <span class="cit-article-title">Maximum likelihood from incomplete data via the EM algorithm</span>. <abbr class="cit-jnl-abbrev">Journal of the Royal Statistical Society, Series B</abbr>, <span class="cit-vol">39</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>, <span class="cit-pub-date">1977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%252C%2BSeries%2BB%26rft.volume%253D39%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2984875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2984875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977DM46400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.8" data-doi="10.1109/78.969520"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Felsberg</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sommer</span></span>. <span class="cit-article-title">The monogenic signal</span>. <abbr class="cit-jnl-abbrev">IEEE transactions on Signal Processing</abbr>, <span class="cit-vol">49</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">3136</span>–<span class="cit-lpage">3144</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2BSignal%2BProcessing%26rft.volume%253D49%26rft.spage%253D3136%26rft_id%253Dinfo%253Adoi%252F10.1109%252F78.969520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/78.969520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. S. A.</span> <span class="cit-name-surname">Gilman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Behera</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jordan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rigaux</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ysebaert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tcherniuk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sourimant</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-F.</span> <span class="cit-name-surname">Eléouët</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sutto-Ortiz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Decroly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Roymans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Jin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">McLellan</span></span>. <span class="cit-article-title">Structure of the Respiratory Syncytial Virus Polymerase Complex</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">179</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">193</span>–<span class="cit-lpage">204</span>.e14, <span class="cit-month">sep</span> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D179%26rft.spage%253D193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.10" data-doi="10.2307/1268518"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Golub</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Heath</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wahba</span></span>. <span class="cit-article-title">Generalized cross-validationas as a method for choosing a good ridge parameter</span>. <abbr class="cit-jnl-abbrev">Technometrics</abbr>, <span class="cit-vol">21</span>:<span class="cit-fpage">215</span>–<span class="cit-lpage">223</span>, <span class="cit-pub-date">1979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D21%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GW16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.11" data-doi="10.7554/eLife.35383"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Grant</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rohou</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Grigorieff</span></span>. <span class="cit-article-title"><em>cis</em>tem, user-friendly software for single-particle image processing</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-vol">7</span>:<span class="cit-fpage">e35383</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De35383%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.35383%26rft_id%253Dinfo%253Apmid%252F29513216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.35383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29513216&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.12" data-doi="10.1016/j.jsb.2006.05.004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Grigorieff</span></span>. <span class="cit-article-title">FREEALIGN: High resolution refinement of single particle structures</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">157</span>:<span class="cit-fpage">117</span>–<span class="cit-lpage">125</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DGrigorieff%26rft.auinit1%253DN.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D125%26rft.atitle%253DFREALIGN%253A%2Bhigh-resolution%2Brefinement%2Bof%2Bsingle%2Bparticle%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2006.05.004%26rft_id%253Dinfo%253Apmid%252F16828314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2006.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16828314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243391100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Harauz</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">van Heel</span></span>. <span class="cit-article-title">Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy</span>. <abbr class="cit-jnl-abbrev">Optik</abbr>, <span class="cit-vol">73</span>:<span class="cit-fpage">146</span>–<span class="cit-lpage">156</span>, <span class="cit-pub-date">1986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOptik%26rft.volume%253D73%26rft.spage%253D146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.14" data-doi="10.1038/s41586-019-1391-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hofmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Januliene</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Mehdipour</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Thomas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Stefan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brüchert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Kuhn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Geertsma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hummer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tampé</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Moeller</span></span>. <span class="cit-article-title">Conformation space of a heterodimeric ABC exporter under turnover conditions</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">571</span>(<span class="cit-issue">7766</span>):<span class="cit-fpage">580</span>–<span class="cit-lpage">583</span>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D571%26rft.spage%253D580%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1391-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-019-1391-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. Z.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wicht</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure and drug resistance of the plasmodium falciparum transporter pfcrt</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-month">Nov</span>. <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.16"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kucukelbir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sigworth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Tagare</span></span>. <span class="cit-article-title">A Bayesian adaptive basis algorithm for single particle reconstruction</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">179</span>:<span class="cit-fpage">56</span>–<span class="cit-lpage">67</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DKucukelbir%26rft.auinit1%253DA.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D67%26rft.atitle%253DA%2BBayesian%2Badaptive%2Bbasis%2Balgorithm%2Bfor%2Bsingle%2Bparticle%2Breconstruction.%26rft_id%253Dinfo%253Apmid%252F22564910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22564910&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kucukelbir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sigworth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Tagare</span></span>. <span class="cit-article-title">Quantifying the local resolution of cryo-em density maps</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">11</span>:<span class="cit-fpage">63</span>–<span class="cit-lpage">65</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D11%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lehtinen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Munkberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hasselgren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Laine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Karras</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Aittala</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Aila</span></span>. <span class="cit-article-title">Noise2Noise: Learning image restoration without clean data</span>. <abbr class="cit-jnl-abbrev">International Conference on Machine Learning</abbr>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. Z.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Carragher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Potter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">D’Arcy</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Luger</span></span>. <span class="cit-article-title">FACT caught in the act of manipulating the nucleosome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R. B.</span> <span class="cit-name-surname">Thomsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Cahill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-Y.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Marcink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. B.</span> <span class="cit-name-surname">Clarke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Heissel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Masoudi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ben-Hail</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Samaan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. P.</span> <span class="cit-name-surname">Dandey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. Z.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mahoney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Triest</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Little</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sunahara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Steyaert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Molina</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. des</span> <span class="cit-name-surname">Georges</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Lefkowitz</span></span>. <span class="cit-article-title">Structure of an endosomal signaling GPCR–G protein–<em>β</em>-arrestin megacomplex</span>. <abbr class="cit-jnl-abbrev">Nature Structural &amp; Molecular Biology</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1123</span>–<span class="cit-lpage">1131</span>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D26%26rft.spage%253D1123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.21" data-doi="10.1038/nature14367"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Paulsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Armache</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cheng</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Julius</span></span>. <span class="cit-article-title">Structure of the TRPA1 ion channel suggests regulatory mechanisms</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">520</span>:<span class="cit-fpage">511</span>–<span class="cit-lpage">517</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D520%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14367%26rft_id%253Dinfo%253Apmid%252F25855297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25855297&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Punjani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Rubinstein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Fleet</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Brubaker</span></span>. <span class="cit-article-title">CryoSPARC: Algorithms for rapid unsupervised cryo-em structure determination</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">14</span>:<span class="cit-fpage">290</span>–<span class="cit-lpage">296</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D14%26rft.spage%253D290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ramlaul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Palmer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. H. S.</span> <span class="cit-name-surname">Aylett</span></span>. <span class="cit-article-title">A Local Agreement Filtering Algorithm for Transmission EM Reconstructions</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">205</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStructural%2BBiology%26rft.volume%253D205%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.24" data-doi="10.1126/science.aay0166"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Rogala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Kedir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abu-Remaileh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Bianchi</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bottino</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Dueholm</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Niehaus</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Overwijn</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Priso Fils</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">S. X.</span> <span class="cit-name-surname">Zhou</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Leary</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. N.</span> <span class="cit-name-surname">Laqtom</span></span>1, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Brignole</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Sabatini</span></span>. <span class="cit-article-title">Structural basis for the docking of mTORC1 on the lysosomal surface</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">366</span>(<span class="cit-issue">6464</span>):<span class="cit-fpage">468</span>–<span class="cit-lpage">475</span>, <span class="cit-month">oct</span> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aay0166%26rft_id%253Dinfo%253Apmid%252F31601708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjYvNjQ2NC80NjgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xNi8yMDE5LjEyLjE1Ljg3NzA5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.25"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Rosenthal</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Henderson</span></span>. <span class="cit-article-title">Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy</span>. <abbr class="cit-jnl-abbrev">J. Molecular Biology</abbr>, <span class="cit-vol">333</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">721</span>–<span class="cit-lpage">745</span>, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMolecular%2BBiology%26rft.volume%253D333%26rft.spage%253D721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.26" data-doi="10.1016/j.jmb.2011.11.010"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. H. W.</span> <span class="cit-name-surname">Scheres</span></span>. <span class="cit-article-title">A Bayesian view on cryo-em structure determination</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>, <span class="cit-vol">415</span>:<span class="cit-fpage">406</span>–<span class="cit-lpage">418</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DScheres%26rft.auinit1%253DS.%2BH.%26rft.volume%253D415%26rft.issue%253D2%26rft.spage%253D406%26rft.epage%253D418%26rft.atitle%253DA%2BBayesian%2Bview%2Bon%2Bcryo-EM%2Bstructure%2Bdetermination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2011.11.010%26rft_id%253Dinfo%253Apmid%252F22100448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2011.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22100448&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.27" data-doi="10.1016/j.jsb.2012.09.006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. H. W.</span> <span class="cit-name-surname">Scheres</span></span>. <span class="cit-article-title">RELION: Implementation of a Bayesian approach to cryo-em structure determination</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">180</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">519</span> – <span class="cit-lpage">530</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DScheres%26rft.auinit1%253DS.%2BH.%26rft.volume%253D180%26rft.issue%253D3%26rft.spage%253D519%26rft.epage%253D530%26rft.atitle%253DRELION%253A%2BImplementation%2Bof%2Ba%2BBayesian%2Bapproach%2Bto%2Bcryo-EM%2Bstructure%2Bdetermination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2012.09.006%26rft_id%253Dinfo%253Apmid%252F23000701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2012.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23000701&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. H. W.</span> <span class="cit-name-surname">Scheres</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span>. <span class="cit-article-title">Prevention of overfitting in cryo-EM structure determination</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">853</span>–<span class="cit-lpage">854</span>, <span class="cit-month">sep</span> <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D9%26rft.spage%253D853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.29" data-doi="10.1016/j.str.2017.12.018"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Vilas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gomez-Blanco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Conesa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Melero</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">de la Rosa-Trevín</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Oton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuenca</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Marabini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Carazo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Vargas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. O. S.</span> <span class="cit-name-surname">Sorzano</span></span>. <span class="cit-article-title">Monores: Automatic and accurate estimation of local resolution for electron microscopy maps</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">337</span>–<span class="cit-lpage">344</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D26%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2017.12.018%26rft_id%253Dinfo%253Apmid%252F29395788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2017.12.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29395788&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vonesch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shkolnisky</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Singer</span></span>. <span class="cit-article-title">Fast wavelet-based single particle reconstruction in cryo-em</span>. <abbr class="cit-jnl-abbrev">In International Symposium on Biomedical Imaging</abbr>, pages <span class="cit-fpage">1950</span>–<span class="cit-lpage">1953</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.31" data-doi="10.1214/aos/1176349743"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wahba</span></span>. <span class="cit-article-title">A comparison of GCV and GML for choosing the smoothing parameter in the generalized spline smoothing problem</span>. <abbr class="cit-jnl-abbrev">The Annals of Statistics</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1378</span>–<span class="cit-lpage">1402</span>, <span class="cit-pub-date">1985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BStatistics%26rft.volume%253D13%26rft.spage%253D1378%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176349743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176349743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AXY8900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.32" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Walls</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xiong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-J.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Tortorici</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Snijder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Quispe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Cameroni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gopal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lanzavecchia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zambon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Rey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Corti</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Veesler</span></span>. <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">176</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1026</span>–<span class="cit-lpage">1039</span>.e15, <span class="cit-month">feb</span> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.33"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rohou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Arthur</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tzakoniati</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Wong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Estevez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kugel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Franke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ciferri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Hackos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Koth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Payandeh</span></span>. <span class="cit-article-title">Structural basis of nav1.7 inhibition by a gating-modifier spider toxin</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zheng</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Doerschuk</span></span>. <span class="cit-article-title">An ab initio algorithm for low-resolution 3d reconstructions from cryoelectron microscopy images</span>. <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>, <span class="cit-vol">133</span>:<span class="cit-fpage">132</span>–<span class="cit-lpage">142</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DYin%26rft.auinit1%253DZ.%26rft.volume%253D133%26rft.issue%253D2-3%26rft.spage%253D132%26rft.epage%253D142%26rft.atitle%253DAn%2Bab%2Binitio%2Balgorithm%2Bfor%2Blow-resolution%2B3-D%2Breconstructions%2Bfrom%2Bcryoelectron%2Bmicroscopy%2Bimages.%26rft_id%253Dinfo%253Apmid%252F11472085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11472085&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.15.877092v1.35" data-doi="10.7554/eLife.42166"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zivanov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nakane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Forsberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kimanius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hagen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lindahl</span></span>. <span class="cit-article-title">New tools for automated high-resolution cryo-em structure determination in RELION-3</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">e42166</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De42166%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.42166%26rft_id%253Dinfo%253Apmid%252F30412051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.42166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30412051&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F16%2F2019.12.15.877092.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.15.877092v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.15.877332v1" title="RhoGAP RGA-8 supports morphogenesis in C. elegans by polarizing epithelia through CDC-42" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 16, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.15.877092v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.15.877092v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.15.877092v1" class="active" data-icon-position="" data-hide-link-title="0">Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1040133" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NyGnZ-HM9q3xNuX1jZGOjWbAjVWfbatkIWush-vEATE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1040133" id="node1040133--2" data-pisa="biorxiv;2019.12.15.877092v1" data-pisa-master="biorxiv;2019.12.15.877092" data-seqnum="1040133" data-apath="/biorxiv/early/2019/12/16/2019.12.15.877092.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ali</span> <span class="nlm-surname">Punjani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Haowei</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Fleet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.15.877092; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.15.877092 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;title=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;title=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;text=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;title=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;t=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;title=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;title=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1040133" id="node1040133--3" data-pisa="biorxiv;2019.12.15.877092v1" data-pisa-master="biorxiv;2019.12.15.877092" data-seqnum="1040133" data-apath="/biorxiv/early/2019/12/16/2019.12.15.877092.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ali</span> <span class="nlm-surname">Punjani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Haowei</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Fleet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.15.877092; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.15.877092 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Punjani%2C+Ali&amp;rft.au=Zhang%2C+Haowei&amp;rft.au=Fleet%2C+David+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1040133/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1040133/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1040133/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1040133/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1040133/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1040133/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1040133/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1040133/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1040133/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1040133/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1040133/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1040133/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;count=horizontal&amp;via=&amp;text=Non-uniform%20refinement%3A%20Adaptive%20regularization%20improves%20single%20particle%20cryo-EM%20reconstruction&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.15.877092v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.15.877092v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">Biophysics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
