<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virus Evol</span><span class="iso-abbrev" title="iso-abbrev">Virus Evol</span><span class="publisher-id" title="publisher-id">vevolu</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virus Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2057-1577</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6057501">6057501</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/ve/vey021">10.1093/ve/vey021</a></span><span class="publisher-id" title="publisher-id">vey021</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Exploratory re-encoding of yellow fever virus genome: new insights for the design of live-attenuated viruses</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7777-0308</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klitting</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><a href="#vey021-cor1" /><a href="#vey021-FM1" /><a href="#vey021-aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riziki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><a href="#vey021-aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moureau</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><a href="#vey021-aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piorkowski</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><a href="#vey021-aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">E A</span></span><a href="#vey021-aff1" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7895-2720</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Lamballerie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><a href="#vey021-FM2" /><a href="#vey021-aff1" /></span></div><span class="citation_author_institution" id="vey021-aff1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Corresponding author: E-mail: <span class="email" tagx="email" title="email">raphaelle.klitting@posteo.de</span></div> <div class="fn-type-" title=""> <p><a href="http://orcid.org/0000-0002-7777-0308">http://orcid.org/0000-0002-7777-0308</a></p> </div> <div class="fn-type-" title=""> <p><a href="http://orcid.org/0000-0001-7895-2720">http://orcid.org/0000-0001-7895-2720</a></p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-7</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-7-7</span></span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">vey021</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</span></span></div><a class="self-uri" href="vey021.pdf" title="self-uri">vey021.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Virus attenuation by genome re-encoding is a pioneering approach for generating effective live-attenuated vaccine candidates. Its core principle is to introduce a large number of synonymous substitutions into the viral genome to produce stable attenuation of the targeted virus. Introduction of large numbers of mutations has also been shown to maintain stability of the attenuated phenotype by lowering the risk of reversion and recombination of re-encoded genomes. Identifying mutations with low fitness cost is pivotal as this increases the number that can be introduced and generates more stable and attenuated viruses. Here, we sought to identify mutations with low deleterious impact on the <i>in vivo</i> replication and virulence of yellow fever virus (YFV). Following comparative bioinformatic analyses of flaviviral genomes, we categorised synonymous transition mutations according to their impact on CpG/UpA composition and secondary RNA structures. We then designed seventeen re-encoded viruses with 100–400 synonymous mutations in the NS2A-to-NS4B coding region of YFV <i>Asibi</i> and <i>Ap7M</i> (hamster-adapted) genomes. Each virus contained a panel of synonymous mutations designed according to the above categorisation criteria. The replication and fitness characteristics of parent and re-encoded viruses were compared <i>in vitro</i> using cell culture competition experiments. <i>In vivo</i> laboratory hamster models were also used to compare relative virulence and immunogenicity characteristics. Most of the re-encoded strains showed no decrease in replicative fitness <i>in vitro</i>. However, they showed reduced virulence and, in some instances, decreased replicative fitness <i>in vivo</i>. Importantly, the most attenuated of the re-encoded strains induced robust, protective immunity in hamsters following challenge with <i>Ap7M</i>, a virulent virus. Overall, the introduction of transitions with no or a marginal increase in the number of CpG/UpA dinucleotides had the mildest impact on YFV replication and virulence <i>in vivo</i>. Thus, this strategy can be incorporated in procedures for the finely tuned creation of substantially re-encoded viral genomes.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
yellow fever virus</div> <div class="kwd" title="kwd">
attenuation</div> <div class="kwd" title="kwd">
genome re-encoding</div> <div class="kwd" title="kwd">
Flavivirus</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">European Virus Archive goes Global</span></div><span class="award-id" tagx="award-id" title="award-id">ANR-14CE14-0001</span></div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The <i>Flavivirus</i> genus (family <i>Flaviviridae)</i> includes fifty-three taxonomically recognised species and at least nineteen ‘tentative’ species (<a href="#vey021-B35">King and Am 2014</a>). The flaviviruses display an exceptional diversity of ecological networks that widely correlate with the phylogenetic relationships within the genus (<a href="#vey021-B79">Zanotto et al. 1995</a>; <a href="#vey021-B19">Gaunt et al. 2001</a>; <a href="#vey021-B22">Grard et al. 2007</a>; <a href="#vey021-B21">Gould and Solomon 2008</a>). To date, the majority of recognised flaviviruses are arthropod-borne viruses (arboviruses). They are transmitted among vertebrate hosts by mosquitoes (MBFV), ticks, or sandflies (TBFV). The genus also includes viruses with no known vector (‘NKV’) and some that infect only insects (insect-specific flaviviruses, ‘ISFV’) (<a href="#vey021-B40">Kuno et al. 1998</a>; <a href="#vey021-B56">Moureau et al. 2015</a>). Several important (re)-emerging human pathogens fall within the MBFV and TBFV groups, notably dengue virus (DENV), yellow fever virus (YFV), tick-borne encephalitis virus (TBEV), Japanese encephalitis virus (JEV), West Nile virus (WNV), and the recently emerged, Zika virus (ZIKV). Flaviviruses are spherical, enveloped particles (virions) of <i>ca.</i> 50 nm diameter, incorporating single-stranded positive sense RNA genomes. The capped, eleven kilobase genome includes a single open reading frame (ORF), flanked at its 5′ and 3′ termini by structured, non-coding regions, essential for viral RNA translation and replication (<a href="#vey021-B59">Ng et al. 2017</a>). ORF translation gives rise to a polyprotein that is cleaved co- and post-translationally into the structural (C-prM-E) and seven non-structural proteins (NS1-2A-2B-3-4A-4B-5) (<a href="#vey021-B6">Bell et al. 1985</a>; <a href="#vey021-B63">Rice et al. 1985</a>).</p> <p>YFV is the type species of the genus <i>Flavivirus</i>, that owes its name (<i>flavus</i> is the Latin word for yellow) to the jaundice associated with the liver dysfunction characteristic of yellow fever disease. In humans, YFV primarily infects the liver, often resulting in severe viscerotropic infections and high fever (<a href="#vey021-B53">Monath and Barrett 2003</a>). The severity of YF infections ranges from ‘inapparent’ (i.e., sub-clinical) to fatal haemorrhagic disease with a mortality rate between 20 and 50 per cent among symptomatic cases (<a href="#vey021-B51">Monath 2008</a>). YFV originated in the tropical African forests where non-human primates (NHP) and a variety of <i>Aedes</i> spp<i>.</i> mosquitoes, inhabit the forest canopy, providing the hosts and vectors for viral maintenance through a sylvatic enzootic transmission cycle. Human–mosquito–human transmission cycles usually arise incidentally when humans encroach on the forest or neighbouring savannah environment and then inadvertently take the virus back to the villages or towns/cities, leading to urban epidemics that can be extensive. The most recent of these large-scale African epidemics happened in Angola and Democratic Republic of Congo in 2015 and 2016 (<a href="#vey021-B2">Ahmed and Memish 2017</a>; <a href="#vey021-B38">Kraemer et al. 2017</a>).YFV transmission is also maintained in the tropical forests of South-America. However, in this case the sylvatic transmission cycle is epizootic and human cases are primarily ‘spill-over’ events from these sylvatic outbreaks (<a href="#vey021-B20">Germain et al. 1981</a>; <a href="#vey021-B16">Ellis and Barrett 2008</a>; <a href="#vey021-B12">Carrington and Auguste 2013</a>; <a href="#vey021-B28">Hanley et al. 2013</a>; <a href="#vey021-B49">Mir et al. 2017</a>), as exemplified by the recent epidemic that occurred in Brazil in 2016–7 (<a href="#vey021-B49">Mir et al. 2017</a>).</p> <p>Over the past five or six centuries YFV has killed millions of humans and primates, hence the need for effective methods of prevention. As a response to this need, the live-attenuated vaccine (strain 17D) was developed in 1936 by M. Theiler (<a href="#vey021-B73">Theiler and Smith 1937</a>). It was first tested in 1938 in Brazil (<a href="#vey021-B5">Barrett 2012</a>; <a href="#vey021-B52">Monath 2012</a>) and two substrains (17DD and 17D-204) are still widely used as the source for vaccine manufacture (<a href="#vey021-B17">Ferguson et al. 2010</a>) in Brazil (17DD) and the Old World (17D-204). Throughout decades of use of 17 D vaccine, with twenty to sixty million doses distributed annually (<a href="#vey021-B54">Monath and Vasconcelos 2015</a>), YF vaccination has proven to be safe and effective, providing long-lasting immunity (<a href="#vey021-B50">Monath 2005</a>) with rare adverse events (<a href="#vey021-B54">Monath and Vasconcelos 2015</a>). The YF 17 D strain was obtained through serial passages (&amp;gt;200) of the wild-type (WT) <i>Asibi</i> strain in mouse and chicken embryo tissues (<a href="#vey021-B50">Monath 2005</a>). Similar empirical methods have also been used for the production of notable live-attenuated vaccines against poliovirus (<a href="#vey021-B65">Sabin and Boulger 1973</a>), measles (<a href="#vey021-B66">Schwarz 1962</a>; <a href="#vey021-B29">Hilleman et al. 1968</a>; <a href="#vey021-B31">Ikic et al. 1972</a>), and mumps (<a href="#vey021-B11">Buynak and Hilleman 1966</a>). This strategy relies on attenuation resulting from a relatively limited number of attenuating mutations that are, most often, non-synonymous. However, with all of these live-attenuated viruses there is a risk of (1) attenuation reversion, as described in the case of the poliovirus vaccine (<a href="#vey021-B48">Minor 2012</a>); (2) generation of new biological properties, as illustrated by the gain of neurovirulence observed for the YFV French neurotropic vaccine strain (<a href="#vey021-B76">Wang et al. 1995</a>); and (3) recombination, as documented for poliovirus (<a href="#vey021-B44">Lukashev et al. 2003</a>; <a href="#vey021-B48">Minor 2012</a>; <a href="#vey021-B43">Liu et al. 2016</a>).</p> <p>A new, promising, approach for virus attenuation called codon re-encoding, was developed a decade ago by <a href="#vey021-B10">Burns et al. (2006)</a> and <a href="#vey021-B57">Mueller et al. (2006)</a>. Based on the initial observation that usage among synonymous codons is highly non-random in the genome of viruses, they hypothesised that this equilibrium could be modified in a manner that should be relatively detrimental to virus replication. They provided the first evidence supporting this concept by introducing slightly detrimental synonymous codons within the encoding region of the poliovirus genome, that is, without modifying the encoded proteins after which they observed a decrease in viral replication capacity. This codon re-encoding strategy bypasses the limitations of empirical attenuation methods: importantly, the high number of mutations involved decreases the risk of reversion of the attenuating mutations and reduces the likelihood of vaccine strain recombination (with either vaccine or wild-type viruses). The use of silent mutations also lowers the risk of emergence of untoward biological properties.</p> <p>The first rationale for codon usage bias was that codon abundance correlated with that of isoaccepting tRNAs and could influence the level of protein production within a given host (<a href="#vey021-B30">Ikemura 1985</a>). Several other possibilities have been proposed since then, including implication in mRNA structure and its folding (<a href="#vey021-B67">Shabalina, Ogurtsov, and Spiridonov 2006</a>; <a href="#vey021-B39">Kudla et al. 2009</a>), microRNA-targeting (<a href="#vey021-B9">Brest et al. 2011</a>; <a href="#vey021-B8">Birnbaum et al. 2012</a>) and enhanced recognition by the immune system (<a href="#vey021-B75">Tulloch et al. 2014</a>), that may vary according to the host (<a href="#vey021-B23">Greenbaum et al. 2008</a>; <a href="#vey021-B32">Jimenez-Baranda et al. 2011</a>). Specific and random re-encoding approaches have been efficiently applied to several human RNA viruses including poliovirus, influenza A virus, human immunodeficiency virus, respiratory syncytial virus, chikungunya virus (CHIKV), JEV, TBEV, and DENV (<a href="#vey021-B10">Burns et al. 2006</a>; <a href="#vey021-B57">Mueller et al. 2006</a>; <a href="#vey021-B60">Nougairede et al. 2013</a>; <a href="#vey021-B15">De Fabritus et al. 2015</a>; <a href="#vey021-B45">Martinez et al. 2016</a>). Considering a major role of genome-wide mutational processes in the shaping of synonymous sites (<a href="#vey021-B13">Chen et al. 2004</a>), specific re-encoding approaches include codon and codon-pair deoptimisation as well as increase of CpG/UpA dinucleotide frequency. On the other hand, the efficiency of random codon re-encoding strategies for the attenuation of CHIKV <i>in vitro</i> and of TBEV <i>in vivo</i> suggests an important influence of local constraints (e.g., cis-acting sequences, miRNA targeting) on genome encoding (<a href="#vey021-B74">Thurner et al. 2004</a>; <a href="#vey021-B77">Watts et al. 2009</a>; <a href="#vey021-B60">Nougairede et al. 2013</a>; <a href="#vey021-B15">De Fabritus et al. 2015</a>).</p> <p>Much remains to be learned to understand the multiple mechanisms that shape synonymous codon usage and its contribution to the attenuation process during re-encoding. In this exploratory work, we defined several types of silent mutations and sought to identify which one(s) had the least detrimental impact on viral replicative fitness. In the future, such silent changes could be introduced in large numbers along the genome of vaccine candidates. YFV provides a suitable experimental model because (1) the virus can be produced conveniently and modified using the Infectious Subgenomic Amplicons (ISA) reverse genetics method (<a href="#vey021-B4">Aubry et al. 2014</a>) and (2) an excellent animal model of infection exists in juvenile Syrian Golden hamsters, to enable the comparison of wild-type and re-encoded strains <i>in vivo</i>. This model was established in our laboratory using the <i>Asibi</i>-derived, YF <i>Ap7M</i> strain (<a href="#vey021-B36">Klitting et al. 2018</a>) derived from the previously described hamster-virulent YF <i>Asibi</i>/hamster p7 strain (<a href="#vey021-B47">McArthur et al. 2003</a>). When inoculated into hamsters, both strains induce a disease with features close to that of the human disease.</p> <p>Starting with a bioinformatic analysis of flaviviral genomes, we defined several types of mutations and used them for the design of eight re-encoded sequences. The re-encoded viruses were derived from the wild-type <i>Asibi</i> strain and the hamster-adapted <i>Ap7M</i> strain, with which they were compared both <i>in vitro</i> and <i>in vivo</i> through fitness assays, infection assays and complete genome deep-sequencing. Although only the most heavily re-encoded virus (741 mutations) showed an observable decrease in replicative fitness <i>in vitro</i>, a range of attenuation levels was observed for the re-encoded variants when tested <i>in vivo</i>. The lowest impact was observed for strains that were re-encoded with transitions that did not include new CpG/UpA dinucleotides. In addition, we showed that the strains with the most attenuated phenotypes <i>in vivo</i> could induce robust protective immunity in hamsters following challenge with the lethal YF <i>Ap7M</i> virus.</p> </div> <div class="2.results" title="sec"> <div class="title" tagx="title" title="title">
2. Results</div> <p>The results of the <i>in silico</i> analysis that served as a basis for the design of the re-encoded strains are provided in the <a href="#sup1">Supplementary Results S2</a> section.</p> <div class="2.1designstrategiesforre-encodedstrains" title="sec"> <div class="title" tagx="title" title="title">
2.1 Design strategies for re-encoded strains</div> <div class="2.1.1frameworkdesign" title="sec"> <div class="title" tagx="title" title="title">
2.1.1 Framework design</div> <p>Previous analyses on the impact of random codon re-encoding on chikungunya (CHIKV) and tick-borne encephalitis (TBEV) viruses, were used as a starting point to obtain observable <i>in vitro</i> and/or <i>in vivo</i> attenuation. <a href="#vey021-B60">Nougairède et al. (2013)</a> observed a decrease in CHIKV replicative fitness <i>in vitro</i>, using 264–882 mutations located within the NSp1, NSp4 and/or envelope protein encoding regions. Clear <i>in vivo</i> attenuation of TBEV was achieved by <a href="#vey021-B15">de Fabritus et al. (2015</a>) using as few as 273 mutations located in the NS5 coding region.</p> <p>Based on those results, we defined a framework for the design of YFV re-encoded strains. For each variant, we used a maximum of 350 synonymous mutations (<i>ca.</i> one change for every ten nucleotides). We chose to re-encode the region located within the coding sequences of proteins NS2A to NS4B, so that re-encoding would not affect the structural proteins, the viral RNA-dependent RNA polymerase, or the highly structured, 5′ and 3′ ends of the genome. The highly re-encoded variant rTs4 was reconstructed by the addition of 388 synonymous mutations in the NS5 encoding region. In all cases, the structural proteins encoding sequence remained unchanged, which was an advantage as the production of hamster-virulent YFV variants required the use of a modified E protein sequence. To lower the risk of major detrimental effects of re-encoding (e.g., disruption of cis-acting sequences), we analysed an alignment of thirty-five YFV complete coding sequences (CDS) and defined only synonymous sites for which at least one mutation could be observed as eligible for re-encoding. These will be referred to as ‘editable sites’. Finally, we avoided creating or removing any rare codons during the re-encoding process (see definition in <a href="#sup1">Supplementary Protocols S1</a>).</p> </div> <div class="2.1.2re-encodingstrategies" title="sec"> <div class="title" tagx="title" title="title">
2.1.2 Re-encoding strategies</div> <p>Following <i>in silico</i> analysis of flaviviral genomes, we defined two major types of synonymous mutations (see below). The creation of additional TCG trinucleotide patterns was considered a ‘specific’ type of mutation. In the genus <i>Flavivirus</i>, TCG trinucleotides are frequent in Insect-Specific Flavivirus (ISF) genomes and sparse in both No Known Vector virus (NKV) and arbovirus genomes (see <a href="#sup1">Supplementary Results</a>). In addition, this pattern may enhance the innate immunity in vertebrate cells, as suggested by Greenbaum, Jimenez-Baranda and colleagues (<a href="#vey021-B24">Greenbaum, Rabadan, and Levine 2009</a>; <a href="#vey021-B32">Jimenez-Baranda et al. 2011</a>). ‘Specific’ mutations also included the creation of CpG and UpA dinucleotides, which were the least frequently introduced dinucleotides (CpG and UpA) in YFV species (see <a href="#sup1">Supplementary Results</a>). This is in accordance with the previous observation of dinucleotide bias in the genomes of many viruses, in a fashion that was suggested to, at least partly, mirror the dinucleotide usage of their host(s) <a href="#vey021-B34">(Karlin, Doerfler, and Cardon 1994</a>; <a href="#vey021-B64">Rima and McFerran 1997</a>). In vertebrates, CpG dinucleotide bias is commonly seen as the consequence of cytosine methylation-deamination (<a href="#vey021-B7">Bird 1980</a>; <a href="#vey021-B37">Kow 2002</a>; <a href="#vey021-B69">Simmen 2008</a>) and the detection of such patterns may be a part of the host-defence mechanism against non-self RNA (<a href="#vey021-B33">Johnson 1970</a>; <a href="#vey021-B42">Leyssen et al. 2006</a>; <a href="#vey021-B1">Adams et al. 2013</a>). The introduction of CpG and UpA dinucleotides has been frequently used in re-encoding studies (<a href="#vey021-B3">Atkinson et al. 2014</a>; <a href="#vey021-B75">Tulloch et al. 2014</a>; <a href="#vey021-B18">Gaunt et al. 2016</a>; <a href="#vey021-B78">Witteveldt, Martin-Gans, and Simmonds 2016</a>). By contrast, ‘non-specific’ types of mutations did not cause any increase in CpG/UpA dinucleotides. This category includes transitions located on and outside sites associated with the putative secondary structures identified in the YFV genome during <i>in silico</i> analysis (see <a href="#sup1">Supplementary Results</a>). All mutation types defined in this study are described in <a href="#vey021-T1">Table 1</a>. <div class="table-wrap_UNKNOWN" id="vey021-T1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> </div></p><p>Mutation categories.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1" /> <th align="left" rowspan="1" colspan="1">Non-specific</th> <th align="left" rowspan="1" colspan="1">Specific</th> </tr> </thead> <tbody> <tr> <td rowspan="2" colspan="1">Mutations</td> <td rowspan="1" colspan="1">Simple transitions</td> <td rowspan="1" colspan="1">CpG/UpA dinucleotide transitions</td> </tr> <tr> <td rowspan="1" colspan="1">Transitions on sites associated with putative secondary structures</td> <td rowspan="1" colspan="1">TCG trinucleotide introductions</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Note: Based on bioinformatic analysis, 2 major types of mutation were defined.</p> </div> </div>  <p /> </div> <div class="2.1.3design" title="sec"> <div class="title" tagx="title" title="title">
2.1.3 Design</div> <p>In total, nine re-encoded viruses were designed based on the <i>Asibi</i> strain sequence. Aside from the highly re-encoded rTs4 virus, all Asibi-derived re-encoded viruses were designed with 100–400 mutations located between positions 3,924 and 6,759 of YF <i>Asibi</i> complete coding sequence (CDS). First, YFV rTs3 was designed by introducing exclusively ‘simple’ transitions (siTs) into the target region, within the CDS of the reference strain <i>Asibi</i> (AY640589). The term ‘simple’ refers to synonymous, non-specific (no CpG/UpA introduction) mutations located outside putative secondary structures. YFV rTs3 included 353 substitutions, that is, the highest possible number that could be introduced into the target region. All other re-encoded viruses except YFV rTCG were designed starting from the rTs3 virus by (1) reverting some of the siTs back to the original sequence (rTs1, rTs2), or (2) substituting some of the siTs for a different type of mutation (rUA, rCG, rSS, and rN). The rationale for this design procedure was to obtain viruses that could be compared one to another, as they would share a common background of mutations.</p> <p>Overall the nine initial re-encoded viruses were designed as follows: </p><ul> <li> <p>YFV rTs1, rTs2, and rTs3, include increasing numbers of ‘simple’ transitions (siTs).</p> </li> <li> <p>YFV rSS includes non-specific transitions on sites corresponding to putative secondary structures (SII-Ts).</p> </li> <li> <p>YFV rUA and rCG include transitions leading to an increase in the number of UpA or CpG dinucleotides into the viral sequence (UA- and CG-Ts), respectively.</p> </li> <li> <p>YFV rN includes both transitions and transversions, some of which lead to the introduction of eighty-three UpA and fifty-four CpG dinucleotides into the viral sequence.</p> </li> <li> <p>YFV rTCG includes transitions leading to an increase in the number of TCG dinucleotides into the viral sequence (TCG-Ts). Importantly, this strain was not derived from rTs3 and exclusively includes TCG-Ts.</p> </li> <li> <p>YFV rTs4 includes 388 additional siTs that were introduced into the genome of rTs3 virus, between positions 6,846 and 9,765 (in CDS).</p> </li> </ul> <p /> <p>Finally, eight hamster-adapted re-encoded strains were created from the <i>Asibi</i>-derived variants (other than the heavily re-encoded rTs4 virus) by introducing exactly the same mutations in the CDS of the hamster-virulent, YFV <i>Ap7M</i> virus. For all re-encoded viruses, the design is further detailed in the ‘Section 4’.</p> <p>Following re-encoding, the GC% and Effective Number of Codons (ENC) were calculated for each sequence and compared with those obtained from a dataset of thirty-five YFV genome sequences (non-vaccine strains) using SSE software (v1.3) (<a href="#vey021-B70">Simmonds 2012</a>). Apart from the most re-encoded, rTs4 (GC%: 52.3%; ENC: 52.1), all the other strains fell within the YFV species range in terms of both GC% (48.9–50.5%) and ENC (52.4–54.7). Details regarding re-encoded strain design are available in <a href="#vey021-T2">Table 2</a>. <div class="table-wrap_UNKNOWN" id="vey021-T2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> </div></p><p>Asibi and Ap7M-derived re-encoded strain description.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">Name</th> <th align="left" rowspan="1" colspan="1">Description</th> <th rowspan="1" colspan="1">Transitions</th> <th rowspan="1" colspan="1">Transversions</th> <th rowspan="1" colspan="1">CpG-Ts</th> <th rowspan="1" colspan="1">UpA-Ts</th> <th rowspan="1" colspan="1">SII-Ts</th> <th rowspan="1" colspan="1">TCG-Ts</th> <th rowspan="1" colspan="1">NE sites-Ts</th> <th rowspan="1" colspan="1">Total</th> <th align="left" rowspan="1" colspan="1">GC content (per cent)</th> <th rowspan="1" colspan="1">ENC</th> <th align="left" rowspan="1" colspan="1">Titre (TCID50/ml)</th> <th align="left" rowspan="1" colspan="1">Name (<i>Ap7M</i>- derived viruses)</th> <th rowspan="1" colspan="1">Titre (TCID50/ml)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><b><i>Asibi</i></b></td> <td rowspan="1" colspan="1"><i>Asibi</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">49.7</td> <td rowspan="1" colspan="1">52.8</td> <td rowspan="1" colspan="1">1E+08</td> <td rowspan="1" colspan="1"><b><i>Ap7M</i></b></td> <td rowspan="1" colspan="1">9E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rTs1</i></b></td> <td rowspan="1" colspan="1">siTs-low</td> <td rowspan="1" colspan="1">101</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">101</td> <td rowspan="1" colspan="1">49.7</td> <td rowspan="1" colspan="1">52.9</td> <td rowspan="1" colspan="1">4E+05</td> <td rowspan="1" colspan="1"><b><i>rTs1h</i></b></td> <td rowspan="1" colspan="1">6E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rTs2</i></b></td> <td rowspan="1" colspan="1">siTs-intermediate</td> <td rowspan="1" colspan="1">247</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">247</td> <td rowspan="1" colspan="1">49.5</td> <td rowspan="1" colspan="1">52.7</td> <td rowspan="1" colspan="1">4E+07</td> <td rowspan="1" colspan="1"><b><i>rTs2h</i></b></td> <td rowspan="1" colspan="1">6E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rTs3</i></b></td> <td rowspan="1" colspan="1">siTs-high</td> <td rowspan="1" colspan="1">353</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">353</td> <td rowspan="1" colspan="1">49.4</td> <td rowspan="1" colspan="1">52.6</td> <td rowspan="1" colspan="1">4E+06</td> <td rowspan="1" colspan="1"><b><i>rTs3h</i></b></td> <td rowspan="1" colspan="1">6E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rUA</i></b></td> <td rowspan="1" colspan="1">UpA-Ts</td> <td rowspan="1" colspan="1">326</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">59</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">326</td> <td rowspan="1" colspan="1">49.6</td> <td rowspan="1" colspan="1">53.5</td> <td rowspan="1" colspan="1">3E+06</td> <td rowspan="1" colspan="1"><b><i>rUAh</i></b></td> <td rowspan="1" colspan="1">4E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rCG</i></b></td> <td rowspan="1" colspan="1">CpG-Ts</td> <td rowspan="1" colspan="1">325</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">59</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">19</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">325</td> <td rowspan="1" colspan="1">49.7</td> <td rowspan="1" colspan="1">52.8</td> <td rowspan="1" colspan="1">3E+06</td> <td rowspan="1" colspan="1"><b><i>rCGh</i></b></td> <td rowspan="1" colspan="1">4E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rSS</i></b></td> <td rowspan="1" colspan="1">Ts on putative secondary structures</td> <td rowspan="1" colspan="1">339</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">50</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">339</td> <td rowspan="1" colspan="1">49.7</td> <td rowspan="1" colspan="1">52.9</td> <td rowspan="1" colspan="1">7E+06</td> <td rowspan="1" colspan="1"><b><i>rSSh</i></b></td> <td rowspan="1" colspan="1">4E+08</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rN</i></b></td> <td rowspan="1" colspan="1">Combinatory strategy</td> <td rowspan="1" colspan="1">179</td> <td rowspan="1" colspan="1">54</td> <td rowspan="1" colspan="1">59</td> <td rowspan="1" colspan="1">83</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">17</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">233</td> <td rowspan="1" colspan="1">49.5</td> <td rowspan="1" colspan="1">52.7</td> <td rowspan="1" colspan="1">4E+06</td> <td rowspan="1" colspan="1"><b><i>rNh</i></b></td> <td rowspan="1" colspan="1">6E+06</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rTCG</i></b></td> <td rowspan="1" colspan="1">Ts to TCG</td> <td rowspan="1" colspan="1">46</td> <td rowspan="1" colspan="1">72</td> <td rowspan="1" colspan="1">111</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">12</td> <td rowspan="1" colspan="1">118</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">118</td> <td rowspan="1" colspan="1">49.4</td> <td rowspan="1" colspan="1">52.6</td> <td rowspan="1" colspan="1">2E+06</td> <td rowspan="1" colspan="1"><b><i>rTCGh</i></b></td> <td rowspan="1" colspan="1">6E+07</td> </tr> <tr> <td rowspan="1" colspan="1"><b><i>rTs4</i></b></td> <td rowspan="1" colspan="1">siTs-very high</td> <td rowspan="1" colspan="1">741</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">741</td> <td rowspan="1" colspan="1">52.3</td> <td rowspan="1" colspan="1">52.1</td> <td rowspan="1" colspan="1">6E+03</td> <td rowspan="1" colspan="1"><i>–</i></td> <td rowspan="1" colspan="1">–</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Note: For each YFV re-encoded strain, the number of mutations introduced within the CDS of the reference strain <i>Asibi</i> (Genbank AN: AY640589) are given both as a total and with details according to the type of mutation used. The following abbreviations were used: CpG-Ts: transition leading to CpG dinucleotide introduction, UpA-Ts: transition leading to UpA dinucleotide introduction; SII-Ts: transition on sites corresponding to putative secondary structures; NE sites-Ts: transition on sites for which no mutation has been reported in the 35 YFV CDS alignment. G+C content was calculated on viral CDS and is presented as percentage. For each viral CDS, the ENC was calculated using SEE software (v1.3) (64). Infectious titres were determined on BHK21 cells and are presented as TCID50/mL for both Asibi-derived viruses and the corresponding, <i>Ap7M</i>-derived viruses.</p> </div> </div>  <p /> </div> </div> <div class="2.2invitrobehaviourofyfvre-encodedstrains" title="sec"> <div class="title" tagx="title" title="title">
2.2 <i>In v</i><i>itro</i> behaviour of YFV re-encoded strains</div> <p>All the strains used in this study (one WT, one hamster-adapted, and seventeen re-encoded) were produced using the ISA method, as detailed in <a href="#vey021-F1">Fig. 1</a> and in the ‘Section 4’. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Subgenomic fragments used for wild-type and re-encoded virus production using the Infectious Subgenomic Amplicon method. *For <i>Ap7M</i> strain and hamster-adapted re-encoded strain (<i>Ap7M</i>-derived) production, the subgenomic fragment FIAp7 was amplified from the corresponding plasmid and combined to subgenomic fragments FII (wild-type or re-encoded) and FIII (wild-type). **For each of the YFV re-encoded strains (both <i>Asibi</i> and <i>Ap7M</i>-derived) production, a different subgenomic fragment FII was amplified from the corresponding plasmid. ***For the production of YFV re-encoded strain rTs4, the subgenomic fragment FIIIR was amplified from the corresponding plasmid and combined to fragments FI (wild-type) and FII (re-encoded).</p>   </div> <div class="2.2.1sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.2.1 Sequence analysis</div> <p>Next Generation Sequencing (NGS) was performed on the cell culture supernatant media (viral stocks, i.e., after three passages). For all viruses, both the integrity of the genome and the intra-population genetic diversity were assessed on the CDS. At the level of intra-population genetic diversity, a variant subpopulation (further referred to as ‘variant’) was regarded as major when its corresponding nucleotide proportion (CNP) was over 75 per cent. Only consensus mutations associated with major variants were considered for further analysis. All sequencing results are shown in <a href="#vey021-T3">Tables 3</a> and <a href="#vey021-T4">4</a>. <div class="table-wrap_UNKNOWN" id="vey021-T3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> </div></p><p>Sequence analysis of Asibi and Asibi-derived re-encoded viruses.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Strain name</th> <th align="left" rowspan="1" colspan="1">Re-encoded region</th> <th align="left" rowspan="1" colspan="1">Position (nt)</th> <th align="left" rowspan="1" colspan="1">Protein</th> <th align="left" rowspan="1" colspan="1">Consensus mutation</th> <th align="left" rowspan="1" colspan="1">Expected sequence (nt)</th> <th align="left" rowspan="1" colspan="1">Observed sequence (first variant (nt))</th> <th align="left" rowspan="1" colspan="1">Observed sequence (second variant (nt))</th> <th align="left" rowspan="1" colspan="1">First variant proportion (per cent)</th> <th align="left" rowspan="1" colspan="1">Second variant proportion (per cent)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><i>Asibi</i></td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs1</i></td> <td rowspan="1" colspan="1">3,927–6,753</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="2" colspan="1"><i>rTs2</i></td> <td rowspan="2" colspan="1">3,927–6,753</td> <td rowspan="1" colspan="1">5,430</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">77.5</td> <td rowspan="1" colspan="1">22.3</td> </tr> <tr> <td rowspan="1" colspan="1">5,442</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">82.0</td> <td rowspan="1" colspan="1">18.0</td> </tr> <tr> <td rowspan="2" colspan="1"><i>rTs3</i></td> <td rowspan="2" colspan="1">3,927–6,753</td> <td rowspan="1" colspan="1">4,357</td> <td rowspan="1" colspan="1">NS2B</td> <td rowspan="1" colspan="1">M99V</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">78.3</td> <td rowspan="1" colspan="1">21.7</td> </tr> <tr> <td rowspan="1" colspan="1">9,150</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">78.5</td> <td rowspan="1" colspan="1">21.5</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rUA</i></td> <td rowspan="1" colspan="1">3,927–6,753</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="4" colspan="1"><i>rCG</i></td> <td rowspan="4" colspan="1">3,924–6,750</td> <td rowspan="1" colspan="1">4,396</td> <td rowspan="1" colspan="1">NS2B</td> <td rowspan="1" colspan="1">F112L</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">88.2</td> <td rowspan="1" colspan="1">11.8</td> </tr> <tr> <td rowspan="1" colspan="1">5,430</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">5,667</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">6,082</td> <td rowspan="1" colspan="1">NS3</td> <td rowspan="1" colspan="1">C544R</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="5" colspan="1"><i>rSS</i></td> <td rowspan="5" colspan="1">3,939–6,759</td> <td rowspan="1" colspan="1">561</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">3,190</td> <td rowspan="1" colspan="1">NS1</td> <td rowspan="1" colspan="1">I286V</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">4,308</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">4,623</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">6,012</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rN</i></td> <td rowspan="1" colspan="1">3,924–6,750</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="6" colspan="1"><i>rTCG</i></td> <td rowspan="6" colspan="1">3,939–6,759</td> <td rowspan="1" colspan="1">2,468</td> <td rowspan="1" colspan="1">NS1</td> <td rowspan="1" colspan="1">I44T</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">2,795</td> <td rowspan="1" colspan="1">NS1</td> <td rowspan="1" colspan="1">Q154R</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">7,629</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">8,565</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">8,916</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">9,297</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">&amp;gt;95</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs4</i></td> <td rowspan="1" colspan="1">3,927–9,765</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Note: For each virus, the consensus changes found in the CDS from the viral stock are detailed above. The expected sequence refers to that of plasmids that served for subgenomic fragment amplification while producing viruses using the ISA method. Only consensus mutations associated with major variants (CNP&amp;gt;75%) are described. Amino Acid (AA) positions are given with reference to the beginning of mature protein sequence (i.e., AA position 107 in prM, written prM 107, corresponds to AA position 228 within the precursor polyprotein). A detailed map of protein positions within the YFV polyprotein sequence is provided in the <a href="#sup1">supplementary Table S11</a>.</p> </div> </div>  <div class="table-wrap_UNKNOWN" id="vey021-T4" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Sequence analysis of Asibi, and both Asibi and Ap7M-derived re-encoded viruses.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="char" char="." span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">VIRUS</th> <th rowspan="1" colspan="1">Consensus mutations</th> <th rowspan="1" colspan="1">Consensus NS mutations</th> <th rowspan="1" colspan="1">Variants</th> <th rowspan="1" colspan="1">Reversions</th> <th rowspan="1" colspan="1">Consensus mutations in re-encoded region</th> <th rowspan="1" colspan="1">Variants in reencoded region</th> <th rowspan="1" colspan="1">Total no. reads</th> <th rowspan="1" colspan="1">Coverage depth (x)</th> <th rowspan="1" colspan="1">Mean coverage/site</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><i>Asibi</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">12</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">317,220</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">6,006</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs1</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">12</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">286,332</td> <td rowspan="1" colspan="1">27</td> <td rowspan="1" colspan="1">5,521</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs2</i></td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">410,433</td> <td rowspan="1" colspan="1">40</td> <td rowspan="1" colspan="1">7,625</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs3</i></td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">167,768</td> <td rowspan="1" colspan="1">16</td> <td rowspan="1" colspan="1">3,208</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rUA</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">16</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">113,558</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">2,075</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rCG</i></td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">419,297</td> <td rowspan="1" colspan="1">40</td> <td rowspan="1" colspan="1">7,900</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rSS</i></td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">55,624</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">1,026</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rN</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">223,184</td> <td rowspan="1" colspan="1">21</td> <td rowspan="1" colspan="1">4,274</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTCG</i></td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">249,701</td> <td rowspan="1" colspan="1">24</td> <td rowspan="1" colspan="1">4,580</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs4</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">15</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1">176,219</td> <td rowspan="1" colspan="1">17</td> <td rowspan="1" colspan="1">3,227</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs1h</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">13</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">619,634</td> <td rowspan="1" colspan="1">60</td> <td rowspan="1" colspan="1">10,949</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs2h</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">22</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">144,211</td> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1">2,442</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs3h</i></td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">13</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">528,403</td> <td rowspan="1" colspan="1">51</td> <td rowspan="1" colspan="1">11,304</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rUAh</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">17</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">159,007</td> <td rowspan="1" colspan="1">15</td> <td rowspan="1" colspan="1">2,724</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rCGh</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">297,217</td> <td rowspan="1" colspan="1">29</td> <td rowspan="1" colspan="1">6,220</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rSSh</i></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">19</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">256,125</td> <td rowspan="1" colspan="1">25</td> <td rowspan="1" colspan="1">4,258</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rNh</i></td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">514,717</td> <td rowspan="1" colspan="1">50</td> <td rowspan="1" colspan="1">10,693</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTCGh</i></td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">163,170</td> <td rowspan="1" colspan="1">15</td> <td rowspan="1" colspan="1">2,413</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Note: Counts of consensus mutations and variant subpopulations are presented as raw numbers. Consensus reversion mutations were defined with reference to YFV strain <sup><i>Asibi</i></sup>. The coverage depth has been calculated given that the total length of YFV CDS is 10,236 nt. No variant was taken into account with a coverage lower than 1000 reads. No consensus mutation was taken into account with a coverage lower than 50 reads.</p> </div> </div> </div> <p /> <p>During virus production, sequence variability could arise as an artefact of the production method itself (PCR amplification of DNA fragments) and as the result of the adaptation of re-encoded strains to culture conditions.</p> <p>No consensus mutation associated with a major variant was identified in the genome of the parent strain YFV <i>Asibi</i>, which only included twelve minor variants. In re-encoded genome sequences, low numbers of consensus mutations were observed, ranging from zero to six, with a mean of two consensus changes/virus and 39 per cent of non-synonymous (NS) mutations on an average. Consensus mutations were equally distributed between re-encoded and non-re-encoded regions and, nearly no convergence was observed between viral sequences. Only two viral sequences (YFV rTs2 and rCG) had consensus mutation reversions. One was common to both strains and corresponded to a siTs mutation (C to T), at CDS position 5,430. The two others were also siTs mutation reversions (C to T and A to G, respectively), located at CDS positions 5,442 and 5,667 of YFV rTs2 and rCG, respectively. The absence of consensus NS change in the sequence of the original <i>Asibi</i> strain and the very limited convergence between viral sequences are in accordance with the low number (3) of <i>in cellulo</i> passages. This strongly suggests that the consensus mutations observed in viruses YFV rTs3, rCG, rSS, and rTCG did not result from selective pressure due to culture conditions but may be artefacts of the production method.</p> <p>The total numbers of variants, per virus, ranged from one to sixteen, with a mean of nine variants per virus. Interestingly, variants were most frequently observed outside the re-encoded regions (75 per cent of variants outside the re-encoded regions), similar to what was observed for the parental sequence, (92 per cent of variants outside the re-encoded region). The low number of reversions and the limited sequence variability at re-encoded sites accord with previous work showing that re-encoded CHIKV strains evolved mostly through compensatory mutations rather than mutation reversions (<a href="#vey021-B60">Nougairede et al. 2013</a>).</p> </div> <div class="2.2.2invitroreplicativefitness" title="sec"> <div class="title" tagx="title" title="title">
2.2.2 <i>In vitro</i> replicative fitness</div> <p>The infectious titre was estimated using a TCID<sub>50</sub> assay in BHK21 cells for both <i>Asibi</i> and <i>Asibi</i>-derived re-encoded strains using infectious cell supernatant medium (viral stock) (results detailed in <a href="#vey021-T2">Table 2</a>). The maximal infectious titre was observed for the wild-type strain <i>Asibi</i> (10<sup>8</sup> TCID<sub>50</sub>/mL). A 1- to 2-log decrease was observed for most re-encoded strains, with infectious titres ranging between 2×10<sup>6</sup> and 4×10<sup>7</sup> TCID<sub>50</sub>/mL. However, the least re-encoded virus, YFV rTs1 (101 mutations), was unexpectedly endowed with a remarkably low infectious titre (4×10<sup>5</sup> TCID<sub>50</sub>/mL) while the most re-encoded virus, YFV rTs4 (741 mutations), exhibited the lowest infectious titre (6×10<sup>3</sup> TCID<sub>50</sub>/mL).</p> <p>For each of the re-encoded viruses, the replicative fitness was compared with that of the wild-type strain using competition assays. Specific virus detection was achieved for one in two passages using next generation sequencing (NGS) methods by amplifying a 256 nucleotide region and counting virus-specific patterns within the read population. For all re-encoded viruses except rTs4 (≤353 mutations), replicative fitness <i>in vitro</i> was comparable to that of the <i>Asibi</i> strain: in all competitions, both competing viruses could still be detected at the tenth passage (see <a href="#vey021-F2">Fig. 2</a>). By contrast, the highly re-encoded rTs4 virus (741 mutations) showed a clear reduction in replicative fitness and could not be detected after the fifth passage. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Competition assays: Wild-type and re-encoded-specific patterns counts at tenth competition passage. For each WT/re-encoded virus ratio and each re-encoded virus, pattern counts were calculated as a mean for the three replicates. For each replicate, the pattern count is the mean count for the three virus-specific patterns. Each specific pattern count is the sum of the sense and reverse patterns counts within a given read population (obtained through NGS sequencing).</p>   </div> </div> </div> <div class="2.3invivobehaviourofyfvre-encodedstrains" title="sec"> <div class="title" tagx="title" title="title">
2.3 <i>In v</i><i>ivo</i> behaviour of YFV re-encoded strains</div> <p>The infectious titres of viral stocks for <i>Ap7M</i> and <i>Ap7M</i>-derived re-encoded strains were established in BHK21 cells, using a TCID<sub>50</sub> assay (results detailed in <a href="#vey021-T2">Table 2</a>). For the majority of re-encoded viruses, no notable difference was observed with the parent strain <i>Ap7M</i> (9×10<sup>8</sup> TCID50/mL), with infectious titres ranging from 4×10<sup>8</sup> to 6×10<sup>8</sup> TCID50/mL. However, viruses YFV rNh and rTCGh showed a one- to two-log reduction in viral titre (6×10<sup>6</sup> and 6×10<sup>7</sup> TCID50/mL, respectively).</p> <div class="2.3.1sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3.1 Sequence analysis</div> <p>NGS was performed on the viral stocks for all <i>Ap7M</i>-derived re-encoded viruses as described in the previous section (results shown in <a href="#vey021-T4">Tables 4</a> and <a href="#vey021-T5">5</a>). <div class="table-wrap_UNKNOWN" id="vey021-T5" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> </div></p><p>Sequence and intra-population diversity in viral culture supernatant media and hamster liver samples for Ap7M and Ap7M-derived re-encoded strains.</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="left" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" valign="top" align="center" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Strain name</th> <th rowspan="2" align="left" colspan="1">Position (nt)</th> <th rowspan="2" align="left" colspan="1">Protein</th> <th rowspan="2" align="left" colspan="1">Consensus mutation</th> <th rowspan="2" align="left" colspan="1">Expected sequence (nt)</th> <th rowspan="2" align="left" colspan="1">Observed sequence (consensus (nt))</th> <th rowspan="2" align="left" colspan="1">AA change</th> <th colspan="2" align="left" rowspan="1">Observed sequence (Inoculum (nt)) <hr /> </th> <th colspan="3" align="left" rowspan="1"> Observed sequence (Hamster (nt)) <hr /> </th> </tr> <tr> <th rowspan="1" colspan="1">First variant</th> <th rowspan="1" colspan="1">Second variant</th> <th rowspan="1" colspan="1">Retrieved sequence</th> <th align="left" rowspan="1" colspan="1">First variant</th> <th align="left" rowspan="1" colspan="1">Second variant</th> </tr> </thead> <tbody> <tr> <td rowspan="3" colspan="1"><i>rTs1h</i></td> <td rowspan="1" colspan="1">836</td> <td rowspan="1" colspan="1">prM</td> <td rowspan="1" colspan="1">A158V</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">A → V</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="3" colspan="1">1–4,806, 5,430–7,566</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">4,574</td> <td rowspan="1" colspan="1">NS3</td> <td rowspan="1" colspan="1">G41A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">G → A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">g</td> </tr> <tr> <td rowspan="1" colspan="1">5,442</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rTs2h</i></td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">1–3,858, 4,653–5,771, 6,083–10,236</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="2" colspan="1"><i>rTs3h</i></td> <td rowspan="1" colspan="1">1,649</td> <td rowspan="1" colspan="1">Env</td> <td rowspan="1" colspan="1">T265I</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">T → I</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">t</td> <td rowspan="2" colspan="1">1–10,236</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">4,604</td> <td rowspan="1" colspan="1">NS3</td> <td rowspan="1" colspan="1">R51M</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">R → M</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">t</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">g</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rUAh</i></td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">1–6,210, 6,492–7,565</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="3" colspan="1"><i>rCGh</i></td> <td rowspan="1" colspan="1">283</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">–</td> <td rowspan="3" colspan="1">1–4,652, 5,433–7,564</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">t</td> </tr> <tr> <td rowspan="1" colspan="1">1,051</td> <td rowspan="1" colspan="1">Env</td> <td rowspan="1" colspan="1">T66A</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">T → A</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">a</td> </tr> <tr> <td rowspan="1" colspan="1">2,541</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1"><i>rSSh</i></td> <td rowspan="1" colspan="1">5,439</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">14–10,228</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="6" colspan="1"><i>rNh</i></td> <td rowspan="1" colspan="1">818</td> <td rowspan="1" colspan="1">prM</td> <td rowspan="1" colspan="1">I152T</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">I → T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="6" colspan="1">7–5,236, 5,422–10,229</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">1,686</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">2,052</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">7,490</td> <td rowspan="1" colspan="1">NS4B</td> <td rowspan="1" colspan="1">N241S</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">N → S</td> <td rowspan="1" colspan="1">g</td> <td rowspan="1" colspan="1">a</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">8,253</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">g</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="1" colspan="1">9,783</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">a</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">–</td> </tr> <tr> <td rowspan="2" colspan="1"><i>rTCGh</i></td> <td rowspan="1" colspan="1">2,222</td> <td rowspan="1" colspan="1">Env</td> <td rowspan="1" colspan="1">I456T</td> <td rowspan="1" colspan="1">T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">I → T</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">t</td> <td rowspan="2" colspan="1">11–10,223</td> <td rowspan="1" colspan="1">C</td> <td rowspan="1" colspan="1">t</td> </tr> <tr> <td rowspan="1" colspan="1">7,107</td> <td rowspan="1" colspan="1">NS4B</td> <td rowspan="1" colspan="1">I113M</td> <td rowspan="1" colspan="1">A</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">I → M</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">a</td> <td rowspan="1" colspan="1">G</td> <td rowspan="1" colspan="1">a</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Note: For each virus, the consensus changes found either in the complete coding sequence from the viral stock or in partial sequences from infected hamster liver homogenates are detailed above. The expected sequence refers to that of plasmids that served for subgenomic fragment amplification while producing viruses using the ISA method. For 1<sup>st</sup> and 2<sup>nd</sup> variant descriptions, capital letters correspond to variant proportions above 75%, lowercase letters to variant proportions between 6 and 75% and dashes to variant proportions below 5%. Amino Acid (AA) positions are given with reference to the beginning of mature protein sequence (i.e., AA position 107 in prM, written prM 107, corresponds to AA position 228 within the precursor polyprotein). A detailed map of protein positions within the YFV polyprotein sequence is provided in the <a href="#sup1">supplementary Table S11</a>.</p> </div> </div>  <p /> <p>As observed for <i>Asibi</i>-re-encoded viruses, the number of consensus mutations associated with major variants were low, ranging from zero to five, with a mean number of one consensus change/virus and 59 per cent of non-synonymous mutations on an average. Consensus mutations were mainly distributed outside re-encoded regions (90 per cent), with no reversion. As observed for <i>Asibi</i>, the parent <i>Ap7M</i> strain showed no NS consensus sequence changes associated with major variants. Moreover, the re-encoded viruses which were given a limited number of <i>in cellulo</i> passages did not exhibit any evidence of convergent evolution. Based on this evidence, the consensus mutations observed in YFV rTs3h, rNh, and rTCGh are likely to be artefacts of the production method rather than the result of adaptation of re-encoded viruses to culture conditions. Notably, for YFV rTs3h, the consensus NS change observed in the inoculum was not essential for virus survival <i>in vivo</i> as it was not maintained following infection in hamsters.</p> <p>The total number of variants per virus varied among strains, ranging from four to twenty-two, with a mean number of thirteen variants per virus. Similar to what was observed for <i>Asibi</i>-derived re-encoded strains, the diversity at re-encoded sites was low, with only a minority of variants being detected within the re-encoded regions (19 per cent).</p> </div> <div class="2.3.2invivophenotype:comparativestudyofpathogenicity" title="sec"> <div class="title" tagx="title" title="title">
2.3.2 <i>In vivo</i> phenotype: comparative study of pathogenicity</div> <p>Groups of 12 three-week-old female hamsters were inoculated intra-peritoneally with 5×10<sup>5</sup> TCID<sub>50</sub> of virus (either <i>Ap7M</i> or <i>Ap7M</i>-re-encoded strains) and a control group of two uninfected hamsters was maintained for weight monitoring. Clinical follow-up included (1) clinical manifestations of the disease (brittle fur, dehydration, prostration, and lethargy), (2) body weight (weight evolution was expressed as a normalised percentage of the initial weight (%IW)), and (3) death. Three and four hamsters were euthanised at days three and six post-infection (dpi), respectively, to conduct virology investigations from liver samples, while the other five in each group were retained for evaluating mortality rate (endpoint in case of survival: 16 dpi). Due to an issue in group dispatching, in the case of rTs2h strain, three hamsters were euthanised at 6 dpi while six hamsters were used for mortality rate evaluation. Virology follow-up was achieved by performing qRT-PCR and next generation sequencing on RNA extracted from the liver homogenates. RNA samples were subsequently pooled to obtain one sequence for each virus, reflecting genetic diversity among the five hamsters kept until death/euthanasia. For all viruses, the <i>in vivo</i> phenotype and the sequencing results are detailed in <a href="#vey021-F3">Fig. 3</a> and <a href="#vey021-T4">Tables 4</a> and <a href="#vey021-T5">5</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p><i>In vivo</i> phenotypes of Ap7M and Ap7M-derived re-encoded viruses. Normalised percentages of initial weight (%IW) (A), viral loads at 3/6 dpi (B), as well as mortality rates (C), are illustrated for all viruses. %IW were evaluated on groups of twelve and nine hamsters at 3 and 6 dpi, respectively. Viral loads were evaluated on groups of three and four hamsters at 3 and 6 dpi, respectively. Of note, for YFV rTs2h, only three hamsters were used for viral load determination at 6 dpi. Mortality rates were evaluated on groups of five hamsters for all viruses, apart from rTs2h, for which six hamsters were used. Computations of %IW and viral loads are detailed in the corresponding paragraphs within the ‘Section 4’. Significant %IW and viral load differences observed between <i>Ap7M</i> and the other viruses at 6 dpi (Wilcoxon rank-sum test P value &amp;lt;0.05) are indicated by a star (*). The ranges corresponding to viral load values recorded at 3 and 6 dpi during <i>Ap7M</i> infection are highlighted by light (3 dpi) and dark (6 dpi) blue rectangles.</p>   </div> <p>Nearly all hamsters inoculated with <i>Ap7M</i> strain developed outward signs of illness such as ruffled fur, prostration, dehydration, and lethargy. One hamster did not show any sign of illness, both its liver and blood were tested negative for YFV genomes and it was excluded from analysis. A high mortality rate (80 per cent) was observed, similar to previous observations (<a href="#vey021-B36">Klitting et al. 2018</a>). Clinical signs of the disease appeared as early as 4 dpi and all animals died within 2/3 days after onset of the symptoms. Weight loss was not observed at 3 dpi (mean of %IW: 97 per cent), but between 3 and 6 dpi, significant weight losses were recorded (mean of %IW: 71 per cent, Wilcoxon rank sum test, P value = 0.00032). All livers were found to be YFV-positive by qRT-PCR, with viral RNA loads ranging between 6×10<sup>7</sup> and 6×10<sup>9</sup> RNA copies per gram of liver and no significant difference between the viral yields at 3 and 6 dpi (Wilcoxon rank sum test, P values &amp;gt;0.05). No consensus change was detected in the viral sequence after propagation in hamsters.</p> <p>A reduction in mortality rate was observed in all groups infected with re-encoded viruses. For most (YFV rTs1h, rUAh, rCGh, rSSh, rNh, and rTCGh), the reduction was significant (Logrank test, P values = 0.03960 in all cases), as no hamsters died. However, for YFV rTs2h and rTs3h, mortality was reduced but still observed (17 and 20 per cent, respectively; Logrank test, P values = 0.05820 and 0.07186).</p> <p>Regardless of the observed mortality rate, clinical signs of illness were observed with varying degrees of severity among groups. No association was observed between mortality and weight evolution at 3 or 6 dpi, with non-lethal viruses inducing limited (rNh, rTs1h) as well as severe weight loss (rCGh, rSSh) in infected hamsters. At 3 dpi, weight evolution was heterogeneous but not significantly different from what was observed in <i>Ap7M</i>-infected group (Wilcoxon rank sum test, P values &amp;gt; 0.05). An increase up to 7 per cent was observed in YFV rTs1h, rTs2h, and rTs3h-infected groups, a slight decrease (between 3 and 4 per cent) in groups infected with YFV rUAh, rNh, and rTCGh, and a greater decrease up to 11 per cent, in YFV rCGh and rSSh-infected groups. At 6 dpi, an important weight loss was observed in most groups (up to 38 per cent) except rTs1h and rNh-infected groups, in which significantly higher %IW values were recorded (8 and 4 per cent, respectively, (Wilcoxon rank sum test, P values = 0.00088 and 0.00062, respectively)).</p> <p>An increase of viral yields at 3 and 6 dpi was not associated with mortality rates. At 3 dpi, viral loads in the liver were close to values observed with YF <i>Ap7M</i>, with values ranging from 7×10<sup>8</sup> to 1×10<sup>10</sup> RNA copies per gram of liver. At 6 dpi, viral loads remained close to those observed with <i>Ap7M</i>. However, in some cases (YFV rTs1h and rNh), significantly lower viral loads were observed (means of 5 and 8×10<sup>7</sup>, respectively. Wilcoxon rank sum test, P value= 0.03038 in both cases).</p> <p>Several possibilities can explain the apparent absence of association between mortality and the evolution of viral loads. First, significant reduction in viral loads were always associated with a complete loss of lethality (for viruses rNh and rTs1h), hence they were, to a reasonable extent, associated to pathogenesis. Furthermore, in terms of replicative fitness and thus, of viral loads, the threshold that corresponds with the loss of lethality may be subtle and undetectable under the experimental conditions used here. In addition, the viral loads that are reported here are RNA copy numbers and may not reflect the actual amount of infectious particles. Furthermore, they have been evaluated from liver samples and may be less sharply correlated with disease evolution than viral loads in the blood. In addition, variation between individual animals will contribute to the final outcome of the data. Finally, during YFV infection, it is likely that the pathogenesis involves an immunological component (<a href="#vey021-B72">Ter Meulen et al. 2004</a>; <a href="#vey021-B5">Barrett 2012</a>; <a href="#vey021-B62">Quaresma et al. 2013</a>). Hence, levels of pro- and anti-inflammatory cytokines may be more closely associated with mortality than viral loads and weight evolution. Such criteria were not included in the experimental design.</p> </div> <div class="2.3.3evaluationofthephenotypicimpactofre-encoding" title="sec"> <div class="title" tagx="title" title="title">
2.3.3 Evaluation of the phenotypic impact of re-encoding</div> <p>Some conclusions can be drawn by analysing the <i>in vivo</i> phenotypes of re-encoded viruses in light of the sequencing data obtained from infected hamsters liver samples (see details in <a href="#vey021-T5">Table 5</a>). For example, the sequences obtained from the livers of hamsters infected with the control strain <i>Ap7M</i> did not include consensus mutations. This indicates that the sequence of the parent virus was stable under the experimental conditions and that adaptative mutations were not required for efficient replication <i>in vivo</i>.</p> </div> <div class="2.3.4impactof‘non-specific’re-encodingstrategies" title="sec"> <div class="title" tagx="title" title="title">
2.3.4 Impact of ‘non-specific’ re-encoding strategies</div> <p>In general, low detrimental effects resulted from the introduction of simple transitions (siTs) (i.e., with no modification of predicted secondary structures) on the <i>in vivo</i> phenotype of YFV. Strain rTs2h (247 siTs) was associated with mild attenuation <i>in vivo</i>, with a decreased mortality rate and a reduction in hamster weight loss at 6 dpi but no significant change in the production of viral RNA (Wilcoxon rank sum, P values &amp;gt; 0.05). As the viral sequence remained unchanged during the course of the experiments, the <i>in vivo</i> phenotype can be interpreted as being the result of genome re-encoding. rTs3h (353 siTs) and rTs1h (101 siTs) viruses, respectively, showed mild and strong attenuation <i>in vivo</i>. For both, consensus NS mutations associated with major variants arose during the <i>in vivo</i> infections possibly resulting from adaptation of the re-encoded viruses to the <i>in vivo</i> conditions during the infection. However, they did not correspond with the recovery of an <i>in vivo</i> phenotype similar to that of the parent strain <i>Ap7M</i>. Surprisingly, the least re-encoded strain, rTs1h (101 siTs) was more attenuated <i>in vivo</i> than the rTs2h strain, that included all rTs1h mutations plus 146 additional siTs. We observed a mild detrimental impact following the introduction of transitions at sites involved in predicted RNA secondary structures. The sequence of strain rSSh involved siTs as well as transitions located on predicted secondary RNA structures and remained stable throughout the <i>in vivo</i> experiments. The variant showed a partially attenuated phenotype <i>in vivo</i> (complete loss of mortality, no alleviation in weight loss or viral loads) that can thus be attributed to this second re-encoding strategy. Overall, these results illustrate the relative efficiency of the use of ‘non-specific’ transitions in providing a stable re-encoded infectious genome exhibiting <i>in vivo</i> attenuation.</p> </div> <div class="2.3.5impactof‘specific’re-encodingstrategies" title="sec"> <div class="title" tagx="title" title="title">
2.3.5 Impact of ‘specific’ re-encoding strategies</div> <p>Variants with increased CpG and UpA dinucleotides exhibited attenuated phenotypes <i>in vivo</i>, with no mortality, alleviated weight loss and no reduction in viral loads. For variant rUAh (fifty-nine UpA), the sequence remained stable throughout the <i>in vivo</i> experiments. Hence, the attenuation resulted from the original sequence re-encoding. The originally modified sequence of variant rCGh (fifty-nine CpG) remained constant and consensus NS changes were associated with major variants arising during the <i>in vivo</i> studies. Such consensus mutations are likely to be adaptive but did not result in complete recovery of virulence during the <i>in vivo</i> studies. Altogether, these results are consistent with the attenuating effect of the introduction of CpG/UpA dinucleotides into viral genomes as reported on previous occasions (<a href="#vey021-B3">Atkinson et al. 2014</a>; <a href="#vey021-B75">Tulloch et al. 2014</a>; <a href="#vey021-B18">Gaunt et al. 2016</a>; <a href="#vey021-B78">Witteveldt, Martin-Gans, and Simmonds 2016</a>).</p> <p>On the other hand, the sequence of the stock variant rTCGh (118 TCG) differed from that of the original construct. To conclude, with respect to the compensatory role of the consensus NS mutations that arose during strain production, further experiments would be required. Nevertheless, the attenuated phenotype (loss of lethality, alleviation of weight loss) of the re-encoded viruses following infection in hamsters suggests a detrimental effect of increasing TCG trinucleotides on YFV virulence <i>in vivo</i> that was not completely outweighed by the presence of additional consensus NS changes. Hence, the introduction of TCGs may impact on the <i>in vivo</i> phenotype of YFV following infection in vertebrate hosts. Of note, a recent study by <a href="#vey021-B55">Moratorio et al. (2017</a>) highlighted the adverse effect of the introduction of codons that are likely to yield stop codons (referred to as ‘1-to-Stop codons’) on the fitness of Influenza A and Coxsackie B3 viruses <i>in vitro</i> and <i>in vivo</i>. Namely, these 1-to-Stop codons are UUA, UUG, UCA, and UCG. Only twenty-eight additional UCG were identified within the sequence of YFV rTCG (which was the only re-encoded virus for which there was a change in the number of 1-to-Stop codons). This is four- to eight-fold lower compared with what was introduced into the genomes of Coxsackie B3 (117 codons) and Influenza A (205 codons) viruses. Nevertheless, further experimentation would be useful to determine whether or not there is an increase in the generation of Stop codons during viral replication of rTCG/rTCGh viruses <i>in vitro</i> and/or <i>in vivo</i>.</p> <p>Finally, for strain rNh, consensus NS mutations were identified in the sequences from both the inoculum and hamster liver samples. It is therefore complicated to determine which of the consensus changes were production artefacts or resulted from adaptation of the virus to <i>in vitro</i> or <i>in vivo</i> conditions. Nevertheless, the virus exhibited a strongly attenuated phenotype <i>in vivo</i> (no mortality, alleviated weight loss and viral loads) indicating that, regardless of the emergence of the—potentially compensatory—consensus NS mutations, the combination of several re-encoding strategies led to an important degree of attenuation considering the low number of mutations used for re-encoding (i.e., 233).</p> </div> <div class="2.3.6adaptationofre-encodedvirusesthroughconsensusnsmutations" title="sec"> <div class="title" tagx="title" title="title">
2.3.6 Adaptation of re-encoded viruses through consensus NS mutations</div> <p>Altogether, these results indicate that consensus mutations can arise and stably establish in re-encoded viruses in response to <i>in vivo</i> conditions. This was observed for most of the re-encoded viruses, notably some with as few as 101 re-encoded sites (rTs1h) or as many as 353 re-encoded sites (rTs3h). In most cases, the consensus mutations did not lead to a full recovery of <i>in vivo</i> viral replicative fitness. Observable changes in the biological properties of the viruses (e.g., change in tropism, as reported for YF strain 17D), were not reported in any of the viruses that exhibited consensus NS mutations.</p> </div> <div class="2.3.7immunisationpotentialofyf-re-encodedstrains:serologicalinvestigations" title="sec"> <div class="title" tagx="title" title="title">
2.3.7 Immunisation potential of YF-re-encoded strains: serological investigations</div> <p>For each of the re-encoded viruses that showed complete loss of mortality <i>in vivo</i>, sera were recovered from infected hamsters at 16 dpi during the comparative study of pathogenicity (see above). Sera from both uninfected hamsters and vaccinated humans were included as negative and positive controls, respectively. Neutralisation tests were performed using serial five-fold dilutions of all sera, the YF <i>Ap7M</i> challenge virus and BHK21 cells. Neutralising antibodies were detected in both hamsters infected with YF-re-encoded viruses (NT<sub>50</sub> ranging between 2×10<sup>−3</sup> and 1×10<sup>−5</sup>) and in a control group of 17D-vaccinated humans (NT<sub>50</sub> between 4 and 5×10<sup>−4</sup>) while the negative control group did not show any neutralisation activity against <i>Ap7M</i> (see <a href="#vey021-F4">Fig. 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Results of YFV serology at 16-day post-infection. Neutralisation assays were performed on sera obtained from hamsters infected with re-encoded viruses that showed complete loss of mortality <i>in vivo</i>. For each virus, serum samples were retrieved at 16 dpi from two hamsters. Sera from both uninfected hamsters (<i>n</i>=2) and vaccinated humans (vaccinees, <i>n</i>=4) were used as negative and positive controls. Results are expressed as 50 per cent neutralisation titre (NT<sub>50</sub>) and were evaluated using Graphpad Prism Software (v7.00 for Windows, GraphPad Software, La Jolla, CA, <a href="http://www.graphpad.com">www.graphpad.com</a>). Thick black lines indicate median values and minimum/maximum values are indicated by dashes.</p>   </div> </div> </div> <div class="2.4immunisationpotentialofyf-re-encodedstrains:challengeexperiments" title="sec"> <div class="title" tagx="title" title="title">
2.4 Immunisation potential of YF-re-encoded strains: challenge experiments</div> <p>Two groups of 11 three-week-old female hamsters were inoculated intraperitoneally with 5×10<sup>5</sup> TCID<sub>50</sub> of virus (either YFV rCGh or rNh). A negative control group of nine hamsters was not immunised before the challenge and a control group of two uninfected hamsters was included for monitoring hamster weight.</p> <p>Unexpectedly, one hamster from the YFV rCGh infected group died at 6 dpi, although it did not show any symptom nor weight loss. Its liver was tested positive for YFV. Twenty-four days after inoculation, two and three hamsters from rCGh and rNh groups, respectively, and a hamster from the non-infected group were euthanised and both serum and liver samples were recovered for neutralisation assay and viral load determination. For all three groups, the eight remaining hamsters were inoculated intraperitoneally with 5×10<sup>5</sup> TCID<sub>50</sub> of virus (<i>Ap7M</i>). Two hamsters were euthanised at three- and six-day post-challenge (dpc) to conduct virology investigations from liver samples while four were kept for evaluating mortality rate (endpoint in case of survival: twelve dpc). The protection was evaluated by determining for each group (1) viral loads in liver at three and six dpc, (2) weight evolution at six and ten dpc, (3) survival. With regard to the number of samples tested for virology and immunology investigations, no statistical test was achieved in this section. Detailed results for this experiment are available in <a href="#vey021-F5">Fig. 5</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Challenge assay. Normalised percent of initial weight (%IW) at day of challenge and 6/10 dpc (A) as well as viral loads at day of challenge and 3/6 dpc (B) are given for all groups (i.e., immunised and control). Viral loads were evaluated on groups of two hamsters at 3/6 dpc. %IW were evaluated on groups of eight, six, and four hamsters at 3, 6, and 10 dpc, respectively. Significant %IW differences observed between non-immunised (Non im.) and the other groups at 6/10 dpc (Wilcoxon rank-sum test P value &amp;lt;0.05) are indicated by a star (*). Viral loads and %IW computations are detailed in the corresponding paragraphs within the ‘Section 4’.</p>   </div> <p>In agreement with previous observations, no mortality was observed post-challenge in either immunised or control groups, indicating that mortality rate in hamsters decreases importantly in adults (greater than six-week old individuals) (<a href="#vey021-B502">Tesh et al. 2001</a>). However, there was a great difference in the evolution of viral loads, with no increase in viral yields at three and six dpc in rCGh and rNh immunised animals while high viral loads were detected in the control animals, around ∼10<sup>9</sup> RNA copies per gram of liver. Evidence of protection was also provided by the fact that there was no weight loss in immunised groups at six and ten dpc, whereas a significant weight loss up to 18 per cent at ten dpc (Wilcoxon rank sum test P value = 0.00239) was recorded in the control group. Altogether, these results indicate efficient immunisation of hamsters inoculated with YFV re-encoded strains rCGh or rNh.</p> </div> </div> <div class="3.discussion" title="sec"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p>Genome re-encoding through synonymous codon replacement is a recognised procedure for attenuating RNA viruses including poliovirus, influenza virus, respiratory syncytial virus, chikungunya virus, tick-borne encephalitis virus and dengue virus (<a href="#vey021-B10">Burns et al. 2006</a>; <a href="#vey021-B57">Mueller et al. 2006</a>; <a href="#vey021-B60">Nougairede et al. 2013</a>; <a href="#vey021-B15">De Fabritus et al. 2015</a>; <a href="#vey021-B45">Martinez et al. 2016</a>). Additional insights into the mechanisms contributing to attenuation of the virus phenotype following synonymous codon replacement should prove valuable for the custom design of, viable, safe, cost-effective vaccine candidates. The initial purpose of this work was to identify the types of synonymous mutations that would impose the least detrimental effect on viral replicative fitness. Previous codon re-encoding studies in our laboratory were achieved for two other arboviruses, CHIKV and TBEV (<a href="#vey021-B60">Nougairede et al. 2013</a>; <a href="#vey021-B15">De Fabritus et al. 2015</a>). In the light of this experience, we planned an exploratory study of the effects of re-encoding on both the <i>in vitro</i> and <i>in vivo</i> phenotypes of yellow fever virus (YFV). Our choice was motivated by the availability of a suitable reverse genetics system for re-encoded strains of YFV production, viz., the ISA method (<a href="#vey021-B4">Aubry et al. 2014</a>) and a robust YF hamster model (<a href="#vey021-B47">McArthur et al. 2003</a>). Hence, we could test first the <i>in vitro</i>, and then the <i>in vivo</i> replicative fitness of our strains before using some of the live-attenuated candidates for hamster immunisation studies.</p> <p>Here, we have described the direct impact of synonymous substitutions on YFV replicative phenotype both <i>in vitro</i> and <i>in vivo</i>. As previously described with TBEV (<a href="#vey021-B15">De Fabritus et al. 2015</a>), the level of re-encoding necessary to observe viral attenuation <i>in vivo</i> was lower than that required to obtain an observable decrease in replicative fitness <i>in vitro</i>. As further suggested by supplementary, basic, growth curve experiments (data not shown), all re-encoded strains with only one re-encoded fragment (101–350 mutations) maintained a replicative phenotype close to that of the parental <i>Asibi</i> strain <i>in vitro</i>. Only YFV rTs4, that included an additional re-encoded fragment (741 mutations in total), showed a severe reduction in replicative fitness <i>in vitro</i>. Previous works on Poliovirus, Echovirus, RSV, Influenza, HIV, CHIKV, DENV, and Marek’s disease virus (<a href="#vey021-B10">Burns et al. 2006</a>; <a href="#vey021-B58">Mueller et al. 2010</a>; <a href="#vey021-B46">Martrus et al. 2013</a>; <a href="#vey021-B60">Nougairede et al. 2013</a>; <a href="#vey021-B3">Atkinson et al. 2014</a>; <a href="#vey021-B41">Le Nouen et al. 2014</a>; <a href="#vey021-B68">Shen et al. 2015</a>; <a href="#vey021-B14">Conrad et al. 2018</a>) brought evidence that the effect of synonymous mutations may vary according to the location within the viral sequence. Hence, a control, <i>Asibi</i>-derived virus including only the additional re-encoded fragment (388 mutations in the NS5) was produced. Contrarily to the viruses re-encoded in the NS2-3-4 region, it showed a decrease in replicative fitness during competition experiments (outcompeted by the wild-type virus at passage no. 10). As three consensus non-synonymous mutations (associated to major variants) arose in the viral sequence during the production of this virus, it is not possible to associate the <i>in vitro</i> phenotype to the re-encoding in the NS5 protein rather than to the additional, consensus, non-synonymous mutations. However, this suggests that in the case of YFV, re-encoding may have a greater effect in the polymerase gene than in other non-structural proteins. In contrast with the <i>in vitro</i> results, we observed a broad range of levels of attenuation among all the strains that were tested in hamsters (101–350 mutations). While the least attenuated viruses showed no observable loss of replicative fitness and retained a degree of lethality, the most attenuated viruses displayed a complete loss of lethality, together with a ten-fold reduction in viral loads in the liver and significant alleviation of weight loss. We did not produce nor test a hamster-adapted version of the rTs4 strain because such a heavily re-encoded virus would have been unlikely to achieve infection <i>in vivo</i>.</p> <p>In contrast with previously published re-encoding studies, we performed deep sequencing of infected hamster liver homogenates to correlate precisely each viral sequence(s) with the observed phenotype. Our results indicate that, detailed genetic information and in particular, that retrieved from <i>in vivo</i> experiments, is critical to determine the significance of synonymous substitutions within the attenuation process. When monitoring the evolution of YFV re-encoded strains after production (i.e., transfection followed by three passages <i>in vitro</i>) or after one passage in hamsters, we observed the spontaneous emergence of both synonymous and non-synonymous substitutions. These consensus changes may be artefacts of the production methods employed (PCR amplification of DNA fragments, stochastic evolution events) or they may have resulted from the adaptation of re-encoded viruses to the differing <i>in vitro</i> and <i>in vivo</i> conditions of replication. In some cases, we observed the emergence of consensus non-synonymous mutations present in major variants of the viral stock used as the inoculum for <i>in vivo</i> experiments. For these viruses, the analysis of the results from <i>in vivo</i> experiments and attempts to correlate them with sequencing data was limited due to consensus NS mutations which impeded the identification of the specific impact of silent mutations on the viral phenotype <i>in vivo</i>.</p> <p>Several conclusions can be drawn regarding the <i>in vivo</i> phenotypic impact of the different strategies used to produce re-encoded YFV strains.</p> <p>First, the introduction of simple transitions (i.e., with no modification of putative secondary structures in the genomic RNA) on the replicative fitness of YFV had only minor impact on viral phenotype. They did not cause larger modification of GC%, ENC, Codon Adaptation index (to the homo sapiens codon usage) or in the proportion of codons that are more likely to yield stop codons (‘1-to-Stop codons’, as defined by <a href="#vey021-B55">Moratorio et al. 2017</a>) than what can be observed among clinical/field isolates, indicating that additional mechanisms are probably involved in the attenuation process. Similarly, the introduction of transitions at sites involved in putative genomic secondary RNA structures had a relatively mild detrimental impact on replicative fitness. Second, contrary to secondary RNA structures that have been identified in the untranslated regions of YFV (<a href="#vey021-B59">Ng et al. 2017</a>) and other flaviviruses (<a href="#vey021-B25">Gritsun and Gould 2006</a>, <a href="#vey021-B26">2007a</a>,<a href="#vey021-B27">b</a>), the putative structures within the virus open reading frame appear unlikely to be important determinants of replicative fitness. Rather, role(s) in finely tuning the efficiency of translation (mRNA or co-translational folding) or in interactions with viral, and non-viral proteins have been proposed (<a href="#vey021-B67">Shabalina, Ogurtsov, and Spiridonov 2006</a>; <a href="#vey021-B39">Kudla et al. 2009</a>; <a href="#vey021-B61">Pechmann and Frydman 2013</a>).</p> <p>Previous studies have shown that increase in CpG/UpA dinucleotides is an efficient attenuation mechanism (<a href="#vey021-B3">Atkinson et al. 2014</a>; <a href="#vey021-B75">Tulloch et al. 2014</a>; <a href="#vey021-B18">Gaunt et al. 2016</a>; <a href="#vey021-B78">Witteveldt, Martin-Gans, and Simmonds 2016</a>) and this was supported by the evidence of a correlation between increased CpG/UpA and attenuation in our hamster models. However, the introduction of TCG trinucleotides affected the <i>in vivo</i> phenotype of YFV. <i>In silico</i> analysis predicted that such patterns are likely to be selected against in flaviviruses infecting vertebrates. In addition, studies on influenza virus produced evidence that CpG dinucleotides in an A/U context (e.g., TCG) may enhance virus detection by the host innate immune system (<a href="#vey021-B23">Greenbaum et al. 2008</a>; <a href="#vey021-B24">Greenbaum, Rabadan, and Levine 2009</a>; <a href="#vey021-B32">Jimenez-Baranda et al. 2011</a>). Such trinucleotides can be used to induce a host-dependent attenuation, as recently demonstrated for dengue virus (<a href="#vey021-B68">Shen et al. 2015</a>; <a href="#vey021-B71">Simmonds et al. 2015</a>).</p> <p>This study approach was exploratory. Hence, we chose to focus on the identification of mutations with low deleterious impact on <i>in vivo</i> replication and virulence of the yellow fever virus (YFV) rather than on the comprehensive characterisation of the biological mechanisms underlying the attenuation process (e.g., protein translation, RNA replication, activation of innate immunity). Mechanistic insights on the specific effect(s) of the new re-encoding strategies would be most valuable, to gain insight into the finer details of viral attenuation and to facilitate fine-tuning of re-encoded viruses for the specific requirements of their intended use.</p> <p>The initial aim of this exploratory work was to identify types of mutation with limited detrimental impact on viral replicative fitness, usable for large scale genome re-encoding. In this regard, ‘non-specific’ transitions (i.e., with no CpG/UpA creation) represent the mutation type that best met our initial criteria. On this basis, we conclude that the introduction of transitions without CpG/UpA increase may constitute a ground rule for the custom-designed large-scale re-encoding of viral genomes. This does not rule out the possible use of mutations that increase the rate of CpG/UpA dinucleotides. We propose that using a limited number of such mutations may ensure that re-encoded genomes are relatively innocuous when tested <i>in vivo</i>, because these patterns ensure enhanced activation of host innate immunity. Thus, procedures combining large, pan-genomic non-specific re-encoding with limited bespoke CpG/UpA re-encoding should lead to the development of safe, stable, and effective live-attenuated vaccine viruses with finely tuned phenotypes.</p> <p>In addition to deciphering the mechanisms of attenuation, evaluating the stability of viral re-encoded variants is a crucial objective for the development of vaccine candidates. Here, we observed no reversion to the parental <i>Ap7M</i> sequence of YFV re-encoded viruses when tested <i>in vivo</i> and virtually no reversion <i>in vitro</i>. However, for several re-encoded strains, the occurrence of consensus non-synonymous mutations in the sequences from the inoculum and/or hamster livers suggests that YFV re-encoded viruses may adapt and evolve both <i>in vitro</i> and <i>in vivo</i>. The molecular evolution of viral re-encoded strains has already been described for HIV and CHIKV-derived strains <i>in vitro</i>. The occurrence of synonymous and non-synonymous consensus mutations correlated with partial recovery of (CHIKV) or full (HIV) WT replicative fitness (<a href="#vey021-B46">Martrus et al. 2013</a>; <a href="#vey021-B60">Nougairede et al. 2013</a>). In accordance with the observations made on CHIKV evolution <i>in vitro</i>, some re-encoded strains retained a strongly attenuated phenotype that was not outweighed by the emergence of adaptative consensus mutations. Altogether, these elements suggest that, in some cases, the accumulation of slightly detrimental mutations could trap the virus, with no alternative mutational pathway with which to enable complete restoration of replicative fitness.</p> <p>As illustrated by the case of poliovirus vaccines, live-attenuated vaccine strains have the potential to evolve following inoculation and in cases of dissemination following vaccination. Hence, the development of re-encoded vaccine candidates should be accompanied by robust studies that address the potential fate of the variants <i>in vivo</i>. However, to the best of our knowledge, the evolutionary behaviour of re-encoded viruses <i>in vivo</i> has never been studied. Since convenient <i>in vivo</i> laboratory models and reverse genetics systems are available, potential live-attenuated YFV re-encoded vaccine candidates would provide a convenient experimental model for future <i>in vivo</i> studies. In the studies described earlier, we focused mainly on YF variants re-encoded in one specific region of the genome (NS2A-NS4B). These studies were appropriate for our experimental objectives but would not be ideal for evolution studies. An optimised and more realistic design would rely on <i>in vivo</i> serial passage and deep sequencing characterisation of evenly distributed synonymous mutations, along the entire genome, each with low fitness impact. The major objective would be to assess the stability of re-encoded strains with synonymously modified codon compositions and in particular, their potential to evolve towards modified phenotypes <i>in vivo</i>.</p> </div> <div class="4.methods" title="sec"> <div class="title" tagx="title" title="title">
4. Methods</div> <p>Detailed protocols regarding <i>in silico</i> analysis, virus, and cell culture (viral production, transfection, and titration (TCID50); competition and virus neutralisation assays), PCR amplification (subgenomic amplicon production), quantitative real-time PCR (qRT-PCR) assays, sample collection, and Next-Generation Sequencing (NGS) stand in the <a href="#sup1">Supplementary Protocol</a>.</p> <div class="4.1cellsandanimals" title="sec"> <div class="title" tagx="title" title="title">
4.1 Cells and animals</div> <p>Viruses were produced in Baby hamster kidney BHK21 (ATCC, number CCL10) and Vero (ATCC, CCL81) cells and titrated in BHK21 cells. <i>In vivo</i> infection was performed in three-week-old female Syrian Golden hamsters (<i>Mesocricetus Auratus</i>, Janvier and Charles River laboratories).</p> </div> <div class="4.2ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
4.2 Ethics statement</div> <p>Animal protocols were reviewed and approved by the ethics committee ‘Comité d’éthique en expérimentation animale de Marseille—C2EA—14’ (protocol number 2015042015136843-V3 #528). All animal experiments were performed in compliance with French national guidelines and in accordance with the European legislation covering the use of animals for scientific purposes (Directive 210/63/EU).</p> </div> <div class="4.3designofre-encodedstrains" title="sec"> <div class="title" tagx="title" title="title">
4.3 Design of re-encoded strains</div> <div class="4.3.1designofre-encodedstrainsincludingonly‘non-specific’transitions" title="sec"> <div class="title" tagx="title" title="title">
4.3.1 Design of re-encoded strains including only ‘non-specific’ transitions</div> <p>Re-encoded strains YFV rTs1, rTs2, and rTs3, were designed by introducing increasing numbers of ‘simple’ transitions (siTs) in the CDS of the reference strain <i>Asibi</i> (AY640589). The term ‘simple’ refers to synonymous, non-specific (no CpG/UpA introduction) mutations located outside putative secondary structures. Strain YFV rTs4 included all mutations of YFV rTs3 and 388 additional non-specific transitions located between position 6,846 and 9,765 of the CDS, (741 transitions in total).</p> <p>Strain YFV rSS was re-encoded using 289 siTs as well as 50 non-specific transitions on sites corresponding to putative secondary structures (SII-Ts) identified in the parent sequence (described in the <a href="#sup1">Supplementary Results S2</a>).</p> </div> <div class="4.3.2designofre-encodedstrainsincluding‘specific’transitions" title="sec"> <div class="title" tagx="title" title="title">
4.3.2 Design of re-encoded strains including ‘specific’ transitions</div> <p>Viruses YFV rUA and rCG were designed by increasing the number of UpA and CpG dinucleotides in the viral sequence. Transitions leading to UpA dinucleotide introduction (UpA-Ts) were combined with siTs to design strain rUA (fifty-nine additional UpA). Similarly, rCG involved a combination of siTs and CpG-Ts (fifty-nine additional CpG).</p> <p>To investigate the impact of introducing TCG patterns on viral replication, mutations allowing the creation of TCG trinucleotides (TCG-Ts) were used for the design of re-encoded strain YFV rTCG (118 additional TCG). Because of the scarcity of editable sites allowing TCG introduction, design rules were relaxed: both transitions and transversions were used and some mutations affected sites corresponding to putative secondary structures (total: twelve) as well as non-editable sites (total: eleven).</p> <p>Finally, a combinatory strategy was used for YFV strain rN design. It included 233 mutations (179 transitions and 54 transversions) that led to the introduction of 83 UpA and 54 CpG dinucleotides into the viral sequence.</p> </div> </div> <div class="4.4recoveryofinfectiousvirusesandstockproduction" title="sec"> <div class="title" tagx="title" title="title">
4.4 Recovery of infectious viruses and stock production</div> <p>All the strains used in this study (one WT, one hamster-adapted and seventeen re-encoded) were produced using the ISA method, that enables recovery of infectious viruses after transfection into permissive cells of overlapping subgenomic DNA fragments covering the entire genome (<a href="#vey021-B4">Aubry et al. 2014</a>). All fragment combination schemes used for virus production are detailed in <a href="#vey021-F1">Fig. 1</a> and in the ‘Fragment combination’ section.</p> <div class="4.4.1fragmentscombinations" title="sec"> <div class="title" tagx="title" title="title">
4.4.1 Fragments combinations</div> <p>For <i>Asibi</i> strain production, three subgenomic fragments were used: FI (positions 1–3,919 in CDS and the 5′-UTR (119 nucleotides (nt)), FII (3,843–6,838) and FIII (6,783–10,236 and 3′-UTR (509 nt)). They correspond to the complete genome of the strain, flanked, respectively, at 5′ and 3′ termini by the human cytomegalovirus promoter (pCMV) and the hepatitis delta ribozyme followed by the simian virus forty polyadenylation signal (HDR/SV40pA) (pCMV and HDR/SV40pA sequences were described by <a href="#vey021-B4">Aubry et al. 2014</a>). The hamster-virulent strain YFV <i>Ap7M</i>, was obtained from <i>Asibi</i> virus by substituting the FI subgenomic fragment by the FIAp7 fragment, that includes ten hamster-virulence mutations as described elsewhere (R. Klitting et al., accepted manuscript).</p> <p>Re-encoded strains rTs1, rTs2, rTs3, rUA, rCG, rSS, rN, and rTCG were derived from <i>Asibi</i> by combining a re-encoded version of the second subgenomic fragment FII to the WT first (FI) and third (FIII) subgenomic fragments. For the heavily re-encoded rTs4 strain, the WT fragment FI was combined to the re-encoded FII fragment of strain rTs3 and to a re-encoded version of FIII fragment, FIIIR.</p> <p>All hamster-adapted re-encoded strains (rTs1h, rTs2h, rTs3h, rUAh, rCGh, rSSh, rNh, and rTCGh) were obtained by replacing the WT FI fragment with the <i>Ap7M</i> FI fragment, FI<i>Ap7</i>.</p> </div> <div class="4.4.2productionandrecovery" title="sec"> <div class="title" tagx="title" title="title">
4.4.2 Production and recovery</div> <p>For producing each of the different YFV strains described in this study, 3 overlapping DNA fragments were synthesised <i>de novo</i> (Genscript) and amplified by High Fidelity PCR using the Platinum PCR SuperMix High Fidelity kit (Life Technologies) and specific sets of primers. After amplification, all DNA fragments were purified using Monarch PCR &amp;amp; DNA Cleanup kit 5 µg (BioLabs) according to the manufacturer’s instructions. Details regarding the subgenomic DNA fragments and the sets of primers used for the amplification step are available in the <a href="#sup1">Supplementary Table S6</a>. The different subgenomic fragment combinations are described in <a href="#vey021-F1">Fig. 1</a>.</p> <p>A final amount of 1 µg of DNA (equimolar mix of subgenomic cDNA fragments) was transfected using Lipofectamine 3000 (Life Technologies) in a 25-cm<sup>2</sup> culture flask of subconfluent cells containing 1 mL of culture medium without antibiotics. The cell supernatant media were harvested at 9-day post-transfection (dpi), aliquoted, and stored at −80°C. Each virus was then passaged twice in Vero (<i>Asibi</i>-derived) or BHK21 (<i>Ap7M</i>-derived) cells and once in BHK21 cells. Clarified cell supernatant media from the second passage (virus stocks) were stored and used to perform viral RNA quantification, TCID50 assays and whole-genome sequencing (see section 4.6, 4.10 and in the <a href="#sup1">supplementary data</a>).</p> </div> </div> <div class="4.5nucleicacidextraction" title="sec"> <div class="title" tagx="title" title="title">
4.5 Nucleic acid extraction</div> <p>Samples (liver homogenate or cell culture supernatant medium) were extracted using either EZ1 Biorobot (EZ1 Virus Mini kit v2) or the QiaCube HT device (CadorPathogen kit) both from Qiagen. Inactivation was performed using either 200 μL of AVL buffer (EZ1) or 100 μL of VXL buffer and 100 μL HBSS (Qiacube) according to the manufacturer’s instructions.</p> </div> <div class="4.6quantitativereal-timert-pcrassays" title="sec"> <div class="title" tagx="title" title="title">
4.6 Quantitative real-time RT-PCR assays</div> <p>All quantitative real-time PCR (qRT-PCR) assays were performed using the EXPRESS SuperScript kit for One-Step qRT-PCR (Invitrogen) and the GoTaq<sup>®</sup> Probe 1-Step RT-qPCR System (Promega). Primers and probe sequences are detailed in the <a href="#sup1">Supplementary Table S6</a>.</p> <p>The amount of viral RNA was calculated from standard curves using a synthetic RNA transcript (concentrations of 10<sup>7</sup>, 10<sup>6</sup>, 10<sup>5</sup>, and 10<sup>4</sup> copies/μL). Results were normalised using amplification (qRT-PCR) of the housekeeping gene actin (as described by Piorkowski and colleagues (<a href="#vey021-B503">Piorkowski et al. 2014</a>)). The standard curves generated for all the YFV-specific assays had coefficients of determination values (<i>R</i><sup>2</sup>) &amp;gt;0.98 and amplification efficiencies were between 93 and 100 per cent.</p> </div> <div class="4.7competitionassays" title="sec"> <div class="title" tagx="title" title="title">
4.7 Competition assays</div> <p>WT virus was grown in competition with each of the nine, <i>Asibi</i>-derived, re-encoded viruses using two RNA ratios in triplicate (WT/re-encoded virus: 10/90 and 50/50). A global estimated MOI of 0.5 was used for the first inoculation. Viruses from each experiment were then passaged nine times with an incubation time of three days. At each passage, the estimated MOI was ∼0.6. After each infection (3 dpi), nucleic acids were extracted from the clarified culture supernatant medium using the QiaCube HT device (see section 4.5) and then used for RT-PCR amplification. The detection of each virus was achieved using next generation sequencing (NGS) methods (see section 4.10), by amplifying a 256 nucleotides region and counting virus-specific patterns within the read population.</p> </div> <div class="4.8invivoexperiments" title="sec"> <div class="title" tagx="title" title="title">
4.8 <i>In v</i><i>ivo</i> experiments</div> <p>The laboratory hamster model reproduces the features of human YF but requires the use of adapted strains (<a href="#vey021-B501">Julander 2016</a>). We recently implemented a hamster model for YFV, based on the use of a strain equivalent to the <i>Asibi</i>/hamster p7 strain described by <a href="#vey021-B47">McArthur et al. (2003)</a>, <i>Ap7M</i>. When inoculated into hamsters, this strain induces a lethal viscerotropic disease similar to that described for YF <i>Ap7</i> virus in terms of (1) clinical signs of illness, (2) weight evolution, (3) viral loads in the liver, and (iv) lethality (100 per cent). This strain was derived from <i>Asibi</i> by including ten mutations in the FI subgenomic region used for viral recovery with the ISA method. As we chose to target the FII subgenomic region for re-encoding, the <i>Ap7M</i> strain was well suited for testing the effects of re-encoding on the biological properties of viruses <i>in vivo</i>.</p> <p>Three-week-old female Syrian Golden hamsters were inoculated intra-peritoneally with 5×10<sup>5</sup> TCID50 of virus in a final volume of 100 μL HBSS. In all experiments, a control group of two hamsters was kept uninfected. The clinical course of the viral infection was monitored by following (1) the clinical manifestation of the disease (brittle fur, dehydration, prostration, and lethargy), (2) weight evolution, and (3) death. Weight was expressed as a normalised percentage of initial weight (%IW) and calculated as follows: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
%</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
W</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
W</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
%</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
W</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div></div></div> (<i>W<sub>dn</sub></i>: weight at day n; <i>W<sub>d0</sub></i>: weight on the day of inoculation or challenge; <i>IW<sub>n</sub></i>: mean of the <i>%IW</i> for control hamsters).<p /> <p>Liver samples were obtained from euthanised hamsters, ground, and treated with proteinase K (PK) before nucleic acid extraction using either the EZ1 Biorobot or the QiaCube HT device (see section 4.5). Serum was recovered from euthanised hamsters and stored (−80°C).</p> </div> <div class="4.9virusneutralisationassay" title="sec"> <div class="title" tagx="title" title="title">
4.9 Virus neutralisation assay</div> <p>Sera were incubated for 20 min at 56°C prior to viral serology. For each serum, serial five-fold dilutions (first dilution: 1:250) of serum were tested on BHK21 cells for <i>Ap7M</i> strain infection inhibition. The plates were incubated for 44 h before nucleic acid extraction. Virus quantification was achieved using the YFV-specific qRT-PCR system as described earlier. Then, a viral RNA yield reduction (per cent of viral inhibition) was calculated for each well as follows: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> </div></div></div> <p /> <p>With <i>qYFV<sub>w</sub></i> the number of RNA copies in the analysed well and qYFV<sub>m</sub> the mean number of RNA copies in the negative control wells. Fifty percent neutralisation titre NT<sub>50</sub> values were determined from plots of per cent of viral neutralisation versus serum dilutions and calculated by non-linear regression (four-parameter dose–response curve) analysis using Graphpad PRISM software (v7.00 for Windows, GraphPad Software, La Jolla, CA, <a href="http://www.graphpad.com">www.graphpad.com</a>).</p> </div> <div class="4.10wholeandpartialgenomesequencing" title="sec"> <div class="title" tagx="title" title="title">
4.10 Whole and partial genome sequencing</div> <p>Nucleotide sequences were determined using NGS methods: overlapping amplicons spanning either the complete genome sequence or a 256 nucleotides region (nucleotide positions: 4,391–4,646) were produced from the extracted RNA using the SuperScript<sup>®</sup> III One-Step RT-PCR System with Platinum<sup>®</sup>Taq High Fidelity kit (Invitrogen) and specific primers (detailed in the <a href="#sup1">Supplementary Table S7</a>). Sequencing was performed using the PGM Ion torrent technology (Thermo Fisher Scientific) following the manufacturer’s instructions.</p> <div class="4.10.1sequencedetermination" title="sec"> <div class="title" tagx="title" title="title">
4.10.1 Sequence determination</div> <p>CDS consensus sequence determination was done using CLC genomics workbench software (CLC bio-Qiagen). Substitutions with a frequency &amp;gt;5 per cent were considered for the analysis of intra-population genetic diversity and major/minor variant subpopulations identification (minor variants: 5 per cent &amp;lt; variants frequency ≤75 per cent; major variants: variants frequency &amp;gt;75 per cent). No variant was taken into account with a coverage lower than 1,000 reads. No consensus mutation was taken into account with a coverage lower than fifty reads.</p> </div> <div class="4.10.2re-encodedvirusquantification" title="sec"> <div class="title" tagx="title" title="title">
4.10.2 Re-encoded virus quantification</div> <p>After sequencing the 256 nucleotide region using the PGM Ion torrent technology. Automated read datasets provided by Torrent software suite 5.0.2 were trimmed according to quality score using CLC genomics workbench software (CLC bio-Qiagen) and six re-encoded or wild-type specific patterns (see <a href="#sup1">Supplementary Tables S8–S10</a>) were counted within the read datasets using in-house software. As a control, two patterns common to both viruses were counted to evaluate the total amount of virus for each sample.</p> </div> </div> <div class="4.11statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.11 Statistical analysis</div> <p>Viral load comparisons were achieved using Wilcoxon rank-sum test with a continuity correction and Kaplan–Meier survival analysis, using Mandel–Cox’s Logrank tests. Both analyses were performed using R software (<a href="#vey021-B504">R Core Team 2013</a>). P values &amp;lt;0.05 were considered as significant.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Data</span><div class="media" title="media"><a href="vey021_supp.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Morgan Seston, Ludivine Molina, and Karine Barthélémy from UMR UVE, "Unité des Virus Émergents�? for their technical assistance. We thank Stéphane Priet from UMR UVE, "Unité des Virus Émergents�? for his careful re-reading of the article.</p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the European Virus Archive goes Global, <a href="http://global.europeanvirus-archive.com/">http://global.europeanvirus-archive.com/</a> (European Union’s Horizon 2020 research and innovation programme under grant agreement no. 653316); and the Agence Nationale de la Recherche, <a href="http://www.agence-nationalerecherche.fr/">http://www.agence-nationalerecherche.fr/</a> (grant ANR-14CE14-0001 RNA Vacci-Code). The funders had no role in study design, data collection, and analysis, decision to publish, or preparation of the article.</p> </div> <div class="supplementarydata" title="sec"> <div class="title" tagx="title" title="title">
Supplementary data</div> <p><a href="#sup1">Supplementary data</a> are available at <i>Virus Evolution</i> online.</p> <p><b>Conflict of interest:</b> None declared.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="vey021-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Pathogenesis of Modoc Virus (Flaviviridae; Flavivirus) in Persistently Infected Hamsters</span>’, <span class="source" tagx="source" title="source">The American Journal of Tropical Medicine and Hygiene</span>, <span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">455</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23358636">23358636</a></span></span></li> <li tag="ref"><a name="vey021-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">Q. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">Z. A.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Yellow Fever from Angola and Congo: A Storm Gathers</span>’, <span class="source" tagx="source" title="source">Tropical Doctor</span>, <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">92</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28424031">28424031</a></span></span></li> <li tag="ref"><a name="vey021-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson</span><span class="given-names" tagx="given-names" title="given-names">N. J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Influence of CpG and UpA Dinucleotide Frequencies on RNA Virus Replication and Characterization of the Innate Cellular Pathways Underlying Virus Attenuation and Enhanced Replication</span>’, <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">4527</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24470146">24470146</a></span></span></li> <li tag="ref"><a name="vey021-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aubry</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Single-Stranded Positive-Sense RNA Viruses Generated in Days Using Infectious Subgenomic Amplicons</span>’, <span class="source" tagx="source" title="source">The Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">2462</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25053561">25053561</a></span></span></li> <li tag="ref"><a name="vey021-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">M. G. S.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="chapter-title" tagx="chapter-title" title="chapter-title">‘Yellow Fever Vaccine’,</span> in <span class="source" tagx="source" title="source">Vaccines</span>, <span class="edition" tagx="edition" title="edition">6th edn</span>, <span class="fpage" tagx="fpage" title="fpage">870</span>–<span class="lpage" tagx="lpage" title="lpage">968</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span> Inc.</span></li> <li tag="ref"><a name="vey021-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1985</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Amino-Terminal Amino Acid Sequences of Structural Proteins of Three Flaviviruses</span>’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">143</span>: <span class="fpage" tagx="fpage" title="fpage">224</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2998002">2998002</a></span></span></li> <li tag="ref"><a name="vey021-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bird</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span></span> (<span class="year" tagx="year" title="year">1980</span>) ‘<span class="mixed-article-title" title="mixed-article-title">DNA Methylation and the Frequency of CpG in Animal DNA</span>’, <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">1499</span>–<span class="lpage" tagx="lpage" title="lpage">504</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6253938">6253938</a></span></span></li> <li tag="ref"><a name="vey021-B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birnbaum</span><span class="given-names" tagx="given-names" title="given-names">R. Y.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Coding Exons Function as Tissue-Specific Enhancers of Nearby Genes</span>’, <span class="source" tagx="source" title="source">Genome Research</span>, <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">1059</span>–<span class="lpage" tagx="lpage" title="lpage">68</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22442009">22442009</a></span></span></li> <li tag="ref"><a name="vey021-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brest</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) ‘<span class="mixed-article-title" title="mixed-article-title">A Synonymous Variant in IRGM Alters a Binding Site for miR-196 and Causes Deregulation of IRGM-Dependent Xenophagy in Crohn’s Disease</span>’, <span class="source" tagx="source" title="source">Nature Genetics</span>, <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">242</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278745">21278745</a></span></span></li> <li tag="ref"><a name="vey021-B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burns</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2006</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Modulation of Poliovirus Replicative Fitness in HeLa Cells by Deoptimization of Synonymous Codon Usage in the Capsid Region</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">3259</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16537593">16537593</a></span></span></li> <li tag="ref"><a name="vey021-B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buynak</span><span class="given-names" tagx="given-names" title="given-names">E. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilleman</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span></span> (<span class="year" tagx="year" title="year">1966</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Live Attenuated Mumps Virus Vaccine. 1. Vaccine Development</span>’, <span class="source" tagx="source" title="source">Proceedings of the Society for Experimental Biology and Medicine. Society for Experimental Biology and Medicine (New York, N.Y.)</span>, <span class="volume" tagx="volume" title="volume">123</span>: <span class="fpage" tagx="fpage" title="fpage">768</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>.</span></li> <li tag="ref"><a name="vey021-B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrington</span><span class="given-names" tagx="given-names" title="given-names">C. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auguste</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Evolutionary and Ecological Factors Underlying the Tempo and Distribution of Yellow Fever Virus Activity</span>’, <span class="source" tagx="source" title="source">Infection, Genetics and Evolution</span>, <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">198</span>–<span class="lpage" tagx="lpage" title="lpage">210</span>.</span></li> <li tag="ref"><a name="vey021-B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S. L.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Codon Usage between Genomes Is Constrained by Genome-Wide Mutational Processes</span>’, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">101</span>: <span class="fpage" tagx="fpage" title="fpage">3480</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14990797">14990797</a></span></span></li> <li tag="ref"><a name="vey021-B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conrad</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation of Marek’s Disease Virus by Codon Pair Deoptimization of a Core Gene</span>’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">516</span>: <span class="fpage" tagx="fpage" title="fpage">219</span>–<span class="lpage" tagx="lpage" title="lpage">26</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29407380">29407380</a></span></span></li> <li tag="ref"><a name="vey021-B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Fabritus</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation of Tick-Borne Encephalitis Virus Using Large-Scale Random Codon Re-Encoding</span>’, <span class="source" tagx="source" title="source">PLoS Pathogens</span>, <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e1004738</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25734338">25734338</a></span></span></li> <li tag="ref"><a name="vey021-B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellis</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Enigma of Yellow Fever in East Africa</span>’, <span class="source" tagx="source" title="source">Reviews in Medical Virology</span>, <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">331</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18615782">18615782</a></span></span></li> <li tag="ref"><a name="vey021-B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) ‘<span class="mixed-article-title" title="mixed-article-title">WHO Working Group on Technical Specifications for Manufacture and Evaluation of Yellow Fever Vaccines, Geneva, Switzerland, 13-14 May 2009</span>’, <span class="source" tagx="source" title="source">Vaccine</span>, <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">8236</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21055492">21055492</a></span></span></li> <li tag="ref"><a name="vey021-B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaunt</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Elevation of CpG Frequencies in Influenza a Genome Attenuates Pathogenicity but Enhances Host Response to Infection</span>’, <span class="source" tagx="source" title="source">eLife</span>, <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e12735</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26878752">26878752</a></span></span></li> <li tag="ref"><a name="vey021-B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaunt</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2001</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylogenetic Relationships of Flaviviruses Correlate with Their Epidemiology, Disease Association and Biogeography</span>’, <span class="source" tagx="source" title="source">Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">82</span>: <span class="fpage" tagx="fpage" title="fpage">1867</span>–<span class="lpage" tagx="lpage" title="lpage">76</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11457992">11457992</a></span></span></li> <li tag="ref"><a name="vey021-B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Germain</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1981</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Sylvatic Yellow Fever in Africa Recent Advances and Present Approach (Author’s Transl</span>’, <span class="source" tagx="source" title="source">Medecine Tropicale</span>, <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">31</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6116147">6116147</a></span></span></li> <li tag="ref"><a name="vey021-B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Pathogenic Flaviviruses</span>’, <span class="source" tagx="source" title="source">Lancet (London, England)</span>, <span class="volume" tagx="volume" title="volume">371</span>: <span class="fpage" tagx="fpage" title="fpage">500</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="vey021-B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grard</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2007</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Genetic Characterization of Tick-Borne Flaviviruses: New Insights into Evolution, Pathogenetic Determinants and Taxonomy</span>’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">361</span>: <span class="fpage" tagx="fpage" title="fpage">80</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17169393">17169393</a></span></span></li> <li tag="ref"><a name="vey021-B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">B. D.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Patterns of Evolution and Host Gene Mimicry in Influenza and Other RNA Viruses</span>’, <span class="source" tagx="source" title="source">PLoS Pathogens</span>, <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e1000079</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18535658">18535658</a></span></span></li> <li tag="ref"><a name="vey021-B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">B. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabadan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levine</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Patterns of Oligonucleotide Sequences in Viral and Host Cell RNA Identify Mediators of the Host Innate Immune System</span>’, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e5969</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19536338">19536338</a></span></span></li> <li tag="ref"><a name="vey021-B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gritsun</span><span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Direct Repeats in the 3’ Untranslated Regions of Mosquito-Borne Flaviviruses: Possible Implications for Virus Transmission</span>’, <span class="source" tagx="source" title="source">The Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">87</span>: <span class="fpage" tagx="fpage" title="fpage">3297</span>–<span class="lpage" tagx="lpage" title="lpage">305</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17030864">17030864</a></span></span></li> <li tag="ref"><a name="vey021-B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gritsun</span><span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span></span> (<span class="year" tagx="year" title="year">2007a</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Direct Repeats in the Flavivirus 3’ Untranslated Region; a Strategy for Survival in the Environment?</span> ’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">358</span>: <span class="fpage" tagx="fpage" title="fpage">258</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17067651">17067651</a></span></span></li> <li tag="ref"><a name="vey021-B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gritsun</span><span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span></span> (<span class="year" tagx="year" title="year">2007b</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Origin and Evolution of Flavivirus 5’UTRs and Panhandles: Trans-Terminal Duplications?</span>’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">366</span>: <span class="fpage" tagx="fpage" title="fpage">8</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17658577">17658577</a></span></span></li> <li tag="ref"><a name="vey021-B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanley</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Fever versus Fever: The Role of Host and Vector Susceptibility and Interspecific Competition in Shaping the Current and Future Distributions of the Sylvatic Cycles of Dengue Virus and Yellow Fever Virus</span>’, <span class="source" tagx="source" title="source">Infection, Genetics and Evolution</span>, <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">292</span>–<span class="lpage" tagx="lpage" title="lpage">311</span>.</span></li> <li tag="ref"><a name="vey021-B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilleman</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1968</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Development and Evaluation of the Moraten Measles Virus Vaccine</span>’, <span class="source" tagx="source" title="source">JAMA</span>, <span class="volume" tagx="volume" title="volume">206</span>: <span class="fpage" tagx="fpage" title="fpage">587</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5695578">5695578</a></span></span></li> <li tag="ref"><a name="vey021-B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikemura</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">1985</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Codon Usage and tRNA Content in Unicellular and Multicellular Organisms</span>’, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">13</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708">3916708</a></span></span></li> <li tag="ref"><a name="vey021-B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikic</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1972</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation and Characterisation of Edmonston-Zagreb Measles Virus</span>’, <span class="source" tagx="source" title="source">Annales Immunologiae Hungaricae</span>, <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4679646">4679646</a></span></span></li> <li tag="ref"><a name="vey021-B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jimenez-Baranda</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Oligonucleotide Motifs That Disappear during the Evolution of Influenza Virus in Humans Increase Alpha Interferon Secretion by Plasmacytoid Dendritic Cells</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">3893</span>–<span class="lpage" tagx="lpage" title="lpage">904</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21307198">21307198</a></span></span></li> <li tag="ref"><a name="vey021-B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">H. N.</span></span></span> (<span class="year" tagx="year" title="year">1970</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Long-Term Persistence of Modoc Virus in Hamster-Kidney Cells. In Vivo and in Vitro Demonstration</span>’, <span class="source" tagx="source" title="source">The American Journal of Tropical Medicine and Hygiene</span>, <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">537</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5446320">5446320</a></span></span></li> <li tag="ref"><a name="vey021-B501" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Julander</span><span class="given-names" tagx="given-names" title="given-names">J. G.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Animal models of yellow fever and their application in clinical research</span>’, <span class="source" tagx="source" title="source">Current Opinion in Virology</span>, <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">64</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27093699">27093699</a></span></span></li> <li tag="ref"><a name="vey021-B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlin</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doerfler</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardon</span><span class="given-names" tagx="given-names" title="given-names">L. R.</span></span></span> (<span class="year" tagx="year" title="year">1994</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Why Is CpG Suppressed in the Genomes of Virtually All Small Eukaryotic Viruses but Not in Those of Large Eukaryotic Viruses?</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">2889</span>–<span class="lpage" tagx="lpage" title="lpage">97</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8151759">8151759</a></span></span></li> <li tag="ref"><a name="vey021-B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Virus Taxonomy</span>’, <span class="source" tagx="source" title="source">Ninth Report of the International Committee on Taxonomy of Viruses</span>. Elsevier Inc.</span></li> <li tag="ref"><a name="vey021-B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klitting</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Molecular Determinants of Yellow Fever Virus Pathogenicity in Syrian Golden Hamsters: One Mutation Away from Virulence</span>’, <span class="source" tagx="source" title="source">Emerging Microbes &amp;amp; Infections</span>, <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29593212">29593212</a></span></span></li> <li tag="ref"><a name="vey021-B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kow</span><span class="given-names" tagx="given-names" title="given-names">Y. W.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Repair of Deaminated Bases in DNA</span>’, <span class="source" tagx="source" title="source">Free Radical Biology &amp;amp; Medicine</span>, <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">886</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12361800">12361800</a></span></span></li> <li tag="ref"><a name="vey021-B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">M. U.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Spread of Yellow Fever Virus Outbreak in Angola and the Democratic Republic of the Congo 2015-16: A Modelling Study</span>’, <span class="source" tagx="source" title="source">The Lancet Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">330</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28017559">28017559</a></span></span></li> <li tag="ref"><a name="vey021-B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudla</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Coding-Sequence Determinants of Gene Expression in Escherichia coli</span>’, <span class="source" tagx="source" title="source">Science (New York, N.Y.)</span>, <span class="volume" tagx="volume" title="volume">324</span>: <span class="fpage" tagx="fpage" title="fpage">255</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="vey021-B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuno</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1998</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylogeny of the Genus Flavivirus</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">72</span>: <span class="fpage" tagx="fpage" title="fpage">73</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9420202">9420202</a></span></span></li> <li tag="ref"><a name="vey021-B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Nouen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation of Human Respiratory Syncytial Virus by Genome-Scale Codon-Pair Deoptimization</span>’, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">111</span>: <span class="fpage" tagx="fpage" title="fpage">13169</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25157129">25157129</a></span></span></li> <li tag="ref"><a name="vey021-B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leyssen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2006</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Acute Encephalitis, a Poliomyelitis-like Syndrome and Neurological Sequelae in a Hamster Model for Flavivirus Infections</span>’, <span class="source" tagx="source" title="source">Brain Pathology</span>, <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">279</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>.</span></li> <li tag="ref"><a name="vey021-B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Characterization of Four Vaccine-Related Polioviruses Including Two Intertypic Type 3/Type 2 Recombinants Associated with Aseptic Encephalitis</span>’, <span class="source" tagx="source" title="source">Virology Journal</span>, <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">162</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27677968">27677968</a></span></span></li> <li tag="ref"><a name="vey021-B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukashev</span><span class="given-names" tagx="given-names" title="given-names">A. N.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2003</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Recombination in Circulating Enteroviruses</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">77</span>: <span class="fpage" tagx="fpage" title="fpage">10423</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12970427">12970427</a></span></span></li> <li tag="ref"><a name="vey021-B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Synonymous Virus Genome Recoding as a Tool to Impact Viral Fitness</span>’, <span class="source" tagx="source" title="source">Trends in Microbiology</span>, <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">134</span>–<span class="lpage" tagx="lpage" title="lpage">47</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26646373">26646373</a></span></span></li> <li tag="ref"><a name="vey021-B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martrus</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Changes in Codon-Pair Bias of Human Immunodeficiency Virus Type 1 Have Profound Effects on Virus Replication in Cell Culture</span>’, <span class="source" tagx="source" title="source">Retrovirology</span>, <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">78</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23885919">23885919</a></span></span></li> <li tag="ref"><a name="vey021-B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McArthur</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2003</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Molecular Characterization of a Hamster Viscerotropic Strain of Yellow Fever Virus</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">77</span>: <span class="fpage" tagx="fpage" title="fpage">1462</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12502861">12502861</a></span></span></li> <li tag="ref"><a name="vey021-B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minor</span><span class="given-names" tagx="given-names" title="given-names">P. D.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Polio-Eradication Programme and Issues of the End Game</span>’, <span class="source" tagx="source" title="source">The Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">93</span>: <span class="fpage" tagx="fpage" title="fpage">457</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22128250">22128250</a></span></span></li> <li tag="ref"><a name="vey021-B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mir</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylodynamics of Yellow Fever Virus in the Americas: New Insights into the Origin of the 2017 Brazilian Outbreak</span>’, <span class="source" tagx="source" title="source">Scientific Reports</span>, <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">7385</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28785067">28785067</a></span></span></li> <li tag="ref"><a name="vey021-B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Yellow Fever Vaccine</span>’, <span class="source" tagx="source" title="source">Expert Review of Vaccines</span>, <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">553</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16117712">16117712</a></span></span></li> <li tag="ref"><a name="vey021-B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Treatment of Yellow Fever</span>’, <span class="source" tagx="source" title="source">Antiviral Research</span>, <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">116</span>–<span class="lpage" tagx="lpage" title="lpage">24</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18061688">18061688</a></span></span></li> <li tag="ref"><a name="vey021-B52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Review of the Risks and Benefits of Yellow Fever Vaccination Including Some New Analyses</span>’, <span class="source" tagx="source" title="source">Expert Review of Vaccines</span>, <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">427</span>–<span class="lpage" tagx="lpage" title="lpage">48</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22551029">22551029</a></span></span></li> <li tag="ref"><a name="vey021-B53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Pathogenesis and Pathophysiology of Yellow Fever</span>’, <span class="source" tagx="source" title="source">Advances in Virus Research</span>, <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">343</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14689698">14689698</a></span></span></li> <li tag="ref"><a name="vey021-B54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">P. F.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Yellow Fever</span>’, <span class="source" tagx="source" title="source">Journal of Clinical Virology</span>, <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">160</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25453327">25453327</a></span></span></li> <li tag="ref"><a name="vey021-B55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moratorio</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation of RNA Viruses by Redirecting Their Evolution in Sequence Space</span>’, <span class="source" tagx="source" title="source">Nature Microbiology</span>, <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">17088</span>.</span></li> <li tag="ref"><a name="vey021-B56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moureau</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">New Insights into Flavivirus Evolution, Taxonomy and Biogeographic History, Extended by Analysis of Canonical and Alternative Coding Sequences</span>’, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0117849</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25719412">25719412</a></span></span></li> <li tag="ref"><a name="vey021-B57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mueller</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2006</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Reduction of the Rate of Poliovirus Protein Synthesis through Large-Scale Codon Deoptimization Causes Attenuation of Viral Virulence by Lowering Specific Infectivity</span>’, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">9687</span>–<span class="lpage" tagx="lpage" title="lpage">96</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973573">16973573</a></span></span></li> <li tag="ref"><a name="vey021-B58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mueller</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Live Attenuated Influenza Virus Vaccines by Computer-Aided Rational Design</span>’, <span class="source" tagx="source" title="source">Nature Biotechnology</span>, <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">723</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="vey021-B59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">W. C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The 5’ and 3’ Untranslated Regions of the Flaviviral Genome</span>’, <span class="source" tagx="source" title="source">Viruses</span>, <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">137</span>.</span></li> <li tag="ref"><a name="vey021-B60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nougairede</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Random Codon Re-Encoding Induces Stable Reduction of Replicative Fitness of Chikungunya Virus in Primate and Mosquito Cells</span>’, <span class="source" tagx="source" title="source">PLoS Pathogens</span>, <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e1003172</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23436995">23436995</a></span></span></li> <li tag="ref"><a name="vey021-B61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pechmann</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frydman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Evolutionary Conservation of Codon Optimality Reveals Hidden Signatures of Cotranslational Folding</span>’, <span class="source" tagx="source" title="source">Nature Structural &amp;amp; Molecular Biology</span>, <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">237</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.</span></li> <li tag="ref"><a name="vey021-B503" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piorkowski</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Development of generic Taqman PCR and RT-PCR assays for the detection of DNA and mRNA of beta-actin-encoding sequences in a wide range of animal species</span>’, <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">202</span>: <span class="fpage" tagx="fpage" title="fpage">101</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24642236">24642236</a></span></span></li> <li tag="ref"><a name="vey021-B62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quaresma</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Immunity and Immune Response, Pathology and Pathologic Changes: Progress and Challenges in the Immunopathology of Yellow Fever</span>’, <span class="source" tagx="source" title="source">Reviews in Medical Virology</span>, <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">305</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23873723">23873723</a></span></span></li> <li tag="ref"><a name="vey021-B504" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: R Core Team</span>.</span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">R: A language and environment for statistical computing</span>’, in <span class="source" tagx="source" title="source">R Foundation for Statistical Computing</span>, Vienna, Austria. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</span></li> <li tag="ref"><a name="vey021-B63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rice</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1985</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Nucleotide Sequence of Yellow Fever Virus: Implications for Flavivirus Gene Expression and Evolution</span>’, <span class="source" tagx="source" title="source">Science (New York, N.Y.)</span>, <span class="volume" tagx="volume" title="volume">229</span>: <span class="fpage" tagx="fpage" title="fpage">726</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>.</span></li> <li tag="ref"><a name="vey021-B64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rima</span><span class="given-names" tagx="given-names" title="given-names">B. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McFerran</span><span class="given-names" tagx="given-names" title="given-names">N. V.</span></span></span> (<span class="year" tagx="year" title="year">1997</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Dinucleotide and Stop Codon Frequencies in Single-Stranded RNA Viruses</span>’, <span class="source" tagx="source" title="source">Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">2859</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9367373">9367373</a></span></span></li> <li tag="ref"><a name="vey021-B65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabin</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulger</span><span class="given-names" tagx="given-names" title="given-names">L. R.</span></span></span> (<span class="year" tagx="year" title="year">1973</span>) ‘<span class="mixed-article-title" title="mixed-article-title">History of Sabin Attenuated Poliovirus Oral Live Vaccine Strains</span>’, <span class="source" tagx="source" title="source">Journal of Biological Standardization</span>, <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">115</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="vey021-B66" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarz</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">1962</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Preliminary Tests of a Highly Attenuated Measles Vaccine</span>’, <span class="source" tagx="source" title="source">American Journal of Diseases of Children (1960)</span>, <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">386</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13909753">13909753</a></span></span></li> <li tag="ref"><a name="vey021-B67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shabalina</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogurtsov</span><span class="given-names" tagx="given-names" title="given-names">A. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spiridonov</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>) ‘<span class="mixed-article-title" title="mixed-article-title">A Periodic Pattern of mRNA Secondary Structure Created by the Genetic Code</span>’, <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">2428</span>–<span class="lpage" tagx="lpage" title="lpage">37</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16682450">16682450</a></span></span></li> <li tag="ref"><a name="vey021-B68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Large-Scale Recoding of an Arbovirus Genome to Rebalance Its Insect versus Mammalian Preference</span>’, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">4749</span>–<span class="lpage" tagx="lpage" title="lpage">54</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25825721">25825721</a></span></span></li> <li tag="ref"><a name="vey021-B69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmen</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Genome-Scale Relationships between Cytosine Methylation and Dinucleotide Abundances in Animals</span>’, <span class="source" tagx="source" title="source">Genomics</span>, <span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">33</span>–<span class="lpage" tagx="lpage" title="lpage">40</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18485662">18485662</a></span></span></li> <li tag="ref"><a name="vey021-B70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">SSE: A Nucleotide and Amino Acid Sequence Analysis Platform</span>’, <span class="source" tagx="source" title="source">BMC Research Notes</span>, <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22264264">22264264</a></span></span></li> <li tag="ref"><a name="vey021-B71" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Attenuation of Dengue (and Other RNA Viruses) with Codon Pair Recoding Can Be Explained by Increased CpG/UpA Dinucleotide Frequencies</span>’, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">E3633</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26071447">26071447</a></span></span></li> <li tag="ref"><a name="vey021-B72" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ter MEULEN</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Activation of the Cytokine Network and Unfavorable Outcome in Patients with Yellow Fever</span>’, <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">190</span>: <span class="fpage" tagx="fpage" title="fpage">1821</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15499539">15499539</a></span></span></li> <li tag="ref"><a name="vey021-B502" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesh</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">‘Experimental yellow fever virus infection in the Golden Hamster (Mesocricetus auratus). I. Virologic, biochemical, and immunologic studies</span>’, <span class="source" tagx="source" title="source">Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">183</span>: <span class="fpage" tagx="fpage" title="fpage">1431</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11319679">11319679</a></span></span></li> <li tag="ref"><a name="vey021-B73" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theiler</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">H. H.</span></span></span> (<span class="year" tagx="year" title="year">1937</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Effect of Prolonged Cultivation in Vitro upon the Pathogenicity of Yellow Fever Virus</span>’, <span class="source" tagx="source" title="source">The Journal of Experimental Medicine</span>, <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">767</span>–<span class="lpage" tagx="lpage" title="lpage">86</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19870633">19870633</a></span></span></li> <li tag="ref"><a name="vey021-B74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thurner</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Conserved RNA Secondary Structures in Flaviviridae Genomes</span>’, <span class="source" tagx="source" title="source">The Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">1113</span>–<span class="lpage" tagx="lpage" title="lpage">24</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15105528">15105528</a></span></span></li> <li tag="ref"><a name="vey021-B75" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tulloch</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">RNA Virus Attenuation by Codon Pair Deoptimisation Is an Artefact of Increases in CpG/UpA Dinucleotide Frequencies</span>’, <span class="source" tagx="source" title="source">eLife</span>, <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e04531</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25490153">25490153</a></span></span></li> <li tag="ref"><a name="vey021-B76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1995</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Comparison of the Genomes of the Wild-Type French Viscerotropic Strain of Yellow Fever Virus with Its Vaccine Derivative French Neurotropic Vaccine</span>’, <span class="source" tagx="source" title="source">The Journal of General Virology</span>, <span class="volume" tagx="volume" title="volume">76</span>: <span class="fpage" tagx="fpage" title="fpage">2749</span>–<span class="lpage" tagx="lpage" title="lpage">55</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7595382">7595382</a></span></span></li> <li tag="ref"><a name="vey021-B77" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watts</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Architecture and Secondary Structure of an Entire HIV-1 RNA Genome</span>’, <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">460</span>: <span class="fpage" tagx="fpage" title="fpage">711</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19661910">19661910</a></span></span></li> <li tag="ref"><a name="vey021-B78" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witteveldt</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Gans</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Enhancement of the Replication of Hepatitis C Virus Replicons of Genotypes 1 to 4 by Manipulation of CpG and UpA Dinucleotide Frequencies and Use of Cell Lines Expressing SECL14L2 for Antiviral Resistance Testing</span>’, <span class="source" tagx="source" title="source">Antimicrobial Agents and Chemotherapy</span>, <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">2981</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26953209">26953209</a></span></span></li> <li tag="ref"><a name="vey021-B79" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zanotto</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1995</span>) ‘<span class="mixed-article-title" title="mixed-article-title">An Arbovirus Cline across the Northern Hemisphere</span>’, <span class="source" tagx="source" title="source">Virology</span>, <span class="volume" tagx="volume" title="volume">210</span>: <span class="fpage" tagx="fpage" title="fpage">152</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7793067">7793067</a></span></span></li> </ul> </div> </div>  </body></html>