<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/05/2020.04.03.024216.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.03.024216v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19" />
<meta name="DC.Identifier" content="10.1101/2020.04.03.024216" />
<meta name="DC.Date" content="2020-04-05" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19." />
<meta name="DC.Contributor" content="Rodrigo Noriega" />
<meta name="DC.Contributor" content="Matthew H. Samore" />
<meta name="article:published_time" content="2020-04-05" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.03.024216v1" />
<meta name="citation_id" content="2020.04.03.024216v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.03.024216" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.03.024216" />
<meta name="citation_author" content="Rodrigo Noriega" />
<meta name="citation_author_institution" content="Department of Chemistry, University of Utah" />
<meta name="citation_author_email" content="noriega@chem.utah.edu" />
<meta name="citation_author" content="Matthew H. Samore" />
<meta name="citation_author_institution" content="Division of Epidemiology, University of Utah" />
<meta name="citation_author_institution" content="Division of Epidemiology, Veterans Affairs Salt Lake City Health Care System" />
<meta name="citation_reference" content="C. Wells, D. Yamin, M. L. Ndeffo-Mbah, N. Wenzel, S. G. Gaffney, J. P. Townsend, L. A. Meyers, M. Fallah, T. G. Nyenswah, F. L. Altice, K. E. Atkins, A. P. Galvani, Harnessing Case Isolation and Ring Vaccination to Control Ebola. PLOS Neglected Tropical Diseases. 9, e0003794 (2015)." />
<meta name="citation_reference" content="G. Chowell, S. Echevarría-Zuno, C. Viboud, L. Simonsen, J. Tamerius, M. A. Miller, V. H. Borja-Aburto, Characterizing the Epidemiology of the 2009 Influenza A/H1N1 Pandemic in Mexico. PLoS Med. 8 (2011), doi:10.1371/journal.pmed.1000436." />
<meta name="citation_reference" content="E. P. Fenichel, C. Castillo-Chavez, M. G. Ceddia, G. Chowell, P. A. G. Parra, G. J. Hickling, G. Holloway, R. Horan, B. Morin, C. Perrings, M. Springborn, L. Velazquez, C. Villalobos, Adaptive human behavior in epidemiological models. PNAS. 108, 6306–6311 (2011)." />
<meta name="citation_reference" content="R. M. May, A. R. McLean, J. Pattison, R. A. Weiss, R. M. Anderson, C. Fraser, A. C. Ghani, C. A. Donnelly, S. Riley, N. M. Ferguson, G. M. Leung, T. H. Lam, A. J. Hedley, Epidemiology, transmission dynamics and control of SARS: the 2002–2003 epidemic. Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences. 359, 1091–1105 (2004)." />
<meta name="citation_reference" content="D. Klinkenberg, C. Fraser, H. Heesterbeek, The Effectiveness of Contact Tracing in Emerging Epidemics. PLOS ONE. 1, e12 (2006)." />
<meta name="citation_reference" content="R. D. Smith, Responding to global infectious disease outbreaks: Lessons from SARS on the role of risk perception, communication and management. Social Science &amp; Medicine. 63, 3113–3123 (2006)." />
<meta name="citation_reference" content="S. Cauchemez, N. M. Ferguson, C. Wachtel, A. Tegnell, G. Saour, B. Duncan, A. Nicoll, Closure of schools during an influenza pandemic. The Lancet Infectious Diseases. 9, 473–481 (2009)." />
<meta name="citation_reference" content="C. del Rio, P. N. Malani, COVID-19—New Insights on a Rapidly Changing Epidemic. JAMA (2020), doi:10.1001/jama.2020.3072." />
<meta name="citation_reference" content="M. Lin, A. Beliavsky, K. Katz, J. E. Powis, W. Ng, V. Williams, M. Science, H. Groves, M. P. Muller, A. Vaisman, S. Hota, J. Johnstone, J. A. Leis, What can early Canadian experience screening for COVID-19 teach us about how to prepare for a pandemic? CMAJ (2020), doi:10.1503/cmaj.200305." />
<meta name="citation_reference" content="Y. Bai, L. Yao, T. Wei, F. Tian, D.-Y. Jin, L. Chen, M. Wang, Presumed Asymptomatic Carrier Transmission of COVID-19. JAMA (2020), doi:10.1001/jama.2020.2565." />
<meta name="citation_reference" content="S-S. Chang, W-H. Hsieh, T-S. Kiu, S-H. Lee, C-H. Wang, H-C. Chou, Y. H. Yeo, C-P. Tseng, C-C. Lee, Multiplex PCR System for Rapid Detection of Pathogens in Patients with Presumed Sepsis –A Systemic Review and Meta-Analysis, PLOS ONE. 8(5), e62323 (2013)." />
<meta name="citation_reference" content="S. C. Kehl, S. Kumar, Utilization of Nucleic Acid Amplification Assays for the Detection of Respiratory Viruses. Clinics in Laboratory Medicine. 29, 661–671 (2009)." />
<meta name="citation_reference" content="J. B. Mahony, Detection of Respiratory Viruses by Molecular Methods. Clinical Microbiology Reviews. 21, 716–747 (2008)." />
<meta name="citation_reference" content="W. Wu, Y.-W. Tang, Emerging Molecular Assays for Detection and Characterization of Respiratory Viruses. Clinics in Laboratory Medicine. 29, 673–693 (2009)." />
<meta name="citation_reference" content="M. J. Espy, J. R. Uhl, L. M. Sloan, S. P. Buckwalter, M. F. Jones, E. A. Vetter, J. D. C. Yao, N. L. Wengenack, J. E. Rosenblatt, F. R. Cockerill, T. F. Smith, Real-Time PCR in Clinical Microbiology: Applications for Routine Laboratory Testing. Clinical Microbiology Reviews. 19, 165–256 (2006)." />
<meta name="citation_reference" content="I. M. Mackay, K. E. Arden, A. Nitsche, Real-time PCR in virology. Nucleic Acids Res. 30, 1292–1305 (2002)." />
<meta name="citation_reference" content="H. Sox, S. Stern, D. Owens, H. L. Abrams, “The use of diagnostic tests: A probabilistic approach” in “Assessment of Diagnostic Technology in Health Care: Rationale, Methods, Problems, and Directions” (National Academies Press (US), 1989." />
<meta name="citation_reference" content="R. S. Ledley, L. B. Lusted, Reasoning Foundations of Medical Diagnosis: Symbolic logic, probability, and value theory aid our understanding of how physicians reason. Science. 130, 9–21 (1959)." />
<meta name="citation_reference" content="H. C. Sox, Diagnostic Decision: Probability Theory in the Use of Diagnostic Tests: An Introduction to Critical Study of the Literature. Ann Intern Med. 104, 60 (1986)." />
<meta name="citation_reference" content="R. Dorfman, The detection of defective numbers of large populations. Annals of Mathematical Statistics 14, 436–440 (1943)." />
<meta name="citation_reference" content="C. Pilcher, S. Fiscus, T. Nguyen, E. Foust, L. Wolf, D. Williams, R. Ashby, J. O’Dowd, J. McPherson, B. Stalzer, L. Hightow, W. Miller, J. Eron, M. Cohen, P. Leone, Detection of acute infections during HIV testing in North Carolina. New England Journal of Medicine 352, 1873–1883 (2005)." />
<meta name="citation_reference" content="C. Lindan, M. Mathur, S. Kumta, H. Jerajani, A. Gogate, J. Schachter, J. Moncada, Utility of pooled urine specimens for detection of Chlamydia trachomatis and Neisseria gonorrhoeae in men attending public sexually transmitted infection clinics in Mumbai, India, by PCR. Journal of Clinical Microbiology 43, 1674–1677 (2005)." />
<meta name="citation_reference" content="T. Van, J. Miller, D. Warshauer, E. Reisdorf, D. Jerrigan D, Humes R, Shult P., Pooling nasopharyngeal/throat swab specimens to increase testing capacity for influenza viruses by PCR. Journal of Clinical Microbiology 50, 891–896 (2012)." />
<meta name="citation_reference" content="M. S. Warasi, J. M. Tebbs, C. S. McMahan,C. R. Bilder, Estimating the prevalence of multiple diseases from two-stage hierarchical pooling. Statist. Med. 35, 3851–3864 (2016)." />
<meta name="citation_reference" content="H.-Y. Kim, M. G. Hudgens, J. M. Dreyfuss, D. J. Westreich, C. D. Pilcher, Comparison of Group Testing Algorithms for Case Identification in the Presence of Test Error. Biometrics 63, 1152–1163 (2007)." />
<meta name="citation_reference" content="S. P. Layne, J. M. Hyman, D. M. Morens, J. K. Taubenberger, New coronavirus outbreak: Framing questions for pandemic prevention. Science Translational Medicine. 12 (2020), doi:10.1126/scitranslmed.abb1469." />
<meta name="citation_reference" content="V. M. Corman, O. Landt, M. Kaiser, R. Molenkamp, A. Meijer, D. K. Chu, T. Bleicker, S. Brünink, J. Schneider, M. L. Schmidt, D. G. Mulders, B. L. Haagmans, B. van der Veer, S. van den Brink, L. Wijsman, G. Goderski, J.-L. Romette, J. Ellis, M. Zambon, M. Peiris, H. Goossens, C. Reusken, M. P. Koopmans, C. Drosten, Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Eurosurveillance. 25, p. 2000045 (2020)." />
<meta name="citation_reference" content="S. M. Samuels, The exact solution to the two-stage group-testing problem. Technometrics 20, 497–500 (1978)." />
<meta name="citation_reference" content="K. Hanson, ARUP COVID-19 rRT-PCR diagnostic testing results. Testing method is the EUA approved Hologic Panther Fusion SARS-COV2 assay: https://www.fda.gov/media/136156/download" />
<meta name="citation_reference" content="W. Wang, Y. Xu, R. Gao, R. Lu, K. Han, G. Wu, W. Tan, Detection of SARS-CoV-2 in Different Types of Clinical Specimens. JAMA (2020), doi:10.1001/jama.2020.3786." />
<meta name="citation_reference" content="J. L. Lewis, V. M. Lockary, S. Kobic, Cost Savings and Increased Efficiency Using a Stratified Specimen Pooling Strategy for Chlamydia trachomatis and Neisseria gonorrhoeae. Sexually Transmitted Diseases 39, 46–48 (2012)." />
<meta name="citation_reference" content="J. Cohen, K. Kupferschmidt, Countries test tactics in ‘war’ against COVID-19. Science. 367, 1287–1288 (2020)." />
<meta name="twitter:title" content="Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19." />
<meta name="og-title" property="og:title" content="Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-05" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.03.024216v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.03.024216v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.03.024216v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1230016" />
    <title>Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"K3NJA64mK7spZwWBT4loBS1NYm3jHU5n_8vDIeBs_Zg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1230016","apath":"\/biorxiv\/early\/2020\/04\/05\/2020.04.03.024216.atom","pisa":"biorxiv;2020.04.03.024216v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.03.024216v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.03.024216v1","title":"Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19","identifier":"node\/1230016"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022elPRe2tYXl0NX86mHmktkDJkHieRODpN0lHWkQNqpEKfnruFwo9iiy6B\\\\\\\/Ov003v8xIYiFnMU0cJFBOzdaJd6ur\\\\\\\/gmR8mP5Ty1Mp1QH9Ftnq0ZqUwTck8+dRQPX5R60Wocm6cJ7drIKXN2G97S5cyv8mehpzPYTP7I5J\\\\\\\/8OByFoUSACi2eGxDjhDJnk\\\\\\\/oMk4DMl5YNy2cbOr\\\\\\\/4y7EGTH8DS7BWej0zrl+ndJmNsYdVSoPu3\\\\\\\/6wSj5WARtD0ZGRGwR2\\\\\\\/WEDUTvQ0J+vl6cdVYV1oiZqL4JnoRL1P8XHP6F\\\\\\\/X\\\\\\\/J4btVuEAjb\\\\\\\/lI\\\\\\\/8XvqTVjdI8rIBrVy1fHtOnvz5Ky5qBLLkzYnVgiR95xcuR3rCc0LvaItBAXqzZq9BQFsRkjiC8OEtsyWs+kWqe5f4Ig6ZLWTQJGDtFMNkTTBBZKgXq45\\\\\\\/jrkC3FlqHNdD+EmkT9kOHRsO10+kF++qKJ51pJgLryKFJK8E0bGCjfrNJOUohatj0O5sDH8bDNz8ZJAx2T+B3KxO+V1MiLn7GRRJzE+KW5bTFO8QA+KvVf3uziLSUbk4MB91blhfIXv9rthjvMbl4TJq\\\\\\\/2Fda6EwMP7s1OQzLxTCAwwVMC0CZMsK\\\\\\\/cPjBRS4I88qRnb8v76e7nbdqeic7E5hwleabgf2f6s8NCX5fQj1iZmQXDORxW\\\\\\\/hBSPWVrf60HhEFwO0N33mABo4HxEmXZ2O7ViYg7xblDgh3+wazoirivSSysN6R43Tp2LXn7cMob2L8VLptrwOTs6SV10V\\\\\\\/XrUROIBgJ9EZsD353GRH+bbG0suL7d7pLOKwF96ZRdsQHUkLKpu\\\\\\\/tcofgb3u5J+keoC7FzHh9SL64v940pNnDfaB\\\\\\\/f1XnMonKPS8bvMfLqH56D96gzNUMwan4hPPQ0Lwya3Z6Gu858Cv9Bx6CVRabH63oQXCmr5HobeJHv9fcag9VGypf33cJ08cbX58aV4G1tC+9q\\\\\\\/LOsS\\\\\\\/MjfnmonC6nhjH2Ayzj\\\\\\\/LKxGrP8fMWUsn49eDxO8zR85XC+bccN4MR4A4g29lQrTU2SxkV0sEkwb\\\\\\\/vL3BHWcCgvlOvBH9zvGStm533cigqEi7hVdZ0wj80oD3M9RshUER3S3SYaXQT+Vb84REbiUXwroK4VvJHrBA9+k2HkuYSj5Iy2yryBRmVeGbAVY75SkM\\\\\\\/ksqa+iqRvBjLRg6izicqX2BJoxs6Ophv3Pp2r7fUP4343J9gj5oww4s10oK3n3XRpuQ7ae9dXq071cto6+6EVVbMPApmAK8Qc4IKC3GyTR\\\\\\\/Y420lrLpi0QQTXY2klPOCvxpvmVOD7+TQHNB7eMqLhAZ3qzi6Gq0hrp143n5MyNK74cSG\\\u0022,\\\u0022iv\\\u0022:\\\u0022CIohgssHQs9DMGtDdeIx4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022dac56510d26991005d4c1b0cd92444fb9ace96c29b54c7bc3d1d4de1780d34e9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.03.024216v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1230016 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.03.024216v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-az2r01H4vK94rjK_q2Q3HwkEiS4K9fpjVesGPzDhC1o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1230016" id="node1230016" data-pisa="biorxiv;2020.04.03.024216v1" data-pisa-master="biorxiv;2020.04.03.024216" data-apath="/biorxiv/early/2020/04/05/2020.04.03.024216.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rodrigo</span> <span class="nlm-surname">Noriega</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Samore</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.03.024216 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1230016" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rodrigo Noriega </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, University of Utah</span>, Salt Lake City, UT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BNoriega%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noriega%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BNoriega%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:noriega@chem.utah.edu" class="" data-icon-position="" data-hide-link-title="0">noriega@chem.utah.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew H. Samore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Epidemiology, University of Utah</span>, Salt Lake City, UT <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Division of Epidemiology, Veterans Affairs Salt Lake City Health Care System</span>, Salt Lake City, Utah</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BH.%2BSamore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samore%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BH.%2BSamore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.03.024216v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1230016" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1230016/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.03.024216v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1230016" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1230016/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.03.024216v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1230016" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1230016/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.03.024216v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1230016" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1230016/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.03.024216v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1230016" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1230016/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19.</p></div><div id="sec-1" class="subsection"><p id="p-3">Emerging infectious diseases pose a global hazard to public health, as exemplified by the COVID-19 pandemic. Key epidemiologic strategies for control of community spread include contact tracing, case isolation, ring containment, and social distancing (<em><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></em>). The use of microbiological testing to identify disease cases is a crucially important element of these strategies. Some countries, including the US, experienced a shortage of kits needed for COVID-19 diagnosis, which resulted in the imposition of restrictive criteria to manage the selection of patients for testing. Constraints in the supply of kits had a particularly significant impact on testing of mildly symptomatic individuals, as well as asymptomatic contacts of confirmed cases. For some facilities that have been overwhelmed by demand for testing as the pandemic progressed, test throughput continues to be a limiting factor (<em><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></em>). Strategies for screening more individuals with a reduced burden on resources are highly desirable. Using a Bayesian formalism, a hierarchical testing protocol based on sample pooling is discussed. Anticipated benefits include easing the demand of constrained resources and enabling more efficient detection of a larger number of cases.</p><p id="p-4">Molecular assays are the predominant testing method for viral and bacterial pathogens (<em><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></em>). Specifically, nucleic acid detection assays typically employ real-time polymerase chain reaction (RT-PCR) for DNA targets and reverse-transcription real-time PCR (rRT-PCT) for RNA targets (<em><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></em>). The popularity of such testing platforms is due to <strong>1)</strong> their high sensitivity and specificity, <strong>2)</strong> the widespread access to sequencing and synthesis technologies for the identification of nucleic acid target sequences and probes, and <strong>3)</strong> the development of fast, user-friendly, and cost-effective equipment. While nucleic acid assays have powered a revolution in diagnostics and delivery of care for individual patients, their application in large-scale infectious disease surveillance is hampered partly by low throughput at a population level.</p><p id="p-5">The information content of a diagnostic test can be evaluated with a Bayesian probability formalism in the context of an individual sample or for repeated sampling from the same patient (<em><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></em>) by taking into account the probability of detecting a positive case (assay sensitivity, or identification rate <em>P</em><sub><em>id</em></sub>) and the probability of a positive result from healthy samples (false positive rate <em>P</em><sub><em>fp</em></sub>). Bayesian inference requires the assessment of a “prior” probability to the presence of disease in a sample, <em>P</em>(<em>D</em>), which is updated to a “posterior” probability given a positive or negative test result with conditional probabilities <em>P</em>(<em>D</em>|+) or <em>P</em>(<em>D</em>|−) (<strong><a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Eq. 1.a-b</a>)</strong>.
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-1.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-2.gif"/></noscript></span>

</span>
where <em>P</em>(+) and <em>P</em>(−) are the overall probabilities of the test yielding a positive or negative result, respectively. These tools can be extended to the somewhat counterintuitive situation where a diagnostic test is conducted on a sample pooled from multiple individuals. The motivation for sample pooling is to screen multiple patients simultaneously and reduce the burden on testing facilities working with limited resources. Pooling schemes have been developed since their introduction in the 1940s for syphilis testing, and have been applied to screen for and estimate prevalence rates of a variety of diseases (<em><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>-<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></em>). Here, a simple two-step hierarchical protocol first introduced by Dorfman is considered: <em>Samples from N</em><sub><em>p</em></sub> <em>patients are collected and randomly pooled into groups of n individual samples each. Pooled samples are interrogated with the diagnostic test. If pooled testing yields a negative result, no further testing is conducted. If pooled testing yields a positive result, all patients in that pool are tested individually.</em></p><p id="p-6">While statistical approaches have been focused on characterizing the performance of various pooling schemes, not all of them include non-ideal test parameters (<em><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></em>). Moreover, testing characteristics at the pooled- and individual-sample levels can be different. Here, the Bayesian inference approach in <strong><a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Eq. 1.a-b</a></strong> is modified to include differences in the assay sensitivity and overall probability of a positive result in pooled vs. individual tests (<strong><a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Eq. S1</a>-<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">2</a></strong>). Sensitivity loss is included as a reduction in the identification rate of pooled-sample tests by a scaling factor <em>γ</em>. Due to the exceptional specificities of nucleic acid assays, the false positive rate is assumed to remain unaffected. Importantly, the posterior probability assessed from a positive pooled test is used as a prior for follow-up individual tests, which yields additional information and enhances the Bayesian inference assessment of those cases (<strong><a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Eq. S3</a></strong>).</p><p id="p-7">For a population of <em>N</em><sub><em>p</em></sub> patients divided into sample pools of size <em>n</em>, the average number of tests is the number of initial pooled tests plus the expected number of follow-up tests (<strong><a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Eq. S4</a></strong>). Throughput increase <em>χ</em> is expressed as the effective number of individuals screened by each diagnostic test (<strong><a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Eq. S5</a></strong>). The individual- and pooled-sample test characteristics determine the pool size that optimizes screening throughput as a function of average disease prevalence in the tested population.</p><p id="p-8">The advantages of pooled-sample screening are discussed in the context of the rapidly evolving COVID-19 pandemic (caused by the novel coronavirus SARS-CoV-2) (<em><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></em>). A recent rRT-PCR assay for COVID-19 reports an identification rate of 95% and no false positives after testing 310 samples including other respiratory pathogens (<em><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></em>). Given reported specificities of commercially-available respiratory panel assays (&gt;99%), an estimated 1% false positive rate was included in the model results reported here. A moderate reduction in the identification rate for a pooled sample (<em>γ</em> = 0.9) was assumed – this variable is discussed in detail below. Consistent with similar implementations of Dorfman-type testing algorithms (<em><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></em>), substantial increases in testing throughput are predicted for low disease prevalence rates (<em>P</em>(<em>D</em>) ≤8%, <strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a></strong>), where throughput more than doubles and optimal pool sizes are 4 ≤ <em>n</em> ≤ 12. At intermediate prevalence rates 0.1 ≤ <em>P</em>(<em>D</em>) ≤ 0.2, the increase in throughput is moderate yet substantial (&gt;30% increase in throughput) and pool sizes are small (<em>n</em> = 3). For high prevalence rates, pooling yields no improvement (optimal pool size <em>n</em> = 1). Average disease prevalence can be re-assessed as information is gained for the tested population to re-optimize pool size. Dynamic self-tuning is a feature of Bayesian inference, a significant asset when a close feedback loop is desirable.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of patients screened by each diagnostic test (&#x3C7;, blue circles) and the relative increase in detection-to-miss ratio for a pooled scheme (&#x394;, black squares), as a function of average disease prevalence. Gray line is the no-pooling scheme reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-641536546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of patients screened by each diagnostic test (&#x3C7;, blue circles) and the relative increase in detection-to-miss ratio for a pooled scheme (&#x394;, black squares), as a function of average disease prevalence. Gray line is the no-pooling scheme reference.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F1.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F1.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-9" class="first-child">Number of patients screened by each diagnostic test (χ, blue circles) and the relative increase in detection-to-miss ratio for a pooled scheme (Δ, black squares), as a function of average disease prevalence. Gray line is the no-pooling scheme reference.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Besides testing throughput, it is informative to assess the increased ability to detect cases using a pooled-sample vs individual-sample scheme, which can be accomplished by comparing the ratio of detected to missed cases in each protocol. Importantly, resource constraints are incorporated into this comparison by accounting for unscreened cases in the standard 1:1 scheme (<strong><a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Eq. S6</a>-<a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">8</a></strong>). The relative increase in the detection-to-miss ratio between the pooled and standard 1:1 schemes exhibits even more significant gains than those observed for testing throughput (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a></strong>).</p><p id="p-11">Loss in screening power associated with sample pooling is assessed by <strong>1)</strong> estimating the portion of cases that would have been identified in an individual test but missed by pooled screening, and <strong>2)</strong> determining the information gained for patients whose pooled screening result was negative.</p><p id="p-12">A reduction in overall pathogen concentration due to pooling in conditions of low disease prevalence can decrease the test’s identification rate, although it is manageable when targeting infectious diseases for which typical pathogen concentrations are non-negligible (<em><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></em>). This concern is examined with a set of 186 positive rRT-PCR diagnostic test results for COVID-19 (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a></strong>), which include nasopharyngeal swab, oropharyngeal swab, and bronchoalveolar lavage (<em><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></em>). Tests have a mean cycle threshold value of ⟨<em>C</em><sub><em>t</em></sub>⟩ = 24.6, and a positive test result is defined as a rRT-PCR reaction with <em>C</em><sub><em>t</em></sub> ≤ 42 – in agreement with early reports for nasal swab COVID-19 rRT-PCR tests with ⟨<em>C</em><sub><em>t</em></sub>⟩ = 24.3 and viral loads of 1.4 × 10<sup>6</sup> copies/mL (<em><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></em>). From these data, the portion of samples that would have had a positive test result even if pooled with an entirely healthy population is estimated using a pool size <em>n</em> = 12 and rRT-PCR geometric efficiencies of ϵ = 1.7 − 2 per cycle. A pooled screening protocol would have detected 95.7-96.8% of these cases (178-180 out of 186). Further support for moderate sensitivity loss can be achieved by dividing the distribution of <em>C</em><sub><em>t</em></sub> values for the test-positive samples into three subpopulations. For the same pooling and rRT-PCR efficiencies stated previously, &gt;95% of the broadest, lowest-load population would be detected.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cycle threshold values for rRT-PCR reactions for confirmed COVID-19 cases (27), described by subpopulations with low, medium, and high viral load. Sensitivity cutoffs shown as vertical lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-641536546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cycle threshold values for rRT-PCR reactions for confirmed COVID-19 cases (&lt;em&gt;27&lt;/em&gt;), described by subpopulations with low, medium, and high viral load. Sensitivity cutoffs shown as vertical lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F2.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F2.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-13" class="first-child">Cycle threshold values for rRT-PCR reactions for confirmed COVID-19 cases (<em><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></em>), described by subpopulations with low, medium, and high viral load. Sensitivity cutoffs shown as vertical lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">The main benefit of hierarchical pooled-sample testing protocols is the ability to screen a larger portion of the population and detect more positive cases. The relative increase in case detection is given by the ratio of the number of cases detected in a pooled setting and the number of cases detected if the same number of tests were used on single patients – the multiplicative factor in testing throughput scaled by the loss in sensitivity, <em>χ</em> · <em>γ</em>. Loss in sensitivity also leads to more screened-yet-missed cases – even in the absence of pooling, diagnostic tests yield false negatives and increasing those odds should not be considered lightly. In the range of disease prevalence with moderate to high increase in throughput and compared to individual testing of every patient, a <em>γ</em> = 0.9 sensitivity loss leads to a relatively low increase in posterior probability of disease after a negative test outcome, <em>P</em>(<em>D</em>|−) (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a></strong>). While sensitivity losses decrease the ability to confidently screen healthy individuals, the threshold for this tradeoff depends on the situation where the screening protocol is deployed. Effective risk communication to screened individuals is needed to prevent an outsized sense of security after a negative pooled-test result – e.g., vigilance to symptom development triggers individual testing.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability of an individual being infected even though their pooled-sample test gave a negative result, as a function of background disease prevalence and for different values of sensitivity loss." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-641536546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Probability of an individual being infected even though their pooled-sample test gave a negative result, as a function of background disease prevalence and for different values of sensitivity loss.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F3.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F3.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.024216/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-15" class="first-child">Probability of an individual being infected even though their pooled-sample test gave a negative result, as a function of background disease prevalence and for different values of sensitivity loss.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">This protocol is amenable to HIPAA regulations – in fact, pooling has been implemented by state laboratories in the recent past (<em><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></em>)– and requires limited additional sample processing. Refining the dependence of sensitivity loss with pool size and coupling to modeling strategies that inform population sampling and prior probabilities will provide further screening improvements. From an epidemiological surveillance standpoint, increased detection of positive cases in a larger portion of the population denotes that a greater fraction of infectious individuals can be isolated. However, pooled testing in less useful in an inpatient clinical setting where the highest sensitivity is needed to minimize risk of hospital transmission from non-isolated patients. As with any change in the delivery of medical care, a discussion including community stakeholders is paramount.</p><p id="p-17">In summary, a pooled testing strategy has the potential to enhance comprehensive surveillance of SARS-CoV-2 particularly when test kits are in short supply. The benefits of surveillance are greatest in the early phases of community spread. Thus, improving the capacity for high-throughput testing has the highest impact when prevalence is low enough that pooled sampling is most beneficial. The ratio of confirmed COVID-19 cases to tests performed varies by country, but it appears that aggressive testing strategies yield a low enough prevalence to benefit from pooled-sample screening – e.g., South Korea’s is 3% (8.3k/270k as of 3/16/2020) (<em><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></em>). While the development of clinical prediction rules and non-testing screening are critical to any epidemiological response, dealing with a novel disease for which data is still sparse and testing capabilities are limited means that maximizing the impact of each individual test can benefit the continued refinement of our strategy.</p></div><div class="section" id="sec-2"><h2 class="">Funding</h2><p id="p-19">none;</p></div><div class="section" id="sec-3"><h2 class="">Author contributions</h2><p id="p-20">R.N. conceived the idea, performed the analysis, prepared the figures, and drafted the manuscript. R.N. and M.H.S. developed the idea and edited the manuscript;</p></div><div class="section" id="sec-4"><h2 class="">Competing interests</h2><p id="p-21">Authors declare no competing interests;</p></div><div class="section" id="sec-5"><h2 class="">Data and materials availability</h2><p id="p-22">All data is available in the main text or the supplementary materials.</p></div><div class="section" id="sec-6"><h2 class="">Materials and Methods</h2><div class="section" id="sec-7"><h3>Bayesian inference implementation</h3><p id="p-23">The process for assessing the posterior probabilities for individuals whose pooled test yielded a positive or negative result (<strong><a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Eq. S1.a-b</a></strong>)
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-6.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-7.gif"/></noscript></span>

</span>
includes the a reduction in the identification rate of the pooled-sample diagnostic test by a factor <em>γ</em> compared to an individual test, so that <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-1.gif"/></noscript></span></span> One must also consider the probability of a positive result in a pooled sample. Assuming that every individual has a prior probability of being infected equal to some background average <em>P</em>(<em>D</em>), the probability of a positive test result in a pool of size <em>n</em> is the probability of having a nonzero number of positive individual samples times the pooled-test identification rate plus the probability of a completely-healthy sample pool yielding a false positive (<strong><a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Eq. S2</a></strong>):
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-8.gif"/></noscript></span>

</span>
</p><p id="p-24">The posterior probability <em>P</em>(<em>D</em>|+) from the pooled test with a positive outcome can be used as a prior for the follow-up individual test Bayesian inference (<strong><a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Eq. S3.a-b</a></strong>).
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-9.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-10.gif"/></noscript></span>

</span>
where the overall probability of an individual’s test being positive includes the probability of correctly identifying a positive sample <em>P</em>(<em>D</em>|+) · <em>P</em><sub><em>id</em></sub> plus the probability of a negative sample yielding a false positive (1 − <em>P</em>(<em>D</em>|+)) · <em>P</em><sub><em>fp</em></sub>.</p><p id="p-25">For a population of <em>N</em><sub><em>p</em></sub> patients divided into sample pools of size <em>n</em>, the average number of tests ⟨<em>N</em><sub><em>test</em></sub>⟩ is given by the number of initial pooled tests plus the expected number of follow-up tests (<strong><a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">Eq. S4</a></strong>):
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-11.gif"/></noscript></span>

</span>
</p><p id="p-26">The throughput increase is expressed as a multiplicative factor representing how many individuals were screened by the use of each diagnostic test (<strong><a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">Eq. S5</a></strong>).
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-12.gif"/></noscript></span>

</span>
</p><div class="section" id="sec-8"><h4>Full vs. partial optimization</h4><p id="p-27">In a fully optimized scheme a pooled-sample’s positive result triggers a cascade of smaller-sized pools, minimizing the number of tests performed. However, an intermediate improvement is chosen due to possible adverse outcomes of a lengthier process with several pooling and Bayesian inference steps – e.g., possible delay of necessary care, increased exposure to infected individuals.</p></div></div><div class="section" id="sec-9"><h3>Advantages in testing throughput and case detection</h3><p id="p-28">We can estimate the ratio of detected-to-missed (including unscreened) cases for each protocol with the following simplified analysis:</p><p id="p-29"><span class="underline"><strong>In the pooled scheme</strong></span>, the number of detected cases are the total number of screened individuals with the disease ⟨<em>N</em><sub><em>test</em></sub>⟩<em>χP</em>(<em>D</em>) multiplied by the effective detection probability <em>γ</em> · <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-2.gif"/></noscript></span></span>, which includes the sequential detection probability at the pooled stage and the follow-up individual stage. The number of missed cases would thus be the difference between detected and total cases, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-3.gif"/></noscript></span></span> The number of false positives is the number of individuals in all-healthy pools yielding a false positive and whose individual test is also a false positive – <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-4.gif"/></noscript></span></span> – plus the healthy individuals pooled with a diseased sample that is detected at the pooled stage and whose follow up test is a false positive, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-5.gif"/></noscript></span></span> The detection-to-miss ratio <em>θ</em><sub><em>pool</em></sub> can be expressed as
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-13.gif"/></noscript></span>

</span>
</p><p id="p-30"><span class="underline"><strong>In a standard 1:1 scheme</strong></span>, the number of detected cases are the number of tested individuals with the disease ⟨<em>N</em><sub><em>test</em></sub>⟩<em>P</em>(<em>D</em>) times the detection efficiency of the individual test <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-6.gif"/></noscript></span></span> The number of missed cases is thus <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-7.gif"/></noscript></span></span> The number of false positives is ⟨<em>N</em><sub><em>test</em></sub>⟩[1 − <em>P</em>(<em>D</em>)]<em>P</em><sub><em>fp</em></sub>. The number of unscreened cases that carry the disease is ⟨<em>N</em><sub><em>test</em></sub>⟩(<em>χ</em> − 1) <em>P</em>(<em>D</em>). The detection-to-miss ratio – considering the population sampled in the pooled scheme – is given by
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-14.gif"/></noscript></span>

</span>
</p><p id="p-31">The relative increase in the detection-to-miss ratio between the pooled and standard 1:1 schemes is thus given by
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/graphic-15.gif"/></noscript></span>

</span>
</p><p id="p-32">To further illustrate this, consider a disease including asymptomatic-yet-infectious individuals and for which clinical predictions are at an early stage, preventing effective triage against conditions presenting similar symptoms. A screening point must decide how to use limited resources (e.g., 4,000 available tests) to detect the maximum number of cases in a population at 5% risk. Using the test characteristics described in the main text <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-8.gif"/></noscript></span></span>, it is possible to compare the pooled-sample screening protocol vs. standard 1:1: testing. On average, a pooled-sample approach allows testing 10,000 individuals, detecting 406 cases while missing 94 and yielding 18 false positives; conversely, a standard 1:1 approach would test 4,000 individuals and detect 190 cases, miss 10, yield 38 false positives, and leave 300 positive cases untested and thus undetected.</p><p id="p-33">What if the rRT-PCR test sensitivity is much lower than expected – say <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.03.024216/embed/inline-graphic-9.gif"/></noscript></span></span> Let us also consider a much more pessimistic estimate for sensitivity loss, <em>γ</em> = 0.8 and no change in the false positive rate (<em>P</em><sub><em>fp</em></sub> = 0.01). For the same 5% prevalence in the population, the screening point with access to 4,000 tests would screen 12,550 individuals (627 of which are infected) using a pooling scheme (for these new parameters, optimal pool size is <em>n</em> = 7). It would detect 246 and miss 381. A 1:1 testing scheme would identify 140 cases, miss 60, and leave 427 untested. Not surprisingly, the number of false negatives increases substantially <span class="underline">in both scenarios</span> due to the lower starting point for the test’s sensitivity. However, in a pooled scheme 381 infected individuals are still at risk of spreading the disease in the community, while in the 1:1 scheme 487 infected individuals remain at risk of further community spread. As mentioned in the main text, effective risk communication is a critical component of any large-scale screening effort with imperfect tests. Symptomatic individuals should be considered at increased risk even after a negative test result, and other diagnostic avenues could be used (e.g., chest CT).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-18">Valuable discussions with Dr. Kimberly Hanson and Dr. Lindsay T. Keegan.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References and Notes</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wells</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Yamin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Ndeffo-Mbah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Wenzel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Gaffney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Townsend</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Meyers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fallah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Nyenswah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. L.</span> <span class="cit-name-surname">Altice</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Atkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Galvani</span></span>, <span class="cit-article-title">Harnessing Case Isolation and Ring Vaccination to Control Ebola</span>. <abbr class="cit-jnl-abbrev">PLOS Neglected Tropical Diseases</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">e0003794</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D9%26rft.spage%253D0003794e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.2" data-doi="10.1371/journal.pmed.1000436"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chowell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Echevarría-Zuno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Viboud</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Simonsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tamerius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. H.</span> <span class="cit-name-surname">Borja-Aburto</span></span>, <span class="cit-article-title">Characterizing the Epidemiology of the 2009 Influenza A/H1N1 Pandemic in Mexico</span>. <abbr class="cit-jnl-abbrev">PLoS Med</abbr>. <span class="cit-vol">8</span> (<span class="cit-pub-date">2011</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pmed.1000436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BMed%26rft.aulast%253DChowell%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1000436%26rft.epage%253De1000436%26rft.atitle%253DCharacterizing%2Bthe%2Bepidemiology%2Bof%2Bthe%2B2009%2Binfluenza%2BA%252FH1N1%2Bpandemic%2Bin%2BMexico.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pmed.1000436%26rft_id%253Dinfo%253Apmid%252F21629683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pmed.1000436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21629683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.3" data-doi="10.1073/pnas.1011250108"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Fenichel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Castillo-Chavez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Ceddia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chowell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A. G.</span> <span class="cit-name-surname">Parra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Hickling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Holloway</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Horan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Morin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Perrings</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Springborn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Velazquez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Villalobos</span></span>, <span class="cit-article-title">Adaptive human behavior in epidemiological models</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>. <span class="cit-vol">108</span>, <span class="cit-fpage">6306</span>–<span class="cit-lpage">6311</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011250108%26rft_id%253Dinfo%253Apmid%252F21444809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE1LzYzMDYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAzLjAyNDIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.4" data-doi="10.1098/rstb.2004.1490"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">May</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">McLean</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pattison</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Weiss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fraser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Ghani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Donnelly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Riley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Ferguson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Leung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Lam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Hedley</span></span>, <span class="cit-article-title">Epidemiology, transmission dynamics and control of SARS: the 2002–2003 epidemic</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences</abbr>. <span class="cit-vol">359</span>, <span class="cit-fpage">1091</span>–<span class="cit-lpage">1105</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DAnderson%26rft.auinit1%253DR.%2BM.%26rft.volume%253D359%26rft.issue%253D1447%26rft.spage%253D1091%26rft.epage%253D1105%26rft.atitle%253DEpidemiology%252C%2Btransmission%2Bdynamics%2Band%2Bcontrol%2Bof%2BSARS%253A%2Bthe%2B2002-2003%2Bepidemic%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2004.1490%26rft_id%253Dinfo%253Apmid%252F15306395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2004.1490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15306395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222688200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.5" data-doi="10.1371/journal.pone.0000012"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Klinkenberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fraser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Heesterbeek</span></span>, <span class="cit-article-title">The Effectiveness of Contact Tracing in Emerging Epidemics</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>. <span class="cit-vol">1</span>, <span class="cit-fpage">e12</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKlinkenberg%26rft.auinit1%253DD.%26rft.volume%253D1%26rft.spage%253De12%26rft.epage%253De12%26rft.atitle%253DThe%2Beffectiveness%2Bof%2Bcontact%2Btracing%2Bin%2Bemerging%2Bepidemics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000012%26rft_id%253Dinfo%253Apmid%252F17183638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17183638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.6" data-doi="10.1016/j.socscimed.2006.08.004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-article-title">Responding to global infectious disease outbreaks: Lessons from SARS on the role of risk perception, communication and management</span>. <abbr class="cit-jnl-abbrev">Social Science &amp; Medicine</abbr>. <span class="cit-vol">63</span>, <span class="cit-fpage">3113</span>–<span class="cit-lpage">3123</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSocial%2BScience%2B%2526%2BMedicine%26rft.volume%253D63%26rft.spage%253D3113%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.socscimed.2006.08.004%26rft_id%253Dinfo%253Apmid%252F16978751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.socscimed.2006.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16978751&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242467000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.7" data-doi="10.1016/S1473-3099(09)70176-8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cauchemez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Ferguson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wachtel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tegnell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Saour</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Duncan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nicoll</span></span>, <span class="cit-article-title">Closure of schools during an influenza pandemic</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2Binfectious%2Bdiseases%26rft.stitle%253DLancet%2BInfect%2BDis%26rft.aulast%253DCauchemez%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DClosure%2Bof%2Bschools%2Bduring%2Ban%2Binfluenza%2Bpandemic.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252809%252970176-8%26rft_id%253Dinfo%253Apmid%252F19628172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(09)70176-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19628172&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268625400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.8" data-doi="10.1001/jama.2020.3072"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">del Rio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. N.</span> <span class="cit-name-surname">Malani</span></span>, <span class="cit-article-title">COVID-19—New Insights on a Rapidly Changing Epidemic</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.3072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.3072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2020.3072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.9" data-doi="10.1503/cmaj.200305"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beliavsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Katz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Powis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Williams</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Science</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Groves</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Muller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vaisman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hota</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Johnstone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Leis</span></span>, <span class="cit-article-title">What can early Canadian experience screening for COVID-19 teach us about how to prepare for a pandemic?</span> <abbr class="cit-jnl-abbrev">CMAJ</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1503/cmaj.200305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCMAJ%26rft_id%253Dinfo%253Adoi%252F10.1503%252Fcmaj.200305%26rft_id%253Dinfo%253Apmid%252F32144097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiY21haiI7czo1OiJyZXNpZCI7czoxMToiMTkyLzEyL0UzMTQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAzLjAyNDIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.10" data-doi="10.1001/jama.2020.2565"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Bai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.-Y.</span> <span class="cit-name-surname">Jin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-article-title">Presumed Asymptomatic Carrier Transmission of COVID-19</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.2565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.2565%26rft_id%253Dinfo%253Apmid%252F32083643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2020.2565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32083643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.11" data-doi="10.1371/journal.pone.0062323"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S-S.</span> <span class="cit-name-surname">Chang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W-H.</span> <span class="cit-name-surname">Hsieh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T-S.</span> <span class="cit-name-surname">Kiu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S-H.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C-H.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H-C.</span> <span class="cit-name-surname">Chou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. H.</span> <span class="cit-name-surname">Yeo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C-P.</span> <span class="cit-name-surname">Tseng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C-C.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-article-title">Multiplex PCR System for Rapid Detection of Pathogens in Patients with Presumed Sepsis –A Systemic Review and Meta-Analysis</span>, <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>. <span class="cit-vol">8</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">e62323</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D8%26rft.spage%253De62323%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062323%26rft_id%253Dinfo%253Apmid%252F23734173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23734173&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.12" data-doi="10.1016/j.cll.2009.07.008"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Kehl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-article-title">Utilization of Nucleic Acid Amplification Assays for the Detection of Respiratory Viruses</span>. <abbr class="cit-jnl-abbrev">Clinics in Laboratory Medicine</abbr>. <span class="cit-vol">29</span>, <span class="cit-fpage">661</span>–<span class="cit-lpage">671</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinics%2Bin%2Blaboratory%2Bmedicine%26rft.stitle%253DClin%2BLab%2BMed%26rft.aulast%253DKehl%26rft.auinit1%253DS.%2BC.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D661%26rft.epage%253D671%26rft.atitle%253DUtilization%2Bof%2Bnucleic%2Bacid%2Bamplification%2Bassays%2Bfor%2Bthe%2Bdetection%2Bof%2Brespiratory%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cll.2009.07.008%26rft_id%253Dinfo%253Apmid%252F19892227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cll.2009.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19892227&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.13" data-doi="10.1128/CMR.00037-07"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Mahony</span></span>, <span class="cit-article-title">Detection of Respiratory Viruses by Molecular Methods</span>. <abbr class="cit-jnl-abbrev">Clinical Microbiology Reviews</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">716</span>–<span class="cit-lpage">747</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DMahony%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D716%26rft.epage%253D747%26rft.atitle%253DDetection%2Bof%2BRespiratory%2BViruses%2Bby%2BMolecular%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00037-07%26rft_id%253Dinfo%253Apmid%252F18854489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNzE2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMy4wMjQyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.14" data-doi="10.1016/j.cll.2009.07.005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-W.</span> <span class="cit-name-surname">Tang</span></span>, <span class="cit-article-title">Emerging Molecular Assays for Detection and Characterization of Respiratory Viruses</span>. <abbr class="cit-jnl-abbrev">Clinics in Laboratory Medicine</abbr>. <span class="cit-vol">29</span>, <span class="cit-fpage">673</span>–<span class="cit-lpage">693</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinics%2Bin%2Blaboratory%2Bmedicine%26rft.stitle%253DClin%2BLab%2BMed%26rft.aulast%253DWu%26rft.auinit1%253DW.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D673%26rft.epage%253D693%26rft.atitle%253DEmerging%2Bmolecular%2Bassays%2Bfor%2Bdetection%2Band%2Bcharacterization%2Bof%2Brespiratory%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cll.2009.07.005%26rft_id%253Dinfo%253Apmid%252F19892228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cll.2009.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19892228&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272162100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.15" data-doi="10.1128/CMR.19.1.165-256.2006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Espy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Uhl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Sloan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Buckwalter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Vetter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D. C.</span> <span class="cit-name-surname">Yao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Wengenack</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Rosenblatt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. R.</span> <span class="cit-name-surname">Cockerill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-article-title">Real-Time PCR in Clinical Microbiology: Applications for Routine Laboratory Testing</span>. <abbr class="cit-jnl-abbrev">Clinical Microbiology Reviews</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">165</span>–<span class="cit-lpage">256</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DEspy%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D256%26rft.atitle%253DReal-Time%2BPCR%2Bin%2BClinical%2BMicrobiology%253A%2BApplications%2Bfor%2BRoutine%2BLaboratory%2BTesting%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.19.1.165-256.2006%26rft_id%253Dinfo%253Apmid%252F16418529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTY1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMy4wMjQyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.16" data-doi="10.1093/nar/30.6.1292"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Mackay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Arden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nitsche</span></span>, <span class="cit-article-title">Real-time PCR in virology</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">30</span>, <span class="cit-fpage">1292</span>–<span class="cit-lpage">1305</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMackay%26rft.auinit1%253DI.%2BM.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253D1292%26rft.epage%253D1305%26rft.atitle%253DReal-time%2BPCR%2Bin%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.6.1292%26rft_id%253Dinfo%253Apmid%252F11884626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.6.1292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11884626&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174467900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.03.024216v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sox</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Stern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Owens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. L.</span> <span class="cit-name-surname">Abrams</span></span>, <span class="cit-chapter-title">“The use of diagnostic tests: A probabilistic approach” in</span> <span class="cit-source">“Assessment of Diagnostic Technology in Health Care: Rationale, Methods, Problems, and Directions”</span> (<span class="cit-publ-name">National Academies Press (US</span>), <span class="cit-pub-date">1989</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.18" data-doi="10.1126/science.130.3366.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Ledley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Lusted</span></span>, <span class="cit-article-title">Reasoning Foundations of Medical Diagnosis: Symbolic logic, probability, and value theory aid our understanding of how physicians reason</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">130</span>, <span class="cit-fpage">9</span>–<span class="cit-lpage">21</span> (<span class="cit-pub-date">1959</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLedley%26rft.auinit1%253DR.%2BS.%26rft.volume%253D130%26rft.issue%253D3366%26rft.spage%253D9%26rft.epage%253D21%26rft.atitle%253DReasoning%2BFoundations%2Bof%2BMedical%2BDiagnosis%253A%2BSymbolic%2Blogic%252C%2Bprobability%252C%2Band%2Bvalue%2Btheory%2Baid%2Bour%2Bunderstanding%2Bof%2Bhow%2Bphysicians%2Breason%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.130.3366.9%26rft_id%253Dinfo%253Apmid%252F13668531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTA6IjEzMC8zMzY2LzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAzLjAyNDIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.19" data-doi="10.7326/0003-4819-104-1-60"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">Sox</span></span>, <span class="cit-article-title">Diagnostic Decision: Probability Theory in the Use of Diagnostic Tests: An Introduction to Critical Study of the Literature</span>. <abbr class="cit-jnl-abbrev">Ann Intern Med</abbr>. <span class="cit-vol">104</span>, <span class="cit-fpage">60</span> (<span class="cit-pub-date">1986</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BInternal%2BMedicine%26rft.stitle%253DANN%2BINTERN%2BMED%26rft.aulast%253DSOX%26rft.auinit1%253DH.%2BC.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D60%26rft.epage%253D66%26rft.atitle%253DDiagnostic%2BDecision%253A%2BProbability%2BTheory%2Bin%2Bthe%2BUse%2Bof%2BDiagnostic%2BTests%253A%2BAn%2BIntroduction%2Bto%2BCritical%2BStudy%2Bof%2Bthe%2BLiterature%26rft_id%253Dinfo%253Adoi%252F10.7326%252F0003-4819-104-1-60%26rft_id%253Dinfo%253Apmid%252F3079637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7326/0003-4819-104-1-60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3079637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986AXL4300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.20" data-doi="10.1214/aoms/1177731363"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Dorfman</span></span>, <span class="cit-article-title">The detection of defective numbers of large populations</span>. <abbr class="cit-jnl-abbrev">Annals of Mathematical Statistics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">436</span>–<span class="cit-lpage">440</span> (<span class="cit-pub-date">1943</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BMathematical%2BStatistics%26rft.volume%253D14%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177731363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177731363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.21" data-doi="10.1056/NEJMoa042291"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pilcher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fiscus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Foust</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wolf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Williams</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ashby</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">O’Dowd</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McPherson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Stalzer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hightow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Eron</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Leone</span></span>, <span class="cit-article-title">Detection of acute infections during HIV testing in North Carolina</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> <span class="cit-vol">352</span>, <span class="cit-fpage">1873</span>–<span class="cit-lpage">1883</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DPilcher%26rft.auinit1%253DC.%2BD.%26rft.volume%253D352%26rft.issue%253D18%26rft.spage%253D1873%26rft.epage%253D1883%26rft.atitle%253DDetection%2Bof%2BAcute%2BInfections%2Bduring%2BHIV%2BTesting%2Bin%2BNorth%2BCarolina%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa042291%26rft_id%253Dinfo%253Apmid%252F15872202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa042291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15872202&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228858600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.03.024216v1.22" data-doi="10.1128/JCM.43.4.1674-1677.2005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lindan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mathur</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jerajani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gogate</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schachter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Moncada</span></span>, <span class="cit-article-title">Utility of pooled urine specimens for detection of Chlamydia trachomatis and Neisseria gonorrhoeae in men attending public sexually transmitted infection clinics in Mumbai, India, by PCR</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">1674</span>–<span class="cit-lpage">1677</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DLindan%26rft.auinit1%253DC.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D1674%26rft.epage%253D1677%26rft.atitle%253DUtility%2Bof%2BPooled%2BUrine%2BSpecimens%2Bfor%2BDetection%2Bof%2BChlamydia%2Btrachomatis%2Band%2BNeisseria%2Bgonorrhoeae%2Bin%2BMen%2BAttending%2BPublic%2BSexually%2BTransmitted%2BInfection%2BClinics%2Bin%2BMumbai%252C%2BIndia%252C%2Bby%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.43.4.1674-1677.2005%26rft_id%253Dinfo%253Apmid%252F15814983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQzLzQvMTY3NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDMuMDI0MjE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.23" data-doi="10.1128/JCM.05631-11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Van</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Warshauer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Reisdorf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jerrigan</span></span> D, <span class="cit-auth"><span class="cit-name-surname">Humes</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shult</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-article-title">Pooling nasopharyngeal/throat swab specimens to increase testing capacity for influenza viruses by PCR</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr> <span class="cit-vol">50</span>, <span class="cit-fpage">891</span>–<span class="cit-lpage">896</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.05631-11%26rft_id%253Dinfo%253Apmid%252F22205820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjUwLzMvODkxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMy4wMjQyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Warasi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Tebbs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">McMahan</span></span>,<span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Bilder</span></span>, <span class="cit-article-title">Estimating the prevalence of multiple diseases from two-stage hierarchical pooling</span>. <abbr class="cit-jnl-abbrev">Statist. Med</abbr>. <span class="cit-vol">35</span>, <span class="cit-fpage">3851</span>–<span class="cit-lpage">3864</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatist.%2BMed%26rft.volume%253D35%26rft.spage%253D3851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.25" data-doi="10.1111/j.1541-0420.2007.00817.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.-Y.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Hudgens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Dreyfuss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Westreich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Pilcher</span></span>, <span class="cit-article-title">Comparison of Group Testing Algorithms for Case Identification in the Presence of Test Error</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">1152</span>–<span class="cit-lpage">1163</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DKim%26rft.auinit1%253DH.%2BY.%26rft.volume%253D63%26rft.issue%253D4%26rft.spage%253D1152%26rft.epage%253D1163%26rft.atitle%253DComparison%2Bof%2Bgroup%2Btesting%2Balgorithms%2Bfor%2Bcase%2Bidentification%2Bin%2Bthe%2Bpresence%2Bof%2Btest%2Berror.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2007.00817.x%26rft_id%253Dinfo%253Apmid%252F17501946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2007.00817.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17501946&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251508300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.26" data-doi="10.1126/scitranslmed.abb1469"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Layne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hyman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Morens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Taubenberger</span></span>, <span class="cit-article-title">New coronavirus outbreak: Framing questions for pandemic prevention</span>. <abbr class="cit-jnl-abbrev">Science Translational Medicine</abbr>. <span class="cit-vol">12</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/scitranslmed.abb1469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DLayne%26rft.auinit1%253DS.%2BP.%26rft.volume%253D12%26rft.issue%253D534%26rft.spage%253Deabb1469%26rft.epage%253Deabb1469%26rft.atitle%253DNew%2Bcoronavirus%2Boutbreak%253A%2BFraming%2Bquestions%2Bfor%2Bpandemic%2Bprevention%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.abb1469%26rft_id%253Dinfo%253Apmid%252F32161107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjE1OiIxMi81MzQvZWFiYjE0NjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAzLjAyNDIxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Landt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kaiser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Molenkamp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Meijer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bleicker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brünink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schneider</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Schmidt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Mulders</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Haagmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">van der Veer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">van den Brink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wijsman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Goderski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Romette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ellis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zambon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Peiris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Goossens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Reusken</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Koopmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr>. <span class="cit-vol">25</span>, p. <span class="cit-fpage">2000045</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEurosurveillance%26rft.volume%253D25%26rft.spage%253D2000045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Samuels</span></span>, <span class="cit-article-title">The exact solution to the two-stage group-testing problem</span>. <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">500</span> (<span class="cit-pub-date">1978</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D20%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.03.024216v1.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hanson</span></span>, <span class="cit-source">ARUP COVID-19 rRT-PCR diagnostic testing results. Testing method is the EUA approved Hologic Panther Fusion SARS-COV2 assay</span>: <a href="https://www.fda.gov/media/136156/download">https://www.fda.gov/media/136156/download</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.30" data-doi="10.1001/jama.2020.3786"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-article-title">Detection of SARS-CoV-2 in Different Types of Clinical Specimens</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.3786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.3786%26rft_id%253Dinfo%253Apmid%252F32159775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2020.3786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32159775&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.31" data-doi="10.1097/OLQ.0b013e318231cd4a"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Lewis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Lockary</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kobic</span></span>, <span class="cit-article-title">Cost Savings and Increased Efficiency Using a Stratified Specimen Pooling Strategy for Chlamydia trachomatis and Neisseria gonorrhoeae</span>. <abbr class="cit-jnl-abbrev">Sexually Transmitted Diseases</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">46</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSexually%2Btransmitted%2Bdiseases%26rft.stitle%253DSex%2BTransm%2BDis%26rft.aulast%253DLewis%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D48%26rft.atitle%253DCost%2Bsavings%2Band%2Bincreased%2Befficiency%2Busing%2Ba%2Bstratified%2Bspecimen%2Bpooling%2Bstrategy%2Bfor%2BChlamydia%2Btrachomatis%2Band%2BNeisseria%2Bgonorrhoeae.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FOLQ.0b013e318231cd4a%26rft_id%253Dinfo%253Apmid%252F22183846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/OLQ.0b013e318231cd4a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22183846&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.024216.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.03.024216v1.32" data-doi="10.1126/science.367.6484.1287"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cohen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kupferschmidt</span></span>, <span class="cit-article-title">Countries test tactics in ‘war’ against COVID-19</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">367</span>, <span class="cit-fpage">1287</span>–<span class="cit-lpage">1288</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%26rft.volume%253D367%26rft.issue%253D6484%26rft.spage%253D1287%26rft.epage%253D1288%26rft.atitle%253DCountries%2Btest%2Btactics%2Bin%2B%2527war%2Bagainst%2BCOVID-19%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.367.6484.1287%26rft_id%253Dinfo%253Apmid%252F32193299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4NC8xMjg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMy4wMjQyMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.03.024216v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.05.025858v1" title="Annulohypoxylon sp. strain MUS1, an Endophyte isolated from Taxus wallichiana Zucc. produces Taxol and Other Bioactive Metabolites" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.05.026104v1" title="Urine proteome changes in an α-synuclein transgenic mouse model of Parkinson’s disease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 05, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.03.024216v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.03.024216v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.03.024216v1" class="active" data-icon-position="" data-hide-link-title="0">Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1230016" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gBD5iPc0U3roSC0E4Rd5kQeASPGUCn2yXnWtdJT95mk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1230016" id="node1230016--2" data-pisa="biorxiv;2020.04.03.024216v1" data-pisa-master="biorxiv;2020.04.03.024216" data-seqnum="1230016" data-apath="/biorxiv/early/2020/04/05/2020.04.03.024216.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rodrigo</span> <span class="nlm-surname">Noriega</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Samore</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.03.024216; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.03.024216 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;title=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;title=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;text=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;title=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;t=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;title=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;title=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1230016" id="node1230016--3" data-pisa="biorxiv;2020.04.03.024216v1" data-pisa-master="biorxiv;2020.04.03.024216" data-seqnum="1230016" data-apath="/biorxiv/early/2020/04/05/2020.04.03.024216.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rodrigo</span> <span class="nlm-surname">Noriega</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Samore</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.03.024216; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.03.024216 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Noriega%2C+Rodrigo&amp;rft.au=Samore%2C+Matthew+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1230016/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1230016/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1230016/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1230016/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1230016/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1230016/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1230016/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1230016/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1230016/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1230016/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1230016/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1230016/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;count=horizontal&amp;via=&amp;text=Increasing%20testing%20throughput%20and%20case%20detection%20with%20a%20pooled-sample%20Bayesian%20approach%20in%20the%20context%20of%20COVID-19&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.03.024216v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.03.024216v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
