<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/19/2020.03.16.993584.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.16.993584v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.16.993584"/>
  <meta name="DC.Date" content="2020-03-19"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="COVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses."/>
  <meta name="DC.Contributor" content="Minfeng Xiao"/>
  <meta name="DC.Contributor" content="Xiaoqing Liu"/>
  <meta name="DC.Contributor" content="Jingkai Ji"/>
  <meta name="DC.Contributor" content="Min Li"/>
  <meta name="DC.Contributor" content="Jiandong Li"/>
  <meta name="DC.Contributor" content="Lin Yang"/>
  <meta name="DC.Contributor" content="Wanying Sun"/>
  <meta name="DC.Contributor" content="Peidi Ren"/>
  <meta name="DC.Contributor" content="Guifang Yang"/>
  <meta name="DC.Contributor" content="Jincun Zhao"/>
  <meta name="DC.Contributor" content="Tianzhu Liang"/>
  <meta name="DC.Contributor" content="Huahui Ren"/>
  <meta name="DC.Contributor" content="Tian Chen"/>
  <meta name="DC.Contributor" content="Huanzi Zhong"/>
  <meta name="DC.Contributor" content="Wenchen Song"/>
  <meta name="DC.Contributor" content="Yanqun Wang"/>
  <meta name="DC.Contributor" content="Ziqing Deng"/>
  <meta name="DC.Contributor" content="Yanping Zhao"/>
  <meta name="DC.Contributor" content="Zhihua Ou"/>
  <meta name="DC.Contributor" content="Daxi Wang"/>
  <meta name="DC.Contributor" content="Jielun Cai"/>
  <meta name="DC.Contributor" content="Xinyi Cheng"/>
  <meta name="DC.Contributor" content="Taiqing Feng"/>
  <meta name="DC.Contributor" content="Honglong Wu"/>
  <meta name="DC.Contributor" content="Yanping Gong"/>
  <meta name="DC.Contributor" content="Huanming Yang"/>
  <meta name="DC.Contributor" content="Jian Wang"/>
  <meta name="DC.Contributor" content="Xun Xu"/>
  <meta name="DC.Contributor" content="Shida Zhu"/>
  <meta name="DC.Contributor" content="Fang Chen"/>
  <meta name="DC.Contributor" content="Yanyan Zhang"/>
  <meta name="DC.Contributor" content="Weijun Chen"/>
  <meta name="DC.Contributor" content="Yimin Li"/>
  <meta name="DC.Contributor" content="Junhua Li"/>
  <meta name="article:published_time" content="2020-03-19"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;COVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.16.993584v2"/>
  <meta name="citation_id" content="2020.03.16.993584v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.993584v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.993584v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.993584v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.16.993584"/>
  <meta name="citation_num_pages" content="25"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.16.993584"/>
  <meta name="citation_author" content="Minfeng Xiao"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0507-7352"/>
  <meta name="citation_author" content="Xiaoqing Liu"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6784-1873"/>
  <meta name="citation_author" content="Jingkai Ji"/>
  <meta name="citation_author_institution" content="School of Future Technology, University of Chinese Academy of Sciences"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Min Li"/>
  <meta name="citation_author_institution" content="BGI Education Center, University of Chinese Academy of Sciences"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Jiandong Li"/>
  <meta name="citation_author_institution" content="BGI Education Center, University of Chinese Academy of Sciences"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Lin Yang"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author" content="Wanying Sun"/>
  <meta name="citation_author_institution" content="BGI Education Center, University of Chinese Academy of Sciences"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Peidi Ren"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Guifang Yang"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author" content="Jincun Zhao"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University"/>
  <meta name="citation_author_institution" content="Institute of Infectious disease, Guangzhou Eighth People’s Hospital of Guangzhou Medical University"/>
  <meta name="citation_author" content="Tianzhu Liang"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Huahui Ren"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author" content="Tian Chen"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author" content="Huanzi Zhong"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9512-1750"/>
  <meta name="citation_author" content="Wenchen Song"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Yanqun Wang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University"/>
  <meta name="citation_author" content="Ziqing Deng"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Yanping Zhao"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Zhihua Ou"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Daxi Wang"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author" content="Jielun Cai"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author" content="Xinyi Cheng"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author" content="Taiqing Feng"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author" content="Honglong Wu"/>
  <meta name="citation_author_institution" content="BGI PathoGenesis Pharmaceutical Technology"/>
  <meta name="citation_author" content="Yanping Gong"/>
  <meta name="citation_author_institution" content="BGI PathoGenesis Pharmaceutical Technology"/>
  <meta name="citation_author" content="Huanming Yang"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="James D. Watson Institute of Genome Science"/>
  <meta name="citation_author" content="Jian Wang"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="James D. Watson Institute of Genome Science"/>
  <meta name="citation_author" content="Xun Xu"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Genome Read and Write, BGI-Shenzhen"/>
  <meta name="citation_author" content="Shida Zhu"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Engineering Laboratory for Innovative Molecular Diagnostics, BGI-Shenzhen"/>
  <meta name="citation_author" content="Fang Chen"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author" content="Yanyan Zhang"/>
  <meta name="citation_author_institution" content="MGI, BGI-Shenzhen"/>
  <meta name="citation_author_email" content="lijunhua@genomics.cn"/>
  <meta name="citation_author_email" content="dryiminli@vip.163.com"/>
  <meta name="citation_author_email" content="chenwj@genomics.com"/>
  <meta name="citation_author_email" content="zhangyanyan@genomics.cn"/>
  <meta name="citation_author" content="Weijun Chen"/>
  <meta name="citation_author_institution" content="BGI PathoGenesis Pharmaceutical Technology"/>
  <meta name="citation_author_institution" content="BGI Education Center, University of Chinese Academy of Sciences"/>
  <meta name="citation_author_email" content="lijunhua@genomics.cn"/>
  <meta name="citation_author_email" content="dryiminli@vip.163.com"/>
  <meta name="citation_author_email" content="chenwj@genomics.com"/>
  <meta name="citation_author_email" content="zhangyanyan@genomics.cn"/>
  <meta name="citation_author" content="Yimin Li"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University"/>
  <meta name="citation_author_email" content="lijunhua@genomics.cn"/>
  <meta name="citation_author_email" content="dryiminli@vip.163.com"/>
  <meta name="citation_author_email" content="chenwj@genomics.com"/>
  <meta name="citation_author_email" content="zhangyanyan@genomics.cn"/>
  <meta name="citation_author" content="Junhua Li"/>
  <meta name="citation_author_institution" content="BGI-Shenzhen"/>
  <meta name="citation_author_institution" content="Shenzhen Key Laboratory of Unknown Pathogen Identification, BGI-Shenzhen"/>
  <meta name="citation_author_email" content="lijunhua@genomics.cn"/>
  <meta name="citation_author_email" content="dryiminli@vip.163.com"/>
  <meta name="citation_author_email" content="chenwj@genomics.com"/>
  <meta name="citation_author_email" content="zhangyanyan@genomics.cn"/>
  <meta name="citation_reference" content="Jiang, S. et al. A distinct name is needed for the new coronavirus. The Lancet (2020)."/>
  <meta name="citation_reference" content="Organization, W. H. in Available from: https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/ (Geneva: WHO, 2020)."/>
  <meta name="citation_reference" content="Dudas, G. et al. Virus genomes reveal factors that spread and sustained the Ebola epidemic. Nature 544, 309–315, doi:10.1038/nature22040 (2017)."/>
  <meta name="citation_reference" content="Dudas, G., Carvalho, L. M., Rambaut, A. &amp;; Bedford, T. Correction: MERS-CoV spillover at the camel-human interface. Elife 7, doi:10.7554/eLife.37324 (2018)."/>
  <meta name="citation_reference" content="Gardy, J. L. &amp;; Loman, N. J. Towards a genomics-informed, real-time, global pathogen surveillance system. Nat Rev Genet 19, 9–20, doi:10.1038/nrg.2017.88 (2018)."/>
  <meta name="citation_reference" content="Gire, S. K. et al. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science 345, 1369–1372, doi:10.1126/science.1259657 (2014)."/>
  <meta name="citation_reference" content="Grubaugh, N. D. et al. Genomic epidemiology reveals multiple introductions of Zika virus into the United States. Nature 546, 401–405, doi:10.1038/nature22400 (2017)."/>
  <meta name="citation_reference" content="Sabir, J. S. et al. Co-circulation of three camel coronavirus species and recombination of MERS-CoVs in Saudi Arabia. Science 351, 81–84, doi:10.1126/science.aac8608 (2016)."/>
  <meta name="citation_reference" content="Lu, R. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574, doi:10.1016/S0140-6736(20)30251-8 (2020)."/>
  <meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 382, 727–733, doi:10.1056/NEJMoa2001017 (2020)."/>
  <meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273, doi:10.1038/s41586-020-2012-7 (2020)."/>
  <meta name="citation_reference" content="Chen, L. et al. RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak. Emerg Microbes Infect 9, 313–319, doi:10.1080/22221751.2020.1725399 (2020)."/>
  <meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature 579, 265–269, doi:10.1038/s41586-020-2008-3 (2020)."/>
  <meta name="citation_reference" content="Drosten, C. et al. Evaluation of advanced reverse transcription-PCR assays and an alternative PCR target region for detection of severe acute respiratory syndrome-associated coronavirus. J Clin Microbiol 42, 2043–2047, doi:10.1128/jcm.42.5.2043-2047.2004 (2004)."/>
  <meta name="citation_reference" content="Jonsdottir, H. R. &amp;; Dijkman, R. Coronaviruses and the human airway: a universal system for virus-host interaction studies. Virol J 13, 24, doi:10.1186/s12985-016-0479-5 (2016)."/>
  <meta name="citation_reference" content="Chan, J. F. et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet 395, 514–523, doi:10.1016/S0140-6736(20)30154-9 (2020)."/>
  <meta name="citation_reference" content="Holshue, M. L. et al. First Case of 2019 Novel Coronavirus in the United States. N Engl J Med 382, 929–936, doi:10.1056/NEJMoa2001191 (2020)."/>
  <meta name="citation_reference" content="McCrone, J. T. et al. Stochastic processes constrain the within and between host evolution of influenza virus. Elife 7, e35962 (2018)."/>
  <meta name="citation_reference" content="Ni, M. et al. Intra-host dynamics of Ebola virus during 2014. Nat Microbiol 1, 16151, doi:10.1038/nmicrobiol.2016.151 (2016)."/>
  <meta name="citation_reference" content="Park, D. J. et al. Ebola Virus Epidemiology, Transmission, and Evolution during Seven Months in Sierra Leone. Cell 161, 1516–1526, doi:10.1016/j.cell.2015.06.007 (2015)."/>
  <meta name="citation_reference" content="Domingo, E., Sheldon, J. &amp;; Perales, C. Viral quasispecies evolution. Microbiol Mol Biol Rev 76, 159–216, doi:10.1128/MMBR.05023-11 (2012)."/>
  <meta name="citation_reference" content="Metsky, H. C. et al. Zika virus evolution and spread in the Americas. Nature 546, 411–415, doi:10.1038/nature22402 (2017)."/>
  <meta name="citation_reference" content="Li, Q. et al. Reliable multiplex sequencing with rare index mis-assignment on DNB-based NGS platform. BMC Genomics 20, 215, doi:10.1186/s12864-019-5569-5 (2019)."/>
  <meta name="citation_reference" content="Xia, Z. et al. Advanced Whole Genome Sequencing Using a Complete PCR-free Massively Parallel Sequencing (MPS) Workflow. bioRxiv (2019)."/>
  <meta name="citation_reference" content="Zou, L. et al. SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients. N Engl J Med, doi:10.1056/NEJMc2001737 (2020)."/>
  <meta name="citation_reference" content="Wood, D. E. &amp;; Salzberg, S. L. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biol 15, R46, doi:10.1186/gb-2014-15-3-r46 (2014)."/>
  <meta name="citation_reference" content="Chen, S., Zhou, Y., Chen, Y. &amp;; Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34, i884–i890, doi:10.1093/bioinformatics/bty560 (2018)."/>
  <meta name="citation_reference" content="Chen, Y. et al. SOAPnuke: a MapReduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data. Gigascience 7, 1–6, doi:10.1093/gigascience/gix120 (2018)."/>
  <meta name="citation_reference" content="Schmieder, R. &amp;; Edwards, R. Quality control and preprocessing of metagenomic datasets. Bioinformatics 27, 863–864, doi:10.1093/bioinformatics/btr026 (2011)."/>
  <meta name="citation_reference" content="Au, C. H., Ho, D. N., Kwong, A., Chan, T. L. &amp;; Ma, E. S. K. BAMClipper: removing primers from alignments to minimize false-negative mutations in amplicon next-generation sequencing. Sci Rep 7, 1567, doi:10.1038/s41598-017-01703-6 (2017)."/>
  <meta name="citation_reference" content="Bankevich, A. et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19, 455–477, doi:10.1089/cmb.2012.0021 (2012)."/>
  <meta name="citation_reference" content="Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS One 9, e112963, doi:10.1371/journal.pone.0112963 (2014)."/>
  <meta name="citation_reference" content="Li, H. &amp;; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–1760, doi:10.1093/bioinformatics/btp324 (2009)."/>
  <meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079, doi:10.1093/bioinformatics/btp352 (2009)."/>
  <meta name="citation_reference" content="Wickham, H. ggplot2. Wiley Interdisciplinary Reviews: Computational Statistics 3, 180–185 (2011)."/>
  <meta name="citation_reference" content="Team, R. C. R: A language and environment for statistical computing. (2013)."/>
  <meta name="citation_reference" content="Kolde, R. &amp;; Kolde, M. R. Package ‘pheatmap’. R Package 1 (2015)."/>
  <meta name="citation_reference" content="Kassambara, A. ggpubr:”ggplot2” based publication ready plots. R package version 0.1 6 (2017)."/>
  <meta name="citation_reference" content="Seemann, T. in Snippy: fast bacterial variant calling from NGS reads (WWW Document, 2015)."/>
  <meta name="citation_reference" content="Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly (Austin) 6, 80–92, doi:10.4161/fly.19695 (2012)."/>
  <meta name="citation_reference" content="Li, R. et al. SOAP2: an improved ultrafast tool for short read alignment. Bioinformatics 25, 1966–1967, doi:10.1093/bioinformatics/btp336 (2009)."/>
  <meta name="citation_reference" content="Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp;; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol 37, 907–915, doi:10.1038/s41587-019-0201-4 (2019)."/>
  <meta name="citation_reference" content="Wood, D. E., Lu, J. &amp;; Langmead, B. Improved metagenomic analysis with Kraken 2. Genome Biol 20, 257, doi:10.1186/s13059-019-1891-0 (2019)."/>
  <meta name="citation_reference" content="Lu, J., Breitwieser, F. P., Thielen, P. &amp;; Salzberg, S. L. Bracken: estimating species abundance in metagenomics data. PeerJ Computer Science 3, e104 (2017)."/>
  <meta name="citation_reference" content="Kircher, M., Sawyer, S. &amp;; Meyer, M. Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform. Nucleic Acids Res 40, fe3, doi:10.1093/nar/gkr771 (2012)."/>
  <meta name="twitter:title" content="Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="COVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses."/>
  <meta name="og-title" property="og:title" content="Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.993584v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="COVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-19"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1202705 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-4">COVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-5">As of 14 March 2020, human coronavirus 2019 (HCoV-19) has surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) in every aspect, infecting over 140,000 people in more than 110 countries, with a mortality of over 5,000
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. So far, coronaviruses have caused three major epidemics in the past two decades, posing a great challenge to global health and economy. Massively parallel sequencing (MPS) of viral genomes has demonstrated enormous capacity as a powerful tool to study emerging infectious diseases, such as SARS, MERS, Zika, and Ebola, in tracing the outbreak origin and drivers, tracking transmission chains, mapping the spread, and monitoring the evolution of the etiological agents
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>. Though by 14 March 2020, fewer than 500 HCoV-19 genomes were published on public databases including China National GeneBank DataBase (CNGBdb), NCBI GenBank, the Global initiative on sharing all influenza data (GISAID), etc, and much remains unknown about the epidemiology and evolution of the virus. One possible explanation of the paucity of published HCoV-19 genomes was the challenges posed by sequencing clinical samples with low virus abundance.
    </p>
    <p id="p-6">The first teams obtained the HCoV-19 genome sequences through metatranscriptomic MPS, supplemented by PCR and Sanger sequencing of a combination of bronchoalveolar-lavage fluid (BALF) and culture
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>–
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup> or from BALF directly
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup>. Experience from studying SARS-CoV showed that BALF from the lower respiratory tract was an ideal sample type with higher viral load
     <sup>
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>. However, BALF was not routinely collected from every patient, and human airway epithelial (HAE) cell culture is very labor-intensive and time-consuming, taking four to six weeks
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup>. The University of Hong Kong team managed to get the whole-genome sequences through metatranscriptomic sequencing with Oxford Nanopore platform supplemented by Sanger sequencing from both nasopharyngeal and sputum specimens after single-primer amplification
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </sup>. The United States scientists published the whole-genome sequence using oropharyngeal and nasopharyngeal specimens through Sanger and meta-transcriptomic sequencing with both Illumina and MinIon
     <sup>
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>. To date, multiplex PCR-based or hybrid capture-based whole-genome sequencing of HCoV-19, as well as comparative studies between different approaches, have not been reported on peer-reviewed journals.
    </p>
    <p id="p-7">Besides inter-individual variations, dissecting intra-individual dynamics of viruses also largely promotes our understanding of viral-host interactions, viral evolution and transmission as demonstrated for Ebola, Zika, Influenza, etc
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>-
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
     </sup>. The analyses of intra-individual single nucleotide variations (iSNVs) and its allele frequency have also contributed to antiviral therapy and drug resistance, e.g., to reveal highly conserved genes during the outbreak that potentially serve as ideal therapeutic targets
     <sup>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup>. However, it is a challenge to accurately detect iSNVs from clinical samples, especially when the samples are subjected to extra steps of enrichment and amplification.
    </p>
    <p id="p-8">Therefore, we aim to comprehensively compare the sensitivity, inter-individual (variant) and intra-individual (iSNV) accuracy, and other general features of different approaches by systematically utilizing ultra-high-throughput metatranscriptomic, hybrid capture-based, and amplicon-based sequencing approaches to obtain genomic information of HCoV-19 from serial dilutions of a cultured isolate and directly from clinical samples. We present a reasonable sequencing strategy that fits into different scenarios, and estimate the minimal amount of sequencing data necessary for downstream HCoV-19 genome analyses. Our study, to-gether with our tailor-made experimental workflows and bioinformatics pipelines, offers very practical solutions to facilitate the studies of HCoV-19 and other emerging viruses in the future and would promote extensive genomic sequencing and analyses of HCoV-19 and other emerging viruses, underpinning more comprehensive real-time virus surveillance and more efficient viral outbreaks managing.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">RESULTS</h2>
    <div id="sec-3" class="subsection">
     <h3>Design of the comparative study</h3>
     <p id="p-9">We sampled eight specimens from COVID-19 patients in February 2020, including throat swab, nasal swab, anal swab and sputum specimens, and the corresponding cycle threshold (Ct) value of HCoV-19 qRT-PCR ranges from 18 to 32 (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We initially tried to boost the coverage and depth of the viral genome by ultra-deep metatranscriptomic sequencing with an average sequencing amount of 1,607,264,105 paired-end reads (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Although we managed to obtain complete viral genome assemblies for each specimen, the sequencing depth varied across specimens. Only 0.002%-0.003% of the total reads were assigned to the HCoV-19 in three samples (GZMU0014, GZMU0030 and GZMU0031) with Ct between 29-32, resulting in inferior sequencing depth (less than 100X) (
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Isolating viruses and enriching them in cell culture might improve the situation, but this requires high-standard laboratory settings and expertise apart from being time-consuming. Also, unwanted mutations that are not concordant with original clinical samples may be introduced during the culturing process.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1207273/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1207273/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1207273" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-10" class="first-child">Metatranscriptomic sequencing data summary of eight HCoV-19 positive clinic al samples collected from Guangzhou in February 2020</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">To enrich adequate viral content for whole-genome sequencing in a convenient manner, we pursued two other methods: multiplex PCR amplification (amplicon) and hybrid capture (capture) (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). We designed a systematic study to comprehensively validate the bias, sensitivity, inter-individual (variant) and intra-individual (iSNV) accuracy of multiple approaches by sequencing serial dilutions of a cultured isolate (unpublished), as well as the eight clinical samples (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). We performed qRT-PCR of 10-fold serial dilutions (D1-D7) of the cultured isolate, and the Ct was 17.3, 20.8, 24.5 for, 28.7, 31.8, 35, and 39.9, respectively, indicating the undiluted RNA (D0) of the cultured isolate contained ~1E+08 genome copies per mL. For amplicon sequencing, we utilized two kits comprising of two set of primers generating PCR products of 300-400 bp and 100-200 bp, respectively. The ~400 bp amplicon-based sequencing was implemented in all samples and analyzed throughout the study, while the ~200 bp amplicon-based sequencing was only applied in the cultured isolate for coverage analysis.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The general workflow of multiple sequencing approaches adopted in this study. We employed unique dual indexing (UDI) strategy and DNB-based (DNA Nanoball) PCR-free MPS platform to minimize index hopping and relevant sequencing errors23,24,46. a, Amplicon-based enrichment, the dual indexing was integrated in the 2nd PCR. Navy, multiplex PCR primers. b, Metatranscriptomic library preparations, the dual indexing was integrated in the universal PCR. c, Library preparations and hybrid capture-based enrichment, the 1st indexing was integrated in the pre-capture PCR while the 2nd indexing was integrated in the post-capture PCR. Ocher, ssDNA probes. Red and green lines represent adaptor sequences; green dots represent phosphate groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180545937" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The general workflow of multiple sequencing approaches adopted in this study.&lt;;/span&gt;; We employed unique dual indexing (UDI) strategy and DNB-based (DNA Nanoball) PCR-free MPS platform to minimize index hopping and relevant sequencing errors&lt;;sup&gt;;23,24,46&lt;;/sup&gt;;. &lt;;strong&gt;;a&lt;;/strong&gt;;, Amplicon-based enrichment, the dual indexing was integrated in the 2nd PCR. Navy, multiplex PCR primers. &lt;;strong&gt;;b&lt;;/strong&gt;;, Metatranscriptomic library preparations, the dual indexing was integrated in the universal PCR. &lt;;strong&gt;;c&lt;;/strong&gt;;, Library preparations and hybrid capture-based enrichment, the 1st indexing was integrated in the pre-capture PCR while the 2nd indexing was integrated in the post-capture PCR. Ocher, ssDNA probes. Red and green lines represent adaptor sequences; green dots represent phosphate groups.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F1.medium.gif" width="440" height="309"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F1.medium.gif" width="440" height="309"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">The general workflow of multiple sequencing approaches adopted in this study.</span>
       <p id="p-12" class="first-child">We employed unique dual indexing (UDI) strategy and DNB-based (DNA Nanoball) PCR-free MPS platform to minimize index hopping and relevant sequencing errors
        <sup>
         <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,
         <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,
         <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>
        </sup>. 
        <strong>a</strong>, Amplicon-based enrichment, the dual indexing was integrated in the 2nd PCR. Navy, multiplex PCR primers. 
        <strong>b</strong>, Metatranscriptomic library preparations, the dual indexing was integrated in the universal PCR. 
        <strong>c</strong>, Library preparations and hybrid capture-based enrichment, the 1st indexing was integrated in the pre-capture PCR while the 2nd indexing was integrated in the post-capture PCR. Ocher, ssDNA probes. Red and green lines represent adaptor sequences; green dots represent phosphate groups.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Overview of the study design. Eight clinical samples and serial dilutions of a cultured isolate were subjected to direct metatranscriptomic library construction, ampliconbased enrichment, and hybrid capture-based enrichment, respectively. Libraries generated from each method were pooled, respectively. DNB, DNA Nanoball. 14, GZMU0014; 16, GZMU0016; 30, GZMU0030; 31, GZMU0031; 42, GZMU0042; 44, GZMU0044; 47, GZMU0047; 48, GZMU0048. D0, undiluted sample of the cultured isolate; D1-D7, seven serial diluted samples of the cultured isolate, ranging from 1E+07 to 1E+01 genome copies per mL, in 10-fold dilution. -, negative controls prepared from nuclease-free water and human nucleic acids. PE100, paired-end 100-nt reads; SE400, single-end 100-nt reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180545937" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Overview of the study design.&lt;;/span&gt;; Eight clinical samples and serial dilutions of a cultured isolate were subjected to direct metatranscriptomic library construction, ampliconbased enrichment, and hybrid capture-based enrichment, respectively. Libraries generated from each method were pooled, respectively. DNB, DNA Nanoball. 14, GZMU0014; 16, GZMU0016; 30, GZMU0030; 31, GZMU0031; 42, GZMU0042; 44, GZMU0044; 47, GZMU0047; 48, GZMU0048. D0, undiluted sample of the cultured isolate; D1-D7, seven serial diluted samples of the cultured isolate, ranging from 1E+07 to 1E+01 genome copies per mL, in 10-fold dilution. -, negative controls prepared from nuclease-free water and human nucleic acids. PE100, paired-end 100-nt reads; SE400, single-end 100-nt reads.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F2.medium.gif" width="342" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F2.medium.gif" width="342" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Overview of the study design.</span>
       <p id="p-13" class="first-child">Eight clinical samples and serial dilutions of a cultured isolate were subjected to direct metatranscriptomic library construction, ampliconbased enrichment, and hybrid capture-based enrichment, respectively. Libraries generated from each method were pooled, respectively. DNB, DNA Nanoball. 14, GZMU0014; 16, GZMU0016; 30, GZMU0030; 31, GZMU0031; 42, GZMU0042; 44, GZMU0044; 47, GZMU0047; 48, GZMU0048. D0, undiluted sample of the cultured isolate; D1-D7, seven serial diluted samples of the cultured isolate, ranging from 1E+07 to 1E+01 genome copies per mL, in 10-fold dilution. -, negative controls prepared from nuclease-free water and human nucleic acids. PE100, paired-end 100-nt reads; SE400, single-end 100-nt reads.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Comparison of evenness and sensitivity</h3>
     <p id="p-14">Theoretically, amplicon sequencing should be the most sensitive and economical method among the three, and is particularly suitable in an outbreak where viral isolates are highly related. Although, there are still potential pitfalls, for instance, the 40 cycle-PCR in our workflow might augment trace amounts of HCoV-19 cross-contamination. To ensure the confidence of the datasets, we included serial dilutions of the cultured isolate and negative controls prepared from nuclease-free water and human nucleic acids since the 1st PCR. All samples in ~400 bp amplicon-based sequencing exhibited &gt;; 99.5% coverage across the HCoV-19 genome except for 1E+01 (95.23%), GZMU0031 (73.65%), HNA (6.17%), water (60.24%), suggesting the primers were well designed and the positive datasets were reliable. We also set stringent and method-specific criteria to filter low-confidence sequencing reads and samples (see Methods), e.g., clinical sample GZMU0031 was excluded for downstream sensitivity and accuracy validation due to inadequate depth in amplicon sequencing (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). Another pitfall is that amplification across the genome can hardly be unbiased, causing difficulties in complete genome assembly. Indeed, amplicon sequencing exhibited a higher level of bias compared with meta-transcriptomic sequencing, in terms of coverage across the viral genomes from the cultural isolate and the clinical samples tested in our study (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b, d</a>, Supplementary Fig. 1). To our surprise, however, capture sequencing was almost as unbiased as meta sequencing, demonstrating better performance than the previous capture method used to enrich ZIKV despite the HCoV-19 genome is ~3 fold larger than ZIKV
      <sup>
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
      </sup> (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b, c</a>). Two reasons amongst others were likely to be accountable to this improvement, 1) we utilized 506 pieces of 120 ssDNA probes covering 2x of the HCoV-19 genome to capture the libraries, 2) we employed the DNBSEQ sequencing technology that features PCR-free rolling circle replication (RCR) of DNA Nanoballs (DNBs)
      <sup>
       <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>,
       <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>
      </sup>.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequencing coverage and depth of the cultured isolate and eight clinical samples. a, Amplicon sequencing coverage by sample (row) across the HCoV-19 genome. Pink, sequencing depth [[#x2265]]100[[#xD7]]; heatmap (bottom) sums coverage across all samples. HNA, negative control prepared from human nucleic acids; water, negative control prepared from nuclease-free water. Green horizontal lines on heatmap, amplicon locations. Overlap regions between amplicons range from 59-209 bp. b-d, Normalized coverage across viral genomes of the clinical samples across methods. e, HCoV-19-RPM sequenced plotted against genome copies per mL for the cultured isolate. Three independent experiments were performed for amplicon sequencing. Pink, ~400 bp amplicon-based sequencing including human and lambda phage nucleic acids background; red, ~200 bp amplicon-based sequencing; orange, ~400 bp amplicon-based sequencing excluding human and lambda phage nucleic acids background (NAB); light blue, capture sequencing. f, HCoV-19-RPM (Reads Per Million) sequenced plotted against qRT-PCR Ct value for the clinical samples. Pink, amplicon; light blue, capture; blue, meta. g, Estimated minimum amount of bases required by each method for high-confidence downstream analyses. Pink, amplicon; light blue, capture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180545937" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sequencing coverage and depth of the cultured isolate and eight clinical samples.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Amplicon sequencing coverage by sample (row) across the HCoV-19 genome. Pink, sequencing depth [[#x2265]]100[[#xD7]]; heatmap (bottom) sums coverage across all samples. HNA, negative control prepared from human nucleic acids; water, negative control prepared from nuclease-free water. Green horizontal lines on heatmap, amplicon locations. Overlap regions between amplicons range from 59-209 bp. &lt;;strong&gt;;b-d&lt;;/strong&gt;;, Normalized coverage across viral genomes of the clinical samples across methods. &lt;;strong&gt;;e&lt;;/strong&gt;;, HCoV-19-RPM sequenced plotted against genome copies per mL for the cultured isolate. Three independent experiments were performed for amplicon sequencing. Pink, ~400 bp amplicon-based sequencing including human and lambda phage nucleic acids background; red, ~200 bp amplicon-based sequencing; orange, ~400 bp amplicon-based sequencing excluding human and lambda phage nucleic acids background (NAB); light blue, capture sequencing. &lt;;strong&gt;;f&lt;;/strong&gt;;, HCoV-19-RPM (Reads Per Million) sequenced plotted against qRT-PCR Ct value for the clinical samples. Pink, amplicon; light blue, capture; blue, meta. &lt;;strong&gt;;g&lt;;/strong&gt;;, Estimated minimum amount of bases required by each method for high-confidence downstream analyses. Pink, amplicon; light blue, capture.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F3.medium.gif" width="440" height="395"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F3.medium.gif" width="440" height="395"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Sequencing coverage and depth of the cultured isolate and eight clinical samples.</span>
       <p id="p-15" class="first-child">
        <strong>a</strong>, Amplicon sequencing coverage by sample (row) across the HCoV-19 genome. Pink, sequencing depth ≥100×; heatmap (bottom) sums coverage across all samples. HNA, negative control prepared from human nucleic acids; water, negative control prepared from nuclease-free water. Green horizontal lines on heatmap, amplicon locations. Overlap regions between amplicons range from 59-209 bp. 
        <strong>b-d</strong>, Normalized coverage across viral genomes of the clinical samples across methods. 
        <strong>e</strong>, HCoV-19-RPM sequenced plotted against genome copies per mL for the cultured isolate. Three independent experiments were performed for amplicon sequencing. Pink, ~400 bp amplicon-based sequencing including human and lambda phage nucleic acids background; red, ~200 bp amplicon-based sequencing; orange, ~400 bp amplicon-based sequencing excluding human and lambda phage nucleic acids background (NAB); light blue, capture sequencing. 
        <strong>f</strong>, HCoV-19-RPM (Reads Per Million) sequenced plotted against qRT-PCR Ct value for the clinical samples. Pink, amplicon; light blue, capture; blue, meta. 
        <strong>g</strong>, Estimated minimum amount of bases required by each method for high-confidence downstream analyses. Pink, amplicon; light blue, capture.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">The sequencing results of amplicon and capture approaches revealed dramatic increases in the ratio of HCoV-19 reads out of the total reads compared with meta sequencing, suggesting the enrichment was highly efficient - 5596-fold in capture method and 5710-fold in amplicon method for each sample on average (Supplementary Table 1-2). To further compare the sensitivity of different methods, we plotted the number of HCoV-19 reads per million (HCoV-19-RPM) of total sequencing reads against the viral concentration for each sample. The productivity was similar between the two methods when the input RNA of the cultured isolate contained 1E+05 genome copies per mL and above (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3e</a>). However, amplicon sequencing produced 10-100 fold more HCoV-19 reads than capture sequencing when the input RNA concentration of the cultured isolate was 1E+04 genome copies per mL and lower, suggesting amplicon-based enrichment was more efficient than capture for more challenging samples (conc. ≤ 1E+04 genome copies per mL, or Ct ≥ 28.7) (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3e</a>). Meta sequencing - as expected - produced dramatically lower HCoV-19-RPM than the other two methods among clinical samples tested with a wide range of Ct values, whereas amplicon and capture were generally comparable to each other (
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3e</a>). Considering the costs for sequencing, storage, and analysis increase substantially with larger datasets, we tried to estimate how much sequencing data must be produced for each approach in order to achieve 10X depth across 95% of the HCoV-19 genome, and the results can be found in Supplementary Table 3. As a practical, cost-effective guidance for future sequencing, we also assessed the minimum amount of data required to pass the stringent filters (≥ 95% coverage and method-specific depth, see Methods) in our pipelines corresponding to different viral loads. We estimated that for high-confidence downstream analyses, amplicon sequencing requires at least 2,757 to 186 Mega bases (Mb) for samples containing 1E+02 to 1E+06 copies of HCoV-19 genome per mL, while capture sequencing requires 24,474 to 9 Mb for the same situation (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3g</a>, Supplementary Table 4).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Investigation of inter-and intra-individual variations</h3>
     <p id="p-17">To determine the accuracy of different approaches in discovering inter-individual genetic diversity, we tested each method in calling the single nucleotide variations (SNVs) and verified some of the SNVs with Sanger sequencing (Supplementary Fig. 2). Two to five SNVs were identified within each clinical sample, and in all the seven samples, SNVs identified by the three methods were concordant except that capture missed one SNV at position 16535 in GZMU0014 (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>). We then investigated the allele frequencies of these sites across methods, and found that alleles identified by capture sequencing displayed lower frequencies than the other two methods, especially for GZMU0014, GZMU0030, and GZMU0042 where the viral load was lower (Ct ? 29), which explained why capture sequencing neglected an SNV in our pipeline when the cutoff of SNV calling was set as 80% allele frequency (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b</a>). These data indicate that amplicon sequencing is more accurate than capture sequencing in identifying SNVs, especially for challenging samples.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Between-sample and within-sample variants of HCoV-19 detected across methods. a, SNVs detected between clinical samples against a reference genome (GISAID accession: EPI_ISL_402119). Alleles with [[#x2265]] 80% frequencies were called. *, SNVs verified by Sanger sequencing. b, Allele frequencies of the identified SNVs. Pink, amplicon; light blue, capture; blue, meta. Minor allele frequencies detected in serial dilutions of the cultured isolate (c) and clinical samples (d) across methods. Pink, amplicon vs meta; light blue, capture vs meta. Minor alleles are defined with [[#x2265]] 5% and &lt;; 50% frequencies. Besides general quality filter, iSNVs had to pass depth and strand bias filter as described in Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180545937" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Between-sample and within-sample variants of HCoV-19 detected across methods.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, SNVs detected between clinical samples against a reference genome (GISAID accession: EPI_ISL_402119). Alleles with [[#x2265]] 80% frequencies were called. *, SNVs verified by Sanger sequencing. &lt;;strong&gt;;b&lt;;/strong&gt;;, Allele frequencies of the identified SNVs. Pink, amplicon; light blue, capture; blue, meta. Minor allele frequencies detected in serial dilutions of the cultured isolate (&lt;;strong&gt;;c&lt;;/strong&gt;;) and clinical samples (&lt;;strong&gt;;d&lt;;/strong&gt;;) across methods. Pink, amplicon vs meta; light blue, capture vs meta. Minor alleles are defined with [[#x2265]] 5% and &lt;; 50% frequencies. Besides general quality filter, iSNVs had to pass depth and strand bias filter as described in Methods.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F4.medium.gif" width="440" height="312"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F4.medium.gif" width="440" height="312"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/19/2020.03.16.993584/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">Between-sample and within-sample variants of HCoV-19 detected across methods.</span>
       <p id="p-18" class="first-child">
        <strong>a</strong>, SNVs detected between clinical samples against a reference genome (GISAID accession: EPI_ISL_402119). Alleles with ≥ 80% frequencies were called. *, SNVs verified by Sanger sequencing. 
        <strong>b</strong>, Allele frequencies of the identified SNVs. Pink, amplicon; light blue, capture; blue, meta. Minor allele frequencies detected in serial dilutions of the cultured isolate (
        <strong>c</strong>) and clinical samples (
        <strong>d</strong>) across methods. Pink, amplicon vs meta; light blue, capture vs meta. Minor alleles are defined with ≥ 5% and &lt;; 50% frequencies. Besides general quality filter, iSNVs had to pass depth and strand bias filter as described in Methods.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-19">To further determine the accuracy of different approaches in identifying HCoV-19 iSNVs, we examined minor allele frequencies in serial dilutions of the cultured HCoV-19 isolate and clinical samples. For serial dilutions of the cultured isolate, the minor allele frequencies detected in capture sequencing datasets were generally approximate to meta sequencing, while most allele frequencies in amplicon sequencing datasets deviated with those in meta sequencing (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4c</a>) A similar pattern was shown for clinical samples, indicating that amplicon sequencing was unreliable of quantifying minor allele frequencies (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4d</a>). Plotting allele frequencies against HCoV-19 concentrations supported the above finding, and further revealed that amplicon sequencing was unreliable of allele frequencies at all concentrations while capture sequencing was reliable at &gt;; 1E+03 genome copies per mL (Supplementary Fig. 3). Referring to the iSNV identified in clinical samples by meta sequencing, we then calculated the false positive rate (FPR) and false negative rate (FNR) of minor alleles called by amplicon and capture methods. The FPR and FNR of minor alleles identified in amplicon sequencing was 0.74% and 66.67%, while that in capture sequencing was 0.02% and 0%, respectively. Together these results suggest amplicon sequencing was not as accurate as capture sequencing in identifying minor alleles, which could be in part due to Matthew effect caused by PCR.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Microbiome in clinical samples</h3>
     <p id="p-20">In addition to target viral genome, metatranscriptomic sequencing has also allowed us to investigate RNA expression patterns of the overall microbiome and host content and thus suitable for discovering new viruses, distinguishing coinfections, and dissecting virus-host interactions. To explore the microbiota, we performed further metatranscriptomics analysis of the clinical samples. We were able to identify host nucleic acids in all of the samples, and over 95% of total reads were from the host in sputum, nasal, and throat samples (Supplementary Fig. 4a). Virus contributed to less than 5% of reads in anal swab and throat swab while more than 50% of reads in nasal swab (Supplementary Fig. 4b). These results suggest nasal swab could be the most ideal sample type for viral detection among the four sample types, which agrees with recent clinical evidence
      <sup>
       <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
      </sup>. Among the viral reads, over 90% were Coronaviridae, which is consistent with clinical diagnostics (Supplementary Fig. 4c). Reads from other viruses were also identified, indicating further measurements could be taken to confirm if co-infection exists (Supplementary Fig. 4). Bacterial composition was also shown, providing support for scientific research, as well as for further confirmation of bacterial infection and antibiotics prescription (Supplementary Fig. 4d-f).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Guidance for virus sequencing</h3>
     <p id="p-21">Taken together, each sequencing scheme elaborated here for massively parallel sequencing of HCoV-19 genomes has its own merits (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We hereby propose a reasonable, cost-effective strategy for sequencing and analyzing HCoV-19 under different situations: 1) if one wants to study other genetic materials than the target viruses, or the viruses become highly diversified via recombinational events, or the viral load within the RNA sample is high (e.g. conc. ≥ 1E+05 viral genome copies per mL, or Ct ≤ 24.5), meta sequencing can be prioritized; 2) if one focuses on intra-individual variations for more challenging samples (e.g. conc. &lt;; 1 E+05 viral genome copies per mL, or Ct &gt;; 24.5), capture sequencing seems to be a justified choice; and, 3) if identifying SNVs is the main purpose, the most convenient, economical strategy would be amplicon sequencing that can support high-confidence analyses of samples containing viral content as low as 1E+02 viral genome copies per mL, or Ct as high as 35.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1207263/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1207263/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1207263" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-22" class="first-child">General characteristics of the three approaches employed in this study</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">DISCUSSION</h2>
    <p id="p-23">Sequencing low-titre viruses directly from clinical samples is challenging, which is further exacerbated by the fact that coronavirus genomes are the largest among RNA viruses (~3 fold larger compared with ZIKV). Compared with direct metatranscriptomic sequencing, high sensitivity of hybrid capture and amplicon sequencing methods come at a price of low accuracy, and neither of the two can be used to sequence highly diverse or recombinant viruses because the primers and probes are specific to known viral genomes. Amplicon sequencing compromises its accuracy, while it becomes the most convenient and economical method of all. Either or a combination of the approaches described here can be chosen to cope with various needs of researchers, e.g., metatranscriptomic sequencing data with insufficient coverage and depth can be pooled with hybrid capture data to generate high quality assemblies
     <sup>
      <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>
     </sup>. Our research, as well as the methods elaborated here, are able to help other researchers to sequence and analyze large viruses from clinical samples and thus benefit investigations on the genomic epidemiology of viruses.
    </p>
    <p id="p-24">Some pros and cons described above might be specific to the experimental workflows and bioinformatics pipelines tailored in this study, e.g., 1) the bias of amplicon sequencing can be improved by reducing the amount of cycles in the 1st PCR or optimize the molar ratios of primers (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>), 2) the amplicon sequencing is particularly convenient compared with previous counterparts since the fragmentation and library construction steps are omitted here by integrating adaptor and barcode ligation in the 2nd PCR and sequencing the amplicons using single-end 400 nt reads (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1a</a>), 3) using anything less than 506 pieces of 120 ssDNA probes in hybrid capture may attenuate the sequencing coverage while increase the bias, 4) metatranscriptomic sequencing was conducted with an ultra-high-throughput sequencing platform so that the successful rate was substantially higher than usual. 5) the minimal amount of data necessary for analyzing the HCoV-19 genome from clinical samples across methods is higher than that predicted by data from the cultured isolate was probably due to the high nucleic acids background from the host and other microbes (Supplementary Table 3-4, Supplementary Fig. 4). Also, we do not take into account the time spent in sequencing since the workflows can be easily adapted in order to be compatible with various platforms including Illumina and Oxford Nanopore Technologies (ONT), besides DNBSEQ of MGI.
    </p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">METHODS</h2>
    <div id="sec-10" class="subsection">
     <h3>Ethics statement</h3>
     <p id="p-25">The Institutional Review Boards (IRB) of the First Affiliated Hospital of Guangzhou Medical University approved the clinical studies. IRB of BGI-Shenzhen approved the sequencing and downstream studies.</p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Sampling, RNA extraction, reverse transcription and qRT-PCR</h3>
     <p id="p-26">Clinical specimens (including throat swab, nasal swab, anal swab, and sputum) were obtained from confirmed COVID-19 cases at the First Affiliated Hospital of Guangzhou Medical University. Total RNA of the cultured isolate of HCoV-19 was obtained from the Academy of Military Medical Science (AMMS), and subjected to 10-fold serial dilutions. Total RNA was extracted with QiAamp RNeasy Mini Kit (Qiagen, Heiden, Germany) following the manufacturer’s instructions without modification. Real-time reverse transcription PCR (qRT-PCR) targeting RdRp gene and N gene of HCoV-19 was used to detect and quantify the viral RNA within clinical samples and serial dilutions of the cultured isolate using the HCoV-19 Nucleic Acid Detection Kit following the manufacture’s protocol (Geneodx, Shanghai, China, and BGI-Shenzhen, Shenzhen, China).</p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Metatranscriptomic library preparation and sequencing</h3>
     <p id="p-27">Host DNA was removed from RNA samples using DNase I, and the concentration of RNA samples was measured by Qubit RNA HS Assay Kit (Thermo Fisher Scientific, Waltham, MA, USA). DNA-depleted and purified RNA was used to construct the single-stranded circular DNA library with MGIEasy RNA Library preparation reagent set (MGI, Shenzhen, China), as follows: 1) RNA was fragmented by incubating with fragmentation buffer at 87°C for 6 min; 2) double-stranded (ds) cDNA was synthesized using random hexamers with fragmented RNA; 3) ds cDNA was subjected to end repair, adaptor ligation, and 18-cycle PCR amplification; 4) PCR products were Unique Dual Indexed (UDI), before going through circularization, and rolling circle replication (RCR) to generate DNA nanoballs (DNBs)-based libraries. DNBs preps of clinical samples were sequenced on the ultra-high-throughput DNB-SEQ-T7 platform (MGI, Shenzhen, China) with paired-end 100 nt strategy, generating 321 Gb sequencing data for each sample on average.</p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Hybrid capture-based enrichment and sequencing</h3>
     <p id="p-28">A hybrid capture technique was used to enrich HCoV-19-specific content from the meta-transcriptomic double-stranded DNA libraries with the 2019-nCoVirus DNA/RNA Capture Panel (BOKE, Jiangsu, China). Manufacturer’s instructions were slightly modified to accommodate the MGISEQ-2000 platform, i.e., blocker oligos and PCR primer oligos were replaced by MGIEasy exon capture assistive kit (MGI, Shenzhen, China). DNBs-based libraries were constructed and sequenced on the MGISEQ-2000 platform with paired-end 100 nt strategy using the same protocol described above, generating 37 Gb sequencing data for each sample on average.</p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Amplicon-based enrichment and sequencing</h3>
     <p id="p-29">Total RNA was reverse transcribed to synthesize the first-strand cDNA with random hexamers and Superscript II reverse transcriptase kit (Invitrogen, Carlsbad, USA). Sequencing was attempted on all samples regardless of Ct value including negative controls prepared from nuclease-free water and NA12878 human gDNA. A two-step HCoV-19 genome amplification was performed with an equimolar mixture of primers using ATOPlex SARS-CoV-2 Full Length Genome Panel following the manufacture’s protocol (MGI, Shenzhen, China), generating 137X ~400 bp amplicons or 299X ~200 bp amplicons and the genome positions of the amplicons are shown in Supplementary Table 5. 20 μl of first-strand cDNA was mixed with the components of the first PCR reaction following the manufacturer’s instructions, including lambda phage gDNA unless specified. 2 ng of Human gDNA was added to each PCR reaction of the cultured isolate. The PCR was performed as follows: 5 min at 37°C, 10 min at 95°C, 15 cycles of (10 s at 95°C, 1 min at 64°C, 1min at 60°C to 10 s at 72°C), 2 min at 72°C. The products were purified with MGI EasyDNA Clean beads (MGI, BGI-Shenzhen, China) at a 5:4 ratio and cleaned with 80% concentration ethanol according to the manufacturer’s instructions. The 2nd PCR was performed under the same regimen as the 1st PCR except for 25 cycles, and the beads-purified products from the first PCR reaction were unique dual indexed. After the 2nd PCR, products were purified following the same procedures as the 1st PCR and quantified using the Qubit dsDNA High Sensitivity assay on Qubit 3.0 (Life Technologies). PCR products of samples yielding sufficient material (&gt;; 5 ng/μl) were pooled at equimolar to a total DNA amount of 300 ng before converting to single-stranded circular DNA. DNBs-based libraries were generated from 20 μl of single-stranded circular DNA pools and sequenced on the MGISEQ-2000 platform with single-end 400 nt strategy, generating 1.8 Gb sequencing data for each sample on average.</p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Identification of HCoV-19-like reads from Massively Parallel Sequencing data</h3>
     <p id="p-30">For metatranscriptomic and hybrid capture sequencing data, total reads were first processed by Kraken v0.10.5
      <sup>
       <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
      </sup> (default parameters) with a self-build database of Coronaviridae genomes (including SARS, MERS and HCoV-19 genome sequences downloaded from GISAID, NCBI and CNGB) to efficiently identify candidate viral reads with a loose manner. These candidate reads were further qualified with fastp v0.19.5
      <sup>
       <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
      </sup> (parameters: -q 20 -u 20 - n 1 -l 50) and SOAPnuke v1.5.6
      <sup>
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
      </sup> (parameters: -l 20 -q 0.2 -E 50 -n 0.02 -5 0 -Q 2 -G -d) to remove low-quality reads, duplications and adaptor contaminations. Low-complexity reads were next filtered by PRINSEQ v0.20.4
      <sup>
       <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
      </sup> (parameters: -lc_method dust -lc_threshold 7). After the above process, HCoV-19-like reads generated from metatranscriptomics and hybrid capture method were obtained.
     </p>
     <p id="p-31">For amplicon sequencing data, SE 400 reads were first processed with fastp v0.19.5
      <sup>
       <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>
      </sup> (parameters: -q 20 -u 20 -n 1 -l 50) to remove low quality-reads and adaptor sequences. Primer sequences and the 21 nt upstream and downstream of primers within the reads were then trimmed with BAMClipper v1.1.1
      <sup>
       <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
      </sup> (Parameters: -n 4 -u 21 -d 21). Reads with low quality bases, adaptors, primers and adjacent sequences completely removed as described above were considered as clean reads for downstream analyses.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Assembling viral genome</h3>
     <p id="p-32">HCoV-19-like reads of metatranscriptomic and hybrid capture sequencing data were 
      <em>de novo</em> assembled with SPAdes (v3.14.0)
      <sup>
       <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>
      </sup> using the default settings to obtain virus genome sequences. To reduce the complexity of the assembling process, identified HCoV-19-like reads of metatranscriptomic and hybrid capture sequencing data were subsampled to the data amount greater than 100X sequencing depth for the HCoV-19 genome. For the two metatranscriptomic samples with a sequencing depth lower than 100X for the HCoV-19 genome (GZMU0014 and GZMU0030), all HCoV-19-like reads were used for assembling viral genomes.
     </p>
     <p id="p-33">Due to the uneven read coverage in amplicon sequencing of HCoV-19, virus consensus sequences of amplicon samples were generated by Pilon v1.23
      <sup>
       <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
      </sup> (parameters: --changes - vcf --changes --vcf --mindepth 1 --fix all, amb). Positions with depth less than 100X or lower five times than negative control samples were masked as ambiguous base N.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Assessment the coverage depth across the viral genome</h3>
     <p id="p-34">HCoV-19-like reads of metatranscriptomic and hybrid capture sequencing data were aligned to the HCoV-19 reference genome (GISAID accession: EPI_ISL_402119) with BWA aln (v0.7.16)
      <sup>
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
      </sup>. Duplications were identified by Picard Markduplicates (v2.10.10)(
      <a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>) with default settings. For each sample, we calculated the depth of coverage at each nucleotide position of the HCoV-19 reference genome with Samtools (v1.9)
      <sup>
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
      </sup> and scaled the values to the mean depth. For each nucleotide position, we calculated the median depth, and 20th and 80th percentiles across all samples. Read coverage and depth across the HCoV-19 reference genome were plotted by a 200-nt sliding window with the ggplot2
      <sup>
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>
      </sup> package in R (v3.6.1)
      <sup>
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </sup>.
     </p>
     <p id="p-35">Amplicon sequencing data were processed as described above, except that duplications were not removed. A heatmap was generated to visualize the viral genome coverage for all samples sequenced by the amplicon method with the pheatmap
      <sup>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </sup> package in R (v3.6.1)
      <sup>
       <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>
      </sup>. The depth at each nucleotide position was binarized, and was shown in pink if the depth was 100x and above.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Relationships between genome copies and method-dependent minimum amount of sequencing data</h3>
     <p id="p-36">HCoV-19 reads of metatranscriptomic and hybrid capture sequencing data were identified by aligning the HcoV-19-like reads to the HCoV-19 reference genome (GISAID accession: EPI_ISL_402119) with BWA in a strict manner of coverage ≥ 95% and identity ≥ 90%. For comparisons of the coverage and depth of the viral genome across samples and methods, we normalized the viral reads to total sequencing reads with HCoV-19 Reads Per Million (HCoV-19-RPM). HCoV-19-RPM for amplicon sequencing data was calculated by the same pipelines we applied for metatranscriptomic and hybrid capture sequencing data.</p>
     <p id="p-37">To estimate the minimum data requirements for genome assembling and intra-individual variation analysis, we applied gradient-based sampling to the HCoV-19 genome alignments (referred to BAM files) to each dataset using Samtools (v1.9)
      <sup>
       <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>
      </sup>. The effective genome coverage was set as 95% for all three MPS methods. Considering the distinct technologies used in different methods, we set method-dependent thresholds of effective depth as follows: 1) ≥ 10X for metatranscriptomic sequencing; 2) ≥ 20X for hybrid capture sequencing; and 3) ≥ 100X for amplicon sequencing. We next calculated the coverage and depth within each subsampled BAM file per sample to determine the minimal BAM file that could meet the above thresholds of both coverage and sequencing depth. The methoddependent minimum amount of sequencing data of each sample were estimated accordingly. We assessed the correlations between the HCoV-19 genome copies per mL in diluted samples of cultured isolates and the minimum amount of sequencing data for amplicon-and capture-based methods using Pearson correlation coefficient (R) with the function 
      <em>scatter</em> from the R package 
      <em>ggpubr</em> (v3.6.1)
      <sup>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
      </sup>.
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Consistency in variants calling performance among methods</h3>
     <p id="p-38">Except for amplicon sequencing samples, variants calling in metatranscriptomic and hybrid capture sequencing samples was performed in the previous BAM files of identified HCoV-19 reads after removing duplications from alignment output by Picard Markduplicates (
      <a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>). To accurately identify SNVs from viral sequencing data of all three methods, we first called SNVs with freebayes (v1.3.1)
      <sup>
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
      </sup> (parameters: -p 1 - q 20 -m 60 --min-coverage 10 -V) and then filtered the low-confidence SNVs with snippy-vcf_filter
      <sup>
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>
      </sup> (parameters: --minqual 100 --mincov 10 --minfrac 0.8). Remaining SNVs post filtering in VCF files generated by freebayes were annotated in HCoV-19 genome assemblies and consensus sequences with SNVeff (v4.3)
      <sup>
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>
      </sup> using default parameters.
     </p>
     <p id="p-39">Next, we calculated SNV allele frequencies and called iSNVs (intra-host Single Nucleotide Variations) for each dataset to assess the consistency of variants calling performance among three methods. We performed pysamstats v1.1.2 (
      <a href="https://github.com/ali-manfoo/pysamstats">https://github.com/ali-manfoo/pysamstats</a>) (parameters: -type variation_strand --min-baseq 20 -D 1000000) to count the number of matches, mismatches, deletions and insertions at each base, estimate nucleotide percentage and determine allele frequencies of SNVs at reference genome positions based on the HCoV-19 alignments from BAM files.
     </p>
     <p id="p-40">For amplicon sequencing data, only base positions covered by ≥100X reads were used for iSNVs calling. For metatranscriptomic and hybrid capture-based sequencing data, the thresholds of depth were set as 10X and 20X, respectively. The candidate iSNVs were further filtered for quality as follows: 1) frequency filtering, only minor alleles (frequency ≥ 5% and &lt;;50%) and major alleles (frequency ≥ 50% and ≤ 95%) were remained; 2) depth filtering, iSNVs with fewer than five forward or reverse reads were removed; and 3) strand bias filtering (not applicable to single-end reads of amplicon sequencing), iSNVs were removed if there were more than a 10-fold strand bias or a 5-fold difference between the strand bias of the variant call and that of the reference call.</p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Taxonomy of clinical samples by unbiased metatranscriptomic sequencing</h3>
     <p id="p-41">For metatranscriptomic sequencing of clinical samples, raw sequencing data of a single sequence lane (approximately 60-75 Gb per sample) was used to simultaneously assess the RNA expression patterns of human, bacteria and viruses in clinical samples from COVID-19 patients. We first used software fastp (v0.19.5)
      <sup>
       <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>
      </sup> to filter low-quality reads and remove adapter with parameters: -5 -3 -q 20 -c -l 30. After QC, we mapped high-quality reads to hg19 and removed human ribosomal RNA (rRNA) reads by SOAP2 v2.21
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>
      </sup> (parameters: - m 0 -x 1000 -s 28 -l 32 -v 5 -r 1), and the remaining RNA reads were then aligned to hg19 by HISAT2
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>
      </sup> with default settings to identify non-rRNA human transcripts as previously described. Next, we applied Kraken 2
      <sup>
       <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>
      </sup> (version 2.0.8-beta, parameters: --threads 24 --confidence 0) to assign microbial taxonomic ranks to non-human RNA reads against the large reference database MiniKraken2 (April 2019, 8GB) built from the Refseq bacteria, archaea, and viral libraries and the h38 human genome. Bracken
      <sup>
       <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>
      </sup> (Bayesian Reestimation of Abundance with Kraken) was further applied to estimate microbial relative abundances based on taxonomic ranks of reads assigned by Kraken2.
     </p>
    </div>
   </div>
   <div class="section" id="sec-22">
    <h2 class="">Data availability</h2>
    <p id="p-42">Sequencing data that support the findings of this study have been deposited in CNGB (
     <a href="https://db.cngb.org/">https://db.cngb.org/</a>) under Project accession CNP0000951 and CNP0000955, and in GISAID under accession EPI_ISL_414663, EPI_ISL_414686 to EPI_ISL_414692.
    </p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">Code availability</h2>
    <p id="p-43">The software and parameters used in data analysis can be found in Supplementary Table 6.</p>
   </div>
   <div class="section" id="sec-24">
    <h2 class="">AUTHOR CONTRIBUTIONS</h2>
    <p id="p-45">J.L., W.C. and M.X. conceived the project. X.L., J.Z, Y.W., and Y.L. sampled and processed the clinical specimen. M.X., Ji.L., M.L., and J.L. designed the experiments. L.Y. and Y.Z. developed the multiplex PCR amplicon-based sequencing method. M.L., Ji.L., Y.L, P.R. W.S., G.Y. and T.C. performed multiplex PCR and amplicon sequencing. J.L., and P.R. performed metatranscriptomic library construction and hybrid capture experiments. J.J., M.L, W.S., T.L., H.R., and H.Z. processed the sequencing data and conducted bioinformatic analyses. J.L., M.X. H.Z., J.J., M.L., and W.S. interpreted the data. M.X., J.J., M.L., and J.L. wrote and polished the manuscript. H.Z., W.S., L.Y., W.C. and Y.Z. contributed substantially to the manuscript revisions. All other authors provided useful suggestions and comments on the project and the manuscript.</p>
   </div>
   <div class="section" id="sec-25">
    <h2 class="">COMPETING INTERESTS</h2>
    <p id="p-46">ATOPlex SARS-CoV-2 Full Length Genome Panel is a proprietary product.</p>
    <p id="p-47">PCR PRIMER PAIR AND APPLICATION THEREOF</p>
    <p id="p-48">Patent applicant: MGI Tech Co.,Ltd</p>
    <p id="p-49">Name of inventor(s): Lin Yang, Ya Gao, Guodong Huang, Yicong Wang, Yuqian wang, Yanyan Zhang, Fang Chen, Na Zhong, Hui Jiang, Xun Xu.</p>
    <p id="p-50">Application number: PCT/CN2017/089195</p>
    <p id="p-51">Any inquires or requests regarding this product should be specifically addressed to Yan-yan Zhang (
     <span class="em-link">
      <span class="em-addr">zhangyanyan{at}genomics.cn</span>
     </span>).
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-44">We attribute this work to the amazing people in this land who dedicate themselves to the battle of mankind against viruses. This work is funded by the State Key Research Development Program of China (2019YFC1200501), the National Major Project for Control and Prevention of Infectious Disease in China (2018ZX10301101-004), the emergency grants for prevention and control of SARS-CoV-2 of Ministry of Science and Technology (2020YFC0841400) and Guangdong province, China (2020B111107001, 2020B111108001, 2018B020207013), Guangdong Provincial Key Laboratory of Genome Read and Write (No. 2017B030301011), Guangdong Provincial Academician Workstation of BGI Synthetic Genomics (No. 2017B090904014), and Shenzhen Engineering Laboratory for Innovative Molecular Diagnostics (DRC-SZ[2016]884). We thank China National GeneBank at Shenzhen for providing sequencing service. We appreciate all the authors who have deposited and shared genome data on GISAID, and a table with genome sequence acknowledgments can be found in Supplementary Table 7.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-2">
      <p id="p-2">
       <span class="fn-label">#</span> These authors jointly supervised this work.
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A distinct name is needed for the new coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.03.16.993584v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Organization</span>, 
          <span class="cit-name-given-names">W. H.</span>
         </span> in Available from: 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/</a> (
         <span class="cit-publ-loc">Geneva</span>: 
         <span class="cit-publ-name">WHO</span>, 
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.3" data-doi="10.1038/nature22040">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dudas</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Virus genomes reveal factors that spread and sustained the Ebola epidemic</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">544</span>, 
         <span class="cit-fpage">309</span>–
         <span class="cit-lpage">315</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nature22040</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D544%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22040%26rft_id%253Dinfo%253Apmid%252F28405027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature22040&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28405027&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.993584v2.4" data-doi="10.7554/eLife.37324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dudas</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carvalho</span>, 
          <span class="cit-name-given-names">L. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Correction: MERS-CoV spillover at the camel-human interface</span>. 
         <abbr class="cit-jnl-abbrev">Elife</abbr>
         <span class="cit-vol">7</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.37324</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.37324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.7554/eLife.37324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.993584v2.5" data-doi="10.1038/nrg.2017.88">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gardy</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Loman</span>, 
          <span class="cit-name-given-names">N. J.</span>
         </span>
         <span class="cit-article-title">Towards a genomics-informed, real-time, global pathogen surveillance system</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">9</span>–
         <span class="cit-lpage">20</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nrg.2017.88</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D19%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2017.88%26rft_id%253Dinfo%253Apmid%252F29129921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg.2017.88&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29129921&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.6" data-doi="10.1126/science.1259657">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gire</span>, 
          <span class="cit-name-given-names">S. K.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">345</span>, 
         <span class="cit-fpage">1369</span>–
         <span class="cit-lpage">1372</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.1259657</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTkvMjAyMC4wMy4xNi45OTM1ODQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.993584v2.7" data-doi="10.1038/nature22400">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grubaugh</span>, 
          <span class="cit-name-given-names">N. D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic epidemiology reveals multiple introductions of Zika virus into the United States</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">546</span>, 
         <span class="cit-fpage">401</span>–
         <span class="cit-lpage">405</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nature22400</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22400%26rft_id%253Dinfo%253Apmid%252F28538723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature22400&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28538723&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.8" data-doi="10.1126/science.aac8608">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sabir</span>, 
          <span class="cit-name-given-names">J. S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Co-circulation of three camel coronavirus species and recombination of MERS-CoVs in Saudi Arabia</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">351</span>, 
         <span class="cit-fpage">81</span>–
         <span class="cit-lpage">84</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aac8608</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac8608%26rft_id%253Dinfo%253Apmid%252F26678874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTEvNjI2OC84MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE5LzIwMjAuMDMuMTYuOTkzNTg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.9" data-doi="10.1016/S0140-6736(20)30251-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">565</span>–
         <span class="cit-lpage">574</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30251-8</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32007145&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.10" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">382</span>, 
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31978945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.11" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">579</span>, 
         <span class="cit-fpage">270</span>–
         <span class="cit-lpage">273</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015507&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.12" data-doi="10.1080/22221751.2020.1725399">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">313</span>–
         <span class="cit-lpage">319</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2020.1725399</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1725399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1725399&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.13" data-doi="10.1038/s41586-020-2008-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">579</span>, 
         <span class="cit-fpage">265</span>–
         <span class="cit-lpage">269</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015508&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.14" data-doi="10.1128/JCM.42.5.2043-2047.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Evaluation of advanced reverse transcription-PCR assays and an alternative PCR target region for detection of severe acute respiratory syndrome-associated coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-vol">42</span>, 
         <span class="cit-fpage">2043</span>–
         <span class="cit-lpage">2047</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/jcm.42.5.2043-2047.2004</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DDrosten%26rft.auinit1%253DC.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D2043%26rft.epage%253D2047%26rft.atitle%253DEvaluation%2Bof%2BAdvanced%2BReverse%2BTranscription-PCR%2BAssays%2Band%2Ban%2BAlternative%2BPCR%2BTarget%2BRegion%2Bfor%2BDetection%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome-Associated%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.42.5.2043-2047.2004%26rft_id%253Dinfo%253Apmid%252F15131168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQyLzUvMjA0MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE5LzIwMjAuMDMuMTYuOTkzNTg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.15" data-doi="10.1186/s12985-016-0479-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jonsdottir</span>, 
          <span class="cit-name-given-names">H. R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dijkman</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Coronaviruses and the human airway: a universal system for virus-host interaction studies</span>. 
         <abbr class="cit-jnl-abbrev">Virol J</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">24</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/s12985-016-0479-5</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D13%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-016-0479-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12985-016-0479-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.16" data-doi="10.1016/S0140-6736(20)30154-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">J. F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">514</span>–
         <span class="cit-lpage">523</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30154-9</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930154-9%26rft_id%253Dinfo%253Apmid%252F31986261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30154-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31986261&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.17" data-doi="10.1056/NEJMoa2001191">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Holshue</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">First Case of 2019 Novel Coronavirus in the United States</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">382</span>, 
         <span class="cit-fpage">929</span>–
         <span class="cit-lpage">936</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001191</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001191&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.18" data-doi="10.7554/eLife.35962">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McCrone</span>, 
          <span class="cit-name-given-names">J. T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Stochastic processes constrain the within and between host evolution of influenza virus</span>. 
         <abbr class="cit-jnl-abbrev">Elife</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">e35962</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D7%26rft.spage%253De35962%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.35962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.7554/eLife.35962&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.19" data-doi="10.1038/nmicrobiol.2016.151">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ni</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Intra-host dynamics of Ebola virus during 2014</span>. 
         <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">16151</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nmicrobiol.2016.151</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D1%26rft.spage%253D16151%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmicrobiol.2016.151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmicrobiol.2016.151&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.20" data-doi="10.1016/j.cell.2015.06.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>, 
          <span class="cit-name-given-names">D. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Ebola Virus Epidemiology, Transmission, and Evolution during Seven Months in Sierra Leone</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">161</span>, 
         <span class="cit-fpage">1516</span>–
         <span class="cit-lpage">1526</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2015.06.007</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1516%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.06.007%26rft_id%253Dinfo%253Apmid%252F26091036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.06.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26091036&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.21" data-doi="10.1128/MMBR.05023-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sheldon</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perales</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-article-title">Viral quasispecies evolution</span>. 
         <abbr class="cit-jnl-abbrev">Microbiol Mol Biol Rev</abbr>
         <span class="cit-vol">76</span>, 
         <span class="cit-fpage">159</span>–
         <span class="cit-lpage">216</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/MMBR.05023-11</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDomingo%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D216%26rft.atitle%253DViral%2BQuasispecies%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.05023-11%26rft_id%253Dinfo%253Apmid%252F22688811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni8yLzE1OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE5LzIwMjAuMDMuMTYuOTkzNTg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.22" data-doi="10.1038/nature22402">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Metsky</span>, 
          <span class="cit-name-given-names">H. C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Zika virus evolution and spread in the Americas</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">546</span>, 
         <span class="cit-fpage">411</span>–
         <span class="cit-lpage">415</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nature22402</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22402%26rft_id%253Dinfo%253Apmid%252F28538734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature22402&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28538734&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.23" data-doi="10.1186/s12864-019-5569-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Reliable multiplex sequencing with rare index mis-assignment on DNB-based NGS platform</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">215</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/s12864-019-5569-5</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D20%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-019-5569-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12864-019-5569-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Advanced Whole Genome Sequencing Using a Complete PCR-free Massively Parallel Sequencing (MPS) Workflow</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.25" data-doi="10.1056/NEJMc2001737">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMc2001737</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMc2001737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMc2001737&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.26" data-doi="10.1186/gb-2014-15-3-r46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wood</span>, 
          <span class="cit-name-given-names">D. E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>
         <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">R46</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/gb-2014-15-3-r46</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24580807&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.27" data-doi="10.1093/bioinformatics/bty560">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">fastp: an ultra-fast all-in-one FASTQ preprocessor</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">i884</span>–
         <span class="cit-lpage">i890</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bty560</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di884%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty560%26rft_id%253Dinfo%253Apmid%252F30423086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty560&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30423086&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.28" data-doi="10.1093/gigascience/gix120">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SOAPnuke: a MapReduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Gigascience</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">6</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/gigascience/gix120</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix120%26rft_id%253Dinfo%253Apmid%252F29300887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/gigascience/gix120&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29300887&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.29" data-doi="10.1093/bioinformatics/btr026">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmieder</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Edwards</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Quality control and preprocessing of metagenomic datasets</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">863</span>–
         <span class="cit-lpage">864</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr026</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr026%26rft_id%253Dinfo%253Apmid%252F21278185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr026&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21278185&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000288277300018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.30" data-doi="10.1038/s41598-017-01703-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Au</span>, 
          <span class="cit-name-given-names">C. H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>, 
          <span class="cit-name-given-names">D. N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kwong</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">T. L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">E. S. K.</span>
         </span>
         <span class="cit-article-title">BAMClipper: removing primers from alignments to minimize false-negative mutations in amplicon next-generation sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">1567</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-017-01703-6</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D1567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-01703-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41598-017-01703-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.31" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Biol</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1089/cmb.2012.0021</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22506599&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.32" data-doi="10.1371/journal.pone.0112963">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walker</span>, 
          <span class="cit-name-given-names">B. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">e112963</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0112963</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25409509&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.33" data-doi="10.1093/bioinformatics/btp324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">1754</span>–
         <span class="cit-lpage">1760</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp324</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19451168&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267665900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.34" data-doi="10.1093/bioinformatics/btp352">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">2078</span>–
         <span class="cit-lpage">2079</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp352</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268808600014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.35" data-doi="10.1002/wics.147.">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wickham</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">ggplot2</span>. 
         <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews: Computational Statistics</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">180</span>–
         <span class="cit-lpage">185</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%253A%2BComputational%2BStatistics%26rft.volume%253D3%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwics.147.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/wics.147.&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Team</span>, 
          <span class="cit-name-given-names">R. C.</span>
         </span>
         <span class="cit-article-title">R: A language and environment for statistical computing</span>. (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kolde</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kolde</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>
         <span class="cit-article-title">Package ‘pheatmap’</span>. 
         <abbr class="cit-jnl-abbrev">R Package</abbr>
         <span class="cit-vol">1</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kassambara</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">ggpubr:”ggplot2” based publication ready plots</span>. 
         <abbr class="cit-jnl-abbrev">R package version 0.1</abbr>
         <span class="cit-vol">6</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Garrison</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Haplotype-based variant detection from short-read sequencing</span>. 
         <abbr class="cit-jnl-abbrev">arXiv preprint</abbr>
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">1207.3907</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seemann</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> in 
         <abbr class="cit-jnl-abbrev">Snippy: fast bacterial variant calling from NGS reads</abbr> (WWW Document, 
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.41" data-doi="10.4161/fly.19695">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cingolani</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</span>. 
         <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">80</span>–
         <span class="cit-lpage">92</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.4161/fly.19695</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DFly%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DA%2Bprogram%2Bfor%2Bannotating%2Band%2Bpredicting%2Bthe%2Beffects%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%252C%2BSnpEff%253A%2BSNPs%2Bin%2Bthe%2Bgenome%2Bof%2BDrosophila%2Bmelanogaster%2Bstrain%2Bw1118%253B%2Biso-2%253B%2Biso-3.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22728672&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305965500003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.42" data-doi="10.1093/bioinformatics/btp336">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SOAP2: an improved ultrafast tool for short read alignment</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">1966</span>–
         <span class="cit-lpage">1967</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp336</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp336%26rft_id%253Dinfo%253Apmid%252F19497933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp336&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19497933&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F19%2F2020.03.16.993584.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268107100019&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.43" data-doi="10.1038/s41587-019-0201-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Paggi</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bennett</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>
         <span class="cit-article-title">Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>
         <span class="cit-vol">37</span>, 
         <span class="cit-fpage">907</span>–
         <span class="cit-lpage">915</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41587-019-0201-4</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D37%26rft.spage%253D907%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-019-0201-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41587-019-0201-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.44" data-doi="10.1186/s13059-019-1891-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wood</span>, 
          <span class="cit-name-given-names">D. E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">Improved metagenomic analysis with Kraken 2</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">257</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/s13059-019-1891-0</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1891-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13059-019-1891-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.45" data-doi="10.7717/peerj-cs.104">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Breitwieser</span>, 
          <span class="cit-name-given-names">F. P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thielen</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>
         <span class="cit-article-title">Bracken: estimating species abundance in metagenomics data</span>. 
         <abbr class="cit-jnl-abbrev">PeerJ Computer Science</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">e104</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPeerJ%2BComputer%2BScience%26rft.volume%253D3%26rft.spage%253D104e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj-cs.104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.7717/peerj-cs.104&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.993584v2.46" data-doi="10.1093/nar/gkr771">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kircher</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sawyer</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Meyer</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">40</span>, 
         <span class="cit-fpage">fe3</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkr771</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D40%26rft.spage%253Dfe3%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkr771&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
