<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/16/2020.01.15.908608.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.15.908608v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family" />
<meta name="DC.Identifier" content="10.1101/2020.01.15.908608" />
<meta name="DC.Date" content="2020-01-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I- β 2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β 2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.

IMPORTANCE Ursidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines." />
<meta name="DC.Contributor" content="Hongyu Yuan" />
<meta name="DC.Contributor" content="Lizhen Ma" />
<meta name="DC.Contributor" content="Lijie Zhang" />
<meta name="DC.Contributor" content="Xiaoying Li" />
<meta name="DC.Contributor" content="Chun Xia" />
<meta name="article:published_time" content="2020-01-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-&lt;i&gt;β&lt;/i&gt;2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and &lt;i&gt;β&lt;/i&gt;2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.&lt;/p&gt;&lt;h3&gt;IMPORTANCE&lt;/h3&gt;
&lt;p&gt;Ursidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.15.908608v1" />
<meta name="citation_id" content="2020.01.15.908608v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.15.908608" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.15.908608" />
<meta name="citation_author" content="Hongyu Yuan" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University" />
<meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author" content="Lizhen Ma" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University" />
<meta name="citation_author" content="Lijie Zhang" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University" />
<meta name="citation_author" content="Xiaoying Li" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University" />
<meta name="citation_author_institution" content="School of Basic Medical Sciences, Xinxiang Medical University" />
<meta name="citation_author" content="Chun Xia" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University" />
<meta name="citation_author_email" content="xiachun@cau.edu.cn" />
<meta name="citation_reference" content="Flajnik MF. 2018. A cold-blooded view of adaptive immunity. Nature Reviews Immunology 18." />
<meta name="citation_reference" content="Kaufman J. 2018. Unfinished Business: Evolution of the MHC and the Adaptive Immune System of Jawed Vertebrates. Annual Review of Immunology 36:383." />
<meta name="citation_reference" content="Garstka MA, Alexander F, Celie PHN, Joosten RP, Janssen GMC, Ilana B, Rieuwert H, Magda S, Lennert J, Huib O. 2015. The first step of peptide selection in antigen presentation by MHC class I molecules. Proceedings of the National Academy of Sciences of the United States of America 112:1505–1510." />
<meta name="citation_reference" content="Liang R, Sun Y, Liu Y, Wang J, Wu Y, Li Z, Ma L, Zhang N, Zhang L, Wei X. 2018. Major histocompatibility complex class I (FLA-E*01801) molecular structure in domestic cats demonstrates species-specific characteristics in presenting viral antigen peptides. Journal of Virology:JVI.01631–01617." />
<meta name="citation_reference" content="Fan S, Wu Y, Wang S, Wang Z, Jiang B, Liu Y, Liang R, Zhou W, Zhang N, Xia C. 2016. Structural and Biochemical Analyses of Swine Major Histocompatibility Complex Class I Complexes and Prediction of the Epitope Map of Important Influenza A Virus Strains. Journal of Virology 90:6625–6641." />
<meta name="citation_reference" content="Xiao J, Xiang W, Chai Y, Haywood J, Qi J, Ba L, Qi P, Wang M, Liu J, Gao GF. 2016. Diversified Anchoring Features the Peptide Presentation of DLA-88*50801: First Structural Insight into Domestic Dog MHC Class I. Journal of Immunology 197:2306." />
<meta name="citation_reference" content="Ladell K, Hashimoto M, Iglesias MC, Wilmann PG, McLaren JE, Gras S, Chikata T, Kuse N, Fastenackels S, Gostick E, Bridgeman JS, Venturi V, Arkoub ZA, Agut H, van Bockel DJ, Almeida JR, Douek DC, Meyer L, Venet A, Takiguchi M, Rossjohn J, Price DA, Appay V. 2013. A molecular basis for the control of preimmune escape variants by HIV-specific CD8+ T cells. Immunity 38:425–436." />
<meta name="citation_reference" content="Li X, Liu J, Qi J, Gao F, Li Q, Li X, Zhang N, Xia C, Gao GF. 2011. Two distinct conformations of a rinderpest virus epitope presented by bovine major histocompatibility complex class I N*01801: a host strategy to present featured peptides. Journal of virology 85:6038–6048." />
<meta name="citation_reference" content="Chu F, Lou Z, Chen Yu. Wai, Liu Y, Gao B, Zong L, Khan AH, Bell JI, Rao Z, Gao GF. 2007. First glimpse of the peptide presentation by rhesus macaque MHC class I: crystal structures of Mamu-A*01 complexed with two immunogenic SIV epitopes and insights into CTL escape. Journal of Immunology 178:944–952." />
<meta name="citation_reference" content="Yewdell JW, Haeryfar SM. 2005. Understanding presentation of viral antigens to CD8+ T cells in vivo: the key to rational vaccine design. Annual Review of Immunology 23:651–682." />
<meta name="citation_reference" content="Gray SM, Kaech SM, Staron MM. 2015. The interface between transcriptional and epigenetic control of effector and memory CD8? T-cell differentiation. Nature Reviews Immunology 261:157–168." />
<meta name="citation_reference" content="Gruta NLL, Gras S, Daley SR, Thomas PG, Rossjohn J. 2018. Understanding the drivers of MHC restriction of T cell receptors. Nature Reviews Immunology 18." />
<meta name="citation_reference" content="Marrack P, Scott-Browne JP, Dai S, Gapin L, Kappler JW. 2008. Evolutionarily conserved amino acids that control TCR-MHC interaction. Annual Review of Immunology 26:171–203." />
<meta name="citation_reference" content="P Anton VDM, Omer D. 2011. Mechanisms for T cell receptor triggering. Nature Reviews Immunology 11:47–55." />
<meta name="citation_reference" content="Huang J. 2011. Two-Stage Cooperative T Cell Receptor-Peptide Major Histocompatibility Complex-CD8 Trimolecular Interactions Amplify Antigen Discrimination. Immunity 34:13–23." />
<meta name="citation_reference" content="Kirksey TJ, Pogue-Caley RR, Frelinger JA, Collins EJ. 1999. The structural basis for the increased immunogenicity of two HIV-reverse transcriptase peptide variant/class I major histocompatibility complexes. Journal of Biological Chemistry 274:37259." />
<meta name="citation_reference" content="Butler N, Theodossis A, Ai, Dunstone M, Nastovska R, Ramarathinam S, Rossjohn J, Purcell A, Perlman S. 2008. Structural and Biological Basis of CTL Escape in Coronavirus-Infected Mice. Journal of Immunology 180:3926–3937." />
<meta name="citation_reference" content="Yao S, Liu J, Qi J, Chen R, Zhang N, Liu Y, Wang J, Wu Y, Gao GF, Xia C. 2016. Structural Illumination of Equine MHC Class I Molecules Highlights Unconventional Epitope Presentation Manner That Is Evolved in Equine Leukocyte Antigen Alleles. Journal of Immunology 196:1943–1954." />
<meta name="citation_reference" content="Zhang J, Yong C, Jianxun Q, Feng G, Yanjie L, Jun L, Xuyu Z, Jim K, Chun X, Gao GF. 2012. Narrow groove and restricted anchors of MHC class I molecule BF2*0401 plus peptide transporter restriction can explain disease susceptibility of B4 chickens. Journal of Immunology 189:4478–4487." />
<meta name="citation_reference" content="Chen Z, Zhang N, Qi J, Chen R, Dijkstra JM, Li X, Wang Z, Wang J, Wu Y, Xia C. 2017. The Structure of the MHC Class I Molecule of Bony Fishes Provides Insights into the Conserved Nature of the Antigen-Presenting System. Journal of Immunology 199:3668." />
<meta name="citation_reference" content="Krause J, Unger T, Noçon A, Malaspinas AS, Kolokotronis SO, Stiller M, Soibelzon L, Spriggs H, Dear PH, Briggs AW. 2008. Mitochondrial genomes reveal an explosive radiation of extinct and extant bears near the Miocene-Pliocene boundary. Bmc Evolutionary Biology 8:220–220." />
<meta name="citation_reference" content="Li R, Fan W, Tian G, Zhu H, He L, Cai J, Huang Q, Cai Q, Li B, Bai Y, Zhang Z, Zhang Y, Wang W, Li J, Wei F, Li H, Jian M, Li J, Zhang Z, Nielsen R, Li D, Gu W, Yang Z, Xuan Z, Ryder OA, Leung FC-C, Zhou Y, Cao J, Sun X, Fu Y, Fang X, Guo X, Wang B, Hou R, Shen F, Mu B, Ni P, Lin R, Qian W, Wang G, Yu C, Nie W, Wang J, Wu Z, Liang H, Min J, Wu Q, Cheng S, Ruan J, Wang M, et al. 2009. The sequence and de novo assembly of the giant panda genome. Nature 463:311." />
<meta name="citation_reference" content="Pan HJ, Wan QH, Fang SG. 2008. Molecular characterization of major histocompatibility complex class I genes from the giant panda (Ailuropoda melanoleuca). Immunogenetics 60:185–193." />
<meta name="citation_reference" content="Ying Z, Sun DD, Ge YF, Yu B, Chen YY, Wan QH. 2013. Isolation and characterization of class I MHC genes in the giant panda (Ailuropoda melanoleuca). Chinese Science Bulletin 58:2140–2147." />
<meta name="citation_reference" content="Zhu Y, Wan Q-H, Yu B, Ge Y-F, Fang S-G. 2013. Patterns of genetic differentiation at MHC class I genes and microsatellites identify conservation units in the giant panda. BMC Evolutionary Biology 13:227." />
<meta name="citation_reference" content="Jin Y, Zhang X, Ma Y, Qiao Y, Liu X, Zhao K, Zhang C, Lin D, Fu X, Xu X. 2017. Canine distemper viral infection threatens the giant panda population in China. Oncotarget 8:113910–113919." />
<meta name="citation_reference" content="Gao FS, Hu GX, Xia XZ, Gao YW, Bai YD, Zou XH. 2009. Isolation and identification of a canine coronavirus strain from giant pandas (Ailuropoda melanoleuca). Journal of Veterinary Science 10:261." />
<meta name="citation_reference" content="Decaro N, Mari V, Elia G, Lanave G, Dowgier G, Colaianni ML, Martella V, Buonavoglia C. 2015. Full-length genome analysis of canine coronavirus type I. Virus Res 210:100–105." />
<meta name="citation_reference" content="Nianzhi Z, Jianxun Q, Sijia F, Feng G, Jun L, Xiaocheng P, Rong C, Qirun L, Zhaosan C, Xiaoying L. 2011. Crystal structure of swine major histocompatibility complex class I SLA-1 0401 and identification of 2009 pandemic swine-origin influenza A H1N1 virus cytotoxic T lymphocyte epitope peptides. Journal of Virology 85:11709–11724." />
<meta name="citation_reference" content="Chih LY, Zhenjun C, Burrows SR, Purcell AW, James MC, Jamie R, Stephanie G. 2012. The energetic basis underpinning T-cell receptor recognition of a super-bulged peptide bound to a major histocompatibility complex class I molecule. Journal of Biological Chemistry 287:12267–12276." />
<meta name="citation_reference" content="Nivarthi UK, Stephanie G, Lars KN, Richard B, Lucet IS, Miles JJ, Tracy SL, Purcell AW, Bowden DS, Margaret H. 2014. An extensive antigenic footprint underpins immunodominant TCR adaptability against a hypervariable viral determinant. Journal of Immunology 193:5402–5413." />
<meta name="citation_reference" content="Stephanie G, Wilmann PG, Zhenjun C, Hanim H, Chih LY, Lars KN, Purcell AW, Burrows SR, James MC, Jamie R. 2012. A structural basis for varied αβ TCR usage against an immunodominant EBV antigen restricted to a HLA-B8 molecule. Journal of Immunology 188:311." />
<meta name="citation_reference" content="Turner SJ, Doherty PC, James MC, Jamie R. 2006. Structural determinants of T-cell receptor bias in immunity. Nature Reviews Immunology 6:883–894." />
<meta name="citation_reference" content="Gao GF, Tormo J, ., Gerth UC, Wyer JR, Mcmichael AJ, Stuart DI, Bell JI, Jones EY, Jakobsen BK. 1997. Crystal structure of the complex between human CD8alpha(alpha) and HLA-A2. Nature 387:630–634." />
<meta name="citation_reference" content="Li Y, Yin Y, Mariuzza RA. 2013. Structural and biophysical insights into the role of CD4 and CD8 in T cell activation. Frontiers in Immunology 4:206." />
<meta name="citation_reference" content="Zhang W, Yang S, Shan T, Hou R, Liu Z, Li W, Guo L, Wang Y, Chen P, Wang X, Feng F, Wang H, Chen C, Shen Q, Zhou C, Hua X, Cui L, Deng X, Zhang Z, Qi D, Delwart E. 2017. Virome comparisons in wild-diseased and healthy captive giant pandas. Microbiome 5:90." />
<meta name="citation_reference" content="Li D, Ling Z, Hengmin C, Shanshan L, Shengtao F, Zhijun Y, Yuancheng Z, Tiecheng W, Jun Q, Xianzhu X. 2014. Influenza A(H1N1)pdm09 virus infection in giant pandas, China. Emerging Infectious Diseases 20:480–483." />
<meta name="citation_reference" content="Castro CD, Luoma AM, Adams EJ. 2015. Coevolution of T-cell receptors with MHC and non-MHC ligands. Immunological Reviews 267:30–55." />
<meta name="citation_reference" content="Tobita T, Oda M, Morii H, Kuroda M, Yoshino A, Azuma T, Kozono H. 2003. A role for the P1 anchor residue in the thermal stability of MHC class II molecule I-A b. Immunology Letters 85:47–52." />
<meta name="citation_reference" content="Parkin S, Hope H. 1998. Macromolecular Cryocrystallography: Cooling, Mounting, Storage and Transportation of Crystals. Journal of Applied Crystallography 31:945–953." />
<meta name="citation_reference" content="Otwinowski Z, Minor W. 1997. Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol 276:307–326." />
<meta name="citation_reference" content="Emsley P, Cowtan K. 2004. Coot: model-building tools for molecular graphics. Acta crystallographica Section D, Biological crystallography 60:2126–2132." />
<meta name="citation_reference" content="Adams PD, Grosse-Kunstleve RW, Hung LW, Ioerger TR, McCoy AJ, Moriarty NW, Read RJ, Sacchettini JC, Sauter NK, Terwilliger TC. 2002. PHENIX: building new software for automated crystallographic structure determination. Acta crystallographica Section D, Biological crystallography 58:1948–1954." />
<meta name="citation_reference" content="Laskowski RA, Moss DS, Thornton JM. 1993. Main-chain bond lengths and bond angles in protein structures. Journal of Molecular Biology 231:1049–1067." />
<meta name="citation_reference" content="Thompson JD, Gibson TJ, Plewniak F, ., Jeanmougin F, ., Higgins DG. 1997. The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res. Nucleic Acids Research 25:4876–4882." />
<meta name="citation_reference" content="Patrice G, Xavier R, Emmanuel C. 2003. ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins. Nucleic Acids Research 31:3320–3323." />
<meta name="twitter:title" content="Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I- β 2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β 2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.

IMPORTANCE Ursidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines." />
<meta name="og-title" property="og:title" content="Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I- β 2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β 2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.

IMPORTANCE Ursidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.15.908608v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.15.908608v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.15.908608v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1107443" />
    <title>Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"amWiXCJFtArMiXWSjrCuelvT0CRR6wbhnGah5k52eLc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1107443","apath":"\/biorxiv\/early\/2020\/01\/16\/2020.01.15.908608.atom","pisa":"biorxiv;2020.01.15.908608v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.15.908608v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.15.908608v1","title":"Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family","identifier":"node\/1107443"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pVu9va2AqZYaNzjZpFJDSRT8fOv4QYmbdCEpzO5Vy3b++5dTbcOKeEdmz6JHrCaQnOOwoNEITukq9IQkUCli8yh\\\\\\\/\\\\\\\/gdhkTAISNxEBlURLPJNC5fo4tGPVpWKPtAmpOltyGAF1v1+R\\\\\\\/mR1GaGjcL7X7NL5FBsa8zGLW4UkBQAJjymIEdzr76yzzyzLsRofzTPmYTe4inn4v6nwFISJksA+vAS7rQpcL5qAAfTg5mS7fIXszsMi+0+nNJyTJ6VZ52orrxZ4jpcPk\\\\\\\/YMHinl\\\\\\\/dDrVAPSyBYhZ98K4mbcOy2pnigypfpMJdeO60mFikBGBcZMvpP2BJ+VO\\\\\\\/yRk5SCLqk4PyBrgXVBEisiH6Y7PIgtTytWgIEP9lieytuXr+jnKLX9vTt2A4CswWLHdug5Ua3slUcAz+5F71UIeNsrD2i7hdIqIPsmLHbbY8ESvmJPodtgLL8qZt1NnWkF4J8r1Q88xq1gV2RyRcparA07ONlPeklAWORatuTTUJPKl+vJiaaMOZs+4hEbW7me0aDZmfUvyQFL2KOESwM9Tmfcjf9\\\\\\\/PFToLqXmo28SetOzwDvePKaTaJYJCVUnA+SmzJ8iLA4Xm6eVNbhF+ucg\\\\\\\/p724N2KZyuONoIxVcEDzP2MMls7ZE1cP125+oD43J+6yIt8Rxbm+y7oVyK8PGaeRlcaFclzBaNUPqaO44Zr3WA4zZMsqHySAjiQPcfn\\\\\\\/pBGJKt8YFL1EEDzMOSjussqO7GqaUykmPiYbGk5anM7Byl7oymyjy9AH3AnP0SMEiY48zRoCK4UNozLxoYVy4LCT4TXk0uRDau7E9kY17kgOJss3ZgSBn5bnn0ASlCI\\\\\\\/ns\\\\\\\/UgU9GDpmF1CIi5qbKYHHX4B3ssWbPtfqzAxjpCWKVvNF6aiDqozh+1UChNomwI6UvI781CVBEgKyi+AMHfI1bhKHW+VSdGti+6pO5PZBDWixygx4QiktowDjxdc8dhw+ncw9D1IOXMWE0FRnLAHRQHNv8446IG\\\\\\\/y5NOWUyO7hUGb7LcB754GcJkTJFbPPDXFRBlY5UmO38zAfU1KiBoylcb9zZ8jt9p5oFMkIXOlle7FpMWxDhdD1IH5fU5oYapCBf3edyGt49XORkxB9JgtKGvaCknhc5BR05TE0eE1n57yOquA8R21QqsjiTzYesuiuklq399ipK4tjsShsjcpiXOiRyl8YkU1jncU0g4BabiaZ\\\\\\\/UvcFHl6jWVHnQhthztDMi\\\\\\\/N6LdMK50cf3B35DsIGYP1Krb+pbREsgZzsZfwVUx5DTQXlcf8oz5cr3bRniTMBX8C7xoSYZ2MsyF2Uh4SQGTZsDwnvu8QkaHC+djzXaC8jn3yVP\\\u0022,\\\u0022iv\\\u0022:\\\u0022638U52XC42XauakIGuRNSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002249c29c63c80da06047a98a45fd0ad38fd74249a0ca8a6a9c8ca3f8005de73e59\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.15.908608v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1107443 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.15.908608v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-S2KFJVOXDtKjyyh3N2YmMzl_e8x6wOqbLboS57w64Ak" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107443" id="node1107443" data-pisa="biorxiv;2020.01.15.908608v1" data-pisa-master="biorxiv;2020.01.15.908608" data-apath="/biorxiv/early/2020/01/16/2020.01.15.908608.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hongyu</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lizhen</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lijie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Xiaoying</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chun</span> <span class="nlm-surname">Xia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.15.908608 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1107443" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hongyu Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University</span>, Haidian District, Beijing 100193, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyu%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyu%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lizhen Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University</span>, Haidian District, Beijing 100193, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lizhen%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALizhen%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lijie Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University</span>, Haidian District, Beijing 100193, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijie%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijie%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xiaoying Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University</span>, Haidian District, Beijing 100193, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>School of Basic Medical Sciences, Xinxiang Medical University</span>, Xinxiang, Henan, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoying%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoying%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chun Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, China Agricultural University</span>, Haidian District, Beijing 100193, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChun%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xiachun@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiachun@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.15.908608v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1107443" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1107443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.15.908608v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1107443" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1107443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.15.908608v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1107443" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1107443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.15.908608v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1107443" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1107443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.15.908608v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1107443" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1107443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">The viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-<em>β</em>2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and <em>β</em>2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>IMPORTANCE</strong> Ursidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-4">The cytotoxic T lymphocytes (CTLs) and their immune responses are common to various genera of jawed vertebrates. Common CTLs include CD8<sup>+</sup> T cells, which recognize only antigens presented by classical major histocompatibility complex class I (MHC-I) molecules (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). MHC-I genes are expressed in most nucleated cells and present antigens derived mostly from intracellular viruses, parasites, and tumors (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Structural studies of MHC-I molecules have shown that the complex is generally composed of an MHC-I, <em>β</em>2-microglobulin (<em>β</em>2m), and peptide (pMHC-I) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). The MHC-I contains α1, α2 and α3 domains in its extracellular region, with the α1 and α2 domains forming the pMHC-I peptide-binding groove (PBG) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Antigen peptides 8-11 amino acids in length are loaded onto the PBG and subsequently recognized by TCR expressed on the surface of CD8<sup>+</sup> T cells (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Interactions between the pMHC-I complex and TCR form the first signal for the activation of antigen-specific CD8<sup>+</sup> T cells (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). The direct interaction between TCR and pMHC-I demonstrates the MHC-I-restriction nature of CD8<sup>+</sup> T cell recognition and indicates a close evolutionary relationship between pMHC-I and TCR (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). It is widely accepted that in addition to direct interaction with TCR, MHC-I engagement of the CD8 coreceptor is required for the functional activation of CD8<sup>+</sup> T cells (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). The transmembrane glycoprotein CD8 molecules bind to pMHC-I and recruit the Src tyrosine kinase p65lck (Lck) to the TCR-pMHC-I complex, resulting in the assembly of a TCR-pMHC-I-CD8 ternary complex (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Therefore, the presentation of various endogenous peptides by the pMHC-I complex is key in determining whether the antiviral CTL immune response is initiated.</p><p id="p-5">The structures and functions of pMHC-I complexes that bind viral peptides or peptides have been investigated in humans (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>), mice (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), monkeys (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>), cattle (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>), pigs (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>), horses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>), dogs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>), cats (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>), chickens (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>), and a bony fish (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). The polymorphism of MHC-I molecules causes variation in PBGs in various mammalian pMHC-I complexes, which triggers various types of CTL immune responses. Although there are many reports of mammalian pMHC-I complexes, no study has yet been conducted on the pMHC-I structure in the bear family (Ursidae), which includes the giant panda. The bear family includes 5 genera and 8 species. Classical MHC-I genes and polymorphisms in the giant panda (<em>Ailuropoda melanoleuca</em>), brown bears and the Asiatic black bear have been reported. Phylogenetic studies indicate that although the giant panda belongs to the bear family, divergence between the giant panda and other bears occurred approximately &gt;17 million years ago (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). The giant panda has an estimated population size of approximately 2500 individuals worldwide, and its classical MHC-I/II genes are located on chromosome 9q (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). The large genetic region of MHC-I is believed to play critical roles in CTL immunity. In addition, the giant panda MHC-I molecules represent an important model for understanding the structural and functional characteristics of these antiviral complexes in the bear family.</p><p id="p-6">Several studies have been performed to identify giant panda MHC-I genes. In early work, three classical MHC-I genes (including <em>Aime-128</em>) were identified in the giant panda, and the <em>Aime-128</em> gene was proven to include ten conserved amino acids critical for viral antigenic peptide binding as described by human leukocyte antigen I (HLA-I) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). Subsequent genetic analysis identified six panda MHC-I genes, four of which are classical MHC-I genes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). High levels of genetic variation have been demonstrated in the panda MHC-I molecules (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>), which is consistent with the selective advantage of MHC polymorphisms when encountering numerous types of pathogens (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>). Pandas appear to be highly susceptible to zoonoses. Cases of canine distemper virus (CDV) and canine coronavirus (CCV) infections have occurred in panda populations (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). Therefore, it is important to elucidate key molecules, such as the pMHC-I complex, and the antiviral CTL immune response of the panda, to aid vaccine development and the development of effective treatments for various diseases. In this report, the panda was chosen as a representative species of the bear family, and its pMHC-I structure (aka pAime-128) was determined by X-ray crystal diffraction. The PBG of panda pMHC-I has distinct features, from the amino acid sequence to the three-dimensional (3D) structure, and pAime-128 also reflects the features of the bear family. The findings reveal specific characteristics of the giant panda pMHC-I structure, greatly enhancing the understanding of MHC-I-related antiviral immunity in the bear family.</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><div id="sec-4" class="subsection"><h3>Structural Framework of Panda pAime-128</h3><p id="p-7">To verify the binding of these viral peptides to the giant panda pMHC-I (pAime-128), <em>in vitro</em> refolding experiments were carried out. The results revealed that four peptides can form stable pAime-128 complexes (<strong>Supplementary Fig 1 and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). The complex of Aime-128 and Aime-<em>β</em>2m with the CCV-NGY9 peptide was crystallized in the P4<sub>3</sub>2<sub>1</sub>2 orthorhombic space group with a resolution of 2.68 Å (<strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>). The viral CCV-NGY9 peptide (CCV-NGY9) was selected from the CCV spike protein (S protein), which is among the structural proteins encoded by ORF2 (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). Similar to the S proteins of other coronaviruses, the S protein of CCV plays a fundamental role in the interaction with the cellular receptor and induces neutralizing antibodies in the natural host (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Two complexes of pAime-128, molecule 1 (aka M1) and molecule 2 (aka M2), were found in one asymmetric unit. The root-mean-square deviation (RMSD) of the two molecules of Aime-128 was 0.584 Å. Because of the high consistency between M1 and M2, the following analysis of pAime-128 was based mainly on M1 (i.e., pAime-128 is M1 unless otherwise specified). The Aime-128 H chain contains the α1 (residues 1 to 90), α2 (residues 91 to 182) and α3 (residues 183 to 275) domains, of which the α1 and α2 domains constitute the PBG (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1A</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">B</a></strong>). The α1 and α2 domains of the Aime-128 can be divided into two portions: one portion (α1, residues 57 to 84; α2, residues 138 to 150 and 152 to 176) forms helices located at the top of PBG, and the remaining portion (residues 3 to 13, 20 to 28, 31 to 37, 46 to 47, 93 to 103, 110 to 118, 121 to 126, and 133 to 135) forms an eight-stranded <em>β</em>-sheet platform at the bottom, named A to H. The CCV-NGY9 is loaded onto PBG of pAime-128 (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1B</a></strong>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1112334/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1112334/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1112334" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 1</span> <p id="p-8" class="first-child">Peptides refolded with the Aime-128 and the Aime-<em>β</em><sub>2</sub>m.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1112323/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1112323/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1112323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 2</span> <p id="p-9" class="first-child">X-ray diffraction data processing and refinement statistics.</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of pAime-128 complex. (A) Overview of M1 and M2 in an asymmetric unit of pAime-128 complex. Aime-128 M1 and M2 are indicated in cyan and yellow, respectively. (B) The formation of the CCV-NGY9 peptide in PBG of pAime-128. (C) Interactions between Aime-128 and Aime-&#x3B2;2m. Aime-128 is shown in cyan and Aime-&#x3B2;2m is shown in magenta. The hydrogen bonds formed between Aime-128 and Aime-&#x3B2;2m are shown as yellow dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure of pAime-128 complex. (&lt;strong&gt;A&lt;/strong&gt;) Overview of M1 and M2 in an asymmetric unit of pAime-128 complex. Aime-128 M1 and M2 are indicated in cyan and yellow, respectively. (&lt;strong&gt;B&lt;/strong&gt;) The formation of the CCV-NGY9 peptide in PBG of pAime-128. (&lt;strong&gt;C&lt;/strong&gt;) Interactions between Aime-128 and Aime-&lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. Aime-128 is shown in cyan and Aime-&lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m is shown in magenta. The hydrogen bonds formed between Aime-128 and Aime-&lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m are shown as yellow dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F1.medium.gif" width="369" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F1.medium.gif" width="369" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIG 1</span> <p id="p-10" class="first-child">Structure of pAime-128 complex. (<strong>A</strong>) Overview of M1 and M2 in an asymmetric unit of pAime-128 complex. Aime-128 M1 and M2 are indicated in cyan and yellow, respectively. (<strong>B</strong>) The formation of the CCV-NGY9 peptide in PBG of pAime-128. (<strong>C</strong>) Interactions between Aime-128 and Aime-<em>β</em><sub>2</sub>m. Aime-128 is shown in cyan and Aime-<em>β</em><sub>2</sub>m is shown in magenta. The hydrogen bonds formed between Aime-128 and Aime-<em>β</em><sub>2</sub>m are shown as yellow dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Further analysis showed that strands and loops of Aime-<em>β</em>2m broadly interact with Aime-128, and the numbers and patterns of hydrogen bonds formed in these interactions were similar to those of other mammalian MHC I molecules. Analysis revealed that a depth of Aime-β2m in Aime-128 up to 1351.1 Å<sup>2</sup> (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig 1C</a></strong>). In addition, the hydrogen bonds involved in the interactions between Aime-128 and Aime-<em>β</em>2m were found to differ from those in humans. Aime-<em>β</em>2m forms only 15 hydrogen bonds with Aime-128, whereas in HLA-B*5101, <em>β</em>2m can form 25 hydrogen bonds with MHC-I (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>), which implies the interactions are weaker than humans (<strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig 1C</a></strong>). Thus, the main function of <em>β</em>2m is to stabilize MHC-I, even though the same <em>β</em>2m can interact with MHC-I in various ways. The main role of β2m is to ensure the stable function of pMHC-I, and the manner of its interaction with MHC-I is thought to be able to promote side effects such as <em>β</em>2m dissociation and even diseases (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>).</p></div><div id="sec-5" class="subsection"><h3>The Unique Pockets for Antigen-binding Peptide Found in pAime-128</h3><p id="p-12">The amino acids in PBG form six functional pockets, named pockets A to F, to accommodate residues of the CCV-NGY9 peptide (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2</a></strong>). Pocket A in pAime-128 consists of residues Met<sup>5</sup>, Tyr<sup>7</sup>, Tyr<sup>59</sup>, Arg<sup>62</sup>, Asn<sup>63</sup>, Tyr<sup>159</sup>, Glu<sup>163</sup>, Trp<sup>167</sup> and Tyr<sup>171</sup> (<strong><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a></strong>). P1-Asn of CCV-NGY9 was inserted into pocket A. Based on the 3D structures of pMHC-I in different species, Asn<sup>63</sup> and Glu<sup>163</sup> of Aime-128 are rare among all of the known class I molecules, i.e., these residues are found in only certain alleles, such as HLA-B*5101 and HLA-B*2705 (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a></strong>). Due to the presence of residue Asn<sup>63</sup> in Aime-128, pocket A of this molecule is considered to be tighter than that of the other pMHC-I complexes (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a></strong>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1112322/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1112322/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1112322" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 3</span> <p id="p-13" class="first-child">Hydrogen bonds and van der Waals interactions between peptides and Aime-128</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure-based amino acid sequence alignment of Aime-128 and other representative mammalian MHC-I molecules with available crystal structures. Cylinders indicate &#x3B1; helices, and black arrows above the alignment indicate &#x3B2; strands. The amino acid identities of different domains between Aime-128 and the listed MHC-I molecules are given on the right-hand side. The residue positions contributing to Aime-128 pockets are highlighted by pocket-specific colored shading." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure-based amino acid sequence alignment of Aime-128 and other representative mammalian MHC-I molecules with available crystal structures. Cylinders indicate &#x3B1; helices, and black arrows above the alignment indicate &lt;em&gt;&#x3B2;&lt;/em&gt; strands. The amino acid identities of different domains between Aime-128 and the listed MHC-I molecules are given on the right-hand side. The residue positions contributing to Aime-128 pockets are highlighted by pocket-specific colored shading.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F2.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F2.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 2</span> <p id="p-14" class="first-child">Structure-based amino acid sequence alignment of Aime-128 and other representative mammalian MHC-I molecules with available crystal structures. Cylinders indicate α helices, and black arrows above the alignment indicate <em>β</em> strands. The amino acid identities of different domains between Aime-128 and the listed MHC-I molecules are given on the right-hand side. The residue positions contributing to Aime-128 pockets are highlighted by pocket-specific colored shading.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural analysis and comparison of the six pockets of pAime-128 complex. The six pockets are shown in surface charge representation (blue, positively charged; red, negatively charged; white, nonpolar). The pocket residues were determined based on interaction with peptide ligand as indicated by CCP4 software and on our visual inspection of pocket continuity. The residues in PBG of pAime-128 complex are shown in stick form. The residues of CCV-NGY9 are shown as cyan sticks, and hydrogen bonds between peptide and pockets are shown as yellow dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Structural analysis and comparison of the six pockets of pAime-128 complex. The six pockets are shown in surface charge representation (blue, positively charged; red, negatively charged; white, nonpolar). The pocket residues were determined based on interaction with peptide ligand as indicated by CCP4 software and on our visual inspection of pocket continuity. The residues in PBG of pAime-128 complex are shown in stick form. The residues of CCV-NGY9 are shown as cyan sticks, and hydrogen bonds between peptide and pockets are shown as yellow dashed lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F3.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F3.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 3</span> <p id="p-15" class="first-child">Structural analysis and comparison of the six pockets of pAime-128 complex. The six pockets are shown in surface charge representation (blue, positively charged; red, negatively charged; white, nonpolar). The pocket residues were determined based on interaction with peptide ligand as indicated by CCP4 software and on our visual inspection of pocket continuity. The residues in PBG of pAime-128 complex are shown in stick form. The residues of CCV-NGY9 are shown as cyan sticks, and hydrogen bonds between peptide and pockets are shown as yellow dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Residues Tyr<sup>7</sup>, Tyr<sup>9</sup>, Ala<sup>24</sup>, Met<sup>45</sup>, Asn<sup>63</sup>, Ile<sup>66</sup>, Ala<sup>67</sup> and His<sup>99</sup> form pocket B of PBG in pAime-128 (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a></strong>). P2-Gly of CCV-NGY9 is tethered by hydrogen bonds to residues in PBG and fits into the small pocket B. In most class I molecules, the residues at positions 63 and 66 are Glu and Lys, respectively, which are both charged residues. The residue at position 99 of Aime-128 is His instead of the conserved Tyr residue; both of these residues have a large side chain, but they have different charges (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2</a></strong>). Pocket B of pAime-128 extends deep under the α1 helix and touches the Met45 residue, which is one of the most polymorphic residues in mammalian MHC-I molecules (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3</a></strong>).</p><p id="p-17">Pocket C of pAime-128 is composed of residues Asn<sup>70</sup>, Phe<sup>74</sup>, Trp<sup>97</sup> and Leu<sup>116</sup> and shows an obvious negative charge, and none of these residues are highly conserved among known mammalian class I molecules (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2</a></strong>). Pocket C is wide but shallow. P5-Phe and P6-Phe of CCV-NGY9 are located in this pocket, and no hydrogen bonds exist between the C pocket and the peptide, only van der Waals forces. The side chain of P6-Phe extends outward into the solvent and may play a role in TCR docking (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3</a></strong>).</p><p id="p-18">Pocket D in pAime-128 accommodates the side chains of P3 and P4 residues. The pocket consists of residues Ile<sup>66</sup>, Trp<sup>97</sup>, His<sup>99</sup>, Ser<sup>114</sup>, Glu<sup>152</sup>, Arg<sup>155</sup>, Tyr<sup>156</sup> and Tyr<sup>159</sup> (<strong><a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a></strong>). In the peptide CCV-NGY9, the residue at position 3 is Tyr, which has a large side chain and, as a result, forms two hydrogen bonds and many van der Waals interactions with pocket D (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3</a></strong>). The side chain of P3 is inserted into pocket D and forms hydrogen bonds with Glu<sup>152</sup> and Arg<sup>155</sup>. Notably, Glu<sup>152</sup> and Arg<sup>155</sup> are not conserved in most mammalian class I molecules, and both of these residues are charged (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2</a></strong>). P3-Tyr is primarily surrounded by the residues Trp<sup>97</sup>, His<sup>99</sup>, Tyr<sup>156</sup> and Tyr<sup>159</sup>, which have large side chains and provide a clear boundary to pocket D. The small residue Ser<sup>114</sup> provides enough space in the pocket to accommodate the large side chain of P3-Tyr (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig 3</a></strong>). The side chain of P4-Asn extends outward into the solvent for potential recognition by TCRs. Different formations of hydrogen bonds between the PBG and the peptide were also observed for peptide residues P3-Tyr and P4-Asn.</p><p id="p-19">Pocket E is composed of residues Ser<sup>114</sup>, Gln<sup>115</sup>, Leu<sup>116</sup>, Trp<sup>133</sup>, Trp<sup>147</sup> and Glu<sup>152</sup> (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a></strong>). The pocket is obviously negatively charged and located relatively deep beneath the α2 helix, and residues Ser<sup>114</sup>, Gln<sup>115</sup> and Leu<sup>116</sup> constitute the bottom of platform. P7-Ser of CCV-NGY9 is inserted into pocket E (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig 3</a></strong>).</p><p id="p-20">Pocket F of pAime-128 PBG consists of the highly conserved residues Ala<sup>117</sup>, Tyr<sup>118</sup>, Tyr<sup>123</sup>, Ile<sup>124</sup>, Thr<sup>143</sup>, Lys<sup>146</sup> and Trp<sup>147</sup>, as well as the poorly conserved residues Ala<sup>73</sup>, Phe<sup>74</sup>, Asp<sup>77</sup>, Thr<sup>80</sup>, Ile<sup>95</sup> and Leu<sup>116</sup> (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 2</a></strong>). Among these residues, Leu116 is not present in any other annotated MHC class I molecule deposited in the Protein Data Bank (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>). Pocket F is therefore narrow but deep enough to accommodate large residues, and residues P8-Thr and P9-Phe are located in this pocket (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig 3</a></strong>). The anchor residue P9-Phe is similar to the anchor residues of HLA-A*01, HLA-B*35, and HLA-B*57, i.e., large residues with an aromatic ring (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). The aromatic ring in CCV-NGY9 is held in close contact with residues in pocket F by strong hydrogen bonds and van der Waals contacts (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig 3</a></strong>).</p></div><div id="sec-6" class="subsection"><h3>Comparison of panda Aime-128 to Other Known MHC-I Molecules</h3><p id="p-21">The amino acid homology of MHC-I molecules in giant pandas is approximately 70-90%, but the homology of the peptide-binding domains composed of the α1 and α2 regions is low (<strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig 3</a></strong>). These results suggest that the antigen-peptide presenting profiles of the panda individuals are different, especially the homology of the α1 region, which can be as low as approximately 57%; thus, the difference is large. The results also show the potential differences in pMHC-I complexes among the bear family. It can be concluded that classical MHC-I binding antigen-peptide profiles of the bear family are diverse.</p><p id="p-22">Structure-based amino acid sequence analysis implied that MHC-I molecules may form different PBGs in pandas. However, the amino acid composition of the A, B, C, D, E and F pockets in the PBG was not identical among pandas, and the PBG regions of the pandas differed extensively from those of other members of the bear family (<strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig 3</a></strong>). Of the ten conserved amino acids in the α1 and α2 domains of Aime-128 predicted to be critical for peptide binding (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>), only eight (Met<sup>5</sup>, Tyr<sup>7</sup>, Tyr<sup>59</sup>, Tyr<sup>84</sup>, Thr<sup>143</sup>, Lys<sup>146</sup>, Tyr<sup>159</sup> and Tyr<sup>171</sup>) were confirmed in our study to have direct contact with the CCV-NGY9 peptide (<strong><a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a></strong>). In addition, the amino acid composition of PBGs among other genera and members of the bear family is not completely conserved, so there are some differences in pockets A to F. There are also great differences among giant pandas; other members of the bear family; other mammals; such as humans and mice; and nonmammals, such as the bony fish grass carp (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig 3</a></strong>).</p></div><div id="sec-7" class="subsection"><h3>Validation of Anchor Sites and Binding Motif in pAime-128</h3><p id="p-23">The peptide binding to pAime-128 PBG used in a featured mode with a central bulge, as revealed by unambiguous electron density (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a></strong>). Extensive comparison of pAime-128 with other pMHC-I structures deposited in the Protein Data Bank showed that the main chain of the CCV-NGY9 peptide has a characteristic conformation similar to that presented by HLA-B*0801 (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a></strong>). Ternary TCR-peptide-HLA-B*0801 structure of epitope HCV-HSK9 (HSKKKCDEL, hepatitis C virus-derived) showed that the protruding residue Lys-4 in HCV-HSK9 plays important roles in TCR recognition (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). In pAime-128, the side chain of P6-Phe pointed upward for solvent exposure, similar to the side chain of Lys-4 in HCV-HSK9 (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4A</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">B</a></strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CCV-NGY peptide conformations for TCR contacts. (A) Electron densities and overall conformations of the structurally defined CCV-NGY9 peptides of pAime-128. (B) Superposition of the CCV-NGY9 peptide presented by pAime-128 with nonapeptides presented by other vertebrate MHC-I molecules. Peptides are shown as ribbons with the following color scheme: cyan, Aime-128-CCV-NGY9; salmon, HLA-B*0801-HSK9 (4QRP); light blue, HLA-B*0801-FLR9 (1MI5); and gray, peptides presented by HLA-A*0201 (2AV1), Mamu-A*02 (3JTT), H-2Kd (1VGK), SLA-1*0401 (3QQ3), BoLA-A*01801 (3PWU), FLA-E*01801 (5XMF), DLA-88*50801 (5F1I) and Anpl-UAA*01 (5GJX). (C) The refolded products of Aime-128 and Aime-&#x3B2;2m in the presence of mutated peptides tested by gel filtration chromatograms. The refolding efficiencies are represented by the relevant concentration ratios and by the heights of the peak for each mutant. The mutated peptides P1A, P5A and P9A clearly yielded low refolding efficiency. (D) CD spectropolarimetry was utilized to assess the thermostabilities of the purified pAime-128 complexes. Shown here are the data fitted to the denaturation curves using the Origin 9.1 program (OriginLab). The Tm values of different peptides are indicated by the gray line at the 50% unfolded level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CCV-NGY peptide conformations for TCR contacts. (&lt;strong&gt;A&lt;/strong&gt;) Electron densities and overall conformations of the structurally defined CCV-NGY9 peptides of pAime-128. (&lt;strong&gt;B&lt;/strong&gt;) Superposition of the CCV-NGY9 peptide presented by pAime-128 with nonapeptides presented by other vertebrate MHC-I molecules. Peptides are shown as ribbons with the following color scheme: cyan, Aime-128-CCV-NGY9; salmon, HLA-B*0801-HSK9 (4QRP); light blue, HLA-B*0801-FLR9 (1MI5); and gray, peptides presented by HLA-A*0201 (2AV1), Mamu-A*02 (3JTT), H-2Kd (1VGK), SLA-1*0401 (3QQ3), BoLA-A*01801 (3PWU), FLA-E*01801 (5XMF), DLA-88*50801 (5F1I) and &lt;em&gt;Anpl&lt;/em&gt;-UAA*01 (5GJX). (&lt;strong&gt;C&lt;/strong&gt;) The refolded products of Aime-128 and Aime-&lt;em&gt;&#x3B2;&lt;/em&gt;2m in the presence of mutated peptides tested by gel filtration chromatograms. The refolding efficiencies are represented by the relevant concentration ratios and by the heights of the peak for each mutant. The mutated peptides P1A, P5A and P9A clearly yielded low refolding efficiency. (&lt;strong&gt;D&lt;/strong&gt;) CD spectropolarimetry was utilized to assess the thermostabilities of the purified pAime-128 complexes. Shown here are the data fitted to the denaturation curves using the Origin 9.1 program (OriginLab). The Tm values of different peptides are indicated by the gray line at the 50% unfolded level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F4.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F4.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 4</span> <p id="p-24" class="first-child">CCV-NGY peptide conformations for TCR contacts. (<strong>A</strong>) Electron densities and overall conformations of the structurally defined CCV-NGY9 peptides of pAime-128. (<strong>B</strong>) Superposition of the CCV-NGY9 peptide presented by pAime-128 with nonapeptides presented by other vertebrate MHC-I molecules. Peptides are shown as ribbons with the following color scheme: cyan, Aime-128-CCV-NGY9; salmon, HLA-B*0801-HSK9 (4QRP); light blue, HLA-B*0801-FLR9 (1MI5); and gray, peptides presented by HLA-A*0201 (2AV1), Mamu-A*02 (3JTT), H-2Kd (1VGK), SLA-1*0401 (3QQ3), BoLA-A*01801 (3PWU), FLA-E*01801 (5XMF), DLA-88*50801 (5F1I) and <em>Anpl</em>-UAA*01 (5GJX). (<strong>C</strong>) The refolded products of Aime-128 and Aime-<em>β</em>2m in the presence of mutated peptides tested by gel filtration chromatograms. The refolding efficiencies are represented by the relevant concentration ratios and by the heights of the peak for each mutant. The mutated peptides P1A, P5A and P9A clearly yielded low refolding efficiency. (<strong>D</strong>) CD spectropolarimetry was utilized to assess the thermostabilities of the purified pAime-128 complexes. Shown here are the data fitted to the denaturation curves using the Origin 9.1 program (OriginLab). The Tm values of different peptides are indicated by the gray line at the 50% unfolded level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">With different peptide presentation modes observed for pAime-128, it is necessary to clarify the critical anchors for the peptide bound to Aime-128. Therefore, the stabilities of the CCV-NGY9 peptide and its mutants with alanine substitution at each position bound with Aime-128 were measured through circular dichroism (CD) spectroscopy (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4C</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">D</a></strong>). The midpoint transition temperature (Tm) value of the wild-type CCV-NGY9 peptide was determined to be 49.3°C. Compared with the CCV-NGY9 peptide, the CCV-P9A-substituted peptide showed a dramatic decrease in stability with the lowest Tm at 39.4°C, while the peptide mutants CCV-P1A and CCV-P5A had lower Tms of 47.2°C and 47.4°C, respectively, indicating a minor decrease in stability. The remaining peptide mutants showed different levels of enhanced stability, with Tms ranging from 49.5°C to 54.8°C (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4C</a> and <a id="xref-fig-4-8" class="xref-fig" href="#F4">D</a></strong>). These data indicated that for peptides bound to Aime-128, PΩ plays a pivotal role in peptide anchoring since it cannot be replaced by alanine. The peptide residues at the N terminus (PN) and position 5 (P5) are also important for stabilization of pAime-128 complex. Notably, the mutant CCV-P2A displayed the highest Tm of 54.8°C, suggesting that the mutation of P2-G to comparatively large amino acids might increase the stability of pAime-128 complexes. Based on this finding and the peptide screening results (<strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>), we identified the motif of pAime-128 presented peptides. The binding motif of pAime-128 is (Ala, Asn, Thr or Glu)-(Ala, Gly, Ser, Thr or Val)-x-x-(Ala, Phe, Arg, Pro, Tyr)-x-x-x-(Phe or Leu).</p></div><div id="sec-8" class="subsection"><h3>Distinctive Interaction of Aime-128 and <em>β</em>2m/TCR/CD8 Molecules</h3><p id="p-26">Superposing pAime-128 molecule on the known pMHC-I structures from different species showed that the AB loop of pAime-128 and other mammal pMHC-I structures does not contact <em>β</em>2m (<strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A-E</a></strong>), whereas the AB loop binds <em>β</em>2m via hydrogen bonds only in nonmammals (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5F, G</a></strong>). Furthermore, the CD and EF loops of mammals are longer than those of nonmammals (<strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5A</a></strong>), because mammalian MHC-I contains two more amino acids than nonmammalian MHC-I (<strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig 2</a></strong>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="pAime-128 complexes show some features that are similar among mammals but different from those of lower vertebrates. (A) Unique details of the higher vertebrates in PBG. (pAime-128 cyan, grass carp-5Y91 light orange, chicken-4E0R hot pink, duck-5GJX magenta and other mammals (1VGK, 2XFX, 3OXS, 3PWU, 3TID, 1ZVS, 3QQ3, 3RWC, 3X11, 4QRQ, 4MJ6, 4MNX, 4N02, 4NT6, 4O2C, 4QOK, 4QRS, 5XMF, 3BUY, 1ZT7, 4WJ5 white). The AB loops of the nonmammals are downward, which can interact with &#x3B2;2m, while the mammals&#x2019; cannot contact &#x3B2;2m. The CD loops of mammals are longer than those of lower vertebrates, causing the 2 more residues after the 40th amino acids of PBG in the higher vertebrates than in the nonmammals. The EF loops of mammals are longer than those of nonmammals. There is a hydrophobic core in the GH loop of grass carp, duck and chicken, so the GH loop is expanded and close to &#x3B2;2m. (B-G) The gray areas indicate the discrepant regions of MHC-I and &#x3B2;2m interfaces among chicken, grass carp, and mammals. (B) In pAime-128, the AB loop of Aime-128 cannot bind to &#x3B2;2m. (C) In mouse pMHC-I structures (3TID), the AB loop cannot bind to &#x3B2;2m. (D) In human pMHC-I structures (3OXS), the AB loop cannot bind to &#x3B2;2m. (E) In monkey pMHC-I structures (3RWC), the AB loop cannot bind to &#x3B2;2m. (F) In chicken, the BF2*0401(4E0R) AB loop can bind to &#x3B2;2m. (G) In grass carp (5Y91), the AB loop can bind to &#x3B2;2m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;pAime-128 complexes show some features that are similar among mammals but different from those of lower vertebrates. (&lt;strong&gt;A&lt;/strong&gt;) Unique details of the higher vertebrates in PBG. (pAime-128 cyan, grass carp-5Y91 light orange, chicken-4E0R hot pink, duck-5GJX magenta and other mammals (1VGK, 2XFX, 3OXS, 3PWU, 3TID, 1ZVS, 3QQ3, 3RWC, 3X11, 4QRQ, 4MJ6, 4MNX, 4N02, 4NT6, 4O2C, 4QOK, 4QRS, 5XMF, 3BUY, 1ZT7, 4WJ5 white). The AB loops of the nonmammals are downward, which can interact with &lt;em&gt;&#x3B2;&lt;/em&gt;2m, while the mammals&#x2019; cannot contact &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. The CD loops of mammals are longer than those of lower vertebrates, causing the 2 more residues after the 40&lt;sup&gt;th&lt;/sup&gt; amino acids of PBG in the higher vertebrates than in the nonmammals. The EF loops of mammals are longer than those of nonmammals. There is a hydrophobic core in the GH loop of grass carp, duck and chicken, so the GH loop is expanded and close to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;B-G&lt;/strong&gt;) The gray areas indicate the discrepant regions of MHC-I and &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m interfaces among chicken, grass carp, and mammals. (&lt;strong&gt;B&lt;/strong&gt;) In pAime-128, the AB loop of Aime-128 cannot bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;C&lt;/strong&gt;) In mouse pMHC-I structures (3TID), the AB loop cannot bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;D&lt;/strong&gt;) In human pMHC-I structures (3OXS), the AB loop cannot bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;E&lt;/strong&gt;) In monkey pMHC-I structures (3RWC), the AB loop cannot bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;F&lt;/strong&gt;) In chicken, the BF2*0401(4E0R) AB loop can bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m. (&lt;strong&gt;G&lt;/strong&gt;) In grass carp (5Y91), the AB loop can bind to &lt;em&gt;&#x3B2;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F5.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F5.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 5</span> <p id="p-27" class="first-child">pAime-128 complexes show some features that are similar among mammals but different from those of lower vertebrates. (<strong>A</strong>) Unique details of the higher vertebrates in PBG. (pAime-128 cyan, grass carp-5Y91 light orange, chicken-4E0R hot pink, duck-5GJX magenta and other mammals (1VGK, 2XFX, 3OXS, 3PWU, 3TID, 1ZVS, 3QQ3, 3RWC, 3X11, 4QRQ, 4MJ6, 4MNX, 4N02, 4NT6, 4O2C, 4QOK, 4QRS, 5XMF, 3BUY, 1ZT7, 4WJ5 white). The AB loops of the nonmammals are downward, which can interact with <em>β</em>2m, while the mammals’ cannot contact <em>β</em><sub>2</sub>m. The CD loops of mammals are longer than those of lower vertebrates, causing the 2 more residues after the 40<sup>th</sup> amino acids of PBG in the higher vertebrates than in the nonmammals. The EF loops of mammals are longer than those of nonmammals. There is a hydrophobic core in the GH loop of grass carp, duck and chicken, so the GH loop is expanded and close to <em>β</em><sub>2</sub>m. (<strong>B-G</strong>) The gray areas indicate the discrepant regions of MHC-I and <em>β</em><sub>2</sub>m interfaces among chicken, grass carp, and mammals. (<strong>B</strong>) In pAime-128, the AB loop of Aime-128 cannot bind to <em>β</em><sub>2</sub>m. (<strong>C</strong>) In mouse pMHC-I structures (3TID), the AB loop cannot bind to <em>β</em><sub>2</sub>m. (<strong>D</strong>) In human pMHC-I structures (3OXS), the AB loop cannot bind to <em>β</em><sub>2</sub>m. (<strong>E</strong>) In monkey pMHC-I structures (3RWC), the AB loop cannot bind to <em>β</em><sub>2</sub>m. (<strong>F</strong>) In chicken, the BF2*0401(4E0R) AB loop can bind to <em>β</em><sub>2</sub>m. (<strong>G</strong>) In grass carp (5Y91), the AB loop can bind to <em>β</em><sub>2</sub>m.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The antigenic peptides presented by pMHC-I complexes are eventually scanned and recognized by TCRs to initiate MHC-I-restricted CD8 T cell immunity. Extensive structural studies have illustrated that the TCR repertoire specific for a certain antigenic peptide is determined by the conformations of both the peptide and the restricted MHC-I element (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). Comparisons with MHC-I structures with known TCR docking strategies revealed that the exposed Aime-128 residues Arg<sup>62</sup>, Arg<sup>65</sup>, Ile<sup>66</sup>, Asp<sup>69</sup>, Gln<sup>72</sup>, Val<sup>76</sup>, Gln<sup>79</sup>, Glu<sup>154</sup>, Arg<sup>155</sup>, Asn<sup>158</sup>, Glu<sup>161</sup>, Glu<sup>163</sup>, Glu<sup>166</sup> and Trp<sup>167</sup> have great potential to interact with TCRs (<strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a></strong>). Although most of these residues are highly conserved or preferentially used among different species, rarely used residues were observed in Aime-128, including Ile<sup>66</sup>, Asp<sup>69</sup>, Arg<sup>155</sup> and Asn<sup>158</sup> (<strong><a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig 2</a></strong>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Putative TCR docking sites on pAime-128 complexes and a unique way to bind CD8. (A) Based on the HLA-B*0801-HSK9-specific TCR, the residues on pAime-128 that may contact the TCR CDR loops (green) are shown as red surfaces, and the CCV-NGY9 peptide is shown on the surface according to its charge. The proximity of the TCR CDR loops to the peptide-binding region of pAime-128 is shown. (B) The major shift in the &#x3B1;3 domain and the variation in the key residues for binding CD8&#x3B1;&#x3B1; in pAime-128 complexes. pAime-128 structure is superposed on the HLA-A2-CD8&#x3B1;&#x3B1; (HLA-A2, white, PDB code: 1AKJ), Ctid-UAAg (cyan, PDB code: 5Y91) and BF2*0401 (hot pink, PDB code: 4E0R) structures in the ribbon. The residues shown in different colors according to their species in the CD loop that are critical for interaction with CD8&#x3B1;&#x3B1; are shown in sticks. The distance between E219 of pCtid-UAAg and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 12.4 &#xC5;, and the distance between Q222 in BF2*0401 (PDB: 4E0R) and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 8.4 &#xC5;. The distance between the superposed CD loops of pAime-128 is approximately 3.1 &#xC5;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Putative TCR docking sites on pAime-128 complexes and a unique way to bind CD8. (&lt;strong&gt;A&lt;/strong&gt;) Based on the HLA-B*0801-HSK9-specific TCR, the residues on pAime-128 that may contact the TCR CDR loops (green) are shown as red surfaces, and the CCV-NGY9 peptide is shown on the surface according to its charge. The proximity of the TCR CDR loops to the peptide-binding region of pAime-128 is shown. (&lt;strong&gt;B&lt;/strong&gt;) The major shift in the &#x3B1;3 domain and the variation in the key residues for binding CD8&#x3B1;&#x3B1; in pAime-128 complexes. pAime-128 structure is superposed on the HLA-A2-CD8&#x3B1;&#x3B1; (HLA-A2, white, PDB code: 1AKJ), &lt;em&gt;Ctid&lt;/em&gt;-UAAg (cyan, PDB code: 5Y91) and BF2*0401 (hot pink, PDB code: 4E0R) structures in the ribbon. The residues shown in different colors according to their species in the CD loop that are critical for interaction with CD8&#x3B1;&#x3B1; are shown in sticks. The distance between E219 of p&lt;em&gt;Ctid&lt;/em&gt;-UAAg and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 12.4 &#xC5;, and the distance between Q222 in BF2*0401 (PDB: 4E0R) and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 8.4 &#xC5;. The distance between the superposed CD loops of pAime-128 is approximately 3.1 &#xC5;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F6.medium.gif" width="440" height="155"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F6.medium.gif" width="440" height="155"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 6</span> <p id="p-29" class="first-child">Putative TCR docking sites on pAime-128 complexes and a unique way to bind CD8. (<strong>A</strong>) Based on the HLA-B*0801-HSK9-specific TCR, the residues on pAime-128 that may contact the TCR CDR loops (green) are shown as red surfaces, and the CCV-NGY9 peptide is shown on the surface according to its charge. The proximity of the TCR CDR loops to the peptide-binding region of pAime-128 is shown. (<strong>B</strong>) The major shift in the α3 domain and the variation in the key residues for binding CD8αα in pAime-128 complexes. pAime-128 structure is superposed on the HLA-A2-CD8αα (HLA-A2, white, PDB code: 1AKJ), <em>Ctid</em>-UAAg (cyan, PDB code: 5Y91) and BF2*0401 (hot pink, PDB code: 4E0R) structures in the ribbon. The residues shown in different colors according to their species in the CD loop that are critical for interaction with CD8αα are shown in sticks. The distance between E219 of p<em>Ctid</em>-UAAg and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 12.4 Å, and the distance between Q222 in BF2*0401 (PDB: 4E0R) and Q226 of HLA-A*0201 (PDB: 1AKJ) is approximately 8.4 Å. The distance between the superposed CD loops of pAime-128 is approximately 3.1 Å.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Further analysis found different structures of the α3 loop (CD loop) in pAime-128 and other pMHC-I molecules. The CD loop of Aime-128, encompassing residues 222-228, is believed to be essential for CD8 interaction (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Of particular note, the highly conserved residues Glu<sup>222</sup> and Gln<sup>226</sup>, which have been proven to directly contact CD8 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>), are also present in pAime-128. Interestingly, the distance between Glu<sup>219</sup> of p<em>Ctid</em>-UAAg and Gln<sup>226</sup> of HLA-A*0201 is approximately 12.4 Å, and the distance between Gln<sup>222</sup> in BF2*0401 and Gln<sup>226</sup> of HLA-A*0201 is approximately 8.4 Å. The distance between the superposed CD loops of pAime-128 and HLA-A*0201 is approximately 3.1 Å (<strong><a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6B</a></strong>).</p></div><div id="sec-9" class="subsection"><h3>Analysis of Peptide-epitopes of Viruses Based on the Motif of Panda pAime-128</h3><p id="p-31">The amino acid sequences of the pathogenic viruses related to viral diseases reported in the giant panda were obtained. Based on the 3D structure of pAime-128, the characteristics of presentation of viral peptides, and the motif from the experiment, the potential viral CTL epitopes of canine distemper virus (CDV), canine corona virus (CCV), giant panda polyomavirus (GPPV), giant panda rotavirus (GPRV), giant panda-associated gemycircularvirus (GPGE), giant panda anellovirus (GPAN), and influenza H1N1 viruses (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>) were reasonably proposed. The results are shown in <strong><a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a></strong>. A series of viral epitope-peptides from the proteomes of viruses, including CCV, CDV, GPP, GPRV, GPGE, GPAN and H1N1, were screened against the panda MHC-I molecules. We obtained a total of 8 nonapeptides conforming to virus peptide motif of Aime-128, and synthesize them for <em>in vitro</em> verification. We found that all the selected virus peptides can bind to Aime-128, only one can bind Aime-128 but cannot tolerate anion-exchange chromatography while the others can tolerate anion-exchange chromatography. Although the result was rational calculated, it also fully demonstrates the potential research value of pAime-128.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted peptides and their binding to Aime-128 evaluated by in vitro refolding." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-296861005" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predicted peptides and their binding to Aime-128 evaluated by &lt;em&gt;in vitro&lt;/em&gt; refolding.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F7.medium.gif" width="440" height="211"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 7" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F7.medium.gif" width="440" height="211"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.15.908608/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 7</span> <p id="p-32" class="first-child">Predicted peptides and their binding to Aime-128 evaluated by <em>in vitro</em> refolding.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-10"><h2 class="">DISCUSSION</h2><p id="p-33">The viral peptides presented by classical MHC-I molecules require the assembly of a pMHC-I complex for TCR recognition, which is critical for the initiation of antiviral CTL immunity in most jawed vertebrates (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). In this study, the 3D structure of giant panda classical MHC-I complexed with viral peptide derived from CCV spike protein was determined for the first time, and several unique features of panda pMHC-I were identified․.</p><p id="p-34">One notable finding was that the homologies of classical MHC-I molecules in pandas are high, at approximately 70-90%", are higher than those of other mammals, being approximately 70-90%, whereas the homology of the antigen-binding domains composed of α1 and α2 regions is much lower. This result implied that the panda MHC-I molecules can form different PBGs. In addition, the amino acid composition of the A, B, C, D, E and F pockets in the PBGs are not identical among pandas, and the PBG regions of pandas and other members the bear family are very different from one another (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig 2</a></strong>). The results suggest that the antigen-peptide presenting profiles of the panda individuals are different, especially the homology of the α1 region, which can be as low as approximately 57%. This difference may be quite remarkable. Thus, the classical MHC-I binding antigen-peptide profile of pandas is diverse. These findings indicate the high-throughput antiviral CTL immune response potential of Ursidae.</p><p id="p-35">A second notable finding is that in the PBG pockets, the 9 residues constituting pocket A can be found in most giant panda MHC-I molecules, though 2 residues, Asn<sup>63</sup> and Glu<sup>163</sup>, are rare among known MHC-I molecules. Due to the presence of the Asn<sup>63</sup> residue, the pAime-128 complex pocket A is reasonably tight. The 8 residues that constitute pocket B can be found in almost all classical panda MHC-I molecules, although Asn<sup>63</sup>, Ile<sup>66</sup>, Ala<sup>67</sup> and His<sup>99</sup> are rare among known panda MHC-I molecules. Moreover, 5 residues constitute pocket C, but these residues are not highly conserved in mammalian MHC-I molecules (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>). Similarly, 5 residues constitute pocket D, but Glu<sup>152</sup> and Arg<sup>155</sup> are not conserved. Therefore, although some amino acids are highly conserved in the composition of certain pockets, the different amino acids result in distinctive pocket conformations and a unique pAime-128 PBG for presenting viral epitope peptides with specific characteristics. Due to pAime-128, the polymorphism of amino acid sequences is easily reflected in its 3D structure, which is also important for studying pMHC-I complexes. In addition, the amino acid composition of PBGs among other genera and members of the bear family is not completely conservative, so there are some differences in A to F pockets. There are also great differences between giant pandas and other members of the bear family. The results also reveal the characteristics of the pMHC-I complexes of pandas and other members of the bear family and differences between them in their viral antigen-presenting peptide profiles.</p><p id="p-36">Furthermore, by analyzing the structure of pAime-128 and comparing it with the structures of other known pMHC-Is, we found that the viral peptide in PBG features the “M” configuration that can activate T cells (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>) and that key for recognition by TCR is the combination of the peptide P4-Asn and P6-Phe amino acids. Through analysis of the pockets in the PBG and the stability of CCV-NGY9 peptide and its alanine substitution mutant determined by CD spectroscopy, we identified the peptide-binding motif of pAime-128 and obtained a series of viral peptides that can bind or can potentially bind with pAime-128. The results provide a new platform for the effective design of bear family viral vaccines.</p><p id="p-37">In summary, the structure of pAime-128, as a representative of the bear family pMHC-I complexes, was elucidated, and unique features of pMHC-I antigen presentation in in the bear family were identified. In addition, the viral peptide presentation profile was proposed in this paper. These results provide a new platform for the further study of bear family antiviral CTL immunology and vaccinology.</p></div><div class="section" id="sec-11"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-12" class="subsection"><h3>Viral Peptide Synthesis</h3><p id="p-38">Four epitope peptides that potentially bind to Aime-128 were predicted by the NetMHCpan4.0 server (<a href="http://www.cbs.dtu.dk/services/NetMHC/">http://www.cbs.dtu.dk/services/NetMHC/</a>) and via artificial intelligence selection based on the CCV spike protein (GenBank accession no. AFG19726.1), the CDV hemagglutinin (H) protein (GenBank accession no. AAD54600.1), CPV virus protein 1 (VP1) (GenBank accession no. AAV36771.1) and the H7N9 influenza A virus (GenBank accession no. AGI60301.1). These peptides were synthesized and purified by reverse-phase high-performance liquid chromatography (HPLC) (SciLight Biotechnology, Beijing, China) with &gt;90% purity. The peptides were stored in lyophilized aliquots at −20°C after synthesis and were dissolved in dimethyl sulfoxide (DMSO) before use.</p></div><div id="sec-13" class="subsection"><h3>Preparation of Expression Constructs</h3><p id="p-39">DNA fragments encoding the extracellular domain of giant panda MHC-I <em>Aime-128</em> (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AM282693&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom">AM282693</a>, residues 1–270 of the mature protein with <em>Nde</em>I and <em>Xho</em>I restriction sites) and <em>Aime-β2m</em> (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AB178590&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom">AB178590</a>, residues 1–98 of the mature protein with restriction sites <em>Nde</em>I and <em>Xho</em>I) were synthesized by Invitrogen Life Technologies (Shanghai, China). The products were ligated into the pET21a vector (Novagen) and then transformed into the <em>Escherichia coli</em> (<em>E. coli</em>) Transetta (DE3) strain. Recombinant plasmids were expressed as inclusion bodies and purified as previously described (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>). Complexes of the CCV-NGY9 peptide with Aime-128 and Aime-<em>β</em>2m (pAime-128) were prepared with refolding assays using the gradual dilution method, as previously described (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>). After 48 hours of incubation at 4°C, the remaining soluble portion of the complex was concentrated and then purified by chromatography on a Superdex200 16/60 column followed by Resource-Q anion-exchange chromatography (GE Healthcare), as described previously (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>).</p></div><div id="sec-14" class="subsection"><h3>Thermostability Measurements Using Circular Dichroism Spectroscopy</h3><p id="p-40">The thermostabilities of Aime-128 with nine peptides were tested by CD spectroscopy. CD spectra were measured at 20°C on a Jasco J-810 spectropolarimeter equipped with a water-circulating cell holder. The protein concentration was 0.1 mg/mL in pH 8.0 Tris buffer (20 mM Tris and 50 mM NaCl). Thermal denaturation curves were determined by monitoring the CD value at 218 nm using a 1-mm optical-path-length cell as the temperature was raised from 25 to 80°C at a rate of 1°C/min. The temperature of the sample solution was directly measured with a thermistor. The fraction of unfolded protein was calculated from the mean residue ellipticity (θ) using a standard method. The unfolded fraction (%) is expressed as (θ−θ<sub>N</sub>)/(θ<sub>U</sub>−θ<sub>N</sub>), where θ<sub>N</sub> and θ<sub>U</sub> are the mean residue ellipticity values in the fully folded and fully unfolded states, respectively. The midpoint transition temperature (T<sub>m</sub>) was determined by fitting the data to the denaturation curves using the Origin 8.0 program (OriginLab) as described previously (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). Based on the CD spectroscopy results, the nonapeptide binding motifs of Aime-128 were extrapolated from potential CTL epitopes of viruses detected from giant panda (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>).</p></div><div id="sec-15" class="subsection"><h3>Crystallization and Data Collection</h3><p id="p-41">The purified pAime-128 complexes were ultimately concentrated to 7.5 mg/ml in crystallization buffer (20 mM Tris–HCl (pH 8.0) and 50 mM NaCl), mixed with reservoir buffer at a 1:1 ratio, and then crystallized by the hanging-drop vapor diffusion technique at 4°C. Index Kits (Hampton Research, Riverside, CA) were used to screen the crystals. After several days, pAime-128 crystals were obtained with Index solution 55 (30% (w/v) polyethylene glycol 3350, 0.05 M magnesium chloride hexahydrate and 0.1 M HEPES (pH 7.5)). Diffraction data were collected using an in-house X-ray source (Rigaku MicroMax007 desktop rotating anode X-ray generator with a Cu target operated at 40 kV and 30 mA) and an R-Axis IV imaging-plate detector at a wavelength of 0.97892 Å. In each case, the crystal was first soaked in reservoir solution containing 25% glycerol as a cryoprotectant for several seconds and then flash-cooled in a stream of gaseous nitrogen at 100 K (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). The collected intensities were indexed, integrated, corrected for absorption, scaled, and merged using the HKL2000 package (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>).</p></div><div id="sec-16" class="subsection"><h3>Structure Determination and Refinement</h3><p id="p-42">The structure of pAime-128 was solved by molecular replacement using the MOLREP program with HLA-B*5101 (PDB code, 1E27) as the search model. Extensive model building was performed by hand using COOT (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>), and restrained refinement was performed using REFMAC5. Further rounds of refinement were performed using the phenix refine program implemented in the PHENIX package (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>) with isotropic ADP refinement and bulk solvent modeling. The stereochemical quality of the final model was assessed with the PROCHECK program (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>).The data collection and refinement statistics are listed in <strong><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a></strong>.</p></div><div id="sec-17" class="subsection"><h3>Structural Analysis and Generation of Illustrations</h3><p id="p-43">Peptide-contacting residues were identified using the program CONTACT and were defined as residues containing an atom within 3.3 Å of the target partner. Structural illustrations and electron density-related Figs were generated using the PyMOL molecular graphics system (<a href="http://www.pymol.org">http://www.pymol.org</a>). Solvent-accessible surface areas and B factors were calculated with CCP4. The multiple sequence alignment was performed by Clustal Omega (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>) (<a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">https://www.ebi.ac.uk/Tools/msa/clustalo/</a>) and ESPript 3.0 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>) (<a href="http://espript.ibcp.fr/ESPript/ESPript/">http://espript.ibcp.fr/ESPript/ESPript/</a>). Accessible surface area (ASA) and buried surface area (BSA) were calculated by the online website PDBePISA.</p></div><div id="sec-18" class="subsection"><h3>Protein Structure Accession Numbers</h3><p id="p-44">The crystal structures have been deposited in the Protein Data Bank (<a href="http://www.pdb.org/pdb/home/home.do">http://www.pdb.org/pdb/home/home.do</a>) with accession numbers (5ZE5).</p></div></div><div class="section" id="sec-19"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-46">H.Y. performed the experiments; H.Y., L.M. and L.Z. analyzed the data and wrote the paper; X.L. provided constructive suggestions; and C.X. designed and supervised the study. All authors critically revised the manuscript and gave their final approval of the version to be submitted.</p></div><div class="section COI-statement" id="sec-20"><h2 class="">CONFLICT OF INTEREST</h2><p id="p-47">The authors declare that they have no competing interests.</p></div><div class="section" id="sec-21"><h2 class="">FUNDING</h2><p id="p-48">This work was supported by the National Natural Science Foundation of China (Grant NO. 31972683 and 31572493).</p></div><div class="section" id="sec-22"><h2 class="">SUPPLEMENTARY MATERIAL</h2><p id="p-49"><strong>Supplementary Fig 1</strong> Refolding efficiency of pAime-128 complex. Aime-128 and Aime-<em>β</em><sub>2</sub>m were corefolded with different nonapeptides. The refolded products were purified by gel filtration chromatography and anion-exchange chromatography. pAime-128 complex b curves are shown in blue. The insets show reducing SDS-PAGE gels (15%) of the peaks that are labeled on the curve. Lane M contains molecular mass markers (labeled in kD). (<strong>A</strong>) Refolding efficiency of Aime-128 and Aime-<em>β</em><sub>2</sub>m with NGY9. (<strong>B</strong>) Refolding efficiency of Aime-128 and Aime-<em>β</em><sub>2</sub>m with TSV9. (<strong>C</strong>) Refolding efficiency of Aime-128 and Aime-<em>β</em><sub>2</sub>m with NTY9. (<strong>D</strong>) Refolding efficiency of Aime-128 and Aime-<em>β</em><sub>2</sub>m with EVW9.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-45">We acknowledge the assistance of the staff at the Shanghai Synchrotron Radiation Facility of China (SSRF).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flajnik</span>  <span class="cit-name-given-names">MF.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A cold-blooded view of adaptive immunity</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Unfinished Business: Evolution of the MHC and the Adaptive Immune System of Jawed Vertebrates</span>. <abbr class="cit-jnl-abbrev">Annual Review of Immunology</abbr> <span class="cit-vol">36</span>:<span class="cit-fpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BImmunology%26rft.volume%253D36%26rft.spage%253D383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.3" data-doi="10.1073/pnas.1416543112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garstka</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Celie</span>  <span class="cit-name-given-names">PHN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joosten</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">GMC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ilana</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rieuwert</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magda</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lennert</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huib</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The first step of peptide selection in antigen presentation by MHC class I molecules</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">1505</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1416543112%26rft_id%253Dinfo%253Apmid%252F25605945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzUvMTUwNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzIwMjAuMDEuMTUuOTA4NjA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Major histocompatibility complex class I (FLA-E*01801) molecular structure in domestic cats demonstrates species-specific characteristics in presenting viral antigen peptides</span>. <abbr class="cit-jnl-abbrev">Journal of Virology:JVI</abbr>.<span class="cit-fpage">01631</span>–<span class="cit-lpage">01617</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.5" data-doi="10.1128/JVI.00119-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Structural and Biochemical Analyses of Swine Major Histocompatibility Complex Class I Complexes and Prediction of the Epitope Map of Important Influenza A Virus Strains</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">6625</span>–<span class="cit-lpage">6641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00119-16%26rft_id%253Dinfo%253Apmid%252F27170754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82NjI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.6" data-doi="10.4049/jimmunol.1600887"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haywood</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Diversified Anchoring Features the Peptide Presentation of DLA-88*50801: First Structural Insight into Domestic Dog MHC Class I</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">197</span>:<span class="cit-fpage">2306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1600887%26rft_id%253Dinfo%253Apmid%252F27511732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny82LzIzMDYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.908608v1.7" data-doi="10.1016/j.immuni.2012.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ladell</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iglesias</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilmann</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gras</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chikata</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuse</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fastenackels</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gostick</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bridgeman</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venturi</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arkoub</span>  <span class="cit-name-given-names">ZA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agut</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Bockel</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Douek</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takiguchi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossjohn</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Appay</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A molecular basis for the control of preimmune escape variants by HIV-specific CD8+ T cells</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">425</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D38%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2012.11.021%26rft_id%253Dinfo%253Apmid%252F23521884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2012.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23521884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330941500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.8" data-doi="10.1128/JVI.00030-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Two distinct conformations of a rinderpest virus epitope presented by bovine major histocompatibility complex class I N*01801: a host strategy to present featured peptides</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">85</span>:<span class="cit-fpage">6038</span>–<span class="cit-lpage">6048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00030-11%26rft_id%253Dinfo%253Apmid%252F21450819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NS8xMi82MDM4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.9" data-doi="10.4049/jimmunol.178.2.944"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Yu. Wai</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zong</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">AH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">JI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">First glimpse of the peptide presentation by rhesus macaque MHC class I: crystal structures of Mamu-A*01 complexed with two immunogenic SIV epitopes and insights into CTL escape</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">178</span>:<span class="cit-fpage">944</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DChu%26rft.auinit1%253DF.%26rft.volume%253D178%26rft.issue%253D2%26rft.spage%253D944%26rft.epage%253D952%26rft.atitle%253DFirst%2BGlimpse%2Bof%2Bthe%2BPeptide%2BPresentation%2Bby%2BRhesus%2BMacaque%2BMHC%2BClass%2BI%253A%2BCrystal%2BStructures%2Bof%2BMamu-A%2A01%2BComplexed%2Bwith%2BTwo%2BImmunogenic%2BSIV%2BEpitopes%2Band%2BInsights%2Binto%2BCTL%2BEscape%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.178.2.944%26rft_id%253Dinfo%253Apmid%252F17202356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6OToiMTc4LzIvOTQ0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.10" data-doi="10.1146/annurev.immunol.23.021704.115702"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yewdell</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haeryfar</span>  <span class="cit-name-given-names">SM.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Understanding presentation of viral antigens to CD8+ T cells in vivo: the key to rational vaccine design</span>. <abbr class="cit-jnl-abbrev">Annual Review of Immunology</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">651</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bimmunology%26rft.stitle%253DAnnu%2BRev%2BImmunol%26rft.aulast%253DYewdell%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D23%26rft.spage%253D651%26rft.epage%253D682%26rft.atitle%253DUnderstanding%2Bpresentation%2Bof%2Bviral%2Bantigens%2Bto%2BCD8%252B%2BT%2Bcells%2Bin%2Bvivo%253A%2Bthe%2Bkey%2Bto%2Brational%2Bvaccine%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.immunol.23.021704.115702%26rft_id%253Dinfo%253Apmid%252F15771583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.immunol.23.021704.115702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15771583&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228947000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaech</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Staron</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The interface between transcriptional and epigenetic control of effector and memory CD8? T-cell differentiation</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">261</span>:<span class="cit-fpage">157</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BImmunology%26rft.volume%253D261%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gruta</span>  <span class="cit-name-given-names">NLL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gras</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daley</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossjohn</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Understanding the drivers of MHC restriction of T cell receptors</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.13" data-doi="10.1146/annurev.immunol.26.021607.090421"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marrack</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott-Browne</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gapin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kappler</span>  <span class="cit-name-given-names">JW.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Evolutionarily conserved amino acids that control TCR-MHC interaction</span>. <abbr class="cit-jnl-abbrev">Annual Review of Immunology</abbr> <span class="cit-vol">26</span>:<span class="cit-fpage">171</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bimmunology%26rft.stitle%253DAnnu%2BRev%2BImmunol%26rft.aulast%253DCahalan%26rft.auinit1%253DM.%2BD.%26rft.volume%253D26%26rft.spage%253D171%26rft.epage%253D203%26rft.atitle%253DEvolutionarily%2Bconserved%2Bamino%2Bacids%2Bthat%2Bcontrol%2BTCR-MHC%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.immunol.26.021607.090421%26rft_id%253Dinfo%253Apmid%252F18304006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.immunol.26.021607.090421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18304006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255349300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.14" data-doi="10.1038/nri2887"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">P Anton</span>  <span class="cit-name-given-names">VDM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omer</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Mechanisms for T cell receptor triggering</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">47</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253Dvan%2Bder%2BMerwe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D55%26rft.atitle%253DMechanisms%2Bfor%2BT%2Bcell%2Breceptor%2Btriggering.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri2887%26rft_id%253Dinfo%253Apmid%252F21127503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri2887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21127503&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285572600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.15" data-doi="10.1016/j.immuni.2010.12.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Two-Stage Cooperative T Cell Receptor-Peptide Major Histocompatibility Complex-CD8 Trimolecular Interactions Amplify Antigen Discrimination</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">34</span>:<span class="cit-fpage">13</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D34%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2010.12.017%26rft_id%253Dinfo%253Apmid%252F21256056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2010.12.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21256056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287336600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.16" data-doi="10.1074/jbc.274.52.37259"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirksey</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pogue-Caley</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frelinger</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">The structural basis for the increased immunogenicity of two HIV-reverse transcriptase peptide variant/class I major histocompatibility complexes</span>. <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-vol">274</span>:<span class="cit-fpage">37259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DKirksey%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D274%26rft.issue%253D52%26rft.spage%253D37259%26rft.epage%253D37264%26rft.atitle%253DThe%2BStructural%2BBasis%2Bfor%2Bthe%2BIncreased%2BImmunogenicity%2Bof%2BTwo%2BHIV-Reverse%2BTranscriptase%2BPeptide%2BVariant%252FClass%2BI%2BMajor%2BHistocompatibility%2BComplexes%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.52.37259%26rft_id%253Dinfo%253Apmid%252F10601290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTIvMzcyNTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.17" data-doi="10.4049/jimmunol.180.6.3926"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Theodossis</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ai</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunstone</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nastovska</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramarathinam</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossjohn</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Structural and Biological Basis of CTL Escape in Coronavirus-Infected Mice</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">180</span>:<span class="cit-fpage">3926</span>–<span class="cit-lpage">3937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DButler%26rft.auinit1%253DN.%2BS.%26rft.volume%253D180%26rft.issue%253D6%26rft.spage%253D3926%26rft.epage%253D3937%26rft.atitle%253DStructural%2Band%2BBiological%2BBasis%2Bof%2BCTL%2BEscape%2Bin%2BCoronavirus-Infected%2BMice%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.180.6.3926%26rft_id%253Dinfo%253Apmid%252F18322201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4MC82LzM5MjYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.18" data-doi="10.4049/jimmunol.1501352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Structural Illumination of Equine MHC Class I Molecules Highlights Unconventional Epitope Presentation Manner That Is Evolved in Equine Leukocyte Antigen Alleles</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">196</span>:<span class="cit-fpage">1943</span>–<span class="cit-lpage">1954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1501352%26rft_id%253Dinfo%253Apmid%252F26764037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzE5NDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.19" data-doi="10.4049/jimmunol.1200885"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yong</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jianxun</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yanjie</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xuyu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jim</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Narrow groove and restricted anchors of MHC class I molecule BF2*0401 plus peptide transporter restriction can explain disease susceptibility of B4 chickens</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">189</span>:<span class="cit-fpage">4478</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1200885%26rft_id%253Dinfo%253Apmid%252F23041567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4OS85LzQ0NzgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.20" data-doi="10.4049/jimmunol.1600229"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dijkstra</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The Structure of the MHC Class I Molecule of Bony Fishes Provides Insights into the Conserved Nature of the Antigen-Presenting System</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">199</span>:<span class="cit-fpage">3668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1600229%26rft_id%253Dinfo%253Apmid%252F29055007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE5OS8xMC8zNjY4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Unger</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noçon</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolokotronis</span>  <span class="cit-name-given-names">SO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soibelzon</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spriggs</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dear</span>  <span class="cit-name-given-names">PH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">AW.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Mitochondrial genomes reveal an explosive radiation of extinct and extant bears near the Miocene-Pliocene boundary</span>. <abbr class="cit-jnl-abbrev">Bmc Evolutionary Biology</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">220</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBmc%2BEvolutionary%2BBiology%26rft.volume%253D8%26rft.spage%253D220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jian</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">OA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">FC-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Min</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The sequence and de novo assembly of the giant panda genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>:<span class="cit-fpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D463%26rft.spage%253D311%26rft_id%253Dinfo%253Apmid%252F20010809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20010809&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273748100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">QH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">SG.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Molecular characterization of major histocompatibility complex class I genes from the giant panda (Ailuropoda melanoleuca)</span>. <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">60</span>:<span class="cit-fpage">185</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DPan%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D60%26rft.issue%253D3-4%26rft.spage%253D185%26rft.epage%253D193%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BI%2Bgenes%2Bfrom%2Bthe%2Bgiant%2Bpanda%2B%2528Ailuropoda%2Bmelanoleuca%2529.%26rft_id%253Dinfo%253Apmid%252F18292994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18292994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">YF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">YY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">QH.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Isolation and characterization of class I MHC genes in the giant panda (Ailuropoda melanoleuca)</span>. <abbr class="cit-jnl-abbrev">Chinese Science Bulletin</abbr> <span class="cit-vol">58</span>:<span class="cit-fpage">2140</span>–<span class="cit-lpage">2147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChinese%2BScience%2BBulletin%26rft.volume%253D58%26rft.spage%253D2140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Q-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">Y-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S-G</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Patterns of genetic differentiation at MHC class I genes and microsatellites identify conservation units in the giant panda</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D13%26rft.spage%253D227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Canine distemper viral infection threatens the giant panda population in China</span>. <abbr class="cit-jnl-abbrev">Oncotarget</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">113910</span>–<span class="cit-lpage">113919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D8%26rft.spage%253D113910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">FS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">GX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">XZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">YW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">YD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">XH.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Isolation and identification of a canine coronavirus strain from giant pandas (Ailuropoda melanoleuca)</span>. <abbr class="cit-jnl-abbrev">Journal of Veterinary Science</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BScience%26rft.volume%253D10%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decaro</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mari</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elia</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanave</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dowgier</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colaianni</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martella</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buonavoglia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Full-length genome analysis of canine coronavirus type I</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">210</span>:<span class="cit-fpage">100</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D210%26rft.spage%253D100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.29" data-doi="10.1128/JVI.05040-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nianzhi</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jianxun</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sijia</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiaocheng</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qirun</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhaosan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiaoying</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Crystal structure of swine major histocompatibility complex class I SLA-1 0401 and identification of 2009 pandemic swine-origin influenza A H1N1 virus cytotoxic T lymphocyte epitope peptides</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">85</span>:<span class="cit-fpage">11709</span>–<span class="cit-lpage">11724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05040-11%26rft_id%253Dinfo%253Apmid%252F21900158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yMi8xMTcwOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzIwMjAuMDEuMTUuOTA4NjA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.30" data-doi="10.1074/jbc.M112.344689"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chih</span>  <span class="cit-name-given-names">LY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhenjun</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burrows</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamie</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephanie</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The energetic basis underpinning T-cell receptor recognition of a super-bulged peptide bound to a major histocompatibility complex class I molecule</span>. <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-vol">287</span>:<span class="cit-fpage">12267</span>–<span class="cit-lpage">12276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.344689%26rft_id%253Dinfo%253Apmid%252F22343629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMTUvMTIyNjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE1LjkwODYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.31" data-doi="10.4049/jimmunol.1401357"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nivarthi</span>  <span class="cit-name-given-names">UK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephanie</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lars</span>  <span class="cit-name-given-names">KN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucet</span>  <span class="cit-name-given-names">IS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miles</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tracy</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Margaret</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">An extensive antigenic footprint underpins immunodominant TCR adaptability against a hypervariable viral determinant</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">193</span>:<span class="cit-fpage">5402</span>–<span class="cit-lpage">5413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1401357%26rft_id%253Dinfo%253Apmid%252F25355921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE5My8xMS81NDAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.32" data-doi="10.4049/jimmunol.1102686"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stephanie</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilmann</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhenjun</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanim</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chih</span>  <span class="cit-name-given-names">LY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lars</span>  <span class="cit-name-given-names">KN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burrows</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamie</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A structural basis for varied αβ TCR usage against an immunodominant EBV antigen restricted to a HLA-B8 molecule</span>. <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">188</span>:<span class="cit-fpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1102686%26rft_id%253Dinfo%253Apmid%252F22140258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMzExIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNS45MDg2MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.33" data-doi="10.1038/nri1977"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamie</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Structural determinants of T-cell receptor bias in immunity</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">883</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DGazzinelli%26rft.auinit1%253DR.%2BT.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D883%26rft.epage%253D894%26rft.atitle%253DStructural%2Bdeterminants%2Bof%2BT-cell%2Breceptor%2Bbias%2Bin%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri1977%26rft_id%253Dinfo%253Apmid%252F17110956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri1977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17110956&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242244100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.34" data-doi="10.1038/42523"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tormo</span>  <span class="cit-name-given-names">J</span></span>, ., <span class="cit-auth"><span class="cit-name-surname">Gerth</span>  <span class="cit-name-given-names">UC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyer</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcmichael</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">JI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">EY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakobsen</span>  <span class="cit-name-given-names">BK.</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Crystal structure of the complex between human CD8alpha(alpha) and HLA-A2</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>:<span class="cit-fpage">630</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGao%26rft.auinit1%253DG.%2BF.%26rft.volume%253D387%26rft.issue%253D6633%26rft.spage%253D630%26rft.epage%253D634%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bcomplex%2Bbetween%2Bhuman%2BCD8alpha%2528alpha%2529%2Band%2BHLA-A2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42523%26rft_id%253Dinfo%253Apmid%252F9177355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9177355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mariuzza</span>  <span class="cit-name-given-names">RA.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Structural and biophysical insights into the role of CD4 and CD8 in T cell activation</span>. <abbr class="cit-jnl-abbrev">Frontiers in Immunology</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BImmunology%26rft.volume%253D4%26rft.spage%253D206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.36" data-doi="10.1186/s40168-017-0308-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Virome comparisons in wild-diseased and healthy captive giant pandas</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D5%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-017-0308-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-017-0308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hengmin</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shanshan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shengtao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhijun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuancheng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiecheng</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xianzhu</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Influenza A(H1N1)pdm09 virus infection in giant pandas, China</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">480</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.38" data-doi="10.1111/imr.12327"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luoma</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Coevolution of T-cell receptors with MHC and non-MHC ligands</span>. <abbr class="cit-jnl-abbrev">Immunological Reviews</abbr> <span class="cit-vol">267</span>:<span class="cit-fpage">30</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunological%2BReviews%26rft.volume%253D267%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fimr.12327%26rft_id%253Dinfo%253Apmid%252F26284470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/imr.12327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26284470&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.39" data-doi="10.1016/S0165-2478(02)00206-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tobita</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morii</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoshino</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azuma</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kozono</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A role for the P1 anchor residue in the thermal stability of MHC class II molecule I-A b</span>. <abbr class="cit-jnl-abbrev">Immunology Letters</abbr> <span class="cit-vol">85</span>:<span class="cit-fpage">47</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%2Bletters%26rft.stitle%253DImmunol%2BLett%26rft.aulast%253DTobita%26rft.auinit1%253DT.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D52%26rft.atitle%253DA%2Brole%2Bfor%2Bthe%2BP1%2Banchor%2Bresidue%2Bin%2Bthe%2Bthermal%2Bstability%2Bof%2BMHC%2Bclass%2BII%2Bmolecule%2BI-Ab.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0165-2478%252802%252900206-7%26rft_id%253Dinfo%253Apmid%252F12505196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0165-2478(02)00206-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12505196&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.40" data-doi="10.1107/S0021889898005305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hope</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Macromolecular Cryocrystallography: Cooling, Mounting, Storage and Transportation of Crystals</span>. <abbr class="cit-jnl-abbrev">Journal of Applied Crystallography</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">945</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BCrystallography%26rft.volume%253D31%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0021889898005305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0021889898005305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.41" data-doi="10.1007/BF00197809"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Otwinowski</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minor</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr> <span class="cit-vol">276</span>:<span class="cit-fpage">307</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol%26rft.volume%253D276%26rft.spage%253D307%26rft.atitle%253DMETHODS%2BENZYMOL%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00197809%26rft_id%253Dinfo%253Apmid%252F8520220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00197809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8520220&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.42" data-doi="10.1107/S0907444904019158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emsley</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowtan</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Coot: model-building tools for molecular graphics</span>. <abbr class="cit-jnl-abbrev">Acta crystallographica Section D, Biological crystallography</abbr> <span class="cit-vol">60</span>:<span class="cit-fpage">2126</span>–<span class="cit-lpage">2132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DEmsley%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253DPt%2B12%2BPt%2B1%26rft.spage%253D2126%26rft.epage%253D2132%26rft.atitle%253DCoot%253A%2Bmodel-building%2Btools%2Bfor%2Bmolecular%2Bgraphics.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444904019158%26rft_id%253Dinfo%253Apmid%252F15572765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444904019158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15572765&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225360500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.43" data-doi="10.1107/S0907444902016657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grosse-Kunstleve</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">LW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ioerger</span>  <span class="cit-name-given-names">TR</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCoy</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moriarty</span>  <span class="cit-name-given-names">NW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacchettini</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauter</span>  <span class="cit-name-given-names">NK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terwilliger</span>  <span class="cit-name-given-names">TC.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <abbr class="cit-jnl-abbrev">Acta crystallographica Section D, Biological crystallography</abbr> <span class="cit-vol">58</span>:<span class="cit-fpage">1948</span>–<span class="cit-lpage">1954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DAdams%26rft.auinit1%253DP.%2BD.%26rft.volume%253D58%26rft.issue%253DPt%2B11%26rft.spage%253D1948%26rft.epage%253D1954%26rft.atitle%253DPHENIX%253A%2Bbuilding%2Bnew%2Bsoftware%2Bfor%2Bautomated%2Bcrystallographic%2Bstructure%2Bdetermination.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444902016657%26rft_id%253Dinfo%253Apmid%252F12393927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444902016657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12393927&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178745000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.44" data-doi="10.1006/jmbi.1993.1351"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laskowski</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">JM.</span></span> <span class="cit-pub-date">1993</span>. <span class="cit-article-title">Main-chain bond lengths and bond angles in protein structures</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr> <span class="cit-vol">231</span>:<span class="cit-fpage">1049</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLaskowski%26rft.auinit1%253DR.%2BA.%26rft.volume%253D231%26rft.issue%253D4%26rft.spage%253D1049%26rft.epage%253D1067%26rft.atitle%253DMain-chain%2Bbond%2Blengths%2Band%2Bbond%2Bangles%2Bin%2Bprotein%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1993.1351%26rft_id%253Dinfo%253Apmid%252F8515464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1993.1351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8515464&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LK36500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plewniak</span>  <span class="cit-name-given-names">F</span></span>, ., <span class="cit-auth"><span class="cit-name-surname">Jeanmougin</span>  <span class="cit-name-given-names">F</span></span>, ., <span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">DG.</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res. Nucleic Acids Research</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">4876</span>–<span class="cit-lpage">4882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%2BNucleic%2BAcids%2BResearch%26rft.volume%253D25%26rft.spage%253D4876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.15.908608v1.46" data-doi="10.1093/nar/gkg556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patrice</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emmanuel</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">3320</span>–<span class="cit-lpage">3323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGouet%26rft.auinit1%253DP.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3320%26rft.epage%253D3323%26rft.atitle%253DESPript%252FENDscript%253A%2Bextracting%2Band%2Brendering%2Bsequence%2Band%2B3D%2Binformation%2Bfrom%2Batomic%2Bstructures%2Bof%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg556%26rft_id%253Dinfo%253Apmid%252F12824317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.15.908608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.15.908608v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.15.907295v1" title="vRNA-vRNA interactions in influenza A virus HA vRNA packaging" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.15.908608v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.15.908608v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.15.908608v1" class="active" data-icon-position="" data-hide-link-title="0">Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1107443" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S2goZZjWXlXGgkXEpiYixjoBms7wMLwN4-bBlJtDjUE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107443" id="node1107443--2" data-pisa="biorxiv;2020.01.15.908608v1" data-pisa-master="biorxiv;2020.01.15.908608" data-seqnum="1107443" data-apath="/biorxiv/early/2020/01/16/2020.01.15.908608.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hongyu</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lizhen</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lijie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Xiaoying</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chun</span> <span class="nlm-surname">Xia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.15.908608; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.15.908608 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;title=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;title=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;text=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;title=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;t=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;title=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;title=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107443" id="node1107443--3" data-pisa="biorxiv;2020.01.15.908608v1" data-pisa-master="biorxiv;2020.01.15.908608" data-seqnum="1107443" data-apath="/biorxiv/early/2020/01/16/2020.01.15.908608.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hongyu</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lizhen</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lijie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Xiaoying</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Chun</span> <span class="nlm-surname">Xia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.15.908608; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.15.908608 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Yuan%2C+Hongyu&amp;rft.au=Ma%2C+Lizhen&amp;rft.au=Zhang%2C+Lijie&amp;rft.au=Li%2C+Xiaoying&amp;rft.au=Xia%2C+Chun"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1107443/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1107443/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1107443/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1107443/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1107443/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1107443/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1107443/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1107443/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1107443/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1107443/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1107443/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1107443/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;count=horizontal&amp;via=&amp;text=Crystal%20structure%20of%20the%20giant%20panda%20MHC%20class%20I%20complex%3A%20first%20insights%20into%20the%20viral%20peptide%20presentation%20profile%20in%20the%20bear%20family&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.908608v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.15.908608v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
