<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/09/18/420463.atom"/>
  <meta name="HW.pisa" content="biorxiv;420463v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="VAPiD: a lightweight cross platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank"/>
  <meta name="DC.Identifier" content="10.1101/420463"/>
  <meta name="DC.Date" content="2018-09-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available. GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions. Results We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1-4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems. Conclusions We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at &lt;;https://github.com/rcs333/VAPiD&gt;;."/>
  <meta name="DC.Contributor" content="Ryan C. Shean"/>
  <meta name="DC.Contributor" content="Negar Makhsous"/>
  <meta name="DC.Contributor" content="Graham D. Stoddard"/>
  <meta name="DC.Contributor" content="Michelle J. Lin"/>
  <meta name="DC.Contributor" content="Alexander L. Greninger"/>
  <meta name="article:published_time" content="2018-09-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="VAPiD: a lightweight cross platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;h3&gt;;Background&lt;;/h3&gt;; &lt;;p&gt;;With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available.&lt;;/p&gt;;&lt;;p&gt;;GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions.&lt;;/p&gt;;&lt;;h3&gt;;Results&lt;;/h3&gt;; &lt;;p&gt;;We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1-4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems.&lt;;/p&gt;;&lt;;h3&gt;;Conclusions&lt;;/h3&gt;; &lt;;p&gt;;We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at https://github.com/rcs333/VAPiD.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;420463v1"/>
  <meta name="citation_id" content="420463v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/420463v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/420463v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/420463v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463.full.pdf"/>
  <meta name="citation_doi" content="10.1101/420463"/>
  <meta name="citation_num_pages" content="21"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="420463"/>
  <meta name="citation_author" content="Ryan C. Shean"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author" content="Negar Makhsous"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author" content="Graham D. Stoddard"/>
  <meta name="citation_author_institution" content="Department of Neurobiology, University of Washington"/>
  <meta name="citation_author" content="Michelle J. Lin"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author" content="Alexander L. Greninger"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_reference" content="Quick J, Loman NJ, Duraffour S, Simpson JT, Severi E, Cowley L, et al. Real-time, portable genome sequencing for Ebola surveillance. Nature. 2016;530:228–32."/>
  <meta name="citation_reference" content="Kozyreva VK, Jospin G, Greninger AL, Watt JP, Eisen JA, Chaturvedi V. Recent Outbreaks of Shigellosis in California Caused by Two Distinct Populations of Shigella sonnei with either Increased Virulence or Fluoroquinolone Resistance. mSphere. 2016;1:e00344–16."/>
  <meta name="citation_reference" content="Naccache SN, Thézé J, Sardi SI, Somasekar S, Greninger AL, Bandeira AC, et al. Distinct Zika Virus Lineage in Salvador, Bahia, Brazil. Emerg Infect Dis. 2016;22."/>
  <meta name="citation_reference" content="Greninger AL, Zerr DM, Qin X, Adler AL, Sampoleo R, Kuypers JM, et al. Rapid Metagenomic Next-Generation Sequencing during an Investigation of Hospital-Acquired Human Parainfluenza Virus 3 Infections. J Clin Microbiol. 2017;55:177–82."/>
  <meta name="citation_reference" content="Greninger AL, Messacar K, Dunnebacke T, Naccache SN, Federman S, Bouquet J, et al. Clinical metagenomic identification of Balamuthia mandrillaris encephalitis and assembly of the draft genome: the continuing case for reference genome sequencing. Genome Med. 2015;7:113."/>
  <meta name="citation_reference" content="Wood DE, Salzberg SL. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biol. 2014;15:R46."/>
  <meta name="citation_reference" content="Naccache SN, Federman S, Veeraraghavan N, Zaharia M, Lee D, Samayoa E, et al. A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples. Genome Res. 2014;24:1180–92."/>
  <meta name="citation_reference" content="Greninger AL, Knudsen GM, Betegon M, Burlingame AL, DeRisi JL. ACBD3 interaction with TBC1 domain 22 protein is differentially affected by enteroviral and kobuviral 3A protein binding. mBio. 2013;4:e00098–00013."/>
  <meta name="citation_reference" content="Gardy J, Loman NJ, Rambaut A. Real-time digital pathogen surveillance - the time is now. Genome Biol. 2015;16:155."/>
  <meta name="citation_reference" content="Yozwiak NL, Schaffner SF, Sabeti PC. Data sharing: Make outbreak research open access. Nature. 2015;518:477–9."/>
  <meta name="citation_reference" content="Allard MW, Strain E, Melka D, Bunning K, Musser SM, Brown EW, et al. Practical Value of Food Pathogen Traceability through Building a Whole-Genome Sequencing Network and Database. J Clin Microbiol. 2016;54:1975–83."/>
  <meta name="citation_reference" content="Kozyreva VK, Truong C-L, Greninger AL, Crandall J, Mukhopadhyay R, Chaturvedi V. Validation and Implementation of Clinical Laboratory Improvements Act-Compliant Whole-Genome Sequencing in the Public Health Microbiology Laboratory. J Clin Microbiol. 2017;55:2502–20."/>
  <meta name="citation_reference" content="Besser J, Carleton HA, Gerner-Smidt P, Lindsey RL, Trees E. Next-generation sequencing technologies and their application to the study and control of bacterial infections. Clin Microbiol Infect Off Publ Eur Soc Clin Microbiol Infect Dis. 2017;"/>
  <meta name="citation_reference" content="Seemann T. Prokka: rapid prokaryotic genome annotation. Bioinforma Oxf Engl. 2014;30:2068–9."/>
  <meta name="citation_reference" content="Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, et al. NCBI prokaryotic genome annotation pipeline. Nucleic Acids Res. 2016;44:6614–24."/>
  <meta name="citation_reference" content="Bao Y, Bolotov P, Dernovoy D, Kiryutin B, Tatusova T. FLAN: a web server for influenza virus genome annotation. Nucleic Acids Res. 2007;35:W280–4."/>
  <meta name="citation_reference" content="Greninger AL. A decade of RNA virus metagenomics is (not) enough. Virus Res. 2018;244:218–29."/>
  <meta name="citation_reference" content="Pelet T, Curran J, Kolakofsky D. The P gene of bovine parainfluenza virus 3 expresses all three reading frames from a single mRNA editing site. EMBO J. 1991;10:443–8."/>
  <meta name="citation_reference" content="Greninger AL, Pepper G, Shean RC, Cent A, Palileo I, Kuypers JM, et al. Myeloablation-associated deletion of ORF4 in a human coronavirus 229E infection. Npj Genomic Med. 2017;2:30."/>
  <meta name="citation_reference" content="Vidal S, Curran J, Kolakofsky D. Editing of the Sendai virus P/C mRNA by G insertion occurs during mRNA synthesis via a virus-encoded activity. J Virol. 1990;64:239–46."/>
  <meta name="citation_reference" content="citation_title=Bioinformatics for DNA Sequence Analysis. Methods in Molecular Biology (Methods and Protocols);citation_volume=537;citation_year=2009"/>
  <meta name="twitter:title" content="VAPiD: a lightweight cross platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available. GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions. Results We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1-4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems. Conclusions We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at &lt;;https://github.com/rcs333/VAPiD&gt;;."/>
  <meta name="og-title" property="og:title" content="VAPiD: a lightweight cross platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/420463v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available. GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions. Results We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1-4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems. Conclusions We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at &lt;;https://github.com/rcs333/VAPiD&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-09-18"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>VAPiD: a lightweight cross platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-126579 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <div id="sec-1" class="subsection">
     <p id="p-3">
      <strong>Background</strong> With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available.
     </p>
     <p id="p-4">GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions.</p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-5">
      <strong>Results</strong> We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1-4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-6">
      <strong>Conclusions</strong> We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at 
      <a href="https://github.com/rcs333/VAPiD">https://github.com/rcs333/VAPiD</a>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Background</h2>
    <p id="p-7">With sequencing technologies becoming cheaper and more accessible, genomic sequencing data is becoming increasingly widespread. Smaller groups are generating more sequencing data than they can analyze alone. In order to extract maximal scientific and public health value out of these data, sharing of assembled consensus genomes and raw sequence data is critical. The democratization of genomics takes a village. This is especially true for infectious diseases, where the searchable sequence databases allow for real-time tracking of viral epidemics and solving of foodborne bacteria outbreaks [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. While recent high-profile cases receive the most attention, the fact remains that almost all infectious diseases exist in the context of an ongoing outbreak [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. In the clinical world, metagenomic analysis pipelines depend on and take advantage of the availability of many infectious disease genomes to allow for faster and accurate alignments [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. In the basic science world, a graduate student studying the function of a singular protein is greatly assisted by being able to pull the world’s history of sequence diversity for that protein before designing experiments [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>].
    </p>
    <p id="p-8">Many foresee a world in which nearly every infectious disease genome is sequenced and archived in a publicly searchable database [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. State and federal public health laboratories have built capacity such that they now sequence more than 6,000 influenza virus genomes and more than 5,000 enteropathogenic bacterial genomes each year [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Major efforts in rationalizing workflows, from nucleic acid extraction to data deposition and analysis, have enabled these rapid growths in throughput [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. These tools allow public health and research laboratories to focus on the epidemiological or scientific insight gained from sequencing rather than rote protocols for data deposition. Specifically in the area of genome annotation, the National Center for Biotechnology Information (NCBI) has created the Prokaryotic Genome Annotation Pipeline, Eukaryotic Genome Annotation Pipeline, and the Influenza Virus Sequence Annotation Tool [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>].
    </p>
    <p id="p-9">Surprisingly, other than for influenza virus, NCBI GenBank does not currently have an automatic viral genome annotation pipeline. The incredible diversity of DNA and RNA viruses presents a challenge for development of a universal annotator [
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. Complex viral life cycles involving RNA editing, ribosomal slippage, and overlapping reading frames create additional annotation issues, along with non-standard nomenclature for viral gene products [
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>–
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>].
    </p>
    <p id="p-10">In order to accept submitted viral genomic data, NCBI GenBank requires 1) viral sequence complete with at least one protein annotation, 2) author/depositor metadata, and 3) viral sequence metadata, such as strain, collection date, collection location, and coverage. While manual annotation of nucleotide sequence can be done for small numbers of viruses, it is extremely time-consuming and labor-intensive. Even after correct annotations have been obtained for all viral sequences, manually integrating author and sample metadata to create files to be submitted is an equally time-consuming effort and not a feasible solution for groups sequencing more than a few viruses.</p>
    <p id="p-11">To date, existing viral annotation tools have focused largely on batch submissions of a single virus species. This may be a holdover from when specific PCR-based methods were required to faithfully recover viral whole genome sequence or due to the focus of researchers on a single virus at a time. With the increasing use of metagenomic or shotgun next-generation sequencing and the availability of more and more sequencing capacity, researchers can confidently batch many different RNA or DNA viruses together on a single sequencing run.</p>
    <p id="p-12">In order to facilitate viral genome annotation, we have developed a lightweight and user-friendly command-line tool that inputs FASTA files of complete or near complete viral genomes, automatically annotates them, and outputs the required files for GenBank submission over email. VAPiD handles batch submissions of multiple viruses of different types without prior knowledge of the viral species, correctly annotates RNA editing and ribosomal slippage, performs spellchecking on annotations, handles batch or individual submission of metadata, runs with a simple one-line command, and creates annotated viral sequence files for GenBank submission.</p>
    <div id="sec-5" class="subsection">
     <h3>Implementation</h3>
     <p id="p-13">VAPiD can be downloaded at 
      <a href="https://github.com/rcs333/VAPiD">https://github.com/rcs333/VAPiD</a>. An installation guide, usage instructions, and test data can also be found at the above website. The invocation of VAPiD is shown in 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, users must provide a standard FASTA format containing all of the viral genomes they wish to annotate. Users also must provide a GenBank submission template (.sbt file) that includes author, publication, and project metadata. The GenBank submission template can be used for multiple viral sequences or submissions and is easily created at the NCBI Submission portal (
      <a href="https://submit.ncbi.nlm.nih.gov/genbank/template/submission/">https://submit.ncbi.nlm.nih.gov/genbank/template/submission/</a>). An optional sample metadata file (.csv file) can also be provided to VAPiD to expedite the process of incorporating sample metadata. This optional file can also be used to include any of the Source Modifiers supported by NCBI (
      <a href="https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html">https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html</a>). If no sample metadata file is provided, VAPiD will prompt the user to input the required sample metadata at runtime. Additionally, users can provide a specified reference from which to annotate all viruses in a run, as well as provide their own BLASTn database or force VAPiD to search NCBI’s NT database over the internet.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Example usage of VAPiD. The two required files are shown as genome.fasta and author_info.sbt. genome.fasta is all of the viral genomes you wish to submit, named as you want them to appear on GenBank. In the example code provided in the github repository this example file is called example.fasta. The author_info.sbt file is an NCBI specific file for attaching sequence author names to sequin files and is a required part of properly submitting sequences to NCBI. This file can be generated at (https://submit.ncbi.nlm.nih.gov/genbank/template/submission/). The first optional command is a comma separated file in which you can include all relevant metadata. You can create additional columns here so long as they correspond to NCBI approved sequence metadata. A list and formatting requirements can be found at (https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html). Note that FASTA sequence names must be identical to names in the optional metadata sheet. Additionally, one could omit the metadata sheet and VAPiD will prompt for strain name, collection-date, country, and coverage data automatically at runtime. The second optional argument is a location of a local BLASTn database, which will force VAPiD to use the specified database instead of the included database. The last optional argument will force VAPiD to send an online search query to NCBI[[#x2019]]s NT database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1108958198" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Example usage of VAPiD. The two required files are shown as genome.fasta and author_info.sbt. genome.fasta is all of the viral genomes you wish to submit, named as you want them to appear on GenBank. In the example code provided in the github repository this example file is called example.fasta. The author_info.sbt file is an NCBI specific file for attaching sequence author names to sequin files and is a required part of properly submitting sequences to NCBI. This file can be generated at (https://submit.ncbi.nlm.nih.gov/genbank/template/submission/). The first optional command is a comma separated file in which you can include all relevant metadata. You can create additional columns here so long as they correspond to NCBI approved sequence metadata. A list and formatting requirements can be found at (https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html). Note that FASTA sequence names must be identical to names in the optional metadata sheet. Additionally, one could omit the metadata sheet and VAPiD will prompt for strain name, collection-date, country, and coverage data automatically at runtime. The second optional argument is a location of a local BLASTn database, which will force VAPiD to use the specified database instead of the included database. The last optional argument will force VAPiD to send an online search query to NCBI[[#x2019]]s NT database.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F1.medium.gif" width="440" height="139"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F1.medium.gif" width="440" height="139"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-14" class="first-child">Example usage of VAPiD. The two required files are shown as genome.fasta and author_info.sbt. genome.fasta is all of the viral genomes you wish to submit, named as you want them to appear on GenBank. In the example code provided in the github repository this example file is called example.fasta. The author_info.sbt file is an NCBI specific file for attaching sequence author names to sequin files and is a required part of properly submitting sequences to NCBI. This file can be generated at (
        <a href="https://submit.ncbi.nlm.nih.gov/genbank/template/submission/">https://submit.ncbi.nlm.nih.gov/genbank/template/submission/</a>). The first optional command is a comma separated file in which you can include all relevant metadata. You can create additional columns here so long as they correspond to NCBI approved sequence metadata. A list and formatting requirements can be found at (
        <a href="https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html">https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html</a>). Note that FASTA sequence names must be identical to names in the optional metadata sheet. Additionally, one could omit the metadata sheet and VAPiD will prompt for strain name, collection-date, country, and coverage data automatically at runtime. The second optional argument is a location of a local BLASTn database, which will force VAPiD to use the specified database instead of the included database. The last optional argument will force VAPiD to send an online search query to NCBI’s NT database.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-15">The VAPiD pipeline is summarized in 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The first step is finding the correct reference sequence. This is accomplished in three ways 1) using the provided reference database (default), 2) forcing VAPiD to execute an online BLASTn search of NCBI’s NT database, or 3) inputting the accession number of a single NCBI sequence to use as the reference.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="General design and information flow of VAPiD. First the provided sequences are used as queries for a local BLAST search (default) or an online BLASTn search. After results have been returned a reference annotation is downloaded, if a specific reference accession number is given than this reference is downloaded. Then the original FASTA file is aligned with the reference FASTA and the resulting alignment is used to map the reference annotations onto the new FASTA. Then custom code runs through the file and handles RNA editing, ribosomal slippage and splicing. These finalized annotations are then plugged into NCBI[[#x2019]]s tbl2asn with the author information and sequin files are generated as well as gbk files which can be used to manually verify accuracy of new annotations. Quality checked .sqn files can be emailed directly to GenBank." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1108958198" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;General design and information flow of VAPiD. First the provided sequences are used as queries for a local BLAST search (default) or an online BLASTn search. After results have been returned a reference annotation is downloaded, if a specific reference accession number is given than this reference is downloaded. Then the original FASTA file is aligned with the reference FASTA and the resulting alignment is used to map the reference annotations onto the new FASTA. Then custom code runs through the file and handles RNA editing, ribosomal slippage and splicing. These finalized annotations are then plugged into NCBI[[#x2019]]s tbl2asn with the author information and sequin files are generated as well as gbk files which can be used to manually verify accuracy of new annotations. Quality checked .sqn files can be emailed directly to GenBank.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F2.medium.gif" width="440" height="205"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F2.medium.gif" width="440" height="205"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/09/18/420463/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-16" class="first-child">General design and information flow of VAPiD. First the provided sequences are used as queries for a local BLAST search (default) or an online BLASTn search. After results have been returned a reference annotation is downloaded, if a specific reference accession number is given than this reference is downloaded. Then the original FASTA file is aligned with the reference FASTA and the resulting alignment is used to map the reference annotations onto the new FASTA. Then custom code runs through the file and handles RNA editing, ribosomal slippage and splicing. These finalized annotations are then plugged into NCBI’s tbl2asn with the author information and sequin files are generated as well as gbk files which can be used to manually verify accuracy of new annotations. Quality checked .sqn files can be emailed directly to GenBank.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">In the default case, NCBI’s BLAST+ tools are called from the command line to search against a reference database that is included with the VAPiD installation. This database was generated by downloading all complete viral genomes in NCBI on May 1, 2018. The best result from this search is passed as the reference into the next steps.</p>
     <p id="p-18">If the online option for finding the reference is specified, VAPiD finds an appropriate reference sequence for each genome to be annotated by performing an online BLASTn search with a word size of 28 using BioPython’s NCBIWWW.qblast() function against the online NCBI NT database. The BLASTn output is parsed for the best scoring alignment among the top 15 results that contains “complete genome” in the reference definition line. If no complete genome is found in the top 15 BLASTn results, it uses the top-scoring hit as the reference sequence.</p>
     <p id="p-19">If a specific reference is provided, VAPiD simply downloads it directly from NCBI. After the correct reference is downloaded, gene locations are stripped from the reference and a pairwise nucleotide alignment between the reference and the submitted sequence is generated using MAFFT [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. The relative locations of the genes on the reference sequence are then mapped onto the new sequence based off the alignment. This putative alignment only requires that start codons are in regions of high homology and does not rely on intergenic spacing or gene lengths. Gene names are taken from the annotated reference sequence GenBank entry. Automatic spellchecking is performed with NCBI’s ESpell module. This module provides automatic spellchecking of many biological strings including protein product names. An optional argument can be provided at execution that disables this automatic step.
     </p>
     <p id="p-20">The diverse array of methods viruses use to encode genes can present problems for any viral genome annotator. Ribosomal slippage allows viruses to produce two proteins from a single mRNA transcript by having the ribosome ‘slip’ one or two nucleotides along the mRNA transcript, thus changing the reading frame. Since ribosomal slippage is well conserved within viral species and complete reference genomes often list exactly where it occurs, custom code was used to strip the correct junction site and include it in the annotation.</p>
     <p id="p-21">RNA editing is another process by which viruses can include multiple proteins in a single gene. In RNA editing, the RNA polymerase co-transcriptionally adds one or two nucleotides that are not on the template. These changes are specifically created during viral mRNA transcription and not during viral genome replication. RNA editing presents an annotation issue because the annotated protein sequence does not match expected translated nucleotide sequence. To correctly annotate genes with RNA editing, VAPiD parses the reference genome viral species, detects the RNA editing locus, and mimics the RNA polymerase. VAPiD adds the correct number of non-templated nucleotides for the viral species and provides an alternative protein translation. This process is hard-coded for human parainfluenza 2-4, Nipah virus, Sendai virus, measles virus, and mumps virus. Although RNA editing occurs in Ebola virus, references for Ebola virus are annotated in the same way as ribosomal slippage, so code written for ribosomal slippage handles Ebola virus annotations.</p>
     <p id="p-22">After ribosomal slippage and RNA editing are processed, files required for GenBank submission are generated with the provided author and sample metadata. VAPiD first generates the .fsa file, .tbl file, and optional .cmt file. Submission files for each viral genome are packaged into a separate folder for each sequence. VAPiD then runs tbl2asn on each folder using the provided GenBank submission template file (.sbt). tbl2asn generates error reports and Sequin (.sqn) and GenBank (.gbk) files for manual verification and GenBank submission via email attachment to 
      <span class="em-link">
       <span class="em-addr">gb-admin{at}ncbi.nlm.nih.gov</span>
      </span>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Results and Discussion</h2>
    <p id="p-23">To illustrate our vision of how VAPiD will be useful to the scientific community we are providing two example use cases.</p>
    <div id="sec-7" class="subsection">
     <h3>Case Study 1</h3>
     <p id="p-24">This first example is the task that the authors originally wrote VAPiD for - annotating large numbers of genomes from different viral species, which mirrors the type of data that many clinical and public health laboratories may encounter. To illustrate this, 30 synthetic viral genomes were created by manually mutagenizing sequences downloaded from NCBI GenBank. Only point mutations that did not lead to stop codons were used and each sequence was given about 10 - 25 changes depending on the viral genome length, equal to roughly one amino acid change per 1,000 nucleotides. The species included were Nipah, Sendai, Measles, Mumps, Parainfluenza 1-4, Ebola, Rotavirus segments, MERS, SARS, Coronavirus 229E, West Nile Virus, HTLV, HIV1, Hepatitis A-C, Hepatitis E, Norovirus, Enterovirus, JC virus, and BK polyomavirus. All genomes were put together into a single FASTA file. Metadata (collection date, country and coverage) for each of the viral genomes was put together into a single csv file. VAPiD was then executed with the command [python vapid.py example.fasta example.sbt --metadata_loc example_metadata.csv]. After running for 123 seconds the program successfully generated 30 complete and correct NCBI submittable annotation files (*.sqn), which could be immediately emailed to NCBI. When these same sequences were run with the online option it took 280 minutes to complete, most of which was waiting for NCBI to return BLAST results. The above example shows how VAPiD would be ideal for busy groups who produce a diverse array of viral sequence such as public health or clinical testing labs.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Case Study 2</h3>
     <p id="p-25">The second example use case highlights both the cross platform functionality of VAPiD as well as the ability to manually enter NCBI required metadata at run time and use an online BLAST search - reducing the number of required files at runtime to two. For this example, a sample human parainfluenza 3 virus FASTA was created. No accompanying metadata file was created (unlike in the first example). Additionally, the syntax is exactly the same to use VAPiD across all operating systems. The single human parainfluenza 3 virus took approximately two minutes to annotate (including waiting about 30 seconds for NCBI blast results to be returned) on a Windows 10 Virtual machine with a single 2.36 GHz core and 2 Gb of RAM. This example highlights how VAPiD can be used to annotate and submit viruses with only two input files (the author template file only needs to be created once) on almost any Mac, Linux, or Windows computer with a python installation.</p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Comparisons</h3>
     <p id="p-26">Two other programs exist to handle viral annotation, JCVI’s VIGOR (
      <a href="https://doi.org/10.1186/1471-2105-11-451">https://doi.org/10.1186/1471-2105-11-451</a>) and the Broad Institute’s viral-ngs suite (
      <a href="https://viral-ngs.readthedocs.io/en/latest/">https://viral-ngs.readthedocs.io/en/latest/</a>) (Table). VIGOR is an extremely fast and automatic viral annotator that is able to correctly annotate any number of virus genomes without prior knowledge of viral species or type. VIGOR also is able to correctly handle RNA editing and ribosomal slippage – needing only an input FASTA file with the genome to be annotated, it can produce NCBI compatible .tbl annotation files. VIGOR works natively on Mac and Linux systems. The annotations that VIGOR produces are identical to those created by VAPiD except that VAPiD only annotated CDS regions whereas VIGOR will annotate all regions. Two primary advantages VAPiD offers over VIGOR are 1) VAPiD can be installed and run on Windows systems, and 2) VAPiD automates metadata and NCBI submission preparation steps. This fits our intended use case of a public health laboratory that may only own Windows machines and does not want to spend extra time worrying about scripting tbl2asn and manually preparing metadata files. For those with powerful Linux machines and some knowledge of bash scripting we recommend stitching VIGOR and tbl2asn together with custom scripts. However, for those without the infrastructure or time to pursue this path VAPiD offers a reasonable alternative.
     </p>
     <p id="p-27">The Broad Institute’s publicly available viral-ngs package, also developed in Python, represents another alternative to VAPiD. This suite of tools is custom developed for internal Broad Institute users but publicly available and well-documented on GitHub. Viral-ngs takes a reference genome and corresponding annotation file and transfers the reference annotations onto a new genome. This is very similar to the way VAPiD transfers annotation, except viral-ngs requires the user to provide a reference, which avoids the potential problem of low quality references propagating errors. Viral-ngs is well suited for annotating large numbers of the same type of virus all at the same time. Another advantage of viral-ngs is that it contains many tools for going from raw sequencing reads to complete viral genomes. While viral-ngs handles ribosomal slippage and most normal viral genes, because NCBI compatible annotation files do not contain enough information to correctly annotate RNA editing viral-ngs fails in these cases. Outside of cases involving RNA editing VAPiD and viral-ngs produce identical CDS annotations, with viral-ngs transferring all other features. This software, like VAPiD, is also cross platform and could be easily run on a Windows machine with a slightly modified Python installation. The main advantages VAPiD offers over viral-ngs are the ease of batching multiple viral types together and also as with VIGOR, the automated metadata input and NCBI packaging.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Limitations</h3>
     <p id="p-28">As with all software tools and especially gene annotation programs, VAPiD is not without limitations. VAPiD was designed purely to expedite the process of submitting large batches of different human virus genomes from a clinical laboratory. As such, a major limitation is that VAPiD expects a “complete genome” to use as a reference to be available for each of the viral sequences submitted to it. VAPiD is not the preferred annotation tool for novel or extremely divergent viral species. However, we find that complete reference genomes are available for most viral species of clinical importance, especially for viruses that are commonly sequenced in clinical or public health laboratories. A further limitation of VAPiD is that it does not perform 
      <em>ab initio</em> gene annotation. Instead VAPiD simply transfers annotations from the closest reference genome with a few quality control steps, such as allowing for slightly different gene lengths and RNA editing. A significant limitation of this strategy is that because the new annotation is generated from a downloaded reference, any errors that are in the downloaded reference will be transferred to the new genome. This means that VAPiD performs best on high-quality and accurate reference sequences. The downside of the VAPiD strategy is that if an inaccurate reference strain is used, error can propagate extremely fast. For example, early in our development we deposited roughly 60 human parainfluenza 3 viruses that had the matrix protein incorrectly annotated as ‘matrix potein’ [sic] due to a misspelling in the official NCBI reference sequence NC_001796 for HPIV3. To combat this problem, we have included a variety of ways to pull references, including directly specifying the reference. However, due to ease of implementation this functionality only works if all the viruses to be submitted are the same type. For those submitting many viruses of the same type we recommend using this feature to ensure annotation quality. VAPiD also contains code that overwrites misspellings in protein product names using NCBI’s ESpell utility. This spell-checking step automatically flags and corrects errors such as the misspelling mentioned above. A helpful message of what was corrected is printed to the console to prevent erroneous corrections. While this takes care of most misspellings, it is not a perfect fix, and viruses that we tested previously could have erroneous references uploaded in the future and further propagate error. It is for this reason that we highly recommend that users inspect and verify the accuracy of their new annotations, which have to potential to be used for future references.
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Conclusions</h2>
    <p id="p-29">VAPiD allows users to go from any number of viral genome sequences to GenBank-ready submission files (.sqn) with a single command. VAPiD runs from the command line with two required arguments and one optional sample metadata file, with the option of inputting minimal sample metadata via the command line at the time of annotation. The latest version of VAPiD can be found at 
     <a href="https://github.com/rcs333/VAPiD">https://github.com/rcs333/VAPiD</a> along with detailed installation and usage instructions. This software will allow public health, clinical virology, and research laboratories that do not have the resources to develop their own in-house genome annotation and submission tools or to adapt other pre-existing tools to quickly and easily share the sequence information that they generate on a regular basis with a minimal effort.
    </p>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Availability and Requirements</h2>
    <p id="p-30">Project name: VAPiD</p>
    <p id="p-31">Project Homepage: 
     <a href="https://github.com/rcs333/VAPiD">https://github.com/rcs333/VAPiD</a>
    </p>
    <p id="p-32">Operating System(s): Mac OS X, Linux, Windows 10</p>
    <p id="p-33">Programing Language: Python</p>
    <p id="p-34">Other requirements: Python 2.7.4 or higher, BioPython, tbl2asn, MAFFT, internet connection</p>
    <p id="p-35">License: MIT</p>
    <p id="p-36">Any restrictions to use by non-academics: None</p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">List of Abbreviations</h2>
    <dl id="def-list-1">
     <dt id="def-item-1">NCBI
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>National Center for Biotechnology Information
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-2">BLAST
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Basic Local Alignment Sequent Tool
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-3">VAPiD
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Viral Annotation Pipeline and iDentification
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-4">sqn
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Sequin file
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-5">sbt
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Submission Template file
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-6">csv
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Comma separated value file</dd>
    </dl>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">Declarations</h2>
    <div class="section" id="sec-15">
     <h3>Ethics approval and consent to participate</h3>
     <p id="p-43">Not applicable</p>
    </div>
    <div class="section" id="sec-16">
     <h3>Consent for publication</h3>
     <p id="p-44">Not applicable</p>
    </div>
    <div class="section" id="sec-17">
     <h3>Availability of data and material</h3>
     <p id="p-45">All data generated or analyzed during this study are included in this published article, GitHub, or NCBI BioProject PRJNA338014. Code and databases are available on GitHub (
      <a href="https://github.com/rcs333/VAPiD">https://github.com/rcs333/VAPiD</a>)
     </p>
    </div>
    <div class="section COI-statement" id="sec-18">
     <h3>Competing interests</h3>
     <p id="p-46">The authors declare that they have no competing interests.</p>
    </div>
    <div class="section" id="sec-19">
     <h3>Funding</h3>
     <p id="p-47">This project was supported by the University of Washington Department of Laboratory Medicine and Virology Division. Ryan Shean was funded by the Mary Gates Research Scholarship at the University of Washington.</p>
    </div>
    <div class="section" id="sec-20">
     <h3>Authors’ contributions</h3>
     <p id="p-48">RS wrote the code. NM generated sequences for beta-testing and development of the software. ML helped implement several features to correctly annotate RNA editing and ribosomal slippage. GS implemented cross-platform functionality. AG conceived of the software and oversaw its development. RS and AG wrote the paper.</p>
     <p id="p-49">All authors contributed to debugging and testing of the final version of the software and all authors read and approved the final manuscript.</p>
    </div>
   </div>
   <div id="sec-21" class="subsection">
    <p id="p-50">
     <strong>Table</strong> – Attribute comparison of VAPiD, VIGOR, and viral-ngs annotation pipelines
    </p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn" id="fn-1">
      <p id="p-1">
       <span class="em-link">
        <span class="em-addr">agrening{at}uw.edu</span>
       </span>, 
       <span class="em-link">
        <span class="em-addr">rcs333{at}uw.edu</span>
       </span>, 
       <span class="em-link">
        <span class="em-addr">negarm{at}uw.edu</span>
       </span>, 
       <span class="em-link">
        <span class="em-addr">gds6{at}uw.edu</span>
       </span>, 
       <span class="em-link">
        <span class="em-addr">mjlin{at}uw.edu</span>
       </span>, 
       <span class="em-link">
        <span class="em-addr">agrening{at}uw.edu</span>
       </span>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.1" data-doi="10.1038/nature16996">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Quick</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loman</span>
          <span class="cit-name-given-names">NJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duraffour</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simpson</span>
          <span class="cit-name-given-names">JT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Severi</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowley</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Real-time, portable genome sequencing for Ebola surveillance</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">530</span>:
         <span class="cit-fpage">228</span>–
         <span class="cit-lpage">32</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16996%26rft_id%253Dinfo%253Apmid%252F26840485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature16996&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26840485&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-420463v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kozyreva</span>
          <span class="cit-name-given-names">VK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jospin</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Watt</span>
          <span class="cit-name-given-names">JP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eisen</span>
          <span class="cit-name-given-names">JA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chaturvedi</span>
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">Recent Outbreaks of Shigellosis in California Caused by Two Distinct Populations of Shigella sonnei with either Increased Virulence or Fluoroquinolone Resistance</span>. 
         <abbr class="cit-jnl-abbrev">mSphere</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">1</span>:
         <span class="cit-fpage">e00344</span>–
         <span class="cit-lpage">16</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.volume%253D1%26rft.spage%253De00344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thézé</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sardi</span>
          <span class="cit-name-given-names">SI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Somasekar</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bandeira</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Distinct Zika Virus Lineage in Salvador, Bahia, Brazil</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-fpage">22</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.4" data-doi="10.1128/JCM.01881-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zerr</span>
          <span class="cit-name-given-names">DM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qin</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Adler</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sampoleo</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuypers</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Rapid Metagenomic Next-Generation Sequencing during an Investigation of Hospital-Acquired Human Parainfluenza Virus 3 Infections</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">55</span>:
         <span class="cit-fpage">177</span>–
         <span class="cit-lpage">82</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.01881-16%26rft_id%253Dinfo%253Apmid%252F27795347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjU1LzEvMTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDkvMTgvNDIwNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.5" data-doi="10.1186/s13073-015-0235-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Messacar</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dunnebacke</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Federman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bouquet</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical metagenomic identification of Balamuthia mandrillaris encephalitis and assembly of the draft genome: the continuing case for reference genome sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Genome Med</abbr>. 
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">7</span>:
         <span class="cit-fpage">113</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BMed%26rft.volume%253D7%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-015-0235-2%26rft_id%253Dinfo%253Apmid%252F26620704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13073-015-0235-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26620704&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-420463v1.6" data-doi="10.1186/gb-2014-15-3-r46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wood</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. 
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">15</span>:
         <span class="cit-fpage">R46</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24580807&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.7" data-doi="10.1101/gr.171934.113">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Federman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Veeraraghavan</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zaharia</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Samayoa</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>. 
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">24</span>:
         <span class="cit-fpage">1180</span>–
         <span class="cit-lpage">92</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171934.113%26rft_id%253Dinfo%253Apmid%252F24899342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE4MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA5LzE4LzQyMDQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.8" data-doi="10.1128/mBio.00098-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Knudsen</span>
          <span class="cit-name-given-names">GM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Betegon</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Burlingame</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DeRisi</span>
          <span class="cit-name-given-names">JL</span>
         </span>. 
         <span class="cit-article-title">ACBD3 interaction with TBC1 domain 22 protein is differentially affected by enteroviral and kobuviral 3A protein binding</span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>. 
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">4</span>:
         <span class="cit-fpage">e00098</span>–
         <span class="cit-lpage">00013</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D4%26rft.spage%253De00098%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00098-13%26rft_id%253Dinfo%253Apmid%252F23572552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/mBio.00098-13&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23572552&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.9" data-doi="10.1186/s13059-015-0726-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gardy</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loman</span>
          <span class="cit-name-given-names">NJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Real-time digital pathogen surveillance - the time is now</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. 
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">16</span>:
         <span class="cit-fpage">155</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D16%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0726-x%26rft_id%253Dinfo%253Apmid%252F27391693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13059-015-0726-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27391693&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.10" data-doi="10.1038/518477a">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yozwiak</span>
          <span class="cit-name-given-names">NL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schaffner</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sabeti</span>
          <span class="cit-name-given-names">PC</span>
         </span>. 
         <span class="cit-article-title">Data sharing: Make outbreak research open access</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">518</span>:
         <span class="cit-fpage">477</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1038%252F518477a%26rft_id%253Dinfo%253Apmid%252F25719649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/518477a&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25719649&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.11" data-doi="10.1128/JCM.00081-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Allard</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Strain</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Melka</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bunning</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Musser</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brown</span>
          <span class="cit-name-given-names">EW</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Practical Value of Food Pathogen Traceability through Building a Whole-Genome Sequencing Network and Database</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">54</span>:
         <span class="cit-fpage">1975</span>–
         <span class="cit-lpage">83</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00081-16%26rft_id%253Dinfo%253Apmid%252F27008877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjU0LzgvMTk3NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA5LzE4LzQyMDQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.12" data-doi="10.1128/JCM.00361-17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kozyreva</span>
          <span class="cit-name-given-names">VK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Truong</span>
          <span class="cit-name-given-names">C-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crandall</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mukhopadhyay</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chaturvedi</span>
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">Validation and Implementation of Clinical Laboratory Improvements Act-Compliant Whole-Genome Sequencing in the Public Health Microbiology Laboratory</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">55</span>:
         <span class="cit-fpage">2502</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00361-17%26rft_id%253Dinfo%253Apmid%252F28592550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjU1LzgvMjUwMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA5LzE4LzQyMDQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-420463v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Besser</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carleton</span>
          <span class="cit-name-given-names">HA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gerner-Smidt</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindsey</span>
          <span class="cit-name-given-names">RL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Trees</span>
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-article-title">Next-generation sequencing technologies and their application to the study and control of bacterial infections</span>. 
         <abbr class="cit-jnl-abbrev">Clin Microbiol Infect Off Publ Eur Soc Clin Microbiol Infect Dis</abbr>. 
         <span class="cit-pub-date">2017</span>;
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seemann</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Prokka: rapid prokaryotic genome annotation</span>. 
         <abbr class="cit-jnl-abbrev">Bioinforma Oxf Engl</abbr>. 
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">30</span>:
         <span class="cit-fpage">2068</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinforma%2BOxf%2BEngl%26rft.volume%253D30%26rft.spage%253D2068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.15" data-doi="10.1093/nar/gkw569">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tatusova</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DiCuccio</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Badretdin</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chetvernin</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nawrocki</span>
          <span class="cit-name-given-names">EP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zaslavsky</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">NCBI prokaryotic genome annotation pipeline</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">44</span>:
         <span class="cit-fpage">6614</span>–
         <span class="cit-lpage">24</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw569%26rft_id%253Dinfo%253Apmid%252F27342282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw569&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27342282&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.16" data-doi="10.1093/nar/gkm354">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bao</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bolotov</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dernovoy</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kiryutin</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tatusova</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">FLAN: a web server for influenza virus genome annotation</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. 
         <span class="cit-pub-date">2007</span>;
         <span class="cit-vol">35</span>:
         <span class="cit-fpage">W280</span>–
         <span class="cit-lpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm354%26rft_id%253Dinfo%253Apmid%252F17545199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkm354&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17545199&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000255311500052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.17" data-doi="10.1016/j.virusres.2017.10.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>. 
         <span class="cit-article-title">A decade of RNA virus metagenomics is (not) enough</span>. 
         <abbr class="cit-jnl-abbrev">Virus Res</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">244</span>:
         <span class="cit-fpage">218</span>–
         <span class="cit-lpage">29</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D244%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pelet</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Curran</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kolakofsky</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">The P gene of bovine parainfluenza virus 3 expresses all three reading frames from a single mRNA editing site</span>. 
         <abbr class="cit-jnl-abbrev">EMBO J</abbr>. 
         <span class="cit-pub-date">1991</span>;
         <span class="cit-vol">10</span>:
         <span class="cit-fpage">443</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DPelet%26rft.auinit1%253DT%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D443%26rft.epage%253D448%26rft.atitle%253DThe%2BP%2Bgene%2Bof%2Bbovine%2Bparainfluenza%2Bvirus%2B3%2Bexpresses%2Ball%2Bthree%2Breading%2Bframes%2Bfrom%2Ba%2Bsingle%2BmRNA%2Bediting%2Bsite%26rft_id%253Dinfo%253Apmid%252F1846805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=1846805&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F09%2F18%2F420463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-420463v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pepper</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shean</span>
          <span class="cit-name-given-names">RC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cent</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palileo</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuypers</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Myeloablation-associated deletion of ORF4 in a human coronavirus 229E infection</span>. 
         <abbr class="cit-jnl-abbrev">Npj Genomic Med</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">2</span>:
         <span class="cit-fpage">30</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNpj%2BGenomic%2BMed%26rft.volume%253D2%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-420463v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vidal</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Curran</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kolakofsky</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Editing of the Sendai virus P/C mRNA by G insertion occurs during mRNA synthesis via a virus-encoded activity</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>. 
         <span class="cit-pub-date">1990</span>;
         <span class="cit-vol">64</span>:
         <span class="cit-fpage">239</span>–
         <span class="cit-lpage">46</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DVidal%26rft.auinit1%253DS%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D246%26rft.atitle%253DEditing%2Bof%2Bthe%2BSendai%2Bvirus%2BP%252FC%2BmRNA%2Bby%2BG%2Binsertion%2Boccurs%2Bduring%2BmRNA%2Bsynthesis%2Bvia%2Ba%2Bvirus-encoded%2Bactivity.%26rft_id%253Dinfo%253Apmid%252F1688384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjY0LzEvMjM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDkvMTgvNDIwNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-book" id="cit-420463v1.21">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Posada</span>
           <span class="cit-name-given-names">D.</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Katoh</span>
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Asimenos</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Toh</span>
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-chapter-title">Multiple Alignment of DNA Sequences with MAFFT</span>. In: 
         <span class="cit-ed">
          <span class="cit-name-surname">Posada</span>
          <span class="cit-name-given-names">D.</span>
         </span> (eds) 
         <span class="cit-source">Bioinformatics for DNA Sequence Analysis. Methods in Molecular Biology (Methods and Protocols)</span>, vol 
         <span class="cit-vol">537</span>. 
         <span class="cit-publ-name">Humana Press</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
