<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/01/19/101774.atom" />
<meta name="HW.pisa" content="biorxiv;101774v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species" />
<meta name="DC.Identifier" content="10.1101/101774" />
<meta name="DC.Date" content="2017-01-19" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Taxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus." />
<meta name="DC.Contributor" content="Gary R. Whittaker" />
<meta name="DC.Contributor" content="Nicole M. André" />
<meta name="DC.Contributor" content="Jean Kaoru Millet" />
<meta name="article:published_time" content="2017-01-19" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Taxonomic classification for the &lt;i&gt;Coronaviridae&lt;/i&gt; can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the &lt;i&gt;Alphacoronavirus&lt;/i&gt; genus, there is a single species &lt;i&gt;Alphacoronavirus 1&lt;/i&gt; that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the &lt;i&gt;Alphacoronavirus&lt;/i&gt; genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the &lt;i&gt;Alphacoronavirus 1&lt;/i&gt; species. &lt;i&gt;Alphacoronavirus 1&lt;/i&gt; clade A encompasses serotype I FCoV and CCoV, and &lt;i&gt;Alphacoronavirus 1&lt;/i&gt; clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed &lt;i&gt;Alphacoronavirus 2&lt;/i&gt; species, as an improved way to classify the diverse &lt;i&gt;Alphacoronavirus&lt;/i&gt; genus.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;101774v1" />
<meta name="citation_id" content="101774v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/101774v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/101774v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/101774v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774.full.pdf" />
<meta name="citation_doi" content="10.1101/101774" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="101774" />
<meta name="citation_author" content="Gary R. Whittaker" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University" />
<meta name="citation_author" content="Nicole M. André" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University" />
<meta name="citation_author" content="Jean Kaoru Millet" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8373-3688" />
<meta name="citation_reference" content="de Groot RJ, Cowley JA, Enjuanes L, Faaberg KS, Perlman S, Rottier PJM, Snijder EJ, Ziebuhr J, Gorbalenya AE. 2012. Order - Nidovirales, p 784–794, In king AMQ, J. Am, B. Ce, J. Le (ed), Virus Taxonomy doi:http://dxdoi.org/10.1016/B978-0-12-384684-6.00066-5 Elsevier, San Diego." />
<meta name="citation_reference" content="Lauber C, Ziebuhr J, Junglen S, Drosten C, Zirkel F, Nga PT, Morita K, Snijder EJ, Gorbalenya AE. 2012. Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses. Arch Virol 157:1623–8." />
<meta name="citation_reference" content="Woo PCY, Lau SKP, Lam CSF, Lau CCY, Tsang AKL, Lau JHN, Bai R, Teng JLL, Tsang CCC, Wang M, Zheng B-J, Chan K-H, Yuen K-Y. 2012. Discovery of Seven Novel Mammalian and Avian Coronaviruses in the Genus Deltacoronavirus Supports Bat Coronaviruses as the Gene Source of Alphacoronavirus and Betacoronavirus and Avian Coronaviruses as the Gene Source of Gammacoronavirus and Deltacoronavirus. Journal of Virology 86:3995–4008." />
<meta name="citation_reference" content="Dudas G, Rambaut A. 2016.MERS-CoV recombination: implications about the reservoir and potential for adaptation. Virus Evolution 2: vev023" />
<meta name="citation_reference" content="Graham RL, Baric RS. 2010. Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission. Journal of Virology 84:3134–3146." />
<meta name="citation_reference" content="Carstens EB. 2010. Ratification vote on taxonomic proposals to the International Committee on Taxonomy of Viruses (2009). Arch Virol 155:133–46." />
<meta name="citation_reference" content="Haijema BJ, Rottier P, de Groot RP. 2007. Feline Coronaviruses: A Tale of Two-Faced Types, p 183–204. In Thiel V (ed), Coronaviruses Molecular and Cellular Biology. Caister Academic Press, Norfolk, United Kingdom." />
<meta name="citation_reference" content="Vennema H, Poland A, Foley J, Pedersen NC. 1998. Feline Infectious Peritonitis Viruses Arise by Mutation from Endemic Feline Enteric Coronaviruses. Virology 243:150–157." />
<meta name="citation_reference" content="Garner MM, Ramsell K, Morera N, Juan-Salles C, Jimenez J, Ardiaca M, Montesinos A, Teifke JP, Lohr CV, Evermann JF, Baszler TV, Nordhausen RW, Wise AG, Maes RK, Kiupel M. 2008. Clinicopathologic features of a systemic coronavirus-associated disease resembling feline infectious peritonitis in the domestic ferret (Mustela putorius). Vet Pathol 45:236–46." />
<meta name="citation_reference" content="Licitra BN, Duhamel GE, Whittaker GR. 2014. Canine enteric coronaviruses: emerging viral pathogens with distinct recombinant spike proteins. Viruses 6:3363–76." />
<meta name="citation_reference" content="Belouzard S, Millet JK, Licitra BN, Whittaker GR. 2012. Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein. Viruses 4:1011–1033." />
<meta name="citation_reference" content="Millet JK, Whittaker GR. 2015. Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis. Virus Research 202:120–134." />
<meta name="citation_reference" content="Pedersen NC, Black JW, Boyle JF, Evermann JF, McKeirnan AJ, Ott RL. 1984. Pathogenic Differences Between Various Feline Coronavirus Isolates, p 365–380. In Rottier PJM, van der Zeijst BAM, Spaan WJM, Horzinek MC (ed), Molecular Biology and Pathogenesis of Coronaviruses doi:10.1007/978-1-4615-9373-7_36. Springer US, Boston, MA." />
<meta name="citation_reference" content="Hohdatsu T, Okada S, Koyama H. 1991. Characterization of monoclonal antibodies against feline infectious peritonitis virus type II and antigenic relationship between feline, porcine, and canine coronaviruses. Archives of Virology 117:85–95." />
<meta name="citation_reference" content="Decaro N, Buonavoglia C. 2008. An update on canine coronaviruses: viral evolution and pathobiology. Vet Microbiol 132:221–34." />
<meta name="citation_reference" content="Hohdatsu T, Izumiya Y, Yokoyama Y, Kida K, Koyama H. 1998. Differences in virus receptor for type I and type II feline infectious peritonitis virus. Archives of Virology 143:839–50." />
<meta name="citation_reference" content="Dye C, Temperton N, Siddell SG. 2007. Type I feline coronavirus spike glycoprotein fails to recognize aminopeptidase N as a functional receptor on feline cell lines. Journal of General Virology 88:1753–1760." />
<meta name="citation_reference" content="Decaro N, Martella V, Elia G, Campolo M, Desario C, Cirone F, Tempesta M, Buonavoglia C. 2007. Molecular characterisation of the virulent canine coronavirus CB/05 strain. Virus Res 125:54–60." />
<meta name="citation_reference" content="Herrewegh AA, Smeenk I, Horzinek MC, Rottier PJ, de Groot RJ. 1998. Feline coronavirus type II strains 79-1683 and 79-1146 originate from a double recombination between feline coronavirus type I and canine coronavirus. Journal of Virology 72:4508–14." />
<meta name="citation_reference" content="Regan AD, Millet JK, Tse LP, Chillag Z, Rinaldi VD, Licitra BN, Dubovi EJ, Town CD, Whittaker GR. 2012. Characterization of a recombinant canine coronavirus with a distinct receptor-binding (S1) domain. Virology 430:90–99." />
<meta name="citation_reference" content="Licitra BN, Millet JK, Regan AD, Hamilton BS, Rinaldi VD, Duhamel GE, Whittaker GR. 2013. Mutation in spike protein cleavage site and pathogenesis of feline coronavirus. Emerg Infect Dis 19:1066–73." />
<meta name="citation_reference" content="de Haan CAM, Haijema BJ, Schellen P, Schreur PW, te Lintelo E, Vennema H, Rottier PJM. 2008. Cleavage of Group 1 Coronavirus Spike Proteins: How Furin Cleavage Is Traded Off against Heparan Sulfate Binding upon Cell Culture Adaptation. Journal of Virology 82:6078–6083." />
<meta name="citation_reference" content="Lamers MM, Smits SL, Hundie GB, Provacia LB, Koopmans MP, Osterhaus AD, Haagmans BL, Raj VS. 2016. Naturally occurring recombination in ferret coronaviruses revealed by complete genome characterization. J Gen Virol doi:10.1099/jgv.0.000520." />
<meta name="citation_reference" content="World Health Organization/World Organisation for Animal HF, Agriculture Organization HNEWG. 2014. Revised and updated nomenclature for highly pathogenic avian influenza A (H5N1) viruses. Influenza Other Respir Viruses 8:384–8." />
<meta name="citation_reference" content="Guindon S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O. 2010. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst Biol 59:307–21." />
<meta name="twitter:title" content="Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Taxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus." />
<meta name="og-title" property="og:title" content="Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/101774v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Taxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-01-19" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/101774v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/101774v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/101774v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/101774v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/28926" />
    <title>Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"riB_vQXLw8dqBirpFEx6cqfCUEofrrjLSXjfsF23D7E","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"28926","apath":"\/biorxiv\/early\/2017\/01\/19\/101774.atom","pisa":"biorxiv;101774v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;101774v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/101774v1","title":"Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species","identifier":"node\/28926"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221O1iIAvBk+Sf9IUB\\\\\\\/BseDWKnmPH\\\\\\\/sly8wiDrgcRN2yLLdiiQRyjv94XyUIeAG6VVDQCuevOwC5RNvPpE3XHrWWCLIMJLYnd0+KGOCu7nSabIjfOKWmJrG8rGVzAGSoXRGtFajX0GfBfLtnWrkLKYBH4HIEAu0SsfmM\\\\\\\/0JGOTPG8KxNC+LGY3pIbDzo+HK\\\\\\\/ILklE6QeEMQG8TIh\\\\\\\/bx\\\\\\\/EW1GL7zmXce2PjVbE+OR8fQUTBCHNErUH1u8TWi3KsoyYXQEXF35kLMBzj+mcfRCa8T7yIav1aiwjeBM772e5Cnxi6j5jo6uiiq7aoQKjP91eA3tZqkS1FxrGGy7mG2nb\\\\\\\/mB2QZBU+fmRUQyOD0eupDYb52Ug4+qW6TsCDOWIN7okSSpV3bQ3WETr94G7eohEMqRtQi3KAXbx2WBOSB+\\\\\\\/srbmD9x8MzOTYAL6ayz9Cbr9TrGk0UHVRLZIUOEyqA8U3G38RTR61fkW8im3n1oDLeqDnLiMCcjHYFx7be8id8mQqC\\\\\\\/RBqJ7WXYQ8xjNfIClSSe2Mhj+4J8XQHnwYFjkuwIQteQ+ipF7xOQj\\\\\\\/3zX\\\\\\\/5XzXlR2vdkq4kRdBTo3vz\\\\\\\/2QzGRJZ\\\\\\\/24zfjoJym3wq50GhMkRSBLLabYtK+kIxYXCRQZqA5bQ9cKWmKgUDkr1t8B+QM49NX2t\\\\\\\/FYBK50Sfmo6ofAn27d6i34foMfXTCEuP+LMPr5pGmTu\\\\\\\/Z3Zqg\\\\\\\/ccMz8kup+iTj40l+haJS6fJSZUhpDlWxT6CwxaeO14oKgy7iLryRPZKKvve8WRNmx8DsqVrvtg\\\\\\\/bph9M+Qns733ZxYpVwNnbx2ON7q8P0g1LlnuGIKuVNYnFlnGxy3iooUPiK1d1PEDcJFjrAn6PQlyf7w2u9NirsBQLO\\\\\\\/9hKBSi59TSg8vQDtwVLPhX0+2Ni3WqXyaZhJ3SPiTGayRFzdCoihpnf8oJPSCWJa8jmQKc\\\\\\\/xkpBIoeTrzr\\\\\\\/df\\\\\\\/bSWF5Pzveg6J0CVncje5bHAJ9Ors2eSLW9cAA0IVXE8lsXdZvi3CErQHi\\\\\\\/NyxgcXjBvLEGDKvyEd2GItl1IB+d8tC5ajPSCjBzwwcw9Ovn\\\\\\\/F40z5dwc6OrVOhr7YGuZEPb3B3gMuS166ZDUEGTDZ\\\\\\\/6cCogPTfV\\\\\\\/d1T4Z9YWMBFVlPYZCIWdCNrwamx2Sl7P33WezkylEKxxnMXhCosBBh12Ikrg\\\\\\\/exwIz\\\\\\\/fenY+DnIFOxEOZtyts\\\\\\\/ovBkvAdPlZDVQAYJe0Wt8fKVWYcK7nrexsDAIhHwBY4ij4QLnEhI4eKWrOtv2qXYkHlQChYCDTEj7ut4X3RlK2srhSTfd88Frkqhumh8Ls=\\\u0022,\\\u0022iv\\\u0022:\\\u0022M4uKVhhSlgnuo5UpPFrxlw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b2f6a1c402189977e5cebd6de0bac23c55bde6f7e07aadc8facb5ef7959c2f0f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/101774v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-28926 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/101774v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NeJdekmTxCxAytfz04vDhSKVmDqKOm_wD7ikbQ2Bbxc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="28926" id="node28926" data-pisa="biorxiv;101774v1" data-pisa-master="biorxiv;101774" data-apath="/biorxiv/early/2017/01/19/101774.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Spike-based phylogenetically defined clades within the <em>Alphacoronavirus 1</em> species</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gary R.</span> <span class="nlm-surname">Whittaker</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nicole M.</span> <span class="nlm-surname">André</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8373-3688" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jean Kaoru</span> <span class="nlm-surname">Millet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/101774 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node28926" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gary R. Whittaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University</span>, Ithaca NY 14853, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BR.%2BWhittaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whittaker%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BR.%2BWhittaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicole M. André </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University</span>, Ithaca NY 14853, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicole%2BM.%2BAndr%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andr%C3%A9%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicole%2BM.%2BAndr%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean Kaoru Millet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, College of Veterinary Medicine, Cornell University</span>, Ithaca NY 14853, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BKaoru%2BMillet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Millet%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BKaoru%2BMillet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8373-3688" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean Kaoru Millet</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/101774v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:28926" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:28926/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/101774v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:28926" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:28926/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/101774v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:28926" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:28926/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/101774v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:28926" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:28926/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/101774v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:28926" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:28926/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Taxonomic classification for the <em>Coronaviridae</em> can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the <em>Alphacoronavirus</em> genus, there is a single species <em>Alphacoronavirus 1</em> that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the <em>Alphacoronavirus</em> genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the <em>Alphacoronavirus 1</em> species. <em>Alphacoronavirus 1</em> clade A encompasses serotype I FCoV and CCoV, and <em>Alphacoronavirus 1</em> clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed <em>Alphacoronavirus 2</em> species, as an improved way to classify the diverse <em>Alphacoronavirus</em> genus.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Members of the <em>Coronaviridae</em> family form a diverse group of enveloped, single-strand, positive-sense RNA viruses. The <em>Coronaviridae</em> family is divided into the <em>Torovirinae</em> and <em>Coronavirinae</em> subfamilies, both of which contain viral species characterized by their exceptionally large RNA genomes in the 26.4-31.7 kb and 26.6-28.5 kb ranges, respectively. <em>Torovirinae</em> and <em>Coronavirinae</em> subfamilies members are able to infect a diverse array of vertebrate species. They have a distinct genomic architecture and replication strategy shared with other members of the <em>Nidovirales</em> order, which also includes the <em>Arteriviridae</em>, <em>Mesoniviridae</em>, and <em>Roniviridae</em> families (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Coronaviruses (CoV) are classified into four genera, with <em>Alphacoronavirus</em> and <em>Betacoronavirus</em> containing members that infect mostly mammalian species, and <em>Gammacoronavirus</em> and <em>Deltacoronavirus</em> grouping viruses infecting both birds and mammals (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). The zoonotic emergence of highly pathogenic human betacoronaviruses, severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002, and Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012, have renewed interest for the study of coronaviruses and highlighted their propensity for recombination and to cross the species barrier (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>).</p><p id="p-4">The <em>Alphacoronavirus</em> genus is composed of viruses infecting bats, ferrets, mink, cats, dogs, pigs, and humans. The prototypical species <em>Alphacoronavirus 1</em> is composed of the following prototypical virus strains (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>): feline coronavirus (FCoV), canine coronavirus (CCoV), and transmissible gastrointestinal enteric virus (TGEV). FCoV is an <em>Alphacoronavirus</em> of particular interest as it manifests as two distinct biotypes or pathotypes with a highly transmissible form, feline enteric coronavirus (FECV) that provokes self-limiting, usually mild enteric tract infections, and a systemic form, feline infectious peritonitis virus (FIPV), typically associated with low transmissibility but high morbidity (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). In the widely accepted “internal mutation” hypothesis, it is believed that genetic mutations in the genome of FCoV occur within an infected animal, giving rise to FIPV (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). A similar FIP-like pathogenesis phenomenon is also observed with ferret coronaviruses (FRCoV) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). While CCoV is a widespread enteric virus of dogs and can occur in highly pathogenic forms, the virus does not manifest itself with FIP-like clinical signs (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>).</p><p id="p-5">The coronavirus spike (S) envelope glycoprotein, the main determinant of virus entry, is an essential structural protein as it governs binding to the host cell receptor, mediates viral membrane fusion, and is typically proteolytically processed by host cell proteases to activate its fusogenicity (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). As such, the coronavirus S protein is a crucially important viral component as it determines to a large extent host species, tissue, and cell tropism as well as pathogenicity and transmission. Previous serological characterizations of alphacoronaviruses, based on the antigenicity of the S glycoprotein, have revealed the existence of two FCoV serotypes (serotype I and II) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Both serotype I and II FCoV can manifest as either FECV and FIPV biotypes. FCoV serotype I is more prevalent in cats than serotype II, but has proved more difficult to culture <em>in vitro</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). Similarly in strains of CCoV, a common enteric virus of dogs, two serotypes (CCoV I and II) have also been characterized, and are distinguished by genetic differences in the S and ORF3 genes. Serotype II CCoV strains can be further subdivided into the IIa, IIb, and IIc subtypes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). CCoV IIa and IIb strains are distinguished by differences in the N-terminal domain of the S protein (NTD), where the IIb NTD is closely related to TGEV NTD. The recently characterized IIc subtype of CCoV has been reported in Sweden and in the United States.</p><p id="p-6">The evolution of strains within the <em>Alphacoronavirus 1</em> species is complex and likely involved a number of recombination events. It is thought that serotype I FCoV and CCoV originated from a common ancestor. A recombination event occurring between a serotype I CCoV with an unknown coronavirus gave rise to serotype II CCoV which acquired a recombinant S protein, distinct from serotype I S envelope glycoprotein. TGEV appears to have originated from a serotype II CCoV (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). Additional, independent recombination events between serotype I FCoV and serotype II CCoV gave rise to serotype AI FCoV, such as FIPV WSU-79-1146 and FECV WSU-79-1683, which acquired a serotype II CCoV S protein (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Furthermore, we have previously shown that CCoV strain A76 also has a recombinant S protein, a product of recombination between serotype I and II CCoV sequences (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). CCoV-A76 S was shown to have a serotype I-like NTD, while the rest of the protein was serotype II-like. Analysis of coronavirus recombination events within the S protein sequence revealed its modular nature, allowing exchanges of functional domains between co-infecting viruses (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>).</p><p id="p-7">Because of the numerous recombination events occurring within the S gene of <em>Alphacoronavirus</em> 1 species, current classification of <em>Alphacoronavirus 1</em> strains are often not well defined and fail to recapitulate the previously established serotype demarcation. However, in addition to serological differences, serotype I and II S proteins are fundamentally different in several biological aspects. While the receptor for serotype II FCoV, CCoV, and TGEV has been shown to be aminopeptidase N (APN, or CD13), the receptor for serotype I strains remains unknown. Furthermore, the S protein of serotype I strains contain an additional cleavage site, the S1/S2 site, which is not present in serotype II or in other <em>Alphacoronavirus</em> S proteins. This site has been shown to be important for cell culture adaptation and pathogenesis of FCoV (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Because of the critical role played by the S glycoprotein in virus entry, pathogenesis, and tropism, and since the S proteins of serotype I and II strains differ greatly, we propose a classification of <em>Alphacoronavirus 1</em> strains into two clades (<em>Alphacoronavirus 1</em> clade A and B), using a functionally-based S protein sequence classification that reflects the previously determined serologically-based demarcation.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-8">As a starting point to gain a better understating of the phylogenetic relationships between alphacoronaviruses, we generated a phylogenetic tree of key representative species and strains based on complete genome nucleotide sequence alignment, a method often used to classify coronaviruses (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). As expected, the <em>Alphacoronavirus 1</em> FCoV, CCoV, and TGEV strains formed a well-defined monophyletic group. The analysis also reveals the clearly delineated branching of coronavirus strains that infect ferrets and mink, FrCoV-NL-2010, MinkCoV-WD1127, and MinkCoV-WD1133, which were recently proposed to form a separate species, <em>Alphacoronavirus 2</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). Two main subgroupings within the <em>Alphacoronavirus 1</em> species were observed with CCoV and TGEV partitioning into one subgroup and FCoV forming a second, separate subgroup. However, as shown by the CCoV/FCoV serotype highlighting, the complete genome-based phylogenetic tree fails to cluster CCoV and FCoV strains according to serotype demarcations, and instead groups strains according to their host species.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic analysis of alphacoronaviruses based on complete genome and ORF1ab protein sequence. Nucleotide sequences of the complete genomes of alphacoronaviruses and of the Betacoronavirus strain MHV-A59 (A) or the complete protein sequences of the ORF1ab polyprotein of the corresponding viruses (B) were aligned using MAFFT (http://mafft.cbrc.jp/alignment/software/) within Geneious 10 software package. The alignment was then used to generate a maximum-likelihood phylogenetic tree using PhyML (25). The tree was rooted with MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete genome nucleotide sequences used: CCoV-23/03 (KP849472.1) , CCoV-1-71 (JQ404409.1), CCoV-A76 (JN856008.2), TGEV-Purdue (AJ271965.2), FCoV-TN406 (EU186072.1), FCoV-RM (FJ938051.1), FCoV-WSU-79-1146 (NC_002306.3), FCoV-WSU-79-1683 (JN634064.1), FRCoV-NL-2010 (NC_030292.1), MinkCoV-WD1127 (HM245925.1), MinkCoV-WD1133 (HM245926.1), PEDV-CV777 (AF353511.1), BatCoV-HKU10 (JQ989270.1), HCoV-229E (KU291448.1), HCoV-NL63 (AY567487.2), MHV-A59 (AY700211.1). Accession numbers for complete ORF1ab protein sequences used: CCoV-23/03 (AKZ66481.1), CCoV-1-71 (AFG19735.1), CCoV-A76 (AEQ61967.2), TGEV-Purdue (P0C6Y5.1), FCoV-TN406 (ABX60144.1), FCoV-RM (ACT10853.1), FCoV-WSU-79-1146 (YP_004070193.2), FCoV-WSU-79-1683 (AFH58022.1), FRCoV-NL-2010 (YP_009256195.1), MinkCoV-WD1127 (ADI80512.1), MinkCoV-WD1133 (ADI80522.1), PEDV-CV777 (P0C6Y4.1), BatCoV-HKU10 (AFU92103.1), HCoV-229E (AOG74782.1), HCoV-NL63 (AAS58176.2), MHV-A59 (AAU06353.1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2098560117" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Phylogenetic analysis of alphacoronaviruses based on complete genome and ORF1ab protein sequence.&lt;/span&gt; Nucleotide sequences of the complete genomes of alphacoronaviruses and of the &lt;em&gt;Betacoronavirus&lt;/em&gt; strain MHV-A59 (&lt;strong&gt;A&lt;/strong&gt;) or the complete protein sequences of the ORF1ab polyprotein of the corresponding viruses (&lt;strong&gt;B&lt;/strong&gt;) were aligned using MAFFT (http://mafft.cbrc.jp/alignment/software/) within Geneious 10 software package. The alignment was then used to generate a maximum-likelihood phylogenetic tree using PhyML (25). The tree was rooted with MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete genome nucleotide sequences used: CCoV-23/03 (KP849472.1) , CCoV-1-71 (JQ404409.1), CCoV-A76 (JN856008.2), TGEV-Purdue (AJ271965.2), FCoV-TN406 (EU186072.1), FCoV-RM (FJ938051.1), FCoV-WSU-79-1146 (NC_002306.3), FCoV-WSU-79-1683 (JN634064.1), FRCoV-NL-2010 (NC_030292.1), MinkCoV-WD1127 (HM245925.1), MinkCoV-WD1133 (HM245926.1), PEDV-CV777 (AF353511.1), BatCoV-HKU10 (JQ989270.1), HCoV-229E (KU291448.1), HCoV-NL63 (AY567487.2), MHV-A59 (AY700211.1). Accession numbers for complete ORF1ab protein sequences used: CCoV-23/03 (AKZ66481.1), CCoV-1-71 (AFG19735.1), CCoV-A76 (AEQ61967.2), TGEV-Purdue (P0C6Y5.1), FCoV-TN406 (ABX60144.1), FCoV-RM (ACT10853.1), FCoV-WSU-79-1146 (YP_004070193.2), FCoV-WSU-79-1683 (AFH58022.1), FRCoV-NL-2010 (YP_009256195.1), MinkCoV-WD1127 (ADI80512.1), MinkCoV-WD1133 (ADI80522.1), PEDV-CV777 (P0C6Y4.1), BatCoV-HKU10 (AFU92103.1), HCoV-229E (AOG74782.1), HCoV-NL63 (AAS58176.2), MHV-A59 (AAU06353.1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F1.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F1.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Phylogenetic analysis of alphacoronaviruses based on complete genome and ORF1ab protein sequence.</span><p id="p-9" class="first-child">Nucleotide sequences of the complete genomes of alphacoronaviruses and of the <em>Betacoronavirus</em> strain MHV-A59 (<strong>A</strong>) or the complete protein sequences of the ORF1ab polyprotein of the corresponding viruses (<strong>B</strong>) were aligned using MAFFT (<a href="http://mafft.cbrc.jp/alignment/software/">http://mafft.cbrc.jp/alignment/software/</a>) within Geneious 10 software package. The alignment was then used to generate a maximum-likelihood phylogenetic tree using PhyML (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). The tree was rooted with MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete genome nucleotide sequences used: CCoV-23/03 (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP849472.1&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom">KP849472.1</a>) , CCoV-1-71 (JQ404409.1), CCoV-A76 (JN856008.2), TGEV-Purdue (AJ271965.2), FCoV-TN406 (EU186072.1), FCoV-RM (FJ938051.1), FCoV-WSU-79-1146 (NC_002306.3), FCoV-WSU-79-1683 (JN634064.1), FRCoV-NL-2010 (NC_030292.1), MinkCoV-WD1127 (HM245925.1), MinkCoV-WD1133 (HM245926.1), PEDV-CV777 (AF353511.1), BatCoV-HKU10 (JQ989270.1), HCoV-229E (KU291448.1), HCoV-NL63 (AY567487.2), MHV-A59 (AY700211.1). Accession numbers for complete ORF1ab protein sequences used: CCoV-23/03 (AKZ66481.1), CCoV-1-71 (AFG19735.1), CCoV-A76 (AEQ61967.2), TGEV-Purdue (P0C6Y5.1), FCoV-TN406 (ABX60144.1), FCoV-RM (ACT10853.1), FCoV-WSU-79-1146 (YP_004070193.2), FCoV-WSU-79-1683 (AFH58022.1), FRCoV-NL-2010 (YP_009256195.1), MinkCoV-WD1127 (ADI80512.1), MinkCoV-WD1133 (ADI80522.1), PEDV-CV777 (P0C6Y4.1), BatCoV-HKU10 (AFU92103.1), HCoV-229E (AOG74782.1), HCoV-NL63 (AAS58176.2), MHV-A59 (AAU06353.1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Analysis based on the ORF1ab polyprotein sequence, reveals very similar phylogenetic relationships within the <em>Alphacoronavirus 1</em> species, with a branching that partitions FCoV strains in one subgroup and CCoV and TGEV strains in another (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). In a similar manner than with the complete genome analysis, the ORF1ab polyprotein-based phylogenetic tree fails to group strains according to FCoV and CCoV serotypes.</p><p id="p-11">One of the most distinguishing features found in serotype I FCoV and CCoV is a S protein cleavage site containing a polybasic furin recognition motif at the junction between the S1 receptor-binding domain and the S2 fusion domain (S1/S2, <a id="xref-fig-2-1" class="xref-fig" href="#F2">fig. 2A</a>). While a furin site at S1/S2 is quite common in betacoronaviruses (e.g. MHV, HCoV-HKU1, and MERS-CoV) and gammacoronaviruses (e.g. IBV), it is not typically found in alphacoronaviruses. This observation highlighted the distinctive nature of serotype I S and prompted us to investigate more closely the phylogenetic relationships of <em>Alphacoronavirus 1</em> members by characterizing their S protein cleavage sites and performing S-protein-based phylogenetic analysis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">fig. 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of Alphacoronavirus S1/S2 and S2&#x2019; cleavage sites and phylogenetic tree based on S protein sequence. The 50 amino acids regions around the S1/S2 (A) and S2&#x2019; (B) S protein sites were aligned with MAFFT using Geneious 10 software package. An asterisk (*) indicates a position which has a single, fully conserved residue. (C) Protein sequences of the complete S protein were aligned using MAFFT within Geneious 10 software package and a maximum likelihood phylogenetic tree of the complete S protein was generated with PhyML. The tree was rooted using MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete spike (S) protein sequences used: CCoV-23/03(AAP72150.1), CCoV-1-71 (AAV65515.1), CCoV-A76 (AEQ61968.1), TGEV-Purdue (ABG89335.1), FCoV-TN406 (BAC05493.1), FCoV-RM (ACT10854.1), FCoV-WSU-79-1146 (YP_004070194.1), FCoV-WSU-79-1683(AFH58021.1), FRCoV-NL-2010(AKG92640.1), MinkCoV-WD1127(ADI80513.1), MinkCoV-WD1133 (ADI80523.1), PEDV-CV777 (AAK38656.1), BatCoV-HKU10 (AFU92104.1), HCoV-229E (BAL45637.1), HCoV-NL63 (AAS58177.1), MHV-A59 (AAA46455.1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2098560117" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of &lt;em&gt;Alphacoronavirus&lt;/em&gt; S1/S2 and S2&#x2019; cleavage sites and phylogenetic tree based on S protein sequence.&lt;/span&gt; The 50 amino acids regions around the S1/S2 (&lt;strong&gt;A&lt;/strong&gt;) and S2&#x2019; (&lt;strong&gt;B&lt;/strong&gt;) S protein sites were aligned with MAFFT using Geneious 10 software package. An asterisk (*) indicates a position which has a single, fully conserved residue. (&lt;strong&gt;C&lt;/strong&gt;) Protein sequences of the complete S protein were aligned using MAFFT within Geneious 10 software package and a maximum likelihood phylogenetic tree of the complete S protein was generated with PhyML. The tree was rooted using MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete spike (S) protein sequences used: CCoV-23/03(AAP72150.1), CCoV-1-71 (AAV65515.1), CCoV-A76 (AEQ61968.1), TGEV-Purdue (ABG89335.1), FCoV-TN406 (BAC05493.1), FCoV-RM (ACT10854.1), FCoV-WSU-79-1146 (YP_004070194.1), FCoV-WSU-79-1683(AFH58021.1), FRCoV-NL-2010(AKG92640.1), MinkCoV-WD1127(ADI80513.1), MinkCoV-WD1133 (ADI80523.1), PEDV-CV777 (AAK38656.1), BatCoV-HKU10 (AFU92104.1), HCoV-229E (BAL45637.1), HCoV-NL63 (AAS58177.1), MHV-A59 (AAA46455.1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F2.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F2.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/19/101774/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Analysis of <em>Alphacoronavirus</em> S1/S2 and S2’ cleavage sites and phylogenetic tree based on S protein sequence.</span><p id="p-12" class="first-child">The 50 amino acids regions around the S1/S2 (<strong>A</strong>) and S2’ (<strong>B</strong>) S protein sites were aligned with MAFFT using Geneious 10 software package. An asterisk (*) indicates a position which has a single, fully conserved residue. (<strong>C</strong>) Protein sequences of the complete S protein were aligned using MAFFT within Geneious 10 software package and a maximum likelihood phylogenetic tree of the complete S protein was generated with PhyML. The tree was rooted using MHV-A59. Numbers at nodes indicate the bootstrap support on 100 replicates. Scale bar indicates the estimated number of substitutions per site. Accession numbers for complete spike (S) protein sequences used: CCoV-23/03(AAP72150.1), CCoV-1-71 (AAV65515.1), CCoV-A76 (AEQ61968.1), TGEV-Purdue (ABG89335.1), FCoV-TN406 (BAC05493.1), FCoV-RM (ACT10854.1), FCoV-WSU-79-1146 (YP_004070194.1), FCoV-WSU-79-1683(AFH58021.1), FRCoV-NL-2010(AKG92640.1), MinkCoV-WD1127(ADI80513.1), MinkCoV-WD1133 (ADI80523.1), PEDV-CV777 (AAK38656.1), BatCoV-HKU10 (AFU92104.1), HCoV-229E (BAL45637.1), HCoV-NL63 (AAS58177.1), MHV-A59 (AAA46455.1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Alignment of protein sequences of various representative alphacoronavirus S1/S2 S cleavage sites reveals a clear demarcation between serotype I and II sequences (<a id="xref-fig-2-3" class="xref-fig" href="#F2">fig. 2A</a>). Serotype I CCoV and FCoV all contain a 16-19 amino acid insert, with a stretch of basic arginine (R) and lysine (K) residues (R/K-R-X-R-R), flanked by fairly well conserved N‐ and C-terminal regions. This insert is absent in all serotype II FCoV, CCoV, and TGEV sequences. It is interesting to note that neither ferret, mink, or other alphacoronaviruses contain such insert. This analysis highlights the distinctive feature that appears to be uniquely found in serotype I <em>Alphacoronavirus 1</em> strains. The lack of an S1/S2 furin site in CCoV-A76 S protein sequence and its good alignment with serotype II sequences is in agreement with the fact that only the NTD of the CCoV-A76 S protein is serotype I-like with the rest of the protein being serotype II-like, including the region around the S1/S2 site.</p><p id="p-14">Alignment of S protein sequences in the region which includes the S2’ cleavage site and the fusion peptide reveals a very similar partitioning between serotype I and II FCoV and CCoV strains (<a id="xref-fig-2-4" class="xref-fig" href="#F2">fig. 2B</a>). In this alignment, the S2’ site demarcates a striking change in amino acid conservation, with only one identical site found upstream of the S2’ cleavage site whereas the region immediately downstream of the cleavage site had 16 identical sites and corresponds to the coronavirus fusion peptide. This analysis also shows the presence of insertions/deletions in the region upstream of the S2’ site. Interestingly, a conserved sequence pattern is observed in serotype II FCoV and CCoV sequences along with TGEV, with the presence of a stretch of basic residues (arginine/lysine) interrupted by a single bulky hydrophobic residue (phenylalanine/tyrosine) upstream of the S2’ site. A less conserved site is found for serotype I FCoV and CCoV, however all are less basic in nature and lack the K-R-K motif observed in all serotype II sequences. In a similar fashion to the alignment at the S1/S2 site, CCoV-A76 S2’ site aligned best with serotype II sequences.</p><p id="p-15">Finally, we performed a phylogenetic analysis based on full-length S protein alignment (<a id="xref-fig-2-5" class="xref-fig" href="#F2">fig. 2C</a>). This analysis reveals a different partitioning of <em>Alphacoronavirus 1</em> strains compared to our previous analyses. Serotype I FCoV and CCoV clustered in one group and serotype II FCoV, CCoV, and TGEV grouped in another. Furthermore, the CCoV-A76 strain is found at an intermediate position between the two serotype groupings (20), a result that reflects the recombinant nature of its S protein. This phylogenetic analysis clearly demarcates strains according to the previously characterized serotypes and not according to which host species the strains infect, as was observed when performing the analysis with complete genome or ORF1ab sequences. Using such S-based phylogenetic analyses, we propose that the <em>Alphacoronavirus 1</em> species be sub-classified as clade A, corresponding to serotype I FCoV and CCoV, and clade B, corresponding to serotype II FCoV and CCoV and TGEV-like viruses.</p></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-16">Current classification within the <em>Alphacoronavirus 1</em> species is not well defined and often fails to recognize the profound differences observed between well-established <em>Alphacoronavirus 1</em> serotypes. Adding to the confusion are the different terms used to designate various <em>Alphacoronavirus 1</em> strains: FCoV serotypes and types; CCoV serotypes, types, and genotypes; and TGEV which is not classified according to FCoV/CCoV serotypes. We propose a more unified classification, based on the important differences between the S proteins of serotype I and II viruses. Our analysis reveals two well-defined clades, clade A and clade B, corresponding to the serotype groupings. Both clades contain FCoV and CCoV strains, while TGEV only belongs to clade B, in agreement with the finding that TGEV is most related to clade B (serotype II) CCoV. We recommend the inclusion of representatives of both clades when performing phylogenetic analysis of alphacoronaviruses. The proposed classification scheme for the alphacoronaviruses is similar to the one used to characterize lineages and clades of avian influenza viruses. Indeed, instead of performing phylogenetic analyses on the entire genomes, avian influenza classifications are based on the surface proteins genes, e.g. hemagglutinin (HA) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>).</p><p id="p-17">In addition to better matching serological and phylogenetic groupings, S-based phylogenies offer other advantages. Because the S gene is frequently shuffled by recombination events, such classifications allows the grouping of viruses that have a shared S gene. In our analyses, using complete genome phylogenies, strains were clustered according to the host they infected whereas the S-based phylogeny allowed to group FCoV and CCoV strains together in different clades. Our approach allows for a better understanding of the complex phylogenetic relationships and evolutionary history observed in coronaviruses. Furthermore, phylogenetic analysis on S proteins can reveal relationships that are not observed using ORF1ab or complete genome-based analysis. In particular, in a study characterizing novel deltacoronaviruses, Woo and colleagues showed that the S proteins of alphacoronaviruses are more closely related to deltacoronaviruses than to other coronavirus genera (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>). During our phylogenetic analyses, we took note of a distinct <em>Alphacoronavirus</em> S protein sequence from the Asian leopard cat coronavirus (accession no. ABQ39958.1) and noticed its much smaller size (1035 amino acids) compared to other alphacoronaviruses (1466 and 1356 amino acids for FCoV-RM and HCoV-NL63 respectively). Phylogenetically, we observed that it was highly divergent and did not cluster with the other alphacoronaviruses analyzed (data not shown). As only an incomplete genomic sequence was available, it was not included in our analyses.</p><p id="p-18">Our alignment analysis of the cleavage sites of alphacoronaviruses revealed relatively well-conserved functional regions within members of the same clade, but with clear divergence when comparing sequences from two different clades. For example, the presence of a furin motif consisting of the basic stretch of residues R/K-R-X-R-R found only in the S1/S2 site of clade A viruses along with the K-R-K motif found only in the S2’ site of clade B viruses, could be used for rapid determination of clade inclusion in samples which are difficult to sequence at a whole genome level.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-19">We thank members of the Whittaker lab for helpful discussions. This work was funded by a research grant from the Morris Animal Foundation (D10FE-511). Work in the authors’ lab is also funded by the Winn Feline Health Foundation (W15-026) and the Cornell Feline Health Center.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-101774v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowley</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faaberg</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-chapter-title">Order - Nidovirales</span>, p <span class="cit-fpage">784</span>–<span class="cit-lpage">794</span>, In <span class="cit-auth"><span class="cit-name-surname">king</span>  <span class="cit-name-given-names">AMQ</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Am</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ce</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Le</span></span> (ed), <span class="cit-source">Virus Taxonomy</span> doi:<a href="http://dxdoi.org/10.1016/B978-0-12-384684-6.00066-5">http://dxdoi.org/10.1016/B978-0-12-384684-6.00066-5</a> <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.2" data-doi="10.1007/s00705-012-1295-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zirkel</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">157</span>:<span class="cit-fpage">1623</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DLauber%26rft.auinit1%253DC.%26rft.volume%253D157%26rft.issue%253D8%26rft.spage%253D1623%26rft.epage%253D1628%26rft.atitle%253DMesoniviridae%253A%2Ba%2Bproposed%2Bnew%2Bfamily%2Bin%2Bthe%2Border%2BNidovirales%2Bformed%2Bby%2Ba%2Bsingle%2Bspecies%2Bof%2Bmosquito-borne%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-012-1295-x%26rft_id%253Dinfo%253Apmid%252F22527862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-012-1295-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22527862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.3" data-doi="10.1128/JVI.06540-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">CSF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">CCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">AKL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">JHN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">JLL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">CCC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">B-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">K-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K-Y.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Discovery of Seven Novel Mammalian and Avian Coronaviruses in the Genus Deltacoronavirus Supports Bat Coronaviruses as the Gene Source of Alphacoronavirus and Betacoronavirus and Avian Coronaviruses as the Gene Source of Gammacoronavirus and Deltacoronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">86</span>:<span class="cit-fpage">3995</span>–<span class="cit-lpage">4008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06540-11%26rft_id%253Dinfo%253Apmid%252F22278237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzcvMzk5NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAxLzE5LzEwMTc3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.4" data-doi="10.1093/ve/vev023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2016</span>.<span class="cit-article-title">MERS-CoV recombination: implications about the reservoir and potential for adaptation</span>. <abbr class="cit-jnl-abbrev">Virus Evolution</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">vev023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvev023%26rft_id%253Dinfo%253Apmid%252F27774293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vev023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.5" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">84</span>:<span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAxLzE5LzEwMTc3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.6" data-doi="10.1007/s00705-009-0547-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">EB</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Ratification vote on taxonomic proposals to the International Committee on Taxonomy of Viruses (2009)</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">155</span>:<span class="cit-fpage">133</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DCarstens%26rft.auinit1%253DE.%2BB.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D133%26rft.epage%253D146%26rft.atitle%253DRatification%2Bvote%2Bon%2Btaxonomic%2Bproposals%2Bto%2Bthe%2BInternational%2BCommittee%2Bon%2BTaxonomy%2Bof%2BViruses%2B%25282009%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-009-0547-x%26rft_id%253Dinfo%253Apmid%252F19960211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-009-0547-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19960211&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273588000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-101774v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haijema</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RP</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-chapter-title">Feline Coronaviruses: A Tale of Two-Faced Types</span>, p <span class="cit-fpage">183</span>–<span class="cit-lpage">204</span>. In <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span> (ed), <span class="cit-source">Coronaviruses Molecular and Cellular Biology</span>. <span class="cit-publ-name">Caister Academic Press</span>, Norfolk, <span class="cit-publ-loc">United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.8" data-doi="10.1006/viro.1998.9045"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vennema</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">NC</span></span>. <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Feline Infectious Peritonitis Viruses Arise by Mutation from Endemic Feline Enteric Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">243</span>:<span class="cit-fpage">150</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DVennema%26rft.auinit1%253DH.%26rft.volume%253D243%26rft.issue%253D1%26rft.spage%253D150%26rft.epage%253D157%26rft.atitle%253DFeline%2Binfectious%2Bperitonitis%2Bviruses%2Barise%2Bby%2Bmutation%2Bfrom%2Bendemic%2Bfeline%2Benteric%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1998.9045%26rft_id%253Dinfo%253Apmid%252F9527924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1998.9045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9527924&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072920500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.9" data-doi="10.1354/vp.45-2-236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramsell</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morera</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juan-Salles</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jimenez</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ardiaca</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montesinos</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teifke</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohr</span>  <span class="cit-name-given-names">CV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evermann</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baszler</span>  <span class="cit-name-given-names">TV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordhausen</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maes</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiupel</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Clinicopathologic features of a systemic coronavirus-associated disease resembling feline infectious peritonitis in the domestic ferret (Mustela putorius)</span>. <abbr class="cit-jnl-abbrev">Vet Pathol</abbr> <span class="cit-vol">45</span>:<span class="cit-fpage">236</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2BPathology%26rft.stitle%253DVeterinary%2BPathology%26rft.aulast%253DGarner%26rft.auinit1%253DM.%2BM.%26rft.volume%253D45%26rft.issue%253D2%26rft.spage%253D236%26rft.epage%253D246%26rft.atitle%253DClinicopathologic%2BFeatures%2Bof%2Ba%2BSystemic%2BCoronavirus-Associated%2BDisease%2BResembling%2BFeline%2BInfectious%2BPeritonitis%2Bin%2Bthe%2BDomestic%2BFerret%2B%2528Mustela%2Bputorius%2529%26rft_id%253Dinfo%253Adoi%252F10.1354%252Fvp.45-2-236%26rft_id%253Dinfo%253Apmid%252F18424841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1354/vp.45-2-236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18424841&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254766100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.10" data-doi="10.3390/v6083363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Licitra</span>  <span class="cit-name-given-names">BN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duhamel</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Canine enteric coronaviruses: emerging viral pathogens with distinct recombinant spike proteins</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">3363</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D6%26rft.spage%253D3363%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv6083363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v6083363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.11" data-doi="10.3390/v4061011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belouzard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Licitra</span>  <span class="cit-name-given-names">BN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">1011</span>–<span class="cit-lpage">1033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DBelouzard%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1011%26rft.epage%253D1033%26rft.atitle%253DMechanisms%2Bof%2Bcoronavirus%2Bcell%2Bentry%2Bmediated%2Bby%2Bthe%2Bviral%2Bspike%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4061011%26rft_id%253Dinfo%253Apmid%252F22816037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4061011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22816037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305801900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.12" data-doi="10.1016/j.virusres.2014.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virus Research</abbr> <span class="cit-vol">202</span>:<span class="cit-fpage">120</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft.volume%253D202%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.021%26rft_id%253Dinfo%253Apmid%252F25445340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25445340&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-101774v1.13" data-doi="10.1007/978-1-4615-9373-7_36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evermann</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKeirnan</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">RL</span></span>. <span class="cit-pub-date">1984</span>. <span class="cit-chapter-title">Pathogenic Differences Between Various Feline Coronavirus Isolates</span>, p <span class="cit-fpage">365</span>–<span class="cit-lpage">380</span>. In <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Zeijst</span>  <span class="cit-name-given-names">BAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span> (ed), <span class="cit-source">Molecular Biology and Pathogenesis of Coronaviruses</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-4615-9373-7_36</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> <span class="cit-publ-name">Springer US</span>, <span class="cit-publ-loc">Boston, MA</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DPedersen%26rft.auinit1%253DN.%2BC.%26rft.volume%253D173%26rft.spage%253D365%26rft.epage%253D380%26rft.atitle%253DPathogenic%2Bdifferences%2Bbetween%2Bvarious%2Bfeline%2Bcoronavirus%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4615-9373-7_36%26rft_id%253Dinfo%253Apmid%252F6331125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4615-9373-7_36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6331125&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-101774v1.14" data-doi="10.1007/BF01310494"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hohdatsu</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koyama</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Characterization of monoclonal antibodies against feline infectious peritonitis virus type II and antigenic relationship between feline, porcine, and canine coronaviruses</span>. <abbr class="cit-jnl-abbrev">Archives of Virology</abbr> <span class="cit-vol">117</span>:<span class="cit-fpage">85</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DHohdatsu%26rft.auinit1%253DT.%26rft.volume%253D117%26rft.issue%253D1-2%26rft.spage%253D85%26rft.epage%253D95%26rft.atitle%253DCharacterization%2Bof%2Bmonoclonal%2Bantibodies%2Bagainst%2Bfeline%2Binfectious%2Bperitonitis%2Bvirus%2Btype%2BII%2Band%2Bantigenic%2Brelationship%2Bbetween%2Bfeline%252C%2Bporcine%252C%2Band%2Bcanine%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01310494%26rft_id%253Dinfo%253Apmid%252F1706593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01310494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1706593&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FD14600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.15" data-doi="10.1016/j.vetmic.2008.06.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decaro</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buonavoglia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">An update on canine coronaviruses: viral evolution and pathobiology</span>. <abbr class="cit-jnl-abbrev">Vet Microbiol</abbr> <span class="cit-vol">132</span>:<span class="cit-fpage">221</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DDecaro%26rft.auinit1%253DN.%26rft.volume%253D132%26rft.issue%253D3-4%26rft.spage%253D221%26rft.epage%253D234%26rft.atitle%253DAn%2Bupdate%2Bon%2Bcanine%2Bcoronaviruses%253A%2Bviral%2Bevolution%2Band%2Bpathobiology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2008.06.007%26rft_id%253Dinfo%253Apmid%252F18635322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2008.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18635322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260973900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.16" data-doi="10.1007/s007050050336"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hohdatsu</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Izumiya</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yokoyama</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kida</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koyama</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Differences in virus receptor for type I and type II feline infectious peritonitis virus</span>. <abbr class="cit-jnl-abbrev">Archives of Virology</abbr> <span class="cit-vol">143</span>:<span class="cit-fpage">839</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DHohdatsu%26rft.auinit1%253DT.%26rft.volume%253D143%26rft.issue%253D5%26rft.spage%253D839%26rft.epage%253D850%26rft.atitle%253DDifferences%2Bin%2Bvirus%2Breceptor%2Bfor%2Btype%2BI%2Band%2Btype%2BII%2Bfeline%2Binfectious%2Bperitonitis%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs007050050336%26rft_id%253Dinfo%253Apmid%252F9645192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s007050050336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9645192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.17" data-doi="10.1099/vir.0.82666-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dye</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temperton</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Type I feline coronavirus spike glycoprotein fails to recognize aminopeptidase N as a functional receptor on feline cell lines</span>. <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">1753</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DDye%26rft.auinit1%253DC.%26rft.volume%253D88%26rft.issue%253D6%26rft.spage%253D1753%26rft.epage%253D1760%26rft.atitle%253DType%2BI%2Bfeline%2Bcoronavirus%2Bspike%2Bglycoprotein%2Bfails%2Bto%2Brecognize%2Baminopeptidase%2BN%2Bas%2Ba%2Bfunctional%2Breceptor%2Bon%2Bfeline%2Bcell%2Blines%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.82666-0%26rft_id%253Dinfo%253Apmid%252F17485536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.82666-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247087900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.18" data-doi="10.1016/j.virusres.2006.12.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decaro</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martella</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elia</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campolo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desario</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cirone</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tempesta</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buonavoglia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Molecular characterisation of the virulent canine coronavirus CB/05 strain</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">125</span>:<span class="cit-fpage">54</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DDecaro%26rft.auinit1%253DN.%26rft.volume%253D125%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D60%26rft.atitle%253DMolecular%2Bcharacterisation%2Bof%2Bthe%2Bvirulent%2Bcanine%2Bcoronavirus%2BCB%252F05%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.12.006%26rft_id%253Dinfo%253Apmid%252F17275120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17275120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245873500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herrewegh</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeenk</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>. <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Feline coronavirus type II strains 79-1683 and 79-1146 originate from a double recombination between feline coronavirus type I and canine coronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">72</span>:<span class="cit-fpage">4508</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHerrewegh%26rft.auinit1%253DA.%2BA.%25A0P.%25A0M.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D4508%26rft.epage%253D4514%26rft.atitle%253DFeline%2BCoronavirus%2BType%2BII%2BStrains%2B79-1683%25A0and%2B79-1146%25A0Originate%2Bfrom%2Ba%2BDouble%2BRecombination%2Bbetween%2BFeline%2BCoronavirus%2BType%2BI%2Band%2BCanine%2BCoronavirus%26rft_id%253Dinfo%253Apmid%252F9557750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcyLzUvNDUwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAxLzE5LzEwMTc3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.20" data-doi="10.1016/j.virol.2012.04.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Regan</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chillag</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">VD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Licitra</span>  <span class="cit-name-given-names">BN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dubovi</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Town</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Characterization of a recombinant canine coronavirus with a distinct receptor-binding (S1) domain</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">430</span>:<span class="cit-fpage">90</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D430%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2012.04.013%26rft_id%253Dinfo%253Apmid%252F22609354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2012.04.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22609354&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.21" data-doi="10.3201/eid1907.121094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Licitra</span>  <span class="cit-name-given-names">BN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regan</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">VD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duhamel</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Mutation in spike protein cleavage site and pathogenesis of feline coronavirus</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">1066</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D19%26rft.spage%253D1066%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1907.121094%26rft_id%253Dinfo%253Apmid%252F23763835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1907.121094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23763835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.22" data-doi="10.1128/JVI.00074-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haijema</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schellen</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schreur</span>  <span class="cit-name-given-names">PW</span></span>, <span class="cit-auth"><span class="cit-name-surname">te Lintelo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vennema</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Cleavage of Group 1 Coronavirus Spike Proteins: How Furin Cleavage Is Traded Off against Heparan Sulfate Binding upon Cell Culture Adaptation</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">6078</span>–<span class="cit-lpage">6083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00074-08%26rft_id%253Dinfo%253Apmid%252F18400867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xMi82MDc4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDEvMTkvMTAxNzc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.23" data-doi="10.1099/jgv.0.000520."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lamers</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hundie</span>  <span class="cit-name-given-names">GB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Provacia</span>  <span class="cit-name-given-names">LB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koopmans</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">VS</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Naturally occurring recombination in ferret coronaviruses revealed by complete genome characterization</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000520.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000520.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000520.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.24"><div class="cit-metadata"><cite><span class="cit-article-title">World Health Organization/World Organisation for Animal HF, Agriculture Organization HNEWG</span>. <span class="cit-pub-date">2014</span>. <abbr class="cit-jnl-abbrev">Revised and updated nomenclature for highly pathogenic avian influenza A (H5N1) viruses. Influenza Other Respir Viruses</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">384</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRevised%2Band%2Bupdated%2Bnomenclature%2Bfor%2Bhighly%2Bpathogenic%2Bavian%2Binfluenza%2BA%2B%2528H5N1%2529%2Bviruses.%2BInfluenza%2BOther%2BRespir%2BViruses%26rft.volume%253D8%26rft.spage%253D384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-101774v1.25" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">59</span>:<span class="cit-fpage">307</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F01%2F19%2F101774.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/101774v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F101774v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/101766v1" title="Novel insights into the molecular heterogeneity of hepatocellular carcinoma" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/101725v1" title="Modern-Day SIV Viral Diversity Generated by Extensive Recombination and Cross-Species Transmission" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 19, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/101774v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/101774v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/101774v1" class="active" data-icon-position="" data-hide-link-title="0">Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/28926" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6BTY7cEOuncbefAmXcXrHu-kGpW28fFwKySw_Mu-2L4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="28926" id="node28926--2" data-pisa="biorxiv;101774v1" data-pisa-master="biorxiv;101774" data-seqnum="28926" data-apath="/biorxiv/early/2017/01/19/101774.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spike-based phylogenetically defined clades within the <em>Alphacoronavirus 1</em> species</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gary R.</span> <span class="nlm-surname">Whittaker</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nicole M.</span> <span class="nlm-surname">André</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jean Kaoru</span> <span class="nlm-surname">Millet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">101774; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/101774 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/101774v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;title=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;title=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;text=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;title=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;t=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;title=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;title=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="28926" id="node28926--3" data-pisa="biorxiv;101774v1" data-pisa-master="biorxiv;101774" data-seqnum="28926" data-apath="/biorxiv/early/2017/01/19/101774.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spike-based phylogenetically defined clades within the <em>Alphacoronavirus 1</em> species</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gary R.</span> <span class="nlm-surname">Whittaker</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nicole M.</span> <span class="nlm-surname">André</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jean Kaoru</span> <span class="nlm-surname">Millet</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">101774; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/101774 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Whittaker%2C+Gary+R.&amp;rft.au=Andr%C3%A9%2C+Nicole+M.&amp;rft.au=Millet%2C+Jean+Kaoru"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/28926/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/28926/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/28926/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/28926/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/28926/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/28926/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/28926/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/28926/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/28926/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/28926/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/28926/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/28926/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;count=horizontal&amp;via=&amp;text=Spike-based%20phylogenetically%20defined%20clades%20within%20the%20Alphacoronavirus%201%20species&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/101774v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/101774v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/101774v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
