<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/embed/graphic-4.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/09/25/738526.atom" />
<meta name="HW.pisa" content="biorxiv;738526v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes" />
<meta name="DC.Identifier" content="10.1101/738526" />
<meta name="DC.Date" content="2019-09-25" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.

Supplementary information is available at [rna.uni-jena.de/supplements/bats][1], the Open Science Framework ([doi.org/10.17605/OSF.IO/4CMDN][2]), and GitHub ([github.com/rnajena/bats_ncrna][3]).

 [1]: http://rna.uni-jena.de/supplements/bats
 [2]: http://doi.org/10.17605/OSF.IO/4CMDN
 [3]: http://github.com/rnajena/bats_ncrna" />
<meta name="DC.Contributor" content="Nelly Mostajo Berrospi" />
<meta name="DC.Contributor" content="Marie Lataretu" />
<meta name="DC.Contributor" content="Sebastian Krautwurst" />
<meta name="DC.Contributor" content="Florian Mock" />
<meta name="DC.Contributor" content="Daniel Desirò" />
<meta name="DC.Contributor" content="Kevin Lamkiewicz" />
<meta name="DC.Contributor" content="Maximilian Collatz" />
<meta name="DC.Contributor" content="Andreas Schoen" />
<meta name="DC.Contributor" content="Friedemann Weber" />
<meta name="DC.Contributor" content="Manja Marz" />
<meta name="DC.Contributor" content="Martin Hölzer" />
<meta name="article:published_time" content="2019-09-25" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.&lt;/p&gt;&lt;p&gt;&lt;b&gt;Supplementary information&lt;/b&gt; is available at rna.uni-jena.de/supplements/bats, the Open Science Framework (doi.org/10.17605/OSF.IO/4CMDN), and GitHub (github.com/rnajena/bats_ncrna).&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;738526v3" />
<meta name="citation_id" content="738526v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/738526v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/738526v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/738526v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526.full.pdf" />
<meta name="citation_doi" content="10.1101/738526" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="738526" />
<meta name="citation_author" content="Nelly Mostajo Berrospi" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="Institute of Virology, Philipps-University Marburg" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author" content="Marie Lataretu" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3637-5870" />
<meta name="citation_author" content="Sebastian Krautwurst" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9413-5701" />
<meta name="citation_author" content="Florian Mock" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1791-4437" />
<meta name="citation_author" content="Daniel Desirò" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3928-9094" />
<meta name="citation_author" content="Kevin Lamkiewicz" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6375-6441" />
<meta name="citation_author" content="Maximilian Collatz" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author" content="Andreas Schoen" />
<meta name="citation_author_institution" content="Institute for Virology, FB10-Veterinary Medicine, Justus-Liebig University" />
<meta name="citation_author_institution" content="German Center for Infection Research (DZIF)" />
<meta name="citation_author" content="Friedemann Weber" />
<meta name="citation_author_institution" content="Institute of Virology, Philipps-University Marburg" />
<meta name="citation_author_institution" content="Institute for Virology, FB10-Veterinary Medicine, Justus-Liebig University" />
<meta name="citation_author_institution" content="German Center for Infection Research (DZIF)" />
<meta name="citation_author" content="Manja Marz" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="FLI Leibniz Institute for Age Research" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4783-8823" />
<meta name="citation_author" content="Martin Hölzer" />
<meta name="citation_author_institution" content="RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center, Friedrich Schiller University Jena" />
<meta name="citation_author_email" content="martin.hoelzer@uni-jena.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7090-8717" />
<meta name="citation_reference" content="C. H. Calisher, J. E. Childs, H. E. Field, K. V. Holmes, and T. Schountz. Bats: important reservoir hosts of emerging viruses. Clin Microbiol Rev, 19(3):531–545, 2006." />
<meta name="citation_reference" content="E. C. Teeling, S. C. Vernes, L. M. Dávalos, D. A. Ray, M. T. P. Gilbert, E. Myers, and B. Consortium. Bat biology, genomes, and the Bat1K Project: To generate Chromosome-Level genomes for all living bat species. Annu Rev Anim Biosci, (6):23–46, 2018." />
<meta name="citation_reference" content="N. B. Simmons. An Eocene big bang for bats. Science, 307(5709):527–528, 2005." />
<meta name="citation_reference" content="E. C. Teeling, G. Jones, and S. J. Rossiter. Phylogeny, genes, and hearing: implications for the evolution of echolocation in bats. In Bat Bioacoustics, pages 25–54. Springer, 2016." />
<meta name="citation_reference" content="A. Gardner, D. Wilson, and D. Reeder. Mammal Species of the World. A Taxonomic and Geographic Reference. Mammal species of the world: a taxonomic and geographic reference, 12, 2005." />
<meta name="citation_reference" content="E. C. Teeling, M. S. Springer, O. Madsen, P. Bates, S. J. O’brien, and W. J. Murphy. A molecular phylogeny for bats illuminates biogeography and the fossil record. Science, 307(5709):580–584, 2005." />
<meta name="citation_reference" content="Y. Prat, L. Azoulay, R. Dor, and Y. Yovel. Crowd vocal learning induces vocal dialects in bats: Playback of conspecifics shapes fundamental frequency usage by pups. PLoS Biol, 15(10):e2002556, 2017." />
<meta name="citation_reference" content="Z. Huang, D. Jebb, and E. C. Teeling. Blood miRNomes and transcriptomes reveal novel longevity mechanisms in the long-lived bat, Myotis myotis. BMC Genom, 17(1):906, 2016." />
<meta name="citation_reference" content="N. M. Foley, G. M. Hughes, Z. Huang, M. Clarke, D. Jebb, C. V. Whelan, E. J. Petit, F. Touzalin, O. Farcy, G. Jones, et al. Growing old, yet staying young: The role of telomeres in bats’ exceptional longevity. Sci Adv, 4(2):eaao0926, 2018." />
<meta name="citation_reference" content="Z. Huang, C. V. Whelan, N. M. Foley, D. Jebb, F. Touzalin, E. J. Petit, S. J. Puechmaille, and E. C. Teeling. Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats. Nat Ecol Evol, 2019." />
<meta name="citation_reference" content="I. Smith and L.-F. Wang. Bats and their virome: an important source of emerging viruses capable of infecting humans. Curr Opin Virol, 3(1):84–91, 2013." />
<meta name="citation_reference" content="L.-F. Wang, P. J. Walker, and L. L. Poon. Mass extinctions, biodiversity and mitochondrial function: are bats “special” as reservoirs for emerging viruses? Curr Opin Virol, 1(6):649–657, 2011." />
<meta name="citation_reference" content="C. E. Brook and A. P. Dobson. Bats as “special” reservoirs for emerging zoonotic pathogens. Trends Microbiol, 23(3):172–180, 2015." />
<meta name="citation_reference" content="S. J. Anthony, C. K. Johnson, D. J. Greig, S. Kramer, X. Che, H. Wells, A. L. Hicks, D. O. Joly, N. D. Wolfe, P. Daszak, et al. Global patterns in coronavirus diversity. Virus Evol, 3(1), 2017." />
<meta name="citation_reference" content="T. J. O’shea, P. M. Cryan, A. A. Cunningham, A. R. Fooks, D. T. Hayman, A. D. Luis, A. J. Peel, R. K. Plowright, and J. L. Wood. Bat flight and zoonotic viruses. Emerg Infect Dis, 20(5):741, 2014." />
<meta name="citation_reference" content="M. Hölzer, V. Krähling, F. Amman, E. Barth, S. H. Bernhart, V. A. O. Carmelo, M. Collatz, G. Doose, F. Eggenhofer, J. Ewald, J. Fallmann, L. M. Feldhahn, M. Fricke, J. Gebauer, A. J. Gruber, F. Hufsky, H. Indrischek, S. Kanton, J. Linde, N. Mostajo, R. Ochsenreiter, K. Riege, L. Rivarola-Duarte, A. H. Sahyoun, S. J. Saunders, S. E. Seemann, A. Tanzer, B. Vogel, S. Wehner, M. T. Wolfinger, R. Backofen, J. Gorodkin, I. Grosse, I. Hofacker, S. Hoffmann, C. Kaleta, P. F. Stadler, S. Becker, and M. Marz. Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells. Sci Rep, (6):34589, 2016." />
<meta name="citation_reference" content="A. T. Papenfuss, M. L. Baker, Z.-P. Feng, M. Tachedjian, G. Crameri, C. Cowled, J. Ng, V. Janardhana, H. E. Field, and L.-F. Wang. The immune gene repertoire of an important viral reservoir, the Australian black flying fox. BMC Genom, 13(1):261, 2012." />
<meta name="citation_reference" content="J. Mattick. Non-coding RNas: the architects of eukaryotic complexity. EMBO Rep, 2(11):986–991, 2001." />
<meta name="citation_reference" content="O. Isakov, R. Ronen, J. Kovarsky, A. Gabay, I. Gan, S. Modai, and N. Shomron. Novel insight into the non-coding repertoire through deep sequencing analysis. Nucleic Acids Res, 40(11):e86–e86, 2012." />
<meta name="citation_reference" content="L. Stein. Genome annotation: from sequence to biology. Nat Rev Genet, 2(7):493, 2001." />
<meta name="citation_reference" content="R. Ekblom and J. Galindo. Applications of next generation sequencing in molecular ecology of non-model organisms. Heredity, 107(1):1, 2011." />
<meta name="citation_reference" content="S. L. Salzberg. Next-generation genome annotation: we still struggle to get it right. Genome Biology, 20(1): 92, May 2019. ISSN 1474-760X." />
<meta name="citation_reference" content="W. A. Haynes, A. Tomczak, and P. Khatri. Gene annotation bias impedes biomedical research. Sci Rep, 8(1):1362, 2018." />
<meta name="citation_reference" content="A. Gurevich, V. Saveliev, N. Vyahhi, and G. Tesler. QUAST: quality assessment tool for genome assemblies. Bioinformatics, 29(8):1072–1075, 2013." />
<meta name="citation_reference" content="K. Lindblad-Toh, M. Garber, O. Zuk, M. F. Lin, B. J. Parker, S. Washietl, P. Kheradpour, J. Ernst, G. Jordan, E. Mauceli, et al. A high-resolution map of human evolutionary constraint using 29 mammals. Nature, 478(7370):476, 2011." />
<meta name="citation_reference" content="G. Zhang, C. Cowled, Z. Shi, Z. Huang, K. A. Bishop-Lilly, X. Fang, J. W. Wynne, Z. Xiong, M. L. Baker, W. Zhao, et al. Comparative analysis of bat genomes provides insight into the evolution of flight and immunity. Science, 339(6118):456–460, 2013." />
<meta name="citation_reference" content="S. S. Pavlovich, S. P. Lovett, G. Koroleva, J. C. Guito, C. E. Arnold, E. R. Nagle, K. Kulcsar, A. Lee, F. Thibaud-Nissen, A. J. Hume, et al. The egyptian rousette genome reveals unexpected features of bat antiviral immunity. Cell, 173(5):1098–1110, 2018." />
<meta name="citation_reference" content="J. Parker, G. Tsagkogeorga, J. A. Cotton, Y. Liu, P. Provero, E. Stupka, and S. J. Rossiter. Genome-wide signatures of convergent evolution in echolocating mammals. Nature, 502(7470):228, 2013." />
<meta name="citation_reference" content="M. Wen, J. H. Ng, F. Zhu, Y. T. Chionh, W. N. Chia, I. H. Mendenhall, B. P.-H. Lee, A. T. Irving, and L.-F. Wang. Exploring the genome and transcriptome of the cave nectar bat Eonycteris spelaea with PacBio long-read sequencing. GigaScience, 7(10):giy116, 2018." />
<meta name="citation_reference" content="D. Dong, M. Lei, P. Hua, Y.-H. Pan, S. Mu, G. Zheng, E. Pang, K. Lin, and S. Zhang. The genomes of two bat species with long constant frequency echolocation calls. Mol Biol Evol, page msw231, 2016." />
<meta name="citation_reference" content="M. L. Z. Mendoza, Z. Xiong, M. Escalera-Zamudio, A. K. Runge, J. Thézé, D. Streicker, H. K. Frank, E. Loza-Rubio, S. Liu, O. A. Ryder, et al. Hologenomic adaptations underlying the evolution of sanguivory in the common vampire bat. Nat Ecol Evol, 2(4):659, 2018." />
<meta name="citation_reference" content="I. Seim, X. Fang, Z. Xiong, A. V. Lobanov, Z. Huang, S. Ma, Y. Feng, A. A. Turanov, Y. Zhu, T. L. Lenz, et al. Genome analysis reveals insights into physiology and longevity of the Brandt’s bat Myotis brandtii. Nat Commun, (4):2212, 2013." />
<meta name="citation_reference" content="W. L. Eckalbar, S. A. Schlebusch, M. K. Mason, Z. Gill, A. V. Parker, B. M. Booker, S. Nishizaki, C. Muswamba-Nday, E. Terhune, K. A. Nevonen, et al. Transcriptomic and epigenomic characterization of the developing bat wing. Nat Genet, 48(5):528, 2016." />
<meta name="citation_reference" content="D. Kim, B. Langmead, and S. L. Salzberg. HISAT: a fast spliced aligner with low memory requirements. Nat Methods, 12(4):357–360, 2015." />
<meta name="citation_reference" content="Y. Liao, G. K. Smyth, and W. Shi. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics, 30(7):923–930, 2014." />
<meta name="citation_reference" content="M. Hölzer, A. Schoen, J. Wulle, M. A. Müller, C. Drosten, M. Marz, and F. Weber. Virus- and interferon alpha-induced transcriptomes of cells from the microbat Myotis daubentonii. iScience, 2019." />
<meta name="citation_reference" content="K. A. Field, J. S. Johnson, T. M. Lilley, S. M. Reeder, E. J. Rogers, M. J. Behr, and D. M. Reeder. The white-nose syndrome transcriptome: activation of anti-fungal host responses in wing tissue of hibernating little brown myotis. PLOS Pathog, 11(10): e1005168, 2015." />
<meta name="citation_reference" content="K. A. Field, B. J. Sewall, J. M. Prokkola, G. G. Turner, M. F. Gagnon, T. M. Lilley, J. Paul White, J. S. Johnson, C. L. Hauer, and D. M. Reeder. Effect of torpor on host transcriptomic responses to a fungal pathogen in hibernating bats. Mol Ecol, 27(18):3727–3743, 2018." />
<meta name="citation_reference" content="A. M. Bolger, M. Lohse, and B. Usadel. Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics, (30):2114–2120, 2014. ISSN 1367-4811." />
<meta name="citation_reference" content="M. Martin. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMB-net.journal, 17(1):10, 2011." />
<meta name="citation_reference" content="R. Schmieder and R. Edwards. Quality control and preprocessing of metagenomic datasets. Bioinformatics, (27):863–864, 2011. ISSN 1367-4811." />
<meta name="citation_reference" content="M. R. Friedländer, S. D. Mackowiak, N. Li, W. Chen, and N. Rajewsky. miRDeep2 accurately identifies known and hundreds of novel microRNA genes in seven animal clades. Nucleic Acids Res, (40):37–52, January 2012. ISSN 1362-4962." />
<meta name="citation_reference" content="M. I. Love, W. Huber, and S. Anders. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol, 15(12):550, 2014." />
<meta name="citation_reference" content="B. Li, V. Ruotti, R. M. Stewart, J. A. Thomson, and C. N. Dewey. RNA-Seq gene expression estimation with read mapping uncertainty. Bioinformatics, (26): 493–500, 2010. ISSN 1367-4811." />
<meta name="citation_reference" content="D. R. Zerbino, P. Achuthan, W. Akanni, M. R. Amode, D. Barrell, J. Bhai, K. Billis, C. Cummins, A. Gall, C. G. Girón, et al. Ensembl 2018. Nucleic Acids Res, 46 (D1):D754–D761, 2017." />
<meta name="citation_reference" content="S. Griffiths-Jones, A. Bateman, M. Marshall, A. Khanna, and S. R. Eddy. Rfam: an RNA family database. Nucleic Acids Res, (31):439–441, January 2003. ISSN 1362-4962." />
<meta name="citation_reference" content="I. Kalvari, J. Argasinska, N. Quinones-Olvera, E. P. Nawrocki, E. Rivas, S. R. Eddy, A. Bateman, R. D. Finn, and A. I. Petrov. Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families. Nucleic Acids Res, 46(D1):D335–D342, 2017." />
<meta name="citation_reference" content="I. Kalvari, E. P. Nawrocki, J. Argasinska, N. Quinones-Olvera, R. D. Finn, A. Bateman, and A. I. Petrov. Non-coding RNA analysis using the Rfam database. Curr Protoc Bioinformatics, page e51, 2018." />
<meta name="citation_reference" content="E. P. Nawrocki, D. L. Kolbe, and S. R. Eddy. Infernal 1.0: inference of RNA alignments. Bioinformatics, (25): 1335–1337, 2009." />
<meta name="citation_reference" content="E. P. Nawrocki and S. R. Eddy. Infernal 1.1: 100-fold faster RNA homology searches. Bioinformatics, (29): 2933–2935, November 2013. ISSN 1367-4811." />
<meta name="citation_reference" content="S. Griffiths-Jones. RALEE—RNA alignment editor in Emacs. Bioinformatics, (21):257–259, 2005." />
<meta name="citation_reference" content="K. Lagesen, P. Hallin, E. A. Rødland, H.-H. Staerfeldt, T. Rognes, and D. W. Ussery. RNAmmer: consistent and rapid annotation of ribosomal RNA genes. Nucleic Acids Res, (35):3100–3108, 2007. ISSN 1362-4962." />
<meta name="citation_reference" content="T. M. Lowe and S. R. Eddy. tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res, (25):955–964, 1997." />
<meta name="citation_reference" content="S. Kehr, S. Bartschat, H. Tafer, P. F. Stadler, and J. Hertel. Matching of Soulmates: coevolution of snoRNAs and their targets. Molec Biol Evol, 31(2): 455–467, 2013." />
<meta name="citation_reference" content="C. Cowled, C. R. Stewart, V. A. Likic, M. R. Friedländer, M. Tachedjian, K. A. Jenkins, M. L. Tizard, P. Cottee, G. A. Marsh, P. Zhou, et al. Characterisation of novel microRNAs in the Black flying fox (Pteropus alecto) by deep sequencing. BMC Genom, 15(1):682, 2014." />
<meta name="citation_reference" content="P.-J. Volders, J. Anckaert, K. Verheggen, J. Nuytens, L. Martens, P. Mestdagh, and J. Vandesompele. LNCipedia 5: towards a reference set of human long non-coding RNAs. Nucleic Acids Res, (47):D135–D139, January 2019. ISSN 1362-4962." />
<meta name="citation_reference" content="M. Bernt, A. Donath, F. Jühling, F. Externbrink, C. Florentz, G. Fritzsch, J. Pütz, M. Middendorf, and P. F. Stadler. MITOS: Improved de novo metazoan mitochondrial genome annotation. Mol Phylogenet Evol, 69(2):313–319, 2013." />
<meta name="citation_reference" content="A. R. Khan, M. T. Pervez, M. E. Babar, N. Naveed, and M. Shoaib. A comprehensive study of de novo genome assemblers: current challenges and future prospective. Evol Bioinform, (14):1176934318758650, 2018." />
<meta name="citation_reference" content="A. Mikheenko, G. Valin, A. Prjibelski, V. Saveliev, and A. Gurevich. Icarus: visualizer for de novo assembly evaluation. Bioinformatics, 32(21):3321–3323, 2016." />
<meta name="citation_reference" content="N. A. O’Leary, M. W. Wright, J. R. Brister, S. Ciufo, D. Haddad, R. McVeigh, B. Rajput, B. Robbertse, B. Smith-White, D. Ako-Adjei, et al. Reference sequence (RefSeq) database at [ncbi: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res, 44(D1):D733–D745, 2015." />
<meta name="citation_reference" content="T. I. Shaw, A. Srivastava, W.-C. Chou, L. Liu, A. Hawkinson, T. C. Glenn, R. Adams, and T. Schountz. Transcriptome sequencing and annotation for the Jamaican fruit bat (Artibeus jamaicensis). PloS One, 7(11):e48472, 2012." />
<meta name="citation_reference" content="L. Yuan, F. Geiser, B. Lin, H. Sun, J. Chen, and S. Zhang. Down but not out: the role of microRNAs in hibernating bats. PloS One, 10(8):e0135064, 2015." />
<meta name="citation_reference" content="P. Schattner, S. Barberan-Soler, and T. M. Lowe. A computational screen for mammalian pseudouridylation guide H/ACA RNAs. RNA, 12(1):15–25, 2006." />
<meta name="citation_reference" content="M. McMahon, A. Contreras, and D. Ruggero. Small RNAs with big implications: new insights into H/ACA snoRNA function and their role in human disease. Wiley Interdisciplinary Reviews: RNA, 6(2):173–189, 2015." />
<meta name="citation_reference" content="J. T. Y. Kung, D. Colognori, and J. T. Lee. Long noncoding RNAs: past, present, and future. Genetics, (193):651–669, March 2013. ISSN 1943-2631." />
<meta name="citation_reference" content="J.-W. Nam, S.-W. Choi, and B.-H. You. Incredible RNA: Dual Functions of Coding and Noncoding. Mol Cells, (39):367–374, May 2016. ISSN 0219-1032." />
<meta name="citation_reference" content="P. Kumari and K. Sampath. cncRNAs: Bi-functional RNAs with protein coding and non-coding functions. Semin Cell Dev Biol, 47-48:40–51, December 2015. ISSN 1096-3634." />
<meta name="citation_reference" content="M. Marz, T. Kirsten, and P. F. Stadler. Evolution of spliceosomal snRNA genes in metazoan animals. J Mol Evol, 67(6):594–607, 2008." />
<meta name="citation_reference" content="J. L. Boore. Animal mitochondrial genomes. Nucleic Acids Res, 27(8):1767–1780, 1999." />
<meta name="citation_reference" content="R. Gerst and M. Hölzer. PCAGO: An interactive web service to analyze RNA-Seq data with principal component analysis. bioRxiv, page 433078, 2018." />
<meta name="citation_reference" content="H. Thorvaldsdóttir, J. T. Robinson, and J. P. Mesirov. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Briefings Bioinf, (14):178–192, 2013. ISSN 1477-4054." />
<meta name="twitter:title" content="A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.

Supplementary information is available at [rna.uni-jena.de/supplements/bats][1], the Open Science Framework ([doi.org/10.17605/OSF.IO/4CMDN][2]), and GitHub ([github.com/rnajena/bats_ncrna][3]).

 [1]: http://rna.uni-jena.de/supplements/bats
 [2]: http://doi.org/10.17605/OSF.IO/4CMDN
 [3]: http://github.com/rnajena/bats_ncrna" />
<meta name="og-title" property="og:title" content="A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/738526v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.

Supplementary information is available at [rna.uni-jena.de/supplements/bats][1], the Open Science Framework ([doi.org/10.17605/OSF.IO/4CMDN][2]), and GitHub ([github.com/rnajena/bats_ncrna][3]).

 [1]: http://rna.uni-jena.de/supplements/bats
 [2]: http://doi.org/10.17605/OSF.IO/4CMDN
 [3]: http://github.com/rnajena/bats_ncrna" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-09-25" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/738526v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/738526v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/738526v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/738526v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/921879" />
    <title>A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UbXxMDzAO_ljJQaRGoHZpsgORv82m082gvEA7C6w3o0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"921879","apath":"\/biorxiv\/early\/2019\/09\/25\/738526.atom","pisa":"biorxiv;738526v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;738526v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/738526v3","title":"A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes","identifier":"node\/921879"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UsXZ261rCE8uixP895el130tKIntnhw6KNZWRIOyBywQJ2ROUbAJtMBvPH35pV9040bexwj9z9sh130f3tvfOgniVQgxIHmr6lcH0UHprkLZfYJhgSR27YSBQkCN\\\\\\\/OgHkAOccp9BFkRa31eY5KzWvvpZPDeNluVPOAughwp1ZsBLR934qAVhNdXC0nGe4tChI37czo4Yrl2srpe3pP5kgqJxJIONtc6PmsMccIwOraXmZDK\\\\\\\/TNxutgLopLECpkt1ezLtYTmhQNWPvLiL6\\\\\\\/gnZxHFrPGaAha6skBtL1J2bpkMvCB0cQYObeob1tw\\\\\\\/ivYjCVdpg86TQBlLzphbUxrCAKu4MBsm20wtGQbGrVVq+Ra9Nms9gv6uJ8ODYYc+jwWRftyA0Vke978I6r+7IUlLjb3zRmViVAv8SIQNXWOF6e6UnD8+6l+hrUtWlP\\\\\\\/b1ARMvnAq4XaA\\\\\\\/meMf4NACwfHJpdmpKoyLOc9O5y0LVTw\\\\\\\/rzs8WdFyy+sPldRdOWNHYUaPVCgq064A3MYI5+4uWDZz9xLOiTP39LKYGvQ45hlEksCUfi\\\\\\\/YyOCsDynZlEP7Q8m1DAsfJgp143kJihn29LadRbIsXZdgZWUHonlIRwYyLoEbIs2SKaZZccnBHZaeudJMnQds8KJt9Ukrw+nyVWPR9+EZ1hQnUTY2ExwfczFO1fKxShb4euYJjP0bhWvsH2GpFaPXqc03Jz2yrICaA002zKk8GW9ErzTIwg8JAaqsIjmtr7x\\\\\\\/qBL1T1Bj4nBhQ1qMaiQ0xtHseZwDkJAGCexOpszEQNz33UDQdY9oW+a3iifgTSI405jOFzNNQazt4m4yLBJWU3kkdTy6A6dm4fmFsSPzEN7EQ7UQBCLcLWybR8BHPb0PFe6Z06kOShYshlPvnMqBzRC+33sN2eIQX5FXulZm30DE8HAMCtQlVT4ByQsA7nzVubuh+k0OTtSwdlw9D613bQ9LGKvy9ZMceBzhU8ZmTi5GKyDlcNUvQP2v9IcbtttZrUkZJnE96xHFtwa3IQLABIJ6R+w7FLbmNibx3pj60jVd8rdeBnKMNGSSKVy0Ce7skzT+OhZIKBGX70PNyRG6RQskXa\\\\\\\/i4CjNsonpgciidpHNyi2c59EM1ntLTaN\\\\\\\/NleMB7d4bEjypzggI5d0q5u6\\\\\\\/zYrJfb7HUYQ\\\\\\\/rmhDAjAvFZIDfPNEfMpKEaoQGWYRrJfIcRKqE3frPwwnIBvRVs1Lt+TAaUlEINYwMf7gwx91yEE3TU2rPunTmx348xnVTslSe0eRBOwdZmRhugHmPH17lNhGuQCPeAw1t4OTL14soMbxZK9Te9Lm0afDM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022c702ChvPdnvHdRo1ItixNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002266414d08a0537aaa82a69e523dc86b259f59bfd1c55bcc448ac7cfb3e2f35cfc\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/738526v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-921879 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/738526v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RzSUK3Ht6DZdcOgiVnuK21ziaUlCstxdnO7-pQCwniU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="921879" id="node921879" data-pisa="biorxiv;738526v3" data-pisa-master="biorxiv;738526" data-apath="/biorxiv/early/2019/09/25/738526.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nelly</span> <span class="nlm-surname">Mostajo Berrospi</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3637-5870" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Lataretu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9413-5701" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Krautwurst</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1791-4437" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Florian</span> <span class="nlm-surname">Mock</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3928-9094" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Desirò</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6375-6441" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kevin</span> <span class="nlm-surname">Lamkiewicz</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Maximilian</span> <span class="nlm-surname">Collatz</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Schoen</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Friedemann</span> <span class="nlm-surname">Weber</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-4783-8823" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Manja</span> <span class="nlm-surname">Marz</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-7090-8717" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Hölzer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/738526 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node921879" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nelly Mostajo Berrospi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Virology, Philipps-University Marburg</span>, Hans-Meerwein-Straße 2, 35043 Marburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nelly%2BMostajo%2BBerrospi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mostajo%20Berrospi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANelly%2BMostajo%2BBerrospi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marie Lataretu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BLataretu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lataretu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarie%2BLataretu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3637-5870" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marie Lataretu</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Krautwurst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BKrautwurst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krautwurst%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASebastian%2BKrautwurst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9413-5701" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sebastian Krautwurst</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Florian Mock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BMock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mock%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFlorian%2BMock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1791-4437" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Florian Mock</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel Desirò </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BDesir%C3%B2%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desir%C3%B2%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BDesir%25C3%25B2%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3928-9094" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Desirò</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kevin Lamkiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BLamkiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lamkiewicz%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BLamkiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6375-6441" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin Lamkiewicz</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maximilian Collatz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maximilian%2BCollatz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collatz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaximilian%2BCollatz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Andreas Schoen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute for Virology, FB10-Veterinary Medicine, Justus-Liebig University</span>, 35392 Gießen, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>German Center for Infection Research (DZIF)</span>, partner sites Marburg and Gießen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BSchoen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BSchoen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Friedemann Weber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Virology, Philipps-University Marburg</span>, Hans-Meerwein-Straße 2, 35043 Marburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute for Virology, FB10-Veterinary Medicine, Justus-Liebig University</span>, 35392 Gießen, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>German Center for Infection Research (DZIF)</span>, partner sites Marburg and Gießen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Friedemann%2BWeber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weber%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFriedemann%2BWeber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Manja Marz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>FLI Leibniz Institute for Age Research</span>, Beutenbergstraße 11, 07745 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manja%2BMarz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManja%2BMarz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4783-8823" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manja Marz</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Martin Hölzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>RNA Bioinformatics and High-Throughput Analysis, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center, Friedrich Schiller University Jena</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BH%C3%B6lzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=H%C3%B6lzer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BH%25C3%25B6lzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7090-8717" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Hölzer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.hoelzer@uni-jena.de" class="" data-icon-position="" data-hide-link-title="0">martin.hoelzer@uni-jena.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/738526v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/738526v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/738526v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/738526v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/738526v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/738526v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:921879" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:921879/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Supplementary information</strong> is available at <a href="http://rna.uni-jena.de/supplements/bats">rna.uni-jena.de/supplements/bats</a>, the Open Science Framework (<a href="http://doi.org/10.17605/OSF.IO/4CMDN">doi.org/10.17605/OSF.IO/4CMDN</a>), and GitHub (<a href="http://github.com/rnajena/bats_ncrna">github.com/rnajena/bats_ncrna</a>).</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-9">Bats (Chiroptera) are the most abundant, ecologically diverse and globally distributed animals within all vertebrates<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>, but representative genome arrangements and corresponding coding and non-coding gene annotations are still incomplete<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Except for the extreme polar regions, bats can be found throughout the globe, feeding on diverse sources such as insects, blood, nectar, fruits, and pollen<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup>. Their origin has been dated in the Cretaceous period, with a diversification explosion process dating back to the Eocene<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>.</p><p id="p-10">The 21 bat families known to date are classified into the suborders <em>Yinpterochiroptera</em> and <em>Yan-gochiroptera</em><sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Although they account for more than 20 % of the total living mammalian diversity<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>, the genomes of only 16 bat species of the estimated more than 1,300 species<sup><a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a></sup> have been sequenced with adequate coverage to date and are publicly available (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="We used available genomes of 16 bat species from eight out of 21 families for non-coding RNA annotation in this study. The tree shows their phylogenetic relationship and is based on a molecular consensus on family relationships of bats 4, further adapted and extended from 2. Bat families and species with published genomes currently available in the NCBI are shown (details see Tab. 1). Bat families still lacking a published genome assembly are written in gray color. RNA-Seq data sets were selected from species marked with an asterisk and additionally obtained from a Myotis daubentonii cell line (see Tab. 2). Bat silhouettes were adapted from artworks created by Fiona Reid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1220078789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;We used available genomes of 16 bat species from eight out of 21 families for non-coding RNA annotation in this study. The tree shows their phylogenetic relationship and is based on a molecular consensus on family relationships of bats &lt;sup&gt;4&lt;/sup&gt;, further adapted and extended from &lt;sup&gt;2&lt;/sup&gt;. Bat families and species with published genomes currently available in the &lt;kbd&gt;NCBI&lt;/kbd&gt; are shown (details see Tab. 1). Bat families still lacking a published genome assembly are written in gray color. RNA-Seq data sets were selected from species marked with an asterisk and additionally obtained from a &lt;em&gt;Myotis daubentonii&lt;/em&gt; cell line (see Tab. 2). Bat silhouettes were adapted from artworks created by Fiona Reid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F1.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F1.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-11" class="first-child">We used available genomes of 16 bat species from eight out of 21 families for non-coding RNA annotation in this study. The tree shows their phylogenetic relationship and is based on a molecular consensus on family relationships of bats <sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>, further adapted and extended from <sup><a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a></sup>. Bat families and species with published genomes currently available in the <kbd>NCBI</kbd> are shown (details see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tab. 1</a>). Bat families still lacking a published genome assembly are written in gray color. RNA-Seq data sets were selected from species marked with an asterisk and additionally obtained from a <em>Myotis daubentonii</em> cell line (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tab. 2</a>). Bat silhouettes were adapted from artworks created by Fiona Reid.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/924970/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/924970/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/924970" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-12" class="first-child">We have annotated ncRNAs within 16 bat genomes of different assembly quality. We introduced three-letter abbreviations for each bat species used throughout the manuscript and in supplemental files and annotations. Genome sizes were estimated (<em>Est.</em>) by using <em>C</em>-values (DNA content per pg) from the animal genome size database (<a href="http://genomesize.com">http://genomesize.com</a>) and by applying the following formula: <em>Genome size</em> = (0.978 · 10<sup>9</sup>) <em>C</em>. If multiple entries for one species were available, an average over all <em>C</em>-values was calculated and used to estimate the genome size. If one species could not be found, an average <em>C</em>-value for the corresponding genus was used. Tab. S1 provides additional assembly statistics calculated by <kbd>QUAST</kbd> (v5.0.2) <sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. <kbd>NCBI</kbd> acc. – GenBank assembly accession without the prefix ‘<kbd>GCA</kbd>_’.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/924976/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/924976/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/924976" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-13" class="first-child">Six RNA-Seq data sets comprising alltogether 98 samples derived from four different bat species were used to evaluate our novel ncRNA annotations. All samples were quality trimmed and individually mapped to all 16 bat assemblies using <kbd>HISAT2</kbd><sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> and transcript abundances were subsequently calculated from all 1,568 mappings by <kbd>featureCounts</kbd><sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. We labeld each RNA-Seq data set based on the first authors last name and the year of data set publication. Raw read data of the enriched sequencing of small RNAs (especially miRNAs) of a <em>M. daubentonii</em> cell line (accompanying GSE121301<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>) have been uploaded in the course of this publication under GEO accession GSE132336 (<em>Weber-2019</em>). polyA* – library preparation with mRNA selection; rRNA- – library preparation with rRNA depletion and size selection (&gt;200 nt); sRNA – library preparation with size selection (&lt;200 nt); se/pe – single-/paired-end sequencing; ss/not-ss – strand-specific/unstranded sequencing; ss<sup><em>s</em></sup>/ss<sup><em>a</em></sup> – strand-specific in sense orientation/in anti-sense orientation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Bats have developed a variety of unique biological features that are the rarest among all mammalian, including laryngeal echolocation<sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, vocal learning<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>, and the ability to fly <sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. They occupy a broad range of different ecological niches<sup><a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a></sup>, have an exceptional longevity <sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> and a natural and unique resilience against various pathogenic viruses<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. For example, bats are the suspected reservoirs for some of the deadliest viral diseases such as Ebola and SARS<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, but appear to be asymptomatic and survive the infection. Possibly, the solution to better understand and fight these pathogens lies in the uniquely developed immune system of bats.<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Studying bats and their genomes is likely to have high impacts on various scientific fields, including healthy ageing, immune and ecosystem functioning, the evolution of sensory perception and human communication, and mammalian genome architecture (see the recent Bat1K review for further details<sup><a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a></sup>).</p><p id="p-15">Despite the unique biological characteristics of these flying mammals and their important role as natural reservoirs for viruses, bats are one of the least studied taxa of all mammalian<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Accordingly, there is little knowledge about the non-protein-coding transcriptome of bats, which plays a crucial role in an extensive number of cellular and regular functions and comprises a very diverse family of untranslated RNA molecules<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. In addition, it is believed that due to the early evolutionary radiation of bats (compared to other mammals) their innate and acquired immune responses have a different set of molecules<sup><a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a></sup>.</p><p id="p-16">Genome assemblies and annotations are essential starting points for many molecular-driven and comparative studies.<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. Especially, studies of non-model organisms play important roles in many investigations<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. In most cases, however, these organisms lack well-annotated genomes<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>, which severely limit our ability to gain a deeper understanding of these species and may further impede biomedical research<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-17">In this study, we comprehensively annotated non-coding RNAs in 16 available bat genome assemblies (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tab. 1</a>). For each bat species, we provide final annotations that are compatible with current <kbd>Ensembl</kbd> and <kbd>NCBI</kbd> (National Center for Biotechnology Information) standards (<kbd>GTF</kbd> format) and that can be directly used in other studies, for example for differential gene expression analysis. We compare our new annotations with the currently available annotations for bats and show that a large number of non-coding genes are simply not annotated and are therefore overlooked by other studies. We used six RNA-Seq data sets comprising different conditions (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tab. 2</a>) to validate our annotations and to determine the expression levels of our newly annotated ncRNAs. Exemplarily, we show that our novel annotations can be used to identify ncRNAs that are significantly differential expressed during viral infections and were missed by previous studies.</p></div><div class="section" id="sec-3"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-4" class="subsection"><h3>Bat species and genomic assembly data</h3><p id="p-18">We downloaded the last recent genome versions for 16 bat species in September 2018 from the <kbd>NCBI</kbd> genome data base (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tab. 1</a>). Within the order of <em>Yinpterochiroptera</em>, nine genomic sequences were obtained covering the bat families <em>Pteropodidae</em>, <em>Hipposideridae</em>, <em>Rhinolophidae</em>, and <em>Megadermatidae</em> whereas for the order of <em>Yangochiroptera</em> another seven genome assemblies were available for the bat families <em>Phyllostomidae</em>, <em>Mormoopidae</em>, <em>Vespertilionidae</em>, and <em>Miniopteridae</em> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). We introduced a unique three-letter abbreviation code (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tab. 1</a>) to easily distinguish between the 16 bat species in the manuscript and intermediate annotation files provided in the Electronic Supplement. We used <kbd>QUAST</kbd> (v5.0.2)<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup> to calculate several assembly statistics for all genomes, shown in Tab. S1.</p><p id="p-19">At the end of 2018, two new bat genomes were presented by the Bat1K project (<a href="http://bat1k.ucd.ie/">http://bat1k.ucd.ie/</a>)<sup><a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">2</a></sup>, comprising a newer version of the greater horseshoe bat genome (<em>Rhinolophus ferrumequinum</em>; <em>Rhinolophidae</em>) and the genome of the pale spear-nose bat (<em>Phyllostomus discolor</em> ; <em>Phyllostomidae</em>). However, these two bat genomes were not included in our current study due to the data use policy of the Bat1K consortium and to support a fair and productive use of these data.</p></div><div id="sec-5" class="subsection"><h3>RNA-Seq data</h3><p id="p-20">To validate our novel ncRNA predictions, we selected six RNA-Seq data sets<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>–<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> comprising alltogether 98 samples gathered from four different bat species. We have labeled each published RNA-Seq data set based on the first authors last name and the year of data set publication (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Tab. 2</a> and Tab. S2).</p><p id="p-21">All samples were quality trimmed using <kbd>Trimmomatic</kbd><sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> (v0.36) with a 4 nt-step sliding-window approach (Q20) and a minimum remaining read length of 20 nt. For the <em>Field-2018</em> data set<sup><a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a></sup> we additionally removed the three leading 5’ nucleotides from the reads of each sample because of a generally low quality observed by <kbd>FastQC</kbd> (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) (v0.11.7). The remaining reads of all processed samples were individually mapped to all 16 bat assemblies using <kbd>HISAT2</kbd><sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup> (v2.1.0) and transcript abundances were subsequently calculated from all resulting 1,568 mappings by <kbd>featureCounts</kbd><sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup> (v1.6.3). If suitable, the appropriate strand-specific counting mode was applied for each data set (see <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Tab. 2</a> for information about the strand specificity). For each bat genome assembly, the merged annotation of already known (<kbd>NCBI</kbd>) and newly identified (this study) ncRNAs was used (Files S1). Due to the size of the annotations and the huge amount of overlaps, long ncRNAs were counted and analyzed for differential expression separately.</p><p id="p-22">To enable a better investigation of small RNAs (sRNAs), we included a data set of the targeted sequencing of sRNAs (especially miRNAs) from <em>M. daubentonii</em> cells (<em>Weber-2019</em>). To obtain this sRNA sequencing data, total RNA of 18 samples, that was obtained using the same procedure like explained for the rRNA-depleted <em>M. daubentonii</em> data<sup><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></sup>, was preprocessed using the Illumina TruSeq smallRNA protocol, sequenced on one HiSeq 2500 lane, and finally uploaded in the course of this study under GEO accession GSE132336. The reads of these 18 sRNA samples were additionally preprocessed by removing potential adapter sequences with <kbd>cutadapt</kbd><sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> (v1.8.3) followed by a quality (Q20) trimming using again a window-size of 4 and a minimum length of 15 nt by <kbd>PrinSeq</kbd><sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> (v0.20.3). The processed sRNA samples were either individually mapped to the 16 bat genomes for differential expression analysis or combined and mapped on each bat genome to predict known and novel miRNAs with <kbd>miRDeep2</kbd><sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>.</p></div><div id="sec-6" class="subsection"><h3>Differential gene expression analysis</h3><p id="p-23">Only uniquely mapped reads were counted and used for the differential gene expression analyses with <kbd>DESeq2</kbd><sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> (v1.16.1). Annotated rRNA genes were removed prior <kbd>DESeq2</kbd> and TPM (transcripts per million) analysis. All raw read counts from samples of one data set were combined and normalized together using the built-in functionality of DESeq2, followed by pair-wise comparisons to detect significant (adjusted p-value &lt; 0.05; absolute log<sub>2</sub> fold change &gt; 2) differential expressed ncRNAs.</p><p id="p-24">Besides the <kbd>DESeq2</kbd> normalization, we calculated TPM values for each ncRNA in each sample as previously described<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-4.gif"/></noscript></span>

</span>
where <em>c</em><sub><em>i</em></sub> is the raw read count of ncRNA <em>i</em>, <em>l</em><sub><em>i</em></sub> is the length of ncRNA <em>i</em> (and the cumulative exon length in the case of IncRNAs) and <em>N</em> is the number of all ncRNAs in the given annotation. To this end, we obtained for each RNA-Seq sample, each bat annotation, and each ncRNA one TPM value representing the normalized expression level of this ncRNA. If available, we calculated all TPM values in relation to the expression of all already known coding and non-coding genes and not only based on our novel ncRNA annotation.</p><p id="p-25">Although we performed mappings, read countings, and normalization for all samples, bat genome assemblies and all six data sets (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Tab. 2</a>; overall 1,568 mappings), we only selected one comparison per data set to exemplarily show novel and significantly differential expressed ncRNAs (Files S2.1-S2.15; divided by data set and input annotation). detected with the help of our extended annotations (Tab. S11). For each data set, we chose the bat species that was also used in the corresponding study. For the <em>Hölzer-2019</em> and <em>Weber-2019</em> data sets we used the closely related <em>M. lucifugus</em> genome assembly and annotation as a reference, because currently no genomic sequence for <em>M. daubentonii</em> is publicly available.</p><div id="sec-7" class="subsection"><h4><em>Field-2015</em> (11 samples, MLU)</h4><p id="p-26">We compared control (<em>mock</em>) samples (5 replicates) with the <em>infected</em> (white-nose syndrome, WNS) samples (6 replicates) obtained from wing tissue of <em>M. lucifugus</em> to identify novel ncRNAs differentially expressed during the infection with the psychrophilic fungus <em>Pseudogymnoascus destructans</em><sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>.</p></div><div id="sec-8" class="subsection"><h4><em>Eckalbar-2016</em> (18 samples, MNA)</h4><p id="p-27">To identify new ncRNAs playing different roles during the development of the bat wing, we compared forelimb samples and hindlimb samples (each of 3 developmental stages in 3 replicates) of <em>M. natalensis</em> independent from the embryonic stages<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup>.</p></div><div id="sec-9" class="subsection"><h4><em>Hölzer-2016</em> (9 samples, RAE)</h4><p id="p-28">Here, we investigated novel ncRNAs that might play a role due to transcriptional changes between un-infected (<em>mock</em>) samples and Ebola/Marburg virus infected samples regardless of the time point of infection and obtained from <em>R. aegyptiacus</em> cells<sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a></sup>. Due to the lack of real biological replicates, we only calculated TPM values and did not include this data set into the differential expression analysis with <kbd>DESeq2</kbd>.</p></div><div id="sec-10" class="subsection"><h4><em>Field-2018</em> (24 samples, MLU)</h4><p id="p-29">As Field <em>et al.</em> were interested in host transcriptomic responses to a fungal pathogen (<em>P. destructans</em>, <em>Pd</em>) during torpor in hibernating bats (<em>M. lucifugus</em>), we searched for novel ncRNAs that were differentially affected between <em>P. destructans</em> positive (<em>Pd+</em>) samples, obtained although the bats were still torpid (3–6°C; 6 replicates), and <em>Pd−</em> samples taken after the bats were allowed to warm to euthermic temperature (6 replicates)<sup><a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a></sup>.</p></div><div id="sec-11" class="subsection"><h4><em>Hölzer-2019</em>/<em>Weber-2019</em> (18 samples, MLU)</h4><p id="p-30">For these studies, RNA was extracted from 18 samples of <em>M. daubentonii</em> tissue either left uninfected (<em>mock</em>), infected with virus (RVFV Clone 13), or stimulated with interferon (IFN) at two different time points<sup><a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a></sup>. Sequencing was performed with two different protocols resulting in data sets: <em>Hölzer-2019</em> (rRNA-depleted) and <em>Weber-2019</em> (smallRNA-enriched). For both data sets we compared <em>mock</em> and virus-infected (Clone 13) samples at 24 h post infection (each with 3 replicates) in more detail (Files S2) and present for all 18 smallRNA-Seq samples (<em>Weber-2019</em>) normalized expression values in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>. Please note, that for <em>M. daubentonii</em> currently no genome assembly is available, so the genome assembly of <em>M. lucifugus</em> was used as a close relative.</p></div></div><div id="sec-12" class="subsection"><h3>Annotation file format</h3><p id="p-31">All of our annotations follow the <em>General Transfer Format</em> (<kbd>GTF</kbd>) as described and defined in the <kbd>Ensembl</kbd><sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> data base (<a href="https://ensembl.org/info/website/upload/gff.html">https://ensembl.org/info/website/upload/gff.html</a>). Therefore, each row of each annotation file is either defined as a <em>gene</em>, <em>transcript</em>, or <em>exon</em> (by the feature column) and strictly following a hierarchical structure, even if only one exon (as for most ncRNAs) is reported. By adhering to this annotation format, our novel annotations can be easily merged with existing ones as derived from <kbd>Ensembl</kbd> or <kbd>NCBI</kbd> and are directly usable as input for computational tools such as <kbd>HISAT2</kbd> for mapping or <kbd>featureCounts</kbd> for transcript abundance estimation. We defined gene, transcript, and exon IDs following the <kbd>Ensembl</kbd> pattern:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-32">For example, the ID:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/25/738526/embed/graphic-6.gif"/></noscript></span>

</span>
denotes the first (00000000001) rRNA (R) gene (G) annotated in the <em>M. lucifugs</em> (MLU) genome. We defined the following abbreviations for different ncRNA types: rRNA (R), tRNA (T), miRNA (M), miRNA with <kbd>mirDeep2</kbd> (D), snoRNA (S), ncRNA/miscRNA/other (N), IncRNA (L), IncRNA hot-spot (H), mitochondrial ncRNA (O).</p></div><div id="sec-13" class="subsection"><h3>Annotation of non-coding RNAs</h3><p id="p-33">In general, we used specialized computational tools for the annotation of specific ncRNA classes (Tab. S3–S10. If not otherwise stated, the main ncRNA-discovery is based on homology searches against the <kbd>Rfam</kbd> database<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>–<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> (v14.0). We used the <kbd>Gorap</kbd> pipeline (<a href="https://github.com/koriege/gorap">https://github.com/koriege/gorap</a>), a specially developed software suite for genome-wide ncRNA screening. <kbd>Gorap</kbd> screens genomic sequences for all ncRNAs present in the <kbd>Rfam</kbd> database using a generalized strategy by applying multiple filters and specialized software tools. To this end, <kbd>Gorap</kbd> takes huge advantage of <kbd>Infernal</kbd> (v1.1.2)<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> to annotate ncRNAs based on input alignment files conserved in sequence and secondary structure (so-called <em>Stockholm</em> alignment files; <kbd>stk</kbd>). All resulting alignment files were automatically pre-filtered by <kbd>Gorap</kbd> based on different ncRNA class-specific parameters including taxonomy, secondary structure, and primary sequence comparisons. Due to repeats, pseudogenes, undiscriminable un-/functional genes, and overlapping results from the different assembly methods, we defined a ncRNA set per species for annotation that includes filtered sequences, but allows for variants and multiple copies. This final annotation set is defined by hand-curating the resulting <kbd>stk</kbd> alignments of <kbd>Gorap</kbd> with the help of <kbd>Emacs RALEE</kbd> mode<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. Due to the removal of sequences in the stockholm alignments, the remaining sequences were extracted and again aligned into stockholm format using <kbd>cmalign -noprob</kbd> from <kbd>Infernal</kbd>. The <kbd>Rfam</kbd>-derived ncRNA alignments were further split into <em>snoRNAs</em> (Tab. S5), <em>miRNAs</em> (Tab. S6), and <em>other ncRNAs</em> including snRNAs, IncRNAs, and other structural RNAs (Tab. S8).</p><p id="p-34">In general, our annotation results give an overview about the amount of different ncRNAs in bat species and intentionally can include false positive hits and duplicates. All ncRNA hits are placed as <kbd>STK</kbd> (if available), <kbd>GTF</kbd> and <kbd>FASTA</kbd>-files in the Electronic Supplement and OSF (<a href="http://doi.org/10.17605/OSF.IO/4CMDN">doi.org/10.17605/OSF.IO/4CMDN</a>).</p><div id="sec-14" class="subsection"><h4>rRNAs</h4><p id="p-35">We used the prediction tool <kbd>RNAmmer</kbd> <sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup> (v1.2) to identify 5.8S, 18S and 28S rRNA genes using hidden markov models. The tools output was transformed into regular <kbd>GTF</kbd> file format. All output files can be found in Tab. S3.</p></div><div id="sec-15" class="subsection"><h4>tRNAs</h4><p id="p-36">For the annotation of tRNAs we applied <kbd>tRNAscan-SE</kbd><sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> (v1.3.1) to the bat contigs using default parameters. The results were filtered by removing any tRNAs of type “Un-det” or “Pseudo” and the tabular output was transformed into the <kbd>GTF</kbd> file format. Additional information about the anticodon and the coverage score were added to the description column. We provide the raw <kbd>tRNAscan-SE</kbd> files in Tab. S4.</p></div><div id="sec-16" class="subsection"><h4>snoRNAs</h4><p id="p-37">We annotated snoRNAs based on available alignments from the <kbd>Rfam</kbd> data base using <kbd>Gorap</kbd> and additionally marked and classified them into box C/D and box H/ACA when intersecting with the set of snoRNAs from <a href="http://www.bioinf.uni-leipzig.de/publications/supplements/12-022">http://www.bioinf.uni-leipzig.de/publications/supplements/12-022</a><sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> (Tab. S5).</p></div><div id="sec-17" class="subsection"><h4>miRNAs</h4><p id="p-38">Additionally to the <kbd>Rfam</kbd>-screening (Tab. S6), miRNAs were predicted by the <kbd>miRDeep2</kbd> pipeline<sup><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup> (v2.0.0.8) using default parameters and the combined smallRNA-Seq data set (<em>Weber-2019</em>; 18 samples) mapped to each individual bat assembly as an input for <kbd>miRDeep2</kbd> (Tab. S7).</p><p id="p-39">To validate the accuracy of our approach, we compared our <kbd>miRDeep2</kbd> annotations of <em>M. lucifugus</em>/<em>P. alecto</em> (based on the transcriptomic data derived from <em>M. daubentonii</em>; <em>Weber-2019</em>) with annotations of miRNAs for transcriptomic data of <em>Myotis myotis</em> <sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></sup> and <em>P. alecto</em><sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>. For reference mapping, Huang <em>et al.</em> also used the <em>M. lucifugus</em> genome, so we were theoretically able to directly compare our annotations with the annotations of both studies. Unfortunately, no positional information (annotation file) of the identified miRNAs derived from the transcriptomic data of <em>M. myotis</em> were given in the manuscript or supplement<sup><a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a></sup>. Therefore, we blasted the precursor miRNA sequences identified with the help of the <em>M. myotis</em> transcriptome against the <em>M. lucifugus</em> genome and retained only hits with a sequence identity of 100 %. The so obtained positional information was further used to calculate the overlap between our predicted miRNAs in <em>M. lucifugus</em>. We used the same approach for the <em>P. alecto</em> comparison. If the annotated location of an miRNA and one of our identified miRNA locations in <em>M. lucifugus</em>/<em>P.alecto</em> were overlapping by at least 85 %, we counted this location as a common prediction.</p></div><div id="sec-18" class="subsection"><h4>IncRNAs</h4><p id="p-40">Long ncRNAs were annotated using a high confidence data set <em>H</em> from the <kbd>LNCipedia</kbd><sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup> (v5.2) data base comprising 107,039 transcripts of potential human IncRNAs. The transcripts were used as input for a <kbd>BLASTn</kbd> (2.7.1+, 1<em>e</em><sup>−10</sup>) search against each of the 16 bat assemblies (compiled as <kbd>BLAST</kbd> databases). The <kbd>BLASTn</kbd> result for each bat assembly was further processed to group single hits into potential transcripts as follows: first, for each query sequence <em>q</em> ∈ <em>H</em>, hits of <em>q</em> found on the same contig <em>c</em> and strand <em>s</em> were selected (<em>hits</em><sub><em>c</em>,<em>s</em>,<em>q</em></sub>) and the longest one, <em>q</em><sub>1</sub>, was chosen as a starting point, so that <em>trscp</em><sub><em>c,s,q</em></sub> = (<em>q</em><sub>1</sub>). Second, all hits <em>q</em><sub><em>i</em></sub> ∈ <em>hits</em><sub><em>c,s,q</em></sub> with <em>q</em><sub><em>i</em></sub> ∉ <em>trscp</em><sub><em>c,s,q</em></sub>, that do not overlap neither in the query <em>q</em> nor in the target sequence and do not exceed a maximum range of 500,000 nt from the most up-stream to the most down-stream target sequence position of all <em>q</em><sub><em>j</em></sub> ∈ <em>trscp</em><sub><em>c,s,q</em></sub> ⋃ <em>q</em><sub><em>i</em></sub>, were added iteratively to <em>trscp</em><sub><em>c,s,q</em></sub>. To this end, we introduced a simple model of exon-intron structures, naturally occurring when using transcript sequences as queries against a target genome assembly. We defined the 500,000 nt search range based on an estimation of IncRNA gene sizes derived from the human <kbd>Ensembl</kbd><sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup> annotation. If the sum of the lengths of all <em>q</em><sub><em>i</em></sub> ∈ <em>trscp</em><sub><em>c,s,q</em></sub> covers the query transcript length <em>length</em><sub><em>q</em></sub> at least for 70 %, <em>trscp</em><sub><em>c,s,q</em></sub> was considered as a transcript and its elements <em>q</em><sub><em>i</em></sub> as exons, otherwise all <em>q</em><sub><em>i</em></sub> ∈ <em>trscp</em><sub><em>c,s,q</em></sub> were withdrawn. This procedure is repeated until all hits ∈ <em>hits</em><sub><em>c,s,q</em></sub> were used or withdrawn. Therefore, each so-defined group of non-overlapping hits derived from the same query sequence and found on the same contig and strand should represent a IncRNA transcript with its (rough) exon structure. The defined transcripts were saved as <kbd>BLAST</kbd>-like output and transformed into <kbd>GTF</kbd> file format. To follow the <kbd>GTF</kbd> annotation format and to harmonize our IncRNA annotations with the other ncRNA annotations, each IncRNA transcript was also saved as a gene annotation and consists of at least one exon.</p><p id="p-41">As we observed a lot of different sequences from <kbd>LNCipedia</kbd> aligning to the same positions in the genomes, we decided to condense exons at the same sequence positions, considering transcripts with one or multiple exons separately. For each contig and strand, starting from the 5’ end, exons with a minimum overlap of 10 nt were grouped together. In the case of multiple exons, groups of exons were merged, if they shared any transcript origin. If all exons in the group originated from the same <kbd>LNCipedia</kbd> gene, the group was considered as one gene with several transcripts and its associated exon(s). Otherwise, we defined a IncRNA <em>hot spot</em> on gene level with several transcripts and their associated exon(s). The <kbd>LNCipedia</kbd> names of the gathered transcripts of a IncRNA hot spot, as well as start and end positions of all exons, are listed in the <kbd>GTF</kbd> gene attribute field (Tab. S9). The scripts used for the identification of IncRNAs can be found at <a href="https://github.com/rnajena/bats_ncrna">https://github.com/rnajena/bats_ncrna</a>.</p></div><div id="sec-19" class="subsection"><h4>Mitochondrial DNA</h4><p id="p-42">As not all of the 16 genome assemblies are including contigs representing the mitochondrial DNA (mtDNA), we downloaded for MLU (KP273591), MBR (KM199849), MDA (KM233172), PAL (NC_023122), PVA (KP214033), PPA (KF752590), RFE (NC_020326), RAE (NC_007393), HAR (NC_018540), and DRO (NC_022423) mitochondrial genomes from the <kbd>NCBI</kbd> (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Tab. 3</a>). For the other six species we used <kbd>BLASTn</kbd> (2.7.1+, 1<em>e</em><sup>−10</sup>) with the MLU and PVA mitochondrial gemomes as queries against the remaining bat genomes. For EHE we found a possible mtDNA contig in full-length (16,141 nt; AWHC01200796) in the genome assembly. Due to the circularization of mtDNA, we rearranged the sequence of this contig to start with the gene coding for the phenylalanin tRNA and to match the gene order of the other mitochondrial genomes. Only for ESP, RSI, MLY, EFU, and MNA, we were not able to detect any possible contigs of mtDNA (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Tab. 3</a>). All eleven mitochondrial genomes were annotated with <kbd>MITOS</kbd><sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. The ncRNA results were filtered by e-value (threshold 0.001), thus one of two small rRNAs in MBR and RFE and one of two large rRNAs in EHE were discarded as false positive hits (Tab. S10). For the five bat assemblies directly including mtDNAs (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Tab. 3</a>), the MITOS annotations were added to the final merged ncRNA annotation. All other mtDNAs and annotations can be found in Tab. S10.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/924972/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/924972/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/924972" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-43" class="first-child">Mitochondrial bat genomes (mtDNA) publicly available and used for annotation with <kbd>MITOS</kbd><sup><a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a></sup>. For 10 out of the 16 bat species investigated in this study, mtDNA could be found in the <kbd>NCBI</kbd>. For four bat species, the mtDNA is also part of the genome assembly as determined using <kbd>BLASTn</kbd>. For <em>E. helvum</em>, no mtDNA could be found in the <kbd>NCBI</kbd>, but we were able to identify a single contig that is part of the genome assembly as mtDNA using <kbd>BLASTn</kbd> and the mitochondrial genomes of the other bats as query. The contig was rearranged to match the gene order of the other mtDNAs. <sup><em>R</em></sup> – found via <kbd>BLASTn</kbd> and rearranged.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-20" class="subsection"><h3>Computational merging of gene annotations</h3><p id="p-44">As we annotated all bat assemblies by using different tools, we needed to merge the resulting <kbd>GTF</kbd> files to resolve overlapping annotations and to recieve a final annotation of ncRNAs for each bat species. Furthermore, we extended the available <kbd>NCBI</kbd> annotations (including protein- and non-coding genes) by integrating our novel ncRNA annotations. The scripts used to merge the different annotation files and to calculate overlaps between annotations can be found at <a href="https://github.com/rnajena/bats_ncrna">https://github.com/rnajena/bats_ncrna</a>. Due to their size, we have not included the IncRNA annotations based on <kbd>LNCipedia</kbd>. These can be downloaded and used separately (Tab. S9).</p><div id="sec-21" class="subsection"><h4>Merge of novel non-coding annotations</h4><p id="p-45">For each bat species we merged the ncRNA annotations (except for IncRNAs) using a custom script (merge_gtf_global_ids.py). After reading in all features and asserting correct file structure, overlaps were resolved in the following manner: (1) exons are considered overlapping if more than 50 % of the shorter one is covered by the larger one. (2) if only one of the overlapping set is of biotype <em>protein-coding</em>, remove all others. (3) for further ties, keep only the exon that is highest on a priority list based on annotation source. (4) for further ties, keep only the longest of the exons. For each exon to be removed the corresponding transcript is deleted, and gene records that lost all transcripts are also deleted.</p></div><div id="sec-22" class="subsection"><h4>Merge of <kbd>NCBI</kbd> and novel annotations</h4><p id="p-46">We first converted and filtered the <kbd>NCBI</kbd> annotations to a compatible format with a custom script (format_ncbi.py) and then combined the results with our merged novel ncRNA annotations using the same strategy to resolve overlaps as above, but imposing less strict format rules (merge_gtf_ncbi.py).</p></div></div></div><div class="section" id="sec-23"><h2 class="">RESULTS</h2><div id="sec-24" class="subsection"><h3>Assembly and annotation quality differs among bat species</h3><p id="p-47">At best, a genome assembly represents the full genetic content of a species at chromosome level. Whereas the first complex eukaryotic genomes were generated using Sanger chemistry, todays technologies such as Illumina short-read sequencing and PacBio or Oxford Nanopore long-read approaches are increasingly used<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. The currently available bat genomes vary wiedly regarding their assembly quality and completness (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Tab. 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>, and Tab. S1) and were predominantly assembled by using short Illumina-derived reads and low (~18 X)<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> up to moderate/higher coverage (77–218 X) approaches<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>–<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a></sup>. A new assembly of the cave nectar bat (<em>Eonycteris spelaea</em>)<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> was exclusivly generated from long-read data derived from the PacBio platform and the genome of the Egyptian fruit bat (<em>Rousettus aegyptiacus</em>)<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> was assembled by using a hybrid-approach of Illumina and PacBio data. These two genomes from the <em>Pteropodidae</em> family are of a generally higher quality (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>, Tab. S1). Regardless of their assembly quality, these genomes need to be annotated to identify regions of interest, for example, encoding for protein- and non-coding genes or other regulatory elements.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A contig size view of all bat genome assemblies annotated in this study. Only the largest 1,000 contigs of each assembly are shown. Gray bars indicate all contigs that are shorter than the 1,000 largest contigs. N50/N75 labels mark the position of the contig with length equal to the N50/N75 value. Due to a low-coverage approach28, the assemblies of E. helvum, R. ferrumequinum, M. lyra, and P. parnellii are of generally lower quality (see Tab. 1 and Tab. S1). Thus, N50/N75 values are not visualized for these species. The figure was adapted from the Icarus viewer59 used as a part of QUAST24." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1220078789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A contig size view of all bat genome assemblies annotated in this study. Only the largest 1,000 contigs of each assembly are shown. Gray bars indicate all contigs that are shorter than the 1,000 largest contigs. N50/N75 labels mark the position of the contig with length equal to the N50/N75 value. Due to a low-coverage approach28, the assemblies of &lt;em&gt;E. helvum&lt;/em&gt;, &lt;em&gt;R. ferrumequinum&lt;/em&gt;, &lt;em&gt;M. lyra&lt;/em&gt;, and &lt;em&gt;P. parnellii&lt;/em&gt; are of generally lower quality (see Tab. 1 and Tab. S1). Thus, N50/N75 values are not visualized for these species. The figure was adapted from the &lt;kbd&gt;Icarus&lt;/kbd&gt; viewer&lt;sup&gt;59&lt;/sup&gt; used as a part of &lt;kbd&gt;QUAST&lt;sup&gt;24&lt;/sup&gt;&lt;/kbd&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F2.medium.gif" width="440" height="94"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F2.medium.gif" width="440" height="94"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-48" class="first-child">A contig size view of all bat genome assemblies annotated in this study. Only the largest 1,000 contigs of each assembly are shown. Gray bars indicate all contigs that are shorter than the 1,000 largest contigs. N50/N75 labels mark the position of the contig with length equal to the N50/N75 value. Due to a low-coverage approach28, the assemblies of <em>E. helvum</em>, <em>R. ferrumequinum</em>, <em>M. lyra</em>, and <em>P. parnellii</em> are of generally lower quality (see <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Tab. 1</a> and Tab. S1). Thus, N50/N75 values are not visualized for these species. The figure was adapted from the <kbd>Icarus</kbd> viewer<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup> used as a part of <kbd>QUAST<sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a></sup></kbd>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Non-coding RNAs are underrepresented in current bat genome annotations</h3><p id="p-49">Current genome annotations, mostly generated by automatic annotation pipelines provided by databases such as the <kbd>NCBI</kbd><sup><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup> or <kbd>Ensembl</kbd><sup><a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a></sup>, are predominantly focusing on protein-coding genes and well studied ncRNAs such as tRNAs and rRNAs. Accordingly, the available bat genome annotations vary a lot regarding their quality, ranging from more comprehensive annotations for long-standing bat genomes such as <em>M. lucifugus</em> or <em>P. vampyrus</em> to annotations on <em>region</em> level, completely missing any coding or non-coding gene annotations at the current <kbd>NCBI</kbd> version (<a id="xref-fig-3-1" class="xref-figure" href="#F3">Fig. 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Tab. 4</a>). Furthermore, and by using strand-specific RNA-Seq data, we could show that some genes (e.g. <em>IFNA5</em>/<em>IFNW2</em> in the <kbd>Ensembl</kbd> annotation of <em>M. lucifugus</em> <sup><a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">36</a></sup>) are annotated on the false strand and are therefore entirely missed by differential expression studies when relying on a strand-specific read quantification. For all publicly available bat genomes, ncRNAs are generally annotated on low levels and are highly incomplete, mostly only comprising some tRNAs, rRNAs, snRNAs, snoRNAs, and IncRNAs (<a id="xref-fig-3-2" class="xref-figure" href="#F3">Fig. 3</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Tab. 4</a>). Therefore, many ncRNAs, especially miRNAs, are simply overlooked by current molecular studies, for example from RNA-Seq studies that aim to call differential expressed genes based on such in-complete genome annotation files. Studies that have made additional effort on annotating ncRNAs in bats<sup><a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup> are not reporting their results on a level that can be directly used for further computational assessment (e.g. as an direct input for RNA-Seq abundance estimation). Currently, in the <kbd>NCBI</kbd> database, five bat assemblies are entirely lacking any coding/non-coding annotations and miRNAs are not annotated at all (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Tab. 4</a>). The <kbd>Rfam</kbd> database<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> contains mainly for <em>M. lucifugus</em> and <em>P. vampyrus</em> 336 ncRNA families. Other ncRNAs are currently unknown from bat genomes or not well documented.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/924971/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/924971/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/924971" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4:</span> <p id="p-50" class="first-child">Currently annotated protein-coding genes and ncRNAs of the 16 bat genome assemblies obtained from <kbd>NCBI</kbd>. Genes were counted by checking the <em>gene biotype</em> tag for all <em>gene</em> entries (third column in the <kbd>GTF</kbd> file). Five bat species (EHE, ESP, RFE, MLY, PPA) are currently missing any protein- and non-coding gene annotations and only provide <em>regions</em> for each scaffold in the assembly. The annotation of <em>E. fuscus</em> is completely missing any <em>gene biotype</em> tags. For the description of the three-letter abbreviations please refer to <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Tab. 1</a>. misc. RNA – miscellaneous RNA, not classifying into the other groups.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Identification and validation of ncRNAs in 16 bat genomes</h3><p id="p-51">The genome assembly status of different bat species varies widely: ranging for example from 29,801 contigs and an N50 of 26,869,735 nt (<em>D. rotundus</em>) to 192,872 contigs and an N50 of only 16,881 nt (<em>M. lyra</em>), see <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Tab. 1</a> and Tab. S1. Accordingly, also the annotation status varies a lot (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Tab. 4</a>). Within this work, we give an overview of potential ncRNA annotations in bats. However, the precise number of ncRNAs remains unclear, because of ncRNAs being present several times in the assemblies, and others still remaining undiscovered.</p><p id="p-52">To give a better estimation of transcribed and potentially functional ncRNAs, we used six Illumina short-read RNA-Seq data sets derived from four bat species (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Tab. 2</a>) to estimate the expression levels of our novel annotations. Note that we refer throughout this paper to an RNA-Seq data set by the first author’s name and the year of the respective data set publication. The only included data set derived from a species of the <em>Yinpterochiroptera</em> suborder (<em>R. aegyptiacus</em>) was obtained from a study dealing with the differential transcriptional responses of Ebola and Marburg virus infections in human and bat cells<sup><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a></sup> (data set: <em>Hölzer-2016</em>). In this study, total RNA of nine samples of R06E-J cells, either challenged by the Ebola or Marburg virus or left un-infected, were harvested at 3, 7, or 23 h post infection (poi) and sequenced. Unfortunately, no biological replicates could be generated for this study. Therefore, we did not use this data set for the differential expression analysis, but also calculated normalized expression values (TPM; transcripts per million) as done for all RNA-Seq data sets. The other five data sets comprise <em>Yangochiroptera</em> species of the <em>Vespertilionidae</em> (<em>M. lucifugus, M. daubentonii</em>) and <em>Miniopteridae</em> (<em>M. natalensis</em>) families (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Tab. 2</a>). Field <em>et al.</em> conducted two transcriptomic studies<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>,<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">38</a></sup> (<em>Field-2015</em>, <em>Field-2018</em>) using wing tissue of the hibernating little brown myotis bat. They were especially interested in transcriptional changes between un-infected wing tissue and adjacent tissue infected with <em>Pseudogymnoascus destructans</em>, the fungal pathogen that causes the white-nose syndrome. Two other data sets were obtained from virus-(RVFV Clone 13) and interferon (IFN) alpha-induced transcriptomes of a <em>Myotis daubentonii</em> kidney cell line (MyDauNi/2c). RNA of mock, IFN and Clone 13 samples were gathered at two time points, 6 h and 24 h poi<sup><a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">36</a></sup>. From the same samples, rRNA-depleted (<em>Hölzer-2019</em>) and smallRNA-concentrated (<em>Weber-2019</em>; see Methods for details) libraries were generated and sequenced. Finally, we used data of the long-fingered bat <em>M. natalensis</em> initially obtained to characterize the developing bat wing<sup><a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">33</a></sup>. Here, total RNA was extracted from paired forelimbs and hindlimbs from three individuals at three developmental stage.</p><p id="p-53">We have mapped each sample to each bat genome, regardless of the origin of the RNA. Expectedly, the mapping rate decreases in bat species that are evolutionary more far away from the original species from which the RNA was sequenced. However, we were interested to find out which ncRNAs are consistently transcribed in all investigated bat species or only in certain bat families and sub-groups.</p><p id="p-54">Over all 16 bat assemblies, we annotated ncRNA families for in total 23 tRNAs, 3 rRNAs, 193 miRNAs (2,680 predicted by <kbd>mirDeep2</kbd>), 162 snoRNAs, 22 mitochondrial (mt-)tRNAs and 2 mt-rRNAs as well as 244 other ncRNAs additionally derived from the <kbd>Rfam</kbd> database<sup><a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup> (selected ncRNAs are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Tab. 5</a>). With a broad approach, we have identified 24,316 potential IncRNAs and defined between 27,149 (<em>M. lyra</em>) and 158,135 (<em>D. rotundus</em>) IncRNA hot spots.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/924969/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/924969/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/924969" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5:</span> <p id="p-55" class="first-child">General genome information for each of the 16 investigated bat assemblies and selected ncRNA examples annotated in this study. We selected ncRNAs with interesting copy number distributions among the investigated bat species. For SNORA38, PVT1_5 and HOTTIP_2 and 3 we additionally found sophisticated differential expression patterns in at least one of the used RNA-Seq data sets (absolute log2 fold-change greater 1, TPM&gt; 10). Full tables and detailed information for each ncRNA class (<kbd>FASTA, STK, GTF</kbd> files) can be found in the Electronic Supplement online (Tab. S3–S10). Members of the <em>Vespertilionidae</em> as well as <em>D. rotundus</em> and <em>M. natalensis</em> appear to have a slightly higher GC-content (+ ~2 %). PVA – <em>Pteropus vampyrus</em>; PAL – <em>Pteropus alecto</em>; RAE – <em>Rousettus aegyptiacus</em>; EHE – <em>Eidolon helvum</em>; ESP – <em>Eonycteris spelaea</em>; HAR – <em>Hipposideros armiger</em> ; RSI – <em>Rhinolophus sinicus</em>; RFE – <em>Rhinolophus ferrumequinum</em>; MLY – <em>Megaderma lyra</em>; DRO – <em>Desmodus rotundus</em>; PPA – <em>Pteronotus parnellii</em>; MLU – <em>Myotis lucifugus</em>; MBR – <em>Myotis brandtii</em>; MDA – <em>Myotis davidii</em> ; EFU – <em>Eptesicus fuscus</em>; MNA – <em>Miniopterus natalensis</em>. ACA64 – RF01225; SNORD112 -RF01169; SCARNA7 – RF01295; SNORA61 – RF00420; SNORA48 – RF00554; SNORA38 – RF00428; mir-454 – RF00746; mir-563 – RF01003; mir-662 – RF00983; mir-626 – RF00968; ZNRD1-AS1_2 – RF02219; HOXA11-AS1_1 – RF02137; LINC00901 – RF01884; PVT1_5 – RF02168; HOTTIP_2, 3 – RF02041, RF02042; U11 – RF00548; U4atac – RF00618; Hammerhead_HH10, HH9 – RF02277, RF02275; G-CSF_SLDE – RF00183;</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">All annotations, separated for each ncRNA class and summarized for each bat species, can be found in the Open Science Frame-work (<a href="http://doi.org/10.17605/OSF.IO/4CMDN">doi.org/10.17605/OSF.IO/4CMDN</a>) and in our Electronic Supplement (<a href="https://www.rna.uni-jena.de/supplements/bats">https://www.rna.uni-jena.de/supplements/bats</a>) and are compatible with the genome assembly versions listed in <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Tab. 1</a>. Thus, our annotations together with the bat genome assemblies obtained from the <kbd>NCBI</kbd> can be directly used for subsequent analysis such as differential gene expression detection.</p></div><div id="sec-27" class="subsection"><h3>rRNAs</h3><p id="p-57">We detected 18S and 28S rRNA for the majority of investigated bat species (Tab. S3). The varying number of rRNAs is in line with all currently available metazoan genomes, lacking the correct composition of rRNAs due to misassemblies. However, the number of 5.8S rRNA varies a lot between 17,919 for <em>P. vampyrus</em> and 51 for <em>M. natalensis</em> (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Tab. 5</a> and Tab. S3). Interestingly, 4 of 5 <em>Pteropodidae</em> show a higher number of 5.8S rRNAs compared to the other species (~50–100 fold). Only the <em>P. alecto</em> assembly is in line with the other bat assemblies in regard to the amount of 5.8S rRNA copies.</p></div><div id="sec-28" class="subsection"><h3>tRNAs</h3><p id="p-58">For all bat species we observed various numbers of tRNAs (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Tab. 5</a> and Tab. S4). We could detect full sets of tRNAs for <em>E. fuscus</em> and <em>M. davidii</em>, whereas between one and four tRNAs are missing from the assemblies of <em>H. armiger</em>, <em>M. brandtii</em>, <em>M. lucifugus</em>, <em>M. lyra</em>, <em>M. natalensis</em>, <em>P. parnellii</em>, <em>R. ferrumequinum</em>, <em>R. sinicus</em>, and <em>D. rotundus</em> and between nine and twelve are missing for <em>E. helvum</em>, <em>P. alecto</em>, <em>P. vampyrus</em>, <em>R. aegyptiacus</em>, and <em>E. spelaea</em> (Tab. S4). Interestingly, we identified a large number of tRNAs (18,235) for <em>M. lyra</em> in comparison to all other bat species (Tab. S4), likely a result of the low assembly quality (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>, Tab. S1). The lowest amount of tRNAs was annotated for <em>P. parnellii</em> and <em>D. rotundus</em> with only 1,059 and 1,041 copies, respectively. In all other bat genomes we found between 1,735 and 4,657 tRNA genes (Tab. S4).</p><p id="p-59">The tRNA encoding for valine (Val) with the anticodon structure TAC had high copy numbers (over 1,000) in all genome assemblies of the <em>Pteropodidae</em> family (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Tab. 5</a>). Similar copy numbers were achieved by the <em>R. ferrumequinum</em> and <em>R. sinicus</em> assemblies regarding the tRNA encoding for isoleucine (Ile) with the anticodon AAT (Tab. S4). For tRNA(Ile) and the anticodon GAT we also observed high copy numbers in <em>H. armiger</em>. Interestingly, all species with high tRNA(Val) and tRNA(Ile) copy numbers had rather low counts (between 0 and 28) of tRNA(SeC) with the anticodon TCA, while this tRNA was found with higher copy numbers in <em>P. parnelli</em> (68) and in <em>D. rotundus</em> (145) and with even higher counts (between 315–498) in all other bat species (Tab. S4). However, high copy numbers might be also occur due to assembly quality and false positive predictions of <kbd>tRNAscan-SE</kbd>.</p></div><div id="sec-29" class="subsection"><h3>snoRNAs</h3><p id="p-60">In Tab. S5 we list all detected snoRNAs, divided into box C/D and box H/ACA types. Overall, we found 162 snoRNA families within the investigated bat species, comprising 88 box C/D, 61 box H/ACA, and 13 unclassified snoRNAs.</p><p id="p-61">Many snoRNAs were found with exactly one copy present in each bat genome assembly (e.g. SCARNA7), whereas others were found in multiple copies for each bat species (e.g. SNORD112) or completely absent from certain bat families (e.g. ACA64), see <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Tab. 5</a>. Exactly one copy of the small nucleolar RNA ACA64 was found within the genomes of the <em>Pteropodidae</em> family and multiple copies for <em>D. rotundus</em>, <em>P. parnellii</em>, <em>M. natalensis</em> and members of the <em>Vespertilionidae</em> family, however, this snoRNA seems to be completely absent from bat species of the <em>Megadermatidae</em> and <em>Rhinolophidae</em> families (<a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Tab. 5</a>). The H/ACA box snoRNA ACA64 is predicted to guide the pseudo-uridylation of 28S rRNA U4331<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>. Interestingly and as another example, SNORA48 was found in higher copy numbers in the genomes of the <em>Vespertilionidae</em> family. Among others, this H/ACA box snoRNA was described to be commonly altered in human disease<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>.</p></div><div id="sec-30" class="subsection"><h3>miRNAs</h3><p id="p-62">Over all bat assemblies, we detected 193 miRNA families based on <kbd>Rfam</kbd> alignments (Tab. S6) and predicted between 349 (<em>E. helvum</em>) and 2,464 (<em>M. davidii</em>) miRNAs based on the 18 combined small RNA-Seq data sets (<em>Weber-2019</em>) using <kbd>miRDeep2</kbd><sup><a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a></sup> (Tab. S7). The higher number of miRNAs predicted for <em>Myotis</em> species can be explained because the small RNA-Seq data set is derived from a <em>Myotis daubentonii</em> cell line.</p><p id="p-63">Similar to other ncRNA classes, we observe various differences in miRNA copy numbers between the bat families. For example, mir-454 and mir-563 are absent in all <em>Vespertilionidae</em> and <em>M. natalensis</em> (<a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Tab. 5</a>), whereas mir-1912 is present in all <em>Yangochiroptera</em> (except <em>D. rotundus</em>) but absent from all <em>Pteropodidae</em> (Tab. S6). The miRNA 541 is absent from all <em>Yangochiroptera</em> except <em>D. rotundus</em>. There are many other examples of absent/present miRNAs in certain bat species/families such as mir-662 (absent from <em>Pteropodidae</em>), mir-767 (absent from <em>Yinpterochiroptera</em>), and mir-626 (absent from <em>Rhinolophidae</em> and <em>Megaderma lyra</em>), see Tab. S6.</p><p id="p-64">With <kbd>miRDeep2</kbd>, we detected hundreds of potential miRNAs for all investigated bat species (Tab. S7). Generally, all 16 bat species can be divided into two groups. For the majority of the bat assemblies (12 out of 16) about 400 miRNAs were predicted. For the other 4 species of the <em>Vespertilionidae</em> family about 5 times as many miRNAs could be found. This is in concordance with the small RNA-Seq data used for the prediction, that was obtained from <em>Myotis daubentonii</em> kidney cells (see Methods). Nevertheless, ~400 conserved miRNAs can be predicted in the evolutionary more distant bat species.</p><p id="p-65">We compared our <kbd>miRDeep2</kbd>-predictions in <em>M. lucifugus</em> and <em>P. alecto</em> with the predictions of two other studies<sup><a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">55</a></sup>. From the 540 published miRNAs based on the transcriptome of <em>M. myotis</em> <sup><a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">8</a></sup> and the 426 published miRNAs based on transcriptomic data of <em>P. alecto</em>, we were able to obtain 490 and 368 miRNAs with positional information using <kbd>BLASTn</kbd> against the <em>M. lucifugus</em> and <em>P. alecto</em> genomes, respectively. From these 490 and 368 miRNAs, our prediction, based on the transcriptome of <em>M. daubentonii</em> (<em>Weber-2019</em>) mapped to <em>M. lucifugus</em> and <em>P. alecto</em>, included 195/490 (39.8 %) and 182/368 (49.5 %) miRNAs.</p></div><div id="sec-31" class="subsection"><h3>IncRNAs</h3><p id="p-66">For the annotation of IncRNAs, we have deliberately chosen a broad <kbd>Blast</kbd>-based approach, using 107,039 transcripts of potential human IncRNAs obtained from the <kbd>LNCipedia</kbd> data base<sup><a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a></sup>.</p><p id="p-67">We have consciously chosen this approach, because IncRNAs have diverse genomic contexts, reveal various functions and act in different biological mechanisms<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>–<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup>. From 107,039 <kbd>LNCipedia</kbd> transcripts we annotated 24,316 genes and 182,451 IncRNA <em>hot spots</em>. We defined regions in a genome as a IncRNA hot spot, if different <kbd>LNCipedia</kbd> transcripts derived from different genes map to the same region (see Methods for detailed description). Overall, we found between 58,425 and 137,161 potential IncRNA genes in <em>M. lucifugus</em> and <em>R. sinicus</em>, respectively, and between 27,149 and 158,135 IncRNA hot spots in <em>M. lucifugus</em> and <em>D. rotundus</em>, respectively. We annotated the previously described IncRNAs Tbx5-as1 and Hottip <sup><a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">33</a></sup> in all bat genomes, except Tbx5-as1 in <em>M. lucifugus</em>, presumably due to the lower genome assembly quality (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Tab. 1</a>).</p></div><div id="sec-32" class="subsection"><h3>Other ncRNA elements</h3><p id="p-68">Based on the <kbd>Rfam</kbd> alignments, we were able to detect 244 other ncRNA families in addition to the rRNAs, tRNAs, snoRNAs, and miRNAs described before (Tab. S8). Overlaps with annotated IncRNAs (Tab. S9) are intentional, because the <kbd>Rfam</kbd> includes only highly structured parts of long ncRNAs.</p><p id="p-69">The highest number of ncRNA copies (951) were detected for the U6 spliceosomal RNA in <em>M. brandtii</em>, already known to have a lot of pseudogenes<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup>. For 50 ncRNAs such as CAESAR (RF00172), G-CSF_SLIDE (RF00183), NRON (RF00636), TUSC7 (RF01879), Xist_exon4 (RF01881), LINC00901 (RF01884), and Hammerhead_HH9 and _HH10 (RF02275, RF02277) we found exactly one copy in each investigated bat genome assembly (Tab. S8). Again, we also observed ncRNA families that are lost for some species or entire families, for example the ribozyme CoTC (RF00621) that seems to be absent from all <em>Vespertilionidae</em> members and <em>M. natalensis</em>.</p></div><div id="sec-33" class="subsection"><h3>Mitochondrial annotation</h3><p id="p-70">For each investigated bat species except <em>E. spelaea</em>, <em>R. sinicus</em>, <em>M. lyra</em>, <em>E.fuscus</em>, and <em>M. natalensis</em> (where no mitochondrial contigs could be identified; <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Tab. 3</a>) mitochondrial protein-coding genes and ncRNAs were annotated (see Methods). In total, 37 mitochondrial genes comprising 22 tRNAs, two rRNAs (12S and 16S) and 13 protein-coding genes were detected for each bat species as known for other metazoans<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup> (Tab. S10). The mitochondrial genome lengths range from 16,343 nt in <em>E. helvum</em> to 17,783 nt in <em>M. davidii</em>. For the five bat species, where the mitochondrial genome could be identified as a part of the <kbd>NCBI</kbd> genome assembly, we appended the mtDNA annotation to the final annotation of ncRNAs.</p></div><div id="sec-34" class="subsection"><h3>Updated annotations provide insights into novel differentially expressed ncRNAs</h3><p id="p-71">Exemplarily, we investigated known and novel differentially expressed (DE) ncRNAs found in the genome of <em>M. lucifugus</em> in more detail. To this end, we used the RNA-Seq data sets <em>Field-2015</em>, <em>Field-2018</em>, <em>Hölzer-2019</em>, and <em>Weber-2019</em> (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Tab. 2</a>) as a basis to identify DE ncRNAs that were newly discovered in this study and were not part of the current <kbd>NCBI</kbd> or <kbd>Ensembl</kbd> genome annotations for this bat species. More detailed DE results can be found in Files S2.</p><p id="p-72">We filtered for novel <em>M. lucifugus</em> DE ncRNAs by a) an absolute log<sub>2</sub> fold change (fc) &gt; 1, b) an adjusted p-value &lt; 0.05, and c) a TPM &gt; 10. We further manually investigated the expression patterns with the <kbd>IGV</kbd><sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup> and discarded DE ncRNAs overlapping with the current <kbd>NCBI</kbd> (Myoluc2.0 annotation release 102) or <kbd>Ensembl</kbd> (Myoluc2.0.96) annotations.</p><p id="p-73">Based on the small RNA-Seq comparison of mock and virus-infected (Clone 13) samples 24 h post infection (<em>Weber-2019</em>), we found several miRNAs (<kbd>Rfam-</kbd> and <kbd>mirDeep2</kbd>-based) and snoRNAs to be differentially expressed (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a> and details in Files S2.13). In general, replicates of virus-infected and IFN-treated samples 24 h post infection tend to cluster together only based on the expression profiles of small ncRNAs (mainly miRNAs) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4A</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">B</a>). Most differences can be observed between the 24 h virus-infected and all other samples, that seem to show no clearly distinguishable expression pattern. Interestingly, at 6 h post infection, we see replicates clustering together regardless of their treatment (mock, IFN, Clone 13). Thus, after only 6 h, few miRNAs are differentially expressed and therefore the samples of each replicate (mock, IFN, Clone 13) cluster togehter, because they have the same passaging history but the passaging history in between the replicates differ<sup><a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">36</a></sup>. After 24 h, more and more miRNAs are significantly differentially expressed and the samples can be better distinguished based on their treatment (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4A</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">B</a>). We observed that, in general, miRNAs tend to be down-regulated (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4B</a>; upper half), while snoRNAs tend to be up-regulated (lower half) after 24 h of Clone 13 infection compared to mock. For example, we found a novel miRNA (MLUGD00000002094 in our annotation; predicted by <kbd>mirDeep2</kbd>; Tab. S7) located in an intron of the protein-coding gene <em>SEMA3G</em>, significantly down-regulated (log<sub>2</sub> fc=−2.56) during Clone 13 infection (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4C</a> and <a id="xref-fig-4-9" class="xref-fig" href="#F4">D</a>). Based on Rfam alignments we further found a histone 3’-UTR stem-loop (RF00032), an RNA element involved in nucleocytoplasmic transport of the histone mRNAs, significantly down-regulated during infection.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The number of already annotated ncRNAs (NCBI) and newly annotated ncRNAs in this study (marked by an asterisk) for each bat species. Due to the general detection approach (see Methods) and their high number IncRNAs are omitted from this figure. Details about IncRNAs can be found in Tab. S9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1220078789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The number of already annotated ncRNAs (&lt;kbd&gt;NCBI&lt;/kbd&gt;) and newly annotated ncRNAs in this study (marked by an asterisk) for each bat species. Due to the general detection approach (see Methods) and their high number IncRNAs are omitted from this figure. Details about IncRNAs can be found in Tab. S9.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F3.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F3.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-74" class="first-child">The number of already annotated ncRNAs (<kbd>NCBI</kbd>) and newly annotated ncRNAs in this study (marked by an asterisk) for each bat species. Due to the general detection approach (see Methods) and their high number IncRNAs are omitted from this figure. Details about IncRNAs can be found in Tab. S9.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results shown here are based on expression patterns obtained from the small RNA-Seq samples (Weber-2019) mapped to the M. lucifugus reference genome (NCBI: GCA_000147115). (A) A PCA performed by PCAGO 70 using the 150 most highly expressed ncRNAs (75 miRNAs, 56 snoRNAs, 10 tRNAs, 9 other small ncRNAs; discarding rRNAs) as input. As also shown in the heat map (B), replicates tend to cluster on the basis of their passaging history, except for virus- and interferon-treated (IFN) samples 24 h post infection. (B) A heat map and hirarchical clustering for the same 150 most highly expressed ncRNAs that were used for PCA (A). The clustering was performed on DESeq2-normalized read counts and scaled on rows to visualize changes in expression on ncRNA level. Most differences can be observed between the Clone 13-infected samples 24 h post infection and all other conditions. Due to minor changes in miRNA expression levels, the other samples tend to cluster on the basis of their passaging history and not on the basis of their treatment. In general, miRNAs tend to be down-regulated (upper half), while snoRNAs tend to be up-regulated (lower half) after 24 h of Clone 13 infection. (C) Selected expression box plots of four highly expressed ncRNAs. Shown are DESeq2-normalized expression counts. The miRNA mir-1307 and a novel mirDeep2-predicted miRNA are significantly (adjusted p-value &lt; 0.05) down-regulated at 24 h post virus infection. The C/D box snoRNA SNORD62 and a Y RNA are up-regulated. (D) IGV-derived expression patterns and raw read counts of the four selected ncRNAs obtained from one replicate of the mock and Clone 13 24 h conditions. Blue bars indicate the annotation and strandness." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1220078789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results shown here are based on expression patterns obtained from the small RNA-Seq samples (&lt;em&gt;Weber-2019&lt;/em&gt;) mapped to the &lt;em&gt;M. lucifugus&lt;/em&gt; reference genome (NCBI: GCA_000147115). &lt;strong&gt;(A)&lt;/strong&gt; A PCA performed by &lt;kbd&gt;PCAGO&lt;/kbd&gt; &lt;sup&gt;70&lt;/sup&gt; using the 150 most highly expressed ncRNAs (75 miRNAs, 56 snoRNAs, 10 tRNAs, 9 other small ncRNAs; discarding rRNAs) as input. As also shown in the heat map (B), replicates tend to cluster on the basis of their passaging history, except for virus- and interferon-treated (IFN) samples 24 h post infection. &lt;strong&gt;(B)&lt;/strong&gt; A heat map and hirarchical clustering for the same 150 most highly expressed ncRNAs that were used for PCA (A). The clustering was performed on &lt;kbd&gt;DESeq2&lt;/kbd&gt;-normalized read counts and scaled on rows to visualize changes in expression on ncRNA level. Most differences can be observed between the Clone 13-infected samples 24 h post infection and all other conditions. Due to minor changes in miRNA expression levels, the other samples tend to cluster on the basis of their passaging history and not on the basis of their treatment. In general, miRNAs tend to be down-regulated (upper half), while snoRNAs tend to be up-regulated (lower half) after 24 h of Clone 13 infection. &lt;strong&gt;(C)&lt;/strong&gt; Selected expression box plots of four highly expressed ncRNAs. Shown are &lt;kbd&gt;DESeq2&lt;/kbd&gt;-normalized expression counts. The miRNA mir-1307 and a novel &lt;kbd&gt;mirDeep2&lt;/kbd&gt;-predicted miRNA are significantly (adjusted p-value &lt; 0.05) down-regulated at 24 h post virus infection. The C/D box snoRNA SNORD62 and a Y RNA are up-regulated. &lt;strong&gt;(D)&lt;/strong&gt; &lt;kbd&gt;IGV&lt;/kbd&gt;-derived expression patterns and raw read counts of the four selected ncRNAs obtained from one replicate of the mock and Clone 13 24 h conditions. Blue bars indicate the annotation and strandness.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F4.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F4.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/25/738526/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-75" class="first-child">Results shown here are based on expression patterns obtained from the small RNA-Seq samples (<em>Weber-2019</em>) mapped to the <em>M. lucifugus</em> reference genome (NCBI: GCA_000147115). <strong>(A)</strong> A PCA performed by <kbd>PCAGO</kbd> <sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup> using the 150 most highly expressed ncRNAs (75 miRNAs, 56 snoRNAs, 10 tRNAs, 9 other small ncRNAs; discarding rRNAs) as input. As also shown in the heat map (B), replicates tend to cluster on the basis of their passaging history, except for virus- and interferon-treated (IFN) samples 24 h post infection. <strong>(B)</strong> A heat map and hirarchical clustering for the same 150 most highly expressed ncRNAs that were used for PCA (A). The clustering was performed on <kbd>DESeq2</kbd>-normalized read counts and scaled on rows to visualize changes in expression on ncRNA level. Most differences can be observed between the Clone 13-infected samples 24 h post infection and all other conditions. Due to minor changes in miRNA expression levels, the other samples tend to cluster on the basis of their passaging history and not on the basis of their treatment. In general, miRNAs tend to be down-regulated (upper half), while snoRNAs tend to be up-regulated (lower half) after 24 h of Clone 13 infection. <strong>(C)</strong> Selected expression box plots of four highly expressed ncRNAs. Shown are <kbd>DESeq2</kbd>-normalized expression counts. The miRNA mir-1307 and a novel <kbd>mirDeep2</kbd>-predicted miRNA are significantly (adjusted p-value &lt; 0.05) down-regulated at 24 h post virus infection. The C/D box snoRNA SNORD62 and a Y RNA are up-regulated. <strong>(D)</strong> <kbd>IGV</kbd>-derived expression patterns and raw read counts of the four selected ncRNAs obtained from one replicate of the mock and Clone 13 24 h conditions. Blue bars indicate the annotation and strandness.</p><div class="sb-div caption-clear"></div></div></div><p id="p-76">For the same comparison of mock and virus-infected samples at 24 h, the rRNA-depleted data set (<em>Hölzer-2019</em>) revealed several DE IncRNAs. For example, we found a IncRNA potentially transcribed in an intron of <em>MX1</em> (MLUGL00000039178) up-regulated (log<sub>2</sub> fc=3.36) during infection. Another IncRNA (MLUGL00000087396), we found potentially transcribed as a part of two exons of the <em>PLAT</em> protein-coding gene and down-regulated (log<sub>2</sub> fc= −1.67) during viral infection (Files S2.11).</p><p id="p-77">Interestingly, based on the <em>Field-2015</em> RNA-Seq data, we found two internal ribosomal entry site (IRES) in the genes <em>VEGFA</em> and <em>ODC</em> with <kbd>Rfam</kbd> IDs RF00461 and RF02535, respectively, to be two-fold up-regulated during <em>P. destructans</em> infection.</p></div></div><div class="section" id="sec-35"><h2 class="">DISCUSSION</h2><p id="p-78">In this study, we comprehensively annotated ncRNAs in 16 readily available bat genomes obtained from the <kbd>NCBI</kbd> database (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Tab. 1</a>). We provide novel annotations in the common <kbd>GTF</kbd> format, following a hierarchical structure of <em>gene</em>, <em>transcript</em>, and <em>exon</em> features to allow direct integration of our annotations into already available ones (Tab. S3–S10). Finally, we provide for each bat genome assembly an extended annotation file merged with the protein- and non-coding gene annotations that were already available by the <kbd>NCBI</kbd> database (Files. S1; leaving out potential IncRNAs that can be downloaded separately Tab. S9). We used six RNA-Seq data sets derived from the transcriptomic sequencing of four bat species (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Tab. 2</a>) to calculate normalized expression values for our newly annotated ncRNAs and exemplarily show significantly differential expressed ncRNAs (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4</a>), that were never before described on such a large scale for any bat species.</p><p id="p-79">In addition to the evolutionarily explainable differences in the pure existence and the amount of annotated ncRNAs in bats, we have observed that the assembly quality can also influence the annotation results. While the effects on the annotation of short ncRNAs seem to be small (with some exceptions), the number of identified lncRNAs and lncRNA hot spots increases with increasing assembly quality (e.g. with a higher N50; see Fig. S1). However, this observation may be true for our data and analyses, but it also depends strongly on the annotation method used.</p><p id="p-80">Recently, the Bat1K project (<a href="http://bat1k.ucd.ie/">http://bat1k.ucd.ie/</a>) was announced as a global effort to sequence, assemble and annotate high-quality genomes of all living bat species<sup><a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">2</a></sup>. We aim to extend our annotation of ncRNAs regulary and whenever new bat genomes become publicly available.</p><p id="p-81">In mid January 2019, 16 new low-coverage bat genomes of nine families were submitted by the Broad Institute to the <kbd>NCBI</kbd> genome database. Unfortunately, our time-consuming and computationally extensive analyses were already completed at this time point. We want to further automate our ncRNA annotation workflow, to easily include these and any new bat (or other mammalian) genomes that will be sequenced and assembled in the future.</p><p id="p-82">Our current identification of ncRNAs in bat species will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.</p></div><div class="section" id="sec-36"><h2 class="">DATA AVAILABILITY</h2><p id="p-83">Detailed information about the bat genomes used in this study, their assembly quality, and all ncRNA candidates (in <kbd>FASTA</kbd>, <kbd>STK</kbd>, and <kbd>GTF</kbd> format) can be found in the Electronic Supplement (rna.uni-jena.de/supplements/bats) Tables S1-S10. The final extended annotations for each investigated bat species can be found in Tab. S11 and the IncRNA annotations in Files S11. To allow full reproducibility of our study, all final and intermediate data files (such as used genome files and mapping files in <kbd>BAM</kbd> format) were uploaded to the Open Science Framework under accession <a href="http://doi.org/10.17605/OSF.IO/4CMDN">doi.org/10.17605/OSF.IO/4CMDN</a>. Python scripts used to filter and merge our annotations were deposited at GitHub (<a href="http://github.com/rnajena/bats_ncrna">github.com/rnajena/bats_ncrna</a>). The virus-infected and IFN-stimulated small RNA-Seq data of the <em>M. daubentonii</em> kidney cell line was uploaded to GEO (GSE132336).</p></div><div class="section" id="sec-37"><h2 class="">FUNDING</h2><p id="p-85">This work was supported in part by grants from the <em>Deutsche Forschungsgemeinschaft</em> SFB 1021 (B06), SPP 1596 (We 2616/7-1, We 2616/7-2, DR772-10/1, and MA 5082/1-1), MA 5082/9-1, and the LOEWE-Schwerpunkt “Medical RNomics” of the Land Hessen and by the RAPID consortium of the Bundesministerium für Bildung und Forschung (BMBF, grant number 01KI1723E). This study is further part of the Collaborative Research Centre AquaDiva (CRC 1076 AquaDiva) of the Friedrich Schiller University Jena, funded by the Deutsche Forschungsgemeinschaft, as well as the German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig (iDIV FZT 118) of the University of Leipzig, and also supported by the Carl Zeiss Foundation, project “Eine virtuelle Werkstatt für die Digitalisierung in den Wissenschaften” (Durchbrüche).</p><div class="section COI-statement" id="sec-38"><h3>Conflict of interest statement</h3><p id="p-86">None declared.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-84">We thank Ivonne Görlich and Marco Groth from  the Core Facility DNA sequencing of the Leibniz Institute on Aging — Fritz Lipmann Institute in Jena for their help with short-read Illumina sequencing. Martin Hölzer appreciates the support of the Joachim Herz Foundation by the add-on fellowship for inter-disciplinary life science. Kevin Lamkiewicz thanks the Carl Zeiss Foundation for the financial support within the scope of the program line “Breakthroughs”. The authors gratefully acknowledge the help of the developer of the Gorap pipeline, Konstantin Riege.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-5">Added citation of Figure 3 and changed affiliation of Andreas. Final submission.</p></li><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-6">
<a href="https://www.rna.uni-jena.de/supplements/bats">https://www.rna.uni-jena.de/supplements/bats</a>
</p></li><li class="fn-dataset fn-group-external-links" id="fn-4"><p id="p-7">
<a href="https://doi.org/10.17605/OSF.IO/4CMDN">https://doi.org/10.17605/OSF.IO/4CMDN</a>
</p></li><li class="fn-dataset fn-group-external-links" id="fn-5"><p id="p-8">
<a href="https://github.com/rnajena/bats_ncrna">https://github.com/rnajena/bats_ncrna</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.1" data-doi="10.1128/CMR.00017-06"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Calisher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Childs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. E.</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Holmes</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schountz</span></span>. <span class="cit-article-title">Bats: important reservoir hosts of emerging viruses</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">531</span>–<span class="cit-lpage">545</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DCalisher%26rft.auinit1%253DC.%2BH.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DBats%253A%2BImportant%2BReservoir%2BHosts%2Bof%2BEmerging%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00017-06%26rft_id%253Dinfo%253Apmid%252F16847084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjUvNzM4NTI2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.2" data-doi="10.1146/annurev-animal-022516-022811"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Teeling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Vernes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Dávalos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Ray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T. P.</span> <span class="cit-name-surname">Gilbert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Myers</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Consortium</span></span>. <span class="cit-article-title">Bat biology, genomes, and the Bat1K Project: To generate Chromosome-Level genomes for all living bat species</span>. <abbr class="cit-jnl-abbrev">Annu Rev Anim Biosci</abbr>, (<span class="cit-vol">6</span>):<span class="cit-fpage">23</span>–<span class="cit-lpage">46</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BAnim%2BBiosci%26rft.volume%253D6%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-022516-022811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-022516-022811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.3" data-doi="10.1126/science.1108871"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. B.</span> <span class="cit-name-surname">Simmons</span></span>. <span class="cit-article-title">An Eocene big bang for bats</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">307</span>(<span class="cit-issue">5709</span>):<span class="cit-fpage">527</span>–<span class="cit-lpage">528</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSimmons%26rft.auinit1%253DN.%2BB.%26rft.volume%253D307%26rft.issue%253D5709%26rft.spage%253D527%26rft.epage%253D528%26rft.atitle%253DEVOLUTION%253A%2BEnhanced%253A%2BAn%2BEocene%2BBig%2BBang%2Bfor%2BBats%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1108871%26rft_id%253Dinfo%253Apmid%252F15681371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwOS81MjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yNS83Mzg1MjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-738526v3.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Teeling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jones</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Rossiter</span></span>. <span class="cit-chapter-title">Phylogeny, genes, and hearing: implications for the evolution of echolocation in bats</span>. In <span class="cit-source">Bat Bioacoustics</span>, pages <span class="cit-fpage">25</span>–<span class="cit-lpage">54</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gardner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wilson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reeder</span></span>. <span class="cit-article-title">Mammal Species of the World. A Taxonomic and Geographic Reference</span>. <abbr class="cit-jnl-abbrev">Mammal species of the world: a taxonomic and geographic reference</abbr>, <span class="cit-vol">12</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.6" data-doi="10.1126/science.1105113"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Teeling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Springer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Madsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">O’brien</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Murphy</span></span>. <span class="cit-article-title">A molecular phylogeny for bats illuminates biogeography and the fossil record</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">307</span>(<span class="cit-issue">5709</span>):<span class="cit-fpage">580</span>–<span class="cit-lpage">584</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTeeling%26rft.auinit1%253DE.%2BC.%26rft.volume%253D307%26rft.issue%253D5709%26rft.spage%253D580%26rft.epage%253D584%26rft.atitle%253DA%2BMolecular%2BPhylogeny%2Bfor%2BBats%2BIlluminates%2BBiogeography%2Band%2Bthe%2BFossil%2BRecord%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105113%26rft_id%253Dinfo%253Apmid%252F15681385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwOS81ODAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yNS83Mzg1MjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.7" data-doi="10.1371/journal.pbio.2002556"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Prat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Azoulay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Dor</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yovel</span></span>. <span class="cit-article-title">Crowd vocal learning induces vocal dialects in bats: Playback of conspecifics shapes fundamental frequency usage by pups</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e2002556</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D15%26rft.spage%253De2002556%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2002556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2002556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jebb</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Teeling</span></span>. <span class="cit-article-title">Blood miRNomes and transcriptomes reveal novel longevity mechanisms in the long-lived bat, <em>Myotis myotis</em></span>. <abbr class="cit-jnl-abbrev">BMC Genom</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">906</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenom%26rft.volume%253D17%26rft.spage%253D906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[9].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-738526v3.9" data-doi="10.1126/sciadv.aao0926"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Foley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Hughes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Clarke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jebb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. V.</span> <span class="cit-name-surname">Whelan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Petit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Touzalin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Farcy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Growing old, yet staying young: The role of telomeres in bats’ exceptional longevity</span>. <abbr class="cit-jnl-abbrev">Sci Adv</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">eaao0926</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DFoley%26rft.auinit1%253DN.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253Deaao0926%26rft.epage%253Deaao0926%26rft.atitle%253DGrowing%2Bold%252C%2Byet%2Bstaying%2Byoung%253A%2BThe%2Brole%2Bof%2Btelomeres%2Bin%2Bbats%2Bexceptional%2Blongevity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aao0926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNC8yL2VhYW8wOTI2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjUvNzM4NTI2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. V.</span> <span class="cit-name-surname">Whelan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Foley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jebb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Touzalin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Petit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Puechmaille</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Teeling</span></span>. <span class="cit-article-title">Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.11" data-doi="10.1016/j.coviro.2012.11.006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Smith</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">L.-F.</span> <span class="cit-name-surname">Wang</span></span>. <span class="cit-article-title">Bats and their virome: an important source of emerging viruses capable of infecting humans</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">84</span>–<span class="cit-lpage">91</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D3%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2012.11.006%26rft_id%253Dinfo%253Apmid%252F23265969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2012.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23265969&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.12" data-doi="10.1016/j.coviro.2011.10.013"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.-F.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Walker</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Poon</span></span>. <span class="cit-article-title">Mass extinctions, biodiversity and mitochondrial function: are bats “special” as reservoirs for emerging viruses?</span> <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">649</span>–<span class="cit-lpage">657</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BOpin%2BVirol%26rft.aulast%253DWang%26rft.auinit1%253DL.%2BF.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253D649%26rft.epage%253D657%26rft.atitle%253DMass%2Bextinctions%252C%2Bbiodiversity%2Band%2Bmitochondrial%2Bfunction%253A%2Bare%2Bbats%2B%2527special%2527%2Bas%2Breservoirs%2Bfor%2Bemerging%2Bviruses%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2011.10.013%26rft_id%253Dinfo%253Apmid%252F22440923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2011.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22440923&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[13].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-738526v3.13" data-doi="10.1016/j.tim.2014.12.004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Brook</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Dobson</span></span>. <span class="cit-article-title">Bats as “special” reservoirs for emerging zoonotic pathogens</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">172</span>–<span class="cit-lpage">180</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D23%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2014.12.004%26rft_id%253Dinfo%253Apmid%252F25572882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2014.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572882&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Anthony</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Johnson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Greig</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kramer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Che</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wells</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Hicks</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. O.</span> <span class="cit-name-surname">Joly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Wolfe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Daszak</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Global patterns in coronavirus diversity</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">1</span>), <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.15" data-doi="10.3201/eid2005.130539"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">O’shea</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Cryan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Cunningham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Fooks</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Hayman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Luis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Peel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Plowright</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Wood</span></span>. <span class="cit-article-title">Bat flight and zoonotic viruses</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">741</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2005.130539%26rft_id%253Dinfo%253Apmid%252F24750692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2005.130539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24750692&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.16" data-doi="10.1038/srep34589"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hölzer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Krähling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Amman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Barth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Bernhart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. A. O.</span> <span class="cit-name-surname">Carmelo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Collatz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Doose</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eggenhofer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ewald</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fallmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Feldhahn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fricke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gebauer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Gruber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hufsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Indrischek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kanton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Linde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Mostajo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ochsenreiter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Riege</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rivarola-Duarte</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Sahyoun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Saunders</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Seemann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tanzer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vogel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wehner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Wolfinger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Backofen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gorodkin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Grosse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hofacker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kaleta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Stadler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Becker</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marz</span></span>. <span class="cit-article-title">Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, (<span class="cit-vol">6</span>):<span class="cit-fpage">34589</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D34589%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep34589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep34589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Papenfuss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-P.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tachedjian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Crameri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cowled</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Janardhana</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. E.</span> <span class="cit-name-surname">Field</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.-F.</span> <span class="cit-name-surname">Wang</span></span>. <span class="cit-article-title">The immune gene repertoire of an important viral reservoir, the Australian black flying fox</span>. <abbr class="cit-jnl-abbrev">BMC Genom</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">261</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenom%26rft.volume%253D13%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.18" data-doi="10.1093/embo-reports/kve230"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mattick</span></span>. <span class="cit-article-title">Non-coding RNas: the architects of eukaryotic complexity</span>. <abbr class="cit-jnl-abbrev">EMBO Rep</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">986</span>–<span class="cit-lpage">991</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft.stitle%253DEMBO%2BRep.%26rft.aulast%253DMattick%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D986%26rft.epage%253D991%26rft.atitle%253DNon-coding%2BRNAs%253A%2Bthe%2Barchitects%2Bof%2Beukaryotic%2Bcomplexity%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fembo-reports%252Fkve230%26rft_id%253Dinfo%253Apmid%252F11713189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiMi8xMS85ODYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yNS83Mzg1MjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.19" data-doi="10.1093/nar/gks228"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Isakov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ronen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kovarsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gabay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Gan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Modai</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Shomron</span></span>. <span class="cit-article-title">Novel insight into the non-coding repertoire through deep sequencing analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">40</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">e86</span>–<span class="cit-lpage">e86</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks228%26rft_id%253Dinfo%253Apmid%252F22406831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22406831&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.20" data-doi="10.1038/35080529"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Stein</span></span>. <span class="cit-article-title">Genome annotation: from sequence to biology</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">493</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DStein%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D493%26rft.epage%253D503%26rft.atitle%253DGenome%2Bannotation%253A%2Bfrom%2Bsequence%2Bto%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35080529%26rft_id%253Dinfo%253Apmid%252F11433356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35080529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11433356&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169681600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ekblom</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galindo</span></span>. <span class="cit-article-title">Applications of next generation sequencing in molecular ecology of non-model organisms</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">107</span>(<span class="cit-issue">1</span>):1, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span>. <span class="cit-article-title">Next-generation genome annotation: we still struggle to get it right</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">92</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1474-760X</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D20%26rft.spage%253D92%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Haynes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tomczak</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Khatri</span></span>. <span class="cit-article-title">Gene annotation bias impedes biomedical research</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1362</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D1362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.24" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gurevich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Saveliev</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vyahhi</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tesler</span></span>. <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.25" data-doi="10.1038/nature10530"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lindblad-Toh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Garber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Zuk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Parker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Washietl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kheradpour</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ernst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jordan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mauceli</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A high-resolution map of human evolutionary constraint using 29 mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">478</span>(<span class="cit-issue">7370</span>):<span class="cit-fpage">476</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLindblad-Toh%26rft.auinit1%253DK.%26rft.volume%253D478%26rft.issue%253D7370%26rft.spage%253D476%26rft.epage%253D482%26rft.atitle%253DA%2Bhigh-resolution%2Bmap%2Bof%2Bhuman%2Bevolutionary%2Bconstraint%2Busing%2B29%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10530%26rft_id%253Dinfo%253Apmid%252F21993624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21993624&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296194200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.26" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cowled</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Bishop-Lilly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Fang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Wynne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xiong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">339</span>(<span class="cit-issue">6118</span>):<span class="cit-fpage">456</span>–<span class="cit-lpage">460</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yNS83Mzg1MjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Pavlovich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Lovett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Koroleva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Guito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Arnold</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Nagle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kulcsar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Thibaud-Nissen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Hume</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The egyptian rousette genome reveals unexpected features of bat antiviral immunity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">173</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1098</span>–<span class="cit-lpage">1110</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.28" data-doi="10.1038/nature12511"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tsagkogeorga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Cotton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Provero</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Stupka</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Rossiter</span></span>. <span class="cit-article-title">Genome-wide signatures of convergent evolution in echolocating mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">502</span>(<span class="cit-issue">7470</span>):<span class="cit-fpage">228</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12511%26rft_id%253Dinfo%253Apmid%252F24005325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24005325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325436100049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Ng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. T.</span> <span class="cit-name-surname">Chionh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. N.</span> <span class="cit-name-surname">Chia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Mendenhall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. P.-H.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Irving</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.-F.</span> <span class="cit-name-surname">Wang</span></span>. <span class="cit-article-title">Exploring the genome and transcriptome of the cave nectar bat <em>Eonycteris spelaea</em> with PacBio long-read sequencing</span>. <abbr class="cit-jnl-abbrev">GigaScience</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">giy116</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D7%26rft.spage%253Dgiy116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hua</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-H.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-article-title">The genomes of two bat species with long constant frequency echolocation calls</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, page <span class="cit-fpage">msw231</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[31].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-738526v3.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. L. Z.</span> <span class="cit-name-surname">Mendoza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xiong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Escalera-Zamudio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Runge</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Thézé</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Streicker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Frank</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Loza-Rubio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Ryder</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Hologenomic adaptations underlying the evolution of sanguivory in the common vampire bat</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">659</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BEcol%2BEvol%26rft.volume%253D2%26rft.spage%253D659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[32].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-738526v3.32" data-doi="10.1038/ncomms3212"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Seim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Fang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xiong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Lobanov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Turanov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Lenz</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome analysis reveals insights into physiology and longevity of the Brandt’s bat <em>Myotis brandtii</em></span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, (<span class="cit-vol">4</span>):<span class="cit-fpage">2212</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D4%26rft.spage%253D2212%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3212%26rft_id%253Dinfo%253Apmid%252F23962925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23962925&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.33" data-doi="10.1038/ng.3537"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W. L.</span> <span class="cit-name-surname">Eckalbar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Schlebusch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Mason</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Gill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Parker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Booker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nishizaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Muswamba-Nday</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Terhune</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Nevonen</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Transcriptomic and epigenomic characterization of the developing bat wing</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr>, <span class="cit-vol">48</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">528</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D48%26rft.spage%253D528%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3537%26rft_id%253Dinfo%253Apmid%252F27019111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27019111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.34" data-doi="10.1038/nmeth.3317"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Langmead</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span>. <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">357</span>–<span class="cit-lpage">360</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751142&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.35" data-doi="10.1093/bioinformatics/btt656"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. K.</span> <span class="cit-name-surname">Smyth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shi</span></span>. <span class="cit-article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">923</span>–<span class="cit-lpage">930</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt656%26rft_id%253Dinfo%253Apmid%252F24227677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24227677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.36"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hölzer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schoen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wulle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marz</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Weber</span></span>. <span class="cit-article-title">Virus- and interferon alpha-induced transcriptomes of cells from the microbat <em>Myotis daubentonii</em></span>. <abbr class="cit-jnl-abbrev">iScience</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.37" data-doi="10.1371/journal.ppat.1005168"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Johnson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Lilley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Reeder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Rogers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Behr</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Reeder</span></span>. <span class="cit-article-title">The white-nose syndrome transcriptome: activation of anti-fungal host responses in wing tissue of hibernating little brown myotis</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1005168</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D11%26rft.spage%253De1005168%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005168%26rft_id%253Dinfo%253Apmid%252F26426272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426272&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Sewall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Prokkola</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. G.</span> <span class="cit-name-surname">Turner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Gagnon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Lilley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Paul White</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Johnson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Hauer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Reeder</span></span>. <span class="cit-article-title">Effect of torpor on host transcriptomic responses to a fungal pathogen in hibernating bats</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">3727</span>–<span class="cit-lpage">3743</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D27%26rft.spage%253D3727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.39" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bolger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span>. <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">30</span>):<span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>, <span class="cit-pub-date">2014</span>. ISSN <span class="cit-issn">1367-4811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.40" data-doi="10.14806.17.1.200"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Martin</span></span>. <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <abbr class="cit-jnl-abbrev">EMB-net.journal</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">10</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMB-net.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806.17.1.200%26rft_id%253Dinfo%253Apmid%252F22955987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955987&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.41" data-doi="10.1093/bioinformatics/btr026"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Schmieder</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Edwards</span></span>. <span class="cit-article-title">Quality control and preprocessing of metagenomic datasets</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">27</span>):<span class="cit-fpage">863</span>–<span class="cit-lpage">864</span>, <span class="cit-pub-date">2011</span>. ISSN <span class="cit-issn">1367-4811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr026%26rft_id%253Dinfo%253Apmid%252F21278185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21278185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.42" data-doi="10.1093/nar/gkr688"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Friedländer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Mackowiak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rajewsky</span></span>. <span class="cit-article-title">miRDeep2 accurately identifies known and hundreds of novel microRNA genes in seven animal clades</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, (<span class="cit-vol">40</span>):<span class="cit-fpage">37</span>–<span class="cit-lpage">52</span>, <span class="cit-month">January</span> <span class="cit-pub-date">2012</span>. ISSN <span class="cit-issn">1362-4962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr688%26rft_id%253Dinfo%253Apmid%252F21911355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21911355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298733500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.43" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">Love</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anders</span></span>. <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">550</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.44" data-doi="10.1093/bioinformatics/btp692"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ruotti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Stewart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Thomson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. N.</span> <span class="cit-name-surname">Dewey</span></span>. <span class="cit-article-title">RNA-Seq gene expression estimation with read mapping uncertainty</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">26</span>): <span class="cit-fpage">493</span>–<span class="cit-lpage">500</span>, <span class="cit-pub-date">2010</span>. ISSN <span class="cit-issn">1367-4811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp692%26rft_id%253Dinfo%253Apmid%252F20022975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20022975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274484300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.45" data-doi="10.1093/nar/gkx1098"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Zerbino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Achuthan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Akanni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Amode</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Barrell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bhai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Billis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cummins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Girón</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ensembl 2018</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">46</span> (<span class="cit-issue">D1</span>):<span class="cit-fpage">D754</span>–<span class="cit-lpage">D761</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD754%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.46" data-doi="10.1093/nar/gkg006"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Griffiths-Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marshall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Khanna</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>. <span class="cit-article-title">Rfam: an RNA family database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, (<span class="cit-vol">31</span>):<span class="cit-fpage">439</span>–<span class="cit-lpage">441</span>, <span class="cit-month">January</span> <span class="cit-pub-date">2003</span>. ISSN <span class="cit-issn">1362-4962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGriffiths-Jones%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D439%26rft.epage%253D441%26rft.atitle%253DRfam%253A%2Ban%2BRNA%2Bfamily%2Bdatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg006%26rft_id%253Dinfo%253Apmid%252F12520045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12520045&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181079700109&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.47" data-doi="10.1093/nar/gkx1038"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Kalvari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Argasinska</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Quinones-Olvera</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Nawrocki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rivas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Finn</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Petrov</span></span>. <span class="cit-article-title">Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">46</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D335</span>–<span class="cit-lpage">D342</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD335%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.48"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Kalvari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Nawrocki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Argasinska</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Quinones-Olvera</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Finn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Petrov</span></span>. <span class="cit-article-title">Non-coding RNA analysis using the Rfam database</span>. <abbr class="cit-jnl-abbrev">Curr Protoc Bioinformatics</abbr>, page <span class="cit-fpage">e51</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.49" data-doi="10.1093/bioinformatics/btp157"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Nawrocki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Kolbe</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>. <span class="cit-article-title">Infernal 1.0: inference of RNA alignments</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">25</span>): <span class="cit-fpage">1335</span>–<span class="cit-lpage">1337</span>, <span class="cit-pub-date">2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp157%26rft_id%253Dinfo%253Apmid%252F19307242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19307242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265950600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.50" data-doi="10.1093/bioinformatics/btt509"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Nawrocki</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>. <span class="cit-article-title">Infernal 1.1: 100-fold faster RNA homology searches</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">29</span>): <span class="cit-fpage">2933</span>–<span class="cit-lpage">2935</span>, <span class="cit-month">November</span> <span class="cit-pub-date">2013</span>. ISSN <span class="cit-issn">1367-4811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt509%26rft_id%253Dinfo%253Apmid%252F24008419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24008419&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326643600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.51" data-doi="10.1093/bioinformatics/bth489"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Griffiths-Jones</span></span>. <span class="cit-article-title">RALEE—RNA alignment editor in Emacs</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (<span class="cit-vol">21</span>):<span class="cit-fpage">257</span>–<span class="cit-lpage">259</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth489%26rft_id%253Dinfo%253Apmid%252F15377506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15377506&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226308500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.52" data-doi="10.1093/nar/gkm160"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lagesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hallin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Rødland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-H.</span> <span class="cit-name-surname">Staerfeldt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Rognes</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Ussery</span></span>. <span class="cit-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, (<span class="cit-vol">35</span>):<span class="cit-fpage">3100</span>–<span class="cit-lpage">3108</span>, <span class="cit-pub-date">2007</span>. ISSN <span class="cit-issn">1362-4962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm160%26rft_id%253Dinfo%253Apmid%252F17452365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17452365&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247350800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.53" data-doi="10.1093/nar/25.5.955"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Lowe</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>. <span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, (<span class="cit-vol">25</span>):<span class="cit-fpage">955</span>–<span class="cit-lpage">964</span>, <span class="cit-pub-date">1997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.5.955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9023104&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kehr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bartschat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tafer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Stadler</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hertel</span></span>. <span class="cit-article-title">Matching of Soulmates: coevolution of snoRNAs and their targets</span>. <abbr class="cit-jnl-abbrev">Molec Biol Evol</abbr>, <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">455</span>–<span class="cit-lpage">467</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolec%2BBiol%2BEvol%26rft.volume%253D31%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.55"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cowled</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Stewart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. A.</span> <span class="cit-name-surname">Likic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Friedländer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tachedjian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Jenkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Tizard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cottee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Marsh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterisation of novel microRNAs in the Black flying fox (<em>Pteropus alecto</em>) by deep sequencing</span>. <abbr class="cit-jnl-abbrev">BMC Genom</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">682</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenom%26rft.volume%253D15%26rft.spage%253D682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.56"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.-J.</span> <span class="cit-name-surname">Volders</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Anckaert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Verheggen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Nuytens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Martens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Mestdagh</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Vandesompele</span></span>. <span class="cit-article-title">LNCipedia 5: towards a reference set of human long non-coding RNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, (<span class="cit-vol">47</span>):<span class="cit-fpage">D135</span>–<span class="cit-lpage">D139</span>, <span class="cit-month">January</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1362-4962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.57" data-doi="10.1016/j.ympev.2012.08.023"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bernt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Donath</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jühling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Externbrink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Florentz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Fritzsch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pütz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Middendorf</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Stadler</span></span>. <span class="cit-article-title">MITOS: Improved <em>de novo</em> metazoan mitochondrial genome annotation</span>. <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr>, <span class="cit-vol">69</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPhylogenet%2BEvol%26rft.volume%253D69%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.08.023%26rft_id%253Dinfo%253Apmid%252F22982435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982435&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.58"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Pervez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Babar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Naveed</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shoaib</span></span>. <span class="cit-article-title">A comprehensive study of de novo genome assemblers: current challenges and future prospective</span>. <abbr class="cit-jnl-abbrev">Evol Bioinform</abbr>, (<span class="cit-vol">14</span>):<span class="cit-fpage">1176934318758650</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BBioinform%26rft.volume%253D14%26rft.spage%253D1176934318758650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.59" data-doi="10.1093/bioinformatics/btw379"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mikheenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Valin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Prjibelski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Saveliev</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gurevich</span></span>. <span class="cit-article-title">Icarus: visualizer for <em>de novo</em> assembly evaluation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">3321</span>–<span class="cit-lpage">3323</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw379%26rft_id%253Dinfo%253Apmid%252F27378299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27378299&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[60].</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.60"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">O’Leary</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Wright</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Brister</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ciufo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Haddad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">McVeigh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rajput</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Robbertse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Smith-White</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ako-Adjei</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Reference sequence (RefSeq) database at [ncbi: current status, taxonomic expansion, and functional annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D733</span>–<span class="cit-lpage">D745</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D44%26rft.spage%253DD733%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26553804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[61].</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.61" data-doi="10.1371/journal.pone.0048472"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. I.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Srivastava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.-C.</span> <span class="cit-name-surname">Chou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hawkinson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Glenn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Adams</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schountz</span></span>. <span class="cit-article-title">Transcriptome sequencing and annotation for the Jamaican fruit bat (<em>Artibeus jamaicensis</em>)</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">e48472</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DShaw%26rft.auinit1%253DT.%2BI.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48472%26rft.epage%253De48472%26rft.atitle%253DTranscriptome%2BSequencing%2Band%2BAnnotation%2Bfor%2Bthe%2BJamaican%2BFruit%2BBat%2B%2528Artibeus%2Bjamaicensis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048472%26rft_id%253Dinfo%253Apmid%252F23166587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23166587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[62].</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.62" data-doi="10.1371/journal.pone.0135064"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Geiser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-article-title">Down but not out: the role of microRNAs in hibernating bats</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">e0135064</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D10%26rft.spage%253De0135064%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0135064%26rft_id%253Dinfo%253Apmid%252F26244645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0135064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26244645&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[63].</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.63" data-doi="10.1261/rna.2210406"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Schattner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Barberan-Soler</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Lowe</span></span>. <span class="cit-article-title">A computational screen for mammalian pseudouridylation guide H/ACA RNAs</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">15</span>–<span class="cit-lpage">25</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DSCHATTNER%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D25%26rft.atitle%253DA%2Bcomputational%2Bscreen%2Bfor%2Bmammalian%2Bpseudouridylation%2Bguide%2BH%252FACA%2BRNAs%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2210406%26rft_id%253Dinfo%253Apmid%252F16373490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjc6IjEyLzEvMTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yNS83Mzg1MjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[64].</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.64"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">McMahon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Contreras</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ruggero</span></span>. <span class="cit-article-title">Small RNAs with big implications: new insights into H/ACA snoRNA function and their role in human disease</span>. <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews: RNA</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">173</span>–<span class="cit-lpage">189</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%253A%2BRNA%26rft.volume%253D6%26rft.spage%253D173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[65].</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.65" data-doi="10.1534/genetics.112.146704"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. T. Y.</span> <span class="cit-name-surname">Kung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Colognori</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Lee</span></span>. <span class="cit-article-title">Long noncoding RNAs: past, present, and future</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, (<span class="cit-vol">193</span>):<span class="cit-fpage">651</span>–<span class="cit-lpage">669</span>, <span class="cit-month">March</span> <span class="cit-pub-date">2013</span>. ISSN <span class="cit-issn">1943-2631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKung%26rft.auinit1%253DJ.%2BT.%2BY.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D651%26rft.epage%253D669%26rft.atitle%253DLong%2BNoncoding%2BRNAs%253A%2BPast%252C%2BPresent%252C%2Band%2BFuture%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146704%26rft_id%253Dinfo%253Apmid%252F23463798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvNjUxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjUvNzM4NTI2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[66].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-738526v3.66"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.-W.</span> <span class="cit-name-surname">Nam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.-W.</span> <span class="cit-name-surname">Choi</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.-H.</span> <span class="cit-name-surname">You</span></span>. <span class="cit-article-title">Incredible RNA: Dual Functions of Coding and Noncoding</span>. <abbr class="cit-jnl-abbrev">Mol Cells</abbr>, (<span class="cit-vol">39</span>):<span class="cit-fpage">367</span>–<span class="cit-lpage">374</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">0219-1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCells%26rft.volume%253D39%26rft.spage%253D367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[67].</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.67"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kumari</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sampath</span></span>. <span class="cit-article-title">cncRNAs: Bi-functional RNAs with protein coding and non-coding functions</span>. <abbr class="cit-jnl-abbrev">Semin Cell Dev Biol</abbr>, 47-48:<span class="cit-fpage">40</span>–<span class="cit-lpage">51</span>, <span class="cit-month">December</span> <span class="cit-pub-date">2015</span>. ISSN <span class="cit-issn">1096-3634</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[68].</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.68" data-doi="10.1007/s00239-008-9149-6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kirsten</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Stadler</span></span>. <span class="cit-article-title">Evolution of spliceosomal snRNA genes in metazoan animals</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr>, <span class="cit-vol">67</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">594</span>–<span class="cit-lpage">607</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMarz%26rft.auinit1%253DM.%26rft.volume%253D67%26rft.issue%253D6%26rft.spage%253D594%26rft.epage%253D607%26rft.atitle%253DEvolution%2Bof%2Bspliceosomal%2BsnRNA%2Bgenes%2Bin%2Bmetazoan%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-008-9149-6%26rft_id%253Dinfo%253Apmid%252F19030770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-008-9149-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19030770&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262837600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[69].</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [69] in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.69" data-doi="10.1093/nar/27.8.1767"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Boore</span></span>. <span class="cit-article-title">Animal mitochondrial genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1767</span>–<span class="cit-lpage">1780</span>, <span class="cit-pub-date">1999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.8.1767%26rft_id%253Dinfo%253Apmid%252F10101183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.8.1767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10101183&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079966900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[70].</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.70"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gerst</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hölzer</span></span>. <span class="cit-article-title">PCAGO: An interactive web service to analyze RNA-Seq data with principal component analysis</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, page <span class="cit-fpage">433078</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[71].</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [71] in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-738526v3.71" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Thorvaldsdóttir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Robinson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span>. <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</span>. <abbr class="cit-jnl-abbrev">Briefings Bioinf</abbr>, (<span class="cit-vol">14</span>):<span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>, <span class="cit-pub-date">2013</span>. ISSN <span class="cit-issn">1477-4054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2BBioinf%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F25%2F738526.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/738526v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F738526v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/770057v2" title="Distinct neural progenitor pools in the ventral telencephalon generate diversity in striatal spiny projection neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 25, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/738526v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/09/25/738526.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/738526v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/738526v3" class="active" data-icon-position="" data-hide-link-title="0">A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/921879" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WlWalzW434prRjEVkDEp6upEZRhFcHfS0rhK3Qz0v7o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="921879" id="node921879--2" data-pisa="biorxiv;738526v3" data-pisa-master="biorxiv;738526" data-seqnum="921879" data-apath="/biorxiv/early/2019/09/25/738526.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nelly</span> <span class="nlm-surname">Mostajo Berrospi</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Lataretu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Krautwurst</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Florian</span> <span class="nlm-surname">Mock</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Desirò</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kevin</span> <span class="nlm-surname">Lamkiewicz</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Maximilian</span> <span class="nlm-surname">Collatz</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Schoen</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Friedemann</span> <span class="nlm-surname">Weber</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Manja</span> <span class="nlm-surname">Marz</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Hölzer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">738526; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/738526 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/738526v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;title=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;title=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;text=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;title=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;t=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;title=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;title=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="921879" id="node921879--3" data-pisa="biorxiv;738526v3" data-pisa-master="biorxiv;738526" data-seqnum="921879" data-apath="/biorxiv/early/2019/09/25/738526.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nelly</span> <span class="nlm-surname">Mostajo Berrospi</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Lataretu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Krautwurst</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Florian</span> <span class="nlm-surname">Mock</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Desirò</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kevin</span> <span class="nlm-surname">Lamkiewicz</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Maximilian</span> <span class="nlm-surname">Collatz</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Schoen</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Friedemann</span> <span class="nlm-surname">Weber</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Manja</span> <span class="nlm-surname">Marz</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Hölzer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">738526; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/738526 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Mostajo+Berrospi%2C+Nelly&amp;rft.au=Lataretu%2C+Marie&amp;rft.au=Krautwurst%2C+Sebastian&amp;rft.au=Mock%2C+Florian&amp;rft.au=Desir%C3%B2%2C+Daniel&amp;rft.au=Lamkiewicz%2C+Kevin&amp;rft.au=Collatz%2C+Maximilian&amp;rft.au=Schoen%2C+Andreas&amp;rft.au=Weber%2C+Friedemann&amp;rft.au=Marz%2C+Manja&amp;rft.au=H%C3%B6lzer%2C+Martin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/921879/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/921879/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/921879/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/921879/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/921879/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/921879/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/921879/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/921879/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/921879/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/921879/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/921879/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/921879/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;count=horizontal&amp;via=&amp;text=A%20comprehensive%20annotation%20and%20differential%20expression%20analysis%20of%20short%20and%20long%20non-coding%20RNAs%20in%2016%20bat%20genomes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/738526v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/738526v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/738526v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
