<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/23/2020.01.29.925354.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.29.925354v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Interpretable detection of novel human viruses from genome sequencing data" />
<meta name="DC.Identifier" content="10.1101/2020.01.29.925354" />
<meta name="DC.Date" content="2020-03-23" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020." />
<meta name="DC.Contributor" content="Jakub M. Bartoszewicz" />
<meta name="DC.Contributor" content="Anja Seidel" />
<meta name="DC.Contributor" content="Bernhard Y. Renard" />
<meta name="article:published_time" content="2020-03-23" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Interpretable detection of novel human viruses from genome sequencing data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.29.925354v3" />
<meta name="citation_id" content="2020.01.29.925354v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.29.925354" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.29.925354" />
<meta name="citation_author" content="Jakub M. Bartoszewicz" />
<meta name="citation_author_institution" content="Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_institution" content="Department of Mathematics and Computer Science, Free University of Berlin" />
<meta name="citation_author_institution" content="Hasso Plattner Institute for Digital Engineering" />
<meta name="citation_author_institution" content="Digital Engineering Faculty, University of Postdam" />
<meta name="citation_author_email" content="jakub.bartoszewicz@hpi.de" />
<meta name="citation_author_email" content="bernhard.renard@hpi.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6893-2371" />
<meta name="citation_author" content="Anja Seidel" />
<meta name="citation_author_institution" content="Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_institution" content="Department of Mathematics and Computer Science, Free University of Berlin" />
<meta name="citation_author_institution" content="Central Research Institute of Ambulatory Health Care" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6856-728X" />
<meta name="citation_author" content="Bernhard Y. Renard" />
<meta name="citation_author_institution" content="Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_institution" content="Hasso Plattner Institute for Digital Engineering" />
<meta name="citation_author_institution" content="Digital Engineering Faculty, University of Postdam" />
<meta name="citation_author_email" content="jakub.bartoszewicz@hpi.de" />
<meta name="citation_author_email" content="bernhard.renard@hpi.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4589-9809" />
<meta name="citation_reference" content="Alipanahi, B., Delong, A., Weirauch, M. T., and Frey, B. J. Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning. Nature Biotechnology, 33(8):831–838, 2015. ISSN 1546-1696. doi: 10.1038/nbt.3300." />
<meta name="citation_reference" content="Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. Basic local alignment search tool. Journal of Molecular Biology, 215(3):403–410, 1990. ISSN 0022-2836. doi: 10.1016/S0022-2836(05)80360-2." />
<meta name="citation_reference" content="Avsec, Ž., Weilert, M., Shrikumar, A., Alexandari, A., Krueger, S., Dalal, K., Fropf, R., McAnany, C., Gagneur, J., Kundaje, A., and Zeitlinger, J. Deep learning at baseresolution reveals motif syntax of the cis-regulatory code. bioRxiv, pp. 737981, August 2019. doi: 10.1101/737981." />
<meta name="citation_reference" content="Babayan, S. A., Orton, R. J., and Streicker, D. G. Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes. Science, 362(6414): 577–580, November 2018. ISSN 0036-8075, 1095-9203. doi: 10.1126/science.aap9072." />
<meta name="citation_reference" content="Bach, S., Binder, A., Montavon, G., Klauschen, F., Müller, K.-R., and Samek, W. On Pixel-Wise Explanations for Non-Linear Classifier Decisions by Layer-Wise Relevance Propagation. PLOS ONE, 10(7):e0130140, July 2015. ISSN 1932-6203. doi: 10.1371/journal.pone.0130140." />
<meta name="citation_reference" content="Bartoszewicz, J. M., Seidel, A., Rentzsch, R., and Renard, B. Y. DeePaC: predicting pathogenic potential of novel DNA with reverse-complement neural networks. Bioinformatics, 36(1):81–89, 07 2019. ISSN 1367-4803. doi: 10.1093/bioinformatics/btz541." />
<meta name="citation_reference" content="Brendel, W. and Bethge, M. Approximating CNNs with bag-of-local-features models works surprisingly well on ImageNet. In International Conference on Learning Representations, 2019." />
<meta name="citation_reference" content="Calistri, A. and Palù, G. Editorial commentary: Unbiased next-generation sequencing and new pathogen discovery: undeniable advantages and still-existing drawbacks. Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America, 60(6):889–891, 2015. ISSN 1537-6591. doi: 10.1093/cid/ciu913." />
<meta name="citation_reference" content="Calvignac-Spencer, S., Schulze, J. M., Zickmann, F., and Renard, B. Y. Clock rooting further demonstrates that guinea 2014 ebov is a member of the zaïre lineage. PLoS currents, 6, 2014." />
<meta name="citation_reference" content="Crooks, G. E., Hon, G., Chandonia, J.-M., and Brenner, S. E. WebLogo: a sequence logo generator. Genome Research, 14(6):1188–1190, June 2004. ISSN 1088-9051. doi: 10.1101/gr.849004." />
<meta name="citation_reference" content="Edwards, R. A., McNair, K., Faust, K., Raes, J., and Dutilh, B. E. Computational approaches to predict bacteriophagehost relationships. FEMS microbiology reviews, 40(2): 258–272, 2016. ISSN 1574-6976. doi: 10.1093/femsre/fuv048." />
<meta name="citation_reference" content="Eng, C. L., Tong, J. C., and Tan, T. W. Predicting host tropism of influenza a virus proteins using random forest. BMC Medical Genomics, 7(3):S1, 2014. ISSN 1755-8794. doi: 10.1186/1755-8794-7-S3-S1." />
<meta name="citation_reference" content="Eraslan, G., Avsec, Ž., Gagneur, J., and Theis, F. J. Deep learning: new computational modelling techniques for genomics. Nature Reviews Genetics, 20(7):389–403, July 2019. ISSN 1471-0064. doi: 10.1038/s41576-019-0122-6." />
<meta name="citation_reference" content="Gałan, W., Bąk, M., and Jakubowska, M. Host taxon predictor - a tool for predicting taxon of the host of a newly discovered virus. Scientific Reports, 9(1):3436, 2019. ISSN 2045-2322. doi: 10.1038/s41598-019-39847-2." />
<meta name="citation_reference" content="Greenside, P., Shimko, T., Fordyce, P., and Kundaje, A. Discovering epistatic feature interactions from neural network models of regulatory DNA sequences. Bioinformatics, 34(17):i629–i637, 2018. ISSN 1367-4803. doi: 10.1093/bioinformatics/bty575." />
<meta name="citation_reference" content="Herfst, S., Schrauwen, E. J. A., Linster, M., Chutinimitkul, S., Wit, E. d., Munster, V. J., Sorrell, E. M., Bestebroer, T. M., Burke, D. F., Smith, D. J., Rimmelzwaan, G. F., Osterhaus, A. D. M. E., and Fouchier, R. A. M. Airborne Transmission of Influenza A/H5n1 Virus Between Ferrets. Science, 336(6088):1534–1541, June 2012. ISSN 0036-8075, 1095-9203. doi: 10.1126/science.1213362." />
<meta name="citation_reference" content="Holtgrewe, M. Mason – a read simulator for second generation sequencing data. Technical Report FU Berlin, 2010." />
<meta name="citation_reference" content="Imai, M., Watanabe, T., Hatta, M., Das, S. C., Ozawa, M., Shinya, K., Zhong, G., Hanson, A., Katsura, H., Watanabe, S., Li, C., Kawakami, E., Yamada, S., Kiso, M., Suzuki, Y., Maher, E. A., Neumann, G., and Kawaoka, Y. Experimental adaptation of an influenza H5 HA confers respiratory droplet transmission to a reassortant H5 HA/H1n1 virus in ferrets. Nature, 486(7403):420–428, June 2012. ISSN 1476-4687. doi: 10.1038/nature10831." />
<meta name="citation_reference" content="Jha, A., Aicher, J. K., Singh, D., and Barash, Y. Improving interpretability of deep learning models: splicing codes as a case study. bioRxiv, 2019. doi: 10.1101/700096." />
<meta name="citation_reference" content="Kelley, D. R., Snoek, J., and Rinn, J. L. Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks. Genome Research, 26 (7):990–999, 2016. ISSN 1088-9051. doi: 10.1101/gr.200535.115." />
<meta name="citation_reference" content="Lanchantin, J., Singh, R., Lin, Z., and Qi, Y. Deep Motif: Visualizing Genomic Sequence Classifications. CoRR, abs/1605.01133, 2016." />
<meta name="citation_reference" content="Lanchantin, J., Singh, R., Wang, B., and Qi, Y. Deep motif dashboard: visualizing and understanding genomic sequences using deep neural networks. Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing, 22:254–265, 2017. ISSN 2335-6936. doi: 10.1142/9789813207813_0025." />
<meta name="citation_reference" content="Lecuit, M. and Eloit, M. The diagnosis of infectious diseases by whole genome next generation sequencing: a new era is opening. Frontiers in Cellular and Infection Microbiology, 4:25, 2014. ISSN 2235-2988. doi: 10.3389/fcimb.2014.00025." />
<meta name="citation_reference" content="Leendertz, S. A. J., Gogarten, J. F., Düx, A., Calvignac-Spencer, S., and Leendertz, F. H. Assessing the evidence supporting fruit bats as the primary reservoirs for ebola viruses. EcoHealth, 13(1):18–25, Mar 2016. ISSN 1612-9210. doi: 10.1007/s10393-015-1053-0." />
<meta name="citation_reference" content="Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annual Review of Virology, 3(1):237–261, 2016. doi: 10.1146/annurev-virology-110615-042301." />
<meta name="citation_reference" content="Li, H. and Sun, F. Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences. Scientific Reports, 8(1):10032, 2018. ISSN 2045-2322. doi: 10.1038/s41598-018-28308-x." />
<meta name="citation_reference" content="Lipsitch, M. and Inglesby, T. V. Moratorium on Research Intended To Create Novel Potential Pandemic Pathogens. mBio, 5(6), December 2014. ISSN 2150-7511. doi: 10.1128/mBio.02366-14." />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_author=S. M. Lundberg;citation_author=S.-I. Lee;citation_pages=4765-4774;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="Marchler-Bauer, A., Bo, Y., Han, L., He, J., Lanczycki, C. J., Lu, S., Chitsaz, F., Derbyshire, M. K., Geer, R. C., Gonzales, N. R., Gwadz, M., Hurwitz, D. I., Lu, F., Marchler, G. H., Song, J. S., Thanki, N., Wang, Z., Yamashita, R. A., Zhang, D., Zheng, C., Geer, L. Y., and Bryant, S. H. CDD/SPARCLE: functional classification of proteins via subfamily domain architectures. Nucleic Acids Research, 45(D1):D200–D203, 2017. ISSN 1362-4962. doi: 10.1093/nar/gkw1129." />
<meta name="citation_reference" content="Mihara, T., Nishimura, Y., Shimizu, Y., Nishiyama, H., Yoshikawa, G., Uehara, H., Hingamp, P., Goto, S., and Ogata, H. Linking virus genomes with host taxonomy. Viruses, 8(3):66, 2016. ISSN 1999-4915. doi: 10.3390/v8030066." />
<meta name="citation_reference" content="Mock, F., Viehweger, A., Barth, E., and Marz, M. Viral host prediction with deep learning. bioRxiv, pp. 575571, 2019. doi: 10.1101/575571." />
<meta name="citation_reference" content="Moustafa, A., Xie, C., Kirkness, E., Biggs, W., Wong, E., Turpaz, Y., Bloom, K., Delwart, E., Nelson, K. E., Venter, J. C., and Telenti, A. The blood DNA virome in 8,000 humans. PLOS Pathogens, 13(3):e1006292, March 2017. ISSN 1553-7374. doi: 10.1371/journal.ppat.1006292." />
<meta name="citation_reference" content="Nair, S., Kim, D. S., Perricone, J., and Kundaje, A. Integrating regulatory DNA sequence and gene expression to predict genome-wide chromatin accessibility across cellular contexts. Bioinformatics, 35(14):i108–i116, July 2019. ISSN 1367-4803. doi: 10.1093/bioinformatics/btz352." />
<meta name="citation_reference" content="Noyce, R. S., Lederman, S., and Evans, D. H. Construction of an infectious horsepox virus vaccine from chemically synthesized DNA fragments. PLOS ONE, 13 (1):e0188453, January 2018. ISSN 1932-6203. doi: 10.1371/journal.pone.0188453." />
<meta name="citation_reference" content="Poplin, R., Chang, P.-C., Alexander, D., Schwartz, S., Colthurst, T., Ku, A., Newburger, D., Dijamco, J., Nguyen, N., Afshar, P. T., Gross, S. S., Dorfman, L., McLean, C. Y., and DePristo, M. A. A universal SNP and small-indel variant caller using deep neural networks. Nature Biotechnology, 36(10):983–987, 2018. ISSN 1546-1696. doi: 10.1038/nbt.4235." />
<meta name="citation_reference" content="Quang, D. and Xie, X. DanQ: a hybrid convolutional and recurrent deep neural network for quantifying the function of DNA sequences. Nucleic Acids Research, 44(11):e107–e107, 2016. ISSN 0305-1048. doi: 10.1093/nar/gkw226." />
<meta name="citation_reference" content="Schneider, T. D. and Stephens, R. M. Sequence logos: a new way to display consensus sequences. Nucleic Acids Research, 18(20):6097–6100, October 1990. ISSN 0305-1048. doi: 10.1093/nar/18.20.6097." />
<meta name="citation_reference" content="Stojkova, P., Spidlova, P., and Stulik, J. Nucleoid-associated protein hu: A lilliputian in gene regulation of bacterial virulence. Frontiers in Cellular and Infection Microbiology, 9:159, 2019. ISSN 2235-2988. doi: 10.3389/fcimb.2019.00159." />
<meta name="citation_reference" content="Sundararajan, M., Taly, A., and Yan, Q. Gradients of Coun-terfactuals. CoRR, abs/1611.02639, 2016." />
<meta name="citation_reference" content="Tampuu, A., Bzhalava, Z., Dillner, J., and Vicente, R. Vi-raMiner: Deep learning on raw DNA sequences for identifying viral genomes in human samples. PLOS ONE, 14 (9):e0222271, September 2019. ISSN 1932-6203. doi: 10.1371/journal.pone.0222271." />
<meta name="citation_reference" content="Thiel, V. Synthetic viruses-Anything new? PLoS pathogens, 14(10):e1007019, 2018. ISSN 1553-7374. doi: 10.1371/journal.ppat.1007019." />
<meta name="citation_reference" content="Thorvaldsdóttir, H., Robinson, J. T., and Mesirov, J. P. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Briefings in Bioinformatics, 14(2):178–192, March 2013. ISSN 1467-5463. doi: 10.1093/bib/bbs017." />
<meta name="citation_reference" content="Trappe, K., Marschall, T., and Renard, B. Y. Detecting horizontal gene transfer by mapping sequencing reads across species boundaries. Bioinformatics, 32(17):i595–i604, September 2016. ISSN 1367-4803. doi: 10.1093/bioinformatics/btw423." />
<meta name="citation_reference" content="Vouga, M. and Greub, G. Emerging bacterial pathogens: the past and beyond. Clinical Microbiology and Infection, 22(1):12–21, January 2016. ISSN 1198-743X. doi: 10.1016/j.cmi.2015.10.010." />
<meta name="citation_reference" content="Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Hu, Y., Song, Z.-G., Tao, Z.-W., Tian, J.-H., Pei, Y.-Y., Yuan, M.-L., Zhang, Y.-L., Dai, F.-H., Liu, Y., Wang, Q.-M., Zheng, J.-J., Xu, L., Holmes, E. C., and Zhang, Y.-Z. Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China. bioRxiv, pp. 2020. 01.24.919183, January 2020. doi: 10.1101/2020.01.24.919183." />
<meta name="citation_reference" content="Zeng, H., Edwards, M. D., Liu, G., and Gifford, D. K. Convolutional neural network architectures for predicting DNA–protein binding. Bioinformatics, 32(12):i121–i127, 2016. ISSN 1367-4803. doi: 10.1093/bioinformatics/btw255." />
<meta name="citation_reference" content="Zhang, Z., Cai, Z., Tan, Z., Lu, C., Jiang, T., Zhang, G., and Peng, Y. Rapid identification of human-infecting viruses. Transboundary and Emerging Diseases, August 2019. ISSN 1865-1682. doi: 10.1111/tbed.13314." />
<meta name="citation_reference" content="Zhou, J. and Troyanskaya, O. G. Predicting effects of noncoding variants with deep learning–based sequence model. Nature Methods, 12(10):931–934, 2015. ISSN 1548-7105. doi: 10.1038/nmeth.3547." />
<meta name="twitter:title" content="Interpretable detection of novel human viruses from genome sequencing data" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020." />
<meta name="og-title" property="og:title" content="Interpretable detection of novel human viruses from genome sequencing data" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-23" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.29.925354v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.29.925354v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.29.925354v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1208641" />
    <title>Interpretable detection of novel human viruses from genome sequencing data | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WG3PFTtXG3oMnuyzF760S8R03uFotXFDDzq64qUYmaQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1208641","apath":"\/biorxiv\/early\/2020\/03\/23\/2020.01.29.925354.atom","pisa":"biorxiv;2020.01.29.925354v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.29.925354v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.29.925354v3","title":"Interpretable detection of novel human viruses from genome sequencing data","identifier":"node\/1208641"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224l9uJsDer47THKJzhoZcgrp\\\\\\\/MXCMa1KLUMSxYZQhLlmrnhG5PBp2EKnWHu3VyuIstX56SD2JhRp2L2GhNzRUk5PHFJnR1kHxJqcMkBY3+HZjuQJK9YGnkRcM7JRfqewBOjBH8efA8g3AhQtJdvEZcPS8cQxdpBrenc5qq+QfJkyDZ8Yql2NpoSoDgtBFsqIq85\\\\\\\/nNVim1LsMBSm3H9WqnRDa68IP7F56jF6nh\\\\\\\/X6hfMw2KYW6tTk6S23iIsxNWOe17PnW7x2fb5d9EoRnVzh8H68RQEt30DzzZ0QiXgC02bc1oycKI9FblCOrnh6NQRRIK3mPod5v6+5tPrj7O8OslWFjOXeIywz8sJDswJIn5F4iZL\\\\\\\/DcofSXrHRruHckPETf\\\\\\\/jNQDOTh7drwZTE60tnSSnR3LwWlUM03eBWD7M98XUcoDB4krEUXNZHR7Zhdh0xSlJxoos6yld2KhK\\\\\\\/083yFHl7qeyQAUOib8DvzAMCk6aQqO7\\\\\\\/7VGVcvK4ugKxTB+MovM4lvHXLn0NCCMNnuuKuyqDcMw0uFY9Jes7M3c5Gb4s2wG6YkkT3Gbdz56fieO9kq+CHmVY87TVziP9quABC5+lLnoAmbg07YbcL7Slsl9WFzVDpSUeEF3BONrEgzFzQYnfwLX83DWF5315qXtrnOqKMokdbJ0NDzrUof9g9WF4VIIKiSWvPTA6zaJjSkjOJmJjhU8qgQ6bjKiutcJruNlM4VEqoJHvanPIqMXo88nu9wt0FeO0zaLuwJ1Jxf1XXYz4ltwpGTNgdATEs2I1ieI53gts0zPskwf+CKryxWKitA5shuPh\\\\\\\/qoDNY1ZXCG0xbM6+uYk2ILskBPmPX\\\\\\\/LXdOmH3Y4+hqq3V7fc6BsvLCBgvBh5w26ZaGbyadtugdZtL92ZZ3DwomdsN\\\\\\\/4in4JDEb7qfczgBvjXN+\\\\\\\/rz1tV5ApcrHAyc8Mpzn1E71Nhsk9Xh+RJcphUwR3xqcg8d36CPftDcFFP7rcwjdomhB7wX6ee3GBCKTrFF4xhZUYsNLbWNry7tX3JT8BuGpCEPsy6e4UW3m49IWB\\\\\\\/ukqKGmKqhpqWenmy+hLiOtc7qkpnxAPEFtO34e\\\\\\\/tojhGiX2DJribZqzES9dch+fOfkF9ycehgsEkZbXY6jxkc6oGHeeshsDGKQVfBlVb6hfFs9\\\\\\\/Bk2D2uvbG2qirE1VLahgdQvGZsheH5ZKzJ7Dj0j7PlHJprbjxR1dO1NT62nNC2BrB8NjZS26lDtwS++baYf\\\\\\\/XA6wBLbgTMZnX6Ce7ezWf8vzCsikPclAhaiEV5ZEUXj0GPz4xewpVHjxhnBPx8+jU6ekshxe1UuyIvBwb5dJwtndg5H\\\u0022,\\\u0022iv\\\u0022:\\\u0022FAfmaG5pwp5aZETEvpmvVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220a36be9b73c705d85a118a8b71d716aee5c20a811d8b2bb0ba68d73265ef28fb\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.29.925354v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1208641 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.29.925354v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9W0K2A4KUh-X1Zi8vkiyHITUpG1oB19WXdGiY2ZSamY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208641" id="node1208641" data-pisa="biorxiv;2020.01.29.925354v3" data-pisa-master="biorxiv;2020.01.29.925354" data-apath="/biorxiv/early/2020/03/23/2020.01.29.925354.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Interpretable detection of novel human viruses from genome sequencing data</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6893-2371" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jakub M.</span> <span class="nlm-surname">Bartoszewicz</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6856-728X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Anja</span> <span class="nlm-surname">Seidel</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4589-9809" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.29.925354 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1208641" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jakub M. Bartoszewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics and Computer Science, Free University of Berlin</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Hasso Plattner Institute for Digital Engineering</span>, Potsdam, Brandenburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Digital Engineering Faculty, University of Postdam</span>, Potsdam, Brandenburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jakub%2BM.%2BBartoszewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bartoszewicz%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJakub%2BM.%2BBartoszewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6893-2371" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jakub M. Bartoszewicz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jakub.bartoszewicz@hpi.de" class="" data-icon-position="" data-hide-link-title="0">jakub.bartoszewicz@hpi.de</a>
<a href="mailto:bernhard.renard@hpi.de" class="" data-icon-position="" data-hide-link-title="0">bernhard.renard@hpi.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anja Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics and Computer Science, Free University of Berlin</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Central Research Institute of Ambulatory Health Care</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anja%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnja%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6856-728X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anja Seidel</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernhard Y. Renard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics (MF1), Department of Methodology and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Hasso Plattner Institute for Digital Engineering</span>, Potsdam, Brandenburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Digital Engineering Faculty, University of Postdam</span>, Potsdam, Brandenburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BY.%2BRenard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Renard%20BY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernhard%2BY.%2BRenard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4589-9809" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bernhard Y. Renard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jakub.bartoszewicz@hpi.de" class="" data-icon-position="" data-hide-link-title="0">jakub.bartoszewicz@hpi.de</a>
<a href="mailto:bernhard.renard@hpi.de" class="" data-icon-position="" data-hide-link-title="0">bernhard.renard@hpi.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.29.925354v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1208641" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1208641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925354v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1208641" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1208641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925354v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1208641" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1208641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925354v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1208641" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1208641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.29.925354v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1208641" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1208641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><div id="sec-2" class="subsection"><h3>1.1 Background</h3><p id="p-3">Within a globally interconnected and densely populated world, pathogens can spread more easily than they ever did before. As the recent outbreaks of Ebola and Zika viruses have shown, the risks posed even by these previously known agents remain unpredictable and their expansion hard to control (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Calvignac-Spencer et al., 2014</a>). What is more, it is almost certain that more unknown pathogen species and strains are yet to be discovered, given their constant, extremely fast-paced evolution and unexplored biodiversity, as well as increasing human exposure (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vouga &amp; Greub, 2016</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Trappe et al., 2016</a>). Some of those novel pathogens may cause epidemics (similar to the SARS and MERS coronavirus outbreaks in 2002 and 2012) or even pandemics (e.g. SARS-CoV-2 and the “swine flu” H1N1/09 strain). Many have more than one host or vector, which makes assessing and predicting the risks even more difficult. For example, Ebola has its natural reservoir most likely in fruit bats (Leen-dertz et al., 2016), but causes deadly epidemics in both humans and chimpanzees. As the state-of-the art approach for the open-view detection of pathogens is genome sequencing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lecuit &amp; Eloit, 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calistri &amp; Palù, 2015</a>), it is crucial to develop automated pipelines for characterizing the infectious potential of currently unidentifiable sequences.</p><p id="p-4">Screening against potentially dangerous subsequences before their synthesis may also be used as a way of ensuring responsible research in synthetic biology. While potentially useful in some applications, engineering of viral genomes could also pose a biosecurity and biosafety threat. Two controversial studies modified the influenza A/H5N1 (“bird flu”) virus to be airborne transmissible in mammals (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Herfst et al., 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Imai et al., 2012</a>). A possibility of modifying coronaviruses to enhance their virulence triggered calls for a moratorium on this kind of research (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lipsitch &amp; Inglesby, 2014</a>). Synthesis of an infectious horsepox virus closely related to the smallpox-causing <em>Variola</em> virus (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Noyce et al., 2018</a>) caused a public uproar and calls for intensified discussion on risk control in synthetic biology (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thiel, 2018</a>).</p></div><div id="sec-3" class="subsection"><h3>1.2. Current tools for host range prediction</h3><p id="p-5">Several computational, genome-based methods exist that allow to predict the host-range of a bacteriophage (a bacteria-infecting virus). A selection of composition-based and alignment-based approaches has been presented in an extensive review by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edwards et al. (2016)</a>. Prediction of eu-kariotic host tropism (including humans) based on known protein sequences was shown for the influenza A virus (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eng et al., 2014</a>). Two recent studies employ <em>k</em>-mer based, <em>k</em>-NN classifiers (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li &amp; Sun, 2018</a>) and deep learning (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mock et al., 2019</a>) to predict host range for a small set of three well-studied species directly from viral sequences. While those approaches are limited to those particular species and do not scale to viral host-range prediction in general, the Host Taxon Predictor (HTP) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gałan et al., 2019</a>) uses logistic regression and support vector machines to predict if a novel virus infects bacteria, plants, vertebrates or arthropods. Yet, the authors argue that it is not possible to use HTP in a read-based manner; it requires long sequences of at least 3,000 nucleotides. This is incompatible with modern metagenomic next-generation sequencing workflows, where the DNA reads obtained are at least 10-20 times shorter. Another study used gradient boosting machines to predict reservoir hosts and transmission via arthropod vectors for known human-infecting viruses (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Babayan et al., 2018</a>).</p><p id="p-6"><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang et al. (2019)</a> designed several classifiers explicitly predicting whether a new virus can potentially infect humans. Their best model, a <em>k</em>-NN classifier, uses <em>k</em>-mer frequencies as features representing the query sequence and can yield predictions for sequences as short as 500 base pairs (bp). It worked also with 150bp-long reads from real DNA sequencing runs, although in this case the reads originated also from the viruses present in the training set (and were therefore not “novel”).</p></div><div id="sec-4" class="subsection"><h3>1.3. Deep Learning for DNA sequences</h3><p id="p-7">While DNA sequences mapped to a reference genome may be represented as images (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Poplin et al., 2018</a>), a majority of studies uses a distributed orthographic representation, where each nucleotide {A, <em>C, G, T</em>} in a sequence is represented by a one-hot encoded vector of length 4. An “unknown” nucleotide (N) can be represented as an all-zero vector. CNNs and LSTMs have been successfully used for a variety of DNA-based prediction tasks. Early works focused mainly on regulation of gene expression in humans (Ali-panahi et al., 2015; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhou &amp; Troyanskaya, 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zeng et al., 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Quang &amp; Xie, 2016</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kelley et al., 2016</a>), which is still an area of active research (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Greenside et al., 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nair et al., 2019</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Avsec et al., 2019</a>). In the field of pathogen genomics, deep learning models trained directly on DNA sequences were developed to predict host ranges of three multi-host viral species (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mock et al., 2019</a>) and to predict pathogenic potentials of novel bacteria (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bartoszewicz et al., 2019</a>). DeepVirFinder (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ren et al., 2018</a>) and ViraMiner (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tampuu et al., 2019</a>) can detect viral sequences in metage-nomic samples, but they cannot predict the host and focus on previously known species. For a broader view on deep learning in genomics we refer to a recent review by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eraslan et al. (2019)</a>.</p><p id="p-8">Interpretability and explainability of deep learning models for genomics is crucial for their wide-spread adoption, as it is necessary for delivering trustworthy and actionable results. Convolutional filters can be visualized by forward-passing multiple sequences through the network and extracting the most-activating subsequences (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alipanahi et al., 2015</a>) to create a position weight matrix (PWM) which can be visualized as a sequence logo (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schneider &amp; Stephens, 1990</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crooks et al., 2004</a>). Direct optimization of input sequences is problematic, as it results in generating a dense matrix even though the input sequences are one-hot encoded (Lan-chantin et al., 2016; 2017). This problem can be alleviated with Integrated Gradients (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sundararajan et al., 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jha et al., 2019</a>) or DeepLIFT, which propagates activation differences relative to a selected reference back to the input, reducing the computational overhead of obtaining accurate gradients (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shrikumar et al., 2019a</a>). If a reference of all-zeros is used, the method is analogous to Layer-wise Relevance Propagation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bach et al., 2015</a>). DeepLIFT is an additive feature attribution method, and may used to approximate Shapley values if the input features are independent (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lundberg &amp; Lee, 2017</a>). TF-MoDISco (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shrikumar et al., 2019b</a>) uses DeepLIFT to discover consolidated, biologically meaningful DNA motifs (transcription factor binding sites).</p></div><div id="sec-5" class="subsection"><h3>1.4. Contributions</h3><p id="p-9">In this paper, we first improve the performance of read-based predictions of the viral host (human or non-human) from next-generation sequencing reads. We show that reversecomplement (RC) neural networks (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bartoszewicz et al., 2019</a>) significantly outperform both the previous state-of-the-art (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Zhang et al., 2019</a>) and the traditional, alignmentbased algorithm – BLAST (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul et al., 1990</a>), which constitutes a gold standard in homology-based bioinformatics analyses. We show that defining the negative (nonhuman) class is non-trivial and compare different ways of constructing the training set. Strikingly, a model trained to distinguish between viruses infecting humans and viruses infecting other chordates (a phylum of animals including vertebrates) generalizes well to evolutionary distant nonhuman hosts, including even bacteria. This suggests that the host-related signal is strong and the learned decision boundary separates human viruses from other DNA sequences surprisingly well.</p><p id="p-10">Next, we propose a new approach for convolutional filter visualization using partial Shapley values to differentiate between simple nucleotide information content and the contribution of each sequence position to the final classification score. To test the biological plausibility of our models, we generate genome-wide maps of “infectious potential” and nucleotide contributions. We show that those maps can be used to visualize and detect virulence-related regions of interest (e.g. genes) in novel genomes. Finally, we analyze a recently discovered SARS-CoV-2 coronavirus, which caused a pandemic in 2020 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wu et al., 2020</a>).</p></div></div><div class="section" id="sec-6"><h2 class="">2. Methods</h2><div id="sec-7" class="subsection"><h3>2.1. Data collection and preprocessing</h3><div id="sec-8" class="subsection"><h4>2.1.1. VHDB <span class="sc">dataset</span></h4><p id="p-11">We accessed the Virus-Host Database (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mihara et al., 2016</a>) on July 31, 2019 and downloaded all the available data. The original dataset contained 14,380 records comprising Ref-Seq IDs for viral sequences and associated metadata. Some viruses are divided into discontiguous segments, which are represented as separate records in VHDB; in those cases the segments were treated as contigs of a single genome in the further analysis. We removed records with unspecified host information and those confusing the highly pathogenic Variola virus with a similarly named genus of fish. Further, we filtered out viroids and satellites. Human-infecting viruses were extracted by searching for records containing “Homo sapiens” in the “host name” field. Note that VHDB contains information about multiple possible hosts for a given virus where appropriate. Any virus infecting humans was assigned to the positive class, also if other, non-human hosts exist. In total, the dataset contained 9,496 viruses, including 1,309 human viruses. We considered both DNA and RNA viruses; RNA sequences were encoded in the DNA alphabet, as in RefSeq.</p></div><div id="sec-9" class="subsection"><h4>2.1.2. D<span class="sc">efining the negative class</span></h4><p id="p-12">While defining a human-infecting class is relatively straightforward, the reference negative class may be conceptualized in a variety of ways. The broadest definition takes all non-human viruses into account, including bacteriophages (bacterial viruses). This is especially important, as most of known bacteriophages are DNA viruses, while many important human (and animal) viruses are RNA viruses. One could expect that the multitude of available bacteriophage genomes dominating the negative class could lower the prediction performance on viruses similar to those infecting humans. This offers an open-view approach covering a wider part of the sequence space, but may lead to misclassification of potentially dangerous mammalian or avian viruses. As they are often involved in clinically relevant host-switching events, a stricter approach must also be considered. In this case, the negative class comprises only viruses infecting Chordata (a group containing vertebrates and closely related taxa). Two intermediate approaches consider all eukaryotic viruses (including plant and fungi viruses), or only animal-infecting viruses. This amounts to four nested host sets: “All” (8,187 non-human viruses), “Eukaryota” (5,114 viruses), “Metazoa” (2,942 viruses) and “Chordata” (2,078 viruses). Auxiliary sets containing only non-eukaryotic viruses (“non-Eukaryota”), non-animal eukaryotic viruses (“non-Metazoa Eukaryota”) etc. can be easily constructed by set subtraction.</p><p id="p-13">For the positive class, we generated a training set containing 80% of the genomes, and validation and test sets with 10% of the genomes each. Importantly, the nested structure was kept also during the training-validation-test split: for example, the species assigned to the smallest test set (“Chordata”) were also present in all the bigger test sets. The same applied to other taxonomic levels, as well as the training and validation sets wherever applicable.</p></div><div id="sec-10" class="subsection"><h4>2.1.3. R<span class="sc">ead simulation</span></h4><p id="p-14">We simulated 250bp long Illumina reads following a modification of a previously described protocol (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bartoszewicz et al., 2019</a>) and using the Mason read simulator (Holt-grewe, 2010). First, we only generated the reads from the genomes of human-infecting viruses. Then, the same steps were applied to each of the four negative class sets. Finally, we also generated a fifth set, “Stratified”, containing an equal number of reads drawn from genomes of the following disjunct host classes: “Chordata” (25%), “non-Chordata Metazoa” (25%), “non-Metazoa Eukaryota” (25%) and “non-Eukaryota” (25%).</p><p id="p-15">In each of the evaluated settings, we used a total of 20 million (80%) reads for training, 2.5 million (10%) reads for validation and 2.5 million (10%) paired reads as the held-out test set. Read number per genome was proportional to genome length, keeping the coverage uniform on average. While the original datasets are heavily imbalanced, we generated the same number of negative and positive data points (reads) regardless of the negative class definition used.</p><p id="p-16">This protocol allowed us to test the impact of defining the negative class, while using the exactly same data as representatives of the positive class. We used three training and validation sets (“All”, “Stratified”, and “Chordata”), representing the fully open-view setting, a setting more balanced with regard to the host taxonomy, and a setting focused on cases most likely to be clinically relevant. In each setting, the validation set matched the composition of the training set. The evaluation was performed using all five test sets to gain a more detailed insight on the effects of negative class definition on the prediction performance.</p></div><div id="sec-11" class="subsection"><h4>2.1.4. H<span class="sc">uman blood virome dataset</span></h4><p id="p-17">Similarily to <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Zhang et al. (2019)</a>, we used the human blood DNA virome dataset (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moustafa et al., 2017</a>) to test the selected classifiers on real data. We obtained 14,242,329 reads of 150bp and searched all of VHDB using blastn (with default parameters) to obtain high-quality reference labels. If a read’s best hit was a human-infecting virus, we assigned it to a positive class; the negative class was assigned if this was not the case. This procedure yielded 14,012,665 “positive” and 229,664 “negative” reads.</p></div></div><div id="sec-12" class="subsection"><h3>2.2. Training</h3><p id="p-18">We used the DeePaC package (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bartoszewicz et al., 2019</a>) to investigate RC-CNN and RC-LSTM architectures, previously shown to accurately predict bacterial pathogenicity. Here, we employ a CNN with two convolutional layers with 512 filters of size 15 each, average pooling and 2 fully connected layers with 256 units each. The LSTM used has 384 units. We use dropout regularization in both cases, together with aggressive input dropout at the rate of 0.2 or 0.25 (tuned for each model). Input dropout may be interpreted as a special case of noise injection, where a fraction of input nucleotides is turned to <em>N</em>s. Representations of forward and reverse-complement strands are summed before the fully connected layers. As two mates in a read pair should originate from the same virus, predictions obtained for them can be averaged for a boost in performance. If a contig or genome is available, averaging predictions for constituting reads yields a prediction for the whole sequence. We used Tesla P100 GPUs for training and an RTX 2080 Ti for visualizations.</p><p id="p-19">We wanted the networks to yield accurate predictions for both 250bp (our data, modelling a sequencing run of an Illumina MiSeq device) and 150bp long reads (as in the Human Blood Virome dataset). As shorter reads are padded with zeros, we expected the CNNs trained using average pooling to misclassify many of them. Therefore, we prepared a modified version of the “Stratified” dataset, in which the last 100bp of each read were turned to zeros, mocking a shorter sequencing run while preserving the error model. Then, we retrained the CNN which had performed best on the original dataset. Since in principle, the Human Blood Virome dataset should not contain viruses infecting non-human Chordata, a “Chordata”-trained classifier was not used in this setting.</p></div><div id="sec-13" class="subsection"><h3>2.3. Benchmarking</h3><p id="p-20">We compare the networks to the <em>k</em>-NN classifier proposed by <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Zhang et al. (2019)</a> and used by them for read-based predictions. We trained the classifier on the “All” dataset as described by the authors, i.e. using non-overlapping, 500bp-long contigs generated from the training genomes (retraining on simulated reads is computationally prohibitive). We also tested the performance of using BLAST to search against an indexed database of labeled genomes. We constructed the database from the “All” training set and used discontiguous megablast to achieve high inter-species sensitivity.</p><p id="p-21">Note that both BLAST and <em>k</em>-NN can yield conflicting predictions for the individual mates in a read pair. What is more, BLAST yields no prediction at all if no match is found. Therefore, similarly to <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Bartoszewicz et al. (2019)</a>, we used the <em>accept anything</em> operator to integrate binary predictions for read pairs and genomes. At least one match is needed to predict a label, and conflicting predictions are treated as if no match was found at all. Missing predictions lower both true positive and true negative rates.</p></div><div id="sec-14" class="subsection"><h3>2.4. Filter visualization</h3><div id="sec-15" class="subsection"><h4>2.4.1. S<span class="sc">ubstring extraction</span></h4><p id="p-22">In order to visualize the learned convolutional filters, we downsample a matching test set to 125,000 reads and pass it through the network. This is modelled after the method presented by <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alipanahi et al. (2015)</a>. For each filter and each input sequence, the authors extracted a subsequence leading to the highest activation, and created sequence logos from the obtained sequence sets (“max-activation”). We used DeepLIFT (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Shrikumar et al., 2019a</a>) to extract score-weighted subsequences with the highest contribution score (“max-contrib”) or all subsequences with non-zero contributions (“all-contrib”). Computing the latter was costly and did not yield better quality logos.</p><p id="p-23">We use an all-zero reference. As reads from real sequencing runs are usually not equally long, shorter reads must be padded with Ns; the “unknown” nucleotide is also called whenever there is not enough evidence to assign any other to the raw sequencing signal. Therefore, Ns are “null” nucleotides and are a natural candidate for the reference input. We do not consider alternative solutions based on GC content or dinucleotide shuffling, as the input reads originate from multiple different species, and the sequence composition may itself be a strong marker of both virus and host taxonomy.</p><p id="p-24">However, some of the training sequences contain Ns themselves. It is therefore possible that a filter will learn only negative weights at a given position, even though there is no biological justification for that. This may lead to assigning only negative contributions to all four possible nucleotides at a given position if the filter’s contribution is positive (and positive nucleotide contributions if the filter’s contribution is negative). To solve the problem, we first normalize the weight matrices position-wise, as described by <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Shrikumar et al. (2019a)</a>. Finally, we calculate average filter contributions to obtain a crude ranking of feature importance with regard to both the positive and negative class.</p></div><div id="sec-16" class="subsection"><h4>2.4.2. P<span class="sc">artial</span> S<span class="sc">hapley values</span></h4><p id="p-25">Building sequence logos involves calculating information content (IC) of each nucleotide at each position in a prospective DNA motif. This can be then interpreted as measure of evolutionary sequence conservation. However, high IC does not necessarily imply that a given nucleotide is relevant in terms of its contribution to the classifier’s output. Some sub-motifs may be present in the sequences used to build the logo, even if they do not contribute to the final prediction (or even a given filter’s activation).</p><p id="p-26">To test this hypothesis, we use partial Shapley values. Intuitively speaking, we capture the contributions of a nucleotide to the network’s output, but only in the context of a given intermediate neuron of the convolutional layer. More precisely, for any given feature <em>x<sub>i</sub></em>, intermediate neuron <em>y<sub>i</sub></em> and the output neuron <em>z</em>, we aim to measure how <em>x<sub>i</sub></em> contributes to <em>z</em> while regarding only the fraction of the total contribution of <em>x<sub>i</sub></em> that influences how <em>y<sub>j</sub></em> contributes to <em>z</em>.</p><p id="p-27">Using the formalism of DeepLIFT’s multipliers (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Shrikumar et al., 2019a</a>) and their reinterpretation in SHAP (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lundberg &amp; Lee, 2017</a>), we backpropagate the activation differences only along the paths “passing through” <em>y<sub>j</sub></em>. In <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Eq. 1</a>, we define partial multipliers <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-1.gif"/></noscript></span></span> and express them in terms of Shapley values <em>ϕ</em> and activation differences w.r.t. the expected activation values (reference activation). Calculating partial multipliers is equivalent to zeroing out the multipliers <em>m<sub>y<sub>k</sub>z</sub></em> for all <em>k</em> = <em>j</em> before backpropagating <em>m<sub>y<sub>j</sub>z</sub></em> further.
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-1.gif"/></noscript></span>

</span></p><p id="p-28">We define partial Shapley values <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-2.gif"/></noscript></span></span> analogously to how Shapley values can be approximated by a product of multipliers and input differences w.r.t. the reference (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Eq. 2</a>):
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-2.gif"/></noscript></span>

</span></p><p id="p-29">From the chain rule for multipliers (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Shrikumar et al., 2019a</a>), it follows that standard multipliers are a sum over all partial multipliers for a given layer <em>y</em>. Therefore, Shapley values as approximated by DeepLIFT are a sum of partial Shapley values for the layer <em>y</em>.</p><p id="p-30">Once we calculate the contributions of convolutional filters for the first layer, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/inline-graphic-3.gif"/></noscript></span></span> for the first convolutional layer of a network with one-hot encoded inputs and an all-zero reference can be efficiently calculated using weight matrices and filter activation differences (<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Eq. 3</a>–<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">4</a>). First, in this case we do not traverse any non-linearities and can directly use the linear rule (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Shrikumar et al., 2019a</a>) to calculate the contributions of <em>x<sub>i</sub></em> to <em>y<sub>j</sub></em> as a product of the weight <em>w<sub>i</sub></em> and the input <em>x<sub>i</sub></em>. Second, the input values may only be 0 or 1.
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-3.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/23/2020.01.29.925354/embed/graphic-4.gif"/></noscript></span>

</span></p><p id="p-31">Resulting partial contributions can be visualized along the IC of each nucleotide of a convolutional kernel. To this end, we design extended sequence logos, where each nucleotide is colored according to its contribution. Positive contributions are shown in red, negative contributions are blue, and near-zero contributions are gray. Therefore, no information is lost compared to standard sequence logos, but the relevance of individual nucleotides and the filter as a whole can be easily seen.</p></div><div id="sec-17" class="subsection"><h4>2.5. Genome-wide phenotype analysis</h4><p id="p-32">We create genome-wide phenotype analysis (GWPA) plots to analyse which parts of a viral genome are associated with the infectious phenotype. We scramble the genome into overlapping, 250bp long subsequences (pseudo-reads) without adding any sequencing noise. For the highest resolution, we use a stride of one nucleotide. We predict the infectious potential of each pseudo-read and average the obtained values at each position of the genome. Analogously, we calculate average contributions of each nucleotide to the final prediction of the convolutional network. We visualize the resulting nucleotide-resolution maps with IGV (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thorvaldsdóttir et al., 2013</a>).</p><p id="p-33">For well-annotated genomes, we compile a ranking of genes (or other genomic features) sorted by the average infectious potential within a given region. In addition to that, we scan the genome with the learned filters of the first convolutional layer to find genes enriched in subsequences yielding nonzero filter activations. We use Gene Ontology to connect the identified genes of interest with their molecular functions and biological processes they are engaged in.</p><p id="p-34">As a proof of concept, we analyze one of the viruses randomly assigned to the test set – the Taï Forest ebolavirus, which has a history of host-switching and can cause a serious disease. To show that the method can also be used for other biological problems, we investigated the networks trained by <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Bartoszewicz et al. (2019)</a> and their predictions on a genome of a pathogenic bacterium <em>Staphylococcus aureus.</em> The authors used this particular species to assess the performance of their method on real sequencing data. In this case, we used a stride of 125bp to generate the pseudo-reads. Finally, we analyzed the SARS-CoV-2 coronavirus, which emerged in December 2019, causing a pneumonia outbreak (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wu et al., 2020</a>).</p></div></div></div><div class="section" id="sec-18"><h2 class="">3. Results</h2><div id="sec-19" class="subsection"><h3>3.1. Negative class definition</h3><p id="p-35">Choosing which viruses should constitute the negative class is application dependent and influences the performance of the trained models. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the prediction accuracy for different combinations of the training and test set composition. The models trained only on human and Chordata-infecting viruses maintain similar, or even better performance when evaluated on viruses infecting a much broader host range, including bacteria. This suggests that the learned decision boundary separates human viruses from all the others surprisingly well. We hypothesize that the human host signal must be relatively strong and contained within the Chordata host signal. Dropout rate of 0.2 resulted in the highest validation accuracy for CNN<sub>Str-150</sub> and LSTM<sub>Str</sub>. A rate of 0.25 was selected for the other models.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1213596/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1213596/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1213596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-36" class="first-child">Classification accuracy depending on the negative class definition, read pairs. Euk. – Eukaryota dataset; Met. – Metazoa dataset, Cho. – Chordata dataset, Str. – Stratified dataset, X-150 – first 150 bp of each read in X. Training set in subscript of the model name; test set in column headers. Recall (Rec.) is identical in all cases, as the positive class remains unchanged. Best performance in bold. CNN<sub>All</sub> achieves best overall accuracy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Adding more diversity to the negative class may still boost performance on more diverse test sets, as in the case of CNN trained on the “All” dataset (CNN<sub>All</sub>). This model performs a bit worse on viruses infecting hosts related to humans, but achieves higher accuracy than the “Chordata”-trained models and the best recall overall. Rebalancing the negative class using the “Stratified” dataset helps to achieve higher performance on animal viruses while maintaing high overall accuracy. The LSTMs are outperformed by the CNNs, but they can be used for shorter reads without retraining (see <a id="xref-sec-12-1" class="xref-sec" href="#sec-12">Sections 2.2</a> and <a id="xref-sec-20-1" class="xref-sec" href="#sec-20">3.2</a>).</p></div><div id="sec-20" class="subsection"><h3>3.2. Prediction performance</h3><p id="p-38">We selected LSTMAll and CNNAll for further evaluation. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> presents the results of a benchmark using the “All” test set. Low performance of the <em>k</em>-NN classifier (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Zhang et al., 2019</a>) is caused by frequent conflicting predictions for each read in a read pair (in a single-read setting it achieves 75.5% accuracy, while our best model – 87.8%). Although BLAST achieves the highest precision, it yields no predictions for over 10% of the samples. CNNAll is the most sensitive and accurate (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1213575/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1213575/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1213575" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-39" class="first-child">Classification performance in the fully open-view setting (all virus hosts), read pairs. Acc. – accuracy, Prec. – precision, Rec. – recall. BLAST yields no predictions for over 10% of the samples. Best performance in bold.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1213589/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1213589/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1213589" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-40" class="first-child">Classification performance, all hosts. Whole available genomes. Negative class is the majority class. Rec. – recall, Spec. – specificity, BAcc. – balanced accuracy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We benchmarked our models against the human blood vi-rome dataset used by <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Zhang et al. (2019)</a>. Our models outperform their <em>k</em>-NN classifier. As the positive class massively outnumbers the negative class, all models achieve over 99% precision. CNN<sub>All-150</sub> performs best (Table. 4). However, the positive class is dominated by viruses which are not necessarily novel. The CNN was more accurate on training data, so we expected it to detect those viruses easily.</p></div><div id="sec-21" class="subsection"><h3>3.3. Filter visualization</h3><p id="p-42">In the <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a> we present example filters, visualized as “max-contrib” sequence logos based on mean partial Shapley values for each nucelotide at each position. All nucleotides of the filters with the highest (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>) or lowest (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a>) score have relatively strong contributions in accordance with the filters’ own contributions. However, we observe that sometimes, there is a “conserved” nucleotide which consistently appears in the activating subsequences, but the sign of its contributions is opposite to the filter’s (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1c</a>). Those “counter-contributions” may arise if a nucleotide with a negative weight forms a frequent motif with others with positive weights strong enough to activate the filter. We comment on this fact in the <a id="xref-sec-25-1" class="xref-sec" href="#sec-25">Section 4.2</a>. Some filters seem to learn gapped motifs resembling a codon structure (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1d</a>). We extracted this filter from a network predicting bacterial pathogenicity trained by <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Bartoszewicz et al. (2019)</a>, but we find similar filters in our networks as well. We scanned a genome of <em>S. aureus</em> with this filter and discovered that the learned motif is indeed significantly enriched in coding sequences (Fisher exact test with Benjamini-Hochberg correction, <em>q</em> &lt; 10<sup>-15</sup>). It is also enriched in a number of specific genes. The one with the most hits (sraP, <em>q</em> &lt; 10<sup>-15</sup>) is associated with virulence in endovascular infection.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleotide contribution logos of example filters. 1a: Highest mean contribution score (CNNAll). 1b: Lowest mean contribution score (CNNAll). 1c: Local counter-contributions (CNNStr-150). 1d: Gaps resembling a codon structure, extracted from Bartoszewicz et al. (2019)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919871314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nucleotide contribution logos of example filters. 1a: Highest mean contribution score (CNN&lt;sub&gt;All&lt;/sub&gt;). 1b: Lowest mean contribution score (CNN&lt;sub&gt;All&lt;/sub&gt;). 1c: Local counter-contributions (CNN&lt;sub&gt;Str-150&lt;/sub&gt;). 1d: Gaps resembling a codon structure, extracted from Bartoszewicz et al. (2019)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F1.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F1.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-43" class="first-child">Nucleotide contribution logos of example filters. 1a: Highest mean contribution score (CNN<sub>All</sub>). 1b: Lowest mean contribution score (CNN<sub>All</sub>). 1c: Local counter-contributions (CNN<sub>Str-150</sub>). 1d: Gaps resembling a codon structure, extracted from <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Bartoszewicz et al. (2019)</a></p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>3.4. Genome-wide phenotype analysis</h3><p id="p-44">We created a GWPA plot for the Taï Forest ebolavirus genome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>). Most genes can be detected by finding peaks of elevated pathogenic potential score predicted by at least one of the models. Intergenic regions are characterized by lower mean scores.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ta&#xEF; Forest ebolavirus and SARS-CoV-2 coronavirus genomes. Top: score predicted by LSTMAll. Middle: score predicted by CNNAII. Heatmap: nucleotide contributions of CNNAII. Bottom, in blue: reference sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919871314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ta&#xEF; Forest ebolavirus and SARS-CoV-2 coronavirus genomes. Top: score predicted by LSTM&lt;sub&gt;All&lt;/sub&gt;. Middle: score predicted by CNN&lt;sub&gt;AII&lt;/sub&gt;. Heatmap: nucleotide contributions of CNN&lt;sub&gt;AII&lt;/sub&gt;. Bottom, in blue: reference sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F2.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F2.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/23/2020.01.29.925354/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-45" class="first-child">Taï Forest ebolavirus and SARS-CoV-2 coronavirus genomes. Top: score predicted by LSTM<sub>All</sub>. Middle: score predicted by CNN<sub>AII</sub>. Heatmap: nucleotide contributions of CNN<sub>AII</sub>. Bottom, in blue: reference sequence.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1213590/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1213590/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1213590" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-46" class="first-child">Classification performance on the human blood virome dataset. Positive class is the majority class. Rec. – recall, Spec. – specificity, BAcc. – balanced accuracy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">We ran a similar analysis of <em>S. aureus</em> using the built-in DeePaC models (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Bartoszewicz et al., 2019</a>) and our interpretation workflow. While a viral genome contains usually only a handful of genes, by compiling a ranking of 870 annotated genes of the analyzed <em>S. aureus</em> strain we could test if the high-ranking regions are indeed associated with pathogenicity. Indeed, out of three top-ranking genes, sarR and sspB are directly engaged in virulence, while hupB regulates expression of virulence-involved genes in many pathogens (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stojkova et al., 2019</a>).</p><p id="p-48"><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a> presents a GWPA plot for the whole genome of the SARS-CoV-2 coronavirus. We highlighted the score peaks aligning with the gene encoding the spike protein, which plays a significant role in host entry (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li, 2016</a>), as well as the E and N genes, which were scored the highest (apart from an unconfirmed ORF10 of just 38aa downstream of N) by the CNN and the LSTM, respectively. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2c</a> shows the nucleotide-level contributions in a small peak within the receptor-binding domain of the S protein, crucial for recognizing the host cell. The domain location was predicted with CD-search (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marchler-Bauer et al., 2017</a>). While this could suggest a host adaptation, more research is needed.</p></div></div><div class="section" id="sec-23"><h2 class="">4. Discussion</h2><div id="sec-24" class="subsection"><h3>4.1. Accurate predictions from short DNA reads</h3><p id="p-49">Compared to the previous state-of-the-art in viral host prediction directly from next-generation sequencing reads (<a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Zhang et al., 2019</a>), our models drastically reduce the error rates. This holds also for novel viruses not present in the training set. In the paired read scenario, the previously described method fails, and standard, alignment-based homology testing algorithm cannot find any matches in more than 10% of the cases, resulting in relatively low accuracy. On a real human virome sample, where a main source of negative class reads is most likely contamination (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Moustafa et al., 2017</a>), our method filters out non-human viruses with high specificity. In this scenario, the BLAST-derived groundtruth labels were mined using the complete database (as opposed to just a training set). In all cases, our results are only as good as the training data used; high quality labels and sequences are needed to develop trustworthy models. Ideally, sources of error should be investigated with an in-depth analysis of a model’s performance on multiple genomes covering a wide selection of taxonomic units. This is especially important as the method assumes no mechanistic link between an input sequence and the phenotype of interest, and the input sequence constitutes only a small fraction of the target genome without a wider biological context. Still, it is possible to predict a label even from those small, local fragments. A similar effect was also observed for image classification with CNNs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brendel &amp; Bethge, 2019</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1213593/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1213593/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1213593" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5.</span> <p id="p-50" class="first-child">Gene ranking for S. <em>aureus</em> (top 3 out of 870). hupB is indirectly engaged in virulence. Our method detects functionally relevant genes using the model of <a id="xref-ref-6-10" class="xref-bibr" href="#ref-6">Bartoszewicz et al. (2019)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>4.2. Nucleotide contribution logos</h3><p id="p-51">Visualizing convolutional filters may help to identify potential problems. If the input data contains Ns, a ReLU network may learn only negative weights at some positions, resulting in counter-contributions for all possible nucleotides at those positions. In our case, as the filters were weight-normalized, the counter-contributions suggest that the information content and the contribution of a nucleotide are not necessarily correlated. Visualizing learned motifs by aligning the activating sequences (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alipanahi et al., 2015</a>) would not fully describe how the filter reacts to presented data. It seems that the assumption of nucleotide independence – which is crucial for treating DeepLIFT as a method of estimating Shapley values for input nucleotides – is broken. Indeed, <em>k</em>-mer distribution profiles are frequently used features for modelling DNA sequences (as shown also by the dimershuffling method of generating reference sequences proposed by <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Shrikumar et al. (2019a)</a>). However, DeepLIFT’s multiple successful applications in genomics indicate that the assumption probably holds approximately. We see information content and DeepLIFT’s contribution values as two complementary channels that can be jointly visualized for better interpretability and explainability of CNNs in genomics.</p></div><div id="sec-26" class="subsection"><h3>4.3. Genome-scale interpretability</h3><p id="p-52">Mapping predictions back to a target genome can be used both as a way of investigating a given model’s performance and as a method of genome analysis. GWPA plots of well-annotated genomes highlight the sequences with erroneous and correct phenotype predictions at both genome and gene level, and nucleotide-resolution contribution maps help track those regions down to individual amino-acids. On the other hand, once a trusted model is developed, it can be used on newly emerging pathogens, as the SARS-CoV-2 virus briefly analyzed in this work. The methods presented here may also be applied to other biological problems outside of the field of pathogen genomics. However, experimental work and traditional sequence analysis are required to truly understand the biology behind host adaptation and distinguish true hits from false positives.</p></div><div id="sec-27" class="subsection"><h3>4.4. Conclusion</h3><p id="p-53">We presented a new approach for predicting a host of a novel virus based on a single DNA read or a read pair, cutting the error rates in half compared to the previous state-of-the-art. For convolutional filters, we jointly visualize nucleotide contributions and information content. Finally, we use GWPA plots to gain insights into the models’ behavior and analyze a recently emerged SARS-CoV-2 virus. Data is available at <a href="https://doi.org/10.5281/zenodo.3630803">https://doi.org/10.5281/zenodo.3630803</a> and the code is submitted in Supplementary Material.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-54">We gratefully acknowledge Yong-Zhen Zhang and the scientists at the Shanghai Public Health Clinical Center &amp; School of Public Health, Fudan University, who shared the sequence of the 2019-nCoV virus ahead of publication. We thank Melania Nowicka (Max Plank Institute for Molecular Genetics) for inspiring discussions on efficient calculations of partial Shapley values and Lothar H. Wieler (Robert Koch Institute) for useful comments on the first draft of the manuscript. This work was supported by the German Academic Scholarship Foundation (JMB) and the BMBF-funded de.NBI Cloud within the German Network for Bioinformatics Infrastructure (de.NBI) (031A537B, 031A533A, 031A538A, 031A533B, 031A535A, 031A537C, 031A534A, 031A532B).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.1" data-doi="10.1038/nbt.3300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alipanahi</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delong</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weirauch</span>,  <span class="cit-name-given-names">M. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Frey</span>,  <span class="cit-name-given-names">B. J.</span></span> <span class="cit-article-title">Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">831</span>–<span class="cit-lpage">838</span>, <span class="cit-pub-date">2015</span>. ISSN <span class="cit-issn">1546-1696</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.3300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D33%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3300%26rft_id%253Dinfo%253Apmid%252F26213851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26213851&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">E. W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>, <span class="cit-vol">215</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>, <span class="cit-pub-date">1990</span>. ISSN <span class="cit-issn">0022-2836</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0022-2836(05)80360-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.3" data-doi="10.1101/737981"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Avsec</span>,  <span class="cit-name-given-names">Ž.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weilert</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shrikumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexandari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krueger</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalal</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fropf</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAnany</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gagneur</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kundaje</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zeitlinger</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Deep learning at baseresolution reveals motif syntax of the cis-regulatory code</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, pp. <span class="cit-fpage">737981</span>, <span class="cit-month">August</span> <span class="cit-pub-date">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/737981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F737981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI3Mzc5ODF2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.4" data-doi="10.1126/science.aap9072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babayan</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orton</span>,  <span class="cit-name-given-names">R. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> <span class="cit-article-title">Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">362</span>(<span class="cit-issue">6414</span>): <span class="cit-fpage">577</span>–<span class="cit-lpage">580</span>, <span class="cit-month">November</span> <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">0036-8075, 1095-9203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aap9072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBabayan%26rft.auinit1%253DS.%2BA.%26rft.volume%253D362%26rft.issue%253D6414%26rft.spage%253D577%26rft.epage%253D580%26rft.atitle%253DPredicting%2Breservoir%2Bhosts%2Band%2Barthropod%2Bvectors%2Bfrom%2Bevolutionary%2Bsignatures%2Bin%2BRNA%2Bvirus%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap9072%26rft_id%253Dinfo%253Apmid%252F30385576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNC81NzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMy8yMDIwLjAxLjI5LjkyNTM1NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.5" data-doi="10.1371/journal.pone.0130140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bach</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binder</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montavon</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klauschen</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">K.-R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Samek</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">On Pixel-Wise Explanations for Non-Linear Classifier Decisions by Layer-Wise Relevance Propagation</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e0130140</span>, <span class="cit-month">July</span> <span class="cit-pub-date">2015</span>. ISSN <span class="cit-issn">1932-6203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0130140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0130140%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0130140%26rft_id%253Dinfo%253Apmid%252F26161953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0130140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26161953&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.6" data-doi="10.1093/bioinformatics/btz541"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bartoszewicz</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seidel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rentzsch</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">DeePaC: predicting pathogenic potential of novel DNA with reverse-complement neural networks</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>, <span class="cit-page-range">07</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1367-4803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btz541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtz541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btz541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brendel</span>,  <span class="cit-name-given-names">W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bethge</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Approximating CNNs with bag-of-local-features models works surprisingly well on ImageNet</span>. In <abbr class="cit-jnl-abbrev">International Conference on Learning Representations</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.8" data-doi="10.1093/cid/ciu913"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calistri</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Palù</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Editorial commentary: Unbiased next-generation sequencing and new pathogen discovery: undeniable advantages and still-existing drawbacks</span>. <abbr class="cit-jnl-abbrev">Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">889</span>–<span class="cit-lpage">891</span>, <span class="cit-pub-date">2015</span>. ISSN <span class="cit-issn">1537-6591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/cid/ciu913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BInfectious%2BDiseases%253A%2BAn%2BOfficial%2BPublication%2Bof%2Bthe%2BInfectious%2BDiseases%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciu913%26rft_id%253Dinfo%253Apmid%252F25572900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciu913&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572900&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulze</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zickmann</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">Clock rooting further demonstrates that guinea 2014 ebov is a member of the zaïre lineage</span>. <abbr class="cit-jnl-abbrev">PLoS currents</abbr>, <span class="cit-fpage">6</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.10" data-doi="10.1101/gr.849004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crooks</span>,  <span class="cit-name-given-names">G. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hon</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandonia</span>,  <span class="cit-name-given-names">J.-M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brenner</span>,  <span class="cit-name-given-names">S. E.</span></span> <span class="cit-article-title">WebLogo: a sequence logo generator</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>, <span class="cit-month">June</span> <span class="cit-pub-date">2004</span>. ISSN <span class="cit-issn">1088-9051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.849004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.11" data-doi="10.1093/femsre/fuv048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McNair</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faust</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raes</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dutilh</span>,  <span class="cit-name-given-names">B. E.</span></span> <span class="cit-article-title">Computational approaches to predict bacteriophagehost relationships</span>. <abbr class="cit-jnl-abbrev">FEMS microbiology reviews</abbr>, <span class="cit-vol">40</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">258</span>–<span class="cit-lpage">272</span>, <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1574-6976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/femsre/fuv048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2Bmicrobiology%2Breviews%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsre%252Ffuv048%26rft_id%253Dinfo%253Apmid%252F26657537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsre/fuv048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26657537&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.12" data-doi="10.1186/1755-8794-7-S3-S1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eng</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">J. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">T. W.</span></span> <span class="cit-article-title">Predicting host tropism of influenza a virus proteins using random forest</span>. <abbr class="cit-jnl-abbrev">BMC Medical Genomics</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">S1</span>, <span class="cit-pub-date">2014</span>. ISSN <span class="cit-issn">1755-8794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1755-8794-7-S3-S1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMedical%2BGenomics%26rft.volume%253D7%26rft.spage%253DS1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1755-8794-7-S3-S1%26rft_id%253Dinfo%253Apmid%252F25079003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1755-8794-7-S3-S1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25079003&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.13" data-doi="10.1038/s41576-019-0122-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eraslan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avsec</span>,  <span class="cit-name-given-names">Ž.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gagneur</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Theis</span>,  <span class="cit-name-given-names">F. J.</span></span> <span class="cit-article-title">Deep learning: new computational modelling techniques for genomics</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">389</span>–<span class="cit-lpage">403</span>, <span class="cit-month">July</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1471-0064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41576-019-0122-6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-019-0122-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41576-019-0122-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.14" data-doi="10.1038/s41598-019-39847-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gałan</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bąk</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jakubowska</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Host taxon predictor - a tool for predicting taxon of the host of a newly discovered virus</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">3436</span>, <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">2045-2322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-019-39847-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D9%26rft.spage%253D3436%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-39847-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-019-39847-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.15" data-doi="10.1093/bioinformatics/bty575"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greenside</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimko</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fordyce</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kundaje</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Discovering epistatic feature interactions from neural network models of regulatory DNA sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">i629</span>–<span class="cit-lpage">i637</span>, <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">1367-4803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bty575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di629%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.16" data-doi="10.1126/science.1213362"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herfst</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schrauwen</span>,  <span class="cit-name-given-names">E. J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linster</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chutinimitkul</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wit</span>,  <span class="cit-name-given-names">E. d.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorrell</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rimmelzwaan</span>,  <span class="cit-name-given-names">G. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>,  <span class="cit-name-given-names">A. D. M. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span> <span class="cit-article-title">Airborne Transmission of Influenza A/H5n1 Virus Between Ferrets</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">336</span>(<span class="cit-issue">6088</span>):<span class="cit-fpage">1534</span>–<span class="cit-lpage">1541</span>, <span class="cit-month">June</span> <span class="cit-pub-date">2012</span>. ISSN <span class="cit-issn">0036-8075, 1095-9203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1213362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHerfst%26rft.auinit1%253DS.%26rft.volume%253D336%26rft.issue%253D6088%26rft.spage%253D1534%26rft.epage%253D1541%26rft.atitle%253DAirborne%2BTransmission%2Bof%2BInfluenza%2BA%252FH5N1%2BVirus%2BBetween%2BFerrets%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1213362%26rft_id%253Dinfo%253Apmid%252F22723413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzYvNjA4OC8xNTM0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjMvMjAyMC4wMS4yOS45MjUzNTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925354v3.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holtgrewe</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Mason – a read simulator for second generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Technical Report FU Berlin</abbr>, <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.18" data-doi="10.1038/nature10831"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Imai</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatta</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Das</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shinya</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katsura</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawakami</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamada</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiso</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maher</span>,  <span class="cit-name-given-names">E. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neumann</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kawaoka</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Experimental adaptation of an influenza H5 HA confers respiratory droplet transmission to a reassortant H5 HA/H1n1 virus in ferrets</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">486</span>(<span class="cit-issue">7403</span>):<span class="cit-fpage">420</span>–<span class="cit-lpage">428</span>, <span class="cit-month">June</span> <span class="cit-pub-date">2012</span>. ISSN <span class="cit-issn">1476-4687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature10831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DImai%26rft.auinit1%253DM.%26rft.volume%253D486%26rft.issue%253D7403%26rft.spage%253D420%26rft.epage%253D428%26rft.atitle%253DExperimental%2Badaptation%2Bof%2Ban%2Binfluenza%2BH5%2BHA%2Bconfers%2Brespiratory%2Bdroplet%2Btransmission%2Bto%2Ba%2Breassortant%2BH5%2BHA%252FH1N1%2Bvirus%2Bin%2Bferrets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10831%26rft_id%253Dinfo%253Apmid%252F22722205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22722205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305466800047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.19" data-doi="10.1101/700096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aicher</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Barash</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Improving interpretability of deep learning models: splicing codes as a case study</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/700096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F700096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI3MDAwOTZ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.20" data-doi="10.1101/gr.200535.115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snoek</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rinn</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-article-title">Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">26</span> (<span class="cit-issue">7</span>):<span class="cit-fpage">990</span>–<span class="cit-lpage">999</span>, <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1088-9051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.200535.115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.200535.115%26rft_id%253Dinfo%253Apmid%252F27197224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzcvOTkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjMvMjAyMC4wMS4yOS45MjUzNTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925354v3.21" data-doi="abs/1605.01133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanchantin</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Z.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Deep Motif: Visualizing Genomic Sequence Classifications</span>. <abbr class="cit-jnl-abbrev">CoRR</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>abs/1605.01133</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCoRR%26rft_id%253Dinfo%253Adoi%252Fabs%252F1605.01133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=abs/1605.01133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925354v3.22" data-doi="10.1142/9789813207813_0025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanchantin</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Deep motif dashboard: visualizing and understanding genomic sequences using deep neural networks</span>. <abbr class="cit-jnl-abbrev">Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</abbr>, <span class="cit-vol">22</span>:<span class="cit-fpage">254</span>–<span class="cit-lpage">265</span>, <span class="cit-pub-date">2017</span>. ISSN <span class="cit-issn">2335-6936</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1142/9789813207813_0025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPacific%2BSymposium%2Bon%2BBiocomputing.%2BPacific%2BSymposium%2Bon%2BBiocomputing%26rft.volume%253D22%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.1142%252F9789813207813_0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/9789813207813_0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.23" data-doi="10.3389/fcimb.2014.00025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lecuit</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Eloit</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The diagnosis of infectious diseases by whole genome next generation sequencing: a new era is opening</span>. <abbr class="cit-jnl-abbrev">Frontiers in Cellular and Infection Microbiology</abbr>, <span class="cit-vol">4</span>:<span class="cit-fpage">25</span>, <span class="cit-pub-date">2014</span>. ISSN <span class="cit-issn">2235-2988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3389/fcimb.2014.00025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BCellular%2Band%2BInfection%2BMicrobiology%26rft.volume%253D4%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffcimb.2014.00025%26rft_id%253Dinfo%253Apmid%252F24639952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fcimb.2014.00025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24639952&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925354v3.24" data-doi="10.1007/s10393-015-1053-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leendertz</span>,  <span class="cit-name-given-names">S. A. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gogarten</span>,  <span class="cit-name-given-names">J. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Düx</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Leendertz</span>,  <span class="cit-name-given-names">F. H.</span></span> <span class="cit-article-title">Assessing the evidence supporting fruit bats as the primary reservoirs for ebola viruses</span>. <abbr class="cit-jnl-abbrev">EcoHealth</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">18</span>–<span class="cit-lpage">25</span>, <span class="cit-month">Mar</span> <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1612-9210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10393-015-1053-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D13%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10393-015-1053-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10393-015-1053-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.25" data-doi="10.1146/annurev-virology-110615-042301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">237</span>–<span class="cit-lpage">261</span>, <span class="cit-pub-date">2016</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-110615-042301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft_id%253Dinfo%253Apmid%252F27578435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27578435&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.26" data-doi="10.1038/s41598-018-28308-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">10032</span>, <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">2045-2322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-28308-x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D10032%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-28308-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-28308-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.27" data-doi="10.1128/mBio.02366-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lipsitch</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Inglesby</span>,  <span class="cit-name-given-names">T. V.</span></span> <span class="cit-article-title">Moratorium on Research Intended To Create Novel Potential Pandemic Pathogens</span>. <abbr class="cit-jnl-abbrev">mBio</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">6</span>), <span class="cit-month">December</span> <span class="cit-pub-date">2014</span>. ISSN <span class="cit-issn">2150-7511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.02366-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.02366-14%26rft_id%253Dinfo%253Apmid%252F25505122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiNS82L2UwMjM2Ni0xNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925354v3.28"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Guyon</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Luxburg</span>,  <span class="cit-name-given-names">U. V.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Bengio</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Wallach</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Fergus</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Vishwanathan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-ed"><span class="cit-name-surname">Garnett</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Lundberg</span>,  <span class="cit-name-given-names">S. M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.-I.</span></span> <span class="cit-chapter-title">A Unified Approach to Interpreting Model Predictions</span>. In <span class="cit-ed"><span class="cit-name-surname">Guyon</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Luxburg</span>,  <span class="cit-name-given-names">U. V.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bengio</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wallach</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fergus</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Vishwanathan</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-ed"><span class="cit-name-surname">Garnett</span>,  <span class="cit-name-given-names">R.</span></span> (eds.), <span class="cit-source">Advances in Neural Information Processing Systems</span> <span class="cit-vol">30</span>, pp. <span class="cit-fpage">4765</span>–<span class="cit-lpage">4774</span>. <span class="cit-publ-name">Curran Associates, Inc</span>., <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2BNeural%2BInformation%2BProcessing%2BSystems%26rft.volume%253D30%26rft.spage%253D4765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.29" data-doi="10.1093/nar/gkw1129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanczycki</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitsaz</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geer</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzales</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gwadz</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurwitz</span>,  <span class="cit-name-given-names">D. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchler</span>,  <span class="cit-name-given-names">G. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thanki</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamashita</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geer</span>,  <span class="cit-name-given-names">L. Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bryant</span>,  <span class="cit-name-given-names">S. H.</span></span> <span class="cit-article-title">CDD/SPARCLE: functional classification of proteins via subfamily domain architectures</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D200</span>–<span class="cit-lpage">D203</span>, <span class="cit-pub-date">2017</span>. ISSN <span class="cit-issn">1362-4962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1129%26rft_id%253Dinfo%253Apmid%252F27899674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.30" data-doi="10.3390/v8030066"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mihara</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishimura</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimizu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishiyama</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uehara</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hingamp</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goto</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ogata</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Linking virus genomes with host taxonomy</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">66</span>, <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1999-4915</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3390/v8030066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D8%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv8030066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v8030066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.31" data-doi="10.1101/575571"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mock</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viehweger</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barth</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marz</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Viral host prediction with deep learning</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, pp. <span class="cit-fpage">575571</span>, <span class="cit-pub-date">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/575571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F575571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1NzU1NzF2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.32" data-doi="10.1371/journal.ppat.1006292"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moustafa</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirkness</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Biggs</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turpaz</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">K. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venter</span>,  <span class="cit-name-given-names">J. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Telenti</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">The blood DNA virome in 8,000 humans</span>. <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e1006292</span>, <span class="cit-month">March</span> <span class="cit-pub-date">2017</span>. ISSN <span class="cit-issn">1553-7374</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D13%26rft.spage%253De1006292%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.33" data-doi="10.1093/bioinformatics/btz352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nair</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perricone</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kundaje</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Integrating regulatory DNA sequence and gene expression to predict genome-wide chromatin accessibility across cellular contexts</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">14</span>):<span class="cit-fpage">i108</span>–<span class="cit-lpage">i116</span>, <span class="cit-month">July</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1367-4803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btz352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253Di108%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtz352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btz352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.34" data-doi="10.1371/journal.pone.0188453"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Noyce</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lederman</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">D. H.</span></span> <span class="cit-article-title">Construction of an infectious horsepox virus vaccine from chemically synthesized DNA fragments</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">13</span> (<span class="cit-issue">1</span>):<span class="cit-fpage">e0188453</span>, <span class="cit-month">January</span> <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">1932-6203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0188453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D13%26rft.spage%253De0188453%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0188453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0188453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.35" data-doi="10.1038/nbt.4235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poplin</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">P.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colthurst</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ku</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newburger</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dijamco</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Afshar</span>,  <span class="cit-name-given-names">P. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gross</span>,  <span class="cit-name-given-names">S. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dorfman</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLean</span>,  <span class="cit-name-given-names">C. Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-article-title">A universal SNP and small-indel variant caller using deep neural networks</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">983</span>–<span class="cit-lpage">987</span>, <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">1546-1696</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.4235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.36" data-doi="10.1093/nar/gkw226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quang</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-article-title">DanQ: a hybrid convolutional and recurrent deep neural network for quantifying the function of DNA sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">44</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">e107</span>–<span class="cit-lpage">e107</span>, <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">0305-1048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw226%26rft_id%253Dinfo%253Apmid%252F27084946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27084946&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahlgren</span>,  <span class="cit-name-given-names">N. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuhrman</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Identifying viruses from metagenomic data by deep learning</span>. <abbr class="cit-jnl-abbrev">arXiv:1806.07810 [q-bio]</abbr>, <span class="cit-month">June</span> <span class="cit-pub-date">2018</span>.<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv: </span><span class="cit-pub-id cit-pub-id-arxiv">1806.07810</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.38" data-doi="10.1093/nar/18.20.6097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">T. D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-article-title">Sequence logos: a new way to display consensus sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">18</span>(<span class="cit-issue">20</span>):<span class="cit-fpage">6097</span>–<span class="cit-lpage">6100</span>, <span class="cit-month">October</span> <span class="cit-pub-date">1990</span>. ISSN <span class="cit-issn">0305-1048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/18.20.6097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSchneider%26rft.auinit1%253DT.%2BD.%26rft.volume%253D18%26rft.issue%253D20%26rft.spage%253D6097%26rft.epage%253D6100%26rft.atitle%253DSequence%2Blogos%253A%2Ba%2Bnew%2Bway%2Bto%2Bdisplay%2Bconsensus%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.20.6097%26rft_id%253Dinfo%253Apmid%252F2172928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.20.6097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2172928&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EF12100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shrikumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenside</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kundaje</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Learning Important Features Through Propagating Activation Differences</span>. <abbr class="cit-jnl-abbrev">arXiv:1704.02685 [cs]</abbr>, <span class="cit-month">October</span> <span class="cit-pub-date">2019a</span>.<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv: </span><span class="cit-pub-id cit-pub-id-arxiv">1704.02685</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shrikumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shcherbina</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avsec</span>,  <span class="cit-name-given-names">Ž.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharmin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nair</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kundaje</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">TF-MoDISco v0.4.2.2-alpha: Technical Note</span>. <abbr class="cit-jnl-abbrev">arXiv:1811.00416 [cs, q-bio, stat]</abbr>, <span class="cit-month">March</span> <span class="cit-pub-date">2019b</span>.<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv: </span><span class="cit-pub-id cit-pub-id-arxiv">1811.00416</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.41" data-doi="10.3389/fcimb.2019.00159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stojkova</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spidlova</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stulik</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Nucleoid-associated protein hu: A lilliputian in gene regulation of bacterial virulence</span>. <abbr class="cit-jnl-abbrev">Frontiers in Cellular and Infection Microbiology</abbr>, <span class="cit-vol">9</span>:<span class="cit-fpage">159</span>, <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">2235-2988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3389/fcimb.2019.00159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BCellular%2Band%2BInfection%2BMicrobiology%26rft.volume%253D9%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffcimb.2019.00159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fcimb.2019.00159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.42" data-doi="abs/1611.02639"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sundararajan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taly</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">Q.</span></span> <span class="cit-article-title">Gradients of Coun-terfactuals</span>. <abbr class="cit-jnl-abbrev">CoRR</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>abs/1611.02639</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCoRR%26rft_id%253Dinfo%253Adoi%252Fabs%252F1611.02639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=abs/1611.02639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.43" data-doi="10.1371/journal.pone.0222271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tampuu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bzhalava</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dillner</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Vicente</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Vi-raMiner: Deep learning on raw DNA sequences for identifying viral genomes in human samples</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">14</span> (<span class="cit-issue">9</span>):<span class="cit-fpage">e0222271</span>, <span class="cit-month">September</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1932-6203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0222271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D14%26rft.spage%253De0222271%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0222271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0222271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.44" data-doi="10.1371/journal.ppat.1007019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thiel</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Synthetic viruses-Anything new?</span> <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1007019</span>, <span class="cit-pub-date">2018</span>. ISSN <span class="cit-issn">1553-7374</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1007019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D14%26rft.spage%253De1007019%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.45" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mesirov</span>,  <span class="cit-name-given-names">J. P.</span></span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</span>. <abbr class="cit-jnl-abbrev">Briefings in Bioinformatics</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>, <span class="cit-month">March</span> <span class="cit-pub-date">2013</span>. ISSN <span class="cit-issn">1467-5463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bib/bbs017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.46" data-doi="10.1093/bioinformatics/btw423"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trappe</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marschall</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">Detecting horizontal gene transfer by mapping sequencing reads across species boundaries</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">i595</span>–<span class="cit-lpage">i604</span>, <span class="cit-month">September</span> <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1367-4803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btw423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw423%26rft_id%253Dinfo%253Apmid%252F27587679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27587679&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.47" data-doi="10.1016/j.cmi.2015.10.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vouga</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Greub</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Emerging bacterial pathogens: the past and beyond</span>. <abbr class="cit-jnl-abbrev">Clinical Microbiology and Infection</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">12</span>–<span class="cit-lpage">21</span>, <span class="cit-month">January</span> <span class="cit-pub-date">2016</span>. ISSN 1198-743X.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cmi.2015.10.010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2Band%2BInfection%26rft.volume%253D22%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmi.2015.10.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmi.2015.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.48" data-doi="10.1101/2020.01.24.919183"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span> <span class="cit-article-title">Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, pp. <span class="cit-fpage">2020</span>. 01.24.919183, <span class="cit-month">January</span> <span class="cit-pub-date">2020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.24.919183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.24.919183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNC45MTkxODN2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIzLzIwMjAuMDEuMjkuOTI1MzU0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.49" data-doi="10.1093/bioinformatics/btw255"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gifford</span>,  <span class="cit-name-given-names">D. K.</span></span> <span class="cit-article-title">Convolutional neural network architectures for predicting DNA–protein binding</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">i121</span>–<span class="cit-lpage">i127</span>, <span class="cit-pub-date">2016</span>. ISSN <span class="cit-issn">1367-4803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btw255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw255%26rft_id%253Dinfo%253Apmid%252F27307608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27307608&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.50" data-doi="10.1111/tbed.13314"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Rapid identification of human-infecting viruses</span>. <abbr class="cit-jnl-abbrev">Transboundary and Emerging Diseases</abbr>, <span class="cit-month">August</span> <span class="cit-pub-date">2019</span>. ISSN <span class="cit-issn">1865-1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/tbed.13314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransboundary%2Band%2BEmerging%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftbed.13314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tbed.13314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925354v3.51" data-doi="10.1038/nmeth.3547"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>,  <span class="cit-name-given-names">O. G.</span></span> <span class="cit-article-title">Predicting effects of noncoding variants with deep learning–based sequence model</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">931</span>–<span class="cit-lpage">934</span>, <span class="cit-pub-date">2015</span>. ISSN <span class="cit-issn">1548-7105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.3547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D12%26rft.spage%253D931%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3547%26rft_id%253Dinfo%253Apmid%252F26301843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26301843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F23%2F2020.01.29.925354.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.29.925354v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.20.998203v2" title="Genome regulation and gene interaction networks inferred from muscle transcriptome underlying feed efficiency in Pigs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/867572v2" title="Differential tolerance of Zymoseptoria tritici to altered optimal moisture conditions during the early stages of wheat infection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 23, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.29.925354v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.29.925354v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.29.925354v3" class="active" data-icon-position="" data-hide-link-title="0">Interpretable detection of novel human viruses from genome sequencing data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1208641" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Riol2jddwi9lwN5sKpbYjFNG8McGfRoiB0MnBHLnpOo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208641" id="node1208641--2" data-pisa="biorxiv;2020.01.29.925354v3" data-pisa-master="biorxiv;2020.01.29.925354" data-seqnum="1208641" data-apath="/biorxiv/early/2020/03/23/2020.01.29.925354.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Interpretable detection of novel human viruses from genome sequencing data</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jakub M.</span> <span class="nlm-surname">Bartoszewicz</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anja</span> <span class="nlm-surname">Seidel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.29.925354; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.29.925354 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;title=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;title=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;text=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;title=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;t=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;title=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;title=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1208641" id="node1208641--3" data-pisa="biorxiv;2020.01.29.925354v3" data-pisa-master="biorxiv;2020.01.29.925354" data-seqnum="1208641" data-apath="/biorxiv/early/2020/03/23/2020.01.29.925354.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Interpretable detection of novel human viruses from genome sequencing data</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jakub M.</span> <span class="nlm-surname">Bartoszewicz</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anja</span> <span class="nlm-surname">Seidel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.29.925354; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.29.925354 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Bartoszewicz%2C+Jakub+M.&amp;rft.au=Seidel%2C+Anja&amp;rft.au=Renard%2C+Bernhard+Y."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1208641/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1208641/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1208641/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1208641/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1208641/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1208641/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1208641/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1208641/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1208641/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1208641/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1208641/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1208641/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;count=horizontal&amp;via=&amp;text=Interpretable%20detection%20of%20novel%20human%20viruses%20from%20genome%20sequencing%20data&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925354v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.925354v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
